
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            The genetic diversity and populational specificity of the human gut virome at single-nucleotide resolution - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AF818AF31EA305AF81001F1BD9CD.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="microbio">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369081/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Microbiome">
<meta name="citation_title" content="The genetic diversity and populational specificity of the human gut virome at single-nucleotide resolution">
<meta name="citation_author" content="Xiuchao Wang">
<meta name="citation_author_institution" content="Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China">
<meta name="citation_author_institution" content="Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China">
<meta name="citation_author" content="Quanbin Dong">
<meta name="citation_author_institution" content="Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China">
<meta name="citation_author_institution" content="Cardiovascular Research Center, The Affiliated Suzhou Hospital of Nanjing Medical University, Nanjing Medical University, Suzhou, China">
<meta name="citation_author" content="Pan Huang">
<meta name="citation_author_institution" content="State Key Laboratory of Food Science and Resources, Jiangnan University, Wuxi, China">
<meta name="citation_author_institution" content="School of Food Science and Technology, Jiangnan University, Wuxi, China">
<meta name="citation_author" content="Shuai Yang">
<meta name="citation_author_institution" content="Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China">
<meta name="citation_author_institution" content="Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China">
<meta name="citation_author" content="Mengke Gao">
<meta name="citation_author_institution" content="Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China">
<meta name="citation_author" content="Chengcheng Zhang">
<meta name="citation_author_institution" content="State Key Laboratory of Food Science and Resources, Jiangnan University, Wuxi, China">
<meta name="citation_author_institution" content="School of Food Science and Technology, Jiangnan University, Wuxi, China">
<meta name="citation_author" content="Chuan Zhang">
<meta name="citation_author_institution" content="State Key Laboratory of Food Science and Resources, Jiangnan University, Wuxi, China">
<meta name="citation_author_institution" content="School of Food Science and Technology, Jiangnan University, Wuxi, China">
<meta name="citation_author" content="Youpeng Deng">
<meta name="citation_author_institution" content="National Key Laboratory of Immune Response and Immunotherapy, Division of Life Sciences and Medicine, University of Science and Technology of China, Hefei, Anhui China">
<meta name="citation_author_institution" content="Center for Advanced Interdisciplinary Science and Biomedicine of IHM and Department of Infectious Diseases, The First Affiliated Hospital of USTC, University of Science and Technology of China, Hefei, Anhui China">
<meta name="citation_author" content="Zijing Huang">
<meta name="citation_author_institution" content="National Key Laboratory of Immune Response and Immunotherapy, Division of Life Sciences and Medicine, University of Science and Technology of China, Hefei, Anhui China">
<meta name="citation_author_institution" content="Center for Advanced Interdisciplinary Science and Biomedicine of IHM and Department of Infectious Diseases, The First Affiliated Hospital of USTC, University of Science and Technology of China, Hefei, Anhui China">
<meta name="citation_author" content="Beining Ma">
<meta name="citation_author_institution" content="Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China">
<meta name="citation_author_institution" content="Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China">
<meta name="citation_author" content="Yuwen Jiao">
<meta name="citation_author_institution" content="Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China">
<meta name="citation_author" content="Yan Zhou">
<meta name="citation_author_institution" content="Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China">
<meta name="citation_author" content="Tingting Wu">
<meta name="citation_author_institution" content="Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China">
<meta name="citation_author" content="Huayiyang Zou">
<meta name="citation_author_institution" content="Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China">
<meta name="citation_author" content="Jing Shi">
<meta name="citation_author_institution" content="Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China">
<meta name="citation_author" content="Yanhui Sheng">
<meta name="citation_author_institution" content="Cardiovascular Research Center, The Affiliated Suzhou Hospital of Nanjing Medical University, Nanjing Medical University, Suzhou, China">
<meta name="citation_author" content="Yifeng Wang">
<meta name="citation_author_institution" content="Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China">
<meta name="citation_author" content="CGVR Consortium">
<meta name="citation_author" content="Liming Tang">
<meta name="citation_author_institution" content="Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China">
<meta name="citation_author" content="Shixian Hu">
<meta name="citation_author_institution" content="Institute of Precision Medicine, The First Affiliated Hospital, Sun Yat-Sen University, Guangzhou, China">
<meta name="citation_author" content="Yi Duan">
<meta name="citation_author_institution" content="National Key Laboratory of Immune Response and Immunotherapy, Division of Life Sciences and Medicine, University of Science and Technology of China, Hefei, Anhui China">
<meta name="citation_author_institution" content="Center for Advanced Interdisciplinary Science and Biomedicine of IHM and Department of Infectious Diseases, The First Affiliated Hospital of USTC, University of Science and Technology of China, Hefei, Anhui China">
<meta name="citation_author" content="Wei Sun">
<meta name="citation_author_institution" content="Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China">
<meta name="citation_author" content="Wei Chen">
<meta name="citation_author_institution" content="State Key Laboratory of Food Science and Resources, Jiangnan University, Wuxi, China">
<meta name="citation_author_institution" content="School of Food Science and Technology, Jiangnan University, Wuxi, China">
<meta name="citation_author" content="Qixiao Zhai">
<meta name="citation_author_institution" content="State Key Laboratory of Food Science and Resources, Jiangnan University, Wuxi, China">
<meta name="citation_author_institution" content="School of Food Science and Technology, Jiangnan University, Wuxi, China">
<meta name="citation_author" content="Xiangqing Kong">
<meta name="citation_author_institution" content="Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China">
<meta name="citation_author_institution" content="Cardiovascular Research Center, The Affiliated Suzhou Hospital of Nanjing Medical University, Nanjing Medical University, Suzhou, China">
<meta name="citation_author" content="Lianmin Chen">
<meta name="citation_author_institution" content="Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China">
<meta name="citation_author_institution" content="Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="13">
<meta name="citation_firstpage" content="188">
<meta name="citation_doi" content="10.1186/s40168-025-02185-9">
<meta name="citation_pmid" content="40836310">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369081/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369081/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369081/pdf/40168_2025_Article_2185.pdf">
<meta name="description" content="Large-scale characterization of gut viral genomes provides strain-resolved insights into host-microbe interactions. However, existing viral genomes are mainly derived from Western populations, limiting our understanding of global gut viral diversity ...">
<meta name="og:title" content="The genetic diversity and populational specificity of the human gut virome at single-nucleotide resolution">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Large-scale characterization of gut viral genomes provides strain-resolved insights into host-microbe interactions. However, existing viral genomes are mainly derived from Western populations, limiting our understanding of global gut viral diversity ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369081/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12369081">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s40168-025-02185-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/40168_2025_Article_2185.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369081%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12369081/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12369081/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369081/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-microbio.png" alt="Microbiome logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Microbiome" title="Link to Microbiome" shape="default" href="https://microbiomejournal.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Microbiome</button></div>. 2025 Aug 20;13:188. doi: <a href="https://doi.org/10.1186/s40168-025-02185-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s40168-025-02185-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Microbiome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Microbiome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Microbiome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Microbiome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>The genetic diversity and populational specificity of the human gut virome at single-nucleotide resolution</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Xiuchao Wang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xiuchao Wang</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China </div>
<div class="p">
<sup>2</sup>Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiuchao Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Quanbin Dong</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Quanbin Dong</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China </div>
<div class="p">
<sup>3</sup>Cardiovascular Research Center, The Affiliated Suzhou Hospital of Nanjing Medical University, Nanjing Medical University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Quanbin Dong</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Pan Huang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Pan Huang</span></h3>
<div class="p">
<sup>4</sup>State Key Laboratory of Food Science and Resources, Jiangnan University, Wuxi, China </div>
<div class="p">
<sup>5</sup>School of Food Science and Technology, Jiangnan University, Wuxi, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pan Huang</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Shuai Yang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Shuai Yang</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China </div>
<div class="p">
<sup>2</sup>Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shuai Yang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Mengke Gao</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Mengke Gao</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mengke Gao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Chengcheng Zhang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Chengcheng Zhang</span></h3>
<div class="p">
<sup>4</sup>State Key Laboratory of Food Science and Resources, Jiangnan University, Wuxi, China </div>
<div class="p">
<sup>5</sup>School of Food Science and Technology, Jiangnan University, Wuxi, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chengcheng Zhang</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Chuan Zhang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Chuan Zhang</span></h3>
<div class="p">
<sup>4</sup>State Key Laboratory of Food Science and Resources, Jiangnan University, Wuxi, China </div>
<div class="p">
<sup>5</sup>School of Food Science and Technology, Jiangnan University, Wuxi, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chuan Zhang</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deng%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Youpeng Deng</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Youpeng Deng</span></h3>
<div class="p">
<sup>6</sup>National Key Laboratory of Immune Response and Immunotherapy, Division of Life Sciences and Medicine, University of Science and Technology of China, Hefei, Anhui China </div>
<div class="p">
<sup>7</sup>Center for Advanced Interdisciplinary Science and Biomedicine of IHM and Department of Infectious Diseases, The First Affiliated Hospital of USTC, University of Science and Technology of China, Hefei, Anhui China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deng%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Youpeng Deng</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Zijing Huang</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Zijing Huang</span></h3>
<div class="p">
<sup>6</sup>National Key Laboratory of Immune Response and Immunotherapy, Division of Life Sciences and Medicine, University of Science and Technology of China, Hefei, Anhui China </div>
<div class="p">
<sup>7</sup>Center for Advanced Interdisciplinary Science and Biomedicine of IHM and Department of Infectious Diseases, The First Affiliated Hospital of USTC, University of Science and Technology of China, Hefei, Anhui China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zijing Huang</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Beining Ma</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Beining Ma</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China </div>
<div class="p">
<sup>2</sup>Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Beining Ma</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiao%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Yuwen Jiao</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Yuwen Jiao</span></h3>
<div class="p">
<sup>2</sup>Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiao%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuwen Jiao</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Yan Zhou</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Yan Zhou</span></h3>
<div class="p">
<sup>2</sup>Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yan Zhou</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Tingting Wu</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Tingting Wu</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tingting Wu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zou%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Huayiyang Zou</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Huayiyang Zou</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zou%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huayiyang Zou</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Jing Shi</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Jing Shi</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jing Shi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sheng%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Yanhui Sheng</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Yanhui Sheng</span></h3>
<div class="p">
<sup>3</sup>Cardiovascular Research Center, The Affiliated Suzhou Hospital of Nanjing Medical University, Nanjing Medical University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sheng%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yanhui Sheng</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Yifeng Wang</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Yifeng Wang</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yifeng Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Consortium%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">CGVR Consortium</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">CGVR Consortium</span></h3>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Consortium%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">CGVR Consortium</span></a>
</div>
</div>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Liming Tang</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Liming Tang</span></h3>
<div class="p">
<sup>2</sup>Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Liming Tang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Shixian Hu</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Shixian Hu</span></h3>
<div class="p">
<sup>8</sup>Institute of Precision Medicine, The First Affiliated Hospital, Sun Yat-Sen University, Guangzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shixian Hu</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Duan%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Yi Duan</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Yi Duan</span></h3>
<div class="p">
<sup>6</sup>National Key Laboratory of Immune Response and Immunotherapy, Division of Life Sciences and Medicine, University of Science and Technology of China, Hefei, Anhui China </div>
<div class="p">
<sup>7</sup>Center for Advanced Interdisciplinary Science and Biomedicine of IHM and Department of Infectious Diseases, The First Affiliated Hospital of USTC, University of Science and Technology of China, Hefei, Anhui China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Duan%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yi Duan</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id22"><span class="name western">Wei Sun</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Wei Sun</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wei Sun</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id23"><span class="name western">Wei Chen</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Wei Chen</span></h3>
<div class="p">
<sup>4</sup>State Key Laboratory of Food Science and Resources, Jiangnan University, Wuxi, China </div>
<div class="p">
<sup>5</sup>School of Food Science and Technology, Jiangnan University, Wuxi, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wei Chen</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhai%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id24"><span class="name western">Qixiao Zhai</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Qixiao Zhai</span></h3>
<div class="p">
<sup>4</sup>State Key Laboratory of Food Science and Resources, Jiangnan University, Wuxi, China </div>
<div class="p">
<sup>5</sup>School of Food Science and Technology, Jiangnan University, Wuxi, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhai%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qixiao Zhai</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kong%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id25"><span class="name western">Xiangqing Kong</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Xiangqing Kong</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China </div>
<div class="p">
<sup>3</sup>Cardiovascular Research Center, The Affiliated Suzhou Hospital of Nanjing Medical University, Nanjing Medical University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kong%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiangqing Kong</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id26"><span class="name western">Lianmin Chen</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">Lianmin Chen</span></h3>
<div class="p">
<sup>1</sup>Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China </div>
<div class="p">
<sup>2</sup>Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lianmin Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Cardiology, The First Affiliated Hospital With Nanjing Medical University, Nanjing Medical University, Nanjing, China </div>
<div id="Aff2">
<sup>2</sup>Changzhou Medical Center, The Third Affiliated Hospital of Nanjing Medical University, Nanjing Medical University, Changzhou, China </div>
<div id="Aff3">
<sup>3</sup>Cardiovascular Research Center, The Affiliated Suzhou Hospital of Nanjing Medical University, Nanjing Medical University, Suzhou, China </div>
<div id="Aff4">
<sup>4</sup>State Key Laboratory of Food Science and Resources, Jiangnan University, Wuxi, China </div>
<div id="Aff5">
<sup>5</sup>School of Food Science and Technology, Jiangnan University, Wuxi, China </div>
<div id="Aff6">
<sup>6</sup>National Key Laboratory of Immune Response and Immunotherapy, Division of Life Sciences and Medicine, University of Science and Technology of China, Hefei, Anhui China </div>
<div id="Aff7">
<sup>7</sup>Center for Advanced Interdisciplinary Science and Biomedicine of IHM and Department of Infectious Diseases, The First Affiliated Hospital of USTC, University of Science and Technology of China, Hefei, Anhui China </div>
<div id="Aff8">
<sup>8</sup>Institute of Precision Medicine, The First Affiliated Hospital, Sun Yat-Sen University, Guangzhou, China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 20; Accepted 2025 Jul 2; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the articleâ€™s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12369081Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40836310/" class="usa-link">40836310</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par13">Large-scale characterization of gut viral genomes provides strain-resolved insights into host-microbe interactions. However, existing viral genomes are mainly derived from Western populations, limiting our understanding of global gut viral diversity and functional variations necessary for personalized medicine and addressing regional health disparities.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par14">Here, we introduce the Chinese Gut Viral Reference (CGVR) set, consisting of 120,568 viral genomes from 3234 deeply sequenced fecal samples collected nationwide, covering 72,751 viral operational taxonomic units (vOTUs), nearly 90% of which are likelyÂ absent from current databases. Analysis of single-nucleotide variations (SNVs) in 233 globally prevalent vOTUs revealed that 18.9% showed significant genetic stratification between Chinese and non-Chinese populations, potentially linked to bacterial infection susceptibility. The predicted bacterial hosts of population-stratified viruses exhibit distinct genetic components associated with health-related functions, including multidrug resistance. Additionally, viral strain diversity at the SNV level correlated with human phenotypic traits, such as age and gastrointestinal issues like constipation. Our analysis also indicates that the human gut bacteriome is specifically shaped by the virome, which mediates associations with human phenotypic traits.</p>
<figure class="fig xbox font-sm" id="MOESM3"><div class="media-box p text-center margin-x-neg-2 tablet:margin-x-0"><video controls="" preload="none" height="360" width="640" poster="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/video_poster.png"><source src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d360/12369081/0e277c993898/40168_2025_2185_MOESM3_ESM-pmcvs_normal.mp4"></source><div class="media-fallback margin-x-2 tablet:margin-x-0">
<div>Your browser is not supporting the HTML5 &lt;video&gt; element.</div>
<div>You may download that video as a file and play it with a player of you choice.</div>
<p><a href="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d360/12369081/0e277c993898/40168_2025_2185_MOESM3_ESM-pmcvs_normal.mp4" class="usa-link">Download video stream</a>.</p>
</div></video></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12369081/bin/40168_2025_2185_MOESM3_ESM.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (56.4MB, mp4) </sup>
</div></div>
<div class="p text-right font-secondary"><a href="figure/MOESM3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Video Abstract</p></figcaption></figure></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par15"> Our analysisÂ underscores the unique genetic makeup of the gut virome across populations and emphasizes the importance of recognizing gut viral genetic heterogeneity for deeper insights into regional health implications.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s40168-025-02185-9.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Virome, Metagenomics, Cohort study, Genomic epidemiology, Bioinformatics</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par16">The human gut virome is a critical part of the microbiome, regulating bacterial communities and impacting human health [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>â€“<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Understanding viral genomic diversity and functional variations across global populations may improve viral discovery and treatment, particularly in addressing regional health disparities. Comprehensive reference viral genomes are vital for genome-resolved metagenomics. Recent large-scale metagenomic assemblies from diverse populations have generated valuable viral reference genomes [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>â€“<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>], enabling deeper insights into strain-level viral diversity and its health implications [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>].</p>
<p id="Par17">Nevertheless, several key aspects remain underexplored, hindering a full understanding of global gut viral genomic diversity and functional variations. First, most available gut viral genomes come from Western populations. While the Unified Human Gut Virome (UHGV) catalog integrates tens of thousands of viral genomes from 12 existing virome datasets[<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>â€“<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>], around 83% of its metagenomic samples are from Western populations, with only about 17% from non-Western sources. Although some studies have explored the Chinese gut virome [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>], large-scale genomic assembly are limited. Given that China accounts for nearly 20% of the global population, expanding gut viral genome recovery from Chinese populations is crucial to enhance our understanding of the gut microbiome and increase population diversity in viral databases. Second, most gut virome phylogenetic research focuses on a few viruses with high morbidity [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>] or the most abundant in the gut [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. A comprehensive comparison of viral genomic diversity across populations and their physiological interactions with hosts remains unexplored. Finally, while advances have been made in understanding viral-bacterial interactions [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>, <a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>], the complex interplay between gut viruses and bacteria, and their effects on human phenotypes, are still poorly understood.</p>
<p id="Par18">Here, we present the Chinese Gut Viral Reference (CGVR), a comprehensive catalog of 120,568 viral genomes derived from a nationwide collection of 3234 fecal samples across Mainland China. Comparative analysis with the UHGV revealed that viral genomes from 89.7% of viral operational taxonomic units (vOTUs) in CGVR are unique to Chinese populations, while 4406 vOTUs are shared between Chinese and non-Chinese populations. Phylogenetic analysis of these shared vOTUs at single-nucleotide resolution uncovered population-specific genomic variations related to bacterial host infection, with predicted bacterial hosts of dominant viruses displaying distinct genetic features relevant for human health, including multidrug resistance. Furthermore, viral genetic diversity within populations was associated with host physiological traits, largely independent of viral abundance. We also observed that individual-specific gut virome significantly shape interindividual bacteriome variations. Lastly, medication analysis showed that associations between gut viral abundance and human phenotypic traits are mediated by the bacteriome.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Human subjects</h3>
<p id="Par19">The study was approved by the Ethics Review Board of Jiangnan University (reference number JNU20220901IRB01) and included 3234 participants from 30 provinces across mainland China. Informed consent was obtained from all participants prior to sample collection. Comprehensive phenotypic data were gathered, encompassing anthropometric traits (e.g., age, sex, and body mass index), geographical factors, residential location (urban or rural), dietary habits, lifestyles, and medical history and medication use. A detailed summary of these phenotypic traits has been previously reported [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>].</p></section><section id="Sec4"><h3 class="pmc_sec_title">Sample collection and metagenomic sequencing</h3>
<p id="Par20">Fecal samples were collected and placed on dry ice within 15 min of production. The samples were then transported to the laboratory, where aliquots were prepared using a 25% pre-reduced anaerobic glycerol solution and 1% cysteine. The aliquots were stored atâ€‰âˆ’â€‰80 Â°C until further processing. DNA extraction from the fecal samples was carried out using the QIAamp Fast DNA Stool Mini Kit (Qiagen, cat. 51604). After DNA extraction, library preparation and whole-genome shotgun sequencing were performed using the MGI DNBSEQ-T7 platform for 3082 samples and the Illumina Nova Seq 6000 platform for 152 samples [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>].</p></section><section id="Sec5"><h3 class="pmc_sec_title">Sequence preprocessing and assembly</h3>
<p id="Par21">All metagenomic data underwent quality control and decontamination of human genomic DNA (NCBI37) using KneadData (v0.12.0) with default parameters. The following key steps were employed to ensure high-quality datasets and assembly of reads into contigs: (i) trimming of adapters and low-quality bases, (ii) identification and masking of human reads, and (iii) assembly of clean reads. The raw reads were trimmed to remove adapters and low-quality bases at both ends using Trimmomatic [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>] (v.0.39). Reads shorter than 50 bp after trimming were discarded. Subsequently, the quality-filtered reads were aligned to the human reference genome using Bowtie2 [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>] (v.2.4.2). Reads for which both paired ends failed to align were retained. After completing data quality control, MegaHIT [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>] (v.1.2.9) was used to assemble the quality-filtered reads, generating 6.8â€‰Ã—â€‰10<sup>7</sup> contigs.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Viral genomic binning and quality assessment</h3>
<p id="Par22">Viral genomic binning was performed using VAMB [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] (v.4.1.3), a deep-learning-based tool that clusters metagenomic contigs into putative biological entities based on assembly sequences. During this process, contigs shorter than 2 kb were excluded to ensure data quality. After binning, all VAMB-generated bins were further refined using the viral binning method PHAMB [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] (v.1.0.1), resulting in the identification of approximately 5.5â€‰Ã—â€‰10<sup>6</sup> putative viral bins. To assess the completeness of the metagenome-assembled viral genomes, we applied CheckV [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>] (v.1.0.1, database v1.5) with the â€œend_to_endâ€ mode and default settings. This step ensured that only high-quality viral genomes were included in further analyses, contributing to a robust dataset for viral characterization.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Generation of viral operational taxonomic units (vOTUs)</h3>
<p id="Par23">To generate viral operational taxonomic units (vOTUs), we first built a nucleotide BLAST database of all 64,377 viral sequences using makeblastdb (option: â€œ-dbtype nuclâ€) from BLAST[<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>] (v.2.14.1). Pairwise comparisons of the viral sequences were generated using an all-vs-all local alignment strategy, executed via blastn with the option â€œ-max_target_seqs 10,000.â€ Next, two custom scripts, anicalc.py and aniclust.py, from the CheckV repositoryâ€™s genome clustering tools, were employed to compute average nucleotide identity (ANI) and alignment fraction (AF) for clustering sequences at the species level. Using the anicalc.py script, pairwise ANI values were calculated by merging local alignments between sequence pairs. Finally, species-level clustering was performed using aniclust.py in a UCLUST-like manner with the recommended parameters (â€œ-min_ani 95, -min_tcov 85, -min_qcov 0â€) from the MIUVIG [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>] protocol, effectively grouping viral genomes into viral operational taxonomic units.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Taxonomic annotation of viral genomes</h3>
<p id="Par24">Each viral genomes were taxonomically classified using uhgv-tools (v0.0.1) with â€œclassifyâ€ mode. The uhgv-tools utilize aligning genome and protein sequences against an integrated viral database (uhgv-db-v0.4) to obtain average DNA identity, alignment fraction, and amino acid similarity scores, which are used to propose viral ICTV taxon lineage [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>].</p></section><section id="Sec9"><h3 class="pmc_sec_title">Functional annotation of genes in viral genomes</h3>
<p id="Par25">The open reading frames (ORFs) of the viral genomes were predicted using Prodigal [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] (v2.6.3) with the option â€œ-p meta.â€ Functional annotation of the genes was performed using eggNOG-mapper [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] (v2.1.12), utilizing the EggNOG database [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] (v5.0.2) with default settings. The resulting functional annotations included categories from COG, KEGG [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>], and Pfam [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>], which were derived from the eggNOG-mapper output. Eleven classes of phage parts were categorized based on their functions, including LYS (lysis), INT (integration), REP (replication), REG (regulation), PAC (packaging), ASB (assembly), INF (infection), EVA (immune evasion), HYP (hypothetical protein), UNS (unsorted), and tRNA according to a previous study [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>].</p></section><section id="Sec10"><h3 class="pmc_sec_title">Comparison of viral genomes with the UHGV</h3>
<p id="Par26">Viral sequences from the human gut virome catalog database (UHGV, <a href="https://github.com/snayfach/UHGV" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/snayfach/UHGV</a>) were obtained for comparison. The UHGV database integrates gut virome collections from 12 recent studies [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>â€“<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>], including MGV, GPD, GVD, GFD, CHVD, Hadza, Benler, mMGE, DEVoC, IMG/VR, HuVirDB, and COPSAC, totaling 2,242,702 putative viral genomes. To ensure data quality, the completeness of all viral sequences was assessed using CheckV. From this assessment, 208,604 high-quality viral sequences were selected for comparative analysis. A nucleotide BLAST database of all quality-controlled viral sequences was then constructed, and pairwise comparisons were performed using all-vs-all local alignments with blastn (v2.14.1).</p></section><section id="Sec11"><h3 class="pmc_sec_title">Single-nucleotide variant (SNV) calling</h3>
<p id="Par27">For each cluster of selected viruses, all protein-coding genes of vOTUs were predicted using Prodigal (v2.6.3) with the parameters -p meta -f gff to generate GFF files. Subsequently, we utilized Roary<sup>59</sup> (v3.12.0) with options -s -i 95 to obtain a pan-genome reference, which serves as a comprehensive and reliable reference for each specific virus species. To detect SNVs within each vOTU, we used the pan_genome_reference.fa file generated by Roary as the reference genome. The detection process was performed using Snippy (v4.6.0), a fast and accurate tool specifically designed to identify single-nucleotide polymorphisms (SNPs) and insertion-deletion events (indels) in draft or complete genomes through a mapping-based approach. A coverage threshold of 10 reads at a site was established to consider an SNV as valid. The SNVs of bacterial hosts were characterized using the same methodology.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Genomic dissimilarity</h3>
<p id="Par28">For each viral and bacterial genome, we focused on the single-nucleotide polymorphism (SNP) variants extracted from the Snippy output, temporarily excluding insertions, deletions, and complex variants. We combined the.vcf files of all genomes associated with the same species into a matrix format, where each row represented a variant site and each column corresponded to the nucleotide type of a genome at that site. Each column was treated as a distinct sequence, which we aligned to generate a phylogenetic distance matrix. Pairwise nucleotide substitution rates between samples were calculated using the Kimura 2-parameter method from the EMBOSS [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>] package. For any NA values in the distance matrix, indicating distances that were too large to compute, we replaced these entries with the maximum distance value plus one [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>].</p></section><section id="Sec13"><h3 class="pmc_sec_title">Phylogenetic trees</h3>
<p id="Par29">To identify distinct sub-species clades or strains, we employed hierarchical clustering using the complete linkage method. This approach was applied to a genomic distance matrix derived from SNP haplotype data, enabling the classification of viral and bacterial strains based on their genetic similarity. Hierarchical clustering of genomes was performed using <em>hclust</em> function. Phylogenetic trees were then visualized using <em>ggtree</em> package (v3.6.2) [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>].</p></section><section id="Sec14"><h3 class="pmc_sec_title">Protein structure and function prediction</h3>
<p id="Par30">To investigate the three-dimensional structures and potential functions of the virus proteins of interest, we employed advanced computational tools, including AlphaFold3 [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>] and DeepFRI [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. After predicting protein structures using AlphaFold3, we utilized DeepFRI to infer potential functions. DeepFRI employs a pre-trained graph-based neural network model to analyze protein sequences and structures, providing a list of predicted functions along with confidence scores for each protein. Protein structures were visualized using PyMOL.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Prediction of viral bacterial hosts</h3>
<p id="Par31">To predict bacterial hosts for vOTUs, we utilized the Global Microbiome Reference (GMR) catalog [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>] to identify CRISPR spacers. This was achieved by integrating the outputs from two CRISPR detection tools, CRT [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>], and PILER-CR [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>], both executed with default settings. The predicted CRISPR spacers were then aligned against the representative vOTUs from the CGVR catalog using blastn (v2.14.1). The alignment was optimized for short sequences with the following options: -max_target_seqs 10,000, -task blastn-short, -evalue 1, -gapopen 10, -gapextend 2, -penalty âˆ’1, -word_size 10, and -perc_identity 100. This approach allowed for accurate matching of CRISPR spacers to viral genomes, facilitating bacterial host prediction.</p>
<p id="Par32">Given that CRISPR-based host prediction is limited to bacteria with active spacer integration, we complemented our analysis with iPHoP [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>] which exhibits higher sensitivity and yields more matches. We ran iPHoP with the parameters set to â€œpredictâ€ and the database to â€œiPHoP_db_Aug23_rw.â€</p></section><section id="Sec16"><h3 class="pmc_sec_title">Functional annotation of genes in bacterial genomes</h3>
<p id="Par33">To investigate the functions of SNVs derived from metagenome-assembled genomes (MAGs) that contribute to the observed genetic stratification in bacterial hosts, we performed functional annotation of bacterial proteins using the UniRef50 [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>] database. This process involved conducting a BLAST alignment of the gene sequences against the UniRef50 database using Diamond with the parameters â€œâ€“evalue 1e-5 â€“id 50.â€</p></section><section id="Sec17"><h3 class="pmc_sec_title">Relative abundance of vOTUs</h3>
<p id="Par34">The TPM (transcripts per kilobase of exon model per million mapped reads) values were used to represent relative abundances of vOTUs in the fecal metagenomic samples estimated by CoverM (v0.6.1). The representatives of vOTUs (<em>n</em>â€‰=â€‰42,803) were utilized to construct the mapping reference genome through bowtie2-build (version 2.4.2). Subsequently, the quality-controlled reads from each individual sample were mapped using bowtie2. Finally, the TPM values were computed with the â€œâ€“methods tpmâ€ option, leveraging custom scripts.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Relative abundance of bacterial species</h3>
<p id="Par35">Bacterial species abundances were generated using MetaPhlAn4 [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>], which leverages approximately 5.1 million unique clade-specific marker genes identified from around 1 million microbial genomes. This extensive database encompasses 26,970 species-level genome bins, facilitating unambiguous taxonomic assignments, accurate estimation of organismal relative abundance, and species-level resolution for bacteria. Only microbial species present in more than 20% of the samples were included for further analysis, resulting in a list of 303 species.</p></section></section><section id="Sec19"><h2 class="pmc_sec_title">Statistical analysis</h2>
<section id="Sec20"><h3 class="pmc_sec_title">Differential comparison</h3>
<p id="Par36">Viral genomic dissimilarities within and between Chinese and non-Chinese populations were assessed using the Kruskalâ€“Wallis test, while genomic dissimilarities between the two groups were evaluated using the Wilcoxon test. <em>P</em>-values were adjusted using the Benjamini-Hochberg (BH) method [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>].</p></section><section id="Sec21"><h3 class="pmc_sec_title">Enrichment of allele type</h3>
<p id="Par37">Differential enrichment of specific allele types in viral genomes between populations and sub-species clades was assessed using Fisherâ€™s exact tests. <em>P</em>-values were adjusted using the BH method.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Associations of within-species viral genomic dissimilarities with phenotypic differences</h3>
<p id="Par38">Spearman correlation was applied to assess whether genomic variations within viral species are associated with human host physiological traits. For inter-individual gut viral genomic dissimilarities, pairwise nucleotide substitution rates between samples were calculated using the Kimura 2-parameter method. The Manhattan dissimilarity metric was applied to measure phenotypic differences. <em>P</em>-values were adjusted using the BH method.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Associations between viral genomic and abundance differences</h3>
<p id="Par39">Spearman correlation was applied to evaluate whether genomic variations within viral species are associated with abundance differences. For inter-individual gut viral genomic dissimilarities, pairwise nucleotide substitution rates were calculated using the Kimura 2-parameter method. The Manhattan dissimilarity metric was used to assess abundance differences, and <em>P</em>-values were adjusted using the BH method.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Viral abundance associations with human phenotypes and bacterial abundance</h3>
<p id="Par40">The relative abundances of vOTUs and bacterial species were centered log-ratio transformed and then subjected to Spearman correlation [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. No transformation was applied to the phenotypic data. For associations identified at <em>FDR</em>â€‰&lt;â€‰0.05, we conducted MaAsLin2 analysis to estimate relationships between viral features, human phenotypes, and bacterial species abundance. Human phenotypes and bacterial species significantly associated with vOTUs were treated as fixed effects in MaAsLin2, and results with FDR controlled at 0.05 were reported.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Association between virome and gut bacteriome overall compositions</h3>
<p id="Par41">We applied the <em>vegdist</em> function from the vegan (version 2.6.4) R package to calculate the Brayâ€“Curtis dissimilarity matrix based on the relative abundances of the virome and bacteriome at the species level. We then assessed the correlation between the two distance matrices using the Spearman correlation test. To estimate the variance in bacterial overall composition explained by the gut virome, linear regression was applied.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Mediation linkage inference</h3>
<p id="Par42">For vOTUs associated with both human phenotypic traits and bacterial species, we first checked whether host phenotypic traits were also associated with bacterial species using both Spearman correlation and MaAsLin2 (<em>FDR</em>â€‰&lt;â€‰0.05). Next, mediation analysis was carried out using the mediate function from the R package mediation (version 4.5.0) to infer the mediation effect of bacterial species on viral associations with human phenotypic traits. <em>P</em>-values were adjusted using the BH method.</p></section></section><section id="Sec27"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec28"><h3 class="pmc_sec_title">A metagenomic compendium of 120,568 gut viral genomes from Chinese populations extends global viral diversity</h3>
<p id="Par43">The gut virome is crucial for human health and disease. To enable strain-resolved metagenomics across diverse populations, a comprehensive reference of viral genomes is essential. The Unified Human Gut Virome Catalog (UHGV) recently integrated 12 existing virome catalogs [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>â€“<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>], establishing itself as the most extensive viral resource to date. However, fewer than 20% of these viral genomes are derived from non-Western populations, which constitute 75% of the global population. This limitation hinders our understanding of global gut viral diversity and its implications for regional health disparities. In this study, we performed gut viral genome assembly using deeply sequenced metagenomic samples (averaging 20 GB) from 3234 individuals across 30 provinces in China [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>].</p>
<p id="Par44">Following a stringent viral detection pipeline (Fig. <a href="#Fig1" class="usa-link">1</a>a), we identified 5,531,572 putative viral bins. Quality assessment using CheckV [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>] yielded 120,568 medium- to high-quality viral genomes, categorized as follows: 8667 complete genomes (100% completeness), 55,710 high-quality genomes (&gt;â€‰90% completeness), and 56,191 medium-quality genomes (50â€“90% completeness) (Fig. <a href="#Fig1" class="usa-link">1</a>b, c; Additional file 1: Table S1). For subsequent analysis, we focused on the 64,377 genomes withâ€‰&gt;â€‰90% completeness to address limitations associated with small genome fragments [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>] and adhere to UHGV quality standards. The median size of the assembled viral genomes was 82 kb (interquartile range: 43â€“173 kb) (Additional file 1: Table S1). We designate this collection of viral genomes as the CGVR.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369081_40168_2025_2185_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d360/12369081/36b3b50ad4a6/40168_2025_2185_Fig1_HTML.jpg" loading="lazy" id="MO1" height="422" width="778" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The Chinese Gut Viral Reference (CGVR) extends global viral diversity.Â <strong>a</strong>Â Overview of the CGVR database construction pipeline. A total of 120,568 qualified viral bins were generated, resulting in 72,751 viral operational taxonomic units (vOTUs), including 42,803 high-quality vOTUs. <strong>b</strong>Â Distribution of assembled viral genomes based on genomic length and quality. <strong>c</strong>Â Summary of viral genomes categorized by completeness (complete, nâ€‰=â€‰8,667; &gt;90% complete, nâ€‰=â€‰55,710; 50â€“90% complete, nâ€‰=â€‰56,191). <strong>d</strong>Â ICTV taxonomic annotation of viral genomes in the CGVR, with numbers indicating the total genomes within each viral class. <strong>e</strong>Â Accumulation curves for vOTUs in the CGVR. Red curves include all vOTUs, while blue curves focus on vOTUs (n = 7,441) with at least two conspecific viral sequences. <strong>f</strong>Â Functional characterization of novel vOTUs compared with the UHGV catalog, with bars color-coded by broad COG categories, blue for cellular processes and signaling, red for information storage and processing, orange for metabolism, and grey for poorly characterized</p></figcaption></figure><p id="Par45">To classify the taxonomy of our CGVR catalog, we clustered all genomes at 95% average nucleotide identity (ANI) and 85% alignment fraction (AF) [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>], resulting in 42,803 viral operational taxonomic units (vOTUs) (Additional file 1: Table S2). Taxonomic annotation using the ICTV classification tool [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>] revealed that 57.22% (24,492) of these vOTUs could be classified at the class level, with Caudoviricetes (22,360), Malgrandaviricetes (1374), Faserviricetes (673), and Arfiviricetes (76) being the most prevalent (Fig. <a href="#Fig1" class="usa-link">1</a>d; Additional file 1: Table S3), consistent with recent studies [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. Notably, 18,311 of the 42,803 (42.78%) vOTUs could not be classified into any known viral classes, highlighting the vast, underexplored diversity of gut viruses in Chinese populations. Additionally, rarefaction analysis indicated that the number of vOTUs detected has not yet reached saturation, suggesting the presence of additional undiscovered viruses (Fig. <a href="#Fig1" class="usa-link">1</a>e). However, these are likely to be rarer members of the Chinese gut virome, as the curve approaches saturation when considering only those with at least two conspecific sequences (Fig. <a href="#Fig1" class="usa-link">1</a>e).</p>
<p id="Par46">Finally, we examined the functional potential of the 18,311 unclassified vOTUs by annotating their protein-coding genes using Prodigal [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] and eggNOG-mapper [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. A total of 3,531,752 viral proteins were predicted, with 56.79% (2,005,582) showing homologous genes in the eggNOG database [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>], enabling functional decoding (Fig. <a href="#Fig1" class="usa-link">1</a>f; Additional file 1: Table S4). Common virus-related proteins involved in processes such as DNA replication, homologous recombination, and transcriptional regulation were frequently identified (Fig. <a href="#Fig1" class="usa-link">1</a>f). However, 43.21% of protein sequences lacked homology to known sequences in existing databases, and this may partly be due to alternative stop codons utilized by some viruses, as we have detected high frequency of TGA (30.0%) and TAG (16.7%) (Additional file 1: Table S5), but also highlighting a significant knowledge gap in our understanding of the human gut virome. Overall, the CGVR catalog significantly enhances our understanding of global gut virome diversity, particularly in Chinese populations, and provides a robust foundation for future comparative studies, yielding valuable insights into the potential role of gut viruses in health, disease, and regional disparities.</p></section><section id="Sec29"><h3 class="pmc_sec_title">Phylogenetic comparison between CGVR and UHGV revealed population-specific gut viruses</h3>
<p id="Par47">We then sought to assess the uniqueness of vOTUs in our CGVR compared to non-Chinese viral genomes in the UHGV. To do this, we clustered our 64,377 high-quality viral genomes with 208,604 viral genomes from the UHGV (released in March 2024, uhgv-db-v0.4) using 95% ANI and 85% AF. This resulted in a total of 101,562 vOTUs, with only 4406 (4.34%) overlapping between the CGVR and UHGV (Fig. <a href="#Fig2" class="usa-link">2</a>a). This finding highlights significant genetic differences in gut viruses between Chinese and non-Chinese populations.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369081_40168_2025_2185_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d360/12369081/79e4839bc8fa/40168_2025_2185_Fig2_HTML.jpg" loading="lazy" id="MO2" height="938" width="778" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Population-specific viruses and genomic stratification of predominant viruses linked to bacterial host infection. <strong>a</strong> Overlap of vOTUs between the CGVR and UHGV based on 95% ANI and 85% AF. <strong>b</strong> Summary of viral genomes in 233 commonly presented vOTUs across Chinese and non-Chinese populations, defined as having at least 3 genomes in a given vOTU and a minimum of 10% representation in either population. <strong>c</strong> Characteristics of the 233 commonly presented vOTUs. Red dot plot indicates viral genome length, blue dot plot represents the number of viral CDSs, and green dot plot shows detected SNV sites. <strong>d</strong> Summary of vOTUs exhibiting significant genomic stratification between Chinese and non-Chinese populations at single-nucleotide resolution. The circular chart shows red for vOTUs with significant population stratification (44 vOTUs, <em>FDR</em>â€‰&lt;â€‰0.05) and blue for those without. <strong>e</strong> Overview of 44 vOTUs with significant genetic stratifications between populations, where green indicates genetic distance between Chinese and non-Chinese populations, red denotes genetic distance within the Chinese population, and blue indicates genetic distance within non-Chinese populations. <strong>f</strong> Summary of SNVs showing significant enrichment between populations. A total of 4950 loci from 44 vOTUs were analyzed with Fisherâ€™s exact tests, identifying 239 loci that were differentially enriched in either population at <em>FDR</em>â€‰&lt;â€‰0.05. <strong>g</strong> Phylogenetic tree of <em>FeceHeNa7_vae_40</em> based on SNVs, with red representing viral genomes from Chinese populations and blue from non-Chinese populations. <strong>h</strong> Genomic annotation of <em>FeceHeNa7_vae_40</em>, highlighting differential SNVs in genes encoding a phage portal protein. Annotated protein-coding genes (arrows) are color-coded by function, including assembly, lysis, integration, replication, and infection. <strong>i</strong> Phylogenetic tree of <em>FNMHLBE15_vae_111</em> based on SNVs, with red indicating genomes from Chinese populations and blue indicating non-Chinese populations. <strong>j</strong> Summary of viral genomes of <em>FNMHLBE15_vae_111</em> with differential allele types in the <em>OMEACPOP_00016</em> gene, marked in red for Chinese and blue for non-Chinese genomes. <strong>k</strong> Predicted protein structure encoded by the <em>OMEACPOP_00016</em> gene, with differential physicochemical properties indicated by gray dashed lines</p></figcaption></figure><p id="Par48">Notably, 38,436 vOTUs from our study, comprising 44,715 viral genomes, did not cluster with any sequences from the UHGV. This substantial divergence underscores the unique viral diversity present in Chinese populations. Given the highly individualized nature of gut viruses at the strain level, developing person-specific viral genome databases is essential for accurate strain tracking [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>, <a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. The fact that 89.7% of our vOTUs lack close relatives in the UHGV indicates that existing viral catalogs are insufficient for analyzing gut viral strains in Chinese populations. These findings emphasize the necessity of population-specific viral genome databases to capture unique viral diversity, enabling more precise investigations into the gut viromeâ€™s role in human health and disease.</p></section><section id="Sec30"><h3 class="pmc_sec_title">Genomic variations in predominant viruses between populations linked to bacterial host infection</h3>
<p id="Par49">In addition to the vOTUs specific to Chinese populations, we examined the 4406 shared vOTUs (Fig. 2a) between Chinese and non-Chinese populations for genomic and functional variations. After excluding viral genomes of unknown ethnic origin, we focused on 233 vOTUs with at least 3 genomes and a minimum prevalence of 10%. (Fig. 2b). These vOTUs averaged 66 kb in genome length and contained around 100 CDSs (coding DNA sequence). Single-nucleotide variation (SNV) analysis using Snippy revealed an average of 2840 SNVs per vOTU (Fig. 2c).</p>
<p id="Par50">We then compared within- and between-population genomic dissimilarity using the Kimura 2-parameter model based on SNVs. Among the 233 vOTUs, 44 (18.88%) displayed significant population stratification (Kruskalâ€“Wallis test, <em>FDR</em>â€‰&lt;â€‰0.05), indicating potential distinct evolutionary trajectories for Chinese and non-Chinese populations (Fig. <a href="#Fig2" class="usa-link">2</a>d, e; Additional file 1: Table S6; Additional file 2: Fig. S1).</p>
<p id="Par51">To assess the functional implications of these genetic differences, we conducted enrichment test on biallelic SNVs (<em>N</em>â€‰=â€‰4,950), identifying 239 SNVs with significant population differences (<em>FDR</em>â€‰&lt;â€‰0.05, Additional file 1: Table S7), including 181 located within CDSs (Fig. <a href="#Fig2" class="usa-link">2</a>f). Functional annotation of these genes encompasses various functionalities related to viral lysis, infection, and integration (Additional file 1: Table S7, Additional file 2: Fig. S2).</p>
<p id="Par52">For instance, <em>FeceHeNa7_vae_40</em>, a member of Caudoviricetes, exhibited two distinct clusters between populations (Fisherâ€™s exact test, <em>P</em>â€‰=â€‰6.6â€‰Ã—â€‰10<sup>-13</sup>) (Fig. <a href="#Fig2" class="usa-link">2</a>g), with eight significant SNVs detected, three of which were within the <em>LCHFMIBM_00032</em> gene encoding a phage portal protein (SPP1 Gp6-like) (Fig. <a href="#Fig2" class="usa-link">2</a>h), crucial for phage infection [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>].</p>
<p id="Par53">Another vOTU, <em>FNMHLBE15_vae_111</em> from the Zeta-crassviridae family, also showed significant genetic stratification (Fisherâ€™s exact test, <em>P</em>â€‰=â€‰3.8â€‰Ã—â€‰10<sup>-14</sup>) driven by 31 SNVs (<em>FDR</em>â€‰&lt;â€‰0.05) (Fig. <a href="#Fig2" class="usa-link">2</a>i). Notably, a non-synonymous SNV in the <em>OMEACPOP_00016</em> gene was enriched in distinct populations, with position 328 showing A in Chinese and T in non-Chinese genomes (Fig. <a href="#Fig2" class="usa-link">2</a>j). Although this gene lacks functional annotation, predictions using AlphaFold3 [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>] and DeepFRI [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] suggest its involvement in binding ions and small molecules, potentially influencing phage interactions with host bacteria (Fig. <a href="#Fig2" class="usa-link">2</a>k). Given the role of cations in phage attachment to host cells [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>], variations in this protein sequence may significantly impact virus-host interactions. Thus, our findings indicate that differences in viral infection proteins may contribute to varying bacteria-virus interactions across populations, reflecting unique viral adaptations.</p></section><section id="Sec31"><h3 class="pmc_sec_title">Predicted bacterial hosts of population-stratified viruses display distinct genetic components linked to various human health-related functions</h3>
<p id="Par54">Given the genomic stratification observed in predominant viruses and the role of mutations in bacterial infection-related proteins, we examined whether their bacterial hosts also exhibit genomic stratification. Using CRISPR spacer sequences from prokaryotic genomes, which serve as molecular records of past viral encounters [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>, <a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>], we predicted bacterial hosts for the 44 vOTUs showing population differences.</p>
<p id="Par55">By comparing CRISPR spacer sequences from 2791 virus genomes associated with the 44 vOTUs to 478,588 global gut bacterial genomes [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>], we identified 889,417 matches spanning 2740 virus genomes and 17,897 bacterial genomes from 452 species. To enhance the confidence of these virus-bacteria pairs, we retained only those supported by at least 20 viral genomes within each vOTU, resulting in 117 pairs involving 25 vOTUs and 68 bacterial species (Additional file 1: Table S10). For 50.6% bacterial genomes without CRISPR spacer sequences, we further applied iPHoP to predict bacterial hosts for the 44 vOTUs, which identified 1020 host-virus pairs involving 517 bacterial species (Additional file 1: Table S8). Comparison with the CRISPR-spacer method (which detected 1373 pairs involving 452 bacterial species) (Additional file 1: Table S9) revealed 394 (394Â /Â 1,020â€‰=â€‰38.6%) overlapping pairs consistently predicted by both approaches, demonstrating partial concordance between the two methods while highlighting their respective strengths in host prediction. Notably, the bacterial hosts in these pairs predominantly belonged to the phyla Bacteroidota and Bacillota_A (Fig. <a href="#Fig3" class="usa-link">3</a>a; Additional file 1: Table S10). Furthermore, comparisons of intragroup genomic dissimilarity among bacterial hosts corresponding to different viral sub-clades revealed significant differences in the genetic makeup of 23 bacterial host species based on their SNVs, as profiled by Snippy and analyzed using a Kimura 2-parameter distance matrix (<em>FDR</em>â€‰&lt;â€‰0.05; Additional file 1: Table S11).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369081_40168_2025_2185_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d360/12369081/a383e79d8770/40168_2025_2185_Fig3_HTML.jpg" loading="lazy" id="MO3" height="352" width="777" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Predicted bacterial hosts of population-stratified viruses exhibit unique genetic components associated with diverse health-related functions including multidrug resistance. <strong>a</strong> Predicted virus-bacteria pairs categorized by bacterial hosts at the phylum level. A total of 117 virus-bacteria pairs involves 25 vOTUs and 68 bacterial species. Red indicates viruses, blue indicates bacterial hosts, and gray boxes represent bacterial hosts categorized by phyla, including Bacillota_A and Bacteroidota. <strong>b</strong> Distinct sub-species clades of <em>FNMHLBE15_vae_111</em> genomes based on SNV profiles, with blue for clade 1 and red for clade 2. <strong>c</strong> Genetic dissimilarity of the predicted bacterial host <em>P. distasonis</em> within the two distinct sub-species clades of <em>FNMHLBE15_vae_111</em>, with blue and red indicating within-clade genomic dissimilarities. <strong>d</strong> Summary of the predicted bacterial host <em>P. distasonis</em> specific to the sub-species clades of <em>FNMHLBE15_vae_111</em>. <strong>e</strong> Differential allele types within the gene <em>PNJHJHOD_03790_999</em> (antibiotic ABC transporter ATP-binding and multidrug export protein) in <em>P. distasonis</em> between the sub-species clades of <em>FNMHLBE15_vae_111</em></p></figcaption></figure><p id="Par56">To further characterize the genomic components driving the observed genetic stratification in bacterial hosts, we conducted an enrichment analysis to identify differential SNVs from bacterial genomes unique to various viral sub-species clades. This analysis revealed 257 differential SNVs across 6 bacterial species, corresponding to 242 CDSs (Additional file 1: Table S12). Annotation of these CDSs using UniRef50 [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>] indicated a diverse range of encoded functionalities, including biosynthesis pathways for amino acids and short-chain fatty acids, as well as mechanisms related to drug resistance (Additional file 1: Table S13).</p>
<p id="Par57">For instance, 284 viral genomes from the <em>FNMHLBE15_vae_111</em> vOTU formed 2 distinct sub-species clades (Fig. <a href="#Fig3" class="usa-link">3</a>b). We predicted 390 metagenome-assembled genomes (MAGs) from <em>Parabacteroides distasonis</em> as their bacterial hosts based on CRISPR spacers, which exhibited significant genetic divergence (<em>P</em><sub>Wilcoxon</sub>â€‰=â€‰2.3â€‰Ã—â€‰10<sup>-70</sup>) (Fig. <a href="#Fig3" class="usa-link">3</a>c; Additional file 1: Table S11). Notably, 18 MAGs were specific to one cluster, while 251 were specific to the other (Fig. <a href="#Fig3" class="usa-link">3</a>d), and we identified 205 SNVs differentially enriched between the two groups. Among these, several SNVs were located in genes associated with drug resistance relevant to human health. In particular, we observed a significant differential enrichment of allele types at a locus within the <em>PNJHJHOD_03790_999</em> gene (Fisherâ€™s exact test, <em>P</em>â€‰=â€‰7.2â€‰Ã—â€‰10<sup>-9</sup>) (Additional file 1: Table S12), which encodes antibiotic ABC transporter ATP-binding [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>] and multidrug export protein [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>] (Fig. <a href="#Fig3" class="usa-link">3</a>e; Additional file 1: Table S13). These findings suggest the potential of leveraging population-specific viral strains to target bacterial hosts harboring antibiotic resistance genes, offering a promising strategy for combating antibiotic resistance through virus-mediated bacterial control.</p></section><section id="Sec32"><h3 class="pmc_sec_title">Within-species viral genetic diversity is related to human phenotypic traits that independent of abundance variations</h3>
<p id="Par58">In addition to the genomic stratification of predominant viruses between populations, which suggests potential susceptibility to bacterial infections, we further examined whether within-species genomic variations are also associated with human host physiological traits. We filtered our cohort to identify vOTUs for which viral genomes could be constructed from at least 10 MAGs, resulting in a total of 446 vOTUs. Correlation analyses between the genetic dissimilarities of these vOTUs and host phenotypic differences revealed 117 significant associations (<em>FDR</em>â€‰&lt;â€‰0.05) across 67 vOTUs and 18 phenotypic traits (Fig. <a href="#Fig4" class="usa-link">4</a>a; Additional file 1: Table S14).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369081_40168_2025_2185_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d360/12369081/39868626661e/40168_2025_2185_Fig4_HTML.jpg" loading="lazy" id="MO4" height="440" width="778" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genetic diversity within viral species is associated with host phenotypic traits. <strong>a</strong> Summary of associations between vOTU genomic dissimilarity and host phenotypic differences. Each bar represents a phenotype, sorted by the number of associations (<em>FDR</em>â€‰&lt;â€‰0.05), with color coding: red for physiological traits, blue for lifestyle factors, and green for dietary habits. <strong>b</strong> Genomic dissimilarity of <em>FSHCM57_vae_33</em> associated with drinking frequency. <strong>c</strong> Genomic dissimilarity of <em>FGZ37M_FDME220052037-1a_vae_29</em> related to host age. <strong>d</strong> Distinct sub-species clades of <em>FeceHuNa150110_FDME220052261-1a_vae_31</em> based on SNV profiles, with red indicating clade 1 and blue indicating clade 2. <strong>e</strong> Sex distribution of hosts for <em>FeceHuNa150110_FDME220052261-1a_vae_31</em>, highlighting the differential distribution of viral genomes between male and female hosts</p></figcaption></figure><p id="Par59">Generally, within-species genetic dissimilarities were mainly linked to dietary habits and intrinsic factors like age and BMI (Fig. <a href="#Fig4" class="usa-link">4</a>a). Notably, participants exhibiting greater phenotypic differences tended to show increased genetic variation in vOTUs. For example, differences in drinking frequency were correlated with genetic variations in <em>FSHCM57_vae_33</em> (<em>r</em><sub>Spearman</sub>â€‰=â€‰0.16, <em>P</em>â€‰=â€‰7.2â€‰Ã—â€‰10<sup>-16</sup>) (Fig. <a href="#Fig4" class="usa-link">4</a>b; Additional file 1: Table S14). Additionally, many associations were observed with intrinsic factors such as age, BMI, and defecation frequency (Additional file 2: Fig. S3). For instance, inter-individual age differences were associated with the genetic dissimilarities of <em>FGZ37M_FDME220052037-1a_vae_29</em> (<em>r</em><sub>Spearman</sub>â€‰= -0.20, <em>P</em>â€‰=â€‰2.1â€‰Ã—â€‰10<sup>-4</sup>) (Fig. <a href="#Fig4" class="usa-link">4</a>c; Additional file 1: Table S14).</p>
<p id="Par60">Given the substantial genetic diversity within viral species, we further investigated whether these viral genomes could form distinct sub-clades within the population. We hypothesized that main types of viral strains with similar genetic makeup might lead to unique functional attributes influencing host physiology. For 31 out of the 67 vOTUs, distinct subspecies clades were observed in phylogenetic analyses (Additional file 2: Fig. S4). By linking these viral subspecies clades to the aforementioned host phenotypic traits, we identified four significant associations between four vOTUs and three host phenotypic traits (<em>FDR</em>â€‰&lt;â€‰0.05, Additional file 1: Table S15). For instance, viral genomes from the <em>FeceHuNa150110_FDME220052261-1a_vae_31</em> vOTU stratified into two clades specifically associated with differences in host sex (<em>P</em>â€‰=â€‰9.1â€‰Ã—â€‰10<sup>-4</sup>, Fig. <a href="#Fig4" class="usa-link">4</a>d, e).</p>
<p id="Par61">To determine whether the observed genetic associations between vOTUs and host phenotypic traits were influenced by abundance differences, we filtered for vOTUs present in at least 20% of participants and comprising a minimum of 10 viral genomes, yielding 408 vOTUs. We profiled their abundance in fecal metagenomic samples using transcripts per kilobase of exon model per million mapped reads (TPM) calculated by CoverM. Among these vOTUs, 87.0% (355 out of 408) showed no significant association between genomic dissimilarity and abundance differences (<em>FDR</em>â€‰&gt;â€‰0.05, Additional file 2: Fig. S5; Additional file 1: Table S16), suggesting that genetic variability among vOTUs provides additional insights independent of their abundance. These findings underscore the critical role of viral strains in influencing host phenotypic traits.</p></section><section id="Sec33"><h3 class="pmc_sec_title">Abundance variations of newly characterized vOTUs widely associated with human phenotypic traits</h3>
<p id="Par62">As demonstrated, within-species viral genomic variations largely operate independently of abundance differences, with only modest associations observed. To provide further biological relevance for the newly characterized vOTUs lacking taxonomic information, we linked their abundances to human phenotypic traits. We profiled the viral abundance of 38,436 unique vOTUs in fecal metagenomic samples, detecting an average of 27,310 vOTUs per sample and 35,549 vOTUs in over 20% of samples.</p>
<p id="Par63">Using MaAsLin2 [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>], a tool for analyzing multivariable associations between host phenotypes and microbiomes, we identified 19,108 significant associations between 13,215 vOTUs and 22 phenotypes (<em>FDR</em>â€‰&lt;â€‰0.05), primarily enriched for geographic, individual intrinsic, and dietary factors (Fig. <a href="#Fig5" class="usa-link">5</a>a; Additional file 1: Table S17). Notably, residential location (urban vs. rural), defecation frequency, and age were the most strongly associated with viral abundance.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369081_40168_2025_2185_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d360/12369081/2353f2a93994/40168_2025_2185_Fig5_HTML.jpg" loading="lazy" id="MO5" height="401" width="778" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Abundance variations of newly characterized vOTUs widely associated with human phenotypes. <strong>a</strong> Summary of associations between vOTU abundances and various phenotypic traits. Human phenotypes are represented in gray, while gut viruses from different viral classes are color-coded. Each line indicates a significant association between a viral factor and a phenotype, with line colors corresponding to the associated viral class. <strong>b</strong> The relative abundance of <em>FSHXHZ2_vae_115</em> is significantly higher in urban populations compared to rural areas. The relative abundance undergoes CLR transformation. <strong>c</strong> The relative abundance of <em>FSDLZ21_vae_435</em> shows a positive correlation with host age, indicating that older individuals tend to harbor higher levels of this viral species. The relative abundance undergoes CLR transformation. <strong>d</strong> The relative abundance of <em>FSDLZ44_vae_143</em> is negatively associated with defecation frequency, suggesting that increased levels of this viral species may be linked to lower frequency of bowel movements. The relative abundance undergoes CLR transformation. <strong>e</strong> The relative abundance of <em>FGSYC78_vae_185</em> is negatively associated with the frequency of bean consumption, indicating that individuals who consume beans more frequently tend to have lower levels of this viral species. The relative abundance undergoes CLR transformation</p></figcaption></figure><p id="Par64">For instance, participants in urban areas exhibited a higher abundance of <em>FSHXHZ2_vae_115</em> (<em>r</em><sub>spearman</sub>â€‰= -0.32; <em>P</em>â€‰=â€‰4.4â€‰Ã—â€‰10<sup>-54</sup>, Fig. <a href="#Fig5" class="usa-link">5</a>b), while the abundance of <em>FSDLZ21_vae_435</em> increased with age (<em>r</em><sub>spearman</sub>â€‰=â€‰0.27; <em>P</em>â€‰=â€‰5.4â€‰Ã—â€‰10<sup>-51</sup>, Fig. <a href="#Fig5" class="usa-link">5</a>c). Additionally, variations in defecation frequency correlated with viral abundance, as observed with <em>FSDLZ44_vae_143</em>, which showed a negative association with increased defecation frequency (<em>r</em><sub>spearman</sub>â€‰= -0.21; <em>P</em>â€‰=â€‰5.2â€‰Ã—â€‰10<sup>â»18</sup>, Fig. <a href="#Fig5" class="usa-link">5</a>d). Regarding dietary habits, the frequency of bean consumption was negatively associated with <em>FGSYC78_vae_185</em>, a member of Caudoviricetes (<em>r</em><sub>spearman</sub>â€‰=â€‰-0.19; <em>P</em>â€‰=â€‰1.6â€‰Ã—â€‰10<sup>-8</sup>, Fig. <a href="#Fig5" class="usa-link">5</a>e). These findings enhance our understanding of the potential roles of taxonomically unclassified gut viruses in human phenotypic traits, elucidating their potential roles in human health and disease.</p></section><section id="Sec34"><h3 class="pmc_sec_title">Inter-individual gut bacteriome abundance differences is specifically shaped by virome</h3>
<p id="Par65">The human gut virome is primarily dominated by bacteriophages, which regulate bacteria by either lysing them or modifying their physiological functions [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. Among the 20,220 out of 38,436 vOTUs with taxonomy information available in our CGVR, 90.3% (18,253) were identified as bacteriophages (Additional file 1: Table S18); notably, the majority of these phages (76.6%) were temperate in their lifestyle (Additional file 2: Fig. S6). To assess the extent to which inter-individual variations in the gut bacteriome are shaped by the virome, we profiled the abundance of 5310 bacterial species using MetaPhlAn4. Our analysis revealed a positive correlation between the number of identified viruses and the number of detected bacterial species per sample (Fig. <a href="#Fig6" class="usa-link">6</a>a).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369081_40168_2025_2185_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d360/12369081/905c6f4c80ef/40168_2025_2185_Fig6_HTML.jpg" loading="lazy" id="MO6" height="547" width="778" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The gut bacteriome composition variations are specific to virome. <strong>a</strong> Relationship between the number of identified viruses and the number of detected bacterial species per sample. Red density represents the number of detected bacteria, while blue density represents the number of detected viruses. Each dot indicates an individual. <strong>b</strong> Correlation of overall virome and bacteriome composition based on Brayâ€“Curtis distance calculated using abundance data. Each dot represents the Brayâ€“Curtis distance between pairs of individuals. Regression lines are shown in white. <strong>c</strong> Summary of viral abundance associations with bacterial abundance confirmed by CRISPR-spacer imprints. The bar plot summarizes the number of vOTUs with significant associations per bacterial species, while the histogram shows the distribution of correlation coefficients based on viral class. <strong>d</strong> Negative association between <em>FQHXN96_vae_219</em> and <em>E. coli</em> abundances. The relative abundance undergoes CLR transformation. <strong>e</strong> Negative association between <em>FSCPS65_vae_71</em> and <em>B. intestinihominis</em> abundances. The relative abundance undergoes CLR transformation</p></figcaption></figure><p id="Par66">By further calculating the Brayâ€“Curtis distance matrix based on the abundance of 18,253 vOTUs and 5310 bacteria, we found a strong correlation between the two matrices (<em>r</em><sub>spearman</sub>â€‰=â€‰0.92, <em>P</em>â€‰&lt;â€‰2.2â€‰Ã—â€‰10<sup>â»16</sup>, Fig. <a href="#Fig6" class="usa-link">6</a>b). Notably, a total of 84.3% of the bacterial compositional variance can be attributed to the gut virome composition estimated by linear regression, indicating that the human gut bacteriome composition is largely shaped by the virome (Fig. <a href="#Fig6" class="usa-link">6</a>b).</p>
<p id="Par67">In addition to analyzing the overall gut virome and bacteriome composition, we identified 41,844 significant associations involving the abundance of 6131 vOTUs and 302 bacterial species using the MaAsLin2 (<em>FDR</em>â€‰&lt;â€‰0.05, Additional file 1: Table S19). These associations were primarily enriched in the phyla Pseudomonadota, Bacillota, Actinobacteria, and Bacteroidota. To assess the concordance with CRISPR-spacer-based analyses, we focused on significant associations that corresponded with CRISPR-spacer imprints, resulting in a total of 3497 associations (Additional file 1: Table S20, Fig. <a href="#Fig6" class="usa-link">6</a>c). Among these, 2.1% were negative (Fig. <a href="#Fig6" class="usa-link">6</a>c), indicating that lower bacterial abundance was linked to higher viral abundance, suggesting potential viral regulation of bacterial composition.</p>
<p id="Par68">At the individual species level, the most significant viral associations were observed with <em>Escherichia coli</em> (Additional file 2: Fig. S7). For instance, we found that a lower abundance of <em>E. coli</em> was associated with a higher level of <em>FQHXN96_vae_219</em> (<em>r</em><sub>spearman</sub>â€‰=â€‰-0.33; <em>P</em>â€‰=â€‰3.7â€‰Ã—â€‰10<sup>-85</sup>, Fig. <a href="#Fig6" class="usa-link">6</a>d). The emergence of antibiotic-resistant <em>E. coli</em> has become a global concern due to the widespread use of antibiotics [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>â€“<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]. Our results suggest the potential for targeted lysis of <em>E. coli</em> cells by specific viruses, thereby addressing antibiotic resistance and infections related to <em>E. coli</em> [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>].</p>
<p id="Par69">Furthermore, our data revealed associations between the abundance of probiotics and viruses. Probiotics like <em>Barnesiella intestinihominis</em> have been shown to enhance immune responses, improving the effectiveness of chemotherapy and immunotherapy [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. In our findings, higher abundances of <em>B. intestinihominis</em> were associated with lower abundances of <em>FSCPS65_vae_71</em> (<em>r</em><sub>spearman</sub>â€‰=â€‰-0.14; <em>P</em>â€‰=â€‰1.1â€‰Ã—â€‰10<sup>-15</sup>, Fig. <a href="#Fig6" class="usa-link">6</a>e). These results further underscore the potential of targeting gut viruses to regulate bacterial populations related to human health and disease.</p></section><section id="Sec35"><h3 class="pmc_sec_title">The gut viral abundance associations to human phenotypic traits are mediated by bacteriome</h3>
<p id="Par70">Since the abundance of the gut virome can be linked to both the bacteriome and human phenotypic traits, and given that gut viruses like bacteriophages are unlikely to interact directly with human hosts, the viral impact on human phenotypic traits may be mediated by the bacteriome. To evaluate this, we conducted mediation analysis focusing on 966 viruses associated with both the bacteriome and human phenotypic traits (Fig. <a href="#Fig7" class="usa-link">7</a>a). We identified 107 significant mediation linkages involving 88 viral species (<em>FDR</em>â€‰&lt;â€‰0.05, Additional file 1: Table S21), and the main associations are related to individualsâ€™ basic characteristics and lifestyles (Fig. <a href="#Fig7" class="usa-link">7</a>b).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369081_40168_2025_2185_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d360/12369081/788f34396922/40168_2025_2185_Fig7_HTML.jpg" loading="lazy" id="MO7" height="610" width="778" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mediation linkages among the gut virome, bacteriome, and human phenotypes. <strong>a</strong> Venn diagram showing the number of viruses associated with phenotypes and bacteria, respectively. <strong>b</strong> Parallel coordinates chart displaying 107 significant mediation effects of gut microbial species. The left panel shows the viral species (vOTUs), the middle panel presents the microbial species, and the right panel illustrates the human phenotypes. Curved lines across the panels indicate mediation effects, with colors representing different phenotypes. <strong>c</strong>. Microbial species <em>B. uniformis</em> mediates the effect of viral species <em>FHLJYC39_FDME220052688-1a_vae_1390</em> on age. <strong>d</strong> Microbial species <em>A. putredinis</em> mediates the effect of viral species <em>FGDLZ19_FDME220051917-1a_vae_242482</em> on defecation frequency</p></figcaption></figure><p id="Par71">Most of these linkages were related to the viral impact on host phenotypes via <em>Gemmiger formicilis</em> (19 linkages) and <em>Bacteroides uniformis</em> (18 linkages), highlighting the profound influence of viruses on the complex host-microbe ecosystem in the intestine. For instance, the abundance of <em>B. uniformis</em> is positively correlated with age, and the virus <em>FHLJYC39_FDME220052688-1a_vae_1390</em> demonstrates a symbiotic relationship with <em>B. uniformis</em>, mediating its abundance (43%, <em>P</em><sub>mediation</sub>â€‰&lt;â€‰0.001, Fig. <a href="#Fig7" class="usa-link">7</a>c). Additionally, the abundance of vOTU <em>FGDLZ19_FDME220051917-1a_vae_242482</em> negatively associated with defecation frequency, a phenotypic trait that relevant for gastrointestinal issues, which correlates with a reduction in <em>Alistipes putredinis</em> (36%, <em>P</em><sub>mediation</sub>â€‰&lt;â€‰0.001, Fig. <a href="#Fig7" class="usa-link">7</a>d). Taken together, these findings provide potential mechanistic underpinnings for virome-microbiome interactions in human phenotypes.</p></section></section><section id="Sec36"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par72">In this study, we present the CGVR set, comprising 120,568 viral genomes from 3234 fecal samples collected nationwide. Through sequence clustering, we identified 72,751 viral operational taxonomic units (vOTUs), with nearly 90% absent from existing databases [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>â€“<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. We found population-specific genomic differences in common gut viral species between Chinese and non-Chinese populations, which are linked to bacterial host infections. Additionally, the predicted bacterial hosts of these population-stratified viruses exhibit distinct genetic components associated with various health-related functions, such as multidrug resistance. Our analysis revealed that within-species viral genomic variations correlate with host physiological traits at the single-nucleotide level, independent of abundance variations. Moreover, we observed that inter-individual differences in the gut bacteriome are specifically shaped by the virome, identifying 19,108 viral abundance associations with host phenotypes, enriched in geographic, intrinsic, and dietary factors. Finally, mediation analysis suggests that the gut virome may influence human phenotypic traits through interactions with the bacteriome.</p>
<p id="Par73">The human gut virome is a critical part of the microbiome, and characterizing viral genomic diversity and functional variations across global populations is crucial in addressing regional health disparities. However, large-scale construction of viral genomes has primarily focused on samples from Western populations [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>â€“<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>], hindering a full understanding of global gut viral genomic diversity and functional variations at strain-level resolution. To address this limitation and expand the global reference of viral genomes, we compared our Chinese dataset with the UHGV dataset, adding 38,436 novel viral operational taxonomic units (vOTUs) to existing reference genomes. This significantly broadens the representation of viral genomes, particularly for non-Western populations. Our analysis of viral operational taxonomic units (vOTUs) revealed significant population-specific stratification. Previous studies have highlighted the geographic variability of viral species [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>], emphasizing the vast diversity of the human gut virome and the necessity for strain-resolved genome databases for functional interpretation.</p>
<p id="Par74">The gut virome is a crucial component of the microbiome [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>], and understanding virus-bacteria interactions is vital [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. While prior studies linked the virome to bacterial hosts [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>], they often overlooked population-specific features of the gut virome. This is essential for comprehending the mechanisms of virus-specific infections. Using the CRISPR spacer method, we identified that bacterial hosts of population-stratified viruses possess distinct genetic elements linked to health-related functions. Furthermore, our findings show that within-species viral genomic diversity correlates with host phenotypic traits independent of abundance variations.</p>
<p id="Par75">In addition to genetic diversity, viral abundance plays a key role in human health and disease. Through associations between newly characterized vOTUs and bacteria, we identified tens of thousands of potential virus-phenotype and virus-bacteria associations. Mediation analysis revealed over a hundred significant associations between gut viral abundance and human phenotypic traits mediated by the bacteriome. While these findings highlight the potential of targeting gut viruses to modulate bacterial populations relevant to human health, we acknowledge that our cross-sectional design primarily captures static associations, which may not fully reflect the dynamic nature of phage-bacteria interactions. To dissect the temporal dynamics and functional equilibrium of these relationships, future studies should integrate longitudinal multi-omics data.</p>
<p id="Par76">Notably, while previous studies demonstrated the role of viruses in regulating bacterial populations and metabolites through experimental evidence [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>], our large-scale analysis provides observational yet comprehensive insights into the viromeâ€™s impact on human health, offering valuable guidance for future research.</p>
<p id="Par77">We acknowledge several limitations in our study. First, while we collected over 3000 samples from 30 provinces, the sample size remains relatively small compared to Chinaâ€™s population. Second, current virus detection tools may have limitations, leading to inaccuracies in identifying viral sequences [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>, <a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. Third, advanced virome analysis techniques, such as long-read metagenomic sequencing or virus-like particle sequencing, have successfully uncovered hidden diversity in human gut viruses [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. Employing these methods could provide a more comprehensive view of the virome in the Chinese population. Fourth, our observation that 43% of predicted protein sequences lacked detectable homology to known proteins may partially reflect alternative stop codon usage and reassignment. This interpretation is supported by the high prevalence of TGA (30.0%) and TAG (16.7%) stop codons in our dataset of nonhomologous proteins. These findings strongly advocate for the incorporation of alternative genetic code models in future human gut virome research [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>â€“<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]. Lastly, multiple methods can be employed for phylogenetic analysis, including orthologous gene-based approaches and whole-genome SNP-based strategies. Given the high genetic diversity and mosaic nature of viral genomes, especially in human-associated viromes, the integration of these complementary methods can enhance phylogenetic resolution across different evolutionary scales. Future studies should explore combined frameworks that leverage both conserved orthologs (where available) and genome-wide variation to achieve a more comprehensive understanding of viral evolution and strain-level diversity.</p></section><section id="Sec37"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par78">Taken together, our study expands the existing collection of gut viral genomes and offers functional insights into genetic diversity and population specificity of the human gut virome at the single-nucleotide level. This work enhances our understanding of host-microbe interactions and lays the groundwork for developing microbiome-based personalized therapies.</p></section><section id="Sec38"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12369081/bin/40168_2025_2185_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">40168_2025_2185_MOESM1_ESM.xlsx</a><sup> (13.1MB, xlsx) </sup><p>Additional file 1.Â Table S1-S21. Table S1. The quality of 120,568 medium-to-high quality viral genomes evaluated by CheckV. Table S2. Summary of 64,377 viral genomes from 42,803 vOTUs. Table S3. Taxonomy annotation of 42,803 vOTUs using ICTV. Table S4. Summary analysis of functional annotation of CDSs of 18,311 unclassified vOTUs. Table S5. Frequency distribution of stop codon types among predicted viral proteins lacking homology to known sequences. Table S6. Comparison of genetic distances among different populations of 233 vOTUs. Table S7. SNV sites that are significantly different between two populations. Table S8. Host prediction of 44 vOTUs for iPHoP method. Table S9. Host prediction of 44 vOTUs for CRISPR method. Table S10. Matching relationships between viruses and bacteria identified by spacer sequences. Table S11. The intra-group genomic dissimilarity of bacterial hosts corresponding to viruses from different clade. Table S12. SNVs that potentially driving the observed genetic stratification in bacterial hosts. Table S13. The UniRef50 annotation results for driver single nucleotide variants (SNVs). Table S14. Significant associations between viral species genomic and host phenotypic differences. Table S15. Correlations between viral subspecies clades and host phenotypic traits. Table S16. Correlations between vOTUs genomic dissimilarity and vOTU abundance differences. Table S17. Significant associations between viral abundance and phenotypes. Table S18. Taxonomic information of 38,436 vOTUs involved in the CGVR. Table S19. Significant associations between virus and bacteria. Table S20. Summary of 3,497 viral associations to bacterial species confirmed by CRISPR-spacer imprints. Table S21. The significant results of bacteriome mediate virus impacts on phenotypic traits.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12369081/bin/40168_2025_2185_MOESM2_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">40168_2025_2185_MOESM2_ESM.zip</a><sup> (17.7MB, zip) </sup><p>Addtional file 2.Â  Fig. S1-S7. Fig. S1. Phylogenetic tree of viral genomes from 44 vOTUs based on SNP haplotype profile. Red represents viral genomes from Chinese population and blue represents the Non-Chinese population.Â Fig. S2. Functional annotation of viral genes with differential SNVs between populations.Â We conducted functional annotation of the 239 SNVs that exhibit significant population differences and classified them into different functional groups based on a previous study.Â Fig. S3. Associations between viral species genomic and host phenotypic differences. a The genetic distance of L1EFC230390_FSHXXA28_S33_L003_vae_13 decrease with increasing variation in defecating frequency. bÂ The genetic distance of FSHCM57_vae_33 decrease with increasing BMI disparity among hosts.Â Fig. S4.Â Distinct viral subspecies clades based on their phylogenetic trees. Two Distinct viral subspecies clades detected in 31 vOTUs based on their SNP haplotypes. Red and blue represent different subspecies clades. Fig. S5.Â Summary of associations between vOTUs genomic dissimilarity and abundance difference. Red indicates proportion of significant associations between vOTUs genomic dissimilarity and abundance difference at FDR &gt; 0.05, while blue for FDR. Fig. S6.Â Lifestyle of 18,235 bacteriophages. The circular visualization represents three predicted lifestyle categories of bacteriophages. Red for bacteriophages with temperate lifestyle, blue for virulent lifestyle and green for the virus with unknow prediction of lifestyle. Fig. S7.Â Significant associations between vOTUs and bacterial species with CRISPR-spacer based confirmation. In total, 3497 significant associations between vOTUs and bacterial species were characterized, with the most viral associations were observed with Escherichia coli.</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements
</h2>
<p>We thank the participants and management staffs for their collaboration and support.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>L.C., X.K. and Q.Z. contributed to conceptualization and funding. The CGVR Consortium Initiative staffs contributed to data and sample collection. X.W., Q.D., L.C., P.H., S.Y. contributed to data analysis. Q.D., X.W., L.C. drafted the manuscript. X.W., Q.D., P.H., S.Y., M.G., C.Z., C.Z., Y.D., Z.H., B.M., Y.J., Y.Z., T.W., H.Z., J.S., Y.S., Y.W., L.T., S.H., Y.D., W.S., W.C., Q.Z., X.K. and L.C. contributed to discussion of the content. All authors read, revised and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This project was funded by the National Natural Science Foundation of China (NSFC, 32394052, 32270077, and Excellent Young Scientists Fund Program Overseas 2022), Jiangsu Shuangchuang Project (JSSCTD202450), the Natural Science Foundation of Jiangsu (BK20220709), High-Level Talent Cultivation Program (CZ0121002010039 &amp; YNRCZN0301) and Major Basic Research Fund (TS202403) of Jiangsu Province Hospital, Nanjing Medical University (CMCM202204 and GSKY20210105), Suzhou Science and Technology Development Plan Foundation (SKY2021010), Gusu Health Talent Program (GSWS202206), Research Funds of Center for Advanced Interdisciplinary Science and Biomedicine of IHM (QYPY20230033), Fundamental Research Funds for the Central Universities (WK9100000063), Research Funds of Strategic Priority Research Program of the Chinese Academy of Sciences (XDB0940000), Development of Jiangsu Higher Education Institutions Priority Academic Program (PAPD). The funders had no role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data Availability</h2>
<p>The genomic assemblies of gut virus are deposited in National Genomics Data Center (NGDC) under accession id OMIX007646. The metagenomic sequencing data used for the analysis presented in this study are available from the Genome Sequence Archive (GSA) under accession id PRJCA017330.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>Scripts used for the analyses in this manuscript are available via <a href="https://github.com/MicrobiomeCardioMetaLab/CGVR_project" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/MicrobiomeCardioMetaLab/CGVR_project</a>.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Declarations</h2>
<p id="Par79">Ethics approval and consent to participate.</p>
<p id="Par80">The study was approved by the Ethics Review Board of Jiangnan University (reference number JNU20220901IRB01) and included 3,234 participants from 30 provinces across Mainland China. Informed consent was obtained from all participants prior to sample collection.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Consent for publication</h2>
<p id="Par81">Not applicable.</p></section><section id="FPar3"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par82">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Authors Xiuchao Wang, Quanbin Dong, Pan Huang, and Shuai Yang contributed equally and share the first author.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Qixiao Zhai, Email: zhaiqixiao@jiangnan.edu.cn.</p>
<p>Xiangqing Kong, Email: kongxq@njmu.edu.cn.</p>
<p>Lianmin Chen, Email: lianminchen@njmu.edu.cn.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Campbell DE, Ly LK, Ridlon JM, Hsiao A, Whitaker RJ, Degnan PH. Infection with Bacteroides phage BV01 alters the host transcriptome and bile acid metabolism in a common human gut microbe. Cell Rep. 2020;32.</cite> [<a href="https://doi.org/10.1016/j.celrep.2020.108142" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8354205/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32937127/" class="usa-link">PubMed</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Norman JM, Handley SA, Baldridge MT, Droit L, Liu CY, Keller BC, Kambal A, Monaco CL, Zhao G, Fleshner P, et al. Disease-specific alterations in the enteric virome in inflammatory bowel disease. Cell. 2015;160:447â€“60.
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4312520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25619688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Norman%20JM,%20Handley%20SA,%20Baldridge%20MT,%20Droit%20L,%20Liu%20CY,%20Keller%20BC,%20Kambal%20A,%20Monaco%20CL,%20Zhao%20G,%20Fleshner%20P,%20et%20al.%20Disease-specific%20alterations%20in%20the%20enteric%20virome%20in%20inflammatory%20bowel%20disease.%20Cell.%202015;160:447%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>de Jonge PA, Wortelboer K, Scheithauer TPM, van den Born BH, Zwinderman AH, Nobrega FL, Dutilh BE, Nieuwdorp M, Herrema H. Gut virome profiling identifies a widespread bacteriophage family associated with metabolic syndrome. Nat Commun. 2022;13:3594.
</cite> [<a href="https://doi.org/10.1038/s41467-022-31390-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9226167/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35739117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Jonge%20PA,%20Wortelboer%20K,%20Scheithauer%20TPM,%20van%20den%20Born%20BH,%20Zwinderman%20AH,%20Nobrega%20FL,%20Dutilh%20BE,%20Nieuwdorp%20M,%20Herrema%20H.%20Gut%20virome%20profiling%20identifies%20a%20widespread%20bacteriophage%20family%20associated%20with%20metabolic%20syndrome.%20Nat%20Commun.%202022;13:3594." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Lang S, Demir M, Martin A, Jiang L, Zhang X, Duan Y, Gao B, Wisplinghoff H, Kasper P, Roderburg C, et al. Intestinal virome signature associated with severity of nonalcoholic fatty liver disease. Gastroenterology. 2020;159:1839â€“52.
</cite> [<a href="https://doi.org/10.1053/j.gastro.2020.07.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8404510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32652145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lang%20S,%20Demir%20M,%20Martin%20A,%20Jiang%20L,%20Zhang%20X,%20Duan%20Y,%20Gao%20B,%20Wisplinghoff%20H,%20Kasper%20P,%20Roderburg%20C,%20et%20al.%20Intestinal%20virome%20signature%20associated%20with%20severity%20of%20nonalcoholic%20fatty%20liver%20disease.%20Gastroenterology.%202020;159:1839%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Yang K, Niu J, Zuo T, Sun Y, Xu Z, Tang W, Liu Q, Zhang J, Ng EKW, Wong SKH, et al. Alterations in the gut virome in obesity and type 2 diabetes mellitus. Gastroenterology. 2021;161(4):1257-1269.e1213.
</cite> [<a href="https://doi.org/10.1053/j.gastro.2021.06.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34175280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20K,%20Niu%20J,%20Zuo%20T,%20Sun%20Y,%20Xu%20Z,%20Tang%20W,%20Liu%20Q,%20Zhang%20J,%20Ng%20EKW,%20Wong%20SKH,%20et%20al.%20Alterations%20in%20the%20gut%20virome%20in%20obesity%20and%20type%202%20diabetes%20mellitus.%20Gastroenterology.%202021;161(4):1257-1269.e1213." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Nayfach S, PÃ¡ez-Espino D, Call L, Low SJ, Sberro H, Ivanova NN, Proal AD, Fischbach MA, Bhatt AS, Hugenholtz P, Kyrpides NC. Metagenomic compendium of 189,680 DNA viruses from the human gut microbiome. Nat Microbiol. 2021;6:960â€“70.
</cite> [<a href="https://doi.org/10.1038/s41564-021-00928-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8241571/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34168315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nayfach%20S,%20P%C3%A1ez-Espino%20D,%20Call%20L,%20Low%20SJ,%20Sberro%20H,%20Ivanova%20NN,%20Proal%20AD,%20Fischbach%20MA,%20Bhatt%20AS,%20Hugenholtz%20P,%20Kyrpides%20NC.%20Metagenomic%20compendium%20of%20189,680%20DNA%20viruses%20from%20the%20human%20gut%20microbiome.%20Nat%20Microbiol.%202021;6:960%E2%80%9370." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Chen J, Sun C, Dong Y, Jin M, Lai S, Jia L, Zhao X, Wang H, Gao NL, Bork P, et al. Efficient recovery of complete gut viral genomes by combined short- and long-read sequencing. Adv Sci (Weinh). 2024;11.</cite> [<a href="https://doi.org/10.1002/advs.202305818" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10987132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38240578/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Zeng S, Almeida A, Li S, Ying J, Wang H, Qu Y, Paul Ross R, Stanton C, Zhou Z, Niu X, et al. A metagenomic catalog of the early-life human gut virome. Nat Commun. 2024;15(1):1864. 10.1038/s41467-024-45793-z.
</cite> [<a href="https://doi.org/10.1038/s41467-024-45793-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10904392/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38424077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng%20S,%20Almeida%20A,%20Li%20S,%20Ying%20J,%20Wang%20H,%20Qu%20Y,%20Paul%20Ross%20R,%20Stanton%20C,%20Zhou%20Z,%20Niu%20X,%20et%20al.%20A%20metagenomic%20catalog%20of%20the%20early-life%20human%20gut%20virome.%20Nat%20Commun.%202024;15(1):1864.%2010.1038/s41467-024-45793-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Gregory AC, Zablocki O, Zayed AA, Howell A, Bolduc B, Sullivan MB. The gut virome database reveals age-dependent patterns of virome diversity in the human gut. Cell Host Microbe. 2020;28(5):724-740.e728. 10.1016/j.chom.2020.08.003.
</cite> [<a href="https://doi.org/10.1016/j.chom.2020.08.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7443397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32841606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gregory%20AC,%20Zablocki%20O,%20Zayed%20AA,%20Howell%20A,%20Bolduc%20B,%20Sullivan%20MB.%20The%20gut%20virome%20database%20reveals%20age-dependent%20patterns%20of%20virome%20diversity%20in%20the%20human%20gut.%20Cell%20Host%20Microbe.%202020;28(5):724-740.e728.%2010.1016/j.chom.2020.08.003." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Camarillo-Guerrero LF, Almeida A, Rangel-Pineros G, Finn RD, Lawley TD. Massive expansion of human gut bacteriophage diversity. Cell. 2021;184:1098-1109.e1099.
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.01.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7895897/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33606979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Camarillo-Guerrero%20LF,%20Almeida%20A,%20Rangel-Pineros%20G,%20Finn%20RD,%20Lawley%20TD.%20Massive%20expansion%20of%20human%20gut%20bacteriophage%20diversity.%20Cell.%202021;184:1098-1109.e1099." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Camargo AP, Nayfach S, Chen I-MA, Palaniappan K, Ratner A, Chu K, Ritter StephanÂ J, Reddy TBK, Mukherjee S, Schulz F, et al: IMG/VR v4: an expanded database of uncultivated virus genomes within a framework of extensive functional, taxonomic, and ecological metadata. Nucleic Acids Res. 2022;51:D733-D743.</cite> [<a href="https://doi.org/10.1093/nar/gkac1037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36399502/" class="usa-link">PubMed</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Lai S, Jia L, Subramanian B, Pan S, Zhang J, Dong Y, Chen W-H, Zhao X-M. mMGE: a database for human metagenomic extrachromosomal mobile genetic elements. Nucleic Acids Res. 2020;49:D783â€“91.</cite> [<a href="https://doi.org/10.1093/nar/gkaa869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7778953/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33074335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lai%20S,%20Jia%20L,%20Subramanian%20B,%20Pan%20S,%20Zhang%20J,%20Dong%20Y,%20Chen%20W-H,%20Zhao%20X-M.%20mMGE:%20a%20database%20for%20human%20metagenomic%20extrachromosomal%20mobile%20genetic%20elements.%20Nucleic%20Acids%20Res.%202020;49:D783%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Tisza MJ, Buck CB. A catalog of tens of thousands of viruses from human metagenomes reveals hidden associations with chronic diseases. Proc Natl Acad Sci USA. 2021;118(23). 10.1073/pnas.2023202118.</cite> [<a href="https://doi.org/10.1073/pnas.2023202118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8201803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34083435/" class="usa-link">PubMed</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Tisza MJ, Buck CB: A catalog of tens of thousands of viruses from human metagenomes reveals hidden associations with chronic diseases. Proc Natl Acad Sci USA. 2021;118.</cite> [<a href="https://doi.org/10.1073/pnas.2023202118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8201803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34083435/" class="usa-link">PubMed</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Soto-Perez P, Bisanz JE, Berry JD, Lam KN, Bondy-Denomy J, Turnbaugh PJ. CRISPR-Cas system of a prevalent human gut bacterium reveals hyper-targeting against phages in a human virome catalog. Cell Host Microbe. 2019;26:325-335.e325.
</cite> [<a href="https://doi.org/10.1016/j.chom.2019.08.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6936622/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31492655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Soto-Perez%20P,%20Bisanz%20JE,%20Berry%20JD,%20Lam%20KN,%20Bondy-Denomy%20J,%20Turnbaugh%20PJ.%20CRISPR-Cas%20system%20of%20a%20prevalent%20human%20gut%20bacterium%20reveals%20hyper-targeting%20against%20phages%20in%20a%20human%20virome%20catalog.%20Cell%20Host%20Microbe.%202019;26:325-335.e325." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Shah SA, Deng L, Thorsen J, Pedersen AG, Dion MB, Castro-MejÃ­a JL, Silins R, Romme FO, Sausset R, Ndela EO, et al. Hundreds of viral families in the healthy infant gut. 2023;8(5):986â€“98. 10.1038/s41564-023-01345-7.</cite> [<a href="https://doi.org/10.1038/s41564-023-01345-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10159846/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37037943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shah%20SA,%20Deng%20L,%20Thorsen%20J,%20Pedersen%20AG,%20Dion%20MB,%20Castro-Mej%C3%ADa%20JL,%20Silins%20R,%20Romme%20FO,%20Sausset%20R,%20Ndela%20EO,%20et%20al.%20Hundreds%20of%20viral%20families%20in%20the%20healthy%20infant%20gut.%202023;8(5):986%E2%80%9398.%2010.1038/s41564-023-01345-7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Benler S, Yutin N, Antipov D, Rayko M, Shmakov S, Gussow AB, Pevzner P, Koonin EV. Thousands of previously unknown phages discovered in whole-community human gut metagenomes. Microbiome. 2021;9(1):78. 10.1186/s40168-021-01017-w.
</cite> [<a href="https://doi.org/10.1186/s40168-021-01017-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8008677/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33781338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Benler%20S,%20Yutin%20N,%20Antipov%20D,%20Rayko%20M,%20Shmakov%20S,%20Gussow%20AB,%20Pevzner%20P,%20Koonin%20EV.%20Thousands%20of%20previously%20unknown%20phages%20discovered%20in%20whole-community%20human%20gut%20metagenomes.%20Microbiome.%202021;9(1):78.%2010.1186/s40168-021-01017-w." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Van Espen L, Bak EG, Beller L, Close L, Deboutte W, Juel HB, Nielsen T, Sinar D, De Coninck L, Frithioff-BÃ¸jsÃ¸e C, et al: A previously undescribed highly prevalent phage identified in a Danish enteric virome catalog. mSystems.Â 2021;6:e0038221.</cite> [<a href="https://doi.org/10.1128/mSystems.00382-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8525569/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34665009/" class="usa-link">PubMed</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Garmaeva S, Gulyaeva A, Sinha T, Shkoporov AN, Clooney AG, Stockdale SR, Spreckels JE, Sutton TDS, Draper LA, Dutilh BE, et al. Stability of the human gut virome and effect of gluten-free diet. Cell Rep. 2021;35.</cite> [<a href="https://doi.org/10.1016/j.celrep.2021.109132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34010651/" class="usa-link">PubMed</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Zuo T, Sun Y, Wan Y, Yeoh YK, Zhang F, Cheung CP, Chen N, Luo J, Wang W, Sung JJY, et al. Human-gut-DNA virome variations across geography, ethnicity, and urbanization. Cell Host Microbe. 2020;28:741-751.e744.
</cite> [<a href="https://doi.org/10.1016/j.chom.2020.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32910902/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zuo%20T,%20Sun%20Y,%20Wan%20Y,%20Yeoh%20YK,%20Zhang%20F,%20Cheung%20CP,%20Chen%20N,%20Luo%20J,%20Wang%20W,%20Sung%20JJY,%20et%20al.%20Human-gut-DNA%20virome%20variations%20across%20geography,%20ethnicity,%20and%20urbanization.%20Cell%20Host%20Microbe.%202020;28:741-751.e744." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Zhao L, Shi Y, Lau HC, Liu W, Luo G, Wang G, Liu C, Pan Y, Zhou Q, Ding Y, et al. Uncovering 1058 novel human enteric DNA viruses through deep long-read third-generation sequencing and their clinical impact. Gastroenterology. 2022;163:699â€“711.
</cite> [<a href="https://doi.org/10.1053/j.gastro.2022.05.048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35679948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20L,%20Shi%20Y,%20Lau%20HC,%20Liu%20W,%20Luo%20G,%20Wang%20G,%20Liu%20C,%20Pan%20Y,%20Zhou%20Q,%20Ding%20Y,%20et%20al.%20Uncovering%201058%20novel%20human%20enteric%20DNA%20viruses%20through%20deep%20long-read%20third-generation%20sequencing%20and%20their%20clinical%20impact.%20Gastroenterology.%202022;163:699%E2%80%93711." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Cui J, Li F, Shi ZL. Origin and evolution of pathogenic coronaviruses. Nat Rev Microbiol. 2019;17:181â€“92.
</cite> [<a href="https://doi.org/10.1038/s41579-018-0118-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7097006/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30531947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cui%20J,%20Li%20F,%20Shi%20ZL.%20Origin%20and%20evolution%20of%20pathogenic%20coronaviruses.%20Nat%20Rev%20Microbiol.%202019;17:181%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Dutilh BE, Cassman N, McNair K, Sanchez SE, Silva GG, Boling L, Barr JJ, Speth DR, Seguritan V, Aziz RK, et al. A highly abundant bacteriophage discovered in the unknown sequences of human faecal metagenomes. Nat Commun. 2014;5:4498.
</cite> [<a href="https://doi.org/10.1038/ncomms5498" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4111155/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25058116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dutilh%20BE,%20Cassman%20N,%20McNair%20K,%20Sanchez%20SE,%20Silva%20GG,%20Boling%20L,%20Barr%20JJ,%20Speth%20DR,%20Seguritan%20V,%20Aziz%20RK,%20et%20al.%20A%20highly%20abundant%20bacteriophage%20discovered%20in%20the%20unknown%20sequences%20of%20human%20faecal%20metagenomes.%20Nat%20Commun.%202014;5:4498." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite> LÃ³pez-BeltrÃ¡n A, Botelho J, Iranzo J: Dynamics of CRISPR-mediated virus-host interactions in the human gut microbiome. Isme j. 2024;18(1):wrae134.</cite> [<a href="https://doi.org/10.1093/ismejo/wrae134" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11307328/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39023219/" class="usa-link">PubMed</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Madi N, Cato ET, Abu Sayeed M, Creasy-Marrazzo A, CuÃ©nod A, Islam K, Khabir MIU, Bhuiyan MTR, Begum YA, Freeman E, et al: Phage predation, disease severity, and pathogen genetic diversity in cholera patients. Science 2024;384:eadj3166.</cite> [<a href="https://doi.org/10.1126/science.adj3166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38669570/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Huang P, Dong Q, Wang Y, Tian Y, Wang S, Zhang C, Yu L, Tian F, Gao X, Guo H, et al. Gut microbial genomes with paired isolates from China illustrate probiotic and cardiometabolic effects. Cell Genom. 2024;4(6):100559. 10.1016/j.xgen.2024.100559.</cite> [<a href="https://doi.org/10.1016/j.xgen.2024.100559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11228888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38740021/" class="usa-link">PubMed</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Bolger AM, Lohse M, Usadel B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics. 2014;30:2114â€“20.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4103590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bolger%20AM,%20Lohse%20M,%20Usadel%20B.%20Trimmomatic:%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data.%20Bioinformatics.%202014;30:2114%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9:357â€“9.
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead%20B,%20Salzberg%20SL.%20Fast%20gapped-read%20alignment%20with%20Bowtie%202.%20Nat%20Methods.%202012;9:357%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Li D, Liu CM, Luo R, Sadakane K, Lam TW. MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph. Bioinformatics. 2015;31:1674â€“6.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25609793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20D,%20Liu%20CM,%20Luo%20R,%20Sadakane%20K,%20Lam%20TW.%20MEGAHIT:%20an%20ultra-fast%20single-node%20solution%20for%20large%20and%20complex%20metagenomics%20assembly%20via%20succinct%20de%20Bruijn%20graph.%20Bioinformatics.%202015;31:1674%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Nissen JN, Johansen J, AllesÃ¸e RL, SÃ¸nderby CK, Armenteros JJA, GrÃ¸nbech CH, Jensen LJ, Nielsen HB, Petersen TN, Winther O, Rasmussen S. Improved metagenome binning and assembly using deep variational autoencoders. Nat Biotechnol. 2021;39:555â€“60.
</cite> [<a href="https://doi.org/10.1038/s41587-020-00777-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33398153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nissen%20JN,%20Johansen%20J,%20Alles%C3%B8e%20RL,%20S%C3%B8nderby%20CK,%20Armenteros%20JJA,%20Gr%C3%B8nbech%20CH,%20Jensen%20LJ,%20Nielsen%20HB,%20Petersen%20TN,%20Winther%20O,%20Rasmussen%20S.%20Improved%20metagenome%20binning%20and%20assembly%20using%20deep%20variational%20autoencoders.%20Nat%20Biotechnol.%202021;39:555%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Johansen J, Plichta DR, Nissen JN, Jespersen ML, Shah SA, Deng L, Stokholm J, Bisgaard H, Nielsen DS, SÃ¸rensen SJ, Rasmussen S. Genome binning of viral entities from bulk metagenomics data. Nat Commun. 2022;13:965.
</cite> [<a href="https://doi.org/10.1038/s41467-022-28581-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8857322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35181661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johansen%20J,%20Plichta%20DR,%20Nissen%20JN,%20Jespersen%20ML,%20Shah%20SA,%20Deng%20L,%20Stokholm%20J,%20Bisgaard%20H,%20Nielsen%20DS,%20S%C3%B8rensen%20SJ,%20Rasmussen%20S.%20Genome%20binning%20of%20viral%20entities%20from%20bulk%20metagenomics%20data.%20Nat%20Commun.%202022;13:965." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Nayfach S, Camargo AP, Schulz F, Eloe-Fadrosh E, Roux S, Kyrpides NC. CheckV assesses the quality and completeness of metagenome-assembled viral genomes. Nat Biotechnol. 2021;39:578â€“85.
</cite> [<a href="https://doi.org/10.1038/s41587-020-00774-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8116208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33349699/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nayfach%20S,%20Camargo%20AP,%20Schulz%20F,%20Eloe-Fadrosh%20E,%20Roux%20S,%20Kyrpides%20NC.%20CheckV%20assesses%20the%20quality%20and%20completeness%20of%20metagenome-assembled%20viral%20genomes.%20Nat%20Biotechnol.%202021;39:578%E2%80%9385." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>McGinnis S, Madden TL. BLAST: at the core of a powerful and diverse set of sequence analysis tools. Nucleic Acids Res. 2004;32:W20-25.
</cite> [<a href="https://doi.org/10.1093/nar/gkh435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC441573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15215342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McGinnis%20S,%20Madden%20TL.%20BLAST:%20at%20the%20core%20of%20a%20powerful%20and%20diverse%20set%20of%20sequence%20analysis%20tools.%20Nucleic%20Acids%20Res.%202004;32:W20-25." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Roux S, Adriaenssens EM, Dutilh BE, Koonin EV, Kropinski AM, Krupovic M, Kuhn JH, Lavigne R, Brister JR, Varsani A, et al. Minimum Information about an Uncultivated Virus Genome (MIUViG). Nat Biotechnol. 2019;37:29â€“37.
</cite> [<a href="https://doi.org/10.1038/nbt.4306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6871006/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30556814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roux%20S,%20Adriaenssens%20EM,%20Dutilh%20BE,%20Koonin%20EV,%20Kropinski%20AM,%20Krupovic%20M,%20Kuhn%20JH,%20Lavigne%20R,%20Brister%20JR,%20Varsani%20A,%20et%20al.%20Minimum%20Information%20about%20an%20Uncultivated%20Virus%20Genome%20(MIUViG).%20Nat%20Biotechnol.%202019;37:29%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Hyatt D, Chen GL, Locascio PF, Land ML, Larimer FW, Hauser LJ. Prodigal: prokaryotic gene recognition and translation initiation site identification. BMC Bioinformatics. 2010;11:119.
</cite> [<a href="https://doi.org/10.1186/1471-2105-11-119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2848648/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20211023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hyatt%20D,%20Chen%20GL,%20Locascio%20PF,%20Land%20ML,%20Larimer%20FW,%20Hauser%20LJ.%20Prodigal:%20prokaryotic%20gene%20recognition%20and%20translation%20initiation%20site%20identification.%20BMC%20Bioinformatics.%202010;11:119." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Cantalapiedra CP, Hernandez-Plaza A, Letunic I, Bork P, Huerta-Cepas J. eggNOG-mapper v2: functional annotation, orthology assignments, and domain prediction at the metagenomic scale. Mol Biol Evol. 2021;38:5825â€“9.
</cite> [<a href="https://doi.org/10.1093/molbev/msab293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8662613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34597405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cantalapiedra%20CP,%20Hernandez-Plaza%20A,%20Letunic%20I,%20Bork%20P,%20Huerta-Cepas%20J.%20eggNOG-mapper%20v2:%20functional%20annotation,%20orthology%20assignments,%20and%20domain%20prediction%20at%20the%20metagenomic%20scale.%20Mol%20Biol%20Evol.%202021;38:5825%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Huerta-Cepas J, Szklarczyk D, Heller D, HernÃ¡ndez-Plaza A, Forslund SK, Cook H, Mende DR, Letunic I, Rattei T, Jensen LJ, et al: eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Nucleic Acids Res 2019;47:D309-d314.</cite> [<a href="https://doi.org/10.1093/nar/gky1085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6324079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30418610/" class="usa-link">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Kanehisa M, Goto S. KEGG: Kyoto Encyclopedia of Genes and Genomes. Nucleic Acids Res. 2000;28:27â€“30.
</cite> [<a href="https://doi.org/10.1093/nar/28.1.27" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa%20M,%20Goto%20S.%20KEGG:%20Kyoto%20Encyclopedia%20of%20Genes%20and%20Genomes.%20Nucleic%20Acids%20Res.%202000;28:27%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Mistry J, Chuguransky S, Williams L, Qureshi M, Salazar GA, Sonnhammer ELL, Tosatto SCE, Paladin L, Raj S, Richardson LJ, et al. Pfam: the protein families database in 2021. Nucleic Acids Res. 2021;49:D412-d419.
</cite> [<a href="https://doi.org/10.1093/nar/gkaa913" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7779014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33125078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mistry%20J,%20Chuguransky%20S,%20Williams%20L,%20Qureshi%20M,%20Salazar%20GA,%20Sonnhammer%20ELL,%20Tosatto%20SCE,%20Paladin%20L,%20Raj%20S,%20Richardson%20LJ,%20et%20al.%20Pfam:%20the%20protein%20families%20database%20in%202021.%20Nucleic%20Acids%20Res.%202021;49:D412-d419." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Song W, Sun HX, Zhang C, Cheng L, Peng Y, Deng Z, Wang D, Wang Y, Hu M, Liu W, et al. Prophage Hunter: an integrative hunting tool for active prophages. Nucleic Acids Res. 2019;47:W74-w80.
</cite> [<a href="https://doi.org/10.1093/nar/gkz380" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6602508/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31114893/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song%20W,%20Sun%20HX,%20Zhang%20C,%20Cheng%20L,%20Peng%20Y,%20Deng%20Z,%20Wang%20D,%20Wang%20Y,%20Hu%20M,%20Liu%20W,%20et%20al.%20Prophage%20Hunter:%20an%20integrative%20hunting%20tool%20for%20active%20prophages.%20Nucleic%20Acids%20Res.%202019;47:W74-w80." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Rice P, Longden I, Bleasby A. EMBOSS: the European Molecular Biology Open Software Suite. Trends Genet. 2000;16:276â€“7.
</cite> [<a href="https://doi.org/10.1016/s0168-9525(00)02024-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10827456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rice%20P,%20Longden%20I,%20Bleasby%20A.%20EMBOSS:%20the%20European%20Molecular%20Biology%20Open%20Software%20Suite.%20Trends%20Genet.%202000;16:276%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Dong Q, Hua D, Wang X, Jiao Y, Liu L, Deng Q, Wu T, Zou H, Zhao C, Wang C, et al: Temporal colonization and metabolic regulation of the gut microbiome in neonatal oxen at single nucleotide resolution. Isme j 2024;18.</cite> [<a href="https://doi.org/10.1093/ismejo/wrad022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10833086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38365257/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Abramson J, Adler J, Dunger J, Evans R, Green T, Pritzel A, Ronneberger O, Willmore L, Ballard AJ, Bambrick J, et al: Accurate structure prediction of biomolecular interactions with AlphaFold 3<strong>.</strong> Nature 2024;630(8016):493-500.</cite> [<a href="https://doi.org/10.1038/s41586-024-07487-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11168924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38718835/" class="usa-link">PubMed</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Gligorijevic V, Renfrew PD, Kosciolek T, Leman JK, Berenberg D, Vatanen T, Chandler C, Taylor BC, Fisk IM, Vlamakis H, et al. Structure-based protein function prediction using graph convolutional networks. Nat Commun. 2021;12:3168.
</cite> [<a href="https://doi.org/10.1038/s41467-021-23303-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8155034/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34039967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gligorijevic%20V,%20Renfrew%20PD,%20Kosciolek%20T,%20Leman%20JK,%20Berenberg%20D,%20Vatanen%20T,%20Chandler%20C,%20Taylor%20BC,%20Fisk%20IM,%20Vlamakis%20H,%20et%20al.%20Structure-based%20protein%20function%20prediction%20using%20graph%20convolutional%20networks.%20Nat%20Commun.%202021;12:3168." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Dong Q, Ma B, Zhou X, Huang P, Gao M, Yang S, Jiao Y, Zhou Y, Shi Z, Deng Q, et al: Expanded gut microbial genomes from Chinese populations reveal population-specific genomic features related to human physiological traits. 2024. 10.1101/2024.10.19.618999.</cite>
</li>
<li id="CR46">
<span class="label">46.</span><cite>Bland C, Ramsey TL, Sabree F, Lowe M, Brown K, Kyrpides NC, Hugenholtz P. CRISPR recognition tool (CRT): a tool for automatic detection of clustered regularly interspaced palindromic repeats. BMC Bioinformatics. 2007;8:209.
</cite> [<a href="https://doi.org/10.1186/1471-2105-8-209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1924867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17577412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bland%20C,%20Ramsey%20TL,%20Sabree%20F,%20Lowe%20M,%20Brown%20K,%20Kyrpides%20NC,%20Hugenholtz%20P.%20CRISPR%20recognition%20tool%20(CRT):%20a%20tool%20for%20automatic%20detection%20of%20clustered%20regularly%20interspaced%20palindromic%20repeats.%20BMC%20Bioinformatics.%202007;8:209." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Edgar RC. PILER-CR: fast and accurate identification of CRISPR repeats. BMC Bioinformatics. 2007;8:18.
</cite> [<a href="https://doi.org/10.1186/1471-2105-8-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1790904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17239253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Edgar%20RC.%20PILER-CR:%20fast%20and%20accurate%20identification%20of%20CRISPR%20repeats.%20BMC%20Bioinformatics.%202007;8:18." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Roux S, Camargo AP, Coutinho FH, Dabdoub SM, Dutilh BE, Nayfach S, Tritt A. iPHoP: an integrated machine learning framework to maximize host prediction for metagenome-derived viruses of archaea and bacteria. PLoS Biol. 2023;21.</cite> [<a href="https://doi.org/10.1371/journal.pbio.3002083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10155999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37083735/" class="usa-link">PubMed</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Steinegger M, SÃ¶ding J. Clustering huge protein sequence sets in linear time. Nat Commun. 2018;9:2542.
</cite> [<a href="https://doi.org/10.1038/s41467-018-04964-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6026198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29959318/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Steinegger%20M,%20S%C3%B6ding%20J.%20Clustering%20huge%20protein%20sequence%20sets%20in%20linear%20time.%20Nat%20Commun.%202018;9:2542." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Blanco-MÃ­guez A, Beghini F, Cumbo F, McIver LJ, Thompson KN, Zolfo M, Manghi P, Dubois L, Huang KD, Thomas AM, et al. Extending and improving metagenomic taxonomic profiling with uncharacterized species using MetaPhlAn 4. Nat Biotechnol. 2023;41:1633â€“44.
</cite> [<a href="https://doi.org/10.1038/s41587-023-01688-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10635831/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36823356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blanco-M%C3%ADguez%20A,%20Beghini%20F,%20Cumbo%20F,%20McIver%20LJ,%20Thompson%20KN,%20Zolfo%20M,%20Manghi%20P,%20Dubois%20L,%20Huang%20KD,%20Thomas%20AM,%20et%20al.%20Extending%20and%20improving%20metagenomic%20taxonomic%20profiling%20with%20uncharacterized%20species%20using%20MetaPhlAn%204.%20Nat%20Biotechnol.%202023;41:1633%E2%80%9344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Benjamini Y, Drai D, Elmer G, Kafkafi N, Golani I. Controlling the false discovery rate in behavior genetics research. Behav Brain Res. 2001;125:279â€“84.
</cite> [<a href="https://doi.org/10.1016/s0166-4328(01)00297-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11682119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Benjamini%20Y,%20Drai%20D,%20Elmer%20G,%20Kafkafi%20N,%20Golani%20I.%20Controlling%20the%20false%20discovery%20rate%20in%20behavior%20genetics%20research.%20Behav%20Brain%20Res.%202001;125:279%E2%80%9384." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Deng Q, Sun Y, Gu M, Fu J, Xu F, Jiao Y, Zhou Y, Ma B, Liu L, Wang X, et al: Food pairing pattern independently associated with cardiometabolic traits beyond dietary indices across eastern and western populations. Engineering 2025.</cite>
</li>
<li id="CR53">
<span class="label">53.</span><cite>The new scope of virus taxonomy. partitioning the virosphere into 15 hierarchical ranks. Nat Microbiol. 2020;5:668â€“74.
</cite> [<a href="https://doi.org/10.1038/s41564-020-0709-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7186216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32341570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?The%20new%20scope%20of%20virus%20taxonomy.%20partitioning%20the%20virosphere%20into%2015%20hierarchical%20ranks.%20Nat%20Microbiol.%202020;5:668%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Nishijima S, Nagata N, Kiguchi Y, Kojima Y, Miyoshi-Akiyama T, Kimura M, Ohsugi M, Ueki K, Oka S, Mizokami M, et al. Extensive gut virome variation and its associations with host and environmental factors in a population-level cohort. Nat Commun. 2022;13:5252.
</cite> [<a href="https://doi.org/10.1038/s41467-022-32832-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9448778/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36068216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nishijima%20S,%20Nagata%20N,%20Kiguchi%20Y,%20Kojima%20Y,%20Miyoshi-Akiyama%20T,%20Kimura%20M,%20Ohsugi%20M,%20Ueki%20K,%20Oka%20S,%20Mizokami%20M,%20et%20al.%20Extensive%20gut%20virome%20variation%20and%20its%20associations%20with%20host%20and%20environmental%20factors%20in%20a%20population-level%20cohort.%20Nat%20Commun.%202022;13:5252." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Lou YC, Chen L, Borges AL, West-Roberts J, Firek BA, Morowitz MJ, Banfield JF. Infant gut DNA bacteriophage strain persistence during the first 3 years of life. Cell Host Microbe. 2024;32:35-47.e36.
</cite> [<a href="https://doi.org/10.1016/j.chom.2023.11.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11156429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38096814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lou%20YC,%20Chen%20L,%20Borges%20AL,%20West-Roberts%20J,%20Firek%20BA,%20Morowitz%20MJ,%20Banfield%20JF.%20Infant%20gut%20DNA%20bacteriophage%20strain%20persistence%20during%20the%20first%203%20years%20of%20life.%20Cell%20Host%20Microbe.%202024;32:35-47.e36." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Shah SA, Deng L, Thorsen J, Pedersen AG, Dion MB, Castro-MejÃ­a JL, Silins R, Romme FO, Sausset R, Jessen LE, et al. Expanding known viral diversity in the healthy infant gut. Nat Microbiol. 2023;8:986â€“98.
</cite> [<a href="https://doi.org/10.1038/s41564-023-01345-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10159846/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37037943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shah%20SA,%20Deng%20L,%20Thorsen%20J,%20Pedersen%20AG,%20Dion%20MB,%20Castro-Mej%C3%ADa%20JL,%20Silins%20R,%20Romme%20FO,%20Sausset%20R,%20Jessen%20LE,%20et%20al.%20Expanding%20known%20viral%20diversity%20in%20the%20healthy%20infant%20gut.%20Nat%20Microbiol.%202023;8:986%E2%80%9398." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Dedeo CL, Cingolani G, Teschke CM. Portal protein: the orchestrator of capsid assembly for the dsDNA tailed bacteriophages and herpesviruses. Annu Rev Virol. 2019;6:141â€“60.
</cite> [<a href="https://doi.org/10.1146/annurev-virology-092818-015819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6947915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31337287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dedeo%20CL,%20Cingolani%20G,%20Teschke%20CM.%20Portal%20protein:%20the%20orchestrator%20of%20capsid%20assembly%20for%20the%20dsDNA%20tailed%20bacteriophages%20and%20herpesviruses.%20Annu%20Rev%20Virol.%202019;6:141%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Carlson HK, Piya D, Moore ML, Magar RT, Elisabeth NH, Deutschbauer AM, Arkin AP, Mutalik VK. Geochemical constraints on bacteriophage infectivity in terrestrial environments. ISME Commun. 2023;3:78.
</cite> [<a href="https://doi.org/10.1038/s43705-023-00297-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10439110/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37596312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carlson%20HK,%20Piya%20D,%20Moore%20ML,%20Magar%20RT,%20Elisabeth%20NH,%20Deutschbauer%20AM,%20Arkin%20AP,%20Mutalik%20VK.%20Geochemical%20constraints%20on%20bacteriophage%20infectivity%20in%20terrestrial%20environments.%20ISME%20Commun.%202023;3:78." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Deveau H, Garneau JE, Moineau S. CRISPR/Cas system and its role in phage-bacteria interactions. Annu Rev Microbiol. 2010;64:475â€“93.
</cite> [<a href="https://doi.org/10.1146/annurev.micro.112408.134123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20528693/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deveau%20H,%20Garneau%20JE,%20Moineau%20S.%20CRISPR/Cas%20system%20and%20its%20role%20in%20phage-bacteria%20interactions.%20Annu%20Rev%20Microbiol.%202010;64:475%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Shmakov SA, Wolf YI, Savitskaya E, Severinov KV, Koonin EV. Mapping CRISPR spaceromes reveals vast host-specific viromes of prokaryotes. Commun Biol. 2020;3:321.
</cite> [<a href="https://doi.org/10.1038/s42003-020-1014-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7308287/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32572116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shmakov%20SA,%20Wolf%20YI,%20Savitskaya%20E,%20Severinov%20KV,%20Koonin%20EV.%20Mapping%20CRISPR%20spaceromes%20reveals%20vast%20host-specific%20viromes%20of%20prokaryotes.%20Commun%20Biol.%202020;3:321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Akhtar AA, Turner DP. The role of bacterial ATP-binding cassette (ABC) transporters in pathogenesis and virulence: therapeutic and vaccine potential. Microb Pathog. 2022;171.</cite> [<a href="https://doi.org/10.1016/j.micpath.2022.105734" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36007845/" class="usa-link">PubMed</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Sodani K, Patel A, Kathawala RJ, Chen ZS. Multidrug resistance associated proteins in multidrug resistance. Chin J Cancer. 2012;31:58â€“72.
</cite> [<a href="https://doi.org/10.5732/cjc.011.10329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3777468/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22098952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sodani%20K,%20Patel%20A,%20Kathawala%20RJ,%20Chen%20ZS.%20Multidrug%20resistance%20associated%20proteins%20in%20multidrug%20resistance.%20Chin%20J%20Cancer.%202012;31:58%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Mallick H, Rahnavard A, McIver LJ, Ma S, Zhang Y, Nguyen LH, Tickle TL, Weingart G, Ren B, Schwager EH, et al. Multivariable association discovery in population-scale meta-omics studies. PLoS Comput Biol. 2021;17.</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1009442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8714082/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34784344/" class="usa-link">PubMed</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Shkoporov AN, Hill C. Bacteriophages of the human gut: the â€œKnown Unknownâ€ of the microbiome. Cell Host Microbe. 2019;25:195â€“209.
</cite> [<a href="https://doi.org/10.1016/j.chom.2019.01.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30763534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shkoporov%20AN,%20Hill%20C.%20Bacteriophages%20of%20the%20human%20gut:%20the%20%E2%80%9CKnown%20Unknown%E2%80%9D%20of%20the%20microbiome.%20Cell%20Host%20Microbe.%202019;25:195%E2%80%93209." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Hammerum AM, Heuer OE. Human health hazards from antimicrobial-resistant Escherichia coli of animal origin. Clin Infect Dis. 2009;48:916â€“21.
</cite> [<a href="https://doi.org/10.1086/597292" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19231979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hammerum%20AM,%20Heuer%20OE.%20Human%20health%20hazards%20from%20antimicrobial-resistant%20Escherichia%20coli%20of%20animal%20origin.%20Clin%20Infect%20Dis.%202009;48:916%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Pormohammad A, Nasiri MJ, Azimi T. Prevalence of antibiotic resistance in Escherichia coli strains simultaneously isolated from humans, animals, food, and the environment: a systematic review and meta-analysis. Infect Drug Resist. 2019;12:1181â€“97.
</cite> [<a href="https://doi.org/10.2147/IDR.S201324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6512575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31190907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pormohammad%20A,%20Nasiri%20MJ,%20Azimi%20T.%20Prevalence%20of%20antibiotic%20resistance%20in%20Escherichia%20coli%20strains%20simultaneously%20isolated%20from%20humans,%20animals,%20food,%20and%20the%20environment:%20a%20systematic%20review%20and%20meta-analysis.%20Infect%20Drug%20Resist.%202019;12:1181%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite> Poirel L, Madec JY, Lupo A, Schink AK, Kieffer N, Nordmann P, Schwarz S: Antimicrobial resistance in Escherichia coli. Microbiol Spectr 2018;6.</cite> [<a href="https://doi.org/10.1128/microbiolspec.arba-0026-2017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11633601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30003866/" class="usa-link">PubMed</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Federici S, Kredo-Russo S, ValdÃ©s-Mas R, Kviatcovsky D, Weinstock E, Matiuhin Y, Silberberg Y, Atarashi K, Furuichi M, Oka A, et al. Targeted suppression of human IBD-associated gut microbiota commensals by phage consortia for treatment of intestinal inflammation. Cell. 2022;185:2879-2898.e2824.
</cite> [<a href="https://doi.org/10.1016/j.cell.2022.07.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35931020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Federici%20S,%20Kredo-Russo%20S,%20Vald%C3%A9s-Mas%20R,%20Kviatcovsky%20D,%20Weinstock%20E,%20Matiuhin%20Y,%20Silberberg%20Y,%20Atarashi%20K,%20Furuichi%20M,%20Oka%20A,%20et%20al.%20Targeted%20suppression%20of%20human%20IBD-associated%20gut%20microbiota%20commensals%20by%20phage%20consortia%20for%20treatment%20of%20intestinal%20inflammation.%20Cell.%202022;185:2879-2898.e2824." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Fongang B, Satizabal C, Kautz TF, Wadop YN, Muhammad JAS, Vasquez E, Mathews J, Gireud-Goss M, Saklad AR, Himali J, et al. Cerebral small vessel disease burden is associated with decreased abundance of gut Barnesiella intestinihominis bacterium in the Framingham Heart Study. Sci Rep. 2023;13:13622.
</cite> [<a href="https://doi.org/10.1038/s41598-023-40872-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10442369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37604954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fongang%20B,%20Satizabal%20C,%20Kautz%20TF,%20Wadop%20YN,%20Muhammad%20JAS,%20Vasquez%20E,%20Mathews%20J,%20Gireud-Goss%20M,%20Saklad%20AR,%20Himali%20J,%20et%20al.%20Cerebral%20small%20vessel%20disease%20burden%20is%20associated%20with%20decreased%20abundance%20of%20gut%20Barnesiella%20intestinihominis%20bacterium%20in%20the%20Framingham%20Heart%20Study.%20Sci%20Rep.%202023;13:13622." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Liu L, Zhou Q, Xu T, Deng Q, Sun Y, Fu J, Chen M, Chen X, Ma Z, Dong Q, et al. Non-differential gut microbes contribute to hypertension and its severity through co-abundances: a multi-regional prospective cohort study. Imeta. 2025;4.</cite> [<a href="https://doi.org/10.1002/imt2.268" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11865328/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40027484/" class="usa-link">PubMed</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Zolfo M, Silverj A, Blanco-MÃ­guez A, Manghi P, Rota-Stabelli O, Heidrich V, Jensen J, Maharjan S, Franzosa E, Menni C, et al: Discovering and exploring the hidden diversity of human gut viruses using highly enriched virome samples. bioRxiv 2024.</cite>
</li>
<li id="CR72">
<span class="label">72.</span><cite>Hsu BB, Gibson TE, Yeliseyev V, Liu Q, Lyon L, Bry L, Silver PA, Gerber GK. Dynamic modulation of the gut microbiota and metabolome by bacteriophages in a mouse model. Cell Host Microbe. 2019;25:803-814.e805.
</cite> [<a href="https://doi.org/10.1016/j.chom.2019.05.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6579560/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31175044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hsu%20BB,%20Gibson%20TE,%20Yeliseyev%20V,%20Liu%20Q,%20Lyon%20L,%20Bry%20L,%20Silver%20PA,%20Gerber%20GK.%20Dynamic%20modulation%20of%20the%20gut%20microbiota%20and%20metabolome%20by%20bacteriophages%20in%20a%20mouse%20model.%20Cell%20Host%20Microbe.%202019;25:803-814.e805." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Li R, Wang Y, Hu H, Tan Y, Ma Y. Metagenomic analysis reveals unexplored diversity of archaeal virome in the human gut. Nat Commun. 2022;13:7978.
</cite> [<a href="https://doi.org/10.1038/s41467-022-35735-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9800368/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36581612/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20R,%20Wang%20Y,%20Hu%20H,%20Tan%20Y,%20Ma%20Y.%20Metagenomic%20analysis%20reveals%20unexplored%20diversity%20of%20archaeal%20virome%20in%20the%20human%20gut.%20Nat%20Commun.%202022;13:7978." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Chibani CM, Shah SA, Schmitz RA, Nayfach S. Inaccurate viral prediction leads to overestimated diversity of the archaeal virome in the human gut. Nat Commun. 2024;15:5976.
</cite> [<a href="https://doi.org/10.1038/s41467-024-49902-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11255274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39019907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chibani%20CM,%20Shah%20SA,%20Schmitz%20RA,%20Nayfach%20S.%20Inaccurate%20viral%20prediction%20leads%20to%20overestimated%20diversity%20of%20the%20archaeal%20virome%20in%20the%20human%20gut.%20Nat%20Commun.%202024;15:5976." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Ivanova NN, Schwientek P, Tripp HJ, Rinke C, Pati A, Huntemann M, Visel A, Woyke T, Kyrpides NC, Rubin EM. Stop codon reassignments in the wild. 2014;344:909â€“13.</cite> [<a href="https://doi.org/10.1126/science.1250691" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24855270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ivanova%20NN,%20Schwientek%20P,%20Tripp%20HJ,%20Rinke%20C,%20Pati%20A,%20Huntemann%20M,%20Visel%20A,%20Woyke%20T,%20Kyrpides%20NC,%20Rubin%20EM.%20Stop%20codon%20reassignments%20in%20the%20wild.%202014;344:909%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Devoto AE, Santini JM, Olm MR, Anantharaman K, Munk P, Tung J, Archie EA, Turnbaugh PJ, Seed KD, Blekhman R, et al. Megaphages infect Prevotella and variants are widespread in gut microbiomes. Nat Microbiol. 2019;4:693â€“700.
</cite> [<a href="https://doi.org/10.1038/s41564-018-0338-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6784885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30692672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Devoto%20AE,%20Santini%20JM,%20Olm%20MR,%20Anantharaman%20K,%20Munk%20P,%20Tung%20J,%20Archie%20EA,%20Turnbaugh%20PJ,%20Seed%20KD,%20Blekhman%20R,%20et%20al.%20Megaphages%20infect%20Prevotella%20and%20variants%20are%20widespread%20in%20gut%20microbiomes.%20Nat%20Microbiol.%202019;4:693%E2%80%93700." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Cook R, Telatin A, Bouras G, Camargo AP, Larralde M, Edwards RA, Adriaenssens EM: Driving through stop signs: predicting stop codon reassignment improves functional annotation of bacteriophages. ISME Communications. 2024;4</cite> [<a href="https://doi.org/10.1093/ismeco/ycae079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11210395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38939532/" class="usa-link">PubMed</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Yutin N, Benler S, Shmakov SA, Wolf YI, Tolstoy I, Rayko M, Antipov D, Pevzner PA, Koonin EV. Analysis of metagenome-assembled viral genomes from the human gut reveals diverse putative CrAss-like phages with unique genomic features. Nat Commun. 2021;12:1044.
</cite> [<a href="https://doi.org/10.1038/s41467-021-21350-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7886860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33594055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yutin%20N,%20Benler%20S,%20Shmakov%20SA,%20Wolf%20YI,%20Tolstoy%20I,%20Rayko%20M,%20Antipov%20D,%20Pevzner%20PA,%20Koonin%20EV.%20Analysis%20of%20metagenome-assembled%20viral%20genomes%20from%20the%20human%20gut%20reveals%20diverse%20putative%20CrAss-like%20phages%20with%20unique%20genomic%20features.%20Nat%20Commun.%202021;12:1044." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Yu G, Smith DK, Zhu H, Guan Y, Lam TT.Â ggtree : an r package for visualization and annotation of phylogenetic trees with their covariates and other associated data Summary. Methods EcolÂ Evol. 2017;8(1):28-36. 10.1111/mee3.2017.8.issue-1. 10.1111/2041-210X.12628.</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369081/bin/40168_2025_2185_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">40168_2025_2185_MOESM1_ESM.xlsx</a><sup> (13.1MB, xlsx) </sup><p>Additional file 1.Â Table S1-S21. Table S1. The quality of 120,568 medium-to-high quality viral genomes evaluated by CheckV. Table S2. Summary of 64,377 viral genomes from 42,803 vOTUs. Table S3. Taxonomy annotation of 42,803 vOTUs using ICTV. Table S4. Summary analysis of functional annotation of CDSs of 18,311 unclassified vOTUs. Table S5. Frequency distribution of stop codon types among predicted viral proteins lacking homology to known sequences. Table S6. Comparison of genetic distances among different populations of 233 vOTUs. Table S7. SNV sites that are significantly different between two populations. Table S8. Host prediction of 44 vOTUs for iPHoP method. Table S9. Host prediction of 44 vOTUs for CRISPR method. Table S10. Matching relationships between viruses and bacteria identified by spacer sequences. Table S11. The intra-group genomic dissimilarity of bacterial hosts corresponding to viruses from different clade. Table S12. SNVs that potentially driving the observed genetic stratification in bacterial hosts. Table S13. The UniRef50 annotation results for driver single nucleotide variants (SNVs). Table S14. Significant associations between viral species genomic and host phenotypic differences. Table S15. Correlations between viral subspecies clades and host phenotypic traits. Table S16. Correlations between vOTUs genomic dissimilarity and vOTU abundance differences. Table S17. Significant associations between viral abundance and phenotypes. Table S18. Taxonomic information of 38,436 vOTUs involved in the CGVR. Table S19. Significant associations between virus and bacteria. Table S20. Summary of 3,497 viral associations to bacterial species confirmed by CRISPR-spacer imprints. Table S21. The significant results of bacteriome mediate virus impacts on phenotypic traits.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369081/bin/40168_2025_2185_MOESM2_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">40168_2025_2185_MOESM2_ESM.zip</a><sup> (17.7MB, zip) </sup><p>Addtional file 2.Â  Fig. S1-S7. Fig. S1. Phylogenetic tree of viral genomes from 44 vOTUs based on SNP haplotype profile. Red represents viral genomes from Chinese population and blue represents the Non-Chinese population.Â Fig. S2. Functional annotation of viral genes with differential SNVs between populations.Â We conducted functional annotation of the 239 SNVs that exhibit significant population differences and classified them into different functional groups based on a previous study.Â Fig. S3. Associations between viral species genomic and host phenotypic differences. a The genetic distance of L1EFC230390_FSHXXA28_S33_L003_vae_13 decrease with increasing variation in defecating frequency. bÂ The genetic distance of FSHCM57_vae_33 decrease with increasing BMI disparity among hosts.Â Fig. S4.Â Distinct viral subspecies clades based on their phylogenetic trees. Two Distinct viral subspecies clades detected in 31 vOTUs based on their SNP haplotypes. Red and blue represent different subspecies clades. Fig. S5.Â Summary of associations between vOTUs genomic dissimilarity and abundance difference. Red indicates proportion of significant associations between vOTUs genomic dissimilarity and abundance difference at FDR &gt; 0.05, while blue for FDR. Fig. S6.Â Lifestyle of 18,235 bacteriophages. The circular visualization represents three predicted lifestyle categories of bacteriophages. Red for bacteriophages with temperate lifestyle, blue for virulent lifestyle and green for the virus with unknow prediction of lifestyle. Fig. S7.Â Significant associations between vOTUs and bacterial species with CRISPR-spacer based confirmation. In total, 3497 significant associations between vOTUs and bacterial species were characterized, with the most viral associations were observed with Escherichia coli.</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The genomic assemblies of gut virus are deposited in National Genomics Data Center (NGDC) under accession id OMIX007646. The metagenomic sequencing data used for the analysis presented in this study are available from the Genome Sequence Archive (GSA) under accession id PRJCA017330.</p>
<p>Scripts used for the analyses in this manuscript are available via <a href="https://github.com/MicrobiomeCardioMetaLab/CGVR_project" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/MicrobiomeCardioMetaLab/CGVR_project</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Microbiome are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s40168-025-02185-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/40168_2025_Article_2185.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.3Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12369081/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12369081/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369081%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369081/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12369081/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12369081/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40836310/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12369081/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40836310/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12369081/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12369081/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="2OMI7Fowg2sSwvWIo8RGYJMP72HpCiHgslpZNrJiWKySykTtbk8KDFNFZEtalmDj">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
