
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Mitogenomic characterization and phylogenetic placement of the Red pandora Pagellus bellottii from the Eastern Atlantic Ocean within the family Sparidae - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152548238AF35A630548230020E4096D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365136/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Mitogenomic characterization and phylogenetic placement of the Red pandora Pagellus bellottii from the Eastern Atlantic Ocean within the family Sparidae">
<meta name="citation_author" content="Arief Wujdi">
<meta name="citation_author_institution" content="Interdisciplinary Program of Marine and Fisheries Sciences and Convergent Technology, Pukyong National University, Busan, 48513 Republic of Korea">
<meta name="citation_author_institution" content="Research Center for Fishery, National Research and Innovation Agency (BRIN), Bogor, 16911 Republic of Indonesia">
<meta name="citation_author" content="Emmanuel Ofosu Mireku Ewusi">
<meta name="citation_author_institution" content="Fisheries Commission, Fisheries Scientific Survey Division, Ministry of Fisheries and Aquaculture Development, Tema, P.O. Box BT 62, Ghana">
<meta name="citation_author" content="Sinar Pagi Sektiana">
<meta name="citation_author_institution" content="Jakarta Technical University of Fisheries, Ministry of Marine Affairs and Fisheries, Jakarta, 12520 Republic of Indonesia">
<meta name="citation_author" content="Muhammad Hilman Fu’adil Amin">
<meta name="citation_author_institution" content="Advance Tropical Biodiversity, Genomics, and Conservation Research Group, Department of Biology, Faculty of Science and Technology, Airlangga University, Surabaya, 60115 Republic of Indonesia">
<meta name="citation_author" content="Estuardo López-Vera">
<meta name="citation_author_institution" content="Laboratorio de Toxinología Marina, Unidad Académica de Ecología y Biodiversidad Acuática, Instituto de Ciencias del Mar y Limnología, Universidad Nacional Autónoma de México, Ciudad de México, 04510 Mexico">
<meta name="citation_author" content="Hyun-Woo Kim">
<meta name="citation_author_institution" content="Advance Tropical Biodiversity, Genomics, and Conservation Research Group, Department of Biology, Faculty of Science and Technology, Airlangga University, Surabaya, 60115 Republic of Indonesia">
<meta name="citation_author_institution" content="Department of Marine Biology, Pukyong National University, Busan, 48513 Republic of Korea">
<meta name="citation_author_institution" content="Research Center for Marine Integrated Bionics Technology, Pukyong National University, Busan, 48513 Republic of Korea">
<meta name="citation_author" content="Hye-Eun Kang">
<meta name="citation_author_institution" content="Institute of Marine Life Science, Pukyong National University, Busan, 48513 Republic of Korea">
<meta name="citation_author" content="Shantanu Kundu">
<meta name="citation_author_institution" content="Ocean and Fisheries Development International Cooperation Institute, College of Fisheries Science, Pukyong National University, Busan, 48513 Republic of Korea">
<meta name="citation_author_institution" content="International Graduate Program of Fisheries Science, Pukyong National University, Busan, 48513 Republic of Korea">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30432">
<meta name="citation_doi" content="10.1038/s41598-025-16187-y">
<meta name="citation_pmid" content="40830652">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365136/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365136/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365136/pdf/41598_2025_Article_16187.pdf">
<meta name="description" content="The systematics and conservation management of the Red pandora (Pagellus bellottii) remain challenging due to the overlapping morphology and distributions with closely related congeners in the Eastern Atlantic Ocean and Mediterranean Sea. These ...">
<meta name="og:title" content="Mitogenomic characterization and phylogenetic placement of the Red pandora Pagellus bellottii from the Eastern Atlantic Ocean within the family Sparidae">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The systematics and conservation management of the Red pandora (Pagellus bellottii) remain challenging due to the overlapping morphology and distributions with closely related congeners in the Eastern Atlantic Ocean and Mediterranean Sea. These ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365136/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365136">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-16187-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_16187.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365136%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365136/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365136/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365136/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 19;15:30432. doi: <a href="https://doi.org/10.1038/s41598-025-16187-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-16187-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Mitogenomic characterization and phylogenetic placement of the Red pandora <em>Pagellus bellottii</em> from the Eastern Atlantic Ocean within the family Sparidae</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wujdi%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Arief Wujdi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Arief Wujdi</span></h3>
<div class="p">
<sup>1</sup>Interdisciplinary Program of Marine and Fisheries Sciences and Convergent Technology, Pukyong National University, Busan, 48513 Republic of Korea </div>
<div class="p">
<sup>2</sup>Research Center for Fishery, National Research and Innovation Agency (BRIN), Bogor, 16911 Republic of Indonesia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wujdi%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Arief Wujdi</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ewusi%20EOM%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Emmanuel Ofosu Mireku Ewusi</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Emmanuel Ofosu Mireku Ewusi</span></h3>
<div class="p">
<sup>3</sup>Fisheries Commission, Fisheries Scientific Survey Division, Ministry of Fisheries and Aquaculture Development, Tema, P.O. Box BT 62, Ghana </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ewusi%20EOM%22%5BAuthor%5D" class="usa-link"><span class="name western">Emmanuel Ofosu Mireku Ewusi</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sektiana%20SP%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Sinar Pagi Sektiana</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Sinar Pagi Sektiana</span></h3>
<div class="p">
<sup>4</sup>Jakarta Technical University of Fisheries, Ministry of Marine Affairs and Fisheries, Jakarta, 12520 Republic of Indonesia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sektiana%20SP%22%5BAuthor%5D" class="usa-link"><span class="name western">Sinar Pagi Sektiana</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Amin%20MHF%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Muhammad Hilman Fu’adil Amin</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Muhammad Hilman Fu’adil Amin</span></h3>
<div class="p">
<sup>5</sup>Advance Tropical Biodiversity, Genomics, and Conservation Research Group, Department of Biology, Faculty of Science and Technology, Airlangga University, Surabaya, 60115 Republic of Indonesia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Amin%20MHF%22%5BAuthor%5D" class="usa-link"><span class="name western">Muhammad Hilman Fu’adil Amin</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22L%C3%B3pez-Vera%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Estuardo López-Vera</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Estuardo López-Vera</span></h3>
<div class="p">
<sup>6</sup>Laboratorio de Toxinología Marina, Unidad Académica de Ecología y Biodiversidad Acuática, Instituto de Ciencias del Mar y Limnología, Universidad Nacional Autónoma de México, Ciudad de México, 04510 Mexico </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22L%C3%B3pez-Vera%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Estuardo López-Vera</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20HW%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Hyun-Woo Kim</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Hyun-Woo Kim</span></h3>
<div class="p">
<sup>5</sup>Advance Tropical Biodiversity, Genomics, and Conservation Research Group, Department of Biology, Faculty of Science and Technology, Airlangga University, Surabaya, 60115 Republic of Indonesia </div>
<div class="p">
<sup>7</sup>Department of Marine Biology, Pukyong National University, Busan, 48513 Republic of Korea </div>
<div class="p">
<sup>8</sup>Research Center for Marine Integrated Bionics Technology, Pukyong National University, Busan, 48513 Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20HW%22%5BAuthor%5D" class="usa-link"><span class="name western">Hyun-Woo Kim</span></a>
</div>
</div>
<sup>5,</sup><sup>7,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kang%20HE%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Hye-Eun Kang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Hye-Eun Kang</span></h3>
<div class="p">
<sup>9</sup>Institute of Marine Life Science, Pukyong National University, Busan, 48513 Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kang%20HE%22%5BAuthor%5D" class="usa-link"><span class="name western">Hye-Eun Kang</span></a>
</div>
</div>
<sup>9,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kundu%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Shantanu Kundu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Shantanu Kundu</span></h3>
<div class="p">
<sup>10</sup>Ocean and Fisheries Development International Cooperation Institute, College of Fisheries Science, Pukyong National University, Busan, 48513 Republic of Korea </div>
<div class="p">
<sup>11</sup>International Graduate Program of Fisheries Science, Pukyong National University, Busan, 48513 Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kundu%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shantanu Kundu</span></a>
</div>
</div>
<sup>10,</sup><sup>11,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Interdisciplinary Program of Marine and Fisheries Sciences and Convergent Technology, Pukyong National University, Busan, 48513 Republic of Korea </div>
<div id="Aff2">
<sup>2</sup>Research Center for Fishery, National Research and Innovation Agency (BRIN), Bogor, 16911 Republic of Indonesia </div>
<div id="Aff3">
<sup>3</sup>Fisheries Commission, Fisheries Scientific Survey Division, Ministry of Fisheries and Aquaculture Development, Tema, P.O. Box BT 62, Ghana </div>
<div id="Aff4">
<sup>4</sup>Jakarta Technical University of Fisheries, Ministry of Marine Affairs and Fisheries, Jakarta, 12520 Republic of Indonesia </div>
<div id="Aff5">
<sup>5</sup>Advance Tropical Biodiversity, Genomics, and Conservation Research Group, Department of Biology, Faculty of Science and Technology, Airlangga University, Surabaya, 60115 Republic of Indonesia </div>
<div id="Aff6">
<sup>6</sup>Laboratorio de Toxinología Marina, Unidad Académica de Ecología y Biodiversidad Acuática, Instituto de Ciencias del Mar y Limnología, Universidad Nacional Autónoma de México, Ciudad de México, 04510 Mexico </div>
<div id="Aff7">
<sup>7</sup>Department of Marine Biology, Pukyong National University, Busan, 48513 Republic of Korea </div>
<div id="Aff8">
<sup>8</sup>Research Center for Marine Integrated Bionics Technology, Pukyong National University, Busan, 48513 Republic of Korea </div>
<div id="Aff9">
<sup>9</sup>Institute of Marine Life Science, Pukyong National University, Busan, 48513 Republic of Korea </div>
<div id="Aff10">
<sup>10</sup>Ocean and Fisheries Development International Cooperation Institute, College of Fisheries Science, Pukyong National University, Busan, 48513 Republic of Korea </div>
<div id="Aff11">
<sup>11</sup>International Graduate Program of Fisheries Science, Pukyong National University, Busan, 48513 Republic of Korea </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 21; Accepted 2025 Aug 13; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365136  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830652/" class="usa-link">40830652</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par14">The systematics and conservation management of the Red pandora (<em>Pagellus bellottii</em>) remain challenging due to the overlapping morphology and distributions with closely related congeners in the Eastern Atlantic Ocean and Mediterranean Sea. These challenges are further exacerbated by population declines driven by intensive coastal fisheries and a paucity of molecular data. In this study, high-throughput sequencing was utilized to assemble the first complete mitogenome of <em>P. bellottii</em> from its native range. The mitogenome (16,666 bp) comprises the typical set of 13 protein-coding genes (PCGs), 22 transfer RNAs (tRNAs), two ribosomal RNAs (rRNAs), and a non-coding control region (CR). The analysis of PCGs revealed a predominance of hydrophobic and neutral amino acids and strong purifying selection. Unlike <em>P. bogaraveo</em>, no evidence of heteroplasmy was observed in the WANCY tRNA gene cluster of <em>P. bellottii</em>. Notably, nucleotide polymorphisms were identified within conserved motifs of the CR. The Bayesian inference and Maximum Likelihood phylogenetic analyses revealed non-monophyly among several Sparidae genera (<em>Pagellus</em>, <em>Dentex</em>, and <em>Pagrus</em>), with <em>P. bellottii</em> clustering most closely with <em>P. erythrinus</em>. Molecular dating estimated the divergence between <em>P. bellottii</em> and <em>P. erythrinus</em> at approximately 6.1 million years ago (MYA), during the Miocene, a period more recent than the divergence of <em>P. bogaraveo</em> and <em>P. acarne</em> (~ 10.5 MYA). These divergence times coincide with major geological events, such as the formation of the Gibraltar Arc and the opening of the Atlantic–Mediterranean gateway, which likely facilitated speciation and lineage diversification of <em>Pagellus</em>. Additionally, the asymmetric biogeographic distribution of <em>P. bellottii</em> and related species may be influenced by hydrographic barriers, climatic gradients across the North and South Atlantic, and historical events such as the Messinian salinity crisis. Overall, this study provides the first comprehensive mitogenomic insights into <em>P. bellottii</em> and underscores the importance of further genome-wide research to elucidate adaptive evolution and phylogeographic history for developing effective conservation and management strategies for all Sparidae in marine ecosystems.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-16187-y.</p></section><section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Phylogenetics, Taxonomy, Zoology, Ichthyology, Genetic variation, Mitochondrial genome, Conservation genomics</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par15">The Sparidae family consists of a diverse group of fish species found in both marine and brackish-water environments<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. The family currently comprises approximately 164 recognized species under 39 genera, with eight new species described within the past decade, highlighting ongoing systematic efforts and the expanding understanding of its biodiversity<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. They are commonly known as seabreams or porgies and are widely dispersed across the tropical and subtropical Atlantic, Pacific, and Indian Oceans<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Based on their dental and jaw morphology, sparids often exhibit herbivorous, piscivorous, and carnivorous feeding behaviors<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Moreover, dentition patterns have become widely recognized as critical diagnostic features in the taxonomic classification of the group<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Among this group, the Red pandora, <em>Pagellus bellottii</em>, inhabits demersal waters in shallow depth ranging from 15 to 70 m<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. This species is, regarded as a vital resource for small to large-scale commercial fisheries for several West African countries, such as Senegal, Mauritius, Ghana, and Guinea. Based on the data reported to the Food and Agriculture Organization (FAO), <em>P. bellottii</em> contributed to 69% among the ~ 2 million tons of total catch of <em>Pagellus</em> genus in FAO Fishing Area 34 (Eastern Atlantic) in last 65 years<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Although <em>P. bellottii</em> is currently classified as ‘Least Concern’ by the IUCN, growing global concern has emerged due to its population decline, indicating overexploitation in several West African countries<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Notably, a progressive increase in fishing pressure in FAO Fishing Area 34 has resulted in a downward trend in annual catch since 2005, highlighting the urgent need for sustainable fisheries management measures<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Therefore, to ensure the conservation of <em>P. bellottii</em>, one of the most economically important species in this region, the integration of both morphological and molecular approaches is urgently demanded.</p>
<p id="Par16">With respect to traditional taxonomy, emerging evidence suggests a range expansion of <em>P. bellottii</em> into the Bay of Biscay within FAO Fishing Area 27 (Northeast Atlantic) and the southeastern Mediterranean Sea<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. This expanded natural distribution of <em>P. bellottii</em> has led to range overlap with its congeners (<em>P. acarne</em>, <em>P. bogaraveo</em>, and <em>P. erythrinus</em>), increasing the likelihood of their co-occurrence and simultaneous harvest within the same fishing grounds<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Hence, the overlapping distributions and shared morphological traits among <em>Pagellus</em> species often pose challenges for accurate species identification<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. The red pandora, <em>P. bellottii</em>, is typically distinguished from other <em>Pagellus</em> species by its silvery-red coloration, the presence of blue spots along the flanks, and a characteristic dark red mark near the lateral line<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Moreover, their systematic position has been frequently revised across different taxonomic groups, often hindering accurate identification and complicating effective management and conservation efforts<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Therefore, the incorporation of molecular approaches is essential for accurate species identification, serving as a prerequisite for systematics research and the development of effective, species-specific fisheries management strategies. So far, several molecular studies have successfully facilitated species identification and characterization within Sparidae, contributing to phylogenetic interpretations, detection of cryptic diversity, and estimation of lineage diversification using partial mitochondrial and nuclear markers, as well as microsatellite DNA<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. In addition, recent mitogenomic studies have rapidly expanded worldwide due to their effectiveness in elucidating genomic features, gene-level variation, and enabling comprehensive phylogenetic analyses across diverse groups of marine fishes, including members of the Sparidae family<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>–<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. To date, three complete mitogenomes of the <em>Pagellus</em> species have been sequenced, all derived from specimens collected in the Mediterranean Sea<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. As a result, the maternal inheritance-driven evolutionary pathways of these Pandora fishes remain poorly understood, largely due to the absence of mitogenomic data for congeners distributed in the Atlantic and western Indian Ocean. Therefore, the present study aims to (i) generate the first complete mitochondrial genome of the <em>P. bellottii</em> based on specimens from its native range in the Eastern Atlantic Ocean; (ii) assess the structural features and genetic variation among closely related congeners; (iii) evaluate its phylogenetic placement within major Sparidae lineages; and (iv) estimate divergence times and correlate them with historical geological events, present-day ocean currents, and climatic conditions to infer potential drivers of diversification. The findings of this study may serve as a foundational resource for future systematic research within the Sparidae family, while the generated genetic data could support further population genetic studies of <em>Pagellus</em> species. Additionally, the study provides valuable insights into the divergence time estimation of <em>P. bellottii</em> and other congeners, particularly within the biogeographic context of marine ecosystems, thereby contributing to a more comprehensive understanding of their evolutionary history and distribution patterns.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Mitogenome organization and gene arrangement</h3>
<p id="Par17">The mitogenome of <em>P. bellottii</em> is totalling 16,666 bp and is available in GenBank under accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ524309" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ524309</a>. It consists of 13 protein-coding genes (PCGs), 22 transfer RNA (tRNA) genes, two ribosomal RNA (rRNA) genes, and a noncoding control region (CR). Out of the 37 genes, 28 are placed on the heavy-strand including 12 PCGs, two rRNAs, and 14 tRNAs, while the remaining nine genes (<em>ND6</em> and eight tRNAs) are accommodated on the light-strand (Fig. <a href="#Fig1" class="usa-link">1</a>; Table <a href="#Tab1" class="usa-link">1</a>). Comparative analysis revealed the total length of mitogenome within <em>Pagellus</em> ranged from 16,486 bp in <em>P. acarne</em> to 16,941 bp in <em>P. bogaraveo</em>. The gene arrangement in light- and heavy-strands closely mirrors that of other <em>Pagellus</em> congeners. The gene organization is consistent across most <em>Pagellus</em> species, except for <em>P. bogaraveo</em>, which has an extra <em>tRNA-Cys</em> pseudogene within the tRNA gene group located on the light-strand. The mitogenome of <em>P. bellottii</em> shows a noticeable nucleotide composition of AT bias (54.93%) with nucleotide composition values of 27.45% adenine (A), 27.48% thymine (T), 28.28% cytosine (C), and 16.79% guanine (G). Other <em>Pagellus</em> species also exhibits the AT-bias, with <em>P. bogaraveo</em> having the lowest AT bias (53.62%) followed by <em>P. acarne</em> (53.69%), <em>P. bellottii</em> (54.93%) and <em>P. erythrinus</em> (55%). The AT and GC skewness values for <em>P. bellottii</em> were manually calculated as −0.001 and − 0.255, respectively. In comparison, the skew values for other <em>Pagellus</em> species ranged from − 0.002 to − 0.261 in <em>P. acarne</em>, 0.005 and − 0.256 in <em>P. bogaraveo</em>, to 0.0004 and − 0.252 in <em>P. erythrinus</em> (Table <a href="#Tab2" class="usa-link">2</a>). The complete mitogenome of <em>P. bellottii</em> includes seven overlapping regions spanning 37 bp, with the longest overlap of 13 bp shared between <em>tRNA-Ser</em> (S2) and <em>COI</em>. Additionally, <em>tRNA-Asn</em> gene shows the largest intergenic space, spanning 38 bp. Comparative analysis showed that three overlapping regions within PCGs are common to all examined <em>Pagellus</em> species, such as both <em>ND4</em> and <em>ND4L</em> sharing 7 bp, <em>ATP8</em> and <em>ATP6</em> sharing 10 bp, and between <em>ND5</em> and <em>ND6</em> sharing 4 bp nucleotide (Supplementary Table<a href="#MOESM1" class="usa-link">S1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365136_41598_2025_16187_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/02ad/12365136/fdc32efc4c77/41598_2025_16187_Fig1_HTML.jpg" loading="lazy" id="d33e694" height="361" width="772" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Geographic distribution map illustrating the IUCN-defined ranges of four <em>Pagellus</em> species across the Euro-African marine ecosystem, overlaid with major ocean currents and bathymetric features. The collection locality of <em>P. bellottii</em> is indicated with a Ghana circular flag symbol, and the species photograph was taken by the second author (E.O.M.E.). The map was generated using ArcGIS v10.6. global-scale ocean current data were sourced from the NOAA National Weather Service and the U.S. Army (<a href="https://data.amerigeoss.org/dataset/major-ocean-currents-arrowpolys-100m-76" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://data.amerigeoss.org/dataset/major-ocean-currents-arrowpolys-100 m-76</a>). Additionally, bathymetric layers derived from the CMIP6 Earth System Models were obtained from the Bio-ORACLE v3.0 database (<a href="https://bio-oracle.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bio-oracle.org/</a>). (<strong>B</strong>) Circular representation of the complete mitochondrial genome of <em>P. bellottii</em>, showing gene arrangement through various colored arcs. This circular map was generated using the MitoFish MitoAnnotator web server and manually edited in Adobe Photoshop CS 8.0.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Mitochondrial genome organization of <em>P. bellottii</em>, covering strand arrangement, size, and intergenic nucleotide (IN). ‘H’ and ‘L’ indicates the position of each gene in the heavy and light strands. ‘-’ marks indicate an incomplete stop codon.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Genes</th>
<th align="left" colspan="1" rowspan="1">Start</th>
<th align="left" colspan="1" rowspan="1">Stop</th>
<th align="left" colspan="1" rowspan="1">Size (bp)</th>
<th align="left" colspan="1" rowspan="1">Strand</th>
<th align="left" colspan="1" rowspan="1">IN</th>
<th align="left" colspan="1" rowspan="1">Start codon</th>
<th align="left" colspan="1" rowspan="1">Stop codon</th>
<th align="left" colspan="1" rowspan="1">Anti-codon</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Phe (F)</em>
</td>
<td align="center" colspan="1" rowspan="1">1</td>
<td align="center" colspan="1" rowspan="1">68</td>
<td align="center" colspan="1" rowspan="1">68</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">GAA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>12 S rRNA</em>
</td>
<td align="center" colspan="1" rowspan="1">69</td>
<td align="center" colspan="1" rowspan="1">1022</td>
<td align="center" colspan="1" rowspan="1">954</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Val (V)</em>
</td>
<td align="center" colspan="1" rowspan="1">1023</td>
<td align="center" colspan="1" rowspan="1">1094</td>
<td align="center" colspan="1" rowspan="1">72</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">TAC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>16 S rRNA</em>
</td>
<td align="center" colspan="1" rowspan="1">1095</td>
<td align="center" colspan="1" rowspan="1">2790</td>
<td align="center" colspan="1" rowspan="1">1696</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Leu (L2)</em>
</td>
<td align="center" colspan="1" rowspan="1">2791</td>
<td align="center" colspan="1" rowspan="1">2863</td>
<td align="center" colspan="1" rowspan="1">73</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">TAA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>ND1</em>
</td>
<td align="center" colspan="1" rowspan="1">2864</td>
<td align="center" colspan="1" rowspan="1">3838</td>
<td align="center" colspan="1" rowspan="1">975</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Ile (I)</em>
</td>
<td align="center" colspan="1" rowspan="1">3844</td>
<td align="center" colspan="1" rowspan="1">3913</td>
<td align="center" colspan="1" rowspan="1">70</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">−1</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">GAT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Gln (Q)</em>
</td>
<td align="center" colspan="1" rowspan="1">3913</td>
<td align="center" colspan="1" rowspan="1">3983</td>
<td align="center" colspan="1" rowspan="1">71</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">−1</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">TTG</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Met (M)</em>
</td>
<td align="center" colspan="1" rowspan="1">3983</td>
<td align="center" colspan="1" rowspan="1">4052</td>
<td align="center" colspan="1" rowspan="1">70</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">CAT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>ND2</em>
</td>
<td align="center" colspan="1" rowspan="1">4053</td>
<td align="center" colspan="1" rowspan="1">5098</td>
<td align="center" colspan="1" rowspan="1">1046</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TA-</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Trp (W)</em>
</td>
<td align="center" colspan="1" rowspan="1">5099</td>
<td align="center" colspan="1" rowspan="1">5168</td>
<td align="center" colspan="1" rowspan="1">70</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">TCA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Ala (A)</em>
</td>
<td align="center" colspan="1" rowspan="1">5169</td>
<td align="center" colspan="1" rowspan="1">5237</td>
<td align="center" colspan="1" rowspan="1">69</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">TGC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Asn (N)</em>
</td>
<td align="center" colspan="1" rowspan="1">5239</td>
<td align="center" colspan="1" rowspan="1">5311</td>
<td align="center" colspan="1" rowspan="1">73</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">38</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">GTT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Cys (C)</em>
</td>
<td align="center" colspan="1" rowspan="1">5350</td>
<td align="center" colspan="1" rowspan="1">5415</td>
<td align="center" colspan="1" rowspan="1">66</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">GCA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Tyr (Y)</em>
</td>
<td align="center" colspan="1" rowspan="1">5416</td>
<td align="center" colspan="1" rowspan="1">5485</td>
<td align="center" colspan="1" rowspan="1">70</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">GTA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>COI</em>
</td>
<td align="center" colspan="1" rowspan="1">5487</td>
<td align="center" colspan="1" rowspan="1">7052</td>
<td align="center" colspan="1" rowspan="1">1566</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">−13</td>
<td align="left" colspan="1" rowspan="1">GTG</td>
<td align="left" colspan="1" rowspan="1">AGG</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Ser (S2)</em>
</td>
<td align="center" colspan="1" rowspan="1">7040</td>
<td align="center" colspan="1" rowspan="1">7110</td>
<td align="center" colspan="1" rowspan="1">71</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">TGA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Asp (D)</em>
</td>
<td align="center" colspan="1" rowspan="1">7114</td>
<td align="center" colspan="1" rowspan="1">7186</td>
<td align="center" colspan="1" rowspan="1">73</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">GTC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>COII</em>
</td>
<td align="center" colspan="1" rowspan="1">7195</td>
<td align="center" colspan="1" rowspan="1">7885</td>
<td align="center" colspan="1" rowspan="1">691</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">T--</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Lys (K)</em>
</td>
<td align="center" colspan="1" rowspan="1">7886</td>
<td align="center" colspan="1" rowspan="1">7959</td>
<td align="center" colspan="1" rowspan="1">74</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">TTT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>ATP8</em>
</td>
<td align="center" colspan="1" rowspan="1">7961</td>
<td align="center" colspan="1" rowspan="1">8128</td>
<td align="center" colspan="1" rowspan="1">168</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">−10</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">T--</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>ATP6</em>
</td>
<td align="center" colspan="1" rowspan="1">8119</td>
<td align="center" colspan="1" rowspan="1">8801</td>
<td align="center" colspan="1" rowspan="1">683</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TA-</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>COIII</em>
</td>
<td align="center" colspan="1" rowspan="1">8802</td>
<td align="center" colspan="1" rowspan="1">9586</td>
<td align="center" colspan="1" rowspan="1">785</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TA-</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Gly (G)</em>
</td>
<td align="center" colspan="1" rowspan="1">9587</td>
<td align="center" colspan="1" rowspan="1">9658</td>
<td align="center" colspan="1" rowspan="1">72</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">TCC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>ND3</em>
</td>
<td align="center" colspan="1" rowspan="1">9659</td>
<td align="center" colspan="1" rowspan="1">10,007</td>
<td align="center" colspan="1" rowspan="1">349</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">T--</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Arg (R)</em>
</td>
<td align="center" colspan="1" rowspan="1">10,008</td>
<td align="center" colspan="1" rowspan="1">10,078</td>
<td align="center" colspan="1" rowspan="1">71</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">TCG</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>ND4L</em>
</td>
<td align="center" colspan="1" rowspan="1">10,079</td>
<td align="center" colspan="1" rowspan="1">10,375</td>
<td align="center" colspan="1" rowspan="1">297</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">−7</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>ND4</em>
</td>
<td align="center" colspan="1" rowspan="1">10,369</td>
<td align="center" colspan="1" rowspan="1">11,749</td>
<td align="center" colspan="1" rowspan="1">1381</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">GTG</td>
<td align="left" colspan="1" rowspan="1">T--</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-His (H)</em>
</td>
<td align="center" colspan="1" rowspan="1">11,750</td>
<td align="center" colspan="1" rowspan="1">11,818</td>
<td align="center" colspan="1" rowspan="1">69</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">GTG</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Ser (S1)</em>
</td>
<td align="center" colspan="1" rowspan="1">11,819</td>
<td align="center" colspan="1" rowspan="1">11,886</td>
<td align="center" colspan="1" rowspan="1">68</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">GCT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Leu (L1)</em>
</td>
<td align="center" colspan="1" rowspan="1">11,893</td>
<td align="center" colspan="1" rowspan="1">11,965</td>
<td align="center" colspan="1" rowspan="1">73</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">TAG</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>ND5</em>
</td>
<td align="center" colspan="1" rowspan="1">11,966</td>
<td align="center" colspan="1" rowspan="1">13,804</td>
<td align="center" colspan="1" rowspan="1">1839</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">−4</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>ND6</em>
</td>
<td align="center" colspan="1" rowspan="1">13,801</td>
<td align="center" colspan="1" rowspan="1">14,322</td>
<td align="center" colspan="1" rowspan="1">522</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAG</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Glu (E)</em>
</td>
<td align="center" colspan="1" rowspan="1">14,323</td>
<td align="center" colspan="1" rowspan="1">14,391</td>
<td align="center" colspan="1" rowspan="1">69</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">TTC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>CYTB</em>
</td>
<td align="center" colspan="1" rowspan="1">14,396</td>
<td align="center" colspan="1" rowspan="1">15,536</td>
<td align="center" colspan="1" rowspan="1">1141</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">T--</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Thr (T)</em>
</td>
<td align="center" colspan="1" rowspan="1">15,537</td>
<td align="center" colspan="1" rowspan="1">15,610</td>
<td align="center" colspan="1" rowspan="1">74</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">−1</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">TGT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tRNA-Pro (P)</em>
</td>
<td align="center" colspan="1" rowspan="1">15,610</td>
<td align="center" colspan="1" rowspan="1">15,679</td>
<td align="center" colspan="1" rowspan="1">70</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">TGG</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Control region</em>
</td>
<td align="center" colspan="1" rowspan="1">15,680</td>
<td align="center" colspan="1" rowspan="1">16,666</td>
<td align="center" colspan="1" rowspan="1">987</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Nucleotide composition of mitochondrial genomes of <em>P. bellottii</em> and other congeners.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Species name</th>
<th align="left" colspan="1" rowspan="1">Size (bp)</th>
<th align="left" colspan="1" rowspan="1">A%</th>
<th align="left" colspan="1" rowspan="1">T%</th>
<th align="left" colspan="1" rowspan="1">G%</th>
<th align="left" colspan="1" rowspan="1">C%</th>
<th align="left" colspan="1" rowspan="1">A + T%</th>
<th align="left" colspan="1" rowspan="1">AT-Skew</th>
<th align="left" colspan="1" rowspan="1">GC-Skew</th>
</tr></thead>
<tbody>
<tr><td align="left" colspan="9" rowspan="1">Complete mitogenome</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. bellottii</em>
</td>
<td align="left" colspan="1" rowspan="1">16,666</td>
<td align="left" colspan="1" rowspan="1">27.45</td>
<td align="left" colspan="1" rowspan="1">27.48</td>
<td align="left" colspan="1" rowspan="1">16.79</td>
<td align="left" colspan="1" rowspan="1">28.28</td>
<td align="left" colspan="1" rowspan="1">54.93</td>
<td align="left" colspan="1" rowspan="1">−0.001</td>
<td align="left" colspan="1" rowspan="1">−0.255</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. acarne</em>
</td>
<td align="left" colspan="1" rowspan="1">16,484</td>
<td align="left" colspan="1" rowspan="1">26.79</td>
<td align="left" colspan="1" rowspan="1">26.90</td>
<td align="left" colspan="1" rowspan="1">17.10</td>
<td align="left" colspan="1" rowspan="1">29.20</td>
<td align="left" colspan="1" rowspan="1">53.69</td>
<td align="left" colspan="1" rowspan="1">−0.002</td>
<td align="left" colspan="1" rowspan="1">−0.261</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. bogaraveo</em>
</td>
<td align="left" colspan="1" rowspan="1">16,941</td>
<td align="left" colspan="1" rowspan="1">26.94</td>
<td align="left" colspan="1" rowspan="1">26.68</td>
<td align="left" colspan="1" rowspan="1">17.25</td>
<td align="left" colspan="1" rowspan="1">29.13</td>
<td align="left" colspan="1" rowspan="1">53.62</td>
<td align="left" colspan="1" rowspan="1">0.005</td>
<td align="left" colspan="1" rowspan="1">−0.256</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. erythrinus</em>
</td>
<td align="left" colspan="1" rowspan="1">16,828</td>
<td align="left" colspan="1" rowspan="1">27.51</td>
<td align="left" colspan="1" rowspan="1">27.49</td>
<td align="left" colspan="1" rowspan="1">16.83</td>
<td align="left" colspan="1" rowspan="1">28.17</td>
<td align="left" colspan="1" rowspan="1">55.00</td>
<td align="left" colspan="1" rowspan="1">0.0004</td>
<td align="left" colspan="1" rowspan="1">−0.252</td>
</tr>
<tr><td align="left" colspan="9" rowspan="1">PCGs</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. bellottii</em>
</td>
<td align="left" colspan="1" rowspan="1">11,443</td>
<td align="left" colspan="1" rowspan="1">25.68</td>
<td align="left" colspan="1" rowspan="1">28.87</td>
<td align="left" colspan="1" rowspan="1">15.40</td>
<td align="left" colspan="1" rowspan="1">30.04</td>
<td align="left" colspan="1" rowspan="1">54.56</td>
<td align="left" colspan="1" rowspan="1">−0.058</td>
<td align="left" colspan="1" rowspan="1">−0.322</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. acarne</em>
</td>
<td align="left" colspan="1" rowspan="1">11,439</td>
<td align="left" colspan="1" rowspan="1">24.09</td>
<td align="left" colspan="1" rowspan="1">29.25</td>
<td align="left" colspan="1" rowspan="1">16.49</td>
<td align="left" colspan="1" rowspan="1">30.17</td>
<td align="left" colspan="1" rowspan="1">53.34</td>
<td align="left" colspan="1" rowspan="1">−0.097</td>
<td align="left" colspan="1" rowspan="1">−0.293</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. bogaraveo</em>
</td>
<td align="left" colspan="1" rowspan="1">11,442</td>
<td align="left" colspan="1" rowspan="1">24.01</td>
<td align="left" colspan="1" rowspan="1">28.95</td>
<td align="left" colspan="1" rowspan="1">16.78</td>
<td align="left" colspan="1" rowspan="1">30.27</td>
<td align="left" colspan="1" rowspan="1">52.95</td>
<td align="left" colspan="1" rowspan="1">−0.093</td>
<td align="left" colspan="1" rowspan="1">−0.287</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. erythrinus</em>
</td>
<td align="left" colspan="1" rowspan="1">11,444</td>
<td align="left" colspan="1" rowspan="1">24.83</td>
<td align="left" colspan="1" rowspan="1">29.88</td>
<td align="left" colspan="1" rowspan="1">16.06</td>
<td align="left" colspan="1" rowspan="1">29.24</td>
<td align="left" colspan="1" rowspan="1">54.70</td>
<td align="left" colspan="1" rowspan="1">−0.092</td>
<td align="left" colspan="1" rowspan="1">−0.291</td>
</tr>
<tr><td align="left" colspan="9" rowspan="1">tRNAs</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. bellottii</em>
</td>
<td align="left" colspan="1" rowspan="1">1556</td>
<td align="left" colspan="1" rowspan="1">29.63</td>
<td align="left" colspan="1" rowspan="1">24.87</td>
<td align="left" colspan="1" rowspan="1">20.44</td>
<td align="left" colspan="1" rowspan="1">25.06</td>
<td align="left" colspan="1" rowspan="1">54.50</td>
<td align="left" colspan="1" rowspan="1">0.087</td>
<td align="left" colspan="1" rowspan="1">−0.102</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. acarne</em>
</td>
<td align="left" colspan="1" rowspan="1">1556</td>
<td align="left" colspan="1" rowspan="1">27.25</td>
<td align="left" colspan="1" rowspan="1">26.35</td>
<td align="left" colspan="1" rowspan="1">24.04</td>
<td align="left" colspan="1" rowspan="1">22.37</td>
<td align="left" colspan="1" rowspan="1">53.56</td>
<td align="left" colspan="1" rowspan="1">0.017</td>
<td align="left" colspan="1" rowspan="1">0.036</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. bogaraveo</em>
</td>
<td align="left" colspan="1" rowspan="1">1557</td>
<td align="left" colspan="1" rowspan="1">27.75</td>
<td align="left" colspan="1" rowspan="1">26.46</td>
<td align="left" colspan="1" rowspan="1">23.89</td>
<td align="left" colspan="1" rowspan="1">21.90</td>
<td align="left" colspan="1" rowspan="1">54.21</td>
<td align="left" colspan="1" rowspan="1">0.024</td>
<td align="left" colspan="1" rowspan="1">0.043</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. erythrinus</em>
</td>
<td align="left" colspan="1" rowspan="1">1559</td>
<td align="left" colspan="1" rowspan="1">27.65</td>
<td align="left" colspan="1" rowspan="1">27.07</td>
<td align="left" colspan="1" rowspan="1">24.25</td>
<td align="left" colspan="1" rowspan="1">21.04</td>
<td align="left" colspan="1" rowspan="1">54.71</td>
<td align="left" colspan="1" rowspan="1">0.011</td>
<td align="left" colspan="1" rowspan="1">0.071</td>
</tr>
<tr><td align="left" colspan="9" rowspan="1">rRNAs</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. bellottii</em>
</td>
<td align="left" colspan="1" rowspan="1">2650</td>
<td align="left" colspan="1" rowspan="1">32.642</td>
<td align="left" colspan="1" rowspan="1">21.36</td>
<td align="left" colspan="1" rowspan="1">21.25</td>
<td align="left" colspan="1" rowspan="1">24.75</td>
<td align="left" colspan="1" rowspan="1">54.00</td>
<td align="left" colspan="1" rowspan="1">0.209</td>
<td align="left" colspan="1" rowspan="1">−0.076</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. acarne</em>
</td>
<td align="left" colspan="1" rowspan="1">2648</td>
<td align="left" colspan="1" rowspan="1">31.382</td>
<td align="left" colspan="1" rowspan="1">21.15</td>
<td align="left" colspan="1" rowspan="1">21.94</td>
<td align="left" colspan="1" rowspan="1">25.53</td>
<td align="left" colspan="1" rowspan="1">52.53</td>
<td align="left" colspan="1" rowspan="1">0.195</td>
<td align="left" colspan="1" rowspan="1">−0.076</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. bogaraveo</em>
</td>
<td align="left" colspan="1" rowspan="1">2648</td>
<td align="left" colspan="1" rowspan="1">31.458</td>
<td align="left" colspan="1" rowspan="1">21.37</td>
<td align="left" colspan="1" rowspan="1">21.53</td>
<td align="left" colspan="1" rowspan="1">25.64</td>
<td align="left" colspan="1" rowspan="1">52.83</td>
<td align="left" colspan="1" rowspan="1">0.191</td>
<td align="left" colspan="1" rowspan="1">−0.087</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. erythrinus</em>
</td>
<td align="left" colspan="1" rowspan="1">2650</td>
<td align="left" colspan="1" rowspan="1">32.491</td>
<td align="left" colspan="1" rowspan="1">21.13</td>
<td align="left" colspan="1" rowspan="1">21.55</td>
<td align="left" colspan="1" rowspan="1">24.83</td>
<td align="left" colspan="1" rowspan="1">53.62</td>
<td align="left" colspan="1" rowspan="1">0.212</td>
<td align="left" colspan="1" rowspan="1">−0.071</td>
</tr>
<tr><td align="left" colspan="9" rowspan="1">Control region</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. bellottii</em>
</td>
<td align="left" colspan="1" rowspan="1">987</td>
<td align="left" colspan="1" rowspan="1">31.51</td>
<td align="left" colspan="1" rowspan="1">31.91</td>
<td align="left" colspan="1" rowspan="1">15.20</td>
<td align="left" colspan="1" rowspan="1">21.38</td>
<td align="left" colspan="1" rowspan="1">63.42</td>
<td align="left" colspan="1" rowspan="1">−0.006</td>
<td align="left" colspan="1" rowspan="1">−0.169</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. acarne</em>
</td>
<td align="left" colspan="1" rowspan="1">781</td>
<td align="left" colspan="1" rowspan="1">32.14</td>
<td align="left" colspan="1" rowspan="1">31.75</td>
<td align="left" colspan="1" rowspan="1">15.62</td>
<td align="left" colspan="1" rowspan="1">20.49</td>
<td align="left" colspan="1" rowspan="1">63.89</td>
<td align="left" colspan="1" rowspan="1">0.006</td>
<td align="left" colspan="1" rowspan="1">−0.135</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. bogaraveo</em>
</td>
<td align="left" colspan="1" rowspan="1">1195</td>
<td align="left" colspan="1" rowspan="1">32.22</td>
<td align="left" colspan="1" rowspan="1">30.38</td>
<td align="left" colspan="1" rowspan="1">15.98</td>
<td align="left" colspan="1" rowspan="1">21.42</td>
<td align="left" colspan="1" rowspan="1">62.59</td>
<td align="left" colspan="1" rowspan="1">0.029</td>
<td align="left" colspan="1" rowspan="1">−0.145</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em> P. erythrinus</em>
</td>
<td align="left" colspan="1" rowspan="1">1154</td>
<td align="left" colspan="1" rowspan="1">31.80</td>
<td align="left" colspan="1" rowspan="1">30.68</td>
<td align="left" colspan="1" rowspan="1">14.38</td>
<td align="left" colspan="1" rowspan="1">23.14</td>
<td align="left" colspan="1" rowspan="1">62.48</td>
<td align="left" colspan="1" rowspan="1">0.018</td>
<td align="left" colspan="1" rowspan="1">−0.233</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">Protein-coding genes features</h3>
<p id="Par21">The complete mitogenome of <em>P. bellottii</em> consisting of 13 PCGs (four cytochromes, two ATP synthases, and seven NADH dehydrogenases) with a combined length of 11,443 base pairs, representing 68.7% of the complete mitogenome. Among the PCGs, <em>ATP8</em> is being the shortest with 168 bp, whereas <em>ND5</em> appears to be the longest with 1,839 bp (Table <a href="#Tab1" class="usa-link">1</a>). Comparative analysis of PCG lengths across other <em>Pagellus</em> species showed a range of 11,439 bp in <em>P. acarne</em>, 11,442 bp in <em>P. bogaraveo</em>, and 11,444 bp in <em>P. erythrinus</em>. An evaluation of AT bias in the PCGs revealed values of 52.95% in <em>P. bogaraveo</em>, 53.34% in <em>P. acarne</em>, 54.56% in <em>P. bellottii</em>, and 54.70% in <em>P. erythrinus</em>. The AT-skewness value ranged from − 0.058 in <em>P. bellottii</em>, −0.097 in <em>P. acarne</em>, and − 0.093 in <em>P. bogaraveo</em> to −0.092 in <em>P. erythrinus</em>. Similarly, the GC-skewness values were calculated as −0.322 in <em>P. bellottii</em>, −0.293 in <em>P. acarne</em>, −0.287 in <em>P. bogaraveo</em>, and − 0.291 in <em>P. erythrinus</em> (Table <a href="#Tab2" class="usa-link">2</a>). Nucleotide diversity (π) analysis on PCGs produced the average value of 0.16187, encompassing 3,023 polymorphic nucleotides with <em>ND4</em> exhibits the highest value of 0.26 (Fig. <a href="#Fig2" class="usa-link">2</a>A). Furthermore, saturation analysis revealed that neither transitions nor transversions reached saturation, even with increasing Tamura-Nei 93 (TN93) divergence values across all PCGs in the Sparidae mitogenomes (Fig. <a href="#Fig2" class="usa-link">2</a>B). The Open Reading Frame (ORF) finder tool successfully identified the initiation and termination codons for all 13 PCGs. The start codon ATG was identified in 11 PCGs (<em>ND1</em>, <em>ND2</em>, <em>COII</em>, <em>COIII</em>, <em>ATP6</em>, <em>ATP8</em>, <em>ND3</em>, <em>ND4L</em>, <em>ND5</em>, <em>ND6</em>, and <em>Cytb</em>), whereas GTG served as the start codon for <em>COI</em> and <em>ND4</em>. In terms of termination codons, three PCGs (<em>ND1</em>, <em>ND4L</em>, and <em>ND5</em>) terminated with TAA, while <em>COI</em> and <em>ND6</em> used GTG and TAG, respectively. The remaining eight PCGs featured incomplete stop codons, either with TA- or T–. Comparative analysis revealed that the start codon GTG in <em>COI</em> is conserved across other <em>Pagellus</em> species, whereas the use of GTG as the start codon in <em>ND4</em> appears to be shared exclusively with <em>P. erythrinus</em>. Notably, the termination codon usage is different among four <em>Pagellus</em>, with seven PCGs having incomplete stop codon in <em>P. acarne</em>, three PCGs in <em>P. bogaraveo</em>, and six PCGs in <em>P. erythrinus</em> (Supplementary Table S2).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365136_41598_2025_16187_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/02ad/12365136/9aac64dab6cf/41598_2025_16187_Fig2_HTML.jpg" loading="lazy" id="d33e2251" height="608" width="778" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mitogenomic characteristics of the protein-coding genes (PCGs): (<strong>A</strong>) Nucleotide diversity (π) across PCGs, illustrating genetic variability among the four <em>Pagellus</em> species. (<strong>B</strong>) Scatter plot of transition (s) and transversion (v) rates relative to genetic divergence in PCGs, calculated using the Tamura-Nei (TN93) distance across all Sparidae species. (<strong>C</strong>) Box plot showing the mean Ka/Ks ratios for each PCG, with values consistently below 1 across Sparidae, indicating purifying selection.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Substitution pattern and relative synonymous codon usage</h3>
<p id="Par23">The estimation of nonsynonymous (Ka) and synonymous (Ks) ratios in the PCGs indicated that all sparids, including <em>P. bellottii</em>, are subject to similar levels of selection pressure. The Ka/Ks ratio analyses across all sparid species revealed that all PCGs exhibited ratios below 1, indicating purifying selection. The mean Ka/Ks values ranged from 0.011 ± 0.004 for <em>COI</em>, representing the lowest ratio, to 0.074 ± 0.018 for <em>ND2</em>, representing the highest. Subsequently, the Ka/Ks ratios across all sparids follow the order: <em>COI</em> &lt; <em>Cytb</em> &lt; <em>COII</em> &lt; <em>ND1</em> &lt; <em>COIII</em> &lt; <em>ND5</em> &lt; <em>ND3</em> &lt; <em>ND4L</em> &lt; <em>ND4</em> &lt; <em>ATP6</em> &lt; <em>ND6</em> &lt; <em>ATP8</em> &lt; <em>ND2</em> (Fig. <a href="#Fig2" class="usa-link">2</a>C, Supplementary Table S3). Moreover, the Relative Synonymous Codon Usage (RSCU) analysis showed that the combination of nucleotides was transcribed into specific amino acids in the PCGs of four <em>Pagellus</em>. In total, the number of amino acids in <em>P. bellottii</em> PCGs are 3,608 without the stop codons. The amino acid contributors were primarily dominated by Leucine (14.4%) and Alanine (6.3%) with hydrophobic hydropathy characteristic, followed by three neutral hydropathic amino acids, such as Proline (11.4%), Serine (10.8%), and Threonine (6.9%). Meanwhile, two hydrophobic amino acids, cysteine (1.5%), and tryptophan (2.9%) and three hydrophilic amino acids, aspartic acid (2.1%), glutamic acid (2.2%), and lysine (3.0%) were found as less abundant (Fig. <a href="#Fig3" class="usa-link">3</a>A, Supplementary Table S4). Comparative analysis showed that the amino acid composition with hydrophobic properties is more dominant compared to others. Two hydrophobic amino acids, such as Leucine and Alanine, followed by three neutral hydropathic amino acids (Proline, Serine, and Threonine) are commonly observed in all four <em>Pagellus</em> species. Notably, six codon variants of leucine and serine were identified, contributing to an increased abundance of these amino acids across all four <em>Pagellus</em> species. Moreover, several codons exhibited higher relative usage, indicated by values greater than 1.5, reflecting their preferential contribution to the translation of specific amino acids. For example, GCC for alanine, CTT for leucine, TCT for serine, and ACC for threonine showed elevated usage in all four <em>Pagellus</em> species (Fig. <a href="#Fig3" class="usa-link">3</a>B, Supplementary Table S4).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365136_41598_2025_16187_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/02ad/12365136/32e6dd745282/41598_2025_16187_Fig3_HTML.jpg" loading="lazy" id="d33e2351" height="794" width="778" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Structural characteristics of amino acids in PCGs across four <em>Pagellus</em> species: (<strong>A</strong>) Comparative amino acid composition. (<strong>B</strong>) Relative synonymous codon usage (RSCU) analysis, highlighting codon preferences contributing to the translational efficiency of each amino acid.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">Ribosomal RNA and transfer RNA structures</h3>
<p id="Par25">The mitogenome of <em>P. bellottii</em> includes two rRNA genes: the <em>12 S rRNA</em> (small subunit) and the <em>16 S rRNA</em> (large subunit), spanning a total of 2,650 bp, which accounts for 15.9% of the overall mitogenome length. Comparative analysis showed that the rRNA length in <em>P. erythrinus</em> is conserved, whereas <em>P. bogaraveo</em> and <em>P. acarne</em> possess slightly shorter rRNAs (2,648 bp). The rRNA genes showed a bias towards AT, ranging from 52.5% in <em>P. acarne</em> to 54.0% in <em>P. bellottii</em>. Additionally, AT-skewness value ranged from 0.191 to 0.212 and GC-skewness values ranged from − 0.087 to − 0.071 among all four <em>Pagellus</em> species (Table <a href="#Tab2" class="usa-link">2</a>).</p>
<p id="Par26">The 22 tRNA genes within the mitogenome of <em>P. bellottii</em> are scattered between rRNA and PCGs regions. It has a collective size of 1,556 bp, accounting for 9.3% of the total mitogenome length. All 22 tRNAs demonstrated a bias towards AT content of 54.5%, showing an AT-skewness and a GC-skewness value of 0.087 and − 0.102, respectively (Table <a href="#Tab2" class="usa-link">2</a>). Furthermore, the secondary structure of the majority of tRNA molecules displayed the typical cloverleaf shape, with the exception of <em>tRNA-Ser1</em>, which failed to form this structure due to the absence of a nucleotide bond in its DHU arm. A total of 17 tRNA genes were found to be structured through a combination of standard Watson-Crick base pairing and Wobble base pairing (G-T, T-T, G-G, G-A), leading to an interesting blend of conventional and mismatched pairings in their overall configuration. In contrast, the remaining five tRNA genes rely exclusively on Watson-Crick base pairing (Supplementary Fig. <a href="#MOESM1" class="usa-link">S1</a>). Comparative analysis showed that all four <em>Pagellus</em> species have a similar anticodon pattern in all 22 tRNA genes (Supplementary Table S5).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Structures of control region</h3>
<p id="Par27">The CR of <em>P. bellottii</em> spans 987 bp located in between <em>tRNA-Pro</em> and <em>tRNA-Phe</em>, contributing 5.9% of the complete mitogenome. Comparative analyses showed that the CR length varies among four <em>Pagellus</em> species, with <em>P. bogaraveo</em> having the longest one (1,195 bp), followed by <em>P. erythrinus</em> (1,154 bp), and <em>P. acarne</em> (781 bp). Notably, the CR of all four <em>Pagellus</em> species showed a similar portion of AT content ranged from 62.5% in <em>P. erythrinus</em> to 63.9% in <em>P. acarne</em>, with AT skewness varies among <em>Pagellus</em>, ranging from − 0.006 in <em>P. bellottii</em> to 0.029 in <em>P. bogaraveo</em> (Table <a href="#Tab2" class="usa-link">2</a>). Since the CR showed a bias towards AT, the GC content is low, resulting in GC-skewness ranging from − 0.233 in <em>P. bogaraveo</em> to −0.135 in <em>P. acarne</em>.</p>
<p id="Par28">The CR of four <em>Pagellus</em> also incorporates four conserved sequence blocks (CSB): CSB-1, CSB-2, CSB-3, and CSB-D. Among these conserved domains, CSB-1 is the longest (21 bp) compared to the others, ranging from 18 bp (CSB-2 and CSB-D) to 19 bp (CSB-3). Comparative analysis of CSB-D showed that <em>P. bellottii</em>, <em>P. acarne</em>, and <em>P. bogaraveo</em> were classified in similar groups, showing a similarity in conserved nucleotides pattern, while <em>P. erythrinus</em> has a single base pair deviation compared to its congeners. In CSB-1, <em>P. bellottii</em> and <em>P. bogaraveo</em> exhibited conserved nucleotide sequences, whereas the other two species displayed nucleotide variations at three positions. In CSB-2 and CSB-3, <em>P. bellottii</em> and <em>P. erythrinus</em> showed a consistent pattern in conserve nucleotide, while <em>P. bogaraveo</em> had 1 bp and 3 bp differences in CSB-2 and CSB-3, respectively. Unfortunately, due to the shorter sequence of CR in <em>P. acarne</em>, the comparison of nucleotide composition within the CSB-2 and CSB-3 is restricted with other three <em>Pagellus</em> species. Moreover, the CR of <em>P. bellottii</em> and <em>P. acarne</em> showed no repetitive sequences, while the CR of the two other congeners exhibited long repetitive nucleotide patterns in the extended termination-associated sequences (ETAS) region, with <em>P. bogaraveo</em> having two copies of 183 bp consensus base pairs and <em>P. erythrinus</em> having two copies of 160 bp consensus base pairs (Fig. <a href="#Fig4" class="usa-link">4</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365136_41598_2025_16187_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/02ad/12365136/571ab7ada816/41598_2025_16187_Fig4_HTML.jpg" loading="lazy" id="d33e2529" height="619" width="776" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Nucleotide features of the control regions (CRs) in the mitogenomes of four <em>Pagellus</em> species, emphasizing conserved sequence blocks. Highly conserved nucleotides are denoted by black asterisks. The graphical panel below depicts the presence or absence of tandem repeats, along with their sequence characteristics in different <em>Pagellus</em> species.</p></figcaption></figure></section><section id="Sec45366"><h3 class="pmc_sec_title">Matrilineal phylogeny and divergence time</h3>
<p id="Par30">Both Bayesian (BA) and Maximum Likelihood (ML) phylogenetic trees revealed a cohesive clustering of Sparidae family members, supported by high posterior probabilities and bootstrap values, in contrast to the outgroup taxa from Lethrinidae and Nemipteridae (Fig. <a href="#Fig5" class="usa-link">5</a>, Supplementary Fig. S2). Within the Sparidae, species members of the Pagellinae, Sparinae, Denticinae, and Boopsinae subfamilies exhibited clear non-monophyletic clustering. Notably, two major clusters were identified among all sparid species, one cluster comprises species from the genera <em>Dentex</em>, <em>Evynnis</em>, <em>Pagellus</em>, <em>Pagrus</em>, <em>Argyrops</em>, <em>Polysteganus</em>, and the monotypic genus <em>Parargyrops</em>, while the other includes species from <em>Calamus</em>, <em>Stenotomus</em>, <em>Archosargus</em>, <em>Boops</em>, <em>Rhabdosargus</em>, <em>Diplodus</em>, <em>Acanthopagrus</em>, <em>Pagellus</em>, and the monotypic genera <em>Lagodon</em> and <em>Sparus</em>. Specifically, the phylogenetic trees revealed a non-monophyletic cladistics pattern in three genera: <em>Pagrus</em>,<em> Dentex</em>, and <em>Pagellus.</em> Within the genus <em>Pagellus</em>, <em>P. bellottii</em> exhibits the closest phylogenetic relationship with <em>P. erythrinus</em>, while <em>P. bogaraveo</em> and <em>P. acarne</em> are resolved as sister species based on the current mitogenomic-based cladistic analysis. Furthermore, divergence time estimation for sparid species indicated a split between the two major clusters at approximately 28.8 million years ago (MYA), likely occurring during the Oligocene epoch. The divergence between <em>P. bellottii</em> and <em>P. erythrinus</em> was estimated to have occurred around 6.1 MYA, during the Miocene epoch (Fig. <a href="#Fig6" class="usa-link">6</a>). As this event is more recent than the divergence between <em>P. bogaraveo</em> and <em>P. acarne</em>, which occurred approximately 10.5 MYA in the Atlantic Ocean and Mediterranean Sea, <em>P. bellottii</em> and <em>P. erythrinus</em> can be considered the ‘younger sister’ species among the four <em>Pagellus</em> congeners.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365136_41598_2025_16187_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/02ad/12365136/1eec5fd53df8/41598_2025_16187_Fig5_HTML.jpg" loading="lazy" id="d33e2666" height="706" width="776" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Bayesian phylogeny based on complete mitogenomes, depicting the non-monophyletic evolutionary relationships of the four <em>Pagellus</em> species within the Sparidae lineage. Posterior probability values are indicated at each node. Subfamilial classifications are represented by colored boxes adjacent to each clade. Distribution ranges of the four species are integrated with the phylogeny to visualize their biogeographic patterns across the Atlantic Ocean and Mediterranean Sea. All distribution maps were produced using ArcGIS version 10.6 based on IUCN data. The photograph of <em>P. bellottii</em> was taken by the second author (E.O.M.E.), while images of <em>P. erythrinus</em>, <em>P. bogaraveo</em>, and <em>P. acarne</em> were sourced from Wikimedia Commons (Creative Commons Attribution 4.0) and edited in Adobe Photoshop CS 8.0.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365136_41598_2025_16187_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/02ad/12365136/875bc4095563/41598_2025_16187_Fig6_HTML.jpg" loading="lazy" id="d33e2711" height="575" width="778" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Maximum Likelihood-based timetree estimating the divergence times among <em>Pagellus</em> congeners, with divergence estimates denoted by red font at each node. Calibration points (orange diamonds) are based on previous fossil-derived data<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>indicating the divergence between <em>P. acarne</em> and <em>P. bogaraveo</em> during the Messinian-Miocene epoch (lower 7–upper 14 Mya). (<strong>B</strong>–<strong>D</strong>) Paleogeographic maps depicting tectonically driven changes in the Atlantic–Mediterranean gateway from the middle Miocene to the present. The black arrows trace the inferred drift path of the Alboran Plate from the early-middle Miocene through the formation of the Gibraltar Arc. The source map was obtained from previously published literature<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>and edited manually with prior permission granted by Walter Capella through personal communication. (<strong>E</strong>) Map showing the present-day sea surface temperature map of the Atlantic Ocean, generated using ArcGIS version 10.6 with environmental layers sourced from Bio-ORACLE v3.0 database (<a href="https://bio-oracle.org/)an" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bio-oracle.org/)an</a>d manually refined in Adobe Photoshop CS 8.0.</p></figcaption></figure></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par33">Despite variations in total length, the mitochondrial genomes of the four <em>Pagellus</em> species exhibit notable similarity, encompassing 37 genes and a control region. The organization and arrangement of these genes align with the typical mitochondrial architecture observed in other teleosts, including Spariformes<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. The mitogenomic base composition of four <em>Pagellus</em> with AT-bias is also similar to vertebrates, including teleosts<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. It is aligned with the hydrophobic character in mitochondrial proteins<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. The results indicate the mitogenome of <em>P. bellottii</em> is conserved similarly to other <em>Pagellus</em> or sparids<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. The overall organization of mitochondrial genomes, particularly gene order, plays a critical role in inferring gene rearrangement events, as comparisons with the putative ancestral gene order provide essential insights into organismal evolutionary trajectories<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Consequently, comprehending gene organization and base composition in mitogenome is pivotal for enhancing the accuracy of species classification. Additionally, placement of mitochondrial genes can impact various biological aspects in fish, including physiology, life history, molecular mechanisms, and other evolutionary processes<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>.</p>
<p id="Par34">The start and stop codon pattern of <em>Pagellus</em> species is consistent with investigation in other subfamilies within the Sparidae family<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. In this study, amino acids with hydrophobic characteristics are more frequently, with Leucine being found as the most frequent than others with hydrophilic or neutral hydropathy properties. The observed high abundance of hydrophobic amino acids is consistent with findings reported in other teleost species<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. The hydropathy characteristics of amino acids fundamentally shape the evolution of mitochondrial proteins, which are vital for regulating tissue-level metabolism and enabling organisms to adapt to fluctuating environmental conditions, thereby underscoring their evolutionary significance<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Given the frequent environmental fluctuations in the Atlantic Ocean and their potential impact on the observed range expansion of <em>P. bellottii</em>, further investigations into the hydropathy characteristics and adaptive modifications of mitochondrial proteins are warranted. Such investigation could provide valuable insights into the role of these modifications in improving protein configuration and their functional purposes under various changes of environmental stressors. Such mitogenomic information facilitates the inference of adaptive mechanisms and metabolic responses that underpin the resilience and tolerance of aquatic organisms, including <em>Pagellus</em> species, within the highly variable habitats of tropical and subtropical Atlantic and Mediterranean marine ecosystems<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>.</p>
<p id="Par35">Moreover, the present findings demonstrate that the codon usage patterns of PCGs are conserved across all four <em>Pagellus</em> species, and that various codon combinations contribute to the translation of specific amino acids. Overall, the RSCU values deviated from 1, indicating varying degrees of codon usage bias among different codons in the <em>Pagellus</em> species. This codon usage bias is closely associated with gene expression levels; highly expressed genes tend to utilize specific codons more frequently than genes with lower expression efficiency<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Compared to low-expression genes, highly expressed genes exhibit a stronger codon usage bias and preferentially employ a subset of synonymous codons. The findings of this study suggest that PCGs in <em>Pagellus</em> species are conserved and likely maintain similar functional roles. In various organisms, codon usage bias arises from a combination of mutation pressure, natural selection, and genetic drift. Furthermore, codon bias is influenced by multiple factors, including tRNA abundance and interactions, recombination rates, mRNA secondary structure, codon position and context within genes, GC content, gene expression levels, gene length, and overall genomic composition<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>.</p>
<p id="Par36">Furthermore, the calculation of Ka/Ks ratio is a widely acknowledged method for estimating selective pressure in line with Darwinian theory, providing insights into selection pressure at the molecular level across both same and closely related taxa<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. This selection pressure is fundamental for understanding the evolutionary dynamics of genes subjected to selective forces and plays a crucial role in elucidating species divergence. Therefore, detailed analysis of the Ka/Ks substitution ratios in PCGs provides insight into the selection pressures acting on each gene, a ratio greater than 1 indicates positive selection, a ratio equal to 1 suggests neutral evolution, and a ratio less than 1 reflects purifying (negative) selection<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. In this study, the Ka/Ks ratios in PCGs were less than 1, indicating strong purifying selection to preserve their functional integrity, with the majority of mutations occurring as synonymous substitutions across all species within the Sparidae family. Among the 13 PCGs, the <em>ND2</em> gene exhibited the highest Ka/Ks ratio, followed by <em>ATP8</em> and <em>ND6</em>, suggesting that <em>ND2</em> has undergone the least selective constraint and evolved at a relatively faster rate. In contrast, <em>COI</em> displayed the lowest Ka/Ks ratio, followed by <em>Cytb</em>, indicating strong purifying selection and slower evolutionary rates. These findings highlight the role of natural selection in eliminating deleterious nucleotide mutations through negative selective pressure, consistent with patterns previously observed in teleost species<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>–<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. The Ka/Ks analysis of <em>Pagellus</em> and related species provides a valuable framework for understanding the subtleties of natural selection. The strong purifying selection acting on PCGs enhances the evolutionary resilience of <em>Pagellus</em> species, contributing to genetic stability and facilitating speciation through ecological adaptation and colonization. This process reflects the evolutionary trajectory and native distribution of <em>Pagellus</em> species, offering insights into the complex interplay between mutational events and selective pressures in shaping their evolution. Such interactions are fundamental to protein evolution, as demonstrated by the composition and relative abundance of PCGs<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>.</p>
<p id="Par37">The complete mitochondrial genome of <em>P. bellottii</em> contains both the small and large rRNA subunits encoded on the heavy strand, consistent with the mitochondrial gene arrangement observed in other <em>Pagellus</em> species<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. These rRNA genes, which are generally maintained as ribonucleoproteins, play a pivotal function during genetic information translation from mRNAs into amino acids, providing constructive understanding into the essential mechanisms to accelerate the protein synthesis<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. The tRNAs in <em>P. bellottii</em> predominantly exhibited the canonical secondary structures, except for <em>tRNA-Ser1</em>, which deviated due to missing or mismatched base pairs in the DHU arm. This missing base-pairing feature is commonly detected in the vertebrates, including the order Spariformes<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Comparative analysis revealed that the organization and structure of tRNA genes are conserved among the three <em>Pagellus</em> species, except in <em>P. bogaraveo</em>, which contains a 66 bp <em>tRNA-Cys</em> pseudogene within the WANCY region<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. This distinct organization of tRNA genes and the presence of heteroplasmy in the WANCY region are critical for the expression and functional integrity of mitochondrial tRNA genes<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>.</p>
<p id="Par38">The CR of <em>P. bellottii</em> exhibits an A-T bias, consistent with other <em>Pagellus</em> congeners and sparid species, and contains four conserved sequence blocks, as observed in sparid fishes like <em>Pagrus major</em><sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. The investigation of these conserved domains is significant due to the presence of highly conserved nucleotides within the highly variable CR. Therefore, the nucleotide variations identified within these conserved domains serve as critical markers that can be employed for species-level differentiation<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. The nucleotide composition of the CR in <em>Pagellus</em> species, including the detection of polymorphic sites within conserved blocks, offers valuable insights for the development of specific molecular markers for species identification and for elucidating population structure. Furthermore, elucidating the complex mechanisms governing the CR including random loss, dimer formation, non-random loss, duplication, and genomic rearrangements is crucial for comprehending the structural diversity of mitochondrial genes and the evolutionary dynamics<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. The detection of tandem repeats within the ETAS region of <em>P. bogaraveo</em> and <em>P. erythrinus</em> further highlights notable diversity and distinct repeat patterns that likely contribute to the formation of stable hairpin loops, which are recognized as sequence-specific signals involved in regulating mitochondrial DNA replication termination<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>.</p>
<p id="Par39">The present cladistic framework derived from mitogenome data corroborates earlier evolutionary hypotheses concerning the non-monophyletic nature of Sparidae<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> as well as certain genera including <em>Pagellus</em>, as previously inferred from partial mitochondrial and nuclear markers<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>–<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. These findings are also consistent with phylogenomic analyses based on six mitochondrial and 21 nuclear genes, which similarly reveal a non-monophyletic pattern among <em>Pagellus</em> species in the marine environment<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Our findings indicate that <em>P. bellottii</em> is closely related to <em>P. erythrinus</em>, while exhibiting a more distant phylogenetic relationship to the other two congeners. This observed phylogenetic affinity between <em>P. bellottii</em> and <em>P. erythrinus</em> may be further substantiated by the notable morphological similarities shared between these two species. Both <em>P. bellottii</em> and <em>P. erythrinus</em> exhibit a more laterally compressed body shape, whereas the other two species display a more oblong and fusiform morphology. Additionally, the pattern of scalation on the dorsal surface of the head terminates anterior to a transverse line through the middle of the eye in <em>P. bellottii</em> and <em>P. erythrinus</em>, in contrast to the posterior termination in <em>P. bogaraveo</em> and <em>P. acarne.</em> The mouth configuration in <em>P. bellottii</em> and <em>P. erythrinus</em> is low-set and slightly oblique, while it is low-set and nearly horizontal in the other two species. The number of gills rakers on the lower and upper limbs of the first arch ranges from 8 to 10 and 5–6, respectively, in <em>P. bellottii</em> and <em>P. erythrinus</em>, whereas higher counts are observed in the other two species. Lateral line scale counts further support this relationship, with <em>P. bellottii</em> exhibiting 54–60 and <em>P. erythrinus</em> 55–65 scales, in contrast to the significantly higher counts in <em>P. bogaraveo</em> (68–74) and <em>P. acarne</em> (65–72). Furthermore, coloration of the oral cavity also differs as it is typically whitish to greyish in <em>P. bellottii</em> and <em>P. erythrinus</em>, while <em>P. bogaraveo</em> and <em>P. acarne</em> display a distinct orange-red hue<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Hence, both the phylogenetic relationships and morphological characteristics provide a more comprehensive understanding of the evolutionary affinities between <em>P. bellottii</em> and <em>P. erythrinus</em>, particularly in contrast to the more distantly related congeners <em>P. bogaraveo</em> and <em>P. acarne</em>.</p>
<p id="Par40">Notably, divergence time estimates indicate an approximate 4.4 MYA interval between the initial divergence of <em>P. acarne</em> and <em>P. bogaraveo</em>, and the subsequent split between <em>P. bellottii</em> and <em>P. erythrinus</em>. These findings suggest that evolutionary processes and reproductive isolation within the <em>Pagellus</em> lineage have occurred over an extended timescale in the marine environment, highlighting the long-term nature of sparid evolution as reported in previous studies<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Based on the estimated divergence times, it can be hypothesized that geological events in the Atlantic–Mediterranean gateway and the formation of Gibraltar Arc may have driven the speciation and lineage diversification of <em>Pagellus</em> in both the Eastern Atlantic Ocean and the Mediterranean Sea. The split divergence of <em>Pagellus</em> species can be traced back to the late Miocene period corresponding to the disconnection of the Atlantic Ocean and Mediterranean Sea followed by the hypersaline event also known as the Messinian salinity crisis<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Subsequently, the connection between the Atlantic and Mediterranean was re-established during the Pliocene (5.4 to 1.8 MYA) leading to a shared geographical distribution for major sparids<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. It is worth discussing that three <em>Pagellus</em> species are widely distributed across the Mediterranean Sea and the northeastern Atlantic Ocean, whereas <em>P. bellottii</em> has extended its native range into the southeastern Atlantic. This asymmetric distribution pattern of <em>P. bellottii</em> may be influenced by hydrographic and climatic conditions in the North and South Atlantic Oceans, which have gradually diverged due to the Coriolis effect driven by Earth’s rotation<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. In the North Atlantic, distinct oceanic gyres form due to oceanic current circulation, with the warm Gulf Stream current flowing away northward and the cold Canary current flowing southward, restricting the distribution of tropical marine fishes species towards the northward region<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. In addition, the existence of cold-Benguela and warm-Agulhas currents along with the South Equatorial warm currents in southern coast of Africa are profound primary physical barrier which may restrict the overlap distribution of the Atlantic <em>Pagellus</em> species with its congeners (<em>P. natalensis</em> and <em>P. affinis</em>) distributed in Western Indian Ocean<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>.</p>
<p id="Par41">Furthermore, the recent northward expansion of <em>P. bellottii</em> into the northeastern Atlantic and southeastern Mediterranean Sea could be attributed to abiotic factors, such as increasing sea surface temperatures, which may be influencing its distribution. A similar pattern of northward range expansion has been observed in several tropical marine fish species, likely driven by the accelerated warming of marine environments over recent decades<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. This indicates that the cold-water current passing through the Canary Islands warms up, may enable <em>P. bellottii</em> to expand its distribution range toward the Bay of Biscay<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Moreover, the range expansion of <em>P. bellottii</em> to Haifa Bay in the southeastern Mediterranean Sea suggests that this species might possess a high tolerance to salinity, as the Mediterranean Sea has exhibited higher salinity levels than the Atlantic Ocean since the end of the Messinian Salinity Crisis<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>.</p>
<section id="Sec10"><h3 class="pmc_sec_title">Limitations and future direction</h3>
<p id="Par42">The present study provides the first complete mitochondrial genome of <em>P. bellottii</em> from the Eastern Atlantic Ocean and renders insights into its structural characteristics and phylogenetic placement within the major Sparidae clade; however, several limitations persist that warrant attention in future research. Notably, the current analysis is based on a single specimen collected from Ghana, which may not adequately capture the species’ genetic diversity or its phylogeographic structure, given its broader distribution from the Eastern Atlantic to the Bay of Biscay in the northward Atlantic region and the south-eastern Mediterranean Sea. Future investigations should therefore incorporate extensive surveys and sampling across the full geographic range of the species, including multiple individuals from different populations. This approach would enable a more in-depth understanding of population structure, gene flow, and demographic history by incorporating both partial and complete mitochondrial genetic data. Moreover, this study incorporates mitogenomic data only from four out of six currently recognised <em>Pagellus</em> species worldwide. Hence, the addition of complete mitogenomes for <em>P. affinis</em> and <em>P. natalensis</em>, which inhabit the Western and Southwestern Indian Ocean, is essential for clarifying the comprehensive matrilineal evolutionary relationships and lineage diversification within this genus. Additionally, the application of whole-genome sequencing and genome-wide single nucleotide polymorphism (SNP) analyses would enable more precise inferences of genetic variation, population divergence, and adaptability of <em>Pagellus</em> species in diverse marine ecosystems. Ultimately, an integrative approach encompassing morphological and genetic analyses, biogeographic patterns, and geological history will yield deeper insights into the evolutionary trajectory and present-day population structure of this species. Such findings will significantly contribute to marine ichthyological systematics, conservation strategies, and aquaculture development.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par43">In the present study, the complete mitochondrial genome of <em>P. bellottii</em> is reported for the first time. The comprehensive analyses offer an in-depth understanding of the genetic structure and variations of this species compared with its closely related congeners. The phylogenetic reconstruction further elucidates the non-monophyletic clustering of <em>Pagellus</em> and inferring the evolutionary relationships within the broader Sparidae family. Considering the overlapping geographic distributions of four <em>Pagellus</em> species and the recently documented range expansion of <em>P. bellottii</em> into the Bay of Biscay (northeastern Atlantic) and Haifa Bay (southeastern Mediterranean), this study discusses potential lineage diversification driven by Miocene-era geological events, subsequently influenced by oceanic currents and ongoing climate change. Overall, the findings underscore the significance of integrating mitogenomic data and demographic history to understand matrilineal speciation and diversification processes, particularly in the context of the biogeographical complexity of the Atlantic Ocean and Mediterranean Sea.</p></section><section id="Sec12"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec13"><h3 class="pmc_sec_title">Sample handling and species identification</h3>
<p id="Par44">A sample of <em>P. bellottii</em> were collected from Tema, Ghana (5.611389 N, 0.044444 W) (Fig. <a href="#Fig1" class="usa-link">1</a>). Muscle tissues between the area of the lateral line and the dorsal fin were carefully excised under sterile conditions, placed in 95% ethanol, and stored in 2 ml tubes. To prevent cross-contamination and DNA hydrolysis, the tissue sample was stored in freezer at − 20 °C. Later on, the specimen was vouchered in 10% formaldehyde and deposited at the Fisheries Scientific Survey Division under Ghana’s Ministry of Fisheries and Aquaculture Development (MOFAD). The study design and all methods were conducted in compliance with the relevant guidelines and regulations of ARRIVE 2.0 (<a href="https://arriveguidelines.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arriveguidelines.org</a>), specifically for animal research<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Then species identification was confirmed through several key morphological features<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. All laboratory examinations were conducted by using the relevant guidelines and regulations approved by the Pukyong National University Institutional Animal Care and Use Committee (PKNU-IACUC) under Approval No. PKNUIACUC-2025-16, dated February 18, 2025.</p></section><section id="Sec14"><h3 class="pmc_sec_title">DNA extraction and COI marker sequencing</h3>
<p id="Par45">The extraction of the genomic DNA was performed using the AccuPrep<sup>®</sup> DNA Extraction Kit (Bioneer, Republic of Korea) according to the standard instructions provided by the manufacturer. A paired-set primer Fish-BCH (5′-TCAACYAATCAYAAAGATATYGGCAC-3′) and Fish-BCL (5′-ACTTCYGGGTGRCCRAARAATCA-3′) was used to amplify the targeted mitochondrial <em>COI</em> partial marker (~ 650 bp)<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Polymerase Chain Reaction (PCR) was conducted using a Takara thermal cycler with a 30 µL reaction mixture consisting of 1 µL each of forward and reverse primers, 0.9 µL of 3% dimethyl sulfoxide (DMSO), 19.9 µL of sterilized deionized water, 3 µL of 10× ExTaq Buffer, 0.2 µL of Ex Taq HS enzyme, 3 µL of dNTPs, and 1 µL of 1/10 diluted target DNA template. The thermal conditions included an initial denaturation at 94º C for 3 min, followed by 40 cycles of denaturation at 94º C for 30 s, annealing at 50º C for 30 s, extension at 72º C for 1 min, and a final extension at 72º C for 5 min. Gel electrophoresis with 1.5% agarose was applied to visualize the DNA integrity, followed by a DNA purification assessment using NanoDrop spectrophotometer (Thermo Fisher Scientific, USA). Successfuly amplified PCR product was sent and read using the Sanger sequencing at Macrogen (Republic of Korea). The PCR product was purified using the AccuPrep<sup>®</sup> Purification Kit (Bioneer, Republic of Korea) to remove impurities, and then sequenced bidirectionally at Macrogen (Daejeon, Republic of Korea). The <em>COI</em> sequence was subjected to species examination based on sequence query and similarity using nucleotide BLAST (<a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://blast.ncbi.nlm.nih.gov/Blast.cgi</a>) and was submitted in the GenBank database (Accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ242656" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ242656</a>).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Mitogenome sequencing and annotation</h3>
<p id="Par46">The high-throughput next-generation sequencing (NGS) with NovaSeq platform was employed for complete mitogenome analysis. A paired-end library with size 2 × 150 bp was generated through the TruSeq library preparation kit for Illumina<sup>®</sup> (Illumina, USA). About 100 ng of extracted DNA was fragmented using a Covaris ultrasonicator technology to generate double-helix DNA with straight ends and 5′ phosphorylation. After end reparation and ‘A’ tailing, the fragments were ligated as stubs with TruSeq UD DNA Indexing adapters. To construct the final library, the purification and PCR enrichment of the resulting products were performed using a bead-based method. Library quantification and size distribution quality assessment were performed using KAPA Library Quantification Kits for Illumina and the Agilent Technologies 4200 TapeStation D1000 (Agilent Technologies, USA), with sequencing conducted by Macrogen.</p>
<p id="Par47">The NGS reading and assembly were constructed using Geneious Prime and referred to the published <em>P. erythrinus</em> mitogenome (Genbank Accession No. <a href="https://www.ncbi.nlm.nih.gov/nuccore/MG653592" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MG653592</a>) using default mapping algorithm. The PCGs were validated by adjusting overlapping regions with MEGA X<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. The genes placements and boundaries were determined by employing the MITOS Galaxy version 1.1.6<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> and MitoFish MitoAnnotator webserver<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Moreover, the boundaries of each PCG, including the start and stop codons were verified using the Open Reading Frames (ORFs) Finder webtool in GenBank (<a href="https://www.ncbi.nlm.nih.gov/orffinder" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/orffinder</a>), with standard genetic code for vertebrate mitochondrial analysis in conjunction with MITOS Galaxy webserver. The final annotated <em>P. bellottii</em> mitogenome was submitted to the GenBank database (Accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ524309" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ524309</a>).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Characterization and comparative analyses</h3>
<p id="Par48">The mitogenome of <em>P. bellottii</em> was represented as a circular map using MitoFish MitoAnnotator webserver. Comparative analyses were performed to assess mitogenomic structural variations in <em>P. bellottii</em> with three other congeners within <em>Pagellus</em> genus: <em>P. bogaraveo</em> (Genbank Accession No. <a href="https://www.ncbi.nlm.nih.gov/nuccore/AB305023" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AB305023</a>)<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup><em>P. acarne</em> (GenBank Accession No. <a href="https://www.ncbi.nlm.nih.gov/nuccore/MG736083" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MG736083</a>)<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>and <em>P. erythrinus</em> (GenBank Accession No. <a href="https://www.ncbi.nlm.nih.gov/nuccore/MG653592" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MG653592</a>)<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Base compositions of PCGs, tRNA, rRNA, and the CR were evaluated using MEGA X, and intergenic and overlapping regions between contiguous genes were calculated manually. The nucleotide skewness, specifically the AT and GC skews, was calculated using the following formulas: AT-skew = (A − T)/(A + T) and GC-skew = (G − C)/(G + C), respectively<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>.</p>
<p id="Par49">Nucleotide diversity (π) for all <em>Pagellus</em> species was scrutinized using a sliding window method in DnaSP ver 6.0<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>, applying a 25 bp step size, which is the size of the shifted sequence within each window, and a 200 bp window size, which means the sequence size at each time. By using these sizes, the accuracy of determining the location of protein coding regions in eukaryotes can be significantly improved<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. The DnaSP software was also employed to calculate the pairwise substitution patterns in PCGs among four <em>Pagellus</em> species and other Sparidae family members, including the ratio of Ks and Ka. In addition, the transition and transversion codon saturation rates across the PCGs of all Sparidae species were assessed using the DAMBE6 software<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. To analyze the RSCU and the distribution of amino acids within the PCGs, the dataset of all <em>Pagellus</em> species was aligned using MEGA X. The secondary structures of tRNAs were reconstructed using the tRNA prediction tools in ARWEN 1.2<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> and ARAGORN<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> which were integrated into the MITOS Galaxy web server. Furthermore, the conserved blocks within the CR were identified using CLUSTAL X<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup> in MEGA X referring to the published CR of <em>P. major</em> in a previous study<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. As repeat sequences serve a vital role for further development markers in population structures studies, we employed the Tandem Repeats Finder program to detect the tandem repeats pattern within the CR<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Dataset construction and phylogenetic analyses</h3>
<p id="Par50">To construct the matrilineal lineage of four <em>Pagellus</em> and other sparids, a dataset encompassing the complete mitogenomes of 32 species was assembled, including a newly sequenced mitogenome from this study (Supplementary Table S6). In addition, two mitogenomes of <em>Lethrinus obsoletus</em> (Accession No. <a href="https://www.ncbi.nlm.nih.gov/nuccore/AP009165" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AP009165</a>)<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> and <em>Nemipterus japonicus</em> (Accession No. <a href="https://www.ncbi.nlm.nih.gov/nuccore/KJ473717" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KJ473717</a>)<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup> under the families Lethrinidae and Nemipteridae, respectively were designated as outgroups. To investigate matrilineal phylogenetic relationships within <em>Pagellus</em> and other Sparidae species, a concatenated dataset of 13 PCGs were generated using the Concatenator Tool in iTaxoTools ver 0.1<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. The ‘GTR + G + I’ model was determined to be the most suitable based on the lowest Bayesian Information Criterion (BIC) value, as assessed using both PartitionFinder ver 2<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup> and the JModelTest program<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. Then, phylogenetic trees were constructed by Bayesian analysis employing a Metropolis-coupled Markov Chain Monte Carlo (MCMC) algorithm in MrBayes 3.1.2<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. The ML-based phylogenetic tree was made with standard settings in PhyML 3.0 online server<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. The two phylogenetic trees were then saved in Newick format (*.nwk) and visualized using Interactive Tree of Life (iTOL)<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Divergence time estimation</h3>
<p id="Par51">The estimation of diverging times among sparids lineage, including the four <em>Pagellus</em> species, were calculated by RelTime<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup> method with the ML topology as the baseline tree in MEGA 12<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. The RelTime method is widely recognized for its efficiency and accuracy in divergence time estimation, making it particularly well-suited for analysing large datasets with reduced computational time<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. Thus, a known fossil calibration was applied to constrain the analysis based on the splitting age between <em>P. acarne</em> and <em>P. bogaraveo</em> by determining the fossil of <em>Pagellus leptosomus</em> from the Messinian diatomites of the Chelif Basin, Algeria (about 7 MYA) as the lower splitting age of both species. Additionally, fossils of <em>Boops</em> sp. were also used as the upper boundary from the Messinian diatomites and middle Miocene (about 14 MYA), as described in the previous study<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. The generated timetree was further refined and visually enhanced using MEGA12 to improve clarity and presentation quality.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Geospatial data acquisition and processing</h3>
<p id="Par52">The generation of the maps entailed a multi-step process involving the acquisition and processing of both vector and raster datasets. Initially, global administrative boundary shapefiles were downloaded from the DIVA-GIS platform (<a href="https://diva-gis.org/data.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://diva-gis.org/data.html</a>), which offers high-resolution vector data suitable for spatial analysis. These shapefiles were subsequently imported into ArcMap within the ArcGIS v10.6 environment for further geospatial processing and overlay analyses. Global-scale Ocean current data were sourced from the NOAA National Weather Service and the U.S. Army (<a href="https://data.amerigeoss.org/dataset/major-ocean-currents-arrowpolys-100m-76" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://data.amerigeoss.org/dataset/major-ocean-currents-arrowpolys-100 m-76</a>; accessed on 15 May 2025). Additionally, bathymetric and sea surface temperature layers from the CMIP6 Earth System Models were obtained via the Bio-ORACLE v3.0 database (<a href="https://bio-oracle.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bio-oracle.org/</a>)<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. All spatial datasets were reprojected to the WGS 84 geographic coordinate system to ensure interoperability and spatial consistency. The processed datasets were then integrated and exported for the creation of the final thematic maps. In addition, paleogeographic maps illustrating tectonically driven changes in the Atlantic–Mediterranean gateway and the evolution of the Gibraltar Arc from the middle Miocene to the present were manually adapted from previously published literature<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>.</p></section></section><section id="Sec21"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365136/bin/41598_2025_16187_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (768.1KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The first author (A.W.) extends heartfelt gratitude to the Ministry of Oceans and Fisheries, Republic of Korea, for their generous support through the ‘Establishing Integrated Ocean Fisheries Technology Training Center and Enhancing Capacity Building in Indonesia (PG54000)’ program, which played a pivotal role in enabling his doctoral studies at PKNU, Republic of Korea. The authors would also like to express sincere thanks to Emmanuel Dovlo and Eunice Nuerkie Ofoli-Anum for their exceptional assistance in obtaining the necessary documentation for the shipment of samples. We also acknowledge the staff of the Fisheries Scientific Survey Division of the Fisheries Commission Ghana, including Audrey Lamptey, Salman Sheini, Peter Baah Essel, Kelvin Kafui Fiadoyor, Linda Yeboah, Efua Ansah Mensah, Ama Nyarkoa Sarpong, and Agness Mensima Arthur, for their invaluable contribution to the sampling efforts. We also thank Walter Capella for granting permission to reproduce and modify maps from his publication, and Imon Abedin for his assistance in generating the distribution map of the studied species. </p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>H.-W.K., and S.K. conceived the study and assisted with the study design; A.W., E.O.M.E., S.P.S., and M.H.F.A., were responsible for the data collection and analysis; A.W., E.O.M.E., H.-E.K. wrote the original draft; E.L.V., H.W.K., and S.K. critically reviewed the manuscript and ensured the accuracy of the findings; H.W.K. and H.-E.K. were responsible for the project administration and funding acquisition; All authors engaged in discussions regarding the findings and made valuable contributions to the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by the National Research Foundation of Korea (NRF) grant funded by the Korea government (MSIT) (RS-2021-NR058802). This work was also supported by the Global Joint Research Program funded by the Pukyong National University (202412070001).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The data presented in this study can be found in GenBank (<a href="https://www.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/</a>), <em>COI</em> Accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ242656" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ242656</a> and complete mitogenome Accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ524309" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ524309</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par54">The authors declare no competing interests.</p></section><section id="FPar4"><h3 class="pmc_sec_title">Ethical accordance for procedures</h3>
<p id="Par55">The study design and all methods were conducted in compliance with the relevant guidelines and regulations approved by the Pukyong National University Institutional Animal Care and Use Committee (PKNU-IACUC) under Approval No. PKNUIACUC-2025-16, dated February 18, 2025.</p></section><section id="FPar5"><h3 class="pmc_sec_title">Adherence to ARRIVE guidelines</h3>
<p id="Par56">The study was conducted in compliance with the relevant guidelines and regulations of ARRIVE 2.0 (<a href="https://arriveguidelines.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arriveguidelines.org</a>), specifically for animal research.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Hye-Eun Kang, Email: kanghe24@pukyong.ac.kr.</p>
<p>Shantanu Kundu, Email: shantanu1984@pknu.ac.kr.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Carpenter, K. E. &amp; Niem, V. H. <em>The Living Marine Resources of the Western Central Pacific. Bony Fishes Part 4 (Labridae To Latimeriidae), Estuarine Crocodiles, Sea Turtles, Sea Snakes and Marine Mammals</em>  <strong>6</strong> (FAO, 2001).</cite>
</li>
<li id="CR2">
<span class="label">2.</span><cite>Fricke, R., Eschmeyer, W. N. &amp; Fong, J. D. <em>Eschmeyer’s Catalog of Fishes: Genera/Species by Family/Subfamily.</em><a href="https://researcharchive.calacademy.org/research/ichthyology/catalog/SpeciesByFamily.asp" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://researcharchive.calacademy.org/research/ichthyology/catalog/SpeciesByFamily.asp</a> (2024).</cite>
</li>
<li id="CR3">
<span class="label">3.</span><cite>Basurco, B., Lovatelli, A. &amp; García, B. <em>Wiley-blackwell, current status of Sparidae aquaculture in Sparidae: Biology and aquaculture of gilthead sea bream and other species</em> (eds Pavlidis, M. A. &amp; Mylonas, C.) 1–50 (2011).</cite>
</li>
<li id="CR4">
<span class="label">4.</span><cite>Golani, D., Azzurro, E., Dulčić, J., Massutí, E. &amp; Orsi-Relini, L. <em>Atlas of Exotic Fishes in the Mediterranean Sea</em> (CIESM, 2021).</cite>
</li>
<li id="CR5">
<span class="label">5.</span><cite>Parenti, P. P. An annotated checklist of the fishes of the family Sparidae. <em>FishTaxa</em><strong>4</strong>, 47–98 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Parenti,%20P.%20P.%20An%20annotated%20checklist%20of%20the%20fishes%20of%20the%20family%20Sparidae.%20FishTaxa4,%2047%E2%80%9398%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Amponsah, S. K. K., Ofori-Danson, P. K., Nunoo, F. K. E. &amp; Ameyaw, G. A. Aspects of population dynamics of red pandora, <em>Pagellus bellottii</em> (Steindachner, 1882) from the coastal waters of Ghana. <em>J. Sci. Innov. Res.</em><strong>5</strong>, 215–224 (2016).</cite> [<a href="https://scholar.google.com/scholar_lookup?Amponsah,%20S.%20K.%20K.,%20Ofori-Danson,%20P.%20K.,%20Nunoo,%20F.%20K.%20E.%20&amp;%20Ameyaw,%20G.%20A.%20Aspects%20of%20population%20dynamics%20of%20red%20pandora,%20Pagellus%20bellottii%20(Steindachner,%201882)%20from%20the%20coastal%20waters%20of%20Ghana.%20J.%20Sci.%20Innov.%20Res.5,%20215%E2%80%93224%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Lazar, N. <em>Baseline Assessment of Demersal Fish Stocks of the Western Region of Ghana</em> (University of Rhode Island, 2017).</cite>
</li>
<li id="CR8">
<span class="label">8.</span><cite>Sanz-Fernandez, V. &amp; Gutierrez-Estrada, J. C. Pulido-Calvo, I. <em>Pagellus</em> genus catches time series in the FAO major fishing areas 27 and 34: analysis of fishery behaviour. <em>Mar. Policy</em>. <strong>136</strong>, 104912 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sanz-Fernandez,%20V.%20&amp;%20Gutierrez-Estrada,%20J.%20C.%20Pulido-Calvo,%20I.%20Pagellus%20genus%20catches%20time%20series%20in%20the%20FAO%20major%20fishing%20areas%2027%20and%2034:%20analysis%20of%20fishery%20behaviour.%20Mar.%20Policy.%20136,%20104912%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Russell, B. &amp; Carpenter, K. E. <em>Pagellus bellottii</em>. <em>The IUCN Red List of Threatened Species 2014</em> e.T170162A1285147. 10.2305/IUCN.UK.2014-3.RLTS.T170162A1285147.en (2014). </cite>
</li>
<li id="CR10">
<span class="label">10.</span><cite>Kouame, A. C., Sylla, S., Yao, S. S., Tia, C. B. &amp; Loukou, G. A. Stock assessment of red Pandora <em>Pagellus bellottii</em> (Steindachner, 1882) in the Ivorian continental shelf (West Africa). <em>Agron. Afr.</em><strong>32</strong>, 423–438 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kouame,%20A.%20C.,%20Sylla,%20S.,%20Yao,%20S.%20S.,%20Tia,%20C.%20B.%20&amp;%20Loukou,%20G.%20A.%20Stock%20assessment%20of%20red%20Pandora%20Pagellus%20bellottii%20(Steindachner,%201882)%20in%20the%20Ivorian%20continental%20shelf%20(West%20Africa).%20Agron.%20Afr.32,%20423%E2%80%93438%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Fricke, R., Golani, D., Sonin, O. &amp; Appelbaum-Golani, B. First record of the red Pandora <em>Pagellus bellottii</em> from israel, south-eastern mediterranean (Teleostei: Sparidae). <em>Mar. Biodivers. Rec</em>. <strong>7</strong>, e130 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?Fricke,%20R.,%20Golani,%20D.,%20Sonin,%20O.%20&amp;%20Appelbaum-Golani,%20B.%20First%20record%20of%20the%20red%20Pandora%20Pagellus%20bellottii%20from%20israel,%20south-eastern%20mediterranean%20(Teleostei:%20Sparidae).%20Mar.%20Biodivers.%20Rec.%207,%20e130%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Iglésias, S. P. &amp; Lorance, P. First record of <em>Pagellus bellottii</em> (Teleostei: Sparidae) in the Bay of biscay, France. <em>Mar. Biodivers. Rec</em>. <strong>9</strong>, 16 (2016).</cite> [<a href="https://scholar.google.com/scholar_lookup?Igl%C3%A9sias,%20S.%20P.%20&amp;%20Lorance,%20P.%20First%20record%20of%20Pagellus%20bellottii%20(Teleostei:%20Sparidae)%20in%20the%20Bay%20of%20biscay,%20France.%20Mar.%20Biodivers.%20Rec.%209,%2016%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Whitehead, P. J. P., Bauchot, M. L., Hureau, J. C., Nielsen, J. &amp; Tortonese, E. <em>Fishes of the North-Eastern Atlantic and the Mediterranean </em><strong>2</strong> (UNESCO, 1986).</cite>
</li>
<li id="CR14">
<span class="label">14.</span><cite>Carpenter, K. E. &amp; De Angelis, N. <em>The living marine resources of the Eastern Central Atlantic. Bony fishes part 2 (Perciformes to Tetradontiformes) and Sea turtles.</em> <strong>4</strong> (FAO,2016).</cite>
</li>
<li id="CR15">
<span class="label">15.</span><cite>Al-Amry, W. N., Al-Ansari, A. S. &amp; Leduc, A. O. H. C. Identification of Sparidae species based on COI and 16S rRNA DNA barcoding along the Coast of Oman. <em>Reg. Stud. Mar. Sci.</em><strong>77</strong>, 103677 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Al-Amry,%20W.%20N.,%20Al-Ansari,%20A.%20S.%20&amp;%20Leduc,%20A.%20O.%20H.%20C.%20Identification%20of%20Sparidae%20species%20based%20on%20COI%20and%2016S%20rRNA%20DNA%20barcoding%20along%20the%20Coast%20of%20Oman.%20Reg.%20Stud.%20Mar.%20Sci.77,%20103677%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Orrell, T. M. &amp; Carpenter, K. E. A phylogeny of the fish family Sparidae (porgies) inferred from mitochondrial sequence data. <em>Mol. Phylogenet Evol.</em><strong>32</strong>, 425–434 (2004).
</cite> [<a href="https://doi.org/10.1016/j.ympev.2004.01.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15223028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Orrell,%20T.%20M.%20&amp;%20Carpenter,%20K.%20E.%20A%20phylogeny%20of%20the%20fish%20family%20Sparidae%20(porgies)%20inferred%20from%20mitochondrial%20sequence%20data.%20Mol.%20Phylogenet%20Evol.32,%20425%E2%80%93434%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Wu, R. et al. DNA barcoding of the family Sparidae along the Coast of China and revelation of potential cryptic diversity in the Indo-West Pacific oceans based on COI and 16S rRNA genes. <em>J. Oceanol. Limnol.</em><strong>36</strong>, 1753–1770 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wu,%20R.%20et%20al.%20DNA%20barcoding%20of%20the%20family%20Sparidae%20along%20the%20Coast%20of%20China%20and%20revelation%20of%20potential%20cryptic%20diversity%20in%20the%20Indo-West%20Pacific%20oceans%20based%20on%20COI%20and%2016S%20rRNA%20genes.%20J.%20Oceanol.%20Limnol.36,%201753%E2%80%931770%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Herrán, R. et al. The molecular phylogeny of the Sparidae (Pisces, Perciformes) based on two satellite DNA families. <em>Heredity</em><strong>87</strong>, 691–697 (2001).
</cite> [<a href="https://doi.org/10.1046/j.1365-2540.2001.00967.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11903564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herr%C3%A1n,%20R.%20et%20al.%20The%20molecular%20phylogeny%20of%20the%20Sparidae%20(Pisces,%20Perciformes)%20based%20on%20two%20satellite%20DNA%20families.%20Heredity87,%20691%E2%80%93697%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Alvarenga, M. et al. Mitochondrial genome structure and composition in 70 fishes: a key resource for fisheries management in the South Atlantic. <em>BMC Genom.</em><strong>25</strong>, 215 (2024).</cite> [<a href="https://doi.org/10.1186/s12864-024-10035-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10898094/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38413941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alvarenga,%20M.%20et%20al.%20Mitochondrial%20genome%20structure%20and%20composition%20in%2070%20fishes:%20a%20key%20resource%20for%20fisheries%20management%20in%20the%20South%20Atlantic.%20BMC%20Genom.25,%20215%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Yamanoue, Y. et al. Phylogenetic position of tetraodontiform fishes within the higher teleosts: bayesian inferences based on 44 whole mitochondrial genome sequences. <em>Mol. Phylogenet Evol.</em><strong>45</strong>, 89–101 (2007).
</cite> [<a href="https://doi.org/10.1016/j.ympev.2007.03.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17490896/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yamanoue,%20Y.%20et%20al.%20Phylogenetic%20position%20of%20tetraodontiform%20fishes%20within%20the%20higher%20teleosts:%20bayesian%20inferences%20based%20on%2044%20whole%20mitochondrial%20genome%20sequences.%20Mol.%20Phylogenet%20Evol.45,%2089%E2%80%93101%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Miya, M., Kawaguchi, A. &amp; Nishida, M. Mitogenomic exploration of higher teleostean phylogenies: A case study for moderate-scale evolutionary genomics with 38 newly determined complete mitochondrial DNA sequences. <em>Mol. Biol. Evol.</em><strong>18</strong>, 1993–2009 (2001).
</cite> [<a href="https://doi.org/10.1093/oxfordjournals.molbev.a003741" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11606696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miya,%20M.,%20Kawaguchi,%20A.%20&amp;%20Nishida,%20M.%20Mitogenomic%20exploration%20of%20higher%20teleostean%20phylogenies:%20A%20case%20study%20for%20moderate-scale%20evolutionary%20genomics%20with%2038%20newly%20determined%20complete%20mitochondrial%20DNA%20sequences.%20Mol.%20Biol.%20Evol.18,%201993%E2%80%932009%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Ponce, M., Infante, C., Jiménez-Cantizano, R. M., Pérez, L. &amp; Manchado, M. Complete mitochondrial genome of the blackspot seabream, <em>Pagellus Bogaraveo</em> (Perciformes: Sparidae), with high levels of length heteroplasmy in the WANCY region. <em>Gene</em><strong>409</strong>, 44–52 (2008).
</cite> [<a href="https://doi.org/10.1016/j.gene.2007.11.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18191919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ponce,%20M.,%20Infante,%20C.,%20Jim%C3%A9nez-Cantizano,%20R.%20M.,%20P%C3%A9rez,%20L.%20&amp;%20Manchado,%20M.%20Complete%20mitochondrial%20genome%20of%20the%20blackspot%20seabream,%20Pagellus%20Bogaraveo%20(Perciformes:%20Sparidae),%20with%20high%20levels%20of%20length%20heteroplasmy%20in%20the%20WANCY%20region.%20Gene409,%2044%E2%80%9352%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Mascolo, C. et al. The complete mitochondrial genome of the axillary seabream, <em>Pagellus acarne</em> (Perciformes: Sparidae). <em>Mitochondrial DNA Part. B</em>. <strong>3</strong>, 434–435 (2018).
</cite> [<a href="https://doi.org/10.1080/23802359.2018.1450674" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7800612/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33474194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mascolo,%20C.%20et%20al.%20The%20complete%20mitochondrial%20genome%20of%20the%20axillary%20seabream,%20Pagellus%20acarne%20(Perciformes:%20Sparidae).%20Mitochondrial%20DNA%20Part.%20B.%203,%20434%E2%80%93435%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Ceruso, M. et al. The complete mitochondrial genome of the common Pandora <em>Pagellus erythrinus</em> (Perciformes: Sparidae). <em>Mitochondrial DNA Part. B</em>. <strong>3</strong>, 624–625 (2018).
</cite> [<a href="https://doi.org/10.1080/23802359.2018.1467235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7800150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33474264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ceruso,%20M.%20et%20al.%20The%20complete%20mitochondrial%20genome%20of%20the%20common%20Pandora%20Pagellus%20erythrinus%20(Perciformes:%20Sparidae).%20Mitochondrial%20DNA%20Part.%20B.%203,%20624%E2%80%93625%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Kundu, S. et al. Mitogenomic architecture of Atlantic emperor <em>Lethrinus atlanticus</em> (Actinopterygii: Spariformes): Insights into the lineage diversification in Atlantic Ocean. <em>Int. J. Mol. Sci.</em><strong>25</strong>, 10700 (2024).
</cite> [<a href="https://doi.org/10.3390/ijms251910700" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11476654/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39409028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kundu,%20S.%20et%20al.%20Mitogenomic%20architecture%20of%20Atlantic%20emperor%20Lethrinus%20atlanticus%20(Actinopterygii:%20Spariformes):%20Insights%20into%20the%20lineage%20diversification%20in%20Atlantic%20Ocean.%20Int.%20J.%20Mol.%20Sci.25,%2010700%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Mascolo, C. et al. The complete mitochondrial genome of the Angolan dentex <em>Dentex angolensis</em> (Perciformes: Sparidae). <em>Mitochondrial DNA Part. B</em>. <strong>4</strong>, 1245–1246 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mascolo,%20C.%20et%20al.%20The%20complete%20mitochondrial%20genome%20of%20the%20Angolan%20dentex%20Dentex%20angolensis%20(Perciformes:%20Sparidae).%20Mitochondrial%20DNA%20Part.%20B.%204,%201245%E2%80%931246%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Naylor, G. J. P., Collins, T. &amp; Brown, W. M. Hydrophobicity and phylogeny. <em>Nature</em><strong>373</strong>, 555–556 (1995).
</cite> [<a href="https://doi.org/10.1038/373565b0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7854410/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Naylor,%20G.%20J.%20P.,%20Collins,%20T.%20&amp;%20Brown,%20W.%20M.%20Hydrophobicity%20and%20phylogeny.%20Nature373,%20555%E2%80%93556%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Gong, L. et al. Novel gene rearrangement pattern in <em>Cynoglossus melampetalus</em> mitochondrial genome: new gene order in genus <em>Cynoglossus</em> (Pleuronectiformes: Cynoglossidae). <em>Int. J. Biol. Macromol.</em><strong>149</strong>, 1232–1240 (2020).
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2020.02.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32032708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gong,%20L.%20et%20al.%20Novel%20gene%20rearrangement%20pattern%20in%20Cynoglossus%20melampetalus%20mitochondrial%20genome:%20new%20gene%20order%20in%20genus%20Cynoglossus%20(Pleuronectiformes:%20Cynoglossidae).%20Int.%20J.%20Biol.%20Macromol.149,%201232%E2%80%931240%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Montaña-Lozano, P. et al. Comparative genomic analysis of vertebrate mitochondrial genomes reveals a differential rate of rearrangements between taxonomic classes. <em>Sci. Rep.</em><strong>12</strong>, 5479 (2022).
</cite> [<a href="https://doi.org/10.1038/s41598-022-09512-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8971445/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35361853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Monta%C3%B1a-Lozano,%20P.%20et%20al.%20Comparative%20genomic%20analysis%20of%20vertebrate%20mitochondrial%20genomes%20reveals%20a%20differential%20rate%20of%20rearrangements%20between%20taxonomic%20classes.%20Sci.%20Rep.12,%205479%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Wang, C. et al. Comparative analysis of four complete mitochondrial genomes of epinephelidae (Perciformes). <em>Genes</em><strong>13</strong>, 660 (2022).
</cite> [<a href="https://doi.org/10.3390/genes13040660" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9029768/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35456466/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20C.%20et%20al.%20Comparative%20analysis%20of%20four%20complete%20mitochondrial%20genomes%20of%20epinephelidae%20(Perciformes).%20Genes13,%20660%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Berthelot, C. Adaptation of proteins to the cold in Antarctic fish: A role for methionine? <em>Genome Biol. Evol.</em><strong>11</strong>, 220–231 (2019).
</cite> [<a href="https://doi.org/10.1093/gbe/evy262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6336007/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30496401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berthelot,%20C.%20Adaptation%20of%20proteins%20to%20the%20cold%20in%20Antarctic%20fish:%20A%20role%20for%20methionine?%20Genome%20Biol.%20Evol.11,%20220%E2%80%93231%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Sebastian, W. et al. Signals of selection in the mitogenome provide insights into adaptation mechanisms in heterogeneous habitats in a widely distributed pelagic fish. <em>Sci. Rep.</em><strong>10</strong>, 9081 (2020).
</cite> [<a href="https://doi.org/10.1038/s41598-020-65905-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7270097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32493917/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sebastian,%20W.%20et%20al.%20Signals%20of%20selection%20in%20the%20mitogenome%20provide%20insights%20into%20adaptation%20mechanisms%20in%20heterogeneous%20habitats%20in%20a%20widely%20distributed%20pelagic%20fish.%20Sci.%20Rep.10,%209081%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Ji, Y. T. et al. Adaptive evolution characteristics of mitochondrial genomes in genus Aparapotamon (Brachyura, Potamidae) of freshwater crabs. <em>BMC Genom.</em><strong>24</strong>, 193 (2023).</cite> [<a href="https://doi.org/10.1186/s12864-023-09290-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10091551/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37041498/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ji,%20Y.%20T.%20et%20al.%20Adaptive%20evolution%20characteristics%20of%20mitochondrial%20genomes%20in%20genus%20Aparapotamon%20(Brachyura,%20Potamidae)%20of%20freshwater%20crabs.%20BMC%20Genom.24,%20193%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Parvathy, S. T., Udayasuriyan, V. &amp; Bhadana, V. Codon usage bias. <em>Mol. Biol. Rep.</em><strong>49</strong>, 539–565 (2022).
</cite> [<a href="https://doi.org/10.1007/s11033-021-06749-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8613526/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34822069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Parvathy,%20S.%20T.,%20Udayasuriyan,%20V.%20&amp;%20Bhadana,%20V.%20Codon%20usage%20bias.%20Mol.%20Biol.%20Rep.49,%20539%E2%80%93565%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Zhao, D., Guo, Y. &amp; Gao, Y. Natural selection drives the evolution of mitogenomes in Acrossocheilus. <em>PLoS ONE</em>. <strong>17</strong>, e0276056 (2022).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0276056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9560497/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36227932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20D.,%20Guo,%20Y.%20&amp;%20Gao,%20Y.%20Natural%20selection%20drives%20the%20evolution%20of%20mitogenomes%20in%20Acrossocheilus.%20PLoS%20ONE.%2017,%20e0276056%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Nei, M. &amp; Kumar, S. <em>Molecular Evolution and Phylogenetics</em> (Oxford University Press, 2000).</cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Ruan, H. et al. Comparative analysis of complete mitochondrial genomes of three Gerres fishes (Perciformes: Gerreidae) and primary exploration of their evolution history. <em>Int. J. Mol. Sci.</em><strong>21</strong>, 1874 (2020).
</cite> [<a href="https://doi.org/10.3390/ijms21051874" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7084342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32182936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ruan,%20H.%20et%20al.%20Comparative%20analysis%20of%20complete%20mitochondrial%20genomes%20of%20three%20Gerres%20fishes%20(Perciformes:%20Gerreidae)%20and%20primary%20exploration%20of%20their%20evolution%20history.%20Int.%20J.%20Mol.%20Sci.21,%201874%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Ewusi, E. O. M. et al. Endemic radiation of African moonfish, <em>Selene dorsalis</em> (Gill 1863), in the Eastern atlantic: mitogenomic characterization and phylogenetic implications of carangids (Teleostei: Carangiformes). <em>Biomolecules</em><strong>14</strong>, 1208 (2024).
</cite> [<a href="https://doi.org/10.3390/biom14101208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11506752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39456141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ewusi,%20E.%20O.%20M.%20et%20al.%20Endemic%20radiation%20of%20African%20moonfish,%20Selene%20dorsalis%20(Gill%201863),%20in%20the%20Eastern%20atlantic:%20mitogenomic%20characterization%20and%20phylogenetic%20implications%20of%20carangids%20(Teleostei:%20Carangiformes).%20Biomolecules14,%201208%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Wujdi, A. et al. Elucidating the mitogenomic blueprint of <em>Pomadasys Perotaei</em> from the Eastern atlantic: characterization and matrilineal phylogenetic insights into haemulid grunts (Teleostei: Lutjaniformes). <em>Biochem. Genet.</em>10.1007/s10528-024-10941-z (2024).
</cite> [<a href="https://doi.org/10.1007/s10528-024-10941-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39453547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wujdi,%20A.%20et%20al.%20Elucidating%20the%20mitogenomic%20blueprint%20of%20Pomadasys%20Perotaei%20from%20the%20Eastern%20atlantic:%20characterization%20and%20matrilineal%20phylogenetic%20insights%20into%20haemulid%20grunts%20(Teleostei:%20Lutjaniformes).%20Biochem.%20Genet.10.1007/s10528-024-10941-z%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Satoh, T. P., Miya, M., Mabuchi, K. &amp; Nishida, M. Structure and variation of the mitochondrial genome of fishes. <em>BMC Genom.</em><strong>17</strong>, 719 (2016).</cite> [<a href="https://doi.org/10.1186/s12864-016-3054-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5015259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27604148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Satoh,%20T.%20P.,%20Miya,%20M.,%20Mabuchi,%20K.%20&amp;%20Nishida,%20M.%20Structure%20and%20variation%20of%20the%20mitochondrial%20genome%20of%20fishes.%20BMC%20Genom.17,%20719%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Yoon, T-H. et al. Mitogenomic analysis reveals the phylogenetic placement of monotypic <em>Parachelon grandisquamis</em> and distinctive structural features of control regions in mullets. <em>Front. Mar. Sci.</em><strong>11</strong>, 1484198 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yoon,%20T-H.%20et%20al.%20Mitogenomic%20analysis%20reveals%20the%20phylogenetic%20placement%20of%20monotypic%20Parachelon%20grandisquamis%20and%20distinctive%20structural%20features%20of%20control%20regions%20in%20mullets.%20Front.%20Mar.%20Sci.11,%201484198%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Santini, F., Carnevale, G. &amp; Sorenson, L. First multi-locus timetree of seabreams and porgies (Percomorpha: Sparidae). <em>Ital. J. Zool.</em><strong>81</strong>, 55–71 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?Santini,%20F.,%20Carnevale,%20G.%20&amp;%20Sorenson,%20L.%20First%20multi-locus%20timetree%20of%20seabreams%20and%20porgies%20(Percomorpha:%20Sparidae).%20Ital.%20J.%20Zool.81,%2055%E2%80%9371%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Chiba, S. N., Iwatsuki, Y., Yoshino, T. &amp; Hanzawa, N. Comprehensive phylogeny of the family Sparidae (Perciformes: Teleostei) inferred from mitochondrial gene analyses. <em>Genes Genet. Syst.</em><strong>84</strong>, 153–170 (2009).
</cite> [<a href="https://doi.org/10.1266/ggs.84.153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19556709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chiba,%20S.%20N.,%20Iwatsuki,%20Y.,%20Yoshino,%20T.%20&amp;%20Hanzawa,%20N.%20Comprehensive%20phylogeny%20of%20the%20family%20Sparidae%20(Perciformes:%20Teleostei)%20inferred%20from%20mitochondrial%20gene%20analyses.%20Genes%20Genet.%20Syst.84,%20153%E2%80%93170%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Hanel, R. &amp; Sturmbauer, C. Multiple recurrent evolution of trophic types in Northeastern Atlantic mediterranean seabrenms (Sparidae, Percoidei). <em>J. Mol. Evol.</em><strong>50</strong>, 276–283 (2000).
</cite> [<a href="https://doi.org/10.1007/s002399910032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10754071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hanel,%20R.%20&amp;%20Sturmbauer,%20C.%20Multiple%20recurrent%20evolution%20of%20trophic%20types%20in%20Northeastern%20Atlantic%20mediterranean%20seabrenms%20(Sparidae,%20Percoidei).%20J.%20Mol.%20Evol.50,%20276%E2%80%93283%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Orrell, T. M., Carpenter, K. E., Musick, J. A. &amp; Graves, J. E. Phylogenetic and biogeographic analysis of the Sparidae (Perciformes: Percoidei) from cytochrome b sequences. <em>Copeia</em><strong>3</strong>, 618–631 (2002).</cite> [<a href="https://scholar.google.com/scholar_lookup?Orrell,%20T.%20M.,%20Carpenter,%20K.%20E.,%20Musick,%20J.%20A.%20&amp;%20Graves,%20J.%20E.%20Phylogenetic%20and%20biogeographic%20analysis%20of%20the%20Sparidae%20(Perciformes:%20Percoidei)%20from%20cytochrome%20b%20sequences.%20Copeia3,%20618%E2%80%93631%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Rabosky, D. L. et al. An inverse latitudinal gradient in speciation rate for marine fishes. <em>Nature</em><strong>559</strong>, 392–395 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0273-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29973726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rabosky,%20D.%20L.%20et%20al.%20An%20inverse%20latitudinal%20gradient%20in%20speciation%20rate%20for%20marine%20fishes.%20Nature559,%20392%E2%80%93395%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Day, J. J. Evolutionary relationships of the Sparidae (Teleostei: Percoidei): integrating fossil and Recent data. <em>Trans. R Soc. Edinb. Earth Sci.</em><strong>93</strong>, 333–353 (2003).</cite> [<a href="https://scholar.google.com/scholar_lookup?Day,%20J.%20J.%20Evolutionary%20relationships%20of%20the%20Sparidae%20(Teleostei:%20Percoidei):%20integrating%20fossil%20and%20Recent%20data.%20Trans.%20R%20Soc.%20Edinb.%20Earth%20Sci.93,%20333%E2%80%93353%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Capella, W. et al. Mediterranean isolation preconditioning the Earth system for late miocene climate cooling. <em>Sci. Rep.</em><strong>9</strong>, 3795 (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-40208-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6405769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30846804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Capella,%20W.%20et%20al.%20Mediterranean%20isolation%20preconditioning%20the%20Earth%20system%20for%20late%20miocene%20climate%20cooling.%20Sci.%20Rep.9,%203795%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Rutterford, L. A., Simpson, S. D., Bogstad, B., Devine, J. A. &amp; Genner, M. J. Sea temperature is the primary driver of recent and predicted fish community structure across Northeast Atlantic shelf seas. <em>Glob Chang. Biol.</em><strong>29</strong>, 2510–2521 (2023).
</cite> [<a href="https://doi.org/10.1111/gcb.16633" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36896634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rutterford,%20L.%20A.,%20Simpson,%20S.%20D.,%20Bogstad,%20B.,%20Devine,%20J.%20A.%20&amp;%20Genner,%20M.%20J.%20Sea%20temperature%20is%20the%20primary%20driver%20of%20recent%20and%20predicted%20fish%20community%20structure%20across%20Northeast%20Atlantic%20shelf%20seas.%20Glob%20Chang.%20Biol.29,%202510%E2%80%932521%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Rose, G. A. On distributional responses of North Atlantic fish to climate change. <em>ICES J. Mar. Sci.</em><strong>62</strong>, 1360–1374 (2005).</cite> [<a href="https://scholar.google.com/scholar_lookup?Rose,%20G.%20A.%20On%20distributional%20responses%20of%20North%20Atlantic%20fish%20to%20climate%20change.%20ICES%20J.%20Mar.%20Sci.62,%201360%E2%80%931374%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Majumder, S. &amp; Schmid, C. A study of the variability in the Benguela current volume transport. <em>Ocean. Sci.</em><strong>14</strong>, 273–283 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Majumder,%20S.%20&amp;%20Schmid,%20C.%20A%20study%20of%20the%20variability%20in%20the%20Benguela%20current%20volume%20transport.%20Ocean.%20Sci.14,%20273%E2%80%93283%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Gordó-Vilaseca, C., Stephenson, F., Coll, M., Lavin, C. &amp; Costello, M. J. Three decades of increasing fish biodiversity across the northeast Atlantic and the Arctic Ocean. Proc. Natl. Acad. Sci. USA. 120, e2120869120
</cite> [<a href="https://doi.org/10.1073/pnas.2120869120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9942854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36656855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gord%C3%B3-Vilaseca,%20C.,%20Stephenson,%20F.,%20Coll,%20M.,%20Lavin,%20C.%20&amp;%20Costello,%20M.%20J.%20Three%20decades%20of%20increasing%20fish%20biodiversity%20across%20the%20northeast%20Atlantic%20and%20the%20Arctic%20Ocean.%20Proc.%20Natl.%20Acad.%20Sci.%20USA.%20120,%20e2120869120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Percie du Sert. The ARRIVE guidelines 2.0: updated guidelines for reporting animal research. <em>PLOS Biol.</em><strong>18</strong>, e3000410 (2020).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.3000410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7360023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32663219/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Percie%20du%20Sert.%20The%20ARRIVE%20guidelines%202.0:%20updated%20guidelines%20for%20reporting%20animal%20research.%20PLOS%20Biol.18,%20e3000410%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Baldwin, C. C., Mounts, J. H., Smith, D. G. &amp; Weigt, L. A. Genetic identification and color descriptions of early life-history stages of Belizean phaeoptyx and astrapogon (Teleostei: Apogonidae), with comments on the identification of adult phaeoptyx. <em>Zootaxa</em><strong>2008</strong>, 1–22 (2009).</cite> [<a href="https://scholar.google.com/scholar_lookup?Baldwin,%20C.%20C.,%20Mounts,%20J.%20H.,%20Smith,%20D.%20G.%20&amp;%20Weigt,%20L.%20A.%20Genetic%20identification%20and%20color%20descriptions%20of%20early%20life-history%20stages%20of%20Belizean%20phaeoptyx%20and%20astrapogon%20(Teleostei:%20Apogonidae),%20with%20comments%20on%20the%20identification%20of%20adult%20phaeoptyx.%20Zootaxa2008,%201%E2%80%9322%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Kumar, S., Stecher, G., Li, M., Knyaz, C. &amp; Tamura, K. MEGA X: molecular evolutionary genetics analysis across computing platforms. <em>Mol. Biol. Evol.</em><strong>35</strong>, 1547–1549 (2018).
</cite> [<a href="https://doi.org/10.1093/molbev/msy096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5967553/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29722887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20S.,%20Stecher,%20G.,%20Li,%20M.,%20Knyaz,%20C.%20&amp;%20Tamura,%20K.%20MEGA%20X:%20molecular%20evolutionary%20genetics%20analysis%20across%20computing%20platforms.%20Mol.%20Biol.%20Evol.35,%201547%E2%80%931549%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Bernt, M. et al. Improved de Novo metazoan mitochondrial genome annotation. <em>Mol. Phylogenet Evol.</em><strong>69</strong>, 313–319 (2013).
</cite> [<a href="https://doi.org/10.1016/j.ympev.2012.08.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22982435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bernt,%20M.%20et%20al.%20Improved%20de%20Novo%20metazoan%20mitochondrial%20genome%20annotation.%20Mol.%20Phylogenet%20Evol.69,%20313%E2%80%93319%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Iwasaki, W. et al. MitoFish and mitoannotator: a mitochondrial genome database of fish with an accurate and automatic annotation pipeline. <em>Mol. Biol. Evol.</em><strong>30</strong>, 2531–2540 (2013).
</cite> [<a href="https://doi.org/10.1093/molbev/mst141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3808866/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23955518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Iwasaki,%20W.%20et%20al.%20MitoFish%20and%20mitoannotator:%20a%20mitochondrial%20genome%20database%20of%20fish%20with%20an%20accurate%20and%20automatic%20annotation%20pipeline.%20Mol.%20Biol.%20Evol.30,%202531%E2%80%932540%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Perna, N. T. &amp; Kocher, T. D. Patterns of nucleotide composition at fourfold degenerate sites of animal mitochondrial genomes. <em>J. Mol. Evol.</em><strong>41</strong>, 353–359 (1995).
</cite> [<a href="https://doi.org/10.1007/BF00186547" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7563121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perna,%20N.%20T.%20&amp;%20Kocher,%20T.%20D.%20Patterns%20of%20nucleotide%20composition%20at%20fourfold%20degenerate%20sites%20of%20animal%20mitochondrial%20genomes.%20J.%20Mol.%20Evol.41,%20353%E2%80%93359%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Rozas, J. et al. DnaSP 6: DNA sequence polymorphism analysis of large data sets. <em>Mol. Biol. Evol.</em><strong>34</strong>, 3299–3302 (2017).
</cite> [<a href="https://doi.org/10.1093/molbev/msx248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29029172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rozas,%20J.%20et%20al.%20DnaSP%206:%20DNA%20sequence%20polymorphism%20analysis%20of%20large%20data%20sets.%20Mol.%20Biol.%20Evol.34,%203299%E2%80%933302%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Rao, N., Lei, X., Guo, J., Huang, H. &amp; Ren, Z. An efficient sliding window strategy for accurate location of eukaryotic protein coding regions. <em>Comput. Biol. Med.</em><strong>39</strong>, 392–395 (2009).
</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2009.01.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19281973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rao,%20N.,%20Lei,%20X.,%20Guo,%20J.,%20Huang,%20H.%20&amp;%20Ren,%20Z.%20An%20efficient%20sliding%20window%20strategy%20for%20accurate%20location%20of%20eukaryotic%20protein%20coding%20regions.%20Comput.%20Biol.%20Med.39,%20392%E2%80%93395%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Xia, X. DAMBE6: new tools for microbial genomics, phylogenetics, and molecular evolution. <em>J. Hered</em>. <strong>108</strong>, 431–437 (2017).
</cite> [<a href="https://doi.org/10.1093/jhered/esx033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5434544/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28379490/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia,%20X.%20DAMBE6:%20new%20tools%20for%20microbial%20genomics,%20phylogenetics,%20and%20molecular%20evolution.%20J.%20Hered.%20108,%20431%E2%80%93437%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Laslett, D. &amp; Canbäck, B. ARWEN: a program to detect tRNA genes in metazoan mitochondrial nucleotide sequences. <em>Bioinformatics</em><strong>24</strong>, 172–175 (2008).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btm573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18033792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laslett,%20D.%20&amp;%20Canb%C3%A4ck,%20B.%20ARWEN:%20a%20program%20to%20detect%20tRNA%20genes%20in%20metazoan%20mitochondrial%20nucleotide%20sequences.%20Bioinformatics24,%20172%E2%80%93175%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Laslett, D. ARAGORN, a program to detect tRNA genes and TmRNA genes in nucleotide sequences. <em>Nucleic Acids Res.</em><strong>32</strong>, 11–16 (2004).
</cite> [<a href="https://doi.org/10.1093/nar/gkh152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC373265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14704338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laslett,%20D.%20ARAGORN,%20a%20program%20to%20detect%20tRNA%20genes%20and%20TmRNA%20genes%20in%20nucleotide%20sequences.%20Nucleic%20Acids%20Res.32,%2011%E2%80%9316%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Thompson, J. D., Gibson, T. J., Plewniak, F., Jeanmougin, F. &amp; Higgins, D. G. The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. <em>Nucleic Acids Res.</em><strong>25</strong>, 4876–4882 (1997).
</cite> [<a href="https://doi.org/10.1093/nar/25.24.4876" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC147148/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9396791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thompson,%20J.%20D.,%20Gibson,%20T.%20J.,%20Plewniak,%20F.,%20Jeanmougin,%20F.%20&amp;%20Higgins,%20D.%20G.%20The%20CLUSTAL_X%20windows%20interface:%20flexible%20strategies%20for%20multiple%20sequence%20alignment%20aided%20by%20quality%20analysis%20tools.%20Nucleic%20Acids%20Res.25,%204876%E2%80%934882%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Benson, G. Tandem repeats finder: a program to analyze DNA sequences. <em>Nucleic Acids Res.</em><strong>27</strong>, 573–580 (1999).
</cite> [<a href="https://doi.org/10.1093/nar/27.2.573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC148217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9862982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Benson,%20G.%20Tandem%20repeats%20finder:%20a%20program%20to%20analyze%20DNA%20sequences.%20Nucleic%20Acids%20Res.27,%20573%E2%80%93580%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Li, Y., Chen, J., Yan, B. &amp; Meng, X. The complete mitochondrial genome of the Japanese threadfin bream, <em>Nemipterus japonicus</em> (Teleostei, Nemipteridae). <em>Mitochondrial DNA DNA Mapp. Seq. Anal.</em><strong>27</strong>, 429–430 (2016).</cite> [<a href="https://doi.org/10.3109/19401736.2014.898290" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24660929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Y.,%20Chen,%20J.,%20Yan,%20B.%20&amp;%20Meng,%20X.%20The%20complete%20mitochondrial%20genome%20of%20the%20Japanese%20threadfin%20bream,%20Nemipterus%20japonicus%20(Teleostei,%20Nemipteridae).%20Mitochondrial%20DNA%20DNA%20Mapp.%20Seq.%20Anal.27,%20429%E2%80%93430%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Vences, M. et al. iTaxoTools 0.1: kickstarting a specimen-based software toolkit for taxonomists. <em>Megataxa</em><strong>6</strong>, 77–92 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Vences,%20M.%20et%20al.%20iTaxoTools%200.1:%20kickstarting%20a%20specimen-based%20software%20toolkit%20for%20taxonomists.%20Megataxa6,%2077%E2%80%9392%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Lanfear, R., Frandsen, P. B., Wright, A. M., Senfeld, T. &amp; Calcott, B. PartitionFinder 2: new methods for selecting partitioned models of evolution for molecular and morphological phylogenetic analyses. <em>Mol. Biol. Evol.</em><strong>34</strong>, 772–773 (2017).
</cite> [<a href="https://doi.org/10.1093/molbev/msw260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28013191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lanfear,%20R.,%20Frandsen,%20P.%20B.,%20Wright,%20A.%20M.,%20Senfeld,%20T.%20&amp;%20Calcott,%20B.%20PartitionFinder%202:%20new%20methods%20for%20selecting%20partitioned%20models%20of%20evolution%20for%20molecular%20and%20morphological%20phylogenetic%20analyses.%20Mol.%20Biol.%20Evol.34,%20772%E2%80%93773%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Darriba, D., Taboada, G. L., Doallo, R. &amp; Posada, D. JModelTest 2: more models, new heuristics and parallel computing. <em>Nat. Methods</em>. <strong>9</strong>, 772 (2012).
</cite> [<a href="https://doi.org/10.1038/nmeth.2109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4594756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22847109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Darriba,%20D.,%20Taboada,%20G.%20L.,%20Doallo,%20R.%20&amp;%20Posada,%20D.%20JModelTest%202:%20more%20models,%20new%20heuristics%20and%20parallel%20computing.%20Nat.%20Methods.%209,%20772%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Ronquist, F. &amp; Huelsenbeck, J. P. MrBayes 3: bayesian phylogenetic inference under mixed models. <em>Bioinformatics</em><strong>19</strong>, 1572–1574 (2003).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btg180" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12912839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ronquist,%20F.%20&amp;%20Huelsenbeck,%20J.%20P.%20MrBayes%203:%20bayesian%20phylogenetic%20inference%20under%20mixed%20models.%20Bioinformatics19,%201572%E2%80%931574%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Guindon, S. et al. New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0. <em>Syst. Biol.</em><strong>59</strong>, 307–321 (2010).
</cite> [<a href="https://doi.org/10.1093/sysbio/syq010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20525638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guindon,%20S.%20et%20al.%20New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies:%20assessing%20the%20performance%20of%20PhyML%203.0.%20Syst.%20Biol.59,%20307%E2%80%93321%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Letunic, I. &amp; Bork, P. Interactive tree of life (iTOL): an online tool for phylogenetic tree display and annotation. <em>Bioinformatics</em><strong>23</strong>, 127–128 (2007).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btl529" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17050570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Letunic,%20I.%20&amp;%20Bork,%20P.%20Interactive%20tree%20of%20life%20(iTOL):%20an%20online%20tool%20for%20phylogenetic%20tree%20display%20and%20annotation.%20Bioinformatics23,%20127%E2%80%93128%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Mello, B. Estimating timetrees with MEGA and the timetree resource. <em>Mol. Biol. Evol.</em><strong>35</strong>, 2334–2342 (2018).
</cite> [<a href="https://doi.org/10.1093/molbev/msy133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29931306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mello,%20B.%20Estimating%20timetrees%20with%20MEGA%20and%20the%20timetree%20resource.%20Mol.%20Biol.%20Evol.35,%202334%E2%80%932342%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Kumar, S. et al. MEGA12: molecular evolutionary genetics analysis version 12 for adaptive and green computing. <em>Mol. Biol. Evol.</em><strong>41</strong>, 1–9 (2024).</cite> [<a href="https://doi.org/10.1093/molbev/msae263" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11683415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39708372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20S.%20et%20al.%20MEGA12:%20molecular%20evolutionary%20genetics%20analysis%20version%2012%20for%20adaptive%20and%20green%20computing.%20Mol.%20Biol.%20Evol.41,%201%E2%80%939%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Tamura, K., Tao, Q. &amp; Kumar, S. Theoretical foundation of the RelTime method for estimating divergence times from variable evolutionary rates. <em>Mol. Biol. Evol.</em><strong>35</strong>, 1770–1782 (2018).
</cite> [<a href="https://doi.org/10.1093/molbev/msy044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5995221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29893954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tamura,%20K.,%20Tao,%20Q.%20&amp;%20Kumar,%20S.%20Theoretical%20foundation%20of%20the%20RelTime%20method%20for%20estimating%20divergence%20times%20from%20variable%20evolutionary%20rates.%20Mol.%20Biol.%20Evol.35,%201770%E2%80%931782%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Assis, J. et al. Bio-ORACLE v3.0. Pushing marine data layers to the CMIP6 Earth system models of climate change research. <em>Glob Ecol. Biogeogr.</em><strong>33</strong>, e13813 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Assis,%20J.%20et%20al.%20Bio-ORACLE%20v3.0.%20Pushing%20marine%20data%20layers%20to%20the%20CMIP6%20Earth%20system%20models%20of%20climate%20change%20research.%20Glob%20Ecol.%20Biogeogr.33,%20e13813%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365136/bin/41598_2025_16187_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (768.1KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data presented in this study can be found in GenBank (<a href="https://www.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/</a>), <em>COI</em> Accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ242656" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ242656</a> and complete mitogenome Accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ524309" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ524309</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-16187-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_16187.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365136/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365136/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365136%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365136/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365136/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365136/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830652/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365136/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830652/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365136/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365136/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="43LeF1oJJyituUwMUK97Gj7Da2V4ZgFCTUlJbn6YDUA1XfsRkRcj4v6983wtFJ2N">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
