
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Ca2+-pumping by PMCA-neuroplastin complexes operates in the kiloHertz-range - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5338F78AF313D30538F70020B7308B.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368035/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Ca2+-pumping by PMCA-neuroplastin complexes operates in the kiloHertz-range">
<meta name="citation_author" content="Cristina E Constantin">
<meta name="citation_author_institution" content="Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany">
<meta name="citation_author" content="Barbara Schmidt">
<meta name="citation_author_institution" content="Center for Biophysics &amp; Department of Physics, Saarland University, Saarbrücken, Germany">
<meta name="citation_author" content="Yvonne Schwarz">
<meta name="citation_author_institution" content="Institute of Physiology, Center for Integrative Physiology and Molecular Medicine, Saarland, Homburg, Germany">
<meta name="citation_author" content="Harumi Harada">
<meta name="citation_author_institution" content="Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany">
<meta name="citation_author" content="Astrid Kollewe">
<meta name="citation_author_institution" content="Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany">
<meta name="citation_author" content="Catrin S Müller">
<meta name="citation_author_institution" content="Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany">
<meta name="citation_author" content="Sebastian Henrich">
<meta name="citation_author_institution" content="Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany">
<meta name="citation_author" content="Botond Gaal">
<meta name="citation_author_institution" content="Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany">
<meta name="citation_author_institution" content="Department of Anatomy, Histology and Embryology, Faculty of Medicine, University of Debrecen, Debrecen, Hungary">
<meta name="citation_author" content="Akos Kulik">
<meta name="citation_author_institution" content="Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany">
<meta name="citation_author" content="Dieter Bruns">
<meta name="citation_author_institution" content="Institute of Physiology, Center for Integrative Physiology and Molecular Medicine, Saarland, Homburg, Germany">
<meta name="citation_author" content="Uwe Schulte">
<meta name="citation_author_institution" content="Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany">
<meta name="citation_author_institution" content="Department of Anatomy, Histology and Embryology, Faculty of Medicine, University of Debrecen, Debrecen, Hungary">
<meta name="citation_author_institution" content="Logopharm GmbH, March-Buchheim, Germany">
<meta name="citation_author" content="Heiko Rieger">
<meta name="citation_author_institution" content="Center for Biophysics &amp; Department of Physics, Saarland University, Saarbrücken, Germany">
<meta name="citation_author" content="Bernd Fakler">
<meta name="citation_author_institution" content="Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany">
<meta name="citation_author_institution" content="Signalling Research Centres BIOSS and CIBSS, Freiburg, Germany">
<meta name="citation_author_institution" content="Center for Basics in NeuroModulation, Freiburg, Germany">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7550">
<meta name="citation_doi" content="10.1038/s41467-025-62735-5">
<meta name="citation_pmid" content="40835599">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368035/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368035/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368035/pdf/41467_2025_Article_62735.pdf">
<meta name="description" content="Ca2+-ATPases in the plasma membrane extrude Ca2+ ions from the cytosol to the extracellular space thereby terminating Ca2+-signals and controlling Ca2+-homeostasis in any type of cell. Recently, these Ca2+-pumps have been identified as protein ...">
<meta name="og:title" content="Ca2+-pumping by PMCA-neuroplastin complexes operates in the kiloHertz-range">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Ca2+-ATPases in the plasma membrane extrude Ca2+ ions from the cytosol to the extracellular space thereby terminating Ca2+-signals and controlling Ca2+-homeostasis in any type of cell. Recently, these Ca2+-pumps have been identified as protein ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368035/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368035">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62735-5"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62735.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368035%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368035/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368035/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368035/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 20;16:7550. doi: <a href="https://doi.org/10.1038/s41467-025-62735-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62735-5</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Ca<sup>2+</sup>-pumping by PMCA-neuroplastin complexes operates in the kiloHertz-range</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Constantin%20CE%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Cristina E Constantin</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Cristina E Constantin</span></h3>
<div class="p">
<sup>1</sup>Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Constantin%20CE%22%5BAuthor%5D" class="usa-link"><span class="name western">Cristina E Constantin</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schmidt%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Barbara Schmidt</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Barbara Schmidt</span></h3>
<div class="p">
<sup>2</sup>Center for Biophysics &amp; Department of Physics, Saarland University, Saarbrücken, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schmidt%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Barbara Schmidt</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schwarz%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yvonne Schwarz</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yvonne Schwarz</span></h3>
<div class="p">
<sup>3</sup>Institute of Physiology, Center for Integrative Physiology and Molecular Medicine, Saarland, Homburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schwarz%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yvonne Schwarz</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Harada%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Harumi Harada</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Harumi Harada</span></h3>
<div class="p">
<sup>1</sup>Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Harada%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Harumi Harada</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kollewe%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Astrid Kollewe</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Astrid Kollewe</span></h3>
<div class="p">
<sup>1</sup>Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kollewe%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Astrid Kollewe</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22M%C3%BCller%20CS%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Catrin S Müller</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Catrin S Müller</span></h3>
<div class="p">
<sup>1</sup>Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22M%C3%BCller%20CS%22%5BAuthor%5D" class="usa-link"><span class="name western">Catrin S Müller</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Henrich%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Sebastian Henrich</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Sebastian Henrich</span></h3>
<div class="p">
<sup>1</sup>Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Henrich%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sebastian Henrich</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gaal%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Botond Gaal</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Botond Gaal</span></h3>
<div class="p">
<sup>1</sup>Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany </div>
<div class="p">
<sup>4</sup>Department of Anatomy, Histology and Embryology, Faculty of Medicine, University of Debrecen, Debrecen, Hungary </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gaal%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Botond Gaal</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kulik%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Akos Kulik</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Akos Kulik</span></h3>
<div class="p">
<sup>1</sup>Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kulik%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Akos Kulik</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bruns%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Dieter Bruns</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Dieter Bruns</span></h3>
<div class="p">
<sup>3</sup>Institute of Physiology, Center for Integrative Physiology and Molecular Medicine, Saarland, Homburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bruns%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dieter Bruns</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schulte%20U%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Uwe Schulte</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Uwe Schulte</span></h3>
<div class="p">
<sup>1</sup>Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany </div>
<div class="p">
<sup>4</sup>Department of Anatomy, Histology and Embryology, Faculty of Medicine, University of Debrecen, Debrecen, Hungary </div>
<div class="p">
<sup>5</sup>Logopharm GmbH, March-Buchheim, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schulte%20U%22%5BAuthor%5D" class="usa-link"><span class="name western">Uwe Schulte</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rieger%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Heiko Rieger</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Heiko Rieger</span></h3>
<div class="p">
<sup>2</sup>Center for Biophysics &amp; Department of Physics, Saarland University, Saarbrücken, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rieger%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Heiko Rieger</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fakler%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Bernd Fakler</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Bernd Fakler</span></h3>
<div class="p">
<sup>1</sup>Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany </div>
<div class="p">
<sup>6</sup>Signalling Research Centres BIOSS and CIBSS, Freiburg, Germany </div>
<div class="p">
<sup>7</sup>Center for Basics in NeuroModulation, Freiburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fakler%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bernd Fakler</span></a>
</div>
</div>
<sup>1,</sup><sup>6,</sup><sup>7,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Institute of Physiology, Faculty of Medicine, University of Freiburg, Freiburg, Germany </div>
<div id="Aff2">
<sup>2</sup>Center for Biophysics &amp; Department of Physics, Saarland University, Saarbrücken, Germany </div>
<div id="Aff3">
<sup>3</sup>Institute of Physiology, Center for Integrative Physiology and Molecular Medicine, Saarland, Homburg, Germany </div>
<div id="Aff4">
<sup>4</sup>Department of Anatomy, Histology and Embryology, Faculty of Medicine, University of Debrecen, Debrecen, Hungary </div>
<div id="Aff5">
<sup>5</sup>Logopharm GmbH, March-Buchheim, Germany </div>
<div id="Aff6">
<sup>6</sup>Signalling Research Centres BIOSS and CIBSS, Freiburg, Germany </div>
<div id="Aff7">
<sup>7</sup>Center for Basics in NeuroModulation, Freiburg, Germany </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jun 16; Accepted 2025 Jul 29; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368035  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835599/" class="usa-link">40835599</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Ca<sup>2+</sup>-ATPases in the plasma membrane extrude Ca<sup>2+</sup> ions from the cytosol to the extracellular space thereby terminating Ca<sup>2+</sup>-signals and controlling Ca<sup>2+</sup>-homeostasis in any type of cell. Recently, these Ca<sup>2+</sup>-pumps have been identified as protein complexes of the transporting subunits PMCAs1-4 and the single-span membrane proteins Neuroplastin (NPTN) or Basigin that are obligatory for efficient trafficking of the pump complexes to the surface membrane. Quantitative investigation of the pumping velocity controlling the time course of Ca<sup>2+</sup>-signals, however, has remained unresolved. Here we show, using Ca<sup>2+</sup>-activated K<sup>+</sup> channels as fast native reporters of intracellular Ca<sup>2+</sup> concentration(s) together with membrane-tethered fluorescent Ca<sup>2+</sup>-indicators, that under cellular conditions PMCA2-NPTN complexes can clear Ca<sup>2+</sup> in the low millisecond-range. Computational modeling exploiting EM-derived densities of Ca<sup>2+</sup>-source(s) and Ca<sup>2+</sup>-transporters in freeze-fracture replicas translated these fast kinetics into transport rates for individual PMCA2-NPTN pumps of more than 5000 cycles/s. Direct comparison with the Na<sup>+</sup>/Ca<sup>2+</sup>-exchanger NCX2, an alternate-access transporter with established cycling rates in the kHz range, indicated similar efficiencies in Ca<sup>2+</sup>-transport. Our results establish PMCA2-NPTN complexes, the most abundant Ca<sup>2+</sup>-clearing tool in the mammalian brain, as transporters with unanticipated high cycling rates and demonstrate that under cellular conditions ATPases may operate in the kHz-range.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Calcium signalling, Transporters in the nervous system</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Ca2 + -ATPases in the plasma membrane extrude Ca2+ ions from the cytosol to the extracellular space and thus terminate Ca2 + -signals and control Ca2 + -homeostasis. Here, Constantin et al. show that these pumps operate at rates of more than 5000 cycles/s.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">The calcium (Ca<sup>2+</sup>) transporting ATPases of the plasma membrane (PMCAs) are P-type ATPases or pumps that are expressed in virtually any type of cell where they extrude Ca<sup>2+</sup> from the cytosol to the extracellular space using ATP as an energy source<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. This extrusion is crucial for the termination of Ca<sup>2+</sup>-signals, transient increases in intracellular Ca<sup>2+</sup> concentration ([Ca<sup>2+</sup>]<sub>i</sub>) that are used to precisely control a variety of fundamental cellular processes, including vesicle release, enzymatic activity, excitability, contraction, and motility<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>.</p>
<p id="Par4">Molecularly, native PMCAs have recently been identified as heteromeric complexes of the ATPase subunits PMCAs1–4 and the single-span membrane proteins Neuroplastin (NPTN) or Basigin (BASI) that co-assemble with the PMCA subunits in the ER and are required for stability and proper trafficking of the Ca<sup>2+</sup>-pump complexes<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>.</p>
<p id="Par5">In neurons, genetic or pharmacological ablation of the PMCA-mediated transport demonstrated that Ca<sup>2+</sup> extrusion occurs within tens of milliseconds or faster suggesting that individual Ca<sup>2+</sup>-pumps operate with rapid transport rates in the cellular context<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. As yet, however, determination of these transport rates has been hampered by technical limitations precluding accurate assessment of both transport activity and number of active Ca<sup>2+</sup>-pumps involved.</p>
<p id="Par6">Here, we use time-resolved monitoring of Ca<sup>2+</sup>-transport and changes in [Ca<sup>2+</sup>]<sub>i</sub> in intact cells together with EM-based protein counting in the plasma membrane to show that individual PMCA2-NPTN complexes operate at cycle-rates in the kHz-range and thus enable Ca<sup>2+</sup>-signaling with millisecond precision.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Fast and effective Ca<sup>2+</sup>-clearing by PMCA-NPTN pumps</h3>
<p id="Par7">The Ca<sup>2+</sup>-pump activity of PMCAs critically depends on intact cellular boundaries, in particular sufficient supply of ATP and physiological levels of phospholipids (including PIP<sub>2</sub> (phosphatidylinositol-4,5-bisphosphate)) in the plasma membrane<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. In fact, our structural analysis of PMCA2-NPTN complexes by cryo-EM identified a PIP<sub>2</sub> molecule that is inserted into a pocket close to the Ca<sup>2+</sup>-passageway and that is obligatory for the transport function of the pump complex (Fig. <a href="#Fig1" class="usa-link">1a</a><sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>). For investigating the Ca<sup>2+</sup>-transport in PMCA-NPTN complexes we, therefore, turned to heterologous expression in CHO cells, an epithelial line established for analysis of membrane proteins with phospholipid-dependence<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>–<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. To reliably determine transport function in intact cells, we chose to monitor intracellular Ca<sup>2+</sup> ([Ca<sup>2+</sup>]<sub>i</sub>, rather than ATP-consumption or pH-changes resulting from the Ca<sup>2+</sup>−2H<sup>+</sup> countertransport) using co-expressed BK-type Ca<sup>2+</sup>-activated K<sup>+</sup> channels (BK<sub>Ca</sub>, homomeric BKα, see Methods) as native sensors. These channels sense apparent [Ca<sup>2+</sup>]<sub>i</sub> in the physiological range (0.1–10 µM<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>) at the plasma membrane and convert changes in [Ca<sup>2+</sup>]<sub>i</sub> into changes in channel gating (and K<sup>+</sup> currents) with ms-resolution (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a><sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>). Importantly, BK<sub>Ca</sub> channels do not interfere with [Ca<sup>2+</sup>]<sub>i</sub> as they do not effectively act as buffers (due to their low number, in contrast to soluble (fluorescent)-indicators that depend on stoichiometric Ca<sup>2+</sup> binding).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Ca<sup>2+</sup>-clearance by PMCA2-NPTN in the millisecond-range.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368035_41467_2025_62735_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0f80/12368035/b09319e6d29c/41467_2025_62735_Fig1_HTML.jpg" loading="lazy" id="d33e559" height="1011" width="704" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Surface-illustration (space-filling mode) of the PMCA2-NPTN complex with bound PIP<sub>2</sub> as determined in cryo-EM analyses<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. <strong>b</strong> Steady-state activation curves of BK<sub>Ca</sub> channels recorded in whole-cell mode with 10 µM free Ca<sup>2+</sup> in the patch-pipette (insets) from CHO(ΔPMCA2)-cells in the absence (squares and line in black, 8 cells) and presence of heterologously expressed PMCA2-NPTN complexes (squares and line in red, 11 cells). Activation curves recorded with 1 µM and 0.1 µM free Ca<sup>2+</sup> in the patch-pipette were added for calibration. Lines are result of a Boltzmann function fitted to the data (mean ± SEM). Note the large pump-mediated shift of the activation curve indicating a reduction of [Ca<sup>2+</sup>]<sub>i</sub> from pipette-delivered 10 µM (dark grey) to values below 0.1 µM (light grey). <strong>c</strong> Representative BK<sub>Ca</sub>-mediated outward K<sup>+</sup> currents recorded with the indicated voltage-protocol in cultured CHO cells in response to a 0.8 ms Ca<sup>2+</sup> influx through voltage-gated Cav2.2 channels in the presence of either 10 mM EGTA (trace in blue) or 0.1 mM EGTA in the absence (control, black trace) or presence of PMCA2-NPTN complexes (trace in red). Note the rapid current decay with 10 mM EGTA and with PMCA2-NPTN at 0.1 mM EGTA as a cytoplasmic buffer. <strong>d</strong> Current traces (in the framed box in (<strong>c</strong>)) at an expanded time scale, the decay phase approximated with a mono-exponential function with indicated time constant (τ). Currents were scaled to the maximum prior to the Ca<sup>2+</sup> influx pulse. <strong>e</strong> Representative fluorescence traces recorded in 10-12 experiments with identical conditions as in (<strong>c</strong>), but with the membrane-tethered fluorescent Ca<sup>2+</sup>-indicator Lck-GCaMP6s used for monitoring changes in [Ca<sup>2+</sup>]<sub>i</sub> (left insets). Gray line is the result of a mono-exponential fitted to the decay phase of the traces with PMCA2-NPTN and 10 EGTA. Right inset: Fluorescent signals (at the framed box) at an expanded time scale. <strong>f</strong> Representative BK<sub>Ca</sub>-currents recorded in response to 1, 2 and 5 Ca<sup>2+</sup> influx pulses applied at 500 Hz (voltage protocol in inset) to CHO cells as in (<strong>a</strong>). <strong>g</strong> Plot summarizing the time constants of the current decay determined in experiments as in (<strong>f</strong>); squares represent mean ± SEM of the indicated number of cells.</p></figcaption></figure><p id="Par8">Three types of experiments were used to analyze PMCA2-NPTN-mediated Ca<sup>2+</sup>-transport in distinct cellular settings, with the first set investigating extrusion of Ca<sup>2+</sup> ions against a high concentration of 10 µM free Ca<sup>2+</sup> constantly infused into the cell by the de-facto unlimited reservoir of a whole-cell patch pipette (Fig. <a href="#Fig1" class="usa-link">1b</a>, left inset). The BK<sub>Ca</sub> activation curves recorded under these steady-state conditions in the absence (control) and presence of the pump complexes were largely different: PMCA2-NPTN fueled by 2.5 mM ATP (through the patch pipette) induced an extensive right-shift of the activation curve compared to control, placing it slightly below the calibration curve determined with 0.1 µM free Ca<sup>2+</sup> and thus indicating a pump-mediated decrease in [Ca<sup>2+</sup>]<sub>i</sub> at the plasma membrane of more than 2 orders of magnitude (Fig. <a href="#Fig1" class="usa-link">1b</a>, right inset).</p>
<p id="Par9">In a second set of experiments, the pumps were challenged with a 0.8 ms-‘[Ca<sup>2+</sup>]<sub>i</sub>-step’ generated by a voltage-commanded Ca<sup>2+</sup> influx through stably expressed N-type voltage-gated Ca<sup>2+</sup> (Cav2.2) channels (see Methods). In CHO cells perfused with 0.1 mM EGTA, the equivalent of cytoplasmic Ca<sup>2+</sup>-buffering in most CNS neurons<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, such an action potential-like Ca<sup>2+</sup> influx resulted in a lasting increase in [Ca<sup>2+</sup>]<sub>i</sub> reflected by activation of persistent (slowly decaying) BK<sub>Ca</sub>-mediated outward K<sup>+</sup> currents (Fig. <a href="#Fig1" class="usa-link">1c</a>, control trace). In contrast, when EGTA (in the perfusing pipette solution) was increased to 10 mM, BK<sub>Ca</sub>-currents activated by the same brief Ca<sup>2+</sup> influx displayed a rapid decay (Fig. <a href="#Fig1" class="usa-link">1c, d</a>, traces in blue) with time constants (τ <sub>decay</sub>) of ∼8 ms (mean ± SEM: 8.4 ± 0.6 ms, n = 9, Fig. <a href="#Fig1" class="usa-link">1g</a>). This τ <sub>decay</sub> perfectly matched the time constant of channel deactivation (at 50 mV, Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>) indicating complete absorption of the incoming Ca<sup>2+</sup> ions by EGTA-buffering within less than a millisecond<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Surprisingly, Ca<sup>2+</sup>-triggered BK<sub>Ca</sub>-currents with very similar decay kinetics could also be recorded at low intracellular EGTA of 0.1 mM upon co-expression of PMCA2-NPTN complexes (Fig. <a href="#Fig1" class="usa-link">1c, d</a>, traces in red). The respective value for τ <sub>decay</sub> was ∼13 ms (mean ± SEM: 13.2 ± 1.2 ms, n = 10, Fig. <a href="#Fig1" class="usa-link">1g</a>) indicative of pump-mediated removal of Ca<sup>2+</sup> at a speed resembling that by buffering with 10 mM EGTA. These conclusions were further corroborated by measuring the [Ca<sup>2+</sup>]<sub>i</sub> transients in the same step experiments with the membrane-tethered Lck-GCaMP6 (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>, Methods<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>;) replacing BK<sub>Ca</sub> channels as Ca<sup>2+</sup>-indicators: The Ca<sup>2+</sup>-triggered fluorescence signal, after an initial increase (Fig. <a href="#Fig1" class="usa-link">1e</a>, right panel), displayed complete decay with both 10 mM EGTA (in the cytoplasm) or PMCA2-NPTN co-expression (at low intracellular EGTA of 0.1 mM), but persisted and declined slowly under control conditions with 0.1 mM EGTA in the cytoplasm (Fig. <a href="#Fig1" class="usa-link">1e</a>). The respective τ<sub>decay</sub> of ∼740 ms determined with 10 mM EGTA and PMCA2-NPTN co-expression closely matched the dissociation-time measured for Ca<sup>2+</sup> ions from GCaMP6s in stopped-flow fluorimetry with one ms-mixing<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>, supporting the fast Ca<sup>2+</sup>-clearing determined with BK<sub>Ca</sub> channels before (Fig. <a href="#Fig1" class="usa-link">1c, d</a>).</p>
<p id="Par10">In a third set of experiments, Ca<sup>2+</sup> extrusion by either PMCA2-NPTN (at 0.1 mM EGTA) or 10 mM EGTA was further probed in experiments where two or five 0.8 ms-pulses of Ca<sup>2+</sup> influx were elicited at a frequency of 500 Hz. These additional Ca<sup>2+</sup> pulses interrupted by 2 ms non-influx intervals (Fig. <a href="#Fig1" class="usa-link">1f</a>, upper inset) led to an increased [Ca<sup>2+</sup>]<sub>i</sub> as reflected by the concomitant increase of the BK<sub>Ca</sub>-current maxima. The subsequent Ca<sup>2+</sup>-clearing, however, appeared only slightly prolonged in either case (Fig. <a href="#Fig1" class="usa-link">1f</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). The latter is indicated by the respective increase in τ <sub>decay</sub> (from 8.4 to 12.4 ms (10 EGTA), and from 13.2 to 18.5 ms (PMCA2-NPTN); Fig. <a href="#Fig1" class="usa-link">1f, g</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">3b</a>). Interestingly, [Ca<sup>2+</sup>]<sub>i</sub> did not prominently increase with the number of Ca<sup>2+</sup> pulses, pointing to sufficient supply of Ca<sup>2+</sup>-free EGTA and to fast extrusion of Ca<sup>2+</sup> by the pump complexes during the 2-ms non-influx intervals, respectively.</p>
<p id="Par11">Together, the results of all three sets of experiments unequivocally indicated that PMCA2-NPTN complexes can terminate increased [Ca<sup>2+</sup>]<sub>i</sub> in the ms-range as already noticed in some neurons<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, and raised the fundamental question whether this fast Ca<sup>2+</sup> extrusion may be due to fast transport of individual pump complexes or rather result from a high-density expression in the plasma membrane combined with slow pumping activity.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Membrane densities and amounts of PMCA pumps in cells and tissues</h3>
<p id="Par12">In pursuit of this question, we next used immunogold electron-microscopy (immuno-EM) on SDS-digested freeze-fracture replicas (SDS-FRL) of brain neurons and CHO cells, as well as quantitative mass spectrometry (MS) on membrane fractions from various tissues. Labelling with a PMCA-specific antibody (PMCA<sub>pan</sub>, see Methods) at saturating concentration showed that PMCAs1-4 are distinctly distributed over the surface membrane of different brain neurons. Thus, on dendrites of cerebellar Purkinje cells the density of PMCAs amounted to ∼250 µm<sup>2</sup> (equivalent to more than 10% of all intra-membraneous particles), while roughly 50 PMCAs/µm<sup>2</sup> were found in the active zone of parallel fiber varicosities (or pre-synaptic terminals) of the granule cell-to-Purkinje-cell synapses (Fig. <a href="#Fig2" class="usa-link">2a</a>, upper panel). Application of the PMCA<sub>pan</sub> antibody to freeze-fracture replicas from CHO cells expressing PMCA2-NPTN complexes indicated a density of ∼55 pump complexes per µm<sup>2</sup> exceeding the endogenous levels of PMCAs in these epithelial cells by more than ten-fold (Fig. <a href="#Fig2" class="usa-link">2a</a>, lower panel). In the same CHO cells the density of stably expressed Cav2.2 channels stained with a target-specific Cav2.2 antibody (Methods) exhibited a mean value of ∼52/µm<sup>2</sup> (Fig. <a href="#Fig2" class="usa-link">2a</a>, lower panel) thus demonstrating about equal amounts for both, Ca<sup>2+</sup>-source and Ca<sup>2+</sup>-transporter, in our testing system (Fig. <a href="#Fig1" class="usa-link">1</a>). Noteworthy, the majority of the Cav2.2 immuno-particles appeared in clusters (of up to 10 proteins), while the PMCA-particles were more homogenously distributed; preferred co-clustering between both Cav2.2 and PMCA2-NPTN was not observed.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Expression and surface density of PMCA-NPTN complexes in mouse brain and CHO cell testing system.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368035_41467_2025_62735_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0f80/12368035/8d706ae8e6db/41467_2025_62735_Fig2_HTML.jpg" loading="lazy" id="d33e819" height="1004" width="680" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Upper panel: Left, Electron micrographs showing distribution of immunogold particles for Cav2.1 channels (6 nm-gold particles) and PMCA1-4 proteins (12 nm-gold particles) on dendritic shafts of Purkinje cells (den) and in presynaptic varicosities (var) of granule cells in the cerebellum. Dashed line demarks the active zone (az); arrowheads depict clusters of Cav2.1 channels. Scale bars represent 200 nm. Right: Graph denoting the density of PMCAs1-4 in Purkinje cell dendrites and granule cell varicosities (bars are mean ± SEM of the 19 dendrites and 37 varicosities shown). Lower panel: Left, Distribution of Cav2.2 channels (6 nm-gold particles) and PMCA2 protein (12 nm-gold particles) on the plasma membrane of CHO cells that stably express Cav2.2 (α1, β1, α2δ1) and that were transiently transfected with PMCA2 and NPTN. Inset: The framed box on the left at expanded scale. Scale bars represent 200 nm. Right: Graph summarizing surface density for the indicated proteins; values are mean ± SEM of 10 (Cav2.2, PMCA2) and 12 (endo) cells, respectively. Statistical testing between groups of data are indicated by horizontal bars, three stars denote p-values &lt; 0.001, Mann-Whitney U-test (two-sided), n.s. is not significant (exact p-values were 0.0000001 (dendrite vs varicosity), 0.00076 (PMCA-NPTN vs PMCA(endo)). <strong>b</strong> Molecular abundance of the Ca<sup>2+</sup>-transporting proteins PMCAs1-4, NCXs1-3 and SERCAs1-3 determined by quantitative mass spectrometry in membrane fractions from pooled whole mouse brains. Bars denote mean values of the duplicate experiments shown. Amounts determined for alpha subunits 1-3 of the Na/K-ATPase were added for comparison. Note that PMCAs are the most abundant Ca<sup>2+</sup>-transporting system in the brain. n.d. no detected. <strong>c</strong> Molecular abundance of all transporting subunits of the Ca<sup>2+</sup>-pumps in plasma membrane and membrane of the sarco-/endoplasmic reticulum, as well as of the Na<sup>+</sup>/Ca<sup>2+</sup>-exchanger in the distinct tissues/organs indicated (skel. is skeletal). Abundance scaling is logarithmic; all protein amounts were determined in equivalent amounts of membrane(s). Note that PMCA pumps outnumber the exchangers in all tissues except heart and kidney (inset), and that the brain is the only organ where the amounts of PMCAs exceed that of the SERCAs.</p></figcaption></figure><p id="Par13">In addition, we assessed the amounts of Ca<sup>2+</sup>-transport systems in brain and other tissues of adult mice by quantitative MS on integral membrane fractions (<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, Methods). These analyses unraveled PMCAs as the most abundant Ca<sup>2+</sup>-transporters in the brain exceeding both the Na<sup>+</sup>/Ca<sup>2+</sup>-exchangers (NCX) and the Ca<sup>2+</sup>-pumps of the endo/sarcoplasmic reticulum (SERCA) by several-fold and indicated that PMCA2 exhibits highest expression among the four PMCA proteins (Fig. <a href="#Fig2" class="usa-link">2b</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). Conversely, PMCA-pumps are largely outnumbered (more than five-fold) in brain membranes by the Na/K-ATPase, another P-type ATPase in the plasma membrane (Fig. <a href="#Fig2" class="usa-link">2b</a>, right panel). Interestingly, the observed excess of PMCAs over both NCX and SERCA is a unique feature of the brain (forebrain and cerebellum), while in all other tissues analyzed, SERCA is the most abundant Ca<sup>2+</sup>-transport system. In these tissues PMCA dominated over NCX in the surface membrane, except for heart and kidney where the PMCA-to-NCX ratio was slightly less than one (values of 0.75 and 0.95, respectively; Fig. <a href="#Fig2" class="usa-link">2c</a>, inset).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Computational modelling of transport cycles in PMCA pumps</h3>
<p id="Par14">With the precise data acquired on [Ca<sup>2+</sup>]<sub>i</sub> dynamics and BK<sub>Ca</sub>-gating (Fig. <a href="#Fig1" class="usa-link">1</a>), as well as on densities of PMCA2-NPTN complexes and Cav2.2 channels in the plasma membrane of CHO cells (Fig. <a href="#Fig2" class="usa-link">2</a>), we next turned to computational modeling for determining the rates of Ca<sup>2+</sup> -transport in individual pump complexes. For realistic representation of the experimental conditions, the following procedures were applied: (1) BK<sub>Ca</sub>-gating was described by the response of a well-established 10-state model<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> whose transition rates were calibrated by machine-learning procedures using experimental data with defined variations in [Ca<sup>2+</sup>]<sub>i</sub> and transmembrane voltage as an input (Methods, Supplementary Tables <a href="#MOESM1" class="usa-link">1</a>-<a href="#MOESM1" class="usa-link">3</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">5a, b</a><sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>,); (2) spatio-temporal profiles for [Ca<sup>2+</sup>]<sub>i</sub> were determined in a three-dimensional diffusion model (of a spherical cell) without and with Ca<sup>2+</sup> efflux mediated by transport of the (homogenously distributed) PMCA2-NPTN pumps at defined density (50 µm<sup>2</sup>), but variable cycling rates (Fig. <a href="#Fig3" class="usa-link">3a</a>, inset); (3) Ca<sup>2+</sup> influx was either stationary through a single reservoir (pipette) or transient (0.8 ms) through a density-defined number of Cav channels (50 µm<sup>2</sup>) providing an experimentally determined mean number of Ca<sup>2+</sup> ions (equations and parameters used are detailed in Methods and Supplementary Tables <a href="#MOESM1" class="usa-link">1</a>-<a href="#MOESM1" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Determination of transport rates in PMCA2-NPTN complexes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368035_41467_2025_62735_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0f80/12368035/790deacde601/41467_2025_62735_Fig3_HTML.jpg" loading="lazy" id="d33e992" height="1006" width="670" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Computed activation curves of BK<sub>Ca</sub> channels (using the stationary model, Supplementary Table <a href="#MOESM1" class="usa-link">2</a>) determined with 10 µM free Ca<sup>2+</sup> delivered through the patch-pipette in the absence (line in black) and presence of PMCA2-NPTN operating at the indicated rates (colored lines). The computed activation for 0.1 µM [Ca<sup>2+</sup>]<sub>i</sub> (line in gay), and the experimentally determined activation curves for [Ca<sup>2+</sup>]<sub>i</sub> of 10 µM and 0.1 µM (dashed lines) are indicated. Inset: [Ca<sup>2+</sup>]<sub>i</sub> profiles obtained with constant Ca<sup>2+</sup> influx (10 µM) from the pipette and Ca<sup>2+</sup> outflux mediated by PMCA complexes transporting at rates of 50 and 5000 cycles/s; color-coding of [Ca<sup>2+</sup>]<sub>i</sub> as indicated. Note close overlay between computed and experimental data for pipette-set [Ca<sup>2+</sup>]<sub>i</sub> of 10 µM and 0.1 µM (calibration), and of computed curves for [Ca<sup>2+</sup>]<sub>i</sub> of 10 µM with PMCA2-NPTN operating at transport rates of 5−10.000 cycles/s with experimental curves recorded with pipette-set [Ca<sup>2+</sup>]<sub>i</sub> of 0.1 µM. <strong>b</strong> Computed responses of BK<sub>Ca</sub> channels (using the non-stationary model, Supplementary Table <a href="#MOESM1" class="usa-link">3</a>) to pulsed increases in [Ca<sup>2+</sup>]<sub>i</sub> either with PMCA2-NPTN operating at the indicated rates (dashed lines in yellow/orange) or with 10 mM EGTA as a cytoplasmic buffer (dashed lines in blue). BK<sub>Ca</sub>-currents experimentally determined with either 10 mM EGTA or PMCA2-NPTN are indicated as lines in red and blue, respectively. Black trace is the model response to a homogenous increase of [Ca<sup>2+</sup>]<sub>i</sub> from 0.1 to 10 µM in the absence of PMCA pumps.</p></figcaption></figure><p id="Par15">In a first step, we recapitulated the pump-mediated shift in activation curves (Fig. <a href="#Fig1" class="usa-link">1b</a>) by computing the response of the calibrated BK<sub>Ca</sub> model to [Ca<sup>2+</sup>]<sub>i</sub> profiles obtained with constant Ca<sup>2+</sup> influx (10 µM) from the pipette and increasing cycle rates for the PMCA complexes. As illustrated in Fig. <a href="#Fig3" class="usa-link">3a</a>, the resulting BK<sub>Ca</sub> response(s) perfectly matched the experimentally determined calibration curves ([Ca<sup>2+</sup>]<sub>i</sub> of 10 and 0.1 µM in the absence of pumps, dashed lines in Fig. <a href="#Fig3" class="usa-link">3a</a>), and showed that increasing cycle rates of the Ca<sup>2+</sup>-pumps shifted BK<sub>Ca</sub> activation towards positive membrane potentials (Fig. <a href="#Fig3" class="usa-link">3</a>). Interestingly, the experimentally observed shift generated by PMCA2-NPTN required the pumps to operate at rates of ≥5.000/s (Fig. <a href="#Fig3" class="usa-link">3a</a>). Currently assumed pump rates of 50 s<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> failed to produce a visible shift of the activation curve.</p>
<p id="Par16">In a second step, we modelled the BK<sub>Ca</sub> current-transients recorded upon the pulsed increases in [Ca<sup>2+</sup>]<sub>i</sub> in the presence of either high buffer concentrations or PMCA complexes (Fig. <a href="#Fig3" class="usa-link">3b</a>, traces in blue and red, respectively). In these calculations, [Ca<sup>2+</sup>]<sub>i</sub> at the end of the 0.8 ms-influx pulse was 10 µM, a minimal value to account for the observed rapid onset of the Ca<sup>2+</sup>-activated BK<sub>Ca</sub> currents<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, while the resting [Ca<sup>2+</sup>]<sub>i</sub> was set to 0.1 µM. The response determined with 10 mM EGTA as a cytoplasmic Ca<sup>2+</sup>-buffer (with known kinetics<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>) was used for validation of the model’s performance. As illustrated in Fig. <a href="#Fig3" class="usa-link">3b</a>, the experimentally determined current traces were closely approximated by the responses of the BK<sub>Ca</sub> model when [Ca<sup>2+</sup>]<sub>i</sub> was removed either by EGTA (dashed line in blue) or by PMCA2-NPTN operating at a rate of ∼20.000 s (dashed line in orange). Lower transport rates failed to match the experimental traces with respect to both peak-current (time to amplitude maximum) and decay kinetics of the current traces (Fig. <a href="#Fig3" class="usa-link">3b</a>, dashed traces in yellow; Supplementary Fig. <a href="#MOESM1" class="usa-link">5c</a>).</p>
<p id="Par17">Together, our modelling data strongly suggest that native Ca<sup>2+</sup>-pumps transport their substrate with rates in the kHz-range, very similar to what was previously reported for native and heterologously expressed NCX-type Ca<sup>2+</sup>-transport proteins<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Comparison of Ca<sup>2+</sup>-transport by PMCA-NPTN and NCX</h3>
<p id="Par18">We, therefore, thought to probe this prediction by direct comparison between both types of transporters in an identical experimental setting. We recapitulated the experiments in Fig. <a href="#Fig1" class="usa-link">1b, c</a> with NCX2, the most abundant NCX isoform of the rodent brain (Fig. <a href="#Fig2" class="usa-link">2b</a>), replacing PMCA2-NPTN complexes. With NCX2 present in the plasma membrane, the steady-state activation curve of BK<sub>Ca</sub> channels measured with a [Ca<sup>2+</sup>]<sub>pip</sub> of 10 µM was shifted to the right indicating that NCX2 decreased [Ca<sup>2+</sup>]<sub>i</sub> underneath the plasma membrane into the sub-micromolar range similar to the activity of PMCA2-NPTN, albeit somewhat less effective (Fig. <a href="#Fig4" class="usa-link">4a</a>). Subsequent probing of the speed of Ca<sup>2+</sup> extrusion by NCX2 with short Ca<sup>2+</sup> influx pulses as in Fig. <a href="#Fig1" class="usa-link">1c</a> showed that the exchanger promoted a rapid decay of the BK<sub>Ca</sub> -currents with values for τ <sub>decay</sub> of ∼26 ms (mean ± SEM: 26.2 ± 2.3 ms, <em>n</em> = 19). This two-fold slowing of the Ca<sup>2+</sup>-clearing compared to that by PMCA2-NPTN is in line with the about two-fold lower density of the NCX2 protein found by immuno-EM in freeze-fractures of the plasma membrane of the respective CHO cells (Fig. <a href="#Fig4" class="usa-link">4b</a>). And, as a consequence of this slower Ca<sup>2+</sup>-clearing, high-frequency application of several voltage-commanded Ca<sup>2+</sup> influx pulses led to a more prominent increase in the amplitude and the time-to-peak interval of the BK<sub>Ca</sub>-currents, as well as in the time constants of their decay (τ <sub>decay</sub> increased from 26.1 to 48.2 ms) compared to PMCA2-NPTN-mediated Ca<sup>2+</sup> removal in our test system (Fig. <a href="#Fig4" class="usa-link">4c, d</a>). These results provided independent evidence that under cellular conditions PMCA2-NPTN complexes are able to transport Ca<sup>2+</sup> with turnover rates at least in the range of 5000/s, a value previously determined for NCX in giant cell-attached patches from cardiomyocytes<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Comparison of Ca<sup>2+</sup>-clearing by NCX2 and PMCA2-NPTN complexes expressed in CHO cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368035_41467_2025_62735_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0f80/12368035/bc3eaa8260b3/41467_2025_62735_Fig4_HTML.jpg" loading="lazy" id="d33e1230" height="485" width="683" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Activation curve of BK<sub>Ca</sub> channels recorded in whole-cell mode with 10 µM free Ca<sup>2+</sup> in the patch pipette (inset) is distinctly shifted to the right by co-expression of either PMC2-NPTN (red symbols, 12 cells) or NCX2 (green symbols, 14 cells). Activation curves determined in experiments with the indicated [Ca<sup>2+</sup>]<sub>i</sub> in the pipette are given as calibration for nominal intracellular Ca<sup>2+</sup> concentrations (data from<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>). Lines are result of a Boltzmann function fitted to the data (mean ± SEM). Note the large shift of the activation curve indicating a reduction of [Ca<sup>2+</sup>]<sub>i</sub> from pipette-delivered 10 µM (dark grey) to values around 0.1 µM (light grey) by both Ca<sup>2+</sup>-transporters. <strong>b</strong> Electron micrograph showing distribution of immunogold particles for NCX2 transiently expressed in CHO cells and bar graph (mean ± SEM of the 13 cells shown) denoting surface density of the NCX2 protein. Data for PMCA2 from Fig. <a href="#Fig2" class="usa-link">2a</a> were added for comparison. <strong>c</strong> Representative BK<sub>Ca</sub>-currents recorded with 0.1 mM EGTA as a cytoplasmic buffer in response to 1, 2, and 5 Ca<sup>2+</sup> influx pulses as in Fig. <a href="#Fig1" class="usa-link">1f</a> in CHO cells expressing Cav2.2 and NCX2. The current-response to 5 pulses obtained with PMCA2-NPTN from Fig. <a href="#Fig1" class="usa-link">1f</a> (trace in grey) was added for comparison. Inset: BK<sub>Ca</sub>-currents recorded upon a single 0.8 ms Ca<sup>2+</sup> influx pulse with NCX2 or PMCA2-NPTN. Note slightly faster decay of the BK<sub>Ca</sub>-currents in the presence of PMCA2-NPTN complexes. <strong>d</strong> Plot summarizing the time constants of the current decay determined in experiments as in (<strong>c</strong>); squares represent mean ± SEM of the indicated number of cells; data for PMCA2-NPTN were added for comparison.</p></figcaption></figure></section></section><section id="Sec7"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par19">Our work demonstrates that under cellular conditions PMCA2-NPTN complexes are able to remove increased intracellular Ca<sup>2+</sup> with both high efficiency (against large high-concentration reservoirs) and high speed (within ms-windows). These transport characteristics are based on the pumps’ ability to operate in the kHz-range ( &gt; 5000 cycles/s, Fig. <a href="#Fig3" class="usa-link">3</a>) and thus at cycling rates similar to those previously reported for NCX2, an alternate access-type transporter for Ca<sup>2+</sup>. Direct comparison of both transporter types in identical experiments emphasized their similar Ca<sup>2+</sup>-transport capabilities.</p>
<p id="Par20">For detailed investigation of the transport properties of PMCA pumps we set up experimental conditions that guaranteed maximal activity and its precise determination (in individual cells): (i) co-expression of NPTN promoted reliable and robust expression of PMCA pumps in the plasma membrane<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>, (ii) saturating physiological levels for both the phospholipid PIP<sub>2</sub> and ATP were maintained throughout the experiments, (iii) [Ca]<sub>i</sub> and its pump-mediated changes were monitored with BK<sub>Ca</sub> channels, a native Ca<sup>2+</sup>-sensor operating in the physiologically-relevant [Ca<sup>2+</sup>]<sub>i</sub>-range with ms-precision and, importantly, without interfering with [Ca<sup>2+</sup>]<sub>i</sub> (in contrast to soluble fluorescence-based reporter systems acting as additional Ca<sup>2+</sup>-buffers), and (iv) the number of PMCA pumps was determined by immuno-EM in freeze-fracture configuration that enables immediate and selective counting of specifically labelled target proteins in the membrane<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. For labelling with an extended dynamic range as displayed by the PMCA<sub>pan</sub> antibody (about three orders of magnitude, Fig. <a href="#Fig2" class="usa-link">2a</a>), the protein counting obtained by this technique is highly accurate and reliable (as shown for ion channels<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>), in contrast to indirect approaches providing rather rough estimates on protein densities/numbers in membranes<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p>
<p id="Par21">The cycling rates in the kHz-range that were finally determined in computational modelling with the acquired experimental data as input are profoundly different from values of about 50–100/s that were estimated from transport-experiments with erythrocyte ghosts, or lipid vesicles containing endogenous or purified/reconstituted PMCA protein<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. The reasons for these differences are most likely buried in the experimental settings and approaches used including detergent-mediated dissociation of PMCA and NPTN/BASI subunits<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>, erroneous data on transporter densities, and, most importantly, insufficient supply of PIP<sub>2</sub> in the vesicle or ghost preparations. As indicated by structure-function analyses, PIP<sub>2</sub>-binding is obligatory for the transport activity of PMCA pumps<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Conversely, transport at high cycling rates could be well observed in intact erythrocytes: While different in settings and dimensions, the measured transport-associated ATP-consumption is compatible with the kHz-rates derived in our experiments (60 mmoles/hl<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> versus ∼120 mmoles/hl calculated with a turnover rate of 5000/s and an estimated PMCA-density of 6/µm<sup>2</sup>).</p>
<p id="Par22">The finding of PMCA2-NPTN-mediated Ca<sup>2+</sup> extrusion occurring in the ms-range has fundamental implications for both the amplitude and the time course of Ca<sup>2+</sup>-signaling (or increased [Ca<sup>2+</sup>]<sub>i</sub>) in virtually any type of cell. In CNS neurons, where fast pump-mediated Ca<sup>2+</sup>-clearing is well known<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> and where PMCA-NPTN is the most abundant Ca<sup>2+</sup>-clearing device (Fig. <a href="#Fig2" class="usa-link">2b, c</a>), rapid termination of increased [Ca<sup>2+</sup>]<sub>i</sub> and prevention of Ca<sup>2+</sup> accumulation is prerequisite for reliable and timely accurate neurotransmission in both dendrites and synapses<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>–<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. In synapses, PMCA-NPTN complexes are expected to act in concert with cytoplasmic Ca<sup>2+</sup>-buffers to effectively promote synchronous over asynchronous vesicle release even at high frequency transmission (100 to more than 1000 Hz)<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. For this purpose, the increases in [Ca<sup>2+</sup>]<sub>i</sub> fueled by action potential-triggered activation of Cav channels must be rapidly cleared. Explicitly, the about 200 Ca<sup>2+</sup> ions delivered by any Cav2 channel per millisecond must be effectively extruded (to prevent buffer saturation), with PMCA pumps contributing at a rate of 10 Ca<sup>2+</sup> ions/ms based on a cycling rate of 10 kHz (Figs. <a href="#Fig1" class="usa-link">1</a>f, <a href="#Fig3" class="usa-link">3</a>).</p>
<p id="Par23">To what extent the fast Ca<sup>2+</sup> extrusion by PMCA-NPTN participates in other signaling processes in the CNS, as well as in other Ca<sup>2+</sup>-driven second-messenger reactions should be (re)-investigated in detail, best by acute removal of all Ca<sup>2+</sup>-ATPases in the plasma membrane as can be achieved by knock-out/down of their obligatory auxiliary subunits NPTN and BASI<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Moreover, it appears worthwhile to investigate the relation between NCX and PMCA-NPTN, as both transport systems are able to drive fast reduction of activity-induced increases in [Ca<sup>2+</sup>]<sub>i</sub> to resting levels of about 100 nM (Figs. <a href="#Fig1" class="usa-link">1</a>, <a href="#Fig4" class="usa-link">4</a>).</p></section><section id="Sec8"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec9"><h3 class="pmc_sec_title">Ethical Information</h3>
<p id="Par24">Ethical approval was provided by Regierungspräsidium Freiburg (Germany).</p></section><section id="Sec10"><h3 class="pmc_sec_title">Cell culture and transfection</h3>
<p id="Par25">CHO cells (in wildtype form) and CHO cells stably expressing Cav2.2 channels (Cav2.2α1; Cavβ1, α2δ1 subunits) were transiently transfected with cDNAs having the following GenBank (<a href="https://www.ncbi.nlm.nih.gov/genbank" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/genbank</a>) accession numbers: <a href="https://www.ncbi.nlm.nih.gov/protein/Q08460" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q08460</a> (BK<sub>Ca</sub>), <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_001036684.3" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_001036684.3</a> (PMCA2), <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_009145.2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_009145.2</a> (Neuroplastin), <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_078619.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_078619.1</a> (NCX2). The fusion proteins of PMCA2 or NCX2 with green fluorescent protein as well as Neuroplastin with red fluorescence protein were prepared and verified by sequencing. Cells were incubated at 37 °C and 5% CO<sub>2</sub> and measured 2 to 4 days after transfection. Lck-GCamp6s (addgene #52924) was subcloned into the eukaryotic expression vector pN1-CMV, verified by sequencing and used for transfection using the standard lipofectamine 2000 protocol; recordings were done 30–36 h after transfection.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Electrophysiology</h3>
<p id="Par26">Whole-cell patch clamp recordings were performed at room temperature using a HEKA EPC 10 amplifier. Currents were low-pass-filtered at 3–10 kHz and sampled at 20 kHz. Leak currents were subtracted with a P/4 protocol at a holding potential of −90 mV. Serial resistance was 50–70% compensated using the internal compensation circuitry. The standard extracellular solution contained (in mM): KCl 5.8; NaCl 144; MgCl<sub>2</sub> 0.9; CaCl<sub>2</sub> 1.3; NaH<sub>2</sub>PO<sub>4</sub> 0.7; D-Glucose 5.6, and HEPES 10 (pH 7.4). Recording pipettes pulled from quartz glass had resistances of 2–3.5 MΩ when filled with internal solution.</p>
<p id="Par27">For recordings of BK<sub>Ca</sub>-currents in unmodified CHO cells intracellular solution contained (in mM): KCl 139.5, MgCl<sub>2</sub> 3.5, DiBrBAPTA 2, HEPES 5, Na<sub>2</sub>ATP 2.5, Na<sub>3</sub>GTP 0.1 (pH 7.3); CaCl<sub>2</sub> was added to obtain the following free [Ca<sup>2+</sup>]<sub>pip</sub>: 100 nM, 1 μM, 10 μM and 50 μM.</p>
<p id="Par28">WEBMAXC STANDARD [<a href="https://somapp.ucdmc.ucdavis.edu/pharmacology/bers/maxchelator/webmaxc/webmaxcS.htm" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://somapp.ucdmc.ucdavis.edu/pharmacology/ /bers/maxchelator/webmaxc/webmaxcS.htm</a>] was used for calculation of the appropriate amount of calcium to be added to the internal solution; final free Ca<sup>2+</sup> concentrations were checked with a Ca<sup>2+</sup>-sensitive electrode (World Precision Instruments, Sarasota, USA). Steady-state activation of BK<sub>Ca</sub> channels at distinct [Ca<sup>2+</sup>]<sub>i</sub> was determined using test pulses ranging from −80 to +200 mV (in 10 mV increments), followed by a repolarization step to 0 or −50 mV. Conductance–voltage relations were determined from tail current amplitudes measured 0.5 ms after repolarization to the fixed membrane potential and normalized to maximum. Data were fitted with a Boltzmann function g/g<sub>max</sub> = g<sub>max</sub>/(1+exp((V<sub>h</sub>-V<sub>m</sub>)/k)), where V<sub>h</sub> is voltage required for half maximal activation and k is the slope factor.</p>
<p id="Par29">BK<sub>Ca</sub>-currents in CHO cells stably expressing Cav2.2 channels were recorded using internal solution containing (in mM): KCl 127.5; MgCl<sub>2</sub> 3.5; HEPES 5; Na<sub>2</sub>ATP 2.5; Na<sub>3</sub>GTP 0.1 and K<sub>2</sub>EGTA 0.1 oder 10 (pH 7.3). For determination of PMCA-mediated Ca<sup>2+</sup>-clearance via the deactivation time course of BK<sub>Ca</sub> channels a previously established voltage-protocol<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> was used. Briefly, a depolarizing voltage-step (from −90 to 70 mV) activated Cav2.2 (and a minor portion of BK<sub>Ca</sub>) channels without inducing Ca<sup>2+</sup> influx (due to the lack of driving force); Ca<sup>2</sup><sup>+</sup> influx was triggered by a 0.8 ms voltage-step to −60 mV. The resulting increase in [Ca<sup>2+</sup>]<sub>i</sub> activated BK<sub>Ca</sub> channels and thus triggered transient (in the presence of PMCA and NPTN) or long-lasting (control) K<sup>+</sup> currents that were measured at a potential of 70 mV. The current decay was fitted with a mono-exponential function yielding values for the respective time constant (τ <sub>decay</sub>). For the multiple calcium influx protocol (Fig. <a href="#Fig1" class="usa-link">1c</a>, Fig. <a href="#Fig4" class="usa-link">4c</a>) 2 or 5 repolarization pulses to −60 mV at 2 ms intervals (500 Hz) were applied between the pre- and the test-pulse. Data analysis and fitting was done using Igor Pro9 (WaveMetrics).</p>
<p id="Par30">All chemicals except DiBrBAPTA (Alfa Aesar) were purchased from Sigma.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Fluorescence microscopy</h3>
<p id="Par31">CHO cells were recorded in the whole-cell voltage-clamp mode using a EPC10 amplifier (Heka, Germany) under control of Pulse 8.5 (HEKA Electronic). Patch-pipettes had a resistance of 2–4 MΩ when filled with the intracellular solution described above. Cells with an average access resistance of 3–12 MΩ, with 50–75% resistance compensation and &lt;20pA leak-current were used for analysis. Current signals were low-pass filtered at 2.9 kHz (four pole Bessel filter EPC10) and digitized at 50 kHz. Ca<sup>2+</sup> signals were recorded simultaneously. Lck-GCaMP6s fluorescence was acquired with an Evolve EMCCD camera (Visitron, Germany) using a Zeiss Plan Apochromat 40x oil immersion objective (NA 1.3) on a Axiovert200 microscope (Zeiss). Fluorescent images were captured at 100 Hz with custom written macros in VisiView (Visitron, Germany), processed offline using ImageJ 1.43 software and Igor Pro5. Background was deducted by subtracting the F<sub>0</sub> image (average of three pre-stimulus images) from all subsequent images (ΔF<sub>n</sub> = F<sub>n</sub>-F<sub>0</sub>). Regions of interest of identical size (10×10 pixels) were placed over the plasma-membrane reacting to the electrical stimulation and fluorescence changes were tracked throughout the stack.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Electron microscopy</h3>
<section id="Sec14"><h4 class="pmc_sec_title">Sample preparation, CHO cells</h4>
<p id="Par32">For determination of the density of Cav2.2, PMCA2 and NCX2 protein(s) in CHO cells the SDS-FRL technique was used as previously described<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> with some modifications. CHO cells constitutively expressing Cav2.2 were transiently transfected with the aforementioned plasmids (coding for PMCA2-GFP, NPTN-RFP, NCX2-GFP) or an empty vector with Jet-PEI transfection reagent according to the manufacture’s instruction. After 2 days of incubation, cells were rinsed with 25 mM phosphate-buffered saline (PBS) and then fixed with 4% paraformaldehyde (Roth, Germany) in PBS for 10 min. Cells were harvested by scraping, pelleted by centrifugation (500 <em>g</em>, 5 min); cell pellets were sandwiched between copper carriers for high-pressure freezing (HPM100, Leica, Austria). Frozen pellets were then fractured into two parts at −120 °C and the fractured facets were coated with carbon (5 nm), platinum-carbon (2 nm) and an additional layer of carbon (20 nm) in a freeze-fracture replica machine (ACE900, Leica). Replicas were digested at 60 °C in a solution containing 2.5 % SDS and 20% sucrose diluted in 15 mM Tris buffer (pH 8.3) for 48 h followed by 37 °C for 20 h. The replicas were washed in washing buffer comprising 0.05 % bovine serum albumin (BSA, Roth, Germany) and 0.1% Tween20 (Tw20, Roth, Germany) in 50 mM Tris-buffered saline (TBS) and then blocked in a solution containing 5% BSA and 0.1% Tween20 in TBS at room temperature (RT) for 1 h. Subsequently, replicas were incubated at 15 °C two overnights (O/Ns) in the following mixtures of primary antibodies in a solution containing 1% BSA and 0.1% Tween20 made up in TBS: (i) PMCApan (5F10, mouse /Ms/, 1:3000, Invitrogen, Cat. MA3-914) and Cav2.2 (rabbit, 1.67 µg/ml, Synaptic Systems, Cat. 152303) or (ii) NCX2 (SLC8A2, 0.5 µg/ml, Alomone, Cat. ANX-012). Replicas were washed in washing buffer then reacted with 6 nm (Rb) or 12 nm (Ms or Rb) gold particle-conjugated secondary antibodies (1:60, Jackson ImmunoResearch Laboratories, PA) at 15 °C overnight.</p></section><section id="Sec15"><h4 class="pmc_sec_title">Sample preparation, cerebellum</h4>
<p id="Par33">Immunogold labeling of replicas was performed as previously described<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Adult mice (<em>n</em> = 3; Charles River, Germany) were narcotized with isoflurane then anesthetized with pentobarbital (80 mg/kg). Cerebelli were removed and immersed into a fixative containing 2% paraformaldehyde and 15% saturated picric acid in 0.1 M phosphate buffer (PB) for 4 h at 4 °C. Cerebellar slices (100 µm) were cut on a vibrotome (VT 1000, Leica) and cryoprotected with 30% glycerol in 0.1 M PB O/N at 4 °C and then frozen by the high pressure freezing machine. Frozen samples were placed onto double replica tables then fractured at −140 °C and coated by deposition with carbon (5 nm), platinum (2 nm) and carbon (18 nm) in a freeze-fracture replica device (BAF 060 BAL-TEC, Liechtenstein). Replicas were digested at 80 °C for 18 h in a solution containing 2.5 % SDS and 20 % sucrose diluted in 15 mM TB. They were washed in 50 mM TBS containing 0.05% BSA and 0.1% Tween20 and then incubated in a blocking solution. Subsequently, replicas were incubated in the mixture of the following primary antibodies: anti-PMCA1-4 (PMCApan, Mouse, 1:3000, Invitrogen, Cat.# M;A3-914) and anti-Cav2.1 (Guinea pig, Gp, 5 μg/mL<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>,) prepared in 50 mM TBS containing 1% BSA and 0.1% Tween20 at RT. After washing them in TBS they were reacted with a mixture of 5 nm (Gp) and 10 nm (Ms) gold particle-conjugated secondary antibodies (1:50, BioCell Research Laboratories, Cardiff, UK) at 15 °C overnight.</p></section><section id="Sec16"><h4 class="pmc_sec_title">Microscopy and quantification</h4>
<p id="Par34">Finally, replicas were washed in TBS followed by distilled water, mounted on Formvar-coated 100 mesh grids, and analyzed with transmission electron microscopes (CM100, Philips or Zeiss LEO 906 E). All antibodies targeted intracellular epitopes of proteins, immunoreactivity was therefore observed on the protoplasmic face (P-face) of the plasma membrane. The density of immuno-particles labeling protein(s) of interest was calculated by dividing the absolute number of gold particles by the surface area of CHO cells, dendritic shafts of Purkinje cells, and varicosities of parallel fibers. Data are given as individual data points and mean values ± SEM. Statistical significance was assessed by the non-parametric Mann-Whitney U-test.</p></section></section><section id="Sec17"><h3 class="pmc_sec_title">Biochemistry and proteomic analysis</h3>
<section id="Sec18"><h4 class="pmc_sec_title">Quantitative MS analysis on membrane fractions</h4>
<p id="Par35">Tissues from adult mice were freshly collected and snap-frozen before they were subjected to a simple total membrane isolation protocol<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. 1 mg of membrane suspension was then extracted with 1 ml 100 mM Na<sub>2</sub>CO<sub>3</sub> (pH 11) to reduce sample complexity. After ultracentrifugation, the pellets were solubilized with 100 µl ComplexioLyte91 (CL-91, Logopharm GmbH, Germany) and aliquots run on a short SDS-PAGE gel. After silver staining, the gel lanes were excised and subjected to standard tryptic in-gel digestion. Extracted and dried peptides were then dissolved in 0.5% (v/v) trifluoroacetic acid and loaded onto a precolumn (C18 PepMap100 (300 µm i.d. x 5 mm; particle size 5 µm, Thermo Scientific, Germany) for 5 min at 20 µL/min with 0.05% (v/v) trifluoroacetic acid using a split-free UltiMate 3000 RSLCnano HPLC (Thermo Scientific, Germany). Bound peptides were then eluted with an aqueous-organic gradient (eluent A: 0.5% (v/v) acetic acid; eluent B: 0.5% (v/v) acetic acid in 80% (v/v) acetonitrile): 5 min 3% B, 120 min from 3% B to 30% B, 20 min from 30% B to 50% B, 10 min from 50% B to 99% B, 5 min 99% B, 5 min from 99% B to 3% B, 15 min 3% B (flow rate 300 nl/min). Eluted peptides were separated in a PicoTip™ emitter (75 µm i.d.) manually packed 23.5 cm with ReproSil-Pur 120 ODS-3 (C18; particle size 3 µm; Dr. Maisch HPLC, Germany) and electrosprayed (2.3 kV; transfer capillary temperature 300 °C; funnel RF level 45.0) in positive ion mode into a LTQ Orbitrap XL tandem mass spectrometer (Thermo Scientific, Germany) with the instrument settings described in ref. <sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>.</p>
<p id="Par36">For each dataset, a peak list was extracted from fragment ion spectra using the “msconvert.exe” tool (part of ProteoWizard; http:// proteowizard.sourceforge.net/; v3.0.11098; Mascot generic format with filter options “peakPicking true 1-” and “threshold count 500 most-intense”) and the precursor m/z values were shifted by the median m/z offset of all peptides assigned to proteins in a preliminary database search with 50 ppm peptide mass tolerance. Corrected peak lists were searched with Mascot 2.6.2 (Matrix Science, UK) against the UniProtKB/Swiss-Prot database (mouse, rat and human entries). Acetyl (Protein N-term), Carbamidomethyl (C), Gln-&gt;pyro-Glu (N-term Q), Glu-&gt;pyro-Glu (N-term E), Oxidation (M), Phospho (S, T, Y), and Propionamide (C) were chosen as variable modifications, peptide and fragment mass tolerance were set to ±5 ppm and ±20 mmu, respectively. One missed tryptic cleavage was allowed. The expect value cut-off for peptide assignment was set to 0.5. Related identified proteins (subset or species homologs) were grouped using the name of the predominant member. Proteins either representing exogenous contaminations (e.g., keratins, trypsin, IgG chains) or identified by only one specific peptide were not considered.</p>
<p id="Par37">Label-free quantification of proteins was carried out following the principles and procedures described in refs. <sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>–<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Peptide signal intensities (peak volumes, PVs) from FT full scans were determined and offline mass calibrated using MaxQuant v1.6.3.3 (<a href="https://www.maxquant.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.maxquant.org</a>). Peptide PV elution times in evaluated datasets were pairwise aligned using LOESS regression (reference times were dynamically calculated from the median peptide elution times over all aligned datasets). PVs were then assigned to peptides based on their m/z and elution time obtained either directly from MS/MS-based identification or indirectly (i.e., from identifications in parallel datasets) using in-house developed software (matching tolerances of ±2 ppm and ±1 min). The obtained matrix of protein-specific peptide intensities / run was then subjected to consistency-based evaluation and used for weighted fitting to a global reference<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Molecular abundances of proteins (Fig. <a href="#Fig2" class="usa-link">2A</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>) were estimated using the abundance<sub>norm</sub>spec measure<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>.</p></section><section id="Sec19"><h4 class="pmc_sec_title">Cryo-slicing BN-MS</h4>
<p id="Par38">Synaptosome-enriched membranes were prepared from whole mouse brain<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>, solubilized with CL-47 (Logopharm GmbH, Germany; 1 ml/mg membrane protein, protease inhibitors added), and resolved on a preparative scale blue native polyacrylamide gradient (BN-PAGE) gel as described<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. The section of interest was excised from the gel lane, embedded and cryo-sliced (stepsize 0,33 mm) as detailed in ref. <sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. In-gel tryptic digestion and LC-MS/MS analysis were performed as follows: Peptides were dissolved in 0.5% (v/v) trifluoroacetic acid and loaded onto a precolumn (C18 PepMap100 (300 µm i.d. x 5 mm; particle size 5 µm, Thermo Scientific, Germany) for 5 min at 20 µL/min with 0.05% (v/v) trifluoroacetic acid using a split-free UltiMate 3000 RSLCnano HPLC (Thermo Scientific, Germany). Bound peptides were then eluted with an aqueous-organic gradient (eluent A: 0.5% (v/v) acetic acid; eluent B: 0.5% (v/v) acetic acid in 80% (v/v) acetonitrile): 5 min 3% B, 120 min from 3% B to 30% B, 20 min from 30% B to 50% B, 10 min from 50% B to 99% B, 5 min 99% B, 5 min from 99% B to 3% B, 10 min 3% B (flow rate 300 nl/min). Eluted peptides were separated in a SilicaTip™ emitter (75 µm i.d.) manually packed 23.5 cm with ReproSil-Pur 120 ODS-3 (C18; particle size 3 µm; Dr. Maisch HPLC, Germany) and electrosprayed (2.3 kV; transfer capillary temperature 300 °C; funnel RF level 45.0) in positive ion mode into a Q Exactive HF-X mass spectrometer (Thermo Scientific, Germany). Full MS scan range was 370–1700 m/z with a target value of 3*10<sup>6 </sup>ions at a nominal resolution of 240,000. Each precursor scan was followed by up to 15 data-dependent MS/MS fragmentation spectra with a target value of 10<sup>5 </sup>ions (min. 8*1<sup>03</sup> IT<sub>max</sub> = 200 ms), isolation window 1.0 m/z, resolution of 15,000. Isotopes and charge states +1, &gt;+8 were skipped, dynamic exclusion was set to 60 s.</p>
<p id="Par39">LC-MS (PV) data were extracted and assigned to peptides as described above and then further processed following the steps outlined in ref. <sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>: Slice-to-slice sample variations in peptide recovery and ionization efficiency of the LC-MS setup were corrected for by calculating the median PV offset of each slice from its predecessor and subsequent PV rescaling of each dataset by the median of all offset values inside a window of ±20 slices. Resulting PV data for each protein were filtered for outliers and false-positive assignments as identified by a correlation-based consistency check. Protein abundance distribution profiles were calculated from the resulting PV matrix by fitting the data to a globally determined abundance<sub>norm</sub>spec reference for each individual protein. Slice numbers were finally converted to apparent complex molecular weights by linear regression (log10(MW) versus slice number of the respective protein profile peak maximum) of the marker protein complexes (from the mitochondrial oxidative respiratory chain) described by Schägger and Pfeiffer<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>.</p></section></section><section id="Sec20"><h3 class="pmc_sec_title">Computational modelling</h3>
<p id="Par40">BK<sub>Ca</sub>-currents in response to voltage-changes at constant [Ca<sup>2+</sup>]<sub>i</sub> (activation curves under steady-state conditions as in Fig. <a href="#Fig1" class="usa-link">1b</a>), as well as to time-dependent changes in [Ca<sup>2+</sup>]<sub>i</sub> at constant voltage(s) ([Ca<sup>2+</sup>]<sub>i</sub> pulse-experiments as in Fig. <a href="#Fig1" class="usa-link">1c</a>) were modelled in a spherical 3D-reaction-diffusion model with dimensions of an average CHO cell (‘model cell’) equipped with PMCA-NPTN complexes and Cav channels at the experimentally derived densities; 30% of the intracellular volume represents freely diffusible space, while 70% are inaccessible to free diffusion (all values and parameters summarized in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>).</p>
<section id="Sec21"><h4 class="pmc_sec_title">Spatio-temporal profiles for intracellular Ca<sup>2+</sup> concentration</h4>
<p id="Par41">Under <em>steady-state conditions</em> (as in the experiments of Figs. <a href="#Fig1" class="usa-link">1b</a> and <a href="#Fig4" class="usa-link">4a</a>), free Ca<sup>2+</sup> in the diffusion-accessible volume obeys the stationary diffusion equation <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e1734"><msup><mrow><msub><mrow><mi>D</mi></mrow><mrow><mi>c</mi></mrow></msub><mo>∇</mo></mrow><mrow><mn>2</mn></mrow></msup><mi>c</mi><mfenced close=")" open="("><mrow><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced><mo>=</mo><mn>0</mn></math></span> with influx and outflux of Ca<sup>2+</sup> defined as follows (see also Supplementary Table <a href="#MOESM1" class="usa-link">1</a>): Dirichlet boundary conditions at the pipette-cell contact area (∂Ω<sub>pip</sub>) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e1769"><mi>c</mi><mfenced close=")" open="("><mrow><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced><msub><mrow><mo>∣</mo></mrow><mrow><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover><mi>ϵ</mi><mi>∂</mi><mi mathvariant="normal">Ω</mi><mi mathvariant="normal">pip</mi></mrow></msub><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mi>p</mi><mi>i</mi><mi>p</mi></mrow></msub></math></span>, and von Neumann boundary conditions at the cell surface (∂Ω<sub>PMCA</sub>) outside the pipette-covered area</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e1809" display="block"><msub><mrow><mi>D</mi></mrow><mrow><mi>c</mi></mrow></msub><msub><mrow><mo>∇</mo></mrow><mrow><mi mathvariant="bold-italic">n</mi></mrow></msub><mi>c</mi><mfenced close=")" open="("><mrow><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced><msub><mrow><mo>∣</mo></mrow><mrow><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover><mi>ϵ</mi><mi>∂</mi><mi mathvariant="normal">Ω</mi><mi mathvariant="normal">pmca</mi></mrow></msub><mo>=</mo><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">j</mi></mrow><mo>→</mo></mover></mrow><mrow><mi>P</mi><mi>M</mi><mi>C</mi><mi>A</mi></mrow></msub><mrow><mo>(</mo><mrow><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow><mo>)</mo></mrow><mo>+</mo><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">j</mi></mrow><mo>→</mo></mover></mrow><mrow><mi>l</mi><mi>e</mi><mi>a</mi><mi>k</mi></mrow></msub></math></td>
<td class="label">1</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e1884"><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">j</mi></mrow><mo>→</mo></mover></mrow><mrow><mi>P</mi><mi>M</mi><mi>C</mi><mi>A</mi></mrow></msub></math></span> is the Ca<sup>2+</sup> current by the PMCA2-NPTN-mediated transport given as</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e1907" display="block"><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">j</mi></mrow><mo>→</mo></mover></mrow><mrow><mi>P</mi><mi>M</mi><mi>C</mi><mi>A</mi></mrow></msub><mfenced close=")" open="("><mrow><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced><mo>=</mo><msub><mrow><mi>ϕ</mi></mrow><mrow><mi>P</mi><mi>M</mi><mi>C</mi><mi>A</mi></mrow></msub><msub><mrow><mi>ρ</mi></mrow><mrow><mi>P</mi><mi>M</mi><mi>C</mi><mi>A</mi></mrow></msub><mi>f</mi><mrow><mo>(</mo><mrow><mi>c</mi><mfenced close=")" open="("><mrow><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced></mrow><mo>)</mo></mrow></math></td>
<td class="label">2</td>
</tr></table>
<p>and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e1962"><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">j</mi></mrow><mo>→</mo></mover></mrow><mrow><mi>l</mi><mi>e</mi><mi>a</mi><mi>k</mi></mrow></msub></math></span> is a constant Ca<sup>2+</sup> leak current. In Eq. <a href="#Equ2" class="usa-link">2</a>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e1988"><msub><mrow><mi>ϕ</mi></mrow><mrow><mi>P</mi><mi>M</mi><mi>C</mi><mi>A</mi></mrow></msub></math></span> denotes the cycle rate of the PMCA pumps, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e2004"><msub><mrow><mi>ρ</mi></mrow><mrow><mi>P</mi><mi>M</mi><mi>C</mi><mi>A</mi></mrow></msub></math></span> is their experimentally determined density of 50/µm<sup>2</sup> (Fig. <a href="#Fig2" class="usa-link">2</a>) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e2025"><mi>f</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></math></span> describes the concentration dependence of PMCA transport-activity; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e2039"><mi>f</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></math></span> is a Hill-function of the local Ca<sup>2+</sup> concentration <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e2056"><mi>c</mi></math></span> at the membrane with values for c<sub>1/2</sub> and Hill-coefficient of 0.43 µM and 2, respectively<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Setting <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e2071"><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">j</mi></mrow><mo>→</mo></mover></mrow><mrow><mi>l</mi><mi>e</mi><mi>a</mi><mi>k</mi></mrow></msub><mo>=</mo><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">j</mi></mrow><mo>→</mo></mover></mrow><mrow><mi>P</mi><mi>M</mi><mi>C</mi><mi>A</mi></mrow></msub><mrow><mo>(</mo><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow></math></span> fixed the net flux of Ca<sup>2+</sup> to zero at the resting Ca<sup>2+</sup> concentration <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e2118"><msub><mrow><mi>c</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> of 0.1 µM. For determination of spatial [Ca<sup>2+</sup>]<sub>i</sub> profiles in the diffusion-accessible intracellular volume under steady-state conditions with [Ca<sup>2+</sup>]<sub>pipette</sub> of 10 µM and variable values for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e2140"><msub><mrow><mi>ϕ</mi></mrow><mrow><mi>P</mi><mi>M</mi><mi>C</mi><mi>A</mi></mrow></msub></math></span>, the stationary diffusion equation with boundary conditions (Eqs. <a href="#Equ1" class="usa-link">1</a>, <a href="#Equ2" class="usa-link">2</a>) was numerically solved using FreeFEM + +<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> (Fig. <a href="#Fig3" class="usa-link">3a</a>).</p>
<p id="Par42">Under the <em>dynamic conditions</em> of the ‘pulsed Ca<sup>2+</sup>-experiments’ (Figs. <a href="#Fig1" class="usa-link">1c, f</a>, <a href="#Fig4" class="usa-link">4c</a>), the time-course of [Ca<sup>2+</sup>]<sub>i</sub> after the 0.8 ms-influx pulse is modeled by reaction-diffusion equations for Ca<sup>2+</sup> and buffer (EGTA) concentration, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e2189"><mi>c</mi><mfenced close=")" open="("><mrow><mi>t</mi><mo>,</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e2206"><mi>b</mi><mfenced close=")" open="("><mrow><mi>t</mi><mo>,</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced></math></span>, respectively:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e2224" display="block"><mfrac><mrow><mi>∂</mi><mi>c</mi><mfenced close=")" open="("><mrow><mi>t</mi><mo>,</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced></mrow><mrow><mi>∂</mi><mi>t</mi></mrow></mfrac><mo>=</mo><msub><mrow><mi>D</mi></mrow><mrow><mi>c</mi></mrow></msub><msup><mrow><mo>∇</mo></mrow><mrow><mn>2</mn></mrow></msup><mi>c</mi><mfenced close=")" open="("><mrow><mi>t</mi><mo>,</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced><mo>−</mo><msub><mrow><mi>k</mi></mrow><mrow><mo>+</mo></mrow></msub><mi>c</mi><mfenced close=")" open="("><mrow><mi>t</mi><mo>,</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced><mi>b</mi><mfenced close=")" open="("><mrow><mi>t</mi><mo>,</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced><mo>+</mo><msub><mrow><mi>k</mi></mrow><mrow><mo>−</mo></mrow></msub><mfenced close=")" open="("><mrow><msub><mrow><mi mathvariant="normal">b</mi></mrow><mrow><mi>t</mi><mi>o</mi><mi>t</mi></mrow></msub><mo>−</mo><mi>b</mi><mfenced close=")" open="("><mrow><mi>t</mi><mo>,</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced></mrow></mfenced></math></td>
<td class="label">3</td>
</tr></table>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e2318" display="block"><mfrac><mrow><mi>∂</mi><mi>b</mi><mfenced close=")" open="("><mrow><mi>t</mi><mo>,</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced></mrow><mrow><mi>∂</mi><mi>t</mi></mrow></mfrac><mo>=</mo><msub><mrow><mi>D</mi></mrow><mrow><mi>b</mi></mrow></msub><msup><mrow><mo>∇</mo></mrow><mrow><mn>2</mn></mrow></msup><mi>b</mi><mfenced close=")" open="("><mrow><mi>t</mi><mo>,</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced><mo>−</mo><msub><mrow><mi>k</mi></mrow><mrow><mo>+</mo></mrow></msub><mi>c</mi><mfenced close=")" open="("><mrow><mi>t</mi><mo>,</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced><mi>b</mi><mfenced close=")" open="("><mrow><mi>t</mi><mo>,</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced><mo>+</mo><msub><mrow><mi>k</mi></mrow><mrow><mo>−</mo></mrow></msub><mfenced close=")" open="("><mrow><msub><mrow><mi mathvariant="normal">b</mi></mrow><mrow><mi>t</mi><mi>o</mi><mi>t</mi></mrow></msub><mo>−</mo><mi>b</mi><mfenced close=")" open="("><mrow><mi>t</mi><mo>,</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced></mrow></mfenced></math></td>
<td class="label">4</td>
</tr></table>
<p id="Par43"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e2412"><msub><mrow><mi>D</mi></mrow><mrow><mi>c</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e2425"><msub><mrow><mi>D</mi></mrow><mrow><mi>b</mi></mrow></msub></math></span> are the diffusion constants for Ca<sup>2+</sup> and EGTA, respectively, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e2440"><msub><mrow><mi>k</mi></mrow><mrow><mo>+</mo></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e2453"><msub><mrow><mi>k</mi></mrow><mrow><mo>−</mo></mrow></msub></math></span> the binding and unbinding rates of Ca<sup>2+</sup> to EGTA<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, and b<sub>tot</sub> the total EGTA concentration (0 or 10 mM, all parameters in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). Ca<sup>2+</sup> influx and outflux are described by von Neumann boundary conditions</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><math id="d33e2481" display="block"><msub><mrow><mi>D</mi></mrow><mrow><mi>c</mi></mrow></msub><msub><mrow><mo>∇</mo></mrow><mrow><mi mathvariant="bold-italic">n</mi></mrow></msub><mi>c</mi><mfenced close=")" open="("><mrow><mi>t</mi><mo>,</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced><msub><mrow><mo>∣</mo></mrow><mrow><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover><mi>ϵ</mi><mi>∂</mi><mi mathvariant="normal">Ω</mi><mi mathvariant="normal">pmca</mi></mrow></msub><mo>=</mo><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">j</mi></mrow><mo>→</mo></mover></mrow><mrow><mi>P</mi><mi>M</mi><mi>C</mi><mi>A</mi></mrow></msub><mo>+</mo><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">j</mi></mrow><mo>→</mo></mover></mrow><mrow><mi>l</mi><mi>e</mi><mi>a</mi><mi>k</mi></mrow></msub><mo>+</mo><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">j</mi></mrow><mo>→</mo></mover></mrow><mrow><mi>C</mi><mi>a</mi><mi>V</mi></mrow></msub></math></td>
<td class="label">5</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e2561"><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">j</mi></mrow><mo>→</mo></mover></mrow><mrow><mi>C</mi><mi>a</mi><mi>V</mi></mrow></msub></math></span> is Cav2-mediated Ca<sup>2+</sup> current (during the 0.8 voltage-pulse), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e2583"><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">j</mi></mrow><mo>→</mo></mover></mrow><mrow><mi>P</mi><mi>M</mi><mi>C</mi><mi>A</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e2603"><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">j</mi></mrow><mo>→</mo></mover></mrow><mrow><mi>l</mi><mi>e</mi><mi>a</mi><mi>k</mi></mrow></msub></math></span> are as in Eqs. <a href="#Equ1" class="usa-link">1</a>, <a href="#Equ2" class="usa-link">2</a>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e2629"><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold-italic">j</mi></mrow><mo>→</mo></mover></mrow><mrow><mi>C</mi><mi>a</mi><mi>V</mi></mrow></msub></math></span> is experimentally determined average number of Ca<sup>2+</sup> ions entering the cell during the 0.8 ms-pulse (1.2 ∙ 10<sup>6</sup> / 0.8 ms) divided by the cell surface (300 µm<sup>2</sup>), and zero otherwise. The accessible intracellular volume (30% of total) resulted in a [Ca<sup>2+</sup>]<sub>i</sub> at the end of the 0.8 ms-influx pulse of around 10 µM, a minimal value to account for the observed rapid onset of the Ca<sup>2+</sup>-activated BK<sub>Ca</sub> currents<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>.</p>
<p id="Par44">For determination of the spatio-temporal evolution of [Ca<sup>2+</sup>]<sub>i</sub> following the 0.8 ms-influx pulse with or without buffer, with variable values for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e2674"><msub><mrow><mi>ϕ</mi></mrow><mrow><mi>P</mi><mi>M</mi><mi>C</mi><mi>A</mi></mrow></msub></math></span>, the above formulated reaction diffusion problem (Eqs. <a href="#Equ3" class="usa-link">3</a>–<a href="#Equ5" class="usa-link">5</a>) was numerically solved using FreeFEM + +.</p></section><section id="Sec22"><h4 class="pmc_sec_title">Gating model of BK<sub>Ca</sub> channels</h4>
<p id="Par45">For theoretical approximation of measured BK<sub>Ca</sub>-currents (Figs. <a href="#Fig1" class="usa-link">1</a>, <a href="#Fig4" class="usa-link">4</a>) we used a previously established model for BK<sub>Ca</sub> channel gating<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Briefly, this model comprises a total of 10 distinct states, five open (O <sub>0-4</sub>) and five closed (C <sub>0-4</sub>) states, and four binding sites for Ca<sup>2+</sup> ions (occupancy designated 0-4). Transitions between states O<sub>i</sub> and C<sub>i</sub> (i = 0,…, 4) are voltage-dependent, while transitions between states O<sub>i</sub> and O<sub>i+1</sub> and C<sub>i</sub> and C<sub>i+1</sub> (i = 0,…, 3) are dependent on [Ca<sup>2+</sup>]<sub>i</sub> (Supplementary Tables <a href="#MOESM1" class="usa-link">2</a>, <a href="#MOESM1" class="usa-link">3</a>).</p>
<p id="Par46">For steady-state conditions, the open probability of the model (P<sub>open</sub>) is given by</p>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><math id="d33e2753" display="block"><msub><mrow><mi>P</mi></mrow><mrow><mi>o</mi><mi>p</mi><mi>e</mi><mi>n</mi></mrow></msub><mo>=</mo><msup><mrow><mfenced close=")" open="("><mrow><mn>1</mn><mo>+</mo><mi>B</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>⋅</mo><mi>L</mi><mrow><mo>(</mo><mrow><mn>0</mn></mrow><mo>)</mo></mrow><mo>⋅</mo><msup><mrow><mi>e</mi></mrow><mrow><mo>−</mo><mfrac><mrow><mi>Q</mi><mi>F</mi><mi>V</mi></mrow><mrow><mi>R</mi><mi>T</mi></mrow></mfrac></mrow></msup></mrow></mfenced></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></math></td>
<td class="label">6</td>
</tr></table>
<p>where V denotes membrane voltage, Q is elementary charge, and R, T and F having their usual meaning; L(0) is open to close equilibrium constant at 0 mV with no Ca<sup>2+</sup> bound. The Ca<sup>2+</sup> concentration dependent factor <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e2810"><mi>B</mi><mfenced close=")" open="("><mrow><mi>c</mi></mrow></mfenced></math></span> is given by</p>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><math id="d33e2824" display="block"><mi>B</mi><mfenced close=")" open="("><mrow><mi>c</mi></mrow></mfenced><mo>=</mo><mfenced close=")" open="("><mrow><mfrac><mrow><msup><mrow><mi>c</mi></mrow><mrow><mn>4</mn></mrow></msup></mrow><mrow><msub><mrow><mi>K</mi></mrow><mrow><mi>C</mi><mn>1</mn></mrow></msub><msub><mrow><mi>K</mi></mrow><mrow><mi>C</mi><mn>2</mn></mrow></msub><msub><mrow><mi>K</mi></mrow><mrow><mi>C</mi><mn>3</mn></mrow></msub><msub><mrow><mi>K</mi></mrow><mrow><mi>C</mi><mn>4</mn></mrow></msub></mrow></mfrac><mo>+</mo><mfrac><mrow><msup><mrow><mi>c</mi></mrow><mrow><mn>3</mn></mrow></msup></mrow><mrow><msub><mrow><mi>K</mi></mrow><mrow><mi>C</mi><mn>1</mn></mrow></msub><msub><mrow><mi>K</mi></mrow><mrow><mi>C</mi><mn>2</mn></mrow></msub><msub><mrow><mi>K</mi></mrow><mrow><mi>C</mi><mn>3</mn></mrow></msub></mrow></mfrac><mo>+</mo><mfrac><mrow><msup><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow><mrow><msub><mrow><mi>K</mi></mrow><mrow><mi>C</mi><mn>1</mn></mrow></msub><msub><mrow><mi>K</mi></mrow><mrow><mi>C</mi><mn>2</mn></mrow></msub></mrow></mfrac><mo>+</mo><mfrac><mrow><mi>c</mi></mrow><mrow><msub><mrow><mi>K</mi></mrow><mrow><mi>C</mi><mn>1</mn></mrow></msub></mrow></mfrac><mo>+</mo><mn>1</mn></mrow></mfenced><mo>/</mo><mfenced close=")" open="("><mrow><mfrac><mrow><msup><mrow><mi>c</mi></mrow><mrow><mn>4</mn></mrow></msup></mrow><mrow><msub><mrow><mi>K</mi></mrow><mrow><mi>O</mi><mn>1</mn></mrow></msub><msub><mrow><mi>K</mi></mrow><mrow><mi>O</mi><mn>2</mn></mrow></msub><msub><mrow><mi>K</mi></mrow><mrow><mi>O</mi><mn>3</mn></mrow></msub><msub><mrow><mi>K</mi></mrow><mrow><mi>O</mi><mn>4</mn></mrow></msub></mrow></mfrac><mo>+</mo><mfrac><mrow><msup><mrow><mi>c</mi></mrow><mrow><mn>3</mn></mrow></msup></mrow><mrow><msub><mrow><mi>K</mi></mrow><mrow><mi>O</mi><mn>1</mn></mrow></msub><msub><mrow><mi>K</mi></mrow><mrow><mi>O</mi><mn>2</mn></mrow></msub><msub><mrow><mi>K</mi></mrow><mrow><mi>O</mi><mn>3</mn></mrow></msub></mrow></mfrac><mo>+</mo><mfrac><mrow><msup><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow><mrow><msub><mrow><mi>K</mi></mrow><mrow><mi>O</mi><mn>1</mn></mrow></msub><msub><mrow><mi>K</mi></mrow><mrow><mi>O</mi><mn>2</mn></mrow></msub></mrow></mfrac><mo>+</mo><mfrac><mrow><mi>c</mi></mrow><mrow><msub><mrow><mi>K</mi></mrow><mrow><mi>O</mi><mn>1</mn></mrow></msub></mrow></mfrac><mo>+</mo><mn>1</mn></mrow></mfenced></math></td>
<td class="label">7</td>
</tr></table>
<p id="Par47">K<sub>Ci</sub> and K<sub>Oi</sub> are the dissociation constants for Ca<sup>2+</sup> binding to closed and open states of the BK<sub>Ca</sub> channels, respectively.</p>
<p id="Par48">This gating model was calibrated with experimentally determined BK<sub>Ca</sub>-activation curves (as in Fig. <a href="#Fig1" class="usa-link">1b</a>) for [Ca<sup>2+</sup>]<sub>i</sub> values of 0.1, 1, 10, 50 µM<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup> via an optimization routine<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. The respective values obtained for L(0), Q and the reaction constants K<sub>Ci</sub> and K<sub>Oi</sub> (detailed in Supplementary Table <a href="#MOESM1" class="usa-link">2</a>) were finally used together with the spatial intracellular [Ca<sup>2+</sup>]<sub>i</sub> profiles, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e3069"><mi>c</mi><mfenced close=")" open="("><mrow><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow></mfenced></math></span>, to calculate the open probability <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e3084"><msub><mrow><mi>P</mi></mrow><mrow><mi>o</mi><mi>p</mi><mi>e</mi><mi>n</mi></mrow></msub><mfenced close=")" open="("><mrow><mi>c</mi><mrow><mo>(</mo><mrow><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow><mo>)</mo></mrow></mrow></mfenced></math></span> of the channels distributed over the entire surface of the model cell. Averages of these <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e3111"><msub><mrow><mi>P</mi></mrow><mrow><mi>o</mi><mi>p</mi><mi>e</mi><mi>n</mi></mrow></msub><mfenced close=")" open="("><mrow><mi>c</mi><mrow><mo>(</mo><mrow><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover></mrow><mo>)</mo></mrow></mrow></mfenced></math></span> provided the activation curves under steady-state conditions with a fixed [Ca<sup>2+</sup>]<sub>pipette</sub> of 10 µM and variable values for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e3142"><msub><mrow><mi>ϕ</mi></mrow><mrow><mi>P</mi><mi>M</mi><mi>C</mi><mi>A</mi></mrow></msub></math></span> (Fig. <a href="#Fig3" class="usa-link">3a</a>).</p>
<p id="Par49">For modelling BK<sub>Ca</sub>-responses to the <em>dynamic conditions</em> given in the ‘Ca<sup>2+</sup>-pulse experiments’ (as in Fig. <a href="#Fig1" class="usa-link">1c</a>), the channel model was extended to include forward and backward rates (Supplementary Table <a href="#MOESM1" class="usa-link">3</a>). The open probability P<sub>open</sub> (t) is given as</p>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><math id="d33e3181" display="block"><msub><mrow><mi>P</mi></mrow><mrow><mi>o</mi><mi>p</mi><mi>e</mi><mi>n</mi></mrow></msub><mfenced close=")" open="("><mrow><mi>t</mi></mrow></mfenced><mo>=</mo><mfrac><mrow><msub><mrow><mi>O</mi></mrow><mrow><mi>t</mi><mi>o</mi><mi>t</mi></mrow></msub><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mrow><mrow><msub><mrow><mi>C</mi></mrow><mrow><mi>t</mi><mi>o</mi><mi>t</mi></mrow></msub><mfenced close=")" open="("><mrow><mi>t</mi></mrow></mfenced><mo>+</mo><msub><mrow><mi>O</mi></mrow><mrow><mi>t</mi><mi>o</mi><mi>t</mi></mrow></msub><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mrow></mfrac><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mn>1</mn><mo>+</mo><msubsup><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>i</mi><mo>=</mo><mn>0</mn></mrow><mrow><mn>4</mn></mrow></msubsup><msub><mrow><mi>C</mi></mrow><mrow><mi>i</mi></mrow></msub><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow><mo>/</mo><msubsup><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>i</mi><mo>=</mo><mn>0</mn></mrow><mrow><mn>4</mn></mrow></msubsup><msub><mrow><mi>O</mi></mrow><mrow><mi>i</mi></mrow></msub><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mrow></mfrac></math></td>
<td class="label">8</td>
</tr></table>
<p id="Par50">The extended model was calibrated with experimental data previously obtained on BK<sub>Ca</sub> channels exposed to sub-millisecond switches of [Ca<sup>2+</sup>]<sub>i</sub> under constant membrane potential conditions (<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>; Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>); the respective values for Q and the reaction constants fitting the experimental data best are summarized in Supplementary Table <a href="#MOESM1" class="usa-link">3</a>.</p>
<p id="Par51">Again, these parameters were used together with the time-dependent intracellular [Ca<sup>2+</sup>]<sub>i</sub> profiles, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e3311"><mi>c</mi><mfenced close=")" open="("><mrow><mover accent="true"><mrow><mi>x</mi></mrow><mo>→</mo></mover><mo>,</mo><mi>t</mi></mrow></mfenced></math></span>, to compute the time course of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e3328"><msub><mrow><mi>P</mi></mrow><mrow><mi>o</mi><mi>p</mi><mi>e</mi><mi>n</mi></mrow></msub><mfenced close=")" open="("><mrow><mi>t</mi></mrow></mfenced></math></span> that provided the response(s) of the BK<sub>Ca</sub>-model to the 0.8 ms-pulses of [Ca<sup>2+</sup>]<sub>I</sub> under defined values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e3354"><msub><mrow><mi>ϕ</mi></mrow><mrow><mi>P</mi><mi>M</mi><mi>C</mi><mi>A</mi></mrow></msub></math></span> (Fig. <a href="#Fig3" class="usa-link">3b</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">5c</a>).</p></section></section><section id="Sec23"><h3 class="pmc_sec_title">Quantification and data analysis</h3>
<p id="Par52">All statistical details are indicated in the figure legends. Data are given as mean ± SEM throughout the manuscript and are analyzed as detailed in the Method section. Significance was assessed by the non-parametric Mann-Whitney U-test, significance levels are indicated (***, **, * for <em>p</em> values &lt; 0.001, 0.01, 0.05, respectively; n.s. is not significant).</p></section><section id="Sec24"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par53">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec25"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12368035/bin/41467_2025_62735_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (1.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12368035/bin/41467_2025_62735_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (186.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12368035/bin/41467_2025_62735_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (15.4KB, pdf) </sup>
</div></div></section></section><section id="Sec26"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12368035/bin/41467_2025_62735_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (16.8KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank V. Flockerzi, G. Zolles and S. Raunser for intense discussions and critical reading of the manuscript, S. Boudkkazi for help with functional recordings, A. Haupt for help with bioinformatics and N. Wernet for assistance with EM work. This work was supported by grants of the DFG to B.F. (TRR152, project ID 23928380; Fa 332/17-1; Fa 332/19-1; SFB 1453, project ID 431984000), to H.R. (SFB 1027, project ID 200049484) and to A.K. (FOR 2143).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>C.E.C. and B.F. conceived the project, C.S.M., S.H., A.Ko., and U.S. performed experiments related to protein biochemistry and proteomic analyses, C.E.C. performed electrophysiological recordings and related data evaluations, Y.S. and D.B. performed fluorescence measurements, C.S.M., H.H., B.G., and A.Ku. did immuno-EM in SDS-FRL configuration, B.S. and H.R. performed computer calculations and modelling, B.F. evaluated data and wrote the manuscript with the support of all authors.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par54"><em>Nature Communications</em> thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Funding</h2>
<p>Open Access funding enabled and organized by Projekt DEAL.</p></section><section id="notes4"><h2 class="pmc_sec_title">Data availability</h2>
<p>The mass spectrometry data (Fig. <a href="#Fig2" class="usa-link">2</a>) have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD060109 [10.6019/PXD060109]. <a href="#Sec26" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par55">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Cristina E. Constantin, Barbara Schmidt.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Heiko Rieger, Email: heiko.rieger@uni-saarland.de.</p>
<p>Bernd Fakler, Email: bernd.fakler@physiologie.uni-freiburg.de.</p></section><section id="sec27"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62735-5.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Brini, M. &amp; Carafoli, E. Calcium pumps in health and disease. <em>Physiol. Rev.</em><strong>89</strong>, 1341–1378 (2009).
</cite> [<a href="https://doi.org/10.1152/physrev.00032.2008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19789383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brini,%20M.%20&amp;%20Carafoli,%20E.%20Calcium%20pumps%20in%20health%20and%20disease.%20Physiol.%20Rev.89,%201341%E2%80%931378%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Dyla, M., Kjaergaard, M., Poulsen, H. &amp; Nissen, P. Structure and mechanism of P-Type ATPase Ion pumps. <em>Annu Rev. Biochem.</em><strong>89</strong>, 583–603 (2020).
</cite> [<a href="https://doi.org/10.1146/annurev-biochem-010611-112801" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31874046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dyla,%20M.,%20Kjaergaard,%20M.,%20Poulsen,%20H.%20&amp;%20Nissen,%20P.%20Structure%20and%20mechanism%20of%20P-Type%20ATPase%20Ion%20pumps.%20Annu%20Rev.%20Biochem.89,%20583%E2%80%93603%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Palmgren, M. G. &amp; Nissen, P. P-type ATPases. <em>Annu. Rev. Biophys.</em><strong>40</strong>, 243–266 (2011).
</cite> [<a href="https://doi.org/10.1146/annurev.biophys.093008.131331" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21351879/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palmgren,%20M.%20G.%20&amp;%20Nissen,%20P.%20P-type%20ATPases.%20Annu.%20Rev.%20Biophys.40,%20243%E2%80%93266%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Augustine, G. J., Santamaria, F. &amp; Tanaka, K. Local calcium signaling in neurons. <em>Neuron</em><strong>40</strong>, 331–346 (2003).
</cite> [<a href="https://doi.org/10.1016/s0896-6273(03)00639-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14556712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Augustine,%20G.%20J.,%20Santamaria,%20F.%20&amp;%20Tanaka,%20K.%20Local%20calcium%20signaling%20in%20neurons.%20Neuron40,%20331%E2%80%93346%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Clapham, D. E. Calcium signaling. <em>Cell</em><strong>131</strong>, 1047–1058 (2007).
</cite> [<a href="https://doi.org/10.1016/j.cell.2007.11.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18083096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clapham,%20D.%20E.%20Calcium%20signaling.%20Cell131,%201047%E2%80%931058%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Fakler, B. &amp; Adelman, J. P. Control of K<sub>Ca</sub> channels by calcium nano/microdomains. <em>Neuron</em><strong>59</strong>, 873–881 (2008).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2008.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18817728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fakler,%20B.%20&amp;%20Adelman,%20J.%20P.%20Control%20of%20KCa%20channels%20by%20calcium%20nano/microdomains.%20Neuron59,%20873%E2%80%93881%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Gong, D. et al. Structure of the human plasma membrane Ca<sup>2+</sup>-ATPase 1 in complex with its obligatory subunit neuroplastin. <em>Nat. Commun.</em><strong>9</strong>, 3623 (2018).
</cite> [<a href="https://doi.org/10.1038/s41467-018-06075-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6127144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30190470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gong,%20D.%20et%20al.%20Structure%20of%20the%20human%20plasma%20membrane%20Ca2+-ATPase%201%20in%20complex%20with%20its%20obligatory%20subunit%20neuroplastin.%20Nat.%20Commun.9,%203623%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Herrera-Molina, R. et al. Neuroplastin deletion in glutamatergic neurons impairs selective brain functions and calcium regulation: implication for cognitive deterioration. <em>Sci. Rep.</em><strong>7</strong>, 7273 (2017).
</cite> [<a href="https://doi.org/10.1038/s41598-017-07839-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5544750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28779130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herrera-Molina,%20R.%20et%20al.%20Neuroplastin%20deletion%20in%20glutamatergic%20neurons%20impairs%20selective%20brain%20functions%20and%20calcium%20regulation:%20implication%20for%20cognitive%20deterioration.%20Sci.%20Rep.7,%207273%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Korthals, M. et al. A complex of neuroplastin and plasma membrane Ca<sup>2+</sup> ATPase controls T cell activation. <em>Sci. Rep.</em><strong>7</strong>, 8358 (2017).
</cite> [<a href="https://doi.org/10.1038/s41598-017-08519-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5566957/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28827723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Korthals,%20M.%20et%20al.%20A%20complex%20of%20neuroplastin%20and%20plasma%20membrane%20Ca2+%20ATPase%20controls%20T%20cell%20activation.%20Sci.%20Rep.7,%208358%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Schmidt, N. et al. Neuroplastin and basigin are essential auxiliary subunits of plasma membrane Ca(2+)-ATPases and key regulators of Ca<sup>2+</sup> clearance. <em>Neuron</em><strong>96</strong>, 827–838 (2017).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2017.09.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29056295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schmidt,%20N.%20et%20al.%20Neuroplastin%20and%20basigin%20are%20essential%20auxiliary%20subunits%20of%20plasma%20membrane%20Ca(2+)-ATPases%20and%20key%20regulators%20of%20Ca2+%20clearance.%20Neuron96,%20827%E2%80%93838%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Empson, R. M., Garside, M. L. &amp; Knopfel, T. Plasma membrane Ca<sup>2+</sup> ATPase 2 contributes to short-term synapse plasticity at the parallel fiber to Purkinje neuron synapse. <em>J. Neurosci.</em><strong>27</strong>, 3753–3758 (2007).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.0069-07.2007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6672407/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17409239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Empson,%20R.%20M.,%20Garside,%20M.%20L.%20&amp;%20Knopfel,%20T.%20Plasma%20membrane%20Ca2+%20ATPase%202%20contributes%20to%20short-term%20synapse%20plasticity%20at%20the%20parallel%20fiber%20to%20Purkinje%20neuron%20synapse.%20J.%20Neurosci.27,%203753%E2%80%933758%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Morgans, C. W., El Far, O., Berntson, A., Wassle, H. &amp; Taylor, W. R. Calcium extrusion from mammalian photoreceptor terminals. <em>J. Neurosci.</em><strong>18</strong>, 2467–2474 (1998).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.18-07-02467.1998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6793104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9502807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morgans,%20C.%20W.,%20El%20Far,%20O.,%20Berntson,%20A.,%20Wassle,%20H.%20&amp;%20Taylor,%20W.%20R.%20Calcium%20extrusion%20from%20mammalian%20photoreceptor%20terminals.%20J.%20Neurosci.18,%202467%E2%80%932474%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Choquette, D. et al. Regulation of plasma membrane Ca<sup>2+</sup> ATPases by lipids of the phosphatidylinositol cycle. <em>Biochem Biophys. Res Commun.</em><strong>125</strong>, 908–915 (1984).
</cite> [<a href="https://doi.org/10.1016/0006-291x(84)91369-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6097254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Choquette,%20D.%20et%20al.%20Regulation%20of%20plasma%20membrane%20Ca2+%20ATPases%20by%20lipids%20of%20the%20phosphatidylinositol%20cycle.%20Biochem%20Biophys.%20Res%20Commun.125,%20908%E2%80%93915%20(1984)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Missiaen, L., Wuytack, F., Raeymaekers, L., De Smedt, H. &amp; Casteels, R. Polyamines and neomycin inhibit the purified plasma-membrane Ca<sup>2+</sup> pump by interacting with associated polyphosphoinositides. <em>Biochem J.</em><strong>261</strong>, 1055–1058 (1989).
</cite> [<a href="https://doi.org/10.1042/bj2611055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1138938/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2552987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Missiaen,%20L.,%20Wuytack,%20F.,%20Raeymaekers,%20L.,%20De%20Smedt,%20H.%20&amp;%20Casteels,%20R.%20Polyamines%20and%20neomycin%20inhibit%20the%20purified%20plasma-membrane%20Ca2+%20pump%20by%20interacting%20with%20associated%20polyphosphoinositides.%20Biochem%20J.261,%201055%E2%80%931058%20(1989)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Niggli, V., Adunyah, E. S. &amp; Carafoli, E. Acidic phospholipids, unsaturated fatty acids, and limited proteolysis mimic the effect of calmodulin on the purified erythrocyte Ca<sup>2+</sup>- ATPase. <em>J. Biol. Chem.</em><strong>256</strong>, 8588–8592 (1981).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/6455424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Niggli,%20V.,%20Adunyah,%20E.%20S.%20&amp;%20Carafoli,%20E.%20Acidic%20phospholipids,%20unsaturated%20fatty%20acids,%20and%20limited%20proteolysis%20mimic%20the%20effect%20of%20calmodulin%20on%20the%20purified%20erythrocyte%20Ca2+-%20ATPase.%20J.%20Biol.%20Chem.256,%208588%E2%80%938592%20(1981)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Vinayagam, D. et al. Molecular mechanism of ultrafast transport of plasma membrane Ca<sup>2+</sup>-ATPases. <em>Nature</em><strong>1131</strong>, 93–129 (2025).</cite> [<a href="https://doi.org/10.1038/s41586-025-09402-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40836084/" class="usa-link">PubMed</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Baukrowitz, T. et al. PIP<sub>2</sub> and PIP as determinants for ATP inhibition of K<sub>ATP</sub> channels. <em>Science</em><strong>282</strong>, 1141–1144 (1998).</cite> [<a href="https://doi.org/10.1126/science.282.5391.1141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9804555/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Choveau, F. S., De la Rosa, V., Bierbower, S. M., Hernandez, C. C. &amp; Shapiro, M. S. Phosphatidylinositol 4,5-bisphosphate (PIP(2)) regulates KCNQ3 K<sup>+</sup> channels by interacting with four cytoplasmic channel domains. <em>J. Biol. Chem.</em><strong>293</strong>, 19411–19428 (2018).
</cite> [<a href="https://doi.org/10.1074/jbc.RA118.005401" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6302169/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30348901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Choveau,%20F.%20S.,%20De%20la%20Rosa,%20V.,%20Bierbower,%20S.%20M.,%20Hernandez,%20C.%20C.%20&amp;%20Shapiro,%20M.%20S.%20Phosphatidylinositol%204,5-bisphosphate%20(PIP(2))%20regulates%20KCNQ3%20K+%20channels%20by%20interacting%20with%20four%20cytoplasmic%20channel%20domains.%20J.%20Biol.%20Chem.293,%2019411%E2%80%9319428%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Oliver, D. et al. Functional conversion between A-type and delayed rectifier K<sup>+</sup> channels by membrane lipids. <em>Science</em><strong>304</strong>, 265–270 (2004).
</cite> [<a href="https://doi.org/10.1126/science.1094113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15031437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oliver,%20D.%20et%20al.%20Functional%20conversion%20between%20A-type%20and%20delayed%20rectifier%20K+%20channels%20by%20membrane%20lipids.%20Science304,%20265%E2%80%93270%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Turecek, R. et al. Auxiliary GABA<sub>B</sub> receptor subunits uncouple G protein betagamma subunits from effector channels to induce desensitization. <em>Neuron</em><strong>82</strong>, 1032–1044 (2014).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2014.04.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24836506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Turecek,%20R.%20et%20al.%20Auxiliary%20GABAB%20receptor%20subunits%20uncouple%20G%20protein%20betagamma%20subunits%20from%20effector%20channels%20to%20induce%20desensitization.%20Neuron82,%201032%E2%80%931044%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Zolles, G. et al. Pacemaking by HCN channels requires interaction with phosphoinositides. <em>Neuron</em><strong>52</strong>, 1027–1036 (2006).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2006.12.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17178405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zolles,%20G.%20et%20al.%20Pacemaking%20by%20HCN%20channels%20requires%20interaction%20with%20phosphoinositides.%20Neuron52,%201027%E2%80%931036%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Berkefeld, H., Fakler, B. &amp; Schulte, U. Ca<sup>2+</sup>-activated K<sup>+</sup> channels: from protein complexes to function. <em>Physiol. Rev.</em><strong>90</strong>, 1437–1459 (2010).
</cite> [<a href="https://doi.org/10.1152/physrev.00049.2009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20959620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berkefeld,%20H.,%20Fakler,%20B.%20&amp;%20Schulte,%20U.%20Ca2+-activated%20K+%20channels:%20from%20protein%20complexes%20to%20function.%20Physiol.%20Rev.90,%201437%E2%80%931459%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Berkefeld, H. &amp; Fakler, B. Ligand-gating by Ca<sup>2+</sup> is rate limiting for physiological operation of BK(Ca) channels. <em>J. Neurosci.</em><strong>33</strong>, 7358–7367 (2013).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.5443-12.2013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6619579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23616542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berkefeld,%20H.%20&amp;%20Fakler,%20B.%20Ligand-gating%20by%20Ca2+%20is%20rate%20limiting%20for%20physiological%20operation%20of%20BK(Ca)%20channels.%20J.%20Neurosci.33,%207358%E2%80%937367%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Berkefeld, H. et al. BKCa-Cav channel complexes mediate rapid and localized Ca<sup>2+</sup>-activated K<sup>+</sup> signaling. <em>Science</em><strong>314</strong>, 615–620 (2006).
</cite> [<a href="https://doi.org/10.1126/science.1132915" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17068255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berkefeld,%20H.%20et%20al.%20BKCa-Cav%20channel%20complexes%20mediate%20rapid%20and%20localized%20Ca2+-activated%20K+%20signaling.%20Science314,%20615%E2%80%93620%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Delvendahl, I. &amp; Hallermann, S. The cerebellar mossy fiber synapse as a model for high-frequency transmission in the mammalian CNS. <em>Trends Neurosci.</em><strong>39</strong>, 722–737 (2016).
</cite> [<a href="https://doi.org/10.1016/j.tins.2016.09.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27771145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Delvendahl,%20I.%20&amp;%20Hallermann,%20S.%20The%20cerebellar%20mossy%20fiber%20synapse%20as%20a%20model%20for%20high-frequency%20transmission%20in%20the%20mammalian%20CNS.%20Trends%20Neurosci.39,%20722%E2%80%93737%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Helassa, N., Podor, B., Fine, A. &amp; Torok, K. Design and mechanistic insight into ultrafast calcium indicators for monitoring intracellular calcium dynamics. <em>Sci. Rep.</em><strong>6</strong>, 38276 (2016).
</cite> [<a href="https://doi.org/10.1038/srep38276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5138832/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27922063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Helassa,%20N.,%20Podor,%20B.,%20Fine,%20A.%20&amp;%20Torok,%20K.%20Design%20and%20mechanistic%20insight%20into%20ultrafast%20calcium%20indicators%20for%20monitoring%20intracellular%20calcium%20dynamics.%20Sci.%20Rep.6,%2038276%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Shigetomi, E., Kracun, S., Sofroniew, M. V. &amp; Khakh, B. S. A genetically targeted optical sensor to monitor calcium signals in astrocyte processes. <em>Nat. Neurosci.</em><strong>13</strong>, 759–766 (2010).
</cite> [<a href="https://doi.org/10.1038/nn.2557" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2920135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20495558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shigetomi,%20E.,%20Kracun,%20S.,%20Sofroniew,%20M.%20V.%20&amp;%20Khakh,%20B.%20S.%20A%20genetically%20targeted%20optical%20sensor%20to%20monitor%20calcium%20signals%20in%20astrocyte%20processes.%20Nat.%20Neurosci.13,%20759%E2%80%93766%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Boudkkazi, S. et al. A Noelin-organized extracellular network of proteins required for constitutive and context-dependent anchoring of AMPA-receptors. <em>Neuron</em><strong>111</strong>, 2544–2556 e2549 (2023).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2023.07.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10441612/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37591201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boudkkazi,%20S.%20et%20al.%20A%20Noelin-organized%20extracellular%20network%20of%20proteins%20required%20for%20constitutive%20and%20context-dependent%20anchoring%20of%20AMPA-receptors.%20Neuron111,%202544%E2%80%932556%20e2549%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Schwenk, J. et al. Regional diversity and developmental dynamics of the AMPA-receptor proteome in the mammalian brain. <em>Neuron</em><strong>84</strong>, 41–54 (2014).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2014.08.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25242221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schwenk,%20J.%20et%20al.%20Regional%20diversity%20and%20developmental%20dynamics%20of%20the%20AMPA-receptor%20proteome%20in%20the%20mammalian%20brain.%20Neuron84,%2041%E2%80%9354%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Cox, D. H., Cui, J. &amp; Aldrich, R. W. Allosteric gating of a large conductance Ca-activated K<sup>+</sup> channel. <em>J. Gen. Physiol.</em><strong>110</strong>, 257–281 (1997).
</cite> [<a href="https://doi.org/10.1085/jgp.110.3.257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2229366/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9276753/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cox,%20D.%20H.,%20Cui,%20J.%20&amp;%20Aldrich,%20R.%20W.%20Allosteric%20gating%20of%20a%20large%20conductance%20Ca-activated%20K+%20channel.%20J.%20Gen.%20Physiol.110,%20257%E2%80%93281%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Garrahan, P. J. &amp; Rega, A. F. in <em>Intracellular Calcium Regulation</em> (ed F. Bronner) Ch. 10, 271-303 (Alan R. Liss, Inc., 1990).</cite>
</li>
<li id="CR32">
<span class="label">32.</span><cite>Naraghi, M. &amp; Neher, E. Linearized buffered Ca<sup>2+</sup> diffusion in microdomains and its implications for calculation of [Ca<sup>2+</sup>] at the mouth of a calcium channel. <em>J. Neurosci.</em><strong>17</strong>, 6961–6973 (1997).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.17-18-06961.1997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6573285/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9278532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Naraghi,%20M.%20&amp;%20Neher,%20E.%20Linearized%20buffered%20Ca2+%20diffusion%20in%20microdomains%20and%20its%20implications%20for%20calculation%20of%20%5BCa2+%5D%20at%20the%20mouth%20of%20a%20calcium%20channel.%20J.%20Neurosci.17,%206961%E2%80%936973%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Hilgemann, D. W., Nicoll, D. A. &amp; Philipson, K. D. Charge movement during Na+ translocation by native and cloned cardiac Na<sup>+</sup>/ Ca<sup>2+</sup> exchanger. <em>Nature</em><strong>352</strong>, 715–718 (1991).
</cite> [<a href="https://doi.org/10.1038/352715a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1876186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hilgemann,%20D.%20W.,%20Nicoll,%20D.%20A.%20&amp;%20Philipson,%20K.%20D.%20Charge%20movement%20during%20Na+%20translocation%20by%20native%20and%20cloned%20cardiac%20Na+/%20Ca2+%20exchanger.%20Nature352,%20715%E2%80%93718%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Masugi-Tokita, M. &amp; Shigemoto, R. High-resolution quantitative visualization of glutamate and GABA receptors at central synapses. <em>Curr. Opin. Neurobiol.</em><strong>17</strong>, 387–393 (2007).
</cite> [<a href="https://doi.org/10.1016/j.conb.2007.04.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17499496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Masugi-Tokita,%20M.%20&amp;%20Shigemoto,%20R.%20High-resolution%20quantitative%20visualization%20of%20glutamate%20and%20GABA%20receptors%20at%20central%20synapses.%20Curr.%20Opin.%20Neurobiol.17,%20387%E2%80%93393%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Masugi-Tokita, M. et al. Number and density of AMPA receptors in individual synapses in the rat cerebellum as revealed by SDS-digested freeze-fracture replica labeling. <em>J. Neurosci.</em><strong>27</strong>, 2135–2144 (2007).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.2861-06.2007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6673557/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17314308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Masugi-Tokita,%20M.%20et%20al.%20Number%20and%20density%20of%20AMPA%20receptors%20in%20individual%20synapses%20in%20the%20rat%20cerebellum%20as%20revealed%20by%20SDS-digested%20freeze-fracture%20replica%20labeling.%20J.%20Neurosci.27,%202135%E2%80%932144%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Tanaka, J. et al. Number and density of AMPA receptors in single synapses in immature cerebellum. <em>J. Neurosci.</em><strong>25</strong>, 799–807 (2005).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.4256-04.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6725634/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15673659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tanaka,%20J.%20et%20al.%20Number%20and%20density%20of%20AMPA%20receptors%20in%20single%20synapses%20in%20immature%20cerebellum.%20J.%20Neurosci.25,%20799%E2%80%93807%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Bortolozzi, M. et al. The novel PMCA2 pump mutation Tommy impairs cytosolic calcium clearance in hair cells and links to deafness in mice. <em>J. Biol. Chem.</em><strong>285</strong>, 37693–37703 (2010).
</cite> [<a href="https://doi.org/10.1074/jbc.M110.170092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2988374/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20826782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bortolozzi,%20M.%20et%20al.%20The%20novel%20PMCA2%20pump%20mutation%20Tommy%20impairs%20cytosolic%20calcium%20clearance%20in%20hair%20cells%20and%20links%20to%20deafness%20in%20mice.%20J.%20Biol.%20Chem.285,%2037693%E2%80%9337703%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Chen, Q. et al. The development, distribution and density of the plasma membrane calcium ATPase 2 calcium pump in rat cochlear hair cells. <em>Eur. J. Neurosci.</em><strong>36</strong>, 2302–2310 (2012).
</cite> [<a href="https://doi.org/10.1111/j.1460-9568.2012.08159.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3412916/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22672315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20Q.%20et%20al.%20The%20development,%20distribution%20and%20density%20of%20the%20plasma%20membrane%20calcium%20ATPase%202%20calcium%20pump%20in%20rat%20cochlear%20hair%20cells.%20Eur.%20J.%20Neurosci.36,%202302%E2%80%932310%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Tiffert, T. &amp; Lew, V. L. Elevated intracellular Ca<sup>2+</sup> reveals a functional membrane nucleotide pool in intact human red blood cells. <em>J. Gen. Physiol.</em><strong>138</strong>, 381–391 (2011).
</cite> [<a href="https://doi.org/10.1085/jgp.201110660" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3182447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21948947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tiffert,%20T.%20&amp;%20Lew,%20V.%20L.%20Elevated%20intracellular%20Ca2+%20reveals%20a%20functional%20membrane%20nucleotide%20pool%20in%20intact%20human%20red%20blood%20cells.%20J.%20Gen.%20Physiol.138,%20381%E2%80%93391%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Empson, R. M., Turner, P. R., Nagaraja, R. Y., Beesley, P. W. &amp; Knopfel, T. Reduced expression of the Ca<sup>2+</sup> transporter protein PMCA2 slows Ca<sup>2+</sup> dynamics in mouse cerebellar Purkinje neurones and alters the precision of motor coordination. <em>J. Physiol.</em><strong>588</strong>, 907–922 (2010).
</cite> [<a href="https://doi.org/10.1113/jphysiol.2009.182196" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2849958/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20083513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Empson,%20R.%20M.,%20Turner,%20P.%20R.,%20Nagaraja,%20R.%20Y.,%20Beesley,%20P.%20W.%20&amp;%20Knopfel,%20T.%20Reduced%20expression%20of%20the%20Ca2+%20transporter%20protein%20PMCA2%20slows%20Ca2+%20dynamics%20in%20mouse%20cerebellar%20Purkinje%20neurones%20and%20alters%20the%20precision%20of%20motor%20coordination.%20J.%20Physiol.588,%20907%E2%80%93922%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Empson, R. M., Huang, H., Nagaraja, R. Y., Roome, C. J. &amp; Knopfel, T. Enhanced synaptic inhibition in the cerebellar cortex of the ataxic PMCA2(-/-) knockout mouse. <em>Cerebellum</em><strong>12</strong>, 667–675 (2013).
</cite> [<a href="https://doi.org/10.1007/s12311-013-0472-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23568408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Empson,%20R.%20M.,%20Huang,%20H.,%20Nagaraja,%20R.%20Y.,%20Roome,%20C.%20J.%20&amp;%20Knopfel,%20T.%20Enhanced%20synaptic%20inhibition%20in%20the%20cerebellar%20cortex%20of%20the%20ataxic%20PMCA2(-/-)%20knockout%20mouse.%20Cerebellum12,%20667%E2%80%93675%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Juhaszova, M., Church, P., Blaustein, M. P. &amp; Stanley, E. F. Location of calcium transporters at presynaptic terminals. <em>Eur. J. Neurosci.</em><strong>12</strong>, 839–846 (2000).
</cite> [<a href="https://doi.org/10.1046/j.1460-9568.2000.00974.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10762313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Juhaszova,%20M.,%20Church,%20P.,%20Blaustein,%20M.%20P.%20&amp;%20Stanley,%20E.%20F.%20Location%20of%20calcium%20transporters%20at%20presynaptic%20terminals.%20Eur.%20J.%20Neurosci.12,%20839%E2%80%93846%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Delvendahl, I. et al. Reduced endogenous Ca<sup>2+</sup> buffering speeds active zone Ca<sup>2+</sup> signaling. <em>Proc. Natl. Acad. Sci. USA</em><strong>112</strong>, E3075–E3084 (2015).
</cite> [<a href="https://doi.org/10.1073/pnas.1508419112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4466756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26015575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Delvendahl,%20I.%20et%20al.%20Reduced%20endogenous%20Ca2+%20buffering%20speeds%20active%20zone%20Ca2+%20signaling.%20Proc.%20Natl.%20Acad.%20Sci.%20USA112,%20E3075%E2%80%93E3084%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Helmchen, F., Borst, J. G. &amp; Sakmann, B. Calcium dynamics associated with a single action potential in a CNS presynaptic terminal. <em>Biophys. J.</em><strong>72</strong>, 1458–1471 (1997).
</cite> [<a href="https://doi.org/10.1016/S0006-3495(97)78792-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1184528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9138591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Helmchen,%20F.,%20Borst,%20J.%20G.%20&amp;%20Sakmann,%20B.%20Calcium%20dynamics%20associated%20with%20a%20single%20action%20potential%20in%20a%20CNS%20presynaptic%20terminal.%20Biophys.%20J.72,%201458%E2%80%931471%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Tabata, S. et al. Electron microscopic detection of single membrane proteins by a specific chemical labeling. <em>iScience</em><strong>22</strong>, 256–268 (2019).
</cite> [<a href="https://doi.org/10.1016/j.isci.2019.11.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6906691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31786521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tabata,%20S.%20et%20al.%20Electron%20microscopic%20detection%20of%20single%20membrane%20proteins%20by%20a%20specific%20chemical%20labeling.%20iScience22,%20256%E2%80%93268%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Booker, S. A. et al. Presynaptic GABAB receptors functionally uncouple somatostatin interneurons from the active hippocampal network. <em>Elife</em><strong>9</strong>, 51156 (2020).</cite> [<a href="https://doi.org/10.7554/eLife.51156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7060044/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32073397/" class="usa-link">PubMed</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Althof, D. et al. Inhibitory and excitatory axon terminals share a common nano-architecture of their Cav2.1 (P/Q-type) Ca<sup>2+</sup> channels. <em>Front. Cell Neurosci.</em><strong>9</strong>, 315 (2015).
</cite> [<a href="https://doi.org/10.3389/fncel.2015.00315" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4531237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26321916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Althof,%20D.%20et%20al.%20Inhibitory%20and%20excitatory%20axon%20terminals%20share%20a%20common%20nano-architecture%20of%20their%20Cav2.1%20(P/Q-type)%20Ca2+%20channels.%20Front.%20Cell%20Neurosci.9,%20315%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Bildl, W. et al. Extending the dynamic range of label-free mass spectrometric quantification of affinity purifications. <em>Mol. Cell Proteom.</em><strong>15</strong>, 669–681 (2012).</cite> [<a href="https://doi.org/10.1074/mcp.M111.007955" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3277747/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22067099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bildl,%20W.%20et%20al.%20Extending%20the%20dynamic%20range%20of%20label-free%20mass%20spectrometric%20quantification%20of%20affinity%20purifications.%20Mol.%20Cell%20Proteom.15,%20669%E2%80%93681%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Kollewe, A. et al. Subunit composition, molecular environment, and activation of native TRPC channels encoded by their interactomes. <em>Neuron</em><strong>110</strong>, 4162–4175 (2022).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2022.09.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36257322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kollewe,%20A.%20et%20al.%20Subunit%20composition,%20molecular%20environment,%20and%20activation%20of%20native%20TRPC%20channels%20encoded%20by%20their%20interactomes.%20Neuron110,%204162%E2%80%934175%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Muller, C. S. et al. Cryo-slicing blue native-mass spectrometry (csBN-MS), a novel technology for high resolution complexome profiling. <em>Mol. Cell Proteom.</em><strong>15</strong>, 669–681 (2016).</cite> [<a href="https://doi.org/10.1074/mcp.M115.054080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4739680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26598645/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Muller,%20C.%20S.%20et%20al.%20Cryo-slicing%20blue%20native-mass%20spectrometry%20(csBN-MS),%20a%20novel%20technology%20for%20high%20resolution%20complexome%20profiling.%20Mol.%20Cell%20Proteom.15,%20669%E2%80%93681%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Schwenk, J. et al. High-resolution proteomics unravel architecture and molecular diversity of native AMPA receptor complexes. <em>Neuron</em><strong>74</strong>, 621–633 (2012).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2012.03.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22632720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schwenk,%20J.%20et%20al.%20High-resolution%20proteomics%20unravel%20architecture%20and%20molecular%20diversity%20of%20native%20AMPA%20receptor%20complexes.%20Neuron74,%20621%E2%80%93633%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Muller, C. S. et al. High-resolution complexome profiling by cryoslicing BN-MS analysis. <em>J. Vis. Exp</em>. <strong>10</strong>, 60096 (2019).</cite> [<a href="https://doi.org/10.3791/60096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31680678/" class="usa-link">PubMed</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Schagger, H. &amp; Pfeiffer, K. Supercomplexes in the respiratory chains of yeast and mammalian mitochondria. <em>EMBO J.</em><strong>19</strong>, 1777–1783 (2000).
</cite> [<a href="https://doi.org/10.1093/emboj/19.8.1777" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC302020/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10775262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schagger,%20H.%20&amp;%20Pfeiffer,%20K.%20Supercomplexes%20in%20the%20respiratory%20chains%20of%20yeast%20and%20mammalian%20mitochondria.%20EMBO%20J.19,%201777%E2%80%931783%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Bautista, D. M., Hoth, M. &amp; Lewis, R. S. Enhancement of calcium signalling dynamics and stability by delayed modulation of the plasma-membrane calcium-ATPase in human T cells. <em>J. Physiol.</em><strong>541</strong>, 877–894 (2002).
</cite> [<a href="https://doi.org/10.1113/jphysiol.2001.016154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2290354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12068047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bautista,%20D.%20M.,%20Hoth,%20M.%20&amp;%20Lewis,%20R.%20S.%20Enhancement%20of%20calcium%20signalling%20dynamics%20and%20stability%20by%20delayed%20modulation%20of%20the%20plasma-membrane%20calcium-ATPase%20in%20human%20T%20cells.%20J.%20Physiol.541,%20877%E2%80%93894%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Hecht, F. New Development in FreeFEM++. <em>J. Numer Math.</em><strong>20</strong>, 251–265 (2012).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hecht,%20F.%20New%20Development%20in%20FreeFEM++.%20J.%20Numer%20Math.20,%20251%E2%80%93265%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Press, W., Teukolsky, S., Vetterling, W. &amp; Flannery, B. <em>Numerical Recipes: The Art of Scientific Computing</em>. (Cambridge University Press, 2007).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368035/bin/41467_2025_62735_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (1.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368035/bin/41467_2025_62735_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (186.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368035/bin/41467_2025_62735_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (15.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368035/bin/41467_2025_62735_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (16.8KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The mass spectrometry data (Fig. <a href="#Fig2" class="usa-link">2</a>) have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD060109 [10.6019/PXD060109]. <a href="#Sec26" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62735-5"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62735.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368035/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368035/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368035%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368035/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368035/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368035/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835599/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368035/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835599/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368035/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368035/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="XpZQIgYY4eKviMeGreHfhbyJaMkyqzTu1flJhdfdQM8s1bURrIWc5zfBf8HpNENo">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
