
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Glucosylsphingosine affects mitochondrial function in a neuronal cell model - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532D6D8AF20503052D6D00447663C9.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="commbiol">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371010/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Communications Biology">
<meta name="citation_title" content="Glucosylsphingosine affects mitochondrial function in a neuronal cell model">
<meta name="citation_author" content="Valeria Nikolaenko">
<meta name="citation_author_institution" content="Translational Mass Spectrometry Research Group, Genetics &amp; Genomic Medicine Dept., UCL Institute of Child Health, London, WC1N 1EH UK">
<meta name="citation_author" content="Reddy Vootukuri">
<meta name="citation_author_institution" content="Translational Mass Spectrometry Research Group, Genetics &amp; Genomic Medicine Dept., UCL Institute of Child Health, London, WC1N 1EH UK">
<meta name="citation_author" content="Simon Eaton">
<meta name="citation_author_institution" content="Developmental Biology and Cancer Programme, UCL Great Ormond Street Institute of Child Health, London, UK">
<meta name="citation_author" content="Jenny Hällqvist">
<meta name="citation_author_institution" content="Translational Mass Spectrometry Research Group, Genetics &amp; Genomic Medicine Dept., UCL Institute of Child Health, London, WC1N 1EH UK">
<meta name="citation_author" content="Tomas Baldwin">
<meta name="citation_author_institution" content="Translational Mass Spectrometry Research Group, Genetics &amp; Genomic Medicine Dept., UCL Institute of Child Health, London, WC1N 1EH UK">
<meta name="citation_author" content="Kevin Mills">
<meta name="citation_author_institution" content="Translational Mass Spectrometry Research Group, Genetics &amp; Genomic Medicine Dept., UCL Institute of Child Health, London, WC1N 1EH UK">
<meta name="citation_author" content="Wendy Heywood">
<meta name="citation_author_institution" content="Translational Mass Spectrometry Research Group, Genetics &amp; Genomic Medicine Dept., UCL Institute of Child Health, London, WC1N 1EH UK">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="8">
<meta name="citation_firstpage" content="1260">
<meta name="citation_doi" content="10.1038/s42003-025-08684-7">
<meta name="citation_pmid" content="40841705">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371010/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371010/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371010/pdf/42003_2025_Article_8684.pdf">
<meta name="description" content="Gaucher disease arises from mutations in glucocerebrosidase resulting in accumulation of glucosylceramide, which is deacylated to glucosylsphingosine. Mutations in glucocerebrosidase are the greatest known genetic risk factor for Parkinson’s ...">
<meta name="og:title" content="Glucosylsphingosine affects mitochondrial function in a neuronal cell model">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Gaucher disease arises from mutations in glucocerebrosidase resulting in accumulation of glucosylceramide, which is deacylated to glucosylsphingosine. Mutations in glucocerebrosidase are the greatest known genetic risk factor for Parkinson’s ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371010/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371010">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s42003-025-08684-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/42003_2025_Article_8684.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371010%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371010/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371010/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371010/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-commbiol.jpg" alt="Communications Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Communications Biology" title="Link to Communications Biology" shape="default" href="https://www.nature.com/commsbio/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Commun Biol</button></div>. 2025 Aug 21;8:1260. doi: <a href="https://doi.org/10.1038/s42003-025-08684-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s42003-025-08684-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Commun%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Commun%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Commun%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Commun%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Glucosylsphingosine affects mitochondrial function in a neuronal cell model</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nikolaenko%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Valeria Nikolaenko</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Valeria Nikolaenko</span></h3>
<div class="p">
<sup>1</sup>Translational Mass Spectrometry Research Group, Genetics &amp; Genomic Medicine Dept., UCL Institute of Child Health, London, WC1N 1EH UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nikolaenko%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Valeria Nikolaenko</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vootukuri%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Reddy Vootukuri</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Reddy Vootukuri</span></h3>
<div class="p">
<sup>1</sup>Translational Mass Spectrometry Research Group, Genetics &amp; Genomic Medicine Dept., UCL Institute of Child Health, London, WC1N 1EH UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vootukuri%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Reddy Vootukuri</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Eaton%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Simon Eaton</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Simon Eaton</span></h3>
<div class="p">
<sup>2</sup>Developmental Biology and Cancer Programme, UCL Great Ormond Street Institute of Child Health, London, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Eaton%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Simon Eaton</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22H%C3%A4llqvist%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jenny Hällqvist</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jenny Hällqvist</span></h3>
<div class="p">
<sup>1</sup>Translational Mass Spectrometry Research Group, Genetics &amp; Genomic Medicine Dept., UCL Institute of Child Health, London, WC1N 1EH UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22H%C3%A4llqvist%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jenny Hällqvist</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baldwin%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Tomas Baldwin</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Tomas Baldwin</span></h3>
<div class="p">
<sup>1</sup>Translational Mass Spectrometry Research Group, Genetics &amp; Genomic Medicine Dept., UCL Institute of Child Health, London, WC1N 1EH UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baldwin%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tomas Baldwin</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mills%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Kevin Mills</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Kevin Mills</span></h3>
<div class="p">
<sup>1</sup>Translational Mass Spectrometry Research Group, Genetics &amp; Genomic Medicine Dept., UCL Institute of Child Health, London, WC1N 1EH UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mills%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kevin Mills</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heywood%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Wendy Heywood</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Wendy Heywood</span></h3>
<div class="p">
<sup>1</sup>Translational Mass Spectrometry Research Group, Genetics &amp; Genomic Medicine Dept., UCL Institute of Child Health, London, WC1N 1EH UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heywood%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wendy Heywood</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Translational Mass Spectrometry Research Group, Genetics &amp; Genomic Medicine Dept., UCL Institute of Child Health, London, WC1N 1EH UK </div>
<div id="Aff2">
<sup>2</sup>Developmental Biology and Cancer Programme, UCL Great Ormond Street Institute of Child Health, London, UK </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 31; Accepted 2025 Aug 6; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371010  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841705/" class="usa-link">40841705</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Gaucher disease arises from mutations in glucocerebrosidase resulting in accumulation of glucosylceramide, which is deacylated to glucosylsphingosine. Mutations in glucocerebrosidase are the greatest known genetic risk factor for Parkinson’s disease. Glucosylsphingosine is a biomarker for Gaucher disease and studies demonstrate its relevance to disease pathology, yet the mechanisms of its toxicity remain little understood. Using proteomics, we show that incubation of SH-Sy5y cells with glucosylsphingosine at physiological plasma concentrations observed in moderate/ severe Gaucher disease negatively effects the TCA cycle, mitochondrial function, glycolysis and protein ubiquitination. Functional analyses confirmed that glucosylsphingosine reduces ATP production, elicits oxidative stress and an increase of glycolysis. Analyses of ubiquitinated proteins and lipid-binding studies demonstrated that glucosylsphingosine has binding affinity for tubulin alpha and induced a specific increase of ubiquitination of α and β tubulins. In conclusion, supranormal levels of glucosylsphingosine affect cellular energy metabolism which may contribute to the pathology in Gaucher disease.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Mechanisms of disease, Proteomics, Neurodegeneration</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Exogenous addition of glucosylsphingosine affects cellular energy metabolism in SH-Sy5y cell line via changes in mitochondrial function and glycolysis. It also induces an increase in protein ubiquitination with tubulin species affected the most.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Homozygous mutations in the <em>GBA</em> gene that encodes for the lysosomal acid beta glucosidase (GBA1) cause the most common lysosomal storage disorder Gaucher disease (GD)<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>.Gaucher disease is a heterogeneous disease and is categorised by the absence (type I) or presence (types II and III) of the central nervous system involvement with GD II being the most severe<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. GBA1 breaks down the glycosphingolipid glucosylceramide. The downstream consequence of glucosylceramide accumulation is its modification to glucosylsphingosine (GlcSph), where the fatty acid moiety is removed from the accumulating glucosylceramide by the lysosomal enzyme acid ceramidase<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Plasma levels of GlcSph strongly correlate with disease burden and response to treatment, with highest levels observed in the brain of neuronopathic GD<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. GlcSph is considered a superior biomarker than the accepted GD biomarker chitotriosidase. However, chitotriosidase is not specific to GD as it can be elevated in other diseases and cannot be measured in some populations who carry a 24–base pair duplication in the chitotriosidase gene<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Exogenous GlcSph causes the same haematological and visceral changes in mice as those observed in GD patients, while endogenous GlcSph interferes with lysosomal biogenesis, promotes a-synuclein aggregation and toxicity, directly correlates with a-synuclein levels in Parkinsonian brain, is toxic to cholinergic neurons and causes loss of dopaminergic neurons<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>.</p>
<p id="Par4">Decreased GBA1 activity results in mitochondrial dysfunction with loss of mitochondrial membrane potential, reduced ATP, increased oxidative stress and fragmented mitochondria and is observed in GD fibroblasts, GD cell and animal models and is a well-established feature of Parkinson’s disease (PD)<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Despite the established connection between GlcSph and some of the common pathology of GD and PD, the exact mechanism of GlcSph cellular toxicity and effect on mitochondria is not clear. We therefore aimed to evaluate the effect of GlcSph on mitochondria at the two physiological concentrations observed in moderate (20 ng/mL) and severe (200 ng/mL) GD phenotypes using a neuronal SH-Sy5y cell model. To determine the specific effect of GlcSph we have used another disease associated lyso-glycosphingolipid, lyso-Gb3, as a disease control. Lyso-Gb3 differs from GlcSph by a trihexoside moiety with two galactose and a glucose molecule instead of a single glucose attached to the ceramide. Lyso-Gb3 is associated with Fabry disease which has a very different clinical phenotype to GD and no neurological involvement. Our previous work has shown it also has its own specific effect on the cell<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. The work presented here expands on the effect of GlcSph on the cell’s metabolic activity and how this could cause the associated metabolic phenotype in GD and potentially shed light on the role of GlcSph in PD.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<p id="Par5">GlcSph entry into cells from culture medium was confirmed by UPLS-MS/MS and was proportionately increased with increasing concentrations of GlcSph supplementation (Supplementary Fig. <a href="#MOESM1" class="usa-link">S1</a>).</p>
<section id="Sec3"><h3 class="pmc_sec_title">GlcSph exposure causes proteomic changes in metabolic pathways in SH-Sy5y cells</h3>
<p id="Par6">Brain tissue concentrations of GlcSph varies from 0.4 to 4.7 mg/kg among GD I – III cases and are 100–1000-fold increased compared to the normal brain, and plasma levels reach 250 ng/mL but depend on the type and the severity of <em>GBA1</em> mutation and residual enzyme activity<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. To gain insights into molecular mechanisms of GlcSph toxicity we incubated SH-Sy5y cells with GlcSph at 20 ng/mL, which is observed in mildly affected GD patients, for 24 h to test for the acute response on the cellular proteome. Cells were also incubated with GlcSph at 200 ng/mL for 72 h to test for a longer-term effect. The disease control lyso-Gb3 was used at the same concentrations. DMSO was used as a vehicle control.</p>
<p id="Par7">As previously reported both GlcSph and lyso-Gb3 affect the cellular proteome, however, GlcSph exposure resulted in a bigger change in proteome expression than lyso-Gb3. At both low and high doses GlcSph affected approximately 10% and 16% of the proteome respectively <em>versus</em> 5.8% and 12.4% compared to lyso-Gb3 (Fig. <a href="#Fig1" class="usa-link">1a</a>). While the 20 ng/mL treatment resulted in 21% reduction in protein level for both lyso-lipids, the larger dose of 200 ng/mL led to a greater increase in the number of downregulated proteins for GlcSph (29% vs 19%) (Fig. <a href="#Fig1" class="usa-link">1b</a>). To gain insights into biological pathways affected by the lyso-lipids we performed a bioinformatics pathway analysis looking at the changes in protein expression after treatment using Ingenuity Pathway Analysis tool (Qiagen). The analysis suggested that GlcSph mainly affects metabolic pathways whilst lyso-Gb3 affects signalling pathways (Supplementary Tables <a href="#MOESM1" class="usa-link">S1</a>–<a href="#MOESM1" class="usa-link">S4</a>). At the low dose of 20 ng/mL GlcSph affected the TCA cycle the most, followed by RAN signalling (involved in metabolic regulation and dependent on the energy status of the cell), mitochondrial dysfunction and sirtuin signalling (Fig. <a href="#Fig1" class="usa-link">1c</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Exposure to GlcSph and lyso-Gb3 at 20 ng/mL and 200 ng/mL significantly alters protein expression in the SH-Sy5y cell line.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371010_42003_2025_8684_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d865/12371010/bb04ef3f7e0b/42003_2025_8684_Fig1_HTML.jpg" loading="lazy" id="d33e314" height="452" width="800" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Percent of significant (<em>p </em>&lt; 0.05) protein changes of the total proteome by the lysolipid and dose. <strong>b</strong> The number of significantly up-/down-regulated proteins. Top 15 significantly affected cellular pathways by GlcSph at <strong>c</strong> 20 ng/mL and <strong>d</strong> 200 ng/mL. Circle radii represent the negative log10 of the pathway enrichment <em>p</em>-value &lt; 0.05 and the proteins affected in the pathway. z-scores &gt; 0 indicate activation while z-scores &lt; 0 indicate deactivation of the pathway. <strong>e</strong> Significantly affected mitochondrial proteins by each lyso-lipid and dose. Source data are provided as a Supplementary Data file.</p></figcaption></figure><p id="Par8">In the high dose of 200 ng/mL and similarly to the low dose mitochondrial dysfunction and sirtuin signalling pathways remained disturbed, however, they were superseded by tRNA charging (involved in protein translation) and protein ubiquitination pathways. In addition. glycolysis was also affected (Fig. <a href="#Fig1" class="usa-link">1d</a>). These data indicate that GlcSph affects cellular metabolism and with increased dose likely starts to affect protein folding and degradation.</p>
<p id="Par9">As mitochondrial dysfunction was a pathway specifically affected by GlcSph exposure in both doses a further analysis was performed looking at mitochondrial protein expression alone and their relation to key mitochondrial processes to get a deeper understanding. Figure <a href="#Fig1" class="usa-link">1e</a> shows the percent of significantly affected proteins in each process defined by the total identified proteins assigned to that process. The low dose of 20 ng/mL appears to affect the TCA cycle, complex I, IV and V but the high 200 ng/mL dose also affects complex III and starts to affect acetyl CoA metabolism and fatty acid degradation.</p></section><section id="Sec4"><h3 class="pmc_sec_title">GlcSph reduces ATP production rate and induces metabolic shift towards glycolysis</h3>
<p id="Par10">To explore further the effect of GlcSph on energy metabolism we investigated ATP production and glycolytic rates in live SH-Sy5y cells exposed to the two doses of GlcSph using Seahorse technology (Agilent, USA). This technology simultaneously measures oxygen consumption and extracellular acidification rates to infer the two major energy producing processes – oxidative phosphorylation (OXPHOS) and glycolysis. Cells were incubated with GlcSph at 20 ng/mL and 200 ng/mL for 72 h and assessed for ATP production and glycolytic rates (basal measurement). To examine whether GlcSph also affects cellular capacity to switch between alternative pathways of ATP production, a mitochondrial pyruvate carrier inhibitor was used (induced measurement) for the ATP production rate.</p>
<p id="Par11">The analyses shown in Fig. <a href="#Fig2" class="usa-link">2</a> demonstrated that cellular ATP production rate was not affected by GlcSph exposure at 20 ng/mL with or without mitochondrial pyruvate carrier inhibition (Fig. <a href="#Fig2" class="usa-link">2a</a>). This suggests at the low 20 ng/mL dose GlcSph did not affect the mitochondrial OXPHOS and the cell’s ability to switch fuels. However, that was not the case for the high dose of 200 ng/mL, which led to a significant 28% drop in ATP production rate (<em>p </em>= 0.0025) and the immediate cellular response to the inhibition of the pyruvate entry into the mitochondria (<em>p </em>= 0.0025) (Fig. <a href="#Fig2" class="usa-link">2b</a>). This was contrasted with the glycolytic rate, which was significantly increased at both dose exposures (Fig. <a href="#Fig2" class="usa-link">2</a>c, d). Basal glycolysis under GlcSph exposure increased by 45% (<em>p </em>= 0.0071) in response to the low dose, which also affected the cell’s ability to compensate for the total inhibition of mitochondrial OXPHOS (compensatory rate) and led only to a 16% increase in glycolytic rate in treated cells (<em>p </em>= 0.015) <em>vs</em> 37% for the DMSO control cells (<em>p </em>= 0.015). At the high dose GlcSph led to an even higher (50%) increase in basal glycolysis (<em>p </em>= 0.0066), while compensatory glycolysis was increased by 58% (<em>p </em>= 0.0066). Furthermore, GlcSph exposed cells at the high dose were able to upregulate glycolysis more than the DMSO control cells (28% <em>vs</em> 21%) with the total inhibition of OXPHOS (Fig. <a href="#Fig2" class="usa-link">2d</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. The effect of GlcSph on energy metabolism and mitochondrial function in live SH-Sy5y cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371010_42003_2025_8684_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d865/12371010/351abba5edf5/42003_2025_8684_Fig2_HTML.jpg" loading="lazy" id="d33e443" height="619" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>ATP production rate in 20 ng/mL (<strong>a</strong>, <strong>e</strong>) and 200 ng/mL (<strong>b</strong>, <strong>f</strong>) of GlcSph and lyso-Gb3 exposures respectively. Glycolytic rate in 20 ng/mL (<strong>c</strong>, <strong>g</strong>) and 200 ng/mL (<strong>d</strong>, <strong>h</strong>) of GlcSph and lysoGb3 exposures respectively. Induced measurement denotes inhibition of pyruvate entry into mitochondria in the ATP analyses, compensatory measurement denotes total OXPHOS inhibition by competitive inhibition of hexokinase by deoxyglucose in glycolysis analyses. DMSO was used as a vehicle control. Box plots show mean ±25% (boxes) and max. and min. values (whiskers) (<em>n </em>= 3 for (<strong>a</strong>–<strong>h</strong>). Statistical significance was determined by a 2-way ANOVA followed by a Sidak’s post-hoc test. The activity of <strong>i</strong> complex I (<em>n </em>= 5), <strong>j</strong> complex II&amp;III (<em>n </em>= 4), and <strong>k</strong> citrate synthase (<em>n </em>= 4) was unaffected. Statistical significance was determined using a Kruskal–Wallis test with a Dunn’s post-hoc test. *<em>p </em>&lt; 0.5, **<em>p </em>&lt; 0.01. Source data are provided as a Supplementary Data file.</p></figcaption></figure><p id="Par12">In contrast, the disease control treatment with lyso-Gb3, at the low 20 ng/mL dose led to a significant 24% reduction in both basal (<em>p </em>= 0.0053) and induced (<em>p </em>= 0.0053) ATP production rates (Fig. <a href="#Fig2" class="usa-link">2e</a>) and suggests that lyso-Gb3 affects the mitochondrial ATP production and the immediate cellular response to the mitochondrial pyruvate entry inhibition. However, this was not the case when the cells were exposed to the high 200 ng/mL dose of lyso-Gb3, which demonstrated no effect on ATP production rate (Fig. <a href="#Fig2" class="usa-link">2f</a>). When looking at the glycolytic rate we observe a different picture. Lyso-Gb3 did not affect glycolysis at the low dose compared to the DMSO control, although a 31% reduction was observed at the 200 ng/mL dose. However, with the total inhibition of mitochondrial OXPHOS (compensatory measurement) lyso-Gb3 treated cells at the 20 ng/mL dose were able to upregulate glycolysis only by 24% <em>vs</em> 41% compared to DMSO control (Fig. <a href="#Fig2" class="usa-link">2g</a>). The same was observed at the high 200 ng/mL dose where lyso-Gb3 treated cells upregulated compensatory glycolysis only by 40% <em>vs</em> 79% observed in the DMSO control (Fig. <a href="#Fig2" class="usa-link">2h</a>). This implies that lyso-Gb3 exposure impaired the cell’s ability to compensate via glycolysis for the inhibited OXPHOS in opposite to GlcSph exposed cells.</p></section><section id="Sec5"><h3 class="pmc_sec_title">GlcSph exposure does not affect the activity of mitochondrial complexes I – III</h3>
<p id="Par13">As proteomic profiling of SH-Sy5y cells exposed to GlcSph at both concentrations demonstrated dysregulation of ETC complex proteins the activity of complex I, which is impaired in both GD and PD, was investigated further<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>–<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Citrate synthase being the first rate limiting enzyme of the TCA cycle is often used as a marker of mitochondrial content and its activity was also measured and used for subsequent normalisation of the activity of complex I and II/III<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. The analysis demonstrated that citrate synthase activity itself was unaffected and stable. Overall, there was no significant difference in the activity of complex I-III between GlcSph treated and control cells (Fig. <a href="#Fig2" class="usa-link">2</a>i, j). However, both analyses demonstrated highly variable activity between biological replicates particularly for complex II/III which could not be explained. Technical causes were ruled out as the same effect was observed when repeated, with stable activity of citrate synthase observed in the DMSO control group (Fig. <a href="#Fig2" class="usa-link">2k</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">GlcSph exposure does not affect the cell’s ability to maintain its cellular energy charge</h3>
<p id="Par14">To ascertain how the reduction in ATP production rate in GlcSph exposure might affect the bioenergetic status of the cell we measured adenylate energy charge using HPLC-chemical detection. No difference in adenylate energy charge was observed in both GlcSph exposures (Fig. <a href="#Fig3" class="usa-link">3a</a>) and suggests that despite the negative effect of GlcSph on the ATP production rate the cell is still able to maintain higher cellular ATP levels relative to ADP and AMP. This is likely compensated by the upregulated glycolysis observed (Fig. <a href="#Fig2" class="usa-link">2c, d</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. The effect of GlcSph on cellular stress in SH-Sy5y cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371010_42003_2025_8684_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d865/12371010/19f9f3a19777/42003_2025_8684_Fig3_HTML.jpg" loading="lazy" id="d33e535" height="820" width="676" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>GlcSph exposure does not affect <strong>a</strong> adenylate energy charge (<em>n </em>= 4), and the activity of <strong>b</strong> cytosolic and <strong>c</strong> mitochondrial malate dehydrogenases (<em>n </em>= 6) in SH-Sy5y cells. <strong>d</strong> GlcSph exposure elicits an increase in the activity of glutathione reductase at 200 ng/mL (<em>n </em>= 9). DMSO was used as a vehicle control. Box plots show mean ±25% (boxes) and max. amd min. values (whiskers). Statistical significance was determined using a Kruskal–Wallis test with a Dunn’s post-hoc test. *<em>p </em>&lt; 0.5. Source data are provided as a Supplementary Data file.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">GlcSph exposure does not affect malate-aspartate shuttle and malate dehydrogenase activity</h3>
<p id="Par15">One way the cell’s energy could be affected is through NAD+ availability. Proteomics analysis indicated altered expression of mitochondrial malate dehydrogenase, therefore we investigated the functional effect of GlcSph on malate dehydrogenase activity.</p>
<p id="Par16">We observed no difference in the activity of cytosolic malate dehydrogenase between GlcSph treated and control cells (Fig. <a href="#Fig3" class="usa-link">3b</a>), which suggests the availability of NADH for its activity was not altered. Although a diminished trend (approx. 18%) was observed for the activity of mitochondrial malate dehydrogenase between GlcSph treated and control cells this decrease was not significant (Fig. <a href="#Fig3" class="usa-link">3c</a>). These results suggest that GlcSph exposure does not affect malate-aspartate shuttle and regeneration of NADH in mitochondria and its consumption in the cytosol.</p></section><section id="Sec8"><h3 class="pmc_sec_title">GlcSph exposure induces cellular stress that is compensated in the short term</h3>
<p id="Par17">Mitochondrial dysfunction and reduced ATP production indicate potential for cellular stress caused by GlcSph. Proteomic analysis indicated upregulation of glutathione reductase (Supplementary Table <a href="#MOESM1" class="usa-link">S5</a>), therefore its activity was assessed. Glutathione reductase is an important enzyme in the glutathione cycle – the main cellular antioxidant defence system. Glutathione is reduced by glutathione reductase using NADPH as an electron donor and in its reduced form (GSH) assists in clearance of hydrogen peroxide and lipid peroxides. GSH also donates electrons to ER resident protein isomerases required for proper protein folding.</p>
<p id="Par18">The analysis demonstrated that the activity of glutathione reductase was increased in both low and high dose GlcSph exposure although this increase was only statistically significant at the high 200 ng/mL dose (<em>p </em>= 0.041) (Fig. <a href="#Fig3" class="usa-link">3d</a>). This result suggests there may be an elevation in reactive oxygen species in GlcSph treated cells.</p></section><section id="Sec9"><h3 class="pmc_sec_title">GlcSph specifically binds and causes ubiquitination of tubulin species</h3>
<p id="Par19">Proteomic analysis indicated protein ubiquitination was a pathway upregulated by exposure to both lyso-lipids. We previously reported, that lyso-Gb3 elicited a greater effect on this pathway, however, the proteins affected by either lipid differ indicating specific lyso-lipid effects<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. To explore which proteins have a high affinity for GlcSph and vice versa, and potentially are prone to ubiquitination we immobilised GlcSph onto magnetic beads and incubated with SH-Sy5y cell lysates. Of note, immobilisation of lyso-lipids is expected to obscure their free amine group as it reacts with the epoxy group on magnetic beads. This may have affected the proteins immobilised GlcSph bound to, and is a limitation of this study. However, the effects on the cell proteome and associated disease phenotypes of the two lyso-lipids are so different we suspect that the differences are largely caused by the different sugar moieties and not just the common amine group that both lyso-lipids have. Hence why a comparison with blank beads and lyso-Gb3 was considered the best way to look for specific effects of GlcSph. Cellular proteins that were ubiquitinated after GlcSph and lyso-Gb3 exposure were isolated using an antibody that recognised mono- and polyubiquitin chains on proteins and captured proteins were analysed by label free proteomics. The analysis demonstrated that GlcSph bound 14 proteins listed in Table <a href="#Tab1" class="usa-link">1</a>, of which only two proteins, namely tubulin alpha 3C and alpha enolase bound exclusively. We further assessed if GlcSph binding to alpha tubulin and alpha enolase may affect their function and cause their ubiquitination. The ubiquitinated immunoprecipitation protein analysis demonstrated that out of the 54 ubiquitinated proteins observed exclusively in the GlcSph exposure both alpha and beta tubulins were the most abundant (Supplementary Table <a href="#MOESM1" class="usa-link">S6</a>, Fig. <a href="#Fig4" class="usa-link">4a</a>), which was different to the lyso-Gb3 exposure (Fig. <a href="#Fig4" class="usa-link">4b</a>). Tubulins were also more ubiquitinated in GlcSph treated cells compared to the DMSO control and lyso-Gb3 treatment, particularly the core tubulins of microtubules TUBB (6-fold), TUBB3 (7-fold) and TUBA1B (4-fold) <em>vs</em> control (Fig. <a href="#Fig4" class="usa-link">4c</a>) thereby supporting proteomic observations that also indicated dysregulation of tubulin species in GlcSph treated cells (Fig. <a href="#Fig4" class="usa-link">4d</a>). These results suggest that GlcSph exposure may specifically render tubulin species less functional or difficult to fold, which could subsequently affect microtubule dynamics and neuronal transport. Gene ontology analysis of the 54 ubiquitinated proteins exclusively observed in the GlcSph exposure also demonstrated enrichment in the ubiquitination of tubulins, chaperones of the 70 kDa family and ATP binding proteins (Table <a href="#Tab2" class="usa-link">2</a>) indicating a possible problem with protein synthesis and folding.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>The fourteen proteins that bound GlcSph</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Gene name</th>
<th colspan="1" rowspan="1">Protein</th>
<th colspan="1" rowspan="1">Sub cellular location</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">TUBA3C</td>
<td align="left" colspan="1" rowspan="1">Tubulin alpha-3C chain</td>
<td align="left" colspan="1" rowspan="1">Cytosolic,</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ENO1</td>
<td align="left" colspan="1" rowspan="1">Alpha enolase</td>
<td align="left" colspan="1" rowspan="1">Cytosolic, Cell membrane</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HCD2</td>
<td align="left" colspan="1" rowspan="1">3-hydroxyacyl-CoA dehydrogenase type-2</td>
<td align="left" colspan="1" rowspan="1">Mitochondria</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LYPLA1</td>
<td align="left" colspan="1" rowspan="1">Acyl protein thioesterase 1</td>
<td align="left" colspan="1" rowspan="1">Cytosolic, CM, NM, ER</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SLC25A5</td>
<td align="left" colspan="1" rowspan="1">ADP/ATP translocase 2</td>
<td align="left" colspan="1" rowspan="1">Mitochondria IM</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SLC25A6</td>
<td align="left" colspan="1" rowspan="1">ADP/ATP translocase 3</td>
<td align="left" colspan="1" rowspan="1">Mitochondria IM</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LDHB</td>
<td align="left" colspan="1" rowspan="1">L-lactase dehydrogenase beta chain</td>
<td align="left" colspan="1" rowspan="1">Cytosolic, Mitochondria IM</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ACTB</td>
<td align="left" colspan="1" rowspan="1">Actin cytoplasmic</td>
<td align="left" colspan="1" rowspan="1">Cytosolic, cytoskeleton</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PHB1</td>
<td align="left" colspan="1" rowspan="1">Prohibitin</td>
<td align="left" colspan="1" rowspan="1">Cytosolic, Mitochondria IM, Nucleus</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HADHA</td>
<td align="left" colspan="1" rowspan="1">Trifunctional enzyme subunit alpha</td>
<td align="left" colspan="1" rowspan="1">Mitochondria IM</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HNRNPU</td>
<td align="left" colspan="1" rowspan="1">Heterogeneous nuclear ribonuclear protein U</td>
<td align="left" colspan="1" rowspan="1">Cytosolic, Nucleus, Cytoskeleton</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PCB1</td>
<td align="left" colspan="1" rowspan="1">Poly(r)C-binding protein 1</td>
<td align="left" colspan="1" rowspan="1">Cytosolic, Nucleus</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PCB2</td>
<td align="left" colspan="1" rowspan="1">Poly(r)C-binding protein 2</td>
<td align="left" colspan="1" rowspan="1">Cytosolic, Nucleus</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H2B1K</td>
<td align="left" colspan="1" rowspan="1">Histone H2B type K-1</td>
<td align="left" colspan="1" rowspan="1">Nucleus</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p25"><p><em>CM</em> Cell membrane, <em>IM</em> inner membrane.</p></div></div></section><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. GlcSph exposure increases ubiquitination of tubulin species in SH-Sy5y cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371010_42003_2025_8684_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d865/12371010/edfc2e180beb/42003_2025_8684_Fig4_HTML.jpg" loading="lazy" id="d33e621" height="561" width="800" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Pie chart showing tubulin species are ubiquitinated the most among the proteins exclusive to GlcSph exposure in contrast to <strong>b</strong> ubiquitinated proteins exclusive to lyso-Gb3 exposure. <strong>c</strong> Tubulin species observed to be differentially ubiquitinated in GlcSph and lyso-Gb3 treated cells vs control. <strong>d</strong> Proteomic analysis demonstrating altered abundance of tubulin species in GlcSph treated cells. Box plots show mean ±25% (boxes) and max. and min. values (whiskers) (<em>n </em>= 2). <strong>e</strong> Polymerisation assay demonstrating no effect of GlcSph at either 20 ng/mL or 200 ng/mL on any of the polymerisation phases. Source data are provided as a Supplementary Data file.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Summary of the GO analyses of the ubiquitinated proteins unique to GlcSph exposure</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">PANTHER cellular component</th>
<th colspan="1" rowspan="1">Homo sapiens (REF)</th>
<th colspan="1" rowspan="1">Found</th>
<th colspan="1" rowspan="1">Expected</th>
<th colspan="1" rowspan="1">Fold enrichment</th>
<th colspan="1" rowspan="1">Raw <em>p</em>-value</th>
<th colspan="1" rowspan="1">FDR</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Intermediate filament cytoskeleton</td>
<td align="left" colspan="1" rowspan="1">51</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">0.13</td>
<td align="center" colspan="1" rowspan="1">39.58</td>
<td align="left" colspan="1" rowspan="1">2.62E−07</td>
<td align="left" colspan="1" rowspan="1">3.34E−05</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cytoskeleton</td>
<td align="left" colspan="1" rowspan="1">751</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="center" colspan="1" rowspan="1">1.86</td>
<td align="center" colspan="1" rowspan="1">4.84</td>
<td align="left" colspan="1" rowspan="1">9.01E−05</td>
<td align="left" colspan="1" rowspan="1">2.07E−03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ribosomal subunit</td>
<td align="left" colspan="1" rowspan="1">80</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">0.20</td>
<td align="center" colspan="1" rowspan="1">20.19</td>
<td align="left" colspan="1" rowspan="1">5.51E−05</td>
<td align="left" colspan="1" rowspan="1">1.29E−02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PANTHER molecular function</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="center" colspan="1" rowspan="1"></td>
<td align="center" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Misfolded protein binding</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">0.4</td>
<td align="center" colspan="1" rowspan="1">75.69</td>
<td align="left" colspan="1" rowspan="1">1.34E−05</td>
<td align="left" colspan="1" rowspan="1">4.57E−04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Heat shock protein binding</td>
<td align="left" colspan="1" rowspan="1">33</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">0.8</td>
<td align="center" colspan="1" rowspan="1">36.7</td>
<td align="left" colspan="1" rowspan="1">9.58E−05</td>
<td align="left" colspan="1" rowspan="1">2.91E−03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ATP binding</td>
<td align="left" colspan="1" rowspan="1">41</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">0.10</td>
<td align="center" colspan="1" rowspan="1">39.39</td>
<td align="left" colspan="1" rowspan="1">4.57E−06</td>
<td align="left" colspan="1" rowspan="1">1.66E−04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PANTHER protein class</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="center" colspan="1" rowspan="1"></td>
<td align="center" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Hsp70 chaperone</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">0.4</td>
<td align="center" colspan="1" rowspan="1">80.74</td>
<td align="left" colspan="1" rowspan="1">1.13E−05</td>
<td align="left" colspan="1" rowspan="1">2.21E−03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Tubulin</td>
<td align="left" colspan="1" rowspan="1">24</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">0.6</td>
<td align="center" colspan="1" rowspan="1">50.46</td>
<td align="left" colspan="1" rowspan="1">3.99E−05</td>
<td align="left" colspan="1" rowspan="1">3.91E−03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ribosomal protein</td>
<td align="left" colspan="1" rowspan="1">177</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">0.44</td>
<td align="center" colspan="1" rowspan="1">11.40</td>
<td align="left" colspan="1" rowspan="1">8.53E−05</td>
<td align="left" colspan="1" rowspan="1">5.57E−03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PANTHER pathway</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="center" colspan="1" rowspan="1"></td>
<td align="center" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Serine glycine biosynthesis</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">0.1</td>
<td align="center" colspan="1" rowspan="1">&gt;100</td>
<td align="left" colspan="1" rowspan="1">1.66E−04</td>
<td align="left" colspan="1" rowspan="1">3.79E−03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Parkinson disease</td>
<td align="left" colspan="1" rowspan="1">101</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">0.25</td>
<td align="center" colspan="1" rowspan="1">15.99</td>
<td align="left" colspan="1" rowspan="1">1.31E−04</td>
<td align="left" colspan="1" rowspan="1">4.21E−03</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par20">With this observation we investigated the possibility of GlcSph potential disruption of tubulin polymerisation, which would disrupt microtubule assembly. Tubulin polymerisation reaction in vitro was performed using a fluorescence-based kit (Cytoskeleton, BK011P). The analysis demonstrated GlcSph had no effect on any of the phases of microtubule formation (nucleation, growth or equilibrium) at the three concentrations tested (Fig. <a href="#Fig4" class="usa-link">4e</a>). This suggests that GlcSph exposure may increase tubulin turnover rather than affect microtubule formation/assembly.</p>
<p id="Par21">The assessment of alpha enolase activity demonstrated no differences between GlcSph treated and control cells and suggests the interaction may be transient (Supplementary Fig. <a href="#MOESM1" class="usa-link">S2</a>).</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par22"><em>GBA1</em> mutations represent the greatest known genetic risk factor for developing PD<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, whose hallmarks are mitochondrial dysfunction, aggregation and toxicity of a-synuclein protein and loss of dopaminergic neurons<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. The activity of the GBA1 enzyme in the brain of PD patients is also reduced irrespective of the <em>GBA1</em> mutation status<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Almost undetectable in healthy cells GlcSph accumulates in both GBA1 carriers and GD patients and has been shown to promote a-synuclein aggregation and toxicity<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, and loss of dopaminergic neurons<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. We sought to investigate the effect of GlcSph on mitochondrial function. Proteomic analysis of SH-Sy5y cells exposed to GlcSph demonstrated altered metabolism even at the lower mild levels observed in GD phenotypes. These changes, shown in Fig. <a href="#Fig1" class="usa-link">1c-d</a>, indicated activated TCA cycle (z-score = 2.4) and mitochondrial dysfunction. The higher level typically observed in severe GD phenotypes indicated activated glycolysis (z-score = 2) and mitochondrial dysfunction. We have confirmed this specific effect of GlcSph on cell metabolism using functional assays to demonstrate changes in ATP production and glycolytic rates in live cell analyses further to support our initial proteomic findings. Our findings show that glycolysis can rescue reduced ATP production rate at the low dose but not adequately at the high dose. As suggested by the proteomic analysis (Fig. <a href="#Fig1" class="usa-link">1e</a>) the TCA cycle may also be contributing to the maintenance of the ATP production rate and warrants further investigation. This contrasted with the disease control lyso-lipid exposure to lyso-Gb3, which demonstrated a different effect on cell metabolism where a 24% reduction in the ATP production rate was observed at the low 20 ng/mL dose (Fig. <a href="#Fig2" class="usa-link">2e</a>). Curiously, no effect at the higher dose was observed indicating a low dose of lyso-Gb3 may be more potent on cell function and suggests there may be some cellular adaptation long term that compensates. We previously observed this effect of lyso-Gb3 on protein ubiquitination<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Intriguingly, lyso-Gb3 had an opposite effect on the glycolytic rate compared to GlcSph where a 31% reduction was observed during OXPHOS inhibition and indicates the cell was not able to compensate adequately (Fig. <a href="#Fig2" class="usa-link">2</a>g, h). Furthermore, compensatory glycolysis for the inhibited OXPHOS was also hampered in lyso-Gb3 exposed cells compared to control. These results suggest that each lyso-lipid has a specific and opposite effect on the ATP production and glycolytic rates whereby GlcSph induces a shift toward glycolysis while lyso-Gb3 does not. This could be due to the significant differences in the lipid structures (lyso-Gb3 is bulkier and more hydrophilic), their ability to cross cellular membranes, be transported in the cell and accumulate in different cellular compartments. They also demonstrate that GlcSph exposure could contribute to the metabolic changes observed in GD and possibly GBA associated PD. Decreased cellular ATP levels are observed in fibroblasts of patients with familial PD<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, PD patients’ brains<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, and platelets<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>, but this effect has been attributed to reduced mitochondrial complex I activity. Reduced ATP levels have also been observed in a neuronopathic GD mouse model<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> and peripheral blood mononuclear cells of GD patients<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, with reduced complex I-III activity in the brains of Gba null mouse model of neuronopathic GD<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. We attempted to confirm if complex I and II/III activity could be the cause for reduced ATP production demonstrated in this study, and increased stress indicated by the proteomic analysis, however, GlcSph exposure resulted in highly variable activity of complex II/III compared to stable activity of the DMSO control (Fig. <a href="#Fig2" class="usa-link">2i, j</a>). The same effect was observed on several occasions when experiment was repeated, so whilst we cannot confirm a significant effect on complex II/III activity it does appear there is some variable effect which could be due to how the lipid behaves/transports within the cell or due to the cell type used in this study. Thus whilst we demonstrate metabolic reprogramming with the shift toward glycolysis and reduction in the ATP production rate that is associated with loss of complex I in dopaminergic neurons in mice<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, further investigations are necessary to determine whether GlcSph truly does not affect the activity of complexes I-III in other, more vulnerable cell types such as dopaminergic neurons that are more reliant on OXPHOS<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Increased glycolysis as observed in this study has been suggested to be not only a compensatory but a protective mechanism in various neurodegenerative diseases including PD<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> and can protect against a-synuclein proteotoxic effect<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> and cellular stress via enhanced chaperone activity of Hsp90<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>.</p>
<p id="Par23">Despite the negative effect of GlcSph on the ATP production rate our findings show the cell appears to be able to maintain its cellular energy charge, i.e. the ability of the cell to maintain its ATP levels higher than those of ADP and AMP as this was unaffected (Fig. <a href="#Fig3" class="usa-link">3a</a>). This may be due to an increased expression of the mitochondrial creatine and adenylate kinases, voltage dependent anion channels (VDACs) and phosphate carrier proteins as was suggested by the proteomic analysis (Supplementary Table <a href="#MOESM1" class="usa-link">S5</a>) and whose job is to facilitate cytosolic ATP availability quickly and maintain global and local ATP/ADP ratios. These data also suggest a well-controlled balance between the ATP producing and ATP consuming reactions, possibly due to switching off ATP consuming processes such as protein synthesis, folding and degradation and increase in catabolic processes such as amino and fatty acid breakdown. Incidentally, pathway analysis demonstrated tRNA charging and protein ubiquitination pathways as the most significantly altered in the 200 ng/mL dose exposure (Fig. <a href="#Fig1" class="usa-link">1e</a>). Gene ontology analysis also indicated misfolded and heat shock protein binding as enriched functions together with heat shock proteins as an enriched class (Table <a href="#Tab2" class="usa-link">2</a>) thereby indicating altered protein homeostasis and adding weight to this hypothesis.</p>
<p id="Par24">In the attempt to ascertain what caused the reduction in the ATP production rate we probed the malate-aspartate shuttle that maintains mitochondrial NAD and cytosolic NADH availability and tested the activity of both cytosolic and mitochondrial malate dehydrogenases involved in the NADH oxidation in the cytosol and its regeneration in the mitochondria respectively. However, the analyses demonstrated no effect of GlcSph exposure on the activity of either enzyme (Fig. <a href="#Fig3" class="usa-link">3b, c</a>). Whilst only two proteins specifically bound GlcSph we did observe proteins that bound both lyso-lipids (Table <a href="#Tab1" class="usa-link">1</a>), six of which were localised to the mitochondria including the inner membrane. Two of these proteins were ADP/ATP translocase 2 &amp; 3 which facilitate ADP/ATP transport in and out of the mitochondria and when disrupted can lead to impaired OXPHOS. Proteomic analysis indicated upregulation of both translocases in GlcSph exposure but only at the higher dose the increase was significant (Supplementary Table <a href="#MOESM1" class="usa-link">S5</a>). This increase may be due to GlcSph binding and rendering the translocase less functional. SLC25A5 is also a necessary component for parkin-mediated mitophagy<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> and loss or reduction of its activity may also lead to mitochondrial dysfunction in GlcSph exposure as suggested by the pathway analysis (Fig. <a href="#Fig1" class="usa-link">1d, e</a>). Whilst the two lyso-lipids bind to ADP/ATP transporters they may have unique, specific effects thus affecting OXPHOS in different ways and this observation warrants further investigation.</p>
<p id="Par25">Another specific effect observed from GlcSph exposure was the binding of tubulin. We determined that GlcSph does not affect microtubule polymerisation in vitro but likely affects tubulin turnover as it elicited greater levels of ubiquitination of tubulins compared to lyso-Gb3 (Fig. <a href="#Fig4" class="usa-link">4a, b</a>). Tubulin is known to associate with VDACs and modulate their permeability thereby affecting ADP/ATP exchange<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Upregulation of VDACs as suggested by the proteomics analysis (Supplementary Table <a href="#MOESM1" class="usa-link">S5</a>) may indicate the cells attempt to increase ADP/ATP exchange. Increased ubiquitination of tubulins indicating their increased turnover and thus altered modulation of VDACs further supports this observation. Microtubule disruption that may result from changes in tubulin turnover can affect many cellular processes and in GD animal models altered lysosome distribution is attributed to cytoskeletal disruption, which occurs in the early stages of disease<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Altered lysosomal distribution can impair lysosome function and lead to neurodegenerative disease<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Changes in tubulin can cause trafficking defects and lead to adult onset movement disorder in mice<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>, disrupted mitochondrial transport in human cells<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, and even cause PD<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Changes in TUBA1C, the most abundant ubiquitinated protein unique to GlcSph dataset in this study (Fig. <a href="#Fig4" class="usa-link">4a</a>), have been demonstrated in rotenone-induced rat PD model<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, highlighting the toxic role GlcSph may play in GBA related PD.</p>
<p id="Par26">A limitation of our study is understanding the combined effect of the GlcCer accumulation that also occurs in GBA1 mutant cells. Our work clearly defines the specific effect of GlcSph but the effect of GlcCer accumulation could also contribute and exacerbate some of the findings we describe here and observed in GD. Previous work has addressed both lipids using a double knock out zebrafish model for <em>Gba1</em> with and without acid ceramidase knocked out<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> This study showed that accumulation of these two lipids does lead to specific effects. GlcCer accumulation was found to be attributed more to neuroinflammation and the absence of GlcSph in <em>Gba1</em> mutants did lead to reduced disease severity with increased lifespan. An interesting finding was the loss of dopaminergic neurons, which was slowed down in the double knock out mutants, confirming GlcSph effect on vulnerable neuronal cells over time. Our findings of how GlcSph specifically affects mitochondria likely explains why these high energy demanding dopaminergic neurons are affected more by GlcSph than GlcCer accumulation.</p>
<p id="Par27">In summary, our data show that GlcSph is neurotoxic and its mechanism may be due to altering pathways responsible for ATP production and concomitant increase in oxidative stress. While the cell is able to maintain its cellular charge and compensate by upregulating glycolysis, a rapid but inefficient way of making ATP, this may be sufficient in a short-term situation, which we can only study in this system. However, these cellular deficiencies and a reduced capacity of neuroprotective mechanisms with age or in a long-term chronic exposure may eventually lead to damage of vulnerable cells such as neurons and potentially answer some of the questions regarding the disease mechanisms associated with GD and GBA related PD.</p>
<p id="Par28">Supplementary information accompanies the manuscript on the Communications Biology website <a href="http://www.nature.com/commsbio" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.nature.com/commsbio</a>.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Cell culture</h3>
<p id="Par29">SH-Sy5y cells were purchased from the European Collection of Cell Cultures (Public Health England, UK). Cells used in this study were between passage 8 and 15. Cells were incubated in DMEM/F12 (1:1) supplemented with 10% human serum (Sigma, UK) due to high level of endogenous glucosylsphingosine present in standard 10% FBS supplemented culture medium. Cell viability was assessed using AlamarBlue™ HS cell viability reagent (Invitrogen, Thermo Fisher) as per manufacturer’s instructions. Cells were plated in 6 well plates at the density of 3 ×105 cells/well and treated with lyso-Gb3/GlcSph (Avanti lipids) dissolved in DMSO or vehicle for 24 h at the 20 ng/mL dose and 72 h at the 200 ng/mL dose. Cells were harvested using trypsin and centrifuged at 500 × <em>g</em> for 5 min, and further washed with PBS.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Label Free proteomics and deep proteomic phenotyping</h3>
<p id="Par30">Pelleted cells were lysed in PBS by three freeze-thaw cycles using 37 °C bath and dry ice. Cell lysate was ice-cold acetone precipitated at a volume of 1:3 and left at –20 °C for 4 h, then centrifuged to pellet precipitated proteins. The supernatant was removed and the protein pellet air dried. Precipitated protein was digested as described previously<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Briefly, protein was solubilised by the addition of 20 µL of 100 mM Tris buffer (pH 7.8) containing 6 M urea, 2 M thiourea and 2% (w/v) amidosulfobetain-14. Disulphide bridges in proteins were broken by the addition of 45 µg of dithioerythritol prepared fresh in 100 mM Tris buffer, pH 7.8 and left to shake for 60 min. Carbamidomethylation of cysteines was performed by the addition of 108 µg of iodoacetamide prepared fresh in buffer as before, and samples incubated in the dark for 45 min. To reduce urea concentration in the samples to below 1 M 160 µL of Milli-Q water was added to samples. 1 µg of trypsin Gold (Promega, Germany) was added to each sample and incubated at 37 °C for 16 h. Peptides were subject to high pH, low pH fractionation using Isolute 100 mg C18 cartridges (Biotage, Sweden) and eluted into 5 fractions at 6–50% ACN. Peptide eluents were dried by centrifugal evaporation and resuspended at a concentration of 300 ng/µL in 3% ACN, 0.1% FA.</p>
<p id="Par31">Ultra-definition mass spectrometry (UDMS<sup>E</sup>) analysis was performed as previously described<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Briefly, 0.3 µg of peptides were loaded onto a Waters NanoAquity Liquid Chromatography (LC) system and separated over 60 min on a 75 µm × 150 mm, 1.7 µm Peptide BEH C18 analytical column (Waters, UK), then injected into a Synapt-G2-Si mass spectrometer with Ion Mobility Separation (IMS) (Waters, UK).</p>
<p id="Par32">Raw data was imported to Progenesis QI version 4.1 (Waters, UK) and each fraction was processed separately before all five fractions were combined into one experiment. MS/MS spectra in the MS raw data files were searched against the UniProt most recent published human reference proteome database with manual addition of porcine trypsin (<a href="https://www.ncbi.nlm.nih.gov/protein/P00761" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P00761</a>). Enzyme digestion was set to trypsin, maximum missed cleavage number was set to 3. Cysteine carbamidomethylation was set as a fixed modification, methionine oxidation, protein N-terminal acetylation, glutamine deamidation and protein N-terminal pyrrolidone carboxylation were set as variable modifications. False discovery Rate (FDR) cut off was set at 1% at both peptide and protein level. Protein quantification was performed using unique peptides only. The resulting protein identifications and quantitative data were exported to Excel (Microsoft) for further analysis.</p></section><section id="Sec14"><h3 class="pmc_sec_title">The immobilisation and the identification of lyso-lipid interacting proteins</h3>
<p id="Par33">Glucosylsphingosine and lyso-Gb3 standards in MeOH were evaporated by centrifugation, then glycosphingolipids were reconstituted in 150 mM phosphate buffered saline (PBS, pH 7.4) to a final concentration of 1 mg/mL. Five milligrams of Dynabeads® M-270 epoxy (Invitrogen, UK) were resuspended in 2 mL of dimethylformamide organic buffer, vortexed and aliquoted into 7 vials (blank, sample and a duplicate) at 3.3 ×10<sup>8</sup> beads per vial. Buffer was removed from the beads using a magnet. Beads were washed with 1 mL of 150 mM PBS (pH 7.4), resuspended in 0.1 mL of 100 mM sodium phosphate buffer (pH 7.4) and vortexed. One hundred microlitres of 1 mg/mL lysoGb3, glucosylsphingosine, or PBS (blank) was added to each aliquot of beads. To enhance glycosphingolipid-bead binding 0.1 mL of ammonium sulphate buffer (1 M final concentration) was added. The mixture was incubated at room temperature while mixing on a rotor for 24 h. The beads with bound lyso-glycosphingolipids were placed on a magnet and supernatants removed. The beads were washed with 0.5 mL of 150 mM PBS (pH 7.4) containing 2.5 mg/mL of blocking agent horse myoglobin and further washed in PBS. Lyso-glycosphingolipid bound beads were resuspended in 0.1 mL of 100 mM sodium phosphate buffer (pH 7.4), 0.035 mg/mL protein from SH-SY5Y cell lysate diluted in 150 mM PBS (pH 7.4) added and incubated at room temperature for 1 hour while mixing on the rotor. Supernatants were removed and beads washed with 0.5 mL of 150 mM PBS containing 0.1% Triton-100 (pH 7.4) for 15 min. Triton wash was removed and beads were washed with 150 mM PBS (pH 7.4) twice. Protein-glycosphingolipid bound partners were eluted from the beads by the addition of 40 µL of 100 mM Tris (pH 7.8), containing 6 M urea, 2 M thiourea, 0.2% ABS-14 (w/v), subjected to in-solution digestion without fractionation and LC-MS/MS proteomics analysis<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Data were imported to ProteinLynx Global SERVER v3 (Waters, UK), protein identification was performed using UniProt’s most recent published human reference proteome FASTA database (<a href="https://www.uniprot.org/proteomes/UP000005640" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.uniprot.org/proteomes/UP000005640</a>) with manual addition of porcine trypsin (<a href="https://www.ncbi.nlm.nih.gov/protein/P00761" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P00761</a>) and horse myoglobin (<a href="https://www.ncbi.nlm.nih.gov/protein/P68082" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P68082</a>). Protein quantitative data were exported to Excel (Microsoft) for further analysis. Proteins identified were considered to interact with a glycosphingolipid if they were not detected in the blank – those found to interact with the beads without glycosphingolipid bound to the beads.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Ubiquitinated protein analysis</h3>
<p id="Par34">Mouse IgG1 antibody that recognises mono and polyubiquitin chains (Clone FK2, Sigma, UK) on proteins was diluted in 200 µL of PBS containing 0.02% of Tween 20 and coupled to protein G magnetic beads (Dynabeads®, Invitrogen, Sigma) at 4 µg antibody/0.5 mg beads overnight while on a rotor at 4 °C. The bead-antibody complex was placed on a magnet, the supernatant removed, the bead-antibody complex washed twice with 200 µL of PBS-Tween 20, then the bead-antibody complex was incubated with 250 µL horse myoglobin (2.5 mg/mL) for 15 min at room temperature while shaking to reduce nonspecific binding. Horse myoglobin was washed off twice with 200 µL of PBS-Tween 20. The bead-antibody complex was incubated for 1 hour at room temperature on the rotor with SH-SY5Y cell lysates prepared as described above. Bead-antibody-antigen complex was washed with 200 µL of PBS-Tween 20 twice, following which antibody-antigen complex was eluted from the beads with 40 µL of digest buffer containing 100 mM Tris (pH 7.8), 6 M urea, 2 M thiourea, 2% (w/v) ASB-14 while rotating for 2 min. Supernatants containing the antibody-antigen complex were subject to proteomic analysis<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Identification of proteins present in the FK2 antibody immunocapture was performed as described in the previous section with the only difference that mouse immunoglobulin gamma-1 chain C (<a href="https://www.ncbi.nlm.nih.gov/protein/P01868" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P01868</a>) from the commercial antibody was also added to the database. Relative protein quantitation was achieved based on the amount of mouse immunoglobulin gamma-1 chain C peptide added to beads. Data were exported to Microsoft Excel for further analysis.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Tubulin polymerisation assay</h3>
<p id="Par35">The assay was performed according to the manufacturer’s protocol using Cytoskeleton BK011P kit. GlcSph and lyso-Gb3 stocks in DMSO and DMSO buffer control were diluted with Milli-Q water at 10 x desired concentrations. Taxol in DMSO and colchicine were diluted in Milli-Q water at 30 µM. Infinite 200 plate reader (Tecan, Switzerland) was set into fluorescence (excitation 340 nm, emission 435 nm) top reading kinetic mode at 37 °C with a 30 s interval, 30 nm gain and 3 flashes per reading. A black, flat bottom, 96-well pate (Corning, costar, #3686) was placed into the reader for 10 min to warm up. DMSO, taxol, colchicine, and GlcSph and lysoGb3 samples were pipetted in duplicates into the plate and an assay buffer containing final concentration of GTP (1 mM), tubulin (2 mg/mL), glycerol (15%, v/v) was added to each well. The plate was immediately placed into the reader, shaken for 10 s in orbital mode, then fluorescence was recorded for 60 min. The data were exported into Excel (Microsoft Inc) for further processing and visualised using Graghpad, v 10.4 (Prism).</p></section><section id="Sec17"><h3 class="pmc_sec_title">Analysis of cellular energy charge</h3>
<p id="Par36">Five hundred millilitres of 1 M ice-cold perchloric acid was added to the cells, they were scraped and collected into chilled tubes. Two hundred microliters of 0.5 M Potassium Hydrogen Carbonate in 1 M Potassium Hydroxide were combined with 250 µL of cell extract, vortexed, centrifuged at 14,000 × <em>g</em> for 5 min at 4 °C and the supernatant collected into a clean tube. This was stored at −80 °C until derivatisation. One hundred µL of 1 M sodium acetate, pH 4.5 and 10 µL of 4 M chloracetaldehyde were added to 100 µL of cell extract. The mixture was incubated at 60 °C for 40 min and the reaction quenched by putting the sample on ice. Once cool the samples were analysed at 4 °C using High-Performance Liquid Chromatography (HPLC) coupled to fluorescence detector. HPLC equipment consisted of PU-1580 intelligent pumps, AS-950 autosampler and DG-1580-53 in-line mobile phase degasser (Jasco Inc.). Etheno-adenine nucleotides were chromatographically separated over a 150 mm × 4.6 mm, 3 µm ODS C18 analytical column (Hypersil) using a 100 - 0% 0.2 M potassium phosphate, pH 5 and 0.2 M potassium phosphate with 10% acetonitrile, pH 5 over a 31-min linear gradient at a flow rate of 0.8 mL/min. The FP-920 fluorescence detector (Jasco) was maintained at 290 nm excitation and 415 nm emission wavelength respectively and the data collected with EZChrom Elite v3.17 software (Jasco Inc.). Chromatographic peaks acquired for ATP, ADP and AMP were integrated using elution times of corresponding standards and were used to calculate adenylate energy charge (AEC) according to equation equation<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>:</p>
<table class="disp-formula p" id="Equ1"><tr><td class="formula"><math id="d33e1240" display="block"><mi>A</mi><mi>E</mi><mi>C</mi><mo>=</mo><mfrac><mrow><mfenced close="]" open="["><mrow><mi mathvariant="normal">ATP</mi></mrow></mfenced><mo>+</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mn>2</mn><mfenced close="]" open="["><mrow><mi mathvariant="normal">ADP</mi></mrow></mfenced></mrow></mfrac></mrow><mrow><mfenced close="]" open="["><mrow><mi>A</mi><mi>T</mi><mi>P</mi></mrow></mfenced><mo>+</mo><mfenced close="]" open="["><mrow><mi>A</mi><mi>D</mi><mi>P</mi></mrow></mfenced><mo>+</mo><mfenced close="]" open="["><mrow><mi>A</mi><mi>M</mi><mi>P</mi></mrow></mfenced></mrow></mfrac></math></td></tr></table></section><section id="Sec18"><h3 class="pmc_sec_title">Analysis of glutathione reductase activity</h3>
<p id="Par37">Cells in PBS were scraped into microcentrifuge tubes, PBS removed by centrifugation for 5 min at 4 °C at 1000 × <em>g</em>, 200 µL of ice-cold assay buffer added, the pellets centrifuged for 15 min at 10,000 × <em>g</em> at 4 °C and supernatants collected and stored at −80 °C until ready for analysis. The assay was performed using ab83461 GR assay kit (Abcam, UK) according to the manufacturer’s instructions. GR reduces GSSG, which in turn reacts with 5,5’-dithiobis-2-nitrobenzoic acid (DTNB) to generate a coloured product (TNB) with strong absorbance at 405 nm. Endogenous GSH was destroyed with 3% hydrogen peroxide (5 µL) added to the sample (100 µL) for 5 min at room temperature, followed by the treatment with catalase (5 µL) for another 5 min. A 0–50 nmol/well TNB standard curve was prepared using a TNB standard and ran alongside the samples, which were added to a 96-well plate (30 µL). A reaction mixture containing GR assay buffer, DTNB, GSSG and NADPH solutions was added to each well (50 µL) and the absorbance read immediately at 405 nm in an Infinite 200 plate reader at room temperature generating an initial reading (A1). The samples were incubated at first for 10 min and the absorbance read again in a kinetic mode with an interval of 1 min for the next hour. TNB standard curve was plotted and a change in absorbance between the initial reading and another time point determined to fall within the linear range of the reaction was applied to the standard curve to interpolate the amount of TNB in each sample. The activity of GR was calculated as follows: GR activity = (∆B*Sample dilution factor)/((T2–T1)*0.9*V) (nmol/min/mL), where ∆B (nmol) is the amount of TNB interpolated from the standard curve, T1 (min) is the time of the initial reading, T2 (min) is the time of the second reading, V (mL) is the sample volume and 0.9 is the sample volume change factor. The resulting value was normalised to total protein determined by the Bradford method to yield GR activity (nmol/min/mg).</p></section><section id="Sec19"><h3 class="pmc_sec_title">Analysis of malate dehydrogenase activity</h3>
<p id="Par38">Cells were collected by scaping. PBS was removed by centrifugation for 5 min at 4 °C at 1000 × <em>g</em> and pellets homogenised in 100 µL of ice-cold assay extraction buffer for 10 min on ice. The pellets were centrifuged at 10,000 × g for 5 min at 4 °C, the supernatants collected and stored at −80 °C until ready for analysis. The assay was performed using an MDH assay kit (Abcam, ab183305, UK) according to the manufacturer’s instructions. One hundred microlitres of ice-cold assay buffer was added to each pellet, placed on ice for 10 min and spun at 10,000 × <em>g</em> for 5 min at 4 °C, the supernatants collected and diluted 1 in 10 in the assay buffer. Ten microlitres of each sample dilution was aliquoted out for total protein measurement by the Bradford method. Samples were analysed in duplicates. A 6-point calibration curve containing 0–12.5 nmol of NADH standard was prepared in duplicates and sample blanks for each sample dilution was run alongside. The reaction mixture for the samples and standards contained samples/standards, assay buffer, enzyme mix, developer and substrate. Sample blanks had all the above except for the substrate. After an initial shake the plate was incubated at 37 °C in Infinite 200 plate reader, the initial absorbance measured at 450 nm, then the absorbance was measured in a kinetic cycle for 30 min every 5 min until the value of the most active sample was greater than the value of the highest standard. The activity of MDH was calculated as follows: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e1305"><mi mathvariant="normal">MDH activity</mi><mo>=</mo><mfrac><mrow><mi mathvariant="normal">Sa</mi></mrow><mrow><mfenced close=")" open="("><mrow><mi mathvariant="normal">Reaction Time</mi></mrow></mfenced><mi mathvariant="normal">x Sv</mi></mrow></mfrac></math></span> where Sa is the amount of NADH (nmol) generated in the sample between the final and initial time of reading (interpolated from the standard curve) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e1327"><mi mathvariant="normal">Reaction Time</mi></math></span> is the difference between the final and initial time of reading (min) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e1337"><mi mathvariant="normal">Sv</mi></math></span> is the sample volume used in the rection (mL). The resulting value was normalised to total protein to yield MDH activity (nmol/min/mg).</p></section><section id="Sec20"><h3 class="pmc_sec_title">Analysis of mitochondrial malate dehydrogenase activity</h3>
<p id="Par39">Samples were prepared in the same manner as for MDH assay with the exception that the pellets were incubated in 200 µL of assay extraction buffer for 20 min on ice. The sample protein was determined by the Bradford assay and each sample was diluted to contain 150 µg of protein. The assay was performed using MDH2 activity kit (ab119693, Abcam, UK) according to the manufacturer’s protocol. Samples were diluted with incubation buffer and together with a dilution series of a control sample (100 µL) were added to the wells to capture mitochondrial MDH2 and incubated at room temperature while shaking at 300 rpm for 3 h. The buffer was removed, samples washed with blocking buffer and the wash repeated. An activity solution (100 µL) containing sodium malate, NAD<sup>+</sup>, coupler, reagent dye and assay buffer giving the final concentration of malate and NAD<sup>+</sup> 5 mM and 4 mM respectively was added to the samples, absorbance read at 450 nm in a kinetic cycle for 25 min with a 20 s interval with shaking before and between readings. MDH2 activity was calculated using the Beer-Lambert law and the dye extinction coefficient was 37 mM<sup>−1</sup> cm<sup>−1</sup>.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Analysis of citrate synthase activity</h3>
<p id="Par40">CS activity was determined by a modified method based on the premise that upon the action of CS Coenzyme A and citrate are liberated and Coenzyme A reacts with DTNB, which causes a colorimetric change and can be monitored at 412 nm<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Cells were collected on ice and centrifuged at 4 °C, the PBS removed and the pellet re-suspended in 500 µL of 10 mM Tris buffer, pH 7.4 containing 320 mM sucrose and 1 mM EDTA. Each pellet was split into two 250 µL aliquots, one for the CS the other for complex I activity assays, and quickly snap frozen on dry ice before storage at −80 °C prior to analyses. Cell lysates were thawed and snap frozen twice using ethanol and dry ice and each sample cuvette was prepared to contain 950 µL of 100 mM Tris/ 0.1% Triton, (v/v), pH 8.0, 20 µL of cell homogenate, 10 µL of 10 mM acetyl-CoA and 10 µL of 20 mM DTNB. The corresponding reference cuvettes were prepared in the same way but had 960 µL of 100 mM Tris/0.1% Triton, (v/v). The cuvettes were covered with parafilm, gently inverted twice before they were put into a Kontron Uvikon 941 spectrophotometer (Nothstar Scientific) maintained at 30 °C. Ten microlitres of 20 mM oxaloacetate was added to the sample cuvettes only, gently mixed, and the reaction monitored at 412 nm for 30 min and measured every 30 s. CS activity was calculated using the Beer-Lambert Law, where path length was 1 cm, total volume was 1 mL and the extinction coefficient of DTNB was 13.6 mM<sup>−1</sup> cm<sup>−1</sup>. CS activity was expressed in nmol/min/mg.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Analysis of mitochondrial complex I activity</h3>
<p id="Par41">A modified method of Ragan et al. was used to determine complex I activity<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Each sample was prepared to contain 800 µL of 25 mM phosphate buffer containing 10 mM magnesium chloride, pH 7.2, 10 µL of 100 mM potassium cyanide, 50 µL of 50 mM BSA, 30 µL of 5 mM NADH, 20 µL of cell homogenate and 80 µL of water. The corresponding reference cuvettes were prepared in the same way but contained 90 µL of water. The cuvettes covered with parafilm were gently inverted twice before being put into a Kontron Uvikon 941 spectrophotometer maintained at 30 °C. Ten microlitres of Coenzyme Q<sub>1</sub> was added to the sample cuvettes only, gently mixed and left to reach 30 °C, and the reaction measured at 340 nm for 5 min at 30 s intervals. After 5 min elapsed 20 µL of 1 mM rotenone was added to the sample cuvettes only and after a 2–3 min wait the measurement continued for a further 5 min. Using the Beer-Lambert Law the concentration of oxidised NADH was calculated where the change in absorbance at 340 nm after rotenone addition was subtracted from the change in absorbance before rotenone addition, the extinction coefficient of NADH was 6.22 mM<sup>−1</sup> cm<sup>−1</sup>, path length was 1 cm and volume was 1 mL. The resulting value expressed as nmol/min/mg was divided by the value of CS activity to obtain a ratio.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Analysis of mitochondrial complex II/III activity</h3>
<p id="Par42">The activity of complex II/III was assayed based on the King method<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. The oxidation of succinate to fumarate by complex II shuttling electrons from Coenzyme QH<sub>2</sub> to reduce cytochrome <em>c</em> by complex III and is measured spectrophotometrically at 550 nm as a succinate dependent antimycin A (AA) sensitive reduction of cytochrome <em>c</em>. Each sample cuvette was prepared to contain a final concentration of 1 mM potassium cyanide, 300 µM EDTA, 100 µM cytochrome <em>c</em>, in a 166 mM potassium phosphate buffer pH 7.4 with addition of 185 µL of water. The corresponding reference cuvette was prepared in the same manner but 225 µL of water was added. 20 µL of sample was then added to each sample cuvette only, gently mixed and left to warm to 30 °C in the spectrophotometer. The reaction was initiated by the addition of 40 µL of 0.5 mM succinate into the sample cuvettes and the absorbance followed for 5 min. The absorbance for each sample was determined by subtracting the difference in absorbance prior to and after the addition of AA and Beer-Lambert law was applied using extinction coefficient of cytochrome <em>c</em> as 19.2 ×103 M<sup>−1</sup> cm<sup>−1</sup>. The resulting values were normalised to the activity of CS and expressed as ratios.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Analysis of alpha enolase activity</h3>
<p id="Par43">The activity was examined using an Abcam ab117994 kit according to manufacturer’s protocol. Cells were washed with ice-cold PBS and collected on ice by scraping. PBS was removed by centrifugation at 500 × <em>g</em> for 5 min at 4 °C and aspiration, pellets washed with 100 µL ice-cold PBS, centrifugation repeated and PBS removed. Pellets were dissolved in 200 µL of the kit extraction buffer while incubated on ice for 20 min, spun at 16,000 × <em>g</em> at 4 °C for 20 min, supernatants collected, total protein determined by the Bradford method and the supernatants stored at −80 °C till ready for analysis. Samples were diluted with incubation buffer to contain 150 µg of protein and a control sample dilution series from 0 to 250 µg of total protein was prepared. The native ENO1 was immunocaptured from the samples during a 2-h incubation at room temperature while shaking at 300 rpm. The buffer was removed, samples washed with blocking buffer and the wash repeated. An activity solution (100 µL), containing 0.5 mM NADH, 0.25 mM 2DG, PK, LDH and assay buffer was added to the samples and absorbance was immediately recorded at 340 nm over a period of 40 min with an interval of 20 s at 37 °C in an Infinite f200 plate reader. ENO1 activity was determined as the change in absorbance per min per µg of protein being proportionate to the consumption of NADH in the reaction mixture by the Beer-Lambert Law using the extinction coefficient of NADH as 6.22 mM<sup>−1</sup> cm<sup>−1</sup>.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Analysis of ATP production and glycolytic rates using Agilent XFp Seahorse analyser</h3>
<section id="Sec26"><h4 class="pmc_sec_title">Sample preparation</h4>
<p id="Par44">XFp cartridges were left hydrating overnight prior to the day of the assay in MilliQ water. Alamar blue cell viability reagent was added to each well (1/10<sup>th</sup> volume) and left to permeate the cells overnight. On the day of the analysis cell viability was assessed by measuring colour change and fluorescence of the dye (excitation 530 nm, emission 590 nm) using an Infinite f200 plate reader (Tecan, Switzerland). Fresh XF DMEM assay medium pH 7.4 was prepared with 1 mM sodium pyruvate, 2 mM glutamine and 10 mM glucose on the day of the assay. Following the 72-h GlcSph treatment cell culture medium was carefully removed and cells checked under the microscope for any perturbations. They were then washed with XF medium, 180 µL of assay medium was added to each well, the plate checked under the microscope again and incubated for 1 h at 37 °C with no CO<sub>2</sub> for 45–60 min to remove CO<sub>2</sub>. After degassing cells were washed with the assay medium, checked under the microscope and the plate placed in the Seahorse analyser for the analysis.</p></section><section id="Sec27"><h4 class="pmc_sec_title">ATP production rate</h4>
<p id="Par45">Port A of the XFp cartridge was designated for acute treatment of both control and lyso-GSL treated cells with 3 µM UK5099 (mitochondrial pyruvate shuttle inhibitor). Manufacturer’s protocol was followed using ATP rate assay kit where port B contained 1.5 µM oligomycin, port C contained 0.5 µM rotenone/antimycin A. Data were exported into Seahorse Wave software (v 2.6.1, Agilent), normalised to cell viability and an ATP rate report was generated.</p></section><section id="Sec28"><h4 class="pmc_sec_title">Glycolytic rate</h4>
<p id="Par46">Port A contained 0.5 µM rotenone/antimycin A and port B contained 500 µM 2-deoxy-D-glucose – an allosteric hexokinase inhibitor for glycolysis inhibition. The data were exported into Seahorse Wave software (v 2.6.1, Agilent), normalised to cell viability and a glycolytic rate report was generated.</p></section><section id="Sec29"><h4 class="pmc_sec_title">Statistics and reproducibility</h4>
<p id="Par47">Data from the ATP and glycolytic rate reports and functional assays were exported into Graphpad Prism software (V9, USA) and analysed by a 2-way ANOVA with a Sidak’s post-hoc multiple comparison test for the ATP production and glycolytic rates, and a Kruskal–Wallis test with a Dunn’s post-hoc test for the activity of mitochondrial complexes I-III, citrate synthase, malate dehydrogenases, glutathione reductase, alpha enolase and cellular energy charge assays. Results were considered to be statistically significant with <em>p</em> value &lt; 0.05.</p>
<p id="Par48">Proteomics data were analysed for pathway enrichment using IPA (QIAGEN Inc, <a href="https://digitalinsights.qiagen.com/products-overview/discovery-insights-portfolio/analysis-and-visualization/qiagen-ipa/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://digitalinsights.qiagen.com/products-overview/discovery-insights-portfolio/analysis-and-visualization/qiagen-ipa/</a>). Input variables were set to proteins that demonstrated a significantly altered expression between the control and lyso-lipid treated cells, with fold-change as expression observation. The output pathways were set to those with <em>p </em>&lt; 0.05. Gene ontology annotations were extracted using PANTHER bioinformatics tool (<a href="https://www.pantherdb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.pantherdb.org/</a>).</p></section></section><section id="Sec30"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par49">Further information on research design is available in the <a href="#MOESM4" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec31"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371010/bin/42003_2025_8684_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (436.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12371010/bin/42003_2025_8684_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">42003_2025_8684_MOESM2_ESM.pdf</a><sup> (28.1KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12371010/bin/42003_2025_8684_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data</a><sup> (40.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12371010/bin/42003_2025_8684_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting summary</a><sup> (1.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12371010/bin/42003_2025_8684_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (689.4KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This study has been part funded by the UCL Biological Mass Spectrometry Centre. All research at Great Ormond Street Hospital NHS Foundation Trust and UCL Great Ormond Street Institute of Child Health is made possible by the NIHR Great Ormond Street Hospital Biomedical Research Centre. The views expressed are those of the author(s) and not necessarily those of the NHS, the NIHR or the Department of Health. We would also like to thank the Peto Foundation for their generous donations.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>V.N. designed and performed experiments, analysed and interpreted data, wrote the original draft and revised the manuscript. R.V., S.E., J.H., and T.B. provided support for data collection, analysis and revised the manuscript. W.H. conceptualised and funded the study, W.H. and K.M. designed the experiments, supervised research and revised the manuscript. All authors read and approved the final version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par50"><em>Communications Biology</em> thanks Hans Aerts, Yu Chen and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editor: Benjamin Bessieres. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD061666 and 10.6019/PXD061666. Source data for all data represented in graphs within the figures are provided in a Supplementary Data file. Supplementary data are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par51">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec32"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s42003-025-08684-7.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Stirnemann, J. et al. Molecular sciences a review of gaucher disease pathophysiology, clinical presentation and treatments. <em>Int. J. Mol. Sci.</em><strong>18</strong>, 441 (2017).
</cite> [<a href="https://doi.org/10.3390/ijms18020441" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5343975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28218669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stirnemann,%20J.%20et%20al.%20Molecular%20sciences%20a%20review%20of%20gaucher%20disease%20pathophysiology,%20clinical%20presentation%20and%20treatments.%20Int.%20J.%20Mol.%20Sci.18,%20441%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Erikson, A., Bembi, B. &amp; Schiffmann, R. Neuronopathic forms of Gaucher’s disease. <em>Baillieres Clin. Haematol.</em><strong>10</strong>, 711–723 (1997).
</cite> [<a href="https://doi.org/10.1016/s0950-3536(97)80035-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9497859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Erikson,%20A.,%20Bembi,%20B.%20&amp;%20Schiffmann,%20R.%20Neuronopathic%20forms%20of%20Gaucher%E2%80%99s%20disease.%20Baillieres%20Clin.%20Haematol.10,%20711%E2%80%93723%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Ferraz, M. J. et al. Lysosomal glycosphingolipid catabolism by acid ceramidase: formation of glycosphingoid bases during deficiency of glycosidases. <em>FEBS Lett.</em><strong>590</strong>, 716–725 (2016).
</cite> [<a href="https://doi.org/10.1002/1873-3468.12104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26898341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ferraz,%20M.%20J.%20et%20al.%20Lysosomal%20glycosphingolipid%20catabolism%20by%20acid%20ceramidase:%20formation%20of%20glycosphingoid%20bases%20during%20deficiency%20of%20glycosidases.%20FEBS%20Lett.590,%20716%E2%80%93725%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Orvisky, E. et al. Glucosylsphingosine accumulation in mice and patients with type 2 Gaucher disease begins early in gestation. <em>Pediatr. Res.</em><strong>48</strong>, 233–237 (2000).</cite> [<a href="https://doi.org/10.1203/00006450-200008000-00018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10926300/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Grabowski, G. A. Gaucher disease: gene frequencies and genotype/phenotype correlations. <em>Genet Test.</em><strong>1</strong>, 5–12 (1997).
</cite> [<a href="https://doi.org/10.1089/gte.1997.1.5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10464619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grabowski,%20G.%20A.%20Gaucher%20disease:%20gene%20frequencies%20and%20genotype/phenotype%20correlations.%20Genet%20Test.1,%205%E2%80%9312%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Orvisky, E. et al. Glucosylsphingosine accumulation in tissues from patients with Gaucher disease: correlation with phenotype and genotype. <em>Mol. Genet Metab.</em><strong>76</strong>, 262–270 (2002).
</cite> [<a href="https://doi.org/10.1016/s1096-7192(02)00117-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12208131/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Orvisky,%20E.%20et%20al.%20Glucosylsphingosine%20accumulation%20in%20tissues%20from%20patients%20with%20Gaucher%20disease:%20correlation%20with%20phenotype%20and%20genotype.%20Mol.%20Genet%20Metab.76,%20262%E2%80%93270%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Peterschmitt, M. J., Foster, M. C., Ji, A. J., Zajdel, M. B. &amp; Cox, G. F. Plasma glucosylsphingosine correlations with baseline disease burden and response to eliglustat in two clinical trials of previously untreated adults with Gaucher disease type 1. <em>Mol. Genet Metab.</em><strong>138</strong>, 107527 (2023).
</cite> [<a href="https://doi.org/10.1016/j.ymgme.2023.107527" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36739645/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peterschmitt,%20M.%20J.,%20Foster,%20M.%20C.,%20Ji,%20A.%20J.,%20Zajdel,%20M.%20B.%20&amp;%20Cox,%20G.%20F.%20Plasma%20glucosylsphingosine%20correlations%20with%20baseline%20disease%20burden%20and%20response%20to%20eliglustat%20in%20two%20clinical%20trials%20of%20previously%20untreated%20adults%20with%20Gaucher%20disease%20type%201.%20Mol.%20Genet%20Metab.138,%20107527%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Guo, Y. et al. Elevated plasma chitotriosidase activity in various lysosomal storage disorders. <em>J. Inherit. Metab. Dis.</em><strong>18</strong>, 717–722 (1995).
</cite> [<a href="https://doi.org/10.1007/BF02436762" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8750610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20Y.%20et%20al.%20Elevated%20plasma%20chitotriosidase%20activity%20in%20various%20lysosomal%20storage%20disorders.%20J.%20Inherit.%20Metab.%20Dis.18,%20717%E2%80%93722%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Michelakakis, H., Dimitriou, E. &amp; Labadaridis, I. The expanding spectrum of disorders with elevated plasma chitotriosidase activity:An update. <em>J. Inherit. Metab. Dis.</em><strong>27</strong>, 705–706 (2004).
</cite> [<a href="https://doi.org/10.1023/b:boli.0000043025.17721.fc" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15669690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Michelakakis,%20H.,%20Dimitriou,%20E.%20&amp;%20Labadaridis,%20I.%20The%20expanding%20spectrum%20of%20disorders%20with%20elevated%20plasma%20chitotriosidase%20activity:An%20update.%20J.%20Inherit.%20Metab.%20Dis.27,%20705%E2%80%93706%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Boot, R. G. et al. Strong induction of members of the chitinase family of proteins in atherosclerosis: chitotriosidase and human cartilage gp-39 expressed in lesion macrophages. <em>Arterioscler. Thromb. Vasc. Biol.</em><strong>19</strong>, 687–694 (1999).
</cite> [<a href="https://doi.org/10.1161/01.atv.19.3.687" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10073974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boot,%20R.%20G.%20et%20al.%20Strong%20induction%20of%20members%20of%20the%20chitinase%20family%20of%20proteins%20in%20atherosclerosis:%20chitotriosidase%20and%20human%20cartilage%20gp-39%20expressed%20in%20lesion%20macrophages.%20Arterioscler.%20Thromb.%20Vasc.%20Biol.19,%20687%E2%80%93694%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Lukas, J. et al. Glucosylsphingosine causes hematological and visceral changes in mice—evidence for a pathophysiological role in gaucher disease. <em>Int. J. Mol. Sci.</em><strong>18</strong>, 2192 (2017).</cite> [<a href="https://doi.org/10.3390/ijms18102192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5666873/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29053611/" class="usa-link">PubMed</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Srikanth, M. P. et al. Elevated glucosylsphingosine in Gaucher disease induced pluripotent stem cell neurons deregulates lysosomal compartment through mammalian target of rapamycin complex 1. <em>Stem Cells Transl. Med.</em><strong>10</strong>, 1081–1094 (2021).
</cite> [<a href="https://doi.org/10.1002/sctm.20-0386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8235124/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33656802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Srikanth,%20M.%20P.%20et%20al.%20Elevated%20glucosylsphingosine%20in%20Gaucher%20disease%20induced%20pluripotent%20stem%20cell%20neurons%20deregulates%20lysosomal%20compartment%20through%20mammalian%20target%20of%20rapamycin%20complex%201.%20Stem%20Cells%20Transl.%20Med.10,%201081%E2%80%931094%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Taguchi, Y. V. et al. Glucosylsphingosine promotes α-synuclein pathology in mutant GBA-associated parkinson’s disease. <em>J. Neurosci.</em><strong>37</strong>, 9617–9631 (2017).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.1525-17.2017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5628407/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28847804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Taguchi,%20Y.%20V.%20et%20al.%20Glucosylsphingosine%20promotes%20%CE%B1-synuclein%20pathology%20in%20mutant%20GBA-associated%20parkinson%E2%80%99s%20disease.%20J.%20Neurosci.37,%209617%E2%80%939631%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Gundner, A. L. et al. Path mediation anlysis reveals GBA impacts Lewy body disease status by increaseing a-synuclein levels. <em>Neurobiol. Dis.</em><strong>121</strong>, 205–213 (2019).
</cite> [<a href="https://doi.org/10.1016/j.nbd.2018.09.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30236861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gundner,%20A.%20L.%20et%20al.%20Path%20mediation%20anlysis%20reveals%20GBA%20impacts%20Lewy%20body%20disease%20status%20by%20increaseing%20a-synuclein%20levels.%20Neurobiol.%20Dis.121,%20205%E2%80%93213%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Cleeter, M. W. J. et al. Glucocerebrosidase inhibition causes mitochondrial dysfunction and free radical damage. <em>Neurochem. Int.</em><strong>62</strong>, 1–7 (2013).
</cite> [<a href="https://doi.org/10.1016/j.neuint.2012.10.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3550523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23099359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cleeter,%20M.%20W.%20J.%20et%20al.%20Glucocerebrosidase%20inhibition%20causes%20mitochondrial%20dysfunction%20and%20free%20radical%20damage.%20Neurochem.%20Int.62,%201%E2%80%937%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>De La Mata, M. et al. Pharmacological chaperones and coenzyme Q10 treatment improves mutant $β$-glucocerebrosidase activity and mitochondrial function in neuronopathic forms of gaucher disease. <em>Sci. Rep.</em><strong>5</strong>, 1–18 (2015).</cite> [<a href="https://doi.org/10.1038/srep10903" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4456666/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26045184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20La%20Mata,%20M.%20et%20al.%20Pharmacological%20chaperones%20and%20coenzyme%20Q10%20treatment%20improves%20mutant%20%24%CE%B2%24-glucocerebrosidase%20activity%20and%20mitochondrial%20function%20in%20neuronopathic%20forms%20of%20gaucher%20disease.%20Sci.%20Rep.5,%201%E2%80%9318%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Osellame, L. D. et al. Mitochondria and quality control defects in a mouse model of gaucher disease - links to Parkinson’s disease. <em>Cell Metab.</em><strong>17</strong>, 941–953 (2013).
</cite> [<a href="https://doi.org/10.1016/j.cmet.2013.04.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3678026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23707074/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Osellame,%20L.%20D.%20et%20al.%20Mitochondria%20and%20quality%20control%20defects%20in%20a%20mouse%20model%20of%20gaucher%20disease%20-%20links%20to%20Parkinson%E2%80%99s%20disease.%20Cell%20Metab.17,%20941%E2%80%93953%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Nikolaenko, V., Warnock, D. G., Mills, K. &amp; Heywood, W. E. Elucidating the toxic effect and disease mechanisms associated with Lyso-Gb3 in Fabry disease. <em>Hum. Mol. Genet.</em><strong>32</strong>, 2464–2472 (2023).
</cite> [<a href="https://doi.org/10.1093/hmg/ddad073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10360391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37145097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nikolaenko,%20V.,%20Warnock,%20D.%20G.,%20Mills,%20K.%20&amp;%20Heywood,%20W.%20E.%20Elucidating%20the%20toxic%20effect%20and%20disease%20mechanisms%20associated%20with%20Lyso-Gb3%20in%20Fabry%20disease.%20Hum.%20Mol.%20Genet.32,%202464%E2%80%932472%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Nilsson, O. &amp; Svennerholm, L. Accumulation of glucosylceramide and glucosylsphingosine (psychosine) in cerebrum and cerebellum in infantile and juvenile Gaucher disease. <em>J. Neurochem.</em><strong>39</strong>, 709–718 (1982).
</cite> [<a href="https://doi.org/10.1111/j.1471-4159.1982.tb07950.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7097276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nilsson,%20O.%20&amp;%20Svennerholm,%20L.%20Accumulation%20of%20glucosylceramide%20and%20glucosylsphingosine%20(psychosine)%20in%20cerebrum%20and%20cerebellum%20in%20infantile%20and%20juvenile%20Gaucher%20disease.%20J.%20Neurochem.39,%20709%E2%80%93718%20(1982)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Rolfs, A. et al. Glucosylsphingosine is a highly sensitive and specific biomarker for primary diagnostic and follow-up monitoring in Gaucher disease in a non-Jewish, Caucasian cohort of Gaucher disease patients. <em>PLoS One</em><strong>8</strong>, e79732 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0079732" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3835853/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24278166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rolfs,%20A.%20et%20al.%20Glucosylsphingosine%20is%20a%20highly%20sensitive%20and%20specific%20biomarker%20for%20primary%20diagnostic%20and%20follow-up%20monitoring%20in%20Gaucher%20disease%20in%20a%20non-Jewish,%20Caucasian%20cohort%20of%20Gaucher%20disease%20patients.%20PLoS%20One8,%20e79732%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Schapira, A. H. et al. Mitochondrial complex I deficiency in Parkinson’s disease. <em>Lancet</em><strong>1</strong>, 1269 (1989).</cite> [<a href="https://doi.org/10.1016/s0140-6736(89)92366-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2566813/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Benecke, R., Strümper, P. &amp; Weiss, H. Electron transfer complexes I and IV of platelets are abnormal in parkinson’s disease but normal in parkinson-plus syndromes. <em>Brain</em><strong>116</strong>, 1451–1463 (1993).
</cite> [<a href="https://doi.org/10.1093/brain/116.6.1451" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8293280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Benecke,%20R.,%20Str%C3%BCmper,%20P.%20&amp;%20Weiss,%20H.%20Electron%20transfer%20complexes%20I%20and%20IV%20of%20platelets%20are%20abnormal%20in%20parkinson%E2%80%99s%20disease%20but%20normal%20in%20parkinson-plus%20syndromes.%20Brain116,%201451%E2%80%931463%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Dasgupta, N. et al. Neuronopathic Gaucher disease: dysregulated mRNAs and miRNAs in brain pathogenesis and effects of pharmacologic chaperone treatment in a mouse model. <em>Hum. Mol. Genet.</em><strong>24</strong>, 7031–7048 (2015).
</cite> [<a href="https://doi.org/10.1093/hmg/ddv404" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4654057/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26420838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dasgupta,%20N.%20et%20al.%20Neuronopathic%20Gaucher%20disease:%20dysregulated%20mRNAs%20and%20miRNAs%20in%20brain%20pathogenesis%20and%20effects%20of%20pharmacologic%20chaperone%20treatment%20in%20a%20mouse%20model.%20Hum.%20Mol.%20Genet.24,%207031%E2%80%937048%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Hargreaves, I. P. et al. Inhibition of mitochondrial complex IV leads to secondary loss complex II-III activity: implications for the pathogenesis and treatment of mitochondrial encephalomyopathies. <em>Mitochondrion</em><strong>7</strong>, 284–287 (2007).
</cite> [<a href="https://doi.org/10.1016/j.mito.2007.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17395552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hargreaves,%20I.%20P.%20et%20al.%20Inhibition%20of%20mitochondrial%20complex%20IV%20leads%20to%20secondary%20loss%20complex%20II-III%20activity:%20implications%20for%20the%20pathogenesis%20and%20treatment%20of%20mitochondrial%20encephalomyopathies.%20Mitochondrion7,%20284%E2%80%93287%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Sidransky, E. et al. Multicenter analysis of glucocerebrosidase mutations in Parkinson’s disease. <em>N. Engl. J. Med.</em><strong>361</strong>, 1651–1661 (2009).
</cite> [<a href="https://doi.org/10.1056/NEJMoa0901281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2856322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19846850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sidransky,%20E.%20et%20al.%20Multicenter%20analysis%20of%20glucocerebrosidase%20mutations%20in%20Parkinson%E2%80%99s%20disease.%20N.%20Engl.%20J.%20Med.361,%201651%E2%80%931661%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Park, J. S., Davis, R. L. &amp; Sue, C. M. Mitochondrial dysfunction in Parkinson’s disease: new mechanistic insights and therapeutic perspectives. <em>Curr. Neurol Neurosci. Rep.</em><strong>18</strong>, 21 (2018).</cite> [<a href="https://doi.org/10.1007/s11910-018-0829-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5882770/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29616350/" class="usa-link">PubMed</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Alcalay, R. N. et al. Glucocerebrosidase activity in Parkinson’s disease with and without GBA mutations. <em>Brain</em><strong>138</strong>, 2648–2658 (2015).
</cite> [<a href="https://doi.org/10.1093/brain/awv179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4564023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26117366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alcalay,%20R.%20N.%20et%20al.%20Glucocerebrosidase%20activity%20in%20Parkinson%E2%80%99s%20disease%20with%20and%20without%20GBA%20mutations.%20Brain138,%202648%E2%80%932658%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Lelieveld, L. T. et al. Consequences of excessive glucosylsphingosine in glucocerebrosidase-deficient zebrafish. <em>J. Lipid Res.</em><strong>64</strong>, 100199 (2022).</cite> [<a href="https://doi.org/10.1016/j.jlr.2022.100199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9058576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35315333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lelieveld,%20L.%20T.%20et%20al.%20Consequences%20of%20excessive%20glucosylsphingosine%20in%20glucocerebrosidase-deficient%20zebrafish.%20J.%20Lipid%20Res.64,%20100199%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Grünewald, A. et al. Differential effects of PINK1 nonsense and missense mutations on mitochondrial function and morphology. <em>Exp. Neurol.</em><strong>219</strong>, 266–273 (2009).
</cite> [<a href="https://doi.org/10.1016/j.expneurol.2009.05.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19500570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gr%C3%BCnewald,%20A.%20et%20al.%20Differential%20effects%20of%20PINK1%20nonsense%20and%20missense%20mutations%20on%20mitochondrial%20function%20and%20morphology.%20Exp.%20Neurol.219,%20266%E2%80%93273%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Osellame, L. D. &amp; Duchen, M. R. Defective quality control mechanisms and accumulation of damaged mitochondria link Gaucher and Parkinson diseases. <em>Autophagy</em><strong>9</strong>, 1633–1635 (2013).
</cite> [<a href="https://doi.org/10.4161/auto.25878" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23989665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Osellame,%20L.%20D.%20&amp;%20Duchen,%20M.%20R.%20Defective%20quality%20control%20mechanisms%20and%20accumulation%20of%20damaged%20mitochondria%20link%20Gaucher%20and%20Parkinson%20diseases.%20Autophagy9,%201633%E2%80%931635%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Ivanova, M. M., Changsila, E., Iaonou, C. &amp; Goker-Alpan, O. Impaired autophagic and mitochondrial functions are partially restored by ERT in Gaucher and Fabry diseases. <em>PLoS One</em><strong>14</strong>, 1–22 (2019).</cite> [<a href="https://doi.org/10.1371/journal.pone.0210617" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6329517/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30633777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ivanova,%20M.%20M.,%20Changsila,%20E.,%20Iaonou,%20C.%20&amp;%20Goker-Alpan,%20O.%20Impaired%20autophagic%20and%20mitochondrial%20functions%20are%20partially%20restored%20by%20ERT%20in%20Gaucher%20and%20Fabry%20diseases.%20PLoS%20One14,%201%E2%80%9322%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>González-Rodríguez, P. et al. Disruption of mitochondrial complex I induces progressive parkinsonism. <em>Nature</em><strong>599</strong>, 650–656 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-04059-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9189968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34732887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gonz%C3%A1lez-Rodr%C3%ADguez,%20P.%20et%20al.%20Disruption%20of%20mitochondrial%20complex%20I%20induces%20progressive%20parkinsonism.%20Nature599,%20650%E2%80%93656%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Smith, A. M. et al. Mitochondrial dysfunction and increased glycolysis in prodromal and early Parkinson’s blood cells. <em>Mov. Disord.</em><strong>33</strong>, 1580–1590 (2018).
</cite> [<a href="https://doi.org/10.1002/mds.104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6221131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30294923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20A.%20M.%20et%20al.%20Mitochondrial%20dysfunction%20and%20increased%20glycolysis%20in%20prodromal%20and%20early%20Parkinson%E2%80%99s%20blood%20cells.%20Mov.%20Disord.33,%201580%E2%80%931590%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Knight, A. L. et al. The glycolytic enzyme, GPI, is a functionally conserved modifier of dopaminergic neurodegeneration in Parkinson’s models. <em>Cell Metab.</em><strong>20</strong>, 145–157 (2014).
</cite> [<a href="https://doi.org/10.1016/j.cmet.2014.04.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4097176/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24882066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Knight,%20A.%20L.%20et%20al.%20The%20glycolytic%20enzyme,%20GPI,%20is%20a%20functionally%20conserved%20modifier%20of%20dopaminergic%20neurodegeneration%20in%20Parkinson%E2%80%99s%20models.%20Cell%20Metab.20,%20145%E2%80%93157%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Chen, X. et al. Terazosin activates Pgk1 and Hsp90 to promote stress resistance. <em>Nat. Chem. Biol.</em><strong>11</strong>, 19–25 (2015).
</cite> [<a href="https://doi.org/10.1038/nchembio.1657" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4412158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25383758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20X.%20et%20al.%20Terazosin%20activates%20Pgk1%20and%20Hsp90%20to%20promote%20stress%20resistance.%20Nat.%20Chem.%20Biol.11,%2019%E2%80%9325%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Hoshino, A. et al. The ADP/ATP translocase drives mitophagy independent of nucleotide exchange. <em>Nature</em><strong>575</strong>, 375–379 (2019).
</cite> [<a href="https://doi.org/10.1038/s41586-019-1667-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6858570/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31618756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoshino,%20A.%20et%20al.%20The%20ADP/ATP%20translocase%20drives%20mitophagy%20independent%20of%20nucleotide%20exchange.%20Nature575,%20375%E2%80%93379%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Rostovtseva, T. K. &amp; Bezrukov, S. M. VDAC inhibition by tubulin and its physiological implications. <em>Biochim. Biophys. Acta Biomembr.</em><strong>1818</strong>, 1526–1535 (2012).</cite> [<a href="https://doi.org/10.1016/j.bbamem.2011.11.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3302949/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22100746/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rostovtseva,%20T.%20K.%20&amp;%20Bezrukov,%20S.%20M.%20VDAC%20inhibition%20by%20tubulin%20and%20its%20physiological%20implications.%20Biochim.%20Biophys.%20Acta%20Biomembr.1818,%201526%E2%80%931535%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Zigdon, H. et al. Altered lysosome distribution is an early neuropathological event in neurological forms of Gaucher disease. <em>FEBS Lett.</em><strong>591</strong>, 774–783 (2017).
</cite> [<a href="https://doi.org/10.1002/1873-3468.12591" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28186340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zigdon,%20H.%20et%20al.%20Altered%20lysosome%20distribution%20is%20an%20early%20neuropathological%20event%20in%20neurological%20forms%20of%20Gaucher%20disease.%20FEBS%20Lett.591,%20774%E2%80%93783%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Oyarzún, J. E. et al. Lysosome motility and distribution: relevance in health and disease. <em>Biochim. Biophys. Acta Mol. Basis Dis.</em><strong>1865</strong>, 1076–1087 (2019).</cite> [<a href="https://doi.org/10.1016/j.bbadis.2019.03.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30904612/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Buscaglia, G., Northington, K. R., Moore, J. K. &amp; Bates, E. A. Reduced TUBA1A tubulin causes defects in trafficking and impaired adult motor behavior. <em>eNeuro</em><strong>7</strong>, 1–19 (2020).</cite> [<a href="https://doi.org/10.1523/ENEURO.0045-20.2020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7218002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32184299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buscaglia,%20G.,%20Northington,%20K.%20R.,%20Moore,%20J.%20K.%20&amp;%20Bates,%20E.%20A.%20Reduced%20TUBA1A%20tubulin%20causes%20defects%20in%20trafficking%20and%20impaired%20adult%20motor%20behavior.%20eNeuro7,%201%E2%80%9319%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Vulinovic, F. et al. Motor protein binding and mitochondrial transport are altered by pathogenic TUBB4A variants. <em>Hum. Mutat.</em><strong>39</strong>, 1901–1915 (2018).
</cite> [<a href="https://doi.org/10.1002/humu.23602" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30079973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vulinovic,%20F.%20et%20al.%20Motor%20protein%20binding%20and%20mitochondrial%20transport%20are%20altered%20by%20pathogenic%20TUBB4A%20variants.%20Hum.%20Mutat.39,%201901%E2%80%931915%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Okada, K. et al. An autopsy case of pure nigropathy with TUBA4A nonsense mutation. <em>Neuropathol. Appl Neurobiol.</em><strong>47</strong>, 891–893 (2021).
</cite> [<a href="https://doi.org/10.1111/nan.12712" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33760283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Okada,%20K.%20et%20al.%20An%20autopsy%20case%20of%20pure%20nigropathy%20with%20TUBA4A%20nonsense%20mutation.%20Neuropathol.%20Appl%20Neurobiol.47,%20891%E2%80%93893%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Yadav, S. K. et al. Identification of altered blood microRNAs and plasma proteins in a rat model of Parkinson’s disease. <em>Mol. Neurobiol.</em><strong>59</strong>, 1781–1798 (2022).
</cite> [<a href="https://doi.org/10.1007/s12035-021-02636-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35023059/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yadav,%20S.%20K.%20et%20al.%20Identification%20of%20altered%20blood%20microRNAs%20and%20plasma%20proteins%20in%20a%20rat%20model%20of%20Parkinson%E2%80%99s%20disease.%20Mol.%20Neurobiol.59,%201781%E2%80%931798%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Bliss, E., Heywood, W. E., Benatti, M., Sebire, N. J. &amp; Mills, K. An optimised method for the proteomic profiling of full thickness human skin. <em>Biol. Proced. Online</em><strong>18</strong>, 15 (2016).</cite> [<a href="https://doi.org/10.1186/s12575-016-0045-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4955162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27445641/" class="usa-link">PubMed</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Atkinson, D. E. The energy charge of adenylate pool as a regulatory parameter. <em>Biochemistry</em><strong>7</strong>, 4030–4034 (1968).</cite> [<a href="https://doi.org/10.1021/bi00851a033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4972613/" class="usa-link">PubMed</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Ragan, C., Wilson, M., Darley-Usmar, V. &amp; Lowe, P. <em>Subfractionation of mitochondria, and isolation of the proteins of oxidative phosphorylation in mitochondria: a practical approach</em>, 79–112 (IRL Press, 1987).</cite>
</li>
<li id="CR47">
<span class="label">47.</span><cite>King, T.E. Preparations of succinate—cytochrome c reductase and the cytochrome b-c1 particle, and reconstitution of succinate-cytochrome c reductase. <em>Methods Enzymol</em>. <strong>10</strong>, 216–225 (1967).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371010/bin/42003_2025_8684_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (436.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371010/bin/42003_2025_8684_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">42003_2025_8684_MOESM2_ESM.pdf</a><sup> (28.1KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371010/bin/42003_2025_8684_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data</a><sup> (40.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371010/bin/42003_2025_8684_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting summary</a><sup> (1.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371010/bin/42003_2025_8684_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (689.4KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD061666 and 10.6019/PXD061666. Source data for all data represented in graphs within the figures are provided in a Supplementary Data file. Supplementary data are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Communications Biology are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s42003-025-08684-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/42003_2025_Article_8684.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371010/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371010/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371010%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371010/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371010/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371010/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841705/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371010/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841705/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371010/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371010/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Oq1izdtBA50qScTn6Exc3mRERHmKNc3osFeqfktehF0lwykbQ4wS2FbuzMztkpWK">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
