
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Machine learning based on pangenome-wide association studies reveals the impact of host source on the zoonotic potential of closely related bacterial pathogens - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="1525471F8AF3435305471F0001403E72.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="commbiol">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368250/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Communications Biology">
<meta name="citation_title" content="Machine learning based on pangenome-wide association studies reveals the impact of host source on the zoonotic potential of closely related bacterial pathogens">
<meta name="citation_author" content="Cheng Han">
<meta name="citation_author_institution" content="State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China">
<meta name="citation_author" content="Shiying Lu">
<meta name="citation_author_institution" content="State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China">
<meta name="citation_author" content="Pan Hu">
<meta name="citation_author_institution" content="State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China">
<meta name="citation_author" content="Jiang Chang">
<meta name="citation_author_institution" content="State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China">
<meta name="citation_author_institution" content="State Key Laboratory of Genetic Engineering, Fudan Microbiome Center, School of Life Sciences, Fudan University, Shanghai, 200433 China">
<meta name="citation_author" content="Deying Zou">
<meta name="citation_author_institution" content="State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China">
<meta name="citation_author" content="Feng Li">
<meta name="citation_author_institution" content="State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China">
<meta name="citation_author_institution" content="Shandong Binzhou Animal Science and Veterinary Medicine Institute, Binzhou, 256600 Shandong Province China">
<meta name="citation_author" content="Yansong Li">
<meta name="citation_author_institution" content="State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China">
<meta name="citation_author" content="Qiang Lu">
<meta name="citation_author_institution" content="State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China">
<meta name="citation_author" content="Honglin Ren">
<meta name="citation_author_institution" content="State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="8">
<meta name="citation_firstpage" content="1253">
<meta name="citation_doi" content="10.1038/s42003-025-08650-3">
<meta name="citation_pmid" content="40835741">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368250/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368250/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368250/pdf/42003_2025_Article_8650.pdf">
<meta name="description" content="Variations in host species significantly impact bacterial growth traits and antibiotic resistance, making it essential to consider host origin when evaluating the zoonotic potential of pathogens. This study focuses on multiple Brucella species, ...">
<meta name="og:title" content="Machine learning based on pangenome-wide association studies reveals the impact of host source on the zoonotic potential of closely related bacterial pathogens">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Variations in host species significantly impact bacterial growth traits and antibiotic resistance, making it essential to consider host origin when evaluating the zoonotic potential of pathogens. This study focuses on multiple Brucella species, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368250/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368250">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s42003-025-08650-3"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/42003_2025_Article_8650.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368250%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368250/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368250/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368250/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-commbiol.jpg" alt="Communications Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Communications Biology" title="Link to Communications Biology" shape="default" href="https://www.nature.com/commsbio/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Commun Biol</button></div>. 2025 Aug 20;8:1253. doi: <a href="https://doi.org/10.1038/s42003-025-08650-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s42003-025-08650-3</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Commun%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Commun%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Commun%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Commun%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Machine learning based on pangenome-wide association studies reveals the impact of host source on the zoonotic potential of closely related bacterial pathogens</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Cheng Han</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Cheng Han</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Cheng Han</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Shiying Lu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Shiying Lu</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shiying Lu</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Pan Hu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Pan Hu</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pan Hu</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jiang Chang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jiang Chang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China </div>
<div class="p">
<sup>2</sup>State Key Laboratory of Genetic Engineering, Fudan Microbiome Center, School of Life Sciences, Fudan University, Shanghai, 200433 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiang Chang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zou%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Deying Zou</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Deying Zou</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zou%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Deying Zou</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Feng Li</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Feng Li</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China </div>
<div class="p">
<sup>3</sup>Shandong Binzhou Animal Science and Veterinary Medicine Institute, Binzhou, 256600 Shandong Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Feng Li</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Yansong Li</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Yansong Li</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yansong Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Qiang Lu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Qiang Lu</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qiang Lu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ren%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Honglin Ren</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Honglin Ren</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ren%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Honglin Ren</span></a>
</div>
</div>
<sup>1,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, Key Laboratory for Zoonosis Research of the Ministry of Education, Institute of Zoonosis, College of Veterinary Medicine, Chongqing Research Institute, Jilin University, Changchun, 130062 Jilin Province China </div>
<div id="Aff2">
<sup>2</sup>State Key Laboratory of Genetic Engineering, Fudan Microbiome Center, School of Life Sciences, Fudan University, Shanghai, 200433 China </div>
<div id="Aff3">
<sup>3</sup>Shandong Binzhou Animal Science and Veterinary Medicine Institute, Binzhou, 256600 Shandong Province China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 31; Accepted 2025 Aug 1; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the articleâ€™s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368250Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835741/" class="usa-link">40835741</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Variations in host species significantly impact bacterial growth traits and antibiotic resistance, making it essential to consider host origin when evaluating the zoonotic potential of pathogens. This study focuses on multiple <em>Brucella</em> species, which share highly similar genetic material, to explore the relationship between host origin and zoonotic potential by integrating pan-genome-wide association studies (pan-GWAS) with machine learning (ML). Our results present an open pangenome of <em>Brucella</em> spp. derived from the whole-genome sequencing (WGS) data of 991 strains and identify 268 genes potentially associated with the zoonotic potential of <em>Brucella</em>. Integrating these genes into an ML model based on the support vector machine (SVM) algorithm allows us to predict the zoonotic potential of various <em>Brucella</em> strains with high accuracy. Our findings reveal that zoonotic potential varies by host origin: <em>Brucella melitensis</em> strains isolated from humans exhibit higher zoonotic potential than those isolated from cattle, goats, and sheep, while <em>Brucella suis</em> biovar 2 strains isolated from domestic pigs display higher zoonotic potential than those isolated from wild boars. Our study proposes a method for predicting and quantifying the zoonotic potential of closely related bacterial pathogens from different host origins, providing valuable insights for risk assessment and public health strategy.</p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/00ff/12368250/6e7c64cebc30/42003_2025_8650_Figa_HTML.jpg" loading="lazy" id="d33e252" height="474" width="633" alt="graphic file with name 42003_2025_8650_Figa_HTML.jpg"></p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Bacteria, Genome informatics</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par3">This study proposed a method for identifying the host origins for zoonotic pathogens by integrating pan-genome-wide association studies with machine learning algorithms. The application of the proposed method on the <em>Brucella</em> species revealed diverse zoonotic host origins and potentials.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par4">Bacteria exhibit an exceptionally broad distribution across natural environments, occupying nearly all ecosystems, including soil, aquatic systems, and the internal environments of plants and animals<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. This extensive distribution underscores the diverse survival strategies of bacteria, enabling them to persist within a variety of hosts and to form intricate interactions with hosts<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>â€“<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Numerous pathogenic bacteria display distinct host specificity, thriving and reproducing solely within defined specific hosts, while some bacteria demonstrate the ability to infect a wide range of host species<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Furthermore, the phenomenon of host jump among bacteria has drawn increasing attention, as it enables bacteria to cross species boundaries, potentially leading to the emergence of novel infectious diseases<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. The transition of pathogenic bacteria from one host to another, particularly when shifting from animal hosts to humans, poses a substantial threat to public health. Consequently, the surveillance and investigation of bacterial host adaptability and their potential for interspecies transmission are critical for the prevention of infectious disease outbreaks in future.</p>
<p id="Par5">The COVID-19 pandemic has underscored the significant challenges that novel pathogens pose to global public health, highlighting that zoonotic pathogen spillovers into human populations can act as precursors to new pandemics. In this context, accurately predicting the zoonotic potential of pathogens has become critically important. The current approach to predicting the zoonotic potential of closely related bacterial pathogens primarily relies on phylogenetic analysis<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. However, the phylogenetic analysis often lacks the resolution required for high-precision zoonotic risk assessments<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. This limitation restricts our ability to reliably forecast the zoonotic potential of closely related bacterial pathogens, consequently hampering the design of effective intervention strategies. Thus, there is an urgent need to develop more refined and accurate assessment tools capable of rapidly identifying and mitigating the threats posed by closely related bacterial pathogens to human health.</p>
<p id="Par6">Brucellosis is an infectious disease caused by members of the genus <em>Brucella</em> that can be transmitted from livestock and wildlife to humans. The genus <em>Brucella</em> consists of highly homogenous species of bacteria with minimal genetic variation, which comprises six classical <em>Brucella</em> species and six atypical <em>Brucella</em> species<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. For a considerable period of time, only six classical species were recognized within the genus <em>Brucella</em>, including the main causative pathogens for human brucellosis: <em>Brucella melitensis</em>, <em>B. abortus</em>, and <em>B. suis</em><sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Except for the three <em>Brucella</em> species as above that are highly harmful to both humans and their natural hosts, <em>B. canis</em> and <em>B. neotomae</em> have been reported to cause human brucellosis, which are also members of the classical <em>Brucella</em> species<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. It is noteworthy that <em>B. neotomae</em> was previously classified as a non-zoonotic species among classical <em>Brucella</em> species, but recent evidence suggests that <em>B. neotomae</em> may have zoonotic potential<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. In recent years, several new atypical species of <em>Brucella</em> have been described, including <em>B. ceti</em>, <em>B. pinnipedialis</em>, <em>B. microti</em>, <em>B. inopinata</em>, <em>B. vulpis</em>, and <em>B. papionis</em>. Nevertheless, the zoonotic potential of these atypical <em>Brucella</em> species remains largely unknown. Therefore, it is necessary to evaluate the zoonotic potential of various <em>Brucella</em> species within the genus <em>Brucella</em> to provide valuable data for the control and prevention of brucellosis.</p>
<p id="Par7">The number of publicly available whole-genome sequencing (WGS) data for the genus <em>Brucella</em> is continually increasing due to advancements in next-generation sequencing technology<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, providing a prerequisite for assessing the zoonotic potential of <em>Brucella</em> based on WGS data. However, it is challenging to extract meaningful information from the large complex WGS data of <em>Brucella</em> using conventional statistical methods<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Machine learning (ML) has the ability to classify, cluster, and recognize patterns for large amounts of data<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. It can be categorized into supervised, unsupervised, and semi-supervised learning depending on the need for human intervention during the training process<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Supervised ML uses labeled datasets to train algorithms such as Random Forest (RF), Decision Trees (DTC), Support Vector Machine (SVM), K-Nearest Neighbors (KNN), and Multilayer Perceptron (MLP) to predict output based on labeled input data<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>â€“<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. There have been numerous studies using ML as a tool to investigate the correlation between pathogenic genetic information and virulence or host preference<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>â€“<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. This showcases the immense application potential of ML.</p>
<p id="Par8">In this study, we performed pangenome-wide association studies (pan-GWAS) on 991 <em>Brucella</em> genomes to identify specific genes associated with zoonotic phenotypes. These genes were then employed as features in ML models. Subsequently, we conducted various ML models using different algorithms, and compared their performances in evaluating the zoonotic potential of <em>Brucella</em> strains in the test dataset. The SVM algorithm was selected for developing the prediction model due to its optimal performance among these ML algorithms. The strong generalization ability of the SVM prediction models facilitated its broad applicability across various strains within the genus <em>Brucella</em>. Furthermore, the SVM models provided a more refined evaluation of the zoonotic potential of <em>Brucella</em> strains from diverse host origins. This capability enables high-resolution predictions of zoonotic potential for strains within the same <em>Brucella</em> species. Model explanations have identified important genes for the evaluation of the SVM prediction models, and these genes may contribute to the zoonotic potential of strains from <em>Brucella</em> species.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Genomic analysis indicates open pangenome of <em>Brucella</em>
</h3>
<p id="Par9">To examine the genomic features of <em>Brucella</em>, we used 11 different <em>Brucella</em> species of publicly available genomes to characterize and define its pangenome. Core genes were defined as a set of genes shared with all 991 strains, accessory genes presented in at least two strain genomes or up to n-1 strain genomes (n: 991, total number of genomes), and the genes presenting only in one strain genome were defined as unique genes. The pangenome was constructed with the dataset consisting of 582 core genes, 4,121 accessory genes, and 2,462 unique genes from 991 <em>Brucella</em> spp. strains. Functional annotation of genes in the pangenome was conducted utilizing the Clusters of Orthologous Genes database (COGs), with the results revealing a distinctive distribution of functional categories across the three different pangenome subsets (Fig.Â <a href="#Fig1" class="usa-link">1A</a>). Unique genes in the pangenome exhibited a higher ratio of genes annotated to the L category (Replication, recombination, and repair) than core and accessory genes (13.3% vs. 2.3% and 5.6%), with the majority of unique genes that belonged to the L category originating from <em>B. inopinata</em> (73.6%) (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">1</a>). Furthermore, we found that the partial L category genes of <em>B. inopinata</em> were related to DNA modification, such as DNA adenine methylation, ISP type restriction/modification enzyme and DNA (cytosine-5-)-methyltransferase, indicating that <em>B. inopinata</em> may possess substantial epigenetic plasticity to aid in niche shift.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. <em>Brucella</em> spp. pangenome statistics.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368250_42003_2025_8650_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/00ff/12368250/49fafa90a1d3/42003_2025_8650_Fig1_HTML.jpg" loading="lazy" id="d33e529" height="585" width="683" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> The <em>Brucella</em> spp. pangenome can be divided into three gene subsets: (i) core gene (the gene is present in n genomes, where n represents the total number of genomes), (ii) accessory gene (the gene is present in a number of genomes ranging between 2 and n-1), and (iii) unique gene (the gene is exclusively present in a single genome). The annotation and categorization of genes were performed by the COGs. COG categories encompass the following: Information storage and processing, which encompasses J (Translation, ribosomal structure and biogenesis), A (RNA processing and modification), K (Transcription), L (Replication, recombination and repair), and B (Chromatin structure and dynamics); Cellular processes and signaling, including D (Cell cycle control, cell division, chromosome partitioning), V (Defense mechanisms), T (Signal transduction mechanisms), M (Cell wall/membrane/envelope biogenesis), N (Cell motility), Z (Cytoskeleton), W (Extracellular structures), U (Intracellular trafficking, secretion, and vesicular transport), and O (Posttranslational modification, protein turnover, chaperones); Metabolism, including C (Energy production and conversion), G (Carbohydrate transport and metabolism), E (Amino acid transport and metabolism), F (Nucleotide transport and metabolism), H (Coenzyme transport and metabolism), I (Lipid transport and metabolism), P (Inorganic ion transport and metabolism), and Q (Secondary metabolites biosynthesis, transport and catabolism); Poorly characterized, including S (Function unknown). <strong>B</strong> The pangenome accumulation plot showed the change in pan genes (purple) and core genes (cyan) with the increasing size of the number of genomes. <strong>C</strong> The scatter plot displayed the number of accessory genes (X-axis) and unique genes (Y-axis) possessed by 991 <em>Brucella</em> strains, with the color of the points representing the species of each strain.</p></figcaption></figure><p id="Par10">Pangenome can be classified into two different types based on openness: open pangenome and closed pangenome. A highly open pangenome indicates that species inhabiting diverse environments possess the ability to exchange genetic material through various mechanisms, while species occupying the strained ecological niches find it challenging to acquire exogenous genes, resulting in a closed pangenome<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Heapâ€™s law was used to characterize the pangenome openness of <em>Brucella</em>, and Î³â€‰&gt;â€‰0 indicated an open pangenome for <em>Brucella</em>. For <em>Brucella</em>, we calculated that the relationship between pangenome size (P) and the number of genomes (N) was Pâ€‰=â€‰1260.58*991^0.25, Î³ equals 0.25, and the pangenome size increased with the addition of new genomes (Fig.Â <a href="#Fig1" class="usa-link">1B</a>), indicating that <em>Brucella</em> had an open pangenome and the pangenome repertoire of these 11 <em>Brucella</em> species was unable to be completely characterized using only the 991 genomes. Subsequently, we counted the number of accessory and unique genes within each strain of <em>Brucella</em>, and the result revealed that the majority of <em>Brucella</em> strains had a low number of unique genes, ranging from 0 to 100. Only five strains, including both two <em>B. inopinata</em> strains, had more than 100 unique genes. The number of accessory genes in most <em>Brucella</em> strains was around 2300. It is worth noting that both <em>B. neotomae</em> strains possessed a relatively low number of accessory genes, and the <em>B. neotomae</em> NCTC10070 strain had 1891 accessory genes, which is significantly lower than in other <em>Brucella</em> strains (Fig.Â <a href="#Fig1" class="usa-link">1C</a>). This lower count is probably ascribed to the comparatively small size of the genome of this particular strain, which has a size of 2.9 Mbp.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Quantifying the zoonotic potential of <em>Brucella</em> spp. via phylogenetic analysis presents substantial complexities</h3>
<p id="Par11">For analyzing the genetic structure among the 991 <em>Brucella</em> strains, we used the core gene alignment result to construct an ML tree of all <em>Brucella</em> strains. In particular, we focused on whether phylogenetic analysis could separate classical zoonotic <em>Brucella</em> species from <em>B. ovis</em>, the only non-zoonotic species identified, and predict the zoonotic potential of other <em>Brucella</em> species accordingly. While <em>Brucella</em> species exhibited a close genetic relationship, they display notable variations in virulence, host preferences, and zoonotic capacity<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. <em>B. melitensis</em>, <em>B. abortus</em>, and <em>B. suis</em> biovar 1 and 3 are the primary causative agents of brucellosis in both domesticated animals and humans<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Nevertheless, <em>B. ovis</em> infects animals exclusively and is the only known non-zoonotic <em>Brucella</em> species<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>â€“<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. As shown in Fig.Â <a href="#Fig2" class="usa-link">2</a>, <em>B.</em>
<em>abortus</em> and <em>B. melitensis</em> each formed a highly differentiated clade, while the other clade consisted of a mixture of various <em>Brucella</em> species. The two atypical <em>Brucella</em> species, <em>B. inopinata</em> and <em>B. vulpis</em>, were distant from classical <em>Brucella</em> strains, and the phylogenetic relationship between the two species was also found to be quite distant. Zoonotic <em>Brucella</em> strains were clearly divided into three distinct groups, while the non-zoonotic species <em>B. ovis</em> were located in a separate clade. However, with the exception of these three zoonotic groups and the isolated clade of non-zoonotic species, it was difficult to determine the zoonotic potential of other species such as <em>B. ceti</em> and <em>B. neotomae</em>, based solely on phylogenetic analysis (Fig.Â <a href="#Fig2" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Phylogenetic tree of <em>Brucella</em> species.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368250_42003_2025_8650_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/00ff/12368250/58bc8f874c41/42003_2025_8650_Fig2_HTML.jpg" loading="lazy" id="d33e706" height="453" width="701" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>This was a phylogenetic tree of 991 <em>Brucella</em> strains, reconstructed using a maximum-likelihood algorithm (raxml-ng) based on core gene alignment. Different species strains were labeled according to different colored dots. The red (zoonotic) and blue (non-zoonotic) background colors designated the zoonotic potential of <em>Brucella</em> strains.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Unsupervised ML algorithms are incapable of distinguishing between zoonotic and non-zoonotic <em>Brucella</em> strains</h3>
<p id="Par12">To acquire suitable input data for ML, we classified <em>B. ovis</em>, <em>B. suis</em> biovar 5, <em>B. abortus</em> 104â€‰M strain, and <em>B. suis</em> S2 strains as non-zoonotic <em>Brucella</em> strains. Conversely, the zoonotic <em>Brucella</em> dataset encompassed all strains of <em>B. melitensis</em>, <em>B. abortus</em>, and part of the <em>B. suis</em> strains (<em>B. suis</em> biovar 1 and <em>B. suis</em> biovar 3), while excluding the aforementioned non-zoonotic strains. In total, 883 <em>Brucella</em> strains were manually annotated with zoonotic labels, among which 861 were classified as zoonotic pathogens and 22 as non-zoonotic pathogens. 268 genes with statistically significant associations to zoonotic or non-zoonotic phenotypes (P.adjust&lt;0.05, Benjamini-Hochberg method) were identified as features through pan-GWAS analysis (Supplementary DataÂ <a href="#MOESM2" class="usa-link">1</a>).</p>
<p id="Par13">We initially employed the PCA algorithm to analyze data from all 991 strains, and PCA plots revealed the presence of non-zoonotic strains that were intermingled with zoonotic strains (Fig.Â <a href="#Fig3" class="usa-link">3A</a>). To ascertain the optimal parameter for the K-Means model, the Elbow method and silhouette coefficient were performed to determine the optimal value for the number of clusters in the unsupervised ML algorithm K-Means, and the results revealed that the optimal number of clusters was three (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">2</a>). The K-Means algorithm classified all strains into three clusters, although most non-zoonotic strains were split into cluster 3, while clusters 1 and 2 contained a mixture of both zoonotic and non-zoonotic strains (Fig.Â <a href="#Fig3" class="usa-link">3B</a>). The performance of the DBSCAN clustering algorithm was inadequate in distinguishing between zoonotic strains and non-zoonotic strains (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3</a>). These findings implied that unsupervised ML algorithms encountered challenges in discerning the distinction between zoonotic and non-zoonotic strains with precision.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Analysis of <em>Brucella</em> strains used the input binarized dataset by unsupervised ML algorithms PCA and K-Means.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368250_42003_2025_8650_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/00ff/12368250/8f261cedb90b/42003_2025_8650_Fig3_HTML.jpg" loading="lazy" id="d33e807" height="265" width="683" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The red point and blue point represented zoonotic <em>Brucella</em> strains and non-zoonotic <em>Brucella</em> strains, respectively. <strong>A</strong> The PCA plot of the <em>Brucella</em> strains. PC1 and PC2 represented principal component one and principal component two, respectively. The green dashed line signified the admixture of zoonotic <em>Brucella</em> strains with non-zoonotic <em>Brucella</em> strains. <strong>B</strong> The K-Means plot of the <em>Brucella</em> strains. The border lines display the ranges of the three clusters.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">The supervised ML models based on SVM algorithm can accurately distinguish between zoonotic and non-zoonotic strains</h3>
<p id="Par14">Due to the limited performance of unsupervised ML algorithms in distinguishing between zoonotic and non-zoonotic strains, five different supervised ML algorithms (RF, DTC, SVM, KNN, and MLP) were used to develop models for predicting the zoonotic potential of <em>Brucella</em> strains. Briefly, we divided 883 <em>Brucella</em> strains manually annotated with zoonotic labels into training and test datasets using random stratified sampling to ensure that both datasets maintained a proportional representation of zoonotic and non-zoonotic strains. The training dataset was then utilized to train each ML model and assessed the effectiveness of the models by stratified 3-fold cross validation. This entire process was repeated 100 times, and the results were averaged to obtain a more factual quality score for the models. The results showed that the models constructed with supervised ML algorithms achieved exceptional performance, with accuracy, recall, F1, and AUPRC mean scores all over 0.90, indicating that the supervised ML models were able to distinguish between zoonotic strains and non-zoonotic strains. Further analysis showed that the SVM ML model obtained the highest mean scores for all metrics except for the AUC and AUPRC score. Matthews correlation coefficient (MCC) score is a reliable metric for evaluating the quality of ML model in an imbalanced dataset. Among the five different ML models, the SVM model obtained the highest mean MCC score (0.78), and its MCC score distribution was more concentrated than that of the other ML models. These results suggested that SVM was the most appropriate algorithm for supervised ML model construction for <em>Brucella</em> zoonotic and non-zoonotic strain classification (Fig.Â <a href="#Fig4" class="usa-link">4A-F</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Violin and box plots showed the performance of ML models using different algorithms (RF, DTC, SVM, KNN, and MLP) in the validation dataset.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368250_42003_2025_8650_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/00ff/12368250/293fd9bb5382/42003_2025_8650_Fig4_HTML.jpg" loading="lazy" id="d33e851" height="269" width="684" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The performance of five algorithms was demonstrated by accuracy score (<strong>A</strong>), recall score (<strong>B</strong>), F1 score (<strong>C</strong>), AUC score (<strong>D</strong>), AUPRC score (<strong>E</strong>), and MCC score (<strong>F</strong>). The box plots marked the median, upper and lower quartiles, and 1.5Ã— inter-quartile range (whiskers); outliers were shown as points (nâ€‰=â€‰100 for each violin plot).</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">The SVM prediction models accurately quantify zoonotic capacity of <em>Brucella</em> strains in the test dataset</h3>
<p id="Par15">As the most suitable ML algorithm, we retrained the models using the hyperparameters of SVM models with the top 10% MCC score previously. To predict the zoonotic potential of <em>Brucella</em> strain, the output results of these retrained SVM models were averaged to obtain decision value for quantifying the zoonotic potential of the strain.</p>
<p id="Par16">The SVM prediction models were designed to generate decision values representing the zoonotic potential of <em>Brucella</em> strains on a continuous scale, with zero established as the threshold. This predictive framework allowed for the classification of zoonotic potential, where positive values indicated zoonotic potential and negative values denoted non-zoonotic potential. Moreover, the magnitude of the decision value reflected the strength of the corresponding classification, providing a quantitative measure of zoonotic potential. As shown in Fig.Â <a href="#Fig5" class="usa-link">5A</a>, most of the <em>Brucella</em> strains from the zoonotic subset of the test dataset obtained decision values above +0.3, while certain <em>B. suis</em> strains demonstrated substantially lower decision values than others within the zoonotic subset. One strain of <em>B. suis</em> bv. 1 from the zoonotic test subset was erroneously assigned a decision value of -0.37 (below the threshold of zero), resulting in its misclassification as a non-zoonotic pathogen. (Fig.Â <a href="#Fig5" class="usa-link">5A</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Kernel density plot showed the quantitative zoonotic potential predictions of zoonotic and non-zoonotic <em>Brucella</em> strains in the test dataset.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368250_42003_2025_8650_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/00ff/12368250/3d2b8d079dbf/42003_2025_8650_Fig5_HTML.jpg" loading="lazy" id="d33e915" height="301" width="683" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The red and blue points represented zoonotic <em>Brucella</em> strains (<strong>A</strong>) and non-zoonotic <em>Brucella</em> strains (<strong>B</strong>), respectively. The position of the point on the X-axis represented the zoonotic potential of the strain, and the points to the right of zero were predicted to be zoonotic <em>Brucella</em> strains, while those positioned to the left were predicted to be non-zoonotic <em>Brucella</em> strains. The red and blue background colors indicated the zoonotic probability density and the non-zoonotic probability density, respectively. The intensity of the background color correlated with the magnitude of zoonotic potential.</p></figcaption></figure><p id="Par17">In addition, the SVM prediction models accurately predicted the zoonotic potential of the non-zoonotic subset in the test dataset. The decision values for all non-zoonotic <em>Brucella</em> strains were below the threshold of zero. Notably, the SVM predictive models accurately predicted that the <em>B. ovis</em> had minimal zoonotic potential. <em>B. ovis</em> are generally recognized as non-zoonotic bacteria, exhibited decision values lower than -0.70, which further supported their classification as non-zoonotic bacteria. Two strains of <em>B. suis</em> bv. 5 were assigned decision values of -0.26 and -0.25, respectively, which were higher than those of <em>B. ovis</em> strains, indicating that their non-zoonotic potential was comparatively weaker than that of <em>B. ovis</em> strains (Fig.Â <a href="#Fig5" class="usa-link">5B</a>).</p></section><section id="Sec8"><h3 class="pmc_sec_title">The SVM prediction models exhibit strong generalization ability</h3>
<p id="Par18">Lipopolysaccharide (LPS) serves as a crucial virulence factor in <em>Brucella</em>, playing a pivotal role in the intracellular survival of <em>Brucella</em> within host cells. <em>Brucella</em> spp. are classified into smooth or rough phenotypes depending on whether O-polysaccharide (O-PS) is present on the LPS, and the rough phenotype <em>Brucella</em> strains frequently exhibit significantly attenuated virulence compared to the smooth phenotype <em>Brucella</em> strains, such as <em>B. melitensis</em> and <em>B. abortus</em><sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. The natural rough phenotype of <em>Brucella</em> includes <em>B. ovis</em> and <em>B. canis</em><sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Although <em>B. canis</em> is currently the only species within <em>Brucella</em> spp. sharing the same LPS as <em>B. ovis</em>, there have been several reports of human infected by <em>B. canis</em> in recent years, indicating its zoonotic potential<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>.</p>
<p id="Par19">While the SVM prediction models demonstrated excellent predictive performance in the test dataset, it is essential to further evaluate their generalization ability. <em>B. canis</em>, which is the only zoonotic species sharing the absence of O-PS with the non-zoonotic <em>B. ovis</em> species in <em>Brucella</em> spp., provided appropriate data for assessing the generalization ability of SVM prediction models. Therefore, we utilized the SVM prediction models to predict the zoonotic potential of 26 strains of <em>B. canis</em>. The mean decision value for <em>B. canis</em> was +0.49. Among the strains of <em>B. canis</em>, the highest positive decision value was +0.70. Notably, <em>B. canis</em> was not included in the training dataset of the SVM prediction models. The prediction results suggest that <em>B. canis</em> exhibits an intermediate zoonotic potential, which is in correspondence with the current understanding of this pathogen. The result indicated that the SVM prediction models demonstrated a strong generalization ability and can be utilized to estimate the zoonotic potential of other species strains within the genus <em>Brucella</em> (Fig.Â <a href="#Fig6" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Kernel density plot showed the quantitative zoonotic capacity predictions of <em>B.</em><em>canis</em>, <em>B</em>. <em>ceti</em>, <em>B</em>. <em>pinnipedialis</em>, <em>B</em>. <em>neotomae</em>, <em>B</em>. <em>microti</em>, <em>B</em>. <em>inopinata</em>, <em>B</em>. <em>suis</em> bv. 2, and <em>B</em>. <em>suis</em> bv. 4 strains.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368250_42003_2025_8650_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/00ff/12368250/60d173435193/42003_2025_8650_Fig6_HTML.jpg" loading="lazy" id="d33e1099" height="1215" width="723" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The color and position of the points indicated the species and predicted zoonotic potential of the strains, respectively. The red and blue background colors indicated the zoonotic probability density and the non-zoonotic probability density, respectively. The intensity of the background color correlated with the magnitude of the zoonotic potential. Animal silhouettes were obtained from PhyloPic (<a href="http://phylopic.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://phylopic.org</a>) under CC0 1.0 Universal Public Domain Dedication license.</p></figcaption></figure></section><section id="Sec9"><h3 class="pmc_sec_title">Multiple various <em>Brucella</em> species display a wide range of zoonotic potential</h3>
<p id="Par20">The high similarity (~97%) at the genomic level within the genus <em>Brucella</em> allows for the utilization of our pan-GWAS-based SVM prediction models among different species of <em>Brucella</em> strains and various host-derived isolates of the same species. The marine mammal-derived <em>Brucellae</em>, including <em>B. ceti</em> and <em>B. pinnipedialis</em>, have been demonstrated to have potential zoonotic capacity<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Here, nine strains of <em>B. ceti</em> were evaluated for zoonotic potential using the SVM prediction models with an average decision value of +0.36. The highest decision value of +0.57 was obtained from a <em>B. ceti</em> strain isolated from a common bottlenose dolphin (<em>Tursiops truncatus</em>), while the lowest decision value of +0.10 was obtained from a <em>B. ceti</em> strain isolated from common dolphin (<em>Delphinus delphis</em>). Similarly, for <em>B. pinnipedialis</em>, the strains of this species exhibited an average decision value of +0.32. The highest decision value of +0.53 was observed in a <em>B. pinnipedialis</em> strain isolated from hooded seal (<em>Cystophora cristata</em>), while the lowest decision value of +0.11 was observed in a <em>B. pinnipedialis</em> strain isolated from harbor seal (<em>Phoca vitulina</em>). These findings highlight the necessity of surveilling zoonotic potential of marine mammal <em>Brucellae</em>. (Fig.Â <a href="#Fig6" class="usa-link">6</a>).</p>
<p id="Par21"><em>B. neotomae</em>, first isolated from desert woodrats (<em>Neotoma lepida</em>) in the United States in 1957, was previously considered a non-zoonotic bacterium. However, two cases of <em>B. neotomae</em> infected human were reported in 2017<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, indicating that humans may be part of its host range. The SVM prediction models predicted an average decision value of +0.17 for five <em>B. neotomae</em> strains. Among these strains, the highest decision value was at +0.26, and the lowest decision value was at +0.03 (Fig.Â <a href="#Fig6" class="usa-link">6</a>).</p>
<p id="Par22"><em>B. microti</em> is the only known species capable of surviving in soil within the <em>Brucella</em> spp.<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. In this study, two strains of <em>B. microti</em> were predicted to have zoonotic potential, one strain was isolated from a vole (<em>Microtus arvalis</em>), while the other was found in a pool frog (<em>Pelophylax lessonae</em>). These strains had an average decision value of +0.37. Notably, the strain of <em>B. microti</em> derived from the vole exhibited a higher decision value of +0.54 compared to the strain isolated from the pool frog, which obtained a decision value of +0.21 (Fig.Â <a href="#Fig6" class="usa-link">6</a>).</p>
<p id="Par23">Apart from <em>B. ceti</em>, <em>B. pinnipedialis</em>, and <em>B. microti</em>, the atypical <em>Brucella</em> species include <em>B. vulpis</em> and <em>B. inopinata</em><sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Two strains of <em>B. vulpis</em> were assessed for their zoonotic potential by the SVM prediction models. The two highly similar strains of <em>B. vulpis</em> were isolated from red foxes (<em>Vulpes vulpes</em>) in 2008<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. As a result of their similarity, the decision values for both strains were identical, with a value of +0.15. This lack of variation in decision value made it impossible to construct a kernel density plot for <em>B. vulpis</em>.</p>
<p id="Par24">A strain of <em>B. inopinata</em> BO1 was isolated from the breast implant wound fluid of a 71-year-old female patient in 2008<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, and its zoonotic potential was predicted to be +0.076 based on the SVM prediction models. Despite the <em>B. inopinata</em> 141012304 strain being isolated from a bluespotted ribbontail ray (<em>Taeniura lymma</em>), its predicted zoonotic potential was slightly higher at +0.080 compared to <em>B. inopinata</em> BO1 (â€‰+â€‰0.076) isolated from the human patient (Fig.Â <a href="#Fig6" class="usa-link">6</a>).</p>
<p id="Par25"><em>B. suis</em> is divided into 5 various biovars, with <em>B. suis</em> bv 1, 3, and 5 being used to construct the SVM prediction models. The zoonotic potential of <em>B. suis</em> bv. 2 was subsequently predicted using the SVM prediction models, with an average decision value of +0.26. Additionally, <em>B. suis</em> bv. 4 strains obtained an average decision value of +0.42. Two distinct strains of <em>B. suis</em> bv. 4 strains isolated from a reindeer (<em>Rangifer tarandus</em>) and a wild boar were assessed for their zoonotic potential, yielding decision values of +0.29 and +0.54, respectively (Fig.Â <a href="#Fig6" class="usa-link">6</a>).</p></section><section id="Sec10"><h3 class="pmc_sec_title">The zoonotic potential of <em>Brucella</em> strain is affected by both species and isolated host</h3>
<p id="Par26">Statistical analysis further corroborated that the SVM prediction models provided species-specific and host-specific zoonotic potential for <em>Brucella</em> strains. The results revealed that the zoonotic potential of <em>B. abortus</em> and <em>B. melitensis</em> is significantly higher than that of most other <em>Brucella</em> species. Notably, the SVM prediction models provided species-specific decision values for <em>B. melitensis</em> strains in the test dataset based solely on the gene features of the training samples, even in the absence of the input species information for all the <em>Brucella</em> strains, which is the most virulent <em>Brucella</em> species for humans<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Conversely, the zoonotic potential of <em>B. ovis</em> was significantly lower compared to <em>B. abortus</em>, <em>B. melitensis</em>, <em>B. suis</em>, <em>B. canis</em>, <em>B. neotomae</em>, and marine mammal <em>Brucella</em> spp. (Fig.Â <a href="#Fig7" class="usa-link">7A, B</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. Impact of species and host factors on the predicted zoonotic potential of <em>Brucella</em> strains.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368250_42003_2025_8650_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/00ff/12368250/29bef4377209/42003_2025_8650_Fig7_HTML.jpg" loading="lazy" id="d33e1400" height="377" width="683" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Scatter plot showed the zoonotic potential of <em>Brucella</em> strains in the test dataset and various <em>Brucella</em> species strains derived from different hosts, the color of each point indicating the decision value for each <em>Brucella</em> strain. <strong>B</strong> Statistical significance of the decision value across 8 <em>Brucella</em> species. The single asterisk (*, yellow), double asterisk (**, blue), triple asterisk (***, red), and â€œnsâ€ indicated statistically significant, very significant, extremely significant, and nonsignificant differences, respectively, from the permutation test. <strong>C</strong> Box plot showed the predicted zoonotic potential of <em>B. melitensis</em> strains derived from various hosts. Statistical significance of the decision values was determined by the Kruskal-Wallis test (**, P.adjust&lt;0.01; ***, P.adjust&lt;0.001). <strong>D</strong> Box plot showed the predicted zoonotic potential of the <em>B. suis</em> bv. 2 strains derived from swine or wild boar. Statistical significance of the decision values was determined by the independent samples t-test (**, Pâ€‰&lt;â€‰0.01).</p></figcaption></figure><p id="Par27">Additionally, <em>B. melitensis</em> strains isolated from <em>Homo sapiens</em> exhibited significantly higher zoonotic potential than those isolated from <em>Bos taurus</em>, <em>Capra hircus</em>, and <em>Ovis aries</em> (Fig.Â <a href="#Fig7" class="usa-link">7C</a>). Among the <em>B. suis</em> bv. 2 strains, the highest decision value was +0.65, isolated from a swine (<em>Sus scrofa domesticus</em>), while the lowest decision value was -0.04, isolated from a wild boar (<em>Sus scrofa</em>) (Fig.Â <a href="#Fig6" class="usa-link">6</a>). The considerable difference between the highest decision value and the lowest decision value indicated a need to compare the zoonotic potential of <em>B. suis</em> bv. 2 strains derived from different hosts. We compared the decision values of <em>Brucella</em> strains isolated from different hosts and found that those isolated from swine exhibited significantly higher zoonotic potential than those isolated from wild boars, despite both belonging to the <em>Sus scrofa</em> species (Fig.Â <a href="#Fig7" class="usa-link">7D</a>).</p></section><section id="Sec11"><h3 class="pmc_sec_title">Model explanations reveal the genes potentially used to estimate the zoonotic potential of <em>Brucella</em> strains</h3>
<p id="Par28">Model explanations were conducted using the SHAP method on a set of 268 genes, which served as the features for constructing the SVM prediction models. The SHAP method offers two distinct perspectives for explaining the impact of features on ML model outputs: a global explanation and a local explanation. As shown in Fig.Â <a href="#Fig8" class="usa-link">8A</a>, all 268 features were ranked according to the mean absolute SHAP values in the global model explanation, identifying the ten most influential features on the predicted results (Fig.Â <a href="#Fig8" class="usa-link">8A</a>). Furthermore, the SHAP dependence plot showed how variations in feature values (<em>gene</em> present or <em>gene</em> absent) affected the predictions for <em>Brucella</em> strains. The feature <em>group_2874</em> was the most significant contributor to the output of SVM prediction models, resulting in predictions towards zoonotic direction when absent and towards non-zoonotic direction when present, with a greater distance of the shift than that observed in the absence of <em>group_2874</em>. It has been preliminarily identified as an uncharacterized protein, and currently, there is no definitive functional characterization available. This indicated that <em>group_2874</em> was more likely to be associated with a diminished zoonotic potential of <em>Brucella</em> strains (Fig.Â <a href="#Fig8" class="usa-link">8B</a>).</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8. Global and local explanations provided by the SHAP method.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368250_42003_2025_8650_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/00ff/12368250/47d9f0e264f7/42003_2025_8650_Fig8_HTML.jpg" loading="lazy" id="d33e1530" height="597" width="683" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> SHAP summary bar plot showed the top 10 features in descending order of the mean SHAP absolute values. <strong>B</strong> SHAP dependence plot showed the impact of 10 features on the decision value output of the SVM prediction models. Each dot represented a single <em>Brucella</em> strain in the test dataset, with red or blue indicating that the feature was present or absent in the genome of the <em>Brucella</em> strain, respectively. <strong>C</strong> SHAP force plot showed the impact of all features on the predicted results for each <em>Brucella</em> strain in the test dataset. The x-axis represented each sample, and the y-axis represented the predicted zoonotic potential of <em>Brucella</em> strains, with the larger part in red indicating greater zoonotic potential of the <em>Brucella</em> strains. <strong>D</strong>, <strong>E</strong> SHAP waterfall plot showed the local explanations for the typical zoonotic pathogen <em>B. melitensis</em> 16â€‰M strain and the non-zoonotic pathogen <em>B. ovis</em> ATCC 25840 strain.</p></figcaption></figure><p id="Par29">The predicted results for each <em>Brucella</em> strain in the test dataset were explained in Fig.Â <a href="#Fig8" class="usa-link">8C</a>. The upper x-axis represented each sample, while the y-axis represented the contributions of features to the predicted results. As the blue part of the sample increased, the likelihood of the <em>Brucella</em> strain being pathogenic to humans was decreased (Fig.Â <a href="#Fig8" class="usa-link">8C</a>). In addition, the local explanation of two <em>Brucella</em> strains, including the typical zoonotic <em>Brucella</em> strain (<em>B. melitensis</em> 16â€‰M strain, GCA_000740415.1) and non-zoonotic <em>Brucella</em> strain (<em>B. ovis</em> ATCC 25840 strain, GCA_000016845.1), provided a personalized analysis of the predicted results. For instance, the absence of <em>group_2974</em> resulted in an increase in the predicted zoonotic potential of the <em>B. melitensis</em> 16â€‰M strain. In contrast, the presence of <em>group_2974</em> in the <em>B. ovis</em> ATCC 25840 strain decreased its predicted zoonotic potential. The local explanation clearly demonstrated the roles that various features play in the zoonotic potential prediction process for individual <em>Brucella</em> strains (Fig.Â <a href="#Fig8" class="usa-link">8D, E</a>).</p></section></section><section id="Sec12"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par30">The construction of the pangenome of <em>Brucella</em> spp. provided a finer resolution insight into the genetic diversity of this zoonotic intracellular pathogen. The pangenome of <em>Brucella</em> spp. consisted of 582 core genes, 4121 accessory genes, and 2462 unique genes. The proportion of genes annotated to the L category among the unique genes was higher than that of core and accessory genes (Fig.Â <a href="#Fig1" class="usa-link">1A</a>). This was likely due to individual <em>Brucella</em> strains facing distinct selective pressures, necessitating the retention of genes that are beneficial for their genetic information transfer, thereby ensuring their survival under such conditions. This result reflected the diversity of ecological niches inhabited by <em>Brucella</em> strains. Furthermore, our study demonstrated that these 11 <em>Brucella</em> species possessed an open pangenome (Fig.Â <a href="#Fig1" class="usa-link">1B</a>), which was consistent with the ongoing expansion of the genus <em>Brucella</em>, indicating once again that pathogens belonging to the <em>Brucella</em> spp. inhabited the varied ecological niches<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. The coexistence of specialist <em>Brucella</em> strains (such as <em>B. melitensis</em> and <em>B. abortus</em>) infecting livestock or wildlife and generalist <em>Brucella</em> strains (such as <em>B. microti</em>) living extensively in the environment underpinned the movement of diverse <em>Brucella</em> strains among the environment, animals, and humans.</p>
<p id="Par31">Given the largely unknown zoonotic potential of novel species of <em>Brucella</em> and the emergence of human brucellosis caused by <em>Brucella</em> strains previously thought to lack zoonotic capability, such as <em>B. neotomae</em> considered a non-zoonotic species over the past years<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, the understanding of the zoonotic potential of <em>B. neotomae</em> has been changed by the isolation of <em>B. neotomae</em> from the cerebrospinal fluid of two patients diagnosed with human brucellosis in Costa Rica<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. This finding necessitates a reconsideration of its host range and implications for public health. It was necessary to develop a prediction model to further characterize the zoonotic potential of <em>Brucella</em> strains. To establish the most effective prediction model, we conducted a comparative analysis of three distinct methods, encompassing phylogenetic analysis, unsupervised ML model, and supervised ML model. Our findings revealed that both phylogenetic analysis and unsupervised ML model were ineffective in distinguishing between artificially categorized zoonotic and non-zoonotic <em>Brucella</em> strains. Conversely, supervised ML models successfully discriminated between zoonotic and non-zoonotic <em>Brucella</em> strains in the test dataset. The SVM-based model exhibited superior discriminative performance on the validation dataset compared to other supervised ML models (Fig.Â <a href="#Fig4" class="usa-link">4</a>). Consequently, we developed SVM prediction models to obtain quantitative indicator decision values, which were then utilized for assessing the zoonotic potential of <em>Brucella</em> strains.</p>
<p id="Par32">The generalization ability of ML model is a crucial metric for evaluating its performance<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. This ability determines whether the developed model can maintain suitable performance for dealing with unseen data. In this study, we assessed the generalization ability of the SVM prediction models using <em>B. canis</em> strains. The SVM prediction models yielded an average decision value of +0.49 for <em>B. canis</em> strains (Fig.Â <a href="#Fig6" class="usa-link">6</a>), ranking closely behind zoonotic <em>Brucella</em> species <em>B. melitensis</em> (+1.11) and <em>B. abortus</em> (+0.91) in the test dataset (Fig.Â <a href="#Fig5" class="usa-link">5A</a>). This precise prediction of the zoonotic potential of unseen data from <em>B. canis</em> strains indicated the strong generalization ability of the SVM prediction models. Furthermore, the SVM prediction models demonstrated a surprising efficacy in evaluating the zoonotic potential characteristics of <em>Brucella</em> species at both interspecific (among different species) and intraspecific (among strains of the same species) levels. Among the various <em>Brucella</em> species, <em>B. melitensis</em> is recognized as the most pathogenic, thereby establishing itself as a critical focus for zoonotic potential assessments<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Notably, the SVM prediction models exhibited the capability to accurately differentiate <em>B. melitensis</em> from other zoonotic <em>Brucella</em> species, even in the absence of any species-related input data (Fig.Â <a href="#Fig7" class="usa-link">7A, B</a>).</p>
<p id="Par33">Holzapfel et al. addressed the virulence of <em>Brucella</em> species isolated from different hosts, revealing that the <em>B. melitensis</em> strains isolated from various hosts did not exhibit any host-specific characteristics. This research focused on <em>B. melitensis</em> strains identified as clones, isolating these pathogens from multiple hosts during the same outbreak over a span of one year. The relatively short isolation period of merely one year may present challenges for <em>B. melitensis</em> in achieving effective host adaptation. This limitation may have influenced the observed relationship between host factors and the characteristics of <em>Brucella</em> strains<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. <em>Brucella</em> have been isolated from several bodily fluids such as urine, semen, breast milk, and blood of infected patients<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>â€“<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. The transmission routes of <em>Brucella</em>, such as maternal transmission, sexual contact, and blood transfusion, have been described<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>â€“<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Our findings further suggest that <em>B. melitensis</em> strains isolated from humans exhibit a higher predictive zoonotic potential. Consequently, this highlights the critical need for further investigations into the impact of diverse hosts on the virulence and zoonotic potential of strains within the same <em>Brucella</em> species. In addition to accurately assessing the zoonotic potential of different <em>Brucella</em> species, the SVM prediction models have also effectively distinguished <em>B. melitensis</em> isolates from human and differentiated between <em>B. suis</em> bv. 2 strains isolated from swine and those isolated from wild boars, despite the lack of host information of isolates (Fig.Â <a href="#Fig7" class="usa-link">7C, D</a>). These results provide valuable insights into the impact of the host on the characteristics of <em>Brucella</em> strains.</p>
<p id="Par34">The circulation at the wildlife-livestock interface is a prominent characteristic for <em>B. melitensis, B. abortus, and B. suis</em><sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. As previously described, the distribution of <em>B. suis</em> bv. 2 is progressively expanding from wild boars and hares to domestic pigs; this transition is particularly concerning because domestic pigs have a significantly closer association with human communities compared to wild boars, turning this pathogen, which is rarely infective to humans, into a significant public health concern<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>â€“<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. Our SVM prediction models indicated that the zoonotic potential associated with <em>B. suis</em> bv. 2 from domestic pigs is greater than that from wild boars. Therefore, it is essential to implement continuous monitoring of <em>B. suis</em> bv. 2 within animal husbandry to find the potential risk of host jumps targeting humans in the future. These findings indicate that the predictions made by the SVM prediction models are informative.</p>
<p id="Par35">The zoonotic potential of <em>Brucella</em> strains varies among different species. Human brucellosis is commonly caused by three <em>Brucella</em> species transmitted from animals to humans: <em>B. melitensis</em>, <em>B. abortus</em>, and <em>B. suis</em> bv. 1 and 3. However, the zoonotic potential of novel members of the genus <em>Brucella</em>, such as <em>B. ceti</em>, <em>B. pinnipedialis</em>, <em>B. inopinata</em>, <em>B. microti</em>, <em>B. vulpis</em>, and <em>B. suis</em> bv. 2 and 4 remains largely unknown, despite the fact that some strains of certain <em>Brucella</em> species have been isolated from patients with brucellosis. The predicted zoonotic potential indicated the necessity for ongoing monitoring among these <em>Brucella</em> species. Due to the limited number of available genomes for these <em>Brucella</em> species, the prediction results generated by the SVM prediction models may not fully characterize the true zoonotic potential of these species. In the future, the performance of the SVM prediction models could be enhanced by increasing the WGS data for <em>Brucella</em>.</p>
<p id="Par36">As an effective ML model explanation tool, the SHAP method identified 10 genes that are most important to evaluate the zoonotic potential of the <em>Brucella</em> strain in SVM prediction models. The gene <em>group_1020</em> encodes a protein belonging to the pentapeptide MXKDX repeat protein family, and this gene was annotated as â€œReduced virulenceâ€ (<a href="http://www.phi-base.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.phi-base.org</a>), which suggests that the gene may enhance the virulence of pathogen. The gene <em>group_1603</em> was predicted encodes ATPase component belong to ABC-type transport system and was also annotated as â€œReduced virulenceâ€, ABC transporter plays an important role in bacterial pathogenesis and virulence<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. In addition to the top 10 genes, the gene <em>group_4450</em> encodes urea transporter, which has been observed to be up-regulated during the acid stress response for <em>B. melitensis</em> 16â€‰M<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Given that the consumption of contaminated food is a common transmission route for <em>Brucella</em>, this observation suggests that <em>group_4450</em> may play an important role in aiding <em>Brucella</em> strains in resisting the acidic environments within the host stomach, thereby facilitating the establishment of infection.</p>
<p id="Par37">The dataset used for training the SVM prediction models is imbalanced, with the number of zoonotic <em>Brucella</em> strains exceeding that of non-zoonotic <em>Brucella</em> strains. This discrepancy may result in an overestimation of the predicted zoonotic potential for <em>Brucella</em> strains<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. To mitigate the sample imbalance, SVMSMOTE was performed to alleviate the skewed class distribution of the training dataset. Furthermore, the MCC score was adopted to assess the performance of ML models, thereby improving the reliability of the results. Although there remains a risk of overestimating zoonotic potential for SVM prediction models, it is more important from a public health perspective to avoid misclassifying zoonotic <em>Brucella</em> strains as non-zoonotic; such errors may result in undetected brucellosis outbreaks and pose a serious threat to public health.</p>
<p id="Par38">Currently, the prediction of zoonotic potential based on ML utilizing genomic data is typically conducted within the same species for bacteria<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. Our study achieves cross-species zoonotic potential predictions using ML on multiple highly conserved <em>Brucella</em> species, demonstrating the applicability of ML to predict zoonotic potential in bacterial pathogens with greater genetic variability. However, an anticipated challenge is that as genetic variability increases, it becomes increasingly difficult to capture sufficient information from the genomes. Relying solely on the presence or absence of genes may become inadequate for predicting zoonotic potential in bacterial pathogens with greater genetic variability. It is therefore necessary to provide ML with more information at the nucleotide-level, which can capture additional genetic variations such as Single-nucleotide polymorphisms (SNPs) and Insertions and Deletions (Indels). SNPs have been shown to grant bacteria the ability to infect new host species or acquire antibiotic resistance<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>,<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>, highlighting the risk that gene-level analyses may miss important trait-associated information. Thus, integrating both gene-level and nucleotide-level features in future studies may improve the accuracy of zoonotic potential prediction for bacterial pathogens with high genetic variability.</p>
<p id="Par39">In conclusion, we successfully combined pan-GWAS with ML to predict the zoonotic potential of closely related bacteria within the genus <em>Brucella</em>. Among the 11 <em>Brucella</em> species, most strains were predicted to be potential zoonotic pathogens. The SVM prediction models delivered species-specific and host-specific predictions of zoonotic potential for these closely related bacteria isolated from various hosts. This technical framework offers a viable analytical method for predicting the zoonotic potential of the closely related bacterial pathogens at the host resolution.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Preparation of the binarized dataset for ML</h3>
<p id="Par40">A total of 1042 assembled genomes from 11 different <em>Brucella</em> species were downloaded from NCBI Assembly database (<a href="https://www.ncbi.nlm.nih.gov/assembly" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/assembly</a>). The quality of these genomes was checked by <a href="https://quast.sourceforge.net" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">QUAST</a>, and a total of 991 <em>Brucella</em> genomes were considered to be of acceptable quality<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. The visualization of species and host source information pertaining to these 991 <em>Brucella</em> strains was generated using the CNSknowall platform (<a href="https://cnsknowall.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cnsknowall.com</a>), a comprehensive web service for data analysis and visualization (Supplementary TableÂ <a href="#MOESM1" class="usa-link">1</a>). The annotation of 991 quality-passed genomes was performed using Prokka, a bioinformatics tool for annotating prokaryotic genome (<a href="https://github.com/tseemann/prokka" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/tseemann/prokka</a>)<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. The pan-genome analysis was performed by Roary (<a href="https://sanger-pathogens.github.io/Roary" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://sanger-pathogens.github.io/Roary</a>)<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. The COG annotation was conducted using the online web service eggNOG-mapper v2 (<a href="http://eggnog-mapper.embl.de" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://eggnog-mapper.embl.de</a>)<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. A Python script was used to calculate Heapâ€™s law to characterize the openness of the <em>Brucella</em> pangenome (Supplementary DataÂ <a href="#MOESM3" class="usa-link">2</a>). To classify <em>Brucella</em> strains according to their zoonotic potential, <em>B. abortus</em>, <em>B. melitensis</em>, and part of <em>B. suis</em> (biovar 1 and biovar 3) strains were assigned to the zoonotic group<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. <em>B. ovis</em> and <em>B. suis</em> biovar 5 strains were assigned to the non-zoonotic group<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. Additionally, the vaccine strains <em>B. abortus</em> 104â€‰M and <em>B. suis</em> S2 were also assigned to the non-zoonotic group<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>,<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. Subsequent to annotating the zoonotic potential information for the <em>Brucella</em> strains, the pan-GWAS tool Scoary was used to identify statistically significant associated genes with either zoonotic or non-zoonotic groups (P.adjust&lt;0.05, Benjamini-Hochberg method) and these genes were used to build the input binarized dataset (Supplementary DataÂ <a href="#MOESM2" class="usa-link">1</a>)<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Phylogenetic analysis and unsupervised ML for <em>Brucella</em> spp. strains</h3>
<p id="Par41">The maximum-likelihood tree based on the core genes was constructed using raxmlGUIv2.0 (<a href="https://antonellilab.github.io/raxmlGUI" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://antonellilab.github.io/raxmlGUI</a>)<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>, the model was set to GTRâ€‰+â€‰G and the reliability of the internal branch was evaluated using 200 bootstrap replicates. The resulting phylogenetic tree was modified using tvBOT (<a href="https://www.chiplot.online/tvbot.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.chiplot.online/tvbot.html</a>)<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. The PCA on the input binarized dataset was performed using the Scikit-learn Python package (<a href="https://scikit-learn.org/stable/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scikit-learn.org/stable/index.html</a>). The K-Means clustering and DBSCAN clustering models were built on the PCA transformed input binarized dataset using the Scikit-learn and SciPy Python package (<a href="https://scipy.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scipy.org</a>). All figures were plotted using the Matplotlib Python packages (<a href="https://matplotlib.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://matplotlib.org</a>).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Supervised ML for <em>Brucella</em> spp. strains</h3>
<p id="Par42">The binarized dataset was used to develop a supervised ML model aimed at generating predictions for the zoonotic potential of <em>Brucella</em> strains. Five different supervised ML algorithms (RF, DTC, SVM, KNN, and MLP) were used to build the prediction models. Due to the limited number of non-zoonotic <em>Brucella</em> strain samples, a stratified sampling approach was employed to allocate training and test datasets in a 50-50% split for the input binarized dataset. To tackle the class imbalance problem, the imbalanced-learn Python package was utilized to perform SVM Synthetic Minority Over-sampling Technique (SVMSMOTE) for oversampling on the training dataset, ensuring an adequate representation of non-zoonotic <em>Brucella</em> strain samples for training the ML model. Hyperparameter optimization for the ML model was carried out using a grid search with 3-fold cross-validation, with the evaluation score set as the MCC score. The ML model was trained on the training dataset using the optimal hyperparameters identified from the grid search: H<sub>1</sub>(class_weightâ€‰=â€‰â€˜balancedâ€™, Câ€‰=â€‰0.005, degreeÂ =Â 0, kernelâ€‰=â€‰â€˜rbfâ€™), H<sub>2</sub>(class_weightâ€‰=â€‰â€˜balancedâ€™, Câ€‰=â€‰0.005, degreeÂ =Â 10, kernelâ€‰=â€‰â€˜polyâ€™), and H<sub>3</sub>(class_weightâ€‰=â€‰â€˜balancedâ€™, Câ€‰=â€‰0.005, degreeÂ =Â 1, kernelâ€‰=â€‰â€˜polyâ€™). Its performance was then evaluated on the test dataset. To mitigate potential random sampling errors, the sample split, model establishment, and validation process were repeated 100 times for different algorithms. The performance of the ML models established with different algorithms was evaluated based on various metrics, including Accuracy score, Recall score, F1 score, AUC score, AUPRC score, and MCC score, calculated as the average of all 100 repetitions. The sample split and ML model establishment were executed using the Scikit-learn Python package.</p></section><section id="Sec17"><h3 class="pmc_sec_title">The establishment of SVM prediction models and quantification of the zoonotic potential of <em>Brucella</em> strains</h3>
<p id="Par43">After identifying SVM as the algorithm for establishing the zoonotic potential prediction model, the hyperparameters of the models with MCC scores in the top 10% were extracted from the results of the grid search. Three distinct SVM prediction models were subsequently developed using these optimal hyperparameters: H<sub>1</sub>, H<sub>2</sub>, and H<sub>3</sub>. For each <em>Brucella</em> strain sample, the decision value is calculated as the average of the raw decision_function outputs from these three SVM models. The SciPy Python package was used to conduct Gaussian kernel density estimation of predicted probabilities for the strains in the test dataset and various species of the genus <em>Brucella</em>. The kernel density plot was generated using the Matplotlib Python package.</p></section><section id="Sec18"><h3 class="pmc_sec_title">SVM prediction models explanation and annotation of gene pathogenicity</h3>
<p id="Par44">The oversampled training dataset was clustered into 100 weighted samples via K-Means clustering and employed as background samples for the SHAP method. Each SVM prediction model used the kernel SHAP method to explain decision values, followed by averaging SHAP values across all models to derive the final output. Feature importance was assessed by ranking the averaged SHAP values, and both global and local explanations were constructed using the shap Python package (<a href="https://shap.readthedocs.io/en/latest" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://shap.readthedocs.io/en/latest</a>). The pathogenicity of genes was annotated using PHIB-BLAST on the Pathogen-Host Interactions database (PHI-base, <a href="http://www.phi-base.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.phi-base.org</a>).</p></section><section id="Sec19"><h3 class="pmc_sec_title">Statistics and Reproducibility</h3>
<p id="Par45">Statistical significance was determined by independent samples t-test for the decision values of <em>B. suis</em> biovar 2 strains isolated from swine or wild boar, which conformed to normal distribution. Since the decision values of the eight <em>Brucella</em> species did not all conform to normal distribution and some <em>Brucella</em> species have small sample sizes, permutation test was performed to determine interspecies statistical significance, with p values were corrected by the Benjamini-Hochberg method. The decision values for <em>B. melitensis</em> strains isolated from different hosts did not conform to normal distribution, and the Kruskal-Wallis test was used to determine statistical significance with p values were corrected by the Bonferroni method. All p values described in the study are two-sided. The independent samples t-test, Kruskal-Wallis test, and Bonferroni method were performed using BONC DSS Statistics 25 software. The permutation test and Benjamini-Hochberg method were performed using SciPy Python package.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par46">Further information on research design is available in theÂ <a href="#MOESM6" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec21"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12368250/bin/42003_2025_8650_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (3.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12368250/bin/42003_2025_8650_MOESM2_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (18.4KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12368250/bin/42003_2025_8650_MOESM3_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (1.7KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12368250/bin/42003_2025_8650_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (81.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12368250/bin/42003_2025_8650_MOESM5_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">42003_2025_8650_MOESM5_ESM.docx</a><sup> (21.2KB, docx) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12368250/bin/42003_2025_8650_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12368250/bin/42003_2025_8650_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.3MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by the Self-established grant from the State Key Laboratory for Diagnosis and Treatment of Severe Zoonotic Infectious Diseases, and the Natural Science Foundation of Chongqing (No. CSTB2023NSCQ-MSX0515).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Cheng Han conceived the ideas, designed the experiments, and wrote the manuscript. Shiying Lu and Pan Hu performed experiments. Jiang Chang contributed to analytic tools. Deying Zou, Feng Li and Yansong Li analyzed data. Qiang Lu contributed to the writing of the manuscript. Honglin Ren contributed to the manuscript revision and supervised the overall study. All authors have read and approved the final version of the manuscript for publication.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par47"><em>Communications Biology</em> thanks Peter Rabinowitz, Kumaragurubaran Karthik and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Yu-Wei Wu and Aylin Bircan, Tobias Goris. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data supporting the findings of this study are available within the paper and its Supplementary Information. The genomic data for all <em>Brucella</em> strains analyzed in this study was obtained from the NCBI Assembly database (<a href="https://www.ncbi.nlm.nih.gov/assembly" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/assembly</a>), with strain-associated metadata available in Supplementary DataÂ <a href="#MOESM4" class="usa-link">3</a>. All formatted data required to reproduce the results are stored in the GitHub repository (<a href="https://github.com/Han-Cheng22/Brucella_zoonotic_potential_prediction_models/blob/main/Data.zip" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Han-Cheng22/Brucella_zoonotic_potential_prediction_models/blob/main/Data.zip</a>).</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>All code to reproduce the analysis and generate the figures in this article is available at: <a href="https://github.com/Han-Cheng22/Brucella_zoonotic_potential_prediction_models" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Han-Cheng22/Brucella_zoonotic_potential_prediction_models</a>.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par48">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisherâ€™s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Cheng Han, Shiying Lu, Pan Hu.</p></div>
</div></section><section id="sec22"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s42003-025-08650-3.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Carroll-Portillo, A. et al. Mucin and Agitation Shape Predation of Escherichia coli by Lytic Coliphage. <em>Microorganisms</em><strong>11</strong>, 508 (2023).
</cite> [<a href="https://doi.org/10.3390/microorganisms11020508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9966288/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36838472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carroll-Portillo,%20A.%20et%20al.%20Mucin%20and%20Agitation%20Shape%20Predation%20of%20Escherichia%20coli%20by%20Lytic%20Coliphage.%20Microorganisms11,%20508%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Pan, Y. et al. Drought-induced assembly of rhizosphere mycobiomes shows beneficial effects on plant growth. <em>mSystems</em><strong>9</strong>, e0035424 (2024).
</cite> [<a href="https://doi.org/10.1128/msystems.00354-24" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11264929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38842321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan,%20Y.%20et%20al.%20Drought-induced%20assembly%20of%20rhizosphere%20mycobiomes%20shows%20beneficial%20effects%20on%20plant%20growth.%20mSystems9,%20e0035424%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Gupta, A. et al. Fungal Endophytes: Microfactories of Novel Bioactive Compounds with Therapeutic Interventions; A Comprehensive Review on the Biotechnological Developments in the Field of Fungal Endophytic Biology over the Last Decade. <em>Biomolecules</em><strong>13</strong>, 1038 (2023).
</cite> [<a href="https://doi.org/10.3390/biom13071038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10377637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37509074/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gupta,%20A.%20et%20al.%20Fungal%20Endophytes:%20Microfactories%20of%20Novel%20Bioactive%20Compounds%20with%20Therapeutic%20Interventions;%20A%20Comprehensive%20Review%20on%20the%20Biotechnological%20Developments%20in%20the%20Field%20of%20Fungal%20Endophytic%20Biology%20over%20the%20Last%20Decade.%20Biomolecules13,%201038%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Chen, S., Pham, S., Terrapon, N., Blom, J. &amp; Walker, E. D. Elizabethkingia anophelis MSU001 Isolated from Anopheles stephensi: Molecular Characterization and Comparative Genome Analysis. <em>Microorganisms</em><strong>12</strong>, 1079 (2024).
</cite> [<a href="https://doi.org/10.3390/microorganisms12061079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11206156/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38930461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20S.,%20Pham,%20S.,%20Terrapon,%20N.,%20Blom,%20J.%20&amp;%20Walker,%20E.%20D.%20Elizabethkingia%20anophelis%20MSU001%20Isolated%20from%20Anopheles%20stephensi:%20Molecular%20Characterization%20and%20Comparative%20Genome%20Analysis.%20Microorganisms12,%201079%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>DerviÅŸ, S. &amp; Ã–zer, G. Plant-Associated Neoscytalidium dimidiatum-Taxonomy, Host Range, Epidemiology, Virulence, and Management Strategies: A Comprehensive Review. <em>J. Fungi (Basel)</em><strong>9</strong>, 1048 (2023).
</cite> [<a href="https://doi.org/10.3390/jof9111048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10672476/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37998855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dervi%C5%9F,%20S.%20&amp;%20%C3%96zer,%20G.%20Plant-Associated%20Neoscytalidium%20dimidiatum-Taxonomy,%20Host%20Range,%20Epidemiology,%20Virulence,%20and%20Management%20Strategies:%20A%20Comprehensive%20Review.%20J.%20Fungi%20(Basel)9,%201048%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Lajoie, G. &amp; Parfrey, L. W. Beyond specialization: re-examining routes of host influence on symbiont evolution. <em>Trends Ecol. Evolution</em><strong>37</strong>, 590â€“598 (2022).</cite> [<a href="https://doi.org/10.1016/j.tree.2022.03.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35466020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lajoie,%20G.%20&amp;%20Parfrey,%20L.%20W.%20Beyond%20specialization:%20re-examining%20routes%20of%20host%20influence%20on%20symbiont%20evolution.%20Trends%20Ecol.%20Evolution37,%20590%E2%80%93598%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Guo, J. J. et al. Diversity and circulation of Jingmen tick virus in ticks and mammals. <em>Virus Evol.</em><strong>6</strong>, veaa051 (2020).
</cite> [<a href="https://doi.org/10.1093/ve/veaa051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8097133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33976906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20J.%20J.%20et%20al.%20Diversity%20and%20circulation%20of%20Jingmen%20tick%20virus%20in%20ticks%20and%20mammals.%20Virus%20Evol.6,%20veaa051%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Kirkwood, M. et al. Ecological niche adaptation of Salmonella Typhimurium U288 is associated with altered pathogenicity and reduced zoonotic potential. <em>Commun. Biol.</em><strong>4</strong>, 498 (2021).
</cite> [<a href="https://doi.org/10.1038/s42003-021-02013-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8065163/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33893390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kirkwood,%20M.%20et%20al.%20Ecological%20niche%20adaptation%20of%20Salmonella%20Typhimurium%20U288%20is%20associated%20with%20altered%20pathogenicity%20and%20reduced%20zoonotic%20potential.%20Commun.%20Biol.4,%20498%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Sterner, B., Elliott, S., Upham, N. &amp; Franz, N. Bats, objectivity, and viral spillover risk. <em>Hist. Philos. Life Sci.</em><strong>43</strong>, 7 (2021).
</cite> [<a href="https://doi.org/10.1007/s40656-021-00366-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7805256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33439354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sterner,%20B.,%20Elliott,%20S.,%20Upham,%20N.%20&amp;%20Franz,%20N.%20Bats,%20objectivity,%20and%20viral%20spillover%20risk.%20Hist.%20Philos.%20Life%20Sci.43,%207%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Occhialini, A. et al. The Retrospective on Atypical Brucella Species Leads to Novel Definitions. <em>Microorganisms</em><strong>10</strong>, 813 (2022).
</cite> [<a href="https://doi.org/10.3390/microorganisms10040813" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9025488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35456863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Occhialini,%20A.%20et%20al.%20The%20Retrospective%20on%20Atypical%20Brucella%20Species%20Leads%20to%20Novel%20Definitions.%20Microorganisms10,%20813%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Wareth, G. et al. MLVA-16 Genotyping of Brucella abortus and Brucella melitensis Isolates from Different Animal Species in Egypt: Geographical Relatedness and the Mediterranean Lineage. <em>Pathogens</em><strong>9</strong>, 498 (2020).
</cite> [<a href="https://doi.org/10.3390/pathogens9060498" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7350383/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32580472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wareth,%20G.%20et%20al.%20MLVA-16%20Genotyping%20of%20Brucella%20abortus%20and%20Brucella%20melitensis%20Isolates%20from%20Different%20Animal%20Species%20in%20Egypt:%20Geographical%20Relatedness%20and%20the%20Mediterranean%20Lineage.%20Pathogens9,%20498%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Whatmore, A. M. &amp; Foster, J. T. Emerging diversity and ongoing expansion of the genus Brucella. <em>Infect., Genet. Evolution</em><strong>92</strong>, 104865 (2021).</cite> [<a href="https://doi.org/10.1016/j.meegid.2021.104865" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33872784/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Whatmore,%20A.%20M.%20&amp;%20Foster,%20J.%20T.%20Emerging%20diversity%20and%20ongoing%20expansion%20of%20the%20genus%20Brucella.%20Infect.,%20Genet.%20Evolution92,%20104865%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Jiang, H. et al. MLVA genotyping of Brucella melitensis and Brucella abortus isolates from different animal species and humans and identification of Brucella suis vaccine strain S2 from cattle in China. <em>PLoS One</em>. <strong>8</strong>, e76332 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0076332" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3790686/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24124546/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang,%20H.%20et%20al.%20MLVA%20genotyping%20of%20Brucella%20melitensis%20and%20Brucella%20abortus%20isolates%20from%20different%20animal%20species%20and%20humans%20and%20identification%20of%20Brucella%20suis%20vaccine%20strain%20S2%20from%20cattle%20in%20China.%20PLoS%20One.%208,%20e76332%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Villalobos-Vindas, J. M. et al. Brucellosis caused by the wood rat pathogen Brucella neotomae: two case reports. <em>J. Med. Case Rep.</em><strong>11</strong>, 352 (2017).
</cite> [<a href="https://doi.org/10.1186/s13256-017-1496-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5735802/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29254496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Villalobos-Vindas,%20J.%20M.%20et%20al.%20Brucellosis%20caused%20by%20the%20wood%20rat%20pathogen%20Brucella%20neotomae:%20two%20case%20reports.%20J.%20Med.%20Case%20Rep.11,%20352%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Kolwijck, E. et al. First Case of Human Brucella canis Infection in the Netherlands. <em>Clin. Infect. Dis.</em><strong>75</strong>, 2250â€“2252 (2022).
</cite> [<a href="https://doi.org/10.1093/cid/ciac425" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35653425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kolwijck,%20E.%20et%20al.%20First%20Case%20of%20Human%20Brucella%20canis%20Infection%20in%20the%20Netherlands.%20Clin.%20Infect.%20Dis.75,%202250%E2%80%932252%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Vergnaud, G., Zygmunt, M. S., Ashford, R. T., Whatmore, A. M. &amp; Cloeckaert, A. Genomic Diversity and Zoonotic Potential of Brucella neotomae. <em>Emerg. Infect. Dis.</em><strong>30</strong>, 155â€“158 (2024).
</cite> [<a href="https://doi.org/10.3201/eid3001.221783" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10756370/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38147057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vergnaud,%20G.,%20Zygmunt,%20M.%20S.,%20Ashford,%20R.%20T.,%20Whatmore,%20A.%20M.%20&amp;%20Cloeckaert,%20A.%20Genomic%20Diversity%20and%20Zoonotic%20Potential%20of%20Brucella%20neotomae.%20Emerg.%20Infect.%20Dis.30,%20155%E2%80%93158%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>SuÃ¡rez-Esquivel, M., Chaves-Olarte, E., Moreno, E. &amp; GuzmÃ¡n-Verri, C. Brucella Genomics: Macro and Micro Evolution. <em>Int J. Mol. Sci.</em><strong>21</strong>, 7749 (2020).
</cite> [<a href="https://doi.org/10.3390/ijms21207749" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7589603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33092044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su%C3%A1rez-Esquivel,%20M.,%20Chaves-Olarte,%20E.,%20Moreno,%20E.%20&amp;%20Guzm%C3%A1n-Verri,%20C.%20Brucella%20Genomics:%20Macro%20and%20Micro%20Evolution.%20Int%20J.%20Mol.%20Sci.21,%207749%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Asada, K. et al. Integrated Analysis of Whole Genome and Epigenome Data Using Machine Learning Technology: Toward the Establishment of Precision Oncology. <em>Front Oncol.</em><strong>11</strong>, 666937 (2021).
</cite> [<a href="https://doi.org/10.3389/fonc.2021.666937" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8149908/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34055633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Asada,%20K.%20et%20al.%20Integrated%20Analysis%20of%20Whole%20Genome%20and%20Epigenome%20Data%20Using%20Machine%20Learning%20Technology:%20Toward%20the%20Establishment%20of%20Precision%20Oncology.%20Front%20Oncol.11,%20666937%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Cerezo, M., Verdon, G., Huang, H.-Y., Cincio, L. &amp; Coles, P. J. Challenges and opportunities in quantum machine learning. <em>Nat. Computational Sci.</em><strong>2</strong>, 567â€“576 (2022).</cite> [<a href="https://doi.org/10.1038/s43588-022-00311-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38177473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cerezo,%20M.,%20Verdon,%20G.,%20Huang,%20H.-Y.,%20Cincio,%20L.%20&amp;%20Coles,%20P.%20J.%20Challenges%20and%20opportunities%20in%20quantum%20machine%20learning.%20Nat.%20Computational%20Sci.2,%20567%E2%80%93576%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Eckardt, J. N., BornhÃ¤user, M., Wendt, K. &amp; Middeke, J. M. Semi-supervised learning in cancer diagnostics. <em>Front Oncol.</em><strong>12</strong>, 960984 (2022).
</cite> [<a href="https://doi.org/10.3389/fonc.2022.960984" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9329803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35912249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eckardt,%20J.%20N.,%20Bornh%C3%A4user,%20M.,%20Wendt,%20K.%20&amp;%20Middeke,%20J.%20M.%20Semi-supervised%20learning%20in%20cancer%20diagnostics.%20Front%20Oncol.12,%20960984%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Schrider, D. R. &amp; Kern, A. D. Supervised Machine Learning for Population Genetics: A New Paradigm. <em>Trends Genet</em>. <strong>34</strong>, 301â€“312 (2018).
</cite> [<a href="https://doi.org/10.1016/j.tig.2017.12.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5905713/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29331490/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schrider,%20D.%20R.%20&amp;%20Kern,%20A.%20D.%20Supervised%20Machine%20Learning%20for%20Population%20Genetics:%20A%20New%20Paradigm.%20Trends%20Genet.%2034,%20301%E2%80%93312%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Uddin, S., Khan, A., Hossain, M. E. &amp; Moni, M. A. Comparing different supervised machine learning algorithms for disease prediction. <em>BMC Med Inf. Decis. Mak.</em><strong>19</strong>, 281 (2019).</cite> [<a href="https://doi.org/10.1186/s12911-019-1004-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6925840/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31864346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Uddin,%20S.,%20Khan,%20A.,%20Hossain,%20M.%20E.%20&amp;%20Moni,%20M.%20A.%20Comparing%20different%20supervised%20machine%20learning%20algorithms%20for%20disease%20prediction.%20BMC%20Med%20Inf.%20Decis.%20Mak.19,%20281%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Chatterjee, A., Saha, J. &amp; Mukherjee, J. Clustering with multi-layered perceptron. <em>Pattern Recognit. Lett.</em><strong>155</strong>, 92â€“99 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Chatterjee,%20A.,%20Saha,%20J.%20&amp;%20Mukherjee,%20J.%20Clustering%20with%20multi-layered%20perceptron.%20Pattern%20Recognit.%20Lett.155,%2092%E2%80%9399%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Mock, F., Viehweger, A., Barth, E. &amp; Marz, M. VIDHOP, viral host prediction with deep learning. <em>Bioinformatics</em><strong>37</strong>, 318â€“325 (2021).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa705" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7454304/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32777818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mock,%20F.,%20Viehweger,%20A.,%20Barth,%20E.%20&amp;%20Marz,%20M.%20VIDHOP,%20viral%20host%20prediction%20with%20deep%20learning.%20Bioinformatics37,%20318%E2%80%93325%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Karabulut, O. C., Karpuzcu, B. A., TÃ¼rk, E., Ibrahim, A. H. &amp; SÃ¼zek, B. E. ML-AdVInfect: A Machine-Learning Based Adenoviral Infection Predictor. <em>Front Mol. Biosci.</em><strong>8</strong>, 647424 (2021).
</cite> [<a href="https://doi.org/10.3389/fmolb.2021.647424" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8139618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34026828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karabulut,%20O.%20C.,%20Karpuzcu,%20B.%20A.,%20T%C3%BCrk,%20E.,%20Ibrahim,%20A.%20H.%20&amp;%20S%C3%BCzek,%20B.%20E.%20ML-AdVInfect:%20A%20Machine-Learning%20Based%20Adenoviral%20Infection%20Predictor.%20Front%20Mol.%20Biosci.8,%20647424%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Bandoy, D. D. R. &amp; Weimer, B. C. Biological Machine Learning Combined with Campylobacter Population Genomics Reveals Virulence Gene Allelic Variants Cause Disease. <em>Microorganisms</em><strong>8</strong>, 549 (2020).
</cite> [<a href="https://doi.org/10.3390/microorganisms8040549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7232492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32290186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bandoy,%20D.%20D.%20R.%20&amp;%20Weimer,%20B.%20C.%20Biological%20Machine%20Learning%20Combined%20with%20Campylobacter%20Population%20Genomics%20Reveals%20Virulence%20Gene%20Allelic%20Variants%20Cause%20Disease.%20Microorganisms8,%20549%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Im, H., Hwang, S. H., Kim, B. S. &amp; Choi, S. H. Pathogenic potential assessment of the Shiga toxin-producing Escherichia coli by a source attribution-considered machine learning model. <em>Proc. Natl Acad. Sci. USA</em><strong>118</strong>, e2018877118 (2021).
</cite> [<a href="https://doi.org/10.1073/pnas.2018877118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8157976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33986113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Im,%20H.,%20Hwang,%20S.%20H.,%20Kim,%20B.%20S.%20&amp;%20Choi,%20S.%20H.%20Pathogenic%20potential%20assessment%20of%20the%20Shiga%20toxin-producing%20Escherichia%20coli%20by%20a%20source%20attribution-considered%20machine%20learning%20model.%20Proc.%20Natl%20Acad.%20Sci.%20USA118,%20e2018877118%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Medini, D., Donati, C., Tettelin, H., Masignani, V. &amp; Rappuoli, R. The microbial pan-genome. <em>Curr. Opin. Genet Dev.</em><strong>15</strong>, 589â€“594 (2005).
</cite> [<a href="https://doi.org/10.1016/j.gde.2005.09.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16185861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Medini,%20D.,%20Donati,%20C.,%20Tettelin,%20H.,%20Masignani,%20V.%20&amp;%20Rappuoli,%20R.%20The%20microbial%20pan-genome.%20Curr.%20Opin.%20Genet%20Dev.15,%20589%E2%80%93594%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Rajput, A. et al. Pangenome analysis reveals the genetic basis for taxonomic classification of the Lactobacillaceae family. <em>Food Microbiol.</em><strong>115</strong>, 104334 (2023).
</cite> [<a href="https://doi.org/10.1016/j.fm.2023.104334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37567624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rajput,%20A.%20et%20al.%20Pangenome%20analysis%20reveals%20the%20genetic%20basis%20for%20taxonomic%20classification%20of%20the%20Lactobacillaceae%20family.%20Food%20Microbiol.115,%20104334%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Mancilla, M. et al. Deletion of the GI-2 integrase and the wbkA flanking transposase improves the stability of Brucella melitensis Rev 1 vaccine. <em>Vet. Res</em>. <strong>44</strong>, 105 (2013).
</cite> [<a href="https://doi.org/10.1186/1297-9716-44-105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4176087/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24176078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mancilla,%20M.%20et%20al.%20Deletion%20of%20the%20GI-2%20integrase%20and%20the%20wbkA%20flanking%20transposase%20improves%20the%20stability%20of%20Brucella%20melitensis%20Rev%201%20vaccine.%20Vet.%20Res.%2044,%20105%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>De Massis, F. et al. Distribution of Brucella field strains isolated from livestock, wildlife populations, and humans in Italy from 2007 to 2015. <em>PLoS One</em><strong>14</strong>, e0213689 (2019).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0213689" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6430384/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30901346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Massis,%20F.%20et%20al.%20Distribution%20of%20Brucella%20field%20strains%20isolated%20from%20livestock,%20wildlife%20populations,%20and%20humans%20in%20Italy%20from%202007%20to%202015.%20PLoS%20One14,%20e0213689%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Rossetti, C. A., Maurizio, E. &amp; Rossi, U. A. Comparative Review of Brucellosis in Small Domestic Ruminants. <em>Front Vet. Sci.</em><strong>9</strong>, 887671 (2022).
</cite> [<a href="https://doi.org/10.3389/fvets.2022.887671" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9133814/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35647101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rossetti,%20C.%20A.,%20Maurizio,%20E.%20&amp;%20Rossi,%20U.%20A.%20Comparative%20Review%20of%20Brucellosis%20in%20Small%20Domestic%20Ruminants.%20Front%20Vet.%20Sci.9,%20887671%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Khurana, S. K. et al. Bovine brucellosis - a comprehensive review. <em>Vet. Q</em><strong>41</strong>, 61â€“88 (2021).
</cite> [<a href="https://doi.org/10.1080/01652176.2020.1868616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7833053/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33353489/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khurana,%20S.%20K.%20et%20al.%20Bovine%20brucellosis%20-%20a%20comprehensive%20review.%20Vet.%20Q41,%2061%E2%80%9388%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Sankarasubramanian, J., Vishnu, U. S., Gunasekaran, P. &amp; Rajendhran, J. A genome-wide SNP-based phylogenetic analysis distinguishes different biovars of Brucella suis. <em>Infect., Genet. Evolution</em><strong>41</strong>, 213â€“217 (2016).</cite> [<a href="https://doi.org/10.1016/j.meegid.2016.04.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27085292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sankarasubramanian,%20J.,%20Vishnu,%20U.%20S.,%20Gunasekaran,%20P.%20&amp;%20Rajendhran,%20J.%20A%20genome-wide%20SNP-based%20phylogenetic%20analysis%20distinguishes%20different%20biovars%20of%20Brucella%20suis.%20Infect.,%20Genet.%20Evolution41,%20213%E2%80%93217%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Tsolis, R. M. et al. Genome degradation in Brucella ovis corresponds with narrowing of its host range and tissue tropism. <em>PLoS One</em><strong>4</strong>, e5519 (2009).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0005519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2677664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19436743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsolis,%20R.%20M.%20et%20al.%20Genome%20degradation%20in%20Brucella%20ovis%20corresponds%20with%20narrowing%20of%20its%20host%20range%20and%20tissue%20tropism.%20PLoS%20One4,%20e5519%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Stranahan, L. W. &amp; Arenas-Gamboa, A. M. When the Going Gets Rough: The Significance of Brucella Lipopolysaccharide Phenotype in Host-Pathogen Interactions. <em>Front Microbiol</em><strong>12</strong>, 713157 (2021).
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.713157" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8319746/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34335551/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stranahan,%20L.%20W.%20&amp;%20Arenas-Gamboa,%20A.%20M.%20When%20the%20Going%20Gets%20Rough:%20The%20Significance%20of%20Brucella%20Lipopolysaccharide%20Phenotype%20in%20Host-Pathogen%20Interactions.%20Front%20Microbiol12,%20713157%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Yang, X. et al. Whole-genome sequencing of rough Brucella melitensis in China provides insights into its genetic features. <em>Emerg. Microbes Infect.</em><strong>9</strong>, 2147â€“2156 (2020).
</cite> [<a href="https://doi.org/10.1080/22221751.2020.1824549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7580580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32936049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20X.%20et%20al.%20Whole-genome%20sequencing%20of%20rough%20Brucella%20melitensis%20in%20China%20provides%20insights%20into%20its%20genetic%20features.%20Emerg.%20Microbes%20Infect.9,%202147%E2%80%932156%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Dentinger, C. M. et al. Human Brucella canis Infection and Subsequent Laboratory Exposures Associated with a Puppy, New York City, 2012. <em>Zoonoses Public Health</em><strong>62</strong>, 407â€“414 (2015).
</cite> [<a href="https://doi.org/10.1111/zph.12163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4639931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25363807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dentinger,%20C.%20M.%20et%20al.%20Human%20Brucella%20canis%20Infection%20and%20Subsequent%20Laboratory%20Exposures%20Associated%20with%20a%20Puppy,%20New%20York%20City,%202012.%20Zoonoses%20Public%20Health62,%20407%E2%80%93414%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Ahmed-Bentley, J. et al. Laboratory Exposures from an Unsuspected Case of Human Infection with Brucella canis. <em>Emerg. Infect. Dis.</em><strong>27</strong>, 2489â€“2491 (2021).
</cite> [<a href="https://doi.org/10.3201/eid2709.204701" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8386805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34424169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahmed-Bentley,%20J.%20et%20al.%20Laboratory%20Exposures%20from%20an%20Unsuspected%20Case%20of%20Human%20Infection%20with%20Brucella%20canis.%20Emerg.%20Infect.%20Dis.27,%202489%E2%80%932491%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Orsini, M., Ianni, A. &amp; Zinzula, L. Brucella ceti and Brucella pinnipedialis genome characterization unveils genetic features that highlight their zoonotic potential. <em>MicrobiologyOpen</em><strong>11</strong>, e1329 (2022).
</cite> [<a href="https://doi.org/10.1002/mbo3.1329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9597259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36314752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Orsini,%20M.,%20Ianni,%20A.%20&amp;%20Zinzula,%20L.%20Brucella%20ceti%20and%20Brucella%20pinnipedialis%20genome%20characterization%20unveils%20genetic%20features%20that%20highlight%20their%20zoonotic%20potential.%20MicrobiologyOpen11,%20e1329%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>RÃ³nai, Z. et al. First isolation and characterization of Brucella microti from wild boar. <em>BMC Vet. Res</em>. <strong>11</strong>, 147 (2015).
</cite> [<a href="https://doi.org/10.1186/s12917-015-0456-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4499207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26163135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?R%C3%B3nai,%20Z.%20et%20al.%20First%20isolation%20and%20characterization%20of%20Brucella%20microti%20from%20wild%20boar.%20BMC%20Vet.%20Res.%2011,%20147%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Scholz, H. C. et al. Brucella vulpis sp. nov., isolated from mandibular lymph nodes of red foxes (Vulpes vulpes). <em>Int. J. Syst. Evolut. Microbiol.</em><strong>66</strong>, 2090â€“2098 (2016).</cite> [<a href="https://doi.org/10.1099/ijsem.0.000998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26928956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scholz,%20H.%20C.%20et%20al.%20Brucella%20vulpis%20sp.%20nov.,%20isolated%20from%20mandibular%20lymph%20nodes%20of%20red%20foxes%20(Vulpes%20vulpes).%20Int.%20J.%20Syst.%20Evolut.%20Microbiol.66,%202090%E2%80%932098%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Scholz, H. C. et al. Brucella inopinata sp. nov., isolated from a breast implant infection. <em>Int. J. Syst. Evolut. Microbiol.</em><strong>60</strong>, 801â€“808 (2010).</cite> [<a href="https://doi.org/10.1099/ijs.0.011148-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19661515/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scholz,%20H.%20C.%20et%20al.%20Brucella%20inopinata%20sp.%20nov.,%20isolated%20from%20a%20breast%20implant%20infection.%20Int.%20J.%20Syst.%20Evolut.%20Microbiol.60,%20801%E2%80%93808%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Yang, Z. et al. Comparative genomic analysis provides insights into the genetic diversity and pathogenicity of the genus Brucella. <em>Front. Microbiol.</em><strong>15</strong>, 1389859 (2024).
</cite> [<a href="https://doi.org/10.3389/fmicb.2024.1389859" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11076708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38721599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20Z.%20et%20al.%20Comparative%20genomic%20analysis%20provides%20insights%20into%20the%20genetic%20diversity%20and%20pathogenicity%20of%20the%20genus%20Brucella.%20Front.%20Microbiol.15,%201389859%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Moreno, E. Retrospective and prospective perspectives on zoonotic brucellosis. <em>Front Microbiol</em><strong>5</strong>, 213 (2014).
</cite> [<a href="https://doi.org/10.3389/fmicb.2014.00213" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4026726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24860561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moreno,%20E.%20Retrospective%20and%20prospective%20perspectives%20on%20zoonotic%20brucellosis.%20Front%20Microbiol5,%20213%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>SuÃ¡rez-Esquivel, M. et al. Brucella neotomae Infection in Humans, Costa Rica. <em>Emerg. Infect. Dis.</em><strong>23</strong>, 997â€“1000 (2017).
</cite> [<a href="https://doi.org/10.3201/eid2306.162018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5443450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28518028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su%C3%A1rez-Esquivel,%20M.%20et%20al.%20Brucella%20neotomae%20Infection%20in%20Humans,%20Costa%20Rica.%20Emerg.%20Infect.%20Dis.23,%20997%E2%80%931000%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Oonishi, H. &amp; Iima, H. in <em>2017</em><em>IEEE</em> Conference on Computational Intelligence and Games (<em>C. I. G</em>.). 232-239.</cite>
</li>
<li id="CR48">
<span class="label">48.</span><cite>Oseguera Montiel, D., Frankena, K., Udo, H. &amp; van der Zijpp, A. Opportunities for Brucellosis Control in Mexico: Views Based on the Sustainable Livelihoods Perspective. <em>Front Vet. Sci.</em><strong>6</strong>, 216 (2019).
</cite> [<a href="https://doi.org/10.3389/fvets.2019.00216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6614285/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31312642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oseguera%20Montiel,%20D.,%20Frankena,%20K.,%20Udo,%20H.%20&amp;%20van%20der%20Zijpp,%20A.%20Opportunities%20for%20Brucellosis%20Control%20in%20Mexico:%20Views%20Based%20on%20the%20Sustainable%20Livelihoods%20Perspective.%20Front%20Vet.%20Sci.6,%20216%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Holzapfel, M. et al. Comparative Genomics and in vitro Infection of Field Clonal Isolates of Brucella melitensis Biovar 3 Did Not Identify Signature of Host Adaptation. <em>Front Microbiol</em>. <strong>9</strong>, 2505 (2018).
</cite> [<a href="https://doi.org/10.3389/fmicb.2018.02505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6204395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30405566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Holzapfel,%20M.%20et%20al.%20Comparative%20Genomics%20and%20in%20vitro%20Infection%20of%20Field%20Clonal%20Isolates%20of%20Brucella%20melitensis%20Biovar%203%20Did%20Not%20Identify%20Signature%20of%20Host%20Adaptation.%20Front%20Microbiol.%209,%202505%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Uncu, H. et al. A case of brucellosis presenting with urinary tract infection]. <em>Mikrobiyol. Bul.</em><strong>40</strong>, 275â€“278 (2006).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/17001859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Uncu,%20H.%20et%20al.%20A%20case%20of%20brucellosis%20presenting%20with%20urinary%20tract%20infection%5D.%20Mikrobiyol.%20Bul.40,%20275%E2%80%93278%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Vandercam, B. et al. Isolation of Brucella melitensis from human sperm. <em>Eur. J. Clin. Microbiol Infect. Dis.</em><strong>9</strong>, 303â€“304 (1990).
</cite> [<a href="https://doi.org/10.1007/BF01968069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2351148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vandercam,%20B.%20et%20al.%20Isolation%20of%20Brucella%20melitensis%20from%20human%20sperm.%20Eur.%20J.%20Clin.%20Microbiol%20Infect.%20Dis.9,%20303%E2%80%93304%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Tikare, N. V., Mantur, B. G. &amp; Bidari, L. H. Brucellar Meningitis in an Infantâ€”Evidence for Human Breast Milk Transmission. <em>J. Tropical Pediatrics</em><strong>54</strong>, 272â€“274 (2008).</cite> [<a href="https://doi.org/10.1093/tropej/fmn017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18292100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tikare,%20N.%20V.,%20Mantur,%20B.%20G.%20&amp;%20Bidari,%20L.%20H.%20Brucellar%20Meningitis%20in%20an%20Infant%E2%80%94Evidence%20for%20Human%20Breast%20Milk%20Transmission.%20J.%20Tropical%20Pediatrics54,%20272%E2%80%93274%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Tian, G. et al. A case report on mother-to-child transmission of Brucella in human, China. <em>BMC Infect. Dis.</em><strong>19</strong>, 666 (2019).
</cite> [<a href="https://doi.org/10.1186/s12879-019-4302-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6661076/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31351446/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tian,%20G.%20et%20al.%20A%20case%20report%20on%20mother-to-child%20transmission%20of%20Brucella%20in%20human,%20China.%20BMC%20Infect.%20Dis.19,%20666%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Meltzer, E. et al. Sexually Transmitted Brucellosis in Humans. <em>Clin. Infect. Dis.</em><strong>51</strong>, e12â€“e15 (2010).
</cite> [<a href="https://doi.org/10.1086/653608" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20550455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meltzer,%20E.%20et%20al.%20Sexually%20Transmitted%20Brucellosis%20in%20Humans.%20Clin.%20Infect.%20Dis.51,%20e12%E2%80%93e15%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>AkÃ§akuÅŸ, M., Esel, D., Cetin, N., Kisaarslan, A. P. &amp; KurtoÄŸlu, S. Brucella melitensis in blood cultures of two newborns due to exchange transfusion. <em>Turk. J. Pediatr.</em><strong>47</strong>, 272â€“274 (2005).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/16250315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ak%C3%A7aku%C5%9F,%20M.,%20Esel,%20D.,%20Cetin,%20N.,%20Kisaarslan,%20A.%20P.%20&amp;%20Kurto%C4%9Flu,%20S.%20Brucella%20melitensis%20in%20blood%20cultures%20of%20two%20newborns%20due%20to%20exchange%20transfusion.%20Turk.%20J.%20Pediatr.47,%20272%E2%80%93274%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Meurens, F. et al. Animal board invited review: Risks of zoonotic disease emergence at the interface of wildlife and livestock systems. <em>Animal</em><strong>15</strong>, 100241 (2021).
</cite> [<a href="https://doi.org/10.1016/j.animal.2021.100241" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8172357/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34091225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meurens,%20F.%20et%20al.%20Animal%20board%20invited%20review:%20Risks%20of%20zoonotic%20disease%20emergence%20at%20the%20interface%20of%20wildlife%20and%20livestock%20systems.%20Animal15,%20100241%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Kreizinger, Z. et al. Genetic relatedness of Brucella suis biovar 2 isolates from hares, wild boars and domestic pigs. <em>Vet. Microbiol.</em><strong>172</strong>, 492â€“498 (2014).
</cite> [<a href="https://doi.org/10.1016/j.vetmic.2014.05.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24962519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kreizinger,%20Z.%20et%20al.%20Genetic%20relatedness%20of%20Brucella%20suis%20biovar%202%20isolates%20from%20hares,%20wild%20boars%20and%20domestic%20pigs.%20Vet.%20Microbiol.172,%20492%E2%80%93498%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Mailles, A. et al. Brucella suis biovar 2 infection in humans in France: emerging infection or better recognition?. <em>Epidemiol. Infect.</em><strong>145</strong>, 2711â€“2716 (2017).
</cite> [<a href="https://doi.org/10.1017/S0950268817001704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9148767/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28784192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mailles,%20A.%20et%20al.%20Brucella%20suis%20biovar%202%20infection%20in%20humans%20in%20France:%20emerging%20infection%20or%20better%20recognition?.%20Epidemiol.%20Infect.145,%202711%E2%80%932716%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Elmonir, W. et al. Isolation and molecular confirmation of Brucella suis biovar 2 from slaughtered pigs: an unanticipated biovar from domestic pigs in Egypt. <em>BMC Vet. Res</em><strong>18</strong>, 224 (2022).
</cite> [<a href="https://doi.org/10.1186/s12917-022-03332-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9195200/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35698071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elmonir,%20W.%20et%20al.%20Isolation%20and%20molecular%20confirmation%20of%20Brucella%20suis%20biovar%202%20from%20slaughtered%20pigs:%20an%20unanticipated%20biovar%20from%20domestic%20pigs%20in%20Egypt.%20BMC%20Vet.%20Res18,%20224%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Akhtar, A. A. &amp; Turner, D. P. J. The role of bacterial ATP-binding cassette (ABC) transporters in pathogenesis and virulence: Therapeutic and vaccine potential. <em>Microb. Pathogenesis</em><strong>171</strong>, 105734 (2022).</cite> [<a href="https://doi.org/10.1016/j.micpath.2022.105734" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36007845/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akhtar,%20A.%20A.%20&amp;%20Turner,%20D.%20P.%20J.%20The%20role%20of%20bacterial%20ATP-binding%20cassette%20(ABC)%20transporters%20in%20pathogenesis%20and%20virulence:%20Therapeutic%20and%20vaccine%20potential.%20Microb.%20Pathogenesis171,%20105734%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Kornspan, D., Zahavi, T. &amp; Salmon-Divon, M. The Acidic Stress Response of the Intracellular Pathogen Brucella melitensis: New Insights from a Comparative, Genome-Wide Transcriptome Analysis. <em>Genes</em>. <strong>11</strong>, 1016 (2020).
</cite> [<a href="https://doi.org/10.3390/genes11091016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7563570/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32872264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kornspan,%20D.,%20Zahavi,%20T.%20&amp;%20Salmon-Divon,%20M.%20The%20Acidic%20Stress%20Response%20of%20the%20Intracellular%20Pathogen%20Brucella%20melitensis:%20New%20Insights%20from%20a%20Comparative,%20Genome-Wide%20Transcriptome%20Analysis.%20Genes.%2011,%201016%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Torres, L. H. M., Arrais, J. P. &amp; Ribeiro, B. Combining graph neural networks and transformers for few-shot nuclear receptor binding activity prediction. <em>J. Cheminformatics</em>. <strong>16</strong>, 109 (2024).</cite> [<a href="https://doi.org/10.1186/s13321-024-00902-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11429188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39334272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Torres,%20L.%20H.%20M.,%20Arrais,%20J.%20P.%20&amp;%20Ribeiro,%20B.%20Combining%20graph%20neural%20networks%20and%20transformers%20for%20few-shot%20nuclear%20receptor%20binding%20activity%20prediction.%20J.%20Cheminformatics.%2016,%20109%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Lupolova, N., Dallman, T. J., Matthews, L., Bono, J. L. &amp; Gally, D. L. Support vector machine applied to predict the zoonotic potential of E. coli O157 cattle isolates. <em>Proc. Natl Acad. Sci. USA</em><strong>113</strong>, 11312â€“11317 (2016).
</cite> [<a href="https://doi.org/10.1073/pnas.1606567113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5056084/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27647883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lupolova,%20N.,%20Dallman,%20T.%20J.,%20Matthews,%20L.,%20Bono,%20J.%20L.%20&amp;%20Gally,%20D.%20L.%20Support%20vector%20machine%20applied%20to%20predict%20the%20zoonotic%20potential%20of%20E.%20coli%20O157%20cattle%20isolates.%20Proc.%20Natl%20Acad.%20Sci.%20USA113,%2011312%E2%80%9311317%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Lupolova, N., Dallman, T. J., Holden, N. J. &amp; Gally, D. L. Patchy promiscuity: Machine learning applied to predict the host specificity of Salmonella enterica and Escherichia coli. <em>Microb. Genomics</em><strong>3</strong>, e000135 (2017).</cite> [<a href="https://doi.org/10.1099/mgen.0.000135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5695212/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29177093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lupolova,%20N.,%20Dallman,%20T.%20J.,%20Holden,%20N.%20J.%20&amp;%20Gally,%20D.%20L.%20Patchy%20promiscuity:%20Machine%20learning%20applied%20to%20predict%20the%20host%20specificity%20of%20Salmonella%20enterica%20and%20Escherichia%20coli.%20Microb.%20Genomics3,%20e000135%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Viana, D. et al. A single natural nucleotide mutation alters bacterial pathogen host tropism. <em>Nat. Genet</em><strong>47</strong>, 361â€“366 (2015).
</cite> [<a href="https://doi.org/10.1038/ng.3219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4824278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25685890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Viana,%20D.%20et%20al.%20A%20single%20natural%20nucleotide%20mutation%20alters%20bacterial%20pathogen%20host%20tropism.%20Nat.%20Genet47,%20361%E2%80%93366%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Yoshida, H., Bogaki, M., Nakamura, M. &amp; Nakamura, S. Quinolone resistance-determining region in the DNA gyrase gyrA gene of Escherichia coli. <em>Antimicrob. Agents Chemother.</em><strong>34</strong>, 1271â€“1272 (1990).
</cite> [<a href="https://doi.org/10.1128/aac.34.6.1271" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC171799/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2168148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshida,%20H.,%20Bogaki,%20M.,%20Nakamura,%20M.%20&amp;%20Nakamura,%20S.%20Quinolone%20resistance-determining%20region%20in%20the%20DNA%20gyrase%20gyrA%20gene%20of%20Escherichia%20coli.%20Antimicrob.%20Agents%20Chemother.34,%201271%E2%80%931272%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Gurevich, A., Saveliev, V., Vyahhi, N. &amp; Tesler, G. QUAST: quality assessment tool for genome assemblies. <em>Bioinformatics</em><strong>29</strong>, 1072â€“1075 (2013).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3624806/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23422339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gurevich,%20A.,%20Saveliev,%20V.,%20Vyahhi,%20N.%20&amp;%20Tesler,%20G.%20QUAST:%20quality%20assessment%20tool%20for%20genome%20assemblies.%20Bioinformatics29,%201072%E2%80%931075%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Seemann, T. Prokka: rapid prokaryotic genome annotation. <em>Bioinformatics</em><strong>30</strong>, 2068â€“2069 (2014).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24642063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seemann,%20T.%20Prokka:%20rapid%20prokaryotic%20genome%20annotation.%20Bioinformatics30,%202068%E2%80%932069%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Page, A. J. et al. Roary: rapid large-scale prokaryote pan genome analysis. <em>Bioinformatics</em><strong>31</strong>, 3691â€“3693 (2015).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4817141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26198102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Page,%20A.%20J.%20et%20al.%20Roary:%20rapid%20large-scale%20prokaryote%20pan%20genome%20analysis.%20Bioinformatics31,%203691%E2%80%933693%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Cantalapiedra, C. P., HernÃ¡ndez-Plaza, A., Letunic, I., Bork, P. &amp; Huerta-Cepas, J. eggNOG-mapper v2: Functional Annotation, Orthology Assignments, and Domain Prediction at the Metagenomic Scale. <em>Mol. Biol. Evolution</em><strong>38</strong>, 5825â€“5829 (2021).</cite> [<a href="https://doi.org/10.1093/molbev/msab293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8662613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34597405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cantalapiedra,%20C.%20P.,%20Hern%C3%A1ndez-Plaza,%20A.,%20Letunic,%20I.,%20Bork,%20P.%20&amp;%20Huerta-Cepas,%20J.%20eggNOG-mapper%20v2:%20Functional%20Annotation,%20Orthology%20Assignments,%20and%20Domain%20Prediction%20at%20the%20Metagenomic%20Scale.%20Mol.%20Biol.%20Evolution38,%205825%E2%80%935829%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Elbehiry, A. et al. The Development of Diagnostic and Vaccine Strategies for Early Detection and Control of Human Brucellosis, Particularly in Endemic Areas. <em>Vaccines (Basel)</em><strong>11</strong>, 654</cite> [<a href="https://doi.org/10.3390/vaccines11030654" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10054502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36992237/" class="usa-link">PubMed</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Aljanazreh, B., Shamseye, A. A., Abuawad, A. &amp; Ashhab, Y. Genomic distribution of the insertion sequence IS711 reveal a potential role in Brucella genome plasticity and host preference. <em>Infect., Genet. Evolution</em><strong>112</strong>, 105457 (2023).</cite> [<a href="https://doi.org/10.1016/j.meegid.2023.105457" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37257801/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aljanazreh,%20B.,%20Shamseye,%20A.%20A.,%20Abuawad,%20A.%20&amp;%20Ashhab,%20Y.%20Genomic%20distribution%20of%20the%20insertion%20sequence%20IS711%20reveal%20a%20potential%20role%20in%20Brucella%20genome%20plasticity%20and%20host%20preference.%20Infect.,%20Genet.%20Evolution112,%20105457%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Deqiu, S., Donglou, X. &amp; Jiming, Y. Epidemiology and control of brucellosis in China. <em>Vet. Microbiol.</em><strong>90</strong>, 165â€“182 (2002).
</cite> [<a href="https://doi.org/10.1016/s0378-1135(02)00252-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12414142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deqiu,%20S.,%20Donglou,%20X.%20&amp;%20Jiming,%20Y.%20Epidemiology%20and%20control%20of%20brucellosis%20in%20China.%20Vet.%20Microbiol.90,%20165%E2%80%93182%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Sun, S. et al. Safety and Transcriptome Analysis of Live Attenuated Brucella Vaccine Strain S2 on Non-pregnant Cynomolgus Monkeys Without Abortive Effect on Pregnant Cynomolgus Monkeys. <em>Front. Vet. Sci.</em><strong>8</strong>, 641022 (2021).
</cite> [<a href="https://doi.org/10.3389/fvets.2021.641022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7985263/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33768120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20S.%20et%20al.%20Safety%20and%20Transcriptome%20Analysis%20of%20Live%20Attenuated%20Brucella%20Vaccine%20Strain%20S2%20on%20Non-pregnant%20Cynomolgus%20Monkeys%20Without%20Abortive%20Effect%20on%20Pregnant%20Cynomolgus%20Monkeys.%20Front.%20Vet.%20Sci.8,%20641022%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Brynildsrud, O., Bohlin, J., Scheffer, L. &amp; Eldholm, V. Rapid scoring of genes in microbial pan-genome-wide association studies with Scoary. <em>Genome Biol.</em><strong>17</strong>, 238 (2016).
</cite> [<a href="https://doi.org/10.1186/s13059-016-1108-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5124306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27887642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brynildsrud,%20O.,%20Bohlin,%20J.,%20Scheffer,%20L.%20&amp;%20Eldholm,%20V.%20Rapid%20scoring%20of%20genes%20in%20microbial%20pan-genome-wide%20association%20studies%20with%20Scoary.%20Genome%20Biol.17,%20238%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Edler, D., Klein, J., Antonelli, A. &amp; Silvestro, D. raxmlGUI 2.0: A graphical interface and toolkit for phylogenetic analyses using RAxML. <em>Methods Ecol. Evolution</em><strong>12</strong>, 373â€“377 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Edler,%20D.,%20Klein,%20J.,%20Antonelli,%20A.%20&amp;%20Silvestro,%20D.%20raxmlGUI%202.0:%20A%20graphical%20interface%20and%20toolkit%20for%20phylogenetic%20analyses%20using%20RAxML.%20Methods%20Ecol.%20Evolution12,%20373%E2%80%93377%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Xie, J. et al. Tree Visualization By One Table (tvBOT): a web application for visualizing, modifying and annotating phylogenetic trees. <em>Nucleic Acids Res.</em><strong>51</strong>, W587â€“W592 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkad359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10320113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37144476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie,%20J.%20et%20al.%20Tree%20Visualization%20By%20One%20Table%20(tvBOT):%20a%20web%20application%20for%20visualizing,%20modifying%20and%20annotating%20phylogenetic%20trees.%20Nucleic%20Acids%20Res.51,%20W587%E2%80%93W592%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368250/bin/42003_2025_8650_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (3.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368250/bin/42003_2025_8650_MOESM2_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (18.4KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368250/bin/42003_2025_8650_MOESM3_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (1.7KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368250/bin/42003_2025_8650_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (81.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368250/bin/42003_2025_8650_MOESM5_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">42003_2025_8650_MOESM5_ESM.docx</a><sup> (21.2KB, docx) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368250/bin/42003_2025_8650_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368250/bin/42003_2025_8650_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.3MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data supporting the findings of this study are available within the paper and its Supplementary Information. The genomic data for all <em>Brucella</em> strains analyzed in this study was obtained from the NCBI Assembly database (<a href="https://www.ncbi.nlm.nih.gov/assembly" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/assembly</a>), with strain-associated metadata available in Supplementary DataÂ <a href="#MOESM4" class="usa-link">3</a>. All formatted data required to reproduce the results are stored in the GitHub repository (<a href="https://github.com/Han-Cheng22/Brucella_zoonotic_potential_prediction_models/blob/main/Data.zip" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Han-Cheng22/Brucella_zoonotic_potential_prediction_models/blob/main/Data.zip</a>).</p>
<p>All code to reproduce the analysis and generate the figures in this article is available at: <a href="https://github.com/Han-Cheng22/Brucella_zoonotic_potential_prediction_models" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Han-Cheng22/Brucella_zoonotic_potential_prediction_models</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Communications Biology are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s42003-025-08650-3"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/42003_2025_Article_8650.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (11.0Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368250/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368250/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368250%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368250/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368250/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368250/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835741/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368250/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835741/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368250/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368250/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="eUMfSBXZCTE7jaNA4eal8BILDaPKYSY053pBHEJln3x0pzorzPGqmpJWNOpKWFRg">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
