
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            “In silico analysis of human TLR3 missense single nucleotide polymorphisms and their potential association with cancer” - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A1BB8AF1DDC305A1BB003CB17438.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373945/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="“In silico analysis of human TLR3 missense single nucleotide polymorphisms and their potential association with cancer”">
<meta name="citation_author" content="Mohini Agarwal">
<meta name="citation_author_institution" content="Therapeutics and Molecular Diagnostic Lab, J-3 Block, Amity Institute of Biotechnology, Amity University, Noida, Uttar Pradesh India">
<meta name="citation_author" content="Manish Kumar">
<meta name="citation_author_institution" content="Therapeutics and Molecular Diagnostic Lab, J-3 Block, Amity Institute of Biotechnology, Amity University, Noida, Uttar Pradesh India">
<meta name="citation_author" content="Sarthak Dahiya">
<meta name="citation_author_institution" content="Amity Institute of Pharmacy, Amity University, Noida, India">
<meta name="citation_author" content="Anoop Kumar">
<meta name="citation_author_institution" content="National Institute of Biologicals, A-32, Sector-62, Noida, Uttar Pradesh India">
<meta name="citation_author" content="Rupal Tripathi">
<meta name="citation_author_institution" content="Rajiv Gandhi Cancer Institute and Research Centre, Delhi, India">
<meta name="citation_author" content="Kumud Bala">
<meta name="citation_author_institution" content="Therapeutics and Molecular Diagnostic Lab, J-3 Block, Amity Institute of Biotechnology, Amity University, Noida, Uttar Pradesh India">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30837">
<meta name="citation_doi" content="10.1038/s41598-025-05599-5">
<meta name="citation_pmid" content="40847033">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373945/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373945/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373945/pdf/41598_2025_Article_5599.pdf">
<meta name="description" content="Cervical cancer is a prevalent malignancy worldwide and represents a significant health burden for women. Toll-like receptors (TLRs) are crucial components of the innate immune system and play a vital role in recognizing pathogen-associated ...">
<meta name="og:title" content="“In silico analysis of human TLR3 missense single nucleotide polymorphisms and their potential association with cancer”">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Cervical cancer is a prevalent malignancy worldwide and represents a significant health burden for women. Toll-like receptors (TLRs) are crucial components of the innate immune system and play a vital role in recognizing pathogen-associated ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373945/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373945">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-05599-5"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_5599.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373945%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373945/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373945/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373945/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 22;15:30837. doi: <a href="https://doi.org/10.1038/s41598-025-05599-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-05599-5</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>“In silico analysis of human TLR3 missense single nucleotide polymorphisms and their potential association with cancer”</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Agarwal%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Mohini Agarwal</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mohini Agarwal</span></h3>
<div class="p">
<sup>1</sup>Therapeutics and Molecular Diagnostic Lab, J-3 Block, Amity Institute of Biotechnology, Amity University, Noida, Uttar Pradesh India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Agarwal%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mohini Agarwal</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Manish Kumar</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Manish Kumar</span></h3>
<div class="p">
<sup>1</sup>Therapeutics and Molecular Diagnostic Lab, J-3 Block, Amity Institute of Biotechnology, Amity University, Noida, Uttar Pradesh India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Manish Kumar</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dahiya%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Sarthak Dahiya</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Sarthak Dahiya</span></h3>
<div class="p">
<sup>2</sup>Amity Institute of Pharmacy, Amity University, Noida, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dahiya%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sarthak Dahiya</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Anoop Kumar</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Anoop Kumar</span></h3>
<div class="p">
<sup>3</sup>National Institute of Biologicals, A-32, Sector-62, Noida, Uttar Pradesh India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anoop Kumar</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tripathi%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Rupal Tripathi</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Rupal Tripathi</span></h3>
<div class="p">
<sup>4</sup>Rajiv Gandhi Cancer Institute and Research Centre, Delhi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tripathi%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rupal Tripathi</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bala%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Kumud Bala</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Kumud Bala</span></h3>
<div class="p">
<sup>1</sup>Therapeutics and Molecular Diagnostic Lab, J-3 Block, Amity Institute of Biotechnology, Amity University, Noida, Uttar Pradesh India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bala%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kumud Bala</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Therapeutics and Molecular Diagnostic Lab, J-3 Block, Amity Institute of Biotechnology, Amity University, Noida, Uttar Pradesh India </div>
<div id="Aff2">
<sup>2</sup>Amity Institute of Pharmacy, Amity University, Noida, India </div>
<div id="Aff3">
<sup>3</sup>National Institute of Biologicals, A-32, Sector-62, Noida, Uttar Pradesh India </div>
<div id="Aff4">
<sup>4</sup>Rajiv Gandhi Cancer Institute and Research Centre, Delhi, India </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 26; Accepted 2025 Jun 3; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373945  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40847033/" class="usa-link">40847033</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Cervical cancer is a prevalent malignancy worldwide and represents a significant health burden for women. Toll-like receptors (TLRs) are crucial components of the innate immune system and play a vital role in recognizing pathogen-associated molecular patterns. Among the TLRs, TLR3 has been implicated in the defense against viral infections, including human papillomavirus (HPV), the primary etiological agent of cervical cancer. Missense single-nucleotide polymorphisms (SNPs) in the TLR3 gene can alter its protein structure and potentially influence its function, leading to variations in immune responses and disease susceptibility. This study aimed to investigate the impact of TLR3 missense SNPs on cervical cancer development through in silico methods. A comprehensive analysis was performed using various computational tools and databases to predict the functional consequences of identified TLR3 missense SNPs. The selection criteria for SNPs included their reported association with cervical cancer or their potential effects on TLR3 structure and function. Three extremely detrimental TLR-3 nsSNPs, namely N284I (rs5743316), C37R(rs752889035), and L360P(rs768091235), have been found among the 150 nsSNPs that have so far been reported in the dbSNP database. The in silico analysis suggests that these genetic variations may contribute to the development and progression of cervical cancer by modulating TLR3 function. Further experimental studies are warranted to validate these findings and elucidate the underlying mechanisms, which may aid in developing novel therapeutic strategies for cervical cancer prevention and treatment.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-05599-5.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> TLR-3, Cervical cancer, HPV, Missense mutation, nsSNP, In silico analysis</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Biotechnology, Cancer, Computational biology and bioinformatics, Structural biology, Oncology</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Cervical cancer ranks as the fourth most prevalent malignancy affecting women worldwide, arising from the epithelial tissue of the cervix<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Cervical cancer is one of the most successfully treated cancers when detected early. Human papillomaviruses (HPV), an extremely prevalent virus spread through sexual contact, are responsible for approximately 99% of cervical cancer occurrences<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. The majority of occurrences of cervical cancer can be avoided with the HPV vaccine and secondary prevention techniques (screening for and treating precancerous lesions). Most HPV infections are eliminated by the immune system; however, persistent high-risk types can cause cervical cell changes that may lead to cancer<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. It is currently unclear how the host immune system defends against HPV infection, and what the key predictors of HPV infection outcome are.</p>
<p id="Par3">Toll-like receptors (TLRs) are pattern-recognition receptors present in the cytoplasm and cell membrane that may detect pathogen-associated molecular patterns (PAMPs). TLRs have been discovered to have a crucial role in HR-HPV-caused cervical cancer<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. TLRs, which are required for both innate and acquired immunity, play an important part in the genesis and progression of many malignant tumors, as well as the immune system’s defense against infectious illnesses<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. TLRs in <em>Homo sapiens</em> can be categorized into ten subclasses based on their main sequence, TLR1 through TLR10<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Among these TLRs, TLR1, TLR2, TLR4, TLR5, and TLR6 are present in the plasma membrane and are required for bacterial and mycobacterial population discrimination. TLR3 and TLR7, both found in endolysosomes, are essential for detecting dsRNA and ssRNA viruses, respectively<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. TLR-8 is a gene that encodes a protein that has been associated with anogenital warts and superficial basal cell carcinoma. TLR9, which is present in the endolysosome, recognizes bacteria and viruses’ cytosine phosphate-guanine oligodeoxynucleotide (CpG-ODN). TLR expression profiles differ between TLRs. The TLR3 gene produces interferon-induced transmembrane protein 1, which is required for pathogen identification and innate defense activation. When a virus is detected, TLR3 is activated, enhancing the production of type I interferons and signaling other cells to strengthen their antiviral defenses<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Genetic polymorphisms in TLR3 have been associated with altered susceptibility to HPV infection and cervical cancer<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>.</p>
<p id="Par4">Single-nucleotide polymorphisms (SNPs) are the most widespread form of genetic variation in humans<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Each SNP represents a change in a single nucleotide, the basic unit of DNA. SNPs are frequently classified into two groups based on where they are found and whether they are coding, non-coding, or intergenic. SNPs in non-coding regions may change mRNA structure, disease susceptibility, and cancer risk. Non-coding SNPs, unlike synonymous and non-synonymous substitutions in coding regions, can affect gene expression levels. By definition, synonymous replacements do not modify a protein’s amino acids but may impact its function in other ways, whereas non-synonymous substitutions are further classified as missense and nonsense variants<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. A missense variant is a type of single nucleotide polymorphism (SNP) in which a single nucleotide change in the DNA sequence leads to the substitution of one amino acid for another in the resulting protein. This alteration can affect the structure and function of the protein, potentially leading to changes in its stability, activity, or interaction with other molecules. Depending on the location and nature of the amino acid change, a missense variant may have benign, deleterious, or even disease-causing effects.</p>
<p id="Par5">SNPs in TLR genes are an important predictor of early cancer susceptibility, notably in cervical cancer. Functional non-synonymous SNPs (nsSNPs) identification requires expensive and difficult experimental approaches among the many SNPs linked to cervical cancer. As the human TLR3 gene has a huge amount of SNP data, it is critical to discover and investigate the detrimental SNPs associated with it. To prioritize the dbSNP database nsSNPs that are detrimental, the current study used a range of computational methods (dry lab work) for disease-related mutations, including PROVEAN, Mutation Assessor, PANTHER, SNAP, PhD-SNP, SNPs&amp;GO, I-Mutant, CUPSAT, and many more on TLR3 SNPs. For the first time, the structural effects produced by the mentioned mutations, its interaction analysis &amp; oncogenic nature of the TLR3 gene against Cervical Cancer investigation were performed.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<p id="Par6">The schematic representation of the in silico analysis performed in this study is shown in Fig. <a href="#Fig1" class="usa-link">1</a>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373945_41598_2025_5599_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/6625a80e8b62/41598_2025_5599_Fig1_HTML.jpg" loading="lazy" id="d33e310" height="439" width="769" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The methodology used for the analysis of the TLR3 missense SNPs.</p></figcaption></figure><section id="Sec3"><h3 class="pmc_sec_title">Data collection</h3>
<p id="Par8">TLR3’s protein sequence was obtained from UniProtKB with accession ID <a href="https://www.ncbi.nlm.nih.gov/protein/O15455" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">O15455</a><sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. The human TLR3 gene’s nsSNPs were acquired from NCBI. The protein structure was found in the protein data bank (PDB), specifically under PDB ID: 5GS0<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. The accession numbers of the SNPs and the direct access links to the software/tools used in the manuscript are provided in the Data availability Section (Table <a href="#Tab1" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>The accession number of the SNPs and direct accessible links of the software/tools mentioned in the manuscript.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">SNP IDs</th>
<th align="left" colspan="1" rowspan="1">Links</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">rs5743316</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/snp/rs5743316#frequency_tab" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">rs5743316 RefSNP Report - dbSNP - NCBI</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs199768900</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/snp/rs199768900" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">rs199768900 RefSNP Report - dbSNP - NCBI</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs200214332</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/snp/rs200214332" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">rs200214332 RefSNP Report - dbSNP - NCBI</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs201540925</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/snp/rs201540925" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">rs201540925 RefSNP Report - dbSNP - NCBI</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs748141748</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/snp/rs748141748" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">rs748141748 RefSNP Report - dbSNP - NCBI</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs752889035</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/snp/rs752889035" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">rs752889035 RefSNP Report - dbSNP - NCBI</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs760275329</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/snp/rs760275329" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">rs760275329 RefSNP Report - dbSNP - NCBI</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs768091235</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.ncbi.nlm.nih.gov/snp/rs768091235" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">rs768091235 RefSNP Report - dbSNP - NCBI</a>
</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">
<strong>Software/Tools Used</strong>
</th>
<th align="left" colspan="1" rowspan="1">
<strong>Links</strong>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Meta-SNP</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://snps.biofold.org/meta-snp/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Meta-SNP - Meta-predictor of disease causing variants</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Mutation Assessor</td>
<td align="left" colspan="1" rowspan="1">
<a href="http://mutationassessor.org/v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://mutationassessor.org/v1</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PANTHER</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://pantherdb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">pantherdb.org</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PredictSNP</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://loschmidt.chemi.muni.cz/predictsnp/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PredictSNP: Predict SNP effect!</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SNAP</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://genetools.org/Manchester/page/snap-screening-nonacceptable-polymorphisms.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SNAP (Screening for NonAcceptable Polymorphisms) | NGRL Manchester</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PhD-SNAP</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://snps.biofold.org/phd-snp/phd-snp.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PhD-SNP: Predictor of human Deleterious Single Nucleotide Polymorphisms</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SNPs&amp;GO</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://snps.biofold.org/snps-and-go/snps-and-go.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SNPs&amp;GO - Predicting disease associated variations using GO terms</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">I-Mutant</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://folding.biofold.org/i-mutant/i-mutant2.0.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Welcome to I-Mutant2.0 Home Page</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CUPSAT</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://cupsat.brenda-enzymes.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CUPSAT</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MUpro</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://mupro.proteomics.ics.uci.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Prediction of Protein Stability Changes upon Mutations</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Align-GVGD</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://agvgd.hci.utah.edu/agvgd_input.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Use - Align GVGD</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ConSurf</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://consurf.tau.ac.il/consurf_index.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ConSurf | Evolutionary conservation profiles of proteins</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetSurfP 2.0</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://services.healthtech.dtu.dk/services/NetSurfP-2.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NetSurfP 2.0 - DTU Health Tech - Bioinformatic Services</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CScape</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://cscape.biocompute.org.uk/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CScape</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CScape Somatic</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://cscape-somatic.biocompute.org.uk/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CScape Somatic</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HOPE</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://www3.cmbi.umcn.nl/hope/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www3.cmbi.umcn.nl/hope/</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">STRING</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://string-db.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">STRING: functional protein association networks</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cytoscape</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://cytoscape.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Cytoscape: An Open Source Platform for Complex Network Analysis and Visualization</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GEPIA2</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://gepia2.cancer-pku.cn/#index" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GEPIA 2</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">muTarget</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://mutarget.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">| muTarget</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AlphaFold</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://deepmind.google/science/alphafold/alphafold-server/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AlphaFold Server - Google DeepMind</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RamPlot</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.ramplot.in/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Ramachandran Map</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TLR3 (UniProtKB with accession ID <a href="https://www.ncbi.nlm.nih.gov/protein/O15455" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">O15455</a>)</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.uniprot.org/uniprotkb/O15455/entry#structure" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">TLR3-Toll-like receptor 3-Homo sapiens (Human)| UniProtKB|UniProt</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PDB ID- 5GS0</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://www.rcsb.org/structure/5GS0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">RCSB PDB- 5GS0: Crystal structure of the complex of TLR3 and bi-specific diabody</a>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">Identification of deleterious nsSNPs through sequence-based tools</h3>
<p id="Par10">Different in silico or dry lab tools that could provide deleterious nsSNPs were used. The functional effect of these could be observed through these tools, and 7 of these were used. The effect of the substitution was predicted by Mutation Assessor<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>PANTHER<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>Meta-SNP<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>PredictSNP<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>SNAP<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>PhD-SNP<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> and SNPs&amp;GO<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Detection of damaging nsSNPs through structure-based tools</h3>
<p id="Par11">I-MUTANT<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>CUPSAT (both heat and denature method)<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>MUpro<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>and Align-GVGD<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> tools were utilized to anticipate the impact of protein stability.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Conservation analysis and effect of nsSNPs on protein stability and solvent accessibility of TLR3</h3>
<p id="Par12">Prediction of the conserved region for TLR3 was done by ConSurf server<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. The ConSurf server is a bioinformatics tool for estimating the evolutionary conservation of amino/nucleic acid positions in a protein/DNA/RNA molecule based on phylogenetic relationships between homologous sequences. The evolutionarily conserved amino acid profile of the most deteriorating nsSNPs was used to gain insight into the protein’s structural and functional effect on the nsSNPs. Also, the usage of certain multiple sequences alignment tools like CLUSTAL-OMEGA and T-Coffee for TLR3 sequence of different species like <em>Homo sapiens</em>,<em> Mus musculus</em>,<em> Bos taurus</em> and <em>Boselaphus tragocamelus</em> with uniport ID <a href="https://www.ncbi.nlm.nih.gov/protein/O15455" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">O15455</a>, <a href="https://www.ncbi.nlm.nih.gov/protein/Q99MB1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q99MB1</a>, <a href="https://www.ncbi.nlm.nih.gov/protein/Q5TJ59" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q5TJ59</a> and <a href="https://www.ncbi.nlm.nih.gov/protein/Q0PV50" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q0PV50</a> was used (Figure <a href="#MOESM1" class="usa-link">S1</a> &amp; <a href="#MOESM1" class="usa-link">S2</a>, Supplementary Information).</p>
<p id="Par13">NetSurfP 2.0 program was used to determine how the most detrimental SNPs affected protein stability and solvent accessibility. Absolute surface accessibility (ASA) and relative surface accessibility (RSA) were used by the NetSurfP 2.0 web server to characterize the cumulative effects of each damaging nsSNP on the amino acids that were exposed or buried. Through the use of the NetSurfP 2.0 server, some physiological characteristics such as protein stability and solvent accessibility were taken into consideration for the potential impact of the most detrimental nsSNPs on the functional behavior of TLR3 protein<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Assessing the oncogenic nature of screened mutations and association of the damaging SNPs with cancer</h3>
<p id="Par14">For the analysis of the oncogenic nature of the screened mutations, different tools named CScape<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> and CScape-somatic<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> were used. Along with it, FATHMM-MKL<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> (Predict the Functional Consequences of Non-Coding and Coding Single Nucleotide Variants) &amp; FATHMM-XF<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> (Enhanced Accuracy in Predicting the Functional Consequences of Non-Coding and Coding Single Nucleotide Variants) were also performed to assess the oncogenic nature of screened mutations and association of the damaging SNPs with cancer.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Analysis of the structural assessment of the most deleterious nsSNPs of TLR3</h3>
<p id="Par15">The deleterious mutants which were filtered after structural analysis databases were used further for visualization and analysis by the HOPE (Have Your Protein Explained) server<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. With the help of this web-based tool, how an amino acid substitution affects the protein TLR3’s physical and chemical characteristics, such as hydrophobicity, charge size differences between the wild and mutant forms, spatial structure, and protein activities can be determined.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Protein-Protein interaction analysis &amp; network analysis of TLR3</h3>
<p id="Par16">To get an insight on the protein-protein interaction, the STRING database which gives the TLR3 affected mutation through analysis with the interaction network and associated functions which participate in it was utilized. Any change to a protein’s structure or function can deviate the way it interacts with other molecules<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>.</p>
<p id="Par17">The network was subsequently transmitted to Cytoscape (version: 3.10) for analysis. We used six methodologies from the cytoHubba plugin, including three local ranking techniques: degree, maximum neighborhood component (MNC), and maximal clique centrality (MCC), as well as three worldwide ranking algorithms: betweenness, radiality, and closeness centrality. The networks were subsequently studied using the cytoHubba plugin of Cytoscape. Nonetheless, the techniques evaluated hub proteins according to their proximity. Hub proteins were ranked according to their connectivity within the entire network, as per the global method<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Analysis of gene expression profile of TLR3 gene in association with cancer</h3>
<p id="Par18">The GEPIA2 database has been incorporated into this by performing box plot analysis of the TLR3 gene to identify the differential gene expression associated with cancer types. The box plot analysis revealed the expression of the TLR-3 gene in CESC (cervical adenocarcinoma) and UCEC (uterine corpus endometrial carcinoma)<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Determining the impact of the most detrimental nsSNPs and TLR3 gene expression on the proximate genes</h3>
<p id="Par19">The muTarget tool for examining the impact of the nsSNPs based on neighboring genes was used, and as two separate analyses “Genotype” and “Target” were present on this server, both UCEC and CESC were performed<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Three-dimensional structural assessment analysis of TLR3 protein</h3>
<p id="Par20">AlphaFold 3 server<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> is used to perform the structural assessment of the TLR3 protein. Then Ramachandran plot was plotted using the Ramplot server.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Molecular dynamics simulation and trajectory analysis</h3>
<p id="Par21">The three-dimensional structure of TLR3 (5GS0) was subjected to molecular dynamics simulations using the GROMACS software package<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. The simulation protocol employed in this study followed previously established methods as detailed in earlier publications<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Protein topology parameters were derived using the CHARMM36 force field<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. A cubic simulation box was constructed via the Gmxeditconf utility. Initially, the system underwent vacuum energy minimization using the steepest descent algorithm for 1500 steps. Solvation was achieved using the simple point-charge water model through the gmx solvate command, followed by system neutralization using gmxgenion. To eliminate steric clashes and optimize geometry, an energy minimization step was performed. This was followed by a two-phase equilibration process. The first phase involved 100 ps of NVT equilibration to stabilize the temperature at 300 K. Subsequently, 100 ps of NPT equilibration were carried out to regulate the system’s pressure and density. After successful equilibration, a 100 ns production molecular dynamics run was executed<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>.</p>
<p id="Par22">Analysis of the simulation trajectories was performed using standard GROMACS utilities. The RMSD and RMSF for both wild-type and mutant proteins were computed using the gmxrms and gmxrmsf tools, respectively. Additionally, the radius of gyration (Rg) and solvent accessible surface area<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> were evaluated using gmx gyrate and gmxsasa. Secondary structure elements were assessed using the do dssp module<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par23">To investigate potential relationships among computationally predicted effects of high-risk nsSNPs in the TLR3 gene, a correlation analysis was conducted using a panel of bioinformatics scores, including SNAP (functional impact), I-Mutant ΔΔG (protein stability), ConSurf (evolutionary conservation), and CScape/FATHMM (oncogenic potential). Spearman’s rank correlation coefficient was calculated to assess non-parametric associations between these variables<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. A correlation matrix was visualized using a heatmap to illustrate the strength and direction of correlations<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Statistical analyses were performed using Python libraries (Pandas, SciPy, Seaborn), with a significance threshold set at <em>p</em> &lt; 0.05<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>–<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>.</p></section></section><section id="Sec15"><h2 class="pmc_sec_title">Results</h2>
<p id="Par24">The results of the study and tools that were used to find deleterious nsSNPs for the TLR 3 gene were summarized in Fig. <a href="#Fig2" class="usa-link">2</a>.</p>
<figure class="fig xbox font-sm" id="Fig2"><h3 class="obj_head">Fig. 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373945_41598_2025_5599_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/b8e01070a4e6/41598_2025_5599_Fig2_HTML.jpg" loading="lazy" id="d33e914" height="428" width="771" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Systematic representation of the results of in silico study conducted.</p></figcaption></figure><p id="Par26">A total of 7843 SNPs of TLR3 gene sequence were collected from dbSNP of NCBI. Out of these, 150 SNPs were screened for selection for their deleterious role in different viral diseases. From these 150 SNPs, 71 of them were filtered to be missense variants while others were in the 3′ and 5′ UTR locations, Synonymous, non-sense, and frameshift mutations (Fig. <a href="#Fig3" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h3 class="obj_head">Fig. 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/75f4475769b3/41598_2025_5599_Fig3_HTML.jpg" loading="lazy" id="d33e929" height="617" width="740" alt="Fig. 3"></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of TLR3 SNPs in dbSNP.</p></figcaption></figure><section id="Sec16"><h3 class="pmc_sec_title">Screening of nsSNPs</h3>
<p id="Par28">In the current study, the TLR3 SNPs that play a damaging impact were selected. A total of 71 nsSNPs from dbSNP out of 150 were retrieved. To determine whether any of these SNPs were detrimental, they were all further assessed using various sequence-based (Table <a href="#Tab2" class="usa-link">2</a>) and structure-based methods (Table <a href="#Tab3" class="usa-link">3</a>).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Missense nsSNPs shown to be detrimental by sequence-based methods.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">SNP IDs</th>
<th align="left" colspan="1" rowspan="1">Mutations</th>
<th align="left" colspan="1" rowspan="1">Meta-SNP</th>
<th align="left" colspan="1" rowspan="1">Mutation Assessor</th>
<th align="left" colspan="1" rowspan="1">PANTHER</th>
<th align="left" colspan="1" rowspan="1">PredictSNP</th>
<th align="left" colspan="1" rowspan="1">SNAP</th>
<th align="left" colspan="1" rowspan="1">PhD-<br>SNP</th>
<th align="left" colspan="1" rowspan="1">SNPs&amp;GO</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">rs5743316</td>
<td align="left" colspan="1" rowspan="1">N284I</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
<td align="left" colspan="1" rowspan="1">High</td>
<td align="left" colspan="1" rowspan="1">Probably damaging</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs199768900</td>
<td align="left" colspan="1" rowspan="1">R867Q</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
<td align="left" colspan="1" rowspan="1">Medium</td>
<td align="left" colspan="1" rowspan="1">Probably damaging</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs200214332</td>
<td align="left" colspan="1" rowspan="1">R791G</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
<td align="left" colspan="1" rowspan="1">Medium</td>
<td align="left" colspan="1" rowspan="1">Probably damaging</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs201540925</td>
<td align="left" colspan="1" rowspan="1">A839D</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
<td align="left" colspan="1" rowspan="1">Medium</td>
<td align="left" colspan="1" rowspan="1">Probably damaging</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs748141748</td>
<td align="left" colspan="1" rowspan="1">A802E</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
<td align="left" colspan="1" rowspan="1">Medium</td>
<td align="left" colspan="1" rowspan="1">Probably damaging</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs752889035</td>
<td align="left" colspan="1" rowspan="1">C37R</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
<td align="left" colspan="1" rowspan="1">Medium</td>
<td align="left" colspan="1" rowspan="1">Probably damaging</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs760275329</td>
<td align="left" colspan="1" rowspan="1">Q538P</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
<td align="left" colspan="1" rowspan="1">Medium</td>
<td align="left" colspan="1" rowspan="1">Probably damaging</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs768091235</td>
<td align="left" colspan="1" rowspan="1">L360P</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
<td align="left" colspan="1" rowspan="1">High</td>
<td align="left" colspan="1" rowspan="1">Probably damaging</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Deleterious</td>
<td align="left" colspan="1" rowspan="1">Disease</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p36">
<p>*Note: PANTHER: Between 0 and 1. (If &gt; 0.5 mutation is predicted Disease).</p>
<p>PhD-SNP: Between 0 and 1.(If &gt;0.5 mutation is predicted Disease).</p>
<p>SNAP: Output normalized between 0 and 1 (If &gt;0.5 mutation is predicted Disease).</p>
<p>Meta-SNP: Between 0 and 1. (If &gt;0.5 mutation is predicted Disease).</p>
<p>SNPs&amp;GO : Disease probability (if &gt;0.5 mutation is predicted Disease).</p>
</div></div></section><section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Set of nsSNPs indicated by structure-based methods to be detrimental.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">SNP IDs</th>
<th align="left" rowspan="2" colspan="1">Mutation</th>
<th align="left" colspan="2" rowspan="1">I-Mutant</th>
<th align="left" colspan="2" rowspan="1">CUPSAT<br>(Thermal)</th>
<th align="left" colspan="2" rowspan="1">CUPSAT<br>(Denature)</th>
<th align="left" colspan="4" rowspan="1">MUpro (Increase ↑ or Decrease ↓)</th>
<th align="left" rowspan="2" colspan="1">Align-GVGD</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">A</th>
<th align="left" colspan="1" rowspan="1">B</th>
<th align="left" colspan="1" rowspan="1">A</th>
<th align="left" colspan="1" rowspan="1">B</th>
<th align="left" colspan="1" rowspan="1">A</th>
<th align="left" colspan="1" rowspan="1">B</th>
<th align="left" colspan="1" rowspan="1">SVM</th>
<th align="left" colspan="1" rowspan="1">NN</th>
<th align="left" colspan="1" rowspan="1">SV0M</th>
<th align="left" colspan="1" rowspan="1">NN</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">rs5743316</td>
<td align="left" colspan="1" rowspan="1">N284I</td>
<td align="left" colspan="1" rowspan="1">↓stability</td>
<td align="left" colspan="1" rowspan="1">↓stability</td>
<td align="left" colspan="1" rowspan="1">Stabilizing</td>
<td align="left" colspan="1" rowspan="1">Stabilizing</td>
<td align="left" colspan="1" rowspan="1">Destabilizing</td>
<td align="left" colspan="1" rowspan="1">Destabilizing</td>
<td align="left" colspan="1" rowspan="1">↑</td>
<td align="left" colspan="1" rowspan="1">↑</td>
<td align="left" colspan="1" rowspan="1">↓</td>
<td align="left" colspan="1" rowspan="1">↓</td>
<td align="left" colspan="1" rowspan="1">Most likely</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs752889035</td>
<td align="left" colspan="1" rowspan="1">C37R</td>
<td align="left" colspan="1" rowspan="1">↓stability</td>
<td align="left" colspan="1" rowspan="1">↓stability</td>
<td align="left" colspan="1" rowspan="1">Destabilizing</td>
<td align="left" colspan="1" rowspan="1">Destabilizing</td>
<td align="left" colspan="1" rowspan="1">Stabilizing</td>
<td align="left" colspan="1" rowspan="1">Stabilizing</td>
<td align="left" colspan="1" rowspan="1">↑</td>
<td align="left" colspan="1" rowspan="1">↓</td>
<td align="left" colspan="1" rowspan="1">↓</td>
<td align="left" colspan="1" rowspan="1">↓</td>
<td align="left" colspan="1" rowspan="1">Most likely</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs760275329</td>
<td align="left" colspan="1" rowspan="1">Q538P</td>
<td align="left" colspan="1" rowspan="1">↓stability</td>
<td align="left" colspan="1" rowspan="1">↓stability</td>
<td align="left" colspan="1" rowspan="1">Stabilizing</td>
<td align="left" colspan="1" rowspan="1">Destabilizing</td>
<td align="left" colspan="1" rowspan="1">Destabilizing</td>
<td align="left" colspan="1" rowspan="1">Destabilizing</td>
<td align="left" colspan="1" rowspan="1">↓</td>
<td align="left" colspan="1" rowspan="1">↓</td>
<td align="left" colspan="1" rowspan="1">↓</td>
<td align="left" colspan="1" rowspan="1">↓</td>
<td align="left" colspan="1" rowspan="1">Most likely</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rs768091235</td>
<td align="left" colspan="1" rowspan="1">L360P</td>
<td align="left" colspan="1" rowspan="1">↓stability</td>
<td align="left" colspan="1" rowspan="1">↓stability</td>
<td align="left" colspan="1" rowspan="1">Destabilizing</td>
<td align="left" colspan="1" rowspan="1">Destabilizing</td>
<td align="left" colspan="1" rowspan="1">Stabilizing</td>
<td align="left" colspan="1" rowspan="1">Stabilizing</td>
<td align="left" colspan="1" rowspan="1">↓</td>
<td align="left" colspan="1" rowspan="1">↓</td>
<td align="left" colspan="1" rowspan="1">↓</td>
<td align="left" colspan="1" rowspan="1">↓</td>
<td align="left" colspan="1" rowspan="1">Most likely</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p43">
<p>*Note: I-Mutant: Free energy change value (DDG) classified into largely unstable if less than − 0.5 kcal/mol, largely stable if more than 0.5 kcal/mol, and neutral if the value lies in between.</p>
<p>Align-GVGD: combines evolutionary conservation and biophysical properties to classify missense mutations from C0 to C65, where C65 indicates the highest likelihood of functional disruption.</p>
<p>MUpro: A score less than 0 means the mutation decreases the protein stability. The smaller the score, the more confident the prediction is. Conversely, a score more than 0 means the mutation increases the protein stability. The bigger the score, the more confident the prediction is.</p>
</div></div></section><p id="Par31">To determine the most detrimental nsSNPs from the selected SNPs, a variety of in silico tools and algorithms, including PANTHER, Mutation Assessor, SNAP, PhD-SNP, Meta-SNP, PredictSNP, and SNPs&amp;GO were employed. The outcomes of these tools demonstrated whether the phenotypic impacts of the amino acid alterations on protein activities were beneficial or detrimental depending on the score. A binary classifier called Meta-SNP uses the random forest-based technique to distinguish between SNPs that are linked to disease and those that are polymorphic and non-synonymous. The output of the four predictors discussed above was provided to Meta-SNP as an eight-element feature vector made up of two sets of four items each. The first group consisted of all the raw output scores from PANTHER, PhD-SNP, and SNAP variant predictions. If one of the input approaches did not generate a prediction, the method-defined default threshold for differentiating neutrals and non-neutrals as input to Meta-SNP was used. We were able to recover 23 SNPs out of 71 by this method, showing disease-causing mutations.</p>
<p id="Par32">The mutation assessor employs a multiple sequence alignment (MSA) partitioned to represent functional specificity to identify the functional importance of a missense variant. A functional impact score was created by adding a conservation score and a specificity score. ‘Neutral’ or ‘low’ variants are expected to have no impact on protein function, whereas ‘medium’ or ‘high’ variants are expected to lead to functional modifications. By utilizing UniProt protein sequences, the mutation assessor generated its own MSA. In order to create aligned sets of families and subfamilies, it was then divided based on the boundaries of the UniProt and Pfam domains. Out of 71 mutations, we had 3 and 26 that exhibit high and medium variations, respectively.</p>
<p id="Par33">Through the PANTHER (Protein ANalysis Through Evolutionary Relationships) Classification System, we obtained 43 mutations that were probably destructive i.e., they have a deleterious nature. The results were divided into two categories: “probably damaging” and “probably benign.” To provide a more reliable and accurate alternative to the predictions provided by individual integrated tools, the PredictSNP is a consensus classifier that combines the six best-performing prediction algorithms. PredictSNP identified 25 mutations out of 71 that were disease-causing. A total of 31 and 22 mutations, respectively were identified as being disease-causing by SNAP and PhD-SNP, two further programmes that were employed to retrieve the detrimental nature of the SNPs. SNPs&amp;GO is an exact approach for determining whether a mutation, starting with a protein sequence, is related to a disease.</p>
<p id="Par34">Further, we also filtered out the nsSNPs that were mostly deleterious through structural method using the PDB ID: 5GS0 through different bioinformatics tools like I-MUTANT, CUPSAT, MUpro and Align-GVGD. We employed these tools for getting a more accurate understanding of its structure and how the nsSNPs impact protein stability based on free energy change (DDG). I-MUTANT, CUPSAT, MUpro and Align-GVGD predicted 5, 3, 3 and 7 mutations according to their structure to be deleterious in nature. Among these we filtered out 4 mutations i.e., N284I, C37R, Q538P, L360P.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Conservation analysis of TLR3</h3>
<p id="Par35">Using ConSurf, the mutations C37R and L360P had a score of 9 which showed that it had a highly conserved region and was buried inside the core region. N284I also had a score of 9 which showed a highly conserved region but was structurally exposed. Q538P had a score of 8 which was less conserved than the other three mutations and was exposed in nature.</p>
<p id="Par36">The score is analyzed on the basis of the amino acid in the variable region or conserved region. The conserved region which is scored to be 9 is color coded to be dark pink which is the most conserved. And those with a score of 1 is color coded to be blue which is the least conserved (Fig. <a href="#Fig4" class="usa-link">4</a>). A deleterious mutation that falls under a highly conserved region is likely to be detrimental in nature.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373945_41598_2025_5599_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/89cefff7e616/41598_2025_5599_Fig4_HTML.jpg" loading="lazy" id="d33e1343" height="348" width="780" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>TLR3 chain A conservation analysis results predicted by the ConSurf server.</p></figcaption></figure><p id="Par39">NetSurfP-2.0 server has been used for predicting secondary structure, solvent, and surface accessibility. According to NetSurfP, four SNPs N284I, C37R, Q538P and L360P were buried inside the backbone of the core protein. The most detrimental nsSNPs have RSA scores ranged from 0.063 to 0.49 and ASA scores ranged from 11.55 to 105.98 (Table <a href="#Tab4" class="usa-link">4</a>) (Figure <a href="#MOESM1" class="usa-link">S3</a>, Supplementary Information).</p>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Analysis of the high-risk nsSNPs in TLR3’s evolutionary conservation profile using the NetSurfP 2.0 database.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Mutations</th>
<th align="left" colspan="1" rowspan="1">Class Assignment</th>
<th align="left" colspan="1" rowspan="1">Secondary Structure</th>
<th align="left" colspan="1" rowspan="1">RSA (%)</th>
<th align="left" colspan="1" rowspan="1">ASA (A²)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">N284I</td>
<td align="left" colspan="1" rowspan="1">Buried</td>
<td align="left" colspan="1" rowspan="1">Coil</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">C37R</td>
<td align="left" colspan="1" rowspan="1">Buried</td>
<td align="left" colspan="1" rowspan="1">Coil</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Q538P</td>
<td align="left" colspan="1" rowspan="1">Buried</td>
<td align="left" colspan="1" rowspan="1">Coil</td>
<td align="center" colspan="1" rowspan="1">11</td>
<td align="center" colspan="1" rowspan="1">20</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">L360P</td>
<td align="left" colspan="1" rowspan="1">Buried</td>
<td align="left" colspan="1" rowspan="1">Strand</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">4</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p56">
<p>RSA: relative surface accessibility [¼ ASA/Max ASA].</p>
<p>ASA: absolute surface accessibility.</p>
</div></div></section></section><section id="Sec18"><h3 class="pmc_sec_title">Oncogenic analysis of TLR3</h3>
<p id="Par40">CScape-somatic mutations isolate cancer driver mutations that arise early in tumor growth from passenger mutations that accumulate once metastasis has begun. Probability estimates, or p-scores in the range (0, 1), are used to represent predictions. Values above 0.5 are projected to be cancer drivers, while values below 0.5 are predicted to be passenger variations. Through CScape- somatic, it was seen that the mutations which had coding score as mentioned in Table <a href="#Tab5" class="usa-link">5</a> showed that the mutations N284I, Q538P, and L360P had values that indicated low-confidence passenger whereas C37R had a value that denoted a low-confidence driver. Cancer drivers occur fairly early in the development of the tumor. Meanwhile, passenger variants accumulate at later stages after a tumor starts to grow and usually correspond to low or no oncogenicity. However, the CScape data presents information about mutations in the chromosome 4, detailing their positions, reference bases, mutant bases, and associated coding scores. The mutation N284I, C37R and L360P were classified as oncogenic while, Q538P, showed no prediction for its coding score (Table <a href="#Tab5" class="usa-link">5</a>).</p>
<section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Coding score of the filtered mutations for their oncogenic property through CScape and CScape-somatic.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr><th align="left" colspan="6" rowspan="1">CScape-Somatic</th></tr>
<tr>
<th align="left" colspan="1" rowspan="1">Chromosome</th>
<th align="left" colspan="1" rowspan="1">Mutation</th>
<th align="left" colspan="1" rowspan="1">Position</th>
<th align="left" colspan="1" rowspan="1">Variant</th>
<th align="left" colspan="1" rowspan="1">Coding Score</th>
<th align="left" colspan="1" rowspan="1">Messages</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">N284I</td>
<td align="left" colspan="1" rowspan="1">187,003,691</td>
<td align="left" colspan="1" rowspan="1">A/T</td>
<td align="left" colspan="1" rowspan="1">0.120850</td>
<td align="left" colspan="1" rowspan="1">Passenger</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">C37R</td>
<td align="left" colspan="1" rowspan="1">186,997,882</td>
<td align="left" colspan="1" rowspan="1">T/C</td>
<td align="left" colspan="1" rowspan="1">0.510624</td>
<td align="left" colspan="1" rowspan="1">Driver</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">Q538P</td>
<td align="left" colspan="1" rowspan="1">187,004,453</td>
<td align="left" colspan="1" rowspan="1">A/C</td>
<td align="left" colspan="1" rowspan="1">0.116281</td>
<td align="left" colspan="1" rowspan="1">Passenger</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">L360P</td>
<td align="left" colspan="1" rowspan="1">187,003,919</td>
<td align="left" colspan="1" rowspan="1">T/C</td>
<td align="left" colspan="1" rowspan="1">0.110609</td>
<td align="left" colspan="1" rowspan="1">Passenger</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr><th align="left" colspan="6" rowspan="1">
<strong>CScape</strong>
</th></tr>
<tr>
<th align="left" colspan="1" rowspan="1">
<strong>Chromosome</strong>
</th>
<th align="left" colspan="1" rowspan="1">
<strong>Mutation</strong>
</th>
<th align="left" colspan="1" rowspan="1">
<strong>Position</strong>
</th>
<th align="left" colspan="1" rowspan="1">
<strong>Ref. Base</strong>
</th>
<th align="left" colspan="1" rowspan="1">
<strong>Mutant Base</strong>
</th>
<th align="left" colspan="1" rowspan="1">
<strong>Coding Score</strong>
</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">N284I</td>
<td align="left" colspan="1" rowspan="1">18,703,691</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">T</td>
<td align="left" colspan="1" rowspan="1">0.790177, oncogenic</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">C37R</td>
<td align="left" colspan="1" rowspan="1">186,997,882</td>
<td align="left" colspan="1" rowspan="1">T</td>
<td align="left" colspan="1" rowspan="1">C</td>
<td align="left" colspan="1" rowspan="1">0.624616, oncogenic</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">Q538P</td>
<td align="left" colspan="1" rowspan="1">18,704,453</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">C</td>
<td align="left" colspan="1" rowspan="1">No prediction found</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">L360P</td>
<td align="left" colspan="1" rowspan="1">187,003,919</td>
<td align="left" colspan="1" rowspan="1">T</td>
<td align="left" colspan="1" rowspan="1">C</td>
<td align="left" colspan="1" rowspan="1">0.790816, oncogenic</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par41">In addition to this, FATHMM-MKL and FATHMM-XF studies were performed and all four mutations- N284I, C37R, Q538P, and L360P show significant pathogenic potential and are most likely oncogenic. Both coding and non-coding regions have incredibly high p-values (all &gt; 0.98) according to FATHMM-MKL, indicating a high likelihood of functional disruption. In particular, there is a considerable chance that N284I (coding: 0.99256, non-coding: 0.99253) and L360P (coding: 0.99103, non-coding: 0.99325) will undergo oncogenic transformation via structural and regulatory changes. Strongly detrimental effects are shown by C37R (coding: 0.98728, non-coding: 0.99346), with its elevated non-coding score raising concerns as it may play a function in dysregulated gene expression linked to cancer (Table <a href="#MOESM1" class="usa-link">S1</a> &amp; <a href="#MOESM1" class="usa-link">S2</a>, Supplementary Information).</p></section><section id="Sec19"><h3 class="pmc_sec_title">Structural effects of deleterious SNPs</h3>
<p id="Par43">HOPE is designed to achieve the goal of developing a server for automatic mutant analysis that can give light on the structural implications of a mutation. The HOPE server allowed structural visualization of the three extremely detrimental nsSNPs. As well as their structural modification, amino acid characteristics, and domain, the structural information for the mutations N284I, C37R, and L360P is provided in Table <a href="#Tab6" class="usa-link">6</a> for all three mutations.</p>
<section class="tw xbox font-sm" id="Tab6"><h4 class="obj_head">Table 6.</h4>
<div class="caption p"><p>Depicts the structural impact of TLR3 mutations retrieved from the HOPE server.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">N284I</th>
<th align="left" colspan="1" rowspan="1">C37R</th>
<th align="left" colspan="1" rowspan="1">L360P</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/1ef20b2d9e47/41598_2025_5599_Figa_HTML.gif" loading="lazy" id="d33e1698" height="71" width="200" alt="graphic file with name 41598_2025_5599_Figa_HTML.gif">
</td>
<td align="left" colspan="1" rowspan="1">
<img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/0880b07aa611/41598_2025_5599_Figb_HTML.gif" loading="lazy" id="d33e1704" height="80" width="196" alt="graphic file with name 41598_2025_5599_Figb_HTML.gif">
</td>
<td align="left" colspan="1" rowspan="1">
<img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/350361db49a5/41598_2025_5599_Figc_HTML.gif" loading="lazy" id="d33e1710" height="63" width="200" alt="graphic file with name 41598_2025_5599_Figc_HTML.gif">
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Mutant and wild-type amino acids differ in size.</td>
<td align="left" colspan="1" rowspan="1">Mutants have differing charges than wild-type amino acids.</td>
<td align="left" colspan="1" rowspan="1">The mutant and wild-type amino acids differ in size.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Wild-type residue is less compact than mutant.</td>
<td align="left" rowspan="2" colspan="1">The mutant residue charges a buried residue, which may disrupt protein folding.</td>
<td align="left" rowspan="2" colspan="1">Compared to wild-type, mutant residues are compact.</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1">Mutations empty the protein’s core.</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Mutant and wild-type residues differ hydrophobically.</td>
<td align="left" colspan="1" rowspan="1">Mutant and wild-type amino acids differ in size.</td>
<td align="left" colspan="1" rowspan="1">The mutation will empty the protein’s core.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Mutations weaken core hydrogen bonding, preventing folding.</td>
<td align="left" colspan="1" rowspan="1">Wild-type residue is smaller than mutant residue.</td>
<td align="left" colspan="1" rowspan="1">The altered residue is near another binding site and in a protein-critical domain.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">The altered residue is near a binding site and protein-critical domain.</td>
<td align="left" colspan="1" rowspan="1">The protein’s core hid the wild-type residue. Due to its size, the mutant residue may not fit.</td>
<td align="left" colspan="1" rowspan="1">The mutation may prevent signal transmission between these sites.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">These domains may interact differently due to the mutation, altering signal transmission.</td>
<td align="left" colspan="1" rowspan="1">Mutant and wild-type residues differ hydrophobically.</td>
<td align="left" rowspan="2" colspan="1">The altered residue is near other domain residues and in a protein-critical area, and this interaction may be essential for protein function.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">The changed residue is near domain residues and protein-critical.</td>
<td align="left" colspan="1" rowspan="1">The mutation removes hydrophobic contacts from the protein’s core. </td>
</tr>
<tr><td align="left" colspan="3" rowspan="1">
<strong>Common Interferences:</strong>
</td></tr>
<tr><td align="left" colspan="3" rowspan="1">The mutation may impact this protein interaction, preventing protein function.</td></tr>
<tr><td align="left" colspan="3" rowspan="1">The mutant residue is near a protein domain which is essential to its activity.</td></tr>
<tr><td align="left" colspan="3" rowspan="1">The mutation may affect how these domains interact, affecting protein function.</td></tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par45">When it comes to N284I, the altered residue is situated at a region critical to the protein’s activity and in close proximity to another region thought to be involved in binding. The mutation may interfere with how these domains interact, which could impact how signals are transmitted between them (Fig. <a href="#Fig5" class="usa-link">5</a>a). The C37R mutation is present in a region of the protein that is crucial for its activity, and it is in close proximity to another region that is thought to be essential in binding. The mutation may interfere with how these domains interact, which could impact how signals are transmitted between them (Fig. <a href="#Fig5" class="usa-link">5</a>b). For L360P, the altered residue is situated at a region critical to the protein’s function and in close proximity to another region thought to be essential in binding. The mutation may interfere with how these domains interact, which could impact how signals are transmitted between them (Fig. <a href="#Fig5" class="usa-link">5</a>c).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373945_41598_2025_5599_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/d4482ec8055f/41598_2025_5599_Fig5_HTML.jpg" loading="lazy" id="d33e1660" height="201" width="792" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The visualization of wild-type (green) and altered (red) amino acid residues for all mutations using the HOPE Server (<strong>a</strong>) N284I, (<strong>b</strong>) C37R, (<strong>c</strong>) L360P.</p></figcaption></figure></section><section id="Sec20"><h3 class="pmc_sec_title">Protein-Protein interaction analysis &amp; network analysis of TLR3</h3>
<p id="Par47">STRING is a database of observed and predicted protein-protein interactions. The interactions result from computational prediction, cross-species knowledge transfer, and interactions acquired from other (primary) databases; they include both direct (physical) and indirect (functional) correlations. The STRING database was used to obtain the TLR3 interaction network. ‘TLR3’ was the input name, while ‘<em>Homo sapiens</em>’ was the organism choice and those with a confidence score of 0.9 and higher were chosen. In Fig. <a href="#Fig6" class="usa-link">6</a>, the results are displayed as nodes and edges that depict how proteins interact with one another. In Fig. <a href="#Fig7" class="usa-link">7</a>, the network analysis reveals that TRAF6, TRAF3, and MYD88 are strongly correlated with TLR3 based on MCC, MNC, closeness, radiality, and degree, whereas apart from these three, TLR3 is also correlated with RIPK1 based on betweenness.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373945_41598_2025_5599_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/60528078a23e/41598_2025_5599_Fig6_HTML.jpg" loading="lazy" id="d33e1801" height="450" width="671" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Protein-protein interaction of TLR3 gene.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373945_41598_2025_5599_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/143a6cef87e7/41598_2025_5599_Fig7_HTML.jpg" loading="lazy" id="d33e1831" height="443" width="793" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Network Analysis of genes obtained from STRING using the cytohubba plugin of Cytoscape, (<strong>a</strong>) Betweenness, (<strong>b</strong>) Closeness, (<strong>c</strong>) Degree, (<strong>d</strong>) MCC, (<strong>e</strong>) MNC, and (<strong>f</strong>) Radiality.</p></figcaption></figure></section><section id="Sec21"><h3 class="pmc_sec_title">Gene expression profile analysis of TLR3</h3>
<p id="Par50">Through performing the box plot analysis using the GEPIA2 database, we can understand the differential gene expression associated with different cancer types of the TLR3 gene. Based on box plots, GEPIA2 offers differential signature score analysis. Through the results, it was observed that the two datasets of cancer types- CESC (cervical adenocarcinoma) and UCEC (Uterine corpus endometrial carcinoma) arise due to the overexpression of the TLR3 gene (Fig. <a href="#Fig8" class="usa-link">8</a>). Generally, this means that most cancers occur due to the upregulation of the TLR3 gene.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373945_41598_2025_5599_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/d47b5290ddda/41598_2025_5599_Fig8_HTML.jpg" loading="lazy" id="d33e1859" height="189" width="669" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> TLR3 gene expression analysis; total % survival rate of patients for both <strong>(B)</strong> cervical squamous cell carcinoma (CESC) and <strong>(C)</strong> uterine corpus endometrial carcinoma (UCEC).</p></figcaption></figure><p id="Par51">We completed the survival analysis of the patients with CESC and UCEC using the GEPIA online database. Patients were divided into two groups, high-expression level groups and low-expression level groups, based on the median level of TLR3 gene expression. Here, the findings indicated that patients with CESC had a longer survival period in both overexpression (roughly 200 months) and underexpression of the TLR3 gene, whereas in UCEC, it was observed that patient survival periods were higher when overexpression (roughly 140) of the TL3 gene was present and lower when underexpression of the TLR3 gene was present.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Effect of most destructive nsSNPs and expression of TLR3 gene with their neighboring genes</h3>
<p id="Par53">The muTarget tool plots an expression plot and assists us in determining how the mutation affects gene expression. We used the muTarget tool for analyzing the correlation between gene expression and mutation based on the TCGA that correlates somatic mutations and gene expression in cancer. Correlations can be analyzed in two ways: The ‘Genotype’ run is for finding changes in gene expression that are related to a specific mutation and the ‘Target’ run is for finding mutations that alter the expression of target genes. The ‘Genotype’ hypothesis findings for uterine cancer showed that the expression of the genes CMC2, CXCL13, CXCL19, LAG3, SRD5A2, and others had changed (Fig. <a href="#Fig9" class="usa-link">9</a>).</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373945_41598_2025_5599_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/d64872628e9d/41598_2025_5599_Fig9_HTML.jpg" loading="lazy" id="d33e1877" height="412" width="635" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genotype analysis of TLR3 gene.</p></figcaption></figure><p id="Par55">The number of patients was too low (less than 10) to make any conclusions, and TLR3 was unable to change the expression of the nearby gene, hence there were no results for cervical cancer presented in the database. The TLR3 gene was used as input for both uterine and cervical cancer in the following Target run. According to our findings, the CHD6 gene exhibited higher expression than the RIMS1, MORC4, FMO5, TIAM1, and other genes in cervical cancer (Fig. <a href="#Fig10" class="usa-link">10</a>A). Alternatively, for uterine cancer, the expression can be influenced by TP53, GABPB1, ZDHHC7, PTEN, MTPAP and other genes. When compared to its counterparts, PTEN in this category exhibited the greatest change in uterine cancer gene expression (Fig. <a href="#Fig10" class="usa-link">10</a>B). Mutations in any interacting gene partners in the gene-gene network affect their neighboring genes, which may result in cancer formation, according to the muTarget study findings.</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373945_41598_2025_5599_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/56a9fd842682/41598_2025_5599_Fig10_HTML.jpg" loading="lazy" id="d33e1902" height="206" width="668" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Target analysis of TLR3 gene <strong>(A)</strong> Cervical cancer and <strong>(B)</strong> Uterine cancer.</p></figcaption></figure></section><section id="Sec23"><h3 class="pmc_sec_title">Three-dimensional structure assessment of TLR3</h3>
<p id="Par57">The Three-dimensional structure assessment of TLR3 without mutations (Fig. <a href="#Fig11" class="usa-link">11</a>) and with mutations (N284I, C37R and L360P) (Figs. <a href="#Fig12" class="usa-link">12</a> and <a href="#Fig13" class="usa-link">13</a>, &amp;<a href="#Fig14" class="usa-link">14</a>) is performed with the help of the Alphafold 3 server. The projected template modeling (pTM) score and the interface predicted template modeling (ipTM) score are both generated from a measure known as the template modeling (TM) score. This assesses the precision of the complete structure. A pTM score exceeding 0.5 indicates that the predicted fold of the complex may closely resemble the actual structure. The ipTM evaluates the precision of the anticipated relative placements of the subunits inside the complex. Values beyond 0.8 indicate robust, high-quality predictions, whilst values below 0.6 imply a probable failure in prediction. ipTM values ranging from 0.6 to 0.8 represent an ambiguous zone in which forecasts may be accurate or inaccurate. The 2D &amp; 3D Ramachandran plots were plotted for both structures without (Fig. <a href="#Fig11" class="usa-link">11</a>b and c) and with mutations (Figs. <a href="#Fig12" class="usa-link">12</a>b and c, <a href="#Fig13" class="usa-link">13</a>b and c and <a href="#Fig14" class="usa-link">14</a>b and c). In Fig. <a href="#Fig11" class="usa-link">11</a>a, a structural assessment of TLR3 without mutations gave a pTM score of 0.92, while Figs. <a href="#Fig12" class="usa-link">12</a>a, <a href="#Fig13" class="usa-link">13</a>a and <a href="#Fig14" class="usa-link">14</a>a respectively showed structural assessment of TLR3 with mutation N284I gave a pTM score of 0.91, TLR-3 with mutation C37R gave a pTM score of 0.92, and TLR3 with mutation L360P gave a pTM score of 0.92, all scores indicating robust and high-quality structural predictions. (Figure <a href="#MOESM1" class="usa-link">S4</a>, <a href="#MOESM1" class="usa-link">S5</a>, <a href="#MOESM1" class="usa-link">S6</a> &amp; <a href="#MOESM1" class="usa-link">S7</a> Supplementary Information).</p>
<figure class="fig xbox font-sm" id="Fig11"><h4 class="obj_head">Fig. 11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373945_41598_2025_5599_Fig11_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/f08bdf49b5c3/41598_2025_5599_Fig11_HTML.jpg" loading="lazy" id="d33e1975" height="296" width="780" alt="Fig. 11"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(a)</strong> Structural assessment of TLR3 without mutations, <strong>(b)</strong> 2-D Ramachandran plot analysis (green, blue, and red (dots/triangles) represent torsion angles of favored, allowed, and disallowed regions respectively; dot represents residues other than glycine and triangles represents glycine), and <strong>(c)</strong> 3-D Ramachandran Plot (bar represents the frequency of torsion angles).</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig12"><h4 class="obj_head">Fig. 12.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373945_41598_2025_5599_Fig12_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/25073894d779/41598_2025_5599_Fig12_HTML.jpg" loading="lazy" id="d33e1994" height="298" width="780" alt="Fig. 12"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(a)</strong> Structural assessment of TLR3 with N284I mutation, <strong>(b)</strong> 2-D Ramachandran plot analysis (green, blue, and red (dots/triangles) represent torsion angles of favored, allowed and disallowed regions respectively; dot represents residues other than glycine and triangles represents glycine), and <strong>(c)</strong> 3-D Ramachandran Plot (bar represents the frequency of torsion angles).</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig13"><h4 class="obj_head">Fig. 13.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373945_41598_2025_5599_Fig13_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/46a11018a86c/41598_2025_5599_Fig13_HTML.jpg" loading="lazy" id="d33e2013" height="296" width="780" alt="Fig. 13"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(a)</strong> Structural assessment of TLR3 with C37R mutation, <strong>(b)</strong> 2-D Ramachandran plot analysis (green, blue, and red (dots/triangles) represent torsion angles of favored, allowed and disallowed regions respectively; dot represents residues other than glycine and triangles represents glycine), and <strong>(c)</strong> 3-D Ramachandran Plot (bar represents the frequency of torsion angles).</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig14"><h4 class="obj_head">Fig. 14.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373945_41598_2025_5599_Fig14_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/9b679018a37d/41598_2025_5599_Fig14_HTML.jpg" loading="lazy" id="d33e2032" height="305" width="780" alt="Fig. 14"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig14/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(a</strong>) Structural assessment of TLR3 with L360P mutation, <strong>(b)</strong> 2-D Ramachandran plot analysis (green, blue, and red (dots/triangles) represent torsion angles of favored, allowed and disallowed regions respectively; dot represents residues other than glycine and triangles represents glycine), and <strong>(c</strong>) 3-D Ramachandran Plot (bar represents the frequency of torsion angles).</p></figcaption></figure></section><section id="Sec24"><h3 class="pmc_sec_title">Molecular dynamics simulation and trajectory analysis</h3>
<p id="Par62">The Root Mean Square Deviation (RMSD) plot provides an overview of the structural stability of the protein throughout the simulation period. The wild-type 5GS0 (blue) shows bigger and more stable variable RMSD values than the mutants in the RMSD vs. Time plot (Fig. <a href="#Fig15" class="usa-link">15</a>a), suggesting greater structural deviation and more extensive stability during the simulation. In contrast, the N284I mutant (orange) exhibits the lowest and most reliable RMSD, indicating a noticeably more stable conformation. The C37R (green) and L360P (red) mutants are in the between, with moderate RMSD variations. According to this pattern, the wild-type structure is more vulnerable to dynamic shifts under the simulated conditions, whereas the N284I mutation may give increased conformational stiffness or structural integrity.</p>
<figure class="fig xbox font-sm" id="Fig15"><h4 class="obj_head">Fig. 15.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373945_41598_2025_5599_Fig15_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ca/12373945/ce95cea7b92e/41598_2025_5599_Fig15_HTML.jpg" loading="lazy" id="d33e2077" height="448" width="792" alt="Fig. 15"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig15/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Protein &amp; its mutation analysis of all four plots—(<strong>a</strong>) RMSD, (<strong>b</strong>) RMSF, (<strong>c</strong>) Radius of gyration, and (<strong>d</strong>) SASA for 100 ns or 100,000 ps [5GS0 (blue), N284I (orange), C37R (green), and L360P (red)].</p></figcaption></figure><p id="Par63">The Root Mean Square Fluctuation (RMSF) plot reveals localized flexibility within the protein structure by displaying the time-averaged fluctuation of each residue. The RMSF versus Residue plot (Fig. <a href="#Fig15" class="usa-link">15</a>b) demonstrates per-residue flexibility. The general profile of all variants is similar, with higher fluctuation in the N- and C-terminal regions-which is to be expected given that they are generally disordered. N284I and C37R show somewhat greater fluctuations in the core region (about residues 300–400), despite the fact that the RMSF trends are generally conserved across all proteins. These results suggest localised instability or conformational flexibility in areas that may be important in functional interactions.</p>
<p id="Par64">The Radius of Gyration (Fig. <a href="#Fig15" class="usa-link">15</a>c), which measures structural compactness, shows that the wild-type 5GS0 exhibits greater fluctuations and tighter packing, indicating sporadic transitions between compact and slightly relaxed states. With continually higher Rg values, N284I stands out once more, suggesting a more expansive structure overall. Compared to 5GS0, C37R and L360P similarly show higher Rg values, albeit they fluctuate more than N284I. This implies that the mutants, especially N284I, retain a stable but less compact shape, which may affect how they function biologically.</p>
<p id="Par65">The Solvent Accessible Surface Area (SASA) plot provides insights into how much of the protein’s surface is accessible to the solvent, reflecting changes in protein folding and hydrophobic core exposure. The Solvent Accessible Surface Area (SASA) plot (Fig. <a href="#Fig15" class="usa-link">15</a>d) showed 5GS0 and C37R (blue and green) comparatively greater surface exposure than L360P and N284I. It’s interesting to note that N284I exhibits lower SASA despite being compact in Rg; this could be a sign of stronger surface contacts or hidden residues that restrict solvent access. L360P exhibits a comparable pattern.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par67">To evaluate the relationships between various computational prediction scores associated with high-risk nsSNPs in the TLR3 gene, we conducted a Spearman correlation analysis. The results are visualized in the correlation heatmap (Figure <a href="#MOESM1" class="usa-link">S10</a>, Supplementary Information). A strong positive correlation was observed between the SNAP score and the CScape score (<em>r</em> = 0.96), suggesting that mutations predicted to be functionally damaging by SNAP are also likely to be predicted as oncogenic by CScape. Similarly, the I-Mutant ΔΔG values showed a strong inverse correlation with both SNAP (<em>r</em> = −0.87) and CScape scores (<em>r</em> = −0.97), indicating that SNPs predicted to decrease protein stability are also associated with higher functional and oncogenic impact. ConSurf scores, which reflect evolutionary conservation, showed weak correlations with other metrics (SNAP: <em>r</em> = 0.17, I-Mutant: <em>r</em> = −0.10, CScape: <em>r</em> = −0.10), suggesting that conservation alone may not be sufficient to predict functional or oncogenic significance in this context.</p></section></section><section id="Sec26"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par68">In this current study, we have shown the role of TLR3 associated with cervical cancer by computational approach. After running a set of databases, we found that among 71 missense mutations, 3 of them were highly detrimental in nature. These retrieved mutations namely N284I, C37R and L360P were further involved with different databases like HOPE server, STRING, GEPIA2 and muTARGET tool for gaining more understanding.</p>
<p id="Par69">Previous studies have demonstrated that TLR3 is aberrantly expressed in several cancers, including cervical cancer. For instance, Hasimu et al., (2011)<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> reported elevated TLR3 expression in cervical intraepithelial neoplasia and cervical squamous cell carcinoma tissues compared to normal cervical tissues, suggesting a role in cervical carcinogenesis. Similarly, studies have shown that TLR3 expression correlates with apoptosis, proliferation, and angiogenesis in hepatocellular carcinoma, and serves as a prognostic biomarker in renal clear cell carcinoma. These findings underscore the significance of TLR3 in tumor biology across various cancer types<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>.</p>
<p id="Par70">Firstly, after retrieving the missense SNPs from NCBI, the mutations were incorporated into certain sequence-based tools like Mutation Assessor, PANTHER, Meta-SNP, PredictSNP, SNAP, PhD-SNP and SNPs&amp;GO. These databases helped us to filter out the most deleterious mutations. To get more precise results of damaging mutations, we considered structural-based methods as well like the I-MUTANT, CUPSAT, MUpro and Align-GVGD. Through this, we sorted out 4 mutations that were highly detrimental in nature and these are N284I, C37R, Q538P and L360P. To understand the oncologic nature of these mutations, we used the databases like CScape, CScape-somatic and FATHMM, which showed that N284I, C37R and L360P were the more damaging mutations that could be interrelated to cancer. N284I has proven to be highly detrimental to different diseases other than cancer. Because of its association with impaired TLR3 signaling in vitro, it’s been said that N284I is highly related to viral infections in humans<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Previous studies have already implicated N284I in impaired TLR3 signaling and increased susceptibility to viral infections<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>suggesting its pathogenic role beyond cancer.</p>
<p id="Par71">In addition to this, C37R of TLR3 mutations has been predicted for the first time through computational analysis (dry lab) to be highly deleterious and potentially associated with cervical cancer. The mutation C37R, although not previously reported in a cancer context to our knowledge, was shown in our study to significantly impact protein structure and stability, and is proposed here for the first time as a novel cancer-related TLR3 mutation. However, further experimental validation is needed to confirm its impact on cancer and other diseases.</p>
<p id="Par72">Another mutation namely L360P, has been proven to be oncogenic. To support this, previous studies has shown that L360P is an HSE-causing (herpes simplex encephalitis) TLR3 mutation among the others that are P554S, E746X, G743D + R811I, and R867Q<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. It has also been reported that heterozygosity for the HSE-causing L360P or G743D + R811I allele causes AD TLR3 deficiency in fibroblasts due to negative dominance and haploinsufficiency, respectively<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Therefore, our study expands on these findings by suggesting that these N284I, C37R and L360P may also contribute to oncogenic processes in cervical cancer, thereby connecting TLR3 immune dysfunction to tumor biology.</p>
<p id="Par73">To explore the functional implications of the identified high-impact nsSNPs (N284I, C37R, Q538P, L360P), we extended our analysis to include protein interaction networks, gene expression profiles, and mutation-expression correlations. STRING-based network analysis revealed strong interactions between TLR3 and key immune signaling proteins such as TRAF3, TRAF6, and MYD88, suggesting that these mutations could affect downstream immune pathways consistent with earlier findings that describe their involvement in TLR3-mediated NF-κB signaling<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>.</p>
<p id="Par74">Gene expression profiling using GEPIA2 revealed that TLR3 expression is upregulated in both cervical squamous cell carcinoma (CESC) and uterine corpus endometrial carcinoma (UCEC). Interestingly, survival analysis showed improved outcomes in UCEC patients with high TLR3 expression, whereas results for CESC were inconclusive, reflecting the context-dependent dual role of TLRs in tumor promotion and suppression. Furthermore, the muTarget analysis provided insight into mutation-associated expression shifts in neighboring genes. CHD6 showed the highest expression in cervical cancer, while TP53 and PTEN exhibited the greatest expression changes in uterine cancer. These findings point to potential network-level interactions, in which TLR3 mutations may influence the regulation of cancer-critical genes indirectly, in line with systems biology studies.</p>
<p id="Par75">To support the structural analysis, AlphaFold was used to generate separate structural models for each point mutation-C37R, N248I, and L360P-rather than combining all mutations into a single structure. This approach was adopted to better understand the individual structural consequences of each mutation without the confounding effects of multiple simultaneous substitutions. By modeling each mutant independently, we were able to observe localized conformational changes specific to each variant, allowing for a clearer interpretation of their potential impact on protein stability and function. This refinement improves the accuracy of structural comparison with the wild-type (PDB ID: 5GS0) and enhances the clarity of our structural analysis. Additionally, Sankari et al., (2024)<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> and Kamal et al., (2025)<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> also utilized Alphafold to generate accurate 3D models, enabling detailed visualization and functional analysis of mutant proteins. Our findings are in line with the study by Jumper et al., (2021)<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>which validates AlphaFold’s accuracy for modeling high-resolution protein structures and guiding mutational impact studies.</p>
<p id="Par76">In our study, the molecular dynamics simulations revealed notable structural differences between the TLR3 wild-type (5GS0) and its mutants (N284I, C37R, and L360P), as assessed through solvent accessible surface area (SASA), radius of gyration (Rg), RMSD, and RMSF. Among these, N284I exhibited the lowest RMSD and the most stable conformation, indicating enhanced structural integrity. This stability was further supported by RMSF analysis, which showed preserved flexibility across the mutants, though some increased fluctuations were noted around residues 300–400 in N284I and C37R. Furthermore, Rg analysis demonstrated that N284I maintained a more stable but expanded structure, potentially facilitating better ligand accommodation. SASA data revealed that N284I and L360P had reduced solvent exposure, suggesting tighter internal packing, a characteristic often linked to structural stabilisation in similar systems. In recent studies, molecular dynamics (MD) simulations have been employed to explore the structural effects of point mutations in TLR3 and TLR4. Mahita et al., (2018)<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> investigated the TLR3 wild-type (WT) homodimer and TLR3 A795P homodimer to understand the impact of key mutations on the protein’s structural behavior. Their study highlighted subtle conformational changes induced by phosphorylation and point mutations in the TIR domains of TLR3, providing deeper insights into the molecular alterations that occur in these regions. Similarly, Prakasam et al., (2023)<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup> used MD simulations to examine structural changes in TLR4 induced by point mutations, further advancing our understanding of how such mutations influence TLR proteins and their functions.</p>
<p id="Par77">In addition to this, correlation analysis further reinforced the findings. A strong positive correlation between SNAP and CScape scores (<em>r</em> = 0.96) indicated that mutations predicted to be functionally damaging are also likely to be oncogenic. Additionally, I-Mutant ΔΔG scores were inversely correlated with both SNAP (<em>r</em> = −0.87) and CScape (<em>r</em> = −0.97) values, highlighting the role of structural destabilization in disease progression. In contrast, ConSurf scores had weak correlations with other metrics, suggesting that while conservation indicates evolutionary importance, it does not alone predict pathogenicity.</p>
<p id="Par78">However, this study concentrates on the most detrimental nsSNPs linked to cervical cancer; yet the impact of these SNPs may also be examined in relation to other serious disorders. Despite the estimations of this study being derived from several well-known tools and algorithms, comprehensive experimental research and large-scale population studies, in conjunction with clinical trials, are requisite prior to the use of the principal findings for clinical reasons. Moreover, these computational predictions are currently being validated through ongoing wet-lab experiments in our laboratory, which will provide experimental support for the in silico findings related to TLR3 SNPs.</p></section><section id="Sec27"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par79">This study has predicted that TLR3 is related to cervical cancer with the help of the missense SNPs retrieved and the different in silico techniques involved. The TLR3 gene has undergone analysis of its nsSNPs since it has been linked to a number of complex illnesses. Four extremely damaging TLR3 nsSNPs namely N284I(rs5743316), C37R(rs752889035), Q538P(rs760275329), and L360P(rs768091235) have been found out of the 150 nsSNPs that have so far been reported in the dbSNP database. Based on a variety of evaluations, N284I, C37R and L360P were identified as the three most detrimental mutations. Although in silico methods cannot completely replace physical and frequently conclusive testing processes and methodologies, the current work is regarded to be useful for future research efforts that target TLR3 to treat cervical cancer.</p></section><section id="Sec28"><h2 class="pmc_sec_title">Electronic supplementary material</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373945/bin/41598_2025_5599_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (4.1MB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors would like to thank Amity Institute of Biotechnology, Amity University, Noida, Uttar Pradesh, India, for providing infrastructure and support.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>M.A.- Conceptualization, Methodology, Formal analysis, Validation, Data curation, Investigation, Writing - Original Draft, Software. M.K.- Conceptualization, Methodology, Formal analysis, Validation, Data curation, Investigation, Writing - Original Draft, Software. S.D.- Software, Methodology, Data curation, Writing - Original Draft. A.K.- Writing - Review &amp; Editing, Investigation, Validation, Visualization. R.T.- Writing - Review &amp; Editing, Supervision, Validation, Investigation, Visualization. K.B- Conceptualization, Supervision, Data curation, Formal analysis, Investigation, Resources, Validation, Visualization, Writing - review &amp; editing, Project administration. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This research did not receive any specific grant from funding agencies in the public, commercial, or not-for-profit sectors.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Data is provided within the manuscript.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par80">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Bray, F. et al. Global cancer statistics 2022: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. <em>CA Cancer J. Clin.</em><strong>74</strong>, 229–263 (2024).
</cite> [<a href="https://doi.org/10.3322/caac.21834" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38572751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bray,%20F.%20et%20al.%20Global%20cancer%20statistics%202022:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries.%20CA%20Cancer%20J.%20Clin.74,%20229%E2%80%93263%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Stelzle, D. et al. Estimates of the global burden of cervical cancer associated with HIV. <em>Lancet Glob Health</em>. <strong>9</strong>, e161–e169 (2021).
</cite> [<a href="https://doi.org/10.1016/S2214-109X(20)30459-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7815633/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33212031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stelzle,%20D.%20et%20al.%20Estimates%20of%20the%20global%20burden%20of%20cervical%20cancer%20associated%20with%20HIV.%20Lancet%20Glob%20Health.%209,%20e161%E2%80%93e169%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Lei, J. et al. HPV vaccination and the risk of invasive cervical Cancer. <em>N. Engl. J. Med.</em><strong>383</strong>, 1340–1348 (2020).
</cite> [<a href="https://doi.org/10.1056/NEJMoa1917338" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32997908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lei,%20J.%20et%20al.%20HPV%20vaccination%20and%20the%20risk%20of%20invasive%20cervical%20Cancer.%20N.%20Engl.%20J.%20Med.383,%201340%E2%80%931348%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Andersen, J. M., Al-Khairy, D. &amp; Ingalls, R. R. Innate immunity at the mucosal surface: role of toll-like receptor 3 and toll-like receptor 9 in cervical epithelial cell responses to microbial pathogens. <em>Biol. Reprod.</em><strong>74</strong>, 824–831 (2006).
</cite> [<a href="https://doi.org/10.1095/biolreprod.105.048629" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16421230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andersen,%20J.%20M.,%20Al-Khairy,%20D.%20&amp;%20Ingalls,%20R.%20R.%20Innate%20immunity%20at%20the%20mucosal%20surface:%20role%20of%20toll-like%20receptor%203%20and%20toll-like%20receptor%209%20in%20cervical%20epithelial%20cell%20responses%20to%20microbial%20pathogens.%20Biol.%20Reprod.74,%20824%E2%80%93831%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Mukherjee, S. et al. Toll-like receptor-guided therapeutic intervention of human cancers: molecular and immunological perspectives. <em>Frontiers in Immunology</em> vol. 14 Preprint at (2023). 10.3389/fimmu.2023.1244345</cite> [<a href="https://doi.org/10.3389/fimmu.2023.1244345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10562563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37822929/" class="usa-link">PubMed</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Behzadi, P. et al. The dual role of toll-like receptors in COVID-19: Balancing protective immunity and immunopathogenesis. <em>International Journal of Biological Macromolecules</em> vol. 284 Preprint at (2025). 10.1016/j.ijbiomac.2024.137836</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2024.137836" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39613064/" class="usa-link">PubMed</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Jin, M. S. &amp; Lee, J. O. Structures of the Toll-like Receptor Family and Its Ligand Complexes. <em>Immunity</em> vol. 29 182–191 Preprint at (2008). 10.1016/j.immuni.2008.07.007</cite> [<a href="https://doi.org/10.1016/j.immuni.2008.07.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18701082/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Ernst, P. B., Takaishi, H. &amp; Crowe, S. E. <em>Helicobacter Pylori Infection as a Model for Gastrointestinal Immunity and Chronic Inflammatory Diseases</em>. <em>Dig Dis</em> vol. 19 (2001). <a href="http://www.karger.comwww.karger.com/journals/ddi" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.karger.comwww.karger.com/journals/ddi</a></cite> [<a href="https://doi.org/10.1159/000050663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11549818/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Yu, L. &amp; Chen, S. Toll-like receptors expressed in tumor cells: Targets for therapy. <em>Cancer Immunology, Immunotherapy</em> vol. 57 1271–1278 Preprint at (2008). 10.1007/s00262-008-0459-8</cite> [<a href="https://doi.org/10.1007/s00262-008-0459-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11031105/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18256828/" class="usa-link">PubMed</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Agarwal, M., Kumar, M., Pathak, R., Bala, K. &amp; Kumar, A. Chapter Six - Exploring TLR signaling pathways as promising targets in cervical cancer: the road less traveled. in Targeting Signaling Pathways in Solid Tumors - Part A (eds Mukherjee, S. &amp; Chatterjee, K.) vol. 385 227–261 (Academic, (2024).</cite> [<a href="https://doi.org/10.1016/bs.ircmb.2023.11.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38663961/" class="usa-link">PubMed</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Behzadi, P. et al. The Interleukin-1 (IL-1) Superfamily Cytokines and Their Single Nucleotide Polymorphisms (SNPs). <em>Journal of Immunology Research</em> vol. 2022 Preprint at (2022). 10.1155/2022/2054431</cite> [<a href="https://doi.org/10.1155/2022/2054431" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8976653/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35378905/" class="usa-link">PubMed</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Aitken, N., Smith, S., Schwarz, C. &amp; Morin, P. A. Single nucleotide polymorphism (SNP) discovery in mammals: A targeted-gene approach. <em>Mol. Ecol.</em><strong>13</strong>, 1423–1431 (2004).
</cite> [<a href="https://doi.org/10.1111/j.1365-294X.2004.02159.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15140087/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aitken,%20N.,%20Smith,%20S.,%20Schwarz,%20C.%20&amp;%20Morin,%20P.%20A.%20Single%20nucleotide%20polymorphism%20(SNP)%20discovery%20in%20mammals:%20A%20targeted-gene%20approach.%20Mol.%20Ecol.13,%201423%E2%80%931431%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Lander, S. et al. Initial Sequencing and Analysis of the Human Genome International Human Genome Sequencing Consortium* The Sanger Centre: Beijing Genomics Institute/Human Genome Center. <em>NATURE</em> vol. 409 www.nature.com (2001).</cite> [<a href="https://doi.org/10.1038/35057062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11237011/" class="usa-link">PubMed</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Bateman, A. et al. UniProt: the universal protein knowledgebase in 2021. <em>Nucleic Acids Res.</em><strong>49</strong>, D480–D489 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkaa1100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7778908/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33237286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bateman,%20A.%20et%20al.%20UniProt:%20the%20universal%20protein%20knowledgebase%20in%202021.%20Nucleic%20Acids%20Res.49,%20D480%E2%80%93D489%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Yoshida, T. et al. A covalent small molecule inhibitor of glutamate-oxaloacetate transaminase 1 impairs pancreatic cancer growth. <em>Biochem. Biophys. Res. Commun.</em><strong>522</strong>, 633–638 (2020).
</cite> [<a href="https://doi.org/10.1016/j.bbrc.2019.11.130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6981064/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31787239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshida,%20T.%20et%20al.%20A%20covalent%20small%20molecule%20inhibitor%20of%20glutamate-oxaloacetate%20transaminase%201%20impairs%20pancreatic%20cancer%20growth.%20Biochem.%20Biophys.%20Res.%20Commun.522,%20633%E2%80%93638%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Reva, B., Antipin, Y. &amp; Sander, C. Predicting the functional impact of protein mutations: application to cancer genomics. <em>Nucleic Acids Res.</em>, <strong>39</strong>(17), e118 (2011).</cite> [<a href="https://doi.org/10.1093/nar/gkr407" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3177186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21727090/" class="usa-link">PubMed</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Thomas, P. D. et al. A library of protein families and subfamilies indexed by function. <em>Genome Res.</em><strong>13</strong>, 2129–2141 (2003).
</cite> [<a href="https://doi.org/10.1101/gr.772403" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC403709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12952881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thomas,%20P.%20D.%20et%20al.%20A%20library%20of%20protein%20families%20and%20subfamilies%20indexed%20by%20function.%20Genome%20Res.13,%202129%E2%80%932141%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Capriotti, E., Altman, R. B. &amp; Bromberg, Y. Collective judgment predicts disease-associated single nucleotide variants. <em>BMC Genomics</em><strong>14</strong> Suppl 3(Suppl 3):S2 (2013).</cite> [<a href="https://doi.org/10.1186/1471-2164-14-S3-S2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3839641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23819846/" class="usa-link">PubMed</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Bendl, J. et al. PredictSNP: robust and accurate consensus classifier for prediction of Disease-Related mutations. <em>PLoS Comput. Biol</em><strong>10</strong>, e1003440 (2014).</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1003440" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3894168/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24453961/" class="usa-link">PubMed</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Bromberg, Y. &amp; Rost, B. S. N. A. P. Predict effect of non-synonymous polymorphisms on function. <em>Nucleic Acids Res.</em><strong>35</strong>, 3823–3835 (2007).
</cite> [<a href="https://doi.org/10.1093/nar/gkm238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1920242/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17526529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bromberg,%20Y.%20&amp;%20Rost,%20B.%20S.%20N.%20A.%20P.%20Predict%20effect%20of%20non-synonymous%20polymorphisms%20on%20function.%20Nucleic%20Acids%20Res.35,%203823%E2%80%933835%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Capriotti, E., Calabrese, R. &amp; Casadio, R. Predicting the insurgence of human genetic diseases associated to single point protein mutations with support vector machines and evolutionary information. <em>Bioinformatics</em><strong>22</strong>, 2729–2734 (2006).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btl423" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16895930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Capriotti,%20E.,%20Calabrese,%20R.%20&amp;%20Casadio,%20R.%20Predicting%20the%20insurgence%20of%20human%20genetic%20diseases%20associated%20to%20single%20point%20protein%20mutations%20with%20support%20vector%20machines%20and%20evolutionary%20information.%20Bioinformatics22,%202729%E2%80%932734%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Capriotti, E. et al. WS-SNPs&amp;GO: a web server for predicting the deleterious effect of human protein variants using functional annotation. <em>BMC Genomics</em><strong>14</strong> Suppl 3, (2013).</cite> [<a href="https://doi.org/10.1186/1471-2164-14-S3-S6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3665478/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23819482/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Capriotti, E., Fariselli, P. &amp; Casadio, R. I-Mutant2.0: predicting stability changes upon mutation from the protein sequence or structure. <em>Nucleic Acids Res</em><strong>33</strong>, (2005).</cite> [<a href="https://doi.org/10.1093/nar/gki375" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1160136/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15980478/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Parthiban, V., Gromiha, M. M. &amp; Schomburg, D. C. U. P. S. A. T. Prediction of protein stability upon point mutations. <em>Nucleic Acids Res</em><strong>34</strong>, (2006).</cite> [<a href="https://doi.org/10.1093/nar/gkl190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1538884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16845001/" class="usa-link">PubMed</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Cheng, J., Randall, A. &amp; Baldi, P. Prediction of protein stability changes for single-site mutations using support vector machines. <em>Proteins: Struct. Function Genet.</em><strong>62</strong>, 1125–1132 (2006).</cite> [<a href="https://doi.org/10.1002/prot.20810" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16372356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20J.,%20Randall,%20A.%20&amp;%20Baldi,%20P.%20Prediction%20of%20protein%20stability%20changes%20for%20single-site%20mutations%20using%20support%20vector%20machines.%20Proteins:%20Struct.%20Function%20Genet.62,%201125%E2%80%931132%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Tavtigian, S. V. et al. Comprehensive statistical study of 452 BRCA1 missense substitutions with classification of eight recurrent substitutions as neutral. <em>J. Med. Genet.</em><strong>43</strong>, 295–305 (2006).
</cite> [<a href="https://doi.org/10.1136/jmg.2005.033878" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2563222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16014699/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tavtigian,%20S.%20V.%20et%20al.%20Comprehensive%20statistical%20study%20of%20452%20BRCA1%20missense%20substitutions%20with%20classification%20of%20eight%20recurrent%20substitutions%20as%20neutral.%20J.%20Med.%20Genet.43,%20295%E2%80%93305%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Ashkenazy, H. et al. ConSurf 2016: an improved methodology to estimate and visualize evolutionary conservation in macromolecules. <em>Nucleic Acids Res.</em><strong>44</strong>, W344–W350 (2016).
</cite> [<a href="https://doi.org/10.1093/nar/gkw408" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4987940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27166375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ashkenazy,%20H.%20et%20al.%20ConSurf%202016:%20an%20improved%20methodology%20to%20estimate%20and%20visualize%20evolutionary%20conservation%20in%20macromolecules.%20Nucleic%20Acids%20Res.44,%20W344%E2%80%93W350%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Klausen, M. S. et al. NetSurfP-2.0: improved prediction of protein structural features by integrated deep learning. <em>Proteins: Struct. Function Bioinf.</em><strong>87</strong>, 520–527 (2019).</cite> [<a href="https://doi.org/10.1002/prot.25674" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30785653/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Klausen,%20M.%20S.%20et%20al.%20NetSurfP-2.0:%20improved%20prediction%20of%20protein%20structural%20features%20by%20integrated%20deep%20learning.%20Proteins:%20Struct.%20Function%20Bioinf.87,%20520%E2%80%93527%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Rogers, M. F., Shihab, H. A., Gaunt, T. R. &amp; Campbell, C. CScape: A tool for predicting oncogenic single-point mutations in the cancer genome. <em>Sci Rep</em><strong>7</strong>, (2017).</cite> [<a href="https://doi.org/10.1038/s41598-017-11746-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5599557/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28912487/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Rogers, M. F., Gaunt, T. R. &amp; Campbell, C. CScape-somatic: distinguishing driver and passenger point mutations in the cancer genome. <em>Bioinformatics</em><strong>36</strong>, 3637–3644 (2020).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa242" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7320610/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32282885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rogers,%20M.%20F.,%20Gaunt,%20T.%20R.%20&amp;%20Campbell,%20C.%20CScape-somatic:%20distinguishing%20driver%20and%20passenger%20point%20mutations%20in%20the%20cancer%20genome.%20Bioinformatics36,%203637%E2%80%933644%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Shihab, H. A. et al. An integrative approach to predicting the functional effects of non-coding and coding sequence variation. <em>Bioinformatics</em><strong>31</strong>, 1536–1543 (2015).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4426838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25583119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shihab,%20H.%20A.%20et%20al.%20An%20integrative%20approach%20to%20predicting%20the%20functional%20effects%20of%20non-coding%20and%20coding%20sequence%20variation.%20Bioinformatics31,%201536%E2%80%931543%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Rogers, M. F. et al. FATHMM-XF: accurate prediction of pathogenic point mutations via extended features. <em>Bioinformatics</em><strong>34</strong>, 511–513 (2018).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx536" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5860356/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28968714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rogers,%20M.%20F.%20et%20al.%20FATHMM-XF:%20accurate%20prediction%20of%20pathogenic%20point%20mutations%20via%20extended%20features.%20Bioinformatics34,%20511%E2%80%93513%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Venselaar, H., te Beek, T. A. H., Kuipers, R. K. P., Hekkelman, M. L. &amp; Vriend, G. Protein structure analysis of mutations causing inheritable diseases. An e-Science approach with life scientist friendly interfaces. <em>BMC Bioinformatics</em><strong>11</strong>, 548 (2010).</cite> [<a href="https://doi.org/10.1186/1471-2105-11-548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2992548/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21059217/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Takeda, K. &amp; Akira, S. Toll-Like receptors. <em>Curr Protoc Immunol</em> 14.12.1–14.12.10 (2015).</cite> [<a href="https://doi.org/10.1002/0471142735.im1412s109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25845562/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Dong, X. et al. A simplified herbal formula improves cardiac function and reduces inflammation in mice through the TLR-Mediated NF-κB signaling pathway. <em>Front Pharmacol</em><strong>13</strong>, 865614 (2022).</cite> [<a href="https://doi.org/10.3389/fphar.2022.865614" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9207450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35734399/" class="usa-link">PubMed</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Shannon, P. et al. Cytoscape: A software environment for integrated models of biomolecular interaction networks. <em>Genome Res.</em><strong>13</strong>, 2498–2504 (2003).
</cite> [<a href="https://doi.org/10.1101/gr.1239303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC403769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shannon,%20P.%20et%20al.%20Cytoscape:%20A%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks.%20Genome%20Res.13,%202498%E2%80%932504%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Tang, Z., Kang, B., Li, C., Chen, T. &amp; Zhang, Z. GEPIA2: an enhanced web server for large-scale expression profiling and interactive analysis. <em>Nucleic Acids Res.</em><strong>47</strong>, W556–W560 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gkz430" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6602440/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31114875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang,%20Z.,%20Kang,%20B.,%20Li,%20C.,%20Chen,%20T.%20&amp;%20Zhang,%20Z.%20GEPIA2:%20an%20enhanced%20web%20server%20for%20large-scale%20expression%20profiling%20and%20interactive%20analysis.%20Nucleic%20Acids%20Res.47,%20W556%E2%80%93W560%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Menyhárt, O. et al. Guidelines for the selection of functional assays to evaluate the hallmarks of cancer. <em>Biochimica et Biophysica Acta - Reviews on Cancer</em> vol. 1866 300–319 Preprint at (2016). 10.1016/j.bbcan.2016.10.002</cite> [<a href="https://doi.org/10.1016/j.bbcan.2016.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27742530/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Abramson, J. et al. Accurate structure prediction of biomolecular interactions with alphafold 3. <em>Nature</em><strong>630</strong>, 493–500 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-024-07487-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11168924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38718835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abramson,%20J.%20et%20al.%20Accurate%20structure%20prediction%20of%20biomolecular%20interactions%20with%20alphafold%203.%20Nature630,%20493%E2%80%93500%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Abraham, M. J. et al. High performance molecular simulations through multi-level parallelism from laptops to supercomputers. <em>SoftwareX</em><strong>1–2</strong>, 19–25 (2015). Gromacs.</cite> [<a href="https://scholar.google.com/scholar_lookup?Abraham,%20M.%20J.%20et%20al.%20High%20performance%20molecular%20simulations%20through%20multi-level%20parallelism%20from%20laptops%20to%20supercomputers.%20SoftwareX1%E2%80%932,%2019%E2%80%9325%20(2015).%20Gromacs." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Scientiflow | Scientific Workflow Automation Tool. <a href="https://scientiflow.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scientiflow.com/</a></cite>
</li>
<li id="CR42">
<span class="label">42.</span><cite>Saxena, S. et al. Structural and functional analysis of disease-associated mutations in GOT1 gene: an in silico study. <em>Comput Biol. Med</em><strong>136</strong>, 104695 (2021).</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2021.104695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34352456/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Huang, J. &amp; Mackerell, A. D. CHARMM36 all-atom additive protein force field: validation based on comparison to NMR data. <em>J. Comput. Chem.</em><strong>34</strong>, 2135–2145 (2013).
</cite> [<a href="https://doi.org/10.1002/jcc.23354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3800559/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23832629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20J.%20&amp;%20Mackerell,%20A.%20D.%20CHARMM36%20all-atom%20additive%20protein%20force%20field:%20validation%20based%20on%20comparison%20to%20NMR%20data.%20J.%20Comput.%20Chem.34,%202135%E2%80%932145%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Eisenhaber, F., Lijnzaad, P., Argos, P., Sander, C. &amp; Scharf, M. <em>The double cubic lattice method: efficient approaches to numerical integration of surface area and volume and to dot surface contouring of molecular assemblies</em>. <em>Journal Comput. Chemistry</em><strong>16</strong> (1995).</cite>
</li>
<li id="CR45">
<span class="label">45.</span><cite>Kabsch, W. &amp; Sander, C. <em>Dictionary of Protein Secondary Structure: Pattern Recognition of Hydrogen-Bonded and Geometrical Features</em>.</cite> [<a href="https://doi.org/10.1002/bip.360221211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6667333/" class="usa-link">PubMed</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Spearman, C. The proof and measurement of association between two things. <em>Source: Am. J. Psychology</em><strong>100(3-4)</strong>, 441–471 (1904).</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/3322052/" class="usa-link">PubMed</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Waskom, M. Seaborn: statistical data visualization. <em>J. Open. Source Softw.</em><strong>6</strong>, 3021 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Waskom,%20M.%20Seaborn:%20statistical%20data%20visualization.%20J.%20Open.%20Source%20Softw.6,%203021%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. <em>Nat. Methods</em>. <strong>17</strong>, 261–272 (2020).
</cite> [<a href="https://doi.org/10.1038/s41592-019-0686-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7056644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32015543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Virtanen,%20P.%20et%20al.%20SciPy%201.0:%20fundamental%20algorithms%20for%20scientific%20computing%20in%20Python.%20Nat.%20Methods.%2017,%20261%E2%80%93272%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Mckinney, W. <em>Data Structures for Statistical Computing in Python</em>. (2010).</cite>
</li>
<li id="CR50">
<span class="label">50.</span><cite>Hasimu, A., Ge, L., Li, Q. Z., Zhang, R. P. &amp; Guo, X. <em>Chinese Anti鄄 Cancer A Ssociation CACA</em>. <a href="http://www.cjcsysu.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.cjcsysu.com</a></cite>
</li>
<li id="CR51">
<span class="label">51.</span><cite>Liao, G., Lv, J., Ji, A., Meng, S. &amp; Chen, C. TLR3 Serves as a Prognostic Biomarker and Associates with Immune Infiltration in the Renal Clear Cell Carcinoma Microenvironment. <em>J Oncol</em> (2021). (2021).</cite> [<a href="https://doi.org/10.1155/2021/3336770" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8440088/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34531911/" class="usa-link">PubMed</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Brown, R. A. &amp; Razonable, R. R. A real-time PCR assay for the simultaneous detection of functional N284I and L412F polymorphisms in the human toll-like receptor 3 gene. <em>J. Mol. Diagn.</em><strong>12</strong>, 493–497 (2010).
</cite> [<a href="https://doi.org/10.2353/jmoldx.2010.090122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2893634/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20413676/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown,%20R.%20A.%20&amp;%20Razonable,%20R.%20R.%20A%20real-time%20PCR%20assay%20for%20the%20simultaneous%20detection%20of%20functional%20N284I%20and%20L412F%20polymorphisms%20in%20the%20human%20toll-like%20receptor%203%20gene.%20J.%20Mol.%20Diagn.12,%20493%E2%80%93497%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Zhang, S. Y. et al. TLR3 deficiency in patients with herpes simplex encephalitis. <em>Sci. (1979)</em>. <strong>317</strong>, 1522–1527 (2007).</cite> [<a href="https://doi.org/10.1126/science.1139522" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17872438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20S.%20Y.%20et%20al.%20TLR3%20deficiency%20in%20patients%20with%20herpes%20simplex%20encephalitis.%20Sci.%20(1979).%20317,%201522%E2%80%931527%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Lim, H. K. et al. Severe influenza pneumonitis in children with inherited TLR3 deficiency. <em>J. Exp. Med.</em><strong>216</strong>, 2038–2056 (2019).
</cite> [<a href="https://doi.org/10.1084/jem.20181621" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6719423/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31217193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lim,%20H.%20K.%20et%20al.%20Severe%20influenza%20pneumonitis%20in%20children%20with%20inherited%20TLR3%20deficiency.%20J.%20Exp.%20Med.216,%202038%E2%80%932056%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Takeuchi, O. &amp; Akira, S. Pattern recognition receptors and inflammation. <em>Cell</em><strong>140</strong>, 805–820 (2010). 10.1016/j.cell.2010.01.022 Preprint at.
</cite> [<a href="https://doi.org/10.1016/j.cell.2010.01.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20303872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takeuchi,%20O.%20&amp;%20Akira,%20S.%20Pattern%20recognition%20receptors%20and%20inflammation.%20Cell140,%20805%E2%80%93820%20(2010).%2010.1016/j.cell.2010.01.022%20Preprint%20at." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Siva Sankari, G. et al. Computational analysis of sodium-dependent phosphate transporter SLC20A1/PiT1 gene identifies missense variations C573F, and T58A as high-risk deleterious snps. <em>J. Biomol. Struct. Dyn.</em><strong>42</strong>, 4072–4086 (2024).
</cite> [<a href="https://doi.org/10.1080/07391102.2023.2218939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37286379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Siva%20Sankari,%20G.%20et%20al.%20Computational%20analysis%20of%20sodium-dependent%20phosphate%20transporter%20SLC20A1/PiT1%20gene%20identifies%20missense%20variations%20C573F,%20and%20T58A%20as%20high-risk%20deleterious%20snps.%20J.%20Biomol.%20Struct.%20Dyn.42,%204072%E2%80%934086%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Kamal, E., Kaddam, L. A., Ahmed, M. &amp; Alabdulkarim, A. Integrating artificial intelligence and bioinformatics methods to identify disruptive STAT1 variants impacting protein stability and function. <em>Genes (Basel)</em><strong>16</strong>(3):303 (2025).</cite> [<a href="https://doi.org/10.3390/genes16030303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11942549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40149454/" class="usa-link">PubMed</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Jumper, J. et al. Highly accurate protein structure prediction with alphafold. <em>Nature</em><strong>596</strong>, 583–589 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jumper,%20J.%20et%20al.%20Highly%20accurate%20protein%20structure%20prediction%20with%20alphafold.%20Nature596,%20583%E2%80%93589%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Mahita, J. &amp; Sowdhamini, R. Probing subtle conformational changes induced by phosphorylation and point mutations in the TIR domains of TLR2 and TLR3. <em>Proteins</em> 86, 524–535 (2018).</cite> [<a href="https://doi.org/10.1002/prot.25471" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29383749/" class="usa-link">PubMed</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Mahita, J. &amp; Sowdhamini, R. Investigating the effect of key mutations on the conformational dynamics of toll-like receptor dimers through molecular dynamics simulations and protein structure networks. <em>Proteins</em><strong>86</strong>, 475–490 (2018).
</cite> [<a href="https://doi.org/10.1002/prot.25467" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29383743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mahita,%20J.%20&amp;%20Sowdhamini,%20R.%20Investigating%20the%20effect%20of%20key%20mutations%20on%20the%20conformational%20dynamics%20of%20toll-like%20receptor%20dimers%20through%20molecular%20dynamics%20simulations%20and%20protein%20structure%20networks.%20Proteins86,%20475%E2%80%93490%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Prakasam, P. &amp; Abdul Salam, A. A. Basheer ahamed, S. I. The pathogenic effect of snps on structure and function of human TLR4 using a computational approach. <em>J. Biomol. Struct. Dyn.</em><strong>41</strong>, 12387–12400 (2023).
</cite> [<a href="https://doi.org/10.1080/07391102.2023.2166998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36648243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prakasam,%20P.%20&amp;%20Abdul%20Salam,%20A.%20A.%20Basheer%20ahamed,%20S.%20I.%20The%20pathogenic%20effect%20of%20snps%20on%20structure%20and%20function%20of%20human%20TLR4%20using%20a%20computational%20approach.%20J.%20Biomol.%20Struct.%20Dyn.41,%2012387%E2%80%9312400%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373945/bin/41598_2025_5599_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (4.1MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data is provided within the manuscript.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-05599-5"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_5599.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373945/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373945/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373945%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373945/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373945/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373945/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40847033/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373945/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40847033/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373945/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373945/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="55ttxw2EfJ249phnlXM2b1sxSwCz0FKqXhasGizdkxOB3GH97XbPBd4Zsf416u1V">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
