
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Correcting for Genomic Inflation Leads to Loss of Power in Large‐Scale Genome‐Wide Association Study Meta‐Analysis - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15251B3F8AEEE2C3051B3F000F290A2F.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327166/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Genetic Epidemiology">
<meta name="citation_title" content="Correcting for Genomic Inflation Leads to Loss of Power in Large‐Scale Genome‐Wide Association Study Meta‐Analysis">
<meta name="citation_author" content="Archit Singh">
<meta name="citation_author_institution" content="Technical University of Munich (TUM), TUM School of Medicine and Health, Graduate School of Experimental Medicine, Munich, Germany">
<meta name="citation_author_institution" content="Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany">
<meta name="citation_author_institution" content="Munich School for Data Science (MUDS), Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany">
<meta name="citation_author" content="Lorraine Southam">
<meta name="citation_author_institution" content="Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany">
<meta name="citation_author" content="Konstantinos Hatzikotoulas">
<meta name="citation_author_institution" content="Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany">
<meta name="citation_author" content="Nigel W Rayner">
<meta name="citation_author_institution" content="Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany">
<meta name="citation_author" content="Ken Suzuki">
<meta name="citation_author_institution" content="Department of Diabetes and Metabolic Diseases, Graduate School of Medicine, University of Tokyo, Tokyo, Japan">
<meta name="citation_author_institution" content="Department of Statistical Genetics, Osaka University Graduate School of Medicine, Suita, Japan">
<meta name="citation_author" content="Henry J Taylor">
<meta name="citation_author_institution" content="Center for Precision Health Research, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, USA">
<meta name="citation_author_institution" content="British Heart Foundation Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK">
<meta name="citation_author_institution" content="Heart and Lung Research Institute, University of Cambridge, Cambridge, UK">
<meta name="citation_author" content="Xianyong Yin">
<meta name="citation_author_institution" content="Department of Epidemiology, School of Public Health, Nanjing Medical University, Nanjing, China">
<meta name="citation_author_institution" content="Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author" content="Ravi Mandla">
<meta name="citation_author_institution" content="Programs in Metabolism and Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, Massachusetts, USA">
<meta name="citation_author_institution" content="Diabetes Unit and Center for Genomic Medicine, Massachusetts General Hospital, Boston, Massachusetts, USA">
<meta name="citation_author" content="Alicia Huerta‐Chagoya">
<meta name="citation_author_institution" content="Programs in Metabolism and Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, Massachusetts, USA">
<meta name="citation_author" content="Andrew P Morris">
<meta name="citation_author_institution" content="Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany">
<meta name="citation_author_institution" content="Centre for Genetics and Genomics Versus Arthritis, Centre for Musculoskeletal Research, Division of Musculoskeletal and Dermatological Sciences, University of Manchester, Manchester, UK">
<meta name="citation_author" content="Eleftheria Zeggini">
<meta name="citation_author_institution" content="Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany">
<meta name="citation_author_institution" content="TUM School of Medicine and Health, Technical University of Munich and Klinikum Rechts der Isar, Munich, Germany">
<meta name="citation_author" content="Ozvan Bocher">
<meta name="citation_author_institution" content="Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany">
<meta name="citation_publication_date" content="2025 Aug 6">
<meta name="citation_volume" content="49">
<meta name="citation_issue" content="6">
<meta name="citation_firstpage" content="e70016">
<meta name="citation_doi" content="10.1002/gepi.70016">
<meta name="citation_pmid" content="40767503">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327166/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327166/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327166/pdf/GEPI-49-0.pdf">
<meta name="description" content="Inflation in genome‐wide association studies (GWAS) summary statistics represents a major challenge, for which correction methods have been developed. These include the genomic control (GC) method, which uses the λ‐value to correct summary ...">
<meta name="og:title" content="Correcting for Genomic Inflation Leads to Loss of Power in Large‐Scale Genome‐Wide Association Study Meta‐Analysis">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Inflation in genome‐wide association studies (GWAS) summary statistics represents a major challenge, for which correction methods have been developed. These include the genomic control (GC) method, which uses the λ‐value to correct summary ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327166/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12327166">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/gepi.70016"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/GEPI-49-0.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12327166%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12327166/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12327166/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327166/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/gepi.70016" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Genet Epidemiol</button></div>. 2025 Aug 6;49(6):e70016. doi: <a href="https://doi.org/10.1002/gepi.70016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/gepi.70016</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Genet%20Epidemiol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Genet%20Epidemiol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Genet%20Epidemiol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Genet%20Epidemiol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Correcting for Genomic Inflation Leads to Loss of Power in Large‐Scale Genome‐Wide Association Study Meta‐Analysis</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Archit Singh</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Archit Singh</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Technical University of Munich (TUM), TUM School of Medicine and Health, Graduate School of Experimental Medicine, Munich, Germany</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Munich School for Data Science (MUDS), Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Archit Singh</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Southam%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Lorraine Southam</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Lorraine Southam</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Southam%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lorraine Southam</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hatzikotoulas%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Konstantinos Hatzikotoulas</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Konstantinos Hatzikotoulas</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hatzikotoulas%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Konstantinos Hatzikotoulas</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rayner%20NW%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Nigel W Rayner</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Nigel W Rayner</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rayner%20NW%22%5BAuthor%5D" class="usa-link"><span class="name western">Nigel W Rayner</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Suzuki%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ken Suzuki</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ken Suzuki</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Diabetes and Metabolic Diseases, Graduate School of Medicine, University of Tokyo, Tokyo, Japan</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>Department of Statistical Genetics, Osaka University Graduate School of Medicine, Suita, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Suzuki%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Ken Suzuki</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Taylor%20HJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Henry J Taylor</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Henry J Taylor</span></h3>
<div class="p">
<sup>
<sup>6</sup>
</sup>Center for Precision Health Research, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, USA</div>
<div class="p">
<sup>
<sup>7</sup>
</sup>British Heart Foundation Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</div>
<div class="p">
<sup>
<sup>8</sup>
</sup>Heart and Lung Research Institute, University of Cambridge, Cambridge, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Taylor%20HJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Henry J Taylor</span></a>
</div>
</div>
<sup>6,</sup><sup>7,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yin%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Xianyong Yin</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Xianyong Yin</span></h3>
<div class="p">
<sup>
<sup>9</sup>
</sup>Department of Epidemiology, School of Public Health, Nanjing Medical University, Nanjing, China</div>
<div class="p">
<sup>
<sup>10</sup>
</sup>Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yin%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xianyong Yin</span></a>
</div>
</div>
<sup>9,</sup><sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mandla%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Ravi Mandla</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Ravi Mandla</span></h3>
<div class="p">
<sup>
<sup>11</sup>
</sup>Programs in Metabolism and Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, Massachusetts, USA</div>
<div class="p">
<sup>
<sup>12</sup>
</sup>Diabetes Unit and Center for Genomic Medicine, Massachusetts General Hospital, Boston, Massachusetts, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mandla%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ravi Mandla</span></a>
</div>
</div>
<sup>11,</sup><sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huerta%E2%80%90Chagoya%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Alicia Huerta‐Chagoya</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Alicia Huerta‐Chagoya</span></h3>
<div class="p">
<sup>
<sup>11</sup>
</sup>Programs in Metabolism and Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, Massachusetts, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huerta%E2%80%90Chagoya%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alicia Huerta‐Chagoya</span></a>
</div>
</div>
<sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Morris%20AP%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Andrew P Morris</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Andrew P Morris</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany</div>
<div class="p">
<sup>
<sup>13</sup>
</sup>Centre for Genetics and Genomics Versus Arthritis, Centre for Musculoskeletal Research, Division of Musculoskeletal and Dermatological Sciences, University of Manchester, Manchester, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Morris%20AP%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew P Morris</span></a>
</div>
</div>
<sup>2,</sup><sup>13</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeggini%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Eleftheria Zeggini</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Eleftheria Zeggini</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany</div>
<div class="p">
<sup>
<sup>14</sup>
</sup>TUM School of Medicine and Health, Technical University of Munich and Klinikum Rechts der Isar, Munich, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeggini%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Eleftheria Zeggini</span></a>
</div>
</div>
<sup>2,</sup><sup>14,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bocher%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Ozvan Bocher</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Ozvan Bocher</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bocher%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Ozvan Bocher</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="gepi70016-aff-0001">
<sup>
<sup>1</sup>
</sup>Technical University of Munich (TUM), TUM School of Medicine and Health, Graduate School of Experimental Medicine, Munich, Germany</div>
<div id="gepi70016-aff-0002">
<sup>
<sup>2</sup>
</sup>Institute of Translational Genomics, Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany</div>
<div id="gepi70016-aff-0003">
<sup>
<sup>3</sup>
</sup>Munich School for Data Science (MUDS), Helmholtz Zentrum München‐ German Research Center for Environmental Health, Neuherberg, Germany</div>
<div id="gepi70016-aff-0004">
<sup>
<sup>4</sup>
</sup>Department of Diabetes and Metabolic Diseases, Graduate School of Medicine, University of Tokyo, Tokyo, Japan</div>
<div id="gepi70016-aff-0005">
<sup>
<sup>5</sup>
</sup>Department of Statistical Genetics, Osaka University Graduate School of Medicine, Suita, Japan</div>
<div id="gepi70016-aff-0006">
<sup>
<sup>6</sup>
</sup>Center for Precision Health Research, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, USA</div>
<div id="gepi70016-aff-0007">
<sup>
<sup>7</sup>
</sup>British Heart Foundation Cardiovascular Epidemiology Unit, Department of Public Health and Primary Care, University of Cambridge, Cambridge, UK</div>
<div id="gepi70016-aff-0008">
<sup>
<sup>8</sup>
</sup>Heart and Lung Research Institute, University of Cambridge, Cambridge, UK</div>
<div id="gepi70016-aff-0009">
<sup>
<sup>9</sup>
</sup>Department of Epidemiology, School of Public Health, Nanjing Medical University, Nanjing, China</div>
<div id="gepi70016-aff-0010">
<sup>
<sup>10</sup>
</sup>Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, Michigan, USA</div>
<div id="gepi70016-aff-0011">
<sup>
<sup>11</sup>
</sup>Programs in Metabolism and Medical and Population Genetics, Broad Institute of Harvard and MIT, Cambridge, Massachusetts, USA</div>
<div id="gepi70016-aff-0012">
<sup>
<sup>12</sup>
</sup>Diabetes Unit and Center for Genomic Medicine, Massachusetts General Hospital, Boston, Massachusetts, USA</div>
<div id="gepi70016-aff-0013">
<sup>
<sup>13</sup>
</sup>Centre for Genetics and Genomics Versus Arthritis, Centre for Musculoskeletal Research, Division of Musculoskeletal and Dermatological Sciences, University of Manchester, Manchester, UK</div>
<div id="gepi70016-aff-0014">
<sup>
<sup>14</sup>
</sup>TUM School of Medicine and Health, Technical University of Munich and Klinikum Rechts der Isar, Munich, Germany</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong> Eleftheria Zeggini (<span>eleftheria.zeggini@helmholtz-munich.de</span>), 
Ozvan Bocher (<span>ozvan.bocher@helmholtz-munich.de</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jul 4; Received 2024 Jul 25; Accepted 2025 Jul 18; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Genetic Epidemiology</em> published by Wiley Periodicals LLC.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12327166  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40767503/" class="usa-link">40767503</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Inflation in genome‐wide association studies (GWAS) summary statistics represents a major challenge, for which correction methods have been developed. These include the genomic control (GC) method, which uses the λ‐value to correct summary statistics, and the linkage disequilibrium score regression (LDSR) method, which uses the LDSR intercept. By using type 2 diabetes (T2D) as an exemplar, we explore factors influencing λ‐values and the impact of these corrections on association signals. We find that larger sample sizes increase λ‐values due to increased captured polygenicity, while including lower frequency variants decreases λ‐values due to reduced power. Comparing T2D genetic associations described in overlapping GWAS meta‐analyses of increasing sample size, we find that GC correction reduces the false positive rate and leads to the loss of robust associations. In one of the largest meta‐analysis, GC correction results in 39.7% loss of independent loci, substantially reducing the number of detected associations. In comparison, the LDSR intercept correction leads to a loss of up to 25.2% of the independent loci, being therefore less conservative than the GC correction. We conclude that in large, well‐powered GWAS meta‐analysis of polygenic traits, both GC and LDSR intercept correction leads to power loss, highlighting the need for improved genomic inflation correction methods.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> genetic associations, genomic control, genomic inflation, GWAS meta‐analysis, LD‐score regression</p></section></section><section id="gepi70016-sec-0010"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>The genetic architecture of polygenic diseases is complex, and underpinned by the contribution of multiple variants of varying frequency and effect size (Boyle et al. <a href="#gepi70016-bib-0005" class="usa-link" aria-describedby="gepi70016-bib-0005">2017</a>). Genome‐wide association studies (GWAS) have been successful in identifying associations between genomic loci and complex traits (Sollis et al. <a href="#gepi70016-bib-0023" class="usa-link" aria-describedby="gepi70016-bib-0023">2022</a>). The power of these association studies has been further enhanced by meta‐analyses, which combine association summary statistics from multiple GWAS and therefore increase sample size and statistical power. When conducting a GWAS, a key consideration is to differentiate between true associations and false positives due to an inflation of the test statistics, which could arise from systematic bias. Such bias can be due to a variety of factors that, for example, contribute to population stratification, including differences in allele frequencies and linkage disequilibrium (LD) structure across populations, as well as cryptic relatedness between the participants (Voight and Pritchard <a href="#gepi70016-bib-0027" class="usa-link" aria-describedby="gepi70016-bib-0027">2005</a>; Price et al. <a href="#gepi70016-bib-0018" class="usa-link" aria-describedby="gepi70016-bib-0018">2010</a>). These biases are usually corrected by including principal components capturing population stratification as covariates, and/or by using mixed models to account for population structure and/or relatedness (Price et al. <a href="#gepi70016-bib-0018" class="usa-link" aria-describedby="gepi70016-bib-0018">2010</a>).</p>
<p>The genomic control (GC) correction method has been developed to quantify systematic inflation and correct for it (Devlin and Roeder <a href="#gepi70016-bib-0008" class="usa-link" aria-describedby="gepi70016-bib-0008">1999</a>). It relies on the genomic inflation factor, also called the <em>λ</em>‐value, which is defined as the ratio between the median of the observed <em>χ</em>
<sup>2</sup>‐association test statistic distribution to its theoretical counterpart under the null hypothesis of no association, where a value greater than 1 is indicative of an inflation in the test statistics (Devlin and Roeder <a href="#gepi70016-bib-0008" class="usa-link" aria-describedby="gepi70016-bib-0008">1999</a>). A key assumption behind this calculation is that only a limited proportion of the variants in the genome are associated with the trait of interest, with most of the genome being under the null model of no association. This assumption has been challenged in recent large genetic studies of complex diseases, where it has been observed that the number of genomic associations detected increases linearly with GWAS sample size, especially due to the detection of variants with smaller effect sizes (Canela‐Xandri et al. <a href="#gepi70016-bib-0007" class="usa-link" aria-describedby="gepi70016-bib-0007">2018</a>). This underlines the polygenicity of complex traits and the contribution of a high number of variants with small individual effects (Visscher et al. <a href="#gepi70016-bib-0026" class="usa-link" aria-describedby="gepi70016-bib-0026">2021</a>). Accordingly, <em>λ</em>‐values have increased in studies with larger sample sizes identifying larger proportions of the genome to be associated with the trait (Yang et al. <a href="#gepi70016-bib-0031" class="usa-link" aria-describedby="gepi70016-bib-0031">2011</a>). Intuitively, a larger <em>λ</em>‐value will result in a more severe GC correction and may be overconservative (Yang et al. <a href="#gepi70016-bib-0031" class="usa-link" aria-describedby="gepi70016-bib-0031">2011</a>; Wang et al. <a href="#gepi70016-bib-0028" class="usa-link" aria-describedby="gepi70016-bib-0028">2012</a>). Yet, it is still frequent practice to GC correct the summary statistics at the individual study level (single GC correction) as well as to GC correct GWAS meta‐analysis summary statistics (double GC correction) by the respective <em>λ</em>‐values. However, it is challenging to discriminate between a high <em>λ</em>‐value that is due to confounding or due to high polygenicity of the trait. To tackle this problem, the LD score regression (LDSR) correction method, based on LD information and the distribution of <em>χ</em>
<sup>2</sup>‐association test statistics, has been developed and widely adopted to measure inflation and correct summary statistics (Bulik‐Sullivan et al. <a href="#gepi70016-bib-0006" class="usa-link" aria-describedby="gepi70016-bib-0006">2015</a>). This method computes LD scores that capture the likelihood of a variant tagging a true causal variant depending upon the degree of LD with its neighboring variants. LDSR has several key assumptions including that the trait being studied is polygenic, that is, each variant contributes a small effect, the LD scores are estimated from an appropriate reference population and variants with higher LD score should have proportionally larger effect sizes on the trait. A regression of <em>χ</em>
<sup>2</sup>‐association test statistics against LD scores is performed, where an intercept value greater than 1 indicates confounding. The LDSR intercept represents the expected test statistics for variants with LD score of zero and can be used to correct the inflation in the <em>χ</em>
<sup>2</sup> test statistics similarly to how the <em>λ</em>‐value is used in GC correction. On the other hand, the slope of the regression line represents the heritability of the trait, which is not independent from the intercept. While it has been established that the λ‐value is sensitive to sample size and polygenicity, the potential loss of association signals and independent loci that could result from GC correction in such studies, and whether LDSR intercept correction could represent an effective alternative, remain questions of interest for the scientific community. Here, we investigate the effect of both correction methods using three of the largest GWAS meta‐analyses of T2D (Mahajan et al. <a href="#gepi70016-bib-0017" class="usa-link" aria-describedby="gepi70016-bib-0017">2018</a>; Mahajan et al. <a href="#gepi70016-bib-0016" class="usa-link" aria-describedby="gepi70016-bib-0016">2022</a>; Suzuki et al. <a href="#gepi70016-bib-0024" class="usa-link" aria-describedby="gepi70016-bib-0024">2024</a>). Our approach entails analysis of GWAS meta‐analyses of various sample sizes to ascertain genomic variants consistently associated with T2D across GWAS meta‐analysis and how they are impacted by inflation correction methods.</p></section><section id="gepi70016-sec-0020"><h2 class="pmc_sec_title">2. Methods</h2>
<section id="gepi70016-sec-0030"><h3 class="pmc_sec_title">2.1. Type 2 Diabetes GWAS Meta‐Analysis Summary Statistics</h3>
<p>We used three of the largest T2D GWAS meta‐analyses from the DIAMANTE Consortium and the Type 2 Diabetes Global genomics Initiative (T2DGGI), all BMI‐unadjusted, which we referred to as DIAMANTE‐18, DIAMANTE‐22, and T2DGGI‐24 (Mahajan et al. <a href="#gepi70016-bib-0017" class="usa-link" aria-describedby="gepi70016-bib-0017">2018</a>; Mahajan et al. <a href="#gepi70016-bib-0016" class="usa-link" aria-describedby="gepi70016-bib-0016">2022</a>; Suzuki et al. <a href="#gepi70016-bib-0024" class="usa-link" aria-describedby="gepi70016-bib-0024">2024</a>). We used only the European‐specific analysis from the DIAMANTE‐22 and the T2DGGI‐24 studies, as these comprise the largest subset of the multi‐ancestry dataset and enabled us to compare findings in a similar ancestry group across the three meta‐analyses (the DIAMANTE‐18 study is entirely based on European‐ancestry GWAS). Additional information about the European subset of each study is provided in Supporting Information S1: Table <a href="#gepi70016-suppl-0001" class="usa-link">S1</a>. To ensure a fair comparison in our analysis, we did not use the λ‐values reported by the respective study authors as they were based on different filtering strategies across studies, especially in DIAMANTE‐18. Instead, we chose to perform our analysis on the intersecting set of variants across the three studies, corresponding to 10,269,674 variants. The description of the three studies, including the sample size, the number of total and genome‐wide significant variants in our analysis, the method of correction (as reported in the original study), the <em>λ</em> value (computed on the shared set of variants or reported in the original study), and the number of independent loci as reported in the original study is provided in Table <a href="#gepi70016-tbl-0001" class="usa-link">1</a>. The DIAMANTE‐18 and DIAMANTE‐22 meta‐analysis summary statistics only report double GC‐corrected <em>p</em> values. We uncorrected the <em>p</em> values by calculating the <em>χ</em>²‐statistics corresponding to the double GC‐corrected <em>p</em> values and by multiplying them by the <em>λ</em>‐value reported in each study (Table <a href="#gepi70016-tbl-0001" class="usa-link">1</a>). These <em>χ</em>²‐statistics were used to determine the uncorrected <em>p</em> values. We then calculated the inflation on the shared set of variants using the GC method to determine the new <em>λ</em> values (Table <a href="#gepi70016-tbl-0001" class="usa-link">1</a>). Finally, we re‐corrected the <em>p</em> values using these new <em>λ</em>‐values to obtain the new double GC‐corrected <em>p</em> values for our analysis. The impact of genomic inflation correction was not assessed in the T2DGGI‐24 meta‐analysis as it is the latest T2D GWAS meta‐analysis. This study, which report uncorrected <em>p</em> values, was employed to ascertain high confidence variants in the DIAMANTE‐18 and the DIAMANTE‐22 meta‐analyses.</p>
<section class="tw xbox font-sm" id="gepi70016-tbl-0001"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Study characteristics of the three T2D GWAS meta‐analyses included in our study.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="left" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<thead valign="bottom"><tr valign="bottom" style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Name of study</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Variants filtering in original study</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Correction for population structure</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>λ</em>‐value (original study)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>λ</em>‐value (new)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Loci identified (original study)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Significant associations (<em>p</em> &lt; = 5 × 10<sup>−8</sup>)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>DIAMANTE‐18</strong> (74,124 cases, 824,006 controls)</td>
<td align="center" valign="top" rowspan="1" colspan="1">HRC reference panel</td>
<td align="center" valign="top" rowspan="1" colspan="1">Double GC correction</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.013</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.387</td>
<td align="center" valign="top" rowspan="1" colspan="1">231</td>
<td align="center" valign="top" rowspan="1" colspan="1">19,328</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>DIAMANTE‐22</strong> (80,154 cases, 853,816 controls)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Overlap of 1000 G and HRC, MAF &gt; 0.5% in at least one ancestry group</td>
<td align="center" valign="top" rowspan="1" colspan="1">Double GC correction</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.096</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.402</td>
<td align="center" valign="top" rowspan="1" colspan="1">277</td>
<td align="center" valign="top" rowspan="1" colspan="1">20,925</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>T2DGGI‐24</strong> (242,283 cases, 1,569,734 controls)</td>
<td align="center" valign="top" rowspan="1" colspan="1">MAF ≥ 0.5% in at least one of the five ancestry groups from 1000 G</td>
<td align="center" valign="top" rowspan="1" colspan="1">Individual study level correction using LDSR intercept</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.283</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.676</td>
<td align="center" valign="top" rowspan="1" colspan="1">611</td>
<td align="center" valign="top" rowspan="1" colspan="1">64,944</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/gepi70016-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="gepi70016-tbl1-note-0001"><p>
<em>Note:</em> Double GC correction in GWAS meta‐analysis means applying GC correction to the individual GWAS as well as to the meta‐analysis summary statistics. <em>λ</em>‐value (original study) indicates the <em>λ</em>‐value reported by the authors of the original study. <em>λ</em>‐value (new) indicates the recalculated <em>λ</em>‐value on the selected set of 10,269,674 variants included across the three meta‐analyses. The substantial difference between the original and the new λ‐values is due to the loss of low‐frequency genetic variants in our dataset compared to the original meta‐analysis.</p></div></div></section></section><section id="gepi70016-sec-0040"><h3 class="pmc_sec_title">2.2. Calculating the <em>λ</em>‐Value and the LDSR Intercept</h3>
<p>The GC method computes the λ‐value which is defined as follows:
</p>
<table class="disp-formula p" id="gepi70016-disp-0001"><tr><td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><mrow><mi mathvariant="normal">λ</mi><mtext>value</mtext><mo>=</mo><mfrac><mrow><mi mathvariant="italic">Median</mi><mo>(</mo><mi mathvariant="italic">observed</mi><msup><mrow><mspace width="1em"></mspace><mi mathvariant="normal">χ</mi></mrow><mn>2</mn></msup><mtext>test statistics</mtext><mo>)</mo></mrow><mrow><mi mathvariant="italic">Median</mi><mo>(</mo><mi mathvariant="italic">expected</mi><mspace width="1em"></mspace><msup><mi mathvariant="normal">χ</mi><mn>2</mn></msup><mtext>distribution</mtext><mo>)</mo></mrow></mfrac></mrow></mrow></mrow></math></td></tr></table>
<p>In this equation, the median of expected <em>χ</em>
<sup>2</sup>‐distribution is 0.454, corresponding to a χ<sup>2</sup>‐distribution with one degree of freedom. For case‐control GWAS, testing for association under an additive model follow a <em>χ</em>
<sup>2</sup>‐distribution with one degree of freedom. To assess the effect of LDSR intercept correction, we ran LDSR using the LDSC software (version 1.0.1) with the ‐‐h2 option and calculated the LDSR intercept (Bulik‐Sullivan et al. <a href="#gepi70016-bib-0006" class="usa-link" aria-describedby="gepi70016-bib-0006">2015</a>). Since we analyzed only the European subsets of the meta‐analyses, we employed precomputed LD scores from the 1000 Genomes European ancestry haplotypes (Auton et al. <a href="#gepi70016-bib-0002" class="usa-link" aria-describedby="gepi70016-bib-0002">2015</a>). The LDSR intercept was used to correct the uncorrected <em>p</em> values from the DIAMANTE‐18 and the DIAMANTE‐22 meta‐analysis summary statistics, following the same approach as the GC correction. The impact of correction methods was not assessed in the T2DGGI‐24 study. Further, the λ‐value and the LDSR intercept were recomputed after removing variants with a minor allele frequency (MAF) below 0.5%, 1%, 5% and 10% by considering the MAF provided in the respective summary statistics.</p></section><section id="gepi70016-sec-0050"><h3 class="pmc_sec_title">2.3. Estimating Polygenicity of Type 2 Diabetes</h3>
<p>We consider here the polygenicity as a reflection of the proportion of the genome significantly associated with T2D. It is a fixed quantity, which is increasingly captured with larger sample size of GWAS. The variance explained by polygenic factors can be captured by calculating the correlation between the χ<sup>2</sup>‐values and the LD scores of the variants. We computed this quantity using summary statistics from the meta‐analyses from the DIAMANTE‐18, DIAMANTE‐22 and T2DGGI studies within each genetic ancestry group (Mahajan et al. <a href="#gepi70016-bib-0017" class="usa-link" aria-describedby="gepi70016-bib-0017">2018</a>; Mahajan et al. <a href="#gepi70016-bib-0016" class="usa-link" aria-describedby="gepi70016-bib-0016">2022</a>; Suzuki et al. <a href="#gepi70016-bib-0024" class="usa-link" aria-describedby="gepi70016-bib-0024">2024</a>). The effective sample size of the ancestry specific meta‐analyses was calculated using the formula (Willer et al. <a href="#gepi70016-bib-0029" class="usa-link" aria-describedby="gepi70016-bib-0029">2010</a>):
</p>
<table class="disp-formula p" id="gepi70016-disp-0002"><tr><td class="formula"><math id="jats-math-2" display="block"><mrow><mrow><mrow><msub><mi>N</mi><mtext>effective</mtext></msub><mo>=</mo><mfrac><mn>4</mn><mrow><mfrac><mn>1</mn><mrow><msub><mi>N</mi><mi mathvariant="italic">cases</mi></msub></mrow></mfrac><mo>+</mo><mfrac><mn>1</mn><mrow><msub><mi>N</mi><mi mathvariant="italic">controls</mi></msub></mrow></mfrac></mrow></mfrac></mrow></mrow></mrow></math></td></tr></table></section><section id="gepi70016-sec-0060"><h3 class="pmc_sec_title">2.4. Confirmation of Associations Across Type 2 Diabetes Meta‐Analyses</h3>
<p>We evaluated the impact of the GC and LDSR intercept corrections on the T2D GWAS meta‐analyses. The impact of employing either of the two methods was studied in the smaller meta‐analyses while, the larger meta‐analyses were employed to ascertain high confidence associations. We performed pairwise analyses between T2D meta‐analyses and investigated associations which are confirmed between smaller T2D GWAS meta‐analysis and a larger one (DIAMANTE‐18 was analyzed with respect to DIAMANTE‐22 and T2DGGI‐24, and DIAMANTE‐22 was analyzed with respect to T2DGGI‐24). We refer to these high confidence associations as “robust associations” in the sense that they remain significant with a concordant direction of effect across successive GWAS meta‐analyses with increasing sample size. Note however that the term “robust associations” has a statistical rather than biological meaning as there was no validation in an experimental setting. Robust associations are defined as genetic variants significant at the genome‐wide significance threshold of 5 × 10<sup>−8</sup> in the earlier and the later study with a consistent direction of effect between the two. The percentage of robust associations between the two studies is computed as follows:
</p>
<table class="disp-formula p" id="gepi70016-disp-0003"><tr><td class="formula"><math id="jats-math-3" display="block" mathsize="8.0pt"><mrow><mrow><mrow><mo>%</mo><mi>R</mi><mi>o</mi><mi>b</mi><mi>u</mi><mi>s</mi><mi>t</mi><mo> </mo><mi>a</mi><mi>s</mi><mi>s</mi><mi>o</mi><mi>c</mi><mi>i</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi><mi>s</mi><mo>=</mo><mfrac><mrow><mi>T</mi><mi>o</mi><mi>t</mi><mi>a</mi><mi>l</mi><mo> </mo><mi>n</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mo> </mo><mi>o</mi><mi>f</mi><mo> </mo><mi>r</mi><mi>o</mi><mi>b</mi><mi>u</mi><mi>s</mi><mi>t</mi><mo> </mo><mi>a</mi><mi>s</mi><mi>s</mi><mi>o</mi><mi>c</mi><mi>i</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi><mi>s</mi><mo> </mo><mi>i</mi><mi>n</mi><mo> </mo><mi>e</mi><mi>a</mi><mi>r</mi><mi>l</mi><mi>i</mi><mi>e</mi><mi>r</mi><mo> </mo><mi>s</mi><mi>t</mi><mi>u</mi><mi>d</mi><mi>y</mi></mrow><mrow><mi>T</mi><mi>o</mi><mi>t</mi><mi>a</mi><mi>l</mi><mo> </mo><mi>n</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mo> </mo><mi>o</mi><mi>f</mi><mo> </mo><mi>s</mi><mi>i</mi><mi>g</mi><mi>n</mi><mi>i</mi><mi>f</mi><mi>i</mi><mi>c</mi><mi>a</mi><mi>n</mi><mi>t</mi><mo> </mo><mi>a</mi><mi>s</mi><mi>s</mi><mi>o</mi><mi>c</mi><mi>i</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi><mi>s</mi><mo> </mo><mi>i</mi><mi>n</mi><mo> </mo><mi>e</mi><mi>a</mi><mi>r</mi><mi>l</mi><mi>i</mi><mi>e</mi><mi>r</mi><mo> </mo><mi>s</mi><mi>t</mi><mi>u</mi><mi>d</mi><mi>y</mi></mrow></mfrac><mo>×</mo><mn>100</mn></mrow></mrow></mrow></math></td></tr></table>
<p>For each of the three analyses, we corrected only the earlier study, while the later study is left uncorrected. To assess the impact of correction in the earlier study, the percentage of robust associations is calculated for uncorrected and corrected GWAS meta‐analysis summary statistics of the earlier study.</p></section><section id="gepi70016-sec-0070"><h3 class="pmc_sec_title">2.5. Investigating Chromosomal Confirmation Rate Using a Leave‐One‐Chromosome‐Out (LOCO) and Random Sampling Approach</h3>
<p>To assess whether confirmation rates significantly vary across chromosomes, we employed a LOCO framework. Using the DIAMANTE‐18 and DIAMANTE‐22 meta‐analyses as an example, we first identified the number of genome‐wide significant variants on each chromosome in DIAMANTE‐18 and determined how many were confirmed in DIAMANTE‐22 (methods). Next, we randomly sampled an equal number of genome‐wide significant variants from the rest of the genome, excluding the chromosome of interest, and assessed the corresponding confirmation rate. We then computed an empirical <em>p</em> value, denoted as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><mrow><mrow><msub><mi>p</mi><mi mathvariant="italic">chr</mi></msub></mrow></mrow></mrow></math></span>, as following:
</p>
<table class="disp-formula p" id="gepi70016-disp-0004"><tr><td class="formula"><math id="jats-math-5" display="block"><mrow><mrow><mrow><msub><mi>p</mi><mi mathvariant="italic">chr</mi></msub><mo>=</mo><mfrac><mrow><msubsup><mo>∑</mo><mrow><mi>s</mi><mo>=</mo><mn>1</mn></mrow><mi>N</mi></msubsup><mi>min</mi><mo>⁡</mo><mrow><mo>(</mo><mrow><mi>I</mi><mrow><mo>(</mo><mrow><msub><mi>C</mi><mi>s</mi></msub><mo>≤</mo><msub><mi>C</mi><mrow><mi mathvariant="italic">obs</mi><mo>,</mo><mi mathvariant="italic">chr</mi></mrow></msub><mo>,</mo><msub><mi>C</mi><mi>s</mi></msub><mo>&gt;</mo><mi>C</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow><mi>N</mi></mfrac></mrow></mrow></mrow></math></td></tr></table>
<p>With <em>N</em> being the total number of samplings here corresponding to 100,000, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><mrow><mrow><mi>I</mi><mo>(</mo><mo>)</mo></mrow></mrow></mrow></math></span> the indicator function, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><mrow><mrow><msub><mi>C</mi><mi>s</mi></msub></mrow></mrow></mrow></math></span> the confirmation rate for the sampling <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><mrow><mrow><mi>s</mi></mrow></mrow></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><mrow><mrow><msub><mi>C</mi><mrow><mi mathvariant="italic">obs</mi><mo>,</mo><mi mathvariant="italic">chr</mi></mrow></msub></mrow></mrow></mrow></math></span> the confirmation rate observed for the investigated chromosome. This analysis was conducted for all chromosomes except chromosome 21, which lacked genome‐wide significant variants in the DIAMANTE‐18 and the DIAMANTE‐22 meta‐analyses. We repeated the same for other pairwise analyses considered in our study, i.e., DIAMANTE‐18 with respect to T2DGGI‐24, and DIAMANTE‐22 with respect to T2DGGI‐24. Further, to assess the effect of correction methods on chromosomal confirmation rate, we performed the described analysis on summary statistics after GC or LDSR intercept correction.</p></section><section id="gepi70016-sec-0080"><h3 class="pmc_sec_title">2.6. False Positive Rates and the True Positive Rates</h3>
<p>To assess the relative cost of losing robust associations in comparison to removing false associations, which is the main aim of the two correction methods, we computed the false positive rate and the true positive rate on the aforementioned pairwise analyses. False positive associations correspond to genetic variants genome‐wide significant in the earlier study but not in the later study, or genome‐wide significant in both but with inconsistent direction of effect. True positive associations correspond to variants genome‐wide significant in both studies with consistent direction of effect. False negative associations correspond to variants genome‐wide significant only in the later study and true negative associations to variants not genome‐wide significant in either study. We then calculate the false positive rate and the true positive rate as:
</p>
<table class="disp-formula p" id="gepi70016-disp-0005"><tr><td class="formula"><math id="jats-math-10" display="block"><mrow><mrow><mrow><mi mathvariant="normal">False Positive Rate</mi><mo>=</mo><mfrac><mtext>False Positive</mtext><mrow><mtext>False Positive</mtext><mo>+</mo><mtext>True Negative</mtext></mrow></mfrac></mrow></mrow></mrow></math></td></tr></table>
<table class="disp-formula p" id="gepi70016-disp-0006"><tr><td class="formula"><math id="jats-math-11" display="block"><mrow><mrow><mrow><mtext>True Positive Rate</mtext><mspace width="0.25em"></mspace><mo>=</mo><mfrac><mtext>True Positive</mtext><mrow><mtext>True Positive</mtext><mo>+</mo><mtext>False Negative</mtext></mrow></mfrac></mrow></mrow></mrow></math></td></tr></table></section><section id="gepi70016-sec-0090"><h3 class="pmc_sec_title">2.7. Assessing the Loss of Independent Loci</h3>
<p>In addition to describing the loss of robust associations from the previous analyses, we investigated if this leads to a loss of an independent locus. An independent associated locus is a genomic region comprising multiple variants in high LD, which are significantly associated with the trait of interest after accounting for other nearby genetic variants. It is described using its chromosomal location, genomic length (usually 1 Mb), and index variant (usually the most significant variant). To do so, we used the study‐defined index variants as proxy for the DIAMANTE‐18 and the DIAMANTE‐22 studies. To assess whether the lost robust associations lead to a loss of independent loci, we used the following analysis workflow (Supporting Information S1: Figure <a href="#gepi70016-suppl-0001" class="usa-link">S1</a>) for each chromosome:
</p>
<ul class="list" style="list-style-type:none">
<li>
<span class="label">1.</span><p class="display-inline">We ran LD clumping using PLINK for DIAMANTE‐18 and DIAMANTE‐22 meta‐analyses using 1000 genomes reference data for Europeans (Purcell et al. <a href="#gepi70016-bib-0020" class="usa-link" aria-describedby="gepi70016-bib-0020">2007</a>).</p>
</li>
<li>
<span class="label">2.</span><p class="display-inline">We calculated the genomic boundaries of the LD clumps as the minimum and maximum genomic position in the clump.</p>
</li>
<li>
<span class="label">3.</span><p class="display-inline">We assessed the study defined T2D index variants to determine the specific LD clump they belong to.</p>
</li>
<li>
<span class="label">4.</span><p class="display-inline">Next, we determined if the lost robust associations fall within the same LD clumps as the study defined T2D index variants.</p>
</li>
<li>
<span class="label">5.</span><p class="display-inline">Lost robust associations part of LD clumps different from those of the study‐defined index variants potentially represent lost independent loci.</p>
</li>
<li>
<span class="label">6.</span><p class="display-inline">Finally, we determined the number of unique LD clumps for these lost robust associations untagged by study defined T2D index variants.</p>
</li>
</ul>
<p>In this, meta‐analysis variants not part of the LD reference panel and falling outside the LD clumps were defined as single variant clumps. These variants were excluded when assessing the loss of independent loci. The LD clumping in PLINK was ran in windows of 500 kb with an r2 threshold of 0.20 and significance thresholds of 5 × 10<sup>−8</sup> and 5 × 10<sup>−6</sup> for the index and secondary signals, respectively.</p>
<p>Further, lost robust associations located at the major histocompatibility complex region chr6:28,477,797‐33,448,354 (GRCh37) were removed because the region is characterized by extremely high LD and high density of genes which makes it difficult to distinguish independent association signals (de Bakker and Raychaudhuri <a href="#gepi70016-bib-0003" class="usa-link" aria-describedby="gepi70016-bib-0003">2012</a>).</p>
<p>To calculate the percentage of lost independent loci, we divided the number of independent loci lost due to correction by the total number of significant independent loci before correction. The total number of independent loci before correction was determined by summing the lost independent loci and the independent loci identified in the smaller study for the respective analysis.</p></section></section><section id="gepi70016-sec-0100"><h2 class="pmc_sec_title">3. Results</h2>
<section id="gepi70016-sec-0110"><h3 class="pmc_sec_title">3.1. Factors Contributing to Increased Genomic Inflation</h3>
<p>We started by investigating factors driving the lambda values by analyzing the 10,269,674 variants included across the three meta‐analyses. We observe a linear trend between the effective sample size of the meta‐analysis and the <em>λ</em>‐values (Supporting Information S1: Figure <a href="#gepi70016-suppl-0001" class="usa-link">S2</a>), as previously reported (Yang et al. <a href="#gepi70016-bib-0030" class="usa-link" aria-describedby="gepi70016-bib-0030">2011</a>). We next investigated the impact of allele frequency cutoffs by recomputing <em>λ</em>‐values using varying MAF thresholds (methods). We observed that <em>λ</em>‐values increase as rare and low‐frequency variants are removed (Supporting Information S1: Figure <a href="#gepi70016-suppl-0001" class="usa-link">S3A</a>), likely reflecting lower statistical power (Pritchard <a href="#gepi70016-bib-0019" class="usa-link" aria-describedby="gepi70016-bib-0019">2001</a>; Li and Leal <a href="#gepi70016-bib-0013" class="usa-link" aria-describedby="gepi70016-bib-0013">2008</a>). This was corroborated by the fact that the <em>λ</em>‐value stabilizes after removing variants with a MAF lower than 5% for each of the three meta‐analyses (Supporting Information S1: Figure <a href="#gepi70016-suppl-0001" class="usa-link">S3</a>). The same was found to be true for LDSR intercept, although this increase was lower compared to the <em>λ</em>‐value (Supporting Information S1: Figure <a href="#gepi70016-suppl-0001" class="usa-link">S3B</a>). Finally, we investigated the impact of the polygenicity of the trait. Polygenicity is a fixed quantity associated with complex traits, for which we have used as a proxy the proportion of variance in the χ<sup>2</sup>‐statistics explained by polygenetic factors (methods). We find a general positive trend between this quantity and the effective sample size of the respective meta‐analyses (Figure <a href="#gepi70016-fig-0001" class="usa-link">1</a>). This highlights that polygenicity is revealed with increasing sample size and contributes to the rise in genomic inflation, concordant with results from Yang et al. (Yang et al. <a href="#gepi70016-bib-0031" class="usa-link" aria-describedby="gepi70016-bib-0031">2011</a>).</p>
<figure class="fig xbox font-sm" id="gepi70016-fig-0001"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e14c/12327166/9a074f31ef8e/GEPI-49-0-g002.jpg" loading="lazy" id="jats-graphic-1" height="608" width="608" alt="Figure 1"></p>
<div class="p text-right font-secondary"><a href="figure/gepi70016-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effective sample size against the correlation between χ<sup>2</sup>‐values and LD scores from ancestry‐specific meta‐analyses part of DIAMANTE‐18, DIAMANTE‐22 and T2DGGI‐24 meta‐analyses.</p></figcaption></figure></section><section id="gepi70016-sec-0120"><h3 class="pmc_sec_title">3.2. Correction Post Meta‐Analysis Leads to Loss of Robust Associations</h3>
<p>While previous studies have investigated factors leading to an increase in genomic inflation, no study has assessed the impact of the corresponding correction on the potential loss of genetic signals. To investigate this question, we assessed whether GC‐correcting the <em>p</em> values would lead to a loss of robust associations. For this, we conducted pairwise analyses between two successive studies (methods). We calculated the proportion of robust associations in the earlier study confirmed in the later study both before and after applying the correction method. As shown in Figure <a href="#gepi70016-fig-0002" class="usa-link">2A</a>, we observed a substantial decline in the proportion of robust associations after GC correction, with as much as 49.08% of the robust associations being lost when analyzing DIAMANTE‐22 and T2DGGI‐24 (Table <a href="#gepi70016-tbl-0002" class="usa-link">2</a>). The robust associations that were lost show <em>p</em> values close to the genome‐wide significance threshold, explaining their loss after correction (Supporting Information S1: Figure <a href="#gepi70016-suppl-0001" class="usa-link">S4</a>). Moreover, examining the DIAMANTE‐18 vs DIAMANTE‐22 analysis, we found that out of the 8359 robust associations lost due to GC correction, 8,015 were retrieved in the larger T2DGGI‐24 meta‐analysis, where they reached genome‐wide significance. This demonstrates the power of sample size in GWAS and confirms that the lost robust associations indeed represent high confidence associations. We compared these findings with a correction based on the LDSR, which integrates LD information to better estimate inflation compared to the λ‐value. For the DIAMANTE‐18 and the DIAMANTE‐22 meta‐analyses, we report lower values of the LDSR intercept compared to the <em>λ</em>‐values (Table <a href="#gepi70016-tbl-0002" class="usa-link">2</a>). By applying a similar procedure as for the GC correction, we again observed a decrease of the proportion of robust associations following correction (Table <a href="#gepi70016-tbl-0002" class="usa-link">2</a>, Figure <a href="#gepi70016-fig-0002" class="usa-link">2B</a>), but which was less severe than when using the GC correction for all analyses. We observed that robust associations that did not remain significant after correction had <em>p</em> values near the genome‐wide significance threshold (Supporting Information S1: Figure <a href="#gepi70016-suppl-0001" class="usa-link">S5</a>). Of the 2509 robust associations lost following LDSR intercept correction between the DIAMANTE‐18 and the DIAMANTE‐22 meta‐analysis, 2367 were recovered in the larger T2DGGI‐24 meta‐analysis. Altogether, we show that both correction methods lead to a loss of robust associations, although this loss is notably smaller with the LDSR intercept correction.</p>
<figure class="fig xbox font-sm" id="gepi70016-fig-0002"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e14c/12327166/c7e58bbca3dd/GEPI-49-0-g001.jpg" loading="lazy" id="jats-graphic-3" height="287" width="608" alt="Figure 2"></p>
<div class="p text-right font-secondary"><a href="figure/gepi70016-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The proportion of robust associations before and after correcting the earlier meta‐analysis in the three analyses. (A) Correction using the GC method. (B) Correction using the LDSR method.</p></figcaption></figure><section class="tw xbox font-sm" id="gepi70016-tbl-0002"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Loss of robust associations after GC and LDSR intercept correction in the three analyses.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="left" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<thead valign="bottom">
<tr valign="bottom" style="border-bottom:solid 1px #000000">
<th rowspan="2" align="left" valign="bottom" colspan="1">Analyzed studies</th>
<th colspan="2" align="center" valign="bottom" rowspan="1">Earlier study</th>
<th rowspan="2" align="center" valign="bottom" colspan="1">Robust associations before correction</th>
<th colspan="2" align="center" valign="bottom" rowspan="1">Robust associations after correction</th>
<th align="center" colspan="2" valign="bottom" rowspan="1">Robust associations lost</th>
</tr>
<tr valign="bottom" style="border-bottom:solid 1px #000000">
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>λ</em>‐value</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">LDSR</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">GC</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">LDSR</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">GC</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">LDSR</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>DIAMANTE‐18 versus DIAMANTE‐22</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.387</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.097</td>
<td align="center" valign="top" rowspan="1" colspan="1">18,266</td>
<td align="center" valign="top" rowspan="1" colspan="1">9907</td>
<td align="center" valign="top" rowspan="1" colspan="1">15,757</td>
<td align="center" valign="top" rowspan="1" colspan="1">8359</td>
<td align="center" valign="top" rowspan="1" colspan="1">2509</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>DIAMANTE‐18 versus T2DGGI‐24</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.387</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.097</td>
<td align="center" valign="top" rowspan="1" colspan="1">18,830</td>
<td align="center" valign="top" rowspan="1" colspan="1">9893</td>
<td align="center" valign="top" rowspan="1" colspan="1">15,713</td>
<td align="center" valign="top" rowspan="1" colspan="1">8937</td>
<td align="center" valign="top" rowspan="1" colspan="1">3117</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>DIAMANTE‐22 versus T2DGGI‐24</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.402</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.091</td>
<td align="center" valign="top" rowspan="1" colspan="1">20,405</td>
<td align="center" valign="top" rowspan="1" colspan="1">10,390</td>
<td align="center" valign="top" rowspan="1" colspan="1">17,364</td>
<td align="center" valign="top" rowspan="1" colspan="1">10,015</td>
<td align="center" valign="top" rowspan="1" colspan="1">3041</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/gepi70016-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="gepi70016-tbl2-note-0001"><p>
<em>Note:</em> LDSR indicates LDSR intercept calculated on the smaller study in the pairwise analysis.</p></div></div></section></section><section id="gepi70016-sec-0130"><h3 class="pmc_sec_title">3.3. Robust Associations Lost Only to GC Correction Have Significantly Lower <em>p</em> values</h3>
<p>Since the λ‐value is substantially higher than LDSR intercept value for the DIAMANTE‐18 and the DIAMANTE‐22 meta‐analyses (Table <a href="#gepi70016-tbl-0002" class="usa-link">2</a>), robust associations lost to LDSR intercept correction are a subset of those lost to GC correction in the three analyses. Given this observation, we analyzed associations lost only to GC correction compared to those lost to both correction methods. When analyzing DIAMANTE‐18 and DIAMANTE‐22, we found 5850 robust associations that were lost only to GC correction and 2509 robust associations that were lost to GC correction and LDSR intercept correction. As expected, we found that robust associations lost only to GC correction have significantly lower <em>p</em> values compared to robust associations lost to GC and LDSR intercept correction (<em>t</em>‐test <em>p</em> &lt; 2 × 10<sup>−16</sup>, Figure <a href="#gepi70016-fig-0003" class="usa-link">3A</a>, Supporting Information S1: Figures <a href="#gepi70016-suppl-0001" class="usa-link">S6</a> and <a href="#gepi70016-suppl-0001" class="usa-link">S7</a>). On average, we observed lower MAF for the associations lost only due to GC correction compared to associations lost with both correction methods (<em>t</em>‐test <em>p</em> &lt; 2 × 10<sup>−16</sup>, Figure <a href="#gepi70016-fig-0003" class="usa-link">3B</a>). We did not find significant differences in LD scores between associations lost only due to GC correction or to both correction methods (<em>t</em>‐test <em>p</em> = 0.346, Figure <a href="#gepi70016-fig-0003" class="usa-link">3C</a>). Similar results were observed in the DIAMANTE‐18 vs. T2DGGI‐24 analysis (Supporting Information S1: Figure <a href="#gepi70016-suppl-0001" class="usa-link">S6</a>), while in the DIAMANTE‐22 vs. T2DGGI‐24 analysis, we observed significant differences in both the <em>p</em> value and LD score distributions (Supporting Information S1: Figure <a href="#gepi70016-suppl-0001" class="usa-link">S7</a>).</p>
<figure class="fig xbox font-sm" id="gepi70016-fig-0003"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e14c/12327166/ac8b5fb965ef/GEPI-49-0-g003.jpg" loading="lazy" id="jats-graphic-5" height="295" width="607" alt="Figure 3"></p>
<div class="p text-right font-secondary"><a href="figure/gepi70016-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distributions of <em>p</em> values, MAF and LD‐scores for robust associations lost only to GC correction compared to those lost to both GC and LDSR intercept correction in the DIAMANTE‐18 and the DIAMANTE‐22 analysis. <em>p</em> value distribution: <em>t</em>‐test <em>p</em> &lt; 2 × 10<sup>−16</sup>. Minor allele frequency distribution: <em>t</em>‐test <em>p</em> &lt; 2 × 10<sup>−16</sup>. LD score distribution: <em>t</em>‐test p = 0.346. (A) <em>P</em>‐value distribution (B) Minor allele frequency distribution (C) LD‐score distribution 0.015</p></figcaption></figure></section><section id="gepi70016-sec-0140"><h3 class="pmc_sec_title">3.4. Confirmation Rates Vary Across the Genome</h3>
<p>We observed that confirmation rates vary across the genome, with or without correcting the <em>p</em> values for genomic inflation (Table <a href="#gepi70016-tbl-0003" class="usa-link">3</a> and Supporting Information S1: Table <a href="#gepi70016-suppl-0001" class="usa-link">S2</a>). To assess whether specific chromosomes show confirmation rates significantly different from the rest of the genome, we employed a LOCO approach with 100,000 samplings (methods). Across all three uncorrected analyses, we consistently observed that the confirmation rates for chromosomes 1 and 14 did not differ from the rest of the genome (Supporting Information S1: Table <a href="#gepi70016-suppl-0001" class="usa-link">S2</a>). In contrast, chromosomes 10, 12, 13, 15, 16 and 17 exhibited significantly higher confirmation rates, while chromosome 13 had a significantly lower confirmation rate compared to the rest of the genome. We then examined whether these confirmation patterns persisted after applying GC or LDSR intercept correction. Following GC correction, chromosomes 10, 12, 15, 16 and 17 continue showing a higher confirmation rate, while chromosome 13 showed lower confirmation rate compared to the rest of the genome (Table <a href="#gepi70016-tbl-0003" class="usa-link">3</a>). After LDSR intercept correction, only chromosomes 10, 12, and 15 continue showing significantly higher confirmation rates than the rest of the genome. While no chromosome showed a lower confirmation rate than the rest of the genome (Table <a href="#gepi70016-tbl-0003" class="usa-link">3</a>).</p>
<section class="tw xbox font-sm" id="gepi70016-tbl-0003"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Chromosomal confirmation rate assessed using a LOCO framework with 100,000 genome‐wide random samplings (after GC or LDSR intercept correction) in the DIAMANTE‐22 vs T2DGGI‐24 analysis as an example.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="left" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<thead valign="bottom">
<tr valign="bottom">
<th rowspan="3" align="left" valign="bottom" colspan="1">Chr</th>
<th colspan="2" align="center" valign="bottom" rowspan="1">Observed confirmation rate</th>
<th align="center" colspan="2" valign="bottom" rowspan="1">Mean confirmation rate in the rest of the genome from simulation (<em>p</em> value)</th>
</tr>
<tr valign="bottom" style="border-bottom:solid 1px #000000">
<th colspan="2" align="left" valign="bottom" rowspan="1">DIAMANTE‐22 versus T2DGGI‐24</th>
<th align="center" colspan="2" valign="bottom" rowspan="1">DIAMANTE‐22 versus T2DGGI‐24</th>
</tr>
<tr valign="bottom" style="border-bottom:solid 1px #000000">
<th align="center" valign="bottom" rowspan="1" colspan="1">GC correction</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">LDSR intercept correction</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">GC correction</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">LDSR intercept correction</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">1</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.81</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.88</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.80 (0.36)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.88 (0.10)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.73</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.86</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.71 (<strong>&lt; 1e‐6</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.85 (<strong>&lt; 1e‐6</strong>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">3</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.77</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.83</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.75 (<strong>1e‐5</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.82 (<strong>&lt; 1e‐6</strong>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">4</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.68</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.7</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.74 (<strong>&lt; 1e‐6</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.73 (<strong>&lt; 1e‐6</strong>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">5</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.73</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.83</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.77 (<strong>&lt; 1e‐6</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.85 (<strong>&lt; 1e‐6</strong>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">6</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.73</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.88</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.75 (<strong>&lt; 1e‐6</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.90 (<strong>&lt; 1e‐6</strong>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">7</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.79</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.91</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.79 (0.24)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.90 (2e‐2)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">8</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.78</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.75</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.77 (1e‐2)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.74 (<strong>1e‐3</strong>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">9</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.76</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.74</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.75 (4e‐2)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.74 (0.11)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">10</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.75</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.91</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.73 (<strong>&lt; 1e‐6</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.90 (<strong>7e‐4</strong>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">11</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.74</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.72</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.73 (<strong>&lt; 1e‐6</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.71 (<strong>2e‐4</strong>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">12</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.77</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.83</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.75 (<strong>&lt; 1e‐6</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.81 (<strong>1e‐5</strong>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">13</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.78</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.69</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.86 (<strong>&lt; 1e‐6</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.69 (0.28)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">14</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.8</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.6</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.80 (0.49)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.60 (9e‐2)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">15</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.82</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.83</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.80 (<strong>&lt; 1e‐6</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.81 (<strong>2e‐5</strong>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">16</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.72</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.86</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.70 (<strong>1e‐4</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.85 (<strong>2e‐3</strong>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">17</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.76</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.83</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.74 (<strong>1e‐5</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.82 (2e‐3)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">18</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.77</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.73</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.75 (7e‐3)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.72 (2e‐2)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">19</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.75</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.82</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.73 (<strong>5e‐4</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.80 (1e‐2)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">20</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.72</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.79</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.75 (<strong>8e‐4</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.79 (0.20)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">22</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.74</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.75</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.76 (<strong>5e‐4</strong>)</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.75 (0.20)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/gepi70016-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="gepi70016-tbl3-note-0001"><p>
<em>Note:</em> Empirical <em>p</em> values highlighted in bold are significant at a Bonferroni significance threshold of 0.0023 corrected for the 21 chromosomes.</p></div>
<div class="fn" id="gepi70016-tbl3-note-0002"><p>Abbreviation: Chr, chromosome.</p></div>
</div></section></section><section id="gepi70016-sec-0150"><h3 class="pmc_sec_title">3.5. Correction Marginally Reduces the False Positive Rate and Decreases the True Positive Rate</h3>
<p>The main purpose of applying correction methods like GC and LDSR is to eliminate false positive associations while retaining robust associations. We therefore evaluated whether the loss of robust associations was compensated by a strong decrease in false positive signals by computing the false positive rate and the true positive rate across the pairwise analyses. While applying the GC correction led to a complete elimination of false positive associations in the DIAMANTE‐18 vs the DIAMANTE‐22 meta‐analysis (Table <a href="#gepi70016-tbl-0004" class="usa-link">4</a>), the true positive rate fell drastically in the same analysis. When using LDSR intercept correction, the false positive rate was 10 times lower than before correction, and the decrease in the true positive rate was smaller compared to the GC correction (Table <a href="#gepi70016-tbl-0004" class="usa-link">4</a>). A similar pattern was observed in the DIAMANTE‐18 vs the T2DGGI‐24 and the DIAMANTE‐22 vs the T2DGGI‐24 analyses (Table <a href="#gepi70016-tbl-0004" class="usa-link">4</a>). Despite the observed decrease in the false positive rate, we show that this rate was small even before applying any correction in the different meta‐analyses. This observation is likely due to the individual studies contributing to the meta‐analysis being corrected at the individual study level using GC. In summary, we conclude that applying the GC correction method in large meta‐analyses reduces the false positive rate, although being already low, and substantially decreases the true positive rate, resulting in a loss of robust associations. In concordance with the previous result of our study, the decrease in true positive rate was smaller using the LDSR intercept correction than the GC correction.</p>
<section class="tw xbox font-sm" id="gepi70016-tbl-0004"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>False positive rate and true positive rate before and after GC and LDSR intercept correction in the three analyses.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="left" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<thead valign="bottom">
<tr valign="bottom" style="border-bottom:solid 1px #000000">
<th rowspan="2" align="left" valign="bottom" colspan="1"></th>
<th colspan="4" align="center" valign="bottom" rowspan="1">GC correction</th>
<th align="center" colspan="4" valign="bottom" rowspan="1">LDSR intercept correction</th>
</tr>
<tr valign="bottom" style="border-bottom:solid 1px #000000">
<th align="left" colspan="2" valign="bottom" rowspan="1">False positive rate</th>
<th colspan="2" align="center" valign="bottom" rowspan="1">True positive rate</th>
<th colspan="2" align="center" valign="bottom" rowspan="1">False positive rate</th>
<th align="center" colspan="2" valign="bottom" rowspan="1">True positive rate</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Before/After correction</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>Before</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>After</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>Before</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>After</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>Before</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>After</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>Before</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>After</strong>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>DIAMANTE‐18 versus DIAMANTE‐22</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.0001</td>
<td align="center" valign="top" rowspan="1" colspan="1">0</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.87</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.47</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.0001</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.00001</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.87</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.75</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>DIAMANTE‐18 versus T2DGGI‐24</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.00004</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.000001</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.28</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.15</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.00004</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.00002</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.28</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.24</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>DIAMANTE‐22 versus T2DGGI‐24</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.00005</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.000004</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.31</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.15</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.00005</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.00002</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.31</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.26</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/gepi70016-tbl-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="gepi70016-sec-0160"><h3 class="pmc_sec_title">3.6. Correction Post Meta‐Analysis Leads to Loss of Independent Loci</h3>
<p>As tested variants in GWAS meta‐analyses are correlated due to LD, we investigated whether the lost robust associations lead to a loss of independent loci, and thus biological insight into the disease. To do so, we evaluated if robust associations lost from each of the three analyses were tagged by any of the predefined T2D index variant in the original study (methods). Robust associations that were untagged were further analyzed and a subset of them were found to represent lost independent loci (methods). Comparing the DIAMANTE‐18 and DIAMANTE‐22 studies while excluding single variant clumps, we found that GC and LDSR intercept corrections resulted in the loss of 149 (39.2%) and 90 (23.6%) independent loci from the DIAMANTE‐18 meta‐analysis, respectively (Table <a href="#gepi70016-tbl-0005" class="usa-link">5</a>). When analyzing the DIAMANTE and the T2DGGI‐24 studies, both correction methods showed a loss of large number of loci (Table <a href="#gepi70016-tbl-0005" class="usa-link">5</a>). The DIAMANTE‐22 meta‐analysis showed a loss of 170 loci due to GC correction which corresponds to a loss of 38.0% of the independent loci before correction in the DIAMANTE‐22 study. We observed that both correction methods result in a substantial loss of biological information, but, in concordance with the previous results, the loss was smaller using the LDSR intercept correction compared to GC correction.</p>
<section class="tw xbox font-sm" id="gepi70016-tbl-0005"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Loss of independent loci after GC and LDSR intercept correction in the three analyses.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="left" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<thead valign="bottom">
<tr valign="bottom">
<th rowspan="2" align="left" valign="bottom" colspan="1">Analyzed studies</th>
<th align="center" valign="bottom" rowspan="1" colspan="1"></th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="center" valign="bottom" rowspan="1">GC correction</th>
<th align="center" colspan="3" style="border-bottom:solid 1px #000000" valign="bottom" rowspan="1">LDSR intercept correction</th>
</tr>
<tr valign="bottom" style="border-bottom:solid 1px #000000">
<th align="center" valign="bottom" rowspan="1" colspan="1">Significant independent loci before correction</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Lost robust associations untagged</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Independent loci lost</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">% of lost independent loci</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Lost robust associations untagged</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Independent loci lost</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">% of lost independent loci</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>DIAMANTE‐18 versus DIAMANTE‐22</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">380</td>
<td align="center" valign="top" rowspan="1" colspan="1">1609</td>
<td align="center" valign="top" rowspan="1" colspan="1">149</td>
<td align="center" valign="top" rowspan="1" colspan="1">39.20%</td>
<td align="center" valign="top" rowspan="1" colspan="1">490</td>
<td align="center" valign="top" rowspan="1" colspan="1">90</td>
<td align="center" valign="top" rowspan="1" colspan="1">23.60%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>DIAMANTE‐18 versus T2DGGI‐24</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">380</td>
<td align="center" valign="top" rowspan="1" colspan="1">1799</td>
<td align="center" valign="top" rowspan="1" colspan="1">151</td>
<td align="center" valign="top" rowspan="1" colspan="1">39.70%</td>
<td align="center" valign="top" rowspan="1" colspan="1">656</td>
<td align="center" valign="top" rowspan="1" colspan="1">96</td>
<td align="center" valign="top" rowspan="1" colspan="1">25.20%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<strong>DIAMANTE‐22 versus T2DGGI‐24</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">447</td>
<td align="center" valign="top" rowspan="1" colspan="1">2593</td>
<td align="center" valign="top" rowspan="1" colspan="1">170</td>
<td align="center" valign="top" rowspan="1" colspan="1">38.00%</td>
<td align="center" valign="top" rowspan="1" colspan="1">1068</td>
<td align="center" valign="top" rowspan="1" colspan="1">102</td>
<td align="center" valign="top" rowspan="1" colspan="1">22.80%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/gepi70016-tbl-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section></section><section id="gepi70016-sec-0170"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>Correction of genomic inflation arising from systematic bias in GWAS is important in genetic association studies. Using T2D as an exemplar polygenic disease and three of the largest T2D GWAS meta‐analyses to date, we explore two widely used correction methods, the GC based on the λ‐value, and the LDSR. We confirm previous observations that λ‐values from meta‐analyses increase with sample size and captured polygenicity of the disease. By further investigating the consequences of correction on associations, we show that it leads to a loss of power, through the loss of confirmed, high confidence associations, as well as independent loci. We further show that, GC correction leads to a larger loss in power compared to LDSR intercept correction, characterized by a smaller loss of robust associations and independent loci.</p>
<p>The emergence of global consortia and the resulting increase in the sample size of large GWAS meta‐analyses have greatly contributed to our understanding of the genetic architecture of complex traits and diseases like height, T2D, and schizophrenia (Trubetskoy et al. <a href="#gepi70016-bib-0025" class="usa-link" aria-describedby="gepi70016-bib-0025">2022</a>; Yengo et al. <a href="#gepi70016-bib-0032" class="usa-link" aria-describedby="gepi70016-bib-0032">2022</a>; Suzuki et al. <a href="#gepi70016-bib-0024" class="usa-link" aria-describedby="gepi70016-bib-0024">2024</a>). This has been accompanied by higher <em>λ</em>‐values for traits with a substantial polygenic background. The polygenicity of a trait is a fixed quantity which is being increasingly captured with larger sample size and higher statistical power to detect variants, asymptotically approaching that fixed quantity. In our study, we proxied polygenicity by the correlation between the χ<sup>2</sup>‐values and LD scores from ancestry specific meta‐analyses from DIAMANTE‐18, DIAMANTE‐22, and T2DGGI‐24 studies. Other methods, such as the attenuation ratio, have been developed to estimate polygenic effects (Loh et al. <a href="#gepi70016-bib-0014" class="usa-link" aria-describedby="gepi70016-bib-0014">2018</a>). This method relies on the observed <em>χ</em>
<sup>2</sup>‐test statistics, which were not available to us for all the individual T2D GWAS comprising the T2DGGI‐24 meta‐analysis. Related to this limitation, we were unable to compute the correlation between LD score and <em>χ</em>² statistics for individual T2D GWAS and relied instead on ancestry‐specific meta‐analysis summary statistics. Our results are in line with the findings of Yang et al. which describes the effect of sample size and polygenicity on <em>λ</em>‐value (Yang et al. <a href="#gepi70016-bib-0031" class="usa-link" aria-describedby="gepi70016-bib-0031">2011</a>). Additionally, we show that <em>λ</em>‐value and LDSR intercept are sensitive to the variant frequency thresholds used in meta‐analysis, with the inclusion of rarer variants leading to a decreased inflation, with an effect less pronounced on the LDSR intercept. Larger sample size leads to increased statistical power, which is, however, still limited for rare and low frequency variants (Lee et al. <a href="#gepi70016-bib-0012" class="usa-link" aria-describedby="gepi70016-bib-0012">2014</a>; Bomba et al. <a href="#gepi70016-bib-0004" class="usa-link" aria-describedby="gepi70016-bib-0004">2017</a>). Even if detected as genome‐wide significant, such rare variants likely have <em>p</em> values close to the significance threshold and will, therefore, also be impacted by GC correction, as highlighted in a previous study by Georgiopoulos &amp; Evangelou (Georgiopoulos and Evangelou <a href="#gepi70016-bib-0010" class="usa-link" aria-describedby="gepi70016-bib-0010">2016</a>).</p>
<p>In addition, we investigate, for the first time, the consequences on information lost from correction. Firstly, we demonstrate that GC correction leads to a loss of robust associations, i.e., associations that were confirmed in a larger GWAS meta‐analysis. By assessing these lost associations in a subsequent study, we further confirm that they correspond to high confidence signals being recapitulated in meta‐analyses with larger sample sizes. Some associations were lost only with GC correction and not LDSR intercept correction due to the larger λ‐values, which were found to show significantly lower <em>p</em> values and MAF. Variants with higher MAF are expected to have longer range LD and higher LD scores compared to variants with lower MAF (Gazal et al. <a href="#gepi70016-bib-0009" class="usa-link" aria-describedby="gepi70016-bib-0009">2017</a>; Lee et al. <a href="#gepi70016-bib-0011" class="usa-link" aria-describedby="gepi70016-bib-0011">2018</a>). However, we found no significant difference between the LD scores of associations being lost only to GC correction compared to associations lost to both correction methods. Analyzing the confirmation rate by chromosome reveals distinct patterns, some of which are consistent while others disappear after GC or LDSR intercept correction. Moreover, reflecting these results, we observe that GC correction led to a steep decline in the true positive rate, as well as a reduction in an already low false positive rate. Compared to GC correction, LDSR intercept correction resulted in a lower loss of robust associations and independent loci but still led to a substantial reduction in statistical power, affecting as much as 25.2% of the T2D loci identified in the DIAMANTE‐18 meta‐analysis. Thus, we demonstrate that using either correction method ultimately leads to a substantial loss of potential biological insights.</p>
<p>Although our study demonstrates the effect of GC and LDSR intercept correction in large meta‐analyses, we focus our main comparative analysis only on the European subset of the meta‐analyses to limit the impact of ancestry‐related differences. Further research is needed to assess the impact of correction in other ancestries. However, current sample sizes for these populations are still smaller compared to those in European GWAS. When sample sizes are equalized, we expect the impact to be at least as significant as observed in European ancestry GWAS. This is particularly true for LDSR intercept correction, which relies on LD estimates that are less precise due to underrepresentation of non‐EUR populations in LD reference panels (Appadurai et al. <a href="#gepi70016-bib-0001" class="usa-link" aria-describedby="gepi70016-bib-0001">2023</a>). Leveraging the power of genetic diversity, multi‐ancestry meta‐analysis tools like MR‐MEGA have been key to the discovery of novel genetic loci associated with complex traits (Mägi et al. <a href="#gepi70016-bib-0015" class="usa-link" aria-describedby="gepi70016-bib-0015">2017</a>). The impact of correction in multi‐ancestry meta‐analysis remains to be explored, especially LDSR intercept correction, which is affected by the aforementioned challenges. In the multi‐ancestry meta‐analyses of DIAMANTE‐22 and T2DGGI‐24, the authors reported lower λ‐values when accounting for ancestry‐correlated allelic effect heterogeneity compared to fixed or random effects meta‐analysis. This highlights the need to use specific methods when performing trans‐ancestry meta‐analyses to better model differences across ancestries. Studying variants with heterogenous effect sizes across ancestral groups, which are detectable in larger and more diverse meta‐analyses, is important for understanding the genetic architecture of the disease. Considering this, it would be important to assess the impact of correction on the confirmation of variants with fixed and heterogeneous effects across ancestral groups. Another limitation of our study is the presence of sample overlap among the three meta‐analyses. Specifically, the DIAMANTE‐18 and DIAMANTE‐22 datasets are fully included within the T2DGGI‐24 meta‐analysis. Such overlap can artificially inflate the observed concordance between studies. However, T2DGGI‐24 includes more than three times the number of T2D cases compared to either DIAMANTE studies, resulting in substantially greater statistical power to detect associations (Skol et al. <a href="#gepi70016-bib-0022" class="usa-link" aria-describedby="gepi70016-bib-0022">2006</a>). This large sample size enables the T2DGGI‐24 meta‐analysis to more comprehensively capture the polygenic architecture of T2D, leading to identification of several hundred novel loci—substantially more than either of the DIAMANTE studies (Rotondi and Bull <a href="#gepi70016-bib-0021" class="usa-link" aria-describedby="gepi70016-bib-0021">2013</a>).</p>
<p>The GC method was originally developed under the assumption that more than 50% of the tested genetic variants are not associated with the trait. However, this assumption is increasingly challenged in large‐scale GWAS meta‐analyses of complex traits, where a substantial portion of the genome shows association. As highlighted from our study, further methodological developments to better assess genomic inflation are needed. One potential direction would be to use different percentiles of the test statistic distribution in the GC correction rather than the median. Alternatively, as statistical power depends on the MAF of the variants (Supporting Information S1: Figure <a href="#gepi70016-suppl-0001" class="usa-link">S3A</a>), GC corrections could be derived representing different frequency strata. Ideally, new methods should be developed at a finer scale to improve calibration and better capture the genetic architecture of complex traits. As meta‐analyses increase in sample size, they gain greater power to detect associations involving low‐frequency variants. However, many of these variants are absent or poorly represented in reference panels, leading to inaccuracies in LD score estimation. This mismatch can result in inflated estimates of confounding in the LDSR, as the model may misattribute the signal from low‐frequency polygenic effects to confounding bias.</p>
<p>In conclusion, our findings suggest that GC and LDSR intercept correction are not optimal for addressing genomic inflation in large meta‐analyses of polygenic traits, as polygenicity is better accounted for in large meta‐analyses. We demonstrate that although LDSR intercept correction reduces the loss of robust associations and independent loci compared to GC correction, it still leads to a loss of power. We expect these issues to persist in upcoming meta‐analyses, which will be larger and thus capturing more polygenic effects of complex traits, as well as more diverse, making the use of LD‐based methods challenging. We therefore highlight the need for methodological development to better capture and correct for genomic inflation in large GWAS meta‐analyses, including the need of more representative LD reference panels to improve the accuracy of LD score calculations.</p></section><section id="sec18"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="gepi70016-suppl-0001"><div class="caption p"><p>
<strong>Supplementary:</strong> Tables and Figures.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12327166/bin/GEPI-49-0-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">GEPI-49-0-s001.docx</a><sup> (8.4MB, docx) </sup>
</div></div></section></section><section id="gepi70016-sec-0180" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>Archit Singh and Dr Ozvan Bocher have received funding from the European Union's Horizon 2020 research and innovation program under Grant Agreement No 101017802 (OPTOMICS). We gratefully acknowledge the Type 2 Diabetes Global Genomics Initiative (T2DGGI) consortium for granting early access to the summary statistics utilized in our analyses. A complete list of contributing authors from the T2DGGI consortium is provided in the Supplementary data. Open Access funding enabled and organized by Projekt DEAL.</p></section><section id="notes1"><section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="gepi70016-note-0001"><p>Eleftheria Zeggini and Ozvan Bocher contributed equally to the study.</p></div></div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Eleftheria Zeggini, Email: eleftheria.zeggini@helmholtz-munich.de.</p>
<p>Ozvan Bocher, Email: ozvan.bocher@helmholtz-munich.de.</p></section><section id="gepi70016-sec-0200"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The Jupyter notebooks associated with the analysis are available in: <a href="https://github.com/hmgu-itg/lambda_project/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/hmgu-itg/lambda_project/</a>.</p></section><section id="gepi70016-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="gepi70016-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="gepi70016-bib-0001">
<cite>

Appadurai, V.
, Bybjerg‐Grauholm J., Krebs M. D., et al. 2023. “Accuracy of Haplotype Estimation and Whole Genome Imputation Affects Complex Trait Analyses in Complex Biobanks.” Communications Biology
6, no. 1: 101.
</cite> [<a href="https://doi.org/10.1038/s42003-023-04477-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9876938/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36697501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Communications%20Biology&amp;title=Accuracy%20of%20Haplotype%20Estimation%20and%20Whole%20Genome%20Imputation%20Affects%20Complex%20Trait%20Analyses%20in%20Complex%20Biobanks&amp;author=V.%20Appadurai&amp;author=J.%20Bybjerg%E2%80%90Grauholm&amp;author=M.%20D.%20Krebs&amp;volume=6&amp;issue=1&amp;publication_year=2023&amp;pages=101&amp;pmid=36697501&amp;doi=10.1038/s42003-023-04477-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0002">
<cite>

Auton, A.
, Abecasis G. R., Altshuler D. M., et al. 2015. “A Global Reference for Human Genetic Variation.” Nature
526, no. 7571: 68–74.
</cite> [<a href="https://doi.org/10.1038/nature15393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4750478/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20Global%20Reference%20for%20Human%20Genetic%20Variation&amp;author=A.%20Auton&amp;author=G.%20R.%20Abecasis&amp;author=D.%20M.%20Altshuler&amp;volume=526&amp;issue=7571&amp;publication_year=2015&amp;pages=68-74&amp;pmid=26432245&amp;doi=10.1038/nature15393&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0003">
<cite>

de Bakker, P. I. W.
, and Raychaudhuri S.. 2012. “Interrogating the Major Histocompatibility Complex With High‐Throughput Genomics.” Human Molecular Genetics
21, no. R1: R29–R36. 10.1093/hmg/dds384.
</cite> [<a href="https://doi.org/10.1093/hmg/dds384" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3459647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22976473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Molecular%20Genetics&amp;title=Interrogating%20the%20Major%20Histocompatibility%20Complex%20With%20High%E2%80%90Throughput%20Genomics&amp;author=P.%20I.%20W.%20de%20Bakker&amp;author=S.%20Raychaudhuri&amp;volume=21&amp;issue=R1&amp;publication_year=2012&amp;pages=R29-R36&amp;pmid=22976473&amp;doi=10.1093/hmg/dds384&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0004">
<cite>

Bomba, L.
, Walter K., and Soranzo N.. 2017. “The Impact of Rare and Low‐Frequency Genetic Variants in Common Disease.” Genome Biology
18: 77.
</cite> [<a href="https://doi.org/10.1186/s13059-017-1212-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5408830/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28449691/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=The%20Impact%20of%20Rare%20and%20Low%E2%80%90Frequency%20Genetic%20Variants%20in%20Common%20Disease&amp;author=L.%20Bomba&amp;author=K.%20Walter&amp;author=N.%20Soranzo&amp;volume=18&amp;publication_year=2017&amp;pages=77&amp;pmid=28449691&amp;doi=10.1186/s13059-017-1212-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0005">
<cite>

Boyle, E. A.
, Li Y. I., and Pritchard J. K.. 2017. “An Expanded View of Complex Traits: From Polygenic to Omnigenic.” Cell
169, no. 7: 1177–1186.
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.05.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5536862/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28622505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=An%20Expanded%20View%20of%20Complex%20Traits:%20From%20Polygenic%20to%20Omnigenic&amp;author=E.%20A.%20Boyle&amp;author=Y.%20I.%20Li&amp;author=J.%20K.%20Pritchard&amp;volume=169&amp;issue=7&amp;publication_year=2017&amp;pages=1177-1186&amp;pmid=28622505&amp;doi=10.1016/j.cell.2017.05.038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0006">
<cite>

Bulik‐Sullivan, B. K.
, Loh P.‐R., Finucane H. K., et al. 2015. “LD Score Regression Distinguishes Confounding From Polygenicity in Genome‐Wide Association Studies.” Nature Genetics
47, no. 3: 291–295.
</cite> [<a href="https://doi.org/10.1038/ng.3211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4495769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25642630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=LD%20Score%20Regression%20Distinguishes%20Confounding%20From%20Polygenicity%20in%20Genome%E2%80%90Wide%20Association%20Studies&amp;author=B.%20K.%20Bulik%E2%80%90Sullivan&amp;author=P.%E2%80%90R.%20Loh&amp;author=H.%20K.%20Finucane&amp;volume=47&amp;issue=3&amp;publication_year=2015&amp;pages=291-295&amp;pmid=25642630&amp;doi=10.1038/ng.3211&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0007">
<cite>

Canela‐Xandri, O.
, Rawlik K., and Tenesa A.. 2018. “An Atlas of Genetic Associations in UK Biobank.” Nature Genetics
50, no. 11: 1593–1599.
</cite> [<a href="https://doi.org/10.1038/s41588-018-0248-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6707814/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30349118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=An%20Atlas%20of%20Genetic%20Associations%20in%20UK%20Biobank&amp;author=O.%20Canela%E2%80%90Xandri&amp;author=K.%20Rawlik&amp;author=A.%20Tenesa&amp;volume=50&amp;issue=11&amp;publication_year=2018&amp;pages=1593-1599&amp;pmid=30349118&amp;doi=10.1038/s41588-018-0248-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0008">
<cite>

Devlin, B.
, and Roeder K.. 1999. “Genomic Control for Association Studies.” Biometrics
55, no. 4: 997–1004.
</cite> [<a href="https://doi.org/10.1111/j.0006-341x.1999.00997.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11315092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic%20Control%20for%20Association%20Studies&amp;author=B.%20Devlin&amp;author=K.%20Roeder&amp;volume=55&amp;issue=4&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;doi=10.1111/j.0006-341x.1999.00997.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0009">
<cite>

Gazal, S.
, Finucane H. K., Furlotte N. A., et al. 2017. “Linkage Disequilibrium–Dependent Architecture of Human Complex Traits Shows Action of Negative Selection.” Nature Genetics
49, no. 10: 1421–1427.
</cite> [<a href="https://doi.org/10.1038/ng.3954" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6133304/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28892061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Linkage%20Disequilibrium%E2%80%93Dependent%20Architecture%20of%20Human%20Complex%20Traits%20Shows%20Action%20of%20Negative%20Selection&amp;author=S.%20Gazal&amp;author=H.%20K.%20Finucane&amp;author=N.%20A.%20Furlotte&amp;volume=49&amp;issue=10&amp;publication_year=2017&amp;pages=1421-1427&amp;pmid=28892061&amp;doi=10.1038/ng.3954&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0010">
<cite>

Georgiopoulos, G.
, and Evangelou E.. 2016. “Power Considerations for λ Inflation Factor in Meta‐Analyses of Genome‐Wide Association Studies.” Genetics Research
98: e9.
</cite> [<a href="https://doi.org/10.1017/S0016672316000069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6865175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27193946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Research&amp;title=Power%20Considerations%20for%20%CE%BB%20Inflation%20Factor%20in%20Meta%E2%80%90Analyses%20of%20Genome%E2%80%90Wide%20Association%20Studies&amp;author=G.%20Georgiopoulos&amp;author=E.%20Evangelou&amp;volume=98&amp;publication_year=2016&amp;pages=e9&amp;pmid=27193946&amp;doi=10.1017/S0016672316000069&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0011">
<cite>

Lee, J. J.
, McGue M., Iacono W. G., and Chow C. C.. 2018. “The Accuracy of LD Score Regression as an Estimator of Confounding and Genetic Correlations in Genome‐Wide Association Studies.” Genetic Epidemiology
42, no. 8: 783–795.
</cite> [<a href="https://doi.org/10.1002/gepi.22161" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6758917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30251275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetic%20Epidemiology&amp;title=The%20Accuracy%20of%20LD%20Score%20Regression%20as%20an%20Estimator%20of%20Confounding%20and%20Genetic%20Correlations%20in%20Genome%E2%80%90Wide%20Association%20Studies&amp;author=J.%20J.%20Lee&amp;author=M.%20McGue&amp;author=W.%20G.%20Iacono&amp;author=C.%20C.%20Chow&amp;volume=42&amp;issue=8&amp;publication_year=2018&amp;pages=783-795&amp;pmid=30251275&amp;doi=10.1002/gepi.22161&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0012">
<cite>

Lee, S.
, Abecasis G. R., Boehnke M., and Lin X.. 2014. “Rare‐Variant Association Analysis: Study Designs and Statistical Tests.” American Journal of Human Genetics
95, no. 1: 5–23.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2014.06.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4085641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24995866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Rare%E2%80%90Variant%20Association%20Analysis:%20Study%20Designs%20and%20Statistical%20Tests&amp;author=S.%20Lee&amp;author=G.%20R.%20Abecasis&amp;author=M.%20Boehnke&amp;author=X.%20Lin&amp;volume=95&amp;issue=1&amp;publication_year=2014&amp;pages=5-23&amp;pmid=24995866&amp;doi=10.1016/j.ajhg.2014.06.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0013">
<cite>

Li, B.
, and Leal S. M.. 2008. “Methods for Detecting Associations With Rare Variants for Common Diseases: Application to Analysis of Sequence Data.” American Journal of Human Genetics
83, no. 3: 311–321.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2008.06.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2842185/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18691683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Methods%20for%20Detecting%20Associations%20With%20Rare%20Variants%20for%20Common%20Diseases:%20Application%20to%20Analysis%20of%20Sequence%20Data&amp;author=B.%20Li&amp;author=S.%20M.%20Leal&amp;volume=83&amp;issue=3&amp;publication_year=2008&amp;pages=311-321&amp;pmid=18691683&amp;doi=10.1016/j.ajhg.2008.06.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0014">
<cite>

Loh, P.‐R.
, Kichaev G., Gazal S., Schoech A. P., and Price A. L.. 2018. “Mixed‐Model Association for Biobank‐Scale Datasets.” Nature Genetics
50, no. 7: 906–908.
</cite> [<a href="https://doi.org/10.1038/s41588-018-0144-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6309610/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29892013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Mixed%E2%80%90Model%20Association%20for%20Biobank%E2%80%90Scale%20Datasets&amp;author=P.%E2%80%90R.%20Loh&amp;author=G.%20Kichaev&amp;author=S.%20Gazal&amp;author=A.%20P.%20Schoech&amp;author=A.%20L.%20Price&amp;volume=50&amp;issue=7&amp;publication_year=2018&amp;pages=906-908&amp;pmid=29892013&amp;doi=10.1038/s41588-018-0144-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0015">
<cite>

Mägi, R.
, Horikoshi M., Sofer T., et al. 2017. “Trans‐Ethnic Meta‐Regression of Genome‐Wide Association Studies Accounting for Ancestry Increases Power for Discovery and Improves Fine‐Mapping Resolution.” Human Molecular Genetics
26, no. 18: 3639–3650.
</cite> [<a href="https://doi.org/10.1093/hmg/ddx280" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5755684/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28911207/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Molecular%20Genetics&amp;title=Trans%E2%80%90Ethnic%20Meta%E2%80%90Regression%20of%20Genome%E2%80%90Wide%20Association%20Studies%20Accounting%20for%20Ancestry%20Increases%20Power%20for%20Discovery%20and%20Improves%20Fine%E2%80%90Mapping%20Resolution&amp;author=R.%20M%C3%A4gi&amp;author=M.%20Horikoshi&amp;author=T.%20Sofer&amp;volume=26&amp;issue=18&amp;publication_year=2017&amp;pages=3639-3650&amp;pmid=28911207&amp;doi=10.1093/hmg/ddx280&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0016">
<cite>

Mahajan, A.
, Spracklen C. N., Zhang W., et al. 2022. “Multi‐Ancestry Genetic Study of Type 2 Diabetes Highlights the Power of Diverse Populations for Discovery and Translation.” Nature Genetics
54, no. 5: 560–572. 10.1038/s41588-022-01058-3.
</cite> [<a href="https://doi.org/10.1038/s41588-022-01058-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9179018/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35551307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Multi%E2%80%90Ancestry%20Genetic%20Study%20of%20Type%202%20Diabetes%20Highlights%20the%20Power%20of%20Diverse%20Populations%20for%20Discovery%20and%20Translation&amp;author=A.%20Mahajan&amp;author=C.%20N.%20Spracklen&amp;author=W.%20Zhang&amp;volume=54&amp;issue=5&amp;publication_year=2022&amp;pages=560-572&amp;pmid=35551307&amp;doi=10.1038/s41588-022-01058-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0017">
<cite>

Mahajan, A.
, Taliun D., Thurner M., et al. 2018. “Fine‐Mapping Type 2 Diabetes Loci to Single‐Variant Resolution Using High‐Density Imputation and Islet‐Specific Epigenome Maps.” Nature Genetics
50, no. 11: 1505–1513.
</cite> [<a href="https://doi.org/10.1038/s41588-018-0241-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6287706/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30297969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Fine%E2%80%90Mapping%20Type%202%20Diabetes%20Loci%20to%20Single%E2%80%90Variant%20Resolution%20Using%20High%E2%80%90Density%20Imputation%20and%20Islet%E2%80%90Specific%20Epigenome%20Maps&amp;author=A.%20Mahajan&amp;author=D.%20Taliun&amp;author=M.%20Thurner&amp;volume=50&amp;issue=11&amp;publication_year=2018&amp;pages=1505-1513&amp;pmid=30297969&amp;doi=10.1038/s41588-018-0241-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0018">
<cite>

Price, A. L.
, Zaitlen N. A., Reich D., and Patterson N.. 2010. “New Approaches to Population Stratification in Genome‐Wide Association Studies.” Nature Reviews Genetics
11, no. 7: 459–463.</cite> [<a href="https://doi.org/10.1038/nrg2813" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2975875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20548291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=New%20Approaches%20to%20Population%20Stratification%20in%20Genome%E2%80%90Wide%20Association%20Studies&amp;author=A.%20L.%20Price&amp;author=N.%20A.%20Zaitlen&amp;author=D.%20Reich&amp;author=N.%20Patterson&amp;volume=11&amp;issue=7&amp;publication_year=2010&amp;pages=459-463&amp;pmid=20548291&amp;doi=10.1038/nrg2813&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0019">
<cite>

Pritchard, J. K.

2001. “Are Rare Variants Responsible for Susceptibility to Complex Diseases?” American Journal of Human Genetics
69, no. 1: 124–137.
</cite> [<a href="https://doi.org/10.1086/321272" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1226027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11404818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Are%20Rare%20Variants%20Responsible%20for%20Susceptibility%20to%20Complex%20Diseases?&amp;author=J.%20K.%20Pritchard&amp;volume=69&amp;issue=1&amp;publication_year=2001&amp;pages=124-137&amp;pmid=11404818&amp;doi=10.1086/321272&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0020">
<cite>

Purcell, S.
, Neale B., Todd‐Brown K., et al. 2007. “Plink: A Tool Set for Whole‐Genome Association and Population‐Based Linkage Analyses.” American Journal of Human Genetics
81, no. 3: 559–575.
</cite> [<a href="https://doi.org/10.1086/519795" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1950838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Plink:%20A%20Tool%20Set%20for%20Whole%E2%80%90Genome%20Association%20and%20Population%E2%80%90Based%20Linkage%20Analyses&amp;author=S.%20Purcell&amp;author=B.%20Neale&amp;author=K.%20Todd%E2%80%90Brown&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;doi=10.1086/519795&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0021">
<cite>

Rotondi, M. A.
, and Bull S. B.. 2013. “Cumulative Meta‐Analysis for Genetic Association: When Is a New Study Worthwhile?” In S. Karger AG Basel, Vol. 74, 61–70.</cite> [<a href="https://doi.org/10.1159/000345604" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23258221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=S.%20Karger%20AG%20Basel&amp;author=M.%20A.%20Rotondi&amp;author=S.%20B.%20Bull&amp;volume=74&amp;publication_year=2013&amp;pages=61-70&amp;pmid=23258221&amp;doi=10.1159/000345604&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0022">
<cite>

Skol, A. D.
, Scott L. J., Abecasis G. R., and Boehnke M.. 2006. “Joint Analysis Is More Efficient Than Replication‐Based Analysis for Two‐Stage Genome‐Wide Association Studies.” Nature Genetics
38, no. 2: 209–213.
</cite> [<a href="https://doi.org/10.1038/ng1706" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16415888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Joint%20Analysis%20Is%20More%20Efficient%20Than%20Replication%E2%80%90Based%20Analysis%20for%20Two%E2%80%90Stage%20Genome%E2%80%90Wide%20Association%20Studies&amp;author=A.%20D.%20Skol&amp;author=L.%20J.%20Scott&amp;author=G.%20R.%20Abecasis&amp;author=M.%20Boehnke&amp;volume=38&amp;issue=2&amp;publication_year=2006&amp;pages=209-213&amp;pmid=16415888&amp;doi=10.1038/ng1706&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0023">
<cite>

Sollis, E.
, Mosaku A., Abid A., et al. 2022. “The NHGRI‐EBI GWAS Catalog: Knowledgebase and Deposition Resource.” Nucleic Acids Research
51, no. D1: D977–D985. 10.1093/nar/gkac1010.</cite> [<a href="https://doi.org/10.1093/nar/gkac1010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36350656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=The%20NHGRI%E2%80%90EBI%20GWAS%20Catalog:%20Knowledgebase%20and%20Deposition%20Resource&amp;author=E.%20Sollis&amp;author=A.%20Mosaku&amp;author=A.%20Abid&amp;volume=51&amp;issue=D1&amp;publication_year=2022&amp;pages=D977-D985&amp;pmid=36350656&amp;doi=10.1093/nar/gkac1010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0024">
<cite>

Suzuki, K.
, Hatzikotoulas K., Southam L., et al. 2024. “Genetic Drivers of Heterogeneity in Type 2 Diabetes Pathophysiology.” Nature
627: 347–357.
</cite> [<a href="https://doi.org/10.1038/s41586-024-07019-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10937372/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38374256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic%20Drivers%20of%20Heterogeneity%20in%20Type%202%20Diabetes%20Pathophysiology&amp;author=K.%20Suzuki&amp;author=K.%20Hatzikotoulas&amp;author=L.%20Southam&amp;volume=627&amp;publication_year=2024&amp;pages=347-357&amp;pmid=38374256&amp;doi=10.1038/s41586-024-07019-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0025">
<cite>

Trubetskoy, V.
, Pardiñas A. F., Qi T., et al. 2022. “Mapping Genomic Loci Implicates Genes and Synaptic Biology in Schizophrenia.” Nature
604, no. 7906: 502–508.
</cite> [<a href="https://doi.org/10.1038/s41586-022-04434-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9392466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35396580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mapping%20Genomic%20Loci%20Implicates%20Genes%20and%20Synaptic%20Biology%20in%20Schizophrenia&amp;author=V.%20Trubetskoy&amp;author=A.%20F.%20Pardi%C3%B1as&amp;author=T.%20Qi&amp;volume=604&amp;issue=7906&amp;publication_year=2022&amp;pages=502-508&amp;pmid=35396580&amp;doi=10.1038/s41586-022-04434-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0026">
<cite>

Visscher, P. M.
, Yengo L., Cox N. J., and Wray N. R.. 2021. “Discovery and Implications of Polygenicity of Common Diseases.” Science
373, no. 6562: 1468–1473.
</cite> [<a href="https://doi.org/10.1126/science.abi8206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9945947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34554790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Discovery%20and%20Implications%20of%20Polygenicity%20of%20Common%20Diseases&amp;author=P.%20M.%20Visscher&amp;author=L.%20Yengo&amp;author=N.%20J.%20Cox&amp;author=N.%20R.%20Wray&amp;volume=373&amp;issue=6562&amp;publication_year=2021&amp;pages=1468-1473&amp;pmid=34554790&amp;doi=10.1126/science.abi8206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0027">
<cite>

Voight, B. F.
, and Pritchard J. K.. 2005. “Confounding From Cryptic Relatedness in Case‐Control Association Studies.” PLoS Genetics
1, no. 3: e32.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.0010032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1200427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16151517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&amp;title=Confounding%20From%20Cryptic%20Relatedness%20in%20Case%E2%80%90Control%20Association%20Studies&amp;author=B.%20F.%20Voight&amp;author=J.%20K.%20Pritchard&amp;volume=1&amp;issue=3&amp;publication_year=2005&amp;pages=e32&amp;pmid=16151517&amp;doi=10.1371/journal.pgen.0010032&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0028">
<cite>

Wang, S.
, Chen W., Chen X., et al. 2012. “Double Genomic Control Is Not Effective to Correct for Population Stratification in Meta‐Analysis for Genome‐Wide Association Studies.” Frontiers in Genetics
3: 300.
</cite> [<a href="https://doi.org/10.3389/fgene.2012.00300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3529452/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23269928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Double%20Genomic%20Control%20Is%20Not%20Effective%20to%20Correct%20for%20Population%20Stratification%20in%20Meta%E2%80%90Analysis%20for%20Genome%E2%80%90Wide%20Association%20Studies&amp;author=S.%20Wang&amp;author=W.%20Chen&amp;author=X.%20Chen&amp;volume=3&amp;publication_year=2012&amp;pages=300&amp;pmid=23269928&amp;doi=10.3389/fgene.2012.00300&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0029">
<cite>

Willer, C. J.
, Li Y., and Abecasis G. R.. 2010. “Metal: Fast and Efficient Meta‐Analysis of Genomewide Association Scans.” Bioinformatics
26, no. 17: 2190–2191.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btq340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2922887/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20616382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Metal:%20Fast%20and%20Efficient%20Meta%E2%80%90Analysis%20of%20Genomewide%20Association%20Scans&amp;author=C.%20J.%20Willer&amp;author=Y.%20Li&amp;author=G.%20R.%20Abecasis&amp;volume=26&amp;issue=17&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;doi=10.1093/bioinformatics/btq340&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0030">
<cite>

Yang, J.
, Lee S. H., Goddard M. E., and Visscher P. M.. 2011. “GCTA: A Tool for Genome‐Wide Complex Trait Analysis.” American Journal of Human Genetics
88, no. 1: 76–82.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2010.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3014363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21167468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=GCTA:%20A%20Tool%20for%20Genome%E2%80%90Wide%20Complex%20Trait%20Analysis&amp;author=J.%20Yang&amp;author=S.%20H.%20Lee&amp;author=M.%20E.%20Goddard&amp;author=P.%20M.%20Visscher&amp;volume=88&amp;issue=1&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;doi=10.1016/j.ajhg.2010.11.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0031">
<cite>

Yang, J.
, Weedon M. N., Purcell S., et al. 2011. “Genomic Inflation Factors Under Polygenic Inheritance.” European Journal of Human Genetics
19, no. 7: 807–812.
</cite> [<a href="https://doi.org/10.1038/ejhg.2011.39" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3137506/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21407268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Human%20Genetics&amp;title=Genomic%20Inflation%20Factors%20Under%20Polygenic%20Inheritance&amp;author=J.%20Yang&amp;author=M.%20N.%20Weedon&amp;author=S.%20Purcell&amp;volume=19&amp;issue=7&amp;publication_year=2011&amp;pages=807-812&amp;pmid=21407268&amp;doi=10.1038/ejhg.2011.39&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70016-bib-0032">
<cite>

Yengo, L.
, Vedantam S., Marouli E., et al. 2022. “A Saturated Map of Common Genetic Variants Associated With Human Height.” Nature
610, no. 7933: 704–712.
</cite> [<a href="https://doi.org/10.1038/s41586-022-05275-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9605867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36224396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20Saturated%20Map%20of%20Common%20Genetic%20Variants%20Associated%20With%20Human%20Height&amp;author=L.%20Yengo&amp;author=S.%20Vedantam&amp;author=E.%20Marouli&amp;volume=610&amp;issue=7933&amp;publication_year=2022&amp;pages=704-712&amp;pmid=36224396&amp;doi=10.1038/s41586-022-05275-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Supplementary:</strong> Tables and Figures.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12327166/bin/GEPI-49-0-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">GEPI-49-0-s001.docx</a><sup> (8.4MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The Jupyter notebooks associated with the analysis are available in: <a href="https://github.com/hmgu-itg/lambda_project/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/hmgu-itg/lambda_project/</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Genetic Epidemiology are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/gepi.70016"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/GEPI-49-0.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (804.6 KB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12327166/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12327166/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12327166%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327166/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12327166/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12327166/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40767503/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12327166/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40767503/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12327166/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12327166/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="ZqVN27j3pxlRLFbXro2QHEoLCom3GkuYTwtI60UiRpxm8o7DjCdADagjmK2xeBoF">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
