
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Whole-genome sequencing and probiotic properties of Lactobacillus helveticus KM7 isolated from the gut of the Chinese honey bee (Apis cerana): A promising exopolysaccharide-producing strain - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4852D8AEF24E305852D003D892794.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcmicr">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374427/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Microbiology">
<meta name="citation_title" content="Whole-genome sequencing and probiotic properties of Lactobacillus helveticus KM7 isolated from the gut of the Chinese honey bee (Apis cerana): A promising exopolysaccharide-producing strain">
<meta name="citation_author" content="Chen Liu">
<meta name="citation_author_institution" content="Faculty of Animal Science and Technology, Yunnan Agricultural University, Panlong district, Kunming, 650201 China">
<meta name="citation_author" content="Le Xu">
<meta name="citation_author_institution" content="Faculty of Animal Science and Technology, Yunnan Agricultural University, Panlong district, Kunming, 650201 China">
<meta name="citation_author_institution" content="Key Laboratory of Animal Nutrition and Feed Science of Yunnan Province, Kunming, 650201 China">
<meta name="citation_author" content="Shiyu Chen">
<meta name="citation_author_institution" content="Faculty of Animal Science and Technology, Yunnan Agricultural University, Panlong district, Kunming, 650201 China">
<meta name="citation_author" content="Sifan Wang">
<meta name="citation_author_institution" content="Faculty of Animal Science and Technology, Yunnan Agricultural University, Panlong district, Kunming, 650201 China">
<meta name="citation_author" content="Mingkui Lv">
<meta name="citation_author_institution" content="Sericulture and Apiculture Research Institute, Yunnan Academy of Agricultural Sciences, Mengzi, 661101 China">
<meta name="citation_author" content="Kun Dong">
<meta name="citation_author_institution" content="Faculty of Animal Science and Technology, Yunnan Agricultural University, Panlong district, Kunming, 650201 China">
<meta name="citation_author" content="Qiuye Lin">
<meta name="citation_author_institution" content="School of Ethnic Medicine, Yunnan Minzu University, Chenggong district, Kunming, 650504 China">
<meta name="citation_author" content="Zhenhui Cao">
<meta name="citation_author_institution" content="Faculty of Animal Science and Technology, Yunnan Agricultural University, Panlong district, Kunming, 650201 China">
<meta name="citation_author_institution" content="Key Laboratory of Animal Nutrition and Feed Science of Yunnan Province, Kunming, 650201 China">
<meta name="citation_publication_date" content="2025 Aug 23">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="530">
<meta name="citation_doi" content="10.1186/s12866-025-04286-9">
<meta name="citation_pmid" content="40847281">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374427/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374427/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374427/pdf/12866_2025_Article_4286.pdf">
<meta name="description" content="Lactobacillus helveticus is a probiotic bacterium widely used in the food industry. In this study, we evaluate the safety and probiotic properties of L. helveticus strain KM7, isolated from the gut of Apis cerana, through whole-genome sequencing and ...">
<meta name="og:title" content="Whole-genome sequencing and probiotic properties of Lactobacillus helveticus KM7 isolated from the gut of the Chinese honey bee (Apis cerana): A promising exopolysaccharide-producing strain">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Lactobacillus helveticus is a probiotic bacterium widely used in the food industry. In this study, we evaluate the safety and probiotic properties of L. helveticus strain KM7, isolated from the gut of Apis cerana, through whole-genome sequencing and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374427/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12374427">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12866-025-04286-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12866_2025_Article_4286.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374427%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12374427/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12374427/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374427/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcmicr.png" alt="BMC Microbiology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Microbiology" title="Link to BMC Microbiology" shape="default" href="https://bmcmicrobiol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Microbiol</button></div>. 2025 Aug 23;25:530. doi: <a href="https://doi.org/10.1186/s12866-025-04286-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12866-025-04286-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Microbiol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Microbiol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Whole-genome sequencing and probiotic properties of <em>Lactobacillus helveticus</em> KM7 isolated from the gut of the Chinese honey bee (<em>Apis cerana</em>): A promising exopolysaccharide-producing strain</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Chen Liu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Chen Liu</span></h3>
<div class="p">
<sup>1</sup>Faculty of Animal Science and Technology, Yunnan Agricultural University, Panlong district, Kunming, 650201 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chen Liu</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Le Xu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Le Xu</span></h3>
<div class="p">
<sup>1</sup>Faculty of Animal Science and Technology, Yunnan Agricultural University, Panlong district, Kunming, 650201 China </div>
<div class="p">
<sup>2</sup>Key Laboratory of Animal Nutrition and Feed Science of Yunnan Province, Kunming, 650201 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Le Xu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Shiyu Chen</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Shiyu Chen</span></h3>
<div class="p">
<sup>1</sup>Faculty of Animal Science and Technology, Yunnan Agricultural University, Panlong district, Kunming, 650201 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shiyu Chen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Sifan Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Sifan Wang</span></h3>
<div class="p">
<sup>1</sup>Faculty of Animal Science and Technology, Yunnan Agricultural University, Panlong district, Kunming, 650201 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sifan Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lv%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Mingkui Lv</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Mingkui Lv</span></h3>
<div class="p">
<sup>4</sup>Sericulture and Apiculture Research Institute, Yunnan Academy of Agricultural Sciences, Mengzi, 661101 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lv%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mingkui Lv</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Kun Dong</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Kun Dong</span></h3>
<div class="p">
<sup>1</sup>Faculty of Animal Science and Technology, Yunnan Agricultural University, Panlong district, Kunming, 650201 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kun Dong</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Qiuye Lin</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Qiuye Lin</span></h3>
<div class="p">
<sup>3</sup>School of Ethnic Medicine, Yunnan Minzu University, Chenggong district, Kunming, 650504 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qiuye Lin</span></a>
</div>
</div>
<sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Zhenhui Cao</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Zhenhui Cao</span></h3>
<div class="p">
<sup>1</sup>Faculty of Animal Science and Technology, Yunnan Agricultural University, Panlong district, Kunming, 650201 China </div>
<div class="p">
<sup>2</sup>Key Laboratory of Animal Nutrition and Feed Science of Yunnan Province, Kunming, 650201 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhenhui Cao</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Faculty of Animal Science and Technology, Yunnan Agricultural University, Panlong district, Kunming, 650201 China </div>
<div id="Aff2">
<sup>2</sup>Key Laboratory of Animal Nutrition and Feed Science of Yunnan Province, Kunming, 650201 China </div>
<div id="Aff3">
<sup>3</sup>School of Ethnic Medicine, Yunnan Minzu University, Chenggong district, Kunming, 650504 China </div>
<div id="Aff4">
<sup>4</sup>Sericulture and Apiculture Research Institute, Yunnan Academy of Agricultural Sciences, Mengzi, 661101 China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jun 11; Accepted 2025 Jul 29; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12374427  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40847281/" class="usa-link">40847281</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1"><em>Lactobacillus helveticus</em> is a probiotic bacterium widely used in the food industry. In this study, we evaluate the safety and probiotic properties of <em>L</em>. <em>helveticus</em> strain KM7, isolated from the gut of <em>Apis cerana</em>, through whole-genome sequencing and in vitro experiments. The complete genome consists of 2,164,024 bp, encoding 2192 genes with an average GC content of 36.82%. The strain lacks identifiable virulence factor genes and antibiotic resistance genes, shows no hemolytic activity, and cannot produce biogenic amines such as spermine, cadaverine, or putrescine. <em>L. helveticus</em> KM7 exhibited strong acid and bile salt tolerance, superior adhesion capacity, and remarkable antioxidant activity, with its genome harboring genes associated with these traits. It contains gene clusters potentially involved in the biosynthesis of bacteriocins Helveticin J and Enterolysin A, as well as encoding multiple genes of carbohydrate-active enzymes. Moreover, KM7 demonstrated excellent exopolysaccharide-producing capability, and glucose supplementation in the medium enhanced yield. These findings indicate that KM7 is a safe strain with promising probiotic properties, highlighting its potential for food applications.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12866-025-04286-9.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> <em>Lactobacillus helveticus</em>, Whole-genome sequencing, Probiotic properties, Safety, Exopolysaccharide</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">
Probiotics are defined as living microorganisms that are beneficial to the health of the host when administered in adequate amounts, and lactic acid bacteria (LAB) are one of the most common groups [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. <em>Lactobacillus helveticus</em> is a homofermentative, Gram-positive, rod-shaped, thermophilic LAB predominantly isolated from cheese, fermented milk, and other dairy products [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Its remarkable adaptability to diverse environmental stresses, including high temperature, low pH, low osmotic pressure, and low oxygen levels, has enabled <em>L. helveticus</em> to be widely used as a probiotic [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. As a dietary supplement, <em>L</em>. <em>helveticus</em> has been demonstrated to improve mood, alleviate alcohol-induced liver injury, and ameliorate intestinal inflammation [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. Furthermore, owing to its unique proteolytic system and capacity for exopolysaccharide (EPS) production, <em>L</em>. <em>helveticus</em> is widely employed in food fermentation [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. For example, during cheese production, it not only suppresses the growth of gas-producing microorganisms by metabolizing residual carbohydrates, but it also effectively reduces bitterness and accelerates the generation of flavor compounds [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. These multifaceted probiotic and technological properties of <em>L</em>. <em>helveticus</em> underscore its broad application potential, which has recently attracted increased research attention.</p>
<p id="Par3">With the expanding applications of probiotics, safety assessment has become increasingly critical [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Systematic evaluation of strain safety and probiotic properties is a fundamental prerequisite for their use in human and animal health [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Beyond the ability to survive gastrointestinal transit, ideal probiotics should have effective intestinal adhesion and colonization capabilities, along with functional characteristics such as antimicrobial activity [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Furthermore, growing concerns regarding food safety and antibiotic resistance have underscored the importance of assessing potential virulence factors and antimicrobial resistance profiles in probiotic strains. Whole-genome sequencing (WGS) has recently emerged as an indispensable tool for strain characterization, offering unprecedented resolution for phylogenetic analysis, functional gene identification, and metabolic pathway elucidation [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Consequently, it has become a highly reliable approach for comprehensive safety and probiotic trait assessment.</p>
<p id="Par4">
The honeybee gut microbiota is highly specific and functionally diverse, playing crucial roles in carbohydrate metabolism, antioxidation, and immune modulation [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. In a previous study, we isolated <em>L</em>. <em>helveticus</em> KM7 from the <em>Apis cerana</em> gut and demonstrated its notable antimicrobial activity and antibiotic susceptibility [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. However, most currently characterized <em>L</em>. <em>helveticus</em> strains originate from fermented dairy products, while insect-derived isolates (particularly those from honeybees) remain understudied and their potential probiotic traits and genomic features poorly characterized. Therefore, we here employ WGS combined with phenotypic assays to systematically evaluate the safety, probiotic properties and EPS-producing capacity of <em>L</em>. <em>helveticus</em> KM7. Our findings provide a theoretical foundation for developing this strain as both a probiotic candidate and a microorganism for fermented food production.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Bacterial strains and growth conditions</h3>
<p id="Par5">We previously isolated <em>L</em>. <em>helveticus</em> KM7 from the intestinal tract of <em>A. cerana</em>, collected in Kunming, China (102°45’30.5’’E, 25°8’5.8’’N). KM7 was revived and anaerobically cultured in MRS medium (Guangdong Huankai Microbial Sci. &amp; Tech. Co., Ltd., Guangzhou, China; cat. no. 027312) at 37℃ for 24 h. The pathogenic strains (<em>Escherichia coli</em> K88, <em>Staphylococcus aureus</em> ATCC 49521, and <em>Salmonella</em> typhi) were revived and aerobically cultured in LB medium (Guangdong Huankai Microbial Sci. &amp; Tech. Co., Ltd., Guangzhou, China; cat. no. 028320) at 37℃ for 24 h.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Genome sequencing and analysis</h3>
<p id="Par6">The activated KM7 culture was inoculated into MRS broth at a 1% (v/v) concentration and incubated at 37 ℃ until reaching the mid-logarithmic growth phase (8 h). The bacteria were then harvested by centrifugation at 2,800 × g for 10 min, followed by three washes with sterile PBS (phosphate-buffered saline) buffer for subsequent DNA extraction, which was performed with a Wizard<sup>®</sup> Genomic DNA Purification Kit (Promega Corporation, WI, USA; cat. no. A1620). Purified genomic DNA was quantified with a TBS-380 fluorometer (Turner BioSystems Inc., CA, USA). High quality DNA (OD<sub>260/280</sub> = 1.8 ~ 2.0, &gt;20 µg) was used for further research.</p>
<p id="Par7">We sequenced the KM7 genome using a combination of PacBio RS II Single Molecule Real-Time (SMRT) and Illumina platforms. For Illumina sequencing, at least 1 µg of genomic DNA per strain was sheared into 400–500 bp fragments using a Covaris M220 Focused Acoustic Shearer (Covaris, Inc., MA, USA). Sequencing libraries were prepared using the NEXTflex™ Rapid DNA-Seq Kit (Bioo Scientific, TX, USA), including end-repair, phosphorylation, A-tailing, adapter ligation, and PCR enrichment. Paired-end sequencing (2 × 150 bp) was performed on an Illumina HiSeq X Ten. For PacBio sequencing, 15 µg of DNA was fragmented using a Covaris g-TUBE (Covaris, Inc., MA, USA) at 6,000 g for 60 s. The fragments were then purified, end-repaired, and ligated with SMRTbell adapters (Pacific Biosciences, CA, USA). The library was purified three times using 0.45 × volumes of Agencourt AMPure XP beads (Beckman Coulter, Inc., CA, USA). An ~ 10 kb insert library was prepared and sequenced on one SMRT cell using standard protocols and the complete genome sequence assembled using both PacBio and Illumina reads. Raw sequence data, generated via base calling, were saved as FASTQ files containing sequence read and quality information. Quality trimming was performed to remove low-quality reads. We assembled the genome using the hierarchical genome assembly process (HGAP) and Canu, followed by manual verification to ensure circularization, resulting in a complete genome with seamless chromosomes and plasmids. Final error correction of the PacBio assembly was performed using Illumina reads with Pilon.</p>
<p id="Par8">Coding sequences (CDSs) were predicted using Glimmer. The predicted CDSs were functionally annotated by aligning against the NR, GO, KEGG and CAZy databases using BLAST, Diamond, Hmmscan and HMMER. Annotations were assigned based on the best-matched hits with an <em>E</em>-value threshold of 10<sup>−5</sup>. Putative genomic islands, prophages, and CRISPR arrays were identified bioinformatically using IslandPath-DIMOB, PHASTER, and CRISPRCasFinder, respectively, applied to the complete genome sequence. Genes associated with probiotic functions were identified following the approaches and utilizing the databases described by Sabino et al. and Rocha et al. [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Putative bacteriocin biosynthesis gene clusters were identified using the BAGEL4 web server (<a href="http://bagel4.molgenrug.nl" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bagel4.molgenrug.nl</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Comparative genome analysis and reconstruction of the pan-core gene set</h3>
<p id="Par9">We performed comparative genome analysis and pan-core gene set reconstruction of <em>L</em>. <em>helveticus</em> KM7 and six other <em>L</em>. <em>helveticus</em> strains from different sources: DS3_8 from commercial dietary supplements (GenBank accession: GCA_003053085.1), CAUH18 from koumiss (GCA_001308285.1), CGMCC from Emmental cheese (GCA_001434945.1), LBGT 162 from sorghum beer fermentation (GCA_019455825.1), LZ-R-5 from yoghurt (GCA_009498395.1), and TMW 12,315 from rice vinegar fermentation (GCA_019455865.1). We calculated the pairwise average nucleotide identity (ANI) between KM7 and the other strains using the integrated prokaryotes genome and pan-genome analysis service (IPGA) v1.09. We reconstructed the pan-genome also using the IPGA, which incorporates the Panaroo tool (v1.3.2) for gene clustering and annotation. Orthologous gene clusters were identified using a 70% nucleotide identity threshold, and core genes were defined as those present in ≥ 95% of the analyzed strains. A stringent support value of 1.0 was applied to ensure high-confidence clustering. Finally, we reconstructed a core gene-based phylogenetic tree using VBCG v1.3 with FastTree to assess sequence divergence among strains.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Safety assessment</h3>
<section id="Sec7"><h4 class="pmc_sec_title">Identification of safety-relevant genes</h4>
<p id="Par10">We predicted potential virulence factor-related genes in the genome of KM7 using the Virulence Factor Database (VFDB) v20240301 (<a href="https://www.mgc.ac.cn/VFs/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.mgc.ac.cn/VFs/</a>) and annotated antibiotic resistance genes via the Comprehensive Antibiotic Resistance Database (CARD) v3.2.9 (<a href="https://card.mcmaster.ca" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://card.mcmaster.ca</a>). In accordance with the Guidelines for Identification and Safety Evaluation of Strains Used in Direct Feeding of Microorganisms and Fermented Products [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>], we set the screening criteria for virulence factors and antibiotic resistance genes as follows: <em>E</em> value &lt; 10<sup>−5</sup>, coverage &gt; 70%, similarity &gt; 80%.</p></section><section id="Sec8"><h4 class="pmc_sec_title">Hemolytic activity assessment</h4>
<p id="Par11">The activated cultures of <em>L</em>. <em>helveticus</em> KM7 and <em>S. aureus</em> ATCC 49521 (negative control) were streaked onto Columbia blood agar plates (Guangdong Huankai Microbial Sci. &amp; Tech. Co., Ltd., Guangzhou, China) and incubated at 37℃ for 48 h. Hemolytic activity was assessed by observing hemolysis zones around the colonies: A clear zone surrounding the colony signifies β-hemolysis, whereas a greenish discoloration indicates α-hemolysis; in contrast, the absence of any hemolytic activity is classified as γ-hemolysis.</p></section><section id="Sec9"><h4 class="pmc_sec_title">Assessment of the ability to produce biogenic amines</h4>
<p id="Par12">The capacity to produce putrescine, cadaverine, and spermidine was assessed using biochemical test tubes (Hangzhou Binhe Microbial Reagent Co., Ltd., Hangzhou, China, cat. nos. A139, A140, A141). KM7 was streaked onto MRS agar plates and incubated at 37℃ for 24 h. Single colonies were then inoculated into test tubes for arginine decarboxylase, lysine decarboxylase, and ornithine decarboxylase assays, along with control tubes, followed by sealing with paraffin oil. After 24 h of incubation at 37℃, color changes were observed. A negative result was recorded if both the test and control tubes turned yellow, whereas a positive result was indicated by a color change in the test tube with the control tube remaining yellow.</p></section></section><section id="Sec10"><h3 class="pmc_sec_title">Probiotic characterization</h3>
<section id="Sec11"><h4 class="pmc_sec_title">Acid and bile salt tolerance</h4>
<p id="Par13">We adjusted the pH of MRS medium to different levels (2.0, 3.0, and 4.0) using 1 mol/L diluted hydrochloric acid (HCl) (Tianjin Fengchuan Chemical Reagent Technology Co., Ltd., Tianjin, China) and inoculated activated KM7 into the adjusted medium at 5% (v/v) concentration, followed by culturing at 37℃. Viable cell counts were determined by plating on MRS agar at 0, 1, 2, and 3 h, followed by survival rate calculation. For bile salt tolerance assessment, we inoculated KM7 at 5% (v/v) concentration into MRS medium containing 0.3% (w/v) bile salt (bovine) (Solarbio Biotechnology Co., Ltd., Beijing, China, cat. no. B8210) and cultured it at 37 ℃. Survival rates were determined at 0, 1, 2, and 3 h as follows:</p>
<table class="disp-formula p" id="Equa"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb7b/12374427/ffd8629a2982/d33e490.gif" loading="lazy" id="d33e490" alt="graphic file with name d33e490.gif"></td></tr></table>
<p id="Par14">where <em>N</em><sub><em>t</em></sub> refers to the number of viable bacteria corresponding to 1, 2 and 3 h and <em>N</em><sub><em>0</em></sub> refers to the number of viable bacteria corresponding to 0 h.</p></section><section id="Sec12"><h4 class="pmc_sec_title">Bile salt hydrolase activity</h4>
<p id="Par15">KM7 was cultured in MRS broth at 37℃ for 2, 4, 6, 8, 10, and 12 h. Subsequently, the cultures were centrifuged at 8,000 × g for 10 min at 4℃. The resulting pellets were washed twice with PBS (pH 7.4) and resuspended in the same buffer. The suspensions were subjected to ultrasonication using an ultrasonic cell disruptor (Ningbo Scientz Biotechnology Co., Ltd., Ningbo, China, cat. no. 950E) operated at 200 W with cycles of 5 s pulse and 5 s pause for 15 min. Cell debris was removed by centrifugation. The Bile salt hydrolase (BSH) activity in the supernatant was quantified using a commercial BSH Activity Assay Kit (Gerisi Bio-Technologies Co., Ltd., Suzhou, China, cat. no. G0933F). One unit of enzyme activity (U) was defined as the amount of enzyme required to catalyze the production of 1 µg of taurine per minute per milliliter of supernatant under the assay conditions.</p></section><section id="Sec15"><h4 class="pmc_sec_title">Adhesion and aggregation</h4>
<section id="FPar1"><h5 class="pmc_sec_title">Cell-surface hydrophobicity</h5>
<p id="Par16">KM7 suspension cultured for 24 h was centrifuged at 4,000 × g for 10 min at 4℃. The collected cells were washed twice with PBS and resuspended in 2 mL PBS to a concentration of 10<sup>8</sup> CFU/mL, followed by measurement of the optical density at 600 nm (<em>A</em><sub><em>0</em></sub>). Then, 2 mL cell suspension was mixed with 2 mL xylene (C<sub>8</sub>H<sub>10</sub>) (Tianjin Zhiyuan Chemical Reagent Co., Ltd., Tianjin, China), vortexed for 5 min, and allowed to stand for 30 min for phase separation. The optical density of the aqueous phase was measured at 600 nm (<em>A</em><sub><em>1</em></sub>). Cell surface hydrophobicity was calculated as follows: </p>
<table class="disp-formula p" id="Equb"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb7b/12374427/8ee76c9a53c0/d33e538.gif" loading="lazy" id="d33e538" alt="graphic file with name d33e538.gif"></td></tr></table></section><section id="FPar2"><h5 class="pmc_sec_title"><em>Auto-aggregation and co-aggregation</em></h5>
<p id="Par17">We cultured KM7 anaerobically in MRS broth at 37℃ until the late exponential phase (OD<sub>600</sub> ≈ 1.0). Cells were harvested by centrifugation (5,000 × g, 10 min, 4℃), washed twice with PBS, and resuspended in the same buffer to a final concentration of 10<sup>8</sup> CFU/mL. The suspension was then vortexed for 10 s and allowed to incubate at 37℃. Aliquots (200 µl) from the upper layer were collected at 0 h (<em>A</em><sub><em>0</em></sub>) and 3 h (<em>A</em><sub><em>2</em></sub>), and OD<sub>600</sub> measured using a spectrophotometer (METASH Co., Ltd., Shanghai, China, cat. no. UV-5100). Self-aggregation ability was calculated as follows:</p>
<table class="disp-formula p" id="Equc"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb7b/12374427/fde7106a23b8/d33e568.gif" loading="lazy" id="d33e568" alt="graphic file with name d33e568.gif"></td></tr></table>
<p id="Par18">The selected pathogenic strains (<em>E. coli</em> K88, <em>S. aureus</em> ATCC 49521, and <em>S.</em> typhi) were cultured aerobically in LB medium at 37℃ for 8 h and adjusted to a concentration of 10<sup>8</sup> CFU/mL. Equal volumes (1:1) of KM7 suspension (10<sup>8</sup> CFU/mL) and each pathogen suspension (10<sup>8</sup> CFU/mL) were thoroughly mixed and co-cultured at 37℃. OD<sub>600</sub> measurements of the upper suspension were performed at identical time (5 h) intervals. Co-aggregation efficiency was determined using the formula:</p>
<table class="disp-formula p" id="Equd"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb7b/12374427/870fc4ec5dbe/d33e593.gif" loading="lazy" id="d33e593" alt="graphic file with name d33e593.gif"></td></tr></table>
<p id="Par19">where <em>A</em><sub><em>2</em></sub> is the OD<sub>600</sub> (10<sup>8</sup> CFU/mL) of the KM7 suspension, <em>A</em><sub><em>3</em></sub> is the OD<sub>600</sub> (10<sup>8</sup> CFU/mL) of each pathogen suspension, and <em>A</em><sub><em>4</em></sub> is the OD<sub>600</sub> of the mixed suspension after standing.</p></section></section><section id="Sec22"><h4 class="pmc_sec_title">Antioxidant capacity</h4>
<p id="Par20">The total antioxidant capacity (T-AOC), hydroxyl radical scavenging ability, 1,1-diphenyl-2-picrylhydrazyl (DPPH) radical scavenging ability, reduced glutathione (GSH) content, and superoxide dismutase (SOD) activity of both fermentation supernatant and cell-free extract of <em>L. helveticus</em> KM7 were quantified using commercial assay kits (Solarbio Biotechnology Co., Ltd., Beijing, China, cat. nos. BC1310, BC1320, BC4750, BC1170, BC0170). For T-AOC determination, FeSO<sub>4</sub> (40 µmol/mL) was used as the standard compound; DPPH radical scavenging ability was quantified using vitamin C (10 mg/mL) as the standard reference; Reduced glutathione (10 mg/mL) served as the standard compound for GSH content measurement. Fermentation supernatant was collected by centrifugation of the bacterial culture supernatant (8,000 × g, 15 min, 4℃). To obtain cell-free extract, cell pellets were washed twice with PBS, disrupted via ultrasonication (200 W, 5 s pulse/5 s pause, 15 min) in ice-cold lysis buffer, and clarified by centrifugation (8,000 × g, 20 min, 4℃).</p></section></section><section id="Sec23"><h3 class="pmc_sec_title">Extraction and quantification of crude exopolysaccharides</h3>
<p id="Par21">Crude EPS were extracted following the method of Senanayake [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. with modifications. Bacterial cultures were centrifuged (8,000 × g, 20 min, 4℃), and the supernatant was treated with an equal volume of 10% (w/v) trichloroacetic acid (C<sub>2</sub>HCl<sub>3</sub>O<sub>2</sub>) (Chengdu Kelong Chemical Co., Ltd., Chengdu, China) to precipitate proteins (homogenized at 90 rpm for 30 min). After centrifugation (8,000 × g, 20 min, 4℃), EPS was precipitated from the clarified supernatant using ice-cold ethanol (1:2 v/v, 4 ℃, 24 h), recovered by centrifugation (8,000 × g, 20 min, 4℃), and lyophilized. The crude EPS were dissolved in deionized water to prepare an aqueous solution for further analysis.</p>
<p id="Par22">Quantification of crude EPS was performed using the sulfuric acid-phenol method. A glucose standard curve was prepared by mixing 0–2.0 mL (in 0.2 mL increments) of 100 µg/mL glucose solution with distilled water (final volume: 2 mL), followed by addition of 2 mL 6% phenol (C<sub>6</sub>H<sub>5</sub>OH) (Guangdong Guanghua Sci-Tech Co., Ltd., Shantou, China) and 10 mL concentrated sulfuric acid (H<sub>2</sub>SO<sub>4</sub>) (Tianjin Fengchuan Chemical Reagent Technology Co., Ltd., Tianjin, China). After vortexing and 30 min incubation at 25℃, absorbance was measured at 490 nm. For crude EPS quantification, 2 mL of aqueous solution of crude exopolysaccharides was similarly processed, and concentration was calculated using the standard curve. The strain was cultured statically in MRS broth (1% inoculum, 37℃), and the content of extracellular polysaccharides was measured at 4, 8, 12, 16, 20 and 24 h. Following the method of Tenea [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>] with minor modifications, the strain was statically cultured on MRS broth supplemented with 20% (w/v) each of glucose (BioFroxx GmbH, Heidelberg, Germany, cat. no. 1179GR500), sucrose (Macklin Biochemical Co., Ltd., Shanghai, China, cat. no. S818046), fructose (Macklin Biochemical Co., Ltd., Shanghai, China, cat. no. D809612), and lactose (Macklin Biochemical Co., Ltd., Shanghai, China, cat. no. L812298) (1% inoculum, 37℃, 24 h). The crude EPS content was subsequently quantified.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par23">Data were analyzed using SPSS. Group differences were evaluated by Student’s t-test or one-way ANOVA followed by Tukey’s post hoc test. Results are reported as mean ± SD. Statistical significance was defined as a two-tailed <em>p</em>-value &lt; 0.05.</p></section></section><section id="Sec25"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="Sec26"><h3 class="pmc_sec_title">Genome characteristics of <em>L. helveticus</em> KM7</h3>
<p id="Par24">The genome of <em>L</em>. <em>helveticu</em>s typically consists of a single circular chromosome ~ 2 Mbp in size, with some variation among strains [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. The KM7 genome has a total length of 2,164,024 bp with a GC content of 36.82%, distributed on one chromosome and three plasmids (Fig. <a href="#Fig1" class="usa-link">1</a>A). It encodes 2,192 genes with an average coding sequence length of 856.32 bp (Table <a href="#Tab1" class="usa-link">1</a>). Of these, 2,145 genes are located on the chromosome, while the remaining 30, 11, and 6 genes are found on plasmid A, B, and C, respectively. The total length of coding sequences is 1,877,052 bp, representing 86.74% of the genome. Genomic analysis identified fifteen genomic islands (GIs) in the KM7 genome, fourteen located on the chromosome and one on plasmidA (Table <a href="#MOESM1" class="usa-link">S1</a>). Three CRISPR arrays were also detected, two on the chromosome and one on plasmidA (Table <a href="#MOESM2" class="usa-link">S2</a>). No genes encoding Cas proteins associated with the CRISPR arrays were detected. The absence of these essential components results in a standalone, defective CRISPR structure, compromising formation of a functional CRISPR-Cas defense system. Consequently, this abolishes the capacity of the system for targeted cleavage of exogenous genetic elements, such as plasmids harboring antibiotic resistance or virulence genes [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Furthermore, no prophages were identified.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374427_12866_2025_4286_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb7b/12374427/c5b1633d4f1b/12866_2025_4286_Fig1_HTML.jpg" loading="lazy" id="d33e730" height="596" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome overview of <em>L</em>. <em>helveticus</em> KM7 and functional annotation in different databases. <strong>A</strong> Chromosomal and plasmid genomic circular map. <strong>B</strong> GO function classification. <strong>C</strong> KEGG function classification</p></figcaption></figure><section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Characteristics of the <em>L</em>. <em>helveticus</em> KM7 genome</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Attributes</th>
<th align="left" colspan="1" rowspan="1">Values</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Genome Size (bp)</td>
<td align="left" colspan="1" rowspan="1">2,164,024</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Chromosome</td>
<td align="left" colspan="1" rowspan="1">2,128,667</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PlasmidA</td>
<td align="left" colspan="1" rowspan="1">23,910</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PlasmidB</td>
<td align="left" colspan="1" rowspan="1">6,414</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PlasmidC</td>
<td align="left" colspan="1" rowspan="1">2,192</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">G + C Content (%)</td>
<td align="left" colspan="1" rowspan="1">36.82</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Chromosome</td>
<td align="left" colspan="1" rowspan="1">36.81</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PlasmidA</td>
<td align="left" colspan="1" rowspan="1">38.53</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PlasmidB</td>
<td align="left" colspan="1" rowspan="1">34.49</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PlasmidC</td>
<td align="left" colspan="1" rowspan="1">36.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Coding Gene Number</td>
<td align="left" colspan="1" rowspan="1">2,192</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Coding Gene Average Length (bp)</td>
<td align="left" colspan="1" rowspan="1">856.32</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Coding Gene Total Length (bp)</td>
<td align="left" colspan="1" rowspan="1">1,877,052</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par27">The Gene Ontology (GO) database offers standardized terms and hierarchical classifications to describe gene and protein functions across diverse organisms, and is continuously updated to enhance biological research [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. GO annotation of the KM7 genome revealed that 1,498 genes (68.34%) were successfully annotated (Fig. <a href="#Fig1" class="usa-link">1</a>B). Within the Biological Process category, 1,194 genes were annotated, with translation-related and phosphorylation-related genes being the most abundant (58 genes each). For Cellular Component, 819 genes were annotated, with membrane-associated functions representing the predominant category (250 genes). Furthermore, 2,423 genes were assigned to Molecular Function, among which ATP-binding proteins constituted the largest group (234 genes). These genes may contribute to the adhesive colonization capacity and environmental stress resistance of <em>L</em>. <em>helveticus</em> KM7.</p>
<p id="Par28">KEGG functional annotation enables the mapping of genes, proteins, and other biomolecules to pathways and functional modules within the KEGG (Kyoto Encyclopedia of Genes and Genomes) database, elucidating their roles and interactions in biological processes such as metabolism and signal transduction [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Our KEGG analysis of the <em>L</em>. <em>helveticus</em> KM7 genome successfully annotated 1581 genes (72.13%), with 77, 143, 165, 87, 1,081, and 28 assigned to Cellular Processes, Environmental Information Processing, Genetic Information Processing, Human Diseases, Metabolism, and Organismal Systems, respectively (Fig. <a href="#Fig1" class="usa-link">1</a>C). Notably, metabolism-related genes constituted the predominant category, including 145 genes involved in carbohydrate metabolism. This metabolic profile aligns remarkably well with the characteristic fermentative properties of LAB. The efficient carbohydrate metabolic capacity of strain likely drives substantial lactate production, suggesting that KM7 holds promising potential for dairy fermentation and functional food applications [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>].</p></section><section id="Sec27"><h3 class="pmc_sec_title">Comparative genomic analysis of <em>L. helveticus</em> KM7 and other <em>L. helveticus</em> strains</h3>
<p id="Par29">Average Nucleotide Identity (ANI) analysis compares homologous sequences between pairwise genomes to evaluate phylogenetic relationships at the whole-genome level, with an ANI threshold &gt; 95% typically indicating conspecificity [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Our comparative analysis of KM7 and six other <em>L</em>. <em>helveticus</em> strains from diverse origins revealed that all pairwise comparisons exceeded the 95% threshold, confirming the accurate taxonomic classification of KM7 as <em>L</em>. <em>helveticus</em> at the genome level (Fig. <a href="#Fig2" class="usa-link">2</a>). Notably, KM7 had the highest ANI value with DS3_8 (99.81%), followed by CAUH18 (98.01%) and LBGT 162 (98.26%). This suggests that these strains have a closer evolutionary relationship with KM7 and thus may have similar metabolic capabilities due to the conservation of specific functional genes.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374427_12866_2025_4286_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb7b/12374427/9a6ccbcc6622/12866_2025_4286_Fig2_HTML.jpg" loading="lazy" id="d33e912" height="595" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative genomic analysis of <em>L</em>. <em>helveticus</em> KM7 and other <em>L</em>. <em>helveticus</em> strains. <strong>A</strong> Heatmap of ANI analysis. <strong>B</strong> Collinearity analysis</p></figcaption></figure><p id="Par31">Genomic synteny analysis enables the elucidation of genetic locus alterations caused by various mechanisms, including inversions and translocations between sequenced and reference strains, as well as sequence insertions and deletions [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Comparative synteny analysis of <em>L</em>. <em>helveticu</em>s KM7 with the six other strains mentioned above revealed extensive collinear conservation of most genes between KM7 and both CAUH18 and LZ-R-5, with only localized genomic rearrangements (e.g., inversions, translocations, and positional variations). This conservation pattern suggests that these strains have undergone relatively constrained genomic evolution. In contrast, KM7 had more extensive genomic rearrangements when compared with DS_8, TMW 12,315, LBGT 162, and CGMCC. These differences in genomic architecture may potentially correlate with the strains’ distinct isolation sources, environmental adaptation histories, or genetic backgrounds.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Construction of the pan-core gene set</h3>
<p id="Par32">Core and pan-genome analyses have been widely employed to study bacterial evolution [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. The pan-genome of <em>L</em>. <em>helveticus</em> KM7 and the six additional strains comprises 6,783 genes, including 1,201 core and 5,582 strain-specific genes (Fig. <a href="#Fig3" class="usa-link">3</a>A). The rarefaction curve revealed decreasing core-gene clusters and increasing pan-gene clusters with the addition of more strains (Fig. <a href="#Fig3" class="usa-link">3</a>B), indicating an open genome architecture and high genetic diversity [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Open pan-genomes are typically associated with broad environmental adaptability and frequent horizontal gene transfer (HGT), often facilitated by the acquisition of adaptive genes (e.g., related to carbohydrate metabolism and stress response) from mobile genetic elements such as phages, plasmids, or transposons in heterogeneous niches like dairy products or the host gut [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. This genomic flexibility underlines the functional versatility of <em>L</em>. <em>helveticus</em> in industrial fermentation and probiotic applications. Our phylogenetic analysis of core gene sequences demonstrates a close evolutionary relationship between KM7 and DS_8 (Fig. <a href="#Fig3" class="usa-link">3</a>C), suggesting that they share biological and physiological traits. As a model strain isolated from commercial dietary supplements, DS_8 is likely to have enhanced probiotic properties and metabolic capabilities for host health benefits [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. The genetic similarity between KM7 and DS_8 implies that KM7 may have comparable beneficial traits, warranting further investigation.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374427_12866_2025_4286_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb7b/12374427/e00da24b330d/12866_2025_4286_Fig3_HTML.jpg" loading="lazy" id="d33e994" height="551" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Construction of the pan-core genome of <em>L</em>. <em>helveticus</em> KM7 and six other <em>L</em>. <em>helveticus</em> strains. <strong>A</strong> Core and unique genes of the seven strains. <strong>B</strong> Dilution curves of the core-genes and pan-genes clusters. <strong>C</strong> Phylogenetic tree reconstructed from core genes</p></figcaption></figure></section><section id="Sec29"><h3 class="pmc_sec_title">Safety assessment of <em>L. helveticus</em> KM7</h3>
<section id="Sec30"><h4 class="pmc_sec_title">Antibiotic resistance</h4>
<p id="Par34">To ensure the safety and efficacy of probiotic strains, they must be devoid of antibiotic resistance genes (ARGs) [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. No ARGs meeting the screening criteria of the “Guidelines for Identification and Safety Evaluation of Strains Used in Direct Feeding of Microorganisms and Fermented Products” (Ministry of Agriculture and Rural Affairs of China) were detected in our analysis of the <em>L</em>. <em>helveticus</em> KM7 genome. Although bacteria may acquire ARGs through HGT, mobile genetic elements (MGEs), or chromosomal mutations, KM7 carries only one transposon located on a plasmid, suggesting a low risk of plasmid-mediated ARG dissemination [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. The transporter proteins detected in GIs (e.g., the MFS transporter in GI11 and the ABC transporter in GI15) are membrane-associated but show no apparent linkage to mobile genetic elements (MGEs), suggesting a low risk of ARG dissemination [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Moreover, consistent with the absence of annotated resistance genes in its genome, our prior phenotypic assays confirmed that KM7 is sensitive to cefotaxime, amoxicillin, cephalothin, penicillin G, ampicillin, kanamycin, and vancomycin, as well as moderately sensitive to novobiocin [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. These results demonstrate that <em>L</em>. <em>helveticus</em> KM7 meets probiotic safety standards at both genomic and phenotypic levels, aligning with regulatory requirements for fermented products.</p></section><section id="Sec31"><h4 class="pmc_sec_title">Virulence factors</h4>
<p id="Par35">Virulence factors play crucial roles in bacterial pathogenesis, including host invasion, colonization, and disease progression [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. Our genomic analysis of <em>L</em>. <em>helveticus</em> KM7 against the Virulence Factor Database identified only one putative virulence factor (capsule). Importantly, many genes annotated in virulence databases may participate in routine metabolic processes or environmental adaptation rather than directly mediating pathogenicity [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. For instance, certain bacterial adhesion-related genes in KM7 might solely facilitate intestinal colonization, which is a desirable trait for potential probiotic applications [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. Notably, the capsule virulence factor is commonly present across <em>Lactobacillus</em> species. As surface polysaccharide structures, bacterial capsules in non-pathogenic strains typically contribute to environmental stress resistance and biofilm formation rather than virulence [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. The GIs within the KM7 strain lack detectable canonical virulence genes (e.g., hemolysins, invasins) and harbor only toxin-antitoxin (TA) systems (GI02/GI03). These systems play a significant role in cellular stress responses, enhancing genomic stability and cell viability without exerting detrimental effects on the host or environment [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. Furthermore, transposases (e.g., those in GI04/GI12) are primarily associated with metabolic genes (such as the GI06 fatty acid biosynthesis cluster) and do not carry exogenous pathogenic islands. These findings demonstrate that KM7 lacks definite virulence factors and complies with safety standards for microbial strains used in fermented food production.</p></section><section id="Sec32"><h4 class="pmc_sec_title">Hemolytic activity</h4>
<p id="Par36">Assessment of hemolytic activity is a critical safety evaluation for probiotic strains [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Our hemolysis assays revealed distinct zones of β-hemolysis surrounding <em>S</em>. <em>aureus</em> colonies (positive control), indicating hemolysin production (e.g., α-toxin or staphylolysin) capable of erythrocyte lysis and systemic infection [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] (Fig. <a href="#Fig4" class="usa-link">4</a>A). In contrast, no hemolytic zones were observed around <em>L</em>. <em>helveticus</em> KM7 colonies, demonstrating γ-hemolysis (i.e., a non-hemolytic phenotype). These results confirm that <em>L</em>. <em>helveticus</em> KM7 lacks the capacity to damage host erythrocytes, consistent with the safety profile expected of LAB strains.</p>
<figure class="fig xbox font-sm" id="Fig4"><h5 class="obj_head">Fig. 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374427_12866_2025_4286_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb7b/12374427/6c544627f9aa/12866_2025_4286_Fig4_HTML.jpg" loading="lazy" id="MO4" height="288" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Safety assessment of <em>L. helveticus</em> KM7. <strong>A</strong> Hemolytic activity assay. <strong>B</strong> Production capacity assays for spermine, cadaverine, and putrescine</p></figcaption></figure></section><section id="Sec33"><h4 class="pmc_sec_title">Biogenic amine production</h4>
<p id="Par38">Biogenic amines (BAs) in food are primarily formed through decarboxylation of free amino acids by microbial amino acid decarboxylases, and their excessive accumulation may lead to food poisoning. For instance, in fermented foods such as cheese and sausages, certain LAB and <em>Enterococcus</em> species can pose safety risks by producing BAs [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. Therefore, we evaluated the BA-producing capacity of <em>L</em>. <em>helveticus</em> KM7. According to our results, the arginine decarboxylase, lysine decarboxylase, and ornithine decarboxylase test tubes showed no color change, indicating that KM7 lacks the ability to decarboxylate arginine, lysine, or ornithine, and thus cannot produce spermine, cadaverine, or putrescine (Fig. <a href="#Fig4" class="usa-link">4</a>B). This further shows the suitability of <em>L</em>. <em>helveticus</em> KM7 for applications in the food industry.</p></section></section><section id="Sec34"><h3 class="pmc_sec_title">Probiotic characterization of <em>L. helveticus</em> KM7</h3>
<section id="Sec35"><h4 class="pmc_sec_title">Acid and bile salt tolerance</h4>
<p id="Par39">LAB encounter acidic environments during fermentation, and their acid tolerance is crucial for survival in the host gut or fermented foods [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. Our results demonstrate that <em>L</em>. <em>helveticus</em> KM7 is highly viable under acidic conditions (pH 3 and 4) (Fig. <a href="#Fig5" class="usa-link">5</a>A). To investigate the mechanisms underlying this acid resistance, we further analyzed the strain’s probiotic-related genes (Table <a href="#MOESM3" class="usa-link">S3</a>). The genome of KM7 harbors a complete F-type H<sup>+</sup>/Na<sup>+</sup>-transporting ATPase system (<em>atpA</em>-<em>atpH</em> genes). This proton pump expels excess intracellular H<sup>+</sup> by ATP hydrolysis, thus maintaining cytoplasmic pH homeostasis [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. The integrity of this system is a key determinant of bacterial acid tolerance, as studies have shown that downregulation of the F-type ATPase β subunit significantly reduces the growth and survival of <em>Streptococcus mutans</em> under acidic conditions [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. The genomic integrity of the F-type ATPase system in KM7 ensures its high survival rate under acidic conditions. The arginine deiminase (ADI) pathway, regulated by arginine deiminase (<em>arcA</em>), ornithine carbamoyltransferase (<em>arcB</em>), and carbamate kinase (<em>arcC</em>), is involved in modulating intracellular pH homeostasis in LAB under acid stress [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. When exposed to adverse environmental conditions, LAB can activate the ADI pathway to enhance acid resistance [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. The presence of partial ADI pathway genes (<em>arcA</em>, <em>arcC</em>) in the KM7 genome suggests its potential reliance on ADI pathway activation for acid stress tolerance. The synergistic action of the F-type ATPase system and the ADI pathway likely underpins the robust acid stress response in <em>L</em>. <em>helveticus</em> KM7. The KM7 genome also encodes additional genes supporting acid adaptation. Key glycolytic enzymes (<em>gapA</em>, <em>pgi</em>, <em>tpiA</em>, <em>pyk</em>) ensure sustained ATP generation, facilitating proton extrusion via the F-type ATPase [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. Furthermore, the DNA repair function mediated by <em>uvrA</em> and cell wall synthesis regulated by <em>ddl</em> contribute to maintaining structural integrity [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>, <a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. Notably, lactate dehydrogenase (<em>ldh</em>) activity contributes to environmental acidification, which could potentially enhance competitive fitness [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. These molecular adaptations not only enhance its survival in harsh environments but also support its potential as a functional probiotic for industrial applications.</p>
<figure class="fig xbox font-sm" id="Fig5"><h5 class="obj_head">Fig. 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374427_12866_2025_4286_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb7b/12374427/60a5efb8c8d0/12866_2025_4286_Fig5_HTML.jpg" loading="lazy" id="MO5" height="629" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Probiotic characterization of <em>L</em>. <em>helveticu</em>s KM7. <strong>A</strong> Acid tolerance. <strong>B</strong> Bile salt tolerance. <strong>C</strong> Antioxidant activity of fermentation supernatant and cell-free extract. Data are presented as mean ± SD (<em>n</em> = 3) and *** indicates <em>P &lt;</em> 0.001</p></figcaption></figure><p id="Par41">Bile salt tolerance represents a fundamental prerequisite for probiotic survival within the host gastrointestinal tract. According to our results, <em>L</em>. <em>helveticus</em> KM7 had good survival rates when exposed to bile salts (Fig. <a href="#Fig5" class="usa-link">5</a>B). We identified the bile salt hydrolase gene <em>cbh</em> in the genome of KM7, suggesting that the strain may directly degrade bile salts enzymatically (Table <a href="#MOESM3" class="usa-link">S3</a>). Bile salt hydrolase (BSH) activity is a central mechanism of bile salt tolerance in <em>L</em>. <em>helveticus</em>, enhancing its survival in the intestine by reducing bile salt damage to cell membranes and proteins [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. The BSH activity assay showed that strain KM7 has the ability to produce bile salt hydrolase (BSH), and its BSH activity level gradually increased with the advancement of the growth phase under the experimental conditions (Fig. <a href="#MOESM1" class="usa-link">S1</a>). This result suggests that strain KM7 is able to continuously express and accumulate BSH enzyme during growth. Our results are consistent with previous studies that BSH activity correlates with bile salt resistance in <em>L</em>. <em>helveticus</em>, and that its expression level and substrate specificity may lead to differences in tolerance among strains [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. The <em>cbh</em> is both present in the KM7 genome, so this strain may expand bile salt hydrolysis capacity through synergistic effects of the two genes to increase the adaptation to the complex bile salt environment. It is worth noting that its adaptability to this environment not only stems from direct detoxification mechanisms, but is also closely related to its ability to maintain the structural integrity of the cell membrane and wall [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. Concerning the regulation of cell membrane stability, genomic analysis of KM7 revealed that the cyclopropane fatty acid synthase encoded by the <em>cfa</em> gene may play a key role [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. This enzyme can maintain the stability of the cell membrane through fatty acid cyclopropanation, thus limiting the permeability damage of bile salt molecules. GlcN-6-P deaminase encoded by <em>nagB</em> may affect peptidoglycan synthesis by regulating the N-acetylglucosamine (NAG) metabolic network [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. We hypothesize that KM7 may form a physical barrier against bile salts by dynamically regulating the rate of cell wall synthesis and repair. Moreover, peptidases (<em>pepF</em>, <em>pepO</em>, <em>pepP</em>) contribute to amino acid acquisition for the synthesis of repair proteins, while pyrophosphatase (<em>ppaC</em>) could potentially provide energy for these processes [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>, <a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. Notably, the DNA-binding protein Dps counteracts bile-induced oxidative DNA damage [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. Thus, the bile salt tolerance in <em>L</em>. <em>helveticus</em> KM7 is not dependent on a single gene function but is the result of synergistic effects of multiple pathways.</p></section><section id="Sec36"><h4 class="pmc_sec_title">Adhesion and aggregation</h4>
<p id="Par42">The hydrophobicity and aggregation capacity of bacterial strains are closely associated with their adhesion properties, serving as crucial prerequisites for probiotic functionality. Our investigations revealed that <em>L</em>. <em>helveticus</em> KM7 has high surface hydrophobicity and auto-aggregation abilities (Table <a href="#Tab2" class="usa-link">2</a>). These characteristics imply the presence of abundant hydrophobic surface molecules, such as lipoteichoic acids (LTAs) and proteins, which may be attributed to genes such as <em>ltaS</em> (encoding lipoteichoic acid synthase) and <em>eno</em> (encoding enolase) we identified in the KM7 genome [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. As a key component of Gram-positive bacterial cell walls, LTAs enhance bacterial adhesion through interactions with host cell surfaces [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. Enolase, a multifunctional surface protein, not only participates in glycolytic metabolism but also functions as an adhesin by directly binding to host extracellular matrix proteins, thereby mediating cellular adhesion [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. These molecular features collectively account for the observed auto-aggregation phenotype and potential intestinal epithelial adhesion advantages of <em>L</em>. <em>helveticus</em> KM7. Furthermore, we identified <em>scpA</em> and <em>scpB</em> in the KM7 genome. While these genes encode fibronectin-binding proteins known to promote bacterial-host adhesion in certain streptococci (typically associated with pathogenic processes), their presence in this probiotic strain may enhance adhesion capacity without conferring pathogenicity [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. Notably, the co-aggregation ability of probiotic strains further facilitates biofilm formation in the intestinal tract, thereby enhancing their inhibitory effects against pathogenic bacteria [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. Studies have shown that various <em>Lactobacillus</em> species including <em>L. crispatus</em>, <em>L. gasseri</em>, <em>L. reuteri</em>, and <em>L</em>. <em>coryniformis</em> can co-aggregate with pathogenic bacteria [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>, <a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. Here, we found that <em>L</em>. <em>helveticus</em> KM7 was able to co-aggregate (at a rate &gt; 30%) with <em>S</em>. <em>aureus</em>, <em>E</em>. <em>coli</em> K88, and <em>S</em>. typhi, suggesting that it can inhibit pathogenic bacterial colonization through competitive binding [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>].</p>
<section class="tw xbox font-sm" id="Tab2"><h5 class="obj_head">Table 2.</h5>
<div class="caption p"><p>Adhesion and aggregation ability of <em>L</em>. <em>helveticus</em> KM7</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Item</th>
<th align="left" colspan="1" rowspan="1">Value (%)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Hydrophobicity</td>
<td align="left" colspan="1" rowspan="1">53.67 ± 1.28</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Self-coagulation</td>
<td align="left" colspan="1" rowspan="1">78.27 ± 2.46</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Co-aggregation with <em>S. aureus</em> ATCC 49521</td>
<td align="left" colspan="1" rowspan="1">49.40 ± 1.42</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Co-aggregation with <em>E. coli</em> K88</td>
<td align="left" colspan="1" rowspan="1">32.05 ± 1.95</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Co-aggregation with S. typhi</td>
<td align="left" colspan="1" rowspan="1">36.32 ± 1.15</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec37"><h4 class="pmc_sec_title">Antioxidant capacity</h4>
<p id="Par44">Maintaining effective metabolic and oxidative homeostasis can counteract aging-related diseases [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]. Probiotics have significant potential as antioxidants to preserve intestinal redox balance [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. Here, we have evaluated the antioxidant capacity of <em>L</em>. <em>helveticus</em> KM7. Our results reveal that both fermentation supernatant and cell-free extract of KM7 have antioxidant (including SOD and GSH) activity, as well as scavenging effects on DPPH and hydroxyl radicals (Fig. <a href="#Fig5" class="usa-link">5</a>C). This enhanced antioxidant capability may be attributed to the synergistic effects of multiple pathways encoded by genes such as <em>trxB</em>, the <em>suf</em> operon, and <em>msrA/B</em> (Table <a href="#MOESM3" class="usa-link">S3</a>). The <em>trxB</em> gene encodes thioredoxin reductase, a core component of the thioredoxin system. This enzyme catalyzes NADPH-dependent reduction reactions to continuously regenerate the reduced form of thioredoxin (Trx-(SH)<sub>₂</sub>), thereby facilitating electron transfer to peroxiredoxins and other antioxidant enzymes to maintain cellular redox homeostasis [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. The <em>suf</em> operon genes encode proteins involved in the synthesis and repair of Fe-S clusters, which are essential for the function of key antioxidant enzymes such as thioredoxin and glutathione reductase [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. The Suf system enhances cellular antioxidant capacity by modulating sulfur metabolism, thereby conferring protection against oxidative stress-induced damage [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. The <em>msrA/B</em> genes encode the methionine sulfoxide reductase system, a critical mechanism for repairing oxidative protein damage. This enzyme system specifically reverses methionine sulfoxidation caused by oxidative stress, restoring the function of damaged proteins [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. Notably, in our experiments, the fermentation supernatant had higher antioxidant capacity than the cell-free extract. LAB can secrete extracellular vesicles to deliver proteins into the surrounding environment [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. <em>L</em>. <em>helveticus</em> KM7 may employ a similar mechanism, potentially contributing to the stronger free radical scavenging capacity of its extracellular products. The coordinated action of these genetic systems suggests that KM7 not only maintains its own oxidative homeostasis through enzymatic antioxidant pathways (SOD/GSH) but may also regulate extracellular redox balance via the secretion of bioactive compounds.</p></section><section id="Sec38"><h4 class="pmc_sec_title">Antibacterial activity and potential bacteriocin genes</h4>
<p id="Par45">Antimicrobial activity is a key mechanism by which probiotics eliminate or inhibit harmful intestinal microorganisms, either through the production of antimicrobial peptides or by lowering environmental pH to kill pathogens [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]. In our previous studies, we found that <em>L</em>. <em>helveticus</em> KM7 exhibits antimicrobial activity against <em>E</em>. <em>coli</em>, <em>Shigella flexneri</em>, and <em>Salmonella typhimurium</em>, surpassing that of the reference strain LGG [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Bacteriocins are one of the natural defense mechanisms bacteria use to compete with other microbes in shared environments. To identify the bacteriocins responsible for the antimicrobial effects of KM7, we performed a genome-wide bacteriocin gene cluster analysis using BAGEL4 (Fig. <a href="#Fig6" class="usa-link">6</a>). Our results revealed three putative gene clusters (Helveticin-J-1, -J-2, and -J-3) potentially involved in the biosynthesis of bacteriocin Helveticin J, with core peptide sequence similarities of 90.980%, 42.066%, and 39.481%, respectively. Helveticin J is a Class III bacteriocin and likely exerts its antimicrobial effects by disrupting bacterial cell walls and increasing membrane permeability [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]. NX371, a novel bacteriocin with high sequence similarity to Helveticin J, exhibits potent activity against both Gram-positive and Gram-negative bacteria [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. In our study, Helveticin J-1 had the highest similarity to the known Helveticin J, suggesting it may be the primary contributor to the antimicrobial activity of KM7. Notably, sequence variations within bacteriocin families may influence target-binding specificity or membrane penetration efficiency, thereby expanding or modulating their antimicrobial spectrum [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>]. The lower sequence similarity of Helveticin J-2 and J-3 suggests potential functional divergence, though further in vitro expression and mechanistic studies are required to validate this. Furthermore, we identified a gene cluster associated with the biosynthesis of the Class II bacteriocin Enterolysin A (EnlA) in the KM7 genome, with a core peptide similarity of 60.248%, indicating possible production of an EnlA-like bacteriocin. EnlA, a peptidase-type bacteriocin, induces bacterial lysis by hydrolyzing the peptidoglycan layer of Gram-positive cell walls [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>]. The synergistic action of multiple bacteriocins may underlie the broad-spectrum activity of <em>L</em>. <em>helveticus</em> KM7 against pathogenic bacteria.</p>
<figure class="fig xbox font-sm" id="Fig6"><h5 class="obj_head">Fig. 6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374427_12866_2025_4286_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb7b/12374427/79b8f4b4b102/12866_2025_4286_Fig6_HTML.jpg" loading="lazy" id="MO6" height="362" width="669" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Potential antimicrobial gene clusters of <em>L</em>. <em>helveticus</em> KM7</p></figcaption></figure></section><section id="Sec39"><h4 class="pmc_sec_title">Carbohydrate-active enzyme analysis</h4>
<p id="Par47">Through comprehensive analysis of carbohydrate-active enzymes (CAZymes) in the <em>L. helveticus</em> KM7 genome, we elucidated its functional characteristics in carbohydrate metabolism. Among the 71 identified CAZyme genes, glycoside hydrolases (GHs, 33 genes) and glycosyl transferases (GTs, 23 genes) predominated, indicating the robust capacity strain for decomposing complex polysaccharides and synthesizing functional oligosaccharides or exopolysaccharides (Fig. <a href="#Fig7" class="usa-link">7</a>). The diversity of GH families (particularly GH13) likely reflects metabolic adaptation to plant-derived polysaccharides and host mucopolysaccharides in the intestinal environment. GH13 family α-amylases may provide carbon sources by degrading α-glucans while competitively limiting carbohydrate utilization by pathogens [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]. Furthermore, GT families (including GT2 and GT4) potentially contribute to cell wall biosynthesis or prebiotic-like molecule production, thereby enhancing intestinal colonization capacity [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>]. Notably, the identification of carbohydrate esterases (CEs, 7 genes) and auxiliary activities (AAs, 8 genes) suggests that KM7 may facilitate degradation of recalcitrant polysaccharides (e.g., chitin or lignocellulose) through deacetylation or redox reactions that complement GH activity [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>]. This metabolic versatility toward diverse substrates positions <em>L</em>. <em>helveticus</em> KM7 as a promising candidate for probiotic formulations or starter cultures, with potential applications in dairy texture modification and dietary fiber-enriched food development.</p>
<figure class="fig xbox font-sm" id="Fig7"><h5 class="obj_head">Fig. 7.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb7b/12374427/2a79ee461fae/12866_2025_4286_Fig7_HTML.jpg" loading="lazy" id="MO7" height="652" width="650" alt="Fig. 7"></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Classification of predicted carbohydrate-active enzymes (CAZy) from <em>L</em>. <em>helveticus</em> KM7. AA: Auxiliary Activities; CE: Carbohydrate Esterases; GH: Glycoside Hydrolases; GT: Glycosyl Transferases</p></figcaption></figure></section></section><section id="Sec40"><h3 class="pmc_sec_title">EPS production capacity of <em>L. helveticus</em> KM7</h3>
<p id="Par49">Genome annotation revealed that KM7 possesses complete gene clusters for sugar metabolism (<em>rfb</em>, <em>gal</em>, and <em>glm</em> clusters) and EPS biosynthesis (<em>epsA</em>/<em>B</em>/<em>F</em>/<em>H</em>/<em>J</em>), indicating genetic basis for complex EPS production (Table <a href="#Tab3" class="usa-link">3</a>). Metabolic network predictions demonstrate ability of KM7 to convert monosaccharides (e.g., glucose, galactose) into diverse nucleotide sugars through four pathways, generating four key EPS precursors: UDP-glucose, UDP-galactose, dTDP-L-rhamnose, and UDP-N-acetyl-D-mannosamine (Fig. <a href="#Fig8" class="usa-link">8</a>A). Unlike most lactic acid bacteria that utilize the wzx/wzy-dependent pathway, KM7 lacks <em>wzx</em>/<em>wzy</em> genes but retains complete ABC transporter-dependent genes (e<em>psA/B</em>), suggesting its EPS secretion primarily relies on the ABC transporter pathway [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. Therefore, we hypothesized that the extracellular polysaccharides produced by KM7 are mainly capsular polysaccharides (CPS). CPS production likely contributes to multifaceted functional profile of KM7, including demonstrated antimicrobial effects through interference with pathogen adhesion and biofilm formation, as well as significant antioxidant capacity via both direct free radical scavenging and modulation of antioxidant enzyme activity [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>, <a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>]. Furthermore, these CPS molecules also improve the functional properties of dairy products by increasing water retention and viscosity, which enhances texture and mouthfeel [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>, <a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>]. These properties position KM7 as a potential candidate strain for functional food development.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Genes related to EPS production of <em>L</em>. <em>helveticus</em> KM7</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Gene locus</th>
<th align="left" colspan="1" rowspan="1">Gene name</th>
<th align="left" colspan="1" rowspan="1">Definition</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">gene0693</td>
<td align="left" colspan="1" rowspan="1">
<em>glk</em>
</td>
<td align="left" colspan="1" rowspan="1">glucokinase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene1548</td>
<td align="left" colspan="1" rowspan="1">
<em>pgm</em>
</td>
<td align="left" colspan="1" rowspan="1">phosphoglucomutase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene1616, gene1840</td>
<td align="left" colspan="1" rowspan="1">
<em>galU</em>
</td>
<td align="left" colspan="1" rowspan="1">UTP–glucose-1-phosphate uridylyltransferase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene1874</td>
<td align="left" colspan="1" rowspan="1">
<em>rfbA</em>
</td>
<td align="left" colspan="1" rowspan="1">glucose-1-phosphate thymidylyltransferase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene1875</td>
<td align="left" colspan="1" rowspan="1">
<em>rfbB</em>
</td>
<td align="left" colspan="1" rowspan="1">dTDP-glucose 4,6-dehydratase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene1873</td>
<td align="left" colspan="1" rowspan="1">
<em>rfbC</em>
</td>
<td align="left" colspan="1" rowspan="1">dTDP-4-dehydrorhamnose 3,5-epimerase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene1854, gene1872</td>
<td align="left" colspan="1" rowspan="1">
<em>rfbD</em>
</td>
<td align="left" colspan="1" rowspan="1">dTDP-4-dehydrorhamnose reductase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene0198, gene0734, gene1234</td>
<td align="left" colspan="1" rowspan="1">
<em>galM</em>
</td>
<td align="left" colspan="1" rowspan="1">aldose 1-epimerase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene0732</td>
<td align="left" colspan="1" rowspan="1">
<em>galK</em>
</td>
<td align="left" colspan="1" rowspan="1">galactokinase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene0733</td>
<td align="left" colspan="1" rowspan="1">
<em>galT</em>
</td>
<td align="left" colspan="1" rowspan="1">UDPglucose–hexose-1-phosphate uridylyltransferase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene0708</td>
<td align="left" colspan="1" rowspan="1">
<em>galE</em>
</td>
<td align="left" colspan="1" rowspan="1">UDP-glucose 4-epimerase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene0709, gene0710</td>
<td align="left" colspan="1" rowspan="1">
<em>lacZ</em>
</td>
<td align="left" colspan="1" rowspan="1">beta-galactosidase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene0408</td>
<td align="left" colspan="1" rowspan="1">
<em>sacA</em>
</td>
<td align="left" colspan="1" rowspan="1">beta-fructofuranosidase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene1485</td>
<td align="left" colspan="1" rowspan="1">
<em>pgi</em>
</td>
<td align="left" colspan="1" rowspan="1">glucose-6-phosphate isomerase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene1772</td>
<td align="left" colspan="1" rowspan="1">
<em>glmS</em>
</td>
<td align="left" colspan="1" rowspan="1">glutamine—fructose-6-phosphate transaminase (isomerizing)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene1516</td>
<td align="left" colspan="1" rowspan="1">
<em>glmM</em>
</td>
<td align="left" colspan="1" rowspan="1">phosphoglucosamine mutase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene0233</td>
<td align="left" colspan="1" rowspan="1">
<em>glmU</em>
</td>
<td align="left" colspan="1" rowspan="1">bifunctional UDP-N-acetylglucosamine pyrophosphorylase/glucosamine-1-phosphate N-acetyltransferase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene1621</td>
<td align="left" colspan="1" rowspan="1">
<em>wecB</em>
</td>
<td align="left" colspan="1" rowspan="1">UDP-N-acetylglucosamine 2-epimerase (non-hydrolysing)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene1900</td>
<td align="left" colspan="1" rowspan="1">
<em>epsA</em>
</td>
<td align="left" colspan="1" rowspan="1">protein tyrosine kinase modulator</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene1899</td>
<td align="left" colspan="1" rowspan="1">
<em>epsB</em>
</td>
<td align="left" colspan="1" rowspan="1">protein-tyrosine kinase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene1893</td>
<td align="left" colspan="1" rowspan="1">
<em>epsF</em>
</td>
<td align="left" colspan="1" rowspan="1">glycosyltransferase EpsF</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene0137</td>
<td align="left" colspan="1" rowspan="1">
<em>epsH</em>
</td>
<td align="left" colspan="1" rowspan="1">glycosyltransferase EpsH</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gene1892</td>
<td align="left" colspan="1" rowspan="1">
<em>epsJ</em>
</td>
<td align="left" colspan="1" rowspan="1">glycosyltransferase EpsJ</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374427_12866_2025_4286_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb7b/12374427/2d41d46ad51f/12866_2025_4286_Fig8_HTML.jpg" loading="lazy" id="MO8" height="595" width="669" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Predicted EPS metabolic network and production capacity of KM7. <strong>A</strong> Predicted EPS metabolic network. <strong>B</strong> Crude EPS yield by KM7 at different time points. <strong>C</strong> Crude EPS yield of KM7 supplemented with various carbon sources. Different letters indicate significant differences between age stages (<em>P</em> &lt; 0.05)</p></figcaption></figure><p id="Par52">KM7 exhibited a short lag phase, entering the exponential growth phase after 4 h of cultivation with concomitant EPS production (Fig. <a href="#Fig8" class="usa-link">8</a>B). The EPS accumulation reached a plateau (1.08 mg/mL crude EPS) at 24 h when the culture transitioned into the stationary phase, a yield superior to several documented <em>L</em>. <em>helveticus</em> strains and other lactic acid bacteria [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>, <a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>]. This high productivity may be attributed to strain-specific glycosyltransferases enhancing monosaccharide polymerization efficiency, and gene dosage effects (e.g., multi-copy <em>galU</em>) augmenting sugar nucleotide biosynthesis. The high-EPS-producing phenotype likely underpins enhanced adhesion of capacity and antioxidant activity. Carbon source utilization significantly influenced EPS yield, with glucose supporting the highest production, followed by sucrose, lactose, and fructose (Fig. <a href="#Fig8" class="usa-link">8</a>C). This hierarchy aligns with genomic predictions of carbon transport and metabolic pathways. Notably, KM7 demonstrated reduced EPS biosynthesis efficiency from sucrose relative to glucose, along with compromised fructose metabolism. Genomic analysis revealed the absence of a fructose-6-phosphate isomerase (<em>gpi</em>) gene, potentially explaining the fructose assimilation bottleneck [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>]. Given these findings, glucose supplementation represents the most effective strategy for maximizing EPS production in KM7. Furthermore, its efficient lactose metabolism supports its potential application in dairy fermentations.</p></section></section><section id="Sec41"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par53">In summary, we characterized the safety, probiotic properties and EPS-producing capacity of <em>L</em>. <em>helveticus</em> KM7 through genomic analysis and in vitro assays. The absence of identifiable virulence factor genes, antibiotic resistance genes, hemolytic activity, and biogenic amine production confirmed the safety of this strain. Its bile salt and acid tolerance, strong adhesion capacity, notable antioxidant activity, and the presence of genes potentially encoding bacteriocin and diverse carbohydrate-active enzymes endowed it with outstanding probiotic attributes. Furthermore, KM7 has an excellent EPS-producing capacity. Collectively, our findings demonstrate that <em>L</em>. <em>helveticus</em> KM7 is a safe and highly promising probiotic candidate with substantial potential for food applications.</p></section><section id="Sec42"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12374427/bin/12866_2025_4286_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (64KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12374427/bin/12866_2025_4286_MOESM2_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (252.4KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12374427/bin/12866_2025_4286_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3.</a><sup> (11.5KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12374427/bin/12866_2025_4286_MOESM4_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4.</a><sup> (20.6KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Not applicable.</p></section><section id="notes1"><h2 class="pmc_sec_title">Authors' contributions</h2>
<p>C.L.: Investigation, Data Curation, Formal analysis, Visualization, Writing - Original Draft. L.X.: Investigation, Data Curation, Formal analysis, Visualization. S.C.: Investigation, Data Curation, Formal analysis. S.W.: Data Curation, Formal analysis. M.L.: Data Curation, Formal analysis. K.D.: Conceptualization, Methodology, Supervision, Funding acquisition. Q.L.: Conceptualization, Methodology, Supervision, Writing - review &amp; editing, Funding acquisition. Z.C.: Conceptualization, Methodology, Funding acquisition, Project administration, Supervision.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This project was supported by the Yunnan Provincial Middle-Young Academic and Technical Leader Candidate (202305AC160040), Young Talent of Yunnan Xingdian Support Project for High Level Talents (YNWR-QNBJ-2018-137), and the China Agriculture Research System of MOF and MARA (CARS-44-KXJ13).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The KM7 genome data were deposited at the NCBI database, with the BioProject accession number: PRJNA1271779.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar200"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par54">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par55">Not applicable.</p></section><section id="FPar4"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par56">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Chen Liu and Le Xu contributed contributed equally to this study.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Qiuye Lin, Email: linqiuye@ymu.edu.cn.</p>
<p>Zhenhui Cao, Email: caozhenhui@ynau.edu.cn.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Hill C, Guarner F, Reid G, Gibson GR, Merenstein DJ, Pot B, Morelli L, Canani RB, Flint HJ, Salminen S. Expert consensus document: the international scientific association for probiotics and prebiotics consensus statement on the scope and appropriate use of the term probiotic. Nat Rev Gastroenterol Hepatol. 2014;11(8):506–14.
</cite> [<a href="https://doi.org/10.1038/nrgastro.2014.66" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24912386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hill%20C,%20Guarner%20F,%20Reid%20G,%20Gibson%20GR,%20Merenstein%20DJ,%20Pot%20B,%20Morelli%20L,%20Canani%20RB,%20Flint%20HJ,%20Salminen%20S.%20Expert%20consensus%20document:%20the%20international%20scientific%20association%20for%20probiotics%20and%20prebiotics%20consensus%20statement%20on%20the%20scope%20and%20appropriate%20use%20of%20the%20term%20probiotic.%20Nat%20Rev%20Gastroenterol%20Hepatol.%202014;11(8):506%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Skrzypczak K, Gustaw W, Waśko A. Health-promoting properties exhibited by <em>Lactobacillus helveticus</em> strains. Acta Biochim Pol. 2015;62(4):713–20.
</cite> [<a href="https://doi.org/10.18388/abp.2015_1116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26601325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Skrzypczak%20K,%20Gustaw%20W,%20Wa%C5%9Bko%20A.%20Health-promoting%20properties%20exhibited%20by%20Lactobacillus%20helveticus%20strains.%20Acta%20Biochim%20Pol.%202015;62(4):713%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Cremonesi. Genome sequence and analysis of <em>Lactobacillus helveticus</em>. Front Microbiol. 2012;3:435. </cite> [<a href="https://doi.org/10.3389/fmicb.2012.00435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3542869/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23335916/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Valentina T, Simone G. Health-promoting properties of <em>Lactobacillus helveticus</em>. Front Microbiol. 2012;3:392.
</cite> [<a href="https://doi.org/10.3389/fmicb.2012.00392" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3500876/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23181058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Valentina%20T,%20Simone%20G.%20Health-promoting%20properties%20of%20Lactobacillus%20helveticus.%20Front%20Microbiol.%202012;3:392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Zago M, Massimiliano L, Bonvini B, Penna G, Giraffa G, Rescigno M. Functional characterization and immunomodulatory properties of <em>Lactobacillus helveticus</em> strains isolated from Italian hard cheeses. PLoS One. 2021;16(1):e0245903.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0245903" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7833162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33493208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zago%20M,%20Massimiliano%20L,%20Bonvini%20B,%20Penna%20G,%20Giraffa%20G,%20Rescigno%20M.%20Functional%20characterization%20and%20immunomodulatory%20properties%20of%20Lactobacillus%20helveticus%20strains%20isolated%20from%20Italian%20hard%20cheeses.%20PLoS%20One.%202021;16(1):e0245903." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Gao K, Chen CL, Ke XQ, Yu YX, Chen S, Liu GC, Wang HF, Li YJ. Ingestion of <em>Lactobacillus helveticus</em> WHH1889 improves depressive and anxiety symptoms induced by chronic unpredictable mild stress in mice. Benef Microbes. 2022;13(6):473–88.
</cite> [<a href="https://doi.org/10.3920/BM2022.0052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36377577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao%20K,%20Chen%20CL,%20Ke%20XQ,%20Yu%20YX,%20Chen%20S,%20Liu%20GC,%20Wang%20HF,%20Li%20YJ.%20Ingestion%20of%20Lactobacillus%20helveticus%20WHH1889%20improves%20depressive%20and%20anxiety%20symptoms%20induced%20by%20chronic%20unpredictable%20mild%20stress%20in%20mice.%20Benef%20Microbes.%202022;13(6):473%E2%80%9388." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Mutoh N, Kakiuchi I, Kato K, Xu C, Iwabuchi N, Ayukawa M, Kiyosawa K, Igarashi K, Tanaka M, Nakamura M. Heat-killed <em>L. helveticus</em> enhances positive mood states: a randomized, double-blind, placebo-controlled study. Brain Sci. 2023;13(6):973. 
</cite> [<a href="https://doi.org/10.3390/brainsci13060973" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10296070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37371451/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mutoh%20N,%20Kakiuchi%20I,%20Kato%20K,%20Xu%20C,%20Iwabuchi%20N,%20Ayukawa%20M,%20Kiyosawa%20K,%20Igarashi%20K,%20Tanaka%20M,%20Nakamura%20M.%20Heat-killed%20L.%20helveticus%20enhances%20positive%20mood%20states:%20a%20randomized,%20double-blind,%20placebo-controlled%20study.%20Brain%20Sci.%202023;13(6):973." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Lv J, Lang G, Wang Q, Zhao W, Shi D, Zhou Z, Shen Y, Xia H, Han S, Li L. <em>Lactobacillus helveticus</em> attenuates alcoholic liver injury via regulation of gut microecology in mice. Microb Biotechnol. 2024;17(10): e70016.
</cite> [<a href="https://doi.org/10.1111/1751-7915.70016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11492535/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39431804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lv%20J,%20Lang%20G,%20Wang%20Q,%20Zhao%20W,%20Shi%20D,%20Zhou%20Z,%20Shen%20Y,%20Xia%20H,%20Han%20S,%20Li%20L.%20Lactobacillus%20helveticus%20attenuates%20alcoholic%20liver%20injury%20via%20regulation%20of%20gut%20microecology%20in%20mice.%20Microb%20Biotechnol.%202024;17(10):%20e70016." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Zhao X, Lin T, Jiang W, Lin Y, Xiao L, Tian Y, Ma K, Zhang C, Ji F, Mahsa GC, et al. <em>Lactobacillus helveticus</em> LZ-R-5 ameliorates DSS-induced colitis in mice by modulating gut microbiota and enhancing intestinal barrier function. J Agric Food Chem. 2025;73(1):464–77.
</cite> [<a href="https://doi.org/10.1021/acs.jafc.4c07895" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39688942/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20X,%20Lin%20T,%20Jiang%20W,%20Lin%20Y,%20Xiao%20L,%20Tian%20Y,%20Ma%20K,%20Zhang%20C,%20Ji%20F,%20Mahsa%20GC,%20et%20al.%20Lactobacillus%20helveticus%20LZ-R-5%20ameliorates%20DSS-induced%20colitis%20in%20mice%20by%20modulating%20gut%20microbiota%20and%20enhancing%20intestinal%20barrier%20function.%20J%20Agric%20Food%20Chem.%202025;73(1):464%E2%80%9377." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Sadat-Mekmene L, Genay M, Atlan D, Lortal S, Gagnaire V. Original features of cell-envelope proteinases of <em>Lactobacillus helveticus</em>. a review. Int J Food Microbiol. 2011;146(1):1–13.
</cite> [<a href="https://doi.org/10.1016/j.ijfoodmicro.2011.01.039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21354644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sadat-Mekmene%20L,%20Genay%20M,%20Atlan%20D,%20Lortal%20S,%20Gagnaire%20V.%20Original%20features%20of%20cell-envelope%20proteinases%20of%20Lactobacillus%20helveticus.%20a%20review.%20Int%20J%20Food%20Microbiol.%202011;146(1):1%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Li W, Ji J, Rui X, Yu J, Tang W, Chen X, Jiang M, Dong M. Production of exopolysaccharides by <em>Lactobacillus helveticus</em> MB2-1 and its functional characteristics in vitro. LWT. 2014;59(2):732–9.</cite> [<a href="https://scholar.google.com/scholar_lookup?Li%20W,%20Ji%20J,%20Rui%20X,%20Yu%20J,%20Tang%20W,%20Chen%20X,%20Jiang%20M,%20Dong%20M.%20Production%20of%20exopolysaccharides%20by%20Lactobacillus%20helveticus%20MB2-1%20and%20its%20functional%20characteristics%20in%20vitro.%20LWT.%202014;59(2):732%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Broadbent JR, Cai H, Larsen RL, Hughes JE, Welker DL, Carvalho VGD, Tompkins TA, Ard Y, Vogensen F, Lorentiis AD. Genetic diversity in proteolytic enzymes and amino acid metabolism among <em>Lactobacillus helveticus</em> strains. J Dairy Sci. 2011;94(9):4313–28.
</cite> [<a href="https://doi.org/10.3168/jds.2010-4068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21854904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Broadbent%20JR,%20Cai%20H,%20Larsen%20RL,%20Hughes%20JE,%20Welker%20DL,%20Carvalho%20VGD,%20Tompkins%20TA,%20Ard%20Y,%20Vogensen%20F,%20Lorentiis%20AD.%20Genetic%20diversity%20in%20proteolytic%20enzymes%20and%20amino%20acid%20metabolism%20among%20Lactobacillus%20helveticus%20strains.%20J%20Dairy%20Sci.%202011;94(9):4313%E2%80%9328." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>O’Sullivan DJ, Mcsweeney PLH, Cotter PD, Giblin L, Sheehan JJ. Compromised <em>Lactobacillus helveticus</em> starter activity in the presence of facultative heterofermentative <em>Lactobacillus casei</em> DPC6987 results in atypical eye formation in Swiss-type cheese. J Dairy Sci. 2016;99(4):2625–40.
</cite> [<a href="https://doi.org/10.3168/jds.2015-10503" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26805985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?O%E2%80%99Sullivan%20DJ,%20Mcsweeney%20PLH,%20Cotter%20PD,%20Giblin%20L,%20Sheehan%20JJ.%20Compromised%20Lactobacillus%20helveticus%20starter%20activity%20in%20the%20presence%20of%20facultative%20heterofermentative%20Lactobacillus%20casei%20DPC6987%20results%20in%20atypical%20eye%20formation%20in%20Swiss-type%20cheese.%20J%20Dairy%20Sci.%202016;99(4):2625%E2%80%9340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Merenstein D, Pot B, Leyer G, Ouwehand A, Preidis GA, Elkins CA, Hill C, Lewis ZT, Shane A, Zmora N. Emerging issues in probiotic safety: 2023 perspectives. Gut Microbes. 2023;15(1):2185034.
</cite> [<a href="https://doi.org/10.1080/19490976.2023.2185034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10026873/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36919522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Merenstein%20D,%20Pot%20B,%20Leyer%20G,%20Ouwehand%20A,%20Preidis%20GA,%20Elkins%20CA,%20Hill%20C,%20Lewis%20ZT,%20Shane%20A,%20Zmora%20N.%20Emerging%20issues%20in%20probiotic%20safety:%202023%20perspectives.%20Gut%20Microbes.%202023;15(1):2185034." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Wu JJ, Zhou Qy L, Dm, Xiong J, Liang MH, Tang J, Xu Y. Evaluation of the safety and probiotic properties of <em>Lactobacillus gasseri</em> LGZ1029 based on whole genome and phenotype analysis. LWT. 2023;184:114759.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wu%20JJ,%20Zhou%20Qy%20L,%20Dm,%20Xiong%20J,%20Liang%20MH,%20Tang%20J,%20Xu%20Y.%20Evaluation%20of%20the%20safety%20and%20probiotic%20properties%20of%20Lactobacillus%20gasseri%20LGZ1029%20based%20on%20whole%20genome%20and%20phenotype%20analysis.%20LWT.%202023;184:114759." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Yang J, Sun Y, Lei X, Zhao L, Luo R, Liu W. Evaluation of novel isolates of <em>Lacticaseibacillus rhamnosus</em> Probio-M9 derived through space mutagenesis. Food Biosci. 2023;52:102456.</cite> [<a href="https://scholar.google.com/scholar_lookup?Yang%20J,%20Sun%20Y,%20Lei%20X,%20Zhao%20L,%20Luo%20R,%20Liu%20W.%20Evaluation%20of%20novel%20isolates%20of%20Lacticaseibacillus%20rhamnosus%20Probio-M9%20derived%20through%20space%20mutagenesis.%20Food%20Biosci.%202023;52:102456." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Engel P, Kwong WK, Mcfrederick Q, Anderson KE, Dainat B. The bee microbiome: impact on bee health and model for evolution and ecology of host-microbe interactions. mBio. 2016;7(2):e02164–02115.
</cite> [<a href="https://doi.org/10.1128/mBio.02164-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4850275/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27118586/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Engel%20P,%20Kwong%20WK,%20Mcfrederick%20Q,%20Anderson%20KE,%20Dainat%20B.%20The%20bee%20microbiome:%20impact%20on%20bee%20health%20and%20model%20for%20evolution%20and%20ecology%20of%20host-microbe%20interactions.%20mBio.%202016;7(2):e02164%E2%80%9302115." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Bonilla-Rosso G, Engel P. Functional roles and metabolic niches in the honey bee gut microbiota. Curr Opin Microbiol. 2018;43:69–76.
</cite> [<a href="https://doi.org/10.1016/j.mib.2017.12.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29309997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bonilla-Rosso%20G,%20Engel%20P.%20Functional%20roles%20and%20metabolic%20niches%20in%20the%20honey%20bee%20gut%20microbiota.%20Curr%20Opin%20Microbiol.%202018;43:69%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Lv M, Wang S, Yin H, Dong K, Liu Y, Pan H, Lin Q, Cao Z. Probiotic potential and effects on gut microbiota composition and immunity of Indigenous gut lactobacilli in <em>Apis Cerana</em>. Probiotics Antimicrob Proteins. 2022;14:252–62.
</cite> [<a href="https://doi.org/10.1007/s12602-022-09935-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35325390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lv%20M,%20Wang%20S,%20Yin%20H,%20Dong%20K,%20Liu%20Y,%20Pan%20H,%20Lin%20Q,%20Cao%20Z.%20Probiotic%20potential%20and%20effects%20on%20gut%20microbiota%20composition%20and%20immunity%20of%20Indigenous%20gut%20lactobacilli%20in%20Apis%20Cerana.%20Probiotics%20Antimicrob%20Proteins.%202022;14:252%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Sabino YNV, Paiva AD, Fonseca BR, Medeiros JD, Machado ABF. Deciphering probiotic potential: a comprehensive guide to probiogenomic analyses. Future Microbiol. 2025;20(7–9):611–22.
</cite> [<a href="https://doi.org/10.1080/17460913.2025.2492472" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12153390/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40227157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sabino%20YNV,%20Paiva%20AD,%20Fonseca%20BR,%20Medeiros%20JD,%20Machado%20ABF.%20Deciphering%20probiotic%20potential:%20a%20comprehensive%20guide%20to%20probiogenomic%20analyses.%20Future%20Microbiol.%202025;20(7%E2%80%939):611%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Rocha BMO, Sabino YNV, de Almeida TC, Palacio FB, Rotta IS, Dias VC, da Silva VL, Diniz CG, Azevedo VAC, Brenig B, et al. Unlocking probiotic potential: genomic insights into <em>Weissella paramesenteroides</em> UFTM 2.6.1. Probiotics Antimicrob Proteins. 2024. </cite> [<a href="https://doi.org/10.1007/s12602-024-10409-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39633035/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>MARA. Notice from the general office of the ministry of agriculture and rural affairs on the issuance of the guidelines for identification and safety evaluation of strains used in direct feeding of microorganisms and fermented products. Bull Ministry Agric Rural Affairs People’s Repub China. 2021;11:97–111.</cite> [<a href="https://scholar.google.com/scholar_lookup?MARA.%20Notice%20from%20the%20general%20office%20of%20the%20ministry%20of%20agriculture%20and%20rural%20affairs%20on%20the%20issuance%20of%20the%20guidelines%20for%20identification%20and%20safety%20evaluation%20of%20strains%20used%20in%20direct%20feeding%20of%20microorganisms%20and%20fermented%20products.%20Bull%20Ministry%20Agric%20Rural%20Affairs%20People%E2%80%99s%20Repub%20China.%202021;11:97%E2%80%93111." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Senanayake D, Ramarao-Milne P, Pandey G, Hlaing MM, Chandrapala J, Torley PJ, Terefe NS. Genomic insights into exopolysaccharide biosynthesis pathways in novel <em>Lactiplantibacillus plantarum</em> and <em>Leuconostoc mesenteroides</em> strains. LWT. 2025;225:117863.</cite> [<a href="https://scholar.google.com/scholar_lookup?Senanayake%20D,%20Ramarao-Milne%20P,%20Pandey%20G,%20Hlaing%20MM,%20Chandrapala%20J,%20Torley%20PJ,%20Terefe%20NS.%20Genomic%20insights%20into%20exopolysaccharide%20biosynthesis%20pathways%20in%20novel%20Lactiplantibacillus%20plantarum%20and%20Leuconostoc%20mesenteroides%20strains.%20LWT.%202025;225:117863." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Tenea GN, Hidalgo J, Pepinos J, Ortega C. Genome characterization of <em>Leuconostoc pseudomesenteroides</em> UTNElla29 isolated from <em>Morus Nigra</em> (L.) fruits: a promising exopolysaccharides producing strain. LWT. 2024;206:116594.</cite> [<a href="https://scholar.google.com/scholar_lookup?Tenea%20GN,%20Hidalgo%20J,%20Pepinos%20J,%20Ortega%20C.%20Genome%20characterization%20of%20Leuconostoc%20pseudomesenteroides%20UTNElla29%20isolated%20from%20Morus%20Nigra%20(L.)%20fruits:%20a%20promising%20exopolysaccharides%20producing%20strain.%20LWT.%202024;206:116594." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Zhang Q, Ye Y. Not all predicted CRISPR-Cas systems are equal: isolated Cas genes and classes of CRISPR like elements. BMC Bioinformatics. 2017;18(1):92.
</cite> [<a href="https://doi.org/10.1186/s12859-017-1512-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5294841/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28166719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Q,%20Ye%20Y.%20Not%20all%20predicted%20CRISPR-Cas%20systems%20are%20equal:%20isolated%20Cas%20genes%20and%20classes%20of%20CRISPR%20like%20elements.%20BMC%20Bioinformatics.%202017;18(1):92." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Vu TTD, Jung J. Protein function prediction with gene ontology: from traditional to deep learning models. PeerJ. 2021;9:e12019.
</cite> [<a href="https://doi.org/10.7717/peerj.12019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8395570/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34513334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vu%20TTD,%20Jung%20J.%20Protein%20function%20prediction%20with%20gene%20ontology:%20from%20traditional%20to%20deep%20learning%20models.%20PeerJ.%202021;9:e12019." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Kanehisa M, Furumichi M, Sato Y, Kawashima M, Ishiguro-Watanabe M. KEGG for taxonomy-based analysis of pathways and genomes. Nucleic Acids Res. 2023;51(D1):D587-92.
</cite> [<a href="https://doi.org/10.1093/nar/gkac963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825424/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36300620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa%20M,%20Furumichi%20M,%20Sato%20Y,%20Kawashima%20M,%20Ishiguro-Watanabe%20M.%20KEGG%20for%20taxonomy-based%20analysis%20of%20pathways%20and%20genomes.%20Nucleic%20Acids%20Res.%202023;51(D1):D587-92." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Dan T, Chen H, Li T, Tian J, Ren W, Zhang H, Sun T. Influence of <em>Lactobacillus plantarum</em> P-8 on fermented milk flavor and storage stability. Front Microbiol. 2019;9:3133.
</cite> [<a href="https://doi.org/10.3389/fmicb.2018.03133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6333906/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30687239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dan%20T,%20Chen%20H,%20Li%20T,%20Tian%20J,%20Ren%20W,%20Zhang%20H,%20Sun%20T.%20Influence%20of%20Lactobacillus%20plantarum%20P-8%20on%20fermented%20milk%20flavor%20and%20storage%20stability.%20Front%20Microbiol.%202019;9:3133." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Chirag J, Luis M, Rodriguez -R, Adam M, Phillippy, Konstantinos T, Konstantinidis. Srinivas, aluru. High throughput ANI analysis of 90K prokaryotic genomes reveals clear species boundaries. Nat Commun. 2018;9(1):5114.
</cite> [<a href="https://doi.org/10.1038/s41467-018-07641-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6269478/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30504855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chirag%20J,%20Luis%20M,%20Rodriguez%20-R,%20Adam%20M,%20Phillippy,%20Konstantinos%20T,%20Konstantinidis.%20Srinivas,%20aluru.%20High%20throughput%20ANI%20analysis%20of%2090K%20prokaryotic%20genomes%20reveals%20clear%20species%20boundaries.%20Nat%20Commun.%202018;9(1):5114." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Zhao J, Liu S, Ren H, Afriyie OE, Zhang M, Xu D, Huang X. Genome-wide identification and comparative evolution of 14-3-3 gene family members in five brassicaceae species. BMC Genomics. 2025;26(1):309.
</cite> [<a href="https://doi.org/10.1186/s12864-025-11513-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11954322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40155852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20J,%20Liu%20S,%20Ren%20H,%20Afriyie%20OE,%20Zhang%20M,%20Xu%20D,%20Huang%20X.%20Genome-wide%20identification%20and%20comparative%20evolution%20of%2014-3-3%20gene%20family%20members%20in%20five%20brassicaceae%20species.%20BMC%20Genomics.%202025;26(1):309." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Lee JY, Han GG, Kim EB, Choi YJ. Comparative genomics of <em>Lactobacillus salivarius</em> strains focusing on their host adaptation. Microbiol Res. 2017;205:48–58.
</cite> [<a href="https://doi.org/10.1016/j.micres.2017.08.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28942844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20JY,%20Han%20GG,%20Kim%20EB,%20Choi%20YJ.%20Comparative%20genomics%20of%20Lactobacillus%20salivarius%20strains%20focusing%20on%20their%20host%20adaptation.%20Microbiol%20Res.%202017;205:48%E2%80%9358." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Tettelin H, Masignani V, Cieslewicz M, Donati C, Medini D, Ward N, Angiuoli S, Crabtree J, Jones A, Durkin A. Genome analysis of multiple pathogenic isolates of <em>Streptococcus agalactiae</em>: implications for the microbial pan-genome. Proc Natl Acad Sci U S A. 2005;102(39):13950–5.
</cite> [<a href="https://doi.org/10.1073/pnas.0506758102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1216834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16172379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tettelin%20H,%20Masignani%20V,%20Cieslewicz%20M,%20Donati%20C,%20Medini%20D,%20Ward%20N,%20Angiuoli%20S,%20Crabtree%20J,%20Jones%20A,%20Durkin%20A.%20Genome%20analysis%20of%20multiple%20pathogenic%20isolates%20of%20Streptococcus%20agalactiae:%20implications%20for%20the%20microbial%20pan-genome.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202005;102(39):13950%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Oksana L, David W, Ussery, Trudy M. Wassenaar. Comparative genomics of <em>Bifidobacterium</em>, <em>Lactobacillus</em> and related probiotic genera. Microb Ecol. 2011;63(3):651–73.
</cite> [<a href="https://doi.org/10.1007/s00248-011-9948-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3324989/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22031452/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oksana%20L,%20David%20W,%20Ussery,%20Trudy%20M.%20Wassenaar.%20Comparative%20genomics%20of%20Bifidobacterium,%20Lactobacillus%20and%20related%20probiotic%20genera.%20Microb%20Ecol.%202011;63(3):651%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Latif A, Shehzad A, Niazi S, Zahid A, Ashraf W, Iqbal MW, Rehman A, Riaz T, Aadil RM, Khan IM, et al. Probiotics: mechanism of action, health benefits and their application in food industries. Front Microbiol. 2023;14:1216674.
</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1216674" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10470842/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37664108/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Latif%20A,%20Shehzad%20A,%20Niazi%20S,%20Zahid%20A,%20Ashraf%20W,%20Iqbal%20MW,%20Rehman%20A,%20Riaz%20T,%20Aadil%20RM,%20Khan%20IM,%20et%20al.%20Probiotics:%20mechanism%20of%20action,%20health%20benefits%20and%20their%20application%20in%20food%20industries.%20Front%20Microbiol.%202023;14:1216674." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Duche RT, Singh A, Wandhare AG, Sangwan V, Sihag MK, Nwagu TNT, Panwar H, Ezeogu LI. Antibiotic resistance in potential probiotic lactic acid bacteria of fermented foods and human origin from Nigeria. BMC Microbiol. 2023;23(1):142.
</cite> [<a href="https://doi.org/10.1186/s12866-023-02883-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10197481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37208603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duche%20RT,%20Singh%20A,%20Wandhare%20AG,%20Sangwan%20V,%20Sihag%20MK,%20Nwagu%20TNT,%20Panwar%20H,%20Ezeogu%20LI.%20Antibiotic%20resistance%20in%20potential%20probiotic%20lactic%20acid%20bacteria%20of%20fermented%20foods%20and%20human%20origin%20from%20Nigeria.%20BMC%20Microbiol.%202023;23(1):142." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Rana C, Vikas V, Awasthi S, Gautam D, Vats A, Rajput S, Behera M, Ludri A, Berwal A, Singh D. Antimicrobial resistance genes and associated mobile genetic elements in <em>Escherichia coli</em> from human, animal and environment. Chemosphere. 2024;369:143808. </cite> [<a href="https://doi.org/10.1016/j.chemosphere.2024.143808" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39608649/" class="usa-link">PubMed</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Drew D, North RA, Nagarathinam K, Tanabe M. Structures and general transport mechanisms by the major facilitator superfamily (MFS). Chem Rev. 2021;121(9):5289–335.
</cite> [<a href="https://doi.org/10.1021/acs.chemrev.0c00983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8154325/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33886296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Drew%20D,%20North%20RA,%20Nagarathinam%20K,%20Tanabe%20M.%20Structures%20and%20general%20transport%20mechanisms%20by%20the%20major%20facilitator%20superfamily%20(MFS).%20Chem%20Rev.%202021;121(9):5289%E2%80%93335." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Sharma AK, Dhasmana N, Dubey N, Kumar N, Gangwal A, Gupta M, Singh Y. Bacterial virulence factors: secreted for survival. Indian J Microbiol. 2017;57(1):1–10.
</cite> [<a href="https://doi.org/10.1007/s12088-016-0625-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5243249/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28148975/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sharma%20AK,%20Dhasmana%20N,%20Dubey%20N,%20Kumar%20N,%20Gangwal%20A,%20Gupta%20M,%20Singh%20Y.%20Bacterial%20virulence%20factors:%20secreted%20for%20survival.%20Indian%20J%20Microbiol.%202017;57(1):1%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Ahmad M, Aduru SV, Smith RP, Zhao Z, Lopatkin AJ. The role of bacterial metabolism in antimicrobial resistance. Nat Rev Microbiol. 2025;23(7):439–54. </cite> [<a href="https://doi.org/10.1038/s41579-025-01155-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12173792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39979446/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Colautti A, Arnoldi M, Comi G, Iacumin L. Antibiotic resistance and virulence factors in lactobacilli: something to carefully consider. Food Microbiol. 2022;103:103934.
</cite> [<a href="https://doi.org/10.1016/j.fm.2021.103934" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35082060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Colautti%20A,%20Arnoldi%20M,%20Comi%20G,%20Iacumin%20L.%20Antibiotic%20resistance%20and%20virulence%20factors%20in%20lactobacilli:%20something%20to%20carefully%20consider.%20Food%20Microbiol.%202022;103:103934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Yao P, Mohd Esah E, Zhao C. Regulatory mechanisms and applications of <em>Lactobacillus</em> biofilms in the food industry. Front Microbiol. 2025;15:1465373.
</cite> [<a href="https://doi.org/10.3389/fmicb.2024.1465373" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11753222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39845052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yao%20P,%20Mohd%20Esah%20E,%20Zhao%20C.%20Regulatory%20mechanisms%20and%20applications%20of%20Lactobacillus%20biofilms%20in%20the%20food%20industry.%20Front%20Microbiol.%202025;15:1465373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Zhao Y, Wang W, Yao J, Wang X, Liu D, Wang P. The HipAB toxin-antitoxin system stabilizes a composite genomic island in <em>Shewanella putrefaciens</em> CN-32. Front Microbiol. 2022;13:858857.
</cite> [<a href="https://doi.org/10.3389/fmicb.2022.858857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8978831/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35387082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20Y,%20Wang%20W,%20Yao%20J,%20Wang%20X,%20Liu%20D,%20Wang%20P.%20The%20HipAB%20toxin-antitoxin%20system%20stabilizes%20a%20composite%20genomic%20island%20in%20Shewanella%20putrefaciens%20CN-32.%20Front%20Microbiol.%202022;13:858857." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Yasmin I, Saeed M, Khan WA, Khaliq A, Tanweer S. In vitro probiotic potential and safety evaluation (hemolytic, cytotoxic activity) of <em>Bifidobacterium</em> strains isolated from raw camel milk. Microorganisms. 2020;8(3):354.
</cite> [<a href="https://doi.org/10.3390/microorganisms8030354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7143641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32131456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yasmin%20I,%20Saeed%20M,%20Khan%20WA,%20Khaliq%20A,%20Tanweer%20S.%20In%20vitro%20probiotic%20potential%20and%20safety%20evaluation%20(hemolytic,%20cytotoxic%20activity)%20of%20Bifidobacterium%20strains%20isolated%20from%20raw%20camel%20milk.%20Microorganisms.%202020;8(3):354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Zhu Z, Hu Z, Li S, Fang R, Ono HK, Hu DL. Molecular characteristics and pathogenicity of <em>Staphylococcus aureus</em> exotoxins. Int J Mol Sci. 2024;25(1):25.</cite> [<a href="https://doi.org/10.3390/ijms25010395" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10778951/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38203566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20Z,%20Hu%20Z,%20Li%20S,%20Fang%20R,%20Ono%20HK,%20Hu%20DL.%20Molecular%20characteristics%20and%20pathogenicity%20of%20Staphylococcus%20aureus%20exotoxins.%20Int%20J%20Mol%20Sci.%202024;25(1):25." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Linares DM, Martin MC, Ladero V, Alvarez MA, Fernández M. Biogenic amines in dairy products. Crit Rev Food Sci Nutr. 2011;51(7):691–703.
</cite> [<a href="https://doi.org/10.1080/10408398.2011.582813" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21793728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Linares%20DM,%20Martin%20MC,%20Ladero%20V,%20Alvarez%20MA,%20Fern%C3%A1ndez%20M.%20Biogenic%20amines%20in%20dairy%20products.%20Crit%20Rev%20Food%20Sci%20Nutr.%202011;51(7):691%E2%80%93703." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Derunets AS, Selimzyanova AI, Rykov SV, Kuznetsov AE, Berezina OV. Strategies to enhance stress tolerance in lactic acid bacteria across diverse stress conditions. World J Microbiol Biotechnol. 2024;40(4):126.
</cite> [<a href="https://doi.org/10.1007/s11274-024-03905-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38446232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Derunets%20AS,%20Selimzyanova%20AI,%20Rykov%20SV,%20Kuznetsov%20AE,%20Berezina%20OV.%20Strategies%20to%20enhance%20stress%20tolerance%20in%20lactic%20acid%20bacteria%20across%20diverse%20stress%20conditions.%20World%20J%20Microbiol%20Biotechnol.%202024;40(4):126." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Zharova TV, Grivennikova VG, Borisov VB. F-1·F-0 ATP synthase/atpase: contemporary view on unidirectional catalysis. Int J Mol Sci. 2023;24(6):5417.
</cite> [<a href="https://doi.org/10.3390/ijms24065417" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10049701/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36982498/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zharova%20TV,%20Grivennikova%20VG,%20Borisov%20VB.%20F-1%C2%B7F-0%20ATP%20synthase/atpase:%20contemporary%20view%20on%20unidirectional%20catalysis.%20Int%20J%20Mol%20Sci.%202023;24(6):5417." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Sekiya M, Ikeda K, Yonai A, Ishikawa T, Shimoyama Y, Kodama Y, Sasaki M, Nakanishi-Matsui M. F-type proton-pumping ATPase mediates acid tolerance in <em>Streptococcus mutans</em>. J Appl Microbiol. 2023;134(4): lxad073.
</cite> [<a href="https://doi.org/10.1093/jambio/lxad073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37055370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sekiya%20M,%20Ikeda%20K,%20Yonai%20A,%20Ishikawa%20T,%20Shimoyama%20Y,%20Kodama%20Y,%20Sasaki%20M,%20Nakanishi-Matsui%20M.%20F-type%20proton-pumping%20ATPase%20mediates%20acid%20tolerance%20in%20Streptococcus%20mutans.%20J%20Appl%20Microbiol.%202023;134(4):%20lxad073." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Yang H, Wang D, Jin Y, Zhou R, Huang J, Wu C. Arginine deiminase pathway of <em>Tetragenococcus halophilus</em> contributes to improve the acid tolerance of lactic acid bacteria. Food Microbiol. 2023;113:104281.
</cite> [<a href="https://doi.org/10.1016/j.fm.2023.104281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37098426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20H,%20Wang%20D,%20Jin%20Y,%20Zhou%20R,%20Huang%20J,%20Wu%20C.%20Arginine%20deiminase%20pathway%20of%20Tetragenococcus%20halophilus%20contributes%20to%20improve%20the%20acid%20tolerance%20of%20lactic%20acid%20bacteria.%20Food%20Microbiol.%202023;113:104281." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Lorena Díez, Ana S, Fernández-Pérez Rocío. Miriam, gonzález, carmen, tenorio. Transcriptome analysis shows activation of the arginine deiminase pathway in <em>Lactococcus lactis</em> as a response to ethanol stress. Int J Food Microbiol. 2017;257:41–8.
</cite> [<a href="https://doi.org/10.1016/j.ijfoodmicro.2017.05.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28644989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lorena%20D%C3%ADez,%20Ana%20S,%20Fern%C3%A1ndez-P%C3%A9rez%20Roc%C3%ADo.%20Miriam,%20gonz%C3%A1lez,%20carmen,%20tenorio.%20Transcriptome%20analysis%20shows%20activation%20of%20the%20arginine%20deiminase%20pathway%20in%20Lactococcus%20lactis%20as%20a%20response%20to%20ethanol%20stress.%20Int%20J%20Food%20Microbiol.%202017;257:41%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Zhang W, Chen X, Sun W, Nie T, Quanquin N, Sun Y. <em>Escherichia coli</em> increases its ATP concentration in weakly acidic environments principally through the glycolytic pathway. Genes. 2020;11(9):991.
</cite> [<a href="https://doi.org/10.3390/genes11090991" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7563387/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32854287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20W,%20Chen%20X,%20Sun%20W,%20Nie%20T,%20Quanquin%20N,%20Sun%20Y.%20Escherichia%20coli%20increases%20its%20ATP%20concentration%20in%20weakly%20acidic%20environments%20principally%20through%20the%20glycolytic%20pathway.%20Genes.%202020;11(9):991." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Pilhofer M, Rappl K, Eckl C, Bauer AP, Ludwig W, Schleifer KH, Petroni G. Characterization and evolution of cell division and cell wall synthesis genes in the bacterial phyla verrucomicrobia, lentisphaerae, chlamydiae, and planctomycetes and phylogenetic comparison with rRNA genes. J Bacteriol. 2008;190(9):3192–202.
</cite> [<a href="https://doi.org/10.1128/JB.01797-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2347405/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18310338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pilhofer%20M,%20Rappl%20K,%20Eckl%20C,%20Bauer%20AP,%20Ludwig%20W,%20Schleifer%20KH,%20Petroni%20G.%20Characterization%20and%20evolution%20of%20cell%20division%20and%20cell%20wall%20synthesis%20genes%20in%20the%20bacterial%20phyla%20verrucomicrobia,%20lentisphaerae,%20chlamydiae,%20and%20planctomycetes%20and%20phylogenetic%20comparison%20with%20rRNA%20genes.%20J%20Bacteriol.%202008;190(9):3192%E2%80%93202." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Pakotiprapha D, Samuels M, Shen K, Hu JH, Jeruzalmi D. Structure and mechanism of the UvrA-UvrB DNA damage sensor. Nat Struct Mol Biol. 2012;19(3):291–8.
</cite> [<a href="https://doi.org/10.1038/nsmb.2240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22307053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pakotiprapha%20D,%20Samuels%20M,%20Shen%20K,%20Hu%20JH,%20Jeruzalmi%20D.%20Structure%20and%20mechanism%20of%20the%20UvrA-UvrB%20DNA%20damage%20sensor.%20Nat%20Struct%20Mol%20Biol.%202012;19(3):291%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Chen C, Yu L, Tian F, Zhao J, Zhai Q. Identification of novel bile salt-tolerant genes in <em>Lactobacillus</em> using comparative genomics and its application in the rapid screening of tolerant strains. Microorganisms. 2022;10(12):2371.
</cite> [<a href="https://doi.org/10.3390/microorganisms10122371" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9786149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36557624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20C,%20Yu%20L,%20Tian%20F,%20Zhao%20J,%20Zhai%20Q.%20Identification%20of%20novel%20bile%20salt-tolerant%20genes%20in%20Lactobacillus%20using%20comparative%20genomics%20and%20its%20application%20in%20the%20rapid%20screening%20of%20tolerant%20strains.%20Microorganisms.%202022;10(12):2371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Fontana A, Falasconi I, Molinari P, Treu L, Basile A, Vezzi A, Campanaro S, Morelli L. Genomic comparison of <em>Lactobacillus helveticus</em> strains highlights probiotic potential. Front Microbiol. 2019;10:1380.
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.01380" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6606699/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31293536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fontana%20A,%20Falasconi%20I,%20Molinari%20P,%20Treu%20L,%20Basile%20A,%20Vezzi%20A,%20Campanaro%20S,%20Morelli%20L.%20Genomic%20comparison%20of%20Lactobacillus%20helveticus%20strains%20highlights%20probiotic%20potential.%20Front%20Microbiol.%202019;10:1380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Wu T, Jiang J, Zhang H, Liu J, Ruan H. Transcending membrane barriers: advances in membrane engineering to enhance the production capacity of microbial cell factories. Microb Cell Fact. 2024;23(1):154.
</cite> [<a href="https://doi.org/10.1186/s12934-024-02436-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11128114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38796463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20T,%20Jiang%20J,%20Zhang%20H,%20Liu%20J,%20Ruan%20H.%20Transcending%20membrane%20barriers:%20advances%20in%20membrane%20engineering%20to%20enhance%20the%20production%20capacity%20of%20microbial%20cell%20factories.%20Microb%20Cell%20Fact.%202024;23(1):154." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Geng J, Long J, Hu Q, Liu M, Ge A, Du Y, Zhang T, Jin Y, Yang H, Chen S. Current status of cyclopropane fatty acids on bacterial cell membranes characteristics and physiological functions. Microb Pathog. 2025;200:107295.
</cite> [<a href="https://doi.org/10.1016/j.micpath.2025.107295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39805345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Geng%20J,%20Long%20J,%20Hu%20Q,%20Liu%20M,%20Ge%20A,%20Du%20Y,%20Zhang%20T,%20Jin%20Y,%20Yang%20H,%20Chen%20S.%20Current%20status%20of%20cyclopropane%20fatty%20acids%20on%20bacterial%20cell%20membranes%20characteristics%20and%20physiological%20functions.%20Microb%20Pathog.%202025;200:107295." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Muhammad A, Sulman S, Irfan M, Birgitta HN, Kuipers OP. N-acetylglucosamine-mediated expression of <em>NagA</em> and <em>NagB</em> in <em>Streptococcus pneumoniae</em>. Front Cell Infect Microbiol. 2016;6:158.
</cite> [<a href="https://doi.org/10.3389/fcimb.2016.00158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5110562/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27900287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Muhammad%20A,%20Sulman%20S,%20Irfan%20M,%20Birgitta%20HN,%20Kuipers%20OP.%20N-acetylglucosamine-mediated%20expression%20of%20NagA%20and%20NagB%20in%20Streptococcus%20pneumoniae.%20Front%20Cell%20Infect%20Microbiol.%202016;6:158." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Zhang H, Xu M, Hu S, Zhao H, Zhang B. The enzyme gene expression of protein utilization and metabolism by <em>Lactobacillus helveticus</em> CICC 22171. Microorganisms. 2022;10(9):1724.
</cite> [<a href="https://doi.org/10.3390/microorganisms10091724" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9501481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36144326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20H,%20Xu%20M,%20Hu%20S,%20Zhao%20H,%20Zhang%20B.%20The%20enzyme%20gene%20expression%20of%20protein%20utilization%20and%20metabolism%20by%20Lactobacillus%20helveticus%20CICC%2022171.%20Microorganisms.%202022;10(9):1724." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>García-Contreras R, de la Mora J, Mora-Montes HM, Martínez-Álvarez JA, Vicente-Gómez M, Padilla-Vaca F, Vargas-Maya NI, Franco B. The inorganic pyrophosphatases of microorganisms: a structural and functional review. PeerJ. 2024;12:e17496.
</cite> [<a href="https://doi.org/10.7717/peerj.17496" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11210485/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38938619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garc%C3%ADa-Contreras%20R,%20de%20la%20Mora%20J,%20Mora-Montes%20HM,%20Mart%C3%ADnez-%C3%81lvarez%20JA,%20Vicente-G%C3%B3mez%20M,%20Padilla-Vaca%20F,%20Vargas-Maya%20NI,%20Franco%20B.%20The%20inorganic%20pyrophosphatases%20of%20microorganisms:%20a%20structural%20and%20functional%20review.%20PeerJ.%202024;12:e17496." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Hamon E, Horvatovich P, Izquierdo E, Bringel F, Marchioni E, Aoudé-Werner D, Ennahar S. Comparative proteomic analysis of <em>Lactobacillus plantarum</em> for the identification of key proteins in bile tolerance. BMC Microbiol. 2011;11:63.
</cite> [<a href="https://doi.org/10.1186/1471-2180-11-63" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3073879/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21447177/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hamon%20E,%20Horvatovich%20P,%20Izquierdo%20E,%20Bringel%20F,%20Marchioni%20E,%20Aoud%C3%A9-Werner%20D,%20Ennahar%20S.%20Comparative%20proteomic%20analysis%20of%20Lactobacillus%20plantarum%20for%20the%20identification%20of%20key%20proteins%20in%20bile%20tolerance.%20BMC%20Microbiol.%202011;11:63." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Lebeer S, Vanderleyden J, De Keersmaecker SCJ. Host interactions of probiotic bacterial surface molecules: comparison with commensals and pathogens. Nat Rev Microbiol. 2010;8(3):171–84.</cite> [<a href="https://doi.org/10.1038/nrmicro2297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20157338/" class="usa-link">PubMed</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Payen S, Giroux MC, Gisch N, Schombel U, Fittipaldi N, Segura M, Gottschalk M. Lipoteichoic acids influence cell shape and bacterial division of <em>Streptococcus suis</em> serotype 2, but play a limited role in the pathogenesis of the infection. Vet Res. 2024;55(1):14.
</cite> [<a href="https://doi.org/10.1186/s13567-024-01287-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10953176/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38504299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Payen%20S,%20Giroux%20MC,%20Gisch%20N,%20Schombel%20U,%20Fittipaldi%20N,%20Segura%20M,%20Gottschalk%20M.%20Lipoteichoic%20acids%20influence%20cell%20shape%20and%20bacterial%20division%20of%20Streptococcus%20suis%20serotype%202,%20but%20play%20a%20limited%20role%20in%20the%20pathogenesis%20of%20the%20infection.%20Vet%20Res.%202024;55(1):14." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Schreiner SA, Sokoli A, Felder KM, Wittenbrink MM, Schwarzenbach S, Guhl B, Hoelzle K, Hoelzle LE. The surface-localised α-enolase of <em>Mycoplasma suis</em> is an adhesion protein. Vet Microbiol. 2012;156(1–2):88–95.
</cite> [<a href="https://doi.org/10.1016/j.vetmic.2011.10.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22047714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schreiner%20SA,%20Sokoli%20A,%20Felder%20KM,%20Wittenbrink%20MM,%20Schwarzenbach%20S,%20Guhl%20B,%20Hoelzle%20K,%20Hoelzle%20LE.%20The%20surface-localised%20%CE%B1-enolase%20of%20Mycoplasma%20suis%20is%20an%20adhesion%20protein.%20Vet%20Microbiol.%202012;156(1%E2%80%932):88%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Lindahl G, Stalhammar-Carlemalm M, Areschoug T. Surface proteins of <em>Streptococcus agalactiae</em> and related proteins in other bacterial pathogens. Clin Microbiol Rev. 2005;18(1):102–27.
</cite> [<a href="https://doi.org/10.1128/CMR.18.1.102-127.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC544178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15653821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lindahl%20G,%20Stalhammar-Carlemalm%20M,%20Areschoug%20T.%20Surface%20proteins%20of%20Streptococcus%20agalactiae%20and%20related%20proteins%20in%20other%20bacterial%20pathogens.%20Clin%20Microbiol%20Rev.%202005;18(1):102%E2%80%9327." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Lee J, Jo J, Wan J, Seo H, Han SW, Shin YJ, Kim DH. In vitro evaluation of probiotic properties and anti-pathogenic effects of <em>Lactobacillus</em> and <em>Bifidobacterium</em> strains as potential probiotics. Foods. 2024;13(14):2301.
</cite> [<a href="https://doi.org/10.3390/foods13142301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11276478/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39063385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20J,%20Jo%20J,%20Wan%20J,%20Seo%20H,%20Han%20SW,%20Shin%20YJ,%20Kim%20DH.%20In%20vitro%20evaluation%20of%20probiotic%20properties%20and%20anti-pathogenic%20effects%20of%20Lactobacillus%20and%20Bifidobacterium%20strains%20as%20potential%20probiotics.%20Foods.%202024;13(14):2301." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Schachtsiek M, Hammes WP, Hertel C. Characterization of <em>Lactobacillus coryniformis</em> DSM 20001T surface protein Cpf mediating coaggregation with and aggregation among pathogens. Appl Environ Microbiol. 2004;70(12):7078–85.</cite> [<a href="https://doi.org/10.1128/AEM.70.12.7078-7085.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC535140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15574903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schachtsiek%20M,%20Hammes%20WP,%20Hertel%20C.%20Characterization%20of%20Lactobacillus%20coryniformis%20DSM%2020001T%20surface%20protein%20Cpf%20mediating%20coaggregation%20with%20and%20aggregation%20among%20pathogens.%20Appl%20Environ%20Microbiol.%202004;70(12):7078%E2%80%9385." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Mizuno K, Mizuno M, Yamauchi M, Takemura AJ, Medrano Romero V, Morikawa K. Adjacent-possible ecological niche: growth of <em>Lactobacillus</em> species co-cultured with <em>Escherichia coli</em> in a synthetic minimal medium. Sci Rep. 2017;7(1):12880.
</cite> [<a href="https://doi.org/10.1038/s41598-017-12894-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5643319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29038545/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mizuno%20K,%20Mizuno%20M,%20Yamauchi%20M,%20Takemura%20AJ,%20Medrano%20Romero%20V,%20Morikawa%20K.%20Adjacent-possible%20ecological%20niche:%20growth%20of%20Lactobacillus%20species%20co-cultured%20with%20Escherichia%20coli%20in%20a%20synthetic%20minimal%20medium.%20Sci%20Rep.%202017;7(1):12880." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Vinayamohan P, Joseph D, Viju LS, Baskaran SA, Venkitanarayanan K. Efficacy of probiotics in reducing pathogenic potential of infectious agents. Fermentation. 2024;10(12):599.</cite> [<a href="https://scholar.google.com/scholar_lookup?Vinayamohan%20P,%20Joseph%20D,%20Viju%20LS,%20Baskaran%20SA,%20Venkitanarayanan%20K.%20Efficacy%20of%20probiotics%20in%20reducing%20pathogenic%20potential%20of%20infectious%20agents.%20Fermentation.%202024;10(12):599." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Rusu ME, Fizean I, Vlase L, Popa DS. Antioxidants in age-related diseases and anti-aging strategies. Antioxidants. 2022;11(10):1868.
</cite> [<a href="https://doi.org/10.3390/antiox11101868" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9598595/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36290589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rusu%20ME,%20Fizean%20I,%20Vlase%20L,%20Popa%20DS.%20Antioxidants%20in%20age-related%20diseases%20and%20anti-aging%20strategies.%20Antioxidants.%202022;11(10):1868." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Mafe AN, Iruogheneedo G, Akpoghelie PO, Gaaz TS, Yousif E, Zainulabdeen K, Isoje EF, Igbuku UA, Opiti RA, Garba Y. Probiotics and food bioactives: unraveling their impact on gut microbiome, inflammation, and metabolic health. Probiotics Antimicrob Proteins. 2025:1–42.</cite> [<a href="https://doi.org/10.1007/s12602-025-10452-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39808399/" class="usa-link">PubMed</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Tanaka R, Kosugi K, Mizukami M, Ishibashi M, Tokunaga H, Tokunaga M. Expression and purification of thioredoxin (TrxA) and thioredoxin reductase (TrxB) from <em>Brevibacillus choshinensis</em>. Protein Expr Purif. 2004;37(2):385–91.
</cite> [<a href="https://doi.org/10.1016/j.pep.2004.06.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15358361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tanaka%20R,%20Kosugi%20K,%20Mizukami%20M,%20Ishibashi%20M,%20Tokunaga%20H,%20Tokunaga%20M.%20Expression%20and%20purification%20of%20thioredoxin%20(TrxA)%20and%20thioredoxin%20reductase%20(TrxB)%20from%20Brevibacillus%20choshinensis.%20Protein%20Expr%20Purif.%202004;37(2):385%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Lee JH, Yeo WS, Roe JH. Induction of the SufA operon encoding Fe-S assembly proteins by superoxide generators and hydrogen peroxide: involvement of oxyr, IHF and an unidentified oxidant‐responsive factor. Mol Microbiol. 2004;51(6):1745–55.
</cite> [<a href="https://doi.org/10.1111/j.1365-2958.2003.03946.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15009899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20JH,%20Yeo%20WS,%20Roe%20JH.%20Induction%20of%20the%20SufA%20operon%20encoding%20Fe-S%20assembly%20proteins%20by%20superoxide%20generators%20and%20hydrogen%20peroxide:%20involvement%20of%20oxyr,%20IHF%20and%20an%20unidentified%20oxidant%E2%80%90responsive%20factor.%20Mol%20Microbiol.%202004;51(6):1745%E2%80%9355." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Riboldi GP, Bierhals CG, Mattos EPD, Frazzon APG, D’Azevedo PA, Frazzon J. Oxidative stress enhances the expression of sulfur assimilation genes: preliminary insights on the <em>Enterococcus faecalis</em> iron-sulfur cluster machinery regulation. Mem Inst Oswaldo Cruz. 2018;109(4):408–13.</cite> [<a href="https://doi.org/10.1590/0074-0276140006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4155840/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24936909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Riboldi%20GP,%20Bierhals%20CG,%20Mattos%20EPD,%20Frazzon%20APG,%20D%E2%80%99Azevedo%20PA,%20Frazzon%20J.%20Oxidative%20stress%20enhances%20the%20expression%20of%20sulfur%20assimilation%20genes:%20preliminary%20insights%20on%20the%20Enterococcus%20faecalis%20iron-sulfur%20cluster%20machinery%20regulation.%20Mem%20Inst%20Oswaldo%20Cruz.%202018;109(4):408%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Singh V, Singh K, Baum K. The role of methionine sulfoxide reductases in oxidative stress tolerance and virulence of <em>Staphylococcus aureus</em> and other bacteria. Antioxidants. 2018;7(10):128.
</cite> [<a href="https://doi.org/10.3390/antiox7100128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6210949/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30274148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh%20V,%20Singh%20K,%20Baum%20K.%20The%20role%20of%20methionine%20sulfoxide%20reductases%20in%20oxidative%20stress%20tolerance%20and%20virulence%20of%20Staphylococcus%20aureus%20and%20other%20bacteria.%20Antioxidants.%202018;7(10):128." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Plavec TV, Žagar Soderžnik K, Della Pelle G, Zupančič Š, Vidmar R, Berlec A. Incorporation of recombinant proteins into extracellular vesicles by <em>Lactococcus cremoris</em>. Sci Rep. 2025;15(1):1768.
</cite> [<a href="https://doi.org/10.1038/s41598-025-86492-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11736121/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39815011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Plavec%20TV,%20%C5%BDagar%20Soder%C5%BEnik%20K,%20Della%20Pelle%20G,%20Zupan%C4%8Di%C4%8D%20%C5%A0,%20Vidmar%20R,%20Berlec%20A.%20Incorporation%20of%20recombinant%20proteins%20into%20extracellular%20vesicles%20by%20Lactococcus%20cremoris.%20Sci%20Rep.%202025;15(1):1768." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Szajewska H, Scott KP, Meij TD, Forslund-Startceva SK, Knight R, Koren O, Little P, Johnston BC, Ukasik J, Suez J. Antibiotic-perturbed microbiota and the role of probiotics. Nat Rev Gastroenterol Hepatol. 2025;22(3):155–72.
</cite> [<a href="https://doi.org/10.1038/s41575-024-01023-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39663462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Szajewska%20H,%20Scott%20KP,%20Meij%20TD,%20Forslund-Startceva%20SK,%20Knight%20R,%20Koren%20O,%20Little%20P,%20Johnston%20BC,%20Ukasik%20J,%20Suez%20J.%20Antibiotic-perturbed%20microbiota%20and%20the%20role%20of%20probiotics.%20Nat%20Rev%20Gastroenterol%20Hepatol.%202025;22(3):155%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Angelescu IR, Grosu-Tudor S, Cojoc L, Maria G, Zamfir M. Isolation, characterization, and applicability of helveticin 34.9, a class III bacteriocin produced by <em>Lactobacillus helveticus</em> 34.9. World J Microbiol Biotechnol. 2021;38(12):220.</cite> [<a href="https://doi.org/10.1007/s11274-022-03408-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36083397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Angelescu%20IR,%20Grosu-Tudor%20S,%20Cojoc%20L,%20Maria%20G,%20Zamfir%20M.%20Isolation,%20characterization,%20and%20applicability%20of%20helveticin%2034.9,%20a%20class%20III%20bacteriocin%20produced%20by%20Lactobacillus%20helveticus%2034.9.%20World%20J%20Microbiol%20Biotechnol.%202021;38(12):220." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Meng F, Zhu X, Zhao H, Nie T, Lu F, Lu Z, Lu Y. A class III bacteriocin with broad-spectrum antibacterial activity from <em>Lactobacillus acidophilus</em> NX2-6 and its preservation in milk and cheese. Food Control. 2021;121:107597.</cite> [<a href="https://scholar.google.com/scholar_lookup?Meng%20F,%20Zhu%20X,%20Zhao%20H,%20Nie%20T,%20Lu%20F,%20Lu%20Z,%20Lu%20Y.%20A%20class%20III%20bacteriocin%20with%20broad-spectrum%20antibacterial%20activity%20from%20Lactobacillus%20acidophilus%20NX2-6%20and%20its%20preservation%20in%20milk%20and%20cheese.%20Food%20Control.%202021;121:107597." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Cotter PD, Hill C, Ross RP. Bacteriocins: developing innate immunity for food. Nat Rev Microbiol. 2005;3(10):777–88.
</cite> [<a href="https://doi.org/10.1038/nrmicro1273" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16205711/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cotter%20PD,%20Hill%20C,%20Ross%20RP.%20Bacteriocins:%20developing%20innate%20immunity%20for%20food.%20Nat%20Rev%20Microbiol.%202005;3(10):777%E2%80%9388." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Nilsen T, Nes Ingolf F, Holo Helge. Enterolysin A, a cell wall-degrading bacteriocin from <em>Enterococcus faecalis</em> LMG 2333. Appl Environ Microbiol. 2003;69(5):2975–84.</cite> [<a href="https://doi.org/10.1128/AEM.69.5.2975-2984.2003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC154489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12732574/" class="usa-link">PubMed</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Cantarel BL, Coutinho PM, Corinne R, Thomas B, Vincent L, Bernard H. The carbohydrate-active enzymes database (CAZy): an expert resource for glycogenomics. Nucleic Acids Res. 2009;37:D233–8. 
</cite> [<a href="https://doi.org/10.1093/nar/gkn663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2686590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18838391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cantarel%20BL,%20Coutinho%20PM,%20Corinne%20R,%20Thomas%20B,%20Vincent%20L,%20Bernard%20H.%20The%20carbohydrate-active%20enzymes%20database%20(CAZy):%20an%20expert%20resource%20for%20glycogenomics.%20Nucleic%20Acids%20Res.%202009;37:D233%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Vincent L, Hemalatha GR, Elodie D, Coutinho PM, Bernard H. The carbohydrate-active enzymes database (CAZy) in 2013. Nucleic Acids Res. 2014;42:D490–5.
</cite> [<a href="https://doi.org/10.1093/nar/gkt1178" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3965031/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24270786/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vincent%20L,%20Hemalatha%20GR,%20Elodie%20D,%20Coutinho%20PM,%20Bernard%20H.%20The%20carbohydrate-active%20enzymes%20database%20(CAZy)%20in%202013.%20Nucleic%20Acids%20Res.%202014;42:D490%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Levasseur A, Drula E, Lombard V. Expansion of the enzymatic repertoire of the CAZy database to integrate auxiliary redox enzymes. Biotechnol Biofuels. 2013;6(1):41.
</cite> [<a href="https://doi.org/10.1186/1754-6834-6-41" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3620520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23514094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Levasseur%20A,%20Drula%20E,%20Lombard%20V.%20Expansion%20of%20the%20enzymatic%20repertoire%20of%20the%20CAZy%20database%20to%20integrate%20auxiliary%20redox%20enzymes.%20Biotechnol%20Biofuels.%202013;6(1):41." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Nguyen HT, Pham TT, Nguyen PT, Le-Buanec H, Rabetafika HN, Razafindralambo HL. Advances in microbial exopolysaccharides: present and future applications. Biomolecules. 2024;14(9):1162.
</cite> [<a href="https://doi.org/10.3390/biom14091162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11430787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39334928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen%20HT,%20Pham%20TT,%20Nguyen%20PT,%20Le-Buanec%20H,%20Rabetafika%20HN,%20Razafindralambo%20HL.%20Advances%20in%20microbial%20exopolysaccharides:%20present%20and%20future%20applications.%20Biomolecules.%202024;14(9):1162." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Nguyen PT, Nguyen HT. Antioxidant property and structural characteristics of exopolysaccharides derived from <em>Lactiplantibacillus plantarum</em> VAL6. Discover Bacteria. 2024;1(1):1–10.</cite> [<a href="https://scholar.google.com/scholar_lookup?Nguyen%20PT,%20Nguyen%20HT.%20Antioxidant%20property%20and%20structural%20characteristics%20of%20exopolysaccharides%20derived%20from%20Lactiplantibacillus%20plantarum%20VAL6.%20Discover%20Bacteria.%202024;1(1):1%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Yang Z, Li S, Zhang X, Zeng X, Li D, Zhao Y, Zhang J. Capsular and slime-polysaccharide production by <em>Lactobacillus rhamnosus</em> JAAS8 isolated from Chinese sauerkraut: potential application in fermented milk products. J Biosci Bioeng. 2010;110(1):53–7.
</cite> [<a href="https://doi.org/10.1016/j.jbiosc.2009.12.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20541116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20Z,%20Li%20S,%20Zhang%20X,%20Zeng%20X,%20Li%20D,%20Zhao%20Y,%20Zhang%20J.%20Capsular%20and%20slime-polysaccharide%20production%20by%20Lactobacillus%20rhamnosus%20JAAS8%20isolated%20from%20Chinese%20sauerkraut:%20potential%20application%20in%20fermented%20milk%20products.%20J%20Biosci%20Bioeng.%202010;110(1):53%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Behera HT, Mojumdar A, Mohapatra C, Ray L. Microbial polysaccharides with potential industrial applications: diversity, synthesis, and their applications. Microb Polym. 2021;521:546.</cite> [<a href="https://scholar.google.com/scholar_lookup?Behera%20HT,%20Mojumdar%20A,%20Mohapatra%20C,%20Ray%20L.%20Microbial%20polysaccharides%20with%20potential%20industrial%20applications:%20diversity,%20synthesis,%20and%20their%20applications.%20Microb%20Polym.%202021;521:546." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Sørensen HM, Rochfort KD, Maye S, MacLeod G, Brabazon D, Loscher C, Freeland B. Exopolysaccharides of lactic acid bacteria: production, purification and health benefits towards functional food. Nutrients. 2022;14(14):2938.
</cite> [<a href="https://doi.org/10.3390/nu14142938" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9319976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35889895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?S%C3%B8rensen%20HM,%20Rochfort%20KD,%20Maye%20S,%20MacLeod%20G,%20Brabazon%20D,%20Loscher%20C,%20Freeland%20B.%20Exopolysaccharides%20of%20lactic%20acid%20bacteria:%20production,%20purification%20and%20health%20benefits%20towards%20functional%20food.%20Nutrients.%202022;14(14):2938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Zhang M, Zhao D, Yang H, Jiao X, Zhou R, Zheng J, Wu C. Lactic acid bacteria-derived exopolysaccharide: biosynthesis and antibacterial characterization. Trends Food Sci Technol. 2025;160:105033.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhang%20M,%20Zhao%20D,%20Yang%20H,%20Jiao%20X,%20Zhou%20R,%20Zheng%20J,%20Wu%20C.%20Lactic%20acid%20bacteria-derived%20exopolysaccharide:%20biosynthesis%20and%20antibacterial%20characterization.%20Trends%20Food%20Sci%20Technol.%202025;160:105033." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374427/bin/12866_2025_4286_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (64KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374427/bin/12866_2025_4286_MOESM2_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (252.4KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374427/bin/12866_2025_4286_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3.</a><sup> (11.5KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374427/bin/12866_2025_4286_MOESM4_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4.</a><sup> (20.6KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The KM7 genome data were deposited at the NCBI database, with the BioProject accession number: PRJNA1271779.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Microbiology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12866-025-04286-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12866_2025_Article_4286.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12374427/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12374427/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374427%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374427/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12374427/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12374427/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40847281/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12374427/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40847281/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12374427/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12374427/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="BLL6Hya8lm0z2hi6Pobh3KnTFFumlbNQ0cXhXgSzufjYVsAe8PFjMkP3fe2mvMt2">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
