
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            KYNU is a potential metabolic-related biomarker for nasopharyngeal carcinoma by Raman spectroscopy, metabolomics, and transcriptomics analysis - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532BBD8AF1D5D3052BBD0031AEB227.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="discon">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373589/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Discover Oncology">
<meta name="citation_title" content="KYNU is a potential metabolic-related biomarker for nasopharyngeal carcinoma by Raman spectroscopy, metabolomics, and transcriptomics analysis">
<meta name="citation_author" content="Ziman Wu">
<meta name="citation_author_institution" content="School of Medical Technology, Xinxiang Medical University, Xinxiang, 453003 China">
<meta name="citation_author_institution" content="Department of Clinical Laboratory Medicine, The Third Affiliated Hospital of Shenzhen University, Shenzhen, 518001 China">
<meta name="citation_author" content="Haiyan Yang">
<meta name="citation_author_institution" content="Shantou University Medical College, Shantou, 515041 China">
<meta name="citation_author_institution" content="Department of Clinical Laboratory Medicine, The Third Affiliated Hospital of Shenzhen University, Shenzhen, 518001 China">
<meta name="citation_author" content="Yafei Xu">
<meta name="citation_author_institution" content="Department of Cell Biology and Genetics, Shenzhen University Health Science Center, Shenzhen, 518060 China">
<meta name="citation_author" content="Xiang Ji">
<meta name="citation_author_institution" content="Department of Clinical Laboratory Medicine, The Third Affiliated Hospital of Shenzhen University, Shenzhen, 518001 China">
<meta name="citation_author" content="Dayang Chen">
<meta name="citation_author_institution" content="Department of Clinical Laboratory Medicine, The Third Affiliated Hospital of Shenzhen University, Shenzhen, 518001 China">
<meta name="citation_author" content="Chuang Zhang">
<meta name="citation_author_institution" content="School of Medicine, Anhui University of Science and Technology, Huainan, 232000 China">
<meta name="citation_author" content="Mingjie Liang">
<meta name="citation_author_institution" content="Shantou University Medical College, Shantou, 515041 China">
<meta name="citation_author" content="Xinying Li">
<meta name="citation_author_institution" content="School of Medicine, Anhui University of Science and Technology, Huainan, 232000 China">
<meta name="citation_author" content="Xiuming Zhang">
<meta name="citation_author_institution" content="School of Medical Technology, Xinxiang Medical University, Xinxiang, 453003 China">
<meta name="citation_author_institution" content="Shantou University Medical College, Shantou, 515041 China">
<meta name="citation_author_institution" content="Department of Clinical Laboratory Medicine, The Third Affiliated Hospital of Shenzhen University, Shenzhen, 518001 China">
<meta name="citation_author_institution" content="School of Medicine, Anhui University of Science and Technology, Huainan, 232000 China">
<meta name="citation_author_institution" content="Medical Center of Shenzhen Luohu Hospital Group, 33 Zhongyuan Road, Buji Street, Longgang District, Shenzhen City, Guangdong Province China">
<meta name="citation_author" content="Dan Xiong">
<meta name="citation_author_institution" content="Shantou University Medical College, Shantou, 515041 China">
<meta name="citation_author_institution" content="Department of Clinical Laboratory Medicine, The Third Affiliated Hospital of Shenzhen University, Shenzhen, 518001 China">
<meta name="citation_author_institution" content="School of Medicine, Anhui University of Science and Technology, Huainan, 232000 China">
<meta name="citation_author_institution" content="Medical Center of Shenzhen Luohu Hospital Group, 33 Zhongyuan Road, Buji Street, Longgang District, Shenzhen City, Guangdong Province China">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="1595">
<meta name="citation_doi" content="10.1007/s12672-025-03349-7">
<meta name="citation_pmid" content="40844682">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373589/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373589/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373589/pdf/12672_2025_Article_3349.pdf">
<meta name="description" content="Nasopharyngeal carcinoma (NPC) is a malignant tumor with high incidence in Southeast Asia and Southern China, characterized by difficulties in early diagnosis and high recurrence rates after treatment. Metabolic reprogramming plays a crucial role in ...">
<meta name="og:title" content="KYNU is a potential metabolic-related biomarker for nasopharyngeal carcinoma by Raman spectroscopy, metabolomics, and transcriptomics analysis">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Nasopharyngeal carcinoma (NPC) is a malignant tumor with high incidence in Southeast Asia and Southern China, characterized by difficulties in early diagnosis and high recurrence rates after treatment. Metabolic reprogramming plays a crucial role in ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373589/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373589">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1007/s12672-025-03349-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12672_2025_Article_3349.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373589%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373589/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373589/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373589/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-discon.png" alt="Discover Oncology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Discover Oncology" title="Link to Discover Oncology" shape="default" href="https://doi.org/10.1007/s12672-025-03349-7" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Discov Oncol</button></div>. 2025 Aug 22;16:1595. doi: <a href="https://doi.org/10.1007/s12672-025-03349-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1007/s12672-025-03349-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Discov%20Oncol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Discov%20Oncol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Discov%20Oncol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Discov%20Oncol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>KYNU is a potential metabolic-related biomarker for nasopharyngeal carcinoma by Raman spectroscopy, metabolomics, and transcriptomics analysis</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Ziman Wu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Ziman Wu</span></h3>
<div class="p">
<sup>1</sup>School of Medical Technology, Xinxiang Medical University, Xinxiang, 453003 China </div>
<div class="p">
<sup>3</sup>Department of Clinical Laboratory Medicine, The Third Affiliated Hospital of Shenzhen University, Shenzhen, 518001 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Ziman Wu</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Haiyan Yang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Haiyan Yang</span></h3>
<div class="p">
<sup>2</sup>Shantou University Medical College, Shantou, 515041 China </div>
<div class="p">
<sup>3</sup>Department of Clinical Laboratory Medicine, The Third Affiliated Hospital of Shenzhen University, Shenzhen, 518001 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haiyan Yang</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yafei Xu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yafei Xu</span></h3>
<div class="p">
<sup>5</sup>Department of Cell Biology and Genetics, Shenzhen University Health Science Center, Shenzhen, 518060 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yafei Xu</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ji%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Xiang Ji</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Xiang Ji</span></h3>
<div class="p">
<sup>3</sup>Department of Clinical Laboratory Medicine, The Third Affiliated Hospital of Shenzhen University, Shenzhen, 518001 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ji%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiang Ji</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Dayang Chen</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Dayang Chen</span></h3>
<div class="p">
<sup>3</sup>Department of Clinical Laboratory Medicine, The Third Affiliated Hospital of Shenzhen University, Shenzhen, 518001 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dayang Chen</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Chuang Zhang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Chuang Zhang</span></h3>
<div class="p">
<sup>4</sup>School of Medicine, Anhui University of Science and Technology, Huainan, 232000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chuang Zhang</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Mingjie Liang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Mingjie Liang</span></h3>
<div class="p">
<sup>2</sup>Shantou University Medical College, Shantou, 515041 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mingjie Liang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Xinying Li</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Xinying Li</span></h3>
<div class="p">
<sup>4</sup>School of Medicine, Anhui University of Science and Technology, Huainan, 232000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xinying Li</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Xiuming Zhang</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Xiuming Zhang</span></h3>
<div class="p">
<sup>1</sup>School of Medical Technology, Xinxiang Medical University, Xinxiang, 453003 China </div>
<div class="p">
<sup>2</sup>Shantou University Medical College, Shantou, 515041 China </div>
<div class="p">
<sup>3</sup>Department of Clinical Laboratory Medicine, The Third Affiliated Hospital of Shenzhen University, Shenzhen, 518001 China </div>
<div class="p">
<sup>4</sup>School of Medicine, Anhui University of Science and Technology, Huainan, 232000 China </div>
<div class="p">
<sup>6</sup>Medical Center of Shenzhen Luohu Hospital Group, 33 Zhongyuan Road, Buji Street, Longgang District, Shenzhen City, Guangdong Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiuming Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>6,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiong%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Dan Xiong</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Dan Xiong</span></h3>
<div class="p">
<sup>2</sup>Shantou University Medical College, Shantou, 515041 China </div>
<div class="p">
<sup>3</sup>Department of Clinical Laboratory Medicine, The Third Affiliated Hospital of Shenzhen University, Shenzhen, 518001 China </div>
<div class="p">
<sup>4</sup>School of Medicine, Anhui University of Science and Technology, Huainan, 232000 China </div>
<div class="p">
<sup>6</sup>Medical Center of Shenzhen Luohu Hospital Group, 33 Zhongyuan Road, Buji Street, Longgang District, Shenzhen City, Guangdong Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiong%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dan Xiong</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>6,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>School of Medical Technology, Xinxiang Medical University, Xinxiang, 453003 China </div>
<div id="Aff2">
<sup>2</sup>Shantou University Medical College, Shantou, 515041 China </div>
<div id="Aff3">
<sup>3</sup>Department of Clinical Laboratory Medicine, The Third Affiliated Hospital of Shenzhen University, Shenzhen, 518001 China </div>
<div id="Aff4">
<sup>4</sup>School of Medicine, Anhui University of Science and Technology, Huainan, 232000 China </div>
<div id="Aff5">
<sup>5</sup>Department of Cell Biology and Genetics, Shenzhen University Health Science Center, Shenzhen, 518060 China </div>
<div id="Aff6">
<sup>6</sup>Medical Center of Shenzhen Luohu Hospital Group, 33 Zhongyuan Road, Buji Street, Longgang District, Shenzhen City, Guangdong Province China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 15; Accepted 2025 Jul 29; Collection date 2025 Dec.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the articleâ€™s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373589Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40844682/" class="usa-link">40844682</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Nasopharyngeal carcinoma (NPC) is a malignant tumor with high incidence in Southeast Asia and Southern China, characterized by difficulties in early diagnosis and high recurrence rates after treatment. Metabolic reprogramming plays a crucial role in the development and progression of tumors. In-depth studies on the metabolic characteristics and molecular mechanisms of NPC are essential to identify novel diagnostic and therapeutic targets.</p></section><section id="sec2"><h3 class="pmc_sec_title">Objectives</h3>
<p id="Par2">This study aimed to systematically reveal the metabolic characteristics and molecular mechanisms of NPC cell lines by integrating untargeted metabolomics, transcriptomics, and confocal micro-Raman spectroscopy (CMRS), and to explore potential biomarkers for prognostic evaluation and precision treatment of NPC. Methods: We performed an integrated analysis of transcriptomic, metabolomic, and Raman spectral data on five NPC cell lines (CNE1, CNE2, 5â€“8Â F, 6-10B, and SUNE1) and the immortalized nasopharyngeal epithelial cell line NPEC1-BMI1. The analysis included association analysis of differentially expressed metabolites (DEMs) and differentially expressed genes (DEGs), pathway enrichment analysis, and network analysis to elucidate the interplay between gene expression and metabolic alterations. Furthermore, we employed machine learning models to achieve efficient discrimination between NPC cell lines and NPEC1-BMI1 using Raman spectroscopy. Finally, we validated the expression levels of selected DEGs using quantitative polymerase chain reaction (qPCR), Western blotting (WB), and immunohistochemistry (IHC).</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">Significant differences in metabolic and gene expression profiles were observed between NPC cells and normal cells. CMRS analysis, combined with a multilayer perceptron (MLP) model, achieved high-precision discrimination between NPC cells and normal cells (accuracy 99.3%, AUCâ€‰=â€‰1.00). Further integrated analysis revealed significant correlations between KYNU and other DEGs, multiple DEMs, and specific Raman spectral features, suggesting their potential as diagnostic and prognostic biomarkers. Validation using the The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) databases showed high KYNU expression in head and neck squamous cell carcinoma (HNSCC) and NPC tissues. Consistent high expression of KYNU was confirmed in NPC cell lines and tissues by qPCR, WB, and IHC.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par4">This study elucidated the unique metabolic characteristics and molecular signatures of NPC, clarified how molecular changes regulate gene expression, and provided new potential targets for prognostic evaluation and precision treatment of NPC.</p></section><section id="sec5"><h3 class="pmc_sec_title">Graphical abstract</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373589_12672_2025_3349_Figa_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b755/12373589/afbea4c6348f/12672_2025_3349_Figa_HTML.jpg" loading="lazy" id="d33e293" height="653" width="714" alt="graphic file with name 12672_2025_3349_Figa_HTML.jpg"></a></p></section><section id="sec6"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1007/s12672-025-03349-7.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Nasopharyngeal carcinoma, Multi-omics, Metabolomics, Transcriptomics, Confocal micro-Raman spectroscopy, Machine learning</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par6">NPC primarily originates from the pharyngeal recess of the nasopharynx, with over 95% of cases classified as non-keratinizing squamous cell carcinoma [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. It exhibits a significant geographical distribution, with over 75% of cases occurring in Southeast Asia and Southern China [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Epstein-Barr virus (EBV) infection and genetic susceptibility are the main pathogenic factors of NPC. Specifically, EBV infection interacts with genetic susceptibility through multiple mechanisms to promote NPC development. Firstly, genetic factors, such as specific allelic variations in human leukocyte antigen (HLA) genes, influence individual susceptibility and immune response to EBV infection. For instance, certain HLA alleles are associated with an increased risk of NPC [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Secondly, upon EBV infection, the virus can alter host cell gene expression and signaling pathways, such as activating the PI3K/Akt/mTOR and NF-ÎºB pathways through the viral protein LMP1, thereby promoting tumor cell growth and proliferation [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Furthermore, EBV infection may induce epigenetic changes in host cells, such as DNA methylation and histone modification, further affecting gene expression [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Other factors, such as environmental exposures (e.g., diet, smoking), also contribute to NPC development [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Despite significant advances in NPC treatment, including chemotherapy, radiotherapy, targeted therapy, and immunotherapy, challenges remain, such as drug resistance, tumor heterogeneity, frequent recurrence and metastasis, and a narrow therapeutic window, which often limit treatment efficacy [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Tumor cell metabolism is closely related to NPC cell growth, proliferation, invasion, and metastasis [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>â€“<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Multiple studies have shown that tumor cells frequently undergo metabolic reprogramming to adapt to the challenging tumor microenvironment [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>].</p>
<p id="Par7">Metabolomics, as a rapidly evolving analytical tool, has demonstrated significant potential in elucidating tumor metabolic phenotypes, aiding in diagnosis, and predicting prognosis [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. By systematically analyzing metabolites within biological organisms, metabolomics reveals notable metabolic differences between tumor cells and normal cells. For instance, tumor cells frequently exhibit the Warburg effect, a metabolic reprogramming that provides energy and biosynthetic building blocks for rapid proliferation [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Studies have unveiled significant metabolic alterations in NPC cells compared to normal cells, offering valuable insights into the underlying mechanisms of NPC development and progression [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. In-depth transcriptomic analyses of NPC patients have identified key genes associated with tumorigenesis, progression, invasion, and metastasis. Aberrant gene expression drives tumor malignancy and significantly influences tumor cell metabolism by regulating the expression of metabolic enzymes and transporters [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Integrating metabolomics and transcriptomics can further elucidate the specific roles of these genes in modulating tumor metabolic networks.</p>
<p id="Par8">CMRS is an emerging non-invasive spectroscopic technique that obtains the â€œmolecular fingerprintâ€ of cells by detecting Raman scattered light generated from molecular vibrations [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Compared to traditional radiological and histological methods, CMRS enable more direct detection of intracellular biomolecular changes, revealing cellular metabolic states. For example, CMRS can sensitively detect changes in biomolecules such as DNA, proteins, and lipids, as well as metabolites like glucose and lactate [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. Thus, CMRS provide a novel perspective for studying cellular functions and diseases. However, CMRS also present limitations and potential challenges. Firstly, the Raman scattering signal is weak and susceptible to fluorescence interference, leading to a reduced signal-to-noise ratio. Secondly, sample preparation processes may introduce artifacts, affecting the accuracy of the spectra. Additionally, CMRS data analysis is complex, requiring specialized knowledge and experience. Lastly, the high cost of equipment and limited penetration depth also restrict its application scope [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>].</p>
<p id="Par9">Although reports on the CMRS spectral characteristics of NPC at the single-cell level are limited, studies have demonstrated that CMRS, by analyzing cellular biomolecular vibrational information and integrating machine learning models, can effectively distinguish between normal and tumor cells. A novel blood test using Raman spectroscopy has been proposed for the detection of NPC. By examining blood samples from NPC patients and healthy individuals, the results revealed high sensitivity and specificity, effectively differentiating NPC from normal samples, with similar detection efficacy observed at two wavelengths. This suggests the potential application of Raman spectroscopy-based blood tests in NPC detection [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. This approach offers a promising avenue for the screening and validation of NPC-related biomarkers [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>].</p>
<p id="Par10">Advances in technologies such as metabolomics, transcriptomics, and Raman spectroscopy have provided powerful tools for studying tumor metabolism. By systematically analyzing the metabolism of NPC cells, we can gain deeper insights into their molecular mechanisms, laying the foundation for improving the prognosis of NPC patients. In recent years, advances in omics technologies have opened new avenues for cancer biomarker research. Multi-omics technologies have been widely applied in common cancers such as lung cancer [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>] and breast cancer [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>], achieving significant progress. Currently, metabolomics and transcriptomics studies of NPC cells still have significant limitations and data scarcity, severely hindering our comprehensive understanding of NPC metabolic characteristics [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. Specifically, in transcriptomics, there is insufficient functional validation and clinical application of miRNA, and lncRNA and RNA sequencing studies lack widely recognized biomarkers [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. In metabolomics, the application of high-throughput technologies is lagging, and clinical validation and long-term follow-up data are insufficient [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Therefore, there is an urgent need to conduct systematic multi-omics studies, using methods such as technological innovation, clinical validation, optimized design, and multi-omics integration, to deeply reveal the metabolic mechanisms of NPC and provide new scientific evidence for the early diagnosis and precise treatment of NPC. Integrating metabolomic and transcriptomic analyses, combined with real-time metabolite information provided by Raman spectroscopy, can more comprehensively reveal the molecular changes in NPC. Transcriptomic data provide information at the gene expression level, metabolomic data reflect the actual levels of metabolites, and Raman spectroscopy directly captures the dynamic changes of intracellular metabolic activities. The combination of these three can reveal how changes in gene expression affect metabolic pathways and how metabolites regulate gene expression, thereby providing more effective strategies for the screening and validation of NPC-related biomarkers.</p>
<p id="Par11">This study aimed to elucidate the changes in metabolomics, transcriptomics, and CMRS in NPC cell lines and to validate the impact of these changes on gene expression. We hypothesized that integrating untargeted metabolomics, transcriptomics, and CMRS analyses could identify DEGs specifically associated with NPC and elucidate how molecular changes drive alterations in gene expression. Specifically, we performed untargeted metabolomics, transcriptomics, and CMRS analyses on five NPC cell lines (CNE1, CNE2, 5â€“8Â F, 6-10B, and SUNE1) and the immortalized nasopharyngeal epithelial cell line NPEC1-BMI1 to identify DEMsand DEGs. Subsequently, we analyzed the relative expression levels of DEGs significantly correlated with DEMs in HNSCC using The TCGA database and validated these findings in the GEO database. To validate the roles of the screened DEGs in NPC, we employed qPCR and WB techniques, which revealed high expression of KYNU in NPC cell lines. Furthermore, we successfully validated the high expression of KYNU on NPC tissue microarrays. By systematically analyzing metabolomic, transcriptomic, and CMRS data, combined with validation, we not only revealed the unique molecular characteristics and heterogeneity of NPC but also elucidated how molecular changes regulate gene expression, providing new insights into the pathogenesis of NPC. Looking forward, we will further investigate the functional roles of these key DEGs in the development and progression of NPC and explore their potential as therapeutic targets. We believe that this study lays the foundation for precision treatment strategies for NPC and provides an important reference for future research directions.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Cell culture</h3>
<p id="Par12">The development and progression of NPC are multifactorial processes, with EBV infection being one of the most significant etiological factors. This study utilized a variety of NPC cell lines, including EBV-negative cell lines (SUNE1, 5â€“8Â F, 6-10B, CNE1, CNE2, HK1, HONE1) and EBV-positive cell lines (CNE2-EBV, HK1-EBV). The inclusion of both EBV-positive and EBV-negative cell lines aimed to investigate the impact of EBV infection status on the biological characteristics and potential differences of NPC cells, given the critical role of EBV infection in NPC pathogenesis.</p>
<p id="Par13">The immortalized nasopharyngeal epithelial cell lines NP69 and NPEC1-BMI1 were cultured as non-cancer controls under the following conditions. NP69 cells were maintained in Keratinocyte Serum-Free Medium (KSFM) supplemented with epidermal growth factor and bovine pituitary extract. NPEC1-BMI1 cells, immortalized nasopharyngeal epithelial cells induced by Bmi-1, were cultured in KSFM (Invitrogen). All NPC cell lines were cultured in RPMI 1640 medium (GIBCO) supplemented with 5% fetal bovine serum (FBS, GIBCO). All cells were cultured at 37Â Â°C in a humidified incubator with 5% CO2. Among these NPC cell lines, 5â€“8Â F exhibited high metastatic potential, 6-10B exhibited low metastatic potential, CNE1 was a well-differentiated squamous cell line, and CNE2 was a poorly differentiated squamous cell line. All cell lines mentioned above were kindly provided by Professor Musheng Zeng from Sun Yat-sen University Cancer Center.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Ultra-performance liquid chromatography-tandem mass spectrometry (UPLC-MS/MS) metabolomics assay and data analysis</h3>
<p id="Par14">Untargeted metabolomics analysis was performed as previously described [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] at the Calibra Metabolomics laboratory (DlAN Diagnostics, Hangzhou, Zhejiang, China). Small-molecule metabolites were extracted using 500ÂµL of 80% methanol extraction buffer, followed by centrifugation. The analysis was conducted using four different UPLC-MS/MS methods on a Waters ACQUITY 2D UPLC system coupled to a Thermo Fisher Q Exactive mass spectrometer. The chromatographic separations employed C18 reversed-phase (UPLC BEH C18, 2.1â€‰Ã—â€‰100Â mm, 1.7Â Î¼m) and HILIC (UPLC BEH Amide, 2.1â€‰Ã—â€‰150Â mm, 1.7Â Î¼m) columns. For each method, samples were dried under nitrogen and reconstituted in a suitable solvent prior to injection. The QE mass spectrometer was operated in full scan mode over a mass range of 70â€“1000Â m/z with a resolution of 35,000.</p>
<p id="Par15">Raw mass spectrometry data were pre-processed using proprietary in-house metabolomics software, including ion feature extraction and quality control. Metabolite identification was performed by comparing the data to an in-house reference library, requiring matches in chromatographic retention index, ion mass accuracy within 10 ppm, and high spectral similarity scores to achieve MSI Tier 1 confidence levels. Identified metabolites were median-normalized, log2-transformed, and statistically analyzed using Orthogonal Partial Least Squares Discriminant Analysis (OPLS-DA) to classify NPEC1-BMI1 cells and NPC cells. OPLS-DA was chosen for its efficacy in maximizing inter-group separation and providing VIP scores for key metabolite identification. DEMs were identified based on VIP valuesâ€‰&gt;â€‰1.0 and <em>P</em>â€‰&lt;â€‰0.05, determined through t-tests or non-parametric statistical tests. Random forest (RF) analysis was also employed to identify potential biomarkers in NPC cells compared to NPEC1-BMI1 cells. RF was selected for its ability to handle high-dimensional data, robustness against overfitting, and feature importance assessment for biomarker identification. The original mass spectrometry data are available in Supplementary Raw Data 1.</p></section><section id="Sec5"><h3 class="pmc_sec_title">RNA extraction, transcriptome sequencing, and data analysis</h3>
<p id="Par16">Total RNA was extracted using TRIzol Reagent (Thermo Fisher Scientific, 15596018), and samples with an RNA Integrity Number (RIN) greater than 7.0 were assessed using a Bioanalyzer 2100. 5Â Âµg of RNA was purified using Dynabeads Oligo(dT) beads (Thermo Fisher Scientific) and fragmented at 94Â Â°C using the Magnesium RNA Fragmentation Module. cDNA was synthesized using SuperScript II Reverse Transcriptase (Invitrogen, cat. 1896649) and ligated to a bi-exponential adapter. The cDNA library was amplified by PCR (95Â Â°C for 3Â min, followed by 8 cycles of 98Â Â°C for 15Â s, 60Â Â°C for 15Â s, and 72Â Â°C for 30Â s) to generate an average insert size of 300â€‰Â±â€‰50Â bp. The library was then sequenced on an Illumina Novaseq 6000 platform (LC-Bio Technology CO., Ltd, Hangzhou, China) using 2â€‰Ã—â€‰150Â bp paired-end sequencing. Differential gene expression analysis was performed between NPC cell lines and NPEC1-BMI1 cells using both DESeq2 and edgeR software. Genes with an adjusted p-value (FDR) less than 0.05 and an absolute fold change greater than or equal to 2 were considered differentially expressed. Gene Ontology (GO) term and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analyses were performed on these DEGs. In this study, GO reference gene set was obtained from the official website of the Gene Ontology Consortium (<a href="https://geneontology.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://geneontology.org/</a>), and the KEGG reference gene set was obtained from the official website of KEGG (<a href="https://www.kegg.jp/).Th" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.kegg.jp/).Th</a>e original transcriptome data are available in Supplementary Raw Data 2.</p></section><section id="Sec6"><h3 class="pmc_sec_title">CMRS detection and single-cell Raman spectroscopy pretreatment</h3>
<p id="Par17">NPEC1-BMI1 and NPC cells were dissociated into single cells, washed once with PBS, fixed with 4% paraformaldehyde for 15Â min, and washed three times with PBS. Cell suspensions were air-dried on aluminum Raman slides, preserving cellular morphology. Single-cell Raman spectra were acquired using a Witec A300 confocal Raman spectrometer equipped with a 532Â nm laser and a 100Ã—/0.9 NA objective lens. The laser power was set to 12 mW. The spectrometer was calibrated in silico, with the Raman peak located at 520.73Â cmâ»Â¹. Spectra were collected over a spectral range of 279â€“2187Â cmâ»Â¹ using a 1200 lines/mm diffraction grating with a 9-second integration time. Five spectra were acquired from different positions on each cell, resulting in a total of 40 single-cell spectra per sample. Three technical replicates and three biological replicates were performed for each sample. After quality control, a total of 5685 single-cell Raman spectra were obtained, including 846 from NPEC1-BMI1 cells, 964 from 5 to 8Â F cells, 1006 from 6-10B cells, 1023 from SUNE1 cells, 820 from CNE1 cells, and 1026 from CNE2 cells. All Raman spectroscopy data are available in Supplementary Raw Data 3.</p>
<p id="Par18">Raman spectra were pre-processed using Labspec6 software (Horiba). The spectral range was limited to 279â€“2187Â cmâ»Â¹ to remove cosmic ray spikes and reduce background noise. Baseline correction was performed using a 10th-order polynomial fitting algorithm. Spectral normalization was achieved by dividing each spectrum by its total area. Biomolecule quantification was performed by integrating the area under the corresponding Raman bands. Unpaired two-sided t-tests were used to compare the mean values of different samples, and p-values were adjusted for multiple comparisons using the Bonferroni correction. Significantly different wavenumbers and their corresponding biomolecular structures are listed in TableÂ <a href="#Tab1" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Biological attribution of Raman wave numbers</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Raman wavenumber (cm<sup>âˆ’â€‰1</sup>)</th>
<th align="left" colspan="1" rowspan="1">Biomolecule assignment</th>
<th align="left" colspan="1" rowspan="1">References</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">405</td>
<td align="left" colspan="1" rowspan="1">Glucose</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">525</td>
<td align="left" colspan="1" rowspan="1">DNA/RNA backbone vibrations, phosphodiester bond stretching or bending</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">666</td>
<td align="left" colspan="1" rowspan="1">Guanine, Thymine (ring breathing modes in the DNA bases); tyrosine-G backbone in RNA</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">780</td>
<td align="left" colspan="1" rowspan="1">Cytosine/uracil ring breathing (nucleotide)</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">860</td>
<td align="left" colspan="1" rowspan="1">Tryptophan</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">876</td>
<td align="left" colspan="1" rowspan="1">Hydroxyproline</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">900</td>
<td align="left" colspan="1" rowspan="1">Saccharides</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">934</td>
<td align="left" colspan="1" rowspan="1">Proline, valine</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1125</td>
<td align="left" colspan="1" rowspan="1">Amide III vibrations of proteins</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1247</td>
<td align="left" colspan="1" rowspan="1">Guanine, cytosine (NH2)</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1265</td>
<td align="left" colspan="1" rowspan="1">Unsaturated lipids</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1305</td>
<td align="left" colspan="1" rowspan="1">Amide III vibrations of proteins</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1440</td>
<td align="left" colspan="1" rowspan="1">CH2 bending mode of proteins and lipids</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1580</td>
<td align="left" colspan="1" rowspan="1">C-C stretching</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1603</td>
<td align="left" colspan="1" rowspan="1">C=C in-plane bending mode of phenylalanine and tyrosine</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1616</td>
<td align="left" colspan="1" rowspan="1">C=C stretching mode of tyrosine and tryptophan</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1655</td>
<td align="left" colspan="1" rowspan="1">Amide I vibrations of proteins</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par20">Considering the potential for high dimensionality, non-linearity, and complex inter-class distribution characteristics inherent in Raman spectroscopy data, this study selected and comparatively analyzed a variety of machine learning models to identify the most suitable classification method for this type of data. The models investigated include MLP, Gated Recurrent Unit (GRU), K-Nearest Neighbors (KNN), Gradient Boosting (GB), Linear Support Vector Machine (LSVM), Quadratic Discriminant Analysis (QDA), RF, Linear Discriminant Analysis (LDA), Logistic Regression (LR), and Naive Bayes (NB). All model analyses were implemented using the Python 3.9 programming language. The libraries utilized encompassed, but were not limited to: SciPy for statistical analysis, Scikit-learn for machine learning models, TensorFlow and PyTorch for deep learning models, and Matplotlib and Seaborn for data visualization.</p>
<p id="Par21">For non-image or sequence models, feature data were initially subjected to Principal Component Analysis (PCA) for dimensionality reduction. This step aimed to remove redundant information and ensure the independence of variables, thereby enhancing model training efficiency and generalization capability. Raman spectroscopy data typically encompass a substantial number of correlated wavelength features; PCA can effectively extract the principal components of variance, reduce feature dimensionality, mitigate the risk of overfitting, and provide more stable inputs for subsequent linear models.</p>
<p id="Par22">Subsequently, we employed the following traditional machine learning models: LSVM, suitable for high-dimensional data and typically exhibiting good generalization capability on small sample datasets, was adopted as one of the baseline models in this study to examine the classification performance of spectral data under linearly separable conditions. The ability of LSVM to identify the optimal hyperplane in high-dimensional space theoretically enables it to effectively distinguish linearly separable or approximately linearly separable spectral data. LDA, a linear model that performs classification concurrently with dimensionality reduction, is appropriate for data where class discrimination primarily relies on linear decision boundaries. It was used to evaluate the classification efficacy of spectral data under the assumption that the data follow a normal distribution and each class shares a common covariance matrix. The selection of LDA aimed to investigate whether significant linear separability exists between the classes of Raman spectroscopy data under conditions where its assumptions are met. QDA, an extension of LDA that permits different classes to possess distinct covariance matrices, can handle more complex non-linear data distributions, particularly suitable for scenarios where the distribution shapes of spectral features from different cell types are expected to exhibit significant variations. QDA was employed to explore the modelâ€™s classification performance when the distribution of Raman spectroscopy data from different classes demonstrates evident non-linearity and distinct covariance structures. RF, a robust ensemble learning method that constructs multiple decision trees and performs voting, can effectively process high-dimensional data and exhibits good robustness to noise and missing values, making it suitable for investigating classification capabilities under complex spectral features. RF is capable of handling high-dimensional and non-linear data, and possesses certain advantages in feature selection and feature importance assessment, aiding in the understanding of which wavelength ranges are crucial for classification. GB is another ensemble learning method that enhances model performance by sequentially constructing residuals of multiple weak learners (typically decision trees). It excels in handling non-linear problems and complex data structures, making it suitable for extracting subtle discriminatory features within spectral data. GB often achieves superior performance in complex, non-linear classification tasks and can capture finer patterns in the data. LR is a simple yet effective linear classification model, particularly applicable to binary classification problems. It was used to evaluate the classification performance of PCA-reduced spectral data under approximately linearly separable conditions and to provide insights into the relationships between features and classes. Serving as a complement to linear models, LR investigates whether the reduced-dimensional data can be effectively distinguished by a linear model and analyzes the strength and direction of the relationships between features and classes. NB is a probabilistic classifier based on Bayesâ€™ theorem. Despite its assumption of independence between features, it is computationally efficient and can achieve reasonable classification results in scenarios with a large number of features but a relatively small sample size, serving as a simple and fast probabilistic model for comparison. The high computational efficiency of NB and its unexpectedly good performance on certain high-dimensional, small sample datasets make it a useful rapid baseline model. KNN is a non-parametric, instance-based learning algorithm whose classification decisions are based on neighborhood voting and does not make any assumptions about the underlying data distribution, making it suitable for identifying local structures within the data. Given that Raman spectroscopy data may exhibit complex local patterns, the selection of KNN aimed to explore whether classification patterns based on local similarity exist in the data without relying on specific distribution assumptions.</p>
<p id="Par23">In contrast, for spectral data exhibiting potential sequential characteristics, we employed deep learning models. GRU, a type of recurrent neural network particularly well-suited for processing sequential data, can capture potential long-range dependencies within the spectral data, which may be crucial for distinguishing subtly different spectral features. The selection of GRU was motivated by the consideration that Raman spectroscopy data might, in certain contexts, possess â€œsequentialâ€ information. For instance, the specific arrangement patterns of spectral peaks or the data acquisition process itself could contain class-discriminative information, and the recurrent structure of GRU enables it to capture such sequential dependencies. MLP, a fundamental feedforward neural network with strong non-linear fitting capabilities, can learn complex combinations of PCA-reduced spectral features, serving as a baseline for comparing the performance of deep learning models. The choice of MLP was based on its status as a classic and powerful non-linear model capable of learning intricate feature representations from high-dimensional data, thus providing a comparative benchmark for evaluating the performance of deep learning models.</p>
<p id="Par24">Traditional machine learning models underwent hyperparameter optimization via GridSearchCV (cvâ€‰=â€‰5), and the selected optimal parameters were used for prediction on the test set. Deep learning models, including MLP and GRU, were fine-tuned based on common architectures, with a validation split ratio of validation_splitâ€‰=â€‰0.2. All models were trained with a batch size of 128 and a learning rate of 0.001, employing the ReduceLROnPlateau strategy for automatic learning rate adjustment to enhance training efficiency. The single-cell Raman spectroscopy data were randomly partitioned into training and test sets at an 80:20 ratio to evaluate model performance and generalization capability. Receiver Operating Characteristic (ROC) curves were generated using the test set, and the Area Under the Curve (AUC) was calculated. Sensitivity, specificity, and overall accuracy were computed using the binary classification confusion matrix, according to the following formulas: Sensitivityâ€‰=â€‰TP / (TPâ€‰+â€‰FN), Specificityâ€‰=â€‰TN / (TNâ€‰+â€‰FP), Overall Accuracy = (TNâ€‰+â€‰TP) / (TNâ€‰+â€‰FPâ€‰+â€‰FNâ€‰+â€‰TP), where TP represents true positives, TN represents true negatives, FP represents false positives, and FN represents false negatives.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Differential metabolite correlation analysis with differential genes and Raman spectroscopy</h3>
<p id="Par25">This study used transcriptomics and untargeted metabolomics analyses to investigate gene expression and metabolic differences between NPEC1-BMI1 and NPC cells. DEGs and DEMs were identified and mapped to the KEGG pathway database to identify shared pathways. The top 32 DEGs and DEMs from these shared pathways were selected for further analysis. Spearman correlation coefficients were calculated between these top 32 DEGs and DEMs using R (4.1.3), and the results were visualized as cluster heatmaps. Additionally, Spearman correlation coefficients were calculated between these top 32 DEGs and biologically significant Raman spectral features, and the results were also visualized as cluster heat maps.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Differential gene expression in NPC and HNSCC</h3>
<p id="Par26">Spearman correlation analysis was employed to identify DEGs that regulate multiple metabolites, potentially serving as biomarkers. TCGA database (<a href="http://portal.gbc.cancer.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://portal.gbc.cancer.gov/</a>) is a comprehensive cancer genomics data resource encompassing genomic, transcriptomic, epigenetic, and proteomic information for 33 cancer types. We utilized the UALCAN portal (<a href="http://ualcan.path.uab/index.hrml" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://ualcan.path.uab/index.hrml</a>) to analyze the online expression of these target genes in HNSCC within the TCGA database. Additionally, the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE53819" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE53819</a> microarray dataset (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL6480" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GPL6480</a> platform) from the GEO database (<a href="https://www.ncbi.nlm.nih.gov/geo/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/</a>) was used to validate the expression of these target genes in NPC. This dataset includes 18 tumor tissues and 18 normal tissues.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Immunohistochemistry analysis</h3>
<p id="Par27">The NPC tissue microarray used in this study was provided by the Sun Yat-sen University Cancer Center and contained tumor tissue samples from 157 NPC patients. IHC staining was performed using a KYNU polyclonal antibody (Wuhan Fine Biotech Co., Ltd.). Detailed experimental procedures and reagent information are provided in Supplementary documentation 2.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Western blot analysis</h3>
<p id="Par28">Following cell lysis, total protein was extracted and subjected to Western blot analysis. The protein expression level of KYNU was detected using a KYNU polyclonal antibody (Wuhan Fine Biotech Co., Ltd.), with GAPDH serving as an internal control. Detailed experimental procedures and reagent information are provided in Supplementary documentation 2.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Quantitative polymerase chain reaction</h3>
<p id="Par29">QPCR was performed using the 2Ã—RealStar Fast SYBR qPCR Mix (Low ROX) from GenStar Biosolutions (Beijing, China) to validate the expression levels of relevant genes. Primer sequences are provided in Supplementary documentation 2. Relative gene expression levels were calculated using the 2^(-Î”Î”Ct) method, with GAPDH serving as an internal control. Detailed reaction conditions and cycling parameters are provided in Supplementary documentation 2.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par30">This study presented the differential Raman spectral intensities and the expression levels of DEGs as meanâ€‰Â±â€‰standard deviation (SD). Parametric t-tests, ANOVA, non-parametric Wilcoxon rank-sum tests, or Kruskal-Wallis tests were employed to assess the statistical significance of differences in spectral intensities and DEG expression levels among different cell lines. Data visualization was performed using R (4.1.3) and GraphPad Prism (9.5.0). Most of the data visualization conducted in this study was performed using the OmicStudio tools at <a href="https://www.omicstudio.cn/tool" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.omicstudio.cn/tool</a> and was plotted by <a href="https://www.bioinformatics.com.cn" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.bioinformatics.com.cn</a> (last accessed on 10 Dec 2024), an online platform for data analysis and visualization. Unless otherwise specified, statistical significance was determined at a p-value threshold 0.05.</p></section></section><section id="Sec13"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Differential metabolite screening and metabolic pathway analysis</h3>
<p id="Par31">To characterize the metabolic profiles of NPC cell lines (SUNE1, CNE1, CNE2, 5â€“8Â F, and 6-10B) and immortalized nasopharyngeal epithelial cells (NPEC1-BMI1), an untargeted metabolomics analysis was performed using UPLC-MS/MS. A total of 600 metabolites were identified. PCA revealed a clear clustering pattern among the six cell lines, indicating distinct metabolic profiles after dimensionality reduction (Fig.Â <a href="#Fig1" class="usa-link">1</a>A).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373589_12672_2025_3349_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b755/12373589/8f05bb6299fd/12672_2025_3349_Fig1_HTML.jpg" loading="lazy" id="d33e751" height="957" width="714" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differential metabolite profiling and pathway analysis. <strong>A</strong> PCA clearly separated the six cell lines into distinct clusters in a two-dimensional space. <strong>B</strong> OPLS-DA effectively distinguished NPC samples from non-NPC samples. VIP values indicates the contribution of each variable to the model, with higher values indicating a more significant contribution. <strong>C</strong> Volcano plot illustrating DEMs between NPC and non-NPC groups. Metabolites with VIP valuesâ€‰&gt;â€‰1.0 and p-valuesâ€‰&lt;â€‰0.05 were considered statistically significant. Red dots represent up-regulated metabolites, while blue dots represent down-regulated metabolites. <strong>D</strong> Heatmap showing the expression patterns of the top 100 DEMs between NPC and non-NPC groups. The color intensity represents the relative abundance of each metabolite, with red indicating up-regulation and blue indicating down-regulation. <strong>E</strong> RF analysis identified the top 50 most important metabolites for differentiating between the two groups. <strong>F</strong> Enrichment analysis of differential metabolites in KEGG pathways</p></figcaption></figure><p id="Par33">The metabolomics data of NPC cell lines (SUNE1, CNE1, CNE2, 5â€“8Â F, and 6-10B) were combined into an NPC group, while NPEC1-BMI1 cells were designated as the non-NPC group. Inter-cell line variability was mitigated by standardizing the data, applying log2 transformation, and averaging metabolite values, generating a comprehensive NPC group metabolic profile. All cell line data underwent identical preprocessing, and quality control samples ensured data consistency.An orthogonal partial least squares-discriminant analysis (OPLS-DA) model was constructed, yielding a coefficient of determination (RÂ²) of 0.999 and a predictive power (QÂ²) of 0.994 (Fig.Â <a href="#Fig1" class="usa-link">1</a>B). Volcano plots were generated based on variable importance in projection (VIP) scoresâ€‰&gt;â€‰1.0 and p-valuesâ€‰&lt;â€‰0.05 (Fig.Â <a href="#Fig1" class="usa-link">1</a>C). Both the OPLS-DA model and volcano plot analysis indicated significant metabolic differences between the NPC and non-NPC groups. The top 100 DEMs, ranked by q-value, were selected for hierarchical cluster analysis (Fig.Â <a href="#Fig1" class="usa-link">1</a>D). Subsequently, a supervised RF analysis was performed to identify the top 50 most important metabolites for distinguishing between the NPC and non-NPC groups (Fig.Â <a href="#Fig1" class="usa-link">1</a>E). Fourteen metabolites were identified as potential biomarkers based on their ability to discriminate between the two groups in both the hierarchical clustering and RF analyses, which were contained taurine, sphingomyelin (d18:1/18:1, d18:2/18:0), myristoyl dihydrosphingomyelin (d18:0/14:0)*, imidazole propionate, N-acetyl-aspartyl-glutamate (NAAG), 3-ureidopropionate, asparagine, beta-alanine, cysteine, glutamate gamma-methyl ester, mannitol/sorbitol, 10-nonadecenoate (19:1n9), aspartate, and carnitine.</p>
<p id="Par34">We performed a KEGG pathway enrichment analysis to elucidate the biological implications of the top 100 DEMs between NPC and non-NPC groups. Fifty metabolic pathways with q-values significantly less than 0.05 were identified (Fig.Â <a href="#Fig1" class="usa-link">1</a>F). Notable differences were observed in the NPC and non-NPC groups for pathways including carnitine metabolism (deoxycarnitine, carnitine), fatty acid metabolism (particularly dihydroxybutyrate), and short-chain acylcarnitine metabolism (acetylcarnitine (C2)). These significant alterations in metabolites and metabolic pathways may contribute to the development and progression of NPC.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Transcriptome data analysis and pathway analysis by RNA-seq</h3>
<p id="Par35">Transcriptome sequencing identified 35,279 genes expressed across the six cell lines. PCA revealed distinct clustering patterns among the cell lines (Fig.Â <a href="#Fig2" class="usa-link">2</a>A). To identify DEGs associated with NPC, we compared the five NPC cell lines (SUNE1, CNE1, CNE2, 5â€“8Â F, and 6-10B) with the NPEC1-BMI1 cell line. To mitigate inter-cell line variability, data from the five NPC cell lines were combined, generating a composite NPC group gene expression profile through standardization and averaging. All cell line data underwent identical batch processing to ensure consistency.Genes with an adjusted p-value (q-value)â€‰&lt;â€‰0.05 and an absolute log2 fold change (|log2FC|)â€‰&gt;â€‰1 were considered significantly differentially expressed. A total of 5020 DEGs were identified, comprising 2436 upregulated genes and 2584 downregulated genes (Fig.Â <a href="#Fig2" class="usa-link">2</a>B). Hierarchical clustering analysis of the top 500 DEGs, ranked by q-value, further highlighted the distinct transcriptional profiles of NPC cells (Fig.Â <a href="#Fig2" class="usa-link">2</a>C).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373589_12672_2025_3349_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b755/12373589/7ca2c822823e/12672_2025_3349_Fig2_HTML.jpg" loading="lazy" id="d33e809" height="591" width="714" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>DEGs between NPC and non-NPC groups and functional enrichment analysis of these DEGs. <strong>A</strong> PCA of the six cell lines based on transcriptomic data demonstrated that the distinct cell lines are separated in the two-dimensional feature space. <strong>B</strong> Differential gene volcano plot. Genes with a q-valueâ€‰&lt;â€‰0.05 and |log2FC| &gt;1 were considered significantly differentially expressed. Red and blue dots represent upregulated and downregulated genes, respectively. <strong>C</strong> Hierarchical clustering analysis of DEGs between NPC and non-NPC groups. Each column corresponds to one sample, and the color gradient indicates the gene expression levels. <strong>D</strong> Enrichment analysis of DEGs in KEGG Level 1 and Level 2 pathways. <strong>E</strong> GO functional enrichment analysis of DEGs. BP, CC, and MF denote biological processes, cellular components, and molecular functions</p></figcaption></figure><p id="Par37">To further explore the biological functions of the DEGs, we performed a KEGG pathway enrichment analysis. A total of 6 and 44 pathways were significantly enriched at KEGG levels 1 and 2, respectively (Fig.Â <a href="#Fig2" class="usa-link">2</a>D). Signal transduction was the most significantly enriched pathway among the DEGs.GO enrichment analysis identified 362 significantly enriched pathways. The top 10 significantly enriched pathways in biological process (BP), cellular component (CC), and molecular function (MF) categories are shown in Fig.Â <a href="#Fig2" class="usa-link">2</a>E.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Integrated analysis of transcriptomics and metabolomics data</h3>
<p id="Par38">We integrated transcriptomic and metabolomic data to gain deeper insights into the molecular mechanisms underlying NPC. A joint pathway enrichment analysis identified 35 shared pathways significantly enriched by both DEGs and DEMs (Fig.Â <a href="#Fig3" class="usa-link">3</a>A). Notably, pathways such as oxidative phosphorylation, retrograde endocannabinoid signaling, sphingolipid signaling, purine metabolism, and phosphatidylinositol signaling were prominently enriched (Fig.Â <a href="#Fig3" class="usa-link">3</a>B). Further analysis revealed that 18 genes and 28 metabolites were significantly upregulated, while 14 genes and 4 metabolites were significantly downregulated within these shared pathways (Fig.Â <a href="#Fig3" class="usa-link">3</a>C-D). Spearman correlation analysis was performed to identify significant correlations between DEGs and DEMs within these pathways, resulting in a robust transcript-metabolite interaction network (<em>P</em>â€‰&lt;â€‰0.05) (Fig.Â <a href="#Fig3" class="usa-link">3</a>E-F). Among them, differentially up-regulated genes such as LPCAT1, KYNU, PDE10A, ALPP, GNAO1, TYMS, and NME1 were significantly positively correlated with the following metabolites: (R)-3-hydroxybutyrylcarnitine (VIPâ€‰=â€‰1.2449, q valueâ€‰=â€‰1.6442E-05, Fold Changeâ€‰=â€‰13.204), oleoylcarnitine (C18:1) (VIPâ€‰=â€‰1.2344, q valueâ€‰=â€‰2.3710E-05, Fold Changeâ€‰=â€‰184.93), 1-ribosyl-imidazoleacetate (VIPâ€‰=â€‰1.1910, q valueâ€‰=â€‰3.9062E-05, Fold Changeâ€‰=â€‰0.092), N-stearoyltaurine (VIPâ€‰=â€‰1.2449, q valueâ€‰=â€‰6.5875E-06, Fold Changeâ€‰=â€‰29.466), docosapentaenoate (n3 DPA;22:5n3) ( VIPâ€‰=â€‰1.2286, q valueâ€‰=â€‰7.9329E-07, Fold Changeâ€‰=â€‰27.544), eicosenoylcarnitine (C20:1) (VIPâ€‰=â€‰1.2366, q valueâ€‰=â€‰4.1058E-05, Fold Changeâ€‰=â€‰92.492), allantoin (VIPâ€‰=â€‰1.2457, q valueâ€‰=â€‰2.3581E-07, Fold Changeâ€‰=â€‰121.402), homoarginine (VIPâ€‰=â€‰1.2438, q valueâ€‰=â€‰2.5925E-06, Fold Changeâ€‰=â€‰52.256), trigonelline (Nâ€™-methylnicotinate) (VIPâ€‰=â€‰1.2442, q valueâ€‰=â€‰2.5217E-06, Fold Changeâ€‰=â€‰6.156), pipecolate (VIPâ€‰=â€‰1.2324, q valueâ€‰=â€‰7.8871E-07, Fold Changeâ€‰=â€‰5.406), 3-ureidopropionate (VIPâ€‰=â€‰1.2394, q valueâ€‰=â€‰4.6160E-06, Fold Changeâ€‰=â€‰75.092), glycochenodeoxycholate (VIPâ€‰=â€‰1.2398, q valueâ€‰=â€‰1.2258E- 05, Fold Changeâ€‰=â€‰10.092),tyrosine (VIPâ€‰=â€‰1.2043, q valueâ€‰=â€‰4.2138E-05, Fold Changeâ€‰=â€‰3.018), imidazole propionate (VIPâ€‰=â€‰1.2356, q valueâ€‰=â€‰5.5467E-08, Fold Changeâ€‰=â€‰34.628) and pyridoxine (Vitamin B6) (VIPâ€‰=â€‰1.2361, q valueâ€‰=â€‰4.1367E-07, Fold Changeâ€‰=â€‰57.93). However, the significantly downregulated differential gene CBS exhibited a strong negative correlation with various metabolites, including homoarginine, trigonelline (Nâ€™-methylnicotinate), and pipecolate. TableÂ <a href="#Tab2" class="usa-link">2</a> shows that 21 DEGs from common pathways in the KEGG pathway analysis were assigned to six metabolic pathways. To further explore the potential molecular mechanisms underlying the different subtypes of NPC, we performed a comparative analysis of DEGs and DEMs between the highly differentiated CNE1 and lowly differentiated CNE2 cell lines, as well as between the highly metastatic 5â€“8Â F and lowly metastatic 6-10B cell lines. Hierarchical clustering analysis was used to visualize the distinct expression patterns of these genes and metabolites (Supplementary Fig.Â 1A-D). Notably, genes such as NME1, TYMS, and KYNU, along with metabolites like 3-ureidopropionate, 1-ribosyl-imidazoleacetate, and pyridoxine (vitamin B6), exhibited distinct clustering patterns between highly and lowly differentiated cell lines. Conversely, genes like TYMS, PDE10A, and GNAO1, as well as metabolites like pipecolate and imidazole propionate, showed significant clustering differences between high and low metastatic cell lines. These findings suggest that specific gene-metabolite correlations may underlie the molecular characteristics of different NPC subtypes.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373589_12672_2025_3349_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b755/12373589/b77f4aef45ab/12672_2025_3349_Fig3_HTML.jpg" loading="lazy" id="d33e870" height="852" width="714" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Integration of Transcriptomic and Metabolomic Analyses. <strong>A</strong> Venn diagram showing the number of KEGG pathways co-enriched in the analysis of DEGs and differentially expressed DEMs.<strong>B</strong> Histogram showing the distribution of KEGG pathways co-enriched in DEGs and DEMs. <strong>C</strong>, <strong>D</strong> Lists of DEMs and DEGs involved in the common pathways, respectively. <strong>E</strong> Spearman correlation coefficient matrix heatmap showing the correlation between DEMs and DEGs involved in the common pathway. Red and blue colors indicate positive and negative correlations, respectively. *qâ€‰&lt;â€‰0.05; **qâ€‰&lt;â€‰0.01. <strong>F</strong> Correlation networks of DEMs and DEGs involved in common pathways constructed by Cytoscape. Nodes represent DEMs or DEGs, edges represent Spearman correlations between them, and the edgesâ€™ color and thickness indicate the correlationâ€™s strength and direction. *qâ€‰&lt;â€‰0.05 correlations were retained</p></figcaption></figure><section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>DEGs enriched in metabolic pathways</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Classification level 3</th>
<th align="left" colspan="1" rowspan="1">Classification level 1</th>
<th align="left" colspan="1" rowspan="1">Gene number</th>
<th align="left" colspan="1" rowspan="1">Rich. Factor</th>
<th align="left" colspan="1" rowspan="1">Genes</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Metabolic pathways</td>
<td align="left" colspan="1" rowspan="1">Metabolism</td>
<td align="center" colspan="1" rowspan="1">19</td>
<td align="center" colspan="1" rowspan="1">0.30</td>
<td align="left" colspan="1" rowspan="1">NME1 â†‘;HKDC1 â†‘; CHPT1 â†‘; NAT8L â†‘; TYMS â†‘; ENPP1 â†‘; KYNU â†‘; PLAAT3 â†‘; LPCAT1 â†‘;ALPP â†‘; PDE4A â†‘; GLUL â†“; ITPKC â†“; NAGS â†“; ABAT â†“; ARG2 â†“; PIK3CD â†“; CBS â†“; PCBD1 â†“</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Propanoate metabolism</td>
<td align="left" colspan="1" rowspan="1">Metabolism</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">0.47</td>
<td align="left" colspan="1" rowspan="1">ABAT â†“; SUCLG2 â†“</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Pyrimidine metabolism</td>
<td align="left" colspan="1" rowspan="1">Metabolism</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">0.41</td>
<td align="left" colspan="1" rowspan="1">NME1 â†‘; TYMS â†‘; ENPP1 â†‘</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Carbon metabolism</td>
<td align="left" colspan="1" rowspan="1">Metabolism</td>
<td align="center" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">0.35</td>
<td align="left" colspan="1" rowspan="1">HKDC1 â†‘; PIK3R1 â†“; SUCLG2 â†“; PIK3CD â†“</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Arginine and proline metabolism</td>
<td align="left" colspan="1" rowspan="1">Metabolism</td>
<td align="center" colspan="1" rowspan="1">1</td>
<td align="center" colspan="1" rowspan="1">0.42</td>
<td align="left" colspan="1" rowspan="1">ARG2 â†“</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Folate biosynthesis</td>
<td align="left" colspan="1" rowspan="1">Metabolism</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">0.47</td>
<td align="left" colspan="1" rowspan="1">PCBD1 â†“; ALPP â†‘</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec17"><h3 class="pmc_sec_title">Machine learning analysis of Raman spectroscopy data structures of NPC cell lines and NPEC1-BMI1 cell lines</h3>
<p id="Par41">In our previous study, we successfully identified cell-specific lipid metabolic signatures in NPC cells using CMRS coupled with machine learning, and established a high-precision diagnostic model [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Building upon this foundation, we integrated single-cell Raman spectroscopy data from five NPC cell lines (SUNE1, CNE1, CNE2, 5â€“8Â F, and 6-10B) and the NPEC1-BMI1 cell line to comprehensively characterize the metabolic profiles of NPC cells using CMRS. Initially, following background removal and normalization, we obtained single-cell Raman spectra within the spectral range of 500â€“2000Â cmâ»Â¹ for the five NPC cell lines and the NPEC1-BMI1 cell line (Fig.Â <a href="#Fig4" class="usa-link">4</a>A). Analysis using t-distributed stochastic neighbor embedding (t-SNE), a non-linear dimensionality reduction technique (Fig.Â <a href="#Fig4" class="usa-link">4</a>B), revealed a clear distinction between the Raman spectra of the NPEC1-BMI1 cell line and the NPC cell lines. Subsequently, we consolidated the Raman spectroscopy data from the five NPC cell lines into a single NPC group. To minimize inter-cell line variability, we merged the Raman spectroscopy data of these five NPC cell lines and obtained single-cell Raman spectra for the NPC group and the NPEC1-BMI1 cell line within the spectral range of 300â€“1800Â cmâ»Â¹ through standardization and averaging (Fig.Â <a href="#Fig4" class="usa-link">4</a>C). To ensure data consistency, all cell line data underwent identical batch processing.Specifically, 17 wavenumbers exhibited substantial variations in the NPC group compared to the non-NPC group (TableÂ <a href="#Tab1" class="usa-link">1</a>), providing insights into the underlying biomolecular changes associated with NPC. Semi-quantitative analysis revealed that the NPC group exhibited a relative increase in spectral intensity at 666 and1247 cmâ»Â¹ (Nucleic Acid), 780, 1580, 1603, and 1616Â cmâ»Â¹ (Protein), while showing a relative decrease at 405Â cmâ»Â¹ (Hydrocarbon), 525Â cmâ»Â¹ (Nucleic Acid), 1265Â cmâ»Â¹ (Lipid), 1440Â cmâ»Â¹ (Protein, Lipid), 900Â cmâ»Â¹ (Carbohydrate),860, 876, 934, 1125, 1305and 1655Â cmâ»Â¹(Protein). Supplementary Table 2 provides a comprehensive list of the biomolecular structures and their biological significance corresponding to these differentially expressed Raman spectral peaks.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373589_12672_2025_3349_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b755/12373589/4016e1b305d4/12672_2025_3349_Fig4_HTML.jpg" loading="lazy" id="d33e1012" height="762" width="714" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Discrimination of NPC and Non-NPC Cell Lines by Single-Cell Raman Spectroscopy (SCRS). <strong>A</strong> Normalized average Raman spectra of five NPC cell lines (SUNE1, CNE1, CNE2, 5â€“8Â F, and 6-10B) and the NPEC1-BMI1 cell line. Spectral data were baseline-corrected and normalized to the total area. <strong>B</strong> Visualization of high-dimensional data using the non-linear dimensionality reduction technique t-SNE (t-Distributed Stochastic Neighbor Embedding). <strong>C</strong> Normalized average Raman spectra of the NPC group (red) and the non-NPC group (blue, NPEC1-BMI1). Spectral data were baseline-corrected and normalized to the total area. <strong>D</strong> Analysis of SCRS data from the NPC and non-NPC cell lines using the first two principal components of the PCA model. PCA analysis revealed significant differences in Raman spectral features between the two groups. <strong>E</strong> Confusion matrix of the PC-MLP model for classifying NPC and non-NPC groups. <strong>F</strong> ROC curve demonstrating the excellent performance of the PC-MLP model in classifying NPC and non-NPC cells, with an AUC value of 1.00, indicating extremely high classification accuracy. These results suggest that single-cell Raman spectroscopy combined with machine learning models can effectively distinguish between NPC and non-NPC cells</p></figcaption></figure><p id="Par43">To validate the classification performance of Raman spectroscopy data, single-cell Raman spectroscopy data were randomly divided into training and test sets at an 80:20 ratio, and dimensionality reduction was performed using PCA to 50 principal components. On the training set, hyperparameters of various machine learning models were optimized using 5-fold cross-validation (cvâ€‰=â€‰5). Subsequently, ten machine learning models, including MLP, GRU, KNN, GB, LSVM, QDA, RF, LDA, LR, and NB, were trained on the training set, and their prediction results were compared on the test set. The results demonstrated that MLP exhibited the best performance in Raman spectroscopy prediction (Supplementary Table 1), with its training results (Supplementary Fig.Â 2) indicating good convergence and generalization capabilities, effectively learning and generalizing data features, thus providing a reliable foundation for subsequent classification tasks.</p>
<p id="Par44">PCA revealed a clear separation between the NPC and non-NPC groups in the two-dimensional PCA space, indicating significant spectral differences (Fig.Â <a href="#Fig4" class="usa-link">4</a>D). Based on the biomolecular attributions summarized in TableÂ <a href="#Tab1" class="usa-link">1</a>, we focused on several key Raman spectral regions, including 860, 1125, 1305, 1440, 1655, 780, 1247, 1580, 1603, 1616, and 1265Â cmâ»Â¹. These spectral regions contributed significantly to PC1 and PC2 in the PCA analysis. Notably, the Raman bands at 860, 1125, 1305, 1440, and 1655Â cmâ»Â¹ exhibited positive loadings on both PC1 and PC2, while the bands at 780, 1247, 1580, 1603, 1616, and 1265Â cmâ»Â¹ showed positive loadings on PC1 and negative loadings on PC2. To validate the PCA results, a PC-MLP model was developed using the PCA scores as input features and an MLP as the classifier. The binary confusion matrix for the NPC and non-NPC groups is presented in Fig.Â <a href="#Fig4" class="usa-link">4</a>E-F, demonstrating a prediction accuracy of 99.3% (TableÂ <a href="#Tab3" class="usa-link">3</a>). Additionally, the ROC curve for the test set yielded an AUC of 1.00. These results indicate that the PC-MLP model, based on Raman spectroscopy, effectively discriminates between NPC and non-NPC groups with high accuracy.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>PC-MLP model distinguishes NPC group cell lines from non-NPC group cell lines</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">PC-MLP</th>
<th align="left" colspan="2" rowspan="1">Reference</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">NPC</th>
<th align="left" colspan="1" rowspan="1">NPEC-BMI1</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">NPC</td>
<td align="left" colspan="1" rowspan="1">158</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NPEC-BMI1</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">159</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Sensitivity(%)</td>
<td align="left" colspan="1" rowspan="1">99.4</td>
<td align="left" colspan="1" rowspan="1">â€“</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Specificity(%)</td>
<td align="left" colspan="1" rowspan="1">â€“</td>
<td align="left" colspan="1" rowspan="1">98.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Accuracy(%)</td>
<td align="left" colspan="2" rowspan="1">99.3</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec41"><h3 class="pmc_sec_title">Correlation analysis between DEMs and DEGs with differential Raman spectra and differential analysis of potential Raman spectral biomarkers</h3>
<p id="Par46">We identified significant correlations between 32 DEMs and 32 DEGs within the integrated metabolic and transcriptomic pathways (Fig.Â <a href="#Fig3" class="usa-link">3</a>E-F). Furthermore, we performed Spearman correlation analysis between the 17 biomolecularly significant Raman spectral features (TableÂ <a href="#Tab1" class="usa-link">1</a>) and the 32 DEGs and DEMs (Fig.Â <a href="#Fig5" class="usa-link">5</a>A-B). Our study revealed that elevated Raman spectral intensities at 666 and 1616Â cmâ»Â¹ in the NPC group were significantly positively correlated with the expression of genes such as NAT8L and PDE10A. Conversely, decreased Raman spectral intensities at 900 and 934Â cmâ»Â¹ were significantly positively correlated with the expression of PCBD1 and PIK3R1, respectively. Additionally, Raman shifts at 876, 1305, and 1655Â cmâ»Â¹ were significantly negatively correlated with NDUFS6, CHKA, CHPT1, and GDA expression. In the correlation analysis between Raman spectra and DEMs, elevated Raman spectral intensities at 1603, 1616, 780, 1580, and 666Â cmâ»Â¹ in the NPC group were significantly positively correlated with metabolites such as eicosenoylcarnitine (C20:1), trans-4-hydroxyproline, docosapentaenoate (n-3 DPA; 22:5n3), (R)-3-hydroxybutyrylcarnitine, allantoin, propionylcarnitine (C3), homoarginine, trigonelline (Nâ€™-methylnicotinate), butyrylcarnitine (C4), pipecolate, 3-ureidopropionate, and glycochenodeoxycholate. Interestingly, a negative correlation was observed between the 780Â cmâ»Â¹ Raman band and adenine levels. Decreased Raman spectral intensities at 1440, 1305, 1655, 860, and 1125Â cmâ»Â¹ exhibited significant negative correlations with metabolites such as mannitol/sorbitol, 10-nonadecenoate (19:1n9), betaine, homoarginine, trigonelline, N-acetylaspartate (NAA), N, N, N-trimethyl-5-aminovalerate, allantoin, and propionylcarnitine (C3). Specifically, 860 and 1125Â cmâ»Â¹ bands showed significant positive correlations with adenine and pantothenate, respectively.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373589_12672_2025_3349_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b755/12373589/cd894b2d4051/12672_2025_3349_Fig5_HTML.jpg" loading="lazy" id="d33e1121" height="791" width="714" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Correlation Analysis of DEMs and DEGs with Raman Spectra. <strong>A</strong>, <strong>B</strong> Heatmaps demonstrating Spearman correlations between DEMs and DEGs in common pathways and differential Raman peak intensities. Red and blue colors indicate positive and negative correlations, respectively. Color labels indicate the absolute values of correlation coefficients. *qâ€‰&lt;â€‰0.05, **qâ€‰&lt;â€‰0.01. <strong>C</strong>â€“<strong>J</strong> Differences in the intensity of Raman peaks of potential biomarkers in different groups were compared. *qâ€‰&lt;â€‰0.05, **qâ€‰&lt;â€‰0.01, ***qâ€‰&lt;â€‰0.001, ****qâ€‰&lt;â€‰0.0001</p></figcaption></figure><p id="Par48">Our analysis revealed that nine Raman spectral features (666, 1616, 876, 1305, 1655, 1603, 1440, 780, and 860Â cmâ»Â¹) exhibited significant correlations with multiple DEGs and DEMs involved in integrated metabolic and transcriptomic pathways (Fig.Â <a href="#Fig5" class="usa-link">5</a>C-J). Among these, the Raman spectral features at 860, 1603, 1616, and 1655Â cmâ»Â¹ showed significant differences between the NPC and non-NPC groups. Furthermore, these spectral features also discriminated between high and low metastatic and high and low differentiated NPC cell lines, suggesting their potential as biomarkers for subtype-specific molecular features. These findings suggest a complex interplay between Raman spectral information, metabolic profiles, and gene expression patterns in NPC.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Expression analysis of potential biomarkers in HNSCC and NPC</h3>
<p id="Par49">Based on the correlation analysis between DEGs and DEMs within the integrated metabolic and transcriptomic pathways, eight key genes were identified: LPCAT1, KYNU, PDE10A, ALPP, GNAO1, TYMS, CBS, and NME1. Notably, the results demonstrated significantly higher expression of these genes in HNSC tumor samples (<em>n</em>â€‰=â€‰520) compared to adjacent normal tissues (<em>n</em>â€‰=â€‰44) (<em>P</em>â€‰&lt;â€‰0.0001) (Fig.Â <a href="#Fig6" class="usa-link">6</a>A-H).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373589_12672_2025_3349_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b755/12373589/2167b51726a5/12672_2025_3349_Fig6_HTML.jpg" loading="lazy" id="d33e1173" height="984" width="714" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differential Expression of Potential Transcriptomic Biomarkers in TCGA HNSCC Samples and GEO NPC Tissue Samples.(A)-(H) Expression of LPCAT1, KYNU, PDE10A, ALPP, GNAO1, TYMS, CBS, and NME1 was significantly elevated in HNSCC tissues compared to normal tissues. <sup>****</sup><em>P</em>â€‰&lt;â€‰0.0001. (I)-(T) KYNU, NME1, and TYMS expression were significantly elevated in NPC tissues compared with normal tissues as well as in the TCGA database, there were significant differences between KYNU, NME1, and TYMS with cancer stage, tumor grade, and nodal metastasis in HNSC patients. <sup>*</sup><em>P</em>â€‰&lt;â€‰0.05, **<em>P</em>â€‰&lt;â€‰0.01, <sup>***</sup><em>P</em>â€‰&lt;â€‰0.001, <sup>****</sup><em>P</em>â€‰&lt;â€‰0.0001</p></figcaption></figure><p id="Par51">To validate the TCGA database findings, we selected the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE53819" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE53819</a> dataset from the GEO database for analysis. This dataset included whole-genome expression profiling of 18 primary NPC tumors and 18 non-cancerous nasopharyngeal tissues. The median age of NPC patients was 46 years (range: 19â€“77 years), and the median age of the non-cancerous cohort was 45 years (range: 18â€“78 years), with nearly one-third of the patients being female. All samples were collected prior to any anti-cancer treatment. This detailed clinical information, along with the paired tumor and normal tissue samples, provided an important clinical context and a reliable basis for validating the differential expression of genes in NPC tumor tissues. Our analysis results confirmed the overexpression of KYNU, TYMS, and NME1 in NPC tissues (Fig.Â <a href="#Fig6" class="usa-link">6</a>I-T). Notably, these genes exhibited significantly higher expression levels in HNSC tumors compared to normal controls, regardless of tumor stage (I, II, III, and IV) or lymph node metastasis (N0 and N1) (<em>P</em>â€‰&lt;â€‰0.05). These findings suggest that KYNU, TYMS, and NME1 may serve as potential biomarkers for the diagnosis and prognosis of NPC.</p></section><section id="Sec19"><h3 class="pmc_sec_title">KYNU is highly expressed in NPC cell lines and tissues</h3>
<p id="Par52">The results of this study demonstrated a significant trend of KYNU overexpression in NPC. Firstly, qPCR validation revealed that KYNU mRNA levels were significantly elevated in NPC cell lines SUNE1, 5â€“8Â F, 6-10B, and HONE1 compared to the normal nasopharyngeal epithelial cell line NP69 (Fig.Â <a href="#Fig7" class="usa-link">7</a>A). Subsequently, Western blot experiments further confirmed the high expression of KYNU protein in these NPC cell lines (Fig.Â <a href="#Fig7" class="usa-link">7</a>C). Notably, KYNU mRNA and protein levels were also significantly higher in the EBV-positive NPC cell lines CNE2-EBV and HK1-EBV compared to their corresponding EBV-negative cell lines CNE2 and HK1 (Fig.Â <a href="#Fig7" class="usa-link">7</a>D-F).Supplementary documentation 1 provides the full uncropped Gels and Blots images. Furthermore, immunohistochemical (IHC) staining of NPC tissue microarrays revealed three distinct expression levels of KYNU (low, medium, and high) in clinical samples (Fig.Â <a href="#Fig7" class="usa-link">7</a>G), suggesting a potential correlation with the clinicopathological features of NPC. In summary, the high expression of KYNU in NPC cell lines, EBV-positive NPC cell lines, and its differential expression in clinical samples indicate its potential crucial role in the development and progression of NPC.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373589_12672_2025_3349_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b755/12373589/d4a29c15c09e/12672_2025_3349_Fig7_HTML.jpg" loading="lazy" id="d33e1289" height="1079" width="714" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Expression of KYNU in NPC Cell Lines and Tissues. <strong>A</strong> Quantitative PCR (qPCR) results revealed a significant increase in KYNU mRNA levels in NPC cell lines SUNE1, 5â€“8Â F, 6-10B, and HONE1 compared to the normal nasopharyngeal epithelial cell line NP69 (**<em>P</em>â€‰&lt;â€‰0.01, <sup>***</sup><em>P</em>â€‰&lt;â€‰0.001, <sup>****</sup><em>P</em>â€‰&lt;â€‰0.0001). <strong>B</strong>, <strong>C</strong> Western Blot analysis demonstrated elevated KYNU protein levels in SUNE1, 5â€“8Â F, 6-10B, and HONE1 cell lines compared to NP69 (<sup>*</sup><em>P</em>â€‰&lt;â€‰0.05, **<em>P</em>â€‰&lt;â€‰0.01, <sup>***</sup><em>P</em>â€‰&lt;â€‰0.001, <sup>****</sup><em>P</em>â€‰&lt;â€‰0.0001). <strong>D</strong> qPCR results showed a significant increase in KYNU mRNA levels in EBV-positive NPC cell lines CNE2-EBV and HK1-EBV compared to EBV-negative cell lines CNE2 and HK1 (<sup>*</sup><em>P</em>â€‰&lt;â€‰0.05, **<em>P</em>â€‰&lt;â€‰0.01, <sup>***</sup><em>P</em>â€‰&lt;â€‰0.001, <sup>****</sup><em>P</em>â€‰&lt;â€‰0.0001). <strong>E</strong>, <strong>F</strong> Western Blot analysis indicated higher KYNU protein levels in CNE2-EBV and HK1-EBV compared to CNE2 and HK1 (<sup>*</sup><em>P</em>â€‰&lt;â€‰0.05, **<em>P</em>â€‰&lt;â€‰0.01, <sup>***</sup><em>P</em>â€‰&lt;â€‰0.001, <sup>****</sup><em>P</em>â€‰&lt;â€‰0.0001). <strong>G</strong> Immunohistochemical (IHC) staining of a NPC tissue microarray showed varying levels of KYNU protein expression (low, medium, and high) in clinical samples (100Ã— and 200Ã— magnification). HE staining illustrates the morphological characteristics of NPC tissues</p></figcaption></figure></section></section><section id="Sec20"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par54">The development and progression of NPC involve a complex interplay of geographical location, environmental factors, genetic susceptibility, and EBV infection [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Metabolic reprogramming is considered a key characteristic of NPC cells adapting to high energy demands and promoting rapid tumor growth [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. This study integrated Raman spectroscopy, metabolomics, and transcriptomics analyses to elucidate metabolic-related gene alterations in NPC cells and to explore the potential of KYNU as a candidate metabolic-related biomarker.</p>
<p id="Par55">This study integrated metabolomics data to more comprehensively elucidate the molecular mechanisms of NPC, overcoming the limitations of transcriptomics in reflecting dynamic metabolic states and signaling pathway regulation [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Through multi-omics integrated analysis, we identified 35 significantly altered common pathways and found significant correlations between DEGs (including KYNU, NME1, and TYMS) and multiple DEMs within these pathways, suggesting that these genes may play a central role in metabolic dysregulation in NPC. We observed that the upregulation of KYNU in NPC cells was positively correlated with the levels of (R)-3-hydroxybutyryl carnitine, oleoyl carnitine (C18:1), 1-ribosylimidazoleacetic acid*, and N-stearoyl taurine, which supports the role of KYNU as a key enzyme in tryptophan catabolism, potentially driving fatty acid oxidation and ketone body accumulation to meet the energy demands of tumor cells by promoting the production of metabolites such as kynurenine [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. Metabolomic analysis revealed significant alterations in the tryptophan metabolism pathway in NPC cells (Fig.Â <a href="#Fig1" class="usa-link">1</a>F), further corroborating the potential crucial role of KYNU in NPC metabolic reprogramming. The upregulation of NME1 and TYMS and their positive correlations with specific metabolites suggest that they may synergistically promote NPC cell proliferation by participating in multiple pathways such as nucleotide synthesis, amino acid metabolism, and energy metabolism [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>, <a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>].</p>
<p id="Par56">CMRS combined with the PC-MLP model enabled high-precision discrimination between NPC cells and NPEC1-BMI1 cells, outperforming the traditional SVM model [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>], which highlights the potential of Raman spectroscopy in label-free detection and revealing the metabolic heterogeneity of cancer cells [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. The differences in Raman spectral intensity at specific wavenumbers in NPC cells, correlated with DEGs and DEMs, suggest that alterations in these genes and metabolites may lead to changes in cellular molecular composition and structure [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>â€“<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>].</p>
<p id="Par57">Notably, we found a close association between the upregulation of the KYNU gene in NPC cells and significant alterations in Raman spectra, suggesting that KYNU expression levels may influence the overall molecular composition of the cells. Specifically, the increase in intensity at 1603Â cmâ»Â¹ associated with KYNU upregulation may reflect enhanced aromatic amino acid metabolism and changes in protein structure, while the decrease in intensity at 1655Â cmâ»Â¹ suggests alterations in protein secondary structure. These observations are consistent with the elevated levels of KYNU-positively correlated metabolites, including (R)-3-hydroxybutyryl carnitine, oleoyl carnitine (C18:1), 1-ribosylimidazoleacetic acid*, and N-stearoyl taurine, further supporting the critical role of aromatic amino acid metabolism in providing energy and influencing cell signaling in NPC tumor cells. KYNU, as a key enzyme in the tryptophan metabolic pathway, directly affects the metabolic flux in NPC cells by accelerating the conversion of tryptophan to kynurenine, which is not only a precursor for NADâ€‰+â€‰synthesis but also activates AhR, regulating downstream gene expression. The metabolic changes we observed may be related to the increased levels of tryptophan catabolites mediated by KYNU, subsequently promoting fatty acid oxidation and ketone body production to provide energy and biosynthetic precursors for the rapid proliferation of NPC cells. The upregulation of TYMS and its positive correlation with decreased intensity at 1655Â cmâ»Â¹ and elevated levels of metabolites such as L-arginine, L-tyrosine, 3-ureidopropionic acid, and glycochenodeoxycholic acid highlight the crucial role of nucleic acid and protein synthesis in NPC progression. The accumulation of these metabolites may provide necessary precursors for DNA and protein synthesis in tumor cells, thereby promoting cell proliferation and differentiation. In summary, the integration of Raman spectroscopy, metabolomics, and genomics provides a novel strategy for comprehensively analyzing metabolic alterations in NPC and demonstrates its potential value in distinguishing different NPC subtypes.</p>
<p id="Par58">Through integrated omics analyses, we identified eight DEGs closely associated with metabolic alterations in NPC cells, among which KYNU, TYMS, and NME1 were significantly upregulated in NPC tissues and were closely correlated with tumor stage, grade, and lymph node metastasis, suggesting their potential as biomarkers for NPC progression. KYNU, as a key enzyme in tryptophan metabolism, is involved in immunosuppression [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. Our validation confirmed the elevated expression of KYNU in NPC tissues and cell lines (especially in EBV-positive cell lines), further supporting its potential as an NPC biomarker. Based on the significant correlations between KYNU and various metabolites and Raman spectral features, we speculate that KYNU may influence NPC development and progression by regulating specific metabolic pathways. Our findings are consistent with observations of KYNU upregulation in other malignancies. For instance, Zhao et al. [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>] found that high KYNU expression was associated with tumor clinical stage in colorectal cancer (CRC), and inhibiting KYNU significantly reduced the proliferation, migration, and invasion abilities of CRC cells, which aligns with our speculation that KYNU may promote tumor progression in NPC. The study by Lu et al. [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>] showed that KYNU expression was elevated in cisplatin-resistant esophageal cancer cells and tissues, suggesting that KYNU may be involved in tumor drug resistance mechanisms. Analysis of multiple cancer types by LeÃ³n-Letelier et al. [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>] revealed that high KYNU expression was significantly associated with an immunosuppressive tumor microenvironment and with high expression of immune checkpoint genes, including PD-L1, PD-1, and CTLA4, implying that KYNU may affect the immunotherapeutic response in NPC patients. The significant alterations in the tryptophan metabolic pathway observed in our study (Fig.Â <a href="#Fig1" class="usa-link">1</a>F) also support the notion that KYNU may regulate the immune microenvironment by influencing the production of metabolites such as kynurenine. Similarly, Shen et al. [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>] reported that high KYNU expression was associated with a poor immune microenvironment and poor patient prognosis in gastric cancer, and that KYNU knockdown inhibited the proliferation and invasion of gastric cancer cells. In this study, we validated the high expression of KYNU in NPC tissues and cell lines and observed its close correlation with tumor stage, grade, and lymph node metastasis, echoing the findings of Shen et al. in gastric cancer regarding the association of high KYNU expression with poor prognosis, further suggesting KYNU as a potential indicator of poor prognosis in NPC patients. In conclusion, KYNU plays a significant role in various cancers and may promote tumor progression and influence treatment response by affecting the malignant biological behavior of tumor cells and regulating the tumor microenvironment. Therefore, in-depth investigation of the functional mechanisms of KYNU in NPC and its potential clinical value, such as its use as a diagnostic and prognostic biomarker and as a target for therapeutic strategies, is of significant importance.</p>
<p id="Par59">This study integrated Raman spectroscopy, metabolomics, and transcriptomics analyses, preliminarily revealing the potential roles of KYNU, TYMS, and NME1 in NPC metabolic reprogramming and highlighting the value of KYNU as a potential biomarker. However, this study also has several limitations, including a limited sample size, the complexity of Raman spectroscopy and metabolomics analyses, potential discrepancies between transcriptomic data and protein expression, and insufficient functional validation. Considering potential sex- or demographic-based biases in interpreting our metabolomics and transcriptomics data, we noted that the GEO dataset <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE53819" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE53819</a> used for validation had a sample size of 18 pairs of NPC tumors and normal tissues, with a median patient age of 46 years and approximately one-third being female. While this dataset provided an important resource for validation, its sample size and specific demographic distribution may limit our comprehensive assessment of sex- or age-related differences. In our cell line studies, we primarily used cell lines derived from Asian populations, which may limit the generalizability of our findings to other ethnic groups. Therefore, future studies should expand the sample size, investigate the specific mechanisms of these genes in more depth, incorporate proteomics data for validation, and further optimize the analytical methods for Raman spectroscopy and metabolomics. Simultaneously, future research should aim to include larger and more diverse patient cohorts and perform more detailed stratified analyses to explore potential sex- or demographic-specific differences in the metabolic and transcriptomic profiles of NPC, thereby leading to a more comprehensive understanding of the diseaseâ€™s molecular mechanisms.</p>
<p id="Par60">Despite the significant scientific value of this study, we acknowledge the challenges in translating complex multi-omics data into practical clinical applications such as early diagnostic tools or personalized treatment strategies, necessitating careful evaluation. Nevertheless, the potential biomarker KYNU identified in this study shows promise for clinical translation. As a key enzyme significantly overexpressed in NPC tissues and cell lines and correlated with tumor stage, grade, and lymph node metastasis, KYNU could be further developed as a diagnostic or prognostic marker for NPC. Future research should focus on validating the clinical diagnostic and prognostic value of KYNU in larger, independent patient cohorts and exploring the development of convenient detection methods based on KYNU expression levels, such as IHC or serological assays. Furthermore, a deeper understanding of the specific functional mechanisms of KYNU in NPC progression may provide new directions for developing therapeutic strategies targeting KYNU. The limitations of sample size or technology may restrict the applicability of our findings to specific patient subgroups, underscoring the importance of clearly articulating the studyâ€™s limitations and their impact on data interpretation. Additionally, multi-omics research involves substantial patient data, necessitating discussion of potential ethical and social implications, including patient data privacy and security and strategies to prevent misuse of research findings. Therefore, by balancing the potential value of multi-omics with its inherent risks and challenges, we can better facilitate the translation of this research towards clinical applications for NPC.</p></section><section id="Sec21"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par61">This study, through the integration of Raman spectroscopy, metabolomics, and transcriptomics analyses, elucidated key genes and pathways involved in the metabolic reprogramming of NPC, clarified the significant roles of KYNU, TYMS, and NME1 in NPC tumorigenesis and progression, and demonstrated that Raman spectroscopy coupled with a machine learning model can effectively discriminate NPC cells, particularly highlighting the value of KYNU as a potential metabolism-related biomarker.</p></section><section id="Sec22"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12672_2025_3349_MOESM1_ESM.docx</a><sup> (381.5KB, docx) </sup><p>Supplementary documentation 1.The full uncropped Gels and Blots images</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12672_2025_3349_MOESM2_ESM.docx</a><sup> (21.3KB, docx) </sup><p>Supplementary documentation 2.Supplementary Materials and Methods</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM3_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">12672_2025_3349_MOESM3_ESM.tif</a><sup> (29MB, tif) </sup><p>Figure S1. Clustering Analysis of DEGs and DEMs Expression in Different Subtypes of NPC Cell Lines(A) (C) Cluster analysis of DEGs and DEMs between CNE1 and CNE2 cell lines. Each column represents a sample and each row represents a gene or metabolite. (B) (D) Cluster analysis of DEGs and DEMs between 5-8F and 6-10B cell lines. Each column represents a sample and each row represents a gene or metabolite</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM4_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">12672_2025_3349_MOESM4_ESM.tif</a><sup> (16.2MB, tif) </sup><p>Figure S2. Training Results of the MLP Model. Demonstrates the performance of the MLP model during 100 epochs of training, including the accuracy and loss changes in the training and validation sets. The results show that with the increase in training epochs, the accuracy of the model in both the training and validation sets rapidly improved and stabilized, with a high consistency between the validation and training set accuracies. Simultaneously, the loss values rapidly decreased and converged, indicating that the model possesses good convergence and generalization capabilities, effectively learning and generalizing data features, thus providing a reliable foundation for subsequent classification tasks</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM5_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">12672_2025_3349_MOESM5_ESM.zip</a><sup> (3.5MB, zip) </sup><p>Supplementary raw data 1.Original metabolomics data</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM6_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6</a><sup> (14.3MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM7_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7</a><sup> (14.3MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM8_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">12672_2025_3349_MOESM8_ESM.zip</a><sup> (14.4MB, zip) </sup><p>Supplementary raw data 2.Original transcriptomics data</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM9_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 9</a><sup> (14.3MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM10_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 10</a><sup> (14.4MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM11_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">12672_2025_3349_MOESM11_ESM.zip</a><sup> (8MB, zip) </sup><p>Supplementary raw data 3.Original single-cell Raman spectroscopy data</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM12"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM12_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 12</a><sup> (8.3MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM13"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM13_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 13</a><sup> (6.8MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM14"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM14_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 14</a><sup> (8.5MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM15"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM15_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 15</a><sup> (7MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM16"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM16_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 16</a><sup> (8.5MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM17"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM17_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 17</a><sup> (12.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM18"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM18_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 18</a><sup> (14.1KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Professor Mu-Sheng Zeng (Cancer Center, Sun Yat-sen University) for providing the NPEC1-BMI1, SUNE1, 5â€“8Â F, 6-10B, CNE1, and CNE2 cell lines.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Ziman Wu and Haiyan Yang contributed equally to the design and writing of this manuscript and should be considered co-first authors; Yafei Xu, Xiang Ji, and Dayang Chen collected and analyzed the data; Chuang Zhang, Xinying Li and Mingjie Liang performed literature searches and were responsible for data visualization; Xiuming Zhang and Dan Xiong carefully and rigorously revised the manuscript; all authors have read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study was supported by the National Natural Science Foundation of China (Grant No. 81772921), the Science and Technology Planning Project of Shenzhen Municipality, China (JCYJ20230807142806014), and the Shenzhen Key Medical Discipline (No. SZXK054).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Data availability Statement: The authors declare that the main data supporting the findings of this study are available within the article and its Supplementary files.Extra dataare available from the corresponding author upon request.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar2"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par62"> For this study, the NPC tissue microarray was supplied by Sun Yat-sen University Cancer Center. Written informed consent was obtained from all patients regarding the use of their biopsies, with approval from the Ethics Committee of Sun Yat-sen University Cancer Center (Ethics Approval Number: GZR2018939). This study strictly adhered to the Declaration of Helsinki (as revised) and all relevant ethical guidelines and regulations. No animal experiments were involved in this study; therefore, the ARRIVE guidelines are not applicable.</p></section><section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par63">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisherâ€™s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Ziman Wu and Haiyan Yang should be considered joint first authors.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Xiuming Zhang, Email: zhangxiuming0760@163.com.</p>
<p>Dan Xiong, Email: sunny543@126.com.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Chen YP, Chan ATC, Le QT, Blanchard P, Sun Y, Ma J. Nasopharyngeal carcinoma. Lancet. 2019;394(10192):64â€“80.
</cite> [<a href="https://doi.org/10.1016/S0140-6736(19)30956-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31178151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20YP,%20Chan%20ATC,%20Le%20QT,%20Blanchard%20P,%20Sun%20Y,%20Ma%20J.%20Nasopharyngeal%20carcinoma.%20Lancet.%202019;394(10192):64%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Huang H, Yao Y, Deng X, Huang Z, Chen Y, Wang Z, Hong H, Huang H, Lin T. Immunotherapy for nasopharyngeal carcinoma: current status and prospects (Review). Int J Oncol. 2023;63(2):97.
</cite> [<a href="https://doi.org/10.3892/ijo.2023.5545" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10367053/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37417358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20H,%20Yao%20Y,%20Deng%20X,%20Huang%20Z,%20Chen%20Y,%20Wang%20Z,%20Hong%20H,%20Huang%20H,%20Lin%20T.%20Immunotherapy%20for%20nasopharyngeal%20carcinoma:%20current%20status%20and%20prospects%20(Review).%20Int%20J%20Oncol.%202023;63(2):97." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Tsao SW, Tsang CM, Lo KW. Epstein-Barr virus infection and nasopharyngeal carcinoma. Philos Trans R Soc Lond B Biol Sci. 2017;372(1732):20160270.
</cite> [<a href="https://doi.org/10.1098/rstb.2016.0270" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5597737/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28893937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsao%20SW,%20Tsang%20CM,%20Lo%20KW.%20Epstein-Barr%20virus%20infection%20and%20nasopharyngeal%20carcinoma.%20Philos%20Trans%20R%20Soc%20Lond%20B%20Biol%20Sci.%202017;372(1732):20160270." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Trinh CTH, Tran DN, Nguyen LTT, Tran NT, Nguyen MTG, Nguyen VTP, Vu NTH, Dang KD, Van Vo K, Chau HC, Phan PTP. Truc phuong MH. LMP1-EBV gene deletion mutations and HLA genotypes of nasopharyngeal cancer patients in Vietnam. Pathophysiology. 2023;30(1):1â€“12.
</cite> [<a href="https://doi.org/10.3390/pathophysiology30010001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9844464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36649009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trinh%20CTH,%20Tran%20DN,%20Nguyen%20LTT,%20Tran%20NT,%20Nguyen%20MTG,%20Nguyen%20VTP,%20Vu%20NTH,%20Dang%20KD,%20Van%20Vo%20K,%20Chau%20HC,%20Phan%20PTP.%20Truc%20phuong%20MH.%20LMP1-EBV%20gene%20deletion%20mutations%20and%20HLA%20genotypes%20of%20nasopharyngeal%20cancer%20patients%20in%20Vietnam.%20Pathophysiology.%202023;30(1):1%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Guo R, Liang JH, Zhang Y, Lutchenkov M, Li Z, Wang Y, Trujillo-Alonso V, Puri R, Giulino-Roth L, Gewurz BE. Methionine metabolism controls the B cell EBV epigenome and viral latency. Cell Metab. 2022;34(9):1280â€“e12979.
</cite> [<a href="https://doi.org/10.1016/j.cmet.2022.08.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9482757/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36070681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20R,%20Liang%20JH,%20Zhang%20Y,%20Lutchenkov%20M,%20Li%20Z,%20Wang%20Y,%20Trujillo-Alonso%20V,%20Puri%20R,%20Giulino-Roth%20L,%20Gewurz%20BE.%20Methionine%20metabolism%20controls%20the%20B%20cell%20EBV%20epigenome%20and%20viral%20latency.%20Cell%20Metab.%202022;34(9):1280%E2%80%93e12979." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Renaud S, Lefebvre A, Mordon S, MoralÃ¨s O, Delhem N. Novel therapies boosting T cell immunity in epstein barr Virus-Associated nasopharyngeal carcinoma. Int J Mol Sci. 2020;21(12):4292.
</cite> [<a href="https://doi.org/10.3390/ijms21124292" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7352617/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32560253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Renaud%20S,%20Lefebvre%20A,%20Mordon%20S,%20Moral%C3%A8s%20O,%20Delhem%20N.%20Novel%20therapies%20boosting%20T%20cell%20immunity%20in%20epstein%20barr%20Virus-Associated%20nasopharyngeal%20carcinoma.%20Int%20J%20Mol%20Sci.%202020;21(12):4292." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Juarez-Vignon Whaley JJ, Afkhami M, Onyshchenko M, Massarelli E, Sampath S, Amini A, Bell D, Villaflor VM. Recurrent/Metastatic nasopharyngeal carcinoma treatment from present to future: where are we and where are we heading?? Curr Treat Options Oncol. 2023;24(9):1138â€“66.
</cite> [<a href="https://doi.org/10.1007/s11864-023-01101-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10477128/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37318724/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Juarez-Vignon%20Whaley%20JJ,%20Afkhami%20M,%20Onyshchenko%20M,%20Massarelli%20E,%20Sampath%20S,%20Amini%20A,%20Bell%20D,%20Villaflor%20VM.%20Recurrent/Metastatic%20nasopharyngeal%20carcinoma%20treatment%20from%20present%20to%20future:%20where%20are%20we%20and%20where%20are%20we%20heading??%20Curr%20Treat%20Options%20Oncol.%202023;24(9):1138%E2%80%9366." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Tang M, Dong X, Xiao L, Tan Z, Luo X, Yang L, Li W, Shi F, Li Y, Zhao L, Liu N, Du Q, Xie L, Hu J, Weng X, Fan J, Zhou J, Gao Q, Wu W, Zhang X, Liao W, Bode AM, Cao Y. CPT1A-mediated fatty acid oxidation promotes cell proliferation via nucleoside metabolism in nasopharyngeal carcinoma. Cell Death Dis. 2022;13(4):331.
</cite> [<a href="https://doi.org/10.1038/s41419-022-04730-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9001659/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35411000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang%20M,%20Dong%20X,%20Xiao%20L,%20Tan%20Z,%20Luo%20X,%20Yang%20L,%20Li%20W,%20Shi%20F,%20Li%20Y,%20Zhao%20L,%20Liu%20N,%20Du%20Q,%20Xie%20L,%20Hu%20J,%20Weng%20X,%20Fan%20J,%20Zhou%20J,%20Gao%20Q,%20Wu%20W,%20Zhang%20X,%20Liao%20W,%20Bode%20AM,%20Cao%20Y.%20CPT1A-mediated%20fatty%20acid%20oxidation%20promotes%20cell%20proliferation%20via%20nucleoside%20metabolism%20in%20nasopharyngeal%20carcinoma.%20Cell%20Death%20Dis.%202022;13(4):331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Namba T, Nardelli J, Gressens P, Huttner WB. Metabolic regulation of neocortical expansion in development and evolution. Neuron. 2021;109(3):408â€“19.
</cite> [<a href="https://doi.org/10.1016/j.neuron.2020.11.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33306962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Namba%20T,%20Nardelli%20J,%20Gressens%20P,%20Huttner%20WB.%20Metabolic%20regulation%20of%20neocortical%20expansion%20in%20development%20and%20evolution.%20Neuron.%202021;109(3):408%E2%80%9319." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Li L, Tang Q, Ge J, Wang D, Mo Y, Zhang Y, Wang Y, Xiong F, Yan Q, Liao Q, Guo C, Wang F, Zhou M, Xiang B, Zeng Z, Shi L, Chen P, Xiong W. METTL14 promotes lipid metabolism reprogramming and sustains nasopharyngeal carcinoma progression via enhancing mA modification of ANKRD22 mRNA. Clin Transl Med. 2024;14(7):e1766.
</cite> [<a href="https://doi.org/10.1002/ctm2.1766" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11255023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39021049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20L,%20Tang%20Q,%20Ge%20J,%20Wang%20D,%20Mo%20Y,%20Zhang%20Y,%20Wang%20Y,%20Xiong%20F,%20Yan%20Q,%20Liao%20Q,%20Guo%20C,%20Wang%20F,%20Zhou%20M,%20Xiang%20B,%20Zeng%20Z,%20Shi%20L,%20Chen%20P,%20Xiong%20W.%20METTL14%20promotes%20lipid%20metabolism%20reprogramming%20and%20sustains%20nasopharyngeal%20carcinoma%20progression%20via%20enhancing%20mA%20modification%20of%20ANKRD22%20mRNA.%20Clin%20Transl%20Med.%202024;14(7):e1766." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Zanotelli MR, Zhang J, Reinhart-King CA. Mechanoresponsive metabolism in cancer cell migration and metastasis. Cell Metab. 2021;33(7):1307â€“21.
</cite> [<a href="https://doi.org/10.1016/j.cmet.2021.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9015673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33915111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zanotelli%20MR,%20Zhang%20J,%20Reinhart-King%20CA.%20Mechanoresponsive%20metabolism%20in%20cancer%20cell%20migration%20and%20metastasis.%20Cell%20Metab.%202021;33(7):1307%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Fuller H, Zhu Y, Nicholas J, Chatelaine HA, Drzymalla EM, Sarvestani AK, JuliÃ¡n-Serrano S, Tahir UA, Sinnott-Armstrong N, Raffield LM, Rahnavard A, Hua X, Shutta KH, Darst BF. Metabolomic epidemiology offers insights into disease aetiology. Nat Metab. 2023;5(10):1656â€“72.
</cite> [<a href="https://doi.org/10.1038/s42255-023-00903-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11164316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37872285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fuller%20H,%20Zhu%20Y,%20Nicholas%20J,%20Chatelaine%20HA,%20Drzymalla%20EM,%20Sarvestani%20AK,%20Juli%C3%A1n-Serrano%20S,%20Tahir%20UA,%20Sinnott-Armstrong%20N,%20Raffield%20LM,%20Rahnavard%20A,%20Hua%20X,%20Shutta%20KH,%20Darst%20BF.%20Metabolomic%20epidemiology%20offers%20insights%20into%20disease%20aetiology.%20Nat%20Metab.%202023;5(10):1656%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Nascentes Melo LM, Lesner NP, Sabatier M, Ubellacker JM, Tasdogan A. Emerging metabolomic tools to study cancer metastasis. Trends Cancer. 2022;8(12):988â€“1001.
</cite> [<a href="https://doi.org/10.1016/j.trecan.2022.07.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35909026/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nascentes%20Melo%20LM,%20Lesner%20NP,%20Sabatier%20M,%20Ubellacker%20JM,%20Tasdogan%20A.%20Emerging%20metabolomic%20tools%20to%20study%20cancer%20metastasis.%20Trends%20Cancer.%202022;8(12):988%E2%80%931001." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Zhou J, Deng Y, Huang Y, Wang Z, Zhan Z, Cao X, Cai Z, Deng Y, Zhang L, Huang H, Li C, Lv X. An individualized prognostic model in patients with locoregionally advanced nasopharyngeal carcinoma based on serum metabolomic profiling. Life (Basel). 2023;13(5):1167.
</cite> [<a href="https://doi.org/10.3390/life13051167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10224521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37240811/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20J,%20Deng%20Y,%20Huang%20Y,%20Wang%20Z,%20Zhan%20Z,%20Cao%20X,%20Cai%20Z,%20Deng%20Y,%20Zhang%20L,%20Huang%20H,%20Li%20C,%20Lv%20X.%20An%20individualized%20prognostic%20model%20in%20patients%20with%20locoregionally%20advanced%20nasopharyngeal%20carcinoma%20based%20on%20serum%20metabolomic%20profiling.%20Life%20(Basel).%202023;13(5):1167." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Xue K, Cao J, Wang Y, Zhao X, Yu D, Jin C, Xu C. Identification of potential therapeutic gene markers in nasopharyngeal carcinoma based on bioinformatics analysis. Clin Transl Sci. 2020;13(2):265â€“74.
</cite> [<a href="https://doi.org/10.1111/cts.12690" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7070980/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31863646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xue%20K,%20Cao%20J,%20Wang%20Y,%20Zhao%20X,%20Yu%20D,%20Jin%20C,%20Xu%20C.%20Identification%20of%20potential%20therapeutic%20gene%20markers%20in%20nasopharyngeal%20carcinoma%20based%20on%20bioinformatics%20analysis.%20Clin%20Transl%20Sci.%202020;13(2):265%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Zhuang Z, Li N, Guo Z, Zhu M, Xiong K, Chen S. Study of molecule variations in renal tumor based on confocal micro-Raman spectroscopy. J Biomed Opt. 2013;18(3):31103.
</cite> [<a href="https://doi.org/10.1117/1.JBO.18.3.031103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23174950/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhuang%20Z,%20Li%20N,%20Guo%20Z,%20Zhu%20M,%20Xiong%20K,%20Chen%20S.%20Study%20of%20molecule%20variations%20in%20renal%20tumor%20based%20on%20confocal%20micro-Raman%20spectroscopy.%20J%20Biomed%20Opt.%202013;18(3):31103." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Yue S, Cheng JX. Deciphering single cell metabolism by coherent Raman scattering microscopy. Curr Opin Chem Biol. 2016;33:46â€“57.
</cite> [<a href="https://doi.org/10.1016/j.cbpa.2016.05.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5018428/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27288951/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yue%20S,%20Cheng%20JX.%20Deciphering%20single%20cell%20metabolism%20by%20coherent%20Raman%20scattering%20microscopy.%20Curr%20Opin%20Chem%20Biol.%202016;33:46%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Elumalai S, ManagÃ³ S, De Luca AC. Raman microscopy: progress in research on cancer cell sensing. Sens (Basel). 2020;20(19):5525.</cite> [<a href="https://doi.org/10.3390/s20195525" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7582629/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32992464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elumalai%20S,%20Manag%C3%B3%20S,%20De%20Luca%20AC.%20Raman%20microscopy:%20progress%20in%20research%20on%20cancer%20cell%20sensing.%20Sens%20(Basel).%202020;20(19):5525." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Lin H, Zhou J, Wu Q, Hung TM, Chen W, Yu Y, Chang JT, Pan J, Qiu S, Chen R. Human blood test based on surface-enhanced Raman spectroscopy technology using different excitation light for nasopharyngeal cancer detection. IET Nanobiotechnol. 2019;13(9):942â€“5.
</cite> [<a href="https://doi.org/10.1049/iet-nbt.2019.0221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8676329/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31811763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20H,%20Zhou%20J,%20Wu%20Q,%20Hung%20TM,%20Chen%20W,%20Yu%20Y,%20Chang%20JT,%20Pan%20J,%20Qiu%20S,%20Chen%20R.%20Human%20blood%20test%20based%20on%20surface-enhanced%20Raman%20spectroscopy%20technology%20using%20different%20excitation%20light%20for%20nasopharyngeal%20cancer%20detection.%20IET%20Nanobiotechnol.%202019;13(9):942%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Chan JW, Taylor DS, Zwerdling T, Lane SM, Ihara K, Huser T. Micro-Raman spectroscopy detects individual neoplastic and normal hematopoietic cells. Biophys J. 2006;90(2):648â€“56.
</cite> [<a href="https://doi.org/10.1529/biophysj.105.066761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1367069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16239327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chan%20JW,%20Taylor%20DS,%20Zwerdling%20T,%20Lane%20SM,%20Ihara%20K,%20Huser%20T.%20Micro-Raman%20spectroscopy%20detects%20individual%20neoplastic%20and%20normal%20hematopoietic%20cells.%20Biophys%20J.%202006;90(2):648%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Song Y, Zhou J, Zhao X, Zhang Y, Xu X, Zhang D, Pang J, Bao H, Ji Y, Zhan M, Wang Y, Ou Q, Hu J. Lineage tracing for multiple lung cancer by Spatiotemporal heterogeneity using a multi-omics analysis method integrating genomic, transcriptomic, and immune-related features. Front Oncol. 2023;13:1237308.
</cite> [<a href="https://doi.org/10.3389/fonc.2023.1237308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10548834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37799479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song%20Y,%20Zhou%20J,%20Zhao%20X,%20Zhang%20Y,%20Xu%20X,%20Zhang%20D,%20Pang%20J,%20Bao%20H,%20Ji%20Y,%20Zhan%20M,%20Wang%20Y,%20Ou%20Q,%20Hu%20J.%20Lineage%20tracing%20for%20multiple%20lung%20cancer%20by%20Spatiotemporal%20heterogeneity%20using%20a%20multi-omics%20analysis%20method%20integrating%20genomic,%20transcriptomic,%20and%20immune-related%20features.%20Front%20Oncol.%202023;13:1237308." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Tao M, Song T, Du W, Han S, Zuo C, Li Y, Wang Y, Yang Z. Classifying breast cancer subtypes using multiple kernel learning based on omics data. Genes (Basel). 2019;10(3):200.
</cite> [<a href="https://doi.org/10.3390/genes10030200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6471546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30866472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tao%20M,%20Song%20T,%20Du%20W,%20Han%20S,%20Zuo%20C,%20Li%20Y,%20Wang%20Y,%20Yang%20Z.%20Classifying%20breast%20cancer%20subtypes%20using%20multiple%20kernel%20learning%20based%20on%20omics%20data.%20Genes%20(Basel).%202019;10(3):200." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Zhang SQ, Pan SM, Liang SX, Han YS, Chen HB, Li JC. Research status and prospects of biomarkers for nasopharyngeal carcinoma in the era of highthroughput omics (Review). Int J Oncol. 2021;58(4):9.
</cite> [<a href="https://doi.org/10.3892/ijo.2021.5188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7910009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33649830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20SQ,%20Pan%20SM,%20Liang%20SX,%20Han%20YS,%20Chen%20HB,%20Li%20JC.%20Research%20status%20and%20prospects%20of%20biomarkers%20for%20nasopharyngeal%20carcinoma%20in%20the%20era%20of%20highthroughput%20omics%20(Review).%20Int%20J%20Oncol.%202021;58(4):9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Spence T, Bruce J, Yip KW, Liu FF. MicroRNAs in nasopharyngeal carcinoma. Chin Clin Oncol. 2016;5(2):17.
</cite> [<a href="https://doi.org/10.21037/cco.2016.03.09" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27121877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spence%20T,%20Bruce%20J,%20Yip%20KW,%20Liu%20FF.%20MicroRNAs%20in%20nasopharyngeal%20carcinoma.%20Chin%20Clin%20Oncol.%202016;5(2):17." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Tang T, Zhou Z, Chen M, Li N, Sun J, Chen Z, Xiao T, Wang X, Zhang L, Wang Y, Zhang H, Zheng X, Chen B, Ye F, Guan J. Plasma metabolic Profiles-Based prediction of induction chemotherapy efficacy in nasopharyngeal carcinoma: results of a bidirectional clinical trial. Clin Cancer Res. 2024;30(14):2925â€“36.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-23-3608" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11247322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38713248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang%20T,%20Zhou%20Z,%20Chen%20M,%20Li%20N,%20Sun%20J,%20Chen%20Z,%20Xiao%20T,%20Wang%20X,%20Zhang%20L,%20Wang%20Y,%20Zhang%20H,%20Zheng%20X,%20Chen%20B,%20Ye%20F,%20Guan%20J.%20Plasma%20metabolic%20Profiles-Based%20prediction%20of%20induction%20chemotherapy%20efficacy%20in%20nasopharyngeal%20carcinoma:%20results%20of%20a%20bidirectional%20clinical%20trial.%20Clin%20Cancer%20Res.%202024;30(14):2925%E2%80%9336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>GÃ³ral J, Zichy V. Fourier transform Raman studies of materials and compounds of biological importance. Spectrochimica Acta Part A: Mol Spectrosc. 1990;46(2):253â€“75.</cite> [<a href="https://scholar.google.com/scholar_lookup?G%C3%B3ral%20J,%20Zichy%20V.%20Fourier%20transform%20Raman%20studies%20of%20materials%20and%20compounds%20of%20biological%20importance.%20Spectrochimica%20Acta%20Part%20A:%20Mol%20Spectrosc.%201990;46(2):253%E2%80%9375." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Wiercigroch E, Szafraniec E, Czamara K, Pacia MZ, Majzner K, Kochan K, et al. Raman and infrared spectroscopy of carbohydrates: A review. Spectrochim Acta Part A Mol Biomol Spectrosc. 2017;185:317â€“35.</cite> [<a href="https://doi.org/10.1016/j.saa.2017.05.045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28599236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wiercigroch%20E,%20Szafraniec%20E,%20Czamara%20K,%20Pacia%20MZ,%20Majzner%20K,%20Kochan%20K,%20et%20al.%20Raman%20and%20infrared%20spectroscopy%20of%20carbohydrates:%20A%20review.%20Spectrochim%20Acta%20Part%20A%20Mol%20Biomol%20Spectrosc.%202017;185:317%E2%80%9335." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Lin C, Li Y, Peng Y, Zhao S, Xu M, Zhang L, Huang Z, Shi J, Yang Y. Recent development of surface-enhanced Raman scattering for biosensing. J Nanobiotechnol. 2023;21(1):149.</cite> [<a href="https://doi.org/10.1186/s12951-023-01890-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10163864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37149605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20C,%20Li%20Y,%20Peng%20Y,%20Zhao%20S,%20Xu%20M,%20Zhang%20L,%20Huang%20Z,%20Shi%20J,%20Yang%20Y.%20Recent%20development%20of%20surface-enhanced%20Raman%20scattering%20for%20biosensing.%20J%20Nanobiotechnol.%202023;21(1):149." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Pezzotti G. Raman spectroscopy in cell biology and microbiology. J Raman Spectrosc. 2021;52(12):2348â€“443.</cite> [<a href="https://scholar.google.com/scholar_lookup?Pezzotti%20G.%20Raman%20spectroscopy%20in%20cell%20biology%20and%20microbiology.%20J%20Raman%20Spectrosc.%202021;52(12):2348%E2%80%93443." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Zhu G, Zhu X, Fan Q, Wan X. Raman spectra of amino acids and their aqueous solutions. Spectrochim Acta Part A Mol Biomol Spectrosc. 2011;78(3):1187â€“95.</cite> [<a href="https://doi.org/10.1016/j.saa.2010.12.079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21242101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20G,%20Zhu%20X,%20Fan%20Q,%20Wan%20X.%20Raman%20spectra%20of%20amino%20acids%20and%20their%20aqueous%20solutions.%20Spectrochim%20Acta%20Part%20A%20Mol%20Biomol%20Spectrosc.%202011;78(3):1187%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Barth A, Zscherp C. What vibrations tell Us about proteins. Q Rev Biophys. 2002;35(4):369â€“430.
</cite> [<a href="https://doi.org/10.1017/s0033583502003815" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12621861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barth%20A,%20Zscherp%20C.%20What%20vibrations%20tell%20Us%20about%20proteins.%20Q%20Rev%20Biophys.%202002;35(4):369%E2%80%93430." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Ruiz-Chica AJ, Medina MA, SÃ¡nchezâ€JimÃ©nez F, RamÃ­rez FJ. Characterization by Raman spectroscopy of conformational changes on guanineâ€“cytosine and adenineâ€“thymine oligonucleotides induced by aminooxy analogues of spermidine. J Raman Spectrosc. 2004;35(2):93â€“100.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ruiz-Chica%20AJ,%20Medina%20MA,%20S%C3%A1nchez%E2%80%90Jim%C3%A9nez%20F,%20Ram%C3%ADrez%20FJ.%20Characterization%20by%20Raman%20spectroscopy%20of%20conformational%20changes%20on%20guanine%E2%80%93cytosine%20and%20adenine%E2%80%93thymine%20oligonucleotides%20induced%20by%20aminooxy%20analogues%20of%20spermidine.%20J%20Raman%20Spectrosc.%202004;35(2):93%E2%80%93100." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Czamara K, Majzner K, Pacia MZ, Kochan K, Kaczor A, Baranska M. Raman spectroscopy of lipids: a review. J Raman Spectrosc. 2015;46(1):4â€“20.</cite> [<a href="https://scholar.google.com/scholar_lookup?Czamara%20K,%20Majzner%20K,%20Pacia%20MZ,%20Kochan%20K,%20Kaczor%20A,%20Baranska%20M.%20Raman%20spectroscopy%20of%20lipids:%20a%20review.%20J%20Raman%20Spectrosc.%202015;46(1):4%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Movasaghi Z, Rehman S, Rehman DIU. Raman spectroscopy of biological tissues. Appl Spectrosc Rev. 2007;42(5):493â€“541.</cite> [<a href="https://scholar.google.com/scholar_lookup?Movasaghi%20Z,%20Rehman%20S,%20Rehman%20DIU.%20Raman%20spectroscopy%20of%20biological%20tissues.%20Appl%20Spectrosc%20Rev.%202007;42(5):493%E2%80%93541." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Stone N, Kendall C, Shepherd N, Crow P, Barr H. Near-infrared Raman spectroscopy for the classification of epithelial pre-cancers and cancers. J Raman Spectrosc. 2002;33(7):564â€“73.</cite> [<a href="https://scholar.google.com/scholar_lookup?Stone%20N,%20Kendall%20C,%20Shepherd%20N,%20Crow%20P,%20Barr%20H.%20Near-infrared%20Raman%20spectroscopy%20for%20the%20classification%20of%20epithelial%20pre-cancers%20and%20cancers.%20J%20Raman%20Spectrosc.%202002;33(7):564%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Xu J, Chen D, Wu W, Ji X, Dou X, Gao X, Li J, Zhang X, Huang WE, Xiong D. A metabolic map and artificial intelligence-aided identification of nasopharyngeal carcinoma via a single-cell Raman platform. Br J Cancer. 2024;130(10):1635â€“46.
</cite> [<a href="https://doi.org/10.1038/s41416-024-02637-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11091122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38454165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20J,%20Chen%20D,%20Wu%20W,%20Ji%20X,%20Dou%20X,%20Gao%20X,%20Li%20J,%20Zhang%20X,%20Huang%20WE,%20Xiong%20D.%20A%20metabolic%20map%20and%20artificial%20intelligence-aided%20identification%20of%20nasopharyngeal%20carcinoma%20via%20a%20single-cell%20Raman%20platform.%20Br%20J%20Cancer.%202024;130(10):1635%E2%80%9346." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Chang ET, Ye W, Zeng YX, Adami HO. The evolving epidemiology of nasopharyngeal carcinoma. Cancer Epidemiol Biomarkers Prev. 2021;30(6):1035â€“47.
</cite> [<a href="https://doi.org/10.1158/1055-9965.EPI-20-1702" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33849968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chang%20ET,%20Ye%20W,%20Zeng%20YX,%20Adami%20HO.%20The%20evolving%20epidemiology%20of%20nasopharyngeal%20carcinoma.%20Cancer%20Epidemiol%20Biomarkers%20Prev.%202021;30(6):1035%E2%80%9347." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Huang H, Li S, Tang Q, Zhu G. Metabolic reprogramming and immune evasion in nasopharyngeal carcinoma. Front Immunol. 2021;12:680955.
</cite> [<a href="https://doi.org/10.3389/fimmu.2021.680955" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8458828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34566954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20H,%20Li%20S,%20Tang%20Q,%20Zhu%20G.%20Metabolic%20reprogramming%20and%20immune%20evasion%20in%20nasopharyngeal%20carcinoma.%20Front%20Immunol.%202021;12:680955." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Maan K, Baghel R, Dhariwal S, Sharma A, Bakhshi R, Rana P. Metabolomics and transcriptomics based multi-omics integration reveals radiation-induced altered pathway networking and underlying mechanism. NPJ Syst Biol Appl. 2023;9(1):42.
</cite> [<a href="https://doi.org/10.1038/s41540-023-00305-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10492812/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37689794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maan%20K,%20Baghel%20R,%20Dhariwal%20S,%20Sharma%20A,%20Bakhshi%20R,%20Rana%20P.%20Metabolomics%20and%20transcriptomics%20based%20multi-omics%20integration%20reveals%20radiation-induced%20altered%20pathway%20networking%20and%20underlying%20mechanism.%20NPJ%20Syst%20Biol%20Appl.%202023;9(1):42." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Al-Mansoob M, Gupta I, Stefan Rusyniak R, Ouhtit A. KYNU, a novel potential target that underpins CD44-promoted breast tumour cell invasion. J Cell Mol Med. 2021;25(5):2309â€“14.
</cite> [<a href="https://doi.org/10.1111/jcmm.16296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7933956/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33486887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Al-Mansoob%20M,%20Gupta%20I,%20Stefan%20Rusyniak%20R,%20Ouhtit%20A.%20KYNU,%20a%20novel%20potential%20target%20that%20underpins%20CD44-promoted%20breast%20tumour%20cell%20invasion.%20J%20Cell%20Mol%20Med.%202021;25(5):2309%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Attwood PV, Muimo R. The actions of NME1/NDPK-A and NME2/NDPK-B as protein kinases. Lab Invest. 2018;98(3):283â€“90.
</cite> [<a href="https://doi.org/10.1038/labinvest.2017.125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29200201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Attwood%20PV,%20Muimo%20R.%20The%20actions%20of%20NME1/NDPK-A%20and%20NME2/NDPK-B%20as%20protein%20kinases.%20Lab%20Invest.%202018;98(3):283%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Furuta E, Okuda H, Kobayashi A, Watabe K. Metabolic genes in cancer: their roles in tumor progression and clinical implications. Biochim Biophys Acta. 2010;1805(2):141â€“52.
</cite> [<a href="https://doi.org/10.1016/j.bbcan.2010.01.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2850259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20122995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Furuta%20E,%20Okuda%20H,%20Kobayashi%20A,%20Watabe%20K.%20Metabolic%20genes%20in%20cancer:%20their%20roles%20in%20tumor%20progression%20and%20clinical%20implications.%20Biochim%20Biophys%20Acta.%202010;1805(2):141%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Overman SA, Thomas GJ Jr. Raman spectroscopy of the filamentous virus Ff (fd, fl, M13): structural interpretation for coat protein aromatics. Biochemistry. 1995;34(16):5440â€“51.
</cite> [<a href="https://doi.org/10.1021/bi00016a015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7727402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Overman%20SA,%20Thomas%20GJ%20Jr.%20Raman%20spectroscopy%20of%20the%20filamentous%20virus%20Ff%20(fd,%20fl,%20M13):%20structural%20interpretation%20for%20coat%20protein%20aromatics.%20Biochemistry.%201995;34(16):5440%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Guleken Z, Ceylan Z, Aday A, Bayrak AG, Hindilerden Ä°Y, NalÃ§acÄ± M, Jakubczyk P, Jakubczyk D, Kula-Maximenko M, Depciuch J. Detection of primary myelofibrosis in blood serum via Raman spectroscopy assisted by machine learning approaches; correlation with clinical diagnosis. Nanomedicine. 2023;53:102706.
</cite> [<a href="https://doi.org/10.1016/j.nano.2023.102706" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37633405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guleken%20Z,%20Ceylan%20Z,%20Aday%20A,%20Bayrak%20AG,%20Hindilerden%20%C4%B0Y,%20Nal%C3%A7ac%C4%B1%20M,%20Jakubczyk%20P,%20Jakubczyk%20D,%20Kula-Maximenko%20M,%20Depciuch%20J.%20Detection%20of%20primary%20myelofibrosis%20in%20blood%20serum%20via%20Raman%20spectroscopy%20assisted%20by%20machine%20learning%20approaches;%20correlation%20with%20clinical%20diagnosis.%20Nanomedicine.%202023;53:102706." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Miura T, Thomas GJ Jr. Structure and dynamics of interstrand guanine association in quadruplex telomeric DNA. Biochemistry. 1995;34(29):9645â€“54.
</cite> [<a href="https://doi.org/10.1021/bi00029a042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7626633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miura%20T,%20Thomas%20GJ%20Jr.%20Structure%20and%20dynamics%20of%20interstrand%20guanine%20association%20in%20quadruplex%20telomeric%20DNA.%20Biochemistry.%201995;34(29):9645%E2%80%9354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Bai Y, Yu Z, Yi S, Yan Y, Huang Z, Qiu L. Raman spectroscopy-based biomarker screening by studying the fingerprint characteristics of chronic lymphocytic leukemia and diffuse large B-cell lymphoma. J Pharm Biomed Anal. 2020;190:113514.
</cite> [<a href="https://doi.org/10.1016/j.jpba.2020.113514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32827998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bai%20Y,%20Yu%20Z,%20Yi%20S,%20Yan%20Y,%20Huang%20Z,%20Qiu%20L.%20Raman%20spectroscopy-based%20biomarker%20screening%20by%20studying%20the%20fingerprint%20characteristics%20of%20chronic%20lymphocytic%20leukemia%20and%20diffuse%20large%20B-cell%20lymphoma.%20J%20Pharm%20Biomed%20Anal.%202020;190:113514." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>hen K, Chen B, Yang L, Gao W. KYNU as a biomarker of Tumor-Associated macrophages and correlates with immunosuppressive microenvironment and poor prognosis in gastric cancer. Int J Genomics. 2023;2023:4662480.
</cite> [<a href="https://doi.org/10.1155/2023/4662480" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10635752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37954130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?hen%20K,%20Chen%20B,%20Yang%20L,%20Gao%20W.%20KYNU%20as%20a%20biomarker%20of%20Tumor-Associated%20macrophages%20and%20correlates%20with%20immunosuppressive%20microenvironment%20and%20poor%20prognosis%20in%20gastric%20cancer.%20Int%20J%20Genomics.%202023;2023:4662480." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Zhao L, Wang B, Yang C, Lin Y, Zhang Z, Wang S, Ye Y, Shen Z. TDO2 knockdown inhibits colorectal cancer progression via TDO2-KYNU-AhR pathway. Gene. 2021;792:145736.
</cite> [<a href="https://doi.org/10.1016/j.gene.2021.145736" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34051337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20L,%20Wang%20B,%20Yang%20C,%20Lin%20Y,%20Zhang%20Z,%20Wang%20S,%20Ye%20Y,%20Shen%20Z.%20TDO2%20knockdown%20inhibits%20colorectal%20cancer%20progression%20via%20TDO2-KYNU-AhR%20pathway.%20Gene.%202021;792:145736." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Lu Y, Zhao X, Yuan M, Zhao M, Liu K, Zhang M, Qiu X, Yu X, Liu X, Wei D, Xie J, Cheng Z. KYNU expression promotes cisplatin resistance in esophageal cancer. J Cancer. 2024;15(9):2475â€“85. 10.7150/jca.93229. PMID: 38577600; PMCID: PMC10988315.
</cite> [<a href="https://doi.org/10.7150/jca.93229" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10988315/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38577600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20Y,%20Zhao%20X,%20Yuan%20M,%20Zhao%20M,%20Liu%20K,%20Zhang%20M,%20Qiu%20X,%20Yu%20X,%20Liu%20X,%20Wei%20D,%20Xie%20J,%20Cheng%20Z.%20KYNU%20expression%20promotes%20cisplatin%20resistance%20in%20esophageal%20cancer.%20J%20Cancer.%202024;15(9):2475%E2%80%9385.%2010.7150/jca.93229.%20PMID:%2038577600;%20PMCID:%20PMC10988315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>LeÃ³n-Letelier RA, Abdel Sater AH, Chen Y, Park S, Wu R, Irajizad E, Dennison JB, Katayama H, Vykoukal JV, Hanash S, Ostrin EJ, Fahrmann JF. Kynureninase upregulation is a prominent feature of NFR2-Activated cancers and is associated with tumor immunosuppression and poor prognosis. Cancers (Basel). 2023;15(3):834.
</cite> [<a href="https://doi.org/10.3390/cancers15030834" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9913753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36765792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Le%C3%B3n-Letelier%20RA,%20Abdel%20Sater%20AH,%20Chen%20Y,%20Park%20S,%20Wu%20R,%20Irajizad%20E,%20Dennison%20JB,%20Katayama%20H,%20Vykoukal%20JV,%20Hanash%20S,%20Ostrin%20EJ,%20Fahrmann%20JF.%20Kynureninase%20upregulation%20is%20a%20prominent%20feature%20of%20NFR2-Activated%20cancers%20and%20is%20associated%20with%20tumor%20immunosuppression%20and%20poor%20prognosis.%20Cancers%20(Basel).%202023;15(3):834." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Shen K, Chen B, Yang L, Gao W. KYNU as a biomarker of Tumor-Associated macrophages and correlates with immunosuppressive microenvironment and poor prognosis in gastric cancer. Int J Genomics. 2023;2023:4662480.
</cite> [<a href="https://doi.org/10.1155/2023/4662480" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10635752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37954130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen%20K,%20Chen%20B,%20Yang%20L,%20Gao%20W.%20KYNU%20as%20a%20biomarker%20of%20Tumor-Associated%20macrophages%20and%20correlates%20with%20immunosuppressive%20microenvironment%20and%20poor%20prognosis%20in%20gastric%20cancer.%20Int%20J%20Genomics.%202023;2023:4662480." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12672_2025_3349_MOESM1_ESM.docx</a><sup> (381.5KB, docx) </sup><p>Supplementary documentation 1.The full uncropped Gels and Blots images</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12672_2025_3349_MOESM2_ESM.docx</a><sup> (21.3KB, docx) </sup><p>Supplementary documentation 2.Supplementary Materials and Methods</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM3_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">12672_2025_3349_MOESM3_ESM.tif</a><sup> (29MB, tif) </sup><p>Figure S1. Clustering Analysis of DEGs and DEMs Expression in Different Subtypes of NPC Cell Lines(A) (C) Cluster analysis of DEGs and DEMs between CNE1 and CNE2 cell lines. Each column represents a sample and each row represents a gene or metabolite. (B) (D) Cluster analysis of DEGs and DEMs between 5-8F and 6-10B cell lines. Each column represents a sample and each row represents a gene or metabolite</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM4_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">12672_2025_3349_MOESM4_ESM.tif</a><sup> (16.2MB, tif) </sup><p>Figure S2. Training Results of the MLP Model. Demonstrates the performance of the MLP model during 100 epochs of training, including the accuracy and loss changes in the training and validation sets. The results show that with the increase in training epochs, the accuracy of the model in both the training and validation sets rapidly improved and stabilized, with a high consistency between the validation and training set accuracies. Simultaneously, the loss values rapidly decreased and converged, indicating that the model possesses good convergence and generalization capabilities, effectively learning and generalizing data features, thus providing a reliable foundation for subsequent classification tasks</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM5_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">12672_2025_3349_MOESM5_ESM.zip</a><sup> (3.5MB, zip) </sup><p>Supplementary raw data 1.Original metabolomics data</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM6_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6</a><sup> (14.3MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM7_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7</a><sup> (14.3MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM8_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">12672_2025_3349_MOESM8_ESM.zip</a><sup> (14.4MB, zip) </sup><p>Supplementary raw data 2.Original transcriptomics data</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM9_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 9</a><sup> (14.3MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM10_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 10</a><sup> (14.4MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM11_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">12672_2025_3349_MOESM11_ESM.zip</a><sup> (8MB, zip) </sup><p>Supplementary raw data 3.Original single-cell Raman spectroscopy data</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM12_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 12</a><sup> (8.3MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM13_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 13</a><sup> (6.8MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material14_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM14_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 14</a><sup> (8.5MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material15_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM15_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 15</a><sup> (7MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material16_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM16_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 16</a><sup> (8.5MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material17_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM17_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 17</a><sup> (12.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material18_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373589/bin/12672_2025_3349_MOESM18_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 18</a><sup> (14.1KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data availability Statement: The authors declare that the main data supporting the findings of this study are available within the article and its Supplementary files.Extra dataare available from the corresponding author upon request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Discover Oncology are provided here courtesy of <strong>Springer</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1007/s12672-025-03349-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12672_2025_Article_3349.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.6Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373589/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373589/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373589%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373589/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373589/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373589/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40844682/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373589/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40844682/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373589/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373589/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="ugnke4oH1dyO4ue7bEdXcoWso5wte1oXD57MNl42wJU37JiPNicmh2LpXM30ellQ">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
