
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Exploring biogeochemical transformation in a hypertrophic lake with dual focus on environmental cues and potential functional fingerprints of littoral bacteria - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152552E88AF43D630552E80025939992.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375747/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Exploring biogeochemical transformation in a hypertrophic lake with dual focus on environmental cues and potential functional fingerprints of littoral bacteria">
<meta name="citation_author" content="Milap Dashora">
<meta name="citation_author_institution" content="Department of Environmental Science, School of Earth Sciences, Central University of Rajasthan, Bandarsindri, Kishangarh, Ajmer, Rajasthan 305817 India">
<meta name="citation_author" content="Manish Kumar">
<meta name="citation_author_institution" content="Department of Environmental Studies, Faculty of Science, University of Delhi, New Delhi, 110007 India">
<meta name="citation_author" content="Garima Kaushik">
<meta name="citation_author_institution" content="Department of Environmental Science, School of Earth Sciences, Central University of Rajasthan, Bandarsindri, Kishangarh, Ajmer, Rajasthan 305817 India">
<meta name="citation_author" content="Alok Kumar">
<meta name="citation_author_institution" content="Department of Environmental Studies, Faculty of Science, University of Delhi, New Delhi, 110007 India">
<meta name="citation_publication_date" content="2025 Aug 24">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="31115">
<meta name="citation_doi" content="10.1038/s41598-025-16527-y">
<meta name="citation_pmid" content="40851025">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375747/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375747/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375747/pdf/41598_2025_Article_16527.pdf">
<meta name="description" content="The Biogeochemical cycles are crucial for aquatic ecosystems, particularly the functioning of nutrient cycles. Growing anthropogenic pressure alters their functioning, causing accelerated eutrophication. Understanding the scale of alteration in ...">
<meta name="og:title" content="Exploring biogeochemical transformation in a hypertrophic lake with dual focus on environmental cues and potential functional fingerprints of littoral bacteria">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The Biogeochemical cycles are crucial for aquatic ecosystems, particularly the functioning of nutrient cycles. Growing anthropogenic pressure alters their functioning, causing accelerated eutrophication. Understanding the scale of alteration in ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375747/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375747">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-16527-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_16527.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375747%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375747/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375747/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375747/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 24;15:31115. doi: <a href="https://doi.org/10.1038/s41598-025-16527-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-16527-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Exploring biogeochemical transformation in a hypertrophic lake with dual focus on environmental cues and potential functional fingerprints of littoral bacteria</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dashora%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Milap Dashora</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Milap Dashora</span></h3>
<div class="p">
<sup>1</sup>Department of Environmental Science, School of Earth Sciences, Central University of Rajasthan, Bandarsindri, Kishangarh, Ajmer, Rajasthan 305817 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dashora%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Milap Dashora</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Manish Kumar</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Manish Kumar</span></h3>
<div class="p">
<sup>2</sup>Department of Environmental Studies, Faculty of Science, University of Delhi, New Delhi, 110007 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Manish Kumar</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaushik%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Garima Kaushik</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Garima Kaushik</span></h3>
<div class="p">
<sup>1</sup>Department of Environmental Science, School of Earth Sciences, Central University of Rajasthan, Bandarsindri, Kishangarh, Ajmer, Rajasthan 305817 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaushik%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Garima Kaushik</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Alok Kumar</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Alok Kumar</span></h3>
<div class="p">
<sup>2</sup>Department of Environmental Studies, Faculty of Science, University of Delhi, New Delhi, 110007 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alok Kumar</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Environmental Science, School of Earth Sciences, Central University of Rajasthan, Bandarsindri, Kishangarh, Ajmer, Rajasthan 305817 India </div>
<div id="Aff2">
<sup>2</sup>Department of Environmental Studies, Faculty of Science, University of Delhi, New Delhi, 110007 India </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 4; Accepted 2025 Aug 18; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375747  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40851025/" class="usa-link">40851025</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The Biogeochemical cycles are crucial for aquatic ecosystems, particularly the functioning of nutrient cycles. Growing anthropogenic pressure alters their functioning, causing accelerated eutrophication. Understanding the scale of alteration in ecological functions could assist with combat strategies. Bacterial communities regulate the transition of functional traits in aquatic ecosystems. However, current knowledge is still elusive regarding the variability of the environmental drivers dictating the functional roles in dynamic regimes like littoral zones. This study assessed how seasonal and circumlimnal conditions influenced sediment characteristics, bacterioplankton communities, and putative ecological functions (specifically biogeochemical cycling of carbon, nitrogen, and sulphur) in the hypertrophic freshwater Lake Pichola (Rajasthan, India). Findings suggest a significant seasonal drivers’ influence on sediment characteristics, while the anthropopressure-aided niche partitioning promoted species richness and diversity in the hypertrophic lake. Proteobacteria, Bacteroidota, and Firmicutes were abundant phyla; chemoheterotrophy and fermentation are the dominant potential putative functions; methylotrophy, nitrate reduction, and sulfate reduction are prominent nutrient cycling biogeochemical functions. Redundancy analysis accurately explained the influence of sediment characteristics on the variability of biogeochemical functions. Shifts in bacterioplankton communities were evident within circumlimnal regimes with greater anthropogenic stress, promoting higher abundance and diversity of putative bacteria orchestrating the ecological functions. This study emphasizes the importance of considering the interplay among the functional traits and environmental factors dynamics in littoral sediments for studying biogeochemical transformation in lakes.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Nutrients, Hypertrophic lakes, Carbon, Nitrogen, Sulphur, FAPROTAX</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Biogeochemistry, Environmental sciences, Limnology</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Human activities and climate change are foremost responsible for the extensive alteration of freshwater ecosystems on the Earth. Freshwater lakes constitute 90% of the global freshwater resources, covering just 2.8% of the land surface<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Pertinent wastewater discharge from industries, sewage outflow from dense urban centers, and agricultural runoff primarily add nutrients (carbon (C), nitrogen (N), &amp; phosphorous (P)) and other pollutants in lakes<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Although nutrients are crucial for sustaining life, excessive nutrient loadings (C, N, &amp; P) through anthropogenic activities cause alteration in these systems, including changes in trophic state, physico-chemical characteristics, and microbial communities<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Long-term accumulation of these nutrients could accelerate eutrophication, causing ecosystem degradation.</p>
<p id="Par3">Microorganisms, being sensitive, are the first to sustain the impacts of these alterations and could serve as ecosystem monitors and influence several ecological roles governed by microbiota, primarily energy transport and material cycling (C, N, P, and sulphur (S))<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. From the perspective of biogeochemical cycling, bacteria, and archaea are fundamentally crucial for ecological roles like the transformation of organic matter, nutrients, and pollutant degradation<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Examining their community structure could provide insights into the scale of these processes and assess ecosystem damage and their resilience toward these alterations. Technological advancements have replaced conventional laboratory cultures to study microbial communities with culture-independent high-throughput sequencing of the variable regions of 16S ribosomal rRNA gene with the help of efficient sequencing platforms like Illumina MiSeq<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. As the material cycling and energy transport occur at the sediment–water interface, lake sediments, and above-water columns have become extensive subjects of metagenomic analysis and next-generation sequencing, generating vast amplicon and shotgun metagenomic records<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>.</p>
<p id="Par4">The standalone richness and diversity estimate for prokaryotic species and community composition do not provide a clear understanding of the ecosystem functions<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>; therefore, in addition to community composition data, it is equally important to assess functional profiles. Shotgun metagenomics accurately represents both taxonomic and functional profiles for microbes, but has constraints like selective gene amplification, relatively higher cost per sample, and challenges with data processing<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Alternatively, in ecosystems with little background information on community composition or initial screening is the study’s objective, certain bioinformatics tools can be used to predict the functional profile of microbial communities based on taxonomic data obtained after sequencing<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>.</p>
<p id="Par5">Biogeochemical cycling is a crucial ecosystem function performed by bacteria present in lake sediments and water columns. Since bacterial community composition is highly dynamic, sharp changes in water regimes likely have far-reaching effects on biogeochemical cycles and ecosystem functioning in lakes. Nutrient cycling is a complex multi-step process, and apart from its physico-chemical counterparts, a variety of bacterial taxa might be involved at each step, exhibiting interactions within its community. A holistic approach should account for seasonal and circumlimnal conditions variability in the relationship between sediment characteristics and bacterial communities for a comprehensive understanding of sediment functions. Moreover, assessing bacterial community composition in eutrophic and hypertrophic lakes might be common. Still, only a few studies have tried to determine the ecological functions of bacterial communities<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. In contrast, a systematic study should account for community composition explaining the interactions<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, ecological and biogeochemical functions for functional profiling<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, the role of spatial and temporal changes in modulating the transformative rates of biogeochemical processes<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, and the influence of physico-chemical and environmental drivers<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> under the freshwater ecosystems.</p>
<p id="Par6">Current knowledge about the association between freshwater bacterial diversity and the environmental drivers governing the functional roles of freshwater bacterial communities is far from comprehensive, and their variability along different seasons and circumlimnal constraints is not very well accounted for within the highly dynamic littoral sediments of freshwater lakes. Considering the above rationale and the research gap, we framed a hypothesis that the interplay among circumliminal conditions and seasonal changes in dynamic littoral zones moderates environmental factors (physico-chemical characteristics), which in turn mediates the microbial community and associated biogeochemical functions. Therefore, this study aims to quantify key sediment physico-chemical parameters of littoral sediments, followed by estimation of bacterial community characteristics to infer overall putative ecological functions, mainly targeting the functions associated with biogeochemical cycling and influence from seasonal and circumlimnal factors. This study could provide deep insights into environmental forces that determine circumlimnal and seasonal differences in bacterial communities’ composition and alterations within the biogeochemical functions. Understanding the nutrient cycles in lakes could contribute to designing, evaluating, and establishing countermeasures to combat eutrophication-associated negative socio-economic impacts and maintain the aesthetic value of lakes.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Study site</h3>
<p id="Par7">The sampling site was Lake Pichola (LP) from Udaipur District, Rajasthan State, located in the northwest part of India at a mean elevation of 598 m with semi-arid climatic conditions. The district receives most of its rainfall during the monsoon season (July–September), and the region recorded 805 mm of mean annual rainfall in the past decade. The monthly mean temperatures range from 7 ºC in winter and peak up to 39.8 ºC during summer. Udaipur is a popular destination among international and national tourists, addressed by epithets like “City of Lakes”, and “Venice of the East”<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>–<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>.</p>
<p id="Par8">LP is an artificial freshwater lake that extends within 73º41′12′′ E – 73º39′40″ E longitude and 24º33′20″ N – 24º35′30″ N latitude (Fig. <a href="#Fig1" class="usa-link">1</a>), with water spread covering a 6.96 km<sup>2</sup> area. The lake was constructed to fulfill domestic water demand during the dry season, due to which major establishments settled around the northern part of the lake. Presently, with an increase in its popularity, the lake is under severe anthropogenic stress<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> due to activities like leakage from conventional sewage pipelines situated within the lake boundary, an increase in rooftop restaurants and luxury hotels that discharge their untreated organic waste directly into the lake, along with recreational activities like boat rides<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. This has resulted in the production of algal bloom (<em>Microcystis sp.</em>)<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> and water hyacinth (<em>Eichhornia crassipes)</em> as the floating type, <em>Hydrilla verticillata</em> as the submerged type, and <em>Paspalidium germinatum</em> as the emergent type macrophyte growth within the lake<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Conversely, the southern part of the lake is covered with scrub vegetation and less permeable clayey loam soil that serves as a nesting site for several local and migratory birds. The western part had inflow from the Sisarma River channel, and the eastern part had an embankment with three outflows: two towards the north and one towards the southern end.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375747_41598_2025_16527_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/45d2/12375747/5168a98f2ebd/41598_2025_16527_Fig1_HTML.jpg" loading="lazy" id="MO1" height="557" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Study area map with littoral sediment sampling sites of Lake Pichola in Udaipur District of Rajasthan State, India. Lake Pichola sampling sites (n = 19) with different markers represent a separate composite group based on the four (PI-PIV) distinct circumlimnal conditions around the lake. ArcGIS v10.6 (ESRI Inc., Redlands, CA, USA) is used to create the study area map. Land Use/Land Cover (LULC) thematic data was sourced as a Web Map Service (WMS) layer from NRSC Bhuvan portal (National Remote Sensing Centre (NRSC), ISRO. (2015–2016). Land Use Land Cover (1:50,000) – Rajasthan [Web Map Service layer]. Bhuvan. <a href="https://bhuvan-vec2.nrsc.gov.in/bhuvan/wms" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bhuvan-vec2.nrsc.gov.in/bhuvan/wms</a> (Accessed on May 24, 2024)).</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Littoral sediment sampling</h3>
<p id="Par9">Littoral sites in Lake Pichola were surveyed for sampling site selection, deeming the ease of access, considering circumlimnal conditions like point sources for pollutants, land use land cover (LULC), anthropogenic activity, and stream inflow. Accordingly, LP was divided into four groups (PI-PIV), and nineteen sites were selected to collect littoral sediment samples; representing all circumlimnal conditions. The sampling was conducted for three seasons, i.e. pre-monsoon (PR) in May month, monsoon (MO) in the month of September, and post-monsoon (PM) in the month of January within the year 2022–2023. The samples (approx. 10 g) were collected using a 22 cm long sterile stainless-steel spatula which can dig up to 15–20 cm below the surface. Where emergent macrophytes were available, rhizospheric sediments were collected. The sediment slurry was transferred into the sterile 50 mL falcon tubes. The caps of the falcons were sealed with parafilm and later transported in an ice box to the laboratory, where they were stored under −20 ºC before analysis. Additionally, replicate littoral sediment samples (~ 50 g) from all the sites were collected for physico-chemical analysis of samples in zipped bags and were stored at 4 ºC for physico-chemical and elemental analysis.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Sediment physico-chemical analysis</h3>
<p id="Par10">Sediment pH and oxidation–reduction potential (ORP) were measured in situ using portable electrodes Lutron PH-220S and Labtronics-LT50, respectively. Bulk density (BD) was estimated through the replicate sediment core collected using a corer, and dry mass was measured, which was divided against the known volume of the corer to obtain density. Soil moisture (SM) was estimated by a soil moisture meter (Precisa XM 60). Sediment organic matter (SOM) was estimated through titrimetric measurement of total organic carbon content in sediment samples<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. All measurements were repeated 5 times. 5 g of sediment sample from all the sites was oven dried at 60 ºC, sieved (63 µ) to obtain a homogenous fraction, and stored in air-tight capped vials for elemental analysis. Total nitrogen (TN) and total carbon (TC) were estimated for all sediment samples (~ 40 mg) through the elemental analyser (Elementar-Vario EL cube).</p></section></section><section id="Sec6"><h2 class="pmc_sec_title">Microbiological analysis</h2>
<section id="Sec7"><h3 class="pmc_sec_title">Metagenomic DNA extraction</h3>
<p id="Par11">Approximately 0.5 g (wet mass) of twelve composite sediment samples (PI-PIV) for three seasons (PR, MO, &amp; PM) was used to extract metagenomic DNA through a commercially available Nucleospin Soil Kit (MN) following the manufacturer’s instructions. The quality (purity and concentration) of the extracted metagenomic DNA from 12 samples was analyzed using NanoDrop by determining the A<sub>260</sub>/A<sub>280</sub> ratio.</p></section><section id="Sec8"><h3 class="pmc_sec_title">16S-rRNA amplicon sequencing and bacterial community analysis</h3>
<p id="Par12">High-throughput sequencing of bacteria was performed by selecting highly variable V3-V4 regions to generate the first amplicons. The primers used during polymerase chain reaction (PCR) for amplification were 341F (5′-GCCTACGGGNGGCWGCAG-3′) and 806R (5′-ACTACHVGGGTATCTAATCC-3′), designed and synthesized at Eurofins Genomics Lab. The PCR reaction used the Kapa HiFi HotStart mix (Roche). The reaction mixture (50 µL) contains 2 × pre-mix Taq (25 µL), forward and reverse primer (1 µL), DNA (3 µL), and nuclease-free water (20 µL). The PCR conditions are as follows: (1) Initial denaturation (one cycle) at 95 ºC for 3 min (2) Twenty-five cycles of a) denaturation at 95 ºC for 30 s b) renaturation at 55 ºC for 30 s c) extension at 72 ºC for 30 s (3) Final extension (1 cycle) at 72 ºC for 5 min followed by 4 ºC hold. For quality assessment of the PCR product, 3 µL of the product was resolved on 1.2% Agarose gel along with 100 bp DNA ladder RTU (GeneDireX) at 120 V for approximately 60 min or until the sample reached 3/4<sup>th</sup> of the gel.</p>
<p id="Par13">The amplicon library was prepared using Nextera XT Index Kit (Illumina Inc.) as per the 16S Metagenomic Sequencing Library Preparation protocol (Part # 15,044,223 Rev. B, Illumina, USA). The QC-passed amplicons with Illumina adaptors were amplified using i5 and i7 primers to add multiplexing index sequences and common adapters for cluster generation as per the standard Illumina protocol. The amplified libraries were purified by AMPureXP beads and quantified using Qubit 4.0 (Thermo Fisher). The quality of amplified libraries was analyzed on Bioanalyser 4200 Tape Station (Agilent Technologies) using D1000 Screen Tape as per manufacturer instructions. Based on the mean peak size from the tape station profile, libraries were loaded onto the Illumina MiSeq Platform using 2 × 300 bp chemistry at an appropriate concentration (10–20 pM) for cluster generation and sequencing.</p>
<p id="Par14">High-quality pair end (PE) reads were obtained using Trimmomatic v0.38 by removing adapter sequences, ambiguous reads having unknown nucleotides “N” &gt; 5%, and low-quality sequences (reads with more than 10% quality threshold (QV) &lt; 25 phred score) along with a sliding window of 20 bp and a minimum length of 100 bp. High-quality PE reads were stitched into the single-end (SE) reads using fluorescent-amplification-based specific hybridization (FLASH) v1.2.11<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> with an error rate of 2%, over 98% of read pairs correctly with default parameters. Denoised chimeric sequences through Divisive Amplicon Denoising Algorithm (DADA2)<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup> and filtered them as per Qualitative Insights into Microbial Ecology (QIIME 2)<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Diversity analysis was performed based on observed features; besides the rarefaction curves, Taxonomic classification was performed on the amplicon sequence variants (ASV) with the q2-feature classifier using a pre-trained classifier based on the SILVA database<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Richness estimators, Chao1, and abundance-based coverage estimate (ACE), along with diversity metrics (α-diversity: Shannon’s index, Simpson index, Simpson reciprocal) were generated using QIIME and R based on normalized data.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Putative function predictions for biogeochemical cycling</h3>
<p id="Par15">The functional prediction was performed using the Functional Annotation of Prokaryotic Taxa (FAPROTAX) v1.2.10, a manually constructed database, which can determine the functions capable of governing ecological and metabolic functional profiles based on a database compiled from peer-reviewed literature and identify corresponding putative strains by utilizing the OTU (operational taxonomic unit) table generated from high-throughput sequencing<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. It uses a Python script to match the OTUs table to the FAPROTAX database of functional annotations and provides the functional profiles specifically for marine and lake biogeochemical cycles<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>.</p>
<p id="Par16">Although a few limitations are associated with the FAPROTAX database; it only includes cultured members of the strains. Secondly, it also assumes that if a cultured member of a taxon can perform any function, all the members (cultured and non-cultured) can perform the same function<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p>
<p id="Par17">Even after considering the above shortcomings, several recent studies from diverse domains have used the FAPROTAX database to study pathogen communities in polluted rivers<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, effects of nutrients on bacterial community structure in surface sediments in Lake system<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>, metabolic functions of the microbial community in indoor dust of university dormitories<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, variation in potential functions in cow manure during its transit in earthworm’s gut<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, functional properties of the bacterial community in surface sediment of nutrient-saturated lakes<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> and many more.</p>
<p id="Par18">Researchers also use other advanced tools for functional annotations, like the Phylogenetic Investigation of Communities by Reconstruction of Unobserved States (PICRUSt 2)<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> and Tax4Fun<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Our study majorly focuses on biogeochemical cycles, and the FAPROTAX database has functional annotations for major biogeochemical cycles such as carbon, nitrogen, and sulphur<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> and to account for the dynamics of biogeochemical functions within lake ecosystems. We considered FAPROTAX as it was a preliminary study, and we needed to screen a large number of samples (<em>n</em> = 12) and being a cost-effective alternative to shotgun sequencing (functional metagenomics)<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> other, researchers also prefer the putative functional profiles prediction approach over sequence homology-based genomic prediction approaches<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par19">The sediment physico-chemical data in triplicate was obtained for all 19 sites from LP, and the mean value was calculated for each composite group’s parameters. One-way analysis of variance with Fisher posthoc pair-wise comparison was applied at a 95% confidence interval (p &lt; 0.05) to determine significant differences within circumlimnal groups and seasons in Minitab 21.2. Redundancy analysis (RDA) was performed to determine the response of biogeochemical functional traits to sediment physico-chemical characteristics.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Physico-chemical properties of littoral sediments</h3>
<p id="Par20">Here, Fig. <a href="#Fig2" class="usa-link">2</a>A presents physico-chemical analyses of sediment samples, with the upper row showing seasonal variations (means across circumlimnal conditions) and the lower row comparing circumlimnal variations (means across seasons). Sediment pH was alkaline during PR and PM seasons but significantly decreased (p &lt; 0.01) to near neutrality during the MO season. ORP was significantly higher (p &lt; 0.001) during PR and PM seasons, indicating oxidizing conditions, while the MO season showed lower ORP, suggesting a reducing environment. Notably, the SM content was significantly (p &lt; 0.05) elevated during the PR season compared to the MO season. Considering the circumlimnal aspect, the PII group (Sisarma river inflow) exhibited a significantly higher SOM content (p &lt; 0.05) than the PIV group (embankment, greater depth). Furthermore, the C/N ratio was significantly higher (p &lt; 0.05) during the MO season than the PR and PM seasons.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375747_41598_2025_16527_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/45d2/12375747/e47410244f4a/41598_2025_16527_Fig2_HTML.jpg" loading="lazy" id="MO2" height="650" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Box plots representing pair-wise comparison for physico-chemical characteristics of composite littoral sediments from Lake Pichola with four circumlimnal site-group in the lower row (PI, PII, PIII, and PIV) and three seasons in upper row (PR, MO, and PM), (<strong>B</strong>) Pair-wise comparison of alpha diversity indices and observed features for 12 samples from Lake Pichola. The lowercase alphabet represents group labels, and groups that do not share the same letter have significantly different mean values. Fisher-Least Significant Difference (LSD) was applied for pair-wise comparison after one-way analysis of variance (ANOVA); PI-PIV represents circumlimnal groups in Lake Pichola while PR, MO and PM represent three seasons. For other abbreviations, refer to the main text.</p></figcaption></figure></section><section id="Sec13"><h3 class="pmc_sec_title">Amplicon sequencing and metagenomics</h3>
<p id="Par21">Sequencing of 12 composite littoral sediment samples across four circumlimnal groups and three seasons yielded 4,511,742 raw paired end (PE) reads. Following adapter and ambiguous read removal (FASTQC v0.12.1), 2,753,986 high quality (HQ) PE reads were retained (Supplementary Fig. <a href="#MOESM1" class="usa-link">S1</a>). These were stitched (FLASH), denoised by chimera filtering, and taxonomically classified. For Lake Pichola (16,349), OTUs were generated, and diversity indices and richness estimates were calculated. Rarefaction curves (Supplementary Fig. <a href="#MOESM1" class="usa-link">S2</a>) indicate optimal sequencing depth and representative bacterial OTUs for the lake sediment.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Alpha diversity</h3>
<p id="Par22">Within-sample diversity metrics of the microbial community from twelve samples were analyzed post-Illumina sequencing. Good’s coverage (~ 1) indicates that bacterial profiles accurately represent the original communities. Richness estimators (Chao1 and ACE) matched observed features, likely due to few unique features with similar abundance. Figure <a href="#Fig2" class="usa-link">2</a>B shows within-sample diversity, with the Shannon index considering rare taxa and the Simpson index considering abundance as a function for diversity (0–1, higher value = higher diversity)<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>; Simpson reciprocal was also calculated, which is the inverse of the Simpson index; it ranges from one to infinity, and a higher value denotes higher diversity. Figure <a href="#Fig2" class="usa-link">2</a>B (upper row) compares alpha diversity within circumlimnal groups across seasons, and the lower row compares across circumlimnal groups within seasons. The PI group exhibited significantly higher (p &lt; 0.05) observed features and diversity indices compared to the PII site, while no significant seasonal differences were observed.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Changes in bacterial community composition at phylum level</h3>
<p id="Par23">PR_PI sample exhibited a distinct bacterial profile (Fig. <a href="#Fig3" class="usa-link">3</a>), characterized by exceptionally high relative abundances of Patescibacteria (13%) and Planctomycetota (4%), alongside a relatively low Actinobacteriota abundance (2%). Desulfobacterota (6%) displayed a discrete rise in relative abundance within the PR_PII sample along with a low relative abundance of Actinobacteriota (2%). A notable Bacteroidota (24%) dominance was observed in PR_PIII sample rather than Proteobacteria phylum. Likewise, PR_PIV sample had the highest relative abundance for Acidobacteriota (7%) and Verrucomicrobiota (4%). Furthermore, MO_PI was characterized by the highest relative abundance of the Proteobacteria phylum, while MO_PII (21%) and MO_PIII (17%) shared the highest relative abundance for the Firmicutes phylum. Subsequently, MO_PIII exhibited the highest Actinobacteriota and Acidobacteriota relative abundance (7%), along with the lowest relative abundance for Patescibacteria phylum (1%). Planctomycetota (4%) and Chloroflexi (3%) displayed the highest relative abundance within the MO_PIV sample. Successively, during the PM period, PM_PI was marked by the highest Chloroflexi abundance (8%), while PM_PII was dominated by Bacteroidota (27%) and Firmicutes (21%), with a remarkably lowest Proteobacteria abundance (19%). Similarly, PM_PIII shared the highest Desulfobacterota abundance (6%) and the lowest relative abundance of Chloroflexi, Patescibacteria, Planctomycetota, and Verrucomicrobiota phyla. Lastly, PM_PIV showed high Verrucomicrobiota (4%) and low Acidobacteriota (2%). Interestingly, Actinobacteriota abundance was comparatively lowest within all PR season samples, Desulfobacterota phyla had consistently the lowest abundance during the MO season, while Planctomycetota phyla had the lowest relative abundance during the PM season.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375747_41598_2025_16527_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/45d2/12375747/de2ce31d9321/41598_2025_16527_Fig3_HTML.jpg" loading="lazy" id="MO3" height="382" width="708" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of bacterial community composition at phylum level based on the relative abundance of the first ten abundant phyla within twelve samples from Lake Pichola. Here, PR, MO, and PM represent three seasons, and PI_PIV represents four circumlimnal groups from Lake Pichola.</p></figcaption></figure></section><section id="Sec16"><h3 class="pmc_sec_title">FAPROTAX putative functions</h3>
<p id="Par24">The input feature table for LP had 16,349 OTU records. Using FAPROTAX database with 92 functional groups<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, 3,809 records (23.29%) were assigned to 73 functional groups in LP, and was consistent with recent findings<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Chemoheterotrophy was the dominant function in LP, followed by aerobic chemoheterotrophy, fermentation, animal parasites and symbionts, respiration of sulfur compounds, sulfate respiration, nitrate reduction, and nitrogen respiration (Supplementary Fig. <a href="#MOESM1" class="usa-link">S3</a>), aligning with previous studies<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. For convenience, all the FAPROTAX functions are categorized into seven broad function classes (Fig. <a href="#Fig4" class="usa-link">4</a>), but some functions associated with one class might be associated with other classes as well. One-way ANOVA followed by Fischer LSD post hoc test was used to assess significant differences in OTUs associated with each function, comparing either circumlimnal conditions (accounting for seasonal variability) or seasonal variations (accounting for circumlimnal variability).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375747_41598_2025_16527_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/45d2/12375747/051886fee039/41598_2025_16527_Fig4_HTML.jpg" loading="lazy" id="MO4" height="444" width="668" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Pair-wise comparison of variability from seasonal and circumlimnal factors within the available FAPROTAX ecological and metabolic functions for composite grouped samples (n = 12) of Lake Pichola, represented through a polar heatmap with color representing z-score based on normalization across samples. Hierarchical clustering generated 12 clusters for functions and 9 for samples in Lake Pichola. One-way analysis of variance was applied followed by the Fisher LSD post hoc test with appropriate normality and homogeneity checks; wherever required, and the box-cox transformation was applied. The significant differences were represented by group labels, where lowercase and uppercase labels separated by “;” represent three seasons (PR, MO, and PM) respectively separated by “,” and 4 circumlimnal groups (PI, PII, PIII, and PIV) for Lake Pichola. Superscript annotations on group labels represent * is p &lt; 0.05, ** is p &lt; 0.01, *** is p &lt; 0.001, and ns is not significant. The diamond-shaped label with numbers represents the respective function classes as per the legend at the bottom-left part of the plot.</p></figcaption></figure><p id="Par25">Looking at the carbon cycling function class, out of the 8 carbon cycle functions at the OTU level (Fig. <a href="#Fig4" class="usa-link">4</a>), methanogenesis, methanogenesis by CO<sub>2</sub> reduction with H<sub>2</sub>, hydrogenotrophic methanogenesis, methanogenesis by reduction of methyl compounds with H<sub>2</sub>, methanol oxidation, and methanotrophy do not differ significantly. However, methanogenesis using formate had significantly higher OTUs in the PI group (p &lt; 0.05) compared to PII and PIV, and also during the PR season (p &lt; 0.05) compared to the PM season. In the case of methylotrophy as well, there was no seasonal significant difference in methylotrophic OTUs, but PIV has significantly higher (p &lt; 0.05) methylotrophic bacteria OTUs than PIII and PII groups. Clustering of OTUs for carbon cycle functions like methanogenesis, methanogenesis by CO<sub>2</sub> reduction with H<sub>2</sub>, hydrogenotrophic methanogenesis, methanogenesis by reduction of methyl compounds with H<sub>2</sub>, methanogenesis using formate-associated OTUs were higher in the PI group during the PR period. Likewise, methanol oxidation OTUs were also prevalent in the PM season within the PI group. Methylotrophy and methanotrophy functions are clustered; their corresponding OTUs were higher in the MO season. OTU features for putative functions like acetoclastic methanogenesis and methanogenesis through disproportionation of methyl groups are absent in Lake Pichola.</p>
<p id="Par26">Statistical analysis revealed distinct seasonal and circumlimnal patterns in the distribution of operational taxonomic units (OTUs) associated with 14 nitrogen cycling functions (Fig. <a href="#Fig4" class="usa-link">4</a>). Notably, ureolysis, nitrogen respiration, and nitrate respiration exhibited no significant differences. However, nitrogen fixation-associated OTUs were significantly elevated (p &lt; 0.05) during the PM season compared to the MO season. From seasonal purview, the denitrification function (nitrate and nitrite) was significantly higher (p &lt; 0.05) in the PM season than in the PR and MO seasons, whereas from the circumlimnal prospect, the PI group had significantly higher (p &lt; 0.05) denitrification OTUs than the PIII and PIV groups. Conversely, nitrous oxide denitrification and anammox functions had no associated OTUs in LP. Looking at oxidative processes in N cycling, aerobic nitrite oxidation-associated OTUs were significantly higher (p &lt; 0.05) in the PR season and at the PIV site group highlighting the favorable conditions for the process. In the PM season, aerobic ammonia oxidation and nitrification-associated OTUs were significantly lower (p &lt; 0.01), indicating the possible unfavorable state. The clustering of these functions further supports this observation, with higher Z score values for the above functions during the PR period. Likewise, for reductive processes like nitrite/nitrate ammonification, the interaction of seasonal and circumlimnal conditions could govern these functions. Statistically, nitrite ammonification-associated OTUs were significantly lower (p &lt; 0.05) in the MO season but significantly higher (p &lt; 0.05) at the PIV site group. On the other hand, nitrate ammonification-associated OTUs were significantly higher (p &lt; 0.05) in the PM season and significantly lower (p &lt; 0.05) in the PIII group. Further, nitrite respiration-associated OTUs were significantly higher (p &lt; 0.05) in the PI group than PII group within the MO season. Nitrate reduction OTUs were significantly lower (p &lt; 0.01) in the PR season, with no significant circumlimnal differences. Conversely, ammonification clustering (nitrite and nitrate) was prevalent during the PR and PM seasons. Ureolysis, nitrogen respiration, nitrate respiration, and nitrite respiration functions clustered together and were substantially prevalent during the MO and PM periods. Finally, nitrite and nitrate denitrification, representing overall denitrification, were prevalent in the PM season.</p>
<p id="Par27">Nine putative sulphur cycling functions were assigned to OTUs in LP (Fig. <a href="#Fig4" class="usa-link">4</a>), broadly categorized as respiration and dark oxidation of sulfur compounds. Beginning with reduction functions, statistically, sulfate respiration function and associated OTUs show no seasonal significant difference, but the PIII group has significantly higher (p &lt; 0.01) assigned OTUs than the PIV and PII groups. Sulfite respiration and assigned OTUs also do not show any seasonal significant differences, but here again, the circumlimnal group PIII has significantly higher (p &lt; 0.05) associated OTUs than the PIV group. In contrast, for the sulfur respiration, no statistically significant differences were observed for either of the factors. Out of all the respiration functions, only thiosulfate respiration associated OTUs are significantly higher (p &lt; 0.001) in the PM season, at circumlimnal groups PII and PIII (p &lt; 0.05). Further, in the case of oxidation functions, OTUs associated with dark thiosulfate oxidation are significantly higher (p &lt; 0.01) in the PR season as compared to MO and PM seasons, but no significant differences were observed in associated OTUs from circumlimnal purview. Statistically, for the dark sulfur oxidation function, the assigned OTU features are significantly lower (p &lt; 0.05) within the MO season, but not significant within the circumlimnal aspect. Similarly, for dark sulfide oxidation, the assigned OTUs to the function are significantly higher during the PR period than during MO and PM seasons, with non-significant differences existing for circumlimnal groups. Dark sulfite oxidation function was not associated with any OTUs in LP. As per the clustering, it can be inferred that all the oxidative processes for sulphur cycling were clustered together, indicating the resemblance in functioning and governing factors. Instead, the functions categorized under respiration of sulfur compounds are all part of distinct clusters, except sulfur respiration and thiosulfate respiration.</p></section></section><section id="Sec17"><h2 class="pmc_sec_title">Variability of bacteria associated with biogeochemical cycling functions</h2>
<section id="Sec18"><h3 class="pmc_sec_title">Carbon cycle-associated functions and bacteria</h3>
<p id="Par28">For clarity, four FAPROTAX functions (methanogenesis, methanotrophy, methylotrophy, and methanol oxidation) were grouped under carbon cycling (Fig. <a href="#Fig5" class="usa-link">5</a>). Methanogenesis comprised six sub-categories (methanogenesis using formate, methanogenesis by CO<sub>2</sub> reduction with H<sub>2</sub>, methanogenesis by reduction of methyl compounds with H<sub>2</sub>, hydrogenotrophic methanogenesis, methanogenesis by disproportionation of methyl groups, and acetoclastic methanogenesis), with the last two methanogenesis sub-categories absent in Lake Pichola. In total, 272 putative bacteria/archaea were associated with carbon cycling in LP (Fig. <a href="#Fig5" class="usa-link">5</a>), contributing 27.21% to methanogenesis, 27.94% to methanotrophy, 38.66% (highest) to methylotrophy, and 6.25% (lowest) to methanol oxidation (Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>). Seasonally, functional contributions to carbon cycling were relatively higher within the PI and PIII groups, in the PR season, replaced by PIV and PI groups during the MO period, ultimately shifting to the PIV and PII groups in the PM season. From a circumlimnal aspect, the PI group follows the order (PR &gt; MO &gt; PM), PII (PM &gt; PR &gt; MO), PIII (PR &gt; MO ~ PM), and PIV (MO &gt; PM &gt; PR) for functional contribution to carbon cycling. Within the carbon cycling function class, methanogenesis involved fourteen types of putative bacteria/archaea (family-species level) across all twelve LP samples. Two of them (<em>Methanomassiliicoccus sp.</em> and the member from the Methanomassiliicoccaceae family) were contributing to methylotrophy function as well so their counts appear double in Fig. <a href="#Fig5" class="usa-link">5</a>. Assuming that they might equally contribute to both functions, their counts will be considered as half. Looking into the sub-categories of methanogenesis function and associated bacteria, based on the pathway (Fig. <a href="#Fig5" class="usa-link">5</a>), were: methanogenesis using formate (<em>Methanospirillum sp.</em>); CO<sub>2</sub> reduction with H<sub>2</sub> (<em>Methanobacterium sp.</em>, <em>Methanocorpusculum sp.</em>, <em>Methanolinea sp.</em>, <em>Methanoregula sp.</em>, <em>Methanospirillum sp.</em>, <em>Methanosarcina sp.</em>, and a Methanomicrobiales family member); reduction of methyl compounds with H<sub>2</sub> (<em>Methanomonassiliicoccus sp.</em> and a Methanomonassiliicoccaceae member); hydrogenotrophic methanogenesis ( <em>Methanobacterium sp.</em>, <em>Methanocorpusculum sp.</em>, <em>Methanolinea sp.</em>, <em>Methanoregula sp.</em>, <em>Methanosphaerula sp.</em>, <em>Methanospirillum sp.</em>, <em>Methanosarcina sp.</em>, <em>Methanomonassiliicoccus sp.</em>, and members from families Methanobacteriaceae, Methanomicrobiales, and Methanomonassiliicoccaceae). Looking into other functions; methanotrophy and methanol oxidation are distinct subsets of methylotrophy function. Methylotrophs encompass methanotrophs, methanol oxidizers, and methylotrophic methanogens utilizing single-carbon sources other than methane and methanol. Eighteen types of putative methanotrophs and twenty four methylotrophs with their samplewise contributions (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>), across all twelve LP samples, are shown in Fig. <a href="#Fig5" class="usa-link">5</a>.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375747_41598_2025_16527_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/45d2/12375747/2b6a5a44cb69/41598_2025_16527_Fig5_HTML.jpg" loading="lazy" id="MO5" height="524" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Alluvial plot representing variability from seasonal and circumlimnal factors within the carbon cycle associated putative functions and conforming bacterium (family/genus/species level) and their absolute counts predicted using FAPROTAX database in twelve samples from Lake Pichola. The bacterium names have a prefix separated by a “_” sign for designating taxonomic level where ‘f’ stands for family, ‘g’ for genus, and ‘s’ for species name.</p></figcaption></figure></section><section id="Sec19"><h3 class="pmc_sec_title">Nitrogen cycle-associated functions and bacteria</h3>
<p id="Par29">Conveniently for the nitrogen cycle, we considered the broad seven FAPROTAX functions (nitrogen fixation, nitrification, denitrification, ammonification, nitrogen respiration, nitrate reduction, and ureolysis) in the nitrogen cycling function class (Fig. <a href="#Fig4" class="usa-link">4</a>). Around 480 putative bacteria were associated with these functions across all twelve LP samples (Fig. <a href="#Fig6" class="usa-link">6</a>). Nitrate reduction (35%) had the highest overall contribution, followed by nitrogen respiration (20.63%), nitrification (20.42%), nitrogen fixation (10.42%), and ureolysis (9.79%). Denitrification and ammonification each contributed the lowest (1.88%) (Supplementary Table <a href="#MOESM1" class="usa-link">S2</a>). Seasonally, PR and MO seasons showed relatively higher contributions in PI and PIV groups, while the PM season showed higher contributions in PIV and PIII groups. From the circumlimnal perspective, in the PI group, contribution to nitrogen cycling in different seasons follows the order MO &gt; PR &gt; PM, for PII (PM &gt; MO &gt; PR), PIII (PM &gt; MO &gt; PR), lastly, for the PIV group, the order is PM &gt; MO &gt; PR, indicating the distinct patterns at the PI group. The nitrogen fixation function involved approximately sixteen distinct OTUs across all LP samples, with some potentially contributing to nitrogen respiration (2.50%), nitrate reduction (2.3%), ammonification (0.84%), and ureolysis (1.46%). Nitrification involved around fifteen putative bacteria (Fig. <a href="#Fig6" class="usa-link">6</a>), including <em>Nitrosomonas nitrosa</em> (0.21%), which also performs ureolysis. There are two sub-categories within nitrification: aerobic ammonia oxidation (involving all listed nitrifiers) and aerobic nitrite oxidation (only <em>Candidatus Nitrotoga sp.</em>). Subsequently, ammonification in LP involved around five bacteria (Fig. <a href="#Fig6" class="usa-link">6</a>). <em>Clostridium butyricum</em>, also a nitrogen-fixer, contributed to both nitrogen fixation and ammonification in certain samples (PR_PIII, PR_PIV, PM_PI, and PM_PIV) with 0.21% contribution within each. <em>Lactobacillus brevis</em> (MO_PII only) was an ammonifier not associated with nitrate reduction, unlike other ammonifiers (linked to both nitrogen respiration and nitrate reduction)<em>.</em> Ammonification is an aggregate of nitrate ammonification (<em>Sulfurospirilllum arsenophillum</em>, <em>Desulfovibrio vulgaris</em>, and <em>Anaeromyxobacter dehalogenans</em>, also capable of nitrite ammonification) and nitrite ammonification (<em>Lactobacillus brevis</em> and <em>Clostridium butyricum</em>). Additionally, within a few samples (PR_PI, PR_PII, MO_PIII, MO_PIV, and PM_PIII), ammonification-associated putative bacteria were absent. Four denitrifiers were associated with all LP samples (Supplementary Table <a href="#MOESM1" class="usa-link">S2</a>), all are also involved in nitrogen respiration and nitrate reduction, and absent in certain samples (PR_PI, PR_PII, PR_PIV, and PM_PIII) of LP (Fig. <a href="#Fig6" class="usa-link">6</a>). Within denitrification (nitrate, nitrite, and nitrous oxide), all putative denitrifiers performed nitrate and nitrite denitrification, but none performed nitrous oxide denitrification (Fig. <a href="#Fig6" class="usa-link">6</a>). Further, twenty-five OTUs (Fig. <a href="#Fig6" class="usa-link">6</a>) are associated within the nitrogen respiration function, which includes the nitrate-reducing and the nitrite ammonifying bacteria. Some are also linked to nitrogen fixation (12%), denitrification (12%), and ammonification (20%). Subsequently, nitrate reduction involved around forty-one bacteria, exclusively fifteen (Supplementary Table <a href="#MOESM1" class="usa-link">S2</a>), while others overlapping with nitrogen respiration (twenty-five), ammonification (<em>D. vulgaris</em>, <em>A. dehalogenans</em>, and <em>S. arsenophilum</em>), denitrification (<em>Paracoccus sp.</em>, <em>Zoogloea sp.</em>, and <em>D. multivorans</em>), and nitrogen fixation (<em>Azospira sp.</em>, <em>Magnetospirillum sp.</em>, and <em>D. tiedjei</em>). Ureolysis involved around fifteen distinct bacteria (Fig. <a href="#Fig6" class="usa-link">6</a>), exclusively nine, while the rest also involved in nitrate reduction (<em>B. funiculus</em> and <em>S. thalpophilum</em>), nitrification (<em>N. nitrosa</em>), and nitrogen fixation (<em>Azotobacter sp.</em>, <em>Azospirillum sp.</em>, and <em>Mesorhizobium sp.</em>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375747_41598_2025_16527_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/45d2/12375747/62924b652e4b/41598_2025_16527_Fig6_HTML.jpg" loading="lazy" id="MO6" height="573" width="669" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Alluvial plot representing variability from seasonal and circumlimnal factors within the nitrogen cycle associated putative functions and conforming bacterium (family/genus/species level) and their absolute counts predicted using FAPROTAX database in twelve samples from Lake Pichola. The bacterium names have a prefix separated by a “_” sign for designating taxonomic level where ‘f’ stands for family, ‘g’ for genus, and ‘s’ for species name.</p></figcaption></figure></section><section id="Sec20"><h3 class="pmc_sec_title">Sulphur cycle-associated functions and bacteria</h3>
<p id="Par30">For the sulphur cycle, considering seven FAPROTAX functions (sulfate respiration, sulfur respiration, sulfite respiration, thiosulfate respiration, dark sulfide oxidation, dark sulfur oxidation, and dark thiosulfate oxidation) (Fig. <a href="#Fig4" class="usa-link">4</a>), are broadly included under two major classes (respiration of sulfur compounds and dark oxidation of sulfur compounds), but to prevent repetition, they were excluded from the text and figures (Fig. <a href="#Fig7" class="usa-link">7</a>). Around 451 putative bacteria were associated with these functions across all twelve LP samples. Sulfate respiration (63.86%) had the highest contribution, followed by thiosulfate respiration (15.3%), dark sulfide oxidation (7.1%), sulfite respiration (6.43%), and dark thiosulfate oxidation (1.77%). Dark sulfur oxidation had the lowest contribution (0.89%) (Supplementary Table <a href="#MOESM1" class="usa-link">S3</a>). Seasonally, the PR season showed higher sulphur cycling contributions at PI and PIII, and the MO and PM seasons at PIII and PIV. Circumlimnally, PI and PII groups followed the order PR &gt; PM &gt; MO, PIII followed PR &gt; PM = MO, and PIV followed PM &gt; MO &gt; PR. Sulfate respiration involved twenty-four distinct OTUs (Fig. <a href="#Fig7" class="usa-link">7</a>), with <em>Desulfobulbus sp.</em> (3.77%), <em>D. biactus</em> (0.22%), and <em>D. multivorans</em> (0.22%) also contributing to sulfite respiration, whereas <em>Desulforhopalus sp.</em> (2.22%), <em>D. vulgaris</em> (0.22%), and <em>D. tiedjei</em> (0.22%) also contributing to thiosulfate respiration (Supplementary Table <a href="#MOESM1" class="usa-link">S3</a>). Sulfur respiration involved around five OTUs, with <em>S. arsenophilum</em> (0.44%) also associated with thiosulfate respiration. Sulfite respiration included four bacteria already linked to sulfate respiration (Fig. <a href="#Fig7" class="usa-link">7</a>). Thiosulfate respiration involved around six bacteria (Fig. <a href="#Fig7" class="usa-link">7</a>; Supplementary Table <a href="#MOESM1" class="usa-link">S3</a>). Dark oxidation of sulfur compounds involved around four bacteria, with <em>Thiobacillus sp.</em> (6.21%) being the major contributor to dark sulfide oxidation, followed by <em>Magnetovibrio sp.</em> (0.89%), which also contributed to dark thiosulfate oxidation along with <em>S. variabilis</em> (0.89%), which also contributed to dark sulfur oxidation (0.89%).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375747_41598_2025_16527_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/45d2/12375747/605b95fa2322/41598_2025_16527_Fig7_HTML.jpg" loading="lazy" id="MO7" height="524" width="669" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Alluvial plot representing variability from seasonal and circumlimnal factors within the sulphur cycle associated putative functions and conforming bacterium (family/genus/species level) and their absolute counts predicted using FAPROTAX database in twelve samples from Lake Pichola. The bacterium names have a prefix separated by a “_” sign for designating taxonomic level where ‘f’ stands for family, ‘g’ for genus, and ‘s’ for species name.</p></figcaption></figure></section><section id="Sec21"><h3 class="pmc_sec_title">Influence of environmental variables on functions associated with biogeochemical cycling</h3>
<p id="Par31">RDA was employed to assess variability in FAPROTAX putative functions related to C, N, and S biogeochemical cycling based on sediment physico-chemical parameters in twelve LP samples. The triplots (Fig. <a href="#Fig8" class="usa-link">8</a>) illustrate the influence of seasonal and circumlimnal factors on the inertia between sediment characteristics and biogeochemical functions. Figure <a href="#Fig8" class="usa-link">8</a>a,c,e on the left, depicts within seasonal variability across four circumlimnal groups, while the Fig. <a href="#Fig8" class="usa-link">8</a>b,d,f represents variability within the circumlimnal groups based on the three seasons for the littoral sediment characteristics and biogeochemical functions.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375747_41598_2025_16527_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/45d2/12375747/005e12b3e97a/41598_2025_16527_Fig8_HTML.jpg" loading="lazy" id="MO8" height="780" width="669" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Redundancy analysis (RDA) based estimation of the association between sediment physico-chemical characteristics and OTU counts associated with FAPROTAX putative function specifically explaining the variability of functions associated with biogeochemical cycling of carbon (<strong>a</strong>) &amp; (<strong>b</strong>), nitrogen (<strong>c</strong>) &amp; (<strong>d</strong>), and sulphur (<strong>e</strong>) &amp; (<strong>f</strong>) from Lake Pichola. Plots on the left side depict seasonal variability, while on the right represent variability among circumlimnal groups. The sample label in different colors indicates three seasons PR (Red), MO (Green), and PM (Purple), whereas dashed vectors with indigo labels represent FAPROTAX functions and solid black vectors represent sediment physico-chemical parameters. Higher magnitudes and closer distances of vectors represent a higher association.</p></figcaption></figure><p id="Par32">RDA for carbon cycling in Lake Pichola yielded four components (78.63% cumulative inertia). C-RDA I (51.14%) showed positive loadings for methanotrophy (0.65) and methylotrophy (0.64), negatively correlated with SOM (-0.44), ORP (-0.378), SM (-0.34), and pH (-0.29). Methanogenesis (0.49) in C-RDA II (16.68%) was negatively correlated with BD (-0.43) and C/N ratio (-0.35), and positively with ORP (0.32), TN (0.23), and SOM (0.22). Methanol oxidation (0.39) in C-RDA III (10.80%) was positively correlated with ORP (0.417), pH (0.414), and SM (0.32), and negatively with BD (-0.39) and C/N ratio (-0.28).</p>
<p id="Par33">RDA for nitrogen cycling generated five components (52.42% cumulative inertia). N-RDA I (20.51%) showed positive loadings for nitrogen fixation (0.43), nitrate ammonification (0.39), nitrate reduction (0.29), and nitrogen respiration (0.20), negatively correlated with SOM (-0.530), TC (-0.413), SM (-0.302), and C/N (-0.25), and positively with ORP (0.308) and BD (0.293). N-RDA II (15.13%) showed positive loading for nitrite ammonification (0.53), correlated positively with ORP (0.65) and SM (0.60), and negatively with C/N ratio (-0.35) and SOM (-0.34). N-RDA III (10.20%) showed positive loadings for nitrification (0.35) and nitrate reduction (0.34), correlated positively with C/N (0.80) and BD (0.52), and negatively with pH (-0.92), ORP (-0.46), and SM (-0.31). N-RDA IV (3.93%) showed positive loadings for nitrogen fixation and ureolysis, negatively correlated with TC (-0.65), TN (-0.59), and SOM (-0.34), and positively with BD (0.30). N-RDA V (2.65%) showed a negative loading for nitrogen respiration and a positive loading for denitrification, correlated positively with C/N ratio (0.32), TC (0.27), BD (0.23), and pH (0.18), and negatively with SOM (-0.48).</p>
<p id="Par34">RDA for sulphur cycling in LP sediments yielded three components (68.16% cumulative inertia). S-RDA I (42.69%) showed higher positive loadings for dark thiosulfate oxidation (0.61), dark sulfide oxidation (0.47), dark oxidation of sulfur compounds (0.46), dark sulfur oxidation (0.45), and sulfate respiration (0.23), and higher negative loadings for thiosulfate respiration (-0.40), sulfur respiration (-0.33), and respiration of sulfur compounds (-0.25), explained by positive loadings for SM (0.56), ORP (0.54), TN (0.30), TC (0.28) and negative loading for BD (-0.39). S-RDA II (15.00%) showed higher positive loadings for sulfate respiration (0.54), respiration of sulfur compounds (0.26), and sulfite respiration (0.22), explained by positive loadings for TN (0.72), TC (0.43), and SM (0.26), and negative loading for BD (-0.28). S-RDA III (10.47%) showed higher positive loadings for thiosulfate respiration (0.28) and respiration of sulfur compounds (0.26), and a negative loading for sulfur respiration (-0.30), explained by positive loadings for pH (0.85), ORP (0.68), and SM (0.23) and negative loadings for C/N ratio (-0.43) and SOM (-0.33).</p></section></section><section id="Sec22"><h2 class="pmc_sec_title">Discussion</h2>
<section id="Sec23"><h3 class="pmc_sec_title">Physico-chemical properties of littoral sediments</h3>
<p id="Par35">Physico-chemical characteristics and their seasonal and circumlimnal variability had crucial role in influencing microbial abundance and diversity<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. pH is an important parameter that governs the abundance and diversity of microbes<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Onsite pH estimation indicated neutral to weakly alkaline conditions. Sediment characteristics (except SOM) had no significant differences based on circumlimnal conditions, seasonal variations were significant, with pH lower (near neutral) during the MO season in Lake Pichola, potentially due to rainfall dilution<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup> and organic load from stream runoffs leading to organic acid production<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>; alternatively, the addition of suspended matter might limit photosynthetic activity in the water column and carbon dioxide removal causes pH shifts<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. ORP, a redox state indicator, had no significant differences among circumlimnal groups, but oxidizing conditions in PR and PM seasons significantly dropped during the MO season, possibly due to organic load influx from monsoon runoff and microbial decomposition<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. BD, influenced by SM, SOM, and fine particulates, was below 1 g/cm<sup>3</sup> in LP across seasons and groups, likely due to fine silt and clay and biogenic siliceous diatomaceous material<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, supported by the inverse relation between SM and BD in most groups and seasons. Significantly higher SM in the PR season is likely due to abundant finer fractions retaining excess moisture. Allochthonous SOM was significantly higher at PII than PIV, possibly from upstream organic matter input at PII during the MO season from agricultural and sewage discharges via the Sisarma River; autochthonous SOM production occurred at outflow sites PI and PIII later in the PR season. TC content showed the highest seasonal variability in the PII group and the lowest in the PIV group, with the relatively highest TC in the PR season, gradually reducing in MO and reaching the lowest in PM, a trend mirrored by SOM, suggesting a major organic carbon contribution. TN showed no significant seasonal or inter-group differences in Lake Pichola, but sites PII and PI had substantially higher TN levels from agricultural runoff and untreated sewage near the Sisarma River catchment. Anthropogenic inputs at PI from densely populated areas and sewage seepage might contribute to TN concentration in PR season with reduced water levels<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. The MO season in LP had a significantly higher C/N ratio, with substantially higher values at PI and PIII sites, indicating organic-rich load transport from the PII group towards the two outflows. Groups PII and PIV had relatively lower C/N ratios, significantly lower in PR and PM seasons, suggesting TN retention and TC removal within Lake Pichola.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Spatio-temporal variability in richness and diversity estimators</h3>
<p id="Par36">Observed features (OTUs) displayed intra-lake variability, with significantly higher mean values at the PI group after accounting for annual seasonal changes. The Shannon index indicated substantially higher species richness at PI, while Simpson reciprocal values showed significant intra-lake diversity differences due to variations in species evenness between the PI and PII groups. This aligns with global studies on anthropogenically influenced freshwater lakes, where nutrient enrichment and pollution tend to increase bacterial diversity by creating diverse ecological niches<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Seasonally, considering the entire lake, significantly higher OTUs were present in the PR season at LP. Furthermore, the Shannon index revealed intra-season differences in the MO season, with significantly higher bacterial species richness in LP, as seasonal changes in temperature and nutrient levels are key drivers of bacterial community structure in lakes and can significantly affect OTU richness and diversity<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Variability in bacterial community representation at the phylum level</h3>
<p id="Par37">Proteobacteria, Bacteroidota, and Firmicutes abundance exhibit a stronger seasonal influence than circumlimnal differences. Although less seasonally variable in tropical aquatic ecosystems, both are influenced by monsoon cycles and agricultural runoff<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>, contrasting with temperate regions showing their higher abundance in summer and autumn due to increased organic matter and primary productivity<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Other studies on Lake Erie (USA), Lake Geneva (Switzerland), and Lake Taihu (China) have also reported seasonal variation in Proteobacteria dominance, particularly in regions with high nutrient availability and organic matter inflow<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Likewise, Bacteriodota occurs in wastewater-impacted lakes, eutrophic reservoirs, and post-monsoon floodplains; conversely, firmicutes, occur under a hypoxic sedimentary environment<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Firmicutes, having the ability of spore formation, can survive even in extremely harsh conditions, suggesting an adaptive response to seasonal shifts, perhaps could ferment a wide range of organic compounds like butyrate and acetate<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Patescibacteria is widely detected in deep subsurface environments and oligotrophic lakes like Lake Baikal (Russia), Lake Michigan (USA), and Lake Birwa (Japan), and also among the freshwater sediments<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. They are metabolically streamlined and lack key biosynthetic pathways, mostly dependent on host bacteria or symbiotic association for survival but have been linked to fermentation-based metabolism and interactions with biogeochemical cycling<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. The significantly higher abundance of Patescibacteria in Lake Pichola at the PI site during the PR season is possibly due to the shallow water regime, low turbulence, and organic matter accumulation. Although seasonal variations are less studied in this phylum, their higher abundance might correlate with the host activity level<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. Chloroflexi, prevalent in wastewater treatment plants (WWTPs), freshwater, and marine sediments with notably high organic matter inputs and includes photoheterotrophic, chemoheterotrophic, and anaerobic bacteria with crucial roles like organic matter degradation, nitrogen removal, and hydrogen metabolism<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. Their relatively higher abundance at the PI group is explained by the availability of organic matter and nutrients in a low-energy environment and low oxygen availability, possibly from domestic sewage pollution at this site, where they could assist in remineralizing organic matter<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Both, Actinobacteriota and Acidobacteriota abundance were influenced by season rather than circumlimnal conditions, having a moderate abundance, indicating their consistent presence. Actinobacteriota is commonly prevalent in rhizospheric sediments of aquatic plants. As per a study on the Heihe River Basin in China, they play a crucial role in nitrogen cycling and organic matter degradation<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. Similarly, Acidobacteriota aids in carbon cycling within the aquatic ecosystems through their ability to utilize diverse carbon sources, including xylan and chitin, acting as key players in organic matter decomposition<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Their presence is often associated with healthy microbial communities as they participate in nutrient cycling and contribute to ecosystem stability and resilience<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. Desulfobacterota, formerly (Deltaproteobacteria), is a group of sulfate-reducing bacteria (SRB) commonly found in marine sediments, freshwater wetlands, and tidal flats. They prefer anaerobic environments with sulfate availability, such as deep-sea hydrothermal vents, hot springs, and coastal sediments, and their distribution is influenced by environmental factors such as sulfate concentration, organic carbon availability, and redox conditions<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. Desulfobacterota performs aliphatic/aromatic hydrocarbon degradation and sulfate reduction through symbiotic association with fermenting microbes, utilizing the molecular hydrogen as a fermentation product for sulfate reduction and nitrate reduction, mitigating nitrate reduction in coastal and mangrove ecosystems. In peatlands and freshwater wetlands, they assist with carbon mineralization and methane mitigation by competing with methanogenic archaea for substrates like acetate and hydrogen, limiting methane emissions<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. Verrucomicriobiota usually thrive in organic-rich freshwater and marine environments, where they participate in organic matter degradation and nutrient cycling<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>; here, they showed significantly higher abundance in the PIV group in LP, suggesting niche preference. Planctomycetota, despite with lower abundance in LP, is ecologically important in lakes for its involvement in the degradation of complex organic materials, performing anaerobic processes like fermentation, sulfur reduction, and denitrification<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. Additionally, they can thrive in different habitats, including sediments, water columns, and algal mats, exhibiting higher adaptability and survival strategies across different chemical gradients<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>.</p></section></section><section id="Sec26"><h2 class="pmc_sec_title">Influence of sediment characteristics on variability of putative OTUs associated with biogeochemical transformations</h2>
<section id="Sec27"><h3 class="pmc_sec_title">Carbon transformation</h3>
<p id="Par38">Lake Pichola, considering carbon cycling putative functions, exhibited a dominance of methylotrophic OTUs, with nearly equal methanogen and methanotroph abundance, and the lowest representation of methanol oxidizers, likely due to high substrate availability in its hypertrophic state<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. Methylotrophic bacteria are comparatively abundant in Lake Pichola and can oxidize simpler single-carbon compounds. Methanotrophs, which are a type of methylotroph<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>, are more abundant in LP as compared to methanol oxidizing bacteria, particularly Type I rather than Type II methanotrophs. It implies that the sediment has high oxygen levels and methanotrophs are more responsive to even lower methane concentrations<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. The methanotrophic communities in the lake are typically dominated by gammaproteobacterial methanotrophs (Methylococcales order) and substantially lower alphaproteobacterial methanotrophs (Type II)<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. LP has two distinct genera of Type II methanotrophs (<em>Methylocella sp.</em> and <em>Methylocystis sp.</em>), possibly as hypertrophic conditions may provide abundant substrates for methanogenesis, leading to increased methane production. This, in turn, supports a higher abundance of Type II methanotrophs<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>, as compared to oligotrophic lakes<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. C-RDA I had high positive loading for methanotrophy (0.65) for MO_PIV sample, followed by PM_PIV and MO_PI. In MO_PIV sample, the abundant methanotrophs was <em>Methylobacter sp.</em> (1.84%). Being a methanotroph, it could share its methane-derived oxidized carbon produced in PR season with methylotrophs like <em>Methylotenera sp.</em> (MO_PIV and PM_PIV) during MO and PM seasons for further oxidation, which does not require the cell to cell transfer but is aided by nitrate in eutrophic lakes<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. Other associated methanotrophs with the MO_PIV sample were <em>Methylomicrobium sp.</em> (1.47%), an obligatory methanotroph and members from the family Methylomonadaceae (1.10%). The environmental factors based on C-RDA I indicates the combination of relatively lower ORP and lower moisture content during the MO season in LP which likely creates more favorable micro-oxic conditions and better gas exchange within the upper sediment, allowing methanotrophs to access both methane (from deeper anoxic layers) and oxygen (from the water column/diffusion) more effectively. The relatively lower organic matter might reduce extreme anoxia and competition, while the slightly alkaline pH is within the preferred range for many methanotrophic species. This suite of conditions in MO_PIV and PM_PIV samples possibly shifts the balance, favoring methane consumption over methane production or other anaerobic processes at the sediment–water interface. In the MO_PI sample, as well, there are additional abundant OTUs like <em>Methylomonas sp.</em> along with <em>Methylomicrobium sp.</em>, both being Type I methanotroph along with <em>Crenothrix sp.,</em> which is although associated with manganese and iron oxidation, but here it is oxidizing methane, acting as a connecting link between these cycles.</p>
<p id="Par39">Considering the C-RDA II with high positive loading for Methanogenesis (0.49), the combination of high organic matter, relatively lower bulk density, a C/N ratio that supports an active anaerobic food web, and ORP conditions indicative of active anoxic micro-niches collectively created highly favorable conditions for the growth and activity of diverse methanogenic archaea, leading to their relatively higher abundance in the samples (PR_PI, PM_PI, PM_PII, and PR_PIV). Particularly PR_PI sample has the possible dominance of the hydrogenotrophic methanogenesis pathway in LP, like other eutrophic lakes<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>, based on the abundance of <em>Methanobacterium sp.</em> (1.10%) and <em>Methanospirillum sp.</em> (0.74%), which is also linked with CO<sub>2</sub> reduction by H<sub>2</sub> and methanogenesis using the formate pathway. Additionally, the members of Methanomassillicoccaceae are also abundant (0.74%) in PR_PIV and are methylotrophic methanogens, capable of reducing methyl compounds with H<sub>2</sub> and producing methane. They consume non-competitive methylated substrates like methanol and methylamines when substrate limitation constricts the hydrogenotrophic and acetoclastic methanogenesis pathways<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. This is evident in (Fig. <a href="#Fig4" class="usa-link">4</a>) where no putative OTUs were associated with these functions, possibly due to the abundance of hydrogenotrophic methanogens.</p>
<p id="Par40">The methanol oxidation process, a part of the broader methylotrophy pathway for one-carbon compounds, utilizes methanol produced during CH<sub>4</sub> oxidation through methanotrophs and converts it to formaldehyde or further oxidize it to formate and CO<sub>2</sub> with the aid of enzymes like methanol dehydrogenases, phenazine methosulphate-dependent pteridine-type enzyme<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>. Considering the C-RDA III, the methanol oxidation function had positive loading (0.39), the reduced oxidation potential, relatively lower alkaline pH, higher BD and severe nitrogen limitation (high C/N) had selectively favoured the proliferation of <em>Methylobacillus sp.</em> and Methylophilaceae during MO season at PI site, whereas the combined effect of more oxic, alkaline, moist, and less compacted sediment with a favorable C/N ratio, likely created an optimal niche for <em>Paracoccus sp.</em> during PM season, leading to its higher abundance and association with methanol oxidation, demonstrating the influence of environmental parameters driving shifts in the functional composition of the methanol-oxidizing community. Interestingly, the Methylophilaceae and <em>Paracoccus sp.</em> are both capable of denitrification, but seasonal changes from MO to PM have shifted the community adapted to anaerobic denitrification in MO to aerobic denitrification in the PM season, indicating the ecosystem’s resilience and continuous removal of nitrogen.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Nitrogen transformation</h3>
<p id="Par41">Looking at the nitrogen cycle, N-RDA I with high positive loading for the nitrogen fixation and the nitrate ammonification-associated OTUs. Sediment characteristics at the PM_PIII site, a specific combination of moderately lower organic matter and moisture, a favorable C/N ratio, and crucially, microoxic conditions (higher ORP), collectively created an environment that selectively promoted the abundance of <em>Azospirillum sp.</em> and <em>Magnetospirillum sp.</em>, enabling their key role in nitrogen fixation within the hypertrophic lake. <em>Magnetospirillum sp.</em> abundance suggests the presence of dissolved iron, which is often released from the sediments during reducing conditions, and has been reported to biomineralize magnetite (Fe<sub>3</sub>O<sub>4</sub>) through magnetosome formation<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup> aided by diazotrophs like <em>Azospirillum sp.</em>, ensuring nutrient availability. Likewise, at the PII site, relatively higher SOM and BD than the PM_PIII sample, with an abundance of symbionts and cyanobacteria like <em>Nostoc sp.</em> PCC-7107 could be attributed to the presence of algal blooms. In PR_PI <em>Bradyrhizobium sp.</em> is primarily an abundant diazotroph and is also a phosphate mineralizing bacteria (PMB) that exhibit symbiotic association with aquatic plants and algae. Nitrate ammonifiers associated with N-RDA I include OTUs <em>Anaeromyxobacter dehalogenans</em> (PM_PI and PM_PII), along with <em>Desulfovibrio vulgaris</em> (MO_PI) and <em>Sulfurospirillum arsenophilum</em> (PM_PIV). <em>Anaeromyxobacter dehalogenans</em> is present only in PM season samples and is capable of both reduction of nitrate to ammonium and nitrous oxide to nitrogen gas, even without the typical nitrite reductase genes (<em>nir</em>S and <em>nir</em>K) by a combination of biotic and abiotic reactions, particularly in the presence of Iron through chemodenitrification, highlighting the bacterium’s ability to contribute to N loss in the environment with Iron abundance<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>. Additionally, <em>S. arsenophilum</em> could thrive in environments with lower hydrogen concentrations and perform respiration by using arsenate (As(V)) as a terminal electron acceptor similar to those of denitrifiers or Fe (III) reducers, providing a competitive advantage over other anaerobic respirators<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup>, indicating its capacity to modulate microbial processes like sulphur and iron cycling, influencing geochemistry of lake sediments<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. Similarly, another sulfate reducer, <em>D. vulgaris</em>, is also associated with ammonification in LP that thrives in environments with heavy metal contamination<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup> as well as facilitating the decomposition of organic compounds along with sulfate reduction even in the presence of oxygen<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup>. Although all nitrate ammonifiers are also associated with nitrite ammonification except <em>Lactobacillus brevis</em> (MO_PII), a fermenter that degrades organic matter and could indirectly reduce nitrite only, and <em>Clostridium butyricum</em> (PM_PIV, PM_PI, PR_PIII and PR_PIV), primarily associated with nitrogen fixation. The abundance of <em>C. butyricum</em> in PR and PM season is explained by N-RDA II with a specific balance of ORP (indicating a suitable redox environment), high moisture, and a favorable C/N and SOM supporting its metabolism, leading to significant nitrite ammonification. On the other hand, in the MO season, the significantly lower ORP in the monsoon season might be too extreme for the optimal activity of <em>Clostridium butyricum</em> for nitrite ammonification. The overall lower ORP and potentially less bioavailable SOM could reduce the overall capacity for efficient nitrite ammonification, leading to a shift in the dominant community member that can still perform the function, even if to a lesser extent. N-RDA III explains the abundance of nitrifiers and nitrate reducers, which are primarily explained by the C/N ratio and ORP. Major contributors to nitrification in LP were <em>MND</em>1, <em>Ellin</em>6067, and <em>mle</em>1-7. <em>MND</em>1 abundance (PR_PI, PR_PIV and MO_PIII) is dependent on relatively higher C/N as compared to PR_PIII and PM_PIV samples with <em>Ellin</em>6067 abundance. <em>MND</em>1 presence indicates active organic matter turnover, particularly degrading organic matter and release of methanol and methylamines during decomposition<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>; being a nitrifier, <em>MND</em>1 utilizes ammonia derived from methylamine degradation for nitrification reflecting functional interconnection<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>. <em>Ellin</em>6067, a member of Acidobacteria, often found in terrestrial soils, might be transported to lakes through runoff and erosion processes, explaining their abundance in the littoral zone<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. Whereas <em>mle</em>1-7 apart from nitrification, is also involved in the degradation of dissolved organic matter (DOM), particularly recalcitrant compounds, and their abundance can reflect the response of lakes to anthropogenic stress and natural environmental variations<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup>. <em>Nitrosomonas sp.,</em> along with <em>N. nitrosa</em> and <em>N. communis,</em> are abundant in LP and are a part of ammonia-oxidizing bacteria (AOB) that convert ammonia to nitrite<sup><a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup>, and they might compete for ammonia with ammonia-oxidizing archaea (AOA)<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup>. These species are indicators of nutrient pollution, particularly in eutrophic lakes with excessive nitrogen inputs (fertilizers and sewage), promoting ammonia accumulation<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>. <em>Nitrospira sp.</em> is also present in LP, which belongs to nitrite-oxidizing bacteria (NOB) and converts nitrite to nitrate, but its lower abundance is possibly influenced either by its competitive dynamics with Proteobacteria and Bacteroidetes<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup> that are relatively abundant in LP or, the suppression by high pollutant levels<sup><a href="#CR92" class="usa-link" aria-describedby="CR92">92</a></sup>. In case of nitrate reduction associated OTUs too, the relatively higher C/N value and ORP possibly influence the abundance of <em>Thauera sp.</em> at MO_PI. Studies have reported <em>Thauera sp.</em> as a core member of the microbial community involved in denitrification, is highly efficient in nitrogen removal as compared to other well-known denitrifiers, specifically in mixotrophic systems where both sulfur autotrophic and heterotrophic denitrifiers are present<sup><a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup>. In systems coupling partial denitrification with the anammox process, <em>Thauera sp.</em> plays a crucial role in nitrate reduction primarily through activating <em>nap</em>A/B/C genes involved in the nitrate reduction process<sup><a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup>. In PR_PI, PM_PIII, and PM_PIV samples, <em>Ferribacterium sp.</em> is a dominant nitrate reducer, and its abundance is primarily influenced by a relatively low C/N ratio and high positive ORP. Being an obligate anaerobe, it exists in microaerophilic environments, and few strains can reduce metals like Iron, Manganese, and Zinc<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup>. Relatively lower ORP and higher C/N ratio influence the abundance of <em>Rubrobacter sp.</em> (MO_PIII and PIV). Belonging to the Actinobacteria phylum, they show resilience against extreme environmental conditions like high temperatures, desiccation, and radiation, typically present in arid environments with high alkalinity and salinity<sup><a href="#CR96" class="usa-link" aria-describedby="CR96">96</a></sup>. It also possesses the ability to tolerate oxidative stress and higher concentrations of metals like iron and copper<sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup>. <em>A. butzleri</em> is abundant in LP and is commonly found in water systems contaminated with faecal matter, suggesting the presence of sewage or animal waste; its abundance may indicate poor water quality and inadequate sanitation in or around the lake<sup><a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>,<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a></sup>. Being a potential enteric pathogen, it often harbors virulence genes and may pose health risks if transmitted through contaminated water and cause gastrointestinal illness in humans<sup><a href="#CR100" class="usa-link" aria-describedby="CR100">100</a></sup>. <em>Veillonella sp.</em> might also play important roles in the microbial food web and biogeochemical cycles<sup><a href="#CR101" class="usa-link" aria-describedby="CR101">101</a></sup>, with another study reporting multiple antibiotic resistance (MARi) traits<sup><a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup>. Another enteric pathogenic bacteria <em>Aeromonas sp.</em>, is present in LP, with MARi traits; specifically, two virulent species <em>A. hydrophila</em> and <em>A. veronii,</em> were reported based on a study within lakes of Udaipur<sup><a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup> and Lake Erie<sup><a href="#CR103" class="usa-link" aria-describedby="CR103">103</a></sup>. <em>Dechloromonas sp.</em> apart from the nitrogen respiration is additionally associated with the degradation of contaminants, including dechlorination and nitrate reduction<sup><a href="#CR104" class="usa-link" aria-describedby="CR104">104</a></sup> under anaerobic conditions where it might take part in the degradation of microcystins<sup><a href="#CR105" class="usa-link" aria-describedby="CR105">105</a></sup>, also being a putative polyphosphate accumulating organism (PAO) possibly contributing to its removal in Lake Pichola explaining its presence in all seasons. Denitrification is primarily controlled by <em>Paracoccus sp.</em> followed by <em>Zoogloea sp.</em> and <em>Desulfococcus multivorans</em> in LP. <em>Zoogloea sp.</em> is an aerobic denitrifier reported to be present in oligotrophic lakes<sup><a href="#CR106" class="usa-link" aria-describedby="CR106">106</a></sup>, while strains like <em>Z. ramigera</em> occur majorly within eutrophic and mesotrophic lakes and perform N removal<sup><a href="#CR107" class="usa-link" aria-describedby="CR107">107</a></sup>. <em>D. multivorans</em> had been reported to perform other functions like sulfate reduction in anaerobic environments and completely metabolize carbon with the acetyl-CoA pathway under aerobic conditions, along with mercury methylation<sup><a href="#CR108" class="usa-link" aria-describedby="CR108">108</a></sup> and dechlorination of polychlorinated biphenyls (PCB), thereby playing a crucial role in decontamination of sediments<sup><a href="#CR109" class="usa-link" aria-describedby="CR109">109</a></sup>. Looking at Ureolysis function, <em>Roseomonas sp.</em> is abundant, particularly at the PI site during the PR season, being a part of the potential pathogenic community<sup><a href="#CR110" class="usa-link" aria-describedby="CR110">110</a></sup>, exists in environments with high levels of contaminants like nitrate and radionuclides, incorporating genes associated with nitrate reduction and resistance against heavy metal pollution<sup><a href="#CR111" class="usa-link" aria-describedby="CR111">111</a></sup>. Lake Pichola had three species of <em>Roseomonas</em> from which a novel strain of <em>R. lacus</em> was previously isolated from the sediments of Lake Taihu in China<sup><a href="#CR111" class="usa-link" aria-describedby="CR111">111</a></sup>, but further research is required to infer other ecological roles of the <em>R. ludipueritiae</em> and <em>R. terrae</em> in such systems.</p></section><section id="Sec29"><h3 class="pmc_sec_title">Sulphur transformation</h3>
<p id="Par42">The bacterial community associated with sulphur cycling had major proportions from bacteria associated with the respiration of sulfur compounds, under which the major contributors are from sulfate respiration, followed by thiosulfate, sulfite, and sulfur respiration. Sulfate respiration associated OTUs or sulfate reducing bacteria SRBs are abundant in PR_PI and PR_PIII sites explained by S-RDA II, under the influence of high TN content, and sediment moisture availability. <em>Desulfovibrio sp.</em>, being a SRB, is relatively more abundant in LP, suggesting active sulfate reduction within anoxic layers of sediments common in eutrophic lakes due to stratification and organic matter decomposition<sup><a href="#CR112" class="usa-link" aria-describedby="CR112">112</a></sup>. This bacterium could utilize different organic substrates and electron acceptors like sulfate, sulfite, and thiosulfate and is also reported to participate in direct interspecies electron transfer (DIET), facilitating methanogenesis by transferring electrons to methanogenic partners, aided by prevailing organic matter degradation under eutrophic conditions<sup><a href="#CR113" class="usa-link" aria-describedby="CR113">113</a></sup>. OTUs like members associated with the Syntrophobacteraceae family in LP are known for their syntrophic association with methanogens, particularly by the degradation of fatty acids and other organic compounds for utilization by methanogens to produce methane<sup><a href="#CR114" class="usa-link" aria-describedby="CR114">114</a></sup>. Likewise, members from the Desulfocapsaceae family are abundant in LP, and their presence is often associated with polluted environments, particularly those with high levels of organic pollutants and alkalinity exhibiting higher richness and diversity in nutrients-rich environments. <em>Synthrophobacter sp.</em> is also abundant in LP, and from a eutrophication context, members from Syntrophobacteraceae, along with other genera like <em>Geobacter</em> and <em>Methanocella,</em> may serve as indicators of nutrient-rich conditions and the ongoing biogeochemical transformations in the lake sediments<sup><a href="#CR115" class="usa-link" aria-describedby="CR115">115</a></sup>. In the case of sulfur respiration, <em>Sulfurospirillum sp.</em> is the major contributor, particularly <em>S. cavolei</em>, and <em>S. arsenophilum</em>, indicating complete metabolic pathways as a part of chemolithoautotrophic processes<sup><a href="#CR116" class="usa-link" aria-describedby="CR116">116</a></sup>, which can reduce different electron acceptors like thiosulfate, sulphur, nitrate and arsenate and might interact with iron cycling<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. <em>Fusibacter sp.</em> is a major contributor to thiosulfate respiration in LP and is capable of converting sulfur compounds like sulfate, thiosulfate, and elemental sulfur to sulfide and fermenting various carbohydrates, producing end products like acetate, ethanol, carbon dioxide, acetate, and hydrogen<sup><a href="#CR117" class="usa-link" aria-describedby="CR117">117</a></sup>. Additionally, it possesses genes for pathways like the Embden-Meyerhof glycolytic pathway and pentose phosphate pathway<sup><a href="#CR118" class="usa-link" aria-describedby="CR118">118</a></sup>. In the case of dark oxidation of sulfur compounds, major OTUs are associated with dark sulfide oxidation, followed by thiosulfate and sulfur oxidation. Overall, the variability in OTUs associated with dark oxidation of sulfur compounds is explained through S-RDA I with possible influence from sediment characteristics like positive ORP suggesting oxic conditions, followed by SM availability, and relatively lower BD, favoring dissolution of gases and exchange in sediment pore spaces. In Lake Pichola, <em>Thiobacillus sp.</em> is a major contributor in dark sulfide oxidation, followed by <em>Magnetovibrio sp.</em> and <em>Sulfurifustis variabilis</em>. <em>Thiobacillus sp.</em> belongs to the category of sulfur-oxidizing bacteria (SOB) that can oxidize sulfur compounds to sulfate in two phases: a sulfate producing phase and a sulfur and sulfate producing phase to adapt with variable sulfide concentration in case of obligate chemolithoautotrophs<sup><a href="#CR119" class="usa-link" aria-describedby="CR119">119</a></sup>. Alternatively, certain <em>Thiobacillus sp.</em> utilize both inorganic and organic substrates by growing mixotrophically<sup><a href="#CR120" class="usa-link" aria-describedby="CR120">120</a></sup>. A study on its use in bioreactors mentioned its potential roles in bio-augmenting processes like autotrophic denitrification and desulfurization<sup><a href="#CR121" class="usa-link" aria-describedby="CR121">121</a></sup>. <em>Magnetovibrio sp.</em> are microaerophiles, capable of both aerobic and anaerobic growth, exhibiting the chemolithoautotrophic growth on thiosulfate and sulfide with oxygen as a terminal acceptor and under anaerobic conditions, N<sub>2</sub>O acts as a terminal acceptor. They show a distinct lineage with <em>Magnetospirillum sp.</em>, exhibit magnetotactic behavior, and support chemo-organoautotrophic and methylotrophic growth under microaerophilic conditions<sup><a href="#CR122" class="usa-link" aria-describedby="CR122">122</a></sup>. <em>S. variabilis</em>, another SOB present in LP, is an autotrophic bacterium that utilizes thiosulfate, tetrathionate, and elemental sulphur as electron donors to support its chemolithoautotrophic growth<sup><a href="#CR123" class="usa-link" aria-describedby="CR123">123</a></sup> and genomic analysis revealed that it carries redundant genes in sulfur oxidation and inorganic carbon fixation pathways<sup><a href="#CR124" class="usa-link" aria-describedby="CR124">124</a></sup>.</p></section></section><section id="Sec30"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par43">Littoral sediments demonstrated dynamic functional potential for all considered biogeochemical putative functions that depend upon variable sediment characteristics, distinct circumlinmal, and seasonal variability. Ideally, seasonal variability is the major driver for altering sediment characteristics, which in turn alters functional traits. Conversely, for the altered trophic state, the interaction of seasonal and circumlimnal conditions governs the functional trait variability. The shifts within bacterial community composition counter the variable environmental conditions, ensuring the continuity of the functional roles even under unfavorable conditions and highlighting the ecosystem’s resilience. The interactions between the bacteria associated with the three biogeochemical cycles (C,N, and S), particularly the electron transfer and donor–acceptor association, were also evident and should be accounted for in future studies, even if the focus is on a specific element. The FAPROTAX database is comprehensive, and we recommend its use for getting insights into ecological functions. Although it covers major biogeochemical functions, it would be wholesome if phosphate metabolism functions were present as well.</p>
<p id="Par44">This study was the first attempt of its kind in the freshwater lakes from the semi-arid part of India. It demonstrated that the approach used in this study for estimating putative functions provides significant insights into quantitative estimates of functional profiles and their variability at an ecosystem scale. This approach is by far a cheaper and more reliable alternative where limited scientific literature exists about functional profiles, either only taxonomic features (OTUs) data is available, or screening of ecological functions is the objective in ecosystem-scale studies. Although thorough analysis, functional genomics, and enzymatic pathway analysis are recommended approaches in microcosm studies. Apart from providing a comprehensive outlook on variability assessment for biogeochemical function and underpinning the responsible factors, this study also emphasizes the invaluable importance of metagenomic analysis and functional profiling in assessing biogeochemical transformation.</p>
<p id="Par45">In Lake Pichola, the methylotrophs and chemoheterotrophs are the frontline defenders that combat environmental degradation. At the same time, nitrate reducers consistently reduce mobile forms of nitrogen, implying substantial N immobilization either through assimilation by phytoplankton or macrophyte assemblages. Sulfate reduction by SRB aids in direct interspecies electron transfer and exhibits symbiotic association with methanogens and denitrifiers for metabolizing complex organic compounds, acting as a connecting link for the C and N cycles. Emerging patterns suggest a strong interplay between anthropogenic pressures and seasonal cycles. Sites with significant anthropogenic inputs, such as PI and PIV, often exhibited distinct functional profiles across all three biogeochemical cycles, underscoring the direct influence of human activity on specific biogeochemical pathways. The higher circumlimnal variability consistently observed in the PM season across most cycles suggests that post-monsoon conditions might foster greater microbial functional diversity and differentiation across the littoral zone compared to other seasons. These findings collectively emphasize that while seasonality imposes a broad framework, localized anthropogenic pressures and micro-environmental variations are critical modulators of the microbial communities orchestrating the biogeochemical transformations in this hypertrophic lake.</p></section><section id="Sec31"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12375747/bin/41598_2025_16527_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information.</a><sup> (501.8KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>MD would like to thank the Department of Environmental Science, Central University of Rajasthan, Ajmer, and the Department of Environmental Studies, University of Delhi, for providing laboratory facilities for the completion of the analytical part of this work. MK and AK would gratefully acknowledge the generous financial support from DST-ANRF for providing a project grant (File no. EEQ/2022/000936). All the authors have reviewed the manuscript.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>M.D.: conceptualization, methodology, sample collection, data analysis, writing original draft; M.K.: sample collection, formal analysis of manuscript, review and editing; GK: supervision, review and editing, validation; AK: conceptualization, supervision, validation, editing and finalizing the manuscript, funding acquisition.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study was funded by the Anusandhan National Research Foundation (ANRF) under the framework of the Department of Science and Technology (DST), under the project File no. EEQ/2022/000936.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All raw sequences were deposited at the NCBI Sequence Read Archive as part of the Bioproject PRJNA1270394 (<a href="https://www.ncbi.nlm.nih.gov/sra/PRJNA1270394" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/sra/PRJNA1270394</a>) with all the accessions SRR33778681- SRR33778692. A detailed overview can be found in Supplementary Table <a href="#MOESM1" class="usa-link">S4</a>.” Also, the Supplementary Table <a href="#MOESM1" class="usa-link">S4</a> has been added to the Supplementary files.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par46">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="sec32"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>The online version contains supplementary material available at 10.1038/s41598-025-16527-y.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Downing, J. A. et al. The global abundance and size distribution of lakes, ponds, and impoundments. <em>Limnol. Oceanogr.</em><strong>51</strong>, 2388–2397. 10.4319/lo.2006.51.5.2388 (2006).</cite> [<a href="https://scholar.google.com/scholar_lookup?Downing,%20J.%20A.%20et%20al.%20The%20global%20abundance%20and%20size%20distribution%20of%20lakes,%20ponds,%20and%20impoundments.%20Limnol.%20Oceanogr.51,%202388%E2%80%932397.%2010.4319/lo.2006.51.5.2388%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Bhagowati, B. &amp; Ahamad, K. U. A review on lake eutrophication dynamics and recent developments in lake modeling. <em>Ecohydrol. Hydrobiol.</em><strong>19</strong>, 155–166. 10.1016/j.ecohyd.2018.03.002 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bhagowati,%20B.%20&amp;%20Ahamad,%20K.%20U.%20A%20review%20on%20lake%20eutrophication%20dynamics%20and%20recent%20developments%20in%20lake%20modeling.%20Ecohydrol.%20Hydrobiol.19,%20155%E2%80%93166.%2010.1016/j.ecohyd.2018.03.002%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Malone, T. C. &amp; Newton, A. The globalization of cultural eutrophication in the coastal ocean: Causes and consequences. <em>Front. Mar. Sci.</em><strong>7</strong>, 30. 10.3389/fmars.2020.00670 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Malone,%20T.%20C.%20&amp;%20Newton,%20A.%20The%20globalization%20of%20cultural%20eutrophication%20in%20the%20coastal%20ocean:%20Causes%20and%20consequences.%20Front.%20Mar.%20Sci.7,%2030.%2010.3389/fmars.2020.00670%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Wang, W. D., Liu, W. Y., Wu, D., Wang, X. X. &amp; Zhu, G. B. Differentiation of nitrogen and microbial community in the littoral and limnetic sediments of a large shallow eutrophic lake (Chaohu Lake, China). <em>J. Soils Sediments</em><strong>19</strong>, 1005–1016. 10.1007/s11368-018-2090-4 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang,%20W.%20D.,%20Liu,%20W.%20Y.,%20Wu,%20D.,%20Wang,%20X.%20X.%20&amp;%20Zhu,%20G.%20B.%20Differentiation%20of%20nitrogen%20and%20microbial%20community%20in%20the%20littoral%20and%20limnetic%20sediments%20of%20a%20large%20shallow%20eutrophic%20lake%20(Chaohu%20Lake,%20China).%20J.%20Soils%20Sediments19,%201005%E2%80%931016.%2010.1007/s11368-018-2090-4%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Wang, Y. et al. Shifts in microbial communities in shallow lakes depending on trophic states: Feasibility as an evaluation index for eutrophication. <em>Ecol. Ind.</em><strong>136</strong>, 11. 10.1016/j.ecolind.2022.108691 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Y.%20et%20al.%20Shifts%20in%20microbial%20communities%20in%20shallow%20lakes%20depending%20on%20trophic%20states:%20Feasibility%20as%20an%20evaluation%20index%20for%20eutrophication.%20Ecol.%20Ind.136,%2011.%2010.1016/j.ecolind.2022.108691%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Kumar, R. et al. Getting started with microbiome analysis: sample acquisition to bioinformatics. <em>Curr. Prot. Hum. Genet.</em>10.1002/0471142905.hg1808s82 (2014).</cite> [<a href="https://doi.org/10.1002/0471142905.hg1808s82" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4383038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25042718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20R.%20et%20al.%20Getting%20started%20with%20microbiome%20analysis:%20sample%20acquisition%20to%20bioinformatics.%20Curr.%20Prot.%20Hum.%20Genet.10.1002/0471142905.hg1808s82%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Parducci, L. et al. Shotgun environmental DNA, pollen, and macrofossil analysis of lateglacial lake sediments from southern Sweden. <em>Front. Ecol. Evol.</em>10.3389/fevo.2019.00189 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Parducci,%20L.%20et%20al.%20Shotgun%20environmental%20DNA,%20pollen,%20and%20macrofossil%20analysis%20of%20lateglacial%20lake%20sediments%20from%20southern%20Sweden.%20Front.%20Ecol.%20Evol.10.3389/fevo.2019.00189%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Cardinale, B. J. et al. Effects of biodiversity on the functioning of trophic groups and ecosystems. <em>Nature</em><strong>443</strong>, 989–992. 10.1038/nature05202 (2006).
</cite> [<a href="https://doi.org/10.1038/nature05202" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17066035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cardinale,%20B.%20J.%20et%20al.%20Effects%20of%20biodiversity%20on%20the%20functioning%20of%20trophic%20groups%20and%20ecosystems.%20Nature443,%20989%E2%80%93992.%2010.1038/nature05202%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Burgoyne, L., Koh, L. Y. &amp; Catcheside, D. Arbitrarily primed PCR for comparison of meta genomes and extracting useful loci from them. <em>Methods Mol. Biol.</em><strong>1620</strong>, 267–280. 10.1007/978-1-4939-7060-5_18 (2017).
</cite> [<a href="https://doi.org/10.1007/978-1-4939-7060-5_18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28540714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Burgoyne,%20L.,%20Koh,%20L.%20Y.%20&amp;%20Catcheside,%20D.%20Arbitrarily%20primed%20PCR%20for%20comparison%20of%20meta%20genomes%20and%20extracting%20useful%20loci%20from%20them.%20Methods%20Mol.%20Biol.1620,%20267%E2%80%93280.%2010.1007/978-1-4939-7060-5_18%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Aßhauer, K. P., Wemheuer, B., Daniel, R. &amp; Meinicke, P. Tax4Fun: predicting functional profiles from metagenomic 16S rRNA data. <em>Bioinformatics</em><strong>31</strong>, 2882–2884. 10.1093/bioinformatics/btv287 (2015).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv287" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4547618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25957349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?A%C3%9Fhauer,%20K.%20P.,%20Wemheuer,%20B.,%20Daniel,%20R.%20&amp;%20Meinicke,%20P.%20Tax4Fun:%20predicting%20functional%20profiles%20from%20metagenomic%2016S%20rRNA%20data.%20Bioinformatics31,%202882%E2%80%932884.%2010.1093/bioinformatics/btv287%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Langille, M. G. I. et al. Predictive functional profiling of microbial communities using 16S rRNA marker gene sequences. <em>Nat. Biotechnol.</em>10.1038/nbt.2676 (2013).
</cite> [<a href="https://doi.org/10.1038/nbt.2676" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3819121/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23975157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langille,%20M.%20G.%20I.%20et%20al.%20Predictive%20functional%20profiling%20of%20microbial%20communities%20using%2016S%20rRNA%20marker%20gene%20sequences.%20Nat.%20Biotechnol.10.1038/nbt.2676%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Louca, S., Parfrey, L. W. &amp; Doebeli, M. Decoupling function and taxonomy in the global ocean microbiome. <em>Science</em><strong>353</strong>, 1272–1277. 10.1126/science.aaf4507 (2016).
</cite> [<a href="https://doi.org/10.1126/science.aaf4507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27634532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Louca,%20S.,%20Parfrey,%20L.%20W.%20&amp;%20Doebeli,%20M.%20Decoupling%20function%20and%20taxonomy%20in%20the%20global%20ocean%20microbiome.%20Science353,%201272%E2%80%931277.%2010.1126/science.aaf4507%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Klump, J. V., Fitzgerald, S. A. &amp; Waples, J. T. Benthic biogeochemical cycling, nutrient stoichiometry, and carbon and nitrogen mass balances in a eutrophic freshwater bay. <em>Limnol. Oceanogr.</em><strong>54</strong>, 692–712. 10.4319/lo.2009.54.3.0692 (2009).</cite> [<a href="https://scholar.google.com/scholar_lookup?Klump,%20J.%20V.,%20Fitzgerald,%20S.%20A.%20&amp;%20Waples,%20J.%20T.%20Benthic%20biogeochemical%20cycling,%20nutrient%20stoichiometry,%20and%20carbon%20and%20nitrogen%20mass%20balances%20in%20a%20eutrophic%20freshwater%20bay.%20Limnol.%20Oceanogr.54,%20692%E2%80%93712.%2010.4319/lo.2009.54.3.0692%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Wan, Y., Ruan, X. H., Wang, J. &amp; Shi, X. J. Spatial and seasonal variations in the abundance of nitrogen-transforming genes and the microbial community structure in freshwater lakes with different trophic statuses. <em>Int. J. Environ. Res. Public Health</em><strong>16</strong>, 19. 10.3390/ijerph16132298 (2019).</cite> [<a href="https://doi.org/10.3390/ijerph16132298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6651097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31261730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wan,%20Y.,%20Ruan,%20X.%20H.,%20Wang,%20J.%20&amp;%20Shi,%20X.%20J.%20Spatial%20and%20seasonal%20variations%20in%20the%20abundance%20of%20nitrogen-transforming%20genes%20and%20the%20microbial%20community%20structure%20in%20freshwater%20lakes%20with%20different%20trophic%20statuses.%20Int.%20J.%20Environ.%20Res.%20Public%20Health16,%2019.%2010.3390/ijerph16132298%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Gallardo, B., Clavero, M., Sánchez, M. I. &amp; Vilà, M. Global ecological impacts of invasive species in aquatic ecosystems. <em>Glob. Change Biol.</em><strong>22</strong>, 151–163. 10.1111/gcb.13004 (2016).</cite> [<a href="https://doi.org/10.1111/gcb.13004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26212892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gallardo,%20B.,%20Clavero,%20M.,%20S%C3%A1nchez,%20M.%20I.%20&amp;%20Vil%C3%A0,%20M.%20Global%20ecological%20impacts%20of%20invasive%20species%20in%20aquatic%20ecosystems.%20Glob.%20Change%20Biol.22,%20151%E2%80%93163.%2010.1111/gcb.13004%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Graves, C. J. et al. Functional responses of salt marsh microbial communities to long-term nutrient enrichment. <em>Appl. Environ. Microbiol.</em><strong>82</strong>, 2862–2871. 10.1128/aem.03990-15 (2016).
</cite> [<a href="https://doi.org/10.1128/AEM.03990-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4836423/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26944843/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Graves,%20C.%20J.%20et%20al.%20Functional%20responses%20of%20salt%20marsh%20microbial%20communities%20to%20long-term%20nutrient%20enrichment.%20Appl.%20Environ.%20Microbiol.82,%202862%E2%80%932871.%2010.1128/aem.03990-15%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Soininen, J., Heino, J. &amp; Wang, J. J. A meta-analysis of nestedness and turnover components of beta diversity across organisms and ecosystems. <em>Glob. Ecol. Biogeogr.</em><strong>27</strong>, 96–109. 10.1111/geb.12660 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Soininen,%20J.,%20Heino,%20J.%20&amp;%20Wang,%20J.%20J.%20A%20meta-analysis%20of%20nestedness%20and%20turnover%20components%20of%20beta%20diversity%20across%20organisms%20and%20ecosystems.%20Glob.%20Ecol.%20Biogeogr.27,%2096%E2%80%93109.%2010.1111/geb.12660%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Ma, B. et al. Geographic patterns of co-occurrence network topological features for soil microbiota at continental scale in eastern China. <em>Isme J.</em><strong>10</strong>, 1891–1901. 10.1038/ismej.2015.261 (2016).
</cite> [<a href="https://doi.org/10.1038/ismej.2015.261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5029158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26771927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20B.%20et%20al.%20Geographic%20patterns%20of%20co-occurrence%20network%20topological%20features%20for%20soil%20microbiota%20at%20continental%20scale%20in%20eastern%20China.%20Isme%20J.10,%201891%E2%80%931901.%2010.1038/ismej.2015.261%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Auddy, N. et al. Assessing influence of anthropogenic and environmental factors on surface water quality for sustainable management of urban lakes: A case study from Udaipur, India. <em>Water Practice Technol.</em><strong>17</strong>, 1615–1634. 10.2166/wpt.2022.078 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Auddy,%20N.%20et%20al.%20Assessing%20influence%20of%20anthropogenic%20and%20environmental%20factors%20on%20surface%20water%20quality%20for%20sustainable%20management%20of%20urban%20lakes:%20A%20case%20study%20from%20Udaipur,%20India.%20Water%20Practice%20Technol.17,%201615%E2%80%931634.%2010.2166/wpt.2022.078%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Rao, G. &amp; Nandan, M. J. <em>Encyclopedia of Lakes and Reservoirs</em> (Springer, 2012).</cite> [<a href="https://scholar.google.com/scholar_lookup?Rao,%20G.%20&amp;%20Nandan,%20M.%20J.%20Encyclopedia%20of%20Lakes%20and%20Reservoirs%20(Springer,%202012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Samant, S. <em>Structural Studies, Repairs Maintenance of Heritage Architecture</em> (WIT Press, 2007).</cite> [<a href="https://scholar.google.com/scholar_lookup?Samant,%20S.%20Structural%20Studies,%20Repairs%20Maintenance%20of%20Heritage%20Architecture%20(WIT%20Press,%202007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Nair, N., Saini, D. &amp; Mahecha, G. S. Ecological research gaps in limnological studies: A case study on city of lakes, Udaipur. <em>Int. J. Sci. Res.</em><strong>5</strong>, 739–743 (2016).</cite> [<a href="https://scholar.google.com/scholar_lookup?Nair,%20N.,%20Saini,%20D.%20&amp;%20Mahecha,%20G.%20S.%20Ecological%20research%20gaps%20in%20limnological%20studies:%20A%20case%20study%20on%20city%20of%20lakes,%20Udaipur.%20Int.%20J.%20Sci.%20Res.5,%20739%E2%80%93743%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Mangal, H. &amp; Pathania, S. <em>Aquatic Ecosystem: Biodiversity, Ecology and Conservation</em> (Springer, 2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mangal,%20H.%20&amp;%20Pathania,%20S.%20Aquatic%20Ecosystem:%20Biodiversity,%20Ecology%20and%20Conservation%20(Springer,%202015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Billore, D. &amp; Vyas, L. <em>Wetlands Ecology Management</em> 45–54. (1982).</cite>
</li>
<li id="CR25">
<span class="label">25.</span><cite>Walkley, A. &amp; Black, I. A. An examination of the Degtjareff method for determining soil organic matter, and a proposed modification of the chromic acid titration method. <em>Soil Sci. Soc. Am. J.</em><strong>37</strong>, 29–38 (1934).</cite> [<a href="https://scholar.google.com/scholar_lookup?Walkley,%20A.%20&amp;%20Black,%20I.%20A.%20An%20examination%20of%20the%20Degtjareff%20method%20for%20determining%20soil%20organic%20matter,%20and%20a%20proposed%20modification%20of%20the%20chromic%20acid%20titration%20method.%20Soil%20Sci.%20Soc.%20Am.%20J.37,%2029%E2%80%9338%20(1934)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Magoc, T. &amp; Salzberg, S. L. FLASH: fast length adjustment of short reads to improve genome assemblies. <em>Bioinformatics</em><strong>27</strong>, 2957–2963. 10.1093/bioinformatics/btr507 (2011).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3198573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21903629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Magoc,%20T.%20&amp;%20Salzberg,%20S.%20L.%20FLASH:%20fast%20length%20adjustment%20of%20short%20reads%20to%20improve%20genome%20assemblies.%20Bioinformatics27,%202957%E2%80%932963.%2010.1093/bioinformatics/btr507%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Rosen, M. J., Callahan, B. J., Fisher, D. S. &amp; Holmes, S. P. Denoising PCR-amplified metagenome data. <em>BMC Bioinform.</em><strong>13</strong>, 16. 10.1186/1471-2105-13-283 (2012).</cite> [<a href="https://doi.org/10.1186/1471-2105-13-283" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3563472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23113967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosen,%20M.%20J.,%20Callahan,%20B.%20J.,%20Fisher,%20D.%20S.%20&amp;%20Holmes,%20S.%20P.%20Denoising%20PCR-amplified%20metagenome%20data.%20BMC%20Bioinform.13,%2016.%2010.1186/1471-2105-13-283%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Bolyen, E. et al. Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2. <em>Nat. Biotechnol.</em><strong>37</strong>, 852–857. 10.1038/s41587-019-0209-9 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0209-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7015180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31341288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bolyen,%20E.%20et%20al.%20Reproducible,%20interactive,%20scalable%20and%20extensible%20microbiome%20data%20science%20using%20QIIME%202.%20Nat.%20Biotechnol.37,%20852%E2%80%93857.%2010.1038/s41587-019-0209-9%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Quast, C. et al. The SILVA ribosomal RNA gene database project: improved data processing and web-based tools. <em>Nucleic Acids Res.</em><strong>41</strong>, D590–D596. 10.1093/nar/gks1219 (2013).
</cite> [<a href="https://doi.org/10.1093/nar/gks1219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3531112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23193283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quast,%20C.%20et%20al.%20The%20SILVA%20ribosomal%20RNA%20gene%20database%20project:%20improved%20data%20processing%20and%20web-based%20tools.%20Nucleic%20Acids%20Res.41,%20D590%E2%80%93D596.%2010.1093/nar/gks1219%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Yan, D. B., Xia, P. H., Song, X., Lin, T. &amp; Cao, H. P. Community structure and functional diversity of epiphytic bacteria and planktonic bacteria on submerged macrophytes in Caohai Lake, southwest of China. <em>Ann. Microbiol.</em><strong>69</strong>, 933–944. 10.1007/s13213-019-01485-4 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yan,%20D.%20B.,%20Xia,%20P.%20H.,%20Song,%20X.,%20Lin,%20T.%20&amp;%20Cao,%20H.%20P.%20Community%20structure%20and%20functional%20diversity%20of%20epiphytic%20bacteria%20and%20planktonic%20bacteria%20on%20submerged%20macrophytes%20in%20Caohai%20Lake,%20southwest%20of%20China.%20Ann.%20Microbiol.69,%20933%E2%80%93944.%2010.1007/s13213-019-01485-4%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Kumar, A., Ng, D. H. P., Wu, Y. C. &amp; Cao, B. Microbial community composition and putative biogeochemical functions in the sediment and water of tropical granite quarry lakes. <em>Microb. Ecol.</em><strong>77</strong>, 1–11. 10.1007/s00248-018-1204-2 (2019).
</cite> [<a href="https://doi.org/10.1007/s00248-018-1204-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29808411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20A.,%20Ng,%20D.%20H.%20P.,%20Wu,%20Y.%20C.%20&amp;%20Cao,%20B.%20Microbial%20community%20composition%20and%20putative%20biogeochemical%20functions%20in%20the%20sediment%20and%20water%20of%20tropical%20granite%20quarry%20lakes.%20Microb.%20Ecol.77,%201%E2%80%9311.%2010.1007/s00248-018-1204-2%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Yang, Y. Z., Hou, Y., Ma, M. &amp; Zhan, A. B. Potential pathogen communities in highly polluted river ecosystems: Geographical distribution and environmental influence. <em>Ambio</em><strong>49</strong>, 197–207. 10.1007/s13280-019-01184-z (2020).
</cite> [<a href="https://doi.org/10.1007/s13280-019-01184-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6888796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31020611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20Y.%20Z.,%20Hou,%20Y.,%20Ma,%20M.%20&amp;%20Zhan,%20A.%20B.%20Potential%20pathogen%20communities%20in%20highly%20polluted%20river%20ecosystems:%20Geographical%20distribution%20and%20environmental%20influence.%20Ambio49,%20197%E2%80%93207.%2010.1007/s13280-019-01184-z%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Chen, J. et al. Bacterial community structure in the surface sediments of different habitats of Baiyangdian Lake, Northern China: effects of nutrient conditions. <em>J. Soils Sediments</em><strong>21</strong>, 1866–1874. 10.1007/s11368-021-02901-6 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Chen,%20J.%20et%20al.%20Bacterial%20community%20structure%20in%20the%20surface%20sediments%20of%20different%20habitats%20of%20Baiyangdian%20Lake,%20Northern%20China:%20effects%20of%20nutrient%20conditions.%20J.%20Soils%20Sediments21,%201866%E2%80%931874.%2010.1007/s11368-021-02901-6%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Wu, Z. N. et al. Microbial community in indoor dusts from university dormitories: Characteristics, potential pathogens and influence factors. <em>Atmos. Pollut. Res.</em><strong>12</strong>, 274–286. 10.1016/j.apr.2020.12.018 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wu,%20Z.%20N.%20et%20al.%20Microbial%20community%20in%20indoor%20dusts%20from%20university%20dormitories:%20Characteristics,%20potential%20pathogens%20and%20influence%20factors.%20Atmos.%20Pollut.%20Res.12,%20274%E2%80%93286.%2010.1016/j.apr.2020.12.018%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Wang, N. et al. Variations in bacterial taxonomic profiles and potential functions in response to the gut transit of earthworms (<em>Eisenia fetida</em>) feeding on cow manure. <em>Sci. Total Environ.</em><strong>787</strong>, 10. 10.1016/j.scitotenv.2021.147392 (2021).</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2021.147392" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34000543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20N.%20et%20al.%20Variations%20in%20bacterial%20taxonomic%20profiles%20and%20potential%20functions%20in%20response%20to%20the%20gut%20transit%20of%20earthworms%20(Eisenia%20fetida)%20feeding%20on%20cow%20manure.%20Sci.%20Total%20Environ.787,%2010.%2010.1016/j.scitotenv.2021.147392%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Campos, M. A. et al. Structure and functional properties of bacterial communities in surface sediments of the recently declared nutrient-saturated lake Villarrica in Southern Chile. <em>Microb. Ecol.</em><strong>86</strong>, 1513–1533. 10.1007/s00248-023-02173-2 (2023).
</cite> [<a href="https://doi.org/10.1007/s00248-023-02173-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36752910/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Campos,%20M.%20A.%20et%20al.%20Structure%20and%20functional%20properties%20of%20bacterial%20communities%20in%20surface%20sediments%20of%20the%20recently%20declared%20nutrient-saturated%20lake%20Villarrica%20in%20Southern%20Chile.%20Microb.%20Ecol.86,%201513%E2%80%931533.%2010.1007/s00248-023-02173-2%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Douglas, G. M. et al. PICRUSt2 for prediction of metagenome functions. <em>Nat. Biotechnol.</em><strong>38</strong>, 685–688. 10.1038/s41587-020-0548-6 (2020).
</cite> [<a href="https://doi.org/10.1038/s41587-020-0548-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7365738/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32483366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Douglas,%20G.%20M.%20et%20al.%20PICRUSt2%20for%20prediction%20of%20metagenome%20functions.%20Nat.%20Biotechnol.38,%20685%E2%80%93688.%2010.1038/s41587-020-0548-6%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Xu, Z. C. et al. Relating bacterial dynamics and functions to gaseous emissions during composting of kitchen and garden wastes. <em>Sci. Total Environ.</em><strong>767</strong>, 10. 10.1016/j.scitotenv.2020.144210 (2021).</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2020.144210" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33429280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20Z.%20C.%20et%20al.%20Relating%20bacterial%20dynamics%20and%20functions%20to%20gaseous%20emissions%20during%20composting%20of%20kitchen%20and%20garden%20wastes.%20Sci.%20Total%20Environ.767,%2010.%2010.1016/j.scitotenv.2020.144210%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Xie, N. et al. Community composition and function of bacteria in activated sludge of municipal wastewater treatment plants. <em>Water</em><strong>13</strong>, 13. 10.3390/w13060852 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Xie,%20N.%20et%20al.%20Community%20composition%20and%20function%20of%20bacteria%20in%20activated%20sludge%20of%20municipal%20wastewater%20treatment%20plants.%20Water13,%2013.%2010.3390/w13060852%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Yang, J. X., Zhao, B., Zhang, P., Chen, D. Y. &amp; Chen, Y. P. Improvement in nitrogen removal and changes in community structure in a sequencing batch reactor bioaugmented with P. stutzeri strain XL-2. <em>Bioresour. Technol.</em><strong>317</strong>, 9. 10.1016/j.biortech.2020.123976 (2020).</cite> [<a href="https://doi.org/10.1016/j.biortech.2020.123976" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32805485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20J.%20X.,%20Zhao,%20B.,%20Zhang,%20P.,%20Chen,%20D.%20Y.%20&amp;%20Chen,%20Y.%20P.%20Improvement%20in%20nitrogen%20removal%20and%20changes%20in%20community%20structure%20in%20a%20sequencing%20batch%20reactor%20bioaugmented%20with%20P.%20stutzeri%20strain%20XL-2.%20Bioresour.%20Technol.317,%209.%2010.1016/j.biortech.2020.123976%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Li, W. et al. Variations in bacterial and archaeal community structure and diversity along the soil profiles of a peatland in Southwest China. <em>Environ. Sci. Pollut. Res.</em><strong>29</strong>, 2276–2286. 10.1007/s11356-021-15774-6 (2022).</cite> [<a href="https://doi.org/10.1007/s11356-021-15774-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34365597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20W.%20et%20al.%20Variations%20in%20bacterial%20and%20archaeal%20community%20structure%20and%20diversity%20along%20the%20soil%20profiles%20of%20a%20peatland%20in%20Southwest%20China.%20Environ.%20Sci.%20Pollut.%20Res.29,%202276%E2%80%932286.%2010.1007/s11356-021-15774-6%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Hartman, W. H., Richardson, C. J., Vilgalys, R. &amp; Bruland, G. L. Environmental and anthropogenic controls over bacterial communities in wetland soils. <em>Proc. Natl. Acad. Sci. U. S. A.</em><strong>105</strong>, 17842–17847. 10.1073/pnas.0808254105 (2008).
</cite> [<a href="https://doi.org/10.1073/pnas.0808254105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2584698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19004771/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hartman,%20W.%20H.,%20Richardson,%20C.%20J.,%20Vilgalys,%20R.%20&amp;%20Bruland,%20G.%20L.%20Environmental%20and%20anthropogenic%20controls%20over%20bacterial%20communities%20in%20wetland%20soils.%20Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.105,%2017842%E2%80%9317847.%2010.1073/pnas.0808254105%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Reddy, D. V., Madhav, T., Chandrakala, P. &amp; Nagabhushanam, P. A perspective of alkaline Lonar Lake, Maharashtra, India with reference to its hydrochemistry. <em>Curr. Sci.</em><strong>109</strong>, 965–975 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Reddy,%20D.%20V.,%20Madhav,%20T.,%20Chandrakala,%20P.%20&amp;%20Nagabhushanam,%20P.%20A%20perspective%20of%20alkaline%20Lonar%20Lake,%20Maharashtra,%20India%20with%20reference%20to%20its%20hydrochemistry.%20Curr.%20Sci.109,%20965%E2%80%93975%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Dhanush, S., Murthy, M., Verma, R. &amp; Sathish, A. Assessment of lake water quality and suitability for consumption, irrigation and industrial purpose using multivariate statistical approach, water quality indices (WQI) and geographic information system (GIS). <em>Mysore J. Agric. Sci.</em><strong>58</strong>, 5 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Dhanush,%20S.,%20Murthy,%20M.,%20Verma,%20R.%20&amp;%20Sathish,%20A.%20Assessment%20of%20lake%20water%20quality%20and%20suitability%20for%20consumption,%20irrigation%20and%20industrial%20purpose%20using%20multivariate%20statistical%20approach,%20water%20quality%20indices%20(WQI)%20and%20geographic%20information%20system%20(GIS).%20Mysore%20J.%20Agric.%20Sci.58,%205%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Nazneen, S. &amp; Raju, N. J. <em>Wastewater Reuse and Watershed Management</em> (Apple Academic Press, 2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Nazneen,%20S.%20&amp;%20Raju,%20N.%20J.%20Wastewater%20Reuse%20and%20Watershed%20Management%20(Apple%20Academic%20Press,%202019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Yan, G. H. et al. Effects of summer and autumn drought on eutrophication and the phytoplankton community in Dongting Lake in 2022. <em>Toxics</em><strong>11</strong>, 17. 10.3390/toxics11100822 (2023).</cite> [<a href="https://doi.org/10.3390/toxics11100822" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10610670/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37888674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan,%20G.%20H.%20et%20al.%20Effects%20of%20summer%20and%20autumn%20drought%20on%20eutrophication%20and%20the%20phytoplankton%20community%20in%20Dongting%20Lake%20in%202022.%20Toxics11,%2017.%2010.3390/toxics11100822%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Håkanson, L. &amp; Jansson, M. <em>Principles of Lake Sedimentology</em> (Springer, 1983).</cite> [<a href="https://scholar.google.com/scholar_lookup?H%C3%A5kanson,%20L.%20&amp;%20Jansson,%20M.%20Principles%20of%20Lake%20Sedimentology%20(Springer,%201983)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Lal, D. P. &amp; Sharma, B. K. Water quality status of Lake Pichhola, Udaipur, Rajasthan. <em>Int. J. Fish. Aquat. Stud.</em><strong>5</strong>, 181–187 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lal,%20D.%20P.%20&amp;%20Sharma,%20B.%20K.%20Water%20quality%20status%20of%20Lake%20Pichhola,%20Udaipur,%20Rajasthan.%20Int.%20J.%20Fish.%20Aquat.%20Stud.5,%20181%E2%80%93187%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Newton, R. J. &amp; McLellan, S. L. A unique assemblage of cosmopolitan freshwater bacteria and higher community diversity differentiate an urbanized estuary from oligotrophic Lake Michigan. <em>Front. Microbiol.</em>10.3389/fmicb.2015.01028 (2015).
</cite> [<a href="https://doi.org/10.3389/fmicb.2015.01028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4586452/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26483766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Newton,%20R.%20J.%20&amp;%20McLellan,%20S.%20L.%20A%20unique%20assemblage%20of%20cosmopolitan%20freshwater%20bacteria%20and%20higher%20community%20diversity%20differentiate%20an%20urbanized%20estuary%20from%20oligotrophic%20Lake%20Michigan.%20Front.%20Microbiol.10.3389/fmicb.2015.01028%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Liu, H. et al. Bacterial community assembly driven by temporal succession rather than spatial heterogeneity in Lake Bosten: a large lake suffering from eutrophication and salinization. <em>Front. Microbiol.</em><strong>14</strong>, 14. 10.3389/fmicb.2023.1261079 (2023).</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1261079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10552925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37808304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20H.%20et%20al.%20Bacterial%20community%20assembly%20driven%20by%20temporal%20succession%20rather%20than%20spatial%20heterogeneity%20in%20Lake%20Bosten:%20a%20large%20lake%20suffering%20from%20eutrophication%20and%20salinization.%20Front.%20Microbiol.14,%2014.%2010.3389/fmicb.2023.1261079%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Mikhailov, I. S. et al. Seasonal succession and coherence among bacteria and microeukaryotes in lake Baikal. <em>Microb. Ecol.</em><strong>84</strong>, 404–422. 10.1007/s00248-021-01860-2 (2022).
</cite> [<a href="https://doi.org/10.1007/s00248-021-01860-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34510242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mikhailov,%20I.%20S.%20et%20al.%20Seasonal%20succession%20and%20coherence%20among%20bacteria%20and%20microeukaryotes%20in%20lake%20Baikal.%20Microb.%20Ecol.84,%20404%E2%80%93422.%2010.1007/s00248-021-01860-2%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Huang, J. J., Gao, K. L., Yang, L. &amp; Lu, Y. H. Successional action of Bacteroidota and Firmicutes in decomposing straw polymers in a paddy soil. <em>Environ. Microbiome</em><strong>18</strong>, 12. 10.1186/s40793-023-00533-6 (2023).
</cite> [<a href="https://doi.org/10.1186/s40793-023-00533-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10576277/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37838745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20J.%20J.,%20Gao,%20K.%20L.,%20Yang,%20L.%20&amp;%20Lu,%20Y.%20H.%20Successional%20action%20of%20Bacteroidota%20and%20Firmicutes%20in%20decomposing%20straw%20polymers%20in%20a%20paddy%20soil.%20Environ.%20Microbiome18,%2012.%2010.1186/s40793-023-00533-6%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Kim, H. S., Lee, S. H., Jo, H. Y., Finneran, K. T. &amp; Kwon, M. J. Diversity and composition of soil Acidobacteria and Proteobacteria communities as a bacterial indicator of past land-use change from forest to farmland. <em>Sci. Total Environ.</em><strong>797</strong>, 10. 10.1016/j.scitotenv.2021.148944 (2021).</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2021.148944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34298360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20H.%20S.,%20Lee,%20S.%20H.,%20Jo,%20H.%20Y.,%20Finneran,%20K.%20T.%20&amp;%20Kwon,%20M.%20J.%20Diversity%20and%20composition%20of%20soil%20Acidobacteria%20and%20Proteobacteria%20communities%20as%20a%20bacterial%20indicator%20of%20past%20land-use%20change%20from%20forest%20to%20farmland.%20Sci.%20Total%20Environ.797,%2010.%2010.1016/j.scitotenv.2021.148944%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Eiler, A., Heinrich, F. &amp; Bertilsson, S. Coherent dynamics and association networks among lake bacterioplankton taxa. <em>Isme J.</em><strong>6</strong>, 330–342. 10.1038/ismej.2011.113 (2012).
</cite> [<a href="https://doi.org/10.1038/ismej.2011.113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3260505/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21881616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eiler,%20A.,%20Heinrich,%20F.%20&amp;%20Bertilsson,%20S.%20Coherent%20dynamics%20and%20association%20networks%20among%20lake%20bacterioplankton%20taxa.%20Isme%20J.6,%20330%E2%80%93342.%2010.1038/ismej.2011.113%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Ruiz-González, C., Niño-García, J. P. &amp; del Giorgio, P. A. Terrestrial origin of bacterial communities in complex boreal freshwater networks. <em>Ecol. Lett.</em><strong>18</strong>, 1198–1206. 10.1111/ele.12499 (2015).
</cite> [<a href="https://doi.org/10.1111/ele.12499" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26306742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ruiz-Gonz%C3%A1lez,%20C.,%20Ni%C3%B1o-Garc%C3%ADa,%20J.%20P.%20&amp;%20del%20Giorgio,%20P.%20A.%20Terrestrial%20origin%20of%20bacterial%20communities%20in%20complex%20boreal%20freshwater%20networks.%20Ecol.%20Lett.18,%201198%E2%80%931206.%2010.1111/ele.12499%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Machado, D. T. et al. Uncovering new Firmicutes species in vertebrate hosts through metagenome-assembled genomes with potential for sporulation. <em>Microbiol. Spectr.</em><strong>12</strong>, 25. 10.1128/spectrum.02113-24 (2024).</cite> [<a href="https://doi.org/10.1128/spectrum.02113-24" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11536998/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39283121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Machado,%20D.%20T.%20et%20al.%20Uncovering%20new%20Firmicutes%20species%20in%20vertebrate%20hosts%20through%20metagenome-assembled%20genomes%20with%20potential%20for%20sporulation.%20Microbiol.%20Spectr.12,%2025.%2010.1128/spectrum.02113-24%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Wrighton, K. C. et al. Fermentation, hydrogen, and sulfur metabolism in multiple uncultivated bacterial phyla. <em>Science</em><strong>337</strong>, 1661–1665. 10.1126/science.1224041 (2012).
</cite> [<a href="https://doi.org/10.1126/science.1224041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23019650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wrighton,%20K.%20C.%20et%20al.%20Fermentation,%20hydrogen,%20and%20sulfur%20metabolism%20in%20multiple%20uncultivated%20bacterial%20phyla.%20Science337,%201661%E2%80%931665.%2010.1126/science.1224041%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Brown, C. T. et al. Unusual biology across a group comprising more than 15% of domain Bacteria. <em>Nature</em><strong>523</strong>, 208-U173. 10.1038/nature14486 (2015).
</cite> [<a href="https://doi.org/10.1038/nature14486" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26083755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown,%20C.%20T.%20et%20al.%20Unusual%20biology%20across%20a%20group%20comprising%20more%20than%2015%%20of%20domain%20Bacteria.%20Nature523,%20208-U173.%2010.1038/nature14486%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Fujii, N. et al. Metabolic potential of the superphylum patescibacteria reconstructed from activated sludge samples from a municipal wastewater treatment plant. <em>Microbes Environ.</em><strong>37</strong>, 10. 10.1264/jsme2.ME22012 (2022).</cite> [<a href="https://doi.org/10.1264/jsme2.ME22012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9530719/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35768268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fujii,%20N.%20et%20al.%20Metabolic%20potential%20of%20the%20superphylum%20patescibacteria%20reconstructed%20from%20activated%20sludge%20samples%20from%20a%20municipal%20wastewater%20treatment%20plant.%20Microbes%20Environ.37,%2010.%2010.1264/jsme2.ME22012%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Hanada, S. The phylum Chloroflexi, the family Chloroflexaceae, and the related phototrophic families Oscillochloridaceae and Roseiflexaceae. <em>Prokaryotes</em>10.1007/978-3-642-38954-2_165 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hanada,%20S.%20The%20phylum%20Chloroflexi,%20the%20family%20Chloroflexaceae,%20and%20the%20related%20phototrophic%20families%20Oscillochloridaceae%20and%20Roseiflexaceae.%20Prokaryotes10.1007/978-3-642-38954-2_165%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Speirs, L. B. M., Rice, D. T. F., Petrovski, S. &amp; Seviour, R. J. The phylogeny, biodiversity, and ecology of the chloroflexi in activated sludge. <em>Front. Microbiol.</em><strong>10</strong>, 28. 10.3389/fmicb.2019.02015 (2019).
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.02015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6753630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31572309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Speirs,%20L.%20B.%20M.,%20Rice,%20D.%20T.%20F.,%20Petrovski,%20S.%20&amp;%20Seviour,%20R.%20J.%20The%20phylogeny,%20biodiversity,%20and%20ecology%20of%20the%20chloroflexi%20in%20activated%20sludge.%20Front.%20Microbiol.10,%2028.%2010.3389/fmicb.2019.02015%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Zhang, B. L. et al. Variation in actinobacterial community composition and potential function in different soil ecosystems belonging to the arid Heihe River Basin of Northwest China. <em>Front. Microbiol.</em><strong>10</strong>, 11. 10.3389/fmicb.2019.02209 (2019).
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.02209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6769177/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31608036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20B.%20L.%20et%20al.%20Variation%20in%20actinobacterial%20community%20composition%20and%20potential%20function%20in%20different%20soil%20ecosystems%20belonging%20to%20the%20arid%20Heihe%20River%20Basin%20of%20Northwest%20China.%20Front.%20Microbiol.10,%2011.%2010.3389/fmicb.2019.02209%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Crits-Christoph, A., Diamond, S., Al-Shayeb, B., Valentin-Alvarado, L. &amp; Banfield, J. F. A widely distributed genus of soil Acidobacteria genomically enriched in biosynthetic gene clusters. <em>ISME Commun.</em><strong>2</strong>, 70 (2022).
</cite> [<a href="https://doi.org/10.1038/s43705-022-00140-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9723581/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37938723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crits-Christoph,%20A.,%20Diamond,%20S.,%20Al-Shayeb,%20B.,%20Valentin-Alvarado,%20L.%20&amp;%20Banfield,%20J.%20F.%20A%20widely%20distributed%20genus%20of%20soil%20Acidobacteria%20genomically%20enriched%20in%20biosynthetic%20gene%20clusters.%20ISME%20Commun.2,%2070%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Kielak, A. M., Barreto, C. C., Kowalchuk, G. A., van Veen, J. A. &amp; Kuramae, E. E. The ecology of acidobacteria: Moving beyond genes and genomes. <em>Front. Microbiol.</em><strong>7</strong>, 16. 10.3389/fmicb.2016.00744 (2016).
</cite> [<a href="https://doi.org/10.3389/fmicb.2016.00744" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4885859/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27303369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kielak,%20A.%20M.,%20Barreto,%20C.%20C.,%20Kowalchuk,%20G.%20A.,%20van%20Veen,%20J.%20A.%20&amp;%20Kuramae,%20E.%20E.%20The%20ecology%20of%20acidobacteria:%20Moving%20beyond%20genes%20and%20genomes.%20Front.%20Microbiol.7,%2016.%2010.3389/fmicb.2016.00744%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Langwig, M. V. et al. Large-scale protein level comparison of Deltaproteobacteria reveals cohesive metabolic groups. <em>Isme J.</em><strong>16</strong>, 307–320. 10.1038/s41396-021-01057-y (2022).
</cite> [<a href="https://doi.org/10.1038/s41396-021-01057-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8692467/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34331018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langwig,%20M.%20V.%20et%20al.%20Large-scale%20protein%20level%20comparison%20of%20Deltaproteobacteria%20reveals%20cohesive%20metabolic%20groups.%20Isme%20J.16,%20307%E2%80%93320.%2010.1038/s41396-021-01057-y%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Dyksma, S. &amp; Pester, M. Growth of sulfate-reducing Desulfobacterota and Bacillota at periodic oxygen stress of 50% air-O2 saturation. <em>Microbiome</em><strong>12</strong>, 16. 10.1186/s40168-024-01909-7 (2024).
</cite> [<a href="https://doi.org/10.1186/s40168-024-01909-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11451228/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39367500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dyksma,%20S.%20&amp;%20Pester,%20M.%20Growth%20of%20sulfate-reducing%20Desulfobacterota%20and%20Bacillota%20at%20periodic%20oxygen%20stress%20of%2050%%20air-O2%20saturation.%20Microbiome12,%2016.%2010.1186/s40168-024-01909-7%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Freitas, S. et al. Global distribution and diversity of marine Verrucomicrobia. <em>Isme J.</em><strong>6</strong>, 1499–1505. 10.1038/ismej.2012.3 (2012).
</cite> [<a href="https://doi.org/10.1038/ismej.2012.3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3400412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22318305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Freitas,%20S.%20et%20al.%20Global%20distribution%20and%20diversity%20of%20marine%20Verrucomicrobia.%20Isme%20J.6,%201499%E2%80%931505.%2010.1038/ismej.2012.3%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Storesund, J. E. et al. Planctomycetes as a vital constituent of the microbial communities inhabiting different layers of the meromictic lake SÆlenvannet (Norway). <em>Microorganisms</em><strong>8</strong>, 18. 10.3390/microorganisms8081150 (2020).</cite> [<a href="https://doi.org/10.3390/microorganisms8081150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7464441/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32751313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Storesund,%20J.%20E.%20et%20al.%20Planctomycetes%20as%20a%20vital%20constituent%20of%20the%20microbial%20communities%20inhabiting%20different%20layers%20of%20the%20meromictic%20lake%20S%C3%86lenvannet%20(Norway).%20Microorganisms8,%2018.%2010.3390/microorganisms8081150%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Steven, B., Dowd, S. E., Schulmeyer, K. H. &amp; Ward, N. L. Phylum-targeted pyrosequencing reveals diverse planctomycete populations in a eutrophic lake. <em>Aquat. Microb. Ecol.</em><strong>64</strong>, 41–49. 10.3354/ame01507 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Steven,%20B.,%20Dowd,%20S.%20E.,%20Schulmeyer,%20K.%20H.%20&amp;%20Ward,%20N.%20L.%20Phylum-targeted%20pyrosequencing%20reveals%20diverse%20planctomycete%20populations%20in%20a%20eutrophic%20lake.%20Aquat.%20Microb.%20Ecol.64,%2041%E2%80%9349.%2010.3354/ame01507%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Yang, Y. Y. et al. Eutrophication influences methanotrophic activity, abundance and community structure in freshwater lakes. <em>Sci. Total Environ.</em><strong>662</strong>, 863–872. 10.1016/j.scitotenv.2019.01.307 (2019).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2019.01.307" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30708301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20Y.%20Y.%20et%20al.%20Eutrophication%20influences%20methanotrophic%20activity,%20abundance%20and%20community%20structure%20in%20freshwater%20lakes.%20Sci.%20Total%20Environ.662,%20863%E2%80%93872.%2010.1016/j.scitotenv.2019.01.307%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Samad, M. S. &amp; Bertilsson, S. Seasonal variations in abundance and diversity of bacterial mathanotrophs in five temperate lakes. <em>Front. Microbiol.</em><strong>8</strong>, 12. 10.3389/fmicb.2017.00142 (2017).
</cite> [<a href="https://doi.org/10.3389/fmicb.2017.00142" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5289968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28217121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Samad,%20M.%20S.%20&amp;%20Bertilsson,%20S.%20Seasonal%20variations%20in%20abundance%20and%20diversity%20of%20bacterial%20mathanotrophs%20in%20five%20temperate%20lakes.%20Front.%20Microbiol.8,%2012.%2010.3389/fmicb.2017.00142%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Henckel, T., Roslev, P. &amp; Conrad, R. Effects of O2 and CH4 on presence and activity of the indigenous methanotrophic community in rice field soil. <em>Environ. Microbiol.</em><strong>2</strong>, 666–679. 10.1046/j.1462-2920.2000.00149.x (2000).
</cite> [<a href="https://doi.org/10.1046/j.1462-2920.2000.00149.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11214799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Henckel,%20T.,%20Roslev,%20P.%20&amp;%20Conrad,%20R.%20Effects%20of%20O2%20and%20CH4%20on%20presence%20and%20activity%20of%20the%20indigenous%20methanotrophic%20community%20in%20rice%20field%20soil.%20Environ.%20Microbiol.2,%20666%E2%80%93679.%2010.1046/j.1462-2920.2000.00149.x%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>van Grinsven, S. et al. Redox zone and trophic state as drivers of methane-oxidizing bacterial abundance and community structure in lake sediments. <em>Front. Environ. Sci.</em><strong>10</strong>, 22. 10.3389/fenvs.2022.857358 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?van%20Grinsven,%20S.%20et%20al.%20Redox%20zone%20and%20trophic%20state%20as%20drivers%20of%20methane-oxidizing%20bacterial%20abundance%20and%20community%20structure%20in%20lake%20sediments.%20Front.%20Environ.%20Sci.10,%2022.%2010.3389/fenvs.2022.857358%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Lyautey, E., Billard, E., Tissot, N., Jacquet, S. &amp; Domaizon, I. Seasonal dynamics of abundance, structure, and diversity of methanogens and methanotrophs in lake sediments. <em>Microb. Ecol.</em><strong>82</strong>, 559–571. 10.1007/s00248-021-01689-9 (2021).
</cite> [<a href="https://doi.org/10.1007/s00248-021-01689-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33538855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lyautey,%20E.,%20Billard,%20E.,%20Tissot,%20N.,%20Jacquet,%20S.%20&amp;%20Domaizon,%20I.%20Seasonal%20dynamics%20of%20abundance,%20structure,%20and%20diversity%20of%20methanogens%20and%20methanotrophs%20in%20lake%20sediments.%20Microb.%20Ecol.82,%20559%E2%80%93571.%2010.1007/s00248-021-01689-9%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Van Grinsven, S., Damsté, J. S. S., Harrison, J., Polerecky, L. &amp; Villanueva, L. Nitrate promotes the transfer of methane-derived carbon from the methanotroph Methylobacter sp. to the methylotroph Methylotenera sp. in eutrophic lake water. <em>Limnol. Oceanogr.</em><strong>66</strong>, 878–891. 10.1002/lno.11648 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Van%20Grinsven,%20S.,%20Damst%C3%A9,%20J.%20S.%20S.,%20Harrison,%20J.,%20Polerecky,%20L.%20&amp;%20Villanueva,%20L.%20Nitrate%20promotes%20the%20transfer%20of%20methane-derived%20carbon%20from%20the%20methanotroph%20Methylobacter%20sp.%20to%20the%20methylotroph%20Methylotenera%20sp.%20in%20eutrophic%20lake%20water.%20Limnol.%20Oceanogr.66,%20878%E2%80%93891.%2010.1002/lno.11648%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Yang, Y. Y., Chen, J. F., Tong, T. L., Xie, S. G. &amp; Liu, Y. Influences of eutrophication on methanogenesis pathways and methanogenic microbial community structures in freshwater lakes. <em>Environ. Pollut.</em><strong>260</strong>, 9. 10.1016/j.envpol.2020.114106 (2020).</cite> [<a href="https://doi.org/10.1016/j.envpol.2020.114106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32041086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20Y.%20Y.,%20Chen,%20J.%20F.,%20Tong,%20T.%20L.,%20Xie,%20S.%20G.%20&amp;%20Liu,%20Y.%20Influences%20of%20eutrophication%20on%20methanogenesis%20pathways%20and%20methanogenic%20microbial%20community%20structures%20in%20freshwater%20lakes.%20Environ.%20Pollut.260,%209.%2010.1016/j.envpol.2020.114106%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>de Mesquita, C. P. B., Wu, D. Y. &amp; Tringe, S. G. Methyl-based methanogenesis: an ecological and genomic review. <em>Microbiol. Mol. Biol. Rev.</em><strong>87</strong>, 29. 10.1128/mmbr.00024-22 (2023).</cite> [<a href="https://doi.org/10.1128/mmbr.00024-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10029344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36692297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Mesquita,%20C.%20P.%20B.,%20Wu,%20D.%20Y.%20&amp;%20Tringe,%20S.%20G.%20Methyl-based%20methanogenesis:%20an%20ecological%20and%20genomic%20review.%20Microbiol.%20Mol.%20Biol.%20Rev.87,%2029.%2010.1128/mmbr.00024-22%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Chu, F. &amp; Lidstrom, M. E. XoxF acts as the predominant methanol dehydrogenase in the type I methanotroph methylomicrobium buryatense. <em>J. Bacteriol.</em><strong>198</strong>, 1317–1325. 10.1128/jb.00959-15 (2016).
</cite> [<a href="https://doi.org/10.1128/JB.00959-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4859581/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26858104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chu,%20F.%20&amp;%20Lidstrom,%20M.%20E.%20XoxF%20acts%20as%20the%20predominant%20methanol%20dehydrogenase%20in%20the%20type%20I%20methanotroph%20methylomicrobium%20buryatense.%20J.%20Bacteriol.198,%201317%E2%80%931325.%2010.1128/jb.00959-15%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Faivre, D. &amp; Schüler, D. Magnetotactic bacteria and magnetosomes. <em>Chem. Rev.</em><strong>108</strong>, 4875–4898. 10.1021/cr078258w (2008).
</cite> [<a href="https://doi.org/10.1021/cr078258w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18855486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Faivre,%20D.%20&amp;%20Sch%C3%BCler,%20D.%20Magnetotactic%20bacteria%20and%20magnetosomes.%20Chem.%20Rev.108,%204875%E2%80%934898.%2010.1021/cr078258w%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Onley, J. R., Ahsan, S., Sanford, R. A. &amp; Löffler, F. E. Denitrification by Anaeromyxobacter dehalogenans, a common soil bacterium lacking the nitrite reductase genes nirS, and nirK. <em>Appl. Environ. Microbiol.</em><strong>84</strong>, 14. 10.1128/aem.01985-17 (2018).</cite> [<a href="https://doi.org/10.1128/AEM.01985-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5795083/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29196287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Onley,%20J.%20R.,%20Ahsan,%20S.,%20Sanford,%20R.%20A.%20&amp;%20L%C3%B6ffler,%20F.%20E.%20Denitrification%20by%20Anaeromyxobacter%20dehalogenans,%20a%20common%20soil%20bacterium%20lacking%20the%20nitrite%20reductase%20genes%20nirS,%20and%20nirK.%20Appl.%20Environ.%20Microbiol.84,%2014.%2010.1128/aem.01985-17%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Heimann, A. C. et al. Hydrogen thresholds and steady-state concentrations associated with microbial arsenate respiration. <em>Environ. Sci. Technol.</em><strong>41</strong>, 2311–2317. 10.1021/es062067d (2007).
</cite> [<a href="https://doi.org/10.1021/es062067d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17438780/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heimann,%20A.%20C.%20et%20al.%20Hydrogen%20thresholds%20and%20steady-state%20concentrations%20associated%20with%20microbial%20arsenate%20respiration.%20Environ.%20Sci.%20Technol.41,%202311%E2%80%932317.%2010.1021/es062067d%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Berg, J. S. et al. Microbial diversity involved in iron and cryptic sulfur cycling in the ferruginous, low-sulfate waters of Lake Pavin. <em>PLoS ONE</em><strong>14</strong>, 21. 10.1371/journal.pone.0212787 (2019).</cite> [<a href="https://doi.org/10.1371/journal.pone.0212787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6386445/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30794698/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berg,%20J.%20S.%20et%20al.%20Microbial%20diversity%20involved%20in%20iron%20and%20cryptic%20sulfur%20cycling%20in%20the%20ferruginous,%20low-sulfate%20waters%20of%20Lake%20Pavin.%20PLoS%20ONE14,%2021.%2010.1371/journal.pone.0212787%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Walker, C. B. et al. Contribution of mobile genetic elements to Desulfovibrio vulgaris genome plasticity. <em>Environ. Microbiol.</em><strong>11</strong>, 2244–2252. 10.1111/j.1462-2920.2009.01946.x (2009).
</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2009.01946.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19737303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Walker,%20C.%20B.%20et%20al.%20Contribution%20of%20mobile%20genetic%20elements%20to%20Desulfovibrio%20vulgaris%20genome%20plasticity.%20Environ.%20Microbiol.11,%202244%E2%80%932252.%2010.1111/j.1462-2920.2009.01946.x%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Cypionka, H. Oxygen respiration by Desulfovibrio species. <em>Annu. Rev. Microbiol.</em><strong>54</strong>, 827–848. 10.1146/annurev.micro.54.1.827 (2000).
</cite> [<a href="https://doi.org/10.1146/annurev.micro.54.1.827" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11018146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cypionka,%20H.%20Oxygen%20respiration%20by%20Desulfovibrio%20species.%20Annu.%20Rev.%20Microbiol.54,%20827%E2%80%93848.%2010.1146/annurev.micro.54.1.827%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Chistoserdova, L. Modularity of methylotrophy, revisited. <em>Environ. Microbiol.</em><strong>13</strong>, 2603–2622. 10.1111/j.1462-2920.2011.02464.x (2011).
</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2011.02464.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21443740/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chistoserdova,%20L.%20Modularity%20of%20methylotrophy,%20revisited.%20Environ.%20Microbiol.13,%202603%E2%80%932622.%2010.1111/j.1462-2920.2011.02464.x%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Krause, S. M. B. et al. Lanthanide-dependent cross-feeding of methane-derived carbon is linked by microbial community interactions. <em>Proc. Natl. Acad. Sci. U. S. A.</em><strong>114</strong>, 358–363. 10.1073/pnas.1619871114 (2017).
</cite> [<a href="https://doi.org/10.1073/pnas.1619871114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5240692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28028242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krause,%20S.%20M.%20B.%20et%20al.%20Lanthanide-dependent%20cross-feeding%20of%20methane-derived%20carbon%20is%20linked%20by%20microbial%20community%20interactions.%20Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.114,%20358%E2%80%93363.%2010.1073/pnas.1619871114%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Ghai, R., Mizuno, C. M., Picazo, A., Camacho, A. &amp; Rodriguez-Valera, F. Metagenomics uncovers a new group of low GC and ultra-small marine Actinobacteria. <em>Sci. Rep.</em><strong>3</strong>, 8. 10.1038/srep02471 (2013).</cite> [<a href="https://doi.org/10.1038/srep02471" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3747508/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23959135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ghai,%20R.,%20Mizuno,%20C.%20M.,%20Picazo,%20A.,%20Camacho,%20A.%20&amp;%20Rodriguez-Valera,%20F.%20Metagenomics%20uncovers%20a%20new%20group%20of%20low%20GC%20and%20ultra-small%20marine%20Actinobacteria.%20Sci.%20Rep.3,%208.%2010.1038/srep02471%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Prosser, J. I. &amp; Nicol, G. W. Archaeal and bacterial ammonia-oxidisers in soil: the quest for niche specialisation and differentiation. <em>Trends Microbiol.</em><strong>20</strong>, 523–531. 10.1016/j.tim.2012.08.001 (2012).
</cite> [<a href="https://doi.org/10.1016/j.tim.2012.08.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22959489/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prosser,%20J.%20I.%20&amp;%20Nicol,%20G.%20W.%20Archaeal%20and%20bacterial%20ammonia-oxidisers%20in%20soil:%20the%20quest%20for%20niche%20specialisation%20and%20differentiation.%20Trends%20Microbiol.20,%20523%E2%80%93531.%2010.1016/j.tim.2012.08.001%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Hou, J., Song, C. L., Cao, X. Y. &amp; Zhou, Y. Y. Shifts between ammonia-oxidizing bacteria and archaea in relation to nitrification potential across trophic gradients in two large Chinese lakes (Lake Taihu and Lake Chaohu). <em>Water Res.</em><strong>47</strong>, 2285–2296. 10.1016/j.watres.2013.01.042 (2013).
</cite> [<a href="https://doi.org/10.1016/j.watres.2013.01.042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23473400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hou,%20J.,%20Song,%20C.%20L.,%20Cao,%20X.%20Y.%20&amp;%20Zhou,%20Y.%20Y.%20Shifts%20between%20ammonia-oxidizing%20bacteria%20and%20archaea%20in%20relation%20to%20nitrification%20potential%20across%20trophic%20gradients%20in%20two%20large%20Chinese%20lakes%20(Lake%20Taihu%20and%20Lake%20Chaohu).%20Water%20Res.47,%202285%E2%80%932296.%2010.1016/j.watres.2013.01.042%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Camargo, J. A. &amp; Alonso, A. Ecological and toxicological effects of inorganic nitrogen pollution in aquatic ecosystems: A global assessment. <em>Environ. Int.</em><strong>32</strong>, 831–849. 10.1016/j.envint.2006.05.002 (2006).
</cite> [<a href="https://doi.org/10.1016/j.envint.2006.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16781774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Camargo,%20J.%20A.%20&amp;%20Alonso,%20A.%20Ecological%20and%20toxicological%20effects%20of%20inorganic%20nitrogen%20pollution%20in%20aquatic%20ecosystems:%20A%20global%20assessment.%20Environ.%20Int.32,%20831%E2%80%93849.%2010.1016/j.envint.2006.05.002%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Li, Y. et al. The responses of bacterial community and N2O emission to nitrogen input in lake sediment: Estrogen as a co-pollutant. <em>Environ. Res.</em><strong>179</strong>, 8. 10.1016/j.envres.2019.108769 (2019).</cite> [<a href="https://doi.org/10.1016/j.envres.2019.108769" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31574450/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Y.%20et%20al.%20The%20responses%20of%20bacterial%20community%20and%20N2O%20emission%20to%20nitrogen%20input%20in%20lake%20sediment:%20Estrogen%20as%20a%20co-pollutant.%20Environ.%20Res.179,%208.%2010.1016/j.envres.2019.108769%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Xu, Y. F. et al. The diversity of comammox bacteria and the effect of sewage discharge on their abundance in eutrophic lake sediments. <em>J. Soils Sediments</em><strong>20</strong>, 2495–2503. 10.1007/s11368-020-02618-y (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Xu,%20Y.%20F.%20et%20al.%20The%20diversity%20of%20comammox%20bacteria%20and%20the%20effect%20of%20sewage%20discharge%20on%20their%20abundance%20in%20eutrophic%20lake%20sediments.%20J.%20Soils%20Sediments20,%202495%E2%80%932503.%2010.1007/s11368-020-02618-y%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Zheng, S. H. et al. The nitrogen removal performance and microbial community on mixotrophic denitrification process. <em>Bioresour. Technol.</em><strong>363</strong>, 9. 10.1016/j.biortech.2022.127901 (2022).</cite> [<a href="https://doi.org/10.1016/j.biortech.2022.127901" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36075349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20S.%20H.%20et%20al.%20The%20nitrogen%20removal%20performance%20and%20microbial%20community%20on%20mixotrophic%20denitrification%20process.%20Bioresour.%20Technol.363,%209.%2010.1016/j.biortech.2022.127901%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Zhang, W. H. et al. Nitrogen rather than phosphorus driving the biogeographic patterns of abundant bacterial taxa in a eutrophic plateau lake. <em>Sci. Total Environ.</em><strong>806</strong>, 10. 10.1016/j.scitotenv.2021.150947 (2022).</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2021.150947" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34655620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20W.%20H.%20et%20al.%20Nitrogen%20rather%20than%20phosphorus%20driving%20the%20biogeographic%20patterns%20of%20abundant%20bacterial%20taxa%20in%20a%20eutrophic%20plateau%20lake.%20Sci.%20Total%20Environ.806,%2010.%2010.1016/j.scitotenv.2021.150947%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Cummings, D. E., Caccavo, F., Spring, S. &amp; Rosenzweig, R. F. Ferribacterium limneticum, gen. nov., sp. nov., an Fe(III)-reducing microorganism isolated from mining-impacted freshwater lake sediments. <em>Arch. Microbiol.</em><strong>171</strong>, 183–188. 10.1007/s002030050697 (1999).</cite> [<a href="https://scholar.google.com/scholar_lookup?Cummings,%20D.%20E.,%20Caccavo,%20F.,%20Spring,%20S.%20&amp;%20Rosenzweig,%20R.%20F.%20Ferribacterium%20limneticum,%20gen.%20nov.,%20sp.%20nov.,%20an%20Fe(III)-reducing%20microorganism%20isolated%20from%20mining-impacted%20freshwater%20lake%20sediments.%20Arch.%20Microbiol.171,%20183%E2%80%93188.%2010.1007/s002030050697%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Zeng, F. F. et al. Metagenomic analysis of the soil microbial composition and salt tolerance mechanism in Yuncheng Salt Lake, Shanxi Province. <em>Front. Microbiol.</em><strong>13</strong>, 14. 10.3389/fmicb.2022.1004556 (2022).</cite> [<a href="https://doi.org/10.3389/fmicb.2022.1004556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9549588/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36225369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng,%20F.%20F.%20et%20al.%20Metagenomic%20analysis%20of%20the%20soil%20microbial%20composition%20and%20salt%20tolerance%20mechanism%20in%20Yuncheng%20Salt%20Lake,%20Shanxi%20Province.%20Front.%20Microbiol.13,%2014.%2010.3389/fmicb.2022.1004556%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Liu, J. et al. Correlating microbial diversity patterns with geochemistry in an extreme and heterogeneous environment of mine tailings. <em>Appl. Environ. Microbiol.</em><strong>80</strong>, 3677–3686. 10.1128/aem.00294-14 (2014).
</cite> [<a href="https://doi.org/10.1128/AEM.00294-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4054149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24727268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20J.%20et%20al.%20Correlating%20microbial%20diversity%20patterns%20with%20geochemistry%20in%20an%20extreme%20and%20heterogeneous%20environment%20of%20mine%20tailings.%20Appl.%20Environ.%20Microbiol.80,%203677%E2%80%933686.%2010.1128/aem.00294-14%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Collado, L., Inza, I., Guarro, J. &amp; Figueras, M. J. Presence of Arcobacter spp. in environmental waters correlates with high levels of fecal pollution. <em>Environ. Microbiol.</em><strong>10</strong>, 1635–1640. 10.1111/j.1462-2920.2007.01555.x (2008).
</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2007.01555.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18215159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Collado,%20L.,%20Inza,%20I.,%20Guarro,%20J.%20&amp;%20Figueras,%20M.%20J.%20Presence%20of%20Arcobacter%20spp.%20in%20environmental%20waters%20correlates%20with%20high%20levels%20of%20fecal%20pollution.%20Environ.%20Microbiol.10,%201635%E2%80%931640.%2010.1111/j.1462-2920.2007.01555.x%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Collado, L. et al. Occurrence and diversity of Arcobacter spp. along the Llobregat River catchment, at sewage effluents and in a drinking water treatment plant. <em>Water Res.</em><strong>44</strong>, 3696–3702. 10.1016/j.watres.2010.04.002 (2010).
</cite> [<a href="https://doi.org/10.1016/j.watres.2010.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20427071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Collado,%20L.%20et%20al.%20Occurrence%20and%20diversity%20of%20Arcobacter%20spp.%20along%20the%20Llobregat%20River%20catchment,%20at%20sewage%20effluents%20and%20in%20a%20drinking%20water%20treatment%20plant.%20Water%20Res.44,%203696%E2%80%933702.%2010.1016/j.watres.2010.04.002%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Webb, A. L., Taboada, E. N., Selinger, L. B., Boras, V. F. &amp; Inglis, G. D. Prevalence and diversity of waterborne <em>Arcobacter butzleri</em> in southwestern Alberta, Canada. <em>Can. J. Microbiol.</em><strong>63</strong>, 330–340. 10.1139/cjm-2016-0745 (2017).
</cite> [<a href="https://doi.org/10.1139/cjm-2016-0745" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28177789/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Webb,%20A.%20L.,%20Taboada,%20E.%20N.,%20Selinger,%20L.%20B.,%20Boras,%20V.%20F.%20&amp;%20Inglis,%20G.%20D.%20Prevalence%20and%20diversity%20of%20waterborne%20Arcobacter%20butzleri%20in%20southwestern%20Alberta,%20Canada.%20Can.%20J.%20Microbiol.63,%20330%E2%80%93340.%2010.1139/cjm-2016-0745%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Gong, Y., Tang, X., Shao, K., Hu, Y. &amp; Gao, G. Dynamics of bacterial abundance and the related environmental factors in large shallow eutrophic Lake Taihu. <em>J. Freshw. Ecol.</em><strong>32</strong>, 133–145. 10.1080/02705060.2016.1248506 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gong,%20Y.,%20Tang,%20X.,%20Shao,%20K.,%20Hu,%20Y.%20&amp;%20Gao,%20G.%20Dynamics%20of%20bacterial%20abundance%20and%20the%20related%20environmental%20factors%20in%20large%20shallow%20eutrophic%20Lake%20Taihu.%20J.%20Freshw.%20Ecol.32,%20133%E2%80%93145.%2010.1080/02705060.2016.1248506%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Rawal, I., Joshi, H. &amp; Chaudhary, B. L. Isolation, identification, and antibiotics resistance of Aeromonas spp. from lakes of Udaipur (Rajasthan), India. <em>Asian J. Pharm. (AJP)</em>10.2237/ajp.v10i2.612 (2016).</cite> [<a href="https://scholar.google.com/scholar_lookup?Rawal,%20I.,%20Joshi,%20H.%20&amp;%20Chaudhary,%20B.%20L.%20Isolation,%20identification,%20and%20antibiotics%20resistance%20of%20Aeromonas%20spp.%20from%20lakes%20of%20Udaipur%20(Rajasthan),%20India.%20Asian%20J.%20Pharm.%20(AJP)10.2237/ajp.v10i2.612%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Skwor, T., Shinko, J., Augustyniak, A., Gee, C. &amp; Andraso, G. <em>Aeromonas</em><em>hydrophila</em> and <em>Aeromonas</em><em>veronii</em> Predominate among potentially pathogenic ciprofloxacin- and tetracycline- resistant Aeromonas isolates from Lake Erie. <em>Appl. Environ. Microbiol.</em><strong>80</strong>, 841–848. 10.1128/aem.03645-13 (2014).
</cite> [<a href="https://doi.org/10.1128/AEM.03645-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3911203/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24242249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Skwor,%20T.,%20Shinko,%20J.,%20Augustyniak,%20A.,%20Gee,%20C.%20&amp;%20Andraso,%20G.%20Aeromonashydrophila%20and%20Aeromonasveronii%20Predominate%20among%20potentially%20pathogenic%20ciprofloxacin-%20and%20tetracycline-%20resistant%20Aeromonas%20isolates%20from%20Lake%20Erie.%20Appl.%20Environ.%20Microbiol.80,%20841%E2%80%93848.%2010.1128/aem.03645-13%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Haller, L. et al. Composition of bacterial and archaeal communities in freshwater sediments with different contamination levels (Lake Geneva, Switzerland). <em>Water Res.</em><strong>45</strong>, 1213–1228. 10.1016/j.watres.2010.11.018 (2011).
</cite> [<a href="https://doi.org/10.1016/j.watres.2010.11.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21145090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haller,%20L.%20et%20al.%20Composition%20of%20bacterial%20and%20archaeal%20communities%20in%20freshwater%20sediments%20with%20different%20contamination%20levels%20(Lake%20Geneva,%20Switzerland).%20Water%20Res.45,%201213%E2%80%931228.%2010.1016/j.watres.2010.11.018%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Conty, A., García-Criado, F. &amp; Bécares, E. Changes in bacterial and ciliate densities with trophic status in Mediterranean shallow lakes. <em>Hydrobiologia</em><strong>584</strong>, 327–335. 10.1007/s10750-007-0585-x (2007).</cite> [<a href="https://scholar.google.com/scholar_lookup?Conty,%20A.,%20Garc%C3%ADa-Criado,%20F.%20&amp;%20B%C3%A9cares,%20E.%20Changes%20in%20bacterial%20and%20ciliate%20densities%20with%20trophic%20status%20in%20Mediterranean%20shallow%20lakes.%20Hydrobiologia584,%20327%E2%80%93335.%2010.1007/s10750-007-0585-x%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>Huang, T. L. et al. Nitrogen removal characteristics of a newly isolated indigenous aerobic Denitrifier from oligotrophic drinking water reservoir, Zoogloea sp N299. <em>Int. J. Mol. Sci.</em><strong>16</strong>, 10038–10060. 10.3390/ijms160510038 (2015).
</cite> [<a href="https://doi.org/10.3390/ijms160510038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4463631/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25946341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20T.%20L.%20et%20al.%20Nitrogen%20removal%20characteristics%20of%20a%20newly%20isolated%20indigenous%20aerobic%20Denitrifier%20from%20oligotrophic%20drinking%20water%20reservoir,%20Zoogloea%20sp%20N299.%20Int.%20J.%20Mol.%20Sci.16,%2010038%E2%80%9310060.%2010.3390/ijms160510038%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Lu, F. H., Lukasik, J. &amp; Farrah, S. R. Immunological methods for the study of Zoogloea strains in natural environments. <em>Water Res.</em><strong>35</strong>, 4011–4018. 10.1016/s0043-1354(01)00141-5 (2001).
</cite> [<a href="https://doi.org/10.1016/s0043-1354(01)00141-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11791830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20F.%20H.,%20Lukasik,%20J.%20&amp;%20Farrah,%20S.%20R.%20Immunological%20methods%20for%20the%20study%20of%20Zoogloea%20strains%20in%20natural%20environments.%20Water%20Res.35,%204011%E2%80%934018.%2010.1016/s0043-1354(01)00141-5%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite>Ekstrom, E. B. &amp; Morel, F. M. M. Cobalt limitation of growth and mercury methylation in sulfate-reducing bacteria. <em>Environ. Sci. Technol.</em><strong>42</strong>, 93–99. 10.1021/es0705644 (2008).
</cite> [<a href="https://doi.org/10.1021/es0705644" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18350881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ekstrom,%20E.%20B.%20&amp;%20Morel,%20F.%20M.%20M.%20Cobalt%20limitation%20of%20growth%20and%20mercury%20methylation%20in%20sulfate-reducing%20bacteria.%20Environ.%20Sci.%20Technol.42,%2093%E2%80%9399.%2010.1021/es0705644%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR109">
<span class="label">109.</span><cite>Zanaroli, G. et al. Characterization of the microbial community from the marine sediment of the Venice lagoon capable of reductive dechlorination of coplanar polychlorinated biphenyls (PCBs). <em>J. Hazard. Mater.</em><strong>178</strong>, 417–426. 10.1016/j.jhazmat.2010.01.097 (2010).
</cite> [<a href="https://doi.org/10.1016/j.jhazmat.2010.01.097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20153926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zanaroli,%20G.%20et%20al.%20Characterization%20of%20the%20microbial%20community%20from%20the%20marine%20sediment%20of%20the%20Venice%20lagoon%20capable%20of%20reductive%20dechlorination%20of%20coplanar%20polychlorinated%20biphenyls%20(PCBs).%20J.%20Hazard.%20Mater.178,%20417%E2%80%93426.%2010.1016/j.jhazmat.2010.01.097%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR110">
<span class="label">110.</span><cite>Wan, X. L. et al. Biogeographic patterns of potential pathogenic bacteria in the middle and lower reaches of the Yangtze River as well as its two adjoining lakes, China. <em>Front. Microbiol.</em><strong>13</strong>, 12. 10.3389/fmicb.2022.972243 (2022).</cite> [<a href="https://doi.org/10.3389/fmicb.2022.972243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9479215/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36118197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wan,%20X.%20L.%20et%20al.%20Biogeographic%20patterns%20of%20potential%20pathogenic%20bacteria%20in%20the%20middle%20and%20lower%20reaches%20of%20the%20Yangtze%20River%20as%20well%20as%20its%20two%20adjoining%20lakes,%20China.%20Front.%20Microbiol.13,%2012.%2010.3389/fmicb.2022.972243%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR111">
<span class="label">111.</span><cite>Jiang, C. Y., Dai, X., Wang, B. J., Zhou, Y. G. &amp; Liu, S. J. Roseomonas lacus sp nov., isolated from freshwater lake sediment. <em>Int. J. Syst. Evol. Microbiol.</em><strong>56</strong>, 25–28. 10.1099/ijs.0.63938-0 (2006).
</cite> [<a href="https://doi.org/10.1099/ijs.0.63938-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16403861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang,%20C.%20Y.,%20Dai,%20X.,%20Wang,%20B.%20J.,%20Zhou,%20Y.%20G.%20&amp;%20Liu,%20S.%20J.%20Roseomonas%20lacus%20sp%20nov.,%20isolated%20from%20freshwater%20lake%20sediment.%20Int.%20J.%20Syst.%20Evol.%20Microbiol.56,%2025%E2%80%9328.%2010.1099/ijs.0.63938-0%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR112">
<span class="label">112.</span><cite>Gam, Z. B. et al. Characterization of Desulfovibrio salinus sp nov., a slightly halophilic sulfate-reducing bacterium isolated from a saline lake in Tunisia. <em>Int. J. Syst. Evol. Microbiol.</em><strong>68</strong>, 715–720. 10.1099/ijsem.0.002567 (2018).
</cite> [<a href="https://doi.org/10.1099/ijsem.0.002567" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29458461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gam,%20Z.%20B.%20et%20al.%20Characterization%20of%20Desulfovibrio%20salinus%20sp%20nov.,%20a%20slightly%20halophilic%20sulfate-reducing%20bacterium%20isolated%20from%20a%20saline%20lake%20in%20Tunisia.%20Int.%20J.%20Syst.%20Evol.%20Microbiol.68,%20715%E2%80%93720.%2010.1099/ijsem.0.002567%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR113">
<span class="label">113.</span><cite>Zheng, S. L., Li, M., Liu, Y. &amp; Liu, F. H. Desulfovibrio feeding Methanobacterium with electrons in conductive methanogenic aggregates from coastal zones. <em>Water Res.</em><strong>202</strong>, 11. 10.1016/j.watres.2021.117490 (2021).</cite> [<a href="https://doi.org/10.1016/j.watres.2021.117490" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34364064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20S.%20L.,%20Li,%20M.,%20Liu,%20Y.%20&amp;%20Liu,%20F.%20H.%20Desulfovibrio%20feeding%20Methanobacterium%20with%20electrons%20in%20conductive%20methanogenic%20aggregates%20from%20coastal%20zones.%20Water%20Res.202,%2011.%2010.1016/j.watres.2021.117490%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR114">
<span class="label">114.</span><cite>Schwarz, J. I. K., Eckert, W. &amp; Conrad, R. Community structure of Archaea and Bacteria in a profundal lake sediment Lake Kinneret (Israel). <em>Syst. Appl. Microbiol.</em><strong>30</strong>, 239–254. 10.1016/j.syapm.2006.05.004 (2007).
</cite> [<a href="https://doi.org/10.1016/j.syapm.2006.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16857336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schwarz,%20J.%20I.%20K.,%20Eckert,%20W.%20&amp;%20Conrad,%20R.%20Community%20structure%20of%20Archaea%20and%20Bacteria%20in%20a%20profundal%20lake%20sediment%20Lake%20Kinneret%20(Israel).%20Syst.%20Appl.%20Microbiol.30,%20239%E2%80%93254.%2010.1016/j.syapm.2006.05.004%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR115">
<span class="label">115.</span><cite>Xie, Z. et al. Bacterial and archaeal communities in Erhai lake sediments: abundance and metabolic insight into a plateau lake at the edge of eutrophication. <em>Microorganisms</em><strong>12</strong>, 19. 10.3390/microorganisms12081617 (2024).</cite> [<a href="https://doi.org/10.3390/microorganisms12081617" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11356345/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39203459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie,%20Z.%20et%20al.%20Bacterial%20and%20archaeal%20communities%20in%20Erhai%20lake%20sediments:%20abundance%20and%20metabolic%20insight%20into%20a%20plateau%20lake%20at%20the%20edge%20of%20eutrophication.%20Microorganisms12,%2019.%2010.3390/microorganisms12081617%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR116">
<span class="label">116.</span><cite>Vannier, P. et al. Metagenomic analyses of a microbial assemblage in a subglacial lake beneath the Vatnajokull ice cap, Iceland. <em>Front. Microbiol.</em><strong>14</strong>, 12. 10.3389/fmicb.2023.1122184 (2023).</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1122184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10098204/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37065146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vannier,%20P.%20et%20al.%20Metagenomic%20analyses%20of%20a%20microbial%20assemblage%20in%20a%20subglacial%20lake%20beneath%20the%20Vatnajokull%20ice%20cap,%20Iceland.%20Front.%20Microbiol.14,%2012.%2010.3389/fmicb.2023.1122184%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR117">
<span class="label">117.</span><cite>Fadhlaoui, K. et al. Fusibacter fontis sp nov., a sulfur-reducing, anaerobic bacterium isolated from a mesothermic Tunisian spring. <em>Int. J. Syst. Evol. Microbiol.</em><strong>65</strong>, 3501–3506. 10.1099/ijsem.0.000445 (2015).
</cite> [<a href="https://doi.org/10.1099/ijsem.0.000445" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26296995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fadhlaoui,%20K.%20et%20al.%20Fusibacter%20fontis%20sp%20nov.,%20a%20sulfur-reducing,%20anaerobic%20bacterium%20isolated%20from%20a%20mesothermic%20Tunisian%20spring.%20Int.%20J.%20Syst.%20Evol.%20Microbiol.65,%203501%E2%80%933506.%2010.1099/ijsem.0.000445%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR118">
<span class="label">118.</span><cite>Brioukhanov, A. L., Kadnikov, V. V., Beletsky, A. V. &amp; Savvichev, A. S. Aerotolerant thiosulfate-reducing bacterium fusibacter sp. strain WBS isolated from littoral bottom sediments of the white sea-biochemical and genome analysis. <em>Microorganisms</em><strong>11</strong>, 11. 10.3390/microorganisms11071642 (2023).</cite> [<a href="https://doi.org/10.3390/microorganisms11071642" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10386464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37512815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brioukhanov,%20A.%20L.,%20Kadnikov,%20V.%20V.,%20Beletsky,%20A.%20V.%20&amp;%20Savvichev,%20A.%20S.%20Aerotolerant%20thiosulfate-reducing%20bacterium%20fusibacter%20sp.%20strain%20WBS%20isolated%20from%20littoral%20bottom%20sediments%20of%20the%20white%20sea-biochemical%20and%20genome%20analysis.%20Microorganisms11,%2011.%2010.3390/microorganisms11071642%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR119">
<span class="label">119.</span><cite>Visser, J. M., Robertson, L. A., VanVerseveld, H. W. &amp; Kuenen, J. G. Sulfur production by obligately chemolithoautotrophic Thiobacillus species. <em>Appl. Environ. Microbiol.</em><strong>63</strong>, 2300–2305. 10.1128/aem.63.6.2300-2305.1997 (1997).
</cite> [<a href="https://doi.org/10.1128/aem.63.6.2300-2305.1997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1389182/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16535627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Visser,%20J.%20M.,%20Robertson,%20L.%20A.,%20VanVerseveld,%20H.%20W.%20&amp;%20Kuenen,%20J.%20G.%20Sulfur%20production%20by%20obligately%20chemolithoautotrophic%20Thiobacillus%20species.%20Appl.%20Environ.%20Microbiol.63,%202300%E2%80%932305.%2010.1128/aem.63.6.2300-2305.1997%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR120">
<span class="label">120.</span><cite>Kuenen, J. &amp; Beudeker, R. Microbiology of thiobacilli and other sulphur-oxidizing autotrophs, mixotrophs and heterotrophs. <em>Philos. Trans. Royal Soc. London B Biol. Sci.</em><strong>298</strong>, 473–497. 10.1098/rstb.1982.0093 (1982).</cite> [<a href="https://doi.org/10.1098/rstb.1982.0093" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6127737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuenen,%20J.%20&amp;%20Beudeker,%20R.%20Microbiology%20of%20thiobacilli%20and%20other%20sulphur-oxidizing%20autotrophs,%20mixotrophs%20and%20heterotrophs.%20Philos.%20Trans.%20Royal%20Soc.%20London%20B%20Biol.%20Sci.298,%20473%E2%80%93497.%2010.1098/rstb.1982.0093%20(1982)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR121">
<span class="label">121.</span><cite>Huang, C. et al. Bioaugmentation with Thiobacillus sp. H1 in an autotrophic denitrification desulfurization microbial reactor: Microbial community changes and relationship. <em>Environ. Res.</em><strong>189</strong>, 8. 10.1016/j.envres.2020.109927 (2020).</cite> [<a href="https://doi.org/10.1016/j.envres.2020.109927" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32678744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20C.%20et%20al.%20Bioaugmentation%20with%20Thiobacillus%20sp.%20H1%20in%20an%20autotrophic%20denitrification%20desulfurization%20microbial%20reactor:%20Microbial%20community%20changes%20and%20relationship.%20Environ.%20Res.189,%208.%2010.1016/j.envres.2020.109927%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR122">
<span class="label">122.</span><cite>Bazylinski, D. A. et al. Magnetovibrio blakemorei gen. nov., sp nov., a magnetotactic bacterium Alphaproteobacteria: Rhodospirillaceae isolated from a salt marsh. <em>Int. J. Syst. Evol. Microbiol.</em><strong>63</strong>, 1824–1833. 10.1099/ijs.0.044453-0 (2013).
</cite> [<a href="https://doi.org/10.1099/ijs.0.044453-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22984137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bazylinski,%20D.%20A.%20et%20al.%20Magnetovibrio%20blakemorei%20gen.%20nov.,%20sp%20nov.,%20a%20magnetotactic%20bacterium%20Alphaproteobacteria:%20Rhodospirillaceae%20isolated%20from%20a%20salt%20marsh.%20Int.%20J.%20Syst.%20Evol.%20Microbiol.63,%201824%E2%80%931833.%2010.1099/ijs.0.044453-0%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR123">
<span class="label">123.</span><cite>Kojima, H., Shinohara, A. &amp; Fukui, M. Sulfurifustis variabilis gen. nov., sp nov., a sulfur oxidizer isolated from a lake, and proposal of Acidiferrobacteraceae fam. nov and Acidiferrobacterales ord. nov. Int. <em>J. Syst. Evol. Microbiol.</em><strong>65</strong>, 3709–3713. 10.1099/ijsem.0.000479 (2015).</cite> [<a href="https://doi.org/10.1099/ijsem.0.000479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26220671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kojima,%20H.,%20Shinohara,%20A.%20&amp;%20Fukui,%20M.%20Sulfurifustis%20variabilis%20gen.%20nov.,%20sp%20nov.,%20a%20sulfur%20oxidizer%20isolated%20from%20a%20lake,%20and%20proposal%20of%20Acidiferrobacteraceae%20fam.%20nov%20and%20Acidiferrobacterales%20ord.%20nov.%20Int.%20J.%20Syst.%20Evol.%20Microbiol.65,%203709%E2%80%933713.%2010.1099/ijsem.0.000479%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR124">
<span class="label">124.</span><cite>Umezawa, K., Watanabe, T., Miura, A., Kojima, H. &amp; Fukui, M. The complete genome sequences of sulfur-oxidizing Gammaproteobacteria Sulfurifustis variabilis skN76T and Sulfuricaulis limicola HA5T. <em>Stand. Genomic Sci.</em><strong>11</strong>, 8. 10.1186/s40793-016-0196-0 (2016).
</cite> [<a href="https://doi.org/10.1186/s40793-016-0196-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5024460/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27651857/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Umezawa,%20K.,%20Watanabe,%20T.,%20Miura,%20A.,%20Kojima,%20H.%20&amp;%20Fukui,%20M.%20The%20complete%20genome%20sequences%20of%20sulfur-oxidizing%20Gammaproteobacteria%20Sulfurifustis%20variabilis%20skN76T%20and%20Sulfuricaulis%20limicola%20HA5T.%20Stand.%20Genomic%20Sci.11,%208.%2010.1186/s40793-016-0196-0%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375747/bin/41598_2025_16527_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information.</a><sup> (501.8KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All raw sequences were deposited at the NCBI Sequence Read Archive as part of the Bioproject PRJNA1270394 (<a href="https://www.ncbi.nlm.nih.gov/sra/PRJNA1270394" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/sra/PRJNA1270394</a>) with all the accessions SRR33778681- SRR33778692. A detailed overview can be found in Supplementary Table <a href="#MOESM1" class="usa-link">S4</a>.” Also, the Supplementary Table <a href="#MOESM1" class="usa-link">S4</a> has been added to the Supplementary files.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-16527-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_16527.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375747/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375747/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375747%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375747/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375747/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375747/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40851025/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375747/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40851025/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375747/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375747/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="z2FXrejjafxjlwD9fEIY5SD3dz2SfsfImWNPmFZ9lX0BpyQoIGViGVVpeslG6HzZ">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
