
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Innovative approaches to combat antibiotic resistance: integrating CRISPR/Cas9 and nanoparticles against biofilm-driven infections - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="1525426D8AF2D02305426D004FC54097.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcmedicine">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366227/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Medicine">
<meta name="citation_title" content="Innovative approaches to combat antibiotic resistance: integrating CRISPR/Cas9 and nanoparticles against biofilm-driven infections">
<meta name="citation_author" content="Abolfazl Saffari Natanzi">
<meta name="citation_author_institution" content="Anatomical Sciences Research Center, Institute for Basic Sciences, Kashan University of Medical Sciences, Kashan, Iran">
<meta name="citation_author" content="Mohsen Poudineh">
<meta name="citation_author_institution" content="Department of Bacteriology and Virology, School of Medicine, Isfahan University of Medical Sciences, Isfahan, Iran">
<meta name="citation_author" content="Elham Karimi">
<meta name="citation_author_institution" content="Department of Nanobiotechnology, Faculty of Biological Science, Tarbiat Modares University, Tehran, Iran">
<meta name="citation_author" content="Azad Khaledi">
<meta name="citation_author_institution" content="Infectious Diseases Research Center, Kashan University of Medical Sciences, Kashan, Iran">
<meta name="citation_author" content="Hamed Haddad Kashani">
<meta name="citation_author_institution" content="Anatomical Sciences Research Center, Institute for Basic Sciences, Kashan University of Medical Sciences, Kashan, Iran">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="23">
<meta name="citation_firstpage" content="486">
<meta name="citation_doi" content="10.1186/s12916-025-04323-4">
<meta name="citation_pmid" content="40830872">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366227/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366227/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366227/pdf/12916_2025_Article_4323.pdf">
<meta name="description" content="The increasing prevalence of antibiotic-resistant bacterial infections is a major global health concern, with biofilms playing a key role in bacterial persistence and resistance. Biofilms provide a protective matrix that limits antibiotic ...">
<meta name="og:title" content="Innovative approaches to combat antibiotic resistance: integrating CRISPR/Cas9 and nanoparticles against biofilm-driven infections">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The increasing prevalence of antibiotic-resistant bacterial infections is a major global health concern, with biofilms playing a key role in bacterial persistence and resistance. Biofilms provide a protective matrix that limits antibiotic ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366227/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366227">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12916-025-04323-4"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12916_2025_Article_4323.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366227%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366227/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366227/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366227/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcmedicine.png" alt="BMC Medicine logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Medicine" title="Link to BMC Medicine" shape="default" href="https://bmcmedicine.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Med</button></div>. 2025 Aug 20;23:486. doi: <a href="https://doi.org/10.1186/s12916-025-04323-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12916-025-04323-4</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Med%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Med%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Med%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Med%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Innovative approaches to combat antibiotic resistance: integrating CRISPR/Cas9 and nanoparticles against biofilm-driven infections</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saffari%20Natanzi%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Abolfazl Saffari Natanzi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Abolfazl Saffari Natanzi</span></h3>
<div class="p">
<sup>1</sup>Anatomical Sciences Research Center, Institute for Basic Sciences, Kashan University of Medical Sciences, Kashan, Iran </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saffari%20Natanzi%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Abolfazl Saffari Natanzi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Poudineh%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Mohsen Poudineh</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Mohsen Poudineh</span></h3>
<div class="p">
<sup>2</sup>Department of Bacteriology and Virology, School of Medicine, Isfahan University of Medical Sciences, Isfahan, Iran </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Poudineh%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mohsen Poudineh</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Karimi%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Elham Karimi</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Elham Karimi</span></h3>
<div class="p">
<sup>3</sup>Department of Nanobiotechnology, Faculty of Biological Science, Tarbiat Modares University, Tehran, Iran </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Karimi%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Elham Karimi</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khaledi%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Azad Khaledi</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Azad Khaledi</span></h3>
<div class="p">
<sup>4</sup>Infectious Diseases Research Center, Kashan University of Medical Sciences, Kashan, Iran </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khaledi%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Azad Khaledi</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Haddad%20Kashani%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Hamed Haddad Kashani</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Hamed Haddad Kashani</span></h3>
<div class="p">
<sup>1</sup>Anatomical Sciences Research Center, Institute for Basic Sciences, Kashan University of Medical Sciences, Kashan, Iran </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Haddad%20Kashani%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hamed Haddad Kashani</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Anatomical Sciences Research Center, Institute for Basic Sciences, Kashan University of Medical Sciences, Kashan, Iran </div>
<div id="Aff2">
<sup>2</sup>Department of Bacteriology and Virology, School of Medicine, Isfahan University of Medical Sciences, Isfahan, Iran </div>
<div id="Aff3">
<sup>3</sup>Department of Nanobiotechnology, Faculty of Biological Science, Tarbiat Modares University, Tehran, Iran </div>
<div id="Aff4">
<sup>4</sup>Infectious Diseases Research Center, Kashan University of Medical Sciences, Kashan, Iran </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 14; Accepted 2025 Aug 5; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366227  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830872/" class="usa-link">40830872</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The increasing prevalence of antibiotic-resistant bacterial infections is a major global health concern, with biofilms playing a key role in bacterial persistence and resistance. Biofilms provide a protective matrix that limits antibiotic penetration, enhances horizontal gene transfer, and enables bacterial survival in hostile environments. Conventional antimicrobial therapies are often ineffective against biofilm-associated infections, necessitating the development of novel therapeutic strategies. The CRISPR/Cas9 gene-editing system has emerged as a revolutionary tool for precision genome modification, offering targeted disruption of antibiotic resistance genes, quorum sensing pathways, and biofilm-regulating factors. However, the clinical application of CRISPR-based antibacterials faces significant challenges, particularly in efficient delivery and stability within bacterial populations. Nanoparticles (NPs) present an innovative solution, serving as effective carriers for CRISPR/Cas9 components while exhibiting intrinsic antibacterial properties. Nanoparticles can enhance CRISPR delivery by improving cellular uptake, increasing target specificity, and ensuring controlled release within biofilm environments. Recent advances have demonstrated that liposomal CRISPR-Cas9 formulations can reduce <em>Pseudomonas aeruginosa</em> biofilm biomass by over 90% in vitro, while gold nanoparticle carriers enhance editing efficiency up to 3.5-fold compared to non-carrier systems. These hybrid platforms also enable co-delivery with antibiotics, producing synergistic antibacterial effects and superior biofilm disruption. Additionally, they can facilitate co-delivery of antibiotics or antimicrobial peptides, further enhancing therapeutic efficacy. This review explores the synergistic integration of CRISPR/Cas9 and nanoparticles in combating biofilm-associated antibiotic resistance. We discuss the mechanisms of action, recent advancements, and current challenges in translating this approach into clinical practice. While CRISPR-nanoparticle hybrid systems hold immense potential for next-generation precision antimicrobial therapies, further research is required to optimize delivery platforms, minimize off-target effects, and assess long-term safety. Understanding and overcoming these challenges will be critical for developing effective biofilm-targeted antibacterial strategies.</p>
<section id="sec1"><h3 class="pmc_sec_title">Graphical Abstract</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366227_12916_2025_4323_Figa_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f063/12366227/f88ac353b29d/12916_2025_4323_Figa_HTML.jpg" loading="lazy" id="MO1" height="437" width="778" alt="graphic file with name 12916_2025_4323_Figa_HTML.jpg"></a></p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Antibiotic resistance, Biofilm, CRISPR/Cas9, Nanoparticles, Gene editing, Antibacterial therapy</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par17">Antibiotic-resistant infections pose a significant threat to global health, with biofilm-associated infections being particularly challenging due to their inherent resistance to conventional antimicrobial therapies [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Biofilms are structured communities of microorganisms such as bacteria or fungi that adhere to surfaces and are embedded in self-produced extracellular polymeric substances (EPS) [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. This matrix creates microenvironments that can reduce antibiotic efficacy by altering bacterial metabolism and limiting the penetration of antibiotics [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Antimicrobial resistance in bacterial populations arises via two distinct but often co-occurring mechanisms. The first involves heritable genetic adaptations, such as the acquisition of resistance genes (e.g., bla, mecA, ndm-1) through plasmids or transposons, allowing bacteria to enzymatically degrade antibiotics or evade their binding. In contrast, biofilm-mediated resistance is primarily phenotypic, driven by the protective extracellular polymeric substance (EPS) matrix, reduced metabolic activity of persister cells, and quorum sensing–regulated efflux systems. Notably, biofilms can exhibit up to 1000-fold greater tolerance to antibiotics compared to planktonic cells [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. This mechanistic divergence highlights the need for a dual-action approach: CRISPR–Cas systems offer precise targeting of genetic resistance determinants, while nanoparticle-based delivery enhances penetration through biofilm barriers—together forming a synergistic antimicrobial strategy. The CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) is a natural defense system found in bacteria and archaea that helps them fight viral infections. The CRISPR/Cas9 system is a powerful tool for gene editing, opening new avenues for combating antibiotic resistance. CRISPR/Cas9 allows for precise targeting and modification of specific genetic sequences, offering potential for eradicating antibiotic resistance genes [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. This system consists of two key components: the Cas9 nuclease, which introduces double-strand breaks in DNA, and a guide RNA (gRNA) that directs Cas9 to the specific genomic sequence. By designing gRNAs to target resistance genes, researchers can disrupt these genes, thereby resensitizing bacteria to antibiotics [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Recent advancements have highlighted the synergistic potential of combining the CRISPR/Cas9 system with nanoparticles to enhance the delivery and efficacy of antimicrobial agents against biofilm-associated infections. Nanoparticles, due to their unique physicochemical properties, can facilitate targeted delivery, improve cellular uptake, and protect genetic material from degradation [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Various types of nanoparticles, such as lipid-based nanoparticles, polymeric nanoparticles, and metallic nanoparticles, have been explored for this purpose. These carriers can be engineered to possess surface modifications that enhance their interaction with biofilm components, ensuring efficient penetration and delivery of CRISPR/Cas9 constructs directly to the bacterial cells. For instance, liposomal Cas9 formulations reduced P. aeruginosa biofilm biomass by over 90% in vitro [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>], and CRISPR–gold nanoparticle hybrids demonstrated a 3.5-fold increase in gene-editing efficiency while promoting synergistic action with antibiotics [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. This combination strategy not only improves the precision of CRISPR/Cas9 targeting but also overcomes the barriers posed by biofilm matrices, thereby enhancing the treatment outcomes of resistant infections [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. Additionally, the use of nanoparticles can facilitate the co-delivery of antibiotics alongside CRISPR/Cas9, creating a multifaceted approach that attacks bacteria through both genetic disruption and traditional antimicrobial mechanisms. This integrated method holds promise for addressing the growing crisis of antibiotic resistance, particularly in chronic and device-associated infections where biofilms are prevalent [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. However, current mono-therapeutic strategies—whether based solely on genetic editing or on nanoparticle-mediated delivery—often fall short in overcoming the complex barriers of biofilm-associated resistance. Therefore, there is an urgent need for interdisciplinary combinatorial strategies that integrate molecular gene-targeting tools such as CRISPR/Cas with advanced nanocarriers for precise and effective antimicrobial therapy [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. This paper aims to provide an in-depth exploration of the integration of CRISPR/Cas9 technology with nanoparticle-based delivery systems, focusing on their combined efficacy in managing and treating antibiotic-resistant infections associated with biofilms.</p></section><section id="Sec2"><h2 class="pmc_sec_title">The antibiotic resistance disaster: a global challenge</h2>
<p id="Par18">Antibiotic resistance represents one of the most urgent threats to global health, food security, and modern development. The rapid emergence and spread of resistant bacteria negatively affect the efficacy of antibiotics, which are crucial for treating bacterial infections. According to the World Health Organization (WHO), antibiotic resistance causes an estimated 700,000 deaths annually, a figure projected to rise dramatically if current trends continue (WHO, 2020). Antibiotic resistance is induced through genetic mutations or the acquisition of resistance genes via horizontal gene transfer. The primary mechanisms include enzymatic degradation or modification where bacteria produce enzymes such as beta-lactamases that deactivate antibiotics [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>], alteration of target sites through mutations in bacterial proteins that prevent antibiotic binding [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>], efflux pumps that bacteria use to expel antibiotics before they can exert their effects [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>], and reduced permeability resulting from changes in the bacterial cell membrane that decrease antibiotic uptake [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Several factors accelerate the development and spread of antibiotic resistance, including overuse and misuse of antibiotics through inappropriate prescribing, self-medication, and over-the-counter availability [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>], agricultural use where antibiotics are used in livestock for growth promotion and disease prevention fostering resistant strains [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>], poor infection control with inadequate hygiene and sanitation in healthcare settings and communities facilitating the spread of resistant bacteria [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>], and global travel and trade which increase the dissemination of resistant strains across borders [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. The consequences of antibiotic resistance are profound, leading to increased morbidity and mortality as resistant infections are harder to treat, resulting in longer illnesses and higher death rates [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. It also imposes an economic burden due to higher healthcare costs from prolonged hospital stays, additional tests, and more expensive treatments (Smith &amp; Coast, 2013), and threatens medical procedures like surgeries, cancer chemotherapy, and organ transplants, which become riskier without effective antibiotics [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Addressing this crisis requires a multifaceted approach, including antibiotic stewardship to promote the appropriate use of antibiotics in healthcare and agriculture [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>], enhanced global monitoring of antibiotic resistance patterns (WHO, 2020), investment in new antibiotics, vaccines, and diagnostic tools [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>], raising public awareness about the risks of antibiotic misuse, and strengthening hygiene practices, vaccination programs, and infection control measures. Through stewardship, surveillance, research, and education, the spread of resistance can be slowed, preserving the efficacy of existing antibiotics for future generations [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>].</p></section><section id="Sec3"><h2 class="pmc_sec_title">Biofilm structure and ultrastructural review of bacterial biofilms</h2>
<p id="Par19">The biofilm structure is highly organized, often displaying a heterogeneous architecture characterized by microcolonies interspersed with water channels that facilitate nutrient distribution and waste removal [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. Biofilm formation begins with the adhesion of microorganisms to a surface, followed by the production of EPS, which holds the cells together and anchors the biofilm to the substrate. The process involves multiple stages, starting from initial attachment, irreversible attachment, maturation, and finally dispersion. Biofilms can consist of a single species of bacteria or multiple species, often including fungi and other microorganisms in complex communities. The extracellular matrix, composed primarily of polysaccharides, proteins, and extracellular DNA (eDNA), forms a protective barrier that limits the penetration of antibiotics and plays a pivotal role in maintaining biofilm integrity and resilience. This matrix is not only a physical impediment but also a source of microenvironments within the biofilm, where bacterial cells experience [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. The heterogeneous structure of biofilms creates microenvironments with varying levels of nutrient availability, pH, oxygen, and waste products, which can contribute to the survival of microorganisms even under challenging conditions [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. At the ultrastructural level, bacterial biofilms exhibit a stratified organization. The basal layer, in close contact with the surface, consists of densely packed cells that form strong adhesions via adhesins and pili, contributing to the initial attachment phase. Above this layer, microcolonies develop, often surrounded by a dense EPS matrix that acts as a protective barrier against antibiotics and immune responses [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>, <a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>–<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. The uppermost layers are less densely packed, with cells exhibiting phenotypic heterogeneity, including persister cells that contribute to antibiotic resistance [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Advanced imaging techniques such as confocal laser scanning microscopy (CLSM) and scanning electron microscopy (SEM) have shown the dynamic and complex nature of biofilm ultrastructure. The intricate architecture and ultrastructural complexity of bacterial biofilms underscore their role in chronic infections and their resilience against antimicrobial treatments. Understanding these structural nuances is critical for developing effective strategies to disrupt biofilm formation and enhance antibiotic efficacy [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. The resistance mechanisms employed by bacteria within biofilms have garnered significant attention due to their role in persistent infections and their ability to circumvent traditional antimicrobial therapies. The development of biofilm-associated resistance involves a variety of mechanisms, including structural barriers, altered metabolic states, genetic adaptation, and intercellular communication. The molecular basis of biofilm resistance is complex, and ongoing research aims to elucidate the key factors involved in this phenomenon [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>].</p></section><section id="Sec4"><h2 class="pmc_sec_title">The importance and mechanism of biofilms in antibiotic resistance</h2>
<p id="Par20">The formation of biofilms allows bacteria to survive in hostile environments, including under the stress of antibiotic treatment. This resistance mechanism is particularly significant in chronic infections and hospital settings, where biofilm-associated infections are prevalent [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. One of the most concerning aspects of biofilms is their ability to contribute to antibiotic resistance. Several mechanisms have been identified by which biofilms enhance bacterial resistance to antibiotics by biofilms. Many antibiotics are less effective against biofilm-associated bacteria because they cannot reach the cells in the inner regions of the biofilm. Within biofilms, bacteria exist in a slower-growing or dormant state. Antibiotics, particularly those targeting cell wall synthesis or protein synthesis, are less effective against slow-growing or dormant cells. As a result, biofilm bacteria can survive even in the presence of therapeutic concentrations of antibiotics [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. The “persisters,” a subset of bacterial cells within biofilms, exhibit a dormant phenotype that is highly resistant to antibiotics. The role of persister cells in biofilm-associated resistance is another critical aspect of the resistance phenotype. Persister cells are a small subpopulation of metabolically inactive or slow-growing cells that can survive exposure to high concentrations of antibiotics. These cells are thought to be present in biofilms as a result of stress-induced dormancy. The mechanisms underlying persister cell formation remain incompletely understood, but several studies suggest that changes in bacterial gene expression, including the upregulation of stress response pathways, are involved [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. The ability of persister cells to endure antimicrobial treatments contributes to the recalcitrance of biofilm infections, as these cells can “wake up” and repopulate the biofilm once the antibiotic pressure is relieved. One of the most significant mechanisms is the upregulation of efflux pumps, which actively expel antibiotics from bacterial cells, reducing the intracellular concentration of the drug. Efflux pumps are often regulated by quorum sensing, a bacterial communication system that involves the release and detection of small signaling molecules. In biofilms, quorum sensing regulates the expression of genes responsible for efflux pump production, as well as other resistance factors, such as oxidative stress response proteins [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Quorum sensing also modulates the production of extracellular matrix components, further promoting biofilm formation and resistance to antimicrobial agents [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Biofilm-associated bacteria often express higher levels of efflux pumps, which can actively expel antibiotics from bacterial cells, thereby reducing their efficacy. Additionally, biofilms can increase the production of enzymes like β-lactamases that degrade antibiotics [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. The physical and chemical structure of the biofilm plays a crucial role in protecting bacterial cells from antimicrobial agents. In addition to the physical barrier provided by the extracellular matrix, biofilm-forming bacteria are equipped with a range of molecular mechanisms that enhance their resistance to antimicrobial agents. Biofilms provide an environment that facilitates genetic exchange among microorganisms. Horizontal gene transfer (HGT), including the transfer of antibiotic resistance genes, is more frequent within biofilms. This can lead to the rapid spread of resistance traits among bacterial populations. The genetic diversity of bacteria within biofilms also plays a pivotal role in the development of resistance. Biofilm environments are conducive to horizontal gene transfer, allowing for the exchange of antibiotic resistance genes among bacterial cells. The dense, cooperative structure of biofilms facilitates the transfer of plasmids, transposons, and other mobile genetic elements, which may carry resistance determinants. This genetic exchange not only accelerates the acquisition of resistance but also results in the establishment of stable, resistant subpopulations within the biofilm [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Furthermore, biofilm formation itself has been shown to induce genetic mutations that contribute to the development of resistance, particularly in response to prolonged antibiotic exposure [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Several recent studies have focused on the role of the biofilm’s microenvironment in modulating antibiotic resistance. The physical characteristics of biofilms, including their thickness and density, influence the diffusion of antimicrobial agents and create microenvironments with reduced drug penetration. This limited access to antibiotics enhances the persistence of bacteria within biofilms, even in the presence of otherwise effective treatments [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Moreover, biofilms in vivo are frequently exposed to sub-lethal concentrations of antibiotics, which may induce adaptive resistance mechanisms such as the activation of efflux pumps, DNA repair pathways, and changes in membrane permeability. These adaptive responses allow bacteria to survive under conditions that would otherwise be fatal [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. Biofilm-associated infections are frequently encountered in clinical settings, particularly among patients with indwelling medical devices such as catheters, prosthetic joints, and cardiac valves. These abiotic surfaces facilitate microbial adhesion and subsequent biofilm development, contributing to persistent infections that exhibit a high degree of tolerance to conventional antimicrobial therapies. Key bacterial species implicated in these infections include <em>Staphylococcus aureus</em>, <em>Pseudomonas aeruginosa</em>, <em>Escherichia coli</em>, and <em>Enterococcus</em> spp. Research about biofilm-targeting strategies has grown in recent years, aiming to develop treatments that can either prevent biofilm formation, disrupt established biofilms, as well as methods to inhibit the initial attachment of bacteria or block the signaling pathways that regulate biofilm formation (e.g., quorum sensing inhibitors) are under investigation. Enzymes that degrade components of the EPS matrix, such as DNases or polysaccharide-degrading enzymes, can enhance the effectiveness of antibiotics by allowing better penetration into the biofilm. Combining antibiotics with agents that disrupt biofilms or enhance antibiotic penetration is a promising strategy. This approach could reduce the likelihood of resistance development by targeting multiple aspects of biofilm-associated bacterial survival. Nanomaterials such as silver nanoparticles or nanostructured surfaces are being explored for their ability to disrupt biofilms and enhance antibiotic efficacy [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>, <a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Bacteriophages, viruses that infect and kill bacteria, are being investigated for their potential to target and disrupt biofilms without promoting antibiotic resistance [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>–<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. The ability of bacteria within biofilms to resist antibiotic treatment, evade immune responses, and exchange resistance genes makes biofilm-associated infections particularly difficult to treat. Understanding the mechanisms underlying biofilm formation and resistance is essential for the development of more effective treatment strategies. Continued research into biofilm biology and innovative therapeutic approaches will be key in addressing the growing challenge of antibiotic resistance [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. Biofilm-associated antibiotic resistance represents a complex and multifactorial challenge in both clinical and environmental contexts. Understanding the underlying mechanisms is crucial for developing effective therapeutic strategies [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Understanding the mechanisms of biofilm resistance has significant clinical implications, particularly for the treatment of chronic and recurrent infections. The recalcitrance of biofilm-associated bacteria to antibiotics necessitates the development of novel therapeutic strategies aimed at disrupting biofilm formation, enhancing drug penetration, or targeting specific resistance mechanisms. Recent advances have explored the use of enzymatic agents that degrade the extracellular matrix, as well as the development of compounds that inhibit quorum sensing or efflux pump activity [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. Additionally, combining traditional antibiotics with agents that disrupt biofilm structure or function may improve treatment outcomes and help overcome the challenges associated with biofilm-associated infections [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>].</p></section><section id="Sec5"><h2 class="pmc_sec_title">CRISPR-Cas system</h2>
<p id="Par21">The CRISPR-Cas system is a revolutionary genetic tool that has reshaped the landscape of molecular biology and genetic engineering. Originally discovered in bacteria and archaea as an adaptive immune mechanism, CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and its associated protein, Cas (CRISPR-associated proteins), have been repurposed for genome editing in diverse organisms. The CRISPR/Cas system roles in microbial defense involve storing segments of viral DNA in the form of CRISPR sequences and using these sequences to guide the Cas proteins to target and cleave foreign DNA during subsequent infections [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. The discovery of the programmable nature of CRISPR-Cas9, in particular, has enabled precise modifications to the genome of various organisms (Table <a href="#Tab1" class="usa-link">1</a>), ushering in an era of unprecedented genetic manipulation [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. The gene-editing system consists of two primary components: the Cas9 endonuclease and a guide RNA (gRNA). The Cas9 protein introduces double-strand breaks (DSBs) in the target DNA at a precise location defined by the gRNA. Once the DSB occurs, the bacterial repair mechanisms, such as non-homologous end joining (NHEJ) or homology-directed repair (HDR), attempt to repair the break, often resulting in gene knockouts or insertions that can disrupt biofilm-related processes. This precise targeting ability enables researchers to investigate the molecular mechanisms underpinning biofilm formation, maintenance, and eradication [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. This precision allows researchers to cut and edit specific genes, making it an invaluable tool for genetic engineering and research. When the DNA is cut by Cas9, the cell attempts to repair the break through one of two pathways: non-homologous end joining (NHEJ) or homology-directed repair (HDR). NHEJ is error-prone and often introduces small insertions or deletions (indels) that can disrupt gene function, while HDR can be used for precise edits if a donor DNA template is provided [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. This system has been harnessed not only for gene knockout experiments but also for more complex tasks, such as gene insertion, correction of mutations, and even gene activation or repression, depending on the modifications made to the Cas9 protein or the RNA guide [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. The applications of CRISPR-Cas9 have revolutionized molecular biology and genetic research, including in fields such as agriculture, medicine, and biotechnology. It has enabled the development of genetically modified organisms (GMOs) with enhanced traits, such as crops resistant to diseases or environmental stressors, as well as the potential for treating genetic diseases through gene therapy. In human medicine, CRISPR-Cas9 has shown promise for correcting genetic defects at the DNA level, with ongoing research into its potential for treating conditions like cystic fibrosis, sickle cell anemia, and muscular dystrophy [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. However, ethical concerns, including the potential for off-target effects and germline editing, remain significant hurdles that need to be addressed as the technology continues to evolve [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. In recent years, improvements to the CRISPR-Cas9 system have been made, including the development of more precise and efficient versions like CRISPR-Cas12 and CRISPR-Cas13, which target DNA and RNA, respectively. These advancements have expanded the utility of CRISPR beyond genome editing to other applications such as RNA editing and diagnostics, offering even more potential for biomedical and therapeutic innovations [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>].
</p>
<section class="tw xbox font-sm" id="Tab1"><h3 class="obj_head">Table 1.</h3>
<div class="caption p"><p>Representative CRISPR/Cas9-nanoparticle delivery systems with antibacterial, anti-inflammatory, or biofilm-relevant applications</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Nanoparticle platform</th>
<th align="left" colspan="1" rowspan="1">CRISPR-Cas9 form</th>
<th align="left" colspan="1" rowspan="1">Target cells/organism</th>
<th align="left" colspan="1" rowspan="1">Target gene</th>
<th align="left" colspan="1" rowspan="1">Model</th>
<th align="left" colspan="1" rowspan="1">Application</th>
<th align="left" colspan="1" rowspan="1">Key findings</th>
<th align="left" colspan="1" rowspan="1">Ref</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Polymeric NPs</strong></td>
<td align="left" colspan="1" rowspan="1">RNPs</td>
<td align="left" colspan="1" rowspan="1">
<em>Staphylococcus aureus</em> (MRSA)</td>
<td align="left" colspan="1" rowspan="1"><strong>mecA</strong></td>
<td align="left" colspan="1" rowspan="1">In vitro</td>
<td align="left" colspan="1" rowspan="1">Antibacterial (resistance)</td>
<td align="left" colspan="1" rowspan="1">Disrupted <em>mecA</em> gene; suppressed resistance to methicillin; restored antibiotic susceptibility</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>PEI Magnetic NPs</strong></td>
<td align="left" colspan="1" rowspan="1">Plasmid DNA</td>
<td align="left" colspan="1" rowspan="1">HEK293 cells</td>
<td align="left" colspan="1" rowspan="1"><strong>TLR3</strong></td>
<td align="left" colspan="1" rowspan="1">In vitro</td>
<td align="left" colspan="1" rowspan="1">Immunomodulatory/host defense</td>
<td align="left" colspan="1" rowspan="1">Magnetic field-assisted CRISPR delivery enabled site-specific genome editing; potential application in modulating host defense genes against infection</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Lipid-based NPs</strong></td>
<td align="left" colspan="1" rowspan="1">Cas9 mRNA + sgRNA</td>
<td align="left" colspan="1" rowspan="1">BMDMs/C57BL/6 mice</td>
<td align="left" colspan="1" rowspan="1"><strong>NLRP3</strong></td>
<td align="left" colspan="1" rowspan="1">In vitro/in vivo</td>
<td align="left" colspan="1" rowspan="1">Anti-inflammatory/sepsis control</td>
<td align="left" colspan="1" rowspan="1">70.2% NLRP3 knockout in vitro; modulated macrophage inflammation pathways; potentially reducing inflammatory damage in infections</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Lipid-based NPs</strong></td>
<td align="left" colspan="1" rowspan="1">Cas9 mRNA + sgRNA</td>
<td align="left" colspan="1" rowspan="1">HEK293 endothelial cells</td>
<td align="left" colspan="1" rowspan="1"><strong>ICAM-2</strong></td>
<td align="left" colspan="1" rowspan="1">In vivo</td>
<td align="left" colspan="1" rowspan="1">Biofilm penetration (vascular)</td>
<td align="left" colspan="1" rowspan="1">Endothelial targeting improved CRISPR delivery; may enhance vascular targeting of antimicrobials</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Gold-lipid hybrid NPs</strong></td>
<td align="left" colspan="1" rowspan="1">Plasmid DNA</td>
<td align="left" colspan="1" rowspan="1">Melanoma + Bacteria models</td>
<td align="left" colspan="1" rowspan="1"><strong>PLK-1</strong></td>
<td align="left" colspan="1" rowspan="1">In vitro/in vivo</td>
<td align="left" colspan="1" rowspan="1">Dual antibacterial/antitumor</td>
<td align="left" colspan="1" rowspan="1">Photothermal CRISPR system disrupted cell proliferation and potentially sensitized microenvironments to therapy</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Lipidoid NPs</strong></td>
<td align="left" colspan="1" rowspan="1">RNPs</td>
<td align="left" colspan="1" rowspan="1">HeLa-DsRed cells (biofilm test)</td>
<td align="left" colspan="1" rowspan="1"><strong>GFP reporter</strong></td>
<td align="left" colspan="1" rowspan="1">In vivo</td>
<td align="left" colspan="1" rowspan="1">CRISPR delivery validation</td>
<td align="left" colspan="1" rowspan="1">Achieved ~ 70% knockout with low toxicity; relevant for CRISPR-based biosensors or delivery in biofilm-like environments</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Polymeric NPs</strong></td>
<td align="left" colspan="1" rowspan="1">Plasmid DNA</td>
<td align="left" colspan="1" rowspan="1">HFD-induced diabetic mice</td>
<td align="left" colspan="1" rowspan="1"><strong>NE (elastase)</strong></td>
<td align="left" colspan="1" rowspan="1">In vivo</td>
<td align="left" colspan="1" rowspan="1">Inflammatory/metabolic infection</td>
<td align="left" colspan="1" rowspan="1">Knockout of neutrophil elastase reduced inflammation; improved insulin sensitivity, suggesting infection-linked inflammation modulation</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Core–shell Fe/PEI NPs</strong></td>
<td align="left" colspan="1" rowspan="1">Plasmid DNA</td>
<td align="left" colspan="1" rowspan="1">Porcine fibroblasts</td>
<td align="left" colspan="1" rowspan="1"><strong>H11</strong></td>
<td align="left" colspan="1" rowspan="1">In vitro</td>
<td align="left" colspan="1" rowspan="1">Gene delivery platform</td>
<td align="left" colspan="1" rowspan="1">3.5 × improvement in delivery over lipofection; potential for CRISPR editing in antimicrobial peptide research</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Polymeric NPs</strong></td>
<td align="left" colspan="1" rowspan="1">Plasmid DNA</td>
<td align="left" colspan="1" rowspan="1">HEK293 + HeLa</td>
<td align="left" colspan="1" rowspan="1"><strong>GFP/iRFP</strong></td>
<td align="left" colspan="1" rowspan="1">In vitro</td>
<td align="left" colspan="1" rowspan="1">CRISPR validation in cell platforms</td>
<td align="left" colspan="1" rowspan="1">70% knockout; useful for building infection reporters or delivery optimization in biofilm models</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec6"><h2 class="pmc_sec_title">Mechanism of action of the CRISPR-Cas9 system</h2>
<p id="Par22">The sgRNA is engineered to be complementary to a target DNA sequence and directs Cas9 to that sequence for cleavage. The precise targeting is facilitated by the presence of a protospacer adjacent motif (PAM) in the DNA sequence, which is required for Cas9 recognition (Fig. <a href="#Fig1" class="usa-link">1</a>). The PAM sequence, typically “NGG” in <em>Streptococcus pyogenes</em> Cas9, serves as a critical determinant for the binding of Cas9 to the target DNA and the subsequent DNA cleavage event [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. Upon recognition of the PAM sequence, the Cas9 protein undergoes a conformational change, which allows it to unwind the DNA around the target sequence. The sgRNA then binds to the complementary DNA strand, forming a DNA-RNA hybrid structure. This binding is essential for the specificity of the system, as the Cas9 protein will only activate its nuclease activity when the guide RNA is fully paired with the target DNA [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>, <a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. Cas9 has two distinct nuclease domains: The RuvC and HNH domains, which play complementary roles in cleaving the DNA. The HNH domain cleaves the strand of the DNA that is complementary to the guide RNA, while the RuvC domain cleaves the non-complementary strand, creating a double-strand break. This precise, staggered cleavage generates sticky ends that can be repaired through the cell's repair mechanisms, leading to either random mutations or targeted insertions, depending on the repair pathway activated [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>].</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366227_12916_2025_4323_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f063/12366227/26237ed57d92/12916_2025_4323_Fig1_HTML.jpg" loading="lazy" id="MO2" height="389" width="778" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mechanisms of CRISPR-Cas9 in combating bacterial biofilms: <strong>a</strong> Resistance gene disruption: CRISPR-Cas9 targets antibiotic resistance genes via sequence-specific guide RNA (gRNA), inducing double-strand breaks at the protospacer adjacent motif (PAM) site. Cas9 cleavage disrupts resistance gene integrity, restoring bacterial susceptibility. <strong>b</strong> Quorum sensing inhibition: CRISPR-Cas9 disrupts quorum sensing (QS) by targeting transcriptional regulators (e.g., LuxR, BMIR proteins), blocking autoinducer-mediated signaling and biofilm coordination. <strong>c</strong> EPS degradation: CRISPR editing of genes encoding extracellular polymeric substances (EPS), such as phosphorylated cellulose (Pel polysaccharide), destabilizes the biofilm matrix, enhancing structural collapse. <strong>d</strong> Suppressing horizontal gene transfer: CRISPR-Cas systems degrade foreign DNA (e.g., plasmids) carrying resistance genes, preventing horizontal gene transfer (HGT) within biofilm communities</p></figcaption></figure><p id="Par23">Once the DSB is induced, the cell responds with one of two major DNA repair mechanisms: non-homologous end joining (NHEJ) or homology-directed repair (HDR). NHEJ is the predominant repair pathway, particularly in mammalian cells. It is an error-prone mechanism that directly ligates the broken ends of DNA without a homologous template, often leading to insertions or deletions (indels) at the break site. These indels can cause frameshift mutations, effectively disrupting the targeted gene’s function. Alternatively, HDR is a more accurate repair mechanism that utilizes a homologous template to repair the break. This template could be provided exogenously, such as a donor plasmid or a single-stranded oligodeoxynucleotide (ssODN), and can be used for precise gene edits or insertions. HDR is less efficient in non-dividing cells, which makes it challenging for precise editing in certain tissues, but is still invaluable when precise gene editing is required [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. The precise action of CRISPR-Cas9 in editing genomes also hinges on its specificity. A major limitation of the system is its propensity for off-target effects, where Cas9 induces DNA breaks at unintended genomic locations that have sequence homology to the target sequence. Off-target cleavage typically occurs when the sgRNA binds to a similar but non-identical sequence elsewhere in the genome, especially when the off-target sequence contains a PAM sequence [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>, <a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. These off-target effects could lead to unwanted genetic mutations or disruptions in functional genes, which could have unintended consequences, particularly in therapeutic applications. To minimize such risks, several strategies have been developed to increase the specificity of the CRISPR-Cas9 system. One of the most notable approaches involves the engineering of high-fidelity Cas9 variants that exhibit reduced off-target cleavage while maintaining editing efficiency. These include Cas9 variants such as eSpCas9, HypaCas9, and SpCas9-HF1, all of which have been shown to reduce off-target activity significantly without compromising the system’s on-target editing efficacy [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. Additionally, research into alternative CRISPR-associated proteins, such as Cpf1 (now known as Cas12), has shown promise for improving target specificity. Cas12 has distinct features, such as a different PAM sequence requirement (“TTTN”), and an ability to generate staggered cuts, which may reduce off-target activity and expand the range of editable loci [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. The applications of CRISPR-Cas9 extend beyond basic genome editing to therapeutic and biotechnological realms. The ability to induce precise gene modifications, correct genetic mutations, or knock out genes has profound implications for the treatment of genetic disorders, such as sickle cell anemia and Duchenne muscular dystrophy [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. Furthermore, CRISPR-Cas9 enables functional genomics, allowing researchers to systematically study gene function by creating knockout or knock-in cell lines, animal models, and organoids. Despite its immense potential, challenges such as delivery mechanisms, off-target effects, and efficiency of HDR in non-dividing cells remain areas of active research [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. Recent improvements in CRISPR technology, including advanced delivery techniques and the development of next-generation CRISPR-Cas systems, are addressing some of these limitations and will likely accelerate the therapeutic and industrial applications of genome editing [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>].</p></section><section id="Sec7"><h2 class="pmc_sec_title">The CRISPR-Cas system and biofilm-related antibacterial resistance</h2>
<p id="Par24">The CRISPR-Cas system has gained significant attention for its potential in antimicrobial strategies, particularly in targeting antibiotic-resistant bacterial populations within biofilms. One of the main reasons biofilm-associated bacteria exhibit increased antibiotic resistance is due to limited drug penetration, altered metabolic states, and horizontal gene transfer (HGT), which facilitates the spread of resistance determinants [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. The CRISPR-Cas system plays a dual role in biofilm-related resistance: while it can function as a defense mechanism against phages and plasmids carrying antibiotic resistance genes, it also influences biofilm formation through regulatory pathways [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. CRISPR-Cas9 technology, originally discovered as a bacterial immune system against viral infections, has rapidly evolved as a powerful tool for genome editing, with promising applications in combating antibiotic resistance. One of the promising applications of CRISPR-Cas9 in this field is its ability to specifically target and disrupt antibiotic resistance genes, thereby restoring bacterial susceptibility to antibiotics. Recent studies have demonstrated the ability of CRISPR-Cas9 to target and remove resistance genes from the genomes of pathogenic bacteria. For example, one study successfully utilized CRISPR-Cas9 to cleave the mcr-1 gene, which confers resistance to colistin, in urgent need for novel antimicrobial Escherichia coli strains. This approach effectively restored the bacteria’s sensitivity to colistin, a last-resort antibiotic, showcasing CRISPR-Cas9’s potential to reverse antimicrobial resistance (AMR) [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>, <a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]. Additionally, CRISPR-Cas9 has been applied to target and eliminate plasmids carrying antibiotic resistance genes, such as extended-spectrum beta-lactamase (ESBL) genes, in Klebsiella pneumoniae and Pseudomonas aeruginosa, further proving its ability to reverse resistance mechanisms [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. Moreover, CRISPR-Cas9 has been adapted for use in gene drives, a system that ensures the inheritance of specific genes in a population. Researchers have explored the use of CRISPR-based gene drives to target resistance genes across bacterial populations. This strategy has shown promise in preventing the spread of antibiotic resistance in clinical and environmental settings [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. One of the main advantages of using CRISPR-Cas9 over traditional antibiotic therapies is its precision; while antibiotics indiscriminately kill both pathogenic and beneficial bacteria, CRISPR-Cas9 can specifically target resistance genes without affecting the surrounding microbial community, minimizing collateral damage to the microbiome [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>, <a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. The widespread and clinical application of the CRISPR system still faces challenges, with the most significant challenge being the delivery of CRISPR complex components. Issues such as delivery efficiency, particularly in the context of human infections, and the potential for off-target effects need to be addressed. The design of safe and effective delivery systems, such as viral vectors or nanoparticles, is critical for successful clinical application [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>, <a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. Additionally, there are concerns about the potential for resistance to CRISPR-Cas9 systems themselves, as bacteria may evolve mechanisms to resist CRISPR-mediated genome editing [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]. Despite these hurdles, the application of CRISPR-Cas9 systems represents a transformative approach in the fight against antibiotic resistance. Ongoing research and clinical trials are crucial to overcoming these challenges and unlocking the full potential of CRISPR-based therapies to combat resistant infections [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. Studies have shown that certain CRISPR-Cas systems directly regulate biofilm formation. For example, Type I and Type II CRISPR-Cas systems can modulate quorum sensing (QS) pathways, which are critical for biofilm development. In some bacteria, the disruption of CRISPR-Cas components has been linked to enhanced biofilm formation, suggesting that CRISPR interference can downregulate genes associated with biofilm stability. Conversely, biofilm-forming bacteria often exhibit reduced CRISPR activity, allowing them to acquire foreign genetic material, including antibiotic resistance genes, via HGT [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>, <a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>].</p></section><section id="Sec8"><h2 class="pmc_sec_title">CRISPR-based strategies for biofilm eradication</h2>
<p id="Par25">Because biofilm-associated resistance has created many challenges, researchers have explored CRISPR-based approaches to disrupt biofilms and re-sensitize bacteria to antibiotics. CRISPR-Cas9 and CRISPR-Cas3 nucleases have been employed to selectively degrade antibiotic resistance genes in biofilm-embedded bacteria, leading to increased susceptibility to conventional antibiotics [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. Additionally, engineered phage-delivered CRISPR systems can selectively lyse biofilm-forming pathogens by targeting essential survival genes. These strategies have shown promising results against clinically relevant pathogens, including <em>Pseudomonas aeruginosa</em>, <em>Escherichia coli</em>, and <em>Staphylococcus aureus</em> [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>, <a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>, <a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. Recent advancements have also explored CRISPR-guided antimicrobials, wherein programmable CRISPR systems induce targeted double-stranded DNA breaks within resistance genes, effectively eliminating resistant subpopulations from biofilms. Additionally, the combination of CRISPR-based disruption with conventional antibiotics has demonstrated synergistic effects, significantly enhancing bacterial clearance [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>, <a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>].</p></section><section id="Sec9"><h2 class="pmc_sec_title">Mechanisms of CRISPR/Cas9 editing in biofilm control</h2>
<p id="Par26">In the context of biofilm control, CRISPR/Cas9 can be used to manipulate several key genetic pathways that regulate biofilm formation. The CRISPR/Cas9 system is guided by RNA molecules to specific DNA sequences, often those that regulate the synthesis of extracellular polymeric substances (EPS), which form the matrix of the biofilm. For example, in Staphylococcus aureus, the icaA gene encodes an enzyme involved in the biosynthesis of PIA, a key component of the biofilm matrix. Disrupting icaA via CRISPR/Cas9 results in reduced biofilm formation and impaired bacterial adhesion, a crucial process for biofilm establishment [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. One of the central mechanisms by which biofilms are formed is quorum sensing (QS), a process by which bacteria communicate to regulate group behaviors such as biofilm formation, virulence, and antibiotic resistance. In Pseudomonas aeruginosa, the las and rhl QS systems are vital for biofilm development. These systems involve the production and detection of signaling molecules, which regulate the expression of genes involved in biofilm formation and antibiotic resistance. CRISPR/Cas9 can be used to knock out key genes in these pathways, such as lasR, rhlR, and luxR, to prevent QS and biofilm formation. By targeting these genes, the system could effectively reduce biofilm biomass and make bacterial populations more susceptible to antibiotics [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>, <a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. Another important target is the cyclic-di-GMP signaling pathway. Cyclic-di-GMP is a second messenger molecule that plays a critical role in the regulation of biofilm formation in many bacteria. High levels of cyclic-di-GMP promote biofilm formation, while low levels encourage planktonic growth. Genes like pelA, which encode enzymes involved in cyclic-di-GMP synthesis, are key targets for CRISPR/Cas9 editing. By knocking out or downregulating these genes, researchers have successfully reduced biofilm formation in Pseudomonas aeruginosa. This approach can also be extended to other bacteria where cyclic-di-GMP is involved in biofilm regulation, such as Escherichia coli [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>, <a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>, <a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>, <a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>, <a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>].</p></section><section id="Sec10"><h2 class="pmc_sec_title">Engineering CRISPR/Cas9 for biofilm disruption</h2>
<p id="Par27">For the efficient application of CRISPR/Cas9 in biofilm-targeting strategies, the delivery of CRISPR components into biofilm-embedded bacteria presents a significant challenge. Biofilms act as physical barriers, and the dense EPS matrix often limits the penetration of genetic material, including CRISPR/Cas9 constructs. Various strategies have been explored to overcome this limitation. One approach is the use of nanoparticles or liposomes to deliver CRISPR components (Fig. <a href="#Fig2" class="usa-link">2</a>). These carriers can penetrate biofilm matrices, facilitate the uptake of plasmid DNA or CRISPR/Cas9 complexes, and enhance gene editing efficacy [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. Additionally, bacteriophage-mediated delivery has shown promise in overcoming delivery barriers. Engineered bacteriophages that carry CRISPR/Cas9 components can specifically target and infect bacterial cells within biofilms, thus delivering the gene-editing machinery directly to the desired location [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. Phage-based CRISPR systems have been successfully used to edit genes in <em>Klebsiella pneumoniae</em> biofilms, offering a targeted and efficient means of biofilm disruption. Phage-based CRISPR delivery may also offer reduced cytotoxicity to surrounding host cells, making it an attractive strategy for clinical applications [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>, <a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]. In addition to plasmid-based or phage-based delivery, some studies have explored electroporation and other physical methods to facilitate the uptake of CRISPR/Cas9 components in biofilm-forming bacteria. These techniques can increase the permeability of bacterial cell membranes, allowing the introduction of CRISPR plasmids or proteins into biofilm-associated bacteria [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>, <a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>].</p>
<figure class="fig xbox font-sm" id="Fig2"><h3 class="obj_head">Fig. 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366227_12916_2025_4323_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f063/12366227/38b46b090c3c/12916_2025_4323_Fig2_HTML.jpg" loading="lazy" id="MO3" height="385" width="778" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Engineered nanoparticle platform for CRISPR-Cas9 delivery: mechanisms and applications in biofilm eradication and resistance gene suppression. The figure illustrates multi-purpose nanoparticles vacillating between lipid-based polymeric nanoparticles, such as PLGA and chitosan, and inorganic nanoparticles, such as Au and iron oxide, all types of delivery systems for CRISPR-Cas9 into bacterial biofilms. Nanoparticles scatter across the extracellular polymeric matrix while being taken up by bacterial cells through the receptor-mediated or clathrin/caveolin-dependent endocytosis pathway [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. Following uptake, the CRISPR-Cas9 RNPs are released via endosomal escape mechanisms [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>], allowing for genome editing</p></figcaption></figure></section><section id="Sec11"><h2 class="pmc_sec_title">Quorum sensing inhibition and biofilm disruption</h2>
<p id="Par28">Quorum sensing (QS) is a bacterial communication system that regulates biofilm formation and antibiotic resistance. CRISPR-Cas9 has been employed to disrupt QS-regulated genes, such as <em>luxS</em>, <em>lasI</em>, and <em>rhlR</em>, leading to biofilm dispersal and increased bacterial susceptibility to antimicrobials. Nanoparticle formulations enhance this strategy by enabling precise delivery and controlled release of CRISPR-Cas9 components within biofilm structures. Recent studies have highlighted the efficacy of NP-CRISPR systems in QS inhibition, effectively weakening biofilm resilience [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>].</p></section><section id="Sec12"><h2 class="pmc_sec_title">Direct eradication of biofilm-forming bacteria</h2>
<p id="Par29">Beyond targeting antibiotic resistance genes and QS pathways, CRISPR-Cas9 can be programmed to disrupt essential biofilm-associated genes. Genes involved in adhesion (<em>fimH</em>, <em>bap</em>) and EPS production (<em>psl</em>, <em>pel</em>, <em>icaA</em>) are prime targets for biofilm eradication [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>, <a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. NP-based CRISPR systems allow targeted gene knockouts, resulting in structural destabilization and enhanced antimicrobial penetration. This strategy has shown significant promise in reducing biofilm biomass and restoring bacterial clearance by antibiotics and host immune defenses. Nanoparticle-mediated CRISPR-Cas9 delivery presents a revolutionary approach for tackling biofilm-associated antibiotic resistance. By ensuring precise targeting of resistance genes, quorum sensing pathways, and biofilm integrity, NP-CRISPR strategies hold great potential for clinical applications. However, challenges such as delivery efficiency, immune responses, and scalability need to be addressed for translational success. Future research should focus on optimizing nanoparticle formulations, enhancing bacterial specificity, and exploring combination therapies with existing antimicrobials. The integration of NP-CRISPR technology with other antimicrobial strategies may pave the way for next-generation biofilm control and infection management [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>].</p></section><section id="Sec13"><h2 class="pmc_sec_title">Relationships between the CRISPR-Cas system and antibiotic resistance</h2>
<p id="Par30">The CRISPR-Cas system has shown a complex relationship with antibiotic resistance, as it not only serves as a defense against phage attacks but also influences the dynamics of horizontal gene transfer (HGT) and the spread of antibiotic resistance genes (ARGs). The role of the CRISPR-Cas system in antibiotic resistance can be twofold. On one hand, it can limit the acquisition and dissemination of resistance genes through HGT. By cleaving the foreign plasmids or DNA that carry these resistance genes, CRISPR-Cas systems can act as a barrier to the spread of resistance within microbial populations [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. This has been demonstrated in studies where bacteria with active CRISPR systems were less likely to acquire plasmids carrying ARGs compared to CRISPR-deficient strains [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]. On the other hand, some studies suggest that bacteria may develop resistance to the CRISPR-Cas system itself, allowing the persistence of antibiotic resistance. For example, mutations in the CRISPR sequences or the Cas proteins can result in the loss of immune function, facilitating the accumulation and spread of resistance genes. In some cases, bacteria have developed mechanisms to suppress CRISPR activity or acquire immunity to specific Cas proteins, which could allow the persistence of resistance genes even in the presence of CRISPR-mediated defense [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>]. Moreover, CRISPR-Cas systems have been shown to influence the evolution of multidrug resistance in clinical isolates, where CRISPR interference could contribute to the selection of resistant strains by altering the bacterial genomic landscape [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>]. CRISPR interference with mobile genetic elements carrying ARGs may also be influenced by bacterial population dynamics. In densely populated communities, the horizontal transfer of resistance genes can be a key factor in the spread of resistance. The CRISPR-Cas system's role in limiting or enhancing this transfer is context-dependent, varying based on the strain, environmental factors, and the specific type of CRISPR system present. For instance, Type I and Type III CRISPR systems, which primarily target RNA and DNA, may exert different selective pressures on the spread of resistance compared to the more widely studied Type II system, which is used in genome editing [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>, <a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>]. Furthermore, the relationship between CRISPR-Cas systems and antibiotic resistance is not always direct. Other factors, such as the presence of specific bacteriophages or the dynamics of microbial communities, can modify the impact of CRISPR on antibiotic resistance. For example, phage therapy has been proposed as an alternative to antibiotics, where the CRISPR-Cas system could prevent bacterial adaptation to phages, but it may also interfere with the phage’s ability to deliver genetic material, including resistance traits, to bacterial hosts [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>].</p></section><section id="Sec14"><h2 class="pmc_sec_title">Applications of nanoparticles in antibiotic therapy</h2>
<p id="Par31">Nanoparticles (NPs) are increasingly explored for their potential in enhancing antibiotic therapy, primarily due to their unique physicochemical properties, such as high surface area, size-dependent behavior, and the ability to modify surface interactions. These features enable nanoparticles to overcome the challenges posed by conventional antibiotics, such as poor bioavailability, rapid clearance, and the emergence of antibiotic-resistant bacteria. The ability of nanoparticles to deliver drugs more effectively, enhance their stability, and offer targeted therapy has made them a promising tool in combating antibiotic-resistant infections [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>, <a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>]. Nanoparticles, especially liposomes, micelles, dendrimers, and polymeric NPs, have shown promising applications as carriers for antibiotics. The high surface-to-volume ratio of NPs facilitates the encapsulation of hydrophobic antibiotics, thereby improving their solubility, stability, and bioavailability. Furthermore, targeted drug delivery is achieved by functionalizing NPs with ligands that selectively bind to bacterial cells, ensuring enhanced local concentrations of antibiotics at the site of infection [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>].</p></section><section id="Sec15"><h2 class="pmc_sec_title">Drug/gene delivery to combat bacterial infection</h2>
<p id="Par32">The mechanism of action of drug and gene delivery systems is essential to understand in order to optimize the therapeutic efficacy against bacterial infections. The fundamental goal of both drug and gene delivery is to selectively target bacterial cells or infected tissues, allowing for the efficient release of therapeutic agents at the site of infection while minimizing systemic toxicity. Several mechanisms are employed in the delivery of drugs and genes to combat bacterial infections, with the most prominent ones being passive targeting, active targeting, and stimuli-responsive release [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>, <a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>].</p></section><section id="Sec16"><h2 class="pmc_sec_title">Drug delivery mechanisms</h2>
<section id="Sec17"><h3 class="pmc_sec_title">Passive targeting</h3>
<p id="Par33">Passive targeting relies on the physicochemical properties of the drug delivery system and the biological environment. One of the most common strategies for passive targeting is the use of nanoparticles, which are designed to exploit the enhanced permeability and retention (EPR) effect. The EPR effect occurs in the inflamed or infected tissues, where blood vessels become leaky, allowing nanoparticles to accumulate at higher concentrations. This mechanism is particularly useful for delivering antibiotics to sites of infection, such as abscesses or chronic wounds [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>].</p></section><section id="Sec18"><h3 class="pmc_sec_title">Active targeting</h3>
<p id="Par34">Active targeting involves the modification of the surface of drug carriers to enable recognition by specific bacterial cells or infected tissue. This can be achieved by functionalizing nanoparticles with ligands (such as antibodies, peptides, or aptamers) that bind to specific bacterial surface markers. For example, liposomes or polymeric nanoparticles can be conjugated with antibodies against bacterial antigens like lipopolysaccharides on Gram-negative bacteria, allowing for targeted delivery of antibiotics [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>]. Additionally, bacterial receptors such as pili or fimbriae can be exploited for targeting bacterial cells directly [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>].</p></section><section id="Sec19"><h3 class="pmc_sec_title">Controlled release and nanocarriers</h3>
<p id="Par35">Nanocarriers, such as liposomes, micelles, and dendrimers, can encapsulate drugs and control their release at the site of infection. These carriers often employ diffusion, degradation, or swelling mechanisms to release the drug over time. For example, a pH-sensitive polymeric carrier will release its contents in an acidic environment, which is typical of infected tissues. Additionally, some nanocarriers respond to the enzymatic activity in the bacterial cell wall or biofilms, ensuring that the antibiotic is released only when it reaches the targeted site [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>].</p></section></section><section id="Sec20"><h2 class="pmc_sec_title">Gene delivery mechanisms</h2>
<section id="Sec21"><h3 class="pmc_sec_title">Plasmid DNA delivery</h3>
<p id="Par36">One of the most straightforward gene delivery mechanisms involves the use of plasmid DNA, which is delivered directly into bacterial cells or host tissues to express antimicrobial peptides, enzymes, or other therapeutic proteins. This is typically achieved using viral vectors, such as adenoviruses or lentiviruses, which can deliver genes into the target cells. Alternatively, non-viral vectors such as cationic liposomes or nanoparticles can facilitate the uptake of plasmid DNA through endocytosis [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>]. Upon internalization, the plasmid DNA is released into the cytoplasm, where it can be transcribed and translated into the desired therapeutic proteins [<a href="#CR95" class="usa-link" aria-describedby="CR95">95</a>].</p></section><section id="Sec22"><h3 class="pmc_sec_title">CRISPR-Cas9 gene editing</h3>
<p id="Par37">Delivery of CRISPR-Cas9 components, such as the Cas9 protein and guide RNA, is achieved using nanoparticles or viral vectors. These systems allow for precise and targeted gene editing, which can either knock out bacterial resistance genes or enhance the host immune response against the infection [<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>].</p></section><section id="Sec23"><h3 class="pmc_sec_title">Bacteriophage-based gene delivery</h3>
<p id="Par38">In this approach, phages can be modified to deliver therapeutic genes directly to bacterial cells. The mechanism of action involves the bacteriophage attaching to specific receptors on bacterial cell walls, injecting its genetic material into the bacterium, and replicating inside the cell. This method can be used to deliver genes encoding antimicrobial peptides or enzymes that degrade bacterial cell walls, thus helping to combat infection. Furthermore, bacteriophage-derived systems can be used to disrupt bacterial biofilms, a common mechanism of antibiotic resistance [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>, <a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>].</p></section><section id="Sec24"><h3 class="pmc_sec_title">Stimuli-responsive gene delivery</h3>
<p id="Par39">Gene delivery can also be enhanced by incorporating stimuli-responsive carriers. These carriers respond to environmental changes, such as pH, temperature, or specific enzymes found in the infection site, which trigger the release of therapeutic genes. For example, pH-sensitive nanoparticles can release their cargo when exposed to the acidic conditions present in infected tissues, ensuring that the gene therapy is localized to the site of infection [<a href="#CR97" class="usa-link" aria-describedby="CR97">97</a>]. Such controlled release mechanisms are critical for minimizing side effects and improving therapeutic outcomes. Both drug and gene delivery mechanisms aim to improve the precision, efficacy, and safety of therapies used to combat bacterial infections. While drug delivery systems primarily focus on enhancing drug stability, bioavailability, and targeted delivery, gene delivery systems are focused on enhancing the host’s defense mechanisms or directly disrupting bacterial virulence. The integration of nanoparticles, CRISPR technologies, and bacteriophage-based systems holds great promise in overcoming the challenges posed by antibiotic resistance and improving the overall management of bacterial infections [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>].</p></section></section><section id="Sec25"><h2 class="pmc_sec_title">Enhancing antibiotic efficacy and overcoming resistance</h2>
<p id="Par40">NPs can enhance the efficacy of antibiotics, particularly against multidrug-resistant (MDR) pathogens. They act as potent adjuvants, increasing bacterial membrane permeability and facilitating the uptake of antibiotics. Some NPs can also disrupt biofilms, a common feature of chronic infections caused by antibiotic-resistant bacteria. Metal-based NPs, such as silver (Ag NPs), gold (Au NPs), and copper oxide (CuO NPs), possess intrinsic antimicrobial properties, which, when combined with antibiotics, exhibit synergistic effects that can overcome resistance mechanisms [<a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>, <a href="#CR99" class="usa-link" aria-describedby="CR99">99</a>]. Beyond their role in drug delivery, some nanoparticles possess direct antibacterial properties. Metal and metal oxide nanoparticles interact with bacterial cells through mechanisms such as reactive oxygen species (ROS) generation, disruption of bacterial cell walls, and interference with cellular components like proteins and DNA. For instance, silver nanoparticles (Ag NPs) are known to release silver ions that damage bacterial cell membranes, leading to cell death. Similarly, copper oxide nanoparticles (CuO NPs) are effective against both Gram-positive and Gram-negative bacteria due to their ability to generate oxidative stress [<a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>, <a href="#CR101" class="usa-link" aria-describedby="CR101">101</a>]. One of the major challenges with conventional antibiotics is their systemic toxicity and side effects. NPs help reduce these effects by providing localized drug release, which limits the exposure of non-target tissues to high drug concentrations. Additionally [<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>] or targeting moieties can further minimize toxicity while enhancing therapeutic outcomes. The integration of nanoparticles with antibiotics in combination therapies has proven to be a powerful strategy. Multifunctional nanoparticles can simultaneously deliver antibiotics and other therapeutic agents, such as anti-inflammatory or anticancer drugs, for a more comprehensive treatment approach. These NPs can also be engineered to respond to external stimuli like pH, temperature, or light, allowing for controlled and site-specific drug release [<a href="#CR103" class="usa-link" aria-describedby="CR103">103</a>]. Despite the promising applications of nanoparticles in antibiotic therapy, several challenges remain. Issues such as potential toxicity, immune system interactions, and the large-scale production of nanoparticles need to be addressed. Furthermore, the long-term effects of NPs, particularly in terms of environmental and human health, require thorough investigation. Continued advancements in nanomaterial design, biocompatibility, and regulatory approval are essential for the clinical translation of nanoparticle-based antibiotic therapies [<a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>].</p></section><section id="Sec26"><h2 class="pmc_sec_title">Nanoparticles: carrier of the CRISPR-Cas system</h2>
<p id="Par41">Traditional antimicrobial strategies often fail against biofilms, necessitating alternative approaches. CRISPR-Cas9, a genome-editing tool capable of precise genetic modifications, has emerged as a promising strategy for targeting biofilm-associated resistance genes. However, efficient delivery of CRISPR components into biofilms remains a major hurdle. Nanoparticle-based delivery systems offer a viable solution by enhancing CRISPR-Cas9 stability, biofilm penetration, and targeted bacterial eradication [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. Nanoparticles have emerged as promising carriers for the CRISPR-Cas system, addressing the challenges of efficient gene editing by improving the delivery of CRISPR components to target cells, tissues, or organs [<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>]. The CRISPR-Cas system, especially the CRISPR-Cas9 gene-editing tool, requires efficient delivery to ensure successful genetic modification (Fig. <a href="#Fig2" class="usa-link">2</a>). Nanoparticles, due to their unique physical and chemical properties such as high surface-area-to-volume ratio and ability to encapsulate both hydrophilic and hydrophobic substances, provide an effective solution [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Different types of nanoparticles have been explored for CRISPR delivery (Table <a href="#Tab2" class="usa-link">2</a>). Liposomes, lipid-based nanoparticles, are among the most commonly used because of their ability to fuse with cell membranes, facilitating the transport of CRISPR components such as Cas9 proteins and guide RNAs (gRNAs) [<a href="#CR105" class="usa-link" aria-describedby="CR105">105</a>]. Polymeric nanoparticles, created from synthetic or natural polymers, have also shown promise due to their capacity to package CRISPR components and control release, protecting the system from degradation [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Inorganic nanoparticles, including gold, silica, and iron oxide nanoparticles, offer additional benefits due to their stability and ability to be functionalized for gene delivery or imaging purposes. Exosome-mimetic nanoparticles, which are modeled after naturally occurring exosomes, also present a unique approach to delivering CRISPR systems, capitalizing on their inherent ability to interact with cells [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>, <a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>, <a href="#CR99" class="usa-link" aria-describedby="CR99">99</a>, <a href="#CR105" class="usa-link" aria-describedby="CR105">105</a>]. The mechanisms of nanoparticle-mediated CRISPR-Cas delivery involve the uptake of nanoparticles through cellular pathways such as endocytosis, which can occur via clathrin-mediated or caveolin-mediated routes. Once inside the cell, the nanoparticles must release the CRISPR components into the cytoplasm, often overcoming barriers such as the endosomal membrane. Once the Cas9 protein reaches the nucleus, it induces double-strand breaks in the target DNA, leading to gene knockout or insertion via the cell's DNA repair machinery [<a href="#CR106" class="usa-link" aria-describedby="CR106">106</a>, <a href="#CR107" class="usa-link" aria-describedby="CR107">107</a>]. Various nanoparticles (NPs) have been designed to enhance CRISPR-Cas9 delivery for biofilm eradication. Lipid nanoparticles (LNPs) provide high transfection efficiency and protect CRISPR components from enzymatic degradation. Polymeric nanoparticles, such as poly (lactic-co-glycolic acid) (PLGA) and chitosan-based NPs, offer controlled release and biofilm-targeting capabilities. Inorganic nanoparticles, including gold and silica-based NPs, enhance stability and can be functionalized for specific targeting. Additionally, cell-penetrating peptide-modified NPs facilitate uptake by bacterial cells and biofilm penetration [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR108" class="usa-link" aria-describedby="CR108">108</a>]. While CRISPR-Cas9 systems are often scrutinized for their off-target gene-editing effects, the nanoparticle carriers themselves introduce additional risks related to delivery specificity. In particular, cationic nanoparticles, which are frequently employed for complexing negatively charged nucleic acids, exhibit a high affinity for anionic host cell membranes. This property can result in nonspecific cellular uptake, leading to off-target delivery in mammalian cells and reduced therapeutic precision [<a href="#CR109" class="usa-link" aria-describedby="CR109">109</a>]. Surface modifications, such as PEGylation or zwitterionic coatings, have been shown to reduce unintended interactions and improve biocompatibility, but optimization must be tailored to the specific delivery context [<a href="#CR110" class="usa-link" aria-describedby="CR110">110</a>]. However, the nanoparticle-mediated CRISPR delivery systems have several challenges. Toxicity is a concern, as some nanoparticles can damage cells or organs. Furthermore, not all cells take up nanoparticles efficiently, which can hinder the overall delivery efficacy. Despite advances, off-target gene editing remains a problem for CRISPR/Cas9 system, although improved targeting strategies with nanoparticles are being explored to enhance precision. Additionally, immune responses against synthetic nanoparticles can limit their therapeutic potential [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>]. Another critical consideration is the bioaccumulation potential of inorganic nanoparticles, particularly metal-based carriers such as gold (AuNPs) or silver nanoparticles (AgNPs). These particles, while effective in enhancing CRISPR delivery and biofilm penetration, are known to accumulate in the liver, spleen, and other reticuloendothelial tissues, potentially triggering long-term toxicity or immune responses [<a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>]. Chronic exposure studies in animal models have shown that even low-dose accumulation of metal nanoparticles may alter hepatic function and induce oxidative stress [<a href="#CR112" class="usa-link" aria-describedby="CR112">112</a>]. Thus, comprehensive in vivo biosafety assessments are essential before clinical translation of nanoparticle-assisted CRISPR therapeutics. Recent developments focus on optimizing the size, charge, and surface functionalization of nanoparticles to improve CRISPR delivery [<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>]. Surface functionalization, using targeting ligands such as antibodies or peptides, helps guide nanoparticles to specific cell types and reduces unintended interactions. Moreover, stimuli-responsive nanoparticles that respond to environmental cues such as pH or temperature offer controlled release of CRISPR components at the desired target site [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>, <a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>, <a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>, <a href="#CR105" class="usa-link" aria-describedby="CR105">105</a>]. The efficacy of CRISPR–nanoparticle systems in treating biofilm-associated infections depends not only on the engineering of advanced nanocarriers but also on a nuanced understanding of the microbiological microenvironment. Mature biofilms exhibit localized acidic conditions (pH 5.0–6.5) due to anaerobic bacterial metabolism, which provides a unique physicochemical cue for targeted drug release. pH-responsive nanoparticles, typically constructed from acid-labile linkers or protonatable polymers (e.g., poly(histidine), poly(β-amino esters)), are designed to remain stable at physiological pH but undergo destabilization or payload release in acidic niches [<a href="#CR113" class="usa-link" aria-describedby="CR113">113</a>]. This allows CRISPR-Cas cargos to be released precisely within the biofilm matrix, minimizing premature degradation and maximizing localized gene editing. For example, Tu et al. (2020) demonstrated that acid-sensitive polymeric nanoparticles could effectively release CRISPR/Cas9 payloads in acidic tumor microenvironments, a strategy directly translatable to acidic biofilm cores [<a href="#CR114" class="usa-link" aria-describedby="CR114">114</a>]. Such interdisciplinary integration of material science and microbiological insight is essential for achieving spatially controlled delivery and enhancing therapeutic specificity. The synergistic effect between CRISPR/Cas9 and nanoparticle carriers arises from the complementary strengths of both systems, particularly in overcoming biofilm-associated delivery barriers. Cationic or surface-functionalized nanoparticles can disrupt the extracellular polymeric substance (EPS) matrix through electrostatic interaction or enzymatic co-loading, thereby improving the penetration of CRISPR components into deeper biofilm layers [<a href="#CR115" class="usa-link" aria-describedby="CR115">115</a>]. Additionally, ligand-modified nanoparticles, such as those functionalized with bacterium-specific antibodies or antimicrobial peptides, enhance targeting specificity, reducing off-target effects and increasing CRISPR localization at infection sites [<a href="#CR114" class="usa-link" aria-describedby="CR114">114</a>]. Moreover, pH-responsive or redox-responsive nanocarriers are designed to release CRISPR cargos selectively within acidic or oxidative microenvironments typical of mature biofilms, ensuring spatially controlled gene editing [<a href="#CR113" class="usa-link" aria-describedby="CR113">113</a>]. In some designs, nanoparticles act as co-delivery platforms, carrying both CRISPR constructs and traditional antibiotics or EPS-degrading enzymes, achieving a multi-pronged disruption of biofilm defenses and promoting synergistic bacterial clearance [<a href="#CR116" class="usa-link" aria-describedby="CR116">116</a>]. Nanoparticle-mediated CRISPR-Cas delivery has broad applications. Gene therapy has the potential to treat genetic disorders by enabling precise genetic modifications in somatic cells. In cancer therapy, CRISPR-Cas9 can be used to modify immune cells, such as T cells, to enhance their ability to target cancer cells. Furthermore, agricultural biotechnology benefits from CRISPR technology for crop modification, with nanoparticles improving the delivery of CRISPR systems into plant cells to generate traits such as disease resistance or increased yield [<a href="#CR107" class="usa-link" aria-describedby="CR107">107</a>]. Recent studies have emphasized the need for safer, more efficient nanoparticle formulations. Innovations in nanoparticle design, including biocompatible materials, targeted delivery, and responsive systems, are paving the way for the broader application of CRISPR-Cas9 in clinical and agricultural settings [<a href="#CR117" class="usa-link" aria-describedby="CR117">117</a>, <a href="#CR118" class="usa-link" aria-describedby="CR118">118</a>].
</p>
<section class="tw xbox font-sm" id="Tab2"><h3 class="obj_head">Table 2.</h3>
<div class="caption p"><p>Integrated classification and comparative analysis of CRISPR-Cas system subtypes with respect to antimicrobial mechanisms, nanoparticle delivery compatibility, and translational potential</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Class</th>
<th align="left" colspan="1" rowspan="1">Type</th>
<th align="left" colspan="1" rowspan="1">Sub-type</th>
<th align="left" colspan="1" rowspan="1">Description</th>
<th align="left" colspan="1" rowspan="1">Target</th>
<th align="left" colspan="1" rowspan="1">Effector</th>
<th align="left" colspan="1" rowspan="1">CRISPR Protein</th>
<th align="left" colspan="1" rowspan="1">PAM/PFS</th>
<th align="left" colspan="1" rowspan="1">Associated type-subtype</th>
<th align="left" colspan="1" rowspan="1">Function</th>
<th align="left" colspan="1" rowspan="1">Mechanism of antimicrobial action</th>
<th align="left" colspan="1" rowspan="1">NP delivery compatibility</th>
<th align="left" colspan="1" rowspan="1">Translational readiness</th>
<th align="left" colspan="1" rowspan="1">Ethical/regulatory note</th>
<th align="left" colspan="1" rowspan="1">Application potential</th>
<th align="left" colspan="1" rowspan="1">Ref</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="19" colspan="1">Class 1</td>
<td align="left" rowspan="10" colspan="1">Type I</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" rowspan="10" colspan="1">Multi-subunit crRNA-effector complex</td>
<td align="left" rowspan="10" colspan="1">dsDNA</td>
<td align="left" colspan="1" rowspan="1">Cascade</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 1</p>
<p><strong>Cas 3</strong></p>
<p>Cas 6</p>
</td>
<td align="left" rowspan="19" colspan="1">-</td>
<td align="left" rowspan="10" colspan="1">I, II, III-A, III-B, IV, possibly VI</td>
<td align="left" rowspan="10" colspan="1">DNA Nuclease and helicase activity</td>
<td align="left" rowspan="10" colspan="1">Cas3 degrades DNA unidirectionally after Cascade complex binding; enables"genome shredding"of resistance plasmids</td>
<td align="left" rowspan="10" colspan="1">Limited due to large complex size (multi-protein); needs innovative nano-encapsulation</td>
<td align="left" rowspan="10" colspan="1">In vitro efficacy confirmed; no in vivo or clinical delivery reported</td>
<td align="left" rowspan="10" colspan="1">DNA destruction is irreversible; biosecurity and containment are concerns</td>
<td align="left" rowspan="10" colspan="1">Multi-subunit system for processive DNA degradation. Experimentally explored to remove resistance plasmids and disrupt essential bacterial genes in persistent infections. Potential use in anti-biofilm strategies via genome shredding</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>–<a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B</td>
<td align="left" colspan="1" rowspan="1">Cascade</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 1</p>
<p><strong>Cas 3</strong></p>
<p>Cas 6</p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>–<a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">C</td>
<td align="left" colspan="1" rowspan="1">Cascade</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 1</p>
<p><strong>Cas 3</strong></p>
<p>Cas 5</p>
<p>Cas 6</p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>–<a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">C variant</td>
<td align="left" colspan="1" rowspan="1">Cascade</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 5</p>
<p>Cas 6</p>
<p><strong>Cas3</strong></p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>–<a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D</td>
<td align="left" colspan="1" rowspan="1">Hybrid features of Type I and III</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 1</p>
<p>Cas 6</p>
<p><strong>Cas 3</strong></p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR121" class="usa-link" aria-describedby="CR121">121</a>–<a href="#CR123" class="usa-link" aria-describedby="CR123">123</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E</td>
<td align="left" colspan="1" rowspan="1">Cascade</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 1</p>
<p><strong>Cas 3</strong></p>
<p>Cas 6</p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR121" class="usa-link" aria-describedby="CR121">121</a>, <a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>, <a href="#CR124" class="usa-link" aria-describedby="CR124">124</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">F</td>
<td align="left" colspan="1" rowspan="1">Csy-complex</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 1</p>
<p><strong>Cas 3</strong></p>
<p>Cas 6</p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR121" class="usa-link" aria-describedby="CR121">121</a>, <a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">F variant 1</td>
<td align="left" colspan="1" rowspan="1">Csy-complex</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 1</p>
<p><strong>Cas 3</strong></p>
<p>Cas 6</p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR121" class="usa-link" aria-describedby="CR121">121</a>, <a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">F variant 2</td>
<td align="left" colspan="1" rowspan="1">Csy-complex</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 1</p>
<p><strong>Cas 3</strong></p>
<p>Cas 6</p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR121" class="usa-link" aria-describedby="CR121">121</a>, <a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">U</td>
<td align="left" colspan="1" rowspan="1">Cascade</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 1</p>
<p><strong>Cas 3</strong></p>
<p>Cas 6</p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR121" class="usa-link" aria-describedby="CR121">121</a>, <a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>]</td>
</tr>
<tr>
<td align="left" rowspan="6" colspan="1">Type III</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" rowspan="6" colspan="1">Multi-subunit crRNA-effector complex</td>
<td align="left" rowspan="6" colspan="1">ssRNA</td>
<td align="left" colspan="1" rowspan="1">Csm complex</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 2</p>
<p>Cas 6</p>
<p>Cas 8</p>
<p><strong>Cas 10</strong></p>
</td>
<td align="left" rowspan="6" colspan="1">I, II, III-A, III-B, V, some VI</td>
<td align="left" rowspan="6" colspan="1">
<p>RNA Nuclease</p>
<p>(Cleaving)</p>
</td>
<td align="left" rowspan="6" colspan="1">Dual targeting of DNA and RNA; suitable for adaptive multi-pathway suppression of resistance and virulence</td>
<td align="left" rowspan="6" colspan="1">Complex structure limits delivery; still not validated in NP systems</td>
<td align="left" rowspan="6" colspan="1">Experimental phase; theoretical antimicrobial utility</td>
<td align="left" rowspan="6" colspan="1">Dual-action raises complexity in predicting off-targets and safety</td>
<td align="left" rowspan="6" colspan="1">Dual-action targeting of DNA and RNA makes it promising for managing complex infections involving horizontal gene transfer or mobile elements. May be valuable for real-time bacterial control but requires further optimization for nanocarrier compatibility</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR120" class="usa-link" aria-describedby="CR120">120</a>, <a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>, <a href="#CR125" class="usa-link" aria-describedby="CR125">125</a>, <a href="#CR126" class="usa-link" aria-describedby="CR126">126</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A variant</td>
<td align="left" colspan="1" rowspan="1">Csm complex</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 2</p>
<p>Cas 6</p>
<p>Cas 8</p>
<p><strong>Cas 10</strong></p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR120" class="usa-link" aria-describedby="CR120">120</a>, <a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>, <a href="#CR126" class="usa-link" aria-describedby="CR126">126</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B</td>
<td align="left" colspan="1" rowspan="1">Cmr complex</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 2</p>
<p>Cas 6</p>
<p>Cas 8</p>
<p><strong>Cas 10</strong></p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR120" class="usa-link" aria-describedby="CR120">120</a>, <a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>, <a href="#CR126" class="usa-link" aria-describedby="CR126">126</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B variant</td>
<td align="left" colspan="1" rowspan="1">Cascade</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 2</p>
<p>Cas 6</p>
<p>Cas 8</p>
<p><strong>Cas 10</strong></p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR120" class="usa-link" aria-describedby="CR120">120</a>, <a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>, <a href="#CR126" class="usa-link" aria-describedby="CR126">126</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">C</td>
<td align="left" colspan="1" rowspan="1">Cascade</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 2</p>
<p>Cas 6</p>
<p>Cas 8</p>
<p><strong>Cas 10</strong></p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR120" class="usa-link" aria-describedby="CR120">120</a>, <a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>, <a href="#CR126" class="usa-link" aria-describedby="CR126">126</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D</td>
<td align="left" colspan="1" rowspan="1">Cascade</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 2</p>
<p>Cas 6</p>
<p>Cas 8</p>
<p><strong>Cas 10</strong></p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR120" class="usa-link" aria-describedby="CR120">120</a>, <a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>, <a href="#CR126" class="usa-link" aria-describedby="CR126">126</a>]</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">Type IV</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" rowspan="3" colspan="1">Multi-subunit crRNA-effector complex</td>
<td align="left" rowspan="3" colspan="1">dsDNA</td>
<td align="left" colspan="1" rowspan="1">Cascade</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 3</p>
<p><strong>Csf1</strong>(Cas8)</p>
</td>
<td align="left" rowspan="3" colspan="1">I</td>
<td align="left" rowspan="3" colspan="1">Unknown/putative interference role (under investigation)</td>
<td align="left" rowspan="3" colspan="1">Suspected to restrict plasmid transfer and horizontal gene resistance spread</td>
<td align="left" rowspan="3" colspan="1">Not studied yet; lack of effector characterization limits formulation</td>
<td align="left" rowspan="3" colspan="1">Conceptual only; no in vitro or in vivo validations</td>
<td align="left" rowspan="3" colspan="1">No known risks; field underexplored</td>
<td align="left" rowspan="3" colspan="1">Poorly characterized; possibly involved in plasmid interference and restriction of mobile genetic elements. Could be useful for removing resistance plasmids in the future. Clinical applications remain speculative due to lack of adaptation machinery</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR120" class="usa-link" aria-describedby="CR120">120</a>, <a href="#CR127" class="usa-link" aria-describedby="CR127">127</a>–<a href="#CR130" class="usa-link" aria-describedby="CR130">130</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A variant</td>
<td align="left" colspan="1" rowspan="1">Cascade</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 3</p>
<p><strong>Csf1</strong>(Cas8)</p>
</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B</td>
<td align="left" colspan="1" rowspan="1">Cascade</td>
<td align="left" colspan="1" rowspan="1">
<strong>Csf1</strong>(Cas8)</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR120" class="usa-link" aria-describedby="CR120">120</a>, <a href="#CR127" class="usa-link" aria-describedby="CR127">127</a>–<a href="#CR130" class="usa-link" aria-describedby="CR130">130</a>]</td>
</tr>
<tr>
<td align="left" rowspan="12" colspan="1">Class 2</td>
<td align="left" rowspan="4" colspan="1">Type II</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" rowspan="4" colspan="1">Single-subunit crRNA-effector module (Cas9)</td>
<td align="left" colspan="1" rowspan="1">dsDNA</td>
<td align="left" colspan="1" rowspan="1">SpCas9</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 5</p>
<p><strong>Cas 9</strong></p>
</td>
<td align="left" colspan="1" rowspan="1">NGG SpCas9</td>
<td align="left" colspan="1" rowspan="1">Type I, I</td>
<td align="left" colspan="1" rowspan="1">
<p>Ribonuclease responsible for converting pre-crRNA to mature crRNA</p>
<p>(Cleaving)</p>
</td>
<td align="left" rowspan="4" colspan="1">Creates dsDNA breaks in resistance genes (e.g., <em>mecA</em>, <em>blaNDM-1</em>), blocking replication or plasmid stability</td>
<td align="left" rowspan="4" colspan="1">Highly compatible with lipid-based NPs, polymeric NPs, and exosomes. Cas9 protein or mRNA can be co-packaged with sgRNA</td>
<td align="left" rowspan="4" colspan="1">Currently in preclinical and early-phase human trials for antimicrobial and genetic diseases</td>
<td align="left" rowspan="4" colspan="1">Risk of off-target editing and immunogenicity. Regulatory guidelines under development for microbial gene-editing therapies</td>
<td align="left" rowspan="4" colspan="1">Precise DNA editing enables knockout of resistance genes (e.g., <em>mecA</em>, <em>blaNDM-1</em>) and disruption of biofilm-associated loci (e.g., <em>icaA</em>). Widely used in antimicrobial gene targeting and restoration of antibiotic susceptibility in drug-resistant bacteria. Compatible with diverse NP carriers for targeted delivery</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR131" class="usa-link" aria-describedby="CR131">131</a>–<a href="#CR136" class="usa-link" aria-describedby="CR136">136</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A variant</td>
<td align="left" colspan="1" rowspan="1">dsDNA</td>
<td align="left" colspan="1" rowspan="1">SaCas9</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 5</p>
<p><strong>Cas 9</strong></p>
</td>
<td align="left" colspan="1" rowspan="1">NNGRRT SaCas9</td>
<td align="left" colspan="1" rowspan="1">Type I, I</td>
<td align="left" colspan="1" rowspan="1">
<p>Ribonuclease responsible for converting pre-crRNA to mature crRNA</p>
<p>(Cleaving)</p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR131" class="usa-link" aria-describedby="CR131">131</a>–<a href="#CR136" class="usa-link" aria-describedby="CR136">136</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B</td>
<td align="left" colspan="1" rowspan="1">dsDNA/ssRNA</td>
<td align="left" colspan="1" rowspan="1">FnCas9</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 6</p>
<p><strong>Cas 9</strong></p>
</td>
<td align="left" colspan="1" rowspan="1">NGG FnCas9</td>
<td align="left" colspan="1" rowspan="1">Most type IIIB and type I</td>
<td align="left" colspan="1" rowspan="1">
<p>Pre-crRNA is converted to mature crRNA by ribonuclease</p>
<p>(Cleaving)</p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR131" class="usa-link" aria-describedby="CR131">131</a>–<a href="#CR135" class="usa-link" aria-describedby="CR135">135</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">C</td>
<td align="left" colspan="1" rowspan="1">dsDNA</td>
<td align="left" colspan="1" rowspan="1">NmCas9</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 7</p>
<p><strong>Cas 9</strong></p>
</td>
<td align="left" colspan="1" rowspan="1">NNNNGATT NmCas9</td>
<td align="left" colspan="1" rowspan="1">I, III, IV</td>
<td align="left" colspan="1" rowspan="1">
<p>It binds with crRNA and comprises of an RNA recognition motif</p>
<p>(Cleaving)</p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>, <a href="#CR132" class="usa-link" aria-describedby="CR132">132</a>–<a href="#CR135" class="usa-link" aria-describedby="CR135">135</a>]</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">Type V</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" rowspan="3" colspan="1">Single-subunit crRNA-effector module (Cas12)</td>
<td align="left" rowspan="3" colspan="1">dsDNA</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas12a (Cpf1)</p>
<p>RuvC + Nuc domain</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 8</p>
<p>Cas 12</p>
<p>Cpf1</p>
</td>
<td align="left" colspan="1" rowspan="1">50 AT-rich PAM TTTV</td>
<td align="left" colspan="1" rowspan="1">Most type I</td>
<td align="left" colspan="1" rowspan="1">
<p>It forms effector complex large subunit in type I</p>
<p>(Cleaving with cpf1)</p>
</td>
<td align="left" rowspan="3" colspan="1">Cuts target DNA and triggers collateral ssDNA cleavage, enabling broader bacterial genome targeting and biofilm matrix degradation</td>
<td align="left" rowspan="3" colspan="1">Small size and self-processing capability make it ideal for lipid and polymeric NP encapsulation</td>
<td align="left" rowspan="3" colspan="1">Used in diagnostics (DETECTR), preclinical therapeutic designs underway</td>
<td align="left" rowspan="3" colspan="1">Lower immunogenicity; collateral activity may affect safety. Monitoring required in clinical translation</td>
<td align="left" rowspan="3" colspan="1">Offers gene editing with collateral cleavage potential, allowing both gene disruption and DNA degradation. Its self-processing ability simplifies NP delivery design. Applied in biofilm removal, bacterial genome disruption, and CRISPR-based diagnostics</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR132" class="usa-link" aria-describedby="CR132">132</a>, <a href="#CR137" class="usa-link" aria-describedby="CR137">137</a>–<a href="#CR142" class="usa-link" aria-describedby="CR142">142</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas12b (C2c1)</p>
<p>RuvC</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 9</p>
<p>Cas 12</p>
<p>Cpf1</p>
</td>
<td align="left" colspan="1" rowspan="1">50 AT-rich PAM TTTV</td>
<td align="left" colspan="1" rowspan="1">II only</td>
<td align="left" colspan="1" rowspan="1">
<p>DNA nuclease</p>
<p>(Cleaving with cpf1)</p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR132" class="usa-link" aria-describedby="CR132">132</a>, <a href="#CR137" class="usa-link" aria-describedby="CR137">137</a>–<a href="#CR143" class="usa-link" aria-describedby="CR143">143</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">C</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas12c (C2c3)</p>
<p>RuvC</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 10</p>
<p>Cas 12</p>
<p>Cpf1</p>
</td>
<td align="left" colspan="1" rowspan="1">50 AT-rich PAM TTTV</td>
<td align="left" colspan="1" rowspan="1">Some type I, most type III</td>
<td align="left" colspan="1" rowspan="1">
<p>It forms effector complex large subunit in type III</p>
<p>(Cleaving with cpf1)</p>
</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR132" class="usa-link" aria-describedby="CR132">132</a>, <a href="#CR137" class="usa-link" aria-describedby="CR137">137</a>–<a href="#CR143" class="usa-link" aria-describedby="CR143">143</a>]</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">Type VI</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" rowspan="5" colspan="1">Single-subunit crRNA-effector module (Cas13)</td>
<td align="left" rowspan="5" colspan="1">ssRNA</td>
<td align="left" colspan="1" rowspan="1">Cas13a (C2c2)</td>
<td align="left" colspan="1" rowspan="1">
<p>Cas 12 (cpf1)</p>
<p>Cas 13</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>30</p>
<p>PFS: non-G</p>
</td>
<td align="left" colspan="1" rowspan="1">V</td>
<td align="left" colspan="1" rowspan="1">crRNA sorting, DNA nuclease</td>
<td align="left" rowspan="5" colspan="1">Degrades target RNA (e.g., resistance or virulence mRNAs), blocking protein synthesis; collateral RNA degradation in pathogens</td>
<td align="left" rowspan="5" colspan="1">Ideal for transient therapies; loaded as RNP or mRNA into lipid/polymer/hydrogel NPs</td>
<td align="left" rowspan="5" colspan="1">SHERLOCK-based diagnostics approved under EUA by FDA; therapeutic uses under study</td>
<td align="left" rowspan="5" colspan="1">Considered safer than Cas9 due to RNA-only targeting; potential for off-pathway RNA collateral cleavage</td>
<td align="left" rowspan="5" colspan="1">Targets bacterial or viral RNA, allowing suppression of gene expression without altering DNA. Promising for transient gene control in pathogens and CRISPR-based antimicrobials or diagnostics (e.g., SHERLOCK). Suitable for resistant infections involving RNA regulation</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR132" class="usa-link" aria-describedby="CR132">132</a>, <a href="#CR144" class="usa-link" aria-describedby="CR144">144</a>–<a href="#CR149" class="usa-link" aria-describedby="CR149">149</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B-1</td>
<td align="left" colspan="1" rowspan="1">Cas13b (C2c4)</td>
<td align="left" colspan="1" rowspan="1">Cas 13 (C2c2)</td>
<td align="left" colspan="1" rowspan="1">
<p>50</p>
<p>PFS: non-C; 30</p>
<p>PFS:NAN/NNA</p>
</td>
<td align="left" colspan="1" rowspan="1">VI</td>
<td align="left" colspan="1" rowspan="1">crRNA sorting, RNA nuclease</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR145" class="usa-link" aria-describedby="CR145">145</a>–<a href="#CR151" class="usa-link" aria-describedby="CR151">151</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B-2</td>
<td align="left" colspan="1" rowspan="1">Cas13b (C2c4)</td>
<td align="left" colspan="1" rowspan="1">Cas 13 (C2c2)</td>
<td align="left" colspan="1" rowspan="1">
<p>50</p>
<p>PFS: non-C; 30</p>
<p>PFS:NAN/NNA</p>
</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">crRNA sorting, RNA nuclease</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR146" class="usa-link" aria-describedby="CR146">146</a>–<a href="#CR151" class="usa-link" aria-describedby="CR151">151</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">C</td>
<td align="left" colspan="1" rowspan="1">Cas13c (C2c7)</td>
<td align="left" colspan="1" rowspan="1">
<p>Csm, Cmr</p>
<p>Cas 13</p>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">III</td>
<td align="left" colspan="1" rowspan="1">Nucleases for single-stranded DNA and RNA</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR144" class="usa-link" aria-describedby="CR144">144</a>, <a href="#CR146" class="usa-link" aria-describedby="CR146">146</a>–<a href="#CR150" class="usa-link" aria-describedby="CR150">150</a>, <a href="#CR152" class="usa-link" aria-describedby="CR152">152</a>, <a href="#CR153" class="usa-link" aria-describedby="CR153">153</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D</td>
<td align="left" colspan="1" rowspan="1">Cas13d</td>
<td align="left" colspan="1" rowspan="1">
<p>RNase III</p>
<p>Cas 13</p>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">II</td>
<td align="left" colspan="1" rowspan="1">tracrRNA is processed, and crRNA maturation is aided by this system</td>
<td align="left" colspan="1" rowspan="1"> [<a href="#CR146" class="usa-link" aria-describedby="CR146">146</a>–<a href="#CR150" class="usa-link" aria-describedby="CR150">150</a>, <a href="#CR154" class="usa-link" aria-describedby="CR154">154</a>]</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec27"><h2 class="pmc_sec_title">Future directions and challenges</h2>
<p id="Par42">Despite its potential, CRISPR-based anti-biofilm strategies face several hurdles. The delivery of CRISPR components into biofilm-embedded bacteria remains a major challenge, necessitating the use of bacteriophages, nanoparticles, or electroporation techniques. Moreover, the risk of off-target effects and bacterial evolution against CRISPR-based attacks requires further investigation. Nevertheless, the development of CRISPR-Cas antimicrobials represents a promising frontier in tackling biofilm-associated bacterial resistance, with potential applications in clinical and industrial settings [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>, <a href="#CR108" class="usa-link" aria-describedby="CR108">108</a>]. The application of CRISPR/Cas9 in biofilm control presents several anticipated opportunities, novel hypotheses, and inherent challenges, particularly when focusing on molecular mechanisms and biofilm-related genes. CRISPR/Cas9 allows sequence-specific targeting of genes essential for biofilm formation, maintenance, and dispersal. One of the major strategies involves disrupting quorum sensing (QS) pathways, which regulate biofilm development [<a href="#CR155" class="usa-link" aria-describedby="CR155">155</a>]. The <em>lasR</em> and <em>rhlR</em> genes in <em>Pseudomonas aeruginosa</em>, encoding transcriptional regulators of QS, are prime targets for CRISPR/Cas9-mediated disruption, effectively reducing biofilm formation [<a href="#CR156" class="usa-link" aria-describedby="CR156">156</a>]. Similarly, targeting <em>icaA</em>, a critical gene in polysaccharide intercellular adhesin (PIA) synthesis in <em>Staphylococcus aureus</em>, significantly weakens biofilm integrity [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Another promising approach is the utilization of CRISPR/Cas9-based antimicrobials to selectively eliminate antibiotic-resistant bacteria within biofilms. By designing guide RNAs (gRNAs) to target antibiotic resistance genes such as <em>bla</em>, <em>mecA</em>, and <em>ndm-1</em>, researchers have successfully sensitized biofilm-associated bacteria to conventional antibiotics. Furthermore, biofilm dispersal can be promoted by targeting regulators of cyclic-di-GMP signaling, such as <em>pelA</em> and <em>bssR</em>, which control biofilm stability in <em>Escherichia coli</em> and <em>Pseudomonas aeruginosa</em> [<a href="#CR157" class="usa-link" aria-describedby="CR157">157</a>]. One major hurdle is the efficient delivery of CRISPR components into biofilm-embedded bacteria. The dense EPS matrix acts as a barrier, reducing the penetration of CRISPR/Cas9 systems delivered via plasmids, liposomes, or nanoparticles. Bacteriophage-mediated delivery shows potential, yet phage-host specificity limits its broad applicability. Another challenge is the emergence of CRISPR resistance in bacterial populations. Bacteria may develop mutations in the protospacer adjacent motif (PAM) sequence, preventing Cas9 recognition. Additionally, some bacterial species possess anti-CRISPR proteins (Acrs) that inhibit Cas9 activity, reducing its effectiveness. Strategies to overcome this limitation include using CRISPR/Cas variations such as Cas12a or Cas13, which have distinct PAM requirements and broader targeting potential. Furthermore, the unintended ecological consequences of biofilm-targeting CRISPR approaches require careful consideration. Biofilms play essential roles in environmental and industrial settings, including wastewater treatment and bioremediation. Uncontrolled disruption of biofilms could negatively impact these beneficial processes [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>, <a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>, <a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>, <a href="#CR157" class="usa-link" aria-describedby="CR157">157</a>]. Therefore, the development of inducible CRISPR systems, where biofilm targeting is activated only under specific conditions, is a potential solution [<a href="#CR108" class="usa-link" aria-describedby="CR108">108</a>]. These resistance mechanisms may limit the long-term effectiveness of CRISPR-based biofilm therapies. To overcome these challenges, researchers are exploring alternative Cas9 variants, such as Cpf1 (also known as Cas12a), which recognize different PAM sequences and may bypass some of the resistance mechanisms. Moreover, concerns about off-target effects, where CRISPR/Cas9 inadvertently edits unintended genomic loci, must be carefully considered, particularly in clinical or environmental applications. Advances in high-fidelity Cas9 variants and improved gRNA design strategies are being pursued to minimize these risks. This approach allows for more precise regulation of gene expression, offering a reversible means of controlling biofilm formation without the risk of inducing permanent genetic changes [<a href="#CR158" class="usa-link" aria-describedby="CR158">158</a>]. CRISPRi (CRISPR interference) has already been applied to modulate biofilm-related genes in <em>Pseudomonas aeruginosa</em>, providing a potential alternative to gene knockouts [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. The combination of CRISPR/Cas9 with synthetic biology tools, such as inducible gene circuits and biosensors, could further enhance the specificity and control of biofilm-targeting strategies. By engineering bacteria to respond to environmental cues or to self-regulate CRISPR activity, it may be possible to create dynamic systems that can selectively disrupt biofilms under specific conditions, reducing the risk of unwanted side effects [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>, <a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>].</p></section><section id="Sec28"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par43">Bacterial biofilms represent a major challenge in healthcare and industry due to their enhanced resistance to antibiotics. The biofilm structure, characterized by an extracellular polymeric substance (EPS) matrix, provides mechanical protection and restricts antibiotic penetration. Furthermore, the ultrastructural organization of biofilms contributes to their persistence, metabolic heterogeneity, and intercellular communication through quorum sensing. The resistance mechanisms of bacterial biofilms are multifaceted and collectively render conventional antibiotic therapies less effective, necessitating novel approaches to combat biofilm-associated infections. One promising avenue is the CRISPR-Cas system, a prokaryotic adaptive immune mechanism that can be repurposed for precision genome editing. Specifically, CRISPR-Cas9, a widely studied variant, functions by introducing targeted double-stranded DNA breaks, facilitating genetic modifications. In the context of bacterial biofilms and antibiotic resistance, the CRISPR-Cas system offers potential solutions by selectively targeting resistance genes, biofilm-regulating pathways, and virulence factors. However, several hypotheses and challenges must be addressed, such as efficient delivery methods, off-target effects, bacterial immune evasion, and regulatory considerations. The interplay between CRISPR-Cas systems and antibiotic resistance is complex, as some bacterial strains possess endogenous CRISPR elements that can mediate resistance gene uptake or restriction, further complicating therapeutic strategies. Nanotechnology presents an innovative tool to enhance antibiotic therapy. Nanoparticles (NPs) exhibit intrinsic antibacterial properties and serve as efficient carriers for antibiotics or gene-editing tools. They can improve drug solubility, bioavailability, and targeted delivery, reducing off-target toxicity and overcoming bacterial resistance mechanisms. Nanoparticles in CRISPR-Cas9 delivery are particularly promising, offering enhanced protection of CRISPR components, controlled release, and improved cellular uptake within biofilms. The integration of CRISPR-Cas9 and nanotechnology holds immense potential for eradicating biofilm-related infections, but several hurdles remain, including optimizing nanoparticle formulations, ensuring biocompatibility, and scaling up production for clinical applications. Future research should focus on refining CRISPR-nanoparticle hybrid systems, assessing their long-term effects, and developing regulatory frameworks for safe and effective deployment. Overall, a multidisciplinary approach combining molecular genetics, nanotechnology, and bioengineering is essential to overcome biofilm-associated antibiotic resistance and revolutionize bacterial infection treatments.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We would like to thank all our colleagues for their assistance. The facility for this study was provided by Research Deputy of Kashan University of Medical Sciences.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>CRISPR</dt>
<dd><p id="Par3">Clustered Regularly Interspaced Short Palindromic Repeats</p></dd>
<dt>Cas9</dt>
<dd><p id="Par4">CRISPR-associated protein 9</p></dd>
<dt>NPs</dt>
<dd><p id="Par5">Nanoparticles</p></dd>
<dt>PAM</dt>
<dd><p id="Par6">Protospacer adjacent motif</p></dd>
<dt>gRNA</dt>
<dd><p id="Par7">Guide RNA</p></dd>
<dt>HDR</dt>
<dd><p id="Par8">Homology-directed repair</p></dd>
<dt>NHEJ</dt>
<dd><p id="Par9">Non-homologous end joining</p></dd>
<dt>EPS</dt>
<dd><p id="Par10">Extracellular polymeric substances</p></dd>
<dt>PLGA</dt>
<dd><p id="Par11">Poly (lactic-co-glycolic acid)</p></dd>
<dt>DNA</dt>
<dd><p id="Par12">Deoxyribonucleic acid</p></dd>
<dt>RNA</dt>
<dd><p id="Par13">Ribonucleic acid</p></dd>
<dt>gDNA</dt>
<dd><p id="Par14">Genomic DNA</p></dd>
<dt>qPCR</dt>
<dd><p id="Par15">Quantitative polymerase chain reaction</p></dd>
<dt>PCR</dt>
<dd><p id="Par16">Polymerase chain reaction</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>MP, EK, ASN, AKH, HHK conducted the literature and drafted the manuscript. All authors reviewed the manuscript and made significant revisions and approved the final version of the manuscript draft.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>No financial support was received.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>No datasets were generated or analysed during the current study.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par44">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par45">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par46">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Flemming H-C, Wingender J, Szewzyk U, Steinberg P, Rice SA, Kjelleberg S. Biofilms: an emergent form of bacterial life. Nat Rev Microbiol. 2016;14(9):563–75.
</cite> [<a href="https://doi.org/10.1038/nrmicro.2016.94" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27510863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Flemming%20H-C,%20Wingender%20J,%20Szewzyk%20U,%20Steinberg%20P,%20Rice%20SA,%20Kjelleberg%20S.%20Biofilms:%20an%20emergent%20form%20of%20bacterial%20life.%20Nat%20Rev%20Microbiol.%202016;14(9):563%E2%80%9375." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Zuberi A, Ahmad N, Ahmad H, Saeed M, Ahmad I. Beyond antibiotics: CRISPR/Cas9 triumph over biofilm-associated antibiotic resistance infections. Front Cell Infect Microbiol. 2024;14:1408569.
</cite> [<a href="https://doi.org/10.3389/fcimb.2024.1408569" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11257871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39035353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zuberi%20A,%20Ahmad%20N,%20Ahmad%20H,%20Saeed%20M,%20Ahmad%20I.%20Beyond%20antibiotics:%20CRISPR/Cas9%20triumph%20over%20biofilm-associated%20antibiotic%20resistance%20infections.%20Front%20Cell%20Infect%20Microbiol.%202024;14:1408569." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Ciofu O, Tolker-Nielsen T. Tolerance and resistance of <em>Pseudomonas aeruginosa</em> biofilms to antimicrobial agents-how <em>P. aeruginosa</em> can escape antibiotics. Front Microbiol. 2019;10:913.
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.00913" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6509751/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31130925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ciofu%20O,%20Tolker-Nielsen%20T.%20Tolerance%20and%20resistance%20of%20Pseudomonas%20aeruginosa%20biofilms%20to%20antimicrobial%20agents-how%20P.%20aeruginosa%20can%20escape%20antibiotics.%20Front%20Microbiol.%202019;10:913." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Jinek M, Chylinski K, Fonfara I, Hauer M, Doudna JA, Charpentier E. A programmable dual-RNA–guided DNA endonuclease in adaptive bacterial immunity. Science. 2012;337(6096):816–21.
</cite> [<a href="https://doi.org/10.1126/science.1225829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6286148/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22745249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jinek%20M,%20Chylinski%20K,%20Fonfara%20I,%20Hauer%20M,%20Doudna%20JA,%20Charpentier%20E.%20A%20programmable%20dual-RNA%E2%80%93guided%20DNA%20endonuclease%20in%20adaptive%20bacterial%20immunity.%20Science.%202012;337(6096):816%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Ekwebelem OC, Aleke J, Ofielu E, Nnorom-Dike O. CRISPR-Cas9 system: a revolutionary tool in the fight against antimicrobial resistance: retracted. Infect Microbes Dis. 2021;3(2):51–6.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ekwebelem%20OC,%20Aleke%20J,%20Ofielu%20E,%20Nnorom-Dike%20O.%20CRISPR-Cas9%20system:%20a%20revolutionary%20tool%20in%20the%20fight%20against%20antimicrobial%20resistance:%20retracted.%20Infect%20Microbes%20Dis.%202021;3(2):51%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Fletcher RB, Stokes LD, Kelly IB 3rd, Henderson KM, Vallecillo-Viejo IC, Colazo JM, et al. Nonviral in vivo delivery of CRISPR-Cas9 using protein-agnostic, high-loading porous silicon and polymer nanoparticles. ACS Nano. 2023;17(17):16412–31.
</cite> [<a href="https://doi.org/10.1021/acsnano.2c12261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11129837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37582231/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fletcher%20RB,%20Stokes%20LD,%20Kelly%20IB%203rd,%20Henderson%20KM,%20Vallecillo-Viejo%20IC,%20Colazo%20JM,%20et%20al.%20Nonviral%20in%20vivo%20delivery%20of%20CRISPR-Cas9%20using%20protein-agnostic,%20high-loading%20porous%20silicon%20and%20polymer%20nanoparticles.%20ACS%20Nano.%202023;17(17):16412%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Wan F, Draz MS, Gu M, Yu W, Ruan Z, Luo Q. Novel strategy to combat antibiotic resistance: a sight into the combination of CRISPR/Cas9 and nanoparticles. Pharmaceutics. 2021;13(3):352.
</cite> [<a href="https://doi.org/10.3390/pharmaceutics13030352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7998274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33800235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wan%20F,%20Draz%20MS,%20Gu%20M,%20Yu%20W,%20Ruan%20Z,%20Luo%20Q.%20Novel%20strategy%20to%20combat%20antibiotic%20resistance:%20a%20sight%20into%20the%20combination%20of%20CRISPR/Cas9%20and%20nanoparticles.%20Pharmaceutics.%202021;13(3):352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Zohra T, Numan M, Ikram A, Salman M, Khan T, Din M, et al. Cracking the challenge of antimicrobial drug resistance with CRISPR/Cas9, nanotechnology and other strategies in ESKAPE pathogens. Microorganisms. 2021;9(5):954.
</cite> [<a href="https://doi.org/10.3390/microorganisms9050954" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8145940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33946643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zohra%20T,%20Numan%20M,%20Ikram%20A,%20Salman%20M,%20Khan%20T,%20Din%20M,%20et%20al.%20Cracking%20the%20challenge%20of%20antimicrobial%20drug%20resistance%20with%20CRISPR/Cas9,%20nanotechnology%20and%20other%20strategies%20in%20ESKAPE%20pathogens.%20Microorganisms.%202021;9(5):954." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Kim M, Hwang Y, Lim S, Jang H-K, Kim H-O. Advances in nanoparticles as non-viral vectors for efficient delivery of CRISPR/Cas9. Pharmaceutics. 2024;16(9):1197.
</cite> [<a href="https://doi.org/10.3390/pharmaceutics16091197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11434874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39339233/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20M,%20Hwang%20Y,%20Lim%20S,%20Jang%20H-K,%20Kim%20H-O.%20Advances%20in%20nanoparticles%20as%20non-viral%20vectors%20for%20efficient%20delivery%20of%20CRISPR/Cas9.%20Pharmaceutics.%202024;16(9):1197." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Gold K, Slay B, Knackstedt M, Gaharwar AK. Antimicrobial activity of metal and metal-oxide based nanoparticles. Adv Ther. 2018;1(3):1700033.</cite> [<a href="https://scholar.google.com/scholar_lookup?Gold%20K,%20Slay%20B,%20Knackstedt%20M,%20Gaharwar%20AK.%20Antimicrobial%20activity%20of%20metal%20and%20metal-oxide%20based%20nanoparticles.%20Adv%20Ther.%202018;1(3):1700033." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Jiang Y, Wu R, Zhang W, Xin F, Jiang M. Construction of stable microbial consortia for effective biochemical synthesis. Trends Biotechnol. 2023;41(11):1430–41.
</cite> [<a href="https://doi.org/10.1016/j.tibtech.2023.05.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37330325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20Y,%20Wu%20R,%20Zhang%20W,%20Xin%20F,%20Jiang%20M.%20Construction%20of%20stable%20microbial%20consortia%20for%20effective%20biochemical%20synthesis.%20Trends%20Biotechnol.%202023;41(11):1430%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Bush K, Bradford PA. β-lactams and β-lactamase inhibitors: an overview. Cold Spring Harb Perspect Med. 2016;6(8):a025247.
</cite> [<a href="https://doi.org/10.1101/cshperspect.a025247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4968164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27329032/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bush%20K,%20Bradford%20PA.%20%CE%B2-lactams%20and%20%CE%B2-lactamase%20inhibitors:%20an%20overview.%20Cold%20Spring%20Harb%20Perspect%20Med.%202016;6(8):a025247." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Davies J, Davies D. Origins and evolution of antibiotic resistance. Microbiol Mol Biol Rev. 2010;74(3):417–33.
</cite> [<a href="https://doi.org/10.1128/MMBR.00016-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2937522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20805405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Davies%20J,%20Davies%20D.%20Origins%20and%20evolution%20of%20antibiotic%20resistance.%20Microbiol%20Mol%20Biol%20Rev.%202010;74(3):417%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Hay SI, Rao PC, Dolecek C, Day NP, Stergachis A, Lopez AD, Murray CJ: Measuring and mapping ‎the global burden of antimicrobial resistance. BMC medicine 2018, 16(1):78.‎
</cite> [<a href="https://doi.org/10.1186/s12916-018-1073-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5985573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29860943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hay%20SI,%20Rao%20PC,%20Dolecek%20C,%20Day%20NP,%20Stergachis%20A,%20Lopez%20AD,%20Murray%20CJ:%20Measuring%20and%20mapping%20%E2%80%8Ethe%20global%20burden%20of%20antimicrobial%20resistance.%20BMC%20medicine%202018,%2016(1):78.%E2%80%8E" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Delcour AH. Outer membrane permeability and antibiotic resistance. Biochim Biophys Acta. 2009;1794(5):808–16.
</cite> [<a href="https://doi.org/10.1016/j.bbapap.2008.11.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2696358/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19100346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Delcour%20AH.%20Outer%20membrane%20permeability%20and%20antibiotic%20resistance.%20Biochim%20Biophys%20Acta.%202009;1794(5):808%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Laxminarayan R, Duse A, Wattal C, Zaidi AK, Wertheim HF, Sumpradit N, et al. Antibiotic resistance—the need for global solutions. Lancet Infect Dis. 2013;13(12):1057–98.
</cite> [<a href="https://doi.org/10.1016/S1473-3099(13)70318-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24252483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laxminarayan%20R,%20Duse%20A,%20Wattal%20C,%20Zaidi%20AK,%20Wertheim%20HF,%20Sumpradit%20N,%20et%20al.%20Antibiotic%20resistance%E2%80%94the%20need%20for%20global%20solutions.%20Lancet%20Infect%20Dis.%202013;13(12):1057%E2%80%9398." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>van Belkum A, Soriaga LB, LaFave MC, Akella S, Veyrieras J-B, Barbu EM, et al. Phylogenetic distribution of CRISPR-Cas systems in antibiotic-resistant Pseudomonas aeruginosa. MBio. 2015;6(6). 10.1128/mBio.01796-15.</cite> [<a href="https://doi.org/10.1128/mBio.01796-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4669384/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26604259/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Allegranzi B, Gayet-Ageron A, Damani N, Bengaly L, McLaws M-L, Moro M-L, et al. Global implementation of WHO’s multimodal strategy for improvement of hand hygiene: a quasi-experimental study. Lancet Infect Dis. 2013;13(10):843–51.
</cite> [<a href="https://doi.org/10.1016/S1473-3099(13)70163-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23972825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Allegranzi%20B,%20Gayet-Ageron%20A,%20Damani%20N,%20Bengaly%20L,%20McLaws%20M-L,%20Moro%20M-L,%20et%20al.%20Global%20implementation%20of%20WHO%E2%80%99s%20multimodal%20strategy%20for%20improvement%20of%20hand%20hygiene:%20a%20quasi-experimental%20study.%20Lancet%20Infect%20Dis.%202013;13(10):843%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Tängdén T, Giske C. Global dissemination of extensively drug-resistant carbapenemase-producing <em>E</em> nterobacteriaceae: clinical perspectives on detection, treatment and infection control. J Intern Med. 2015;277(5):501–12.
</cite> [<a href="https://doi.org/10.1111/joim.12342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25556628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?T%C3%A4ngd%C3%A9n%20T,%20Giske%20C.%20Global%20dissemination%20of%20extensively%20drug-resistant%20carbapenemase-producing%20E%20nterobacteriaceae:%20clinical%20perspectives%20on%20detection,%20treatment%20and%20infection%20control.%20J%20Intern%20Med.%202015;277(5):501%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Laxminarayan R, Impalli I, Rangarajan R, Cohn J, Ramjeet K, Trainor BW, Strathdee S, Sumpradit N, ‎Berman D, Wertheim H: Expanding antibiotic, vaccine, and diagnostics development and access ‎to tackle antimicrobial resistance. The Lancet 2024;403(10443):2534–2550.‎</cite> [<a href="https://doi.org/10.1016/S0140-6736(24)00878-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38797178/" class="usa-link">PubMed</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Spellberg B, Guidos R, Gilbert D, Bradley J, Boucher HW, Scheld WM, et al. The epidemic of antibiotic-resistant infections: a call to action for the medical community from the Infectious Diseases Society of America. Clin Infect Dis. 2008;46(2):155–64.
</cite> [<a href="https://doi.org/10.1086/524891" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18171244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spellberg%20B,%20Guidos%20R,%20Gilbert%20D,%20Bradley%20J,%20Boucher%20HW,%20Scheld%20WM,%20et%20al.%20The%20epidemic%20of%20antibiotic-resistant%20infections:%20a%20call%20to%20action%20for%20the%20medical%20community%20from%20the%20Infectious%20Diseases%20Society%20of%20America.%20Clin%20Infect%20Dis.%202008;46(2):155%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Dyar O, Pagani L, Pulcini C. Strategies and challenges of antimicrobial stewardship in long-term care facilities. Clin Microbiol Infect. 2015;21(1):10–9.
</cite> [<a href="https://doi.org/10.1016/j.cmi.2014.09.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25636921/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dyar%20O,%20Pagani%20L,%20Pulcini%20C.%20Strategies%20and%20challenges%20of%20antimicrobial%20stewardship%20in%20long-term%20care%20facilities.%20Clin%20Microbiol%20Infect.%202015;21(1):10%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Coates AR, Halls G, Hu Y. Novel classes of antibiotics or more of the same? Br J Pharmacol. 2011;163(1):184–94.
</cite> [<a href="https://doi.org/10.1111/j.1476-5381.2011.01250.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3085877/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21323894/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coates%20AR,%20Halls%20G,%20Hu%20Y.%20Novel%20classes%20of%20antibiotics%20or%20more%20of%20the%20same?%20Br%20J%20Pharmacol.%202011;163(1):184%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Pandey P, Sirisha VL. From gene editing to biofilm busting: CRISPR-CAS9 against antibiotic resistance—a review. Cell Biochem Biophys. 2024;82:1–12.
</cite> [<a href="https://doi.org/10.1007/s12013-024-01276-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38702575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pandey%20P,%20Sirisha%20VL.%20From%20gene%20editing%20to%20biofilm%20busting:%20CRISPR-CAS9%20against%20antibiotic%20resistance%E2%80%94a%20review.%20Cell%20Biochem%20Biophys.%202024;82:1%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Juszczuk-Kubiak E. Molecular aspects of the functioning of pathogenic bacteria biofilm based on quorum sensing (QS) signal-response system and innovative non-antibiotic strategies for their elimination. Int J Mol Sci. 2024;25(5):2655.
</cite> [<a href="https://doi.org/10.3390/ijms25052655" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10931677/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38473900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Juszczuk-Kubiak%20E.%20Molecular%20aspects%20of%20the%20functioning%20of%20pathogenic%20bacteria%20biofilm%20based%20on%20quorum%20sensing%20(QS)%20signal-response%20system%20and%20innovative%20non-antibiotic%20strategies%20for%20their%20elimination.%20Int%20J%20Mol%20Sci.%202024;25(5):2655." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Saharan B, Beniwal N, Duhan J. From formulation to function: a detailed review of microbial biofilms and their polymer-based extracellular substances. The Microbe. 2024;5:100194.</cite> [<a href="https://scholar.google.com/scholar_lookup?Saharan%20B,%20Beniwal%20N,%20Duhan%20J.%20From%20formulation%20to%20function:%20a%20detailed%20review%20of%20microbial%20biofilms%20and%20their%20polymer-based%20extracellular%20substances.%20The%20Microbe.%202024;5:100194." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Costerton JW, Stewart PS, Greenberg EP. Bacterial biofilms: a common cause of persistent infections. Science. 1999;284(5418):1318–22.
</cite> [<a href="https://doi.org/10.1126/science.284.5418.1318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10334980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Costerton%20JW,%20Stewart%20PS,%20Greenberg%20EP.%20Bacterial%20biofilms:%20a%20common%20cause%20of%20persistent%20infections.%20Science.%201999;284(5418):1318%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Nichols W. Biofilms, antibiotics and penetration. Rev Med Microbiol. 1991;2:177–81.</cite> [<a href="https://scholar.google.com/scholar_lookup?Nichols%20W.%20Biofilms,%20antibiotics%20and%20penetration.%20Rev%20Med%20Microbiol.%201991;2:177%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Xie Y, Liu H, Teng Z, Ma J, Liu G. Nanomaterial-enabled anti-biofilm strategies: new opportunities for treatment of bacterial infections. Nanoscale. 2025;17(10):5605–28.
</cite> [<a href="https://doi.org/10.1039/d4nr04774e" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39918303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie%20Y,%20Liu%20H,%20Teng%20Z,%20Ma%20J,%20Liu%20G.%20Nanomaterial-enabled%20anti-biofilm%20strategies:%20new%20opportunities%20for%20treatment%20of%20bacterial%20infections.%20Nanoscale.%202025;17(10):5605%E2%80%9328." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Domenech M, García E, Moscoso M. Biofilm formation in <em>Streptococcus pneumoniae</em>. Microb Biotechnol. 2012;5(4):455–65.
</cite> [<a href="https://doi.org/10.1111/j.1751-7915.2011.00294.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3815323/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21906265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Domenech%20M,%20Garc%C3%ADa%20E,%20Moscoso%20M.%20Biofilm%20formation%20in%20Streptococcus%20pneumoniae.%20Microb%20Biotechnol.%202012;5(4):455%E2%80%9365." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Balcázar JL, Subirats J, Borrego CM. The role of biofilms as environmental reservoirs of antibiotic resistance. Front Microbiol. 2015;6:1216.
</cite> [<a href="https://doi.org/10.3389/fmicb.2015.01216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4628128/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26583011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Balc%C3%A1zar%20JL,%20Subirats%20J,%20Borrego%20CM.%20The%20role%20of%20biofilms%20as%20environmental%20reservoirs%20of%20antibiotic%20resistance.%20Front%20Microbiol.%202015;6:1216." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Keren I, Kaldalu N, Spoering A, Wang Y, Lewis K. Persister cells and tolerance to antimicrobials. FEMS Microbiol Lett. 2004;230(1):13–8.
</cite> [<a href="https://doi.org/10.1016/S0378-1097(03)00856-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14734160/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keren%20I,%20Kaldalu%20N,%20Spoering%20A,%20Wang%20Y,%20Lewis%20K.%20Persister%20cells%20and%20tolerance%20to%20antimicrobials.%20FEMS%20Microbiol%20Lett.%202004;230(1):13%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Já K. Biofilm dispersal: mechanisms, clinical implications, and potential therapeutic uses. J Dent Res. 2010;89(3):205–18.
</cite> [<a href="https://doi.org/10.1177/0022034509359403" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3318030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20139339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?J%C3%A1%20K.%20Biofilm%20dispersal:%20mechanisms,%20clinical%20implications,%20and%20potential%20therapeutic%20uses.%20J%20Dent%20Res.%202010;89(3):205%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Patriquin GM, Banin E, Gilmour C, Tuchman R, Greenberg EP, Poole K. Influence of quorum sensing and iron on twitching motility and biofilm formation in <em>Pseudomonas aeruginosa</em>. J Bacteriol. 2008;190(2):662–71.
</cite> [<a href="https://doi.org/10.1128/JB.01473-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2223716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17993517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patriquin%20GM,%20Banin%20E,%20Gilmour%20C,%20Tuchman%20R,%20Greenberg%20EP,%20Poole%20K.%20Influence%20of%20quorum%20sensing%20and%20iron%20on%20twitching%20motility%20and%20biofilm%20formation%20in%20Pseudomonas%20aeruginosa.%20J%20Bacteriol.%202008;190(2):662%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Abebe GM. The role of bacterial biofilm in antibiotic resistance and food contamination. Int J Microbiol. 2020;2020(1):1705814.
</cite> [<a href="https://doi.org/10.1155/2020/1705814" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7468660/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32908520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abebe%20GM.%20The%20role%20of%20bacterial%20biofilm%20in%20antibiotic%20resistance%20and%20food%20contamination.%20Int%20J%20Microbiol.%202020;2020(1):1705814." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Hancock RE, Speert DP. Antibiotic resistance in <em>Pseudomonas aeruginosa</em>: mechanisms and impact on treatment. Drug Resist Updat. 2000;3(4):247–55.
</cite> [<a href="https://doi.org/10.1054/drup.2000.0152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11498392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hancock%20RE,%20Speert%20DP.%20Antibiotic%20resistance%20in%20Pseudomonas%20aeruginosa:%20mechanisms%20and%20impact%20on%20treatment.%20Drug%20Resist%20Updat.%202000;3(4):247%E2%80%9355." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Redfern J, Wallace J, van Belkum A, Jaillard M, Whittard E, Ragupathy R, et al. Biofilm associated genotypes of multiple antibiotic resistant Pseudomonas aeruginosa. BMC Genomics. 2021;22:1–16.
</cite> [<a href="https://doi.org/10.1186/s12864-021-07818-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8314537/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34311706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Redfern%20J,%20Wallace%20J,%20van%20Belkum%20A,%20Jaillard%20M,%20Whittard%20E,%20Ragupathy%20R,%20et%20al.%20Biofilm%20associated%20genotypes%20of%20multiple%20antibiotic%20resistant%20Pseudomonas%20aeruginosa.%20BMC%20Genomics.%202021;22:1%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Luo Y, Yang Q, Zhang D, Yan W. Mechanisms and control strategies of antibiotic resistance in pathological biofilms. J Microbiol Biotechnol. 2020;31(1):1.</cite> [<a href="https://doi.org/10.4014/jmb.2010.10021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9706009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33323672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luo%20Y,%20Yang%20Q,%20Zhang%20D,%20Yan%20W.%20Mechanisms%20and%20control%20strategies%20of%20antibiotic%20resistance%20in%20pathological%20biofilms.%20J%20Microbiol%20Biotechnol.%202020;31(1):1." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Haddad Kashani H, Schmelcher M, Sabzalipoor H, Seyed Hosseini E, Moniri R: Recombinant endolysins as potential therapeutics against antibiotic-resistant Staphylococcus aureus: current status of research and novel delivery strategies. Clinical microbiology reviews 2018;31(1). 10.1128/cmr.00071-00017.</cite> [<a href="https://doi.org/10.1128/CMR.00071-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5740972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29187396/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Kashani HH, Moniri R. Expression of recombinant pET22b-LysK-cysteine/histidine-dependent amidohydrolase/peptidase bacteriophage therapeutic protein in <em>Escherichia coli</em> BL21 (DE3). Osong Public Health Res Perspect. 2015;6(4):256–60.
</cite> [<a href="https://doi.org/10.1016/j.phrp.2015.08.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4588433/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26473093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kashani%20HH,%20Moniri%20R.%20Expression%20of%20recombinant%20pET22b-LysK-cysteine/histidine-dependent%20amidohydrolase/peptidase%20bacteriophage%20therapeutic%20protein%20in%20Escherichia%20coli%20BL21%20(DE3).%20Osong%20Public%20Health%20Res%20Perspect.%202015;6(4):256%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Hosseini ES, Moniri R, Goli YD, Kashani HH. Purification of antibacterial CHAPK protein using a self-cleaving fusion tag and its activity against methicillin-resistant <em>Staphylococcus aureus</em>. Probiotics Antimicrob Proteins. 2016;8(4):202–10.
</cite> [<a href="https://doi.org/10.1007/s12602-016-9236-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27797005/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hosseini%20ES,%20Moniri%20R,%20Goli%20YD,%20Kashani%20HH.%20Purification%20of%20antibacterial%20CHAPK%20protein%20using%20a%20self-cleaving%20fusion%20tag%20and%20its%20activity%20against%20methicillin-resistant%20Staphylococcus%20aureus.%20Probiotics%20Antimicrob%20Proteins.%202016;8(4):202%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Van Hoogstraten S, Kuik C, Arts J, Cillero-Pastor B: Molecular imaging of bacterial biofilms—a ‎systematic review. Critical reviews in microbiology 2024;50(6):971–992.‎</cite> [<a href="https://doi.org/10.1080/1040841X.2023.2223704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11523921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37452571/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Hall CW, Mah T-F. Molecular mechanisms of biofilm-based antibiotic resistance and tolerance in pathogenic bacteria. FEMS Microbiol Rev. 2017;41(3):276–301.
</cite> [<a href="https://doi.org/10.1093/femsre/fux010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28369412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hall%20CW,%20Mah%20T-F.%20Molecular%20mechanisms%20of%20biofilm-based%20antibiotic%20resistance%20and%20tolerance%20in%20pathogenic%20bacteria.%20FEMS%20Microbiol%20Rev.%202017;41(3):276%E2%80%93301." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Li P, Wan P, Zhao R, Chen J, Li X, Li J, et al. Targeted elimination of bla NDM-5 gene in <em>Escherichia coli</em> by conjugative CRISPR-Cas9 system. Infect Drug Resist. 2022. 10.2147/IDR.S357470.
</cite> [<a href="https://doi.org/10.2147/IDR.S357470" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9004731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35422639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20P,%20Wan%20P,%20Zhao%20R,%20Chen%20J,%20Li%20X,%20Li%20J,%20et%20al.%20Targeted%20elimination%20of%20bla%20NDM-5%20gene%20in%20Escherichia%20coli%20by%20conjugative%20CRISPR-Cas9%20system.%20Infect%20Drug%20Resist.%202022.%2010.2147/IDR.S357470." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Marraffini LA. CRISPR-Cas immunity in prokaryotes. Nature. 2015;526(7571):55–61.
</cite> [<a href="https://doi.org/10.1038/nature15386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26432244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marraffini%20LA.%20CRISPR-Cas%20immunity%20in%20prokaryotes.%20Nature.%202015;526(7571):55%E2%80%9361." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Vigouroux A, Bikard D. CRISPR tools to control gene expression in bacteria. Microbiol Mol Biol Rev. 2020;84(2). 10.1128/MMBR.00077-19.</cite> [<a href="https://doi.org/10.1128/MMBR.00077-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7117552/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32238445/" class="usa-link">PubMed</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Singh V, Gohil N, Ramirez Garcia R, Braddick D, Fofié CK. Recent advances in CRISPR-Cas9 genome editing technology for biological and biomedical investigations. J Cell Biochem. 2018;119(1):81–94.
</cite> [<a href="https://doi.org/10.1002/jcb.26165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28544016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh%20V,%20Gohil%20N,%20Ramirez%20Garcia%20R,%20Braddick%20D,%20Fofi%C3%A9%20CK.%20Recent%20advances%20in%20CRISPR-Cas9%20genome%20editing%20technology%20for%20biological%20and%20biomedical%20investigations.%20J%20Cell%20Biochem.%202018;119(1):81%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Sapranauskas R, Gasiunas G, Fremaux C, Barrangou R, Horvath P, Siksnys V. The <em>Streptococcus thermophilus</em> CRISPR/Cas system provides immunity in <em>Escherichia coli</em>. Nucleic Acids Res. 2011;39(21):9275–82.
</cite> [<a href="https://doi.org/10.1093/nar/gkr606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3241640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21813460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sapranauskas%20R,%20Gasiunas%20G,%20Fremaux%20C,%20Barrangou%20R,%20Horvath%20P,%20Siksnys%20V.%20The%20Streptococcus%20thermophilus%20CRISPR/Cas%20system%20provides%20immunity%20in%20Escherichia%20coli.%20Nucleic%20Acids%20Res.%202011;39(21):9275%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Wu Y, Battalapalli D, Hakeem MJ, Selamneni V, Zhang P, Draz MS, et al. Engineered CRISPR-Cas systems for the detection and control of antibiotic-resistant infections. J Nanobiotechnol. 2021;19:1–26.</cite> [<a href="https://doi.org/10.1186/s12951-021-01132-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8642896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34863214/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20Y,%20Battalapalli%20D,%20Hakeem%20MJ,%20Selamneni%20V,%20Zhang%20P,%20Draz%20MS,%20et%20al.%20Engineered%20CRISPR-Cas%20systems%20for%20the%20detection%20and%20control%20of%20antibiotic-resistant%20infections.%20J%20Nanobiotechnol.%202021;19:1%E2%80%9326." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Zhang L, Wang L, Xie Y, Wang P, Deng S, Qin A, et al. Triple-targeting delivery of CRISPR/Cas9 to reduce the risk of cardiovascular diseases. Angew Chem Int Ed Engl. 2019;58(36):12404–8.
</cite> [<a href="https://doi.org/10.1002/anie.201903618" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31318118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20L,%20Wang%20L,%20Xie%20Y,%20Wang%20P,%20Deng%20S,%20Qin%20A,%20et%20al.%20Triple-targeting%20delivery%20of%20CRISPR/Cas9%20to%20reduce%20the%20risk%20of%20cardiovascular%20diseases.%20Angew%20Chem%20Int%20Ed%20Engl.%202019;58(36):12404%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Timin AS, Muslimov AR, Lepik KV, Epifanovskaya OS, Shakirova AI, Mock U, et al. Efficient gene editing via non-viral delivery of CRISPR-Cas9 system using polymeric and hybrid microcarriers. Nanomedicine. 2018;14(1):97–108.
</cite> [<a href="https://doi.org/10.1016/j.nano.2017.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28917642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Timin%20AS,%20Muslimov%20AR,%20Lepik%20KV,%20Epifanovskaya%20OS,%20Shakirova%20AI,%20Mock%20U,%20et%20al.%20Efficient%20gene%20editing%20via%20non-viral%20delivery%20of%20CRISPR-Cas9%20system%20using%20polymeric%20and%20hybrid%20microcarriers.%20Nanomedicine.%202018;14(1):97%E2%80%93108." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Gratacap RL, Regan T, Dehler CE, Martin SAM, Boudinot P, Collet B, et al. Efficient CRISPR/Cas9 genome editing in a salmonid fish cell line using a lentivirus delivery system. BMC Biotechnol. 2020;20(1):35.
</cite> [<a href="https://doi.org/10.1186/s12896-020-00626-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7310381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32576161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gratacap%20RL,%20Regan%20T,%20Dehler%20CE,%20Martin%20SAM,%20Boudinot%20P,%20Collet%20B,%20et%20al.%20Efficient%20CRISPR/Cas9%20genome%20editing%20in%20a%20salmonid%20fish%20cell%20line%20using%20a%20lentivirus%20delivery%20system.%20BMC%20Biotechnol.%202020;20(1):35." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Glass Z, Li Y, Xu Q. Nanoparticles for CRISPR-Cas9 delivery. Nat Biomed Eng. 2017;1(11):854–5.
</cite> [<a href="https://doi.org/10.1038/s41551-017-0158-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6107070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30147994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Glass%20Z,%20Li%20Y,%20Xu%20Q.%20Nanoparticles%20for%20CRISPR-Cas9%20delivery.%20Nat%20Biomed%20Eng.%202017;1(11):854%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Sago CD, Lokugamage MP, Paunovska K, Vanover DA, Monaco CM, Shah NN, et al. High-throughput in vivo screen of functional mRNA delivery identifies nanoparticles for endothelial cell gene editing. Proc Natl Acad Sci U S A. 2018;115(42):E9944–52.
</cite> [<a href="https://doi.org/10.1073/pnas.1811276115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6196543/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30275336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sago%20CD,%20Lokugamage%20MP,%20Paunovska%20K,%20Vanover%20DA,%20Monaco%20CM,%20Shah%20NN,%20et%20al.%20High-throughput%20in%20vivo%20screen%20of%20functional%20mRNA%20delivery%20identifies%20nanoparticles%20for%20endothelial%20cell%20gene%20editing.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202018;115(42):E9944%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Zhang R, Xu W, Shao S, Wang Q. Gene silencing through CRISPR interference in bacteria: current advances and future prospects. Front Microbiol. 2021;12:635227.
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.635227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8044314/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33868193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20R,%20Xu%20W,%20Shao%20S,%20Wang%20Q.%20Gene%20silencing%20through%20CRISPR%20interference%20in%20bacteria:%20current%20advances%20and%20future%20prospects.%20Front%20Microbiol.%202021;12:635227." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Lee B, Lee K, Panda S, Gonzales-Rojas R, Chong A, Bugay V, et al. Nanoparticle delivery of CRISPR into the brain rescues a mouse model of fragile X syndrome from exaggerated repetitive behaviours. Nat Biomed Eng. 2018;2(7):497–507.
</cite> [<a href="https://doi.org/10.1038/s41551-018-0252-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6544395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30948824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20B,%20Lee%20K,%20Panda%20S,%20Gonzales-Rojas%20R,%20Chong%20A,%20Bugay%20V,%20et%20al.%20Nanoparticle%20delivery%20of%20CRISPR%20into%20the%20brain%20rescues%20a%20mouse%20model%20of%20fragile%20X%20syndrome%20from%20exaggerated%20repetitive%20behaviours.%20Nat%20Biomed%20Eng.%202018;2(7):497%E2%80%93507." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Liu Y, Zhao G, Xu CF, Luo YL, Lu ZD, Wang J. Systemic delivery of CRISPR/Cas9 with PEG-PLGA nanoparticles for chronic myeloid leukemia targeted therapy. Biomater Sci. 2018;6(6):1592–603.
</cite> [<a href="https://doi.org/10.1039/c8bm00263k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29725684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20Y,%20Zhao%20G,%20Xu%20CF,%20Luo%20YL,%20Lu%20ZD,%20Wang%20J.%20Systemic%20delivery%20of%20CRISPR/Cas9%20with%20PEG-PLGA%20nanoparticles%20for%20chronic%20myeloid%20leukemia%20targeted%20therapy.%20Biomater%20Sci.%202018;6(6):1592%E2%80%93603." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Shahbazi R, Sghia-Hughes G, Reid JL, Kubek S, Haworth KG, Humbert O, et al. Targeted homology-directed repair in blood stem and progenitor cells with CRISPR nanoformulations. Nat Mater. 2019;18(10):1124–32.
</cite> [<a href="https://doi.org/10.1038/s41563-019-0385-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6754292/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31133730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shahbazi%20R,%20Sghia-Hughes%20G,%20Reid%20JL,%20Kubek%20S,%20Haworth%20KG,%20Humbert%20O,%20et%20al.%20Targeted%20homology-directed%20repair%20in%20blood%20stem%20and%20progenitor%20cells%20with%20CRISPR%20nanoformulations.%20Nat%20Mater.%202019;18(10):1124%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Kadkhoda H, Gholizadeh P, Kafil HS, Ghotaslou R, Pirzadeh T, Rezaee MA, et al. Role of CRISPR-Cas systems and anti-CRISPR proteins in bacterial antibiotic resistance. Heliyon. 2024. 10.1016/j.heliyon.2024.e34692.
</cite> [<a href="https://doi.org/10.1016/j.heliyon.2024.e34692" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11325803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39149034/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kadkhoda%20H,%20Gholizadeh%20P,%20Kafil%20HS,%20Ghotaslou%20R,%20Pirzadeh%20T,%20Rezaee%20MA,%20et%20al.%20Role%20of%20CRISPR-Cas%20systems%20and%20anti-CRISPR%20proteins%20in%20bacterial%20antibiotic%20resistance.%20Heliyon.%202024.%2010.1016/j.heliyon.2024.e34692." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Nath A, Bhattacharjee R, Nandi A, Sinha A, Kar S, Manoharan N, et al. Phage delivered CRISPR-Cas system to combat multidrug-resistant pathogens in gut microbiome. Biomed Pharmacother. 2022;151:113122.
</cite> [<a href="https://doi.org/10.1016/j.biopha.2022.113122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35594718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nath%20A,%20Bhattacharjee%20R,%20Nandi%20A,%20Sinha%20A,%20Kar%20S,%20Manoharan%20N,%20et%20al.%20Phage%20delivered%20CRISPR-Cas%20system%20to%20combat%20multidrug-resistant%20pathogens%20in%20gut%20microbiome.%20Biomed%20Pharmacother.%202022;151:113122." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Wang H, La Russa M, Qi LS. CRISPR/Cas9 in genome editing and beyond. Annu Rev Biochem. 2016;85(1):227–64.
</cite> [<a href="https://doi.org/10.1146/annurev-biochem-060815-014607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27145843/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20H,%20La%20Russa%20M,%20Qi%20LS.%20CRISPR/Cas9%20in%20genome%20editing%20and%20beyond.%20Annu%20Rev%20Biochem.%202016;85(1):227%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Asmamaw M, Zawdie B. Mechanism and applications of CRISPR/Cas-9-mediated genome editing. Biologics. 2021;15:353–61.
</cite> [<a href="https://doi.org/10.2147/BTT.S326422" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8388126/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34456559/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Asmamaw%20M,%20Zawdie%20B.%20Mechanism%20and%20applications%20of%20CRISPR/Cas-9-mediated%20genome%20editing.%20Biologics.%202021;15:353%E2%80%9361." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Barrangou R, Fremaux C, Deveau H, Richards M, Boyaval P, Moineau S, et al. CRISPR provides acquired resistance against viruses in prokaryotes. Science. 2007;315(5819):1709–12.
</cite> [<a href="https://doi.org/10.1126/science.1138140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17379808/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barrangou%20R,%20Fremaux%20C,%20Deveau%20H,%20Richards%20M,%20Boyaval%20P,%20Moineau%20S,%20et%20al.%20CRISPR%20provides%20acquired%20resistance%20against%20viruses%20in%20prokaryotes.%20Science.%202007;315(5819):1709%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Angel PASRY, Raghul M, Gowsalya S, Paulkumar K, Murugan K: CRISPR interference system: a ‎potential strategy to inhibit pathogenic biofilm in the agri-food sector. In: CRISPR and RNAi ‎Systems. edn.: Elsevier; 2021:387–403.‎</cite>
</li>
<li id="CR65">
<span class="label">65.</span><cite>Kim J-S, Cho D-H, Park M, Chung W-J, Shin D, Ko KS, et al. CRISPR/Cas9-mediated re-sensitization of antibiotic-resistant <em>Escherichia coli</em> harboring extended-spectrum β-lactamases. J Microbiol Biotechnol. 2016;26(2):394–401.
</cite> [<a href="https://doi.org/10.4014/jmb.1508.08080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26502735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20J-S,%20Cho%20D-H,%20Park%20M,%20Chung%20W-J,%20Shin%20D,%20Ko%20KS,%20et%20al.%20CRISPR/Cas9-mediated%20re-sensitization%20of%20antibiotic-resistant%20Escherichia%20coli%20harboring%20extended-spectrum%20%CE%B2-lactamases.%20J%20Microbiol%20Biotechnol.%202016;26(2):394%E2%80%93401." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Tao S, Chen H, Li N, Liang W. The application of the CRISPR-Cas system in antibiotic resistance. Infect Drug Resist. 2022. 10.2147/IDR.S370869.
</cite> [<a href="https://doi.org/10.2147/IDR.S370869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9356603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35942309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tao%20S,%20Chen%20H,%20Li%20N,%20Liang%20W.%20The%20application%20of%20the%20CRISPR-Cas%20system%20in%20antibiotic%20resistance.%20Infect%20Drug%20Resist.%202022.%2010.2147/IDR.S370869." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Palacios Araya D, Palmer KL, Duerkop BA. Crispr-based antimicrobials to obstruct antibiotic-resistant and pathogenic bacteria. PLoS Pathog. 2021;17(7):e1009672.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1009672" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8266055/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34237097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palacios%20Araya%20D,%20Palmer%20KL,%20Duerkop%20BA.%20Crispr-based%20antimicrobials%20to%20obstruct%20antibiotic-resistant%20and%20pathogenic%20bacteria.%20PLoS%20Pathog.%202021;17(7):e1009672." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Luo M, Jia Y-Y, Jing Z-W, Li C, Zhou S-Y, Mei Q-B, et al. Construction and optimization of pH-sensitive nanoparticle delivery system containing PLGA and UCCs-2 for targeted treatment of <em>Helicobacter pylori</em>. Colloids Surf, B. 2018;164:11–9.</cite> [<a href="https://doi.org/10.1016/j.colsurfb.2018.01.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29367052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luo%20M,%20Jia%20Y-Y,%20Jing%20Z-W,%20Li%20C,%20Zhou%20S-Y,%20Mei%20Q-B,%20et%20al.%20Construction%20and%20optimization%20of%20pH-sensitive%20nanoparticle%20delivery%20system%20containing%20PLGA%20and%20UCCs-2%20for%20targeted%20treatment%20of%20Helicobacter%20pylori.%20Colloids%20Surf,%20B.%202018;164:11%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Khambhati K, Bhattacharjee G, Gohil N, Dhanoa GK, Sagona AP, Mani I, et al. Phage engineering and phage-assisted CRISPR-Cas delivery to combat multidrug-resistant pathogens. Bioeng Transl Med. 2023;8(2):e10381.
</cite> [<a href="https://doi.org/10.1002/btm2.10381" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10013820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36925687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khambhati%20K,%20Bhattacharjee%20G,%20Gohil%20N,%20Dhanoa%20GK,%20Sagona%20AP,%20Mani%20I,%20et%20al.%20Phage%20engineering%20and%20phage-assisted%20CRISPR-Cas%20delivery%20to%20combat%20multidrug-resistant%20pathogens.%20Bioeng%20Transl%20Med.%202023;8(2):e10381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Gliźniewicz M, Miłek D, Olszewska P, Czajkowski A, Serwin N, Cecerska-Heryć E, et al. Advances in bacteriophage-mediated strategies for combating polymicrobial biofilms. Front Microbiol. 2024;14:1320345.
</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1320345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10797108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38249486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gli%C5%BAniewicz%20M,%20Mi%C5%82ek%20D,%20Olszewska%20P,%20Czajkowski%20A,%20Serwin%20N,%20Cecerska-Hery%C4%87%20E,%20et%20al.%20Advances%20in%20bacteriophage-mediated%20strategies%20for%20combating%20polymicrobial%20biofilms.%20Front%20Microbiol.%202024;14:1320345." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Radovic-Moreno AF, Lu TK, Puscasu VA, Yoon CJ, Langer R, Farokhzad OC. Surface charge-switching polymeric nanoparticles for bacterial cell wall-targeted delivery of antibiotics. ACS Nano. 2012;6(5):4279–87.
</cite> [<a href="https://doi.org/10.1021/nn3008383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3779925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22471841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Radovic-Moreno%20AF,%20Lu%20TK,%20Puscasu%20VA,%20Yoon%20CJ,%20Langer%20R,%20Farokhzad%20OC.%20Surface%20charge-switching%20polymeric%20nanoparticles%20for%20bacterial%20cell%20wall-targeted%20delivery%20of%20antibiotics.%20ACS%20Nano.%202012;6(5):4279%E2%80%9387." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Zaman QU, Chu W, Hao M, Shi Y, Sun M, Sang S-F, et al. CRISPR/Cas9-mediated multiplex genome editing of JAGGED gene in <em>Brassica napus</em> L. Biomolecules. 2019;9(11):725.
</cite> [<a href="https://doi.org/10.3390/biom9110725" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6921047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31726660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zaman%20QU,%20Chu%20W,%20Hao%20M,%20Shi%20Y,%20Sun%20M,%20Sang%20S-F,%20et%20al.%20CRISPR/Cas9-mediated%20multiplex%20genome%20editing%20of%20JAGGED%20gene%20in%20Brassica%20napus%20L.%20Biomolecules.%202019;9(11):725." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Bondy-Denomy J. Protein inhibitors of CRISPR-Cas9. ACS Chem Biol. 2018;13(2):417–23.
</cite> [<a href="https://doi.org/10.1021/acschembio.7b00831" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6049822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29251498/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bondy-Denomy%20J.%20Protein%20inhibitors%20of%20CRISPR-Cas9.%20ACS%20Chem%20Biol.%202018;13(2):417%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Chabas H, Nicot A, Meaden S, Westra ER, Tremblay DM, Pradier L, et al. Variability in the durability of CRISPR-Cas immunity. Philos Trans R Soc Lond B Biol Sci. 2019;374(1772):20180097.
</cite> [<a href="https://doi.org/10.1098/rstb.2018.0097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6452261/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30905283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chabas%20H,%20Nicot%20A,%20Meaden%20S,%20Westra%20ER,%20Tremblay%20DM,%20Pradier%20L,%20et%20al.%20Variability%20in%20the%20durability%20of%20CRISPR-Cas%20immunity.%20Philos%20Trans%20R%20Soc%20Lond%20B%20Biol%20Sci.%202019;374(1772):20180097." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Aslam B, Rasool M, Idris A, Muzammil S, Alvi RF, Khurshid M, et al. CRISPR-Cas system: a potential alternative tool to cope antibiotic resistance. Antimicrob Resist Infect Control. 2020;9:1–3.
</cite> [<a href="https://doi.org/10.1186/s13756-020-00795-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7418376/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32778162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aslam%20B,%20Rasool%20M,%20Idris%20A,%20Muzammil%20S,%20Alvi%20RF,%20Khurshid%20M,%20et%20al.%20CRISPR-Cas%20system:%20a%20potential%20alternative%20tool%20to%20cope%20antibiotic%20resistance.%20Antimicrob%20Resist%20Infect%20Control.%202020;9:1%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Ortiz-Cartagena C, Fernández-Grela P, Armán L, Blasco L, Pablo-Marcos D, Bleriot I, Fernández-‎García L, Ibarguren-Quiles C, Fernández-Cuenca F, Barrio-Pujante A:.The LAMP-CRISPR-Cas13a ‎technique for detecting the CBASS mechanism of phage resistance in bacteria. Frontiers in ‎Microbiology. 2025;16:1550534.‎</cite> [<a href="https://doi.org/10.3389/fmicb.2025.1550534" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11973324/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40196034/" class="usa-link">PubMed</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Gupta A, Saleh NM, Das R, Landis RF, Bigdeli A, Motamedchaboki K, et al. Synergistic antimicrobial therapy using nanoparticles and antibiotics for the treatment of multidrug-resistant bacterial infection. Nano Futures. 2017;1(1):015004.</cite> [<a href="https://scholar.google.com/scholar_lookup?Gupta%20A,%20Saleh%20NM,%20Das%20R,%20Landis%20RF,%20Bigdeli%20A,%20Motamedchaboki%20K,%20et%20al.%20Synergistic%20antimicrobial%20therapy%20using%20nanoparticles%20and%20antibiotics%20for%20the%20treatment%20of%20multidrug-resistant%20bacterial%20infection.%20Nano%20Futures.%202017;1(1):015004." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Fatima F, Siddiqui S, Khan WA. Nanoparticles as novel emerging therapeutic antibacterial agents in the antibiotics resistant era. Biol Trace Elem Res. 2021;199(7):2552–64.
</cite> [<a href="https://doi.org/10.1007/s12011-020-02394-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33030657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fatima%20F,%20Siddiqui%20S,%20Khan%20WA.%20Nanoparticles%20as%20novel%20emerging%20therapeutic%20antibacterial%20agents%20in%20the%20antibiotics%20resistant%20era.%20Biol%20Trace%20Elem%20Res.%202021;199(7):2552%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Mulens-Arias V, Rojas JM, Barber DF. The intrinsic biological identities of iron oxide nanoparticles and their coatings: unexplored territory for combinatorial therapies. Nanomaterials. 2020;10(5):837.
</cite> [<a href="https://doi.org/10.3390/nano10050837" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7712800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32349362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mulens-Arias%20V,%20Rojas%20JM,%20Barber%20DF.%20The%20intrinsic%20biological%20identities%20of%20iron%20oxide%20nanoparticles%20and%20their%20coatings:%20unexplored%20territory%20for%20combinatorial%20therapies.%20Nanomaterials.%202020;10(5):837." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Jacob EM, Borah A, Sakthi KD. CRISPR/Cas9 Nano-delivery Approaches for Targeted Gene Therapy. Pharmaceutical Nanobiotechnology for Targeted Therapy: Springer; 2022. p. 27–64.</cite> [<a href="https://scholar.google.com/scholar_lookup?Jacob%20EM,%20Borah%20A,%20Sakthi%20KD.%20CRISPR/Cas9%20Nano-delivery%20Approaches%20for%20Targeted%20Gene%20Therapy.%20Pharmaceutical%20Nanobiotechnology%20for%20Targeted%20Therapy:%20Springer;%202022.%20p.%2027%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Karimi M, Ghasemi A, Zangabad PS, Rahighi R, Basri SMM, Mirshekari H, et al. Smart micro/nanoparticles in stimulus-responsive drug/gene delivery systems. Chem Soc Rev. 2016;45(5):1457–501.
</cite> [<a href="https://doi.org/10.1039/c5cs00798d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4775468/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26776487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karimi%20M,%20Ghasemi%20A,%20Zangabad%20PS,%20Rahighi%20R,%20Basri%20SMM,%20Mirshekari%20H,%20et%20al.%20Smart%20micro/nanoparticles%20in%20stimulus-responsive%20drug/gene%20delivery%20systems.%20Chem%20Soc%20Rev.%202016;45(5):1457%E2%80%93501." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Chavanpatil MD, Khdair A, Panyam J. Nanoparticles for cellular drug delivery: mechanisms and factors influencing delivery. J Nanosci Nanotechnol. 2006;6(9–10):2651–63.
</cite> [<a href="https://doi.org/10.1166/jnn.2006.443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17048473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chavanpatil%20MD,%20Khdair%20A,%20Panyam%20J.%20Nanoparticles%20for%20cellular%20drug%20delivery:%20mechanisms%20and%20factors%20influencing%20delivery.%20J%20Nanosci%20Nanotechnol.%202006;6(9%E2%80%9310):2651%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Anarjan FS. Active targeting drug delivery nanocarriers: Ligands. Nano Struct Nano Objects. 2019;19:100370.</cite> [<a href="https://scholar.google.com/scholar_lookup?Anarjan%20FS.%20Active%20targeting%20drug%20delivery%20nanocarriers:%20Ligands.%20Nano%20Struct%20Nano%20Objects.%202019;19:100370." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Pack DW, Hoffman AS, Pun S, Stayton PS. Design and development of polymers for gene delivery. Nat Rev Drug Discovery. 2005;4(7):581–93.
</cite> [<a href="https://doi.org/10.1038/nrd1775" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16052241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pack%20DW,%20Hoffman%20AS,%20Pun%20S,%20Stayton%20PS.%20Design%20and%20development%20of%20polymers%20for%20gene%20delivery.%20Nat%20Rev%20Drug%20Discovery.%202005;4(7):581%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Almeciga-Diaz CJ, Barrera LA. Design and applications of gene therapy vectors for mucopolysaccharidosis in Colombia. Gene Ther. 2020;27(1):104–7.
</cite> [<a href="https://doi.org/10.1038/s41434-019-0086-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31267008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Almeciga-Diaz%20CJ,%20Barrera%20LA.%20Design%20and%20applications%20of%20gene%20therapy%20vectors%20for%20mucopolysaccharidosis%20in%20Colombia.%20Gene%20Ther.%202020;27(1):104%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Leal AF, Cifuentes J, Torres CE, Suárez D, Quezada V, Gómez SC, et al. Delivery and assessment of a CRISPR/nCas9-based genome editing system on <em>in vitro</em> models of mucopolysaccharidoses IVA assisted by magnetite-based nanoparticles. Sci Rep. 2022;12(1):15045.
</cite> [<a href="https://doi.org/10.1038/s41598-022-19407-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9440901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36057729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leal%20AF,%20Cifuentes%20J,%20Torres%20CE,%20Su%C3%A1rez%20D,%20Quezada%20V,%20G%C3%B3mez%20SC,%20et%20al.%20Delivery%20and%20assessment%20of%20a%20CRISPR/nCas9-based%20genome%20editing%20system%20on%20in%20vitro%20models%20of%20mucopolysaccharidoses%20IVA%20assisted%20by%20magnetite-based%20nanoparticles.%20Sci%20Rep.%202022;12(1):15045." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Zhang Q, Kuang G, Li W, Wang J, Ren H, Zhao Y. Stimuli-responsive gene delivery nanocarriers for cancer therapy. Nano Micro Lett. 2023;15(1):44.</cite> [<a href="https://doi.org/10.1007/s40820-023-01018-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9908819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36752939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Q,%20Kuang%20G,%20Li%20W,%20Wang%20J,%20Ren%20H,%20Zhao%20Y.%20Stimuli-responsive%20gene%20delivery%20nanocarriers%20for%20cancer%20therapy.%20Nano%20Micro%20Lett.%202023;15(1):44." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Brooks BD, Brooks AE. Therapeutic strategies to combat antibiotic resistance. Adv Drug Deliv Rev. 2014;78:14–27.
</cite> [<a href="https://doi.org/10.1016/j.addr.2014.10.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25450262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brooks%20BD,%20Brooks%20AE.%20Therapeutic%20strategies%20to%20combat%20antibiotic%20resistance.%20Adv%20Drug%20Deliv%20Rev.%202014;78:14%E2%80%9327." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Slavin YN, Asnis J, Hńfeli UO, Bach H. Metal nanoparticles: understanding the mechanisms behind antibacterial activity. J Nanobiotechnology. 2017;15:1–20.
</cite> [<a href="https://doi.org/10.1186/s12951-017-0308-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5627441/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28974225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Slavin%20YN,%20Asnis%20J,%20H%C5%84feli%20UO,%20Bach%20H.%20Metal%20nanoparticles:%20understanding%20the%20mechanisms%20behind%20antibacterial%20activity.%20J%20Nanobiotechnology.%202017;15:1%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Nagarajan P, Subramaniyan V, Elavarasan V, Mohandoss N, Subramaniyan P, Vijayakumar S. Biofabricated aluminium oxide nanoparticles derived from <em>Citrus aurantium</em> L.: antimicrobial, anti-proliferation, and photocatalytic efficiencies. Sustainability. 2023;15(2):1743.</cite> [<a href="https://scholar.google.com/scholar_lookup?Nagarajan%20P,%20Subramaniyan%20V,%20Elavarasan%20V,%20Mohandoss%20N,%20Subramaniyan%20P,%20Vijayakumar%20S.%20Biofabricated%20aluminium%20oxide%20nanoparticles%20derived%20from%20Citrus%20aurantium%20L.:%20antimicrobial,%20anti-proliferation,%20and%20photocatalytic%20efficiencies.%20Sustainability.%202023;15(2):1743." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Flores-López LZ, Espinoza-Gómez H, Somanathan R. Silver nanoparticles: electron transfer, reactive oxygen species, oxidative stress, beneficial and toxicological effects. J Appl Toxicol. 2019;39(1):16–26.
</cite> [<a href="https://doi.org/10.1002/jat.3654" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29943411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Flores-L%C3%B3pez%20LZ,%20Espinoza-G%C3%B3mez%20H,%20Somanathan%20R.%20Silver%20nanoparticles:%20electron%20transfer,%20reactive%20oxygen%20species,%20oxidative%20stress,%20beneficial%20and%20toxicological%20effects.%20J%20Appl%20Toxicol.%202019;39(1):16%E2%80%9326." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Khan SS, Ullah I, Ullah S, An R, Xu H, Nie K, et al. Recent advances in the surface functionalization of nanomaterials for antimicrobial applications. Materials. 2021;14(22):6932.
</cite> [<a href="https://doi.org/10.3390/ma14226932" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8623114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34832332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khan%20SS,%20Ullah%20I,%20Ullah%20S,%20An%20R,%20Xu%20H,%20Nie%20K,%20et%20al.%20Recent%20advances%20in%20the%20surface%20functionalization%20of%20nanomaterials%20for%20antimicrobial%20applications.%20Materials.%202021;14(22):6932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Fabrega J, Fawcett SR, Renshaw JC, Lead JR. Silver nanoparticle impact on bacterial growth: effect of pH, concentration, and organic matter. Environ Sci Technol. 2009;43(19):7285–90.
</cite> [<a href="https://doi.org/10.1021/es803259g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19848135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fabrega%20J,%20Fawcett%20SR,%20Renshaw%20JC,%20Lead%20JR.%20Silver%20nanoparticle%20impact%20on%20bacterial%20growth:%20effect%20of%20pH,%20concentration,%20and%20organic%20matter.%20Environ%20Sci%20Technol.%202009;43(19):7285%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Murthy SK. Nanoparticles in modern medicine: state of the art and future challenges. Int J Nanomed. 2007;2(2):129–41.</cite> [<a href="/articles/PMC2673971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17722542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murthy%20SK.%20Nanoparticles%20in%20modern%20medicine:%20state%20of%20the%20art%20and%20future%20challenges.%20Int%20J%20Nanomed.%202007;2(2):129%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Chen Z, Liu F, Chen Y, Liu J, Wang X, Chen AT, et al. Targeted delivery of CRISPR/Cas9-mediated cancer gene therapy via liposome-templated hydrogel nanoparticles. Adv Funct Mater. 2017;27(46):1703036.
</cite> [<a href="https://doi.org/10.1002/adfm.201703036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5939593/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29755309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20Z,%20Liu%20F,%20Chen%20Y,%20Liu%20J,%20Wang%20X,%20Chen%20AT,%20et%20al.%20Targeted%20delivery%20of%20CRISPR/Cas9-mediated%20cancer%20gene%20therapy%20via%20liposome-templated%20hydrogel%20nanoparticles.%20Adv%20Funct%20Mater.%202017;27(46):1703036." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Chowdhry R, Lu SZ, Lee S, Godhulayyagari S, Ebrahimi SB, Samanta D. Enhancing CRISPR/Cas systems with nanotechnology. Trends Biotechnol. 2023;41(12):1549–64.
</cite> [<a href="https://doi.org/10.1016/j.tibtech.2023.06.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37451945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chowdhry%20R,%20Lu%20SZ,%20Lee%20S,%20Godhulayyagari%20S,%20Ebrahimi%20SB,%20Samanta%20D.%20Enhancing%20CRISPR/Cas%20systems%20with%20nanotechnology.%20Trends%20Biotechnol.%202023;41(12):1549%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Saw PE, Cui Gh, Xu X. Nanoparticles-mediated CRISPR/Cas gene editing delivery system. ChemMedChem. 2022;17(9):e202100777.
</cite> [<a href="https://doi.org/10.1002/cmdc.202100777" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35261159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saw%20PE,%20Cui%20Gh,%20Xu%20X.%20Nanoparticles-mediated%20CRISPR/Cas%20gene%20editing%20delivery%20system.%20ChemMedChem.%202022;17(9):e202100777." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Pandey P, Vavilala SL. From gene editing to biofilm busting: CRISPR-CAS9 against antibiotic resistance—a review. Cell Biochem Biophys. 2024;82(2):549–60.
</cite> [<a href="https://doi.org/10.1007/s12013-024-01276-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38702575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pandey%20P,%20Vavilala%20SL.%20From%20gene%20editing%20to%20biofilm%20busting:%20CRISPR-CAS9%20against%20antibiotic%20resistance%E2%80%94a%20review.%20Cell%20Biochem%20Biophys.%202024;82(2):549%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Albanese A, Tang PS, Chan WCW. The effect of nanoparticle size, shape, and surface chemistry on biological systems. Annu Rev Biomed Eng. 2012;14:1–16.
</cite> [<a href="https://doi.org/10.1146/annurev-bioeng-071811-150124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22524388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Albanese%20A,%20Tang%20PS,%20Chan%20WCW.%20The%20effect%20of%20nanoparticle%20size,%20shape,%20and%20surface%20chemistry%20on%20biological%20systems.%20Annu%20Rev%20Biomed%20Eng.%202012;14:1%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Blanco E, Shen H, Ferrari M. Principles of nanoparticle design for overcoming biological barriers to drug delivery. Nat Biotechnol. 2015;33(9):941–51.
</cite> [<a href="https://doi.org/10.1038/nbt.3330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4978509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26348965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blanco%20E,%20Shen%20H,%20Ferrari%20M.%20Principles%20of%20nanoparticle%20design%20for%20overcoming%20biological%20barriers%20to%20drug%20delivery.%20Nat%20Biotechnol.%202015;33(9):941%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Schrand AM, Rahman MF, Hussain SM, Schlager JJ, Smith DA, Syed AF. Metal-based nanoparticles and their toxicity assessment. WIREs Nanomed Nanobiotechnol. 2010;2(5):544–68.</cite> [<a href="https://doi.org/10.1002/wnan.103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20681021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schrand%20AM,%20Rahman%20MF,%20Hussain%20SM,%20Schlager%20JJ,%20Smith%20DA,%20Syed%20AF.%20Metal-based%20nanoparticles%20and%20their%20toxicity%20assessment.%20WIREs%20Nanomed%20Nanobiotechnol.%202010;2(5):544%E2%80%9368." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Fujii E, Kimura K, Mizoguchi K, Kato A, Takanashi H, Itoh Z, et al. Assessment of the carcinogenic potential of mitemcinal (GM-611): increased incidence of malignant lymphoma in a rat carcinogenicity study. Toxicol Appl Pharmacol. 2008;228(1):1–7.
</cite> [<a href="https://doi.org/10.1016/j.taap.2007.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18355581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fujii%20E,%20Kimura%20K,%20Mizoguchi%20K,%20Kato%20A,%20Takanashi%20H,%20Itoh%20Z,%20et%20al.%20Assessment%20of%20the%20carcinogenic%20potential%20of%20mitemcinal%20(GM-611):%20increased%20incidence%20of%20malignant%20lymphoma%20in%20a%20rat%20carcinogenicity%20study.%20Toxicol%20Appl%20Pharmacol.%202008;228(1):1%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Kara G, Calin GA, Ozpolat B. Rnai-based therapeutics and tumor targeted delivery in cancer. Adv Drug Deliv Rev. 2022;182:114113.
</cite> [<a href="https://doi.org/10.1016/j.addr.2022.114113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35063535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kara%20G,%20Calin%20GA,%20Ozpolat%20B.%20Rnai-based%20therapeutics%20and%20tumor%20targeted%20delivery%20in%20cancer.%20Adv%20Drug%20Deliv%20Rev.%202022;182:114113." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Tu K, Deng H, Kong L, Wang Y, Yang T, Hu Q, et al. Reshaping tumor immune microenvironment through acidity-responsive nanoparticles featured with CRISPR/Cas9-mediated programmed death-ligand 1 attenuation and chemotherapeutics-induced immunogenic cell death. ACS Appl Mater Interfaces. 2020;12(14):16018–30.
</cite> [<a href="https://doi.org/10.1021/acsami.9b23084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32192326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tu%20K,%20Deng%20H,%20Kong%20L,%20Wang%20Y,%20Yang%20T,%20Hu%20Q,%20et%20al.%20Reshaping%20tumor%20immune%20microenvironment%20through%20acidity-responsive%20nanoparticles%20featured%20with%20CRISPR/Cas9-mediated%20programmed%20death-ligand%201%20attenuation%20and%20chemotherapeutics-induced%20immunogenic%20cell%20death.%20ACS%20Appl%20Mater%20Interfaces.%202020;12(14):16018%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Xie R, Wang Y, Gong S. External stimuli-responsive nanoparticles for spatially and temporally controlled delivery of CRISPR-Cas genome editors. Biomater Sci. 2021;9(18):6012–22.
</cite> [<a href="https://doi.org/10.1039/d1bm00558h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8440484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34286726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie%20R,%20Wang%20Y,%20Gong%20S.%20External%20stimuli-responsive%20nanoparticles%20for%20spatially%20and%20temporally%20controlled%20delivery%20of%20CRISPR-Cas%20genome%20editors.%20Biomater%20Sci.%202021;9(18):6012%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>Zhao Z, Ding C, Wang Y, Tan H, Li J. PH-Responsive polymeric nanocarriers for efficient killing of cariogenic bacteria in biofilms. Biomater Sci. 2019;7:1643–51.
</cite> [<a href="https://doi.org/10.1039/c8bm01640b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30723851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20Z,%20Ding%20C,%20Wang%20Y,%20Tan%20H,%20Li%20J.%20PH-Responsive%20polymeric%20nanocarriers%20for%20efficient%20killing%20of%20cariogenic%20bacteria%20in%20biofilms.%20Biomater%20Sci.%202019;7:1643%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Cheng MHY, Leung J, Zhang Y, Strong C, Basha G, Momeni A, et al. Induction of bleb structures in lipid nanoparticle formulations of mRNA leads to improved transfection potency. Adv Mater. 2023;35(31):2303370.</cite> [<a href="https://doi.org/10.1002/adma.202303370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37172950/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng%20MHY,%20Leung%20J,%20Zhang%20Y,%20Strong%20C,%20Basha%20G,%20Momeni%20A,%20et%20al.%20Induction%20of%20bleb%20structures%20in%20lipid%20nanoparticle%20formulations%20of%20mRNA%20leads%20to%20improved%20transfection%20potency.%20Adv%20Mater.%202023;35(31):2303370." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite>Chan Y-T, Lu Y, Wu J, Zhang C, Tan H-Y, Bian Z-x, et al. CRISPR-Cas9 library screening approach for anti-cancer drug discovery: overview and perspectives. Theranostics. 2022;12(7):3329.
</cite> [<a href="https://doi.org/10.7150/thno.71144" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9065202/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35547744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chan%20Y-T,%20Lu%20Y,%20Wu%20J,%20Zhang%20C,%20Tan%20H-Y,%20Bian%20Z-x,%20et%20al.%20CRISPR-Cas9%20library%20screening%20approach%20for%20anti-cancer%20drug%20discovery:%20overview%20and%20perspectives.%20Theranostics.%202022;12(7):3329." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR109">
<span class="label">109.</span><cite>Ding X, Yin B, Qian L, Zeng Z, Yang Z, Li H, et al. Screening for novel quorum-sensing inhibitors to interfere with the formation of <em>Pseudomonas aeruginosa</em> biofilm. J Med Microbiol. 2011;60(12):1827–34.
</cite> [<a href="https://doi.org/10.1099/jmm.0.024166-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21852522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ding%20X,%20Yin%20B,%20Qian%20L,%20Zeng%20Z,%20Yang%20Z,%20Li%20H,%20et%20al.%20Screening%20for%20novel%20quorum-sensing%20inhibitors%20to%20interfere%20with%20the%20formation%20of%20Pseudomonas%20aeruginosa%20biofilm.%20J%20Med%20Microbiol.%202011;60(12):1827%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR110">
<span class="label">110.</span><cite>De Kievit T. Quorum sensing in Pseudomonas aeruginosa biofilms. Environ Microbiol. 2009;11(2):279–88.
</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2008.01792.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19196266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Kievit%20T.%20Quorum%20sensing%20in%20Pseudomonas%20aeruginosa%20biofilms.%20Environ%20Microbiol.%202009;11(2):279%E2%80%9388." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR111">
<span class="label">111.</span><cite>Ates A, Tastan C, Ermertcan S. CRISPR-Cas9-mediated targeting of multidrug resistance genes in methicillin-resistant <em>Staphylococcus aureus</em>. CRISPR J. 2024;7(6):374–84.
</cite> [<a href="https://doi.org/10.1089/crispr.2024.0001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39514329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ates%20A,%20Tastan%20C,%20Ermertcan%20S.%20CRISPR-Cas9-mediated%20targeting%20of%20multidrug%20resistance%20genes%20in%20methicillin-resistant%20Staphylococcus%20aureus.%20CRISPR%20J.%202024;7(6):374%E2%80%9384." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR112">
<span class="label">112.</span><cite>Agha ASA, Al-Samydai A, Aburjai T. New frontiers in CRISPR: Addressing antimicrobial resistance with Cas9, Cas12, Cas13, and Cas14. Heliyon. 2025;11:e42013.
</cite> [<a href="https://doi.org/10.1016/j.heliyon.2025.e42013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11791237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39906792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Agha%20ASA,%20Al-Samydai%20A,%20Aburjai%20T.%20New%20frontiers%20in%20CRISPR:%20Addressing%20antimicrobial%20resistance%20with%20Cas9,%20Cas12,%20Cas13,%20and%20Cas14.%20Heliyon.%202025;11:e42013." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR113">
<span class="label">113.</span><cite>Knott GJ, Doudna JA. CRISPR-Cas guides the future of genetic engineering. Science. 2018;361(6405):866–9.
</cite> [<a href="https://doi.org/10.1126/science.aat5011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6455913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30166482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Knott%20GJ,%20Doudna%20JA.%20CRISPR-Cas%20guides%20the%20future%20of%20genetic%20engineering.%20Science.%202018;361(6405):866%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR114">
<span class="label">114.</span><cite>Gholizadeh P, Aghazadeh M, Ghotaslou R, Rezaee MA, Pirzadeh T, Cui L, et al. Role of CRISPR-Cas system on antibiotic resistance patterns of Enterococcus faecalis. Ann Clin Microbiol Antimicrob. 2021;20:1–12.
</cite> [<a href="https://doi.org/10.1186/s12941-021-00455-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8317297/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34321002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gholizadeh%20P,%20Aghazadeh%20M,%20Ghotaslou%20R,%20Rezaee%20MA,%20Pirzadeh%20T,%20Cui%20L,%20et%20al.%20Role%20of%20CRISPR-Cas%20system%20on%20antibiotic%20resistance%20patterns%20of%20Enterococcus%20faecalis.%20Ann%20Clin%20Microbiol%20Antimicrob.%202021;20:1%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR115">
<span class="label">115.</span><cite>Abdul R, Wang M-R, Zhong C-J, Liu Y-Y, Hou W, Xiong H-R. An updated review on the antimicrobial and pharmacological properties of <em>Uncaria</em> (Rubiaceae). J Herbal Med. 2022;34:100573.</cite> [<a href="https://scholar.google.com/scholar_lookup?Abdul%20R,%20Wang%20M-R,%20Zhong%20C-J,%20Liu%20Y-Y,%20Hou%20W,%20Xiong%20H-R.%20An%20updated%20review%20on%20the%20antimicrobial%20and%20pharmacological%20properties%20of%20Uncaria%20(Rubiaceae).%20J%20Herbal%20Med.%202022;34:100573." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR116">
<span class="label">116.</span><cite>Zhang F, Wen Y, Guo X. CRISPR/Cas9 for genome editing: progress, implications and challenges. Hum Mol Genet. 2014;23(R1):R40–6.
</cite> [<a href="https://doi.org/10.1093/hmg/ddu125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24651067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20F,%20Wen%20Y,%20Guo%20X.%20CRISPR/Cas9%20for%20genome%20editing:%20progress,%20implications%20and%20challenges.%20Hum%20Mol%20Genet.%202014;23(R1):R40%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR117">
<span class="label">117.</span><cite>Wang S-W, Gao C, Zheng Y-M, Yi L, Lu J-C, Huang X-Y, et al. Current applications and future perspective of CRISPR/Cas9 gene editing in cancer. Mol Cancer. 2022;21(1):57.
</cite> [<a href="https://doi.org/10.1186/s12943-022-01518-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8862238/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35189910/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20S-W,%20Gao%20C,%20Zheng%20Y-M,%20Yi%20L,%20Lu%20J-C,%20Huang%20X-Y,%20et%20al.%20Current%20applications%20and%20future%20perspective%20of%20CRISPR/Cas9%20gene%20editing%20in%20cancer.%20Mol%20Cancer.%202022;21(1):57." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR118">
<span class="label">118.</span><cite>Savić N, Schwank G. Advances in therapeutic CRISPR/Cas9 genome editing. Transl Res. 2016;168:15–21.
</cite> [<a href="https://doi.org/10.1016/j.trsl.2015.09.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26470680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Savi%C4%87%20N,%20Schwank%20G.%20Advances%20in%20therapeutic%20CRISPR/Cas9%20genome%20editing.%20Transl%20Res.%202016;168:15%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR119">
<span class="label">119.</span><cite>Makarova KS, Wolf YI, Alkhnbashi OS, Costa F, Shah SA, Saunders SJ, et al. An updated evolutionary classification of CRISPR-Cas systems. Nat Rev Microbiol. 2015;13(11):722–36.
</cite> [<a href="https://doi.org/10.1038/nrmicro3569" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5426118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26411297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Makarova%20KS,%20Wolf%20YI,%20Alkhnbashi%20OS,%20Costa%20F,%20Shah%20SA,%20Saunders%20SJ,%20et%20al.%20An%20updated%20evolutionary%20classification%20of%20CRISPR-Cas%20systems.%20Nat%20Rev%20Microbiol.%202015;13(11):722%E2%80%9336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR120">
<span class="label">120.</span><cite>Makarova KS, Wolf YI, Iranzo J, Shmakov SA, Alkhnbashi OS, Brouns SJJ, et al. Evolutionary classification of CRISPR-Cas systems: a burst of class 2 and derived variants. Nat Rev Microbiol. 2020;18(2):67–83.
</cite> [<a href="https://doi.org/10.1038/s41579-019-0299-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8905525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31857715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Makarova%20KS,%20Wolf%20YI,%20Iranzo%20J,%20Shmakov%20SA,%20Alkhnbashi%20OS,%20Brouns%20SJJ,%20et%20al.%20Evolutionary%20classification%20of%20CRISPR-Cas%20systems:%20a%20burst%20of%20class%202%20and%20derived%20variants.%20Nat%20Rev%20Microbiol.%202020;18(2):67%E2%80%9383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR121">
<span class="label">121.</span><cite>Morisaka H, Yoshimi K, Okuzaki Y, Gee P, Kunihiro Y, Sonpho E, et al. Crispr-Cas3 induces broad and unidirectional genome editing in human cells. Nat Commun. 2019;10(1):5302.
</cite> [<a href="https://doi.org/10.1038/s41467-019-13226-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6897959/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31811138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morisaka%20H,%20Yoshimi%20K,%20Okuzaki%20Y,%20Gee%20P,%20Kunihiro%20Y,%20Sonpho%20E,%20et%20al.%20Crispr-Cas3%20induces%20broad%20and%20unidirectional%20genome%20editing%20in%20human%20cells.%20Nat%20Commun.%202019;10(1):5302." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR122">
<span class="label">122.</span><cite>Rostøl JT, Xie W, Kuryavyi V, Maguin P, Kao K, Froom R, et al. The Card1 nuclease provides defence during type III CRISPR immunity. Nature. 2021;590(7847):624–9.
</cite> [<a href="https://doi.org/10.1038/s41586-021-03206-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7906951/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33461211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rost%C3%B8l%20JT,%20Xie%20W,%20Kuryavyi%20V,%20Maguin%20P,%20Kao%20K,%20Froom%20R,%20et%20al.%20The%20Card1%20nuclease%20provides%20defence%20during%20type%20III%20CRISPR%20immunity.%20Nature.%202021;590(7847):624%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR123">
<span class="label">123.</span><cite>Lin J, Fuglsang A, Kjeldsen AL, Sun K, Bhoobalan-Chitty Y, Peng X. DNA targeting by subtype I-D CRISPR-Cas shows type I and type III features. Nucleic Acids Res. 2020;48(18):10470–8.
</cite> [<a href="https://doi.org/10.1093/nar/gkaa749" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7544194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32960267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20J,%20Fuglsang%20A,%20Kjeldsen%20AL,%20Sun%20K,%20Bhoobalan-Chitty%20Y,%20Peng%20X.%20DNA%20targeting%20by%20subtype%20I-D%20CRISPR-Cas%20shows%20type%20I%20and%20type%20III%20features.%20Nucleic%20Acids%20Res.%202020;48(18):10470%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR124">
<span class="label">124.</span><cite>Jackson RN, Golden SM, van Erp PB, Carter J, Westra ER, Brouns SJ, et al. Structural biology. Crystal structure of the CRISPR RNA-guided surveillance complex from Escherichia coli. Science. 2014;345(6203):1473–9.
</cite> [<a href="https://doi.org/10.1126/science.1256328" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4188430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25103409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jackson%20RN,%20Golden%20SM,%20van%20Erp%20PB,%20Carter%20J,%20Westra%20ER,%20Brouns%20SJ,%20et%20al.%20Structural%20biology.%20Crystal%20structure%20of%20the%20CRISPR%20RNA-guided%20surveillance%20complex%20from%20Escherichia%20coli.%20Science.%202014;345(6203):1473%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR125">
<span class="label">125.</span><cite>Karneyeva K, Kolesnik M, Livenskyi A, Zgoda V, Zubarev V, Trofimova A, et al. Interference requirements of type III CRISPR-Cas systems from <em>Thermus thermophilus</em>. J Mol Biol. 2024;436(6):168448.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2024.168448" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38266982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karneyeva%20K,%20Kolesnik%20M,%20Livenskyi%20A,%20Zgoda%20V,%20Zubarev%20V,%20Trofimova%20A,%20et%20al.%20Interference%20requirements%20of%20type%20III%20CRISPR-Cas%20systems%20from%20Thermus%20thermophilus.%20J%20Mol%20Biol.%202024;436(6):168448." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR126">
<span class="label">126.</span><cite>Rostøl JT, Marraffini LA. Non-specific degradation of transcripts promotes plasmid clearance during type III-A CRISPR-Cas immunity. Nat Microbiol. 2019;4(4):656–62.
</cite> [<a href="https://doi.org/10.1038/s41564-018-0353-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6430669/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30692669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rost%C3%B8l%20JT,%20Marraffini%20LA.%20Non-specific%20degradation%20of%20transcripts%20promotes%20plasmid%20clearance%20during%20type%20III-A%20CRISPR-Cas%20immunity.%20Nat%20Microbiol.%202019;4(4):656%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR127">
<span class="label">127.</span><cite>Koonin EV, Makarova KS. Origins and evolution of CRISPR-Cas systems. Philos Trans R Soc Lond B Biol Sci. 2019;374(1772):20180087.
</cite> [<a href="https://doi.org/10.1098/rstb.2018.0087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6452270/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30905284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koonin%20EV,%20Makarova%20KS.%20Origins%20and%20evolution%20of%20CRISPR-Cas%20systems.%20Philos%20Trans%20R%20Soc%20Lond%20B%20Biol%20Sci.%202019;374(1772):20180087." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR128">
<span class="label">128.</span><cite>Moya-Beltrán A, Makarova KS, Acuña LG, Wolf YI, Covarrubias PC, Shmakov SA, et al. Evolution of type IV CRISPR-Cas systems: insights from CRISPR loci in integrative conjugative elements of Acidithiobacillia. Crispr j. 2021;4(5):656–72.
</cite> [<a href="https://doi.org/10.1089/crispr.2021.0051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8658065/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34582696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moya-Beltr%C3%A1n%20A,%20Makarova%20KS,%20Acu%C3%B1a%20LG,%20Wolf%20YI,%20Covarrubias%20PC,%20Shmakov%20SA,%20et%20al.%20Evolution%20of%20type%20IV%20CRISPR-Cas%20systems:%20insights%20from%20CRISPR%20loci%20in%20integrative%20conjugative%20elements%20of%20Acidithiobacillia.%20Crispr%20j.%202021;4(5):656%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR129">
<span class="label">129.</span><cite>Pinilla-Redondo R, Mayo-Muñoz D, Russel J, Garrett RA, Randau L, Sørensen SJ, et al. Type IV CRISPR-Cas systems are highly diverse and involved in competition between plasmids. Nucleic Acids Res. 2020;48(4):2000–12.
</cite> [<a href="https://doi.org/10.1093/nar/gkz1197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7038947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31879772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pinilla-Redondo%20R,%20Mayo-Mu%C3%B1oz%20D,%20Russel%20J,%20Garrett%20RA,%20Randau%20L,%20S%C3%B8rensen%20SJ,%20et%20al.%20Type%20IV%20CRISPR-Cas%20systems%20are%20highly%20diverse%20and%20involved%20in%20competition%20between%20plasmids.%20Nucleic%20Acids%20Res.%202020;48(4):2000%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR130">
<span class="label">130.</span><cite>Faure G, Makarova KS, Koonin EV. CRISPR-Cas: complex functional networks and multiple roles beyond adaptive immunity. J Mol Biol. 2019;431(1):3–20.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2018.08.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30193985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Faure%20G,%20Makarova%20KS,%20Koonin%20EV.%20CRISPR-Cas:%20complex%20functional%20networks%20and%20multiple%20roles%20beyond%20adaptive%20immunity.%20J%20Mol%20Biol.%202019;431(1):3%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR131">
<span class="label">131.</span><cite>Jiang F, Doudna JA. CRISPR-Cas9 structures and mechanisms. Annu Rev Biophys. 2017;46:505–29.
</cite> [<a href="https://doi.org/10.1146/annurev-biophys-062215-010822" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28375731/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20F,%20Doudna%20JA.%20CRISPR-Cas9%20structures%20and%20mechanisms.%20Annu%20Rev%20Biophys.%202017;46:505%E2%80%9329." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR132">
<span class="label">132.</span><cite>Shmakov S, Smargon A, Scott D, Cox D, Pyzocha N, Yan W, et al. Diversity and evolution of class 2 CRISPR-Cas systems. Nat Rev Microbiol. 2017;15(3):169–82.
</cite> [<a href="https://doi.org/10.1038/nrmicro.2016.184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5851899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28111461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shmakov%20S,%20Smargon%20A,%20Scott%20D,%20Cox%20D,%20Pyzocha%20N,%20Yan%20W,%20et%20al.%20Diversity%20and%20evolution%20of%20class%202%20CRISPR-Cas%20systems.%20Nat%20Rev%20Microbiol.%202017;15(3):169%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR133">
<span class="label">133.</span><cite>Yosef I, Goren MG, Qimron U. Proteins and DNA elements essential for the CRISPR adaptation process in <em>Escherichia coli</em>. Nucleic Acids Res. 2012;40(12):5569–76.
</cite> [<a href="https://doi.org/10.1093/nar/gks216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3384332/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22402487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yosef%20I,%20Goren%20MG,%20Qimron%20U.%20Proteins%20and%20DNA%20elements%20essential%20for%20the%20CRISPR%20adaptation%20process%20in%20Escherichia%20coli.%20Nucleic%20Acids%20Res.%202012;40(12):5569%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR134">
<span class="label">134.</span><cite>Bikard D, Euler CW, Jiang W, Nussenzweig PM, Goldberg GW, Duportet X, et al. Exploiting CRISPR-Cas nucleases to produce sequence-specific antimicrobials. Nat Biotechnol. 2014;32(11):1146–50.
</cite> [<a href="https://doi.org/10.1038/nbt.3043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4317352/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25282355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bikard%20D,%20Euler%20CW,%20Jiang%20W,%20Nussenzweig%20PM,%20Goldberg%20GW,%20Duportet%20X,%20et%20al.%20Exploiting%20CRISPR-Cas%20nucleases%20to%20produce%20sequence-specific%20antimicrobials.%20Nat%20Biotechnol.%202014;32(11):1146%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR135">
<span class="label">135.</span><cite>Doudna JA, Charpentier E. Genome editing. The new frontier of genome engineering with CRISPR-Cas9. Science. 2014;346(6213):1258096.
</cite> [<a href="https://doi.org/10.1126/science.1258096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25430774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Doudna%20JA,%20Charpentier%20E.%20Genome%20editing.%20The%20new%20frontier%20of%20genome%20engineering%20with%20CRISPR-Cas9.%20Science.%202014;346(6213):1258096." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR136">
<span class="label">136.</span><cite>Ledford H. Major CRISPR patent decision won’t end tangled dispute. Nature. 2022;603(7901):373–4.
</cite> [<a href="https://doi.org/10.1038/d41586-022-00629-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35264807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ledford%20H.%20Major%20CRISPR%20patent%20decision%20won%E2%80%99t%20end%20tangled%20dispute.%20Nature.%202022;603(7901):373%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR137">
<span class="label">137.</span><cite>Koonin EV, Makarova KS, Zhang F. Diversity, classification and evolution of CRISPR-Cas systems. Curr Opin Microbiol. 2017;37:67–78.
</cite> [<a href="https://doi.org/10.1016/j.mib.2017.05.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5776717/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28605718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koonin%20EV,%20Makarova%20KS,%20Zhang%20F.%20Diversity,%20classification%20and%20evolution%20of%20CRISPR-Cas%20systems.%20Curr%20Opin%20Microbiol.%202017;37:67%E2%80%9378." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR138">
<span class="label">138.</span><cite>Yan WX, Hunnewell P, Alfonse LE, Carte JM, Keston-Smith E, Sothiselvam S, et al. Functionally diverse type V CRISPR-Cas systems. Science. 2019;363(6422):88–91.
</cite> [<a href="https://doi.org/10.1126/science.aav7271" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11258546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30523077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan%20WX,%20Hunnewell%20P,%20Alfonse%20LE,%20Carte%20JM,%20Keston-Smith%20E,%20Sothiselvam%20S,%20et%20al.%20Functionally%20diverse%20type%20V%20CRISPR-Cas%20systems.%20Science.%202019;363(6422):88%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR139">
<span class="label">139.</span><cite>Chen JS, Ma E, Harrington LB, Da Costa M, Tian X, Palefsky JM, et al. CRISPR-Cas12a target binding unleashes indiscriminate single-stranded DNase activity. Science. 2018;360(6387):436–9.
</cite> [<a href="https://doi.org/10.1126/science.aar6245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6628903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29449511/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20JS,%20Ma%20E,%20Harrington%20LB,%20Da%20Costa%20M,%20Tian%20X,%20Palefsky%20JM,%20et%20al.%20CRISPR-Cas12a%20target%20binding%20unleashes%20indiscriminate%20single-stranded%20DNase%20activity.%20Science.%202018;360(6387):436%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR140">
<span class="label">140.</span><cite>Li SY, Cheng QX, Wang JM, Li XY, Zhang ZL, Gao S, et al. CRISPR-Cas12a-assisted nucleic acid detection. Cell Discov. 2018;4:20.
</cite> [<a href="https://doi.org/10.1038/s41421-018-0028-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5913299/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29707234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20SY,%20Cheng%20QX,%20Wang%20JM,%20Li%20XY,%20Zhang%20ZL,%20Gao%20S,%20et%20al.%20CRISPR-Cas12a-assisted%20nucleic%20acid%20detection.%20Cell%20Discov.%202018;4:20." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR141">
<span class="label">141.</span><cite>Li L, Li S, Wu N, Wu J, Wang G, Zhao G, et al. HOLMESv2: a CRISPR-Cas12b-assisted platform for nucleic acid detection and DNA methylation quantitation. ACS Synth Biol. 2019;8(10):2228–37.
</cite> [<a href="https://doi.org/10.1021/acssynbio.9b00209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31532637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20L,%20Li%20S,%20Wu%20N,%20Wu%20J,%20Wang%20G,%20Zhao%20G,%20et%20al.%20HOLMESv2:%20a%20CRISPR-Cas12b-assisted%20platform%20for%20nucleic%20acid%20detection%20and%20DNA%20methylation%20quantitation.%20ACS%20Synth%20Biol.%202019;8(10):2228%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR142">
<span class="label">142.</span><cite>Broughton JP, Deng X, Yu G, Fasching CL, Servellita V, Singh J, et al. CRISPR-Cas12-based detection of SARS-CoV-2. Nat Biotechnol. 2020;38(7):870–4.
</cite> [<a href="https://doi.org/10.1038/s41587-020-0513-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9107629/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32300245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Broughton%20JP,%20Deng%20X,%20Yu%20G,%20Fasching%20CL,%20Servellita%20V,%20Singh%20J,%20et%20al.%20CRISPR-Cas12-based%20detection%20of%20SARS-CoV-2.%20Nat%20Biotechnol.%202020;38(7):870%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR143">
<span class="label">143.</span><cite>Rananaware SR, Meister KS, Shoemaker GM, Vesco EK, Sandoval LSW, Lewis JG, et al. PAM-free diagnostics with diverse type V CRISPR-Cas systems. medRxiv. 2024.</cite>
</li>
<li id="CR144">
<span class="label">144.</span><cite>Kordyś M, Sen R, Warkocki Z. Applications of the versatile CRISPR-Cas13 RNA targeting system. WIREs RNA. 2022;13(3):e1694.
</cite> [<a href="https://doi.org/10.1002/wrna.1694" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34553495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kordy%C5%9B%20M,%20Sen%20R,%20Warkocki%20Z.%20Applications%20of%20the%20versatile%20CRISPR-Cas13%20RNA%20targeting%20system.%20WIREs%20RNA.%202022;13(3):e1694." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR145">
<span class="label">145.</span><cite>O’Connell MR. Molecular mechanisms of RNA targeting by Cas13-containing type VI CRISPR-Cas systems. J Mol Biol. 2019;431(1):66–87.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2018.06.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29940185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?O%E2%80%99Connell%20MR.%20Molecular%20mechanisms%20of%20RNA%20targeting%20by%20Cas13-containing%20type%20VI%20CRISPR-Cas%20systems.%20J%20Mol%20Biol.%202019;431(1):66%E2%80%9387." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR146">
<span class="label">146.</span><cite>Abudayyeh OO, Gootenberg JS, Konermann S, Joung J, Slaymaker IM, Cox DB, et al. C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector. Science. 2016;353(6299):aaf5573.
</cite> [<a href="https://doi.org/10.1126/science.aaf5573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5127784/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27256883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abudayyeh%20OO,%20Gootenberg%20JS,%20Konermann%20S,%20Joung%20J,%20Slaymaker%20IM,%20Cox%20DB,%20et%20al.%20C2c2%20is%20a%20single-component%20programmable%20RNA-guided%20RNA-targeting%20CRISPR%20effector.%20Science.%202016;353(6299):aaf5573." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR147">
<span class="label">147.</span><cite>Gootenberg JS, Abudayyeh OO, Kellner MJ, Joung J, Collins JJ, Zhang F. Multiplexed and portable nucleic acid detection platform with Cas13, Cas12a, and Csm6. Science. 2018;360(6387):439–44.
</cite> [<a href="https://doi.org/10.1126/science.aaq0179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5961727/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29449508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gootenberg%20JS,%20Abudayyeh%20OO,%20Kellner%20MJ,%20Joung%20J,%20Collins%20JJ,%20Zhang%20F.%20Multiplexed%20and%20portable%20nucleic%20acid%20detection%20platform%20with%20Cas13,%20Cas12a,%20and%20Csm6.%20Science.%202018;360(6387):439%E2%80%9344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR148">
<span class="label">148.</span><cite>Abudayyeh OO, Gootenberg JS, Essletzbichler P, Han S, Joung J, Belanto JJ, et al. RNA targeting with CRISPR-Cas13. Nature. 2017;550(7675):280–4.
</cite> [<a href="https://doi.org/10.1038/nature24049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5706658/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28976959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abudayyeh%20OO,%20Gootenberg%20JS,%20Essletzbichler%20P,%20Han%20S,%20Joung%20J,%20Belanto%20JJ,%20et%20al.%20RNA%20targeting%20with%20CRISPR-Cas13.%20Nature.%202017;550(7675):280%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR149">
<span class="label">149.</span><cite>Kellner MJ, Koob JG, Gootenberg JS, Abudayyeh OO, Zhang F. SHERLOCK: nucleic acid detection with CRISPR nucleases. Nat Protoc. 2019;14(10):2986–3012.
</cite> [<a href="https://doi.org/10.1038/s41596-019-0210-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6956564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31548639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kellner%20MJ,%20Koob%20JG,%20Gootenberg%20JS,%20Abudayyeh%20OO,%20Zhang%20F.%20SHERLOCK:%20nucleic%20acid%20detection%20with%20CRISPR%20nucleases.%20Nat%20Protoc.%202019;14(10):2986%E2%80%933012." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR150">
<span class="label">150.</span><cite>Altae-Tran H, Kannan S, Suberski AJ, Mears KS, Demircioglu FE, Moeller L, et al. Uncovering the functional diversity of rare CRISPR-Cas systems with deep terascale clustering. Science. 2023;382(6673):eadi1910.
</cite> [<a href="https://doi.org/10.1126/science.adi1910" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10910872/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37995242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Altae-Tran%20H,%20Kannan%20S,%20Suberski%20AJ,%20Mears%20KS,%20Demircioglu%20FE,%20Moeller%20L,%20et%20al.%20Uncovering%20the%20functional%20diversity%20of%20rare%20CRISPR-Cas%20systems%20with%20deep%20terascale%20clustering.%20Science.%202023;382(6673):eadi1910." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR151">
<span class="label">151.</span><cite>Smargon AA, Cox DBT, Pyzocha NK, Zheng K, Slaymaker IM, Gootenberg JS, et al. Cas13b Is a Type VI-B CRISPR-Associated RNA-Guided RNase Differentially Regulated by Accessory Proteins Csx27 and Csx28. Mol Cell. 2017;65(4):618-30.e7.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2016.12.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5432119/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28065598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smargon%20AA,%20Cox%20DBT,%20Pyzocha%20NK,%20Zheng%20K,%20Slaymaker%20IM,%20Gootenberg%20JS,%20et%20al.%20Cas13b%20Is%20a%20Type%20VI-B%20CRISPR-Associated%20RNA-Guided%20RNase%20Differentially%20Regulated%20by%20Accessory%20Proteins%20Csx27%20and%20Csx28.%20Mol%20Cell.%202017;65(4):618-30.e7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR152">
<span class="label">152.</span><cite>Hong T, Luo Q. Advances in the RNA-targeting CRISPR-Cas systems. Sheng Wu Gong Cheng Xue Bao. 2023;39(4):1363–73.
</cite> [<a href="https://doi.org/10.13345/j.cjb.220633" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37154311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hong%20T,%20Luo%20Q.%20Advances%20in%20the%20RNA-targeting%20CRISPR-Cas%20systems.%20Sheng%20Wu%20Gong%20Cheng%20Xue%20Bao.%202023;39(4):1363%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR153">
<span class="label">153.</span><cite>Perčulija V, Lin J, Zhang B, Ouyang S. Functional features and current applications of the RNA-targeting type VI CRISPR-Cas systems. Adv Sci. 2021;8(13):2004685.</cite> [<a href="https://doi.org/10.1002/advs.202004685" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8209922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34254038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Per%C4%8Dulija%20V,%20Lin%20J,%20Zhang%20B,%20Ouyang%20S.%20Functional%20features%20and%20current%20applications%20of%20the%20RNA-targeting%20type%20VI%20CRISPR-Cas%20systems.%20Adv%20Sci.%202021;8(13):2004685." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR154">
<span class="label">154.</span><cite>Yan WX, Chong S, Zhang H, Makarova KS, Koonin EV, Cheng DR, et al. Cas13d is a compact RNA-targeting type VI CRISPR effector positively modulated by a WYL-domain-containing accessory protein. Mol Cell. 2018;70(2):327-39.e5.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2018.02.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5935466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29551514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan%20WX,%20Chong%20S,%20Zhang%20H,%20Makarova%20KS,%20Koonin%20EV,%20Cheng%20DR,%20et%20al.%20Cas13d%20is%20a%20compact%20RNA-targeting%20type%20VI%20CRISPR%20effector%20positively%20modulated%20by%20a%20WYL-domain-containing%20accessory%20protein.%20Mol%20Cell.%202018;70(2):327-39.e5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR155">
<span class="label">155.</span><cite>Pinto-Alphandary H, Andremont A, Couvreur P. Targeted delivery of antibiotics using liposomes and nanoparticles: research and applications. Int J Antimicrob Agents. 2000;13(3):155–68.
</cite> [<a href="https://doi.org/10.1016/s0924-8579(99)00121-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10724019/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pinto-Alphandary%20H,%20Andremont%20A,%20Couvreur%20P.%20Targeted%20delivery%20of%20antibiotics%20using%20liposomes%20and%20nanoparticles:%20research%20and%20applications.%20Int%20J%20Antimicrob%20Agents.%202000;13(3):155%E2%80%9368." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR156">
<span class="label">156.</span><cite>Tyagi P, Wu P-C, Chancellor M, Yoshimura N, Huang L. Recent advances in intravesical drug/gene delivery. Mol Pharm. 2006;3(4):369–79.
</cite> [<a href="https://doi.org/10.1021/mp060001j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2504416/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16889430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tyagi%20P,%20Wu%20P-C,%20Chancellor%20M,%20Yoshimura%20N,%20Huang%20L.%20Recent%20advances%20in%20intravesical%20drug/gene%20delivery.%20Mol%20Pharm.%202006;3(4):369%E2%80%9379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR157">
<span class="label">157.</span><cite>Yang Y, Wang D, Lü P, Ma S, Chen K. Research progress on nucleic acid detection and genome editing of CRISPR/Cas12 system. Mol Biol Rep. 2023;50(4):3723–38.
</cite> [<a href="https://doi.org/10.1007/s11033-023-08240-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9843688/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36648696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20Y,%20Wang%20D,%20L%C3%BC%20P,%20Ma%20S,%20Chen%20K.%20Research%20progress%20on%20nucleic%20acid%20detection%20and%20genome%20editing%20of%20CRISPR/Cas12%20system.%20Mol%20Biol%20Rep.%202023;50(4):3723%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR158">
<span class="label">158.</span><cite>Doudna JA, Charpentier E. The new frontier of genome engineering with CRISPR-Cas9. Science. 2014;346(6213):1258096.
</cite> [<a href="https://doi.org/10.1126/science.1258096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25430774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Doudna%20JA,%20Charpentier%20E.%20The%20new%20frontier%20of%20genome%20engineering%20with%20CRISPR-Cas9.%20Science.%202014;346(6213):1258096." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>No datasets were generated or analysed during the current study.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Medicine are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12916-025-04323-4"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12916_2025_Article_4323.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366227/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366227/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366227%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366227/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366227/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366227/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830872/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366227/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830872/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366227/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366227/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="mABkJYgNpcSiVN6nh2W2MfdjS15ILqBphFxtStUX6ueSW2RU3IHzhVtutol3XXmt">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
