
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Leveraging sequences missing from the human genome to diagnose cancer - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532D6D8AF20503052D6D00439FCDFE.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="commsmed">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371106/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Communications Medicine">
<meta name="citation_title" content="Leveraging sequences missing from the human genome to diagnose cancer">
<meta name="citation_author" content="Ilias Georgakopoulos-Soares">
<meta name="citation_author_institution" content="Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Institute for Personalized Medicine, Department of Molecular and Precision Medicine, The Pennsylvania State University College of Medicine, Hershey, PA USA">
<meta name="citation_author" content="Ofer Yizhar-Barnea">
<meta name="citation_author_institution" content="Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Ioannis Mouratidis">
<meta name="citation_author_institution" content="Institute for Personalized Medicine, Department of Molecular and Precision Medicine, The Pennsylvania State University College of Medicine, Hershey, PA USA">
<meta name="citation_author_institution" content="Huck Institutes of the Life Sciences, Penn State University, University Park, PA USA">
<meta name="citation_author" content="Candace S Y Chan">
<meta name="citation_author_institution" content="Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Michail Patsakis">
<meta name="citation_author_institution" content="Institute for Personalized Medicine, Department of Molecular and Precision Medicine, The Pennsylvania State University College of Medicine, Hershey, PA USA">
<meta name="citation_author" content="Akshatha Nayak">
<meta name="citation_author_institution" content="Institute for Personalized Medicine, Department of Molecular and Precision Medicine, The Pennsylvania State University College of Medicine, Hershey, PA USA">
<meta name="citation_author" content="Rachael Bradley">
<meta name="citation_author_institution" content="Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Mayank Mahajan">
<meta name="citation_author_institution" content="The Gene Lay Institute of Immunology and Inflammation, Brigham and Women’s Hospital, Massachusetts General Hospital and Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Jasmine Sims">
<meta name="citation_author_institution" content="Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Dianne Laboy Cintron">
<meta name="citation_author_institution" content="Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Ryder Easterlin">
<meta name="citation_author_institution" content="Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Julia S Kim">
<meta name="citation_author_institution" content="Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Emmalyn Chen">
<meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Geovanni Pineda">
<meta name="citation_author_institution" content="Division of Gynecologic Oncology, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Guillermo E Parada">
<meta name="citation_author_institution" content="Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Toronto, ON Canada">
<meta name="citation_author" content="John S Witte">
<meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Department of Epidemiology and Population Health, Stanford University, Stanford, CA USA">
<meta name="citation_author" content="Christopher A Maher">
<meta name="citation_author_institution" content="Division of Oncology, Department of Internal Medicine, Washington University School of Medicine, St. Louis, MO USA">
<meta name="citation_author_institution" content="Siteman Cancer Center, Washington University School of Medicine, St. Louis, MO USA">
<meta name="citation_author_institution" content="Department of Biomedical Engineering, Washington University School of Medicine, St. Louis, MO USA">
<meta name="citation_author" content="Felix Feng">
<meta name="citation_author_institution" content="Division of Hematology/Oncology, Department of Medicine, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Helen Diller Comprehensive Cancer Center, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Department of Radiation Oncology, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Department of Urology, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Ioannis Vathiotis">
<meta name="citation_author_institution" content="Third Department of Internal Medicine, Sotiria Hospital, National and Kapodistrian University of Athens, School of Medicine, Athens, Greece">
<meta name="citation_author_institution" content="Department of Pathology, Yale School of Medicine, New Haven, CT USA">
<meta name="citation_author" content="Nikolaos Syrigos">
<meta name="citation_author_institution" content="Third Department of Internal Medicine, Sotiria Hospital, National and Kapodistrian University of Athens, School of Medicine, Athens, Greece">
<meta name="citation_author_institution" content="Breast Oncology, Dana-Farber Brigham Cancer Center, Boston, MA USA">
<meta name="citation_author" content="Emmanouil Panagiotou">
<meta name="citation_author_institution" content="Third Department of Internal Medicine, Sotiria Hospital, National and Kapodistrian University of Athens, School of Medicine, Athens, Greece">
<meta name="citation_author" content="Andriani Charpidou">
<meta name="citation_author_institution" content="Third Department of Internal Medicine, Sotiria Hospital, National and Kapodistrian University of Athens, School of Medicine, Athens, Greece">
<meta name="citation_author" content="Konstantinos Syrigos">
<meta name="citation_author_institution" content="Third Department of Internal Medicine, Sotiria Hospital, National and Kapodistrian University of Athens, School of Medicine, Athens, Greece">
<meta name="citation_author" content="Jocelyn Chapman">
<meta name="citation_author_institution" content="Helen Diller Comprehensive Cancer Center, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Department of Urology, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Mark Kvale">
<meta name="citation_author_institution" content="Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Kaiser Permanente, Oakland, CA USA">
<meta name="citation_author" content="Martin Hemberg">
<meta name="citation_author_institution" content="The Gene Lay Institute of Immunology and Inflammation, Brigham and Women’s Hospital, Massachusetts General Hospital and Harvard Medical School, Boston, MA USA">
<meta name="citation_author_institution" content="Wellcome Sanger Institute, Hinxton, Cambridge, UK">
<meta name="citation_author" content="Nadav Ahituv">
<meta name="citation_author_institution" content="Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Helen Diller Family Comprehensive Cancer Center, University of California San Francisco, San Francisco, CA USA">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="5">
<meta name="citation_firstpage" content="363">
<meta name="citation_doi" content="10.1038/s43856-025-01067-3">
<meta name="citation_pmid" content="40841759">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371106/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371106/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371106/pdf/43856_2025_Article_1067.pdf">
<meta name="description" content="Cancer diagnosis using cell-free DNA (cfDNA) has the potential to improve treatment and survival but has several technical limitations. In this study, we developed a prediction model based on neomers, DNA sequences 13–17 nucleotides in length that ...">
<meta name="og:title" content="Leveraging sequences missing from the human genome to diagnose cancer">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Cancer diagnosis using cell-free DNA (cfDNA) has the potential to improve treatment and survival but has several technical limitations. In this study, we developed a prediction model based on neomers, DNA sequences 13–17 nucleotides in length that ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371106/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371106">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s43856-025-01067-3"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/43856_2025_Article_1067.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371106%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371106/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371106/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371106/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-commsmed.jpg" alt="Communications Medicine logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Communications Medicine" title="Link to Communications Medicine" shape="default" href="https://www.nature.com/commsmed/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Commun Med (Lond)</button></div>. 2025 Aug 21;5:363. doi: <a href="https://doi.org/10.1038/s43856-025-01067-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s43856-025-01067-3</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Commun%20Med%20(Lond)%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Commun%20Med%20(Lond)%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Commun%20Med%20(Lond)%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Commun%20Med%20(Lond)%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Leveraging sequences missing from the human genome to diagnose cancer</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Georgakopoulos-Soares%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Ilias Georgakopoulos-Soares</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Ilias Georgakopoulos-Soares</span></h3>
<div class="p">
<sup>1</sup>Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>2</sup>Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>3</sup>Institute for Personalized Medicine, Department of Molecular and Precision Medicine, The Pennsylvania State University College of Medicine, Hershey, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Georgakopoulos-Soares%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ilias Georgakopoulos-Soares</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yizhar-Barnea%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ofer Yizhar-Barnea</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ofer Yizhar-Barnea</span></h3>
<div class="p">
<sup>1</sup>Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>2</sup>Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yizhar-Barnea%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Ofer Yizhar-Barnea</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mouratidis%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Ioannis Mouratidis</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Ioannis Mouratidis</span></h3>
<div class="p">
<sup>3</sup>Institute for Personalized Medicine, Department of Molecular and Precision Medicine, The Pennsylvania State University College of Medicine, Hershey, PA USA </div>
<div class="p">
<sup>4</sup>Huck Institutes of the Life Sciences, Penn State University, University Park, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mouratidis%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ioannis Mouratidis</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chan%20CSY%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Candace S Y Chan</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Candace S Y Chan</span></h3>
<div class="p">
<sup>1</sup>Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>2</sup>Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chan%20CSY%22%5BAuthor%5D" class="usa-link"><span class="name western">Candace S Y Chan</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Patsakis%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Michail Patsakis</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Michail Patsakis</span></h3>
<div class="p">
<sup>3</sup>Institute for Personalized Medicine, Department of Molecular and Precision Medicine, The Pennsylvania State University College of Medicine, Hershey, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Patsakis%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michail Patsakis</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nayak%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Akshatha Nayak</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Akshatha Nayak</span></h3>
<div class="p">
<sup>3</sup>Institute for Personalized Medicine, Department of Molecular and Precision Medicine, The Pennsylvania State University College of Medicine, Hershey, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nayak%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Akshatha Nayak</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bradley%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Rachael Bradley</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Rachael Bradley</span></h3>
<div class="p">
<sup>1</sup>Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>2</sup>Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bradley%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rachael Bradley</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mahajan%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Mayank Mahajan</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Mayank Mahajan</span></h3>
<div class="p">
<sup>5</sup>The Gene Lay Institute of Immunology and Inflammation, Brigham and Women’s Hospital, Massachusetts General Hospital and Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mahajan%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mayank Mahajan</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sims%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Jasmine Sims</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Jasmine Sims</span></h3>
<div class="p">
<sup>1</sup>Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>2</sup>Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sims%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jasmine Sims</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cintron%20DL%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Dianne Laboy Cintron</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Dianne Laboy Cintron</span></h3>
<div class="p">
<sup>1</sup>Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>2</sup>Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cintron%20DL%22%5BAuthor%5D" class="usa-link"><span class="name western">Dianne Laboy Cintron</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Easterlin%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Ryder Easterlin</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Ryder Easterlin</span></h3>
<div class="p">
<sup>1</sup>Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>2</sup>Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Easterlin%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ryder Easterlin</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20JS%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Julia S Kim</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Julia S Kim</span></h3>
<div class="p">
<sup>1</sup>Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>2</sup>Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20JS%22%5BAuthor%5D" class="usa-link"><span class="name western">Julia S Kim</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Emmalyn Chen</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Emmalyn Chen</span></h3>
<div class="p">
<sup>6</sup>Department of Epidemiology and Biostatistics, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Emmalyn Chen</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pineda%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Geovanni Pineda</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Geovanni Pineda</span></h3>
<div class="p">
<sup>7</sup>Division of Gynecologic Oncology, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pineda%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Geovanni Pineda</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Parada%20GE%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Guillermo E Parada</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Guillermo E Parada</span></h3>
<div class="p">
<sup>8</sup>Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Toronto, ON Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Parada%20GE%22%5BAuthor%5D" class="usa-link"><span class="name western">Guillermo E Parada</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Witte%20JS%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">John S Witte</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">John S Witte</span></h3>
<div class="p">
<sup>6</sup>Department of Epidemiology and Biostatistics, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>9</sup>Department of Epidemiology and Population Health, Stanford University, Stanford, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Witte%20JS%22%5BAuthor%5D" class="usa-link"><span class="name western">John S Witte</span></a>
</div>
</div>
<sup>6,</sup><sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maher%20CA%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Christopher A Maher</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Christopher A Maher</span></h3>
<div class="p">
<sup>10</sup>Division of Oncology, Department of Internal Medicine, Washington University School of Medicine, St. Louis, MO USA </div>
<div class="p">
<sup>11</sup>Siteman Cancer Center, Washington University School of Medicine, St. Louis, MO USA </div>
<div class="p">
<sup>12</sup>Department of Biomedical Engineering, Washington University School of Medicine, St. Louis, MO USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maher%20CA%22%5BAuthor%5D" class="usa-link"><span class="name western">Christopher A Maher</span></a>
</div>
</div>
<sup>10,</sup><sup>11,</sup><sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feng%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Felix Feng</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Felix Feng</span></h3>
<div class="p">
<sup>13</sup>Division of Hematology/Oncology, Department of Medicine, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>14</sup>Helen Diller Comprehensive Cancer Center, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>15</sup>Department of Radiation Oncology, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>16</sup>Department of Urology, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feng%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Felix Feng</span></a>
</div>
</div>
<sup>13,</sup><sup>14,</sup><sup>15,</sup><sup>16</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vathiotis%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Ioannis Vathiotis</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Ioannis Vathiotis</span></h3>
<div class="p">
<sup>17</sup>Third Department of Internal Medicine, Sotiria Hospital, National and Kapodistrian University of Athens, School of Medicine, Athens, Greece </div>
<div class="p">
<sup>18</sup>Department of Pathology, Yale School of Medicine, New Haven, CT USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vathiotis%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ioannis Vathiotis</span></a>
</div>
</div>
<sup>17,</sup><sup>18</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Syrigos%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Nikolaos Syrigos</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Nikolaos Syrigos</span></h3>
<div class="p">
<sup>17</sup>Third Department of Internal Medicine, Sotiria Hospital, National and Kapodistrian University of Athens, School of Medicine, Athens, Greece </div>
<div class="p">
<sup>19</sup>Breast Oncology, Dana-Farber Brigham Cancer Center, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Syrigos%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nikolaos Syrigos</span></a>
</div>
</div>
<sup>17,</sup><sup>19</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Panagiotou%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Emmanouil Panagiotou</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Emmanouil Panagiotou</span></h3>
<div class="p">
<sup>17</sup>Third Department of Internal Medicine, Sotiria Hospital, National and Kapodistrian University of Athens, School of Medicine, Athens, Greece </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Panagiotou%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Emmanouil Panagiotou</span></a>
</div>
</div>
<sup>17</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Charpidou%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id22"><span class="name western">Andriani Charpidou</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Andriani Charpidou</span></h3>
<div class="p">
<sup>17</sup>Third Department of Internal Medicine, Sotiria Hospital, National and Kapodistrian University of Athens, School of Medicine, Athens, Greece </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Charpidou%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andriani Charpidou</span></a>
</div>
</div>
<sup>17</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Syrigos%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id23"><span class="name western">Konstantinos Syrigos</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Konstantinos Syrigos</span></h3>
<div class="p">
<sup>17</sup>Third Department of Internal Medicine, Sotiria Hospital, National and Kapodistrian University of Athens, School of Medicine, Athens, Greece </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Syrigos%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Konstantinos Syrigos</span></a>
</div>
</div>
<sup>17</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chapman%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id24"><span class="name western">Jocelyn Chapman</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Jocelyn Chapman</span></h3>
<div class="p">
<sup>14</sup>Helen Diller Comprehensive Cancer Center, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>16</sup>Department of Urology, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chapman%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jocelyn Chapman</span></a>
</div>
</div>
<sup>14,</sup><sup>16</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kvale%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id25"><span class="name western">Mark Kvale</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Mark Kvale</span></h3>
<div class="p">
<sup>2</sup>Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>20</sup>Kaiser Permanente, Oakland, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kvale%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mark Kvale</span></a>
</div>
</div>
<sup>2,</sup><sup>20</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hemberg%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id26"><span class="name western">Martin Hemberg</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">Martin Hemberg</span></h3>
<div class="p">
<sup>5</sup>The Gene Lay Institute of Immunology and Inflammation, Brigham and Women’s Hospital, Massachusetts General Hospital and Harvard Medical School, Boston, MA USA </div>
<div class="p">
<sup>21</sup>Wellcome Sanger Institute, Hinxton, Cambridge, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hemberg%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Hemberg</span></a>
</div>
</div>
<sup>5,</sup><sup>21,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahituv%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id27"><span class="name western">Nadav Ahituv</span></a><div hidden="hidden" id="id27">
<h3><span class="name western">Nadav Ahituv</span></h3>
<div class="p">
<sup>1</sup>Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>2</sup>Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>22</sup>Helen Diller Family Comprehensive Cancer Center, University of California San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahituv%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nadav Ahituv</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>22,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Bioengineering and Therapeutic Sciences, University of California San Francisco, San Francisco, CA USA </div>
<div id="Aff2">
<sup>2</sup>Institute for Human Genetics, University of California San Francisco, San Francisco, CA USA </div>
<div id="Aff3">
<sup>3</sup>Institute for Personalized Medicine, Department of Molecular and Precision Medicine, The Pennsylvania State University College of Medicine, Hershey, PA USA </div>
<div id="Aff4">
<sup>4</sup>Huck Institutes of the Life Sciences, Penn State University, University Park, PA USA </div>
<div id="Aff5">
<sup>5</sup>The Gene Lay Institute of Immunology and Inflammation, Brigham and Women’s Hospital, Massachusetts General Hospital and Harvard Medical School, Boston, MA USA </div>
<div id="Aff6">
<sup>6</sup>Department of Epidemiology and Biostatistics, University of California San Francisco, San Francisco, CA USA </div>
<div id="Aff7">
<sup>7</sup>Division of Gynecologic Oncology, University of California San Francisco, San Francisco, CA USA </div>
<div id="Aff8">
<sup>8</sup>Donnelly Centre for Cellular and Biomolecular Research, University of Toronto, Toronto, ON Canada </div>
<div id="Aff9">
<sup>9</sup>Department of Epidemiology and Population Health, Stanford University, Stanford, CA USA </div>
<div id="Aff10">
<sup>10</sup>Division of Oncology, Department of Internal Medicine, Washington University School of Medicine, St. Louis, MO USA </div>
<div id="Aff11">
<sup>11</sup>Siteman Cancer Center, Washington University School of Medicine, St. Louis, MO USA </div>
<div id="Aff12">
<sup>12</sup>Department of Biomedical Engineering, Washington University School of Medicine, St. Louis, MO USA </div>
<div id="Aff13">
<sup>13</sup>Division of Hematology/Oncology, Department of Medicine, University of California San Francisco, San Francisco, CA USA </div>
<div id="Aff14">
<sup>14</sup>Helen Diller Comprehensive Cancer Center, University of California San Francisco, San Francisco, CA USA </div>
<div id="Aff15">
<sup>15</sup>Department of Radiation Oncology, University of California San Francisco, San Francisco, CA USA </div>
<div id="Aff16">
<sup>16</sup>Department of Urology, University of California San Francisco, San Francisco, CA USA </div>
<div id="Aff17">
<sup>17</sup>Third Department of Internal Medicine, Sotiria Hospital, National and Kapodistrian University of Athens, School of Medicine, Athens, Greece </div>
<div id="Aff18">
<sup>18</sup>Department of Pathology, Yale School of Medicine, New Haven, CT USA </div>
<div id="Aff19">
<sup>19</sup>Breast Oncology, Dana-Farber Brigham Cancer Center, Boston, MA USA </div>
<div id="Aff20">
<sup>20</sup>Kaiser Permanente, Oakland, CA USA </div>
<div id="Aff21">
<sup>21</sup>Wellcome Sanger Institute, Hinxton, Cambridge, UK </div>
<div id="Aff22">
<sup>22</sup>Helen Diller Family Comprehensive Cancer Center, University of California San Francisco, San Francisco, CA USA </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jun 17; Accepted 2025 Jul 28; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371106  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841759/" class="usa-link">40841759</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Cancer diagnosis using cell-free DNA (cfDNA) has the potential to improve treatment and survival but has several technical limitations.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par2">In this study, we developed a prediction model based on neomers, DNA sequences 13–17 nucleotides in length that are predominantly absent from the genomes of healthy individuals and are created by tumor-associated mutations.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">We show that neomer-based classifiers can accurately detect cancer, including early stages, and distinguish subtypes and features. Analysis of 2577 cancer genomes from 21 cancer types shows that neomers can distinguish tumor types with higher accuracy than state-of-the-art methods. Generation and analysis of 465 cfDNA whole-genome sequences demonstrates that neomers can precisely detect lung and ovarian cancer, including early stages, with an area under the curve ranging from 0.89 to 0.94. By testing various promoters or over 9000 candidate enhancer sequences with massively parallel reporter assays, we show that neomers can identify cancer-associated mutations that alter regulatory activity.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par4">Combined, our results identify a sensitive, specific, and simple cancer diagnostic tool that can also identify cancer-associated mutations in gene regulatory elements.</p></section><section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Cancer genomics, Cancer epigenetics</p></section></section><section class="abstract" id="Abs2"><h2>Plain language summary</h2>
<p id="Par5">DNA fragments circulating in the blood are often present in people with cancer. Current tests using DNA fragments circulating in the blood have limitations in detecting cancer. We developed a cancer diagnostic approach that uses neomers, short DNA sequences that are mostly absent in healthy individuals but are present in tumors as a consequence of changes in the DNA (DNA mutations) in cancer cells. We show that this approach can accurately detect lung and ovarian cancers from DNA fragments extracted from blood and identify changes in the DNA associated with cancer. Our diagnostic tool could be used to improve cancer detection and identify how changes in the DNA affect development of cancer.</p></section><section class="abstract" id="Abs3"><hr class="headless">
<p id="Par6">Georgakopoulos-Soares, Yizhar-Barnea, Mouratidis et al. identify neomers, short (16 base pair) DNA sequences that are absent in the genomes of healthy individuals but appear in tumors due to mutations. They show that neomers can accurately detect cancer from cell-free DNA and that neomers found in gene regulatory elements can alter regulatory activity.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par7">Cancer is the second leading cause of death worldwide<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. For most cancer types, survival is significantly higher if the tumor is detected at an early stage<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Currently, mass population screening is applicable only for breast and cervical cancers and utilizes physical tests like mammography and cytology screens. Detection for other cancer types, done both <em>en masse</em> and in a low and affordable-resource setting, still poses a major challenge for the scientific and clinical communities<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. In particular, a major hurdle is to identify reliable biomarkers for the detection of cancer at a presymptomatic stage. Detection at such an early stage would allow not only for improved survival but would also decrease treatment toxicity and provide an opportunity for providing personalized treatments.</p>
<p id="Par8">Circulating cell-free DNA (cfDNA) has a lot of potential to become an effective biomarker for cancer diagnostics<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. It has a short life span (16 min to 2.5 h), making it an efficient temporal indicator of biological processes in the body. With improvements in sequencing technologies and their costs becoming lower, cfDNA can also be a rapid and cheap biomarker. Being able to identify circulating tumor DNA (ctDNA) is also minimally invasive and can allow not only for cancer detection but also to monitor patients already diagnosed with cancer. To identify cancerous cells, their tissue of origin, cancer type, minimal residual disease, and other cancer features, current cfDNA sequencing technologies rely on the detection of somatic mutations<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> and epigenetic marks, such as DNA methylation or histone modifications that can determine the cancerous tissue of origin<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. For example, one study showed that cell-free DNA methylation patterns can be used for the accurate detection of early-stage breast cancer<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Furthermore, cfDNA has been used as a biomarker for treatment response and recurrence<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. However, ctDNA still has many hurdles and caveats that need to be overcome<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Some of the major hurdles include: (1) Circulating cfDNA is found in fragment sizes ranging between 120 and 220 base pairs (bp), with most of the fragments being 167 bp<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> in length, thus making it challenging to extract; (2) Tumor-derived DNA makes up around 0.4% of the total cfDNA (there is some variance between tumor types), thus needing extremely sensitive biomarkers that can detect the presence of cancerous cells; (3) Prior knowledge of specific mutations or methylation marks is required for targeted screening, leading to biased screening primarily of coding mutations which make up only a small fraction of cancer-associated mutations. (4) Somatic alterations in white blood cells can confound cfDNA mutations and epigenetic changes<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>; (5) The techniques used to detect methylation or histone marks from cfDNA are technologically complex and have low sensitivity and specificity, in particular in early stages<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. To provide the most optimal cancer treatment, it needs to be diagnosed at early stages when the tumor is small (~5 mm in diameter). At these stages, the tumor produces very low levels of ctDNA that are difficult to detect using current methods<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>.</p>
<p id="Par9">Nullomers are short DNA sequences (13–17 base pairs) that are absent from the human genome<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>–<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. While the absence of nullomers could be due to chance, we and others have shown that a significant proportion of them is under negative selection pressure<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, suggesting that they may have a deleterious effect on the genome. We have also shown that these sequences could be used as DNA “fingerprints” to identify specific human populations<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. As nullomers generally do not exist in a human genome, their appearance due to mutagenesis followed by clonal expansion could be exploited as a diagnostic method for diseases associated with a mutational burden, such as cancer. Nullomers have been used for the detection of five different cancers using cell-free RNA<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>.</p>
<p id="Par10">Here, we set out to test whether nullomers could be used as a diagnostic tool to detect cancer and various additional tumor features. Throughout this manuscript, we refer to nullomers uniquely found in multiple tumor genomes as <em>neomers</em> to distinguish them from the more general category. We first analyzed The Cancer Genome Atlas (TCGA<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>) database, finding that neomers created by somatic mutations are able to detect cancer subtypes with higher accuracy than leading methods<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> as well as additional cancer features. Further analyses of cfDNA whole-genome sequencing (WGS) datasets found that these neomers can also be used to detect cancer subtypes. Using WGS on cfDNA from 465 individuals with ovarian or lung cancer and normal controls found an enrichment for neomers. Our neomer-based cancer detection models had area under the curve (AUC) of 0.93 for lung and 0.88 for ovarian cancers, and an AUC of 0.94 for lung and 0.93 for ovarian cancer at early stages. This is particularly noteworthy for ovarian cancer as currently there is no effective screening test<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, with most women being diagnosed at later stages, IIIC or IV, where 5-year survival rates are 39% and 17%, respectively<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Finally, utilizing both promoter assays and a massively parallel reporter assay (MPRA), we show that neomers alter regulatory activity and can be used to detect cancer-associated mutations in gene regulatory elements. Combined, our results show that neomers can be used as a rapid, sensitive, specific, and simple cancer diagnostic tool and also aid in the identification of gene regulatory mutations associated with cancer.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Computational characterization of nullomers</h3>
<p id="Par11">The GRCh38 reference assembly of the human genome was used throughout the study. Nullomer extraction was performed for kmer lengths up to 17 base pairs using the algorithm described in our previous work<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. The reverse complement of a nullomer will also be a nullomer and throughout this manuscript when counting nullomers, the reverse complement of nullomer <em>i</em> was also considered separately, unless <em>i</em> is a palindrome. Substitutions and indels that were not present in healthy tissues were identified from WGS of tumor samples from 2577 individuals across 21 tissues were obtained from open access release data at <a href="https://docs.icgc-argo.org/docs/data-access/icgc-25k-data" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://docs.icgc-argo.org/docs/data-access/icgc-25k-data</a><sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Recurrent nullomers (neomers) (<em>r</em><sub><em>i</em></sub>) were annotated as those that resulted from substitutions or indels across two or more patients within a cancer type. When possible, <em>r</em><sub><em>i</em></sub> was chosen to get ~10,000 neomers from each tissue; otherwise, it was set to 2 (Data <a href="#MOESM3" class="usa-link">S1</a>). Driver mutation-derived neomers were defined as nullomers detected from driver mutations and were identified using the catalog of driver mutations derived from the intOGen database<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, for which we grouped driver mutations by tissue of origin.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Classification of tumor tissue of origin using neomers</h3>
<p id="Par12">We trained a classifier to distinguish classified tissue of origin for a cancer sample based on observed neomers using the libSVM package for Julia<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> with default parameters to train a support vector machine (SVM) classifier with a linear kernel. We used 10-fold cross-validation, whereby the classifier was evaluated on a held-out fraction of the data. The set of neomers for each cancer type was recalculated for each round to only include the patients in the training set.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Supervised selection of nullomers</h3>
<p id="Par13">The MMR status of each biopsy sample was derived from ref. <sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. The model was trained on neomers identified in MSI samples and the performance of the algorithm evaluated. Previous research has indicated that mismatch repair deficiency is associated with mutations at mononucleotide repeat tracts<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. For the MSI versus the microsatellite stable (MSS) samples, we counted the number of neomers that contained either AAAAAAAA or TTTTTTTT repeats, since MSI cancers have been associated with mutations of polyA/T repeats<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. The threshold for determining MSI or MSS was set as the harmonic mean of the maximum number of counts in the MSS set and the minimum number of counts in the MSI set. The POLE deficiency status of each biopsy sample was derived from ref. <sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> and we used a similar strategy to that of MMR status, but instead counted neomers created through either a TCT &gt; TAT or TCG &gt; TTG mutation. Since the number of patients in each category was limited, we used a 5-fold cross validation. For ovarian cancer analyses, two sets of nullomers were used. From the nullomers of length <em>k</em> = 13 identified in ref. <sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, we retained only the ones that could not be created by any of the single base-pair substitutions identified in gnomAD v2<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Similarly, we selected all nullomers of length <em>k</em> = 15 of order 1, i.e., nullomers that can only be created from the human reference genome through at least two insertions, deletions, or substitutions.</p></section><section id="Sec6"><h3 class="pmc_sec_title">cfDNA extraction and WGS</h3>
<p id="Par14">Lung and ovarian cancer samples, as well as age and gender-matched samples were purchased from ProteoGenex Inc. (Inglewood, CA, USA) and IndivuMed GmbH (Hamburg, Germany). In addition, cancer samples, and when available, healthy age and gender-matched samples, were also obtained from the labs of K. Syrigos (Sotiria Hospital, National and Kapodistrian University of Athens, School of Medicine, Athens, Greece), J. Chapman (UCSF, San Francisco, USA), J. Witte (UCSF, San Francisco, USA), and C. Maher (WashU, St. Louis, USA). The samples that included ancestry information were all Caucasian (Data <a href="#MOESM3" class="usa-link">S3</a>). All samples were collected with informed consent. All human sample work was carried out under the approved UCSF human research protection program institutional review board protocol number 21-35920. cfDNA was extracted from 1 mL of plasma, following centrifugation at 4 °C at 34 × <em>g</em> for 3 min, to remove larger debris, using the QIAamp Circulating Nucleic Acid Kit (Qiagen). cfDNA was eluted in 50 μL of elution buffer and measured using the Qubit High-Sensitivity dsDNA kit, and validated for size distribution (160–180 bp) using an Agilent BioAnalyzer 2100 Sensitivity DNA chip. Up to 10 ng of cfDNA was used for sequencing library construction using the library preparation enzymatic fragmentation kit 2.0 (Twist Bioscience) adjusted with IDT’s xGen UDI-UMI 96 barcodes system (IDT) to replace the Twist universal adapter, and using KAPA HiFi polymerase instead of the polymerase provided by the kit. Libraries for lung, ovarian, and their respective controls were sequenced as PE150 using a NovaSeq 6000 S4 system (Illumina) leveraging UMIs to remove PCR duplicates, aiming for 5–10X coverage per multiplexed library, via Novogene. Samples were sequenced in batches of 96 (maximal UDI by the xGen kit) and were equally spread across multiple lanes. Each batch contains a consistent mixture of healthy controls and cancer samples.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Neomer identification in cfDNA samples</h3>
<p id="Par15">To improve the detection of lung cancer mutations in cfDNA, we used neomers that are present in ≥2 patients rather than ≥3 as was used for the tissue-of-origin classifier. The FASTQ files were scanned for neomers by searching for exact matches to the 16-mers of interest. We disregarded any neomer that could arise due to a common germline variant (VAF &gt; 1%). To reduce errors, only reads where the nullomer was found in both pairs were kept. To evaluate the ability to detect cancer from cfDNA, we first randomly split samples into training and validation sets. Prior to the split, samples were randomly removed from either the controls or patients to ensure that the two groups were of equal size. From the training set we determined a threshold, <em>t</em> = max (number of neomers in control samples from the training set)— 1. Samples in the validation set were then designated as having cancer if the number of neomers detected was ≥<em>t</em>. To calculate F1 scores, this procedure was repeated 100 times. For the downsampling analysis, each read pair in sample <em>i</em> was retained with a probability of <em>c/C</em><sub><em>i</em></sub>, where <em>c</em> is the target coverage and <em>C</em><sub><em>i</em></sub> is the full coverage of the sample, i.e., sampling without replacement. For each value of <em>c</em> the downsampling was repeated 100 times.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Classification model to detect cancer patients from cfDNA</h3>
<p id="Par16">Neomers were identified for each donor and combined as a neomer density × sample matrix, <em>d</em><sub>ij</sub> = <em>n</em><sub>ij</sub>/<em>C</em><sub>j</sub>, where <em>n</em><sub>ij</sub> is the count of neomer <em>i</em> in sample j and <em>C</em><sub>j</sub> is the sample coverage. Neomer-based classification models were generated to examine the ability of neomers to detect cancer for lung, colorectal, ovarian, pancreatic, and prostate cancers. We tested PCA and feature hashing to deal with the sparse nature of our dataset and used feature hashing with 8192 features as the best performing method. We evaluated five classification models; random forest, linear, and radial basis function SVMs and linear regression (LR) models with L1 and L2 regularization, each evaluated with a 10 × 10-nested cross-validation. Our benchmarks indicated LR with an L2 penalty as the best performing model across all cancer types. The results obtained were also examined regarding tumor staging for colorectal and lung cancers, in which we had sufficient sample sizes. The nested cross validation (CV) used a 10-fold inner CV to optimize model parameters and hyperparameters and a 10-fold outer CV to test the optimized classifier on data not used for parameter and hyperparameter optimization. This design yields an unbiased estimation of performance while allowing for full optimization of the model. Model parameters were optimized with respect to the balanced accuracy metric, as were hyperparameters across the inner CV. A grid search was used to determine the optimal hyperparameters. Model performance in the outer CV was evaluated using balanced accuracy, AUC, and other binary classification metrics. Choice of model type and of number of hashed features were made outside of the nested CV framework. In practice, the classifier performance tended to plateau or achieve a mild maximum with respect to the number of hashed features, so the number of hashed features chosen was sufficiently large to place the classifier in the performance plateau. No one model type consistently dominated the others in terms of performance, so the model type used was determined by the needs of the analysis and the different tradeoffs with respect to complexity and interpretability.</p>
<p id="Par17">Fragmentomics-based classification models were generated to compare its performance against the neomer method on the lung and ovarian cancer datasets generated from this study. Paired-end sequencing reads were aligned to the GRCh38 reference genome using Burrows-Wheeler Aligner - Maximal Exact Matches (BWA-MEM).</p>
<p id="Par18">Genome-wide fragmentation profiles for each sample (defined as the ratio of short to long fragments) were created using <a href="https://github.com/mhanbioinfo/fragmentomicsBEDPE" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/mhanbioinfo/fragmentomicsBEDPE</a>, where GC-corrected counts were binned into 5 Mb windows. The fragmentation profile was then used in a stochastic gradient boosting model with 10-fold cross validation as described in ref. <sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Alternatively, the read coverage on each of the 39 autosomal arms was calculated and compared to the average coverage to generate coverage <em>z</em>-scores for each sample as described in ref. <sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> and used as features for a stochastic gradient boosting model with 10-fold cross validation. For end motif analysis, duplicates from aligned reads were removed using Picard MarkDuplicates, and 167 basepair fragments were retained and converted to BEDPE format using samtools<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. 256 tetranucleotide motifs at the breakpoints of 167 basepair fragments were used in a SVM and LR model using leave-one-out cross-validation as described in ref. <sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Promoter luciferase assays</h3>
<p id="Par19">Promoter sequences with and without the neomer (Data <a href="#MOESM3" class="usa-link">S4</a>) were synthetically generated and cloned into the modified Promega promoter assay luciferase vector pGL4.11b (a gift from Dr. Rick Myers, HudsonAlpha) by BioMatik Inc. and Sanger sequence verified. LNCaP cells were plated at an initial density of 2 × 10<sup>5</sup> cells/well in 24-well tissue culture plates and maintained in RPMI medium, 10% FBS supplemented with L-Glutamine and Penicillin/Streptomycin. Plasmids together with a Renilla expressing plasmid, pGL4.74 (Promega), at a ratio of 10:1 luciferase:Renilla were transfected using the X-tremeGENE™ HP DNA Transfection Reagent (Roche) using 1:4 ratio of DNA (μg) to reagent (μl). Seventy-two hours post-transfection, Luciferase and Renilla levels were measured using the Dual-Luciferase Reporter Assay System (Promega) following the manufacturer’s protocol using a GloMax Explorer Multimode Microplate Reader (Promega). Luciferase activity was normalized to Renilla levels and presented as Relative Luciferase Units (RLU). Statistical analysis was performed using Prism version 9.0.2 (GraphPad). All values were reported as means (AVG) and standard errors (SE). <em>p</em> &lt; 0.05 were considered statistically significant.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Massively parallel reporter assays</h3>
<p id="Par20">The lentivirus-based MPRA was performed following a previously published protocol<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. An oligonucleotide library of 230 bp long fragments bearing: (1) 4609 sequences with and without a recurring neomer that was found in prostate cancer patients; (2) 100 scrambled control sequences randomly selected from the library. A pool of 230-nt oligos containing each of these 200-nt sequences flanked by 15-base pair primer sequences on either side was synthesized (Agilent Technologies), amplified, and cloned into the lentiMPRA plasmid. Briefly, amplified inserts were cloned upstream of a minimally active promoter (mP), 5’UTR barcode (BC), and EGFP reporter transcript. We aimed for ~100 BCs per sequence when constructing and extracting the plasmid library. Each insert was associated with a set of BCs using paired-end (PE150) sequencing on an Illumina NextSeq 500(see Gordon et al. Step 83<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>). Lentivirus was produced and titered and used to infect LNCaP cells at an MOI of 50 virus particles per cell. Three days post-infection, to remove non-integrating virus, DNA and RNA were extracted from the three replicates and used for library construction and multiplexing for sequencing. All MPRA-related sequencing was performed using an Illumina NextSeq 500 with either PE150 for the CRS-BC association library or PE15 for the DNA/RNA BC count portion of the protocol. MPRA analysis was carried out using MPRAflow<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Comparison to validated neoantigens</h3>
<p id="Par21">We downloaded a list of 1967 validated neoantigens from <a href="http://biopharm.zju.edu.cn/download.neoantigen/iedb_validated.zip" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://biopharm.zju.edu.cn/download.neoantigen/iedb_validated.zip</a>. Requiring both predicted strong binding and a positive validation, provided 1700 neoantigens. To evaluate the enrichment of neoantigens corresponding to neomers, we assumed a hypergeometric distribution with 1700 draws from an urn with 188,659 white balls (total number of neomers) and 186,067,892 black balls (number of nullomers found with lower recurrency than what was required for neomers).</p></section><section id="Sec12"><h3 class="pmc_sec_title">Indel identification</h3>
<p id="Par22">Somatic indel calls were performed using three pipelines from four somatic variant callers. These included those from the Wellcome Sanger Institute, the DKFZ/EMBL, and the Broad Institute, which reported a somatic variant false discovery rate of 2.5%<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Statistics and reproducibility</h3>
<p id="Par23">Statistical tests are described in the context where they are used. All code is publicly available as described above for reproducibility of computational analyses.</p></section></section><section id="Sec14"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec15"><h3 class="pmc_sec_title">Annotation of mutations that lead to nullomers</h3>
<p id="Par24">As cancer is associated with a large number of somatic DNA mutations, we investigated if a subset of them could result in the resurfacing of nullomers (Fig. <a href="#Fig1" class="usa-link">1a</a>). Using our previously characterized human nullomers<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, we analyzed WGS results from 2577 patients across 21 cancer types from TCGA<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> for resurfacing nullomers (Fig. <a href="#MOESM1" class="usa-link">S1A</a>). We focused on 16 bp nullomers, as it is the shortest length where we detect a sufficient number of nullomers per patient, with the human reference genome having only 37.24% of all possible 16 mers. The majority of the 44,599,472 single-nucleotide substitutions that were tumor-unique give rise to multiple nullomers, allowing us to identify 213,164,038 resurfacing nullomers across all cancer types. Furthermore, we identified 2,470,091 nullomers resulting from short insertions and deletions (1–100 base pairs). The median number of nullomers created by each substitution was two, and for indels, four (Fig. <a href="#MOESM1" class="usa-link">S1B, C</a>). On average, 58.29% of substitutions in a patient resulted in one or more nullomers, with 2.1% of the nullomers residing in coding regions. The median number of nullomers found across cancer patients was 9107 (Fig. <a href="#Fig1" class="usa-link">1b, c</a>) and the number of nullomers was directly proportional to the number of mutations (Fig. <a href="#MOESM1" class="usa-link">S1D</a>). As mutations were identified by comparisons to healthy tissues, germline variants of each individual were not included<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Neomers can detect cancer tissue of origin.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371106_43856_2025_1067_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/70f6/12371106/96e7fa38c36e/43856_2025_1067_Fig1_HTML.jpg" loading="lazy" id="d33e1033" height="999" width="701" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic overview of neomer cancer diagnostic pipeline. Created in Biorender. Ahituv, N. (2025) <a href="https://BioRender.com/9sgstbs" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/9sgstbs</a>. <strong>b–f</strong> 2577 patients across 21 cancer types were used. <strong>b</strong> Number of neomers per patient sample across tissues. Each dot represents a patient sample. The boxplot displays the median as the central line. The bottom and top edges of the box represent the first quartile (25th percentile) and third quartile (75th percentile), respectively. The lower and upper whiskers extend to the smallest and largest data values that fall within 1.5 times the interquartile range above the 25th percentile and below the 75th percentile. <strong>c</strong> Number of neomers and the number of substitutions for 2577 patients (Spearman’s <em>ρ</em> = 0.75). <strong>d</strong> Identification of nullomers at driver mutation sites across cancer tissues. The majority of cancer driver mutations result in the resurfacing of nullomers. The proportion of mutations that cause the resurfacing of nullomers is indicated under each cancer type. <strong>e</strong> Heatmap showing the Jaccard index for the overlap of neomer sets associated with different cancer types. <strong>f</strong> Heatmap showing the occurrence of neomers across patients for each cancer type. Each row represents a cancer type and each column a patient. The intensity of the heatmap (log2-scale) shows the number of neomers for each tissue set.</p></figcaption></figure><p id="Par25">To further prioritize nullomers that could be used as cancer biomarkers, we focused on the subset of nullomers that are recurrent, i.e., those found in more than one patient for a specific cancer type, termed hereafter as <em>neomers</em>. The number of neomers was proportional to the total number of mutations (Fig. <a href="#Fig1" class="usa-link">1c</a> and Data <a href="#MOESM3" class="usa-link">S1</a>). As both the number of patients per cancer type and the mutational load varied, the median number of neomers for each tissue type ranged from 0 to 98. Analysis of the most frequent neomers revealed several previously known cancer-associated driver mutations (Table <a href="#Tab1" class="usa-link">1</a>). For example, some of the most recurrent coding neomers were the result of either the Gly12Asp, Gly12Val, or Gly12Cys missense mutation in the KRAS proto-oncogene GTPase (KRAS), which are known to make up 80% of cancer-associated KRAS mutations and lead to KRAS being constitutively active<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Although KRAS mutations have been associated with several cancer types, 190/215 (88%) of these mutations were found in pancreatic cancers. Several frequently occurring coding neomers were also found in other known cancer-associated genes, such as tumor protein p53 (<em>TP53</em>), B-Raf proto-oncogene serine/threonine kinase (<em>BRAF</em>), and phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (<em>PIK3CA</em>). The most frequent neomer was located in a noncoding region, within the telomerase reverse transcriptase (<em>TERT</em>) promoter, which is known to be associated with numerous cancer types<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> and poor prognosis<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. This mutation, called −124C &gt; T or C228T, is extremely common in numerous cancer types<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup> and is thought to disrupt a G-quadruplex<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> leading to the binding of GABP<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, an ETS transcription factor, resulting in increased <em>TERT</em> expression. We found this mutation in 97 patients, with the highest incidence in glioblastoma (51%), fitting with its known high prevalence rate and diagnostic use for this cancer type<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Common cancer-associated neomers</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups"><tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Neomer</td>
<td align="left" colspan="1" rowspan="1">Locus</td>
<td align="left" colspan="1" rowspan="1">Coordinate (hg38)</td>
<td align="left" colspan="1" rowspan="1">Mutation</td>
<td align="left" colspan="1" rowspan="1">Name</td>
<td align="left" colspan="1" rowspan="1">Patient #</td>
<td align="left" colspan="1" rowspan="1">Cancer #</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AGGGCCCGGA<strong><span class="text-underline">A</span></strong>GGGGC</td>
<td align="left" colspan="1" rowspan="1"><em>TERT</em></td>
<td align="left" colspan="1" rowspan="1">chr5:1295113</td>
<td align="left" colspan="1" rowspan="1">G &gt; A</td>
<td align="left" colspan="1" rowspan="1">−124C &gt; T C228T</td>
<td align="left" colspan="1" rowspan="1">97</td>
<td align="left" colspan="1" rowspan="1">7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TCTTGCCTACGCCA<strong><span class="text-underline">T</span></strong>C</td>
<td align="left" colspan="1" rowspan="1"><em>KRAS</em></td>
<td align="left" colspan="1" rowspan="1">chr12:25245350</td>
<td align="left" colspan="1" rowspan="1">C &gt; T</td>
<td align="left" colspan="1" rowspan="1">c.35G &gt; AGly12Asp</td>
<td align="left" colspan="1" rowspan="1">89</td>
<td align="left" colspan="1" rowspan="1">6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CTG<strong><span class="text-underline">T</span></strong>TGGCGTAGGCAA</td>
<td align="left" colspan="1" rowspan="1"><em>KRAS</em></td>
<td align="left" colspan="1" rowspan="1">chr12:25245350</td>
<td align="left" colspan="1" rowspan="1">C &gt; A</td>
<td align="left" colspan="1" rowspan="1">c.35G &gt; TGly12Val</td>
<td align="left" colspan="1" rowspan="1">79</td>
<td align="left" colspan="1" rowspan="1">6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ACGCCACGA<strong><span class="text-underline">G</span></strong>CTCCAA</td>
<td align="left" colspan="1" rowspan="1"><em>KRAS</em></td>
<td align="left" colspan="1" rowspan="1">chr12:25245351</td>
<td align="left" colspan="1" rowspan="1">C &gt; G</td>
<td align="left" colspan="1" rowspan="1">c.34G &gt; T Gly12Cys</td>
<td align="left" colspan="1" rowspan="1">47</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GGTGC<strong><span class="text-underline">A</span></strong>TGTTTGTGCC</td>
<td align="left" colspan="1" rowspan="1"><em>TP53</em></td>
<td align="left" colspan="1" rowspan="1">chr17:7673802</td>
<td align="left" colspan="1" rowspan="1">C &gt; T</td>
<td align="left" colspan="1" rowspan="1">c.818G &gt; A Arg273Pro</td>
<td align="left" colspan="1" rowspan="1">35</td>
<td align="left" colspan="1" rowspan="1">13</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GTGGGGGCAG<strong><span class="text-underline">T</span></strong>GCCTC</td>
<td align="left" colspan="1" rowspan="1"><em>TP53</em></td>
<td align="left" colspan="1" rowspan="1">chr17:7675088</td>
<td align="left" colspan="1" rowspan="1">C &gt; T</td>
<td align="left" colspan="1" rowspan="1">c.524G &gt; A Arg175His</td>
<td align="left" colspan="1" rowspan="1">28</td>
<td align="left" colspan="1" rowspan="1">11</td>
</tr>
</tbody></table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p28"><p>Six of the most common neomers created by a single mutation. The nucleotide in bold font and underlined is the neomer causing mutation.</p></div></div></section><p id="Par26">We also identified several neomers that are frequently created by different mutations (Table <a href="#Tab2" class="usa-link">2</a>). Interestingly, some of these frequently recurrent neomers are created by different mutations, yet are predominantly found in one cancer. For example, GTTTTTCTCCTAGACC is found 41 times in skin cancer at 32 different loci while CTGGCAGTGAGCCACG is found 26 times in liver cancer across 23 loci. The majority (98%) of these frequent neomers reside in noncoding regions, and many of them reside in intronic regions (35%). For example, CGACGTTCTGCCCACT is found in 32 loci, primarily in pancreatic and stomach cancer. Of those loci, 21/32 (65.6%) were found in noncoding regions nearby pancreatic cancer-associated genes. These include, for example, the C-C motif chemokine ligand (<em>CCL4</em>)<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, the POM121 transmembrane nucleoporin like 12 (<em>POM121L12</em>), which is commonly mutated in gastrointestinal cancers<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> and the potassium voltage-gated channel modifier subfamily V member 1 (<em>KCNV1</em>) gene, where promoter hypermethylation has been associated with both pancreatic<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup> and esophageal cancer<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Frequent cancer-associated neomers</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups"><tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Neomer</td>
<td align="left" colspan="1" rowspan="1">Loci #</td>
<td align="left" colspan="1" rowspan="1">Patient #</td>
<td align="left" colspan="1" rowspan="1">Cancer #</td>
<td align="left" colspan="1" rowspan="1">Cancer type</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CGACGTTCTGCCCACT</td>
<td align="left" colspan="1" rowspan="1">32</td>
<td align="left" colspan="1" rowspan="1">74</td>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">Mainly pancreatic and stomach</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GTTTTTCTCCTAGACC</td>
<td align="left" colspan="1" rowspan="1">32</td>
<td align="left" colspan="1" rowspan="1">41</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">All but one in skin</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CTGCAGTGGCGCAATA</td>
<td align="left" colspan="1" rowspan="1">30</td>
<td align="left" colspan="1" rowspan="1">30</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">33% of the neomers in colon cancer</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TTATAGGGGTCCAGTG</td>
<td align="left" colspan="1" rowspan="1">25</td>
<td align="left" colspan="1" rowspan="1">26</td>
<td align="left" colspan="1" rowspan="1">1 (Five of the top frequently recurring neomers created by several different mutations)</td>
<td align="left" colspan="1" rowspan="1">Colorectal only</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CTGGCAGTGAGCCACG</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">26</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">21 in liver, 5 esophagus</td>
</tr>
</tbody></table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p31"><p>Five of the top frequently recurring neomers created by several different mutations.</p></div></div></section><p id="Par27">To further validate that nullomers can detect cancer-associated mutations, we examined whether they are enriched in known cancer driver mutations. We annotated a cohort of driver mutations from 28 different cancer types<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup> for their overlap with 16 bp nullomers. In the pan-cancer analysis, we identified 19,594,212 nullomers resulting from 50,167 putative driver mutations (Fig. <a href="#Fig1" class="usa-link">1f</a> and Data <a href="#MOESM3" class="usa-link">S2</a>). For specific cancer types, we found that on average, 81% of driver mutations resulted in one or more nullomer, ranging between 63.88% and 86.50% in pancreatic and lung cancer, respectively (Data <a href="#MOESM3" class="usa-link">S2</a>). The number of nullomers also varied by tissue type, ranging between 92 in thyroid and 9434 in skin (Fig. <a href="#Fig1" class="usa-link">1d</a>). In general, driver mutations were 1.4-fold more likely to result in the creation of a nullomer (permutation test, <em>p</em>-value &lt; 0.001). Taken together, our results suggest that a large subset of clinically relevant cancer mutations are associated with neomers, i.e., recurrent nullomers.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Cancer subtype neomer classifier</h3>
<p id="Par28">We next set out to assess whether neomers can be used to distinguish between cancer types. We filtered neomers by keeping only those that appeared &gt;=<em>r</em><sub><em>i</em></sub> times in specific cancer type <em>i</em> (Data <a href="#MOESM3" class="usa-link">S1</a>). Comparison of the set of neomers associated with each cancer type revealed a small overlap, as indicated by the Jaccard index which is &lt;0.04, suggesting that each cancer type has a distinct neomer signature (Fig. <a href="#Fig1" class="usa-link">1e</a>). We also counted the number of times neomers are found in each patient, finding that patients are strongly enriched for only one set of cancer-specific neomers (Fig. <a href="#Fig1" class="usa-link">1f</a>).</p>
<p id="Par29">We then tested if our annotated cancer-specific neomers can be used to classify tumor samples. We trained a SVM classifier to identify tumor type. The classifier takes as input a 21-dimensional vector indicating the number of neomers found for each cancer-specific set. For the classification model, we divide the samples into train and validation and test groups, and only filter the neomers and construct a classifier, based on the train samples. Evaluation using 10-fold cross-validation revealed that our classifier achieves both high sensitivity and specificity, with an F1 score of 0.92 and an accuracy of 0.99 (Fig. <a href="#Fig2" class="usa-link">2a, b</a>). Performance was better than a recent deep learning model<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> and also requires less computational resources to train as once neomers have been extracted, training and testing take only a few minutes.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Neomers can distinguish cancer features.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371106_43856_2025_1067_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/70f6/12371106/7c6bec6f47b2/43856_2025_1067_Fig2_HTML.jpg" loading="lazy" id="d33e1444" height="606" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Boxplots of classifier accuracy (<strong>a</strong>) using an unsupervised classifier and F1 (<strong>b</strong>) score for the same classifier for each of the twenty-one cancer types. Box plot lines indicate lower, middle, and upper interquartile values. <strong>c</strong> Separation of 10 MSI and 10 MSS samples using a supervised selection of neomers. <strong>d</strong> Separation of 25 POLE proficient and 25 deficient samples using nullomers. In (<strong>c</strong>, <strong>d</strong>) the vertical line displays the harmonic mean.</p></figcaption></figure></section><section id="Sec17"><h3 class="pmc_sec_title">Neomers distinguish additional cancer features</h3>
<p id="Par30">We next tested whether a supervised feature selection approach, i.e., using neomers that are thought to be informative based on prior biological knowledge, would improve performance. For this approach, we initially utilized microsatellite unstable (MSI) and MSS cancers. MSI is associated with better cancer prognosis, increased benefits from surgery and higher sensitivity to immunotherapy, but with a lack of efficacy from adjuvant treatment<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Since MSI cancers are associated with mutations at polyA and polyT stretches<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>, we hypothesized that neomers containing these motifs would be able to effectively distinguish these two cancer types. We identified ten MSI samples from a cohort of 560 breast cancers<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> and compared them to ten randomly selected MSS samples from the same cohort. We found that the polyA/T neomers were able to separate the two categories with an accuracy of 100% (Fig. <a href="#Fig2" class="usa-link">2c</a>).</p>
<p id="Par31">We next applied a similar strategy to distinguish patients with DNA polymerase epsilon catalytic subunit (POLE) deficiency, as these tumors are known to respond more favorably to immune checkpoint inhibitors<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. We identified 25 patients from the TCGA dataset labeled as POLE deficient, and searched for neomers created through a TCT &gt; TAT or TCG &gt; TTG mutation, which are the most common types of mutations in this context<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Comparing against POLE proficient tumors, we found that the number of neomers identified for each group have very little overlap (Fig. <a href="#Fig2" class="usa-link">2d</a>), and the classifier achieved an accuracy of 96%.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Neomers detect early-stage lung cancer</h3>
<p id="Par32">We next tested whether neomers could be used to diagnose cancer in cfDNA. We first used lung cancer as our test case, due to having an ample amount of cancer-associated neomers (Data <a href="#MOESM3" class="usa-link">S1</a>), difficulty to diagnose using current techniques, with early stages being mostly asymptomatic<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>, and being the leading cause of cancer-associated mortality worldwide<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. We assembled a cohort of cfDNA from the plasma of 381 individuals. These included 200 lung cancer patients from all stages with a strong emphasis on early stages: 77 stage I (38% of cancer cases), 36 stage II (18%), 34 stage III (17%), and 53 stage IV (27%). The majority of our cases were adenocarcinoma (50.2% of cancer samples), followed by squamous cell carcinoma (37.8%) (Data <a href="#MOESM3" class="usa-link">S3</a>). The median age was 64 ± 7.8 years old, with patients as young as 42 years old. We also had 180 controls, considered cancer free (e.g., “healthy”, no cancer of any kind detected &gt;2 years after sampling) with a median age of 58 ± 8.1 years and 90% fitting the criteria for the United States Preventive Services Task Force (USPTF) age interval for early screening for lung cancer. Of the healthy individuals, 21% were active smokers (37.2% were noted as unknown), while 64.2% of the cancer patients were noted as active or past smokers (complying with USPTF guidelines for lung cancer screening) (additional information on the cohort is available in Data <a href="#MOESM3" class="usa-link">S3</a>). Samples were paired-end sequenced at a genomic depth of 5-10× coverage (<em>C</em>) and analyzed for neomers. For each sample and each neomer, all occurrences of the neomer were extracted from the FASTQ file, and the resulting neomer count was normalized by dividing by the sample read depth, forming a neomer density measure. Putting all the neomer densities into a table with sample rows and neomer columns yields a neomer density matrix (NDM). The NDM with an added column for the sample status forms the input for our classification models. To reduce the impact of sequencing errors, we only considered neomers that were found on both read-pairs from a single pair-end read, resulting in 18,090–55,409 read-pairs per sample. To further avoid spurious neomers, only neomers that overlapped at least two read-pairs and overlapped the human genome were used for subsequent analyses. Of note, an important computational advantage compared to conventional mutation calling pipelines is that only a single pass is made across the reads to identify those containing neomers, and only those reads are aligned to the reference genome. Using these data, we trained an LR classifier, achieving an AUC score of 0.93 using 10-fold cross validation and a sensitivity of 92.5% (specificity of 85%) (Fig. <a href="#Fig3" class="usa-link">3a, b</a>). Importantly, our classifier also performed well for early stages, with an AUC of 0.94 for stage I and sensitivity of 96.1% (specificity of 85%) (Fig. <a href="#Fig3" class="usa-link">3c, d</a>). To showcase the ability of the method to recognize different cancer types, we tested the performance of the classifier on lung cancer samples when utilizing the ovarian neomer list. The performance for all models is clearly lower than the performance of the optimal model trained on lung cancer neomers, indicating the ability of the classifier to discriminate between cancer types—a sample will be given a much higher chance of being from a cancer patient when using the corresponding cancer neomer list (Fig. <a href="#MOESM1" class="usa-link">S3</a>). Finally, by carrying out a downsampling analysis, we find that already at 3x coverage we can obtain similar accuracy (Fig. <a href="#MOESM1" class="usa-link">S2</a>). In summary, these data suggest that our neomer lung cancer profile can effectively detect lung cancer from cfDNA with high accuracy, including early stages.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Neomers can detect lung and ovarian cancer from cfDNA.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371106_43856_2025_1067_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/70f6/12371106/488f9d8db6cb/43856_2025_1067_Fig3_HTML.jpg" loading="lazy" id="d33e1559" height="1249" width="748" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Receiver operator characteristics (ROC) curve (<strong>a</strong>) and confusion matrix (<strong>b</strong>) for the neomer based classifier for cfDNA from 200 lung cancer patients and 180 controls. ROC curve (<strong>c</strong>) and confusion matrix (<strong>d</strong>) from 77 stage I lung cancer cases and 180 controls. ROC curve (<strong>e</strong>) and confusion matrix (<strong>f</strong>) for ovarian cancer from 70 cases and 123 controls. ROC curve (<strong>g</strong>) and confusion matrix (<strong>h</strong>) for 24 stage I and 8 stage II ovarian cancer and 123 controls.</p></figcaption></figure></section><section id="Sec19"><h3 class="pmc_sec_title">Neomers detect early-stage ovarian cancer</h3>
<p id="Par33">We next set out to test how our neomer classifier performs with a low amount of tumor-associated neomers. We focused on ovarian cancer which only has 12,788 associated neomers (amongst our analyzed TCGA WGS datasets) (Data <a href="#MOESM3" class="usa-link">S1</a>). In addition, as ovarian cancer does not have an effective screening test<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, most women are diagnosed at stages IIIC and IV, when 5-year survival rates are 39% and 17%, respectively<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>, reflecting a pressing need for early detection methods. We obtained plasma samples and generated cfDNA from 70 patients (Data <a href="#MOESM3" class="usa-link">S3</a>) with a median age of 57 ± 16.1 years. Our analyzed cohort comprises 45.7% early stage samples (24 stage I and 8 stage II), and the rest from late stage (34 stage III and 4 stage IV). Samples were similarly paired-end sequenced at a genomic depth of 5–10× coverage (<em>C</em>) and analyzed for neomers. As controls, we used female donors from our control donor group, totaling 123 control cfDNA samples with a median age of 58 ± 7.9 years (additional information on this cohort is provided in Data <a href="#MOESM3" class="usa-link">S3</a>). Using the same classifier as before, we obtained an AUC of 0.89 and a sensitivity of 62.86% with a specificity of 95.12% for all stages (Fig. <a href="#Fig3" class="usa-link">3e, f</a>). Importantly, for early stages (stage I and II), we obtained an AUC of 0.93 and sensitivity of 71.9% with specificity of 95.12% (Fig. <a href="#Fig3" class="usa-link">3g, h</a>). In addition, we carried out a downsampling analysis and we find that already at 3× coverage we can obtain high accuracy (Fig. <a href="#MOESM1" class="usa-link">S2</a>). Combined, these results suggest that neomers can detect cancer even with a low tumor mutation burden and, more importantly, could potentially be used to diagnose early-stage ovarian cancer for which there is currently no available detection method. We also compared our approach to fragmentomics-based algorithms. We found that after implementing a fragmentomics-based stochastic gradient boosting model on our dataset, using 10-fold cross-validation, we obtain AUCs of 0.83 for lung cancer and 0.87 for ovarian cancer (Fig. <a href="#MOESM1" class="usa-link">S6A, B</a>)<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Additionally, we extracted 256 end motifs from all 167 basepair fragments across the genome and used a SVM and LR model to differentiate healthy from cancer samples<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. This model had an AUC of 0.75 and 0.81, respectively for lung cancer and AUC of 0.93 and 0.94 for ovarian cancer (Fig. <a href="#MOESM1" class="usa-link">S6C, D</a>). Finally, we also implemented a stochastic gradient boosting model for lung cancer detection using coverage <em>Z</em>-scores derived from 39 autosomal arms (AUC 0.87 for lung cancer, AUC 0.93 for ovarian cancer) (Fig. <a href="#MOESM1" class="usa-link">S6E, F</a>)<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Our analyses show that the neomer-based method performs comparably or better than existing fragmentomic approaches for cancer detection using the same cohort and preprocessing pipeline. Notably, it achieved higher or similar AUCs across models for both lung and ovarian cancer, highlighting its robustness and potential clinical utility.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Neomers alter promoter activity</h3>
<p id="Par34">Only a small number of mutations in gene regulatory elements have been found to be associated with cancer<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>,<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. As our results show that driver mutations are enriched for neomers (Fig. <a href="#Fig1" class="usa-link">1d</a>) and the majority of our neomers (98%) lie in noncoding regions, we hypothesized that neomer-creating mutations in noncoding regions could be used to detect cancer-associated mutations in gene regulatory elements that have a functional consequence. Of note, the top patient recurrent neomer was in the <em>TERT</em> promoter (Table <a href="#Tab1" class="usa-link">1</a>), which is associated with numerous cancers<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. We chose to focus on prostate cancer since it is associated with a relatively small number of neomers (<em>N</em> = 4621; median per patient = 29.5), allowing us to characterize all the detected neomers for this cancer via MPRA in the following section. Moreover, 80–90% of prostate cancer is dependent on androgen receptor (AR) signaling<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>, allowing us to use LNCaP-FGC cells, which are androgen-dependent, for our reporter assays. We initially set out to test neomers found in promoters for luciferase reporter assays (Fig. <a href="#Fig4" class="usa-link">4a</a>) using the following criteria: (i) neomers that reside in a promoter based on ENCODE annotations<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>; (ii) the gene regulated by the promoter is associated with prostate cancer. Our list (Data <a href="#MOESM3" class="usa-link">S4</a>) included neomers in: (1) a promoter between two divergent genes, <em>RPS2</em> and the lncRNA gene <em>SNHG9</em> (Fig. <a href="#Fig4" class="usa-link">4b</a>), both of which are overexpressed in prostate cancer<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>; (2) a promoter between two divergent genes, <em>TMEM127</em> and <em>CIAO1</em> (Fig. <a href="#Fig4" class="usa-link">4c</a>), with the former being downregulated in prostate cancer<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>; (3) a promoter between two divergent genes, <em>TTC23</em> and <em>LRRC28</em>, with the former showing aberrant splicing that relates to therapy resistance in prostate cancer cells<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>; (4) The promoter of <em>GNAI2</em>, a protein that interacts with <em>CXCR5</em>, which positively correlates with prostate cancer progression<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>; (5) A promoter between two divergent genes, <em>PRICKLE4</em> and <em>FRS3</em>, with the latter thought to affect malignant but not benign prostate cells<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Neomers alter the activity of gene regulatory elements.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371106_43856_2025_1067_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/70f6/12371106/d7b231f7b465/43856_2025_1067_Fig4_HTML.jpg" loading="lazy" id="d33e1785" height="999" width="762" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic describing the luciferase reporter assay. Created in Biorender. Ahituv, N. (2025) <a href="https://biorender.com/08yc3gj" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://biorender.com/08yc3gj</a>. Integrative Genomics Viewer track snapshot showing the location of a prostate cancer-associated neomer in the promoters of <em>RPS2</em>-<em>SNHG9</em> (<strong>b</strong>) and <em>TMEM127</em>-<em>CIAO1</em> (<strong>c</strong>). The top track shows the gene location (GENCODE V36), beneath it the ENCODE layered H3K27ac marks in this region, below that the promoter location (red line) according to ENCODE cCRE annotations and the most lower track shows the neomer sequence. <strong>d</strong> Relative luciferase units from a luciferase promoter assay (<em>n</em> = 4 per condition) containing either the wild-type (WT) or nullomer variant. Transfection efficiency was normalized using Renilla luciferase and significance is calculated using a two-way ANOVA with multiple testing and Šidák correction. <strong>e</strong> Schematic showing the lentiMPRA workflow testing 4906 wild type (WT) and neomer (NEO) sequences. <strong>f</strong> The regulatory impact of 4609 neomer creating mutations, presented as log2 fold change of neomers containing sequence over wild type sequence (<em>x</em>-axis) and ordered from highest to lowest (<em>y</em>-axis). <strong>g</strong> A neomer in the <em>CTNNA1</em> locus had the highest activating score and is postulated to lead to a gain of a TCF7 motif, as predicted by FIMO<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>.</p></figcaption></figure><p id="Par35">We cloned the promoter sequences with and without the neomer into a luciferase promoter assay vector and compared their activity in LNCaP cells (Fig. <a href="#Fig4" class="usa-link">4a</a>). Two out of the five assayed promoters showed a significant effect on reporter activity between the neomer and wild type allele (Fig. <a href="#Fig4" class="usa-link">4d</a>). For the <em>RPS2</em>-<em>SNHG9</em> promoter, the neomer led to significantly increased activity, in line with this gene being overexpressed in cancer<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. For the <em>TMEM127</em>-<em>CIAO1</em> promoter, the neomer completely abolished activity, fitting with its observed downregulation in prostate cancer<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Combined, our experimental results show that neomers could have a significant effect on promoter activity and could potentially be used to identify cancer-associated gene regulatory mutations.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Neomers alter enhancer activity</h3>
<p id="Par36">We next set out to test in a high-throughput manner whether neomers could affect the function of active gene regulatory elements, such as promoters and enhancers. We annotated all the prostate-associated neomers that reside in noncoding regions, selected for those that were recurrent in at least three patients (to increase likelihood to be cancer-associated and reduce sequences to be assayed due to MPRA limitations), obtaining 4609 loci and generated a lentivirus-based MPRA (lentiMPRA) to test the regulatory activity with and without the neomer of all these sequences (Fig. <a href="#Fig4" class="usa-link">4e</a>). Two hundred base pair sequences, where the position of the neomer is used as a center, were synthesized and cloned upstream of a minimal promoter followed by a GFP reporter gene. Lentivirus was generated and the library was infected using three technical replicates into LNCaP cells and RNA and DNA barcodes were sequenced to determine regulatory activity, as previously described<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. We observed a good correlation between RNA/DNA ratios (Pearson 0.72-0.77; Fig. <a href="#MOESM1" class="usa-link">S4A</a>) as well as for DNA and RNA barcodes separately (Fig. <a href="#MOESM1" class="usa-link">S4B, C</a>) across all three technical replicates. Out of the 4609 loci, 567 showed significant regulatory activity due to the neomer (RNA/DNA ratio ≥ 1.5). Amongst them, 94 neomers and wild type sequence pairs showed significant (<em>p</em>-value ≤ 0.05, U-Mann–Whitney test) differential enhancer activity, with 48 having increased activity due to the neomer (NEO/WT ≥ 2) and 46 decreased activity (NEO/WT ≤ 0.5) (Fig. <a href="#Fig4" class="usa-link">4F</a> and Data <a href="#MOESM3" class="usa-link">S5</a>).</p>
<p id="Par37">The neomer that showed the highest level of increased activity compared to reference (4.09-fold) is located in an intron of the catenin alpha 1 (<em>CTNNA1</em>) gene. This gene is a core member of the cadherin/catenin complex and is involved in the regulation of the Wnt/beta-catenin pathway, which has been widely studied in cancer<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>, including prostate cancer<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>, and is the target of several therapies<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>,<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. TFBS analysis of this neomer using FIMO<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> and JASPAR<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup> found that it leads to a gain of a TCF7/TCFL1 motif (MA0769.2) (Fig. <a href="#Fig4" class="usa-link">4G</a>), which is known to play a role in prostate cancer malignancy<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>,<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. A neomer residing in the third intron of the HECT and RLD domain containing E3 ubiquitin protein ligase (<em>HERC3</em>) gene led to the lowest downregulation of reporter activity (2.7-fold; Fig. <a href="#Fig4" class="usa-link">4F</a>). HERC3 is an AR target and its dysregulation has been associated with epigenetic modulations and poor outcome in prostate cancer<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. In addition, for colorectal cancer, this gene was found to be downregulated, is known to inhibit metastasis and is associated with overall lower survival<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. Taken together, our MPRA results suggest that neomers could be utilized to identify gene regulatory mutations in cancer.</p></section></section><section id="Sec22"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par38">Cancer is a DNA mutation-associated disease. Here, we show that by analyzing cancer WGS, both from tumors and cfDNA, we can find cancer-associated DNA mutations that lead to the generation of neomers, short sequences that are predominantly absent from genomes of healthy individuals. Further analyses of these sets of neomers show that they can be used not only to classify cancer tissue of origin, but also to identify additional cancer features, such as MSI or POLE deficiency, with high accuracy. Analysis of cfDNA WGS generated from lung and ovarian cancer patients finds that neomers could be used to detect cancer, including in early stages, with high specificity and sensitivity. This is particularly important for ovarian cancer, for which there is currently no effective screening test<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> with most women being diagnosed at later stages when survival rates are low<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. We were able to detect early-stage (stage I and II) cancer with an AUC of 0.93. Finally, using promoter assays and MPRA, we show that neomers can be used to detect cancer-associated mutations that have a functional effect on regulatory element activity.</p>
<p id="Par39">We utilized 2577 patients from 21 different cancerous tissues to develop a cancer tissue of origin classifier. Overall, we observed almost no overlap between neomers found in different cancer types. This allowed us to detect cancer tissue of origin with extremely high specificity and accuracy (F1 score of 0.92 and an accuracy of 0.99), performing better than recent deep learning models<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. In general, the classifier has better performance for cancer types with more patients and high mutation burden (Fig. <a href="#Fig2" class="usa-link">2a, b</a>). Our analyses also showed that other than tissue origin, nullomers can also be used to detect additional cancer features. Future work that will utilize additional cases and metadata would allow us to test whether nullomers and neomers can diagnose additional tumor features and also detect other cancer characteristics such as chance of recurrence, predict and/or monitor response to treatment, disease-free survival, and overall survival.</p>
<p id="Par40">For cfDNA, despite various known challenges, including fragmentation and low levels of ctDNA, we obtained high AUC scores, and our results for lung cancer compare favorably with a recent study based on metabolite profiling<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. Lung cancer, being one of five cancer types with a widespread approved screening option, utilizes a low-dosage CT scan (LDCT) for populations at risk (age, smoking, etc.). LDCT’s sensitivity was 93%, specificity was 84% when taking all stages together<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>–<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>. Recently, the employment of a sequencing-based fragmentomics assay followed by LDCT provided a stage-weighted overall sensitivity of 84% (95% CI: 78–90%)<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup>. For ovarian cancer, which is associated with a low tumor mutation burden and has hitherto been extremely challenging to detect at early stages, we managed to detect, even with low sample numbers (<em>N</em> = 70), both late and more importantly early stages (stages I and II), obtaining an AUC of 0.93 and sensitivity of 71.9% with specificity of 94.44%. Our higher AUC scores in early stages could be due to larger sample sizes at this stage and are in line with other publications that find higher scores in these early stages compared to late stages<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>,<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup>.</p>
<p id="Par41">It is worth noting that our classifier did not benefit from removing neomers that can arise due to common germline variants. Nevertheless, we expect that knowing the donor’s ancestry could allow us to further improve the classifier. However, this would require more detailed metadata than was available in the present study. Since the number of neomers increases with the number of tumor samples profiled, obtaining additional WGS datasets from tumors and matched controls would also be beneficial in improving the ability of our classifier to diagnose both these and other cancers. Additional testing across multiple cancer types and in a larger number of samples is required to determine if the specificity that we obtained when using neomers to classify ICGC primary tumor samples can be maintained. We assume that, as cfDNA only contains a small fraction of tumor DNA, the signal will be much weaker.</p>
<p id="Par42">Our cfDNA detection approach has several advantages over current methods: (1) Detecting short DNA sequences that are enriched in cancer samples provides an easy-to-use diagnostic that could allow detection from low amounts of ctDNA. In addition to sequencing-based assays, alternate techniques could potentially be used, such as CRISPR-based diagnostic tools<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup>, that can also allow the testing of thousands of sequences in parallel<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>. In addition, with neomer-based diagnostics potentially not needing large amounts of starting material, cfDNA could be collected from urine, sputum, saliva, or other bodily fluids, which were shown to be a viable but reduced source of cfDNA<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>,<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup>. (2) Our WGS approach is applicable to more sparsely sequenced samples. As also shown in our downsampling analysis, we obtain similar accuracy for both lung and ovarian cancer already at 3× coverage (Fig. <a href="#MOESM1" class="usa-link">S2</a>). It is worth noting that this study was performed using standard Illumina sequencing. The use of sequencing technologies that provide longer reads<sup><a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup> and/or improved accuracy<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup> could further improve the ability of neomers to detect cancer. (3) As our diagnostic is based on neomer detection followed by genome alignments of nullomer encompassing reads, it is extremely effective from a computational standpoint. 4) Neomers could easily be combined with other sequence or analyte-based cancer biomarkers and risk factors to improve the diagnostic positive predictive value. For example, it was shown that combining a blood test that detects both protein biomarkers and DNA mutations, along with positron emission tomography-computed tomography (PET-CT), could detect multiple cancers<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>. In another example, the use of cfDNA fragmentation patterns combined with CT imaging, clinical risk factors, and serum levels of carcinoembryonic antigen significantly increased the ability to diagnose lung cancer<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Adding neomers to other known cancer-associated mutations (coding, regulatory, CNVs, etc.) in the screening of cfDNA could also increase sensitivity and specificity. This should be relatively straightforward, as our neomer detection method is based on standard sequencing technologies. In summary, coupling neomer-based diagnostics to existing cancer biomarkers and risk factors could improve the power to detect various cancer subtypes.</p>
<p id="Par43">As nullomers/neomers do not exist in the human genome they could also be exceptional candidates as suicide genes, as shown previously for three nullomers<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup>, or for neoantigens, to be targeted via immunotherapy. Previous work has shown that minimal absent words, short sequences that are absent from a genome or proteome, could be used to identify phosphorylation sites of high confidence, some of which could be associated with cancer<sup><a href="#CR92" class="usa-link" aria-describedby="CR92">92</a></sup>. Analysis of the Immune Epitope Database of validated antigens<sup><a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup> found that 13 of the recurrent coding neomers can create neoantigens with predicted strong binding levels that were subsequently validated (Data <a href="#MOESM3" class="usa-link">S6</a>). From the 1700 neoantigens with strong binding levels, only 1.72 (<em>p</em>-value &lt; 1e-8, hypergeometric test) is expected to correspond to a neomer, suggesting that missense mutations also resulting in neomers are 7-fold more likely to also generate strongly binding neoantigens.</p>
<p id="Par44">Neomers can be used as a tool to identify cancer-associated gene regulatory mutations. Amongst the 210 prostate cancer promoter neomers, we selected five promoters and found that two of them significantly affected promoter activity due to the neomer. Their difference in activity was in line with the gene’s expression change in prostate cancer, with <em>RPS2</em>-<em>SNHG9</em> having increased activity, fitting with <em>RPS2</em> overexpression in prostate cancer<sup><a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup> and <em>TMEM127</em>-<em>CIAO1</em> abolishing activity, in line with <em>TMEM127</em> observed downregulation in cancer<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Our MPRA library of 4609 neomer-causing mutations in enhancers revealed that 2.6% can change gene expression by &gt;1.5-fold, suggesting that a subset of these mutations could have important functional consequences in cancer. While we obtained good correlations between our three MPRA replicates, MPRA has some caveats that should be noted: (1) testing genes outside of their genomic context; (2) not being able to assign the target gene; (3) utilizing a minimal promoter instead of the target promoter; (4) using cell lines and one prostate cancer cell type. Nonetheless, understanding tumor onset and its transition to a metastatic cancer has been limited mostly to coding genes or their pre-determined regulatory sequences<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a>,<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a></sup>. With the vast majority of mutations falling into non-coding regions, making them much more challenging to interpret, novel approaches are required for identifying mutations that have functional consequences. Here, we show that by prioritizing mutations that result in neomers and utilizing MPRA, it is possible to shortlist candidates that can have a phenotypic impact. This approach can be applied to any cancer type and thus holds the potential to expand our catalog of functionally relevant cancer driver non-coding mutations.</p>
<p id="Par45">In summary, we show that neomers can provide a powerful tool for cancer diagnosis. As they can easily be detected via sequence or CRISPR-based tools, it should be straightforward to integrate them in current routine cancer diagnostic tests and their use could increase the sensitivity and specificity of these tests. Combining neomer-based screening with clinical characteristics and additional diagnostic tools and features could increase the positive predictive value. In addition, as cfDNA could also be isolated from urine and saliva, and detection of these sequences only requires a relatively small amount of DNA, neomer-based diagnosis could be carried out in a non-invasive manner. The utilization of neomers in the clinic will require additional work. This includes standardization of the collection and processing of patient samples, clinical trials that use neomers for cancer detection in a Clinical Laboratory Improvement Amendments certified lab and further refinement and standardization of our algorithms. Our work also suggests that neomers could be used to highlight cancer-associated gene regulatory mutations, which have been difficult to identify. Further high-throughput characterization of these mutations could allow the detection of bona fide cancer-associated functional regulatory mutations that could be used for diagnosis and treatment.</p>
<section id="Sec23"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par46">Further information on research design is available in the <a href="#MOESM4" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371106/bin/43856_2025_1067_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (1.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12371106/bin/43856_2025_1067_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">43856_2025_1067_MOESM2_ESM.pdf</a><sup> (32.9KB, pdf) </sup><p>Description of Additional Supplementary files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12371106/bin/43856_2025_1067_MOESM3_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplemental Data 1-6</a><sup> (672KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12371106/bin/43856_2025_1067_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting summary</a><sup> (2.2MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We wish to thank the individuals who participated in this study. We dedicate this article to Felix Feng (UCSF) and thank him for his advice, support and inspiration for this study. This work was supported in part by: Benioff Initiative for Prostate Cancer Research (NA). UCSF Catalyst award (NA). UCSF Innovations Ventures Philanthropy Fund (NA). National Human Genome Research Institute grant number UM1HG011966 (NA). Food and Drug Administration (FDA) of the U.S. Department of Health and Human Services (HHS) as part of a financial assistance award [Center of Excellence in Regulatory Science and Innovation grant to University of California, San Francisco, U01FD005978] award number 3U01FD005978-09S3 (NA, JC, MH). Core funding from the Wellcome Trust (MH). Core funding from the Evergrande Center (MH). Core funding from the Penn State College of Medicine (IGS). Elsa U. Pardee Foundation (IGS). Four Diamonds (IGS). Penn State Cancer Institute (IGS). The contents are those of the author(s) and do not necessarily represent the official views of, nor an endorsement, by FDA/HHS, or the U.S. Government.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: I.G.S., O.Y.B., I.M., M.H., N.A. Computational methodology: I.G.S., I.M., C.C., R.E., E.C., G.E.P., M.K., M.H. Analysis and visualization: I.G.S., O.Y.B., I.M., C.C., M.P., A.N., M.M., M.K., M.H., N.A. Provided samples: G.P., J.S.W., C.A.M., F.F., I.V., N.S., E.P., A.C., K.S., J.C. Experimental assays: O.Y.B., C.C., R.B., J.S., D.L.C., R.E., J.K. Funding acquisition: I.G.S., M.H., N.A. Project administration: I.G.S., M.H., N.A. Supervision: I.G.S., M.H., N.A. Writing—original draft: I.G.S., O.Y.B., I.M., M.H., and N.A.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par47"><em>Communications Medicine</em> thanks the anonymous reviewers for their contribution to the peer review of this work.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The source data for Fig. <a href="#Fig1" class="usa-link">1</a> is found in Data <a href="#MOESM3" class="usa-link">S1</a>, for Fig. <a href="#Fig3" class="usa-link">3</a> in Data <a href="#MOESM3" class="usa-link">S3</a>, and for Fig. <a href="#Fig4" class="usa-link">4</a> in Data <a href="#MOESM3" class="usa-link">S5</a>. MPRA sequencing data was deposited in the NCBI short read archive (SRA) with accession code PRJNA917083. ICGC samples are publicly available and can be accessed from <a href="https://docs.icgc-argo.org/docs/data-access/icgc-25k-data" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://docs.icgc-argo.org/docs/data-access/icgc-25k-data</a>. Due to patient data confidentiality, WGS sequencing data is only available upon request. For WGS sequencing, contact Nadav Ahituv.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>We generated an easy-to-use software package that enables performing nullomer cancer analyses from sequence-based datasets<sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup>. The package is composed of six functions: (1) EnumerateNullomers, which extracts all nullomers of specified kmer lengths in a FASTA sample; (2) ExtractMutationNullomers, which finds all mutations that cause the resurfacing of a list of nullomers; (3) IdentifyRecurrentNullomers, which identifies nullomers that recur in a dataset through mutagenesis; (4) FindAlmostNullomers, which identifies the positions that can create a list of nullomers genome-wide for every possible substitution and single base-pair insertion and deletion; (5) FindNullomerVariants, which removes nullomers that are likely to result from common variants in a user specified variant VCF file; (6) FindDNANullomersFromReads, which performs the identification of nullomers in raw read samples. The package can be found at 10.5281/zenodo.1091017397 and the documentation on Read the Docs (<a href="https://nullomerator.readthedocs.io/en/latest/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://nullomerator.readthedocs.io/en/latest/#</a>) provides in-depth details on how to use the software.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par48">The authors declare the following competing interests: I.G.S., O.Y.B., I.M., M.H., and N.A. have filed patent applications covering embodiments and concepts disclosed in the manuscript.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Ilias Georgakopoulos-Soares, Ofer Yizhar-Barnea, Ioannis Mouratidis.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Ilias Georgakopoulos-Soares, Email: izg5139@psu.edu.</p>
<p>Martin Hemberg, Email: mhemberg@bwh.harvard.edu.</p>
<p>Nadav Ahituv, Email: nadav.ahituv@ucsf.edu.</p></section><section id="sec29"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s43856-025-01067-3.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Cancer. <a href="https://www.who.int/news-room/fact-sheets/detail/cancer" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.who.int/news-room/fact-sheets/detail/cancer</a>.</cite>
</li>
<li id="CR2">
<span class="label">2.</span><cite>Siegel, R. L., Miller, K. D. &amp; Jemal, A. Cancer statistics, 2020. <em>CA Cancer J. Clin.</em><strong>70</strong>, 7–30 (2020).
</cite> [<a href="https://doi.org/10.3322/caac.21590" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31912902/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Siegel,%20R.%20L.,%20Miller,%20K.%20D.%20&amp;%20Jemal,%20A.%20Cancer%20statistics,%202020.%20CA%20Cancer%20J.%20Clin.70,%207%E2%80%9330%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Hawkes, N. Cancer survival data emphasise importance of early diagnosis. <em>BMJ</em><strong>364</strong>, l408 (2019).</cite> [<a href="https://doi.org/10.1136/bmj.l408" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30683652/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Etzioni, R. et al. The case for early detection. <em>Nat. Rev. Cancer</em><strong>3</strong>, 243–252 (2003).
</cite> [<a href="https://doi.org/10.1038/nrc1041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12671663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Etzioni,%20R.%20et%20al.%20The%20case%20for%20early%20detection.%20Nat.%20Rev.%20Cancer3,%20243%E2%80%93252%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Heitzer, E., Auinger, L. &amp; Speicher, M. R. Cell-free DNA and apoptosis: how dead cells inform about the living. <em>Trends Mol. Med.</em><strong>26</strong>, 519–528 (2020).
</cite> [<a href="https://doi.org/10.1016/j.molmed.2020.01.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32359482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heitzer,%20E.,%20Auinger,%20L.%20&amp;%20Speicher,%20M.%20R.%20Cell-free%20DNA%20and%20apoptosis:%20how%20dead%20cells%20inform%20about%20the%20living.%20Trends%20Mol.%20Med.26,%20519%E2%80%93528%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Bronkhorst, A. J., Ungerer, V. &amp; Holdenrieder, S. The emerging role of cell-free DNA as a molecular marker for cancer management. <em>Biomol. Detect. Quantif.</em><strong>17</strong>, 100087 (2019).
</cite> [<a href="https://doi.org/10.1016/j.bdq.2019.100087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6425120/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30923679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bronkhorst,%20A.%20J.,%20Ungerer,%20V.%20&amp;%20Holdenrieder,%20S.%20The%20emerging%20role%20of%20cell-free%20DNA%20as%20a%20molecular%20marker%20for%20cancer%20management.%20Biomol.%20Detect.%20Quantif.17,%20100087%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Brill, J. V. Screening for cancer: the economic, medical, and psychosocial issues. <em>Am. J. Manag. Care</em><strong>26</strong>, S300–S306 (2020).
</cite> [<a href="https://doi.org/10.37765/ajmc.2020.88534" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33200894/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brill,%20J.%20V.%20Screening%20for%20cancer:%20the%20economic,%20medical,%20and%20psychosocial%20issues.%20Am.%20J.%20Manag.%20Care26,%20S300%E2%80%93S306%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Zill, O. A., Banks, K. C., Fairclough, S. R. &amp; Mortimer, S. A. The landscape of actionable genomic alterations in cell-free circulating tumor DNA from 21,807 advanced cancer patients. <em>Clin. Cancer Res</em>. 3528–3538 (2018).</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-17-3837" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29776953/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Saghafinia, S., Mina, M., Riggi, N., Hanahan, D. &amp; Ciriello, G. Pan-cancer landscape of aberrant DNA methylation across human tumors. <em>Cell Rep.</em><strong>25</strong>, 1066–1080.e8 (2018).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2018.09.082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30355485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saghafinia,%20S.,%20Mina,%20M.,%20Riggi,%20N.,%20Hanahan,%20D.%20&amp;%20Ciriello,%20G.%20Pan-cancer%20landscape%20of%20aberrant%20DNA%20methylation%20across%20human%20tumors.%20Cell%20Rep.25,%201066%E2%80%931080.e8%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Sadeh, R. et al. ChIP-seq of plasma cell-free nucleosomes identifies gene expression programs of the cells of origin. <em>Nat. Biotechnol</em>. 10.1038/s41587-020-00775-6 (2021).</cite> [<a href="https://doi.org/10.1038/s41587-020-00775-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7610786/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33432199/" class="usa-link">PubMed</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Barbany, G. et al. Cell-free tumour DNA testing for early detection of cancer-a potential future tool. <em>J. Intern. Med.</em><strong>286</strong>, 118–136 (2019).
</cite> [<a href="https://doi.org/10.1111/joim.12897" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30861222/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barbany,%20G.%20et%20al.%20Cell-free%20tumour%20DNA%20testing%20for%20early%20detection%20of%20cancer-a%20potential%20future%20tool.%20J.%20Intern.%20Med.286,%20118%E2%80%93136%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Liu, J. et al. Genome-wide cell-free DNA methylation analyses improve accuracy of non-invasive diagnostic imaging for early-stage breast cancer. <em>Mol. Cancer</em><strong>20</strong>, 36 (2021).
</cite> [<a href="https://doi.org/10.1186/s12943-021-01330-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7893735/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33608029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20J.%20et%20al.%20Genome-wide%20cell-free%20DNA%20methylation%20analyses%20improve%20accuracy%20of%20non-invasive%20diagnostic%20imaging%20for%20early-stage%20breast%20cancer.%20Mol.%20Cancer20,%2036%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Magbanua, M. J. M. et al. Cell-free DNA concentration as a biomarker of response and recurrence in HER2-negative breast cancer receiving neoadjuvant chemotherapy. <em>Clin. Cancer Res.</em><strong>30</strong>, 2444–2451 (2024).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-23-2928" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11147708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38470545/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Magbanua,%20M.%20J.%20M.%20et%20al.%20Cell-free%20DNA%20concentration%20as%20a%20biomarker%20of%20response%20and%20recurrence%20in%20HER2-negative%20breast%20cancer%20receiving%20neoadjuvant%20chemotherapy.%20Clin.%20Cancer%20Res.30,%202444%E2%80%932451%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Razavi, P. et al. High-intensity sequencing reveals the sources of plasma circulating cell-free DNA variants. <em>Nat. Med.</em><strong>25</strong>, 1928–1937 (2019).
</cite> [<a href="https://doi.org/10.1038/s41591-019-0652-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7061455/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31768066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Razavi,%20P.%20et%20al.%20High-intensity%20sequencing%20reveals%20the%20sources%20of%20plasma%20circulating%20cell-free%20DNA%20variants.%20Nat.%20Med.25,%201928%E2%80%931937%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Shi, J., Zhang, R., Li, J. &amp; Zhang, R. Size profile of cell-free DNA: a beacon guiding the practice and innovation of clinical testing. <em>Theranostics</em><strong>10</strong>, 4737–4748 (2020).
</cite> [<a href="https://doi.org/10.7150/thno.42565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7163439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32308746/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi,%20J.,%20Zhang,%20R.,%20Li,%20J.%20&amp;%20Zhang,%20R.%20Size%20profile%20of%20cell-free%20DNA:%20a%20beacon%20guiding%20the%20practice%20and%20innovation%20of%20clinical%20testing.%20Theranostics10,%204737%E2%80%934748%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Ji, L. et al. Methylated DNA is over-represented in whole-genome bisulfite sequencing data. <em>Front. Genet.</em><strong>5</strong>, 341 (2014).
</cite> [<a href="https://doi.org/10.3389/fgene.2014.00341" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4204604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25374580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ji,%20L.%20et%20al.%20Methylated%20DNA%20is%20over-represented%20in%20whole-genome%20bisulfite%20sequencing%20data.%20Front.%20Genet.5,%20341%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Worm Ørntoft, M.-B. Review of blood-based colorectal cancer screening: how far are circulating cell-free DNA methylation markers from clinical implementation? <em>Clin. Colorectal Cancer</em><strong>17</strong>, e415–e433 (2018).
</cite> [<a href="https://doi.org/10.1016/j.clcc.2018.02.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29678513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Worm%20%C3%98rntoft,%20M.-B.%20Review%20of%20blood-based%20colorectal%20cancer%20screening:%20how%20far%20are%20circulating%20cell-free%20DNA%20methylation%20markers%20from%20clinical%20implementation?%20Clin.%20Colorectal%20Cancer17,%20e415%E2%80%93e433%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Warton, K. &amp; Samimi, G. Methylation of cell-free circulating DNA in the diagnosis of cancer. <em>Front. Mol. Biosci.</em><strong>2</strong>, 13 (2015).
</cite> [<a href="https://doi.org/10.3389/fmolb.2015.00013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4428375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25988180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Warton,%20K.%20&amp;%20Samimi,%20G.%20Methylation%20of%20cell-free%20circulating%20DNA%20in%20the%20diagnosis%20of%20cancer.%20Front.%20Mol.%20Biosci.2,%2013%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Hampikian, G. &amp; Andersen, T. Absent sequences: nullomers and primes. in <em>Biocomputing 2007</em> 355–366 10.1142/9789812772435_0034 (World Scientific, 2006).</cite> [<a href="https://doi.org/10.1142/9789812772435_0034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17990505/" class="usa-link">PubMed</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Vergni, D. &amp; Santoni, D. Nullomers and high order nullomers in genomic sequences. <em>PLoS ONE</em><strong>11</strong>, e0164540 (2016).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0164540" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5132333/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27906971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vergni,%20D.%20&amp;%20Santoni,%20D.%20Nullomers%20and%20high%20order%20nullomers%20in%20genomic%20sequences.%20PLoS%20ONE11,%20e0164540%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Georgakopoulos-Soares, I., Yizhar-Barnea, O., Mouratidis, I., Hemberg, M. &amp; Ahituv, N. Absent from DNA and protein: genomic characterization of nullomers and nullpeptides across functional categories and evolution. <em>Genome Biol.</em><strong>22</strong>, 245 (2021).
</cite> [<a href="https://doi.org/10.1186/s13059-021-02459-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8386077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34433494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Georgakopoulos-Soares,%20I.,%20Yizhar-Barnea,%20O.,%20Mouratidis,%20I.,%20Hemberg,%20M.%20&amp;%20Ahituv,%20N.%20Absent%20from%20DNA%20and%20protein:%20genomic%20characterization%20of%20nullomers%20and%20nullpeptides%20across%20functional%20categories%20and%20evolution.%20Genome%20Biol.22,%20245%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Montgomery, A. et al. Utilizing nullomers in cell-free RNA for early cancer detection. <em>Cancer Gene Ther.</em><strong>31</strong>, 861–870 (2024).
</cite> [<a href="https://doi.org/10.1038/s41417-024-00741-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11192629/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38351138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Montgomery,%20A.%20et%20al.%20Utilizing%20nullomers%20in%20cell-free%20RNA%20for%20early%20cancer%20detection.%20Cancer%20Gene%20Ther.31,%20861%E2%80%93870%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>The Cancer Genome Atlas Program. <a href="https://www.cancer.gov/tcga" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cancer.gov/tcga</a> (2018).</cite>
</li>
<li id="CR24">
<span class="label">24.</span><cite>Jiao, W. et al. A deep learning system accurately classifies primary and metastatic cancers using passenger mutation patterns. <em>Nat. Commun.</em><strong>11</strong>, 728 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-019-13825-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7002586/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32024849/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiao,%20W.%20et%20al.%20A%20deep%20learning%20system%20accurately%20classifies%20primary%20and%20metastatic%20cancers%20using%20passenger%20mutation%20patterns.%20Nat.%20Commun.11,%20728%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Henderson, J. T., Webber, E. M. &amp; Sawaya, G. F. <em>Screening for Ovarian Cancer: An Updated Evidence Review for the U.S. Preventive Services Task Force</em> (Agency for Healthcare Research and Quality (US), 2018).</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/29648765/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Heintz, A. P. M. et al. Carcinoma of the ovary. <em>Int. J. Gynecol. Obstet.</em><strong>95</strong>, S161–S192 (2006).</cite> [<a href="https://doi.org/10.1016/S0020-7292(06)60033-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29644669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heintz,%20A.%20P.%20M.%20et%20al.%20Carcinoma%20of%20the%20ovary.%20Int.%20J.%20Gynecol.%20Obstet.95,%20S161%E2%80%93S192%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Gonzalez-Perez, A. et al. IntOGen-mutations identifies cancer drivers across tumor types. <em>Nat. Methods</em><strong>10</strong>, 1081–1082 (2013).
</cite> [<a href="https://doi.org/10.1038/nmeth.2642" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5758042/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24037244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gonzalez-Perez,%20A.%20et%20al.%20IntOGen-mutations%20identifies%20cancer%20drivers%20across%20tumor%20types.%20Nat.%20Methods10,%201081%E2%80%931082%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Chang, C.-C. &amp; Lin, C.-J. LIBSVM: a library for support vector machines. <em>ACM Trans. Intell. Syst. Technol.</em><strong>2</strong>, 1–27 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Chang,%20C.-C.%20&amp;%20Lin,%20C.-J.%20LIBSVM:%20a%20library%20for%20support%20vector%20machines.%20ACM%20Trans.%20Intell.%20Syst.%20Technol.2,%201%E2%80%9327%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Zou, X. et al. A systematic CRISPR screen defines mutational mechanisms underpinning signatures caused by replication errors and endogenous DNA damage. <em>Nat. Cancer</em><strong>2</strong>, 643–657 (2021).
</cite> [<a href="https://doi.org/10.1038/s43018-021-00200-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7611045/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34164627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zou,%20X.%20et%20al.%20A%20systematic%20CRISPR%20screen%20defines%20mutational%20mechanisms%20underpinning%20signatures%20caused%20by%20replication%20errors%20and%20endogenous%20DNA%20damage.%20Nat.%20Cancer2,%20643%E2%80%93657%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Georgakopoulos-Soares, I. et al. Transcription-coupled repair and mismatch repair contribute towards preserving genome integrity at mononucleotide repeat tracts. <em>Nat. Commun.</em><strong>11</strong>, 1980 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-15901-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7181645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32332764/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Georgakopoulos-Soares,%20I.%20et%20al.%20Transcription-coupled%20repair%20and%20mismatch%20repair%20contribute%20towards%20preserving%20genome%20integrity%20at%20mononucleotide%20repeat%20tracts.%20Nat.%20Commun.11,%201980%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Pećina-Šlaus, N., Kafka, A., Salamon, I. &amp; Bukovac, A. Mismatch repair pathway, genome stability and cancer. <em>Front. Mol. Biosci.</em><strong>7</strong>, 122 (2020).
</cite> [<a href="https://doi.org/10.3389/fmolb.2020.00122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7332687/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32671096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pe%C4%87ina-%C5%A0laus,%20N.,%20Kafka,%20A.,%20Salamon,%20I.%20&amp;%20Bukovac,%20A.%20Mismatch%20repair%20pathway,%20genome%20stability%20and%20cancer.%20Front.%20Mol.%20Biosci.7,%20122%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Garmezy, B. et al. Correlation of pathogenic POLE mutations with clinical benefit to immune checkpoint inhibitor therapy. <em>J. Clin. Orthod.</em><strong>38</strong>, 3008–3008 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Garmezy,%20B.%20et%20al.%20Correlation%20of%20pathogenic%20POLE%20mutations%20with%20clinical%20benefit%20to%20immune%20checkpoint%20inhibitor%20therapy.%20J.%20Clin.%20Orthod.38,%203008%E2%80%933008%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. <em>Nature</em><strong>581</strong>, 434–443 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2308-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7334197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32461654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karczewski,%20K.%20J.%20et%20al.%20The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141,456%20humans.%20Nature581,%20434%E2%80%93443%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Cristiano, S. et al. Genome-wide cell-free DNA fragmentation in patients with cancer. <em>Nature</em><strong>570</strong>, 385–389 (2019).
</cite> [<a href="https://doi.org/10.1038/s41586-019-1272-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6774252/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31142840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cristiano,%20S.%20et%20al.%20Genome-wide%20cell-free%20DNA%20fragmentation%20in%20patients%20with%20cancer.%20Nature570,%20385%E2%80%93389%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Danecek, P. et al. Twelve years of SAMtools and BCFtools. <em>Gigascience</em><strong>10</strong>, giab008 (2021).</cite> [<a href="https://doi.org/10.1093/gigascience/giab008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7931819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33590861/" class="usa-link">PubMed</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Wong, D. et al. Cell-free DNA from germline TP53 mutation carriers reflect cancer-like fragmentation patterns. <em>Nat. Commun.</em><strong>15</strong>, 7386 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-51529-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11349871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39191772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wong,%20D.%20et%20al.%20Cell-free%20DNA%20from%20germline%20TP53%20mutation%20carriers%20reflect%20cancer-like%20fragmentation%20patterns.%20Nat.%20Commun.15,%207386%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Gordon, M. G. et al. lentiMPRA and MPRAflow for high-throughput functional characterization of gene regulatory elements. <em>Nat. Protoc.</em><strong>15</strong>, 2387–2412 (2020).
</cite> [<a href="https://doi.org/10.1038/s41596-020-0333-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7550205/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32641802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gordon,%20M.%20G.%20et%20al.%20lentiMPRA%20and%20MPRAflow%20for%20high-throughput%20functional%20characterization%20of%20gene%20regulatory%20elements.%20Nat.%20Protoc.15,%202387%E2%80%932412%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium. Pan-cancer analysis of whole genomes. <em>Nature</em><strong>578</strong>, 82–93 (2020).</cite>
</li>
<li id="CR39">
<span class="label">39.</span><cite>Prior, I. A., Lewis, P. D. &amp; Mattos, C. A comprehensive survey of Ras mutations in cancer. <em>Cancer Res.</em><strong>72</strong>, 2457–2467 (2012).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-11-2612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3354961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22589270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prior,%20I.%20A.,%20Lewis,%20P.%20D.%20&amp;%20Mattos,%20C.%20A%20comprehensive%20survey%20of%20Ras%20mutations%20in%20cancer.%20Cancer%20Res.72,%202457%E2%80%932467%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Muñoz-Maldonado, C., Zimmer, Y. &amp; Medová, M. A comparative analysis of individual RAS mutations in cancer biology. <em>Front. Oncol.</em><strong>9</strong>, 1088 (2019).
</cite> [<a href="https://doi.org/10.3389/fonc.2019.01088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6813200/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31681616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mu%C3%B1oz-Maldonado,%20C.,%20Zimmer,%20Y.%20&amp;%20Medov%C3%A1,%20M.%20A%20comparative%20analysis%20of%20individual%20RAS%20mutations%20in%20cancer%20biology.%20Front.%20Oncol.9,%201088%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Vinagre, J. et al. Frequency of TERT promoter mutations in human cancers. <em>Nat. Commun.</em><strong>4</strong>, 2185 (2013).
</cite> [<a href="https://doi.org/10.1038/ncomms3185" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23887589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vinagre,%20J.%20et%20al.%20Frequency%20of%20TERT%20promoter%20mutations%20in%20human%20cancers.%20Nat.%20Commun.4,%202185%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Dratwa, M., Wysoczańska, B., Łacina, P., Kubik, T. &amp; Bogunia-Kubik, K. TERT-regulation and roles in cancer formation. <em>Front. Immunol.</em><strong>11</strong>, 589929 (2020).
</cite> [<a href="https://doi.org/10.3389/fimmu.2020.589929" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7717964/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33329574/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dratwa,%20M.,%20Wysocza%C5%84ska,%20B.,%20%C5%81acina,%20P.,%20Kubik,%20T.%20&amp;%20Bogunia-Kubik,%20K.%20TERT-regulation%20and%20roles%20in%20cancer%20formation.%20Front.%20Immunol.11,%20589929%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Heidenreich, B., Rachakonda, P. S., Hemminki, K. &amp; Kumar, R. TERT promoter mutations in cancer development. <em>Curr. Opin. Genet. Dev.</em><strong>24</strong>, 30–37 (2014).
</cite> [<a href="https://doi.org/10.1016/j.gde.2013.11.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24657534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heidenreich,%20B.,%20Rachakonda,%20P.%20S.,%20Hemminki,%20K.%20&amp;%20Kumar,%20R.%20TERT%20promoter%20mutations%20in%20cancer%20development.%20Curr.%20Opin.%20Genet.%20Dev.24,%2030%E2%80%9337%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Song, J. H. et al. Small-molecule-targeting hairpin loop of hTERT promoter G-quadruplex induces cancer cell death. <em>Cell Chem. Biol.</em><strong>26</strong>, 1110–1121.e4 (2019).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2019.04.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6713458/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31155510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song,%20J.%20H.%20et%20al.%20Small-molecule-targeting%20hairpin%20loop%20of%20hTERT%20promoter%20G-quadruplex%20induces%20cancer%20cell%20death.%20Cell%20Chem.%20Biol.26,%201110%E2%80%931121.e4%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Bell, R. J. A. et al. Cancer. The transcription factor GABP selectively binds and activates the mutant TERT promoter in cancer. <em>Science</em><strong>348</strong>, 1036–1039 (2015).
</cite> [<a href="https://doi.org/10.1126/science.aab0015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4456397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25977370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bell,%20R.%20J.%20A.%20et%20al.%20Cancer.%20The%20transcription%20factor%20GABP%20selectively%20binds%20and%20activates%20the%20mutant%20TERT%20promoter%20in%20cancer.%20Science348,%201036%E2%80%931039%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Powter, B. et al. Human TERT promoter mutations as a prognostic biomarker in glioma. <em>J. Cancer Res. Clin. Oncol.</em><strong>147</strong>, 1007–1017 (2021).
</cite> [<a href="https://doi.org/10.1007/s00432-021-03536-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7954705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33547950/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Powter,%20B.%20et%20al.%20Human%20TERT%20promoter%20mutations%20as%20a%20prognostic%20biomarker%20in%20glioma.%20J.%20Cancer%20Res.%20Clin.%20Oncol.147,%201007%E2%80%931017%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Romero, J. M. et al. A four-chemokine signature is associated with a T-cell-inflamed phenotype in primary and metastatic pancreatic cancer. <em>Clin. Cancer Res.</em><strong>26</strong>, 1997–2010 (2020).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-19-2803" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31964786/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Romero,%20J.%20M.%20et%20al.%20A%20four-chemokine%20signature%20is%20associated%20with%20a%20T-cell-inflamed%20phenotype%20in%20primary%20and%20metastatic%20pancreatic%20cancer.%20Clin.%20Cancer%20Res.26,%201997%E2%80%932010%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Antal, C. E. et al. Cancer-associated protein kinase C mutations reveal kinase’s role as tumor suppressor. <em>Cell</em><strong>160</strong>, 489–502 (2015).</cite> [<a href="https://doi.org/10.1016/j.cell.2015.01.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4313737/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25619690/" class="usa-link">PubMed</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Vincent, A., Omura, N., Hong, S. M., Jaffe, A. &amp; Eshleman, J. Genome-wide analysis of promoter methylation associated with gene expression profile in pancreatic adenocarcinoma. <em>Clin. Cancer Res</em>. <strong>17</strong>, 4341–4354(2011).</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-10-3431" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3131423/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21610144/" class="usa-link">PubMed</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Xu, E. et al. Genome-wide methylation analysis shows similar patterns in Barrett’s esophagus and esophageal adenocarcinoma. <em>Carcinogenesis</em><strong>34</strong>, 2750–2756 (2013).
</cite> [<a href="https://doi.org/10.1093/carcin/bgt286" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3845893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23996928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20E.%20et%20al.%20Genome-wide%20methylation%20analysis%20shows%20similar%20patterns%20in%20Barrett%E2%80%99s%20esophagus%20and%20esophageal%20adenocarcinoma.%20Carcinogenesis34,%202750%E2%80%932756%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Strand, M., Earley, M. C., Crouse, G. F. &amp; Petes, T. D. Mutations in the MSH3 gene preferentially lead to deletions within tracts of simple repetitive DNA in Saccharomyces cerevisiae. <em>Proc. Natl. Acad. Sci. USA</em><strong>92</strong>, 10418–10421 (1995).
</cite> [<a href="https://doi.org/10.1073/pnas.92.22.10418" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC40808/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7479796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Strand,%20M.,%20Earley,%20M.%20C.,%20Crouse,%20G.%20F.%20&amp;%20Petes,%20T.%20D.%20Mutations%20in%20the%20MSH3%20gene%20preferentially%20lead%20to%20deletions%20within%20tracts%20of%20simple%20repetitive%20DNA%20in%20Saccharomyces%20cerevisiae.%20Proc.%20Natl.%20Acad.%20Sci.%20USA92,%2010418%E2%80%9310421%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Nik-Zainal, S. et al. Landscape of somatic mutations in 560 breast cancer whole-genome sequences. <em>Nature</em><strong>534</strong>, 47–54 (2016).
</cite> [<a href="https://doi.org/10.1038/nature17676" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4910866/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27135926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nik-Zainal,%20S.%20et%20al.%20Landscape%20of%20somatic%20mutations%20in%20560%20breast%20cancer%20whole-genome%20sequences.%20Nature534,%2047%E2%80%9354%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Wang, F. et al. Evaluation of POLE and POLD1 mutations as biomarkers for immunotherapy outcomes across multiple cancer types. <em>JAMA Oncol.</em><strong>5</strong>, 1504–1506 (2019).
</cite> [<a href="https://doi.org/10.1001/jamaoncol.2019.2963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6696731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31415061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20F.%20et%20al.%20Evaluation%20of%20POLE%20and%20POLD1%20mutations%20as%20biomarkers%20for%20immunotherapy%20outcomes%20across%20multiple%20cancer%20types.%20JAMA%20Oncol.5,%201504%E2%80%931506%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Alexandrov, L. B. et al. Signatures of mutational processes in human cancer. <em>Nature</em><strong>500</strong>, 415–421 (2013).
</cite> [<a href="https://doi.org/10.1038/nature12477" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3776390/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23945592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alexandrov,%20L.%20B.%20et%20al.%20Signatures%20of%20mutational%20processes%20in%20human%20cancer.%20Nature500,%20415%E2%80%93421%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Thai, A. A., Solomon, B. J., Sequist, L. V., Gainor, J. F. &amp; Heist, R. S. Lung cancer. <em>Lancet</em><strong>398</strong>, 535–554 (2021).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(21)00312-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34273294/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thai,%20A.%20A.,%20Solomon,%20B.%20J.,%20Sequist,%20L.%20V.,%20Gainor,%20J.%20F.%20&amp;%20Heist,%20R.%20S.%20Lung%20cancer.%20Lancet398,%20535%E2%80%93554%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Blandin Knight, S. et al. Progress and prospects of early detection in lung cancer. <em>Open Biol</em>. <strong>7</strong>, 170070 (2017).</cite> [<a href="https://doi.org/10.1098/rsob.170070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5627048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28878044/" class="usa-link">PubMed</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>de Groot, P. M., Wu, C. C., Carter, B. W. &amp; Munden, R. F. The epidemiology of lung cancer. <em>Transl. Lung Cancer Res.</em><strong>7</strong>, 220–233 (2018).
</cite> [<a href="https://doi.org/10.21037/tlcr.2018.05.06" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6037963/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30050761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Groot,%20P.%20M.,%20Wu,%20C.%20C.,%20Carter,%20B.%20W.%20&amp;%20Munden,%20R.%20F.%20The%20epidemiology%20of%20lung%20cancer.%20Transl.%20Lung%20Cancer%20Res.7,%20220%E2%80%93233%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Poulos, R. C., Sloane, M. A., Hesson, L. B. &amp; Wong, J. W. H. The search for cis-regulatory driver mutations in cancer genomes. <em>Oncotarget</em><strong>6</strong>, 32509–32525 (2015).
</cite> [<a href="https://doi.org/10.18632/oncotarget.5085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4741709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26356674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Poulos,%20R.%20C.,%20Sloane,%20M.%20A.,%20Hesson,%20L.%20B.%20&amp;%20Wong,%20J.%20W.%20H.%20The%20search%20for%20cis-regulatory%20driver%20mutations%20in%20cancer%20genomes.%20Oncotarget6,%2032509%E2%80%9332525%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Elliott, K. &amp; Larsson, E. Non-coding driver mutations in human cancer. <em>Nat. Rev. Cancer</em><strong>21</strong>, 500–509 (2021).
</cite> [<a href="https://doi.org/10.1038/s41568-021-00371-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34230647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elliott,%20K.%20&amp;%20Larsson,%20E.%20Non-coding%20driver%20mutations%20in%20human%20cancer.%20Nat.%20Rev.%20Cancer21,%20500%E2%80%93509%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Aurilio, G. et al. Androgen receptor signaling pathway in prostate cancer: from genetics to clinical applications. <em>Cells</em><strong>9</strong>, 2653 (2020).</cite> [<a href="https://doi.org/10.3390/cells9122653" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7763510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33321757/" class="usa-link">PubMed</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. <em>Nature</em><strong>489</strong>, 57–74 (2012).</cite> [<a href="https://doi.org/10.1038/nature11247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3439153/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22955616/" class="usa-link">PubMed</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Ohkia, A., Hu, Y., Wang, M., Garcia, F. U. &amp; Stearns, M. E. Evidence for prostate cancer-associated diagnostic marker-1: immunohistochemistry and in situ hybridization studies. <em>Clin. Cancer Res.</em><strong>10</strong>, 2452–2458 (2004).
</cite> [<a href="https://doi.org/10.1158/1078-0432.ccr-03-0170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15073124/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ohkia,%20A.,%20Hu,%20Y.,%20Wang,%20M.,%20Garcia,%20F.%20U.%20&amp;%20Stearns,%20M.%20E.%20Evidence%20for%20prostate%20cancer-associated%20diagnostic%20marker-1:%20immunohistochemistry%20and%20in%20situ%20hybridization%20studies.%20Clin.%20Cancer%20Res.10,%202452%E2%80%932458%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Qin, Y. et al. The tumor susceptibility gene TMEM127 is mutated in renal cell carcinomas and modulates endolysosomal function. <em>Hum. Mol. Genet.</em><strong>23</strong>, 2428–2439 (2014).
</cite> [<a href="https://doi.org/10.1093/hmg/ddt638" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3976335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24334765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qin,%20Y.%20et%20al.%20The%20tumor%20susceptibility%20gene%20TMEM127%20is%20mutated%20in%20renal%20cell%20carcinomas%20and%20modulates%20endolysosomal%20function.%20Hum.%20Mol.%20Genet.23,%202428%E2%80%932439%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Bowler, E. et al. Hypoxia leads to significant changes in alternative splicing and elevated expression of CLK splice factor kinases in PC3 prostate cancer cells. <em>BMC Cancer</em><strong>18</strong>, 355 (2018).
</cite> [<a href="https://doi.org/10.1186/s12885-018-4227-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5879922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29606096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bowler,%20E.%20et%20al.%20Hypoxia%20leads%20to%20significant%20changes%20in%20alternative%20splicing%20and%20elevated%20expression%20of%20CLK%20splice%20factor%20kinases%20in%20PC3%20prostate%20cancer%20cells.%20BMC%20Cancer18,%20355%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>El-Haibi, C. P. et al. Differential G protein subunit expression by prostate cancer cells and their interaction with CXCR5. <em>Mol. Cancer</em><strong>12</strong>, 64 (2013).
</cite> [<a href="https://doi.org/10.1186/1476-4598-12-64" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3720210/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23773523/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?El-Haibi,%20C.%20P.%20et%20al.%20Differential%20G%20protein%20subunit%20expression%20by%20prostate%20cancer%20cells%20and%20their%20interaction%20with%20CXCR5.%20Mol.%20Cancer12,%2064%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Valencia, T. et al. Role and expression of FRS2 and FRS3 in prostate cancer. <em>BMC Cancer</em><strong>11</strong>, 484 (2011).
</cite> [<a href="https://doi.org/10.1186/1471-2407-11-484" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3231952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22078327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Valencia,%20T.%20et%20al.%20Role%20and%20expression%20of%20FRS2%20and%20FRS3%20in%20prostate%20cancer.%20BMC%20Cancer11,%20484%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Yu, F. et al. Wnt/β-catenin signaling in cancers and targeted therapies. <em>Signal Transduct. Target. Ther.</em><strong>6</strong>, 307 (2021).
</cite> [<a href="https://doi.org/10.1038/s41392-021-00701-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8403677/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34456337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20F.%20et%20al.%20Wnt/%CE%B2-catenin%20signaling%20in%20cancers%20and%20targeted%20therapies.%20Signal%20Transduct.%20Target.%20Ther.6,%20307%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Kypta, R. M. &amp; Waxman, J. Wnt/β-catenin signalling in prostate cancer. <em>Nat. Rev. Urol.</em><strong>9</strong>, 418–428 (2012).
</cite> [<a href="https://doi.org/10.1038/nrurol.2012.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22710668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kypta,%20R.%20M.%20&amp;%20Waxman,%20J.%20Wnt/%CE%B2-catenin%20signalling%20in%20prostate%20cancer.%20Nat.%20Rev.%20Urol.9,%20418%E2%80%93428%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Schneider, J. A. &amp; Logan, S. K. Revisiting the role of Wnt/β-catenin signaling in prostate cancer. <em>Mol. Cell. Endocrinol.</em><strong>462</strong>, 3–8 (2018).
</cite> [<a href="https://doi.org/10.1016/j.mce.2017.02.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5550366/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28189566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schneider,%20J.%20A.%20&amp;%20Logan,%20S.%20K.%20Revisiting%20the%20role%20of%20Wnt/%CE%B2-catenin%20signaling%20in%20prostate%20cancer.%20Mol.%20Cell.%20Endocrinol.462,%203%E2%80%938%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Wang, C., Chen, Q. &amp; Xu, H. Wnt/β-catenin signal transduction pathway in prostate cancer and associated drug resistance. <em>Discov. Oncol.</em><strong>12</strong>, 40 (2021).
</cite> [<a href="https://doi.org/10.1007/s12672-021-00433-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8777554/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35201496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20C.,%20Chen,%20Q.%20&amp;%20Xu,%20H.%20Wnt/%CE%B2-catenin%20signal%20transduction%20pathway%20in%20prostate%20cancer%20and%20associated%20drug%20resistance.%20Discov.%20Oncol.12,%2040%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Grant, C. E., Bailey, T. L. &amp; Noble, W. S. FIMO: scanning for occurrences of a given motif. <em>Bioinformatics</em><strong>27</strong>, 1017–1018 (2011).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3065696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21330290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grant,%20C.%20E.,%20Bailey,%20T.%20L.%20&amp;%20Noble,%20W.%20S.%20FIMO:%20scanning%20for%20occurrences%20of%20a%20given%20motif.%20Bioinformatics27,%201017%E2%80%931018%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Castro-Mondragon, J. A. et al. JASPAR 2022: the 9th release of the open-access database of transcription factor binding profiles. <em>Nucleic Acids Res.</em><strong>50</strong>, D165–D173 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkab1113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34850907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Castro-Mondragon,%20J.%20A.%20et%20al.%20JASPAR%202022:%20the%209th%20release%20of%20the%20open-access%20database%20of%20transcription%20factor%20binding%20profiles.%20Nucleic%20Acids%20Res.50,%20D165%E2%80%93D173%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Vatapalli, R. et al. Histone methyltransferase DOT1L coordinates AR and MYC stability in prostate cancer. <em>Nat. Commun.</em><strong>11</strong>, 4153 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-18013-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7438336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32814769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vatapalli,%20R.%20et%20al.%20Histone%20methyltransferase%20DOT1L%20coordinates%20AR%20and%20MYC%20stability%20in%20prostate%20cancer.%20Nat.%20Commun.11,%204153%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Dhingra, P. et al. Identification of novel prostate cancer drivers using RegNetDriver: a framework for integration of genetic and epigenetic alterations with tissue-specific regulatory network. <em>Genome Biol</em>. <strong>18</strong>, (2017).</cite> [<a href="https://doi.org/10.1186/s13059-017-1266-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5530464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28750683/" class="usa-link">PubMed</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Zhang, Z. et al. HERC3 regulates epithelial-mesenchymal transition by directly ubiquitination degradation EIF5A2 and inhibits metastasis of colorectal cancer. <em>Cell Death Dis.</em><strong>13</strong>, 1–12 (2022).</cite> [<a href="https://doi.org/10.1038/s41419-022-04511-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8782983/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35064108/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Z.%20et%20al.%20HERC3%20regulates%20epithelial-mesenchymal%20transition%20by%20directly%20ubiquitination%20degradation%20EIF5A2%20and%20inhibits%20metastasis%20of%20colorectal%20cancer.%20Cell%20Death%20Dis.13,%201%E2%80%9312%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Schult, T. A. et al. Screening human lung cancer with predictive models of serum magnetic resonance spectroscopy metabolomics. <em>Proc. Natl. Acad. Sci. USA</em>. <strong>118</strong>, e2110633118 (2021).</cite> [<a href="https://doi.org/10.1073/pnas.2110633118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8713787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34903652/" class="usa-link">PubMed</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Moyer, V. A. &amp; Preventive Services Task Force, U. S. Screening for lung cancer: U.S. Preventive Services Task Force recommendation statement. <em>Ann. Intern. Med.</em><strong>160</strong>, 330–338 (2014).
</cite> [<a href="https://doi.org/10.7326/M13-2771" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24378917/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moyer,%20V.%20A.%20&amp;%20Preventive%20Services%20Task%20Force,%20U.%20S.%20Screening%20for%20lung%20cancer:%20U.S.%20Preventive%20Services%20Task%20Force%20recommendation%20statement.%20Ann.%20Intern.%20Med.160,%20330%E2%80%93338%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Smith, R. A. et al. Cancer screening in the United States, 2019: a review of current American Cancer Society guidelines and current issues in cancer screening. <em>CA Cancer J. Clin.</em><strong>69</strong>, 184–210 (2019).
</cite> [<a href="https://doi.org/10.3322/caac.21557" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30875085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20R.%20A.%20et%20al.%20Cancer%20screening%20in%20the%20United%20States,%202019:%20a%20review%20of%20current%20American%20Cancer%20Society%20guidelines%20and%20current%20issues%20in%20cancer%20screening.%20CA%20Cancer%20J.%20Clin.69,%20184%E2%80%93210%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Wood, D. E. et al. Lung Cancer Screening, version 3.2018, NCCN Clinical Practice Guidelines in Oncology. <em>J. Natl. Compr. Cancer Netw.</em><strong>16</strong>, 412–441 (2018).</cite> [<a href="https://doi.org/10.6004/jnccn.2018.0020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6476336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29632061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wood,%20D.%20E.%20et%20al.%20Lung%20Cancer%20Screening,%20version%203.2018,%20NCCN%20Clinical%20Practice%20Guidelines%20in%20Oncology.%20J.%20Natl.%20Compr.%20Cancer%20Netw.16,%20412%E2%80%93441%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Rendle, K. A. et al. Rates of downstream procedures and complications associated with lung cancer screening in routine clinical practice: A retrospective cohort study. <em>Ann. Intern. Med.</em><strong>177</strong>, 18–28 (2024).
</cite> [<a href="https://doi.org/10.7326/M23-0653" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11111256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38163370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rendle,%20K.%20A.%20et%20al.%20Rates%20of%20downstream%20procedures%20and%20complications%20associated%20with%20lung%20cancer%20screening%20in%20routine%20clinical%20practice:%20A%20retrospective%20cohort%20study.%20Ann.%20Intern.%20Med.177,%2018%E2%80%9328%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Mazzone, P. J. et al. Clinical validation of a cell-free DNA fragmentome assay for augmentation of lung cancer early detection. <em>Cancer Discov</em>. 10.1158/2159-8290.CD-24-0519 (2024).</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-24-0519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11528203/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38829053/" class="usa-link">PubMed</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Lin, W.-H. et al. Circulating tumor DNA methylation marker MYO1-G for diagnosis and monitoring of colorectal cancer. <em>Clin. Epigenetics</em><strong>13</strong>, 232 (2021).
</cite> [<a href="https://doi.org/10.1186/s13148-021-01216-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8713401/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34961566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20W.-H.%20et%20al.%20Circulating%20tumor%20DNA%20methylation%20marker%20MYO1-G%20for%20diagnosis%20and%20monitoring%20of%20colorectal%20cancer.%20Clin.%20Epigenetics13,%20232%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Bae, M. et al. Integrative modeling of tumor genomes and epigenomes for enhanced cancer diagnosis by cell-free DNA. <em>Nat. Commun.</em><strong>14</strong>, 2017 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-37768-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10085982/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37037826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bae,%20M.%20et%20al.%20Integrative%20modeling%20of%20tumor%20genomes%20and%20epigenomes%20for%20enhanced%20cancer%20diagnosis%20by%20cell-free%20DNA.%20Nat.%20Commun.14,%202017%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Kaminski, M. M., Abudayyeh, O. O., Gootenberg, J. S., Zhang, F. &amp; Collins, J. J. CRISPR-based diagnostics. <em>Nat. Biomed. Eng.</em><strong>5</strong>, 643–656 (2021).
</cite> [<a href="https://doi.org/10.1038/s41551-021-00760-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34272525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaminski,%20M.%20M.,%20Abudayyeh,%20O.%20O.,%20Gootenberg,%20J.%20S.,%20Zhang,%20F.%20&amp;%20Collins,%20J.%20J.%20CRISPR-based%20diagnostics.%20Nat.%20Biomed.%20Eng.5,%20643%E2%80%93656%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Ackerman, C. M. et al. Massively multiplexed nucleic acid detection with Cas13. <em>Nature</em><strong>582</strong>, 277–282 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2279-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7332423/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32349121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ackerman,%20C.%20M.%20et%20al.%20Massively%20multiplexed%20nucleic%20acid%20detection%20with%20Cas13.%20Nature582,%20277%E2%80%93282%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Augustus, E. et al. The art of obtaining a high yield of cell-free DNA from urine. <em>PLoS ONE</em><strong>15</strong>, e0231058 (2020).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0231058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7135229/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32251424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Augustus,%20E.%20et%20al.%20The%20art%20of%20obtaining%20a%20high%20yield%20of%20cell-free%20DNA%20from%20urine.%20PLoS%20ONE15,%20e0231058%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Ding, S. et al. Saliva-derived cfDNA is applicable for EGFR mutation detection but not for quantitation analysis in non-small cell lung cancer. <em>Thorac. Cancer</em><strong>10</strong>, 1973–1983 (2019).
</cite> [<a href="https://doi.org/10.1111/1759-7714.13178" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6775000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31441578/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ding,%20S.%20et%20al.%20Saliva-derived%20cfDNA%20is%20applicable%20for%20EGFR%20mutation%20detection%20but%20not%20for%20quantitation%20analysis%20in%20non-small%20cell%20lung%20cancer.%20Thorac.%20Cancer10,%201973%E2%80%931983%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Logsdon, G. A., Vollger, M. R. &amp; Eichler, E. E. Long-read human genome sequencing and its applications. <em>Nat. Rev. Genet.</em><strong>21</strong>, 597–614 (2020).
</cite> [<a href="https://doi.org/10.1038/s41576-020-0236-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7877196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32504078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Logsdon,%20G.%20A.,%20Vollger,%20M.%20R.%20&amp;%20Eichler,%20E.%20E.%20Long-read%20human%20genome%20sequencing%20and%20its%20applications.%20Nat.%20Rev.%20Genet.21,%20597%E2%80%93614%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Cheng, C., Fei, Z. &amp; Xiao, P. Methods to improve the accuracy of next-generation sequencing. <em>Front. Bioeng. Biotechnol.</em><strong>11</strong>, 982111 (2023).
</cite> [<a href="https://doi.org/10.3389/fbioe.2023.982111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9895957/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36741756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20C.,%20Fei,%20Z.%20&amp;%20Xiao,%20P.%20Methods%20to%20improve%20the%20accuracy%20of%20next-generation%20sequencing.%20Front.%20Bioeng.%20Biotechnol.11,%20982111%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Lennon, A. M. et al. Feasibility of blood testing combined with PET-CT to screen for cancer and guide intervention. <em>Science</em><strong>369</strong>, eabb9601 (2020).</cite> [<a href="https://doi.org/10.1126/science.abb9601" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7509949/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32345712/" class="usa-link">PubMed</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Alileche, A. &amp; Hampikian, G. The effect of Nullomer-derived peptides 9R, 9S1R and 124R on the NCI-60 panel and normal cell lines. <em>BMC Cancer</em><strong>17</strong>, 533 (2017).
</cite> [<a href="https://doi.org/10.1186/s12885-017-3514-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5551024/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28793867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alileche,%20A.%20&amp;%20Hampikian,%20G.%20The%20effect%20of%20Nullomer-derived%20peptides%209R,%209S1R%20and%20124R%20on%20the%20NCI-60%20panel%20and%20normal%20cell%20lines.%20BMC%20Cancer17,%20533%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Koulouras, G. &amp; Frith, M. C. Significant non-existence of sequences in genomes and proteomes. <em>Nucleic Acids Res</em>. 10.1093/nar/gkab139 (2021).</cite> [<a href="https://doi.org/10.1093/nar/gkab139" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8034619/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33693858/" class="usa-link">PubMed</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Vita, R. et al. The Immune Epitope Database (IEDB): 2018 update. <em>Nucleic Acids Res.</em><strong>47</strong>, D339–D343 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gky1006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6324067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30357391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vita,%20R.%20et%20al.%20The%20Immune%20Epitope%20Database%20(IEDB):%202018%20update.%20Nucleic%20Acids%20Res.47,%20D339%E2%80%93D343%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Edlind, M. P. &amp; Hsieh, A. C. PI3K-AKT-mTOR signaling in prostate cancer progression and androgen deprivation therapy resistance. <em>Asian J. Androl.</em><strong>16</strong>, 378–386 (2014).
</cite> [<a href="https://doi.org/10.4103/1008-682X.122876" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4023363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24759575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Edlind,%20M.%20P.%20&amp;%20Hsieh,%20A.%20C.%20PI3K-AKT-mTOR%20signaling%20in%20prostate%20cancer%20progression%20and%20androgen%20deprivation%20therapy%20resistance.%20Asian%20J.%20Androl.16,%20378%E2%80%93386%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Chiang, A. C. &amp; Massagué, J. Molecular basis of metastasis. <em>N. Engl. J. Med.</em><strong>359</strong>, 2814–2823 (2008).
</cite> [<a href="https://doi.org/10.1056/NEJMra0805239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4189180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19109576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chiang,%20A.%20C.%20&amp;%20Massagu%C3%A9,%20J.%20Molecular%20basis%20of%20metastasis.%20N.%20Engl.%20J.%20Med.359,%202814%E2%80%932823%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Nguyen, D. X. &amp; Massagué, J. Genetic determinants of cancer metastasis. <em>Nat. Rev. Genet.</em><strong>8</strong>, 341–352 (2007).
</cite> [<a href="https://doi.org/10.1038/nrg2101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17440531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen,%20D.%20X.%20&amp;%20Massagu%C3%A9,%20J.%20Genetic%20determinants%20of%20cancer%20metastasis.%20Nat.%20Rev.%20Genet.8,%20341%E2%80%93352%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Easterlin, R. et al. Nullomerator. <em>Zenodo</em>. 10.5281/zenodo.10910173 (2024).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371106/bin/43856_2025_1067_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (1.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371106/bin/43856_2025_1067_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">43856_2025_1067_MOESM2_ESM.pdf</a><sup> (32.9KB, pdf) </sup><p>Description of Additional Supplementary files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371106/bin/43856_2025_1067_MOESM3_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplemental Data 1-6</a><sup> (672KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371106/bin/43856_2025_1067_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting summary</a><sup> (2.2MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The source data for Fig. <a href="#Fig1" class="usa-link">1</a> is found in Data <a href="#MOESM3" class="usa-link">S1</a>, for Fig. <a href="#Fig3" class="usa-link">3</a> in Data <a href="#MOESM3" class="usa-link">S3</a>, and for Fig. <a href="#Fig4" class="usa-link">4</a> in Data <a href="#MOESM3" class="usa-link">S5</a>. MPRA sequencing data was deposited in the NCBI short read archive (SRA) with accession code PRJNA917083. ICGC samples are publicly available and can be accessed from <a href="https://docs.icgc-argo.org/docs/data-access/icgc-25k-data" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://docs.icgc-argo.org/docs/data-access/icgc-25k-data</a>. Due to patient data confidentiality, WGS sequencing data is only available upon request. For WGS sequencing, contact Nadav Ahituv.</p>
<p>We generated an easy-to-use software package that enables performing nullomer cancer analyses from sequence-based datasets<sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup>. The package is composed of six functions: (1) EnumerateNullomers, which extracts all nullomers of specified kmer lengths in a FASTA sample; (2) ExtractMutationNullomers, which finds all mutations that cause the resurfacing of a list of nullomers; (3) IdentifyRecurrentNullomers, which identifies nullomers that recur in a dataset through mutagenesis; (4) FindAlmostNullomers, which identifies the positions that can create a list of nullomers genome-wide for every possible substitution and single base-pair insertion and deletion; (5) FindNullomerVariants, which removes nullomers that are likely to result from common variants in a user specified variant VCF file; (6) FindDNANullomersFromReads, which performs the identification of nullomers in raw read samples. The package can be found at 10.5281/zenodo.1091017397 and the documentation on Read the Docs (<a href="https://nullomerator.readthedocs.io/en/latest/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://nullomerator.readthedocs.io/en/latest/#</a>) provides in-depth details on how to use the software.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Communications Medicine are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s43856-025-01067-3"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/43856_2025_Article_1067.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371106/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371106/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371106%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371106/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371106/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371106/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841759/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371106/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841759/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371106/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371106/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="8nSTf6IlahMxkoyma2ynLoxnYxFqDSWbwC0Lain1xjl0B6HoYv4qitLDaz4UAojI">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
