
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Using extension‐based mRNA display to design antibody‐like proteinogenic peptides for human PD‐L1 - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15251E048AEF26F3051E040014CDBD31.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="prosci">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376068/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Protein Science : A Publication of the Protein Society">
<meta name="citation_title" content="Using extension‐based mRNA display to design antibody‐like proteinogenic peptides for human PD‐L1">
<meta name="citation_author" content="Justin N Ong">
<meta name="citation_author_institution" content="Deparment of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas, USA">
<meta name="citation_author_institution" content="Mork Family Department of Chemical Engineering and Materials Science, University of Southern California, Los Angeles, California, USA">
<meta name="citation_author" content="Brian J Grindel">
<meta name="citation_author_institution" content="Deparment of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas, USA">
<meta name="citation_author" content="Scott A Rankin">
<meta name="citation_author_institution" content="Department of Chemistry, University of Southern California, Los Angeles, California, USA">
<meta name="citation_author" content="Sarah H Naylon">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, Florida Atlantic University, Boca Raton, Florida, USA">
<meta name="citation_author" content="Anupallavi Srinivasamani">
<meta name="citation_author_institution" content="Department of Immunology, University of Texas MD Anderson Cancer Center, Houston, Texas, USA">
<meta name="citation_author" content="Guillaume J Trusz">
<meta name="citation_author_institution" content="Department of Immunology, University of Texas MD Anderson Cancer Center, Houston, Texas, USA">
<meta name="citation_author" content="Xiaowen Liang">
<meta name="citation_author_institution" content="Department of Experimental Therapeutics, University of Texas MD Anderson Cancer Center, Houston, Texas, USA">
<meta name="citation_author" content="Md Nasir Uddin">
<meta name="citation_author_institution" content="Deparment of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas, USA">
<meta name="citation_author" content="Lauren Fuller">
<meta name="citation_author_institution" content="Deparment of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas, USA">
<meta name="citation_author" content="Michael Curran">
<meta name="citation_author_institution" content="Department of Immunology, University of Texas MD Anderson Cancer Center, Houston, Texas, USA">
<meta name="citation_author" content="Stephane P Roche">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, Florida Atlantic University, Boca Raton, Florida, USA">
<meta name="citation_author_institution" content="Center for Molecular Biology and Biotechnology, Florida Atlantic University, Jupiter, Florida, USA">
<meta name="citation_author" content="Terry T Takahashi">
<meta name="citation_author_institution" content="Department of Chemistry, University of Southern California, Los Angeles, California, USA">
<meta name="citation_author" content="Richard W Roberts">
<meta name="citation_author_institution" content="Mork Family Department of Chemical Engineering and Materials Science, University of Southern California, Los Angeles, California, USA">
<meta name="citation_author_institution" content="Department of Chemistry, University of Southern California, Los Angeles, California, USA">
<meta name="citation_author_institution" content="Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA">
<meta name="citation_author" content="Steven W Millward">
<meta name="citation_author_institution" content="Deparment of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas, USA">
<meta name="citation_publication_date" content="2025 Aug 25">
<meta name="citation_volume" content="34">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70268">
<meta name="citation_doi" content="10.1002/pro.70268">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376068/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376068/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376068/pdf/PRO-34-e70268.pdf">
<meta name="description" content="Many peptide drugs rely on nonproteinogenic amino acids and chemical modifications for improved activity and proteolytic stability. However, these features also make drug production expensive and challenging to scale. Here, we engineered small, ...">
<meta name="og:title" content="Using extension‐based mRNA display to design antibody‐like proteinogenic peptides for human PD‐L1">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Many peptide drugs rely on nonproteinogenic amino acids and chemical modifications for improved activity and proteolytic stability. However, these features also make drug production expensive and challenging to scale. Here, we engineered small, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376068/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12376068">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/pro.70268"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/PRO-34-e70268.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12376068%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12376068/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12376068/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376068/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-prosci.gif" alt="Protein Science : A Publication of the Protein Society logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Protein Science : A Publication of the Protein Society" title="Link to Protein Science : A Publication of the Protein Society" shape="default" href="http://www.proteinscience.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Protein Sci</button></div>. 2025 Aug 25;34(9):e70268. doi: <a href="https://doi.org/10.1002/pro.70268" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/pro.70268</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Protein%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Protein%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Using extension‐based mRNA display to design antibody‐like proteinogenic peptides for human PD‐L1
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ong%20JN%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Justin N Ong</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Justin N Ong</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Deparment of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Mork Family Department of Chemical Engineering and Materials Science, University of Southern California, Los Angeles, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ong%20JN%22%5BAuthor%5D" class="usa-link"><span class="name western">Justin N Ong</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grindel%20BJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Brian J Grindel</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Brian J Grindel</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Deparment of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grindel%20BJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Brian J Grindel</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rankin%20SA%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Scott A Rankin</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Scott A Rankin</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Chemistry, University of Southern California, Los Angeles, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rankin%20SA%22%5BAuthor%5D" class="usa-link"><span class="name western">Scott A Rankin</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Naylon%20SH%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Sarah H Naylon</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Sarah H Naylon</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Chemistry and Biochemistry, Florida Atlantic University, Boca Raton, Florida, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Naylon%20SH%22%5BAuthor%5D" class="usa-link"><span class="name western">Sarah H Naylon</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Srinivasamani%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Anupallavi Srinivasamani</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Anupallavi Srinivasamani</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Department of Immunology, University of Texas MD Anderson Cancer Center, Houston, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Srinivasamani%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anupallavi Srinivasamani</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Trusz%20GJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Guillaume J Trusz</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Guillaume J Trusz</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Department of Immunology, University of Texas MD Anderson Cancer Center, Houston, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Trusz%20GJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Guillaume J Trusz</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Xiaowen Liang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Xiaowen Liang</span></h3>
<div class="p">
<sup>
<sup>6</sup>
</sup>Department of Experimental Therapeutics, University of Texas MD Anderson Cancer Center, Houston, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaowen Liang</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Uddin%20MN%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Md Nasir Uddin</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Md Nasir Uddin</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Deparment of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Uddin%20MN%22%5BAuthor%5D" class="usa-link"><span class="name western">Md Nasir Uddin</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fuller%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Lauren Fuller</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Lauren Fuller</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Deparment of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fuller%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lauren Fuller</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Curran%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Michael Curran</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Michael Curran</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Department of Immunology, University of Texas MD Anderson Cancer Center, Houston, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Curran%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael Curran</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roche%20SP%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Stephane P Roche</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Stephane P Roche</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Chemistry and Biochemistry, Florida Atlantic University, Boca Raton, Florida, USA</div>
<div class="p">
<sup>
<sup>7</sup>
</sup>Center for Molecular Biology and Biotechnology, Florida Atlantic University, Jupiter, Florida, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roche%20SP%22%5BAuthor%5D" class="usa-link"><span class="name western">Stephane P Roche</span></a>
</div>
</div>
<sup>4,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Takahashi%20TT%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Terry T Takahashi</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Terry T Takahashi</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Chemistry, University of Southern California, Los Angeles, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Takahashi%20TT%22%5BAuthor%5D" class="usa-link"><span class="name western">Terry T Takahashi</span></a>
</div>
</div>
<sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roberts%20RW%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Richard W Roberts</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Richard W Roberts</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Mork Family Department of Chemical Engineering and Materials Science, University of Southern California, Los Angeles, California, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Chemistry, University of Southern California, Los Angeles, California, USA</div>
<div class="p">
<sup>
<sup>8</sup>
</sup>Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roberts%20RW%22%5BAuthor%5D" class="usa-link"><span class="name western">Richard W Roberts</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>8,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Millward%20SW%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Steven W Millward</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Steven W Millward</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Deparment of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Millward%20SW%22%5BAuthor%5D" class="usa-link"><span class="name western">Steven W Millward</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="pro70268-aff-0001">
<sup>
<sup>1</sup>
</sup>Deparment of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, Texas, USA</div>
<div id="pro70268-aff-0002">
<sup>
<sup>2</sup>
</sup>Mork Family Department of Chemical Engineering and Materials Science, University of Southern California, Los Angeles, California, USA</div>
<div id="pro70268-aff-0003">
<sup>
<sup>3</sup>
</sup>Department of Chemistry, University of Southern California, Los Angeles, California, USA</div>
<div id="pro70268-aff-0004">
<sup>
<sup>4</sup>
</sup>Department of Chemistry and Biochemistry, Florida Atlantic University, Boca Raton, Florida, USA</div>
<div id="pro70268-aff-0005">
<sup>
<sup>5</sup>
</sup>Department of Immunology, University of Texas MD Anderson Cancer Center, Houston, Texas, USA</div>
<div id="pro70268-aff-0006">
<sup>
<sup>6</sup>
</sup>Department of Experimental Therapeutics, University of Texas MD Anderson Cancer Center, Houston, Texas, USA</div>
<div id="pro70268-aff-0007">
<sup>
<sup>7</sup>
</sup>Center for Molecular Biology and Biotechnology, Florida Atlantic University, Jupiter, Florida, USA</div>
<div id="pro70268-aff-0008">
<sup>
<sup>8</sup>
</sup>Norris Comprehensive Cancer Center, University of Southern California, Los Angeles, California, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, 
Terry T. Takahashi, Department of Chemistry, University of Southern California, Los Angeles, CA 90089, USA. 
Email: <span>tttakaha@usc.edu</span>
, 
Richard W. Roberts, Mork Family Department of Chemical Engineering and Materials Science, University of Southern California, Los Angeles, CA 90089, USA. 
Email: <span>richrob@usc.edu</span>
, 
Steven W. Millward, Department of Cancer Systems Imaging, University of Texas MD Anderson Cancer Center, Houston, TX 77054, USA. 
Email: <span>smillward@mdanderson.org</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jul 18; Received 2025 Feb 20; Accepted 2025 Jul 31; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Protein Science</em> published by Wiley Periodicals LLC on behalf of The Protein Society.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12376068</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Many peptide drugs rely on nonproteinogenic amino acids and chemical modifications for improved activity and proteolytic stability. However, these features also make drug production expensive and challenging to scale. Here, we engineered small, linear, proteinogenic peptides that bind human programmed death‐ligand 1 (hPD‐L1) with high affinity and stability using mRNA display affinity maturation. The resulting peptides, SPAM2 and SPAM3, have antibody‐like affinities for hPD‐L1 (dissociation constants between ~250 and 300 pM) and are selective for hPD‐L1. Both SPAM2 and SPAM3 compete with hPD‐L1 ligands known to interact with the programmed cell death protein 1 site and are stable in human serum. SPAM3 bound human glioma D423 cells with high affinity in flow cytometry experiments comparable to that of a clinical therapeutic antibody. These results support the use of affinity maturation selections to dramatically enhance the biophysical properties of linear, proteinogenic peptides for translational applications.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> affinity maturation, immune checkpoint, mRNA display, PD‐L1, peptide</p></section></section><section id="pro70268-sec-0001"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Cancer cells commonly exploit immunosuppressive pathways to evade immune destruction. For example, the interaction between programmed cell death protein 1 (PD‐1) and human programmed death‐ligand 1 (hPD‐L1) mediates T‐cell growth, function, and survival (Bardhan et al., <a href="#pro70268-bib-0001" class="usa-link" aria-describedby="pro70268-bib-0001">2016</a>; Freeman et al., <a href="#pro70268-bib-0010" class="usa-link" aria-describedby="pro70268-bib-0010"><sup>2000</sup></a>). Many monoclonal antibodies target this interaction for therapeutic and diagnostic applications (Doki et al., <a href="#pro70268-bib-0008" class="usa-link" aria-describedby="pro70268-bib-0008">2022</a>; Larson et al., <a href="#pro70268-bib-0022" class="usa-link" aria-describedby="pro70268-bib-0022"><sup>2022</sup></a>; Melisi et al., <a href="#pro70268-bib-0033" class="usa-link" aria-describedby="pro70268-bib-0033"><sup>2021</sup></a>; Saxena et al., <a href="#pro70268-bib-0042" class="usa-link" aria-describedby="pro70268-bib-0042"><sup>2021</sup></a>; Seto et al., <a href="#pro70268-bib-0045" class="usa-link" aria-describedby="pro70268-bib-0045"><sup>2022</sup></a>). While some patients have achieved complete remission with antibody‐based immune checkpoint therapy; many patients receive no benefit and some experience severe immune‐related adverse events (Li et al., <a href="#pro70268-bib-0027" class="usa-link" aria-describedby="pro70268-bib-0027">2025</a>).</p>
<p>Although anti‐PD‐L1 antibodies remain clinical workhorses in immune checkpoint therapy, they are challenging to adapt for imaging and radiotherapeutic applications. In part, this is due to their slow systemic clearance rates, which are partially driven by their high molecular weights (Rhoden &amp; Wittrup, <a href="#pro70268-bib-0040" class="usa-link" aria-describedby="pro70268-bib-0040">2012</a>; Thurber &amp; Wittrup, <a href="#pro70268-bib-0048" class="usa-link" aria-describedby="pro70268-bib-0048"><sup>2012</sup></a>). For imaging, this results in long delays between radiotracer injection and image acquisition. In the case of targeted systemic radiotherapy, it results in sustained exposure to ionizing radiation with concomitant damage to healthy tissues.</p>
<p>As lower molecular weight alternatives to antibodies, peptides have been developed to target hPD‐L1. WL12, a macrocyclic peptide that potently inhibits the binding of therapeutic antibodies to hPD‐L1 (IC<sub>50</sub> = 22 nM), is currently undergoing development as a clinical Positron Emission Tomography (PET) radiotracer (Zhou, Jiang, et al., <a href="#pro70268-bib-0052" class="usa-link" aria-describedby="pro70268-bib-0052">2022</a>). Recently, Bristol Myers Squibb reported the discovery of BMS‐986238, which is undergoing investigation as an orally administered therapeutic (Scola, <a href="#pro70268-bib-0044" class="usa-link" aria-describedby="pro70268-bib-0044">2025</a>). However, both peptides incorporate nonproteinogenic amino acids and chemical modifications. Although the peptides described above are highly potent and metabolically stable, nonproteinogenic amino acids and chemical functionalities increase the monetary costs and synthetic challenges associated with scaling up production for commercial use. Moreover, these compounds cannot be readily incorporated into other biological constructs, making them unsuitable as targeting vectors for viruses and vaccines or as biological tags for reagent localization.</p>
<p>A fully proteinogenic peptide with high metabolic stability and antibody‐like hPD‐L1 affinity would provide an outstanding targeting vector for next‐generation theragnostic development. This peptide could also be genetically encoded to impart hPD‐L1 affinity into current and next‐generation biologics. One such proteinogenic peptide, TPP‐1, was developed using bacterial surface display and bound hPD‐L1 with <em>K</em>
<sub>D</sub> = 95 nM (Li et al., <a href="#pro70268-bib-0026" class="usa-link" aria-describedby="pro70268-bib-0026">2018</a>). However, TPP‐1 suffered from poor serum stability and required PEGylation for pre‐clinical PET studies (Hu et al., <a href="#pro70268-bib-0019" class="usa-link" aria-describedby="pro70268-bib-0019">2019</a>). To our knowledge, there are no proteinogenic PD‐L1‐targeted peptides with antibody‐like affinity and sufficient serum stability for in vivo applications.</p>
<p>Previously, we developed SPAM (MPIFLDHILNKFLILHYA), a short, linear, proteinogenic peptide with mid‐nanomolar affinity for hPD‐L1 (Kamalinia et al., <a href="#pro70268-bib-0020" class="usa-link" aria-describedby="pro70268-bib-0020">2020</a>). Although this peptide showed acceptable serum stability (ex vivo), its affinity (<em>K</em>
<sub>D</sub> = 67 ± 21 nM) is likely insufficient for further pre‐clinical development. In this work, we sought to improve the SPAM scaffold and identify peptides that bind hPD‐L1 with sub‐nanomolar affinity. Previous studies have validated mRNA display as a platform for affinity maturation of proteinogenic peptides and proteins (Cetin et al., <a href="#pro70268-bib-0004" class="usa-link" aria-describedby="pro70268-bib-0004">2017</a>; Kamalinia et al., <a href="#pro70268-bib-0020" class="usa-link" aria-describedby="pro70268-bib-0020"><sup>2020</sup></a>; Nichols et al., <a href="#pro70268-bib-0035" class="usa-link" aria-describedby="pro70268-bib-0035"><sup>2018</sup></a>). Our results indicate that modest extensions of the core SPAM motif can impart considerable affinity enhancements without compromising target selectivity. Indeed, the resulting sequences demonstrate mid‐picomolar target affinity, acceptable serum stability, exquisite selectivity for hPD‐L1, and robust activity in living cells.</p></section><section id="pro70268-sec-0002"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="pro70268-sec-0003"><h3 class="pmc_sec_title">2.1. Recombinant protein expression and purification</h3>
<p>Recombinant protein produced in mammalian cells (extracellular domains of hPD‐L1, mouse PD‐L1 [mPD‐L1], and human PD‐L2 [hPD‐L2]) for mRNA display and binding was purified as previously described (Grindel et al., <a href="#pro70268-bib-0016" class="usa-link" aria-describedby="pro70268-bib-0016">2020</a>, <a href="#pro70268-bib-0017" class="usa-link" aria-describedby="pro70268-bib-0017">2022</a>; Kamalinia et al., <a href="#pro70268-bib-0020" class="usa-link" aria-describedby="pro70268-bib-0020"><sup>2020</sup></a>). Briefly, constructs (GeneART/Thermo Fisher Scientific) were designed with a C‐terminal poly‐histidine tag, a BirA biotinylation site, and a thrombin cleavage site in a pcDNA3.4 backbone. A CMV‐driven promoter and <em>Gaussia princeps</em> signal peptide were included to induce high expression and extracellular secretion of the recombinant proteins. Constructs were transiently transfected into HEK293F cells using an ExpiFectamine Transfection kit (Thermo Fisher Scientific A14524) according to the manufacturer's directions. Conditioned media was dialyzed, and proteins were purified by Ni‐nitriloacetic acid chromatography. Following buffer exchange, the protein was biotinylated by recombinant biotin ligase (BirA) overnight at 4°C and exchanged into storage buffer: 50 mM 4‐(2‐hydroxyethyl) piperazine‐1‐ethanesulfonic acid ‐NaOH (HEPES‐NaOH), pH 8.0, 250 mM NaCl, 5% (v/v) glycerol, and 0.01% (w/v) sodium azide.</p></section><section id="pro70268-sec-0004"><h3 class="pmc_sec_title">2.2. 
mRNA display</h3>
<p>A DNA library encoding 22‐mer peptides was constructed for mRNA display based on mutational analysis data for SPAM (Kamalinia et al., <a href="#pro70268-bib-0020" class="usa-link" aria-describedby="pro70268-bib-0020">2020</a>). The open reading frame had the design 5′‐ATG NNS NNS NNS NNS 213 313 143 342 113 112 443 422 141 342 313 413 231 NNS NNS NNS NNS GGC AGC GGA TCC AGT GGA GGC‐3′, where <em>N</em> = A, C, T, or G, S = C or G, 1 = 65.9% A, 9.8% G, 14.6% C, 9.8% T, 2 = 17.6% A, 52.9% G, 17.6% C, 11.8% T, 3 = 14.6% A, 9.8% G, 65.9% C, 9.8% T, and 4 = 17.6% A, 11.8% G, 17.6% C, 52.9% T. This enabled the translation of a library of peptides with the format MXXXX<span class="text-underline">DHILNKFWILHYA</span>XXXXGSSSSGS, where <em>X</em> represents a random amino acid, GSSSSGS is a constant C‐terminal spacer, and the underlined positions were doped with 70% wildtype nucleotide probability and 10% probability for each of the other three nucleotides, when adjusted for the different nucleotide coupling rates during chemical synthesis. Peptide‐mRNA fusions were prepared as previously described (Kamalinia et al., <a href="#pro70268-bib-0020" class="usa-link" aria-describedby="pro70268-bib-0020">2020</a>). Additional details regarding library construction and peptide‐mRNA fusion preparation are included in the <a href="#pro70268-supitem-0001" class="usa-link">Supporting Information</a>.</p>
<p>Biotinylated hPD‐L1 was immobilized on neutravidin agarose beads (Thermo Fisher 29201) in selection buffer (20 mM HEPES‐KOH, pH 7.5, 150 mM NaCl, 0.2% (v/v) Tween‐20, 1 mg mL<sup>−1</sup> BSA, 50 μg mL<sup>−1</sup> tRNA) at 4°C for 1 h. The concentration of immobilized hPD‐L1 in 1 mL selection buffer varied by selection round: 300 nM for Round 1, 100 nM for Round 2, 50 nM for Rounds 3 and 4, and 25 nM for Rounds 5, 6, and 7. The beads were then washed with and incubated in blocking buffer (selection buffer + 0.02 mM biotin) at 4°C for 2 h to saturate any remaining unbound neutravidin sites with biotin.</p>
<p>After reverse transcription, the cDNA–peptide fusion library was added to the immobilized hPD‐L1 and incubated for 1 h in 1 mL blocking buffer. This binding occurred at 4°C for Rounds 1, 2, 6, and 7 and at 22°C for Rounds 3, 4, and 5. The beads were washed seven times with blocking buffer to remove nonfunctional sequences. Rounds 6 and 7 included an additional off‐rate‐based selection step after the initial washing step. The off‐rate selection was performed by adding non‐biotinylated hPD‐L1 to the washed beads at a final concentration of 2500 nM in 1 mL of selection buffer followed by incubation at 4°C. After 15 min for Round 6 and 15 h for Round 7, the beads were separated from solution using a Spin‐X centrifuge filter tube (Corning CLS8161). For all rounds, the washed beads were added to a PCR reaction and amplified to regenerate the DNA library for the next selection round.</p></section><section id="pro70268-sec-0005"><h3 class="pmc_sec_title">2.3. Radioligand binding assays</h3>
<p>Radiolabeled peptide‐mRNA fusions were prepared as above but with the addition of 2 μL <sup>35</sup>S‐methionine (~1–2 μCi, Perkin Elmer) per 25 μL translation. After translation, fusion formation, and dT purification, 1% of the fusion volume was measured in a liquid scintillation counter (Beckman Coulter LS6500) to estimate the total counts. hPD‐L1 was immobilized onto neutravidin agarose beads as above with the amount required for each assay. 80,000–100,000 counts of radiolabeled fusions were added to the hPD‐L1 immobilized beads and incubated for 1 h at the temperature required for each assay. The supernatant was removed, and the beads were washed three times with blocking buffer. The supernatant, washes, and resuspended beads were counted in a liquid scintillation counter. The fraction of fusions bound was calculated as the ratio of counts of the resuspended beads (the bound fraction) over the sum of counts of the supernatant, washes, and resuspended beads (total counts).</p>
<p>For competition binding assays, the desired competitors (synthetic SPAM, synthetic SPAM2, synthetic SPAM3, atezolizumab [SelleckChem A2004], and WL12 [CPC Scientific CANC‐022A]) were dissolved in dimethyl sulfoxide (DMSO) or water (for atezolizumab) to create stocks at 1000X the concentrations required for each assay. 10 nM hPD‐L1 was immobilized onto beads as above and 1 μL of each competitor stock was added and incubated at 4°C for 1 h. The radiolabeled fusions were then added and incubated at 4°C for 1 h. The beads were washed and counted as described above to determine the fraction of fusions bound.</p></section><section id="pro70268-sec-0006"><h3 class="pmc_sec_title">2.4. Biolayer interferometry analysis</h3>
<p>The binding kinetics of the SPAM peptides with hPD‐L1 were measured by biolayer interferometry (BLI) using an Octet RH16 (Sartorius). Biotinylated hPD‐L1 (1 μg mL<sup>−1</sup>) was immobilized onto a streptavidin‐coated biosensor (Sartorius 18‐5019) to a thickness of 1.4 nm, and biosensors without PD‐L1 were used for reference. Dilutions of SPAM (333, 111, 37, 12.4, and 4.12 nM), SPAM2 (6.25, 1.56, 0.391, 0.098, and 0.024 nM), SPAM3 (6.25, 1.56, 0.391, 0.098, and 0.024 nM), and SPAM4 (6.25, 1.56, 0.391, 0.098, and 0.024 nM) were prepared in assay buffer (20 mM HEPES‐KOH, pH 7.5, 150 mM NaCl, 0.05% [v/v] Tween‐20, 5% [v/v] glycerol, 1% [w/v] BSA, 0.1% [v/v] DMSO). The peptides were allowed to associate with the biosensors for 180 s and then allowed to dissociate in assay buffer for 900 s. The sensorgrams were normalized to no‐peptide values and globally fit to a 1:1 Langmuir binding model using Octet Analysis Studio 13.0 software. Each measurement was carried out in duplicate, and the mean values and fitting errors of the kinetic parameters were calculated.</p></section><section id="pro70268-sec-0007"><h3 class="pmc_sec_title">2.5. Flow cytometry</h3>
<p>The peptide reagents were tested for binding targets exogenously expressed in Chinese hamster ovary (CHO) cells. Test cells were either untransfected (parental) or stably transfected with hPD‐L1, hPD‐L2, or mPD‐L2 according to previous methods (Grindel et al., <a href="#pro70268-bib-0017" class="usa-link" aria-describedby="pro70268-bib-0017">2022</a>; Kamalinia et al., <a href="#pro70268-bib-0020" class="usa-link" aria-describedby="pro70268-bib-0020"><sup>2020</sup></a>). All cells were cultured in Dulbecco's Modified Eagle Medium (Corning, 10‐013‐CV) supplemented with 10% (v/v) heat‐inactivated fetal bovine serum (Sigma‐Aldrich, F4135) and 1× penicillin–streptomycin (Sigma‐Aldrich, P4333). For flow cytometry, cells were briefly washed with Mg<sup>+2</sup>‐ and Ca<sup>+2</sup>‐free phosphate‐buffered saline (PBS) and dissociated into nonadherent single cells with trypsin (0.25% (w/v) trypsin, 0.2% (w/v) EDTA; Sigma‐Aldrich, T4049). Trypsin was deactivated with full media, cells were centrifuged at 400<em>g</em> for 2 min, and the pellet was resuspended in serum‐free (SF) media. Cells were divided into v‐bottom plates (Corning, 3894) (200,000 cells per well), centrifuged, and resuspended in a dilution series of a fluorescein‐conjugated peptide reagent diluted in cold 1:1 PBS:SF media. Cells were incubated without agitation with the reagents for 1 h at 4°C in the dark. This protocol was followed for glioma D423 cells and breast cancer MDA‐MB‐231 cells and MDA‐MB‐468 cells. For antibody competitive binding, durvalumab or trastuzumab control dilutions were included in the mixture with 500 nM fluorescein‐labeled peptide. After binding, cells were centrifuged at 400<em>g</em> and washed three times with cold PBS, resuspended, and immediately acquired for live single‐cell fluorescent intensity in a Cytek® Northern Lights™ flow cytometer (488‐nm laser excitation). The gain was adjusted so that the highest observed fluorescence intensity was below saturation intensity and background signal was well above zero. Parental CHO cells were used to normalize nonspecific uptake/background. Dissociation constants were determined by GraphPad Prism using the one‐site normalized specific binding model. For competitive binding, nonlinear regression models using one‐site Fit Ki were used to generate a dissociation constant for the antibody.</p>
<p>For peptide competition with fluorescently labeled durvalumab (R&amp;D Systems, Biosimilar, FAB10355G), cells were processed as above but were instead incubated with a dilution series of unlabeled SPAM2, SPAM3, SPAM4, WL12, or control % (v/v) matched DMSO for 45 min at 4°C. The fluorescent antibody or IgG control antibody (Biolegend, 40010, clone MOPC‐21) for background was added at a final concentration of 0.35 nM following incubation. Cells were incubated for an additional 40 min at 4°C in the dark and processed as above for flow cytometry. Values were subtracted for IgG control and normalized to DMSO controls per peptide concentration.</p></section><section id="pro70268-sec-0008"><h3 class="pmc_sec_title">2.6. Peptide solubility and aggregation</h3>
<p>Lyophilized peptides were reconstituted in phosphate buffer (PB, 15 mM at pH 6.5, or 50 mM at pH 4.5) with small amounts of 10%–20% (v/v) MeOH or 2,2,2‐trifluoroethanol (TFE) as organic cosolvents. Samples were vortexed; then sonicated for 5 min for maximum solubilization. Sample concentrations were determined by UV‐absorbance using a JASCO V‐670 spectrophotometer based on the combined molar absorptivity at 280 nm of Trp and Tyr residues present in the peptide sequence (<em>ε</em>
<sub>280</sub> = 5500 M<sup>−1</sup> cm<sup>−1</sup> per Trp, <em>ε</em>
<sub>280</sub> = 1490 M<sup>−1</sup> cm<sup>−1</sup> per Tyr). At higher concentrations of peptide, UV‐absorbances were prone to instability; thus, a constant absorbance over the course of 20 min was confirmed for each measurement to ensure a lack of aggregation. This instability was not observed for lower concentrations and higher percentages of organic solvents. In addition, high concentrations (70–100 μM) and temperatures (40–50°C) were found to trigger irreversible aggregation, as evident in the far‐UV circular dichroism (CD) spectra. Observing this onset of aggregation is strong evidence that SPAM1‐3 are in a monomeric state at physiological temperature (37°C) and at concentrations up to 40 μM.</p></section><section id="pro70268-sec-0009"><h3 class="pmc_sec_title">2.7. Far‐UV CD measurements and denaturation studies</h3>
<p>Samples were prepared in a mixture of PB (15 mM at pH 6.5 or 50 mM at pH 4.5) and methanol (PB/MeOH 80:20 vol/vol) in a concentration range between 17 and 70 μM, determined by UV absorbance measurements. In brief, raw CD spectra at varying temperatures (0–90°C) were recorded as previously described (Greenfield, <a href="#pro70268-bib-0015" class="usa-link" aria-describedby="pro70268-bib-0015">2006</a>) on a JASCO J‐810 spectropolarimeter with a temperature controller module JASCO PFD‐425S in a quartz cuvette (0.1 cm). The instrument was calibrated using (+)‐10‐camphorsulfonic acid provided by JASCO. CD spectra were recorded in mdeg from 190 to 270 nm, and CD spectra of the blank (solvent only) were collected for subtraction. A temperature ramp from 0 to 90°C with a rate of 45°C per hour under nitrogen flow rate (2 L min<sup>−1</sup>) was used to obtain CD spectra (averaged of eight scans) at set temperatures representing the unfolding transitions for each peptide. Raw spectra were smoothed, and the baseline was set to 0 between 260 and 270 nm using SpectraGryph 1.2 (Menges, <a href="#pro70268-bib-0034" class="usa-link" aria-describedby="pro70268-bib-0034">2022</a>). <a href="http://www.effemm2.de/spectragryph/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">(http://www.effemm2.de/spectragryph/</a>). The ellipticity scale was then converted from mdeg into molar ellipticity (<em>θ</em>) (deg cm<sup>2</sup> dmol<sup>−1</sup>) by dividing with the corresponding number of amide bonds (17 for SPAM and 21 for SPAM2 and SPAM3), the concentration, and the pathlength. To determine the thermal stability of the peptides, a melting curve was generated from CD spectra by plotting temperature versus molar ellipticity at 220 nm for SPAM2 and 222 nm for SPAM3. Maxima obtained from the first derivative plots of these CD‐melts were used to identify the melting temperatures of 40 ± 5°C for both SPAM2 and SPAM3.</p></section></section><section id="pro70268-sec-0010"><h2 class="pmc_sec_title">3. RESULTS AND DISCUSSION</h2>
<section id="pro70268-sec-0011"><h3 class="pmc_sec_title">3.1. Maturation selection and identification of SPAM2
</h3>
<p>We sought to build upon our previous development efforts to identify linear, proteinogenic peptides with high affinity for hPD‐L1. Our design strategy rested upon using extension libraries with a “doped” core SPAM motif flanked by fully randomized tetrapeptide sequences at the N‐ and C‐termini (Figure <a href="#pro70268-fig-0001" class="usa-link">1</a>). Previous mutational analysis of SPAM showed residues 6–18 (DHILNKFLILHYA) represented a core motif for hPD‐L1 binding (Kamalinia et al., <a href="#pro70268-bib-0020" class="usa-link" aria-describedby="pro70268-bib-0020">2020</a>). Based on mutation analysis and high‐throughput sequencing data of SPAM, this core was expected to conserve the affinity, binding site, selectivity, and stability of the parental SPAM peptide. Therefore, we designed an mRNA display library where the core region of SPAM was doped at a wildtype amino acid frequency of ~50% (Figure <a href="#pro70268-fig-0001" class="usa-link">1</a>). This mutational frequency is sufficient to allow a search for mutations in the core segment that might confer higher affinities, yet low enough that the core was still highly similar to SPAM. We also included four fully random amino acids at the N‐ and C‐terminus to extend the PD‐L1 contact surface and improve peptide affinity.</p>
<figure class="fig xbox font-sm" id="pro70268-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b14e/12376068/0f1bc2a3f9ba/PRO-34-e70268-g002.jpg" loading="lazy" id="jats-graphic-1" height="89" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/pro70268-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Affinity maturation of the SPAM peptide. An mRNA display library of 22‐mer peptides was designed based on our previous SPAM peptide (top). The core motif of SPAM was doped at ~50% wildtype amino acid frequency (red) and flanked by eight fully randomized amino acids (green) to provide N‐ and C‐terminal extensions.</p></figcaption></figure><p>Using this library, we performed several rounds of selection using mRNA display to select sequences that bind hPD‐L1 (Figure <a href="#pro70268-supitem-0001" class="usa-link">S1</a>). The average binding of each selection pool was measured using radioligand binding assays to monitor selection progress (Figure <a href="#pro70268-supitem-0001" class="usa-link">S2</a>). Significant target‐specific binding emerged in Round 2, and stringent selection pressures were applied in subsequent rounds to further enrich high‐affinity sequences. To increase the stringency of selection, the concentration of immobilized hPD‐L1 was reduced from 300 to 25 nM, and the binding temperature was increased from 4 to 22°C. In addition, in Rounds 6 and 7, an off‐rate selection was performed by adding 100‐fold excess non‐immobilized hPD‐L1 as a specific competitor, enabling sequences with slower off‐rates to be selected (Boder &amp; Wittrup, <a href="#pro70268-bib-0003" class="usa-link" aria-describedby="pro70268-bib-0003">1998</a>).</p>
<p>We performed next‐generation sequencing on the final DNA pool, and several sequences with the highest frequencies were evaluated using radioligand binding assays (Figure <a href="#pro70268-supitem-0001" class="usa-link">S3</a>). All tested peptides showed binding to immobilized hPD‐L1 and little binding to beads without target. Of these peptides, SPAM2 (MWSRSDHNLNKFWILHYSANPS) demonstrated the highest binding. Residues 6–18 of SPAM2 are 77% identical to the core motif from SPAM, and this high similarity suggests that SPAM2 likely retains the same selectivity and binding site as SPAM. Further, the conservation of these core residues validates the choice to use the core region from SPAM as a starting point to select for peptides with enhanced affinity.</p></section><section id="pro70268-sec-0012"><h3 class="pmc_sec_title">3.2. Mutational analysis of SPAM2 and identification of SPAM3
</h3>
<p>We then performed alanine scanning mutagenesis on SPAM2 to evaluate its structure–activity relationship and identify potential positions or substitutions that might further optimize the sequence. Alanine scanning is a well‐established technique to determine the contribution of individual amino acids in a protein–protein interaction (Clackson &amp; Wells, <a href="#pro70268-bib-0007" class="usa-link" aria-describedby="pro70268-bib-0007">1995</a>). All positions of SPAM2 were individually mutated to alanine, and the resulting sequences were evaluated in radioligand binding assays (Figure <a href="#pro70268-fig-0002" class="usa-link">2a</a>). The D6A, H7A, N10A, F12A, I14A, and H16A mutations all abolished binding, indicating that these positions are vital for interaction with hPD‐L1. These residues were all randomized in the starting mRNA display library, and their reversion to the wildtype residue in SPAM2 also supports the hypothesis that these wildtype residues are critical for binding hPD‐L1. On the other hand, the N8A, S18A, P21A, and S22A mutants showed no statistical difference in binding relative to the wildtype SPAM2 sequence, suggesting that these positions do not contribute significantly to the binding of hPD‐L1. Many of these less critical positions are located at the C‐terminus, suggesting that SPAM2 could be truncated to a shorter, yet still active peptide.</p>
<figure class="fig xbox font-sm" id="pro70268-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b14e/12376068/2aed3b07de3a/PRO-34-e70268-g005.jpg" loading="lazy" id="jats-graphic-3" height="554" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/pro70268-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mutational analysis of SPAM2. Radiolabeled peptide‐mRNA fusions of SPAM2 mutants were incubated with immobilized hPD‐L1, and the fraction of bound fusions was measured by scintillation counting. (a) SPAM2 mutants with individual alanine mutations and (b) SPAM2 truncations were incubated with 5 nM immobilized hPD‐L1 at 4°C for 1 h. (c) The binding of SPAM2 and SPAM3 was measured at the indicated binding temperatures and hPD‐L1 concentrations. Each measurement was carried out in triplicate, and error bars indicate the standard deviation from the mean (SD). Student's <em>t</em>‐test was used to compare each mutant with SPAM2: *<em>p</em> &lt; 0.05; **<em>p</em> &lt; 0.01; ***<em>p</em> &lt; 0.005; ****<em>p</em> &lt; 0.001 or #<em>p</em> &lt; 0.001. The amino acid sequences of each mutant are listed in Table <a href="#pro70268-supitem-0001" class="usa-link">S1</a>. hPD‐L1, human programmed death‐ligand 1.</p></figcaption></figure><p>We tested several truncations of SPAM2 to determine if a shorter peptide might retain binding of the parental peptide. We tested SPAM2 with a deletion of residues 2–5 (no N), a deletion of residues 19–22 (no C), or a deletion of residues 2–5 and 19–22 (no NC). These deletion constructs were evaluated using radioligand binding assays to determine the relative contributions of each terminus and to further define the boundaries of the core motif region (positions 6–18) (Figure <a href="#pro70268-fig-0002" class="usa-link">2b</a>). All truncated sequences showed significantly reduced binding relative to the SPAM2 wildtype sequence. Truncation of the N‐terminus showed a greater decrease in binding than truncation of the C‐terminus, and truncation of both termini almost completely abrogated binding. The larger loss of binding upon truncation of the N‐terminus as compared with the C‐terminus suggests that the N‐terminal residues of SPAM2 contribute significantly to hPD‐L1 binding. On the other hand, in line with the alanine scanning experiments above, the C‐terminal residues contributed less strongly to SPAM2 binding. These results are corroborated by next generation sequencing data, which showed high sequence conservation at the N‐terminus and only modest conservation at the C‐terminus (Figure <a href="#pro70268-supitem-0001" class="usa-link">S4</a>).</p>
<p>Taken together, alanine scanning and truncation data suggest that SPAM2 cannot be truncated without significant loss of affinity for hPD‐L1 and that the core motif alone (residues 6–18) is not sufficient to recapitulate SPAM2 binding. The importance of these residues to hPD‐L1 binding supports our strategy of using an extension selection to add flanking residues to a core peptide region to increase binding affinity. Further, the alanine scanning data suggest that the identity of the amino acids in the N‐ and C‐terminal extensions of SPAM2 is less important, yet they still contribute to hPD‐L1 binding, suggesting that these regions may interact with hPD‐L1 via main chain or backbone contacts or through intramolecular stabilization of the core SPAM2 motif.</p>
<p>While SPAM2 demonstrated the highest binding of all the sequences we tested, the theoretical diversity of the library (20<sup>21</sup> = ~10<sup>27</sup>) far exceeds the maximum practical throughput of mRNA display selections (~10<sup>12</sup> to 10<sup>14</sup>). It is thus likely that there are other 22‐mer sequences with even higher hPD‐L1 affinity. Previously, we showed the frequency of a given sequence present in late‐round mRNA display selection pools broadly correlates with functionality (Olson et al., <a href="#pro70268-bib-0036" class="usa-link" aria-describedby="pro70268-bib-0036">2012</a>). Additionally, Olson et al. (<a href="#pro70268-bib-0037" class="usa-link" aria-describedby="pro70268-bib-0037">2014</a>) demonstrated that epistasis or nonadditive synergy between mutations is rare in directed evolution systems such as mRNA display. Therefore, we hypothesized that an even higher affinity sequence could be constructed from next generation sequencing data by using the most frequent amino acid at each position (Figure <a href="#pro70268-supitem-0001" class="usa-link">S4</a>) even if that sequence did not appear in next generation sequencing. We thus constructed a “synthetic winner” sequence (MYSSTDHILNKFLILHYALNPT), which we designated as SPAM3. SPAM3 contains eight mutations compared with SPAM2, including at positions 8, 18, and 22, which did not contribute significantly to hPD‐L1 binding in the alanine scanning experiments. SPAM3 demonstrated higher binding than SPAM2 at every tested condition (Figure <a href="#pro70268-fig-0002" class="usa-link">2c</a>) in radioligand binding assays with variable hPD‐L1 concentration and binding temperature. These data suggest SPAM3 has higher affinity and supports a frequency‐based approach for identifying and designing new sequences based on high‐throughput sequencing data. Scrambled versions of SPAM2 and SPAM3 showed essentially no hPD‐L1 binding (Figure <a href="#pro70268-supitem-0001" class="usa-link">S5</a>), indicating that the binding of SPAM2 and SPAM3 is sequence‐dependent. SPAM2 and SPAM3 are nonhomologous to other known hPD‐L1 peptide ligands (Table <a href="#pro70268-supitem-0001" class="usa-link">S2</a>).</p></section><section id="pro70268-sec-0013"><h3 class="pmc_sec_title">3.3. 
SPAM2 and SPAM3 are stable in human serum</h3>
<p>Serum hydrolases can degrade linear, proteinogenic peptides and reduce their in vivo effectiveness. To improve proteolytic stability, many peptide drugs include nonproteinogenic amino acids, macrocyclic structure, and other chemical modifications (McGregor, <a href="#pro70268-bib-0032" class="usa-link" aria-describedby="pro70268-bib-0032">2008</a>). SPAM2 and SPAM3 are linear peptides comprised solely of proteinogenic amino acids and, as such, would be expected to be rapidly degraded by proteases in human serum. To study this, SPAM2 and SPAM3 were incubated in 25% (v/v) human serum at 37°C (ex vivo). Aliquots at 0, 15, 30, 120, and 300 min were analyzed by liquid chromatography‐mass spectrometry and analytical HPLC to quantify the peak area of each intact peptide (Figure <a href="#pro70268-supitem-0001" class="usa-link">S6</a>). SPAM3 showed only modest time‐dependent degradation, with ~70% of the parent compound remaining intact after 300 min of incubation, while SPAM2 showed no appreciable degradation even after 300 min of incubation. In contrast, GiBP, a linear peptide previously studied in our lab, was shown to be unstable in serum and was rapidly degraded over the course of the experiment (Howell et al., <a href="#pro70268-bib-0018" class="usa-link" aria-describedby="pro70268-bib-0018">2014</a>). This experiment suggests that both SPAM2 and SPAM3 are largely stable to degradation in serum. In both cases, on the time scale of a typical PET imaging experiment (&lt;120 min), the majority of each peptide would likely remain intact, suggesting that these peptides have sufficient stability for short half‐life nuclear imaging applications.</p></section><section id="pro70268-sec-0014"><h3 class="pmc_sec_title">3.4. 
SPAM3 binds hPD‐L1 with antibody‐like affinity at the PD‐1 interface</h3>
<p>We determined the kinetic binding parameters of SPAM, SPAM2, and SPAM3 with hPD‐L1 using bio‐layer interferometry (BLI) (Figures <a href="#pro70268-fig-0003" class="usa-link">3</a> and <a href="#pro70268-supitem-0001" class="usa-link">S7</a>) and surface plasmon resonance (SPR) (Figure <a href="#pro70268-supitem-0001" class="usa-link">S8</a>). These parameters are listed in Table <a href="#pro70268-tbl-0001" class="usa-link">1</a>. The equilibrium dissociation constant for SPAM (<em>K</em>
<sub>D</sub> = 38.2 ± 1.0 nM by BLI and 36 ± 29 nM by SPR) agrees with our previously published value from fluorescence polarization experiments (Kamalinia et al., <a href="#pro70268-bib-0020" class="usa-link" aria-describedby="pro70268-bib-0020">2020</a>). More impressively, SPAM2 (<em>K</em>
<sub>D</sub> = 303 ± 7 pM) and SPAM3 (<em>K</em>
<sub>D</sub> = 249 ± 3 pM) bind hPD‐L1 with ~100× higher affinity than SPAM (<em>K</em>
<sub>D</sub> = 38 ± 1 nM), indicating the extension selection was successful at improving the affinity of the core SPAM motif. The kinetic data also indicate SPAM2 and SPAM3 have both increased association and decreased dissociation rates compared with SPAM, demonstrating the improvements made by the mRNA display selection were not limited solely to reductions in the off‐rate. Additionally, we measured the affinity of SPAM4, a dimeric form of SPAM3, which is discussed further below (Figure <a href="#pro70268-supitem-0001" class="usa-link">S9</a>).</p>
<figure class="fig xbox font-sm" id="pro70268-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b14e/12376068/3ca408098eb7/PRO-34-e70268-g004.jpg" loading="lazy" id="jats-graphic-5" height="385" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/pro70268-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Biolayer interferometry of SPAM peptides with hPD‐L1. The binding kinetics of the interactions between hPD‐L1 and (a) SPAM, (b) SPAM2, (c) SPAM3, and (d) SPAM4 were measured using biolayer interferometry. The indicated peptide concentrations were allowed to associate onto biosensors coated in biotinylated hPD‐L1 for 180 s and then allowed to dissociate for 900 s. The sensorgrams were globally fit to a 1:1 (Langmuir) binding model; the fits are plotted alongside the measured data. hPD‐L1, human programmed death‐ligand 1.</p></figcaption></figure><section class="tw xbox font-sm" id="pro70268-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Binding kinetics of the SPAM peptides with hPD‐L1.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Peptide</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>k</em>
<sub>on</sub> (10<sup>4</sup> M<sup>−1</sup> s<sup>−1</sup>)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>k</em>
<sub>off</sub> (10<sup>−3</sup> s<sup>−1</sup>)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<em>K</em>
<sub>D</sub> (pM)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">SPAM</td>
<td align="center" valign="top" rowspan="1" colspan="1">141 ± 3</td>
<td align="center" valign="top" rowspan="1" colspan="1">49.0 ± 0.9</td>
<td align="center" valign="top" rowspan="1" colspan="1">38,200 ± 1000</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">SPAM2</td>
<td align="center" valign="top" rowspan="1" colspan="1">878 ± 21</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.64 ± 0.02</td>
<td align="center" valign="top" rowspan="1" colspan="1">303 ± 7</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">SPAM3</td>
<td align="center" valign="top" rowspan="1" colspan="1">578 ± 8</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.46 ± 0.01</td>
<td align="center" valign="top" rowspan="1" colspan="1">249 ± 3</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">SPAM4</td>
<td align="center" valign="top" rowspan="1" colspan="1">223 ± 11</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.219 ± 0.004</td>
<td align="center" valign="top" rowspan="1" colspan="1">99 ± 1</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pro70268-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="pro70268-note-0118"><p>
<em>Note</em>: Mean on‐rates (<em>k</em>
<sub>on</sub>), off‐rates (<em>k</em>
<sub>off</sub>), and dissociation constants (<em>K</em>
<sub>D</sub>) are shown along with fitting errors.</p></div>
<div class="fn" id="pro70268-note-1118"><p>Abbreviation: hPD‐L1, human programmed death‐ligand 1.</p></div>
</div></section><p>The affinity of SPAM3 is remarkable when compared to natural ligands and approved therapeutics targeting hPD‐L1. SPAM3 binds hPD‐L1 with more than 10,000‐fold higher affinity than the natural ligand PD‐1 (<em>K</em>
<sub>D</sub> = 8.2 ± 0.11 μM), highlighting its potential for hPD‐L1 binding in an active tumor‐immune microenvironment (Cheng et al., <a href="#pro70268-bib-0006" class="usa-link" aria-describedby="pro70268-bib-0006">2013</a>). Additionally, SPAM3 binds with higher affinity than the single‐chain fragment variable (scFv) of atezolizumab (<em>K</em>
<sub>D</sub> = 1750 pM) and durvalumab (<em>K</em>
<sub>D</sub> = 667 pM) and with slightly lower affinity than the scFv of avelumab (<em>K</em>
<sub>D</sub> = 46.7 pM), making its binding comparable to that of therapeutic antibodies (Tan et al., <a href="#pro70268-bib-0047" class="usa-link" aria-describedby="pro70268-bib-0047">2018</a>).</p>
<p>CD spectroscopy is routinely used to study the secondary structures of linear peptides and conformational changes induced by solvent, pH, or temperature. Far‐UV CD spectra of SPAM peptides were recorded in aqueous buffers at pH 4.5 or 6.5 with the addition of methanol (for solubility) or a more structuring cosolvent TFE (Figure <a href="#pro70268-fig-0004" class="usa-link">4</a>). For SPAM, a drastic pH‐dependent change of folding could be observed with a β‐pleated sheet preference at pH 4.5 marked by a large negative band at 214 ± 2 nm to a more helical structure at pH 6.5 shown by the negative minima at about 206 and 221 ± 2 nm (Figure <a href="#pro70268-fig-0004" class="usa-link">4a</a>). In addition, the thermal denaturation of SPAM recorded at pH 6.5 between 0 and 90°C (Figure <a href="#pro70268-fig-0004" class="usa-link">4d</a>) revealed not only the unfolding of the β‐sheet but also a change of state leading to an irreversible aggregation above 50°C. In contrast, SPAM2 was found to have a more stable and well‐behaved antiparallel β‐sheet fold in all solvent and pH conditions tested (positive maxima at 197 ± 2 nm and minima at 214 ± 2 nm) (Figure <a href="#pro70268-fig-0004" class="usa-link">4b</a>). In this case, the thermal denaturation study revealed a much smoother transition with a β‐sheet fraying leading to a random coil state (200 ± 2 nm) above 50°C (Figure <a href="#pro70268-fig-0004" class="usa-link">4e</a>). Whereas at low pH SPAM3 presents a strong α‐helix character (208/221 ± 2 nm) especially in the presence of NaF (Figure <a href="#pro70268-fig-0004" class="usa-link">4c</a>), the peptide fold turns into a mixture of helical and β‐sheet structures at pH 6.5. Unlike SPAM2, SPAM3's antiparallel β‐sheet fold is certainly in equilibrium with some helical structures leading to a complex thermal denaturation mechanism that is likely occurring along with significant aggregation (Figure <a href="#pro70268-fig-0004" class="usa-link">4f</a>). Yet the CD‐melts transitions of SPAM2 and SPAM3 at ~40 ± 5°C indicate that these peptides are more thermostable than SPAM. Under all conditions tested, the low solubility of SPAM4 made it challenging to reach a workable window of concentrations for CD spectroscopy.</p>
<figure class="fig xbox font-sm" id="pro70268-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b14e/12376068/1fcc478a0496/PRO-34-e70268-g001.jpg" loading="lazy" id="jats-graphic-7" height="366" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/pro70268-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Circular dichroism (CD) of SPAM peptides. The CD spectra of (a) SPAM, (b) SPAM2, and (c) SPAM3 recorded in (1) 50 mM phosphate buffer (PB) with 20% MeOH (solid green line, pH 4.5; solid blue line, pH 6.5), (2) 50 mM PB with 150 mM NaF in 20% MeOH (dotted green line, pH = 6.5; dotted blue line, pH = 4.50), and (3) 50 mM PB, pH = 6.5 in 20% TFE (black line). Representative thermal denaturations of (d) SPAM, (e) SPAM2, and (f) SPAM3 in 50 mM PB (pH 6.5) and methanol (1:4 vol/vol) with 150 mM NaF (only e, f), between 0 and 90°C showing that SPAM rapidly aggregates whereas the β‐sheet and helical structures of SPAM2 and SPAM3 unfold progressively upon warming. The denatured SPAM peptides did not refold upon cooling. TFE, 2,2,2‐trifluoroethanol.</p></figcaption></figure><p>These experimental results suggest that SPAM2 adopts a predominantly antiparallel β‐sheet fold in solution. If the SPAM peptides bind hPD‐L1 as β‐sheets, then the increased propensity of SPAM2 and SPAM3 for this conformation could contribute to their increased affinity for hPD‐L1 relative to SPAM. To investigate this hypothesis, we used molecular docking and molecular dynamics simulation to model the binding interaction between SPAM2 and hPD‐L1. The simulation indicated SPAM2 binds hPD‐L1 at the PD‐1 binding site and adopts a β‐hairpin conformation (Figures <a href="#pro70268-supitem-0001" class="usa-link">S10</a> and <a href="#pro70268-supitem-0001" class="usa-link">S11</a>). We corroborated this model using competition radioligand binding assays to assess the binding sites of SPAM2 and SPAM3 on hPD‐L1. We measured the binding of radiolabeled SPAM2 and SPAM3 fusions to resin‐immobilized hPD‐L1 in the presence of atezolizumab (a therapeutic anti‐PD‐L1 antibody), WL12 (a macrocyclic peptide), SPAM, SPAM2, SPAM3, or a DMSO‐only control (Figure <a href="#pro70268-supitem-0001" class="usa-link">S12</a>). The presence of atezolizumab or WL12, which both occlude the PD‐1 binding site (Chatterjee et al., <a href="#pro70268-bib-0005" class="usa-link" aria-describedby="pro70268-bib-0005">2017</a>; Zhang et al., <a href="#pro70268-bib-0050" class="usa-link" aria-describedby="pro70268-bib-0050"><sup>2017</sup></a>), almost completely eliminated binding of SPAM2 and SPAM3 to hPD‐L1, indicating that SPAM2 and SPAM3 bind at the PD‐1 site. These experiments provide a preliminary hypothesis for the increased affinity of SPAM2 and SPAM3.</p></section><section id="pro70268-sec-0015"><h3 class="pmc_sec_title">3.5. 
SPAM2 and SPAM3 specifically bind hPD‐L1 on the surface of living cells</h3>
<p>With clear indications that SPAM2 and SPAM3 bind hPD‐L1 with picomolar affinity in vitro, we next sought to determine their affinity for hPD‐L1 in a cellular context by testing their binding to hPD‐L1 expressed on the surface of living cells. Flow cytometry demonstrated robust binding of fluorescein‐labeled SPAM2, SPAM3, and SPAM4 (discussed below) to hPD‐L1 on the surface of CHO cells (Figure <a href="#pro70268-fig-0005" class="usa-link">5</a>). In flow cytometry experiments, SPAM2‐fluorescein bound to CHO cells with stably expressing hPD‐L1 (Figure <a href="#pro70268-fig-0005" class="usa-link">5a</a>) yet showed no significant binding to CHO cells expressing mouse PD‐L1 (mPD‐L1) or the closely related human PD‐L2 (hPD‐L2). This indicates that selectivity for hPD‐L1 is maintained in the affinity‐matured sequences despite hPD‐L1 being 77% and 34% sequence identical to mPD‐L1 and hPD‐L2, respectively (Latchman et al., <a href="#pro70268-bib-0023" class="usa-link" aria-describedby="pro70268-bib-0023">2001</a>; Lazar‐Molnar et al., <a href="#pro70268-bib-0024" class="usa-link" aria-describedby="pro70268-bib-0024"><sup>2008</sup></a>; Lin et al., <a href="#pro70268-bib-0028" class="usa-link" aria-describedby="pro70268-bib-0028"><sup>2008</sup></a>). SPAM2‐fluorescein also bound to endogenous hPD‐L1 in glioblastoma (D423) and triple‐negative breast cancer cell lines (MDA‐MB‐231) in a concentration‐dependent manner. Meanwhile, hPD‐L1‐negative MDA‐MB‐486 cells showed no concentration‐dependent labeling with SPAM2‐fluorescein. We note that the enhanced maximum signal observed in CHO hPD‐L1 relative to MDA‐MB‐231 and D423 reflects the significantly higher hPD‐L1 expression in the engineered CHO cell lines relative to the cancer cell lines. The higher signal in CHO‐hPD‐L1 cells in comparison to D423 and MDA‐MB‐231 was confirmed by flow cytometry with a biosimilar version of the durvalumab anti‐PD‐L1 antibody (Figure <a href="#pro70268-supitem-0001" class="usa-link">S13</a>) and western blot analysis of total hPD‐L1 expression (Figure <a href="#pro70268-supitem-0001" class="usa-link">S14</a>).</p>
<figure class="fig xbox font-sm" id="pro70268-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b14e/12376068/d224800782bc/PRO-34-e70268-g006.jpg" loading="lazy" id="jats-graphic-9" height="338" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/pro70268-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Binding of SPAM2 and its variants with hPD‐L1 on the surface of live cells. (a) SPAM2‐fluorescein was assessed for concentration‐dependent binding in multiple cell lines. (b) SPAM2 binding was compared directly to SPAM3 and SPAM4 in CHO hPD‐L1 expressing cells. Values were normalized to those observed with parental CHO cells. <em>K</em>
<sub>D</sub> values with 95% confidence interval (CI) values were determined using specific binding models in GraphPad software. CHO, Chinese hamster ovary; hPD‐L1, human programmed death‐ligand 1.</p></figcaption></figure><p>The native IgG forms of therapeutic anti‐hPD‐L1 antibodies likely benefit from avidity effects and bind hPD‐L1 with higher apparent affinities than their corresponding scFvs. With the hope of leveraging bivalent avidity, we developed dimeric forms of SPAM2 and SPAM3 by synthesizing two monomers of each peptide linked via a central bis‐Fmoc lysine at the C‐terminus. We used polyethylene glycol (PEG) spacers with either 5 or 10 repeats to test several different spacers that we hypothesized would be necessary to impart sufficient distance between each peptide arm to facilitate bivalent interactions with hPD‐L1.</p>
<p>We triaged these dimeric peptide variants against CHO‐hPD‐L1 and D423 using flow cytometry to assess concentration‐dependent binding (Figure <a href="#pro70268-supitem-0001" class="usa-link">S15</a>). Of all the constructs, the SPAM3 dimer with PEG<sub>10</sub> spacers showed the highest hPD‐L1 affinity in both cell lines and was designated SPAM4 (Figure <a href="#pro70268-supitem-0001" class="usa-link">S9</a>). In general, bivalent forms of SPAM3 performed better than their monovalent forms, but surprisingly SPAM2 dimers bound worse than their monomeric counterparts. Among all bivalent peptides, the PEG<sub>10</sub> spacer was universally superior to the PEG<sub>5</sub> spacer, suggesting increased affinity from bivalency requires appropriate geometry and distance. More importantly, SPAM4 showed a greater improvement over SPAM3 in D423 cells (~20×) than in CHO‐hPD‐L1 cells (~5×) while having similar affinity for hPD‐L1 across both cell types. BLI analysis indicated a 99 pM affinity for SPAM4 (Figure <a href="#pro70268-fig-0003" class="usa-link">3</a> and Table <a href="#pro70268-tbl-0001" class="usa-link">1</a>).</p>
<p>To quantitatively determine the relative binding of these peptides, we determined the binding of SPAM2, SPAM3, and SPAM4 to hPD‐L1 on the surface of CHO hPD‐L1 cells using flow cytometry (Figure <a href="#pro70268-fig-0005" class="usa-link">5b</a>). SPAM2 (<em>K</em>
<sub>D</sub> = 15.3 nM) is the weakest binder, agreeing with the in vitro binding data above. SPAM4 showed the strongest binding (<em>K</em>
<sub>D</sub> = 3.47 nM), followed by SPAM3 (<em>K</em>
<sub>D</sub> = 5.15 nM); though this difference is not statistically significant (overlapping 95% confidence intervals). The <em>K</em>
<sub>D</sub>s of SPAM3 and SPAM4 are very close to that of durvalumab in the CHO‐hPD‐L1 cell line (Figure <a href="#pro70268-supitem-0001" class="usa-link">S13A</a>; 2.5 nM), further demonstrating their antibody‐like affinities. The scrambled versions of SPAM2, SPAM3, and SPAM4 showed no significant binding to CHO‐hPD‐L1 cells (Figure <a href="#pro70268-supitem-0001" class="usa-link">S15D</a>), confirming that the peptide: protein interaction is sequence‐dependent. Further, SPAM3 and SPAM4 showed no binding to CHO‐mPD‐L1 and CHO‐hPD‐L2 cells at the highest concentration tested (1200 nM; Figure <a href="#pro70268-supitem-0001" class="usa-link">S15E</a>), demonstrating conservation of target selectivity.</p>
<p>With confirmation of high affinity binding across multiple cell lines, we sought to use cell‐based flow cytometry inhibition experiments to confirm that the binding site of the SPAM peptides overlaps with PD‐1 as determined by the in vitro antibody displacement radioligand competition assays above (Figure <a href="#pro70268-supitem-0001" class="usa-link">S12</a>). CHO hPD‐L1 (Figure <a href="#pro70268-fig-0006" class="usa-link">6a</a>), D423 (Figure <a href="#pro70268-fig-0006" class="usa-link">6b</a>), and MDA‐MB‐231 (Figure <a href="#pro70268-fig-0006" class="usa-link">6c</a>) cells were incubated with increasing concentrations of SPAM2, SPAM3, SPAM4, WL12 (positive control), or DMSO (as a background control), followed by the addition of fluorescein‐labeled biosimilar durvalumab mAb. Durvalumab and WL12 both inhibit hPD‐1/PD‐L1 engagement by binding to the PD‐1 interface (Lesniak et al., <a href="#pro70268-bib-0025" class="usa-link" aria-describedby="pro70268-bib-0025">2019</a>; Tan et al., <a href="#pro70268-bib-0047" class="usa-link" aria-describedby="pro70268-bib-0047"><sup>2018</sup></a>). Thus, if the peptides displace durvalumab, then they likely bind to the PD‐1 interface in vivo. SPAM2, SPAM3, and SPAM4, along with the control WL12 peptide, competed with fluorescent durvalumab in a concentration‐dependent manner in all tested cell lines. This displacement indicates at least a partial overlap of the binding surface with durvalumab and, therefore, the PD‐1 binding interface. These data further validate the in vitro competition data showing that the SPAM peptides compete with atezolizumab, which also binds the PD‐1 site (Figure <a href="#pro70268-supitem-0001" class="usa-link">S12</a>). Interestingly, IC<sub>50</sub> values for SPAM2, SPAM3, SPAM4, and the WL12‐positive control showed significant variance across cell lines. The SPAM peptides showed similar IC<sub>50</sub> values as WL12 in CHO‐hPD‐L1 cells but showed higher IC<sub>50</sub> values in D423 and MDA‐MB‐231 cell lines (Figure <a href="#pro70268-fig-0006" class="usa-link">6d</a>). The binding of the biosimilar durvalumab antibody also varied across cell lines (Figure <a href="#pro70268-supitem-0001" class="usa-link">S13A–C</a>) with CHO‐hPD‐L1 cells showing the lowest affinity (2.5 nM).</p>
<figure class="fig xbox font-sm" id="pro70268-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b14e/12376068/4991cb6d0c36/PRO-34-e70268-g003.jpg" loading="lazy" id="jats-graphic-11" height="344" width="709" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/pro70268-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Durvalumab inhibition with SPAM2, SPAM3, and SPAM4. Fluorescein‐labeled durvalumab was inhibited with increasing concentrations of SPAM2, SPAM3, and SPAM4 or control‐matched % (v/v) DMSO on (a) CHO hPD‐L1 cells, (b) D423 cells, or (c) MDA‐MB‐231 cells. (d) Competition experiments IC<sub>50</sub> values with absolute‐value hill slopes (|<em>h</em>|) ([Inhibitor] vs. Response–Variable slope 4 parameters, GraphPad). CHO, Chinese hamster ovary; hPD‐L1, human programmed death‐ligand 1.</p></figcaption></figure><p>To further explore this interaction, we also performed the competition assay in reverse with SPAM2‐fluorescein binding in competition with unlabeled durvalumab on CHO hPD‐L1 cells (Figure <a href="#pro70268-supitem-0001" class="usa-link">S16A</a>). SPAM2‐fluorescein was incubated with CHO hPD‐L1 cells in the presence of increasing concentrations of durvalumab or trastuzumab (Herceptin, anti‐Her2) as a control. As expected, trastuzumab shows no effect on the binding of SPAM2‐fluorescein, while durvalumab is a potent competitor (<em>K</em>
<sub>
<em>i</em>
</sub> = 0.4 nM, 95% CI −0.12 to 1 nM). This provides additional evidence that SPAM2 binds at the PD‐1/hPD‐L1 interface and therefore potentially disrupts PD‐1 engagement.</p>
<p>If SPAM peptides block the PD‐1 interface, they should be able to reactivate T‐cells, which we tested using a cell‐based bioluminescent T‐cell reactivation assay. In this assay, compounds that block the interaction between hPD‐L1 on the antigen‐presenting cell and PD‐1 on the T cell (Jurkat cell) result in re‐activation of the NFAT reporter and induction of luciferase activity. Increasing amounts of SPAM2 and SPAM3, along with vehicle controls, were added, and the bioluminescence at each concentration was measured and normalized as relative luminescence units (RLUs). As seen in Figure <a href="#pro70268-supitem-0001" class="usa-link">S16B</a>, SPAM2 and SPAM3 reactivated T cells in a concentration‐dependent manner, restoring more than 50% of signal at 5 μM, further confirming that SPAM2 and SPAM3 block PD‐1 binding. The positive control atezolizumab (Figure <a href="#pro70268-supitem-0001" class="usa-link">S16C</a>) demonstrates the assay works as intended. Atezolizumab has an IC<sub>50</sub> of 0.76 nM (0.61–1.03 nM, 95% CI). Vehicle CHO‐K1 indicates the maximal RLU values where PD‐L1 is absent and unable to interfere with T‐cell activity. Taken together, these results suggest that later generation SPAM peptides could function as standalone PD‐1/PD‐L1 checkpoint inhibitors.</p></section></section><section id="pro70268-sec-0016"><h2 class="pmc_sec_title">4. CONCLUSION</h2>
<p>Many peptide development approaches rely on nonproteinogenic peptides to impart conformational constraint and enhanced metabolic stability. While these strategies have proven successful (Fiacco et al., <a href="#pro70268-bib-0009" class="usa-link" aria-describedby="pro70268-bib-0009">2016</a>; Fujino et al., <a href="#pro70268-bib-0011" class="usa-link" aria-describedby="pro70268-bib-0011"><sup>2016</sup></a>; Goto et al., <a href="#pro70268-bib-0013" class="usa-link" aria-describedby="pro70268-bib-0013"><sup>2008</sup></a>; Gray et al., <a href="#pro70268-bib-0014" class="usa-link" aria-describedby="pro70268-bib-0014"><sup>2021</sup></a>; Howell et al., <a href="#pro70268-bib-0018" class="usa-link" aria-describedby="pro70268-bib-0018"><sup>2014</sup></a>; Kawakami et al., <a href="#pro70268-bib-0021" class="usa-link" aria-describedby="pro70268-bib-0021"><sup>2008</sup></a>; Maini et al., <a href="#pro70268-bib-0030" class="usa-link" aria-describedby="pro70268-bib-0030"><sup>2019</sup></a>; Passioura &amp; Suga, <a href="#pro70268-bib-0039" class="usa-link" aria-describedby="pro70268-bib-0039"><sup>2017</sup></a>; Subtelny et al., <a href="#pro70268-bib-0046" class="usa-link" aria-describedby="pro70268-bib-0046"><sup>2008</sup></a>), nonproteinogenic functionalities and architectures increase the cost and complexity of synthesis and preclude genetic encoding of the resulting sequence into higher‐order biologics. The primary goal of this work was to design high‐affinity, proteinogenic peptides using an extension mRNA display selection based on a previously identified lead sequence (SPAM). This approach yielded SPAM2 and SPAM3 as fully proteinogenic peptides that bind recombinant hPD‐L1 in vitro and hPD‐L1 on the cell surface with high affinity. Indeed, the affinities of SPAM2, SPAM3, and SPAM4 are 100‐fold higher than the parental SPAM peptide and are comparable with the affinities of clinical anti‐hPD‐L1 antibodies. This improvement in affinity was achieved with a modest increase in molecular weight, demonstrating the molecular efficiency of the extension selection approach. SPAM2 and SPAM3 also showed relatively high stability to serum‐mediated hydrolysis in vitro suggesting, in addition to our previous work (Kamalinia et al., <a href="#pro70268-bib-0020" class="usa-link" aria-describedby="pro70268-bib-0020">2020</a>) and other works, that proteinogenic, linear peptides may be sufficiently stable for in vivo applications.</p>
<p>Our preliminary spectroscopic and computational analyses suggest that the enhanced PD‐L1 affinity of SPAM2 and SPAM3 may arise from increased structural pre‐organization. If so, the N‐ and C‐terminal extensions may act to stabilize a β‐hairpin conformation rather than introducing additional contacts with the hPD‐L1 target. Indeed, antiparallel β‐sheets are a common structural motif in endogenous and designed ligands that recognize the PD‐1 binding site on hPD‐L1 (Figure <a href="#pro70268-supitem-0001" class="usa-link">S11</a>). Although high‐resolution structural studies are required to confirm this binding model, it does suggest additional design strategies to further enhance affinity and stability.</p>
<p>While the SPAM3 sequence could theoretically have been present in the extension selection library, it was not found in the final pool despite having a higher affinity for hPD‐L1 than SPAM2. We calculate that the probability of SPAM3 being present in the original synthetic library was ~5 × 10<sup>−16</sup>, and with an mRNA display library size of 10<sup>12</sup>, roughly a 1 in 2000 chance of being found by our mRNA display selection. SPAM3 shows that even if selections are performed using the largest experimentally accessible libraries (10<sup>12</sup> to 10<sup>14</sup> sequences), these selections only search a very small fraction of all the theoretical sequences possible (20<sup>21</sup> to 10<sup>27</sup> possible sequences). Based on the findings of this work, we propose that higher affinity binders could be identified by applying aggregate analyses to high‐throughput sequencing data. Indeed, we were able to design a sequence with modestly higher affinity by simply using the most frequent amino acid at each position in the final selection pool. More powerful analyses could be implemented by applying machine learning models to next‐generation sequencing data, an approach which has been explored in other works (Liu et al., <a href="#pro70268-bib-0029" class="usa-link" aria-describedby="pro70268-bib-0029">2020</a>; Makowski et al., <a href="#pro70268-bib-0031" class="usa-link" aria-describedby="pro70268-bib-0031"><sup>2022</sup></a>; Parkinson et al., <a href="#pro70268-bib-0038" class="usa-link" aria-describedby="pro70268-bib-0038"><sup>2023</sup></a>; Saka et al., <a href="#pro70268-bib-0041" class="usa-link" aria-describedby="pro70268-bib-0041"><sup>2021</sup></a>; Schmidt &amp; Hildebrandt, <a href="#pro70268-bib-0043" class="usa-link" aria-describedby="pro70268-bib-0043"><sup>2021</sup></a>; Vinogradov et al., <a href="#pro70268-bib-0049" class="usa-link" aria-describedby="pro70268-bib-0049"><sup>2022</sup></a>).</p>
<p>We note that all three types of hPD‐L1 ligands (antibodies, SPAM peptides, and WL12) behave differently in different cell lines. Although we would predict that <em>B</em>
<sub>Max</sub> (the maximal signal) would vary across cell lines due to differential hPD‐L1 expression levels, we also observed large changes in <em>K</em>
<sub>D</sub> and <em>K</em>
<sub>
<em>i</em>
</sub> for all three ligands. This argues that the structure and behavior of hPD‐L1 itself may depend on the cellular context. Indeed, previous experiments have shown that glycosylation of hPD‐L1 has a significant effect on the affinity of clinical hPD‐L1 antibodies and that dimerization may also play a critical role in governing PD‐L1 behavior (Benicky et al., <a href="#pro70268-bib-0002" class="usa-link" aria-describedby="pro70268-bib-0002">2021</a>; Gao et al., <a href="#pro70268-bib-0012" class="usa-link" aria-describedby="pro70268-bib-0012"><sup>2020</sup></a>; Zhou, Chai, et al., <a href="#pro70268-bib-0051" class="usa-link" aria-describedby="pro70268-bib-0051"><sup>2022</sup></a>). Future work will focus on addressing the effects of these properties on ligand affinity and selectivity and the effects of ligand binding on hPD‐L1 function.</p></section><section id="pro70268-sec-0021"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Justin N. Ong:</strong> Formal analysis; validation; methodology; conceptualization; investigation; writing – original draft; writing – review and editing. <strong>Brian J. Grindel:</strong> Conceptualization; investigation; validation; writing – review and editing; writing – original draft; formal analysis; methodology. <strong>Scott A. Rankin:</strong> Validation; methodology. <strong>Sarah H. Naylon:</strong> Methodology; writing – review and editing. <strong>Anupallavi Srinivasamani:</strong> Methodology; validation. <strong>Guillaume J. Trusz:</strong> Methodology; writing – review and editing. <strong>Xiaowen Liang:</strong> Methodology; validation; writing – review and editing. <strong>Md. Nasir Uddin:</strong> Writing – review and editing; methodology; validation. <strong>Lauren Fuller:</strong> Methodology; validation; writing – review and editing. <strong>Michael Curran:</strong> Investigation; funding acquisition; writing – review and editing; supervision. <strong>Stephane P. Roche:</strong> Investigation; methodology; writing – review and editing. <strong>Terry T. Takahashi:</strong> Conceptualization; investigation; funding acquisition; writing – review and editing; validation; supervision. <strong>Richard W. Roberts:</strong> Conceptualization; funding acquisition; investigation; writing – review and editing; project administration; supervision. <strong>Steven W. Millward:</strong> Conceptualization; investigation; funding acquisition; writing – original draft; writing – review and editing; formal analysis; project administration; supervision.</p></section><section id="pro70268-sec-0018"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>JNO, BJG, TTT, SWM, and RWR are co‐inventors on US Patent Application 17/997,507, which incorporates some of the work described in this manuscript.</p></section><section id="sec19"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pro70268-supitem-0001"><div class="caption p"><p>
<strong>Data S1.</strong> Experimental details.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12376068/bin/PRO-34-e70268-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70268-s001.docx</a><sup> (3.2MB, docx) </sup>
</div></div></section></section><section id="pro70268-sec-0017" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This work was funded by a G.E. In‐kind Multi‐Investigator Imaging (MI2) Research Award (SWM, RWR, TTT), an MD Anderson Institutional Research Grant (SWM), and an MD Anderson Quantitative Imaging Analysis Core Partnership in Research Program Pilot Award (SWM). Surface plasmon resonance measurements were performed at the NanoBiophysics Core Facility at USC.</p></section><section id="notes1"><p>

Ong JN, Grindel BJ, Rankin SA, Naylon SH, Srinivasamani A, Trusz GJ, et al. Using extension‐based mRNA display to design antibody‐like proteinogenic peptides for human PD‐L1
. Protein Science. 2025;34(9):e70268. 10.1002/pro.70268

</p>
<section id="pro70268-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="pro70268-note-0001"><p>Justin N. Ong and Brian J. Grindel contributed equally to this study.</p></div>
<div class="fn p" id="pro70268-note-0002"><p>
<strong>Review Editor:</strong> Aitziber L. Cortajarena</p></div>
</div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Terry T. Takahashi, Email: tttakaha@usc.edu.</p>
<p>Richard W. Roberts, Email: richrob@usc.edu.</p>
<p>Steven W. Millward, Email: smillward@mdanderson.org.</p></section><section id="pro70268-sec-0020"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The data that support the findings of this study are available from the corresponding author upon reasonable request.</p></section><section id="pro70268-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="pro70268-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="pro70268-bib-0001">
<cite>
Bardhan K, Anagnostou T, Boussiotis VA. The PD1:PD‐L1/2 pathway from discovery to clinical implementation. Front Immunol. 2016;7:550. 10.3389/fimmu.2016.00550

</cite> [<a href="https://doi.org/10.3389/fimmu.2016.00550" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5149523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28018338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=The%20PD1:PD%E2%80%90L1/2%20pathway%20from%20discovery%20to%20clinical%20implementation&amp;author=K%20Bardhan&amp;author=T%20Anagnostou&amp;author=VA%20Boussiotis&amp;volume=7&amp;publication_year=2016&amp;pages=550&amp;pmid=28018338&amp;doi=10.3389/fimmu.2016.00550&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0002">
<cite>
Benicky J, Sanda M, Brnakova Kennedy Z, Grant OC, Woods RJ, Zwart A, et al. PD‐L1 glycosylation and its impact on binding to clinical antibodies. J Proteome Res. 2021;20(1):485–497. 10.1021/acs.jproteome.0c00521

</cite> [<a href="https://doi.org/10.1021/acs.jproteome.0c00521" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8158060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33073996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Proteome%20Res&amp;title=PD%E2%80%90L1%20glycosylation%20and%20its%20impact%20on%20binding%20to%20clinical%20antibodies&amp;author=J%20Benicky&amp;author=M%20Sanda&amp;author=Z%20Brnakova%20Kennedy&amp;author=OC%20Grant&amp;author=RJ%20Woods&amp;volume=20&amp;issue=1&amp;publication_year=2021&amp;pages=485-497&amp;pmid=33073996&amp;doi=10.1021/acs.jproteome.0c00521&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0003">
<cite>
Boder ET, Wittrup KD. Optimal screening of surface‐displayed polypeptide libraries. Biotechnol Prog. 1998;14(1):55–62. 10.1021/bp970144q

</cite> [<a href="https://doi.org/10.1021/bp970144q" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10858036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol%20Prog&amp;title=Optimal%20screening%20of%20surface%E2%80%90displayed%20polypeptide%20libraries&amp;author=ET%20Boder&amp;author=KD%20Wittrup&amp;volume=14&amp;issue=1&amp;publication_year=1998&amp;pages=55-62&amp;pmid=10858036&amp;doi=10.1021/bp970144q&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0004">
<cite>
Cetin M, Evenson WE, Gross GG, Jalali‐Yazdi F, Krieger D, Arnold D, et al. RasIns: genetically encoded intrabodies of activated Ras proteins. J Mol Biol. 2017;429(4):562–573. 10.1016/j.jmb.2016.11.008

</cite> [<a href="https://doi.org/10.1016/j.jmb.2016.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5798998/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27865780/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=RasIns:%20genetically%20encoded%20intrabodies%20of%20activated%20Ras%20proteins&amp;author=M%20Cetin&amp;author=WE%20Evenson&amp;author=GG%20Gross&amp;author=F%20Jalali%E2%80%90Yazdi&amp;author=D%20Krieger&amp;volume=429&amp;issue=4&amp;publication_year=2017&amp;pages=562-573&amp;pmid=27865780&amp;doi=10.1016/j.jmb.2016.11.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0005">
<cite>
Chatterjee S, Lesniak WG, Miller MS, Lisok A, Sikorska E, Wharram B, et al. Rapid PD‐L1 detection in tumors with PET using a highly specific peptide. Biochem Biophys Res Commun. 2017;483(1):258–263. 10.1016/j.bbrc.2016.12.156

</cite> [<a href="https://doi.org/10.1016/j.bbrc.2016.12.156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5253331/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28025143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&amp;title=Rapid%20PD%E2%80%90L1%20detection%20in%20tumors%20with%20PET%20using%20a%20highly%20specific%20peptide&amp;author=S%20Chatterjee&amp;author=WG%20Lesniak&amp;author=MS%20Miller&amp;author=A%20Lisok&amp;author=E%20Sikorska&amp;volume=483&amp;issue=1&amp;publication_year=2017&amp;pages=258-263&amp;pmid=28025143&amp;doi=10.1016/j.bbrc.2016.12.156&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0006">
<cite>
Cheng X, Veverka V, Radhakrishnan A, Waters LC, Muskett FW, Morgan SH, et al. Structure and interactions of the human programmed cell death 1 receptor. J Biol Chem. 2013;288(17):11771–11785. 10.1074/jbc.M112.448126

</cite> [<a href="https://doi.org/10.1074/jbc.M112.448126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3636866/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23417675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Structure%20and%20interactions%20of%20the%20human%20programmed%20cell%20death%201%20receptor&amp;author=X%20Cheng&amp;author=V%20Veverka&amp;author=A%20Radhakrishnan&amp;author=LC%20Waters&amp;author=FW%20Muskett&amp;volume=288&amp;issue=17&amp;publication_year=2013&amp;pages=11771-11785&amp;pmid=23417675&amp;doi=10.1074/jbc.M112.448126&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0007">
<cite>
Clackson T, Wells JA. A hot spot of binding energy in a hormone‐receptor interface. Science. 1995;267(5196):383–386. 10.1126/science.7529940

</cite> [<a href="https://doi.org/10.1126/science.7529940" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7529940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A%20hot%20spot%20of%20binding%20energy%20in%20a%20hormone%E2%80%90receptor%20interface&amp;author=T%20Clackson&amp;author=JA%20Wells&amp;volume=267&amp;issue=5196&amp;publication_year=1995&amp;pages=383-386&amp;pmid=7529940&amp;doi=10.1126/science.7529940&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0008">
<cite>
Doki Y, Ajani JA, Kato K, Xu J, Wyrwicz L, Motoyama S, et al. Nivolumab combination therapy in advanced esophageal squamous‐cell carcinoma. N Engl J Med. 2022;386(5):449–462. 10.1056/NEJMoa2111380

</cite> [<a href="https://doi.org/10.1056/NEJMoa2111380" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35108470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&amp;title=Nivolumab%20combination%20therapy%20in%20advanced%20esophageal%20squamous%E2%80%90cell%20carcinoma&amp;author=Y%20Doki&amp;author=JA%20Ajani&amp;author=K%20Kato&amp;author=J%20Xu&amp;author=L%20Wyrwicz&amp;volume=386&amp;issue=5&amp;publication_year=2022&amp;pages=449-462&amp;pmid=35108470&amp;doi=10.1056/NEJMoa2111380&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0009">
<cite>
Fiacco SV, Kelderhouse LE, Hardy A, Peleg Y, Hu B, Ornelas A, et al. Directed evolution of scanning unnatural‐protease‐resistant (SUPR) peptides for in vivo applications. Chembiochem. 2016;17(17):1643–1651. 10.1002/cbic.201600253

</cite> [<a href="https://doi.org/10.1002/cbic.201600253" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5167532/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27465925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chembiochem&amp;title=Directed%20evolution%20of%20scanning%20unnatural%E2%80%90protease%E2%80%90resistant%20(SUPR)%20peptides%20for%20in%20vivo%20applications&amp;author=SV%20Fiacco&amp;author=LE%20Kelderhouse&amp;author=A%20Hardy&amp;author=Y%20Peleg&amp;author=B%20Hu&amp;volume=17&amp;issue=17&amp;publication_year=2016&amp;pages=1643-1651&amp;pmid=27465925&amp;doi=10.1002/cbic.201600253&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0010">
<cite>
Freeman GJ, Long AJ, Iwai Y, Bourque K, Chernova T, Nishimura H, et al. Engagement of the PD‐1 immunoinhibitory receptor by a novel B7 family member leads to negative regulation of lymphocyte activation. J Exp Med. 2000;192(7):1027–1034. 10.1084/jem.192.7.1027

</cite> [<a href="https://doi.org/10.1084/jem.192.7.1027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2193311/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11015443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&amp;title=Engagement%20of%20the%20PD%E2%80%901%20immunoinhibitory%20receptor%20by%20a%20novel%20B7%20family%20member%20leads%20to%20negative%20regulation%20of%20lymphocyte%20activation&amp;author=GJ%20Freeman&amp;author=AJ%20Long&amp;author=Y%20Iwai&amp;author=K%20Bourque&amp;author=T%20Chernova&amp;volume=192&amp;issue=7&amp;publication_year=2000&amp;pages=1027-1034&amp;pmid=11015443&amp;doi=10.1084/jem.192.7.1027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0011">
<cite>
Fujino T, Goto Y, Suga H, Murakami H. Ribosomal synthesis of peptides with multiple beta‐amino acids. J Am Chem Soc. 2016;138(6):1962–1969. 10.1021/jacs.5b12482

</cite> [<a href="https://doi.org/10.1021/jacs.5b12482" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26807980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=Ribosomal%20synthesis%20of%20peptides%20with%20multiple%20beta%E2%80%90amino%20acids&amp;author=T%20Fujino&amp;author=Y%20Goto&amp;author=H%20Suga&amp;author=H%20Murakami&amp;volume=138&amp;issue=6&amp;publication_year=2016&amp;pages=1962-1969&amp;pmid=26807980&amp;doi=10.1021/jacs.5b12482&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0012">
<cite>
Gao Y, Nihira NT, Bu X, Chu C, Zhang J, Kolodziejczyk A, et al. Acetylation‐dependent regulation of PD‐L1 nuclear translocation dictates the efficacy of anti‐PD‐1 immunotherapy. Nat Cell Biol. 2020;22(9):1064–1075. 10.1038/s41556-020-0562-4

</cite> [<a href="https://doi.org/10.1038/s41556-020-0562-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7484128/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32839551/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Cell%20Biol&amp;title=Acetylation%E2%80%90dependent%20regulation%20of%20PD%E2%80%90L1%20nuclear%20translocation%20dictates%20the%20efficacy%20of%20anti%E2%80%90PD%E2%80%901%20immunotherapy&amp;author=Y%20Gao&amp;author=NT%20Nihira&amp;author=X%20Bu&amp;author=C%20Chu&amp;author=J%20Zhang&amp;volume=22&amp;issue=9&amp;publication_year=2020&amp;pages=1064-1075&amp;pmid=32839551&amp;doi=10.1038/s41556-020-0562-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0013">
<cite>
Goto Y, Murakami H, Suga H. Initiating translation with D‐amino acids. RNA. 2008;14(7):1390–1398. 10.1261/rna.1020708

</cite> [<a href="https://doi.org/10.1261/rna.1020708" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2441986/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18515548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RNA&amp;title=Initiating%20translation%20with%20D%E2%80%90amino%20acids&amp;author=Y%20Goto&amp;author=H%20Murakami&amp;author=H%20Suga&amp;volume=14&amp;issue=7&amp;publication_year=2008&amp;pages=1390-1398&amp;pmid=18515548&amp;doi=10.1261/rna.1020708&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0014">
<cite>
Gray JP, Uddin MN, Chaudhari R, Sutton MN, Yang H, Rask P, et al. Directed evolution of cyclic peptides for inhibition of autophagy. Chem Sci. 2021;12(10):3526–3543. 10.1039/d0sc03603j

</cite> [<a href="https://doi.org/10.1039/d0sc03603j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8179393/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34163626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem%20Sci&amp;title=Directed%20evolution%20of%20cyclic%20peptides%20for%20inhibition%20of%20autophagy&amp;author=JP%20Gray&amp;author=MN%20Uddin&amp;author=R%20Chaudhari&amp;author=MN%20Sutton&amp;author=H%20Yang&amp;volume=12&amp;issue=10&amp;publication_year=2021&amp;pages=3526-3543&amp;pmid=34163626&amp;doi=10.1039/d0sc03603j&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0015">
<cite>
Greenfield NJ. Using circular dichroism collected as a function of temperature to determine the thermodynamics of protein unfolding and binding interactions. Nat Protocol. 2006;1(6):2527–2535. 10.1038/nprot.2006.204
</cite> [<a href="https://doi.org/10.1038/nprot.2006.204" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2752288/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17406506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Protocol&amp;title=Using%20circular%20dichroism%20collected%20as%20a%20function%20of%20temperature%20to%20determine%20the%20thermodynamics%20of%20protein%20unfolding%20and%20binding%20interactions&amp;author=NJ%20Greenfield&amp;volume=1&amp;issue=6&amp;publication_year=2006&amp;pages=2527-2535&amp;pmid=17406506&amp;doi=10.1038/nprot.2006.204&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0016">
<cite>
Grindel BJ, Engel BJ, Hall CG, Kelderhouse LE, Lucci A, Zacharias NM, et al. Mammalian expression and in situ biotinylation of extracellular protein targets for directed evolution. ACS Omega. 2020;5(39):25440–25455. 10.1021/acsomega.0c03990

</cite> [<a href="https://doi.org/10.1021/acsomega.0c03990" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7542843/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33043224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Omega&amp;title=Mammalian%20expression%20and%20in%20situ%20biotinylation%20of%20extracellular%20protein%20targets%20for%20directed%20evolution&amp;author=BJ%20Grindel&amp;author=BJ%20Engel&amp;author=CG%20Hall&amp;author=LE%20Kelderhouse&amp;author=A%20Lucci&amp;volume=5&amp;issue=39&amp;publication_year=2020&amp;pages=25440-25455&amp;pmid=33043224&amp;doi=10.1021/acsomega.0c03990&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0017">
<cite>
Grindel BJ, Engel BJ, Ong JN, Srinivasamani A, Liang X, Zacharias NM, et al. Directed evolution of PD‐L1‐targeted affibodies by mRNA display. ACS Chem Biol. 2022;17(6):1543–1555. 10.1021/acschembio.2c00218

</cite> [<a href="https://doi.org/10.1021/acschembio.2c00218" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10691555/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35611948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Chem%20Biol&amp;title=Directed%20evolution%20of%20PD%E2%80%90L1%E2%80%90targeted%20affibodies%20by%20mRNA%20display&amp;author=BJ%20Grindel&amp;author=BJ%20Engel&amp;author=JN%20Ong&amp;author=A%20Srinivasamani&amp;author=X%20Liang&amp;volume=17&amp;issue=6&amp;publication_year=2022&amp;pages=1543-1555&amp;pmid=35611948&amp;doi=10.1021/acschembio.2c00218&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0018">
<cite>
Howell SM, Fiacco SV, Takahashi TT, Jalali‐Yazdi F, Millward SW, Hu B, et al. Serum stable natural peptides designed by mRNA display. Sci Rep. 2014;4:6008. 10.1038/srep06008

</cite> [<a href="https://doi.org/10.1038/srep06008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4168267/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25234472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Serum%20stable%20natural%20peptides%20designed%20by%20mRNA%20display&amp;author=SM%20Howell&amp;author=SV%20Fiacco&amp;author=TT%20Takahashi&amp;author=F%20Jalali%E2%80%90Yazdi&amp;author=SW%20Millward&amp;volume=4&amp;publication_year=2014&amp;pages=6008&amp;pmid=25234472&amp;doi=10.1038/srep06008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0019">
<cite>
Hu K, Kuan H, Hanyu M, Masayuki H, Xie L, Zhang Y, et al. Developing native peptide‐based radiotracers for PD‐L1 PET imaging and improving imaging contrast by pegylation. Chem Commun. 2019;55(29):4162–4165. 10.1039/c9cc00445a
</cite> [<a href="https://doi.org/10.1039/c9cc00445a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30810133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem%20Commun&amp;title=Developing%20native%20peptide%E2%80%90based%20radiotracers%20for%20PD%E2%80%90L1%20PET%20imaging%20and%20improving%20imaging%20contrast%20by%20pegylation&amp;author=K%20Hu&amp;author=H%20Kuan&amp;author=M%20Hanyu&amp;author=H%20Masayuki&amp;author=L%20Xie&amp;volume=55&amp;issue=29&amp;publication_year=2019&amp;pages=4162-4165&amp;pmid=30810133&amp;doi=10.1039/c9cc00445a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0020">
<cite>
Kamalinia G, Engel BJ, Srinivasamani A, Grindel BJ, Ong JN, Curran MA, et al. mRNA display discovery of a novel programmed death ligand 1 (PD‐L1) binding peptide (a peptide ligand for PD‐L1). ACS Chem Biol. 2020;15(6):1630–1641. 10.1021/acschembio.0c00264

</cite> [<a href="https://doi.org/10.1021/acschembio.0c00264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7416570/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32352272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Chem%20Biol&amp;title=mRNA%20display%20discovery%20of%20a%20novel%20programmed%20death%20ligand%201%20(PD%E2%80%90L1)%20binding%20peptide%20(a%20peptide%20ligand%20for%20PD%E2%80%90L1)&amp;author=G%20Kamalinia&amp;author=BJ%20Engel&amp;author=A%20Srinivasamani&amp;author=BJ%20Grindel&amp;author=JN%20Ong&amp;volume=15&amp;issue=6&amp;publication_year=2020&amp;pages=1630-1641&amp;pmid=32352272&amp;doi=10.1021/acschembio.0c00264&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0021">
<cite>
Kawakami T, Murakami H, Suga H. Ribosomal synthesis of polypeptoids and peptoid‐peptide hybrids. J Am Chem Soc. 2008;130(50):16861–16863. 10.1021/ja806998v

</cite> [<a href="https://doi.org/10.1021/ja806998v" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19053417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=Ribosomal%20synthesis%20of%20polypeptoids%20and%20peptoid%E2%80%90peptide%20hybrids&amp;author=T%20Kawakami&amp;author=H%20Murakami&amp;author=H%20Suga&amp;volume=130&amp;issue=50&amp;publication_year=2008&amp;pages=16861-16863&amp;pmid=19053417&amp;doi=10.1021/ja806998v&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0022">
<cite>
Larson K, Russ A, Arif‐Tiwari H, Mahadevan D, Elliott A, Bhattacharyya A, et al. Pembrolizumab achieves a complete response in an NF‐1 mutated, PD‐L1 positive malignant peripheral nerve sheath tumor: a case report and review of the benchmarks. J Immunother. 2022;45(4):222–226. 10.1097/CJI.0000000000000410

</cite> [<a href="https://doi.org/10.1097/CJI.0000000000000410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35020691/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Immunother&amp;title=Pembrolizumab%20achieves%20a%20complete%20response%20in%20an%20NF%E2%80%901%20mutated,%20PD%E2%80%90L1%20positive%20malignant%20peripheral%20nerve%20sheath%20tumor:%20a%20case%20report%20and%20review%20of%20the%20benchmarks&amp;author=K%20Larson&amp;author=A%20Russ&amp;author=H%20Arif%E2%80%90Tiwari&amp;author=D%20Mahadevan&amp;author=A%20Elliott&amp;volume=45&amp;issue=4&amp;publication_year=2022&amp;pages=222-226&amp;pmid=35020691&amp;doi=10.1097/CJI.0000000000000410&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0023">
<cite>
Latchman Y, Wood CR, Chernova T, Chaudhary D, Borde M, Chernova I, et al. PD‐L2 is a second ligand for PD‐1 and inhibits T cell activation. Nat Immunol. 2001;2(3):261–268. 10.1038/85330

</cite> [<a href="https://doi.org/10.1038/85330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11224527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&amp;title=PD%E2%80%90L2%20is%20a%20second%20ligand%20for%20PD%E2%80%901%20and%20inhibits%20T%20cell%20activation&amp;author=Y%20Latchman&amp;author=CR%20Wood&amp;author=T%20Chernova&amp;author=D%20Chaudhary&amp;author=M%20Borde&amp;volume=2&amp;issue=3&amp;publication_year=2001&amp;pages=261-268&amp;pmid=11224527&amp;doi=10.1038/85330&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0024">
<cite>
Lazar‐Molnar E, Yan Q, Cao E, Ramagopal U, Nathenson SG, Almo SC. Crystal structure of the complex between programmed death‐1 (PD‐1) and its ligand PD‐L2. Proc Natl Acad Sci USA. 2008;105(30):10483–10488. 10.1073/pnas.0804453105

</cite> [<a href="https://doi.org/10.1073/pnas.0804453105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2492495/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18641123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Crystal%20structure%20of%20the%20complex%20between%20programmed%20death%E2%80%901%20(PD%E2%80%901)%20and%20its%20ligand%20PD%E2%80%90L2&amp;author=E%20Lazar%E2%80%90Molnar&amp;author=Q%20Yan&amp;author=E%20Cao&amp;author=U%20Ramagopal&amp;author=SG%20Nathenson&amp;volume=105&amp;issue=30&amp;publication_year=2008&amp;pages=10483-10488&amp;pmid=18641123&amp;doi=10.1073/pnas.0804453105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0025">
<cite>
Lesniak WG, Mease RC, Chatterjee S, Kumar D, Lisok A, Wharram B, et al. Development of [(18)F]FPy‐WL12 as a PD‐L1 specific PET imaging peptide. Mol Imaging. 2019;18:1536012119852189. 10.1177/1536012119852189

</cite> [<a href="https://doi.org/10.1177/1536012119852189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6563393/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31187691/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Imaging&amp;title=Development%20of%20%5B(18)F%5DFPy%E2%80%90WL12%20as%20a%20PD%E2%80%90L1%20specific%20PET%20imaging%20peptide&amp;author=WG%20Lesniak&amp;author=RC%20Mease&amp;author=S%20Chatterjee&amp;author=D%20Kumar&amp;author=A%20Lisok&amp;volume=18&amp;publication_year=2019&amp;pages=1536012119852189&amp;pmid=31187691&amp;doi=10.1177/1536012119852189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0026">
<cite>
Li C, Zhang N, Zhou J, Ding C, Jin Y, Cui X, et al. Peptide blocking of PD‐1/PD‐L1 interaction for cancer immunotherapy. Cancer Immunol Res. 2018;6(2):178–188. 10.1158/2326-6066.CIR-17-0035

</cite> [<a href="https://doi.org/10.1158/2326-6066.CIR-17-0035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29217732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Immunol%20Res&amp;title=Peptide%20blocking%20of%20PD%E2%80%901/PD%E2%80%90L1%20interaction%20for%20cancer%20immunotherapy&amp;author=C%20Li&amp;author=N%20Zhang&amp;author=J%20Zhou&amp;author=C%20Ding&amp;author=Y%20Jin&amp;volume=6&amp;issue=2&amp;publication_year=2018&amp;pages=178-188&amp;pmid=29217732&amp;doi=10.1158/2326-6066.CIR-17-0035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0027">
<cite>
Li Z, Zhu N, Liu Y, Yu Y, Wang T, Zou C, et al. A disproportionality analysis of real‐world events from the FDA adverse event reporting system (FAERS) for Atezolizumab. BMC Pharmacol Toxicol. 2025;26(1):51. 10.1186/s40360-025-00879-2

</cite> [<a href="https://doi.org/10.1186/s40360-025-00879-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11881481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40038564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Pharmacol%20Toxicol&amp;title=A%20disproportionality%20analysis%20of%20real%E2%80%90world%20events%20from%20the%20FDA%20adverse%20event%20reporting%20system%20(FAERS)%20for%20Atezolizumab&amp;author=Z%20Li&amp;author=N%20Zhu&amp;author=Y%20Liu&amp;author=Y%20Yu&amp;author=T%20Wang&amp;volume=26&amp;issue=1&amp;publication_year=2025&amp;pages=51&amp;pmid=40038564&amp;doi=10.1186/s40360-025-00879-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0028">
<cite>
Lin DY, Tanaka Y, Iwasaki M, Gittis AG, Su HP, Mikami B, et al. The PD‐1/PD‐L1 complex resembles the antigen‐binding Fv domains of antibodies and T cell receptors. Proc Natl Acad Sci USA. 2008;105(8):3011–3016. 10.1073/pnas.0712278105 From NLM Medline.
</cite> [<a href="https://doi.org/10.1073/pnas.0712278105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2268576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18287011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=The%20PD%E2%80%901/PD%E2%80%90L1%20complex%20resembles%20the%20antigen%E2%80%90binding%20Fv%20domains%20of%20antibodies%20and%20T%20cell%20receptors&amp;author=DY%20Lin&amp;author=Y%20Tanaka&amp;author=M%20Iwasaki&amp;author=AG%20Gittis&amp;author=HP%20Su&amp;volume=105&amp;issue=8&amp;publication_year=2008&amp;pages=3011-3016&amp;pmid=18287011&amp;doi=10.1073/pnas.0712278105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0029">
<cite>
Liu G, Zeng H, Mueller J, Carter B, Wang Z, Schilz J, et al. Antibody complementarity determining region design using high‐capacity machine learning. Bioinformatics. 2020;36(7):2126–2133. 10.1093/bioinformatics/btz895

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz895" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7141872/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31778140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Antibody%20complementarity%20determining%20region%20design%20using%20high%E2%80%90capacity%20machine%20learning&amp;author=G%20Liu&amp;author=H%20Zeng&amp;author=J%20Mueller&amp;author=B%20Carter&amp;author=Z%20Wang&amp;volume=36&amp;issue=7&amp;publication_year=2020&amp;pages=2126-2133&amp;pmid=31778140&amp;doi=10.1093/bioinformatics/btz895&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0030">
<cite>
Maini R, Kimura H, Takatsuji R, Katoh T, Goto Y, Suga H. Ribosomal formation of thioamide bonds in polypeptide synthesis. J Am Chem Soc. 2019;141(51):20004–20008. 10.1021/jacs.9b11097

</cite> [<a href="https://doi.org/10.1021/jacs.9b11097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31815469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=Ribosomal%20formation%20of%20thioamide%20bonds%20in%20polypeptide%20synthesis&amp;author=R%20Maini&amp;author=H%20Kimura&amp;author=R%20Takatsuji&amp;author=T%20Katoh&amp;author=Y%20Goto&amp;volume=141&amp;issue=51&amp;publication_year=2019&amp;pages=20004-20008&amp;pmid=31815469&amp;doi=10.1021/jacs.9b11097&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0031">
<cite>
Makowski EK, Kinnunen PC, Huang J, Wu L, Smith MD, Wang T, et al. Co‐optimization of therapeutic antibody affinity and specificity using machine learning models that generalize to novel mutational space. Nat Commun. 2022;13(1):3788. 10.1038/s41467-022-31457-3

</cite> [<a href="https://doi.org/10.1038/s41467-022-31457-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9249733/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35778381/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Co%E2%80%90optimization%20of%20therapeutic%20antibody%20affinity%20and%20specificity%20using%20machine%20learning%20models%20that%20generalize%20to%20novel%20mutational%20space&amp;author=EK%20Makowski&amp;author=PC%20Kinnunen&amp;author=J%20Huang&amp;author=L%20Wu&amp;author=MD%20Smith&amp;volume=13&amp;issue=1&amp;publication_year=2022&amp;pages=3788&amp;pmid=35778381&amp;doi=10.1038/s41467-022-31457-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0032">
<cite>
McGregor DP. Discovering and improving novel peptide therapeutics. Curr Opin Pharmacol. 2008;8(5):616–619. 10.1016/j.coph.2008.06.002

</cite> [<a href="https://doi.org/10.1016/j.coph.2008.06.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18602024/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Pharmacol&amp;title=Discovering%20and%20improving%20novel%20peptide%20therapeutics&amp;author=DP%20McGregor&amp;volume=8&amp;issue=5&amp;publication_year=2008&amp;pages=616-619&amp;pmid=18602024&amp;doi=10.1016/j.coph.2008.06.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0033">
<cite>
Melisi D, Oh DY, Hollebecque A, Calvo E, Varghese A, Borazanci E, et al. Safety and activity of the TGFβ receptor I kinase inhibitor galunisertib plus the anti‐PD‐L1 antibody durvalumab in metastatic pancreatic cancer. J Immunother Cancer. 2021;9(3):e002068. 10.1136/jitc-2020-002068

</cite> [<a href="https://doi.org/10.1136/jitc-2020-002068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7944986/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33688022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Immunother%20Cancer&amp;title=Safety%20and%20activity%20of%20the%20TGF%CE%B2%20receptor%20I%20kinase%20inhibitor%20galunisertib%20plus%20the%20anti%E2%80%90PD%E2%80%90L1%20antibody%20durvalumab%20in%20metastatic%20pancreatic%20cancer&amp;author=D%20Melisi&amp;author=DY%20Oh&amp;author=A%20Hollebecque&amp;author=E%20Calvo&amp;author=A%20Varghese&amp;volume=9&amp;issue=3&amp;publication_year=2021&amp;pages=e002068&amp;pmid=33688022&amp;doi=10.1136/jitc-2020-002068&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0034">
<cite>
Menges F. Spectragryph—optical spectroscopy software, Version 1.2.16.1. 2022.  . <a href="http://www.effemm2.de/spectragryph" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.effemm2.de/spectragryph</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Spectragryph%E2%80%94optical%20spectroscopy%20software&amp;author=F%20Menges&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0035">
<cite>
Nichols AL, Noridomi K, Hughes CR, Jalali‐Yazdi F, Eaton JB, Lai LH, et al. alpha1‐FANGs: protein ligands selective for the alpha‐bungarotoxin site of the alpha1‐nicotinic acetylcholine receptor. ACS Chem Biol. 2018;13(9):2568–2576. 10.1021/acschembio.8b00513

</cite> [<a href="https://doi.org/10.1021/acschembio.8b00513" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8763392/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30059207/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Chem%20Biol&amp;title=alpha1%E2%80%90FANGs:%20protein%20ligands%20selective%20for%20the%20alpha%E2%80%90bungarotoxin%20site%20of%20the%20alpha1%E2%80%90nicotinic%20acetylcholine%20receptor&amp;author=AL%20Nichols&amp;author=K%20Noridomi&amp;author=CR%20Hughes&amp;author=F%20Jalali%E2%80%90Yazdi&amp;author=JB%20Eaton&amp;volume=13&amp;issue=9&amp;publication_year=2018&amp;pages=2568-2576&amp;pmid=30059207&amp;doi=10.1021/acschembio.8b00513&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0036">
<cite>
Olson CA, Nie J, Diep J, et al. Single‐round, multiplexed antibody mimetic design through mRNA display. Angew Chem Int Ed Engl. 2012;51(50):12449–12453. 10.1002/anie.201207005

</cite> [<a href="https://doi.org/10.1002/anie.201207005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3832244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23125174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew%20Chem%20Int%20Ed%20Engl&amp;title=Single%E2%80%90round,%20multiplexed%20antibody%20mimetic%20design%20through%20mRNA%20display&amp;author=CA%20Olson&amp;author=J%20Nie&amp;author=J%20Diep&amp;volume=51&amp;issue=50&amp;publication_year=2012&amp;pages=12449-12453&amp;pmid=23125174&amp;doi=10.1002/anie.201207005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0037">
<cite>
Olson CA, Wu NC, Sun R. A comprehensive biophysical description of pairwise epistasis throughout an entire protein domain. Curr Biol. 2014;24(22):2643–2651. 10.1016/j.cub.2014.09.072

</cite> [<a href="https://doi.org/10.1016/j.cub.2014.09.072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4254498/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25455030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&amp;title=A%20comprehensive%20biophysical%20description%20of%20pairwise%20epistasis%20throughout%20an%20entire%20protein%20domain&amp;author=CA%20Olson&amp;author=NC%20Wu&amp;author=R%20Sun&amp;volume=24&amp;issue=22&amp;publication_year=2014&amp;pages=2643-2651&amp;pmid=25455030&amp;doi=10.1016/j.cub.2014.09.072&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0038">
<cite>
Parkinson J, Hard R, Wang W. The RESP AI model accelerates the identification of tight‐binding antibodies. Nat Commun. 2023;14(1):454. 10.1038/s41467-023-36028-8

</cite> [<a href="https://doi.org/10.1038/s41467-023-36028-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9884274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36709319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=The%20RESP%20AI%20model%20accelerates%20the%20identification%20of%20tight%E2%80%90binding%20antibodies&amp;author=J%20Parkinson&amp;author=R%20Hard&amp;author=W%20Wang&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=454&amp;pmid=36709319&amp;doi=10.1038/s41467-023-36028-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0039">
<cite>
Passioura T, Suga H. A RaPID way to discover nonstandard macrocyclic peptide modulators of drug targets. Chem Commun. 2017;53(12):1931–1940. 10.1039/c6cc06951g
</cite> [<a href="https://doi.org/10.1039/c6cc06951g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28091672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem%20Commun&amp;title=A%20RaPID%20way%20to%20discover%20nonstandard%20macrocyclic%20peptide%20modulators%20of%20drug%20targets&amp;author=T%20Passioura&amp;author=H%20Suga&amp;volume=53&amp;issue=12&amp;publication_year=2017&amp;pages=1931-1940&amp;pmid=28091672&amp;doi=10.1039/c6cc06951g&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0040">
<cite>
Rhoden JJ, Wittrup KD. Dose dependence of intratumoral perivascular distribution of monoclonal antibodies. J Pharm Sci. 2012;101(2):860–867. 10.1002/jps.22801

</cite> [<a href="https://doi.org/10.1002/jps.22801" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3971539/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22057714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Sci&amp;title=Dose%20dependence%20of%20intratumoral%20perivascular%20distribution%20of%20monoclonal%20antibodies&amp;author=JJ%20Rhoden&amp;author=KD%20Wittrup&amp;volume=101&amp;issue=2&amp;publication_year=2012&amp;pages=860-867&amp;pmid=22057714&amp;doi=10.1002/jps.22801&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0041">
<cite>
Saka K, Kakuzaki T, Metsugi S, Kashiwagi D, Yoshida K, Wada M, et al. Antibody design using LSTM based deep generative model from phage display library for affinity maturation. Sci Rep. 2021;11(1):5852. 10.1038/s41598-021-85274-7

</cite> [<a href="https://doi.org/10.1038/s41598-021-85274-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7955064/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33712669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Antibody%20design%20using%20LSTM%20based%20deep%20generative%20model%20from%20phage%20display%20library%20for%20affinity%20maturation&amp;author=K%20Saka&amp;author=T%20Kakuzaki&amp;author=S%20Metsugi&amp;author=D%20Kashiwagi&amp;author=K%20Yoshida&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pages=5852&amp;pmid=33712669&amp;doi=10.1038/s41598-021-85274-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0042">
<cite>
Saxena K, Herbrich SM, Pemmaraju N, Kadia TM, DiNardo CD, Borthakur G, et al. A phase 1b/2 study of azacitidine with PD‐L1 antibody avelumab in relapsed/refractory acute myeloid leukemia. Cancer. 2021;127(20):3761–3771. 10.1002/cncr.33690

</cite> [<a href="https://doi.org/10.1002/cncr.33690" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34171128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=A%20phase%201b/2%20study%20of%20azacitidine%20with%20PD%E2%80%90L1%20antibody%20avelumab%20in%20relapsed/refractory%20acute%20myeloid%20leukemia&amp;author=K%20Saxena&amp;author=SM%20Herbrich&amp;author=N%20Pemmaraju&amp;author=TM%20Kadia&amp;author=CD%20DiNardo&amp;volume=127&amp;issue=20&amp;publication_year=2021&amp;pages=3761-3771&amp;pmid=34171128&amp;doi=10.1002/cncr.33690&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0043">
<cite>
Schmidt B, Hildebrandt A. Deep learning in next‐generation sequencing. Drug Discov Today. 2021;26(1):173–180. 10.1016/j.drudis.2020.10.002

</cite> [<a href="https://doi.org/10.1016/j.drudis.2020.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7550123/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33059075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Discov%20Today&amp;title=Deep%20learning%20in%20next%E2%80%90generation%20sequencing&amp;author=B%20Schmidt&amp;author=A%20Hildebrandt&amp;volume=26&amp;issue=1&amp;publication_year=2021&amp;pages=173-180&amp;pmid=33059075&amp;doi=10.1016/j.drudis.2020.10.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0044">
<cite>
Scola PM. Discovery of BMS‐986238, a second‐generation macrocyclic peptide inhibitor of programmed death‐ligand 1 (PD‐L1). ACS Spring 2025, March 26th, 2025 . <a href="https://acs.digitellinc.com/live/34/session/547208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://acs.digitellinc.com/live/34/session/547208</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=ACS%20Spring%202025&amp;author=PM%20Scola&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0045">
<cite>
Seto T, Nosaki K, Shimokawa M, et al. Phase II study of atezolizumab with bevacizumab for non‐squamous non‐small cell lung cancer with high PD‐L1 expression (@Be study). J Immunother Cancer. 2022;10(2):e004025. 10.1136/jitc-2021-004025

</cite> [<a href="https://doi.org/10.1136/jitc-2021-004025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8808447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35105689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Immunother%20Cancer&amp;title=Phase%20II%20study%20of%20atezolizumab%20with%20bevacizumab%20for%20non%E2%80%90squamous%20non%E2%80%90small%20cell%20lung%20cancer%20with%20high%20PD%E2%80%90L1%20expression%20(@Be%20study)&amp;author=T%20Seto&amp;author=K%20Nosaki&amp;author=M%20Shimokawa&amp;volume=10&amp;issue=2&amp;publication_year=2022&amp;pages=e004025&amp;pmid=35105689&amp;doi=10.1136/jitc-2021-004025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0046">
<cite>
Subtelny AO, Hartman MC, Szostak JW. Ribosomal synthesis of N‐methyl peptides. J Am Chem Soc. 2008;130(19):6131–6136. 10.1021/ja710016v

</cite> [<a href="https://doi.org/10.1021/ja710016v" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2728122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18402453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=Ribosomal%20synthesis%20of%20N%E2%80%90methyl%20peptides&amp;author=AO%20Subtelny&amp;author=MC%20Hartman&amp;author=JW%20Szostak&amp;volume=130&amp;issue=19&amp;publication_year=2008&amp;pages=6131-6136&amp;pmid=18402453&amp;doi=10.1021/ja710016v&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0047">
<cite>
Tan S, Liu K, Chai Y, Zhang CW, Gao S, Gao GF, et al. Distinct PD‐L1 binding characteristics of therapeutic monoclonal antibody durvalumab. Protein Cell. 2018;9(1):135–139. 10.1007/s13238-017-0412-8

</cite> [<a href="https://doi.org/10.1007/s13238-017-0412-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5777972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28488247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Cell&amp;title=Distinct%20PD%E2%80%90L1%20binding%20characteristics%20of%20therapeutic%20monoclonal%20antibody%20durvalumab&amp;author=S%20Tan&amp;author=K%20Liu&amp;author=Y%20Chai&amp;author=CW%20Zhang&amp;author=S%20Gao&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pages=135-139&amp;pmid=28488247&amp;doi=10.1007/s13238-017-0412-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0048">
<cite>
Thurber GM, Dane Wittrup K. A mechanistic compartmental model for total antibody uptake in tumors. J Theor Biol. 2012;314:57–68. 10.1016/j.jtbi.2012.08.034

</cite> [<a href="https://doi.org/10.1016/j.jtbi.2012.08.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3729444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22974563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Theor%20Biol&amp;title=A%20mechanistic%20compartmental%20model%20for%20total%20antibody%20uptake%20in%20tumors&amp;author=GM%20Thurber&amp;author=K%20Dane%20Wittrup&amp;volume=314&amp;publication_year=2012&amp;pages=57-68&amp;pmid=22974563&amp;doi=10.1016/j.jtbi.2012.08.034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0049">
<cite>
Vinogradov AA, Chang JS, Onaka H, Goto Y, Suga H. Accurate models of substrate preferences of post‐translational modification enzymes from a combination of mRNA display and deep learning. ACS Cent Sci. 2022;8(6):814–824. 10.1021/acscentsci.2c00223

</cite> [<a href="https://doi.org/10.1021/acscentsci.2c00223" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9228559/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35756369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Cent%20Sci&amp;title=Accurate%20models%20of%20substrate%20preferences%20of%20post%E2%80%90translational%20modification%20enzymes%20from%20a%20combination%20of%20mRNA%20display%20and%20deep%20learning&amp;author=AA%20Vinogradov&amp;author=JS%20Chang&amp;author=H%20Onaka&amp;author=Y%20Goto&amp;author=H%20Suga&amp;volume=8&amp;issue=6&amp;publication_year=2022&amp;pages=814-824&amp;pmid=35756369&amp;doi=10.1021/acscentsci.2c00223&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0050">
<cite>
Zhang F, Qi X, Wang X, Wei D, Wu J, Feng L, et al. Structural basis of the therapeutic anti‐PD‐L1 antibody atezolizumab. Oncotarget. 2017;8(52):90215–90224. 10.18632/oncotarget.21652

</cite> [<a href="https://doi.org/10.18632/oncotarget.21652" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5685743/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29163822/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Structural%20basis%20of%20the%20therapeutic%20anti%E2%80%90PD%E2%80%90L1%20antibody%20atezolizumab&amp;author=F%20Zhang&amp;author=X%20Qi&amp;author=X%20Wang&amp;author=D%20Wei&amp;author=J%20Wu&amp;volume=8&amp;issue=52&amp;publication_year=2017&amp;pages=90215-90224&amp;pmid=29163822&amp;doi=10.18632/oncotarget.21652&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0051">
<cite>
Zhou L, Chai F, He Y, Zhou Z, Guo S, Li P, et al. Homodimerized cytoplasmic domain of PD‐L1 regulates its complex glycosylation in living cells. Commun Biol. 2022;5(1):887. 10.1038/s42003-022-03845-4

</cite> [<a href="https://doi.org/10.1038/s42003-022-03845-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9427764/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36042378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Commun%20Biol&amp;title=Homodimerized%20cytoplasmic%20domain%20of%20PD%E2%80%90L1%20regulates%20its%20complex%20glycosylation%20in%20living%20cells&amp;author=L%20Zhou&amp;author=F%20Chai&amp;author=Y%20He&amp;author=Z%20Zhou&amp;author=S%20Guo&amp;volume=5&amp;issue=1&amp;publication_year=2022&amp;pages=887&amp;pmid=36042378&amp;doi=10.1038/s42003-022-03845-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70268-bib-0052">
<cite>
Zhou X, Jiang J, Yang X, Liu T, Ding J, Nimmagadda S, et al. First‐in‐humans evaluation of a PD‐L1‐binding peptide PET radiotracer in non‐small cell lung cancer patients. J Nucl Med. 2022;63(4):536–542. 10.2967/jnumed.121.262045

</cite> [<a href="https://doi.org/10.2967/jnumed.121.262045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8973283/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34326125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Nucl%20Med&amp;title=First%E2%80%90in%E2%80%90humans%20evaluation%20of%20a%20PD%E2%80%90L1%E2%80%90binding%20peptide%20PET%20radiotracer%20in%20non%E2%80%90small%20cell%20lung%20cancer%20patients&amp;author=X%20Zhou&amp;author=J%20Jiang&amp;author=X%20Yang&amp;author=T%20Liu&amp;author=J%20Ding&amp;volume=63&amp;issue=4&amp;publication_year=2022&amp;pages=536-542&amp;pmid=34326125&amp;doi=10.2967/jnumed.121.262045&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Data S1.</strong> Experimental details.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12376068/bin/PRO-34-e70268-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70268-s001.docx</a><sup> (3.2MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are available from the corresponding author upon reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Protein Science : A Publication of the Protein Society are provided here courtesy of <strong>The Protein Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/pro.70268"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/PRO-34-e70268.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12376068/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12376068/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12376068%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376068/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12376068/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12376068/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12376068/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12376068/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12376068/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="90zXatcHqXzaULEI33QO2ac1arZCsny7d961DH5cBNWNYkveWKF6yfZYMck6w0R7">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
