
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Spatial host-microbiome profiling demonstrates bacterial-associated host transcriptional alterations in pediatric ileal Crohn’s disease - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4852D8AEF24E305852D003BF570F7.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="microbio">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374449/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Microbiome">
<meta name="citation_title" content="Spatial host-microbiome profiling demonstrates bacterial-associated host transcriptional alterations in pediatric ileal Crohn’s disease">
<meta name="citation_author" content="Sooyoung Jang">
<meta name="citation_author_institution" content="Department of Biomedical Systems Informatics, Yonsei University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author" content="Eun Joo Lee">
<meta name="citation_author_institution" content="Division of Gastroenterology, Hepatology and Nutrition, Department of Pediatrics, Yonsei University College of Medicine, Severance Fecal Microbiota Transplantation Center, Severance Hospital, Seoul, Republic of Korea">
<meta name="citation_author" content="Sowon Park">
<meta name="citation_author_institution" content="Department of Pediatrics, University of Ulsan College of Medicine, Asan Medical Center Children’s Hospital, Seoul, Republic of Korea">
<meta name="citation_author" content="Hyeji Lim">
<meta name="citation_author_institution" content="Division of Gastroenterology, Hepatology and Nutrition, Department of Pediatrics, Yonsei University College of Medicine, Severance Fecal Microbiota Transplantation Center, Severance Hospital, Seoul, Republic of Korea">
<meta name="citation_author" content="Byungsoo Ahn">
<meta name="citation_author_institution" content="Department of Biomedical Systems Informatics, Yonsei University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author" content="Yoon Huh">
<meta name="citation_author_institution" content="Department of Biomedical Systems Informatics, Yonsei University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author" content="Hong Koh">
<meta name="citation_author_institution" content="Division of Gastroenterology, Hepatology and Nutrition, Department of Pediatrics, Yonsei University College of Medicine, Severance Fecal Microbiota Transplantation Center, Severance Hospital, Seoul, Republic of Korea">
<meta name="citation_author" content="Yu Rang Park">
<meta name="citation_author_institution" content="Department of Biomedical Systems Informatics, Yonsei University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_publication_date" content="2025 Aug 23">
<meta name="citation_volume" content="13">
<meta name="citation_firstpage" content="189">
<meta name="citation_doi" content="10.1186/s40168-025-02178-8">
<meta name="citation_pmid" content="40849632">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374449/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374449/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374449/pdf/40168_2025_Article_2178.pdf">
<meta name="description" content="Crohn’s disease (CD) is a chronic inflammatory bowel disease involving complex relationships between the gut microbiome and host immune system. However, the spatial relationships between tissue-resident bacteria and host cells in CD pathogenesis ...">
<meta name="og:title" content="Spatial host-microbiome profiling demonstrates bacterial-associated host transcriptional alterations in pediatric ileal Crohn’s disease">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Crohn’s disease (CD) is a chronic inflammatory bowel disease involving complex relationships between the gut microbiome and host immune system. However, the spatial relationships between tissue-resident bacteria and host cells in CD pathogenesis ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374449/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12374449">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s40168-025-02178-8"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/40168_2025_Article_2178.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374449%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12374449/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12374449/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374449/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-microbio.png" alt="Microbiome logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Microbiome" title="Link to Microbiome" shape="default" href="https://microbiomejournal.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Microbiome</button></div>. 2025 Aug 23;13:189. doi: <a href="https://doi.org/10.1186/s40168-025-02178-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s40168-025-02178-8</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Microbiome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Microbiome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Microbiome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Microbiome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Spatial host-microbiome profiling demonstrates bacterial-associated host transcriptional alterations in pediatric ileal Crohn’s disease</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jang%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Sooyoung Jang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sooyoung Jang</span></h3>
<div class="p">
<sup>1</sup>Department of Biomedical Systems Informatics, Yonsei University College of Medicine, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jang%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sooyoung Jang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20EJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Eun Joo Lee</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Eun Joo Lee</span></h3>
<div class="p">
<sup>2</sup>Division of Gastroenterology, Hepatology and Nutrition, Department of Pediatrics, Yonsei University College of Medicine, Severance Fecal Microbiota Transplantation Center, Severance Hospital, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20EJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Eun Joo Lee</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Sowon Park</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Sowon Park</span></h3>
<div class="p">
<sup>3</sup>Department of Pediatrics, University of Ulsan College of Medicine, Asan Medical Center Children’s Hospital, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sowon Park</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lim%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Hyeji Lim</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Hyeji Lim</span></h3>
<div class="p">
<sup>2</sup>Division of Gastroenterology, Hepatology and Nutrition, Department of Pediatrics, Yonsei University College of Medicine, Severance Fecal Microbiota Transplantation Center, Severance Hospital, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lim%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hyeji Lim</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahn%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Byungsoo Ahn</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Byungsoo Ahn</span></h3>
<div class="p">
<sup>1</sup>Department of Biomedical Systems Informatics, Yonsei University College of Medicine, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahn%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Byungsoo Ahn</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huh%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Yoon Huh</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yoon Huh</span></h3>
<div class="p">
<sup>1</sup>Department of Biomedical Systems Informatics, Yonsei University College of Medicine, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huh%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yoon Huh</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koh%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Hong Koh</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Hong Koh</span></h3>
<div class="p">
<sup>2</sup>Division of Gastroenterology, Hepatology and Nutrition, Department of Pediatrics, Yonsei University College of Medicine, Severance Fecal Microbiota Transplantation Center, Severance Hospital, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koh%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hong Koh</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20YR%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Yu Rang Park</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Yu Rang Park</span></h3>
<div class="p">
<sup>1</sup>Department of Biomedical Systems Informatics, Yonsei University College of Medicine, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20YR%22%5BAuthor%5D" class="usa-link"><span class="name western">Yu Rang Park</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Biomedical Systems Informatics, Yonsei University College of Medicine, Seoul, Republic of Korea </div>
<div id="Aff2">
<sup>2</sup>Division of Gastroenterology, Hepatology and Nutrition, Department of Pediatrics, Yonsei University College of Medicine, Severance Fecal Microbiota Transplantation Center, Severance Hospital, Seoul, Republic of Korea </div>
<div id="Aff3">
<sup>3</sup>Department of Pediatrics, University of Ulsan College of Medicine, Asan Medical Center Children’s Hospital, Seoul, Republic of Korea </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 18; Accepted 2025 Jun 20; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12374449  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40849632/" class="usa-link">40849632</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Crohn’s disease (CD) is a chronic inflammatory bowel disease involving complex relationships between the gut microbiome and host immune system. However, the spatial relationships between tissue-resident bacteria and host cells in CD pathogenesis remain poorly understood. We developed a spatial host-microbiome profiling approach to simultaneously detect host transcriptomics and bacterial species at high taxonomic resolution in pediatric ileal CD tissues.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">In this prospective case–control study, we analyzed 14 terminal ileal tissue samples from six pediatric patients with ileal CD and two controls. Spatial host-microbiome sequencing, combined spatial transcriptomics and in-situ polyadenylation, and bulk shotgun metagenome sequencing were performed. We developed a comprehensive bioinformatics pipeline to identify bacterial species and analyze host-microbiome interactions at cellular resolution, resulting in 13,876 analyzed cells. Our approach revealed increased bacterial abundance in CD tissues compared with controls. The extent of bacterial infiltration at diagnosis correlated with disease prognosis and severity of endoscopic findings. We identified 16 potentially beneficial and nine pathogenic microbiome members in ileal CD, including several newly discovered risk-modulating bacterial species. Cell-type-specific host gene expression analysis revealed transcriptome alterations related to bacterial defense mechanisms in the presence of various bacterial species.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">Our spatial host-microbiome profiling approach enables simultaneous species-level identification of bacteria and host transcriptomics. It reveals the intricate interactions between host cells and bacteria, providing cellular-level insights into CD pathogenesis. Our approach offers a powerful tool for investigating host-microbiome interactions in various microbiome-associated diseases to direct new strategies for microbiome-based therapeutics and prognostic markers.</p>
<figure class="fig xbox font-sm" id="MOESM4"><div class="media-box p text-center margin-x-neg-2 tablet:margin-x-0"><video controls="" preload="none" height="360" width="640" poster="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/video_poster.png"><source src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2b24/12374449/ac8a123835be/40168_2025_2178_MOESM4_ESM-pmcvs_normal.mp4"></source><div class="media-fallback margin-x-2 tablet:margin-x-0">
<div>Your browser is not supporting the HTML5 &lt;video&gt; element.</div>
<div>You may download that video as a file and play it with a player of you choice.</div>
<p><a href="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2b24/12374449/ac8a123835be/40168_2025_2178_MOESM4_ESM-pmcvs_normal.mp4" class="usa-link">Download video stream</a>.</p>
</div></video></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12374449/bin/40168_2025_2178_MOESM4_ESM.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (48.4MB, mp4) </sup>
</div></div>
<div class="p text-right font-secondary"><a href="figure/MOESM4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Video Abstract</p></figcaption></figure></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s40168-025-02178-8.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Gastrointestinal microbiome, Multi-omics, Host-microbiome interactions</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par8">The etiology of Crohn’s disease (CD) remains incompletely understood, although accumulating evidence has emphasized the potential importance of interactions between the gut microbiome and aberrant host immune responses in genetically predisposed patients [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Several studies have emphasized the importance of bacterial translocation in CD pathogenesis. Genome-wide association studies have identified CD-associated defects in microbe sensing, epithelial barrier function, microbicidal mechanisms, cytokine regulation, and adaptive immunity that potentially compromise host defense against bacterial translocation [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Furthermore, CD mouse models develop the disease in the presence of bacteria, but not in germ-free conditions [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Recently, Ha et al. reported that a subset of mucosa-associated gut bacteria translocate into intestinal tissue in CD, potentially contributing to disease progression [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>].</p>
<p id="Par9">Consequently, numerous studies have attempted to characterize the gut microbiome composition in patients with CD [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. They suggested that the gut microbiome in CD is characterized by an increase in pathogenic bacteria such as <em>Cutibacterium acnes</em> and <em>Haemophilus parainfluenzae</em>, and a decrease in beneficial bacteria like <em>Faecalibacterium prausnitzii</em> [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. Understanding these microbial alterations is crucial for developing microbiome-based therapeutics to reduce disease incidence and severity [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. However, previous bulk metagenome studies on CD tissue microbiome have been limited by the use of homogenized tissue samples with fecal remnant, hindering the identification of spatial relationships between the bacteria and specific host cell types [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. This limitation underscores the need for more advanced techniques to reveal the precise localization and interactions of microbes within the host intestinal tissues for understanding CD pathogenesis.</p>
<p id="Par10">Recent advancements in spatial transcriptomics have revolutionized our understanding of cellular heterogeneity and spatial organization in complex biological systems [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. However, previous methods relied on poly(A) tail-targeting approaches, limiting their application to eukaryotic mRNA, such as those of humans and mice [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Identifying non-host RNA, particularly bacterial RNA, which lacks poly(A) tails became challenging [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. This hindered a comprehensive understanding of host-microbiome interactions and their potential roles in the pathogenesis of microbiome-associated diseases, such as CD. Nevertheless, Galeano et al. recently demonstrated that bacteria within tumors may play a crucial role in cancer metastasis using a spatial transcriptomics method targeting bacterial 16S rRNA [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. However, this approach has limited taxonomic resolution when profiling the microbiome, restricting bacterial identification to the genus level, thereby hindering species-level discrimination [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>–<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>].</p>
<p id="Par11">To overcome these limitations and advance our understanding of CD pathogenesis, we developed a spatial host-microbiome profiling approach that allows for species-level identification of bacteria while simultaneously capturing host transcriptomics. Using this method, we examined bacterial localization patterns in pediatric CD tissues and their association with host transcriptome alterations at cellular resolution. Additionally, we identified and characterized microbiomes potentially associated with beneficial or pathogenic effects in pediatric CD. This approach provides insights into CD pathophysiology at the cellular level and offers potential applications for understanding host-microbiome interactions in various microbiome-associated diseases.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Participant and ethics</h3>
<p id="Par12">This prospective study included patients with CD from the Sinchon Severance Hospital, in Seoul. Ethical approval was obtained from the Institutional Review Board of Severance Hospital, Yonsei University College of Medicine (IRB number: 4–2022-1127). The study was conducted in adherence to the STROBE guidelines for observational studies (Additional file 1).</p>
<p id="Par13">We recruited pediatric patients aged 7–18 years between November 01, 2022, and April 30, 2024. For the CD group, we included patients newly diagnosed with ileal CD. The control group consisted of patients with irritable bowel syndrome who underwent endoscopy but showed no evidence of inflammation on endoscopic examination, histopathological analysis of biopsy specimens, or blood tests (Supplementary Table S1). Both the patients and their legal guardians provided informed consent for participation in the study. Clinical relapse of pediatric CD was defined as a Pediatric Crohn’s Disease Activity Index score ≥ 30, indicating worsening of the disease [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>].</p></section><section id="Sec4"><h3 class="pmc_sec_title">Sample collection and storage</h3>
<p id="Par14">Tissue samples were collected from the terminal ileum during endoscopic examination. For patients with CD, both inflamed and non-inflamed tissues were obtained, while control group samples consisted of only non-inflamed tissue. Immediately after collection, these tissue samples were embedded in optimum cutting temperature compound (SciGen Scientific, Gardena, CA, USA), frozen, and stored at − 80 °C. The stored samples were subsequently processed for spatial total RNA sequencing and bulk shotgun metagenome sequencing.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Spatial total RNA sequencing</h3>
<p id="Par15">Tissue block quality was evaluated by measuring the RNA Integrity Number (RIN) using an Agilent 4200 TapeStation system (Agilent Technologies, Santa Clara, CA, USA). Samples with RIN ≥ 4 were included. Due to the small size of the tissues, multiple sections were placed on each sample block for analysis. In the CD group, both inflamed and non-inflamed tissue sections from the same patient were mounted side by side, while only non-inflamed tissue sections were mounted for the control group.</p>
<p id="Par16">Following the approach by McKellar et al., we conducted in-situ polyadenylation [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. This included equilibration with wash buffer containing Protector RNase Inhibitor (Roche, catalog no. 3335402001), followed by incubation with an enzyme mix containing yeast poly(A) polymerase (Thermo Scientific, catalog no. 74225Z25KU). After the in-situ polyadenylation step, we prepared final sequencing libraries using standard Visium library preparation protocol (<a href="https://www.ncbi.nlm.nih.gov/nuccore/CG000239" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CG000239</a>, Rev F, 10 × Genomics). The prepared libraries were sequenced on a NovaSeq 6000 system (Illumina, San Diego, CA, USA), using the NovaSeq 6000 S1 Reagent Kit v1.5 (200 cycles, 20,028,318, Illumina), to a depth of approximately 120 M reads per sample.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Bulk shotgun metagenome sequencing</h3>
<p id="Par17">Genomic DNA quality and quantity were assessed via fluorometry (Qubit, Invitrogen) and gel electrophoresis. Samples with a DNA Integrity Number ≥ 6, as measured by an Agilent 4200 TapeStation, were used. The library was prepared using the Illumina TruSeq Nano DNA Library Prep Kit. For each sample, 100 ng of DNA was fragmented to 350 bp using a Qsonica 800 R2, followed by Illumina adapter ligation and PCR amplification. Libraries sized 500–600 bp were selected and quantified using TapeStation 4200 (Agilent Technologies) and KAPA Library Quantification Kit (Kapa Biosystems). Sequencing was performed on an Illumina NovaSeq 6000 platform using 150 bp paired-end reads.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Host transcriptome analysis</h3>
<p id="Par18">Host transcriptomics data were preprocessed using Space Ranger v1.3.1 (10 × Genomics). Downstream analyses with the output count matrices were performed using Scanpy v1.9.3 [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. As the Visium method captures multiple cells within each spot, we employed Cell2location, a computational tool that maps single-cell RNA sequencing data onto spatial transcriptomics data [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. For this process, we utilized previous single-cell RNA sequencing data from pediatric patients with CD [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. Cell2location integrates this single-cell data with our spatial transcriptomics data, allowing us to identify areas enriched with specific cell types across the analyzed tissue samples. Low-quality spots with &gt; 40% mitochondrial reads were excluded from the analysis [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>].</p></section><section id="Sec8"><h3 class="pmc_sec_title">Spatial microbiome profiling</h3>
<p id="Par19">We used Bowtie2 v2.5.1 to align the spatial transcriptomics read to the GRCh38 human reference genome for human read removal [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. UMI-tools v1.1.1 was used to separate the fastq files by cell barcode for independent analysis per spots [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Bacterial reads were identified using Kraken2 v2.1.1 with the publicly available PlusPF database (version 2022/09/08, <a href="https://benlangmead.github.io/aws-indexes/k2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://benlangmead.github.io/aws-indexes/k2</a>) [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. It contains RefSeq sequences for archaea, bacteria, viruses, plasmids, protozoa, fungi, and humans, as well as UniVec_Core sequences. We calculated counts per million (CPM) mapped reads by dividing bacterial read counts by total sequencing reads for normalization.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Microbiome profiling in bulk shotgun metagenome data</h3>
<p id="Par20">In the bulk shotgun metagenome data, we employed a process similar to our spatial microbiome detection method. Bowtie2 aligned reads to the GRCh38 human reference genome for human read removal [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. Subsequently, we identified bacterial reads using Kraken2 with the same PlusPF database used in our spatial profiling analysis [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>].</p></section><section id="Sec10"><h3 class="pmc_sec_title">Spatial microbiome decontamination process</h3>
<p id="Par21">We developed a stepwise decontamination process to reduce false positive species in our spatial microbiome profiling data. Initially, we excluded species with low spatial microbiome sequencing read counts across all samples, specifically those with &lt; 50 total reads across all spatial microbiome sequencing samples. We then compared the profiling results with the bulk metagenome sequencing data from the same tissue samples. Species with &lt; 50 reads from bulk sequencing or with &gt; 1.5 ratio of spatial and bulk sequencing read percentages were removed from our spatial microbiome dataset.</p>
<p id="Par22">To mitigate potential false positives arising from host read contamination, we evaluated the results of single and double human read removal processes [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. The single human read removal process used Bowtie2, while the double human read removal process used both Bowtie2 and BWA v0.7.17; all aligned with the GRCh38 human reference genome [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. This process led to two observations. First, certain species exhibited a drastic reduction in read counts following the double human read removal, indicating that they might be false positives; second, the read counts from genuine gut-residing species also decreased moderately, suggesting that certain true signals were affected (Supplementary Fig. S1). We classified the species as potential false positives if their read counts after double removal dropped to &lt; 2% of those observed after the single removal.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Comparative analysis of bacterial infiltration</h3>
<p id="Par23">Average number of bacterial reads, in CPM, per spot was compared across the control, CD non-inflamed, and CD inflamed tissues. Number of cells with bacterial infiltration was assessed for each cell type. For each patient, we compared the average number of bacterial reads per spot with their time to relapse and the Rutgeerts score of ileal endoscopic findings [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>].</p></section><section id="Sec12"><h3 class="pmc_sec_title">Differential gene expression and gene set enrichment analysis</h3>
<p id="Par24">Differentially expressed genes were identified based on the presence of specific bacterial species. This analysis was conducted separately for each detected bacteria, and we assessed the consistency of gene expression changes across different bacterial species. Each cell type identified by Cell2location was analyzed, enabling cell type-specific characterization of bacterial-associated transcriptional changes. Gene set enrichment analysis was performed using the BioPlanet 2019 database to identify biological pathways associated with genes consistently expressed differentially in the presence of bacteria [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Additionally, we conducted an analysis of tissue-specific gene expression between cells exposed to beneficial and pathogenic microbiomes.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Quantification of microbial effects on cell viability</h3>
<p id="Par25">We investigated the impact of specific bacterial species on cell viability using the percentage of mitochondrial reads, a widely used marker of cell viability in single-cell RNA sequencing and spatial transcriptomics analyses [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Cells with ≤ 10% mitochondrial reads were defined as viable, while those with &gt; 10% were defined as damaged. We calculated the relative risk (RR) of reduced cell viability following exposure to specific bacterial species to quantify the effect of bacterial exposure on cellular integrity. This analysis was performed exclusively on samples from participants with CD. Multiple testing correction was performed using the Benjamini–Hochberg method to control the false discovery rate at 0.05. The RR was calculated as follows:</p>
<table class="disp-formula p" id="Equa"><tr><td class="formula"><math id="d33e454" display="block"><mrow><mi>R</mi><mi>R</mi><mo>=</mo><mfrac><mrow><mi>N</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mspace width="0.277778em"></mspace><mi>o</mi><mi>f</mi><mspace width="0.277778em"></mspace><mi>d</mi><mi>a</mi><mi>m</mi><mi>a</mi><mi>g</mi><mi>e</mi><mi>d</mi><mspace width="0.277778em"></mspace><mi>c</mi><mi>e</mi><mi>l</mi><mi>l</mi><mi>s</mi><mspace width="0.277778em"></mspace><mi>e</mi><mi>x</mi><mi>p</mi><mi>o</mi><mi>s</mi><mi>e</mi><mi>d</mi><mspace width="0.277778em"></mspace><mi>t</mi><mi>o</mi><mspace width="0.277778em"></mspace><mi>b</mi><mi>a</mi><mi>c</mi><mi>t</mi><mi>e</mi><mi>r</mi><mi>i</mi><mi>a</mi><mo stretchy="false">/</mo><mi>T</mi><mi>o</mi><mi>t</mi><mi>a</mi><mi>l</mi><mspace width="0.277778em"></mspace><mi>n</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mspace width="0.277778em"></mspace><mi>o</mi><mi>f</mi><mspace width="0.277778em"></mspace><mi>c</mi><mi>e</mi><mi>l</mi><mi>l</mi><mi>s</mi><mspace width="0.277778em"></mspace><mi>e</mi><mi>x</mi><mi>p</mi><mi>o</mi><mi>s</mi><mi>e</mi><mi>d</mi><mspace width="0.277778em"></mspace><mi>t</mi><mi>o</mi><mspace width="0.277778em"></mspace><mi>b</mi><mi>a</mi><mi>c</mi><mi>t</mi><mi>e</mi><mi>r</mi><mi>i</mi><mi>a</mi></mrow><mrow><mi>N</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mspace width="0.277778em"></mspace><mi>o</mi><mi>f</mi><mspace width="0.277778em"></mspace><mi>d</mi><mi>a</mi><mi>m</mi><mi>a</mi><mi>g</mi><mi>e</mi><mi>d</mi><mspace width="0.277778em"></mspace><mi>c</mi><mi>e</mi><mi>l</mi><mi>l</mi><mi>s</mi><mspace width="0.277778em"></mspace><mi>n</mi><mi>o</mi><mi>t</mi><mspace width="0.277778em"></mspace><mi>e</mi><mi>x</mi><mi>p</mi><mi>o</mi><mi>s</mi><mi>e</mi><mi>d</mi><mspace width="0.277778em"></mspace><mi>t</mi><mi>o</mi><mspace width="0.277778em"></mspace><mi>b</mi><mi>a</mi><mi>c</mi><mi>t</mi><mi>e</mi><mi>r</mi><mi>i</mi><mi>a</mi><mo stretchy="false">/</mo><mi>T</mi><mi>o</mi><mi>t</mi><mi>a</mi><mi>l</mi><mspace width="0.277778em"></mspace><mi>n</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mspace width="0.277778em"></mspace><mi>o</mi><mi>f</mi><mspace width="0.277778em"></mspace><mi>c</mi><mi>e</mi><mi>l</mi><mi>l</mi><mi>s</mi><mspace width="0.277778em"></mspace><mi>n</mi><mi>o</mi><mi>t</mi><mspace width="0.277778em"></mspace><mi>e</mi><mi>x</mi><mi>p</mi><mi>o</mi><mi>s</mi><mi>e</mi><mi>d</mi><mspace width="0.277778em"></mspace><mi>t</mi><mi>o</mi><mspace width="0.277778em"></mspace><mi>b</mi><mi>a</mi><mi>c</mi><mi>t</mi><mi>e</mi><mi>r</mi><mi>i</mi><mi>a</mi></mrow></mfrac></mrow></math></td></tr></table>
<p id="Par26">We also calculated Population Attributable Risk Percent (PARP) by integrating the RR values of specific bacteria with their tissue prevalence. This provided a measure of the overall contribution of each bacterial species to cellular damage within the entire tissue environment. The PARP was calculated using the following formula, where <em>P</em> is the proportion of cells exposed to the specific bacteria:</p>
<table class="disp-formula p" id="Equb"><tr><td class="formula"><math id="d33e668" display="block"><mrow><mi>P</mi><mi>A</mi><mi>R</mi><mi>P</mi><mspace width="0.277778em"></mspace><mrow><mo stretchy="false">(</mo><mo>%</mo><mo stretchy="false">)</mo></mrow><mo>=</mo><mfenced close=")" open="("><mfrac><mrow><mi>P</mi><mo>×</mo><mfenced close=")" open="("><mi>R</mi><mi>R</mi><mo>-</mo><mn>1</mn></mfenced></mrow><mrow><mi>P</mi><mo>×</mo><mfenced close=")" open="("><mi>R</mi><mi>R</mi><mo>-</mo></mfenced><mo>+</mo><mn>1</mn></mrow></mfrac></mfenced><mo>×</mo><mn>100</mn></mrow></math></td></tr></table></section><section id="Sec16"><h3 class="pmc_sec_title">Correlation analysis of microbiome and host transcriptome</h3>
<p id="Par27">We performed Pearson correlation analysis to evaluate the relationship between the presence of each bacterial species and host gene expression levels. This analysis was performed exclusively on samples from participants with CD. Principal component analysis (PCA) was then applied to the resulting correlation matrix for dimension reduction. We analyzed the contribution of individual genes to principal component 2 (PC2) and displayed the top 20 genes with the highest absolute contribution.</p></section></section><section id="Sec17"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec18"><h3 class="pmc_sec_title">Spatial microbiome profiling reveals bacterial localization patterns in pediatric Crohn’s disease</h3>
<p id="Par28">A total of 14 terminal ileal tissue samples were biopsied, including 12 from inflamed and non-inflamed tissues of six children with pediatric ileal CD (mean age 13.5 years, SD 2.1) and two from non-inflamed tissues of two children without CD as the control group (mean age 15.0 years, SD 1.0). Spatial host-microbiome sequencing and bulk shotgun metagenome sequencing were performed on tissues. For the CD group, tissues were collected at the time of diagnosis.</p>
<p id="Par29">To capture host RNA and bacterial RNA simultaneously, in-situ polyadenylation was performed using yeast poly(A) polymerase prior to host spatial transcriptome sequencing [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. Spatial microbiome profiling was conducted by removing human reads using Bowtie2, followed by the identification of bacterial reads using Kraken2 [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Bacterial species that were nearly absent in the bulk metagenome shotgun sequencing performed on the same tissue were excluded from the spatial microbiome profiling results. Furthermore, bacterial species that were markedly reduced after additional human read removal using BWA were also excluded, as they were considered potential human read contaminants (Fig. <a href="#Fig1" class="usa-link">1</a>A) [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. As a result, 95 species of bacteria were identified, which are commonly found in the gut.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374449_40168_2025_2178_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2b24/12374449/ef22da89c9c7/40168_2025_2178_Fig1_HTML.jpg" loading="lazy" id="MO1" height="993" width="704" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Spatial host-microbiome profiling and bacterial localization patterns in pediatric Crohn’s disease (CD). <strong>A</strong> Schematic illustration of the spatial host-microbiome profiling approach, created using BioRender.com. <strong>B</strong> Cellular composition of areas enriched with various cell types, as identified by Cell2location. <strong>C</strong> Relative abundance of these areas, as identified by Cell2location. <strong>D</strong> Spatial distribution of bacterial read counts (normalized to counts per million mapped reads [CPM]) in tissue sections. The samples shown include CD7 (non-inflamed on the left side of the slide, inflamed on the right side of the slide) and CTRL2. <strong>E</strong> Distribution of cell type-enriched areas in tissue sections corresponding to panel <strong>D</strong>. <strong>F</strong> Average bacterial read count in control, non-inflamed CD, and inflamed CD tissues. Error bars represent standard error of the mean. ∗ <em>P</em> &lt; 0.05; ∗ ∗ <em>P</em> &lt; 0.005; ∗ ∗ ∗ <em>P</em> &lt; 0.0005 (Mann–Whitney two-sided test). <strong>G</strong> Average bacterial read count across different cell type-enriched areas</p></figcaption></figure><p id="Par30">Cell2location was used to map the spatial distribution of cell types in our spatial transcriptomics data by integrating previously published single-cell RNA sequencing data from pediatric patients with CD [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. This approach identified various areas enriched with certain cell-types, such as enterocyte-rich, immune cell-rich, and Tuft cell-rich areas (Fig. <a href="#Fig1" class="usa-link">1</a>B). Considering the samples from the same participant, the immune cell-rich areas were found to be more abundant in the inflamed CD tissues compared to the non-inflamed tissues (Fig. <a href="#Fig1" class="usa-link">1</a>C). The spatial distribution of bacterial reads showed distinct patterns across tissue regions, which were validated by Gram staining of adjacent tissue sections (Fig. <a href="#Fig1" class="usa-link">1</a>D, Supplementary Fig. S2 and S3).</p>
<p id="Par31">Bacterial read count per cell was higher in both non-inflamed (0.409 ± 0.028, mean ± standard error) and inflamed CD tissues (0.892 ± 0.115) compared with that in the controls (0.011 ± 0.002), with the highest count observed in inflamed CD tissues (Fig. <a href="#Fig1" class="usa-link">1</a>E, G). Notably, we observed increased counts in M cell-rich areas (8.660 ± 2.516) (Fig. <a href="#Fig1" class="usa-link">1</a>H), consistent with the known function of M cells in sampling luminal bacteria [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>].</p></section><section id="Sec19"><h3 class="pmc_sec_title">Bacterial abundance predicts Crohn’s disease prognosis</h3>
<p id="Par32">The bacterial read count per cell was highest in the CD group with relapse, followed by the CD group without relapse, and lowest in the control group. Within the relapse group, the time to relapse was shorter for participants who had a higher count at the time of diagnosis (Fig. <a href="#Fig2" class="usa-link">2</a>A). We also observed an association between bacterial read counts and the severity of endoscopic findings in the ileum, which was assessed by the Rutgeerts score (Fig. <a href="#Fig2" class="usa-link">2</a>B) [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. These findings suggest the potential for predicting CD prognosis by assessing the extent of bacterial translocation in intestinal tissues.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374449_40168_2025_2178_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2b24/12374449/a66f41b91026/40168_2025_2178_Fig2_HTML.jpg" loading="lazy" id="MO2" height="421" width="779" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Correlation of bacterial abundance with prognostic indicators and ileal endoscopic findings in pediatric Crohn’s disease (CD). <strong>A</strong> Patients were categorized into three groups: Control (Ctrl), CD without relapse, and CD with relapse. Within each group, individuals were arranged by increasing bacterial read count. The blue bars (left <em>y</em>-axis) show the average bacterial read count, normalized to counts per million mapped reads (CPM) for each patient. The orange bars (right <em>y</em>-axis) represent the time to relapse (in months) for patients with CD who experienced relapse. The red line indicates the Rutgeerts score, which assesses the severity of endoscopic findings in CD (ranging from i0 [no lesions] to i4 [diffuse inflammation with large ulcers]). Error bars represent the standard error of the mean. <strong>B</strong> Endoscopic images of the terminal ileum in patients with CD are presented, corresponding to the order of patients in panel <strong>A</strong></p></figcaption></figure></section><section id="Sec20"><h3 class="pmc_sec_title">Bacterial presence activates host immune responses</h3>
<p id="Par33">Genes differentially expressed based on the presence of 95 bacterial species were identified. Consistency of this expression pattern was examined across different bacterial species to identify genes whose expression was consistently induced or suppressed by the presence of bacteria. Furthermore, using the BioPlanet database, we investigated the functions of gene that showed consistent overexpression [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Gene expression analysis in all cells showed that bacterial presence induced upregulation of immune system components, including B cell receptor signaling (Fig. <a href="#Fig3" class="usa-link">3</a>A, B). Meanwhile, lipid absorption pathway of the intestine, such as chylomicron-mediated lipid transport, were downregulated in the presence of bacteria (Fig. <a href="#Fig3" class="usa-link">3</a>C).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374449_40168_2025_2178_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2b24/12374449/37a07702440f/40168_2025_2178_Fig3_HTML.jpg" loading="lazy" id="MO3" height="905" width="779" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differential gene expression (DGE) analysis in response to bacterial presence in Crohn’s disease. <strong>A</strong> DGE analysis in all cells. The top panel shows the rate of significant DGE in response to various bacteria. The heatmap below displays <em>Z</em>-scores of DGE for various bacterial species. <strong>B</strong> Gene set enrichment analysis (GSEA) of upregulated genes in all cells in response to bacterial presence. <strong>C</strong> GSEA of downregulated genes in all cells in response to bacterial presence. <strong>D</strong> DGE analysis in immune cell-rich areas. <strong>E</strong> DGE analysis in enterocyte-rich areas. <strong>F</strong> GSEA of upregulated genes in immune cell-rich areas in response to bacterial presence. <strong>G</strong> GSEA of upregulated genes in enterocyte-rich areas in response to bacterial presence</p></figcaption></figure><p id="Par34">Gene expression analysis of immune cell-rich and enterocyte-rich areas demonstrated consistent upregulation of genes encoding immunoglobulin components in response to bacterial presence, including <em>IGHG4</em>, <em>IGKC</em>, <em>IGHA1</em>, <em>JCHAIN</em>, and <em>IGHG3</em> in immune cell-rich areas and <em>IGHG1</em>, <em>IGHG3</em>, <em>IGHG4</em>, <em>IGLC2</em>, <em>IGKC, IGHA1</em>, <em>IGLC3</em>, and <em>IGHM</em> in enterocyte-rich areas (Fig. <a href="#Fig3" class="usa-link">3</a>D, E) [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. Functional enrichment analysis revealed that upregulated genes in both areas were involved in bacterial defense mechanisms, including interleukin signaling and innate immune system pathway (Fig. <a href="#Fig3" class="usa-link">3</a>F, G) [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Moreover, the genes in the immune cell-rich areas showed a higher degree of consistency in their differential expression compared with those in the enterocyte-rich areas, suggesting that the transcriptional response of immune cells to bacterial presence is more uniform and robust, underlining their central role in the bacterial defense response.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Bacterial species distinctly impact cell viability in Crohn’s disease</h3>
<p id="Par35">The RR values of reduced cell viability were calculated following exposure to specific bacterial species. We performed multiple testing corrections using the Benjamini–Hochberg method and visualized significant results after this correction. This analysis was performed solely on samples from participants with CD. The RR values and their 95% confidence intervals were displayed (Fig. <a href="#Fig4" class="usa-link">4</a>A, Supplementary Table S2). Based on Supplementary Table S3, species previously reported to increase CD risk are shown in red, while those known to decrease it are shown in green [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. This demonstrates that previously known beneficial and pathogenic microbiomes were consistently observed in our study.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374449_40168_2025_2178_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2b24/12374449/bdb729b0c1f0/40168_2025_2178_Fig4_HTML.jpg" loading="lazy" id="MO4" height="893" width="779" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Relative risks (RR) of reduced cell viability associated with exposure of various bacterial species in pediatric Crohn’s disease (CD). <strong>A</strong>–<strong>D</strong> RR of reduced cell viability upon exposure to various bacteria in <strong>A</strong> all cells, <strong>B</strong> enterocyte-rich areas, <strong>C</strong> immune cell-rich areas, and <strong>D</strong> tuft cell-rich areas. Statistically significant results are presented (Benjamini–Hochberg method, significance threshold of 0.05). Green dots indicate bacteria previously reported to decrease CD risk, while red dots indicate bacteria known to increase CD risk. Error bars represent 95% confidence intervals. <strong>E</strong> Comparison of the fraction of damaged cells (&gt; 10% mitochondrial reads) between bacteria-exposed and non-exposed cells across different cell types. ∗ <em>P</em> &lt; 0.05; ∗ ∗ <em>P</em> &lt; 0.005; ∗ ∗ ∗ <em>P</em> &lt; 0.0005 (Mann–Whitney two-sided test). <strong>F</strong> RR of reduced cell viability for various <em>Escherichia coli</em> strains</p></figcaption></figure><p id="Par36"><em>Bacteroides caccae</em> (RR = 0.53; 95% confidence interval [CI], 0.39–0.71), <em>F. prausnitzii</em> (RR = 0.54; 95% CI, 0.49–0.60), <em>Phocaeicola vulgatus</em> (RR = 0.85; 95% CI, 0.80–0.92), and <em>Ruminococcus gnavus</em> (RR = 0.91; 95% CI, 0.84–0.98) were confirmed as beneficial microbes, aligning with previous findings. <em>Anaerostipes hadrus</em> (RR = 0.64; 95% CI, 0.48–0.85), <em>Hungatella hathewayi</em> (RR = 0.43; 95% CI, 0.33–0.57), <em>Faecalibacterium sp</em>. <em>I3333</em> (RR = 0.57; 95% CI, 0.41–0.79), and <em>sp. I3389</em> (RR = 0.38; 95% CI, 0.31–0.47) were previously identified as beneficial at the genus level, even though their species-level association with CD were not reported. <em>Cutibacterium acnes</em> (RR = 1.50; 95% CI, 1.22–1.83), <em>H. parainfluenzae</em> (RR = 1.40; 95% CI, 1.10–1.17), and <em>Sutterella wadsorthensis</em> (RR = 1.11; 95% CI, 1.03–1.19) were confirmed as pathogenic microbes at the species level, consistent with prior studies.</p>
<p id="Par37">We also calculated the PARP by integrating the RR values of specific bacteria with their gut prevalence. This measured the overall contribution of each bacterial species to intestinal barrier disruption within the entire gut environment. <em>F. prausnitzii</em>, a highly abundant gut bacterium known for its beneficial properties in CD, was the most influential in reducing intestinal barrier disruption (by − 4.6%) (Supplementary Fig. S4).</p>
<p id="Par38">When the RR was examined specifically in enterocyte-rich areas (Fig. <a href="#Fig4" class="usa-link">4</a>B), all bacteria were found to reduce cell viability, even species such as <em>Phocaeicola vulgatus</em>, which was initially found to inhibit this reduction during analysis of the entire cell population. In contrast, analyses of immune cell- and tuft cell-rich areas revealed that some bacteria exhibited a protective effect by inhibiting the reduction in cell viability (Fig. <a href="#Fig4" class="usa-link">4</a>C, D). Additionally, bacterial exposure increased the fraction of damaged cells in enterocyte-rich areas, but decreased it in tuft cell-rich, immune cell-rich, and myofibroblast-rich areas. This suggests that the beneficial effect of certain gut microbiomes may be facilitated through interactions with cell types other than enterocytes, potentially including immune and tuft cells. These findings highlight the complex interplay between various host gut cell types and the resident bacteria.</p>
<p id="Par39">To further investigate cell type-specific bacterial associations, we calculated co-existence frequencies between each bacterial species and 42 cell types by multiplying Cell2location-derived cell type fractions with bacterial counts per spot (Supplementary Fig. S5). This analysis revealed that beneficial microbiome candidates were particularly enriched in M cells and memory B cells, while pathogenic microbiome candidates showed higher prevalence in enterocytes and tuft cells.</p>
<p id="Par40">To account for the heterogeneity of <em>Escherichia coli</em>, a highly abundant species with diverse strains ranging from commensal to pathogenic, we calculated the RR values for 42 different <em>E. coli</em> strains characterized at the strain level, although only 15.2% were identified at this taxonomic resolution (Fig. <a href="#Fig4" class="usa-link">4</a>F, Supplementary Fig. S6, Supplementary Table S4). We identified <em>E. coli APEC O1</em> (RR = 1.66; 95% CI, 1.29–2.14), <em>O157:H7</em> (RR = 1.43; 95% CI, 1.22–1.68), and <em>CFT073</em> (RR = 1.32; 95% CI, 1.14–1.53) as part of the pathogenic microbiome, which is consistent with previous reports [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>–<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>].</p></section><section id="Sec22"><h3 class="pmc_sec_title">Beneficial and pathogenic microbiomes distinctly modulate host transcription</h3>
<p id="Par41">We analyzed differentially expressed genes between cells exposed to beneficial and pathogenic microbiomes. Using 16 beneficial and nine pathogenic members identified in this study, we formed 144 beneficial-pathogenic microbiome pairs. For each pair, we compared gene expressions in response to the beneficial versus pathogenic microbiome. We assessed the consistency of these changes across the different pairs, identifying genes consistently upregulated or downregulated in the presence of beneficial or pathogenic bacteria.</p>
<p id="Par42">Across all cells, beneficial microbiomes increased the expression of immunoglobulin genes, including <em>IGHG3</em>, <em>IGHG4</em>, <em>IGLC1</em>, <em>IGHM</em>, <em>IGKC</em>, <em>IGLC2</em>, <em>IGHG2</em>, and <em>IGHA1</em> (Fig. <a href="#Fig5" class="usa-link">5</a>A) [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. This suggests that beneficial microbiomes may exert their effects through immune cells. In immune cell-rich areas, genes such as <em>REG1A</em> and <em>TNFRSF6B</em> showed increased expression in regions with beneficial microbiomes (Fig. <a href="#Fig5" class="usa-link">5</a>D). Similarly, in enterocytes, genes including <em>IGKC</em> and <em>REG1A</em> were upregulated in such areas (Fig. <a href="#Fig5" class="usa-link">5</a>E).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374449_40168_2025_2178_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2b24/12374449/7cc10bebcaf8/40168_2025_2178_Fig5_HTML.jpg" loading="lazy" id="MO5" height="903" width="779" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differential gene expression (DGE) analysis between cells exposed to beneficial and pathogenic microbiomes and Principal Component Analysis (PCA) of the correlation matrix between bacterial presence and host gene expression in pediatric Crohn’s disease. <strong>A</strong>, <strong>D</strong>, <strong>E</strong> DGE analysis in <strong>A</strong> all cells, <strong>D</strong> immune cell-rich areas, and <strong>E</strong> enterocyte-rich areas. The top panel shows the rate of significant DGE between cells exposed to beneficial and pathogenic microbiome members. The heatmaps display <em>Z</em>-scores of DGE. <strong>B</strong> PCA of the correlation matrix between bacterial presence and host gene expression levels. Pathogenic microbiome members are shown in red, and beneficial microbiome members are shown in green. <strong>C</strong> Bar chart displaying the top 20 genes with the highest absolute contribution to Principal Component 2</p></figcaption></figure><p id="Par43">To assess species-specific bacterial effects on host gene expression, we calculated Pearson correlations for bacterial presence and gene expression levels. We then applied PCA for dimension reduction of the correlation matrix. The results showed a clear distinction between beneficial and pathogenic microbial species along PC2 (Fig. <a href="#Fig5" class="usa-link">5</a>B). Furthermore, the expression of genes such as <em>REG1A</em> and <em>TNFRSF6B</em> had substantial contributions in PC2 (Fig. <a href="#Fig5" class="usa-link">5</a>C, Supplementary Fig. S7).</p></section></section><section id="Sec23"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par44">The critical roles of bacteria-host interactions in CD are increasingly acknowledged. However, the intricate relationships between various cell types and bacterial species at the cellular level are still poorly understood [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. Therefore, we developed a novel spatial host-microbiome profiling approach that, to the best of our knowledge, is the first to enable simultaneous species-level identification of bacteria and host transcriptomics. Using this method, we demonstrated increased bacterial presence in pediatric CD tissues, with a significant association between bacterial abundance and disease prognosis, while also revealing distinct host transcriptome alterations in response to various bacterial species. Furthermore, we identified and characterized potentially beneficial and pathogenic microbial species associated with CD, including several newly discovered risk-modulating bacterial species. Our spatial host-microbiome profiling approach not only provides profound insights regarding CD, but also offers potential applications for studying various microbiome-associated diseases such as gastrointestinal cancer and infectious diseases. It reveals spatial associations between gut bacteria and various host cell types in CD pathophysiology at the cellular level. Moreover, the identification of beneficial and pathogenic microbiome members enables the development of microbiome-based therapeutic strategies.</p>
<p id="Par45">Our results highlighted the importance of bacterial presence in CD, revealing that CD tissues exhibited increased bacterial abundance compared with controls. Furthermore, within CD samples, inflamed tissues showed higher bacterial abundance compared with non-inflamed tissues. These findings suggest a more active bacterial invasion in CD, potentially due to impaired host defense mechanisms [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>], which aligns with the findings of Sun, D, et al., who also reported an increase in bacterial translocation in inflamed tissues compared with non-inflamed tissues in CD [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Moreover, the bacterial abundance at diagnosis not only predicted disease prognosis in patients with CD but also showed a strong association with the severity of endoscopic findings in the ileum. This indicates that the level of bacterial infiltration in intestinal tissues could serve as a potential prognostic marker for predicting disease course in pediatric CD.</p>
<p id="Par46">Analysis of differentially expressed genes in response to the presence of bacterial species revealed a specific and targeted response to bacterial infiltration. Consistent upregulation of genes encoding immunoglobulin components appears to be a response to bacterial presence, which was further confirmed by gene enrichment analysis revealing the involvement of these upregulated genes in bacterial defense functions. Notably, the higher degree of consistency in the differential expression of genes in immune cell-rich areas compared with enterocyte-rich areas shows the uniform and robust transcriptional response of immune cells to bacterial presence. Furthermore, our data showed the upregulation of lncRNA, such as LINC02739, in response to bacterial infiltration, suggesting their potential involvement in the host-microbe interaction and disease pathogenesis. While the specific functions of these lncRNAs remain unclear, their differential expression warrants further investigation into their roles in modulating the immune response and disease progression in CD [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>].</p>
<p id="Par47">To identify potentially beneficial and pathogenic microbes in pediatric CD, we assessed the impact of specific bacterial species on intestinal barrier integrity by determining the RRs of reduced cell viability upon exposure to these bacteria. Our analysis identified 16 beneficial and nine pathogenic microbiome candidates. Among these, six were previously reported to have a confirmed impact on CD at the species level, while eight were known to affect only at the genus level [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Interestingly, the majority of the beneficial microbiome members we identified (13 out of 16 species) belong to <em>Bacillota</em> (<em>Firmicutes</em>), with a substantial number (11 species) belonging to <em>Clostridium</em> clusters XIVa and IV, which have been previously associated with a reduced risk of CD (Supplementary Table S2) [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>–<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. In contrast, five out of nine pathogenic microbiome members, including <em>Citrobacter freundii</em>, <em>Escherichia fergusonii</em>, <em>H. parainfluenzae</em>, <em>Escherichia albertii</em>, and <em>Sutterella wadsworthensis</em>, belong to <em>Pseudomonadota</em> (<em>Proteobaccteria</em>). This aligns with previous studies suggesting an association between increased abundance of <em>Pseudomonadota</em> (<em>Proteobaccteria</em>) and CD [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>, <a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. Notably, we identified five beneficial microbiome members that were not reported in previous studies. Four of these newly identified beneficial microbes belong to <em>Bacillota</em> (<em>Firmicutes</em>), showing similar phylogeny to other known beneficial microbes. Interestingly, three of these newly identified species, <em>Sellimonas intestinalis</em>, <em>Turicimonas muris</em>, and <em>Dysosmobacter welbionis</em>, have been reclassified or newly discovered after the publication of the reference studies we used (Gevers et al., 2014; Kansal et al., 2019) [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>, <a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>, <a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. These findings demonstrate that our newly identified beneficial microbes are aligned with those from previous reports and highlight how our understanding of the microbiome deepens as bacterial databases expand and taxonomic classifications become more precise.</p>
<p id="Par48">Our study highlights that the impact of gut microbes on CD pathogenesis is not uniform, even within the same genus, as demonstrated by the <em>Faecalibacterium</em> genus. While <em>F. prausnitzii</em>, <em>Faecalibacterium sp. I3389</em>, and <em>Faecalibacterium sp. I3333</em> were identified as beneficial microbes in this study, <em>Faecalibacterium sp. I2392</em>, <em>Faecalibacterium sp. I4179</em>, <em>Faecalibacterium sp. I4384</em>, and <em>Faecalibacterium sp. IP329</em> did not have statistically significant RR values. Additionally, our spatial microbiome analysis at the strain level confirmed that <em>E. coli</em>, which has various strains ranging from commensal to pathogenic, showed varying associations with CD risk [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>–<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>, <a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>].</p>
<p id="Par49">Comparing tissue-specific gene expression between cells exposed to beneficial and pathogenic microbiomes, we observed increased expression of genes such as <em>REG1A</em> and <em>TNFRSF6B</em> in the presence of beneficial microbiomes. According to Mao et al., <em>REG1A</em> plays a crucial role in tissue regeneration and the repair of intestinal epithelial damage [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. In their study, inducing <em>REG1A</em> expression in a DSS colitis mouse model promoted the recovery of the intestinal barrier. TNFRSF6B, also known as DcR3, is a well-known inhibitor of FASL and LIGHT, which are essential for cell apoptosis [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. The upregulation of <em>TNFRSF6B</em> in the presence of beneficial bacteria suggests that these microbes may be associated with the maintenance of intestinal epithelial integrity, potentially through their influence on host cell apoptosis pathways. These findings provide insights into the potential mechanisms by which beneficial microbiomes may provide protection against CD pathogenesis, emphasizing the need for further research to understand the complex interactions between gut microbes and host cellular processes.</p>
<p id="Par50">Our spatial host-microbiome sequencing approach offers several advantages over traditional methods. Unlike dissociation-based single-cell RNA sequencing methods such as Chromium, our approach allows for the accurate identification of host cells exposed to bacteria, as it avoids the potential dissociation of bacteria and host cells during sample preparation [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. Additionally, our method minimizes the risk of contamination by ambient RNA, a common issue in droplet-based methods, resulting from cell lysis and RNA release within the microfluidic droplets, ensuring that the detected bacterial reads accurately reflect the true bacterial distribution within the tissue [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. Furthermore, our approach enables the analysis of damaged cells, which are typically removed in dead cell removal processes prior to single-cell RNA sequencing studies [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. This is considerable in the context of CD, where removing damaged cells may inadvertently exclude the more inflamed portion of the tissue, leading to a potential bias in the results.</p>
<p id="Par51">Previous studies using 16 s rRNA targeting approaches identified bacterial distributions only at the genus level [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. In contrast, the Kraken2-based shotgun metagenomic profiling approach employed in our study allows for the detection of even small amounts of bacteria and enables differentiation at the strain level [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Moreover, our approach eliminates the need for creating probes, allowing for the evaluation of a wide range of bacteria, including those less frequently reported in the literature, some of which may have potential as therapeutic targets [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>–<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Unlike probe-based methods that restrict analysis to pre-selected bacterial targets, our shotgun metagenomic approach enables comprehensive identification of all bacterial species present in the sample. This is achieved through a simple sample preparation step with the addition of yeast poly(A) polymerase, eliminating the need for the cumbersome preparation of new probes for specific bacteria.</p>
<p id="Par52">This study has several limitations. First, the small sample size and limited age range may introduce biases, although the total number of cells analyzed was substantial (13,876). Future studies should include more samples from diverse ethnic backgrounds to improve generalizability. Second, our control group consisted of pediatric IBS patients rather than healthy individuals, as obtaining intestinal biopsies from healthy children presents ethical and practical challenges. We selected patients who volunteered for endoscopy and showed no inflammatory markers on blood tests, endoscopic examination, or histopathological analysis. Third, the spatial transcriptomic technology lacks single-cell resolution, resulting in the mixing of various cell types within spots [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. Although we employed Cell2location to classify areas based on cell combinations, heterogeneity within areas of the same cell-type may still exist, meaning that there could be slight differences in cellular composition among areas classified as the same type. This heterogeneity could potentially influence the comparisons made within these areas, such as those for differentially expressed genes in immune cell-rich areas due to bacterial exposure. However, the emergence of high-resolution single-cell spatial transcriptomics techniques offers opportunities to apply our algorithm to more refined spatial data in future investigations, which could help reduce the impact of cellular heterogeneity on our analyses [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>].</p>
<p id="Par53">Additionally, future studies should incorporate diverse modalities including metabolomics data and leverage various analytical methods and multi-omics databases to comprehensively characterize the complex interactions between bacteria and host responses in CD [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. Improvements in reference databases, more sophisticated taxonomic classification algorithms, and enhanced spatial transcriptomics methods for bacterial RNA capture will be essential for further advancement and broader applications of spatial microbiome profiling. Moreover, experimental validation of the beneficial microbiomes and host response mechanisms identified in this study will be important research directions.</p></section><section id="Sec24"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par54">In conclusion, our study introduces a novel spatial host-microbiome profiling approach that enables the simultaneous profiling of the host transcriptome and bacterial species at a high taxonomic resolution in the ileal tissues of pediatric patients with CD. This approach allowed us to identify increased bacterial presence in CD tissues, as well as the potential prognostic value of assessing bacterial infiltration in intestinal tissues. We also discovered specific beneficial and pathogenic microbiomes associated with CD pathogenesis and suggested potential mechanisms by which these microbes may influence disease progression, such as the modulation of host cell apoptosis pathways. The identification of several newly discovered beneficial microbiomes provides promising candidates for the development of novel microbiome-based therapeutics for CD. Our spatial host-microbiome sequencing approach offers a valuable method to understand bacterial-associated host transcriptional alterations in the context of CD pathogenesis.</p></section><section id="Sec118"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12374449/bin/40168_2025_2178_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40168_2025_2178_MOESM1_ESM.docx</a><sup> (37.6KB, docx) </sup><p>Additional file 1: STROBE Statement—checklist of items that should be included in reports of observational studies.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12374449/bin/40168_2025_2178_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40168_2025_2178_MOESM2_ESM.docx</a><sup> (29.1MB, docx) </sup><p>Additional file 2: Supplementary methods. Supplementary Figure S1 Comparison of spatial versus bulk sequencing detection and single versus double human read removal for gut-residing and false positive bacterial species (A, B) Comparison between spatial sequencing and bulk shotgun sequencing detection for genuine gut-residing species (Faecalibacterium prausnitzii and Escherichia coli) and potential false positive species (Guillardia theta and Puccinia striiformis). (C, D) Comparison of bacterial read counts between single (Bowtie2 only) and double (Bowtie2 + BWA) human read removal processes for genuine gut-residing species (Faecalibacterium prausnitzii and Escherichia coli) and potential false positive species (Guillardia theta and Puccinia striiformis). Supplementary Figure S2 Validation of spatial microbiome profiling by Gram staining of adjacent tissue sections. (A) Spatial distribution of bacterial reads (red intensity) across tissue sections from CD patients and controls, as detected by spatial host-microbiome profiling. (B) Histological appearance (H&amp;E staining) of corresponding tissue sections showing tissue architecture. (C) Gram staining of adjacent tissue sections demonstrating bacterial localization patterns with Gram-positive bacteria appearing as dark purple clusters. Supplementary Figure S3 High-magnification correlation between spatial microbiome sequencing and Gram staining. (A-C) Higher magnification views of selected regions showing correlation between spatial sequencing bacterial signals and Gram-positive bacterial clusters (dark purple) within tissue structures. The spatial distribution patterns of bacteria detected by sequencing show overall similarity with bacterial localization observed in Gram-stained adjacent sections, validating the spatial microbiome profiling approach. Supplementary Figure S4 Population Attributable Risk Percent (PARP) analysis of bacterial species in pediatric Crohn's disease. PARP values for bacterial species, calculated by integrating relative risk values with tissue prevalence. Green bars indicate beneficial bacteria that reduce intestinal barrier disruption, while red bars represent pathogenic bacteria that increase tissue damage. Supplementary Figure S5 Cell type-specific co-localization probabilities of bacterial species in pediatric ileal Crohn's disease tissues. (A) Heatmap showing co-localization probabilities between bacterial species and immune cell types. Color intensity represents the strength of co-localization probability calculated by multiplying Cell2location-derived cell type fractions with bacterial counts per spot. (B) Co-localization probabilities between bacterial species and non-immune cell types. (C) Log2 fold change comparison of co-localization probabilities between beneficial and pathogenic microbiomes across different cell types. Green bars indicate cell types with higher co-localization probability for beneficial microbiomes, while red bars indicate higher probability for pathogenic microbiomes. Statistical significance is indicated by asterisks (*<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001). Supplementary Figure S6 Strain-level relative risk analysis of bacterial species associated with reduced cell viability in pediatric Crohn's disease. Relative risk (RR) values for strain-level bacterial species and their association with reduced cell viability in CD tissues. Red circles indicate pathogenic strains with RR &gt; 1.0, while gray circles reƒpresent beneficial strains with RR &lt; 1.0. Error bars represent 95% confidence intervals. Only statistically significant results after Benjamini–Hochberg correction are shown. Supplementary Figure S7 Gene contribution to Principal Component 1 in bacterial presence correlation analysis. Bar chart displaying the top 20 genes with the highest absolute contribution to Principal Component 1 (PC1) in the correlation analysis between bacterial presence and host gene expression levels.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12374449/bin/40168_2025_2178_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">40168_2025_2178_MOESM3_ESM.xlsx</a><sup> (24.2KB, xlsx) </sup><p>Additional file 3: Supplementary Table S1. Baseline clinical, laboratory, and endoscopic characteristics of pediatric CD patients and IBS control subjects. Supplementary Table S2. Bacterial relative risk analysis for reduced cell viability in pediatric Crohn's disease. Supplementary Table S3. Previous literature reports of bacterial associations with Crohn's disease risk according to Gevers et al. (2014) and Kansal et al. (2019). Supplementary Table S4. Strain-level bacterial relative risk values for reduced cell viability in pediatric Crohn's disease.</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Not applicable</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>CD</dt>
<dd><p id="Par4">Crohn’s disease)</p></dd>
<dt>RR</dt>
<dd><p id="Par5">Relative risk</p></dd>
<dt>RIN</dt>
<dd><p id="Par6">RNA Integrity Number</p></dd>
<dt>PARP</dt>
<dd><p id="Par7">Population Attributable Risk Percent</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>Sooyoung Jang wrote the main manuscript text and prepared all figures. Sooyoung Jang led the data curation, conceptualization, investigation, and writing of the original draft, while sharing equal responsibility for project administration. Eun Joo Lee, Sowon Park, and Hyeji Lim provided supporting roles in data curation and conceptualization. Yoon Huh contributed in a supporting capacity to the investigation. Byungsoo Ahn prepared Supplementary figure 2 and 3. Hong Koh supported project administration, shared equal supervision responsibilities, and contributed to review and editing. Yu Rang Park led project administration, provided methodological support, shared equal supervision responsibilities, and contributed equally to the review and editing process.
</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study was supported by grants from the MD-PhD Physician-Scientist Training Program from the Korea Health Industry Development Institute (KHIDI), Ministry of Health and Welfare of the Republic of Korea, Technology Innovation Program (20018770, Development of Human Microbiome Treatment Technology for Inflammatory Bowel Disease) funded by the Ministry of Trade Industry &amp; Energy (MOTIE) of the Republic of Korea, and Bio &amp; Medical Technology Development Program of the National Research Foundation (NRF) funded by the Ministry of Science &amp; ICT (RS-2021-NR056579).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability
</h2>
<p>The raw sequencing data generated in this study have been deposited in the NCBI Sequence Read Archive (SRA) under the accession number PRJNA1159478. Prior to publication, the data can be accessed at <a href="https://dataview.ncbi.nlm.nih.gov/object/PRJNA1159478?reviewer=sjgo3s8uv1itu8c37csovv3qc5" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dataview.ncbi.nlm.nih.gov/object/PRJNA1159478?reviewer=sjgo3s8uv1itu8c37csovv3qc5</a>.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Ethics approval and consent to participate</h2>
<p id="Par55">Ethical approval was obtained from the Institutional Review Board of Severance Hospital, Yonsei University College of Medicine (IRB number: 4–2022-1127). Both the patients and their legal guardians provided informed consent for participation in the study.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Consent for publication</h2>
<p id="Par56">Not applicable.</p></section><section id="FPar3"><h2 class="pmc_sec_title">Competing interests
</h2>
<p id="Par57">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Hong Koh, Email: khong@yuhs.ac.</p>
<p>Yu Rang Park, Email: yurangpark@yuhs.ac.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Neurath MF. Targeting immune cell circuits and trafficking in inflammatory bowel disease. Nat Immunol. 2019;20:970–9. Available from: <a href="https://www.nature.com/articles/s41590-019-0415-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/s41590-019-0415-0</a>. Cited 4 Apr 2023.
</cite> [<a href="https://doi.org/10.1038/s41590-019-0415-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31235952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Neurath%20MF.%20Targeting%20immune%20cell%20circuits%20and%20trafficking%20in%20inflammatory%20bowel%20disease.%20Nat%20Immunol.%202019;20:970%E2%80%939.%C2%A0Available%20from:%C2%A0https://www.nature.com/articles/s41590-019-0415-0.%20Cited%204%20Apr%202023." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Raygoza Garay JA, Turpin W, Lee S-H, Smith MI, Goethel A, Griffiths AM, et al. Gut microbiome composition is associated with future onset of Crohn’s disease in healthy first-degree relatives. Gastroenterology. 2023;165:670–81. Available from: <a href="https://www.sciencedirect.com/science/article/pii/S0016508523008053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sciencedirect.com/science/article/pii/S0016508523008053</a>. Cited 30 May 2025.
</cite> [<a href="https://doi.org/10.1053/j.gastro.2023.05.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37263307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raygoza%20Garay%20JA,%20Turpin%20W,%20Lee%20S-H,%20Smith%20MI,%20Goethel%20A,%20Griffiths%20AM,%20et%20al.%20Gut%20microbiome%20composition%20is%20associated%20with%20future%20onset%20of%20Crohn%E2%80%99s%20disease%20in%20healthy%20first-degree%20relatives.%20Gastroenterology.%202023;165:670%E2%80%9381.%C2%A0Available%20from:%C2%A0https://www.sciencedirect.com/science/article/pii/S0016508523008053.%C2%A0Cited%2030%20May%202025." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Schaubeck M, Clavel T, Calasan J, Lagkouvardos I, Haange SB, Jehmlich N, et al. Dysbiotic gut microbiota causes transmissible Crohn’s disease-like ileitis independent of failure in antimicrobial defence. 2016 ; Available from: <a href="https://gut.bmj.com/content/65/2/225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gut.bmj.com/content/65/2/225</a>. Cited 30 May 2025.</cite> [<a href="https://doi.org/10.1136/gutjnl-2015-309333" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4752651/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25887379/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Graham DB, Xavier RJ. Pathway paradigms revealed from the genetics of inflammatory bowel disease. Nature. 2020;578:527–39. Available from: <a href="https://www.nature.com/articles/s41586-020-2025-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/s41586-020-2025-2</a>. Cited 16 Nov 2023.</cite> [<a href="https://doi.org/10.1038/s41586-020-2025-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7871366/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32103191/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Balish E, Warner T. Enterococcus faecalis induces inflammatory bowel disease in interleukin-10 knockout mice. American J Pathol. 2002;160:2253–7. Available from: <a href="https://www.sciencedirect.com/science/article/pii/S0002944010611728" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sciencedirect.com/science/article/pii/S0002944010611728</a>. Cited 16 Nov 2023.</cite> [<a href="https://doi.org/10.1016/S0002-9440(10)61172-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1850822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12057927/" class="usa-link">PubMed</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Kim SC, Tonkonogy SL, Albright CA, Tsang J, Balish EJ, Braun J, et al. Variable phenotypes of enterocolitis in interleukin 10-deficient mice monoassociated with two different commensal bacteria. Gastroenterol. 2005;128:891–906. Available from: <a href="https://www.sciencedirect.com/science/article/pii/S0016508505001782" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sciencedirect.com/science/article/pii/S0016508505001782</a>. Cited 16 Nov 2023.</cite> [<a href="https://doi.org/10.1053/j.gastro.2005.02.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15825073/" class="usa-link">PubMed</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Ha CWY, Martin A, Sepich-Poore GD, Shi B, Wang Y, Gouin K, et al. Translocation of viable gut microbiota to mesenteric adipose drives formation of creeping fat in humans. Cell. 2020;183:666–683.e17. Available from: <a href="https://www.sciencedirect.com/science/article/pii/S0092867420311508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sciencedirect.com/science/article/pii/S0092867420311508</a>. Cited 9 Dec 2023.</cite> [<a href="https://doi.org/10.1016/j.cell.2020.09.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7521382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32991841/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Wright EK, Kamm MA, Teo SM, Inouye M, Wagner J, Kirkwood CD. Recent Advances in Characterizing the Gastrointestinal Microbiome in Crohn’s Disease: A Systematic Review. Inflammatory Bowel Diseases. 2015;21:1219–28. Available from: 10.1097/MIB.0000000000000382. Cited 19 Nov 2023.</cite> [<a href="https://doi.org/10.1097/MIB.0000000000000382" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4450900/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25844959/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Gevers D, Kugathasan S, Denson LA, Vázquez-Baeza Y, Van Treuren W, Ren B, et al. The treatment-naive microbiome in new-onset Crohn’s disease. Cell Host Microbe. 2014;15:382–92.
</cite> [<a href="https://doi.org/10.1016/j.chom.2014.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4059512/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24629344/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gevers%20D,%20Kugathasan%20S,%20Denson%20LA,%20V%C3%A1zquez-Baeza%20Y,%20Van%20Treuren%20W,%20Ren%20B,%20et%20al.%20The%20treatment-naive%20microbiome%20in%20new-onset%20Crohn%E2%80%99s%20disease.%20Cell%20Host%20Microbe.%202014;15:382%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Kansal S, Catto-Smith AG, Boniface K, Thomas S, Cameron DJ, Oliver M, et al. The microbiome in paediatric Crohn’s disease—a longitudinal, prospective, single-centre study. J Crohns Colitis. 2019;13:1044–54. Available from: 10.1093/ecco-jcc/jjz016. Cited 11 Apr 2024.</cite> [<a href="https://doi.org/10.1093/ecco-jcc/jjz016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30927743/" class="usa-link">PubMed</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Haberman Y, Tickle TL, Dexheimer PJ, Kim M-O, Tang D, Karns R, et al. Pediatric Crohn disease patients exhibit specific ileal transcriptome and microbiome signature. J Clin Invest. 2014;124:3617–33.
</cite> [<a href="https://doi.org/10.1172/JCI75436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4109533/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25003194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haberman%20Y,%20Tickle%20TL,%20Dexheimer%20PJ,%20Kim%20M-O,%20Tang%20D,%20Karns%20R,%20et%20al.%20Pediatric%20Crohn%20disease%20patients%20exhibit%20specific%20ileal%20transcriptome%20and%20microbiome%20signature.%20J%20Clin%20Invest.%202014;124:3617%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Hansen R, Russell RK, Reiff C, Louis P, McIntosh F, Berry SH, et al. Microbiota ofDe-NovoPediatric IBD: IncreasedFaecalibacterium Prausnitziiand reduced bacterial diversity in Crohn’s but not in ulcerative colitis. Official J Am Coll Gastroenterol| ACG. 2012;107:1913. Available from: <a href="https://journals.lww.com/ajg/Fulltext/2012/12000/Microbiota_ofDe_NovoPediatric_IBD_.26.aspx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.lww.com/ajg/Fulltext/2012/12000/Microbiota_ofDe_NovoPediatric_IBD_.26.aspx</a>. Cited 19 Nov 2023.</cite> [<a href="https://doi.org/10.1038/ajg.2012.335" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23044767/" class="usa-link">PubMed</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Sorbara MT, Pamer EG. Microbiome-based therapeutics. Nat Rev Microbiol. 2022;20:365–80. Available from: <a href="https://www.nature.com/articles/s41579-021-00667-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/s41579-021-00667-9</a>. Cited 11 Apr 2024.</cite> [<a href="https://doi.org/10.1038/s41579-021-00667-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34992261/" class="usa-link">PubMed</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Mohebali N, Weigel M, Hain T, Sütel M, Bull J, Kreikemeyer B, et al. Faecalibacterium prausnitzii, Bacteroides faecis and Roseburia intestinalis attenuate clinical symptoms of experimental colitis by regulating Treg/Th17 cell balance and intestinal barrier integrity. Biomed Pharmacother. 2023;167:115568.
</cite> [<a href="https://doi.org/10.1016/j.biopha.2023.115568" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37793274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mohebali%20N,%20Weigel%20M,%20Hain%20T,%20S%C3%BCtel%20M,%20Bull%20J,%20Kreikemeyer%20B,%20et%20al.%20Faecalibacterium%20prausnitzii,%20Bacteroides%20faecis%20and%20Roseburia%20intestinalis%20attenuate%20clinical%20symptoms%20of%20experimental%20colitis%20by%20regulating%20Treg/Th17%20cell%20balance%20and%20intestinal%20barrier%20integrity.%20Biomed%20Pharmacother.%202023;167:115568." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Tian L, Chen F, Macosko EZ. The expanding vistas of spatial transcriptomics. Nat Biotechnol. 2023;41:773–82. Available from: <a href="https://www.nature.com/articles/s41587-022-01448-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/s41587-022-01448-2</a>. Cited 25 Jun 2024.</cite> [<a href="https://doi.org/10.1038/s41587-022-01448-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10091579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36192637/" class="usa-link">PubMed</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Ståhl PL, Salmén F, Vickovic S, Lundmark A, Navarro JF, Magnusson J, et al. Visualization and analysis of gene expression in tissue sections by spatial transcriptomics. Sci. 2016;353:78–82. Available from: <a href="https://www.science.org/doi/full/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.science.org/doi/full/</a>. 10.1126/science.aaf2403. Cited 25 Jun 2024.</cite> [<a href="https://doi.org/10.1126/science.aaf2403" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27365449/" class="usa-link">PubMed</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>McKellar DW, Mantri M, Hinchman MM, Parker JSL, Sethupathy P, Cosgrove BD, et al. Spatial mapping of the total transcriptome by in situ polyadenylation. Nat Biotechnol. 2022;1–8. Available from: <a href="https://www.nature.com/articles/s41587-022-01517-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/s41587-022-01517-6</a>. Cited 24 Mar 2023.</cite> [<a href="https://doi.org/10.1038/s41587-022-01517-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10110464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36329320/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Galeano Niño JL, Wu H, LaCourse KD, Kempchinsky AG, Baryiames A, Barber B, et al. Effect of the intratumoral microbiota on spatial and cellular heterogeneity in cancer. Nature. 2022;611:810–7. Available from: <a href="https://www.nature.com/articles/s41586-022-05435-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/s41586-022-05435-0</a>. Cited 4 Apr 2023.</cite> [<a href="https://doi.org/10.1038/s41586-022-05435-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9684076/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36385528/" class="usa-link">PubMed</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Saarenpää S, Shalev O, Ashkenazy H, Carlos V, Lundberg DS, Weigel D, et al. Spatial metatranscriptomics resolves host–bacteria–fungi interactomes. Nat Biotechnol. 2023;1–10. Available from: <a href="https://www.nature.com/articles/s41587-023-01979-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/s41587-023-01979-2</a>. Cited 2 Jan 2024.</cite> [<a href="https://doi.org/10.1038/s41587-023-01979-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11392817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37985875/" class="usa-link">PubMed</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Lötstedt B, Stražar M, Xavier R, Regev A, Vickovic S. Spatial host–microbiome sequencing reveals niches in the mouse gut. Nat Biotechnol. 2023;1–10. Available from: <a href="https://www.nature.com/articles/s41587-023-01988-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/s41587-023-01988-1</a>. Cited 2 Jan 2024.</cite> [<a href="https://doi.org/10.1038/s41587-023-01988-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11392810/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37985876/" class="usa-link">PubMed</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Hyams JS, Ferry GD, Mandel FS, Gryboski JD, Kibort PM, Kirschner BS, et al. Development and validation of a pediatric Crohn’s disease activity index. J Pediatr Gastroenterol Nutr. 1991;12:439–47.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/1678008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hyams%20JS,%20Ferry%20GD,%20Mandel%20FS,%20Gryboski%20JD,%20Kibort%20PM,%20Kirschner%20BS,%20et%20al.%20Development%20and%20validation%20of%20a%20pediatric%20Crohn%E2%80%99s%20disease%20activity%20index.%20J%20Pediatr%20Gastroenterol%20Nutr.%201991;12:439%E2%80%9347." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Wolf FA, Angerer P, Theis FJ. SCANPY: large-scale single-cell gene expression data analysis. Genome Biology. 2018;19:15. Available from: 10.1186/s13059-017-1382-0. Cited 13 Dec 2023.</cite> [<a href="https://doi.org/10.1186/s13059-017-1382-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5802054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29409532/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Kleshchevnikov V, Shmatko A, Dann E, Aivazidis A, King HW, Li T, et al. Cell2location maps fine-grained cell types in spatial transcriptomics. Nat Biotechnol. 2022;40:661–71. Available from: <a href="https://www.nature.com/articles/s41587-021-01139-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/s41587-021-01139-4</a>. Cited 25 Jun 2024.</cite> [<a href="https://doi.org/10.1038/s41587-021-01139-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35027729/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Elmentaite R, Ross ADB, Roberts K, James KR, Ortmann D, Gomes T, et al. Single-cell sequencing of developing human gut reveals transcriptional links to childhood Crohn’s disease. Dev Cell. 2020;55:771-783.e5.
</cite> [<a href="https://doi.org/10.1016/j.devcel.2020.11.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7762816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33290721/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elmentaite%20R,%20Ross%20ADB,%20Roberts%20K,%20James%20KR,%20Ortmann%20D,%20Gomes%20T,%20et%20al.%20Single-cell%20sequencing%20of%20developing%20human%20gut%20reveals%20transcriptional%20links%20to%20childhood%20Crohn%E2%80%99s%20disease.%20Dev%20Cell.%202020;55:771-783.e5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Elmentaite R, Kumasaka N, Roberts K, Fleming A, Dann E, King HW, et al. Cells of the human intestinal tract mapped across space and time. Nature. 2021;597:250–5. Available from: <a href="https://www.nature.com/articles/s41586-021-03852-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/s41586-021-03852-1</a>. Cited 25 Jul 2024.</cite> [<a href="https://doi.org/10.1038/s41586-021-03852-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8426186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34497389/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9:357–9. Available from: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322381/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322381/</a>. Cited 10 Dec 2023.</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Smith T, Heger A, Sudbery I. UMI-tools: modeling sequencing errors in Unique Molecular Identifiers to improve quantification accuracy. Genome Res. 2017;27:491–9. Available from: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5340976/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5340976/</a>. Cited 10 Dec 2023.</cite> [<a href="https://doi.org/10.1101/gr.209601.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5340976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28100584/" class="usa-link">PubMed</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Wood DE, Lu J, Langmead B. Improved metagenomic analysis with Kraken 2. Genome Biology. 2019;20:257. Available from: 10.1186/s13059-019-1891-0. Cited 29 Aug 2023.</cite> [<a href="https://doi.org/10.1186/s13059-019-1891-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6883579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31779668/" class="usa-link">PubMed</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Garrido-Sanz L, Senar MÀ, Piñol J. Drastic reduction of false positive species in samples of insects by intersecting the default output of two popular metagenomic classifiers. PLOS ONE. 2022;17:e0275790. Available from: https://journals.plos.org/plosone/article?id= 10.1371/journal.pone.0275790. Cited 7 Aug 2024.</cite> [<a href="https://doi.org/10.1371/journal.pone.0275790" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9595558/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36282811/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Li H, Durbin R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics. 2009;25:1754–60. Available from: 10.1093/bioinformatics/btp324. Cited 11 Dec 2023.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2705234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" class="usa-link">PubMed</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Rutgeerts P, Geboes K, Vantrappen G, Beyls J, Kerremans R, Hiele M. Predictability of the postoperative course of Crohn’s disease. Gastroenterol. 1990;99:956–63. Available from: <a href="https://www.sciencedirect.com/science/article/pii/0016508590906136" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sciencedirect.com/science/article/pii/0016508590906136</a>. Cited 2 Aug 2024.</cite> [<a href="https://doi.org/10.1016/0016-5085(90)90613-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2394349/" class="usa-link">PubMed</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Huang R, Grishagin I, Wang Y, Zhao T, Greene J, Obenauer JC, et al. The NCATS BioPlanet – an integrated platform for exploring the universe of cellular signaling pathways for toxicology, systems biology, and chemical genomics. Front Pharmacol. 2019;10. Available from: <a href="https://www.frontiersin.org/journals/pharmacology/articles/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.frontiersin.org/journals/pharmacology/articles/</a>. 10.3389/fphar.2019.00445/full. Cited 2 Aug 2024.</cite> [<a href="https://doi.org/10.3389/fphar.2019.00445" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6524730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31133849/" class="usa-link">PubMed</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Ilicic T, Kim JK, Kolodziejczyk AA, Bagger FO, McCarthy DJ, Marioni JC, et al. Classification of low quality cells from single-cell RNA-seq data. Genome Biology. 2016;17:29. Available from: 10.1186/s13059-016-0888-1. Cited 25 Jun 2024.</cite> [<a href="https://doi.org/10.1186/s13059-016-0888-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4758103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26887813/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Dillon A, Lo DD. M Cells: Intelligent Engineering of Mucosal Immune Surveillance. Front Immunol. 2019;10. Available from: <a href="https://www.frontiersin.org/journals/immunology/articles/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.frontiersin.org/journals/immunology/articles/</a>. 10.3389/fimmu.2019.01499/full. Cited 10 Jul 2024.</cite> [<a href="https://doi.org/10.3389/fimmu.2019.01499" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6614372/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31312204/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Mikocziova I, Greiff V, Sollid LM. Immunoglobulin germline gene variation and its impact on human disease. Genes Immun. 2021;22:205–17. Available from: <a href="https://www.nature.com/articles/s41435-021-00145-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/s41435-021-00145-5</a>. Cited 5 Aug 2024.</cite> [<a href="https://doi.org/10.1038/s41435-021-00145-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8234759/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34175903/" class="usa-link">PubMed</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Johnson TJ, Kariyawasam S, Wannemuehler Y, Mangiamele P, Johnson SJ, Doetkott C, et al. The genome sequence of avian pathogenic Escherichia coli strain O1:K1:H7 shares strong similarities with human extraintestinal pathogenic E. coli genomes. J Bacteriol. 2007;189:3228–36.
</cite> [<a href="https://doi.org/10.1128/JB.01726-06" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1855855/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17293413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johnson%20TJ,%20Kariyawasam%20S,%20Wannemuehler%20Y,%20Mangiamele%20P,%20Johnson%20SJ,%20Doetkott%20C,%20et%20al.%20The%20genome%20sequence%20of%20avian%20pathogenic%20Escherichia%20coli%20strain%20O1:K1:H7%20shares%20strong%20similarities%20with%20human%20extraintestinal%20pathogenic%20E.%20coli%20genomes.%20J%20Bacteriol.%202007;189:3228%E2%80%9336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Riley LW, Remis RS, Helgerson SD, McGee HB, Wells JG, Davis BR, et al. Hemorrhagic colitis associated with a rare Escherichia coli serotype. N Engl J Med. 1983;308:681–5.
</cite> [<a href="https://doi.org/10.1056/NEJM198303243081203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6338386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Riley%20LW,%20Remis%20RS,%20Helgerson%20SD,%20McGee%20HB,%20Wells%20JG,%20Davis%20BR,%20et%20al.%20Hemorrhagic%20colitis%20associated%20with%20a%20rare%20Escherichia%20coli%20serotype.%20N%20Engl%20J%20Med.%201983;308:681%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Kao JS, Stucker DM, Warren JW, Mobley HL. Pathogenicity island sequences of pyelonephritogenic Escherichia coli CFT073 are associated with virulent uropathogenic strains. Infect Immun. 1997;65:2812–20.
</cite> [<a href="https://doi.org/10.1128/iai.65.7.2812-2820.1997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC175396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9199454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kao%20JS,%20Stucker%20DM,%20Warren%20JW,%20Mobley%20HL.%20Pathogenicity%20island%20sequences%20of%20pyelonephritogenic%20Escherichia%20coli%20CFT073%20are%20associated%20with%20virulent%20uropathogenic%20strains.%20Infect%20Immun.%201997;65:2812%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Sun D, Ge X, Tang S, Liu Y, Sun J, Zhou Y, et al. Bacterial characteristics of intestinal tissues from patients with Crohn’s disease. Front Cell Infect Microbiol. 2021;11:711680. Available from: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8635149/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8635149/</a>. Cited 18 Jun 2024.</cite> [<a href="https://doi.org/10.3389/fcimb.2021.711680" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8635149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34869050/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Atianand MK, Caffrey DR, Fitzgerald KA. Immunobiology of long noncoding RNAs. Annu Rev Immunol. 2017;35:177–98. Available from: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6449690/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6449690/</a>. Cited 10 Jul 2024.</cite> [<a href="https://doi.org/10.1146/annurev-immunol-041015-055459" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6449690/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28125358/" class="usa-link">PubMed</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Andoh A, Imaeda H, Aomatsu T, Inatomi O, Bamba S, Sasaki M, et al. Comparison of the fecal microbiota profiles between ulcerative colitis and Crohn’s disease using terminal restriction fragment length polymorphism analysis. J Gastroenterol. 2011;46:479–86. Available from: 10.1007/s00535-010-0368-4. Cited 19 Nov 2023.</cite> [<a href="https://doi.org/10.1007/s00535-010-0368-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21253779/" class="usa-link">PubMed</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Liu C, Finegold SM, Song Y, Lawson PA. Reclassification of Clostridium coccoides, Ruminococcus hansenii, Ruminococcus hydrogenotrophicus, Ruminococcus luti, Ruminococcus productus and Ruminococcus schinkii as Blautia coccoides gen. nov., comb. nov., Blautia hansenii comb. nov., Blautia hydrogenotrophica comb. nov., Blautia luti comb. nov., Blautia producta comb. nov., Blautia schinkii comb. nov., and description of Blautia wexlerae sp. nov., isolated from human faeces. Int J Syst Evol Microbiol. 2008;58:1896–902.
</cite> [<a href="https://doi.org/10.1099/ijs.0.65208-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18676476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20C,%20Finegold%20SM,%20Song%20Y,%20Lawson%20PA.%20Reclassification%20of%20Clostridium%20coccoides,%20Ruminococcus%20hansenii,%20Ruminococcus%20hydrogenotrophicus,%20Ruminococcus%20luti,%20Ruminococcus%20productus%20and%20Ruminococcus%20schinkii%20as%20Blautia%20coccoides%20gen.%20nov.,%20comb.%20nov.,%20Blautia%20hansenii%20comb.%20nov.,%20Blautia%20hydrogenotrophica%20comb.%20nov.,%20Blautia%20luti%20comb.%20nov.,%20Blautia%20producta%20comb.%20nov.,%20Blautia%20schinkii%20comb.%20nov.,%20and%20description%20of%20Blautia%20wexlerae%20sp.%20nov.,%20isolated%20from%20human%20faeces.%20Int%20J%20Syst%20Evol%20Microbiol.%202008;58:1896%E2%80%93902." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Seo B, Yoo JE, Lee YM, Ko G. Sellimonas intestinalis gen. nov., sp. nov., isolated from human faeces. Int J Syst Evol Microbiol. 2016;66:951–6.
</cite> [<a href="https://doi.org/10.1099/ijsem.0.000817" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26637816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seo%20B,%20Yoo%20JE,%20Lee%20YM,%20Ko%20G.%20Sellimonas%20intestinalis%20gen.%20nov.,%20sp.%20nov.,%20isolated%20from%20human%20faeces.%20Int%20J%20Syst%20Evol%20Microbiol.%202016;66:951%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Taras D, Simmering R, Collins MD, Lawson PA, Blaut M. Reclassification of Eubacterium formicigenerans Holdeman and Moore 1974 as Dorea formicigenerans gen. nov., comb. nov., and description of Dorea longicatena sp. nov., isolated from human faeces. Int J Syst Evol Microbiol. 2002;52:423–8.
</cite> [<a href="https://doi.org/10.1099/00207713-52-2-423" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11931151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Taras%20D,%20Simmering%20R,%20Collins%20MD,%20Lawson%20PA,%20Blaut%20M.%20Reclassification%20of%20Eubacterium%20formicigenerans%20Holdeman%20and%20Moore%201974%20as%20Dorea%20formicigenerans%20gen.%20nov.,%20comb.%20nov.,%20and%20description%20of%20Dorea%20longicatena%20sp.%20nov.,%20isolated%20from%20human%20faeces.%20Int%20J%20Syst%20Evol%20Microbiol.%202002;52:423%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Mahowald MA, Rey FE, Seedorf H, Turnbaugh PJ, Fulton RS, Wollam A, et al. Characterizing a model human gut microbiota composed of members of its two dominant bacterial phyla. Proc Natl Acad Sci U S A. 2009;106:5859–64.
</cite> [<a href="https://doi.org/10.1073/pnas.0901529106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2660063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19321416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mahowald%20MA,%20Rey%20FE,%20Seedorf%20H,%20Turnbaugh%20PJ,%20Fulton%20RS,%20Wollam%20A,%20et%20al.%20Characterizing%20a%20model%20human%20gut%20microbiota%20composed%20of%20members%20of%20its%20two%20dominant%20bacterial%20phyla.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202009;106:5859%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Allen-Vercoe E, Daigneault M, White A, Panaccione R, Duncan SH, Flint HJ, et al. Anaerostipes hadrus comb. nov., a dominant species within the human colonic microbiota; reclassification of Eubacterium hadrum Moore et al. 1976. Anaerobe. 2012;18:523–9.
</cite> [<a href="https://doi.org/10.1016/j.anaerobe.2012.09.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22982042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Allen-Vercoe%20E,%20Daigneault%20M,%20White%20A,%20Panaccione%20R,%20Duncan%20SH,%20Flint%20HJ,%20et%20al.%20Anaerostipes%20hadrus%20comb.%20nov.,%20a%20dominant%20species%20within%20the%20human%20colonic%20microbiota;%20reclassification%20of%20Eubacterium%20hadrum%20Moore%20et%C2%A0al.%201976.%20Anaerobe.%202012;18:523%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Schoch CL, Ciufo S, Domrachev M, Hotton CL, Kannan S, Khovanskaya R, et al. NCBI Taxonomy: a comprehensive update on curation, resources and tools. Database (Oxford). 2020;2020:baaa062.</cite> [<a href="https://doi.org/10.1093/database/baaa062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7408187/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32761142/" class="usa-link">PubMed</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Haas KN, Blanchard JL. Reclassification of the Clostridium clostridioforme and Clostridium sphenoides clades as Enterocloster gen. nov. and Lacrimispora gen. nov., including reclassification of 15 taxa. Int J Syst Evol Microbiol. 2020;70:23–34.
</cite> [<a href="https://doi.org/10.1099/ijsem.0.003698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31782700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haas%20KN,%20Blanchard%20JL.%20Reclassification%20of%20the%20Clostridium%20clostridioforme%20and%20Clostridium%20sphenoides%20clades%20as%20Enterocloster%20gen.%20nov.%20and%20Lacrimispora%20gen.%20nov.,%20including%20reclassification%20of%2015%20taxa.%20Int%20J%20Syst%20Evol%20Microbiol.%202020;70:23%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Kaur S, Yawar M, Kumar PA, Suresh K. Hungatella effluvii gen. nov., sp. nov., an obligately anaerobic bacterium isolated from an effluent treatment plant, and reclassification of Clostridium hathewayi as Hungatella hathewayi gen. nov., comb. nov. Int J Syst Evol Microbiol. 2014;64:710–8. Available from: <a href="https://www.microbiologyresearch.org/content/journal/ijsem/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.microbiologyresearch.org/content/journal/ijsem/</a>. 10.1099/ijs.0.056986-0. Cited 11 Apr 2024.</cite> [<a href="https://doi.org/10.1099/ijs.0.056986-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24186873/" class="usa-link">PubMed</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Le Roy T, Van der Smissen P, Paquot A, Delzenne N, Muccioli GG, Collet J-F, et al. Dysosmobacter welbionis gen. nov., sp. nov., isolated from human faeces and emended description of the genus Oscillibacter. Int J Syst Evol Microbiol. 2020;70:4851–8. Available from: <a href="https://www.microbiologyresearch.org/content/journal/ijsem/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.microbiologyresearch.org/content/journal/ijsem/</a>. 10.1099/ijsem.0.003547. Cited 10 Jul 2024.</cite> [<a href="https://doi.org/10.1099/ijsem.0.003547" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31232680/" class="usa-link">PubMed</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Vester-Andersen MK, Mirsepasi-Lauridsen HC, Prosberg MV, Mortensen CO, Träger C, Skovsen K, et al. Increased abundance of proteobacteria in aggressive Crohn’s disease seven years after diagnosis. Sci Rep. 2019;9:13473. Available from: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6748953/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6748953/</a>. Cited 7 Aug 2024.</cite> [<a href="https://doi.org/10.1038/s41598-019-49833-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6748953/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31530835/" class="usa-link">PubMed</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Lagkouvardos I, Pukall R, Abt B, Foesel BU, Meier-Kolthoff JP, Kumar N, et al. The Mouse Intestinal Bacterial Collection (miBC) provides host-specific insight into cultured diversity and functional potential of the gut microbiota. Nat Microbiol. 2016;1:16131.
</cite> [<a href="https://doi.org/10.1038/nmicrobiol.2016.131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27670113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lagkouvardos%20I,%20Pukall%20R,%20Abt%20B,%20Foesel%20BU,%20Meier-Kolthoff%20JP,%20Kumar%20N,%20et%20al.%20The%20Mouse%20Intestinal%20Bacterial%20Collection%20(miBC)%20provides%20host-specific%20insight%20into%20cultured%20diversity%20and%20functional%20potential%20of%20the%20gut%20microbiota.%20Nat%20Microbiol.%202016;1:16131." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Kaper JB, Nataro JP, Mobley HLT. Pathogenic Escherichia coli. Nat Rev Microbiol. 2004;2:123–40. Available from: <a href="https://www.nature.com/articles/nrmicro818" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/nrmicro818</a>. Cited 31 May 2024.</cite> [<a href="https://doi.org/10.1038/nrmicro818" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15040260/" class="usa-link">PubMed</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Mao H, Jia J, Sheng J, Zhang S, Huang K, Li H, et al. Protective and anti-inflammatory role of REG1A in inflammatory bowel disease induced by JAK/STAT3 signaling axis. Int Immunopharmacol. 2021;92:107304.
</cite> [<a href="https://doi.org/10.1016/j.intimp.2020.107304" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33513463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mao%20H,%20Jia%20J,%20Sheng%20J,%20Zhang%20S,%20Huang%20K,%20Li%20H,%20et%20al.%20Protective%20and%20anti-inflammatory%20role%20of%20REG1A%20in%20inflammatory%20bowel%20disease%20induced%20by%20JAK/STAT3%20signaling%20axis.%20Int%20Immunopharmacol.%202021;92:107304." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Su J, Tong Z, Wu S, Zhou F, Chen Q. Research progress of DcR3 in the diagnosis and treatment of sepsis. Int J Mol Sci. 2023;24:12916.
</cite> [<a href="https://doi.org/10.3390/ijms241612916" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10454171/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37629097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su%20J,%20Tong%20Z,%20Wu%20S,%20Zhou%20F,%20Chen%20Q.%20Research%20progress%20of%20DcR3%20in%20the%20diagnosis%20and%20treatment%20of%20sepsis.%20Int%20J%20Mol%20Sci.%202023;24:12916." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Zheng GXY, Terry JM, Belgrader P, Ryvkin P, Bent ZW, Wilson R, et al. Massively parallel digital transcriptional profiling of single cells. Nat Commun. 2017;8:14049. Available from: <a href="https://www.nature.com/articles/ncomms14049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/ncomms14049</a>. Cited 10 Dec 2023.</cite> [<a href="https://doi.org/10.1038/ncomms14049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5241818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28091601/" class="usa-link">PubMed</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Caglayan E, Liu Y, Konopka G. Neuronal ambient RNA contamination causes misinterpreted and masked cell types in brain single-nuclei datasets. Neuron. 2022;110:4043–4056.e5. Available from: <a href="https://www.sciencedirect.com/science/article/pii/S0896627322008157" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sciencedirect.com/science/article/pii/S0896627322008157</a>. Cited 7 Aug 2024.</cite> [<a href="https://doi.org/10.1016/j.neuron.2022.09.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9789184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36240767/" class="usa-link">PubMed</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Durazzi F, Sala C, Castellani G, Manfreda G, Remondini D, De Cesare A. Comparison between 16S rRNA and shotgun sequencing data for the taxonomic characterization of the gut microbiota. Sci Rep. 2021;11:3030.
</cite> [<a href="https://doi.org/10.1038/s41598-021-82726-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7862389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33542369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Durazzi%20F,%20Sala%20C,%20Castellani%20G,%20Manfreda%20G,%20Remondini%20D,%20De%20Cesare%20A.%20Comparison%20between%2016S%20rRNA%20and%20shotgun%20sequencing%20data%20for%20the%20taxonomic%20characterization%20of%20the%20gut%20microbiota.%20Sci%20Rep.%202021;11:3030." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Chen A, Sun Y, Lei Y, Li C, Liao S, Meng J, et al. Single-cell spatial transcriptome reveals cell-type organization in the macaque cortex. Cell. 2023;186:3726–3743.e24. Available from: <a href="https://linkinghub.elsevier.com/retrieve/pii/S0092867423006797" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://linkinghub.elsevier.com/retrieve/pii/S0092867423006797</a>. Cited 7 Aug 2024.</cite> [<a href="https://doi.org/10.1016/j.cell.2023.06.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37442136/" class="usa-link">PubMed</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Zhang Y, Thomas JP, Korcsmaros T, Gul L. Integrating multi-omics to unravel host-microbiome interactions in inflammatory bowel disease. CR Med. 2024;5. Available from: <a href="https://www.cell.com/cell-reports-medicine/abstract/S2666-3791(24)00468-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cell.com/cell-reports-medicine/abstract/S2666-3791(24)00468-3</a>. Cited 30 May 2025.</cite> [<a href="https://doi.org/10.1016/j.xcrm.2024.101738" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11525031/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39293401/" class="usa-link">PubMed</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374449/bin/40168_2025_2178_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40168_2025_2178_MOESM1_ESM.docx</a><sup> (37.6KB, docx) </sup><p>Additional file 1: STROBE Statement—checklist of items that should be included in reports of observational studies.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374449/bin/40168_2025_2178_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">40168_2025_2178_MOESM2_ESM.docx</a><sup> (29.1MB, docx) </sup><p>Additional file 2: Supplementary methods. Supplementary Figure S1 Comparison of spatial versus bulk sequencing detection and single versus double human read removal for gut-residing and false positive bacterial species (A, B) Comparison between spatial sequencing and bulk shotgun sequencing detection for genuine gut-residing species (Faecalibacterium prausnitzii and Escherichia coli) and potential false positive species (Guillardia theta and Puccinia striiformis). (C, D) Comparison of bacterial read counts between single (Bowtie2 only) and double (Bowtie2 + BWA) human read removal processes for genuine gut-residing species (Faecalibacterium prausnitzii and Escherichia coli) and potential false positive species (Guillardia theta and Puccinia striiformis). Supplementary Figure S2 Validation of spatial microbiome profiling by Gram staining of adjacent tissue sections. (A) Spatial distribution of bacterial reads (red intensity) across tissue sections from CD patients and controls, as detected by spatial host-microbiome profiling. (B) Histological appearance (H&amp;E staining) of corresponding tissue sections showing tissue architecture. (C) Gram staining of adjacent tissue sections demonstrating bacterial localization patterns with Gram-positive bacteria appearing as dark purple clusters. Supplementary Figure S3 High-magnification correlation between spatial microbiome sequencing and Gram staining. (A-C) Higher magnification views of selected regions showing correlation between spatial sequencing bacterial signals and Gram-positive bacterial clusters (dark purple) within tissue structures. The spatial distribution patterns of bacteria detected by sequencing show overall similarity with bacterial localization observed in Gram-stained adjacent sections, validating the spatial microbiome profiling approach. Supplementary Figure S4 Population Attributable Risk Percent (PARP) analysis of bacterial species in pediatric Crohn's disease. PARP values for bacterial species, calculated by integrating relative risk values with tissue prevalence. Green bars indicate beneficial bacteria that reduce intestinal barrier disruption, while red bars represent pathogenic bacteria that increase tissue damage. Supplementary Figure S5 Cell type-specific co-localization probabilities of bacterial species in pediatric ileal Crohn's disease tissues. (A) Heatmap showing co-localization probabilities between bacterial species and immune cell types. Color intensity represents the strength of co-localization probability calculated by multiplying Cell2location-derived cell type fractions with bacterial counts per spot. (B) Co-localization probabilities between bacterial species and non-immune cell types. (C) Log2 fold change comparison of co-localization probabilities between beneficial and pathogenic microbiomes across different cell types. Green bars indicate cell types with higher co-localization probability for beneficial microbiomes, while red bars indicate higher probability for pathogenic microbiomes. Statistical significance is indicated by asterisks (*<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001). Supplementary Figure S6 Strain-level relative risk analysis of bacterial species associated with reduced cell viability in pediatric Crohn's disease. Relative risk (RR) values for strain-level bacterial species and their association with reduced cell viability in CD tissues. Red circles indicate pathogenic strains with RR &gt; 1.0, while gray circles reƒpresent beneficial strains with RR &lt; 1.0. Error bars represent 95% confidence intervals. Only statistically significant results after Benjamini–Hochberg correction are shown. Supplementary Figure S7 Gene contribution to Principal Component 1 in bacterial presence correlation analysis. Bar chart displaying the top 20 genes with the highest absolute contribution to Principal Component 1 (PC1) in the correlation analysis between bacterial presence and host gene expression levels.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374449/bin/40168_2025_2178_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">40168_2025_2178_MOESM3_ESM.xlsx</a><sup> (24.2KB, xlsx) </sup><p>Additional file 3: Supplementary Table S1. Baseline clinical, laboratory, and endoscopic characteristics of pediatric CD patients and IBS control subjects. Supplementary Table S2. Bacterial relative risk analysis for reduced cell viability in pediatric Crohn's disease. Supplementary Table S3. Previous literature reports of bacterial associations with Crohn's disease risk according to Gevers et al. (2014) and Kansal et al. (2019). Supplementary Table S4. Strain-level bacterial relative risk values for reduced cell viability in pediatric Crohn's disease.</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The raw sequencing data generated in this study have been deposited in the NCBI Sequence Read Archive (SRA) under the accession number PRJNA1159478. Prior to publication, the data can be accessed at <a href="https://dataview.ncbi.nlm.nih.gov/object/PRJNA1159478?reviewer=sjgo3s8uv1itu8c37csovv3qc5" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dataview.ncbi.nlm.nih.gov/object/PRJNA1159478?reviewer=sjgo3s8uv1itu8c37csovv3qc5</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Microbiome are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s40168-025-02178-8"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/40168_2025_Article_2178.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12374449/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12374449/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374449%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374449/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12374449/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12374449/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40849632/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12374449/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40849632/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12374449/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12374449/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="QNbL2cF6uasopQKEinB6tcdFyDL2RBYt5S7JlB51a6ECQIrYBLgAG7afaoRr8MSl">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
