
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Harnessing big data for enhanced genome-wide prediction in winter wheat breeding - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5329878AF194C305298700751FDEF5.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="springeropen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373547/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik">
<meta name="citation_title" content="Harnessing big data for enhanced genome-wide prediction in winter wheat breeding">
<meta name="citation_author" content="Ravindra Reddy Gundala">
<meta name="citation_author_institution" content="Leibniz Institute for Plant Genetics and Crop Plant Research, Corrensstraße 3, 06466 Seeland, Germany">
<meta name="citation_author" content="Ulrike Avenhaus">
<meta name="citation_author_institution" content="W. Von Borries-Eckendorf GmbH &amp; Co. KG, Hovedisser Str. 94, 33818 Leopoldshoehe, Germany">
<meta name="citation_author" content="Jost Doernte">
<meta name="citation_author_institution" content="Deutsche Saatveredelung AG, Weissenburger Straße 5, 59557 Lippstadt, Germany">
<meta name="citation_author" content="Wera Maria Eckhoff">
<meta name="citation_author_institution" content="KWS Saat SE, Grimsehlstr. 31, 57574 Einbeck, Germany">
<meta name="citation_author" content="Jutta Foerster">
<meta name="citation_author_institution" content="SU BIOTEC GmbH, Am Schwabeplan 6B, 60439 Gatersleben, Germany">
<meta name="citation_author" content="Mario Gils">
<meta name="citation_author_institution" content="Nordsaat Saatzucht GmbH, Böhnshauser Str. 1, 38895 Langenstein, Germany">
<meta name="citation_author" content="Michael Koch">
<meta name="citation_author_institution" content="Deutsche Saatveredelung AG, Weissenburger Straße 5, 59557 Lippstadt, Germany">
<meta name="citation_author" content="Martin Kirchhoff">
<meta name="citation_author_institution" content="Nordsaat Saatzucht GmbH, Böhnshauser Str. 1, 38895 Langenstein, Germany">
<meta name="citation_author_institution" content="Present Address: Nordzucker AG, Magdeburger Landstraße 1-5, 39164 Wanzleben-Boerde, Germany">
<meta name="citation_author" content="Sonja Kollers">
<meta name="citation_author_institution" content="KWS Saat SE, Grimsehlstr. 31, 57574 Einbeck, Germany">
<meta name="citation_author" content="Nina Pfeiffer">
<meta name="citation_author_institution" content="KWS Lochow GmbH, Wetze 3, 37154 Northeim, Germany">
<meta name="citation_author" content="Matthias Rapp">
<meta name="citation_author_institution" content="W. Von Borries-Eckendorf GmbH &amp; Co. KG, Hovedisser Str. 94, 33818 Leopoldshoehe, Germany">
<meta name="citation_author" content="Monika Spiller">
<meta name="citation_author_institution" content="KWS Lochow GmbH, Wetze 3, 37154 Northeim, Germany">
<meta name="citation_author" content="Valentin Wimmer">
<meta name="citation_author_institution" content="KWS Saat SE, Grimsehlstr. 31, 57574 Einbeck, Germany">
<meta name="citation_author_institution" content="Present Address: Aardevo B.V. Johannes, Postweg 8, 8308 PB Nagele, Netherlands">
<meta name="citation_author" content="Markus Wolf">
<meta name="citation_author_institution" content="SU BIOTEC GmbH, Am Schwabeplan 6B, 60439 Gatersleben, Germany">
<meta name="citation_author" content="Yusheng Zhao">
<meta name="citation_author_institution" content="Leibniz Institute for Plant Genetics and Crop Plant Research, Corrensstraße 3, 06466 Seeland, Germany">
<meta name="citation_author" content="Jochen Christoph Reif">
<meta name="citation_author_institution" content="Leibniz Institute for Plant Genetics and Crop Plant Research, Corrensstraße 3, 06466 Seeland, Germany">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="138">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="224">
<meta name="citation_doi" content="10.1007/s00122-025-05007-6">
<meta name="citation_pmid" content="40844639">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373547/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373547/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373547/pdf/122_2025_Article_5007.pdf">
<meta name="description" content="By combining data from different public and private breeding programs for genomic selection, we have increased the size and diversity of the training population, which has led to better predictions of grain yield and plant height in winter wheat ...">
<meta name="og:title" content="Harnessing big data for enhanced genome-wide prediction in winter wheat breeding">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="By combining data from different public and private breeding programs for genomic selection, we have increased the size and diversity of the training population, which has led to better predictions of grain yield and plant height in winter wheat ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373547/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373547">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1007/s00122-025-05007-6"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/122_2025_Article_5007.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373547%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373547/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373547/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373547/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-springeropen.png" alt="Springer logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Springer" title="Link to Springer" shape="default" href="https://doi.org/10.1007/s00122-025-05007-6" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Theor Appl Genet</button></div>. 2025 Aug 22;138(9):224. doi: <a href="https://doi.org/10.1007/s00122-025-05007-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1007/s00122-025-05007-6</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Theor%20Appl%20Genet%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Theor%20Appl%20Genet%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Theor%20Appl%20Genet%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Theor%20Appl%20Genet%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Harnessing big data for enhanced genome-wide prediction in winter wheat breeding</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gundala%20RR%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Ravindra Reddy Gundala</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Ravindra Reddy Gundala</span></h3>
<div class="p">
<sup>1</sup>Leibniz Institute for Plant Genetics and Crop Plant Research, Corrensstraße 3, 06466 Seeland, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gundala%20RR%22%5BAuthor%5D" class="usa-link"><span class="name western">Ravindra Reddy Gundala</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Avenhaus%20U%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ulrike Avenhaus</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ulrike Avenhaus</span></h3>
<div class="p">
<sup>2</sup>W. Von Borries-Eckendorf GmbH &amp; Co. KG, Hovedisser Str. 94, 33818 Leopoldshoehe, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Avenhaus%20U%22%5BAuthor%5D" class="usa-link"><span class="name western">Ulrike Avenhaus</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doernte%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jost Doernte</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jost Doernte</span></h3>
<div class="p">
<sup>3</sup>Deutsche Saatveredelung AG, Weissenburger Straße 5, 59557 Lippstadt, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doernte%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jost Doernte</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Eckhoff%20WM%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Wera Maria Eckhoff</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Wera Maria Eckhoff</span></h3>
<div class="p">
<sup>4</sup>KWS Saat SE, Grimsehlstr. 31, 57574 Einbeck, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Eckhoff%20WM%22%5BAuthor%5D" class="usa-link"><span class="name western">Wera Maria Eckhoff</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Foerster%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Jutta Foerster</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jutta Foerster</span></h3>
<div class="p">
<sup>5</sup>SU BIOTEC GmbH, Am Schwabeplan 6B, 60439 Gatersleben, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Foerster%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jutta Foerster</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gils%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Mario Gils</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Mario Gils</span></h3>
<div class="p">
<sup>6</sup>Nordsaat Saatzucht GmbH, Böhnshauser Str. 1, 38895 Langenstein, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gils%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mario Gils</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koch%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Michael Koch</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Michael Koch</span></h3>
<div class="p">
<sup>3</sup>Deutsche Saatveredelung AG, Weissenburger Straße 5, 59557 Lippstadt, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koch%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael Koch</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kirchhoff%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Martin Kirchhoff</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Martin Kirchhoff</span></h3>
<div class="p">
<sup>6</sup>Nordsaat Saatzucht GmbH, Böhnshauser Str. 1, 38895 Langenstein, Germany </div>
<div class="p">
<sup>8</sup>Present Address: Nordzucker AG, Magdeburger Landstraße 1-5, 39164 Wanzleben-Boerde, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kirchhoff%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Kirchhoff</span></a>
</div>
</div>
<sup>6,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kollers%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Sonja Kollers</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Sonja Kollers</span></h3>
<div class="p">
<sup>4</sup>KWS Saat SE, Grimsehlstr. 31, 57574 Einbeck, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kollers%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sonja Kollers</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pfeiffer%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Nina Pfeiffer</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Nina Pfeiffer</span></h3>
<div class="p">
<sup>7</sup>KWS Lochow GmbH, Wetze 3, 37154 Northeim, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pfeiffer%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nina Pfeiffer</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rapp%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Matthias Rapp</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Matthias Rapp</span></h3>
<div class="p">
<sup>2</sup>W. Von Borries-Eckendorf GmbH &amp; Co. KG, Hovedisser Str. 94, 33818 Leopoldshoehe, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rapp%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Matthias Rapp</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Spiller%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Monika Spiller</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Monika Spiller</span></h3>
<div class="p">
<sup>7</sup>KWS Lochow GmbH, Wetze 3, 37154 Northeim, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Spiller%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Monika Spiller</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wimmer%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Valentin Wimmer</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Valentin Wimmer</span></h3>
<div class="p">
<sup>4</sup>KWS Saat SE, Grimsehlstr. 31, 57574 Einbeck, Germany </div>
<div class="p">
<sup>9</sup>Present Address: Aardevo B.V. Johannes, Postweg 8, 8308 PB Nagele, Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wimmer%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Valentin Wimmer</span></a>
</div>
</div>
<sup>4,</sup><sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolf%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Markus Wolf</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Markus Wolf</span></h3>
<div class="p">
<sup>5</sup>SU BIOTEC GmbH, Am Schwabeplan 6B, 60439 Gatersleben, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolf%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Markus Wolf</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Yusheng Zhao</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Yusheng Zhao</span></h3>
<div class="p">
<sup>1</sup>Leibniz Institute for Plant Genetics and Crop Plant Research, Corrensstraße 3, 06466 Seeland, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yusheng Zhao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reif%20JC%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Jochen Christoph Reif</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Jochen Christoph Reif</span></h3>
<div class="p">
<sup>1</sup>Leibniz Institute for Plant Genetics and Crop Plant Research, Corrensstraße 3, 06466 Seeland, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reif%20JC%22%5BAuthor%5D" class="usa-link"><span class="name western">Jochen Christoph Reif</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Leibniz Institute for Plant Genetics and Crop Plant Research, Corrensstraße 3, 06466 Seeland, Germany </div>
<div id="Aff2">
<sup>2</sup>W. Von Borries-Eckendorf GmbH &amp; Co. KG, Hovedisser Str. 94, 33818 Leopoldshoehe, Germany </div>
<div id="Aff3">
<sup>3</sup>Deutsche Saatveredelung AG, Weissenburger Straße 5, 59557 Lippstadt, Germany </div>
<div id="Aff4">
<sup>4</sup>KWS Saat SE, Grimsehlstr. 31, 57574 Einbeck, Germany </div>
<div id="Aff5">
<sup>5</sup>SU BIOTEC GmbH, Am Schwabeplan 6B, 60439 Gatersleben, Germany </div>
<div id="Aff6">
<sup>6</sup>Nordsaat Saatzucht GmbH, Böhnshauser Str. 1, 38895 Langenstein, Germany </div>
<div id="Aff7">
<sup>7</sup>KWS Lochow GmbH, Wetze 3, 37154 Northeim, Germany </div>
<div id="Aff8">
<sup>8</sup>Present Address: Nordzucker AG, Magdeburger Landstraße 1-5, 39164 Wanzleben-Boerde, Germany </div>
<div id="Aff9">
<sup>9</sup>Present Address: Aardevo B.V. Johannes, Postweg 8, 8308 PB Nagele, Netherlands </div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>Communicated by Philomin Juliana.</p></div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 12; Accepted 2025 Jul 29; Issue date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373547  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40844639/" class="usa-link">40844639</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Key message</h3>
<p id="Par1">By combining data from different public and private breeding programs for genomic selection, we have increased the size and diversity of the training population, which has led to better predictions of grain yield and plant height in winter wheat compared to using individual training sets.</p></section><section id="sec2"><h3 class="pmc_sec_title">Abstract</h3>
<p id="Par2">The accuracy of genome-wide prediction is anticipated to improve with an increase in training population size. In our study, we assembled a comprehensive wheat data set consisting of about 18,000 inbred lines and phenotypic data from about 250,000 plots. We evaluated the potential to train genome-wide prediction models using this big data set through data from post-registration trials conducted across a wide range of environments. Our findings demonstrated that using big data can enhance the prediction ability by up to 97% for grain yield and 44% for plant height, outperforming individual training sets. This improvement is primarily attributed to the expansion of the training set size relative to the genetic diversity. In conclusion, big data holds significant potential to accelerate genetic gain in winter wheat predictive breeding, making it a compelling option.</p></section><section id="sec3"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1007/s00122-025-05007-6.</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Wheat is a global staple food, and to meet the demands of a growing population by 2050, wheat production must be doubled. Achieving this goal at the current rate of yield increase represents a significant challenge (Ray et al. <a href="#CR38" class="usa-link" aria-describedby="CR38">2013</a>). Research suggests that genomic selection could potentially triple the genetic gain compared to existing phenotypic selection (Meuwissen et al. <a href="#CR31" class="usa-link" aria-describedby="CR31">2001</a>; Voss-Fels et al. <a href="#CR46" class="usa-link" aria-describedby="CR46">2019</a>; Tessema et al. <a href="#CR44" class="usa-link" aria-describedby="CR44">2020</a>). Therefore, it is proposed as a promising strategy to help bridge the yield gap. However, the primary challenge lies in enhancing the prediction ability of genomic selection models to maximize the genetic gain.</p>
<p id="Par4">Prediction ability depends on a variety of factors and their interactions, including heritability, the genetic architecture of the trait, marker density, population structure, genetic relationship between training and test sets, size of the training population, and genetic diversity (Combs and Bernardo <a href="#CR9" class="usa-link" aria-describedby="CR9">2013</a>; Lian et al. <a href="#CR26" class="usa-link" aria-describedby="CR26">2014</a>; Windhausen et al. <a href="#CR51" class="usa-link" aria-describedby="CR51">2012</a>; Würschum et al. <a href="#CR52" class="usa-link" aria-describedby="CR52">2017</a>; Werner et al. <a href="#CR50" class="usa-link" aria-describedby="CR50">2020</a>; Zhao et al. <a href="#CR57" class="usa-link" aria-describedby="CR57">2021</a>; Alemu et al. <a href="#CR1" class="usa-link" aria-describedby="CR1">2024</a>). Prediction ability can be enhanced by increasing the training set size in proportion to the genetic diversity (Zhao et al. <a href="#CR57" class="usa-link" aria-describedby="CR57">2021</a>). This can be achieved cost-effectively by combining phenotypic and genotypic data from breeding programs and public–private partnerships into big data, as demonstrated by Zhao et al. (<a href="#CR57" class="usa-link" aria-describedby="CR57">2021</a>) and Lell et al. (<a href="#CR25" class="usa-link" aria-describedby="CR25">2025</a>). However, in these studies, utilization of test sets drawn from the accumulated data complicates the ability to determine the extent to which a comprehensive training set including a substantial number of early-stage genotypes from breeding programs can accurately predict the true genetic values of those genotypes, especially when accounting for genotype-by-environment interactions. Further, the improvement in the prediction ability appears to be saturated well before reaching the expected maximum defined by the square root of the heritability (Lell et al. <a href="#CR25" class="usa-link" aria-describedby="CR25">2025</a>). One possible explanation for this observation is the moderate heritability of the test datasets used, which typically is based on a limited number of environments. Furthermore, these imprecise test datasets also hamper efforts to identify additional factors that could enhance prediction ability. These reasons underscore the need for using test datasets where diverse genotypes are tested in a wide range of environments.</p>
<p id="Par5">In addition to the need for appropriate test datasets, the impact of merging heterogeneous SNP array platforms, which results in a block-like missing value structure (Lell et al. <a href="#CR25" class="usa-link" aria-describedby="CR25">2025</a>), remains understudied. The integration of disparate genotypic data sets introduces a significant number of non-random missing values and minor alleles, ultimately compromising the overall quality of the genotypic data. This limitation hinders the potential for achieving optimal prediction abilities. Therefore, there is a need for a robust strategy to filter out missing values and minor alleles.</p>
<p id="Par6">Moreover, several studies have shown that the inclusion of genome-wide epistasis can improve the prediction ability, and this magnitude of improvement can range from 4 to 25%, depending on the crop, population and trait under consideration (Crossa et al. <a href="#CR10" class="usa-link" aria-describedby="CR10">2010</a>; Jiang and Reif <a href="#CR22" class="usa-link" aria-describedby="CR22">2015</a>; Raffo et al. <a href="#CR37" class="usa-link" aria-describedby="CR37">2022</a>). Whereas other studies demonstrated that genome-wide epistasis can be detrimental to the prediction ability (Lorenzana and Bernardo <a href="#CR27" class="usa-link" aria-describedby="CR27">2009</a>). Further, Santantonio et al. (<a href="#CR42" class="usa-link" aria-describedby="CR42">2019</a>) and Cuevas et al. (<a href="#CR11" class="usa-link" aria-describedby="CR11">2024</a>) modelled epistasis within the subgenome level and found no improvement compared to modelling genome-wide epistasis. However, in most of these studies, the conclusions are drawn based on a single population, but the impact of epistasis on genomic predictions at the scale of big data, particularly when integrating diverse populations, remains unexplored.</p>
<p id="Par7">In this study, the existing wheat big data set was expanded (Zhao et al. <a href="#CR57" class="usa-link" aria-describedby="CR57">2021</a>; Lell et al. <a href="#CR24" class="usa-link" aria-describedby="CR24">2024</a>), resulting in data from approximately 18,000 wheat lines evaluated in approximately 250,000 yield plots. The objective was to combine it with a very powerful test population from post-regional trials to assess the potential of big data. Specifically, our objectives were to: (1) develop a strategy for filtering missing values and minor alleles resulting from the merging of unbalanced genotypic data from disparate sources to reach optimum prediction ability; (2) identify the key drivers of prediction ability in genomic predictions; and (3) examine the value of incorporating genome-wide, subgenome, and chromosome-level epistasis, in addition to additive effects, in genomic predictions at the level of big data.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Plant materials</h3>
<p id="Par8">The genotypic and phenotypic data from which the training populations were assembled came from eight large-scale experimental series of winter wheat conducted in Central Europe (Exp-1 to 8). The genotypic and phenotypic data of the first four series (Exp-1 to 4) have been described in detail previously (Zhao et al. <a href="#CR56" class="usa-link" aria-describedby="CR56">2015</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">2021</a>; Gogna et al. <a href="#CR15" class="usa-link" aria-describedby="CR15">2022</a>). Exp-5 to 8 comprise genotypic and phenotypic data from four wheat breeding programs generated in 2020 and 2021 (Lell et al. <a href="#CR25" class="usa-link" aria-describedby="CR25">2025</a>). The data has been extended by ~ 9000 lines phenotyped in ~ 50,000 plots in 2022 and 2023 as described in detail below.</p>
<p id="Par9">Exp-5 consists of 3211 winter wheat lines that have been tested by KWS LOCHOW GmbH (Bergen, Germany) for grain yield and plant height at 5 to 9 locations in Germany and Poland from 2020 to 2023. Each year, the breeding company performed nine trials with 56 to 154 genotypes per trial and one to two replications. The trials were connected by 10 to 11 registered lines. Plot size ranged between 5 to 15 m<sup>2</sup>.</p>
<p id="Par10">Exp-6 comprises 3906 winter wheat lines evaluated by Deutsche Saatveredelung AG (Lippstadt, Germany) for grain yield and plant height at up to 20 locations in Germany from 2020 to 2023. Each year, the breeding company performed 12 to 17 trials with 25 to 188 genotypes per trial and one to two replications. The trials were connected by up to 5 released lines. Plot size ranged between 5.25 to 18 m<sup>2</sup>.</p>
<p id="Par11">Exp-7 is based on 2918 winter wheat lines tested for grain yield and plant height in the Nordsaat Saatzucht GmbH (Langenstein, Germany) breeding program from 2020 to 2023. The lines were evaluated in 4 to 17 trials at up to 11 locations in Germany. The trials were connected with 6 to 18 released varieties. Between 50 to 281 lines were tested per trial with one to two replicates. Plot size ranged from 3.75 to 14.3 m<sup>2</sup>.</p>
<p id="Par12">Exp-8 is based on 8394 winter wheat lines, which have been evaluated by W. von Borries-Eckendorf GmbH &amp; Co. KG (Leopoldshöhe, Germany) for grain yield and plant height at up to 12 sites in 4 to 23 trials from 2020 to 2023. Trials were connected with up to 7 released varieties. Per trial, 49 to 1115 lines were tested in one or two replications. Plot size ranged from 5.8 to 12 m<sup>2</sup>.</p>
<p id="Par13">As a test set, we utilized phenotypic data from the years 2017 to 2022 obtained from the German post-registration winter wheat trials (Exp-PRT; Supplementary Table 1; Bundessortenversuch <a href="#CR58" class="usa-link" aria-describedby="CR58">2017</a>), along with the corresponding genotypic profiles. The Exp-PRT data includes grain yield trials across Germany at 26 to 31 locations (Supplementary Fig. 1). In total, the dataset represents approximately 170 environments, where each environment is defined as a unique combination of year and location. Around 20 genotypes were tested each year with intensive and extensive treatments. In our study, we used only data from the intensive treatment, which included treatments with fungicides and growth regulators, depending on local conditions. The integrated Exp-PRT data set consists of 98 genotypes: 90 inbred lines and 8 hybrids. Genotypic and phenotypic data was available for 83 of the 90 inbred lines. The EXP-PRT dataset becomes highly topical and commercially relevant for the industry due to the genotypes tested in vast number of environments across Germany and representing majority of the current German seed propagation area.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Phenotypic data curation and analysis</h3>
<p id="Par14">We implemented an un-weighted two-stage analysis of the phenotypic data. This decision is based on previous findings showing that the difference between weighted versus unweighted approaches was negligible (Möhring and Piepho <a href="#CR32" class="usa-link" aria-describedby="CR32">2009</a>). For Exp-1–4 and Exp-PRT, curated data was used (Gogna et al. <a href="#CR15" class="usa-link" aria-describedby="CR15">2022</a>; Zhao et al. <a href="#CR57" class="usa-link" aria-describedby="CR57">2021</a>), and within-environment Best Linear Unbiased Estimates (BLUEs) were available. Given the heterogeneity of the experimental designs in Exp-5–8, linear mixed models tailored to each experimental design derived from model (1) were used to calculate BLUEs for each environment for both traits, grain yield and plant height:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e496" display="block"><mrow><msub><mi>y</mi><mrow><mi mathvariant="italic">ijkl</mi></mrow></msub><mo>∼</mo><mi>μ</mi><mo>+</mo><msub><mi>g</mi><mi>i</mi></msub><mo>+</mo><msub><mi>t</mi><mi>j</mi></msub><mo>+</mo><msub><mi>r</mi><mrow><mi mathvariant="italic">jk</mi></mrow></msub><mo>+</mo><msub><mi>b</mi><mrow><mi mathvariant="italic">jkl</mi></mrow></msub><mo>+</mo><msub><mi>ε</mi><mrow><mi mathvariant="italic">ijkl</mi></mrow></msub><mo>,</mo></mrow></math></td>
<td class="label">1</td>
</tr></table>
<p id="Par15">where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e541"><mi>y</mi></math></span>
<em>is the plot level trait measurement,</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e553"><mi>μ</mi></math></span> is the mean, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e562"><msub><mi>g</mi><mi>i</mi></msub></math></span> is the effect of <em>i</em>th genotype, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e577"><msub><mi>t</mi><mi>j</mi></msub></math></span> is the <em>j</em>th trial effect, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e591"><msub><mi>r</mi><mrow><mi mathvariant="italic">jk</mi></mrow></msub></math></span> refers to the effect of <em>k</em>th replication in <em>j</em>th trial, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e610"><msub><mi>b</mi><mrow><mi mathvariant="italic">jkl</mi></mrow></msub></math></span> indicates the effect of <em>l</em>th block in the <em>k</em>th replication in the <em>j</em>th trial and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e633"><mi>ε</mi></math></span> relates to the residuals. Further, all the effects except for the <em>genotype</em> are fitted random. Outlier correction was performed using method 4 “Bonferroni-Holm with re-scaled median absolute deviation standardized residuals” as described by Bernal-Vasquez, Utz, and Piepho (<a href="#CR60" class="usa-link" aria-describedby="CR60">2016</a>).</p>
<p id="Par16">Based on intra-environmental BLUEs, across-environmental Best Linear Unbiased Predictions (BLUPs) were computed for each experimental series, as well as for all the combined datasets using the following model (<a href="#Equ2" class="usa-link">2</a>):</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e654" display="block"><mrow><msub><mi>y</mi><mrow><mi mathvariant="italic">ij</mi></mrow></msub><mo>∼</mo><mi>μ</mi><mo>+</mo><msub><mi>g</mi><mi>i</mi></msub><mo>+</mo><msub><mi>e</mi><mi>j</mi></msub><mo>+</mo><msub><mi>ε</mi><mrow><mi mathvariant="italic">ij</mi></mrow></msub><mo>,</mo></mrow></math></td>
<td class="label">2</td>
</tr></table>
<p id="Par17">In model (2), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e686"><mi>y</mi></math></span>
<em>are the</em> intra-environmental BLUEs<em>,</em>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e701"><mi>μ</mi></math></span> is the mean, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e710"><mi>g</mi></math></span> are the genotype effects, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e720"><mi>e</mi></math></span> are the environment (field × location × year) effects, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e729"><mi>ε</mi></math></span> relates to the residuals. Further, all the effects are fitted as random. The estimated variance components were used to calculate broad-sense heritability using the following model (<a href="#Equ3" class="usa-link">3</a>):</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e742" display="block"><mrow><msup><mrow><mi>H</mi></mrow><mn>2</mn></msup><mo>=</mo><msubsup><mi>σ</mi><mrow><mi>g</mi></mrow><mn>2</mn></msubsup><mo stretchy="false">/</mo><mfenced close=")" open="("><msubsup><mi>σ</mi><mrow><mi>g</mi></mrow><mn>2</mn></msubsup><mo>+</mo><mfrac><msubsup><mi>σ</mi><mrow><mi>ε</mi></mrow><mn>2</mn></msubsup><msub><mi>N</mi><mrow><mi mathvariant="italic">Env</mi></mrow></msub></mfrac></mfenced><mo>,</mo></mrow></math></td>
<td class="label">3</td>
</tr></table>
<p>In model (<a href="#Equ3" class="usa-link">3</a>), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e785"><msubsup><mi>σ</mi><mrow><mi>g</mi></mrow><mn>2</mn></msubsup></math></span> being the genotypic variance, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e799"><msubsup><mi>σ</mi><mrow><mi>ε</mi></mrow><mn>2</mn></msubsup></math></span> refers to the residual variance, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e812"><msub><mi>N</mi><mrow><mi mathvariant="italic">Env</mi></mrow></msub></math></span> the average number of environments a genotype was measured in. This calculated broad-sense heritability (H<sup>2</sup>) does not correspond to the fundamental heritability defined in quantitative genetics. Please note that, due to potential deviations from underlying assumptions such as random mating and Hardy–Weinberg equilibrium, Bernardo (<a href="#CR4" class="usa-link" aria-describedby="CR4">2020</a>) suggested using the term “reliability” instead of “broad-sense heritability” for greater clarity. Of course, the concept of “reliability” as defined by Bernardo (<a href="#CR4" class="usa-link" aria-describedby="CR4">2020</a>) should not be confused with the more commonly used interpretation of “reliability,” which refers to the bias in predicting an individual’s genetic value based solely on genomic data.</p>
<p id="Par18">All the calculations were made using the R Statistical Software (v4.0.1; R Core Team <a href="#CR59" class="usa-link" aria-describedby="CR59">2021</a>) and the “AsReml” package (v4.1.0; David Butler <a href="#CR3" class="usa-link" aria-describedby="CR3">2019</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Genomic data integration and analysis</h3>
<p id="Par19">The genotypic data was generated using SNP arrays of different densities ranging from 7000 to 90,000 markers derived from the publicly available 90k SNP array (Wang et al. <a href="#CR47" class="usa-link" aria-describedby="CR47">2014</a>). After merging the SNP array data across all experimental series, the resulting dataset contained 84,754 SNP markers for 28,606 genotypes. Markers without SNP calls, lacking variation, unmapped markers, and those with conflicting positional information were excluded. Genotypes without phenotypic measurements were excluded (Supplementary Table 1). After these filters, a refined set of 30,175 SNP markers for 18,211 genotypes remained. Markers with less than 5% minor allele frequency (MAF) and more than 80% missing values were also excluded, resulting in SNP profiles for 13,105 markers and 18,211 genotypes. Missing values were imputed using the Beagle software (v5.0, Browning et al. <a href="#CR7" class="usa-link" aria-describedby="CR7">2018</a>). The Rogers’ distance (Rogers <a href="#CR41" class="usa-link" aria-describedby="CR41">1972</a>) was calculated among all genotypes and a principal coordinate analysis (PCoA) (Gower <a href="#CR16" class="usa-link" aria-describedby="CR16">1966</a>) was performed using the “cmdscale” function of the R package “stats” (R Core Team <a href="#CR59" class="usa-link" aria-describedby="CR59">2021</a>). The effective population size <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e861"><msub><mi>N</mi><mi>e</mi></msub></math></span> (Hill <a href="#CR20" class="usa-link" aria-describedby="CR20">1981</a>; Waples <a href="#CR49" class="usa-link" aria-describedby="CR49">2006</a>) for different experimental series, and the integrated dataset were calculated.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Genome-wide prediction models</h3>
<p id="Par20">We used an extended genomic best linear unbiased prediction model that includes additive and epistasis effects (A + E model; <a href="#Equ4" class="usa-link">4</a>) (Jiang &amp; Reif <a href="#CR22" class="usa-link" aria-describedby="CR22">2015</a>). In addition, we tested a model that considers only additive effects (A model). The full model is as follows (<a href="#Equ4" class="usa-link">4</a>):</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e892" display="block"><mrow><mi>y</mi><mo>=</mo><mi mathvariant="normal">μ</mi><mo>+</mo><msub><mi>g</mi><mi>A</mi></msub><mo>+</mo><msub><mi>g</mi><mi>E</mi></msub><mo>+</mo><mi>ε</mi><mo>,</mo></mrow></math></td>
<td class="label">4</td>
</tr></table>
<p id="Par21">where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e917"><mi>y</mi></math></span> is the vector of across-environment genotype BLUPs, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e926"><mi>μ</mi></math></span> is the intercept, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e935"><msub><mi>g</mi><mi>A</mi></msub></math></span> is the vector of additive genetic values, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e946"><msub><mi>g</mi><mi>E</mi></msub></math></span> is the vector of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e957"><mrow><mi>a</mi><mi>d</mi><mi>d</mi><mi>i</mi><mi>t</mi><mi>i</mi><mi>v</mi><mi>e</mi><mo>×</mo><mi>a</mi><mi>d</mi><mi>d</mi><mi>i</mi><mi>t</mi><mi>i</mi><mi>v</mi><mi>e</mi></mrow></math></span> epistatic genetic values, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e985"><mi>ε</mi></math></span> is the vector of residuals. The random vector <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e994"><mrow><msub><mi>g</mi><mi>A</mi></msub><mo>=</mo><mrow><mo stretchy="false">(</mo><msub><mi>g</mi><mrow><mi>A</mi><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>g</mi><mrow><mi>A</mi><mn>2</mn></mrow></msub><mo>,</mo><mo>⋯</mo><mo>,</mo><msub><mi>g</mi><mrow><mi mathvariant="italic">An</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mrow></math></span> follows a normal distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e1031"><mrow><msub><mi>g</mi><mi>A</mi></msub><mo>∼</mo><mi>N</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msub><mi>G</mi><mi>A</mi></msub><msubsup><mi>σ</mi><mrow><mi>A</mi></mrow><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow></mrow></math></span>; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e1060"><msubsup><mi>σ</mi><mrow><mi>A</mi></mrow><mn>2</mn></msubsup></math></span> is the additive genetic variance. The covariance matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e1073"><msub><mi>G</mi><mi>A</mi></msub></math></span> of additive genetic effects was calculated according to VanRaden (<a href="#CR36" class="usa-link" aria-describedby="CR36">2008</a>; “first method”). The random vector <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e1088"><mrow><msub><mi>g</mi><mi>E</mi></msub><mo>=</mo><mrow><mo stretchy="false">(</mo><msub><mi>g</mi><mrow><mi>E</mi><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>g</mi><mrow><mi>E</mi><mn>2</mn></mrow></msub><mo>,</mo><mo>⋯</mo><mo>,</mo><msub><mi>g</mi><mrow><mi mathvariant="italic">En</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mrow></math></span> follows a normal distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e1125"><mrow><msub><mi>g</mi><mi>E</mi></msub><mo>∼</mo><mi>N</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msub><mi>G</mi><mi>E</mi></msub><msubsup><mi>σ</mi><mrow><mi>E</mi></mrow><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow></mrow></math></span>; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e1154"><msubsup><mi>σ</mi><mrow><mi>E</mi></mrow><mn>2</mn></msubsup></math></span> is the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e1167"><mrow><mi>a</mi><mi>d</mi><mi>d</mi><mi>i</mi><mi>t</mi><mi>i</mi><mi>v</mi><mi>e</mi><mo>×</mo><mi>a</mi><mi>d</mi><mi>d</mi><mi>i</mi><mi>t</mi><mi>i</mi><mi>v</mi><mi>e</mi></mrow></math></span> epistatic genetic variance. The covariance matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e1194"><msub><mi>G</mi><mi>E</mi></msub></math></span> of epistatic genetic effects was calculated as the Hadamard product of the additive relationship matrix (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><math id="d33e1205"><msub><mi>G</mi><mi>A</mi></msub></math></span>) as suggested by Henderson (<a href="#CR19" class="usa-link" aria-describedby="CR19">1985</a>) and further proved by Jiang and Reif (<a href="#CR22" class="usa-link" aria-describedby="CR22">2015</a>).</p>
<p id="Par22">We investigated the relevance of considering epistasis in genome-wide predictions in more detail and divided it into subgenomic (<a href="#Equ5" class="usa-link">5</a>-<a href="#Equ8" class="usa-link">8</a>) and chromosomal levels as follows (<a href="#Equ9" class="usa-link">9</a>-<a href="#Equ12" class="usa-link">12</a>):</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><math id="d33e1238" display="block"><mrow><mi>y</mi><mo>=</mo><mi>μ</mi><mo>+</mo><msub><mi>g</mi><mi>A</mi></msub><mo>+</mo><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>G</mi></msub></mrow></msub><mo>+</mo><mi>ε</mi><mo>,</mo></mrow></math></td>
<td class="label">5</td>
</tr></table>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><math id="d33e1265" display="block"><mrow><mi>y</mi><mo>=</mo><mi>μ</mi><mo>+</mo><msub><mi>g</mi><mi>A</mi></msub><mo>+</mo><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>G</mi></msub></mrow></msub><mo>+</mo><mi>ε</mi><mo>,</mo></mrow></math></td>
<td class="label">6</td>
</tr></table>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><math id="d33e1293" display="block"><mrow><mi>y</mi><mo>=</mo><mi>μ</mi><mo>+</mo><msub><mi>g</mi><mi>A</mi></msub><mo>+</mo><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>G</mi></msub></mrow></msub><mo>+</mo><mi>ε</mi><mo>,</mo></mrow></math></td>
<td class="label">7</td>
</tr></table>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><math id="d33e1320" display="block"><mrow><mi>y</mi><mo>=</mo><mi>μ</mi><mo>+</mo><msub><mi>g</mi><mi>A</mi></msub><mo>+</mo><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>G</mi></msub></mrow></msub><mo>+</mo><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>G</mi></msub></mrow></msub><mo>+</mo><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>G</mi></msub></mrow></msub><mo>+</mo><mi>ε</mi><mo>,</mo></mrow></math></td>
<td class="label">8</td>
</tr></table>
<p id="Par23">Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><math id="d33e1365"><mi>y</mi></math></span> is the vector of across-environment genotype BLUPs, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><math id="d33e1374"><mi>μ</mi></math></span> is the intercept, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><math id="d33e1383"><msub><mi>g</mi><mi>A</mi></msub></math></span> is the vector of additive genetic values and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><math id="d33e1394"><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>G</mi></msub></mrow></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><math id="d33e1409"><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>G</mi></msub></mrow></msub></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><math id="d33e1425"><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>G</mi></msub></mrow></msub></math></span> are the vectors of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><math id="d33e1440"><mrow><mi>a</mi><mi>d</mi><mi>d</mi><mi>i</mi><mi>t</mi><mi>i</mi><mi>v</mi><mi>e</mi><mo>×</mo><mi>a</mi><mi>d</mi><mi>d</mi><mi>i</mi><mi>t</mi><mi>i</mi><mi>v</mi><mi>e</mi></mrow></math></span> epistatic genetic values of subgenome A, B, and D, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><math id="d33e1467"><mi>ε</mi></math></span> is the vector of residuals. The random vectors <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><math id="d33e1476"><msub><mi>g</mi><mi>A</mi></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><math id="d33e1487"><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>G</mi></msub></mrow></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><math id="d33e1502"><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>G</mi></msub></mrow></msub></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><math id="d33e1518"><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>G</mi></msub></mrow></msub></math></span> are assumed to follow a normal distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><math id="d33e1533"><mrow><msub><mi>g</mi><mi>A</mi></msub><mo>∼</mo><mi>N</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msub><mi>G</mi><mi>A</mi></msub><msubsup><mi>σ</mi><mrow><mi>A</mi></mrow><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><math id="d33e1562"><mrow><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>G</mi></msub></mrow></msub><mo>∼</mo><mi>N</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msub><mi>G</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>G</mi></msub></mrow></msub><msubsup><mi>σ</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>G</mi></msub></mrow><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><math id="d33e1602"><mrow><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>G</mi></msub></mrow></msub><mo>∼</mo><mi>N</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msub><mi>G</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>G</mi></msub></mrow></msub><msubsup><mi>σ</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>G</mi></msub></mrow><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><math id="d33e1642"><mrow><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>G</mi></msub></mrow></msub><mo>∼</mo><mi>N</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msub><mi>G</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>G</mi></msub></mrow></msub><msubsup><mi>σ</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>G</mi></msub></mrow><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow></mrow></math></span>; where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><math id="d33e1682"><msubsup><mi>σ</mi><mrow><mi>A</mi></mrow><mn>2</mn></msubsup></math></span> is the additive genetic variance and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><math id="d33e1696"><msubsup><mi>σ</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>G</mi></msub></mrow><mn>2</mn></msubsup></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><math id="d33e1712"><msubsup><mi>σ</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>G</mi></msub></mrow><mn>2</mn></msubsup></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><math id="d33e1728"><msubsup><mi>σ</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>G</mi></msub></mrow><mn>2</mn></msubsup></math></span> are the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><math id="d33e1744"><mrow><mi>a</mi><mi>d</mi><mi>d</mi><mi>i</mi><mi>t</mi><mi>i</mi><mi>v</mi><mi>e</mi><mo>×</mo><mi>a</mi><mi>d</mi><mi>d</mi><mi>i</mi><mi>t</mi><mi>i</mi><mi>v</mi><mi>e</mi></mrow></math></span> epistatic genetic variance of subgenomes A, B, and D, respectively. The related covariance matrices for subgenomes A, B, and D were calculated the same as in model (4), but using only the SNP markers of the respective subgenome. As a next step, we considered epistasis separately for each chromosome within each subgenome as follows (<a href="#Equ9" class="usa-link">9</a>-<a href="#Equ12" class="usa-link">12</a>):</p>
<table class="disp-formula p" id="Equ9"><tr>
<td class="formula"><math id="d33e1779" display="block"><mrow><mi>y</mi><mo>=</mo><mn>1</mn><mi>μ</mi><mo>+</mo><msub><mi>g</mi><mi>A</mi></msub><mo>+</mo><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>7</mn></munderover><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>i</mi></msub></mrow></msub><mo>+</mo><mi>ε</mi><mo>,</mo></mrow></math></td>
<td class="label">9</td>
</tr></table>
<table class="disp-formula p" id="Equ10"><tr>
<td class="formula"><math id="d33e1815" display="block"><mrow><mi>y</mi><mo>=</mo><mn>1</mn><mi>μ</mi><mo>+</mo><msub><mi>g</mi><mi>A</mi></msub><mo>+</mo><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>7</mn></munderover><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>i</mi></msub></mrow></msub><mo>+</mo><mi>ε</mi><mo>,</mo></mrow></math></td>
<td class="label">10</td>
</tr></table>
<table class="disp-formula p" id="Equ11"><tr>
<td class="formula"><math id="d33e1851" display="block"><mrow><mi>y</mi><mo>=</mo><mn>1</mn><mi>μ</mi><mo>+</mo><msub><mi>g</mi><mi>A</mi></msub><mo>+</mo><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>7</mn></munderover><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>i</mi></msub></mrow></msub><mo>+</mo><mi>ε</mi><mo>,</mo></mrow></math></td>
<td class="label">11</td>
</tr></table>
<table class="disp-formula p" id="Equ12"><tr>
<td class="formula"><math id="d33e1887" display="block"><mrow><mi>y</mi><mo>=</mo><mn>1</mn><mi>μ</mi><mo>+</mo><msub><mi>g</mi><mi>A</mi></msub><mo>+</mo><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>7</mn></munderover><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>i</mi></msub></mrow></msub><mo>+</mo><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>7</mn></munderover><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>i</mi></msub></mrow></msub><mo>+</mo><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>7</mn></munderover><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>i</mi></msub></mrow></msub><mo>+</mo><mi>ε</mi><mo>,</mo></mrow></math></td>
<td class="label">12</td>
</tr></table>
<p id="Par24">Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><math id="d33e1954"><mi>y</mi></math></span> is the vector of across-environment genotype BLUPs, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><math id="d33e1963"><mi>μ</mi></math></span> is the intercept, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><math id="d33e1972"><msub><mi>g</mi><mi>A</mi></msub></math></span> is the vector of additive genetic values, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><math id="d33e1983"><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>i</mi></msub></mrow></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><math id="d33e1998"><mrow><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>i</mi></msub></mrow></msub><mo>,</mo><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>i</mi></msub></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><math id="d33e2023"><mrow><mi>i</mi><mo>=</mo><mn>1</mn><mo>-</mo><mn>7</mn></mrow></math></span> are the vectors of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><math id="d33e2037"><mrow><mi>a</mi><mi>d</mi><mi>d</mi><mi>i</mi><mi>t</mi><mi>i</mi><mi>v</mi><mi>e</mi><mo>×</mo><mi>a</mi><mi>d</mi><mi>d</mi><mi>i</mi><mi>t</mi><mi>i</mi><mi>v</mi><mi>e</mi></mrow></math></span> epistatic genetic values of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><math id="d33e2064"><msup><mrow><mi>i</mi></mrow><mrow><mi mathvariant="italic">th</mi></mrow></msup></math></span> chromosome of the subgenomes A, B, and D. The random vectors <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><math id="d33e2078"><msub><mi>g</mi><mi>A</mi></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><math id="d33e2089"><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>i</mi></msub></mrow></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><math id="d33e2104"><mrow><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>i</mi></msub></mrow></msub><mo>,</mo><mi>a</mi><mi>n</mi><mi>d</mi><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>i</mi></msub></mrow></msub></mrow></math></span> are assumed to follow a normal distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><math id="d33e2132"><mrow><msub><mi>g</mi><mi>A</mi></msub><mo>∼</mo><mi>N</mi><mfenced close=")" open="("><mn>0</mn><mo>,</mo><msub><mi>G</mi><mi>A</mi></msub><msubsup><mi>σ</mi><mrow><mi>A</mi></mrow><mn>2</mn></msubsup></mfenced><mo>,</mo><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>i</mi></msub></mrow></msub><mo>∼</mo><mi>N</mi><mfenced close=")" open="("><mn>0</mn><mo>,</mo><msub><mi>G</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>i</mi></msub></mrow></msub><msubsup><mi>σ</mi><mrow><mi>E</mi><msub><mi>A</mi><mi>i</mi></msub></mrow><mn>2</mn></msubsup></mfenced><mo>,</mo><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>i</mi></msub></mrow></msub><mo>∼</mo><mi>N</mi><mfenced close=")" open="("><mn>0</mn><mo>,</mo><msub><mi>G</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>i</mi></msub></mrow></msub><msubsup><mi>σ</mi><mrow><mi>E</mi><msub><mi>B</mi><mi>i</mi></msub></mrow><mn>2</mn></msubsup></mfenced></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><math id="d33e2214"><mrow><msub><mi>g</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>i</mi></msub></mrow></msub><mo>∼</mo><mi>N</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msub><mi>G</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>i</mi></msub></mrow></msub><msubsup><mi>σ</mi><mrow><mi>E</mi><msub><mi>D</mi><mi>i</mi></msub></mrow><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow></mrow></math></span>. The related covariance matrices were calculated the same as in model (4), but using only the SNP markers of the <em>i</em>th chromosome of the subgenomes A, B, and D, respectively.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Scenarios for validating genome-wide predictions</h3>
<p id="Par25">In scenario 1, the A + E model (<a href="#Equ4" class="usa-link">4</a>) was used for the genomic predictions. The data of individual experimental series and combinations of them (Supplementary Table 2) were used as training sets to predict the performance of the Exp-PRT dataset. The Exp-PRT data was particularly useful as a test set because of the large number of environments used to evaluate grain yield. The prediction ability was computed as the correlation between the observed and predicted phenotypic values. For all the training sets the parameters like genotypic variance (σ<sup>2</sup><sub>g</sub>), broad-sense heritability (H<sup>2</sup>), the average number of test environments per genotype (N<sub>env</sub>), coefficient of variation of BLUPs (CV), population size (N), effective population size (N<sub>e</sub>), and the ratio N/N<sub>e</sub> were calculated. These parameters were then correlated with the prediction ability to identify associations between the parameters and the prediction ability.</p>
<p id="Par26">Scenario 2 was used exclusively to study the relevance of epistasis in the genome-wide predictions. We randomly sampled training sets of 500, 1,000, 2,500, 5,000, and 10,000 genotypes 100 times, used genomic prediction models 4–12, and validated the prediction accuracies based on the Exp-PRT dataset. All genotypes occurring in the Exp-PRT data set were excluded from the training data sets of both scenarios. All computations were executed using the R package “BGLR” (Pérez and de los Campos <a href="#CR35" class="usa-link" aria-describedby="CR35">2014</a>).</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec9"><h3 class="pmc_sec_title">Absence of genetically distinct subpopulations</h3>
<p id="Par27">The 18,210 genotypes included in our study cover a wide diversity of winter wheat with an effective population size of <em>N</em><sub><em>e</em></sub> = 91 individuals estimated from the SNP data (Supplementary Table 3). All datasets, except Exp-3, contain candidate elite varieties or released varieties developed by different breeding companies for Central Europe. In addition to elite varieties, Exp-3 also includes 239 plant genetic resources maintained at the IPK genebank (Gatersleben, Germany). Accordingly, the distribution of the Rogers’ distances within each experimental series showed a significantly (<em>p</em> &lt; 0.05) higher diversity for Exp-3 with an average distance of 0.37 (Exp-3) compared to the other datasets with average distances of 0.33 to 0.36 (Supplementary Table 4, Fig. <a href="#Fig1" class="usa-link">1</a>a). Further Exp-PRT showed a significantly (<em>p</em> &lt; 0.05) higher diversity than the Exp-5 to 8 (Supplementary Table 4, Fig. <a href="#Fig1" class="usa-link">1</a>a). In addition, the average pairwise Rogers’ distances between the genotypes of the different experimental series were significantly higher (<em>p</em> &lt; 0.05) at &gt; 0.36 for the comparisons with Exp-3 compared to most of the other pairwise comparisons (Fig. <a href="#Fig1" class="usa-link">1</a>b, Supplementary Table 5). The released varieties tested in the Exp-PRT trials showed similar distributions of Rogers’ distances with genotypes of different experimental series (Fig. <a href="#Fig1" class="usa-link">1</a>c), with a mean distance ranging from 0.35 to 0.37 (Supplementary Table 6), accompanied by an absence of distinct subpopulations within elite lines (Fig. <a href="#Fig1" class="usa-link">1</a>d). Overall, therefore, the use of the Exp-PRT trials as a test set should not bias the results from a genetic point of view and favor a particular composition of training sets.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373547_122_2025_5007_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9916/12373547/13bc92eca17c/122_2025_5007_Fig1_HTML.jpg" loading="lazy" id="MO1" height="913" width="708" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genetic diversity within and among experimental series (Exp). Distribution of Rogers’ distances <strong>a</strong> within Exp, <strong>b</strong> pairwise between all the other Exp except Exp-PRT, <strong>c</strong> pairwise between Exp-PRT and other Exp. The purple color boxes indicate all pairs involving the Exp-3. <strong>d</strong> Principal coordinate analysis (PCoA) based on the Rogers’ distance matrix of all the inbred lines. The percentage in the parenthesis refers to the proportion of genetic variance explained by the respective principal coordinate (PC). <strong>e</strong> Cumulative variance explained by the PCs</p></figcaption></figure></section><section id="Sec10"><h3 class="pmc_sec_title">Exp-PRT is particularly well suited for use as a test set</h3>
<p id="Par28">We assembled extensive grain yield and plant height data from ~ 256,000 plots. Individual genotypes were evaluated for grain yield and plant height on average in 4.8 and 4.5 environments across Central Europe, ranging from ~ 2 (Exp-8) to ~ 39 (Exp-PRT) environments (Supplementary Table 7). The estimated broad-sense heritability within the experimental series ranged from 0.61 (Exp-8) to 0.96 (Exp-PRT) for grain yield and 0.76 (Exp-6) to 0.99 (Exp-1) for plant height. For the analyses across the experimental series, the heritability was 0.80 for grain yield and 0.92 for plant height (Supplementary Fig. <a href="#Fig2" class="usa-link">2</a>a, Supplementary Table 7). The genotypic variances within the experimental series for grain yield and plant height were comparable across the series (Fig. <a href="#Fig2" class="usa-link">2</a>b-c, Supplementary Fig. <a href="#Fig2" class="usa-link">2</a>b-c, Supplementary Table 7), with one outlier from Exp-3 for grain yield and one outlier from Exp-1 for plant height. The pronounced genotypic variation, coupled with high to very high heritabilities for both traits, reflect the excellent quality of the phenotypic data underlying our study. In particular, the high heritabilities for the Exp-PRT dataset, with estimates of 0.96 for grain yield and 0.98 for plant height, make this an ideal test set.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373547_122_2025_5007_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9916/12373547/bf4d3aba18f5/122_2025_5007_Fig2_HTML.jpg" loading="lazy" id="MO2" height="491" width="676" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phenotypic diversity within and across experimental series (Exp). <strong>a</strong> Broad-sense heritabilities, <strong>b</strong> genotypic variance ((dt ha<sup>−1</sup>)<sup>2</sup>), and <strong>c</strong> distribution of best linear unbiased predictions of grain yield (dt ha<sup>−1</sup>) within and across Exp</p></figcaption></figure></section><section id="Sec11"><h3 class="pmc_sec_title">Merging data across experimental series increases the ability to predict grain yield and plant height</h3>
<p id="Par29">The prediction ability based on a single experimental series for the Exp-PRT genotypes ranged between 0.31 (Exp-1) to 0.55 (Exp-7) for grain yield and 0.48 (Exp-1) to 0.65 (Exp-5) for plant height (Supplementary Table 8). The integrated big data across the experimental series reached a prediction ability of 0.61 for grain yield and 0.69 for plant height (Supplementary Table 8). The minimum improvement in predictive ability when moving from a single Exp to an integrated big data approach for grain yield was 11.12% for Exp-7 and the maximum was 97.04% for Exp-1 (Fig. <a href="#Fig3" class="usa-link">3</a>a). For plant height, the improvement ranged from 6.09% for Exp-5 to 43.92% for Exp-1 (Fig. <a href="#Fig3" class="usa-link">3</a>b).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9916/12373547/9c5f3dfd3b81/122_2025_5007_Fig3_HTML.jpg" loading="lazy" id="MO3" height="808" width="645" alt="Fig. 3"></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gain in prediction ability when merging data across experimental series (Exp). <strong>a</strong> Grain yield and <strong>b</strong> plant height plots, the X-axis indicates the training set used to predict the Exp-PRT, and the Y-axis shows the prediction ability. The brown line indicates the prediction abilities with the integrated training set. The numbers above the line refer to the percentage of improvement in the prediction ability due to use of integrated training set compared to the corresponding single experimental series (for grain yield Exp-2 was not used for the comparison due to its small size, instead we used Exp-2 + 3)</p></figcaption></figure></section><section id="Sec12"><h3 class="pmc_sec_title">Increasing the sample size relative to the diversity of the underlying populations enhances the prediction ability</h3>
<p id="Par30">There was a strong positive association between prediction ability and N/N<sub>e</sub>, i.e. the ratio of the size of the training population and the effective population size, with Pearson moment correlation coefficients of r = 0.60 (<em>p</em> &lt; 0.001) for grain yield and r = 0.62 (<em>p</em> &lt; 0.001) for plant height (Fig. <a href="#Fig4" class="usa-link">4</a>a-b). The H<sup>2</sup> values exhibited positive correlation (r = 0.21) with the prediction ability for plant height but a significant negative correlation (r = − 0.49; <em>p</em> &lt; 0.001) was observed for grain yield (Fig. <a href="#Fig4" class="usa-link">4</a>a-b). The correlations between the prediction ability for grain yield and CV (r = − 0.57; <em>p</em> &lt; 0.001), σ<sup>2</sup><sub>g</sub> (r = − 0.58; <em>p</em> &lt; 0.001), and N<sub>env</sub> (r = − 0.52; <em>p</em> &lt; 0.001) were significantly negative, while the correlation between the prediction ability for plant height and σ<sup>2</sup><sub>g</sub> (r = −0.31; <em>p</em> &lt; 0.05) was significantly negative as well (Fig. <a href="#Fig4" class="usa-link">4</a>a–b). Nevertheless, the correlation between the prediction ability for plant height and both N<sub>env</sub> (r = 0) and CV (r = − 0.17) was not statistically significant and is relatively weak (Fig. <a href="#Fig4" class="usa-link">4</a>b).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373547_122_2025_5007_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9916/12373547/e8067c9c0aed/122_2025_5007_Fig4_HTML.jpg" loading="lazy" id="MO4" height="330" width="676" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Pearson correlation values between prediction ability (PA) of <strong>a</strong> grain yield and <strong>b</strong> plant height with the effective population size (N<sub>e</sub>), training set size (N), ratio of training set size to the effective population size (N/N<sub>e</sub>), coefficient of variation of BLUPs (CV), genotypic variance (σ<sup>2</sup><sub>g</sub>), broad-sense heritability (H<sup>2</sup>), and the average number of environments genotypes tested (N<sub>env</sub>), and the. The number of the *marks in colored tiles indicate the significance at respective thresholds</p></figcaption></figure></section><section id="Sec13"><h3 class="pmc_sec_title">Quality filtering for missing values and minor allele frequencies improves the prediction ability</h3>
<p id="Par31">For large training populations, filtering out markers with &gt; 50% missing values resulted in higher prediction abilities independent of filtering for minor allele frequency (Supplementary Table 9, Supplementary Table 10). For smaller training sets, removing markers with more than 80% missing values for grain yield and for plant height resulted in higher prediction abilities (Fig. <a href="#Fig5" class="usa-link">5</a>a–b, Supplementary Fig. <a href="#Fig3" class="usa-link">3</a>a-b). Regardless of the threshold for missing values and the size of the training set, filtering out minor alleles (at 5%) did not lead to better prediction than including minor alleles for either trait (Fig. <a href="#Fig5" class="usa-link">5</a>c–f, Supplementary Fig. <a href="#Fig3" class="usa-link">3</a>c–f). Thus, our data suggests the advantages of careful quality filtering.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373547_122_2025_5007_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9916/12373547/6017ab7f30bb/122_2025_5007_Fig5_HTML.jpg" loading="lazy" id="MO5" height="1017" width="708" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>For grain yield, a-b Prediction ability vs Training set size at different NA thresholds and <strong>a</strong> no minor allele filtering (Min MAF 0), <strong>b</strong> minor allele filtering at 5% (Min MAF 0.05). <strong>c</strong>–<strong>f</strong> Prediction ability vs Training set size at different minor allele frequency thresholds and missing values allowed per marker <strong>c</strong> up to 100%, <strong>d</strong> up to 80%, <strong>e</strong> up to 50%, and <strong>f</strong> up to 30%. The colored smoothing lines indicate respective minor allele frequency and missing value filtering thresholds employed and drawn using the formula y ~ a + b * log(x)</p></figcaption></figure></section><section id="Sec14"><h3 class="pmc_sec_title">The inclusion of epistasis improves the prediction ability given the training set size</h3>
<p id="Par32">The incorporation of genome-wide epistasis effects alongside additive effects enhances the prediction ability compared to the sole inclusion of additive effects and the magnitude of this improvement diminishes with the expansion of the training set size for both traits (Fig. <a href="#Fig6" class="usa-link">6</a>a-b). It was observed that fitting epistasis within subgenome A for grain yield and subgenome B for plant height consistently demonstrated inferior performance compared to fitting genome-wide epistasis. Fitting the within subgenome B epistasis alone as well as fitting epistasis within subgenome A, B, and D together yielded higher prediction abilities compared to fitting genome-wide epistasis for grain yield. But for plant height, fitting the within subgenome A epistasis alone as well as fitting epistasis within subgenome A, B, and D together yielded higher prediction abilities compared to fitting genome-wide epistasis at larger training set size (Fig. <a href="#Fig6" class="usa-link">6</a>c-d). However, none of the chromosome-level epistasis models outperforms the genome-wide epistasis model for grain yield and for plant height epistasis within chromosome A, and fitting epistasis within chromosome A, B, and D together produced higher prediction abilities compared to fitting genome-wide epistasis (Fig. <a href="#Fig6" class="usa-link">6</a>e-f).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373547_122_2025_5007_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9916/12373547/d4826dbe0eb8/122_2025_5007_Fig6_HTML.jpg" loading="lazy" id="MO6" height="906" width="708" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>In all the plots, X-axis indicates the training set size and Y-axis indicate the mean prediction ability. <strong>a</strong>–<strong>b</strong> comparison between the additive model (A) and additive plus epistasis model (A + E) for <strong>a</strong> grain yield and <strong>b</strong> plant height. <strong>c</strong>–<strong>d</strong> comparison between the additive plus epistasis model (A + E) and additive plus subgenome epistasis models (A + sgA, sgB, sgD) for <strong>c</strong> grain yield and <strong>d</strong> plant height. <strong>e</strong>–<strong>f</strong> comparison between the additive plus epistasis model (A + E) and additive plus chromosome epistasis models (A + chrA, chrB, chrD) for <strong>e</strong> grain yield and <strong>f</strong> plant height. The color of the lines indicates the different models used, and at each point standard error is indicated for over 100 runs</p></figcaption></figure></section></section><section id="Sec15"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par33">Zhao et al. (<a href="#CR57" class="usa-link" aria-describedby="CR57">2021</a>) and Lell et al. (<a href="#CR25" class="usa-link" aria-describedby="CR25">2025</a>) demonstrated that combining small to medium datasets from public and private breeding programs improved the prediction ability for wheat hybrids and inbred lines. However, the test sets used were drawn from the combined dataset, limiting the generalizability of their results. In this study, we expanded Lell et al.'s dataset to 18,000 genotypes and validated the potential of big data using the Exp-PRT as a test set. The Exp-PRT contains varieties from different breeding companies and were tested at many locations representing diverse environments in Germany (Supplementary Fig. <a href="#Fig1" class="usa-link">1</a>). Our validation yielded reliable and broadly applicable results, with a focus on genomic selection for improved varieties. However, we did not explore the potential of the comprehensive training data for parental selection, as this would require detailed information about the underlying pedigree of the inbred lines included (Rembe et al. <a href="#CR39" class="usa-link" aria-describedby="CR39">2022</a>).</p>
<p id="Par34">We found that the integrated training set improved the prediction ability by up to 97% for grain yield and 44% for plant height in the Exp-PRT dataset; outperforming individual training sets (Fig. <a href="#Fig3" class="usa-link">3</a>a-b). Furthermore, it was observed that no dataset for both traits exhibited a prediction ability comparable to that of the integrated dataset (Fig. <a href="#Fig3" class="usa-link">3</a>a-b). This highlights the power of big data to enhance genomic prediction and accelerate genetic gain compared to phenotypic selection (Heffner et al. <a href="#CR18" class="usa-link" aria-describedby="CR18">2010</a>; Zhang et al. <a href="#CR55" class="usa-link" aria-describedby="CR55">2019</a>). However, data sharing between the companies remains a challenge due to confidentiality concerns. A well-defined data-sharing ecosystem is essential to overcome this, along with incorporating multi-omic and environmental data, high-throughput phenotyping, and AI-driven genomic models for further improvement (Xu et al. <a href="#CR53" class="usa-link" aria-describedby="CR53">2022</a>; Wang et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2023</a>; Alemu et al. <a href="#CR1" class="usa-link" aria-describedby="CR1">2024</a>; Kaushal et al. <a href="#CR23" class="usa-link" aria-describedby="CR23">2024</a>).</p>
<p id="Par35">For grain yield, prediction ability plateaus at around 5000 genotypes in our study (Fig. <a href="#Fig6" class="usa-link">6</a>a). Similarly, Lell et al. (<a href="#CR25" class="usa-link" aria-describedby="CR25">2025</a>) observed a plateau at ~ 4000 genotypes using 9,000 genotypes. With 18,000 genotypes, the plateau shifts higher, indicating that saturation depends on training set size. This trend is specific to the Central European winter wheat pool; more diverse panels may plateau at different points. Notably, this pattern reflects EGBLUP performance, whereas AI models like CNNs may require even larger datasets (Lell et al. <a href="#CR25" class="usa-link" aria-describedby="CR25">2025</a>). As increased training size shows no negative impact, we recommend using the full dataset to maximize prediction ability.</p>
<p id="Par36">We also examined the parameters of the cumulative datasets that drive the prediction ability. The strong associations between the prediction ability and the parameters training set size (N), effective population size (N<sub>e</sub>), and N/N<sub>e</sub> align with previous studies (Habier et al. <a href="#CR17" class="usa-link" aria-describedby="CR17">2007</a>; Lorenzana and Bernardo <a href="#CR27" class="usa-link" aria-describedby="CR27">2009</a>; Meuwissen <a href="#CR30" class="usa-link" aria-describedby="CR30">2009</a>; Norman et al. <a href="#CR33" class="usa-link" aria-describedby="CR33">2018</a>; Zhao et al. <a href="#CR57" class="usa-link" aria-describedby="CR57">2021</a>; and Lell et al. <a href="#CR25" class="usa-link" aria-describedby="CR25">2025</a>) and justify the superiority of integrated training sets. However, the observed negative correlation between the heritability and its positively associated parameters (CV, N<sub>env</sub>, σ<sup>2</sup><sub>g</sub>) with the prediction ability for grain yield contradicts previous findings (Combs and Bernardo <a href="#CR9" class="usa-link" aria-describedby="CR9">2013</a>; Lian et al. <a href="#CR26" class="usa-link" aria-describedby="CR26">2014</a>; Zhang et al. <a href="#CR61" class="usa-link" aria-describedby="CR61">2017</a>). This can be explained by the confounding genotype-by-environment interactions observed in the historical experimental series Exp-1 to 4 and the presence of plant genetic resources in Exp-3. Since these experimental series happen to have higher values for the H<sup>2</sup>, CV, N<sub>env</sub>, and σ<sup>2</sup><sub>g</sub>, their combination with the rest of the datasets inflates these parameters and decreases the prediction ability (Dawson et al. <a href="#CR13" class="usa-link" aria-describedby="CR13">2013</a>; Jarquín et al. <a href="#CR21" class="usa-link" aria-describedby="CR21">2014</a>; Berro et al. <a href="#CR5" class="usa-link" aria-describedby="CR5">2019</a>; Rogers and Holland <a href="#CR40" class="usa-link" aria-describedby="CR40">2022</a>) compared to the other combinations, leading to negative correlations (Supplementary Table 12). Additionally, datasets with lower H<sup>2</sup> but larger sizes show higher prediction ability and lead to negative correlations (Daetwyler et al. <a href="#CR12" class="usa-link" aria-describedby="CR12">2008</a>) (Supplementary Table 12). Furthermore, the negative correlation between the prediction ability and the other parameters (CV, N<sub>env</sub>, and σ<sup>2</sup><sub>g</sub>) with regard to plant height can be attributed to Exp-1 and Exp-3 with their divergent parameter values. Consequently, when these datasets are merged with the other datasets, the resulting combined datasets exhibit inflated or deflated values of these parameters, resulting in negative correlations (Supplementary Table 12). In summary, increasing the size of the dataset relative to diversity may enhance prediction ability, although correlations should be interpreted with caution.</p>
<p id="Par37">The handling of missing values and minor alleles in heterogeneous SNP data is crucial for maintaining high genotypic data quality and ensuring optimal prediction ability. Our analysis of four missing value thresholds and two minor allele frequency thresholds (Supplementary Table 9) demonstrated that the inclusion of minor alleles does not harm prediction ability. In fact, the inclusion of minor alleles can enhance prediction ability, particularly when the training set size is substantial (Fig. <a href="#Fig5" class="usa-link">5</a>c-f, Supplementary Fig. <a href="#Fig3" class="usa-link">3</a>c-f), due to the increased power to detect QTL associated with the minor alleles (Zakieh et al. <a href="#CR54" class="usa-link" aria-describedby="CR54">2023</a>; Lell et al. <a href="#CR24" class="usa-link" aria-describedby="CR24">2024</a>). Additionally, our findings indicate that excluding low-quality markers by strict filtering (30%) does not enhance the prediction ability. The potential reason for this could be due to uneven marker genome coverage (Daetwyler et al. <a href="#CR12" class="usa-link" aria-describedby="CR12">2008</a>), insufficient genotype differentiation, or removal of markers linked to QTLs (Ali et al. <a href="#CR2" class="usa-link" aria-describedby="CR2">2020</a>). Conversely, using all markers without filtering also decreases the overall quality of the integrated SNP dataset (Ali et al. <a href="#CR2" class="usa-link" aria-describedby="CR2">2020</a>). Therefore, we recommend a threshold of 50% missing values as a balance between marker number and quality to ensure even genome coverage (Fig. <a href="#Fig5" class="usa-link">5</a>a-b and Supplementary Fig. <a href="#Fig3" class="usa-link">3</a>a-b). These guidelines apply specifically to quality control of integrated SNP datasets with heterogeneous marker density and genotyping platforms.</p>
<p id="Par38">As the application of theoretical quantitative genetics assumptions such as random mating and Hardy–Weinberg equilibrium is often violated in breeding populations, the use of classical broad-sense heritability as a benchmark can lead to biased estimations of the empirical upper limit of predictive ability (Bernardo <a href="#CR4" class="usa-link" aria-describedby="CR4">2020</a>). Therefore, in this study broad-sense heritability was defined in practical terms, and its square root was used as an empirical approximation of the maximum achievable prediction ability. With the availability of big data, we now achieved a predictive power 0.61 for grain yield, which is highly promising. Given the high heritability of Exp-PRT (&gt; 0.96), the genetic variance explained by the predicted value is around 36%. This indicates that about 60% of the genetic variance remains unexplained, highlighting the issue of missing heritability (Manolio et al. <a href="#CR29" class="usa-link" aria-describedby="CR29">2009</a>). A wide range of explanations has been proposed to explain missing heritability, including factors such as lack of modelling non-additive effects, inadequate population size, uneven genome coverage, rare genetic variants, undetected copy number variation effects and over-estimated heritability (Clarke and Cooper <a href="#CR8" class="usa-link" aria-describedby="CR8">2010</a>; Park et al. <a href="#CR34" class="usa-link" aria-describedby="CR34">2010</a>; Visscher et al. <a href="#CR45" class="usa-link" aria-describedby="CR45">2006</a>; Bodmer and Tomlinson <a href="#CR6" class="usa-link" aria-describedby="CR6">2010</a>; Forer et al. <a href="#CR14" class="usa-link" aria-describedby="CR14">2010</a>; Makowsky et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2011</a>). In light of this, we considered different epistasis models to explore hidden genetic mechanisms and assess whether they can help to explain any of the 60% of heritability. It is important to note that the epistatic models employed in this study are non-orthogonal, which can potentially introduce bias in the estimation of additive and epistatic variance components (Raffo et al. <a href="#CR37" class="usa-link" aria-describedby="CR37">2022</a>). However, non-orthogonal models can achieve predictive abilities comparable to those of orthogonal models (Jiang and Reif <a href="#CR22" class="usa-link" aria-describedby="CR22">2015</a>). Given that our primary aim was to evaluate the contribution of epistasis to prediction ability, rather than to obtain precise variance component estimates, we consider the use of non-orthogonal models to be justified in this context.</p>
<p id="Par39">Interestingly, the impact of epistasis on prediction ability varies across crops and even within the same crop depending on the population and trait under consideration (Crossa et al. <a href="#CR10" class="usa-link" aria-describedby="CR10">2010</a>; Jiang and Reif <a href="#CR22" class="usa-link" aria-describedby="CR22">2015</a>; Raffo et al. <a href="#CR37" class="usa-link" aria-describedby="CR37">2022</a>). This study shows that the improvement in prediction ability resulting from genome-wide epistasis is contingent upon the genetic architecture of the trait and the size of the training population. Predicting polygenic traits such as grain yield benefits more from modelling epistasis than oligogenic such as plant height. However, this improvement decreases as the training population size increases, probably due to fewer spurious linkage disequilibrium (LD) between unlinked markers (Supplementary Table 13). Smaller populations exhibit inflated LD, detecting more interactions and leading to greater prediction improvements (Supplementary Table 13) (Schrauf et al. <a href="#CR43" class="usa-link" aria-describedby="CR43">2020</a>).</p>
<p id="Par40">Previously, Santantonio et al. (<a href="#CR42" class="usa-link" aria-describedby="CR42">2019</a>) and Cuevas et al. (<a href="#CR11" class="usa-link" aria-describedby="CR11">2024</a>) suggested that modelling within subgenome and between subgenome epistatic interaction effects can provide similar prediction abilities as modelling genome-wide epistatic interaction effects. In contrast to the models previously described, our findings show that models incorporating within-subgenome epistasis combined with additive effects outperform the genome-wide epistasis model. Furthermore, incorporating chromosome-level interaction effects did not improve prediction ability for grain yield. Further research is needed to explore the role of subgenome and chromosome level interactions in improving prediction ability. Furthermore, predictions based on both additive and epistatic effects should be used to select the best-performing genotypes, but not for selecting superior parents. This is because epistatic effects are not heritable, and predicted values that include both additive and epistatic components may rank genotypes differently compared to those based solely on additive (heritable) effects (Raffo et al. <a href="#CR37" class="usa-link" aria-describedby="CR37">2022</a>). In summary, it is strongly recommended that epistasis effects be employed in genomic prediction for wheat, regardless of whether they are apparent or real, particularly for polygenic trait predictions for the purpose of varietal release.</p></section><section id="Sec16"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373547/bin/122_2025_5007_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file1 (PDF 1176 KB)</a><sup> (1.1MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors acknowledge funding within the Wheat BigData Project (German Federal Ministry of Food and Agriculture, FKZ2818408B18). The authors thank Alexandre Pinheiro and Uwe Scholz for their contributions to a smooth data curation and transmission.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>RRG, YZ, and JCR conceived the study. UA, MR, JD, MK, WME, SK, NP, MS, VW, MG, and MK collected and curated the data. JF and MW curated the data. RRG conducted the data analyses, wrote the original draft. RRG, YZ, and JCR interpreted the results. YZ and JCR edited and reviewed the manuscript, supervision. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open Access funding enabled and organized by Projekt DEAL. This project was funded by the German Federal Ministry of Food and Agriculture, FKZ2818408B18.</p></section><section id="notes3"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Conflict of interest</h3>
<p id="Par41">On behalf of all authors, the corresponding author state that there is no conflict of interest.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par42">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Ethics approval</h3>
<p id="Par43">Not applicable.</p></section><section id="FPar4"><h3 class="pmc_sec_title">Consent to participate</h3>
<p id="Par44">Not applicable.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn2">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ol class="ref-list font-sm">
<li id="CR1">
<cite>Alemu A, Åstrand J, Montesinos-López OA, Isidro y Sánchez J, Fernández-Gónzalez J, Tadesse W, Vetukuri RR et al (2024) Genomic selection in plant breeding: key factors shaping two decades of progress. Mol Plant 17(4):552–578. 10.1016/j.molp.2024.03.007
</cite> [<a href="https://doi.org/10.1016/j.molp.2024.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38475993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alemu%20A,%20%C3%85strand%20J,%20Montesinos-L%C3%B3pez%20OA,%20Isidro%20y%20S%C3%A1nchez%20J,%20Fern%C3%A1ndez-G%C3%B3nzalez%20J,%20Tadesse%20W,%20Vetukuri%20RR%20et%20al%20(2024)%20Genomic%20selection%20in%20plant%20breeding:%20key%20factors%20shaping%20two%20decades%20of%20progress.%20Mol%20Plant%2017(4):552%E2%80%93578.%2010.1016/j.molp.2024.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<cite>Ali M, Zhang Y, Rasheed A, Wang J, Zhang L (2020) Genomic prediction for grain yield and yield-related traits in Chinese winter wheat. Int J Mol Sci 21(4):1342. 10.3390/ijms21041342
</cite> [<a href="https://doi.org/10.3390/ijms21041342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7073225/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32079240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ali%20M,%20Zhang%20Y,%20Rasheed%20A,%20Wang%20J,%20Zhang%20L%20(2020)%20Genomic%20prediction%20for%20grain%20yield%20and%20yield-related%20traits%20in%20Chinese%20winter%20wheat.%20Int%20J%20Mol%20Sci%2021(4):1342.%2010.3390/ijms21041342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<cite>Bernal-Vasquez AM, Utz H-F, Piepho H-P (2016) Outlier detection methods for generalized lattices: a case study on the transition from ANOVA to REML. Theor Appl Genet 129(4):787–804. 
10.1007/s00122-016-2666-6</cite> [<a href="https://doi.org/10.1007/s00122-016-2666-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26883044/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<cite>Bernardo R (2020) Reinventing quantitative genetics for plant breeding: something old, something new, something borrowed, something BLUE. Heredity 125(6):375–385. 10.1038/s41437-020-0312-1
</cite> [<a href="https://doi.org/10.1038/s41437-020-0312-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7784685/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32296132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bernardo%20R%20(2020)%20Reinventing%20quantitative%20genetics%20for%20plant%20breeding:%20something%20old,%20something%20new,%20something%20borrowed,%20something%20BLUE.%20Heredity%20125(6):375%E2%80%93385.%2010.1038/s41437-020-0312-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<cite>Berro I, Lado B, Nalin RS, Quincke M, Gutiérrez L (2019) Training population optimization for genomic selection. Plant Genome 12(3):190028. 10.3835/plantgenome2019.04.0028</cite> [<a href="https://doi.org/10.3835/plantgenome2019.04.0028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33016595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berro%20I,%20Lado%20B,%20Nalin%20RS,%20Quincke%20M,%20Guti%C3%A9rrez%20L%20(2019)%20Training%20population%20optimization%20for%20genomic%20selection.%20Plant%20Genome%2012(3):190028.%2010.3835/plantgenome2019.04.0028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<cite>Bodmer W, Tomlinson I (2010) Rare genetic variants and the risk of cancer. Curr Opin Genet Dev 20(3):262–267. 10.1016/j.gde.2010.04.016
</cite> [<a href="https://doi.org/10.1016/j.gde.2010.04.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20554195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bodmer%20W,%20Tomlinson%20I%20(2010)%20Rare%20genetic%20variants%20and%20the%20risk%20of%20cancer.%20Curr%20Opin%20Genet%20Dev%2020(3):262%E2%80%93267.%2010.1016/j.gde.2010.04.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<cite>Browning BL, Zhou Y, Browning SR (2018) A one-penny imputed genome from next-generation reference panels. Am J Hum Genet 103(3):338–348. 10.1016/j.ajhg.2018.07.015
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2018.07.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6128308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30100085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Browning%20BL,%20Zhou%20Y,%20Browning%20SR%20(2018)%20A%20one-penny%20imputed%20genome%20from%20next-generation%20reference%20panels.%20Am%20J%20Hum%20Genet%20103(3):338%E2%80%93348.%2010.1016/j.ajhg.2018.07.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58"><cite>Bundessortenversuch (2017) “Ergebnisse.” June 13, 2017. <a href="http://www.bundessortenversuch.de/index.php?id=4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.bundessortenversuch.de/index.php?id=4</a></cite></li>
<li id="CR8">
<cite>Clarke AJ, Cooper DN (2010) GWAS: heritability missing in action? Eur J Hum Genet 18(8):859–861. 10.1038/ejhg.2010.35
</cite> [<a href="https://doi.org/10.1038/ejhg.2010.35" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2987389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20234388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clarke%20AJ,%20Cooper%20DN%20(2010)%20GWAS:%20heritability%20missing%20in%20action?%20Eur%20J%20Hum%20Genet%2018(8):859%E2%80%93861.%2010.1038/ejhg.2010.35" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<cite>Combs E, Bernardo R (2013) Accuracy of genomewide selection for different traits with constant population size, heritability, and number of markers. Plant Genome 6(1):plantgenome2012.11.0030. 10.3835/plantgenome2012.11.0030</cite> [<a href="https://scholar.google.com/scholar_lookup?Combs%20E,%20Bernardo%20R%20(2013)%20Accuracy%20of%20genomewide%20selection%20for%20different%20traits%20with%20constant%20population%20size,%20heritability,%20and%20number%20of%20markers.%20Plant%20Genome%206(1):plantgenome2012.11.0030.%2010.3835/plantgenome2012.11.0030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<cite>Crossa J, de los Campos G, Pérez P, Gianola D, Burgueño J, Araus JL, Makumbi D et al (2010) Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers. Genetics 186(2):713–724. 10.1534/genetics.110.118521</cite> [<a href="https://doi.org/10.1534/genetics.110.118521" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2954475/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20813882/" class="usa-link">PubMed</a>]</li>
<li id="CR11">
<cite>Cuevas J, González-Diéguez D, Dreisigacker S, Martini JWR, Crespo-Herrera L, Lozano-Ramirez N, Singh PK, He X, Huerta J, Crossa J (2024) Modeling within and between sub-genomes epistasis of synthetic hexaploid wheat for genome-enabled prediction of diseases. Genes 15(3):262. 10.3390/genes15030262
</cite> [<a href="https://doi.org/10.3390/genes15030262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10970072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38540321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cuevas%20J,%20Gonz%C3%A1lez-Di%C3%A9guez%20D,%20Dreisigacker%20S,%20Martini%20JWR,%20Crespo-Herrera%20L,%20Lozano-Ramirez%20N,%20Singh%20PK,%20He%20X,%20Huerta%20J,%20Crossa%20J%20(2024)%20Modeling%20within%20and%20between%20sub-genomes%20epistasis%20of%20synthetic%20hexaploid%20wheat%20for%20genome-enabled%20prediction%20of%20diseases.%20Genes%2015(3):262.%2010.3390/genes15030262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<cite>Daetwyler HD, Villanueva B, Woolliams JA (2008) Accuracy of predicting the genetic risk of disease using a genome-wide approach. PLoS One 3(10):e3395. 10.1371/journal.pone.0003395
</cite> [<a href="https://doi.org/10.1371/journal.pone.0003395" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2561058/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18852893/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Daetwyler%20HD,%20Villanueva%20B,%20Woolliams%20JA%20(2008)%20Accuracy%20of%20predicting%20the%20genetic%20risk%20of%20disease%20using%20a%20genome-wide%20approach.%20PLoS%20One%203(10):e3395.%2010.1371/journal.pone.0003395" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3"><cite>David Butler (2019) asreml: Fits the Linear Mixed Model. R package version 4.1.0.110. <a href="http://www.vsni.co.uk" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.vsni.co.uk</a></cite></li>
<li id="CR13">
<cite>Dawson JC, Endelman JB, Heslot N, Crossa J, Poland J, Dreisigacker S, Manès Y, Sorrells ME, Jannink J-L (2013) The use of unbalanced historical data for genomic selection in an international wheat breeding program. Field Crop Res 154:12–22. 10.1016/j.fcr.2013.07.020</cite> [<a href="https://scholar.google.com/scholar_lookup?Dawson%20JC,%20Endelman%20JB,%20Heslot%20N,%20Crossa%20J,%20Poland%20J,%20Dreisigacker%20S,%20Man%C3%A8s%20Y,%20Sorrells%20ME,%20Jannink%20J-L%20(2013)%20The%20use%20of%20unbalanced%20historical%20data%20for%20genomic%20selection%20in%20an%20international%20wheat%20breeding%20program.%20Field%20Crop%20Res%20154:12%E2%80%9322.%2010.1016/j.fcr.2013.07.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<cite>Forer L, Schönherr S, Weissensteiner H, Haider F, Kluckner T, Gieger C, Wichmann H-E, Specht G, Kronenberg F, Kloss-Brandstätter A (2010) CONAN: copy number variation analysis software for genome-wide association studies. BMC Bioinformatics 11(1):318. 10.1186/1471-2105-11-318
</cite> [<a href="https://doi.org/10.1186/1471-2105-11-318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2894823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20546565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Forer%20L,%20Sch%C3%B6nherr%20S,%20Weissensteiner%20H,%20Haider%20F,%20Kluckner%20T,%20Gieger%20C,%20Wichmann%20H-E,%20Specht%20G,%20Kronenberg%20F,%20Kloss-Brandst%C3%A4tter%20A%20(2010)%20CONAN:%20copy%20number%20variation%20analysis%20software%20for%20genome-wide%20association%20studies.%20BMC%20Bioinformatics%2011(1):318.%2010.1186/1471-2105-11-318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<cite>Gogna A, Schulthess AW, Röder MS, Ganal MW, Reif JC (2022) Gabi wheat a panel of European elite lines as central stock for wheat genetic research. Sci Data 9(1):538. 10.1038/s41597-022-01651-5
</cite> [<a href="https://doi.org/10.1038/s41597-022-01651-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9440043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36056030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gogna%20A,%20Schulthess%20AW,%20R%C3%B6der%20MS,%20Ganal%20MW,%20Reif%20JC%20(2022)%20Gabi%20wheat%20a%20panel%20of%20European%20elite%20lines%20as%20central%20stock%20for%20wheat%20genetic%20research.%20Sci%20Data%209(1):538.%2010.1038/s41597-022-01651-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<cite>Gower JC (1966) Some distance properties of latent root and vector methods used in multivariate analysis. Biometrika 53(3–4):325–338. 10.1093/biomet/53.3-4.325</cite> [<a href="https://scholar.google.com/scholar_lookup?Gower%20JC%20(1966)%20Some%20distance%20properties%20of%20latent%20root%20and%20vector%20methods%20used%20in%20multivariate%20analysis.%20Biometrika%2053(3%E2%80%934):325%E2%80%93338.%2010.1093/biomet/53.3-4.325" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<cite>Habier D, Fernando RL, Dekkers JCM (2007) The impact of genetic relationship information on genome-assisted breeding values. Genetics 177(4):2389–2397. 10.1534/genetics.107.081190
</cite> [<a href="https://doi.org/10.1534/genetics.107.081190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2219482/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18073436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Habier%20D,%20Fernando%20RL,%20Dekkers%20JCM%20(2007)%20The%20impact%20of%20genetic%20relationship%20information%20on%20genome-assisted%20breeding%20values.%20Genetics%20177(4):2389%E2%80%932397.%2010.1534/genetics.107.081190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<cite>Heffner EL, Lorenz AJ, Jannink J-L, Sorrells ME (2010) Plant breeding with genomic selection: gain per unit time and cost. Crop Sci 50(5):1681–1690. 10.2135/cropsci2009.11.0662</cite> [<a href="https://scholar.google.com/scholar_lookup?Heffner%20EL,%20Lorenz%20AJ,%20Jannink%20J-L,%20Sorrells%20ME%20(2010)%20Plant%20breeding%20with%20genomic%20selection:%20gain%20per%20unit%20time%20and%20cost.%20Crop%20Sci%2050(5):1681%E2%80%931690.%2010.2135/cropsci2009.11.0662" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<cite>Henderson CR (1985) Best linear unbiased prediction of nonadditive genetic merits in noninbred populations. J Anim Sci 60(1):111–117. 10.2527/jas1985.601111x</cite> [<a href="https://scholar.google.com/scholar_lookup?Henderson%20CR%20(1985)%20Best%20linear%20unbiased%20prediction%20of%20nonadditive%20genetic%20merits%20in%20noninbred%20populations.%20J%20Anim%20Sci%2060(1):111%E2%80%93117.%2010.2527/jas1985.601111x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<cite>Hill WG (1981) Estimation of effective population size from data on linkage disequilibrium. Genet Res 38(3):209–216. 10.1017/S0016672300020553</cite> [<a href="https://scholar.google.com/scholar_lookup?Hill%20WG%20(1981)%20Estimation%20of%20effective%20population%20size%20from%20data%20on%20linkage%20disequilibrium.%20Genet%20Res%2038(3):209%E2%80%93216.%2010.1017/S0016672300020553" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<cite>Jarquín D, Crossa J, Lacaze X, Cheyron PD, Daucourt J, Lorgeou J, Piraux F et al (2014) A reaction norm model for genomic selection using high-dimensional genomic and environmental data. Theor Appl Genet 127(3):595–607. 10.1007/s00122-013-2243-1
</cite> [<a href="https://doi.org/10.1007/s00122-013-2243-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3931944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24337101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jarqu%C3%ADn%20D,%20Crossa%20J,%20Lacaze%20X,%20Cheyron%20PD,%20Daucourt%20J,%20Lorgeou%20J,%20Piraux%20F%20et%20al%20(2014)%20A%20reaction%20norm%20model%20for%20genomic%20selection%20using%20high-dimensional%20genomic%20and%20environmental%20data.%20Theor%20Appl%20Genet%20127(3):595%E2%80%93607.%2010.1007/s00122-013-2243-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<cite>Jiang Y, Reif JC (2015) Modeling epistasis in genomic selection. Genetics 201(2):759–768. 10.1534/genetics.115.177907
</cite> [<a href="https://doi.org/10.1534/genetics.115.177907" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4596682/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26219298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20Y,%20Reif%20JC%20(2015)%20Modeling%20epistasis%20in%20genomic%20selection.%20Genetics%20201(2):759%E2%80%93768.%2010.1534/genetics.115.177907" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<cite>Kaushal S, Gill HS, Billah MM, Khan SN, Halder J, Bernardo A, Amand PS, Bai G, Glover K, Maimaitijiang M, Sehgal SK (2024) Enhancing the potential of phenomic and genomic prediction in winter wheat breeding using high-throughput phenotyping and deep learning. Front Plant Sci. 10.3389/fpls.2024.1410249
</cite> [<a href="https://doi.org/10.3389/fpls.2024.1410249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11169824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38872880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaushal%20S,%20Gill%20HS,%20Billah%20MM,%20Khan%20SN,%20Halder%20J,%20Bernardo%20A,%20Amand%20PS,%20Bai%20G,%20Glover%20K,%20Maimaitijiang%20M,%20Sehgal%20SK%20(2024)%20Enhancing%20the%20potential%20of%20phenomic%20and%20genomic%20prediction%20in%20winter%20wheat%20breeding%20using%20high-throughput%20phenotyping%20and%20deep%20learning.%20Front%20Plant%20Sci.%2010.3389/fpls.2024.1410249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<cite>Lell M, Zhao Y, Reif JC (2024) Leveraging the potential of big genomic and phenotypic data for genome-wide association mapping in wheat. Crop J 12(3):803–813. 10.1016/j.cj.2024.03.005</cite> [<a href="https://scholar.google.com/scholar_lookup?Lell%20M,%20Zhao%20Y,%20Reif%20JC%20(2024)%20Leveraging%20the%20potential%20of%20big%20genomic%20and%20phenotypic%20data%20for%20genome-wide%20association%20mapping%20in%20wheat.%20Crop%20J%2012(3):803%E2%80%93813.%2010.1016/j.cj.2024.03.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<cite>Lell M et al (2025) Breaking down data silos across companies to train genome-wide predictions: a feasibility study in wheat. Plant Biotechnol J. 10.1111/pbi.70095
</cite> [<a href="https://doi.org/10.1111/pbi.70095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12205884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40253615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lell%20M%20et%20al%20(2025)%20Breaking%20down%20data%20silos%20across%20companies%20to%20train%20genome-wide%20predictions:%20a%20feasibility%20study%20in%20wheat.%20Plant%20Biotechnol%20J.%2010.1111/pbi.70095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<cite>Lian L, Jacobson A, Zhong S, Bernardo R (2014) Genomewide prediction accuracy within 969 maize biparental populations. Crop Sci 54(4):1514–1522. 10.2135/cropsci2013.12.0856</cite> [<a href="https://scholar.google.com/scholar_lookup?Lian%20L,%20Jacobson%20A,%20Zhong%20S,%20Bernardo%20R%20(2014)%20Genomewide%20prediction%20accuracy%20within%20969%20maize%20biparental%20populations.%20Crop%20Sci%2054(4):1514%E2%80%931522.%2010.2135/cropsci2013.12.0856" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<cite>Lorenzana RE, Bernardo R (2009) Accuracy of genotypic value predictions for marker-based selection in biparental plant populations. Theor Appl Genet 120(1):151–161. 10.1007/s00122-009-1166-3
</cite> [<a href="https://doi.org/10.1007/s00122-009-1166-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19841887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lorenzana%20RE,%20Bernardo%20R%20(2009)%20Accuracy%20of%20genotypic%20value%20predictions%20for%20marker-based%20selection%20in%20biparental%20plant%20populations.%20Theor%20Appl%20Genet%20120(1):151%E2%80%93161.%2010.1007/s00122-009-1166-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<cite>Makowsky R, Pajewski NM, Klimentidis YC, Vazquez AI, Duarte CW, Allison DB, de los Campos G (2011) Beyond missing heritability: prediction of complex traits. PLoS Genet 7(4):e1002051. 10.1371/journal.pgen.1002051
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1002051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3084207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21552331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Makowsky%20R,%20Pajewski%20NM,%20Klimentidis%20YC,%20Vazquez%20AI,%20Duarte%20CW,%20Allison%20DB,%20de%20los%20Campos%20G%20(2011)%20Beyond%20missing%20heritability:%20prediction%20of%20complex%20traits.%20PLoS%20Genet%207(4):e1002051.%2010.1371/journal.pgen.1002051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<cite>Manolio TA, Collins FS, Cox NJ, Goldstein DB, Hindorff LA, Hunter DJ, McCarthy MI et al (2009) Finding the missing heritability of complex diseases. Nature 461(7265):747–753. 10.1038/nature08494</cite> [<a href="https://doi.org/10.1038/nature08494" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2831613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19812666/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<cite>Meuwissen THE (2009) Accuracy of breeding values of ‘unrelated’ individuals predicted by dense SNP genotyping. Genet Sel Evol 41(1):35. 10.1186/1297-9686-41-35
</cite> [<a href="https://doi.org/10.1186/1297-9686-41-35" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2708128/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19519896/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meuwissen%20THE%20(2009)%20Accuracy%20of%20breeding%20values%20of%20%E2%80%98unrelated%E2%80%99%20individuals%20predicted%20by%20dense%20SNP%20genotyping.%20Genet%20Sel%20Evol%2041(1):35.%2010.1186/1297-9686-41-35" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<cite>Meuwissen TH, Hayes BJ, Goddard ME (2001) Prediction of total genetic value using genome-wide dense marker maps. Genetics 157(4):1819–1829. 10.1093/genetics/157.4.1819
</cite> [<a href="https://doi.org/10.1093/genetics/157.4.1819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11290733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meuwissen%20TH,%20Hayes%20BJ,%20Goddard%20ME%20(2001)%20Prediction%20of%20total%20genetic%20value%20using%20genome-wide%20dense%20marker%20maps.%20Genetics%20157(4):1819%E2%80%931829.%2010.1093/genetics/157.4.1819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<cite>Möhring J, Piepho H-P (2009) Comparison of weighting in two-stage analysis of plant breeding trials. Crop Sci 49(6):1977–1988. 10.2135/cropsci2009.02.0083</cite> [<a href="https://scholar.google.com/scholar_lookup?M%C3%B6hring%20J,%20Piepho%20H-P%20(2009)%20Comparison%20of%20weighting%20in%20two-stage%20analysis%20of%20plant%20breeding%20trials.%20Crop%20Sci%2049(6):1977%E2%80%931988.%2010.2135/cropsci2009.02.0083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<cite>Norman A, Taylor J, Edwards J, Kuchel H (2018) Optimising genomic selection in wheat: effect of marker density, population size and population structure on prediction accuracy. G3 Genes Genomes Genet 8(9):2889–2899. 10.1534/g3.118.200311</cite> [<a href="https://doi.org/10.1534/g3.118.200311" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6118301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29970398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Norman%20A,%20Taylor%20J,%20Edwards%20J,%20Kuchel%20H%20(2018)%20Optimising%20genomic%20selection%20in%20wheat:%20effect%20of%20marker%20density,%20population%20size%20and%20population%20structure%20on%20prediction%20accuracy.%20G3%20Genes%20Genomes%20Genet%208(9):2889%E2%80%932899.%2010.1534/g3.118.200311" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<cite>Park J-H, Wacholder S, Gail MH, Peters U, Jacobs KB, Chanock SJ, Chatterjee N (2010) Estimation of effect size distribution from genome-wide association studies and implications for future discoveries. Nat Genet 42(7):570–575. 10.1038/ng.610
</cite> [<a href="https://doi.org/10.1038/ng.610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4615599/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20562874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Park%20J-H,%20Wacholder%20S,%20Gail%20MH,%20Peters%20U,%20Jacobs%20KB,%20Chanock%20SJ,%20Chatterjee%20N%20(2010)%20Estimation%20of%20effect%20size%20distribution%20from%20genome-wide%20association%20studies%20and%20implications%20for%20future%20discoveries.%20Nat%20Genet%2042(7):570%E2%80%93575.%2010.1038/ng.610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<cite>Pérez P, de los Campos G (2014) Genome-wide regression and prediction with the BGLR statistical package. Genetics 198(2):483. 10.1534/genetics.114.164442
</cite> [<a href="https://doi.org/10.1534/genetics.114.164442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4196607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25009151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?P%C3%A9rez%20P,%20de%20los%20Campos%20G%20(2014)%20Genome-wide%20regression%20and%20prediction%20with%20the%20BGLR%20statistical%20package.%20Genetics%20198(2):483.%2010.1534/genetics.114.164442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<cite>Raffo MA, Sarup P, Guo X, Liu H, Andersen JR, Orabi J, Jahoor A, Jensen J (2022) Improvement of genomic prediction in advanced wheat breeding lines by including additive-by-additive epistasis. Theor Appl Genet 135(3):965–978. 10.1007/s00122-021-04009-4
</cite> [<a href="https://doi.org/10.1007/s00122-021-04009-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8942904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34973112/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raffo%20MA,%20Sarup%20P,%20Guo%20X,%20Liu%20H,%20Andersen%20JR,%20Orabi%20J,%20Jahoor%20A,%20Jensen%20J%20(2022)%20Improvement%20of%20genomic%20prediction%20in%20advanced%20wheat%20breeding%20lines%20by%20including%20additive-by-additive%20epistasis.%20Theor%20Appl%20Genet%20135(3):965%E2%80%93978.%2010.1007/s00122-021-04009-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<cite>Ray DK, Mueller ND, West PC, Foley JA (2013) Yield trends are insufficient to double global crop production by 2050. PLoS One 8(6):e66428. 10.1371/journal.pone.0066428
</cite> [<a href="https://doi.org/10.1371/journal.pone.0066428" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3686737/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23840465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ray%20DK,%20Mueller%20ND,%20West%20PC,%20Foley%20JA%20(2013)%20Yield%20trends%20are%20insufficient%20to%20double%20global%20crop%20production%20by%202050.%20PLoS%20One%208(6):e66428.%2010.1371/journal.pone.0066428" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59"><cite>R Core Team (2021). R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R-project.org/</a></cite></li>
<li id="CR39">
<cite>Rembe M, Zhao Y, Wendler N, Oldach K, Korzun V, Reif JC (2022) The potential of genome-wide prediction to support parental selection, evaluated with data from a commercial barley breeding program. Plants 11(19):Article 19. 10.3390/plants11192564</cite> [<a href="https://doi.org/10.3390/plants11192564" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9571379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36235430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rembe%20M,%20Zhao%20Y,%20Wendler%20N,%20Oldach%20K,%20Korzun%20V,%20Reif%20JC%20(2022)%20The%20potential%20of%20genome-wide%20prediction%20to%20support%20parental%20selection,%20evaluated%20with%20data%20from%20a%20commercial%20barley%20breeding%20program.%20Plants%2011(19):Article%2019.%2010.3390/plants11192564" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<cite>Rogers AR, Holland JB (2022) Environment-specific genomic prediction ability in maize using environmental covariates depends on environmental similarity to training data. G3 Genes|Genomes|Genetics 12(2):jkab440. 10.1093/g3journal/jkab440
</cite> [<a href="https://doi.org/10.1093/g3journal/jkab440" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9245610/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35100364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rogers%20AR,%20Holland%20JB%20(2022)%20Environment-specific%20genomic%20prediction%20ability%20in%20maize%20using%20environmental%20covariates%20depends%20on%20environmental%20similarity%20to%20training%20data.%20G3%20Genes%7CGenomes%7CGenetics%2012(2):jkab440.%2010.1093/g3journal/jkab440" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41"><cite>Rogers JS (1972) Measures of Genetic Similarity and Genetic Distance. In: Studies in Genetics VII, University of Texas Publication 7213, Austin, 145–153</cite></li>
<li id="CR42">
<cite>Santantonio N, Jannink JL, Sorrells M (2019) Prediction of subgenome additive and interaction effects in allohexaploid wheat. G3 Genes|genomes|genetics 9(3):685–698. 10.1534/g3.118.200613
</cite> [<a href="https://doi.org/10.1534/g3.118.200613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6404612/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30455185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Santantonio%20N,%20Jannink%20JL,%20Sorrells%20M%20(2019)%20Prediction%20of%20subgenome%20additive%20and%20interaction%20effects%20in%20allohexaploid%20wheat.%20G3%20Genes%7Cgenomes%7Cgenetics%209(3):685%E2%80%93698.%2010.1534/g3.118.200613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<cite>Schrauf MF, Martini JWR, Simianer H, de los Campos G, Cantet R, Freudenthal J, Korte A, Munilla S (2020) Phantom epistasis in genomic selection: on the predictive ability of epistatic models. G3 Genes|genomes|genetics 10(9):3137–3145. 10.1534/g3.120.401300
</cite> [<a href="https://doi.org/10.1534/g3.120.401300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7466977/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32709618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schrauf%20MF,%20Martini%20JWR,%20Simianer%20H,%20de%20los%20Campos%20G,%20Cantet%20R,%20Freudenthal%20J,%20Korte%20A,%20Munilla%20S%20(2020)%20Phantom%20epistasis%20in%20genomic%20selection:%20on%20the%20predictive%20ability%20of%20epistatic%20models.%20G3%20Genes%7Cgenomes%7Cgenetics%2010(9):3137%E2%80%933145.%2010.1534/g3.120.401300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<cite>Tessema BB, Liu H, Sørensen AC, Andersen JR, Jensen J (2020) Strategies using genomic selection to increase genetic gain in breeding programs for wheat. Front Genet. 10.3389/fgene.2020.578123
</cite> [<a href="https://doi.org/10.3389/fgene.2020.578123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7748061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33343626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tessema%20BB,%20Liu%20H,%20S%C3%B8rensen%20AC,%20Andersen%20JR,%20Jensen%20J%20(2020)%20Strategies%20using%20genomic%20selection%20to%20increase%20genetic%20gain%20in%20breeding%20programs%20for%20wheat.%20Front%20Genet.%2010.3389/fgene.2020.578123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<cite>Van Raden PM (2008) Efficient methods to compute genomic predictions. J Dairy Sci 91(11):4414–4423. 10.3168/jds.2007-0980
</cite> [<a href="https://doi.org/10.3168/jds.2007-0980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18946147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20Raden%20PM%20(2008)%20Efficient%20methods%20to%20compute%20genomic%20predictions.%20J%20Dairy%20Sci%2091(11):4414%E2%80%934423.%2010.3168/jds.2007-0980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<cite>Visscher PM, Medland SE, Ferreira MAR, Morley KI, Zhu Gu, Cornes BK, Montgomery GW, Martin NG (2006) Assumption-free estimation of heritability from genome-wide identity-by-descent sharing between full siblings. PLoS Genet 2(3):e41. 10.1371/journal.pgen.0020041
</cite> [<a href="https://doi.org/10.1371/journal.pgen.0020041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1413498/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16565746/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Visscher%20PM,%20Medland%20SE,%20Ferreira%20MAR,%20Morley%20KI,%20Zhu%20Gu,%20Cornes%20BK,%20Montgomery%20GW,%20Martin%20NG%20(2006)%20Assumption-free%20estimation%20of%20heritability%20from%20genome-wide%20identity-by-descent%20sharing%20between%20full%20siblings.%20PLoS%20Genet%202(3):e41.%2010.1371/journal.pgen.0020041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<cite>Voss-Fels KP, Cooper M, Hayes BJ (2019) Accelerating crop genetic gains with genomic selection. Theor Appl Genet 132(3):669–686. 10.1007/s00122-018-3270-8
</cite> [<a href="https://doi.org/10.1007/s00122-018-3270-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30569365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Voss-Fels%20KP,%20Cooper%20M,%20Hayes%20BJ%20(2019)%20Accelerating%20crop%20genetic%20gains%20with%20genomic%20selection.%20Theor%20Appl%20Genet%20132(3):669%E2%80%93686.%2010.1007/s00122-018-3270-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<cite>Wang S, Wong D, Forrest K, Allen A, Chao S, Huang BE, Maccaferri M et al (2014) Characterization of polyploid wheat genomic diversity using a high-density 90,000 single nucleotide polymorphism array. Plant Biotechnol J 12(6):787–796. 10.1111/pbi.12183
</cite> [<a href="https://doi.org/10.1111/pbi.12183" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4265271/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24646323/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20S,%20Wong%20D,%20Forrest%20K,%20Allen%20A,%20Chao%20S,%20Huang%20BE,%20Maccaferri%20M%20et%20al%20(2014)%20Characterization%20of%20polyploid%20wheat%20genomic%20diversity%20using%20a%20high-density%2090,000%20single%20nucleotide%20polymorphism%20array.%20Plant%20Biotechnol%20J%2012(6):787%E2%80%93796.%2010.1111/pbi.12183" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<cite>Wang K, Abid MA, Rasheed A, Crossa J, Hearne S, Li H (2023) DNNGP, a deep neural network-based method for genomic prediction using multi-omics data in plants. Mol Plant 16(1):279–293. 10.1016/j.molp.2022.11.004
</cite> [<a href="https://doi.org/10.1016/j.molp.2022.11.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36366781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20K,%20Abid%20MA,%20Rasheed%20A,%20Crossa%20J,%20Hearne%20S,%20Li%20H%20(2023)%20DNNGP,%20a%20deep%20neural%20network-based%20method%20for%20genomic%20prediction%20using%20multi-omics%20data%20in%20plants.%20Mol%20Plant%2016(1):279%E2%80%93293.%2010.1016/j.molp.2022.11.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<cite>Waples RS (2006) A bias correction for estimates of effective population size based on linkage disequilibrium at unlinked gene loci*. Conserv Genet 7(2):167–184. 10.1007/s10592-005-9100-y</cite> [<a href="https://scholar.google.com/scholar_lookup?Waples%20RS%20(2006)%20A%20bias%20correction%20for%20estimates%20of%20effective%20population%20size%20based%20on%20linkage%20disequilibrium%20at%20unlinked%20gene%20loci*.%20Conserv%20Genet%207(2):167%E2%80%93184.%2010.1007/s10592-005-9100-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<cite>Werner CR, Gaynor RC, Gorjanc G, Hickey JM, Kox T, Abbadi A, Leckband G, Snowdon RJ, Stahl A (2020) How population structure impacts genomic selection accuracy in cross-validation: implications for practical breeding. Front Plant Sci. 10.3389/fpls.2020.592977
</cite> [<a href="https://doi.org/10.3389/fpls.2020.592977" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7772221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33391305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Werner%20CR,%20Gaynor%20RC,%20Gorjanc%20G,%20Hickey%20JM,%20Kox%20T,%20Abbadi%20A,%20Leckband%20G,%20Snowdon%20RJ,%20Stahl%20A%20(2020)%20How%20population%20structure%20impacts%20genomic%20selection%20accuracy%20in%20cross-validation:%20implications%20for%20practical%20breeding.%20Front%20Plant%20Sci.%2010.3389/fpls.2020.592977" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<cite>Windhausen VS, Atlin GN, Hickey JM, Crossa J, Jannink J-L, Sorrells ME, Raman B et al (2012) Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments. G3 Genes|genomes|genetics 2(11):1427–1436. 10.1534/g3.112.003699
</cite> [<a href="https://doi.org/10.1534/g3.112.003699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3484673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23173094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Windhausen%20VS,%20Atlin%20GN,%20Hickey%20JM,%20Crossa%20J,%20Jannink%20J-L,%20Sorrells%20ME,%20Raman%20B%20et%20al%20(2012)%20Effectiveness%20of%20genomic%20prediction%20of%20maize%20hybrid%20performance%20in%20different%20breeding%20populations%20and%20environments.%20G3%20Genes%7Cgenomes%7Cgenetics%202(11):1427%E2%80%931436.%2010.1534/g3.112.003699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<cite>Würschum T, Maurer HP, Weissmann S, Hahn V, Leiser WL (2017) Accuracy of within- and among-family genomic prediction in Triticale. Plant Breed 136(2):230–236. 10.1111/pbr.12465</cite> [<a href="https://scholar.google.com/scholar_lookup?W%C3%BCrschum%20T,%20Maurer%20HP,%20Weissmann%20S,%20Hahn%20V,%20Leiser%20WL%20(2017)%20Accuracy%20of%20within-%20and%20among-family%20genomic%20prediction%20in%20Triticale.%20Plant%20Breed%20136(2):230%E2%80%93236.%2010.1111/pbr.12465" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<cite>Xu Y, Zhang X, Li H, Zheng H, Zhang J, Olsen MS, Varshney RK, Prasanna BM, Qian Q (2022) Smart breeding driven by big data, artificial intelligence, and integrated genomic-enviromic prediction. Mol Plant 15(11):1664–1695. 10.1016/j.molp.2022.09.001
</cite> [<a href="https://doi.org/10.1016/j.molp.2022.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36081348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20Y,%20Zhang%20X,%20Li%20H,%20Zheng%20H,%20Zhang%20J,%20Olsen%20MS,%20Varshney%20RK,%20Prasanna%20BM,%20Qian%20Q%20(2022)%20Smart%20breeding%20driven%20by%20big%20data,%20artificial%20intelligence,%20and%20integrated%20genomic-enviromic%20prediction.%20Mol%20Plant%2015(11):1664%E2%80%931695.%2010.1016/j.molp.2022.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<cite>Zakieh M, Alemu A, Henriksson T, Pareek N, Singh PK, Chawade A (2023) Exploring GWAS and genomic prediction to improve septoria tritici blotch resistance in wheat. Sci Rep 13(1):15651. 10.1038/s41598-023-42856-x
</cite> [<a href="https://doi.org/10.1038/s41598-023-42856-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10511425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37730954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zakieh%20M,%20Alemu%20A,%20Henriksson%20T,%20Pareek%20N,%20Singh%20PK,%20Chawade%20A%20(2023)%20Exploring%20GWAS%20and%20genomic%20prediction%20to%20improve%20septoria%20tritici%20blotch%20resistance%20in%20wheat.%20Sci%20Rep%2013(1):15651.%2010.1038/s41598-023-42856-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<cite>Zhang A, Wang H, Beyene Y, Semagn K, Liu Y, Cao S, Cui Z, Ruan Y, Burgueño J, San Vicente F, Olsen M, Prasanna BM, Crossa J, Yu H, Zhang X (2017) Effect of trait heritability, training population size and marker density on genomic prediction accuracy estimation in 22 bi-parental tropical maize populations. Front Plant Sci. 10.3389/fpls.2017.01916</cite> [<a href="https://doi.org/10.3389/fpls.2017.01916" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5683035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29167677/" class="usa-link">PubMed</a>]</li>
<li id="CR55">
<cite>Zhang H, Yin L, Wang M, Yuan X, Liu X (2019) Factors affecting the accuracy of genomic selection for agricultural economic traits in maize, cattle, and pig populations. Front Genet. 10.3389/fgene.2019.00189
</cite> [<a href="https://doi.org/10.3389/fgene.2019.00189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6426750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30923535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20H,%20Yin%20L,%20Wang%20M,%20Yuan%20X,%20Liu%20X%20(2019)%20Factors%20affecting%20the%20accuracy%20of%20genomic%20selection%20for%20agricultural%20economic%20traits%20in%20maize,%20cattle,%20and%20pig%20populations.%20Front%20Genet.%2010.3389/fgene.2019.00189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<cite>Zhao Y, Li Z, Liu G, Jiang Y, Maurer HP, Würschum T, Mock H-P et al (2015) Genome-based establishment of a high-yielding heterotic pattern for hybrid wheat breeding. Proc Natl Acad Sci U S A 112(51):15624–15629. 10.1073/pnas.1514547112
</cite> [<a href="https://doi.org/10.1073/pnas.1514547112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4697414/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26663911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20Y,%20Li%20Z,%20Liu%20G,%20Jiang%20Y,%20Maurer%20HP,%20W%C3%BCrschum%20T,%20Mock%20H-P%20et%20al%20(2015)%20Genome-based%20establishment%20of%20a%20high-yielding%20heterotic%20pattern%20for%20hybrid%20wheat%20breeding.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20112(51):15624%E2%80%9315629.%2010.1073/pnas.1514547112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<cite>Zhao Y, Thorwarth P, Jiang Y, Philipp N, Schulthess AW, Gils M, Boeven PH et al (2021) Unlocking big data doubled the accuracy in predicting the grain yield in hybrid wheat. Sci Adv 7(24):9106</cite> [<a href="https://doi.org/10.1126/sciadv.abf9106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8195483/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34117061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20Y,%20Thorwarth%20P,%20Jiang%20Y,%20Philipp%20N,%20Schulthess%20AW,%20Gils%20M,%20Boeven%20PH%20et%20al%20(2021)%20Unlocking%20big%20data%20doubled%20the%20accuracy%20in%20predicting%20the%20grain%20yield%20in%20hybrid%20wheat.%20Sci%20Adv%207(24):9106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373547/bin/122_2025_5007_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file1 (PDF 1176 KB)</a><sup> (1.1MB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik are provided here courtesy of <strong>Springer</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1007/s00122-025-05007-6"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/122_2025_Article_5007.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373547/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373547/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373547%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373547/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373547/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373547/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40844639/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373547/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40844639/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373547/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373547/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="99nFlRh0NEwyQOAWTHlDSs7g1qvMlLQXdIisFIEiW9WOCSmjMNOtmJTW8Yn756DD">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
