
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Skin irritation testing using human iPSCs derived 3D skin equivalent model - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4BD3D8AF448F305BD3D000F2D0512.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plosone">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360594/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS One">
<meta name="citation_title" content="Skin irritation testing using human iPSCs derived 3D skin equivalent model">
<meta name="citation_author" content="Hyewon Shin">
<meta name="citation_author_institution" content="College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea">
<meta name="citation_author" content="Se-Eun Kim">
<meta name="citation_author_institution" content="College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea">
<meta name="citation_author" content="C-Yoon Kim">
<meta name="citation_author_institution" content="College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea">
<meta name="citation_author" content="Suemin Lee">
<meta name="citation_author_institution" content="College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea">
<meta name="citation_author" content="Ji-Heon Lee">
<meta name="citation_author_institution" content="R&amp;D Center, CLECELL Inc., Beobwon-ro, Songpa-gu, Seoul, Korea">
<meta name="citation_author" content="Jieun Baek">
<meta name="citation_author_institution" content="R&amp;D Center, CLECELL Inc., Beobwon-ro, Songpa-gu, Seoul, Korea">
<meta name="citation_author" content="Gujin Chung">
<meta name="citation_author_institution" content="R&amp;D Center, CLECELL Inc., Beobwon-ro, Songpa-gu, Seoul, Korea">
<meta name="citation_author" content="Min Woo Kim">
<meta name="citation_author_institution" content="College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea">
<meta name="citation_author" content="Jeong-Seop Oh">
<meta name="citation_author_institution" content="Department of Veterinary Pathology, College of Veterinary Medicine, Seoul National University, Gwanak-Gu, Seoul, Republic of Korea">
<meta name="citation_author" content="Shinhye Park">
<meta name="citation_author_institution" content="College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea">
<meta name="citation_author" content="Yun Hyeong Lee">
<meta name="citation_author_institution" content="College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea">
<meta name="citation_author" content="Youngin Jeong">
<meta name="citation_author_institution" content="College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea">
<meta name="citation_author" content="Jeong Hwan Park">
<meta name="citation_author_institution" content="College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea">
<meta name="citation_author" content="Yoonseo Kim">
<meta name="citation_author_institution" content="College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea">
<meta name="citation_author" content="Myeonghee Lee">
<meta name="citation_author_institution" content="College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea">
<meta name="citation_author" content="Seul-Gi Lee">
<meta name="citation_author_institution" content="College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea">
<meta name="citation_author" content="Hyung Min Chung">
<meta name="citation_author_institution" content="Department of Stem Cell Biology, School of Medicine, Konkuk University, Gwangjin-Gu, Seoul, Republic of Korea">
<meta name="citation_author_institution" content="Miracell Bio CO. Ltd., Gangnam-gu, Seoul, Korea">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="20">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e0330306">
<meta name="citation_doi" content="10.1371/journal.pone.0330306">
<meta name="citation_pmid" content="40825023">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360594/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360594/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360594/pdf/pone.0330306.pdf">
<meta name="description" content="Artificial skin models have emerged as valuable tools for evaluating cosmetic ingredients and developing treatments for skin regeneration. Among them, 3D skin equivalent models (SKEs) using human primary skin cells are widely utilized and supported ...">
<meta name="og:title" content="Skin irritation testing using human iPSCs derived 3D skin equivalent model">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Artificial skin models have emerged as valuable tools for evaluating cosmetic ingredients and developing treatments for skin regeneration. Among them, 3D skin equivalent models (SKEs) using human primary skin cells are widely utilized and supported ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360594/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12360594">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pone.0330306"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pone.0330306.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12360594%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12360594/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12360594/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360594/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plosone.png" alt="PLOS One logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS One" title="Link to PLOS One" shape="default" href="https://doi.org/10.1371/journal.pone.0330306" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS One</button></div>. 2025 Aug 18;20(8):e0330306. doi: <a href="https://doi.org/10.1371/journal.pone.0330306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330306</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20One%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20One%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Skin irritation testing using human iPSCs derived 3D skin equivalent model</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shin%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Hyewon Shin</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Hyewon Shin</span></h3>
<div class="p">
<sup>1</sup>College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea</div>
<div>Conceptualization, Data curation, Formal analysis, Project administration, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shin%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hyewon Shin</span></a>
</div>
</div>
<sup>1,</sup><sup>☯</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20SE%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Se-Eun Kim</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Se-Eun Kim</span></h3>
<div class="p">
<sup>1</sup>College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea</div>
<div>Data curation, Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20SE%22%5BAuthor%5D" class="usa-link"><span class="name western">Se-Eun Kim</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20CY%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">C-Yoon Kim</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">C-Yoon Kim</span></h3>
<div class="p">
<sup>1</sup>College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea</div>
<div>Data curation, Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20CY%22%5BAuthor%5D" class="usa-link"><span class="name western">C-Yoon Kim</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Suemin Lee</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Suemin Lee</span></h3>
<div class="p">
<sup>1</sup>College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea</div>
<div>Data curation, Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Suemin Lee</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20JH%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ji-Heon Lee</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ji-Heon Lee</span></h3>
<div class="p">
<sup>2</sup>R&amp;D Center, CLECELL Inc., Beobwon-ro, Songpa-gu, Seoul, Korea</div>
<div>Data curation, Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20JH%22%5BAuthor%5D" class="usa-link"><span class="name western">Ji-Heon Lee</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baek%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jieun Baek</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jieun Baek</span></h3>
<div class="p">
<sup>2</sup>R&amp;D Center, CLECELL Inc., Beobwon-ro, Songpa-gu, Seoul, Korea</div>
<div>Data curation, Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baek%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jieun Baek</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chung%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Gujin Chung</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Gujin Chung</span></h3>
<div class="p">
<sup>2</sup>R&amp;D Center, CLECELL Inc., Beobwon-ro, Songpa-gu, Seoul, Korea</div>
<div>Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chung%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gujin Chung</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20MW%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Min Woo Kim</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Min Woo Kim</span></h3>
<div class="p">
<sup>1</sup>College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea</div>
<div>Data curation, Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20MW%22%5BAuthor%5D" class="usa-link"><span class="name western">Min Woo Kim</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Oh%20JS%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Jeong-Seop Oh</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Jeong-Seop Oh</span></h3>
<div class="p">
<sup>3</sup>Department of Veterinary Pathology, College of Veterinary Medicine, Seoul National University, Gwanak-Gu, Seoul, Republic of Korea</div>
<div>Data curation, Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Oh%20JS%22%5BAuthor%5D" class="usa-link"><span class="name western">Jeong-Seop Oh</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Shinhye Park</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Shinhye Park</span></h3>
<div class="p">
<sup>1</sup>College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea</div>
<div>Data curation, Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shinhye Park</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20YH%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Yun Hyeong Lee</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Yun Hyeong Lee</span></h3>
<div class="p">
<sup>1</sup>College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea</div>
<div>Data curation, Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20YH%22%5BAuthor%5D" class="usa-link"><span class="name western">Yun Hyeong Lee</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jeong%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Youngin Jeong</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Youngin Jeong</span></h3>
<div class="p">
<sup>1</sup>College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea</div>
<div>Data curation, Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jeong%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Youngin Jeong</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20JH%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Jeong Hwan Park</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Jeong Hwan Park</span></h3>
<div class="p">
<sup>1</sup>College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea</div>
<div>Data curation, Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20JH%22%5BAuthor%5D" class="usa-link"><span class="name western">Jeong Hwan Park</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Yoonseo Kim</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Yoonseo Kim</span></h3>
<div class="p">
<sup>1</sup>College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea</div>
<div>Data curation, Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yoonseo Kim</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Myeonghee Lee</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Myeonghee Lee</span></h3>
<div class="p">
<sup>1</sup>College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea</div>
<div>Data curation, Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Myeonghee Lee</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20SG%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Seul-Gi Lee</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Seul-Gi Lee</span></h3>
<div class="p">
<sup>1</sup>College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea</div>
<div>Conceptualization, Data curation, Formal analysis, Project administration, Supervision, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20SG%22%5BAuthor%5D" class="usa-link"><span class="name western">Seul-Gi Lee</span></a>
</div>
</div>
<sup>1,</sup><sup>‡,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chung%20HM%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Hyung Min Chung</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Hyung Min Chung</span></h3>
<div class="p">
<sup>4</sup>Department of Stem Cell Biology, School of Medicine, Konkuk University, Gwangjin-Gu, Seoul, Republic of Korea</div>
<div class="p">
<sup>5</sup>Miracell Bio CO. Ltd., Gangnam-gu, Seoul, Korea</div>
<div>Conceptualization, Formal analysis, Project administration, Supervision, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chung%20HM%22%5BAuthor%5D" class="usa-link"><span class="name western">Hyung Min Chung</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>‡,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Paulo Lee Ho</span><sup>6</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>College of Veterinary Medicine, Konkuk University, Gwangjin, Seoul, Republic of Korea</div>
<div id="aff002">
<sup>2</sup>R&amp;D Center, CLECELL Inc., Beobwon-ro, Songpa-gu, Seoul, Korea</div>
<div id="aff003">
<sup>3</sup>Department of Veterinary Pathology, College of Veterinary Medicine, Seoul National University, Gwanak-Gu, Seoul, Republic of Korea</div>
<div id="aff004">
<sup>4</sup>Department of Stem Cell Biology, School of Medicine, Konkuk University, Gwangjin-Gu, Seoul, Republic of Korea</div>
<div id="aff005">
<sup>5</sup>Miracell Bio CO. Ltd., Gangnam-gu, Seoul, Korea</div>
<div id="edit1">
<sup>6</sup>Instituto Butantan, BRAZIL</div>
<div class="author-notes p">
<div class="fn" id="coi001"><p><strong>Competing Interests: </strong>The authors have declared that no competing interests exist</p></div>
<div class="fn" id="econtrib001"><p>☯ This author contributed to this work as first author.</p></div>
<div class="fn" id="econtrib002"><p>‡ These authors contributed equally to this work as corresponding authors, respectively.</p></div>
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>maxwisdom@konkuk.ac.kr</span> (S-GL); <span>hmchung@kku.ac.kr</span> (HMC)</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Hyewon Shin</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Project administration, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Se-Eun Kim</span></strong>: <span class="role">Data curation, Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">C-Yoon Kim</span></strong>: <span class="role">Data curation, Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Suemin Lee</span></strong>: <span class="role">Data curation, Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ji-Heon Lee</span></strong>: <span class="role">Data curation, Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jieun Baek</span></strong>: <span class="role">Data curation, Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Gujin Chung</span></strong>: <span class="role">Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Min Woo Kim</span></strong>: <span class="role">Data curation, Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jeong-Seop Oh</span></strong>: <span class="role">Data curation, Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Shinhye Park</span></strong>: <span class="role">Data curation, Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yun Hyeong Lee</span></strong>: <span class="role">Data curation, Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Youngin Jeong</span></strong>: <span class="role">Data curation, Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jeong Hwan Park</span></strong>: <span class="role">Data curation, Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yoonseo Kim</span></strong>: <span class="role">Data curation, Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Myeonghee Lee</span></strong>: <span class="role">Data curation, Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Seul-Gi Lee</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Project administration, Supervision, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Hyung Min Chung</span></strong>: <span class="role">Conceptualization, Formal analysis, Project administration, Supervision, Writing – original draft</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Paulo Lee Ho</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 9; Accepted 2025 Jul 29; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Shin et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12360594  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40825023/" class="usa-link">40825023</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Artificial skin models have emerged as valuable tools for evaluating cosmetic ingredients and developing treatments for skin regeneration. Among them, 3D skin equivalent models (SKEs) using human primary skin cells are widely utilized and supported by standardized testing guidelines. However, primary cells face limitations such as restricted donor availability and challenges in conducting genotype-specific studies. To overcome these issues, recent approaches have focused on differentiating skin cells from human-induced pluripotent stem cells (hiPSCs). In this study, we developed a protocol to differentiate high-purity skin cells, such as fibroblasts (hFIBROs) and keratinocytes (hKERAs), from hiPSCs. To construct the hiPSC-derived SKE (hiPSC-SKE), a dermis was first formed by culturing a collagen and hFIBROs mixture within an insert. Subsequently, hKERAs were seeded onto the dermis, and keratinization was induced under air-liquid culture conditions to establish an epidermis. Histological analysis with hematoxylin and eosin staining confirmed that the hiPSC-SKE recapitulated the layered architecture of native human skin and expressed appropriate epidermal and dermal markers. Moreover, exposure to Triton X-100, a known skin irritant, led to marked epidermal damage and significantly reduced cell viability, validating the model’s functional responsiveness. These findings indicate that the hiPSC-SKE model represents a promising alternative for various skin-related applications, including the replacement of animal testing.</p></section><section id="sec001"><h2 class="pmc_sec_title">Introduction</h2>
<p>The skin, as the body’s outermost tissue, serves as a barrier against environmental stimuli and performs essential physiological functions [<a href="#pone.0330306.ref001" class="usa-link" aria-describedby="pone.0330306.ref001">1</a>,<a href="#pone.0330306.ref002" class="usa-link" aria-describedby="pone.0330306.ref002">2</a>]. It is primarily composed of the epidermis and dermis. The epidermis, the outermost layer of the skin, is mainly made up of keratinocytes and serves as a physical barrier protecting the body from external contaminants [<a href="#pone.0330306.ref001" class="usa-link" aria-describedby="pone.0330306.ref001">1</a>]. Beneath the epidermis lies the dermis, which is composed primarily of fibroblasts and collagen. This connective tissue provides elasticity and strength to the skin [<a href="#pone.0330306.ref002" class="usa-link" aria-describedby="pone.0330306.ref002">2</a>]. To replicate skin <em>in vitro</em>, 3D skin equivalent models (SKEs) have been developed using primary skin cells [<a href="#pone.0330306.ref003" class="usa-link" aria-describedby="pone.0330306.ref003">3</a>–<a href="#pone.0330306.ref005" class="usa-link" aria-describedby="pone.0330306.ref005">5</a>]. SKEs can be fabricated relatively easily using insert-based methods [<a href="#pone.0330306.ref006" class="usa-link" aria-describedby="pone.0330306.ref006">6</a>]. When keratinocytes are cultured on a surface exposed to air, they form the epidermis-like layer, creating an environment and structural complexity similar to that of human skin. The organization of fibroblasts within collagen helps maintain the elasticity and structure of SKEs, supported by the extracellular matrix (ECM) secreted by fibroblasts. Due to these features, SKEs are widely used in studies related to drug toxicity, disease modeling, and wound healing applications [<a href="#pone.0330306.ref005" class="usa-link" aria-describedby="pone.0330306.ref005">5</a>].</p>
<p>SKEs are widely used for skin irritation and corrosion testing, guided by international standards like the Organization for Economic Cooperation and Development test guideline 439 (OECD TG 439). This testing is crucial for evaluating the safety of cosmetics, pharmaceuticals, and chemicals, and OECD TG 439 promotes artificial skin models as a replacement for animal testing [<a href="#pone.0330306.ref007" class="usa-link" aria-describedby="pone.0330306.ref007">7</a>–<a href="#pone.0330306.ref009" class="usa-link" aria-describedby="pone.0330306.ref009">9</a>]. Recently, the U.S. Food and Drug Administration (FDA) introduced the FDA Modernization Act 2.0 in 2023, officially recognizing alternatives to animal testing in drug development and safety evaluation [<a href="#pone.0330306.ref010" class="usa-link" aria-describedby="pone.0330306.ref010">10</a>,<a href="#pone.0330306.ref011" class="usa-link" aria-describedby="pone.0330306.ref011">11</a>]. Similarly, the European Medicines Agency (EMA) announced that animal testing will be phased out in all product development processes starting in 2024. In response to these global trends, the use of SKEs has increased, with many studies addressing the limitations of existing models.</p>
<p>Artificial skin models used according to established guidelines typically include reconstructed human epidermis (RhE) products such as Episkin™, EpiDerm™, and SkinEthic™ RHE, as well as dermo-epidermal SKEs constructed from commercially available primary skin cells [<a href="#pone.0330306.ref007" class="usa-link" aria-describedby="pone.0330306.ref007">7</a>–<a href="#pone.0330306.ref009" class="usa-link" aria-describedby="pone.0330306.ref009">9</a>]. In addition to safety evaluations, skin cells are commonly used in research on skin diseases and therapeutic agents [<a href="#pone.0330306.ref006" class="usa-link" aria-describedby="pone.0330306.ref006">6</a>,<a href="#pone.0330306.ref012" class="usa-link" aria-describedby="pone.0330306.ref012">12</a>–<a href="#pone.0330306.ref015" class="usa-link" aria-describedby="pone.0330306.ref015">15</a>]. While primary cells reflect <em>in vivo</em> maturity, their use is limited by supply shortages, high costs, and senescence during passaging. Additionally, modeling genetic skin diseases is challenging due to difficulties in producing patient-specific products, and immune rejection complicates therapeutic transplantation. To address these issues, human induced pluripotent stem cells (hiPSCs) have emerged as a promising source [<a href="#pone.0330306.ref016" class="usa-link" aria-describedby="pone.0330306.ref016">16</a>,<a href="#pone.0330306.ref017" class="usa-link" aria-describedby="pone.0330306.ref017">17</a>]. Advantages of hiPSC-derived cells include: (1) unlimited supply through self-renewal, (2) disease modeling via patient-specific genetics or gene editing, and (3) improved therapeutic potential with better engraftment and low immunogenicity [<a href="#pone.0330306.ref016" class="usa-link" aria-describedby="pone.0330306.ref016">16</a>,<a href="#pone.0330306.ref017" class="usa-link" aria-describedby="pone.0330306.ref017">17</a>]. Several studies have successfully generated 3D SKEs from hiPSC-derived skin cells [<a href="#pone.0330306.ref018" class="usa-link" aria-describedby="pone.0330306.ref018">18</a>–<a href="#pone.0330306.ref021" class="usa-link" aria-describedby="pone.0330306.ref021">21</a>]. However, reports on hiPSC-derived SKEs (hiPSC-SKEs) remain limited, and a comprehensive study database is needed to optimize these models. In this study, we established a protocol to differentiate skin cells from hiPSCs and generated hiPSC-SKEs. Morphological analyses confirmed that hiPSC-SKEs replicate human skin features, and skin irritation tests using Triton X-100 were successfully performed. These datasets are expected to contribute significantly to cosmetics, chemicals, and drug development as hiPSC-SKEs advance as skin models.</p></section><section id="sec002"><h2 class="pmc_sec_title">Materials &amp; methods</h2>
<section id="sec003"><h3 class="pmc_sec_title">Generation of human induced pluripotent stem cells (hiPSCs)</h3>
<p>1.5 × 10⁶ human dermal fibroblasts (BJ cells; ATCC, VA, USA) were transfected with episomal vectors (pCLXE-hOCT4/p53, pCLXE-hSOX2/KLF4, and pCLXE-hL-MYC/LIN28A; Addgene, MA, USA) using the P2 Primary Cell 4D-Nucleofector Kit (Lonza, Basel, Switzerland) according to the manufacturer’s protocol [<a href="#pone.0330306.ref022" class="usa-link" aria-describedby="pone.0330306.ref022">22</a>,<a href="#pone.0330306.ref023" class="usa-link" aria-describedby="pone.0330306.ref023">23</a>]. The mixture of cells and vectors was transferred into a Nucleocuvette Vessel and subjected to electroporation. Transfected cells were plated onto 1:100 Matrigel (Corning, NY, USA)-coated 6-well plates and cultured for 2 days in Dulbecco’s Modified Eagle’s Medium (DMEM; Gibco, NY, USA) supplemented with 10% fetal bovine serum (FBS; Sigma-Aldrich, St. Louis, MO, USA), 0.1 mM non-essential amino acids (NEAA; Gibco), and 1% penicillin/streptomycin (P/S; Gibco). After 2 days, the medium was replaced with TeSR-E7 medium (STEMCELL Technologies, Vancouver, Canada), which was changed daily. Cells were cultured for an additional 10–14 days until PSC-like colonies appeared. After picking the PSC-like colonies, they were seeded onto Matrigel-coated 4-well plates in mTeSR1 (STEMCELL Technologies Inc; Vancouver, BC, Canada) medium supplemented with 10 µM Y-27632 (Tocris Bioscience, Bristol, UK). The medium was changed daily with fresh mTeSR1 during subsequent culture.</p></section><section id="sec004"><h3 class="pmc_sec_title">Generation of hiPSCs derived fibroblasts (hFIBROs)</h3>
<p>hiPSCs were seeded onto 1:100 Matrigel-coated 60 mm cell culture dishes and maintained in mTeSR1 medium with daily medium changes. Upon reaching 90–100% confluency after 3 days, the cells were gently dissociated using DPBS (Welgene, Gyeongsan, South Korea) containing 0.5 mM EDTA (Gibco). The detached hiPSCs were resuspended in 6 ml mTeSR1 supplemented with 10 µM Y-27632 and transferred to F127-coated (Sigma-Aldrich) 60 mm Petri dishes to initiate embryoid body (EB) formation (D0). Cells were cultured on a shaker at 65–70 rpm in a 37 °C incubator with 5% CO₂ for 24 hr. On the following day (D1), EB formation was confirmed, and mTeSR1 was replaced daily for two additional days (until D3). Approximately 150–200 EBs were then transferred onto Matrigel-coated 60 mm culture dishes and cultured in fibroblast differentiation medium 1 (FDM1) for 3 days (D3-D6). FDM1 consisted of a 3:1 mixture of DMEM (Gibco) and F12 (Gibco), supplemented with 5% FBS, 5 µg/ml insulin (Sigma-Aldrich), and 10 ng/ml epidermal growth factor (EGF; PeproTech, NJ, USA). After 3 days, the medium was changed to fibroblast differentiation medium 2 (FDM2), which is FDM1 supplemented with 25 ng/ml bone morphogenetic protein 4 (BMP4; PeproTech), and cultured for another 3 days (D6-D9). Subsequently, the cells were cultured in fibroblast differentiation medium 3 (FDM3) for 7 days (D9-D16). FDM3 consisted of a 1:1 mixture of DMEM and F12 supplemented with 5% FBS and 1% NEAA. Upon reaching 70–80% confluency (Day 16), cells were dissociated using a 5:1 mixture of TrypLE (Gibco) and 0.25% Trypsin-EDTA (Gibco). The detached cells were replated on Matrigel-coated dishes in FDM1 and maintained until Day 30 (D30) (<a href="#pone.0330306.g001" class="usa-link">Fig 1A</a>).</p>
<figure class="fig xbox font-sm" id="pone.0330306.g001"><h4 class="obj_head">Fig 1. Generation of hFIBROs and hKERAs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360594_pone.0330306.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cd98/12360594/de5738abaea7/pone.0330306.g001.jpg" loading="lazy" height="872" width="762" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330306.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Differentiation protocol of hFIBROs. (B) The morphology of hFIBROs at each day (Day 4, 16 and 30). Left scale bar: 300 µm and right scale bar: 100 µm. (C) IF staining for fibronectin and vimentin. Scale bar: 100 µm. (D) FACS for Vimentin<sup>+</sup> cells in hFIBROs. (E) Differentiation protocol of hKERAs. (F) The morphology of hKERAs at each day (Day 4, 17 and 30). Left scale bar: 300 µm and right scale bar: 100 µm. (G) IF staining for KRT14 and ΔNp63. Scale bar: 100 µm. (H) FACS for KRT14<sup>+</sup> cells in hKERAs.</p></figcaption></figure></section><section id="sec005"><h3 class="pmc_sec_title">Generation of hiPSCs derived keratinocytes (hKERAs)</h3>
<p>70−80 EBs were seeded onto Matrigel-coated 60 mm cell culture dishes and cultured in keratinocyte differentiation medium 1 (KDM1) for 7 days, with daily medium changes (D3-D10). KDM1 consisted of a 3:1 mixture of DMEM and F12, supplemented with 2% FBS, 5 µg/mL insulin, 25 ng/ml EGF, 25 ng/ml BMP4, and 1 µg/ml retinoic acid (RA; Sigma-Aldrich). After 7 days, the medium was replaced with keratinocyte differentiation medium 2 (KDM2) and cells were cultured for an additional 7 days (D10-D17). KDM2 consisted of Defined Keratinocyte Serum-Free Medium (DKSFM; Gibco) supplemented with 0.5% FBS, 5 µg/ml insulin, 20 ng/ml EGF, 20 ng/ml BMP4, and 1 µg/ml RA. Subsequently, the cells were cultured for 7 days in CnT-07 medium (CELLnTEC, Bern, Switzerland) (D17-D24). When cells reached 70–80% confluency (D24), they were dissociated using a 5:1 mixture of TrypLE (Gibco) and 0.25% Trypsin-EDTA (Gibco). The detached cells were replated on Matrigel-coated culture dishes and maintained in CnT-07 medium until Day 30 (D30) (<a href="#pone.0330306.g001" class="usa-link">Fig 1E</a>).</p></section><section id="sec006"><h3 class="pmc_sec_title">Generation of human iPSCs derived 3D Skin Equivalent Model</h3>
<p>To construct the dermal layer, a mixture of 4.5 mg/ml collagen type I (Corning, NY, USA, Cat# 354249) and 6 × 10⁴ hFIBROs was poured into a porous polyester membrane insert (Corning) and allowed to solidify in a 37 °C incubator with 5% CO₂ for 30 min. Then, the construct was cultured in FDM1 medium with daily medium changes for 3 days (D0-D3). Subsequently, 1.5 × 10⁶ hKERAs were seeded onto the surface of the dermal layer in CnT-07 medium, which was replaced daily for 3 days (D3-D6). The culture was then switched to CnT-airlift medium (CELLnTEC) supplemented with 1 mM calcium chloride (CaCl<sub>2</sub>; PromoCell, Heidelberg, Germany) and maintained for an additional 4 days (D6-D10). To induce epidermal keratinization, the medium was completely removed from the insert, and an air–liquid culture was maintained for 20 days (D10-D30). CnT-airlift medium was added beneath the insert and replaced every other day [<a href="#pone.0330306.ref024" class="usa-link" aria-describedby="pone.0330306.ref024">24</a>].</p></section><section id="sec007"><h3 class="pmc_sec_title">Flow cytometry analysis (FACS)</h3>
<p>hFIBROs and hKERAs were fixed in 4% Paraformaldehyde solution (PFA; Biosesang, Yongin, Korea) for 20 min and resuspended in FACS solution after washing. Then, fixed hFIBROs and hKERAs were stained each using PE-conjugated Mouse Anti-Human Vimentin (BD Biosciences; CA, USA, Cat# 562337; 1:25) and FITC-conjugated Anti-Cytokeratin 14 antibody (KRT14; abcam, Cambridge, England, Cat# ab77684; 1:25) for 30 min at 4 °C. To confirm Vimentin and KRT14 expression rate, analysis was performed using FACS Calibur and Cell Quest software (BD Biosciences).</p></section><section id="sec008"><h3 class="pmc_sec_title">Immunofluorescence analysis</h3>
<p>hFIBROs and hKERAs were fixed with 4% PFA for 25 min at 4 °C and replaced with PBS. The preparation was blocked and permeabilized with blocking solution (PBS containing 0.1% Triton X100 and 3% normal goat serum) on shaker for 30 min at room temperature (RT). Next, it was treated to the primary antibodies diluted in blocking solution overnight (O/N) at 4 °C. As primary antibodies, anti-Vimentin (abcam, Cat# ab8978; 1:200), anti-Fibronectin (abcam, Cat# ab2413; 1:200), anti-ΔNp63 (abcam, Cat#203826; 1:100), and anti-keratin 14 (KRT14; abcam, Cat# ab181595; 1:200) were used. After O/N, it was washed twice with PBS for 15 min each and treated with secondary antibodies diluted in blocking solution for 2 hr under light protection. As secondary antibodies, Alexa 488 goat anti-mouse IgG (Invitrogen, Grand Island, NY), Alexa 488 goat anti-rabbit IgG (Invitrogen) and Alexa 594 goat anti-rabbit IgG (Invitrogen) were used for 1:700. Next, it was washed twice with PBS for 15 min each and treated with DAPI (4′,6-diamidino-2-phenylindole; Sigma-Aldrich, Cat# D9542) for 1 min. DAPI was used as the nuclear counterstain.</p>
<p>hiPSC-SKEs were fixed with 4% PFA on D30 at 4 °C O/N. The fixed samples were dehydrated in graded sucrose (15% to 30%) for 2 days and planted in OCT compound (Sakura Finetek, Tokyo, Japan) for cryosection. Cryosections were sliced by sectioning them into about 8 µm size on a cryotome. The sections were blocked and permeabilized with blocking solution for 30 min at RT. Subsequently, they were treated to the primary antibodies diluted in blocking solution O/N at 4 °C in wet chamber. As primary antibodies, anti-Vimentin (abcam, Cat# ab8978; 1:200), anti-Fibronectin (abcam, Cat# ab2413; 1:200), anti-ΔNp63 (abcam, Cat#203826; 1:100), anti-KRT14 (abcam, Cat# ab181595; 1:200), anti-platelet-derived growth factor receptor α (PDGFRα; abcam, ab32570), anti-KRT10 (abcam, Cat# ab76318; 1:200), and anti-Loricirin (abcam, Cat# ab198994; 1:100) were used. Next day, they were washed 3–5 times with PBS and treated with secondary antibodies diluted in blocking solution for 2 hr at RT under light protection. As secondary antibodies, Alexa 488 goat anti-mouse IgG, Alexa 488 goat anti-rabbit IgG, and Alexa 594 goat anti-rabbit IgG were used for 1:700. Then, they were washed 3 times with PBS and treated with DAPI. After that, they were washed 3 times with PBS and mounted through mounting solution (Vector labs, CA, USA). The fluorescent dye color of the images was changed for visual effect.</p>
<p>All images were analyzed by using a fluorescence microscope, Nikon TE2000-U (Nikon, Tokyo, Japan).</p></section><section id="sec009"><h3 class="pmc_sec_title">Gel electrophoresis</h3>
<p>To analyze the gene expression in hFIBROs and hKERAs, cells were harvested on day 30 of differentiation. Total RNA was isolated using TRIzol reagent (Invitrogen, Grand Island, NY, USA) following the manufacturer’s instructions. 1 μg RNA was used for cDNA synthesis with AccuPower RT PreMix and Oligo(dT) primers (Bioneer, Daejeon, Korea). The synthesized cDNA was then mixed with gene-specific primers and AccuPower® PCR PreMix (S1 Table in <a href="#pone.0330306.s001" class="usa-link">S1 File</a>). PCR was performed and the presence of target gene bands was verified for each sample by electrophoresis. The information of primers was described in S1 Table in <a href="#pone.0330306.s001" class="usa-link">S1 File</a>. Also, we used GAPDH as PCR control.</p></section><section id="sec010"><h3 class="pmc_sec_title">Histological analysis</h3>
<p>The fixed samples for paraffin block were subjected to tissue processing and paraffin embedding. Paraffin sections were prepared by sectioning at 5 µm through a microtome. Hematoxylin and Eosin (H&amp;E) staining was performed in Hematoxylin for 6 min and washed 3 times in 0.5% HCl-ethanol. Subsequently, they were neutralized with 0.5% Ammonia solution, and stained in Eosin Y for 7 min. After that, they were dehydrated and mounted. Human circum skin tissue and mouse dorsal skin tissue were sampled as native skin.</p></section><section id="sec011"><h3 class="pmc_sec_title">Skin irritation testing</h3>
<p>Cell Counting Kit-8 (CCK-8; Dojindo, Kumamoto, Japan) was used to measure cell viability. 300 µl CCK-8 working solution (a 1:9 mixture of CCK-8 reagent and CnT-airlift medium) was added to the insert containing D30 hiPSC-SKE and the baseline absorbance was measured. After incubation in 37 °C, 5% <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330306.e001"><math id="equation1" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>C</mi><mi>O</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span> incubator for 2 hr, 100 µl of the solution in the insert was transferred to flat bottom 96 well plate and measure the baseline absorbance. Then, the solution in the insert was replaced with fresh media and incubated for 30 min. After washing, 3% Triton X-100 diluted in medium was treated in the drug treatment group and incubated for 1.5 hr. Next, it was replaced with 300 µl CCK-8 working solution and incubated for 2 hr. After incubation, 100 µl of the medium in the insert was transferred to a 96 well plate and measure the absorbance. The absorbance was measured at a wavelength of 450 nm using an Epoch microplate reader (Agilent, CA, USA).</p></section><section id="sec012"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p>All experiments were performed at least three times. Statistical analyses were performed using the GraphPad Prism software (La Jolla, CA, USA; Version 10). Data were presented as mean ± SEM, and the statistical significance of the experimental results was calculated using either a T-test or one-way ANOVA with Tukey’s method for comparison between groups. A p value (*<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01 and ***<em>p</em> &lt; 0.001) was considered statistically significant.</p></section></section><section id="sec013"><h2 class="pmc_sec_title">Results</h2>
<section id="sec014"><h3 class="pmc_sec_title">Differentiation and characterization of hFIBROs and hKERAs</h3>
<p>To generate human induced pluripotent stem cells (hiPSCs) derived fibroblasts (hFIBROs), we established a protocol based on cytokines and small molecules by referring to previous reports (<a href="#pone.0330306.g001" class="usa-link">Fig 1A</a>) [<a href="#pone.0330306.ref021" class="usa-link" aria-describedby="pone.0330306.ref021">21</a>,<a href="#pone.0330306.ref025" class="usa-link" aria-describedby="pone.0330306.ref025">25</a>]. During differentiation, morphological changes consistent with mesenchymal transition were observed. Embryoid bodies (EBs) plated onto Matrigel-coated dishes proliferated and began adopting a spindle-shaped, fibroblast-like morphology by day 16 (D16), which was maintained after passaging up to day 30 (D30) (<a href="#pone.0330306.g001" class="usa-link">Fig 1B</a>). IF analysis was performed for vimentin, a fibroblast marker [<a href="#pone.0330306.ref026" class="usa-link" aria-describedby="pone.0330306.ref026">26</a>], and fibronectin, an extracellular matrix (ECM) protein secreted by fibroblasts [<a href="#pone.0330306.ref027" class="usa-link" aria-describedby="pone.0330306.ref027">27</a>]. Most of the differentiated cells (DAPI) expressed vimentin, and fibronectin was secreted beneath the vimentin<sup>+</sup> cells (<a href="#pone.0330306.g001" class="usa-link">Fig 1C</a>). The purity of hFIBROs was assessed by analyzing the density of vimentin<sup>+</sup> cells per field, which was 97.01 ± 0.73% (S1A Fig in <a href="#pone.0330306.s001" class="usa-link">S1 File</a>). Also, FACS analysis confirmed that 92.94% of the cells were vimentin<sup>+</sup>, consistent with IF staining. (<a href="#pone.0330306.g001" class="usa-link">Fig 1D</a>). Next, we established a protocol for generating hiPSC derived keratinocytes (hKERAs) (<a href="#pone.0330306.g001" class="usa-link">Fig 1E</a>) [<a href="#pone.0330306.ref021" class="usa-link" aria-describedby="pone.0330306.ref021">21</a>,<a href="#pone.0330306.ref024" class="usa-link" aria-describedby="pone.0330306.ref024">24</a>,<a href="#pone.0330306.ref025" class="usa-link" aria-describedby="pone.0330306.ref025">25</a>]. Following EB attachment, cells began to exhibit a cobblestone morphology characteristic of keratinocytes by day 17 (D17), which persisted through D30 (<a href="#pone.0330306.g001" class="usa-link">Fig 1F</a>). To confirm the expression of keratinocyte-specific markers in hKERAs, we performed staining for KRT14 and ΔNp63. KRT14 is a type I intermediate filament protein predominantly expressed in the basal keratinocytes of the epidermis, while ΔNp63 is a transcription factor involved in the regulation of early keratinocyte differentiation and proliferation [<a href="#pone.0330306.ref028" class="usa-link" aria-describedby="pone.0330306.ref028">28</a>–<a href="#pone.0330306.ref030" class="usa-link" aria-describedby="pone.0330306.ref030">30</a>]. Most of the differentiated cells (DAPI<sup>\</sup>) expressed ΔNp63 and KRT14 (<a href="#pone.0330306.g001" class="usa-link">Fig 1G</a>). The purity of hKERAs was assessed by analyzing the density of ΔNp63<sup>+</sup> cells per field, which was 98.09 ± 0.24% (S1B Fig in <a href="#pone.0330306.s001" class="usa-link">S1 File</a>). FACS analysis further confirmed the efficiency of differentiation, with 97.48% of cells expressing KRT14 (<a href="#pone.0330306.g001" class="usa-link">Fig 1H</a>). To further confirm the absence of residual undifferentiated cells in the differentiated hFIBROs and hKERAs, gel electrophoresis analysis for pluripotency markers was performed (S2 Fig in <a href="#pone.0330306.s001" class="usa-link">S1 File</a>). Pluripotent markers such as SOX2 and OCT4 were strongly expressed in undifferentiated hiPSCs but were undetectable in both hFIBROs and hKERAs. In contrast, fibroblast-specific markers (PDGFRα, Col3A1, and fibronectin) were exclusively expressed in hFIBROs (S2A Fig in <a href="#pone.0330306.s001" class="usa-link">S1 File</a>), while keratinocyte-specific markers (KRT14 and ΔNp63) were detected only in hKERAs (S2B Fig in <a href="#pone.0330306.s001" class="usa-link">S1 File</a>). These results confirm successful lineage-specific differentiation and effective elimination of PSCs, demonstrating that the established protocols reliably generate hFIBROs and hKERAs.</p></section><section id="sec015"><h3 class="pmc_sec_title">Generation of hiPSC-SKE</h3>
<p>To fabricate skin equivalents using hiPSC derived skin cells (hiPSC-SKE), a previously reported method based on insert and collagen was employed (<a href="#pone.0330306.g002" class="usa-link">Fig 2A</a>) [<a href="#pone.0330306.ref006" class="usa-link" aria-describedby="pone.0330306.ref006">6</a>,<a href="#pone.0330306.ref012" class="usa-link" aria-describedby="pone.0330306.ref012">12</a>,<a href="#pone.0330306.ref025" class="usa-link" aria-describedby="pone.0330306.ref025">25</a>]. After inserting a mixture of hFIBROs and collagen into the insert, the dermis was formed through gelation and subsequent culture (<a href="#pone.0330306.g002" class="usa-link">Fig 2A</a> and <a href="#pone.0330306.g002" class="usa-link">2B</a>). Subsequently, hKERAs were seeded onto the constructed dermis to form an epidermal layer, and keratinization was induced via air-liquid (ALI) culture (<a href="#pone.0330306.g002" class="usa-link">Fig 2A</a> and <a href="#pone.0330306.g002" class="usa-link">2B</a>). As differentiation progressed, the upper surface of hiPSC-SKE exhibited changes. After ALI culture, the epidermal layer became uneven, with some edges contracting. (<a href="#pone.0330306.g002" class="usa-link">Fig 2B</a>). H&amp;E staining was performed to evaluate epidermal morphology at early (D20; ALI culture for 10 days) and late (D30; ALI culture for 20 days) stages of ALI culture (<a href="#pone.0330306.g002" class="usa-link">Fig 2C</a>). Keratinization of the epidermis was more prominent at ALI-20 days than at ALI-10 days of ALI culture. The epidermal thickness was significantly increased at ALI-20 days (ALI-10 days: 38.82 ± 2.20 μm vs. ALI-20 days: 63.62 ± 8.06 μm), accompanied by thickening of the stratum basale (SB) and stratum spinosum (SS)-like layers. Furthermore, a stratum corneum (SC)-like structure was observed at the air-exposed surface at ALI-20 days (<a href="#pone.0330306.g002" class="usa-link">Fig 2C</a> and <a href="#pone.0330306.g002" class="usa-link">2D</a>) [<a href="#pone.0330306.ref013" class="usa-link" aria-describedby="pone.0330306.ref013">13</a>]. Comparative H&amp;E analysis revealed that both the dermal and epidermal layers of hiPSC-SKE closely resemble those of native human and mouse skin in structure (<a href="#pone.0330306.g002" class="usa-link">Fig 2E</a>).</p>
<figure class="fig xbox font-sm" id="pone.0330306.g002"><h4 class="obj_head">Fig 2. Generation of hiPSC-SKE.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360594_pone.0330306.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cd98/12360594/bad2c7986553/pone.0330306.g002.jpg" loading="lazy" height="739" width="750" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330306.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Protocol for generation of hiPSC-SKE. (B) Images of hiPSC-SKE in the inserts at each differentiation stage. (C) Comparison of epidermis formation according to ALI culture duration (10 and 20 days) through H&amp;E staining. SC: stratum corneum. SG: stratum granulosum. SB: stratum basale. Scale bar: 50 µm. (D) Comparison of epidermis thickness according to ALI culture duration (10 and 20 days) (n = 4). (E) Morphology between native skin (human and mouse) and hiPSC-SKE through H&amp;E staining. Scale bar: 25 and 50 µm. Yellow dotted boxes: enlarged images. Data are presented as mean values ± SEM. <em>p</em> value: *<em>p</em> &lt; 0.05.</p></figcaption></figure></section><section id="sec016"><h3 class="pmc_sec_title">Characterization of Cellular and ECM Distribution in hiPSC-SKE</h3>
<p>To investigate the progression of keratinization induced by ALI culture in the epidermis, representative markers of major epidermal layers were examined at 10 and 20 days of ALI culture. IF staining for KRT14 and ΔNp63, representative markers of keratinocytes predominantly expressed in the SB, showed strong expression in the lower part of the epidermis (<a href="#pone.0330306.g003" class="usa-link">Fig 3A</a> and <a href="#pone.0330306.g003" class="usa-link">3B</a>). Notably, ΔNp63 expression gradually decreased toward the upper layers at ALI-20 days (D30), compared to ALI-10 days (D20) (<a href="#pone.0330306.g003" class="usa-link">Fig 3B</a>). KRT10, a marker primarily expressed in the SS, was strongly detected above the SB, and its expression domain became thicker at ALI-20 days compared to ALI-10 days (<a href="#pone.0330306.g003" class="usa-link">Fig 3C</a>). Furthermore, loricrin, a late differentiation marker expressed in the SC, was not detected at ALI-10 days but was clearly expressed in a band-like pattern at ALI-20 days (<a href="#pone.0330306.g004" class="usa-link">Fig 4D</a>) [<a href="#pone.0330306.ref020" class="usa-link" aria-describedby="pone.0330306.ref020">20</a>,<a href="#pone.0330306.ref028" class="usa-link" aria-describedby="pone.0330306.ref028">28</a>,<a href="#pone.0330306.ref031" class="usa-link" aria-describedby="pone.0330306.ref031">31</a>]. Next, the distribution of hFIBROs within the dermis was assessed using the fibroblast-specific markers Vimentin and PDGFRα. Vimentin<sup>+</sup> and PDGFRα<sup>+</sup> cells exhibited mesenchymal morphology and were distributed within the collagen-containing dermis (<a href="#pone.0330306.g003" class="usa-link">Fig 3E</a> and <a href="#pone.0330306.g003" class="usa-link">3F</a>). Fibronectin, an ECM component secreted by fibroblasts, was expressed in the dermis but not in the epidermis. Notably, fibronectin showed strong expression in the dermal region adjacent to the epidermis (<a href="#pone.0330306.g003" class="usa-link">Fig 3G</a>). These results demonstrated that the application of hFIBROs and hKERAs can sufficiently implement artificial skin that can reflect the main skin constituent layers and the keratinization of epidermis.</p>
<figure class="fig xbox font-sm" id="pone.0330306.g003"><h4 class="obj_head">Fig 3. Verification of skin related markers in hiPSC-SKE.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360594_pone.0330306.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cd98/12360594/9776a27aae08/pone.0330306.g003.jpg" loading="lazy" height="569" width="750" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330306.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>IF staining for (A) KRT14 and (B) ΔNp63 identifying SB distribution in hiPSC-SKE according to ALI culture duration (10 and 20 days). IF staining for (C) KRT10 to identify SS distribution and (D) Loricrin to identify SC distribution. IF staining for (E) vimentin, (F) PDGFRα, and (G) fibronectin to identify hFIBROs in hiPSC-SKE at D30 (ALI culture for 20 days). Scale bar: 50 µm. Yellow dotted boxes: enlarged images.</p></figcaption></figure><figure class="fig xbox font-sm" id="pone.0330306.g004"><h4 class="obj_head">Fig 4. Skin irritation testing using hiPSC-SKEs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360594_pone.0330306.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cd98/12360594/28c8e2643a7f/pone.0330306.g004.jpg" loading="lazy" height="462" width="750" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330306.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Schematic diagram of skin irritation induced chemical stimulation. (B) Protocol of the skin irritation testing by CCK-8 assay. Arrow: O.D value measurement time. (C) Comparison of color changes before (Pre-treat) and after treatment (Post-treat) with the control group and 3% Triton using CCK-8 solution. (D) Changes in O.D value of CTL and 3% Triton-treated groups (n = 4). Data are presented as mean values ± SEM. n.s: not significant. <em>p</em> value: ***<em>p</em> &lt; 0.0001. (E) Comparison of morphology between the control and 3% Triton using H&amp;E staining. Scale bar: 100 µm. Yellow dotted boxes: enlarged images.</p></figcaption></figure></section><section id="sec017"><h3 class="pmc_sec_title">Skin irritation testing using hiPSC-SKE</h3>
<p>Skin irritation testing is a type of alternative animal testing that evaluates cell damage in artificial skin models following exposure to chemical substances (<a href="#pone.0330306.g004" class="usa-link">Fig 4A</a>). In this test, sodium dodecyl sulfate (SDS) or Triton X-100, which are known to cause cytotoxicity as a side effect, are used as a positive control, and the degree of tissue damage is assessed through an absorbance-based cell viability test [<a href="#pone.0330306.ref032" class="usa-link" aria-describedby="pone.0330306.ref032">32</a>]. To evaluate the utility of hiPSC-SKEs in skin irritation testing, a CCK-8 based cell viability assay using 3% Triton X-100 was conducted. Triton X-100 is a nonionic detergent that disrupts cell membranes by specifically binding to membrane lipids, damaging the structural integrity of the membrane. This increases membrane permeability and causes ion imbalance, ultimately disrupting cellular homeostasis and inducing cell death. Additionally, increased cellular stress damages the mitochondrial membrane potential, further promoting cell death [<a href="#pone.0330306.ref033" class="usa-link" aria-describedby="pone.0330306.ref033">33</a>–<a href="#pone.0330306.ref036" class="usa-link" aria-describedby="pone.0330306.ref036">36</a>]. Triton X-100 induces reproducible skin irritation under standardized conditions with minimal nonspecific cytotoxicity, making it a widely used positive control in skin irritation testing. Prior to treatment with 3% Triton, the baseline (pre-treatment; pre-treat) optical density (O.D.) of each hiPSC-SKE was measured using the CCK-8 assay. To ensure proper penetration of the treatment agent, 3% Triton was applied inside the insert containing the hiPSC-SKEs, while only basic medium was added to the outer well. As a negative control, culture medium (CnT-Prime Airlift) was used. After 1.5 hr of treatment, the hiPSC-SKEs were washed, and the post-treatment (post-treat) O.D. was measured again using the CCK-8 assay (<a href="#pone.0330306.g004" class="usa-link">Fig 4B</a>). In this assay, WST-8 is reduced by mitochondrial dehydrogenase to produce orange-colored formazan [<a href="#pone.0330306.ref037" class="usa-link" aria-describedby="pone.0330306.ref037">37</a>]. As shown in <a href="#pone.0330306.g004" class="usa-link">Fig 4C</a>, similar color changes were observed between the control and 3% Triton groups at the pre-treat stage, with no significant differences in O.D. (<a href="#pone.0330306.g004" class="usa-link">Fig 4D</a>). However, after post-treat stage, the control group showed no change in color, whereas the 3% Triton group exhibited only a faint orange color (<a href="#pone.0330306.g004" class="usa-link">Fig 4C</a>). Correspondingly, a significant decrease in O.D. was observed in the 3% Triton group (pre-treat: 1.053 ± 0.158; post-treat: 0.045 ± 0.002), while the control group maintained relatively stable O.D. (pre-treat: 1.336 ± 0.109; post-treat: 1.272 ± 0.179) (<a href="#pone.0330306.g004" class="usa-link">Fig 4D</a>). These results suggest that 3% Triton induced cell death in hiPSC-SKEs, leading to reduced viability. Morphological analysis using H&amp;E staining further confirmed severe damage to the epidermal layer near the air interface in the 3% Triton group, which was not observed in the control group (<a href="#pone.0330306.g004" class="usa-link">Fig 4E</a>). In summary, hiPSC-SKE has been proven to be a valuable model for skin safety assessment of various skin irritation.</p></section></section><section id="sec018"><h2 class="pmc_sec_title">Discussion</h2>
<p>Our study demonstrated that hiPSC-derived skin equivalents (hiPSC-SKEs) can serve as a viable alternative to conventional primary skin cell-based models. hiPSC-SKE was generated by differentiating hiPSC-derived fibroblasts and keratinocytes. The formation of skin composition layers resembling native skin was confirmed, along with the cellular distribution within each layer. Furthermore, treatment of hiPSC-SKEs with 3% Triton X-100, followed by assessments of cell viability and morphological changes, supported their potential utility in skin irritation testing.</p>
<p>Previous studies have reported the development of SKEs using hiPSC-derived skin cells as substitutes for primary skin cells. Some of these models incorporated a mixture of hiPSC-derived and primary skin cells [<a href="#pone.0330306.ref018" class="usa-link" aria-describedby="pone.0330306.ref018">18</a>,<a href="#pone.0330306.ref020" class="usa-link" aria-describedby="pone.0330306.ref020">20</a>], while others were constructed entirely from hiPSC-derived fibroblasts and keratinocytes [<a href="#pone.0330306.ref018" class="usa-link" aria-describedby="pone.0330306.ref018">18</a>,<a href="#pone.0330306.ref019" class="usa-link" aria-describedby="pone.0330306.ref019">19</a>,<a href="#pone.0330306.ref021" class="usa-link" aria-describedby="pone.0330306.ref021">21</a>]. When compared to previous reports, our SKEs showed morphological differences—such as variations in epidermal and SC thickness and dermal cell distribution. These differences may result from factors such as cell density, culture period, and media composition. Despite these variations, most SKEs, including ours, exhibited structural patterns comparable to native human skin. Epidermal maturation in native skin proceeds through keratinization, starting from the SB and forming the SS, SG, and SC toward the air interface. In the SB, proliferating keratinocytes predominantly express KRT14 and ΔNp63, as reported in native skin [<a href="#pone.0330306.ref018" class="usa-link" aria-describedby="pone.0330306.ref018">18</a>–<a href="#pone.0330306.ref021" class="usa-link" aria-describedby="pone.0330306.ref021">21</a>]. Consistent with this, our results confirmed that KRT14 and ΔNp63 were strongly expressed in the basal region adjacent to the dermis in hiPSC-SKEs generated by ALI culture, with expression levels gradually diminishing toward the surface (<a href="#pone.0330306.g003" class="usa-link">Fig 3A</a> and <a href="#pone.0330306.g003" class="usa-link">3B</a>) [<a href="#pone.0330306.ref038" class="usa-link" aria-describedby="pone.0330306.ref038">38</a>]. In native skin, KRT1 and KRT10 are primarily expressed in the SS, while loricrin and filaggrin are mainly expressed in the SC [<a href="#pone.0330306.ref039" class="usa-link" aria-describedby="pone.0330306.ref039">39</a>]. In our hiPSC-SKEs, we also demonstrated that the KRT10<sup>+</sup> layer thickened as ALI culture progressed, and loricrin, which was not observed at ALI-10 days, became detectable by ALI-20 days (<a href="#pone.0330306.g003" class="usa-link">Fig 3C</a> and <a href="#pone.0330306.g003" class="usa-link">3D</a>). In the dermis, fibroblasts exhibit a spindle-shaped morphology typical of mesenchymal cells [<a href="#pone.0330306.ref015" class="usa-link" aria-describedby="pone.0330306.ref015">15</a>]. Similarly, our IF analyses for vimentin and PDGFRα demonstrated that fibroblasts in hiPSC-SKE were distributed in a spindle shape (<a href="#pone.0330306.g003" class="usa-link">Fig 3C</a> and <a href="#pone.0330306.g003" class="usa-link">3D</a>). The epidermis-dermis junction, also known as the BM, connects the dermis to the SB layer of the epidermis [<a href="#pone.0330306.ref040" class="usa-link" aria-describedby="pone.0330306.ref040">40</a>]. Type IV collagen is the major structural component of the BM, forming a meshwork that incorporates laminin and fibronectin [<a href="#pone.0330306.ref040" class="usa-link" aria-describedby="pone.0330306.ref040">40</a>]. Fibronectin contributes to basal lamina synthesis and promotes keratinocyte regeneration in the epidermis by binding to the collagen basal lamina [<a href="#pone.0330306.ref041" class="usa-link" aria-describedby="pone.0330306.ref041">41</a>]. IF analysis revealed that fibronectin was expressed in the dermal layer of hiPSC-SKE, with particularly high expression in the BM region (<a href="#pone.0330306.g003" class="usa-link">Fig 3E</a>).</p>
<p>Skin irritants penetrate the SC and damage basal skin cells, triggering an inflammatory response in the dermis that leads to erythema and edema, which are key signs of skin irritation. To evaluate this <em>in vitro</em>, artificial skin models have been developed as alternatives to animal testing, supported by a robust database that now serves as a guideline. Skin irritation testing using SKEs typically involves cell viability assays, such as the MTT assay, after topical application of test substances [<a href="#pone.0330306.ref007" class="usa-link" aria-describedby="pone.0330306.ref007">7</a>–<a href="#pone.0330306.ref009" class="usa-link" aria-describedby="pone.0330306.ref009">9</a>]. According to the United Nations Globally Harmonized System of Classification and Labeling of Chemicals (UN GHS), a substance is classified as non-irritant if the average tissue viability after treatment exceeds 50%. In contrast, substances resulting in ≤50% viability are classified as irritants, and further assessment for skin corrosivity is required for detailed categorization. In this study, the % change in post-treat O.D. values relative to pre-treat values was analyzed for each group. The viability rate of hiPSC-SKEs was 94.206 ± 8.371% in the control group, compared to 4.621 ± 0.738% in the group treated with 3% Triton (<a href="#pone.0330306.g004" class="usa-link">Fig 4D</a>). These results confirm that Triton X-100, a well-established positive control for skin irritation in primary cell-based SKE models, also induces significant irritation in hiPSC-SKEs.</p>
<p>hiPSC-SKEs demonstrated marker expression and irritation responses comparable to primary skin cell-based SKEs, supporting their potential as reliable alternatives. hiPSCs offer advantages such as overcoming limitations in cell supply, cost, and donor variability. However, a key challenge remains as hiPSC-derived cells often exhibit an immature phenotype [<a href="#pone.0330306.ref042" class="usa-link" aria-describedby="pone.0330306.ref042">42</a>,<a href="#pone.0330306.ref043" class="usa-link" aria-describedby="pone.0330306.ref043">43</a>]. Similarly, hiPSC-derived skin organoids lack structural completeness, such as Collagen VII and immune or endothelial cells [<a href="#pone.0330306.ref044" class="usa-link" aria-describedby="pone.0330306.ref044">44</a>,<a href="#pone.0330306.ref045" class="usa-link" aria-describedby="pone.0330306.ref045">45</a>]. This immaturity may also affect hiPSC-derived skin cells, potentially explaining the need for ALI culture durations exceeding 20 days. Moreover, SKEs generated from primary skin cells often display a thicker SC compared to hiPSC-SKEs [<a href="#pone.0330306.ref020" class="usa-link" aria-describedby="pone.0330306.ref020">20</a>,<a href="#pone.0330306.ref024" class="usa-link" aria-describedby="pone.0330306.ref024">24</a>]. To enhance model maturity, future studies should incorporate additional skin constituent cells to better reflect <em>in vivo</em> conditions. Batch-to-batch variability during hiPSC differentiation also poses challenges for reproducibility. To address this, we are exploring previously reported droplet-based bioprinting and nebulization technologies to standardize hiPSC-SKE fabrication. In summary, the hiPSC-SKE model offers a scalable, physiologically relevant platform with promising applications in safety testing, disease modeling, and regenerative medicine.</p></section><section id="sec019"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pone.0330306.s001"><div class="caption p">
<span>S1 File</span><p><strong>S1 Table. Infromation of primers.</strong> Genes: GAPDH (housekeeping gene for control), puripotency marekrs (OCT4 and SOX2), fibroblast-specific markers (PDGFRα, Col3A1, and Fibronectin), keratinocyte-specific markers (KRT14 and ΔNp63). <strong>S1 Fig. IF staining for purity check in hFIBRO and hKERA.</strong> (A) Staining for vimentin, a fibroblast-specific marker, in hFIBRO and distribution of vimentin+ cells per fields (n = 3). (B) Staining for ΔNp63, a keratinocyte-specific marker, in hKERA and distribution of ΔNp63 + cells per fields (n = 3). White arrows: DAPI where each marker is not expressed. Scale bar: 100 µm. <strong>S2 Fig. Electrophoresis analysis for pluripotency and skin cell-specific markers by hiPSC, hFIBRO, and hKERA.</strong> Genes: GAPDH (housekeeping gene for control), puripotency marekrs (OCT4 and SOX2), fibroblast-specific markers (PDGFRα, Col3A1, and Fibronectin), keratinocyte-specific markers (KRT14 and ΔNp63).</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360594/bin/pone.0330306.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330306.s001.docx</a><sup> (826.8KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank all members of the team for their great support.</p></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>All relevant data are within the paper and its <a href="#sec019" class="usa-link">Supporting Information</a> files.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This work was supported by the Industrial Strategic Technology Development Program (RS-2024-00448561) funded by the Ministry of Trade, Industry &amp; Energy (MOTIE, Korea) in 2025. This paper was supported by the KU Research Professor Program of Konkuk University.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pone.0330306.ref001">
<span class="label">1.</span><cite>Simpson CL, Patel DM, Green KJ. Deconstructing the skin: cytoarchitectural determinants of epidermal morphogenesis. Nat Rev Mol Cell Biol. 2011;12(9):565–80. doi: 10.1038/nrm3175

</cite> [<a href="https://doi.org/10.1038/nrm3175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3280198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21860392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&amp;title=Deconstructing%20the%20skin:%20cytoarchitectural%20determinants%20of%20epidermal%20morphogenesis&amp;author=CL%20Simpson&amp;author=DM%20Patel&amp;author=KJ%20Green&amp;volume=12&amp;issue=9&amp;publication_year=2011&amp;pages=565-80&amp;pmid=21860392&amp;doi=10.1038/nrm3175&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref002">
<span class="label">2.</span><cite>Brown TM, Krishnamurthy K. Histology, Dermis. StatPearls. Treasure Island (FL) ineligible companies. Disclosure: Karthik Krishnamurthy declares no relevant financial relationships with ineligible companies. StatPearls Publishing; 2024.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Disclosure:%20Karthik%20Krishnamurthy%20declares%20no%20relevant%20financial%20relationships%20with%20ineligible%20companies&amp;author=TM%20Brown&amp;author=K%20Krishnamurthy&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref003">
<span class="label">3.</span><cite>Choudhury S, Das A. Advances in generation of three-dimensional skin equivalents: pre-clinical studies to clinical therapies. Cytotherapy. 2021;23(1):1–9. doi: 10.1016/j.jcyt.2020.10.001

</cite> [<a href="https://doi.org/10.1016/j.jcyt.2020.10.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8079860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33189572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cytotherapy&amp;title=Advances%20in%20generation%20of%20three-dimensional%20skin%20equivalents:%20pre-clinical%20studies%20to%20clinical%20therapies&amp;author=S%20Choudhury&amp;author=A%20Das&amp;volume=23&amp;issue=1&amp;publication_year=2021&amp;pages=1-9&amp;pmid=33189572&amp;doi=10.1016/j.jcyt.2020.10.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref004">
<span class="label">4.</span><cite>Hofmann E, Schwarz A, Fink J, Kamolz LP, Kotzbeck P. Modelling the Complexity of Human Skin In Vitro. Biomedicines. 2023;11(3).</cite> [<a href="https://doi.org/10.3390/biomedicines11030794" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10045055/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36979772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomedicines&amp;title=Modelling%20the%20Complexity%20of%20Human%20Skin%20In%20Vitro&amp;author=E%20Hofmann&amp;author=A%20Schwarz&amp;author=J%20Fink&amp;author=LP%20Kamolz&amp;author=P%20Kotzbeck&amp;volume=11&amp;issue=3&amp;publication_year=2023&amp;pmid=36979772&amp;doi=10.3390/biomedicines11030794&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref005">
<span class="label">5.</span><cite>Randall MJ, Jüngel A, Rimann M, Wuertz-Kozak K. Advances in the Biofabrication of 3D Skin in vitro: Healthy and Pathological Models. Front Bioeng Biotechnol. 2018;6:154. doi: 10.3389/fbioe.2018.00154

</cite> [<a href="https://doi.org/10.3389/fbioe.2018.00154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6220074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30430109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Bioeng%20Biotechnol&amp;title=Advances%20in%20the%20Biofabrication%20of%203D%20Skin%20in%20vitro:%20Healthy%20and%20Pathological%20Models&amp;author=MJ%20Randall&amp;author=A%20J%C3%BCngel&amp;author=M%20Rimann&amp;author=K%20Wuertz-Kozak&amp;volume=6&amp;publication_year=2018&amp;pages=154&amp;pmid=30430109&amp;doi=10.3389/fbioe.2018.00154&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref006">
<span class="label">6.</span><cite>Rossi A, Appelt-Menzel A, Kurdyn S, Walles H, Groeber F. Generation of a three-dimensional full thickness skin equivalent and automated wounding. Journal of Visualized Experiments: JoVE. 2015;96.</cite> [<a href="https://doi.org/10.3791/52576" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4354673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25741763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Visualized%20Experiments:%20JoVE&amp;title=Generation%20of%20a%20three-dimensional%20full%20thickness%20skin%20equivalent%20and%20automated%20wounding&amp;author=A%20Rossi&amp;author=A%20Appelt-Menzel&amp;author=S%20Kurdyn&amp;author=H%20Walles&amp;author=F%20Groeber&amp;volume=96&amp;publication_year=2015&amp;pmid=25741763&amp;doi=10.3791/52576&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref007">
<span class="label">7.</span><cite>Kang N-H, Oh J. The Accomplishments of KoCVAM in the Development and Implementation of Alternative Methods in Korea. Altern Lab Anim. 2024;52(3):166–76. doi: 10.1177/02611929241244520

</cite> [<a href="https://doi.org/10.1177/02611929241244520" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38626463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Altern%20Lab%20Anim&amp;title=The%20Accomplishments%20of%20KoCVAM%20in%20the%20Development%20and%20Implementation%20of%20Alternative%20Methods%20in%20Korea&amp;author=N-H%20Kang&amp;author=J%20Oh&amp;volume=52&amp;issue=3&amp;publication_year=2024&amp;pages=166-76&amp;pmid=38626463&amp;doi=10.1177/02611929241244520&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref008">
<span class="label">8.</span><cite>Liao C-C, Wu C-Y, Lin M-H, Hsieh F-K, Hsu L-T, Chang S-Y, et al. Validation study of a new reconstructed human epidermis model EPiTRI for in vitro skin irritation test according to OECD guidelines. Toxicol In Vitro. 2021;75:105197. doi: 10.1016/j.tiv.2021.105197

</cite> [<a href="https://doi.org/10.1016/j.tiv.2021.105197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34062233/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Toxicol%20In%20Vitro&amp;title=Validation%20study%20of%20a%20new%20reconstructed%20human%20epidermis%20model%20EPiTRI%20for%20in%20vitro%20skin%20irritation%20test%20according%20to%20OECD%20guidelines&amp;author=C-C%20Liao&amp;author=C-Y%20Wu&amp;author=M-H%20Lin&amp;author=F-K%20Hsieh&amp;author=L-T%20Hsu&amp;volume=75&amp;publication_year=2021&amp;pages=105197&amp;pmid=34062233&amp;doi=10.1016/j.tiv.2021.105197&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref009">
<span class="label">9.</span><cite>Jung K-M, Lee S-H, Jang W-H, Jung H-S, Heo Y, Park Y-H, et al. KeraSkin-VM: a novel reconstructed human epidermis model for skin irritation tests. Toxicol In Vitro. 2014;28(5):742–50. doi: 10.1016/j.tiv.2014.02.014

</cite> [<a href="https://doi.org/10.1016/j.tiv.2014.02.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24625437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Toxicol%20In%20Vitro&amp;title=KeraSkin-VM:%20a%20novel%20reconstructed%20human%20epidermis%20model%20for%20skin%20irritation%20tests&amp;author=K-M%20Jung&amp;author=S-H%20Lee&amp;author=W-H%20Jang&amp;author=H-S%20Jung&amp;author=Y%20Heo&amp;volume=28&amp;issue=5&amp;publication_year=2014&amp;pages=742-50&amp;pmid=24625437&amp;doi=10.1016/j.tiv.2014.02.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref010">
<span class="label">10.</span><cite>Wadman M. FDA no longer has to require animal testing for new drugs. Science. 2023;379(6628):127–8. doi: 10.1126/science.adg6276

</cite> [<a href="https://doi.org/10.1126/science.adg6276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36634170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=FDA%20no%20longer%20has%20to%20require%20animal%20testing%20for%20new%20drugs&amp;author=M%20Wadman&amp;volume=379&amp;issue=6628&amp;publication_year=2023&amp;pages=127-8&amp;pmid=36634170&amp;doi=10.1126/science.adg6276&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref011">
<span class="label">11.</span><cite>Han JJ. FDA Modernization Act 2.0 Allows for Alternatives to Animal Testing. Artificial Organs. 2023;47(3):449–50.
</cite> [<a href="https://doi.org/10.1111/aor.14503" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36762462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Artificial%20Organs&amp;title=FDA%20Modernization%20Act%202.0%20Allows%20for%20Alternatives%20to%20Animal%20Testing&amp;author=JJ%20Han&amp;volume=47&amp;issue=3&amp;publication_year=2023&amp;pages=449-50&amp;pmid=36762462&amp;doi=10.1111/aor.14503&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref012">
<span class="label">12.</span><cite>Reijnders CM, van Lier A, Roffel S, Kramer D, Scheper RJ, Gibbs S. Development of a full-thickness human skin equivalent in vitro model derived from TERT-immortalized keratinocytes and fibroblasts. Tissue engineering Part A. 2015;21(17–18):2448–59.
</cite> [<a href="https://doi.org/10.1089/ten.tea.2015.0139" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4554934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26135533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Tissue%20engineering%20Part%20A&amp;title=Development%20of%20a%20full-thickness%20human%20skin%20equivalent%20in%20vitro%20model%20derived%20from%20TERT-immortalized%20keratinocytes%20and%20fibroblasts&amp;author=CM%20Reijnders&amp;author=A%20van%20Lier&amp;author=S%20Roffel&amp;author=D%20Kramer&amp;author=RJ%20Scheper&amp;volume=21&amp;publication_year=2015&amp;pages=2448-59&amp;pmid=26135533&amp;doi=10.1089/ten.tea.2015.0139&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref013">
<span class="label">13.</span><cite>Risueño I, Valencia L, Jorcano JL, Velasco D. Skin-on-a-chip models: General overview and future perspectives. APL Bioeng. 2021;5(3):030901. doi: 10.1063/5.0046376

</cite> [<a href="https://doi.org/10.1063/5.0046376" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8270645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34258497/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=APL%20Bioeng&amp;title=Skin-on-a-chip%20models:%20General%20overview%20and%20future%20perspectives&amp;author=I%20Risue%C3%B1o&amp;author=L%20Valencia&amp;author=JL%20Jorcano&amp;author=D%20Velasco&amp;volume=5&amp;issue=3&amp;publication_year=2021&amp;pages=030901&amp;pmid=34258497&amp;doi=10.1063/5.0046376&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref014">
<span class="label">14.</span><cite>Mieremet A, Vázquez García A, Boiten W, van Dijk R, Gooris G, Bouwstra JA, et al. Human skin equivalents cultured under hypoxia display enhanced epidermal morphogenesis and lipid barrier formation. Sci Rep. 2019;9(1):7811. doi: 10.1038/s41598-019-44204-4

</cite> [<a href="https://doi.org/10.1038/s41598-019-44204-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6534609/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31127151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Human%20skin%20equivalents%20cultured%20under%20hypoxia%20display%20enhanced%20epidermal%20morphogenesis%20and%20lipid%20barrier%20formation&amp;author=A%20Mieremet&amp;author=A%20V%C3%A1zquez%20Garc%C3%ADa&amp;author=W%20Boiten&amp;author=R%20van%20Dijk&amp;author=G%20Gooris&amp;volume=9&amp;issue=1&amp;publication_year=2019&amp;pages=7811&amp;pmid=31127151&amp;doi=10.1038/s41598-019-44204-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref015">
<span class="label">15.</span><cite>Lee S-G, Lee S, Bae H-K, Lee KY, Park C, Kim M sup, et al. Evaluation of the therapeutic efficacy of human skin equivalents manufactured through droplet-based bioprinting/nebulization technology. Mol Cell Toxicol. 2024;20(1):129–38. doi: 10.1007/s13273-023-00330-9</cite> [<a href="https://doi.org/10.1007/s13273-023-00330-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Toxicol&amp;title=Evaluation%20of%20the%20therapeutic%20efficacy%20of%20human%20skin%20equivalents%20manufactured%20through%20droplet-based%20bioprinting/nebulization%20technology&amp;author=S-G%20Lee&amp;author=S%20Lee&amp;author=H-K%20Bae&amp;author=KY%20Lee&amp;author=C%20Park&amp;volume=20&amp;issue=1&amp;publication_year=2024&amp;pages=129-38&amp;doi=10.1007/s13273-023-00330-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref016">
<span class="label">16.</span><cite>Gorecka J, Kostiuk V, Fereydooni A, Gonzalez L, Luo J, Dash B, et al. The potential and limitations of induced pluripotent stem cells to achieve wound healing. Stem Cell Res Ther. 2019;10(1):87. doi: 10.1186/s13287-019-1185-1

</cite> [<a href="https://doi.org/10.1186/s13287-019-1185-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6416973/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30867069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stem%20Cell%20Res%20Ther&amp;title=The%20potential%20and%20limitations%20of%20induced%20pluripotent%20stem%20cells%20to%20achieve%20wound%20healing&amp;author=J%20Gorecka&amp;author=V%20Kostiuk&amp;author=A%20Fereydooni&amp;author=L%20Gonzalez&amp;author=J%20Luo&amp;volume=10&amp;issue=1&amp;publication_year=2019&amp;pages=87&amp;pmid=30867069&amp;doi=10.1186/s13287-019-1185-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref017">
<span class="label">17.</span><cite>Cerneckis J, Cai H, Shi Y. Induced pluripotent stem cells (iPSCs): molecular mechanisms of induction and applications. Signal Transduct Target Ther. 2024;9(1):112. doi: 10.1038/s41392-024-01809-0

</cite> [<a href="https://doi.org/10.1038/s41392-024-01809-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11053163/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38670977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduct%20Target%20Ther&amp;title=Induced%20pluripotent%20stem%20cells%20(iPSCs):%20molecular%20mechanisms%20of%20induction%20and%20applications&amp;author=J%20Cerneckis&amp;author=H%20Cai&amp;author=Y%20Shi&amp;volume=9&amp;issue=1&amp;publication_year=2024&amp;pages=112&amp;pmid=38670977&amp;doi=10.1038/s41392-024-01809-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref018">
<span class="label">18.</span><cite>Itoh M, Umegaki-Arao N, Guo Z, Liu L, Higgins CA, Christiano AM. Generation of 3D skin equivalents fully reconstituted from human induced pluripotent stem cells (iPSCs). PLoS One. 2013;8(10):e77673. doi: 10.1371/journal.pone.0077673

</cite> [<a href="https://doi.org/10.1371/journal.pone.0077673" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3795682/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24147053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Generation%20of%203D%20skin%20equivalents%20fully%20reconstituted%20from%20human%20induced%20pluripotent%20stem%20cells%20(iPSCs)&amp;author=M%20Itoh&amp;author=N%20Umegaki-Arao&amp;author=Z%20Guo&amp;author=L%20Liu&amp;author=CA%20Higgins&amp;volume=8&amp;issue=10&amp;publication_year=2013&amp;pmid=24147053&amp;doi=10.1371/journal.pone.0077673&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref019">
<span class="label">19.</span><cite>Gledhill K, Guo Z, Umegaki-Arao N, Higgins CA, Itoh M, Christiano AM. Melanin Transfer in Human 3D Skin Equivalents Generated Exclusively from Induced Pluripotent Stem Cells. PLoS One. 2015;10(8):e0136713. doi: 10.1371/journal.pone.0136713

</cite> [<a href="https://doi.org/10.1371/journal.pone.0136713" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4550351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26308443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Melanin%20Transfer%20in%20Human%203D%20Skin%20Equivalents%20Generated%20Exclusively%20from%20Induced%20Pluripotent%20Stem%20Cells&amp;author=K%20Gledhill&amp;author=Z%20Guo&amp;author=N%20Umegaki-Arao&amp;author=CA%20Higgins&amp;author=M%20Itoh&amp;volume=10&amp;issue=8&amp;publication_year=2015&amp;pmid=26308443&amp;doi=10.1371/journal.pone.0136713&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref020">
<span class="label">20.</span><cite>Kajiwara K, Tanemoto T, Wada S, Karibe J, Ihara N, Ikemoto Y, et al. Fetal Therapy Model of Myelomeningocele with Three-Dimensional Skin Using Amniotic Fluid Cell-Derived Induced Pluripotent Stem Cells. Stem Cell Reports. 2017;8(6):1701–13. doi: 10.1016/j.stemcr.2017.05.013

</cite> [<a href="https://doi.org/10.1016/j.stemcr.2017.05.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5470234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28591652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stem%20Cell%20Reports&amp;title=Fetal%20Therapy%20Model%20of%20Myelomeningocele%20with%20Three-Dimensional%20Skin%20Using%20Amniotic%20Fluid%20Cell-Derived%20Induced%20Pluripotent%20Stem%20Cells&amp;author=K%20Kajiwara&amp;author=T%20Tanemoto&amp;author=S%20Wada&amp;author=J%20Karibe&amp;author=N%20Ihara&amp;volume=8&amp;issue=6&amp;publication_year=2017&amp;pages=1701-13&amp;pmid=28591652&amp;doi=10.1016/j.stemcr.2017.05.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref021">
<span class="label">21.</span><cite>Kim Y, Park N, Rim YA, Nam Y, Jung H, Lee K, et al. Establishment of a complex skin structure via layered co-culture of keratinocytes and fibroblasts derived from induced pluripotent stem cells. Stem Cell Res Ther. 2018;9(1):217. doi: 10.1186/s13287-018-0958-2

</cite> [<a href="https://doi.org/10.1186/s13287-018-0958-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6090613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30103800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stem%20Cell%20Res%20Ther&amp;title=Establishment%20of%20a%20complex%20skin%20structure%20via%20layered%20co-culture%20of%20keratinocytes%20and%20fibroblasts%20derived%20from%20induced%20pluripotent%20stem%20cells&amp;author=Y%20Kim&amp;author=N%20Park&amp;author=YA%20Rim&amp;author=Y%20Nam&amp;author=H%20Jung&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pages=217&amp;pmid=30103800&amp;doi=10.1186/s13287-018-0958-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref022">
<span class="label">22.</span><cite>Bang JS, Choi NY, Lee M, Ko K, Lee HJ, Park YS, et al. Optimization of episomal reprogramming for generation of human induced pluripotent stem cells from fibroblasts. Anim Cells Syst (Seoul). 2018;22(2):132–9. doi: 10.1080/19768354.2018.1451367

</cite> [<a href="https://doi.org/10.1080/19768354.2018.1451367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6138300/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30460090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anim%20Cells%20Syst%20(Seoul)&amp;title=Optimization%20of%20episomal%20reprogramming%20for%20generation%20of%20human%20induced%20pluripotent%20stem%20cells%20from%20fibroblasts&amp;author=JS%20Bang&amp;author=NY%20Choi&amp;author=M%20Lee&amp;author=K%20Ko&amp;author=HJ%20Lee&amp;volume=22&amp;issue=2&amp;publication_year=2018&amp;pages=132-9&amp;pmid=30460090&amp;doi=10.1080/19768354.2018.1451367&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref023">
<span class="label">23.</span><cite>Lee S-G, Kim Y-J, Son M-Y, Oh M-S, Kim J, Ryu B, et al. Generation of human iPSCs derived heart organoids structurally and functionally similar to heart. Biomaterials. 2022;290:121860. doi: 10.1016/j.biomaterials.2022.121860

</cite> [<a href="https://doi.org/10.1016/j.biomaterials.2022.121860" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36274511/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomaterials&amp;title=Generation%20of%20human%20iPSCs%20derived%20heart%20organoids%20structurally%20and%20functionally%20similar%20to%20heart&amp;author=S-G%20Lee&amp;author=Y-J%20Kim&amp;author=M-Y%20Son&amp;author=M-S%20Oh&amp;author=J%20Kim&amp;volume=290&amp;publication_year=2022&amp;pages=121860&amp;pmid=36274511&amp;doi=10.1016/j.biomaterials.2022.121860&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref024">
<span class="label">24.</span><cite>Domingues S, Darle A, Masson Y, Saidani M, Lagoutte E, Bejanariu A, et al. Clinical Grade Human Pluripotent Stem Cell-Derived Engineered Skin Substitutes Promote Keratinocytes Wound Closure In Vitro. Cells. 2022;11(7):1151. doi: 10.3390/cells11071151

</cite> [<a href="https://doi.org/10.3390/cells11071151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8998132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35406716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=Clinical%20Grade%20Human%20Pluripotent%20Stem%20Cell-Derived%20Engineered%20Skin%20Substitutes%20Promote%20Keratinocytes%20Wound%20Closure%20In%20Vitro&amp;author=S%20Domingues&amp;author=A%20Darle&amp;author=Y%20Masson&amp;author=M%20Saidani&amp;author=E%20Lagoutte&amp;volume=11&amp;issue=7&amp;publication_year=2022&amp;pages=1151&amp;pmid=35406716&amp;doi=10.3390/cells11071151&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref025">
<span class="label">25.</span><cite>Kim Y, Ju J. Generation of 3D skin organoid from cord blood-derived induced pluripotent stem cells. Journal of Visualized Experiments. 2019.</cite> [<a href="https://doi.org/10.3791/59297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31058887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Visualized%20Experiments&amp;title=Generation%20of%203D%20skin%20organoid%20from%20cord%20blood-derived%20induced%20pluripotent%20stem%20cells&amp;author=Y%20Kim&amp;author=J%20Ju&amp;publication_year=2019&amp;pmid=31058887&amp;doi=10.3791/59297&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref026">
<span class="label">26.</span><cite>Yajing S, Rajput IR, Ying H, Fei Y, Sanganyado E, Ping L, et al. Establishment and characterization of pygmy killer whale (Feresa attenuata) dermal fibroblast cell line. PLoS One. 2018;13(3):e0195128. doi: 10.1371/journal.pone.0195128

</cite> [<a href="https://doi.org/10.1371/journal.pone.0195128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5875847/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29596530/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Establishment%20and%20characterization%20of%20pygmy%20killer%20whale%20(Feresa%20attenuata)%20dermal%20fibroblast%20cell%20line&amp;author=S%20Yajing&amp;author=IR%20Rajput&amp;author=H%20Ying&amp;author=Y%20Fei&amp;author=E%20Sanganyado&amp;volume=13&amp;issue=3&amp;publication_year=2018&amp;pmid=29596530&amp;doi=10.1371/journal.pone.0195128&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref027">
<span class="label">27.</span><cite>Petrini I, Barachini S, Carnicelli V, Galimberti S, Modeo L, Boni R, et al. ED-B fibronectin expression is a marker of epithelial-mesenchymal transition in translational oncology. Oncotarget. 2017;8(3):4914–21. doi: 10.18632/oncotarget.13615

</cite> [<a href="https://doi.org/10.18632/oncotarget.13615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5354880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27902486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=ED-B%20fibronectin%20expression%20is%20a%20marker%20of%20epithelial-mesenchymal%20transition%20in%20translational%20oncology&amp;author=I%20Petrini&amp;author=S%20Barachini&amp;author=V%20Carnicelli&amp;author=S%20Galimberti&amp;author=L%20Modeo&amp;volume=8&amp;issue=3&amp;publication_year=2017&amp;pages=4914-21&amp;pmid=27902486&amp;doi=10.18632/oncotarget.13615&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref028">
<span class="label">28.</span><cite>Gatti V, Fierro C, Compagnone M, La Banca V, Mauriello A, Montanaro M, et al. ΔNp63-Senataxin circuit controls keratinocyte differentiation by promoting the transcriptional termination of epidermal genes. Proc Natl Acad Sci U S A. 2022;119(10):e2104718119. doi: 10.1073/pnas.2104718119

</cite> [<a href="https://doi.org/10.1073/pnas.2104718119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8915885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35235452/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=%CE%94Np63-Senataxin%20circuit%20controls%20keratinocyte%20differentiation%20by%20promoting%20the%20transcriptional%20termination%20of%20epidermal%20genes&amp;author=V%20Gatti&amp;author=C%20Fierro&amp;author=M%20Compagnone&amp;author=V%20La%20Banca&amp;author=A%20Mauriello&amp;volume=119&amp;issue=10&amp;publication_year=2022&amp;pmid=35235452&amp;doi=10.1073/pnas.2104718119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref029">
<span class="label">29.</span><cite>Wu N, Castel D, Debily M-A, Vigano MA, Alibert O, Mantovani R, et al. Large scale RNAi screen reveals that the inhibitor of DNA binding 2 (ID2) protein is repressed by p53 family member p63 and functions in human keratinocyte differentiation. J Biol Chem. 2011;286(23):20870–9. doi: 10.1074/jbc.M110.169433

</cite> [<a href="https://doi.org/10.1074/jbc.M110.169433" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3121489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21478550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Large%20scale%20RNAi%20screen%20reveals%20that%20the%20inhibitor%20of%20DNA%20binding%202%20(ID2)%20protein%20is%20repressed%20by%20p53%20family%20member%20p63%20and%20functions%20in%20human%20keratinocyte%20differentiation&amp;author=N%20Wu&amp;author=D%20Castel&amp;author=M-A%20Debily&amp;author=MA%20Vigano&amp;author=O%20Alibert&amp;volume=286&amp;issue=23&amp;publication_year=2011&amp;pages=20870-9&amp;pmid=21478550&amp;doi=10.1074/jbc.M110.169433&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref030">
<span class="label">30.</span><cite>Guo Y, Redmond CJ, Leacock KA, Brovkina MV, Ji S, Jaskula-Ranga V. Keratin 14-dependent disulfides regulate epidermal homeostasis and barrier function via 14-3-3σ and YAP1. eLife. 2020;9.</cite> [<a href="https://doi.org/10.7554/eLife.53165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7250575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32369015/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Keratin%2014-dependent%20disulfides%20regulate%20epidermal%20homeostasis%20and%20barrier%20function%20via%2014-3-3%CF%83%20and%20YAP1&amp;author=Y%20Guo&amp;author=CJ%20Redmond&amp;author=KA%20Leacock&amp;author=MV%20Brovkina&amp;author=S%20Ji&amp;volume=9&amp;publication_year=2020&amp;pmid=32369015&amp;doi=10.7554/eLife.53165&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref031">
<span class="label">31.</span><cite>Alam H, Sehgal L, Kundu ST, Dalal SN, Vaidya MM. Novel function of keratins 5 and 14 in proliferation and differentiation of stratified epithelial cells. Mol Biol Cell. 2011;22(21):4068–78. doi: 10.1091/mbc.E10-08-0703

</cite> [<a href="https://doi.org/10.1091/mbc.E10-08-0703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3204069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21900500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Cell&amp;title=Novel%20function%20of%20keratins%205%20and%2014%20in%20proliferation%20and%20differentiation%20of%20stratified%20epithelial%20cells&amp;author=H%20Alam&amp;author=L%20Sehgal&amp;author=ST%20Kundu&amp;author=SN%20Dalal&amp;author=MM%20Vaidya&amp;volume=22&amp;issue=21&amp;publication_year=2011&amp;pages=4068-78&amp;pmid=21900500&amp;doi=10.1091/mbc.E10-08-0703&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref032">
<span class="label">32.</span><cite>Bialas I, Zelent-Kraciuk S, Jurowski K. The skin sensitisation of cosmetic ingredients: review of actual regulatory status. Toxics. 2023;11(4).</cite> [<a href="https://doi.org/10.3390/toxics11040392" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10146005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37112619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Toxics&amp;title=The%20skin%20sensitisation%20of%20cosmetic%20ingredients:%20review%20of%20actual%20regulatory%20status&amp;author=I%20Bialas&amp;author=S%20Zelent-Kraciuk&amp;author=K%20Jurowski&amp;volume=11&amp;issue=4&amp;publication_year=2023&amp;pmid=37112619&amp;doi=10.3390/toxics11040392&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref033">
<span class="label">33.</span><cite>Shahraki S, Bideskan AE, Aslzare M, Tavakkoli M, Bahrami AR, Hosseinian S, et al. Decellularization with triton X-100 provides a suitable model for human kidney bioengineering using human mesenchymal stem cells. Life Sci. 2022;295:120167. doi: 10.1016/j.lfs.2021.120167

</cite> [<a href="https://doi.org/10.1016/j.lfs.2021.120167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34822795/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Life%20Sci&amp;title=Decellularization%20with%20triton%20X-100%20provides%20a%20suitable%20model%20for%20human%20kidney%20bioengineering%20using%20human%20mesenchymal%20stem%20cells&amp;author=S%20Shahraki&amp;author=AE%20Bideskan&amp;author=M%20Aslzare&amp;author=M%20Tavakkoli&amp;author=AR%20Bahrami&amp;volume=295&amp;publication_year=2022&amp;pages=120167&amp;pmid=34822795&amp;doi=10.1016/j.lfs.2021.120167&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref034">
<span class="label">34.</span><cite>Mattei B, Lira RB, Perez KR, Riske KA. Membrane permeabilization induced by Triton X-100: The role of membrane phase state and edge tension. Chem Phys Lipids. 2017;202:28–37. doi: 10.1016/j.chemphyslip.2016.11.009

</cite> [<a href="https://doi.org/10.1016/j.chemphyslip.2016.11.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27913102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem%20Phys%20Lipids&amp;title=Membrane%20permeabilization%20induced%20by%20Triton%20X-100:%20The%20role%20of%20membrane%20phase%20state%20and%20edge%20tension&amp;author=B%20Mattei&amp;author=RB%20Lira&amp;author=KR%20Perez&amp;author=KA%20Riske&amp;volume=202&amp;publication_year=2017&amp;pages=28-37&amp;pmid=27913102&amp;doi=10.1016/j.chemphyslip.2016.11.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref035">
<span class="label">35.</span><cite>Rusanov AL, Nakhod KV, Nakhod VI, Poverennaya EV, Petushkova NA, Luzgina NG. Changes in the Proteome of HaCaT Keratinocytes Induced by Cytotoxic Substance Triton X-100. Bull Exp Biol Med. 2017;163(5):620–2. doi: 10.1007/s10517-017-3863-2

</cite> [<a href="https://doi.org/10.1007/s10517-017-3863-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28952047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bull%20Exp%20Biol%20Med&amp;title=Changes%20in%20the%20Proteome%20of%20HaCaT%20Keratinocytes%20Induced%20by%20Cytotoxic%20Substance%20Triton%20X-100&amp;author=AL%20Rusanov&amp;author=KV%20Nakhod&amp;author=VI%20Nakhod&amp;author=EV%20Poverennaya&amp;author=NA%20Petushkova&amp;volume=163&amp;issue=5&amp;publication_year=2017&amp;pages=620-2&amp;pmid=28952047&amp;doi=10.1007/s10517-017-3863-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref036">
<span class="label">36.</span><cite>Ip K, Song G, Banov D, Bassani AS, Liu Y, Song H. Evaluation of the in vitro human skin percutaneous absorption of ketoprofen in topical anhydrous and aqueous gels. Skin Res Technol. 2024;30(3):e13589.</cite> [<a href="https://doi.org/10.1111/srt.13589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10891364/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38396354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Skin%20Res%20Technol&amp;title=Evaluation%20of%20the%20in%20vitro%20human%20skin%20percutaneous%20absorption%20of%20ketoprofen%20in%20topical%20anhydrous%20and%20aqueous%20gels&amp;author=K%20Ip&amp;author=G%20Song&amp;author=D%20Banov&amp;author=AS%20Bassani&amp;author=Y%20Liu&amp;volume=30&amp;issue=3&amp;publication_year=2024&amp;pmid=38396354&amp;doi=10.1111/srt.13589&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref037">
<span class="label">37.</span><cite>Chen Y-H, Wu C-P, Lin H-L, Liaw R-B, Lai Y-Y, Wu M-C, et al. Tetrazolium Salt WST-8 as a Novel and Reliable Chromogenic Indicator for the Assessment of Boar Semen Quality. Animals (Basel). 2020;10(12):2293. doi: 10.3390/ani10122293

</cite> [<a href="https://doi.org/10.3390/ani10122293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7761879/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33291566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Animals%20(Basel)&amp;title=Tetrazolium%20Salt%20WST-8%20as%20a%20Novel%20and%20Reliable%20Chromogenic%20Indicator%20for%20the%20Assessment%20of%20Boar%20Semen%20Quality&amp;author=Y-H%20Chen&amp;author=C-P%20Wu&amp;author=H-L%20Lin&amp;author=R-B%20Liaw&amp;author=Y-Y%20Lai&amp;volume=10&amp;issue=12&amp;publication_year=2020&amp;pages=2293&amp;pmid=33291566&amp;doi=10.3390/ani10122293&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref038">
<span class="label">38.</span><cite>Lee J, Lee P, Wu X. Molecular and cytoskeletal regulations in epidermal development. Semin Cell Dev Biol. 2017;69:18–25. doi: 10.1016/j.semcdb.2017.05.018

</cite> [<a href="https://doi.org/10.1016/j.semcdb.2017.05.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5583032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28577925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Semin%20Cell%20Dev%20Biol&amp;title=Molecular%20and%20cytoskeletal%20regulations%20in%20epidermal%20development&amp;author=J%20Lee&amp;author=P%20Lee&amp;author=X%20Wu&amp;volume=69&amp;publication_year=2017&amp;pages=18-25&amp;pmid=28577925&amp;doi=10.1016/j.semcdb.2017.05.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref039">
<span class="label">39.</span><cite>Lefèvre-Utile A, Braun C, Haftek M, Aubin F. Five Functional Aspects of the Epidermal Barrier. Int J Mol Sci. 2021;22(21):11676. doi: 10.3390/ijms222111676

</cite> [<a href="https://doi.org/10.3390/ijms222111676" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8583944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34769105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Five%20Functional%20Aspects%20of%20the%20Epidermal%20Barrier&amp;author=A%20Lef%C3%A8vre-Utile&amp;author=C%20Braun&amp;author=M%20Haftek&amp;author=F%20Aubin&amp;volume=22&amp;issue=21&amp;publication_year=2021&amp;pages=11676&amp;pmid=34769105&amp;doi=10.3390/ijms222111676&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref040">
<span class="label">40.</span><cite>Pfisterer K, Shaw LE, Symmank D, Weninger W. The Extracellular Matrix in Skin Inflammation and Infection. Front Cell Dev Biol. 2021;9:682414. doi: 10.3389/fcell.2021.682414

</cite> [<a href="https://doi.org/10.3389/fcell.2021.682414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8290172/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34295891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Dev%20Biol&amp;title=The%20Extracellular%20Matrix%20in%20Skin%20Inflammation%20and%20Infection&amp;author=K%20Pfisterer&amp;author=LE%20Shaw&amp;author=D%20Symmank&amp;author=W%20Weninger&amp;volume=9&amp;publication_year=2021&amp;pages=682414&amp;pmid=34295891&amp;doi=10.3389/fcell.2021.682414&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref041">
<span class="label">41.</span><cite>Bush KA, Pins GD. Carbodiimide conjugation of fibronectin on collagen basal lamina analogs enhances cellular binding domains and epithelialization. Tissue Eng Part A. 2010;16(3):829–38. doi: 10.1089/ten.TEA.2009.0514

</cite> [<a href="https://doi.org/10.1089/ten.TEA.2009.0514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2862607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19778179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Tissue%20Eng%20Part%20A&amp;title=Carbodiimide%20conjugation%20of%20fibronectin%20on%20collagen%20basal%20lamina%20analogs%20enhances%20cellular%20binding%20domains%20and%20epithelialization&amp;author=KA%20Bush&amp;author=GD%20Pins&amp;volume=16&amp;issue=3&amp;publication_year=2010&amp;pages=829-38&amp;pmid=19778179&amp;doi=10.1089/ten.TEA.2009.0514&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref042">
<span class="label">42.</span><cite>Koivumäki JT, Naumenko N, Tuomainen T, Takalo J, Oksanen M, Puttonen KA. Structural immaturity of human iPSC-derived cardiomyocytes: in silico investigation of effects on function and disease modeling. Frontiers in Physiology. 2018;9:80. doi: 10.3389/fphys.2018.00080

</cite> [<a href="https://doi.org/10.3389/fphys.2018.00080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5808345/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29467678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Physiology&amp;title=Structural%20immaturity%20of%20human%20iPSC-derived%20cardiomyocytes:%20in%20silico%20investigation%20of%20effects%20on%20function%20and%20disease%20modeling&amp;author=JT%20Koivum%C3%A4ki&amp;author=N%20Naumenko&amp;author=T%20Tuomainen&amp;author=J%20Takalo&amp;author=M%20Oksanen&amp;volume=9&amp;publication_year=2018&amp;pages=80&amp;pmid=29467678&amp;doi=10.3389/fphys.2018.00080&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref043">
<span class="label">43.</span><cite>Tao Y, Zhang S-C. Neural Subtype Specification from Human Pluripotent Stem Cells. Cell Stem Cell. 2016;19(5):573–86. doi: 10.1016/j.stem.2016.10.015

</cite> [<a href="https://doi.org/10.1016/j.stem.2016.10.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5127287/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27814479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Stem%20Cell&amp;title=Neural%20Subtype%20Specification%20from%20Human%20Pluripotent%20Stem%20Cells&amp;author=Y%20Tao&amp;author=S-C%20Zhang&amp;volume=19&amp;issue=5&amp;publication_year=2016&amp;pages=573-86&amp;pmid=27814479&amp;doi=10.1016/j.stem.2016.10.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref044">
<span class="label">44.</span><cite>Sandoval AGW, Gim KY, Huang JT, Koehler KR. Applications of Human Pluripotent Stem Cell-Derived Skin Organoids in Dermatology. J Invest Dermatol. 2023;143(10):1872–6. doi: 10.1016/j.jid.2023.07.017

</cite> [<a href="https://doi.org/10.1016/j.jid.2023.07.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10518840/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37739763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Invest%20Dermatol&amp;title=Applications%20of%20Human%20Pluripotent%20Stem%20Cell-Derived%20Skin%20Organoids%20in%20Dermatology&amp;author=AGW%20Sandoval&amp;author=KY%20Gim&amp;author=JT%20Huang&amp;author=KR%20Koehler&amp;volume=143&amp;issue=10&amp;publication_year=2023&amp;pages=1872-6&amp;pmid=37739763&amp;doi=10.1016/j.jid.2023.07.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330306.ref045">
<span class="label">45.</span><cite>Oceguera-Yanez F, Avila-Robinson A, Woltjen K. Differentiation of pluripotent stem cells for modeling human skin development and potential applications. Front Cell Dev Biol. 2022;10:1030339. doi: 10.3389/fcell.2022.1030339

</cite> [<a href="https://doi.org/10.3389/fcell.2022.1030339" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9728031/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36506084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Dev%20Biol&amp;title=Differentiation%20of%20pluripotent%20stem%20cells%20for%20modeling%20human%20skin%20development%20and%20potential%20applications&amp;author=F%20Oceguera-Yanez&amp;author=A%20Avila-Robinson&amp;author=K%20Woltjen&amp;volume=10&amp;publication_year=2022&amp;pages=1030339&amp;pmid=36506084&amp;doi=10.3389/fcell.2022.1030339&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section></section><article class="sub-article" id="pone.0330306.r001"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330306.r001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330306.r001</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 0</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ho%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Paulo Ho</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Paulo Ho</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ho%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Paulo Ho</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.c" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.c" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.c" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Paulo Ho</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.c" class="d-panel p" style="display: none">
<div>© 2025 Paulo Ho</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>25 Mar 2025</em>
</p>
<p>--&gt;PONE-D-25-01505--&gt;--&gt;Skin Irritation Testing using Human iPSCs derived 3D Skin Equivalent Model--&gt;--&gt;PLOS ONE</p>
<p>Dear Dr. Chung,</p>
<p>Thank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE’s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.</p>
<p>1) I am not confident that we have a good quality of 3D skin. As pointed out by one of the reviewers, "There is no stratum granulosum and no stratum corneum formed. Differences in stratum basale and spinosum in H&amp;E staining are only minor";--&gt;--&gt;2) Please, see the comments raised by both the reviewers.</p>
<p>Please submit your revised manuscript by May 09 2025 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at <span>plosone@plos.org</span> . When you're ready to submit your revision, log on to <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/pone/</a> and select the 'Submissions Needing Revision' folder to locate your manuscript file.</p>
<p>Please include the following items when submitting your revised manuscript:--&gt;</p>
<ul class="list" style="list-style-type:disc">
<li><p>A rebuttal letter that responds to each point raised by the academic editor and reviewer(s). You should upload this letter as a separate file labeled 'Response to Reviewers'.</p></li>
<li><p>A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled 'Revised Manuscript with Track Changes'.</p></li>
<li><p>An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled 'Manuscript'.</p></li>
</ul>
<p>If you would like to make changes to your financial disclosure, please include your updated statement in your cover letter. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter.</p>
<p>If applicable, we recommend that you deposit your laboratory protocols in protocols.io to enhance the reproducibility of your results. Protocols.io assigns your protocol its own identifier (DOI) so that it can be cited independently in the future. For instructions see: <a href="https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols</a> . Additionally, PLOS ONE offers an option for publishing peer-reviewed Lab Protocol articles, which describe protocols hosted on protocols.io. Read more information on sharing protocols at <a href="https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols</a> .</p>
<p>We look forward to receiving your revised manuscript.</p>
<p>Kind regards,</p>
<p>Paulo Lee Ho, Ph.D.</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>Journal Requirements:</p>
<p>When submitting your revision, we need you to address these additional requirements.</p>
<p>1. Please ensure that your manuscript meets PLOS ONE's style requirements, including those for file naming. The PLOS ONE style templates can be found at </p>
<p><a href="https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf</a> and </p>
<p>
<a href="https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf</a>
</p>
<p>[Note: HTML markup is below. Please do not edit.]</p>
<p>Reviewers' comments:</p>
<p>Reviewer's Responses to Questions</p>
<p>--&gt;<strong>Comments to the Author</strong></p>
<p>1. Is the manuscript technically sound, and do the data support the conclusions?</p>
<p>The manuscript must describe a technically sound piece of scientific research with data that supports the conclusions. Experiments must have been conducted rigorously, with appropriate controls, replication, and sample sizes. The conclusions must be drawn appropriately based on the data presented. --&gt;</p>
<p>Reviewer #1: Partly</p>
<p>Reviewer #2: Partly</p>
<p>**********</p>
<p>--&gt;2. Has the statistical analysis been performed appropriately and rigorously? --&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: N/A</p>
<p>**********</p>
<p>--&gt;3. Have the authors made all data underlying the findings in their manuscript fully available?</p>
<p>The <a href="http://www.plosone.org/static/policies.action#sharing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a> requires authors to make all data underlying the findings described in their manuscript fully available without restriction, with rare exception (please refer to the Data Availability Statement in the manuscript PDF file). The data should be provided as part of the manuscript or its supporting information, or deposited to a public repository. For example, in addition to summary statistics, the data points behind means, medians and variance measures should be available. If there are restrictions on publicly sharing data—e.g. participant privacy or use of data from a third party—those must be specified.--&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>--&gt;4. Is the manuscript presented in an intelligible fashion and written in standard English?</p>
<p>PLOS ONE does not copyedit accepted manuscripts, so the language in submitted articles must be clear, correct, and unambiguous. Any typographical or grammatical errors should be corrected at revision, so please note any specific errors here.--&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: No</p>
<p>**********</p>
<p>--&gt;5. Review Comments to the Author</p>
<p>Please use the space provided to explain your answers to the questions above. You may also include additional comments for the author, including concerns about dual publication, research ethics, or publication ethics. (Please upload your review as an attachment if it exceeds 20,000 characters)--&gt;</p>
<p>Reviewer #1: All comments are being submitted in a separate file in order to respect the limit of characters.</p>
<p>Please find the attached document for answer the required questions. I don't have any concern about dual publication or research ethics.</p>
<p>Reviewer #2: In the manuscript by Shin and colleagues human induced pluripotent stem cells were produced and were differentiated into fibroblasts and keratinocytes. Those cells were then used to establish 3D skin equivalents. Establishing high trough put in vitro skin models is important to meet the high demand of new models needed to replace and reduce animal experiments, especially with regard to chemical testing.</p>
<p>The manuscript is nicely written follows a clear path presenting results, although there are some major draw backs</p>
<p>1. In General, the writing is not clear, especially in the methods section. The English should be edited by a native speaker.</p>
<p>2. Methods: Generation of hi-PSC should be explained in more detail.</p>
<p>3. Formation of 3D SKE: Although there is a good separation into dermis and epidermis, the epidermis is not differentiated properly. There is no stratum granulosum and no stratum corneum formed. Differences in stratum basale and spinosum in H&amp;E staining are only minor. Skin models should be improved so that sublayers of the epidermis form properly as seen in 3D SKE made from primary skin cells. Have different timings been tested? 30 days are quite long compared to other published 3D skin models.</p>
<p>4. Further, proper epidermal differentiation should not only be assessed by H&amp;H staining but also IHC, staining for markers specific for certain sublayers, e.g. Fillagrin, Involucrin, Ki-67 or different keratins.</p>
<p>**********</p>
<p>--&gt;6. PLOS authors have the option to publish the peer review history of their article (<a href="https://journals.plos.org/plosone/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a> ). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a> .--&gt;</p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: No</p>
<p>**********</p>
<p>[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link "View Attachments". If this link does not appear, there are no attachment files.]</p>
<p>While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool, <a href="https://pacev2.apexcovantage.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pacev2.apexcovantage.com/</a> . PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at <span>figures@plos.org</span> . Please note that Supporting Information files do not need this step.</p>
<section class="sm xbox font-sm" id="pone.0330306.s002"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Comments.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360594/bin/pone.0330306.s002.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330306.s002.docx</a><sup> (17.1KB, docx) </sup>
</div></div></section></section></article><article class="sub-article" id="pone.0330306.r002"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. 2025 Aug 18;20(8):e0330306. doi: <a href="https://doi.org/10.1371/journal.pone.0330306.r002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330306.r002</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response to Decision Letter 1</h1></hgroup><ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="anp_a.d" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.d" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="anp_a.d" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub2" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a.d" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>26 Jun 2025</em>
</p>
<p>PONE-D-25-01505</p>
<p>Skin Irritation Testing using Human iPSCs derived 3D Skin Equivalent Model</p>
<p>PLOS ONE</p>
<p>Dear Dr. Chung,</p>
<p>Thank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE’s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.</p>
<p>1) I am not confident that we have a good quality of 3D skin. As pointed out by one of the reviewers, "There is no stratum granulosum and no stratum corneum formed. Differences in stratum basale and spinosum in H&amp;E staining are only minor";</p>
<p>2) Please, see the comments raised by both the reviewers.</p>
<p>Please submit your revised manuscript by May 09 2025 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at plosone@plos.org. When you're ready to submit your revision, log on to <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/pone/</a> and select the 'Submissions Needing Revision' folder to locate your manuscript file.</p>
<p>Please include the following items when submitting your revised manuscript:</p>
<p>• A rebuttal letter that responds to each point raised by the academic editor and reviewer(s). You should upload this letter as a separate file labeled 'Response to Reviewers'.</p>
<p>• A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled 'Revised Manuscript with Track Changes'.</p>
<p>• An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled 'Manuscript'.</p>
<p>If you would like to make changes to your financial disclosure, please include your updated statement in your cover letter. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter.</p>
<p>If applicable, we recommend that you deposit your laboratory protocols in protocols.io to enhance the reproducibility of your results. Protocols.io assigns your protocol its own identifier (DOI) so that it can be cited independently in the future. For instructions see: <a href="https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols</a>. Additionally, PLOS ONE offers an option for publishing peer-reviewed Lab Protocol articles, which describe protocols hosted on protocols.io. Read more information on sharing protocols at <a href="https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols</a>.</p>
<p>We look forward to receiving your revised manuscript.</p>
<p>Kind regards,</p>
<p>Paulo Lee Ho, Ph.D.</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>Response to Editor:</p>
<p>We sincerely thank the Editor’s genuine interest in our study and are grateful for permitting this revision of our manuscript. We did our best to experiment and consider in response to reviewers' comments. We hope that this revision coupled with our responses, are able to sufficiently address the critiques so our information and experience may be shared with the readership of PLOS ONE. </p>
<p>Correction part</p>
<p>&lt;Manuscript&gt;</p>
<p>1) Title page</p>
<p>1. Excluding author ‘Se Eun Kim’ and ‘Yun Hyeong Lee’.</p>
<p>2) Abstract</p>
<p>1. Research on artificial skin has advanced significantly, supporting cosmetic ingredient evaluation and skin regeneration treatment development. Artificial skin-based tests are well established, with standardized guidelines and widespread use of three-dimensional (3D) skin equivalent models (SKE) constructed from human primary skin cells. However, in the case of primary cells, there are problems with supply due to limited patients and difficulties in genetic-specific research. To address these issues, recent efforts have focused on differentiating skin cells from human-induced pluripotent stem cells (hiPSCs). � Artificial skin models have emerged as valuable tools for evaluating cosmetic ingredients and developing treatments for skin regeneration. Among them, 3D skin equivalent models (SKEs) using human primary skin cells are widely utilized and supported by standardized testing guidelines. However, primary cells face limitations such as restricted donor availability and challenges in conducting genotype-specific studies. To overcome these issues, recent approaches have focused on differentiating skin cells from human-induced pluripotent stem cells (hiPSCs).</p>
<p>2. The resulting hiPSC-SKE model replicated the primary layers of native skin morphologically, as demonstrated by hematoxylin and eosin staining, and expressed specific markers characteristic of each layer. Furthermore, treatment of hiPSC-SKEs with Triton X-100, a positive control for skin irritation testing, caused severe epidermal cell damage and reduced cell viability. These findings suggest that hiPSC-SKE serves as a promising alternative for skin-related applications, including animal testing replacement and genetic skin disease modeling. � Histological analysis with hematoxylin and eosin staining confirmed that the hiPSC-SKE recapitulated the layered architecture of native human skin and expressed appropriate epidermal and dermal markers. Moreover, exposure to Triton X-100, a known skin irritant, led to marked epidermal damage and significantly reduced cell viability, validating the model’s functional responsiveness. These findings indicate that the hiPSC-SKE model represents a promising alternative for various skin-related applications, including the replacement of animal testing.</p>
<p>3) Introduction</p>
<p>1. The skin is the primary tissue that exists on the body's exterior and performs various physiological functions in response to external environmental exposure (1, 2). � The skin, as the body's outermost tissue, serves as a barrier against environmental stimuli and performs essential physiological functions (1, 2).</p>
<p>2. Numerous studies have focused on creating human 3D skin equivalents (SKEs) using primary skin cells to replicate human skin in vitro (3-5). One advantage of SKEs is their relatively simple production through insert-based methods (6). In addition, when cultured keratinocytes on the surface in contact with the air forms epidermal-like layer, it forms an environment and structural complexity similar to human skin. � To replicate skin in vitro, 3D skin equivalent models (SKEs) have been developed using primary skin cells (3-5). SKEs can be fabricated relatively easily using insert-based methods (6). When keratinocytes are cultured on a surface exposed to air, they form the epidermis-like layer, creating an environment and structural complexity similar to that of human skin.</p>
<p>3. As a result, SKE has been used in various experiments related to skin drug toxicity evaluation, disease modeling, and transplantation therapeutics as wound models (5). � Due to these features, SKEs are widely used in studies related to drug toxicity, disease modeling, and wound healing applications (5).</p>
<p>4. Among the various applications of SKEs, skin irritation and corrosion testing are prominent (7-9). International guidelines for skin irritation testing are well-established, with the Organization for Economic Cooperation and Development test guideline 439 (OECD TG 439) serving as a key reference (7-9). This testing is critical for verifying the safety of products such as cosmetics, pharmaceuticals, and chemicals by evaluating whether a substance causes physical or chemical irritation or damage to the skin. And OECD TG 439 proposes a method that uses an artificial skin model to replace traditional animal testing. � SKEs are widely used for skin irritation and corrosion testing, guided by international standards like the Organization for Economic Cooperation and Development test guideline 439 (OECD TG 439). This testing is crucial for evaluating the safety of cosmetics, pharmaceuticals, and chemicals, and OECD TG 439 promotes artificial skin models as a replacement for animal testing (7-9).</p>
<p>5. Artificial skin used according to these guidelines typically includes Reconstructed Human Epidermis (such as RhE; EpiskinTM, EpiDermTM, SkinEthicTM RHE) and Dermo-Epidermal SKEs created from commercial primary skin cells (7-9). In addition to safety evaluations, keratinocytes and fibroblasts derived from human tissues are commonly used in research on skin diseases and therapeutic agents (6, 12-15). While primary cells reflect in vivo maturity, challenges such as limited supply, high cost, and senescence from passaging remain. Additionally, simulating genetic disease models is difficult due to limitations in manufacturing patient-specific products, and immune rejection complicates therapeutic agent transplantation. To address these issues, human induced pluripotent stem cells (hiPSCs), known for their self-renewal and pluripotency, have emerged as a promising source (16, 17). The advantages of hiPSC-derived cells include: 1) overcoming cell supply constraints through self-renewal, 2) disease modeling using patient-derived genetic samples or gene editing, and 3) superior therapeutic effects through greater engraftment with low immunogenicity (16, 17). Some studies have successfully produced 3D SKEs from hiPSC-derived skin cells (18-21). However, reports on the production and use of hiPSC-derived SKEs (hiPSC-SKEs) are still limited, and creating a comprehensive database of studies is essential to refine these models. In this study, we established a protocol for producing skin cells from hiPSCs and successfully generated hiPSC-SKEs. Morphological analysis confirmed that hiPSC-SKEs replicate human skin characteristics, and skin irritation tests using Triton X-100 were successfully conducted. These databases are expected to play a significant role in fields such as medicine, cosmetics, and drug development, as hiPSC-SKEs evolve into more advanced skin models in the future. � Artificial skin models used according to established guidelines typically include reconstructed human epidermis (RhE) products such as Episkin™, EpiDerm™, and SkinEthic™ RHE, as well as dermo-epidermal SKEs constructed from commercially available primary skin cells  (7-9). In addition to safety evaluations, skin cells are commonly used in research on skin diseases and therapeutic agents (6, 12-15). While primary cells reflect in vivo maturity, their use is limited by supply shortages, high costs, and senescence during passaging. Additionally, modeling genetic skin diseases is challenging due to difficulties in producing patient-specific products, and immune rejection complicates therapeutic transplantation. To address these issues, human induced pluripotent stem cells (hiPSCs) have emerged as a promising source (16, 17). Advantages of hiPSC-derived cells include: (1) unlimited supply through self-renewal, (2) disease modeling via patient-specific genetics or gene editing, and (3) improved therapeutic potential with better engraftment and low immunogenicity (16, 17). Several studies have successfully generated 3D SKEs from hiPSC-derived skin cells (18-21). However, reports on hiPSC-derived SKEs (hiPSC-SKEs) remain limited, and a comprehensive study database is needed to optimize these models. In this study, we established a protocol to differentiate skin cells from hiPSCs and generated hiPSC-SKEs. Morphological analyses confirmed that hiPSC-SKEs replicate human skin features, and skin irritation tests using Triton X-100 were successfully performed. These datasets are expected to contribute significantly to cosmetics, chemicals, and drug development as hiPSC-SKEs advance as skin models.</p>
<p>4) Materials &amp; Methods</p>
<p>1. Added sentence ‘(Generation of human induced pluripotent stem cells (hiPSCs)) 1.5 × 10⁶ human dermal fibroblasts (BJ cells; ATCC, VA, USA) were transfected with episomal vectors (pCLXE-hOCT4/p53, pCLXE-hSOX2/KLF4, and pCLXE-hL-MYC/LIN28A; Addgene, MA, USA) using the P2 Primary Cell 4D-Nucleofector Kit (Lonza, Basel, Switzerland) according to the manufacturer’s protocol (22, 23). The mixture of cells and vectors was transferred into a Nucleocuvette Vessel and subjected to electroporation. Transfected cells were plated onto 1:100 Matrigel (Corning, NY, USA)-coated 6-well plates and cultured for 2 days in Dulbecco’s Modified Eagle’s Medium (DMEM; Gibco, NY, USA) supplemented with 10% fetal bovine serum (FBS; Sigma-Aldrich, St. Louis, MO, USA), 0.1 mM non-essential amino acids (NEAA; Gibco), and 1% penicillin/streptomycin (P/S; Gibco). After 2 days, the medium was replaced with TeSR-E7 medium (STEMCELL Technologies, Vancouver, Canada), which was changed daily. Cells were cultured for an additional 10–14 days until PSC-like colonies appeared. After picking the PSC-like colonies, they were seeded onto Matrigel-coated 4-well plates in mTeSR1 (STEMCELL Technologies Inc; Vancouver, BC, Canada) medium supplemented with 10 µM Y-27632 (Tocris Bioscience, Bristol, UK). The medium was changed daily with fresh mTeSR1 during subsequent culture.’.</p>
<p>2. hiPSCs were produced using a pluripotency-related reprogramming episomal vectors (addgene, MA, USA) in human dermal fibroblasts (ATCC, VA, USA) (22, 23). hiPSCs were seeded on 60 mm cell culture dish coated with 1:100 Matrigel (Corning Inc.; Corning, NY, USA) and cultured by replacing the mTeSR1 (STEMCELL Technologies Inc; Vancouver, BC, Canada) medium every day. After 3 days, when 90-100% confluency was achieved, hiPSCs were detached using DPBS (welgene, Gyeongsan, South Korea) containing 0.5 mM EDTA (Gibco, NY, USA). Detached hiPSCs were collected in 6 ml mTeSR1 containing 10 µM Y-27632 (Tocris Bioscience, Bristol, UK) and placed in a 60 mm petri dish coated with F127 (Sigma-Aldrich, St. Louis, USA). Afterwards, they were cultured for one day in a shaker set at 65-70 rpm in a 37 °C, 5% CO2 incubator. The next day, embryonic bodies (EBs) formation was confirmed and mTeSR1 was replaced every day for 2 days (D3). 150-200 EBs were seeded on Matrigel-coated 60 mm cell culture dishes using fibroblast differentiation media 1 (FDM1) and replaced for 3 days. FDM1 is a 3:1 mixture of DMEM (Gibco, NY, USA) and F12 (Gibco) supplemented with 5% fetal bovine serum (FBS; Sigma-Aldrich), 5 µg/mL Insulin (Sigma-Aldrich) and 10 ng/ml epithelial growth factor (EGF; PeproTech, NJ, USA). After 3 days, FDM2 was replaced and cultured for 3 days. FDM2 is a medium supplemented with 25 ng/ml bone morphogenic protein 4 (BMP4; Peprotech) to FDM1. Next, it was cultured for 7 days after replacing with FDM3. FDM3 is a 1:1 mixture of DMEM and F12 supplemented with 1% non-essential amino acids solution (NEAA; Gibco). After 7 days, it was replaced with FDM1 medium again and cultured for 4 days. When it reached 70-80% confluency, it was detached using a 5:1 mixture of TrypLE (Gibco) and 0.25% Trypsin/EDTA (T/E; Gibco) (D20). The detached cells were seeded on matrigel-coated cell culture dishes with FDM1 medium and maintained in culture until D30 (Fig. 1A). � hiPSCs were seeded onto 1:100 Matrigel-coated 60 mm cell culture dishes and maintained in mTeSR1 medium with daily medium changes. Upon reaching 90–100% confluency after 3 days, the cells were gently dissociated using DPBS (Welgene, Gyeongsan, South Korea) containing 0.5 mM EDTA (Gibco). The detached hiPSCs were resuspended in 6 ml mTeSR1 supplemented with 10 µM Y-27632 and transferred to F127-coated (Sigma-Aldrich) 60 mm Petri dishes to initiate embryoid body (EB) formation (D0). Cells were cultured on a shaker at 65–70 rpm in a 37 °C incubator with 5% CO₂ for 24 hr. On the following day (D1), EB formation was confirmed, and mTeSR1 was replaced daily for two additional days (until D3). Approximately 150–200 EBs were then transferred onto Matrigel-coated 60 mm culture dishes and cultured in fibroblast differentiation medium 1 (FDM1) for 3 days (D3-D6). FDM1 consisted of a 3:1 mixture of DMEM (Gibco) and F12 (Gibco), supplemented with 5% FBS, 5 µg/ml insulin (Sigma-Aldrich), and 10 ng/ml epidermal growth factor (EGF; PeproTech, NJ, USA). After 3 days, the medium was changed to fibroblast differentiation medium 2 (FDM2), which is FDM1 supplemented with 25 ng/ml bone morphogenetic protein 4 (BMP4; PeproTech), and cultured for another 3 days (D6-D9). Subsequently, the cells were cultured in fibroblast differentiation medium 3 (FDM3) for 7 days (D9-D16). FDM3 consisted of a 1:1 mixture of DMEM and F12 supplemented with 5% FBS</p>
<section class="sm xbox font-sm" id="pone.0330306.s003"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response to reviewer_PLOS ONE.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360594/bin/pone.0330306.s003.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330306.s003.docx</a><sup> (4.9MB, docx) </sup>
</div></div></section></section></article><article class="sub-article" id="pone.0330306.r003"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330306.r003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330306.r003</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 1</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ho%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Paulo Ho</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Paulo Ho</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ho%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Paulo Ho</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.e" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.e" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.e" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Paulo Ho</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.e" class="d-panel p" style="display: none">
<div>© 2025 Paulo Ho</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>30 Jul 2025</em>
</p>
<p>Skin Irritation Testing using Human iPSCs derived 3D Skin Equivalent Model</p>
<p>PONE-D-25-01505R1</p>
<p>Dear Dr. Chung,</p>
<p>We’re pleased to inform you that your manuscript has been judged scientifically suitable for publication and will be formally accepted for publication once it meets all outstanding technical requirements.</p>
<p>Within one week, you’ll receive an e-mail detailing the required amendments. When these have been addressed, you’ll receive a formal acceptance letter and your manuscript will be scheduled for publication.</p>
<p>An invoice will be generated when your article is formally accepted. Please note, if your institution has a publishing partnership with PLOS and your article meets the relevant criteria, all or part of your publication costs will be covered. Please make sure your user information is up-to-date by logging into Editorial Manager at <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Editorial Manager®</a>  and clicking the ‘Update My Information' link at the top of the page. For questions related to billing, please contact <a href="https://plos.my.site.com/s/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">billing support</a> .</p>
<p>If your institution or institutions have a press office, please notify them about your upcoming paper to help maximize its impact. If they’ll be preparing press materials, please inform our press team as soon as possible -- no later than 48 hours after receiving the formal acceptance. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact onepress@plos.org.</p>
<p>Kind regards,</p>
<p>Paulo Lee Ho, Ph.D.</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>Additional Editor Comments (optional):</p>
<p>Reviewers' comments:</p>
<p>Reviewer's Responses to Questions</p>
<p>--&gt;<strong>Comments to the Author</strong></p>
<p>1. If the authors have adequately addressed your comments raised in a previous round of review and you feel that this manuscript is now acceptable for publication, you may indicate that here to bypass the “Comments to the Author” section, enter your conflict of interest statement in the “Confidential to Editor” section, and submit your "Accept" recommendation.--&gt;</p>
<p>Reviewer #1: All comments have been addressed</p>
<p>Reviewer #2: All comments have been addressed</p>
<p>**********</p>
<p>--&gt;2. Is the manuscript technically sound, and do the data support the conclusions?</p>
<p>The manuscript must describe a technically sound piece of scientific research with data that supports the conclusions. Experiments must have been conducted rigorously, with appropriate controls, replication, and sample sizes. The conclusions must be drawn appropriately based on the data presented. --&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>--&gt;3. Has the statistical analysis been performed appropriately and rigorously? --&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>--&gt;4. Have the authors made all data underlying the findings in their manuscript fully available?</p>
<p>The <a href="http://www.plosone.org/static/policies.action#sharing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a> requires authors to make all data underlying the findings described in their manuscript fully available without restriction, with rare exception (please refer to the Data Availability Statement in the manuscript PDF file). The data should be provided as part of the manuscript or its supporting information, or deposited to a public repository. For example, in addition to summary statistics, the data points behind means, medians and variance measures should be available. If there are restrictions on publicly sharing data—e.g. participant privacy or use of data from a third party—those must be specified.--&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>--&gt;5. Is the manuscript presented in an intelligible fashion and written in standard English?</p>
<p>PLOS ONE does not copyedit accepted manuscripts, so the language in submitted articles must be clear, correct, and unambiguous. Any typographical or grammatical errors should be corrected at revision, so please note any specific errors here.--&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>--&gt;6. Review Comments to the Author</p>
<p>Please use the space provided to explain your answers to the questions above. You may also include additional comments for the author, including concerns about dual publication, research ethics, or publication ethics. (Please upload your review as an attachment if it exceeds 20,000 characters)--&gt;</p>
<p>Reviewer #1: The focus on methodological detail, quantitative rigor, and manuscript clarity is appropriate and consistent with standards for publication.</p>
<p>1. General response to reviewer</p>
<p>Could be slightly more specific about where/how improvements were made to increase the credibility of the response.</p>
<p>2. Cell purity and characterization</p>
<p>Provides new data and clearly explains the methods used to address the concern. Supplementary figure addition is appropriate.</p>
<p>3. 3D model construction and media rationale</p>
<p>They explained both the rationale and optimization efforts. The inclusion of comparisons to past protocols strengthens the response.</p>
<p>4. Triton X-100 rationale</p>
<p>Clear explanation grounded in OECD guidelines. Appropriately referenced standard practice.</p>
<p>5. Statistical analysis</p>
<p>They addressed the concern well by explicitly adding the use of Tukey’s test. Including the method directly in the manuscript improvede transparency.</p>
<p>6. Mechanism of Triton X-100</p>
<p>They contextualized the choice and clarified the scope of the study.</p>
<p>7. Methods section clarity</p>
<p>They directly addressed with added detail. This strengthens reproducibility.</p>
<p>8. hiPSC generation protocol</p>
<p>Fully addressed with specifics on episomal vectors, timing, and selection methods.</p>
<p>9. Differentiation protocols</p>
<p>Providing both text and schematic (Figure 1) was an effective way to present complex information.</p>
<p>10. hiPSC-SKE construction</p>
<p>Well-clarified with all relevant parameters now included (collagen type, seeding density, medium details).</p>
<p>11. Immunofluorescence</p>
<p>Adequately addressed with catalog numbers, companies, and DAPI info.</p>
<p>12. Results section improvements</p>
<p>Strong improvement by adding quantification (cell marker % and thickness). Reinforces the conclusions with numbers.</p>
<p>13. Discussion section: comparisons and limitations</p>
<p>They’ve acknowledged model limitations, compared to commercial standards, and outlined future directions clearly.</p>
<p>Overall comment</p>
<p>The revision is highly suitable and demonstrates a robust scientific response. The authors addressed their concerns with specific data and methodological transparency, and significantly improved the manuscript clarity, rigor, and completeness. The integration of supplemental figures, quantitative analysis, and expanded methodological descriptions shows strong scientific maturity.</p>
<p>Minor improvements could still be made by weaving more of the changes directly into the responses (e.g., quoting or referencing the exact location in the revised manuscript), but overall, the quality is good.</p>
<p>Reviewer #2: Although epidermal differentiation is still not perfekt, those skins are a valuable tool for toxicity testing.</p>
<p>**********</p>
<p>--&gt;7. PLOS authors have the option to publish the peer review history of their article (<a href="https://journals.plos.org/plosone/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a> ). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a> .--&gt;</p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: No</p>
<p>**********</p></section></article><article class="sub-article" id="pone.0330306.r004"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330306.r004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330306.r004</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Acceptance letter</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ho%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Paulo Ho</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Paulo Ho</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ho%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Paulo Ho</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.f" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.f" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.f" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Paulo Ho</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.f" class="d-panel p" style="display: none">
<div>© 2025 Paulo Ho</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>PONE-D-25-01505R1</p>
<p>PLOS ONE</p>
<p>Dear Dr. Chung,</p>
<p>I'm pleased to inform you that your manuscript has been deemed suitable for publication in PLOS ONE. Congratulations! Your manuscript is now being handed over to our production team.</p>
<p>At this stage, our production department will prepare your paper for publication. This includes ensuring the following:</p>
<p>* All references, tables, and figures are properly cited</p>
<p>* All relevant supporting information is included in the manuscript submission,</p>
<p>* There are no issues that prevent the paper from being properly typeset</p>
<p>You will receive further instructions from the production team, including instructions on how to review your proof when it is ready. Please keep in mind that we are working through a large volume of accepted articles, so please give us a few days to review your paper and let you know the next and final steps.</p>
<p>Lastly, if your institution or institutions have a press office, please let them know about your upcoming paper now to help maximize its impact. If they'll be preparing press materials, please inform our press team within the next 48 hours. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact onepress@plos.org.</p>
<p>You will receive an invoice from PLOS for your publication fee after your manuscript has reached the completed accept phase. If you receive an email requesting payment before acceptance or for any other service, this may be a phishing scheme. Learn how to identify phishing emails and protect your accounts at <a href="https://explore.plos.org/phishing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://explore.plos.org/phishing</a>.</p>
<p>If we can help with anything else, please email us at customercare@plos.org.</p>
<p>Thank you for submitting your work to PLOS ONE and supporting open access.</p>
<p>Kind regards,</p>
<p>PLOS ONE Editorial Office Staff</p>
<p>on behalf of</p>
<p>Dr. Paulo Lee Ho</p>
<p>Academic Editor</p>
<p>PLOS ONE</p></section></article><article class="sub-article" id="_ad93_"><section class="pmc-layout__citation font-secondary font-xs"><div></div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Associated Data</h1></hgroup><ul class="d-buttons inline-list"></ul>
<div class="d-panels font-secondary-light"></div>
<div></div>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
</div></section><section class="body sub-article-body"><section id="_adsm93_" lang="en" class="supplementary-materials"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 File</span><p><strong>S1 Table. Infromation of primers.</strong> Genes: GAPDH (housekeeping gene for control), puripotency marekrs (OCT4 and SOX2), fibroblast-specific markers (PDGFRα, Col3A1, and Fibronectin), keratinocyte-specific markers (KRT14 and ΔNp63). <strong>S1 Fig. IF staining for purity check in hFIBRO and hKERA.</strong> (A) Staining for vimentin, a fibroblast-specific marker, in hFIBRO and distribution of vimentin+ cells per fields (n = 3). (B) Staining for ΔNp63, a keratinocyte-specific marker, in hKERA and distribution of ΔNp63 + cells per fields (n = 3). White arrows: DAPI where each marker is not expressed. Scale bar: 100 µm. <strong>S2 Fig. Electrophoresis analysis for pluripotency and skin cell-specific markers by hiPSC, hFIBRO, and hKERA.</strong> Genes: GAPDH (housekeeping gene for control), puripotency marekrs (OCT4 and SOX2), fibroblast-specific markers (PDGFRα, Col3A1, and Fibronectin), keratinocyte-specific markers (KRT14 and ΔNp63).</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360594/bin/pone.0330306.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330306.s001.docx</a><sup> (826.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Comments.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360594/bin/pone.0330306.s002.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330306.s002.docx</a><sup> (17.1KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response to reviewer_PLOS ONE.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360594/bin/pone.0330306.s003.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330306.s003.docx</a><sup> (4.9MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>All relevant data are within the paper and its <a href="#sec019" class="usa-link">Supporting Information</a> files.</p></section></section></article><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS One are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pone.0330306"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pone.0330306.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12360594/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12360594/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12360594%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360594/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12360594/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12360594/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40825023/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12360594/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40825023/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12360594/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12360594/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="wWojhUhZLix0i96X6v5xXId8w9FcgHLfApIsxzlcUm7MzjplPiv3HhZGKaFR9CqS">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
