
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genome‐wide association analysis of winter survival in a diverse Canadian winter wheat population - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15251A4A8AEED0B3051A4A001E56475D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365865/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Genome‐wide association analysis of winter survival in a diverse Canadian winter wheat population">
<meta name="citation_author" content="Rachel Whiting">
<meta name="citation_author_institution" content="Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada">
<meta name="citation_author" content="Alexandra Ficht">
<meta name="citation_author_institution" content="Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada">
<meta name="citation_author" content="Yi Chen">
<meta name="citation_author_institution" content="Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada">
<meta name="citation_author" content="Davoud Torkamaneh">
<meta name="citation_author_institution" content="Faculté des sciences de l&#x27;agriculture et de l&#x27;alimentation, Université Laval, Pavillon Paul‐Comtois, Québec City, Québec, Canada">
<meta name="citation_author" content="Joseph Colasanti">
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, University of Guelph, Guelph, Ontario, Canada">
<meta name="citation_author" content="Eric M Lyons">
<meta name="citation_author_institution" content="Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70091">
<meta name="citation_doi" content="10.1002/tpg2.70091">
<meta name="citation_pmid" content="40832815">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365865/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365865/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365865/pdf/TPG2-18-e70091.pdf">
<meta name="description" content="Vrn‐A1 (VERNALIZATION A1) and Fr‐A2 (FROST RESISTANCE A2) have been associated with variation in winter survival of wheat (Triticum aestivum L.). The beneficial alleles of Vrn‐A1 and Fr‐A2 are largely fixed in Canadian winter wheat germplasm, ...">
<meta name="og:title" content="Genome‐wide association analysis of winter survival in a diverse Canadian winter wheat population">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Vrn‐A1 (VERNALIZATION A1) and Fr‐A2 (FROST RESISTANCE A2) have been associated with variation in winter survival of wheat (Triticum aestivum L.). The beneficial alleles of Vrn‐A1 and Fr‐A2 are largely fixed in Canadian winter wheat germplasm, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365865/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365865">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70091"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70091.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365865%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365865/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365865/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365865/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70091" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Aug 20;18(3):e70091. doi: <a href="https://doi.org/10.1002/tpg2.70091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70091</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genome‐wide association analysis of winter survival in a diverse Canadian winter wheat population</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Whiting%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Rachel Whiting</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Rachel Whiting</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Whiting%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rachel Whiting</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ficht%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Alexandra Ficht</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Alexandra Ficht</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ficht%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexandra Ficht</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yi Chen</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yi Chen</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yi Chen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Torkamaneh%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Davoud Torkamaneh</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Davoud Torkamaneh</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Faculté des sciences de l'agriculture et de l'alimentation, Université Laval, Pavillon Paul‐Comtois, Québec City, Québec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Torkamaneh%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Davoud Torkamaneh</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Colasanti%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Joseph Colasanti</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Joseph Colasanti</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Molecular and Cellular Biology, University of Guelph, Guelph, Ontario, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Colasanti%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Joseph Colasanti</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lyons%20EM%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Eric M Lyons</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Eric M Lyons</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lyons%20EM%22%5BAuthor%5D" class="usa-link"><span class="name western">Eric M Lyons</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270091-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada</div>
<div id="tpg270091-aff-0002">
<sup>
<sup>2</sup>
</sup>Faculté des sciences de l'agriculture et de l'alimentation, Université Laval, Pavillon Paul‐Comtois, Québec City, Québec, Canada</div>
<div id="tpg270091-aff-0003">
<sup>
<sup>3</sup>
</sup>Department of Molecular and Cellular Biology, University of Guelph, Guelph, Ontario, Canada</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Eric M. Lyons, Department of Plant Agriculture, University of Guelph, 50 Stone Road East, Guelph, Ontario, N1G 2W1, Canada. Email: <span>elyons@uoguelph.ca</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 27; Received 2025 Mar 5; Accepted 2025 Jul 2; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365865  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40832815/" class="usa-link">40832815</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>
<em>Vrn‐A1</em> (<em>VERNALIZATION A1</em>) and <em>Fr‐A2</em> (<em>FROST RESISTANCE A2</em>) have been associated with variation in winter survival of wheat (<em>Triticum aestivum</em> L.). The beneficial alleles of <em>Vrn‐A1</em> and <em>Fr‐A2</em> are largely fixed in Canadian winter wheat germplasm, rendering the associated molecular markers ineffective for marker‐assisted selection (MAS) in elite populations. The objectives were to (i) identify quantitative trait loci (QTLs) for winter survival in eastern Canada and determine their usefulness for MAS and (ii) explore the underlying genetic mechanisms of superior winter survival in the region. A subpopulation (<em>n</em> = 321) of the Canadian Winter Wheat Diversity Panel, consisting of genotypes that were fixed for the beneficial alleles of <em>vrn‐A1</em> and <em>Fr‐A2</em>, was previously evaluated for winter survival in three eastern Canadian environments (Elora 2016–2017, CÉROM 2017–2018, and Elora 2017–2018). Genome‐wide association mapping identified three significant QTLs for winter survival, a previously identified QTL on chromosome 5A, and two novel QTLs on chromosomes 5D and 7B. These QTLs were of low‐to‐moderate marker utility (0.1473–0.4796) and conferred a 0.7%–1.8% increase in mean winter survival. In silico analyses revealed that an array of biotic and abiotic stress responses are implicated in winter survival in eastern Canada, which challenges the notion that lethal temperature is the primary cause of winterkill in some regions. As significant winterkill events are sporadic in the region, it may be beneficial to identify individual components of winter survival that can be examined in artificial environments.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270091-list-10001" class="list" style="list-style-type:disc">
<li><p>Significant quantitative trait loci (QTLs) were found in the 3T (winter tolerant) subpopulation.</p></li>
<li><p>Applicability for marker‐assisted selection is limited in elite population.</p></li>
<li><p>Different environments initiated unique physiological responses, which present a challenge when breeding for abiotic stressors.</p></li>
</ul></section><section class="abstract" id="abstract3"><h2>Plain Language Summary</h2>
<p>The ability for winter wheat to withstand the variable conditions of winter is key to their survival. This study sought to (i) identify genetic markers for winter survival in eastern Canadian wheat and (ii) determine if they had regional significance. A large panel of diverse winter wheat genotypes was evaluated across three environments from 2016 to 2018. A genome‐wide association study identified three significant markers that had the potential to be used in winter wheat breeding; however, there was little overlap across environments. As significant winterkill events are sporadic, future studies should utilize individual components of winter survival that can be examined in artificial environments.</p></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270091-dl-0001">
<dt>CWWDP</dt>
<dd><p>Canadian Winter Wheat Diversity Panel</p></dd>
<dt>GBS
</dt>
<dd><p>genotyping‐by‐sequencing</p></dd>
<dt>GS</dt>
<dd><p>genomic selection</p></dd>
<dt>GWASs</dt>
<dd><p>genome‐wide association studies</p></dd>
<dt>LD</dt>
<dd><p>linkage disequilibrium</p></dd>
<dt>MAS</dt>
<dd><p>marker‐assisted selection</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus</p></dd>
<dt>ROS</dt>
<dd><p>reactive oxygen species</p></dd>
<dt>SNP</dt>
<dd><p>single‐nucleotide polymorphism</p></dd>
</dl>
<section id="tpg270091-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Plant breeders require an understanding of the genetic elements that underlie a trait of interest to inform breeding strategies. For example, genomic selection (GS) is effective for quantitative traits, those controlled by numerous genes of small effect on the population mean, while traits that are controlled by few genes of large effect can be improved using marker‐assisted selection (MAS) (Pandurangan et al., <a href="#tpg270091-bib-0056" class="usa-link" aria-describedby="tpg270091-bib-0056">2021</a>; Watson et al., <a href="#tpg270091-bib-0092" class="usa-link" aria-describedby="tpg270091-bib-0092">2019</a>). Winter survival in eastern Canadian winter wheat (<em>Triticum aestivum</em> L.) is currently maintained through selection indices; however, the improvement of adaptive traits in an increasingly variable climate may require a more intensive approach (Crain et al., <a href="#tpg270091-bib-0016" class="usa-link" aria-describedby="tpg270091-bib-0016">2018</a>, <a href="#tpg270091-bib-0017" class="usa-link" aria-describedby="tpg270091-bib-0017">2017</a>; T. Zhang et al., <a href="#tpg270091-bib-0100" class="usa-link" aria-describedby="tpg270091-bib-0100">2022</a>). Numerous studies have noted the complex nature of winter survival in cereal species (S. Liu, Yang, et al., <a href="#tpg270091-bib-0043" class="usa-link" aria-describedby="tpg270091-bib-0043">2014</a>; Soleimani et al., <a href="#tpg270091-bib-0072" class="usa-link" aria-describedby="tpg270091-bib-0072">2022</a>; Zhao et al., <a href="#tpg270091-bib-0102" class="usa-link" aria-describedby="tpg270091-bib-0102">2013</a>), and there has been renewed interest in the improvement of winter survival using both MAS and GS (Beil et al., <a href="#tpg270091-bib-0004" class="usa-link" aria-describedby="tpg270091-bib-0004">2019</a>; Michel et al., <a href="#tpg270091-bib-0048" class="usa-link" aria-describedby="tpg270091-bib-0048">2019</a>).</p>
<p>In cereal species, winter survival is often equated with superior cold tolerance (Larsen et al., <a href="#tpg270091-bib-0041" class="usa-link" aria-describedby="tpg270091-bib-0041">2018</a>; Willick et al., <a href="#tpg270091-bib-0093" class="usa-link" aria-describedby="tpg270091-bib-0093">2020</a>). Two functional molecular markers for cold tolerance, <em>Fr‐1</em> (<em>FROST RESISTANCE 1</em>) and <em>Fr‐A2</em> (<em>FROST RESISTANCE A2</em>), are widely reported (Sutka &amp; Snape, <a href="#tpg270091-bib-0078" class="usa-link" aria-describedby="tpg270091-bib-0078">1989</a>; Vágújfalvi et al., <a href="#tpg270091-bib-0087" class="usa-link" aria-describedby="tpg270091-bib-0087">2003</a>). <em>Fr‐1</em> is the pleiotropic effect of <em>Vrn‐A1</em> (<em>VERNALIZATION A1</em>), a key regulatory gene of the floral induction pathway (Dhillon et al., <a href="#tpg270091-bib-0019" class="usa-link" aria-describedby="tpg270091-bib-0019">2010</a>). <em>Vrn‐A1</em> exists in different copy numbers, with a higher copy number of the recessive <em>vrn‐A1</em> allele associated with a longer vernalization requirement (Díaz et al., <a href="#tpg270091-bib-0020" class="usa-link" aria-describedby="tpg270091-bib-0020">2012</a>) and increased cold tolerance (Würschum et al., <a href="#tpg270091-bib-0094" class="usa-link" aria-describedby="tpg270091-bib-0094">2015</a>; Zhu et al., <a href="#tpg270091-bib-0104" class="usa-link" aria-describedby="tpg270091-bib-0104">2014</a>). <em>Fr‐A2</em> consists of a cluster of 11 <em>CBF</em> (<em>C‐repeat binding factor</em>) genes that are upregulated by low temperature (Miller et al., <a href="#tpg270091-bib-0049" class="usa-link" aria-describedby="tpg270091-bib-0049">2006</a>; Vágújfalvi et al., <a href="#tpg270091-bib-0086" class="usa-link" aria-describedby="tpg270091-bib-0086">2005</a>). A higher copy number of the core <em>CBF‐A14</em> gene (referred to as the tolerant “T” haplotype) is also associated with increased freezing tolerance (Sieber et al., <a href="#tpg270091-bib-0069" class="usa-link" aria-describedby="tpg270091-bib-0069">2016</a>; Zhu et al., <a href="#tpg270091-bib-0104" class="usa-link" aria-describedby="tpg270091-bib-0104">2014</a>). However, the use of molecular markers for the improvement of winter survival has been limited in Canadian wheat breeding programs (Randhawa et al., <a href="#tpg270091-bib-0061" class="usa-link" aria-describedby="tpg270091-bib-0061">2013</a>; Toth et al., <a href="#tpg270091-bib-0085" class="usa-link" aria-describedby="tpg270091-bib-0085">2019</a>).</p>
<p>Intense selective pressures from the environment often fix major beneficial alleles for adaptive traits in regional populations (Hoffmann &amp; Hercus, <a href="#tpg270091-bib-0031" class="usa-link" aria-describedby="tpg270091-bib-0031">2000</a>; Olson‐Manning et al., <a href="#tpg270091-bib-0053" class="usa-link" aria-describedby="tpg270091-bib-0053">2012</a>; Orr, <a href="#tpg270091-bib-0055" class="usa-link" aria-describedby="tpg270091-bib-0055">1998</a>), and there is a noted geographic distribution of beneficial alleles for these traits (Kreyling et al., <a href="#tpg270091-bib-0040" class="usa-link" aria-describedby="tpg270091-bib-0040">2019</a>). Functional molecular markers associated with these beneficial alleles are useful when moving exotic germplasm into breeding populations (Pandurangan et al., <a href="#tpg270091-bib-0056" class="usa-link" aria-describedby="tpg270091-bib-0056">2021</a>). However, these molecular markers are fixed for beneficial alleles (i.e., have low marker utility) and have little impact on the phenotypic mean in populations derived from elite, locally adapted cultivars (Bernardo, <a href="#tpg270091-bib-0006" class="usa-link" aria-describedby="tpg270091-bib-0006">2020</a>; Platten et al., <a href="#tpg270091-bib-0059" class="usa-link" aria-describedby="tpg270091-bib-0059">2019</a>). This is the case for <em>Vrn‐A1</em> and <em>Fr‐A2</em> in Canadian winter wheat germplasm. The Canadian Winter Wheat Diversity Panel (CWWDP; <em>n</em> = 450) is a diverse collection of Canadian wheat genotypes curated by the University of Guelph (Sidhu, <a href="#tpg270091-bib-0068" class="usa-link" aria-describedby="tpg270091-bib-0068">2020</a>). This panel comprises ancestral, historical, and modern genotypes, as well as germplasm sourced from various Canadian wheat breeding programs. A significant proportion (70.8%) of the CWWDP was fixed for three copies of the recessive <em>vrn‐A1</em> allele and the “T” haplotype of <em>Fr‐A2</em> (Figure <a href="#tpg270091-fig-0001" class="usa-link">1</a>; Chen, <a href="#tpg270091-bib-0013" class="usa-link" aria-describedby="tpg270091-bib-0013">2019</a>). However, genotypic variation (<em>p</em> &lt; 0.0001) was still observed in this subpopulation (hereafter referred to as “3T”) (Chen, <a href="#tpg270091-bib-0013" class="usa-link" aria-describedby="tpg270091-bib-0013">2019</a>). Other genomic regions underlying regional adaptation to eastern Canadian production regions may explain this variation and are of interest for functional molecular marker development.</p>
<figure class="fig xbox font-sm" id="tpg270091-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2c6c/12365865/c1c3ba76fac9/TPG2-18-e70091-g001.jpg" loading="lazy" id="jats-graphic-1" height="350" width="560" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270091-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of winter survival by <em>Vrn‐A1</em> copy number and <em>Fr‐A2</em> haplotype variation in the Canadian Winter Wheat Diversity Panel (CWWDP), which was assessed in three eastern Canadian environments: Elora 2016–2017, CÉROM 2017–2018, and Elora 2017–2018 and adapted from Chen (<a href="#tpg270091-bib-0013" class="usa-link" aria-describedby="tpg270091-bib-0013">2019</a>). S, susceptible; T, tolerant.</p></figcaption></figure><p>Genome‐wide association studies (GWASs) are commonly used to identify quantitative trait loci (QTLs) for phenotypic variation in plants (Tibbs Cortes et al., <a href="#tpg270091-bib-0080" class="usa-link" aria-describedby="tpg270091-bib-0080">2021</a>). Diverse populations are commonly used for GWASs to maximize linkage disequilibrium (LD) and to identify results that are broadly applicable. By evaluating a regional population, where phenotypic differences are largely restricted to variation in the founder germplasm, there is increased statistical power to detect QTL of local significance (Brachi et al., <a href="#tpg270091-bib-0007" class="usa-link" aria-describedby="tpg270091-bib-0007">2011</a>). Association mapping also provides the opportunity to explore causal genes underlying a trait of interest. At present, our understanding of cold tolerance mechanisms is based on fundamental research in model species (<em>Arabidopsis thaliana</em>, <em>Oryza sativa</em>, etc.) or specific wheat cultivars studied in controlled environments (Ding et al., <a href="#tpg270091-bib-0021" class="usa-link" aria-describedby="tpg270091-bib-0021">2019</a>; Guo et al., <a href="#tpg270091-bib-0025" class="usa-link" aria-describedby="tpg270091-bib-0025">2018</a>; Hassan et al., <a href="#tpg270091-bib-0029" class="usa-link" aria-describedby="tpg270091-bib-0029">2021</a>). Winter survival is largely equated to superior cold tolerance in western Canadian winter wheat (Larsen et al., <a href="#tpg270091-bib-0041" class="usa-link" aria-describedby="tpg270091-bib-0041">2018</a>; Willick et al., <a href="#tpg270091-bib-0093" class="usa-link" aria-describedby="tpg270091-bib-0093">2020</a>). The primary cause of winterkill in winter wheat grown in eastern Canada is not known, and research in other species suggests it is the cumulative effect of multiple stress tolerances, including extreme temperature, cold‐induced desiccation, frost heaving, low‐temperature pathogens, flooding and ice encasement, and thaw‐refreeze cycles (Bélanger et al., <a href="#tpg270091-bib-0005" class="usa-link" aria-describedby="tpg270091-bib-0005">2006</a>; He et al., <a href="#tpg270091-bib-0030" class="usa-link" aria-describedby="tpg270091-bib-0030">2019</a>).</p>
<p>Causal genes underlying winter survival in elite eastern Canadian winter wheat populations warrant further investigation and may result in a more targeted trait of interest for the University of Guelph Winter Wheat Breeding Program. Therefore, the objectives of this study were to (i) identify QTLs for winter survival in eastern Canada and determine their usefulness for MAS and (ii) explore the underlying genetic mechanisms of superior winter survival in the region.</p></section><section id="tpg270091-sec-0020"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270091-sec-0030"><h3 class="pmc_sec_title">2.1. Plant materials</h3>
<p>A notable subpopulation (<em>n</em> = 321) of the CWWDP harbored three copies of the recessive <em>vrn‐A1</em> allele and the tolerant “T” haplotype of <em>Fr‐A2</em>. This subpopulation (“3T”) had significantly higher winter survival when compared to other <em>vrn‐A1</em>/<em>Fr‐A2</em> combinations (Figure <a href="#tpg270091-fig-0001" class="usa-link">1</a>; Chen et al., <a href="#tpg270091-bib-0014" class="usa-link" aria-describedby="tpg270091-bib-0014">2024</a>). However, significant variation (<em>p</em> &lt; 0.0001; Table <a href="#tpg270091-tbl-0001" class="usa-link">1</a>) in winter survival was also observed among the “3T” genotypes and prompted further investigation. A complete list of “3T” genotypes can be found in Supporting Information <a href="#tpg270091-supinfo-0001" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="tpg270091-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Combined analysis of variance for winter survival in the “3T” subpopulation of the Canadian Winter Wheat Diversity Panel (CWWDP).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Effect</th>
<th align="left" rowspan="1" colspan="1">Num. <em>df</em>
</th>
<th align="left" rowspan="1" colspan="1">Den. <em>df</em>
</th>
<th align="left" rowspan="1" colspan="1">
<em>F</em>‐value</th>
<th align="left" rowspan="1" colspan="1">Pr &gt; <em>F</em>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Genotype</td>
<td align="left" rowspan="1" colspan="1">320</td>
<td align="left" rowspan="1" colspan="1">960</td>
<td align="center" rowspan="1" colspan="1">2.18</td>
<td align="center" rowspan="1" colspan="1">&lt;0.0001</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Environment</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="center" rowspan="1" colspan="1">7.38</td>
<td align="center" rowspan="1" colspan="1">0.0695</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Genotype × environment</td>
<td align="left" rowspan="1" colspan="1">640</td>
<td align="left" rowspan="1" colspan="1">960</td>
<td align="center" rowspan="1" colspan="1">1.45</td>
<td align="center" rowspan="1" colspan="1">&lt;0.0001</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Covariance parameter</th>
<th align="left" rowspan="1" colspan="1">Estimate</th>
<th align="left" rowspan="1" colspan="1">Standard error</th>
<th align="left" rowspan="1" colspan="1">
<em>Z</em> value</th>
<th align="left" rowspan="1" colspan="1">Pr &gt; <em>F</em>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Block (environment)</td>
<td align="center" rowspan="1" colspan="1">34.0502</td>
<td align="center" rowspan="1" colspan="1">28.1802</td>
<td align="center" rowspan="1" colspan="1">1.21</td>
<td align="center" rowspan="1" colspan="1">0.1135</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Residual</td>
<td align="center" rowspan="1" colspan="1">148.76</td>
<td align="center" rowspan="1" colspan="1">6.7899</td>
<td align="center" rowspan="1" colspan="1">21.91</td>
<td align="center" rowspan="1" colspan="1">&lt;0.0001</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270091-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270091-tbl1-note-0001"><p>
<em>Note</em>: The CWWDP population was assessed in three eastern Canadian environments: Elora 2016–2017, CÉROM 2017–2018, and Elora 2017–2018.</p></div></div></section><section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270091-sec-0040"><h4 class="pmc_sec_title">Core Ideas</h4>
<ul id="tpg270091-list-0001" class="list" style="list-style-type:disc">
<li><p>Significant quantitative trait loci (QTLs) were found in the 3T (winter tolerant) subpopulation.</p></li>
<li><p>Applicability for marker‐assisted selection is limited in elite population.</p></li>
<li><p>Different environments initiated unique physiological responses, which present a challenge when breeding for abiotic stressors.</p></li>
</ul></section></section></section><section id="tpg270091-sec-0050"><h3 class="pmc_sec_title">2.2. Evaluation of winter survival and statistical analyses</h3>
<p>Field survival data for the CWWDP was obtained from Chen (<a href="#tpg270091-bib-0013" class="usa-link" aria-describedby="tpg270091-bib-0013">2019</a>). Genotypes were grown in a two‐replicate alpha‐lattice design across three location‐year environments, at the Elora Research Station in Elora, ON, during the 2016–2017 and 2017–2018 seasons and at CÉROM in Saint‐Mathiew‐de‐Beloeil, QC, during the 2017–2018 season. Each plot comprised six rows with 7.8‐cm row spacing and varied in length, either 1 or 2 m. Winter survival was visually assessed and scored on a scale of 0 (no plants remaining) to 100 (all plants remaining) for each plot immediately following the spring thaw (Chen, <a href="#tpg270091-bib-0013" class="usa-link" aria-describedby="tpg270091-bib-0013">2019</a>).</p>
<p>A combined analysis of variance (ANOVA) for winter survival was conducted using the PROC GLIMMIX procedure in SAS 9.4 (SAS Institute Inc.). The fixed effects were “Genotype,” “Environment,” and “Genotype × Environment” interaction, while the random effect was “Block(Environment).” ANOVAs were also performed for each individual environment, with “Genotype” as a fixed effect and “Block” as a random effect. Incomplete blocks were not significant and were removed from the models. Least‐square mean (LSMEAN) values were generated for each genotype in the individual and combined environments. Broad‐sense heritability (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-1" display="inline"><mrow><mrow><msup><mi>H</mi><mn>2</mn></msup><mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span> was calculated for individual and combined environments using the following equation:
</p>
<table class="disp-formula p" id="tpg270091-disp-0001"><tr>
<td class="formula"><math id="jats-math-2" display="block"><mrow><mrow><msup><mi>H</mi><mn>2</mn></msup><mo linebreak="badbreak">=</mo><mfrac><msub><mi>v</mi><mi>g</mi></msub><msub><mi>v</mi><mi>p</mi></msub></mfrac><mo>,</mo></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline"><mrow><msub><mi>v</mi><mi>g</mi></msub></mrow></math></span> is the genetic variance and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><msub><mi>v</mi><mi>p</mi></msub></mrow></math></span> is the total phenotypic variance.</p></section><section id="tpg270091-sec-0060"><h3 class="pmc_sec_title">2.3. Genotyping</h3>
<p>Genotyping was performed by Chen (<a href="#tpg270091-bib-0013" class="usa-link" aria-describedby="tpg270091-bib-0013">2019</a>) and used a genotyping‐by‐sequencing (GBS) approach following the methodology outlined by Poland et al. (<a href="#tpg270091-bib-0060" class="usa-link" aria-describedby="tpg270091-bib-0060">2012</a>). DNA extraction was carried out using the DNeasy Plant Mini Kit (Qiagen) in accordance with the manufacturer's protocol. Single‐end sequencing of multiplex GBS libraries was performed on two chips (per 96‐plex GBS library) of Ion Torrent (Proton; Thermo‐Fisher Scientific) at Plateforme d'analyses génomiques, Institut de Biologie Intégrative et des Systèmes, Université Laval (Quebec City, QC, Canada).</p>
<p>Processing of next‐generation sequence reads was performed using the Fast‐GBS pipeline (Torkamaneh et al., <a href="#tpg270091-bib-0084" class="usa-link" aria-describedby="tpg270091-bib-0084">2017</a>). Demultiplexing of FASTQ files based on barcode sequences was followed by read trimming and mapping to the IWGSC Chinese Spring v1.0 wheat reference genome (IWGSC, <a href="#tpg270091-bib-0036" class="usa-link" aria-describedby="tpg270091-bib-0036">2018</a>). Identification of nucleotide variants from the mapped reads was carried out, and variants were filtered to exclude those with two or more alternate alleles, QUAL score &lt; 32, MQ score &lt; 30, and read depth &lt; 2.</p>
<p>A genotypic data file for the “3T” subpopulation with 35,634 single‐nucleotide polymorphisms (SNPs) was extracted using TASSEL 5.0 (Bradbury et al., <a href="#tpg270091-bib-0008" class="usa-link" aria-describedby="tpg270091-bib-0008">2007</a>). Missing data imputation was performed using Beagle v5 (Browning &amp; Browning, <a href="#tpg270091-bib-0009" class="usa-link" aria-describedby="tpg270091-bib-0009">2007</a>) within VCFtools v.0.1.16 (Danecek et al., <a href="#tpg270091-bib-0018" class="usa-link" aria-describedby="tpg270091-bib-0018">2011</a>), following the methods described in Torkamaneh and Belzile (<a href="#tpg270091-bib-0082" class="usa-link" aria-describedby="tpg270091-bib-0082">2015</a>) and Torkamaneh et al. (<a href="#tpg270091-bib-0083" class="usa-link" aria-describedby="tpg270091-bib-0083">2018</a>). The SNP dataset was further filtered based on a minor allele frequency ≥0.05 and maximum missing data of 80%. Heterozygous SNPs were treated as missing data. The resulting genotypic data file had 20,645 SNPs.</p></section><section id="tpg270091-sec-0070"><h3 class="pmc_sec_title">2.4. GWAS analyses</h3>
<p>GWAS analyses were performed for individual and combined environments using the Genome Association and Prediction Integrated Tool (GAPIT3; Wang &amp; Zhang, <a href="#tpg270091-bib-0089" class="usa-link" aria-describedby="tpg270091-bib-0089">2021</a>) in RStudio (v. 2023.3.0.277; Posit Software). To account for population structure and relatedness, both kinship and principal component analysis (PCA) were integrated into the analysis, with <em>P</em> + <em>K</em>* identified as the optimal model fit. The significance threshold was determined by Bonferroni correction using the negative log(1/<em>n</em>) method (Noble, <a href="#tpg270091-bib-0052" class="usa-link" aria-describedby="tpg270091-bib-0052">2009</a>; M. Wang et al., <a href="#tpg270091-bib-0090" class="usa-link" aria-describedby="tpg270091-bib-0090">2012</a>). For the identification of QTL related to winter survival, two statistical methods were utilized: FarmCPU (X. Liu et al., <a href="#tpg270091-bib-0045" class="usa-link" aria-describedby="tpg270091-bib-0045">2016</a>) and BLINK (Huang et al., <a href="#tpg270091-bib-0034" class="usa-link" aria-describedby="tpg270091-bib-0034">2018</a>). The mean LD decay for the entire genome was determined through PopLDDecay (C. Zhang et al., <a href="#tpg270091-bib-0099" class="usa-link" aria-describedby="tpg270091-bib-0099">2019</a>).</p></section><section id="tpg270091-sec-0080"><h3 class="pmc_sec_title">2.5. Metrics of marker suitability</h3>
<p>To evaluate the suitability of a QTL for MAS, two key metrics were employed. The first metric, marker utility (<em>U</em>), was measured as the proportion of genotypes within the population that lacked the beneficial allele associated with the QTL (Platten et al., <a href="#tpg270091-bib-0059" class="usa-link" aria-describedby="tpg270091-bib-0059">2019</a>):
</p>
<table class="disp-formula p" id="tpg270091-disp-0002"><tr>
<td class="formula"><math id="jats-math-5" display="block"><mrow><mrow><mi>Utility</mi><mspace width="0.28em"></mspace><mfenced open="(" close=")"><mi>U</mi></mfenced><mo linebreak="badbreak">=</mo><mfrac><mfenced separators="" open="(" close=")"><mrow><mi>Number</mi><mspace width="0.28em"></mspace><mi>of</mi><mspace width="0.28em"></mspace><mi>genotypes</mi><mspace width="0.28em"></mspace><mi>without</mi><mspace width="0.28em"></mspace><mi>beneficial</mi><mspace width="0.28em"></mspace><mi>allele</mi></mrow></mfenced><mfenced separators="" open="(" close=")"><mrow><mi>Total</mi><mspace width="0.28em"></mspace><mi>genotypes</mi><mspace width="0.28em"></mspace><mi>assessed</mi></mrow></mfenced></mfrac></mrow></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>The second metric was the predicted change in mean value for winter survival, which was determined by subtracting the population mean from the LSMEAN values for each. The LSMEAN values were obtained using the PROC GLIMMIX procedure in SAS 9.4, with “Genotype” filtered by allele.</p></section><section id="tpg270091-sec-0090"><h3 class="pmc_sec_title">2.6. In silico analyses</h3>
<p>An in silico search for candidate genes was conducted using the Wheat IWGSC RefSeq v1.0 reference genome. WheatMine (<a href="https://urgi.versailles.inra.fr/WheatMine/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://urgi.versailles.inra.fr/WheatMine/</a>) served as the primary tool for identifying candidate genes within the 254 kb region surrounding each significant QTL, as determined by LD decay. To identify candidate genes with known molecular functions, Protein Basic Local Alignment Search Tool was used to match <em>Triticum aestivum</em> L. hypothetical proteins against those with established functions in the NCBI database (<a href="https://www.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/</a>). Furthermore, orthologues of the candidate genes were identified in related species, including <em>Arabidopsis thaliana</em> and <em>Oryza sativa</em>, through the UniProtKB database (<a href="http://uniprot.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://uniprot.org/</a>).</p></section></section><section id="tpg270091-sec-0100"><h2 class="pmc_sec_title">3. RESULTS AND DISCUSSION</h2>
<section id="tpg270091-sec-0110"><h3 class="pmc_sec_title">3.1. Phenotypic evaluation</h3>
<p>The CWWDP was assessed for winter survival in a three‐location multi‐environment trial during the 2016–2017 and 2017–2018 seasons (Chen, <a href="#tpg270091-bib-0013" class="usa-link" aria-describedby="tpg270091-bib-0013">2019</a>). Population mean, minimum, and maximum values for winter survival in the “3T” subpopulation are presented in Table <a href="#tpg270091-tbl-0002" class="usa-link">2</a>. CÉROM 2017–2018 exhibited the lowest mean winter survival (58.9 ± 7.0%, standard error) and the highest variation (CV = 29.54%). Moderate winter conditions were observed at Elora in 2016–2017 (77.9 ± 1.0%; CV = 10.64) and 2017–2018 (78.9 ± 1.5%; CV = 9.97; Figure <a href="#tpg270091-supinfo-0001" class="usa-link">S1</a>). Combined winter survival data displayed negative skewness with a mean survival of 71.9 ± 2.5% (Figure <a href="#tpg270091-supinfo-0001" class="usa-link">S2</a>; Table <a href="#tpg270091-tbl-0001" class="usa-link">1</a>). “Genotype” and “Genotype × Environment” were significant (<em>p</em> &lt; 0.0001) in the combined dataset, while “Environment” was not significant (<em>p</em> = 0.0695; Table <a href="#tpg270091-tbl-0001" class="usa-link">1</a>). “Genotype” was also significant (<em>p</em> ranged from &lt; 0.0001 to 0.0107) in each individual environment (Tables <a href="#tpg270091-supinfo-0001" class="usa-link">S1–S3</a>). Broad‐sense heritability (<em>H</em>
<sup>2</sup>) was calculated for each environment and ranged from 0.2273 (CÉROM 2017–2018) to 0.6584 (Elora 2017–2018). <em>H</em>
<sup>2</sup> was 0.3341 in the combined dataset (Table <a href="#tpg270091-tbl-0002" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="tpg270091-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Descriptive statistics for winter survival in the “3T” subpopulation (<em>n</em> = 321) of the Canadian Winter Wheat Diversity Panel (CWWDP), including minimum, maximum, and mean winter survival, coefficients of variation (CV) and broad‐sense heritability (<em>H</em>
<sup>2</sup>).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Environment</th>
<th align="left" rowspan="1" colspan="1">Min.</th>
<th align="left" rowspan="1" colspan="1">Max.</th>
<th align="left" rowspan="1" colspan="1">Mean (standard error)<a href="#tpg270091-tbl2-note-0002" class="usa-link">
<sup>a</sup>
</a>
</th>
<th align="left" rowspan="1" colspan="1">CV (%)</th>
<th align="left" rowspan="1" colspan="1">
<em>H</em>
<sup>2</sup>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Elora 2016–2017</td>
<td align="center" rowspan="1" colspan="1">17.5</td>
<td align="center" rowspan="1" colspan="1">97.5</td>
<td align="center" rowspan="1" colspan="1">77.9 (1.0)</td>
<td align="center" rowspan="1" colspan="1">10.64</td>
<td align="center" rowspan="1" colspan="1">0.6044</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CÉROM 2017–2018</td>
<td align="center" rowspan="1" colspan="1">12.5</td>
<td align="center" rowspan="1" colspan="1">82.5</td>
<td align="center" rowspan="1" colspan="1">58.9 (7.0)</td>
<td align="center" rowspan="1" colspan="1">29.54</td>
<td align="center" rowspan="1" colspan="1">0.2273</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Elora 2017–2018</td>
<td align="center" rowspan="1" colspan="1">25.0</td>
<td align="center" rowspan="1" colspan="1">95.0</td>
<td align="center" rowspan="1" colspan="1">78.9 (1.5)</td>
<td align="center" rowspan="1" colspan="1">9.97</td>
<td align="center" rowspan="1" colspan="1">0.6584</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Combined</td>
<td align="center" rowspan="1" colspan="1">40.8</td>
<td align="center" rowspan="1" colspan="1">86.7</td>
<td align="center" rowspan="1" colspan="1">71.9 (0.4)</td>
<td align="center" rowspan="1" colspan="1">16.97</td>
<td align="center" rowspan="1" colspan="1">0.3341</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270091-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270091-tbl2-note-0001"><p>
<em>Note</em>: The CWWDP population was assessed in three eastern Canadian environments: Elora 2016–2017, CÉROM 2017–2018 and Elora 2017–2018.</p></div>
<div class="fn" id="tpg270091-tbl2-note-0002">
<sup>
<sup>a</sup>
</sup><p class="display-inline">
<em>α</em> = 0.05.</p>
</div>
</div></section></section><section id="tpg270091-sec-0120"><h3 class="pmc_sec_title">3.2. GWAS and marker suitability</h3>
<p>Note that 20,645 high‐quality SNPs were used for genome‐wide association mapping. PCA did not reveal genetically distinct subsets in the “3T” subpopulation (Figure <a href="#tpg270091-supinfo-0001" class="usa-link">S3</a>). One QTL associated with winter survival was identified on chromosomes 5A, 5D, and 7B, respectively, for the combined environments (Figure <a href="#tpg270091-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270091-tbl-0003" class="usa-link">3</a>). QTLs were also identified in individual environments, Elora 2016–2017 (three QTLs) and CÉROM 2017–2018 (two QTLs); however, none were identified in Elora 2017–2018 (Figure <a href="#tpg270091-supinfo-0001" class="usa-link">S4</a>; Tables <a href="#tpg270091-supinfo-0001" class="usa-link">S4</a> and <a href="#tpg270091-supinfo-0001" class="usa-link">S5</a>).</p>
<figure class="fig xbox font-sm" id="tpg270091-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2c6c/12365865/750eb6cf05d8/TPG2-18-e70091-g003.jpg" loading="lazy" id="jats-graphic-3" height="319" width="800" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270091-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Manhattan and <em>Q</em>–<em>Q</em> plots for winter survival in the “3T” subpopulation of the Canadian Winter Wheat Diversity Panel. Combined least square mean values were generated from Elora 2016–2017, CÉROM 2017–2018, and Elora 2017–2018 winter survival data. The FarmCPU and BLINK methods in the Genome Association and Prediction Integrated Tool (GAPIT3) were used to identify quantitative trait loci, with a significance threshold of −Log10(1/<em>n</em>) = 5.62.</p></figcaption></figure><section class="tw xbox font-sm" id="tpg270091-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Significant quantitative trait loci for winter survival in the “3T” subpopulation of the Canadian Winter Wheat Diversity Panel (CWWDP).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">QTL<a href="#tpg270091-tbl3-note-0002" class="usa-link">
<sup>a</sup>
</a>
</th>
<th align="left" rowspan="1" colspan="1">SNP name</th>
<th align="left" rowspan="1" colspan="1">Method</th>
<th align="left" rowspan="1" colspan="1">Chr.</th>
<th align="left" rowspan="1" colspan="1">Position</th>
<th align="left" rowspan="1" colspan="1">
<em>p</em>‐value</th>
<th align="left" rowspan="1" colspan="1">Allele frequency</th>
<th align="left" rowspan="1" colspan="1">Effect (SE)<a href="#tpg270091-tbl3-note-0003" class="usa-link">
<sup>b</sup>
</a>
</th>
<th align="left" rowspan="1" colspan="1">
<em>R</em>
<sup>2</sup>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>
<em>QWs.ugw‐5A.5</em>
</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>
<em>BobWhite_c5540_416</em>
</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>FarmCPU</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>5A</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>519905799</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>3.45E‐08</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>85% C: 15% T</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<p>
<strong>C = 72.6% (0.4)</strong>
</p>
<p>
<strong>T = 67.6% (1.0)</strong>
</p>
</td>
<td align="center" rowspan="1" colspan="1">
<strong>12.41%</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>QWs.ugw‐5D.1</em>
</td>
<td align="left" rowspan="1" colspan="1">
<em>wsnp_CAP11_c209_198467</em>
</td>
<td align="left" rowspan="1" colspan="1">FarmCPU</td>
<td align="left" rowspan="1" colspan="1">5D</td>
<td align="left" rowspan="1" colspan="1">547273607</td>
<td align="left" rowspan="1" colspan="1">7.09E‐07</td>
<td align="left" rowspan="1" colspan="1">54% C: 46% T</td>
<td align="left" rowspan="1" colspan="1">
<p>C = 73.4% (0.5)</p>
<p>T = 70.2% (0.6)</p>
</td>
<td align="center" rowspan="1" colspan="1">5.40%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>QWs.ugw‐7B.2</em>
</td>
<td align="left" rowspan="1" colspan="1">
<em>Kukri_c12581_163</em>
</td>
<td align="left" rowspan="1" colspan="1">BLINK</td>
<td align="left" rowspan="1" colspan="1">7B</td>
<td align="left" rowspan="1" colspan="1">732152999</td>
<td align="left" rowspan="1" colspan="1">4.23E‐07</td>
<td align="left" rowspan="1" colspan="1">52% T: 48% C</td>
<td align="left" rowspan="1" colspan="1">
<p>T = 73.7% (0.6)</p>
<p>C = 70.0% (0.6)</p>
</td>
<td align="center" rowspan="1" colspan="1">16.31%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270091-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270091-tbl3-note-0001"><p>
<em>Note</em>: Combined least square mean values were generated from Elora 2016–2017, CÉROM 2017–2018, and Elora 2017–2018 winter survival data. The FarmCPU and BLINK methods in the Genome Association and Prediction Integrated Tool (GAPIT3) were used to identify quantitative trait loci, with a significance threshold of −Log10(1/<em>n</em>) = 5.62. <em>QWs.ugw.5A.5</em> was also identified in CÉROM 2017–2018 with FarmCPU (in bold).</p></div>
<div class="fn" id="tpg270091-tbl3-note-0002">
<sup>
<sup>a</sup>
</sup><p class="display-inline">QTL were named in continuity with previous studies utilizing the Canadian Winter Wheat Diversity Panel (Chen, <a href="#tpg270091-bib-0013" class="usa-link" aria-describedby="tpg270091-bib-0013">2019</a>; Chen et al., <a href="#tpg270091-bib-0015" class="usa-link" aria-describedby="tpg270091-bib-0015">2019</a>).</p>
</div>
<div class="fn" id="tpg270091-tbl3-note-0003">
<sup>
<sup>b</sup>
</sup><p class="display-inline">Mean winter survival by allele.</p>
</div>
</div></section><p>
<em>QWs.ugw‐7B.2</em> (<em>Kukri_c12581_163</em>) had the strongest effect on winter survival and explained 16.31% of observed phenotypic variation in the combined dataset (Table <a href="#tpg270091-tbl-0003" class="usa-link">3</a>). Marker utility is beneficial for selection in a given genetic background. In the case of winter survival, markers with higher utility may introduce new alleles into the gene pool, which could then play a role in increasing winter survival. QWS.UGW‐7B.2 had moderate marker utility (<em>U = </em>0.4796), with the beneficial allele present in 52.0% of the population (Table <a href="#tpg270091-tbl-0004" class="usa-link">4</a>). <em>QWs.ugw‐5A.5</em> (<em>BobWhite_c5540_416</em>) also had a strong effect on winter survival (12.41% variance explained; Table <a href="#tpg270091-tbl-0003" class="usa-link">3</a>); however, the beneficial allele was largely fixed in the population (∼85.3%), resulting in a low marker utility score (<em>U = </em>0.1473; Table <a href="#tpg270091-tbl-0004" class="usa-link">4</a>). <em>QWs.ugw‐5D.1</em> (<em>wsnp_CAP11_c209_198467</em>) had the smallest impact on winter survival (5.40% variance explained; Table <a href="#tpg270091-tbl-0003" class="usa-link">3</a>) but had moderate marker utility (<em>U = </em>0.4639). The beneficial allele occurred in 53.6% of the population (Table <a href="#tpg270091-tbl-0004" class="usa-link">4</a>).</p>
<section class="tw xbox font-sm" id="tpg270091-tbl-0004"><h4 class="obj_head">TABLE 4.</h4>
<div class="caption p"><p>Suitability metrics of significant winter survival quantitative trait loci for functional marker development in the “3T” subpopulation of the Canadian Winter Wheat Diversity Panel (CWWDP).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">QTL<a href="#tpg270091-tbl4-note-0002" class="usa-link">
<sup>a</sup>
</a>
</th>
<th align="left" rowspan="1" colspan="1">SNP name</th>
<th align="left" rowspan="1" colspan="1">Utility score</th>
<th align="left" rowspan="1" colspan="1">Predicted change<a href="#tpg270091-tbl4-note-0003" class="usa-link">
<sup>b</sup>
</a>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>QWs.ugw‐5A.5</em>
</td>
<td align="left" rowspan="1" colspan="1">
<em>BobWhite_c5540_416</em>
</td>
<td align="center" rowspan="1" colspan="1">0.1473</td>
<td align="center" rowspan="1" colspan="1">0.7%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>QWs.ugw‐5D.1</em>
</td>
<td align="left" rowspan="1" colspan="1">
<em>wsnp_CAP11_c209_198467</em>
</td>
<td align="center" rowspan="1" colspan="1">0.4639</td>
<td align="center" rowspan="1" colspan="1">1.5%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>QWs.ugw‐7B.2</em>
</td>
<td align="left" rowspan="1" colspan="1">
<em>Kukri_c12581_163</em>
</td>
<td align="center" rowspan="1" colspan="1">0.4796</td>
<td align="center" rowspan="1" colspan="1">1.8%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270091-tbl-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270091-tbl4-note-0001"><p>
<em>Note</em>: Utility score defines the ability of a marker to introgress a beneficial allele into a target population, while predicted change defines the expected increase in mean winter survival.</p></div>
<div class="fn" id="tpg270091-tbl4-note-0002">
<sup>
<sup>a</sup>
</sup><p class="display-inline">QTL were named in continuity with previous studies utilizing the Canadian Winter Wheat Diversity Panel (Chen, <a href="#tpg270091-bib-0013" class="usa-link" aria-describedby="tpg270091-bib-0013">2019</a>; Chen et al., <a href="#tpg270091-bib-0015" class="usa-link" aria-describedby="tpg270091-bib-0015">2019</a>).</p>
</div>
<div class="fn" id="tpg270091-tbl4-note-0003">
<sup>
<sup>b</sup>
</sup><p class="display-inline">Change in mean winter survival.</p>
</div>
</div></section><p>Superior winter survival was conferred by a distinct genotypic combination (5A[C]/5D[C]/7B[T]; Figure <a href="#tpg270091-fig-0003" class="usa-link">3</a>); genotypes carrying the beneficial alleles of all three QTLs. This group had significantly greater winter survival (76.1 ± 0.9%) compared to the population mean (+4.2% above the population mean). Individually, the beneficial alleles of <em>QWs.ugw‐5A.5</em>, <em>QWs.ugw‐5D.1</em>, and <em>QWs.ugw‐7B.2</em> provided a 0.7%, 1.5%, and 1.8% advantage in winter survival (Table <a href="#tpg270091-tbl-0004" class="usa-link">4</a>).</p>
<figure class="fig xbox font-sm" id="tpg270091-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2c6c/12365865/0cbbfd03a9cc/TPG2-18-e70091-g002.jpg" loading="lazy" id="jats-graphic-5" height="312" width="560" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270091-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Winter survival in the “3T” subpopulation of the Canadian Winter Wheat Diversity Panel, grouped by allelic combination (“C” or “T”) at <em>QWs.ugw‐5A.5</em>, <em>QWs.ugw‐5D.1</em>, and <em>QWs.ugw‐7B.2</em>. Boxes indicate the median (black line), 25th percentile and 75th percentile of winter survival data, while dots represent outliers. The number of lines in each group (<em>n</em>) are also displayed.</p></figcaption></figure></section><section id="tpg270091-sec-0130"><h3 class="pmc_sec_title">3.3. Putative candidate genes</h3>
<p>Nineteen putative candidate genes were identified in the flanking regions (±254 kb; Figure <a href="#tpg270091-supinfo-0001" class="usa-link">S5</a>) of <em>QWs.ugw‐5A.5</em>, <em>QWs.ugw‐5D.1</em>, and <em>QWs.ugw‐7B.2</em> (Table <a href="#tpg270091-tbl-0005" class="usa-link">5</a>), and 12 had previous associations with stress response (Table <a href="#tpg270091-tbl-0005" class="usa-link">5</a>; in bold). A MYC2‐like transcription factor (<em>TraesCS5A02G306700</em>), a cell division cycle (CDC) 5‐like protein (<em>TraesCS5A02G307100</em>), and two serine carboxypeptidase‐like (SCPL) proteins (<em>TraesCS5A02G307200</em> and <em>TraesCS5A02G307400</em>) were associated with <em>QWs.ugw‐5A.5</em>. An At3g13620‐like probable polyamine transporter (<em>TraesCS5D02G531700</em>) and an eIF1AD eukaryotic translation factor protein (<em>TraesCS5D02G532300</em>) were associated with <em>QWs.ugw‐5D.1</em>. Two cytosolic sulfotransferase (SOT) 5‐like proteins (<em>TraesCS7B02G475800</em> and <em>TraesCS7B02G476000</em>), an Asp1‐like aspartic proteinase (<em>TraesCS7B02G475600</em>), and three flavin‐containing monooxygenase (FMO) 1‐like proteins (<em>TraesCS7B02G475700</em>, <em>TraesCS7B02G475900</em>, and <em>TraesCS7B02G476100</em>) were associated with <em>QWs.ugw‐7B.2</em>.</p>
<section class="tw xbox font-sm" id="tpg270091-tbl-0005"><h4 class="obj_head">TABLE 5.</h4>
<div class="caption p"><p>Putative candidate genes for significant quantitative trait loci associated with winter survival in the “3T” subpopulation of the Canadian Winter Wheat Diversity Panel (CWWDP).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">QTL</th>
<th align="left" rowspan="1" colspan="1">Gene</th>
<th align="left" rowspan="1" colspan="1">Location</th>
<th align="left" rowspan="1" colspan="1">Protein</th>
<th align="left" rowspan="1" colspan="1">% Identical</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>QWs.ugw‐5A.5</em>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>TraesCS5A02G306700</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>chr5A:519693115–519693939</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>transcription factor MYC2‐like (<em>Triticum dicoccoides</em>)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>100.00</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">TraesCS5A02G306800</td>
<td align="left" rowspan="1" colspan="1">chr5A:519852675–519854655</td>
<td align="left" rowspan="1" colspan="1">DNA polymerase delta subunit 4 (<em>Triticum urartu</em>)</td>
<td align="left" rowspan="1" colspan="1">100.00</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">
<strong>TraesCS5A02G307100</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>chr5A:519893625–519899833</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>cell division cycle 5‐like protein (<em>Triticum aestivum</em>)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>100.00</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">
<strong>TraesCS5A02G307200</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>chr5A:519903856–519906212</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>serine carboxypeptidase‐like 2 (<em>Triticum aestivum</em>)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>100.00</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">
<strong>TraesCS5A02G307400</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>chr5A:520092612–520096620</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>serine carboxypeptidase‐like 15 (<em>Triticum aestivum</em>)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>99.71</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>QWs.ugw‐5D.1</em>
</td>
<td align="left" rowspan="1" colspan="1">TraesCS5D02G531600</td>
<td align="left" rowspan="1" colspan="1">chr5D:547049945–547057967</td>
<td align="left" rowspan="1" colspan="1">inner membrane protein PPF‐1, chloroplastic‐like (<em>Triticum aestivum</em>)</td>
<td align="left" rowspan="1" colspan="1">100.00</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">TraesCS5D02G531600</td>
<td align="left" rowspan="1" colspan="1">chr5D:547049945–547057967</td>
<td align="left" rowspan="1" colspan="1">inner membrane protein PPF‐1, chloroplastic‐like (<em>Triticum aestivum</em>)</td>
<td align="left" rowspan="1" colspan="1">99.01</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">
<strong>TraesCS5D02G531700</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>chr5D:547065459–547068033</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>probable polyamine transporter At3g13620 (A<em>egilops tauschii</em> subsp. strangulata)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>100.00</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">TraesCS5D02G531800</td>
<td align="left" rowspan="1" colspan="1">chr5D:547069168–547071896</td>
<td align="left" rowspan="1" colspan="1">ER lumen protein‐retaining receptor (<em>Aegilops tauschii</em> subsp. strangulata)</td>
<td align="left" rowspan="1" colspan="1">100.00</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">TraesCS5D02G532000</td>
<td align="left" rowspan="1" colspan="1">chr5D:547260823–547262224</td>
<td align="left" rowspan="1" colspan="1">pectate lyase 1‐like (<em>Aegilops tauschii</em> subsp. strangulata)</td>
<td align="left" rowspan="1" colspan="1">100.00</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">TraesCS5D02G532100</td>
<td align="left" rowspan="1" colspan="1">chr5D:547273461–547277281</td>
<td align="left" rowspan="1" colspan="1">4‐hydroxy‐3‐methylbut‐2‐enyl diphosphate reductase‐like isoform X2 (<em>Triticum aestivum</em>)</td>
<td align="left" rowspan="1" colspan="1">100.00</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">
<strong>TraesCS5D02G532300</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>chr5D:547430974–547431494</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>probable RNA‐binding protein EIF1AD (<em>Triticum aestivum</em>)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>100.00</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">TraesCS5D02G532500</td>
<td align="left" rowspan="1" colspan="1">chr5D:547501182–547528872</td>
<td align="left" rowspan="1" colspan="1">putative disease resistance protein RGA4 isoform X3 (<em>Triticum aestivum</em>)</td>
<td align="left" rowspan="1" colspan="1">100.00</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>QWs.ugw‐7B.2</em>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>TraesCS7B02G475600</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>chr7B:731958896–731962236</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>aspartic proteinase Asp1‐like isoform X2 (<em>Triticum aestivum</em>)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>99.75</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">
<strong>TraesCS7B02G475700</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>chr7B:732143178–732143636</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>probable flavin‐containing monooxygenase 1 (<em>Triticum aestivum</em>)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>100.00</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">
<strong>TraesCS7B02G475800</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>chr7B:732157936–732159273</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>cytosolic sulfotransferase 5‐like (<em>Triticum aestivum</em>)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>100.00</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">
<strong>TraesCS7B02G475900</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>chr7B:732191900–732195779</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>probable flavin‐containing monooxygenase 1 isoform X1 (<em>Triticum aestivum</em>)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>100.00</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">
<strong>TraesCS7B02G476000</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>chr7B:732199719–732201144</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>cytosolic sulfotransferase 5‐like (<em>Triticum aestivum</em>)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>100.00</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">
<strong>TraesCS7B02G476100</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>chr7B:732382086–732384175</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>probable flavin‐containing monooxygenase 1 (<em>Triticum aestivum</em>)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>95.86</strong>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270091-tbl-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270091-tbl5-note-0001"><p>
<em>Note</em>: The CWWDP population was assessed in three eastern Canadian environments: Elora 2016–2017, CÉROM 2017–2018, and Elora 2017–2018. The “3T” subpopulation consists of genotypes with three copies of the recessive <em>vrn‐A1</em> allele and the tolerant ‘T’ haplotype at <em>Fr‐A2</em>. Bolded rows correspond to putative candidate genes that had previous associations with stress response.</p></div></div></section><p>Although winterkill is the cumulative result of numerous biotic and abiotic stresses, lethal temperature is often considered the primary cause of winterkill. As a result, many mapping studies conducted in winter wheat have focused on the identification of cold tolerance‐related QTL in controlled environments (Baga et al., <a href="#tpg270091-bib-0003" class="usa-link" aria-describedby="tpg270091-bib-0003">2007</a>; Case et al., <a href="#tpg270091-bib-0011" class="usa-link" aria-describedby="tpg270091-bib-0011">2014</a>; Fowler et al., <a href="#tpg270091-bib-0023" class="usa-link" aria-describedby="tpg270091-bib-0023">2016</a>; Kandel et al., <a href="#tpg270091-bib-0038" class="usa-link" aria-describedby="tpg270091-bib-0038">2018</a>; Motomura et al., <a href="#tpg270091-bib-0051" class="usa-link" aria-describedby="tpg270091-bib-0051">2013</a>; Rojas‐Gutierrez et al., <a href="#tpg270091-bib-0064" class="usa-link" aria-describedby="tpg270091-bib-0064">2022</a>; Sofalian et al., <a href="#tpg270091-bib-0071" class="usa-link" aria-describedby="tpg270091-bib-0071">2008</a>; Vaitkeviciute et al., <a href="#tpg270091-bib-0088" class="usa-link" aria-describedby="tpg270091-bib-0088">2023</a>). Several additional studies have identified winter survival‐related QTLs in the natural environment, with putative QTLs located on all 21 chromosomes (Fowler et al., <a href="#tpg270091-bib-0023" class="usa-link" aria-describedby="tpg270091-bib-0023">2016</a>; Pang et al., <a href="#tpg270091-bib-0057" class="usa-link" aria-describedby="tpg270091-bib-0057">2021</a>; Vaitkeviciute et al., <a href="#tpg270091-bib-0088" class="usa-link" aria-describedby="tpg270091-bib-0088">2023</a>; Zhao et al., <a href="#tpg270091-bib-0103" class="usa-link" aria-describedby="tpg270091-bib-0103">2020</a>; Zhao et al., <a href="#tpg270091-bib-0102" class="usa-link" aria-describedby="tpg270091-bib-0102">2013</a>).</p>
<p>As cold tolerance is just one component of winter survival and most winter survival‐related QTLs were identified in different environments compared to the Great Lakes Basin, these loci may have limited applicability for MAS in eastern Canada. Chen (<a href="#tpg270091-bib-0013" class="usa-link" aria-describedby="tpg270091-bib-0013">2019</a>) was the first to identify putative winter survival‐related QTLs in eastern Canada using the CWWDP. In this study, three additional winter survival‐related QTLs were identified in the “3T” subpopulation of the CWWDP. These QTLs were located on chromosomes 5A, 5D, and 7B and had been masked by the major effects of <em>Vrn‐A1</em> and <em>Fr‐A2</em> in the initial study. The QTL on chromosome 5A (<em>QWS.ugw‐5A.5</em>) was previously identified by Chen (<a href="#tpg270091-bib-0013" class="usa-link" aria-describedby="tpg270091-bib-0013">2019</a>) but had a small effect on the phenotypic mean of the CWWDP.</p>
<p>
<em>QWs.ugw‐7B.2</em> explained 16.31% of phenotypic variation in the “3T” subpopulation but did not colocalize with known cold tolerance or winter survival‐related QTLs and therefore is novel. Instead, this genomic region had previous associations with root architecture and drought tolerance, and no known KASP markers have been developed (Iannucci et al., <a href="#tpg270091-bib-0035" class="usa-link" aria-describedby="tpg270091-bib-0035">2017</a>; Maccaferri et al., <a href="#tpg270091-bib-0047" class="usa-link" aria-describedby="tpg270091-bib-0047">2016</a>). Candidate genes for <em>QWs.ugw‐7B.2</em> included two cytosolic SOT 5‐like proteins (<em>TraesCS7B02G475800</em> and <em>TraesCS7B02G476000</em>), an Asp1‐like aspartic proteinase (<em>TraesCS7B02G475600</em>), and three FMO 1‐like proteins (<em>TraesCS7B02G475700</em>, <em>TraesCS7B02G475900</em>, and <em>TraesCS7B02G476100</em>; Table <a href="#tpg270091-tbl-0005" class="usa-link">5</a>). AtSOT5 is a negative regulator of kaempferol, a well‐characterized flavonoid with antioxidant properties that has been implicated in drought and cold tolerance (Hashiguchi et al., <a href="#tpg270091-bib-0027" class="usa-link" aria-describedby="tpg270091-bib-0027">2013</a>; Jan et al., <a href="#tpg270091-bib-0037" class="usa-link" aria-describedby="tpg270091-bib-0037">2022</a>; Zhao et al., <a href="#tpg270091-bib-0103" class="usa-link" aria-describedby="tpg270091-bib-0103">2020</a>). Upregulation of Asp1‐like aspartic proteinases has been observed under salt stress, drought, and low nitrogen conditions (Alshameri et al., <a href="#tpg270091-bib-0001" class="usa-link" aria-describedby="tpg270091-bib-0001">2020</a>; Luo et al., <a href="#tpg270091-bib-0046" class="usa-link" aria-describedby="tpg270091-bib-0046">2019</a>; Xin et al., <a href="#tpg270091-bib-0095" class="usa-link" aria-describedby="tpg270091-bib-0095">2021</a>), but their specific role in abiotic stress response is unclear. Overexpression of <em>FMO1</em> has been associated with increased systemic acquired pathogen resistance (via detoxification of pathogen‐derived compounds), accumulation of reactive oxygen species (ROS), and programmed cell death (Koch et al., <a href="#tpg270091-bib-0039" class="usa-link" aria-describedby="tpg270091-bib-0039">2006</a>; Mishina &amp; Zeier, <a href="#tpg270091-bib-0050" class="usa-link" aria-describedby="tpg270091-bib-0050">2006</a>; Olszak et al., <a href="#tpg270091-bib-0054" class="usa-link" aria-describedby="tpg270091-bib-0054">2006</a>).</p>
<p>
<em>QWS.ugw‐5A.5</em> explained 12.41% of phenotypic variation in the “3T” subpopulation and colocalized with a well‐characterized region associated with cold tolerance and winter survival, and no known KASP markers have been developed (Case et al., <a href="#tpg270091-bib-0011" class="usa-link" aria-describedby="tpg270091-bib-0011">2014</a>; Soleimani et al., <a href="#tpg270091-bib-0072" class="usa-link" aria-describedby="tpg270091-bib-0072">2022</a>; Zhao et al., <a href="#tpg270091-bib-0103" class="usa-link" aria-describedby="tpg270091-bib-0103">2020</a>). Interestingly, the beneficial allele of <em>QWs.ugw‐5A.5</em> occurred at high frequency (85.3%) in the “3T” subpopulation and may highlight a regional adaptation for increased winter survival in eastern Canada (Table <a href="#tpg270091-tbl-0003" class="usa-link">3</a>). Candidate genes for <em>QWs.ugw‐5A.5</em> included a MYC2‐like transcription factor (<em>TraesCS5A02G306700</em>), a CDC 5‐like protein (<em>TraesCS5A02G307100</em>), and two SCPL proteins (<em>TraesCS5A02G307200</em> and <em>TraesCS5A02G307400</em>; Table <a href="#tpg270091-tbl-0005" class="usa-link">5</a>). In wheat, TaMYC2 activates TaICE41, a homolog of AtICE1, to induce the <em>ICE‐CBF‐COR</em> cold‐responsive pathway (R. Wang et al., <a href="#tpg270091-bib-0091" class="usa-link" aria-describedby="tpg270091-bib-0091">2022</a>). The role of MYC2 has also been reported in jasmonic acid‐mediated drought tolerance, biosynthesis of secondary metabolites, and biotic stress tolerance (reviewed in Song et al., [<a href="#tpg270091-bib-0073" class="usa-link" aria-describedby="tpg270091-bib-0073">2022</a>]). CDC5 has been implicated in the miRNA‐mediated regulation of abscisic acid signaling and floral induction (Ferdous et al., <a href="#tpg270091-bib-0022" class="usa-link" aria-describedby="tpg270091-bib-0022">2017</a>; Hua et al., <a href="#tpg270091-bib-0033" class="usa-link" aria-describedby="tpg270091-bib-0033">2019</a>), and a number of <em>TaSCPL</em> genes have been identified with drought‐, ABA‐, and methyl jasmonate‐responsive elements located in the promoter sequences (Xu et al., <a href="#tpg270091-bib-0096" class="usa-link" aria-describedby="tpg270091-bib-0096">2021</a>). However, the role of CDC5 and TaSCPL proteins in cold tolerance has yet to be examined in wheat.</p>
<p>
<em>QWs.ugw‐5D.1</em> explained 5.40% of phenotypic variation in the “3T” subpopulation and did not colocalize with known cold tolerance or winter survival‐related QTLs and therefore is novel. This genomic region had previous associations with disease resistance, and a functional KASP marker has been developed (Hu et al., <a href="#tpg270091-bib-0032" class="usa-link" aria-describedby="tpg270091-bib-0032">2020</a>; S. Liu, Yang, et al., <a href="#tpg270091-bib-0043" class="usa-link" aria-describedby="tpg270091-bib-0043">2014</a>). Candidate genes for <em>QWs.ugw‐5D.1</em> included an At3g13620‐like probable polyamine transporter (<em>TraesCS5D02G531700</em>) and an eIF1AD eukaryotic translation factor protein (<em>TraesCS5D02G532300</em>; Table <a href="#tpg270091-tbl-0005" class="usa-link">5</a>). <em>At3g13620</em> encodes a probable polyamine transporter with homology to LmPOT1 (<em>Leishmania major</em>) and TcPAT12 (<em>Trypanosoma cruzi</em>), high‐affinity transporters for putrescine and spermidine (Carrillo et al., <a href="#tpg270091-bib-0010" class="usa-link" aria-describedby="tpg270091-bib-0010">2006</a>; Ge, <a href="#tpg270091-bib-0024" class="usa-link" aria-describedby="tpg270091-bib-0024">2011</a>; Hasne &amp; Ullman, <a href="#tpg270091-bib-0028" class="usa-link" aria-describedby="tpg270091-bib-0028">2005</a>). Increased antioxidant activity has been associated with the accumulation of polyamines (Seo et al., <a href="#tpg270091-bib-0066" class="usa-link" aria-describedby="tpg270091-bib-0066">2019</a>; Sheteiwy et al., <a href="#tpg270091-bib-0067" class="usa-link" aria-describedby="tpg270091-bib-0067">2017</a>). The accumulation of putrescine and spermidine has been noted in wheat during cold acclimation, with significantly higher quantities observed in cold‐hardy wheat cultivars compared to non‐hardy cultivars (Asghar et al., <a href="#tpg270091-bib-0002" class="usa-link" aria-describedby="tpg270091-bib-0002">2021</a>; Handa &amp; Mattoo, <a href="#tpg270091-bib-0026" class="usa-link" aria-describedby="tpg270091-bib-0026">2010</a>). The eIF1 protein subfamily is also known to mitigate cellular damage by regulating antioxidant and enzymatic activity (Yang et al., <a href="#tpg270091-bib-0097" class="usa-link" aria-describedby="tpg270091-bib-0097">2017</a>), and the overexpression of <em>eIF1A</em> has been widely reported in response to salt and osmotic stress in wheat (Latha et al., <a href="#tpg270091-bib-0042" class="usa-link" aria-describedby="tpg270091-bib-0042">2004</a>; Rangan et al., <a href="#tpg270091-bib-0062" class="usa-link" aria-describedby="tpg270091-bib-0062">2009</a>; Rausell et al., <a href="#tpg270091-bib-0063" class="usa-link" aria-describedby="tpg270091-bib-0063">2003</a>; Sun &amp; Hong, <a href="#tpg270091-bib-0077" class="usa-link" aria-describedby="tpg270091-bib-0077">2013</a>; Yang et al., <a href="#tpg270091-bib-0097" class="usa-link" aria-describedby="tpg270091-bib-0097">2017</a>).</p>
<p>Several of the underlying candidate genes identified in this study were associated with the accumulation of ROS or with ROS scavenging. This suggests that redox homeostasis, in addition to cold tolerance, is important for winter survival in eastern Canada. ROS are important signaling molecules in systemic acquired acclimation and allow plants to regulate a variety of biological responses to environmental stimuli (Suzuki et al., <a href="#tpg270091-bib-0079" class="usa-link" aria-describedby="tpg270091-bib-0079">2013</a>; Zandalinas et al., <a href="#tpg270091-bib-0098" class="usa-link" aria-describedby="tpg270091-bib-0098">2019</a>). However, the low heritability of winter survival in the combined dataset (<em>H</em>
<sup>2</sup> = 0.3341) may suggest that the primary cause of winterkill differs across eastern Canada. Comparing field survival indices across a large geographic area may have identified a generic stress response rather than the unique genetic mechanisms required for survival in individual macro‐environments.</p>
<p>For example, <em>QWs.ugw‐5A.5</em> was also identified in CÉROM 2017–2018 (13.29% observed phenotypic variation) and suggests that lethal temperature has a larger impact on winter survival in southern Quebec than it does in southern Ontario. QTLs associated with cold tolerance were not identified in Elora 2016–2017. In this environment, 54.4% of observed phenotypic variation was attributed to a single QTL on chromosome 2B (<em>QWs.ugw‐2B.1</em>; Table <a href="#tpg270091-supinfo-0001" class="usa-link">S4</a>). This QTL colocalized with <em>TraesCS2B02G209300</em>, a gene encoding an IMPAIRED IN BABA‐INDUCED STERILITY 1 (IBS)‐like protein (Table <a href="#tpg270091-supinfo-0001" class="usa-link">S6</a>) involved in the priming of salicylic acid‐dependent stress response pathways (Slaughter et al., <a href="#tpg270091-bib-0070" class="usa-link" aria-describedby="tpg270091-bib-0070">2012</a>; Ton et al., <a href="#tpg270091-bib-0081" class="usa-link" aria-describedby="tpg270091-bib-0081">2005</a>). Significant winter survival‐related QTLs were not identified in Elora 2017–2018. This environment had low variation in winter survival (CV = 9.97%) and highlights an additional challenge for winter wheat breeders in eastern Canada. Significant winterkill events occur sporadically in southern and central Ontario (approximately one in 8 years; Statistics Canada, <a href="#tpg270091-bib-0074" class="usa-link" aria-describedby="tpg270091-bib-0074">1997</a>, <a href="#tpg270091-bib-0075" class="usa-link" aria-describedby="tpg270091-bib-0075">2007</a>, <a href="#tpg270091-bib-0076" class="usa-link" aria-describedby="tpg270091-bib-0076">2019</a>). Germplasm is often evaluated in moderate winter environments (similar to Elora 2016–2017 and Elora 2017–2018), which are unlikely to represent significant winterkill events in the region. Additional research is required to define macro‐environments for winterkill in eastern Canada and identify the primary cause of winterkill in southern and central Ontario.</p>
<p>Previous research has elucidated differences in outputs from models including FarmCPU and BLINK (Huang et al., <a href="#tpg270091-bib-0034" class="usa-link" aria-describedby="tpg270091-bib-0034">2018</a>). Comparative studies have identified an increase in the false positive rate associated with FarmCPU, which may explain some of the lack of consensus between the two models utilized in this study (Cebeci et al., <a href="#tpg270091-bib-0012" class="usa-link" aria-describedby="tpg270091-bib-0012">2023</a>; Sandhu et al., <a href="#tpg270091-bib-0065" class="usa-link" aria-describedby="tpg270091-bib-0065">2024</a>). Furthermore, the high degree of genotype‐by‐environment variability observed across location‐years in this study may have also played a factor in the absence of overlap in QTL markers.</p></section></section><section id="tpg270091-sec-0140"><h2 class="pmc_sec_title">4. CONCLUSION</h2>
<p>Three significant winter survival‐related QTLs were identified in this study (on chromosomes 5A, 5D, and 7B). As their impact on mean winter survival was minimal (0.7%–1.8%), these QTLs are unlikely to improve winter survival in populations derived from elite eastern Canadian cultivars via MAS. However, the beneficial alleles for these QTLs were prevalent in Canadian winter wheat germplasm. Functional markers derived from these QTLs may be useful in introgressing non‐adapted germplasm into eastern Canadian winter wheat breeding programs. Candidate genes associated with these QTLs functioned in redox homeostasis and known cold response pathways. Candidate genes identified in individual environments highlighted varying biological responses associated with winter survival.</p>
<p>Additional research is required to identify macro‐environments with similar causes of winterkill in eastern Canada. Field survival indices within these macro‐environments may provide higher quality phenotypic data and highlight specific biological responses required for winter survival in these regions. Winterkill events also occur sporadically in the main winter wheat production regions of eastern Canada. This creates an additional challenge for plant breeders, as many testing environments are not representative of significant winterkill events. The development of functional molecular markers with high marker utility associated with specific causes of winterkill or the use of phenotyping methods (e.g., low‐temperature 50, threshold induction temperature, and vegetative‐to‐reproductive transition) under artificial conditions may help to differentiate between genotypes in the absence of significant environmental stress.</p></section><section id="tpg270091-sec-0150"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Rachel Whiting</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; validation; visualization; writing—original draft; writing—review and editing. <strong>Alexandra Ficht</strong>: Conceptualization; data curation; formal analysis; methodology; project administration; validation; visualization; writing—original draft; writing—review and editing. <strong>Yi Chen</strong>: Formal analysis; methodology; writing—review and editing. <strong>Davoud Torkamaneh</strong>: Software; writing—review and editing. <strong>Joseph Colasanti</strong>: Supervision; writing—review and editing. <strong>Eric M. Lyons</strong>: Conceptualization; funding acquisition; supervision; writing—review and editing.</p></section><section id="tpg270091-sec-0170"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interests.</p></section><section id="sec17"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270091-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270091-supitem-0001"><div class="caption p"><p>Supplemental File S1 includes a table of all the genotypes that comprise the CWWDP. The supplemental figures and tables include climatic data for all environments, additional data analysis for winter survival and QTL information that was not required in the main text.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365865/bin/TPG2-18-e70091-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70091-s002.docx</a><sup> (41.7KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270091-supitem-0002"><div class="caption p"><p>Supplementary Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365865/bin/TPG2-18-e70091-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70091-s001.docx</a><sup> (3.3MB, docx) </sup>
</div></div></section></section><section id="tpg270091-sec-0160" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>The authors would like to thank the University of Guelph Wheat Breeding Program and the staff at CÉROM for their help throughout the project.</p></section><section id="notes1"><p>


Whiting, R.
, 
Ficht, A.
, 
Chen, Y.
, 
Torkamaneh, D.
, 
Colasanti, J.
, &amp; 
Lyons, E. M.
 (2025). Genome‐wide association analysis of winter survival in a diverse Canadian winter wheat population. The Plant Genome, 18, e70091. 10.1002/tpg2.70091

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270091-note-0001"><p>Assigned to Associate Editor Susanne Dreisigacker.</p></div></div></section></section><section id="tpg270091-sec-0190"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The datasets generated during and/or analyzed during the current study are available from the corresponding author on reasonable request.</p></section><section id="tpg270091-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270091-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270091-bib-0001">
<cite>

Alshameri, A.
, 
Al‐Qurainy, F.
, 
Gaafar, A. R.
, 
Khan, S.
, 
Nadeem, M.
, 
Alansi, S.
, 
Shaikhaldein, H. O.
, &amp; 
Salih, A. M.
 (2020). Identification of differentially expressed drought‐responsive genes in guar [<em>Cyamopsis tetragonoloba</em> (L.) Taub]. International Journal of Genomics, 2020, 4147615.
</cite> [<a href="https://doi.org/10.1155/2020/4147615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7732403/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33344629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Genomics&amp;title=Identification%20of%20differentially%20expressed%20drought%E2%80%90responsive%20genes%20in%20guar%20%5BCyamopsis%20tetragonoloba%20(L.)%20Taub%5D&amp;author=A.%20Alshameri&amp;author=F.%20Al%E2%80%90Qurainy&amp;author=A.%20R.%20Gaafar&amp;author=S.%20Khan&amp;author=M.%20Nadeem&amp;volume=2020&amp;publication_year=2020&amp;pages=4147615&amp;pmid=33344629&amp;doi=10.1155/2020/4147615&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0002">
<cite>

Asghar, M. A.
, 
Mednyánszky, Z.
, 
Simon‐Sarkadi, L.
, &amp; 
Kocsy, G.
 (2021). Different induction of biogenic amine accumulation during cold acclimation in <em>Triticeae</em> genotypes with varying freezing tolerance. Brazilian Journal of Botany, 44, 11–15. 10.1007/s40415-020-00690-9
</cite> [<a href="https://doi.org/10.1007/s40415-020-00690-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brazilian%20Journal%20of%20Botany&amp;title=Different%20induction%20of%20biogenic%20amine%20accumulation%20during%20cold%20acclimation%20in%20Triticeae%20genotypes%20with%20varying%20freezing%20tolerance&amp;author=M.%20A.%20Asghar&amp;author=Z.%20Medny%C3%A1nszky&amp;author=L.%20Simon%E2%80%90Sarkadi&amp;author=G.%20Kocsy&amp;volume=44&amp;publication_year=2021&amp;pages=11-15&amp;doi=10.1007/s40415-020-00690-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0003">
<cite>

Baga, M.
, 
Chodaparambil, S. V.
, 
Limin, A. E.
, 
Pecar, M.
, 
Fowler, D. B.
, &amp; 
Chibbar, R. N.
 (2007). Identification of quantitative trait loci and associated candidate genes for low‐temperature tolerance in cold‐hardy winter wheat. Functional &amp; Integrative Genomics, 7, 53–68.
</cite> [<a href="https://doi.org/10.1007/s10142-006-0030-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16775685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20&amp;%20Integrative%20Genomics&amp;title=Identification%20of%20quantitative%20trait%20loci%20and%20associated%20candidate%20genes%20for%20low%E2%80%90temperature%20tolerance%20in%20cold%E2%80%90hardy%20winter%20wheat&amp;author=M.%20Baga&amp;author=S.%20V.%20Chodaparambil&amp;author=A.%20E.%20Limin&amp;author=M.%20Pecar&amp;author=D.%20B.%20Fowler&amp;volume=7&amp;publication_year=2007&amp;pages=53-68&amp;pmid=16775685&amp;doi=10.1007/s10142-006-0030-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0004">
<cite>

Beil, C. T.
, 
Anderson, V. A.
, 
Morgounov, A.
, &amp; 
Haley, S. D.
 (2019). Genomic selection for winter survival ability among a diverse collection of facultative and winter wheat genotypes. Molecular Breeding, 39, 29. 10.1007/s11032-018-0925-8
</cite> [<a href="https://doi.org/10.1007/s11032-018-0925-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Genomic%20selection%20for%20winter%20survival%20ability%20among%20a%20diverse%20collection%20of%20facultative%20and%20winter%20wheat%20genotypes&amp;author=C.%20T.%20Beil&amp;author=V.%20A.%20Anderson&amp;author=A.%20Morgounov&amp;author=S.%20D.%20Haley&amp;volume=39&amp;publication_year=2019&amp;pages=29&amp;doi=10.1007/s11032-018-0925-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0005">
<cite>

Bélanger, G.
, 
Castonguay, Y.
, 
Bertrand, A.
, 
Dhont, C.
, 
Rochette, P.
, 
Couture, L.
, 
Drapeau, R.
, 
Mongrain, D.
, 
Chalifour, F. P.
, &amp; 
Michaud, R.
 (2006). Winter damage to perennial forage crops in eastern Canada: Causes, mitigation, and prediction. Canadian Journal of Plant Science, 86, 33–47. 10.4141/P04-171
</cite> [<a href="https://doi.org/10.4141/P04-171" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Canadian%20Journal%20of%20Plant%20Science&amp;title=Winter%20damage%20to%20perennial%20forage%20crops%20in%20eastern%20Canada:%20Causes,%20mitigation,%20and%20prediction&amp;author=G.%20B%C3%A9langer&amp;author=Y.%20Castonguay&amp;author=A.%20Bertrand&amp;author=C.%20Dhont&amp;author=P.%20Rochette&amp;volume=86&amp;publication_year=2006&amp;pages=33-47&amp;doi=10.4141/P04-171&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0006">
<cite>

Bernardo, R.
 (2020). Reinventing quantitative genetics for plant breeding: Something old, something new, something borrowed, something BLUE. Heredity, 125, 375–385. 10.1038/s41437-020-0312-1

</cite> [<a href="https://doi.org/10.1038/s41437-020-0312-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7784685/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32296132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Reinventing%20quantitative%20genetics%20for%20plant%20breeding:%20Something%20old,%20something%20new,%20something%20borrowed,%20something%20BLUE&amp;author=R.%20Bernardo&amp;volume=125&amp;publication_year=2020&amp;pages=375-385&amp;pmid=32296132&amp;doi=10.1038/s41437-020-0312-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0007">
<cite>

Brachi, B.
, 
Morris, G. P.
, &amp; 
Borevitz, J. O.
 (2011). Genome‐wide association studies in plants: The missing heritability is in the field. Genome Biology, 12, 1–8. 10.1186/gb-2011-12-10-232
</cite> [<a href="https://doi.org/10.1186/gb-2011-12-10-232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3333769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22035733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Genome%E2%80%90wide%20association%20studies%20in%20plants:%20The%20missing%20heritability%20is%20in%20the%20field&amp;author=B.%20Brachi&amp;author=G.%20P.%20Morris&amp;author=J.%20O.%20Borevitz&amp;volume=12&amp;publication_year=2011&amp;pages=1-8&amp;pmid=22035733&amp;doi=10.1186/gb-2011-12-10-232&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0008">
<cite>

Bradbury, P. J.
, 
Zhang, Z.
, 
Kroon, D. E.
, 
Casstevens, T. M.
, 
Ramdoss, Y.
, &amp; 
Buckler, E. S.
 (2007). TASSEL: Software for association mapping of complex traits in diverse samples. Bioinformatics, 23, 2633–2635. 10.1093/bioinformatics/btm308

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btm308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17586829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=TASSEL:%20Software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples&amp;author=P.%20J.%20Bradbury&amp;author=Z.%20Zhang&amp;author=D.%20E.%20Kroon&amp;author=T.%20M.%20Casstevens&amp;author=Y.%20Ramdoss&amp;volume=23&amp;publication_year=2007&amp;pages=2633-2635&amp;pmid=17586829&amp;doi=10.1093/bioinformatics/btm308&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0009">
<cite>

Browning, S. R.
, &amp; 
Browning, B. L.
 (2007). Rapid and accurate haplotype phasing and missing‐data inference for whole‐genome association studies by use of localized haplotype clustering. American Journal of Human Genetics, 81, 1084–1097. 10.1086/521987

</cite> [<a href="https://doi.org/10.1086/521987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2265661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17924348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Rapid%20and%20accurate%20haplotype%20phasing%20and%20missing%E2%80%90data%20inference%20for%20whole%E2%80%90genome%20association%20studies%20by%20use%20of%20localized%20haplotype%20clustering&amp;author=S.%20R.%20Browning&amp;author=B.%20L.%20Browning&amp;volume=81&amp;publication_year=2007&amp;pages=1084-1097&amp;pmid=17924348&amp;doi=10.1086/521987&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0010">
<cite>

Carrillo, C.
, 
Canepa, G. E.
, 
Algranati, I. D.
, &amp; 
Pereira, C. A.
 (2006). Molecular and functional characterization of a spermidine transporter (<em>Tc</em>PAT12) from <em>Trypanosoma cruzi</em>
. Biochemical and Biophysical Research Communications, 344, 936–940. 10.1016/j.bbrc.2006.03.215

</cite> [<a href="https://doi.org/10.1016/j.bbrc.2006.03.215" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16631600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemical%20and%20Biophysical%20Research%20Communications&amp;title=Molecular%20and%20functional%20characterization%20of%20a%20spermidine%20transporter%20(TcPAT12)%20from%20Trypanosoma%20cruzi&amp;author=C.%20Carrillo&amp;author=G.%20E.%20Canepa&amp;author=I.%20D.%20Algranati&amp;author=C.%20A.%20Pereira&amp;volume=344&amp;publication_year=2006&amp;pages=936-940&amp;pmid=16631600&amp;doi=10.1016/j.bbrc.2006.03.215&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0011">
<cite>

Case, A. J.
, 
Skinner, D. Z.
, 
Garland‐Campbell, K. A.
, &amp; 
Carter, A. H.
 (2014). Freezing tolerance‐associated quantitative trait loci in the Brundage × coda wheat recombinant inbred line population. Crop Science, 54, 982–992. 10.2135/cropsci2013.08.0526
</cite> [<a href="https://doi.org/10.2135/cropsci2013.08.0526" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Freezing%20tolerance%E2%80%90associated%20quantitative%20trait%20loci%20in%20the%20Brundage%20%C3%97%20coda%20wheat%20recombinant%20inbred%20line%20population&amp;author=A.%20J.%20Case&amp;author=D.%20Z.%20Skinner&amp;author=K.%20A.%20Garland%E2%80%90Campbell&amp;author=A.%20H.%20Carter&amp;volume=54&amp;publication_year=2014&amp;pages=982-992&amp;doi=10.2135/cropsci2013.08.0526&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0012">
<cite>

Cebeci, Z.
, 
Bayraktar, M.
, &amp; 
Gökçe, G.
 (2023). Comparison of the statistical methods for genome‐wide association studies on simulated quantitative traits of domesticated goats (<em>Capra hircus</em> L.). Small Ruminant Research, 227, 107053. 10.1016/j.smallrumres.2023.107053
</cite> [<a href="https://doi.org/10.1016/j.smallrumres.2023.107053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Small%20Ruminant%20Research&amp;title=Comparison%20of%20the%20statistical%20methods%20for%20genome%E2%80%90wide%20association%20studies%20on%20simulated%20quantitative%20traits%20of%20domesticated%20goats%20(Capra%20hircus%20L.)&amp;author=Z.%20Cebeci&amp;author=M.%20Bayraktar&amp;author=G.%20G%C3%B6k%C3%A7e&amp;volume=227&amp;publication_year=2023&amp;pages=107053&amp;doi=10.1016/j.smallrumres.2023.107053&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0013">
<cite>

Chen, Y.
 (2019). Breeding for winter‐hardiness in winter wheat (Triticum aestivum L.) [Master's thesis]. University of Guelph.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Breeding%20for%20winter%E2%80%90hardiness%20in%20winter%20wheat%20(Triticum%20aestivum%20L.)&amp;author=Y.%20Chen&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0014">
<cite>

Chen, Y.
, 
Kaviani, M.
, 
Najafabadi, M. Y.
, 
McElroy, M.
, 
Rajcan, I.
, &amp; 
Navabi, A.
 (2024). Copy number variation at <em>Vrn‐A1</em> and haplotype diversity at <em>Fr‐A2</em> are major determinants of winter survival of winter wheat (<em>Triticum aestivum</em> L.) in Eastern Canada. Canadian Journal of Plant Science, 104, 4. 10.1139/cjps-2023-0101
</cite> [<a href="https://doi.org/10.1139/cjps-2023-0101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Canadian%20Journal%20of%20Plant%20Science&amp;title=Copy%20number%20variation%20at%20Vrn%E2%80%90A1%20and%20haplotype%20diversity%20at%20Fr%E2%80%90A2%20are%20major%20determinants%20of%20winter%20survival%20of%20winter%20wheat%20(Triticum%20aestivum%20L.)%20in%20Eastern%20Canada&amp;author=Y.%20Chen&amp;author=M.%20Kaviani&amp;author=M.%20Y.%20Najafabadi&amp;author=M.%20McElroy&amp;author=I.%20Rajcan&amp;volume=104&amp;publication_year=2024&amp;pages=4&amp;doi=10.1139/cjps-2023-0101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0015">
<cite>

Chen, Y.
, 
Sidhu, H. S.
, 
Kaviani, M.
, 
McElroy, M. S.
, 
Pozniak, C. J.
, &amp; 
Navabi, A.
 (2019). Application of image‐based phenotyping tools to identify QTL for in‐field winter survival of winter wheat (<em>Triticum aestivum</em> L.). Theoretical and Applied Genetics, 132, 2591–2604. 10.1007/s00122-019-03373-6

</cite> [<a href="https://doi.org/10.1007/s00122-019-03373-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31177292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Application%20of%20image%E2%80%90based%20phenotyping%20tools%20to%20identify%20QTL%20for%20in%E2%80%90field%20winter%20survival%20of%20winter%20wheat%20(Triticum%20aestivum%20L.)&amp;author=Y.%20Chen&amp;author=H.%20S.%20Sidhu&amp;author=M.%20Kaviani&amp;author=M.%20S.%20McElroy&amp;author=C.%20J.%20Pozniak&amp;volume=132&amp;publication_year=2019&amp;pages=2591-2604&amp;pmid=31177292&amp;doi=10.1007/s00122-019-03373-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0016">
<cite>

Crain, J.
, 
Mondal, S.
, 
Rutkoski, J.
, 
Singh, R. P.
, &amp; 
Poland, J.
 (2018). Combining high‐throughput phenotyping and genomic information to increase prediction and selection accuracy in wheat breeding. The Plant Genome, 11, 170043. 10.3835/plantgenome2017.05.0043
</cite> [<a href="https://doi.org/10.3835/plantgenome2017.05.0043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29505641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Combining%20high%E2%80%90throughput%20phenotyping%20and%20genomic%20information%20to%20increase%20prediction%20and%20selection%20accuracy%20in%20wheat%20breeding&amp;author=J.%20Crain&amp;author=S.%20Mondal&amp;author=J.%20Rutkoski&amp;author=R.%20P.%20Singh&amp;author=J.%20Poland&amp;volume=11&amp;publication_year=2018&amp;pages=170043&amp;pmid=29505641&amp;doi=10.3835/plantgenome2017.05.0043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0017">
<cite>

Crain, J.
, 
Reynolds, M.
, &amp; 
Poland, J.
 (2017). Utilizing high‐throughput phenotypic data for improved phenotypic selection of stress‐adaptive traits in wheat. Crop Science, 57, 648–659. 10.2135/cropsci2016.02.0135
</cite> [<a href="https://doi.org/10.2135/cropsci2016.02.0135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Utilizing%20high%E2%80%90throughput%20phenotypic%20data%20for%20improved%20phenotypic%20selection%20of%20stress%E2%80%90adaptive%20traits%20in%20wheat&amp;author=J.%20Crain&amp;author=M.%20Reynolds&amp;author=J.%20Poland&amp;volume=57&amp;publication_year=2017&amp;pages=648-659&amp;doi=10.2135/cropsci2016.02.0135&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0018">
<cite>

Danecek, P.
, 
Auton, A.
, 
Abecasis, G.
, 
Albers, C. A.
, 
Banks, E.
, 
DePristo, M. A.
, 
Handsaker, R. E.
, 
Lunter, G.
, 
Marth, G. T.
, 
Sherry, S. T.
, 
McVean, G.
, &amp; 
Durbin, R.
 (2011). The variant call format and VCFtools. Bioinformatics, 27, 2156–2158. 10.1093/bioinformatics/btr330

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3137218/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21653522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The%20variant%20call%20format%20and%20VCFtools&amp;author=P.%20Danecek&amp;author=A.%20Auton&amp;author=G.%20Abecasis&amp;author=C.%20A.%20Albers&amp;author=E.%20Banks&amp;volume=27&amp;publication_year=2011&amp;pages=2156-2158&amp;pmid=21653522&amp;doi=10.1093/bioinformatics/btr330&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0019">
<cite>

Dhillon, T.
, 
Pearce, S. P.
, 
Stockinger, E. J.
, 
Distelfeld, A.
, 
Li, C.
, 
Knox, A. K.
, 
Vashegyi, I.
, 
Vagujfalvi, A.
, 
Galiba, G.
, &amp; 
Dubcovsky, J.
 (2010). Regulation of freezing tolerance and flowering in temperate cereals: The VRN‐1 connection. Plant Physiology, 153, 1846–1858. 10.1104/pp.110.159079

</cite> [<a href="https://doi.org/10.1104/pp.110.159079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2923912/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20571115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Regulation%20of%20freezing%20tolerance%20and%20flowering%20in%20temperate%20cereals:%20The%20VRN%E2%80%901%20connection&amp;author=T.%20Dhillon&amp;author=S.%20P.%20Pearce&amp;author=E.%20J.%20Stockinger&amp;author=A.%20Distelfeld&amp;author=C.%20Li&amp;volume=153&amp;publication_year=2010&amp;pages=1846-1858&amp;pmid=20571115&amp;doi=10.1104/pp.110.159079&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0020">
<cite>

Diaz, A.
, 
Zikhali, M.
, 
Turner, A. S.
, 
Isaac, P.
, &amp; 
Laurie, D. A.
 (2012). Copy number variation affecting the <em>Photoperiod‐B1</em> and <em>Vernalization‐A1</em> genes is associated with altered flowering time in wheat (<em>Triticum aestivum</em>). PLoS One, 7, e33234. 10.1371/journal.pone.0033234

</cite> [<a href="https://doi.org/10.1371/journal.pone.0033234" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3310869/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22457747/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Copy%20number%20variation%20affecting%20the%20Photoperiod%E2%80%90B1%20and%20Vernalization%E2%80%90A1%20genes%20is%20associated%20with%20altered%20flowering%20time%20in%20wheat%20(Triticum%20aestivum)&amp;author=A.%20Diaz&amp;author=M.%20Zikhali&amp;author=A.%20S.%20Turner&amp;author=P.%20Isaac&amp;author=D.%20A.%20Laurie&amp;volume=7&amp;publication_year=2012&amp;pages=e33234&amp;pmid=22457747&amp;doi=10.1371/journal.pone.0033234&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0021">
<cite>

Ding, Y.
, 
Shi, Y.
, &amp; 
Yang, S.
 (2019). Advances and challenges in uncovering cold tolerance regulatory mechanisms in plants. New Phytology, 222, 1690–1704. 10.1111/nph.15696
</cite> [<a href="https://doi.org/10.1111/nph.15696" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30664232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytology&amp;title=Advances%20and%20challenges%20in%20uncovering%20cold%20tolerance%20regulatory%20mechanisms%20in%20plants&amp;author=Y.%20Ding&amp;author=Y.%20Shi&amp;author=S.%20Yang&amp;volume=222&amp;publication_year=2019&amp;pages=1690-1704&amp;pmid=30664232&amp;doi=10.1111/nph.15696&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0022">
<cite>

Ferdous, J.
, 
Sanchez‐Ferrero, J. C.
, 
Langridge, P.
, 
Milne, L.
, 
Chowdhury, J.
, 
Brien, C.
, &amp; 
Tricker, P. J.
 (2017). Differential expression of microRNAs and potential targets under drought stress in barley. Plant, Cell, &amp; Environment, 40, 11–24.</cite> [<a href="https://doi.org/10.1111/pce.12764" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27155357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Cell,%20&amp;%20Environment&amp;title=Differential%20expression%20of%20microRNAs%20and%20potential%20targets%20under%20drought%20stress%20in%20barley&amp;author=J.%20Ferdous&amp;author=J.%20C.%20Sanchez%E2%80%90Ferrero&amp;author=P.%20Langridge&amp;author=L.%20Milne&amp;author=J.%20Chowdhury&amp;volume=40&amp;publication_year=2017&amp;pages=11-24&amp;pmid=27155357&amp;doi=10.1111/pce.12764&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0023">
<cite>

Fowler, D. B.
, 
N'Diaye, A.
, 
Laudencia‐Chingcuanco, D.
, &amp; 
Pozniak, C. J.
 (2016). Quantitative trait loci associated with phenological development, low‐temperature tolerance, grain quality, and agronomic characters in wheat (<em>Triticum aestivum</em> L.). PLoS One, 11, e0152185. 10.1371/journal.pone.0152185

</cite> [<a href="https://doi.org/10.1371/journal.pone.0152185" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4809511/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27019468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Quantitative%20trait%20loci%20associated%20with%20phenological%20development,%20low%E2%80%90temperature%20tolerance,%20grain%20quality,%20and%20agronomic%20characters%20in%20wheat%20(Triticum%20aestivum%20L.)&amp;author=D.%20B.%20Fowler&amp;author=A.%20N'Diaye&amp;author=D.%20Laudencia%E2%80%90Chingcuanco&amp;author=C.%20J.%20Pozniak&amp;volume=11&amp;publication_year=2016&amp;pages=e0152185&amp;pmid=27019468&amp;doi=10.1371/journal.pone.0152185&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0024">
<cite>

Ge, L.
 (2011). Characterization of three novel polyamine transporters in plants [Thesis]. Bowling Green State University.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20three%20novel%20polyamine%20transporters%20in%20plants&amp;author=L.%20Ge&amp;publication_year=2011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0025">
<cite>

Guo, X.
, 
Liu, D.
, &amp; 
Chong, K.
 (2018). Cold signaling in plants: Insights into mechanisms and regulation. Journal of Integrative Plant Biology, 60, 745–756. 10.1111/jipb.12706

</cite> [<a href="https://doi.org/10.1111/jipb.12706" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30094919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Integrative%20Plant%20Biology&amp;title=Cold%20signaling%20in%20plants:%20Insights%20into%20mechanisms%20and%20regulation&amp;author=X.%20Guo&amp;author=D.%20Liu&amp;author=K.%20Chong&amp;volume=60&amp;publication_year=2018&amp;pages=745-756&amp;pmid=30094919&amp;doi=10.1111/jipb.12706&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0026">
<cite>

Handa, A. K.
, &amp; 
Mattoo, A. K.
 (2010). Differential and functional interactions emphasize the multiple roles of polyamines in plants. Plant Physiology and Biochemistry, 48, 540–546. 10.1016/j.plaphy.2010.02.009

</cite> [<a href="https://doi.org/10.1016/j.plaphy.2010.02.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20227284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology%20and%20Biochemistry&amp;title=Differential%20and%20functional%20interactions%20emphasize%20the%20multiple%20roles%20of%20polyamines%20in%20plants&amp;author=A.%20K.%20Handa&amp;author=A.%20K.%20Mattoo&amp;volume=48&amp;publication_year=2010&amp;pages=540-546&amp;pmid=20227284&amp;doi=10.1016/j.plaphy.2010.02.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0027">
<cite>

Hashiguchi, T.
, 
Sakakibara, Y.
, 
Hara, Y.
, 
Shimohira, T.
, 
Kurogi, K.
, 
Akashi, R.
, 
Liu, M. C.
, &amp; 
Suiko, M.
 (2013). Identification and characterization of a novel kaempferol sulfotransferase from <em>Arabidopsis thaliana</em>
. Biochemical and Biophysical Research Communications, 434, 829–835. 10.1016/j.bbrc.2013.04.022

</cite> [<a href="https://doi.org/10.1016/j.bbrc.2013.04.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23611783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemical%20and%20Biophysical%20Research%20Communications&amp;title=Identification%20and%20characterization%20of%20a%20novel%20kaempferol%20sulfotransferase%20from%20Arabidopsis%20thaliana&amp;author=T.%20Hashiguchi&amp;author=Y.%20Sakakibara&amp;author=Y.%20Hara&amp;author=T.%20Shimohira&amp;author=K.%20Kurogi&amp;volume=434&amp;publication_year=2013&amp;pages=829-835&amp;pmid=23611783&amp;doi=10.1016/j.bbrc.2013.04.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0028">
<cite>

Hasne, M. P.
, &amp; 
Ullman, B.
 (2005). Identification and characterization of a polyamine permease from the protozoan parasite Leishmania major. Journal of Biological Chemistry, 280, 15188–15194. 10.1074/jbc.M411331200

</cite> [<a href="https://doi.org/10.1074/jbc.M411331200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15632173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Identification%20and%20characterization%20of%20a%20polyamine%20permease%20from%20the%20protozoan%20parasite%20Leishmania%20major&amp;author=M.%20P.%20Hasne&amp;author=B.%20Ullman&amp;volume=280&amp;publication_year=2005&amp;pages=15188-15194&amp;pmid=15632173&amp;doi=10.1074/jbc.M411331200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0029">
<cite>

Hassan, M. A.
, 
Xiang, C.
, 
Farooq, M.
, 
Muhammad, N.
, 
Yan, Z.
, 
Hui, X.
, 
Yuanyuan, K.
, 
Bruno, A. K.
, 
Lele, Z.
, &amp; 
Jincai, L.
 (2021). Cold stress in wheat: Plant acclimation responses and management strategies. Frontiers in Plant Science, 12, 676884. 10.3389/fpls.2021.676884

</cite> [<a href="https://doi.org/10.3389/fpls.2021.676884" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8299469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34305976/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Cold%20stress%20in%20wheat:%20Plant%20acclimation%20responses%20and%20management%20strategies&amp;author=M.%20A.%20Hassan&amp;author=C.%20Xiang&amp;author=M.%20Farooq&amp;author=N.%20Muhammad&amp;author=Z.%20Yan&amp;volume=12&amp;publication_year=2021&amp;pages=676884&amp;pmid=34305976&amp;doi=10.3389/fpls.2021.676884&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0030">
<cite>

He, W.
, 
Grant, B. B.
, 
Smith, W. N.
, 
VanderZaag, A. C.
, 
Piquette, S.
, 
Qian, B.
, 
Jing, Q.
, 
Rennie, T. J.
, 
Bélanger, G.
, 
Jégo, G.
, &amp; 
Deen, B.
 (2019). Assessing alfalfa production under historical and future climate in eastern Canada: DNDC model development and application. Environmental Modelling &amp; Software, 122, 104540.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20Modelling%20&amp;%20Software&amp;title=Assessing%20alfalfa%20production%20under%20historical%20and%20future%20climate%20in%20eastern%20Canada:%20DNDC%20model%20development%20and%20application&amp;author=W.%20He&amp;author=B.%20B.%20Grant&amp;author=W.%20N.%20Smith&amp;author=A.%20C.%20VanderZaag&amp;author=S.%20Piquette&amp;volume=122&amp;publication_year=2019&amp;pages=104540&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0031">
<cite>

Hoffmann, A. A.
, &amp; 
Hercus, M. J.
 (2000). Environmental stress as an evolutionary force. Bioscience, 50, 217–226. 10.1641/0006-3568(2000)050[0217:ESAAEF]2.3.CO;2
</cite> [<a href="https://doi.org/10.1641/0006-3568(2000)050%5B0217:ESAAEF%5D2.3.CO;2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioscience&amp;title=Environmental%20stress%20as%20an%20evolutionary%20force&amp;author=A.%20A.%20Hoffmann&amp;author=M.%20J.%20Hercus&amp;volume=50&amp;publication_year=2000&amp;pages=217-226&amp;doi=10.1641/0006-3568(2000)050%5B0217:ESAAEF%5D2.3.CO;2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0032">
<cite>

Hu, W.
, 
Gao, D.
, 
Wu, H.
, 
Liu, J.
, 
Zhang, C.
, 
Wang, J.
, 
Jiang, Z.
, 
Liu, Y.
, 
Li, D.
, 
Zhang, Y.
, &amp; 
Lu, C.
 (2020). Genome‐wide association mapping revealed syntenic loci <em>QFhb‐4AL</em> and <em>QFhb‐5DL</em> for <em>Fusarium</em> head blight resistance in common wheat (<em>Triticum aestivum</em> L.). BMC Plant Biology, 20, 29. 10.1186/s12870-019-2177-0

</cite> [<a href="https://doi.org/10.1186/s12870-019-2177-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6971946/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31959107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Genome%E2%80%90wide%20association%20mapping%20revealed%20syntenic%20loci%20QFhb%E2%80%904AL%20and%20QFhb%E2%80%905DL%20for%20Fusarium%20head%20blight%20resistance%20in%20common%20wheat%20(Triticum%20aestivum%20L.)&amp;author=W.%20Hu&amp;author=D.%20Gao&amp;author=H.%20Wu&amp;author=J.%20Liu&amp;author=C.%20Zhang&amp;volume=20&amp;publication_year=2020&amp;pages=29&amp;pmid=31959107&amp;doi=10.1186/s12870-019-2177-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0033">
<cite>

Hua, Y.
, 
Zhang, C.
, 
Shi, W.
, &amp; 
Chen, H.
 (2019). High‐throughput sequencing reveals microRNAs and their targets in response to drought stress in wheat (<em>Triticum aestivum</em> L.). Biotechnology &amp; Biotechnological Equipment, 33, 465–471.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnology%20&amp;%20Biotechnological%20Equipment&amp;title=High%E2%80%90throughput%20sequencing%20reveals%20microRNAs%20and%20their%20targets%20in%20response%20to%20drought%20stress%20in%20wheat%20(Triticum%20aestivum%20L.)&amp;author=Y.%20Hua&amp;author=C.%20Zhang&amp;author=W.%20Shi&amp;author=H.%20Chen&amp;volume=33&amp;publication_year=2019&amp;pages=465-471&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0034">
<cite>

Huang, M.
, 
Liu, X.
, 
Zhou, Y.
, 
Summers, R. M.
, &amp; 
Zhang, Z.
 (2018). BLINK: A package for the next level of genome‐wide association studies with both individuals and markers in the millions. GigaScience, 8, giy154. 10.1093/gigascience/giy154
</cite> [<a href="https://doi.org/10.1093/gigascience/giy154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6365300/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30535326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GigaScience&amp;title=BLINK:%20A%20package%20for%20the%20next%20level%20of%20genome%E2%80%90wide%20association%20studies%20with%20both%20individuals%20and%20markers%20in%20the%20millions&amp;author=M.%20Huang&amp;author=X.%20Liu&amp;author=Y.%20Zhou&amp;author=R.%20M.%20Summers&amp;author=Z.%20Zhang&amp;volume=8&amp;publication_year=2018&amp;pages=giy154&amp;pmid=30535326&amp;doi=10.1093/gigascience/giy154&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0035">
<cite>

Iannucci, A.
, 
Marone, D.
, 
Russo, M. A.
, 
De Vita, P.
, 
Miullo, V.
, 
Ferragonio, P.
, 
Blanco, A.
, 
Gadaleta, A.
, &amp; 
Mastrangelo, A. M.
 (2017). Mapping QTL for root and shoot morphological traits in a durum wheat × <em>T. dicoccum</em> Segregating population at seedling stage. International Journal of Genomics, 2017, 6876393. 10.1155/2017/6876393

</cite> [<a href="https://doi.org/10.1155/2017/6876393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5563412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28845431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Genomics&amp;title=Mapping%20QTL%20for%20root%20and%20shoot%20morphological%20traits%20in%20a%20durum%20wheat%20%C3%97%20T.%20dicoccum%20Segregating%20population%20at%20seedling%20stage&amp;author=A.%20Iannucci&amp;author=D.%20Marone&amp;author=M.%20A.%20Russo&amp;author=P.%20De%20Vita&amp;author=V.%20Miullo&amp;volume=2017&amp;publication_year=2017&amp;pages=6876393&amp;pmid=28845431&amp;doi=10.1155/2017/6876393&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0036">
<cite>
International Wheat Genome Sequencing (IWGSC)
. (2018). Shifting the limits in wheat research and breeding using a fully annotated reference genome. Science, 361, eaar7191. 10.1126/science.aar7191

</cite> [<a href="https://doi.org/10.1126/science.aar7191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30115783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Shifting%20the%20limits%20in%20wheat%20research%20and%20breeding%20using%20a%20fully%20annotated%20reference%20genome&amp;volume=361&amp;publication_year=2018&amp;pages=eaar7191&amp;pmid=30115783&amp;doi=10.1126/science.aar7191&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0037">
<cite>

Jan, R.
, 
Khan, M. A.
, 
Asaf, S.
, Lubna, 
Waqas, M.
, 
Park, J.‐R.
, 
Asif, S.
, 
Kim, N.
, 
Lee, I. J.
, &amp; 
Kim, K. M.
 (2022). Drought and UV radiation stress tolerance in rice is improved by overaccumulation of non‐enzymatic antioxidant flavonoids. Antioxidants, 11, 917. 10.3390/antiox11050917

</cite> [<a href="https://doi.org/10.3390/antiox11050917" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9137601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35624781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antioxidants&amp;title=Drought%20and%20UV%20radiation%20stress%20tolerance%20in%20rice%20is%20improved%20by%20overaccumulation%20of%20non%E2%80%90enzymatic%20antioxidant%20flavonoids&amp;author=R.%20Jan&amp;author=M.%20A.%20Khan&amp;author=S.%20Asaf&amp;author=%20Lubna&amp;author=M.%20Waqas&amp;volume=11&amp;publication_year=2022&amp;pages=917&amp;pmid=35624781&amp;doi=10.3390/antiox11050917&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0038">
<cite>

Kandel, J. S.
, 
Huang, M.
, 
Zhang, Z.
, 
Skinner, D.
, &amp; 
See, D.
 (2018). Genetic diversity of clinal freezing tolerance variation in winter wheat landraces. Agronomy, 8, 95. 10.3390/agronomy8060095
</cite> [<a href="https://doi.org/10.3390/agronomy8060095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=Genetic%20diversity%20of%20clinal%20freezing%20tolerance%20variation%20in%20winter%20wheat%20landraces&amp;author=J.%20S.%20Kandel&amp;author=M.%20Huang&amp;author=Z.%20Zhang&amp;author=D.%20Skinner&amp;author=D.%20See&amp;volume=8&amp;publication_year=2018&amp;pages=95&amp;doi=10.3390/agronomy8060095&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0039">
<cite>

Koch, M.
, 
Vorwerk, S.
, 
Masur, C.
, 
Sharifi‐Sirchi, G.
, 
Olivieri, N.
, &amp; 
Schlaich, N. L.
 (2006). A role for a flavin‐containing mono‐oxygenase in resistance against microbial pathogens in Arabidopsis. The Plant Journal, 47, 629–639. 10.1111/j.1365-313X.2006.02813.x

</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2006.02813.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16856982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=A%20role%20for%20a%20flavin%E2%80%90containing%20mono%E2%80%90oxygenase%20in%20resistance%20against%20microbial%20pathogens%20in%20Arabidopsis&amp;author=M.%20Koch&amp;author=S.%20Vorwerk&amp;author=C.%20Masur&amp;author=G.%20Sharifi%E2%80%90Sirchi&amp;author=N.%20Olivieri&amp;volume=47&amp;publication_year=2006&amp;pages=629-639&amp;pmid=16856982&amp;doi=10.1111/j.1365-313X.2006.02813.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0040">
<cite>

Kreyling, J.
, 
Puechmaille, S. J.
, 
Malyshev, A. V.
, &amp; 
Valladares, F.
 (2019). Phenotypic plasticity closely linked to climate at origin and resulting in increased mortality under warming and frost stress in a common grass. Ecology &amp; Evolution, 9, 1344–1352.
</cite> [<a href="https://doi.org/10.1002/ece3.4848" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6374657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30805164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20&amp;%20Evolution&amp;title=Phenotypic%20plasticity%20closely%20linked%20to%20climate%20at%20origin%20and%20resulting%20in%20increased%20mortality%20under%20warming%20and%20frost%20stress%20in%20a%20common%20grass&amp;author=J.%20Kreyling&amp;author=S.%20J.%20Puechmaille&amp;author=A.%20V.%20Malyshev&amp;author=F.%20Valladares&amp;volume=9&amp;publication_year=2019&amp;pages=1344-1352&amp;pmid=30805164&amp;doi=10.1002/ece3.4848&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0041">
<cite>

Larsen, R. J.
, 
Beres, B. L.
, 
Blackshaw, R.
, &amp; 
Graf, R. J.
 (2018). Extending the growing season‐winter cereals in western Canada. Canadian Journal of Plant Science, 98, 267–277.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Canadian%20Journal%20of%20Plant%20Science&amp;title=Extending%20the%20growing%20season%E2%80%90winter%20cereals%20in%20western%20Canada&amp;author=R.%20J.%20Larsen&amp;author=B.%20L.%20Beres&amp;author=R.%20Blackshaw&amp;author=R.%20J.%20Graf&amp;volume=98&amp;publication_year=2018&amp;pages=267-277&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0042">
<cite>

Latha, R.
, 
Salekdeh, G. H.
, 
Bennett, J.
, &amp; 
Swaminathan, M. S.
 (2004). Molecular analysis of a stress‐induced cDNA encoding the translation initiation factor, <em>eIF1</em>, from the salt‐tolerant wild relative of rice, <em>Porteresia coarctata</em>
. Functional Plant Biology, 31, 1035–1042. 10.1071/FP03233

</cite> [<a href="https://doi.org/10.1071/FP03233" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32688972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20Plant%20Biology&amp;title=Molecular%20analysis%20of%20a%20stress%E2%80%90induced%20cDNA%20encoding%20the%20translation%20initiation%20factor,%20eIF1,%20from%20the%20salt%E2%80%90tolerant%20wild%20relative%20of%20rice,%20Porteresia%20coarctata&amp;author=R.%20Latha&amp;author=G.%20H.%20Salekdeh&amp;author=J.%20Bennett&amp;author=M.%20S.%20Swaminathan&amp;volume=31&amp;publication_year=2004&amp;pages=1035-1042&amp;pmid=32688972&amp;doi=10.1071/FP03233&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0043">
<cite>

Liu, S.
, 
Yang, X.
, 
Zhang, D.
, 
Bai, G.
, 
Chao, S.
, &amp; 
Bockus, W.
 (2014). Genome‐wide association analysis identified SNPs closely linked to a gene resistant to soil‐borne wheat mosaic virus. Theoretical and Applied Genetics, 127, 1039–1047. 10.1007/s00122-014-2277-z

</cite> [<a href="https://doi.org/10.1007/s00122-014-2277-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24522724/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genome%E2%80%90wide%20association%20analysis%20identified%20SNPs%20closely%20linked%20to%20a%20gene%20resistant%20to%20soil%E2%80%90borne%20wheat%20mosaic%20virus&amp;author=S.%20Liu&amp;author=X.%20Yang&amp;author=D.%20Zhang&amp;author=G.%20Bai&amp;author=S.%20Chao&amp;volume=127&amp;publication_year=2014&amp;pages=1039-1047&amp;pmid=24522724&amp;doi=10.1007/s00122-014-2277-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0045">
<cite>

Liu, X.
, 
Huang, M.
, 
Fan, B.
, 
Buckler, E. S.
, &amp; 
Zhang, Z.
 (2016). Iterative usage of fixed and random effect models for powerful and efficient genome‐wide association studies. PLoS Genetics, 12, e1005767. 10.1371/journal.pgen.1005767

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1005767" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4734661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26828793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&amp;title=Iterative%20usage%20of%20fixed%20and%20random%20effect%20models%20for%20powerful%20and%20efficient%20genome%E2%80%90wide%20association%20studies&amp;author=X.%20Liu&amp;author=M.%20Huang&amp;author=B.%20Fan&amp;author=E.%20S.%20Buckler&amp;author=Z.%20Zhang&amp;volume=12&amp;publication_year=2016&amp;pages=e1005767&amp;pmid=26828793&amp;doi=10.1371/journal.pgen.1005767&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0046">
<cite>

Luo, Q.
, 
Teng, W.
, 
Fang, S.
, 
Li, H.
, 
Li, B.
, 
Chu, J.
, 
Li, Z.
, &amp; 
Zheng, Q.
 (2019). Transcriptome analysis of salt‐stress response in three seedling tissues of common wheat. The Crop Journal, 7, 378–392. 10.1016/j.cj.2018.11.009
</cite> [<a href="https://doi.org/10.1016/j.cj.2018.11.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Crop%20Journal&amp;title=Transcriptome%20analysis%20of%20salt%E2%80%90stress%20response%20in%20three%20seedling%20tissues%20of%20common%20wheat&amp;author=Q.%20Luo&amp;author=W.%20Teng&amp;author=S.%20Fang&amp;author=H.%20Li&amp;author=B.%20Li&amp;volume=7&amp;publication_year=2019&amp;pages=378-392&amp;doi=10.1016/j.cj.2018.11.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0047">
<cite>

Maccaferri, M.
, 
El‐Feki, W.
, 
Nazemi, G.
, 
Salvi, S.
, 
Cane, M. A.
, 
Colalongo, M. C.
, 
Stefanelli, S.
, &amp; 
Tuberosa, R.
 (2016). Prioritizing quantitative trait loci for root system architecture in tetraploid wheat. Journal of Experimental Botany, 67, 1161–1178. 10.1093/jxb/erw039

</cite> [<a href="https://doi.org/10.1093/jxb/erw039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4753857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26880749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Prioritizing%20quantitative%20trait%20loci%20for%20root%20system%20architecture%20in%20tetraploid%20wheat&amp;author=M.%20Maccaferri&amp;author=W.%20El%E2%80%90Feki&amp;author=G.%20Nazemi&amp;author=S.%20Salvi&amp;author=M.%20A.%20Cane&amp;volume=67&amp;publication_year=2016&amp;pages=1161-1178&amp;pmid=26880749&amp;doi=10.1093/jxb/erw039&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0048">
<cite>

Michel, S.
, 
Loschenberger, F.
, 
Hellinger, J.
, 
Strasser, V.
, 
Ametz, C.
, 
Pachler, B.
, 
Sparry, E.
, &amp; 
Burstmayr, H.
 (2019). Improving and maintaining winter hardiness and frost tolerance in bread wheat by genomic selection. Frontiers in Plant Science, 10, 1195. 10.3389/fpls.2019.01195

</cite> [<a href="https://doi.org/10.3389/fpls.2019.01195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6781858/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31632427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Improving%20and%20maintaining%20winter%20hardiness%20and%20frost%20tolerance%20in%20bread%20wheat%20by%20genomic%20selection&amp;author=S.%20Michel&amp;author=F.%20Loschenberger&amp;author=J.%20Hellinger&amp;author=V.%20Strasser&amp;author=C.%20Ametz&amp;volume=10&amp;publication_year=2019&amp;pages=1195&amp;pmid=31632427&amp;doi=10.3389/fpls.2019.01195&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0049">
<cite>

Miller, A. K.
, 
Galiba, G.
, &amp; 
Dubcovsky, J.
 (2006). A cluster of 11 <em>CBF</em> transcription factors is located at the frost tolerance locus <em>Fr‐Am2</em> in <em>Triticum monococcum</em>
. Molecular Genetics and Genomics, 275, 193–203. 10.1007/s00438-005-0076-6

</cite> [<a href="https://doi.org/10.1007/s00438-005-0076-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16362370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Genetics%20and%20Genomics&amp;title=A%20cluster%20of%2011%20CBF%20transcription%20factors%20is%20located%20at%20the%20frost%20tolerance%20locus%20Fr%E2%80%90Am2%20in%20Triticum%20monococcum&amp;author=A.%20K.%20Miller&amp;author=G.%20Galiba&amp;author=J.%20Dubcovsky&amp;volume=275&amp;publication_year=2006&amp;pages=193-203&amp;pmid=16362370&amp;doi=10.1007/s00438-005-0076-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0050">
<cite>

Mishina, T. E.
, &amp; 
Zeier, J.
 (2006). The Arabidopsis flavin‐dependent monooxygenase FMO1 is an essential component of biologically induced systemic acquired resistance. Plant Physiology, 141, 1666–1675. 10.1104/pp.106.081257

</cite> [<a href="https://doi.org/10.1104/pp.106.081257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1533925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16778014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=The%20Arabidopsis%20flavin%E2%80%90dependent%20monooxygenase%20FMO1%20is%20an%20essential%20component%20of%20biologically%20induced%20systemic%20acquired%20resistance&amp;author=T.%20E.%20Mishina&amp;author=J.%20Zeier&amp;volume=141&amp;publication_year=2006&amp;pages=1666-1675&amp;pmid=16778014&amp;doi=10.1104/pp.106.081257&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0051">
<cite>

Motomura, Y.
, 
Kobayashi, F.
, 
Iehisa, J. C.
, &amp; 
Takumi, S.
 (2013). A major quantitative trait locus for cold‐responsive gene expression is linked to frost‐resistance gene <em>Fr‐A2</em> in common wheat. Breeding Science, 63, 58–67. 10.1270/jsbbs.63.58

</cite> [<a href="https://doi.org/10.1270/jsbbs.63.58" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3621446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23641182/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Breeding%20Science&amp;title=A%20major%20quantitative%20trait%20locus%20for%20cold%E2%80%90responsive%20gene%20expression%20is%20linked%20to%20frost%E2%80%90resistance%20gene%20Fr%E2%80%90A2%20in%20common%20wheat&amp;author=Y.%20Motomura&amp;author=F.%20Kobayashi&amp;author=J.%20C.%20Iehisa&amp;author=S.%20Takumi&amp;volume=63&amp;publication_year=2013&amp;pages=58-67&amp;pmid=23641182&amp;doi=10.1270/jsbbs.63.58&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0052">
<cite>

Noble, W. S.
 (2009). How does multiple testing correction work?
Nature Biotechnology, 27, 1135–1137. 10.1038/nbt1209-1135
</cite> [<a href="https://doi.org/10.1038/nbt1209-1135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2907892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20010596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=How%20does%20multiple%20testing%20correction%20work?&amp;author=W.%20S.%20Noble&amp;volume=27&amp;publication_year=2009&amp;pages=1135-1137&amp;pmid=20010596&amp;doi=10.1038/nbt1209-1135&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0053">
<cite>

Olson‐Manning, C. F.
, 
Wagner, M. R.
, &amp; 
Mitchell‐Olds, T.
 (2012). Adaptive evolution: Evaluating empirical support for theoretical predictions. Nature Reviews Genetics, 13, 867–877. 10.1038/nrg3322
</cite> [<a href="https://doi.org/10.1038/nrg3322" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3748133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23154809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=Adaptive%20evolution:%20Evaluating%20empirical%20support%20for%20theoretical%20predictions&amp;author=C.%20F.%20Olson%E2%80%90Manning&amp;author=M.%20R.%20Wagner&amp;author=T.%20Mitchell%E2%80%90Olds&amp;volume=13&amp;publication_year=2012&amp;pages=867-877&amp;pmid=23154809&amp;doi=10.1038/nrg3322&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0054">
<cite>

Olszak, B.
, 
Malinovsky, F. G.
, 
Brodersen, P.
, 
Grell, M.
, 
Giese, H.
, 
Petersen, M.
, &amp; 
Mundy, J.
 (2006). A putative flavin‐containing mono‐oxygenase as a marker for certain defense and cell death pathways. Plant Science, 170, 614–623. 10.1016/j.plantsci.2005.10.016
</cite> [<a href="https://doi.org/10.1016/j.plantsci.2005.10.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Science&amp;title=A%20putative%20flavin%E2%80%90containing%20mono%E2%80%90oxygenase%20as%20a%20marker%20for%20certain%20defense%20and%20cell%20death%20pathways&amp;author=B.%20Olszak&amp;author=F.%20G.%20Malinovsky&amp;author=P.%20Brodersen&amp;author=M.%20Grell&amp;author=H.%20Giese&amp;volume=170&amp;publication_year=2006&amp;pages=614-623&amp;doi=10.1016/j.plantsci.2005.10.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0055">
<cite>

Orr, H. A.
 (1998). The population genetics of adaptation: The distribution of factors fixed during adaptive evolution. Evolution; International Journal of Organic Evolution, 52, 935–949. 10.2307/2411226

</cite> [<a href="https://doi.org/10.2307/2411226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28565213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolution;%20International%20Journal%20of%20Organic%20Evolution&amp;title=The%20population%20genetics%20of%20adaptation:%20The%20distribution%20of%20factors%20fixed%20during%20adaptive%20evolution&amp;author=H.%20A.%20Orr&amp;volume=52&amp;publication_year=1998&amp;pages=935-949&amp;pmid=28565213&amp;doi=10.2307/2411226&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0056">
<cite>

Pandurangan, S.
, 
Workman, C.
, 
Nilsen, K.
, &amp; 
Kumar, S.
 (2021). Introduction to marker‐assisted selection in wheat breeding. In Bilichak A. &amp; Laurie J. D. (Eds.), Accelerated breeding of cereal crops (pp. 77–117). Humana.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Accelerated%20breeding%20of%20cereal%20crops&amp;author=S.%20Pandurangan&amp;author=C.%20Workman&amp;author=K.%20Nilsen&amp;author=S.%20Kumar&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0057">
<cite>

Pang, Y.
, 
Wu, Y.
, 
Liu, C.
, 
Li, W.
, 
St Amand, P.
, 
Bernardo, A.
, 
Wang, D.
, 
Dong, L.
, 
Yuan, X.
, 
Zhang, H.
, 
Zhao, M.
, 
Li, L.
, 
Wang, L.
, 
He, F.
, 
Liang, Y.
, 
Yan, Q.
, 
Lu, Y.
, 
Su, Y.
, 
Jiang, H.
, … 
Liu, S.
 (2021). High‐resolution genome‐wide association study and genomic prediction for disease resistance and cold tolerance in wheat. Theoretical and Applied Genetics, 134, 2857–2873. 10.1007/s00122-021-03863-6

</cite> [<a href="https://doi.org/10.1007/s00122-021-03863-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34075443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=High%E2%80%90resolution%20genome%E2%80%90wide%20association%20study%20and%20genomic%20prediction%20for%20disease%20resistance%20and%20cold%20tolerance%20in%20wheat&amp;author=Y.%20Pang&amp;author=Y.%20Wu&amp;author=C.%20Liu&amp;author=W.%20Li&amp;author=P.%20St%20Amand&amp;volume=134&amp;publication_year=2021&amp;pages=2857-2873&amp;pmid=34075443&amp;doi=10.1007/s00122-021-03863-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0059">
<cite>

Platten, J. D.
, 
Cobb, J. N.
, &amp; 
Zantua, R. E.
 (2019). Criteria for evaluating molecular markers: Comprehensive quality metrics to improve marker‐assisted selection. PLoS One, 14, e0210529. 10.1371/journal.pone.0210529

</cite> [<a href="https://doi.org/10.1371/journal.pone.0210529" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6333336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30645632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Criteria%20for%20evaluating%20molecular%20markers:%20Comprehensive%20quality%20metrics%20to%20improve%20marker%E2%80%90assisted%20selection&amp;author=J.%20D.%20Platten&amp;author=J.%20N.%20Cobb&amp;author=R.%20E.%20Zantua&amp;volume=14&amp;publication_year=2019&amp;pages=e0210529&amp;pmid=30645632&amp;doi=10.1371/journal.pone.0210529&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0060">
<cite>

Poland, J.
, 
Endelman, J.
, 
Dawson, J.
, 
Rutkoski, J.
, 
Wu, S.
, 
Manes, Y.
, 
Dreisigacker, S.
, 
Crossa, J.
, 
Sánchez‐Villeda, H.
, 
Sorrells, M.
, &amp; 
Jannink, J. L.
 (2012). Genomic selection in wheat breeding using genotyping‐by‐sequencing. The Plant Genome, 5, 103–113.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20selection%20in%20wheat%20breeding%20using%20genotyping%E2%80%90by%E2%80%90sequencing&amp;author=J.%20Poland&amp;author=J.%20Endelman&amp;author=J.%20Dawson&amp;author=J.%20Rutkoski&amp;author=S.%20Wu&amp;volume=5&amp;publication_year=2012&amp;pages=103-113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0061">
<cite>

Randhawa, H. S.
, 
Asif, M.
, 
Pozniak, C.
, 
Clarke, J. M.
, 
Graf, R. J.
, 
Fox, S. L.
, 
Humphreys, D. G.
, 
Knox, R. E.
, 
DePauw, R. M.
, 
Singh, A. K.
, 
Cuthbert, R. D.
, 
Hucl, P.
, 
Spaner, D.
, &amp; 
Gupta, P.
 (2013). Application of molecular markers to wheat breeding in Canada. Plant Breeding, 132, 458–471. 10.1111/pbr.12057
</cite> [<a href="https://doi.org/10.1111/pbr.12057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Breeding&amp;title=Application%20of%20molecular%20markers%20to%20wheat%20breeding%20in%20Canada&amp;author=H.%20S.%20Randhawa&amp;author=M.%20Asif&amp;author=C.%20Pozniak&amp;author=J.%20M.%20Clarke&amp;author=R.%20J.%20Graf&amp;volume=132&amp;publication_year=2013&amp;pages=458-471&amp;doi=10.1111/pbr.12057&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0062">
<cite>

Rangan, L.
, 
Rout, A.
, 
Sudarshan, M.
, &amp; 
Gregorio, G.
 (2009). Molecular cloning, expression and mapping of the translational initiation factor <em>eIF1</em> gene in <em>Oryza sativa</em>
. Functional Plant Biology, 36, 442–452. 10.1071/FP08276

</cite> [<a href="https://doi.org/10.1071/FP08276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32688658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20Plant%20Biology&amp;title=Molecular%20cloning,%20expression%20and%20mapping%20of%20the%20translational%20initiation%20factor%20eIF1%20gene%20in%20Oryza%20sativa&amp;author=L.%20Rangan&amp;author=A.%20Rout&amp;author=M.%20Sudarshan&amp;author=G.%20Gregorio&amp;volume=36&amp;publication_year=2009&amp;pages=442-452&amp;pmid=32688658&amp;doi=10.1071/FP08276&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0063">
<cite>

Rausell, A.
, 
Kanhonou, R.
, 
Yenush, L.
, 
Serrano, R.
, &amp; 
Ros, R.
 (2003). The translation initiation factor eIF1A is an important determinant in the tolerance to NaCl stress in yeast and plants. The Plant Journal, 34, 257–267. 10.1046/j.1365-313X.2003.01719.x

</cite> [<a href="https://doi.org/10.1046/j.1365-313X.2003.01719.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12713533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=The%20translation%20initiation%20factor%20eIF1A%20is%20an%20important%20determinant%20in%20the%20tolerance%20to%20NaCl%20stress%20in%20yeast%20and%20plants&amp;author=A.%20Rausell&amp;author=R.%20Kanhonou&amp;author=L.%20Yenush&amp;author=R.%20Serrano&amp;author=R.%20Ros&amp;volume=34&amp;publication_year=2003&amp;pages=257-267&amp;pmid=12713533&amp;doi=10.1046/j.1365-313X.2003.01719.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0064">
<cite>

Rojas‐Gutierrez, J. D.
, 
Lee, G.
, 
Sanderson, B. J.
, 
Jameel, M. I.
, &amp; 
Oakley, C. G.
 (2022). Genome‐wide association analysis of freezing tolerance in soft red winter wheat. Crop Science, 62, 637–647. 10.1002/csc2.20682
</cite> [<a href="https://doi.org/10.1002/csc2.20682" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genome%E2%80%90wide%20association%20analysis%20of%20freezing%20tolerance%20in%20soft%20red%20winter%20wheat&amp;author=J.%20D.%20Rojas%E2%80%90Gutierrez&amp;author=G.%20Lee&amp;author=B.%20J.%20Sanderson&amp;author=M.%20I.%20Jameel&amp;author=C.%20G.%20Oakley&amp;volume=62&amp;publication_year=2022&amp;pages=637-647&amp;doi=10.1002/csc2.20682&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0065">
<cite>

Sandhu, K. S.
, 
Burke, A. B.
, 
Merrick, L. F.
, 
Pumphrey, M. O.
, &amp; 
Carter, A. H.
 (2024). Comparing performances of different statistical models and multiple threshold methods in a nested association mapping population of wheat. Frontiers in Plant Science, 15, 1460353. 10.3389/fpls.2024.1460353

</cite> [<a href="https://doi.org/10.3389/fpls.2024.1460353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11482037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39416483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Comparing%20performances%20of%20different%20statistical%20models%20and%20multiple%20threshold%20methods%20in%20a%20nested%20association%20mapping%20population%20of%20wheat&amp;author=K.%20S.%20Sandhu&amp;author=A.%20B.%20Burke&amp;author=L.%20F.%20Merrick&amp;author=M.%20O.%20Pumphrey&amp;author=A.%20H.%20Carter&amp;volume=15&amp;publication_year=2024&amp;pages=1460353&amp;pmid=39416483&amp;doi=10.3389/fpls.2024.1460353&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0066">
<cite>

Seo, S. Y.
, 
Kim, Y. J.
, &amp; 
Park, K. Y.
 (2019). Increasing polyamine contents enhances the stress tolerance via reinforcement of antioxidative properties. Frontiers in Plant Science, 10, 1331. 10.3389/fpls.2019.01331

</cite> [<a href="https://doi.org/10.3389/fpls.2019.01331" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6834694/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31736992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Increasing%20polyamine%20contents%20enhances%20the%20stress%20tolerance%20via%20reinforcement%20of%20antioxidative%20properties&amp;author=S.%20Y.%20Seo&amp;author=Y.%20J.%20Kim&amp;author=K.%20Y.%20Park&amp;volume=10&amp;publication_year=2019&amp;pages=1331&amp;pmid=31736992&amp;doi=10.3389/fpls.2019.01331&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0067">
<cite>

Sheteiwy, M.
, 
Shen, H.
, 
Xu, J.
, 
Guan, Y.
, 
Song, W.
, &amp; 
Hu, J.
 (2017). Seed polyamines metabolism induced by seed priming with spermidine and 5‐aminolevulinic acid for chilling tolerance improvement in rice (<em>Oryza sativa</em> L.) seedlings. Environmental and Experimental Botany, 137, 58–72. 10.1016/j.envexpbot.2017.02.007
</cite> [<a href="https://doi.org/10.1016/j.envexpbot.2017.02.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20and%20Experimental%20Botany&amp;title=Seed%20polyamines%20metabolism%20induced%20by%20seed%20priming%20with%20spermidine%20and%205%E2%80%90aminolevulinic%20acid%20for%20chilling%20tolerance%20improvement%20in%20rice%20(Oryza%20sativa%20L.)%20seedlings&amp;author=M.%20Sheteiwy&amp;author=H.%20Shen&amp;author=J.%20Xu&amp;author=Y.%20Guan&amp;author=W.%20Song&amp;volume=137&amp;publication_year=2017&amp;pages=58-72&amp;doi=10.1016/j.envexpbot.2017.02.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0068">
<cite>

Sidhu, H. S.
 (2020). Fusarium head blight in Canadian winter wheat: Utilizing association mapping and genomic selection for resistance breeding. Department of Plant Agriculture. University of Guelph.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Fusarium%20head%20blight%20in%20Canadian%20winter%20wheat:%20Utilizing%20association%20mapping%20and%20genomic%20selection%20for%20resistance%20breeding&amp;author=H.%20S.%20Sidhu&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0069">
<cite>

Sieber, A. N.
, 
Longin, C. F.
, 
Leiser, W. L.
, &amp; 
Wurschum, T.
 (2016). Copy number variation of <em>CBF‐A14</em> at the <em>Fr‐A2</em> locus determines frost tolerance in winter durum wheat. Theoretical and Applied Genetics, 129, 1087–1097. 10.1007/s00122-016-2685-3

</cite> [<a href="https://doi.org/10.1007/s00122-016-2685-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26883046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Copy%20number%20variation%20of%20CBF%E2%80%90A14%20at%20the%20Fr%E2%80%90A2%20locus%20determines%20frost%20tolerance%20in%20winter%20durum%20wheat&amp;author=A.%20N.%20Sieber&amp;author=C.%20F.%20Longin&amp;author=W.%20L.%20Leiser&amp;author=T.%20Wurschum&amp;volume=129&amp;publication_year=2016&amp;pages=1087-1097&amp;pmid=26883046&amp;doi=10.1007/s00122-016-2685-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0070">
<cite>

Slaughter, A.
, 
Daniel, X.
, 
Flors, V.
, 
Luna, E.
, 
Hohn, B.
, &amp; 
Mauch‐Mani, B.
 (2012). Descendants of primed Arabidopsis plants exhibit resistance to biotic stress. Plant Physiology, 158, 835–843. 10.1104/pp.111.191593

</cite> [<a href="https://doi.org/10.1104/pp.111.191593" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3271771/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22209872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Descendants%20of%20primed%20Arabidopsis%20plants%20exhibit%20resistance%20to%20biotic%20stress&amp;author=A.%20Slaughter&amp;author=X.%20Daniel&amp;author=V.%20Flors&amp;author=E.%20Luna&amp;author=B.%20Hohn&amp;volume=158&amp;publication_year=2012&amp;pages=835-843&amp;pmid=22209872&amp;doi=10.1104/pp.111.191593&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0071">
<cite>

Sofalian, O.
, 
Mohammadi, S. A.
, 
Aharizad, S.
, 
Moghaddam, M.
, &amp; 
Shakiba, M. R.
 (2008). Mapping of QTLs for frost tolerance and heading time using SSR markers in bread wheat. African Journal of Biotechnology, 9, 5260–5264.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=African%20Journal%20of%20Biotechnology&amp;title=Mapping%20of%20QTLs%20for%20frost%20tolerance%20and%20heading%20time%20using%20SSR%20markers%20in%20bread%20wheat&amp;author=O.%20Sofalian&amp;author=S.%20A.%20Mohammadi&amp;author=S.%20Aharizad&amp;author=M.%20Moghaddam&amp;author=M.%20R.%20Shakiba&amp;volume=9&amp;publication_year=2008&amp;pages=5260-5264&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0072">
<cite>

Soleimani, B.
, 
Lehnert, H.
, 
Babben, S.
, 
Keilwagen, J.
, 
Koch, M.
, 
Arana‐Ceballos, F. A.
, 
Chesnokov, Y.
, 
Pshenichnikova, T.
, 
Schondelmaier, J.
, 
Ordon, F.
, 
Borner, A.
, &amp; 
Perovic, D.
 (2022). Genome wide association study of frost tolerance in wheat. Scientific Reports, 12, 5275. 10.1038/s41598-022-08706-y

</cite> [<a href="https://doi.org/10.1038/s41598-022-08706-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8960795/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35347161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Genome%20wide%20association%20study%20of%20frost%20tolerance%20in%20wheat&amp;author=B.%20Soleimani&amp;author=H.%20Lehnert&amp;author=S.%20Babben&amp;author=J.%20Keilwagen&amp;author=M.%20Koch&amp;volume=12&amp;publication_year=2022&amp;pages=5275&amp;pmid=35347161&amp;doi=10.1038/s41598-022-08706-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0073">
<cite>

Song, C.
, 
Cao, Y.
, 
Dai, J.
, 
Li, G.
, 
Manzoor, M. A.
, 
Chen, C.
, &amp; 
Deng, H.
 (2022). The multifaceted roles of MYC2 in plants: Toward transcriptional reprogramming and stress tolerance by Jasmonate Signaling. Frontiers in Plant Science, 13, 868874. 10.3389/fpls.2022.868874

</cite> [<a href="https://doi.org/10.3389/fpls.2022.868874" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9082941/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35548315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=The%20multifaceted%20roles%20of%20MYC2%20in%20plants:%20Toward%20transcriptional%20reprogramming%20and%20stress%20tolerance%20by%20Jasmonate%20Signaling&amp;author=C.%20Song&amp;author=Y.%20Cao&amp;author=J.%20Dai&amp;author=G.%20Li&amp;author=M.%20A.%20Manzoor&amp;volume=13&amp;publication_year=2022&amp;pages=868874&amp;pmid=35548315&amp;doi=10.3389/fpls.2022.868874&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0074">
<cite>
Statistics Canada
. (1997). March intentions of principal field crop areas. Government of Canada.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=March%20intentions%20of%20principal%20field%20crop%20areas&amp;publication_year=1997&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0075">
<cite>
Statistics Canada
. (2007). Situation report. Government of Canada.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Situation%20report&amp;publication_year=2007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0076">
<cite>
Statistics Canada
. (2019). Production of principal field crops. Government of Canada.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Production%20of%20principal%20field%20crops&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0077">
<cite>

Sun, Y. L.
, &amp; 
Hong, S. K.
 (2013). Sensitivity of translation initiation factor <em>eIF1</em> as a molecular target of salt toxicity to sodic‐alkaline stress in the halophytic grass <em>Leymus chinensis</em>
. Biochemical Genetics, 51, 101–118. 10.1007/s10528-012-9546-9

</cite> [<a href="https://doi.org/10.1007/s10528-012-9546-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23112090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemical%20Genetics&amp;title=Sensitivity%20of%20translation%20initiation%20factor%20eIF1%20as%20a%20molecular%20target%20of%20salt%20toxicity%20to%20sodic%E2%80%90alkaline%20stress%20in%20the%20halophytic%20grass%20Leymus%20chinensis&amp;author=Y.%20L.%20Sun&amp;author=S.%20K.%20Hong&amp;volume=51&amp;publication_year=2013&amp;pages=101-118&amp;pmid=23112090&amp;doi=10.1007/s10528-012-9546-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0078">
<cite>

Sutka, J.
, &amp; 
Snape, J. W.
 (1989). Location of a gene for frost resistance on chromosome 5A of wheat. Euphytica, 42, 41–44. 10.1007/BF00042613
</cite> [<a href="https://doi.org/10.1007/BF00042613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Location%20of%20a%20gene%20for%20frost%20resistance%20on%20chromosome%205A%20of%20wheat&amp;author=J.%20Sutka&amp;author=J.%20W.%20Snape&amp;volume=42&amp;publication_year=1989&amp;pages=41-44&amp;doi=10.1007/BF00042613&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0079">
<cite>

Suzuki, N.
, 
Miller, G.
, 
Salazar, C.
, 
Mondal, H. A.
, 
Shulaev, E.
, 
Cortes, D. F.
, 
Shuman, J. L.
, 
Luo, X.
, 
Shah, J.
, 
Schlauch, K.
, 
Shulaev, V.
, &amp; 
Mittler, R.
 (2013). Temporal‐spatial interaction between reactive oxygen species and abscisic acid regulates rapid systemic acclimation in plants. Plant Cell, 25, 3553–3569. 10.1105/tpc.113.114595

</cite> [<a href="https://doi.org/10.1105/tpc.113.114595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3809549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24038652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Temporal%E2%80%90spatial%20interaction%20between%20reactive%20oxygen%20species%20and%20abscisic%20acid%20regulates%20rapid%20systemic%20acclimation%20in%20plants&amp;author=N.%20Suzuki&amp;author=G.%20Miller&amp;author=C.%20Salazar&amp;author=H.%20A.%20Mondal&amp;author=E.%20Shulaev&amp;volume=25&amp;publication_year=2013&amp;pages=3553-3569&amp;pmid=24038652&amp;doi=10.1105/tpc.113.114595&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0080">
<cite>

Tibbs Cortes, L.
, 
Zhang, Z.
, &amp; 
Yu, J.
 (2021). Status and prospects of genome‐wide association studies in plants. The Plant Genome, 14, e20077. 10.1002/tpg2.20077

</cite> [<a href="https://doi.org/10.1002/tpg2.20077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33442955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Status%20and%20prospects%20of%20genome%E2%80%90wide%20association%20studies%20in%20plants&amp;author=L.%20Tibbs%20Cortes&amp;author=Z.%20Zhang&amp;author=J.%20Yu&amp;volume=14&amp;publication_year=2021&amp;pages=e20077&amp;pmid=33442955&amp;doi=10.1002/tpg2.20077&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0081">
<cite>

Ton, J.
, 
Jakab, G.
, 
Toquin, V.
, 
Flors, V.
, 
Iavicoli, A.
, 
Maeder, M. N.
, 
Metraux, J. P.
, &amp; 
Mauch‐Mani, B.
 (2005). Dissecting the beta‐aminobutyric acid‐induced priming phenomenon in Arabidopsis. Plant Cell, 17, 987–999. 10.1105/tpc.104.029728

</cite> [<a href="https://doi.org/10.1105/tpc.104.029728" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1069713/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15722464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Dissecting%20the%20beta%E2%80%90aminobutyric%20acid%E2%80%90induced%20priming%20phenomenon%20in%20Arabidopsis&amp;author=J.%20Ton&amp;author=G.%20Jakab&amp;author=V.%20Toquin&amp;author=V.%20Flors&amp;author=A.%20Iavicoli&amp;volume=17&amp;publication_year=2005&amp;pages=987-999&amp;pmid=15722464&amp;doi=10.1105/tpc.104.029728&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0082">
<cite>

Torkamaneh, D.
, &amp; 
Belzile, F.
 (2015). Scanning and filling: Ultra‐dense SNP genotyping combining genotyping‐by‐sequencing, SNP array and whole‐genome resequencing data. PLoS One, 10, e0131533. 10.1371/journal.pone.0131533

</cite> [<a href="https://doi.org/10.1371/journal.pone.0131533" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4498655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26161900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Scanning%20and%20filling:%20Ultra%E2%80%90dense%20SNP%20genotyping%20combining%20genotyping%E2%80%90by%E2%80%90sequencing,%20SNP%20array%20and%20whole%E2%80%90genome%20resequencing%20data&amp;author=D.%20Torkamaneh&amp;author=F.%20Belzile&amp;volume=10&amp;publication_year=2015&amp;pages=e0131533&amp;pmid=26161900&amp;doi=10.1371/journal.pone.0131533&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0083">
<cite>

Torkamaneh, D.
, 
Boyle, B.
, &amp; 
Belzile, F.
 (2018). Efficient genome‐wide genotyping strategies and data integration in crop plants. Theoretical and Applied Genetics, 131, 499–511. 10.1007/s00122-018-3056-z

</cite> [<a href="https://doi.org/10.1007/s00122-018-3056-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29352324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Efficient%20genome%E2%80%90wide%20genotyping%20strategies%20and%20data%20integration%20in%20crop%20plants&amp;author=D.%20Torkamaneh&amp;author=B.%20Boyle&amp;author=F.%20Belzile&amp;volume=131&amp;publication_year=2018&amp;pages=499-511&amp;pmid=29352324&amp;doi=10.1007/s00122-018-3056-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0084">
<cite>

Torkamaneh, D.
, 
Laroche, J.
, 
Bastien, M.
, 
Abed, A.
, &amp; 
Belzile, F.
 (2017). Fast‐GBS: A new pipeline for the efficient and highly accurate calling of SNPs from genotyping‐by‐sequencing data. BMC Bioinformatics, 18, 5. 10.1186/s12859-016-1431-9

</cite> [<a href="https://doi.org/10.1186/s12859-016-1431-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5210301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28049422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Fast%E2%80%90GBS:%20A%20new%20pipeline%20for%20the%20efficient%20and%20highly%20accurate%20calling%20of%20SNPs%20from%20genotyping%E2%80%90by%E2%80%90sequencing%20data&amp;author=D.%20Torkamaneh&amp;author=J.%20Laroche&amp;author=M.%20Bastien&amp;author=A.%20Abed&amp;author=F.%20Belzile&amp;volume=18&amp;publication_year=2017&amp;pages=5&amp;pmid=28049422&amp;doi=10.1186/s12859-016-1431-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0085">
<cite>

Toth, J.
, 
Pandurangan, S.
, 
Burt, A.
, 
Mitchell Fetch, J.
, 
Kumar, S.
, &amp; 
Willenborg, C.
 (2019). Marker‐assisted breeding of hexaploid spring wheat in the Canadian prairies. Canadian Journal of Plant Science, 99, 111–127. 10.1139/cjps-2018-0183
</cite> [<a href="https://doi.org/10.1139/cjps-2018-0183" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Canadian%20Journal%20of%20Plant%20Science&amp;title=Marker%E2%80%90assisted%20breeding%20of%20hexaploid%20spring%20wheat%20in%20the%20Canadian%20prairies&amp;author=J.%20Toth&amp;author=S.%20Pandurangan&amp;author=A.%20Burt&amp;author=J.%20Mitchell%20Fetch&amp;author=S.%20Kumar&amp;volume=99&amp;publication_year=2019&amp;pages=111-127&amp;doi=10.1139/cjps-2018-0183&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0086">
<cite>

Vagujfalvi, A.
, 
Aprile, A.
, 
Miller, A.
, 
Dubcovsky, J.
, 
Delugu, G.
, 
Galiba, G.
, &amp; 
Cattivelli, L.
 (2005). The expression of several <em>Cbf</em> genes at the <em>Fr‐A2</em> locus is linked to frost resistance in wheat. Molecular Genetics and Genomics, 274, 506–514. 10.1007/s00438-005-0047-y

</cite> [<a href="https://doi.org/10.1007/s00438-005-0047-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16200412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Genetics%20and%20Genomics&amp;title=The%20expression%20of%20several%20Cbf%20genes%20at%20the%20Fr%E2%80%90A2%20locus%20is%20linked%20to%20frost%20resistance%20in%20wheat&amp;author=A.%20Vagujfalvi&amp;author=A.%20Aprile&amp;author=A.%20Miller&amp;author=J.%20Dubcovsky&amp;author=G.%20Delugu&amp;volume=274&amp;publication_year=2005&amp;pages=506-514&amp;pmid=16200412&amp;doi=10.1007/s00438-005-0047-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0087">
<cite>

Vagujfalvi, A.
, 
Galiba, G.
, 
Cattivelli, L.
, &amp; 
Dubcovsky, J.
 (2003). The cold‐regulated transcriptional activator <em>Cbf3</em> is linked to the frost‐tolerance locus <em>Fr‐A2</em> on wheat chromosome 5A. Molecular Genetics and Genomics, 269, 60–67. 10.1007/s00438-003-0806-6

</cite> [<a href="https://doi.org/10.1007/s00438-003-0806-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4743881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12715154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Genetics%20and%20Genomics&amp;title=The%20cold%E2%80%90regulated%20transcriptional%20activator%20Cbf3%20is%20linked%20to%20the%20frost%E2%80%90tolerance%20locus%20Fr%E2%80%90A2%20on%20wheat%20chromosome%205A&amp;author=A.%20Vagujfalvi&amp;author=G.%20Galiba&amp;author=L.%20Cattivelli&amp;author=J.%20Dubcovsky&amp;volume=269&amp;publication_year=2003&amp;pages=60-67&amp;pmid=12715154&amp;doi=10.1007/s00438-003-0806-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0088">
<cite>

Vaitkeviciute, G.
, 
Chawade, A.
, 
Lillemo, M.
, 
Liatukas, Z.
, 
Aleliunas, A.
, &amp; 
Armoniene, R.
 (2023). Genome‐wide association analysis of freezing tolerance and winter hardiness in winter wheat of Nordic origin. Plants, 12, 4014.
</cite> [<a href="https://doi.org/10.3390/plants12234014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10708462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38068649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Genome%E2%80%90wide%20association%20analysis%20of%20freezing%20tolerance%20and%20winter%20hardiness%20in%20winter%20wheat%20of%20Nordic%20origin&amp;author=G.%20Vaitkeviciute&amp;author=A.%20Chawade&amp;author=M.%20Lillemo&amp;author=Z.%20Liatukas&amp;author=A.%20Aleliunas&amp;volume=12&amp;publication_year=2023&amp;pages=4014&amp;pmid=38068649&amp;doi=10.3390/plants12234014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0089">
<cite>

Wang, J.
, &amp; 
Zhang, Z.
 (2021). GAPIT version 3: Boosting power and accuracy for genomic association and prediction. Genomics, Proteomics &amp; Bioinformatics, 19, 629–640. 10.1016/j.gpb.2021.08.005
</cite> [<a href="https://doi.org/10.1016/j.gpb.2021.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9121400/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34492338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics,%20Proteomics%20&amp;%20Bioinformatics&amp;title=GAPIT%20version%203:%20Boosting%20power%20and%20accuracy%20for%20genomic%20association%20and%20prediction&amp;author=J.%20Wang&amp;author=Z.%20Zhang&amp;volume=19&amp;publication_year=2021&amp;pages=629-640&amp;pmid=34492338&amp;doi=10.1016/j.gpb.2021.08.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0090">
<cite>

Wang, M.
, 
Yan, J.
, 
Zhao, J.
, 
Song, W.
, 
Zhang, X.
, 
Xiao, Y.
, &amp; 
Zheng, Y.
 (2012). Genome‐wide association study (GWAS) of resistance to head smut in maize. Plant Science, 196, 125–131. 10.1016/j.plantsci.2012.08.004

</cite> [<a href="https://doi.org/10.1016/j.plantsci.2012.08.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23017907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Science&amp;title=Genome%E2%80%90wide%20association%20study%20(GWAS)%20of%20resistance%20to%20head%20smut%20in%20maize&amp;author=M.%20Wang&amp;author=J.%20Yan&amp;author=J.%20Zhao&amp;author=W.%20Song&amp;author=X.%20Zhang&amp;volume=196&amp;publication_year=2012&amp;pages=125-131&amp;pmid=23017907&amp;doi=10.1016/j.plantsci.2012.08.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0091">
<cite>

Wang, R.
, 
Yu, M.
, 
Xia, J.
, 
Xing, J.
, 
Fan, X.
, 
Xu, Q.
, 
Cang, J.
, &amp; 
Zhang, D.
 (2022). Overexpression of <em>TaMYC2</em> confers freeze tolerance by ICE‐CBF‐COR module in <em>Arabidopsis thaliana</em>
. Frontiers in Plant Science, 13, 1042889. 10.3389/fpls.2022.1042889

</cite> [<a href="https://doi.org/10.3389/fpls.2022.1042889" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9710523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36466238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Overexpression%20of%20TaMYC2%20confers%20freeze%20tolerance%20by%20ICE%E2%80%90CBF%E2%80%90COR%20module%20in%20Arabidopsis%20thaliana&amp;author=R.%20Wang&amp;author=M.%20Yu&amp;author=J.%20Xia&amp;author=J.%20Xing&amp;author=X.%20Fan&amp;volume=13&amp;publication_year=2022&amp;pages=1042889&amp;pmid=36466238&amp;doi=10.3389/fpls.2022.1042889&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0092">
<cite>

Watson, A.
, 
Hickey, L. T.
, 
Christopher, J.
, 
Rutkoski, J.
, 
Poland, J.
, &amp; 
Hayes, B. J.
 (2019). Multivariate genomic selection and potential of rapid indirect selection with speed breeding in spring wheat. Crop Science, 59, 1945–1959. 10.2135/cropsci2018.12.0757
</cite> [<a href="https://doi.org/10.2135/cropsci2018.12.0757" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Multivariate%20genomic%20selection%20and%20potential%20of%20rapid%20indirect%20selection%20with%20speed%20breeding%20in%20spring%20wheat&amp;author=A.%20Watson&amp;author=L.%20T.%20Hickey&amp;author=J.%20Christopher&amp;author=J.%20Rutkoski&amp;author=J.%20Poland&amp;volume=59&amp;publication_year=2019&amp;pages=1945-1959&amp;doi=10.2135/cropsci2018.12.0757&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0093">
<cite>

Willick, I. R.
, 
Tanino, K. K.
, &amp; 
Gusta, L. V.
 (2020). The impact of global climate change on the freezing tolerance of winter cereals in Western Canada. Journal of Agronomy and Crop Science, 207, 88–99. 10.1111/jac.12447
</cite> [<a href="https://doi.org/10.1111/jac.12447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Agronomy%20and%20Crop%20Science&amp;title=The%20impact%20of%20global%20climate%20change%20on%20the%20freezing%20tolerance%20of%20winter%20cereals%20in%20Western%20Canada&amp;author=I.%20R.%20Willick&amp;author=K.%20K.%20Tanino&amp;author=L.%20V.%20Gusta&amp;volume=207&amp;publication_year=2020&amp;pages=88-99&amp;doi=10.1111/jac.12447&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0094">
<cite>

Wurschum, T.
, 
Boeven, P. H.
, 
Langer, S. M.
, 
Longin, C. F.
, &amp; 
Leiser, W. L.
 (2015). Multiply to conquer: Copy number variations at <em>Ppd‐B1</em> and <em>Vrn‐A1</em> facilitate global adaptation in wheat. BMC Genetics, 16, 96. 10.1186/s12863-015-0258-0

</cite> [<a href="https://doi.org/10.1186/s12863-015-0258-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4518651/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26219856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genetics&amp;title=Multiply%20to%20conquer:%20Copy%20number%20variations%20at%20Ppd%E2%80%90B1%20and%20Vrn%E2%80%90A1%20facilitate%20global%20adaptation%20in%20wheat&amp;author=T.%20Wurschum&amp;author=P.%20H.%20Boeven&amp;author=S.%20M.%20Langer&amp;author=C.%20F.%20Longin&amp;author=W.%20L.%20Leiser&amp;volume=16&amp;publication_year=2015&amp;pages=96&amp;pmid=26219856&amp;doi=10.1186/s12863-015-0258-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0095">
<cite>

Xin, W.
, 
Zhang, L.
, 
Gao, J.
, 
Zhang, W.
, 
Yi, J.
, 
Zhen, X.
, 
Bi, C.
, 
He, D.
, 
Liu, S.
, &amp; 
Zhao, X.
 (2021). Adaptation mechanism of roots to low and high nitrogen revealed by proteomic analysis. Rice, 14, 5. 10.1186/s12284-020-00443-y

</cite> [<a href="https://doi.org/10.1186/s12284-020-00443-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7790981/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33411084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rice&amp;title=Adaptation%20mechanism%20of%20roots%20to%20low%20and%20high%20nitrogen%20revealed%20by%20proteomic%20analysis&amp;author=W.%20Xin&amp;author=L.%20Zhang&amp;author=J.%20Gao&amp;author=W.%20Zhang&amp;author=J.%20Yi&amp;volume=14&amp;publication_year=2021&amp;pages=5&amp;pmid=33411084&amp;doi=10.1186/s12284-020-00443-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0096">
<cite>

Xu, X.
, 
Zhang, L.
, 
Zhao, W.
, 
Fu, L.
, 
Han, Y.
, 
Wang, K.
, 
Yan, L.
, 
Li, Y.
, 
Zhang, X. H.
, &amp; 
Min, D. H.
 (2021). Genome‐wide analysis of the serine carboxypeptidase‐like protein family in <em>Triticum aestivum</em> reveals <em>TaSCPL184‐6D</em> is involved in abiotic stress response. BMC Genomics, 22, 350. 10.1186/s12864-021-07647-6

</cite> [<a href="https://doi.org/10.1186/s12864-021-07647-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8126144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33992092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Genome%E2%80%90wide%20analysis%20of%20the%20serine%20carboxypeptidase%E2%80%90like%20protein%20family%20in%20Triticum%20aestivum%20reveals%20TaSCPL184%E2%80%906D%20is%20involved%20in%20abiotic%20stress%20response&amp;author=X.%20Xu&amp;author=L.%20Zhang&amp;author=W.%20Zhao&amp;author=L.%20Fu&amp;author=Y.%20Han&amp;volume=22&amp;publication_year=2021&amp;pages=350&amp;pmid=33992092&amp;doi=10.1186/s12864-021-07647-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0097">
<cite>

Yang, G.
, 
Yu, L.
, 
Wang, Y.
, 
Wang, C.
, &amp; 
Gao, C.
 (2017). The translation initiation factor 1A (<em>TheIF1A</em>) from Tamarix hispida is regulated by a Dof transcription factor and increased abiotic stress tolerance. Frontiers in Plant Science, 8, 513.
</cite> [<a href="https://doi.org/10.3389/fpls.2017.00513" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5383729/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28439284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=The%20translation%20initiation%20factor%201A%20(TheIF1A)%20from%20Tamarix%20hispida%20is%20regulated%20by%20a%20Dof%20transcription%20factor%20and%20increased%20abiotic%20stress%20tolerance&amp;author=G.%20Yang&amp;author=L.%20Yu&amp;author=Y.%20Wang&amp;author=C.%20Wang&amp;author=C.%20Gao&amp;volume=8&amp;publication_year=2017&amp;pages=513&amp;pmid=28439284&amp;doi=10.3389/fpls.2017.00513&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0098">
<cite>

Zandalinas, S. I.
, 
Sengupta, S.
, 
Burks, D.
, 
Azad, R. K.
, &amp; 
Mittler, R.
 (2019). Identification and characterization of a core set of ROS wave‐associated transcripts involved in the systemic acquired acclimation response of Arabidopsis to excess light. The Plant Journal, 98, 126–141. 10.1111/tpj.14205

</cite> [<a href="https://doi.org/10.1111/tpj.14205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6850305/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30556340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=Identification%20and%20characterization%20of%20a%20core%20set%20of%20ROS%20wave%E2%80%90associated%20transcripts%20involved%20in%20the%20systemic%20acquired%20acclimation%20response%20of%20Arabidopsis%20to%20excess%20light&amp;author=S.%20I.%20Zandalinas&amp;author=S.%20Sengupta&amp;author=D.%20Burks&amp;author=R.%20K.%20Azad&amp;author=R.%20Mittler&amp;volume=98&amp;publication_year=2019&amp;pages=126-141&amp;pmid=30556340&amp;doi=10.1111/tpj.14205&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0099">
<cite>

Zhang, C.
, 
Dong, S. S.
, 
Xu, J. Y.
, 
He, W. M.
, &amp; 
Yang, T. L.
 (2019). PopLDdecay: A fast and effective tool for linkage disequilibrium decay analysis based on variant call format files. Bioinformatics, 35, 1786–1788. 10.1093/bioinformatics/bty875

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty875" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30321304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PopLDdecay:%20A%20fast%20and%20effective%20tool%20for%20linkage%20disequilibrium%20decay%20analysis%20based%20on%20variant%20call%20format%20files&amp;author=C.%20Zhang&amp;author=S.%20S.%20Dong&amp;author=J.%20Y.%20Xu&amp;author=W.%20M.%20He&amp;author=T.%20L.%20Yang&amp;volume=35&amp;publication_year=2019&amp;pages=1786-1788&amp;pmid=30321304&amp;doi=10.1093/bioinformatics/bty875&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0100">
<cite>

Zhang, T.
, 
He, Y.
, 
DePauw, R.
, 
Jin, Z.
, 
Garvin, D.
, 
Yue, X.
, 
Anderson, W.
, 
Li, T.
, 
Dong, X.
, 
Zhang, T.
, &amp; 
Yang, X.
 (2022). Climate change may outpace current wheat breeding yield improvements in North America. Nature Communications, 13, 5591. 10.1038/s41467-022-33265-1
</cite> [<a href="https://doi.org/10.1038/s41467-022-33265-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9525655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36180462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Climate%20change%20may%20outpace%20current%20wheat%20breeding%20yield%20improvements%20in%20North%20America&amp;author=T.%20Zhang&amp;author=Y.%20He&amp;author=R.%20DePauw&amp;author=Z.%20Jin&amp;author=D.%20Garvin&amp;volume=13&amp;publication_year=2022&amp;pages=5591&amp;pmid=36180462&amp;doi=10.1038/s41467-022-33265-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0102">
<cite>

Zhao, Y.
, 
Gowda, M.
, 
Wurschum, T.
, 
Longin, C. F.
, 
Korzun, V.
, 
Kollers, S.
, 
Schachschneider, R.
, 
Zeng, J.
, 
Fernando, R.
, 
Dubcovsky, J.
, &amp; 
Reif, J. C.
 (2013). Dissecting the genetic architecture of frost tolerance in Central European winter wheat. Journal of Experimental Botany, 64, 4453–4460. 10.1093/jxb/ert259

</cite> [<a href="https://doi.org/10.1093/jxb/ert259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3808325/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24006418/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Dissecting%20the%20genetic%20architecture%20of%20frost%20tolerance%20in%20Central%20European%20winter%20wheat&amp;author=Y.%20Zhao&amp;author=M.%20Gowda&amp;author=T.%20Wurschum&amp;author=C.%20F.%20Longin&amp;author=V.%20Korzun&amp;volume=64&amp;publication_year=2013&amp;pages=4453-4460&amp;pmid=24006418&amp;doi=10.1093/jxb/ert259&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0103">
<cite>

Zhao, Y.
, 
Li, J.
, 
Zhao, R.
, 
Xu, K.
, 
Xiao, Y.
, 
Zhang, S.
, 
Tian, J.
, &amp; 
Yang, X.
 (2020). Genome‐wide association study reveals the genetic basis of cold tolerance in wheat. Molecular Breeding, 40, 36. 10.1007/s11032-020-01115-x
</cite> [<a href="https://doi.org/10.1007/s11032-020-01115-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Genome%E2%80%90wide%20association%20study%20reveals%20the%20genetic%20basis%20of%20cold%20tolerance%20in%20wheat&amp;author=Y.%20Zhao&amp;author=J.%20Li&amp;author=R.%20Zhao&amp;author=K.%20Xu&amp;author=Y.%20Xiao&amp;volume=40&amp;publication_year=2020&amp;pages=36&amp;doi=10.1007/s11032-020-01115-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270091-bib-0104">
<cite>

Zhu, J.
, 
Pearce, S.
, 
Burke, A.
, 
See, D. R.
, 
Skinner, D. Z.
, 
Dubcovsky, J.
, &amp; 
Garland‐Campbell, K.
 (2014). Copy number and haplotype variation at the <em>VRN‐A1</em> and central <em>FR‐A2</em> loci are associated with frost tolerance in hexaploid wheat. Theoretical and Applied Genetics, 127, 1183–1197. 10.1007/s00122-014-2290-2

</cite> [<a href="https://doi.org/10.1007/s00122-014-2290-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4876961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24626953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Copy%20number%20and%20haplotype%20variation%20at%20the%20VRN%E2%80%90A1%20and%20central%20FR%E2%80%90A2%20loci%20are%20associated%20with%20frost%20tolerance%20in%20hexaploid%20wheat&amp;author=J.%20Zhu&amp;author=S.%20Pearce&amp;author=A.%20Burke&amp;author=D.%20R.%20See&amp;author=D.%20Z.%20Skinner&amp;volume=127&amp;publication_year=2014&amp;pages=1183-1197&amp;pmid=24626953&amp;doi=10.1007/s00122-014-2290-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Supplemental File S1 includes a table of all the genotypes that comprise the CWWDP. The supplemental figures and tables include climatic data for all environments, additional data analysis for winter survival and QTL information that was not required in the main text.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365865/bin/TPG2-18-e70091-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70091-s002.docx</a><sup> (41.7KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>Supplementary Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365865/bin/TPG2-18-e70091-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70091-s001.docx</a><sup> (3.3MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated during and/or analyzed during the current study are available from the corresponding author on reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70091"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70091.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365865/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365865/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365865%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365865/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365865/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365865/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40832815/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365865/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40832815/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365865/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365865/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Z5qISfO2rbtEgoXBSlz6ZZITExyGK3MmMEqQvbthqJj7jyE9JstfVitMxp5pAaE1">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
