
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            NetStart 2.0: prediction of eukaryotic translation initiation sites using a protein language model - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15254BF88AF3A983054BF80046F54728.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcbioi">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366053/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Bioinformatics">
<meta name="citation_title" content="NetStart 2.0: prediction of eukaryotic translation initiation sites using a protein language model">
<meta name="citation_author" content="Line Sandvad Nielsen">
<meta name="citation_author_institution" content="Section for Computational and RNA Biology, Department of Biology, University of Copenhagen, 2200 Copenhagen, Denmark">
<meta name="citation_author" content="Anders Gorm Pedersen">
<meta name="citation_author_institution" content="Section for Bioinformatics, Department of Health Technology, Technical University of Denmark, 2800 Kongens Lyngby, Denmark">
<meta name="citation_author" content="Ole Winther">
<meta name="citation_author_institution" content="Section for Computational and RNA Biology, Department of Biology, University of Copenhagen, 2200 Copenhagen, Denmark">
<meta name="citation_author_institution" content="Section for Cognitive Systems, Department of Applied Mathematics and Computer Science, Technical University of Denmark, 2800 Kongens Lyngby, Denmark">
<meta name="citation_author" content="Henrik Nielsen">
<meta name="citation_author_institution" content="Section for Bioinformatics, Department of Health Technology, Technical University of Denmark, 2800 Kongens Lyngby, Denmark">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="216">
<meta name="citation_doi" content="10.1186/s12859-025-06220-2">
<meta name="citation_pmid" content="40830753">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366053/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366053/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366053/pdf/12859_2025_Article_6220.pdf">
<meta name="description" content="Accurate identification of translation initiation sites is essential for the proper translation of mRNA into functional proteins. In eukaryotes, the choice of the translation initiation site is influenced by multiple factors, including its proximity ...">
<meta name="og:title" content="NetStart 2.0: prediction of eukaryotic translation initiation sites using a protein language model">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Accurate identification of translation initiation sites is essential for the proper translation of mRNA into functional proteins. In eukaryotes, the choice of the translation initiation site is influenced by multiple factors, including its proximity ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366053/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366053">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12859-025-06220-2"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12859_2025_Article_6220.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366053%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366053/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366053/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366053/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcbioi.png" alt="BMC Bioinformatics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Bioinformatics" title="Link to BMC Bioinformatics" shape="default" href="https://bmcbioinformatics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Bioinformatics</button></div>. 2025 Aug 19;26:216. doi: <a href="https://doi.org/10.1186/s12859-025-06220-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12859-025-06220-2</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Bioinformatics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Bioinformatics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Bioinformatics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Bioinformatics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>NetStart 2.0: prediction of eukaryotic translation initiation sites using a protein language model</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nielsen%20LS%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Line Sandvad Nielsen</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Line Sandvad Nielsen</span></h3>
<div class="p">
<sup>1</sup>Section for Computational and RNA Biology, Department of Biology, University of Copenhagen, 2200 Copenhagen, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nielsen%20LS%22%5BAuthor%5D" class="usa-link"><span class="name western">Line Sandvad Nielsen</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pedersen%20AG%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Anders Gorm Pedersen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Anders Gorm Pedersen</span></h3>
<div class="p">
<sup>2</sup>Section for Bioinformatics, Department of Health Technology, Technical University of Denmark, 2800 Kongens Lyngby, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pedersen%20AG%22%5BAuthor%5D" class="usa-link"><span class="name western">Anders Gorm Pedersen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Winther%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Ole Winther</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Ole Winther</span></h3>
<div class="p">
<sup>1</sup>Section for Computational and RNA Biology, Department of Biology, University of Copenhagen, 2200 Copenhagen, Denmark </div>
<div class="p">
<sup>3</sup>Section for Cognitive Systems, Department of Applied Mathematics and Computer Science, Technical University of Denmark, 2800 Kongens Lyngby, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Winther%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Ole Winther</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nielsen%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Henrik Nielsen</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Henrik Nielsen</span></h3>
<div class="p">
<sup>2</sup>Section for Bioinformatics, Department of Health Technology, Technical University of Denmark, 2800 Kongens Lyngby, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nielsen%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Henrik Nielsen</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Section for Computational and RNA Biology, Department of Biology, University of Copenhagen, 2200 Copenhagen, Denmark </div>
<div id="Aff2">
<sup>2</sup>Section for Bioinformatics, Department of Health Technology, Technical University of Denmark, 2800 Kongens Lyngby, Denmark </div>
<div id="Aff3">
<sup>3</sup>Section for Cognitive Systems, Department of Applied Mathematics and Computer Science, Technical University of Denmark, 2800 Kongens Lyngby, Denmark </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 1; Accepted 2025 Jul 4; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366053  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830753/" class="usa-link">40830753</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Accurate identification of translation initiation sites is essential for the proper translation of mRNA into functional proteins. In eukaryotes, the choice of the translation initiation site is influenced by multiple factors, including its proximity to the 5<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/81fec6beadce/d33e193.gif" loading="lazy" id="d33e193" alt="Inline graphic"></span> end and the local start codon context. Translation initiation sites mark the transition from non-coding to coding regions. This fact motivates the expectation that the upstream sequence, if translated, would assemble a nonsensical order of amino acids, while the downstream sequence would correspond to the structured beginning of a protein. This distinction suggests potential for predicting translation initiation sites using a protein language model.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">We present NetStart 2.0, a deep learning-based model that integrates the ESM-2 protein language model with the local sequence context to predict translation initiation sites across a broad range of eukaryotic species. NetStart 2.0 was trained as a single model across multiple species, and despite the broad phylogenetic diversity represented in the training data, it consistently relied on features marking the transition from non-coding to coding regions.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par3">By leveraging “protein-ness”, NetStart 2.0 achieves state-of-the-art performance in predicting translation initiation sites across a diverse range of eukaryotic species. This success underscores the potential of protein language models to bridge transcript- and peptide-level information in complex biological prediction tasks. The NetStart 2.0 webserver is available at: <a href="https://services.healthtech.dtu.dk/services/NetStart-2.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://services.healthtech.dtu.dk/services/NetStart-2.0/</a>.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12859-025-06220-2.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Protein language models, Translation initiation sites, Start codons, Deep learning, “Protein-ness”, Coding potential, Eukaryotic translation initiation</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par16">Eukaryotic translation initiation is a highly regulated process that marks the beginning of protein synthesis. Thousands of proteins that possess important structural, catalytic, and regulatory roles are encoded in eukaryotic genomes, and the identification of the right translation initiation site (TIS), defined as the codon from which translation is initiated, is an important task for ensuring proper translation of mRNAs. For most eukaryotic mRNAs, this process is accomplished by the widely accepted “scanning mechanism”, which was first proposed by Marilyn Kozak in 1978 [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. This mechanism describes how the 40S ribosomal subunit scans along the 5<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/81fec6beadce/d33e300.gif" loading="lazy" id="d33e300" alt="Inline graphic"></span> leader of the mRNA, base by base, until it encounters a start codon in a favorable context for initiating translation [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. In vertebrates, the preferred context flanking the TIS is commonly known as the Kozak sequence, denoted as GCCRCC<em>AUG</em>G (where R represents a purine and <em>AUG</em> is the initiating codon) [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. In particular, the presence of a purine three nucleotides upstream, and a guanine immediately downstream of the start codon has been shown to strongly influence the TIS selection in vertebrates, while the importance of the remaining positions in the Kozak sequence appears to be more variable [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. Expanding beyond vertebrate context, studies of phylogenetically diverse eukaryotic transcripts have shown substantial variation in initiation signals among different eukaryotic groups, suggesting that the preferred initiation context roughly reflects the evolutionary relationships among species [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>].</p>
<p id="Par17">The importance of the local context for TIS selection is evident in the event of leaky scanning, where an AUG codon in a weak context is bypassed by the 40S ribosomal subunit, leading to translation initiation at a downstream start codon [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Notably, approximately 40% of eukaryotic mRNAs in GenBank [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] contain at least one AUG upstream of the annotated main open reading frame (mORF) [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. With the advent of ribosome profiling techniques, recent studies have shown that short ORFs (sORFs) with start codons located in the 5<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/81fec6beadce/d33e368.gif" loading="lazy" id="d33e368" alt="Inline graphic"></span> untranslated region (UTR) are very prevalent [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>], appearing in approximately 64% of human mRNAs and 54% of <em>Arabidopsis</em> mRNAs [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. These upstream ORFs (uORFs) generally play regulatory roles by influencing the translation of downstream mORFs, either through ribosome sequestering or competition, rather than by encoding functional proteins [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Zhang et al. [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>], for instance, found that the start codon contexts of uORFs tend to deviate more from the Kozak consensus than those of mORFs, based on data from 478 phylogenetically diverse eukaryotic species. All of the above findings highlight that the identification of mORF TISs is complex and non-trivial.</p>
<p id="Par18">Integrating detailed knowledge of translation initiation with advanced machine learning enables a wide range of biologically important tasks, including the discovery of novel proteins and alternative TISs, genome and transcriptome annotation, and deeper insight into protein synthesis, RNA coding potential, and the impact of nucleotide mutations on protein products [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>–<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Over time, various computational methods for TIS prediction in eukaryotes have been suggested, evolving from simple neural networks such as NetStart 1.0, which was developed in 1997 [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>], to a range of more complex frameworks [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. In particular, deep learning models have become popular because of their automated feature learning when trained on large, credible datasets [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. An example is the model TIS Transformer, developed by Clauwaert et al. [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>], which is trained from scratch on the human transcriptome and uses the transformer architecture with self-attention to predict multiple TIS locations in transcripts, including those of sORFs and within long non-coding RNAs. In addition to transcript-level TIS predictors, several gene prediction tools incorporate TIS prediction as part of their pipelines. A well-established example is AUGUSTUS, which employs a fourth-order interpolated generalized hidden Markov model to classify sequence features, including exons, introns, splice sites and TISs. AUGUSTUS is trained to predict alternative splice sites and has a broad range of species-specific models available [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. Recently, deep learning models such as Tiberius [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] have further refined the accuracy of eukaryotic gene prediction. Tiberius integrates convolutional and long short-term memory layers with a differentiable HMM layer, predicting probabilities for 15 gene structure classes, including the initial CDS (Coding Sequence) where the TIS is located [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Tiberius is trained on data from 34 mammalian genomes, and does not predict alternative splice forms [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>].</p>
<p id="Par19">The advent of nucleotide and protein language models has significantly enhanced capabilities in biological sequence modeling. These models learn grammatical and semantic relationships between tokens by learning patterns in the training data, enabling them to assign probabilities to previously unseen tokens [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. This ability is particularly effective for understanding the contextual dependencies inherent in DNA, RNA, and protein sequences [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>, <a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. A profound advancement in sequence analysis has been the introduction of the transformer architecture, which employs a self-attention mechanism to efficiently capture long-range dependencies across entire sequences [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. Given the extensive amounts of unlabeled biological sequence data available, pretraining language models in a self-supervised setting allows them to learn the ‘language’ of biological sequences [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. In this setup, models often predict the identities of randomly masked tokens in a sequence on the basis of their surrounding context, as seen in protein language models such as ProtT5 [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>] and ESM-2 [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Following pretraining, these models can be fine-tuned on smaller labeled datasets for specific downstream tasks, leveraging their understanding of general sequence patterns to enhance both task-specific performance and computational efficiency [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>].</p>
<p id="Par20">In this paper, we introduce NetStart 2.0, a novel deep learning-based model designed to predict TISs of protein-coding ORFs in eukaryotic transcripts. NetStart 2.0 takes as input a transcript sequence and the corresponding species name. Its main objective is to accurately identify the correct mORF TIS within transcripts containing several ATG codons. As part of the modeling framework, NetStart 2.0 leverages peptide-level information for its nucleotide-level predictions, using the pretrained protein language model ESM-2 [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>] to encode translated transcript sequences. With this approach, NetStart 2.0 integrates protein context with nucleotide-level features that capture the local start codon context, across sequences from 60 phylogenetically diverse eukaryotic species. After optimizing and defining the model architecture, we benchmarked NetStart 2.0 against state-of-the-art methods in TIS prediction. Our results highlight the potential of incorporating peptide-level context into transcript-level prediction tasks, indicating a promising direction for future research.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Implementation</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Dataset creation</h3>
<p id="Par21">The raw datasets consisted of RefSeq-assembled genomes and corresponding annotation data from NCBI’s Eukaryotic Genome Annotation Pipeline Database, which was collected for 60 diverse eukaryotic species (Supplementary Table <a href="#MOESM1" class="usa-link">A1</a>) [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>].</p>
<p id="Par22">For the positive-labeled part of the dataset (hereafter referred to as the <em>TIS-labeled dataset</em>) we extracted all mRNA transcripts from nuclear genes with an annotated TIS ATG, labeling the position of the A in the translation initiating ATG. The sequences from the assembled genomes were processed by splicing out introns as defined by the annotated exons, and locating the TIS as defined by the beginning of the first CDS annotation (Fig. <a href="#Fig1" class="usa-link">1</a>A). We removed all poorly annotated mRNA sequences that did not fulfill the following criteria: (1) the CDS had a stop codon (TAG, TAA or TGA) as the last codon, (2) the CDS did not have an in-frame stop codon, (3) the CDS had a complete number of codon triplets, and (4) the CDS contained only known nucleotides (A, T, G, C).. Kin\\\</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366053_12859_2025_6220_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/c95012b7404e/12859_2025_6220_Fig1_HTML.jpg" loading="lazy" id="MO1" height="834" width="704" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>High-level visualization of the dataset creation approach. The red marks illustrate non-TIS ATGs, and the green marks illustrate TIS ATGs</p></figcaption></figure><p id="Par23">The negative-labeled part of the dataset (hereafter referred to as the <em>non-TIS labeled dataset</em>) consisted of intergenic sequences, intron sequences, and sequences from mRNA transcripts, where a non-TIS ATG was labeled (Fig. <a href="#Fig1" class="usa-link">1</a>B–D). For each non-TIS labeled sequence, we randomly picked out an ATG, labeled it, and extracted a subsequence of 500 nucleotides upstream and downstream of it. We extracted approximately an equal number of intron and intergenic samples compared to the TIS-labeled sequences of each species, sampling randomly to get sequences spread across the genome. Distinct proteoforms of the same gene can have different TISs, which we kept track of, and then located all ATGs upstream and downstream, respectively. We extracted all non-TIS ATGs located upstream of the first annotated TIS where the 5<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/81fec6beadce/d33e514.gif" loading="lazy" id="d33e514" alt="Inline graphic"></span> UTR was known, and randomly extracted three non-TIS ATGs downstream of the last annotated TIS. Pilot studies showed that the model had the most difficulty classifying downstream ATGs in the same reading frame as the TIS ATG. To better represent these challenging cases, we extracted three non-TIS ATGs downstream of the last annotated TIS: two in the same reading frame as the TIS ATG and one in an alternative reading frame.</p>
<p id="Par24">When available, we extracted annotations sourced from RefSeq. In cases where RefSeq annotations were not available, we collected annotations from Gnomon, which are based on a combination of homology searching and <em>ab initio</em> modeling [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. The reason for including Gnomon annotations was to increase the range of species covered in our training data.</p>
<p id="Par25">Due to the intrinsic similarities found in biological sequences, our dataset contains several highly similar entries, including genes belonging to the same family, mRNA splice variants of the same gene, and homologous genes present in different organisms, resulting in redundancy in our data [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. To address this issue, we employed the homology partitioning algorithm GraphPart [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>] to partition the data prior to training NetStart 2.0. We applied MMseqs2 [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] for alignment and chose a pairwise identity threshold of 50% at the nucleotide level to ensure that no pair of sequences with a higher sequence identity would end up in different partitions [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. We extracted subsequences of 603 nucleotides from each sample, with the labeled ATG being positioned at the center of each subsequence. This was done to execute GraphPart faster, and because NetStart 2.0 takes this sequence window as input. GraphPart was run separately on the different sequence types, while we specified the organism origin for each sequence to ensure an approximately even distribution in each partition considering both organism origin and sequence type. Using this approach, the data was divided into five equally-sized partitions (k = 5). The final dataset, distributed on the 5 partitions, contains 9,912,708 sequences, of which 1,162,194 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/326a904731a8/d33e553.gif" loading="lazy" id="d33e553" alt="Inline graphic"></span>) were TIS-labeled, and 8,750,514 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/64bb52e7b045/d33e559.gif" loading="lazy" id="d33e559" alt="Inline graphic"></span>) were non-TIS labeled. The complete composition of the dataset can be seen in Supplementary Table <a href="#MOESM1" class="usa-link">A2</a>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">NetStart 2.0 architecture and training</h3>
<section id="Sec5"><h4 class="pmc_sec_title">Prediction task and objective</h4>
<p id="Par26">The aim of NetStart 2.0 is to predict TISs of protein-coding mORFs in mRNA sequences from diverse species across the eukaryotic domain. Although the translation of protein-coding mORFs can occasionally initiate at non-ATG codons, such instances are, to current knowledge, relatively rare and not often annotated [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>] (Supplementary Table <a href="#MOESM1" class="usa-link">A3</a>). For this reason, we defined the modeling objective of NetStart 2.0 to be a binary classification task aimed at predicting whether each occurrence of an ATG in a sequence is a TIS or not. Upon prediction of the TIS, the translation termination site can be identified directly from the sequence as the first occurring downstream in-frame stop codon.</p></section><section id="Sec6"><h4 class="pmc_sec_title">Model architecture</h4>
<p id="Par27">The NetStart 2.0 architecture integrates three windows that each process input independently to extract different kinds of information relevant for predicting TIS (Fig. <a href="#Fig2" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h5 class="obj_head">Fig. 2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366053_12859_2025_6220_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/7c3aa05f3756/12859_2025_6220_Fig2_HTML.jpg" loading="lazy" id="MO2" height="684" width="709" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>High-level schematic of the full NetStart 2.0 architecture. As input, NetStart 2.0 takes a transcript sequence centered on a candidate ATG, and a species name, which are processed through three separate windows. The purple window encodes the species input using learned taxonomic embeddings, summed across taxonomic levels. The blue (‘local’) window extracts a short nucleotide subsequence surrounding the candidate ATG and passes it through multiple feed-forward layers to capture patterns in the start codon context. The green (‘global’) window extracts a longer nucleotide subsequence and translates it into the corresponding amino acid sequence. The amino acid sequence is processed by a fine-tuned version of the pretrained protein language model ESM-2 to identify shifts from non-coding to coding regions by evaluating the likelihood that a given subsequence is protein-coding. The resulting embeddings from the three windows are concatenated and passed through a shared feed-forward layer connected to a binary classification head. The model runs independently on each candidate ATG within the input transcript</p></figcaption></figure><p id="Par28">The first window takes a species name as input and represents the distinct sequence patterns of the individual species (Fig. <a href="#Fig2" class="usa-link">2</a>, purple window). Given the evolutionary relationships among organisms, they can be classified at various taxonomical levels. Adopting the approach by Teufel et al. [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>], we represented individual species based on seven taxonomical levels denoted as <em>L</em>: Kingdom, Phylum, Class, Order, Family, Genus, and Species, as defined by the NCBI Taxonomy classification system [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. For each level, embeddings of the same dimension were learned independently from scratch across all species present in the dataset. The embedding dimension was defined as a hyperparameter (Supplementary Tables <a href="#MOESM1" class="usa-link">A4</a> and <a href="#MOESM1" class="usa-link">A5</a>), and the full representation of a species was expressed as the sum of its taxonomic rank embeddings:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/df806db6aa56/d33e615.gif" loading="lazy" id="d33e615" alt="graphic file with name d33e615.gif"></td>
<td class="label">1</td>
</tr></table>
<p>The remaining two windows take the nucleotide sequence as input and process it independently. The ‘local’ sequence window (Fig. <a href="#Fig2" class="usa-link">2</a>, blue window) takes a short nucleotide region surrounding the candidate ATG as input. The window width is defined as a hyperparameter in the range from 10 to 30 nucleotides upstream and downstream of the ATG, respectively, to identify patterns in the immediate start codon context (see Supplementary Tables <a href="#MOESM1" class="usa-link">A4</a> and <a href="#MOESM1" class="usa-link">A5</a>). This input is one-hot encoded as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/e0182566ac1b/d33e631.gif" loading="lazy" id="d33e631" alt="Inline graphic"></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/8f58b99a4e0f/d33e638.gif" loading="lazy" id="d33e638" alt="Inline graphic"></span> and processed by a number of separate feed-forward layers, defined as a hyperparameter that ranged in depth from 2 to 5 for the distinct data splits (Supplementary Table <a href="#MOESM1" class="usa-link">A5</a>). Initially, we experimented with letting a nucleotide language model [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>] encode the local sequence window, but it did not improve performance.</p>
<p id="Par29">The ‘global’ sequence window (Fig. <a href="#Fig2" class="usa-link">2</a>, green window) aims to identify shifts from non-coding to coding regions, examining whether subsequences upstream or downstream of an ATG can be translated into protein-like structures. This window takes a translated nucleotide sequence of 100 amino acids upstream and downstream of the labeled ATG, respectively, as input (this input size was determined based on initial experiments and the results shown in Fig. <a href="#Fig3" class="usa-link">3</a>). The amino acid sequence was tokenized and encoded with the smallest version of the pretrained protein language model ESM-2 (8 M parameters) [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. The input embedding is denoted as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/d74e962552a1/d33e661.gif" loading="lazy" id="d33e661" alt="Inline graphic"></span>, with <em>n</em> being the number of input tokens (i.e., amino acids), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/a8d9dd4c8e32/d33e671.gif" loading="lazy" id="d33e671" alt="Inline graphic"></span> being the embedding dimension of each token (320), and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/0f143894cae5/d33e677.gif" loading="lazy" id="d33e677" alt="Inline graphic"></span> denoting the vector embedding at token <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/505b2bc60370/d33e683.gif" loading="lazy" id="d33e683" alt="Inline graphic"></span> [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. Stop codons (TAA, TAG, and TGA) were encoded as unknown tokens (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/124ebb3456d7/d33e695.gif" loading="lazy" id="d33e695" alt="Inline graphic"></span>unk<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/10d523d2befc/d33e702.gif" loading="lazy" id="d33e702" alt="Inline graphic"></span>). Sequences labeled with a TIS, that had a 5<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/81fec6beadce/d33e708.gif" loading="lazy" id="d33e708" alt="Inline graphic"></span> UTR shorter than the pre-defined input length (25% of the dataset) were padded with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/124ebb3456d7/d33e714.gif" loading="lazy" id="d33e714" alt="Inline graphic"></span>pad<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/10d523d2befc/d33e720.gif" loading="lazy" id="d33e720" alt="Inline graphic"></span> tokens. To ensure consistency between TIS- and non-TIS labeled sequences, we masked the upstream nucleotides in 25% of non-TIS sequences, matching the padding-length distribution observed in the TIS-labeled sequences. We obtained a contextualized representation of the amino acid sequence as the last hidden state of the encoder, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/79709ec02e51/d33e726.gif" loading="lazy" id="d33e726" alt="Inline graphic"></span>, which was fed into a separate, down-scaling feed-forward layer. The embeddings from the three separate windows (i.e., the organism embedding window, the ‘local’ nucleotide sequence window, and the ‘global’ amino acid sequence window) were then concatenated and fed through a shared feed-forward layer, directed to a binary classification layer outputting the probability of an ATG being a TIS.</p>
<figure class="fig xbox font-sm" id="Fig3"><h5 class="obj_head">Fig. 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366053_12859_2025_6220_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/685241ae5ecf/12859_2025_6220_Fig3_HTML.jpg" loading="lazy" id="MO3" height="952" width="709" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Learned sequence representations obtained from the last hidden state of the ESM-2 encoder, projected onto the first two principal components for a random subset of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/ebee973442b1/d33e922.gif" loading="lazy" id="d33e922" alt="Inline graphic"></span> sequences from the non-homologous test set. The objective is to distinguish TIS ATGs from non-TIS ATGs. Sequences are colored by origin: Green = TIS ATG from mRNA sequence. Dark blue = non-TIS ATG from intergenic sequence. Brown = non-TIS ATG from intron sequence. Red = non-TIS ATG from mRNA sequence, upstream of and in an alternative reading frame to the TIS ATG. Orange = non-TIS ATG from mRNA sequence, upstream of and in the same reading frame as the TIS ATG. Cyan = non-TIS ATG from mRNA sequence, downstream of and in an alternative reading frame to the TIS ATG. Pink = non-TIS ATG from mRNA sequence, downstream of and in the same reading frame as the TIS ATG. <strong>A–C</strong> Results for the ESM-2 encoder without fine-tuning for different sequence lengths. <strong>D–F</strong> Results after fine-tuning, trained on one of the four data splits for different sequence lengths. Supplementary Figs. <a href="#MOESM1" class="usa-link">A2</a>, <a href="#MOESM1" class="usa-link">A3</a>, and <a href="#MOESM1" class="usa-link">A4</a> show results for fine-tuning on the other data splits</p></figcaption></figure></section><section id="Sec7"><h4 class="pmc_sec_title">Training procedure</h4>
<p id="Par30">The full training procedure was conducted utilizing unnested 4-fold cross validation, with four data partitions being used as training and validation sets in rotation, and the fifth data partition serving as the independent test set. To reduce computational time, the ESM-2 encoder was fine-tuned separately prior to end-to-end training. The fine-tuning was performed using the EsmForSequenceClassification class from the Hugging Face transformers library, which attaches a classification head to the embedding of the first token from the last hidden layer (used as a sequence summary) [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. The fine-tuning objective was defined as a binary classification task, predicting whether the labeled ATG within a given sequence represents a TIS or not. All ESM-2 weights were updated during fine-tuning. The fine-tuning allowed ESM-2 to adapt to the task of detecting shifts from non-coding to coding sequence, while this discrimination has not been part of the pretraining. We conducted preliminary experiments involving fine-tuning the model on different sequence input lengths (cf. Fig. <a href="#Fig3" class="usa-link">3</a>) and treating the amino acid sequence input length as a hyperparameter. These results showed that including a longer upstream context consistently improved performance. Based on these findings, we used a context window of 201 amino acids centered around the labeled ATG in the final model.</p>
<p id="Par31">Subsequently, a range of hyperparameters were tuned using Optuna (Supplementary Tables <a href="#MOESM1" class="usa-link">A4</a> and <a href="#MOESM1" class="usa-link">A5</a>). We then trained the full model end-to-end with the fine-tuned ESM-2 and optimized hyperparameters defined for each data split. The final model was constructed as an ensemble, averaging the probabilities predicted by the four models trained on distinct data splits. The fine-tuning and training took approximately 22 h and 20 h, respectively, on an NVIDIA L40S GPU.</p>
<p id="Par32">For both the fine-tuning of ESM-2 and the training of the full NetStart 2.0 model, we used weighted Binary Cross-Entropy (BCE) as the loss function, employed the Adam algorithm as optimizer [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>], and implemented early stopping by monitoring the BCE loss on the validation set [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. For the weighted BCE loss, the weight for TIS-labeled sequences was set to three times that of non-TIS labeled sequences, increasing the penalty for incorrect predictions of TIS samples to address the class imbalance. These weights were chosen based on initial experiments. For NetStart 2.0, we applied dropout to each feed-forward layer [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>].</p></section><section id="Sec8"><h4 class="pmc_sec_title">Ablation studies</h4>
<p id="Par33">To assess the relative performance contributions provided by the different input windows to NetStart 2.0, we conducted two ablation studies following the same procedure of training on the 4 data splits and constructing the models as ensembles. The first ablation model uses only the fine-tuned ESM-2 encoder with an attached classification head and is referred to as “NetStart 2.0A”. With the second ablation model, we aimed to mimic the architecture of NetStart 1.0 [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>], using the local nucleotide input window of NetStart 2.0 but with a larger subsequence surrounding the labeled ATG, optimized as a hyperparameter (Supplementary Table <a href="#MOESM1" class="usa-link">A6</a>). We additionally included the organism input due to the high diversity of species in the training data, and refer to this ablation model as “NetStart 1.0A”.</p></section></section><section id="Sec9"><h3 class="pmc_sec_title">Model evaluation and benchmarking</h3>
<p id="Par34">We compared the performance of NetStart 2.0 to that of TIS Transformer [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>], as well as that of the <em>ab initio</em> gene finders AUGUSTUS [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>] and the recently developed Tiberius [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>].</p>
<p id="Par35">TIS Transformer is trained on the human transcriptome, Tiberius on a range of vertebrate genomes, and AUGUSTUS on diverse eukaryotic species. Since AUGUSTUS requires the user to specify a species, we selected the most closely related available species based on the NCBI Taxonomy classification [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>] for those not directly supported in our dataset (Supplementary Table <a href="#MOESM1" class="usa-link">A7</a>). Tiberius was run in <em>ab initio</em>-mode, and had high memory- and time demands (run on an NVIDIA L40S GPU node), likely due to it being a novel model [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] with experimental code. For these reasons, we included predictions for only one species per defined organism group with Tiberius, namely <em>Homo sapiens</em>, <em>Drosophila melanogaster</em>, <em>Cryptococcus neoformans</em>, <em>Toxoplasma gondii</em>, and <em>Arabidopsis thaliana</em>, selected on the criterion of having good RefSeq coverage (Supplementary Table <a href="#MOESM1" class="usa-link">A8</a>). By default, the raw datasets were already softmasked, and Tiberius and AUGUSTUS were run with the softmasking input [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>, <a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>].</p>
<section id="Sec10"><h4 class="pmc_sec_title">Construction of benchmark test sets</h4>
<p id="Par36">NetStart 2.0 is trained on nucleotide sequences up to 603 nucleotides, TIS Transformer is trained on full human transcript sequences (up to 30,000 nucleotides), and AUGUSTUS and Tiberius on genomic sequences. Given these differences in training data, we aimed at creating test sets that would provide a fair evaluation of all models. The test sets were based on the NetStart 2.0 test partition, with a few modifications: (1) We extracted the full transcripts for the TIS-labeled sequences. (2) For transcripts without an annotated transcription start site (where mRNA and CDS annotations begin at the same position), we added 180 nucleotides upstream of the TIS to approximate a 5<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/81fec6beadce/d33e831.gif" loading="lazy" id="d33e831" alt="Inline graphic"></span> UTR (Supplementary Fig. <a href="#MOESM1" class="usa-link">A1</a>). (3) We excluded transcript sequences longer than 30,000 nucleotides. (4) We extracted 500 nucleotides upstream and downstream of the labeled ATG for non-TIS sequences. (5) We excluded sequences with unknown nucleotides (denoted by any other letter than A, T, G or C). This test set was used as the foundation for the benchmark, referred to as the <em>non-homologous test set</em> (Supplementary Table <a href="#MOESM1" class="usa-link">A9</a>).</p>
<p id="Par37">We used the non-homologous test set, encompassing sequence with a single labeled TIS ATG or non-TIS ATG as the foundation for two additional test sets. We extracted all transcripts with a labeled TIS from the non-homologous test set and with an annotated transcription start site to assess the transcript-level accuracy. We refer to this test set as the <em>transcript-level test set</em>. While NetStart 2.0 is trained for transcript-level predictions, we also wanted to assess its applicability on the genomic level, and extracted all genes corresponding to the transcripts with a labeled TIS from the non-homologous test set, which we merged with the non-TIS labeled sequences from the non-homologous test set. We refer to this test set as the <em>genomic test set</em>. As the promoter region is very rarely annotated, we added 1000 nucleotides upstream of each gene to account for this (eukaryotic promoter regions typically span from 100 to 1000 nucleotides) [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. For AUGUSTUS and Tiberius, we included an additional DNA region of 1000 nucleotides upstream and downstream of each gene to represent a more realistic use case for these models with surrounding context. We removed all duplicates arising from distinct mRNA variants with the same TIS, as well as all genes longer than 30,000 nucleotides (Supplementary Table <a href="#MOESM1" class="usa-link">A10</a>).</p></section><section id="Sec11"><h4 class="pmc_sec_title">Performance metrics</h4>
<p id="Par38">For evaluation, we calculated several performance metrics to provide a comprehensive assessment. We calculated the area under the Receiver Operating Characteristic curve (AUC) and the Average Precision Score (APS) as threshold-independent measures (i.e., they summarize model performance across all possible classification thresholds). AUC reflects the model’s ability to distinguish between classes [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. The APS is calculated as the weighted mean of the precisions obtained along the precision-recall curve, and is approximately equivalent to the area under the precision-recall curve, but unlike interpolated AUPR estimates, it is less sensitive to local fluctuations and data sparsity [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>, <a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>].</p>
<p id="Par39">We also used Matthews correlation coefficient (MCC), which incorporates counts of True Positives (TP), True Negatives (TN), False Positives (FP) and False Negatives (FN) to provide a balanced evaluation of model performance. MCC was calculated across thresholds, with the optimal threshold defined as the one maximizing MCC:</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/97870556d3f6/d33e875.gif" loading="lazy" id="d33e875" alt="graphic file with name d33e875.gif"></td>
<td class="label">2</td>
</tr></table>
<p>At the optimal threshold, error rates were calculated for the different sequence types in the dataset. For example, the error rate on non-TIS ATGs placed upstream of and in the same reading frame as the TIS, was calculated as:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/595c87a10ff2/d33e882.gif" loading="lazy" id="d33e882" alt="graphic file with name d33e882.gif"></td>
<td class="label">3</td>
</tr></table></section></section></section><section id="Sec12"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec13"><h3 class="pmc_sec_title">Impact of fine-tuning of ESM-2</h3>
<p id="Par40">We analyzed the sequence representations learned by the ESM-2 encoder for varying sequence input lengths, both before and after fine-tuning. Amino acid embeddings were extracted from the last hidden state of the encoder, where each amino acid in a sequence is represented by a vector of length 320 (the embedding dimension). For each input sequence, the per-residue embeddings were flattened into a single vector by concatenation, resulting in a representation with dimensionality equal to the number of amino acids multiplied by 320. Each vector was standardized, and a principal component analysis (PCA) was performed on the standardized embeddings. The objective of the PCA was to visualize the learned representations in two-dimensional space and assess the extent to which fine-tuning ESM-2 altered the underlying amino acid representations. We considered three input sequence windows, each with a sequence length of 100 amino acids downstream of the TIS but with a varying sequence length upstream of the TIS, specifically: (1) 0 amino acids, (2) 50 amino acids, and (3) 100 amino acids (Fig. <a href="#Fig3" class="usa-link">3</a>).</p>
<p id="Par41">The aim was to qualitatively assess the influence that fine-tuning and including translated nucleotide context would have on distinguishing TIS ATGs from non-TIS ATGs. Note that there are two feed-forward layers with non-linear (ReLU) activation between this representation and the model output (cf. Fig. <a href="#Fig2" class="usa-link">2</a>) and that the first two principal components explain only a minor part of the variance. These circumstances mean that there is not a 1:1 correspondence between the PCA and the model performance. Still, visual separation of clusters in the PCA serves as a strong indication that the downstream layers will be able to computationally separate the groups.</p>
<p id="Par42">Without fine-tuning, there was substantial overlap between the ESM-2 embeddings for TIS- and non-TIS labeled ATGs (Fig. <a href="#Fig3" class="usa-link">3</a>A–C). However, including 100 amino acids upstream and downstream of the TIS, resulted in the embeddings for a subset of the TIS-labeled sequences to be clearly separated from those of non-TIS labeled sequences, indicating that ESM-2 can capture relevant information without being fine-tuned, if provided enough context (Fig. <a href="#Fig3" class="usa-link">3</a>C). After fine-tuning, the model version which was provided only downstream context still struggled to distinguish TIS ATGs from certain non-TIS ATGs located downstream and in-frame relative to the true TIS (Fig. <a href="#Fig3" class="usa-link">3</a>D). This outcome was expected, as the input windows for both sequence types are fully protein-coding. In contrast, when using a sequence window that included 100 amino acids both upstream and downstream of the TIS, the embeddings of the fine-tuned model showed clear separation of TIS- and non-TIS ATGs, highlighting the importance of the model being allowed to learn the transition from the non-coding to coding region (Fig. <a href="#Fig3" class="usa-link">3</a>F). Three well-defined clusters were observed: one comprised of TIS-labeled sequences, one comprised of downstream, in-frame, non-TIS mRNA sequences, and a broader cluster encompassing the remaining non-TIS sequence types. The observed patterns suggest that some downstream, in-frame, non-TIS ATGs are learned differently compared to other non-TIS sequence types, despite the fine-tuning objective solely being to distinguish TIS from non-TIS ATGs. This could be expected as this non-TIS sequence type is protein-coding while the remaining non-TIS sequence types are not. Overall, these results highlight the importance of including context both upstream and downstream of the labeled ATG, in order for the encoder to utilize transitions from non-coding to protein-coding regions in the overall assessment.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Benchmarking of TIS prediction models</h3>
<section id="Sec15"><h4 class="pmc_sec_title">Transcript-level accuracy</h4>
<p id="Par43">To evaluate NetStart 2.0’s performance on identifying the correct mORF TIS in a transcript containing several ATG codons, we calculated the transcript-level accuracy (based on the transcript-level test set, see “<a href="#Sec9" class="usa-link">Model evaluation and benchmarking</a>” section, <em>Construction of Benchmark Test Sets</em>). We define the transcript-level accuracy as the fraction of transcripts for which the annotated TIS ATG received the highest predicted probability among all ATGs present in that transcript. It should be noted that TIS Transformer is trained to predict various kinds of TISs, including those of sORFs within mRNA transcripts. Quantifying this potential bias is challenging since TIS Transformer does not differentiate between specific types of TIS in its predictions, meaning that all predictions are given as the binary output of “TIS” or “Non-TIS”. In this setup, we assume TIS Transformer to predict the TIS of the mORF with a higher probability than of any other potential ORF in the transcript.</p>
<p id="Par44">Overall, NetStart 2.0 achieves the highest transcript-level accuracies among the evaluated models across nearly all organism groups (Table <a href="#Tab1" class="usa-link">1</a>). Notably, NetStart 2.0 outperforms other methods on human transcripts (<em>H. sapiens</em>), particularly when considering only RefSeq-annotated sequences. TIS Transformer, despite being exclusively trained on human transcripts, achieves the highest accuracy within the fungal phylum, surpassing NetStart 2.0 in this specific group.</p>
<section class="tw xbox font-sm" id="Tab1"><h5 class="obj_head">Table 1.</h5>
<div class="caption p"><p>Transcript-level accuracies (%), defined as the percentage of transcripts in which the TIS ATG is predicted with higher probability than all other ATGs in that transcript</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"><em>H. sapiens</em></th>
<th align="left" colspan="1" rowspan="1">Vertebrates</th>
<th align="left" colspan="1" rowspan="1">Invertebrates</th>
<th align="left" colspan="1" rowspan="1">Plants</th>
<th align="left" colspan="1" rowspan="1">Fungi</th>
<th align="left" colspan="1" rowspan="1">Protozoans</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Transcript Count</td>
<td align="left" colspan="1" rowspan="1">9542 (5280)</td>
<td align="left" colspan="1" rowspan="1">106,046 (15,113)</td>
<td align="left" colspan="1" rowspan="1">15,069 (4671)</td>
<td align="left" colspan="1" rowspan="1">65,780 (6412)</td>
<td align="left" colspan="1" rowspan="1">4705</td>
<td align="left" colspan="1" rowspan="1">954</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AUGUSTUS</td>
<td align="left" colspan="1" rowspan="1">60.31 (63.66)</td>
<td align="left" colspan="1" rowspan="1">64.99 (67.66)</td>
<td align="left" colspan="1" rowspan="1">68.41 (71.50)</td>
<td align="left" colspan="1" rowspan="1">45.93 (45.91)</td>
<td align="left" colspan="1" rowspan="1">57.32</td>
<td align="left" colspan="1" rowspan="1">27.88</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Tiberius</td>
<td align="left" colspan="1" rowspan="1">72.55 (74.49)</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 1.0A</td>
<td align="left" colspan="1" rowspan="1">43.75 (49.47)</td>
<td align="left" colspan="1" rowspan="1">47.16 (53.70)</td>
<td align="left" colspan="1" rowspan="1">52.05 (56.50)</td>
<td align="left" colspan="1" rowspan="1">60.05 (63.24)</td>
<td align="left" colspan="1" rowspan="1">62.57</td>
<td align="left" colspan="1" rowspan="1">56.71</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TIS Transformer</td>
<td align="left" colspan="1" rowspan="1">89.39 (86.86)</td>
<td align="left" colspan="1" rowspan="1">93.55 (90.52)</td>
<td align="left" colspan="1" rowspan="1">94.80 (<strong>92</strong>.<strong>16</strong>)</td>
<td align="left" colspan="1" rowspan="1">92.61 (93.25)</td>
<td align="left" colspan="1" rowspan="1">
<strong>93</strong>.<strong>14</strong>
</td>
<td align="left" colspan="1" rowspan="1">88.68</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 2.0A</td>
<td align="left" colspan="1" rowspan="1">91.25 (90.57)</td>
<td align="left" colspan="1" rowspan="1">94.45 (92.26)</td>
<td align="left" colspan="1" rowspan="1">94.55 (91.42)</td>
<td align="left" colspan="1" rowspan="1">95.42 (95.12)</td>
<td align="left" colspan="1" rowspan="1">91.41</td>
<td align="left" colspan="1" rowspan="1">87.94</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 2.0</td>
<td align="left" colspan="1" rowspan="1">
<strong>91</strong>.<strong>89</strong> (<strong>91</strong>.<strong>12</strong>)</td>
<td align="left" colspan="1" rowspan="1">
<strong>94</strong>.<strong>75</strong> (<strong>92</strong>.<strong>56</strong>)</td>
<td align="left" colspan="1" rowspan="1">
<strong>94</strong>.<strong>92</strong> (91.56)</td>
<td align="left" colspan="1" rowspan="1">
<strong>95</strong>.<strong>52</strong> (<strong>95</strong>.<strong>24</strong>)</td>
<td align="left" colspan="1" rowspan="1">91.63</td>
<td align="left" colspan="1" rowspan="1">
<strong>90</strong>.<strong>04</strong>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p38"><p>The results are based on the transcript-level test set. The gene finders (AUGUSTUS and Tiberius) do not output a probability score but instead provide annotations when a CDS is predicted. For transcripts where no CDS was predicted, we treated these cases as the prediction being wrong (equivalent to a non-TIS ATG being predicted with a higher probability than the actual TIS ATG). The accuracies in parenthesis correspond to the RefSeq-annotated transcripts only. For the fungal and protozoan groups, all transcripts were RefSeq-annotated. The numbers marked in bold highlight the highest accuracy across models within each systematic group. NetStart 1.0A and NetStart 2.0A are the ablations described in “<a href="#Sec4" class="usa-link">NetStart 2.0 architecture and training</a>” section, <em>Ablation Studies</em></p></div></div></section><p id="Par45">The ablation study reveals that the simplified model utilizing the ESM-2 protein language model to encode translated transcript sequences, NetStart 2.0A, still performs exceptionally well, suggesting that peptide-level context alone captures substantial predictive power. In contrast, NetStart 1.0A, which mimics the original NetStart 1.0 architecture, achieves considerably lower accuracy compared to all other benchmarked models.</p>
<p id="Par46">AUGUSTUS consistently performs substantially worse than NetStart 2.0, especially for plant and protozoan transcripts, while Tiberius generally outperforms AUGUSTUS, but still attains markedly lower accuracies than NetStart 2.0. A notable exception is the fungus <em>C. neoformans</em>, where Tiberius performed worst among all evaluated methods (see Supplementary Tables <a href="#MOESM1" class="usa-link">A11</a> and <a href="#MOESM1" class="usa-link">A12</a>).</p></section><section id="Sec16"><h4 class="pmc_sec_title">Performance on the non-homologous test set</h4>
<p id="Par47">We evaluated performance of each model on the non-homologous test set, excluding introns and intergenic sequences (see “<a href="#Sec9" class="usa-link">Model evaluation and benchmarking</a>” section, <em>Construction of Benchmark Test Sets</em>). Additionally, we calculated sequence identity to TIS Transformer’s training set and removed transcripts with more than <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/dfd3a2f035b5/d33e1192.gif" loading="lazy" id="d33e1192" alt="Inline graphic"></span> sequence identity (Supplementary Table <a href="#MOESM1" class="usa-link">A13</a>). The threshold-independent metrics AUC and APS were calculated for the TIS prediction models (AUGUSTUS and Tiberius provide the TIS only if predicted, and not a probability), and MCC was calculated for all models. For the models outputting a probability for TIS, we defined the optimal threshold as the one maximizing MCC, which was found at 0.05 for TIS Transformer, and 0.625 for NetStart 2.0 and the ablation models (Table <a href="#Tab2" class="usa-link">2</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">A5</a>). Our results indicate that NetStart 2.0 consistently achieves slightly higher performance than the other evaluated models across organism groups, although the performances of both TIS Transformer and NetStart 2.0A are generally comparable. The results further indicate that using the full NetStart 2.0 architecture rather than NetStart 2.0A has the most substantial impact on underrepresented groups, especially protozoan and fungal species. In contrast, the differences between NetStart 2.0 and NetStart 2.0A were minimal for the remaining organism groups.</p>
<section class="tw xbox font-sm" id="Tab2"><h5 class="obj_head">Table 2.</h5>
<div class="caption p"><p>Predictive performance of models measured on the non-homologous test set of each organism group (binary classification of TIS versus non-TIS ATGs): Matthews Correlation Coefficient (MCC), Area Under the ROC Curve (AUC), and Average Precision Score (APS)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"><em>H. sapiens</em></th>
<th align="left" colspan="1" rowspan="1">Vertebrates</th>
<th align="left" colspan="1" rowspan="1">Invertebrates</th>
<th align="left" colspan="1" rowspan="1">Plants</th>
<th align="left" colspan="1" rowspan="1">Fungi</th>
<th align="left" colspan="1" rowspan="1">Protozoa</th>
</tr></thead>
<tbody>
<tr><td align="left" colspan="7" rowspan="1"><em>MCC</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">AUGUSTUS</td>
<td align="left" colspan="1" rowspan="1">0.695 (0.704)</td>
<td align="left" colspan="1" rowspan="1">0.750 (0.757)</td>
<td align="left" colspan="1" rowspan="1">0.741 (0.701)</td>
<td align="left" colspan="1" rowspan="1">0.616 (0.626)</td>
<td align="left" colspan="1" rowspan="1">0.647</td>
<td align="left" colspan="1" rowspan="1">0.436</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Tiberius</td>
<td align="left" colspan="1" rowspan="1">0.790 (0.794)</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 1.0A</td>
<td align="left" colspan="1" rowspan="1">0.602 (0.641)</td>
<td align="left" colspan="1" rowspan="1">0.695 (0.752)</td>
<td align="left" colspan="1" rowspan="1">0.769 (0.787)</td>
<td align="left" colspan="1" rowspan="1">0.775 (0.810)</td>
<td align="left" colspan="1" rowspan="1">0.809</td>
<td align="left" colspan="1" rowspan="1">0.794</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TIS Transformer</td>
<td align="left" colspan="1" rowspan="1">0.883 (0.862)</td>
<td align="left" colspan="1" rowspan="1">0.919 (0.910)</td>
<td align="left" colspan="1" rowspan="1">0.917 (0.897)</td>
<td align="left" colspan="1" rowspan="1">0.907 (0.908)</td>
<td align="left" colspan="1" rowspan="1">0.863</td>
<td align="left" colspan="1" rowspan="1">0.826</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 2.0A</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>912</strong> (<strong>0</strong>.<strong>930</strong>)</td>
<td align="left" colspan="1" rowspan="1">0.933 (<strong>0</strong>.<strong>942</strong>)</td>
<td align="left" colspan="1" rowspan="1">0.928 (0.911)</td>
<td align="left" colspan="1" rowspan="1">0.936 (0.945)</td>
<td align="left" colspan="1" rowspan="1">0.905</td>
<td align="left" colspan="1" rowspan="1">0.900</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 2.0</td>
<td align="left" colspan="1" rowspan="1">0.910 (0.923)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>937</strong> (0.940)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>936</strong> (<strong>0</strong>.<strong>926</strong>)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>939</strong> (<strong>0</strong>.<strong>950</strong>)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>925</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>925</strong>
</td>
</tr>
<tr><td align="left" colspan="7" rowspan="1"><em>AUC</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 1.0A</td>
<td align="left" colspan="1" rowspan="1">0.949 (0.947)</td>
<td align="left" colspan="1" rowspan="1">0.959 (0.964)</td>
<td align="left" colspan="1" rowspan="1">0.962 (0.964)</td>
<td align="left" colspan="1" rowspan="1">0.964 (0.968)</td>
<td align="left" colspan="1" rowspan="1">0.967</td>
<td align="left" colspan="1" rowspan="1">0.959</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TIS Transformer</td>
<td align="left" colspan="1" rowspan="1">0.994 (0.990)</td>
<td align="left" colspan="1" rowspan="1">0.997 (0.994)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>995</strong> (0.992)</td>
<td align="left" colspan="1" rowspan="1">0.994 (0.993)</td>
<td align="left" colspan="1" rowspan="1">0.987</td>
<td align="left" colspan="1" rowspan="1">0.981</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 2.0A</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>998</strong> (<strong>0</strong>.<strong>997</strong>)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>998</strong> (<strong>0</strong>.<strong>997</strong>)</td>
<td align="left" colspan="1" rowspan="1">0.994 (0.991)</td>
<td align="left" colspan="1" rowspan="1">0.995 (0.995)</td>
<td align="left" colspan="1" rowspan="1">0.989</td>
<td align="left" colspan="1" rowspan="1">0.990</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 2.0</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>998</strong> (<strong>0</strong>.<strong>997</strong>)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>998</strong> (<strong>0</strong>.<strong>997</strong>)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>995</strong> (<strong>0</strong>.<strong>993</strong>)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>996</strong> (<strong>0</strong>.<strong>996</strong>)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>993</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>993</strong>
</td>
</tr>
<tr><td align="left" colspan="7" rowspan="1"><em>APS</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 1.0A</td>
<td align="left" colspan="1" rowspan="1">0.667 (0.718)</td>
<td align="left" colspan="1" rowspan="1">0.789 (0.857)</td>
<td align="left" colspan="1" rowspan="1">0.898 (0.917)</td>
<td align="left" colspan="1" rowspan="1">0.897 (0.930)</td>
<td align="left" colspan="1" rowspan="1">0.941</td>
<td align="left" colspan="1" rowspan="1">0.923</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TIS Transformer</td>
<td align="left" colspan="1" rowspan="1">0.955 (0.944)</td>
<td align="left" colspan="1" rowspan="1">0.980 (0.978)</td>
<td align="left" colspan="1" rowspan="1">0.985 (0.981)</td>
<td align="left" colspan="1" rowspan="1">0.979 (0.983)</td>
<td align="left" colspan="1" rowspan="1">0.974</td>
<td align="left" colspan="1" rowspan="1">0.955</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 2.0A</td>
<td align="left" colspan="1" rowspan="1">0.972 (<strong>0</strong>.<strong>979</strong>)</td>
<td align="left" colspan="1" rowspan="1">0.984 (<strong>0</strong>.<strong>987</strong>)</td>
<td align="left" colspan="1" rowspan="1">0.984 (0.981)</td>
<td align="left" colspan="1" rowspan="1">0.985 (0.990)</td>
<td align="left" colspan="1" rowspan="1">0.981</td>
<td align="left" colspan="1" rowspan="1">0.979</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 2.0</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>974</strong> (<strong>0</strong>.<strong>979</strong>)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>986</strong> (0.986)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>987</strong> (<strong>0</strong>.<strong>984</strong>)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>987</strong> (<strong>0</strong>.<strong>992</strong>)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>987</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>986</strong>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p45"><p>MCCs were calculated based on the best universal threshold for each model (0.05 for TIS Transformer, and 0.625 for NetStart 2.0 and the ablation models). Performance measured only on the RefSeq-annotated sequences from the non-homologous test set is stated in parentheses. For the fungal and protozoan groups, all transcripts were annotated with RefSeq. The numbers marked in bold highlight the best performance of the given metric comparing models within each systematic group</p></div></div></section><p id="Par48">The optimal thresholds found for the distinct models align well with the observed probability distributions (Fig. <a href="#Fig4" class="usa-link">4</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">A6</a>). NetStart 2.0’s predicted probabilities are concentrated towards 0 and 1 for non-TIS and TIS ATGs, respectively, whereas TIS Transformer’s non-TIS predictions are highly concentrated near 0 but with the TIS ATGs showing a clear two-peaked distribution of probabilities.</p>
<figure class="fig xbox font-sm" id="Fig4"><h5 class="obj_head">Fig. 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/d8fee432fd33/12859_2025_6220_Fig4_HTML.jpg" loading="lazy" id="MO4" height="1039" width="708" alt="Fig. 4"></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distributions of predicted probabilities with NetStart 2.0 and TIS Transformer, based on the non-homologous test set</p></figcaption></figure><p id="Par49">We also calculated error rates for each specific sequence type based on the MCC-optimized thresholds for each model (Fig. <a href="#Fig5" class="usa-link">5</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">A7</a>). Both AUGUSTUS and Tiberius exhibit high specificity but low sensitivity, resulting in low error rates across all non-TIS ATGs but high error rates on the TIS ATGs. Among the different types of non-TIS sequences, TIS Transformer, NetStart 2.0A, and NetStart 2.0 show the highest error rates for ATGs located downstream of and in the same reading frame as the TIS, indicating that distinguishing these ATGs from true TIS ATGs remains the greatest challenge for these models (see Supplementary Figs. <a href="#MOESM1" class="usa-link">A8</a> and <a href="#MOESM1" class="usa-link">A9</a> for further results regarding sequence type- and species-specific error rates calculated for NetStart 2.0).</p>
<figure class="fig xbox font-sm" id="Fig5"><h5 class="obj_head">Fig. 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366053_12859_2025_6220_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/42adcfb1f70d/12859_2025_6220_Fig5_HTML.jpg" loading="lazy" id="MO5" height="356" width="708" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Error rates calculated for the various non-TIS sequence types and the TIS sequences separately on the non-homologous test set, based on the best universal threshold for each model (0.05 for TIS Transformer, and 0.625 for NetStart 2.0 and the ablation models). “Upstream”, refers to the labeled ATG being placed in the 5<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/81fec6beadce/d33e1715.gif" loading="lazy" id="d33e1715" alt="Inline graphic"></span> UTR of the transcript, whereas “Downstream” refers to the labeled ATG being placed in the coding region of the transcript. “In frame” refers to the labeled ATG being placed in the same reading frame as the TIS ATG, and “out of frame” refers to the labeled ATG being placed in a reading frame alternative to the TIS ATG</p></figcaption></figure></section><section id="Sec17"><h4 class="pmc_sec_title">Performance at the genomic level</h4>
<p id="Par50">To assess the applicability of NetStart 2.0 on the genomic level, we benchmarked it using the genomic test set (see “<a href="#Sec9" class="usa-link">Model evaluation and benchmarking</a>” section, <em>Construction of Benchmark Test Sets</em>). Although NetStart 2.0 outperforms TIS Transformer across most organism groups, both models exhibit a substantial drop in performance at the genomic level, limiting their applicability for this problem. The predicted TIS probability for both TIS Transformer and NetStart 2.0 is strongly influenced by the position of the first downstream intron relative to the TIS (Fig. <a href="#Fig6" class="usa-link">6</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">A10</a>). In contrast, the gene finders generally outperform the transcript-level TIS predictors across most organism groups (Table <a href="#Tab3" class="usa-link">3</a> and Supplementary Table <a href="#MOESM1" class="usa-link">A14</a>). An exception occurs in protozoan species, where NetStart 2.0 achieves the best performance, likely due to the low prevalence of introns in certain protozoan groups, such as Trypanosomes and Leishmania species [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>, <a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. However, the overall shift in relative performance arises primarily from a larger decline in the accuracy of transcript-level predictors, as the gene finders also show slight performance decreases at the genomic level compared to transcript level.</p>
<figure class="fig xbox font-sm" id="Fig6"><h5 class="obj_head">Fig. 6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366053_12859_2025_6220_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/8e86d2b242c0/12859_2025_6220_Fig6_HTML.jpg" loading="lazy" id="MO6" height="467" width="708" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Predicted TIS probability for TIS ATGs at gene level, shown as a function of the position of the first downstream intron relative to the TIS. The dashed lines symbolize the MCC-optimized threshold for each model on the genomic dataset</p></figcaption></figure><section class="tw xbox font-sm" id="Tab3"><h5 class="obj_head">Table 3.</h5>
<div class="caption p"><p>MCC Scores with best universal threshold on the genomic test set for each model (0.025 for TIS Transformer, 0.75 for NetStart 1.0A, 0.7 for NetStart 2.0A, and 0.625 for NetStart 2.0)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"><em>H. sapiens</em></th>
<th align="left" colspan="1" rowspan="1">Vertebrates</th>
<th align="left" colspan="1" rowspan="1">Invertebrates</th>
<th align="left" colspan="1" rowspan="1">Plants</th>
<th align="left" colspan="1" rowspan="1">Fungi</th>
<th align="left" colspan="1" rowspan="1">Protozoa</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">AUGUSTUS</td>
<td align="left" colspan="1" rowspan="1">0.612 (0.640)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>651</strong> (<strong>0</strong>.<strong>682</strong>)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>636</strong> (<strong>0</strong>.<strong>635</strong>)</td>
<td align="left" colspan="1" rowspan="1">0.642 (<strong>0</strong>.<strong>598</strong>)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>710</strong>
</td>
<td align="left" colspan="1" rowspan="1">0.473</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Tiberius</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>739</strong> (<strong>0</strong>.<strong>752</strong>)</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 1.0A</td>
<td align="left" colspan="1" rowspan="1">0.329 (0.404)</td>
<td align="left" colspan="1" rowspan="1">0.422 (0.451)</td>
<td align="left" colspan="1" rowspan="1">0.491 (0.517)</td>
<td align="left" colspan="1" rowspan="1">0.634 (0.539)</td>
<td align="left" colspan="1" rowspan="1">0.650</td>
<td align="left" colspan="1" rowspan="1">0.732</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TIS Transformer</td>
<td align="left" colspan="1" rowspan="1">0.337 (0.410)</td>
<td align="left" colspan="1" rowspan="1">0.407 (0.423)</td>
<td align="left" colspan="1" rowspan="1">0.371 (0.390)</td>
<td align="left" colspan="1" rowspan="1">0.553 (0.451)</td>
<td align="left" colspan="1" rowspan="1">0.499</td>
<td align="left" colspan="1" rowspan="1">0.561</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 2.0A</td>
<td align="left" colspan="1" rowspan="1">0.486 (0.518)</td>
<td align="left" colspan="1" rowspan="1">0.530 (0.487)</td>
<td align="left" colspan="1" rowspan="1">0.514 (0.505)</td>
<td align="left" colspan="1" rowspan="1">0.685 (0.551)</td>
<td align="left" colspan="1" rowspan="1">0.649</td>
<td align="left" colspan="1" rowspan="1">0.827</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NetStart 2.0</td>
<td align="left" colspan="1" rowspan="1">0.465 (0.505)</td>
<td align="left" colspan="1" rowspan="1">0.532 (0.489)</td>
<td align="left" colspan="1" rowspan="1">0.522 (0.520)</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>697</strong> (0.559)</td>
<td align="left" colspan="1" rowspan="1">0.666</td>
<td align="left" colspan="1" rowspan="1">
<strong>0</strong>.<strong>841</strong>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p52"><p>Performance on RefSeq-annotated sequences only is stated in parentheses. For the fungal and protozoan groups, all transcripts were annotated with RefSeq. The numbers marked in bold highlight the best performance across models within each systematic group</p></div></div></section></section></section><section id="Sec18"><h3 class="pmc_sec_title">Additional evaluations of NetStart 2.0</h3>
<section id="Sec19"><h4 class="pmc_sec_title">Impact of taxonomic input information</h4>
<p id="Par51">To evaluate the robustness of NetStart 2.0 when full species-level information is unavailable, we tested the model under three conditions (see Fig. <a href="#Fig2" class="usa-link">2</a>, purple window): (a) organism embedding included species-level detail (using learned embeddings from all 7 taxonomic ranks), (b) organism embeddings were limited to phylum-level detail (using learned embeddings from only kingdom and phylum ranks), and (c) no organism information was provided (organism embeddings were set to 0). We calculated group-specific MCCs based on the non-homologous test set for each condition based on the optimized threshold (Fig. <a href="#Fig7" class="usa-link">7</a>). For vertebrates, reducing taxonomic detail from species level to phylum level does not affect MCC performance, which only slightly decreases when no taxonomic information is included. For plants, a similar trend is observed with a drop in MCC of only 0.003 when providing phylum-level information and an additional drop of 0.004 when no taxonomic information is provided. This indicates that phylum-level embeddings retain most of the useful taxonomic information for predictions made on organisms of vertebrate and plant origin. For the invertebrate, fungal, and protozoan species, the declines in performance are more pronounced. Using phylum-level information, the drop in MCC within these groups ranges from 0.009 for the invertebrates to 0.02 for the protozoans. Performances further decline when no taxonomic information is provided, with the largest decreases observed in the fungal and protozoan groups. These findings highlight that correct taxonomic information is more important for these groups, which generally exhibit more diverse start codon contexts (Nielsen, L.S. et al., manuscript in preparation). It should be emphasized that these results were obtained on the test set comprising sequences from the same organisms that are represented in the training set. Thus, performance on sequences from entirely unseen species could be significantly lower and remains to be investigated.</p>
<figure class="fig xbox font-sm" id="Fig7"><h5 class="obj_head">Fig. 7.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366053_12859_2025_6220_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/63b23850fba6/12859_2025_6220_Fig7_HTML.jpg" loading="lazy" id="MO7" height="345" width="708" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MCCs measured on the non-homologous test set with NetStart 2.0 under three species input settings at the universally optimized threshold (0.625). “Species input” uses the learned embeddings from all 7 ranks for the organism embedding, “Phylum input” truncates the organism embedding to use information until phylum level, and “No organism input” omits taxonomic information entirely</p></figcaption></figure></section><section id="Sec20"><h4 class="pmc_sec_title">Effect of group-specific fine-tuning</h4>
<p id="Par52">To assess the extent to which NetStart 2.0 learned sequence patterns specific to each organism group, we fine-tuned it independently using training data exclusively from each group (vertebrates, invertebrates, plants, fungi, and protozoa) saving individual checkpoints (i.e., snapshots of the optimized model parameters for each group). This yielded only a marginal increase in performance, mostly notable for protozoan species (Supplementary Table <a href="#MOESM1" class="usa-link">A15</a>). Given the limited improvement and reduced flexibility, we opted to implement NetStart 2.0 with a shared checkpoint for all species.</p></section><section id="Sec21"><h4 class="pmc_sec_title">Gnomon annotations: impact on performance</h4>
<p id="Par53">It may seem surprising that the transcript-level accuracies (Table <a href="#Tab1" class="usa-link">1</a>) for NetStart 2.0, NetStart 2.0A, and TIS Transformer are slightly lower for the RefSeq-annotated transcripts only (numbers in parentheses). However, it is not immediately obvious that one should expect a higher performance for data with stronger experimental support (RefSeq). On the one hand, the Gnomon data could be expected to have more noise, since annotations are of a lower quality. On the other hand, Gnomon data could be expected to be more regular, since they may be biased towards genes that are easy to predict. In order to test whether including Gnomon-annotated data had an adverse effect on training, we trained a version of the model using only the vertebrate sequences annotated with RefSeq for each of the 4 data partitions used for model development (473,481 sequences across the 4 partitions, of which <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/ce9dc7a0aa6d/d33e1974.gif" loading="lazy" id="d33e1974" alt="Inline graphic"></span> were TIS-labeled). The training procedure described in “<a href="#Sec4" class="usa-link">NetStart 2.0 architecture and training</a>” section, <em>Training Procedure</em> was followed. We selected the vertebrate group for this experiment, as it is the systematic group with both most species and sequences annotated with RefSeq included in our dataset. The trained model was run on the vertebrate sequences from both the transcript-level test set and the non-homologous test set. Across all metrics (transcript-level accuracy, MCC, AUC, and APS), the performance was slightly lower compared to the original NetStart 2.0 model, even when evaluating on RefSeq-annotated test sequences only (see Supplementary Tables <a href="#MOESM1" class="usa-link">A16</a> and <a href="#MOESM1" class="usa-link">A17</a>). We interpret these results as demonstrating that the bias introduced by including Gnomon-annotated sequences is of negligible order for this scope, as well as showcasing the consequence of decreasing the amount of data used for training.</p></section></section></section><section id="Sec22"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par54">The emergence of pretrained protein language models has significantly advanced protein sequence modeling, and the performance of NetStart 2.0 demonstrates the potential of integrating peptide-level information to improve nucleotide-level sequence predictions. The comparison of NetStart 2.0A to NetStart 2.0 showed that its predictive ability is largely derived from the “global” sequence window (Fig. <a href="#Fig2" class="usa-link">2</a>, green window), which was implemented to understand the “protein-ness” of the sequence, assessing shifts from non-coding to coding regions. Notably, this remained true despite the broad phylogenetic diversity represented in the NetStart 2.0 training set. Our results specifically show that species-level taxonomic information had only minor impact on vertebrates and plants, indicating that NetStart 2.0 relies minimally on detailed species-specific representations for these groups. In contrast, including species-level taxonomical information has a bigger impact on the remaining organisms. Considering diversity in local start codon contexts among organisms, these findings could be expected, given that organisms of plant or vertebrate origins, respectively, have very similar start codon contexts, whereas these patterns vary more for the remaining organism groups (Nielsen, L.S. et al., manuscript in preparation). Our observations suggest that while most of NetStart 2.0’s predictive power arises from the “global” sequence window, assessment of the start codon context also contributes to the overall performance. However, across all groups there was no substantial loss in performance when representation of a species to the model was reduced to the phylum level (Fig. <a href="#Fig7" class="usa-link">7</a>). These findings illustrate the remarkable potential of utilizing protein language models in detecting biologically relevant signals, even beyond their primary training scope.</p>
<p id="Par55">The biggest challenge for NetStart 2.0 was distinguishing true TIS ATGs from downstream non-TIS ATGs located within the same reading frame. This challenge likely originates from the architectural reliance on the global sequence window, as these two sequence types share similar protein-like contexts downstream of the labeled ATG. However, the overall ability of NetStart 2.0A to identify transitions between coding and non-coding regions raises compelling questions about the extent to which such models can comprehend coding potential or “protein-ness” within sequences. Despite being pretrained on full-length proteins, our findings raise the intriguing possibility of using protein language models to detect functional subsequences within proteins.</p>
<p id="Par56">The substantial performance gap between NetStart 1.0A and TIS Transformer underscores the importance of architectural complexity in TIS prediction, given the similarity in their underlying data inputs. Despite being trained exclusively on human transcript sequences, TIS Transformer excelled in learning sequence patterns beyond human context, achieving high performance across diverse eukaryotic groups and pointing to a universal signal in TIS prediction transcending species boundaries despite high diversity in start codon context patterns (Nielsen, L.S. et al., manuscript in preparation).</p>
<p id="Par57">Although fine-tuning ESM-2 substantially altered its amino acid representations (Fig. <a href="#Fig3" class="usa-link">3</a>), we cannot exclude the possibility that downstream feed-forward layers alone could adequately adapt to TIS prediction potentially making fine-tuning unnecessary. However, early modeling experiments showed less optimal results without fine-tuning, leading us to focus on the fine-tuned version of ESM-2. While some tasks using full protein sequences might negate this necessity, fine-tuning has proven beneficial for a broad range of downstream tasks [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. Furthermore, our approach specifically leveraged ESM-2 to detect transitions from non-coding to coding regions, slightly shifting its scope from the original pretraining objective.</p>
<p id="Par58">While NetStart 2.0 slightly outperformed TIS Transformer at the transcript level, both models exhibited limited performance on the genomic test set. This performance drop was strongly influenced by the position of the first downstream intron, highlighting a broader challenge in applying transcript-level TIS predictors to genomic contexts. While both Tiberius and AUGUSTUS in comparison performed notably better on the genomic test set, their performance also declined compared to the transcript-level tests. This suggests considerable potential for improving models specifically designed for genomic-level TIS prediction.</p>
<p id="Par59">Finally, NetStart 2.0’s training was limited to predict canonical start codons (ATG) of main protein-coding ORFs. Recent evidence from ribosome profiling has revealed a substantial presence of sORFs with both canonical and non-canonical start codons in mRNA transcripts [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. However, annotations of such features are currently very limited in genomic annotation databases such as NCBI’s Eukaryotic Genome Annotation Pipeline [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Future research incorporating predictions on both sORFs and non-canonical start codons could significantly expand the scope and practical utility of TIS prediction models such as NetStart 2.0.</p></section><section id="Sec23"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par60">The integration of peptide-level information through a pretrained protein language model significantly improves the accuracy of predicting translation initiation sites in eukaryotic mRNAs, achieving state-of-the-art performance. The success of NetStart 2.0 in leveraging protein-level context highlights the broader potential of protein language models in bridging nucleotide- and peptide information for diverse biological prediction tasks.</p></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366053/bin/12859_2025_6220_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary material 1.</a><sup> (1.5MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We would like to acknowledge Peter Wad Sackett for his help with setting up the NetStart 2.0 webserver.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>APS</dt>
<dd><p id="Par4">Average precision score</p></dd>
<dt>AUC</dt>
<dd><p id="Par5">Area under the receiver operating characteristic curve</p></dd>
<dt>BCE</dt>
<dd><p id="Par6">Binary cross entropy</p></dd>
<dt>CDS</dt>
<dd><p id="Par7">Coding sequence</p></dd>
<dt>ESM</dt>
<dd><p id="Par8">Evolutionary scale modeling</p></dd>
<dt>MCC</dt>
<dd><p id="Par9">Matthews correlation coefficient</p></dd>
<dt>mORF</dt>
<dd><p id="Par10">Main open reading frame</p></dd>
<dt>ORF</dt>
<dd><p id="Par11">Open reading frame</p></dd>
<dt>sORF</dt>
<dd><p id="Par12">Short open reading frame</p></dd>
<dt>TIS</dt>
<dd><p id="Par13">Translation initiation site</p></dd>
<dt>uORF</dt>
<dd><p id="Par14">Upstream open reading frame</p></dd>
<dt>UTR</dt>
<dd><p id="Par15">Untranslated region</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>H.N. proposed the conceptual framework. L.S.N. developed the source code and models. L.S.N. drafted the manuscript with support from H.N. and A.G.P. A.G.P., H.N., and O.W. supervised the project. All authors provided substantial inputs to and reviewed the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open access funding provided by Copenhagen University. L.S.N. and O.W. were in part funded by the Novo Nordisk Foundation through the Center for Basic Machine Learning Research in Life Science (NNF20OC0062606) and CAZAI (NNF22OC0077058). O.W. and L.S.N. acknowledge support from the Pioneer Center for AI, DNRF Grant Number P1.</p></section><section id="notes3"><h2 class="pmc_sec_title">Availability of data and materials</h2>
<p>The datasets supporting the conclusions of this article are available at the NetStart 2.0 Webserver site, accessible at: <a href="https://services.healthtech.dtu.dk/services/NetStart-2.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://services.healthtech.dtu.dk/services/NetStart-2.0/</a> (See the “Data” section at the webserver site). The raw datasets, comprised by the RefSeq genomes and corresponding annotations, were collected from the FTP server at NCBI: <a href="https://ftp.ncbi.nlm.nih.gov/genomes/refseq/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp.ncbi.nlm.nih.gov/genomes/refseq/</a>. Specifically, for each species the subpath./&lt;organism_group&gt;/&lt;genus_species&gt;was followed. The NetStart 2.0 online server is available at <a href="https://services.healthtech.dtu.dk/services/NetStart-2.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://services.healthtech.dtu.dk/services/NetStart-2.0/</a>. For large datasets, the program can be downloaded locally from github repository <a href="https://github.com/lsandvad/netstart2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/lsandvad/netstart2</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par61">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par62">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par63">The authors declare that they have no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Line Sandvad Nielsen, Email: line.s.nielsen@bio.ku.dk.</p>
<p>Henrik Nielsen, Email: henni@dtu.dk.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Kozak M. How do eucaryotic ribosomes select initiation regions in messenger RNA? Cell. 1978;15(4):1109–23. 10.1016/0092-8674(78)90039-9.
</cite> [<a href="https://doi.org/10.1016/0092-8674(78)90039-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/215319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kozak%20M.%20How%20do%20eucaryotic%20ribosomes%20select%20initiation%20regions%20in%20messenger%20RNA?%20Cell.%201978;15(4):1109%E2%80%9323.%2010.1016/0092-8674(78)90039-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Jackson RJ, Hellen CU, Pestova TV. The mechanism of eukaryotic translation initiation and principles of its regulation. Nat Rev Mol Cell Biol. 2010;11(2):113–27. 10.1038/nrm2838.
</cite> [<a href="https://doi.org/10.1038/nrm2838" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4461372/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20094052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jackson%20RJ,%20Hellen%20CU,%20Pestova%20TV.%20The%20mechanism%20of%20eukaryotic%20translation%20initiation%20and%20principles%20of%20its%20regulation.%20Nat%20Rev%20Mol%20Cell%20Biol.%202010;11(2):113%E2%80%9327.%2010.1038/nrm2838." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Kozak M. The scanning model for translation: an update. J Cell Biol. 1989;108(2):229–41. 10.1083/jcb.108.2.229.
</cite> [<a href="https://doi.org/10.1083/jcb.108.2.229" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2115416/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2645293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kozak%20M.%20The%20scanning%20model%20for%20translation:%20an%20update.%20J%20Cell%20Biol.%201989;108(2):229%E2%80%9341.%2010.1083/jcb.108.2.229." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Andreev DE, Loughran G, Fedorova AD, Mikhaylova MS, Shatsky IN, Baranov PV. Non-AUG translation initiation in mammals. Genome Biol. 2022;23(1):111. 10.1186/s13059-022-02674-2.
</cite> [<a href="https://doi.org/10.1186/s13059-022-02674-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9082881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35534899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andreev%20DE,%20Loughran%20G,%20Fedorova%20AD,%20Mikhaylova%20MS,%20Shatsky%20IN,%20Baranov%20PV.%20Non-AUG%20translation%20initiation%20in%20mammals.%20Genome%20Biol.%202022;23(1):111.%2010.1186/s13059-022-02674-2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Kozak M. Point mutations define a sequence flanking the AUG initiator codon that modulates translation by eukaryotic ribosomes. Cell. 1986;44(2):283–92. 10.1016/0092-8674(86)90762-2.
</cite> [<a href="https://doi.org/10.1016/0092-8674(86)90762-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3943125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kozak%20M.%20Point%20mutations%20define%20a%20sequence%20flanking%20the%20AUG%20initiator%20codon%20that%20modulates%20translation%20by%20eukaryotic%20ribosomes.%20Cell.%201986;44(2):283%E2%80%9392.%2010.1016/0092-8674(86)90762-2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Kozak M. Pushing the limits of the scanning mechanism for initiation of translation. Gene. 2002;299(1–2):1–34. 10.1016/S0378-1119(02)01056-9.
</cite> [<a href="https://doi.org/10.1016/S0378-1119(02)01056-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7126118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12459250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kozak%20M.%20Pushing%20the%20limits%20of%20the%20scanning%20mechanism%20for%20initiation%20of%20translation.%20Gene.%202002;299(1%E2%80%932):1%E2%80%9334.%2010.1016/S0378-1119(02)01056-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Hernández G, Osnaya VG, Pérez-Martínez X. Conservation and variability of the AUG initiation codon context in eukaryotes. Trends Biochem Sci. 2019;44(12):1009–21. 10.1016/j.tibs.2019.07.001.
</cite> [<a href="https://doi.org/10.1016/j.tibs.2019.07.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31353284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hern%C3%A1ndez%20G,%20Osnaya%20VG,%20P%C3%A9rez-Mart%C3%ADnez%20X.%20Conservation%20and%20variability%20of%20the%20AUG%20initiation%20codon%20context%20in%20eukaryotes.%20Trends%20Biochem%20Sci.%202019;44(12):1009%E2%80%9321.%2010.1016/j.tibs.2019.07.001." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Zhang H, Wang Y, Wu X, Tang X, Wu C, Lu J. Determinants of genome-wide distribution and evolution of uORFs in eukaryotes. Nat Commun. 2021;12(1):1076. 10.1038/s41467-021-21394-y.
</cite> [<a href="https://doi.org/10.1038/s41467-021-21394-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7889888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33597535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20H,%20Wang%20Y,%20Wu%20X,%20Tang%20X,%20Wu%20C,%20Lu%20J.%20Determinants%20of%20genome-wide%20distribution%20and%20evolution%20of%20uORFs%20in%20eukaryotes.%20Nat%20Commun.%202021;12(1):1076.%2010.1038/s41467-021-21394-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Miesfeld RL, McEvoy MM. Biochemistry of mRNA translation. In: Biochemistry, 1st ed. New York: W. W. Norton &amp; Company; 2017. pp. 1118–1119.</cite>
</li>
<li id="CR10">
<span class="label">10.</span><cite>Pisarev AV, Kolupaeva VG, Pisareva VP, Merrick WC, Hellen CU, Pestova TV. Specific functional interactions of nucleotides at key<img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/028b/12366053/b959e01a8315/12859_2025_6220_Article_IEq24.gif" loading="lazy" alt="Inline graphic"> 3 and + 4 positions flanking the initiation codon with components of the mammalian 48S translation initiation complex. Genes Dev. 2006;20(5):624–36. 10.1101/gad.1397906.
</cite> [<a href="https://doi.org/10.1101/gad.1397906" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1410799/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16510876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pisarev%20AV,%20Kolupaeva%20VG,%20Pisareva%20VP,%20Merrick%20WC,%20Hellen%20CU,%20Pestova%20TV.%20Specific%20functional%20interactions%20of%20nucleotides%20at%20key%203%20and%20+%204%20positions%20flanking%20the%20initiation%20codon%20with%20components%20of%20the%20mammalian%2048S%20translation%20initiation%20complex.%20Genes%20Dev.%202006;20(5):624%E2%80%9336.%2010.1101/gad.1397906." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Dever TE, Ivanov IP, Hinnebusch AG. Translational regulation by uORFs and start codon selection stringency. Genes Dev. 2023;37(11–12):474–89. 10.1101/gad.350752.123.
</cite> [<a href="https://doi.org/10.1101/gad.350752.123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10393191/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37433636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dever%20TE,%20Ivanov%20IP,%20Hinnebusch%20AG.%20Translational%20regulation%20by%20uORFs%20and%20start%20codon%20selection%20stringency.%20Genes%20Dev.%202023;37(11%E2%80%9312):474%E2%80%9389.%2010.1101/gad.350752.123." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Pedersen AG, Nielsen H. Neural network prediction of translation initiation sites in eukaryotes: perspectives for EST and genome analysis. In: Gaasterland T, Karp PD, Karplus K, Ouzounis C, Sander C, Valencia A, editors. Proceedings of the 5th international conference on intelligent systems for molecular biology. Washington, DC: AAAI Press; 1997. pp. 226–233.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/9322041/" class="usa-link">PubMed</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Nakagawa S, Niimura Y, Gojobori T, Tanaka H, Miura K. Diversity of preferred nucleotide sequences around the translation initiation codon in eukaryote genomes. Nucleic Acids Res. 2008;36(3):861–71. 10.1093/nar/gkm1102.
</cite> [<a href="https://doi.org/10.1093/nar/gkm1102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2241899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18086709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakagawa%20S,%20Niimura%20Y,%20Gojobori%20T,%20Tanaka%20H,%20Miura%20K.%20Diversity%20of%20preferred%20nucleotide%20sequences%20around%20the%20translation%20initiation%20codon%20in%20eukaryote%20genomes.%20Nucleic%20Acids%20Res.%202008;36(3):861%E2%80%9371.%2010.1093/nar/gkm1102." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Krebs JE, Goldstein ES, Kilpatrick ST. Translation. In: Lewin’s Genes XII, 12th ed. Burlington: Jones &amp; Bartlett Learning; 2018. pp. 592–596.</cite>
</li>
<li id="CR15">
<span class="label">15.</span><cite>Xu H, Wang P, Fu Y, Zheng Y, Tang Q, Si L, et al. Length of the ORF, position of the first AUG and the Kozak motif are important factors in potential dual-coding transcripts. Cell Res. 2010;20(4):445–57. 10.1038/cr.2010.25.
</cite> [<a href="https://doi.org/10.1038/cr.2010.25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20157333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20H,%20Wang%20P,%20Fu%20Y,%20Zheng%20Y,%20Tang%20Q,%20Si%20L,%20et%20al.%20Length%20of%20the%20ORF,%20position%20of%20the%20first%20AUG%20and%20the%20Kozak%20motif%20are%20important%20factors%20in%20potential%20dual-coding%20transcripts.%20Cell%20Res.%202010;20(4):445%E2%80%9357.%2010.1038/cr.2010.25." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Sayers EW, Cavanaugh M, Clark K, Pruitt KD, Sherry ST, Yankie L, et al. GenBank 2024 update. Nucleic Acids Res. 2023;52(D1):D134–7. 10.1093/nar/gkad903.</cite> [<a href="https://doi.org/10.1093/nar/gkad903" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767886/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37889039/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sayers%20EW,%20Cavanaugh%20M,%20Clark%20K,%20Pruitt%20KD,%20Sherry%20ST,%20Yankie%20L,%20et%20al.%20GenBank%202024%20update.%20Nucleic%20Acids%20Res.%202023;52(D1):D134%E2%80%937.%2010.1093/nar/gkad903." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Goel N, Singh S, Aseri TC. Global sequence features based translation initiation site prediction in human genomic sequences. Heliyon. 2020;6(9):e04825. 10.1016/j.heliyon.2020.e04825.
</cite> [<a href="https://doi.org/10.1016/j.heliyon.2020.e04825" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7490824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32964155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goel%20N,%20Singh%20S,%20Aseri%20TC.%20Global%20sequence%20features%20based%20translation%20initiation%20site%20prediction%20in%20human%20genomic%20sequences.%20Heliyon.%202020;6(9):e04825.%2010.1016/j.heliyon.2020.e04825." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Xiang Y, Huang W, Tan L, Chen T, He Y, Irving PS, et al. Pervasive downstream RNA hairpins dynamically dictate start-codon selection. Nature. 2023;621(7978):423–30. 10.1038/s41586-023-06500-y.
</cite> [<a href="https://doi.org/10.1038/s41586-023-06500-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10499604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37674078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiang%20Y,%20Huang%20W,%20Tan%20L,%20Chen%20T,%20He%20Y,%20Irving%20PS,%20et%20al.%20Pervasive%20downstream%20RNA%20hairpins%20dynamically%20dictate%20start-codon%20selection.%20Nature.%202023;621(7978):423%E2%80%9330.%2010.1038/s41586-023-06500-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Cao X, Slavoff SA. Non-AUG start codons: expanding and regulating the small and alternative ORFeome. Exp Cell Res. 2020;391(1):111973. 10.1016/j.yexcr.2020.111973.
</cite> [<a href="https://doi.org/10.1016/j.yexcr.2020.111973" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7256928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32209305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cao%20X,%20Slavoff%20SA.%20Non-AUG%20start%20codons:%20expanding%20and%20regulating%20the%20small%20and%20alternative%20ORFeome.%20Exp%20Cell%20Res.%202020;391(1):111973.%2010.1016/j.yexcr.2020.111973." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Clauwaert J, McVey Z, Gupta R, Menschaert G. TIS transformer: remapping the human proteome using deep learning. NAR Genomics Bioinform. 2023;5(1):lqad021. 10.1093/nargab/lqad021.</cite> [<a href="https://doi.org/10.1093/nargab/lqad021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9985340/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36879896/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clauwaert%20J,%20McVey%20Z,%20Gupta%20R,%20Menschaert%20G.%20TIS%20transformer:%20remapping%20the%20human%20proteome%20using%20deep%20learning.%20NAR%20Genomics%20Bioinform.%202023;5(1):lqad021.%2010.1093/nargab/lqad021." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Jin Y, Ivanov M, Dittrich AN, Nelson AD, Marquardt S. LncRNA FLAIL affects alternative splicing and represses flowering in Arabidopsis. EMBO J. 2023;42(11):e110921. 10.15252/embj.2022110921.
</cite> [<a href="https://doi.org/10.15252/embj.2022110921" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10233378/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37051749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin%20Y,%20Ivanov%20M,%20Dittrich%20AN,%20Nelson%20AD,%20Marquardt%20S.%20LncRNA%20FLAIL%20affects%20alternative%20splicing%20and%20represses%20flowering%20in%20Arabidopsis.%20EMBO%20J.%202023;42(11):e110921.%2010.15252/embj.2022110921." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Frikstad K, Molinari E, Thoresen M, Ramsbottom SA, Hughes F, Letteboer SJ, et al. A CEP104-CSPP1 complex is required for formation of primary cilia competent in hedgehog signaling. Cell Rep. 2019;28(7):1907–22. 10.1016/j.celrep.2019.07.025.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2019.07.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6702141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31412255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Frikstad%20K,%20Molinari%20E,%20Thoresen%20M,%20Ramsbottom%20SA,%20Hughes%20F,%20Letteboer%20SJ,%20et%20al.%20A%20CEP104-CSPP1%20complex%20is%20required%20for%20formation%20of%20primary%20cilia%20competent%20in%20hedgehog%20signaling.%20Cell%20Rep.%202019;28(7):1907%E2%80%9322.%2010.1016/j.celrep.2019.07.025." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Fuster-García C, García-García G, Jaijo T, Fornés N, Ayuso C, Fernández-Burriel M, et al. High-throughput sequencing for the molecular diagnosis of Usher syndrome reveals 42 novel mutations and consolidates CEP250 as Usher-like disease causative. Sci Rep. 2018;8(1):17113. 10.1038/s41598-018-35085-0.
</cite> [<a href="https://doi.org/10.1038/s41598-018-35085-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6244211/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30459346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fuster-Garc%C3%ADa%20C,%20Garc%C3%ADa-Garc%C3%ADa%20G,%20Jaijo%20T,%20Forn%C3%A9s%20N,%20Ayuso%20C,%20Fern%C3%A1ndez-Burriel%20M,%20et%20al.%20High-throughput%20sequencing%20for%20the%20molecular%20diagnosis%20of%20Usher%20syndrome%20reveals%2042%20novel%20mutations%20and%20consolidates%20CEP250%20as%20Usher-like%20disease%20causative.%20Sci%20Rep.%202018;8(1):17113.%2010.1038/s41598-018-35085-0." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Williams JL, Paudyal A, Awad S, Nicholson J, Grzesik D, Botta J, et al. Mylk3 null C57BL/6N mice develop cardiomyopathy, whereas Nnt null C57BL/6J mice do not. Life Sci Alliance. 2020. 10.26508/lsa.201900593.
</cite> [<a href="https://doi.org/10.26508/lsa.201900593" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7103425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32213617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Williams%20JL,%20Paudyal%20A,%20Awad%20S,%20Nicholson%20J,%20Grzesik%20D,%20Botta%20J,%20et%20al.%20Mylk3%20null%20C57BL/6N%20mice%20develop%20cardiomyopathy,%20whereas%20Nnt%20null%20C57BL/6J%20mice%20do%20not.%20Life%20Sci%20Alliance.%202020.%2010.26508/lsa.201900593." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Lenglez S, Sablon A, Fénelon G, Boland A, Deleuze JF, Boutoleau-Bretonnière C, et al. Distinct functional classes of PDGFRB pathogenic variants in primary familial brain calcification. Hum Mol Genet. 2022;31(3):399–409. 10.1093/hmg/ddab258.
</cite> [<a href="https://doi.org/10.1093/hmg/ddab258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34494111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lenglez%20S,%20Sablon%20A,%20F%C3%A9nelon%20G,%20Boland%20A,%20Deleuze%20JF,%20Boutoleau-Bretonni%C3%A8re%20C,%20et%20al.%20Distinct%20functional%20classes%20of%20PDGFRB%20pathogenic%20variants%20in%20primary%20familial%20brain%20calcification.%20Hum%20Mol%20Genet.%202022;31(3):399%E2%80%93409.%2010.1093/hmg/ddab258." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Jankovic B, Gojobori T. From shallow to deep: some lessons learned from application of machine learning for recognition of functional genomic elements in human genome. Hum Genomics. 2022;16(1):7. 10.1186/s40246-022-00376-1.
</cite> [<a href="https://doi.org/10.1186/s40246-022-00376-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8855580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35180894/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jankovic%20B,%20Gojobori%20T.%20From%20shallow%20to%20deep:%20some%20lessons%20learned%20from%20application%20of%20machine%20learning%20for%20recognition%20of%20functional%20genomic%20elements%20in%20human%20genome.%20Hum%20Genomics.%202022;16(1):7.%2010.1186/s40246-022-00376-1." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Kalkatawi M, Magana-Mora A, Jankovic B, Bajic VB. DeepGSR: an optimized deep-learning structure for the recognition of genomic signals and regions. Bioinformatics. 2019;35(7):1125–32. 10.1093/bioinformatics/bty752.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty752" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6449759/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30184052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kalkatawi%20M,%20Magana-Mora%20A,%20Jankovic%20B,%20Bajic%20VB.%20DeepGSR:%20an%20optimized%20deep-learning%20structure%20for%20the%20recognition%20of%20genomic%20signals%20and%20regions.%20Bioinformatics.%202019;35(7):1125%E2%80%9332.%2010.1093/bioinformatics/bty752." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Zhang S, Hu H, Jiang T, Zhang L, Zeng J. TITER: predicting translation initiation sites by deep learning. Bioinformatics. 2017;33(14):i234–42. 10.1093/bioinformatics/btx247.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5870772/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28881981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20S,%20Hu%20H,%20Jiang%20T,%20Zhang%20L,%20Zeng%20J.%20TITER:%20predicting%20translation%20initiation%20sites%20by%20deep%20learning.%20Bioinformatics.%202017;33(14):i234%E2%80%9342.%2010.1093/bioinformatics/btx247." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Liu Q, Fang H, Wang X, Wang M, Li S, Coin LJ, et al. DeepGenGrep: a general deep learning-based predictor for multiple genomic signals and regions. Bioinformatics. 2022;38(17):4053–61. 10.1093/bioinformatics/btac454.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btac454" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35799358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20Q,%20Fang%20H,%20Wang%20X,%20Wang%20M,%20Li%20S,%20Coin%20LJ,%20et%20al.%20DeepGenGrep:%20a%20general%20deep%20learning-based%20predictor%20for%20multiple%20genomic%20signals%20and%20regions.%20Bioinformatics.%202022;38(17):4053%E2%80%9361.%2010.1093/bioinformatics/btac454." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Eraslan G, Avsec Ž, Gagneur J, Theis FJ. Deep learning: new computational modelling techniques for genomics. Nat Rev Genet. 2019;20(7):389–403. 10.1038/s41576-019-0122-6.
</cite> [<a href="https://doi.org/10.1038/s41576-019-0122-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30971806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eraslan%20G,%20Avsec%20%C5%BD,%20Gagneur%20J,%20Theis%20FJ.%20Deep%20learning:%20new%20computational%20modelling%20techniques%20for%20genomics.%20Nat%20Rev%20Genet.%202019;20(7):389%E2%80%93403.%2010.1038/s41576-019-0122-6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Stanke M, Steinkamp R, Waack S, Morgenstern B. AUGUSTUS: a web server for gene finding in eukaryotes. Nucleic Acids Res. 2004;32(suppl-2):W309–12. 10.1093/nar/gkh379.
</cite> [<a href="https://doi.org/10.1093/nar/gkh379" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC441517/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15215400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stanke%20M,%20Steinkamp%20R,%20Waack%20S,%20Morgenstern%20B.%20AUGUSTUS:%20a%20web%20server%20for%20gene%20finding%20in%20eukaryotes.%20Nucleic%20Acids%20Res.%202004;32(suppl-2):W309%E2%80%9312.%2010.1093/nar/gkh379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Stanke M, Morgenstern B. AUGUSTUS: a web server for gene prediction in eukaryotes that allows user-defined constraints. Nucleic Acids Res. 2005;33(suppl-2):W465–7. 10.1093/nar/gki458.
</cite> [<a href="https://doi.org/10.1093/nar/gki458" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1160219/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15980513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stanke%20M,%20Morgenstern%20B.%20AUGUSTUS:%20a%20web%20server%20for%20gene%20prediction%20in%20eukaryotes%20that%20allows%20user-defined%20constraints.%20Nucleic%20Acids%20Res.%202005;33(suppl-2):W465%E2%80%937.%2010.1093/nar/gki458." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Scalzitti N, Jeannin-Girardon A, Collet P, Poch O, Thompson JD. A benchmark study of ab initio gene prediction methods in diverse eukaryotic organisms. BMC Genomics. 2020;21:1–20. 10.1186/s12864-020-6707-9.</cite> [<a href="https://doi.org/10.1186/s12864-020-6707-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7147072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32272892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scalzitti%20N,%20Jeannin-Girardon%20A,%20Collet%20P,%20Poch%20O,%20Thompson%20JD.%20A%20benchmark%20study%20of%20ab%20initio%20gene%20prediction%20methods%20in%20diverse%20eukaryotic%20organisms.%20BMC%20Genomics.%202020;21:1%E2%80%9320.%2010.1186/s12864-020-6707-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Hoff KJ, Stanke M. WebAUGUSTUS—a web service for training AUGUSTUS and predicting genes in eukaryotes. Nucleic Acids Res. 2013;41(W1):W123–8. 10.1093/nar/gkt418.
</cite> [<a href="https://doi.org/10.1093/nar/gkt418" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3692069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23700307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoff%20KJ,%20Stanke%20M.%20WebAUGUSTUS%E2%80%94a%20web%20service%20for%20training%20AUGUSTUS%20and%20predicting%20genes%20in%20eukaryotes.%20Nucleic%20Acids%20Res.%202013;41(W1):W123%E2%80%938.%2010.1093/nar/gkt418." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Stanke M, Keller O, Gunduz I, Hayes A, Waack S, Morgenstern B. AUGUSTUS: ab initio prediction of alternative transcripts. Nucleic Acids Res. 2006;34(suppl-2):W435–9. 10.1093/nar/gkl200.
</cite> [<a href="https://doi.org/10.1093/nar/gkl200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1538822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16845043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stanke%20M,%20Keller%20O,%20Gunduz%20I,%20Hayes%20A,%20Waack%20S,%20Morgenstern%20B.%20AUGUSTUS:%20ab%20initio%20prediction%20of%20alternative%20transcripts.%20Nucleic%20Acids%20Res.%202006;34(suppl-2):W435%E2%80%939.%2010.1093/nar/gkl200." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Gabriel L, Becker F, Hoff KJ, Stanke M. Tiberius: end-to-end deep learning with an HMM for gene prediction. Bioinformatics. 2024;40(12):btae685. 10.1093/bioinformatics/btae685.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btae685" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11645249/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39558581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gabriel%20L,%20Becker%20F,%20Hoff%20KJ,%20Stanke%20M.%20Tiberius:%20end-to-end%20deep%20learning%20with%20an%20HMM%20for%20gene%20prediction.%20Bioinformatics.%202024;40(12):btae685.%2010.1093/bioinformatics/btae685." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Khurana D, Koli A, Khatter K, Singh S. Natural language processing: state of the art, current trends and challenges. Multimed Tools Appl. 2023;82(3):3713–44. 10.1007/s11042-022-13428-4.
</cite> [<a href="https://doi.org/10.1007/s11042-022-13428-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9281254/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35855771/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khurana%20D,%20Koli%20A,%20Khatter%20K,%20Singh%20S.%20Natural%20language%20processing:%20state%20of%20the%20art,%20current%20trends%20and%20challenges.%20Multimed%20Tools%20Appl.%202023;82(3):3713%E2%80%9344.%2010.1007/s11042-022-13428-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Elnaggar A, Heinzinger M, Dallago C, Rehawi G, Wang Y, Jones L, et al. ProtTrans: toward understanding the language of life through self-supervised learning. IEEE Trans Pattern Anal Mach Intell. 2021;44(10):7112–27. 10.1109/TPAMI.2021.3095381.</cite> [<a href="https://doi.org/10.1109/TPAMI.2021.3095381" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34232869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elnaggar%20A,%20Heinzinger%20M,%20Dallago%20C,%20Rehawi%20G,%20Wang%20Y,%20Jones%20L,%20et%20al.%20ProtTrans:%20toward%20understanding%20the%20language%20of%20life%20through%20self-supervised%20learning.%20IEEE%20Trans%20Pattern%20Anal%20Mach%20Intell.%202021;44(10):7112%E2%80%9327.%2010.1109/TPAMI.2021.3095381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Lin Z, Akin H, Rao R, Hie B, Zhu Z, Lu W, et al. Evolutionary-scale prediction of atomic-level protein structure with a language model. Science. 2023;379(6637):1123–30. 10.1126/science.ade2574.
</cite> [<a href="https://doi.org/10.1126/science.ade2574" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36927031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20Z,%20Akin%20H,%20Rao%20R,%20Hie%20B,%20Zhu%20Z,%20Lu%20W,%20et%20al.%20Evolutionary-scale%20prediction%20of%20atomic-level%20protein%20structure%20with%20a%20language%20model.%20Science.%202023;379(6637):1123%E2%80%9330.%2010.1126/science.ade2574." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Vaswani A, Shazeer N, Parmar N, Uszkoreit J, Jones L, Gomez AN, et al. Attention is all you need. In: Guyon I, von Luxburg U, Bengio S, Wallach H, Fergus S, Vishwanathan S, et al., editors. Advances in neural information processing systems, vol. 30. Red Hook: Curran Associates, Inc.; 2017. Available from: <a href="https://proceedings.neurips.cc/paper/2017/hash/3f5ee243547dee91fbd053c1c4a845aa-Abstract.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://proceedings.neurips.cc/paper/2017/hash/3f5ee243547dee91fbd053c1c4a845aa-Abstract.html</a>.</cite>
</li>
<li id="CR41">
<span class="label">41.</span><cite>Brandes N, Ofer D, Peleg Y, Rappoport N, Linial M. ProteinBERT: a universal deep-learning model of protein sequence and function. Bioinformatics. 2022;38(8):2102–10. 10.1093/bioinformatics/btac020.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btac020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9386727/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35020807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brandes%20N,%20Ofer%20D,%20Peleg%20Y,%20Rappoport%20N,%20Linial%20M.%20ProteinBERT:%20a%20universal%20deep-learning%20model%20of%20protein%20sequence%20and%20function.%20Bioinformatics.%202022;38(8):2102%E2%80%9310.%2010.1093/bioinformatics/btac020." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>O’Leary NA, Wright MW, Brister JR, Ciufo S, Haddad D, McVeigh R, et al. Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation. Nucleic Acids Res. 2015;44(D1):D733–45. 10.1093/nar/gkv1189.
</cite> [<a href="https://doi.org/10.1093/nar/gkv1189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4702849/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26553804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?O%E2%80%99Leary%20NA,%20Wright%20MW,%20Brister%20JR,%20Ciufo%20S,%20Haddad%20D,%20McVeigh%20R,%20et%20al.%20Reference%20sequence%20(RefSeq)%20database%20at%20NCBI:%20current%20status,%20taxonomic%20expansion,%20and%20functional%20annotation.%20Nucleic%20Acids%20Res.%202015;44(D1):D733%E2%80%9345.%2010.1093/nar/gkv1189." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Thibaud-Nissen F, Souvorov A, Murphy T, DiCuccio M, Kitts P. Eukaryotic genome annotation pipeline. In: The NCBI handbook [Internet], 2nd ed. Bethesda: National Center for Biotechnology Information (US); 2013. Available from: <a href="https://www.ncbi.nlm.nih.gov/books/NBK169439/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/books/NBK169439/</a>.</cite>
</li>
<li id="CR44">
<span class="label">44.</span><cite>Souvorov A, Kapustin Y, Kiryutin B, Chetvernin V, Tatusova T, Lipman D. Gnomon—NCBI eukaryotic gene prediction tool. National Center for Biotechnology Information; 2010. Available from: <a href="https://www.ncbi.nlm.nih.gov/core/assets/genome/files/Gnomon-description.pdf" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/core/assets/genome/files/Gnomon-description.pdf</a>.</cite>
</li>
<li id="CR45">
<span class="label">45.</span><cite>Teufel F, Gíslason MH, Almagro Armenteros JJ, Johansen AR, Winther O, Nielsen H. GraphPart: homology partitioning for biological sequence analysis. NAR Genomics Bioinform. 2023;5(4):lqad088. 10.1093/nargab/lqad088.</cite> [<a href="https://doi.org/10.1093/nargab/lqad088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10578201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37850036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Teufel%20F,%20G%C3%ADslason%20MH,%20Almagro%20Armenteros%20JJ,%20Johansen%20AR,%20Winther%20O,%20Nielsen%20H.%20GraphPart:%20homology%20partitioning%20for%20biological%20sequence%20analysis.%20NAR%20Genomics%20Bioinform.%202023;5(4):lqad088.%2010.1093/nargab/lqad088." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Steinegger M, Söding J. MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets. Nat Biotechnol. 2017;35(11):1026–8. 10.1038/nbt.3988.
</cite> [<a href="https://doi.org/10.1038/nbt.3988" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29035372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Steinegger%20M,%20S%C3%B6ding%20J.%20MMseqs2%20enables%20sensitive%20protein%20sequence%20searching%20for%20the%20analysis%20of%20massive%20data%20sets.%20Nat%20Biotechnol.%202017;35(11):1026%E2%80%938.%2010.1038/nbt.3988." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Teufel F, Stahlhut C, Refsgaard J, Nielsen H, Winther O, Madsen D. SecretoGen: towards prediction of signal peptides for efficient protein secretion. In: NeurIPS 2023 generative AI and biology (GenBio) workshop; 2023. Available from: <a href="https://openreview.net/forum?id=vXXEfmYsvS" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://openreview.net/forum?id=vXXEfmYsvS</a>.</cite>
</li>
<li id="CR48">
<span class="label">48.</span><cite>Schoch CL, Ciufo S, Domrachev M, Hotton CL, Kannan S, Khovanskaya R, et al. NCBI taxonomy: a comprehensive update on curation, resources and tools. Database. 2020;2020:baaa062. 10.1093/database/baaa062.
</cite> [<a href="https://doi.org/10.1093/database/baaa062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7408187/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32761142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schoch%20CL,%20Ciufo%20S,%20Domrachev%20M,%20Hotton%20CL,%20Kannan%20S,%20Khovanskaya%20R,%20et%20al.%20NCBI%20taxonomy:%20a%20comprehensive%20update%20on%20curation,%20resources%20and%20tools.%20Database.%202020;2020:baaa062.%2010.1093/database/baaa062." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Dalla-Torre H, Gonzalez L, Mendoza-Revilla J, Lopez Carranza N, Grzywaczewski AH, Oteri F, et al. Nucleotide transformer: building and evaluating robust foundation models for human genomics. Nat Methods. 2024. 10.1038/s41592-024-02523-z.
</cite> [<a href="https://doi.org/10.1038/s41592-024-02523-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11810778/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39609566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dalla-Torre%20H,%20Gonzalez%20L,%20Mendoza-Revilla%20J,%20Lopez%20Carranza%20N,%20Grzywaczewski%20AH,%20Oteri%20F,%20et%20al.%20Nucleotide%20transformer:%20building%20and%20evaluating%20robust%20foundation%20models%20for%20human%20genomics.%20Nat%20Methods.%202024.%2010.1038/s41592-024-02523-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>ESM. Documentation of ESM-2 from HuggingFace. Available from: <a href="https://huggingface.co/docs/transformers/v4.52.2/en/model_doc/esm#transformers" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://huggingface.co/docs/transformers/v4.52.2/en/model_doc/esm#transformers</a>.</cite>
</li>
<li id="CR51">
<span class="label">51.</span><cite>Kingma DP. Adam: a method for stochastic optimization. arXiv preprint <a href="http://arxiv.org/abs/1412.6980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">arXiv:1412.6980</a>. 2014. 10.48550/arXiv.1412.6980.</cite>
</li>
<li id="CR52">
<span class="label">52.</span><cite>Prechelt L. Early stopping—But when? In: Orr GB, Müller KR, editors. Neural networks: tricks of the trade. Berlin: Springer; 2002. p. 55–69.</cite> [<a href="https://scholar.google.com/scholar_lookup?Prechelt%20L.%20Early%20stopping%E2%80%94But%20when?%20In:%20Orr%20GB,%20M%C3%BCller%20KR,%20editors.%20Neural%20networks:%20tricks%20of%20the%20trade.%20Berlin:%20Springer;%202002.%20p.%2055%E2%80%9369." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Srivastava N, Hinton G, Krizhevsky A, Sutskever I, Salakhutdinov R. Dropout: a simple way to prevent neural networks from overfitting. J Mach Learn Res. 2014;15(1):1929–58.</cite> [<a href="https://scholar.google.com/scholar_lookup?Srivastava%20N,%20Hinton%20G,%20Krizhevsky%20A,%20Sutskever%20I,%20Salakhutdinov%20R.%20Dropout:%20a%20simple%20way%20to%20prevent%20neural%20networks%20from%20overfitting.%20J%20Mach%20Learn%20Res.%202014;15(1):1929%E2%80%9358." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Morgulis A, Gertz EM, Schäffer AA, Agarwala R. WindowMasker: window-based masker for sequenced genomes. Bioinformatics. 2005;22(2):134–41. 10.1093/bioinformatics/bti774.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bti774" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16287941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morgulis%20A,%20Gertz%20EM,%20Sch%C3%A4ffer%20AA,%20Agarwala%20R.%20WindowMasker:%20window-based%20masker%20for%20sequenced%20genomes.%20Bioinformatics.%202005;22(2):134%E2%80%9341.%2010.1093/bioinformatics/bti774." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Zhi D, Raphael BJ, Price AL, Tang H, Pevzner PA. Identifying repeat domains in large genomes. Genome Biol. 2006;7:1–14. 10.1186/gb-2006-7-1-r7.</cite> [<a href="https://doi.org/10.1186/gb-2006-7-1-r7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1431705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16507140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhi%20D,%20Raphael%20BJ,%20Price%20AL,%20Tang%20H,%20Pevzner%20PA.%20Identifying%20repeat%20domains%20in%20large%20genomes.%20Genome%20Biol.%202006;7:1%E2%80%9314.%2010.1186/gb-2006-7-1-r7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Le N, Yapp E, Nagasundaram N, Yeh HY. Classifying promoters by interpreting the hidden information of DNA sequences via deep learning and combination of continuous fasttext N-grams. Front Bioeng Biotechnol. 2019;7:305. 10.3389/fbioe.2019.00305.
</cite> [<a href="https://doi.org/10.3389/fbioe.2019.00305" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6848157/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31750297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Le%20N,%20Yapp%20E,%20Nagasundaram%20N,%20Yeh%20HY.%20Classifying%20promoters%20by%20interpreting%20the%20hidden%20information%20of%20DNA%20sequences%20via%20deep%20learning%20and%20combination%20of%20continuous%20fasttext%20N-grams.%20Front%20Bioeng%20Biotechnol.%202019;7:305.%2010.3389/fbioe.2019.00305." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Richardson E, Trevizani R, Greenbaum JA, Carter H, Nielsen M, Peters B. The receiver operating characteristic curve accurately assesses imbalanced datasets. Patterns. 2024. 10.1016/j.patter.2024.100994.
</cite> [<a href="https://doi.org/10.1016/j.patter.2024.100994" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11240176/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39005487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Richardson%20E,%20Trevizani%20R,%20Greenbaum%20JA,%20Carter%20H,%20Nielsen%20M,%20Peters%20B.%20The%20receiver%20operating%20characteristic%20curve%20accurately%20assesses%20imbalanced%20datasets.%20Patterns.%202024.%2010.1016/j.patter.2024.100994." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Conte AD, Mehdiabadi M, Bouhraoua A, Miguel Monzon A, Tosatto SC, Piovesan D. Critical assessment of protein intrinsic disorder prediction (CAID)-Results of round 2. Proteins: structure. Funct Bioinform. 2023;91(12):1925–34.</cite> [<a href="https://doi.org/10.1002/prot.26582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37621223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Conte%20AD,%20Mehdiabadi%20M,%20Bouhraoua%20A,%20Miguel%20Monzon%20A,%20Tosatto%20SC,%20Piovesan%20D.%20Critical%20assessment%20of%20protein%20intrinsic%20disorder%20prediction%20(CAID)-Results%20of%20round%202.%20Proteins:%20structure.%20Funct%20Bioinform.%202023;91(12):1925%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Keilwagen J, Grosse I, Grau J. Area under precision-recall curves for weighted and unweighted data. PLoS ONE. 2014;9(3):e92209. 10.1371/journal.pone.0092209.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0092209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3961324/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24651729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keilwagen%20J,%20Grosse%20I,%20Grau%20J.%20Area%20under%20precision-recall%20curves%20for%20weighted%20and%20unweighted%20data.%20PLoS%20ONE.%202014;9(3):e92209.%2010.1371/journal.pone.0092209." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Günzl A. The pre-mRNA splicing machinery of trypanosomes: Complex or simplified? Eukaryot Cell. 2010;9(8):1159–70. 10.1128/ec.00113-10.
</cite> [<a href="https://doi.org/10.1128/EC.00113-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2918933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20581293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?G%C3%BCnzl%20A.%20The%20pre-mRNA%20splicing%20machinery%20of%20trypanosomes:%20Complex%20or%20simplified?%20Eukaryot%20Cell.%202010;9(8):1159%E2%80%9370.%2010.1128/ec.00113-10." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Grünebast J, Clos J. Leishmania: responding to environmental signals and challenges without regulated transcription. Comput Struct Biotechnol J. 2020;18:4016–23. 10.1016/j.csbj.2020.11.058.
</cite> [<a href="https://doi.org/10.1016/j.csbj.2020.11.058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7744640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33363698/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gr%C3%BCnebast%20J,%20Clos%20J.%20Leishmania:%20responding%20to%20environmental%20signals%20and%20challenges%20without%20regulated%20transcription.%20Comput%20Struct%20Biotechnol%20J.%202020;18:4016%E2%80%9323.%2010.1016/j.csbj.2020.11.058." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Schmirler R, Heinzinger M, Rost B. Fine-tuning protein language models boosts predictions across diverse tasks. Nat Commun. 2024;15(1):7407. 10.1038/s41467-024-51844-2.
</cite> [<a href="https://doi.org/10.1038/s41467-024-51844-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11358375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39198457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schmirler%20R,%20Heinzinger%20M,%20Rost%20B.%20Fine-tuning%20protein%20language%20models%20boosts%20predictions%20across%20diverse%20tasks.%20Nat%20Commun.%202024;15(1):7407.%2010.1038/s41467-024-51844-2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366053/bin/12859_2025_6220_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary material 1.</a><sup> (1.5MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets supporting the conclusions of this article are available at the NetStart 2.0 Webserver site, accessible at: <a href="https://services.healthtech.dtu.dk/services/NetStart-2.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://services.healthtech.dtu.dk/services/NetStart-2.0/</a> (See the “Data” section at the webserver site). The raw datasets, comprised by the RefSeq genomes and corresponding annotations, were collected from the FTP server at NCBI: <a href="https://ftp.ncbi.nlm.nih.gov/genomes/refseq/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp.ncbi.nlm.nih.gov/genomes/refseq/</a>. Specifically, for each species the subpath./&lt;organism_group&gt;/&lt;genus_species&gt;was followed. The NetStart 2.0 online server is available at <a href="https://services.healthtech.dtu.dk/services/NetStart-2.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://services.healthtech.dtu.dk/services/NetStart-2.0/</a>. For large datasets, the program can be downloaded locally from github repository <a href="https://github.com/lsandvad/netstart2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/lsandvad/netstart2</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Bioinformatics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12859-025-06220-2"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12859_2025_Article_6220.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366053/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366053/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366053%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366053/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366053/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366053/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830753/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366053/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830753/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366053/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366053/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="mvkbA6LJRHZunrZOCtVgZmcLWrt0L3VM4KCSe9SIyQu2yjiWRsxoyeiicPpJ00U7">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
