
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Better together: Subgenomes for allotetraploid potato wild relative Solanum acaule Bitt. reveal origins in Petota Clade 3 and 4 - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5311378AEF1243051137000CBFE9FF.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368328/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Better together: Subgenomes for allotetraploid potato wild relative Solanum acaule Bitt. reveal origins in Petota Clade 3 and 4">
<meta name="citation_author" content="S Achakkagari">
<meta name="citation_author_institution" content="Department of Plant Science, McGill University, Sainte‐Anne‐de‐Bellevue, Quebec, Canada">
<meta name="citation_author" content="J C Camargo‐Tavares">
<meta name="citation_author_institution" content="Department of Plant Science, McGill University, Sainte‐Anne‐de‐Bellevue, Quebec, Canada">
<meta name="citation_author" content="D Praslickova">
<meta name="citation_author_institution" content="Department of Plant Science, McGill University, Sainte‐Anne‐de‐Bellevue, Quebec, Canada">
<meta name="citation_author" content="C Martini">
<meta name="citation_author_institution" content="Department of Plant Science, McGill University, Sainte‐Anne‐de‐Bellevue, Quebec, Canada">
<meta name="citation_author" content="B Bizimungu">
<meta name="citation_author_institution" content="Agriculture and Agri‐Food Canada Fredericton Research and Development Centre, Fredericton, New Brunswick, Canada">
<meta name="citation_author" content="N L Anglin">
<meta name="citation_author_institution" content="International Potato Center (CIP), Lima, Peru">
<meta name="citation_author_institution" content="USDA ARS Small Grains and Potato Germplasm Research, Aberdeen, Idaho, USA">
<meta name="citation_author" content="N Manrique‐Carpintero">
<meta name="citation_author_institution" content="International Potato Center (CIP), Lima, Peru">
<meta name="citation_author" content="H Lindqvist‐Kreuze">
<meta name="citation_author_institution" content="International Potato Center (CIP), Lima, Peru">
<meta name="citation_author" content="H H Tai">
<meta name="citation_author_institution" content="Agriculture and Agri‐Food Canada Fredericton Research and Development Centre, Fredericton, New Brunswick, Canada">
<meta name="citation_author" content="M V Strömvik">
<meta name="citation_author_institution" content="Department of Plant Science, McGill University, Sainte‐Anne‐de‐Bellevue, Quebec, Canada">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70095">
<meta name="citation_doi" content="10.1002/tpg2.70095">
<meta name="citation_pmid" content="40836401">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368328/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368328/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368328/pdf/TPG2-18-e70095.pdf">
<meta name="description" content="Potato wild relatives (Solanum section Petota) are a source of genetic diversity for climate‐smart improvement of modern cultivars (Solanum tuberosum). The allotetraploid Solanum acaule Bitter. has been previously used to introgress frost tolerance ...">
<meta name="og:title" content="Better together: Subgenomes for allotetraploid potato wild relative Solanum acaule Bitt. reveal origins in Petota Clade 3 and 4">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Potato wild relatives (Solanum section Petota) are a source of genetic diversity for climate‐smart improvement of modern cultivars (Solanum tuberosum). The allotetraploid Solanum acaule Bitter. has been previously used to introgress frost tolerance ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368328/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368328">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70095"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70095.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368328%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368328/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368328/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368328/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70095" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Aug 20;18(3):e70095. doi: <a href="https://doi.org/10.1002/tpg2.70095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70095</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Better together: Subgenomes for allotetraploid potato wild relative <em>Solanum acaule</em> Bitt. reveal origins in <em>Petota</em> Clade 3 and 4</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Achakkagari%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">S Achakkagari</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">S Achakkagari</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Science, McGill University, Sainte‐Anne‐de‐Bellevue, Quebec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Achakkagari%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">S Achakkagari</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Camargo%E2%80%90Tavares%20JC%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">J C Camargo‐Tavares</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">J C Camargo‐Tavares</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Science, McGill University, Sainte‐Anne‐de‐Bellevue, Quebec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Camargo%E2%80%90Tavares%20JC%22%5BAuthor%5D" class="usa-link"><span class="name western">J C Camargo‐Tavares</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Praslickova%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">D Praslickova</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">D Praslickova</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Science, McGill University, Sainte‐Anne‐de‐Bellevue, Quebec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Praslickova%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">D Praslickova</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Martini%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">C Martini</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">C Martini</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Science, McGill University, Sainte‐Anne‐de‐Bellevue, Quebec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Martini%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">C Martini</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bizimungu%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">B Bizimungu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">B Bizimungu</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Agriculture and Agri‐Food Canada Fredericton Research and Development Centre, Fredericton, New Brunswick, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bizimungu%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">B Bizimungu</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anglin%20NL%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">N L Anglin</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">N L Anglin</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>International Potato Center (CIP), Lima, Peru</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>USDA ARS Small Grains and Potato Germplasm Research, Aberdeen, Idaho, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anglin%20NL%22%5BAuthor%5D" class="usa-link"><span class="name western">N L Anglin</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Manrique%E2%80%90Carpintero%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">N Manrique‐Carpintero</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">N Manrique‐Carpintero</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>International Potato Center (CIP), Lima, Peru</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Manrique%E2%80%90Carpintero%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">N Manrique‐Carpintero</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lindqvist%E2%80%90Kreuze%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">H Lindqvist‐Kreuze</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">H Lindqvist‐Kreuze</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>International Potato Center (CIP), Lima, Peru</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lindqvist%E2%80%90Kreuze%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">H Lindqvist‐Kreuze</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tai%20HH%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">H H Tai</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">H H Tai</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Agriculture and Agri‐Food Canada Fredericton Research and Development Centre, Fredericton, New Brunswick, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tai%20HH%22%5BAuthor%5D" class="usa-link"><span class="name western">H H Tai</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Str%C3%B6mvik%20MV%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">M V Strömvik</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">M V Strömvik</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Science, McGill University, Sainte‐Anne‐de‐Bellevue, Quebec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Str%C3%B6mvik%20MV%22%5BAuthor%5D" class="usa-link"><span class="name western">M V Strömvik</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270095-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Plant Science, McGill University, Sainte‐Anne‐de‐Bellevue, Quebec, Canada</div>
<div id="tpg270095-aff-0002">
<sup>
<sup>2</sup>
</sup>Agriculture and Agri‐Food Canada Fredericton Research and Development Centre, Fredericton, New Brunswick, Canada</div>
<div id="tpg270095-aff-0003">
<sup>
<sup>3</sup>
</sup>International Potato Center (CIP), Lima, Peru</div>
<div id="tpg270095-aff-0004">
<sup>
<sup>4</sup>
</sup>USDA ARS Small Grains and Potato Germplasm Research, Aberdeen, Idaho, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Martina Strömvik, Department of Plant Science, McGill University, Sainte‐Anne‐de‐Bellevue, QC, Canada. Email: <span>martina.stromvik@mcgill.ca</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 2; Received 2025 Mar 23; Accepted 2025 Jun 25; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368328  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40836401/" class="usa-link">40836401</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Potato wild relatives (<em>Solanum</em> section <em>Petota</em>) are a source of genetic diversity for climate‐smart improvement of modern cultivars (<em>Solanum tuberosum</em>). The allotetraploid <em>Solanum acaule</em> Bitter. has been previously used to introgress frost tolerance into potato cultivars such as Yukon Gold and Viking. The current study presents the subgenomes of <em>S. acaule</em>. In a phylogenetic context, the placement of the individual subgenomes shows that <em>Solanum megistacrolobum</em> (Clade 4) is the progenitor species of <em>S. acaule</em> subgenome 2, while the progenitor species of subgenome 1 (mapping to Clade 3) is still unknown and may be extinct. The genome assembly is 1.34 Gb in 24 chromosomes with an N50 of 56.2 Mb, making the theoretical tetraploid <em>S. acaule</em> genome 2.68 Gb. The C‐repeat binding factor (CBF) cold response genes were identified in the subgenomes, and in a freeze‐challenged transcriptome analysis of <em>S. acaule</em> compared with autotetraploid <em>S. tuberosum</em> ‘Atlantic’, <em>CBF1</em> was induced in both species, while <em>CBF4</em> (subgenome 2) was induced only in <em>S. acaule</em>. A general subgenome expression bias was not detected in a set of homoeologous genes, though the subgenomes diverged in the number of differentially expressed genes and their functional roles.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270095-list-10001" class="list" style="list-style-type:disc">
<li><p>The chromosome‐level genome assembly of allotetraploid <em>Solanum acaule</em> (section Petota) shows distinct subgenomes.</p></li>
<li><p>The two progenitor species group with Clade 3 (unknown species) and Clade 4 (<em>Solanum megistacrolobum</em>), respectively.</p></li>
<li><p>The cold‐challenged transcriptome shows subgenome specificities, including differential expression of C‐repeat binding factor genes.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270095-dl-0001">
<dt>CBF</dt>
<dd><p>C‐repeat binding factor</p></dd>
<dt>CBL</dt>
<dd><p>calcineurin B‐like</p></dd>
<dt>CDS</dt>
<dd><p>coding sequence</p></dd>
<dt>DE</dt>
<dd><p>differentially expressed</p></dd>
<dt>EBN</dt>
<dd><p>endosperm balance number</p></dd>
<dt>ERF</dt>
<dd><p>ethylene‐responsive transcription factor</p></dd>
<dt>GO</dt>
<dd><p>gene ontology</p></dd>
<dt>ML</dt>
<dd><p>maximum likelihood</p></dd>
<dt>MS</dt>
<dd><p>Murashige and Skoog</p></dd>
<dt>QC</dt>
<dd><p>quality control</p></dd>
<dt>REIL</dt>
<dd><p>REI1‐LIKE</p></dd>
<dt>TF</dt>
<dd><p>transcription factor</p></dd>
<dt>WGS</dt>
<dd><p>whole genome sequence</p></dd>
</dl>
<section id="tpg270095-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Wild potato relatives are an important source of genetic material for developing modern climate‐smart potato cultivars, which need to be “<em>en garde</em>” for the increasing frequencies of dramatic weather events such as sudden frosts and droughts (Tai et al., <a href="#tpg270095-bib-0126" class="usa-link" aria-describedby="tpg270095-bib-0126">2025</a>). Over 100 wild potato species are recognized in the <em>Solanum</em> section <em>Petota</em> (tuber‐bearing members of the <em>Solanum</em> genus), and genome data for many of them have been released (e.g., Achakkagari et al., <a href="#tpg270095-bib-0001" class="usa-link" aria-describedby="tpg270095-bib-0001">2024</a>; Aversano et al., <a href="#tpg270095-bib-0006" class="usa-link" aria-describedby="tpg270095-bib-0006">2015</a>; Hardigan et al., <a href="#tpg270095-bib-0049" class="usa-link" aria-describedby="tpg270095-bib-0049">2017</a>; Hosaka et al., <a href="#tpg270095-bib-0057" class="usa-link" aria-describedby="tpg270095-bib-0057">2022</a>; Kyriakidou, Achakkagari et al., <a href="#tpg270095-bib-0072" class="usa-link" aria-describedby="tpg270095-bib-0072">2020</a>; Kyriakidou, Anglin et al., <a href="#tpg270095-bib-0073" class="usa-link" aria-describedby="tpg270095-bib-0073">2020</a>; Leisner et al., <a href="#tpg270095-bib-0075" class="usa-link" aria-describedby="tpg270095-bib-0075">2018</a>; Y. Li et al., <a href="#tpg270095-bib-0084" class="usa-link" aria-describedby="tpg270095-bib-0084">2018</a>; D. Tang et al., <a href="#tpg270095-bib-0127" class="usa-link" aria-describedby="tpg270095-bib-0127">2022</a>), as have larger compilations such as the <em>Solanum</em> section <em>Petota</em> super pangenome (Bozan et al., <a href="#tpg270095-bib-0012" class="usa-link" aria-describedby="tpg270095-bib-0012">2023</a>). Accessions representing breeding hybrids between <em>Solanum tuberosum</em> and wild relatives have also been published (e.g., Achakkagari et al., <a href="#tpg270095-bib-0001" class="usa-link" aria-describedby="tpg270095-bib-0001">2024</a>, <a href="#tpg270095-bib-0002" class="usa-link" aria-describedby="tpg270095-bib-0002">2022</a>). While the majority of modern potato cultivars are autotetraploid and the majority of wild relatives are diploid, the allopolyploid potato species, such as <em>Solanum acaule</em> and <em>Solanum stoloniferum</em> (both allotetraploid), <em>Solanum demissum</em> and <em>Solanum albicans</em> (both hexaploid), <em>Solanum juzepzukii</em> (triploid), and <em>Solanum curtilobum</em> (pentaploid), have long fascinated potato researchers.</p>
<p>One of the most extreme of the allopolyploid species is the allotetraploid <em>S. acaule</em> (2<em>n</em> = 4<em>x</em> = 48) (Bitter G. <a href="#tpg270095-bib-0009" class="usa-link" aria-describedby="tpg270095-bib-0009">1912</a>; Dvorak, <a href="#tpg270095-bib-0034" class="usa-link" aria-describedby="tpg270095-bib-0034">1983</a>; Hawkes, <a href="#tpg270095-bib-0052" class="usa-link" aria-describedby="tpg270095-bib-0052">1990</a>; Ochoa, <a href="#tpg270095-bib-0098" class="usa-link" aria-describedby="tpg270095-bib-0098">2004</a>), which grows at the highest altitudes (from 2400 to 4700 m), tolerates some of the lowest temperatures (Vega &amp; Bamberg, <a href="#tpg270095-bib-0130" class="usa-link" aria-describedby="tpg270095-bib-0130">1995</a>), and has the widest geographical spread of all potato polyploids, from Peru to Argentina (Ochoa, <a href="#tpg270095-bib-0098" class="usa-link" aria-describedby="tpg270095-bib-0098">2004</a>). <em>Solanum acaule</em> has a typical cold‐tolerant growth habit—low to the ground, rosette shaped (<em>acaule</em> means “stemless”), and sports plenty of trichomes. Early studies showed that it can tolerate frosts down to −5.5°C or −6.5°C (P. H. Li, <a href="#tpg270095-bib-0081" class="usa-link" aria-describedby="tpg270095-bib-0081">1977</a>; Sukumaran &amp; Weiser, <a href="#tpg270095-bib-0124" class="usa-link" aria-describedby="tpg270095-bib-0124">1972</a>). Several studies were carried out on <em>S. acaule</em> cold resistance genes (Karlsson &amp; Palta, <a href="#tpg270095-bib-0064" class="usa-link" aria-describedby="tpg270095-bib-0064">1999</a>; F. Li, Xu, et al., <a href="#tpg270095-bib-0077" class="usa-link" aria-describedby="tpg270095-bib-0077">2014</a>), and one transcriptome study found that putrescine, saccharides, amino acids, and other metabolites accumulated after exposure to −1°C for 12 h (Kou et al., <a href="#tpg270095-bib-0069" class="usa-link" aria-describedby="tpg270095-bib-0069">2018</a>). L. Chen et al. (<a href="#tpg270095-bib-0023" class="usa-link" aria-describedby="tpg270095-bib-0023">2022</a>) identified SaCBL1‐like (where CBL is calcineurin B‐like) as playing a vital role in conferring freezing tolerance in potato, and a potential basis for a targeted potato breeding for frost‐hardiness, and Aversano et al. (<a href="#tpg270095-bib-0006" class="usa-link" aria-describedby="tpg270095-bib-0006">2015</a>) identified several important genes.</p>
<p>
<em>Solanum acaule</em> has long been used in potato breeding programs primarily due to its frost tolerance, but also because of its resistance to cyst and root knot nematodes and to several viruses including potato virus X, potato virus Y, and potato leafroll virus (Ross, <a href="#tpg270095-bib-0108" class="usa-link" aria-describedby="tpg270095-bib-0108">1986</a>). A frost‐resistant variety, Alaska Frostless, resulting from the progeny of crosses involving <em>S. acaule</em>, was described by Dearborn (<a href="#tpg270095-bib-0030" class="usa-link" aria-describedby="tpg270095-bib-0030">1969</a>), and at least 20 additional potato cultivars, including Viking and Yukon Gold, have <em>S. acaule</em> in their pedigree (Love, <a href="#tpg270095-bib-0086" class="usa-link" aria-describedby="tpg270095-bib-0086">1999</a>). The high‐altitude habitat of <em>S. acaule</em> rarely harbors other <em>Solanum</em> species to naturally intercross with <em>S. acaule</em> (Hawkes, <a href="#tpg270095-bib-0052" class="usa-link" aria-describedby="tpg270095-bib-0052">1990</a>; Hosaka &amp; Spooner, <a href="#tpg270095-bib-0147" class="usa-link" aria-describedby="tpg270095-bib-0147">1992</a>). It is strongly self‐compatible, has disomic inheritance, and does not hybridize with autotetraploids due to mismatched endosperm balance number (EBN, EBN 2 in <em>S. acaule</em> and 4 in <em>S. tuberosum</em>), leading to a breakdown in the endosperm. Hybridization with diploids does occur with diploids of the same EBN number (2), but it results in sterile triploids such as the landrace <em>S. juzepzukii</em> (Hawkes, <a href="#tpg270095-bib-0051" class="usa-link" aria-describedby="tpg270095-bib-0051">1962</a>; Schmiediche et al., <a href="#tpg270095-bib-0110" class="usa-link" aria-describedby="tpg270095-bib-0110">1980</a>, <a href="#tpg270095-bib-0111" class="usa-link" aria-describedby="tpg270095-bib-0111">1982</a>), so gene flow into <em>S. acaule</em> is limited. Ploidy manipulations and bridge strategies are therefore needed for breeding <em>S. acaule</em> successfully with autotetraploids. Further, identification of nuclear and cytoplasm genome composition is required after hybrids are produced via bridge crossing or somatic hybridization (Park, <a href="#tpg270095-bib-0100" class="usa-link" aria-describedby="tpg270095-bib-0100">2021</a>).</p>
<p>The <em>Solanum</em> section <em>Petota</em> is known from several phylogenetic studies to be divided into several clades. Using orthologs of nuclear genes, plastid genomes, genome‐wide single nucleotide polymorphisms, and whole‐genome presence/absence variation, three main clades were recovered and are referred to as Clade 1+2, Clade 3, and Clade 4 (Bozan et al., <a href="#tpg270095-bib-0012" class="usa-link" aria-describedby="tpg270095-bib-0012">2023</a>; Huang et al., <a href="#tpg270095-bib-0060" class="usa-link" aria-describedby="tpg270095-bib-0060">2019</a>; Y. Li et al., <a href="#tpg270095-bib-0084" class="usa-link" aria-describedby="tpg270095-bib-0084">2018</a>; Spooner et al., <a href="#tpg270095-bib-0119" class="usa-link" aria-describedby="tpg270095-bib-0119">2008</a>; Yan et al., <a href="#tpg270095-bib-0140" class="usa-link" aria-describedby="tpg270095-bib-0140">2023</a>). Clade 1+2 contain North and Central American diploid species, while Clade 3 contains diploids from southern Colombia, Ecuador, and Peru, and Clade 4, which contains <em>S. tuberosum</em> and other cultivated species, is further divided into Clade 4 North (species from Peru) and Clade 4 South (species from Bolivia and south) (Y. Li et al., <a href="#tpg270095-bib-0084" class="usa-link" aria-describedby="tpg270095-bib-0084">2018</a>; Spooner et al., <a href="#tpg270095-bib-0120" class="usa-link" aria-describedby="tpg270095-bib-0120">2018</a>).</p>
<p>Studies on the origins of genomes in section <em>Petota</em> reference species groupings based on the taxonomy from Hawkes (Hawkes, <a href="#tpg270095-bib-0050" class="usa-link" aria-describedby="tpg270095-bib-0050">1958</a>, <a href="#tpg270095-bib-0051" class="usa-link" aria-describedby="tpg270095-bib-0051">1962</a>, <a href="#tpg270095-bib-0052" class="usa-link" aria-describedby="tpg270095-bib-0052">1990</a>) as well as Spooner (Spooner et al., <a href="#tpg270095-bib-0118" class="usa-link" aria-describedby="tpg270095-bib-0118">2014</a>, <a href="#tpg270095-bib-0119" class="usa-link" aria-describedby="tpg270095-bib-0119">2008</a>). Observation of bivalents during meiosis in interspecies hybrids was used to assign five genome groups: A, B, C, D, and P (Matsubayashi, <a href="#tpg270095-bib-0089" class="usa-link" aria-describedby="tpg270095-bib-0089">1991</a>). The A genome group covered all South American diploids and some polyploids, and the B, C, D, and P genome groups did not include diploids. <em>S. acaule</em> was found to be an allotetraploid with two sets of variant A genomes (Matsubayashi, <a href="#tpg270095-bib-0089" class="usa-link" aria-describedby="tpg270095-bib-0089">1991</a>). Analysis using genetic markers and genomic in situ hybridization supported the A genome assignment (Pendinen et al., <a href="#tpg270095-bib-0101" class="usa-link" aria-describedby="tpg270095-bib-0101">2012</a>; Rodriguez &amp; Spooner, <a href="#tpg270095-bib-0107" class="usa-link" aria-describedby="tpg270095-bib-0107">2009</a>; Spooner et al., <a href="#tpg270095-bib-0119" class="usa-link" aria-describedby="tpg270095-bib-0119">2008</a>). However, there are many species with A genomes and the species origin of the <em>S. acaule</em> genomes remained unknown. Plastid genome sequencing of <em>S. acaule</em> demonstrated grouping in Clade 4 (Park, <a href="#tpg270095-bib-0100" class="usa-link" aria-describedby="tpg270095-bib-0100">2021</a>; Yan et al., <a href="#tpg270095-bib-0140" class="usa-link" aria-describedby="tpg270095-bib-0140">2023</a>), though that only indicates the maternal origin. By using six nuclear DNA orthologues as markers, Cai et al. (<a href="#tpg270095-bib-0020" class="usa-link" aria-describedby="tpg270095-bib-0020">2012</a>) showed that <em>S. acaule</em> carries alleles that match homologues in both <em>Petota</em> Clade 3 and 4 species, though no conclusive progenitors were identified.</p>
<p>In the present study, we examine a clone of <em>S. acaule</em>, AA9, which was propagated from a single seed sourced from wild seed collected in Ancash, Peru (PI 498202). The collection site was Puca Chaca, between Tarica and Sihuas, at 4100 m. The high elevation is typical of <em>S. acaule</em> habitats. We have sequenced and assembled the nuclear subgenomes of <em>S. acaule</em> and placed them in a phylogeny consisting of a larger range of <em>Solanum</em> section <em>Petota</em> genomes. The cladistic groupings inferred from the phylogeny were used to understand the origins of subgenomes and progenitors of <em>S. acaule</em>. Furthermore, we have explored the <em>S. acaule</em> cold‐responsive transcriptome in comparison with that of autotetraploid <em>S. tuberosum</em> ‘Atlantic’ and shown subgenome preferential differential expression of several cold‐responsive genes in <em>S. acaule</em>. The <em>S. acaule</em> AA9 assembly is reference quality and can be used as an aid for breeding purposes and for research on the allopolyploid species in the <em>Acaulia</em> series of <em>Solanum</em> section <em>Petota</em>.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270095-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270095-list-0001" class="list" style="list-style-type:disc">
<li><p>The chromosome‐level genome assembly of allotetraploid <em>Solanum acaule</em> (section Petota) shows distinct subgenomes.</p></li>
<li><p>The two progenitor species group with Clade 3 (unknown species) and Clade 4 (<em>Solanum megistacrolobum</em>), respectively.</p></li>
<li><p>The cold‐challenged transcriptome shows subgenome specificities, including differential expression of C‐repeat binding factor genes.</p></li>
</ul></section></section></section><section id="tpg270095-sec-0030"><h2 class="pmc_sec_title">2. METHODS</h2>
<section id="tpg270095-sec-0040"><h3 class="pmc_sec_title">2.1. Plant material for genome sequencing</h3>
<p>The <em>Solanum acaule</em> AA9 clone (a genotype maintained as an in vitro plantlet under Canadian Potato Genebank accession CN# 124026, originating from true seeds of CIP 761464, USDA PI 498202, OCH 13005; <a href="https://doi.org/10.18730/DVW%7E" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.18730/DVW~</a>) was chosen because of its use in breeding for cold stress tolerance. The AA9 accession was originally collected in 1978 in Ancash, Peru, according to the passport information PI 498202 GRIN‐Global (<a href="https://npgsweb.ars-grin.gov/gringlobal/accessiondetail?id=1393138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://npgsweb.ars‐grin.gov/gringlobal/accessiondetail?id=1393138</a>) by Carlos Ochoa of the International Potato Centre (CIP). The collection site was Puca Chaca, between Tarica and Sihuas, at 4100 m. The high elevation is typical of <em>S. acaule</em> habitats. The appearance of AA9 plants was similar to other <em>S. acaule</em> plants (Solanaceae source: <a href="https://solanaceaesource.myspecies.info/content/solanum-acaule" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://solanaceaesource.myspecies.info/content/solanum‐acaule</a>; Correll, <a href="#tpg270095-bib-0028" class="usa-link" aria-describedby="tpg270095-bib-0028">1962</a>). The AA9 genome was sequenced using Pacific Biosciences (PacBio), Oxford Nanopore (ONT), and Illumina technologies; the transcriptome and chromatin interactions (Hi‐C) were sequenced using Illumina. The PacBio libraries were constructed as described in Achakkagari et al. (<a href="#tpg270095-bib-0001" class="usa-link" aria-describedby="tpg270095-bib-0001">2024</a>) and sequenced on Sequel II system using the SMRT technology (CLR [Continuous Long Reads]) at a depth of 100x. The Nanopore ONT libraries were prepared and sequenced on the PromethION at a depth of 100x as described in Achakkagari et al. (<a href="#tpg270095-bib-0001" class="usa-link" aria-describedby="tpg270095-bib-0001">2024</a>), and Illumina short‐read sequencing was performed on the NovaSeq 6000 platform in a paired‐end mode (2 × 150 bp) at 100x depth (Novogene Co., Ltd.). The Hi‐C libraries were prepared using Phase Genomics Proximo Plant Hi‐C version 4.0 and sequenced on the Illumina NovaSeq 6000 platform in PE 100 bp mode with ∼3 billion reads.</p></section><section id="tpg270095-sec-0050"><h3 class="pmc_sec_title">2.2. Plant material for freezing transcriptome analysis</h3>
<p>
<em>Solanum acaule</em> (DPM‐AA9, CIP 761464, PI 498202, and CN# 124026) was the same clone used for genome sequencing and <em>Solanum tuberosum</em> ‘Atlantic’ was sourced from the potato breeding collection of Agriculture and Agri‐Food Canada in Fredericton, NB, Canada Agriculture and Agri‐Food Canada. Plants were grown in Murashige and Skoog (MS) (Sigma) medium containing MS vitamin solution (Sigma) 1 mL/L, 30 g/L of sucrose (Sigma), and 8 g/L of agar (Sigma) in a growth chamber at 22°C, 150 µmol m‐2 s‐1 and 18‐h light 6‐h darkness and transferred to fresh medium every 4 weeks. Magenta boxes with MS medium containing shoot explants were placed in a growth chamber where the temperature gradually decreased from 22°C to −4°C at a rate of 2°C per hour. For the freezing stress treatment, plants were taken from the chamber 1 h after it reached the final temperature; they were immediately flash frozen in liquid nitrogen and stored at −80°C until RNA extraction. Long‐day conditions (as described) were used as the control treatment. Three biological replicates were analyzed for each species and condition.</p></section><section id="tpg270095-sec-0060"><h3 class="pmc_sec_title">2.3. Genome assembly and phasing</h3>
<p>The PacBio raw data were used to assemble the AA9 genome using Canu assembler v2.2 (Nurk et al., <a href="#tpg270095-bib-0097" class="usa-link" aria-describedby="tpg270095-bib-0097">2020</a>) with the parameters <em>batOptions = ‐dg 3 ‐db 3 ‐dr 1 ‐ca 500 ‐cp 50</em>. The nanopore raw reads were trimmed using Porechop v0.2.4 (Bonenfant et al., <a href="#tpg270095-bib-0011" class="usa-link" aria-describedby="tpg270095-bib-0011">2022</a>) and filtered for at least Q10 quality scores using NanoFilt v2.8.0 (De Coster et al., <a href="#tpg270095-bib-0031" class="usa-link" aria-describedby="tpg270095-bib-0031">2018</a>). The filtered reads were used to generate contigs using Flye v2.9‐b1774 (Kolmogorov et al., <a href="#tpg270095-bib-0068" class="usa-link" aria-describedby="tpg270095-bib-0068">2019</a>) and NextDenovo v2.5.0 (Hu et al., <a href="#tpg270095-bib-0059" class="usa-link" aria-describedby="tpg270095-bib-0059">2024</a>) assemblers. The organellar sequences were filtered from the PacBio and ONT assemblies. The PacBio assembly had superior contiguity and a higher N50 compared to the Nanopore assemblies, and was therefore selected for subsequent Hi‐C scaffolding. The Hi‐C reads were trimmed using fastp v0.23.1 (S. Chen et al., <a href="#tpg270095-bib-0024" class="usa-link" aria-describedby="tpg270095-bib-0024">2018</a>) and QC'ed (where QC is quality control) using HiC‐QC v1.0 (<a href="https://github.com/phasegenomics/hic_qc" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/phasegenomics/hic_qc</a>). The trimmed Hi‐C reads were mapped to the filtered PacBio contigs using BWA v0.7.17 (H. Li, <a href="#tpg270095-bib-0078" class="usa-link" aria-describedby="tpg270095-bib-0078">2013</a>) and filtered and sorted using samtools v1.16.1 (H. Li et al., <a href="#tpg270095-bib-0079" class="usa-link" aria-describedby="tpg270095-bib-0079">2009</a>). Then the contigs were corrected based on the Hi‐C signals using ALLHiC correct (X. Zhang et al., <a href="#tpg270095-bib-0145" class="usa-link" aria-describedby="tpg270095-bib-0145">2019</a>). The Hi‐C reads were then mapped back to the corrected contigs and filtered to remove low‐quality hits using ALLHiC and samtools with ‐q 40. Then the contigs were scaffolded using YaHS v1.1 (Zhou et al., <a href="#tpg270095-bib-0146" class="usa-link" aria-describedby="tpg270095-bib-0146">2023</a>) with <em>‐e GATC ‐r 1000,2000,5000,10000,20000,50000,100000,200000,500000,1000000,2000000, 5000000,10000000,20000000 –no‐mem‐check ‐l 500 –no‐contig‐ec –no‐scaffold‐ec</em> parameters. The resulting scaffolds were manually reviewed in JuiceBox v1.11.08 (Durand et al., <a href="#tpg270095-bib-0033" class="usa-link" aria-describedby="tpg270095-bib-0033">2016</a>) and generated 24 chromosomes. Contigs containing assembled telomeres were correctly oriented to the terminal ends by searching for the TTTAGGG repeat (or the reverse complement CCCTAAA) using tidk (Brown et al., <a href="#tpg270095-bib-0013" class="usa-link" aria-describedby="tpg270095-bib-0013">2023</a>) and FindTelomeres (<a href="https://github.com/JanaSperschneider/FindTelomeres" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/JanaSperschneider/FindTelomeres</a>). Then ALLHiC Rescue was used to assign unanchored contigs to specific clusters based on their Hi‐C signal density and then manually reviewed in JuiceBox. Finally, the nanopore assemblies were used within LRScaf v1.1.11 (Qin et al., <a href="#tpg270095-bib-0104" class="usa-link" aria-describedby="tpg270095-bib-0104">2019</a>) to generate links between unanchored contigs and chromosomes and then were manually reviewed and added using JuiceBox. The resulting assembly was polished using the freebayes‐polish pipeline from VGP (<a href="https://github.com/VGP/vgp-assembly/tree/master/pipeline/freebayes-polish" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/VGP/vgp‐assembly/tree/master/pipeline/freebayes‐polish</a>). Any duplicated contigs with respect to the chromosomes were removed from the assembly. The gaps introduced by scaffolding were closed using TGSgapcloser v1.2.1 (M. Xu et al., <a href="#tpg270095-bib-0139" class="usa-link" aria-describedby="tpg270095-bib-0139">2020</a>) with PacBio and nanopore reads, followed by another round of freebayes‐polish to generate a cleaned chromosome‐scale assembly. The 24 chromosomes were phased into two sub‐genomes based on repetitive k‐mers as differential signatures using SubPhaser v1.2.6 (Jia et al., <a href="#tpg270095-bib-0062" class="usa-link" aria-describedby="tpg270095-bib-0062">2022</a>). Unplaced contigs were screened for contamination using BlobTools v1.1.1 (Laetsch &amp; Blaxter, <a href="#tpg270095-bib-0074" class="usa-link" aria-describedby="tpg270095-bib-0074">2017</a>) and by aligning against NCBI databases. This analysis identified contigs matching microbial taxa such as <em>Streptomyces</em> sp., <em>Clostridium</em> sp., and <em>Bradyrhizobium</em> sp., which were subsequently removed from the final assembly.</p></section><section id="tpg270095-sec-0070"><h3 class="pmc_sec_title">2.4. Synteny analysis</h3>
<p>Syntenic blocks between the two sub‐genomes were detected using MCScanX 1.0.0 (Y. Wang et al., <a href="#tpg270095-bib-0133" class="usa-link" aria-describedby="tpg270095-bib-0133">2012</a>), and the synteny plots were visualized using SynVisio (<a href="https://synvisio.github.io" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://synvisio.github.io</a>) (Bandi et al., <a href="#tpg270095-bib-0007" class="usa-link" aria-describedby="tpg270095-bib-0007">2022</a>).</p></section><section id="tpg270095-sec-0080"><h3 class="pmc_sec_title">2.5. Transcriptome (RNA) sequencing and assembly</h3>
<p>Total RNA was extracted from shoot frozen samples using the RNeasy Plant Mini Kit (Qiagen). Library preparation and sequencing was performed at the Centre d'expertise et de service Génome Québec using an Illumina NovaSeq 6000 sequencer, according to manufacturer's instructions. Reads were trimmed and quality filtered with Trimmomatic v0.39 (Bolger et al., <a href="#tpg270095-bib-0010" class="usa-link" aria-describedby="tpg270095-bib-0010">2014</a>) and then cleaned of viral sequences with a custom pipeline. Transcriptomes for each species were assembled by generating a de novo and a reference guided assemblies with <em>Trinity v2.14.0</em> (Haas et al., <a href="#tpg270095-bib-0048" class="usa-link" aria-describedby="tpg270095-bib-0048">2013</a>) and merging them with <em>EvidentialGene v2022may07</em> (Gilbert, <a href="#tpg270095-bib-0044" class="usa-link" aria-describedby="tpg270095-bib-0044">2019</a>). Clean reads were used to get the Trinity de novo using default parameters. To obtain the Trinity reference assemblies, reads were first mapped to their respective reference phased genomes using <em>Hisat2 v 2.2.1</em> (Kim et al., <a href="#tpg270095-bib-0067" class="usa-link" aria-describedby="tpg270095-bib-0067">2019</a>). For <em>S. tuberosum</em>, the Atlantic cultivar haplotype level genome was used as reference (Hoopes et al., <a href="#tpg270095-bib-0056" class="usa-link" aria-describedby="tpg270095-bib-0056">2022</a>), and for <em>S. acaule</em>, the subgenome level assembly generated in this study was used. Mapped reads were used to get the Trinity reference assembly using the genome‐guided option. After merging both reference and de novo assemblies with EvidentialGene, final assemblies were cleaned of viral sequences with custom pipeline. Assembly completeness was assessed with <em>BUSCO v5.3.2</em> (Simão et al., <a href="#tpg270095-bib-0114" class="usa-link" aria-describedby="tpg270095-bib-0114">2015</a>). The RNA reads are uploaded to NCBI and linked with BioProject PRJNA779368. Coding sequences of transcripts were annotated with TransDecoder v5.7.1 (<a href="https://github.com/TransDecoder" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/TransDecoder</a>) (Haas et al., <a href="#tpg270095-bib-0048" class="usa-link" aria-describedby="tpg270095-bib-0048">2013</a>). For transcriptome length comparisons, only protein‐coding transcripts with complete CDS were considered.</p>
<p>Syntenic gene sets were obtained using <em>MCscanX 1.0.0</em> (Y. Wang et al., <a href="#tpg270095-bib-0133" class="usa-link" aria-describedby="tpg270095-bib-0133">2012</a>). In <em>S. acaule</em>, homoeologous gene pairs between subgenomes 1 and 2 were obtained, whereas in <em>S. tuberosum</em> ‘Atlantic’, homologous gene sets were obtained for all haplotype combinations; only sets of genes meeting a 1:1, 1:1:1, or 1:1:1:1 were selected.</p>
<p>Transcriptome assemblies were mapped to the respective reference genome to assign each transcript to their respective gene using GMAP v2023‐12‐01 (Wu &amp; Watanabe, <a href="#tpg270095-bib-0137" class="usa-link" aria-describedby="tpg270095-bib-0137">2005</a>) with a k‐mer size of 16 and a mapping quality cutoff of 2. Transcripts were thus annotated as belonging to different subgenomes/haplotypes and to respective syntenic gene sets.</p></section><section id="tpg270095-sec-0090"><h3 class="pmc_sec_title">2.6. Differential expression and functional analysis</h3>
<p>Transcript abundance estimation was calculated with <em>Kallisto v0.46.1</em> (Bray et al., <a href="#tpg270095-bib-0148" class="usa-link" aria-describedby="tpg270095-bib-0148">2016</a>), followed by differential expression analyses calculated with <em>DESeq2</em> v1.38.3 (Love et al., <a href="#tpg270095-bib-0085" class="usa-link" aria-describedby="tpg270095-bib-0085">2014</a>). Coding regions within transcripts were predicted using <em>TransDecoder v5.0.0</em> (Haas et al., <a href="#tpg270095-bib-0048" class="usa-link" aria-describedby="tpg270095-bib-0048">2013</a>), and functional annotation was assigned using <em>Trinotate v4.0.0</em> (Bryant et al., <a href="#tpg270095-bib-0017" class="usa-link" aria-describedby="tpg270095-bib-0017">2017</a>) with default databases (UniProt, EggNOG, and Pfam). Enrichment analyses were performed using the package <em>GOseq v1.50.0</em> (Young et al., <a href="#tpg270095-bib-0143" class="usa-link" aria-describedby="tpg270095-bib-0143">2010</a>) on a subset of genes selected with a minimum of twofold differential expression (|log2Foldchange | &gt; 1) and <em>p</em>adj &lt; 0.05.</p></section><section id="tpg270095-sec-0100"><h3 class="pmc_sec_title">2.7. Homoeologous expression bias</h3>
<p>Homoeologous gene pairs with 1:1 ratio of <em>S. acaule</em> obtained with MCscanX were used to calculate expression bias between subgenomes for each condition tested, control and freezing. Log 2 fold changes were calculated with DEseq2 v1.38.3 (Love et al., <a href="#tpg270095-bib-0085" class="usa-link" aria-describedby="tpg270095-bib-0085">2014</a>). The built‐in Wald test was used to test significant differences; genes with false discovery rate <em>p</em>adj &lt; 0.05 were considered.</p></section><section id="tpg270095-sec-0110"><h3 class="pmc_sec_title">2.8. Quality control and analysis</h3>
<p>QC was performed on the raw sequencing data (whole genome sequence [WGS] and RNA‐Seq) using <em>FastQC v0.11.9</em> (Andrews, <a href="#tpg270095-bib-0004" class="usa-link" aria-describedby="tpg270095-bib-0004">2019</a>). An adaptor removal and read quality trimming was performed on the raw reads using <em>Trimmomatic v0.39</em> (Bolger et al., <a href="#tpg270095-bib-0010" class="usa-link" aria-describedby="tpg270095-bib-0010">2014</a>). Genome heterozygosity and estimated size were determined using the corrected PacBio reads. First, k‐mer frequencies were calculated (<em>k</em> = 21) and a k‐mer histogram was generated using <em>Jellyfish v2.3.0</em> (Marçais &amp; Kingsford, <a href="#tpg270095-bib-0088" class="usa-link" aria-describedby="tpg270095-bib-0088">2011</a>). The k‐mer histogram was used to generate various genome characteristics with <em>GenomeScope 2.0</em> (Ranallo‐Benavidez et al., <a href="#tpg270095-bib-0105" class="usa-link" aria-describedby="tpg270095-bib-0105">2020</a>). The quality of the raw WGS and RNA‐Seq reads was checked using <em>FastQC</em>. Various metrics such as per base sequence quality, overall read quality score, GC content, N content, and overrepresented sequences were analyzed to detect low‐quality sequences or biases in the data. Amount of viral genome sequences in the RNA‐Seq data was estimated using <em>kraken2 v2.1.2</em> (Wood et al., <a href="#tpg270095-bib-0136" class="usa-link" aria-describedby="tpg270095-bib-0136">2019</a>) by searching against its own genome assembly and potato virus sequences downloaded from NCBI. AA9 had 95.87% of the RNAseq reads classified as <em>Solanum</em> with 3.17% classified as Potato Virus X. The genome assembly quality was assessed by <em>QUAST v5.0.2</em> (Gurevich et al., <a href="#tpg270095-bib-0047" class="usa-link" aria-describedby="tpg270095-bib-0047">2013</a>), <em>merqury</em> (Rhie et al., <a href="#tpg270095-bib-0106" class="usa-link" aria-describedby="tpg270095-bib-0106">2020</a>), <em>BUSCO v5.3.2</em> (Simão et al., <a href="#tpg270095-bib-0114" class="usa-link" aria-describedby="tpg270095-bib-0114">2015</a>), where contiguity and completeness of the assemblies were analyzed, and with CRAQ v1.0.9 (K. Li et al., <a href="#tpg270095-bib-0080" class="usa-link" aria-describedby="tpg270095-bib-0080">2023</a>), AutoHiC v1.3.3 (Jiang et al., <a href="#tpg270095-bib-0063" class="usa-link" aria-describedby="tpg270095-bib-0063">2024</a>), and 3D‐DNA v201008 (Dudchenko et al., <a href="#tpg270095-bib-0032" class="usa-link" aria-describedby="tpg270095-bib-0032">2017</a>).</p></section><section id="tpg270095-sec-0120"><h3 class="pmc_sec_title">2.9. Gene prediction and functional annotation</h3>
<p>The genome assembly was annotated using the RNA‐Seq data as the evidence for gene prediction along with homology‐based prediction. The repeat sequences in the genome were identified using RepeatModeler v2.0.3 (Flynn et al., <a href="#tpg270095-bib-0038" class="usa-link" aria-describedby="tpg270095-bib-0038">2020</a>), annotated using common runtime language scripts, and excluded the gene fragments from the unknown repeats to generate a final custom repeat library. This repeat library was used to softmask the genome using RepeatMasker v4.1.2‐pl (Smit et al., <a href="#tpg270095-bib-0115" class="usa-link" aria-describedby="tpg270095-bib-0115">2015</a>). Structural annotation was performed using BRAKER v3.0.3(Gabriel et al., <a href="#tpg270095-bib-0040" class="usa-link" aria-describedby="tpg270095-bib-0040">2024</a>) with both evidence‐based and homology‐based predictions. For the evidence‐based prediction, trimmed RNA‐Seq reads were aligned to the genome assembly using HiSAT2 v2.2.1 (Kim et al., <a href="#tpg270095-bib-0067" class="usa-link" aria-describedby="tpg270095-bib-0067">2019</a>) and sorted and indexed using SAMtools v1.16.1. The alignment file was used as input for the BRAKER. For the homology‐based prediction, Solanales protein sequences from OrthoDB, along with the protein sequences from potato reference genomes, were combined to create a protein database for BRAKER. The BRAKER pipeline (Buchfink et al., <a href="#tpg270095-bib-0018" class="usa-link" aria-describedby="tpg270095-bib-0018">2015</a>; Bruna et al., <a href="#tpg270095-bib-0014" class="usa-link" aria-describedby="tpg270095-bib-0014">2021</a>, <a href="#tpg270095-bib-0016" class="usa-link" aria-describedby="tpg270095-bib-0016">2023</a>; Gabriel et al., <a href="#tpg270095-bib-0041" class="usa-link" aria-describedby="tpg270095-bib-0041">2021</a>; Gotoh, <a href="#tpg270095-bib-0045" class="usa-link" aria-describedby="tpg270095-bib-0045">2008</a>; Hoff et al., <a href="#tpg270095-bib-0054" class="usa-link" aria-describedby="tpg270095-bib-0054">2016</a>, <a href="#tpg270095-bib-0055" class="usa-link" aria-describedby="tpg270095-bib-0055">2019</a>; Iwata &amp; Gotoh, <a href="#tpg270095-bib-0061" class="usa-link" aria-describedby="tpg270095-bib-0061">2012</a>; Kovaka et al., <a href="#tpg270095-bib-0070" class="usa-link" aria-describedby="tpg270095-bib-0070">2019</a>; Pertea &amp; Pertea, <a href="#tpg270095-bib-0103" class="usa-link" aria-describedby="tpg270095-bib-0103">2020</a>; Stanke et al., <a href="#tpg270095-bib-0121" class="usa-link" aria-describedby="tpg270095-bib-0121">2008</a>, <a href="#tpg270095-bib-0122" class="usa-link" aria-describedby="tpg270095-bib-0122">2006</a>) used AUGUSTUS and GeneMark‐ETP to train and predict highly reliable genes. It used TSEBRA v1.0.3 to generate a combined set of genes containing only the genes with very high support from extrinsic evidence. The annotations were curated using gFACs with <em>–rem‐all‐incompletes –rem‐genes‐without‐start‐codon –rem‐genes‐without‐stop‐codon –allow‐alternate‐starts</em> parameters. A functional annotation was performed using AHRD (<a href="https://github.com/groupschoof/AHRD.git" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/groupschoof/AHRD.git</a>) against Trembl, Swiss‐Prot, and Araprot databases and InterProScan v5.52‐86 against the PFAM database (Mistry et al., <a href="#tpg270095-bib-0092" class="usa-link" aria-describedby="tpg270095-bib-0092">2021</a>). To assess annotation quality, we used a combination of structural, functional, and evolutionary metrics. BUSCO v5.3.2 (Simão et al., <a href="#tpg270095-bib-0114" class="usa-link" aria-describedby="tpg270095-bib-0114">2015</a>) was run in protein mode with solanales_odb10 to estimate gene model completeness. OMArk v0.3.0 (Nevers et al., <a href="#tpg270095-bib-0096" class="usa-link" aria-describedby="tpg270095-bib-0096">2025</a>) was used to evaluate both completeness, via conserved Solanaceae HOGs, and consistency, based on placement of proteins into known gene families. PSAURON v1.0.6 (Sommer et al., <a href="#tpg270095-bib-0116" class="usa-link" aria-describedby="tpg270095-bib-0116">2025</a>) was used to score coding sequences and assess genome‐wide annotation confidence. Structural annotation statistics (e.g., gene, exon, intron counts, and genome coverage) were computed using the AGAT toolkit v1.4.2 (Dainat et al., <a href="#tpg270095-bib-0029" class="usa-link" aria-describedby="tpg270095-bib-0029">2020</a>).</p></section><section id="tpg270095-sec-0130"><h3 class="pmc_sec_title">2.10. Phylogenetic analysis of gene families</h3>
<p>OrthoFinder v2.5.5.2 (Emms &amp; Kelly, <a href="#tpg270095-bib-0037" class="usa-link" aria-describedby="tpg270095-bib-0037">2019</a>) was run with the protein coding genes from AA9 subgenomes, 44 diploid genomes from Tang et al. (<a href="#tpg270095-bib-0127" class="usa-link" aria-describedby="tpg270095-bib-0127">2022</a>), and 23 wild accessions (Achakkagari et al., <a href="#tpg270095-bib-0001" class="usa-link" aria-describedby="tpg270095-bib-0001">2024</a>) to identify orthologous groups. From the resulting orthogroups, we selected 260 high‐confidence single‐copy orthologous genes that were present in all species, contained exactly one gene per species, and had full‐length protein sequences suitable for alignment. These genes were used to construct a coalescent‐based phylogenetic tree, where individual maximum likelihood (ML) gene trees were generated using IQTREE2 v2.2.7 (Minh et al., <a href="#tpg270095-bib-0091" class="usa-link" aria-describedby="tpg270095-bib-0091">2020</a>) and then used ASTRAL v5.7.8 (C. Zhang et al., <a href="#tpg270095-bib-0144" class="usa-link" aria-describedby="tpg270095-bib-0144">2018</a>) to generate a species tree. The phylogenetic trees were visualized in <em>FigTree v1.4.4</em> (<a href="http://tree.bio.ed.ac.uk/software/figtree/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://tree.bio.ed.ac.uk/software/figtree/</a>).</p></section><section id="tpg270095-sec-0140"><h3 class="pmc_sec_title">2.11. C‐repeat binding factor genes analyses</h3>
<p>CBF genes were investigated in <em>S. acaule</em> and <em>S. tuberosum</em> ‘Atlantic’ genomes and transcriptomes. Genes and coding sequences of 11 <em>S. tuberosum CBFs</em> were retrieved from GenBank and Phytozome. Reference CBF genes from Pennycooke et al. (<a href="#tpg270095-bib-0102" class="usa-link" aria-describedby="tpg270095-bib-0102">2008</a>) were retrieved from GenBank [<a href="https://www.ncbi.nlm.nih.gov/nuccore/EU365384" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EU365384</a>.1_cds_ACB45095.1_1 (CBF3), <a href="https://www.ncbi.nlm.nih.gov/nuccore/EU365384" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EU365384</a>.1_cds_ACB45096.1_2 (CBF1), <a href="https://www.ncbi.nlm.nih.gov/nuccore/EU365384" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EU365384</a>.1_cds_ACB45097.1_3 (CBF2A), <a href="https://www.ncbi.nlm.nih.gov/nuccore/EU365384" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EU365384</a>.1_cds_ACB45098.1_4 (CBF2B), <a href="https://www.ncbi.nlm.nih.gov/nuccore/EU365379" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EU365379</a>.1_cds_ACB45083.1_2 (CBF4), <a href="https://www.ncbi.nlm.nih.gov/nuccore/EU365379" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EU365379</a>.1_cds_ACB45082.1_1 (CBF5)] and supplemented with additional ones from W. Li et al. (<a href="#tpg270095-bib-0083" class="usa-link" aria-describedby="tpg270095-bib-0083">2020</a>) retrieved from Phytozome ID = PGSC0003DMT400090651|ST4.03ch11 (CBF6), ID = PGSC0003DMT400010876|ST4.03ch12 (CBF7), ID = PGSC0003DMT400032623|ST4.03ch08 (CBF8), ID = PGSC0003DMT400032622|ST4.03ch08 (CBF9), and ID = PGSC0003DMT400032621|ST4.03ch08 (CBF10). Gene names from Pennycooke et al. (<a href="#tpg270095-bib-0102" class="usa-link" aria-describedby="tpg270095-bib-0102">2008</a>) and W. Li et al. (<a href="#tpg270095-bib-0083" class="usa-link" aria-describedby="tpg270095-bib-0083">2020</a>) do not match. The names from the former were used. GenBank: <a href="https://www.ncbi.nlm.nih.gov/genbank" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/genbank</a>. Phytozome: <a href="https://phytozome-next.jgi.doe.gov" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phytozome‐next.jgi.doe.gov</a>.</p>
<p>The reference genes were used as queries in nucleotide sequence searches of the genomes of <em>S. acaule</em> and the <em>S. tuberosum</em> ‘Atlantic’ (Hoopes et al., <a href="#tpg270095-bib-0056" class="usa-link" aria-describedby="tpg270095-bib-0056">2022</a>) using <em>BLAST+ v.2.14.0</em> (Camacho et al., <a href="#tpg270095-bib-0021" class="usa-link" aria-describedby="tpg270095-bib-0021">2009</a>). Query and candidate CBF sequences were aligned using <em>MUSCLE</em> (Edgar, <a href="#tpg270095-bib-0035" class="usa-link" aria-describedby="tpg270095-bib-0035">2004</a>) as implemented in <em>MEGAX v.10.2.5</em> (Kumar et al., <a href="#tpg270095-bib-0071" class="usa-link" aria-describedby="tpg270095-bib-0071">2018</a>); default parameters were used with the neighbor‐joining clustering method. A ML phylogenetic tree was constructed with <em>MEGAX</em> using the Jones–Taylor–Thornton substitution model and gamma distribution with invariant sites and 100 bootstrap replications. The tree figure was edited with <em>TreeViewer v.2.2.0</em> (Bianchini &amp; Sánchez‐Baracaldo, <a href="#tpg270095-bib-0008" class="usa-link" aria-describedby="tpg270095-bib-0008">2024</a>).</p></section></section><section id="tpg270095-sec-0150"><h2 class="pmc_sec_title">3. RESULTS AND DISCUSSION</h2>
<section id="tpg270095-sec-0160"><h3 class="pmc_sec_title">3.1. Reference quality subgenome assemblies of allotetraploid <em>S. acaule</em>
</h3>
<p>
<em>Solanum acaule</em> clone AA9 was sequenced, assembled, and phased using PacBio, Illumina, Nanopore, and Hi‐C data with a custom bioinformatics pipeline. The assembly size is 1.34 Gb in 24 chromosomes and 188 unplaced contigs (6.7 MB) with an N50 of 56.2 Mb. The individual haplotypes of each of the two subgenomes cannot be distinguished, which indicates that each subgenome is homozygous and that the original hybridization may have been between two diploids (<em>n</em> + <em>n</em>) followed by duplication. Subgenome 1 is smaller (650 Mb) than subgenome 2 (688 Mb) with an N50 of 54.1 and 56.3 Mb, respectively (Table <a href="#tpg270095-tbl-0001" class="usa-link">1</a>). A total of 61,417 high‐confidence protein‐coding genes were found in the AA9 genome, with 56,610 genes having functional annotations. Subgenome 1 has fewer genes than subgenome 2, with 29,521 and 31,822 genes, respectively. The total classified repeats make up 64%, or 857 Mb of the genome, and the majority (28% of the repeats) are long terminal repeats (Table <a href="#tpg270095-supinfo-0001" class="usa-link">S1</a>). Figure <a href="#tpg270095-fig-0001" class="usa-link">1a</a> shows the synteny between the chromosomes of the two subgenomes of <em>S. acaule</em> clone AA9. Overall, 38,899 (63%) genes are syntenic between the two subgenomes. The dotplot of the two subgenomes (Figure <a href="#tpg270095-supinfo-0001" class="usa-link">S1</a>) shows high correspondence, with chromosome 1 showing high divergence and an inverted region. The inverted regions correspond with findings from earlier studies indicating cryptic differences between the two A genomes of <em>S. acaule</em> (Matsubayashi, <a href="#tpg270095-bib-0089" class="usa-link" aria-describedby="tpg270095-bib-0089">1991</a>). Large pericentromeric inversions were also found between A and B genomes of the allotetraploid <em>S. stoloniferum</em> (Hosaka et al., <a href="#tpg270095-bib-0058" class="usa-link" aria-describedby="tpg270095-bib-0058">2025</a>).</p>
<section class="tw xbox font-sm" id="tpg270095-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Assembly statistics for allotetraploid <em>Solanum acaule</em> (AA9) reference genome and subgenomes.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Assembly</th>
<th align="left" rowspan="1" colspan="1">Total assembly length</th>
<th align="left" rowspan="1" colspan="1">BUSCO (solanales_odb10, <em>n</em> = 5950)<a href="#tpg270095-tbl1-note-0001" class="usa-link">
<sup>a</sup>
</a>
</th>
<th align="left" rowspan="1" colspan="1">Completeness k‐mer (merqury)</th>
<th align="left" rowspan="1" colspan="1">Consensus quality value QV (merqury)</th>
<th align="left" rowspan="1" colspan="1">N50 (Mb)</th>
<th align="left" rowspan="1" colspan="1">Largest chromosome (Mb)</th>
<th align="left" rowspan="1" colspan="1">Number of genes</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Total <em>S. acaule</em> AA9</strong>
</td>
<td align="left" rowspan="1" colspan="1">1.34 Gb</td>
<td align="left" rowspan="1" colspan="1">
<p>99.1%</p>
<p>C:99.1%[S:4.4%,D:94.7%], F:0.0%, M:0.9%</p>
</td>
<td align="left" rowspan="1" colspan="1">99.6%</td>
<td align="left" rowspan="1" colspan="1">48.2</td>
<td align="left" rowspan="1" colspan="1">56.2</td>
<td align="left" rowspan="1" colspan="1">79.5</td>
<td align="left" rowspan="1" colspan="1">61,417</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Subgenome 1</strong>
</td>
<td align="left" rowspan="1" colspan="1">650 Mb</td>
<td align="left" rowspan="1" colspan="1">
<p>98.0%</p>
<p>C:98.0%[S:95.6%,D:2.4%], F:0.2%, M:1.8%</p>
</td>
<td align="left" rowspan="1" colspan="1">59.3%</td>
<td align="left" rowspan="1" colspan="1">49.1</td>
<td align="left" rowspan="1" colspan="1">54.1</td>
<td align="left" rowspan="1" colspan="1">73.5</td>
<td align="left" rowspan="1" colspan="1">29,521 (48%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Subgenome 2</strong>
</td>
<td align="left" rowspan="1" colspan="1">688 Mb</td>
<td align="left" rowspan="1" colspan="1">
<p>98.1%</p>
<p>C:98.1%[S:95.5%,D:2.6%], F:0.1%, M:1.8%</p>
</td>
<td align="left" rowspan="1" colspan="1">60.9%</td>
<td align="left" rowspan="1" colspan="1">48.7</td>
<td align="left" rowspan="1" colspan="1">56.3</td>
<td align="left" rowspan="1" colspan="1">79.5</td>
<td align="left" rowspan="1" colspan="1">31,822 (52%)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270095-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270095-tbl1-note-0001">
<sup>
<sup>a</sup>
</sup><p class="display-inline">BUSCO C = complete and single‐copy, S = single‐copy, D = duplicated, F = fragmented, M = missing.</p>
</div></div></section><figure class="fig xbox font-sm" id="tpg270095-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1375/12368328/a71d70558b8f/TPG2-18-e70095-g003.jpg" loading="lazy" id="jats-graphic-1" height="462" width="546" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270095-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Synteny of the chromosomes of the subgenomes of allotetraploid <em>Solanum acaule</em> AA9. (a) Synteny between the subgenomes of <em>S. acaule</em> AA9 shows conservation of syntenic colinear gene blocks between chromosomes of the phased <em>S. acaule</em> subgenomes with gene density (red) and repeat density tracks (blue). (b) Hi‐C contact maps of the <em>S. acaule</em> AA9 genome assembly. Hi‐C reads were mapped to the phased assembly using HiC‐Pro (Servant et al., <a href="#tpg270095-bib-0113" class="usa-link" aria-describedby="tpg270095-bib-0113">2015</a>) and visualized using HiCExplorer (Wolff et al., <a href="#tpg270095-bib-0135" class="usa-link" aria-describedby="tpg270095-bib-0135">2020</a>). (c) BUSCO quality assessment results of <em>S. acaule</em> whole genome and subgenome assemblies. The bar plot shows the percent of BUSCO genes (<em>solanales_odb 10</em> dataset) present in each genome assembly. chr, chromosome; sub, subgenome.</p></figcaption></figure><p>A k‐mer analysis shows that the AA9 genome is highly heterozygous (5.5%) compared to cultivated potato tetraploids, for example, Q9 (1.91%), Otava (1.53%), and Atlantic (1.78%) (Hoopes et al., <a href="#tpg270095-bib-0056" class="usa-link" aria-describedby="tpg270095-bib-0056">2022</a>; Sun et al., <a href="#tpg270095-bib-0125" class="usa-link" aria-describedby="tpg270095-bib-0125">2022</a>; F. Wang et al., <a href="#tpg270095-bib-0131" class="usa-link" aria-describedby="tpg270095-bib-0131">2022</a>; Figure <a href="#tpg270095-supinfo-0001" class="usa-link">S2</a>; Table <a href="#tpg270095-supinfo-0001" class="usa-link">S2</a>), and that it is within the range of heterozygosity (3.7%–12%) seen in other wild polyploid potato species (Kyriakidou, Achakkagari et al., <a href="#tpg270095-bib-0072" class="usa-link" aria-describedby="tpg270095-bib-0072">2020</a>; Kyriakidou, Anglin et al., <a href="#tpg270095-bib-0073" class="usa-link" aria-describedby="tpg270095-bib-0073">2020</a>) but that it is lower than the range in modern tetraploid hybrids between <em>S. tuberosum</em> and wild species (10%–11%) (Achakkagari et al., <a href="#tpg270095-bib-0001" class="usa-link" aria-describedby="tpg270095-bib-0001">2024</a>; Table <a href="#tpg270095-supinfo-0001" class="usa-link">S2</a>).</p>
<p>The BUSCO analysis of the assembly shows that the two subgenomes enhance each other in terms of gene complement, as the whole assembly is more complete (99.1%) than either of the subgenomes (98.0% and 98.1%, respectively) (Figure <a href="#tpg270095-fig-0001" class="usa-link">1</a>; Table <a href="#tpg270095-tbl-0001" class="usa-link">1</a>). Combining genomes from different <em>Petota</em> species leads to an overall larger genome in terms of gene complement, as seen with, for example, the <em>Petota</em> super pangenome (Bozan et al., <a href="#tpg270095-bib-0012" class="usa-link" aria-describedby="tpg270095-bib-0012">2023</a>). It is known that subgenomes of allopolyploids often unevenly retain their gene complements (Steige &amp; Slotte, <a href="#tpg270095-bib-0123" class="usa-link" aria-describedby="tpg270095-bib-0123">2016</a>) and together with the advantage of heterosis (Comai, <a href="#tpg270095-bib-0026" class="usa-link" aria-describedby="tpg270095-bib-0026">2005</a>), it is part of their success. The Merqury analysis shows 99.6% of k‐mer completeness indicative of a high‐quality assembly, and also shows an overlap of 10%–20% of the reads that map equally well to both subgenomes. The consensus quality value is high, with 48.2, 49.1, and 48.7 for the total assembly, subgenome 1, and 2, respectively. The assembly quality was also estimated using CRAQ (K. Li et al., <a href="#tpg270095-bib-0080" class="usa-link" aria-describedby="tpg270095-bib-0080">2023</a>), which assesses regional and structural errors. An overall assembly quality of (S‐AQI) 92% and regional assembly quality (R‐AQI) of 97.3% resulted. The Hi‐C contact map (Figure <a href="#tpg270095-fig-0001" class="usa-link">1b</a>) shows a strong, continuous diagonal line indicating the contigs and scaffolds are correctly ordered and oriented, and the intra‐chromosomal contacts are well defined with high contact frequencies (Figure <a href="#tpg270095-supinfo-0001" class="usa-link">S3</a>). There are regions with no detectable Hi‐C contacts, corresponding to gaps in the interaction matrix, which likely represent centromeric regions, similar to what was found in other studies (Q. Li et al., <a href="#tpg270095-bib-0082" class="usa-link" aria-describedby="tpg270095-bib-0082">2019</a>). These gaps have also been used to assist with detecting centromeres (e.g., D. Xu et al., <a href="#tpg270095-bib-0149" class="usa-link" aria-describedby="tpg270095-bib-0149">2024</a>). The Hi‐C maps were manually reviewed, as well as reviewed using AutoHiC (Jiang et al., <a href="#tpg270095-bib-0063" class="usa-link" aria-describedby="tpg270095-bib-0063">2024</a>), detecting some “debris” corresponding to the no signal regions but no assembly errors such as inversions or translocations (Figure <a href="#tpg270095-supinfo-0001" class="usa-link">S4</a>). Three mismatches were identified at the start of two chromosomes by 3D‐DNA (Dudchenko et al., <a href="#tpg270095-bib-0032" class="usa-link" aria-describedby="tpg270095-bib-0032">2017</a>), but upon closer inspection, these regions were found to be misannotated as mismatches. These segments, each less than 700 bp, do not represent true assembly errors but likely result from algorithmic sensitivity to small structural variations or boundary artifacts.</p>
<p>In addition to the assembly assessment, QC was also performed on the genome annotation to evaluate its completeness, structural accuracy, and functional depth. The overall gene models exhibited high completeness, with 95.9% of conserved genes identified by BUSCO. Subgenome 1 and subgenome 2 individually showed completeness scores of 92.3% and 93.3%, respectively (Figure <a href="#tpg270095-supinfo-0001" class="usa-link">S5</a>). Psauron (Sommer et al., <a href="#tpg270095-bib-0116" class="usa-link" aria-describedby="tpg270095-bib-0116">2025</a>) assigned a high confidence score of 96.5, supporting the overall accuracy of predicted gene models. OMARK (Nevers et al., <a href="#tpg270095-bib-0096" class="usa-link" aria-describedby="tpg270095-bib-0096">2025</a>) assessment using Solanaceae‐specific HOGs found 95.4% of conserved genes present, with 79.3% duplicated and only 4.6% missing (Figure <a href="#tpg270095-supinfo-0001" class="usa-link">S6</a>). Consistency analysis showed that 92.6% of proteins matched gene families from the expected lineage, only 1.0% were inconsistent, and no contaminants were detected. Structurally, the annotation comprises 57,592 genes and 61,417 mRNAs, with a mean of 4.9 exons and 3.9 introns per transcript. Functional annotation showed strong coverage: 92% of gene models (56,610) had matches in Araport/SwissProt/TrEMBL, 72% (44,265) matched InterPro domains, and 45% (27,663) were associated with gene ontology (GO) terms. Together, these metrics indicate a well‐supported, high‐quality annotation suitable for downstream genomic and comparative analyses.</p></section><section id="tpg270095-sec-0170"><h3 class="pmc_sec_title">3.2. The progenitor species of allotetraploid <em>S. acaule</em> are in Clade 3 and 4</h3>
<p>The success of allotetraploid <em>S. acaule</em> as a cold‐adapted species, typically placed in Clade 4, is attributed to its hybrid origin, though the progenitor species have been unknown to date. Previous studies found that <em>S. acaule</em> likely has origins in both Clade 3 and 4 (e.g., Cai et al., <a href="#tpg270095-bib-0020" class="usa-link" aria-describedby="tpg270095-bib-0020">2012</a>), though the duplicated nature of plant genomes, the frequent interspecies hybridization in the <em>Solanum</em> genus, together with technical limitations on earlier methods made complete analyses difficult. While organellar data may be able to pinpoint the maternal progenitor species of an allopolyploid, it is typically problematic to include allopolyploids in phylogenies based on nuclear data as their hybrid nature adds confusion. However, with subgenome assemblies and an increasing number of comparator genomes available, studies are now better poised to answer such questions. Letting <em>S. acaule</em> subgenome 1 and 2 represent their respective progenitor species, Figure <a href="#tpg270095-fig-0002" class="usa-link">2</a> shows their respective placement in a <em>Solanum</em> section <em>Petota</em> phylogeny. Subgenome 2 places securely with <em>S. megistacolobum</em> genomes (Achakkagari et al., <a href="#tpg270095-bib-0001" class="usa-link" aria-describedby="tpg270095-bib-0001">2024</a>). Previous studies using plastid data also placed <em>S. acaule</em> with <em>Solanum megistacrolobum</em> (Nakagawa &amp; Hosaka, <a href="#tpg270095-bib-0093" class="usa-link" aria-describedby="tpg270095-bib-0093">2002</a>; Park, <a href="#tpg270095-bib-0100" class="usa-link" aria-describedby="tpg270095-bib-0100">2021</a>; Yan et al., <a href="#tpg270095-bib-0140" class="usa-link" aria-describedby="tpg270095-bib-0140">2023</a>), showing that this is undoubtedly the maternal progenitor species of <em>S. acaule</em>. Previous taxonomic studies on <em>S. megistacrolobum</em> and <em>S. acaule</em> using restriction fragment length polymorphism markers and nitrate reductase sequence data similarly indicated close relationship between the two species (Giannattasio &amp; Spooner, <a href="#tpg270095-bib-0043" class="usa-link" aria-describedby="tpg270095-bib-0043">1994</a>; Rodriguez &amp; Spooner, <a href="#tpg270095-bib-0107" class="usa-link" aria-describedby="tpg270095-bib-0107">2009</a>), and natural hybridization between the two species was found (Okada &amp; Clausen, <a href="#tpg270095-bib-0099" class="usa-link" aria-describedby="tpg270095-bib-0099">1982</a>). While <em>S. megistacrolobum</em> groups with Clade 4 species, in the present study as well as in studies based on the whole <em>Petota</em> super pangenome, it is found that <em>S. megistacrolobum</em> species form a separate and basal clade within Clade 4 South (Achakkagari et al., <a href="#tpg270095-bib-0001" class="usa-link" aria-describedby="tpg270095-bib-0001">2024</a>; Bozan et al., <a href="#tpg270095-bib-0012" class="usa-link" aria-describedby="tpg270095-bib-0012">2023</a>). Subgenome 1 forms a basal group in Clade 3, which explains early results showing that <em>S. acaule</em> carries Clade 3 alleles (Cai et al., <a href="#tpg270095-bib-0020" class="usa-link" aria-describedby="tpg270095-bib-0020">2012</a>). However, no close relative was found for subgenome 1. <em>S. acaule</em> may be a case where the genome of a now extinct species lives on only as a “ghost” part of this unique allopolyploid (Luo et al., <a href="#tpg270095-bib-0087" class="usa-link" aria-describedby="tpg270095-bib-0087">2017</a>), or alternatively the progenitor of subgenome 1 is a rare species that is yet to be sequenced. The results are also consistent with the genomes of Clade 3 species being generally slightly smaller than those of the Clade 4 species (Bozan et al., <a href="#tpg270095-bib-0012" class="usa-link" aria-describedby="tpg270095-bib-0012">2023</a>).</p>
<figure class="fig xbox font-sm" id="tpg270095-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1375/12368328/acfed4f56a2e/TPG2-18-e70095-g001.jpg" loading="lazy" id="jats-graphic-3" height="386" width="546" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270095-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>A multispecies coalescent based phylogenetic tree places the subgenomes of <em>Solanum acaule</em> with the closest relatives to identify progenitor species. Subgenome 1 is with Clade 3 species, but there is no close relative identified. Subgenome 2 groups with the <em>Solanum megistacrolobum</em> genomes with Clade 4.</p></figcaption></figure></section><section id="tpg270095-sec-0180"><h3 class="pmc_sec_title">3.3. C‐repeat binding factor genes</h3>
<p>The habitat of <em>S. acaule</em> in extreme high altitudes requires frost tolerance. One of the most fundamental gene families involved in plant response to cold temperature is the <em>C‐repeat binding factor</em> (<em>CBF</em>) gene family. To study the presence and evolution of <em>CBF</em> genes in frost‐tolerant <em>S. acaule</em> and frost‐sensitive <em>S. tuberosum</em> ‘Atlantic’, coding sequences were retrieved by a nucleotide BLAST search of their genome assemblies. Candidate genes from the two <em>S. acaule</em> subgenomes and the four <em>S. tuberosum</em> ‘Atlantic’ haplotypes (Hoopes et al., <a href="#tpg270095-bib-0056" class="usa-link" aria-describedby="tpg270095-bib-0056">2022</a>) were aligned, and a phylogenetic tree was constructed (Figure <a href="#tpg270095-fig-0003" class="usa-link">3a</a>). Most of the 11 <em>CBF</em> query sequences were found in all subgenomes or haplotypes with a few exceptions. <em>CBF1</em>, <em>CBF2A</em>, <em>CBF3</em>, <em>CBF4</em>, and <em>CBF7</em> genes were found in all haplotypes and subgenomes. The cluster of genes <em>CBF3</em>, <em>CBF1</em>, <em>CBF2A</em>, <em>CBF2B</em>, <em>CBF5</em>, and <em>CBF4</em> is organized within a small region (∼25 kb) on chromosome 3 in the mentioned order (Figure <a href="#tpg270095-fig-0003" class="usa-link">3b</a>). A similar gene organization has been previously reported in various <em>Solanum</em> species, for example, <em>Solanum commersonii</em> and <em>S. tuberosum</em> (Aversano et al., <a href="#tpg270095-bib-0006" class="usa-link" aria-describedby="tpg270095-bib-0006">2015</a>; Pennycooke et al., <a href="#tpg270095-bib-0102" class="usa-link" aria-describedby="tpg270095-bib-0102">2008</a>). While the order of the genes within this cluster is conserved across all haplotypes and subgenomes, the presence of all genes is not uniformly conserved. <em>CBF5</em> was only found in <em>S. acaule</em> subgenome 1 and in <em>S. tuberosum</em> ‘Atlantic’ haplotypes 1 and 2 (Figure <a href="#tpg270095-fig-0003" class="usa-link">3a</a>). <em>CBF5</em> was previously noted in <em>S. tuberosum</em>, but missing in <em>S. commersonii</em> (Aversano et al., <a href="#tpg270095-bib-0006" class="usa-link" aria-describedby="tpg270095-bib-0006">2015</a>). It is interesting that <em>S. acaule</em> subgenome 2, which is phylogenetically close to <em>S. commersonii</em> (Figure <a href="#tpg270095-fig-0002" class="usa-link">2</a>), also lacks <em>CBF5</em>. <em>CBF6</em> is located on chromosome 11 but is absent in <em>S. tuberosum</em> haplotypes 2 and 4. <em>CBF7</em> was located on chromosome 7 in both subgenomes of <em>S. acaule</em> and all haplotypes of <em>S. tuberosum</em> ‘Atlantic’. Several <em>CBF8</em>, <em>CBF9</em>, and <em>CBF10</em> genes were identified in the genome assemblies on chromosome 8; however, in <em>S. tuberosum</em>, some of these sequences originated from unphased contigs and thus these three genes were not included in further analyses.</p>
<figure class="fig xbox font-sm" id="tpg270095-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1375/12368328/8da0f8fa1609/TPG2-18-e70095-g002.jpg" loading="lazy" id="jats-graphic-5" height="434" width="546" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270095-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Maximum likelihood phylogenetic tree of C‐repeat binding factor (CBF) amino acid sequences from <em>Solanum acaule</em>, <em>Solanum tuberosum</em> ‘Atlantic’, and <em>Solanum tuberosum</em> DM1‐3. Bootstrap values are calculated with 100 replications using MEGAX <em>v.10.2.5</em> (Kumar et al., <a href="#tpg270095-bib-0071" class="usa-link" aria-describedby="tpg270095-bib-0071">2018</a>). Colored circles at nodes indicate bootstrap values. Bars on the right represent multiple sequence alignments of C‐repeat binding factor (CBF) protein sequences. (b) Diagram of synteny of <em>CBF</em> genes in chromosome 3 across subgenomes and haplotypes of <em>S. acaule</em> and <em>S. tuberosum</em> ‘Atlantic’. Complete <em>CBF</em> genes are in green, and pseudogenes are orange. (c) Dotplot of alignment region of chromosome 3 of subgenomes 1 and 2 of <em>S. acaule</em> containing CBF gene cluster. Annotated outside of axes are genes (blue arrows) and repetitive elements (red lines). Indels are marked as green bars. Large orange circle marks the location of the pseudogene <em>CBF2Bψ</em>. Small orange circle indicates the location of the pseudogene <em>CBF2Bψ2</em> in subgenome 1 that aligns with <em>CBF2B</em> of subgenome 2.</p></figcaption></figure><p>Notably, in the four <em>S. tuberosum</em> haplotypes and two <em>S. acaule</em> subgenomes, <em>CBF2B</em> is absent only in <em>S. acaule</em> subgenome 1. A missing <em>CBF2B</em> was previously noted in <em>S. commersonii</em> (Aversano et al., <a href="#tpg270095-bib-0006" class="usa-link" aria-describedby="tpg270095-bib-0006">2015</a>; Pennycooke et al., <a href="#tpg270095-bib-0102" class="usa-link" aria-describedby="tpg270095-bib-0102">2008</a>). It was suggested that <em>CBF2B</em> was lost from the <em>S. commersonii</em> genome, leaving behind a pseudogene, <em>CBF2Bψ</em>, as a remnant (Pennycooke et al., <a href="#tpg270095-bib-0102" class="usa-link" aria-describedby="tpg270095-bib-0102">2008</a>). However, our analysis of the <em>CBF</em> region reveals that this pseudogene (<em>CBF2Bψ</em>) is present in both subgenomes of <em>S. acaule</em> (Figure <a href="#tpg270095-fig-0003" class="usa-link">3c</a>, orange circle) as well as in haplotypes 1, 3, and 4 in the <em>S. tuberosum</em> ‘Atlantic’, always located between the copies of <em>CBF2A</em> and <em>CBF2B</em> (except in <em>S. acaule</em> subgenome 1, which lacks the <em>CBF2B</em> coding sequence and where the pseudogene is downstream of the <em>CBF2</em>; Figure <a href="#tpg270095-fig-0003" class="usa-link">3b</a>). This suggests that the pseudogene in <em>S. commersonii</em> (Pennycooke et al., <a href="#tpg270095-bib-0102" class="usa-link" aria-describedby="tpg270095-bib-0102">2008</a>) is not a mutated <em>CBF2B</em>, but a different copy of a <em>CBF</em> gene that was already a pseudogene in a common ancestor of these species. Additionally, a ∼30 bp small segment (that we label <em>CBF2Bψ2</em>) of the authentic <em>CBF2B</em> upstream sequence was identified in <em>S. acaule</em> chr3_1 (subgenome 1) at the edge of a deletion where a <em>CBF2B</em> gene would be expected (Figure <a href="#tpg270095-fig-0003" class="usa-link">3b</a>; Figure <a href="#tpg270095-fig-0003" class="usa-link">3c</a>, orange arrow, deletions marked as green lines). <em>Solanum commersonii</em> also has a <em>CBF3</em> pseudogene <em>ψCBF3</em> (Aversano et al., <a href="#tpg270095-bib-0006" class="usa-link" aria-describedby="tpg270095-bib-0006">2015</a>) that was not detected in either <em>S. acaule</em> or <em>S. tuberosum</em>. The Figure <a href="#tpg270095-fig-0003" class="usa-link">3c</a> dot plot also shows two additional significant indels (green lines): an insertion in chr3_2 (subgenome 2), containing an unidentified gene that seems to be a duplication of a different region from chromosome 3, and a deletion representing the loss of <em>CBF5</em> in subgenome 2. Our findings support the idea that the <em>CBF</em> gene family in <em>Solanum</em> is undergoing rapid evolution driven by processes of expansion via gene duplication followed by gene loss and pseudogenization (Aversano et al., <a href="#tpg270095-bib-0006" class="usa-link" aria-describedby="tpg270095-bib-0006">2015</a>).</p></section><section id="tpg270095-sec-0190"><h3 class="pmc_sec_title">3.4. Transcriptomes of cold‐adapted <em>S. acaule</em> AA9 and non‐cold‐adapted <em>S. tuberosum</em> ‘Atlantic’</h3>
<p>To analyze freezing response‐related differences, <em>S. acaule</em> AA9 and <em>S. tuberosum</em> ‘Atlantic’ plants were subjected to freezing conditions and their transcriptomes sequenced and assembled. The <em>S. acaule</em> AA9 transcriptome (control and freezing) contains 245,390 total transcripts and that of <em>S. tuberosum</em> ‘Atlantic’ contains 252,591 transcripts, with total transcriptome assembly lengths of 360 Mb and 470 Mb, respectively. A BUSCO completeness assessment shows that both transcriptome assemblies are highly complete with 96.2% and 95.9% for <em>S. acaule</em> and <em>S. tuberosum</em>, respectively. Because not all genes are expressed at the same time, the BUSCO scores for the transcriptomes are lower than the scores of the genome assemblies.</p>
<p>Generally, while the transcriptomes of <em>S. acaule</em> and <em>S. tuberosum</em> both have a similar CG (cytosine guanine) content (39.06% and 39.16%, respectively), the <em>S. acaule</em> transcripts are shorter, averaging 1469 bp, compared to the average of 1863 bp for <em>S. tuberosum</em> ‘Atlantic’. Differences in average transcript length between <em>Solanum</em> species have previously been observed (Gramazio et al., <a href="#tpg270095-bib-0046" class="usa-link" aria-describedby="tpg270095-bib-0046">2016</a>). When considering only protein‐coding transcripts, the median transcript length is 2069 bp in <em>S. acaule</em> and 2312 bp in <em>S. tuberosum</em>, with the latter being significantly longer (Mann–Whitney U test, <em>p</em> &lt; 0.001). Within <em>S. acaule</em>, subgenome 2 has slightly longer transcripts (2114 bp) and coding sequence (CDS) (1136 bp) on average than subgenome 1 (2057 bp transcripts and 1062 bp CDS; <em>p</em> &lt; 0.001). In contrast, <em>S. tuberosum</em> transcript and CDS lengths remain consistent across the four haplotypes (protein‐coding transcript average ranges from 2316 to 2323 bp; CDS from 1193 to 1203 bp; <em>p</em> &gt; 0.01 in all comparisons). This difference in transcript length may be partly explained by differences in the set of genes being expressed in each species. However, both transcriptomes exhibit comparable BUSCO completeness scores and total transcript numbers, suggesting that neither assembly is missing major gene content. An alternative explanation is that <em>S. tuberosum</em> expresses a higher proportion of longer transcript isoforms, possibly due to gene structural differences. The longer CDS average in <em>S. tuberosum</em> suggests that this is the case, rather than the primary source of difference being from alternative splicing and/or UTR length.</p>
<p>Transcripts were mapped to their respective reference subgenome or haplotype. In <em>S. acaule</em>, 229,953 transcripts or 93.7% of all transcripts were successfully mapped, and in <em>S. tuberosum</em> ‘Atlantic’, 226,933 transcripts or 89.8% were successfully mapped. The lower mapping success rate in the latter may be due to the higher number of unphased scaffolds in the <em>S. tuberosum</em> ‘Atlantic’ assembly. In <em>S. acaule</em>, 55.1% of the transcripts mapped to subgenome 2 (Table <a href="#tpg270095-tbl-0002" class="usa-link">2</a>), which contains 52% of all <em>S. acaule</em> genes (Table <a href="#tpg270095-tbl-0001" class="usa-link">1</a>), and 44.8% of the transcripts mapped to subgenome 1 (Table <a href="#tpg270095-tbl-0002" class="usa-link">2</a>), which contains 48% of the genes (Table <a href="#tpg270095-tbl-0001" class="usa-link">1</a>). In <em>S. tuberosum</em>, haplotypes 1 and 2 exhibit lower rates of mapped transcripts (19.2% and 22.5%, respectively) than haplotypes 3 and 4 (25.0% and 24.3%; Table <a href="#tpg270095-tbl-0002" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="tpg270095-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Proportion of differentially expressed transcripts in each subgenome in <em>Solanum acaule</em> and in each haplotype in <em>Solanum tuberosum</em> ‘Atlantic’.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1"> </th>
<th align="left" rowspan="1" colspan="1">Total transcripts mapped (%)</th>
<th align="left" rowspan="1" colspan="1">DE of mapped transcripts (LFC &gt; 1; <em>p</em>
<sub>adj</sub> &lt; 0.05)</th>
<th align="left" rowspan="1" colspan="1">DE Freeze up regulated (%)</th>
<th align="left" rowspan="1" colspan="1">DE Freeze down regulated (%)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>S. acaule</em> total (total transcripts 245,390)</td>
<td align="left" rowspan="1" colspan="1">229,953 (93.7% of total transcripts)</td>
<td align="left" rowspan="1" colspan="1">3761 (100)</td>
<td align="left" rowspan="1" colspan="1">2238 (100)</td>
<td align="left" rowspan="1" colspan="1">1523 (100)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>S. acaule</em> sub 1</td>
<td align="left" rowspan="1" colspan="1">102,964 (44.8)</td>
<td align="left" rowspan="1" colspan="1">1574 (41.9)</td>
<td align="left" rowspan="1" colspan="1">1057 (47.2)</td>
<td align="left" rowspan="1" colspan="1">517 (33.9)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>S. acaule</em> sub 2</td>
<td align="left" rowspan="1" colspan="1">126,694 (55.1)</td>
<td align="left" rowspan="1" colspan="1">2089 (55.5)</td>
<td align="left" rowspan="1" colspan="1">1136 (50.8)</td>
<td align="left" rowspan="1" colspan="1">953 (62.6)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Scaffolds (transcripts mapping to unplaced genome scaffolds)</td>
<td align="left" rowspan="1" colspan="1">306 (0.1)</td>
<td align="left" rowspan="1" colspan="1">98 (2.6)</td>
<td align="left" rowspan="1" colspan="1">45 (2.0)</td>
<td align="left" rowspan="1" colspan="1">53 (3.5)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>S. tuberosum</em> total (total transcripts 252,591)</td>
<td align="left" rowspan="1" colspan="1">226,933 (89.8% of total transcripts)</td>
<td align="left" rowspan="1" colspan="1">16,516 (100)</td>
<td align="left" rowspan="1" colspan="1">9403 (100)</td>
<td align="left" rowspan="1" colspan="1">7113 (100)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>S. tuberosum</em> hap 1</td>
<td align="left" rowspan="1" colspan="1">43,611 (19.2)</td>
<td align="left" rowspan="1" colspan="1">2910 (17.6)</td>
<td align="left" rowspan="1" colspan="1">1647 (17.5)</td>
<td align="left" rowspan="1" colspan="1">1263 (17.8)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>S. tuberosum</em> hap 2</td>
<td align="left" rowspan="1" colspan="1">50,971 (22.5)</td>
<td align="left" rowspan="1" colspan="1">3289 (19.9)</td>
<td align="left" rowspan="1" colspan="1">1869 (19.9)</td>
<td align="left" rowspan="1" colspan="1">1420 (20.0)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>S. tuberosum</em> hap 3</td>
<td align="left" rowspan="1" colspan="1">56,817 (25.0)</td>
<td align="left" rowspan="1" colspan="1">3805 (23.0)</td>
<td align="left" rowspan="1" colspan="1">2178 (23.2)</td>
<td align="left" rowspan="1" colspan="1">1627 (22.9)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>S. tuberosum</em> hap 4</td>
<td align="left" rowspan="1" colspan="1">55,036 (24.3)</td>
<td align="left" rowspan="1" colspan="1">3612 (21.9)</td>
<td align="left" rowspan="1" colspan="1">2068 (22.0)</td>
<td align="left" rowspan="1" colspan="1">1544 (21.7)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Scaffolds (transcripts mapping to unplaced genome scaffolds)</td>
<td align="left" rowspan="1" colspan="1">20,498 (9.0)</td>
<td align="left" rowspan="1" colspan="1">2900 (17.6)</td>
<td align="left" rowspan="1" colspan="1">1641 (17.5)</td>
<td align="left" rowspan="1" colspan="1">1259 (17.7)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270095-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270095-tbl2-note-0001"><p>
<em>Note</em>: Differentially expressed transcripts were selected with the cutoff values |log2Foldchange (LFC )| &gt; 1 and <em>p</em>adj &lt; 0.05.</p></div>
<div class="fn" id="tpg270095-tbl2-note-0002"><p>Abbreviations: hap, haplotype; sub, subgenome.</p></div>
</div></section></section><section id="tpg270095-sec-0200"><h3 class="pmc_sec_title">3.5. Overall transcription of homoeologous gene pairs in <em>S. acaule</em> shows no subgenome preference</h3>
<p>To detect any potential overall expression bias toward either <em>S. acaule</em> subgenome, relative expression levels of homoeologous genes from both subgenomes were compared. Because the two subgenomes have partially different gene complements, a subset of 16,325 genes that have homoeologs on both <em>S. acaule</em> subgenomes was identified using MCScanX (Y. Wang et al., <a href="#tpg270095-bib-0133" class="usa-link" aria-describedby="tpg270095-bib-0133">2012</a>). A total of 12,673 of these homoeologous gene pairs (77.6%) were expressed in either one or both subgenomes in freezing conditions, while in control conditions, 12,561 gene pairs (76.9%) were expressed. Under freezing conditions, for 50.32% of the gene pairs the copy from subgenome 2 was favored, compared to the 49.68% of gene pairs where the copy from subgenome 1 was favored (Figure <a href="#tpg270095-fig-0004" class="usa-link">4a</a>). Similarly, under control conditions, for 50.23% of the expressed gene pairs, the gene from subgenome 2 was favored, while for 49.77% the gene from subgenome 1 was favored. These results suggest an overall balanced expression pattern between the subgenomes across both conditions in this subset of homoeologous genes. Furthermore, it suggests that the progenitors of <em>S. acaule</em> co‐existed in the cold high‐altitude range in the high Andes as both transcriptomes contribute to adaptation to cold and frost, which concurs with reported cold and frost tolerance for <em>S. megistacrolobum</em> and Clade 3 species (Hijmans et al., <a href="#tpg270095-bib-0053" class="usa-link" aria-describedby="tpg270095-bib-0053">2003</a>) as well as the <em>Petota</em> super pangenome analysis of core genes in the Clade 3 species showing an enrichment of cold and abiotic stress‐related genes (Bozan et al., <a href="#tpg270095-bib-0012" class="usa-link" aria-describedby="tpg270095-bib-0012">2023</a>).</p>
<figure class="fig xbox font-sm" id="tpg270095-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1375/12368328/345ebfec3795/TPG2-18-e70095-g004.jpg" loading="lazy" id="jats-graphic-7" height="549" width="546" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270095-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Overlap of differentially expressed (DE) homeologous gene pairs across subgenomes in <em>Solanum acaule</em>, and homologous gene sets across haplotypes in <em>Solanum tuberosum</em> ‘Atlantic’. Genes were selected as DE with cutoff criteria of |log2Foldchange | &gt; 1 and <em>p</em>adj &lt; 0.05. (b) Distribution of expression bias of 1:1 homeologous gene pairs between subgenomes of <em>S. acaule</em> under control and freezing conditions. Negative log2 values indicate bias toward subgenome 1 and positive values indicate bias toward subgenome 2. Genes are only considered if the differential expression is significant with <em>p</em>adj &gt; 0.05 (dark colored bars).</p></figcaption></figure></section><section id="tpg270095-sec-0210"><h3 class="pmc_sec_title">3.6. Differential gene expression analysis reveals <em>S. acaule</em> subgenome 2 preference in downregulated transcripts after freezing treatment</h3>
<p>Differential gene expression analysis of freezing treatment versus control was carried out for each of the two species. Overall, the results are very similar to the proportion of total number of transcripts mapped to each subgenome and haplotype. That is, <em>S. acaule</em> subgenome 1 has a smaller proportion of the DE transcripts (41.9%) than its proportion of genes in the whole <em>S. acaule</em> genome (48%), and subgenome 2 has a larger proportion (55.5%) than its proportion of genes (52%) (Table <a href="#tpg270095-tbl-0002" class="usa-link">2</a>). Interestingly though, in <em>S. acaule</em> subjected to freezing, only 33.9% of the downregulated differentially expressed (DE) transcripts originate from subgenome 1, while 62.6% originate from subgenome 2 (Table <a href="#tpg270095-tbl-0002" class="usa-link">2</a>). In <em>S. tuberosum</em> ‘Atlantic’ the DE transcripts are mapped more or less equally to all four haplotypes—around 20% (Table <a href="#tpg270095-tbl-0002" class="usa-link">2</a>).</p>
<p>In all <em>S. tuberosum</em> haplotypes, about 26% fewer genes are differentially downregulated than upregulated in freezing conditions. In <em>S. acaule</em>, on the other hand, subgenome 2 had only 16% fewer genes downregulated than upregulated, while subgenome 1 had 52% fewer genes downregulated than upregulated. This indicates a possibility that genes that need to be downregulated in freezing conditions were already downregulated in subgenome 1, and so show no differential expression (reflecting a cold‐adapted Clade 3 member), similar to other cold‐adapted plants, for example, <em>Oxytropis arctobia</em> (Archambault &amp; Strömvik, <a href="#tpg270095-bib-0005" class="usa-link" aria-describedby="tpg270095-bib-0005">2011</a>). Alternatively, it may be that those genes that need to be downregulated are favoring the copy in subgenome 2, which had many more genes downregulated in freezing conditions than subgenome 1. Subgenome 2, essentially <em>S. megistacrolobum</em>, which is known for its frost tolerance (Hijmans et al., <a href="#tpg270095-bib-0053" class="usa-link" aria-describedby="tpg270095-bib-0053">2003</a>), and its transcriptome responds more strongly and with different gene homologs to freezing conditions.</p></section><section id="tpg270095-sec-0220"><h3 class="pmc_sec_title">3.7. Differential expression of homoeologous gene pairs (<em>S. acaule</em>) or homologous sets (Atlantic)</h3>
<p>To detect potential bias in the DE gene pairs rather than in all expressed genes, a set similar to the homoeologous gene pairs in <em>S. acaule</em> was generated for <em>S. tuberosum</em> using MCScanX (Y. Wang et al., <a href="#tpg270095-bib-0133" class="usa-link" aria-describedby="tpg270095-bib-0133">2012</a>). A set of 9331 genes that have homologs on at least two of the Atlantic haplotypes resulted. To investigate expression preference in <em>S. acaule</em> and <em>S. tuberosum</em>, DE transcripts (both up‐ and downregulated) were assigned to the homoeologous gene pairs (<em>S. acaule</em>) or homologous gene sets (<em>S. tuberosum</em>). As shown in the Venn diagram (Figure <a href="#tpg270095-fig-0004" class="usa-link">4b</a>), a greater number of homoeologous DE genes originate from <em>S. acaule</em> subgenome 2 (1013) compared to the homoeologous DE genes that originate from subgenome 1 (760). This supports the above finding that subgenome 2 has more differentially expressed genes overall, even when looking at the same set of gene complement (the set of homoeologous gene pairs). Only a small set of homoeologous genes (143) were differentially expressed in both subgenomes (Figure <a href="#tpg270095-fig-0004" class="usa-link">4b</a>). In <em>S. tuberosum</em>, the homolog preference in DE genes ranged between 1605 and 2232 for the four haplotypes, and similarly to <em>S. acaule</em>, only 144 genes had homologs DE in all four haplotypes.</p>
<p>These results are supported by studies of other allopolyploids, where one subgenome exhibits higher overall expression levels after hybridization (Alger &amp; Edger, <a href="#tpg270095-bib-0003" class="usa-link" aria-describedby="tpg270095-bib-0003">2020</a>; Steige &amp; Slotte, <a href="#tpg270095-bib-0123" class="usa-link" aria-describedby="tpg270095-bib-0123">2016</a>). In <em>S. acaule</em>, overall, subgenome 2 has more genes than subgenome 1 (52% vs. 48% of all genes in the genome), more transcripts (55% vs. 45%), and more downregulated genes in freezing conditions (62.6% vs. 33.9%). When looking at a set of expressed homoeologous gene pairs, each subgenome had about 50% of the favored homoeologs, though importantly, looking at the DE homoeologous gene pairs, subgenome 2 has many more (1013 vs. 760). This contrast in gene regulation could be linked to the origins of each parent species, each with different regulatory thresholds for gene expression as have been observed in other allopolyploid species (Chalhoub et al., <a href="#tpg270095-bib-0022" class="usa-link" aria-describedby="tpg270095-bib-0022">2014</a>; Garsmeur et al., <a href="#tpg270095-bib-0042" class="usa-link" aria-describedby="tpg270095-bib-0042">2014</a>; VanBuren et al., <a href="#tpg270095-bib-0129" class="usa-link" aria-describedby="tpg270095-bib-0129">2020</a>; Xing et al., <a href="#tpg270095-bib-0138" class="usa-link" aria-describedby="tpg270095-bib-0138">2023</a>). Expression bias toward one subgenome is commonly observed (Buggs et al., <a href="#tpg270095-bib-0019" class="usa-link" aria-describedby="tpg270095-bib-0019">2011</a>; Edger et al., <a href="#tpg270095-bib-0036" class="usa-link" aria-describedby="tpg270095-bib-0036">2017</a>; Li et al., <a href="#tpg270095-bib-0076" class="usa-link" aria-describedby="tpg270095-bib-0076">2014</a>; Schnable et al., <a href="#tpg270095-bib-0112" class="usa-link" aria-describedby="tpg270095-bib-0112">2011</a>; H. Tang et al., <a href="#tpg270095-bib-0128" class="usa-link" aria-describedby="tpg270095-bib-0128">2012</a>; Yang et al., <a href="#tpg270095-bib-0141" class="usa-link" aria-describedby="tpg270095-bib-0141">2016</a>; Yoo et al., <a href="#tpg270095-bib-0142" class="usa-link" aria-describedby="tpg270095-bib-0142">2012</a>). In the case of the allopolyploid <em>Isoetes sinensis</em>, it was found that one of the subgenomes provided more contributions to high‐altitude adaptation, which corresponds with the parental species that is better suited to high altitudes (Wei et al., <a href="#tpg270095-bib-0134" class="usa-link" aria-describedby="tpg270095-bib-0134">2022</a>).</p></section><section id="tpg270095-sec-0230"><h3 class="pmc_sec_title">3.8. Abiotic stress response‐related genes are preferentially expressed in <em>S. acaule</em> subgenome 2</h3>
<p>Two <em>S. acaule CBF</em> genes, <em>CBF1</em> from subgenome 1 and <em>CBF4</em> from subgenome 2, were significantly upregulated in freezing conditions compared to the control. In contrast, in <em>S. tuberosum</em> ‘Atlantic’, only <em>CBF1</em> is upregulated in freezing conditions and only the copy on haplotype 1. Interestingly, overexpression of <em>S. tuberosum CBF1</em> and <em>CBF4</em> in transgenic <em>Arabidopsis thaliana</em> has been shown to increase tolerance to low temperatures and to affect expression of <em>COR</em> genes, with <em>CBF4</em> having a greater effect than <em>CBF1</em> (Song et al., <a href="#tpg270095-bib-0117" class="usa-link" aria-describedby="tpg270095-bib-0117">2021</a>). Our results support previous reports from <em>Solanum</em> species, indicating that among all the <em>CBF</em> genes in the cluster, only <em>CBF1</em> and <em>CBF4</em> are cold induced (Pennycooke et al., <a href="#tpg270095-bib-0102" class="usa-link" aria-describedby="tpg270095-bib-0102">2008</a>; Zhang et al., <a href="#tpg270095-bib-0150" class="usa-link" aria-describedby="tpg270095-bib-0150">2004</a>); they also contrast with other findings in cold‐tolerant potato and tomato species (Aversano et al., <a href="#tpg270095-bib-0006" class="usa-link" aria-describedby="tpg270095-bib-0006">2015</a>; Mboup et al., <a href="#tpg270095-bib-0090" class="usa-link" aria-describedby="tpg270095-bib-0090">2012</a>), where all <em>CBF</em> genes are cold‐induced. This inconsistency in <em>CBF</em> gene expression under low‐temperature conditions appears to be common across <em>Solanum</em> species, highlighting the complexity of the CBF regulatory system landscape.</p>
<p>CBL protein, a Ca<sup>2+</sup> signal transducer, was previously identified as being significantly upregulated in frost‐tolerant <em>Solanum</em> species, including <em>S. acaule</em> (Chen et al. <a href="#tpg270095-bib-0023" class="usa-link" aria-describedby="tpg270095-bib-0023">2022</a>). However, in our study, the <em>CBL</em> gene was upregulated in <em>S. tuberosum</em>, while in <em>S. acaule</em>, though present in the transcriptomes of both control and cold conditions, no significant differential expression was detected. These discrepancies may stem from differences in cold treatment protocols. Despite this, a protein that decodes Ca<sup>2+</sup> signaling together with CBL, CBL‐interacting protein kinase, was found in our study to be DE in both <em>S. acaule</em> and the <em>S. tuberosum</em> ‘Atlantic’ in all subgenomes and haplotypes. Differences in experimental design, particularly growth conditions, likely contribute to the varying reports of which <em>CBF</em> genes are cold‐induced or cold‐responsive. To minimize the confounding effects of other abiotic factors, our approach was to investigate plants grown in vitro rather than in soil. Though this is a common practice, in vitro culture has been shown to impact transcriptome expression profiles. For instance, in <em>Camellia sinensis</em>, the upregulation of lignin biosynthesis, cytochrome, and strigolactone response genes was higher in plants grown in vitro compared to those grown in vivo under nitrogen deficiency (Samarina et al., <a href="#tpg270095-bib-0109" class="usa-link" aria-describedby="tpg270095-bib-0109">2024</a>). Cold treatment protocols may also contribute to the discrepancies observed across studies. For instance, L. Chen et al. (<a href="#tpg270095-bib-0023" class="usa-link" aria-describedby="tpg270095-bib-0023">2022</a>) exposed plants to 4°C for 3 h, whereas our experiment used a gradual temperature decrease from 22°C to −4°C, with chilling conditions (&lt;4°C) maintained for 5 h. Collectively, these differences in growth environment likely account for variation in the expression of <em>CBF</em> and <em>CBL</em> genes across <em>Solanum</em> species.</p>
<p>A GO enrichment analysis on upregulated genes revealed the most abundant terms in the subgenome 2 transcriptome, such as ribosome biogenesis, cell wall biogenesis, DNA‐binding transcription factor (TF) activity, and phospholipid biosynthesis. Among these genes, TFs such as CBF4, calmodulin‐binding transcription activators, and auxin response factors were preferentially upregulated in subgenome 2, while other genes, such as the REI1‐LIKE (REIL) protein, a ribosome biogenesis factor, were exclusively upregulated in subgenome 2. REIL proteins are cytosolic 60S ribosomal biogenesis factors crucial for cold acclimation, as they accelerate de novo ribosome synthesis at low temperatures (Cheong et al., <a href="#tpg270095-bib-0025" class="usa-link" aria-describedby="tpg270095-bib-0025">2021</a>). In contrast, genes upregulated in subgenome 1 were enriched for gene silencing (e.g., Argonaute), circadian regulation, beta‐amylase activity, and nitrate transmembrane transporter activity. These findings indicate that frost tolerance in <em>S. acaule</em> is likely the combined result of the two subgenomes; that is, the greater cold‐responsiveness indicates that especially subgenome 2 is cold‐acclimating and responds to cold temperatures by inducing genes such as <em>CBF4</em> and <em>REIL</em>, while other important genes such as <em>CBL</em> are already expressed.</p></section><section id="tpg270095-sec-0240"><h3 class="pmc_sec_title">3.9. 
<em>Solanum acaule</em> and <em>S. tuberosum</em> have processes in common in freezing conditions</h3>
<p>Enriched GO terms among DE genes were analyzed to investigate potential functional differences and similarities between <em>S. acaule</em> and <em>S. tuberosum</em> plants under freezing conditions. The results show that the two species share similar overrepresented processes (Table <a href="#tpg270095-tbl-0003" class="usa-link">3</a>). Among the most upregulated functions are “transcription factor activity” and “DNA‐binding transcription factor activity,” including TFs such as WRKY TFs, ethylene‐responsive transcription factor (ERFs), NAC (NAM, ATAF, and CUC) TFs, and cycling DOF factors. WRKY is a large plant‐specific family of genes responsible for regulating abiotic stress response in plants (Khoso et al., <a href="#tpg270095-bib-0066" class="usa-link" aria-describedby="tpg270095-bib-0066">2022</a>). ERFs are also a conserved TF family in plants related to the Apetala 2 domain involved in the ethylene signal transduction pathway of the abiotic stress response (Fujimoto et al., <a href="#tpg270095-bib-0039" class="usa-link" aria-describedby="tpg270095-bib-0039">2000</a>). DOF TFs influence multiple aspects of growth and development but are also involved in regulating many abiotic stress‐responsive genes (Corrales et al., <a href="#tpg270095-bib-0027" class="usa-link" aria-describedby="tpg270095-bib-0027">2014</a>).</p>
<section class="tw xbox font-sm" id="tpg270095-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Gene ontology enrichment analysis on differentially expressed genes from <em>Solanum acaule</em> and <em>Solanum tuberosum</em>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead>
<tr>
<th colspan="4" style="border-bottom:solid 1px #000000" align="left" rowspan="1">
<em>S. acaule</em> AA9</th>
<th colspan="4" style="border-bottom:solid 1px #000000" align="left" rowspan="1">
<em>S. tuberosum</em> ‘Atlantic’</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1">Upregulated</th>
<th align="left" rowspan="1" colspan="1">
<em>p</em>‐value</th>
<th align="left" rowspan="1" colspan="1">Downregulated</th>
<th align="left" rowspan="1" colspan="1">
<em>p</em>‐value</th>
<th align="left" rowspan="1" colspan="1">Upregulated</th>
<th align="left" rowspan="1" colspan="1">
<em>p</em>‐value</th>
<th align="left" rowspan="1" colspan="1">Downregulated</th>
<th align="left" rowspan="1" colspan="1">
<em>p</em>‐value</th>
</tr>
<tr style="border-bottom:solid 1px #000000"><th colspan="8" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Common to both species</th></tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Fruit ripening</td>
<td align="left" rowspan="1" colspan="1">9.E‐16</td>
<td align="left" rowspan="1" colspan="1">Light harvesting in photosystem I</td>
<td align="left" rowspan="1" colspan="1">1.E‐58</td>
<td align="left" rowspan="1" colspan="1">Fruit ripening</td>
<td align="left" rowspan="1" colspan="1">2.E‐26</td>
<td align="left" rowspan="1" colspan="1">Light harvesting in photosystem I</td>
<td align="left" rowspan="1" colspan="1">5.E‐77</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cell wall</td>
<td align="left" rowspan="1" colspan="1">9.E‐14</td>
<td align="left" rowspan="1" colspan="1">Photosystem I</td>
<td align="left" rowspan="1" colspan="1">1.E‐52</td>
<td align="left" rowspan="1" colspan="1">Cell wall</td>
<td align="left" rowspan="1" colspan="1">3.E‐15</td>
<td align="left" rowspan="1" colspan="1">Photosystem I</td>
<td align="left" rowspan="1" colspan="1">2.E‐67</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Xyloglucan metabolic process</td>
<td align="left" rowspan="1" colspan="1">9.E‐13</td>
<td align="left" rowspan="1" colspan="1">Chlorophyll binding</td>
<td align="left" rowspan="1" colspan="1">1.E‐53</td>
<td align="left" rowspan="1" colspan="1">Xyloglucan metabolic process</td>
<td align="left" rowspan="1" colspan="1">9.E‐21</td>
<td align="left" rowspan="1" colspan="1">Chlorophyll binding</td>
<td align="left" rowspan="1" colspan="1">1.E‐57</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">DNA‐binding transcription factor activity</td>
<td align="left" rowspan="1" colspan="1">7.E‐10</td>
<td align="left" rowspan="1" colspan="1">Photosystem II</td>
<td align="left" rowspan="1" colspan="1">1.E‐46</td>
<td align="left" rowspan="1" colspan="1">DNA‐binding transcription factor activity</td>
<td align="left" rowspan="1" colspan="1">1.E‐27</td>
<td align="left" rowspan="1" colspan="1">Photosystem II</td>
<td align="left" rowspan="1" colspan="1">3.E‐48</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Transcription cis‐regulatory region binding</td>
<td align="left" rowspan="1" colspan="1">8.E‐11</td>
<td align="left" rowspan="1" colspan="1">Response to light stimulus</td>
<td align="left" rowspan="1" colspan="1">6.E‐29</td>
<td align="left" rowspan="1" colspan="1">Transcription cis‐regulatory region binding</td>
<td align="left" rowspan="1" colspan="1">5.E‐20</td>
<td align="left" rowspan="1" colspan="1">Response to light stimulus</td>
<td align="left" rowspan="1" colspan="1">9.E‐26</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Polysaccharide binding</td>
<td align="left" rowspan="1" colspan="1">3.E‐10</td>
<td align="left" rowspan="1" colspan="1">Chloroplast thylakoid membrane</td>
<td align="left" rowspan="1" colspan="1">1.E‐23</td>
<td align="left" rowspan="1" colspan="1">Polysaccharide binding</td>
<td align="left" rowspan="1" colspan="1">1.E‐12</td>
<td align="left" rowspan="1" colspan="1">Chloroplast thylakoid membrane</td>
<td align="left" rowspan="1" colspan="1">2.E‐16</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Glutathione metabolic process (photoprotection)</td>
<td align="left" rowspan="1" colspan="1">2.E‐07</td>
<td align="left" rowspan="1" colspan="1">Response to heat</td>
<td align="left" rowspan="1" colspan="1">5.E‐14</td>
<td align="left" rowspan="1" colspan="1">Glutathione metabolic process (photoprotection)</td>
<td align="left" rowspan="1" colspan="1">2.E‐14</td>
<td align="left" rowspan="1" colspan="1">Response to heat</td>
<td align="left" rowspan="1" colspan="1">7.E‐10</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cellular response to hypoxia</td>
<td align="left" rowspan="1" colspan="1">7.E‐09</td>
<td align="left" rowspan="1" colspan="1">Protein refolding</td>
<td align="left" rowspan="1" colspan="1">2.E‐10</td>
<td align="left" rowspan="1" colspan="1">Cellular response to hypoxia</td>
<td align="left" rowspan="1" colspan="1">1.E‐21</td>
<td align="left" rowspan="1" colspan="1">Protein refolding</td>
<td align="left" rowspan="1" colspan="1">5.E‐08</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000"><th colspan="8" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Unique to either species</th></tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Response to mechanical stimulus</td>
<td align="left" rowspan="1" colspan="1">4.E‐11</td>
<td align="left" rowspan="1" colspan="1">Unfolded protein binding</td>
<td align="left" rowspan="1" colspan="1">2.E‐09</td>
<td align="left" rowspan="1" colspan="1">Apoplast</td>
<td align="left" rowspan="1" colspan="1">2.E‐15</td>
<td align="left" rowspan="1" colspan="1">DNA‐templated transcription elongation</td>
<td align="left" rowspan="1" colspan="1">3.E‐08</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Response to cold</td>
<td align="left" rowspan="1" colspan="1">6.E‐09</td>
<td align="left" rowspan="1" colspan="1">Protein complex oligomerization</td>
<td align="left" rowspan="1" colspan="1">6.E‐08</td>
<td align="left" rowspan="1" colspan="1">Defense response</td>
<td align="left" rowspan="1" colspan="1">4.E‐15</td>
<td align="left" rowspan="1" colspan="1">Entrainment of circadian clock</td>
<td align="left" rowspan="1" colspan="1">8.E‐08</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270095-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Other upregulated processes that were identified in common between the two species include fruit ripening, comprising genes implicated in the metabolism of plant hormones such as ethylene (1‐aminocyclopropane‐1‐carboxylate oxidase 1) and abscisic acid (abscisic acid 8′‐hydroxylase 1), as well as genes involved in cell wall modification (polygalacturonase‐1 non‐catalytic subunit beta, xyloglucan endotransglucosylase/hydrolase).</p></section></section><section id="tpg270095-sec-0250"><h2 class="pmc_sec_title">4. CONCLUSIONS</h2>
<p>The genome of allotetraploid potato wild relative <em>Solanum acaule</em> Bitt. (AA9) was sequenced and phased to the subgenome level. The progenitor species of subgenome 2 is determined to be <em>S. megistacrolobum</em> of <em>Solanum</em> section <em>Petota</em> Clade 4, while subgenome 1 is likely a “ghost” genome whose progenitor species is extinct but which would have been associated with Clade 3. The two progenitor species would have grown in proximity in the same high‐altitude environment that required frost tolerance, and while subgenome 1 is smaller and contains fewer genes than subgenome 2, together the subgenomes make the species more complete and competitive. Interestingly, subgenome 2 and <em>S. megistacrolobum</em> are phylogenetically close to <em>S. commersonii</em> (Clade 4), which is well‐known for its frost tolerance. Several similarities were found between <em>S. acaule</em> and <em>S. commersonii</em> in the <em>CBF</em> gene family and also contrasted with the haplotypes of <em>S. tuberosum</em> ‘Atlantic’. Furthermore, while there is no overall subgenome expression bias in a set of homoeologous genes, DE genes showed noticeable differences between the <em>S. acaule</em> subgenomes, both in the number of regulated genes and in their functional roles. Subgenome 2 overexpresses genes related to abiotic stress responses, including cold acclimation, transcription regulation, and ribosome biogenesis, while subgenome 1 specializes in processes such as RNA silencing, circadian regulation, and metabolic activities like beta‐amylase and nitrate transport. This functional divergence observed between the subgenomes may reflect their distinct evolutionary origins and roles in responding to environmental stresses. Taken together, our results suggest that both subgenomes contribute to the cold tolerance and success of <em>S. acaule</em> as a species—subgenome 1 is cold adapted, while subgenome 2 (<em>S. megistacrolobum</em>) is cold acclimating and responds stronger to freezing conditions.</p></section><section id="tpg270095-sec-0260"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>S. Achakkagari</strong>: Formal analysis; investigation; methodology; software; validation; visualization; writing—original draft; writing—review and editing. <strong>J. C. Camargo‐Tavares</strong>: Data curation; formal analysis; investigation; methodology; validation; visualization; writing—original draft. <strong>D. Praslickova</strong>: Investigation; methodology. <strong>C. Martini</strong>: Formal analysis; validation. <strong>B. Bizimungu</strong>: Conceptualization; resources; writing—review and editing. <strong>N. L. Anglin</strong>: Conceptualization; funding acquisition; resources; writing—review and editing. <strong>N. Manrique‐Carpintero</strong>: Conceptualization; funding acquisition; project administration; resources; writing—review and editing. <strong>H. Lindqvist‐Kreuze</strong>: Funding acquisition; project administration; resources; writing—review and editing. <strong>H. H. Tai</strong>: Conceptualization; formal analysis; funding acquisition; project administration; resources; writing—review and editing. <strong>M. V. Strömvik</strong>: Conceptualization; data curation; funding acquisition; investigation; methodology; project administration; resources; software; supervision; writing—original draft; writing—review and editing.</p></section><section id="tpg270095-sec-0280"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec28"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270095-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270095-supitem-0001"><div class="caption p"><p>Supplementary Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12368328/bin/TPG2-18-e70095-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70095-s001.pdf</a><sup> (11.2MB, pdf) </sup>
</div></div></section></section><section id="tpg270095-sec-0270" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>The authors would like to acknowledge the potato breeding and genebank teams at Agriculture and Agri‐Food Canada (Sylvia Steeves) for maintenance and propagation of the <em>Solanum acaule</em> AA9 clone (Canadian Potato Genebank accession CN# 124026), <em>Solanum tuberosum</em> ‘Atlantic’ (Taylor Gervais and Cuijuan Zeng), and Daniel Philippe Matton (Université de Montreal, Canada) for germplasm transferred to the Canadian Potato Gene Resources Genebank. The authors would also like to acknowledge Emily McCoy at Agriculture and Agri‐Food Canada who prepared the Hi‐C libraries. Funding for the study was provided by a grant from Agriculture and Agri‐Food Canada (International Collaboration Program) “Genome sequencing of wild Solanum diploids” to CIP; a grant from the Agriculture and Agri‐Food Canada Genomics Research and Development Initiative to H. H. Tai; an AAFC‐Génome Québec joint call Management Driven Genomics award “Revolutionizing potato variety development for climate smart potato” to H. H. Tai and M. V. Strömvik (J‐002367/GQ‐AAC‐2019‐2); a Digital Research Alliance of Canada RPP award (Research Portals and Platforms: “The Potato Genome Diversity Portal”), a Digital Research Alliance of Canada Resources for Research Groups award (“Structural variation analyses of complex plant genomes in search of climate smart adaptations”), and an NSERC Discovery grant (RGPIN‐2020‐04990) to M. V. Strömvik; and a 10‐year project to strengthen food and nutrition security worldwide by supporting the conservation and use of crop diversity supported by the Government of Norway (NORAD) (“Biodiversity for Opportunities, Livelihoods and Development (BOLD)” (Project CONT‐0791 “CWR‐derived potatoes integrated in breeding pipelines for climate change resilience of farming communities of Ecuador, Kenya and Peru” to CIP), managed by the Crop Trust and implemented in partnership with national and international genebanks and seed system actors and researchers around the world.</p></section><section id="notes1"><p>


Achakkagari, S.
, 
Camargo‐Tavares, J. C.
, 
Praslickova, D.
, 
Martini, C.
, 
Bizimungu, B.
, 
Anglin, N. L.
, 
Manrique‐Carpintero, N.
, 
Lindqvist‐Kreuze, H.
, 
Tai, H. H.
, &amp; 
Strömvik, M. V.
 (2025). Better together: Subgenomes for allotetraploid potato wild relative <em>Solanum acaule</em> Bitt. reveal origins in <em>Petota</em> Clade 3 and 4. The Plant Genome, 18, e70095. 10.1002/tpg2.70095

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="tpg270095-note-0001">
<p>
<strong>Present address</strong>
</p>
<p>N. Manrique‐Carpintero, Alliance of Bioversity International and International Center for Tropical Agriculture (CIAT), Cali, Colombia.</p>
</div>
<div class="fn p" id="tpg270095-note-0002"><p>Assigned to Associate Editor Manuel Spannagl.</p></div>
</div></section></section><section id="tpg270095-sec-0300"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All software with their specific versions used for data processing are clearly described in the methods section. If no specific variable or parameters are mentioned for a software, the default parameters were used. All sequence data in this study are deposited in NCBI and available under BioProject PRJNA779368 unless otherwise specified. The AA9 genome sequencing data are available in NCBI under BioSample SAMN23440999, SRR17078425 (Illumina reads), SRR20871493 (Nanopore reads), SRR20871494 (PacBio), and SRR26082557 (HiC). The assemblies are available in NCBI under PRJNA1023364 accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/JAYKPG000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JAYKPG000000000</a>, “Phased genome assembly of <em>Solanum acaule</em> (AA9)—Subgenome 1,” accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/JAYKPF000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JAYKPF000000000</a>, and PRJNA1023365 “Phased genome assembly of <em>Solanum acaule</em> (AA9)—Subgenome 2.” The annotations are available in DRYAD under <a href="https://doi.org/10.5061/dryad.vdncjsz6q" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5061/dryad.vdncjsz6q</a>. The plastome is available under accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/OM638050" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OM638050</a>. The RNA‐Seq data are available under BioSamples SAMN25867789, SAMN25867802, SAMN25867820, and SAMN25867834, and the short reads archive accession numbers SRR17993274‐SRR17993285 (Illumina RNA‐Seq). The transcriptome expression data are available in NCBI GEO under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE268611" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE268611</a>.</p></section><section id="tpg270095-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270095-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270095-bib-0001">
<cite>

Achakkagari, S. R.
, 
Bozan, I.
, 
Camargo‐Tavares, J. C.
, 
Mccoy, H. J.
, 
Portal, L.
, 
Soto, J.
, 
Bizimungu, B.
, 
Anglin, N. L.
, 
Manrique‐Carpintero, N.
, 
Lindqvist‐Kreuze, H.
, 
Tai, H. H.
, &amp; 
Strömvik, M. V.
 (2024). The phased <em>Solanum okadae</em> genome and <em>Petota</em> pangenome analysis of 23 other potato wild relatives and hybrids. Scientific Data, 11, Article 454. 10.1038/s41597-024-03300-5

</cite> [<a href="https://doi.org/10.1038/s41597-024-03300-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11069515/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38704417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Data&amp;title=The%20phased%20Solanum%20okadae%20genome%20and%20Petota%20pangenome%20analysis%20of%2023%20other%20potato%20wild%20relatives%20and%20hybrids&amp;author=S.%20R.%20Achakkagari&amp;author=I.%20Bozan&amp;author=J.%20C.%20Camargo%E2%80%90Tavares&amp;author=H.%20J.%20Mccoy&amp;author=L.%20Portal&amp;volume=11&amp;publication_year=2024&amp;pages=454&amp;pmid=38704417&amp;doi=10.1038/s41597-024-03300-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0002">
<cite>

Achakkagari, S. R.
, 
Kyriakidou, M.
, 
Gardner, K. M.
, 
De Koeyer, D.
, 
De Jong, H.
, 
Strömvik, M. V.
, &amp; 
Tai, H. H.
 (2022). Genome sequencing of adapted diploid potato clones. Frontiers in Plant Science, 13, 954933. 10.3389/fpls.2022.954933

</cite> [<a href="https://doi.org/10.3389/fpls.2022.954933" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9394749/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36003817/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%20sequencing%20of%20adapted%20diploid%20potato%20clones&amp;author=S.%20R.%20Achakkagari&amp;author=M.%20Kyriakidou&amp;author=K.%20M.%20Gardner&amp;author=D.%20De%20Koeyer&amp;author=H.%20De%20Jong&amp;volume=13&amp;publication_year=2022&amp;pages=954933&amp;pmid=36003817&amp;doi=10.3389/fpls.2022.954933&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0003">
<cite>

Alger, E. I.
, &amp; 
Edger, P. P.
 (2020). One subgenome to rule them all: Underlying mechanisms of subgenome dominance. Current Opinion in Plant Biology, 54, 108–113. 10.1016/j.pbi.2020.03.004

</cite> [<a href="https://doi.org/10.1016/j.pbi.2020.03.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32344327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Plant%20Biology&amp;title=One%20subgenome%20to%20rule%20them%20all:%20Underlying%20mechanisms%20of%20subgenome%20dominance&amp;author=E.%20I.%20Alger&amp;author=P.%20P.%20Edger&amp;volume=54&amp;publication_year=2020&amp;pages=108-113&amp;pmid=32344327&amp;doi=10.1016/j.pbi.2020.03.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0004">
<cite>

Andrews, S.
 (2019). FastQC: A Quality Control Tool for High Throughput Sequence Data (Version 0.11.9) [Computer software]. Babraham Bioinformatics, Babraham Institute. <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Andrews,%20S.%20(2019).%20FastQC:%20A%20Quality%20Control%20Tool%20for%20High%20Throughput%20Sequence%20Data%20(Version%200.11.9)%20%5BComputer%20software%5D.%20Babraham%20Bioinformatics,%20Babraham%20Institute.%20https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0005">
<cite>

Archambault, A.
, &amp; 
Strömvik, M. V.
 (2011). PR‐10, defensin and cold dehydrin genes are among those over expressed in <em>Oxytropis</em> (Fabaceae) species adapted to the arctic. Functional and Integrative Genomics, 11, 497–505. 10.1007/s10142-011-0223-6

</cite> [<a href="https://doi.org/10.1007/s10142-011-0223-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3156302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21499864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20and%20Integrative%20Genomics&amp;title=PR%E2%80%9010,%20defensin%20and%20cold%20dehydrin%20genes%20are%20among%20those%20over%20expressed%20in%20Oxytropis%20(Fabaceae)%20species%20adapted%20to%20the%20arctic&amp;author=A.%20Archambault&amp;author=M.%20V.%20Str%C3%B6mvik&amp;volume=11&amp;publication_year=2011&amp;pages=497-505&amp;pmid=21499864&amp;doi=10.1007/s10142-011-0223-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0006">
<cite>

Aversano, R.
, 
Contaldi, F.
, 
Ercolano, M. R.
, 
Grosso, V.
, 
Iorizzo, M.
, 
Tatino, F.
, 
Xumerle, L.
, 
Dal Molin, A.
, 
Avanzato, C.
, 
Ferrarini, A.
, 
Delledonne, M.
, 
Sanseverino, W.
, 
Cigliano, R. A.
, 
Capella‐Gutierrez, S.
, 
Gabaldón, T.
, 
Frusciante, L.
, 
Bradeen, J. M.
, &amp; 
Carputo, D.
 (2015). The <em>Solanum commersonii</em> genome sequence provides insights into adaptation to stress conditions and genome evolution of wild potato relatives. Plant Cell, 27, 954–968. 10.1105/tpc.114.135954

</cite> [<a href="https://doi.org/10.1105/tpc.114.135954" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4558694/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25873387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20Solanum%20commersonii%20genome%20sequence%20provides%20insights%20into%20adaptation%20to%20stress%20conditions%20and%20genome%20evolution%20of%20wild%20potato%20relatives&amp;author=R.%20Aversano&amp;author=F.%20Contaldi&amp;author=M.%20R.%20Ercolano&amp;author=V.%20Grosso&amp;author=M.%20Iorizzo&amp;volume=27&amp;publication_year=2015&amp;pages=954-968&amp;pmid=25873387&amp;doi=10.1105/tpc.114.135954&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0007">
<cite>

Bandi, V.
, 
Gutwin, C.
, 
Siri, J. N.
, 
Neufeld, E.
, 
Sharpe, A.
, &amp; 
Parkin, I.
 (2022). Visualization tools for genomic conservation. Methods in Molecular Biology, 2443, 285–308.
</cite> [<a href="https://doi.org/10.1007/978-1-0716-2067-0_16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35037213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20in%20Molecular%20Biology&amp;title=Visualization%20tools%20for%20genomic%20conservation&amp;author=V.%20Bandi&amp;author=C.%20Gutwin&amp;author=J.%20N.%20Siri&amp;author=E.%20Neufeld&amp;author=A.%20Sharpe&amp;volume=2443&amp;publication_year=2022&amp;pages=285-308&amp;pmid=35037213&amp;doi=10.1007/978-1-0716-2067-0_16&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0008">
<cite>

Bianchini, G.
, &amp; 
Sánchez‐Baracaldo, P.
 (2024). TreeViewer: Flexible, modular software to visualise and manipulate phylogenetic trees. Ecology and Evolution, 14, e10873. 10.1002/ece3.10873

</cite> [<a href="https://doi.org/10.1002/ece3.10873" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10834882/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38314311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20and%20Evolution&amp;title=TreeViewer:%20Flexible,%20modular%20software%20to%20visualise%20and%20manipulate%20phylogenetic%20trees&amp;author=G.%20Bianchini&amp;author=P.%20S%C3%A1nchez%E2%80%90Baracaldo&amp;volume=14&amp;publication_year=2024&amp;pages=e10873&amp;pmid=38314311&amp;doi=10.1002/ece3.10873&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0009">
<cite>

Bitter, G.
 (1912). 
<em>Solana</em> nova vel minus cogita. II. Feddes Repertorium, 11, 1–18. 10.1002/fedr.19120110103
</cite> [<a href="https://doi.org/10.1002/fedr.19120110103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Feddes%20Repertorium&amp;title=Solana%20nova%20vel%20minus%20cogita.%20II&amp;author=G.%20Bitter&amp;volume=11&amp;publication_year=1912&amp;pages=1-18&amp;doi=10.1002/fedr.19120110103&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0010">
<cite>

Bolger, A. M.
, 
Lohse, M.
, &amp; 
Usadel, B.
 (2014). Trimmomatic: A flexible trimmer for Illumina sequence data. Bioinformatics, 30, 2114–2120. 10.1093/bioinformatics/btu170

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4103590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Trimmomatic:%20A%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;volume=30&amp;publication_year=2014&amp;pages=2114-2120&amp;pmid=24695404&amp;doi=10.1093/bioinformatics/btu170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0011">
<cite>

Bonenfant, Q.
, 
Noé, L.
, &amp; 
Touzet, H.
 (2022). Porechop_ABI: Discovering unknown adapters in Oxford Nanopore Technology sequencing reads for downstream trimming. Bioinformatics Advances, 3, vbac085. 10.1093/bioadv/vbac085

</cite> [<a href="https://doi.org/10.1093/bioadv/vbac085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9869717/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36698762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics%20Advances&amp;title=Porechop_ABI:%20Discovering%20unknown%20adapters%20in%20Oxford%20Nanopore%20Technology%20sequencing%20reads%20for%20downstream%20trimming&amp;author=Q.%20Bonenfant&amp;author=L.%20No%C3%A9&amp;author=H.%20Touzet&amp;volume=3&amp;publication_year=2022&amp;pages=vbac085&amp;pmid=36698762&amp;doi=10.1093/bioadv/vbac085&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0012">
<cite>

Bozan, I.
, 
Achakkagari, S. R.
, 
Anglin, N. L.
, 
Ellis, D.
, 
Tai, H.
, &amp; 
Strömvik, M. V.
 (2023). Pangenome analyses reveal impact of transposable elements and ploidy on the evolution of potato species. PNAS, 120, e2211117120. 10.1073/pnas.2211117120

</cite> [<a href="https://doi.org/10.1073/pnas.2211117120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10401005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37487084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Pangenome%20analyses%20reveal%20impact%20of%20transposable%20elements%20and%20ploidy%20on%20the%20evolution%20of%20potato%20species&amp;author=I.%20Bozan&amp;author=S.%20R.%20Achakkagari&amp;author=N.%20L.%20Anglin&amp;author=D.%20Ellis&amp;author=H.%20Tai&amp;volume=120&amp;publication_year=2023&amp;pages=e2211117120&amp;pmid=37487084&amp;doi=10.1073/pnas.2211117120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0148">
<cite>

Bray, N. L.
, 
Pimentel, H.
, 
Melsted, P.
, &amp; 
Pachter, L.
 (2016). Near‐optimal probabilistic RNA‐seq quantification. Nature Biotechnology, 34(5), 525–527. 10.1038/nbt.3519
</cite> [<a href="https://doi.org/10.1038/nbt.3519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27043002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Near%E2%80%90optimal%20probabilistic%20RNA%E2%80%90seq%20quantification&amp;author=N.%20L.%20Bray&amp;author=H.%20Pimentel&amp;author=P.%20Melsted&amp;author=L.%20Pachter&amp;volume=34&amp;issue=5&amp;publication_year=2016&amp;pages=525-527&amp;pmid=27043002&amp;doi=10.1038/nbt.3519&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0013">
<cite>

Brown, M.
, 
González De la Rosa, P. M.
, &amp; 
Mark, B.
 (2023). 
A telomere identification toolkit
. Zenodo.</cite> [<a href="https://scholar.google.com/scholar_lookup?Brown,%20M.%20,%20Gonz%C3%A1lez%20De%20la%20Rosa,%20P.%20M.%20,%20&amp;%20Mark,%20B.%20(2023).%20A%20telomere%20identification%20toolkit%20.%20Zenodo." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0014">
<cite>

Brůna, T.
, 
Hoff, K. J.
, 
Lomsadze, A.
, 
Stanke, M.
, &amp; 
Borodovsky, M.
 (2021). BRAKER2: Automatic eukaryotic genome annotation with GeneMark‐EP+ and AUGUSTUS supported by a protein database. NAR Genomics &amp; Bioinformatics, 3, lqaa108. 10.1093/nargab/lqaa108

</cite> [<a href="https://doi.org/10.1093/nargab/lqaa108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7787252/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33575650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NAR%20Genomics%20&amp;%20Bioinformatics&amp;title=BRAKER2:%20Automatic%20eukaryotic%20genome%20annotation%20with%20GeneMark%E2%80%90EP+%20and%20AUGUSTUS%20supported%20by%20a%20protein%20database&amp;author=T.%20Br%C5%AFna&amp;author=K.%20J.%20Hoff&amp;author=A.%20Lomsadze&amp;author=M.%20Stanke&amp;author=M.%20Borodovsky&amp;volume=3&amp;publication_year=2021&amp;pages=lqaa108&amp;pmid=33575650&amp;doi=10.1093/nargab/lqaa108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0016"><cite>

Bruna, T.
, 
Lomsadze, A.
, &amp; 
Borodovsky, M.
 (2023). GeneMark‐ETP: Automatic gene finding in eukaryotic genomes in consistence with extrinsic data. <em>BioRxiv</em>, 2023–2001.</cite></li>
<li id="tpg270095-bib-0017">
<cite>

Bryant, D. M.
, 
Johnson, K.
, 
Ditommaso, T.
, 
Tickle, T.
, 
Couger, M. B.
, 
Payzin‐Dogru, D.
, 
Lee, T. J.
, 
Leigh, N. D.
, 
Kuo, T.‐H.
, 
Davis, F. G.
, 
Bateman, J.
, 
Bryant, S.
, 
Guzikowski, A. R.
, 
Tsai, S. L.
, 
Coyne, S.
, 
Ye, W. W.
, 
Freeman, R. M.
, 
Peshkin, L.
, 
Tabin, C. J.
, … 
Whited, J. L.
 (2017). A tissue‐mapped axolotl de novo transcriptome enables identification of limb regeneration factors. Cell Reports, 18, 762–776. 10.1016/j.celrep.2016.12.063

</cite> [<a href="https://doi.org/10.1016/j.celrep.2016.12.063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5419050/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28099853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Reports&amp;title=A%20tissue%E2%80%90mapped%20axolotl%20de%20novo%20transcriptome%20enables%20identification%20of%20limb%20regeneration%20factors&amp;author=D.%20M.%20Bryant&amp;author=K.%20Johnson&amp;author=T.%20Ditommaso&amp;author=T.%20Tickle&amp;author=M.%20B.%20Couger&amp;volume=18&amp;publication_year=2017&amp;pages=762-776&amp;pmid=28099853&amp;doi=10.1016/j.celrep.2016.12.063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0018">
<cite>

Buchfink, B.
, 
Xie, C.
, &amp; 
Huson, D. H.
 (2015). Fast and sensitive protein alignment using DIAMOND. Nature Methods, 12, 59–60. 10.1038/nmeth.3176

</cite> [<a href="https://doi.org/10.1038/nmeth.3176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25402007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Fast%20and%20sensitive%20protein%20alignment%20using%20DIAMOND&amp;author=B.%20Buchfink&amp;author=C.%20Xie&amp;author=D.%20H.%20Huson&amp;volume=12&amp;publication_year=2015&amp;pages=59-60&amp;pmid=25402007&amp;doi=10.1038/nmeth.3176&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0019">
<cite>

Buggs, R. J. A.
, 
Zhang, L.
, 
Miles, N.
, 
Tate, J. A.
, 
Gao, L.
, 
Wei, W.
, 
Schnable, P. S.
, 
Barbazuk, W. B.
, 
Soltis, P. S.
, &amp; 
Soltis, D. E.
 (2011). Transcriptomic shock generates evolutionary novelty in a newly formed, natural allopolyploid plant. Current Biology, 21, 551–556. 10.1016/j.cub.2011.02.016

</cite> [<a href="https://doi.org/10.1016/j.cub.2011.02.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21419627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Biology&amp;title=Transcriptomic%20shock%20generates%20evolutionary%20novelty%20in%20a%20newly%20formed,%20natural%20allopolyploid%20plant&amp;author=R.%20J.%20A.%20Buggs&amp;author=L.%20Zhang&amp;author=N.%20Miles&amp;author=J.%20A.%20Tate&amp;author=L.%20Gao&amp;volume=21&amp;publication_year=2011&amp;pages=551-556&amp;pmid=21419627&amp;doi=10.1016/j.cub.2011.02.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0020">
<cite>

Cai, D.
, 
Rodríguez, F.
, 
Teng, Y.
, 
Ané, C.
, 
Bonierbale, M.
, 
Mueller, L. A.
, &amp; 
Spooner, D. M.
 (2012). Single copy nuclear gene analysis of polyploidy in wild potatoes (<em>Solanum</em> section <em>Petota</em>). BMC Evolutionary Biology, 12, Article 70. 10.1186/1471-2148-12-70

</cite> [<a href="https://doi.org/10.1186/1471-2148-12-70" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3416576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22624678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Evolutionary%20Biology&amp;title=Single%20copy%20nuclear%20gene%20analysis%20of%20polyploidy%20in%20wild%20potatoes%20(Solanum%20section%20Petota)&amp;author=D.%20Cai&amp;author=F.%20Rodr%C3%ADguez&amp;author=Y.%20Teng&amp;author=C.%20An%C3%A9&amp;author=M.%20Bonierbale&amp;volume=12&amp;publication_year=2012&amp;pages=70&amp;pmid=22624678&amp;doi=10.1186/1471-2148-12-70&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0021">
<cite>

Camacho, C.
, 
Coulouris, G.
, 
Avagyan, V.
, 
Ma, N.
, 
Papadopoulos, J.
, 
Bealer, K.
, &amp; 
Madden, T. L.
 (2009). BLAST+: Architecture and applications. BMC Bioinformatics, 10, Article 421. 10.1186/1471-2105-10-421

</cite> [<a href="https://doi.org/10.1186/1471-2105-10-421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2803857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20003500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=BLAST+:%20Architecture%20and%20applications&amp;author=C.%20Camacho&amp;author=G.%20Coulouris&amp;author=V.%20Avagyan&amp;author=N.%20Ma&amp;author=J.%20Papadopoulos&amp;volume=10&amp;publication_year=2009&amp;pages=421&amp;pmid=20003500&amp;doi=10.1186/1471-2105-10-421&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0022">
<cite>

Chalhoub, B.
, 
Denoeud, F.
, 
Liu, S.
, 
Parkin, I. A. P.
, 
Tang, H.
, 
Wang, X.
, 
Chiquet, J.
, 
Belcram, H.
, 
Tong, C.
, 
Samans, B.
, 
Corréa, M.
, 
Da Silva, C.
, 
Just, J.
, 
Falentin, C.
, 
Koh, C. S.
, 
Le Clainche, I.
, 
Bernard, M.
, 
Bento, P.
, 
Noel, B.
, … 
Wincker, P.
 (2014). Early allopolyploid evolution in the post‐neolithic <em>Brassica napus</em> oilseed genome. Science, 345, 950–953. 10.1126/science.1253435

</cite> [<a href="https://doi.org/10.1126/science.1253435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25146293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Early%20allopolyploid%20evolution%20in%20the%20post%E2%80%90neolithic%20Brassica%20napus%20oilseed%20genome&amp;author=B.%20Chalhoub&amp;author=F.%20Denoeud&amp;author=S.%20Liu&amp;author=I.%20A.%20P.%20Parkin&amp;author=H.%20Tang&amp;volume=345&amp;publication_year=2014&amp;pages=950-953&amp;pmid=25146293&amp;doi=10.1126/science.1253435&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0023">
<cite>

Chen, L.
, 
Zhao, H.
, 
Chen, Y.
, 
Jiang, F.
, 
Zhou, F.
, 
Liu, Q.
, 
Fan, Y.
, 
Liu, T.
, 
Tu, W.
, 
Walther, D.
, &amp; 
Song, B.
 (2022). Comparative transcriptomics analysis reveals a calcineurin B‐like gene to positively regulate constitutive and acclimated freezing tolerance in potato. Plant, Cell &amp; Environment, 45, 3305–3321. 10.1111/pce.14432
</cite> [<a href="https://doi.org/10.1111/pce.14432" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36041917/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Cell%20&amp;%20Environment&amp;title=Comparative%20transcriptomics%20analysis%20reveals%20a%20calcineurin%20B%E2%80%90like%20gene%20to%20positively%20regulate%20constitutive%20and%20acclimated%20freezing%20tolerance%20in%20potato&amp;author=L.%20Chen&amp;author=H.%20Zhao&amp;author=Y.%20Chen&amp;author=F.%20Jiang&amp;author=F.%20Zhou&amp;volume=45&amp;publication_year=2022&amp;pages=3305-3321&amp;pmid=36041917&amp;doi=10.1111/pce.14432&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0024">
<cite>

Chen, S.
, 
Zhou, Y.
, 
Chen, Y.
, &amp; 
Gu, J.
 (2018). 
<em>fastp</em>: An ultra‐fast all‐in‐one FASTQ preprocessor. Bioinformatics, 34, i884–i890. 10.1093/bioinformatics/bty560

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6129281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=fastp:%20An%20ultra%E2%80%90fast%20all%E2%80%90in%E2%80%90one%20FASTQ%20preprocessor&amp;author=S.%20Chen&amp;author=Y.%20Zhou&amp;author=Y.%20Chen&amp;author=J.%20Gu&amp;volume=34&amp;publication_year=2018&amp;pages=i884-i890&amp;pmid=30423086&amp;doi=10.1093/bioinformatics/bty560&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0025">
<cite>

Cheong, B. E.
, 
Beine‐Golovchuk, O.
, 
Gorka, M.
, 
Ho, W. W. H.
, 
Martinez‐Seidel, F.
, 
Firmino, A. A. P.
, 
Skirycz, A.
, 
Roessner, U.
, &amp; 
Kopka, J.
 (2021). Arabidopsis REI‐LIKE proteins activate ribosome biogenesis during cold acclimation. Scientific Reports, 11(1), 1–25.
</cite> [<a href="https://doi.org/10.1038/s41598-021-81610-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7844247/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33510206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Arabidopsis%20REI%E2%80%90LIKE%20proteins%20activate%20ribosome%20biogenesis%20during%20cold%20acclimation&amp;author=B.%20E.%20Cheong&amp;author=O.%20Beine%E2%80%90Golovchuk&amp;author=M.%20Gorka&amp;author=W.%20W.%20H.%20Ho&amp;author=F.%20Martinez%E2%80%90Seidel&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pages=1-25&amp;pmid=33510206&amp;doi=10.1038/s41598-021-81610-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0026">
<cite>

Comai, L.
 (2005). The advantages and disadvantages of being polyploid. Nature Reviews Genetics, 6, 836–846. 10.1038/nrg1711
</cite> [<a href="https://doi.org/10.1038/nrg1711" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16304599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=The%20advantages%20and%20disadvantages%20of%20being%20polyploid&amp;author=L.%20Comai&amp;volume=6&amp;publication_year=2005&amp;pages=836-846&amp;pmid=16304599&amp;doi=10.1038/nrg1711&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0027">
<cite>

Corrales, A.‐R.
, 
Nebauer, S. G.
, 
Carrillo, L.
, 
Fernández‐Nohales, P.
, 
Marqués, J.
, 
Renau‐Morata, B.
, 
Granell, A.
, 
Pollmann, S.
, 
Vicente‐Carbajosa, J.
, 
Molina, R.‐V.
, &amp; 
Medina, J.
 (2014). Characterization of tomato Cycling Dof Factors reveals conserved and new functions in the control of flowering time and abiotic stress responses. Journal of Experimental Botany, 65, 995–1012. 10.1093/jxb/ert451

</cite> [<a href="https://doi.org/10.1093/jxb/ert451" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24399177/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Characterization%20of%20tomato%20Cycling%20Dof%20Factors%20reveals%20conserved%20and%20new%20functions%20in%20the%20control%20of%20flowering%20time%20and%20abiotic%20stress%20responses&amp;author=A.%E2%80%90R.%20Corrales&amp;author=S.%20G.%20Nebauer&amp;author=L.%20Carrillo&amp;author=P.%20Fern%C3%A1ndez%E2%80%90Nohales&amp;author=J.%20Marqu%C3%A9s&amp;volume=65&amp;publication_year=2014&amp;pages=995-1012&amp;pmid=24399177&amp;doi=10.1093/jxb/ert451&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0028">
<cite>

Correll, D. S.
 (1962). The potato and its wild relatives. Texas Research Foundation.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20potato%20and%20its%20wild%20relatives&amp;author=D.%20S.%20Correll&amp;publication_year=1962&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0029">
<cite>

Dainat, J.
, 
Hereñú, D.
, &amp; 
Pucholt, P.
 (2020). AGAT: Another Gff analysis toolkit to handle annotations in any GTF/GFF format. <em>Zenodo</em>
. 10.5281/zenodo.3552717
</cite> [<a href="https://doi.org/10.5281/zenodo.3552717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270095-bib-0030">
<cite>

Dearborn, C. H.
 (1969). Alaska Frostless, an inherently frost resistant potato variety. American Potato Journal, 46, 1–4. 10.1007/BF02862503
</cite> [<a href="https://doi.org/10.1007/BF02862503" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Potato%20Journal&amp;title=Alaska%20Frostless,%20an%20inherently%20frost%20resistant%20potato%20variety&amp;author=C.%20H.%20Dearborn&amp;volume=46&amp;publication_year=1969&amp;pages=1-4&amp;doi=10.1007/BF02862503&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0031">
<cite>

De Coster, W.
, 
D'Hert, S.
, 
Schultz, D. T.
, 
Cruts, M.
, &amp; 
Van Broeckhoven, C.
 (2018). NanoPack: Visualizing and processing long‐read sequencing data. Bioinformatics, 34, 2666–2669. 10.1093/bioinformatics/bty149

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6061794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29547981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=NanoPack:%20Visualizing%20and%20processing%20long%E2%80%90read%20sequencing%20data&amp;author=W.%20De%20Coster&amp;author=S.%20D'Hert&amp;author=D.%20T.%20Schultz&amp;author=M.%20Cruts&amp;author=C.%20Van%20Broeckhoven&amp;volume=34&amp;publication_year=2018&amp;pages=2666-2669&amp;pmid=29547981&amp;doi=10.1093/bioinformatics/bty149&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0032">
<cite>

Dudchenko, O.
, 
Batra, S. S.
, 
Omer, A. D.
, 
Nyquist, S. K.
, 
Hoeger, M.
, 
Durand, N. C.
, 
Shamim, M. S.
, 
Machol, I.
, 
Lander, E. S.
, 
Aiden, A. P.
, &amp; 
Aiden, E. L.
 (2017). De novo assembly of the <em>Aedes aegypti</em> genome using Hi‐C yields chromosome‐length scaffolds. Science, 356, 92–95. 10.1126/science.aal3327

</cite> [<a href="https://doi.org/10.1126/science.aal3327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5635820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28336562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=De%20novo%20assembly%20of%20the%20Aedes%20aegypti%20genome%20using%20Hi%E2%80%90C%20yields%20chromosome%E2%80%90length%20scaffolds&amp;author=O.%20Dudchenko&amp;author=S.%20S.%20Batra&amp;author=A.%20D.%20Omer&amp;author=S.%20K.%20Nyquist&amp;author=M.%20Hoeger&amp;volume=356&amp;publication_year=2017&amp;pages=92-95&amp;pmid=28336562&amp;doi=10.1126/science.aal3327&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0033">
<cite>

Durand, N. C.
, 
Shamim, M. S.
, 
Machol, I.
, 
Rao, S. S. P.
, 
Huntley, M. H.
, 
Lander, E. S.
, &amp; 
Aiden, E. L.
 (2016). Juicer provides a one‐click system for analyzing loop‐resolution Hi‐C experiments. Cell Systems, 3, 95–98. 10.1016/j.cels.2016.07.002

</cite> [<a href="https://doi.org/10.1016/j.cels.2016.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5846465/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27467249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Systems&amp;title=Juicer%20provides%20a%20one%E2%80%90click%20system%20for%20analyzing%20loop%E2%80%90resolution%20Hi%E2%80%90C%20experiments&amp;author=N.%20C.%20Durand&amp;author=M.%20S.%20Shamim&amp;author=I.%20Machol&amp;author=S.%20S.%20P.%20Rao&amp;author=M.%20H.%20Huntley&amp;volume=3&amp;publication_year=2016&amp;pages=95-98&amp;pmid=27467249&amp;doi=10.1016/j.cels.2016.07.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0034">
<cite>

Dvořák, J.
 (1983). Evidence for genetic suppression of heterogenetic chromosome pairing in polyploid species of <em>Solanum</em>, sect. <em>Petota</em>
. Canadian Journal of Genetics and Cytology, 25, 530–539. 10.1139/g83-080
</cite> [<a href="https://doi.org/10.1139/g83-080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Canadian%20Journal%20of%20Genetics%20and%20Cytology&amp;title=Evidence%20for%20genetic%20suppression%20of%20heterogenetic%20chromosome%20pairing%20in%20polyploid%20species%20of%20Solanum,%20sect.%20Petota&amp;author=J.%20Dvo%C5%99%C3%A1k&amp;volume=25&amp;publication_year=1983&amp;pages=530-539&amp;doi=10.1139/g83-080&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0035">
<cite>

Edgar, R. C.
 (2004). MUSCLE: Multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Research, 32, 1792–1797. 10.1093/nar/gkh340

</cite> [<a href="https://doi.org/10.1093/nar/gkh340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC390337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15034147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=MUSCLE:%20Multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=R.%20C.%20Edgar&amp;volume=32&amp;publication_year=2004&amp;pages=1792-1797&amp;pmid=15034147&amp;doi=10.1093/nar/gkh340&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0036">
<cite>

Edger, P. P.
, 
Smith, R.
, 
Mckain, M. R.
, 
Cooley, A. M.
, 
Vallejo‐Marin, M.
, 
Yuan, Y.
, 
Bewick, A. J.
, 
Ji, L.
, 
Platts, A. E.
, 
Bowman, M. J.
, 
Childs, K. L.
, 
Washburn, J. D.
, 
Schmitz, R. J.
, 
Smith, G. D.
, 
Pires, J. C.
, &amp; 
Puzey, J. R.
 (2017). Subgenome dominance in an interspecific hybrid, synthetic allopolyploid, and a 140‐year‐old naturally established neo‐allopolyploid monkeyflower. Plant Cell, 29, 2150–2167. 10.1105/tpc.17.00010

</cite> [<a href="https://doi.org/10.1105/tpc.17.00010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5635986/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28814644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Subgenome%20dominance%20in%20an%20interspecific%20hybrid,%20synthetic%20allopolyploid,%20and%20a%20140%E2%80%90year%E2%80%90old%20naturally%20established%20neo%E2%80%90allopolyploid%20monkeyflower&amp;author=P.%20P.%20Edger&amp;author=R.%20Smith&amp;author=M.%20R.%20Mckain&amp;author=A.%20M.%20Cooley&amp;author=M.%20Vallejo%E2%80%90Marin&amp;volume=29&amp;publication_year=2017&amp;pages=2150-2167&amp;pmid=28814644&amp;doi=10.1105/tpc.17.00010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0037">
<cite>

Emms, D. M.
, &amp; 
Kelly, S.
 (2019). OrthoFinder: Phylogenetic orthology inference for comparative genomics. Genome Biology, 20, Article 238. 10.1186/s13059-019-1832-y

</cite> [<a href="https://doi.org/10.1186/s13059-019-1832-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6857279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31727128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=OrthoFinder:%20Phylogenetic%20orthology%20inference%20for%20comparative%20genomics&amp;author=D.%20M.%20Emms&amp;author=S.%20Kelly&amp;volume=20&amp;publication_year=2019&amp;pages=238&amp;pmid=31727128&amp;doi=10.1186/s13059-019-1832-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0038">
<cite>

Flynn, J. M.
, 
Hubley, R.
, 
Goubert, C.
, 
Rosen, J.
, 
Clark, A. G.
, 
Feschotte, C.
, &amp; 
Smit, A. F.
 (2020). RepeatModeler2 for automated genomic discovery of transposable element families. PNAS, 117, 9451–9457. 10.1073/pnas.1921046117

</cite> [<a href="https://doi.org/10.1073/pnas.1921046117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7196820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32300014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=RepeatModeler2%20for%20automated%20genomic%20discovery%20of%20transposable%20element%20families&amp;author=J.%20M.%20Flynn&amp;author=R.%20Hubley&amp;author=C.%20Goubert&amp;author=J.%20Rosen&amp;author=A.%20G.%20Clark&amp;volume=117&amp;publication_year=2020&amp;pages=9451-9457&amp;pmid=32300014&amp;doi=10.1073/pnas.1921046117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0039">
<cite>

Fujimoto, S. Y.
, 
Ohta, M.
, 
Usui, A.
, 
Shinshi, H.
, &amp; 
Ohme‐Takagi, M.
 (2000). Arabidopsis ethylene‐responsive element binding factors act as transcriptional activators or repressors of GCC box‐mediated gene expression. Plant Cell, 12, 393–404.
</cite> [<a href="https://doi.org/10.1105/tpc.12.3.393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC139839/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10715325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Arabidopsis%20ethylene%E2%80%90responsive%20element%20binding%20factors%20act%20as%20transcriptional%20activators%20or%20repressors%20of%20GCC%20box%E2%80%90mediated%20gene%20expression&amp;author=S.%20Y.%20Fujimoto&amp;author=M.%20Ohta&amp;author=A.%20Usui&amp;author=H.%20Shinshi&amp;author=M.%20Ohme%E2%80%90Takagi&amp;volume=12&amp;publication_year=2000&amp;pages=393-404&amp;pmid=10715325&amp;doi=10.1105/tpc.12.3.393&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0040">
<cite>

Gabriel, L.
, 
Brůna, T.
, 
Hoff, K. J.
, 
Ebel, M.
, 
Lomsadze, A.
, 
Borodovsky, M.
, &amp; 
Stanke, M.
 (2024). BRAKER3: Fully automated genome annotation using RNA‐seq and protein evidence with GeneMark‐ETP, AUGUSTUS and TSEBRA. Genome Research, 34, 769–777. <a href="https://www.genome.org/cgi/doi/10.1101/gr.278090.123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.genome.org/cgi/doi/10.1101/gr.278090.123</a>

</cite> [<a href="https://doi.org/10.1101/gr.278090.123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11216308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38866550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=BRAKER3:%20Fully%20automated%20genome%20annotation%20using%20RNA%E2%80%90seq%20and%20protein%20evidence%20with%20GeneMark%E2%80%90ETP,%20AUGUSTUS%20and%20TSEBRA&amp;author=L.%20Gabriel&amp;author=T.%20Br%C5%AFna&amp;author=K.%20J.%20Hoff&amp;author=M.%20Ebel&amp;author=A.%20Lomsadze&amp;volume=34&amp;publication_year=2024&amp;pages=769-777&amp;pmid=38866550&amp;doi=10.1101/gr.278090.123&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0041">
<cite>

Gabriel, L.
, 
Hoff, K. J.
, 
Brůna, T.
, 
Borodovsky, M.
, &amp; 
Stanke, M.
 (2021). TSEBRA: Transcript selector for BRAKER. BMC Bioinformatics, 22, Article 566. 10.1186/s12859-021-04482-0

</cite> [<a href="https://doi.org/10.1186/s12859-021-04482-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8620231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34823473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=TSEBRA:%20Transcript%20selector%20for%20BRAKER&amp;author=L.%20Gabriel&amp;author=K.%20J.%20Hoff&amp;author=T.%20Br%C5%AFna&amp;author=M.%20Borodovsky&amp;author=M.%20Stanke&amp;volume=22&amp;publication_year=2021&amp;pages=566&amp;pmid=34823473&amp;doi=10.1186/s12859-021-04482-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0042">
<cite>

Garsmeur, O.
, 
Schnable, J. C.
, 
Almeida, A.
, 
Jourda, C.
, 
D'Hont, A.
, &amp; 
Freeling, M.
 (2014). Two evolutionarily distinct classes of paleopolyploidy. Molecular Biology and Evolution, 31, 448–454. 10.1093/molbev/mst230

</cite> [<a href="https://doi.org/10.1093/molbev/mst230" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24296661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=Two%20evolutionarily%20distinct%20classes%20of%20paleopolyploidy&amp;author=O.%20Garsmeur&amp;author=J.%20C.%20Schnable&amp;author=A.%20Almeida&amp;author=C.%20Jourda&amp;author=A.%20D'Hont&amp;volume=31&amp;publication_year=2014&amp;pages=448-454&amp;pmid=24296661&amp;doi=10.1093/molbev/mst230&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0043">
<cite>

Giannattasio, R. B.
, &amp; 
Spooner, D. M.
 (1994). A reexamination of species boundaries and hypotheses of hybridization concerning <em>Solanum megistacrolobum</em> and <em>S. toralapanum</em> (<em>Solanum</em> Sect. <em>Petota</em>, series <em>Megistacroloba</em>): Molecular data. Systematic Botany, 19, 106–115. 10.2307/2419715
</cite> [<a href="https://doi.org/10.2307/2419715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Systematic%20Botany&amp;title=A%20reexamination%20of%20species%20boundaries%20and%20hypotheses%20of%20hybridization%20concerning%20Solanum%20megistacrolobum%20and%20S.%20toralapanum%20(Solanum%20Sect.%20Petota,%20series%20Megistacroloba):%20Molecular%20data&amp;author=R.%20B.%20Giannattasio&amp;author=D.%20M.%20Spooner&amp;volume=19&amp;publication_year=1994&amp;pages=106-115&amp;doi=10.2307/2419715&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0044">
<cite>

Gilbert, D. G.
 (2019). Longest protein, longest transcript or most expression, for accurate gene reconstruction of transcriptomes? <em>BioRxiv</em>
. 10.1101/829184
</cite> [<a href="https://doi.org/10.1101/829184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270095-bib-0045">
<cite>

Gotoh, O.
 (2008). A space‐efficient and accurate method for mapping and aligning cDNA sequences onto genomic sequence. Nucleic Acids Research, 36, 2630–2638. 10.1093/nar/gkn105

</cite> [<a href="https://doi.org/10.1093/nar/gkn105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2377433/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18344523/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=A%20space%E2%80%90efficient%20and%20accurate%20method%20for%20mapping%20and%20aligning%20cDNA%20sequences%20onto%20genomic%20sequence&amp;author=O.%20Gotoh&amp;volume=36&amp;publication_year=2008&amp;pages=2630-2638&amp;pmid=18344523&amp;doi=10.1093/nar/gkn105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0046">
<cite>

Gramazio, P.
, 
Blanca, J.
, 
Ziarsolo, P.
, 
Herraiz, F. J.
, 
Plazas, M.
, 
Prohens, J.
, &amp; 
Vilanova, S.
 (2016). Transcriptome analysis and molecular marker discovery in <em>Solanum incanum</em> and <em>S. aethiopicum</em>, two close relatives of the common eggplant (<em>Solanum melongena</em>) with interest for breeding. BMC Genomics, 17, Article 300. 10.1186/s12864-016-2631-4

</cite> [<a href="https://doi.org/10.1186/s12864-016-2631-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4841963/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27108408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Transcriptome%20analysis%20and%20molecular%20marker%20discovery%20in%20Solanum%20incanum%20and%20S.%20aethiopicum,%20two%20close%20relatives%20of%20the%20common%20eggplant%20(Solanum%20melongena)%20with%20interest%20for%20breeding&amp;author=P.%20Gramazio&amp;author=J.%20Blanca&amp;author=P.%20Ziarsolo&amp;author=F.%20J.%20Herraiz&amp;author=M.%20Plazas&amp;volume=17&amp;publication_year=2016&amp;pages=300&amp;pmid=27108408&amp;doi=10.1186/s12864-016-2631-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0047">
<cite>

Gurevich, A.
, 
Saveliev, V.
, 
Vyahhi, N.
, &amp; 
Tesler, G.
 (2013). QUAST: Quality assessment tool for genome assemblies. Bioinformatics, 29, 1072–1075. 10.1093/bioinformatics/btt086

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3624806/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23422339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=QUAST:%20Quality%20assessment%20tool%20for%20genome%20assemblies&amp;author=A.%20Gurevich&amp;author=V.%20Saveliev&amp;author=N.%20Vyahhi&amp;author=G.%20Tesler&amp;volume=29&amp;publication_year=2013&amp;pages=1072-1075&amp;pmid=23422339&amp;doi=10.1093/bioinformatics/btt086&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0048">
<cite>

Haas, B. J.
, 
Papanicolaou, A.
, 
Yassour, M.
, 
Grabherr, M.
, 
Blood, P. D.
, 
Bowden, J.
, 
Couger, M. B.
, 
Eccles, D.
, 
Li, B.
, 
Lieber, M.
, 
Macmanes, M. D.
, 
Ott, M.
, 
Orvis, J.
, 
Pochet, N.
, 
Strozzi, F.
, 
Weeks, N.
, 
Westerman, R.
, 
William, T.
, 
Dewey, C. N.
, … 
Regev, A.
 (2013). 
<em>De novo</em> transcript sequence reconstruction from RNA‐seq using the Trinity platform for reference generation and analysis. Nature Protocols, 8, 1494–1512. 10.1038/nprot.2013.084

</cite> [<a href="https://doi.org/10.1038/nprot.2013.084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3875132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23845962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Protocols&amp;title=De%20novo%20transcript%20sequence%20reconstruction%20from%20RNA%E2%80%90seq%20using%20the%20Trinity%20platform%20for%20reference%20generation%20and%20analysis&amp;author=B.%20J.%20Haas&amp;author=A.%20Papanicolaou&amp;author=M.%20Yassour&amp;author=M.%20Grabherr&amp;author=P.%20D.%20Blood&amp;volume=8&amp;publication_year=2013&amp;pages=1494-1512&amp;pmid=23845962&amp;doi=10.1038/nprot.2013.084&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0049">
<cite>

Hardigan, M. A.
, 
Laimbeer, F. P. E.
, 
Newton, L.
, 
Crisovan, E.
, 
Hamilton, J. P.
, 
Vaillancourt, B.
, 
Wiegert‐Rininger, K.
, 
Wood, J. C.
, 
Douches, D. S.
, 
Farré, E. M.
, 
Veilleux, R. E.
, &amp; 
Buell, C. R.
 (2017). Genome diversity of tuber‐bearing <em>Solanum</em> uncovers complex evolutionary history and targets of domestication in the cultivated potato. Proceedings of the National Academy of Sciences of the United States of America, 114, E9999–E10008. 10.1073/pnas.1714380114

</cite> [<a href="https://doi.org/10.1073/pnas.1714380114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5699086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29087343/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Genome%20diversity%20of%20tuber%E2%80%90bearing%20Solanum%20uncovers%20complex%20evolutionary%20history%20and%20targets%20of%20domestication%20in%20the%20cultivated%20potato&amp;author=M.%20A.%20Hardigan&amp;author=F.%20P.%20E.%20Laimbeer&amp;author=L.%20Newton&amp;author=E.%20Crisovan&amp;author=J.%20P.%20Hamilton&amp;volume=114&amp;publication_year=2017&amp;pages=E9999-E10008&amp;pmid=29087343&amp;doi=10.1073/pnas.1714380114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0050">
<cite>

Hawkes, J. G.
 (1958). Significance of wild species and primitive forms for potato breeding. Euphytica, 7(3), 257–270. 10.1007/BF00025267
</cite> [<a href="https://doi.org/10.1007/BF00025267" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Significance%20of%20wild%20species%20and%20primitive%20forms%20for%20potato%20breeding&amp;author=J.%20G.%20Hawkes&amp;volume=7&amp;issue=3&amp;publication_year=1958&amp;pages=257-270&amp;doi=10.1007/BF00025267&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0051">
<cite>

Hawkes, J. G.
 (1962). The origin of <em>Solanum juzepczukii</em> Buk. and <em>S. curtilobum</em> Juz. et Buk. Zeitschrift für Pflanzenzüchtung, 47, 1–14.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Zeitschrift%20f%C3%BCr%20Pflanzenz%C3%BCchtung&amp;title=The%20origin%20of%20Solanum%20juzepczukii%20Buk.%20and%20S.%20curtilobum%20Juz.%20et%20Buk&amp;author=J.%20G.%20Hawkes&amp;volume=47&amp;publication_year=1962&amp;pages=1-14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0052">
<cite>

Hawkes, J. G.
 (1990). The potato: Evolution, biodiversity and genetic resources. Belhaven Press.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20potato:%20Evolution,%20biodiversity%20and%20genetic%20resources&amp;author=J.%20G.%20Hawkes&amp;publication_year=1990&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0053">
<cite>

Hijmans, R. J.
, 
Jacobs, M.
, 
Bamberg, J. B.
, &amp; 
Spooner, D. M.
 (2003). Frost tolerance in wild potato species: Assessing the predictivity of taxonomic, geographic, and ecological factors. Euphytica, 130, 47–59. 10.1023/A:1022344327669
</cite> [<a href="https://doi.org/10.1023/A:1022344327669" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Frost%20tolerance%20in%20wild%20potato%20species:%20Assessing%20the%20predictivity%20of%20taxonomic,%20geographic,%20and%20ecological%20factors&amp;author=R.%20J.%20Hijmans&amp;author=M.%20Jacobs&amp;author=J.%20B.%20Bamberg&amp;author=D.%20M.%20Spooner&amp;volume=130&amp;publication_year=2003&amp;pages=47-59&amp;doi=10.1023/A:1022344327669&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0054">
<cite>

Hoff, K. J.
, 
Lange, S.
, 
Lomsadze, A.
, 
Borodovsky, M.
, &amp; 
Stanke, M.
 (2016). BRAKER1: Unsupervised RNA‐seq‐based genome annotation with GeneMark‐ET and AUGUSTUS. Bioinformatics, 32, 767–769. 10.1093/bioinformatics/btv661

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv661" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6078167/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26559507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BRAKER1:%20Unsupervised%20RNA%E2%80%90seq%E2%80%90based%20genome%20annotation%20with%20GeneMark%E2%80%90ET%20and%20AUGUSTUS&amp;author=K.%20J.%20Hoff&amp;author=S.%20Lange&amp;author=A.%20Lomsadze&amp;author=M.%20Borodovsky&amp;author=M.%20Stanke&amp;volume=32&amp;publication_year=2016&amp;pages=767-769&amp;pmid=26559507&amp;doi=10.1093/bioinformatics/btv661&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0055">
<cite>

Hoff, K. J.
, 
Lomsadze, A.
, 
Borodovsky, M.
, &amp; 
Stanke, M.
 (2019). Whole‐genome annotation with BRAKER. Methods in Molecular Biology, 1962, 65–95.
</cite> [<a href="https://doi.org/10.1007/978-1-4939-9173-0_5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6635606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31020555/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20in%20Molecular%20Biology&amp;title=Whole%E2%80%90genome%20annotation%20with%20BRAKER&amp;author=K.%20J.%20Hoff&amp;author=A.%20Lomsadze&amp;author=M.%20Borodovsky&amp;author=M.%20Stanke&amp;volume=1962&amp;publication_year=2019&amp;pages=65-95&amp;pmid=31020555&amp;doi=10.1007/978-1-4939-9173-0_5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0056">
<cite>

Hoopes, G.
, 
Meng, X.
, 
Hamilton, J. P.
, 
Achakkagari, S. R.
, 
De Alves Freitas Guesdes, F.
, 
Bolger, M. E.
, 
Coombs, J. J.
, 
Esselink, D.
, 
Kaiser, N. R.
, 
Kodde, L.
, 
Kyriakidou, M.
, 
Lavrijssen, B.
, 
Van Lieshout, N.
, 
Shereda, R.
, 
Tuttle, H. K.
, 
Vaillancourt, B.
, 
Wood, J. C.
, 
De Boer, J. M.
, 
Bornowski, N.
, … 
Finkers, R.
 (2022). Phased, chromosome‐scale genome assemblies of tetraploid potato reveal a complex genome, transcriptome, and predicted proteome landscape underpinning genetic diversity. Molecular Plant, 15, 520–536. 10.1016/j.molp.2022.01.003

</cite> [<a href="https://doi.org/10.1016/j.molp.2022.01.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35026436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=Phased,%20chromosome%E2%80%90scale%20genome%20assemblies%20of%20tetraploid%20potato%20reveal%20a%20complex%20genome,%20transcriptome,%20and%20predicted%20proteome%20landscape%20underpinning%20genetic%20diversity&amp;author=G.%20Hoopes&amp;author=X.%20Meng&amp;author=J.%20P.%20Hamilton&amp;author=S.%20R.%20Achakkagari&amp;author=F.%20De%20Alves%20Freitas%20Guesdes&amp;volume=15&amp;publication_year=2022&amp;pages=520-536&amp;pmid=35026436&amp;doi=10.1016/j.molp.2022.01.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0057">
<cite>

Hosaka, A. J.
, 
Sanetomo, R.
, &amp; 
Hosaka, K.
 (2022). A de novo genome assembly of <em>Solanum verrucosum</em> Schlechtendal, a Mexican diploid species geographically isolated from other diploid A‐genome species of potato relatives. G3 Genes|Genomes|Genetics, 12, jkac166. 10.1093/g3journal/jkac166

</cite> [<a href="https://doi.org/10.1093/g3journal/jkac166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9339273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35775942/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=A%20de%20novo%20genome%20assembly%20of%20Solanum%20verrucosum%20Schlechtendal,%20a%20Mexican%20diploid%20species%20geographically%20isolated%20from%20other%20diploid%20A%E2%80%90genome%20species%20of%20potato%20relatives&amp;author=A.%20J.%20Hosaka&amp;author=R.%20Sanetomo&amp;author=K.%20Hosaka&amp;volume=12&amp;publication_year=2022&amp;pages=jkac166&amp;pmid=35775942&amp;doi=10.1093/g3journal/jkac166&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0058">
<cite>

Hosaka, A. J.
, 
Sanetomo, R.
, &amp; 
Hosaka, K.
 (2025). Allotetraploid nature of a wild potato species, <em>Solanum stoloniferum</em> Schlechtd. et Bché., as revealed by whole‐genome sequencing. Plant Journal, 121, e17158. 10.1111/tpj.17158
</cite> [<a href="https://doi.org/10.1111/tpj.17158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11703546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39585203/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Journal&amp;title=Allotetraploid%20nature%20of%20a%20wild%20potato%20species,%20Solanum%20stoloniferum%20Schlechtd.%20et%20Bch%C3%A9.,%20as%20revealed%20by%20whole%E2%80%90genome%20sequencing&amp;author=A.%20J.%20Hosaka&amp;author=R.%20Sanetomo&amp;author=K.%20Hosaka&amp;volume=121&amp;publication_year=2025&amp;pages=e17158&amp;pmid=39585203&amp;doi=10.1111/tpj.17158&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0147">
<cite>

Hosaka, K.
, &amp; 
Spooner, D. M.
 (1992). RFLP analysis of the wild potato species, <em>Solanum acaule Bitter</em> (<em>Solanum sect. Petota</em>). Theoretical and Applied Genetics, 84(7–8), 851–858. 10.1007/bf00227396

</cite> [<a href="https://doi.org/10.1007/bf00227396" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24201486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=RFLP%20analysis%20of%20the%20wild%20potato%20species,%20Solanum%20acaule%20Bitter%20(Solanum%20sect.%20Petota)&amp;author=K.%20Hosaka&amp;author=D.%20M.%20Spooner&amp;volume=84&amp;issue=7%E2%80%938&amp;publication_year=1992&amp;pages=851-858&amp;pmid=24201486&amp;doi=10.1007/bf00227396&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0059">
<cite>

Hu, J.
, 
Wang, Z.
, 
Sun, Z.
, 
Hu, B.
, 
Ayoola, A. O.
, 
Liang, F.
, 
Li, J.
, 
Sandoval, J. R.
, 
Cooper, D. N.
, 
Ye, K.
, 
Ruan, J.
, 
Xiao, C.‐L.
, 
Wang, D.
, 
Wu, D.‐D.
, &amp; 
Wang, S.
 (2024). NextDenovo: An efficient error correction and accurate assembly tool for noisy long reads. Genome Biology, 25, Article 107. 10.1186/s13059-024-03252-4

</cite> [<a href="https://doi.org/10.1186/s13059-024-03252-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11046930/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38671502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=NextDenovo:%20An%20efficient%20error%20correction%20and%20accurate%20assembly%20tool%20for%20noisy%20long%20reads&amp;author=J.%20Hu&amp;author=Z.%20Wang&amp;author=Z.%20Sun&amp;author=B.%20Hu&amp;author=A.%20O.%20Ayoola&amp;volume=25&amp;publication_year=2024&amp;pages=107&amp;pmid=38671502&amp;doi=10.1186/s13059-024-03252-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0060">
<cite>

Huang, B.
, 
Ruess, H.
, 
Liang, Q.
, 
Colleoni, C.
, &amp; 
Spooner, D. M.
 (2019). Analyses of 202 plastid genomes elucidate the phylogeny of <em>Solanum</em> section <em>Petota</em>
. Scientific Reports, 9(1), Article 4454. 10.1038/s41598-019-40790-5

</cite> [<a href="https://doi.org/10.1038/s41598-019-40790-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6418237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30872631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Analyses%20of%20202%20plastid%20genomes%20elucidate%20the%20phylogeny%20of%20Solanum%20section%20Petota&amp;author=B.%20Huang&amp;author=H.%20Ruess&amp;author=Q.%20Liang&amp;author=C.%20Colleoni&amp;author=D.%20M.%20Spooner&amp;volume=9&amp;issue=1&amp;publication_year=2019&amp;pages=4454&amp;pmid=30872631&amp;doi=10.1038/s41598-019-40790-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0061">
<cite>

Iwata, H.
, &amp; 
Gotoh, O.
 (2012). Benchmarking spliced alignment programs including Spaln2, an extended version of Spaln that incorporates additional species‐specific features. Nucleic Acids Research, 40, e161. 10.1093/nar/gks708

</cite> [<a href="https://doi.org/10.1093/nar/gks708" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3488211/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22848105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Benchmarking%20spliced%20alignment%20programs%20including%20Spaln2,%20an%20extended%20version%20of%20Spaln%20that%20incorporates%20additional%20species%E2%80%90specific%20features&amp;author=H.%20Iwata&amp;author=O.%20Gotoh&amp;volume=40&amp;publication_year=2012&amp;pages=e161&amp;pmid=22848105&amp;doi=10.1093/nar/gks708&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0062">
<cite>

Jia, K.‐H.
, 
Wang, Z.‐X.
, 
Wang, L.
, 
Li, G.‐Y.
, 
Zhang, W.
, 
Wang, X.‐L.
, 
Xu, F.‐J.
, 
Jiao, S.‐Q.
, 
Zhou, S.‐S.
, 
Liu, H.
, 
Ma, Y.
, 
Bi, G.
, 
Zhao, W.
, 
El‐Kassaby, Y. A.
, 
Porth, I.
, 
Li, G.
, 
Zhang, R.‐G.
, &amp; 
Mao, J.‐F.
 (2022). SubPhaser: A robust allopolyploid subgenome phasing method based on subgenome‐specific <em>k</em>‐mers. New Phytologist, 235, 801–809. 10.1111/nph.18173

</cite> [<a href="https://doi.org/10.1111/nph.18173" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35460274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=SubPhaser:%20A%20robust%20allopolyploid%20subgenome%20phasing%20method%20based%20on%20subgenome%E2%80%90specific%20k%E2%80%90mers&amp;author=K.%E2%80%90H.%20Jia&amp;author=Z.%E2%80%90X.%20Wang&amp;author=L.%20Wang&amp;author=G.%E2%80%90Y.%20Li&amp;author=W.%20Zhang&amp;volume=235&amp;publication_year=2022&amp;pages=801-809&amp;pmid=35460274&amp;doi=10.1111/nph.18173&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0063">
<cite>

Jiang, Z.
, 
Peng, Z.
, 
Wei, Z.
, 
Sun, J.
, 
Luo, Y.
, 
Bie, L.
, 
Zhang, G.
, &amp; 
Wang, Y.
 (2024). A deep learning‐based method enables the automatic and accurate assembly of chromosome‐level genomes. Nucleic Acids Research, 52, e92. 10.1093/nar/gkae789

</cite> [<a href="https://doi.org/10.1093/nar/gkae789" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11514472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39287126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=A%20deep%20learning%E2%80%90based%20method%20enables%20the%20automatic%20and%20accurate%20assembly%20of%20chromosome%E2%80%90level%20genomes&amp;author=Z.%20Jiang&amp;author=Z.%20Peng&amp;author=Z.%20Wei&amp;author=J.%20Sun&amp;author=Y.%20Luo&amp;volume=52&amp;publication_year=2024&amp;pages=e92&amp;pmid=39287126&amp;doi=10.1093/nar/gkae789&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0064">
<cite>

Karlsson, B. H.
, &amp; 
Palta, J. P.
 (1999). Expression of heat‐stable and putative dehydrin proteins during nonacclimated and cold‐acclimated conditions in several tuber‐bearing <em>Solanum</em> species. Journal of the American Society for Horticultural Science, 124, 245–251. 10.21273/JASHS.124.3.245
</cite> [<a href="https://doi.org/10.21273/JASHS.124.3.245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Horticultural%20Science&amp;title=Expression%20of%20heat%E2%80%90stable%20and%20putative%20dehydrin%20proteins%20during%20nonacclimated%20and%20cold%E2%80%90acclimated%20conditions%20in%20several%20tuber%E2%80%90bearing%20Solanum%20species&amp;author=B.%20H.%20Karlsson&amp;author=J.%20P.%20Palta&amp;volume=124&amp;publication_year=1999&amp;pages=245-251&amp;doi=10.21273/JASHS.124.3.245&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0066">
<cite>

Khoso, M. A.
, 
Hussain, A.
, 
Ritonga, F. N.
, 
Ali, Q.
, 
Channa, M. M.
, 
Alshegaihi, R. M.
, 
Meng, Q.
, 
Ali, M.
, 
Zaman, W.
, 
Brohi, R. D.
, 
Liu, F.
, &amp; 
Manghwar, H.
 (2022). WRKY transcription factors (TFs): Molecular switches to regulate drought, temperature, and salinity stresses in plants. Frontiers in Plant Science, 13, 1039329. 10.3389/fpls.2022.1039329

</cite> [<a href="https://doi.org/10.3389/fpls.2022.1039329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9679293/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36426143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=WRKY%20transcription%20factors%20(TFs):%20Molecular%20switches%20to%20regulate%20drought,%20temperature,%20and%20salinity%20stresses%20in%20plants&amp;author=M.%20A.%20Khoso&amp;author=A.%20Hussain&amp;author=F.%20N.%20Ritonga&amp;author=Q.%20Ali&amp;author=M.%20M.%20Channa&amp;volume=13&amp;publication_year=2022&amp;pages=1039329&amp;pmid=36426143&amp;doi=10.3389/fpls.2022.1039329&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0067">
<cite>

Kim, D.
, 
Paggi, J. M.
, 
Park, C.
, 
Bennett, C.
, &amp; 
Salzberg, S. L.
 (2019). Graph‐based genome alignment and genotyping with HISAT2 and HISAT‐genotype. Nature Biotechnology, 37, 907–915. 10.1038/s41587-019-0201-4
</cite> [<a href="https://doi.org/10.1038/s41587-019-0201-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7605509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31375807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Graph%E2%80%90based%20genome%20alignment%20and%20genotyping%20with%20HISAT2%20and%20HISAT%E2%80%90genotype&amp;author=D.%20Kim&amp;author=J.%20M.%20Paggi&amp;author=C.%20Park&amp;author=C.%20Bennett&amp;author=S.%20L.%20Salzberg&amp;volume=37&amp;publication_year=2019&amp;pages=907-915&amp;pmid=31375807&amp;doi=10.1038/s41587-019-0201-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0068">
<cite>

Kolmogorov, M.
, 
Yuan, J.
, 
Lin, Y.
, &amp; 
Pevzner, P. A.
 (2019). Assembly of long, error‐prone reads using repeat graphs. Nature Biotechnology, 37, 540–546. 10.1038/s41587-019-0072-8
</cite> [<a href="https://doi.org/10.1038/s41587-019-0072-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30936562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Assembly%20of%20long,%20error%E2%80%90prone%20reads%20using%20repeat%20graphs&amp;author=M.%20Kolmogorov&amp;author=J.%20Yuan&amp;author=Y.%20Lin&amp;author=P.%20A.%20Pevzner&amp;volume=37&amp;publication_year=2019&amp;pages=540-546&amp;pmid=30936562&amp;doi=10.1038/s41587-019-0072-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0069">
<cite>

Kou, S.
, 
Chen, L.
, 
Tu, W.
, 
Scossa, F.
, 
Wang, Y.
, 
Liu, J.
, 
Fernie, A. R.
, 
Song, B.
, &amp; 
Xie, C.
 (2018). The arginine decarboxylase gene <em>ADC1</em>, associated to the putrescine pathway, plays an important role in potato cold‐acclimated freezing tolerance as revealed by transcriptome and metabolome analyses. Plant Journal, 96, 1283–1298. 10.1111/tpj.14126
</cite> [<a href="https://doi.org/10.1111/tpj.14126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30307077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Journal&amp;title=The%20arginine%20decarboxylase%20gene%20ADC1,%20associated%20to%20the%20putrescine%20pathway,%20plays%20an%20important%20role%20in%20potato%20cold%E2%80%90acclimated%20freezing%20tolerance%20as%20revealed%20by%20transcriptome%20and%20metabolome%20analyses&amp;author=S.%20Kou&amp;author=L.%20Chen&amp;author=W.%20Tu&amp;author=F.%20Scossa&amp;author=Y.%20Wang&amp;volume=96&amp;publication_year=2018&amp;pages=1283-1298&amp;pmid=30307077&amp;doi=10.1111/tpj.14126&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0070">
<cite>

Kovaka, S.
, 
Zimin, A. V.
, 
Pertea, G. M.
, 
Razaghi, R.
, 
Salzberg, S. L.
, &amp; 
Pertea, M.
 (2019). Transcriptome assembly from long‐read RNA‐seq alignments with StringTie2. Genome Biology, 20, Article 278. 10.1186/s13059-019-1910-1

</cite> [<a href="https://doi.org/10.1186/s13059-019-1910-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6912988/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31842956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Transcriptome%20assembly%20from%20long%E2%80%90read%20RNA%E2%80%90seq%20alignments%20with%20StringTie2&amp;author=S.%20Kovaka&amp;author=A.%20V.%20Zimin&amp;author=G.%20M.%20Pertea&amp;author=R.%20Razaghi&amp;author=S.%20L.%20Salzberg&amp;volume=20&amp;publication_year=2019&amp;pages=278&amp;pmid=31842956&amp;doi=10.1186/s13059-019-1910-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0071">
<cite>

Kumar, S.
, 
Stecher, G.
, 
Li, M.
, 
Knyaz, C.
, &amp; 
Tamura, K.
 (2018). MEGA X: Molecular evolutionary genetics analysis across computing platforms. Molecular Biology and Evolution, 35, 1547–1549. 10.1093/molbev/msy096

</cite> [<a href="https://doi.org/10.1093/molbev/msy096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5967553/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29722887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=MEGA%20X:%20Molecular%20evolutionary%20genetics%20analysis%20across%20computing%20platforms&amp;author=S.%20Kumar&amp;author=G.%20Stecher&amp;author=M.%20Li&amp;author=C.%20Knyaz&amp;author=K.%20Tamura&amp;volume=35&amp;publication_year=2018&amp;pages=1547-1549&amp;pmid=29722887&amp;doi=10.1093/molbev/msy096&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0072">
<cite>

Kyriakidou, M.
, 
Achakkagari, S. R.
, 
Gálvez López, J. H.
, 
Zhu, X.
, 
Tang, C. Y.
, 
Tai, H. H.
, 
Anglin, N. L.
, 
Ellis, D.
, &amp; 
Strömvik, M. V.
 (2020). Structural genome analysis in cultivated potato taxa. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 133, 951–966. 10.1007/s00122-019-03519-6

</cite> [<a href="https://doi.org/10.1007/s00122-019-03519-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7021743/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31893289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Structural%20genome%20analysis%20in%20cultivated%20potato%20taxa&amp;author=M.%20Kyriakidou&amp;author=S.%20R.%20Achakkagari&amp;author=J.%20H.%20G%C3%A1lvez%20L%C3%B3pez&amp;author=X.%20Zhu&amp;author=C.%20Y.%20Tang&amp;volume=133&amp;publication_year=2020&amp;pages=951-966&amp;pmid=31893289&amp;doi=10.1007/s00122-019-03519-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0073">
<cite>

Kyriakidou, M.
, 
Anglin, N. L.
, 
Ellis, D.
, 
Tai, H. H.
, &amp; 
Strömvik, M. V.
 (2020). Genome assembly of six polyploid potato genomes. Scientific Data, 7, Article 88. 10.1038/s41597-020-0428-4

</cite> [<a href="https://doi.org/10.1038/s41597-020-0428-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7066127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32161269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Data&amp;title=Genome%20assembly%20of%20six%20polyploid%20potato%20genomes&amp;author=M.%20Kyriakidou&amp;author=N.%20L.%20Anglin&amp;author=D.%20Ellis&amp;author=H.%20H.%20Tai&amp;author=M.%20V.%20Str%C3%B6mvik&amp;volume=7&amp;publication_year=2020&amp;pages=88&amp;pmid=32161269&amp;doi=10.1038/s41597-020-0428-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0074">
<cite>

Laetsch, D. R.
, &amp; 
Blaxter, M. L.
 (2017). BlobTools: Interrogation of genome assemblies [version 1; peer review: 2 approved with reservations]. F1000Research, 6, 1287. 10.12688/f1000research.12232.1
</cite> [<a href="https://doi.org/10.12688/f1000research.12232.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=F1000Research&amp;title=BlobTools:%20Interrogation%20of%20genome%20assemblies%20%5Bversion%201;%20peer%20review:%202%20approved%20with%20reservations%5D&amp;author=D.%20R.%20Laetsch&amp;author=M.%20L.%20Blaxter&amp;volume=6&amp;publication_year=2017&amp;pages=1287&amp;doi=10.12688/f1000research.12232.1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0075">
<cite>

Leisner, C. P.
, 
Hamilton, J. P.
, 
Crisovan, E.
, 
Manrique‐Carpintero, N. C.
, 
Marand, A. P.
, 
Newton, L.
, 
Pham, G. M.
, 
Jiang, J.
, 
Douches, D. S.
, 
Jansky, S. H.
, &amp; 
Buell, C. R.
 (2018). Genome sequence of M6, a diploid inbred clone of the high‐glycoalkaloid‐producing tuber‐bearing potato species <em>Solanum chacoense</em>, reveals residual heterozygosity. Plant Journal, 94, 562–570.</cite> [<a href="https://doi.org/10.1111/tpj.13857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29405524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Journal&amp;title=Genome%20sequence%20of%20M6,%20a%20diploid%20inbred%20clone%20of%20the%20high%E2%80%90glycoalkaloid%E2%80%90producing%20tuber%E2%80%90bearing%20potato%20species%20Solanum%20chacoense,%20reveals%20residual%20heterozygosity&amp;author=C.%20P.%20Leisner&amp;author=J.%20P.%20Hamilton&amp;author=E.%20Crisovan&amp;author=N.%20C.%20Manrique%E2%80%90Carpintero&amp;author=A.%20P.%20Marand&amp;volume=94&amp;publication_year=2018&amp;pages=562-570&amp;pmid=29405524&amp;doi=10.1111/tpj.13857&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0076">
<cite>

Li, A.
, 
Liu, D.
, 
Wu, J.
, 
Zhao, X.
, 
Hao, M.
, 
Geng, S.
, 
Yan, J.
, 
Jiang, X.
, 
Zhang, L.
, 
Wu, J.
, 
Yin, L.
, 
Zhang, R.
, 
Wu, L.
, 
Zheng, Y.
, &amp; 
Mao, L.
 (2014). mRNA and small RNA transcriptomes reveal insights into dynamic homoeolog regulation of allopolyploid heterosis in nascent hexaploid wheat. Plant Cell, 26, 1878–1900. 10.1105/tpc.114.124388

</cite> [<a href="https://doi.org/10.1105/tpc.114.124388" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4079356/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24838975/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=mRNA%20and%20small%20RNA%20transcriptomes%20reveal%20insights%20into%20dynamic%20homoeolog%20regulation%20of%20allopolyploid%20heterosis%20in%20nascent%20hexaploid%20wheat&amp;author=A.%20Li&amp;author=D.%20Liu&amp;author=J.%20Wu&amp;author=X.%20Zhao&amp;author=M.%20Hao&amp;volume=26&amp;publication_year=2014&amp;pages=1878-1900&amp;pmid=24838975&amp;doi=10.1105/tpc.114.124388&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0077">
<cite>

Li, F.
, 
Xu, J.‐F.
, 
Liu, J.
, 
Duan, S.‐G.
, 
Bian, C.‐S.
, 
Jiwan, P. P.
, &amp; 
Jin, L.‐P.
 (2014). Molecular cloning and expression analysis of <em>FAD2</em> gene from three wild potato species with different levels of freezing Tolerance. Acta Agronomica Sinica, 40, 45–53. 10.3724/SP.J.1006.2014.00045
</cite> [<a href="https://doi.org/10.3724/SP.J.1006.2014.00045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Agronomica%20Sinica&amp;title=Molecular%20cloning%20and%20expression%20analysis%20of%20FAD2%20gene%20from%20three%20wild%20potato%20species%20with%20different%20levels%20of%20freezing%20Tolerance&amp;author=F.%20Li&amp;author=J.%E2%80%90F.%20Xu&amp;author=J.%20Liu&amp;author=S.%E2%80%90G.%20Duan&amp;author=C.%E2%80%90S.%20Bian&amp;volume=40&amp;publication_year=2014&amp;pages=45-53&amp;doi=10.3724/SP.J.1006.2014.00045&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0078">
<cite>

Li, H.
 (2013). Aligning sequence reads, clone sequences and assembly contigs with BWA‐MEM. <em>arXiv</em>
. 10.48550/arXiv.1303.3997
</cite> [<a href="https://doi.org/10.48550/arXiv.1303.3997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270095-bib-0079">
<cite>

Li, H.
, 
Handsaker, B.
, 
Wysoker, A.
, 
Fennell, T.
, 
Ruan, J.
, 
Homer, N.
, 
Marth, G.
, 
Abecasis, G.
, &amp; 
Durbin, R.
 (2009). The sequence alignment/map format and SAMtools. Bioinformatics, 25, 2078–2079. 10.1093/bioinformatics/btp352

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2723002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The%20sequence%20alignment/map%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;author=T.%20Fennell&amp;author=J.%20Ruan&amp;volume=25&amp;publication_year=2009&amp;pages=2078-2079&amp;pmid=19505943&amp;doi=10.1093/bioinformatics/btp352&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0080">
<cite>

Li, K.
, 
Xu, P.
, 
Wang, J.
, 
Yi, X.
, &amp; 
Jiao, Y.
 (2023). Identification of errors in draft genome assemblies at single‐nucleotide resolution for quality assessment and improvement. Nature Communications, 14, Article 6556. 10.1038/s41467-023-42336-w
</cite> [<a href="https://doi.org/10.1038/s41467-023-42336-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10582259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37848433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Identification%20of%20errors%20in%20draft%20genome%20assemblies%20at%20single%E2%80%90nucleotide%20resolution%20for%20quality%20assessment%20and%20improvement&amp;author=K.%20Li&amp;author=P.%20Xu&amp;author=J.%20Wang&amp;author=X.%20Yi&amp;author=Y.%20Jiao&amp;volume=14&amp;publication_year=2023&amp;pages=6556&amp;pmid=37848433&amp;doi=10.1038/s41467-023-42336-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0081">
<cite>

Li, P. H.
 (1977). Frost killing temperatures of 60 tuber‐bearing <em>Solanum</em> species. American Journal of Potato Research, 54, 452–456. 10.1007/BF02852059
</cite> [<a href="https://doi.org/10.1007/BF02852059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Potato%20Research&amp;title=Frost%20killing%20temperatures%20of%2060%20tuber%E2%80%90bearing%20Solanum%20species&amp;author=P.%20H.%20Li&amp;volume=54&amp;publication_year=1977&amp;pages=452-456&amp;doi=10.1007/BF02852059&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0082">
<cite>

Li, Q.
, 
Li, H.
, 
Huang, W.
, 
Xu, Y.
, 
Zhou, Q.
, 
Wang, S.
, 
Ruan, J.
, 
Huang, S.
, &amp; 
Zhang, Z.
 (2019). A chromosome‐scale genome assembly of cucumber (<em>Cucumis sativus</em> L.). GigaScience, 8, giz072. 10.1093/gigascience/giz072

</cite> [<a href="https://doi.org/10.1093/gigascience/giz072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6582320/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31216035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GigaScience&amp;title=A%20chromosome%E2%80%90scale%20genome%20assembly%20of%20cucumber%20(Cucumis%20sativus%20L.)&amp;author=Q.%20Li&amp;author=H.%20Li&amp;author=W.%20Huang&amp;author=Y.%20Xu&amp;author=Q.%20Zhou&amp;volume=8&amp;publication_year=2019&amp;pages=giz072&amp;pmid=31216035&amp;doi=10.1093/gigascience/giz072&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0083">
<cite>

Li, W.
, 
Chen, Y.
, 
Ye, M. H.
, 
Lu, H. B.
, 
Wang, D. D.
, &amp; 
Chen, Q.
 (2020). Evolutionary history of the C‐repeat binding factor/dehydration‐responsive element‐binding 1 (CBF/DREB1) protein family in 43 plant species and characterization of CBF/DREB1 proteins in <em>Solanum tuberosum</em>
. BMC Evolutionary Biology, 20, Article 142. 10.1186/s12862-020-01710-8

</cite> [<a href="https://doi.org/10.1186/s12862-020-01710-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7607821/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33143637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Evolutionary%20Biology&amp;title=Evolutionary%20history%20of%20the%20C%E2%80%90repeat%20binding%20factor/dehydration%E2%80%90responsive%20element%E2%80%90binding%201%20(CBF/DREB1)%20protein%20family%20in%2043%20plant%20species%20and%20characterization%20of%20CBF/DREB1%20proteins%20in%20Solanum%20tuberosum&amp;author=W.%20Li&amp;author=Y.%20Chen&amp;author=M.%20H.%20Ye&amp;author=H.%20B.%20Lu&amp;author=D.%20D.%20Wang&amp;volume=20&amp;publication_year=2020&amp;pages=142&amp;pmid=33143637&amp;doi=10.1186/s12862-020-01710-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0084">
<cite>

Li, Y.
, 
Colleoni, C.
, 
Zhang, J.
, 
Liang, Q.
, 
Hu, Y.
, 
Ruess, H.
, 
Simon, R.
, 
Liu, Y.
, 
Liu, H.
, 
Yu, G.
, 
Schmitt, E.
, 
Ponitzki, C.
, 
Liu, G.
, 
Huang, H.
, 
Zhan, F.
, 
Chen, L.
, 
Huang, Y.
, 
Spooner, D.
, &amp; 
Huang, B.
 (2018). Genomic analyses yield markers for identifying agronomically important genes in Potato. Molecular Plant, 11, 473–484. 10.1016/j.molp.2018.01.009

</cite> [<a href="https://doi.org/10.1016/j.molp.2018.01.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29421339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=Genomic%20analyses%20yield%20markers%20for%20identifying%20agronomically%20important%20genes%20in%20Potato&amp;author=Y.%20Li&amp;author=C.%20Colleoni&amp;author=J.%20Zhang&amp;author=Q.%20Liang&amp;author=Y.%20Hu&amp;volume=11&amp;publication_year=2018&amp;pages=473-484&amp;pmid=29421339&amp;doi=10.1016/j.molp.2018.01.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0085">
<cite>

Love, M. I.
, 
Huber, W.
, &amp; 
Anders, S.
 (2014). Moderated estimation of fold change and dispersion for RNA‐seq data with DESeq2. Genome Biology, 15, Article 550. 10.1186/s13059-014-0550-8

</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA%E2%80%90seq%20data%20with%20DESeq2&amp;author=M.%20I.%20Love&amp;author=W.%20Huber&amp;author=S.%20Anders&amp;volume=15&amp;publication_year=2014&amp;pages=550&amp;pmid=25516281&amp;doi=10.1186/s13059-014-0550-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0086">
<cite>

Love, S. L.
 (1999). Founding clones, major contributing ancestors, and exotic progenitors of prominent North American potato cultivars. American Journal of Potato Research, 76, 263–272. 10.1007/BF02853624
</cite> [<a href="https://doi.org/10.1007/BF02853624" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Potato%20Research&amp;title=Founding%20clones,%20major%20contributing%20ancestors,%20and%20exotic%20progenitors%20of%20prominent%20North%20American%20potato%20cultivars&amp;author=S.%20L.%20Love&amp;volume=76&amp;publication_year=1999&amp;pages=263-272&amp;doi=10.1007/BF02853624&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0087">
<cite>

Luo, X.
, 
Hu, Q.
, 
Zhou, P.
, 
Zhang, D.
, 
Wang, Q.
, 
Abbott, R. J.
, &amp; 
Liu, J.
 (2017). Chasing ghosts: Allopolyploid origin of <em>Oxyria sinensis</em> (Polygonaceae) from its only diploid congener and an unknown ancestor. Molecular Ecology, 26, 3037–3049. 10.1111/mec.14097

</cite> [<a href="https://doi.org/10.1111/mec.14097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28295838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Chasing%20ghosts:%20Allopolyploid%20origin%20of%20Oxyria%20sinensis%20(Polygonaceae)%20from%20its%20only%20diploid%20congener%20and%20an%20unknown%20ancestor&amp;author=X.%20Luo&amp;author=Q.%20Hu&amp;author=P.%20Zhou&amp;author=D.%20Zhang&amp;author=Q.%20Wang&amp;volume=26&amp;publication_year=2017&amp;pages=3037-3049&amp;pmid=28295838&amp;doi=10.1111/mec.14097&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0088">
<cite>

Marçais, G.
, &amp; 
Kingsford, C.
 (2011). A fast, lock‐free approach for efficient parallel counting of occurrences of k‐mers. Bioinformatics, 27, 764–770.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3051319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21217122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20fast,%20lock%E2%80%90free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k%E2%80%90mers&amp;author=G.%20Mar%C3%A7ais&amp;author=C.%20Kingsford&amp;volume=27&amp;publication_year=2011&amp;pages=764-770&amp;pmid=21217122&amp;doi=10.1093/bioinformatics/btr011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0089">
<cite>

Matsubayashi, M.
 (1991). Phylogenetic relationships in the potato and its related species. In Tsuchiya T. &amp; Gupta P. K. (Eds.), Chromosome engineering in plants: Genetics, breeding, evolution. Part B (pp. 93–118). Elsevier.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Chromosome%20engineering%20in%20plants:%20Genetics,%20breeding,%20evolution.%20Part%20B&amp;author=M.%20Matsubayashi&amp;publication_year=1991&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0090">
<cite>

Mboup, M.
, 
Fischer, I.
, 
Lainer, H.
, &amp; 
Stephan, W.
 (2012). Trans‐species polymorphism and allele‐specific expression in the <em>CBF</em> gene family of wild tomatoes. Molecular Biology and Evolution, 29(12), 3641–3652. 10.1093/molbev/mss176

</cite> [<a href="https://doi.org/10.1093/molbev/mss176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22787283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=Trans%E2%80%90species%20polymorphism%20and%20allele%E2%80%90specific%20expression%20in%20the%20CBF%20gene%20family%20of%20wild%20tomatoes&amp;author=M.%20Mboup&amp;author=I.%20Fischer&amp;author=H.%20Lainer&amp;author=W.%20Stephan&amp;volume=29&amp;issue=12&amp;publication_year=2012&amp;pages=3641-3652&amp;pmid=22787283&amp;doi=10.1093/molbev/mss176&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0091">
<cite>

Minh, B. Q.
, 
Schmidt, H. A.
, 
Chernomor, O.
, 
Schrempf, D.
, 
Woodhams, M. D.
, 
Von Haeseler, A.
, &amp; 
Lanfear, R.
 (2020). IQ‐TREE 2: New models and efficient methods for phylogenetic inference in the genomic era. Molecular Biology and Evolution, 37, 1530–1534. 10.1093/molbev/msaa015

</cite> [<a href="https://doi.org/10.1093/molbev/msaa015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7182206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32011700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=IQ%E2%80%90TREE%202:%20New%20models%20and%20efficient%20methods%20for%20phylogenetic%20inference%20in%20the%20genomic%20era&amp;author=B.%20Q.%20Minh&amp;author=H.%20A.%20Schmidt&amp;author=O.%20Chernomor&amp;author=D.%20Schrempf&amp;author=M.%20D.%20Woodhams&amp;volume=37&amp;publication_year=2020&amp;pages=1530-1534&amp;pmid=32011700&amp;doi=10.1093/molbev/msaa015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0092">
<cite>

Mistry, J.
, 
Chuguransky, S.
, 
Williams, L.
, 
Qureshi, M.
, 
Salazar, G. A.
, 
Sonnhammer, E. L. L.
, 
Tosatto, S. C. E.
, 
Paladin, L.
, 
Raj, S.
, 
Richardson, L. J.
, 
Finn, R. D.
, &amp; 
Bateman, A.
 (2021). Pfam: The protein families database in 2021. Nucleic Acids Research, 49, D412–D419. 10.1093/nar/gkaa913

</cite> [<a href="https://doi.org/10.1093/nar/gkaa913" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7779014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33125078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Pfam:%20The%20protein%20families%20database%20in%202021&amp;author=J.%20Mistry&amp;author=S.%20Chuguransky&amp;author=L.%20Williams&amp;author=M.%20Qureshi&amp;author=G.%20A.%20Salazar&amp;volume=49&amp;publication_year=2021&amp;pages=D412-D419&amp;pmid=33125078&amp;doi=10.1093/nar/gkaa913&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0093">
<cite>

Nakagawa, K.
, &amp; 
Hosaka, K.
 (2002). Species relationships between a wild tetraploid potato species, <em>Solanum acaule</em> Bitter, and its related species as revealed by RFLPs of chloroplast and nuclear DNA. American Journal of Potato Research, 79, 85–98. 10.1007/BF02881517
</cite> [<a href="https://doi.org/10.1007/BF02881517" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Potato%20Research&amp;title=Species%20relationships%20between%20a%20wild%20tetraploid%20potato%20species,%20Solanum%20acaule%20Bitter,%20and%20its%20related%20species%20as%20revealed%20by%20RFLPs%20of%20chloroplast%20and%20nuclear%20DNA&amp;author=K.%20Nakagawa&amp;author=K.%20Hosaka&amp;volume=79&amp;publication_year=2002&amp;pages=85-98&amp;doi=10.1007/BF02881517&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0096">
<cite>

Nevers, Y.
, 
Warwick Vesztrocy, A.
, 
Rossier, V.
, 
Train, C.‐M.
, 
Altenhoff, A.
, 
Dessimoz, C.
, &amp; 
Glover, N. M.
 (2025). Quality assessment of gene repertoire annotations with OMArk. Nature Biotechnology, 43, 124–133. 10.1038/s41587-024-02147-w
</cite> [<a href="https://doi.org/10.1038/s41587-024-02147-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11738984/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38383603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Quality%20assessment%20of%20gene%20repertoire%20annotations%20with%20OMArk&amp;author=Y.%20Nevers&amp;author=A.%20Warwick%20Vesztrocy&amp;author=V.%20Rossier&amp;author=C.%E2%80%90M.%20Train&amp;author=A.%20Altenhoff&amp;volume=43&amp;publication_year=2025&amp;pages=124-133&amp;pmid=38383603&amp;doi=10.1038/s41587-024-02147-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0097">
<cite>

Nurk, S.
, 
Walenz, B. P.
, 
Rhie, A.
, 
Vollger, M. R.
, 
Logsdon, G. A.
, 
Grothe, R.
, 
Miga, K. H.
, 
Eichler, E. E.
, 
Phillippy, A. M.
, &amp; 
Koren, S.
 (2020). HiCanu: Accurate assembly of segmental duplications, satellites, and allelic variants from high‐fidelity long reads. Genome Research, 30, 1291–1305. 10.1101/gr.263566.120

</cite> [<a href="https://doi.org/10.1101/gr.263566.120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7545148/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32801147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=HiCanu:%20Accurate%20assembly%20of%20segmental%20duplications,%20satellites,%20and%20allelic%20variants%20from%20high%E2%80%90fidelity%20long%20reads&amp;author=S.%20Nurk&amp;author=B.%20P.%20Walenz&amp;author=A.%20Rhie&amp;author=M.%20R.%20Vollger&amp;author=G.%20A.%20Logsdon&amp;volume=30&amp;publication_year=2020&amp;pages=1291-1305&amp;pmid=32801147&amp;doi=10.1101/gr.263566.120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0098">
<cite>

Ochoa, C. M.
 (2004). The potatoes of South America. Peru International Potato Center.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20potatoes%20of%20South%20America&amp;author=C.%20M.%20Ochoa&amp;publication_year=2004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0099">
<cite>

Okada, K. A.
, &amp; 
Clausen, A. M.
 (1982). Natural hybridization between <em>Solanum acaule</em> Bitt. and <em>S. megistacrolobum</em> Bitt. in the province of Jujuy, Argentina. Euphytica, 31, 817–835. 10.1007/BF00039223
</cite> [<a href="https://doi.org/10.1007/BF00039223" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Natural%20hybridization%20between%20Solanum%20acaule%20Bitt.%20and%20S.%20megistacrolobum%20Bitt.%20in%20the%20province%20of%20Jujuy,%20Argentina&amp;author=K.%20A.%20Okada&amp;author=A.%20M.%20Clausen&amp;volume=31&amp;publication_year=1982&amp;pages=817-835&amp;doi=10.1007/BF00039223&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0100">
<cite>

Park, T. H.
 (2021). Complete chloroplast genome sequence of the wild diploid potato relative, <em>Solanum acaule</em>
. Mitochondrial DNA Part B, 6, 1189–1191. 10.1080/23802359.2021.1902414

</cite> [<a href="https://doi.org/10.1080/23802359.2021.1902414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8008929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33829083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mitochondrial%20DNA%20Part%20B&amp;title=Complete%20chloroplast%20genome%20sequence%20of%20the%20wild%20diploid%20potato%20relative,%20Solanum%20acaule&amp;author=T.%20H.%20Park&amp;volume=6&amp;publication_year=2021&amp;pages=1189-1191&amp;pmid=33829083&amp;doi=10.1080/23802359.2021.1902414&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0101">
<cite>

Pendinen, G.
, 
Spooner, D. M.
, 
Jiang, J.
, &amp; 
Gavrilenko, T.
 (2012). Genomic in situ hybridization reveals both auto‐ and allopolyploid origins of different North and Central American hexaploid potato (<em>Solanum</em> sect. <em>Petota</em>) species. Genome, 55, 407–415. 10.1139/g2012-027

</cite> [<a href="https://doi.org/10.1139/g2012-027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22594521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome&amp;title=Genomic%20in%20situ%20hybridization%20reveals%20both%20auto%E2%80%90%20and%20allopolyploid%20origins%20of%20different%20North%20and%20Central%20American%20hexaploid%20potato%20(Solanum%20sect.%20Petota)%20species&amp;author=G.%20Pendinen&amp;author=D.%20M.%20Spooner&amp;author=J.%20Jiang&amp;author=T.%20Gavrilenko&amp;volume=55&amp;publication_year=2012&amp;pages=407-415&amp;pmid=22594521&amp;doi=10.1139/g2012-027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0102">
<cite>

Pennycooke, J. C.
, 
Cheng, H.
, 
Roberts, S. M.
, 
Yang, Q.
, 
Rhee, S. Y.
, &amp; 
Stockinger, E. J.
 (2008). The low temperature‐responsive, <em>Solanum CBF1</em> genes maintain high identity in their upstream regions in a genomic environment undergoing gene duplications, deletions, and rearrangements. Plant Molecular Biology, 67, 483–497. 10.1007/s11103-008-9333-5

</cite> [<a href="https://doi.org/10.1007/s11103-008-9333-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18415686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Molecular%20Biology&amp;title=The%20low%20temperature%E2%80%90responsive,%20Solanum%20CBF1%20genes%20maintain%20high%20identity%20in%20their%20upstream%20regions%20in%20a%20genomic%20environment%20undergoing%20gene%20duplications,%20deletions,%20and%20rearrangements&amp;author=J.%20C.%20Pennycooke&amp;author=H.%20Cheng&amp;author=S.%20M.%20Roberts&amp;author=Q.%20Yang&amp;author=S.%20Y.%20Rhee&amp;volume=67&amp;publication_year=2008&amp;pages=483-497&amp;pmid=18415686&amp;doi=10.1007/s11103-008-9333-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0103">
<cite>

Pertea, G.
, &amp; 
Pertea, M.
 (2020). GFF utilities: GffRead and GffCompare. F1000 Research, 9, Article 304. 10.12688/f1000research.23297.2
</cite> [<a href="https://doi.org/10.12688/f1000research.23297.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7222033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32489650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=F1000%20Research&amp;title=GFF%20utilities:%20GffRead%20and%20GffCompare&amp;author=G.%20Pertea&amp;author=M.%20Pertea&amp;volume=9&amp;publication_year=2020&amp;pages=304&amp;pmid=32489650&amp;doi=10.12688/f1000research.23297.2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0104">
<cite>

Qin, M.
, 
Wu, S.
, 
Li, A.
, 
Zhao, F.
, 
Feng, H.
, 
Ding, L.
, &amp; 
Ruan, J.
 (2019). LRScaf: Improving draft genomes using long noisy reads. BMC Genomics, 20, Article 955. 10.1186/s12864-019-6337-2

</cite> [<a href="https://doi.org/10.1186/s12864-019-6337-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6902338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31818249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=LRScaf:%20Improving%20draft%20genomes%20using%20long%20noisy%20reads&amp;author=M.%20Qin&amp;author=S.%20Wu&amp;author=A.%20Li&amp;author=F.%20Zhao&amp;author=H.%20Feng&amp;volume=20&amp;publication_year=2019&amp;pages=955&amp;pmid=31818249&amp;doi=10.1186/s12864-019-6337-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0105">
<cite>

Ranallo‐Benavidez, T. R.
, 
Jaron, K. S.
, &amp; 
Schatz, M. C.
 (2020). GenomeScope 2.0 and Smudgeplot for reference‐free profiling of polyploid genomes. Nature Communications, 11, Article 1432. 10.1038/s41467-020-14998-3
</cite> [<a href="https://doi.org/10.1038/s41467-020-14998-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7080791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32188846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=GenomeScope%202.0%20and%20Smudgeplot%20for%20reference%E2%80%90free%20profiling%20of%20polyploid%20genomes&amp;author=T.%20R.%20Ranallo%E2%80%90Benavidez&amp;author=K.%20S.%20Jaron&amp;author=M.%20C.%20Schatz&amp;volume=11&amp;publication_year=2020&amp;pages=1432&amp;pmid=32188846&amp;doi=10.1038/s41467-020-14998-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0106">
<cite>

Rhie, A.
, 
Walenz, B. P.
, 
Koren, S.
, &amp; 
Phillippy, A. M.
 (2020). Merqury: Reference‐free quality, completeness, and phasing assessment for genome assemblies. Genome Biology, 21, Article 245. 10.1186/s13059-020-02134-9

</cite> [<a href="https://doi.org/10.1186/s13059-020-02134-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7488777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32928274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Merqury:%20Reference%E2%80%90free%20quality,%20completeness,%20and%20phasing%20assessment%20for%20genome%20assemblies&amp;author=A.%20Rhie&amp;author=B.%20P.%20Walenz&amp;author=S.%20Koren&amp;author=A.%20M.%20Phillippy&amp;volume=21&amp;publication_year=2020&amp;pages=245&amp;pmid=32928274&amp;doi=10.1186/s13059-020-02134-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0107">
<cite>

Rodríguez, F.
, &amp; 
Spooner, D. M.
 (2009). Phylogenetic relationships of <em>Solanum</em> Series <em>Conicibaccata</em> and related species in <em>Solanum</em> section <em>Petota</em> inferred from five conserved ortholog sequences. Systematic Botany, 34, 207–219.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Systematic%20Botany&amp;title=Phylogenetic%20relationships%20of%20Solanum%20Series%20Conicibaccata%20and%20related%20species%20in%20Solanum%20section%20Petota%20inferred%20from%20five%20conserved%20ortholog%20sequences&amp;author=F.%20Rodr%C3%ADguez&amp;author=D.%20M.%20Spooner&amp;volume=34&amp;publication_year=2009&amp;pages=207-219&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0108">
<cite>

Ross, H.
 (1986). Potato breeding—Problems and perspectives. Verlag PaulParey.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Potato%20breeding%E2%80%94Problems%20and%20perspectives&amp;author=H.%20Ross&amp;publication_year=1986&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0109">
<cite>

Samarina, L.
, 
Malyukova, L.
, 
Wang, S.
, 
Bobrovskikh, A.
, 
Doroshkov, A.
, 
Shkhalakhova, R.
, 
Manakhova, K.
, 
Koninskaya, N.
, 
Matskiv, A.
, 
Ryndin, A.
, 
Khlestkina, E.
, &amp; 
Orlov, Y.
 (2024). In vitro vs. in vivo transcriptomic approach revealed core pathways of nitrogen deficiency response in tea plant (<em>Camellia sinensis</em> (L.) Kuntze). International Journal of Molecular Sciences, 25(21), 11726. 10.3390/ijms252111726

</cite> [<a href="https://doi.org/10.3390/ijms252111726" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11547157/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39519276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=In%20vitro%20vs.%20in%20vivo%20transcriptomic%20approach%20revealed%20core%20pathways%20of%20nitrogen%20deficiency%20response%20in%20tea%20plant%20(Camellia%20sinensis%20(L.)%20Kuntze)&amp;author=L.%20Samarina&amp;author=L.%20Malyukova&amp;author=S.%20Wang&amp;author=A.%20Bobrovskikh&amp;author=A.%20Doroshkov&amp;volume=25&amp;issue=21&amp;publication_year=2024&amp;pages=11726&amp;pmid=39519276&amp;doi=10.3390/ijms252111726&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0110">
<cite>

Schmiediche, P. E.
, 
Hawkes, J. G.
, &amp; 
Ochoa, C. M.
 (1980). Breeding of the cultivated potato species <em>Solanum × juzepczukii</em> Buk. and <em>S. × curtilobum</em> Juz. et Buk. I. A study of the natural variation of <em>Solanum × juzepczukii</em>, <em>S. × curtilobum</em> and their wild progenitor, <em>S. acaule</em>. Bitt. Euphytica, 29, 685–704. 10.1007/BF00023216
</cite> [<a href="https://doi.org/10.1007/BF00023216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Breeding%20of%20the%20cultivated%20potato%20species%20Solanum%20%C3%97%20juzepczukii%20Buk.%20and%20S.%20%C3%97%20curtilobum%20Juz.%20et%20Buk.%20I.%20A%20study%20of%20the%20natural%20variation%20of%20Solanum%20%C3%97%20juzepczukii,%20S.%20%C3%97%20curtilobum%20and%20their%20wild%20progenitor,%20S.%20acaule.%20Bitt&amp;author=P.%20E.%20Schmiediche&amp;author=J.%20G.%20Hawkes&amp;author=C.%20M.%20Ochoa&amp;volume=29&amp;publication_year=1980&amp;pages=685-704&amp;doi=10.1007/BF00023216&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0111">
<cite>

Schmiediche, P. E.
, 
Hawkes, J. G.
, &amp; 
Ochoa, C. M.
 (1982). 
<em>Solanum × juzepczukii</em> Buk. and <em>S. × curtilobum</em> Juz. et Buk. II. The resynthesis of <em>Solanum × juzepczukii</em> and <em>S. × curtilobum</em>
. Euphytica, 31, 695–708. 10.1007/BF00039208
</cite> [<a href="https://doi.org/10.1007/BF00039208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Solanum%20%C3%97%20juzepczukii%20Buk.%20and%20S.%20%C3%97%20curtilobum%20Juz.%20et%20Buk.%20II.%20The%20resynthesis%20of%20Solanum%20%C3%97%20juzepczukii%20and%20S.%20%C3%97%20curtilobum&amp;author=P.%20E.%20Schmiediche&amp;author=J.%20G.%20Hawkes&amp;author=C.%20M.%20Ochoa&amp;volume=31&amp;publication_year=1982&amp;pages=695-708&amp;doi=10.1007/BF00039208&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0112">
<cite>

Schnable, J. C.
, 
Springer, N. M.
, &amp; 
Freeling, M.
 (2011). Differentiation of the maize subgenomes by genome dominance and both ancient and ongoing gene loss. PNAS, 108, 4069–4074. 10.1073/pnas.1101368108

</cite> [<a href="https://doi.org/10.1073/pnas.1101368108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3053962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21368132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Differentiation%20of%20the%20maize%20subgenomes%20by%20genome%20dominance%20and%20both%20ancient%20and%20ongoing%20gene%20loss&amp;author=J.%20C.%20Schnable&amp;author=N.%20M.%20Springer&amp;author=M.%20Freeling&amp;volume=108&amp;publication_year=2011&amp;pages=4069-4074&amp;pmid=21368132&amp;doi=10.1073/pnas.1101368108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0113">
<cite>

Servant, N.
, 
Varoquaux, N.
, 
Lajoie, B. R.
, 
Viara, E.
, 
Chen, C.‐J.
, 
Vert, J.‐P.
, 
Heard, E.
, 
Dekker, J.
, &amp; 
Barillot, E.
 (2015). HiC‐Pro: An optimized and flexible pipeline for Hi‐C processing. Genome Biology, 16, Article 259. 10.1186/s13059-015-0831-x

</cite> [<a href="https://doi.org/10.1186/s13059-015-0831-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4665391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26619908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=HiC%E2%80%90Pro:%20An%20optimized%20and%20flexible%20pipeline%20for%20Hi%E2%80%90C%20processing&amp;author=N.%20Servant&amp;author=N.%20Varoquaux&amp;author=B.%20R.%20Lajoie&amp;author=E.%20Viara&amp;author=C.%E2%80%90J.%20Chen&amp;volume=16&amp;publication_year=2015&amp;pages=259&amp;pmid=26619908&amp;doi=10.1186/s13059-015-0831-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0114">
<cite>

Simão, F. A.
, 
Waterhouse, R. M.
, 
Ioannidis, P.
, 
Kriventseva, E. V.
, &amp; 
Zdobnov, E. M.
 (2015). BUSCO: Assessing genome assembly and annotation completeness with single‐copy orthologs. Bioinformatics, 31, 3210–3212. 10.1093/bioinformatics/btv351

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26059717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BUSCO:%20Assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single%E2%80%90copy%20orthologs&amp;author=F.%20A.%20Sim%C3%A3o&amp;author=R.%20M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.%20V.%20Kriventseva&amp;author=E.%20M.%20Zdobnov&amp;volume=31&amp;publication_year=2015&amp;pages=3210-3212&amp;pmid=26059717&amp;doi=10.1093/bioinformatics/btv351&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0115"><cite>

Smit, A. F. A.
, 
Hubley, R.
, &amp; 
Green, P.
 (2015). 
<em>RepeatMasker Open‐4.0</em>. 2013–2015. RepeatMasker. <a href="http://www.repeatmasker.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.repeatmasker.org</a>
</cite></li>
<li id="tpg270095-bib-0116">
<cite>

Sommer, M. J.
, 
Zimin, A. V.
, &amp; 
Salzberg, S. L.
 (2025). PSAURON: A tool for assessing protein annotation across a broad range of species. Nucleic Acids Research Genomics and Bioinformatics, 7, lqae189. 10.1093/nargab/lqae189
</cite> [<a href="https://doi.org/10.1093/nargab/lqae189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11704789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39781514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research%20Genomics%20and%20Bioinformatics&amp;title=PSAURON:%20A%20tool%20for%20assessing%20protein%20annotation%20across%20a%20broad%20range%20of%20species&amp;author=M.%20J.%20Sommer&amp;author=A.%20V.%20Zimin&amp;author=S.%20L.%20Salzberg&amp;volume=7&amp;publication_year=2025&amp;pages=lqae189&amp;pmid=39781514&amp;doi=10.1093/nargab/lqae189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0117">
<cite>

Song, Q.
, 
Wang, X.
, 
Li, J.
, 
Chen, T.
, 
Liu, Y.
, &amp; 
Yang, X.
 (2021). 
<em>CBF1</em> and <em>CBF4</em> in <em>Solanum tuberosum</em> L. differ in their effect on low‐temperature tolerance and development. Environmental and Experimental Botany, 185, 104416. 10.1016/j.envexpbot.2021.104416
</cite> [<a href="https://doi.org/10.1016/j.envexpbot.2021.104416" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20and%20Experimental%20Botany&amp;title=CBF1%20and%20CBF4%20in%20Solanum%20tuberosum%20L.%20differ%20in%20their%20effect%20on%20low%E2%80%90temperature%20tolerance%20and%20development&amp;author=Q.%20Song&amp;author=X.%20Wang&amp;author=J.%20Li&amp;author=T.%20Chen&amp;author=Y.%20Liu&amp;volume=185&amp;publication_year=2021&amp;pages=104416&amp;doi=10.1016/j.envexpbot.2021.104416&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0118">
<cite>

Spooner, D. M.
, 
Ghislain, M.
, 
Simon, R.
, 
Jansky, S. H.
, &amp; 
Gavrilenko, T.
 (2014). Systematics, diversity, genetics, and evolution of wild and cultivated potatoes. The Botanical Review, 80, 283–383. 10.1007/s12229-014-9146-y
</cite> [<a href="https://doi.org/10.1007/s12229-014-9146-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Botanical%20Review&amp;title=Systematics,%20diversity,%20genetics,%20and%20evolution%20of%20wild%20and%20cultivated%20potatoes&amp;author=D.%20M.%20Spooner&amp;author=M.%20Ghislain&amp;author=R.%20Simon&amp;author=S.%20H.%20Jansky&amp;author=T.%20Gavrilenko&amp;volume=80&amp;publication_year=2014&amp;pages=283-383&amp;doi=10.1007/s12229-014-9146-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0119">
<cite>

Spooner, D. M.
, 
Rodríguez, F.
, 
Polgár, Z.
, 
Ballard, H. E.
, &amp; 
Jansky, S. H.
 (2008). Genomic origins of potato polyploids: GBSSI gene sequencing data. Crop Science, 48, S27–S36.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genomic%20origins%20of%20potato%20polyploids:%20GBSSI%20gene%20sequencing%20data&amp;author=D.%20M.%20Spooner&amp;author=F.%20Rodr%C3%ADguez&amp;author=Z.%20Polg%C3%A1r&amp;author=H.%20E.%20Ballard&amp;author=S.%20H.%20Jansky&amp;volume=48&amp;publication_year=2008&amp;pages=S27-S36&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0120">
<cite>

Spooner, D. M.
, 
Ruess, H.
, 
Arbizu, C. I.
, 
Rodríguez, F.
, &amp; 
Solís‐Lemus, C.
 (2018). Greatly reduced phylogenetic structure in the cultivated potato clade (<em>Solanum</em> section <em>Petota</em> pro parte). American Journal of Botany, 105, 60–70. 10.1002/ajb2.1008

</cite> [<a href="https://doi.org/10.1002/ajb2.1008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29532930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Botany&amp;title=Greatly%20reduced%20phylogenetic%20structure%20in%20the%20cultivated%20potato%20clade%20(Solanum%20section%20Petota%20pro%20parte)&amp;author=D.%20M.%20Spooner&amp;author=H.%20Ruess&amp;author=C.%20I.%20Arbizu&amp;author=F.%20Rodr%C3%ADguez&amp;author=C.%20Sol%C3%ADs%E2%80%90Lemus&amp;volume=105&amp;publication_year=2018&amp;pages=60-70&amp;pmid=29532930&amp;doi=10.1002/ajb2.1008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0121">
<cite>

Stanke, M.
, 
Diekhans, M.
, 
Baertsch, R.
, &amp; 
Haussler, D.
 (2008). Using native and syntenically mapped cDNA alignments to improve <em>de novo</em> gene finding. Bioinformatics, 24, 637–644. 10.1093/bioinformatics/btn013

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btn013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18218656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Using%20native%20and%20syntenically%20mapped%20cDNA%20alignments%20to%20improve%20de%20novo%20gene%20finding&amp;author=M.%20Stanke&amp;author=M.%20Diekhans&amp;author=R.%20Baertsch&amp;author=D.%20Haussler&amp;volume=24&amp;publication_year=2008&amp;pages=637-644&amp;pmid=18218656&amp;doi=10.1093/bioinformatics/btn013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0122">
<cite>

Stanke, M.
, 
Schöffmann, O.
, 
Morgenstern, B.
, &amp; 
Waack, S.
 (2006). Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources. BMC Bioinformatics, 7, Article 62. 10.1186/1471-2105-7-62

</cite> [<a href="https://doi.org/10.1186/1471-2105-7-62" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1409804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16469098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Gene%20prediction%20in%20eukaryotes%20with%20a%20generalized%20hidden%20Markov%20model%20that%20uses%20hints%20from%20external%20sources&amp;author=M.%20Stanke&amp;author=O.%20Sch%C3%B6ffmann&amp;author=B.%20Morgenstern&amp;author=S.%20Waack&amp;volume=7&amp;publication_year=2006&amp;pages=62&amp;pmid=16469098&amp;doi=10.1186/1471-2105-7-62&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0123">
<cite>

Steige, K. A.
, &amp; 
Slotte, T.
 (2016). Genomic legacies of the progenitors and the evolutionary consequences of allopolyploidy. Current Opinion in Plant Biology, 30, 88–93. 10.1016/j.pbi.2016.02.006

</cite> [<a href="https://doi.org/10.1016/j.pbi.2016.02.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26943938/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Plant%20Biology&amp;title=Genomic%20legacies%20of%20the%20progenitors%20and%20the%20evolutionary%20consequences%20of%20allopolyploidy&amp;author=K.%20A.%20Steige&amp;author=T.%20Slotte&amp;volume=30&amp;publication_year=2016&amp;pages=88-93&amp;pmid=26943938&amp;doi=10.1016/j.pbi.2016.02.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0124">
<cite>

Sukumaran, N. P.
, &amp; 
Weiser, C. J.
 (1972). Freezing injury in potato leaves. Plant Physiology, 50, 564–567. 10.1104/pp.50.5.564

</cite> [<a href="https://doi.org/10.1104/pp.50.5.564" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC366190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16658217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Freezing%20injury%20in%20potato%20leaves&amp;author=N.%20P.%20Sukumaran&amp;author=C.%20J.%20Weiser&amp;volume=50&amp;publication_year=1972&amp;pages=564-567&amp;pmid=16658217&amp;doi=10.1104/pp.50.5.564&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0125">
<cite>

Sun, H.
, 
Jiao, W. B.
, 
Krause, K.
, 
Campoy, J. A.
, 
Goel, M.
, 
Folz‐Donahue, K.
, 
Kukat, C.
, 
Huettel, B.
, &amp; 
Schneeberger, K.
 (2022). Chromosome‐scale and haplotype‐resolved genome assembly of a tetraploid potato cultivar. Nature Genetics, 54, 342–348. 10.1038/s41588-022-01015-0

</cite> [<a href="https://doi.org/10.1038/s41588-022-01015-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8920897/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35241824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Chromosome%E2%80%90scale%20and%20haplotype%E2%80%90resolved%20genome%20assembly%20of%20a%20tetraploid%20potato%20cultivar&amp;author=H.%20Sun&amp;author=W.%20B.%20Jiao&amp;author=K.%20Krause&amp;author=J.%20A.%20Campoy&amp;author=M.%20Goel&amp;volume=54&amp;publication_year=2022&amp;pages=342-348&amp;pmid=35241824&amp;doi=10.1038/s41588-022-01015-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0126">
<cite>

Tai, H.
, 
Shannon, L.
, &amp; 
Strömvik, M.
 (2025). Polyploidy in potatoes: Challenges and possibilities for climate resilience. Trends in Genetics. 10.1016/j.tig.2025.03.006
</cite> [<a href="https://doi.org/10.1016/j.tig.2025.03.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40268598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Genetics&amp;title=Polyploidy%20in%20potatoes:%20Challenges%20and%20possibilities%20for%20climate%20resilience&amp;author=H.%20Tai&amp;author=L.%20Shannon&amp;author=M.%20Str%C3%B6mvik&amp;publication_year=2025&amp;pmid=40268598&amp;doi=10.1016/j.tig.2025.03.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0127">
<cite>

Tang, D.
, 
Jia, Y.
, 
Zhang, J.
, 
Li, H.
, 
Cheng, L.
, 
Wang, P.
, 
Bao, Z.
, 
Liu, Z.
, 
Feng, S.
, 
Zhu, X.
, 
Li, D.
, 
Zhu, G.
, 
Wang, H.
, 
Zhou, Y.
, 
Zhou, Y.
, 
Bryan, G. J.
, 
Buell, C. R.
, 
Zhang, C.
, &amp; 
Huang, S.
 (2022). Genome evolution and diversity of wild and cultivated potatoes. Nature, 606, 535–541. 10.1038/s41586-022-04822-x

</cite> [<a href="https://doi.org/10.1038/s41586-022-04822-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9200641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35676481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome%20evolution%20and%20diversity%20of%20wild%20and%20cultivated%20potatoes&amp;author=D.%20Tang&amp;author=Y.%20Jia&amp;author=J.%20Zhang&amp;author=H.%20Li&amp;author=L.%20Cheng&amp;volume=606&amp;publication_year=2022&amp;pages=535-541&amp;pmid=35676481&amp;doi=10.1038/s41586-022-04822-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0128">
<cite>

Tang, H.
, 
Woodhouse, M. R.
, 
Cheng, F.
, 
Schnable, J. C.
, 
Pedersen, B. S.
, 
Conant, G.
, 
Wang, X.
, 
Freeling, M.
, &amp; 
Pires, J. C.
 (2012). Altered patterns of fractionation and exon deletions in <em>Brassica rapa</em> support a two‐step model of paleohexaploidy. Genetics, 190, 1563–1574. 10.1534/genetics.111.137349

</cite> [<a href="https://doi.org/10.1534/genetics.111.137349" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3316664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22308264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Altered%20patterns%20of%20fractionation%20and%20exon%20deletions%20in%20Brassica%20rapa%20support%20a%20two%E2%80%90step%20model%20of%20paleohexaploidy&amp;author=H.%20Tang&amp;author=M.%20R.%20Woodhouse&amp;author=F.%20Cheng&amp;author=J.%20C.%20Schnable&amp;author=B.%20S.%20Pedersen&amp;volume=190&amp;publication_year=2012&amp;pages=1563-1574&amp;pmid=22308264&amp;doi=10.1534/genetics.111.137349&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0129">
<cite>

VanBuren, R.
, 
Man Wai, C.
, 
Wang, X.
, 
Pardo, J.
, 
Yocca, A. E.
, 
Wang, H.
, 
Chaluvadi, S. R.
, 
Han, G.
, 
Bryant, D.
, 
Edger, P. P.
, 
Messing, J.
, 
Sorrells, M. E.
, 
Mockler, T. C.
, 
Bennetzen, J. L.
, &amp; 
Michael, T. P.
 (2020). Exceptional subgenome stability and functional divergence in the allotetraploid Ethiopian cereal teff. Nature Communications, 11(1), Article 884. 10.1038/s41467-020-14724-z
</cite> [<a href="https://doi.org/10.1038/s41467-020-14724-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7021729/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32060277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Exceptional%20subgenome%20stability%20and%20functional%20divergence%20in%20the%20allotetraploid%20Ethiopian%20cereal%20teff&amp;author=R.%20VanBuren&amp;author=C.%20Man%20Wai&amp;author=X.%20Wang&amp;author=J.%20Pardo&amp;author=A.%20E.%20Yocca&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=884&amp;pmid=32060277&amp;doi=10.1038/s41467-020-14724-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0130">
<cite>

Vega, S. E.
, &amp; 
Bamberg, J. B.
 (1995). Screening the US potato collection for frost hardiness. American Potato Journal, 72, 13–21. 10.1007/BF02874375
</cite> [<a href="https://doi.org/10.1007/BF02874375" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Potato%20Journal&amp;title=Screening%20the%20US%20potato%20collection%20for%20frost%20hardiness&amp;author=S.%20E.%20Vega&amp;author=J.%20B.%20Bamberg&amp;volume=72&amp;publication_year=1995&amp;pages=13-21&amp;doi=10.1007/BF02874375&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0131">
<cite>

Wang, F.
, 
Xia, Z.
, 
Zou, M.
, 
Zhao, L.
, 
Jiang, S.
, 
Zhou, Y.
, 
Zhang, C.
, 
Ma, Y.
, 
Bao, Y.
, 
Sun, H.
, 
Wang, W.
, &amp; 
Wang, J.
 (2022). The autotetraploid potato genome provides insights into highly heterozygous species. Plant Biotechnology Journal, 20, 1996–2005. 10.1111/pbi.13883

</cite> [<a href="https://doi.org/10.1111/pbi.13883" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9491450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35767385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=The%20autotetraploid%20potato%20genome%20provides%20insights%20into%20highly%20heterozygous%20species&amp;author=F.%20Wang&amp;author=Z.%20Xia&amp;author=M.%20Zou&amp;author=L.%20Zhao&amp;author=S.%20Jiang&amp;volume=20&amp;publication_year=2022&amp;pages=1996-2005&amp;pmid=35767385&amp;doi=10.1111/pbi.13883&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0133">
<cite>

Wang, Y.
, 
Tang, H.
, 
Debarry, J. D.
, 
Tan, X.
, 
Li, J.
, 
Wang, X.
, 
Lee, T.‐H.
, 
Jin, H.
, 
Marler, B.
, 
Guo, H.
, 
Kissinger, J. C.
, &amp; 
Paterson, A. H.
 (2012). MCScanX: A toolkit for detection and evolutionary analysis of gene synteny and collinearity. Nucleic Acids Research, 40, e49. 10.1093/nar/gkr1293

</cite> [<a href="https://doi.org/10.1093/nar/gkr1293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3326336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22217600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=MCScanX:%20A%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20collinearity&amp;author=Y.%20Wang&amp;author=H.%20Tang&amp;author=J.%20D.%20Debarry&amp;author=X.%20Tan&amp;author=J.%20Li&amp;volume=40&amp;publication_year=2012&amp;pages=e49&amp;pmid=22217600&amp;doi=10.1093/nar/gkr1293&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0134">
<cite>

Wei, P.
, 
Yu, X.‐L.
, 
Yang, Y.‐J.
, 
Chen, Z.‐Y.
, 
Zhao, S.‐Q.
, 
Li, X.‐Z.
, 
Zhang, W.‐C.
, 
Liu, C.‐L.
, 
Li, X.‐Y.
, &amp; 
Liu, X.
 (2022). Biased gene expression reveals the contribution of subgenome to altitude adaptation in allopolyploid Isoetes sinensis. Ecology and Evolution, 12, e9677. 10.1002/ece3.9677

</cite> [<a href="https://doi.org/10.1002/ece3.9677" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9797765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36619709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20and%20Evolution&amp;title=Biased%20gene%20expression%20reveals%20the%20contribution%20of%20subgenome%20to%20altitude%20adaptation%20in%20allopolyploid%20Isoetes%20sinensis&amp;author=P.%20Wei&amp;author=X.%E2%80%90L.%20Yu&amp;author=Y.%E2%80%90J.%20Yang&amp;author=Z.%E2%80%90Y.%20Chen&amp;author=S.%E2%80%90Q.%20Zhao&amp;volume=12&amp;publication_year=2022&amp;pages=e9677&amp;pmid=36619709&amp;doi=10.1002/ece3.9677&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0135">
<cite>

Wolff, J.
, 
Rabbani, L.
, 
Gilsbach, R.
, 
Richard, G.
, 
Manke, T.
, 
Backofen, R.
, &amp; 
Grüning, B. A.
 (2020). Galaxy HiCExplorer 3: A web server for reproducible Hi‐C, capture Hi‐C and single‐cell Hi‐C data analysis, quality control and visualization. Nucleic Acids Research, 48, W177–W184. 10.1093/nar/gkaa220

</cite> [<a href="https://doi.org/10.1093/nar/gkaa220" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7319437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32301980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Galaxy%20HiCExplorer%203:%20A%20web%20server%20for%20reproducible%20Hi%E2%80%90C,%20capture%20Hi%E2%80%90C%20and%20single%E2%80%90cell%20Hi%E2%80%90C%20data%20analysis,%20quality%20control%20and%20visualization&amp;author=J.%20Wolff&amp;author=L.%20Rabbani&amp;author=R.%20Gilsbach&amp;author=G.%20Richard&amp;author=T.%20Manke&amp;volume=48&amp;publication_year=2020&amp;pages=W177-W184&amp;pmid=32301980&amp;doi=10.1093/nar/gkaa220&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0136">
<cite>

Wood, D. E.
, 
Lu, J.
, &amp; 
Langmead, B.
 (2019). Improved metagenomic analysis with Kraken 2. Genome Biology, 20, Article 257. 10.1186/s13059-019-1891-0

</cite> [<a href="https://doi.org/10.1186/s13059-019-1891-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6883579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31779668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Improved%20metagenomic%20analysis%20with%20Kraken%202&amp;author=D.%20E.%20Wood&amp;author=J.%20Lu&amp;author=B.%20Langmead&amp;volume=20&amp;publication_year=2019&amp;pages=257&amp;pmid=31779668&amp;doi=10.1186/s13059-019-1891-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0137">
<cite>

Wu, T. D.
, &amp; 
Watanabe, C. K.
 (2005). GMAP: A genomic mapping and alignment program for mRNA and EST sequences. Bioinformatics, 21, 1859–1875. 10.1093/bioinformatics/bti310

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bti310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15728110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GMAP:%20A%20genomic%20mapping%20and%20alignment%20program%20for%20mRNA%20and%20EST%20sequences&amp;author=T.%20D.%20Wu&amp;author=C.%20K.%20Watanabe&amp;volume=21&amp;publication_year=2005&amp;pages=1859-1875&amp;pmid=15728110&amp;doi=10.1093/bioinformatics/bti310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0138">
<cite>

Xing, L.
, 
Wang, M.
, 
He, Q.
, 
Zhang, H.
, 
Liang, H.
, 
Zhou, Q.
, 
Liu, Y.
, 
Liu, Z.
, 
Wang, Y.
, 
Du, C.
, 
Xiao, Y.
, 
Liu, J.
, 
Li, W.
, 
Liu, G.
, &amp; 
Du, H.
 (2023). Differential subgenome expression underlies biomass accumulation in allotetraploid <em>Pennisetum giganteum</em>
. BMC Biology, 21, Article 161. 10.1186/s12915-023-01643-w

</cite> [<a href="https://doi.org/10.1186/s12915-023-01643-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10362693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37480118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Biology&amp;title=Differential%20subgenome%20expression%20underlies%20biomass%20accumulation%20in%20allotetraploid%20Pennisetum%20giganteum&amp;author=L.%20Xing&amp;author=M.%20Wang&amp;author=Q.%20He&amp;author=H.%20Zhang&amp;author=H.%20Liang&amp;volume=21&amp;publication_year=2023&amp;pages=161&amp;pmid=37480118&amp;doi=10.1186/s12915-023-01643-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0149">
<cite>

Xu, D.
, 
Yang, J.
, 
Wen, H.
, 
Feng, W.
, 
Zhang, X.
, 
Hui, X.
, 
Yue, J.
, 
Xu, Y.
, 
Chen, F.
, &amp; 
Pan, W.
 (2024). CentIER: Accurate centromere identification for plant genomes. Plant Communications, 5, 101046. 10.1016/j.xplc.2024.101046

</cite> [<a href="https://doi.org/10.1016/j.xplc.2024.101046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11573919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39118326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Communications&amp;title=CentIER:%20Accurate%20centromere%20identification%20for%20plant%20genomes&amp;author=D.%20Xu&amp;author=J.%20Yang&amp;author=H.%20Wen&amp;author=W.%20Feng&amp;author=X.%20Zhang&amp;volume=5&amp;publication_year=2024&amp;pages=101046&amp;pmid=39118326&amp;doi=10.1016/j.xplc.2024.101046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0139">
<cite>

Xu, M.
, 
Guo, L.
, 
Gu, S.
, 
Wang, O.
, 
Zhang, R.
, 
Peters, B. A.
, 
Fan, G.
, 
Liu, X.
, 
Xu, X.
, 
Deng, L.
, &amp; 
Zhang, Y.
 (2020). TGS‐GapCloser: A fast and accurate gap closer for large genomes with low coverage of error‐prone long reads. GigaScience, 9, giaa094. 10.1093/gigascience/giaa094

</cite> [<a href="https://doi.org/10.1093/gigascience/giaa094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7476103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32893860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GigaScience&amp;title=TGS%E2%80%90GapCloser:%20A%20fast%20and%20accurate%20gap%20closer%20for%20large%20genomes%20with%20low%20coverage%20of%20error%E2%80%90prone%20long%20reads&amp;author=M.%20Xu&amp;author=L.%20Guo&amp;author=S.%20Gu&amp;author=O.%20Wang&amp;author=R.%20Zhang&amp;volume=9&amp;publication_year=2020&amp;pages=giaa094&amp;pmid=32893860&amp;doi=10.1093/gigascience/giaa094&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0140">
<cite>

Yan, L.
, 
Zhu, Z.
, 
Wang, P.
, 
Fu, C.
, 
Guan, X.
, 
Kear, P.
, 
Zhang, C.
, &amp; 
Zhu, G.
 (2023). Comparative analysis of 343 plastid genomes of <em>Solanum</em> section <em>Petota</em>: Insight into diversity, phylogeny, and species discrimination. Journal of Systematics and Evolution, 61, 599–612. 10.1111/jse.12898
</cite> [<a href="https://doi.org/10.1111/jse.12898" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Systematics%20and%20Evolution&amp;title=Comparative%20analysis%20of%20343%20plastid%20genomes%20of%20Solanum%20section%20Petota:%20Insight%20into%20diversity,%20phylogeny,%20and%20species%20discrimination&amp;author=L.%20Yan&amp;author=Z.%20Zhu&amp;author=P.%20Wang&amp;author=C.%20Fu&amp;author=X.%20Guan&amp;volume=61&amp;publication_year=2023&amp;pages=599-612&amp;doi=10.1111/jse.12898&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0141">
<cite>

Yang, J.
, 
Liu, D.
, 
Wang, X.
, 
Ji, C.
, 
Cheng, F.
, 
Liu, B.
, 
Hu, Z.
, 
Chen, S.
, 
Pental, D.
, 
Ju, Y.
, 
Yao, P.
, 
Li, X.
, 
Xie, K.
, 
Zhang, J.
, 
Wang, J.
, 
Liu, F.
, 
Ma, W.
, 
Shopan, J.
, 
Zheng, H.
, … 
Zhang, M.
 (2016). The genome sequence of allopolyploid <em>Brassica juncea</em> and analysis of differential homoeolog gene expression influencing selection. Nature Genetics, 48(10), 1225–1232. 10.1038/ng.3657

</cite> [<a href="https://doi.org/10.1038/ng.3657" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27595476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=The%20genome%20sequence%20of%20allopolyploid%20Brassica%20juncea%20and%20analysis%20of%20differential%20homoeolog%20gene%20expression%20influencing%20selection&amp;author=J.%20Yang&amp;author=D.%20Liu&amp;author=X.%20Wang&amp;author=C.%20Ji&amp;author=F.%20Cheng&amp;volume=48&amp;issue=10&amp;publication_year=2016&amp;pages=1225-1232&amp;pmid=27595476&amp;doi=10.1038/ng.3657&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0142">
<cite>

Yoo, M. J.
, 
Szadkowski, E.
, &amp; 
Wendel, J. F.
 (2012). Homoeolog expression bias and expression level dominance in allopolyploid cotton. Heredity, 110(2), 171–180. 10.1038/hdy.2012.94

</cite> [<a href="https://doi.org/10.1038/hdy.2012.94" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3554454/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23169565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Homoeolog%20expression%20bias%20and%20expression%20level%20dominance%20in%20allopolyploid%20cotton&amp;author=M.%20J.%20Yoo&amp;author=E.%20Szadkowski&amp;author=J.%20F.%20Wendel&amp;volume=110&amp;issue=2&amp;publication_year=2012&amp;pages=171-180&amp;pmid=23169565&amp;doi=10.1038/hdy.2012.94&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0143">
<cite>

Young, M. D.
, 
Wakefield, M. J.
, 
Smyth, G. K.
, &amp; 
Oshlack, A.
 (2010). Gene ontology analysis for RNA‐seq: Accounting for selection bias. Genome Biology, 11, Article R14. 10.1186/gb-2010-11-2-r14

</cite> [<a href="https://doi.org/10.1186/gb-2010-11-2-r14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2872874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20132535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Gene%20ontology%20analysis%20for%20RNA%E2%80%90seq:%20Accounting%20for%20selection%20bias&amp;author=M.%20D.%20Young&amp;author=M.%20J.%20Wakefield&amp;author=G.%20K.%20Smyth&amp;author=A.%20Oshlack&amp;volume=11&amp;publication_year=2010&amp;pages=R14&amp;pmid=20132535&amp;doi=10.1186/gb-2010-11-2-r14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0144">
<cite>

Zhang, C.
, 
Rabiee, M.
, 
Sayyari, E.
, &amp; 
Mirarab, S.
 (2018). ASTRAL‐III: Polynomial time species tree reconstruction from partially resolved gene trees. BMC Bioinformatics, 19, Article 153. 10.1186/s12859-018-2129-y

</cite> [<a href="https://doi.org/10.1186/s12859-018-2129-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5998893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29745866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=ASTRAL%E2%80%90III:%20Polynomial%20time%20species%20tree%20reconstruction%20from%20partially%20resolved%20gene%20trees&amp;author=C.%20Zhang&amp;author=M.%20Rabiee&amp;author=E.%20Sayyari&amp;author=S.%20Mirarab&amp;volume=19&amp;publication_year=2018&amp;pages=153&amp;pmid=29745866&amp;doi=10.1186/s12859-018-2129-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0150">
<cite>

Zhang, X.
, 
Fowler, S. G.
, 
Cheng, H.
, 
Lou, Y.
, 
Rhee, S. Y.
, 
Stockinger, E. J.
, &amp; 
Thomashow, M. F.
 (2004). Freezing‐sensitive tomato has a functional CBF cold response pathway, but a CBF regulon that differs from that of freezing‐tolerant <em>Arabidopsis</em>
. The Plant Journal, 39(6), 905–919. 10.1111/j.1365-313x.2004.02176.x

</cite> [<a href="https://doi.org/10.1111/j.1365-313x.2004.02176.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15341633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=Freezing%E2%80%90sensitive%20tomato%20has%20a%20functional%20CBF%20cold%20response%20pathway,%20but%20a%20CBF%20regulon%20that%20differs%20from%20that%20of%20freezing%E2%80%90tolerant%20Arabidopsis&amp;author=X.%20Zhang&amp;author=S.%20G.%20Fowler&amp;author=H.%20Cheng&amp;author=Y.%20Lou&amp;author=S.%20Y.%20Rhee&amp;volume=39&amp;issue=6&amp;publication_year=2004&amp;pages=905-919&amp;pmid=15341633&amp;doi=10.1111/j.1365-313x.2004.02176.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0145">
<cite>

Zhang, X.
, 
Zhang, S.
, 
Zhao, Q.
, 
Ming, R.
, &amp; 
Tang, H.
 (2019). Assembly of allele‐aware, chromosomal‐scale autopolyploid genomes based on Hi‐C data. Nature Plants, 5, 833–845. 10.1038/s41477-019-0487-8

</cite> [<a href="https://doi.org/10.1038/s41477-019-0487-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31383970/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Plants&amp;title=Assembly%20of%20allele%E2%80%90aware,%20chromosomal%E2%80%90scale%20autopolyploid%20genomes%20based%20on%20Hi%E2%80%90C%20data&amp;author=X.%20Zhang&amp;author=S.%20Zhang&amp;author=Q.%20Zhao&amp;author=R.%20Ming&amp;author=H.%20Tang&amp;volume=5&amp;publication_year=2019&amp;pages=833-845&amp;pmid=31383970&amp;doi=10.1038/s41477-019-0487-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270095-bib-0146">
<cite>

Zhou, C.
, 
Mccarthy, S. A.
, &amp; 
Durbin, R.
 (2023). YaHS: Yet another Hi‐C scaffolding tool. Bioinformatics, 39, btac808. 10.1093/bioinformatics/btac808

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btac808" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9848053/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36525368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=YaHS:%20Yet%20another%20Hi%E2%80%90C%20scaffolding%20tool&amp;author=C.%20Zhou&amp;author=S.%20A.%20Mccarthy&amp;author=R.%20Durbin&amp;volume=39&amp;publication_year=2023&amp;pages=btac808&amp;pmid=36525368&amp;doi=10.1093/bioinformatics/btac808&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Supplementary Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12368328/bin/TPG2-18-e70095-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70095-s001.pdf</a><sup> (11.2MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All software with their specific versions used for data processing are clearly described in the methods section. If no specific variable or parameters are mentioned for a software, the default parameters were used. All sequence data in this study are deposited in NCBI and available under BioProject PRJNA779368 unless otherwise specified. The AA9 genome sequencing data are available in NCBI under BioSample SAMN23440999, SRR17078425 (Illumina reads), SRR20871493 (Nanopore reads), SRR20871494 (PacBio), and SRR26082557 (HiC). The assemblies are available in NCBI under PRJNA1023364 accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/JAYKPG000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JAYKPG000000000</a>, “Phased genome assembly of <em>Solanum acaule</em> (AA9)—Subgenome 1,” accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/JAYKPF000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JAYKPF000000000</a>, and PRJNA1023365 “Phased genome assembly of <em>Solanum acaule</em> (AA9)—Subgenome 2.” The annotations are available in DRYAD under <a href="https://doi.org/10.5061/dryad.vdncjsz6q" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5061/dryad.vdncjsz6q</a>. The plastome is available under accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/OM638050" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OM638050</a>. The RNA‐Seq data are available under BioSamples SAMN25867789, SAMN25867802, SAMN25867820, and SAMN25867834, and the short reads archive accession numbers SRR17993274‐SRR17993285 (Illumina RNA‐Seq). The transcriptome expression data are available in NCBI GEO under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE268611" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE268611</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70095"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70095.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368328/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368328/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368328%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368328/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368328/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368328/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40836401/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368328/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40836401/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368328/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368328/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Zrwbq05PAwYWynJdKQEGWvX2aG1QsOtykAICdUNbR2Qv6DRXgvsI9FSr1kIWyJcm">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
