
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Production and purification of methionine gamma-lyase from Iranian soil mulds: investigation of physicochemical properties and anticancer effects - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5336558AF2C4430536550064187DC7.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmccanc">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366409/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Cancer">
<meta name="citation_title" content="Production and purification of methionine gamma-lyase from Iranian soil mulds: investigation of physicochemical properties and anticancer effects">
<meta name="citation_author" content="Matin Nasirian">
<meta name="citation_author_institution" content="Genetics Dept, Faculty of Basic Science, Shahrekord University, Shahrekord, Iran">
<meta name="citation_author" content="Ali Arab-SadeghAbadi">
<meta name="citation_author_institution" content="Genetics Dept, Faculty of Basic Science, Shahrekord University, Shahrekord, Iran">
<meta name="citation_author" content="Mohsen Mobini-Dehkordi">
<meta name="citation_author_institution" content="Genetics Dept, Faculty of Basic Science, Shahrekord University, Shahrekord, Iran">
<meta name="citation_author" content="Sadegh Farhadian">
<meta name="citation_author_institution" content="Biology Dept, Faculty of Basic Science, Shahrekord University, Shahrekord, Iran">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="1339">
<meta name="citation_doi" content="10.1186/s12885-025-14754-0">
<meta name="citation_pmid" content="40830757">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366409/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366409/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366409/pdf/12885_2025_Article_14754.pdf">
<meta name="description" content="Cancer’s rising global incidence necessitates innovative therapies targeting the unique metabolic vulnerabilities of tumor cells. This study investigates microbial methionine gamma-lyase (MGL), an enzyme that depletes methionine, as a potential ...">
<meta name="og:title" content="Production and purification of methionine gamma-lyase from Iranian soil mulds: investigation of physicochemical properties and anticancer effects">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Cancer’s rising global incidence necessitates innovative therapies targeting the unique metabolic vulnerabilities of tumor cells. This study investigates microbial methionine gamma-lyase (MGL), an enzyme that depletes methionine, as a potential ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366409/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366409">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12885-025-14754-0"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12885_2025_Article_14754.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366409%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366409/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366409/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366409/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmccanc.png" alt="BMC Cancer logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Cancer" title="Link to BMC Cancer" shape="default" href="https://doi.org/10.1186/s12885-025-14754-0" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Cancer</button></div>. 2025 Aug 20;25:1339. doi: <a href="https://doi.org/10.1186/s12885-025-14754-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12885-025-14754-0</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Cancer%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Cancer%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Cancer%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Cancer%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Production and purification of methionine gamma-lyase from Iranian soil mulds: investigation of physicochemical properties and anticancer effects</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nasirian%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Matin Nasirian</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Matin Nasirian</span></h3>
<div class="p">
<sup>1</sup>Genetics Dept, Faculty of Basic Science, Shahrekord University, Shahrekord, Iran </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nasirian%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Matin Nasirian</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arab-SadeghAbadi%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ali Arab-SadeghAbadi</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ali Arab-SadeghAbadi</span></h3>
<div class="p">
<sup>1</sup>Genetics Dept, Faculty of Basic Science, Shahrekord University, Shahrekord, Iran </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arab-SadeghAbadi%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Ali Arab-SadeghAbadi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mobini-Dehkordi%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Mohsen Mobini-Dehkordi</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Mohsen Mobini-Dehkordi</span></h3>
<div class="p">
<sup>1</sup>Genetics Dept, Faculty of Basic Science, Shahrekord University, Shahrekord, Iran </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mobini-Dehkordi%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mohsen Mobini-Dehkordi</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Farhadian%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Sadegh Farhadian</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Sadegh Farhadian</span></h3>
<div class="p">
<sup>2</sup>Biology Dept, Faculty of Basic Science, Shahrekord University, Shahrekord, Iran </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Farhadian%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sadegh Farhadian</span></a>
</div>
</div>
<sup>2</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Genetics Dept, Faculty of Basic Science, Shahrekord University, Shahrekord, Iran </div>
<div id="Aff2">
<sup>2</sup>Biology Dept, Faculty of Basic Science, Shahrekord University, Shahrekord, Iran </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 18; Accepted 2025 Jul 24; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366409  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830757/" class="usa-link">40830757</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Cancer’s rising global incidence necessitates innovative therapies targeting the unique metabolic vulnerabilities of tumor cells. This study investigates microbial methionine gamma-lyase (MGL), an enzyme that depletes methionine, as a potential anticancer agent. Elevated methionine levels drive tumor progression by promoting S-adenosyl methionine (SAM) synthesis, leading to hypermethylation of tumor suppressor genes. By reducing methionine availability, MGL may inhibit SAM formation, preserving tumor suppressor function and hindering cancer growth. Three novel Iranian fungal strains—<em>Penicillium flavigenum</em>, <em>Stagonosporopsis cucurbitacearum</em>, and <em>Penicillium allii</em>—were identified as MGL producers through ITS sequencing. MGL was purified from <em>P. flavigenum</em>, and its biochemical properties, including stability, optimal pH, and temperature, were characterized, revealing high affinity for L-methionine and strong catalytic efficiency. Expression analysis of apoptotic genes (BCL-2 and caspase-3) demonstrated MGL’s role in enhancing apoptosis. In vitro assays confirmed its effectiveness against breast (MCF-7), liver (Hep G2), leukemia (MOLT-4), and glioblastoma (U87MG) cancer cell lines, with minimal toxicity to normal cells. This study underscores the therapeutic potential of fungal-derived MGL as a novel anticancer agent, highlighting its ability to modulate apoptotic pathways and providing a foundation for future clinical applications.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Muld native strains, Methionine gamma-lyase, Anti-cancer enzyme, Gene expression, BCL-2, Caspase-3</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">With the rising incidence of cancer worldwide, there is a pressing need for new and effective therapeutic strategies for cancer treatment. One method that has gained attention in recent years is supportive therapy utilizing enzymes such as L-asparaginase [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>], L-glutaminase [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>], and L-methionase [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Cancer cells require increased amounts of specific metabolic and nutritional substances, such as glucose and amino acids, particularly methionine, to support their rapid proliferation [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>].</p>
<p id="Par3">Methionine gamma-lyase is an enzyme that depletes methionine from the blood, leading to cell death in cancer cells by reducing their methionine intake. The antitumor mechanism of methionine gamma-lyase is primarily attributed to the depletion of molecules synthesized from methionine. High levels of methionine promote cancer growth as it serves as a precursor to S-adenosylmethionine (SAM). Elevated SAM levels can lead to hypermethylation of promoter regions in tumor suppressor genes, thus increasing the risk of tumor formation. By reducing methionine levels, the formation of S-adenosylmethionine is prevented, allowing tumor suppressor genes to remain unmethylated and thereby inhibiting cancer progression [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. Additionally, methionine gamma-lyase breaks down methionine, which reduces polyamine synthesis. This reduction in polyamines is significant, as polyamines have been associated with increased cell proliferation and decreased apoptosis, both of which contribute to cancer risk. Recent research indicates that methionine gamma-lyase is effective in treating various types of cancer, often resulting in tumor shrinkage or disappearance. However, it is essential to note that methionine gamma-lyase is not used as a standalone treatment; instead, it is typically employed as a supportive or adjuvant therapy [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>].</p>
<p id="Par4">Laboratory studies, animal studies, and clinical trials conducted since the 1990 s have demonstrated that methionine restriction is an effective anticancer strategy. In vitro experiments using various cell lines have shown that recombinant methionine gamma-lyase is effective in treating colon cancer (Colo205 and SW620), skin cancer (A375), ovarian cancer (A2780), and breast cancer (MCF-7) [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Diets supplemented with methionine gamma-lyase significantly inhibited prostate tumor growth in mice. Additionally, levels of L-methionine, its conversion product L-cysteine, and the oxidative stress marker malondialdehyde were significantly reduced in vivo. These findings suggest a method for reducing L-methionine in foods through engineered methionine gamma-lyase, which effectively inhibits tumor growth in mice [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>].</p>
<p id="Par5">Methionine gamma-lyase has also been tested as a potent antiproliferative enzyme for human lung and colon cancers, glioblastoma, and neuroblastoma. Reports indicate that the growth of human tumors in vivo and in vitro (xenografted in nude mice) is inhibited compared to normal cells after treatment with polyethylene glycol conjugated with recombinant methionine gamma-lyase. Mouse models of colon, lung, and brain cancer have shown efficacy and synergy when treated with methionine gamma-lyase, either alone or in combination with chemotherapeutic agents such as cisplatin, 5-fluorouracil, and vincristine [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. Moreover, methionine gamma-lyase has shown the ability to inhibit and treat prostate cancer, reducing or stabilizing prostate-specific tumor markers in patients with the disease [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. In some cases, it has been effective in halting Lewis’s lung cancers, certain types of colon cancer, glioblastoma, neuroblastoma, and other cancer types [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>].</p>
<p id="Par6">Recent significant efforts in cancer research are focused on developing new forms of chemotherapy that leverage the clear distinction between cancer cells and normal cells, similar to the approach used with MGL [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. One distinct characteristic of certain types of cancer cells is their dependency on methionine [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Cancer cells have a firm reliance on methionine for their growth, and when they are deprived of it using MGL, their proliferation is significantly hindered. These cells cannot adapt to the loss of methionine, which makes them vulnerable. Because of this, using MGL in enzymotherapy is seen as a promising strategy for effectively targeting cancer cells [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Furthermore, neither healthy volunteers nor cancer patients have reported any adverse side effects from the oral administration of MGL [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>].</p>
<p id="Par7">Methionine gamma-lyase is found in nearly all organisms, including bacteria, fungi, protozoa, and plants, but is absent in mammals. Most bacteria, protozoa, and plants produce this enzyme intracellularly, while fungal sources tend to produce it extracellularly [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Following initial studies, several researchers have concentrated on purifying, biochemically characterizing, and evaluating the therapeutic potential of methionine gamma-lyase as an anticancer agent against various human cancer cell lines [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>–<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Despite its wide applications, there have been few studies focusing specifically on eukaryotic (especially fungal) enzymes or comparing them with bacterial sources. In general, the therapeutic effects of bacterial enzymes are often associated with high immunogenicity, low substrate specificity, and potential toxicity to the kidneys and liver [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. In contrast, these issues do not appear to be significant for fungal enzymes. Immunogenicity problems related to bacterial (prokaryotic) methionine gamma-lyase can be addressed through strategies such as polyethylene glycol-polyglycolylation or by using immunization techniques that combine T-cell epitope deletion with neutralizing drift. Additionally, fungal sources tend to produce higher quantities of the enzyme compared to bacterial sources [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>].</p>
<p id="Par8">This study primarily focuses on screening for L-methionine-producing mulds found in soil. It reports for the first time the three strains of bacteria, <em>Penicillium flavigenum</em>, <em>Stagonosporopsis cucurbitacearum</em>, and <em>Penicillium allii</em>, which are novel producers of L-methioninase. These strains may serve as promising alternative resources for cancer treatment. Additionally, we have purified methionine gamma-lyase from novel strains of <em>Penicillium flavigenum</em> and reviewed its pharmaceutical formulation. Furthermore, the anticancer efficacy of the purified enzyme should be evaluated against various cancer cell lines in comparison to regular cell lines. This study is also the first to investigate the impact of methionine gamma-lyase enzyme expression on the apoptotic genes BCL-2 and caspase-3 in both cancerous and healthy cells. This research aims to identify potential targeted cancer treatments and to understand the physicochemical properties of the enzyme for drug development. It also explores the effects and toxicity of the methionine gamma-lyase enzyme on the expression of apoptotic genes in both cancerous and healthy cells.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Microorganism</h3>
<section id="Sec4"><h4 class="pmc_sec_title">Enhancement, testing, separation, and analysis of Muld isolates that produce MGL</h4></section></section><section id="Sec5"><h3 class="pmc_sec_title">Sample collection</h3>
<p id="Par9">Soil samples were collected from twenty different locations across Iran at a depth of 10 to 15 cm from the soil surface (to maintain some moisture) and transferred to sterile sampling containers.</p>
<section id="Sec6"><h4 class="pmc_sec_title">Purification of Muld samples</h4>
<p id="Par10">After the incubation period, a small square piece of muld was cut from the edge of each muld colony, which had a distinct shape and color. Using a sterile loop, the samples were transferred along with the underlying agar to a Petri dish containing dextrose agar culture medium enriched with yeast extract and containing the antibiotic ampicillin, all while maintaining sterile conditions. These cultures were then incubated at 30 degrees Celsius for 7 to 10 days. After the growth period, the mulds were examined for uniformity in shape and color across the entire surface of the culture medium, and pure cultures were prepared from any impure mulds.</p></section><section id="Sec7"><h4 class="pmc_sec_title">Production culture to induce L-methioninase production in mulds</h4>
<p id="Par11">A modified liquid Zapec-Dox culture medium was prepared using the ingredients listed in Table <a href="#Tab1" class="usa-link">1</a>, and its pH was adjusted to 6. The prepared culture medium was sterilized through a 0.22 μm syringe filter. Subsequently, 15 ml of the sterilized medium was distributed into empty 50 ml Erlenmeyer flasks that had also been sterilized at 121 °C and 15 psi for 17 min. Purified mulds were cut from the slant culture into 0.4 cm x 0.4 cm pieces using a sterile loop and transferred into the Erlenmeyer flasks containing the culture medium, ensuring maintaining sterile conditions. The cultures were then incubated for 72 h in a shaking incubator set at 30 °C and 130 RPM.</p>
<section class="tw xbox font-sm" id="Tab1"><h5 class="obj_head">Table 1.</h5>
<div class="caption p"><p>Ingredients and concentrations needed to prepare the modified liquid Zapec-Dox culture medium</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Substance</th>
<th align="left" colspan="1" rowspan="1">Concentration</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Sucrose</td>
<td align="left" colspan="1" rowspan="1">3%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">L-methionine</td>
<td align="left" colspan="1" rowspan="1">0.5%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Dipotassium hydrogen phosphate</td>
<td align="left" colspan="1" rowspan="1">0.1%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Magnesium sulfate</td>
<td align="left" colspan="1" rowspan="1">0.05%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Potassium chloride</td>
<td align="left" colspan="1" rowspan="1">0.05%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ferric chloride</td>
<td align="left" colspan="1" rowspan="1">0.001%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec8"><h4 class="pmc_sec_title">Quantitative assay of L-methioninase production by mulds</h4>
<p id="Par13">After 72 h of incubation, 0.2 ml of the crude enzyme was extracted from the muld samples’ liquid culture medium, ensuring that the colonies growing in the medium remained undisturbed. This was added to a Falcon tube containing 1.8 ml of a 20 mM L-methionine solution in a 50 mM Tris-HCl buffer with a pH of 7.2. The reaction solution was then incubated for 60 min at 30℃. After incubation, 0.2 ml of 15% trichloroacetic acid was added to stop the enzymatic reaction. The reaction solution was centrifuged for 10 min at 4000 rpm to precipitate the mycelium and any other solid materials that may have been present. After centrifugation, 0.2 ml of the supernatant was removed and transferred to a microtube containing 1.6 ml of distilled water, followed by vortexing. Next, 0.2 ml of Nessler’s reagent was added to the microtube, which was immediately vortexed and incubated for 10 min at room temperature. The optical absorbance of the assay sample was measured immediately after the incubation period at a wavelength of 425 nm, compared to its corresponding blank sample.</p>
<p id="Par14">Each assay sample had its own preparation for the blank sample. In the first step, 0.2 ml of crude enzyme was extracted from the muld culture medium for the assay sample. In comparison, an equal volume of crude enzyme was simultaneously added to a Falcon tube containing 1.8 ml of 50 mM Tris-HCl buffer without L-methionine. The remaining steps for the blank sample were performed in the same manner and concurrently with the assay sample.</p></section><section id="Sec9"><h4 class="pmc_sec_title">Molecular characterization of Muld isolates using ITS-rDNA</h4>
<p id="Par15">The muld isolates that exhibited the highest activity of MGL were characterized molecularly using ITS-rDNA sequencing. For ITS-rDNA amplification, the primers ITS1 and ITS4 were utilized as forward and reverse primers, respectively. The merged ITS-rDNA amplified sequences were analyzed using the nucleotide BLAST (BLASTn) tool from the NCBI database. The results were evaluated based on similarity, total score, and coverage, ultimately identifying the genus and species of the muld in question. Additionally, the desired sequence and the top twenty sequences from the database displaying the highest similarity were recorded in FASTA format using MEGA software version 11.0.13. These sequences were then aligned with the ALIGN tool, employing the MUSCLE algorithm. Finally, a phylogenetic tree was constructed using the neighbor-joining method.</p></section><section id="Sec10"><h4 class="pmc_sec_title">Protein determined</h4>
<p id="Par16">Protein levels were determined utilizing the Bradford method, using bovine serum albumin as the standard [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>].</p></section><section id="Sec11"><h4 class="pmc_sec_title">Purification of L-methioninase from <em>Penicillium flavigenom</em>
</h4>
<p id="Par17">The culture medium underwent incubation for 72 h, followed by centrifugation for 10 min at a speed of 6,000 rpm. The resulting clear supernatant, evaluated for enzymatic activity, served as the crude enzyme preparation for subsequent purification steps. This research was conducted in three distinct stages to purify the L-methioninase enzyme.</p>
<p id="Par18">The clear supernatant was subjected to thermal treatment at 55 °C and 60 °C for 20, 10, and 30 min, respectively. This step was designed to selectively denature unwanted proteins while preserving the enzymatic activity of L-methioninase. Following this treatment, the supernatant was rapidly cooled on ice for one hour to stabilize the enzyme, and the denatured proteins were subsequently removed through centrifugation at 5,000 rpm at 4 °C for 15 min.</p>
<p id="Par19">In the second purification phase, the sample demonstrating L-methioninase activity underwent dialysis. It was introduced into a Sephadex C-50 ion exchange chromatography column (25 × 25 cm), previously equilibrated with a 25 mM potassium phosphate buffer at pH 7.2. The chromatography columns were washed with two bed volumes of the same phosphate buffer to elute unbound proteins. Bound proteins were eluted stepwise using a linear sodium chloride gradient from 1.0 to 0.1 M in the phosphate buffer at a flow rate of 70 ml/h. Collected fractions were analyzed for protein concentration at an absorbance of 280 nm using the Bradford assay, and both bound and unbound protein fractions were assessed for their L-methioninase activity. Active fractions were then combined and concentrated through subsequent dialysis and ultrafiltration steps [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>].</p>
<p id="Par20">In the final stage, to achieve greater purity, in addition to the previous steps, the concentrated protein samples were further analyzed using gel filtration chromatography on a Sephadex G-100 column (2 × 50 cm). The fractions were eluted with a 25 mM potassium phosphate buffer at pH 7.2, at a flow rate of 30 ml/h, with an initial sample volume of 5 ml. Protein concentration was again estimated by measuring the absorbance at 280 nm, while L-methioninase activity was evaluated for each fraction. Active fractions were subsequently concentrated through freeze-drying and stored at −80 °C for future use.</p>
<p id="Par21">SDS-PAGE analysis assessed the purity of the fractions exhibiting the highest protein concentration and L-methioninase activity, comparing the results against standard marker proteins.</p></section></section><section id="Sec12"><h3 class="pmc_sec_title">Biochemical characterizations of lmethioninase from <em>Penicillium flavigenom</em>
</h3>
<section id="Sec13"><h4 class="pmc_sec_title">Thermal stability and the ideal temperature for the MGL</h4>
<p id="Par22">To evaluate the thermal stability of the methionine gamma-lyase enzyme, we took 1 ml of the enzyme solution and subjected it to temperatures of 4, 25, 45, 50, 60, 70, 80, 90, and 98 ℃ for periods of 30, 60, and 120 min. After each temperature exposure, we extracted 0.2 ml of the sample and conducted an enzyme assay, which was then compared to a control.</p>
<p id="Par23">Furthermore, we determined the optimal temperature for the activity of methionine gamma-lyase produced <em>by P. flavigenom</em>. This was achieved by combining 0.2 ml of the enzyme solution with 1.8 ml of methionine substrate at various temperatures of 20, 30, 40, 50, 60, 70, 80, 90, and 100 ℃, and the Nessler detection method assessed enzyme activity and compared the results to the control.</p></section><section id="Sec14"><h4 class="pmc_sec_title">The consistency of pH and the ideal pH for the MGL reaction</h4>
<p id="Par24">To assess the enzyme’s stability across different pH levels, we mixed 1 mL of the enzyme solution with 1 mL of phosphate buffer (or phosphate citrate) within a pH range of 3 to 8. The mixture was refrigerated at 4 ℃ for two hours before measuring the enzyme activity. We also compared these results against a control.</p>
<p id="Par25">Additionally, to ascertain the optimal pH for the reaction involving MGL, we combined 0.2 mL of the enzyme with 1.8 mL of 0.1 M phosphate-buffered saline (PBS) at varying pH levels of 5, 6, and 7, and the culture medium pH of 6.5. After introducing methionine into the mixture, we evaluated and compared the enzyme activity with the control using Nessler’s method.</p></section><section id="Sec15"><h4 class="pmc_sec_title">The influence of various inhibitors and activators on the function of the enzyme MGL</h4>
<p id="Par26">To assess the activators and inhibitors, a range of ions at a concentration of 3 mM—including K+, Na+, Zn2+, Mg2+, Ba2+, Ca2+, Hg2+, Cu2+, Al3+, and Fe3+—were employed alongside harmful amino acid analogs such as sodium dodecyl sulfate (SDS), hydroxylamine, hydrogen peroxide, ethanol, methanol, guanidine thiocyanate, and 2-mercaptoethanol at a concentration of 10 mM. These substances were stored in a refrigerator at 4 °C for 2 h. Subsequently, the enzyme activity was measured using the standard procedure and compared to the control.</p>
<p id="Par27">NaCl at a concentration of 2 mM, which enhanced the MGL activity, was further tested at concentrations ranging from 1 to 10 mM.</p></section><section id="Sec16"><h4 class="pmc_sec_title">The preference of the MGL enzyme for various substrates</h4>
<p id="Par28">Enzymes demonstrate specific functions, but because of the structural similarities among different amino acids, they can interact with one another’s substrates. To evaluate the affinity of methionine gamma-lyase from <em>Penicillium flavigenom</em> for various amino acids, we combined 0.2 mL of the enzyme solution with 1.8 mL of substrate for each of the following amino acids: L-methionine, L-cysteine, homocysteine, L-ornithine, L-arginine, L-glycine, L-tyrosine, and L-asparagine. We assessed the enzyme activity using the standard method and compared the outcomes to a control. This study used purified L-methioninase to calculate kinetic parameters, such as the Michaelis-Menten constant (Km) and maximum velocity (Vmax).</p></section></section><section id="Sec17"><h3 class="pmc_sec_title">In vitro evaluation of the anticancer properties of purified MGL derived from <em>Penicillium flavigenom</em> on transformed cell lines</h3>
<section id="Sec18"><h4 class="pmc_sec_title">Investigating the toxicity and lethality of the enzyme on various cell lines</h4>
<p id="Par29">A variety of cell lines, including MCF7 (breast cancer), MOLT-4 (leukemia), Hep G2 (liver cancer), U87MG (human glioblastoma), and a regular human melanocyte cell line (HFB4), were obtained from the Pasteur Institute in Iran. All cell lines were cultured in DMEM medium supplemented with 10% fetal bovine serum, 100 IU/mL penicillin, and 100 µg/mL streptomycin, maintained at 37 °C in a 5% CO2 atmosphere.</p>
<p id="Par30">The viability of cancer cells was assessed in the presence of MGL derived from <em>P. flavigenom</em> using the MTT assay. Suspensions of MCF7, MOLT-4, Hep G2, and U87MG cells (1.2 × 10^4 cells/mL) were plated in 96-well plates, with 200 µL of cell suspension added to each well and cultured for 24 h. Following this, the cells were treated with various concentrations of purified MGL (1000, 500, 250, 120, 60, 30, 15, 8, and 0 µg/mL) for an additional 24 h.</p>
<p id="Par31">Following the treatment, the MTT assay was conducted under dark conditions by incubating for 2 h at 37 °C with MTT (5 mg/mL in distilled water; 20 µL per well). Formazan crystals were solubilized using DMSO (100 µL), and the optical density of the 96-well plate was measured at 570 nm with an automated plate reader. For each 96-well plate, six vehicle controls containing either media or 0.5% DMSO were included. The MTT assay assessed the number of viable cells after 24 h of incubation. The viability and inhibition ratios of MGL were calculated using the following equations:</p>
<table class="disp-formula p" id="Equa"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d6df/12366409/49b93ef3d87f/d33e463.gif" loading="lazy" id="d33e463" alt="graphic file with name d33e463.gif"></td></tr></table>
<table class="disp-formula p" id="Equb"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d6df/12366409/55ca2adc59ef/d33e468.gif" loading="lazy" id="d33e468" alt="graphic file with name d33e468.gif"></td></tr></table>
<section id="Sec19"><h5 class="pmc_sec_title">Investigation of apoptotic and anti-apoptotic gene expression in cancer cell lines following treatment with methionine gamma-lyase</h5>
<p id="Par32">This study assessed the expression levels of apoptotic genes BCL-2 and caspase 3 using Real-Time PCR. Initially, RNA was extracted from cells treated with a concentration of IC 50 MGL and from the control group (untreated) using an RNA extraction kit (Qiagen, USA). The concentration of the extracted RNA was then measured with a Nanodrop device (Scientific Fisher Thermo, USA). Subsequently, cDNA synthesis was conducted using a kit (Takara, Japan). The reaction mixture, comprised of 5 µl of buffer and 1.8 µl of RNA, was diluted to a final volume of 10 µl with double-distilled water. The temperature-time program for cDNA synthesis included primer binding for 5 min at a specified temperature, followed by cDNA synthesis at 42 °C for 60 min, and inactivation of the reverse transcription enzyme at 85 °C. The GAPDH gene was utilized as the internal control (housekeeping gene). The primers for the Bcl-2, caspase 3, and GAPDH genes are detailed in Table <a href="#Tab2" class="usa-link">2</a>. The Real-Time PCR reaction mixture consisted of 5 µl of Master PCR Green Syber, 0.4 µl of reverse primers for each gene, 1 µl of cDNA, and 0.4 µl of Dye Reference, brought to a final volume of 10 µl with double-distilled water. The PCR was performed using the One Step Qia quant 96 plex (USA) with the following temperature conditions: 95 °C for 30 s, followed by 40 cycles of 95 °C for 10 s and 60 °C for 34 s. This investigation focused on the expression of apoptotic and anti-apoptotic genes in cancer cell lines following treatment with methionine gamma-lyase.</p>
<section class="tw xbox font-sm" id="Tab2"><h6 class="obj_head">Table 2.</h6>
<div class="caption p"><p>Sequence of primers of Bcl-2, caspase 3, and GAPDH genes</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">gene</th>
<th align="left" colspan="1" rowspan="1">Primer 5′ to 3′</th>
<th align="left" colspan="1" rowspan="1">Refrence</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Bcl-2</td>
<td align="left" colspan="1" rowspan="1">
<p>F: 5′-ACCTACCCAGCCTCCGTTAT-3′</p>
<p>R: 5′-GAACTGGGGGAGGATTGTGG-3′</p>
</td>
<td align="left" rowspan="3" colspan="1">(65)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Caspase 3</td>
<td align="left" colspan="1" rowspan="1">
<p>F: 5′-GCTGGATGCCGTCTAGAGTC-3′</p>
<p>R: 5′-ATGTGTGGATGATGCTGCCA-3′</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAPDH</td>
<td align="left" colspan="1" rowspan="1">
<p>F: 5′-GAA GGT GAA GGT CGG AGT C-3′</p>
<p>R: 5′-GAA GAT GGT GAT GGG ATT TC-3′</p>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section></section><section id="Sec20"><h4 class="pmc_sec_title">Statistical analysis</h4>
<p id="Par34">The tests in this study were conducted in triplicate, and statistical analysis was performed using GraphPad Prism software version 10. The cytotoxicity and gene expression data were evaluated using one-way analysis of variance (ANOVA). A p-value of less than 0.05 was deemed statistically significant, and results were presented as mean ± standard deviation (SD). Furthermore, the difference in threshold cycles between the tested samples (cells treated with a concentration of IC 50 MGL) and the control samples (cells untreated with the combination mentioned above) was calculated using the real-time PCR method and the ΔΔCt formula. The gene ratio of the target gene to the reference gene (GAPDH) was determined using the formula 2<sup>−ΔΔCt</sup>.</p></section></section></section><section id="Sec21"><h2 class="pmc_sec_title">Result</h2>
<section id="Sec22"><h3 class="pmc_sec_title">Screening and characterization of MGL-producing Muld isolates</h3>
<p id="Par35">A dilution series was prepared from soil samples collected using the described method and cultured on Petri dishes containing dextrose agar enriched with yeast extract and antibiotic ampicillin. The cultures were incubated at 30 °C for 3 to 7 days. The muld samples were quantitatively assessed for L-methioninase production using the Nessler method. Some samples that did not exhibit growth in the culture medium were excluded from the assay.</p>
<p id="Par36">During this process, three indicator species associated with the production of the methionine gamma-lyase enzyme were identified. Each of these species underwent molecular evaluation using the ITS-rDNA method, allowing for the determination of their genus and species. The three muld species identified as producing the enzyme in this study were <em>Penicillium flavigenum</em>,<em> Stagonosporopsis cucurbitacearum</em>, and <em>Penicillium allii</em>. A phylogenetic tree was constructed for each species, and the findings were subsequently registered in the NCBI database (Fig. <a href="#Fig1" class="usa-link">1</a>). </p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366409_12885_2025_14754_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d6df/12366409/0dc90403fcd9/12885_2025_14754_Fig1_HTML.jpg" loading="lazy" id="d33e574" height="1270" width="767" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>ITS-rDNA phylogenetic tree: (<strong>A</strong>) <em>Penicillium flavigenum isolate MA01</em>, (<strong>B</strong>) <em>Stagonosporopsis cucurbitacearum isolate MA01</em>, (<strong>C</strong>) <em>Penicillium allii isolate MA01</em></p></figcaption></figure></section><section id="Sec23"><h3 class="pmc_sec_title">Purification of MGL from <em>Penicillium flavigenum</em>
</h3>
<p id="Par38">Among the three muld species that produce methionine gamma-lyase enzyme, <em>Penicillium flavigenum</em>, which had the highest crude enzyme production in the culture medium extract, was selected for enzyme purification and further studies.</p>
<p id="Par39">To isolate the active extracellular crude L-methioninase from the <em>Penicillium flavigenum</em>, the supernatant was harvested using a cooling centrifuge at 6000×g for 10 min. The resulting crude enzyme exhibited an activity of 23.13 U/mL and a protein content of 2.30 mg/mL. This crude enzyme was subsequently purified to apparent homogeneity through three purification steps: heat treatment, a Sephadex C-50 ion exchange column, and a Sephadex G100 column. The heat treatment was implemented to eliminate denatured proteins from the supernatant, which resulted in a slight decrease in enzyme activity to 17.64 U/mL and a 52.2% reduction in protein content. The purification fold achieved was 1.59, with a specific activity of 16.03 U/mg and a recovery yield of 76.26%.</p>
<p id="Par40">Additionally, this step effectively reduced the protein content in the precipitation fraction, which resulted in the highest enzyme recovery. Further purification steps were essential to attain a greater purification fold. Data obtained indicated that the purification of L-methioninase on CM Sephadex C-50 revealed that fractions 6 to 15 exhibited enzyme activity with a purification fold of 2.96 and a 54.04% yield. Enzymatic activity was also detected in fractions 15 to 18, which displayed higher protein content. The active fractions were pooled together, increasing specific activity (68.66 U/mg) and a purification fold of 6.83 (Table <a href="#Tab3" class="usa-link">3</a>).</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Purification steps, purification folds, and recovery yields of L-methioninase from <em>P.flavigenom</em></p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Sample</th>
<th align="left" colspan="1" rowspan="1">Total protein (mg/ml)</th>
<th align="left" colspan="1" rowspan="1">Total activity(U/ml)</th>
<th align="left" colspan="1" rowspan="1">Specific activity (U/mg)</th>
<th align="left" colspan="1" rowspan="1">Enzyme recovery (%)</th>
<th align="left" colspan="1" rowspan="1">Purification fold</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Curd MGL</td>
<td align="center" colspan="1" rowspan="1">2.30</td>
<td align="center" colspan="1" rowspan="1">23.13</td>
<td align="center" colspan="1" rowspan="1">10.05</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Heated treatment</td>
<td align="center" colspan="1" rowspan="1">1.10</td>
<td align="center" colspan="1" rowspan="1">17.64</td>
<td align="center" colspan="1" rowspan="1">16.03</td>
<td align="left" colspan="1" rowspan="1">76.26</td>
<td align="left" colspan="1" rowspan="1">1.59</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CM Sephadex C-50</td>
<td align="center" colspan="1" rowspan="1">0.42</td>
<td align="center" colspan="1" rowspan="1">12.50</td>
<td align="center" colspan="1" rowspan="1">29.76</td>
<td align="left" colspan="1" rowspan="1">54.04</td>
<td align="left" colspan="1" rowspan="1">2.96</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Sephadex G100 (Gel filtration)</td>
<td align="center" colspan="1" rowspan="1">0.12</td>
<td align="center" colspan="1" rowspan="1">8.24</td>
<td align="center" colspan="1" rowspan="1">68.66</td>
<td align="left" colspan="1" rowspan="1">35.62</td>
<td align="left" colspan="1" rowspan="1">6.83</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par42">SDS-PAGE was conducted following the concentration of the active fractions of Sephadex G100 to verify enzyme purity and molecular weight. The results displayed a single band at an approximate molecular mass of 62 kDa, indicating the presence of two identical subunits, which confirmed the dimeric structure of L-methioninase from <em>P. flavigenom</em> (Fig. <a href="#Fig2" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d6df/12366409/d971b76a14fb/12885_2025_14754_Fig2_HTML.jpg" loading="lazy" id="MO2" height="484" width="650" alt="Fig. 2"></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SDS-PAGE analysis of different stages during the purification of MGL from <em>P. flavigenom</em> Line 1: marker, Line 2:After Sephadex G-100,Line 3: After CM Sephadex C-50,Line 4:After Heating</p></figcaption></figure></section><section id="Sec24"><h3 class="pmc_sec_title">Optimization of reaction parameters to increase the activity of L-methioninase from <em>P. flavigenom</em>
</h3>
<section id="Sec25"><h4 class="pmc_sec_title">Impact of incubation temperature and buffer pH on MGL activity and stability</h4>
<p id="Par44">The stability of the methionine gamma-lyase enzyme was tested at temperatures of 4, 25, 45, 50, 60, 70, 80, 90, and 98 ℃ for 30, 60, and 120 min without substrate presence. Subsequently, the enzyme activity was measured using Nessler’s reagent. The enzyme’s stability at temperatures below 45 ℃remained unchanged over 2 h. However, at 50 ℃, a 30% decline in enzyme activity was observed after 120 min. At 60 ℃, the enzyme activity decreased by 40% of its initial level after 60 min, and it reached half of its initial activity after 80 min. At temperatures exceeding 90 ℃, the enzyme exhibited no activity after 120 min. Specifically, at 98 ℃—the boiling point of water at the testing location—no enzyme activity was detected after just 60 min. (Fig. <a href="#Fig3" class="usa-link">3</a>A).</p>
<figure class="fig xbox font-sm" id="Fig3"><h5 class="obj_head">Fig. 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366409_12885_2025_14754_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d6df/12366409/9d1d62cba16a/12885_2025_14754_Fig3_HTML.jpg" loading="lazy" id="MO3" height="1270" width="704" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effect of pH and temperature on <em>P.falvigenom</em> L-methioninase: (<strong>A</strong>) thermal stability of methionine gamma-lyase from <em>P.falvigenom</em>; (<strong>B</strong>) effect of temperature on <em>P.falvigenom</em> L-methioninase activity; (<strong>C</strong>) effect of pH on <em>P.falvigenom</em> L-methioninase activity; (<strong>D</strong>) effect of pH on <em>P.falvigenom</em> L-methioninase stability</p></figcaption></figure><p id="Par46">Enzyme activity was assessed at various temperatures, 20, 30, 40, 50, 60, 70, 80, 90, and 100 ℃. The highest enzyme activity was observed at temperatures between 30 and 37 ℃. At 25 ℃, considered ambient temperature, enzyme activity was nearly equivalent to that at 50 ℃. The lowest enzyme activity was recorded at 100 ℃ (Fig. 3B). To evaluate the enzyme’s stability at different pH levels, the enzyme was incubated without substrate at pH values ranging from 3 to 11 for 60 min. The enzyme demonstrated the highest and most stable activity at pH levels of 5 to 7. It proved to be more resistant in acidic conditions, experiencing a significant decline in stability as the pH became more alkaline. The lowest enzyme activity was noted at pH 11 (Fig. <a href="#Fig3" class="usa-link">3</a>C). Additionally, we examined enzyme activity across the same pH range. In the acidic range, the lowest activity occurred at pH 3, while the highest activity was found at pH 7, where enzyme activity reached 100%. The acidic environment had minimal impact on enzyme activity; even at pH 3, it retained 60% of its activity. In contrast, the enzyme was more sensitive to alkaline conditions as activity approached zero at pH 11 (Fig. <a href="#Fig3" class="usa-link">3</a>D).</p></section><section id="Sec26"><h4 class="pmc_sec_title">Influence of metal ions and buffer additives on the activity of purified MGL</h4>
<p id="Par47">Our experiments found that copper sulfate had the most substantial inhibitory effect on the enzyme methionine gamma-lyase, followed by iron sulfate. We observed no increase in enzyme activity when incubated with these ions. Notably, methionine gamma-lyase requires no metal coenzymes and functions independently (Table <a href="#Tab4" class="usa-link">4</a>).</p>
<section class="tw xbox font-sm" id="Tab4"><h5 class="obj_head">Table 4.</h5>
<div class="caption p"><p>Effect of some metal ions and chemical reagents on <em>P.flavigenom</em> L-methioninase</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">
<strong>Metal ions (2 mM)</strong>
</th>
<th align="left" colspan="1" rowspan="1">
<strong>Relative activity (%)</strong>
</th>
<th align="left" colspan="1" rowspan="1">
<strong>Chemical reagent(10mM)</strong>
</th>
<th align="left" colspan="1" rowspan="1">
<strong>Relative activity(%)</strong>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">control</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">Hydroxylamine</td>
<td align="left" colspan="1" rowspan="1">31</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KCl</td>
<td align="left" colspan="1" rowspan="1">81</td>
<td align="left" colspan="1" rowspan="1">Guanyldine thiocyanate</td>
<td align="left" colspan="1" rowspan="1">40</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NaCl</td>
<td align="left" colspan="1" rowspan="1">95</td>
<td align="left" colspan="1" rowspan="1">2- mercaptoethanol</td>
<td align="left" colspan="1" rowspan="1">14</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ZnCl<sub>2</sub>
</td>
<td align="left" colspan="1" rowspan="1">82</td>
<td align="left" colspan="1" rowspan="1">H2O2</td>
<td align="left" colspan="1" rowspan="1">20</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MgSO<sub>4</sub>
</td>
<td align="left" colspan="1" rowspan="1">30</td>
<td align="left" colspan="1" rowspan="1">SDS</td>
<td align="left" colspan="1" rowspan="1">48</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BaCl<sub>2</sub>
</td>
<td align="left" colspan="1" rowspan="1">31</td>
<td align="left" colspan="1" rowspan="1">EDTA</td>
<td align="left" colspan="1" rowspan="1">100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CaCl<sub>2</sub>
</td>
<td align="left" colspan="1" rowspan="1">89</td>
<td align="left" colspan="1" rowspan="1">Ethanol</td>
<td align="left" colspan="1" rowspan="1">98</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HgCl<sub>2</sub>
</td>
<td align="left" colspan="1" rowspan="1">62</td>
<td align="left" colspan="1" rowspan="1">Methanol</td>
<td align="left" colspan="1" rowspan="1">100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CuSO<sub>4</sub>
</td>
<td align="left" colspan="1" rowspan="1">25</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AlCl<sub>3</sub>
</td>
<td align="left" colspan="1" rowspan="1">91</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FeCl<sub>3</sub>
</td>
<td align="left" colspan="1" rowspan="1">90</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FeSo<sub>4</sub>
</td>
<td align="left" colspan="1" rowspan="1">28</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par49">Various chemical reagents were assessed for their inhibitory effects on the methionine gamma-lyase enzyme. Among the reagents tested, 2-mercaptoethanol and hydrogen peroxide showed the most significant inhibitory impact, with over 80% inhibition achieved. Hydroxylamine and guanidine thiocyanate also exhibited notable inhibitory effects, exceeding 60% (Table <a href="#Tab4" class="usa-link">4</a>). The methionine gamma-lyase enzyme from <em>P. flavigenom</em> is particularly promising for pharmaceuticals and biotechnology due to its resilience to ethanol and methanol.</p>
<p id="Par50">Chlorine ions also decreased enzyme activity, with the most significant reductions observed for barium chloride and mercury chloride. To further investigate, we examined the effects of different concentrations of sodium chloride, ranging from 2 to 10 mM, on the enzyme. We found that sodium chloride concentrations up to 4 mM did not impact enzyme activity, but higher concentrations decreased activity. (Fig. <a href="#Fig4" class="usa-link">4</a>A).</p>
<figure class="fig xbox font-sm" id="Fig4"><h5 class="obj_head">Fig. 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d6df/12366409/e54be813ac5c/12885_2025_14754_Fig4_HTML.jpg" loading="lazy" id="MO4" height="374" width="650" alt="Fig. 4"></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Determination of: (<strong>A</strong>) Effect on the activity of purified MGL by different consternation of NaCl; (<strong>B</strong>) substrate specificity; (<strong>C</strong>) Km and Vmax; (<strong>D</strong>) reaction time</p></figcaption></figure></section><section id="Sec27"><h4 class="pmc_sec_title">Determination of substrate specificity, km, and vmax, and half-life of the purified MGL</h4>
<p id="Par51">This study was conducted to assess the specificity. Purified L-methioninase was tested with various substrates. Equal amounts (20 mM) of specific substrates, including L-methionine, L-cysteine, homocysteine, L-ornithine, L-arginine, L-glycine, L-tyrosine, and L-asparagine, were added individually to the reaction mixture and incubated under optimal assay conditions. The results, presented as relative activities in Fig. <a href="#Fig4" class="usa-link">4</a>(B), demonstrated that the enzyme displayed the highest affinity for L-methionine, which served as the standard substrate. Furthermore, cysteine exhibited a degradation level of 80% in comparison to the activity of L-methionine. The activity of the enzyme toward homocysteine and L-ornithine was found to be less than 60% and 40%, respectively, compared to its activity with L-methionine. Studies on the substrate specificity of <em>P. flavigenom</em> L-methioninase showed that this enzyme exhibits different activity levels for various amino acids, with a significant preference for L-methionine. Additionally, L-cysteine and homocysteine serve as effective substrates.</p>
<p id="Par52">The kinetic parameters for purified methionine gamma-lyase derived from <em>P. flavigenom</em> were ascertained through the application of a Lineweaver-Burk plot, plotting 1/V against 1/[S]. The values obtained for the Michaelis constant (Km) and the maximum velocity (Vmax) of the enzyme were found to be 19.25 mM and 0.8131 U/mL/min, respectively (Fig. <a href="#Fig4" class="usa-link">4</a>C). The enzyme exhibited maximal activity at substrate concentrations of 60, 70, 80, 90, 100, and 120 mM for L-methionine. This suggests that the enzyme’s active site became saturated with the substrate at concentrations above 50 mM. The relationship between substrate concentrations and the reaction velocity demonstrated a hyperbolic pattern rather than a sigmoidal one, indicating that this enzyme may be classified as allosteric. The apparent Km value of methionine gamma-lyase indicates a high affinity for its substrate, highlighting its potential as an efficient antitumor agent. Furthermore, the catalytic efficiency of fungal L-methioninase was superior to that of the bacterial variants, as evidenced by the Km values for L-methionine.</p>
<p id="Par53">The reaction time was systematically varied from 5 to 35 min to investigate its effect on enzyme activity. The highest activity of MGL was observed after 20 min of incubation. Following this, a gradual decline in activity was noted. The half-life (t1/2) of purified methionine gamma-lyase from <em>P. flavigenom</em> is 503.32 min (approximately 8.38 h) (Fig. <a href="#Fig4" class="usa-link">4</a>D).</p></section></section><section id="Sec28"><h3 class="pmc_sec_title">Impact of the purified methionine gamma-lyase derived from <em>P. flavigenome</em> on transformed cell lines</h3>
<section id="Sec29"><h4 class="pmc_sec_title">Impact of purified methionine gamma-lyase derived from <em>P. flavigenom</em> on both cancerous and normal cell lines in vitro</h4>
<p id="Par54">Different concentrations of purified methionine gamma-lyase from <em>P. flavigenome</em>, specifically 1000, 500, 250, 120, 60, 30, 15, 8, and 0 µg/ml, were tested for their ability to inhibit the growth of transformed cells, including MCF-7, Hep G-2, MOLT-4, and U87MG. The results showed that L-methioninase has a strong capacity to combat cancer cells. IC50 values for various cancer cell lines were determined to be over 47.63 µg/ml for breast cancer cells (MCF-7), 249 µg/ml for liver cancer cells (HepG-2), 257 µg/ml for leukemia cells (MOLT-4), and 267 µg/ml for human glioblastoma cells (U87MG). These results indicate that the purified methionine gamma-lyase from <em>P. flavigenome</em> is effective in destroying cancerous cells, as demonstrated by the MTT assay and analyzed using GraphPad Prism 10.2.3. In contrast, the L-methioninase enzyme did not affect the growth of normal melanocytes (HFB4 cells), even at concentrations significantly higher than the IC50, which was set at 1000 µg/ml. Methionine gamma-lyase is considerably less toxic to normal HFB4 melanocyte cells than to cancer cells. (Fig. <a href="#Fig5" class="usa-link">5</a>)</p>
<figure class="fig xbox font-sm" id="Fig5"><h5 class="obj_head">Fig. 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366409_12885_2025_14754_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d6df/12366409/40b03137e8d0/12885_2025_14754_Fig5_HTML.jpg" loading="lazy" id="MO5" height="508" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cell viability of transformed cell lines after 24 h treatment with purified MGL from <em>P.falvigenom</em>; (<strong>A</strong>) MCF-7; (<strong>B</strong>) Hep G-2;(<strong>C</strong>) U87MG and (<strong>D</strong>)MOLT-4;(<strong>E</strong>) Cytotoxicity of L-methioninase from <em>P.falvigenom</em> on the regular cell (HFB4)</p></figcaption></figure><p id="Par56">In this study, the most significant effect of methionine gamma-lyase, purified from <em>P. flavigenom</em>, on cancer cell lines was observed in the human glioblastoma cell line MCF-7.</p></section><section id="Sec30"><h4 class="pmc_sec_title">Expression analysis of Caspase-3 and BCL-2 genes in cancerous and normal cell lines in vitro under the influence of methionine gamma-lyase</h4>
<p id="Par57">This study examined the expression profiles of the caspase-3 and BCL-2 genes in various cancer cell lines treated with the IC50 concentration of methionine gamma-lyase derived from <em>P. flavigenom</em> over 24 h. Melting curve analysis was utilized to confirm the accuracy of the gene amplification. The results demonstrated that the GAPDH, caspase-3, and BCL-2 genes were amplified successfully. Notably, the expression of the caspase-3 gene exhibited a significant increase compared to the reference gene GAPDH across all cancer cell lines subjected to methionine gamma-lyase treatment after 24 h. In contrast, the expression ratio of the BCL-2 gene relative to the GAPDH reference gene showed a significant decrease in all cancer cell lines treated with methionine gamma-lyase during the same time frame (<em>p</em> &lt; 0.05) (Fig. <a href="#Fig6" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h5 class="obj_head">Fig. 6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366409_12885_2025_14754_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d6df/12366409/423c5af3870c/12885_2025_14754_Fig6_HTML.jpg" loading="lazy" id="MO6" height="1269" width="666" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Expressions of BCL-2 and Caspase-3 in the various cell lines after 24 h</p></figcaption></figure><p id="Par59">The MCF-7 cell line exhibited the most significant increase in caspase 3 expression, along with a reduced expression of Bcl-2, compared to the control group. In contrast, the regular HFB4 cell line showed no changes in the expression of either the caspase-3 or BCL-2 genes compared to the control group.</p></section></section></section><section id="Sec31"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par60">Fungi represent a highly diverse group of organisms, ranging from yeasts and mulds to larger fungi. Most fungal species spend part of their life cycle in the soil or direct contact with it [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. Mulds, in particular, exhibit a wide variety of enzymatic activities, making them potential candidates for industrial enzyme production [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Various microorganisms, including mulds, have been reported to produce L-methioninase, a therapeutic enzyme. However, there is still significant potential for further research to isolate Indigenous mulds from soils in different regions of the world to identify the highest producers of industrial enzyme applications [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Soils in different regions harbor unique microbial populations. Soil samples were collected from 20 distinct locations across Iran to enhance the diversity of the isolation process. Collecting samples from various locations leads to the identification of different strains, even when isolating mulds of the same genus and species. This variance is particularly significant, as previous studies on the isolation of L-methioninase-producing mulds from soil were conducted in countries like Egypt and India rather than in Iran [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. A dilution series was prepared using soil samples, similar to methods described in previous articles, to reduce density and facilitate purification [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. In this study, the optimal dilution for the soil samples was found to be 1,600 times, eliminating the need for successive re-cultivations that could alter the enzymatic activity of the muld in the laboratory environment. The diluted samples were then cultured in a medium containing antibiotics. Ultimately, 65 mulds were purified and stored based on their morphological differences observed in the soil sample cultures. The production process was conducted using a modified Zapec-Dox culture medium and carried out in a submerged environment. In this process, the amino acid L-methionine was used as a nitrogen source instead of sodium nitrate to enhance enzyme production. The production period lasted 72 h, which allowed samples with faster growth and production to excel in the quantitative assay compared to those that grew slowly and yielded later. This shorter production time helped the superior samples become more competitive against bacterial producers, which typically have a production cycle of only 24 h. There are several methods for quantitatively measuring L-methioninase activity, depending on which products of L-methionine degradation are being analyzed. In this study, we used a quantitative method involving Nessler’s reagent, where ammonia produced from the enzymatic reaction of L-methioninase is measured. The three indicator mulds were samples C2, K4, and S2, which exhibited enzyme activities of 23.13 U/ml, 11.45 U/ml, and 11.4 U/ml, respectively.</p>
<p id="Par61">The identification of fungi has traditionally relied on their phenotypic and physiological characteristics. However, the distinctive features of fungi can complicate identification and classification based on morphology. Therefore, molecular methods can provide a more accurate determination of taxonomy. In this study, the ITS region of the genomic DNA from various mulds was sequenced and analyzed using the NCBI database, leading to the identification of the genera and species of mulds that produce L-methioninase.</p>
<p id="Par62">The results revealed that the C2 muld was identified as <em>Penicillium flavigenum</em> isolate MA01, the K4 muld as <em>Stagonosporopsis cucurbitacearum</em> isolate MA01, and the S2 muld as <em>Penicillium allii</em> isolate MA01. The production of the enzyme L-methioninase has been documented in several species of the genus <em>Penicillium</em>, including <em>Penicillium digitatum</em>,<em> Penicillium notatum</em>,<em> Penicillium cassiculum</em>,<em> Penicillium citrinum</em>, and <em>Penicillium oxalicum</em> [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. However, this study reports the production of L-methioninase from <em>Penicillium flavigenum</em> and <em>Penicillium alli</em> for the first time. Additionally, the production of L-methioninase from the genus <em>Stagonosporopsis</em>, specifically the species <em>Stagonosporopsis cucurbitaceum</em>, is also reported for the first time in this research.</p>
<p id="Par63">In this study, the primary enzyme was purified through a three-step process that involved heat treatment, Sephadex C50, and Sephadex C100 gel filtration. Upon completion of the purification, the enzyme exhibited a specific activity of 68.66 U/ml and a recovery rate of 35.62%. This purification strategy, characterized by its minimal steps and impressive enzyme yield, has proven to be both practical and economically beneficial for the industry [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. The heating process was crucial in reducing unwanted proteins by 47.8%, as demonstrated in the purification profile (Table <a href="#Tab3" class="usa-link">3</a>). In a previous study focused on the purification of L-methioninase from <em>Citrobacter freundii</em>, a heat treatment at 60 °C was applied, resulting in a 21% recovery using an initial DEAE-cellulose column [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Furthermore, heat treatment of L-methioninase derived from Streptomyces DMMHM4 achieved an even greater purity, eliminating over 50% of the protein content. In another investigation involving A. fumigatus, a similar two-step approach—consisting of heat treatment followed by Sephadex chromatography—was employed, successfully removing approximately 45% of the heat-denatured proteins while preserving 81.2% of the enzyme’s activity [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>, <a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Also, other studies have indicated that the species <em>Trichoderma harzianum</em> exhibits the highest enzyme activity reported among mulds. This activity was achieved following several stages of enzyme purification, resulting in the following specific activities: crude enzyme 6.66 U/ml)10.4U/mg), ammonium sulfate precipitation 39.47 U/ml (12.24 U/mg), ion exchange 56.47 U/ml (41.03 U/mg), and gel filtration chromatography 88.83 U/ml (74.4 U/mg) [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Another study reported enzyme production by Candida tropicalis, which achieved a crude specific enzyme activity of 1.5 U/mg, which increased to 64.78U/mg after several purification stages [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>] Additionally, the enzymatic activity of <em>Aspergillus flavipes</em> was measured at 22.65 U/ml (43.4 U/mg) in a soybean solid culture medium and 19.55 U/ml (40 U/mg) in a chicken feather solid culture medium. Notably, with the alkaline pretreatment of the chicken feather medium, this enzymatic activity increased to 71 (U/mg) [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>].</p>
<p id="Par64">The molecular weight of methionine gamma-lyase isolated from <em>P. flavigenom</em> was determined to be 62 kDa using SDS-PAGE. Notably, the molecular weights of L-methioninase purified from various other microorganisms show significant variation. For instance, <em>A. favipes</em> has been reported to have a molecular weight of 47 kDa [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>], while <em>T. harzianum</em> exhibits a molecular weight of 48 kDa [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] Similarly, <em>A. ustus</em> and <em>C. tropicalis</em> both have molecular weights of 46 kDa [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>, <a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]., whereas (A) thaliana has a molecular weight of 212 kDa [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>], <em>(B) linens</em> has 170 kDa [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>], and <em>A. flavipes</em> shows 180 kDa [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. These discrepancies in molecular weight across different microorganisms may arise from the presence or absence of denaturing agents during the purification process. Additionally, the enzyme purification method utilized in this study diverges from those employed in previous research.</p>
<p id="Par65">The storage and utilization of the drug product should adhere to specific temperature guidelines to preserve its stability and efficacy. It is advisable to maintain the product at a temperature of 25 °C to ensure optimal stability and sustained activity. Enzyme activity peaks at temperatures between 30 °C and 40 °C, although this optimum range can differ depending on the microbial source. For instance, <em>Aspergillus flavipes</em> thrives at 20–40 °C, <em>Candida tropicalis</em> operates best at 45–55 °C, <em>Streptomyces sp.</em> functions optimally at 70 °C, and <em>Bacterium leucovorum</em> is most effective at 25–30 °C. Notably, enzyme activity tends to diminish when temperatures exceed the specified optimum range [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. Research on bacteria and fungi indicates that the methionine gamma-lyase enzyme derived from fungal sources exhibits better resistance to temperature variations and can maintain functionality at elevated temperatures compared to its bacterial counterpart.</p>
<p id="Par66">In this context, methionine gamma-lyase (MGL) from <em>P. flavigenom</em> exhibits maximum activity at pH values between 6 and 7. The fungal enzyme demonstrates more significant activity in acidic environments compared to its bacterial counterpart. The fungal enzyme has an impressive ability to work well even when the pH levels fluctuate. It shows excellent resilience in acidic conditions, keeping about 60% of its activity even at pH 3. However, it doesn’t perform as well in alkaline environments, where its activity drops significantly. This lower activity in alkaline conditions might be linked to the naturally acidic environment in which these mulds thrive within the soil. The MGL enzyme remains active within a pH range of 4.0 to 8.0. Previous research indicates that the purified MGL from <em>Aspergillus flavipes</em> has optimal activity at pH 7 to 8, aligning with pH ranges between 6.8 and 8.0 [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>].</p>
<p id="Par67">In contrast, the purified MGL from <em>C. tropicalis</em> and <em>Streptomyces sp.</em> demonstrates optimal activities at pH 6.5 and 6.0, respectively. The purified MGL from Candida tropicalis achieved peak activity at pH 6.5 in sodium citrate and potassium phosphate buffers, with consistent performance across pH ranges from 5.5 to 7.0. Also, the purified MGL from Streptomyces sp. showed maximum activity in acetate buffer (50 mM) at pH 6.0. In comparison, in potassium phosphate buffer (50 mM), optimal MGL activity was observed at pH levels ranging from 6.5 to 8.0 [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>].</p>
<p id="Par68">The stability of methionine gamma-lyase from <em>P. flavigenom</em> was examined across various pH buffers. It was found that the enzyme exhibited significant catalytic stability over a broad pH range, specifically between 4.0 and 9.0. Notably, the enzyme’s activity decreased considerably under both acidic and alkaline conditions. Similarly, investigations into the pH stability of L-methioninase from <em>Aspergillus ustus</em>, <em>A. flavipes</em>, and <em>Trichoderma harzianum</em> indicated that optimal activity was present within the pH range of 6.5 to 8.5 [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>, <a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>, <a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Additionally, a study on L-methioninase from <em>Candida tropicalis</em> revealed that the enzyme retained activity at pH levels from 5.5 to 7.5, with only a slight decline noted at pH 8.0. The most significant drop-in activity was observed in the acidic range, particularly at pH 5.0 [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>].</p>
<p id="Par69">Based on the results obtained, none of the metal ions had a positive effect on the activity of the methionine gamma-lyase enzyme derived from <em>P.</em> flavigenum. Additionally, this enzyme is resistant to ethanol and methanol, which makes the enzyme from this muld a promising candidate for pharmaceutical and biotechnological applications. Previous studies indicate that methionine-gamma-lyase produced by various microorganisms exhibits enhanced enzyme activity in the presence of specific metal ions and chemical reagents, such as Cu2+, Mg2+, Co2+, and Mn2+. For instance, studies involving <em>Aspergillus flavipes</em> [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>] and <em>Candida tropicalis</em> [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] revealed that Na+, Ni2+, and Mg2 + could increase enzyme activity. However, most studies have reported no significant increase in enzyme activity due to metal ions and chemical reagents, aligning with our findings. These investigations have shown that Cu2+, Fe2+, and Hg2 + inhibit methionine-gamma-lyase activity in <em>Aspergillus flavipes</em> and <em>Streptomyces sp</em> [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>, <a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. Additionally, Ca2+, Mg2+, Zn2+, and Cd2 + exhibited a minor inhibitory effect on methionine-gamma-lyase produced by <em>T. harzianum</em> [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Methionine-gamma-lyase from <em>A. fumigatus</em> was notably inhibited by Hg2+, Cu2+, Li2+, and Mn2+, while only slight inhibition was observed with Ni2+, Fe3+, and Cr3+ [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. Other studies have explored the effects of chemical reagents on methionine-gamma-lyase; for example, enzymes from K. oxytoca demonstrated increased activity with EDTA and Tween-80. In contrast, reagents such as SDS, β-mercaptoethanol, DTT, and PMSF were found to decrease enzyme activity in <em>Streptomyces sp.</em> and <em>Aspergillus flavipes</em> [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>, <a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. Furthermore, EDTA and PMSF reduced enzyme activity in <em>Streptomyces variabilis</em> [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>], while PMSF and EDTA also diminished enzyme activity, and iodoacetate inhibited activity in methionine-gamma-lyase from <em>Candida tropicalis</em> [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>].</p>
<p id="Par70">The findings from this study showed that methionine gamma-lyase obtained from <em>P. flavigenom</em> exhibited a strong affinity for L-methionine. After L-methionine, it also showed a considerable affinity for L-cysteine. Several researchers noted that it had a clear preference for both the primary substrate and its related compounds, L-cysteine and homocysteine. Studies have indicated that L-methioninases from <em>A. flavipes</em> and <em>T. harzianum</em> display different levels of catalytic activity towards various amino acids. They showed a high specificity for L-methionine, followed by L-cystine and L-cysteine [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>, <a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Conversely, <em>A. stus</em> L-methioninase exhibited the most excellent activity against the standard substrate while demonstrating reduced activity towards other substrates. The hydrolytic function of L-methioninase in cleaving C-S and C-O bonds is predominantly seen in enzymes sourced from bacteria [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>–<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>], protozoal [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>], and fungi rather than in those interacting with C-C bonds (23.45). Additionally, research on the substrate specificity of L-methioninase from C. tropicalis revealed differing relative activities towards various sulfur-containing amino acids [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>].</p>
<p id="Par71">The Km and Vmax values for purified methionine gamma-lyase from <em>P. flavigenom</em>, utilizing L-methionine as the substrate, were found to be 19.25 mM and 0.8131 U/mL/min, respectively. These findings suggest that methionine gamma-lyase has a high affinity for its substrate (L-met), which supports its potential use as an anti-cancer therapeutic agent. The Km value provides insight into how effectively an enzyme binds to its substrate, reflecting the enzyme’s affinity and can be influenced by various factors such as temperature and pH. A higher Km signifies a lower affinity, indicating that more substrate is required for the enzyme to achieve half of its maximum activity. Conversely, Vmax is not a constant value for an enzyme; it can vary based on the enzyme concentration in a reaction, meaning it does not always represent the fundamental traits of the enzyme. Nonetheless, Vmax remains an essential indicator of an enzyme’s catalytic efficiency, offering information on how many substrate molecules can be converted into a product by a single enzyme molecule within a minute [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>, <a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>, <a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. Our research showed the peak enzyme activity at the 20-minute time point, aligning with earlier studies that reported methionine gamma-lyases from <em>Arabidopsis</em> and <em>Candida tropicalis</em> exhibited their highest activity at the same duration [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>, <a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>].</p>
<p id="Par72">The results from the cancer cell lines used in this study indicate that the examined enzyme significantly affects these cells, prompting them to undergo apoptosis. Additionally, past research has observed that cancer cells exhibit a specific metabolic defect related to the essential amino acid methionine, which may be connected to their increased dependence on tumor growth [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>, <a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. The anticancer efficacy of the purified enzyme, L-methioninase, was assessed in vitro through a series of experiments involving various cancerous and normal cell lines. The results demonstrated a pronounced effectiveness of L-methioninase in targeting cancer cell lines while exhibiting a minimal cytotoxic impact on normal cells. This selective activity suggests a promising therapeutic potential for L-methioninase in cancer treatment, warranting further investigation into its mechanisms and clinical applications.</p>
<p id="Par73">Various researchers have studied the effects of this enzyme from different sources on cancer cell lines and found similar results. The most notable impact of purified methionine gamma-lyase from <em>P. flavigenom</em> was observed in the human breast cancer cell line MCF-7. A prior study involving L-methioninase from <em>T. harzianum</em> demonstrated comparable anticancer activity against Hep2-G cell lines, followed by MCF-7 cell lines, with inhibition ratios of 47.62% and 33.84%, respectively, at a concentration of 1 U/mL [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Purified L-methioninase derived from <em>A. favipes</em> demonstrated considerable anti proliferative activity against five tested cell lines. The enzyme exhibited a concentration-dependent efficacy against various cancer cells, including lung, breast, liver, prostate, and colon [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. These findings are consistent with previous studies [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>, <a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>, <a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. Another investigation assessed the use of recombinant L-methioninase to inhibit the growth of MOLT-4 leukemia cells, yielding results similar to those observed with methionine gamma-lyase from <em>P. flavigenom</em>. This study confirmed its capability to impede the growth of cancer cells under laboratory conditions. The U87MG cell line has been evaluated in earlier research involving L-methioninase from <em>K. oxytoca</em> and a variant employing RBC for MGL transport, both of which corroborate the effects of L-methioninase in this context [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>] Extensive studies in this field indicate that L-methioninase is effective against most human cancer cell lines, primarily due to cancer cells’ dependency on methionine for survival [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>, <a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>].</p>
<p id="Par74">The findings revealed that L-methioninase demonstrated a much higher IC50 of over 1000 µg/ml, indicating that its toxicity towards normal melanocyte HFB4 cells is minimal compared to its effects on cancer cells. Normal cells possess L-methionine synthase, which enables them to produce L-methionine from homocysteine (Hcy) by utilizing methyltetrahydrofolate and betaine as methyl group donors. Hcy, a non-standard amino acid, has a structure resembling L-methionine but lacks a methyl group. Research has shown that tumor cell lines dependent on L-methionine synthase generally exhibit lower enzyme levels compared to their normal cell equivalents [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>, <a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>].</p>
<p id="Par75">For the first time in this study, we assessed the expression of apoptotic genes, specifically Caspase 3 and BCL-2, in cancer cell lines treated with methionine gamma-lyase derived from <em>P. flavigenom</em>. Following treatment with this enzyme, we observed an increase in the expression of Caspase 3, while the expression of BCL-2, which is an anti-apoptotic factor, decreased. This change suggests an increased rate of apoptosis in cancer cells exposed to methionine gamma-lyase from <em>P. flavigenom</em>. The MCF-7 cell line exhibited the most significant rise in Caspase 3 expression and a decrease in BCL-2expression, which aligns with the results obtained from the MTT assay for the purified methionine gamma-lyase from <em>P. flavigenom</em>.For the first time in this study, we assessed the expression of apoptotic genes, specifically Caspase 3 and BCL-2, in cancer cell lines treated with methionine gamma-lyase derived from <em>P. flavigenom</em>. Following treatment with this enzyme, we observed an increase in the expression of Caspase 3, while the expression of BCL-2, which is an anti-apoptotic factor, decreased. This change suggests an increased apoptosis rate in cancer cells exposed to methionine gamma-lyase from <em>P. flavigenom</em>. The MCF-7 cell line exhibited the most significant rise in Caspase 3 expression and a decrease in BCL-2expression, which aligns with the results obtained from the MTT assay for the purified methionine gamma-lyase from <em>P. flavigenom</em>.</p>
<p id="Par76">In multicellular organisms, the careful regulation of programmed cell death, known as apoptosis, is just as essential as cell growth. Although the mechanisms controlling cell death are not yet fully understood, caspases play a crucial role in apoptosis. They are activated in a specific sequence that leads to distinct apoptotic changes. Despite their essential function in this process, the activation of caspases is influenced by a variety of other factors [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>].</p>
<p id="Par77">Moreover, the BCL-2 family plays a crucial role in triggering caspases and deciding a cell’s destiny—whether it continues to live or undergoes apoptosis. The members of the BCL-2 family mainly interact with the mitochondria, reacting to circumstances that cause irreversible cellular harm. Earlier molecular research has suggested a negative correlation between the levels of Caspase 3 and BCL-2 expression [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>].</p>
<p id="Par78">Ultimately, Increased expression of caspase-3 and decreased expression of BCL-2in cancer cells treated with methionine gamma-lyase likely activate the mitochondrial apoptotic pathway [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>–<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>].</p></section><section id="Sec32"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par79">This study sheds light on the exciting potential of methionine gamma-lyase (MGL) from the <em>P. flavigenom</em> as a promising new treatment option for human cancer. The researchers successfully purified and characterized this enzyme, achieving a notable 6.83-fold enhancement in activity along with a respectable 35.62% recovery rate, all while confirming that it has a molecular weight of 62 kDa. These findings suggest that MGL could be effectively delivered using nanoparticle systems designed for the precise targeting of cancer cells.</p>
<p id="Par80">The enzyme shows impressive stability, remaining active up to 45 °C, with peak performance between 30 and 40 °C and at a pH of 6 to 7. This robustness makes MGL a strong candidate for various therapeutic applications. What’s particularly noteworthy is its significant anticancer activity against multiple cancer cell lines, including MCF-7 (breast cancer), Hep G-2 (liver cancer), MOLT-4 (leukemia), and U87MG (brain cancer), while leaving healthy cells unharmed.</p>
<p id="Par81">The way MGL works is quite fascinating. It triggers a process called apoptosis—or programmed cell death—via the mitochondrial pathway. This process aims to enhance the activity of the caspase-3 gene while diminishing the activity of the BCL-2gene. Both genes play crucial roles in regulating cellular life and death. Gaining insight into how this mechanism operates not only clarifies the role of the enzyme but also indicates potential therapeutic advantages.</p>
<p id="Par82">Additionally, since MGL is derived from a eukaryotic source, it’s likely to be more compatible with the human immune system, which reduces the chances of adverse reactions that sometimes occur with non-native proteins. The findings of this research have significant implications, indicating that MGL may serve as a pivotal agent in oncological therapies while also potentially contributing to the management of various other health conditions. These include the regulation of elevated homocysteine levels, cardiovascular diseases, neurodegenerative disorders such as Alzheimer’s, as well as metabolic concerns including obesity and the aging process.</p>
<p id="Par83">This study highlights the therapeutic potential of fungal-derived methionine gamma-lyase (MGL) as a promising anticancer agent, particularly its ability to modulate apoptotic pathways and inhibit tumor growth. However, several limitations should be acknowledged to pave the way for future research. Firstly, while in vitro experiments demonstrated significant anticancer effects, the enzyme’s performance in vivo remains unexplored, necessitating preclinical animal studies to evaluate its efficacy, pharmacokinetics, and safety. Secondly, the stability and activity of MGL under physiological conditions require further investigation to ensure its suitability for clinical applications. Additionally, the potential immunogenicity or adverse effects of fungal-derived MGL in human subjects must be rigorously assessed. Lastly, while this study focused on a limited set of cancer cell lines, expanding the scope to other cancer types and exploring combination therapies with existing treatments could enhance the therapeutic applicability of MGL.</p>
<p id="Par84">Future research should address these challenges, focusing on optimizing MGL production, improving its stability, and exploring its broader anticancer potential in vivo. Investigations into its molecular mechanisms of action, particularly in modulating apoptotic pathways, could provide deeper insights into its therapeutic benefits and support its transition to clinical use.</p>
<p id="Par85">In summary, these findings open the door to exciting new research possibilities, not just for cancer therapy but for a range of other conditions. The study highlights the promising role of methionine gamma-lyase in molecular biology and oncology, encouraging scientists further to explore its therapeutic applications in a clinical setting.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>M.N. wrote the main manuscript text and conducted experiments related to the purification and examination of the physicochemical properties of the methionine gamma-lyase enzyme and examined the effect of this enzyme on various cancer cell lines and prepared all tables and all figures except Figure 1.A.A. Isolation of molds from the soil, identification of enzyme-producing molds, and determination of the genus and species of molds by molecular methods prepared Figure 1.M.M. Determining the study methodology and team supervisor and reviewed the manuscript.S.F. Determining the study methodology and team advisor and reviewed the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>The author(s) declare that no financial support was received for the research, authorship, and/or publication of this article.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The datasets used and/or analyzed during the current study are available from the corresponding author on reasonable request.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par86">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par87">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par88">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Alrumman SA, Mostafa YS, Al-Izran KA, Alfaifi MY, Taha TH, Elbehairi SE. Production and anticancer activity of an L-asparaginase from <em>Bacillus licheniformis</em> isolated from the Red Sea, Saudi Arabia. Sci Rep. 2019;9(1): 3756.
</cite> [<a href="https://doi.org/10.1038/s41598-019-40512-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6403232/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30842557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alrumman%20SA,%20Mostafa%20YS,%20Al-Izran%20KA,%20Alfaifi%20MY,%20Taha%20TH,%20Elbehairi%20SE.%20Production%20and%20anticancer%20activity%20of%20an%20L-asparaginase%20from%20Bacillus%20licheniformis%20isolated%20from%20the%20Red%20Sea,%20Saudi%20Arabia.%20Sci%20Rep.%202019;9(1):%203756." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Sahoo SM, Sahoo S. l-Asparaginase and Methioninase as Prospective Anticancer Enzymes: Current Applications and Production Approaches. In: Thatoi H, Mohapatra S, Das SK, editors. Bioprospecting of Enzymes in Industry, Healthcare and Sustainable Environment. Singapore: Springer Singapore; 2021. pp. 349–62. Available from: 10.1007/978-981-33-4195-1_16</cite>
</li>
<li id="CR3">
<span class="label">3.</span><cite>Maharem TM, Emam MA, Said YA. Purification and characterization of L-glutaminase enzyme from camel liver: enzymatic anticancer property. Int J Biol Macromol. 2020;150:1213–22.
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2019.10.131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31743703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maharem%20TM,%20Emam%20MA,%20Said%20YA.%20Purification%20and%20characterization%20of%20L-glutaminase%20enzyme%20from%20camel%20liver:%20enzymatic%20anticancer%20property.%20Int%20J%20Biol%20Macromol.%202020;150:1213%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Orabi H, El-Fakharany E, Abdelkhalek E, Sidkey N. Production, optimization, purification, characterization, and anti-cancer application of extracellular l-glutaminase produced from the marine bacterial isolate. Prep Biochem Biotechnol. 2020;50(4):408–18.
</cite> [<a href="https://doi.org/10.1080/10826068.2019.1703193" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31846380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Orabi%20H,%20El-Fakharany%20E,%20Abdelkhalek%20E,%20Sidkey%20N.%20Production,%20optimization,%20purification,%20characterization,%20and%20anti-cancer%20application%20of%20extracellular%20l-glutaminase%20produced%20from%20the%20marine%20bacterial%20isolate.%20Prep%20Biochem%20Biotechnol.%202020;50(4):408%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Chaturvedi S, Hoffman RM, Bertino JR. Exploiting methionine restriction for cancer treatment. Biochem Pharmacol. 2018;154:170–3.
</cite> [<a href="https://doi.org/10.1016/j.bcp.2018.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29733806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chaturvedi%20S,%20Hoffman%20RM,%20Bertino%20JR.%20Exploiting%20methionine%20restriction%20for%20cancer%20treatment.%20Biochem%20Pharmacol.%202018;154:170%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Alshehri WA. Bacterium <em>hafnia alvei</em> secretes l-methioninase enzyme: optimization of the enzyme secretion conditions. Saudi J Biol Sci. 2020;27(5):1222–7.
</cite> [<a href="https://doi.org/10.1016/j.sjbs.2020.02.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7182987/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32346328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alshehri%20WA.%20Bacterium%20hafnia%20alvei%20secretes%20l-methioninase%20enzyme:%20optimization%20of%20the%20enzyme%20secretion%20conditions.%20Saudi%20J%20Biol%20Sci.%202020;27(5):1222%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Raboni S, Faggiano S, Bettati S, Mozzarelli A. Methionine gamma lyase: structure-activity relationships and therapeutic applications. Biochimica et Biophysica Acta (BBA). 2024;1872(3): 140991.</cite> [<a href="https://doi.org/10.1016/j.bbapap.2023.140991" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38147934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raboni%20S,%20Faggiano%20S,%20Bettati%20S,%20Mozzarelli%20A.%20Methionine%20gamma%20lyase:%20structure-activity%20relationships%20and%20therapeutic%20applications.%20Biochimica%20et%20Biophysica%20Acta%20(BBA).%202024;1872(3):%20140991." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Dhankhar R, Gupta V, Kumar S, Kapoor RK, Gulati P. Microbial enzymes for deprivation of amino acid metabolism in malignant cells: biological strategy for cancer treatment. Appl Microbiol Biotechnol. 2020;104:2857–69.
</cite> [<a href="https://doi.org/10.1007/s00253-020-10432-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32037468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dhankhar%20R,%20Gupta%20V,%20Kumar%20S,%20Kapoor%20RK,%20Gulati%20P.%20Microbial%20enzymes%20for%20deprivation%20of%20amino%20acid%20metabolism%20in%20malignant%20cells:%20biological%20strategy%20for%20cancer%20treatment.%20Appl%20Microbiol%20Biotechnol.%202020;104:2857%E2%80%9369." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Shim J, Shin Y, Lee I, Kim SY. L-Methionine production. Amino Acid Ferment. 2017;153–77.</cite> [<a href="https://doi.org/10.1007/10_2016_30" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27872967/" class="usa-link">PubMed</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Kawaguchi K, Miyake K, Han Q, Li S, Tan Y, Igarashi K, et al. Oral recombinant methioninase (o-rMETase) is superior to injectable rMETase and overcomes acquired gemcitabine resistance in pancreatic cancer. Cancer Lett. 2018;432:251–9.
</cite> [<a href="https://doi.org/10.1016/j.canlet.2018.06.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29928962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kawaguchi%20K,%20Miyake%20K,%20Han%20Q,%20Li%20S,%20Tan%20Y,%20Igarashi%20K,%20et%20al.%20Oral%20recombinant%20methioninase%20(o-rMETase)%20is%20superior%20to%20injectable%20rMETase%20and%20overcomes%20acquired%20gemcitabine%20resistance%20in%20pancreatic%20cancer.%20Cancer%20Lett.%202018;432:251%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Hu Y, Liu Y, Zhang J, Zhou Z, Wang J, Chen H, et al. Depletion of L-Methionine in foods with an engineered thermophilic methionine γ-lyase efficiently inhibits tumor growth. J Agric Food Chem. 2023;71(45):17141–52.</cite> [<a href="https://doi.org/10.1021/acs.jafc.3c05293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37909421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu%20Y,%20Liu%20Y,%20Zhang%20J,%20Zhou%20Z,%20Wang%20J,%20Chen%20H,%20et%20al.%20Depletion%20of%20L-Methionine%20in%20foods%20with%20an%20engineered%20thermophilic%20methionine%20%CE%B3-lyase%20efficiently%20inhibits%20tumor%20growth.%20J%20Agric%20Food%20Chem.%202023;71(45):17141%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Liu TP, Lo HL, Wei LS, Hsiao HH, yun, Yang PM. S-adenosyl-L-methionine-competitive inhibitors of the histone methyltransferase EZH2 induce autophagy and enhance drug sensitivity in cancer cells. Anticancer Drugs. 2015;26(2):139–47.
</cite> [<a href="https://doi.org/10.1097/CAD.0000000000000166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4276571/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25203626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20TP,%20Lo%20HL,%20Wei%20LS,%20Hsiao%20HH,%20yun,%20Yang%20PM.%20S-adenosyl-L-methionine-competitive%20inhibitors%20of%20the%20histone%20methyltransferase%20EZH2%20induce%20autophagy%20and%20enhance%20drug%20sensitivity%20in%20cancer%20cells.%20Anticancer%20Drugs.%202015;26(2):139%E2%80%9347." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>El-Sayed AS. Microbial. L-methioninase: production, molecular characterization, and therapeutic applications. Appl Microbiol Biotechnol. 2010;86:445–67.
</cite> [<a href="https://doi.org/10.1007/s00253-009-2303-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20146062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?El-Sayed%20AS.%20Microbial.%20L-methioninase:%20production,%20molecular%20characterization,%20and%20therapeutic%20applications.%20Appl%20Microbiol%20Biotechnol.%202010;86:445%E2%80%9367." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Abdelraof M, Selim MH, Elsoud MMA, Ali MM. Statistically optimized production of extracellular l-methionine γ-lyase by streptomyces sp. DMMMH60 and evaluation of purified enzyme in sub-culturing cell lines. Biocatal Agric Biotechnol. 2019;18:101074.</cite> [<a href="https://scholar.google.com/scholar_lookup?Abdelraof%20M,%20Selim%20MH,%20Elsoud%20MMA,%20Ali%20MM.%20Statistically%20optimized%20production%20of%20extracellular%20l-methionine%20%CE%B3-lyase%20by%20streptomyces%20sp.%20DMMMH60%20and%20evaluation%20of%20purified%20enzyme%20in%20sub-culturing%20cell%20lines.%20Biocatal%20Agric%20Biotechnol.%202019;18:101074." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Selim MH, Elshikh HH, Saad MM, Mostafa EE, Mahmoud MA. Purification and characterization of a novel thermo stable L-methioninase from streptomyces sp. DMMMH4 and its evaluation for anticancer activity. J Appl Pharm Sci. 2016;6(7):53–60.</cite> [<a href="https://scholar.google.com/scholar_lookup?Selim%20MH,%20Elshikh%20HH,%20Saad%20MM,%20Mostafa%20EE,%20Mahmoud%20MA.%20Purification%20and%20characterization%20of%20a%20novel%20thermo%20stable%20L-methioninase%20from%20streptomyces%20sp.%20DMMMH4%20and%20its%20evaluation%20for%20anticancer%20activity.%20J%20Appl%20Pharm%20Sci.%202016;6(7):53%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Salim N, Santhiagu A, Joji K. Process modeling and optimization of high yielding L-methioninase from a newly isolated <em>Trichoderma harzianum</em> using response surface methodology and artificial neural network coupled genetic algorithm. Biocatal Agric Biotechnol. 2019;17:299–308.</cite> [<a href="https://scholar.google.com/scholar_lookup?Salim%20N,%20Santhiagu%20A,%20Joji%20K.%20Process%20modeling%20and%20optimization%20of%20high%20yielding%20L-methioninase%20from%20a%20newly%20isolated%20Trichoderma%20harzianum%20using%20response%20surface%20methodology%20and%20artificial%20neural%20network%20coupled%20genetic%20algorithm.%20Biocatal%20Agric%20Biotechnol.%202019;17:299%E2%80%93308." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Hoffman RM, Han Q. Oral methioninase for covid-19 methionine-restriction therapy. In Vivo. 2020;34(3 suppl):1593–6.
</cite> [<a href="https://doi.org/10.21873/invivo.11948" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8378026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32503816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoffman%20RM,%20Han%20Q.%20Oral%20methioninase%20for%20covid-19%20methionine-restriction%20therapy.%20In%20Vivo.%202020;34(3%20suppl):1593%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Nasirian M, Mobini-Dehkordi M, Khosravian P. The native Iranian soil bacteria with high potential to produce extracellular methionine gamma-lyase. Front Microbiol. 2024;15:1504742.
</cite> [<a href="https://doi.org/10.3389/fmicb.2024.1504742" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11669666/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39726960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nasirian%20M,%20Mobini-Dehkordi%20M,%20Khosravian%20P.%20The%20native%20Iranian%20soil%20bacteria%20with%20high%20potential%20to%20produce%20extracellular%20methionine%20gamma-lyase.%20Front%20Microbiol.%202024;15:1504742." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Sato D, Nozaki T. Methionine gamma-lyase: the unique reaction mechanism, physiological roles, and therapeutic applications against infectious diseases and cancers. 61, IUBMB Life. 2009. pp. 1019–28.</cite> [<a href="https://doi.org/10.1002/iub.255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19859976/" class="usa-link">PubMed</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Odaa NH. Cytotoxicity of purified methionine γ-lyase produced by Pseudomonas putida on several cell lines. Iraqi J Cancer Med Genet. 2013;6(1).</cite>
</li>
<li id="CR21">
<span class="label">21.</span><cite>Sharma B, Devi S, Kumar R, Kanwar SS. Screening, characterization and anti-cancer application of purified intracellular MGL. Int J Biol Macromol. 2022;217:96–110.
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2022.07.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35817235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sharma%20B,%20Devi%20S,%20Kumar%20R,%20Kanwar%20SS.%20Screening,%20characterization%20and%20anti-cancer%20application%20of%20purified%20intracellular%20MGL.%20Int%20J%20Biol%20Macromol.%202022;217:96%E2%80%93110." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Hendy MH, Hashem AH, Sulieman WB, Sultan MH, Abdelraof M, Purification. Characterization and anticancer activity of L-methionine γ-lyase from thermo-tolerant Aspergillus fumigatus. Microb Cell Fact. 2023;22(1).</cite> [<a href="https://doi.org/10.1186/s12934-023-02019-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9837997/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36635695/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Kulikova VV, Morozova EA, Revtovich SV, Kotlov MI, Anufrieva NV, Bazhulina NP, et al. Gene cloning, characterization, and cytotoxic activity of methionine γ-lyase from <em>Clostridium novyi</em>. IUBMB Life. 2017;69(9):668–76.</cite> [<a href="https://doi.org/10.1002/iub.1649" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28681503/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>El-Sayed AS, Shouman SA, Nassrat HM. Pharmacokinetics, immunogenicity and anticancer efficiency of <em>Aspergillus flavipes</em> L-methioninase. Enzyme Microb Technol. 2012;51(4):200–10.
</cite> [<a href="https://doi.org/10.1016/j.enzmictec.2012.06.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22883554/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?El-Sayed%20AS,%20Shouman%20SA,%20Nassrat%20HM.%20Pharmacokinetics,%20immunogenicity%20and%20anticancer%20efficiency%20of%20Aspergillus%20flavipes%20L-methioninase.%20Enzyme%20Microb%20Technol.%202012;51(4):200%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Bradford MM. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal Biochem. 1976;72(1–2):248–54.
</cite> [<a href="https://doi.org/10.1016/0003-2697(76)90527-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/942051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bradford%20MM.%20A%20rapid%20and%20sensitive%20method%20for%20the%20quantitation%20of%20microgram%20quantities%20of%20protein%20utilizing%20the%20principle%20of%20protein-dye%20binding.%20Anal%20Biochem.%201976;72(1%E2%80%932):248%E2%80%9354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Srivastava N, Singh P. Degradation of toxic pollutants from pulp &amp; paper mill effluent. Discovery. 2015;40(183):221–7.</cite> [<a href="https://scholar.google.com/scholar_lookup?Srivastava%20N,%20Singh%20P.%20Degradation%20of%20toxic%20pollutants%20from%20pulp%20&amp;%20paper%20mill%20effluent.%20Discovery.%202015;40(183):221%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Maijala P, Kleen M, Westin C, Poppius-Levlin K, Herranen K, Lehto JH, et al. Biomechanical pulping of softwood with enzymes and white-rot fungus <em>physisporinus rivulosus</em>. Enzyme Microb Technol. 2008;43(2):169–77.</cite> [<a href="https://scholar.google.com/scholar_lookup?Maijala%20P,%20Kleen%20M,%20Westin%20C,%20Poppius-Levlin%20K,%20Herranen%20K,%20Lehto%20JH,%20et%20al.%20Biomechanical%20pulping%20of%20softwood%20with%20enzymes%20and%20white-rot%20fungus%20physisporinus%20rivulosus.%20Enzyme%20Microb%20Technol.%202008;43(2):169%E2%80%9377." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Rao MB, Tanksale AM, Ghatge MS, Deshpande VV. Molecular and biotechnological aspects of microbial proteases. Microbiol Mol Biol Rev. 1998;62(3):597–635.
</cite> [<a href="https://doi.org/10.1128/mmbr.62.3.597-635.1998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC98927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9729602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rao%20MB,%20Tanksale%20AM,%20Ghatge%20MS,%20Deshpande%20VV.%20Molecular%20and%20biotechnological%20aspects%20of%20microbial%20proteases.%20Microbiol%20Mol%20Biol%20Rev.%201998;62(3):597%E2%80%93635." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Choudhary RB, Jana AK, Jha MK. Enzyme technology applications in leather processing. 2004.</cite>
</li>
<li id="CR30">
<span class="label">30.</span><cite>Araujo R, Casal M, Cavaco-Paulo A. Application of enzymes for textile fibres processing. Biocatal Biotransform. 2008;26(5):332–49.</cite> [<a href="https://scholar.google.com/scholar_lookup?Araujo%20R,%20Casal%20M,%20Cavaco-Paulo%20A.%20Application%20of%20enzymes%20for%20textile%20fibres%20processing.%20Biocatal%20Biotransform.%202008;26(5):332%E2%80%9349." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Kuhad RC, Gupta R, Singh A. Microbial cellulases and their industrial applications. Enzyme Res. 2011;2011(1):280696.
</cite> [<a href="https://doi.org/10.4061/2011/280696" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3168787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21912738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuhad%20RC,%20Gupta%20R,%20Singh%20A.%20Microbial%20cellulases%20and%20their%20industrial%20applications.%20Enzyme%20Res.%202011;2011(1):280696." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Keshwani A, Malhotra B, Kharkwal H. Natural polymer based detergents for stain removal. World J Pharm Pharm Sci. 2015;4(4):490–508.</cite> [<a href="https://scholar.google.com/scholar_lookup?Keshwani%20A,%20Malhotra%20B,%20Kharkwal%20H.%20Natural%20polymer%20based%20detergents%20for%20stain%20removal.%20World%20J%20Pharm%20Pharm%20Sci.%202015;4(4):490%E2%80%93508." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Pio TF, Macedo GA. Cutinases:: properties and industrial applications. Adv Appl Microbiol. 2009;66:77–95.
</cite> [<a href="https://doi.org/10.1016/S0065-2164(08)00804-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19203649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pio%20TF,%20Macedo%20GA.%20Cutinases::%20properties%20and%20industrial%20applications.%20Adv%20Appl%20Microbiol.%202009;66:77%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Hasan F, Shah AA, Javed S, Hameed A. Enzymes used in detergents: lipases. Afr J Biotechnol. 2010;9(31):4836–44.</cite> [<a href="https://scholar.google.com/scholar_lookup?Hasan%20F,%20Shah%20AA,%20Javed%20S,%20Hameed%20A.%20Enzymes%20used%20in%20detergents:%20lipases.%20Afr%20J%20Biotechnol.%202010;9(31):4836%E2%80%9344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Baldwin R. Structure and mechanism in protein science. A guide to enzyme catalysis and protein folding, by A. Fersht. 1999. New York: Freeman. 631 pp. $67.95 (hardcover). Protein Science. 2000;9:207.</cite>
</li>
<li id="CR36">
<span class="label">36.</span><cite>Salim N, Santhiagu A, Joji K. Purification, characterization and anticancer evaluation of l-methioninase from <em>Trichoderma harzianum</em>. 3 Biotech. 2020;10(11):501.
</cite> [<a href="https://doi.org/10.1007/s13205-020-02494-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7606426/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33163320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Salim%20N,%20Santhiagu%20A,%20Joji%20K.%20Purification,%20characterization%20and%20anticancer%20evaluation%20of%20l-methioninase%20from%20Trichoderma%20harzianum.%203%20Biotech.%202020;10(11):501." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Hamed AA, Khedr M, Abdelraof M. Activation of <em>LacZ</em> gene in <em>Escherichia coli</em> DH5α via α-complementation mechanism for β-galactosidase production and its biochemical characterizations. J Genetic Eng Biotechnol. 2020;18(1):80.</cite> [<a href="https://doi.org/10.1186/s43141-020-00096-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7710787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33263861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hamed%20AA,%20Khedr%20M,%20Abdelraof%20M.%20Activation%20of%20LacZ%20gene%20in%20Escherichia%20coli%20DH5%CE%B1%20via%20%CE%B1-complementation%20mechanism%20for%20%CE%B2-galactosidase%20production%20and%20its%20biochemical%20characterizations.%20J%20Genetic%20Eng%20Biotechnol.%202020;18(1):80." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>El-Sayed ASA. Purification and characterization of a new L-methioninase from solid cultures of <em>Aspergillus flavipes</em>. J Microbiol. 2011;49:130–40.
</cite> [<a href="https://doi.org/10.1007/s12275-011-0259-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21369990/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?El-Sayed%20ASA.%20Purification%20and%20characterization%20of%20a%20new%20L-methioninase%20from%20solid%20cultures%20of%20Aspergillus%20flavipes.%20J%20Microbiol.%202011;49:130%E2%80%9340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Abu-Tahon MA, Isaac GS. Comparative study of a new alkaline L-methioninase production by Aspergillus Ustus AUMC 10151 in submerged and solid-state fermentation. Brazilian Archives Biology Technol. 2016;59:e16150484.</cite> [<a href="https://scholar.google.com/scholar_lookup?Abu-Tahon%20MA,%20Isaac%20GS.%20Comparative%20study%20of%20a%20new%20alkaline%20L-methioninase%20production%20by%20Aspergillus%20Ustus%20AUMC%2010151%20in%20submerged%20and%20solid-state%20fermentation.%20Brazilian%20Archives%20Biology%20Technol.%202016;59:e16150484." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Goyer A, Collakova E, Shachar-Hill Y, Hanson AD. Functional characterization of a methionine γ-lyase in Arabidopsis and its implication in an alternative to the reverse trans-sulfuration pathway. Plant Cell Physiol. 2007;48(2):232–42.
</cite> [<a href="https://doi.org/10.1093/pcp/pcl055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17169919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goyer%20A,%20Collakova%20E,%20Shachar-Hill%20Y,%20Hanson%20AD.%20Functional%20characterization%20of%20a%20methionine%20%CE%B3-lyase%20in%20Arabidopsis%20and%20its%20implication%20in%20an%20alternative%20to%20the%20reverse%20trans-sulfuration%20pathway.%20Plant%20Cell%20Physiol.%202007;48(2):232%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Huang KY, Hu HY, Tang YL, Xia FG, Luo XQ, Liu JZ. High-level expression, purification and large-scale production of L-Methionine γ-lyase from <em>Idiomarina</em> as a novel anti-leukemic drug. Mar Drugs. 2015;13(8):5492–507.
</cite> [<a href="https://doi.org/10.3390/md13085492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4557031/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26308011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20KY,%20Hu%20HY,%20Tang%20YL,%20Xia%20FG,%20Luo%20XQ,%20Liu%20JZ.%20High-level%20expression,%20purification%20and%20large-scale%20production%20of%20L-Methionine%20%CE%B3-lyase%20from%20Idiomarina%20as%20a%20novel%20anti-leukemic%20drug.%20Mar%20Drugs.%202015;13(8):5492%E2%80%93507." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Dias B, Weimer B. Conversion of methionine to thiols by lactococci, lactobacilli, and brevibacteria. Appl Environ Microbiol. 1998;64(9):3320–6.
</cite> [<a href="https://doi.org/10.1128/aem.64.9.3320-3326.1998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC106727/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9726877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dias%20B,%20Weimer%20B.%20Conversion%20of%20methionine%20to%20thiols%20by%20lactococci,%20lactobacilli,%20and%20brevibacteria.%20Appl%20Environ%20Microbiol.%201998;64(9):3320%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Pinnamaneni R, Gangula S, Koona S, Potti R. Isolation screening and assaying of methioninase of <em>Brevibacterium linens</em>. Int J Sci Nat. 2012;3(4):773–9.</cite> [<a href="https://scholar.google.com/scholar_lookup?Pinnamaneni%20R,%20Gangula%20S,%20Koona%20S,%20Potti%20R.%20Isolation%20screening%20and%20assaying%20of%20methioninase%20of%20Brevibacterium%20linens.%20Int%20J%20Sci%20Nat.%202012;3(4):773%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Selim MH, Karm Eldin EZ, Saad MM, Mostafa ESE, Shetia YH, Anise AAH. Purification, characterization of L-methioninase from Candida tropicalis, and its application as an anticancer. Biotechnol Res Int. 2015;2015. </cite> [<a href="https://doi.org/10.1155/2015/173140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4672112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26691554/" class="usa-link">PubMed</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>El Awady ME, Selim MS, Abd El-Razek AS, Asker M. Production, purification and characterization of L-methioninase from streptomyces variabilis 3MA2016. Res J Pharm Biol Chem Sci. 2017;8(3):906–21.</cite> [<a href="https://scholar.google.com/scholar_lookup?El%20Awady%20ME,%20Selim%20MS,%20Abd%20El-Razek%20AS,%20Asker%20M.%20Production,%20purification%20and%20characterization%20of%20L-methioninase%20from%20streptomyces%20variabilis%203MA2016.%20Res%20J%20Pharm%20Biol%20Chem%20Sci.%202017;8(3):906%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>El-Sayed ASA, Ruff LE, Ghany SEA, Ali GS, Esener S. Molecular and spectroscopic characterization of <em>A spergillus flavipes</em> and <em>Pseudomonas putida</em> L-methionine γ-lyase in vitro. Appl Biochem Biotechnol. 2017;181:1513–32.
</cite> [<a href="https://doi.org/10.1007/s12010-016-2299-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27796875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?El-Sayed%20ASA,%20Ruff%20LE,%20Ghany%20SEA,%20Ali%20GS,%20Esener%20S.%20Molecular%20and%20spectroscopic%20characterization%20of%20A%20spergillus%20flavipes%20and%20Pseudomonas%20putida%20L-methionine%20%CE%B3-lyase%20in%20vitro.%20Appl%20Biochem%20Biotechnol.%202017;181:1513%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Manukhov IV, Mamaeva DV, Morozova EA, Rastorguev SM, Faleev NG, Demidkina TV, et al. L-methionine γ-lyase from <em>Citrobacter freundii</em>: cloning of the gene and kinetic parameters of the enzyme. Biochemistry (Moscow). 2006;71:361–9.
</cite> [<a href="https://doi.org/10.1134/s0006297906040031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16615855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manukhov%20IV,%20Mamaeva%20DV,%20Morozova%20EA,%20Rastorguev%20SM,%20Faleev%20NG,%20Demidkina%20TV,%20et%20al.%20L-methionine%20%CE%B3-lyase%20from%20Citrobacter%20freundii:%20cloning%20of%20the%20gene%20and%20kinetic%20parameters%20of%20the%20enzyme.%20Biochemistry%20(Moscow).%202006;71:361%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Selim MH, Elshikh HH, El-Hadedy DE, Saad MM, Eliwa E, Abdelraof M. L-methioninase from some streptomyces isolates I: isolation, identification of best producers and some properties of the crude enzyme produced. J Genetic Eng Biotechnol. 2015;13(2):129–37.</cite> [<a href="https://doi.org/10.1016/j.jgeb.2015.08.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6299813/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30647576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Selim%20MH,%20Elshikh%20HH,%20El-Hadedy%20DE,%20Saad%20MM,%20Eliwa%20E,%20Abdelraof%20M.%20L-methioninase%20from%20some%20streptomyces%20isolates%20I:%20isolation,%20identification%20of%20best%20producers%20and%20some%20properties%20of%20the%20crude%20enzyme%20produced.%20J%20Genetic%20Eng%20Biotechnol.%202015;13(2):129%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Kreis W, Hession C. Isolation and purification of l-methionine-α-deamino-γ-mercaptomethane-lyase (l-methioninase) from <em>clostridium sporogenes</em>. Cancer Res. 1973;33(8):1862–5.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/4720797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kreis%20W,%20Hession%20C.%20Isolation%20and%20purification%20of%20l-methionine-%CE%B1-deamino-%CE%B3-mercaptomethane-lyase%20(l-methioninase)%20from%20clostridium%20sporogenes.%20Cancer%20Res.%201973;33(8):1862%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Tanaka H, Esaki N, Yamamoto T, Soda K. Purification and properties of methioninase from <em>Pseudomonas ovalis</em>. FEBS Lett. 1976;66(2):307–11.
</cite> [<a href="https://doi.org/10.1016/0014-5793(76)80528-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/955094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tanaka%20H,%20Esaki%20N,%20Yamamoto%20T,%20Soda%20K.%20Purification%20and%20properties%20of%20methioninase%20from%20Pseudomonas%20ovalis.%20FEBS%20Lett.%201976;66(2):307%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Martínez-Cuesta MC, Peláez C, Eagles J, Gasson MJ, Requena T, Hanniffy SB. YtjE from <em>Lactococcus lactis</em> IL1403 is a CS lyase with α, γ-elimination activity toward methionine. Appl Environ Microbiol. 2006;72(7):4878–84.
</cite> [<a href="https://doi.org/10.1128/AEM.00712-06" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1489307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16820483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mart%C3%ADnez-Cuesta%20MC,%20Pel%C3%A1ez%20C,%20Eagles%20J,%20Gasson%20MJ,%20Requena%20T,%20Hanniffy%20SB.%20YtjE%20from%20Lactococcus%20lactis%20IL1403%20is%20a%20CS%20lyase%20with%20%CE%B1,%20%CE%B3-elimination%20activity%20toward%20methionine.%20Appl%20Environ%20Microbiol.%202006;72(7):4878%E2%80%9384." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Mosca L, Pagano M, Pecoraro A, Borzacchiello L, Mele L, Cacciapuoti G, et al. S-adenosyl-L-methionine overcomes uL3-mediated drug resistance in p53 deleted colon cancer cells. Int J Mol Sci. 2020;22(1): 103.
</cite> [<a href="https://doi.org/10.3390/ijms22010103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7795960/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33374288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mosca%20L,%20Pagano%20M,%20Pecoraro%20A,%20Borzacchiello%20L,%20Mele%20L,%20Cacciapuoti%20G,%20et%20al.%20S-adenosyl-L-methionine%20overcomes%20uL3-mediated%20drug%20resistance%20in%20p53%20deleted%20colon%20cancer%20cells.%20Int%20J%20Mol%20Sci.%202020;22(1):%20103." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Tan Y, Xu M, Hoffman RM. Broad selective efficacy of recombinant methioninase and polyethylene glycol-modified recombinant methioninase on cancer cells in vitro. Anticancer Res. 2010;30(4):1041–6.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/20530407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tan%20Y,%20Xu%20M,%20Hoffman%20RM.%20Broad%20selective%20efficacy%20of%20recombinant%20methioninase%20and%20polyethylene%20glycol-modified%20recombinant%20methioninase%20on%20cancer%20cells%20in%20vitro.%20Anticancer%20Res.%202010;30(4):1041%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Gay F, Aguera K, Sénéchal K, Tainturier A, Berlier W, Maucort-Boulch D, et al. Methionine tumor starvation by erythrocyte‐encapsulated methionine gamma‐lyase activity controlled with per Os vitamin B6. Cancer Med. 2017;6(6):1437–52.
</cite> [<a href="https://doi.org/10.1002/cam4.1086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5463067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28544589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gay%20F,%20Aguera%20K,%20S%C3%A9n%C3%A9chal%20K,%20Tainturier%20A,%20Berlier%20W,%20Maucort-Boulch%20D,%20et%20al.%20Methionine%20tumor%20starvation%20by%20erythrocyte%E2%80%90encapsulated%20methionine%20gamma%E2%80%90lyase%20activity%20controlled%20with%20per%20Os%20vitamin%20B6.%20Cancer%20Med.%202017;6(6):1437%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>El-Sayed ASA. L‐methioninase production by <em>Aspergillus flavipes</em> under solid‐state fermentation. J Basic Microbiol. 2009;49(4):331–41.
</cite> [<a href="https://doi.org/10.1002/jobm.200800318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19455514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?El-Sayed%20ASA.%20L%E2%80%90methioninase%20production%20by%20Aspergillus%20flavipes%20under%20solid%E2%80%90state%20fermentation.%20J%20Basic%20Microbiol.%202009;49(4):331%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Liteplo RG, Hipwell SE, Rosenblatt DS, Sillaots S, Lue-Shing H. Changes in cobalamin metabolism are associated with the altered methionine auxotrophy of highly growth autonomous human melanoma cells. J Cell Physiol. 1991;149(2):332–8.
</cite> [<a href="https://doi.org/10.1002/jcp.1041490222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1748723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liteplo%20RG,%20Hipwell%20SE,%20Rosenblatt%20DS,%20Sillaots%20S,%20Lue-Shing%20H.%20Changes%20in%20cobalamin%20metabolism%20are%20associated%20with%20the%20altered%20methionine%20auxotrophy%20of%20highly%20growth%20autonomous%20human%20melanoma%20cells.%20J%20Cell%20Physiol.%201991;149(2):332%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Hoffman RM. Altered methionine metabolism and transmethylation in cancer. Anticancer Res. 1985;5(1):1–30.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/3888043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoffman%20RM.%20Altered%20methionine%20metabolism%20and%20transmethylation%20in%20cancer.%20Anticancer%20Res.%201985;5(1):1%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Qoura LA, Balakin KV, Hoffman RM, Pokrovsky VS. The potential of methioninase for cancer treatment. Biochimica et Biophysica Acta (BBA). 2024. 10.1016/j.bbcan.2024.189122.</cite> [<a href="https://doi.org/10.1016/j.bbcan.2024.189122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38796027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qoura%20LA,%20Balakin%20KV,%20Hoffman%20RM,%20Pokrovsky%20VS.%20The%20potential%20of%20methioninase%20for%20cancer%20treatment.%20Biochimica%20et%20Biophysica%20Acta%20(BBA).%202024.%2010.1016/j.bbcan.2024.189122." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Davis CD, Uthus EO. DNA methylation, cancer susceptibility, and nutrient interactions. Exp Biol Med. 2004;229(10):988–95.</cite> [<a href="https://doi.org/10.1177/153537020422901002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15522834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Davis%20CD,%20Uthus%20EO.%20DNA%20methylation,%20cancer%20susceptibility,%20and%20nutrient%20interactions.%20Exp%20Biol%20Med.%202004;229(10):988%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>İpek SL, Özdemir MD, Göktürk D. Cytotoxic effect of L-methioninase from Brevibacterium linens BL2 in combination with Etoposide against glioblastoma cells. Appl Sci. 2023;13(16):9382.</cite> [<a href="https://scholar.google.com/scholar_lookup?%C4%B0pek%20SL,%20%C3%96zdemir%20MD,%20G%C3%B6kt%C3%BCrk%20D.%20Cytotoxic%20effect%20of%20L-methioninase%20from%20Brevibacterium%20linens%20BL2%20in%20combination%20with%20Etoposide%20against%20glioblastoma%20cells.%20Appl%20Sci.%202023;13(16):9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Burlacu A. Regulation of apoptosis by Bcl-2 family proteins. J Cell Mol Med. 2003;7(3):249–57.
</cite> [<a href="https://doi.org/10.1111/j.1582-4934.2003.tb00225.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6741335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14594549/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Burlacu%20A.%20Regulation%20of%20apoptosis%20by%20Bcl-2%20family%20proteins.%20J%20Cell%20Mol%20Med.%202003;7(3):249%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Brunelle JK, Letai A. Control of mitochondrial apoptosis by the BCL-2 family. J Cell Sci. 2009;122(4):437–41.
</cite> [<a href="https://doi.org/10.1242/jcs.031682" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2714431/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19193868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brunelle%20JK,%20Letai%20A.%20Control%20of%20mitochondrial%20apoptosis%20by%20the%20BCL-2%20family.%20J%20Cell%20Sci.%202009;122(4):437%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Alotaibi MR, Hassan ZK, Al-Rejaie SS, Alshammari MA, Almutairi MM, Alhoshani AR, et al. Characterization of apoptosis in a breast cancer cell line after IL-10 silencing. Asian Pac J Cancer Prev. 2018;19(3): 777.
</cite> [<a href="https://doi.org/10.22034/APJCP.2018.19.3.777" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5980855/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29582634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alotaibi%20MR,%20Hassan%20ZK,%20Al-Rejaie%20SS,%20Alshammari%20MA,%20Almutairi%20MM,%20Alhoshani%20AR,%20et%20al.%20Characterization%20of%20apoptosis%20in%20a%20breast%20cancer%20cell%20line%20after%20IL-10%20silencing.%20Asian%20Pac%20J%20Cancer%20Prev.%202018;19(3):%20777." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets used and/or analyzed during the current study are available from the corresponding author on reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Cancer are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12885-025-14754-0"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12885_2025_Article_14754.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366409/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366409/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366409%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366409/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366409/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366409/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830757/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366409/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830757/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366409/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366409/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="MaslLyNqJlWhpQBKZQFfLKWXj70qPdsoks5M9meNk4SInUOCt4J31fWrjrR9cGEj">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
