
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Gymnosperm-specific CYP90Js enable biflavonoid biosynthesis and microbial production of amentoflavone - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A2288AF1E33305A22800662BD7B2.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370966/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Gymnosperm-specific CYP90Js enable biflavonoid biosynthesis and microbial production of amentoflavone">
<meta name="citation_author" content="Xue-Hui Dai">
<meta name="citation_author_institution" content="Laboratory of Synthetic Biology, State Key Laboratory of Plant Trait Design, CAS Center for Excellence in Molecular Plant Sciences, Institute of Plant Physiology and Ecology, Chinese Academy of Sciences, Shanghai, China">
<meta name="citation_author_institution" content="State Key Laboratory of Bioreactor Engineering, School of Biotechnology, East China University of Science and Technology, Shanghai, China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Jiang-Ming Zhu">
<meta name="citation_author_institution" content="Laboratory of Synthetic Biology, State Key Laboratory of Plant Trait Design, CAS Center for Excellence in Molecular Plant Sciences, Institute of Plant Physiology and Ecology, Chinese Academy of Sciences, Shanghai, China">
<meta name="citation_author_institution" content="State Key Laboratory of Bioreactor Engineering, School of Biotechnology, East China University of Science and Technology, Shanghai, China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Guang-Yi Wang">
<meta name="citation_author_institution" content="Laboratory of Synthetic Biology, State Key Laboratory of Plant Trait Design, CAS Center for Excellence in Molecular Plant Sciences, Institute of Plant Physiology and Ecology, Chinese Academy of Sciences, Shanghai, China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Yu-Hong Ren">
<meta name="citation_author_institution" content="State Key Laboratory of Bioreactor Engineering, School of Biotechnology, East China University of Science and Technology, Shanghai, China">
<meta name="citation_author" content="Hai-Li Liu">
<meta name="citation_author_institution" content="Laboratory of Synthetic Biology, State Key Laboratory of Plant Trait Design, CAS Center for Excellence in Molecular Plant Sciences, Institute of Plant Physiology and Ecology, Chinese Academy of Sciences, Shanghai, China">
<meta name="citation_author_institution" content="State Key Laboratory of Bioreactor Engineering, School of Biotechnology, East China University of Science and Technology, Shanghai, China">
<meta name="citation_author" content="Yong Wang">
<meta name="citation_author_institution" content="Laboratory of Synthetic Biology, State Key Laboratory of Plant Trait Design, CAS Center for Excellence in Molecular Plant Sciences, Institute of Plant Physiology and Ecology, Chinese Academy of Sciences, Shanghai, China">
<meta name="citation_author_institution" content="State Key Laboratory of Bioreactor Engineering, School of Biotechnology, East China University of Science and Technology, Shanghai, China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7792">
<meta name="citation_doi" content="10.1038/s41467-025-62990-6">
<meta name="citation_pmid" content="40841362">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370966/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370966/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370966/pdf/41467_2025_Article_62990.pdf">
<meta name="description" content="Biflavonoids, a unique subclass of flavonoids with superior clinical activity compared to their monomeric counterparts, offer distinct therapeutic benefits by targeting multiple pathways in neurodegenerative disorders. However, the mechanism of ...">
<meta name="og:title" content="Gymnosperm-specific CYP90Js enable biflavonoid biosynthesis and microbial production of amentoflavone">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Biflavonoids, a unique subclass of flavonoids with superior clinical activity compared to their monomeric counterparts, offer distinct therapeutic benefits by targeting multiple pathways in neurodegenerative disorders. However, the mechanism of ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370966/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370966">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62990-6"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62990.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370966%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370966/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370966/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370966/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7792. doi: <a href="https://doi.org/10.1038/s41467-025-62990-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62990-6</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Gymnosperm-specific CYP90Js enable biflavonoid biosynthesis and microbial production of amentoflavone</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dai%20XH%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Xue-Hui Dai</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xue-Hui Dai</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Synthetic Biology, State Key Laboratory of Plant Trait Design, CAS Center for Excellence in Molecular Plant Sciences, Institute of Plant Physiology and Ecology, Chinese Academy of Sciences, Shanghai, China </div>
<div class="p">
<sup>2</sup>State Key Laboratory of Bioreactor Engineering, School of Biotechnology, East China University of Science and Technology, Shanghai, China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dai%20XH%22%5BAuthor%5D" class="usa-link"><span class="name western">Xue-Hui Dai</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20JM%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jiang-Ming Zhu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jiang-Ming Zhu</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Synthetic Biology, State Key Laboratory of Plant Trait Design, CAS Center for Excellence in Molecular Plant Sciences, Institute of Plant Physiology and Ecology, Chinese Academy of Sciences, Shanghai, China </div>
<div class="p">
<sup>2</sup>State Key Laboratory of Bioreactor Engineering, School of Biotechnology, East China University of Science and Technology, Shanghai, China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20JM%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiang-Ming Zhu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20GY%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Guang-Yi Wang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Guang-Yi Wang</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Synthetic Biology, State Key Laboratory of Plant Trait Design, CAS Center for Excellence in Molecular Plant Sciences, Institute of Plant Physiology and Ecology, Chinese Academy of Sciences, Shanghai, China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20GY%22%5BAuthor%5D" class="usa-link"><span class="name western">Guang-Yi Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ren%20YH%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yu-Hong Ren</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yu-Hong Ren</span></h3>
<div class="p">
<sup>2</sup>State Key Laboratory of Bioreactor Engineering, School of Biotechnology, East China University of Science and Technology, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ren%20YH%22%5BAuthor%5D" class="usa-link"><span class="name western">Yu-Hong Ren</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20HL%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Hai-Li Liu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Hai-Li Liu</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Synthetic Biology, State Key Laboratory of Plant Trait Design, CAS Center for Excellence in Molecular Plant Sciences, Institute of Plant Physiology and Ecology, Chinese Academy of Sciences, Shanghai, China </div>
<div class="p">
<sup>2</sup>State Key Laboratory of Bioreactor Engineering, School of Biotechnology, East China University of Science and Technology, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20HL%22%5BAuthor%5D" class="usa-link"><span class="name western">Hai-Li Liu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Yong Wang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yong Wang</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Synthetic Biology, State Key Laboratory of Plant Trait Design, CAS Center for Excellence in Molecular Plant Sciences, Institute of Plant Physiology and Ecology, Chinese Academy of Sciences, Shanghai, China </div>
<div class="p">
<sup>2</sup>State Key Laboratory of Bioreactor Engineering, School of Biotechnology, East China University of Science and Technology, Shanghai, China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yong Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Laboratory of Synthetic Biology, State Key Laboratory of Plant Trait Design, CAS Center for Excellence in Molecular Plant Sciences, Institute of Plant Physiology and Ecology, Chinese Academy of Sciences, Shanghai, China </div>
<div id="Aff2">
<sup>2</sup>State Key Laboratory of Bioreactor Engineering, School of Biotechnology, East China University of Science and Technology, Shanghai, China </div>
<div id="Aff3">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 1; Accepted 2025 Jul 31; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370966  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841362/" class="usa-link">40841362</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Biflavonoids, a unique subclass of flavonoids with superior clinical activity compared to their monomeric counterparts, offer distinct therapeutic benefits by targeting multiple pathways in neurodegenerative disorders. However, the mechanism of flavonoid dimerization in plants remains enigmatic. Here, we identify CYP90J orthologs as the missing link in biflavonoid biosynthesis. We demonstrate that gymnosperm-specific CYP90Js catalyze intermolecular C–C bond formation in the biosynthesis of biaryl natural products. Together with the identified <em>O</em>-methyltransferases, CYP90Js are responsible for the production of ginkgo biflavonoids. Phylogenetic analysis reveals that the CYP90J subfamily evolved from CYP90E of lycophytes and is found exclusively in gymnosperms. Molecular dynamics simulations show that regioselective dimerization of apigenin to amentoflavone is driven by spatial constraints and π–π stacking interactions. QM/MM calculations support a heme-induced diradical coupling mechanism. Notably, the de novo reconstruction of amentoflavone was achieved in an engineered <em>L</em>-tyrosine <em>E. coli</em> strain, with a titer of 4.75 mg/L. Overall, the discovery of CYP90Js represents a crucial step toward understanding flavone dimerization, and the engineering of biflavonoids in microorganism provides a promising biotechnology platform for expanding therapeutic applications of biflavonoids.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Plant molecular biology, Metabolic engineering, Enzyme mechanisms</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">The authors show that gymnosperm-specific CYP90J enzymes catalyze biflavonoid C-C coupling, reveal a diradical-based mechanism, and achieve microbial production of amentoflavone via synthetic biology.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">The global surge in neurodegenerative disorders, projected to affect 139 million people by 2050<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>, urgently demands innovative therapeutic approaches. Recent breakthroughs in drug development have highlighted natural biflavonoids as promising candidates, particularly for their unique ability to engage multiple therapeutic targets<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Among various biflavonoids, amentoflavone-type derivatives, characterized by a C3’, C8”-linked asymmetric skeleton, have emerged as leads due to their superior drug-like properties and robust target engagement<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Notably, these derivatives exhibited the most potent inhibitory activity against amyloid-β peptide (Aβ40) aggregation with IC<sub>50</sub> values ranging from 5 to 10 μM<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, a hallmark pathological feature of Alzheimer’s disease. Moreover, the regioselective formation of the intermolecular C–C bonds is crucial for achieving their distinct therapeutic benefits<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>.</p>
<p id="Par4">A prominent clinical application of amentoflavone-type biflavonoids is EGb 761, a standardized <em>Ginkgo biloba</em> leaf extract known for its cognitive-enhancing and neuroprotective properties<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Its clinical efficacy in treating mild cognitive impairment was demonstrated in a 24-week randomized controlled trial, showing significant improvement in neuropsychiatric symptoms and cognitive abilities compared to placebo<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. The key active components, including amentoflavone (<strong>1</strong>), bilobetin (<strong>2</strong>), ginkgetin (<strong>3</strong>), and isoginkgetin (<strong>4</strong>), account for 0.047% to 1.68% of the extract and are responsible for its multi-target therapeutic effects<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. However, despite their promising potential, the broader clinical application of these compounds is severely limited by their scarce natural abundance (typical isolation yield &lt;0.001% from plant sources)<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, creating a critical bottleneck in drug development and highlighting the urgent need for efficient synthetic strategies.</p>
<p id="Par5">To address such limitations, chemical synthesis of biflavonoids has been explored as a potential strategy. Nevertheless, these approaches require high temperatures, strong oxidants, and multi-step procedures, leading to low-to-moderate yields and limited practical applications<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. In contrast, biosynthetic technology has emerged as a promising alternative, offering high efficiency and selectivity under mild reaction conditions for cost-effective biflavonoid production<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Although biflavonoids were discovered nearly a century ago, beginning with the isolation of <strong>3</strong> from <em>G. biloba</em><sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, the enzymes responsible for flavonoid dimerization remain unknown, hindering further development and applications.</p>
<p id="Par6">In the broader context of biaryl natural product biosynthesis<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>, various oxidative enzymes, including peroxidases<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, laccases<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, polyphenol oxidases (PPOs)<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, and cytochromes P450 monooxygenases (CYPs)<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, are known to catalyze cross-coupling reactions. Among them, peroxidases, laccases, and PPOs exhibit limited control over post-oxidation bond formation. In contrast, CYPs mediate highly selective oxidation reactions, including regio- and stereoselective dimerization<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> in the biosynthesis of dinaphthalenes<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, diketopiperazine alkaloids<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, and bicoumarins<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Structural analyses of CYPs reveal diverse substrate-binding pockets crucial for defining substrate specificity<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. This structural diversity enables CYPs to recognize specific substrates and catalyze dimeric reactions with precise connectivity, driven by steric and electronic biases and facilitated by radical-mediated mechanisms<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Although such catalytic activities for CYPs are exclusively identified in bacteria and fungi<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, the widespread presence of biflavonoids in plants suggests that similar enzymatic processes may also operate in the plant kingdom.</p>
<p id="Par7">Here, we report the identification of CYP90J and CYP90E subfamily members, which catalyze the regioselective C–C cross-coupling dimerization of apigenin (<strong>5</strong>) and its related compounds. Unlike previously known CYP90 family members that primarily catalyze the sterols hydroxylation, including canonical brassinosteroids<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>, our identified CYP90J and CYP90E members demonstrate distinct catalytic functions. Our findings link the presence of CYP90J orthologs to the biflavonoids in plants. Through comprehensive QM/MM calculations, we reveal that GbCYP90J6 catalyzes the intermolecular C–C bond formation via an energetically favorable diradical coupling mechanism, with key active site residues stabilizing intermediates and directing regioselective coupling. The subsequent methylation modification pathway of biflavonoids <strong>2</strong>–<strong>4</strong> and <strong>8</strong>–<strong>10</strong> is mediated by GbOMT1-GbOMT5. Based on these findings, we successfully engineer a synthetic pathway in <em>Escherichia coli</em> for de novo production of <strong>1</strong>, achieving a titer of 4.75 mg/L. This study reveals unique plant-derived CYPs that expand our understanding of atypical P450 functions and establishes further insights into the biosynthetic pathways of biflavonoids, providing opportunities for the application of biflavonoids in biomanufacturing and therapeutic development.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Unprecedented GbCYP90J6 drives the dimerization of biflavonoids</h3>
<p id="Par8"><em>G. biloba</em> is rich in 3’,8”-linked amentoflavone-type biflavonoids, including <strong>1</strong>–<strong>4</strong><sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. The recent availability of its nearly complete genome<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> makes it a perfect model for elucidating the molecular mechanism underlining the dimerization of biflavonoids. To identify candidate genes, we examined the tissue-specific distribution of biflavonoids, quantitatively assessing biflavonoids <strong>1</strong>–<strong>4</strong> across different tissues. Our results showed that biflavonoids predominantly accumulated in the leaves, regardless of developmental stage (Fig. <a href="#Fig1" class="usa-link">1a, b</a>). Then, we conducted RNA-Seq analyses using total RNA from various tissues and annotated the transcriptome profiles with the reference genome<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Genes with high leaf-specific expression [fragments per kilo base of transcript per million mapped fragments (FPKM) &gt; 20] were prioritized for further study. Giving the roles of CYPs, peroxidases, laccases, and PPOs in C–C oxidative cross-coupling in biaryl natural products<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, we identified six CYPs (chr1.3004, chr5.1346, chr5.1347, chr12.44, chr12.47, and chr12.764) and three peroxidases (chr2.1867, chr6.552, and chr3.1671) as candidates for functional characterization, while no suitable laccases or PPOs were selected (Fig. <a href="#Fig1" class="usa-link">1c</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Leaf-specific abundant amentoflavone-type biflavonoids and its unique dimerase GbCYP90J6 in <em>G. biloba.</em>
</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370966_41467_2025_62990_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7f81/12370966/00de4b086298/41467_2025_62990_Fig1_HTML.jpg" loading="lazy" id="d33e498" height="484" width="796" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Detailed schematic of plant tissue sampling. YL young leaf, uML unmature leaf, ML mature leaf, YS young shoot, OS old shoot, OR old root, MR main root, FR fibrous root. <strong>b</strong> Dry weight content of amentoflavone (<strong>1</strong>), bilobetin (<strong>2</strong>), ginkgetin (<strong>3</strong>), and isoginkgetin (<strong>4</strong>) in selected tissues of <em>G. biloba</em>. All data represent the means values ± SD; error bars show the standard deviation of three biological replicates. Source data are provided as a Source Data file. <strong>c</strong> Expression patterns of screened peroxidases and CYPs in different organs. The figure displays the values as log<sub>2</sub>(average FPKM of biological replicates). <strong>d</strong> LC-MS/MS analysis of tobacco leave crude extract expressing the chr1.3004 (GbCYP90J6) candidate with the filtration of substrate apigenin (<strong>5</strong>). The empty vector was used as a negative control. The MS<sup>2</sup> spectra of the <strong>1</strong> standard (black) and the product from GbCYP90J6-expressing tobacco (blue) were shown on the right. <strong>e</strong> The dimeric reaction catalyzed by GbCYP90J6 with <strong>5</strong> as substrate.</p></figcaption></figure><p id="Par9">Since <strong>1</strong> is the most primitive C–C type biflavonoid<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> and <strong>5</strong> functions as the essential structural unit, we employed <strong>5</strong> as the substrate for our primary screening. <em>Nicotiana benthamiana</em> (tobacco) was chosen as the initial host due to its suitability for expressing functional CYPs and peroxidases. The full-length cDNAs of the candidate genes were individually cloned into the pEAQ-<em>HT</em> vector and expressed in tobacco leaves using <em>Agrobacterium</em>-mediated transformation. Upon the infiltration of the substrate <strong>5</strong>, only the transient expression of the chr1.3004 protein, designated GbCYP90J6, produced a detectable product (Fig. <a href="#Fig1" class="usa-link">1d</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). Sequential liquid chromatography coupled with high-resolution mass spectrometry (LC-HRMS) analysis revealed that the newly formed product had a molecular weight 2 Da less than twice that of <strong>5</strong> (Fig. <a href="#Fig1" class="usa-link">1d</a>), implying that GbCYP90J6 catalyzed the dimerization of two molecules of <strong>5</strong> to form bi-apigenin (Fig. <a href="#Fig1" class="usa-link">1e</a>). This bi-apigenin exhibited identical retention time and MS<sup>2</sup> spectra as the authentic standard <strong>1</strong> (Fig. <a href="#Fig1" class="usa-link">1d</a>), thereby proving its identification. To eliminate the involvement of native enzymes in tobacco, we conducted assays with <em>N</em>-terminal modified GbCYP90J6 (trGbCYP90J6), heterologously expressed in <em>E. coli</em>, alongside its electron transport chain partner, <em>N</em>-terminal modified CPR from <em>Arabidopsis thaliana</em> (trAtCPR). As in tobacco, <em>E. coli</em> cells carrying trGbCYP90J6 yielded <strong>1</strong> when fed with <strong>5</strong> (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). Further kinetic analysis revealed that GbCYP90J6 had a high affinity for <strong>5</strong>, with a <em>K</em><sub>m</sub> value of 0.4925 μM (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>), and remarkable regioselectivity, catalyzing the formation of 3’,8”-linked biflavonoids. To our knowledge, GbCYP90J6 represented a rare example of a plant-derived CYP enzyme that catalyzed the intermolecular C–C dimerization in the biosynthesis of biaryl natural products.</p>
<p id="Par10">To further investigate the substrate specificity of GbCYP90J6, we designed synthetic routes to assess whether methylated monomers could serve as substrates for dimerization into putative methylated derivatives of <strong>1</strong> (Supplementary Fig. <a href="#MOESM1" class="usa-link">4a</a>). In an <em>E. coli</em> reaction system, incubation of <strong>5</strong> with acacetin (<strong>6</strong>) was intended to generate <strong>2</strong> and <strong>4</strong>, and co-incubation with apigenin 7,4’-dtimethyl ether (<strong>7</strong>) aimed to produce <strong>3</strong>. However, GbCYP90J6 exclusively catalyzed the dimerization of non-methylated <strong>5</strong>, while methoxylated analogs <strong>6</strong> and <strong>7</strong> as substrates were not recognized (Supplementary Fig. <a href="#MOESM1" class="usa-link">4b, c</a>). This observation suggested that the dimerization preceded the methylation of phenol groups in <strong>5</strong>. Moreover, <strong>1</strong> likely served as the key precursor for the biosynthesis of <strong>2</strong>–<strong>4</strong>, consistent with its early evolutionary origin<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>.</p>
<p id="Par11">To validate this, we conducted gene co-expression analysis using <em>GbCYP90J6</em> as a bait gene, revealing candidates for <em>O</em>-methyltransferases (GbOMTs, Fig. <a href="#Fig2" class="usa-link">2a</a>). The resulting <strong>2</strong>, which was modified by <em>O</em>-methyltransferases GbOMT1 and GbOMT2 from <strong>1</strong>, verified this hypothesis (Fig. <a href="#Fig2" class="usa-link">2b</a>). Subsequently, GbOMT3 further modified <strong>2</strong> to produce <strong>3</strong>, while GbOMT4 and GbOMT5 convert <strong>2</strong> into <strong>4</strong> (Fig. <a href="#Fig2" class="usa-link">2c</a>). Beyond <strong>2</strong>–<strong>4</strong>, we identified several naturally occurring <em>O</em>-methylated biflavonoids, including podocarpusflavone A (<strong>8</strong>)<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>, sequoiaflavone (<strong>9</strong>)<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> (Fig. <a href="#Fig2" class="usa-link">2d</a>), with their methylation sites confirmed by NOESY correlations and comparison to authentic standards (Supplementary Figs. <a href="#MOESM1" class="usa-link">5</a>–<a href="#MOESM1" class="usa-link">11</a>). A low-yield product, tentatively identified as 4’,7”-<em>O</em>-methylamentoflavone (<strong>10</strong>)<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> (Fig. <a href="#Fig2" class="usa-link">2e</a>), was also detected. Based on unmodified 7-OH and 4’”-OH groups retained in <strong>3</strong> and <strong>4</strong>, the methylation in <strong>10</strong> was assigned to the 7”-OH position (Fig. <a href="#Fig2" class="usa-link">2f</a>). Interestingly, biochemical characterization revealed that each GbOMT exhibited distinct regioselectivity. GbOMT1 showed preference for the 4’-OH and 7”-OH positions adjacent to the intramolecular C3’-C8” linkage, indicating high activity toward 4’-OH but limited activity toward 7”-OH. GbOMT2 selectively methylated the 4’-OH position, while GbOMT3 selectively methylated the 7-OH position. Both GbOMT4 and GbOMT5 demonstrated selective methylation of the 4”’-OH position (Fig. <a href="#Fig2" class="usa-link">2f</a>). Based on these findings, we concluded that the biosynthetic pathway of amentoflavone-type biflavonoids began with oxidative cross-coupling reactions, followed by subsequent modifications such as regioselective <em>O</em>-methylation mediated by specific GbOMTs (Fig. <a href="#Fig2" class="usa-link">2f</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. The biosynthesis pathway from biflavonoids 1 to its methyl modified derives in <em>G. biloba.</em>
</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370966_41467_2025_62990_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7f81/12370966/69bebb070e7f/41467_2025_62990_Fig2_HTML.jpg" loading="lazy" id="d33e938" height="1000" width="768" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>For GbOMT characterization, crude enzymes generated from BL21(DE3) carrying the empty vector pET28a were used as a negative control. <strong>a</strong> Expression patterns of screened <em>O</em>-methyltransferases in different organs. The figure displayed the values as log<sub>2</sub>FPKM. <strong>b</strong> LC-MS/MS analysis of <em>G. biloba</em> OMTs chr7.385 (GbOMT1) and chr8.315 (GbOMT2) activities on substrates <strong>1</strong> in vitro. GbOMT1 and GbOMT2 catalyzed substrate <strong>1</strong> to produce bilobetin (<strong>2</strong>). MS<sup>2</sup> fragmentation spectra for authentic <strong>2</strong> and products of GbOMT1 and GbOMT2 were shown on the right. <strong>c</strong> LC-MS/MS analysis of <em>G. biloba</em> OMTs chr1.879 (GbOMT3), chr10.1253 (GbOMT4) and chr12.613 (GbOMT5) activities on substrates <strong>2</strong> in vitro. GbOMT3 catalyzed <strong>2</strong> to produce ginkgetin (<strong>3</strong>), GbOMT4 and GbOMT5 catalyzed <strong>2</strong> to produce isoginkgetin (<strong>4</strong>). <strong>d</strong> LC-MS/MS analysis of <em>G. biloba</em> OMTs chr1.879 (GbOMT3), chr10.1253 (GbOMT4), and chr12.613 (GbOMT5) activities on substrate <strong>1</strong> in vitro. GbOMT4 and GbOMT5 catalyzed substrates <strong>1</strong> to podocarpusflavone A (<strong>8</strong>). GbOMT3 catalyzed <strong>1</strong> to sequoiaflavone (<strong>9</strong>). MS<sup>2</sup> fragmentation spectra of products <strong>8</strong> and <strong>9</strong> were shown on the right side. <strong>e</strong> LC-MS/MS analysis of <em>G. biloba</em> OMTs chr7.385 (GbOMT1) activities on substrates <strong>2</strong> in vitro. GbOMT1 catalyzed <strong>2</strong> to <strong>10</strong>, which was speculated as 4,7”-<em>O</em>-methylamentoflavone. <strong>f</strong> The biosynthetic pathway of biflavonoid <strong>1</strong> and its methylated derivatives <strong>2</strong>–<strong>4</strong> and <strong>8</strong>–<strong>10</strong> in <em>G. biloba</em>. GbCYP90J6 dimerized <strong>5</strong> to generate <strong>1</strong>, which served as the common precursor for <strong>2</strong>–<strong>4</strong> and <strong>8</strong>–<strong>10</strong>. GbOMT1 and GbOMT2 catalyze the conversion of <strong>1</strong> to <strong>2</strong>. Subsequently, GbOMT3 can modify <strong>2</strong> to produce <strong>3</strong>, while GbOMT4 and GbOMT5 convert <strong>2</strong> into <strong>4</strong>. <strong>8</strong>–<strong>10</strong> are shunt products.</p></figcaption></figure><p id="Par12">Additionally, we probed the substrate specificity of GbCYP90J6 toward a broader range of flavonoids, as hundreds of C–C linkage biflavonoids were reported to form via flavonoids dimerization<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. This range extended beyond <strong>5</strong> and its methylated derivatives. To this end, we selected a series of unmodified aglycones, including flavanones, flavones, flavonols, and flavanonols, as substrates in the biosynthetic route of flavonoid (Supplementary Fig. <a href="#MOESM1" class="usa-link">12a</a>). LC-HRMS results revealed that GbCYP90J6 catalyzed the dimerization of (<em>S</em>)-naringenin (<strong>11</strong>), a precursor in the biosynthesis of <strong>5</strong>, generating two bi-naringenins <strong>11a</strong> and <strong>11b</strong> (Supplementary Fig. <a href="#MOESM1" class="usa-link">12b</a>), albeit with a conversion rate of less than 3%. Additionally, a trace amount of bi-tricetin <strong>12a</strong> was also detected when using tricetin (<strong>12</strong>) as the substrate (Supplementary Fig. <a href="#MOESM1" class="usa-link">12c</a>), although its exact structure also remains to be elucidated. Nevertheless, <strong>5</strong> remained the preferred substrate for GbCYP90J6. Thus, the CYP90J subfamily likely played a crucial role in flavonoid dimerization, contributing to the formation of biflavonoid scaffolds and sparking further interest in this unique CYP subfamily.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Gymnosperm-specific CYP90J subfamily enzymes specially catalyze the intermolecular C–C dimerization of flavonoids</h3>
<p id="Par13">Phylogenetically, CYP90 members were initially identified in lycophytes<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> and later maintained their presence in ferns, gymnosperms, and angiosperms. To investigate the evolutionary origin of GbCYP90J6, we conducted a systematic phylogenetic analysis of CYP90 proteins across representative species of lycophytes<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>, ferns<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, gymnosperms<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>–<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, and angiosperms<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> using available genome data (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). Additionally, we incorporated known angiosperm CYP90 proteins collected in the Plant Cytochrome P450 database<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. Three additional <em>G. biloba</em> CYP90 family members: chr1.681 (GbCYP90B61), chr9.869 (GbCYP90A68), and chr10.96 (GbCYP90D59) were included, each sharing less than 45% protein sequence identity with GbCYP90J6 (Supplementary Fig. <a href="#MOESM1" class="usa-link">1c</a>). Notably, the three <em>G. biloba</em> CYP90 proteins were dispersed and clustered with functionally known AtCYP90, suggesting their potential typical role in the oxidation of polyhydroxylated steroids<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> (Fig. <a href="#Fig3" class="usa-link">3a</a>). In contrast, GbCYP90J6 clusters within the gymnosperms CYP90J subclade, closely related to lycophytes CYP90E subclade, with no angiosperm CYP90 members in this lineage (Fig. <a href="#Fig3" class="usa-link">3a</a>). This suggested that the CYP90J subfamily may be unique to gymnosperms, and the proximity of lycophyte CYP90E members implied a potential evolutionary relationship between these two subclades. Furthermore, we constructed the phylogenetic tree of GbCYP90 subfamily orthologs in land plants, spanning from bryophytes to gymnosperms (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>), to investigate the evolutionary distribution of GbCYP90 members. Based on this analysis, we extracted relevant orthologs distribution patterns and presented a simplified overview in Fig. <a href="#Fig4" class="usa-link">4</a>. Notably, the phylogenetic relationships revealed a strong correlation between the presence of GbCYP90J6 orthologs and the reported occurrence of biflavonoids (Fig. <a href="#Fig4" class="usa-link">4</a>). GbCYP90J6 orthologs were predominantly found in plant species documented to produce biflavonoids (Supplementary Data <a href="#MOESM4" class="usa-link">2</a>), while they were generally absent in plant species lacking such reports. For instance, biflavonoids and their corresponding orthologous genes are present in mosses but absent in hornworts and liverworts<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. A similar pattern was observed in gymnosperms, such as Gnetales<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> (including <em>Ephedra</em>, <em>Gnetum</em>, and <em>Welwitschia</em>) and Pinaceae<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>, which lacked both biflavonoids and orthologous genes. Although the catalytic function of each ortholog remained to be fully validated, their phylogenetic association with biflavonoid-producing species suggested a potentially conserved role in biflavonoid biosynthesis. We acknowledge, however, that some uncertainty persists in species lacking comprehensive phytochemical characterization. The co-distribution of biflavonoids and GbCYP90J6 orthologs suggested a plausible evolutionary trajectory from mosses to gymnosperms in biflavonoid biosynthesis, with the CYP90J playing a specialized role in biflavonoid synthesis within gymnosperms.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Phylogenetic analysis of the CYP90 subfamily and validation results of dimerase activity of other gymnosperm CYP90J and lycophyte CYP90E members towards substrate 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370966_41467_2025_62990_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7f81/12370966/ad31c0f54e74/41467_2025_62990_Fig3_HTML.jpg" loading="lazy" id="d33e1142" height="931" width="748" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> CYP90s from representative species of lycophytes, ferns, gymnosperms, and angiosperms, along with known CYP90s from the Plant Cytochrome P450 database<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, were selected to construct a phylogenetic tree of plant CYP90s. All sequences used in the phylogenetic analysis were listed in Supplementary Data 1. Protein sequences were aligned using MUSCLE<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup> and then used to construct a Maximum Likelihood (ML) tree with IQ-TREE<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup> with 1000 bootstrap replicates. The resulting tree was visualized using iTOL<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup>. In the tree, the purple clade represented the CYP90J subfamily from gymnosperms, with GbCYP90J6 located within it. The pink clade represented the CYP90E subfamily from lycophytes, and the blue clade represented the known CYP90A-D subfamily that catalyzed the hydroxylation of steroids from angiosperms. Additionally, the green leaf represented three other CYP90 members in <em>Ginkgo biloba</em>: GbCYP90A68, GbCYP90B61, and GbCYP90D59, which were adjacent to known steroid hydroxylases. <strong>b</strong> LC-MS/MS results characterized the dimerase activity of additional CYP90Js from gymnosperms and SmCYP90E3v1 from lycophytes towards apigenin (<strong>5</strong>). Functional characterization was conducted in vivo in <em>E. coli</em> JM109(DE3), with the strain carrying the pFe00 vector used as a negative control. MS<sup>2</sup> fragmentation spectra for authentic amentoflavone (<strong>1</strong>) and products of ChCYP90J7 and SmCYP90E3v1 were shown on the right. All these members were also tested for dimerase activity toward substrate (<em>S</em>)-naringenin (<strong>11</strong>) (Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>), with the results summarized on the right side of <strong>b</strong>.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. The relationship between the distribution of biflavonoids and GbCYP90J6 orthologs in land plants, ranging from bryophytes to gymnosperms.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370966_41467_2025_62990_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7f81/12370966/f5f664625f5c/41467_2025_62990_Fig4_HTML.jpg" loading="lazy" id="d33e1230" height="609" width="750" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Plants containing biflavonoids possess GbCYP90J6 orthologs. These include mosses (<em>Sphagnum fallax</em>, <em>Sphagnum magellanicum</em>, <em>Physcomitrium patens</em>, <em>Ceratodon purpureus</em>, and <em>Pohlia nutans</em>), the lycophyte <em>Selaginella moellendorffii</em>, ferns (<em>Alsophila spinulosa</em> and <em>Adiantum capillus-veneris</em>), and gymnosperms (<em>Ginkgo biloba</em>, <em>Cycas panzhihuaensis</em>, <em>Taxus chinensis</em>, <em>Thuja plicata</em>, <em>Metasequoia glyptostroboides</em>, and <em>Sequoiadendron giganteum</em>). Conversely, in plants lacking biflavonoids, GbCYP90J6 orthologs are absent. These include hornworts (<em>Anthoceros angustus</em>, <em>Anthoceros agrestis,</em> and <em>Anthoceros punctatus</em>), liverworts (<em>Marchantia polymorpha</em>), Pinaceae (<em>Pseudotsuga menziesii</em>, <em>Picea babies,</em> and <em>Pinus teada</em>), and Gnetales (<em>Gnetum montanum</em> and <em>Welwitschia mirabilis</em>). Thus, the distribution of biflavonoids is highly correlated with the presence of GbCYP90J6 orthologs, indicating the important role of these orthologs in the dimerization of biflavonoids. Additionally, we provide the detailed distribution of orthologs in plants in Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a> and detailed plant biflavonoids dispersed information in Supplementary Data <a href="#MOESM4" class="usa-link">2</a>.</p></figcaption></figure><p id="Par14">To determine whether biflavonoid synthesis was common within CYP90J, we characterized additional gymnosperm CYP90J members: TcCYP90J7 from <em>Taxus chinensis</em>, CpCYP90J8 from <em>Cycas panzhihuaensis</em>, ChCYP90J7 from <em>Cephalotaxus harringtonia</em>, MgCYP90J1 from <em>Metasequoia glyptostroboides</em>, and TpCYP90J1v1, TpCYP90J9, and TpCYP90J10 from <em>Thuja plicata</em>. Despite their high sequence similarity (68–71%), their catalytic activities varied. Notably, ChCYP90J7, similar to GbCYP90J6, exhibited dimerization activities that were more inclined towards <strong>5</strong> than <strong>11</strong>, as evidenced by its higher catalytic efficiency (Fig. <a href="#Fig3" class="usa-link">3b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>). In contrast, MgCYP90J1 catalyzed only the dimerization of <strong>11</strong> (Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>), while the other CYP90J members did not show these activities. Given the rich diversity of C–C type biflavonoids in gymnosperms<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>, it was plausible that these CYP90J enzymes participate in the dimerization of additional flavonoid substrates. Additionally, to further elucidate the evolutionary relationship, SmCYP90E3v1 from <em>Selaginella moellendorffii</em> in CYP90E subclade was also chosen for screening. It exhibited weak catalytic activity toward <strong>5</strong> and <strong>11</strong>, producing trace amounts of <strong>1</strong> and <strong>11a</strong> (Fig. <a href="#Fig3" class="usa-link">3b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>). Based on the above observations and the established plant phylogeny, gymnosperm CYP90J subfamily members likely evolved from ancestral lycophyte CYP90Es and subsequently diverged to specialize in biflavonoid synthase. Their substrate scope, however, required further investigation.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Ligand conformation controlled by surrounding amino acids determines enzyme dimerization catalytic activity</h3>
<p id="Par15">Although the unique gymnosperms CYP90Js specialized in flavonoid dimerization, only GbCYP90J6 and ChCYP90J7 catalyzed the dimerization of <strong>5</strong> to <strong>1</strong> via an intermolecular 3’,8”-linkage (Figs. <a href="#Fig1" class="usa-link">1d</a> and <a href="#Fig3" class="usa-link">3b</a>). To explain this phenomenon, we investigated the unique function of GbCYP90J6. Using molecular docking, we constructed a structural model of the GbCYP90J6-apigenin (<strong>5D</strong>)-apigenin (<strong>5U</strong>) complex (Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>). The alanine scanning mutagenesis targeting residues within 5 Å of <strong>5D</strong> and <strong>5U</strong> in the active pocket revealed 31 residues that significantly impact activity (Fig. <a href="#Fig5" class="usa-link">5a</a>). Residues A357 and A460 were excluded due to their native alanine identity, and T293 (in the conserved AGxxT motif) and the heme-coordinating C425 were not mutated due to their essential roles. Most deleterious mutations were clustered in the substrate-binding region, spanning M53-H60 in <em>α</em>A, H83-P88 in loop 1, Y113-L127 in loop 2-<em>α</em>B-loop 3, Y208-I216 in <em>α</em>C, L284-M296 in <em>α</em>D, N355-R362 in loop 4, and F458-V461 in loop 6 (Fig. <a href="#Fig5" class="usa-link">5a, b</a>). Notably, alanine mutations at the tunnel entrance (M53, F458, P459, V461) and within the catalytic center (Y113, P114, F121, I126, F286, L361, R362) abolished or nearly abolished dimerization. In contrast, only V215A mutation in <em>α</em>C, located near the entrance, significantly enhanced the activity. To further probe the structure-function relationship, we examined the homologous mutants derived from inactive CYP90Js, as well as the selected mutations (V215T, F286Y, L382N) based on hydrophobicity and polarity differences between active and inactive enzymes (Fig. <a href="#Fig5" class="usa-link">5c</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">16a</a>). We noticed that different homologous mutants at the same site often resulted in divergent functional outcomes, reinforcing the importance of spatial constraints in maintaining catalytic activity (Supplementary Fig. <a href="#MOESM1" class="usa-link">16a</a>). For instance, at position M287 in <em>α</em>D, M287A retained approximately 14% activity, M287L showed near wild-type activity, while M287Y and M287F nearly abolished function. Similarly, mutations at L382 in loop 5 at the bottom of the substrate tunnel revealed contrasting outcomes: L382N eliminated activity, whereas L382F doubled it. Comparable trends were observed at M53, L127, V215, L284, F286, and L361, further highlighting the critical role of spatial constraints in substrate positioning. Moreover, none of the single-site mutations introduced into the inactive TcCYP90J7 conferred dimeric activity (Supplementary Fig. <a href="#MOESM1" class="usa-link">16b</a>). Western blot analyses confirmed the comparable expression across all mutants, indicating that the observed activity differences were attributable to functional effects of the mutations rather than expression level (Supplementary Fig. <a href="#MOESM1" class="usa-link">17</a>). These mutation tests suggest that the steric hindrance within the substrate-binding cavity creates a sophisticated enzymatic environment crucial for substrate dimerization, governing substrate entry and arrangement.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Mutations analysis of GbCYP90J6 revealed critical amino acid residues within the substrate pocket that determine biflavonoid dimerization specificity.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370966_41467_2025_62990_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7f81/12370966/b03bc7bafaf0/41467_2025_62990_Fig5_HTML.jpg" loading="lazy" id="d33e1427" height="813" width="722" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Results of alanine scanning within a 5 Å range of the <strong>5</strong> ligands of GbCYP90J6. The <em>x</em>-axis represented the mutagenesis sites of GbCYP90J6, and the <em>y</em>-axis showed the relative enzyme activity compared to the wild-type (WT). Each value was presented as mean ± S.D. (<em>n</em> = 3). Experiments were conducted in vivo using <em>E. coli</em> JM109(DE3). “nd” indicated no production of <strong>1</strong>. All mutagenesis within a 5 Å range of the <strong>5</strong> ligands showed significant variation compared to the WT. Source data are provided as a Source Data file. <strong>b</strong> GbCYP90J6-apigenin (<strong>5D</strong>)-apigenin (<strong>5U</strong>) model generated by molecular docking and MD simulation, showing the spatial relationship between key residues identified by mutagenesis and the substrate molecules. Two ligands maintained a head-to-tail, anti-parallel configuration, with distance from C4’-OH of ligand 1 (<strong>5D</strong>) and C7”-OH of ligand 2 (<strong>5U</strong>) to the heme center of 2.2 Å and 4.9 Å, separately. The distance between C3’ of <strong>5D</strong> and C8’ of <strong>5U</strong> was 3.5 Å. Key sites from alanine scanning mutagenesis were highlighted in different colors. <strong>c</strong> Protein sequence alignment of gymnosperm CYP90J, ordered by the sequence of GbCYP90J6, revealing conservation patterns of functionally important residues across active and inactive enzymes.</p></figcaption></figure><p id="Par16">To clarify the catalytic mechanism underlying biflavonoid formation, we conducted molecular dynamics (MD) simulations on the protein variants GbCYP90J6<sub>L382F</sub> and GbCYP90J6<sub>L382N</sub> due to their opposing catalytic activities at the same site (Supplementary Fig. <a href="#MOESM1" class="usa-link">16a</a>). Additionally, we included the active ChCYP90J7 and the inactive TcCYP90J7 for comparative analysis. Structural alignment revealed that even subtle sequence variations significantly alter the conformations of surrounding residues and bound ligands (Fig. <a href="#Fig6" class="usa-link">6a</a>). While the specific locations of these conformational changes varied among variants, they consistently altered ligand positioning relative to the catalytic center. This tread was observed both in the single-site mutant comparison (Fig. <a href="#Fig6" class="usa-link">6a</a>) and across active (GbCYP90J6, ChCYP90J7) versus inactive (TcCYP90J7) enzymes (Fig. <a href="#Fig6" class="usa-link">6b</a>). These observations prompted further investigation into which specific conformational arrangements favor the regiospecific catalysis of C3’-C8” linked biflavonoids.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Structural basis for altered catalytic activity due to conformational changes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370966_41467_2025_62990_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7f81/12370966/3a0914b2a960/41467_2025_62990_Fig6_HTML.jpg" loading="lazy" id="d33e1491" height="1250" width="782" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Structural alignment of active sites among GbCYP90J6 wild-type, high-activity mutant GbCYP90J6<sub>L382F</sub>, and low-activity mutant GbCYP90J6<sub>L382N</sub>. Regions targeted in alanine scanning were shown as cartoon representations, with L382, L382F, and L382N displayed as sticks. Critical distances were indicated in the lower left corner of each structure panel: d1 (distance between Fe = O and 4’-OH of <strong>5D</strong>), d2 (distance between Fe = O and 7”-OH of <strong>5U</strong>), and d3 (distance between C3’of <strong>5D</strong> and C8” of <strong>5U</strong>). These results demonstrated how single-point variations in the active site significantly altered both protein pocket and substrate conformations. <strong>b</strong> Structural comparison of active sites between functional amentoflavone synthases (GbCYP90J6 and ChCYP90J7) and inactive TcCYP90J7. Regions corresponding to the alanine scanning area in GbCYP90J6 were shown as cartoon representations. The three critical distances (d1, d2, and d3) were displayed in the lower left corner of each structural panel, highlighting conformational differences between active and inactive enzymes. <strong>c</strong> Binding free energies (kcal/mol) of ligand 1 (<strong>5D</strong>) and ligand 2 (<strong>5U</strong>) in GbCYP90J6, GbCYP90J6<sub>L382F</sub>, GbCYP90J6<sub>L382N</sub>, TcCYP90J7, and ChCYP90J7 as determined by gmx_MMPBSA analysis.</p></figcaption></figure><p id="Par17">Our analysis revealed a critical correlation between enzyme activity and substrate positioning. In active enzyme models (Fig. <a href="#Fig6" class="usa-link">6</a>), the two apigenin molecules (<strong>5U</strong> and <strong>5D</strong>), were positioned closer to the heme center than in inactive variants (Fig. <a href="#Fig6" class="usa-link">6</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>). This proximity directly correlated with enhanced dimerization activity, as exemplified by GbCYP90J6<sub>L382F</sub> with shorter critical distances (d1 = 2.0 Å, d2 = 3.3 Å, d3 = 3.1 Å) showing higher activity than wild-type GbCYP90J6 (d1 = 2.2 Å, d2 = 4.9 Å, d3 = 3.5 Å) (Fig. <a href="#Fig6" class="usa-link">6a</a>). Conversely, increased distances from the heme center in GbCYP90J6<sub>L382N</sub> (d1 = 4.2 Å, d2 = 4.0 Å, d3 = 5.1 Å) correlated with substantially reduced activity (Fig. <a href="#Fig6" class="usa-link">6a</a>).</p>
<p id="Par18">To elucidate the molecular basis underlying substrate positioning, we conducted detailed protein-ligand interaction analysis using PLIP<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>), complementing the structural comparisons in Fig. <a href="#Fig6" class="usa-link">6</a>. In wide-type GbCYP90J6, residues S358 formed hydrogen bonds with substrates and L117, V215, I216, T293, and A357 contributed hydrophobic interactions (Supplementary Fig. <a href="#MOESM1" class="usa-link">18a</a>). The crucial π–π stacking interaction between <strong>5U</strong> and <strong>5D</strong>, stabilized the substrates in an almost head-tail anti-parallel configuration<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>, positioning C3’ of <strong>5D</strong> close to C8” of <strong>5U</strong> (d3 = 3.5 Å) (Supplementary Fig. <a href="#MOESM1" class="usa-link">18a</a>). The enhanced activity of GbCYP90J6<sub>L382F</sub> resulted from the larger hydrophobic side chain of F382 disrupting the original hydrogen bonding network, accompanied by new hydrogen bonds forming between A357 and <strong>5D</strong>, and between H84 and <strong>5U</strong>. These altered interactions, along with hydrophobic contacts from P114, F121, V215, and A357 (Supplementary Fig. <a href="#MOESM1" class="usa-link">18b</a>), effectively positioned the substrates closer to the heme center, enhancing catalytic efficiency. In contrast, the introduction of a polar asparagine in GbCYP90J6<sub>L382N</sub> destabilized the substrate-binding network. N382 and H84 formed new hydrogen bonds with <strong>5U</strong> and <strong>5D</strong>, respectively, disrupting the original interaction pattern. Hydrophobic interactions between L117, V215, and A357 with <strong>5U</strong> and <strong>5D</strong>. While under these interactions, the inappropriate π–π stacking between the ligands pulled <strong>5D</strong> away from <strong>5U</strong>, increasing d3 to 5.1 Å and diminishing activity (Supplementary Fig. <a href="#MOESM1" class="usa-link">18c</a>). A similar ligand arrangement was observed in the active ChCYP90J7, though driven by a distinct set of residues (Supplementary Fig. <a href="#MOESM1" class="usa-link">18d</a>). In contrast, the inactive TcCYP90J7 failed to support π–π stacking between <strong>5D</strong> and <strong>5U</strong>, resulting in the displacement of <strong>5D</strong> away from both <strong>5U</strong> and the heme center (d1 = 30.3 Å, d2 = 6.2 Å, d3 = 24.5 Å) (Supplementary Fig. <a href="#MOESM1" class="usa-link">18e</a>). Therefore, suitable protein-ligand interactions were crucial for positioning the ligands near the heme center. Changes in surrounding amino acid residues will disrupt the original interactions and form new networks, leading to alterations in dimerization activity.</p>
<p id="Par19">Based on the analysis of protein-ligand models and experimental results, we concluded that the key factors influencing CYP90J dimerization activity are as follows: (1) Non-covalent interactions between amino acid residues in the active pocket and ligands create a spatial constraint that brings the ligands close to the heme center, which is essential for dimerization activity; (2) In addition to constraints from surrounding amino acid residues, π-stacking between <strong>5D</strong> and <strong>5U</strong> in a head-tail manner stabilizes their anti-parallel alignment, positioning C3’ near C8”. This arrangement prevents random ligand coupling and prompts the regiospecific formation of <strong>1</strong> (3’,8”-biapigenin); (3) Mutations within the pocket disrupt the existing interaction network. A new network is formed, changing the distance between the ligands and the heme center. These alterations in conformation and interactions indicate that a tight steric hindrance between the substrates and pocket amino acids determines the dimerization activity. This altered distance affects dimerization activity. Regiospecificity in biflavonoids biosynthesis is intricately tied to the conformational orientation and relative positioning of the substrates within the active pocket.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Dimerization mechanism validated by QM/MM calculation</h3>
<p id="Par20">After confirming that the stable conformation of ligands close to the heme center determined GbCYP90J6 dimerization activity, we proposed two plausible C–C bond formation mechanisms: a radical addition pathway (A) and a diradical coupling pathway (B), inspired by mechanical insights from fungal and bacterial P450s<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. As illustrated in Fig. <a href="#Fig7" class="usa-link">7a</a>, both pathways began with a hydrogen atom transfer (HAT) from the C4’-OH of the proximal unit (<strong>5D</strong>) (Fig. <a href="#Fig6" class="usa-link">6a</a>) to the highly reactive Cpd I, producing a delocalized apigenin radical <strong>R5d</strong>. In pathway A, <strong>R5d</strong> attacked the C8” position of a distal apigenin (<strong>5U</strong>), forming a C3’-C8” linked dimeric radical intermediate (<strong>R5du</strong>), which was subsequently oxidized by Cpd II via HAT from C7”-OH, yielding the keto form of <strong>1</strong>. Alternatively, in pathway B, a second HAT from the C7”-OH of <strong>5U</strong>, also mediated by Cpd II, generated a second delocalized radical (<strong>R5u</strong>). The resulting radicals (<strong>R5d</strong> and <strong>R5u</strong>) then underwent intermolecular coupling to form the same C–C linked keto intermediate. This species spontaneously tautomerized to the final enol form of <strong>1</strong><sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. Mechanistic pathways for the formation of dimer 1 catalyzed by GbCYP90J6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370966_41467_2025_62990_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7f81/12370966/454dff1a8970/41467_2025_62990_Fig7_HTML.jpg" loading="lazy" id="d33e1730" height="691" width="798" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Two proposed dimerization pathways (A and B) for the formation of <strong>1</strong>. Pathway A represented a radical addition mechanism, initiated by formation of the apigenin radical <strong>R5d</strong>. This radical underwent rearrangement and attacked C8” of <strong>5U</strong>, forming the C3’-C8” linked dimeric radical intermediate <strong>R5du</strong>. Cpd II then abstracted hydrogen from C7”-OH, generating <strong>1</strong> (keto form), which underwent tautomerization to produce <strong>1</strong> (enol form). Pathway B described a diradical coupling pathway. Specifically, apigenin radicals (<strong>R5d</strong> and <strong>R5u</strong>) were generated via heme-mediated sequential hydrogen extraction reactions. These radicals underwent rearrangement, intermolecular radical-radical coupling, and aromatized to produce <strong>1</strong>. <strong>b</strong> QM/MM calculated energy profiles for the formation of <strong>1</strong> catalyzed by GbCYP90J6 via two proposed pathways. The relative Gibbs free energy changes (ΔG, kcal/mol) during product <strong>1</strong> formation were shown with pathway A in red and pathway B in blue. Key points along the reaction coordinate included: RC (reactant complex), TS1–5 (transition states 1–5), IM1–4 (intermediates 1–4), and PC (product complex). The QM/MM optimazed structures were shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>.</p></figcaption></figure><p id="Par21">To assess the feasibility of both pathways, we performed QM/MM calculations based on a representative conformation derived from 200 ns MD simulation. The QM region was carefully defined to include key residues involved in hydrogen bonding and steric interactions near the reactive center. The computed energy profiles, the optimized structures of key intermediates and transition states were shown in Fig. <a href="#Fig7" class="usa-link">7b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>. The initial HAT step from the C4’-OH of <strong>5D</strong> to Cpd I proceeded with a low activation barrier of 12.5 kcal/mol, generating intermediate IM1, which was 7.9 kcal/mol above the reactant complex (RC). Spin population analysis revealed that the radical in IM1 was delocalized across the aromatic ring of <strong>R5d</strong><sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>, with a high spin densities at the C-3’/C-5’ (−0.27/−0.25), consistent with its potential as a radical reaction site. In pathway A, the subsequent radical addition (IM1 → IM2) and HAT from C7”-OH (IM2 → IM3) catalyzed by Cpd II proceeds via a high energy barrier of 32.8 kcal/mol (TS3 relative to RC), attributed to the suboptimal geometry and the distal positioning of the hydroxyl group. This renders the pathway kinetically unfavorable. In contrast, pathway B involved a second HAT (IM1 → IM4) with a lower barrier of 22.5 kcal/mol compared to TS3. This was followed by a radical-radical coupling (IM4 → IM3) via TS5, which required only 4.6 kcal/mol from IM4 to TS5. Notably, this C–C formation barrier was substantially lower than that in pathway A (15.5 kcal/mol from IM1 to TS2). The final keto-enol tautomerization (IM3 → PC) occured spontaneously, driven by a sharp energy drop of over 42 kcal/mol (from 18.2 to −24.2 kcal/mol), thereby stabilizing the coupled product <strong>1</strong> (Fig. <a href="#Fig7" class="usa-link">7b</a>)<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. These results strongly supported the diradical coupling mechanism (pathway B) as the thermodynamically preferred pathway for GbCYP90J6-catalyzed dimerization.</p>
<p id="Par22">To further elucidate the conformational changes required for C–C bond formation, we analyzed the structural transition from IM1 through the pre-coupling intermediate IM4 to the transition state TS5. QM/MM-optimized structures (Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>) revealed a progressive decrease in the C3’-C8” distance, from 4.7 Å in IM1 to 1.8 Å in IM4 and 1.7 Å in TS5. This reorganization was primarily driven by counterclockwise rotation and reorientation of <strong>R5d</strong>, which brought its C3’ atom into close proximity with the C8” atom of <strong>R5u</strong>, thereby enabling selective C3’-C8” coupling. QM/MM calculations further identified residues V215, I216, E292, and G359 were crucial for stabilizing high-energy transition states and intermediates via hydrogen bonding interaction (Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>). While mutagenesis and MD simulations revealed additional residues essential for catalytic activity, these QM/MM-identified residues specifically exemplified how the active site environment precisely orchestrates substrate alignment, thereby supporting the central role of the conformational control in the GbCYP90J6-catalyzed dimerization.</p></section><section id="Sec7"><h3 class="pmc_sec_title">De novo biosynthesis of amentoflavone in <em>E. coli</em>
</h3>
<p id="Par23">Based on our discovery of biflavonoid dimerases, we established a synthetic pathway for producing <strong>1</strong> in <em>E. coli</em>. To this end, we first engineered a <em>L</em>-tyrosine (<em>L</em>-Tyr) over-producing strain GJA3, as <em>L</em>-Tyr is the key precursor of apigenin (<strong>5</strong>). The engineered <em>E. coli</em> GJA3 was derived from JM109(DE3) by deleting <em>tyrR</em> and <em>pheA</em> genes, followed by integration of <em>aroG</em><sup><em>fbr</em></sup>, <em>tyrA</em><sup><em>fbr</em></sup>, <em>ydiB</em>, <em>tktA,</em> and <em>ppsA</em> genes into the <em>tyrR</em> locus (Fig. <a href="#Fig8" class="usa-link">8a</a>)<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. This engineered GJA3 produced 1.89 g/L <em>L</em>-Tyr after 72 h cultivation, reaching an OD<sub>600</sub> of 15.5 (Fig. <a href="#Fig8" class="usa-link">8b</a>).</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8. De novo biosynthesis of 1 in <em>E. coli.</em>
</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370966_41467_2025_62990_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7f81/12370966/e90252ef6f69/41467_2025_62990_Fig8_HTML.jpg" loading="lazy" id="d33e1900" height="832" width="775" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Metabolic engineering strategy for <em>L</em>-Tyr over-production in <em>E. coli</em> JM109(DE3). <strong>b</strong>
<em>L</em>-Tyr production by the engineered strain GJA3 after 72 h fermentation. Each value represents mean ± S.D. (<em>n</em> = 3). <strong>c</strong> Biosynthetic modules for <strong>1</strong> production from <em>L</em>-Tyr in GJA3 strain. <strong>d</strong> Titers of <strong>1</strong> and its precursor <strong>5</strong>. Each value represents mean ± S.D. (<em>n</em> = 3). “nd”: not detected. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par24">We then assembled two functional modules for <strong>1</strong> production: an apigenin monomer generation module and a dimerization synthesis module. The apigenin generation module, carried on plasmid pYH57<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>, includes four known functional proteins (RtPAL from <em>Rhodotorula toruloides</em>, Pc4CL from <em>Petroselinum crispum</em>, PhCHS from <em>Petunia hybrida</em>, and MsCHI from <em>Medicago sativa</em>) to produce <strong>5</strong> from <em>L</em>-Tyr in <em>E. coli</em> (Fig. <a href="#Fig8" class="usa-link">8c</a>). Strain sXH138 (GJA3 carrying pYH57) produced 236.93 mg/L <strong>5</strong> after 72 h cultivation, reaching an OD<sub>600</sub> of 18.8 (Fig. <a href="#Fig8" class="usa-link">8d</a>). The dimerization synthesis module was implemented through plasmid pBIF21, carrying the apigenin dimerase variant trGbCYP90J6<sub>L382F</sub> along with its electron transfer partners AtCPR2 and RsCYB5, required for CYP-catalyzed oxidation reaction (Fig. <a href="#Fig8" class="usa-link">8c</a>). Finally, strain sXH140 (GJA3 carrying pYH57 and pBIF21) successfully produced <strong>1</strong> at the titer of 4.75 mg/L after 72 h cultivation in shake flasks (Fig. <a href="#Fig8" class="usa-link">8d</a>).</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par25">Plant CYP enzymes are crucial for synthesizing and modifying natural compounds, engaging in typical monooxygenation and hydroxylation reactions, as well as unique processes like oxidative rearrangements, oxidative C–C bond cleavage, and phenol coupling<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. While intramolecular C–C and intermolecular C-O phenol coupling reactions have been previously documented<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>,<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>, intermolecular C–C phenol coupling remains largely unexplored. C–C linkages in biflavonoids, which are widespread in plants, play an essential role in conferring distinct biological activities, differentiating them from mono-flavonoids<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>,<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Despite their significance, the enzymes responsible for biflavonoid biosynthesis have long been unknown.</p>
<p id="Par26">In this work, we identified key enzymes involved in this pathway, especially focusing on the synthesis of the biflavonoid skeleton amentoflavone (<strong>1</strong>), which is produced by the action of GbCYP90J6, ChCYP90J7, and SmCYP90E3v1. Beyond native substrate apigenin (<strong>5</strong>), these enzymes, together with MgCYP90J1, also catalyzed the oxidative dimerization of (<em>S</em>)-naringenin (<strong>11</strong>) to form products <strong>11a</strong> and <strong>11b</strong>, with SmCYP90E3v1 producing only <strong>11a</strong>. Although low conversion efficiency prevented precise structural characterization, MD simulations suggested these dimers may arise from C3’-C8”, C3-C8”, or C7-O-C3” coupling modes, depending on substrate’s conformational alignment (Supplementary Fig. <a href="#MOESM1" class="usa-link">20</a>). In <em>G. biloba</em>, <strong>1</strong> underwent successive methylation by GbOMT1-GbOMT5 to yield prominent biflavonoids including bilobetin (<strong>2</strong>), ginkgetin (<strong>3</strong>), and isoginkgetin (<strong>4</strong>), podocarpusflavone A (<strong>8</strong>), sequoiaflavone (<strong>9</strong>), and likely 4’,7”-O-methylamentoflavone (<strong>10</strong>) (Fig. <a href="#Fig2" class="usa-link">2</a>).</p>
<p id="Par27">Unlike classical CYP90 enzymes known for catalyzing the steroid hydroxylation<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, these CYP90J subfamily enzymes facilitate the formation of an intermolecular C–C bond, a function previously attributed solely to microorganisms CYPs<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. This represents a rare identification of CYP enzymes in plants capable of catalyzing intermolecular C–C coupling, marking a significant advance in plant biochemistry. The distinctive functional divergence of CYP90J enzymes prompted us to investigate their evolutionary origins.</p>
<p id="Par28">Given the unique role of GbCYP90J6 compared to other CYP90 members in <em>G. biloba</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">1d</a>), we investigated the origin and distribution of CYP90J to understand its evolutionary background. Phylogenetic analysis of plant-derived CYP90s (Fig. <a href="#Fig3" class="usa-link">3</a>), together with the known evolutionary history of plant lineages, suggests that CYP90Js in gymnosperms likely evolved from the CYP90E family in lycophytes. This hypothesis was supported by our observation that SmCYP90E3v1 also catalyzes the dimerization of apigenin (<strong>5</strong>) and (<em>S</em>)-naringenin (<strong>11</strong>) (Fig. <a href="#Fig3" class="usa-link">3</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>). While this result provided functional support for the proposed evolutionary link, we noted that only one CYP90E paralog from <em>Selaginella</em> was tested in this study. Additional biochemical validation of closely related CYP90E members was considered necessary to determine whether this activity is broadly conserved within the subclade. In contrast, no CYP90J-like members were found in angiosperms, demonstrating the uniqueness of CYP90J in gymnosperms (Fig. <a href="#Fig3" class="usa-link">3</a>). Further characterization of other gymnosperm-specific CYP90J subfamily members confirmed their specialized in flavonoid dimerization (Fig. <a href="#Fig3" class="usa-link">3b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>), but their varying catalytic activities suggested the need for further investigation into suitable natural substrates, especially considering the divers subtypes and C–C connection patterners of biflavonoids found in gymnosperms<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Moreover, our findings reveal that CYP90J exists exclusively in gymnosperms, suggesting that different enzymes may be responsible for biflavonoid skeleton synthesis in other plants, such as angiosperms. Additionally, while previous studies suggested that the emergence of CYP90 in lycophytes marks the advent of brassinosteroid biosynthesis<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, our analysis reveals that the distribution of CYP90J in gymnosperms aligns with that of biflavonoids, enhancing our understanding of the functional role of the CYP90 family (Fig. <a href="#Fig4" class="usa-link">4</a>). The biosynthesis of brassinosteroids and biflavonoid skeletons shares a common CYP ancestor, suggesting that ancestral subfamilies such as CYP85 and CYP763 likely possessed bifunctional activities in the biosynthesis of both biflavonoids and brassinosteroids<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. However, the evolutionary divergence of these bifunctional activities remains unclear.</p>
<p id="Par29">Although CYP90J subfamily members catalyzed flavonoids dimerization, they exhibit varying catalytic abilities (Fig. <a href="#Fig3" class="usa-link">3</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>). To investigate the reasons behind these differences, we built representative active models GbCYP90J6-apigenin (<strong>5D</strong>)-apigenin (<strong>5U</strong>) for analysis (Fig. <a href="#Fig6" class="usa-link">6</a>). Using these models, we performed alanine scanning and single-site mutation analysis based on the sequences of the inactive CpCYP90J8, MgCYP90J1 and TcCYP90J7, within a 5 Å range of the <strong>5D</strong> and <strong>5U</strong> ligands. These analyses reveal the critical roles of each residue in dimerization activity (Fig. <a href="#Fig5" class="usa-link">5</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">16</a>) and indicate the importance of residues surrounding the ligand in determining dimerization activity. Understanding how single-site mutations affect dimerization activity can also uncover the mechanism behind CYP90J enzymes activity or inactivity. Therefore, we compared several complex models—GbCYP90J6, GbCYP90J6<sub>L382F</sub> GbCYP90J6<sub>L382N</sub>, TcCYP90J7, and ChCYP90J7—based on their minimal differences in active site amino acid residues. From a structural alignment perspective, single-site differences could significantly impact the conformation of residues within the active sites and their interaction with ligands (Fig. <a href="#Fig6" class="usa-link">6</a>). Consequently, this affects the distance between the ligands and the heme center, leading to variations in enzyme activity. The regioselectivity and catalytic efficiency of the dimerization reaction are primarily determined by the substrate conformational orientation and spatial positioning during the formation of the reactive Fe(IV)-oxo intermediate, as demonstrated in microbial CYPs such as NasbB, aryC, and AtuP450<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. At the molecular level, our analysis of protein-ligand interactions revealed that active enzymes establish tight interactions between surrounding residues and ligands through hydrogen bonds and hydrophobic interactions. Additionally, π–π stacking between the two apigenin (<strong>5D</strong> and <strong>5U</strong>) ligands helps stabilize their relative positions<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>, bringing C3’ and C8” into close proximity. Minor changes in this balance could disrupt the molecular conformation, resulting in different interactions and conformations (Fig. <a href="#Fig6" class="usa-link">6</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>). These observations indicated that active enzymes, such as GbCYP90J6, create tight steric constraints between the substrates and pocket residues, which determines dimerization activity. As minor changes in the surrounding residues significantly affect both the protein-substrates conformations and their interactions, such tight steric hindrance typically plays a decisive role in CYP-catalyzed dimerization<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>.</p>
<p id="Par30">To further elucidate the molecular mechanism underlying GbCYP90J6, we performed QM/MM calculations to evaluate the C–C bond formation pathway and to exclude the radical addition mechanism, which was kinetically inaccessible due to a high energy barrier of 32.8 kcal/mol. Instead, we proposed a more feasible diradical coupling mechanism in which residues V215, I216, E292, and G359 play key roles in stabilizing the conformations of intermediates and transition states through hydrogen bonding interactions (Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>). Alongside steric constraints imposed by surrounding residues, these interactions guide the reactive radicals into productive orientations, thereby conferring regioselectivity to the C3’-C8” coupling. Notably, while the intermolecular C–C formation often contributes to the rate-limiting step in previously characterized fungal and bacterial-derived CYPs<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, GbCYP90J6 facilitates this step via a diradical coupling mechanism with a comparatively low energry barrier of only 4.6 kcal/mol from IM4 to TS5. In contrast, the second HAT step to Cpd II represents the rate-limiting step, with a barrier of 22.5 kcal/mol relative to RC, thereby distinguishing the catalytic profile of GbCYP90J6 from microbial CYPs (Fig. <a href="#Fig7" class="usa-link">7</a>). In addition, the requirement for radical formation also explains the lack of GbCYP90J6 activity toward methylated apigenin substrates such as acacetin (<strong>6</strong>) and apigenin 7,4’-dimethyl ether (<strong>7</strong>) (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). Methylation at the C3’-OH and C7-OH positions blocks the initial hydrogen abstraction by the CYP heme center, preventing radical intermediate formation. Additionally, the binding free energy calculations revealed weaker substrate affinity for methylated compounds, with ΔG values of 13.25 and 18.39 kcal/mol for acacetin-containing complexes compared to apigenin (−12.06 kcal/mol) (Supplementary Fig. <a href="#MOESM1" class="usa-link">21</a> and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). These findings demonstrate that successful intermolecular coupling requires both accessible hydroxyl groups for radical generation and optimal substrate positioning within the enzyme active site.</p>
<p id="Par31">Building on the discovery of crucial biflavonoid dimerases responsible for oxidative C–C cross-coupling formation, we successfully engineered an <em>E. coli</em> strain for the de novo production of amentoflavone (<strong>1</strong>), achieving a titer of 4.75 mg/L through the sequential conversion of <em>L</em>-Tyr (1.89 g/L) to apigenin (<strong>5</strong>, 236.93 mg/L), followed by its subsequent dimerization (Fig. <a href="#Fig8" class="usa-link">8d</a>). Notably, this microbial production system proved more efficient than natural sourcing (0.059 mg/g DW, Fig. <a href="#Fig1" class="usa-link">1b</a>). This heterologous biosynthetic platform represents a promising alternative for efficient biflavonoid production and holds significant potential for pharmaceutical manufacturing.</p>
<p id="Par32">In summary, we uncovered the function of CYP90J subfamily members that catalyze the specific <em>C</em><sub>2</sub> unsymmetrical dimerization of biflavonoids with distinct C3’-C8” linkages. These gymnosperm-specific CYP90J enzymes, which our analyses indicate evolved from CYP90E in lycophytes, are essential for C–C type biflavonoid synthesis. Additionally, we elucidated the methylation modification pathway for key ginkgo biflavonoid biosynthesis, including bilobetin, ginkgetin, and isoginkgetin, mediated by GbOMT1 to GbOMT5. Through MD simulations and QM/MM calculations, we discovered that CYP90Js share catalytic mechanisms with bacteria and fungi CYPs, controlling the regiospecificity of C–C cross-coupling reactions. Importantly, GbCYP90J6 catalyzes a diradical coupling reaction, where the steric hindrance from the enzyme pocket and π–π stacking interactions between two apigenin molecules guide their head-tail, anti-parallel alignment, regioselectivity yielding amentoflavone. Finally, building on these mechanistic insights, we successfully engineered a microbial platform for the de novo synthesis of amentoflavone, which holds significant promise for biotechnological applications in the production of valuable biflavonoids.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Plant materials and reagent</h3>
<p id="Par33">The <em>G. biloba</em> L. trees (3 years old) used in this study were collected from a cultivated population in Shanghai, China, under open-air conditions. The main roots (MR), old roots (OR), fibrous roots (FR), and young stems (YS) were collected in March, June, and August 2022, ensuring consistent growth conditions and ages across different trees. Young leaves (YL) and semi-mature leaves (uML) were collected in June 2022, and mature old leaves (ML) were collected in August 2022. The specific appearance of each tissue type is shown in Fig. <a href="#Fig1" class="usa-link">1a</a>. Each tissue type was collected in three biological replicates for transcriptome sequencing and content determination. Tissues used for sequencing were immediately frozen in liquid nitrogen and stored at −80 °C until use. Tissues for content determination were dehydrated in a 37 °C oven until completely dry, followed by extraction and quantification.</p>
<p id="Par34">Biflavonoid standards, including amentoflavone (catalog: FY1576), ginkgetin (catalog: FY1965), and isoginkgetin (catalog: FY3037), were purchased from Feiyubio (Nantong, China). Bilobetin (catalog: <a href="https://www.ncbi.nlm.nih.gov/nuccore/B20029" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">B20029</a>) was obtained from Yuanye (Shanghai, China). Additionally, various mono-flavonoids, such as apigenin (catalog: <a href="https://www.ncbi.nlm.nih.gov/protein/CFN98843" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CFN98843</a>), acacetin (catalog: <a href="https://www.ncbi.nlm.nih.gov/protein/CFN98744" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CFN98744</a>), apigenin 7,4’-dimethyl (catalog: <a href="https://www.ncbi.nlm.nih.gov/protein/CFN98819" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CFN98819</a>), (<em>S</em>)-naringenin (catalog: <a href="https://www.ncbi.nlm.nih.gov/protein/CFN98742" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CFN98742</a>), tricetin (catalog: <a href="https://www.ncbi.nlm.nih.gov/protein/CFN70392" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CFN70392</a>), luteolin (catalog: <a href="https://www.ncbi.nlm.nih.gov/protein/CFN98784" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CFN98784</a>), eriodictyol (catalog: <a href="https://www.ncbi.nlm.nih.gov/protein/CFN99719" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CFN99719</a>), aromadedrin (catalog: <a href="https://www.ncbi.nlm.nih.gov/protein/CFN98736" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CFN98736</a>), kaempferol (catalog: <a href="https://www.ncbi.nlm.nih.gov/protein/CFN98838" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CFN98838</a>), quercetin (catalog: <a href="https://www.ncbi.nlm.nih.gov/protein/CFN99272" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CFN99272</a>), myricetin (catalog: <a href="https://www.ncbi.nlm.nih.gov/protein/CFN98877" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CFN98877</a>), taxifolin (catalog: <a href="https://www.ncbi.nlm.nih.gov/protein/CFN98734" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CFN98734</a>), and ampelopsin (catalog: <a href="https://www.ncbi.nlm.nih.gov/protein/CFN98326" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CFN98326</a>) were sourced from ChemFaces (Wuhan, China). All flavonoids were dissolved in DMSO with a concentration of 50 mM. The stock solutions of all reagents are stored at −20 °C.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Extraction and quantification of biflavonoids from <em>G. biloba</em>
</h3>
<p id="Par35">Biflavonoid extraction from <em>G. biloba</em> tissues was performed according to Mistry et al.<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. In brief, plant tissues were ground into powder using a high-throughput tissue grinder, and 0.1 g of the powder was transferred to a 2 mL microcentrifuge tube. Then, 1 mL of 70% methanol was added, followed by ultrasound treatment for 30 min. The supernatant was collected, and the extraction process was repeated twice. The solvent was evaporated using a vacuum centrifugal concentrator, and the residue was re-dissolved in 1 mL of 70% (v/v) methanol. Detection and quantification were performed using HPLC (Thermo, UltiMate 3000, USA). Three biological replicates were performed for each sample group.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Transcriptome analysis</h3>
<p id="Par36"><em>G. biloba</em> tissues from three different trees were used to prepare the Illumina sequence library. Total RNA was isolated using the mirVana<sup>TM</sup> RNA Isolation Kit (Ambion, USA, catalog AM1560). The sequencing library was constructed from total RNA samples using a TruSeq Stranded mRNA LT Sample Prep Kit (Illumina, San Diego, CA, USA, catalog RS-122-9004DOC). The high-quality library was sequenced with Illumina HiSeq<sup>TM</sup> 2500. The transcriptome sequencing was conducted by OE Biotech Co., Ltd. (Shanghai, China). The genome of <em>G. biloba</em>, published in 2021<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, was used to establish the reference transcriptome.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Candidates screening and gene cloning</h3>
<p id="Par37">Candidate genes were initially identified by extracting peroxidases, PPOs, laccases, and CYPs from the <em>G. biloba</em> genome and transcriptome annotations. Additional candidates were identified using Hmmer (version 3.4)<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> (E &lt; 1 × 10<sup>−10</sup>) with Pfam identifiers: peroxidase (PF00141), PPO (PF12143), laccase (PF02578), and CYP (PF00067). Candidates with complete domains and high leaf-specific expression were selected for functional characterization. Expression levels of candidate biflavonoid synthase genes are provided in Supplementary Data <a href="#MOESM5" class="usa-link">3</a>.</p>
<p id="Par38">For transient expression in tobacco, candidate genes were cloned from leaf cDNA using 2 × Phanta Flash Master Mix enzyme (Vazyme, Nanjing, China, catalog P520) and subcloned into the <em>SalI</em> and <em>XhoI</em> sites in the pEAQ-<em>HT</em> vector using In-Fusion cloning (ClonExpress II One Step Cloning Kit, Vazyme, Nanjing, China, catalog C112). The resulting vectors were transformed into <em>E. coli</em> DH10B. After sequencing by Sangon Biotech (Shanghai, China), the correct recombinant plasmids were transformed into <em>A. tumefaciens</em> GV3101.</p>
<p id="Par39">For the plasmid used in the <em>E. coli</em> expression system, <em>trGbCYP90J6</em> was subcloned into the <em>NcoI</em> and <em>BamHI</em> sites of pFe00. The pFe00 (pCDFduet-AtCPR2-RsCYB5) was constructed from pCDFDuet by inserting the <em>AtCPR2</em>-RBS-<em>RsCYB5</em> fragment into the <em>SalI</em> and <em>XhoI</em> sites. <em>RsCYB5</em> was amplified from pCZ441<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>, and <em>AtCPR2</em> was cloned from <em>A. thaliana</em> (accession number: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_179141" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_179141</a>). <em>TcCYP90J7</em>, <em>ChCYP90J7</em>, <em>CpCYP90J8</em>, <em>MgCYP90J1</em>, <em>TpCYP90J1v1</em>, <em>TpCYP90J9</em>, <em>TpCYP90J10</em>, and <em>SmCYP90E3v1</em> were synthesized by General Biol (Anhui, China) after codon optimization for <em>E. coli</em> and inserted into pFe00 between the <em>NcoI</em> and <em>BamHI</em> sites. All CYPs and AtCPR2 transmembrane sequences were predicted using TMHMM-2.0 and replaced with a synthetic 17<em>α</em>-tag (MALLLAVF)<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>.</p>
<p id="Par40">Mutations were introduced by using mutant-specific primers to amplify plasmids containing the wild-type gene, which served as the template. The amplified fragments were digested with <em>DpnI</em> and then transformed into DH10B. Correct clones were selected by Sanger sequencing.</p>
<p id="Par41">The <em>O</em>-methyltransferases (OMTs) were screened based on transcriptome annotation and subcloned into pET28a between <em>NdeI</em> and <em>EcoRI</em>. Details of primers and plasmids used in this research are listed in Supplementary Data <a href="#MOESM6" class="usa-link">4</a> and <a href="#MOESM7" class="usa-link">5</a>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Phylogenetic analyses</h3>
<p id="Par42">To infer the evolutionary history of the CYP90 family, we conducted phylogenetic analysis based on the existence of the CYP90 family in plants, including representative genomes of lycophytes, ferns, gymnosperms, and angiosperms. Additionally, this analysis incorporated known functional CYP90 members collected from the plant P450 database<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. On the other hand, we used Orthofinder (version 2.5.4)<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup> to identify GbCYP90J6 orthologs in bryophytes, lycophytes, ferns, and gymnosperms, analyzing the relationship between the presence of orthologs and biflavonoid compounds. All sequences used in these analysis are listed in Supplementary Data <a href="#MOESM3" class="usa-link">1</a> and <a href="#MOESM4" class="usa-link">2</a>. Sequences were aligned using MUSCLE (version 5.1) multiple sequence alignment, and phylogenetic analyses were conducted with IQ-tree (version 3.0)<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup> using the maximum likelihood method with bootstrap support and 1000 replications (Fig. <a href="#Fig3" class="usa-link">3</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Transient expression in <em>N. benthamiana</em>
</h3>
<p id="Par43">To screen functional genes, <em>A. tumefaciens</em> carrying candidate genes was used for transient infiltration of tobacco plants. Positive strains were cultured in LB medium supplemented with kanamycin (50 μg·mL<sup>−1</sup>), rifampicin (50 μg·mL<sup>−1</sup>), and gentamycin (50 μg·mL<sup>−1</sup>) at 28 °C with shaking until they reached an OD<sub>600</sub> of 1. The culture was then centrifuged at 4629 g for 5 min, and the resulting pellet was resuspended in infiltration buffer (10 mM MES,10 mM MgCl<sub>2</sub>, 150 μM acetosyringone, pH 5.6) to achieve an OD<sub>600</sub> of 0.8. The resuspended cultures were incubated at room temperature for 2–3 h before being infiltrated into the leaves of 4-week-old tobacco plants. Each sample was infiltrated into three different plants as biological replicates, with <em>A. tumefaciens</em> carrying the empty vector pEAQ-<em>HT</em> serving as a negative control. Three days post-infiltration, the leaves were infiltrated with 100 μM apigenin (dissolved in 4.75 mM methyl-β-cyclodextrin (MCD) solution) and further cultivated for two days. Leaf tissues (0.3 g) from each treatment were frozen in liquid nitrogen, ground into fine powder, and subjected to triple extraction with 1 mL of water-saturated n-butanol. After extraction, the resulting extracts were evaporated to dryness and reconstituted in 200 μL of methanol for subsequent analysis using HPLC and Q-Exactive mass spectrometry.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Characterization of CYPs in <em>E. coli</em>
</h3>
<p id="Par44"><em>E. coli</em> JM109(DE3) (Promega, USA) was used for the characterization of CYPs in vivo. Plasmids utilized in these experiments are described in Supplementary Data <a href="#MOESM7" class="usa-link">5</a>. The starter cultures of strains carrying plasmids were precultured at 37 °C in LB medium overnight and then inoculated (1:100) into 10 mL of TB medium (with 20 g·L<sup>−1</sup> glycerol), with three replicates for each experiment. After reaching an OD<sub>600</sub> of 0.6–0.8, isopropyl β-D-1-thiogalactopyranoside (IPTG) was added to the cooled cultures to a final concentration of 0.1 mM. Concurrently, flavonoid substrates dissolved in 50 mM MCD were added to a final concentration of 100 μM (with a final concentration of 4.75 mM MCD). Control cultures containing plasmid pFe00 were run in parallel. The cultures were shaken at 22 °C for 72 h. Afterward, 1 mL of fermentation broth was ultrasonicated and extracted three times with 1 mL of ethyl acetate. All extracts were evaporated to dryness and re-dissolved in 200 μL of methanol for subsequent analysis.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Kinetic analysis</h3>
<p id="Par45">The coding region of <em>GbCYP90J6</em> was amplified from the vector pEAQ-1.3004 by PCR using primer pair ESC-GbJ6-F/-R (Supplementary Data <a href="#MOESM6" class="usa-link">4</a>). The PCR fragments were ligated into a predigested pESC/-HIS vector using <em>EcoRI</em> and <em>NotI</em> (NEB, U.S.A.), yielding pESC-GbCYP90J6. The correct plasmids were transformed into the <em>Saccharomyces cerevisiae</em> WAT11. The recombinant yeast strains grew at 30 °C on a synthetic drop-out medium without histidine (SD/-His, containing 2% glucose) plate for 48 h. A single clone was cultured in 200 mL SD/-His (containing 2% glucose) medium until an OD600 reached 1, after which the cells were transferred to SD/-His (containing 2% galactose) for an additional 12 h. Microsomes were prepared as previously described<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>.</p>
<p id="Par46">For kinetic analyses, in vitro reactions were conducted in a total volume of 1 mL. The reaction mixtures consist of 500 μg microsomal protein, 50 mM Tris-HCl (pH 7.5), 1 mM NADPH, 5 μM FAD, 5 μg FMN, 4 mM glucose-6-phosphate, 1 unit glucose-6-phosphate dehydrogenase and apigenin ranging from 0.1 to 30 μM. Three biological replicates were set up for analysis. After incubating the mixtures at 30 °C for 1 h, the reactions were quenched by extracting three times with 1 mL of ethyl acetate. The supernatants were evaporated to dryness, resuspended in 100 μL methanol, and the products were quantified by HPLC. Curve fitting was performed using Michaelis–Menten model.</p>
<p id="Par47">Additionally, for the GbCYP90J6 in vitro flavonoids multi-substrates characterization, substrates were added to 25 μM separately and extracted three times with 1 mL water-saturated n-butanol. The extractions were resuspended in 100 μL of methanol and analyzed by LC-MS/MS.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Characterization of OMTs in vitro</h3>
<p id="Par48">The pET28a vectors carrying the OMTs open reading frame were transformed into <em>E. coli</em> BL21(DE3) for protein expression. The starter cultures of strains harboring the plasmids were grown overnight at 37 °C in an LB medium containing 50 μg/mL kanamycin. These cultures were then inoculated (1:100) into 20 mL of LB medium (containing 50 μg/mL kanamycin). Upon reaching an OD<sub>600</sub> of 0.4, IPTG was added to the cooled cultures to a final concentration of 0.1 mM. The expression system was incubated at 22 °C and 220 rpm for 20 h. Subsequently, the cell pellets were collected and resuspended in 1 mL of resuspension buffer (50 mM Tris-HCl, pH 7.5, 50 mM NaCl, 10% glycerol, 5 mM DTT). After adding protease inhibitor (1% v/v Cocktail, Sangon, China), the cells were lysed using an ultrasonic homogenizer at 30 W power. The resulting crude enzymes were used for in vitro enzyme reactions, while crude enzymes from recombinant strains harboring the pET28a empty vector were used as negative controls.</p>
<p id="Par49">For the active analyses, in vitro reactions were conducted in a total volume of 100 μL. The reaction mixtures consist of 98 μL crude enzymes, 50 μM substrates (amentoflavone), and 0.1 mM <em>S</em>-adenosyl-<em>L</em>-methionine. After incubating the mixtures at 37 °C for 1 h, the reactions were quenched by extracting three times with 100 μL of ethyl acetate. The supernatants were evaporated to dryness and resuspended in 100 μL methanol, and the products were detected by HPLC and LC-MS/MS.</p></section><section id="Sec19"><h3 class="pmc_sec_title">HPLC and LC-MS/MS analyses</h3>
<p id="Par50">For the biflavonoids analysis using the HPLC system, a Dionex Ultimate 3000 HPLC instrument (Thermo Fisher Scientific, USA) equipped with an Athena C18 column (4.6 × 250 mm) (ANPEL Laboratory Technologies, China) was utilized. The mobile phase consisted of solvent A (pure water with 0.1% formic acid) and solvent B (acetonitrile with 0.1% formic acid). Gradient elution began with 95% solvent A and 5% solvent B maintained for 5 min. Subsequently, solvent B increased from 5% to 100% over 15 min and maintained at 100% for 10 min. Thereafter, solvent B was reduced to 5% over 5 min. The flow rate was set at 1 mL·min<sup>−1</sup>, and UV absorbance was monitored at 280 nm. Quantification of biflavonoids was performed by measuring the peak areas in the UV absorbance at 280 nm, followed by calculation using a standard curve. For the catalytic activity analysis of mutants, the peak area of the wild-type was standardized to 100%, and the relative peak areas of mutants were calculated for comparison. All calculations were based on three biological replicates.</p>
<p id="Par51">For <em>L</em>-Tyr detection, HPLC analysis was performed using the same column and mobile phase as for biflavonoids. The gradient elution program was as follows: 5–15% B (0–12 min), 15–95% B (12–12.1 min), held at 95% B (12.1–15.1 min), followed by re-equilibration at 5% B for 5 min. The flow rate was maintained at 1 mL·min<sup>−1</sup>, with UV detection at 278 nm.</p>
<p id="Par52">Samples prepared as described above were subjected to qualitative analysis by LC-MS/MS (<em>n</em> = 1). Characteristic fragment ions of the target compounds were extracted and analyzed using Xcalibur (version 4.4) software. For the LC-MS/MS analyses of flavonoids, a Dionex Ultimate 3000 HPLC instrument (Thermo Fisher Scientific, USA) equipped with a 00D-4462-AN C18 column (2.1 × 100 mm, 2.6 μm) (Phenomenex, USA) coupled to a Q Exactive Hybrid Quadrupole-Orbitrap mass spectrometer (Thermo Scientific, USA) was employed. The mobile phase consists of H<sub>2</sub>O (0.1% formic acid, solvent A) and acetonitrile (solvent B). The elution gradient was as follows: 0–1 min 10% solvent B, 1–15 min with a linear increase from 10% to 99% solvent B, 15–18 min 99% solvent B, and 18–21 min returning to 10% solvent B. The flow was set at 0.3 mL·min<sup>−1</sup>. Electrospray ionization was used as the ion source in negative mode, with full-scan mass spectrometry set with a range of <em>m/z</em> 100–1500 Da.</p></section><section id="Sec20"><h3 class="pmc_sec_title">NMR spectroscopy</h3>
<p id="Par53">The <sup>1</sup>H NMR and NOESY spectra were recorded on a Bruker AVANCE III HD 500 MHz spectrometer. Lyophilized samples (1–2 mg) were dissolved in 450 μL of CD<sub>3</sub>OD (Cambridge Isotope Laboratories), and all spectra were acquired at 25 °C. <sup>1</sup>H chemical shifts were referenced to the residual solvent signal of CD<sub>3</sub>OD at 3.31 ppm. NMR data were processed using Bruker TopSpin (version 3.2) and analyzed with MestReNova (version 16.0.0).</p></section><section id="Sec21"><h3 class="pmc_sec_title">Protein structure prediction, molecular docking, and site-directed mutagenesis</h3>
<p id="Par54">The CYP protein structures containing heme used in this study were predicted by AlphaFold 2, as provided by the tools in PCPD database<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. AutoDockTools (version 1.5.7) was employed to prepare the proteins and ligands<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. Subsequently, the ligands were docked into the pocket near the heme using AutoDock Vina (version 1.1.2)<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. The relative conformations of the two substrates were determined through sequential docking. First, apigenin (<strong>5D</strong>) was docked to identify its optimal binding pose. Then, the second apigenin (<strong>5U</strong>) was docked while keeping the first substrate fixed. Models were selected based on specific criteria: proximity of reactive sites to heme iron, proper orientation of C4’-OH in <strong>5D</strong> and C7”-OH in <strong>5U</strong> toward the catalytic center, compatible positioning for C3’-C8” coupling, and favorable interaction energy scores. No artificial constraints were applied during this process, allowing substrate conformations to emerge naturally from protein-substrate interactions. Amino acids residues within 5 Å of the apigenin ligands were chosen for alanine scanning mutagenesis. ChimeraX (version 1.8)<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup> was used for protein-ligand model analysis and visualization.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Classical MD</h3>
<p id="Par55">Using GROMACS (version 2021.3) software, a 100–200 ns MD simulation of the protein-ligand complex was conducted<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. The AMBER99SB-ILDN force field was selected to generate the protein topology file, and the small molecule ligand topology file was generated using the MCPY module from Amber20 software and the GAFF force field<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. A truncated octahedral TIP3P solvent box was applied to the system with a 10 nm distance, and Na<sup>+</sup>/Cl<sup>−</sup> ions were added to neutralize the system charge. Energy minimization was performed with 2500 steps of steepest descent followed by 2500 steps of conjugate gradient<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. The system was then equilibrated using a 100 ps NVT ensemble simulation and a 100 ps NPT ensemble simulation at a temperature of 298.15 K<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. A 100 ns MD ensemble simulation under periodic boundary conditions was subsequently carried out. The Particle Mesh Ewald method was employed for long-range electrostatic interactions, using a 1 nm cutoff for non-bonded interactions, a collision frequency of 2 ps, system pressure set to 101.325 kPa, an integration step of 2 s, and trajectory recording every 10 ps<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. Stable molecular conformations from the last 20 ns of the equilibrium trajectory were extracted for the analysis of the model and the strength of interactions between the protein and the ligand.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Quantum mechanics/molecular mechanics (QM/MM) calculation</h3>
<p id="Par56">QM/MM calculations were performed to investigate the reaction mechanism at the active site of GbCYP90J6. Representative snapshots were extracted from the MD trajectory by selecting frames that showed both a minimal distance between the oxo group of Cpd I and the target hydroxyl proton, and a close spatial proximity between the C3’ and C8” carbons of the substrates. All QM/MM calculations were conducted using the ChemShell (version 23.0.3) software package<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>. The QM/MM system included all amino acid residues within 12 Å of the protein. The QM region comprised the apigenin substrates, a heme iron porphyrin axially coordinated by C425, and residues V215, I216, E291, E292, T293, S294, P295, G359, and V360 (see Supplementary Fig. <a href="#MOESM1" class="usa-link">22</a>). These residues were selected based on the criterion that they formed hydrogen bonds or hydrophobic interactions with the substrates or facilitated substrates positioning within the active site. The MM region was treated with the AMBER force field<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>. The active region during QM/MM calculations included all the residues and solvent molecules within 12 Å from the Fe center. During the QM/MM geometry optimization, the QM region was treated with the hybrid UB3LYP<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup> density functional. For geometry optimization and scanning calculations, the def2-SVP<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup> basis set was used. Then, the single-point calculations were conducted with the higher-level basis set def2-TZVP<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. The D3(BJ) dispersion<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup> correction was employed for all QM/MM computations. The coordinates of all geometries obtained from the QM/MM calculations are provided in Supplementary Data <a href="#MOESM8" class="usa-link">6</a>.</p></section><section id="Sec24"><h3 class="pmc_sec_title">De novo biosynthesis of amentoflavone in engineered <em>E. coli</em>
</h3>
<p id="Par57">The <em>E. coli</em> GJA3, engineered for high <em>L</em>-Tyr production, was constructed from JM109(DE3) by knocking out <em>tyrR</em> and <em>pheA</em> genes, followed by inserting <em>aroG</em><sup><em>fbr</em></sup>, <em>tyrA</em><sup><em>fbr</em></sup>, <em>ydiB</em>, <em>tktA,</em> and <em>ppsA</em> genes into the <em>tyrR</em> locus using CRISPR-Cas9-assisted genome editing according to the protocol described by Yang et al.<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup>. The CRISPR-related platmids pQZ261<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup> and pQZ265<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>, as well as plasmid pYH57<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> harboring apigenin monomer generation module, were previously constructed in our laboratory.</p>
<p id="Par58">The starter cultures of engineered strains were precultured at 37 °C in LB medium supplemented with appropriate antibiotics overnight and then inoculated (1:100) into 10 mL of TB medium containing 20 g·L<sup>−1</sup> glycerol, with three replicates for each experiment. After reaching an OD<sub>600</sub> of 0.6–0.8, 0.1 mM IPTG was added to the cooled cultures. The cultures were fermented at 22 °C, 220 rpm for 72 h.</p>
<p id="Par59">For <em>L</em>-Tyr quantification, the culture supernatant was acidified with 1 N HCl and analyzed using HPLC. For flavonoid quantification, 1 mL of fermentation broth was ultrasonicated and extracted three times with 1 mL of ethyl acetate. All extracts were evaporated to dryness and re-dissolved in 200 μL of methanol for subsequent analysis.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Statistics &amp; reproducibility</h3>
<p id="Par60">Quantitative experiments were conducted with three independent biological replicates unless otherwise stated. Results, including enzyme activity, metabolite concentrations, and product yields, are presented as mean ± standard deviation (s.d.). No statistical methods were used to predetermine sample size. No data were excluded from the analysis. The experiments were not randomized, and investigators were not blinded to group allocation or outcome assessment.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par61">Further information on research design is available in the <a href="#MOESM9" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec27"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (6.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62990_MOESM2_ESM.pdf</a><sup> (162.8KB, pdf) </sup><p>Description of additional supplementary files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (4.5MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM4_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (4.5MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (196.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM6_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (4.5MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM7_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (4.5MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM8_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 6</a><sup> (4.3MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (2.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM10_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (38MB, pdf) </sup>
</div></div></section></section><section id="Sec28"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM11_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (9.2MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work is financially supported by the National Key R&amp;D Program of China (No. 2024YFA0919900) to Y.W. and H.-L.L., the State Key Laboratory of Plant Trait Design to Y.W., the Key Laboratory of Plant Carbon Capture, Chinese Academy of Sciences to Y.W., and the Shanghai Municipal Science and Technology Major Project to Y.W. We thank Prof. David R. Nelson (University of Tennessee Health Science Center) for his assistance on the classification of the CYP90J subfamily referenced in this study.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Y.W. conceptualized the project. Y.W. and H.-L.L. sourced the funding and managed the whole project. Y.W., H.-L.L., Y.-H.R., and X.-H.D. designed the research. X.-H.D. conducted experiments and analysis. J.-M.Z. prepared the <em>G. biloba</em> plant materials and conducted the transcriptome sequencing. G.-Y.W. proposed the initial catalysis mechanism and constructed the <em>L</em>-tyrosine high-producing strain GJA3. X.-H.D. compiled the data and drafted the manuscript. Y.W., H.-L.L., and X.-H.D. revised the manuscript. All authors reviewed and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par62"><em>Nature Communications</em> thanks Lorenzo Caputi, who co-reviewed with Qi Ding and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. [A peer review file is available].</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The data supporting the findings of this study are available within the paper and its supplementary information files. The raw transcriptome data of <em>G. biloba</em> has been deposited in the Genome Sequence Archive and is accessible under accession number <a href="https://ngdc.cncb.ac.cn/gsa/search?searchTerm=CRA019958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CRA019958</a>. The sequences of GbCYP90s reported in this study have been submitted to the NCBI GenBank under accession numbers GbCYP90J6, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV862377" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV862377</a>; GbCYP90D59, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV862378" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV862378</a>; GbCYP90A68, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV862379" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV862379</a>; GbCYP90B61, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV862380" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV862380</a>. Additional CYP90 sequences of this study are provided in Supplementary Data <a href="#MOESM3" class="usa-link">1</a>. The mass spectrometry data generated in this study are available in Figshare [10.6084/m9.figshare.29552828.v1]. <a href="#Sec28" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par63">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Hai-Li Liu, Email: hlliu@cemps.ac.cn.</p>
<p>Yong Wang, Email: yongwang@cemps.ac.cn.</p></section><section id="sec29"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62990-6.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>World Health Organization. <em>Global status report on the public health response to dementia</em> (World Health Organization, Geneva, 2021). <a href="https://www.who.int/publications/i/item/9789240033245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.who.int/publications/i/item/9789240033245</a>.</cite>
</li>
<li id="CR2">
<span class="label">2.</span><cite>Menezes, J. C. J. M. D. S. &amp; Diederich, M. F. Bioactivity of natural biflavonoids in metabolism-related disease and cancer therapies. <em>Pharmacol. Res.</em><strong>167</strong>, 105525 (2021).
</cite> [<a href="https://doi.org/10.1016/j.phrs.2021.105525" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33667686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Menezes,%20J.%20C.%20J.%20M.%20D.%20S.%20&amp;%20Diederich,%20M.%20F.%20Bioactivity%20of%20natural%20biflavonoids%20in%20metabolism-related%20disease%20and%20cancer%20therapies.%20Pharmacol.%20Res.167,%20105525%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Uddin, M. S. et al. Emerging signal regulating potential of small molecule biflavonoids to combat neuropathological insults of Alzheimer’s disease. <em>Sci. Total Environ.</em><strong>700</strong>, 134836 (2020).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2019.134836" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31704512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Uddin,%20M.%20S.%20et%20al.%20Emerging%20signal%20regulating%20potential%20of%20small%20molecule%20biflavonoids%20to%20combat%20neuropathological%20insults%20of%20Alzheimer%E2%80%99s%20disease.%20Sci.%20Total%20Environ.700,%20134836%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Zhang, Z. et al. Amentoflavone protects hippocampal neurons: anti-inflammatory, antioxidative, and antiapoptotic effects. <em>Neural Regen. Res.</em><strong>10</strong>, 1125 (2015).
</cite> [<a href="https://doi.org/10.4103/1673-5374.160109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4541246/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26330838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Z.%20et%20al.%20Amentoflavone%20protects%20hippocampal%20neurons:%20anti-inflammatory,%20antioxidative,%20and%20antiapoptotic%20effects.%20Neural%20Regen.%20Res.10,%201125%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Tatlı Çankaya, İİ, Devkota, H. P., Zengin, G. &amp; Šamec, D. Neuroprotective potential of biflavone ginkgetin: a review. <em>Life</em><strong>13</strong>, 562 (2023).
</cite> [<a href="https://doi.org/10.3390/life13020562" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9964866/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36836918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tatl%C4%B1%20%C3%87ankaya,%20%C4%B0%C4%B0,%20Devkota,%20H.%20P.,%20Zengin,%20G.%20&amp;%20%C5%A0amec,%20D.%20Neuroprotective%20potential%20of%20biflavone%20ginkgetin:%20a%20review.%20Life13,%20562%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Zetzsche, L. E. et al. Biocatalytic oxidative cross-coupling reactions for biaryl bond formation. <em>Nature</em><strong>603</strong>, 79–85 (2022).
</cite> [<a href="https://doi.org/10.1038/s41586-021-04365-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9213091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35236972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zetzsche,%20L.%20E.%20et%20al.%20Biocatalytic%20oxidative%20cross-coupling%20reactions%20for%20biaryl%20bond%20formation.%20Nature603,%2079%E2%80%9385%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Ihl, R. <em>Gingko biloba</em> extract EGb 761®: clinical data in dementia. <em>Int. Psychogeriatr.</em><strong>24</strong>, S35–S40 (2012).
</cite> [<a href="https://doi.org/10.1017/S1041610212000543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22784427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ihl,%20R.%20Gingko%20biloba%20extract%20EGb%20761%C2%AE:%20clinical%20data%20in%20dementia.%20Int.%20Psychogeriatr.24,%20S35%E2%80%93S40%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Gavrilova, S. I. et al. Efficacy and safety of <em>Ginkgo biloba</em> extract EGb 761® in mild cognitive impairment with neuropsychiatric symptoms: a randomized, placebo-controlled, double-blind, multi-center trial. <em>Int. J. Geriatr. Psychiatry</em><strong>29</strong>, 1087–1095 (2014).
</cite> [<a href="https://doi.org/10.1002/gps.4103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24633934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gavrilova,%20S.%20I.%20et%20al.%20Efficacy%20and%20safety%20of%20Ginkgo%20biloba%20extract%20EGb%20761%C2%AE%20in%20mild%20cognitive%20impairment%20with%20neuropsychiatric%20symptoms:%20a%20randomized,%20placebo-controlled,%20double-blind,%20multi-center%20trial.%20Int.%20J.%20Geriatr.%20Psychiatry29,%201087%E2%80%931095%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Šalić, A. et al. Extraction of polyphenolic compounds from ginkgo leaves using deep eutectic solvents: a potential solution for the sustainable and environmentally friendly isolation of biflavonoids. <em>Ind. Crops Prod.</em><strong>219</strong>, 119068 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?%C5%A0ali%C4%87,%20A.%20et%20al.%20Extraction%20of%20polyphenolic%20compounds%20from%20ginkgo%20leaves%20using%20deep%20eutectic%20solvents:%20a%20potential%20solution%20for%20the%20sustainable%20and%20environmentally%20friendly%20isolation%20of%20biflavonoids.%20Ind.%20Crops%20Prod.219,%20119068%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Rahman, M., Riaz, M. &amp; Desai, U. R. Synthesis of biologically relevant biflavanoids–a review. <em>Chem. Biodivers.</em><strong>4</strong>, 2495–2527 (2007).
</cite> [<a href="https://doi.org/10.1002/cbdv.200790205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18027351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rahman,%20M.,%20Riaz,%20M.%20&amp;%20Desai,%20U.%20R.%20Synthesis%20of%20biologically%20relevant%20biflavanoids%E2%80%93a%20review.%20Chem.%20Biodivers.4,%202495%E2%80%932527%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Fregoso-López, D. &amp; Miranda, L. D. Visible-light mediated radical alkylation of flavones: a modular access to nonsymmetrical 3,3″-biflavones. <em>Org. Lett.</em><strong>24</strong>, 8615–8620 (2022).
</cite> [<a href="https://doi.org/10.1021/acs.orglett.2c03415" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36394479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fregoso-L%C3%B3pez,%20D.%20&amp;%20Miranda,%20L.%20D.%20Visible-light%20mediated%20radical%20alkylation%20of%20flavones:%20a%20modular%20access%20to%20nonsymmetrical%203,3%E2%80%B3-biflavones.%20Org.%20Lett.24,%208615%E2%80%938620%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>He, X., Yang, F. &amp; Huang, X. Proceedings of chemistry, pharmacology, pharmacokinetics and synthesis of biflavonoids. <em>Molecules</em><strong>26</strong>, 6088 (2021).
</cite> [<a href="https://doi.org/10.3390/molecules26196088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8512048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34641631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He,%20X.,%20Yang,%20F.%20&amp;%20Huang,%20X.%20Proceedings%20of%20chemistry,%20pharmacology,%20pharmacokinetics%20and%20synthesis%20of%20biflavonoids.%20Molecules26,%206088%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Baker, W. &amp; Simmonds, W. H. C. 258. Derivatives of 5: 6: 4′-and 5: 8: 4′-trihydroxyflavones, and a note on the structure of ginkgetin. <em>J. Chem. Soc. (Resumed)</em> 1370–1374 10.1039/JR9400001370 (1940).</cite>
</li>
<li id="CR14">
<span class="label">14.</span><cite>Sih, C. J. &amp; Malnar, I. Peroxidase-catalyzed oxidative phenolic coupling of some tripeptides containing tyrosine and hydroxyphenylglycine residues. <em>Biocatal. Biotransfor.</em><strong>18</strong>, 301–310 (2000).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sih,%20C.%20J.%20&amp;%20Malnar,%20I.%20Peroxidase-catalyzed%20oxidative%20phenolic%20coupling%20of%20some%20tripeptides%20containing%20tyrosine%20and%20hydroxyphenylglycine%20residues.%20Biocatal.%20Biotransfor.18,%20301%E2%80%93310%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Kudanga, T., Nemadziva, B. &amp; Le Roes-Hill, M. Laccase catalysis for the synthesis of bioactive compounds. <em>Appl. Microbiol. Biotechnol.</em><strong>101</strong>, 13–33 (2017).
</cite> [<a href="https://doi.org/10.1007/s00253-016-7987-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27872999/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kudanga,%20T.,%20Nemadziva,%20B.%20&amp;%20Le%20Roes-Hill,%20M.%20Laccase%20catalysis%20for%20the%20synthesis%20of%20bioactive%20compounds.%20Appl.%20Microbiol.%20Biotechnol.101,%2013%E2%80%9333%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Guyot, S., Vercauteren, J. &amp; Cheynier, V. Structural determination of colourless and yellow dimers resulting from (+)-catechin coupling catalysed by grape polyphenoloxidase. <em>Phytochemistry</em><strong>42</strong>, 1279–1288 (1996).</cite> [<a href="https://scholar.google.com/scholar_lookup?Guyot,%20S.,%20Vercauteren,%20J.%20&amp;%20Cheynier,%20V.%20Structural%20determination%20of%20colourless%20and%20yellow%20dimers%20resulting%20from%20(+)-catechin%20coupling%20catalysed%20by%20grape%20polyphenoloxidase.%20Phytochemistry42,%201279%E2%80%931288%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Hüttel, W. &amp; Müller, M. Regio- and stereoselective intermolecular phenol coupling enzymes in secondary metabolite biosynthesis. <em>Nat. Prod. Rep.</em><strong>38</strong>, 1011–1043 (2021).
</cite> [<a href="https://doi.org/10.1039/d0np00010h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33196733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?H%C3%BCttel,%20W.%20&amp;%20M%C3%BCller,%20M.%20Regio-%20and%20stereoselective%20intermolecular%20phenol%20coupling%20enzymes%20in%20secondary%20metabolite%20biosynthesis.%20Nat.%20Prod.%20Rep.38,%201011%E2%80%931043%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Liu, J., Liu, A. &amp; Hu, Y. Enzymatic dimerization in the biosynthetic pathway of microbial natural products. <em>Nat. Prod. Rep.</em><strong>38</strong>, 1469–1505 (2021).
</cite> [<a href="https://doi.org/10.1039/d0np00063a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33404031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20J.,%20Liu,%20A.%20&amp;%20Hu,%20Y.%20Enzymatic%20dimerization%20in%20the%20biosynthetic%20pathway%20of%20microbial%20natural%20products.%20Nat.%20Prod.%20Rep.38,%201469%E2%80%931505%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Funa, N., Funabashi, M., Ohnishi, Y. &amp; Horinouchi, S. Biosynthesis of hexahydroxyperylenequinone melanin via oxidative aryl coupling by cytochrome P-450 in <em>Streptomyces griseus</em>. <em>J. Bacteriol.</em><strong>187</strong>, 8149–8155 (2005).
</cite> [<a href="https://doi.org/10.1128/JB.187.23.8149-8155.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1291289/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16291687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Funa,%20N.,%20Funabashi,%20M.,%20Ohnishi,%20Y.%20&amp;%20Horinouchi,%20S.%20Biosynthesis%20of%20hexahydroxyperylenequinone%20melanin%20via%20oxidative%20aryl%20coupling%20by%20cytochrome%20P-450%20in%20Streptomyces%20griseus.%20J.%20Bacteriol.187,%208149%E2%80%938155%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Shende, V. V. et al. Structure and function of NzeB, a versatile C–C and C–N bond-forming diketopiperazine dimerase. <em>J. Am. Chem. Soc.</em><strong>142</strong>, 17413–17424 (2020).
</cite> [<a href="https://doi.org/10.1021/jacs.0c06312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7572911/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32786740/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shende,%20V.%20V.%20et%20al.%20Structure%20and%20function%20of%20NzeB,%20a%20versatile%20C%E2%80%93C%20and%20C%E2%80%93N%20bond-forming%20diketopiperazine%20dimerase.%20J.%20Am.%20Chem.%20Soc.142,%2017413%E2%80%9317424%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Mazzaferro, L. S., Hüttel, W., Fries, A. &amp; Müller, M. Cytochrome P450-catalyzed regio-and stereoselective phenol coupling of fungal natural products. <em>J. Am. Chem. Soc.</em><strong>137</strong>, 12289–12295 (2015).
</cite> [<a href="https://doi.org/10.1021/jacs.5b06776" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26389790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mazzaferro,%20L.%20S.,%20H%C3%BCttel,%20W.,%20Fries,%20A.%20&amp;%20M%C3%BCller,%20M.%20Cytochrome%20P450-catalyzed%20regio-and%20stereoselective%20phenol%20coupling%20of%20fungal%20natural%20products.%20J.%20Am.%20Chem.%20Soc.137,%2012289%E2%80%9312295%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Sun, C. et al. Molecular basis of regio- and stereo-specificity in biosynthesis of bacterial heterodimeric diketopiperazines. <em>Nat. Commun.</em><strong>11</strong>, 6251 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-20022-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7721796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33288748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20C.%20et%20al.%20Molecular%20basis%20of%20regio-%20and%20stereo-specificity%20in%20biosynthesis%20of%20bacterial%20heterodimeric%20diketopiperazines.%20Nat.%20Commun.11,%206251%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Ma, C. et al. Exploring the diverse landscape of fungal cytochrome P450-catalyzed regio- and stereoselective dimerization of diketopiperazines. <em>Adv. Sci.</em><strong>11</strong>, 2310018 (2024).</cite> [<a href="https://doi.org/10.1002/advs.202310018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11234459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38687842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20C.%20et%20al.%20Exploring%20the%20diverse%20landscape%20of%20fungal%20cytochrome%20P450-catalyzed%20regio-%20and%20stereoselective%20dimerization%20of%20diketopiperazines.%20Adv.%20Sci.11,%202310018%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Shende, V. V. et al. Molecular dynamics simulations guide chimeragenesis and engineered control of chemoselectivity in diketopiperazine dimerases. <em>Angew. Chem. Int. Ed.</em><strong>62</strong>, e202210254 (2023).</cite> [<a href="https://doi.org/10.1002/anie.202210254" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10159983/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36610039/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shende,%20V.%20V.%20et%20al.%20Molecular%20dynamics%20simulations%20guide%20chimeragenesis%20and%20engineered%20control%20of%20chemoselectivity%20in%20diketopiperazine%20dimerases.%20Angew.%20Chem.%20Int.%20Ed.62,%20e202210254%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Kardam, V., Bhatt, V. &amp; Dubey, K. D. Structural and mechanistic insights into oxidative biaryl coupling to form the arylomycin core by an engineered CYP450. <em>Dalton Trans.</em><strong>54</strong>, 754–763 (2025).
</cite> [<a href="https://doi.org/10.1039/d4dt02197e" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39575560/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kardam,%20V.,%20Bhatt,%20V.%20&amp;%20Dubey,%20K.%20D.%20Structural%20and%20mechanistic%20insights%20into%20oxidative%20biaryl%20coupling%20to%20form%20the%20arylomycin%20core%20by%20an%20engineered%20CYP450.%20Dalton%20Trans.54,%20754%E2%80%93763%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Fujiyama, K. et al. Structural insights into a key step of brassinosteroid biosynthesis and its inhibition. <em>Nat. Plants</em><strong>5</strong>, 589–594 (2019).
</cite> [<a href="https://doi.org/10.1038/s41477-019-0436-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31182839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fujiyama,%20K.%20et%20al.%20Structural%20insights%20into%20a%20key%20step%20of%20brassinosteroid%20biosynthesis%20and%20its%20inhibition.%20Nat.%20Plants5,%20589%E2%80%93594%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Mohanta, T. K., Tamboli, Y. &amp; Zubaidha, P. K. Phytochemical and medicinal importance of <em>Ginkgo biloba</em> L. <em>Nat. Prod. Res.</em><strong>28</strong>, 746–752 (2014).
</cite> [<a href="https://doi.org/10.1080/14786419.2013.879303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24499319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mohanta,%20T.%20K.,%20Tamboli,%20Y.%20&amp;%20Zubaidha,%20P.%20K.%20Phytochemical%20and%20medicinal%20importance%20of%20Ginkgo%20biloba%20L.%20Nat.%20Prod.%20Res.28,%20746%E2%80%93752%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Liu, H. et al. The nearly complete genome of <em>Ginkgo biloba</em> illuminates gymnosperm evolution. <em>Nat. Plants</em><strong>7</strong>, 748–756 (2021).
</cite> [<a href="https://doi.org/10.1038/s41477-021-00933-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34135482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20H.%20et%20al.%20The%20nearly%20complete%20genome%20of%20Ginkgo%20biloba%20illuminates%20gymnosperm%20evolution.%20Nat.%20Plants7,%20748%E2%80%93756%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Geiger, H. Biflavones from some mosses. <em>Z. Naturforsch. C.</em><strong>43</strong>, 1–4 (1988).</cite> [<a href="https://scholar.google.com/scholar_lookup?Geiger,%20H.%20Biflavones%20from%20some%20mosses.%20Z.%20Naturforsch.%20C.43,%201%E2%80%934%20(1988)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Suárez, A. I., Diaz, M. B., Delle Monache, F. &amp; Compagnone, R. S. Biflavonoids from <em>Podocalyx loranthoides</em>. <em>Fitoterapia</em><strong>74</strong>, 473–475 (2003).
</cite> [<a href="https://doi.org/10.1016/s0367-326x(03)00065-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12837364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su%C3%A1rez,%20A.%20I.,%20Diaz,%20M.%20B.,%20Delle%20Monache,%20F.%20&amp;%20Compagnone,%20R.%20S.%20Biflavonoids%20from%20Podocalyx%20loranthoides.%20Fitoterapia74,%20473%E2%80%93475%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Leong, K. I., Alviarez, P. F., Compagnone, R. S. &amp; Suárez, A. I. Isolation and structural elucidation of chemical constituents of Amanoa almerindae. <em>Pharm. Biol.</em><strong>47</strong>, 496–499 (2009).</cite> [<a href="https://scholar.google.com/scholar_lookup?Leong,%20K.%20I.,%20Alviarez,%20P.%20F.,%20Compagnone,%20R.%20S.%20&amp;%20Su%C3%A1rez,%20A.%20I.%20Isolation%20and%20structural%20elucidation%20of%20chemical%20constituents%20of%20Amanoa%20almerindae.%20Pharm.%20Biol.47,%20496%E2%80%93499%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Li, X.-C. et al. Fatty acid synthase inhibitors from plants: isolation, structure elucidation, and SAR studies. <em>J. Nat. Prod.</em><strong>65</strong>, 1909–1914 (2002).
</cite> [<a href="https://doi.org/10.1021/np020289t" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12502337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20X.-C.%20et%20al.%20Fatty%20acid%20synthase%20inhibitors%20from%20plants:%20isolation,%20structure%20elucidation,%20and%20SAR%20studies.%20J.%20Nat.%20Prod.65,%201909%E2%80%931914%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Šamec, D. et al. 3′-8″- Biflavones: a review of their structural diversity, natural occurrence, role in plants, extraction and identification. <em>Molecules</em><strong>29</strong>, 4634 (2024).
</cite> [<a href="https://doi.org/10.3390/molecules29194634" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11478198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39407564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?%C5%A0amec,%20D.%20et%20al.%203%E2%80%B2-8%E2%80%B3-%20Biflavones:%20a%20review%20of%20their%20structural%20diversity,%20natural%20occurrence,%20role%20in%20plants,%20extraction%20and%20identification.%20Molecules29,%204634%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Nelson, D. &amp; Werck-Reichhart, D. A P450-centric view of plant evolution. <em>Plant J.</em><strong>66</strong>, 194–211 (2011).
</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2011.04529.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21443632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nelson,%20D.%20&amp;%20Werck-Reichhart,%20D.%20A%20P450-centric%20view%20of%20plant%20evolution.%20Plant%20J.66,%20194%E2%80%93211%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Banks, J. A. et al. The <em>Selaginella</em> genome identifies genetic changes associated with the evolution of vascular plants. <em>Science</em><strong>332</strong>, 960–963 (2011).
</cite> [<a href="https://doi.org/10.1126/science.1203810" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3166216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21551031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banks,%20J.%20A.%20et%20al.%20The%20Selaginella%20genome%20identifies%20genetic%20changes%20associated%20with%20the%20evolution%20of%20vascular%20plants.%20Science332,%20960%E2%80%93963%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Li, C. et al. Extraordinary preservation of gene collinearity over three hundred million years revealed in homosporous lycophytes. <em>Proc. Natl. Acad. Sci.</em><strong>121</strong>, e2312607121 (2024).
</cite> [<a href="https://doi.org/10.1073/pnas.2312607121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10823260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38236735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20C.%20et%20al.%20Extraordinary%20preservation%20of%20gene%20collinearity%20over%20three%20hundred%20million%20years%20revealed%20in%20homosporous%20lycophytes.%20Proc.%20Natl.%20Acad.%20Sci.121,%20e2312607121%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Fang, Y. et al. The genome of homosporous maidenhair fern sheds light on the euphyllophyte evolution and defences. <em>Nat. Plants</em><strong>8</strong>, 1024–1037 (2022).
</cite> [<a href="https://doi.org/10.1038/s41477-022-01222-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7613604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36050462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fang,%20Y.%20et%20al.%20The%20genome%20of%20homosporous%20maidenhair%20fern%20sheds%20light%20on%20the%20euphyllophyte%20evolution%20and%20defences.%20Nat.%20Plants8,%201024%E2%80%931037%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Liu, Y. et al. The <em>Cycas</em> genome and the early evolution of seed plants. <em>Nat. Plants</em><strong>8</strong>, 389–401 (2022).
</cite> [<a href="https://doi.org/10.1038/s41477-022-01129-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9023351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35437001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.%20et%20al.%20The%20Cycas%20genome%20and%20the%20early%20evolution%20of%20seed%20plants.%20Nat.%20Plants8,%20389%E2%80%93401%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Wan, T. et al. A genome for gnetophytes and early evolution of seed plants. <em>Nat. Plants</em><strong>4</strong>, 82–89 (2018).
</cite> [<a href="https://doi.org/10.1038/s41477-017-0097-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29379155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wan,%20T.%20et%20al.%20A%20genome%20for%20gnetophytes%20and%20early%20evolution%20of%20seed%20plants.%20Nat.%20Plants4,%2082%E2%80%9389%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Xiong, X. et al. The <em>Taxus</em> genome provides insights into paclitaxel biosynthesis. <em>Nat. Plants</em><strong>7</strong>, 1026–1036 (2021).
</cite> [<a href="https://doi.org/10.1038/s41477-021-00963-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8367818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34267359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiong,%20X.%20et%20al.%20The%20Taxus%20genome%20provides%20insights%20into%20paclitaxel%20biosynthesis.%20Nat.%20Plants7,%201026%E2%80%931036%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Wan, T. et al. The <em>Welwitschia</em> genome reveals a unique biology underpinning extreme longevity in deserts. <em>Nat. Commun.</em><strong>12</strong>, 4247 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-24528-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8275611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34253727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wan,%20T.%20et%20al.%20The%20Welwitschia%20genome%20reveals%20a%20unique%20biology%20underpinning%20extreme%20longevity%20in%20deserts.%20Nat.%20Commun.12,%204247%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Shalev, T. J. et al. The western redcedar genome reveals low genetic diversity in a self-compatible conifer. <em>Genome Res.</em><strong>32</strong>, 1952–1964 (2022).
</cite> [<a href="https://doi.org/10.1101/gr.276358.121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9712635/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36109148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shalev,%20T.%20J.%20et%20al.%20The%20western%20redcedar%20genome%20reveals%20low%20genetic%20diversity%20in%20a%20self-compatible%20conifer.%20Genome%20Res.32,%201952%E2%80%931964%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Fu, F. et al. The <em>Metasequoia</em> genome and evolutionary relationships among redwoods. <em>Plant Commun.</em><strong>4</strong>, 100643 (2023).
</cite> [<a href="https://doi.org/10.1016/j.xplc.2023.100643" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10775903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37381601/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fu,%20F.%20et%20al.%20The%20Metasequoia%20genome%20and%20evolutionary%20relationships%20among%20redwoods.%20Plant%20Commun.4,%20100643%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Wang, B. et al. High-quality <em>Arabidopsis Thaliana</em> genome assembly with nanopore and HiFi long reads. <em>Genomics Proteom. Bioinforma.</em><strong>20</strong>, 4–13 (2022).</cite> [<a href="https://doi.org/10.1016/j.gpb.2021.08.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9510872/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34487862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20B.%20et%20al.%20High-quality%20Arabidopsis%20Thaliana%20genome%20assembly%20with%20nanopore%20and%20HiFi%20long%20reads.%20Genomics%20Proteom.%20Bioinforma.20,%204%E2%80%9313%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Hansen, C. C., Nelson, D. R., Møller, B. L. &amp; Werck-Reichhart, D. Plant cytochrome P450 plasticity and evolution. <em>Mol. Plant</em><strong>14</strong>, 1244–1265 (2021).
</cite> [<a href="https://doi.org/10.1016/j.molp.2021.06.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34216829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hansen,%20C.%20C.,%20Nelson,%20D.%20R.,%20M%C3%B8ller,%20B.%20L.%20&amp;%20Werck-Reichhart,%20D.%20Plant%20cytochrome%20P450%20plasticity%20and%20evolution.%20Mol.%20Plant14,%201244%E2%80%931265%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Commisso, M. et al. Bryo-Activities: a review on how bryophytes are contributing to the arsenal of natural bioactive compounds against fungi. <em>Plants</em><strong>10</strong>, 203 (2021).
</cite> [<a href="https://doi.org/10.3390/plants10020203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7911284/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33494524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Commisso,%20M.%20et%20al.%20Bryo-Activities:%20a%20review%20on%20how%20bryophytes%20are%20contributing%20to%20the%20arsenal%20of%20natural%20bioactive%20compounds%20against%20fungi.%20Plants10,%20203%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Wallace, J. W. C-Glycosylflavones in the <em>Gnetopsida</em>: a preliminary report. <em>Am. J. Bot.</em><strong>66</strong>, 343–346 (1979).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wallace,%20J.%20W.%20C-Glycosylflavones%20in%20the%20Gnetopsida:%20a%20preliminary%20report.%20Am.%20J.%20Bot.66,%20343%E2%80%93346%20(1979)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Geiger, H. &amp; Quinn, C. Biflavonoids. In <em>The Flavonoids</em> 692–742 10.1007/978-1-4899-2909-9_13 (Springer US, Boston, MA, 1975).</cite>
</li>
<li id="CR49">
<span class="label">49.</span><cite>Kariyone, T. &amp; Sawada, T. Studies on flavonoids of the leaves of coniferae and allied plants. I: on the flavonoid from the leaves of <em>Torreya nucifera</em> SIEB. et ZUCC. <em>J. Pharm. Soc. Jpn.</em><strong>78</strong>, 1010–1013 (1958).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kariyone,%20T.%20&amp;%20Sawada,%20T.%20Studies%20on%20flavonoids%20of%20the%20leaves%20of%20coniferae%20and%20allied%20plants.%20I:%20on%20the%20flavonoid%20from%20the%20leaves%20of%20Torreya%20nucifera%20SIEB.%20et%20ZUCC.%20J.%20Pharm.%20Soc.%20Jpn.78,%201010%E2%80%931013%20(1958)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Anand, A. V., Arinchedathu Surendran, V. &amp; Sukumaran, S. T. Bioactive metabolites in gymnosperms. In <em>Plant Metabolites: Methods, Applications and Prospects</em> 317–346 10.1007/978-981-15-5136-9_14 (Springer, Singapore, 2020).</cite>
</li>
<li id="CR51">
<span class="label">51.</span><cite>Adasme, M. F. et al. PLIP 2021: expanding the scope of the protein–ligand interaction profiler to DNA and RNA. <em>Nucleic Acids Res.</em><strong>49</strong>, W530–W534 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab294" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8262720/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33950214/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adasme,%20M.%20F.%20et%20al.%20PLIP%202021:%20expanding%20the%20scope%20of%20the%20protein%E2%80%93ligand%20interaction%20profiler%20to%20DNA%20and%20RNA.%20Nucleic%20Acids%20Res.49,%20W530%E2%80%93W534%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Moreno, A. G. et al. Structure, isomerization and dimerization processes of naringenin flavonoids. <em>Phys. Chem. Chem. Phys.</em><strong>23</strong>, 18068–18077 (2021).
</cite> [<a href="https://doi.org/10.1039/d1cp01161h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34388230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moreno,%20A.%20G.%20et%20al.%20Structure,%20isomerization%20and%20dimerization%20processes%20of%20naringenin%20flavonoids.%20Phys.%20Chem.%20Chem.%20Phys.23,%2018068%E2%80%9318077%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Yu, X. et al. Biosynthesis of strained piperazine alkaloids: uncovering the concise pathway of herquline A. <em>J. Am. Chem. Soc.</em><strong>138</strong>, 13529–13532 (2016).
</cite> [<a href="https://doi.org/10.1021/jacs.6b09464" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5378693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27690412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20X.%20et%20al.%20Biosynthesis%20of%20strained%20piperazine%20alkaloids:%20uncovering%20the%20concise%20pathway%20of%20herquline%20A.%20J.%20Am.%20Chem.%20Soc.138,%2013529%E2%80%9313532%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Li, S.-H. et al. Chemoenzymatic synthesis of fluorinated mycocyclosin enabled by the engineered cytochrome P450-catalyzed biaryl coupling reaction. <em>J. Am. Chem. Soc.</em><strong>146</strong>, 19962–19973 (2024).
</cite> [<a href="https://doi.org/10.1021/jacs.4c03499" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38985576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20S.-H.%20et%20al.%20Chemoenzymatic%20synthesis%20of%20fluorinated%20mycocyclosin%20enabled%20by%20the%20engineered%20cytochrome%20P450-catalyzed%20biaryl%20coupling%20reaction.%20J.%20Am.%20Chem.%20Soc.146,%2019962%E2%80%9319973%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Nakagawa, A. et al. A bacterial platform for fermentative production of plant alkaloids. <em>Nat. Commun.</em><strong>2</strong>, 326 (2011).
</cite> [<a href="https://doi.org/10.1038/ncomms1327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3112539/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21610729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakagawa,%20A.%20et%20al.%20A%20bacterial%20platform%20for%20fermentative%20production%20of%20plant%20alkaloids.%20Nat.%20Commun.2,%20326%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Li, J. et al. Production of plant-specific flavones baicalein and scutellarein in an engineered <em>E. coli</em> from available phenylalanine and tyrosine. <em>Metab. Eng.</em><strong>52</strong>, 124–133 (2019).
</cite> [<a href="https://doi.org/10.1016/j.ymben.2018.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30496827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20J.%20et%20al.%20Production%20of%20plant-specific%20flavones%20baicalein%20and%20scutellarein%20in%20an%20engineered%20E.%20coli%20from%20available%20phenylalanine%20and%20tyrosine.%20Metab.%20Eng.52,%20124%E2%80%93133%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Mizutani, M. &amp; Sato, F. Unusual P450 reactions in plant secondary metabolism. <em>Arch. Biochem. Biophys.</em><strong>507</strong>, 194–203 (2011).
</cite> [<a href="https://doi.org/10.1016/j.abb.2010.09.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20920462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mizutani,%20M.%20&amp;%20Sato,%20F.%20Unusual%20P450%20reactions%20in%20plant%20secondary%20metabolism.%20Arch.%20Biochem.%20Biophys.507,%20194%E2%80%93203%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>An, Z. et al. Lineage-specific CYP80 expansion and benzylisoquinoline alkaloid diversity in early-diverging eudicots. <em>Adv. Sci.</em><strong>11</strong>, 2309990 (2024).</cite> [<a href="https://doi.org/10.1002/advs.202309990" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11109638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38477432/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?An,%20Z.%20et%20al.%20Lineage-specific%20CYP80%20expansion%20and%20benzylisoquinoline%20alkaloid%20diversity%20in%20early-diverging%20eudicots.%20Adv.%20Sci.11,%202309990%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Meng, F. et al. Characterization of two CYP80 enzymes provides insights into aporphine alkaloid skeleton formation in <em>Aristolochia contorta</em>. <em>Plant J.</em><strong>118</strong>, 1439–1454 (2024).
</cite> [<a href="https://doi.org/10.1111/tpj.16686" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38379355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meng,%20F.%20et%20al.%20Characterization%20of%20two%20CYP80%20enzymes%20provides%20insights%20into%20aporphine%20alkaloid%20skeleton%20formation%20in%20Aristolochia%20contorta.%20Plant%20J.118,%201439%E2%80%931454%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Liu, Y. et al. Ginkgetin alleviates inflammation and senescence by targeting STING. <em>Adv. Sci.</em><strong>12</strong>, 2407222 (2025).</cite> [<a href="https://doi.org/10.1002/advs.202407222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11727237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39558862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.%20et%20al.%20Ginkgetin%20alleviates%20inflammation%20and%20senescence%20by%20targeting%20STING.%20Adv.%20Sci.12,%202407222%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Smer-Barreto, V. et al. Discovery of senolytics using machine learning. <em>Nat. Commun.</em><strong>14</strong>, 3445 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-39120-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10257182/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37301862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smer-Barreto,%20V.%20et%20al.%20Discovery%20of%20senolytics%20using%20machine%20learning.%20Nat.%20Commun.14,%203445%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Guo, J., Wu, Y., Jiang, M., Wu, C. &amp; Wang, G. An LC–MS-based metabolomic approach provides insights into the metabolite profiles of <em>Ginkgo biloba</em> L. at different developmental stages and in various organs. <em>Food Res. Int.</em><strong>159</strong>, 111644 (2022).
</cite> [<a href="https://doi.org/10.1016/j.foodres.2022.111644" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35940772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20J.,%20Wu,%20Y.,%20Jiang,%20M.,%20Wu,%20C.%20&amp;%20Wang,%20G.%20An%20LC%E2%80%93MS-based%20metabolomic%20approach%20provides%20insights%20into%20the%20metabolite%20profiles%20of%20Ginkgo%20biloba%20L.%20at%20different%20developmental%20stages%20and%20in%20various%20organs.%20Food%20Res.%20Int.159,%20111644%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Potter, S. C. et al. HMMER web server: 2018 update. <em>Nucleic Acids Res.</em><strong>46</strong>, W200–W204 (2018).
</cite> [<a href="https://doi.org/10.1093/nar/gky448" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6030962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29905871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Potter,%20S.%20C.%20et%20al.%20HMMER%20web%20server:%202018%20update.%20Nucleic%20Acids%20Res.46,%20W200%E2%80%93W204%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Sun, Y. et al. De novo production of versatile oxidized kaurene diterpenes in <em>Escherichia coli</em>. <em>Metab. Eng.</em><strong>73</strong>, 201–213 (2022).
</cite> [<a href="https://doi.org/10.1016/j.ymben.2022.08.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35934176/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20Y.%20et%20al.%20De%20novo%20production%20of%20versatile%20oxidized%20kaurene%20diterpenes%20in%20Escherichia%20coli.%20Metab.%20Eng.73,%20201%E2%80%93213%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Barnes, H. J., Arlotto, M. P. &amp; Waterman, M. R. Expression and enzymatic activity of recombinant cytochrome P450 17 alpha-hydroxylase in <em>Escherichia coli</em>. <em>Proc. Natl. Acad. Sci. USA</em><strong>88</strong>, 5597–5601 (1991).
</cite> [<a href="https://doi.org/10.1073/pnas.88.13.5597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC51924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1829523/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barnes,%20H.%20J.,%20Arlotto,%20M.%20P.%20&amp;%20Waterman,%20M.%20R.%20Expression%20and%20enzymatic%20activity%20of%20recombinant%20cytochrome%20P450%2017%20alpha-hydroxylase%20in%20Escherichia%20coli.%20Proc.%20Natl.%20Acad.%20Sci.%20USA88,%205597%E2%80%935601%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Emms, D. M. &amp; Kelly, S. OrthoFinder: phylogenetic orthology inference for comparative genomics. <em>Genome Biol.</em><strong>20</strong>, 238 (2019).
</cite> [<a href="https://doi.org/10.1186/s13059-019-1832-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6857279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31727128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Emms,%20D.%20M.%20&amp;%20Kelly,%20S.%20OrthoFinder:%20phylogenetic%20orthology%20inference%20for%20comparative%20genomics.%20Genome%20Biol.20,%20238%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Trifinopoulos, J., Nguyen, L.-T., von Haeseler, A. &amp; Minh, B. Q. W-IQ-TREE: a fast online phylogenetic tool for maximum likelihood analysis. <em>Nucleic Acids Res.</em><strong>44</strong>, W232–W235 (2016).
</cite> [<a href="https://doi.org/10.1093/nar/gkw256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4987875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27084950/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trifinopoulos,%20J.,%20Nguyen,%20L.-T.,%20von%20Haeseler,%20A.%20&amp;%20Minh,%20B.%20Q.%20W-IQ-TREE:%20a%20fast%20online%20phylogenetic%20tool%20for%20maximum%20likelihood%20analysis.%20Nucleic%20Acids%20Res.44,%20W232%E2%80%93W235%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Pompon, D., Louerat, B., Bronine, A. &amp; Urban, P. Yeast expression of animal and plant P450s in optimized redox environments. In <em>Methods in Enzymology</em> (eds Abelson, J. N., Simon, M. I., Johnson, E. F. &amp; Waterman, M. R.) <strong>272</strong>, 51–64 (Academic Press, 1996).</cite> [<a href="https://doi.org/10.1016/s0076-6879(96)72008-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8791762/" class="usa-link">PubMed</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Wang, H. et al. PCPD: plant cytochrome P450 database and web-based tools for structural construction and ligand docking. <em>Synth. Syst. Biotechnol.</em><strong>6</strong>, 102–109 (2021).
</cite> [<a href="https://doi.org/10.1016/j.synbio.2021.04.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8094579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33997360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20H.%20et%20al.%20PCPD:%20plant%20cytochrome%20P450%20database%20and%20web-based%20tools%20for%20structural%20construction%20and%20ligand%20docking.%20Synth.%20Syst.%20Biotechnol.6,%20102%E2%80%93109%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Forli, S. et al. Computational protein–ligand docking and virtual drug screening with the AutoDock suite. <em>Nat. Protoc.</em><strong>11</strong>, 905–919 (2016).
</cite> [<a href="https://doi.org/10.1038/nprot.2016.051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4868550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27077332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Forli,%20S.%20et%20al.%20Computational%20protein%E2%80%93ligand%20docking%20and%20virtual%20drug%20screening%20with%20the%20AutoDock%20suite.%20Nat.%20Protoc.11,%20905%E2%80%93919%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Eberhardt, J., Santos-Martins, D., Tillack, A. F. &amp; Forli, S. AutoDock Vina 1.2.0: new docking methods, expanded force field, and Python bindings. <em>J. Chem. Inf. Model.</em><strong>61</strong>, 3891–3898 (2021).
</cite> [<a href="https://doi.org/10.1021/acs.jcim.1c00203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10683950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34278794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eberhardt,%20J.,%20Santos-Martins,%20D.,%20Tillack,%20A.%20F.%20&amp;%20Forli,%20S.%20AutoDock%20Vina%201.2.0:%20new%20docking%20methods,%20expanded%20force%20field,%20and%20Python%20bindings.%20J.%20Chem.%20Inf.%20Model.61,%203891%E2%80%933898%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Meng, E. C. et al. UCSF ChimeraX: tools for structure building and analysis. <em>Protein Sci.</em><strong>32</strong>, e4792 (2023).
</cite> [<a href="https://doi.org/10.1002/pro.4792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10588335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37774136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meng,%20E.%20C.%20et%20al.%20UCSF%20ChimeraX:%20tools%20for%20structure%20building%20and%20analysis.%20Protein%20Sci.32,%20e4792%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>da Silva, T. U. et al. Development of parameters compatible with the CHARMM36 force field for [Fe<sub>4</sub>S<sub>4</sub>]<sup>2+</sup> clusters and molecular dynamics simulations of adenosine-5’-phosphosulfate reductase in GROMACS 2019. <em>J. Biomol. Struct. Dyn.</em><strong>40</strong>, 3481–3491 (2022).
</cite> [<a href="https://doi.org/10.1080/07391102.2020.1847687" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33183173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?da%20Silva,%20T.%20U.%20et%20al.%20Development%20of%20parameters%20compatible%20with%20the%20CHARMM36%20force%20field%20for%20%5BFe4S4%5D2+%20clusters%20and%20molecular%20dynamics%20simulations%20of%20adenosine-5%E2%80%99-phosphosulfate%20reductase%20in%20GROMACS%202019.%20J.%20Biomol.%20Struct.%20Dyn.40,%203481%E2%80%933491%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Man, V. H., Nguyen, P. H. &amp; Derreumaux, P. High-resolution structures of the amyloid-β 1–42 dimers from the comparison of four atomistic force fields. <em>J. Phys. Chem. B</em><strong>121</strong>, 5977–5987 (2017).
</cite> [<a href="https://doi.org/10.1021/acs.jpcb.7b04689" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5944325/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28538095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Man,%20V.%20H.,%20Nguyen,%20P.%20H.%20&amp;%20Derreumaux,%20P.%20High-resolution%20structures%20of%20the%20amyloid-%CE%B2%201%E2%80%9342%20dimers%20from%20the%20comparison%20of%20four%20atomistic%20force%20fields.%20J.%20Phys.%20Chem.%20B121,%205977%E2%80%935987%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Ren, J., Vaid, T. M., Lee, H., Ojeda, I. &amp; Johnson, M. E. Evaluation of interactions between the hepatitis C virus NS3/4A and sulfonamidobenzamide based molecules using molecular docking, molecular dynamics simulations and binding free energy calculations. <em>J. Comput. Aided Mol. Des.</em><strong>37</strong>, 53–65 (2023).
</cite> [<a href="https://doi.org/10.1007/s10822-022-00490-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9839505/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36427108/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ren,%20J.,%20Vaid,%20T.%20M.,%20Lee,%20H.,%20Ojeda,%20I.%20&amp;%20Johnson,%20M.%20E.%20Evaluation%20of%20interactions%20between%20the%20hepatitis%20C%20virus%20NS3/4A%20and%20sulfonamidobenzamide%20based%20molecules%20using%20molecular%20docking,%20molecular%20dynamics%20simulations%20and%20binding%20free%20energy%20calculations.%20J.%20Comput.%20Aided%20Mol.%20Des.37,%2053%E2%80%9365%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Suleiman, M. R. et al. Discovery of small molecule inhibitors through pharmacophore modeling, molecular docking, molecular dynamics simulation and experimental validation against myeloid cell leukemia-1 (Mcl-1). <em>J. Biomol. Struct. Dyn.</em><strong>39</strong>, 2512–2525 (2021).
</cite> [<a href="https://doi.org/10.1080/07391102.2020.1749132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32228162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Suleiman,%20M.%20R.%20et%20al.%20Discovery%20of%20small%20molecule%20inhibitors%20through%20pharmacophore%20modeling,%20molecular%20docking,%20molecular%20dynamics%20simulation%20and%20experimental%20validation%20against%20myeloid%20cell%20leukemia-1%20(Mcl-1).%20J.%20Biomol.%20Struct.%20Dyn.39,%202512%E2%80%932525%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Ulutürk, M., Karabacak Atay, Ç., Dede, B. &amp; Tilki, T. Potentially bioactive novel isophthalic acid based azo molecules: synthesis, characterization, quantum chemical calculations, ADMET properties, molecular docking and molecular dynamics simulations. <em>Polycycl. Aromat. Compd</em>. 1–22 10.1080/10406638.2023.2284797 (2023).</cite>
</li>
<li id="CR78">
<span class="label">78.</span><cite>Metz, S., Kästner, J., Sokol, A. A., Keal, T. W. &amp; Sherwood, P. ChemShell—a modular software package for QM/MM simulations. <em>WIREs Comput. Mol. Sci.</em><strong>4</strong>, 101–110 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?Metz,%20S.,%20K%C3%A4stner,%20J.,%20Sokol,%20A.%20A.,%20Keal,%20T.%20W.%20&amp;%20Sherwood,%20P.%20ChemShell%E2%80%94a%20modular%20software%20package%20for%20QM/MM%20simulations.%20WIREs%20Comput.%20Mol.%20Sci.4,%20101%E2%80%93110%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Zhang, J. &amp; Liu, Y. A QM/MM study of the catalytic mechanism of aspartate ammonia lyase. <em>J. Mol. Graph. Model.</em><strong>51</strong>, 113–119 (2014).
</cite> [<a href="https://doi.org/10.1016/j.jmgm.2014.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24875395/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20J.%20&amp;%20Liu,%20Y.%20A%20QM/MM%20study%20of%20the%20catalytic%20mechanism%20of%20aspartate%20ammonia%20lyase.%20J.%20Mol.%20Graph.%20Model.51,%20113%E2%80%93119%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Gritsenko, O. V., Cordero, N. A., Rubio, A. &amp; Alonso, J. A. Gradient correction to the exchange pair-correlation function of the weighted spin-density approximation in the density functional formalism. <em>Chem. Phys. Lett.</em><strong>296</strong>, 307–312 (1998).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gritsenko,%20O.%20V.,%20Cordero,%20N.%20A.,%20Rubio,%20A.%20&amp;%20Alonso,%20J.%20A.%20Gradient%20correction%20to%20the%20exchange%20pair-correlation%20function%20of%20the%20weighted%20spin-density%20approximation%20in%20the%20density%20functional%20formalism.%20Chem.%20Phys.%20Lett.296,%20307%E2%80%93312%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Bayach, I., Sancho-García, J. C., Di Meo, F., Weber, J.-F. F. &amp; Trouillas, P. π-Stacked polyphenolic dimers: a case study using dispersion-corrected methods. <em>Chem. Phys. Lett.</em><strong>578</strong>, 120–125 (2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bayach,%20I.,%20Sancho-Garc%C3%ADa,%20J.%20C.,%20Di%20Meo,%20F.,%20Weber,%20J.-F.%20F.%20&amp;%20Trouillas,%20P.%20%CF%80-Stacked%20polyphenolic%20dimers:%20a%20case%20study%20using%20dispersion-corrected%20methods.%20Chem.%20Phys.%20Lett.578,%20120%E2%80%93125%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Weigend, F. &amp; Ahlrichs, R. Balanced basis sets of split valence, triple zeta valence and quadruple zeta valence quality for H to Rn: design and assessment of accuracy. <em>Phys. Chem. Chem. Phys.</em><strong>7</strong>, 3297–3305 (2005).
</cite> [<a href="https://doi.org/10.1039/b508541a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16240044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weigend,%20F.%20&amp;%20Ahlrichs,%20R.%20Balanced%20basis%20sets%20of%20split%20valence,%20triple%20zeta%20valence%20and%20quadruple%20zeta%20valence%20quality%20for%20H%20to%20Rn:%20design%20and%20assessment%20of%20accuracy.%20Phys.%20Chem.%20Chem.%20Phys.7,%203297%E2%80%933305%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Zheng, D., Yuan, Y. &amp; Wang, F. Determining the hydration free energies of selected small molecules with MP2 and local MP2 through adaptive force matching. <em>J. Chem. Phys.</em><strong>154</strong>, 104113 (2021).
</cite> [<a href="https://doi.org/10.1063/5.0044712" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8231704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33722038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20D.,%20Yuan,%20Y.%20&amp;%20Wang,%20F.%20Determining%20the%20hydration%20free%20energies%20of%20selected%20small%20molecules%20with%20MP2%20and%20local%20MP2%20through%20adaptive%20force%20matching.%20J.%20Chem.%20Phys.154,%20104113%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Li, Q. et al. A modified pCas/pTargetF system for CRISPR-Cas9-assisted genome editing in <em>Escherichia coli</em>. <em>Acta Biochim. Biophys. Sin.</em><strong>53</strong>, 620–627 (2021).
</cite> [<a href="https://doi.org/10.1093/abbs/gmab036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33764372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Q.%20et%20al.%20A%20modified%20pCas/pTargetF%20system%20for%20CRISPR-Cas9-assisted%20genome%20editing%20in%20Escherichia%20coli.%20Acta%20Biochim.%20Biophys.%20Sin.53,%20620%E2%80%93627%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Zhang, Q., Wu, Y.-H., Huang, X.-S., Liu, H.-L. &amp; Wang, Y. Design and optimization for efficient production of (<em>S</em>)-canadine in <em>Escherichia coli</em>. <em>ACS Sustain. Chem. Eng.</em><strong>12</strong>, 6941–6951 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Q.,%20Wu,%20Y.-H.,%20Huang,%20X.-S.,%20Liu,%20H.-L.%20&amp;%20Wang,%20Y.%20Design%20and%20optimization%20for%20efficient%20production%20of%20(S)-canadine%20in%20Escherichia%20coli.%20ACS%20Sustain.%20Chem.%20Eng.12,%206941%E2%80%936951%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Madeira, F. et al. Search and sequence analysis tools services from EMBL-EBI in 2022. <em>Nucleic Acids Res.</em><strong>50</strong>, W276–W279 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkac240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9252731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35412617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Madeira,%20F.%20et%20al.%20Search%20and%20sequence%20analysis%20tools%20services%20from%20EMBL-EBI%20in%202022.%20Nucleic%20Acids%20Res.50,%20W276%E2%80%93W279%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Letunic, I. &amp; Bork, P. Interactive tree of life (iTOL) v6: recent updates to the phylogenetic tree display and annotation tool. <em>Nucleic Acids Res.</em><strong>52</strong>, W78–W82 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkae268" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11223838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38613393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Letunic,%20I.%20&amp;%20Bork,%20P.%20Interactive%20tree%20of%20life%20(iTOL)%20v6:%20recent%20updates%20to%20the%20phylogenetic%20tree%20display%20and%20annotation%20tool.%20Nucleic%20Acids%20Res.52,%20W78%E2%80%93W82%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (6.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62990_MOESM2_ESM.pdf</a><sup> (162.8KB, pdf) </sup><p>Description of additional supplementary files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (4.5MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM4_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (4.5MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (196.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM6_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (4.5MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM7_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (4.5MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM8_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 6</a><sup> (4.3MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (2.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM10_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (38MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370966/bin/41467_2025_62990_MOESM11_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (9.2MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data supporting the findings of this study are available within the paper and its supplementary information files. The raw transcriptome data of <em>G. biloba</em> has been deposited in the Genome Sequence Archive and is accessible under accession number <a href="https://ngdc.cncb.ac.cn/gsa/search?searchTerm=CRA019958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CRA019958</a>. The sequences of GbCYP90s reported in this study have been submitted to the NCBI GenBank under accession numbers GbCYP90J6, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV862377" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV862377</a>; GbCYP90D59, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV862378" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV862378</a>; GbCYP90A68, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV862379" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV862379</a>; GbCYP90B61, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV862380" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV862380</a>. Additional CYP90 sequences of this study are provided in Supplementary Data <a href="#MOESM3" class="usa-link">1</a>. The mass spectrometry data generated in this study are available in Figshare [10.6084/m9.figshare.29552828.v1]. <a href="#Sec28" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62990-6"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62990.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370966/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370966/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370966%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370966/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370966/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370966/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841362/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370966/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841362/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370966/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370966/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="6S3hDAzOCgdc1GoF9Dsfhg2Hl2z68XNfQpKBLzgi9Dnar3oUa7WEcvb71QJzYKdg">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
