
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            GraphVelo allows for accurate inference of multimodal velocities and molecular mechanisms for single cells - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152537D88AF1D4A30537D80038AD71DC.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373899/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="GraphVelo allows for accurate inference of multimodal velocities and molecular mechanisms for single cells">
<meta name="citation_author" content="Yuhao Chen">
<meta name="citation_author_institution" content="Department of Bioinformatics, College of Life Sciences, Zhejiang University, Hangzhou, 310058 China">
<meta name="citation_author_institution" content="Department of Computational and Systems Biology, School of Medicine, University of Pittsburgh, Pittsburgh, PA USA">
<meta name="citation_author" content="Yan Zhang">
<meta name="citation_author_institution" content="Department of Computational and Systems Biology, School of Medicine, University of Pittsburgh, Pittsburgh, PA USA">
<meta name="citation_author" content="Jiaqi Gan">
<meta name="citation_author_institution" content="Department of Physics and Astronomy, University of Pittsburgh, Pittsburgh, PA USA">
<meta name="citation_author" content="Ke Ni">
<meta name="citation_author_institution" content="Department of Computational and Systems Biology, School of Medicine, University of Pittsburgh, Pittsburgh, PA USA">
<meta name="citation_author" content="Ming Chen">
<meta name="citation_author_institution" content="Department of Bioinformatics, College of Life Sciences, Zhejiang University, Hangzhou, 310058 China">
<meta name="citation_author_institution" content="Zhejiang Key Laboratory of Multi-omics Precision Diagnosis and Treatment of Liver Diseases, Department of General Surgery, Sir Run-Run Shaw Hospital, Zhejiang University School of Medicine, 310016 Hangzhou, China">
<meta name="citation_author" content="Ivet Bahar">
<meta name="citation_author_institution" content="Laufer Center for Physical and Quantitative Biology, and Department of Biochemistry and Cell Biology, Renaissance School of Medicine, Stony Brook University, Stony Brook, NY USA">
<meta name="citation_author" content="Jianhua Xing">
<meta name="citation_author_institution" content="Department of Computational and Systems Biology, School of Medicine, University of Pittsburgh, Pittsburgh, PA USA">
<meta name="citation_author_institution" content="Department of Physics and Astronomy, University of Pittsburgh, Pittsburgh, PA USA">
<meta name="citation_author_institution" content="University of Pittsburgh Hillman Cancer Center, Pittsburgh, PA USA">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7831">
<meta name="citation_doi" content="10.1038/s41467-025-62784-w">
<meta name="citation_pmid" content="40847023">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373899/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373899/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373899/pdf/41467_2025_Article_62784.pdf">
<meta name="description" content="RNA velocities and generalizations emerge as powerful approaches for extracting time-resolved information from high-throughput snapshot single-cell data. Yet, several inherent limitations restrict applying the approaches to genes not suitable for ...">
<meta name="og:title" content="GraphVelo allows for accurate inference of multimodal velocities and molecular mechanisms for single cells">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="RNA velocities and generalizations emerge as powerful approaches for extracting time-resolved information from high-throughput snapshot single-cell data. Yet, several inherent limitations restrict applying the approaches to genes not suitable for ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373899/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373899">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62784-w"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62784.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373899%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373899/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373899/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373899/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 22;16:7831. doi: <a href="https://doi.org/10.1038/s41467-025-62784-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62784-w</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>GraphVelo allows for accurate inference of multimodal velocities and molecular mechanisms for single cells</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yuhao Chen</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yuhao Chen</span></h3>
<div class="p">
<sup>1</sup>Department of Bioinformatics, College of Life Sciences, Zhejiang University, Hangzhou, 310058 China </div>
<div class="p">
<sup>2</sup>Department of Computational and Systems Biology, School of Medicine, University of Pittsburgh, Pittsburgh, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuhao Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yan Zhang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yan Zhang</span></h3>
<div class="p">
<sup>2</sup>Department of Computational and Systems Biology, School of Medicine, University of Pittsburgh, Pittsburgh, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yan Zhang</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gan%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jiaqi Gan</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jiaqi Gan</span></h3>
<div class="p">
<sup>3</sup>Department of Physics and Astronomy, University of Pittsburgh, Pittsburgh, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gan%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiaqi Gan</span></a>
</div>
</div>
<sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ni%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ke Ni</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ke Ni</span></h3>
<div class="p">
<sup>2</sup>Department of Computational and Systems Biology, School of Medicine, University of Pittsburgh, Pittsburgh, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ni%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Ke Ni</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ming Chen</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ming Chen</span></h3>
<div class="p">
<sup>1</sup>Department of Bioinformatics, College of Life Sciences, Zhejiang University, Hangzhou, 310058 China </div>
<div class="p">
<sup>4</sup>Zhejiang Key Laboratory of Multi-omics Precision Diagnosis and Treatment of Liver Diseases, Department of General Surgery, Sir Run-Run Shaw Hospital, Zhejiang University School of Medicine, 310016 Hangzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Ming Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bahar%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Ivet Bahar</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Ivet Bahar</span></h3>
<div class="p">
<sup>5</sup>Laufer Center for Physical and Quantitative Biology, and Department of Biochemistry and Cell Biology, Renaissance School of Medicine, Stony Brook University, Stony Brook, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bahar%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ivet Bahar</span></a>
</div>
</div>
<sup>5,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xing%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jianhua Xing</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jianhua Xing</span></h3>
<div class="p">
<sup>2</sup>Department of Computational and Systems Biology, School of Medicine, University of Pittsburgh, Pittsburgh, PA USA </div>
<div class="p">
<sup>3</sup>Department of Physics and Astronomy, University of Pittsburgh, Pittsburgh, PA USA </div>
<div class="p">
<sup>6</sup>University of Pittsburgh Hillman Cancer Center, Pittsburgh, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xing%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jianhua Xing</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>6,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Bioinformatics, College of Life Sciences, Zhejiang University, Hangzhou, 310058 China </div>
<div id="Aff2">
<sup>2</sup>Department of Computational and Systems Biology, School of Medicine, University of Pittsburgh, Pittsburgh, PA USA </div>
<div id="Aff3">
<sup>3</sup>Department of Physics and Astronomy, University of Pittsburgh, Pittsburgh, PA USA </div>
<div id="Aff4">
<sup>4</sup>Zhejiang Key Laboratory of Multi-omics Precision Diagnosis and Treatment of Liver Diseases, Department of General Surgery, Sir Run-Run Shaw Hospital, Zhejiang University School of Medicine, 310016 Hangzhou, China </div>
<div id="Aff5">
<sup>5</sup>Laufer Center for Physical and Quantitative Biology, and Department of Biochemistry and Cell Biology, Renaissance School of Medicine, Stony Brook University, Stony Brook, NY USA </div>
<div id="Aff6">
<sup>6</sup>University of Pittsburgh Hillman Cancer Center, Pittsburgh, PA USA </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 27; Accepted 2025 Jul 30; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373899  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40847023/" class="usa-link">40847023</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article">
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint posted on bioRxiv on June 14, 2025: "<a href="/articles/PMC11642879/" class="usa-link">
GraphVelo allows for accurate inference of multimodal velocities and molecular mechanisms for single cells
</a>".</div>
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint posted on Research Square on January 15, 2025: "<a href="/articles/PMC11774466/" class="usa-link">GraphVelo allows for accurate inference of multimodal omics velocities and molecular mechanisms for single cells</a>".</div>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">RNA velocities and generalizations emerge as powerful approaches for extracting time-resolved information from high-throughput snapshot single-cell data. Yet, several inherent limitations restrict applying the approaches to genes not suitable for RNA velocity inference due to complex transcriptional dynamics, low expression, or lacking splicing dynamics, or data of non-transcriptomic modality. Here, we present GraphVelo, a graph-based machine learning procedure that uses as input the RNA velocities inferred from existing methods and infers velocity vectors lying in the tangent space of the low-dimensional manifold formed by the single cell data. GraphVelo preserves vector magnitude and direction information during transformations across different data representations. Tests on synthetic and experimental single-cell data, including viral-host interactome, multi-omics, and spatial genomics datasets demonstrate that GraphVelo, together with downstream generalized dynamo analyses, extends RNA velocities to multi-modal data and reveals quantitative nonlinear regulation relations between genes, virus, and host cells, and different layers of gene regulation.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Data processing, Computational biophysics, Machine learning</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">RNA velocity offers insight into cell dynamics but faces key limitations across modalities. Here, authors present GraphVelo, a machine learning framework that refines and extends RNA velocity to multimodal data, enabling quantitative, interpretable cell state transitions.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Cells need to constantly detect and adapt to changes in extracellular and intracellular environments. Regulation of their gene transcription is a common mechanism of response. Multiple factors affect the transcriptional activity of eukaryotic genes, including <em>cis</em> and <em>trans</em> regulatory elements and chromatin structure. High throughput single-cell sequencing data provide the landscape of cell genotype. These data lack, however, information on how the cell state changes over time. Continuous efforts have been made to extract information about gene regulation and develop methods for connecting the cell states to temporal sequences of events captured by single-cell snapshot data. One group of methods that has received extensive attention is based on RNA velocity<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup> for predicting the changes in RNA expression states in the cell. The original RNA velocity method leverages the ratio between nascent and mature transcripts to estimate the rate of change in gene expression. This seminal study has inspired numerous methods for improved RNA velocity estimation based on information from splicing<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>, metabolic labeling<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>, lineage tracing<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, and transcriptional factor binding<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>, etc.</p>
<p id="Par4">The RNA velocity framework has, however, its inherent limitations. First, none of the RNA velocity estimation methods could be applied to any single cell transcriptomic data without restrictions. For example, the splicing-based method is not applicable to prokaryotes or viruses, or organisms without introns. Erroneous inferences of RNA velocities have also been noticed for genes having complex splicing dynamics<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Furthermore, it is difficult to estimate the RNA velocities of genes with low expression, which excludes most transcription factors. Second, multi-omics sequencing technologies provide multifaceted information on cellular states alongside the transcriptome modality, and there exist limited systematic methods to extend such velocity estimations to other modalities<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>.</p>
<p id="Par5">The single-cell transcriptome state is usually defined by the instantaneous distribution of RNA levels, represented by a multidimensional vector of RNA levels for all (measurable) genes. The usual practice is to map such single-cell data onto a reduced space, e.g., transform from a principal component (PC) space to a UMAP representation to facilitate the visualization of the time-evolution of cell state, with each cell state being represented by a point in that space. Numerous dimension reduction and manifold learning algorithms have been developed for representation transformation. In comparison, transforming a velocity vector between representations is a nontrivial task not rigorously addressed in the single-cell field. Even worse, a visually correct vector field does not necessarily imply accurate high-dimensional velocity estimation<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. La Manno et al. proposed a cosine kernel method to address this challenge, which has been adopted since then in most subsequent studies<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Li et al. mathematically proved that the cosine kernel asymptotically gives the correct direction of a velocity vector in the large sampling limit, but the magnitude information is completely lost due to a normalization procedure<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. This loss of information casts concerns when such quantitative information is needed.</p>
<p id="Par6">In this study, we tackle the above challenges through a graph-theoretical representation of RNA velocities, called GraphVelo, with dynamical systems underpinnings. GraphVelo takes an ansatz that the measured single-cell expression profiles and inferred RNA velocities collectively reflect a dynamical process and are connected through a set of dynamical equations. It exploits such additional constraints that couple a high dimensional velocity field and a corresponding single-cell state manifold, and enables the generalization of the approach in the context of multi-modal single-cell data. While the combined expression and velocity information has been widely used to infer cell state transition trajectories<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, GraphVelo presents the advantage of enabling downstream analyses such as that performed by dynamo<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup> to extract quantitative information on causal gene-gene relations that dictate the cell state transitions. Benchmarking of the proposed graph framework against simulated and experimental single-cell data lends support to its broad utility.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">GraphVelo infers manifold-consistent single-cell velocity vectors through tangent space projection and transforms between representations through local linear embedding</h3>
<p id="Par7">Consider that the internal state of a cell can be specified by an <em>N</em>-dimensional state vector <strong>x</strong>, with <em>N</em> ≫ 1 generally. Assume that the temporal evolution of the cell state follows a continuous and smooth curve <strong>x</strong>(<em>t</em>) (see Methods for more general mathematical formulation). The instant velocity vector <strong>v</strong>(<strong>x,</strong>
<em>t</em>) = d<strong>x</strong>(<em>t</em>)/d<em>t</em> is always tangent to the curve of <strong>x</strong>(<em>t</em>) (as a function of <em>t</em>) at <strong>x</strong>. One can generalize to the situation that the trajectories of a swarm of cells form a <em>M</em>-dimensional manifold <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e401"><mi class="MJX-tex-caligraphic" mathvariant="script">M</mi><mrow><mo>(</mo><mrow><mi mathvariant="bold">x</mi></mrow><mo>)</mo></mrow></math></span> embedded in the <em>N</em>-dimensional state space with <em>M</em> ≪ <em>N</em> typically, as revealed by high-throughput single cell omics data. Then under the ansatz that a velocity vector <strong>v</strong>(<strong>x,</strong>
<em>t</em>) dictates the evolution of a state vector <strong>x</strong>(<em>t</em>), <strong>v</strong> must lie in the tangent space of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e446"><mi class="MJX-tex-caligraphic" mathvariant="script">M</mi><mrow><mo>(</mo><mrow><mi mathvariant="bold">x</mi></mrow><mo>)</mo></mrow></math></span>, denoted as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e462"><msub><mrow><mi>T</mi></mrow><mrow><mi>p</mi></mrow></msub><mi class="MJX-tex-caligraphic" mathvariant="script">M</mi></math></span>. In practice, the RNA velocity vectors inferred from existing methods do not automatically satisfy this tangent space requirement (but see<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>).</p>
<p id="Par8">Taking various inferred single-cell RNA velocity vectors, e.g., splicing-based, metabolic labeling-based, or lineage tracing-based, as input, GraphVelo takes advantage of the nature of the low-dimensional cell state manifold to: (1) refine the estimated RNA velocity to satisfy the tangent space requirement; (2) infer the velocities of non-transcriptomic modalities using RNA velocities. GraphVelo thus serves as a plugin that can be seamlessly integrated into existing RNA velocity analysis pipelines, and help process single-cell data for downstream cellular dynamics analyses using methods such as dynamo (Fig. <a href="#Fig1" class="usa-link">1a</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Refining RNA velocity by tangent space projection and transforming between representations using GraphVelo.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373899_41467_2025_62784_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7832/12373899/d3b37613e069/41467_2025_62784_Fig1_HTML.jpg" loading="lazy" id="d33e543" height="998" width="757" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Workflow of RNA velocity-based analyses incoporating GraphVelo. Note GraphVelo takes any form of RNA velocity (i.e., not just splicing-based velocity) as input, and the kNN neighborhood is defined in the full state space (e.g., by both scRNAseq and scATACseq in multi-omics data). <strong>b</strong> Schematic of tangent space projection and velocity transformation between homeomophic manifolds. Left: RNA velocity vectors are projected onto the tangent space defined by the discretized local manifold of neighborhood cell samples. Right: GraphVelo allows for transformation of velocity vectors from a manifold embeded in a higher dimensional space <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e498"><mrow><mo>(</mo><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">M</mi></mrow><mo>)</mo></mrow></math></span> to that in a lower-dimensional space <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e512"><mrow><mo>(</mo><mrow><mi>ℵ</mi></mrow><mo>)</mo></mrow></math></span>, and vice versa. <strong>c</strong> The process of minimizing the loss function of tangent space projection. Noisy velocity vectors (left) generated by adding random components orthogonal to those sampled from an analytical 2D manifold were projected back onto the 2D manifold, resulting in smooth velocity vectors that lie in the tangent space (right). <strong>d</strong> GraphVelo allows whole genome velocity inference based on the robustly estimated MacK genes (see also Fig. <a href="#Fig3" class="usa-link">3</a>). Velocities of genes undergoing variable kinetic rates, such as rapid degradation or transcription burst, are difficult to be correctly inferred by other methods, but can be inferred robustly with GraphVelo. <strong>e</strong> Virus infection dynamics and underlying host-virus interaction mechanisms uncovered by GraphVelo (see also Fig. <a href="#Fig4" class="usa-link">4</a>). Upper: pathways involved in host-virus interactions were identified using GraphVelo. Lower: GraphVelo predicted reversed trajectory of viral infection in response to in silico perturbations of viral factors. <strong>f</strong> GraphVelo provides a consistent view of epigenetic and transcription dynamics (see also Fig. <a href="#Fig5" class="usa-link">5</a>). Upper: GraphVelo analyses on multi-omics data revealed that most cell-cycle dependent genes showed decoupling between transcription dynamics and chromatin accessibility change dynamics. Lower: Effective dose-response curves reconstructed from multi-omics data revealed pioneer transcription factors increased chromatin accessibility then transcription of targe genes.</p></figcaption></figure><p id="Par9">Practically, GraphVelo approximates the tangent space at a cell state <strong>x</strong> by a k-nearest neighbor (kNN) graph following the local linear embedding algorithm<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, and uses the more reliable data manifold <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e558"><mi class="MJX-tex-caligraphic" mathvariant="script">M</mi></math></span> to refine the velocity vectors by imposing the constraint that the <em>N</em>-dimensional velocity vector <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e571"><mi mathvariant="bold">v</mi></math></span> should lie in the tangent space (Fig. <a href="#Fig1" class="usa-link">1b, c</a>). Consider a given point <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e585"><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> on a manifold corresponding to the expression state <em>i</em> of the single cell. Its infinitesimal neighborhood forms an Euclidean space that approximates the tangent space <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e602"><msub><mrow><mi>T</mi></mrow><mrow><mi>p</mi></mrow></msub><mi class="MJX-tex-caligraphic" mathvariant="script">M</mi></math></span>. With a sufficient sampling of the neighboring cell states <em>j</em> in the state space, the incremental displacement vectors between cell state <em>i</em> and its neighboring cell states, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e624"><msub><mrow><mi mathvariant="bold-italic">δ</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span>, form a set of complete albeit possibly redundant and nonorthogonal/non-normalized basis vectors of the Euclidean space in the local region. Then the projection of the measured velocity vector onto <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e653"><msub><mrow><mi>T</mi></mrow><mrow><mi>p</mi></mrow></msub><mi class="MJX-tex-caligraphic" mathvariant="script">M</mi></math></span> can be expressed as a linear combination (see also <a href="#MOESM1" class="usa-link">Supplementary Notes</a> <a href="#MOESM1" class="usa-link">1.1</a>),</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e675" display="block"><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mo>∥</mo></mrow></msub><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>j</mi><mo>∈</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub><msub><mrow><mi>ϕ</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><msub><mrow><mi mathvariant="bold-italic">δ</mi></mrow><mrow><mi>i</mi><mi>j</mi><mo>,</mo></mrow></msub></math></td>
<td class="label">1</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e727"><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> is the neighborhood of cell state <em>i</em>, defined by its <em>k</em> nearest neighbors in the feature space determined by sequencing profiles. Direct application of Eq. <a href="#Equ1" class="usa-link">1</a> to determine the coefficients <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e751"><msub><mrow><mi>ϕ</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></span> is numerically unstable in real data (see <a href="#MOESM1" class="usa-link">Supplementary Notes</a> <a href="#MOESM1" class="usa-link">1.2</a> for detailed discussion). Instead, we performed the projection by optimizing the following tangent space projection (TSP) loss function (Fig. <a href="#Fig1" class="usa-link">1c</a>),</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e776" display="block"><mi class="MJX-tex-caligraphic" mathvariant="script">L</mi><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="normal">ϕ</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>=</mo><mi>a</mi><mo>⋅</mo><msup><mrow><mo>∥</mo><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mo>∥</mo><mi>i</mi></mrow></msub><mo>∥</mo></mrow><mrow><mn>2</mn></mrow></msup><mo>−</mo><mi>b</mi><mo>⋅</mo><mspace width="0.25em"></mspace><mi>cos</mi><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="normal">ϕ</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msubsup><mrow><mi mathvariant="normal">ϕ</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">corr</mi></mrow></msubsup></mrow><mo>)</mo></mrow><mo>+</mo><mi>λ</mi><msup><mrow><mo>∥</mo><msub><mrow><mi mathvariant="normal">ϕ</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>∥</mo></mrow><mrow><mn>2</mn></mrow></msup></math></td>
<td class="label">2</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e859"><mo>∥</mo><mo>⋅</mo><mo>∥</mo></math></span> refers to vector modulus. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e870"><msubsup><mrow><mi mathvariant="bold">ϕ</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">corr</mi></mrow></msubsup></math></span> is a heuristic “cosine kernel” widely used in the RNA velocity analyses for projecting velocity vectors onto a reduced space, the elements of which are <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e887"><msub><mrow><mi>ϕ</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mfenced close=")" open="("><mrow><mi>w</mi><mi>i</mi><mi>t</mi><mi>h</mi><mi>j</mi><mo>∈</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></mfenced></math></span> (see also <a href="#MOESM1" class="usa-link">Supplementary Notes</a> <a href="#MOESM1" class="usa-link">1.3</a>); the second term <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e922"><mi>cos</mi><mfenced close=")" open="("><mrow><mo>⋅</mo><mo>,</mo><mo>⋅</mo></mrow></mfenced></math></span> denotes the cosine similarity. The first term in the loss function learns the correct velocity magnitudes, and the second term retains the reliable direction information based on previous mathematical analyses showing that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e936"><msubsup><mrow><mi mathvariant="bold">ϕ</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">corr</mi></mrow></msubsup></math></span> asymptotically gives correct direction of the velocity vector<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. The L2 regularization is used to bound parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e957"><msub><mrow><mi mathvariant="bold">ϕ</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span>. Hyperparameters <em>a</em>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e975"><mi>b</mi></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e984"><mi>λ</mi></math></span> are for retaining the projection strength, direction, and for regularization, respectively.</p>
<p id="Par10">With local linear embedding, it is straightforward to transform velocity between different representations. Assuming a mapping function <em>f</em> exists connecting manifold <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e998"><mi class="MJX-tex-caligraphic" mathvariant="script">M</mi></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e1008"><mi>ℵ</mi></math></span> such that for cell <em>i</em> with state vector <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e1020"><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e1035"><mi class="MJX-tex-caligraphic" mathvariant="script">M</mi></math></span>, the coordinate of the same cell in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e1045"><mi>ℵ</mi></math></span> is given as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e1054"><msub><mrow><mi mathvariant="bold">y</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><mi>f</mi><mfenced close=")" open="("><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></mfenced></math></span>. Since a given local patch of a continuous manifold is approximated by an Euclidean space, a locally linear transformation connects the patch in the two representations. Consequently, for a vector described by Eq. <a href="#Equ1" class="usa-link">1</a> in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e1081"><mi class="MJX-tex-caligraphic" mathvariant="script">M</mi></math></span>, the velocity vector in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e1091"><mi>ℵ</mi></math></span> is,</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e1102" display="block"><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mo>∥</mo></mrow></msub><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold">y</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>j</mi><mo>∈</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub><msub><mrow><mi>ϕ</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><msubsup><mrow><mi mathvariant="bold-italic">δ</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mo>′</mo></mrow></msubsup></math></td>
<td class="label">3</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e1155"><msubsup><mrow><mi mathvariant="bold-italic">δ</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mo>′</mo></mrow></msubsup><mo>=</mo><msub><mrow><mi mathvariant="bold">y</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">y</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span>. That is, one only needs to change the basis vectors.</p>
<p id="Par11">Therefore, Eqs. <a href="#Equ1" class="usa-link">1</a>–<a href="#Equ3" class="usa-link">3</a> form the mathematical and computational foundation of GraphVelo. With Eq. <a href="#Equ3" class="usa-link">3</a> one can extend velocity inference to datasets that velocity inference is not traditionally applicable such as host-virus interactome and multi-omics datasets based on the Whitney embedding theorem<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Details of the mathematical foundation were given in Methods. With velocity vectors refined with GraphVelo, one can readily perform downstream analyses, as exemplified in Fig.<a href="#Fig1" class="usa-link">1d–f</a>, which will be further elaborated below in the context of specific applications.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Benchmark studies demonstrate the effectiveness of GraphVelo across simulation datasets with diverse topology</h3>
<p id="Par12">To demonstrate the effectiveness of the geometry-constrained projection, we first benchmarked our method on a 3D bifurcation system constrained on a 2D manifold (Methods). We added random components vertical to the tangent plane to mimic the noise. The resulting velocity vectors inferred by GraphVelo through minimizing the TSP loss were consistent with the ground truth vectors (Fig. <a href="#Fig2" class="usa-link">2a</a>). Both GraphVelo and cosine kernel successfully removed the normal components (Fig. <a href="#Fig2" class="usa-link">2b i</a>) and maintained the directional information (Fig. <a href="#Fig2" class="usa-link">2b ii</a>), but only GraphVelo kept the velocity magnitude information (Fig. <a href="#Fig2" class="usa-link">2b iii</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Testing GraphVelo on simulated datasets.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373899_41467_2025_62784_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7832/12373899/bc03e767e3f8/41467_2025_62784_Fig2_HTML.jpg" loading="lazy" id="d33e1259" height="968" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Velocity vectors of an analytical three variables bifurcating vector field constrained to a spherical surface. The data points were colored by simulation time. <strong>b</strong> Violinplots of: (i) normal component of velocity vectors, (ii) cosine similarity and (iii) root mean square error (RMSE) between ground truth and velocity vectors projected by GraphVelo and cosine kernel, respectively. The number of simulated cells is 2000 for statistical test. <strong>c</strong>–<strong>e</strong> Simulation of scRNA-seq data using dyngen under linear, cycling, and bifurcating differentiation models (left), and velocity fields projected on multidimensional scaling (MDS) coodinates (right) using GraphVelo-corrected velocities, respectively. Each simulation consists of 1000 cell states and 100 genes. The cells in different states were colored by their simulation time along trajectory. <strong>f</strong>–<strong>h</strong> Comparisons of cosine similarity, and accuracy between the ground truth velocity vectors and dyngen simulated velocities after projection using GraphVelo TSP loss without cosine regularization, GraphVelo TSP loss with cosine regularization, cosine kernel, and random predictor, respectively. The number of dyngen-simulated cells is 1000 for statistical test. In <strong>b</strong> and <strong>f</strong>–<strong>h</strong>, *** indicates Welch’s independent two-sided t-test at <em>p</em> &lt; 0.05. Violinplot in panel b shows the distribution of data points after grouping by projection methods. Boxplots in <strong>f</strong>–<strong>h</strong> indicate median (middle line), first and third quartiles (box), and the upper whisker extends from the edges to the largest value no further than 1.5 × IQR (interquartile range) from the quartiles, and the lower whisker extends from the edge to the smallest value at most 1.5 × IQR of the edge. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par13">Next, we performed multifaceted evaluations of the ability of GraphVelo to robustly recover the transcriptional dynamics across a range of simulated datasets with different underlining phenotypic structures. We used dyngen<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, a multi-modal scRNA-seq simulation engine, to generate gene-wise dynamics defined by gold-standard transcriptional regulatory networks (Methods). We generated simulated scRNA-seq data for networks with a variety of underlying linear, cyclic, and bifurcating topological structures, and recovered the corresponding vector field using GraphVelo-corrected velocity vectors (Fig. <a href="#Fig2" class="usa-link">2c–e</a>). To comprehensively assess the outcome, we used three diverse metrics, cosine similarity, root-mean-square error (RMSE), and accuracy, which evaluate the correctness of velocity direction, magnitude, and sign, respectively. We presented (Fig. <a href="#Fig2" class="usa-link">2c–e</a>) the comparative results obtained with the cosine kernel and with GraphVelo (TSP with (i.e., Eq. <a href="#Equ2" class="usa-link">2</a> with b ≠ 0) and without (Eq. <a href="#Equ2" class="usa-link">2</a> with b = 0) the cosine regularization term). By minimizing TSP loss, GraphVelo preserved both the direction and magnitude of the vector field (Fig. <a href="#Fig2" class="usa-link">2f–h</a>). With an increase of noise level by adding Gaussian noise to the ground truth vectors, GraphVelo refined the distorted velocity and outperformed the cosine kernel projection consistently (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a-c</a>).</p>
<p id="Par14">Then, we tested whether manifold constraints could preserve the speed of the cell progression across different representations. GraphVelo was able to scale velocity vectors between the original space and the PCA space, showing a high correlation with the ground truth, even as noise levels increased, whereas the cosine kernel failed (Supplementary Fig. <a href="#MOESM1" class="usa-link">1d</a>). The results on UMAP showed less agreement, which is not surprising. UMAP is a convenient representation for visualizing single cell data but is not designed for representing quantitative cell state transition dynamics. That is, UMAP is not a continuous transformation from the original gene space and cannot preserve local distances after projection.</p>
<p id="Par15">To further explore whether GraphVelo could correct the RNA velocity estimated by the splicing kinetics, we took the velocity inferred using different packages (scVelo<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>, dynamo<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup> and VeloVI<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>) as input. The output from GraphVelo agreed significantly better with the ground truth compared to the raw input (Supplementary Fig. <a href="#MOESM1" class="usa-link">1e</a>), highlighting the significant improvement achieved by GraphVelo in evaluating both the direction and magnitude of the velocity vector fields across all datasets.</p></section><section id="Sec5"><h3 class="pmc_sec_title">GraphVelo achieved consistent improvements across multiple real-world datasets</h3>
<p id="Par16">Although GraphVelo is designed as a velocity-correction and prediction model complementing existing velocity estimation tools, we benchmarked the performance of scVelo-based GraphVelo outputs on five independent datasets (FUCCI<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, pancreatic endocrinogenesis<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, dentate gyrus<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>, intestinal organoid<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> and hematopoiesis<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>) against five RNA velocity estimation methods (scVelo<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>, VeloVI<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, UniTVelo<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, DeepVelo<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>, and CellDancer<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>) (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>–<a href="#MOESM1" class="usa-link">4</a>). GraphVelo achieved noticeably improved cross-boundary correctness (CBC) score<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> against input velocity and other advanced methods (see Methods for calculation details). We evaluated velocity consistency<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> across two datasets whose trajectories were estimated using different tools (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). While GraphVelo shows slightly lower overall vector smoothness scores compared to others, we observed that these models tend to produce overly smooth and homogenized velocity fields, which may obscure biologically meaningful heterogeneity. In contrast, GraphVelo preserves fine-grained local transitions and reveals subtle divergence in the vector field, particularly around fate bifurcations in endocrinogenesis data.</p>
<p id="Par17">We examined the cell cycle datasets annotated by the dynamo package, which features a relatively simple geometry, to quantitatively evaluate our method. First, we focused on the CBC score between cell cycle states. The velocity vectors processed by TSP showed greater consistency with the ground truth compared to the unprocessed inputs (Supplementary Fig. <a href="#MOESM1" class="usa-link">5a</a>). To demonstrate the effectiveness of GraphVelo in scaling velocities based on the data manifold, we used the L2 norm of velocity vectors to quantify the cell cycle speed. This analysis revealed a peak in velocity within the M and G1 phases, which was also reflected in the distribution of total UMI counts (Supplementary Fig. <a href="#MOESM1" class="usa-link">5b</a>). We further validated these findings using the cycling A549 cell line sequenced by sci-fate<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">5c</a>) and through the temporal variation of stratified cell cycle speed based on velocities inferred from metabolic-labeling data (Supplementary Fig. <a href="#MOESM1" class="usa-link">5d</a>). Leveraging the quantitative velocity vectors generated by GraphVelo, we classified genes by both the phase and peak magnitude of their velocities (Supplementary Fig. <a href="#MOESM1" class="usa-link">5e</a>). Analysis of phase-magnitude relationships uncovered the sequential activation cascade of marker genes throughout the cell cycle.</p></section><section id="Sec6"><h3 class="pmc_sec_title">GraphVelo infers quantitative genome-wide RNA velocity from a subset of genes with manifold-consistent RNA turnover kinetics</h3>
<p id="Par18">Most RNA velocity methods are based on biophysical models of mRNA turnover dynamics with specific assumptions that may break down in certain cases<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. These methods typically provide velocities of only a subset of (~500 or less) genes, termed velocity genes in the subsequent discussions, out of a larger list of (~2-4 k) highly variable genes in a dataset, and some of the velocities are questionable. For example, the splicing-based RNA velocity may have an erroneous sign for processes under active regulation on mRNA degradation or promotors switching between states with different transcription efficiency (Fig. <a href="#Fig3" class="usa-link">3a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Delineating transcriptome-wise progression with manifold-consistent kinetic genes using GraphVelo.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373899_41467_2025_62784_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7832/12373899/6d5efed63f6c/41467_2025_62784_Fig3_HTML.jpg" loading="lazy" id="d33e1465" height="996" width="733" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematitc of transcritional events mislead RNA velocity estimation in the phase portrait by standard approaches. Left: for genes exhibiting rapid degradation, the cells appear above the steady state line on the phase portrait, whereas the true velocity is negative. Right: For genes exhibiting transcription burst, the transcription rate abruptly increases at intermediate states, leading to a steady state line whose slope is overestimated. <strong>b</strong> Schematic of manifold-consistent score calculation for robustly estimated velocity genes. <strong>c</strong> The projected velocity field from GraphVelo are consistent with the erythroid differentiation by using all highly variable genes. <strong>d</strong> The correlation between GraphVelo vector field-based pseudotime and embryo time for erythroid lineage cells. Spearman correlation coefficients are shown. <strong>e</strong> GO enrichment analyses of top ranked MacK genes. <strong>f</strong> The phase portrait of two transcription burst genes (Smim1, Hba-x). <strong>g</strong> Scatter plots of: i) velocities estimated by scVelo, ii) refined velocities by GraphVelo, and iii) mature mRNA expression of transcription burst genes (Smim1, Hba-x). Cells were colored by corresponding velocity, and mature mRNA abundance, respectively, and visualized on the UMAP representation. <strong>h</strong> Gene regulatory cascade unraveled by GraphVelo-based vector field analyses that drives cell lineage commitment. Gene set enrichment was performed using one-sided Fisher’s exact test, Benjamini–Hochberg correction. Adjusted p-values represent FDR-corrected significance of gene set enrichment. <strong>i</strong> Activation of Gata1 inhibitor TF Spi1 lead to reversed velocity flows in gastrulation erythroid maturation investigated through in silico perturbation analyses on GraphVelo-based vector field. <strong>j</strong> Velocities derived from GraphVelo for the branching lineage in the hematopoiesis development and projected onto a pre-defined TSNE embedding. Directions of the projected cell velocities on TSNE are in agreement with the reported differentiation directions. <strong>k</strong> Terminal states identified by CellRank based on Markov chain formulation derived from GraphVelo velocities. <strong>l</strong> Phase portrait, velocity estimated by scVelo, refined velocity by GraphVelo, and gene expression of mature mRNA of identified rapid degradation genes (NPR3, ANGPT1). The cells were colored by the palantir pseudotime<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> in the phase portrait. The box plots showed cell-specific <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e1459"><mi mathvariant="bold-italic">γ</mi></math></span> for cells divided into bins according to pseudotime ordering in the phase protrait. The number of cells within each time bins from early to late is 653, 650, 654, 657, 657, respectively. Boxplots indicate median (middle line), first and third quartiles (box), and the upper whisker extends from the edges to the largest value no further than 1.5 × IQR (interquartile range) from the quartiles and the lower whisker extends from the edge to the smallest value at most 1.5 × IQR of the edge, while data beyond the end of the whiskers are outlying points that are plotted individually. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par19">GraphVelo uses the velocities of high-confidence genes obtained from any method as input to infer velocities of other genes. One can use several existing approaches to evaluate the confidence scores of inferred RNA velocity values of genes<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Alternatively, we identified a subset of Manifold-consistent Kinetics (MacK) Genes based on their agreement with prior knowledge or additional information acquired from other methods such as lineage tracing (Fig. <a href="#Fig3" class="usa-link">3b</a>), We first applied GraphVelo to a mouse erythroid maturation dataset<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. This study provided a transcriptional landscape of the erythroid lineage with well-documented differentiation trajectory during mouse gastrulation. Previous analyses have shown that the dataset contains genes with multiple rate kinetics, leading to erroneous prediction of the cell state transition direction<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. We selected the top 200 out of 450 velocity genes as MacK genes, representing those with robustly estimated velocities (see Methods for details). The projected vector field in UMAP showed consistency with prior knowledge in developmental biology (Fig. <a href="#Fig3" class="usa-link">3c</a>). We then used the corrected RNA velocities for dynamo velocity field analyses. The vector field-based pseudotime accurately predicted the lineage with scRNA-seq data of temporal mouse embryos (Fig. <a href="#Fig3" class="usa-link">3d</a>).</p>
<p id="Par20">Previous studies identified multiple rate kinetics (MURK) genes showing transcription bursts in the middle of erythroid differentiation<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. For example, two MURK genes (<em>Smim1</em> and <em>Hba-x</em>) showed complex patterns of phase portrait (Fig. <a href="#Fig3" class="usa-link">3f</a>). Consequently, the RNA velocity of <em>Simi1</em> inferred with scVelo was negative along a major part of the developmental axis (Fig. <a href="#Fig3" class="usa-link">3g i</a>), contradicting the trend of increasing <em>Simi1</em> mRNA levels (Fig. <a href="#Fig3" class="usa-link">3g iii</a>). For <em>Hba-x</em>, scVelo even failed to infer its RNA velocity. On the other hand, GraphVelo inferred velocities and predicted correct kinetic patterns of these genes (Fig. <a href="#Fig3" class="usa-link">3g ii</a>). Similar performances have been observed in other MURK genes (Supplementary Fig. <a href="#MOESM1" class="usa-link">7a</a>). To examine the overall prediction of cell state transitions from transcription burst genes, we projected the MURK genes velocity inferred from GraphVelo and scVelo to the predefined UMAP. The velocities from GraphVelo but not scVelo correctly captured the directional flow of differentiation using only MURK genes (Supplementary Fig. <a href="#MOESM1" class="usa-link">7b</a>). We further recapitulated gene succession and oscillation magnitudes along the erythroid trajectory and evaluated the phase-magnitude relationships of all highly variable genes (Supplementary Fig. <a href="#MOESM1" class="usa-link">8a</a>). Compared to non-MURK genes, MURK genes exhibited larger average velocity magnitudes and were predominantly enriched in the late stages of lineage progression. Analysis of genes with larger peak velocity amplitudes identified <em>Fth1</em>, <em>Car2</em>, and <em>Hbb-bs</em> as candidates with altered kinetic parameters. These dynamics patterns were evident in their phase portraits and gene expression trends (Supplementary Fig. <a href="#MOESM1" class="usa-link">8b, c</a>), consistent with previous reports that <em>Car2</em> transcription in erythroid cells is regulated by both the promoter activity and long-range enhancer interactions<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>—such complex regulations lead to its transcription dynamics not well-described by the simple transcription model used in the original splicing -based RNA velocity inference.</p>
<p id="Par21">Next, we evaluated the quantitative performance of GraphVelo in estimating cell-wise transition speed. Specifically, we estimated the speed of cell state transition using the norm of velocity vector in high-dimensional space and identified the transcriptional surge stage (Supplementary Fig. <a href="#MOESM1" class="usa-link">7c</a>). We hypothesized that the MURK genes, which exhibited a sudden increase in transcription rate during this stage, were responsible for the sharp acceleration in cell state transition speed. Using dynamo, we estimated the acceleration derived from the GraphVelo vector field and found that the acceleration value, as the derivative of the velocity vector, demonstrated its potential as a predictor for transcription burst genes (Supplementary Fig. <a href="#MOESM1" class="usa-link">7d</a>).</p>
<p id="Par22">With the velocity estimation extended to the whole gene space, we were able to perform comprehensive mechanistic analyses on the entire genome spectrum. First, we calculated the MacK score for each gene using the corrected RNA velocities. We hypothesized that a gene with a higher MacK score indicated a better agreement between its RNA velocity vector and the developmental axis, suggesting that the gene served as a potential lineage-driver gene. We ranked genes based on their scores and performed GO biological process enrichment analyses for the top genes. Indeed, the enriched processes were associated with erythropoiesis, including the heme biosynthetic process and interleukin-12-mediated signaling pathway<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> (Fig. <a href="#Fig3" class="usa-link">3e</a>).</p>
<p id="Par23">Next, we applied dynamo to perform differential geometry analyses of the vector field and mechanistically dissected the activation cascade of erythroid marker gene <em>Klf1</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">9a, b</a>). Jacobian analyses based on GraphVelo vector field revealed sequential activation of driver transcription factors (TFs) <em>Gata2</em>, <em>Gata1</em>, and <em>Klf1</em> during erythroid lineage differentiation, with <em>Gata1</em> subsequently repressing the expression of <em>Gata2</em> (Fig. <a href="#Fig3" class="usa-link">3h</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">9c</a>)<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. To further demonstrate the crucial role of transcriptional factor <em>Gata1</em> during erythropoiesis, we performed in silico genetic perturbation across all cells. Results showed that both inhibiting <em>Gata1</em> and upregulating the <em>Gata1</em> repressor <em>Spi1</em> lead to a reversal of normal developmental flow (Fig. <a href="#Fig3" class="usa-link">3i</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">9d</a>). The above analyses collectively suggest that activation of <em>Gata1</em> in the blood progenitors biased its differentiation to erythropoiesis, agreeing with experimental reports<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>.</p>
<p id="Par24">To further evaluate GraphVelo, we tested the method on another dataset of human bone marrow development<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. This developmental process has complex progressions from hematopoietic stem cells (HSCs) to three distinct branches: erythroid, monocyte, and common lymphoid progenitor (CLP). Again, we used the top 100 out of 454 velocity genes as MacK genes to predict the RNA velocities of 2000 highly variable genes. The GraphVelo velocity field accurately recovered the fate of cells on the sophisticated transcriptional landscape in contrast to scVelo (Fig. <a href="#Fig3" class="usa-link">3j, k</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">10a</a>). By combining the likelihood estimated by scVelo with the MacK score, we identified rapid degradation and transcription burst genes whose dynamics deviated from the RNA velocity assumptions (Supplementary Fig. <a href="#MOESM1" class="usa-link">10b, c</a>). <em>ANGPT1</em> and <em>RBPMS</em> are two examples which were overall highly expressed in the progenitors and decreased quickly along the trajectories (Fig. <a href="#Fig3" class="usa-link">3l</a>), reminiscent of what was shown in Fig. <a href="#Fig3" class="usa-link">3a</a>. These genes misled RNA velocity inference with scVelo assuming a constant degradation rate constant. GraphVelo revealed a cell context-specific transcription rate <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><math id="d33e1668"><mi>α</mi><mo>=</mo><mi>u</mi><mo>+</mo><mfrac><mrow><mi>d</mi><mi>u</mi></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac></math></span> and degradation constant <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><math id="d33e1687"><mi>γ</mi><mo>=</mo><mrow><mo>(</mo><mrow><mi>u</mi><mo>−</mo><mfrac><mrow><mi>d</mi><mi>s</mi></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac></mrow><mo>)</mo></mrow><mo>/</mo><mi>s</mi></math></span>, thus a degradation wave along the differentiation path (Fig. <a href="#Fig3" class="usa-link">3l</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">10d</a>), consistent with simulation result and reports on regulation of <em>ANGPT1</em> mRNA by microRNAs such as <em>miRNA-153-3p</em><sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">6d</a>).</p>
<p id="Par25">It is straightforward to apply Graphvelo to spatial transcriptomics datasets that permit RNA velocity inference. Note that the transcriptional dynamics of a gene can be affected by both the intracellular expression state and extracellular environmental factors. A data manifold containing additional spatial information allows distinction of cell states with similar expression profiles but distinct extracellular environments. Such a refined manifold leads to more accurate inference of the RNA velocities, which are typically performed over averaging the neighborhood of a cell state<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup> (see Methods) and is also used in GraphVelo for tangent space projection. We applied Graphvelo to the mouse coronal hemibrain dataset<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> processed with a bin size 60, which includes spliced and unspliced transcript information at spatial context (Supplementary Fig. <a href="#MOESM1" class="usa-link">11a</a>). GraphVelo inferred coherent velocity fields across brain regions, with streamlines on UMAP reflecting anatomically structured transitions that align with the spatial annotation (Supplementary Fig. <a href="#MOESM1" class="usa-link">11b</a>). Compared to the uncorrected RNA velocities using the dynamo build-in module, GraphVelo captured sharper transcription speed patterns, particularly in the dentate gyrus (DG), a neurogenic region where cell proliferation and neuronal differentiation persist into adulthood<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">11c, d</a>). Spatial mapping of representative genes revealed that GraphVelo velocities were more spatially confined and aligned with known expression domains, whereas the uncorrected velocities were noisier and less localized (Supplementary Fig. <a href="#MOESM1" class="usa-link">11e</a>). These results highlight the ability of GraphVelo to generate interpretable, spatially structured transcription dynamics in spatial transcriptomic data when splicing information is available.</p></section><section id="Sec7"><h3 class="pmc_sec_title">GraphVelo reconstructs host–pathogen transcriptome dynamics from infection trajectory</h3>
<p id="Par26">The continuous battle between human immune surveillance and viral immune evasion takes place in the host cell system after viral entry. scRNA-seq data provide a massive and parallel way of assessing the time evolution of both host and viral transcripts, unraveling the delicate inherent dynamics of a virus-host system<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. For splicing-based RNA velocity methods, genes lacking sufficient unspliced transcript counts are typically filtered out—which is inherently the case for viral genes due to their absence of introns. However, several studies<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>–<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> have applied tools such as scVelo to host-virus systems by focusing exclusively on host velocity genes that accurately reflect the lineage dynamics, rather than attempting to derive velocities directly from viral transcripts. GraphVelo enables inference of the velocities of virus RNA abundance based on the kinetics of host transcripts velocities, as illustrated next.</p>
<p id="Par27">We analyzed a human cytomegalovirus (HCMV) viral infection dataset to learn viral transcriptomic kinetics in monocyte-derived dendritic cells (moDCs)<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. The result from GraphVelo unraveled how viral infection progressed along the transcriptional space (Fig. <a href="#Fig4" class="usa-link">4a</a>). The velocity vectors pointed to directions consistent with an increasing trend of the percentage of viral RNAs in individual cells, which inherently served as an indicator of the infection time course<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Compared to the trend obtained with the raw RNA velocities from scVelo, the vector field-based pseudotime calculated using GraphVelo-corrected RNA velocities consistently showed higher correlation with the (pseudo)temporal progression of viral infection as reflected by viral RNA percentage (Fig. <a href="#Fig4" class="usa-link">4b</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Using GraphVelo velocities to infer host-virus infection trajectory and identify host-pathogen interactions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373899_41467_2025_62784_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7832/12373899/e1e75d6d9a15/41467_2025_62784_Fig4_HTML.jpg" loading="lazy" id="d33e1841" height="991" width="740" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Viral infection captured by the GraphVelo velocity field. Cells were colored by the percentage of viral RNA within a single cell. <strong>b</strong> Correlation between viral RNA percentage and pseudotime inferred by scVelo or GraphVelo. Correlation was calculated using a two-sided Spearman rank correlation test. <strong>c</strong> Viral RNA velocities infered by GraphVelo along the viral RNA percentage axis. The black dot line highlights the zero velocity. The solid line denotes the mean trend, dashed lines denote 1 s.d. <strong>d</strong> Boxplot summarizing the MacK scores of all viral genes calculated by GraphVelo, CellRank pseudotime kernel and random predictor. The number of viral genes is 67 for the statistical test. *** indicates Welch’s independent two-sided <em>t</em>-test at <em>p</em> &lt; 0.05. Boxplots indicate median (middle line), first and third quartiles (box), and the upper whisker extends from the edges to the largest value no further than 1.5 × IQR (interquartile range) from the quartiles and the lower whisker extends from the edge to the smallest value at most 1.5 × IQR of the edge, while data beyond the end of the whiskers are outlying points that are plotted individually. <strong>e</strong> Correlation between viral infection speed and RNA abundance. Genes were ranked by Spearman correlatioin coefficients. Host and viral genes that contribute to viral DNA synthesis were marked in the left side and those contribute to viral defense response were marked in the right side. Viral genes were highlighted in red. <strong>f</strong> UMAP representation of host and viral genes with distances defined by their dynamic expression patterns along the viral RNA percentage axis. <strong>g</strong> Example dynamic expression patterns within specific clusters (Leiden4, 5, 3, 6 from top to bottom) along the viral RNA percentage axis. Zero velocity was highlighted by black dot line. The solid line denotes mean trend, Shaded region represents 1 s.d. <strong>h</strong> GO enrichment of each cluster in (<strong>g</strong>). Gene set enrichment was performed using one-sided Fisher’s exact test, Benjamini–Hochberg correction. Adjusted <em>p</em>-values represent FDR-corrected significance of gene set enrichment. <strong>i</strong> Top host genes inhibited by each viral factor based on dynamo Jacobian analyses. Host effectors were organized by their involved pathways. <strong>j</strong> Dynamo prediction of total viral RNA change in response to in silico viral factor knockout. Viral factors were ranked by the mean of total viral RNA changes. The number of samples is 1454 for virtual perturbation screen. Boxplots indicate median (middle line), first and third quartiles (box), and the upper whisker extends from the edges to the largest value no further than 1.5 × IQR (interquartile range) from the quartiles and the lower whisker extends from the edge to the smallest value at most 1.5 × IQR of the edge. <strong>k</strong> Vector field change resultant from infinitesmal inhibition of UL123 during the viral infection process. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par28">Furthermore, the examination of individual genes revealed that the RNA velocities consistently predicted the trend of the mRNA expression level change with increasing virus load (Fig. <a href="#Fig4" class="usa-link">4c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">12a, b</a>). Most viral genes started with a fast-increase phase, and the expressions of some genes (e.g., <em>UL22A</em>) gradually saturated at high virus load, together with the corresponding RNA velocities approaching zero. One exception is <em>UL122</em>, whose expression profile increased first then decreased to a steady state level lower than the peak value. This overshooting is characteristic of a negative feedback network structure<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Indeed, a recent study reported that <em>UL122</em> negatively regulates its own promotor<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Furthermore, comparison of MacK scores across GraphVelo, the CellRank pseudotime kernel, and randomized prediction showed that GraphVelo-computed viral RNA velocities aligned best with the transcriptome gradient of viral load (Fig. <a href="#Fig4" class="usa-link">4d</a>). Note that the MacK score also served as a reliable predictor for dynamics-driving factors, specifically viral genes in this case (Supplementary Fig. <a href="#MOESM1" class="usa-link">12d</a>).</p>
<p id="Par29">We further quantified the velocity norm of all viral factors as infection speed, and observed that the transcription of viral factors was significantly restricted initially, then gradually increased along the trajectory (Supplementary Fig. <a href="#MOESM1" class="usa-link">12e</a>). Interestingly, most of the genes that exhibited positive correlation with the infection speed were related to viral DNA synthesis, while those negatively correlated to the infection speed were engaged in host viral defense response<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>(Fig. <a href="#Fig4" class="usa-link">4e</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">12f</a>).</p></section><section id="Sec8"><h3 class="pmc_sec_title">GraphVelo identifies host genomic response modules and predicts host-virus gene interactions</h3>
<p id="Par30">With GraphVelo-inferred RNA velocities, we probed the time evolution of lytic infection and the complex interplay between host and viral functional genomes. By fitting the GraphVelo velocity trends along the viral load axis, we identified genes with similar kinetic patterns (Fig. <a href="#Fig4" class="usa-link">4c</a>). Using the smoothened velocity trends to calculate the distances, we clustered the genes into seven major modules and visualized them on the UMAP space (Fig. <a href="#Fig4" class="usa-link">4f</a>). Not surprisingly, viral genes were concentrated in several enclosed regions, indicating that they formed distinct functional genomic modules during the lytic cycle<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. The genes showed two major dynamical features: acceleration and deceleration along the viral load axis (Fig. <a href="#Fig4" class="usa-link">4g</a>).</p>
<p id="Par31">To systematically investigate whether the dichotomy between host kinetics genes and viral genes share similar dynamics, we performed gene functional enrichment analyses of host genes residing close to the viral gene clusters. Genes located in the deceleration part were associated with repressing viral genome replication, which includes negative regulation of viral life cycle and known restriction factors in antiviral responses activated in DCs such as the induction of cytokine and chemokine responses as well as interactions with neutrophils<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. In parallel, the toll-like receptor signaling pathway, required for antiviral defense of the host, was arrested<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Neutrophil related processes, which typically cooperate closely with DCs to modulate adaptive immune responses<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, were suppressed. Therefore, the deceleration part also showed how critical set of host factors were silenced by viral entry to achieve immune evasion.</p>
<p id="Par32">The acceleration groups, on the other hand, demonstrated how viruses hijacked the host cell endogenous cellular programs for virus replication. Notably, the pathways related to viral genome replication were triggered, promoting DNA replication and transcription, such as negative regulation of G1/S transition of mitotic cell cycle<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, cellular response to DNA damage stimulus<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup> and regulation of transcription from RNA polymerase II promoter<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Cells showed a shift towards a transcriptional signature resembling the G1 phase (Supplementary Fig. <a href="#MOESM1" class="usa-link">12g</a>), agreeing with previous report on HCMV infection<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Along the infection process, antiviral interferon (IFN)-γ response of moDC cells was first activated then suppressed. These results highlighted the organized and antagonistic strategies adopted by both host cells and viruses during their tug-of-war for survival and proliferation.</p>
<p id="Par33">To investigate the crosstalk between host and viral factors systematically in depth, we performed dynamo Jacobian analyses. We scanned the entire spectrum of viral genomes and delineated how the HCMV factors silence IFN and NFκB signaling (Fig. <a href="#Fig4" class="usa-link">4i</a>). A large proportion of the identified viral factors functioned in evading host cell immune responses, a finding supported by several recent studies<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>–<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. In silico virus-directed knock out experiments revealed altered accumulation patterns of viral transcripts (Fig. <a href="#Fig4" class="usa-link">4j</a>). Notably, inhibition of <em>UL123</em>, which ranked first with the total viral RNA inhibition in our analyses, led to a qualitatively distinct trajectory. These results highlight the multifunctional <em>UL123</em> locus in the viral genome as a potential target for antiviral intervention<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. The analyses demonstrated potential usage of GraphVelo-inferred velocities for understanding the interactions between viral and host factors, assessing the effects of perturbations on infection, and designing potential antiviral interventions<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p></section><section id="Sec9"><h3 class="pmc_sec_title">GraphVelo reaveals an abortively infected cell population in SARS-CoV-2 infection</h3>
<p id="Par34">Although HCMV is renowned for its elaborate transcriptional landscape—characterized by extensive alternative splicing, overlapping transcripts, and diverse isoform expression—the RNA virus SARS-CoV-2 transcriptome reveals an even greater level of complexity within its relatively compact ~30 kb genome<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. To characterize the molecular mechanisms of host response which protect cells from productive trajectory, we applied GraphVelo to a SARS-CoV-2 infected Calu-3 cells dataset<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. To focus on the host-virus interactions and their corresponding fate outcomes, we subset the infected cell population for downstream analyses. The infected cluster M, characterized by interferon production genes, was hypothesized to represent a subpopulation of abortively infected cells<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>, similar to those described in herpesviruses HSV-1<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> and HCMV<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. We subsampled the infected cell population for downstream analyses and visualization, where cluster M exhibited distinct connectivity properties compared to the main infected groups (Supplementary Fig. <a href="#MOESM1" class="usa-link">13a</a>). To confirm that M is one of the terminal states of infection outcomes with such low cell abundance (Supplementary Fig. <a href="#MOESM1" class="usa-link">13b</a>), we applied GraphVelo to gain the quantitative velocity with dynamo outcomes as input (Supplementary Fig. <a href="#MOESM1" class="usa-link">13c</a>). Using vector field topology analysis, we classified an initial state with relatively low viral load, two terminal states associated with high apoptosis activity, and a saddle point characterized by high viral load (Supplementary Fig. <a href="#MOESM1" class="usa-link">13d</a>). Notably, the region corresponding to cluster M was identified as an attractor, confirming it represents an abortively infected cell state with a high death rate<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>.</p>
<p id="Par35">We further validated the complex lineage commitments of SARS-CoV-2–infected cells using the CellRank framework<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, which successfully identified three potential terminal states as outcomes of host-virus competition (Supplementary Fig. <a href="#MOESM1" class="usa-link">13e</a>). Interestingly, we characterized the saddle stage in dynamo as a productive terminal state governed by viral genes (Supplementary Fig. <a href="#MOESM1" class="usa-link">13f</a>), exhibiting high viral transcription speed (Supplementary Fig. <a href="#MOESM1" class="usa-link">13g</a>). The pathogen-triggered cell death can be driven for protecting the host or for pathogen dissemination purpose<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. To investigate the underlying causes for host cell death, we performed gene functional enrichment analyses on the top correlated genes along distinct lineages (Supplementary Fig. <a href="#MOESM1" class="usa-link">13h</a>). Active programs in the abortive infection lineage were highly enriched for host defense mechanisms against viral invasion, consistent with previous studies<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. In contrast, the drivers of the apoptosis-associated lineage revealed a distinct profile. Notably, we observed enrichment for cellular responses to unfolded proteins, which have been implicated in facilitating pathogen-mediated dissemination of infected cells<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>,<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Additionally, <em>ERBB2</em> inhibition has been shown to suppress SARS-CoV-2 replication<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. These findings support the hypothesis that similar cell death outcomes may arise from fundamentally different host responses. Abortively infected cells appear to promote efficient pathogen clearance, likely through cytokine-mediated immune activation that eliminates both the infected cell and the virus. Conversely, cells undergoing virus-induced apoptosis fail to clear the virus, with cell death instead serving as a mechanism for viral escape, immune evasion, and potential dissemination to deeper tissue layers or the bloodstream.</p></section><section id="Sec10"><h3 class="pmc_sec_title">GraphVelo permits multi-omics velocity inference and chromatin dynamics analyses</h3>
<p id="Par36">The molecular anatomy during cell development entails multiple layers, and how different layers coordinate to regulate gene expression is a fundamental problem. For example, the anagen hair follicle features distinct lineages branching from a central population of progenitor cells. Ma et al.<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. used SHARE-seq to capture both the transcriptome and the epigenome data simultaneously for the lineage commitment process from transit-amplifying cells (TACs) to the inner root sheath (IRS), cuticle layer, and medulla. Upon robust selection of estimated genes following dynamo criteria (Methods), we further refined the RNA velocities of these genes through tangent space projection and obtained the chromatin open/close dynamics from the corresponding scATAC data using GraphVelo. The resultant vector field in the combined transcriptome-epigenome space proved to reconstruct the correct multilineages differentiation paths during the anagen phase (Fig. <a href="#Fig5" class="usa-link">5a</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Inferring epigenome and transcriptome consistent dynamics in mouse hair follicle development using GraphVelo multi-omics velocities.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373899_41467_2025_62784_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7832/12373899/9a7b4698a7d0/41467_2025_62784_Fig5_HTML.jpg" loading="lazy" id="d33e2121" height="999" width="775" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> GraphVelo velocity fields of mouse hair follicle development. Cells were colored by cell macrostates. <strong>b</strong> Number of terminal states predicted by CellRank using velocities inferred with different methods. <strong>c</strong> Driver genes along multiple lineages identified through CellRank. <strong>d</strong> Topological analyses of GraphVelo vector field identified novel root cells and attractors residing in three terminal states(IRS, hair shaft-cuticle cortex, and medulla). <strong>e</strong> Expression levels of marker genes in novel root cells and expected root cells. Markers identified by Ma et al.63 were highlighted with stars, and newly identified markers were highlighted in bold. <strong>f</strong> Regression results of MSD values along the transition path from the expected root or novel root to IRS. Two genes Runx1 and Shh genes with large MSD originating from the novel root were highlighted. The solid line denotes mean of regression result, Shaded region represents 1 s.d. <strong>g</strong> DTW distance between RNA velocity and chromatin velocity of individual genes. CCD genes were colored in red. The dotted line indicates the elbow point separating the decoupled genes from the rest. <strong>h</strong> GO enrichment of decoupled genes in (<strong>g</strong>). <strong>i</strong> Line plot of nomarlized RNA and chromatin velocity along pseudotime for genes predicted by GraphVelo to have notable decoupling patterns. Chromatin velocity trends were colored as brown and RNA velocity trends were colored as green. <strong>j</strong> Heatmaps of Jacobian element distribution along the axis of regulator RNA abundance of four regulator effector circuits: i) Lef1 versus Wnt3 chromatin accessibilities. ii) Hoxc13 versus Wnt3 chromatin accessibilities. iii) Lef1 versus Wnt3 transcription. iv) Hoxc13 versus Wnt3 transcription. <strong>k</strong> Effective dose-response curves obtained from integrating the averaged Jacobian elements over the corresponding normalized regulator mature mRNA regulator level in (<strong>j</strong>). Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par37">To test the consistency of dynamics across different modalities, we performed CellRank terminate stage analyses<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> from the refined velocity vectors. Using GraphVelo velocities of either the RNA modality or the ATAC modality, we accurately estimated three diverse terminal stages (Fig. <a href="#Fig5" class="usa-link">5b</a>). For comparison, we also performed similar analyses using MultiVelo, scVelo with all velocity genes or robustly estimated genes in above GraphVelo studies and pseudotime-based vector field inferred by CellRank. The 2D projection of these vector field functions also exhibited seemingly correct velocity flow direction (Supplementary Fig. <a href="#MOESM1" class="usa-link">14a</a>). However, none of them captured the cell fate commitment based on coarse-grained transition matrix (Fig. <a href="#Fig5" class="usa-link">5b</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">14b</a>). Notably, the results from the RNA modality and the ATAC modality of MultiVelo gave inconsistent results. GraphVelo-corrected velocities, on the other hand, helped identify the top-correlating genes towards individual terminal populations which showed agreement with previous study<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup> (Fig. <a href="#Fig5" class="usa-link">5c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>).</p>
<p id="Par38">Next, we conducted differential geometry analyses based on the composite GraphVelo vector field. We identified novel root cells, which were also characterized by chromatin potential (Fig. <a href="#Fig5" class="usa-link">5d</a>)<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. These novel root cells expressed distinct marker genes compared to the expected root cells using the Wilcoxon test (Fig. <a href="#Fig5" class="usa-link">5e</a>). Moreover, we unraveled differentially expressed markers identified by the original study<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>, as well as new differentiation-potent genes and validated their initiation properties in another transcriptome dataset (Supplementary Fig. <a href="#MOESM1" class="usa-link">16a</a>)<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. To further investigate how these two distinct groups of root cells convert to other cell types, we performed a least action path (LAP) analysis between different cell phenotypes. The expected and novel root cells converted to the IRS terminal state following two distinct least action paths in the vector field (Supplementary Fig. <a href="#MOESM1" class="usa-link">16b, c</a>). The two paths revealed different temporal change patterns of transcription factor expression profiles (Supplementary Fig. <a href="#MOESM1" class="usa-link">16d</a>). We calculated the mean-squared displacement (MSD) for every transcription factor to explore the dynamics of TFs along the path from novel root to IRS. The result demonstrated that the fate conversion by novel root was mediated by the <em>Shh</em>-<em>Runx1</em> signaling axis (Fig. <a href="#Fig5" class="usa-link">5f</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">16d</a>), which has been demonstrated in human embryonic stem cells<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> and is crucial for hair development<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. In summary, GraphVelo unraveled the multiple molecular mechanisms that orchestrated hair follicle morphogenesis.</p>
<p id="Par39">With available chromatin velocity and RNA velocity, we set to quantify the coupling/decoupling relationships between chromatin structure and gene expression for each gene (see Methods). Here chromatin structure refers to the extent of exposure/accessibility of the gene locus to the environment as indicated by scATACseq data, shortly called open or closed state; and chromatin velocities refer to changes between these states as inferred from scATACseq counts at the examined gene locus. We used dynamic time warping (DTW) distances between the velocities from different omics layers to quantify the similarity between temporal patterns of these two modalities for each gene. A higher DTW value indicates higher similarity. Using the elbow of the ranked distance curve as a cutoff we identified genes that showed decoupled transcription and chromatin structure dynamics. These decoupled genes had an accumulation of cell cycle-dependent (CCD) genes found in previous study<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> (Fig. <a href="#Fig5" class="usa-link">5g</a>). This group of genes showed strong involvement in cell cycle-related processes, as indicated by GO enrichment analyses (Fig. <a href="#Fig5" class="usa-link">5h</a>). Close examinations indicated that the transcription of cell cycle related genes decreased along the differentiation path, while the chromatin structure at the corresponding loci remained open (Fig. <a href="#Fig5" class="usa-link">5i</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">17</a>). To validate this hypothesis, we further applied GraphVelo to a recently published 10x Multiome dataset from developing human cortex<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">18a</a>). Following the same analyses, we identified decoupled genes and found out that most of these genes were related to cell cycle (Supplementary Fig. <a href="#MOESM1" class="usa-link">18b–f</a>). which has also been reported in a previous MultiVelo study<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>.</p>
<p id="Par40">We further performed dynamo differential geometry analyses on the composite transcriptome-chromatin vector field. One intriguing phenomenon observed in lineage dynamics is that <em>Lef1</em> and <em>Hoxc13</em> are the driver TFs correlated with domains of regulatory chromatin (DORCs) of <em>Wnt3</em><sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Differential geometry analyses on the composite vector field can go beyond correlation analyses and provide an underlying casual mechanism. As a prerequisite for such analyses, GraphVelo-inferred RNA and chromatin velocities of the three genes correctly predicted the trend of change of mRNA and ATAC-seq counts (Supplementary Fig. <a href="#MOESM1" class="usa-link">19a, b</a>), in contrast to the performance of MultiVelo (Supplementary Fig. <a href="#MOESM1" class="usa-link">19c</a>). Then, Jacobian analyses on the GraphVelo vector field confirmed that priming activation of <em>Lef1</em> subsequently activated the <em>Hoxc13</em> TF<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">19d, e</a>). Both <em>Lef1</em> and <em>Hoxc13</em> were found to activate the <em>Wnt3</em> target gene, initiating lineage commitment (Supplementary Fig. <a href="#MOESM1" class="usa-link">19f, g</a>). To quantitatively understand how the two TFs affect <em>Wnt3</em> chromatin structure and transcription, we plotted the response heatmap to reflect the distributions of Jacobian elements versus the abundance of mature mRNA for each TF (Fig. <a href="#Fig5" class="usa-link">5j</a>). The two terms <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><math id="d33e2293"><mi>∂</mi><msub><mrow><mi>f</mi></mrow><mrow><mi>W</mi><mi>n</mi><mi>t</mi><mn>3</mn><mo>−</mo><mi mathvariant="normal">chrom</mi></mrow></msub><mo>/</mo><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>L</mi><mi>e</mi><mi>f</mi><mn>1</mn></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><math id="d33e2323"><mi>∂</mi><msub><mrow><mi>f</mi></mrow><mrow><mi>W</mi><mi>n</mi><mi>t</mi><mn>3</mn><mo>−</mo><mi mathvariant="normal">chrom</mi></mrow></msub><mo>/</mo><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>H</mi><mi>o</mi><mi>x</mi><mi>c</mi><mn>13</mn></mrow></msub></math></span> started with positive values at low concentrations of TF mRNA copy numbers then decreased to zero, indicating that increasing the level of either TF lead to further opening of the <em>Wnt3</em> chromatin region, and the effect saturated at high TF expression. The other two term <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><math id="d33e2358"><mi>∂</mi><msub><mrow><mi>f</mi></mrow><mrow><mi>W</mi><mi>n</mi><mi>t</mi><mn>3</mn></mrow></msub><mo>/</mo><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>L</mi><mi>e</mi><mi>f</mi><mn>1</mn></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><math id="d33e2385"><mi>∂</mi><msub><mrow><mi>f</mi></mrow><mrow><mi>W</mi><mi>n</mi><mi>t</mi><mn>3</mn></mrow></msub><mo>/</mo><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>H</mi><mi>o</mi><mi>x</mi><mi>c</mi><mn>13</mn></mrow></msub></math></span> increased with the TF levels, indicating that these two TFs also activated <em>Wnt3</em> transcription. Upon integration of the Jacobian elements over regulator expression changes, we obtained the effective dose-response curves obtained (see Methods), which revealed more transparently the TF dose lag between the opening of the target chromatin region and the initialization of transcription (Fig. <a href="#Fig5" class="usa-link">5k</a>).</p>
<p id="Par41">Our results therefore illustrated the sequential events that these two driver TFs, <em>Lef1 and Hoxc13</em>, drove as pioneer transcription factors (PTFs) to initiate local chromatin opening and then activated the transcription of <em>Wnt3</em>. Notably, computational methods and experimental research confirm that <em>Lef1</em> acts as a nucleosome binder and exhibits diverse binding patterns across various cell lines<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. The <em>Hox</em> family of TFs has also been shown to have the capacity to bind their targets in an inaccessible chromatin context and trigger the switch to an accessible state<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>, consistent with our analyses that <em>Hoxc13</em> revealing that it shared a regulation mechanism similar to that of PTF <em>Lef1</em>.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par42">In this work, we provided a general framework that extends the framework developed for RNA velocity and related approaches to various data modalities such as proteomics, spatial genomics, 3 d genome organization, and imaging data, which were originally beyond the reach of this framework. We validated GraphVelo using various in vivo cellular kinetics models, confirmed its efficacy and robustness in handling complex and noisy multimodal data. Upon application to various datasets, we unraveled gene regulation relations of an extended list of genes, host-virus gene regulations, and coupling between transcription and local chromatin structures. GraphVelo can be seamlessly integrated with broad downstream analyses, such as dynamo continuous vector field analyses, as well as Markovian analyses using graph dynamo or CellRank.</p></section><section id="Sec12"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec13"><h3 class="pmc_sec_title">Dynamical systems theory formulation of cellular state transitions</h3>
<p id="Par43">Assume that a cell state can be represented by the cell volume <em>V</em> (or cellular compartment size) and the copy number of <em>L ≫ 1</em> pairs of gene products (<strong>n</strong><sub><em>m</em></sub>, <strong>n</strong><sub><em>p</em></sub>), where <em>m</em> and <em>p</em> designate mRNA and protein, and the bold fonts indicate vectors. For simplicity here we only consider <em>m</em> and <em>p</em>. It is straightforward to generalize to finer cell state specifications, for example, with distinction of nuclear and cytosol localizations, posttranslational states of proteins, other species such as microRNAs, epigenetic states, etc.</p>
<p id="Par44">The temporal evolution of the cell state is described by a set of chemical master equations. When the copy numbers of molecular species are not too small, and the chemical reactions are not strongly coupled, Gillespie showed that the chemical master equations can be approximated by a set of chemical Langevin equations<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. With extrinsic noises also included, we assume the ansatz that the dynamics of cell state is described by a set of generic stochastic differential equations,</p>
<table class="disp-formula p" id="Equa"><tr><td class="formula"><math id="d33e2495" display="block"><mfrac><mrow><mi>d</mi><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>m</mi></mrow></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mi mathvariant="bold">F</mi><mfenced close=")" open="("><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>m</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>p</mi></mrow></msub></mrow></mfenced><mo>+</mo><mi>ζ</mi><mfenced close=")" open="("><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>m</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>p</mi></mrow></msub><mo>,</mo><mi>t</mi></mrow></mfenced></math></td></tr></table>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e2552" display="block"><mfrac><mrow><mi>d</mi><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>p</mi></mrow></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mi mathvariant="bold">G</mi><mfenced close=")" open="("><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>m</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>p</mi></mrow></msub></mrow></mfenced><mo>+</mo><mi>η</mi><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>m</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>p</mi></mrow></msub><mo>,</mo><mi>t</mi></mrow><mo>)</mo></mrow></math></td>
<td class="label">4</td>
</tr></table>
<p>where the <em>L</em>-dimensional vectors <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><math id="d33e2614"><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>m</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><math id="d33e2628"><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>p</mi></mrow></msub></math></span> are cellular concentrations of <em>m</em> and <em>p</em>, and the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><math id="d33e2649"><mi>ζ</mi></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><math id="d33e2658"><mi>η</mi></math></span> are taken as white noises with zero mean.</p>
<p id="Par45">The low-dimensional manifold assumption is central to machine learning approaches on data analyses. From a dynamical systems theory perspective, after a transient time, a multi-dimensional dynamical system often converges to a low-dimensional slow manifold. In practice, such property has been exploited with techniques such as quasi-steady-state approximation, quasi-equilibrium approximation. For a rigorous formulation, assume that one can identify a set of variables <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><math id="d33e2669"><mfenced close=")" open="("><mrow><mi mathvariant="bold">z</mi><mo>,</mo><mi mathvariant="bold">Z</mi></mrow></mfenced></math></span> with (<em>2L</em>-<em>M</em>) dimensional fast variables <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><math id="d33e2690"><mi mathvariant="bold">z</mi><mo>=</mo><mi mathvariant="bold">z</mi><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>m</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>p</mi></mrow></msub></mrow><mo>)</mo></mrow></math></span> and <em>M</em>-dimensional slower variables <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><math id="d33e2724"><mi mathvariant="bold">Z</mi><mo>=</mo><mi mathvariant="bold">Z</mi><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>m</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>p</mi></mrow></msub></mrow><mo>)</mo></mrow></math></span>. Xing and Kim extended the celebrated Zwanzig-Mori projection<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>,<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup> to a general dynamical system described by Eq. <a href="#Equ4" class="usa-link">4</a><sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. The projection procedure results in a set of stochastic integral-differential equations of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><math id="d33e2767"><mi mathvariant="bold">Z</mi></math></span> with colored noises, which are formally equivalent to Eq. <a href="#Equ4" class="usa-link">4</a>. Then if one assumes clear time scale separation between <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><math id="d33e2781"><mi mathvariant="bold">z</mi></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><math id="d33e2791"><mi mathvariant="bold">Z</mi></math></span>, the equations reduce to a set of Langevin equations with white noises,<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><math id="d33e2801"><mfrac><mrow><mi>d</mi><mi mathvariant="bold">Z</mi></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mi mathvariant="bold">A</mi><mrow><mo>(</mo><mrow><mi mathvariant="bold">Z</mi><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>m</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>p</mi></mrow></msub></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mo>+</mo><mi>η</mi><mfenced close=")" open="("><mrow><mi mathvariant="bold">Z</mi><mo>,</mo><mi>t</mi></mrow></mfenced></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><math id="d33e2851"><mfenced close=")" open="("><mrow><mi mathvariant="bold">Z</mi><mo>,</mo><mi>t</mi></mrow></mfenced></math></span> are white noises with zero mean. Through ensemble averaging over the vicinity of a given point <strong>Z</strong>, one has</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><math id="d33e2869" display="block"><mi mathvariant="bold">v</mi><mo>≡</mo><mo>&lt;</mo><mfrac><mrow><mi>d</mi><mi mathvariant="bold">Z</mi></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><msub><mrow><mo>&gt;</mo></mrow><mrow><mi mathvariant="bold">Z</mi></mrow></msub><mo>=</mo><mi mathvariant="bold">A</mi><mfenced close=")" open="("><mrow><mi mathvariant="bold">Z</mi><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>m</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>p</mi></mrow></msub></mrow><mo>)</mo></mrow></mrow></mfenced><mo>.</mo></math></td>
<td class="label">5</td>
</tr></table>
<p id="Par46">The equations define a <em>M</em>-dimensional manifold embedded in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><math id="d33e2924"><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>m</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>p</mi></mrow></msub></mrow><mo>)</mo></mrow></math></span> space. A scRNA-seq data then measures the corresponding manifold projected to the transcriptomic subspace.</p>
<p id="Par47">One should note that in practice the reported RNA velocity vector of a cell state <em>i</em> is typically obtained through averaging the raw velocity vectors of cell states within its neighborhood <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><math id="d33e2954"><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> on the manifold as a numerical approximation of the ensemble average, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><math id="d33e2968"><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mi mathvariant="bold">x</mi><mi>i</mi></mrow></msub><mo>=</mo><mo>&lt;</mo><mfrac><mrow><mi>d</mi><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi mathvariant="bold">m</mi></mrow></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>&gt;</mo><mo>≈</mo><msub><mrow><mo>∑</mo></mrow><mrow><mi>j</mi><mo>∈</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mi mathvariant="bold">x</mi><mi>j</mi></mrow></msub></math></span>. In this work, we used the k-nearest neighbor (kNN) algorithm to define the neighborhood in single-modality datasets, including spatial-transcriptomics datasets. For multi-omics data, the neighborhood of a cell state was defined using weighted nearest neighbors (WNN)<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup> in the composite cell state space. The procedure of applying GraphVelo is the same for different data types, except using the neighborhoods defined on their corresponding data (cell state) manifold.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Mathematical foundation for applying GraphVelo to multi-omics datasets</h3>
<p id="Par48">Equation <a href="#Equ3" class="usa-link">3</a> in the main text applies to the transformation between a manifold embedded in a state space and in a subspace. According to the Whitney Embedding theorem<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, any smooth real <em>M</em>-dimensional manifold can be embedded in a <em>2M</em>-dimensional real space provided that <em>M</em> &gt; 0. Consider a full set of genes versus a subset in a scRNAseq dataset, or a combined scRNAseq/scATACseq multi-omics dataset versus the scRNAseq subset. Assume that the full cell state space has a dimensionality <em>N</em>, while a single cell data manifold is typically low-dimensional with <em>M</em> ≪ <em>N</em>. Then the Whitney Embedding theorem<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> suggests that with proper choice of the subset the manifolds in the full space and the subspace are homeomorphic or at least piece-wise homeomorphic (Fig. <a href="#Fig1" class="usa-link">1b</a>, <a href="#MOESM1" class="usa-link">Supplementary Notes</a> <a href="#MOESM1" class="usa-link">1.4</a>), i.e., a one-to-one mapping exists between the two. Then applying Eq. <a href="#Equ3" class="usa-link">3</a> allows one to infer the velocity vectors for the full-space representation from those of the subspace.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Denoise velocity vectors in the space of principal components (PCs)</h3>
<p id="Par49">Learning coefficients <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><math id="d33e3076"><msub><mrow><mi mathvariant="bold">ϕ</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> in the gene space directly often fails due to thousands of gene profiles. To avoid the curse of high dimensionality and learn parameters in a compact manifold, we designed a procedure to denoise the velocities in a reduced PCA space. Specifically, we extrapolated the cell state <em>i</em> in the original space using the infinitesimal propagation operator to extrapolate the future state:</p>
<table class="disp-formula p" id="Equb"><tr><td class="formula"><math id="d33e3093" display="block"><msubsup><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>i</mi></mrow><mrow><mo>′</mo></mrow></msubsup><mo>=</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>+</mo><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>⋅</mo><mi>d</mi><msub><mrow><mi>t</mi></mrow><mrow><mi>i</mi></mrow></msub></math></td></tr></table>
<p id="Par50">Moreover, we estimated an optimal step size <em>dt</em> based on the local density to guarantee the cell states are bound to the manifold:</p>
<table class="disp-formula p" id="Equc"><tr><td class="formula"><math id="d33e3134" display="block"><mi>d</mi><msub><mrow><mi>t</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><mi mathvariant="normal">median</mi><mfenced close=")" open="("><mrow><mfrac><mrow><mfrac><mrow><mn>1</mn></mrow><mrow><mi>k</mi></mrow></mfrac><msubsup><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>k</mi></mrow></msubsup><mo>∥</mo><msub><mrow><mi mathvariant="bold-italic">δ</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>∥</mo></mrow><mrow><mo>∥</mo><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>∥</mo></mrow></mfrac></mrow></mfenced></math></td></tr></table>
<p id="Par51">After utilizing the cell-dependent <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><math id="d33e3189"><msub><mrow><mi>t</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> to forcing the predictions inhabiting regions of the phenotypical manifold, we applied dimension reduction to project both current and future status from the gene space to the PCA space through linear transformation. Then we obtained the projected velocity vectors as:</p>
<table class="disp-formula p" id="Equd"><tr><td class="formula"><math id="d33e3202" display="block"><msubsup><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">PCA</mi></mrow></msubsup><mo>=</mo><mfrac><mrow><msubsup><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>i</mi></mrow><mrow><mo>′</mo></mrow></msubsup><mi mathvariant="bold">Q</mi><mo>−</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>i</mi></mrow></msub><mi mathvariant="bold">Q</mi></mrow><mrow><mi>d</mi><msub><mrow><mi>t</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></mfrac></math></td></tr></table>
<p>where <strong>Q</strong> is the PC loading matrix estimated using x that serves as the coordinate transformation matrix.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Manifold-consistent kinetic genes</h3>
<p id="Par52">The kinetic assumptions between nascent and mature RNA fail when the underlying parameters shift along the developmental trajectory<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>, which leads to transcription burst and rapid degradation in the phase portraits (Fig. <a href="#Fig3" class="usa-link">3a</a>). An internal clock exists during cell proliferation and differentiation. Current methods rely on different criteria to select confident estimated velocity genes (see <a href="#MOESM1" class="usa-link">Supplementary Notes</a> <a href="#MOESM1" class="usa-link">1.5</a> for detailed discussion). Here, we presume that the velocity of robustly estimated genes should be consistent with the (pseudo)time derivative estimated under the manifold assumption. We can utilize any available ts inferred from data manifold by either pseudotime, velocity latent time, or lineage tracing to approximate the temporal information and use <em>k</em> nearest neighbors to define the locally linear plane. After ordering cells within the local Euclidean space, we calculate the MacK score for any gene <em>g</em> as an indicator of whether the sign of estimated velocity agrees with the dynamic cascades within manifold,</p>
<table class="disp-formula p" id="Eque"><tr><td class="formula"><math id="d33e3275" display="block"><mi>M</mi><mi>a</mi><mi>c</mi><mi>K</mi><mi>s</mi><mi>c</mi><mi>o</mi><mi>r</mi><mi>e</mi><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi>n</mi></mrow></mfrac><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>j</mi><mi mathvariant="script">\in</mi><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi><mrow><mo>(</mo><mrow><mi>i</mi></mrow><mo>)</mo></mrow></mrow></msub><mi mathvariant="double-struck">I</mi><mfenced close=")" open="("><mrow><mi mathvariant="normal">sgn</mi><mfenced close=")" open="("><mrow><mfrac><mrow><mi mathvariant="normal">Δ</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mrow><mo>(</mo><mrow><mi>g</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi mathvariant="normal">Δ</mi><msub><mrow><mi>t</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mfrac></mrow></mfenced><mo>=</mo><mi mathvariant="normal">sgn</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>v</mi></mrow><mrow><mi>i</mi></mrow></msub><mrow><mo>(</mo><mrow><mi>g</mi></mrow><mo>)</mo></mrow></mrow></mfenced></mrow></mfenced></math></td></tr></table>
<p>Where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><math id="d33e3361"><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> indicates the neighbor points of cell <em>i</em>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><math id="d33e3378"><mi mathvariant="double-struck">l</mi></math></span> represents the indicator function and sgn returns the sign of the values. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><math id="d33e3388"><mi mathvariant="normal">Δ</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mfenced close=")" open="("><mrow><mi>g</mi></mrow></mfenced><mo>,</mo><msub><mrow><mi>v</mi></mrow><mrow><mi>i</mi></mrow></msub><mrow><mo>(</mo><mrow><mi>g</mi></mrow><mo>)</mo></mrow></math></span> are the difference in abundance of gene <em>g</em> between cell <em>i</em> and <em>j</em>, and the velocity of gene <em>g</em> in cell <em>i</em>, respectively. We parallelize the calculation to scale efficiently with the number of genes, which is important due to the number of highly variable genes. We want to point out that one can use methods other than the MacK score to identify genes with reliable velocity estimations.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Dynamo criteria</h3>
<p id="Par53">Dynamo<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup> offers a correction strategy by removing genes with low gene-wise confidence in the phase plane. This allows us to identify genes that appear in incorrect phase portrait positions and contribute to erroneous flow directions (illustrated in Fig3. a). To filter out genes with misleading dynamic patterns, one can supply the established lineage hierarchy information to the <em>dyn.tl.confident_cell_velocities</em> function in dynamo. This function scores each gene based on the agreement of its behavior in the splicing phase diagram with the input lineage hierarchy priors.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Post-GraphVelo analyses</h3>
<section id="Sec19"><h4 class="pmc_sec_title">Reconstruction of extended dynamo vector field from multi-omics data</h4>
<p id="Par54">Dynamo is a general framework of reconstructing dynamical models from scRNAseq data, and it is straightforward to generalize to multi-modal data. The framework is based on specific realizations of Eq. <a href="#Equ5" class="usa-link">5</a>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><math id="d33e3454"><mi mathvariant="bold">v</mi><mo>≡</mo><mo>&lt;</mo><mfrac><mrow><mi>d</mi><mi mathvariant="bold">x</mi></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><msub><mrow><mo>&gt;</mo></mrow><mrow><mi mathvariant="bold">x</mi></mrow></msub><mo>=</mo><mi mathvariant="bold">A</mi><mfenced close=")" open="("><mrow><mi mathvariant="bold-italic">x</mi></mrow></mfenced></math></span>, with state vector <strong>x</strong> being the transcript concentrations for scRNAseq data, and combined transcript concentrations and continuous quantification of locus-specific chromatin open-close state for multi-omics scRNAseq/scATACseq data. The variables <strong>x</strong> can be defined in various representations, e.g., the original gene space, principal component subspace, latent space defined by a variational autoencoder, etc. With GraphVelo it is straightforward to transform <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><math id="d33e3493"><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mi mathvariant="bold">x</mi></mrow></msub></math></span> between different representations.</p>
<p id="Par55">The continuous vector field functions <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><math id="d33e3510"><mi mathvariant="bold">A</mi><mfenced close=")" open="("><mrow><mi mathvariant="bold">x</mi></mrow></mfenced></math></span> contain quantitative regulation relations between genes that are learned from single cell data points of (<strong>x,</strong>
<strong>v</strong><sub><strong>x</strong></sub><strong>)</strong>. Various algorithms can be used to learn the analytical forms of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><math id="d33e3535"><mi mathvariant="bold">A</mi><mfenced close=")" open="("><mrow><mi mathvariant="bold">x</mi></mrow></mfenced></math></span>. The original dynamo paper illustrated a Reproducing Kernel Hilbert Space (RKHS) representation method. The method expresses <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><math id="d33e3550"><mi mathvariant="bold">A</mi><mfenced close=")" open="("><mrow><mi mathvariant="bold">x</mi></mrow></mfenced></math></span> as a linear combination of pre-selected basis functions, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><math id="d33e3564"><mi mathvariant="bold">v</mi><mo>=</mo><mi mathvariant="bold">A</mi><mfenced close=")" open="("><mrow><mi mathvariant="bold">x</mi></mrow></mfenced><mo>=</mo><msub><mrow><mo>∑</mo></mrow><mrow><mi>α</mi></mrow></msub><msub><mrow><mi mathvariant="bold">c</mi></mrow><mrow><mi mathvariant="bold-italic">α</mi></mrow></msub><msub><mrow><mi mathvariant="bold">Γ</mi></mrow><mrow><mi mathvariant="normal">α</mi></mrow></msub><mrow><mo>(</mo><mrow><mi mathvariant="bold">x</mi></mrow><mo>)</mo></mrow></math></span>, similar to the more familiar Taylor expansion that uses a linear combination of polynomial functions to represent a continuous analytical function. It should be noted that the basis functions and so <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><math id="d33e3607"><mi mathvariant="bold">A</mi><mfenced close=")" open="("><mrow><mi mathvariant="bold">x</mi></mrow></mfenced></math></span> are generally nonlinear functions of <strong>x</strong>. Following Qiu et al.<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>., we chose Gaussian functions centered at selected reference points <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><math id="d33e3628"><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold">x</mi></mrow><mo>~</mo></mover></mrow><mrow><mi mathvariant="normal">α</mi></mrow></msub><mo>,</mo><msup><mrow><msub><mrow><mi mathvariant="bold">Γ</mi></mrow><mrow><mi>a</mi></mrow></msub><mfenced close=")" open="("><mrow><mi mathvariant="bold">x</mi><mo>,</mo><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold">x</mi></mrow><mo>~</mo></mover></mrow><mrow><mi mathvariant="normal">α</mi></mrow></msub></mrow></mfenced><mo>=</mo><mi mathvariant="normal">e</mi></mrow><mrow><mo>−</mo><mn>2</mn><mi>w</mi><msup><mrow><mo>∥</mo><mi mathvariant="bold">x</mi><mo>−</mo><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold">x</mi></mrow><mo>~</mo></mover></mrow><mrow><mi>α</mi></mrow></msub><mo>∥</mo></mrow><mrow><mn>2</mn></mrow></msup></mrow></msup></math></span>, with default parameter value of <em>w</em> in the package dynamo. Then we determined the coefficient vectors <strong>c</strong><sub><strong>α</strong></sub> through minimizing the loss function <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><math id="d33e3705"><mi mathvariant="normal">Φ</mi><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold">c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold">c</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mrow><mi mathvariant="bold">c</mi></mrow><mrow><mi mathvariant="normal">m</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>=</mo><msubsup><mrow><mo>∑</mo></mrow><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><msup><mrow><mo>∥</mo><msub><mrow><mi mathvariant="bold">v</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>−</mo><msub><mrow><mo>∑</mo></mrow><mrow><mi>α</mi></mrow></msub><mi mathvariant="normal">Γ</mi><mrow><mo>(</mo><mrow><mi mathvariant="bold">x</mi><mo>,</mo><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold">x</mi></mrow><mo>~</mo></mover></mrow><mrow><mi>α</mi></mrow></msub></mrow><mo>)</mo></mrow><msub><mrow><mi mathvariant="bold">c</mi></mrow><mrow><mi>α</mi></mrow></msub><mo>∥</mo></mrow><mrow><mn>2</mn></mrow></msup><mo>+</mo><mfrac><mrow><mi>λ</mi></mrow><mrow><mn>2</mn></mrow></mfrac><msubsup><mrow><mo>∑</mo></mrow><mrow><mi>α</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>m</mi></mrow></msubsup><msubsup><mrow><mo>∑</mo></mrow><mrow><mi>β</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>m</mi></mrow></msubsup><msubsup><mrow><mi mathvariant="bold">c</mi></mrow><mrow><mi>α</mi></mrow><mrow><mo>⊤</mo></mrow></msubsup><mi>Γ</mi><mrow><mo>(</mo><mrow><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold">x</mi></mrow><mo>~</mo></mover></mrow><mrow><mi>α</mi></mrow></msub><mo>,</mo><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold">x</mi></mrow><mo>~</mo></mover></mrow><mrow><mi>β</mi></mrow></msub></mrow><mo>)</mo></mrow><msub><mrow><mi mathvariant="bold">c</mi></mrow><mrow><mi>β</mi></mrow></msub><mo>,</mo></math></span> where the first sum was over all the data points, and the second term was Tikhonov regularization weighted by <em>λ</em>. The superscript <em>T</em> means matrix transpose. One can also use neural networks, e.g., variational autoencoders, to learn an optimal set of basis functions, and other algorithms such as neural ODE to learn <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><math id="d33e3865"><mi mathvariant="bold">B</mi><mfenced close=")" open="("><mrow><mi mathvariant="bold">x</mi></mrow></mfenced></math></span>. The difference is merely algorithmic under the same framework of dynamical systems theories.</p></section><section id="Sec20"><h4 class="pmc_sec_title">Jacobian analyses and reconstruction of effective dose-response curves of gene regulation</h4>
<p id="Par56">The extended dynamo vector field generally contains a nonlinear relationship about regulation between genes, and between genes and other modalities (e.g., chromatin open/close conformations). Several posterior interpretation methods exist to analyze the vector field. Below, we will describe two of them.</p>
<p id="Par57">With the analytical form of <strong>F</strong>(x), one can calculate efficiently the Jacobian field <strong>J</strong> at any cell state <strong>x</strong>. Each element of <strong>J,</strong>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><math id="d33e3897"><mrow><mo>(</mo><mrow><msub><mrow><mi>J</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mi>∂</mi><msub><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>/</mo><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow><mo>)</mo></mrow></math></span>, can be understood as an in-silico perturbation experiment on how upregulating gene <em>j</em> affects the transcription rate of gene <em>i</em>, with all other gene expression levels kept constant at state <strong>x</strong>. For example, a positive value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><math id="d33e3939"><msub><mrow><mi>J</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></span> indicates that at <strong>x</strong> further increasing the expression of gene <em>j</em> causes increase of the transcription rate of gene <em>i</em>. Note that the sign and value of a Jacobian element alone does not unambiguously reflect the nature of the regulation. A close-to-zero Jacobian element can be associated with either no regulation or the regulator is at a saturating concentration of regulation. The regulation relation can be direct, or indirect through intermediate molecular species not implicitly treated as variables of the vector field function.</p>
<p id="Par58">Complementary to the local Jacobian analysis is to reconstruct effective dose-response (D-R) curves of a regulator-target gene pair. The curve reveals the rate of change of one quantity, e.g., the transcription rate or the chromatin open/close dynamics of the target gene, as a function of the value of a regulator, e.g., the mRNA level of a regulator or the chromatin open/close status of a specific genomic region. Note that the D-R curve is generally a multi-variate function, we designed a procedure to reconstruct an effective one-variate function<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. One can genetically write the regulation on quantity <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><math id="d33e3969"><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> as two terms with and without dependence on variable <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><math id="d33e3982"><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></math></span>,</p>
<table class="disp-formula p" id="Equf"><tr><td class="formula"><math id="d33e3995" display="block"><mfrac><mrow><mi>d</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><msub><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub><mfenced close=")" open="("><mrow><msub><mrow><mi>x</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mrow><mi>x</mi></mrow><mrow><mi>N</mi></mrow></msub></mrow></mfenced><mo>=</mo><msubsup><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow><mrow><mn>1</mn></mrow></msubsup><mfenced close=")" open="("><mrow><msub><mrow><mi>x</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mrow><mi>x</mi></mrow><mrow><mi>N</mi></mrow></msub></mrow></mfenced><mo>+</mo><msubsup><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow><mrow><mn>2</mn></mrow></msubsup><mfenced close=")" open="("><mrow><msub><mrow><mi>x</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi><mo>−</mo><mn>1</mn></mrow></msub><mo>,</mo><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi><mo>+</mo><mn>1</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mrow><mi>x</mi></mrow><mrow><mi>N</mi></mrow></msub></mrow></mfenced><mo>.</mo></math></td></tr></table>
<p id="Par59">Notice that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><math id="d33e4101"><msubsup><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow><mrow><mn>2</mn></mrow></msubsup></math></span> is not a function of gene <em>j</em>. First, we calculated the Jacobian element <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><math id="d33e4119"><mfrac><mrow><mi>∂</mi><msub><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfrac></math></span> for each measured cell state. Note <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><math id="d33e4142"><mfrac><mrow><mi>∂</mi><msub><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfrac><mo>=</mo><mfrac><mrow><mi>∂</mi><msubsup><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow><mrow><mn>1</mn></mrow></msubsup></mrow><mrow><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfrac></math></span>, so the background variation from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq75"><math id="d33e4183"><msubsup><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow><mrow><mn>2</mn></mrow></msubsup></math></span> due to effects of other genes has been numerically removed. Then from the histogram of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq76"><math id="d33e4199"><mfrac><mrow><mi>∂</mi><msub><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfrac></math></span> versus <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq77"><math id="d33e4222"><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></math></span>, we binned <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq78"><math id="d33e4235"><mfrac><mrow><mi>∂</mi><msub><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfrac></math></span> over <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq79"><math id="d33e4258"><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></math></span> and calculated <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq80"><math id="d33e4271"><mo>&lt;</mo><mfrac><mrow><mi>∂</mi><msub><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfrac><msub><mrow><mo>&gt;</mo></mrow><mrow><mi>α</mi></mrow></msub></math></span>, which was averaged over all data points of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq81"><math id="d33e4300"><mfrac><mrow><mi>∂</mi><msub><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfrac></math></span> within bin <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq82"><math id="d33e4324"><mi>α</mi></math></span>. Next, we performed numerical integration to obtain <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq83"><math id="d33e4333"><mover accent="true"><mrow><msubsup><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow><mrow><mn>1</mn></mrow></msubsup></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>=</mo><msubsup><mrow><mi>F</mi></mrow><mrow><mi>i</mi><mn>0</mn></mrow><mrow><mn>1</mn></mrow></msubsup><mo>+</mo><msubsup><mrow><mo>∫</mo></mrow><mrow><mn>0</mn></mrow><mrow><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></msubsup><mo>&lt;</mo><mfrac><mrow><mi>∂</mi><msub><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfrac><msub><mrow><mo>&gt;</mo></mrow><mrow><mi>α</mi></mrow></msub><mi>d</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>≈</mo><msubsup><mrow><mi>F</mi></mrow><mrow><mi>i</mi><mn>0</mn></mrow><mrow><mn>1</mn></mrow></msubsup><mo>+</mo><msub><mrow><mo>∑</mo></mrow><mrow><mi>α</mi></mrow></msub><mo>&lt;</mo><mfrac><mrow><mi>∂</mi><msub><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfrac><msub><mrow><mo>&gt;</mo></mrow><mrow><mi>α</mi></mrow></msub><mi mathvariant="normal">Δ</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi><mi>α</mi></mrow></msub></math></span>. In practice, if <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq84"><math id="d33e4453"><mfrac><mrow><mi>∂</mi><msub><mrow><mi>F</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><mi>∂</mi><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfrac></math></span> shows large variance within each bin of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq85"><math id="d33e4476"><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></math></span>, it may imply that other factors affect the D-R curve. For example, the regulation of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq86"><math id="d33e4489"><msub><mrow><mi>x</mi></mrow><mrow><mi>j</mi></mrow></msub></math></span> on <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq87"><math id="d33e4502"><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> may even be opposite at the presence or absence of a specific cofactor. In this case, one should first cluster cells, e.g., grouping cells based on whether an identified cofactor reaches a threshold value, then perform the D-R curve reconstruction on individual clusters.</p></section><section id="Sec21"><h4 class="pmc_sec_title">Markovian analyses</h4>
<p id="Par60">Marius et al.<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> have developed a framework named CellRank to study cellular dynamics based on a Markov chain formulation. We use CellRank to identify cell state transitions using a velocity kernel and identify terminal states within datasets by the GPCCA function module. In addition, CellRank pseudotime kernel<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> is used for methods comparison in real datasets.</p></section><section id="Sec22"><h4 class="pmc_sec_title">Analytical function form of a vector field and differential geometry analyses</h4>
<p id="Par61">Dynamo learns a nonlinear function form of RNA velocity vector field, providing a physics-informed framework that integrates mechanism modeling and single-cell data analyses. We use dynamo to learn continuous vector field functions and perform differential geometry analyses such as gene acceleration, vector field-based pseudotime, least action path (LAP), Jacobian analyses and in silico perturbation.</p></section><section id="Sec23"><h4 class="pmc_sec_title">Pseudotemporal orderings</h4>
<p id="Par62">GraphVelo itself does not compute an ordering index of cells as we are seeking for a more quantitative method to infer RNA velocity. With an accurate RNA velocity as input, we can approximate the vector field precisely. Thus, we use the scalar potential estimated from the functional form vector field with Hodge decomposition as a proxy of time, which is implemented by dynamo package.</p></section><section id="Sec24"><h4 class="pmc_sec_title">Cross boundary correctness scoring</h4>
<p id="Par63">While the GraphVelo framework is designed to quantify velocity vectors across different representations, known transitions between coarse cell states—such as cell types or cell cycle phases—can be used to evaluate the correctness of velocity directions. Suppose there are two cell populations A and B, with A a progenitor state of B. One can define the set of boundary cells between A and B as</p>
<table class="disp-formula p" id="Equg"><tr><td class="formula"><math id="d33e4539" display="block"><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">C</mi></mrow><mrow><mi mathvariant="normal">A</mi><mo>→</mo><mi mathvariant="normal">B</mi></mrow></msub><mo>=</mo><mfenced close="}" open="{"><mrow><mi>c</mi><mo>∈</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">C</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msub><mo>∣</mo><mo>∃</mo><msup><mrow><mi>c</mi></mrow><mrow><mo>′</mo></mrow></msup><mo>∈</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">C</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msub><mo>∩</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi></mrow><mrow><mi>c</mi></mrow></msub></mrow></mfenced><mo>,</mo></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq88"><math id="d33e4592"><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">C</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msub></math></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq89"><math id="d33e4607"><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">C</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msub></math></span> denote the sets of cells in state A or state B, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq90"><math id="d33e4622"><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi></mrow><mrow><mi>c</mi></mrow></msub></math></span> indicates the kNN of cell <em>c</em>. The CBC score is then defined as</p>
<table class="disp-formula p" id="Equh"><tr><td class="formula"><math id="d33e4640" display="block"><mi mathvariant="normal">CBC</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">score</mi><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi>n</mi></mrow></mfrac><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>c</mi></mrow></msub><mfrac><mrow><mn>1</mn></mrow><mrow><mi mathvariant="normal">#</mi><msup><mrow><mi>c</mi></mrow><mrow><mo>′</mo></mrow></msup><mo>∈</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">C</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msub><mo>∩</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi></mrow><mrow><mi>c</mi></mrow></msub></mrow></mfrac><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><msup><mrow><mi>c</mi></mrow><mrow><mo>′</mo></mrow></msup><mo>∈</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">C</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msub><mo>∩</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi></mrow><mrow><mi>c</mi></mrow></msub></mrow></msub><mfrac><mrow><msub><mrow><mi mathvariant="bold-italic">v</mi></mrow><mrow><mi>c</mi></mrow></msub><mo>⋅</mo><mfenced close=")" open="("><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msup><mrow><mi>c</mi></mrow><mrow><mo>′</mo></mrow></msup></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>c</mi></mrow></msub></mrow></mfenced></mrow><mrow><mfenced close="∣" open="∣"><mrow><msub><mrow><mi mathvariant="bold-italic">v</mi></mrow><mrow><mi>c</mi></mrow></msub></mrow></mfenced><mo>⋅</mo><mfenced close="∣" open="∣"><mrow><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><msup><mrow><mi>c</mi></mrow><mrow><mo>′</mo></mrow></msup></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="bold">x</mi></mrow><mrow><mi>c</mi></mrow></msub></mrow></mfenced></mrow></mfrac></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq91"><math id="d33e4773"><msup><mrow><mi>#</mi><mi>c</mi></mrow><mrow><mo>′</mo></mrow></msup><mo>∈</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">C</mi></mrow><mrow><mi mathvariant="normal">B</mi></mrow></msub><mo>∩</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi></mrow><mrow><mi>c</mi></mrow></msub></math></span> is the number of cells in state B which is also the kNN of cell <em>c</em>. While the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq92"><math id="d33e4805"><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold-italic">x</mi></mrow><mrow><mi>c</mi></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold-italic">v</mi></mrow><mrow><mi>c</mi></mrow></msub></mrow><mo>)</mo></mrow></math></span> can be represented in different basis (raw count, PCA, or UMAP), we computed the CBC score in the original count space to ensure that all genes contribute to the velocity estimation. We deliberately avoided using 2D embeddings like UMAP for this purpose, as such visualizations may distort the true geometric relationships in the high-dimensional space and could lead to misleading interpretations.</p></section><section id="Sec25"><h4 class="pmc_sec_title">Velocity consistency scoring</h4>
<p id="Par64">For most of the cases, we expect the inferred RNA velocity vectors to be coherent in a uni-directional vector field. To quantify the local consistency of the velocity flow for each cell, we calculate the velocity consistency score<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> for each cell <em>i</em> as the mean correlation of its velocity <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq93"><math id="d33e4841"><msub><mrow><mi>v</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> with velocities from neighboring cells,</p>
<table class="disp-formula p" id="Equi"><tr><td class="formula"><math id="d33e4854" display="block"><msub><mrow><mi>c</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mfenced close="∣" open="∣"><mrow><mfenced close="}" open="{"><mrow><mi mathvariant="bold-italic">{</mi><mi>j</mi><mspace width="0.25em"></mspace><mo>∈</mo><mspace width="0.25em"></mspace><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi></mrow><mrow><mi>i</mi></mrow></msub><mi mathvariant="bold-italic">}</mi></mrow></mfenced></mrow></mfenced></mrow></mfrac><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>j</mi><mo>∈</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">N</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub><mi>cos</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>v</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi>v</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced></math></td></tr></table>
<p>where cell <em>j</em> is the neighbor of cell <em>i</em> and cos indicates the cosine similarity. One thing should be clarified is that overly smooth and homogenized velocity fields may obscure biologically meaningful heterogeneity.</p></section><section id="Sec26"><h4 class="pmc_sec_title">Approximate smooth velocity trends with a generalized addictive model</h4>
<p id="Par65">The variation of transcription rates contains the high-order dynamic information of the cell system. To model the dynamic patterns of RNA velocity along the transition path from noisy data, we refine the velocity vectors by local geometry via TSP and further fit GAM to velocity value of each gene that has been refined by GraphVelo. For any gene g, we model the velocity trend for the temporal variable <em>t</em> via</p>
<table class="disp-formula p" id="Equj"><tr><td class="formula"><math id="d33e4931" display="block"><msub><mrow><mi>v</mi></mrow><mrow><mi>g</mi><mi>i</mi></mrow></msub><mo>=</mo><msub><mrow><mi>β</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>f</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>t</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></mfenced></math></td></tr></table>
<p>Where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq94"><math id="d33e4960"><msub><mrow><mi>v</mi></mrow><mrow><mi>g</mi><mi>i</mi></mrow></msub></math></span> indicates the velocity of gene g in cell <em>i</em>, <em>f</em> is built using penalized B-splines which allow us to automatically model non-linear mapping while maintaining additivity<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. To visualize the velocity trends, we select 100 equally spaced testing points along the transition path and predict gene expression at each of them using the fitted model. The estimated velocity trends can be treated as smoothed time series for further analyses.</p></section><section id="Sec27"><h4 class="pmc_sec_title">Clustering velocity trends along infection trajectory</h4>
<p id="Par66">With manifold-constrained velocity estimated by GraphVelo, we are able to cluster genes into different functional modules that are involved in the same regulatory circuit. We recover transcription variation of both host and virus factors along the infection trajectory by fitting GAMs in the temporal indicator, the percentage of viral RNA. Next, we select 100 equally spaced time points and generate the GAM-smoothed velocity trends. We compute a kNN graph and cluster the kNN graph using the Leiden algorithm. We used <em>k</em> = 15 for the velocity-trend kNN graph and the Leiden algorithm with a resolution parameter set to 0.3 to avoid over-clustering the trends. For each recovered cluster, we compute its mean and standard deviation (pointwise, for all generated points that were used for smoothing) and visualize the smoothed trends per cluster.</p></section><section id="Sec28"><h4 class="pmc_sec_title">Characterizing decoupling genes based on the dynamic time warping distance between multi-modality velocities</h4>
<p id="Par67">We perform the DTW distance calculation by dtaidistance package. To eliminate the influence of scale in different modality, we maximum-normalize the chromatin/RNA velocity to the same range of [0, 1]. Then we fit the velocity trends of both modalities along vector field-based pseudotime to yield the smoothed velocity trends. We calculate the DTW distance between velocity trends per gene. To distinguish the decoupling genes based on their multi-modality velocities, we rank the genes based on the DTW distance and identify the elbow point as a cutoff. Genes with a distance metric larger than the cutoff are characterized as decoupling genes and used for visualization and functional analysis.</p></section></section><section id="Sec29"><h3 class="pmc_sec_title">Synthetic datasets</h3>
<section id="Sec30"><h4 class="pmc_sec_title">Generating two genes bifurcation process and mapping it to 3d</h4>
<p id="Par68">The bifurcation data (<em>n</em> = 2000 cells) for the toggle-switch system is simulated using the Gillespie algorithm. We use activation and inhibition Hill functions to model the induction and suppression effects between the two genes:</p>
<table class="disp-formula p" id="Equk"><tr><td class="formula"><math id="d33e5005" display="block"><mover accent="true"><mrow><mi>x</mi></mrow><mrow><mo>°</mo></mrow></mover><mo>=</mo><mfrac><mrow><msub><mrow><mi>a</mi></mrow><mrow><mn>1</mn></mrow></msub><msup><mrow><mi>x</mi></mrow><mrow><mi>n</mi></mrow></msup></mrow><mrow><msubsup><mrow><mi>S</mi></mrow><mrow><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mo>+</mo><msup><mrow><mi>x</mi></mrow><mrow><mi>n</mi></mrow></msup></mrow></mfrac><mo>+</mo><mfrac><mrow><msub><mrow><mi>b</mi></mrow><mrow><mn>1</mn></mrow></msub><msubsup><mrow><mi>K</mi></mrow><mrow><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup></mrow><mrow><msubsup><mrow><mi>K</mi></mrow><mrow><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mo>+</mo><msup><mrow><mi>y</mi></mrow><mrow><mi>n</mi></mrow></msup></mrow></mfrac><mo>−</mo><msub><mrow><mi>γ</mi></mrow><mrow><mn>1</mn></mrow></msub><mi>x</mi></math></td></tr></table>
<table class="disp-formula p" id="Equl"><tr><td class="formula"><math id="d33e5080" display="block"><mover accent="true"><mrow><mi>y</mi></mrow><mrow><mo>°</mo></mrow></mover><mo>=</mo><mfrac><mrow><msub><mrow><mi>a</mi></mrow><mrow><mn>2</mn></mrow></msub><msup><mrow><mi>y</mi></mrow><mrow><mi>n</mi></mrow></msup></mrow><mrow><msubsup><mrow><mi>S</mi></mrow><mrow><mn>2</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mo>+</mo><msup><mrow><mi>y</mi></mrow><mrow><mi>n</mi></mrow></msup></mrow></mfrac><mo>+</mo><mfrac><mrow><msub><mrow><mi>b</mi></mrow><mrow><mn>2</mn></mrow></msub><msubsup><mrow><mi>K</mi></mrow><mrow><mn>2</mn></mrow><mrow><mi>n</mi></mrow></msubsup></mrow><mrow><msubsup><mrow><mi>K</mi></mrow><mrow><mn>2</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mo>+</mo><msup><mrow><mi>x</mi></mrow><mrow><mi>n</mi></mrow></msup></mrow></mfrac><mo>−</mo><msub><mrow><mi>γ</mi></mrow><mrow><mn>2</mn></mrow></msub><mi>y</mi></math></td></tr></table>
<p id="Par69">We use the simulation backend implemented by dynamo with default parameters except the timescale (reset <em>τ</em> = 1) to generate the bifurcating process. We then map the synthetic dataset onto a sphere (radius <em>r</em> = 70) and yield the variable <em>z</em> as:</p>
<table class="disp-formula p" id="Equm"><tr><td class="formula"><math id="d33e5166" display="block"><mi>z</mi><mo>=</mo><msqrt><mrow><mi>max</mi><mfenced close=")" open="("><mrow><msup><mrow><mi>r</mi></mrow><mrow><mn>2</mn></mrow></msup><mo>−</mo><msup><mrow><mi>x</mi></mrow><mrow><mn>2</mn></mrow></msup><mo>−</mo><msup><mrow><mi>y</mi></mrow><mrow><mn>2</mn></mrow></msup><mo>,</mo><mn>0</mn></mrow></mfenced></mrow></msqrt></math></td></tr></table>
<p id="Par70">Then we are able to calculate the correctly-scaled 3d vectors by infinitesimal propagation operator with sufficient small step size (<em>dt</em> = 1 in our case):</p>
<table class="disp-formula p" id="Equn"><tr><td class="formula"><math id="d33e5204" display="block"><mover accent="true"><mrow><mi>z</mi></mrow><mrow><mo>°</mo></mrow></mover><mo>=</mo><msub><mrow><mi mathvariant="normal">lim</mi></mrow><mrow><mi mathvariant="normal">Δ</mi><mi>t</mi><mo>→</mo><mn>0</mn></mrow></msub><mfrac><mrow><msqrt><mrow><mi>max</mi><mfenced close=")" open="("><mrow><msup><mrow><mi>r</mi></mrow><mrow><mn>2</mn></mrow></msup><mo>−</mo><msup><mrow><mfenced close=")" open="("><mrow><mi>x</mi><mo>+</mo><mover accent="true"><mrow><mi>x</mi></mrow><mrow><mo>°</mo></mrow></mover><mi mathvariant="normal">Δ</mi><mi>t</mi></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup><mo>−</mo><msup><mrow><mfenced close=")" open="("><mrow><mi>y</mi><mo>+</mo><mover accent="true"><mrow><mi>y</mi></mrow><mrow><mo>°</mo></mrow></mover><mi mathvariant="normal">Δ</mi><mi>t</mi></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup><mo>,</mo><mn>0</mn></mrow></mfenced></mrow></msqrt><mo>−</mo><mi>z</mi></mrow><mrow><mi mathvariant="normal">Δ</mi><mi>t</mi></mrow></mfrac></math></td></tr></table></section><section id="Sec31"><h4 class="pmc_sec_title">Generating scRNA-seq synthetic data with dyngen.\</h4>
<p id="Par71">To generate high-dimensional single-cell transcriptomic data in silico, we use a multi-modal simulation engine, dyngen, to account for different developmental topologies. We constructe module networks to represent regulatory cascades and feedback loops driving progressive changes in gene expression and influencing cell fate decisions. We generate three datasets with 1000 cells and 100 genes using the linear, cyclic, and bifurcating loop backbones provided by dyngen, with all other parameters set to default values. These datasets include simulated nascent and mature mRNA counts along with ground-truth RNA velocities and known manifold structure.</p></section><section id="Sec32"><h4 class="pmc_sec_title">Simulation of rapid degradation and transcription burst events on phase portrait</h4>
<p id="Par72">As for genes with variable degradation rate, we present a minimal regulatory network with linear model in which an external signal both inhibits transcription and promotes microRNA (miRNA). The miRNA exerts a linear influence on the degradation rate of mRNA. We have miRNA’s velocity as</p>
<table class="disp-formula p" id="Equo"><tr><td class="formula"><math id="d33e5289" display="block"><mfrac><mrow><mi>d</mi><mi>m</mi></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><msub><mrow><mi mathvariant="normal">α</mi></mrow><mrow><mi>m</mi></mrow></msub><mo>−</mo><msub><mrow><mi mathvariant="normal">γ</mi></mrow><mrow><mi>m</mi></mrow></msub><mi>m</mi></math></td></tr></table>
<p id="Par73">The nascent gene transcription rate and mature mRNA’s degradation rate would change to</p>
<table class="disp-formula p" id="Equp"><tr><td class="formula"><math id="d33e5320" display="block"><mi>α</mi><mo>=</mo><msub><mrow><mi mathvariant="normal">α</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">α</mi></mrow></msub><mi>t</mi></math></td></tr></table>
<table class="disp-formula p" id="Equq"><tr><td class="formula"><math id="d33e5343" display="block"><mi mathvariant="normal">γ</mi><mo>=</mo><msub><mrow><mi mathvariant="normal">γ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">γ</mi></mrow></msub><mi>m</mi></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq95"><math id="d33e5368"><msub><mrow><mi mathvariant="normal">α</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq96"><math id="d33e5382"><msub><mrow><mi mathvariant="normal">γ</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> represent the constant transcription rate and degradation rate without the effect of miRNA, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq97"><math id="d33e5396"><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">α</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq98"><math id="d33e5410"><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">γ</mi></mrow></msub></math></span> represent the magnitude of influence from miRNA, <em>t</em> indicates the simulation time to mimic the cell-context change along trajectory.</p>
<table class="disp-formula p" id="Equr"><tr><td class="formula"><math id="d33e5428" display="block"><mfrac><mrow><mi>d</mi><mi>u</mi></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mi>α</mi><mo>−</mo><mi>β</mi><mi>u</mi><mo>=</mo><msub><mrow><mi>α</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><msub><mrow><mi>k</mi></mrow><mrow><mi>α</mi></mrow></msub><mi>m</mi><mo>−</mo><mi>β</mi><mi>u</mi></math></td></tr></table>
<table class="disp-formula p" id="Equs"><tr><td class="formula"><math id="d33e5464" display="block"><mfrac><mrow><mi>d</mi><mi>s</mi></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mi>β</mi><mi>u</mi><mo>−</mo><mi>γ</mi><mi>s</mi><mo>=</mo><mi>β</mi><mi>u</mi><mo>−</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>γ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>k</mi></mrow><mrow><mi>γ</mi></mrow></msub><mi>m</mi></mrow></mfenced><mi>s</mi></math></td></tr></table>
<p>We set the initial condition to a steady state with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq99"><math id="d33e5505"><msub><mrow><mi>u</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>=</mo><msub><mrow><mi>α</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>/</mo><mi>β</mi></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq100"><math id="d33e5526"><msub><mrow><mi>s</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>=</mo><msub><mrow><mi>α</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>/</mo><msub><mrow><mi>γ</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span>, while the miRNA abundance <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq101"><math id="d33e5551"><msub><mrow><mi>m</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>=</mo><mn>0</mn></math></span>. We simulate <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq102"><math id="d33e5567"><msub><mrow><mi>u</mi></mrow><mrow><mi>t</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq103"><math id="d33e5580"><msub><mrow><mi>s</mi></mrow><mrow><mi>t</mi></mrow></msub></math></span> as the microRNA signal <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq104"><math id="d33e5593"><msub><mrow><mi>m</mi></mrow><mrow><mi>t</mi></mrow></msub></math></span> gradually increases. The aim is to evaluate whether the estimated RNA velocity consistently aligns in sign with the ground truth RNA velocity.</p>
<p id="Par74">To generate genes with transcription burst phase portrait, we set the initial condition to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq105"><math id="d33e5608"><msub><mrow><mi>u</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>=</mo><mn>0</mn><mo>,</mo><msub><mrow><mi>s</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>=</mo><mn>0</mn></math></span>, together with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq106"><math id="d33e5631"><mi mathvariant="normal">γ</mi></math></span> as constant and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq107"><math id="d33e5641"><mi>α</mi></math></span> promotes to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq108"><math id="d33e5650"><msup><mrow><mi>α</mi></mrow><mrow><mo>′</mo></mrow></msup><mo>=</mo><mn>3</mn><mi>α</mi></math></span> when the simulation reaches specific time.</p></section></section><section id="Sec33"><h3 class="pmc_sec_title">Processing of sequencing data</h3>
<p id="Par75">All sequencing data in this study are downloaded publicly (see details in the ‘Data availability’ section). Though the number of confident-estimated gene sets differed case by case, GraphVelo predicted the velocity of all highly variable genes and used them for downstream calculations such as CBC score, CellRank velocity kernel, and dynamo vector field learning. We parallelized the TSP optimization to scale efficiently and set the hyperparameters in loss with <em>a</em> = 1, <em>b</em> = 10 and <em>λ</em> = 1 as default in all studies (Supplementary Fig. <a href="#MOESM1" class="usa-link">20</a>).</p>
<section id="Sec34"><h4 class="pmc_sec_title">Analyses of scRNA-seq datasets</h4>
<p id="Par76">For the erythroid lineage of the mouse gastrulation, we follow the standard data pre-processing procedures implemented by scVelo and select 9815 cells and 2000 highly variable genes to construct the kNN graph using 30 nearest neighbors for downstream calculation.</p>
<p id="Par77">For the human bone marrow dataset, we follow the standard data pre-processing procedures implemented by scVelo and selected 5780 cells and 2000 highly variable genes to construct the <em>k</em>-nearest neighbor graph using 30 nearest neighbors for downstream calculation. To estimate the variation of degradation rate <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq109"><math id="d33e5691"><mi>γ</mi></math></span> along the differentiation lineages, we divide the cells into five discrete time bins based on precomputed Palantir pseudotime. We then estimate cell-specific degradation rates and visualized their distribution shifts along the hematopoiesis trajectory.</p></section><section id="Sec35"><h4 class="pmc_sec_title">Analyses of spatial transcriptomics dataset</h4>
<p id="Par78">For the mouse coronal hemibrain spatial dataset, we follow the monocle preprocessing pipeline implemented in dynamo and select 7,765 cells and 2000 highly variable genes. To include spatial information during manifold, we build a spatial kNN graph with <em>k</em> = 8 and then take the union with the kNN graph based on transcriptomic data. The combined graph is used for downstream analyses.</p></section><section id="Sec36"><h4 class="pmc_sec_title">Analyses of HCMV dataset</h4>
<p id="Par79">We sample the cells from donor 1 to eliminate sample-specific variation and further filter out cells lacking immediate early <em>UL123</em> gene expression to focus on the viral infection trajectory. We adapt the monocle preprocessing recipe implemented by dynamo and yield 1454 cells and 2000 highly variable genes for further analyses. The number of nearest neighbors were set to 30 as default. Then 1022 velocity genes are used for downstream analyses.</p>
<p id="Par80">We collect the pathway-related genes from MSigDB and perform the Jacobian analyses implemented by dynamo, using viral genes as regulators and host genes as effectors. We rank the regulation relationships based on the collections of Jacobian elements. We pick the top 50 inhibited effectors of viral genome and select the common set between pathway genes and all the effectors for visualization.</p>
<p id="Par81">In silico knock-out experiments are performed via Dynamo. We suppressed every single virus factors per a time using <em>dynamo.pd.perturbation</em> function and calculated the change of total viral gene expression after perturbation.</p></section><section id="Sec37"><h4 class="pmc_sec_title">Analyses of SARS-CoV−2 dataset</h4>
<p id="Par82">We subsample the cells based on their infection status. We adapt the monocle preprocessing pipeline implemented in dynamo and exclude the UMI reads from lenti-virus. We obtain 5001 infected cells and 1000 highly variable genes for downstream analyses. We use both highly variable genes in host and virus genes to perform PCA and calculate the nearest neighbors with <em>k</em> = 30. Then 269 velocity genes are used for downstream analyses. We use the apoptosis-related genes collected by KEGG to calculate the apoptosis activity score of each cell using <em>dynamo.tl.score_cells</em> function.</p></section></section><section id="Sec38"><h3 class="pmc_sec_title">Analyses of multi-modality datasets</h3>
<section id="Sec39"><h4 class="pmc_sec_title">RNA velocity estimation for multi-modality datasets</h4>
<p id="Par83">In traditional scRNA-seq datasets, RNA velocity methods use smoothed spliced and unspliced RNA counts through nearest-neighbor pooling, based on the PCA space computed from transcripts alone. However, this approach is not suitable for multimodal scenarios, as it overlooks hidden variables by relying on a single modality. To construct a consistent manifold combining information from multi-modal genomics data, we utilized WNN as implemented in MultiVelo<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. The WNN algorithm combines low-dimensional representations from RNA and ATAC omics data. Specifically, we use PCA results from scRNA-seq data and latent semantic indexing (LSI) from scATAC-seq as inputs. The nearest neighbors identified by WNN were then used to calculate the first moment, reducing noise in separate modalities and approximating a unified manifold for GraphVelo.</p></section><section id="Sec40"><h4 class="pmc_sec_title">Mouse skin dataset preprocessing</h4>
<p id="Par84">The preprocessed SHARE-seq mouse skin dataset<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> is adopted directly from MultiVelo data resources. All the procedures are consistent with MultiVelo, except we get the LSI representation processed by SCARlink<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. We construct the WNN graph using 50 nearest neighbors for downstream calculation. We run scVelo with ‘stochastic’ mode to estimate the RNA velocity based on the WNN graph as we discussed above.</p></section><section id="Sec41"><h4 class="pmc_sec_title">Human cortex dataset preprocessing</h4>
<p id="Par85">The preprocessed human cerebral cortex data is adopted directly from the MultiVelo data resources. All the procedures are consistent with MultiVelo, except we get the LSI representation processed by SCARlink. We construct the WNN graph using 50 nearest neighbors for downstream calculation. We run scVelo with ‘stochastic’ mode to estimate the RNA velocity based on the WNN graph.</p></section></section><section id="Sec42"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par86">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec43"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373899/bin/41467_2025_62784_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (4.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373899/bin/41467_2025_62784_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (76.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12373899/bin/41467_2025_62784_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (2.4MB, pdf) </sup>
</div></div></section></section><section id="Sec44"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12373899/bin/41467_2025_62784_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (4.3MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was partially supported by the National Institute of General Medical Sciences (1R01GM148525 to J.X. and 1R01GM139297 to I.B.), and the National Science Foundation (2325149 to J.X.). This work was supported by the National Key Research and Development Program of China (2023YFE0112300 to M.C.); National Natural Sciences Foundation of China (32261133526; 32270709; 32070677 to M.C.); the 151 talent project of Zhejiang Province (first level to M.C.), the Science and Technology Innovation Leading Scientist (2022R52035 to M.C.).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>J.X. conceived and formulated the theoretical framework, Y.C. and J.G. performed tests on simulated data, Y.C. and J.X. analyzed scRNA-seq datasets, Y.C., Y.Z., J.G., and K.N. developed the package graphvelo, Y.C. and J.X. wrote the draft, M.C. and I.B. revised the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par87"><em>Nature Communications</em> thanks Zheng Hu, Shihua Zhang and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All the sequencing raw data are publicly accessible. The A549 dataset can be accessed via <a href="https://figshare.com/ndownloader/files/53666738" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/ndownloader/files/53666738</a>. The FUCCI cell cycle data can be downloaded from <a href="https://figshare.com/ndownloader/files/53705057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/ndownloader/files/53705057</a>. The processed data used for benchmark are available at <a href="https://figshare.com/ndownloader/files/53667548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/ndownloader/files/53667548</a>. The mouse gastrulation subset to erythroid lineage can be extracted using scVelo’s CLI: <em>scvelo.datasets.gastrulation_erythroid()</em> or from the original work under accession number E-MTAB-6967 of ArrayExpress. The human bone marrow can be extracted using scVelo’s CLI: <em>scvelo.datasets.bonemarrow()</em> or through the Human Cell Atlas data portal at <a href="https://data.humancellatlas.org/explore/projects/091cf39b-01bc-42e5-9437-f419a66c8a45" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://data.humancellatlas.org/explore/projects/091cf39b-01bc-42e5-9437-f419a66c8a45</a>. The mouse coronal hemibrain spatial transcriptomic data can be downloaded from (<a href="https://www.dropbox.com/s/c5tu4drxda01m0u/mousebrain_bin60.h5ad?dl=0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.dropbox.com/s/c5tu4drxda01m0u/mousebrain_bin60.h5ad?dl=0</a>). The original HCMV infected moDC data can be accessed via Zenodo (<a href="https://zenodo.org/records/10404879" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/10404879</a>) and the processed data can be downloaded from <a href="https://figshare.com/ndownloader/files/53666756" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/ndownloader/files/53666756</a>. The SARS-CoV-2 data can be accessed via <a href="https://figshare.com/ndownloader/files/53666588" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/ndownloader/files/53666588</a>. The preprocessed mouse skin development dataset can be accessed via <a href="https://figshare.com/articles/dataset/Mouse_Hair_Follicle_RNA_Data/22575307" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Mouse_Hair_Follicle_RNA_Data/22575307</a> and <a href="https://figshare.com/articles/dataset/Mouse_hair_follicle_ATAC_data/22575313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Mouse_hair_follicle_ATAC_data/22575313</a>. The preprocessed human cortex dataset can be downloaded from <a href="https://figshare.com/articles/dataset/Developing_Human_Cortex_RNA_Data/22575376" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Developing_Human_Cortex_RNA_Data/22575376</a> and <a href="https://figshare.com/articles/dataset/Developing_Human_Cortex_ATAC_Data/22575370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Developing_Human_Cortex_ATAC_Data/22575370</a>. <a href="#Sec44" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>The source code of python package GraphVelo<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup> can be downloaded from <a href="https://github.com/xing-lab-pitt/GraphVelo" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/xing-lab-pitt/GraphVelo</a> and is released under the BSD 3-Clause License. Reproducibility and tutorials can be found in <a href="https://github.com/xing-lab-pitt/GraphVelo/tree/main/notebook" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/xing-lab-pitt/GraphVelo/tree/main/notebook</a> and <a href="https://graphvelo.readthedocs.io/en/latest/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://graphvelo.readthedocs.io/en/latest/</a>. The specific version of the code associated with this publication is archived in Zendo and is accessible via 10.5281/zenodo.15852884.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par88">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Yuhao Chen, Yan Zhang, Jiaqi Gan.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Ming Chen, Email: mchen@zju.edu.cn.</p>
<p>Ivet Bahar, Email: bahar@laufercenter.org.</p>
<p>Jianhua Xing, Email: xing1@pitt.edu.</p></section><section id="sec45"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62784-w.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>La Manno, G. et al. RNA velocity of single cells. <em>Nature</em><strong>560</strong>, 494–498 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0414-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6130801/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30089906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?La%20Manno,%20G.%20et%20al.%20RNA%20velocity%20of%20single%20cells.%20Nature560,%20494%E2%80%93498%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Bergen, V., Lange, M., Peidli, S., Wolf, F. A. &amp; Theis, F. J. Generalizing RNA velocity to transient cell states through dynamical modeling. <em>Nat. Biotechnol.</em><strong>38</strong>, 1408–1414 (2020).
</cite> [<a href="https://doi.org/10.1038/s41587-020-0591-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32747759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bergen,%20V.,%20Lange,%20M.,%20Peidli,%20S.,%20Wolf,%20F.%20A.%20&amp;%20Theis,%20F.%20J.%20Generalizing%20RNA%20velocity%20to%20transient%20cell%20states%20through%20dynamical%20modeling.%20Nat.%20Biotechnol.38,%201408%E2%80%931414%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Li, S. et al. A relay velocity model infers cell-dependent RNA velocity. <em>Nat. Biotechnol.</em><strong>42</strong>, 99–108 (2024).
</cite> [<a href="https://doi.org/10.1038/s41587-023-01728-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10545816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37012448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20S.%20et%20al.%20A%20relay%20velocity%20model%20infers%20cell-dependent%20RNA%20velocity.%20Nat.%20Biotechnol.42,%2099%E2%80%93108%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Lederer, A. R. et al. Statistical inference with a manifold-constrained RNA velocity model uncovers cell cycle speed modulations. <em>Nat. Methods</em><strong>21</strong>, 2271–2286 (2024).</cite> [<a href="https://doi.org/10.1038/s41592-024-02471-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11621032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39482463/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Gayoso, A. et al. Deep generative modeling of transcriptional dynamics for RNA velocity analysis in single cells. <em>Nat. Methods</em><strong>21</strong>, 50–59 (2024).
</cite> [<a href="https://doi.org/10.1038/s41592-023-01994-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10776389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37735568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gayoso,%20A.%20et%20al.%20Deep%20generative%20modeling%20of%20transcriptional%20dynamics%20for%20RNA%20velocity%20analysis%20in%20single%20cells.%20Nat.%20Methods21,%2050%E2%80%9359%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Gao, C. F., Vaikuntanathan, S. &amp; Riesenfeld, S. J. Dissection and integration of bursty transcriptional dynamics for complex systems. <em>Proc. Natl Acad. Sci. USA</em><strong>121</strong>, e2306901121 (2024).
</cite> [<a href="https://doi.org/10.1073/pnas.2306901121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11067469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38669186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20C.%20F.,%20Vaikuntanathan,%20S.%20&amp;%20Riesenfeld,%20S.%20J.%20Dissection%20and%20integration%20of%20bursty%20transcriptional%20dynamics%20for%20complex%20systems.%20Proc.%20Natl%20Acad.%20Sci.%20USA121,%20e2306901121%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Qiu, X. et al. Mapping transcriptomic vector fields of single cells. <em>Cell</em><strong>185</strong>, 690–711 e45 (2022).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.12.045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9332140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35108499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qiu,%20X.%20et%20al.%20Mapping%20transcriptomic%20vector%20fields%20of%20single%20cells.%20Cell185,%20690%E2%80%93711%20e45%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Weiler, P., Lange, M., Klein, M., Pe’er, D. &amp; Theis, F. CellRank 2: unified fate mapping in multiview single-cell data. <em>Nat. Methods</em><strong>21</strong>, 1196–1205 (2024).
</cite> [<a href="https://doi.org/10.1038/s41592-024-02303-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11239496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38871986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weiler,%20P.,%20Lange,%20M.,%20Klein,%20M.,%20Pe%E2%80%99er,%20D.%20&amp;%20Theis,%20F.%20CellRank%202:%20unified%20fate%20mapping%20in%20multiview%20single-cell%20data.%20Nat.%20Methods21,%201196%E2%80%931205%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Wang, K. et al. PhyloVelo enhances transcriptomic velocity field mapping using monotonically expressed genes. <em>Nat. Biotechnol.</em><strong>42</strong>, 778–789 (2024).
</cite> [<a href="https://doi.org/10.1038/s41587-023-01887-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37524958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20K.%20et%20al.%20PhyloVelo%20enhances%20transcriptomic%20velocity%20field%20mapping%20using%20monotonically%20expressed%20genes.%20Nat.%20Biotechnol.42,%20778%E2%80%93789%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Li, J., Pan, X., Yuan, Y. &amp; Shen, H. B. TFvelo: gene regulation inspired RNA velocity estimation. <em>Nat. Commun.</em><strong>15</strong>, 1387 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-45661-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11258302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38360714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20J.,%20Pan,%20X.,%20Yuan,%20Y.%20&amp;%20Shen,%20H.%20B.%20TFvelo:%20gene%20regulation%20inspired%20RNA%20velocity%20estimation.%20Nat.%20Commun.15,%201387%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Bergen, V., Soldatov, R. A., Kharchenko, P. V. &amp; Theis, F. J. RNA velocity-current challenges and future perspectives. <em>Mol. Syst. Biol.</em><strong>17</strong>, e10282 (2021).
</cite> [<a href="https://doi.org/10.15252/msb.202110282" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8388041/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34435732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bergen,%20V.,%20Soldatov,%20R.%20A.,%20Kharchenko,%20P.%20V.%20&amp;%20Theis,%20F.%20J.%20RNA%20velocity-current%20challenges%20and%20future%20perspectives.%20Mol.%20Syst.%20Biol.17,%20e10282%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Gorin, G., Svensson, V. &amp; Pachter, L. Protein velocity and acceleration from single-cell multiomics experiments. <em>Genome Biol.</em><strong>21</strong>, 39 (2020).
</cite> [<a href="https://doi.org/10.1186/s13059-020-1945-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7029606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32070398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gorin,%20G.,%20Svensson,%20V.%20&amp;%20Pachter,%20L.%20Protein%20velocity%20and%20acceleration%20from%20single-cell%20multiomics%20experiments.%20Genome%20Biol.21,%2039%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Li, C., Virgilio, M. C., Collins, K. L. &amp; Welch, J. D. Multi-omic single-cell velocity models epigenome-transcriptome interactions and improves cell fate prediction. <em>Nat. Biotechnol.</em><strong>41</strong>, 387–398 (2023).
</cite> [<a href="https://doi.org/10.1038/s41587-022-01476-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10246490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36229609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20C.,%20Virgilio,%20M.%20C.,%20Collins,%20K.%20L.%20&amp;%20Welch,%20J.%20D.%20Multi-omic%20single-cell%20velocity%20models%20epigenome-transcriptome%20interactions%20and%20improves%20cell%20fate%20prediction.%20Nat.%20Biotechnol.41,%20387%E2%80%93398%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Zheng, S. C., Stein-O’Brien, G., Boukas, L., Goff, L. A. &amp; Hansen, K. D. Pumping the brakes on RNA velocity by understanding and interpreting RNA velocity estimates. <em>Genome Biol.</em><strong>24</strong>, 246 (2023).
</cite> [<a href="https://doi.org/10.1186/s13059-023-03065-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10601342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37885016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20S.%20C.,%20Stein-O%E2%80%99Brien,%20G.,%20Boukas,%20L.,%20Goff,%20L.%20A.%20&amp;%20Hansen,%20K.%20D.%20Pumping%20the%20brakes%20on%20RNA%20velocity%20by%20understanding%20and%20interpreting%20RNA%20velocity%20estimates.%20Genome%20Biol.24,%20246%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Tiejun Li, T. L., Jifan Shi, J. S., Yichong Wu, Y. W. &amp; Peijie Zhou, P. Z. On the Mathematics of RNA Velocity I: Theoretical. <em>Anal. CSIAM Trans. Appl. Math.</em><strong>2</strong>, 1–55 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Tiejun%20Li,%20T.%20L.,%20Jifan%20Shi,%20J.%20S.,%20Yichong%20Wu,%20Y.%20W.%20&amp;%20Peijie%20Zhou,%20P.%20Z.%20On%20the%20Mathematics%20of%20RNA%20Velocity%20I:%20Theoretical.%20Anal.%20CSIAM%20Trans.%20Appl.%20Math.2,%201%E2%80%9355%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Lange, M. et al. CellRank for directed single-cell fate mapping. <em>Nat. Methods</em><strong>19</strong>, 159–170 (2022).
</cite> [<a href="https://doi.org/10.1038/s41592-021-01346-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8828480/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35027767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lange,%20M.%20et%20al.%20CellRank%20for%20directed%20single-cell%20fate%20mapping.%20Nat.%20Methods19,%20159%E2%80%93170%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Roweis, S. T. &amp; Saul, L. K. Nonlinear dimensionality reduction by locally linear embedding. <em>Science</em><strong>290</strong>, 2323–2326 (2000).
</cite> [<a href="https://doi.org/10.1126/science.290.5500.2323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11125150/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roweis,%20S.%20T.%20&amp;%20Saul,%20L.%20K.%20Nonlinear%20dimensionality%20reduction%20by%20locally%20linear%20embedding.%20Science290,%202323%E2%80%932326%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Whitney, H. The self-intersections of a smooth n-Manifold in 2n-Space. <em>Ann. Math.</em><strong>45</strong>, 220–246 (1944).</cite> [<a href="https://scholar.google.com/scholar_lookup?Whitney,%20H.%20The%20self-intersections%20of%20a%20smooth%20n-Manifold%20in%202n-Space.%20Ann.%20Math.45,%20220%E2%80%93246%20(1944)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Cannoodt, R., Saelens, W., Deconinck, L. &amp; Saeys, Y. Spearheading future omics analyses using dyngen, a multi-modal simulator of single cells. <em>Nat. Commun.</em><strong>12</strong>, 3942 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-24152-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8225657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34168133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cannoodt,%20R.,%20Saelens,%20W.,%20Deconinck,%20L.%20&amp;%20Saeys,%20Y.%20Spearheading%20future%20omics%20analyses%20using%20dyngen,%20a%20multi-modal%20simulator%20of%20single%20cells.%20Nat.%20Commun.12,%203942%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Mahdessian, D. et al. Spatiotemporal dissection of the cell cycle with single-cell proteogenomics. <em>Nature</em><strong>590</strong>, 649–654 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03232-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33627808/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mahdessian,%20D.%20et%20al.%20Spatiotemporal%20dissection%20of%20the%20cell%20cycle%20with%20single-cell%20proteogenomics.%20Nature590,%20649%E2%80%93654%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Bastidas-Ponce, A. et al. Comprehensive single cell mRNA profiling reveals a detailed roadmap for pancreatic endocrinogenesis. <em>Development</em><strong>146</strong>, dev173849 (2019).</cite> [<a href="https://doi.org/10.1242/dev.173849" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31160421/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Battich, N. et al. Sequencing metabolically labeled transcripts in single cells reveals mRNA turnover strategies. <em>Science</em><strong>367</strong>, 1151–1156 (2020).
</cite> [<a href="https://doi.org/10.1126/science.aax3072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32139547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Battich,%20N.%20et%20al.%20Sequencing%20metabolically%20labeled%20transcripts%20in%20single%20cells%20reveals%20mRNA%20turnover%20strategies.%20Science367,%201151%E2%80%931156%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Gao, M., Qiao, C. &amp; Huang, Y. UniTVelo: temporally unified RNA velocity reinforces single-cell trajectory inference. <em>Nat. Commun.</em><strong>13</strong>, 6586 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-34188-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9633790/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36329018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20M.,%20Qiao,%20C.%20&amp;%20Huang,%20Y.%20UniTVelo:%20temporally%20unified%20RNA%20velocity%20reinforces%20single-cell%20trajectory%20inference.%20Nat.%20Commun.13,%206586%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Cui, H. et al. DeepVelo: deep learning extends RNA velocity to multi-lineage systems with cell-specific kinetics. <em>Genome Biol.</em><strong>25</strong>, 27 (2024).
</cite> [<a href="https://doi.org/10.1186/s13059-023-03148-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10799431/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38243313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cui,%20H.%20et%20al.%20DeepVelo:%20deep%20learning%20extends%20RNA%20velocity%20to%20multi-lineage%20systems%20with%20cell-specific%20kinetics.%20Genome%20Biol.25,%2027%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Qiao, C. &amp; Huang, Y. Representation learning of RNA velocity reveals robust cell transitions. <em>Proc. Natl Acad. Sci. USA</em><strong>118</strong>, e2105859118 (2021).</cite> [<a href="https://doi.org/10.1073/pnas.2105859118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8670433/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34873054/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Cao, J., Zhou, W., Steemers, F., Trapnell, C. &amp; Shendure, J. Sci-fate characterizes the dynamics of gene expression in single cells. <em>Nat. Biotechnol.</em><strong>38</strong>, 980–988 (2020).
</cite> [<a href="https://doi.org/10.1038/s41587-020-0480-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7416490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32284584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cao,%20J.,%20Zhou,%20W.,%20Steemers,%20F.,%20Trapnell,%20C.%20&amp;%20Shendure,%20J.%20Sci-fate%20characterizes%20the%20dynamics%20of%20gene%20expression%20in%20single%20cells.%20Nat.%20Biotechnol.38,%20980%E2%80%93988%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Pijuan-Sala, B. et al. A single-cell molecular map of mouse gastrulation and early organogenesis. <em>Nature</em><strong>566</strong>, 490–495 (2019).
</cite> [<a href="https://doi.org/10.1038/s41586-019-0933-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6522369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30787436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pijuan-Sala,%20B.%20et%20al.%20A%20single-cell%20molecular%20map%20of%20mouse%20gastrulation%20and%20early%20organogenesis.%20Nature566,%20490%E2%80%93495%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Barile, M. et al. Coordinated changes in gene expression kinetics underlie both mouse and human erythroid maturation. <em>Genome Biol.</em><strong>22</strong>, 197 (2021).
</cite> [<a href="https://doi.org/10.1186/s13059-021-02414-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8258993/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34225769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barile,%20M.%20et%20al.%20Coordinated%20changes%20in%20gene%20expression%20kinetics%20underlie%20both%20mouse%20and%20human%20erythroid%20maturation.%20Genome%20Biol.22,%20197%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Lee, J., Krivega, I., Dale, R. K. &amp; Dean, A. The LDB1 Complex Co-opts CTCF for Erythroid lineage-specific long-range enhancer interactions. <em>Cell Rep.</em><strong>19</strong>, 2490–2502 (2017).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.05.072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5564295/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28636938/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20J.,%20Krivega,%20I.,%20Dale,%20R.%20K.%20&amp;%20Dean,%20A.%20The%20LDB1%20Complex%20Co-opts%20CTCF%20for%20Erythroid%20lineage-specific%20long-range%20enhancer%20interactions.%20Cell%20Rep.19,%202490%E2%80%932502%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Dybedal, I., Larsen, S. &amp; Jacobsen, S. E. IL-12 directly enhances in vitro murine erythropoiesis in combination with IL-4 and stem cell factor. <em>J. Immunol.</em><strong>154</strong>, 4950–4955 (1995).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/7537295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dybedal,%20I.,%20Larsen,%20S.%20&amp;%20Jacobsen,%20S.%20E.%20IL-12%20directly%20enhances%20in%20vitro%20murine%20erythropoiesis%20in%20combination%20with%20IL-4%20and%20stem%20cell%20factor.%20J.%20Immunol.154,%204950%E2%80%934955%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Setty, M. et al. Characterization of cell fate probabilities in single-cell data with Palantir. <em>Nat. Biotechnol.</em><strong>37</strong>, 451–460 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0068-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7549125/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30899105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Setty,%20M.%20et%20al.%20Characterization%20of%20cell%20fate%20probabilities%20in%20single-cell%20data%20with%20Palantir.%20Nat.%20Biotechnol.37,%20451%E2%80%93460%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Ning, W. et al. Blocking exosomal miRNA-153-3p derived from bone marrow mesenchymal stem cells ameliorates hypoxia-induced myocardial and microvascular damage by targeting the ANGPT1-mediated VEGF/PI3k/Akt/eNOS pathway. <em>Cell Signal</em><strong>77</strong>, 109812 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cellsig.2020.109812" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33164880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ning,%20W.%20et%20al.%20Blocking%20exosomal%20miRNA-153-3p%20derived%20from%20bone%20marrow%20mesenchymal%20stem%20cells%20ameliorates%20hypoxia-induced%20myocardial%20and%20microvascular%20damage%20by%20targeting%20the%20ANGPT1-mediated%20VEGF/PI3k/Akt/eNOS%20pathway.%20Cell%20Signal77,%20109812%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Qiu, X. et al. Spatiotemporal modeling of molecular holograms. <em>Cell</em><strong>187</strong>, 7351–7373.e61 (2024).
</cite> [<a href="https://doi.org/10.1016/j.cell.2024.10.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39532097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qiu,%20X.%20et%20al.%20Spatiotemporal%20modeling%20of%20molecular%20holograms.%20Cell187,%207351%E2%80%937373.e61%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Yao, Z. et al. A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain. <em>Nature</em><strong>624</strong>, 317–332 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06812-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10719114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38092916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yao,%20Z.%20et%20al.%20A%20high-resolution%20transcriptomic%20and%20spatial%20atlas%20of%20cell%20types%20in%20the%20whole%20mouse%20brain.%20Nature624,%20317%E2%80%93332%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Bost, P. et al. Host-viral infection maps reveal signatures of severe COVID-19 patients. <em>Cell</em><strong>181</strong>, 1475–1488.e12 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7205692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32479746/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bost,%20P.%20et%20al.%20Host-viral%20infection%20maps%20reveal%20signatures%20of%20severe%20COVID-19%20patients.%20Cell181,%201475%E2%80%931488.e12%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Ratnasiri, K., Wilk, A. J., Lee, M. J., Khatri, P. &amp; Blish, C. A. Single-cell RNA-seq methods to interrogate virus-host interactions. <em>Semin Immunopathol.</em><strong>45</strong>, 71–89 (2023).
</cite> [<a href="https://doi.org/10.1007/s00281-022-00972-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9684776/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36414692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ratnasiri,%20K.,%20Wilk,%20A.%20J.,%20Lee,%20M.%20J.,%20Khatri,%20P.%20&amp;%20Blish,%20C.%20A.%20Single-cell%20RNA-seq%20methods%20to%20interrogate%20virus-host%20interactions.%20Semin%20Immunopathol.45,%2071%E2%80%9389%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Wilk, A. J. et al. A single-cell atlas of the peripheral immune response in patients with severe COVID-19. <em>Nat. Med.</em><strong>26</strong>, 1070–1076 (2020).
</cite> [<a href="https://doi.org/10.1038/s41591-020-0944-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7382903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32514174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilk,%20A.%20J.%20et%20al.%20A%20single-cell%20atlas%20of%20the%20peripheral%20immune%20response%20in%20patients%20with%20severe%20COVID-19.%20Nat.%20Med.26,%201070%E2%80%931076%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Edahiro, R. et al. Single-cell analyses and host genetics highlight the role of innate immune cells in COVID-19 severity. <em>Nat. Genet.</em><strong>55</strong>, 753–767 (2023).
</cite> [<a href="https://doi.org/10.1038/s41588-023-01375-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10181941/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37095364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Edahiro,%20R.%20et%20al.%20Single-cell%20analyses%20and%20host%20genetics%20highlight%20the%20role%20of%20innate%20immune%20cells%20in%20COVID-19%20severity.%20Nat.%20Genet.55,%20753%E2%80%93767%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Costa, B. et al. Human cytomegalovirus exploits STING signaling and counteracts IFN/ISG induction to facilitate infection of dendritic cells. <em>Nat. Commun.</em><strong>15</strong>, 1745 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-45614-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10897438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38409141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Costa,%20B.%20et%20al.%20Human%20cytomegalovirus%20exploits%20STING%20signaling%20and%20counteracts%20IFN/ISG%20induction%20to%20facilitate%20infection%20of%20dendritic%20cells.%20Nat.%20Commun.15,%201745%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Hein, M. Y. &amp; Weissman, J. S. Functional single-cell genomics of human cytomegalovirus infection. <em>Nat. Biotechnol.</em><strong>40</strong>, 391–401 (2022).
</cite> [<a href="https://doi.org/10.1038/s41587-021-01059-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34697476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hein,%20M.%20Y.%20&amp;%20Weissman,%20J.%20S.%20Functional%20single-cell%20genomics%20of%20human%20cytomegalovirus%20infection.%20Nat.%20Biotechnol.40,%20391%E2%80%93401%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Alon, U. <em>An introduction to systems biology: design principles of biological circuits</em>, (Chapman &amp; Hall/CRC, 2007).</cite>
</li>
<li id="CR42">
<span class="label">42.</span><cite>Ball, C. B. et al. Human Cytomegalovirus IE2 both activates and represses initiation and modulates elongation in a context-dependent manner. <em>mBio</em><strong>13</strong>, e00337–22 (2022).
</cite> [<a href="https://doi.org/10.1128/mbio.00337-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9239164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35579393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ball,%20C.%20B.%20et%20al.%20Human%20Cytomegalovirus%20IE2%20both%20activates%20and%20represses%20initiation%20and%20modulates%20elongation%20in%20a%20context-dependent%20manner.%20mBio13,%20e00337%E2%80%9322%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Manska, S. &amp; Rossetto, C. C. Identification of cellular proteins associated with human cytomegalovirus (HCMV) DNA replication suggests novel cellular and viral interactions. <em>Virology</em><strong>566</strong>, 26–41 (2022).
</cite> [<a href="https://doi.org/10.1016/j.virol.2021.11.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8720285/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34861458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manska,%20S.%20&amp;%20Rossetto,%20C.%20C.%20Identification%20of%20cellular%20proteins%20associated%20with%20human%20cytomegalovirus%20(HCMV)%20DNA%20replication%20suggests%20novel%20cellular%20and%20viral%20interactions.%20Virology566,%2026%E2%80%9341%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Gredmark-Russ, S. &amp; Soderberg-Naucler, C. Dendritic cell biology in human cytomegalovirus infection and the clinical consequences for host immunity and pathology. <em>Virulence</em><strong>3</strong>, 621–634 (2012).
</cite> [<a href="https://doi.org/10.4161/viru.22239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3545944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23076329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gredmark-Russ,%20S.%20&amp;%20Soderberg-Naucler,%20C.%20Dendritic%20cell%20biology%20in%20human%20cytomegalovirus%20infection%20and%20the%20clinical%20consequences%20for%20host%20immunity%20and%20pathology.%20Virulence3,%20621%E2%80%93634%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Paulus, C., Krauss, S. &amp; Nevels, M. A human cytomegalovirus antagonist of type I IFN-dependent signal transducer and activator of transcription signaling. <em>Proc. Natl Acad. Sci. USA</em><strong>103</strong>, 3840–3845 (2006).
</cite> [<a href="https://doi.org/10.1073/pnas.0600007103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1533784/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16497831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paulus,%20C.,%20Krauss,%20S.%20&amp;%20Nevels,%20M.%20A%20human%20cytomegalovirus%20antagonist%20of%20type%20I%20IFN-dependent%20signal%20transducer%20and%20activator%20of%20transcription%20signaling.%20Proc.%20Natl%20Acad.%20Sci.%20USA103,%203840%E2%80%933845%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Park, A. et al. HCMV-encoded US7 and US8 act as antagonists of innate immunity by distinctively targeting TLR-signaling pathways. <em>Nat. Commun.</em><strong>10</strong>, 4670 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-12641-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6789044/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31604943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Park,%20A.%20et%20al.%20HCMV-encoded%20US7%20and%20US8%20act%20as%20antagonists%20of%20innate%20immunity%20by%20distinctively%20targeting%20TLR-signaling%20pathways.%20Nat.%20Commun.10,%204670%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Yang, D., de la Rosa, G., Tewary, P. &amp; Oppenheim, J. J. Alarmins link neutrophils and dendritic cells. <em>Trends Immunol.</em><strong>30</strong>, 531–537 (2009).
</cite> [<a href="https://doi.org/10.1016/j.it.2009.07.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2767430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19699678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20D.,%20de%20la%20Rosa,%20G.,%20Tewary,%20P.%20&amp;%20Oppenheim,%20J.%20J.%20Alarmins%20link%20neutrophils%20and%20dendritic%20cells.%20Trends%20Immunol.30,%20531%E2%80%93537%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Bogdanow, B. et al. Human cytomegalovirus tegument protein pp150 acts as a cyclin A2-CDK-dependent sensor of the host cell cycle and differentiation state. <em>Proc. Natl Acad. Sci. USA</em><strong>110</strong>, 17510–17515 (2013).
</cite> [<a href="https://doi.org/10.1073/pnas.1312235110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3808649/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24101496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bogdanow,%20B.%20et%20al.%20Human%20cytomegalovirus%20tegument%20protein%20pp150%20acts%20as%20a%20cyclin%20A2-CDK-dependent%20sensor%20of%20the%20host%20cell%20cycle%20and%20differentiation%20state.%20Proc.%20Natl%20Acad.%20Sci.%20USA110,%2017510%E2%80%9317515%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Luftig, M. A. Viruses and the DNA damage response: activation and antagonism. <em>Annu Rev. Virol.</em><strong>1</strong>, 605–625 (2014).
</cite> [<a href="https://doi.org/10.1146/annurev-virology-031413-085548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26958736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luftig,%20M.%20A.%20Viruses%20and%20the%20DNA%20damage%20response:%20activation%20and%20antagonism.%20Annu%20Rev.%20Virol.1,%20605%E2%80%93625%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Ball, C. B. et al. Human Cytomegalovirus Infection Elicits Global Changes in Host Transcription by RNA Polymerases I, II, and III. <em>Viruses</em><strong>14</strong>, 779 (2022).</cite> [<a href="https://doi.org/10.3390/v14040779" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9026722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35458509/" class="usa-link">PubMed</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Bogdanow, B., Phan, Q. V. &amp; Wiebusch, L. Emerging Mechanisms of G(1)/S Cell Cycle Control by Human and Mouse Cytomegaloviruses. <em>mBio</em><strong>12</strong>, e0293421 (2021).
</cite> [<a href="https://doi.org/10.1128/mBio.02934-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8669474/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34903047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bogdanow,%20B.,%20Phan,%20Q.%20V.%20&amp;%20Wiebusch,%20L.%20Emerging%20Mechanisms%20of%20G(1)/S%20Cell%20Cycle%20Control%20by%20Human%20and%20Mouse%20Cytomegaloviruses.%20mBio12,%20e0293421%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Goodwin, C. M., Ciesla, J. H. &amp; Munger, J. Who’s driving? Human Cytomegalovirus, Interferon, and NFkappaB signaling. <em>Viruses</em><strong>10</strong>, 447 (2018).</cite> [<a href="https://doi.org/10.3390/v10090447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6163874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30134546/" class="usa-link">PubMed</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Manandhar, T., Ho, G. T., Pump, W. C., Blasczyk, R. &amp; Bade-Doeding, C. Battle between Host Immune Cellular Responses and HCMV Immune Evasion. <em>Int. J. Mol. Sci.</em><strong>20</strong>, 3626 (2019).</cite> [<a href="https://doi.org/10.3390/ijms20153626" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6695940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31344940/" class="usa-link">PubMed</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Zeng, J. et al. Insights into the Transcriptome of Human Cytomegalovirus: A comprehensive review. <em>Viruses</em><strong>15</strong>, 1703 (2023).</cite> [<a href="https://doi.org/10.3390/v15081703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10458407/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37632045/" class="usa-link">PubMed</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Adamson, C. S. &amp; Nevels, M. M. Bright and early: inhibiting human cytomegalovirus by targeting major immediate-early gene expression or protein function. <em>Viruses</em><strong>12</strong>, 110 (2020).</cite> [<a href="https://doi.org/10.3390/v12010110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7019229/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31963209/" class="usa-link">PubMed</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Kim, D. et al. The architecture of SARS-CoV-2 transcriptome. <em>Cell</em><strong>181</strong>, 914–921.e10 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.04.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7179501/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32330414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20D.%20et%20al.%20The%20architecture%20of%20SARS-CoV-2%20transcriptome.%20Cell181,%20914%E2%80%93921.e10%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Sunshine, S. et al. Systematic functional interrogation of SARS-CoV-2 host factors using Perturb-seq. <em>Nat. Commun.</em><strong>14</strong>, 6245 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-41788-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10558542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37803001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sunshine,%20S.%20et%20al.%20Systematic%20functional%20interrogation%20of%20SARS-CoV-2%20host%20factors%20using%20Perturb-seq.%20Nat.%20Commun.14,%206245%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Drayman, N., Patel, P., Vistain, L. &amp; Tay, S. HSV-1 single-cell analysis reveals the activation of anti-viral and developmental programs in distinct sub-populations. <em>Elife</em><strong>8</strong>, e46339 (2019).</cite> [<a href="https://doi.org/10.7554/eLife.46339" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6570482/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31090537/" class="usa-link">PubMed</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Labbe, K. &amp; Saleh, M. Cell death in the host response to infection. <em>Cell Death Differ.</em><strong>15</strong>, 1339–1349 (2008).
</cite> [<a href="https://doi.org/10.1038/cdd.2008.91" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18566602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Labbe,%20K.%20&amp;%20Saleh,%20M.%20Cell%20death%20in%20the%20host%20response%20to%20infection.%20Cell%20Death%20Differ.15,%201339%E2%80%931349%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Augusto, L. et al. Toxoplasma gondii Co-opts the Unfolded Protein Response To Enhance Migration and Dissemination of Infected Host Cells. <em>mBio</em><strong>11</strong>, e00915-20 (2020).</cite> [<a href="https://doi.org/10.1128/mBio.00915-20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7343987/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32636244/" class="usa-link">PubMed</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Prasad, V. &amp; Greber, U. F. The endoplasmic reticulum unfolded protein response–homeostasis, cell death and evolution in virus infections. <em>FEMS Microbiol. Rev.</em><strong>45</strong>, fuab016 (2021).</cite> [<a href="https://doi.org/10.1093/femsre/fuab016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8498563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33765123/" class="usa-link">PubMed</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Saul, S. et al. Anticancer pan-ErbB inhibitors reduce inflammation and tissue injury and exert broad-spectrum antiviral effects. <em>J. Clin. Invest.</em><strong>133</strong>, e169510 (2023).</cite> [<a href="https://doi.org/10.1172/JCI169510" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10541190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37581931/" class="usa-link">PubMed</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Ma, S. et al. Chromatin potential identified by shared single-cell profiling of RNA and Chromatin. <em>Cell</em><strong>183</strong>, 1103–1116.e20 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.09.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7669735/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33098772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20S.%20et%20al.%20Chromatin%20potential%20identified%20by%20shared%20single-cell%20profiling%20of%20RNA%20and%20Chromatin.%20Cell183,%201103%E2%80%931116.e20%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Joost, S. et al. The molecular anatomy of mouse skin during hair growth and rest. <em>Cell Stem Cell</em><strong>26</strong>, 441–457.e7 (2020).
</cite> [<a href="https://doi.org/10.1016/j.stem.2020.01.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32109378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Joost,%20S.%20et%20al.%20The%20molecular%20anatomy%20of%20mouse%20skin%20during%20hair%20growth%20and%20rest.%20Cell%20Stem%20Cell26,%20441%E2%80%93457.e7%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Pethe, P., Noel, V. S. &amp; Kale, V. Deterministic role of sonic hedgehog signalling pathway in specification of hemogenic versus endocardiogenic endothelium from differentiated human embryonic stem cells. <em>Cells Dev.</em><strong>166</strong>, 203685 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cdev.2021.203685" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33994358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pethe,%20P.,%20Noel,%20V.%20S.%20&amp;%20Kale,%20V.%20Deterministic%20role%20of%20sonic%20hedgehog%20signalling%20pathway%20in%20specification%20of%20hemogenic%20versus%20endocardiogenic%20endothelium%20from%20differentiated%20human%20embryonic%20stem%20cells.%20Cells%20Dev.166,%20203685%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>St-Jacques, B. et al. Sonic hedgehog signaling is essential for hair development. <em>Curr. Biol.</em><strong>8</strong>, 1058–1068 (1998).
</cite> [<a href="https://doi.org/10.1016/s0960-9822(98)70443-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9768360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?St-Jacques,%20B.%20et%20al.%20Sonic%20hedgehog%20signaling%20is%20essential%20for%20hair%20development.%20Curr.%20Biol.8,%201058%E2%80%931068%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Trevino, A. E. et al. Chromatin and gene-regulatory dynamics of the developing human cerebral cortex at single-cell resolution. <em>Cell</em><strong>184</strong>, 5053–5069.e23 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.07.039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34390642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trevino,%20A.%20E.%20et%20al.%20Chromatin%20and%20gene-regulatory%20dynamics%20of%20the%20developing%20human%20cerebral%20cortex%20at%20single-cell%20resolution.%20Cell184,%205053%E2%80%935069.e23%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Wang, S. et al. The inconsistent regulation of HOXC13 on different keratins and the regulation mechanism on HOXC13 in cashmere goat (Capra hircus). <em>BMC Genomics</em><strong>19</strong>, 630 (2018).
</cite> [<a href="https://doi.org/10.1186/s12864-018-5011-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6107959/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30139327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20S.%20et%20al.%20The%20inconsistent%20regulation%20of%20HOXC13%20on%20different%20keratins%20and%20the%20regulation%20mechanism%20on%20HOXC13%20in%20cashmere%20goat%20(Capra%20hircus).%20BMC%20Genomics19,%20630%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Peng, Y. et al. Detection of new pioneer transcription factors as cell-type-specific nucleosome binders. <em>Elife</em><strong>12</strong>, RP88936 (2024).</cite> [<a href="https://doi.org/10.7554/eLife.88936" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10945518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38293962/" class="usa-link">PubMed</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Desanlis, I. et al. HOX13-dependent chromatin accessibility underlies the transition towards the digit development program. <em>Nat. Commun.</em><strong>11</strong>, 2491 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-16317-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7237422/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32427842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Desanlis,%20I.%20et%20al.%20HOX13-dependent%20chromatin%20accessibility%20underlies%20the%20transition%20towards%20the%20digit%20development%20program.%20Nat.%20Commun.11,%202491%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Gillespie, D. T. The chemical Langevin Equation. <em>J. Chem. Phys.</em><strong>113</strong>, 297–306 (2000).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gillespie,%20D.%20T.%20The%20chemical%20Langevin%20Equation.%20J.%20Chem.%20Phys.113,%20297%E2%80%93306%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Zwanzig, R. Ensemble method in the theory of irreversibility. <em>J. Chem. Phys.</em><strong>33</strong>, 1338–1341 (1960).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zwanzig,%20R.%20Ensemble%20method%20in%20the%20theory%20of%20irreversibility.%20J.%20Chem.%20Phys.33,%201338%E2%80%931341%20(1960)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Mori, H. Transport, collective motion, and Brownian motion. <em>Prog. Theor., Phys.</em><strong>33</strong>, 423–455 (1965).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mori,%20H.%20Transport,%20collective%20motion,%20and%20Brownian%20motion.%20Prog.%20Theor.,%20Phys.33,%20423%E2%80%93455%20(1965)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Xing, J. &amp; Kim, K. S. Application of the projection operator formalism to non-Hamiltonian dynamics. <em>J. Chem. Phys.</em><strong>134</strong>, 044132 (2011).</cite> [<a href="https://doi.org/10.1063/1.3530071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21280712/" class="usa-link">PubMed</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Hao, Y. et al. Integrated analysis of multimodal single-cell data. <em>Cell</em><strong>184</strong>, 3573–3587.e29 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.04.048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8238499/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hao,%20Y.%20et%20al.%20Integrated%20analysis%20of%20multimodal%20single-cell%20data.%20Cell184,%203573%E2%80%933587.e29%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Hastie, T. &amp; Tibshirani, R. <em>Generalized additive models</em>, (Chapman and Hall, London; New York, 1990).</cite>
</li>
<li id="CR77">
<span class="label">77.</span><cite>Mitra, S. et al. Single-cell multi-ome regression models identify functional and disease-associated enhancers and enable chromatin potential analysis. <em>Nat. Genet</em><strong>56</strong>, 627–636 (2024).
</cite> [<a href="https://doi.org/10.1038/s41588-024-01689-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11018525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38514783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mitra,%20S.%20et%20al.%20Single-cell%20multi-ome%20regression%20models%20identify%20functional%20and%20disease-associated%20enhancers%20and%20enable%20chromatin%20potential%20analysis.%20Nat.%20Genet56,%20627%E2%80%93636%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Chen, Y. et al. GraphVelo allows for accurate inference of multimodal velocities and molecular mechanisms for single cells. <em>Zendo</em>10.5281/zenodo.15852884 (2025).</cite> [<a href="https://doi.org/10.1038/s41467-025-62784-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12373899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40847023/" class="usa-link">PubMed</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373899/bin/41467_2025_62784_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (4.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373899/bin/41467_2025_62784_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (76.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373899/bin/41467_2025_62784_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (2.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373899/bin/41467_2025_62784_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (4.3MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All the sequencing raw data are publicly accessible. The A549 dataset can be accessed via <a href="https://figshare.com/ndownloader/files/53666738" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/ndownloader/files/53666738</a>. The FUCCI cell cycle data can be downloaded from <a href="https://figshare.com/ndownloader/files/53705057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/ndownloader/files/53705057</a>. The processed data used for benchmark are available at <a href="https://figshare.com/ndownloader/files/53667548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/ndownloader/files/53667548</a>. The mouse gastrulation subset to erythroid lineage can be extracted using scVelo’s CLI: <em>scvelo.datasets.gastrulation_erythroid()</em> or from the original work under accession number E-MTAB-6967 of ArrayExpress. The human bone marrow can be extracted using scVelo’s CLI: <em>scvelo.datasets.bonemarrow()</em> or through the Human Cell Atlas data portal at <a href="https://data.humancellatlas.org/explore/projects/091cf39b-01bc-42e5-9437-f419a66c8a45" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://data.humancellatlas.org/explore/projects/091cf39b-01bc-42e5-9437-f419a66c8a45</a>. The mouse coronal hemibrain spatial transcriptomic data can be downloaded from (<a href="https://www.dropbox.com/s/c5tu4drxda01m0u/mousebrain_bin60.h5ad?dl=0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.dropbox.com/s/c5tu4drxda01m0u/mousebrain_bin60.h5ad?dl=0</a>). The original HCMV infected moDC data can be accessed via Zenodo (<a href="https://zenodo.org/records/10404879" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/10404879</a>) and the processed data can be downloaded from <a href="https://figshare.com/ndownloader/files/53666756" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/ndownloader/files/53666756</a>. The SARS-CoV-2 data can be accessed via <a href="https://figshare.com/ndownloader/files/53666588" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/ndownloader/files/53666588</a>. The preprocessed mouse skin development dataset can be accessed via <a href="https://figshare.com/articles/dataset/Mouse_Hair_Follicle_RNA_Data/22575307" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Mouse_Hair_Follicle_RNA_Data/22575307</a> and <a href="https://figshare.com/articles/dataset/Mouse_hair_follicle_ATAC_data/22575313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Mouse_hair_follicle_ATAC_data/22575313</a>. The preprocessed human cortex dataset can be downloaded from <a href="https://figshare.com/articles/dataset/Developing_Human_Cortex_RNA_Data/22575376" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Developing_Human_Cortex_RNA_Data/22575376</a> and <a href="https://figshare.com/articles/dataset/Developing_Human_Cortex_ATAC_Data/22575370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Developing_Human_Cortex_ATAC_Data/22575370</a>. <a href="#Sec44" class="usa-link">Source data</a> are provided with this paper.</p>
<p>The source code of python package GraphVelo<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup> can be downloaded from <a href="https://github.com/xing-lab-pitt/GraphVelo" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/xing-lab-pitt/GraphVelo</a> and is released under the BSD 3-Clause License. Reproducibility and tutorials can be found in <a href="https://github.com/xing-lab-pitt/GraphVelo/tree/main/notebook" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/xing-lab-pitt/GraphVelo/tree/main/notebook</a> and <a href="https://graphvelo.readthedocs.io/en/latest/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://graphvelo.readthedocs.io/en/latest/</a>. The specific version of the code associated with this publication is archived in Zendo and is accessible via 10.5281/zenodo.15852884.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62784-w"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62784.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (19.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373899/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373899/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373899%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373899/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373899/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373899/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40847023/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373899/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40847023/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373899/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373899/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="UQeWYcdK1lFiZzhVE3ky7eQ0mQ2KQLVRXe0RlffXmhKH8KzF0HhTWmQM46VLeE2p">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
