
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Whole exome sequencing identifies FANCM as a susceptibility gene for estrogen-receptor-negative breast cancer in Hispanic/Latina women - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532F1D8AF228D3052F1D002BDA3620.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370925/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Whole exome sequencing identifies FANCM as a susceptibility gene for estrogen-receptor-negative breast cancer in Hispanic/Latina women">
<meta name="citation_author" content="Jovia L Nierenberg">
<meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author_institution" content="Department of Medicine, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Aaron W Adamson">
<meta name="citation_author_institution" content="Department of Population Sciences, Beckman Research Institute of City of Hope, Duarte, CA USA">
<meta name="citation_author" content="Donglei Hu">
<meta name="citation_author_institution" content="Department of Medicine, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Scott Huntsman">
<meta name="citation_author_institution" content="Department of Medicine, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Carmina Patrick">
<meta name="citation_author_institution" content="Department of Population Sciences, Beckman Research Institute of City of Hope, Duarte, CA USA">
<meta name="citation_author" content="Min Li">
<meta name="citation_author_institution" content="Department of Medicine, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Linda Steele">
<meta name="citation_author_institution" content="Department of Population Sciences, Beckman Research Institute of City of Hope, Duarte, CA USA">
<meta name="citation_author" content="Shu Tao">
<meta name="citation_author_institution" content="Integrative Genomics Shared Resource, Beckman Research Institute of City of Hope, Duarte, CA USA">
<meta name="citation_author" content="Yuan Chun Ding">
<meta name="citation_author_institution" content="Department of Population Sciences, Beckman Research Institute of City of Hope, Duarte, CA USA">
<meta name="citation_author" content="Barry Tong">
<meta name="citation_author_institution" content="Department of Medicine, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Yiwey Shieh">
<meta name="citation_author_institution" content="Department of Population Health Sciences, Weill Cornell Medicine, New York, NY USA">
<meta name="citation_author" content="Laura Fejerman">
<meta name="citation_author_institution" content="Department of Public Health Service, University of California, Davis, Davis, CA USA">
<meta name="citation_author_institution" content="UC Davis Comprehensive Cancer Center, University of California, Davis, Davis, CA USA">
<meta name="citation_author" content="Stephen B Gruber">
<meta name="citation_author_institution" content="Department of Medical Oncology and Center for Precision Medicine, City of Hope National Medical Center, Duarte, CA USA">
<meta name="citation_author" content="Christopher A Haiman">
<meta name="citation_author_institution" content="Department of Preventive Medicine, Norris Comprehensive Cancer Center, Keck School of Medicine, University of Southern California, Los Angeles, CA USA">
<meta name="citation_author" content="Esther M John">
<meta name="citation_author_institution" content="Department of Epidemiology &amp; Population Health, Stanford University School of Medicine, Stanford, CA USA">
<meta name="citation_author_institution" content="Department of Medicine, Stanford University School of Medicine, Stanford, CA USA">
<meta name="citation_author_institution" content="Stanford Cancer Institute, Stanford University School of Medicine, Stanford, CA USA">
<meta name="citation_author" content="Lawrence H Kushi">
<meta name="citation_author_institution" content="Division of Research, Kaiser Permanente Northern California, Oakland, CA USA">
<meta name="citation_author" content="Gabriela Torres-Mejía">
<meta name="citation_author_institution" content="Instituto Nacional de Salud Pública, Cuernavaca, Mexico">
<meta name="citation_author" content="Charité Ricker">
<meta name="citation_author_institution" content="Department of Medicine, Keck School of Medicine, University of Southern California, Los Angeles, CA USA">
<meta name="citation_author" content="Jeffrey N Weitzel">
<meta name="citation_author_institution" content="Division of Precision Prevention, The University of Kansas Comprehensive Cancer Center, Kansas City, KS USA">
<meta name="citation_author" content="Elad Ziv">
<meta name="citation_author_institution" content="Department of Medicine, University of California, San Francisco, San Francisco, CA USA">
<meta name="citation_author" content="Susan L Neuhausen">
<meta name="citation_author_institution" content="Department of Population Sciences, Beckman Research Institute of City of Hope, Duarte, CA USA">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7816">
<meta name="citation_doi" content="10.1038/s41467-025-60564-0">
<meta name="citation_pmid" content="40841357">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370925/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370925/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370925/pdf/41467_2025_Article_60564.pdf">
<meta name="description" content="Breast cancer (BC) is one of the most common cancers globally. Genetic testing facilitates screening and informs targeted risk-reduction and treatments. However, genes included in testing panels are from European-ancestry studies. We conducted a ...">
<meta name="og:title" content="Whole exome sequencing identifies FANCM as a susceptibility gene for estrogen-receptor-negative breast cancer in Hispanic/Latina women">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Breast cancer (BC) is one of the most common cancers globally. Genetic testing facilitates screening and informs targeted risk-reduction and treatments. However, genes included in testing panels are from European-ancestry studies. We conducted a ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370925/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370925">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-60564-0"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_60564.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370925%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370925/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370925/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370925/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7816. doi: <a href="https://doi.org/10.1038/s41467-025-60564-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-60564-0</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Whole exome sequencing identifies <em>FANCM</em> as a susceptibility gene for estrogen-receptor-negative breast cancer in Hispanic/Latina women</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nierenberg%20JL%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Jovia L Nierenberg</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Jovia L Nierenberg</span></h3>
<div class="p">
<sup>1</sup>Department of Epidemiology and Biostatistics, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">
<sup>2</sup>Department of Medicine, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nierenberg%20JL%22%5BAuthor%5D" class="usa-link"><span class="name western">Jovia L Nierenberg</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Adamson%20AW%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Aaron W Adamson</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Aaron W Adamson</span></h3>
<div class="p">
<sup>3</sup>Department of Population Sciences, Beckman Research Institute of City of Hope, Duarte, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Adamson%20AW%22%5BAuthor%5D" class="usa-link"><span class="name western">Aaron W Adamson</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Donglei Hu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Donglei Hu</span></h3>
<div class="p">
<sup>2</sup>Department of Medicine, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Donglei Hu</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huntsman%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Scott Huntsman</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Scott Huntsman</span></h3>
<div class="p">
<sup>2</sup>Department of Medicine, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huntsman%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Scott Huntsman</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Patrick%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Carmina Patrick</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Carmina Patrick</span></h3>
<div class="p">
<sup>3</sup>Department of Population Sciences, Beckman Research Institute of City of Hope, Duarte, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Patrick%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Carmina Patrick</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Min Li</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Min Li</span></h3>
<div class="p">
<sup>2</sup>Department of Medicine, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Min Li</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Steele%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Linda Steele</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Linda Steele</span></h3>
<div class="p">
<sup>3</sup>Department of Population Sciences, Beckman Research Institute of City of Hope, Duarte, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Steele%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Linda Steele</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tao%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Shu Tao</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Shu Tao</span></h3>
<div class="p">
<sup>4</sup>Integrative Genomics Shared Resource, Beckman Research Institute of City of Hope, Duarte, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tao%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shu Tao</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ding%20YC%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Yuan Chun Ding</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Yuan Chun Ding</span></h3>
<div class="p">
<sup>3</sup>Department of Population Sciences, Beckman Research Institute of City of Hope, Duarte, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ding%20YC%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuan Chun Ding</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tong%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Barry Tong</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Barry Tong</span></h3>
<div class="p">
<sup>2</sup>Department of Medicine, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tong%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Barry Tong</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shieh%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Yiwey Shieh</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Yiwey Shieh</span></h3>
<div class="p">
<sup>5</sup>Department of Population Health Sciences, Weill Cornell Medicine, New York, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shieh%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yiwey Shieh</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fejerman%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Laura Fejerman</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Laura Fejerman</span></h3>
<div class="p">
<sup>6</sup>Department of Public Health Service, University of California, Davis, Davis, CA USA </div>
<div class="p">
<sup>7</sup>UC Davis Comprehensive Cancer Center, University of California, Davis, Davis, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fejerman%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Laura Fejerman</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gruber%20SB%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Stephen B Gruber</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Stephen B Gruber</span></h3>
<div class="p">
<sup>8</sup>Department of Medical Oncology and Center for Precision Medicine, City of Hope National Medical Center, Duarte, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gruber%20SB%22%5BAuthor%5D" class="usa-link"><span class="name western">Stephen B Gruber</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Haiman%20CA%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Christopher A Haiman</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Christopher A Haiman</span></h3>
<div class="p">
<sup>9</sup>Department of Preventive Medicine, Norris Comprehensive Cancer Center, Keck School of Medicine, University of Southern California, Los Angeles, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Haiman%20CA%22%5BAuthor%5D" class="usa-link"><span class="name western">Christopher A Haiman</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22John%20EM%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Esther M John</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Esther M John</span></h3>
<div class="p">
<sup>10</sup>Department of Epidemiology &amp; Population Health, Stanford University School of Medicine, Stanford, CA USA </div>
<div class="p">
<sup>11</sup>Department of Medicine, Stanford University School of Medicine, Stanford, CA USA </div>
<div class="p">
<sup>12</sup>Stanford Cancer Institute, Stanford University School of Medicine, Stanford, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22John%20EM%22%5BAuthor%5D" class="usa-link"><span class="name western">Esther M John</span></a>
</div>
</div>
<sup>10,</sup><sup>11,</sup><sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kushi%20LH%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Lawrence H Kushi</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Lawrence H Kushi</span></h3>
<div class="p">
<sup>13</sup>Division of Research, Kaiser Permanente Northern California, Oakland, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kushi%20LH%22%5BAuthor%5D" class="usa-link"><span class="name western">Lawrence H Kushi</span></a>
</div>
</div>
<sup>13</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Torres-Mej%C3%ADa%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Gabriela Torres-Mejía</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Gabriela Torres-Mejía</span></h3>
<div class="p">
<sup>14</sup>Instituto Nacional de Salud Pública, Cuernavaca, Mexico </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Torres-Mej%C3%ADa%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gabriela Torres-Mejía</span></a>
</div>
</div>
<sup>14</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ricker%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Charité Ricker</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Charité Ricker</span></h3>
<div class="p">
<sup>15</sup>Department of Medicine, Keck School of Medicine, University of Southern California, Los Angeles, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ricker%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Charité Ricker</span></a>
</div>
</div>
<sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weitzel%20JN%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Jeffrey N Weitzel</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Jeffrey N Weitzel</span></h3>
<div class="p">
<sup>16</sup>Division of Precision Prevention, The University of Kansas Comprehensive Cancer Center, Kansas City, KS USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weitzel%20JN%22%5BAuthor%5D" class="usa-link"><span class="name western">Jeffrey N Weitzel</span></a>
</div>
</div>
<sup>16</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ziv%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Elad Ziv</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Elad Ziv</span></h3>
<div class="p">
<sup>2</sup>Department of Medicine, University of California, San Francisco, San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ziv%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Elad Ziv</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Neuhausen%20SL%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Susan L Neuhausen</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Susan L Neuhausen</span></h3>
<div class="p">
<sup>3</sup>Department of Population Sciences, Beckman Research Institute of City of Hope, Duarte, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Neuhausen%20SL%22%5BAuthor%5D" class="usa-link"><span class="name western">Susan L Neuhausen</span></a>
</div>
</div>
<sup>3</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Epidemiology and Biostatistics, University of California, San Francisco, San Francisco, CA USA </div>
<div id="Aff2">
<sup>2</sup>Department of Medicine, University of California, San Francisco, San Francisco, CA USA </div>
<div id="Aff3">
<sup>3</sup>Department of Population Sciences, Beckman Research Institute of City of Hope, Duarte, CA USA </div>
<div id="Aff4">
<sup>4</sup>Integrative Genomics Shared Resource, Beckman Research Institute of City of Hope, Duarte, CA USA </div>
<div id="Aff5">
<sup>5</sup>Department of Population Health Sciences, Weill Cornell Medicine, New York, NY USA </div>
<div id="Aff6">
<sup>6</sup>Department of Public Health Service, University of California, Davis, Davis, CA USA </div>
<div id="Aff7">
<sup>7</sup>UC Davis Comprehensive Cancer Center, University of California, Davis, Davis, CA USA </div>
<div id="Aff8">
<sup>8</sup>Department of Medical Oncology and Center for Precision Medicine, City of Hope National Medical Center, Duarte, CA USA </div>
<div id="Aff9">
<sup>9</sup>Department of Preventive Medicine, Norris Comprehensive Cancer Center, Keck School of Medicine, University of Southern California, Los Angeles, CA USA </div>
<div id="Aff10">
<sup>10</sup>Department of Epidemiology &amp; Population Health, Stanford University School of Medicine, Stanford, CA USA </div>
<div id="Aff11">
<sup>11</sup>Department of Medicine, Stanford University School of Medicine, Stanford, CA USA </div>
<div id="Aff12">
<sup>12</sup>Stanford Cancer Institute, Stanford University School of Medicine, Stanford, CA USA </div>
<div id="Aff13">
<sup>13</sup>Division of Research, Kaiser Permanente Northern California, Oakland, CA USA </div>
<div id="Aff14">
<sup>14</sup>Instituto Nacional de Salud Pública, Cuernavaca, Mexico </div>
<div id="Aff15">
<sup>15</sup>Department of Medicine, Keck School of Medicine, University of Southern California, Los Angeles, CA USA </div>
<div id="Aff16">
<sup>16</sup>Division of Precision Prevention, The University of Kansas Comprehensive Cancer Center, Kansas City, KS USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2023 Sep 1; Accepted 2025 May 28; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370925  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841357/" class="usa-link">40841357</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>
<strong>Previous version available:</strong> This article is based on a previously available preprint posted on medRxiv on January 28, 2023: "<a href="/articles/PMC9901069/" class="usa-link">Whole exome sequencing and replication for breast cancer among Hispanic/Latino women identifies <em>FANCM</em> as a susceptibility gene for estrogen-receptor-negative breast cancer</a>".</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Breast cancer (BC) is one of the most common cancers globally. Genetic testing facilitates screening and informs targeted risk-reduction and treatments. However, genes included in testing panels are from European-ancestry studies. We conducted a pooled case-control analysis in self-identified Hispanic/Latina women (4178 cases and 4344 controls), using whole exome sequencing and a targeted panel. We tested the association of loss of function (LoF) variants with overall, estrogen receptor (ER)-positive, and ER-negative BC risk. Using logistic regression, we found a strong association of LoF variants in <em>FANCM</em> with ER-negative BC (<em>p</em> = 4.1 × 10<sup>−</sup><sup>7</sup>), odds ratio [confidence interval]: 6.7 [2.9–15.6]). Among known susceptibility genes, <em>BRCA1</em>, <em>BRCA2</em>, and <em>PALB2</em> strongly associated with BC. <em>FANCM</em> was previously proposed as a possible susceptibility gene for ER-negative BC, but is not routinely tested clinically. Our results demonstrate that <em>FANCM</em> should be added to BC gene panels.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Breast cancer, Risk factors, Cancer genetics, Genetic association study, Cancer genomics</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">The genetic susceptibility to breast cancer remains understudied in non-European populations. Here, the authors analyse pathogenic variants associated with breast cancer susceptibility in Hispanic/Latina women using genomics, and find that loss of function variants in FANCM are strongly associated with ER-negative breast cancer risk.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Breast cancer (BC) is the most common cancer in women<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>, and is influenced by hormonal, environmental, and genetic factors<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Germline pathogenic variants found in <em>BRCA1</em><sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup><em>, BRCA2</em><sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>, <em>PALB2</em><sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, <em>TP53</em><sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup> and others genes<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> are associated with high risk of BC and underlie familial cancer syndromes. Several intermediate-penetrance genes including <em>CHEK2</em><sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> and <em>ATM</em><sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup> also have been identified. Genome-wide association studies (GWAS) have identified many common variants that contribute to BC risk<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. All the pathogenic variants in susceptibility genes and the common risk variants identified to date explain approximately half of the heritability of the disease<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>.</p>
<p id="Par4">Genetic testing for pathogenic variants in BC susceptibility genes is currently used to identify women at high risk of developing BC, who may benefit from increased screening and risk-reducing interventions, for cascade testing in families to identify other individuals at increased risk, and to inform the use of targeted treatments in those who develop cancer<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Two large, recent studies, in predominantly European ancestry participants, confirmed the association of known BC susceptibility genes with increased BC risk in population-based cohorts, reiterating that many of these genes are important to include on clinical genetic testing panels<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p>
<p id="Par5">Most of the knowledge of genetic susceptibility to BC is based on studies in European-ancestry populations, leaving gaps in the understanding of genetic effects in other populations. Among Hispanic/Latina (H/L) women in the United States (US), BC is the most common cancer and leading cause of cancer-related death<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Latin-American populations are genetically diverse, and many H/L individuals have admixed ancestry, comprised primarily of European, Indigenous American, and African components<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. GWAS of BC in H/L women identified a protective variant near <em>ESR1</em>, which is most common in women with high Indigenous American ancestry<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. In addition, unique founder variants have been identified in <em>BRCA1</em><sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, <em>PALB2</em><sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>, and <em>CHEK2</em><sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. To better understand the impact of rare variants in coding sequence of genes on BC risk among self-identified H/L women, we performed a whole exome sequencing (WES) and targeted replication approach in over 8500 H/L women from California and Mexico.</p>
<p id="Par6">Here, we show an exome-wide significant association between loss-of-function (LoF) variants in <em>FANCM</em> and estrogen-receptor (ER) negative BC and a nominal association with overall BC risk. We also show significant associations between known susceptibility genes, <em>BRCA1</em>, <em>BRCA2</em>, and <em>PALB2</em> and overall BC risk, as well as nominally significant associations with <em>CHEK2</em>, <em>RAD51C</em>, and <em>TP53</em>.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<p id="Par7">The mean age at BC diagnosis was 42.6 years (standard deviation [SD]: 8.5) and the mean age of controls at enrollment was 60.3 years (SD: 10.9) in the discovery set (<em>P</em> = 2.4 × 10<sup>−200</sup>) (Table <a href="#Tab1" class="usa-link">1</a>). In the replication set, the mean age at BC diagnosis was 55.3 years (SD: 12.0) and the mean age of controls at enrollment was 54.9 years (SD: 11.4, <em>P</em> = 0.27). European ancestry (EA) proportion was higher among cases (t = −4.53, degrees of freedom [df]= 8594.6, <em>P</em> = 6.0 × 10<sup>−6</sup>) and Indigenous American ancestry (IA) was higher among controls (t = 5.05, df = 8599.5, <em>P</em> = 4.4 × 10<sup>−7</sup>, Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). Cases were more likely to report history of a first-degree relative with BC than controls (χ<sup>2</sup> = 99.57, 1 df, <em>P</em> = 2.5 × 10<sup>−108</sup>). Most of the cases were ER-positive and progesterone receptor (PR)-positive and 21.3% of those tested were human epidermal growth factor receptor 2 (HER2)-positive (Table <a href="#Tab1" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="Tab1"><h3 class="obj_head">Table 1.</h3>
<div class="caption p"><p>Characteristics of study participants included in the analysis</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1"></th>
<th colspan="1" rowspan="1">Cases n = 4264</th>
<th colspan="1" rowspan="1">Controls n = 4350</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">Discovery samples, Whole-exome sequencing (Clinical Research Exome, Agilent)</td>
<td colspan="1" rowspan="1">n = 1043</td>
<td colspan="1" rowspan="1">n = 1188</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> CCGCRN</td>
<td colspan="1" rowspan="1">885 (85%)</td>
<td colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> COH</td>
<td colspan="1" rowspan="1">N/A</td>
<td colspan="1" rowspan="1">313 (26%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> MEC</td>
<td colspan="1" rowspan="1">N/A</td>
<td colspan="1" rowspan="1">875 (74%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> UCSF</td>
<td colspan="1" rowspan="1">52 (5%)</td>
<td colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> USC</td>
<td colspan="1" rowspan="1">106 (10%)</td>
<td colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Replication, targeted sequencing</td>
<td colspan="1" rowspan="1">n = 3221</td>
<td colspan="1" rowspan="1">n = 3162</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> CAMA</td>
<td colspan="1" rowspan="1">1123 (35%)</td>
<td colspan="1" rowspan="1">1122 (36%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> CPMCRI Cohort</td>
<td colspan="1" rowspan="1">12 (0.4%)</td>
<td colspan="1" rowspan="1">847 (27%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> PATHWAYS</td>
<td colspan="1" rowspan="1">412 (13%)</td>
<td colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> MEC</td>
<td colspan="1" rowspan="1">816 (25%)</td>
<td colspan="1" rowspan="1">865 (27%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> NC-BCFR</td>
<td colspan="1" rowspan="1">696 (22%)</td>
<td colspan="1" rowspan="1">54 (2%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> SFBCS</td>
<td colspan="1" rowspan="1">162 (5%)</td>
<td colspan="1" rowspan="1">274 (9%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Unselected for Hereditary Risk<sup>a</sup> (%)</td>
<td colspan="1" rowspan="1">2993 (70%)</td>
<td colspan="1" rowspan="1">3162 (73%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Age (mean (SD))</td>
<td colspan="1" rowspan="1">52.1 (13)</td>
<td colspan="1" rowspan="1">55.9 (12)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">African ancestry (mean (SD))</td>
<td colspan="1" rowspan="1">0.05 (0.07)</td>
<td colspan="1" rowspan="1">0.05 (0.05)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Indigenous American ancestry (mean (SD))</td>
<td colspan="1" rowspan="1">0.40 (0.23)</td>
<td colspan="1" rowspan="1">0.43 (0.22)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">European ancestry (mean (SD))</td>
<td colspan="1" rowspan="1">0.54 (0.22)</td>
<td colspan="1" rowspan="1">0.52 (0.22)</td>
</tr>
<tr><td colspan="3" rowspan="1">Family history of breast cancer in a first degree relative (%)</td></tr>
<tr>
<td colspan="1" rowspan="1"> Yes</td>
<td colspan="1" rowspan="1">891 (21%)</td>
<td colspan="1" rowspan="1">324 (7%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> No</td>
<td colspan="1" rowspan="1">3034 (71%)</td>
<td colspan="1" rowspan="1">3162 (73%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> Missing</td>
<td colspan="1" rowspan="1">339 (8%)</td>
<td colspan="1" rowspan="1">864 (20%)</td>
</tr>
<tr><td colspan="3" rowspan="1">Estrogen receptor status<sup>b</sup> (%)</td></tr>
<tr>
<td colspan="1" rowspan="1"> Positive</td>
<td colspan="1" rowspan="1">2094 (49%)</td>
<td colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> Negative</td>
<td colspan="1" rowspan="1">713 (17%)</td>
<td colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> Missing</td>
<td colspan="1" rowspan="1">1457 (34%)</td>
<td colspan="1" rowspan="1">N/A</td>
</tr>
<tr><td colspan="3" rowspan="1">Progesterone receptor status** (%)</td></tr>
<tr>
<td colspan="1" rowspan="1"> Positive</td>
<td colspan="1" rowspan="1">1594 (37%)</td>
<td colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> Negative</td>
<td colspan="1" rowspan="1">940 (22%)</td>
<td colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> Missing</td>
<td colspan="1" rowspan="1">1730 (41%)</td>
<td colspan="1" rowspan="1">N/A</td>
</tr>
<tr><td colspan="3" rowspan="1">Human epidermal growth factor receptor 2 status (%)</td></tr>
<tr>
<td colspan="1" rowspan="1"> Positive</td>
<td colspan="1" rowspan="1">349 (8%)</td>
<td colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> Negative</td>
<td colspan="1" rowspan="1">1285 (30%)</td>
<td colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> Missing</td>
<td colspan="1" rowspan="1">2630 (62%)</td>
<td colspan="1" rowspan="1">N/A</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p9">
<p><em>CAMA</em> Cancer de Mama, <em>CCGCRN</em> Clinical Cancer Genomics Community Research Network, <em>COH</em> City of Hope, <em>CPMCRI-Cohort</em> California Pacific Medical Center - Breast Health Center, <em>MEC</em> Multi-Ethnic Cohort, <em>PR</em> progesterone receptor, <em>UCSF</em> University of California San Francisco, <em>USC</em> University of Southern California.</p>
<p><sup>a</sup> Selection was not based on hereditary risk in Kaiser, MEC, CPMC Cohort, the San Francisco Bay Area Cancer Study, CAMA, and for some participants in the Northern California Breast Cancer Family Registry.</p>
<p><sup>b</sup>Estrogen receptor and progesterone receptor status was missing in &gt;80% of CAMA cases and in approximately 20% of other cases.</p>
</div></div></section><p id="Par8">In the analysis for overall BC risk focusing on LoF variants, we found significant associations with <em>BRCA1</em>, <em>BRCA2</em> and <em>PALB2</em>, with ORs (95% CI) of 24.9 (6.1–102.5), 7.0 (3.5–14.0), and 6.5 (3.2–13.1), respectively. For ER-negative BC, LoF variants in <em>BRCA1</em>, <em>BRCA2</em> and <em>FANCM</em>, were significantly associated with ORs of 40.7 (8.9-186.5), 10.5 (4.5-24.7), and 6.7 (2.9-15.6), respectively (Table <a href="#Tab2" class="usa-link">2</a> and Fig. <a href="#Fig1" class="usa-link">1</a>). Single LoF variants in each of the significant genes are shown in Supplementary Data <a href="#MOESM6" class="usa-link">3</a>. No exome-wide significant associations were found with ER-positive BC. Three other known BC genes had nominally significant associations with BC: <em>CHEK2</em> with overall and ER-positive BC, <em>RAD51D</em> with ER-negative BC, and <em>TP53</em> with ER-negative BC (Supplementary Table <a href="#MOESM1" class="usa-link">1</a> and Fig. <a href="#Fig1" class="usa-link">1</a>). We found no significant associations of <em>ATM</em>, <em>BARD1</em>, <em>CDH1</em>, <em>RAD51C</em>, or <em>RECQL</em> and BC risk. We ran a sensitivity analysis for <em>ATM</em> using truncating variants only, due to the unexpectedly null results and large number of splice variants, and also found no association with BC (OR = 1.2 [0.8–1.9]). All genes with suggestive (<em>P</em> &lt; 0.01) associations for overall, ER-positive and ER-negative BC are shown in Supplementary Tables <a href="#MOESM1" class="usa-link">2</a> and <a href="#MOESM1" class="usa-link">3</a>. We also performed separate analyses that combined LoF variants with predicted deleterious missense variants (Supplementary Tables <a href="#MOESM1" class="usa-link">5</a> and <a href="#MOESM1" class="usa-link">6</a>). Associations with <em>BRCA1</em>, <em>BRCA2</em> and <em>PALB2</em> remained significant, and the association with <em>FANCM</em> and ER-negative breast cancer was mildly attenuated (<em>p</em> = 2.9 × 10<sup>−6</sup>).</p>
<section class="tw xbox font-sm" id="Tab2"><h3 class="obj_head">Table 2.</h3>
<div class="caption p"><p>Gene-based <em>P</em>-values from joint analysis of LoF variants with exome-wide significance, overall breast cancer risk in estrogen receptor-positive and estrogen receptor-negative breast cancers</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Gene</th>
<th colspan="1" rowspan="1">Chr</th>
<th colspan="1" rowspan="1">Overall</th>
<th colspan="1" rowspan="1">ER + </th>
<th colspan="1" rowspan="1">ER−</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">
<em>BRCA1</em><sup>a</sup>
</td>
<td colspan="1" rowspan="1">17</td>
<td colspan="1" rowspan="1">
<strong>2.3 × 10</strong><sup><strong>−10</strong></sup>
</td>
<td colspan="1" rowspan="1">0.03</td>
<td colspan="1" rowspan="1">
<strong>4.4 × 10</strong><sup><strong>−16</strong></sup>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>BRCA2</em><sup>a</sup>
</td>
<td colspan="1" rowspan="1">13</td>
<td colspan="1" rowspan="1">
<strong>8.4 × 10</strong><sup>−</sup><sup><strong>10</strong></sup>
</td>
<td colspan="1" rowspan="1">3.3 × 10<sup>−4</sup>
</td>
<td colspan="1" rowspan="1">
<strong>8.0 × 10</strong><sup><strong>−15</strong></sup>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>FANCM</em></td>
<td colspan="1" rowspan="1">14</td>
<td colspan="1" rowspan="1">9.8 × 10<sup>−</sup><sup>3</sup>
</td>
<td colspan="1" rowspan="1">0.11</td>
<td colspan="1" rowspan="1">
<strong>4.1 × 10</strong><sup>−<strong>7</strong></sup>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><em>PALB2</em></td>
<td colspan="1" rowspan="1">16</td>
<td colspan="1" rowspan="1">
<strong>1.8 × 10</strong><sup><strong>−8</strong></sup>
</td>
<td colspan="1" rowspan="1">1.3 × 10<sup>−5</sup>
</td>
<td colspan="1" rowspan="1">5.9 × 10<sup>−</sup><sup>5</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p15">
<p><em>P</em>-values are from gene-based SKAT-O analyses. Bold <em>P</em>-values indicate Bonferroni corrected statistical significance, adjusting for multiple comparisons in a two-sided test at an alpha threshold of 0.05/20,000 = 2.5 × 10<sup>−6</sup>. Overall breast cancer analyses included N = 8614 (4264 cases and 4350 controls). Estrogen receptor-positive analyses included N = 6444 independent samples (2094 cases and 4350 controls). Estrogen receptor-analyses included H = 5063 independent samples (713 cases and 4350 controls).</p>
<p><em>Chr</em> chromosome; <em>ER</em> estrogen receptor; <em>LoF</em> loss of function.</p>
<p><sup>a</sup>Discovery participants were selected for being <em>BRCA1/2</em> negative (see section “Methods”), replication results are presented for <em>BRCA1/2</em>.</p>
</div></div></section><figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. Gene-based odds ratios from joint analysis for overall, estrogen receptor-positive, and estrogen receptor-negative disease.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370925_41467_2025_60564_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c1f/12370925/c560f36251b4/41467_2025_60564_Fig1_HTML.jpg" loading="lazy" id="d33e1081" height="806" width="683" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Odds ratios (circles) and 95% confidence intervals (bars) are presented for overall breast cancer, (n = 8614 independent samples, including 4264 cases and 4350 controls), estrogen receptor-positive (n = 6444 independent samples, including 2094 cases and 4350 controls), and estrogen receptor-negative disease (n = 5063 independent samples including 713 cases and 4350 controls). The dot represents the point estimate and the bars represent the upper and lower bound of the 95% confidence intervals. The X-axis describes the odds ratio on a log scale, the Y-axis represents the individual genes. Statistical significance was determined using SKAT-O <em>P</em>-values. The exome-wide significance threshold selected was 0.05/20,000 = 2.5 × 10<sup>−6</sup>, based on 20,000 genes in the genome. Orange circles and bars represent genes with exome-wide significance. Results for a gene were included if the gene had &gt;5 variants or at least one alternate allele in both cases and controls. Genes are listed in alphabetical order. Source data are provided as a Source Data file. ER estrogen receptor.</p></figcaption></figure><p id="Par9">The association between <em>FANCM</em> LoF variants and ER-negative BC was largely driven by two stop-gain variants, rs147021911 (chr14:45658326C&gt;T) and rs144567652 (chr14:45667921C&gt;A/C&gt;T) (Fig. <a href="#Fig2" class="usa-link">2</a>). Among ER-negative cases, 1.7% (12/713) had a LoF variant in <em>FANCM</em>, compared to 0.6% of cases with ER-positive BC and 0.2% of controls (Fig. <a href="#Fig2" class="usa-link">2</a>). We also analyzed the association with ER-negative, progesterone receptor (PR)-negative, Her2-negative breast cancer, frequently referred to as triple negative breast cancer (TNBC). Among the 713 women with ER negative breast cancer, 318 (45%) had missing PR or HER2 status, so TNBC status could not be determined. Among those with complete data on PR and HER2, 266 (67%) had TNBC and 129 (33%) did not. Even with this small sample size, <em>FANCM</em> was associated with TNBC (OR = 4.5, SKAT-O <em>p</em> = 0.00013).</p>
<figure class="fig xbox font-sm" id="Fig2"><h3 class="obj_head">Fig. 2. <em>FANCM</em> loss of function (LoF) variant frequencies by participant group.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c1f/12370925/b7407e52991f/41467_2025_60564_Fig2_HTML.jpg" loading="lazy" id="d33e1276" height="1471" width="663" alt="Fig. 2"></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Loss of function (LoF) variants included frameshift, stopgain, and predicted splice variants. The <em>X</em>-axis shows groups of participants (n = 37 biologically independent samples with FANCM LoF variants). The <em>Y</em>-axis shows the frequency of carriers (all carriers are heterozygous). The variant, rs144567652, is also known as chr14:45667921C&gt;A/C&gt;T, c.5791C&gt;T, and p.Arg1931* (overall OR = 3.2, 95% CI: 1.0–9.8 and ER-negative OR = 8.1, 95% CI: 2.3–29.0). The variant, rs147021911, is also known as chr14:45658326C&gt;T, c.5101C&gt;T, and p.Gln1701* (overall OR = 3.0, 95% CI: 0.6–14.7 and ER-negative OR = 11.3, 95% CI: 2.1–62.2). Other loss of function (LoF) variants found in <em>FANCM</em> include chr14:45605743G&gt;A, rs140760056 (chr14:45633697C&gt;T), rs368728266 (chr14:45636336C&gt;T), rs778176467 (chr14:45642357C&gt;T), chr14:45644584A&gt;AT, rs1566762924 (chr14:45644795T&gt;C), chr14:45644911, chr14:45644926TAG&gt;T, rs755947203 (chr14:45653091C&gt;A,T), rs930692973 (chr14:45658155C&gt;G,T), and rs1379375089 (chr14:45665681C&gt;T). Source data (participant counts) are provided in the figure. Source data are provided as a Source Data file. ER estrogen receptor, FHx family history of breast cancer in first-degree relatives, LoF loss of function.</p></figcaption></figure><p id="Par10">Since ascertainment can affect the OR estimates and our discovery dataset and replication datasets were selected based on different criteria, we analyzed the top genes among replication-set participants who were from studies of cases unselected for hereditary risk (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). The effect sizes for <em>CHEK2</em> and <em>PALB2</em> were similar in all studies and in unselected studies. The OR for <em>CHEK2</em> and ER-positive BC in unselected studies was 10.7, which was similar to the OR of 10.8 found in all studies. We found that the association between <em>FANCM</em> and ER-negative BC was similar in studies of unselected cases (OR = 7.6) and in all studies (OR = 6.7). We also tested for interactions by age and by genetic ancestry. As expected, both rs147021911 (chr14:45658326C&gt;T) and rs144567652 (chr14:45667921C&gt;A/C&gt;T) were more common among women with higher European ancestry. We found a trend towards a greater effect size among women with high Indigenous American ancestry, but this did not reach statistical significance (<em>p</em> = 0.06). We found no significant interaction between <em>FANCM</em> variants and age or study site (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>).</p>
<p id="Par11">Since many of the known genes for BC are involved in repair of double-stranded breaks, we reviewed all suggestive associations in the combined discovery and validation analyses and identified genes that are members of this pathway using a previously curated list<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. In analyses that included both LoF variants and missense variants with high likelihood of being pathogenic, we found suggestive evidence for <em>ATR</em> and <em>FANCG</em> in analyses that included LoF and predicted deleterious missense variants (Supplementary Tables <a href="#MOESM1" class="usa-link">5</a> and <a href="#MOESM1" class="usa-link">6</a>). <em>ATR</em> was more strongly associated with ER-positive disease, while <em>FANCG</em> was more strongly associated with ER-negative BC.</p></section><section id="Sec3"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par12">In this case-control study of BC in over 8,500 H/L women, we found strong evidence of association between ER-negative BC and LoF variants in <em>FANCM</em> largely driven by two <em>FANCM</em> stop-gain variants. In addition, we saw strong associations with the known BC genes <em>BRCA1</em>, <em>BRCA2</em>, and <em>PALB2</em>. <em>FANCM</em> previously had been proposed as a susceptibility gene for development of ER-negative BC although without exome-wide significance<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. The two most common <em>FANCM</em> LoF variants we observed are more common in European populations than other populations. They were first identified in European ancestry <em>BRCA1/2</em>-negative familial BC studies<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, and have been associated with ER-negative or triple negative BC<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Additionally, rs147021911 (chr14:45658326C&gt;T, c.5101C&gt;T, p.Gln1701*) was associated with poor BC survival in a Finnish population<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, and other <em>FANCM</em> LoF mutations were found in small BC case series<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, including a report of bi-allelic <em>FANCM</em> variants in early onset or bilateral BCs<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>.</p>
<p id="Par13">Our study identified exome-wide significant association between <em>FANCM</em> and ER-negative BC in any population, but our results should be interpreted with caution. Since our sample size is relatively modest, the OR may be inflated by the winner’s curse<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>–<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. However, even if the OR is inflated, the higher impact on ER-negative disease is consistent with prior studies and suggests that testing for <em>FANCM</em> LoF variants can help identify women at increased risk for developing ER-negative BC. Another possibility for the high OR that we observed may be an, as yet, unknown modifying factor that affects the risk of <em>FANCM</em> and is more common among Latinas. Prior studies have identified heterogeneity among carriers of pathogenic variants in intermediate penetrance susceptibility genes including pathogenic variants in <em>CHEK2</em> which are associated with higher risk among women with more affected relatives<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Our study did not identify family history as a significant modifier of risk. We did note a trend towards a higher OR among women with higher Indigenous American ancestry, but this was not statistically significant. Our sample size was very limited to detect interactions. Future, larger studies of <em>FANCM</em> will be helpful to improve the precision of risk estimates associated with LoF variants in this gene and in understanding which factors may underlie any heterogeneity among carriers.</p>
<p id="Par14">Similar to most other BC susceptibility genes, <em>FANCM</em> is involved in double-stranded DNA repair. FANCM localizes to stalled replication forks and initiates the response to double-stranded breaks by homologous recombination repair<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. <em>FANCM</em> is a member of the Fanconi Anemia (FA) complex and is the most conserved gene within the FA pathway<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. which is essential for handling DNA interstrand cross linking damage in DNA repair through the homologous recombination repair pathway<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. A stronger association between <em>FANCM</em> and ER-negative disease than ER-positive disease is similar to results for <em>RAD51C</em> and <em>RAD51D</em><sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, which are also key components of the FA pathway.</p>
<p id="Par15">Given our results and the previous associations, we believe the data support consideration of inclusion of <em>FANCM</em> on clinical testing panels. Most current genetic susceptibility breast cancer testing panels do not include <em>FANCM</em><sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> and there are no guidelines from the National Comprehensive Cancer Network on managing women who are carriers<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. The absolute risk of breast cancer among <em>FANCM</em> LoF carriers is likely much lower than other intermediate breast cancer susceptibility genes, since ER-negative and triple negative BC constitute the minority of overall breast cancers and that is the main subtype with an increased risk. However, since the average age of diagnosis for ER-negative BC and TNBC are much lower<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, women with <em>FANCM</em> LoF variants may benefit from earlier BC screening. Moreover, women with BC and <em>FANCM</em> LoF variants may benefit from poly adenosine diphosphate-ribose polymerase (PARP) inhibitors<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, therefore, testing for <em>FANCM</em> LoF variants may identify additional therapeutic options for women with ER-negative BC.</p>
<p id="Par16">We found that carriers of LoF variants in <em>CHEK2</em> had a particularly high risk of ER-positive disease, which is consistent with prior studies<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. However, the effect sizes in our study were substantially higher than those in previous reports<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. In analyses that excluded cases selected for familial BC, the CIs were wide but the lower bound of the CI (2.1) included the OR reported by Dorling and colleagues<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, indicating that our larger OR may be due to chance. The higher OR we observed also may be due to the younger ages of the H/L population and/or to heterogeneity of genetic or hormonal and environmental risk factors across populations. H/L women tend to have more protective non-genetic risk factors such as younger age at first pregnancy, higher parity<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, lower postmenopausal hormone use, and lower alcohol consumption<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>, when compared to US non-Hispanic White populations which may lead to stronger associations with genetic risk factors<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. In contrast to the strong association we observed with <em>CHEK2</em>, we found no association with LoF variants in <em>ATM</em>. The upper bound of the CI for ER-positive disease excludes the reported OR in the BC Association Consortium study<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> but not the OR reported in the CARRIERS study<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. We had previously noted no significant association with <em>ATM</em> LoF variants in Latinas<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> in a dataset that partially overlaps our current report. These results may be due to genetic or hormonal and environmental factors that attenuate the associations with <em>ATM</em> LoF variants in this population. However, until larger studies are done on <em>ATM</em> in H/L populations, it may be prudent to continue to consider these women at moderately increased risk of BC like non-H/L White women.</p>
<p id="Par17">Our study identified strong associations with BC risk genes, including an exome-wide significant association between <em>FANCM</em> and ER-negative disease, demonstrating the importance of conducting genetic studies in understudied populations. Most large, complex trait genetic studies, including those on BC, were conducted in European-ancestry participants<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. The lack of genetic studies in non-European populations exacerbates health disparities as genomics is increasingly used in clinical practice.</p>
<p id="Par18">We found higher European ancestry and lower Indigenous American ancestry among US Latina and Mexican breast cancer cases compared to controls. We had previously evaluated the risk factors underlying this association using data that substantially overlap with this study<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>,<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Part of this association was explained by known reproductive and lifestyle risk factors for breast cancer. However, the association with ancestry remained significant after adjusting for these covariates<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>,<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. We used admixture mapping, genome wide association, and fine mapping to identify a 6q25 variant which is associated with substantial protection from breast cancer, originates in Indigenous American populations, and explains part of the association with ancestry among Latinas<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>.</p>
<p id="Par19">Our study has several strengths. We focused on H/L women, a population that is understudied for breast cancer risk. We included participants from both hereditary-risk studies and unselected cases and controls, and compared results across these two study types, making our findings applicable to both groups. Our study also has several limitations. The WES for discovery was performed in only 1043 cases and 1188 controls and was likely underpowered to find intermediate-penetrance genes. To compensate for this, we sequenced 857 genes for replication in 3221 cases and 3162 controls. This sample size is still much lower than those used in discovery and validation of breast cancer susceptibility genes in European ancestry populations<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Larger studies in H/L populations are needed to provide better point estimates, to identify new candidate genes for BC, and to detect and understand factors contributing to heterogeneity of effect sizes compared to studies in US non-Hispanic White and European populations.</p>
<p id="Par20">Our discovery phase included sequencing data previously collected from a subset of controls in the MEC using a slightly different WES target capture kit. This difference could lead to spurious associations due to technical differences between the different capture kits or to demographic differences in the discovery set cases and controls However, we accounted for these potential differences by adjusting for ancestry and excluding variants that were significantly different in an analysis of our two discovery control populations.</p>
<p id="Par21">In conclusion, our study demonstrates an exome-wide significant association between LoF variants in <em>FANCM</em> and ER-negative BC in H/L women from multiple studies. We also found exome-wide significant associations for <em>BRCA1</em>, <em>BRCA2</em>, and <em>PALB2</em>. Our findings suggest that <em>FANCM</em> should be added to genetic testing panels for BC, which is especially important for H/L women. Additionally, our findings demonstrate the importance of conducting genetic studies in admixed populations.</p></section><section id="Sec4"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec5"><h3 class="pmc_sec_title">Study samples</h3>
<p id="Par22">The research complies with all relevant ethical regulations.All participants were consented using written signed forms and enrolled through center-specific institutional review board-approved protocols. The institutional review boards included the City of Hope Beckman Research Center, the University of California, San Francisco, Stanford University, Kaiser Permanente Northern California, Division of Research, the University of Southern California and the National Institute of Public Health, Cuernavaca, Mexico. We performed discovery and replication in a pooled case-control study of invasive BC among self-identified H/L women in the U.S. and women recruited in Mexico. Cases had been diagnosed with at least one invasive BC and we used age at first BC diagnosis for women diagnosed with two or more metachronous BCs<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Discovery cases were selected for having previously tested negative for <em>BRCA1/2</em>, and were diagnosed at &lt;51 years of age and/or had bilateral (synchronous or metachronous) BC, breast and ovarian cancers, or were diagnosed between 51 and 70 years with a family history of BC in ≥2 first-degree or second-degree relatives diagnosed at age &lt;70 years<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. H/L participants in the replication set were from six studies (Supplemental Methods). We included 8,614 participants in our study and the mean (standard deviation) ages of cases and controls were 52.1 (13) and 55.9 (12) years old.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Ethics &amp; inclusion statement</h3>
<p id="Par23">The research study has included local researchers throughout the research process including study design, data acquisition, data ownership and authorship. The local researchers deemed this research relevant. Roles and responsibilities were agreed upon amongst the collaborators before the research, and capacity building for local researchers was discussed. Local institutional review boards evaluated the research and approved. Participants were consented for research testing of genetic/inherited risk of breast cancer. Local research was considered as part of citations.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Sequencing and genotyping</h3>
<p id="Par24">Whole exome sequencing (WES) from DNA of discovery participants was performed using the SureSelect Clinical Research Exome (Agilent, Santa Clara, CA) kit<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. For participants in the replication set, 857 genes were selected for targeted sequencing based on the discovery results and biological plausibility. A custom SureSelect XT kit (Agilent, Santa Clara, CA) was used to capture the exons of 857 genes (Supplementary Data <a href="#MOESM4" class="usa-link">1</a>) and also included 189 known BC single nucleotide polymorphisms (SNPs) and 100 ancestry informative SNP markers. For all sequencing, we used KAPA Hyper Preparation Kits (Kapa Biosystems, Inc., Wilmington, MA) to generate libraries from 500 ng DNA. One hundred base-pair paired end sequencing on the HiSEQ 2500 Genetic Analyzer (Illumina Inc., San Diego, CA) was performed in the COH Integrative Genomics Core (IGC) to an average fold coverage of 65× for the WES samples and ~75× for the targeted-sequencing samples (Supplementary Data <a href="#MOESM5" class="usa-link">2</a>). Paired-end reads were aligned to human reference genome (hg37) using the Burrows-Wheeler Alignment Tool (BWA, version 0.7.5a-r405) under default settings, and the aligned binary format sequence (BAM) files were sorted and indexed using SAMtools<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. The same FASTA reference file was used to align the MEC control samples. Duplicate reads were removed from the sorted and indexed BAMs using Picard MarkDuplicates (version 1.67, <a href="http://broadinstitute.github.io/picard/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://broadinstitute.github.io/picard/</a>).</p>
<p id="Par25">Variant calling from BAM files from COH IGC and the Broad Sequencing Center were performed together using GATK HaplotypeCaller (<a href="https://software.broadinstitute.org/gatk" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://software.broadinstitute.org/gatk</a>) after local realignment of reads around insertions and deletions (indels) and base quality score recalibration by The Genome Analysis Toolkit (GATK, v3.6-0-g89b7209). Variants were annotated using ANNOVAR<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. We excluded participants with &lt;20-fold average coverage. Variants with call quality &lt;20, read depth &lt;10, less frequent allele depth of &lt;4, or allele fraction ratio &lt;30% were filtered out for low quality. DNA from eight MEC participants were sequenced at both COH IGC and the Broad Sequencing Center with &gt;99.8% concordance for variant calling.</p>
<p id="Par26">From the discovery dataset (whole exome sequencing), we excluded 14 samples due to insufficient coverage (&lt;20X), 4 duplicate samples, and 17 samples who were first- or second-degree relatives. We used PLINK 1.9 (<a href="http://www.cog-genomics.org/plink/1.9/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.cog-genomics.org/plink/1.9/</a>) 28 to exclude first- or second-degree relatives or duplicate samples within the discovery and replication samples. We excluded an additional eight participants from the discovery analysis who were identified to have previously undetected <em>BRCA1/2</em> pathogenic variants. We also excluded 73 participants from the replication set who were duplicates from the discovery set. After all exclusions, there were 4260 cases and 4350 controls (Table <a href="#Tab1" class="usa-link">1</a>).</p>
<p id="Par27">Due to possible differences among samples sequenced at the Broad and COH in our discovery sample, we conducted a control-control analysis and excluded variants that were different between the Broad controls (from the Multi-ethnic cohort) and COH control populations at <em>p</em> &lt; 0.05. We also conducted a sensitivity analysis within the CCGCRN/COH only, and excluded variants where the sensitivity analysis point estimate was outside of the 95% confidence interval of the main analysis.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Ancestry estimation</h3>
<p id="Par28">The Clinical Research Exome included a custom panel of 180 ancestry-informative SNPs selected to be informative for European, Indigenous-American, and African populations<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. In addition, we selected 7691 variants common to our WES data and a data set of Axiom arrays, including African (N = 90), European (N = 90), and Indigenous American (N = 71) populations. We used a subset of 4544 unlinked markers by linkage disequilibrium pruning in PLINK<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. We estimated genetic ancestry using ADMIXTURE 1.3<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup> and performed analyses with unsupervised (including data from ancestral populations, but not specifying the identity of ancestral populations) runs. The same approach was used for the MEC control participants where we selected independent variants (n = 12,758) that overlapped between the Axiom arrays and the MEC dataset and in the replication set where we selected 1,195 SNPs that overlapped across all replication and reference datasets.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par29">Gene-based aggregate rare variant analyses were based on loss of function (LoF) variants, including frameshift, stop gain, and predicted splice variants. Variants with minor allele frequency &gt;0.0025 in our control study population and benign clinical significance in CLINVAR<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> were excluded from the gene-based analyses. Statistical significance for each gene was determined using SKAT-O<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. We used a <em>p</em>-value of 2.5 × 10<sup>−6</sup> for the threshold of genome-wide significance which adjusts for ~20,000 coding sequence genes as has been done previously. Odds ratios (OR) and 95% confidence intervals (CI) for BC associated with any LoF variant for each gene were calculated using logistic regression models in which women with at least one LoF variant in a gene were encoded as “1” and all other women were encoded as “0”. These models included ancestry (European and Indigenous American) as covariates and constituted our burden tests. We only included genes that had &gt;5 variants or at least one alternate allele in both cases and controls. We used the replication sample for all <em>BRCA1/2</em> analyses, as <em>BRCA1/2</em> status was an exclusion criterion for the discovery sample. We looked for the known Mexican founder <em>BRCA1</em> exon 9–12 deletion<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, and verified its presence using the Integrative Genomics Viewer (IGV), and included this variant in analyses, but did not perform additional large deletion analyses. In addition, to LoF variants, we also performed separate analyses that included missense variants that were predicted to be deleterious based on VEST<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> with VEST score &gt;0.8 which were analyzed in combination with LoF variants.</p>
<p id="Par30">Single variant analyses and aggregate rare variant analyses were conducted in discovery and replication separately and a joint discovery and replication analysis. BC subtype-specific analyses were conducted, where cases were restricted to ER-positive and ER-negative disease. Sensitivity analyses were run separately for participants from hereditary BC studies and studies that did not select cases based on BC risk factors.</p>
<p id="Par31">To test for interactions with <em>FANCM</em> variant status, we used logistic regression models and entered presence of a <em>FANCM</em> LoF variant multiplied by either age or genetic ancestry as continuous variables or family history as a dichotomous variable.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par32">Further information on research design is available in the <a href="#MOESM7" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (1.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Peer Review File</a><sup> (371.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_60564_MOESM3_ESM.pdf</a><sup> (100.5KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (19.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (488.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (16.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting summary</a><sup> (2.4MB, pdf) </sup>
</div></div></section></section><section id="Sec12"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (311KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank the participants of the SFBCS/NC-BCFR, Pathways, MEC, CAMA, and COH-CCGCRN studies. This work was funded by the NIH/NCI (R01CA184585, K24CA169004 to E.Z.) and the State of California Initiative to Advance Precision Medicine (OPR18111 to EZ, SN and L.F.). Research reported in this publication included work performed in the City of Hope Integrative Genomics Core supported by the NIH/NCI under grant number P30CA033572. The content and views are solely the responsibility of the authors and should not be construed to represent the views of the National Institutes of Health. S.L.N. and this research were partially funded by the Morris and Horowitz Families Professorship. The Multiethnic Cohort was supported by NIH/NCI grants (R01CA164973, R01CA054281, and CA063464 to C.H.). Sequencing of the MEC controls was conducted as part of the Slim Initiative for Genomic Medicine, a project funded by the Carlos Slim Health Institute in Mexico. Seventy-six of the controls were from the California Teachers Study; the collection and data were funded through the NIH/NCI (R01CA77398). The Northern California Breast Cancer Family Registry was supported by a grant from the National Cancer Institute (UM1 CA164920 (to E.J.). The SFBCS was funded by grants from the NIH/NCI (R01CA063446 and R01CA077305 to EJ), grant from the US Department of Defense (DAMD17-96–1-6071 to E.Z.), and a grant from the California Breast Cancer Research Program (7PB-0068 to E.J.). The Kaiser Permanente Research Program on Genes, Environment and Health was supported by Kaiser Permanente national and regional Community Benefit programs, and grants from the Ellison Medical Foundation, the Wayne and Gladys Valley Foundation, and the Robert Wood Johnson Foundation. The PATHWAYS cohort was supported by grants from the National Cancer Institute (U01 CA195565 R01CA105274 to LK) The CAMA Study was funded by Consejo Nacional de Ciencia y Tecnologıa, Mexico (SALUD-2002-C01–7462 to G.T.M.) and recruitment at the Puebla site was supported by the National Cancer Institute (R01CA120120 to E.Z.). Additional support was provided the NIH/NCI R01CA204797 (to L.F.). Funding to recruit women from for the CCGCRN includes Breast Cancer Research Foundation grant #20-172 (to J.N.W.), and American Society of Clinical Oncology Conquer Cancer® Research Professorship in Breast Cancer Disparities (to J.N.W.) and additional funding was provided by NIH/NCIH RC4 CA153828 (to J.N.W). Support was also provided from National Center for Advancing Translational Sciences under award KL2TR001870 (to YS) and the National Cancer Institute under award K08CA237829 (to Y.S.).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>E.Z. and S.N. designed the study. J.L.N., E.Z. and S.N. drafted the manuscript. J.L.N., D.H., and S.H., conducted statistical analyses. E.Z., S.N., J.L.N., S.T., B.T., S.B.G., C.A.H., E.M.J., L.H.K., G.T., C.R., and J.N.W. contributed samples and data. A.W.A., C.P., and M.L. prepared samples for sequencing. L.S. performed clinical data ascertainment and cleaning. Y.C.D., Y.S. and L.F. made critical revisions to the manuscript. All authors reviewed and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par33"><em>Nature Communications</em> thanks Sandra Romero-Cordoba, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Source data are provided with this manuscript. The genotype data and raw sequencing readsgenerated in this study have been deposited in the dbGAP database and are available at phs003144.v1.p1 [<a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs003144.v1.p1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs003144.v1.p1</a>]. The genotype data are available under restricted access due to the informed consent under which the sample were obtained. Access can be obtained be obtained by applying with a scientific proposal to dbGAP. <a href="#Sec12" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par34">J.L.N. is an employee and shareholder at BridgeBio Pharma, this work was done prior to her work at BridgeBio Pharma. J.N.W. is a consultant to Natera, MyOme and Cancer IQ, holds equity in Natera and is a co-founder of Novi Health. All other authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec13"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-60564-0.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Harbeck, N. &amp; Gnant, M. Breast cancer. <em>Lancet</em><strong>389</strong>, 1134–1150 (2017).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(16)31891-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27865536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harbeck,%20N.%20&amp;%20Gnant,%20M.%20Breast%20cancer.%20Lancet389,%201134%E2%80%931150%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Garcia-Closas, M., Gunsoy, N. B. &amp; Chatterjee, N. Combined associations of genetic and environmental risk factors: implications for prevention of breast cancer. <em>J. Natl. Cancer Inst.</em><strong>106</strong>, 10.1093/jnci/dju305 (2014).</cite> [<a href="https://doi.org/10.1093/jnci/dju305" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4271030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25392194/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Miki, Y. et al. A strong candidate for the breast and ovarian cancer susceptibility gene BRCA1. <em>Science</em><strong>266</strong>, 66–71 (1994).
</cite> [<a href="https://doi.org/10.1126/science.7545954" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7545954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miki,%20Y.%20et%20al.%20A%20strong%20candidate%20for%20the%20breast%20and%20ovarian%20cancer%20susceptibility%20gene%20BRCA1.%20Science266,%2066%E2%80%9371%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Wooster, R. et al. Identification of the breast cancer susceptibility gene BRCA2. <em>Nature</em><strong>378</strong>, 789–792 (1995).
</cite> [<a href="https://doi.org/10.1038/378789a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8524414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wooster,%20R.%20et%20al.%20Identification%20of%20the%20breast%20cancer%20susceptibility%20gene%20BRCA2.%20Nature378,%20789%E2%80%93792%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Rahman, N. et al. PALB2, which encodes a BRCA2-interacting protein, is a breast cancer susceptibility gene. <em>Nat. Genet.</em><strong>39</strong>, 165–167 (2007).
</cite> [<a href="https://doi.org/10.1038/ng1959" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2871593/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17200668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rahman,%20N.%20et%20al.%20PALB2,%20which%20encodes%20a%20BRCA2-interacting%20protein,%20is%20a%20breast%20cancer%20susceptibility%20gene.%20Nat.%20Genet.39,%20165%E2%80%93167%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Malkin, D. et al. Germ line p53 mutations in a familial syndrome of breast cancer, sarcomas, and other neoplasms. <em>Science</em><strong>250</strong>, 1233–1238 (1990).
</cite> [<a href="https://doi.org/10.1126/science.1978757" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1978757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Malkin,%20D.%20et%20al.%20Germ%20line%20p53%20mutations%20in%20a%20familial%20syndrome%20of%20breast%20cancer,%20sarcomas,%20and%20other%20neoplasms.%20Science250,%201233%E2%80%931238%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Srivastava, S., Zou, Z. Q., Pirollo, K., Blattner, W. &amp; Chang, E. H. Germ-line transmission of a mutated p53 gene in a cancer-prone family with Li-Fraumeni syndrome. <em>Nature</em><strong>348</strong>, 747–749 (1990).
</cite> [<a href="https://doi.org/10.1038/348747a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2259385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Srivastava,%20S.,%20Zou,%20Z.%20Q.,%20Pirollo,%20K.,%20Blattner,%20W.%20&amp;%20Chang,%20E.%20H.%20Germ-line%20transmission%20of%20a%20mutated%20p53%20gene%20in%20a%20cancer-prone%20family%20with%20Li-Fraumeni%20syndrome.%20Nature348,%20747%E2%80%93749%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Foulkes, W. D. The ten genes for breast (and ovarian) cancer susceptibility. <em>Nat. Rev. Clin. Oncol.</em><strong>18</strong>, 259–260 (2021).
</cite> [<a href="https://doi.org/10.1038/s41571-021-00491-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33692540/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Foulkes,%20W.%20D.%20The%20ten%20genes%20for%20breast%20(and%20ovarian)%20cancer%20susceptibility.%20Nat.%20Rev.%20Clin.%20Oncol.18,%20259%E2%80%93260%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Meijers-Heijboer, H. et al. Low-penetrance susceptibility to breast cancer due to CHEK2(*)1100delC in noncarriers of BRCA1 or BRCA2 mutations. <em>Nat. Genet.</em><strong>31</strong>, 55–59 (2002).
</cite> [<a href="https://doi.org/10.1038/ng879" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11967536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meijers-Heijboer,%20H.%20et%20al.%20Low-penetrance%20susceptibility%20to%20breast%20cancer%20due%20to%20CHEK2(*)1100delC%20in%20noncarriers%20of%20BRCA1%20or%20BRCA2%20mutations.%20Nat.%20Genet.31,%2055%E2%80%9359%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Thompson, D. et al. Cancer risks and mortality in heterozygous ATM mutation carriers. <em>J. Natl Cancer Inst.</em><strong>97</strong>, 813–822 (2005).
</cite> [<a href="https://doi.org/10.1093/jnci/dji141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15928302/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thompson,%20D.%20et%20al.%20Cancer%20risks%20and%20mortality%20in%20heterozygous%20ATM%20mutation%20carriers.%20J.%20Natl%20Cancer%20Inst.97,%20813%E2%80%93822%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Skol, A. D., Sasaki, M. M. &amp; Onel, K. The genetics of breast cancer risk in the post-genome era: thoughts on study design to move past BRCA and towards clinical relevance. <em>Breast Cancer Res.</em><strong>18</strong>, 99 (2016).
</cite> [<a href="https://doi.org/10.1186/s13058-016-0759-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5048663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27716388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Skol,%20A.%20D.,%20Sasaki,%20M.%20M.%20&amp;%20Onel,%20K.%20The%20genetics%20of%20breast%20cancer%20risk%20in%20the%20post-genome%20era:%20thoughts%20on%20study%20design%20to%20move%20past%20BRCA%20and%20towards%20clinical%20relevance.%20Breast%20Cancer%20Res.18,%2099%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Daly, M. B. et al. Genetic/familial high-risk assessment: breast, ovarian, and pancreatic, version 2.2021, NCCN Clinical Practice Guidelines in Oncology. <em>J. Natl Compr. Cancer Netw.</em><strong>19</strong>, 77–102 (2021).</cite> [<a href="https://doi.org/10.6004/jnccn.2021.0001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33406487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Daly,%20M.%20B.%20et%20al.%20Genetic/familial%20high-risk%20assessment:%20breast,%20ovarian,%20and%20pancreatic,%20version%202.2021,%20NCCN%20Clinical%20Practice%20Guidelines%20in%20Oncology.%20J.%20Natl%20Compr.%20Cancer%20Netw.19,%2077%E2%80%93102%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Easton, D. F. et al. Gene-panel sequencing and the prediction of breast-cancer risk. <em>N. Engl. J. Med.</em><strong>372</strong>, 2243–2257 (2015).
</cite> [<a href="https://doi.org/10.1056/NEJMsr1501341" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4610139/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26014596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Easton,%20D.%20F.%20et%20al.%20Gene-panel%20sequencing%20and%20the%20prediction%20of%20breast-cancer%20risk.%20N.%20Engl.%20J.%20Med.372,%202243%E2%80%932257%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Force, U. S. P. S. T. et al. Risk assessment, genetic counseling, and genetic testing for BRCA-related cancer: US preventive services task force recommendation statement. <em>JAMA</em><strong>322</strong>, 652–665 (2019).
</cite> [<a href="https://doi.org/10.1001/jama.2019.10987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31429903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Force,%20U.%20S.%20P.%20S.%20T.%20et%20al.%20Risk%20assessment,%20genetic%20counseling,%20and%20genetic%20testing%20for%20BRCA-related%20cancer:%20US%20preventive%20services%20task%20force%20recommendation%20statement.%20JAMA322,%20652%E2%80%93665%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>King, M. C., Levy-Lahad, E. &amp; Lahad, A. Population-based screening for BRCA1 and BRCA2: 2014 Lasker Award. <em>JAMA</em><strong>312</strong>, 1091–1092 (2014).
</cite> [<a href="https://doi.org/10.1001/jama.2014.12483" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25198398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?King,%20M.%20C.,%20Levy-Lahad,%20E.%20&amp;%20Lahad,%20A.%20Population-based%20screening%20for%20BRCA1%20and%20BRCA2:%202014%20Lasker%20Award.%20JAMA312,%201091%E2%80%931092%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Dorling, L. et al. Breast cancer risks associated with missense variants in breast cancer susceptibility genes. <em>Genome Med.</em><strong>14</strong>, 51 (2022).
</cite> [<a href="https://doi.org/10.1186/s13073-022-01052-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9116026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35585550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dorling,%20L.%20et%20al.%20Breast%20cancer%20risks%20associated%20with%20missense%20variants%20in%20breast%20cancer%20susceptibility%20genes.%20Genome%20Med.14,%2051%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Hu, C. et al. A population-based study of genes previously implicated in breast cancer. <em>N. Engl. J. Med.</em><strong>384</strong>, 440–451 (2021).
</cite> [<a href="https://doi.org/10.1056/NEJMoa2005936" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8127622/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33471974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu,%20C.%20et%20al.%20A%20population-based%20study%20of%20genes%20previously%20implicated%20in%20breast%20cancer.%20N.%20Engl.%20J.%20Med.384,%20440%E2%80%93451%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Miller, K. D. et al. Cancer statistics for Hispanics/Latinos, 2018. <em>CA Cancer J. Clin.</em><strong>68</strong>, 425–445 (2018).30285281
</cite> [<a href="https://scholar.google.com/scholar_lookup?Miller,%20K.%20D.%20et%20al.%20Cancer%20statistics%20for%20Hispanics/Latinos,%202018.%20CA%20Cancer%20J.%20Clin.68,%20425%E2%80%93445%20(2018).30285281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Bryc, K. et al. Colloquium paper: genome-wide patterns of population structure and admixture among Hispanic/Latino populations. <em>Proc. Natl Acad. Sci. USA</em><strong>107</strong>, 8954–8961 (2010).
</cite> [<a href="https://doi.org/10.1073/pnas.0914618107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3024022/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20445096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bryc,%20K.%20et%20al.%20Colloquium%20paper:%20genome-wide%20patterns%20of%20population%20structure%20and%20admixture%20among%20Hispanic/Latino%20populations.%20Proc.%20Natl%20Acad.%20Sci.%20USA107,%208954%E2%80%938961%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Gonzalez Burchard, E. et al. Latino populations: a unique opportunity for the study of race, genetics, and social environment in epidemiological research. <em>Am. J. Public Health</em><strong>95</strong>, 2161–2168 (2005).
</cite> [<a href="https://doi.org/10.2105/AJPH.2005.068668" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1449501/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16257940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gonzalez%20Burchard,%20E.%20et%20al.%20Latino%20populations:%20a%20unique%20opportunity%20for%20the%20study%20of%20race,%20genetics,%20and%20social%20environment%20in%20epidemiological%20research.%20Am.%20J.%20Public%20Health95,%202161%E2%80%932168%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Fejerman, L. et al. Genome-wide association study of breast cancer in Latinas identifies novel protective variants on 6q25. <em>Nat. Commun.</em><strong>5</strong>, 5260 (2014).
</cite> [<a href="https://doi.org/10.1038/ncomms6260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4204111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25327703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fejerman,%20L.%20et%20al.%20Genome-wide%20association%20study%20of%20breast%20cancer%20in%20Latinas%20identifies%20novel%20protective%20variants%20on%206q25.%20Nat.%20Commun.5,%205260%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Herzog, J. S. et al. Genetic epidemiology of BRCA1- and BRCA2-associated cancer across Latin America. <em>NPJ Breast Cancer</em><strong>7</strong>, 107 (2021).
</cite> [<a href="https://doi.org/10.1038/s41523-021-00317-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8377150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34413315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herzog,%20J.%20S.%20et%20al.%20Genetic%20epidemiology%20of%20BRCA1-%20and%20BRCA2-associated%20cancer%20across%20Latin%20America.%20NPJ%20Breast%20Cancer7,%20107%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Weitzel, J. N. et al. Prevalence and type of BRCA mutations in Hispanics undergoing genetic cancer risk assessment in the southwestern United States: a report from the Clinical Cancer Genetics Community Research Network. <em>J. Clin. Oncol.</em><strong>31</strong>, 210–216 (2013).
</cite> [<a href="https://doi.org/10.1200/JCO.2011.41.0027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3532393/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23233716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weitzel,%20J.%20N.%20et%20al.%20Prevalence%20and%20type%20of%20BRCA%20mutations%20in%20Hispanics%20undergoing%20genetic%20cancer%20risk%20assessment%20in%20the%20southwestern%20United%20States:%20a%20report%20from%20the%20Clinical%20Cancer%20Genetics%20Community%20Research%20Network.%20J.%20Clin.%20Oncol.31,%20210%E2%80%93216%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Weitzel, J. N. et al. Pathogenic and likely pathogenic variants in PALB2, CHEK2, and other known breast cancer susceptibility genes among 1054 BRCA-negative Hispanics with breast cancer. <em>Cancer</em><strong>125</strong>, 2829–2836 (2019).
</cite> [<a href="https://doi.org/10.1002/cncr.32083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7376605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31206626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weitzel,%20J.%20N.%20et%20al.%20Pathogenic%20and%20likely%20pathogenic%20variants%20in%20PALB2,%20CHEK2,%20and%20other%20known%20breast%20cancer%20susceptibility%20genes%20among%201054%20BRCA-negative%20Hispanics%20with%20breast%20cancer.%20Cancer125,%202829%E2%80%932836%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Carter, H., Douville, C., Stenson, P. D., Cooper, D. N. &amp; Karchin, R. Identifying Mendelian disease genes with the variant effect scoring tool. <em>BMC Genomics</em><strong>14</strong>, S3 (2013).
</cite> [<a href="https://doi.org/10.1186/1471-2164-14-S3-S3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3665549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23819870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carter,%20H.,%20Douville,%20C.,%20Stenson,%20P.%20D.,%20Cooper,%20D.%20N.%20&amp;%20Karchin,%20R.%20Identifying%20Mendelian%20disease%20genes%20with%20the%20variant%20effect%20scoring%20tool.%20BMC%20Genomics14,%20S3%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Kiiski, J. I. et al. Exome sequencing identifies FANCM as a susceptibility gene for triple-negative breast cancer. <em>Proc. Natl Acad. Sci. USA</em><strong>111</strong>, 15172–15177 (2014).
</cite> [<a href="https://doi.org/10.1073/pnas.1407909111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4210278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25288723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kiiski,%20J.%20I.%20et%20al.%20Exome%20sequencing%20identifies%20FANCM%20as%20a%20susceptibility%20gene%20for%20triple-negative%20breast%20cancer.%20Proc.%20Natl%20Acad.%20Sci.%20USA111,%2015172%E2%80%9315177%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Peterlongo, P. et al. FANCM c.5791C&gt;T nonsense mutation (rs144567652) induces exon skipping, affects DNA repair activity and is a familial breast cancer risk factor. <em>Hum. Mol. Genet.</em><strong>24</strong>, 5345-–55355 (2015).
</cite> [<a href="https://doi.org/10.1093/hmg/ddv251" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4550823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26130695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peterlongo,%20P.%20et%20al.%20FANCM%20c.5791C&gt;T%20nonsense%20mutation%20(rs144567652)%20induces%20exon%20skipping,%20affects%20DNA%20repair%20activity%20and%20is%20a%20familial%20breast%20cancer%20risk%20factor.%20Hum.%20Mol.%20Genet.24,%205345-%E2%80%9355355%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Kiiski, J. I. et al. FANCM mutation c.5791C&gt;T is a risk factor for triple-negative breast cancer in the Finnish population. <em>Breast Cancer Res. Treat.</em><strong>166</strong>, 217–226 (2017).
</cite> [<a href="https://doi.org/10.1007/s10549-017-4388-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5645429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28702895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kiiski,%20J.%20I.%20et%20al.%20FANCM%20mutation%20c.5791C&gt;T%20is%20a%20risk%20factor%20for%20triple-negative%20breast%20cancer%20in%20the%20Finnish%20population.%20Breast%20Cancer%20Res.%20Treat.166,%20217%E2%80%93226%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Neidhardt, G. et al. Association between loss-of-function mutations within the FANCM gene and early-onset familial breast cancer. <em>JAMA Oncol.</em><strong>3</strong>, 1245–1248 (2017).
</cite> [<a href="https://doi.org/10.1001/jamaoncol.2016.5592" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5824291/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28033443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Neidhardt,%20G.%20et%20al.%20Association%20between%20loss-of-function%20mutations%20within%20the%20FANCM%20gene%20and%20early-onset%20familial%20breast%20cancer.%20JAMA%20Oncol.3,%201245%E2%80%931248%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Figlioli, G. et al. The FANCM:p.Arg658* truncating variant is associated with risk of triple-negative breast cancer. <em>NPJ Breast Cancer</em><strong>5</strong>, 38 (2019).
</cite> [<a href="https://doi.org/10.1038/s41523-019-0127-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6825205/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31700994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Figlioli,%20G.%20et%20al.%20The%20FANCM:p.Arg658*%20truncating%20variant%20is%20associated%20with%20risk%20of%20triple-negative%20breast%20cancer.%20NPJ%20Breast%20Cancer5,%2038%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Kiiski, J. I. et al. FANCM c.5101C&gt;T mutation associates with breast cancer survival and treatment outcome. <em>Int. J. Cancer</em><strong>139</strong>, 2760–2770 (2016).
</cite> [<a href="https://doi.org/10.1002/ijc.30394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5095781/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27542569/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kiiski,%20J.%20I.%20et%20al.%20FANCM%20c.5101C&gt;T%20mutation%20associates%20with%20breast%20cancer%20survival%20and%20treatment%20outcome.%20Int.%20J.%20Cancer139,%202760%E2%80%932770%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Nguyen-Dumont, T. et al. FANCM and RECQL genetic variants and breast cancer susceptibility: relevance to South Poland and West Ukraine. <em>BMC Med Genet.</em><strong>19</strong>, 12 (2018).
</cite> [<a href="https://doi.org/10.1186/s12881-018-0524-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5775547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29351780/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen-Dumont,%20T.%20et%20al.%20FANCM%20and%20RECQL%20genetic%20variants%20and%20breast%20cancer%20susceptibility:%20relevance%20to%20South%20Poland%20and%20West%20Ukraine.%20BMC%20Med%20Genet.19,%2012%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Rashid, M. U., Muhammad, N., Khan, F. A. &amp; Hamann, U. Absence of the FANCM c.5101C&gt;T mutation in BRCA1/2-negative triple-negative breast cancer patients from Pakistan. <em>Breast Cancer Res. Treat.</em><strong>152</strong>, 229–230 (2015).
</cite> [<a href="https://doi.org/10.1007/s10549-015-3457-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26067930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rashid,%20M.%20U.,%20Muhammad,%20N.,%20Khan,%20F.%20A.%20&amp;%20Hamann,%20U.%20Absence%20of%20the%20FANCM%20c.5101C&gt;T%20mutation%20in%20BRCA1/2-negative%20triple-negative%20breast%20cancer%20patients%20from%20Pakistan.%20Breast%20Cancer%20Res.%20Treat.152,%20229%E2%80%93230%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Schubert, S. et al. The identification of pathogenic variants in BRCA1/2 negative, high risk, hereditary breast and/or ovarian cancer patients: High frequency of FANCM pathogenic variants. <em>Int. J. Cancer</em><strong>144</strong>, 2683–2694 (2019).
</cite> [<a href="https://doi.org/10.1002/ijc.31992" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30426508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schubert,%20S.%20et%20al.%20The%20identification%20of%20pathogenic%20variants%20in%20BRCA1/2%20negative,%20high%20risk,%20hereditary%20breast%20and/or%20ovarian%20cancer%20patients:%20High%20frequency%20of%20FANCM%20pathogenic%20variants.%20Int.%20J.%20Cancer144,%202683%E2%80%932694%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Silvestri, V. et al. A possible role of FANCM mutations in male breast cancer susceptibility: Results from a multicenter study in Italy. <em>Breast</em><strong>38</strong>, 92–97 (2018).
</cite> [<a href="https://doi.org/10.1016/j.breast.2017.12.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29287190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Silvestri,%20V.%20et%20al.%20A%20possible%20role%20of%20FANCM%20mutations%20in%20male%20breast%20cancer%20susceptibility:%20Results%20from%20a%20multicenter%20study%20in%20Italy.%20Breast38,%2092%E2%80%9397%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Bogliolo, M. et al. Biallelic truncating FANCM mutations cause early-onset cancer but not Fanconi anemia. <em>Genet Med.</em><strong>20</strong>, 458–463 (2018).
</cite> [<a href="https://doi.org/10.1038/gim.2017.124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28837157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bogliolo,%20M.%20et%20al.%20Biallelic%20truncating%20FANCM%20mutations%20cause%20early-onset%20cancer%20but%20not%20Fanconi%20anemia.%20Genet%20Med.20,%20458%E2%80%93463%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Kraft, P. Curses–winner’s and otherwise–in genetic epidemiology. <em>Epidemiology</em><strong>19</strong>, 649–651 (2008.
</cite> [<a href="https://doi.org/10.1097/EDE.0b013e318181b865" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18703928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kraft,%20P.%20Curses%E2%80%93winner%E2%80%99s%20and%20otherwise%E2%80%93in%20genetic%20epidemiology.%20Epidemiology19,%20649%E2%80%93651%20(2008." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Zhong, H. &amp; Prentice, R. L. Bias-reduced estimators and confidence intervals for odds ratios in genome-wide association studies. <em>Biostatistics</em><strong>9</strong>, 621–634 (2008).
</cite> [<a href="https://doi.org/10.1093/biostatistics/kxn001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2536726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18310059/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhong,%20H.%20&amp;%20Prentice,%20R.%20L.%20Bias-reduced%20estimators%20and%20confidence%20intervals%20for%20odds%20ratios%20in%20genome-wide%20association%20studies.%20Biostatistics9,%20621%E2%80%93634%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Zhong, H. &amp; Prentice, R. L. Correcting “winner’s curse” in odds ratios from genomewide association findings for major complex human diseases. <em>Genet Epidemiol.</em><strong>34</strong>, 78–91 (2010).
</cite> [<a href="https://doi.org/10.1002/gepi.20437" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2796696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19639606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhong,%20H.%20&amp;%20Prentice,%20R.%20L.%20Correcting%20%E2%80%9Cwinner%E2%80%99s%20curse%E2%80%9D%20in%20odds%20ratios%20from%20genomewide%20association%20findings%20for%20major%20complex%20human%20diseases.%20Genet%20Epidemiol.34,%2078%E2%80%9391%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Fletcher, O. et al. Family history, genetic testing, and clinical risk prediction: pooled analysis of CHEK2 1100delC in 1,828 bilateral breast cancers and 7,030 controls. <em>Cancer Epidemiol. Biomark. Prev.</em><strong>18</strong>, 230–234 (2009).</cite> [<a href="https://doi.org/10.1158/1055-9965.EPI-08-0416" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2727696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19124502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fletcher,%20O.%20et%20al.%20Family%20history,%20genetic%20testing,%20and%20clinical%20risk%20prediction:%20pooled%20analysis%20of%20CHEK2%201100delC%20in%201,828%20bilateral%20breast%20cancers%20and%207,030%20controls.%20Cancer%20Epidemiol.%20Biomark.%20Prev.18,%20230%E2%80%93234%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Panday, A. et al. FANCM regulates repair pathway choice at stalled replication forks. <em>Mol. Cell</em><strong>81</strong>, 2428–2444.e2426 (2021).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2021.03.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8180084/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33882298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Panday,%20A.%20et%20al.%20FANCM%20regulates%20repair%20pathway%20choice%20at%20stalled%20replication%20forks.%20Mol.%20Cell81,%202428%E2%80%932444.e2426%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Blackford, A. N. et al. The DNA translocase activity of FANCM protects stalled replication forks. <em>Hum. Mol. Genet.</em><strong>21</strong>, 2005–2016 (2012).
</cite> [<a href="https://doi.org/10.1093/hmg/dds013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22279085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blackford,%20A.%20N.%20et%20al.%20The%20DNA%20translocase%20activity%20of%20FANCM%20protects%20stalled%20replication%20forks.%20Hum.%20Mol.%20Genet.21,%202005%E2%80%932016%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Michl, J., Zimmer, J. &amp; Tarsounas, M. Interplay between Fanconi anemia and homologous recombination pathways in genome integrity. <em>EMBO J.</em><strong>35</strong>, 909–923 (2016).
</cite> [<a href="https://doi.org/10.15252/embj.201693860" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4865030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27037238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Michl,%20J.,%20Zimmer,%20J.%20&amp;%20Tarsounas,%20M.%20Interplay%20between%20Fanconi%20anemia%20and%20homologous%20recombination%20pathways%20in%20genome%20integrity.%20EMBO%20J.35,%20909%E2%80%93923%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Couch, F. J. et al. Associations between cancer predisposition testing panel genes and breast cancer. <em>JAMA Oncol.</em><strong>3</strong>, 1190–1196 (2017).
</cite> [<a href="https://doi.org/10.1001/jamaoncol.2017.0424" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5599323/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28418444/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Couch,%20F.%20J.%20et%20al.%20Associations%20between%20cancer%20predisposition%20testing%20panel%20genes%20and%20breast%20cancer.%20JAMA%20Oncol.3,%201190%E2%80%931196%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Kurian, A. W. et al. Germline genetic testing after cancer diagnosis. <em>JAMA</em><strong>330</strong>, 43–51 (2023).
</cite> [<a href="https://doi.org/10.1001/jama.2023.9526" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10242510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37276540/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kurian,%20A.%20W.%20et%20al.%20Germline%20genetic%20testing%20after%20cancer%20diagnosis.%20JAMA330,%2043%E2%80%9351%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Gierach, G. L., Burke, A. &amp; Anderson, W. F. Epidemiology of triple negative breast cancers. <em>Breast Dis.</em><strong>32</strong>, 5–24 (2010).
</cite> [<a href="https://doi.org/10.3233/BD-2010-0319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4684941/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21965309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gierach,%20G.%20L.,%20Burke,%20A.%20&amp;%20Anderson,%20W.%20F.%20Epidemiology%20of%20triple%20negative%20breast%20cancers.%20Breast%20Dis.32,%205%E2%80%9324%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Stoepker, C. et al. DNA helicases FANCM and DDX11 are determinants of PARP inhibitor sensitivity. <em>DNA Repair</em><strong>26</strong>, 54–64 (2015).
</cite> [<a href="https://doi.org/10.1016/j.dnarep.2014.12.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25583207/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stoepker,%20C.%20et%20al.%20DNA%20helicases%20FANCM%20and%20DDX11%20are%20determinants%20of%20PARP%20inhibitor%20sensitivity.%20DNA%20Repair26,%2054%E2%80%9364%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Gilliland, F. D. et al. Reproductive risk factors for breast cancer in Hispanic and non-Hispanic white women: the New Mexico Women’s Health Study. <em>Am. J. Epidemiol.</em><strong>148</strong>, 683–692 (1998).
</cite> [<a href="https://doi.org/10.1093/aje/148.7.683" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9778175/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gilliland,%20F.%20D.%20et%20al.%20Reproductive%20risk%20factors%20for%20breast%20cancer%20in%20Hispanic%20and%20non-Hispanic%20white%20women:%20the%20New%20Mexico%20Women%E2%80%99s%20Health%20Study.%20Am.%20J.%20Epidemiol.148,%20683%E2%80%93692%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Perez-Stable, E. J., Marin, G. &amp; Marin, B. V. Behavioral risk factors: a comparison of Latinos and non-Latino whites in San Francisco. <em>Am. J. Public Health</em><strong>84</strong>, 971–976 (1994).
</cite> [<a href="https://doi.org/10.2105/ajph.84.6.971" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1614972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8203695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perez-Stable,%20E.%20J.,%20Marin,%20G.%20&amp;%20Marin,%20B.%20V.%20Behavioral%20risk%20factors:%20a%20comparison%20of%20Latinos%20and%20non-Latino%20whites%20in%20San%20Francisco.%20Am.%20J.%20Public%20Health84,%20971%E2%80%93976%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Pike, M. C. et al. Breast cancer in a multiethnic cohort in Hawaii and Los Angeles: risk factor-adjusted incidence in Japanese equals and in Hawaiians exceeds that in whites. <em>Cancer Epidemiol. Biomark. Prev.</em><strong>11</strong>, 795–800 (2002).</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/12223421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pike,%20M.%20C.%20et%20al.%20Breast%20cancer%20in%20a%20multiethnic%20cohort%20in%20Hawaii%20and%20Los%20Angeles:%20risk%20factor-adjusted%20incidence%20in%20Japanese%20equals%20and%20in%20Hawaiians%20exceeds%20that%20in%20whites.%20Cancer%20Epidemiol.%20Biomark.%20Prev.11,%20795%E2%80%93800%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Martin, A. R. et al. Clinical use of current polygenic risk scores may exacerbate health disparities. <em>Nat. Genet.</em><strong>51</strong>, 584–591 (2019).
</cite> [<a href="https://doi.org/10.1038/s41588-019-0379-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6563838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30926966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martin,%20A.%20R.%20et%20al.%20Clinical%20use%20of%20current%20polygenic%20risk%20scores%20may%20exacerbate%20health%20disparities.%20Nat.%20Genet.51,%20584%E2%80%93591%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Fejerman, L. et al. Genetic ancestry and risk of breast cancer among U.S. Latinas. <em>Cancer Res.</em><strong>68</strong>, 9723–9728 (2008).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-08-2039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2674787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19047150/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fejerman,%20L.%20et%20al.%20Genetic%20ancestry%20and%20risk%20of%20breast%20cancer%20among%20U.S.%20Latinas.%20Cancer%20Res.68,%209723%E2%80%939728%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Fejerman, L. et al. European ancestry is positively associated with breast cancer risk in Mexican women. <em>Cancer Epidemiol. Biomark. Prev.</em><strong>19</strong>, 1074–1082 (2010).</cite> [<a href="https://doi.org/10.1158/1055-9965.EPI-09-1193" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2852491/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20332279/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fejerman,%20L.%20et%20al.%20European%20ancestry%20is%20positively%20associated%20with%20breast%20cancer%20risk%20in%20Mexican%20women.%20Cancer%20Epidemiol.%20Biomark.%20Prev.19,%201074%E2%80%931082%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Fejerman, L. et al. Admixture mapping identifies a locus on 6q25 associated with breast cancer risk in US Latinas. <em>Hum. Mol. Genet.</em><strong>21</strong>, 1907–1917 (2012).
</cite> [<a href="https://doi.org/10.1093/hmg/ddr617" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3313799/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22228098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fejerman,%20L.%20et%20al.%20Admixture%20mapping%20identifies%20a%20locus%20on%206q25%20associated%20with%20breast%20cancer%20risk%20in%20US%20Latinas.%20Hum.%20Mol.%20Genet.21,%201907%E2%80%931917%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Wilcox, N. et al. Exome sequencing identifies breast cancer susceptibility genes and defines the contribution of coding variants to breast cancer risk. <em>Nat. Genet.</em><strong>55</strong>, 1435–1439 (2023).
</cite> [<a href="https://doi.org/10.1038/s41588-023-01466-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10484782/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37592023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilcox,%20N.%20et%20al.%20Exome%20sequencing%20identifies%20breast%20cancer%20susceptibility%20genes%20and%20defines%20the%20contribution%20of%20coding%20variants%20to%20breast%20cancer%20risk.%20Nat.%20Genet.55,%201435%E2%80%931439%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Li, H. &amp; Durbin, R. Fast and accurate long-read alignment with Burrows-Wheeler transform. <em>Bioinformatics</em><strong>26</strong>, 589–595 (2010).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2828108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20080505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20H.%20&amp;%20Durbin,%20R.%20Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows-Wheeler%20transform.%20Bioinformatics26,%20589%E2%80%93595%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Li, H. et al. The Sequence Alignment/Map format and SAMtools. <em>Bioinformatics</em><strong>25</strong>, 2078–2079 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2723002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20H.%20et%20al.%20The%20Sequence%20Alignment/Map%20format%20and%20SAMtools.%20Bioinformatics25,%202078%E2%80%932079%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Wang, K., Li, M. &amp; Hakonarson, H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. <em>Nucleic Acids Res.</em><strong>38</strong>, e164 (2010).
</cite> [<a href="https://doi.org/10.1093/nar/gkq603" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2938201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20601685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20K.,%20Li,%20M.%20&amp;%20Hakonarson,%20H.%20ANNOVAR:%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data.%20Nucleic%20Acids%20Res.38,%20e164%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Galanter, J. M. et al. Development of a panel of genome-wide ancestry informative markers to study admixture throughout the Americas. <em>PLoS Genet.</em><strong>8</strong>, e1002554 (2012).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1002554" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3297575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22412386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Galanter,%20J.%20M.%20et%20al.%20Development%20of%20a%20panel%20of%20genome-wide%20ancestry%20informative%20markers%20to%20study%20admixture%20throughout%20the%20Americas.%20PLoS%20Genet.8,%20e1002554%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. <em>Gigascience</em><strong>4</strong>, 7 (2015).
</cite> [<a href="https://doi.org/10.1186/s13742-015-0047-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4342193/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25722852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chang,%20C.%20C.%20et%20al.%20Second-generation%20PLINK:%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets.%20Gigascience4,%207%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Alexander, D. H., Novembre, J. &amp; Lange, K. Fast model-based estimation of ancestry in unrelated individuals. <em>Genome Res.</em><strong>19</strong>, 1655–1664 (2009).
</cite> [<a href="https://doi.org/10.1101/gr.094052.109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2752134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19648217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alexander,%20D.%20H.,%20Novembre,%20J.%20&amp;%20Lange,%20K.%20Fast%20model-based%20estimation%20of%20ancestry%20in%20unrelated%20individuals.%20Genome%20Res.19,%201655%E2%80%931664%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Landrum, M. J. et al. ClinVar: improving access to variant interpretations and supporting evidence. <em>Nucleic Acids Res.</em><strong>46</strong>, D1062–D1067 (2018).
</cite> [<a href="https://doi.org/10.1093/nar/gkx1153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5753237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29165669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Landrum,%20M.%20J.%20et%20al.%20ClinVar:%20improving%20access%20to%20variant%20interpretations%20and%20supporting%20evidence.%20Nucleic%20Acids%20Res.46,%20D1062%E2%80%93D1067%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Lee, S. et al. Optimal unified approach for rare-variant association testing with application to small-sample case-control whole-exome sequencing studies. <em>Am. J. Hum. Genet.</em><strong>91</strong>, 224–237 (2012).
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2012.06.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3415556/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22863193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20S.%20et%20al.%20Optimal%20unified%20approach%20for%20rare-variant%20association%20testing%20with%20application%20to%20small-sample%20case-control%20whole-exome%20sequencing%20studies.%20Am.%20J.%20Hum.%20Genet.91,%20224%E2%80%93237%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Lee, S., Wu, M. C. &amp; Lin, X. Optimal tests for rare variant effects in sequencing association studies. <em>Biostatistics</em><strong>13</strong>, 762–775 (2012).
</cite> [<a href="https://doi.org/10.1093/biostatistics/kxs014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3440237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22699862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20S.,%20Wu,%20M.%20C.%20&amp;%20Lin,%20X.%20Optimal%20tests%20for%20rare%20variant%20effects%20in%20sequencing%20association%20studies.%20Biostatistics13,%20762%E2%80%93775%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (1.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Peer Review File</a><sup> (371.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_60564_MOESM3_ESM.pdf</a><sup> (100.5KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (19.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (488.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (16.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting summary</a><sup> (2.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370925/bin/41467_2025_60564_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (311KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Source data are provided with this manuscript. The genotype data and raw sequencing readsgenerated in this study have been deposited in the dbGAP database and are available at phs003144.v1.p1 [<a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs003144.v1.p1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs003144.v1.p1</a>]. The genotype data are available under restricted access due to the informed consent under which the sample were obtained. Access can be obtained be obtained by applying with a scientific proposal to dbGAP. <a href="#Sec12" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-60564-0"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_60564.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (740.5 KB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370925/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370925/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370925%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370925/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370925/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370925/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841357/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370925/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841357/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370925/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370925/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="8wKTI79bs06b4uub4CcN4nvj8YO5MV1jRKI2FPoi60xNNDfZ6M9V6Iacw8ZhD1uN">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
