
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Integrating computational approaches to uncover β-lactamase-associated resistance in diarrheagenic Escherichia coli from pediatric patients - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A5778AF2305305A57700061EEDE2.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373996/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Integrating computational approaches to uncover β-lactamase-associated resistance in diarrheagenic Escherichia coli from pediatric patients">
<meta name="citation_author" content="Taru Singh">
<meta name="citation_author_institution" content="Department of Microbiology, University College of Medical Sciences, University of Delhi) &amp; GTB Hospital, Delhi, India">
<meta name="citation_author_institution" content="Amity Institute of Biotechnology, Amity University, Noida, Uttar Pradesh India">
<meta name="citation_author_institution" content="Division of Descriptive Research, Indian Council of Medical Research, New Delhi, India">
<meta name="citation_author" content="Chandra Shekhar">
<meta name="citation_author_institution" content="Department of Microbiology, University College of Medical Sciences, University of Delhi) &amp; GTB Hospital, Delhi, India">
<meta name="citation_author" content="Saurabh Singh">
<meta name="citation_author_institution" content="Department of Mechanical Engineering, Delhi Technological University, Delhi, India">
<meta name="citation_author" content="Shukla Das">
<meta name="citation_author_institution" content="Department of Microbiology, University College of Medical Sciences, University of Delhi) &amp; GTB Hospital, Delhi, India">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30921">
<meta name="citation_doi" content="10.1038/s41598-025-16122-1">
<meta name="citation_pmid" content="40846736">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373996/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373996/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373996/pdf/41598_2025_Article_16122.pdf">
<meta name="description" content="Multidrug-resistant diarrheagenic Escherichia coli (MDR-DEC) increasingly complicates treatment of pediatric infections, largely due to β-lactamase-mediated resistance. Despite their clinical relevance, the evolutionary diversity, functional ...">
<meta name="og:title" content="Integrating computational approaches to uncover β-lactamase-associated resistance in diarrheagenic Escherichia coli from pediatric patients">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Multidrug-resistant diarrheagenic Escherichia coli (MDR-DEC) increasingly complicates treatment of pediatric infections, largely due to β-lactamase-mediated resistance. Despite their clinical relevance, the evolutionary diversity, functional ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373996/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373996">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-16122-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_16122.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373996%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373996/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373996/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373996/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 22;15:30921. doi: <a href="https://doi.org/10.1038/s41598-025-16122-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-16122-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Integrating computational approaches to uncover β-lactamase-associated resistance in diarrheagenic <em>Escherichia coli</em> from pediatric patients</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Taru Singh</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Taru Singh</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, University College of Medical Sciences, University of Delhi) &amp; GTB Hospital, Delhi, India </div>
<div class="p">
<sup>2</sup>Amity Institute of Biotechnology, Amity University, Noida, Uttar Pradesh India </div>
<div class="p">
<sup>3</sup>Division of Descriptive Research, Indian Council of Medical Research, New Delhi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Taru Singh</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shekhar%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Chandra Shekhar</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Chandra Shekhar</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, University College of Medical Sciences, University of Delhi) &amp; GTB Hospital, Delhi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shekhar%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chandra Shekhar</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Saurabh Singh</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Saurabh Singh</span></h3>
<div class="p">
<sup>4</sup>Department of Mechanical Engineering, Delhi Technological University, Delhi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Saurabh Singh</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Das%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Shukla Das</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Shukla Das</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, University College of Medical Sciences, University of Delhi) &amp; GTB Hospital, Delhi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Das%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shukla Das</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Microbiology, University College of Medical Sciences, University of Delhi) &amp; GTB Hospital, Delhi, India </div>
<div id="Aff2">
<sup>2</sup>Amity Institute of Biotechnology, Amity University, Noida, Uttar Pradesh India </div>
<div id="Aff3">
<sup>3</sup>Division of Descriptive Research, Indian Council of Medical Research, New Delhi, India </div>
<div id="Aff4">
<sup>4</sup>Department of Mechanical Engineering, Delhi Technological University, Delhi, India </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 12; Accepted 2025 Aug 13; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373996  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846736/" class="usa-link">40846736</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par19">Multidrug-resistant diarrheagenic <em>Escherichia coli</em> (MDR-DEC) increasingly complicates treatment of pediatric infections, largely due to β-lactamase-mediated resistance. Despite their clinical relevance, the evolutionary diversity, functional mechanisms, and therapeutic targeting of β-lactamase enzymes in DEC remain poorly defined. This study addresses that gap by evaluating the prevalence of β-lactamase genes in DEC isolates and exploring their molecular interactions with key antibiotics. This study aims to detect and characterize β-lactamase-producing diarrheagenic <em>Escherichia coli</em> (DEC) isolates from children suffering from diarrhea, and to investigate the underlying drug-target interactions contributing to antimicrobial resistance. A total of 120 <em>E. coli</em> isolates were obtained from pediatric diarrheal, non-diarrheal, and healthy groups. Molecular screening for β-lactamase genes was performed using RT-PCR. Structural modeling, phylogenetic analysis, and molecular docking were used to evaluate evolutionary patterns and drug interactions with ceftriaxone and amoxicillin. β-lactamase enzymes in DEC strains exhibited varying thermal stability and were frequently co-produced. Molecular docking revealed that while amoxicillin had stronger binding affinity for certain enzymes, ceftriaxone demonstrated greater inhibitory potency across key targets. Resistance genes such as <em>bla_TEM</em>, <em>bla_SHV</em>, and <em>bla_CTX-M</em> were central in the resistance network. Additional mechanisms, including efflux pump activity and DNA repair pathways, contributed to the multidrug-resistant phenotype. This study highlights the multifactorial resistance landscape of DEC, driven by diverse β-lactamase variants and supporting mechanisms such as efflux and DNA repair. The differential drug interaction profiles of amoxicillin and ceftriaxone underscore the importance of structure-guided antibiotic selection. These insights may inform more targeted therapeutic strategies and support molecular surveillance efforts to monitor and combat MDR-DEC in pediatric settings.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-16122-1.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Multidrug resistance, Diarrheagenic <em>Escherichia coli</em>, Amoxicillin, Ceftriaxone, β-lactamases</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Microbiology, Medical research</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par20">The rise of β-lactamase-producing multidrug-resistant (MDR) <em>Escherichia coli</em> strains is a significant public health concern, particularly in vulnerable populations such as children under five years of age<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. <em>E. coli</em> is a leading cause of diarrheal diseases in children, and the emergence of strains that produce β-lactamase enzymes has severely limited treatment options, making these infections more difficult to manage<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. lactamases enzymes, including extended spectrum β-lactamases (ESBLs), metallo-β-lactamases (MβLs), and AmpC β-lactamases, confer resistance to a wide range of β-lactam antibiotics, such as penicillin’s and cephalosporins, by hydrolyzing the β-lactam ring, thereby neutralizing the efficacy of the antibiotics<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>.</p>
<p id="Par21">In recent years, there has been a notable increase in the prevalence of MDR diarrheagenic <em>E. coli</em> (DEC) in children, complicating treatment strategies and posing a significant threat to child health<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. The β-lactamase-producing strains are of particular concern because they not only resist standard treatments but also often carry additional resistance genes on plasmids, which can spread rapidly through bacterial populations through horizontal gene transfer (HZT)<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. This situation is exacerbated in pediatric populations, where therapeutic options are limited due to the need for careful dosing and the restricted use of certain antibiotics<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>.</p>
<p id="Par22">Amoxicillin, a commonly used penicillin derivative, and ceftriaxone, a third-generation cephalosporin, are often prescribed for treating bacterial infections in children due to their broad-spectrum activity and safety profiles<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. However, the effectiveness of these antibiotics is increasingly compromised by the presence of β-lactamase-producing DEC, which degrade these drugs before they can effectively eliminate the infection<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. The ability of these pathogens to resist amoxicillin and ceftriaxone is particularly concerning in the context of diarrheal diseases, which remain a leading cause of morbidity and mortality in young children globally<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>.</p>
<p id="Par23">The broad clinical use and varied activity spectra of amoxicillin and ceftriaxone make them penicillin and cephalosporins, respectively<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Unlike other antibiotics, these drugs are frequently targeted by key β-lactamase enzymes, making them ideal for studying resistance mechanisms<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Amoxicillin and ceftriaxone are frequently targeted by β-lactamase enzymes due to their widespread use and structural vulnerability. Amoxicillin is easily hydrolyzed by common β-lactamases, while ceftriaxone is a key target of ESBLs like CTX-M. Their frequent use, especially without β-lactamase inhibitors, creates strong selective pressure, promoting resistance. Additionally, their distinct chemical structures allow for a comparative analysis of how structural differences influence binding and resistance, providing broader insights that might be gained from other antibiotics<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>.</p>
<p id="Par24">This study is relevant for detecting and characterizing β-lactamase-producing diarrheagenic <em>Escherichia coli</em> (DEC) isolates from children with diarrhea, as well as for exploring the underlying drug-target interactions that contribute to antimicrobial resistance. By employing molecular docking studies to analyze the interactions between β-lactamase enzymes and key antibiotics such as amoxicillin and ceftriaxone, this research aims to uncover the structural features that contribute to drug resistance. The study also correlates these molecular interactions with the broader MDR mechanisms, providing a comprehensive understanding of how these enzymes play a central role in the resistance network.</p>
<p id="Par25">This research is crucial for developing targeted interventions to combat MDR <em>E. coli</em> infections in children, particularly in resource-limited settings where the burden of diarrheal diseases is highest<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Understanding the specific mechanisms of resistance in this age group will help in designing better therapeutic approaches and in implementing more effective antibiotic stewardship programs to preserve the efficacy of existing drugs. Ultimately, this study aims to contribute to reducing the incidence and severity of diarrheal diseases caused by MDR DEC in young children, thereby improving health outcomes in this vulnerable population.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Study group</h3>
<p id="Par26">The study population consisted of three groups of children under 5 years of age. Group 1 included 40 children with acute diarrhea who were not receiving any antibiotics. Group 2 comprised 40 hospitalized children receiving antibiotic (oral/I/v) for reasons other than diarrhea admitted to NICU/Ward for the past 72-hour duration (to distinguish the impact of antibiotic exposure from the presence of diarrhea itself on the development, spread and pattern of antimicrobial resistance). Group 3 consisted of 40 healthy children who were not suffering from diarrhea or any other disease. While children with known immunological disorders, chronic gastrointestinal disorders and receiving/received probiotic for the last seven days were excluded from the study. The samples were collected for a period of two years from June 2014 to June 2016.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Ethics</h3>
<p id="Par27">Stool samples were collected after obtaining informed consent from the parents or guardians of the participants. The study was approved by the “institutional ethics committee—human research (IEC-HR) university college of medical sciences, University of Delhi, New Delhi in accordance with WHO guidelines.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Processing of samples</h3>
<p id="Par28">Fecal samples were processed using standard biochemical tests for distinguishing <em>E. coli</em> from other enteric bacteria<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Antimicrobial susceptibility testing was conducted using the Kirby-Bauer disc diffusion method for β-lactamase production using a phenotypic method<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, and subsequently confirmed by quantitative polymerase chain reaction (qPCR) following DNA extraction. The list of primers used for β-lactamase production in this study is presented in Table <a href="#Tab1" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>List of primers for beta-lactamase genes used in this study.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Gene</th>
<th align="left" colspan="1" rowspan="1">Primer Sequence (5’−3’)</th>
<th align="left" colspan="1" rowspan="1">References</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="2" colspan="1">
<em>TEM</em>
</td>
<td align="left" colspan="1" rowspan="1">AGTGCTGCCATAACCATGAGG</td>
<td align="left" rowspan="8" colspan="1">
<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>
</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1">CTGACTCCCCGTCGTGTAGATA</td></tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<em>SHV</em>
</td>
<td align="left" colspan="1" rowspan="1">GATGAACGCTTTCCCATGATG</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1">CGCTGTTATCGCTCATGGTAA</td></tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<em>CTX-M</em>
</td>
<td align="left" colspan="1" rowspan="1">GACAAAGAGAGTGCAACGGATG</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1">TCAGTGCGATCCAGACGAAA</td></tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<em>OXA</em>
</td>
<td align="left" colspan="1" rowspan="1">ATTATCTACAGCAGCGCCAGTG</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1">TGCATCCACGTCTTTGGTG</td></tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<em>NDM-1</em>
</td>
<td align="left" colspan="1" rowspan="1">ATTAGCCGCTGCATTGAT</td>
<td align="left" rowspan="2" colspan="1">
<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>
</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1">CATGTCGAGATAGGAAGTG</td></tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<em>IMP</em>
</td>
<td align="left" colspan="1" rowspan="1">TTGACACTCCATTTACAG</td>
<td align="left" rowspan="4" colspan="1">
<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>
</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1">GATTGAGAATTAAGCCACTCT</td></tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<em>VIM</em>
</td>
<td align="left" colspan="1" rowspan="1">GATGGTGTTTGGTCGCATA</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1">CGAATGCGCAGCACCAG</td></tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<em>CMY</em>
</td>
<td align="left" colspan="1" rowspan="1">GCTGCTCAAGGAGCACAGGAT</td>
<td align="left" rowspan="6" colspan="1">
<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>
</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1">CACATTGACATAGGTGTGGTGC</td></tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<em>DHA</em>
</td>
<td align="left" colspan="1" rowspan="1">AACTTTCACAGGTGTGCTGGGT</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1">CCGTACGCATACTGGCTTTGC</td></tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<em>ACT-1</em>
</td>
<td align="left" colspan="1" rowspan="1">TCGGTAAAG CCGATGTTG CGG</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1">CTT CCA CTG CGG CTG CCA GTT</td></tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec6"><h3 class="pmc_sec_title">Quantitative PCR (qPCR)</h3>
<p id="Par30">SYBR Green-based qPCR was conducted in a 96-well white opaque plate (Roche), with each well containing a total volume of 20 µL. The reaction mixture included 10 µL of SYBR Green I master mix (2X), 1 µL of each primer (forward and reverse), and 3 µL of DNA. DNA amplification was carried out using the Roche LightCycler 480-II, starting with a pre-incubation step at 94 °C for 7 min, followed by 35 cycles of amplification with an annealing at 60 °C for 30 s, and concluding with cooling at 40 °C.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Sequencing</h3>
<p id="Par31">DNA sequencing of selected PCR products was performed using the same set of primers mentioned in our previous studies to verify the authenticity of the gene<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. The sequencing was conducted at Helix Biosciences in Delhi. The sequences have been deposited in genbank, and accession numbers were obtained.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Molecular docking</h3>
<p id="Par32">To predict the binding poses of the selected β-lactamase gene products with drugs exhibiting minimal resistance or high sensitivity, molecular docking was conducted using AutoDock 4.2 and analyzed with Discovery Studio Visualizer 2021, following the general approach described by Reddy et al.<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. The 3D structures of the target drugs were retrieved from the PubChem database<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, and ligand preparation was carried out using the LigPrep module (version 2.8, Schrödinger Suite)<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>, with energy minimization applied via the OPLS_2005 force field to generate low-energy conformers.</p>
<p id="Par33">The β-lactamase protein structures were prepared by removing water molecules, adding polar hydrogens, and assigning Kollman charges, following standard AutoDock protocols<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. The docking grid was centered on active site residues identified through structural analysis, with dimensions set to 60 × 60 × 60 Å and a grid spacing of 0.375 Å, consistent with established docking procedures<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Docking was executed using AutoDock’s Lamarckian Genetic Algorithm, with 100 independent runs, a population size of 150, 2.5 million energy evaluations per run, and mutation and crossover rates of 0.02 and 0.8, respectively<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Binding poses were ranked by binding energy, and the top-scoring conformations were selected for analysis.</p>
<p id="Par34">While docking scores offer a useful approximation of ligand–receptor interaction strength, they are limited by simplified scoring functions that neglect key thermodynamic factors, such as enthalpy and entropy<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Therefore, docking in this study was primarily used for posing prediction and comparative assessment.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Comprehensive genomic and functional analysis</h3>
<p id="Par35">To perform a comprehensive functional analysis, genes associated with MDR-DEC were retrieved from the NCBI database<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Various analyses were then conducted using these sequences. Gene interaction network analysis was carried out using STRING<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> and Cytoscape software<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> to elucidate the interactions and functional relationships between these genes. For functional annotation, gene ontology (GO) enrichment analysis<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> was performed to determine the biological processes, cellular components, and molecular functions associated with the identified genes. Additionally, KEGG pathway analysis was utilized to map these genes into relevant biochemical pathways, providing insights into their roles in metabolic and signaling pathways<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. For phylogenetic analysis, multiple sequence alignments of the identified genes were conducted using Clustal W, and phylogenetic trees were constructed using MEGA software<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> to explore evolutionary relationships and divergence among the gene sequences. These combined analyses provided a comprehensive understanding of the genetic basis of MDR in DEG.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par36">Statistical analysis was performed to compare the prevalence of MDR-DEG among the three study groups. Data from antimicrobial susceptibility testing and β-lactamase production assays were analyzed using the Fisher’s exact test to determine significant differences in resistance patterns between groups. For continuous variables, such as zone of inhibition measurements and temperature stability values, one-way ANOVA followed by Tukey’s post-hoc test was employed to assess group-wise differences. The qPCR data, including gene expression levels, were analyzed using the 2^−ΔΔCt method for relative quantification. Differences in gene expression levels among groups were assessed using one-way ANOVA followed by Bonferroni’s post-hoc test.</p>
<p id="Par37">To evaluate potential predictors associated with multidrug resistance, multiple logistic regression analysis was conducted using SPSS version 25.0, adjusting for relevant clinical and demographic covariates. Phylogenetic analysis was performed using the Maximum Likelihood (ML) method with bootstrap analysis (1,000 replicates) to assess the statistical robustness of tree topologies. Gene interaction network analysis, along with Gene Ontology (GO) and KEGG pathway enrichment analysis<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, was statistically evaluated using hypergeometric tests and corrected for multiple testing using the Benjamini-Hochberg false discovery rate (FDR) method to control for type I error. All statistical analyses were performed using SPSS software version 25.0 or relevant bioinformatics tools, and a p-value of &lt; 0.05 was considered statistically significant.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Thermal stability of different β-lactamase genes: qPCR melting curve analysis</h3>
<p id="Par38">The average melting temperature (<em>T</em><sub><em>m</em></sub>) for each tested β-lactamase gene is presented in Fig. <a href="#Fig1" class="usa-link">1</a>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/0dd3a59e2d17/41598_2025_16122_Fig1_HTML.jpg" loading="lazy" id="d33e650" height="374" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>qPCR results for β-lactamase genes, derived from amplified melting peaks of fluorescence (-(d/dT)) at a wavelength of 483–533 nm. The x-axis represents the genes, and the y-axis represents the average melting temperatures (°C) with error bars indicating the standard deviation.</p></figcaption></figure><p id="Par40">The <em>OXA</em> gene had the highest average melting temperature at 90 °C, indicating it may have the most stable structure under thermal conditions. The <em>SHV</em> gene had the lowest average melting temperature at 81.92 °C, suggesting it may be the least stable under thermal conditions. Most genes exhibited average melting temperatures around the mid to high 80 s °C, with varying degrees of standard deviations, reflecting different levels of structural stability. The standard deviations were relatively small for most genes, indicating consistent melting temperature measurements across replicates.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Distribution of β-lactamase production</h3>
<p id="Par41">A comparative distribution of β-lactamase production (<em>ESBL</em>, <em>MBL</em>, and <em>AmpC</em>) by phenotypic and genotypic methods is depicted in Table <a href="#Tab2" class="usa-link">2</a>. Among the total isolates, 76 were identified as ESBL producers (27 from Group 1, 27 from Group 2, and 22 from Group 3). The most prevalent ESBL genes were <em>TEM</em> (40.83%) and <em>SHV</em> (32.5%), followed by <em>CTX-M</em> (19.16%) and <em>OXA</em> (19.16%). Overall, while co-production of specific β-lactamase genes was common, the complete co-occurrence of all relevant <em>ESBL</em>, <em>MBL</em>, or <em>AmpC</em> genes in a single isolate was not observed, indicating distinct patterns of resistance gene distribution within the bacterial populations studied (Table <a href="#Tab2" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Distribution of -lactamase genes.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">
<strong>β-lactamase</strong>
</th>
<th align="left" colspan="1" rowspan="1">
<strong>Genes</strong>
</th>
<th align="left" colspan="1" rowspan="1">
<strong>Group1 (n = 40)</strong>
</th>
<th align="left" colspan="1" rowspan="1">
<strong>Group-2 (n = 40)</strong>
</th>
<th align="left" colspan="1" rowspan="1">
<strong>Group-3 (n = 40)</strong>
</th>
<th align="left" colspan="1" rowspan="1">
<strong><em>p-</em></strong>
<strong>value</strong>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="4" colspan="1">ESBL</td>
<td align="left" colspan="1" rowspan="1">
<em>TEM</em>
</td>
<td align="left" colspan="1" rowspan="1">19 (47.5)</td>
<td align="left" colspan="1" rowspan="1">14 (35)</td>
<td align="left" colspan="1" rowspan="1">16 (40)</td>
<td align="left" colspan="1" rowspan="1">0.519</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>SHV</em>
</td>
<td align="left" colspan="1" rowspan="1">14 (35)</td>
<td align="left" colspan="1" rowspan="1">14 (35)</td>
<td align="left" colspan="1" rowspan="1">11 (27.5)</td>
<td align="left" colspan="1" rowspan="1">0.710</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>CTX-M</em>
</td>
<td align="left" colspan="1" rowspan="1">7 (17.5)</td>
<td align="left" colspan="1" rowspan="1">8 (20)</td>
<td align="left" colspan="1" rowspan="1">8 (20)</td>
<td align="left" colspan="1" rowspan="1">0.947</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>OXA</em>
</td>
<td align="left" colspan="1" rowspan="1">9 (22.5)</td>
<td align="left" colspan="1" rowspan="1">7 (17.5)</td>
<td align="left" colspan="1" rowspan="1">7 (17.5)</td>
<td align="left" colspan="1" rowspan="1">0.806</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">MBL</td>
<td align="left" colspan="1" rowspan="1">
<em>NDM-1</em>
</td>
<td align="left" colspan="1" rowspan="1">11 (27.5)</td>
<td align="left" colspan="1" rowspan="1">5 (12.5)</td>
<td align="left" colspan="1" rowspan="1">6 (15)</td>
<td align="left" colspan="1" rowspan="1">0.178</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>IMP</em>
</td>
<td align="left" colspan="1" rowspan="1">12 (32.5)</td>
<td align="left" colspan="1" rowspan="1">5 (12.5)</td>
<td align="left" colspan="1" rowspan="1">6 (15)</td>
<td align="left" colspan="1" rowspan="1">0.098</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>VIM</em>
</td>
<td align="left" colspan="1" rowspan="1">8 (20)</td>
<td align="left" colspan="1" rowspan="1">7 (17.5)</td>
<td align="left" colspan="1" rowspan="1">7 (17.5)</td>
<td align="left" colspan="1" rowspan="1">0.945</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">AmpC</td>
<td align="left" colspan="1" rowspan="1">
<em>ACT</em>
</td>
<td align="left" colspan="1" rowspan="1">6 (15)</td>
<td align="left" colspan="1" rowspan="1">8 (20)</td>
<td align="left" colspan="1" rowspan="1">6 (15)</td>
<td align="left" colspan="1" rowspan="1">0.786</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>DHA</em>
</td>
<td align="left" colspan="1" rowspan="1">4 (10)</td>
<td align="left" colspan="1" rowspan="1">2 (5)</td>
<td align="left" colspan="1" rowspan="1">3 (7.5)</td>
<td align="left" colspan="1" rowspan="1">0.697</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>CMY</em>
</td>
<td align="left" colspan="1" rowspan="1">11 (27.5)</td>
<td align="left" colspan="1" rowspan="1">4 (10)</td>
<td align="left" colspan="1" rowspan="1">3 (7.5)</td>
<td align="left" colspan="1" rowspan="1">0.024*</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p29"><p><strong>*</strong>Statistically significant; Gene frequencies are presented as absolute numbers with percentage in parentheses.</p></div></div></section></section><section id="Sec14"><h3 class="pmc_sec_title">Prevalence and co-production patterns of β-lactamase enzymes</h3>
<p id="Par43">The distribution of β-lactamase production either alone or in combination within isolates is depicted in Table <a href="#Tab3" class="usa-link">3</a>. Out of 120 clinical isolates examined, 99 (82.5%) were positive for β-lactamase production, indicating a high prevalence of resistance mechanisms within the sample population. Group 1 showed the highest overall rates of β-lactamase co-production, suggesting a significant burden of MDR strains within this group. These findings highlight the complexity and diversity of resistance mechanisms in clinical isolates, underlining the challenges in treating infections caused by these MDR bacteria<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Distribution of β-lactamase production alone or in combination in positive isolates.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Group</th>
<th align="left" colspan="1" rowspan="1">β-lactamase positive</th>
<th align="left" colspan="1" rowspan="1">ESBL</th>
<th align="left" colspan="1" rowspan="1">MBL</th>
<th align="left" colspan="1" rowspan="1">AmpC</th>
<th align="left" colspan="1" rowspan="1">ESBL + MBL</th>
<th align="left" colspan="1" rowspan="1">ESBL + AmpC</th>
<th align="left" colspan="1" rowspan="1">MBL + AmpC</th>
<th align="left" colspan="1" rowspan="1">ESBL + MBL + AmpC</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">1 (<em>n</em> = 40)</td>
<td align="left" colspan="1" rowspan="1">33</td>
<td align="left" colspan="1" rowspan="1">27 (67.5)</td>
<td align="left" colspan="1" rowspan="1">20 (50)</td>
<td align="left" colspan="1" rowspan="1">13 (32.5)</td>
<td align="left" colspan="1" rowspan="1">1 (2.5)</td>
<td align="left" colspan="1" rowspan="1">10 (25)</td>
<td align="left" colspan="1" rowspan="1">6 (15)</td>
<td align="left" colspan="1" rowspan="1">2 (5)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2 (<em>n</em> = 40)</td>
<td align="left" colspan="1" rowspan="1">31</td>
<td align="left" colspan="1" rowspan="1">27 (67.5)</td>
<td align="left" colspan="1" rowspan="1">13 (32.5)</td>
<td align="left" colspan="1" rowspan="1">7 (17.5)</td>
<td align="left" colspan="1" rowspan="1">6 (15)</td>
<td align="left" colspan="1" rowspan="1">3 (7.5)</td>
<td align="left" colspan="1" rowspan="1">2 (5)</td>
<td align="left" colspan="1" rowspan="1">1 (2.5)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3 (<em>n</em> = 40)</td>
<td align="left" colspan="1" rowspan="1">35</td>
<td align="left" colspan="1" rowspan="1">22 (55)</td>
<td align="left" colspan="1" rowspan="1">10 (25)</td>
<td align="left" colspan="1" rowspan="1">9 (22.5)</td>
<td align="left" colspan="1" rowspan="1">5 (12.5)</td>
<td align="left" colspan="1" rowspan="1">7 (17.5)</td>
<td align="left" colspan="1" rowspan="1">2 (5)</td>
<td align="left" colspan="1" rowspan="1">1 (2.5)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total</td>
<td align="left" colspan="1" rowspan="1">99</td>
<td align="left" colspan="1" rowspan="1">76 (63.3)</td>
<td align="left" colspan="1" rowspan="1">43 (35.83)</td>
<td align="left" colspan="1" rowspan="1">29 (24.16)</td>
<td align="left" colspan="1" rowspan="1">12 (10)</td>
<td align="left" colspan="1" rowspan="1">20 (16.66)</td>
<td align="left" colspan="1" rowspan="1">10 (8.33)</td>
<td align="left" colspan="1" rowspan="1">4 (3.33)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p33"><p>Gene frequencies are presented as absolute numbers with percentage in parentheses.</p></div></div></section></section><section id="Sec15"><h3 class="pmc_sec_title">Age-related association of β-lactamase infections in pediatric patients</h3>
<p id="Par45">Table <a href="#Tab4" class="usa-link">4</a> presents a multivariable logistic regression analysis comparing the odds of β-lactamase infections (ESBL, MBL, and AmpC) between two age groups (&lt; 36 months and 36–60 months), using group 3 as the reference. The odds of having an ESBL infection in group 1 were 3.612 times higher compared to group 3, and in group 2, the odds were 0.527 times lower than in group 3. However, these differences were not statistically significant (<em>p</em> &gt; 0.05), the reason for this maybe lies in the fact that group 2 was hospitalized and likely received antibiotics for other bacterial infections. This prior or ongoing antibiotic exposure could influence microbial resistance patterns or colonization rates, making their microbial profile different from that of healthy community controls, group 2 may have been exposed to selective pressure due to antibiotic treatment, leading to an increased likelihood of carrying resistant organisms and it might also be due to insufficient sample size or variability within the group, which limits the power to detect a real difference even if it exists.</p>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Multivariable logistic regression showing association of β-lactamase infection with two different age groups.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Group</th>
<th align="left" rowspan="2" colspan="1">Beta-lactamase infection</th>
<th align="left" colspan="2" rowspan="1">Age</th>
<th align="left" colspan="1" rowspan="1">
<em>p</em>-value</th>
<th align="left" rowspan="2" colspan="1">Odds ratio 95% CI (lower-upper)</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">(<em>≤</em> 36)<br>months</th>
<th align="left" colspan="1" rowspan="1">(36–60) months</th>
<th align="left" colspan="1" rowspan="1">Group 3 as reference</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="2" colspan="1">1</td>
<td align="left" colspan="1" rowspan="1">ESBL +</td>
<td align="left" colspan="1" rowspan="1">25</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">0.215</td>
<td align="left" rowspan="2" colspan="1">3.612 (0.472–34.28)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ESBL -</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">0.215</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">2</td>
<td align="left" colspan="1" rowspan="1">ESBL +</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" rowspan="2" colspan="1">0.498</td>
<td align="left" rowspan="2" colspan="1">0.527 (0.0652- 2.87)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ESBL -</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">1</td>
<td align="left" colspan="1" rowspan="1">MBL +</td>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" rowspan="2" colspan="1">0.023*</td>
<td align="left" rowspan="2" colspan="1">2.957 (1.153–7.898)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MBL -</td>
<td align="left" colspan="1" rowspan="1">17</td>
<td align="left" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">2</td>
<td align="left" colspan="1" rowspan="1">MBL +</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" rowspan="2" colspan="1">0.472</td>
<td align="left" rowspan="2" colspan="1">1.438 (0.537–3.922)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MBL -</td>
<td align="left" colspan="1" rowspan="1">19</td>
<td align="left" colspan="1" rowspan="1">8</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">1</td>
<td align="left" colspan="1" rowspan="1">AmpC +</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" rowspan="2" colspan="1">0.331</td>
<td align="left" rowspan="2" colspan="1">1.648 (0.605–4.614)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AmpC -</td>
<td align="left" colspan="1" rowspan="1">25</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">2</td>
<td align="left" colspan="1" rowspan="1">AmpC +</td>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" rowspan="2" colspan="1">0.591</td>
<td align="left" rowspan="2" colspan="1">0.733 (0.232–2.251)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AmpC +</td>
<td align="left" colspan="1" rowspan="1">24</td>
<td align="left" colspan="1" rowspan="1">9</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p37"><p><strong>*</strong>Statistically significant.</p></div></div></section><p id="Par47">Overall, this logistic regression analysis reveals that, generally, there is no statistically significant association between age and β-lactamase infections (ESBL, MBL, and AmpC) in the two age groups analyzed, except for ESBL-negative cases in group 1, where younger children were more likely to be ESBL-negative.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Age as a significant predictor of β-lactamase production in pediatric patients</h3>
<p id="Par48">Table <a href="#Tab5" class="usa-link">5</a> presents a multivariable logistic regression analysis identifying age as a significantly associated with β-lactamase production in pediatric patients. Children under 3 years were significantly associated with increased odds of producing ESBL (odds ratio = 10.19), MBL (odds ratio = 8.4), and AmpC β-lactamases (odds ratio = 13.82) compared to those aged 3 to 5 years (<em>p</em> &lt; 0.0001 for all). Gender did not significantly predict β-lactamase production, although a non-significant trend was observed for higher MBL production in males. These results underscore the critical role of age in β-lactamase production among young children.</p>
<section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Multiple logistic regression models exploring certain independent predictors of β-lactamase genes.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Predictors of β-lactamase production</th>
<th align="left" colspan="1" rowspan="1">
<em>n</em> = 120</th>
<th align="left" colspan="1" rowspan="1">
<em>p</em>-value</th>
<th align="left" colspan="1" rowspan="1">Adjusted Odds ratio</th>
<th align="left" colspan="1" rowspan="1">95%CI<br>(Lower)</th>
<th align="left" colspan="1" rowspan="1">95%CI<br>(Upper)</th>
</tr></thead>
<tbody>
<tr><td align="left" colspan="6" rowspan="1">Extended spectrum β-lactamases</td></tr>
<tr><td align="left" colspan="6" rowspan="1">Extended spectrum β-lactamases</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">76</td>
<td align="left" rowspan="2" colspan="1">0.000*</td>
<td align="left" colspan="1" rowspan="1">2.969</td>
<td align="left" rowspan="2" colspan="1">1.76</td>
<td align="left" rowspan="2" colspan="1">5.05</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">44</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr><td align="left" colspan="6" rowspan="1">TEM</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">49</td>
<td align="left" rowspan="2" colspan="1">0.004*</td>
<td align="left" colspan="1" rowspan="1">0.477</td>
<td align="left" rowspan="2" colspan="1">0.284</td>
<td align="left" rowspan="2" colspan="1">0.798</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">71</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr><td align="left" colspan="6" rowspan="1">SHV</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">39</td>
<td align="left" rowspan="2" colspan="1">0.000*</td>
<td align="left" colspan="1" rowspan="1">0.227</td>
<td align="left" rowspan="2" colspan="1">0.131</td>
<td align="left" rowspan="2" colspan="1">0.389</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">81</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr><td align="left" colspan="6" rowspan="1">CTX</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" rowspan="2" colspan="1">0.000*</td>
<td align="left" colspan="1" rowspan="1">0.057</td>
<td align="left" rowspan="2" colspan="1">0.029</td>
<td align="left" rowspan="2" colspan="1">0.107</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">97</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr><td align="left" colspan="6" rowspan="1">OXA</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" rowspan="2" colspan="1">0.000*</td>
<td align="left" colspan="1" rowspan="1">0.057</td>
<td align="left" rowspan="2" colspan="1">0.029</td>
<td align="left" rowspan="2" colspan="1">0.107</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">97</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr><td align="left" colspan="6" rowspan="1">Age</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">&lt; 3 years</td>
<td align="left" colspan="1" rowspan="1">58</td>
<td align="left" rowspan="2" colspan="1">0.000*</td>
<td align="left" colspan="1" rowspan="1">10.19</td>
<td align="left" rowspan="2" colspan="1">4.885</td>
<td align="left" rowspan="2" colspan="1">22.06</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">&gt; 3 to 5 years</td>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr><td align="left" colspan="6" rowspan="1">Gender</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Male</td>
<td align="left" colspan="1" rowspan="1">43</td>
<td align="left" rowspan="2" colspan="1">0.109</td>
<td align="left" colspan="1" rowspan="1">1.692</td>
<td align="left" rowspan="2" colspan="1">0.889</td>
<td align="left" rowspan="2" colspan="1">3.239</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Female</td>
<td align="left" colspan="1" rowspan="1">33</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr><td align="left" colspan="6" rowspan="1">
<strong>Metallo β-lactamases</strong>
</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">67</td>
<td align="left" rowspan="2" colspan="1">0.000*</td>
<td align="left" colspan="1" rowspan="1">0.313</td>
<td align="left" rowspan="2" colspan="1">0.1836</td>
<td align="left" rowspan="2" colspan="1">0.53</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">53</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr><td align="left" colspan="6" rowspan="1">Age</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">&lt; 3 year</td>
<td align="left" colspan="1" rowspan="1">37</td>
<td align="left" rowspan="2" colspan="1">0.000*</td>
<td align="left" colspan="1" rowspan="1">8.4</td>
<td align="left" rowspan="2" colspan="1">3.536</td>
<td align="left" rowspan="2" colspan="1">22.81</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">&gt; 3 to 5 years</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr><td align="left" colspan="6" rowspan="1">Gender</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Male</td>
<td align="left" colspan="1" rowspan="1">27</td>
<td align="left" rowspan="2" colspan="1">0.067</td>
<td align="left" colspan="1" rowspan="1">1.882</td>
<td align="left" rowspan="2" colspan="1">0.957</td>
<td align="left" rowspan="2" colspan="1">3.782</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Female</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr><td align="left" colspan="6" rowspan="1">
<strong>AmpC β-lactamases</strong>
</td></tr>
<tr><td align="left" colspan="6" rowspan="1">AmpC</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Yes</td>
<td align="left" colspan="1" rowspan="1">47</td>
<td align="left" colspan="1" rowspan="1">0.000*</td>
<td align="left" colspan="1" rowspan="1">0.102</td>
<td align="left" rowspan="2" colspan="1">0.056</td>
<td align="left" rowspan="2" colspan="1">0.184</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">73</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr><td align="left" colspan="6" rowspan="1">Age</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">&lt; 3 years</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">0.000*</td>
<td align="left" colspan="1" rowspan="1">13.82</td>
<td align="left" rowspan="2" colspan="1">4.034</td>
<td align="left" rowspan="2" colspan="1">54</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">&gt; 3 to 5 years</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr><td align="left" colspan="6" rowspan="1">Gender</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Male</td>
<td align="left" colspan="1" rowspan="1">17</td>
<td align="left" colspan="1" rowspan="1">0.204</td>
<td align="left" colspan="1" rowspan="1">1.982</td>
<td align="left" rowspan="2" colspan="1">0.693</td>
<td align="left" rowspan="2" colspan="1">5.817</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Female</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p42"><p>*Statistically significant (The y variable is diarrhea whereas the variables mentioned as associated with the independent (x) variables).</p></div></div></section></section><section id="Sec17"><h3 class="pmc_sec_title">Phylogenetic analysis and evolutionary relationships of β-lactamase genes in diverse bacterial strains</h3>
<p id="Par50"> A phylogenetic tree was constructed using the maximum likelihood method in MEGA 11.0, following multiple sequence alignment with Clustal W (Fig. <a href="#Fig2" class="usa-link">2</a>). Sequences generated in this study were deposited in the NCBI GenBank, with accession numbers including <a href="https://www.ncbi.nlm.nih.gov/nuccore/KX941097.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KX941097.1</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY753820.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KY753820.1</a> (<em>TEM</em>), <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY883445.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KY883445.1</a> (<em>SHV</em>), <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY753817" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KY753817</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY883447.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KY883447.1</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY883448.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KY883448.1</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY883449.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KY883449.1</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY883450.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KY883450.1</a> (<em>CTX</em>), <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY913604.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KY913604.1</a> (<em>OXA</em>), <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY753818.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KY753818.1</a> (<em>NDM-1</em>) and <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY753819.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KY753819.1</a> (<em>CMY</em>). The DEC isolates sequenced in this study are indicated with red diamonds in Fig. <a href="#Fig2" class="usa-link">2</a> and represent various β-lactamase genes associated with multidrug resistance (Table <a href="#MOESM1" class="usa-link">S1</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/d2d50710902e/41598_2025_16122_Fig2_HTML.jpg" loading="lazy" id="d33e1656" height="1181" width="680" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>This phylogenetic tree represents the evolutionary relationships among various β-lactamase genes associated with multidrug resistance from different bacterial strains. The tree is derived from multiple sequence alignments and constructed using MEGA software. Each branch corresponds to a specific gene sequence, with labels indicating the gene name and the bacterial strain. Red diamonds highlight the DEC specifically focused on in this study. The tree provides insights into the genetic diversity and evolutionary lineage of β-lactamase genes (Table <a href="#MOESM1" class="usa-link">S1</a>), illustrating the clustering of similar gene sequences within and across different bacterial species.</p></figcaption></figure><p id="Par52">The phylogenetic tree demonstrates that these β-lactamase genes are distributed among distinct clades, reflecting both sequence divergence and horizontal gene transfer events. Clustering of similar β-lactamase variants across different bacterial species indicates frequent inter-species gene exchange, contributing to the dissemination of resistance mechanisms. To further validate the functional conservation of these genes, INTERPROSCAN analysis was performed. This revealed that all β-lactamase gene sequences shared ≥ 98% similarity with known β-lactamase family domains. This high degree of domain conservation, across diverse phylogenetic clusters, supports the evolutionary stability and functional importance of β-lactamase activity despite sequence variability. Together, the phylogenetic and domain analyses suggest that these resistance genes are both evolutionarily conserved and selectively retained, underscoring their central role in antibiotic resistance.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Structural insights into the binding interactions of amoxicillin and ceftriaxone with β-lactamase proteins</h3>
<p id="Par53">In this study, we analyzed the interactions between amoxicillin and ceftriaxone and β-lactamase proteins (<em>TEM</em>, <em>SHV</em>, <em>CTX-M</em>, <em>OXA</em>, <em>NDM-1</em>, and <em>AmpC</em>), which are key mediators of antibiotic resistance in bacterial pathogens, using active site prediction and molecular docking techniques (Figs. <a href="#Fig3" class="usa-link">3</a>, <a href="#Fig4" class="usa-link">4</a>, <a href="#Fig5" class="usa-link">5</a>, <a href="#Fig6" class="usa-link">6</a>, <a href="#Fig7" class="usa-link">7</a>, <a href="#Fig8" class="usa-link">8</a>, <a href="#Fig9" class="usa-link">9</a>, <a href="#Fig10" class="usa-link">10</a>, <a href="#Fig11" class="usa-link">11</a>, <a href="#Fig12" class="usa-link">12</a>, <a href="#Fig13" class="usa-link">13</a>, <a href="#Fig14" class="usa-link">14</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/2f51aca1f885/41598_2025_16122_Fig3_HTML.jpg" loading="lazy" id="d33e1742" height="591" width="731" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Docking interactions of amoxicillin with the TEM β-lactamase protein. (<strong>A</strong>) 2D Interactions: Depicts the molecular interactions of amoxicillin with active site residues of the TEM β-lactamase protein, highlighting hydrogen bonds, hydrophobic interactions, and other non-covalent interactions critical for binding. Generated using Discovery Studio Visualizer. <strong>(B) 3D Interactions</strong>: Shows the spatial conformation of amoxicillin within the active site of the TEM protein, providing insights into its orientation and proximity to critical residues. Generated using PyMOL for 3D visualization of the docked complex. <strong>(C) Surface Area Interactions</strong>: Highlights the binding pocket topology, including hydrophilic and hydrophobic regions, showcasing how amoxicillin interacts with the enzyme’s surface. Created using AutoDock Tools for docking results and PyMOL for surface visualization. <strong>(D) Protein Structure with Binding Site</strong>: Demonstrates the location of the binding pocket within the overall tertiary structure of the TEM protein. Generated using PyMOL to integrate the docking results into the protein structure.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/e38dd79b65f0/41598_2025_16122_Fig4_HTML.jpg" loading="lazy" id="d33e1896" height="568" width="722" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Docking interactions of ceftriaxone with the TEM β-lactamase protein. <strong>(A) 2D Interactions</strong>: Depicts the molecular interactions of ceftriaxone with active site residues of the TEM β-lactamase protein, including hydrogen bonds, van der Waals forces, and other critical non-covalent interactions. Generated using Discovery Studio Visualizer. <strong>(B) 3D Interactions</strong>: Shows the spatial conformation of ceftriaxone within the active site of the TEM protein, highlighting the orientation and spatial proximity to key amino acid residues. Created using PyMOL. <strong>(C) Surface Area Interactions</strong>: Displays the surface topology of the TEM protein binding pocket, emphasizing hydrophilic and hydrophobic interactions with ceftriaxone. Generated using AutoDock Tools and visualized in PyMOL. <strong>(D) Protein Structure with Binding Site</strong>: Demonstrates the position of the binding pocket in the overall 3D structure of the TEM protein, providing a holistic view of the docking interaction. Visualized using PyMOL.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/79fbd7ce9c1b/41598_2025_16122_Fig5_HTML.jpg" loading="lazy" id="d33e1917" height="572" width="730" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Docking interactions of amoxicillin with the SHV-1 β-lactamase protein. <strong>(A) 2D Interactions</strong>: Displays the molecular interactions of amoxicillin with the active site residues of the SHV-1 β-lactamase protein, including hydrogen bonds, van der Waals forces, and other critical interactions stabilizing the drug-enzyme complex. Generated using Discovery Studio Visualizer. <strong>(B) 3D Interactions</strong>: Illustrates the spatial orientation of amoxicillin within the active site, highlighting its proximity to key amino acid residues and the structural compatibility of the molecule within the binding pocket. Visualized using PyMOL. <strong>(C) Surface Area Interactions</strong>: Reveals the topological features of the SHV-1 protein’s binding pocket, emphasizing the interactions of amoxicillin with hydrophilic and hydrophobic regions. Generated using AutoDock Tools and visualized in PyMOL. <strong>(D) Protein Structure with Binding Site</strong>: Demonstrates the location of the amoxicillin binding pocket within the overall tertiary structure of SHV-1, providing a comprehensive view of the interaction. Visualized using PyMOL.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/8f5119941ca5/41598_2025_16122_Fig6_HTML.jpg" loading="lazy" id="d33e1938" height="534" width="732" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Docking interactions of ceftriaxone with the SHV-1 β-lactamase protein. <strong>(A) 2D Interactions</strong>: Highlights the molecular interactions of ceftriaxone with key active site residues of the SHV-1 β-lactamase protein, including hydrogen bonds, van der Waals interactions, and other non-covalent interactions. These interactions stabilize the ceftriaxone-enzyme complex. Generated using Discovery Studio Visualizer. <strong>(B) 3D Interactions</strong>: Shows the spatial orientation of ceftriaxone within the SHV-1 active site, illustrating its interaction with critical residues and the structural fit within the binding pocket. Created using PyMOL. <strong>(C) Surface Area Interactions</strong>: Depicts the binding pocket topology of the SHV-1 protein, emphasizing the hydrophilic and hydrophobic interactions between ceftriaxone and the enzyme surface. Visualized using AutoDock Tools and PyMOL. <strong>(D) Protein Structure with Binding Site</strong>: Presents the overall structure of the SHV-1 β-lactamase protein with the binding pocket highlighted, providing a holistic view of the ceftriaxone binding interaction. Visualized using PyMOL.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/208cbc0780f8/41598_2025_16122_Fig7_HTML.jpg" loading="lazy" id="d33e1959" height="545" width="739" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Docking interactions of amoxicillin with the CTX-M-14 β-lactamase protein. <strong>(A) 2D Interactions</strong>: Shows the molecular interactions of amoxicillin with active site residues of CTX-M-14, including hydrogen bonds, van der Waals forces, and other key non-covalent interactions. These interactions play a crucial role in stabilizing the drug-enzyme complex. Generated using Discovery Studio Visualizer. <strong>(B) 3D Interactions</strong>: Illustrates the spatial orientation of amoxicillin within the active site of CTX-M-14, highlighting its proximity to critical residues and the structural compatibility of the molecule with the binding pocket. Visualized using PyMOL. <strong>(C) Surface Area Interactions</strong>: Depicts the binding pocket topology of CTX-M-14, emphasizing hydrophilic and hydrophobic regions involved in the interaction with amoxicillin. Generated using AutoDock Tools and visualized with PyMOL. <strong>(D) Protein Structure with Binding Site</strong>: Provides an overview of the CTX-M-14 β-lactamase protein’s tertiary structure, showcasing the binding pocket location and the interaction of amoxicillin with surrounding residues. Visualized using PyMOL.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/a6cf51abf4fb/41598_2025_16122_Fig8_HTML.jpg" loading="lazy" id="d33e1980" height="569" width="736" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Docking interactions of ceftriaxone with the CTX-M-14 β-lactamase protein. <strong>(A) 2D Interactions</strong>: Depicts the molecular interactions of ceftriaxone with key active site residues of CTX-M-14, including hydrogen bonds, van der Waals interactions, and other stabilizing forces that contribute to the drug’s binding. Generated using Discovery Studio Visualizer. <strong>(B) 3D Interactions</strong>: Highlights the spatial conformation of ceftriaxone within the CTX-M-14 active site, showing its orientation and proximity to critical residues involved in drug binding. Created using PyMOL. <strong>(C) Surface Area Interactions</strong>: Displays the binding pocket topology, showcasing hydrophilic and hydrophobic regions and their interactions with ceftriaxone. Generated using AutoDock Tools and visualized in PyMOL. <strong>(D) Protein Structure with Binding Site</strong>: Provides an overview of the CTX-M-14 β-lactamase protein’s tertiary structure, pinpointing the binding pocket and illustrating the interactions of ceftriaxone with key residues. Visualized using PyMOL.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/bf554d7332c9/41598_2025_16122_Fig9_HTML.jpg" loading="lazy" id="d33e2002" height="532" width="730" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Docking interactions of amoxicillin with the OXA-1 β-lactamase protein. <strong>(A) 2D Interactions</strong>: Illustrates the molecular interactions between amoxicillin and the active site residues of OXA-1, including hydrogen bonds, van der Waals forces, and other stabilizing interactions that support binding. Generated using Discovery Studio Visualizer. <strong>(B) 3D Interactions</strong>: Depicts the spatial conformation of amoxicillin within the OXA-1 active site, highlighting its orientation and interaction with key residues critical for binding. Visualized using PyMOL. <strong>(C) Surface Area Interactions</strong>: Highlights the binding pocket topology, showing hydrophilic and hydrophobic regions and their interaction with amoxicillin. Generated using AutoDock Tools and visualized with PyMOL. <strong>(D) Protein Structure with Binding Site</strong>: Provides a structural overview of the OXA-1 β-lactamase protein, identifying the binding pocket and its interaction with amoxicillin within the overall tertiary structure. Visualized using PyMOL.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/8b9a745a859e/41598_2025_16122_Fig10_HTML.jpg" loading="lazy" id="d33e2023" height="544" width="669" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Docking interactions of ceftriaxone with the OXA-1 β-lactamase protein. <strong>(A) 2D Interactions</strong>: Highlights the molecular interactions between ceftriaxone and the active site residues of OXA-1, including hydrogen bonds, van der Waals forces, and other stabilizing interactions crucial for binding. Generated using Discovery Studio Visualizer. <strong>(B) 3D Interactions</strong>: Depicts the spatial conformation of ceftriaxone within the OXA-1 active site, showing its orientation and proximity to key residues involved in the binding process. Visualized using PyMOL. <strong>(C) Surface Area Interactions</strong>: Displays the binding pocket topology of OXA-1, emphasizing the hydrophilic and hydrophobic interactions between ceftriaxone and the enzyme’s active site. Generated using AutoDock Tools and visualized with PyMOL. <strong>(D) Protein Structure with Binding Site</strong>: Provides a structural overview of the OXA-1 β-lactamase protein, identifying the binding pocket location and illustrating the interactions of ceftriaxone with key residues in the tertiary structure. Visualized using PyMOL.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig11"><h4 class="obj_head">Fig. 11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig11_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/3d333574b923/41598_2025_16122_Fig11_HTML.jpg" loading="lazy" id="d33e2044" height="531" width="669" alt="Fig. 11"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Docking interactions of amoxicillin with the NDM β-lactamase protein. <strong>(A) 2D Interactions</strong>: Highlights the molecular interactions between amoxicillin and the active site residues of NDM, including hydrogen bonds, van der Waals forces, and π-alkyl interactions. These interactions are essential for stabilizing the drug-enzyme complex. Generated using Discovery Studio Visualizer. <strong>(B) 3D Interactions</strong>: Shows the spatial conformation of amoxicillin within the NDM active site, illustrating its binding orientation and proximity to key residues involved in stabilizing the complex. Visualized using PyMOL. <strong>(C) Surface Area Interactions</strong>: Depicts the binding pocket topology of NDM, emphasizing hydrophilic and hydrophobic interactions between amoxicillin and the enzyme. Generated using AutoDock Tools and visualized with PyMOL. <strong>(D) Protein Structure with Binding Site</strong>: Provides an overview of the NDM β-lactamase protein’s tertiary structure, pinpointing the location of the binding pocket and illustrating the interactions of amoxicillin with the surrounding residues. Visualized using PyMOL.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig12"><h4 class="obj_head">Fig. 12.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig12_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/6adf6b466033/41598_2025_16122_Fig12_HTML.jpg" loading="lazy" id="d33e2065" height="559" width="724" alt="Fig. 12"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Docking interactions of ceftriaxone with the NDM β-lactamase protein. <strong>(A) 2D Interactions</strong>: Highlights the molecular interactions between ceftriaxone and the active site residues of NDM, including hydrogen bonds, van der Waals forces, and π-alkyl interactions. These interactions are critical for stabilizing the drug-enzyme complex. Generated using Discovery Studio Visualizer. <strong>(B) 3D Interactions</strong>: Depicts the spatial conformation of ceftriaxone within the NDM active site, illustrating its binding orientation and proximity to key residues involved in stabilizing the complex. Visualized using PyMOL. <strong>(C) Surface Area Interactions</strong>: Shows the topology of the NDM binding pocket, emphasizing hydrophilic and hydrophobic interactions between ceftriaxone and the enzyme. Generated using AutoDock Tools and visualized with PyMOL. <strong>(D) Protein Structure with Binding Site</strong>: Provides a structural overview of the NDM β-lactamase protein, identifying the binding pocket location and illustrating the interactions of ceftriaxone with key residues in the tertiary structure. Visualized using PyMOL.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig13"><h4 class="obj_head">Fig. 13.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig13_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/59f3b729a23b/41598_2025_16122_Fig13_HTML.jpg" loading="lazy" id="d33e2086" height="540" width="737" alt="Fig. 13"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Docking interactions of amoxicillin with the AmpC β-lactamase protein. <strong>(A) 2D Interactions</strong>: Illustrates the molecular interactions between amoxicillin and the active site residues of AmpC, highlighting hydrogen bonds, van der Waals forces, and π-alkyl interactions that stabilize the drug-enzyme complex. Generated using Discovery Studio Visualizer. <strong>(B) 3D Interactions</strong>: Depicts the spatial conformation of amoxicillin within the AmpC active site, showing its binding orientation and interaction with critical residues. Visualized using PyMOL. <strong>(C) Surface Area Interactions</strong>: Shows the topology of the AmpC binding pocket, emphasizing hydrophilic and hydrophobic interactions between amoxicillin and the enzyme. Generated using AutoDock Tools and visualized with PyMOL. <strong>(D) Protein Structure with Binding Site</strong>: Provides an overview of the AmpC β-lactamase protein, identifying the location of the binding pocket and illustrating amoxicillin’s interactions with surrounding residues in the tertiary structure. Visualized using PyMOL.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig14"><h4 class="obj_head">Fig. 14.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig14_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/569babfd2791/41598_2025_16122_Fig14_HTML.jpg" loading="lazy" id="d33e2107" height="566" width="740" alt="Fig. 14"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig14/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Docking interactions of ceftriaxone with the AmpC β-lactamase protein. <strong>(A) 2D Interactions</strong>: Depicts the molecular interactions between ceftriaxone and active site residues of AmpC, including hydrogen bonds, van der Waals forces, π-alkyl interactions, and other stabilizing forces. These interactions play a key role in forming the drug-enzyme complex. Generated using Discovery Studio Visualizer. <strong>(B) 3D Interactions</strong>: Highlights the spatial orientation of ceftriaxone within the AmpC active site, showing its proximity to key residues involved in stabilizing the binding interaction. Visualized using PyMOL. <strong>(C) Surface Area Interactions</strong>: Shows the binding pocket topology of AmpC, emphasizing hydrophilic and hydrophobic regions that interact with ceftriaxone. Generated using AutoDock Tools and visualized with PyMOL. <strong>(D) Protein Structure with Binding Site</strong>: Provides an overview of the AmpC β-lactamase protein, pinpointing the binding pocket and illustrating the interactions of ceftriaxone with surrounding residues in the tertiary structure. Visualized using PyMOL.</p></figcaption></figure></section><section id="Sec19"><h3 class="pmc_sec_title">Active site prediction</h3>
<p id="Par55">The active sites of the β-lactamase proteins were meticulously identified and compared with their respective template structures to pinpoint conserved binding regions (Figs. <a href="#Fig3" class="usa-link">3</a>, <a href="#Fig4" class="usa-link">4</a>, <a href="#Fig5" class="usa-link">5</a>, <a href="#Fig6" class="usa-link">6</a>, <a href="#Fig7" class="usa-link">7</a>, <a href="#Fig8" class="usa-link">8</a>, <a href="#Fig9" class="usa-link">9</a>, <a href="#Fig10" class="usa-link">10</a>, <a href="#Fig11" class="usa-link">11</a>, <a href="#Fig12" class="usa-link">12</a>, <a href="#Fig13" class="usa-link">13</a>, <a href="#Fig14" class="usa-link">14</a>). The analysis revealed that sites 1 and 2 were highly conserved across most of the proteins, although each enzyme exhibited distinct patterns of amino acid residues at these active sites. For the <em>TEM</em> β-lactamase, key residues such as PHE135, THR120, and MET19 were crucial for binding interactions with inhibitors. In the SHV enzyme, residues like ALA60, GLU121, and THR163 played significant roles in facilitating interactions with beta-lactam antibiotics. The <em>CTX-M</em> β-lactamase’s active site included critical residues such as MET38, LEU15, and ARG12, essential for the enzyme’s function and its binding to inhibitors. Similarly, the OXA enzyme exhibited important conserved residues, including VAL23, TRP134, and MET19, which were vital for its activity. The <em>NDM-1</em> enzyme’s active site was characterized by residues like TRP134, LEU83, and ILE86, which are key for inhibitor binding. Finally, in the AmpC β-lactamase, residues such as TYR48, GLY121, and SER22 were identified as integral to the enzyme’s resistance mechanism. These findings highlight the importance of these conserved residues in the functionality and interactions with inhibitors of β-lactamase enzymes.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Molecular docking interactions</h3>
<p id="Par56">Following the prediction of active sites, molecular docking studies were conducted to evaluate the binding interactions of amoxicillin and ceftriaxone with the identified β-lactamase proteins. The docking results provided detailed insights into the structural and molecular interactions within the active sites, as visualized in Figs. <a href="#Fig3" class="usa-link">3</a>, <a href="#Fig4" class="usa-link">4</a>, <a href="#Fig5" class="usa-link">5</a>, <a href="#Fig6" class="usa-link">6</a>, <a href="#Fig7" class="usa-link">7</a>, <a href="#Fig8" class="usa-link">8</a>, <a href="#Fig9" class="usa-link">9</a>, <a href="#Fig10" class="usa-link">10</a>, <a href="#Fig11" class="usa-link">11</a>, <a href="#Fig12" class="usa-link">12</a>, <a href="#Fig13" class="usa-link">13</a>, <a href="#Fig14" class="usa-link">14</a>. For the TEM protein (Fig. <a href="#Fig3" class="usa-link">3</a> and <a href="#Fig4" class="usa-link">4</a>), amoxicillin interacted within a site volume of 330 cubic Å, forming hydrogen bonds and other molecular interactions with residues such as SER56, GLN62, and ARG57, while ceftriaxone engaged with residues like MET129, GLY130, and LYS16 within the same site volume, highlighting distinct binding profiles. In the SHV protein (Figs. <a href="#Fig5" class="usa-link">5</a> and <a href="#Fig6" class="usa-link">6</a>), amoxicillin docked within a larger site volume of 486 cubic Å, interacting with residues like PHE35, PRO36, and MET38, whereas ceftriaxone showed binding interactions with residues such as GLY14, LEU15, and ILE16, demonstrating a different orientation within the same active site. The CTX-M protein (Figs. <a href="#Fig7" class="usa-link">7</a> and <a href="#Fig8" class="usa-link">8</a>) exhibited a site volume of 259 cubic Å for both drugs, with amoxicillin interacting with ALA47, PRO63, and VAL54, and ceftriaxone binding to residues like THR51, VAL54, and VAL27, further illustrating differences in binding orientation. For the OXA protein (Figs. <a href="#Fig9" class="usa-link">9</a> and <a href="#Fig10" class="usa-link">10</a>), amoxicillin occupied a site volume of 234 cubic Å, interacting with residues such as PHE82, LEU83, and ILE86, while ceftriaxone engaged with residues like SER22, GLN79, and MET19 within the same volume, showcasing distinct interaction profiles. The NDM-1 protein (Figs. <a href="#Fig11" class="usa-link">11</a> and <a href="#Fig12" class="usa-link">12</a>), with a site volume of 264 cubic Å, showed that amoxicillin interacted with residues like ILE16, GLU17, and MET18, whereas ceftriaxone bound to residues such as GLN79, LYS119, and THR120, reflecting potential differences in inhibitory effectiveness. Lastly, in the AmpC protein (Figs. <a href="#Fig13" class="usa-link">13</a> and <a href="#Fig14" class="usa-link">14</a>), amoxicillin docked within a site volume of 313 cubic Å, interacting with residues like MET38, VAL13, and PRO152, while ceftriaxone showed interactions with residues such as PHE82, LEU83, and ILE86, demonstrating how each drug occupies and interacts with the active site differently.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Molecular docking analysis reveals differential affinities of amoxicillin and ceftriaxone for β-lactamase genes in MDR DEC bacteria</h3>
<p id="Par57">To know the affinity between the selected β-lactamase genes and drugs that showed minimum resistance or high sensitivity as in our study of amoxicillin and ceftriaxone, molecular docking was performed<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. The binding energy between drugs and ligands were obtained in K.cal/mol (Fig. <a href="#Fig15" class="usa-link">15</a>).</p>
<figure class="fig xbox font-sm" id="Fig15"><h4 class="obj_head">Fig. 15.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373996_41598_2025_16122_Fig15_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/51c5/12373996/0394ce6c4452/41598_2025_16122_Fig15_HTML.jpg" loading="lazy" id="d33e2154" height="441" width="669" alt="Fig. 15"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig15/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Docking results of amoxicillin and ceftriaxone with β-lactamase genes. This figure illustrates the binding energy (K.cal/mol) and predicted IC<sub>50</sub> values (nM/µM) for various drug targets when interacted with two compounds: amoxicillin and ceftriaxone. The bar plot represents the binding energy, with lighter blue bars for amoxicillin and darker blue bars for ceftriaxone. The line plot overlaying the bars represents the predicted IC<sub>50</sub> values on a logarithmic scale, with a blue line for amoxicillin and a dark blue line for ceftriaxone. The x-axis lists the drug targets (<em>TEM-1</em>, <em>SHV-1</em>, <em>CTX-M</em>, <em>OXA-1</em>, <em>NDM-1</em>, and <em>AmpC</em>), while the y-axes display the binding energy on the left and the predicted IC<sub>50</sub> values on the right. This dual representation allows for a comprehensive comparison of the binding affinity and inhibitory concentration of the two drugs across different targets.</p></figcaption></figure></section><section id="Sec22"><h3 class="pmc_sec_title">Binding affinity</h3>
<p id="Par59">Docking results indicate that amoxicillin shows stronger predicted binding (more negative binding energy) than ceftriaxone to several β-lactamase targets, including <em>TEM-1</em> (−8.41 kcal/mol), <em>OXA-1</em> (−7.85 kcal/mol), and <em>NDM-1</em> (−6.98 kcal/mol). Ceftriaxone shows stronger binding to <em>SHV-1</em> (−6.69 kcal/mol), <em>CTX-M</em> (−5.95 kcal/mol), and <em>AmpC</em> (−7.92 kcal/mol), which may contribute to differences in resistance profiles among β-lactamase-producing bacteria. However, while these docking predictions suggest ceftriaxone can still bind <em>NDM-1</em>, they do not fully account for the clinical observation that <em>NDM-1</em> producers exhibit high resistance to amoxicillin but remain moderately susceptible to ceftriaxone. This discrepancy highlights that binding affinity alone cannot predict resistance, as catalytic efficiency (kcat/Km) and other factors like permeability and expression levels also play crucial roles (Table <a href="#Tab6" class="usa-link">6</a>)<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>.</p>
<section class="tw xbox font-sm" id="Tab6"><h4 class="obj_head">Table 6.</h4>
<div class="caption p"><p>Predicted binding energies and inhibitory potency (IC₅₀) of amoxicillin and ceftriaxone interacting with various β-Lactamase enzymes via molecular Docking.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Drug Target</th>
<th align="left" colspan="1" rowspan="1">Name of the compound</th>
<th align="left" colspan="1" rowspan="1">Binding Energy (K.cal/mol)</th>
<th align="left" colspan="1" rowspan="1">Predicted IC50 value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="2" colspan="1">TEM-1</td>
<td align="left" colspan="1" rowspan="1">Amoxicillin</td>
<td align="left" colspan="1" rowspan="1">−8.41</td>
<td align="left" colspan="1" rowspan="1">686.08 nM</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ceftriaxone</td>
<td align="left" colspan="1" rowspan="1">−6.42</td>
<td align="left" colspan="1" rowspan="1">19.81 μm</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">SHV-1</td>
<td align="left" colspan="1" rowspan="1">Amoxicillin</td>
<td align="left" colspan="1" rowspan="1">−6.26</td>
<td align="left" colspan="1" rowspan="1">25.90 μm</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ceftriaxone</td>
<td align="left" colspan="1" rowspan="1">−6.69</td>
<td align="left" colspan="1" rowspan="1">12.53 μm</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">CTX-M</td>
<td align="left" colspan="1" rowspan="1">Amoxicillin</td>
<td align="left" colspan="1" rowspan="1">−5.68</td>
<td align="left" colspan="1" rowspan="1">68.87 μm</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ceftriaxone</td>
<td align="left" colspan="1" rowspan="1">−5.95</td>
<td align="left" colspan="1" rowspan="1">43.44 μm</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">OXA-1</td>
<td align="left" colspan="1" rowspan="1">Amoxicillin</td>
<td align="left" colspan="1" rowspan="1">−7.85</td>
<td align="left" colspan="1" rowspan="1">1.75 μm</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ceftriaxone</td>
<td align="left" colspan="1" rowspan="1">−7.14</td>
<td align="left" colspan="1" rowspan="1">5.81 μm</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">NDM-1</td>
<td align="left" colspan="1" rowspan="1">Amoxicillin</td>
<td align="left" colspan="1" rowspan="1">−6.98</td>
<td align="left" colspan="1" rowspan="1">7.64 μm</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ceftriaxone</td>
<td align="left" colspan="1" rowspan="1">−6.76</td>
<td align="left" colspan="1" rowspan="1">11.02 μm</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">AmpC β-lactamase</td>
<td align="left" colspan="1" rowspan="1">Amoxicillin</td>
<td align="left" colspan="1" rowspan="1">−7.32</td>
<td align="left" colspan="1" rowspan="1">4.29 μm</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ceftriaxone</td>
<td align="left" colspan="1" rowspan="1">−7.92</td>
<td align="left" colspan="1" rowspan="1">1.56 μm</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec23"><h3 class="pmc_sec_title">Inhibitory potency (IC<sub>50</sub> values)</h3>
<p id="Par61">Despite generally weaker predicted binding affinities, ceftriaxone consistently showed lower IC<sub>50</sub> values across all β-lactamase targets, indicating stronger inhibitory potency. For example, the IC<sub>50</sub> values for ceftriaxone against <em>TEM-1</em> and <em>AmpC</em> were 19.81 nM and 1.56 µM, respectively, compared to 686.08 nM and 4.29 µM for amoxicillin. These results suggest that ceftriaxone is more effective at inhibiting these enzymes, particularly in the context of multidrug-resistant strains.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Comprehensive analysis of MDR mechanisms: distribution, enrichment, and network interconnections in DEC</h3>
<p id="Par62">In this work, we examined important MDR genes, particularly those connected to DEC, to comprehend their functions in DEC-mediated MDR when combined with ceftriaxone and amoxicillin (Table <a href="#Tab7" class="usa-link">7</a>). Using these genes, we conducted a series of analyses to explore the distribution of gene functions, pathway enrichment, and network interactions. These analyses reveal the interconnected mechanisms through which these genes contribute to MDR, providing insights into the complexity and challenges of combating resistant bacterial strains.</p>
<section class="tw xbox font-sm" id="Tab7"><h4 class="obj_head">Table 7.</h4>
<div class="caption p"><p>Genes involved in the mechanism of MDR caused by DEC.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Gene</th>
<th align="left" colspan="1" rowspan="1">Protein/Enzyme Name</th>
<th align="left" colspan="1" rowspan="1">Pathway</th>
<th align="left" colspan="1" rowspan="1">Role</th>
<th align="left" colspan="1" rowspan="1">Reference</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>acrA</em>
</td>
<td align="left" colspan="1" rowspan="1">Multidrug efflux system protein AcrA</td>
<td align="left" rowspan="6" colspan="1">Efflux pump system</td>
<td align="left" colspan="1" rowspan="1">Component of AcrAB-TolC efflux pump, expels a wide range of antibiotics</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>acrB</em>
</td>
<td align="left" colspan="1" rowspan="1">Multidrug efflux system protein AcrB</td>
<td align="left" colspan="1" rowspan="1">Component of AcrAB-TolC efflux pump, expels a wide range of antibiotics</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tolC</em>
</td>
<td align="left" colspan="1" rowspan="1">Outer membrane protein TolC</td>
<td align="left" colspan="1" rowspan="1">Outer membrane channel of AcrAB-TolC efflux pump</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>EmrE</em>
</td>
<td align="left" colspan="1" rowspan="1">Small multidrug resistance protein EmrE</td>
<td align="left" colspan="1" rowspan="1">Multidrug efflux pump</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>MdfA</em>
</td>
<td align="left" colspan="1" rowspan="1">Multidrug efflux pump MdfA</td>
<td align="left" colspan="1" rowspan="1">Multidrug efflux pump</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>NorE</em>
</td>
<td align="left" colspan="1" rowspan="1">Quinolone resistance protein NorE</td>
<td align="left" colspan="1" rowspan="1">Multidrug efflux pump</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>bla_TEM</em>
</td>
<td align="left" colspan="1" rowspan="1">Beta-lactamase TEM</td>
<td align="left" rowspan="3" colspan="1">Beta-lactamase production</td>
<td align="left" colspan="1" rowspan="1">Beta-lactamase enzyme, hydrolyzes beta-lactam antibiotics</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>bla_SHV</em>
</td>
<td align="left" colspan="1" rowspan="1">Beta-lactamase SHV</td>
<td align="left" colspan="1" rowspan="1">Beta-lactamase enzyme, hydrolyzes beta-lactam antibiotics</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>bla_CTX-M</em>
</td>
<td align="left" colspan="1" rowspan="1">Beta-lactamase CTX-M</td>
<td align="left" colspan="1" rowspan="1">Beta-lactamase enzyme, hydrolyzes beta-lactam antibiotics</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>aac(3)-II</em>
</td>
<td align="left" colspan="1" rowspan="1">Aminoglycoside 3’-phosphotransferase II</td>
<td align="left" rowspan="3" colspan="1">Aminoglycoside modification</td>
<td align="left" colspan="1" rowspan="1">Aminoglycoside acetyltransferase, modifies aminoglycosides</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>aac(6’)-Ib</em>
</td>
<td align="left" colspan="1" rowspan="1">Aminoglycoside 6’-N-acetyltransferase</td>
<td align="left" colspan="1" rowspan="1">Aminoglycoside acetyltransferase, modifies aminoglycosides</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>aadA</em>
</td>
<td align="left" colspan="1" rowspan="1">Aminoglycoside adenyltransferase A</td>
<td align="left" colspan="1" rowspan="1">Aminoglycoside adenyltransferase, modifies aminoglycosides</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tetA</em>
</td>
<td align="left" colspan="1" rowspan="1">Tetracycline resistance protein TetA</td>
<td align="left" rowspan="2" colspan="1">Tetracycline resistance</td>
<td align="left" colspan="1" rowspan="1">Efflux pump for tetracycline</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>tetB</em>
</td>
<td align="left" colspan="1" rowspan="1">Tetracycline resistance protein TetB</td>
<td align="left" colspan="1" rowspan="1">Efflux pump for tetracycline</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>qnr</em>
</td>
<td align="left" colspan="1" rowspan="1">Quinolone resistance protein Qnr</td>
<td align="left" rowspan="3" colspan="1">Quinolone resistance</td>
<td align="left" colspan="1" rowspan="1">Protects DNA gyrase and topoisomerase IV from quinolones</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>gyrA</em>
</td>
<td align="left" colspan="1" rowspan="1">DNA gyrase subunit A</td>
<td align="left" colspan="1" rowspan="1">Target site mutation, reduces quinolone binding</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>parC</em>
</td>
<td align="left" colspan="1" rowspan="1">DNA topoisomerase IV subunit A</td>
<td align="left" colspan="1" rowspan="1">Target site mutation, reduces quinolone binding</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>sul1</em>
</td>
<td align="left" colspan="1" rowspan="1">Dihydropteroate synthase I</td>
<td align="left" rowspan="2" colspan="1">Sulfonamide resistance</td>
<td align="left" colspan="1" rowspan="1">Encodes an enzyme not inhibited by sulfonamides</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>sul2</em>
</td>
<td align="left" colspan="1" rowspan="1">Dihydropteroate synthase II</td>
<td align="left" colspan="1" rowspan="1">Encodes an enzyme not inhibited by sulfonamides</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>mphA</em>
</td>
<td align="left" colspan="1" rowspan="1">Macrolide 2’-phosphotransferase</td>
<td align="left" colspan="1" rowspan="1">Macrolide resistance</td>
<td align="left" colspan="1" rowspan="1">Macrolide phosphotransferase, inactivates macrolides</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>cat</em>
</td>
<td align="left" colspan="1" rowspan="1">Chloramphenicol acetyltransferase</td>
<td align="left" colspan="1" rowspan="1">Chloramphenicol resistance</td>
<td align="left" colspan="1" rowspan="1">Chloramphenicol acetyltransferase, inactivates chloramphenicol</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>intI1</em>
</td>
<td align="left" colspan="1" rowspan="1">Integrase 1</td>
<td align="left" colspan="1" rowspan="1">Integron-mediated gene capture</td>
<td align="left" colspan="1" rowspan="1">Integron integrase, captures and integrates resistance genes</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>csgD</em>
</td>
<td align="left" colspan="1" rowspan="1">Regulator of curli production</td>
<td align="left" rowspan="5" colspan="1">Biofilm formation</td>
<td align="left" colspan="1" rowspan="1">Regulator controlling curli and biofilm-associated genes</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>bcsA</em>
</td>
<td align="left" colspan="1" rowspan="1">Cellulose synthase catalytic subunit A</td>
<td align="left" colspan="1" rowspan="1">Catalytic subunit of cellulose synthase, contributes to biofilm matrix</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>pgaC</em>
</td>
<td align="left" colspan="1" rowspan="1">Poly-beta-1,6-N-acetyl-D-glucosamine synthase</td>
<td align="left" colspan="1" rowspan="1">Glycosyltransferase involved in PIA synthesis, part of biofilm matrix</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>fimH</em>
</td>
<td align="left" colspan="1" rowspan="1">Fimbrial adhesin</td>
<td align="left" colspan="1" rowspan="1">Type 1 fimbrial adhesin, important for initial attachment</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>luxS</em>
</td>
<td align="left" colspan="1" rowspan="1">Autoinducer 2 synthase</td>
<td align="left" colspan="1" rowspan="1">Synthesizes AI-2 for quorum sensing and biofilm regulation</td>
<td align="left" colspan="1" rowspan="1">
<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec25"><h3 class="pmc_sec_title">Distribution and diversity of MDR mechanisms in MDR-DEC</h3>
<p id="Par64">Analysis of resistance gene functions in MDR-DEC revealed that multidrug efflux pumps and β-lactamase enzymes were the most prevalent, highlighting their dominant roles in resistance (Fig. <a href="#MOESM1" class="usa-link">S1</a>, Table <a href="#Tab7" class="usa-link">7</a>). Other mechanisms included antibiotic-modifying enzymes (e.g., aminoglycoside acetyltransferase), target site modification genes, and regulators of biofilm formation. This functional diversity illustrates the complex, multifactorial basis of resistance in DEC isolates<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Enriched pathways in MDR</h3>
<p id="Par65">Pathway enrichment analysis revealed significant involvement of efflux pump systems and membrane integrity pathways, suggesting their critical role in antibiotic evasion (Fig. <a href="#MOESM1" class="usa-link">S1</a> and S2). Additional enriched pathways included DNA repair mechanisms, topoisomerase activity, and glycosyltransferase-related processes, reflecting bacterial strategies to survive under antimicrobial pressure.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Network analysis of DEC-MDR genes</h3>
<p id="Par66">Network mapping of resistance genes showed that <em>bla_TEM</em>,<em> bla_SHV</em>, and <em>bla_CTX-M</em> genes were central hubs, with high connectivity to other resistance-related genes (Fig. S3). Efflux-related genes such as acrA, acrB, EmrE, and tolC also exhibited strong network integration. These gene clusters indicate co-expression or co-occurrence patterns that may enhance resistance and support horizontal gene transfer (Fig. <a href="#Fig2" class="usa-link">2</a>), which may facilitate the spread of these resistance traits across different bacterial species<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Interconnected pathways in MDR</h3>
<p id="Par67">The pathway network diagram highlights the interconnectedness of the key pathways involved in MDR (Fig. S4). Notably, glycosyltransferase pathways form a dense cluster, indicating their critical role in modifying bacterial structures like cell walls to resist antibiotic action. Efflux pump and membrane-associated pathways also form a significant cluster, showcasing their importance in actively expelling antibiotics from bacterial cells, thereby reducing drug efficacy. Additionally, pathways related to topoisomerase activity and DNA repair are closely linked, emphasizing their role in maintaining DNA integrity and helping bacteria survive antibiotic-induced stress. The strong interactions between these pathways suggest a coordinated effort by bacteria to employ multiple resistance mechanisms simultaneously, enhancing their overall survival against various antibiotics<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>.</p></section></section><section id="Sec29"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par68">The findings of this study provide valuable insights into the thermal stability, distribution, and co-production patterns of β-lactamase enzymes in DEC, highlighting the complex mechanisms underlying antibiotic resistance in the MDR DEC strains.</p>
<p id="Par69">The thermal stability analysis revealed significant variations in the <em>T</em><sub><em>m</em></sub> of different β-lactamase genes, with the <em>OXA</em> gene exhibiting the highest stability (<em>T</em><sub><em>m</em></sub> = 90 °C) and the <em>SHV</em> gene the lowest (<em>T</em><sub><em>m</em></sub> = 81.92 °C) (Fig. <a href="#Fig1" class="usa-link">1</a>). This variation in thermal stability is crucial as it suggests that enzymes like <em>OXA</em>, with higher <em>T</em><sub><em>m</em></sub>, may be more resilient under various thermal conditions<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>.</p>
<p id="Par70">The distribution analysis of β-lactamase production revealed a high prevalence of resistance genes among the clinical isolates (Table <a href="#Tab2" class="usa-link">2</a>). ESBL production was notably the most common, identified in 63.3% of the isolates, with <em>TEM</em> and <em>SHV</em> genes being the most prevalent. The study also uncovered significant co-occurrence patterns, particularly between the <em>TEM</em> and <em>SHV</em> genes, which were 4.394 times more likely to co-occur than the <em>CTX-M</em> and <em>OXA</em> genes (Table <a href="#Tab3" class="usa-link">3</a>). However, despite the frequent co-production of multiple β-lactamase genes, no single isolate harbored all relevant <em>ESBL</em>,<em> MBL</em>, or <em>AmpC</em> genes simultaneously. This may suggest that despite the common co-production of resistance genes, the complete co-occurrence of all resistance mechanisms within a single bacterial strain is not rare, which may require a combination of antibiotics to treat MDR strains<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>.</p>
<p id="Par71">The logistic regression analysis indicated that age is a significant association of β-lactamase production in pediatric patients (Tables <a href="#Tab4" class="usa-link">4</a> and <a href="#Tab5" class="usa-link">5</a>). Children under 3 years were significantly more likely to produce <em>ESBL</em>,<em> MBL</em>, and <em>AmpC</em> β-lactamases compared to those aged 3 to 5 years (<em>p</em> &lt; 0.05). This increased prevalence in younger children may be attributed to their underdeveloped immune systems, higher frequency of hospital visits or admissions, and greater exposure to broad-spectrum antibiotics, often prescribed empirically for recurrent infections such as pneumonia, otitis media, and diarrhea. Additionally, infants and toddlers are more susceptible to colonization with resistant bacteria due to immature gut microbiota and close contact with caregivers, which may facilitate the horizontal transmission of resistance genes<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>,<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. However, specific data comparing the prevalence between children under 3 years and those aged 3 to 5 years remain limited. This age-related association underscores the vulnerability of younger children to MDR infections, necessitating careful monitoring and targeted treatment strategies in this demographic.</p>
<p id="Par72">The phylogenetic analysis revealed substantial genetic variation among the DEC strains, with multiple β-lactamase genes (<em>blaTEM</em>, <em>blaSHV</em>, <em>blaCTX-M</em>, <em>blaOXA-1</em>) detected across various subgroups (Fig. <a href="#Fig2" class="usa-link">2</a>). The phylogenetic tree highlights the clustering of specific β-lactamase genes within DEC, emphasizing their role in conferring multidrug resistance. Closely related genes, such as <em>blaCTX-M</em> variants, may share similar enzymatic properties and resistance profiles, offering valuable clues for therapeutic targeting. By grouping β-lactamase genes, the tree facilitates the identification of potential targets for β-lactamase inhibitors, as genes within the same clade are likely to respond similarly to inhibitors. The red-highlighted DEC serves as a specific focus, enabling comparative analyses of their β-lactamase genes against those from other species and shedding light on unique or shared resistance mechanisms (Table <a href="#MOESM1" class="usa-link">S1</a>)<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>.</p>
<p id="Par73">The detailed exploration of β-lactamase-mediated resistance mechanisms in DEC uncovers the complex interactions between antibiotics and resistance enzymes, particularly through molecular docking studies and comprehensive analysis of MDR mechanisms.</p>
<p id="Par74">The molecular docking studies provide crucial insights into the binding interactions between key β-lactamase enzymes and the antibiotics amoxicillin and ceftriaxone, which are commonly used in clinical settings to combat bacterial infections<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. The docking results revealed distinct differences in the way these antibiotics interact with the active sites of various β-lactamase proteins (Figs. <a href="#Fig3" class="usa-link">3</a>, <a href="#Fig4" class="usa-link">4</a>, <a href="#Fig5" class="usa-link">5</a>, <a href="#Fig6" class="usa-link">6</a>, <a href="#Fig7" class="usa-link">7</a>, <a href="#Fig8" class="usa-link">8</a>, <a href="#Fig9" class="usa-link">9</a>, <a href="#Fig10" class="usa-link">10</a>, <a href="#Fig11" class="usa-link">11</a>, <a href="#Fig12" class="usa-link">12</a>, <a href="#Fig13" class="usa-link">13</a>, <a href="#Fig14" class="usa-link">14</a>).</p>
<p id="Par75">Amoxicillin demonstrated a stronger binding affinity for several β-lactamase enzymes, including <em>TEM-1</em>, <em>OXA-1</em>, and <em>NDM-1</em> (Fig. <a href="#Fig4" class="usa-link">4</a>). This strong binding, characterized by more negative binding energy values, suggests that amoxicillin is effectively recognized and bound by these enzymes<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>,<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>.</p>
<p id="Par76">Overall, the combination of active site prediction and molecular docking provided a comprehensive understanding of how amoxicillin and ceftriaxone interact with β-lactamase proteins. The distinct binding profiles observed in the docking studies, visualized in Figs. <a href="#Fig3" class="usa-link">3</a>, <a href="#Fig4" class="usa-link">4</a>, <a href="#Fig5" class="usa-link">5</a>, <a href="#Fig6" class="usa-link">6</a>, <a href="#Fig7" class="usa-link">7</a>, <a href="#Fig8" class="usa-link">8</a>, <a href="#Fig9" class="usa-link">9</a>, <a href="#Fig10" class="usa-link">10</a>, <a href="#Fig11" class="usa-link">11</a>, <a href="#Fig12" class="usa-link">12</a>, <a href="#Fig13" class="usa-link">13</a>, <a href="#Fig14" class="usa-link">14</a>, reflect the unique interaction mechanisms of each antibiotic with different β-lactamase enzymes. These findings illustrate the significance of considering specific active site residues and binding interactions when designing inhibitors to counteract antibiotic resistance in bacteria<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>.</p>
<p id="Par77">Ceftriaxone’s ability to inhibit the growth of β-lactamase-producing bacteria is not related to its ability to bind to β-lactamase or lower IC<sub>50</sub> value. β-lactams work by inhibiting PBPs, thus inhibiting peptidoglycan biosynthesis<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. This difference in binding and inhibition profiles between amoxicillin and ceftriaxone may highlight the significance of their binding affinity; as they are β-lactam antibiotics; they don’t inhibit β-lactamases instead the substrates for β-lactamases<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>,<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>.</p>
<p id="Par78">The comprehensive analysis of MDR mechanisms in DEC isolates sheds light on the intricate network of resistance strategies employed by these bacteria (Table <a href="#Tab7" class="usa-link">7</a>; Fig. <a href="#Fig5" class="usa-link">5</a>A and D). The study identified key resistance genes, including those encoding β-lactamase enzymes and multidrug efflux pumps, as central players in the MDR phenotype observed in these isolates.</p>
<p id="Par80">The gene interaction network revealed that β-lactamase genes such as <em>bla_TEM</em>, <em>bla_SHV</em>, and <em>bla_CTX-M</em> are highly interconnected with other resistance genes, indicating their pivotal role in conferring resistance to β-lactam antibiotics. These genes not only facilitate the hydrolysis of β-lactam antibiotics, rendering them ineffective but also contribute to a broader resistance network by co-occurring with other resistance mechanisms, such as efflux pumps and target site modifications<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>,<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. This network of interactions underscores the complexity of bacterial resistance, where multiple mechanisms work synergistically to protect the bacterial cell from the effects of antibiotics<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>.</p>
<p id="Par81">Efflux pumps, particularly those encoded by genes such as <em>acrA</em>, <em>acrB</em>, and <em>tolC</em>, play a critical role in actively expelling antibiotics from the bacterial cell, reducing the intracellular concentration of the drug and thereby diminishing its efficacy<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. This active expulsion mechanism is crucial for maintaining resistance, especially in combination with β-lactamase production, which further neutralizes the antibiotic threat<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>–<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>.</p>
<p id="Par82">Moreover, the pathway enrichment analysis highlighted the involvement of additional resistance mechanisms, such as DNA repair pathways and glycosyltransferase activity. These pathways contribute to the survival and adaptation of DEC strains in the presence of antibiotics by repairing drug-induced damage and modifying bacterial cell structures to prevent drug binding<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>,<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>. The interconnectedness of these pathways suggests a coordinated response by the bacteria to antibiotic pressure, enhancing their ability to survive and proliferate despite treatment<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>.</p></section><section id="Sec30"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par83">This study offers significant contributions beyond its immediate findings on β-lactamase-mediated resistance in DEC isolates. For researchers, the integration of molecular docking with microbiological and genotypic analysis provides a robust framework to investigate resistance mechanisms at both molecular and phenotypic levels. It sets a precedent for applying computational tools to predict antibiotic-enzyme interactions, which can accelerate the identification of potential drug candidates or structural vulnerabilities in pathogens. The observed cross-resistance patterns and enzyme-substrate dynamics in DEC may guide further research into resistance evolution and horizontal gene transfer, especially in community-acquired infections among pediatric populations.</p>
<p id="Par84">For clinicians, the study highlights the limitations of commonly used antibiotics like amoxicillin and ceftriaxone in the face of widespread β-lactamase activity. By correlating binding affinity with enzymatic degradation, it emphasizes the importance of using molecular insights to inform empirical treatment choices, particularly in settings with high rates of multidrug resistance. The findings may influence antibiotic selection, dosing decisions, and the prioritization of alternative therapies or combination regimens involving β-lactamase inhibitors.</p>
<p id="Par85">From a public health and infection control perspective, the high prevalence of MDR strains in children under 36 months points to urgent needs for targeted interventions—such as surveillance programs, early diagnostics, and pediatric-specific antibiotic stewardship strategies. The study’s methodological approach—combining in silico, molecular, and phenotypic data—can be adopted by diagnostic labs or hospital surveillance units to monitor resistance trends in other pathogens like <em>Klebsiella</em>, <em>Salmonella</em>, or <em>Pseudomonas</em>, many of which harbor similar resistance determinants.</p>
<p id="Par86">Overall, the study bridges clinical microbiology, computational biology, and public health, offering a scalable model for investigating resistance in diverse contexts. Its implications extend to drug discovery, policy-making, and clinical practice—making it a valuable reference for a wide range of stakeholders working to combat antimicrobial resistance.</p></section><section id="Sec31"><h2 class="pmc_sec_title">Limitations of the study</h2>
<p id="Par87">While the findings of this study offer valuable insights into β-lactamase-mediated resistance in DEC isolates, several limitations were present. First, the study was conducted on a relatively limited number of clinical isolates from a specific pediatric population, which may affect the generalizability of the results to broader or more diverse patient groups. Expanding the sample size and including isolates from different geographic regions and age groups could provide a more comprehensive understanding of resistance patterns.</p>
<p id="Par88">Second, the study focused primarily on β-lactamase enzymes and did not comprehensively assess the role of other resistance mechanisms, such as efflux pumps, porin mutations, or plasmid-mediated gene transfer, which may also contribute significantly to the MDR phenotype. A more holistic approach integrating genomic, transcriptomic, and proteomic data would provide a deeper understanding of the multifactorial nature of resistance. Addressing these limitations in future research will help refine our understanding of antimicrobial resistance and support the development of more targeted and effective therapeutic interventions.</p></section><section id="Sec32"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373996/bin/41598_2025_16122_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (707.7KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The Council of Scientific and Industrial Research (CSIR), Pusa, New Delhi, India is acknowledged for providing Fellowship [08/532(0007)/2011-EMR-I] to Dr. Taru Singh. The authors wish to thank all the children and their parents for providing the samples.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>TS and SD conceived and designed the study, analysed/interpreted results. TS, and CS collected the samples. TS, CS, SS, and SD collected the data. TS, and CS performed experiments and analyzed the data. TS, and SS carried out the literature search. TS, CS, and SS wrote the manuscript. SD performed manuscript editing. TS, SS, and SD supervised the study and revised the final version of the manuscript. All authors read and approved of the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>No funding was available to conduct the study.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Data is provided within the manuscript or supplementary information files.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par99">The study was approved by the “Institutional Ethics Committee—Human Research (IEC-HR) University College of Medical Sciences, University of Delhi, New Delhi in accordance with WHO guidelines.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par100">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Lin, F. C. et al. Characterization of young infants with fecal carriage of multidrug-resistant Escherichia coli in Southern Taiwan. <em>Pediatr. Neonatol</em>. <strong>65</strong>, 138–144 (2024).
</cite> [<a href="https://doi.org/10.1016/j.pedneo.2023.04.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37718133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20F.%20C.%20et%20al.%20Characterization%20of%20young%20infants%20with%20fecal%20carriage%20of%20multidrug-resistant%20Escherichia%20coli%20in%20Southern%20Taiwan.%20Pediatr.%20Neonatol.%2065,%20138%E2%80%93144%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Zhou, Y. et al. Characteristics of diarrheagenic Escherichia coli among children under 5 years of age with acute diarrhea: a hospital based study. <em>BMC Infect. Dis.</em><strong>18</strong>, 63 (2018).
</cite> [<a href="https://doi.org/10.1186/s12879-017-2936-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5796495/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29390982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20Y.%20et%20al.%20Characteristics%20of%20diarrheagenic%20Escherichia%20coli%20among%20children%20under%205%20years%20of%20age%20with%20acute%20diarrhea:%20a%20hospital%20based%20study.%20BMC%20Infect.%20Dis.18,%2063%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Shahbazi, G. et al. Characteristics of diarrheagenic Escherichia coli pathotypes among children under the age of 10 years with acute diarrhea. <em>Gene Rep.</em><strong>25</strong>, 101318 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Shahbazi,%20G.%20et%20al.%20Characteristics%20of%20diarrheagenic%20Escherichia%20coli%20pathotypes%20among%20children%20under%20the%20age%20of%2010%20years%20with%20acute%20diarrhea.%20Gene%20Rep.25,%20101318%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Kaderabkova, N. et al. The biogenesis of β-lactamase enzymes. <em>Microbiology (Reading) 168</em>. (2022).</cite> [<a href="https://doi.org/10.1099/mic.0.001217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10235803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35943884/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Uzodi, A. S., Lohse, C. M. &amp; Banerjee, R. Risk factors for and outcomes of Multidrug-Resistant Escherichia coli infections in children. <em>Infect. Dis. Ther.</em><strong>6</strong>, 245–257 (2017).
</cite> [<a href="https://doi.org/10.1007/s40121-017-0152-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5446365/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28374267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Uzodi,%20A.%20S.,%20Lohse,%20C.%20M.%20&amp;%20Banerjee,%20R.%20Risk%20factors%20for%20and%20outcomes%20of%20Multidrug-Resistant%20Escherichia%20coli%20infections%20in%20children.%20Infect.%20Dis.%20Ther.6,%20245%E2%80%93257%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Coleman, J. P., Smith, C. J. M. &amp; Resistance <em>In: XPharm: the Comprehensive Pharmacology Reference</em>pp. 1–3 (Elsevier, 2007).</cite>
</li>
<li id="CR7">
<span class="label">7.</span><cite>Yu, D. et al. Antimicrobial resistance in pediatric infectious diseases: antimicrobial resistance, resistance mechanisms and antimicrobial use. <em>Front. Cell. Infect. Microbiol.</em><strong>13</strong>, 1287051 (2023).
</cite> [<a href="https://doi.org/10.3389/fcimb.2023.1287051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10562681/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37822357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20D.%20et%20al.%20Antimicrobial%20resistance%20in%20pediatric%20infectious%20diseases:%20antimicrobial%20resistance,%20resistance%20mechanisms%20and%20antimicrobial%20use.%20Front.%20Cell.%20Infect.%20Microbiol.13,%201287051%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Romandini, A. et al. Antibiotic Resistance in Pediatric Infections: Global Emerging Threats, Predicting the Near Future. <em>Antibiotics (Basel) 10</em>. (2021).</cite> [<a href="https://doi.org/10.3390/antibiotics10040393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8067449/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33917430/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Wang, C. Y., Lu, C. Y., Hsieh, Y. C., Lee, C. Y. &amp; Huang, L. M. Intramuscular ceftriaxone in comparison with oral amoxicillin-clavulanate for the treatment of acute otitis media in infants and children. <em>J. Microbiol. Immunol. Infect.</em><strong>37</strong>, 57–62 (2004).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/15060689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20C.%20Y.,%20Lu,%20C.%20Y.,%20Hsieh,%20Y.%20C.,%20Lee,%20C.%20Y.%20&amp;%20Huang,%20L.%20M.%20Intramuscular%20ceftriaxone%20in%20comparison%20with%20oral%20amoxicillin-clavulanate%20for%20the%20treatment%20of%20acute%20otitis%20media%20in%20infants%20and%20children.%20J.%20Microbiol.%20Immunol.%20Infect.37,%2057%E2%80%9362%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Biner, B. et al. The comparison of single-dose ceftriaxone, five-day azithromycin, and ten-day amoxicillin/clavulanate for the treatment of children with acute otitis media. <em>Turk. J. Pediatr.</em><strong>49</strong>, 390–396 (2007).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/18246740/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Biner,%20B.%20et%20al.%20The%20comparison%20of%20single-dose%20ceftriaxone,%20five-day%20azithromycin,%20and%20ten-day%20amoxicillin/clavulanate%20for%20the%20treatment%20of%20children%20with%20acute%20otitis%20media.%20Turk.%20J.%20Pediatr.49,%20390%E2%80%93396%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Guz, D. et al. Ceftriaxone versus ampicillin for the treatment of community-acquired pneumonia. A propensity matched cohort study. <em>Clin. Microbiol. Infect.</em><strong>29</strong>, 70–76 (2023).
</cite> [<a href="https://doi.org/10.1016/j.cmi.2022.07.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35934196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guz,%20D.%20et%20al.%20Ceftriaxone%20versus%20ampicillin%20for%20the%20treatment%20of%20community-acquired%20pneumonia.%20A%20propensity%20matched%20cohort%20study.%20Clin.%20Microbiol.%20Infect.29,%2070%E2%80%9376%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Cusumano, J. A. et al. Penicillin plus ceftriaxone versus ampicillin plus ceftriaxone synergistic potential against clinical Enterococcus faecalis blood isolates. <em>Microbiol. Spectr.</em><strong>10</strong>, e0062122 (2022).
</cite> [<a href="https://doi.org/10.1128/spectrum.00621-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9430804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35703558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cusumano,%20J.%20A.%20et%20al.%20Penicillin%20plus%20ceftriaxone%20versus%20ampicillin%20plus%20ceftriaxone%20synergistic%20potential%20against%20clinical%20Enterococcus%20faecalis%20blood%20isolates.%20Microbiol.%20Spectr.10,%20e0062122%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Bush, K. &amp; Bradford, P. A. β-Lactams and β-Lactamase Inhibitors: An Overview. <em>Cold Spring Harb Perspect Med 6</em>. (2016).</cite> [<a href="https://doi.org/10.1101/cshperspect.a025247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4968164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27329032/" class="usa-link">PubMed</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>World Health. Organization Diarrhoeal Disease (2024).</cite>
</li>
<li id="CR15">
<span class="label">15.</span><cite>Kowalski, M. et al. Antimicrobial resistance in enterobacterales infections among children in sub-Saharan africa: a systematic review and meta-analysis. <em>EClinicalMedicine</em><strong>70</strong>, 102512 (2024).
</cite> [<a href="https://doi.org/10.1016/j.eclinm.2024.102512" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10940950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38495519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kowalski,%20M.%20et%20al.%20Antimicrobial%20resistance%20in%20enterobacterales%20infections%20among%20children%20in%20sub-Saharan%20africa:%20a%20systematic%20review%20and%20meta-analysis.%20EClinicalMedicine70,%20102512%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Kotloff, K. L. et al. Burden and aetiology of diarrhoeal disease in infants and young children in developing countries (the global enteric multicenter study, GEMS): a prospective, case-control study. <em>Lancet</em><strong>382</strong>, 209–222 (2013).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(13)60844-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23680352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kotloff,%20K.%20L.%20et%20al.%20Burden%20and%20aetiology%20of%20diarrhoeal%20disease%20in%20infants%20and%20young%20children%20in%20developing%20countries%20(the%20global%20enteric%20multicenter%20study,%20GEMS):%20a%20prospective,%20case-control%20study.%20Lancet382,%20209%E2%80%93222%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Ingle, D. J., Levine, M. M., Kotloff, K. L., Holt, K. E. &amp; Robins-Browne, R. M. Dynamics of antimicrobial resistance in intestinal <em>Escherichia coli</em> from children in community settings in South Asia and sub-Saharan Africa. <em>Nat. Microbiol.</em><strong>3</strong>, 1063–1073 (2018).
</cite> [<a href="https://doi.org/10.1038/s41564-018-0217-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6787116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30127495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ingle,%20D.%20J.,%20Levine,%20M.%20M.,%20Kotloff,%20K.%20L.,%20Holt,%20K.%20E.%20&amp;%20Robins-Browne,%20R.%20M.%20Dynamics%20of%20antimicrobial%20resistance%20in%20intestinal%20Escherichia%20coli%20from%20children%20in%20community%20settings%20in%20South%20Asia%20and%20sub-Saharan%20Africa.%20Nat.%20Microbiol.3,%201063%E2%80%931073%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Koneman, E. W. A.S.J.W.Mycology. In Color Atlas and Textbook of Diagnostic Microbiology, in: n.d.</cite>
</li>
<li id="CR19">
<span class="label">19.</span><cite>CLSI Performance Standards for Antimicrobial. Disk Susceptibility Tests, (2024).</cite>
</li>
<li id="CR20">
<span class="label">20.</span><cite>Kim, J. et al. Rapid detection of extended spectrum β-Lactamase (ESBL) for Enterobacteriaceae by use of a multiplex PCR-based method. <em>Infect. Chemother.</em><strong>41</strong>, 181 (2009).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kim,%20J.%20et%20al.%20Rapid%20detection%20of%20extended%20spectrum%20%CE%B2-Lactamase%20(ESBL)%20for%20Enterobacteriaceae%20by%20use%20of%20a%20multiplex%20PCR-based%20method.%20Infect.%20Chemother.41,%20181%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Naas, T., Ergani, A., Carrër, A. &amp; Nordmann, P. Real-time PCR for detection of NDM-1 carbapenemase genes from spiked stool samples. <em>Antimicrob. Agents Chemother.</em><strong>55</strong>, 4038–4043 (2011).
</cite> [<a href="https://doi.org/10.1128/AAC.01734-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3165338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21690281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Naas,%20T.,%20Ergani,%20A.,%20Carr%C3%ABr,%20A.%20&amp;%20Nordmann,%20P.%20Real-time%20PCR%20for%20detection%20of%20NDM-1%20carbapenemase%20genes%20from%20spiked%20stool%20samples.%20Antimicrob.%20Agents%20Chemother.55,%204038%E2%80%934043%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Dallenne, C., Da Costa, A., Decré, D., Favier, C. &amp; Arlet, G. Development of a set of multiplex PCR assays for the detection of genes encoding important beta-lactamases in Enterobacteriaceae. <em>J. Antimicrob. Chemother.</em><strong>65</strong>, 490–495 (2010).
</cite> [<a href="https://doi.org/10.1093/jac/dkp498" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20071363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dallenne,%20C.,%20Da%20Costa,%20A.,%20Decr%C3%A9,%20D.,%20Favier,%20C.%20&amp;%20Arlet,%20G.%20Development%20of%20a%20set%20of%20multiplex%20PCR%20assays%20for%20the%20detection%20of%20genes%20encoding%20important%20beta-lactamases%20in%20Enterobacteriaceae.%20J.%20Antimicrob.%20Chemother.65,%20490%E2%80%93495%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Manoharan, A. et al. Phenotypic &amp; molecular characterization of AmpC β-lactamases among Escherichia coli, Klebsiella spp. &amp; Enterobacter spp. From five Indian medical centers. <em>Indian J. Med. Res.</em><strong>135</strong>, 359–364 (2012).
</cite> [<a href="/articles/PMC3361873/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22561623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manoharan,%20A.%20et%20al.%20Phenotypic%20&amp;%20molecular%20characterization%20of%20AmpC%20%CE%B2-lactamases%20among%20Escherichia%20coli,%20Klebsiella%20spp.%20&amp;%20Enterobacter%20spp.%20From%20five%20Indian%20medical%20centers.%20Indian%20J.%20Med.%20Res.135,%20359%E2%80%93364%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Singh, T. et al. Changing paradigm of antibiotic resistance amongst <em>Escherichia coli</em> isolates in Indian pediatric population. <em>PLoS One</em>. <strong>14</strong>, e0213850 (2019).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0213850" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6469777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30995225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh,%20T.%20et%20al.%20Changing%20paradigm%20of%20antibiotic%20resistance%20amongst%20Escherichia%20coli%20isolates%20in%20Indian%20pediatric%20population.%20PLoS%20One.%2014,%20e0213850%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Singh, T. et al. Transcriptome analysis of beta-lactamase genes in diarrheagenic Escherichia coli. <em>Sci. Rep.</em><strong>9</strong>, 3626 (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-40279-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6403342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30842518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh,%20T.%20et%20al.%20Transcriptome%20analysis%20of%20beta-lactamase%20genes%20in%20diarrheagenic%20Escherichia%20coli.%20Sci.%20Rep.9,%203626%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Reddy, S. V., Reddy, K. T., Kumari, V. V. &amp; Basha, S. H. Molecular Docking and dynamic simulation studies evidenced plausible immunotherapeutic anticancer property by Withaferin A targeting indoleamine 2,3-dioxygenase. <em>J. Biomol. Struct. Dyn.</em><strong>33</strong> (12), 2695–2709 (2015).
</cite> [<a href="https://doi.org/10.1080/07391102.2015.1004834" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25671592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reddy,%20S.%20V.,%20Reddy,%20K.%20T.,%20Kumari,%20V.%20V.%20&amp;%20Basha,%20S.%20H.%20Molecular%20Docking%20and%20dynamic%20simulation%20studies%20evidenced%20plausible%20immunotherapeutic%20anticancer%20property%20by%20Withaferin%20A%20targeting%20indoleamine%202,3-dioxygenase.%20J.%20Biomol.%20Struct.%20Dyn.33%20(12),%202695%E2%80%932709%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Yuan, C. &amp; Hao, X. Antibacterial mechanism of action and in Silico molecular Docking studies of Cupressus funebris essential oil against drug-resistant bacterial strains. <em>Heliyon</em><strong>9</strong>(8), e18742 (2023).
</cite> [<a href="https://doi.org/10.1016/j.heliyon.2023.e18742" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10458342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37636470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan,%20C.%20&amp;%20Hao,%20X.%20Antibacterial%20mechanism%20of%20action%20and%20in%20Silico%20molecular%20Docking%20studies%20of%20Cupressus%20funebris%20essential%20oil%20against%20drug-resistant%20bacterial%20strains.%20Heliyon9(8),%20e18742%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Kim, S. et al. PubChem 2019 update: improved access to chemical data. <em>Nucleic Acids Res.</em><strong>47</strong>, D1102–D1109 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gky1033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6324075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30371825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20S.%20et%20al.%20PubChem%202019%20update:%20improved%20access%20to%20chemical%20data.%20Nucleic%20Acids%20Res.47,%20D1102%E2%80%93D1109%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Schrödinger, L. <em>LigPrep, Version 2.8</em> (Schrödinger, LLC, 2013).</cite>
</li>
<li id="CR30">
<span class="label">30.</span><cite>Schrödinger, L. <em>Glide, Version 6.1</em> (Schrödinger, LLC, 2013).</cite>
</li>
<li id="CR31">
<span class="label">31.</span><cite>Sayers, E. W. et al. Database resources of the National center for biotechnology information. <em>Nucleic Acids Res.</em><strong>50</strong>, D20–D26 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkab1112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34850941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sayers,%20E.%20W.%20et%20al.%20Database%20resources%20of%20the%20National%20center%20for%20biotechnology%20information.%20Nucleic%20Acids%20Res.50,%20D20%E2%80%93D26%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Szklarczyk, D. et al. The STRING database in 2023: protein-protein association networks and functional enrichment analyses for any sequenced genome of interest. <em>Nucleic Acids Res.</em><strong>51</strong>, D638–D646 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkac1000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36370105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Szklarczyk,%20D.%20et%20al.%20The%20STRING%20database%20in%202023:%20protein-protein%20association%20networks%20and%20functional%20enrichment%20analyses%20for%20any%20sequenced%20genome%20of%20interest.%20Nucleic%20Acids%20Res.51,%20D638%E2%80%93D646%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Shannon, P. et al. Cytoscape: a software environment for integrated models of biomolecular interaction networks. <em>Genome Res.</em><strong>13</strong>, 2498–2504 (2003).
</cite> [<a href="https://doi.org/10.1101/gr.1239303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC403769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shannon,%20P.%20et%20al.%20Cytoscape:%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks.%20Genome%20Res.13,%202498%E2%80%932504%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>The Gene Ontology Consortium The Gene Ontology Resource. 20 years and still going strong. <em>Nucleic Acids Res.</em><strong>47</strong>, D330–D338 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gky1055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30395331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?The%20Gene%20Ontology%20Consortium%20The%20Gene%20Ontology%20Resource.%2020%20years%20and%20still%20going%20strong.%20Nucleic%20Acids%20Res.47,%20D330%E2%80%93D338%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Kanehisa, M., Sato, Y., Kawashima, M., Furumichi, M. &amp; Tanabe, M. KEGG as a reference resource for gene and protein annotation. <em>Nucleic Acids Res.</em><strong>44</strong>, D457–D462 (2016).
</cite> [<a href="https://doi.org/10.1093/nar/gkv1070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4702792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26476454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa,%20M.,%20Sato,%20Y.,%20Kawashima,%20M.,%20Furumichi,%20M.%20&amp;%20Tanabe,%20M.%20KEGG%20as%20a%20reference%20resource%20for%20gene%20and%20protein%20annotation.%20Nucleic%20Acids%20Res.44,%20D457%E2%80%93D462%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Tamura, K., Stecher, G. &amp; Kumar, S. MEGA11: molecular evolutionary genetics analysis version 11. <em>Mol. Biol. Evol.</em><strong>38</strong>, 3022–3027 (2021).
</cite> [<a href="https://doi.org/10.1093/molbev/msab120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8233496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33892491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tamura,%20K.,%20Stecher,%20G.%20&amp;%20Kumar,%20S.%20MEGA11:%20molecular%20evolutionary%20genetics%20analysis%20version%2011.%20Mol.%20Biol.%20Evol.38,%203022%E2%80%933027%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Barido-Sottani, J., Chapman, S. D., Kosman, E. &amp; Mushegian, A. R. Measuring similarity between gene interaction profiles. <em>BMC Bioinform.</em><strong>20</strong>, 435 (2019).</cite> [<a href="https://doi.org/10.1186/s12859-019-3024-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6704681/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31438841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barido-Sottani,%20J.,%20Chapman,%20S.%20D.,%20Kosman,%20E.%20&amp;%20Mushegian,%20A.%20R.%20Measuring%20similarity%20between%20gene%20interaction%20profiles.%20BMC%20Bioinform.20,%20435%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Chen, L. et al. Prediction and analysis of essential genes using the enrichments of gene ontology and KEGG pathways. <em>PLoS One</em>. <strong>12</strong>, e0184129 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0184129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5584762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28873455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20L.%20et%20al.%20Prediction%20and%20analysis%20of%20essential%20genes%20using%20the%20enrichments%20of%20gene%20ontology%20and%20KEGG%20pathways.%20PLoS%20One.%2012,%20e0184129%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Prestinaci, F., Pezzotti, P. &amp; Pantosti, A. Antimicrobial resistance: a global multifaceted phenomenon. <em>Pathog Glob Health</em>. <strong>109</strong>, 309–318 (2015).
</cite> [<a href="https://doi.org/10.1179/2047773215Y.0000000030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4768623/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26343252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prestinaci,%20F.,%20Pezzotti,%20P.%20&amp;%20Pantosti,%20A.%20Antimicrobial%20resistance:%20a%20global%20multifaceted%20phenomenon.%20Pathog%20Glob%20Health.%20109,%20309%E2%80%93318%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Baquero, F. et al. Evolutionary pathways and trajectories in antibiotic resistance. <em>Clin. Microbiol. Rev.</em><strong>34</strong>, e0005019 (2021).</cite> [<a href="https://doi.org/10.1128/CMR.00050-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8404696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34190572/" class="usa-link">PubMed</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Munita, J. M. &amp; Arias, C. A. Mechanisms of antibiotic resistance. <em>Microbiol Spectr.</em><strong>4</strong>. 10.1128/microbiolspec.vmbf-0016-2015 (2016).</cite> [<a href="https://doi.org/10.1128/microbiolspec.VMBF-0016-2015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4888801/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27227291/" class="usa-link">PubMed</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Smith, B. L., Fernando, S. &amp; King, M. D. <em>Escherichia coli</em> resistance mechanism AcrAB-TolC efflux pump interactions with commonly used antibiotics: a molecular dynamics study. <em>Sci. Rep.</em><strong>14</strong>, 2742 (2024).
</cite> [<a href="https://doi.org/10.1038/s41598-024-52536-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10834472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38302495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20B.%20L.,%20Fernando,%20S.%20&amp;%20King,%20M.%20D.%20Escherichia%20coli%20resistance%20mechanism%20AcrAB-TolC%20efflux%20pump%20interactions%20with%20commonly%20used%20antibiotics:%20a%20molecular%20dynamics%20study.%20Sci.%20Rep.14,%202742%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Nolivos, S. et al. Role of AcrAB-TolC multidrug efflux pump in drug-resistance acquisition by plasmid transfer. <em>Science</em><strong>364</strong>, 778–782 (2019).
</cite> [<a href="https://doi.org/10.1126/science.aav6390" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31123134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nolivos,%20S.%20et%20al.%20Role%20of%20AcrAB-TolC%20multidrug%20efflux%20pump%20in%20drug-resistance%20acquisition%20by%20plasmid%20transfer.%20Science364,%20778%E2%80%93782%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Chaudhary, M. K., Jadhav, I. &amp; Banjara, M. R. Molecular detection of plasmid mediated blatem, blaCTX-M,and BlaSHV genes in extended spectrum β-Lactamase (ESBL) Escherichia coli from clinical samples. <em>Ann. Clin. Microbiol. Antimicrob.</em><strong>22</strong>, 33 (2023).
</cite> [<a href="https://doi.org/10.1186/s12941-023-00584-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10163748/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37147617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chaudhary,%20M.%20K.,%20Jadhav,%20I.%20&amp;%20Banjara,%20M.%20R.%20Molecular%20detection%20of%20plasmid%20mediated%20blatem,%20blaCTX-M,and%20BlaSHV%20genes%20in%20extended%20spectrum%20%CE%B2-Lactamase%20(ESBL)%20Escherichia%20coli%20from%20clinical%20samples.%20Ann.%20Clin.%20Microbiol.%20Antimicrob.22,%2033%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Ramirez, M. S., Nikolaidis, N. &amp; Tolmasky, M. E. Rise and dissemination of aminoglycoside resistance: the aac(6’)-Ib paradigm. <em>Front. Microbiol.</em><strong>4</strong>, 121 (2013).
</cite> [<a href="https://doi.org/10.3389/fmicb.2013.00121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3656343/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23730301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ramirez,%20M.%20S.,%20Nikolaidis,%20N.%20&amp;%20Tolmasky,%20M.%20E.%20Rise%20and%20dissemination%20of%20aminoglycoside%20resistance:%20the%20aac(6%E2%80%99)-Ib%20paradigm.%20Front.%20Microbiol.4,%20121%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Ramírez-Bayard, I. E. et al. Prevalence of Plasmid-Associated Tetracycline resistance genes in Multidrug-Resistant Escherichia coli strains isolated from environmental, animal and human samples in Panama. <em>Antibiotics (Basel)</em><strong>12</strong>, 280 (2023).</cite> [<a href="https://doi.org/10.3390/antibiotics12020280" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9952377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36830191/" class="usa-link">PubMed</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Malek Jamshidi, M. R., Zandi, H. &amp; Eftekhar, F. Correlation of quinolone-resistance, Qnr genes and integron carriage in multidrug-resistant community isolates of Klebsiella spp. <em>Iran. J. Basic. Med. Sci.</em><strong>22</strong>, 1387–1391 (2019).
</cite> [<a href="https://doi.org/10.22038/IJBMS.2019.13985" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7043881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32133055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Malek%20Jamshidi,%20M.%20R.,%20Zandi,%20H.%20&amp;%20Eftekhar,%20F.%20Correlation%20of%20quinolone-resistance,%20Qnr%20genes%20and%20integron%20carriage%20in%20multidrug-resistant%20community%20isolates%20of%20Klebsiella%20spp.%20Iran.%20J.%20Basic.%20Med.%20Sci.22,%201387%E2%80%931391%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Pourahmad Jaktaji, R. &amp; Mohiti, E. Study of mutations in the DNA gyrase GyrA gene of Escherichia coli. <em>Iran. J. Pharm. Res.</em><strong>9</strong>, 43–48 (2010).
</cite> [<a href="/articles/PMC3869551/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24363705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pourahmad%20Jaktaji,%20R.%20&amp;%20Mohiti,%20E.%20Study%20of%20mutations%20in%20the%20DNA%20gyrase%20GyrA%20gene%20of%20Escherichia%20coli.%20Iran.%20J.%20Pharm.%20Res.9,%2043%E2%80%9348%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Ortega-Balleza, J. L. et al. G.; Bocanegra-García, V. Genomic Analysis of Multidrug-Resistant Escherichia coli Strains Isolated in Tamaulipas, Mexico. Trop Med Infect Dis 8. (2023).</cite> [<a href="https://doi.org/10.3390/tropicalmed8100458" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10610597/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37888586/" class="usa-link">PubMed</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Karczmarczyk, M., Abbott, Y., Walsh, C., Leonard, N. &amp; Fanning, S. Characterization of multidrug-resistant Escherichia coli isolates from animals presenting at a university veterinary hospital. <em>Appl. Environ. Microbiol.</em><strong>77</strong>, 7104–7112 (2011).
</cite> [<a href="https://doi.org/10.1128/AEM.00599-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3194860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21856835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karczmarczyk,%20M.,%20Abbott,%20Y.,%20Walsh,%20C.,%20Leonard,%20N.%20&amp;%20Fanning,%20S.%20Characterization%20of%20multidrug-resistant%20Escherichia%20coli%20isolates%20from%20animals%20presenting%20at%20a%20university%20veterinary%20hospital.%20Appl.%20Environ.%20Microbiol.77,%207104%E2%80%937112%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Xiang, Y. et al. Sun, Y. A new plasmid carrying MphA causes prevalence of Azithromycin resistance in enterotoxigenic Escherichia coli serogroup O6. <em>BMC Microbiol.</em><strong>20</strong>, 247 (2020).
</cite> [<a href="https://doi.org/10.1186/s12866-020-01927-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7418381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32782021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiang,%20Y.%20et%20al.%20Sun,%20Y.%20A%20new%20plasmid%20carrying%20MphA%20causes%20prevalence%20of%20Azithromycin%20resistance%20in%20enterotoxigenic%20Escherichia%20coli%20serogroup%20O6.%20BMC%20Microbiol.20,%20247%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Potrykus, J. &amp; Wegrzyn, G. Chloramphenicol-sensitive Escherichia coli strain expressing the Chloramphenicol acetyltransferase (cat) gene. <em>Antimicrob. Agents Chemother.</em><strong>45</strong>, 3610–3612 (2001).
</cite> [<a href="https://doi.org/10.1128/AAC.45.12.3610-3612.2001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC90880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11709351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Potrykus,%20J.%20&amp;%20Wegrzyn,%20G.%20Chloramphenicol-sensitive%20Escherichia%20coli%20strain%20expressing%20the%20Chloramphenicol%20acetyltransferase%20(cat)%20gene.%20Antimicrob.%20Agents%20Chemother.45,%203610%E2%80%933612%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Li, W., Ma, J., Sun, X., Liu, M. &amp; Wang, H. Antimicrobial resistance and molecular characterization of gene cassettes from class 1 integrons in Escherichia coli strains. <em>Microb. Drug Resist.</em><strong>28</strong>, 413–418 (2022).
</cite> [<a href="https://doi.org/10.1089/mdr.2021.0172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9058876/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35076316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20W.,%20Ma,%20J.,%20Sun,%20X.,%20Liu,%20M.%20&amp;%20Wang,%20H.%20Antimicrobial%20resistance%20and%20molecular%20characterization%20of%20gene%20cassettes%20from%20class%201%20integrons%20in%20Escherichia%20coli%20strains.%20Microb.%20Drug%20Resist.28,%20413%E2%80%93418%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Yan, C. H. et al. The transcription factor CsgD contributes to engineered Escherichia coli resistance by regulating biofilm formation and stress responses. <em>Int. J. Mol. Sci.</em><strong>24</strong>, 13681 (2023).</cite> [<a href="https://doi.org/10.3390/ijms241813681" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10530992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37761984/" class="usa-link">PubMed</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Ochoa, S. A. et al. Multidrug- and extensively Drug-Resistant uropathogenic Escherichia coli clinical strains: phylogenetic groups widely associated with integrons maintain high genetic diversity. <em>Front. Microbiol.</em><strong>7</strong>, 2042 (2016).
</cite> [<a href="https://doi.org/10.3389/fmicb.2016.02042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5174082/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28066364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ochoa,%20S.%20A.%20et%20al.%20Multidrug-%20and%20extensively%20Drug-Resistant%20uropathogenic%20Escherichia%20coli%20clinical%20strains:%20phylogenetic%20groups%20widely%20associated%20with%20integrons%20maintain%20high%20genetic%20diversity.%20Front.%20Microbiol.7,%202042%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Shrestha, R. et al. Extended spectrum β-lactamase producing uropathogenic Escherichia coli and the correlation of biofilm with antibiotics resistance in Nepal. <em>Ann. Clin. Microbiol. Antimicrob.</em><strong>18</strong>, 42 (2019).
</cite> [<a href="https://doi.org/10.1186/s12941-019-0340-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6918583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31847837/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shrestha,%20R.%20et%20al.%20Extended%20spectrum%20%CE%B2-lactamase%20producing%20uropathogenic%20Escherichia%20coli%20and%20the%20correlation%20of%20biofilm%20with%20antibiotics%20resistance%20in%20Nepal.%20Ann.%20Clin.%20Microbiol.%20Antimicrob.18,%2042%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Qin, J. et al. Conserved FimH mutations in the global Escherichia coli ST131 multi-drug resistant lineage weaken interdomain interactions and alter adhesin function. <em>Comput. Struct. Biotechnol. J.</em><strong>20</strong>, 4532–4541 (2022).
</cite> [<a href="https://doi.org/10.1016/j.csbj.2022.08.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9428848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36090810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qin,%20J.%20et%20al.%20Conserved%20FimH%20mutations%20in%20the%20global%20Escherichia%20coli%20ST131%20multi-drug%20resistant%20lineage%20weaken%20interdomain%20interactions%20and%20alter%20adhesin%20function.%20Comput.%20Struct.%20Biotechnol.%20J.20,%204532%E2%80%934541%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Wang, Y., Liu, B., Grenier, D. &amp; Yi, L. Regulatory mechanisms of the LuxS/AI-2 system and bacterial resistance. <em>Antimicrob. Agents Chemother.</em><strong>63</strong>, e01186-19 (2019).</cite> [<a href="https://doi.org/10.1128/AAC.01186-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6761564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31383657/" class="usa-link">PubMed</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Zenebe, T. et al. Distribution of ß-Lactamase genes among Multidrug-Resistant and Extended-Spectrum ß-Lactamase-Producing diarrheagenic Escherichia coli from Under-Five children in Ethiopia. <em>Infect. Drug Resist.</em><strong>16</strong>, 7041–7054 (2023).
</cite> [<a href="https://doi.org/10.2147/IDR.S432743" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10637226/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37954506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zenebe,%20T.%20et%20al.%20Distribution%20of%20%C3%9F-Lactamase%20genes%20among%20Multidrug-Resistant%20and%20Extended-Spectrum%20%C3%9F-Lactamase-Producing%20diarrheagenic%20Escherichia%20coli%20from%20Under-Five%20children%20in%20Ethiopia.%20Infect.%20Drug%20Resist.16,%207041%E2%80%937054%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Wang, Q., Wei, S., Silva, A. F. &amp; Madsen, J. S. Cooperative antibiotic resistance facilitates horizontal gene transfer. <em>ISME J.</em><strong>17</strong>, 846–854 (2023).
</cite> [<a href="https://doi.org/10.1038/s41396-023-01393-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10203111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36949153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Q.,%20Wei,%20S.,%20Silva,%20A.%20F.%20&amp;%20Madsen,%20J.%20S.%20Cooperative%20antibiotic%20resistance%20facilitates%20horizontal%20gene%20transfer.%20ISME%20J.17,%20846%E2%80%93854%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Wallace, V. J., Sakowski, E. G., Preheim, S. P. &amp; Prasse, C. Bacteria exposed to antiviral drugs develop antibiotic cross-resistance and unique resistance profiles. <em>Commun. Biol.</em><strong>6</strong>, 837 (2023).
</cite> [<a href="https://doi.org/10.1038/s42003-023-05177-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10423222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37573457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wallace,%20V.%20J.,%20Sakowski,%20E.%20G.,%20Preheim,%20S.%20P.%20&amp;%20Prasse,%20C.%20Bacteria%20exposed%20to%20antiviral%20drugs%20develop%20antibiotic%20cross-resistance%20and%20unique%20resistance%20profiles.%20Commun.%20Biol.6,%20837%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Li, W. et al. Association between antibiotic resistance and increasing ambient temperature in china: an ecological study with nationwide panel data. <em>Lancet Reg. Health West. Pac.</em><strong>30</strong>, 100628 (2023).
</cite> [<a href="https://doi.org/10.1016/j.lanwpc.2022.100628" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9672962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36406382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20W.%20et%20al.%20Association%20between%20antibiotic%20resistance%20and%20increasing%20ambient%20temperature%20in%20china:%20an%20ecological%20study%20with%20nationwide%20panel%20data.%20Lancet%20Reg.%20Health%20West.%20Pac.30,%20100628%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Hecky, J. &amp; Müller, K. M. Structural perturbation and compensation by directed evolution at physiological temperature leads to thermostabilization of beta-lactamase. <em>Biochemistry</em><strong>44</strong>, 12640–12654 (2005).
</cite> [<a href="https://doi.org/10.1021/bi0501885" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16171379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hecky,%20J.%20&amp;%20M%C3%BCller,%20K.%20M.%20Structural%20perturbation%20and%20compensation%20by%20directed%20evolution%20at%20physiological%20temperature%20leads%20to%20thermostabilization%20of%20beta-lactamase.%20Biochemistry44,%2012640%E2%80%9312654%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Worthington, R. J. &amp; Melander, C. Combination approaches to combat multidrug-resistant bacteria. <em>Trends Biotechnol.</em><strong>31</strong>, 177–184 (2013).
</cite> [<a href="https://doi.org/10.1016/j.tibtech.2012.12.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3594660/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23333434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Worthington,%20R.%20J.%20&amp;%20Melander,%20C.%20Combination%20approaches%20to%20combat%20multidrug-resistant%20bacteria.%20Trends%20Biotechnol.31,%20177%E2%80%93184%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Tola, M. A., Abera, N. A., Gebeyehu, Y. M., Dinku, S. F. &amp; Tullu, K. D. High prevalence of extended-spectrum beta-lactamase-producing Escherichia coli and Klebsiella pneumoniae fecal carriage among children under five years in addis ababa, Ethiopia. <em>PLoS One</em>. <strong>16</strong>, e0258117 (2021).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0258117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8486131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34597328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tola,%20M.%20A.,%20Abera,%20N.%20A.,%20Gebeyehu,%20Y.%20M.,%20Dinku,%20S.%20F.%20&amp;%20Tullu,%20K.%20D.%20High%20prevalence%20of%20extended-spectrum%20beta-lactamase-producing%20Escherichia%20coli%20and%20Klebsiella%20pneumoniae%20fecal%20carriage%20among%20children%20under%20five%20years%20in%20addis%20ababa,%20Ethiopia.%20PLoS%20One.%2016,%20e0258117%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Mansouri, S., Savari, M., Malakian, A. &amp; Abbasi Montazeri, E. High prevalence of multidrug-resistant enterobacterales carrying extended-spectrum beta-lactamase and AmpC genes isolated from neonatal sepsis in ahvaz, Iran. <em>BMC Microbiol.</em><strong>24</strong>, 136 (2024).
</cite> [<a href="https://doi.org/10.1186/s12866-024-03285-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11040821/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38658819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mansouri,%20S.,%20Savari,%20M.,%20Malakian,%20A.%20&amp;%20Abbasi%20Montazeri,%20E.%20High%20prevalence%20of%20multidrug-resistant%20enterobacterales%20carrying%20extended-spectrum%20beta-lactamase%20and%20AmpC%20genes%20isolated%20from%20neonatal%20sepsis%20in%20ahvaz,%20Iran.%20BMC%20Microbiol.24,%20136%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Toprak, E. et al. Evolutionary paths to antibiotic resistance under dynamically sustained drug selection. <em>Nat. Genet.</em><strong>44</strong>, 101–105 (2011).
</cite> [<a href="https://doi.org/10.1038/ng.1034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3534735/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22179135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Toprak,%20E.%20et%20al.%20Evolutionary%20paths%20to%20antibiotic%20resistance%20under%20dynamically%20sustained%20drug%20selection.%20Nat.%20Genet.44,%20101%E2%80%93105%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Hain, E., Adejumo, H., Anger, B., Orenstein, J. &amp; Blaney, L. Advances in antimicrobial activity analysis of fluoroquinolone, macrolide, sulfonamide, and Tetracycline antibiotics for environmental applications through improved bacteria selection. <em>J. Hazard. Mater.</em><strong>415</strong>, 125686 (2021).
</cite> [<a href="https://doi.org/10.1016/j.jhazmat.2021.125686" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34088184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hain,%20E.,%20Adejumo,%20H.,%20Anger,%20B.,%20Orenstein,%20J.%20&amp;%20Blaney,%20L.%20Advances%20in%20antimicrobial%20activity%20analysis%20of%20fluoroquinolone,%20macrolide,%20sulfonamide,%20and%20Tetracycline%20antibiotics%20for%20environmental%20applications%20through%20improved%20bacteria%20selection.%20J.%20Hazard.%20Mater.415,%20125686%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Safdari, H. et al. Potent and selective inhibitors of class A β-lactamase: 7-prenyloxy coumarins. <em>J. Antibiot. (Tokyo)</em>. <strong>67</strong>, 373–377 (2014).
</cite> [<a href="https://doi.org/10.1038/ja.2014.9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24517922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Safdari,%20H.%20et%20al.%20Potent%20and%20selective%20inhibitors%20of%20class%20A%20%CE%B2-lactamase:%207-prenyloxy%20coumarins.%20J.%20Antibiot.%20(Tokyo).%2067,%20373%E2%80%93377%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Yang, K. B. et al. High-resolution landscape of an antibiotic binding site. <em>Nature</em><strong>622</strong>, 180–187 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06495-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10550828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37648864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20K.%20B.%20et%20al.%20High-resolution%20landscape%20of%20an%20antibiotic%20binding%20site.%20Nature622,%20180%E2%80%93187%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Laws, M., Shaaban, A. &amp; Rahman, K. M. Antibiotic resistance breakers: current approaches and future directions. <em>FEMS Microbiol. Rev.</em><strong>43</strong>, 490–516 (2019).
</cite> [<a href="https://doi.org/10.1093/femsre/fuz014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6736374/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31150547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laws,%20M.,%20Shaaban,%20A.%20&amp;%20Rahman,%20K.%20M.%20Antibiotic%20resistance%20breakers:%20current%20approaches%20and%20future%20directions.%20FEMS%20Microbiol.%20Rev.43,%20490%E2%80%93516%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Soothill, J. S., Ward, R. &amp; Girling, A. J. The IC50: an exactly defined measure of antibiotic sensitivity. <em>J. Antimicrob. Chemother.</em><strong>29</strong>, 137–139 (1992).
</cite> [<a href="https://doi.org/10.1093/jac/29.2.137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1506328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Soothill,%20J.%20S.,%20Ward,%20R.%20&amp;%20Girling,%20A.%20J.%20The%20IC50:%20an%20exactly%20defined%20measure%20of%20antibiotic%20sensitivity.%20J.%20Antimicrob.%20Chemother.29,%20137%E2%80%93139%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Huemer, M., Mairpady Shambat, S., Brugger, S. D. &amp; Zinkernagel, A. S. Antibiotic resistance and persistence-Implications for human health and treatment perspectives. <em>EMBO Rep.</em><strong>21</strong>, e51034 (2020).
</cite> [<a href="https://doi.org/10.15252/embr.202051034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7726816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33400359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huemer,%20M.,%20Mairpady%20Shambat,%20S.,%20Brugger,%20S.%20D.%20&amp;%20Zinkernagel,%20A.%20S.%20Antibiotic%20resistance%20and%20persistence-Implications%20for%20human%20health%20and%20treatment%20perspectives.%20EMBO%20Rep.21,%20e51034%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Li, Y., Feng, T. &amp; Wang, Y. The role of bacterial signaling networks in antibiotics response and resistance regulation. <em>Mar. Life Sci. Technol.</em><strong>4</strong>, 163–178 (2022).
</cite> [<a href="https://doi.org/10.1007/s42995-022-00126-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10077285/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37073223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Y.,%20Feng,%20T.%20&amp;%20Wang,%20Y.%20The%20role%20of%20bacterial%20signaling%20networks%20in%20antibiotics%20response%20and%20resistance%20regulation.%20Mar.%20Life%20Sci.%20Technol.4,%20163%E2%80%93178%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Jang, S. AcrAB-TolC, a major efflux pump in gram negative bacteria: toward Understanding its operation mechanism. <em>BMB Rep.</em><strong>56</strong>, 326–334 (2023).
</cite> [<a href="https://doi.org/10.5483/BMBRep.2023-0070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10315565/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37254571/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jang,%20S.%20AcrAB-TolC,%20a%20major%20efflux%20pump%20in%20gram%20negative%20bacteria:%20toward%20Understanding%20its%20operation%20mechanism.%20BMB%20Rep.56,%20326%E2%80%93334%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Gaurav, A., Bakht, P., Saini, M., Pandey, S. &amp; Pathania, R. Role of bacterial efflux pumps in antibiotic resistance, virulence, and strategies to discover novel efflux pump inhibitors. <em>Microbiology (Reading) 169</em>. (2023).</cite> [<a href="https://doi.org/10.1099/mic.0.001333" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10268834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37224055/" class="usa-link">PubMed</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Hernández-Durán, M. et al. Inside-out, antimicrobial resistance mediated by efflux pumps in clinical strains of acinetobacter baumannii isolated from burn wound infections. <em>Braz. J. Microbiol.</em><strong>55</strong>(4), 3629–3641 (2024).</cite> [<a href="https://doi.org/10.1007/s42770-024-01461-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11711420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39044104/" class="usa-link">PubMed</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Revitt-Mills, S. A. et al. Defects in DNA double-strand break repair resensitize antibiotic-resistant Escherichia coli to multiple bacteriodddal antibiotics. <em>Microbiologyopen</em><strong>11</strong>, e1316 (2022).
</cite> [<a href="https://doi.org/10.1002/mbo3.1316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9500592/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36314749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Revitt-Mills,%20S.%20A.%20et%20al.%20Defects%20in%20DNA%20double-strand%20break%20repair%20resensitize%20antibiotic-resistant%20Escherichia%20coli%20to%20multiple%20bacteriodddal%20antibiotics.%20Microbiologyopen11,%20e1316%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Yakovlieva, L., Fülleborn, J. A. &amp; Walvoort, M. T. C. Opportunities and challenges of bacterial glycosylation for the development of novel antibacterial strategies. <em>Front. Microbiol.</em><strong>12</strong>, 745702 (2021).
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.745702" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8498110/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34630370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yakovlieva,%20L.,%20F%C3%BClleborn,%20J.%20A.%20&amp;%20Walvoort,%20M.%20T.%20C.%20Opportunities%20and%20challenges%20of%20bacterial%20glycosylation%20for%20the%20development%20of%20novel%20antibacterial%20strategies.%20Front.%20Microbiol.12,%20745702%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Deter, H. S., Hossain, T. &amp; Butzin, N. C. Antibiotic tolerance is associated with a broad and complex transcriptional response in E. coli. <em>Sci. Rep.</em><strong>11</strong>, 6112 (2021).
</cite> [<a href="https://doi.org/10.1038/s41598-021-85509-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7969968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33731833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deter,%20H.%20S.,%20Hossain,%20T.%20&amp;%20Butzin,%20N.%20C.%20Antibiotic%20tolerance%20is%20associated%20with%20a%20broad%20and%20complex%20transcriptional%20response%20in%20E.%20coli.%20Sci.%20Rep.11,%206112%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373996/bin/41598_2025_16122_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (707.7KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data is provided within the manuscript or supplementary information files.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-16122-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_16122.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373996/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373996/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373996%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373996/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373996/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373996/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846736/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373996/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846736/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373996/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373996/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="yPpJUtEm9UzS5T7rf5KyNOM9Cp8s1ob7IQdAtlyehUa1byJ9k73Lce68fVhRDGyP">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
