
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Comprehensive mapping of Arabidopsis alternative splicing landscape reveals key insights into plant development and immunity - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="37B2C32A7FFF9BD306C32A000BEF74C4.m_1" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11953613/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Comprehensive mapping of Arabidopsis alternative splicing landscape reveals key insights into plant development and immunity">
<meta name="citation_author" content="Teura Barff">
<meta name="citation_author_institution" content="Department of Chemistry, Biochemistry and Physics and Groupe de Recherche en Biologie Végétale, Université du Québec à Trois‐Rivières, Trois‐Rivières, Québec, Canada">
<meta name="citation_author" content="Ingrid Berenice Sanchez Carrillo">
<meta name="citation_author_institution" content="Department of Chemistry, Biochemistry and Physics and Groupe de Recherche en Biologie Végétale, Université du Québec à Trois‐Rivières, Trois‐Rivières, Québec, Canada">
<meta name="citation_author" content="Valeria Paola Parra Gutiérrez">
<meta name="citation_author_institution" content="Department of Chemistry, Biochemistry and Physics and Groupe de Recherche en Biologie Végétale, Université du Québec à Trois‐Rivières, Trois‐Rivières, Québec, Canada">
<meta name="citation_author" content="Mélodie B Plourde">
<meta name="citation_author_institution" content="Department of Chemistry, Biochemistry and Physics and Groupe de Recherche en Biologie Végétale, Université du Québec à Trois‐Rivières, Trois‐Rivières, Québec, Canada">
<meta name="citation_author" content="David L Joly">
<meta name="citation_author_institution" content="Département de biologie, Université de Moncton, Moncton, New Brunswick, Canada">
<meta name="citation_author" content="Hugo Germain">
<meta name="citation_author_institution" content="Department of Chemistry, Biochemistry and Physics and Groupe de Recherche en Biologie Végétale, Université du Québec à Trois‐Rivières, Trois‐Rivières, Québec, Canada">
<meta name="citation_publication_date" content="2025 Mar 28">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="2">
<meta name="citation_firstpage" content="e70022">
<meta name="citation_doi" content="10.1002/tpg2.70022">
<meta name="citation_pmid" content="40156198">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11953613/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11953613/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11953613/pdf/TPG2-18-e70022.pdf">
<meta name="description" content="The different steps of alternative splicing (AS) in plants and its regulatory mechanisms have already been studied extensively. Its broader impact on cell identity, plant immunity‐related genes, and their study as a whole remains to be investigated. ...">
<meta name="og:title" content="Comprehensive mapping of Arabidopsis alternative splicing landscape reveals key insights into plant development and immunity">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The different steps of alternative splicing (AS) in plants and its regulatory mechanisms have already been studied extensively. Its broader impact on cell identity, plant immunity‐related genes, and their study as a whole remains to be investigated. ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11953613/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11953613">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70022"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70022.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11953613%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11953613/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11953613/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11953613/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70022" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Mar 28;18(2):e70022. doi: <a href="https://doi.org/10.1002/tpg2.70022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70022</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Comprehensive mapping of Arabidopsis alternative splicing landscape reveals key insights into plant development and immunity</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Barff%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Teura Barff</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Teura Barff</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, Biochemistry and Physics and Groupe de Recherche en Biologie Végétale, Université du Québec à Trois‐Rivières, Trois‐Rivières, Québec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Barff%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Teura Barff</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanchez%20Carrillo%20IB%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ingrid Berenice Sanchez Carrillo</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ingrid Berenice Sanchez Carrillo</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, Biochemistry and Physics and Groupe de Recherche en Biologie Végétale, Université du Québec à Trois‐Rivières, Trois‐Rivières, Québec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanchez%20Carrillo%20IB%22%5BAuthor%5D" class="usa-link"><span class="name western">Ingrid Berenice Sanchez Carrillo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Parra%20Guti%C3%A9rrez%20VP%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Valeria Paola Parra Gutiérrez</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Valeria Paola Parra Gutiérrez</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, Biochemistry and Physics and Groupe de Recherche en Biologie Végétale, Université du Québec à Trois‐Rivières, Trois‐Rivières, Québec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Parra%20Guti%C3%A9rrez%20VP%22%5BAuthor%5D" class="usa-link"><span class="name western">Valeria Paola Parra Gutiérrez</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22B%20Plourde%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mélodie B Plourde</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mélodie B Plourde</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, Biochemistry and Physics and Groupe de Recherche en Biologie Végétale, Université du Québec à Trois‐Rivières, Trois‐Rivières, Québec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22B%20Plourde%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mélodie B Plourde</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joly%20DL%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">David L Joly</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">David L Joly</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Département de biologie, Université de Moncton, Moncton, New Brunswick, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joly%20DL%22%5BAuthor%5D" class="usa-link"><span class="name western">David L Joly</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Germain%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Hugo Germain</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Hugo Germain</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, Biochemistry and Physics and Groupe de Recherche en Biologie Végétale, Université du Québec à Trois‐Rivières, Trois‐Rivières, Québec, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Germain%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hugo Germain</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270022-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, Biochemistry and Physics and Groupe de Recherche en Biologie Végétale, Université du Québec à Trois‐Rivières, Trois‐Rivières, Québec, Canada</div>
<div id="tpg270022-aff-0002">
<sup>
<sup>2</sup>
</sup>Département de biologie, Université de Moncton, Moncton, New Brunswick, Canada</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Hugo Germain, Department of Chemistry, Biochemistry and Physics and Groupe de Recherche en Biologie Végétale, Université du Québec à Trois‐Rivières, Trois‐Rivières, Québec, G9A 5H9, Canada. Email: <span>Hugo.germain@uqtr.ca</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Feb 24; Received 2024 Sep 16; Accepted 2025 Feb 27; Issue date 2025 Jun.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11953613  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40156198/" class="usa-link">40156198</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The different steps of alternative splicing (AS) in plants and its regulatory mechanisms have already been studied extensively. Its broader impact on cell identity, plant immunity‐related genes, and their study as a whole remains to be investigated. Using transgenic plants, we sorted 11 different <em>Arabidopsis thaliana</em> cell types ranging from root to aerial organs using fluorescence‐activated cell sorting. RNA‐seq data were analyzed with vast‐tools and enabled us to generate a high‐resolution AS landscape across multiple cell types, all collected through the same experimental procedure. The analysis of cell type‐specific gene expression and alternative splicing events highlights the importance of AS on transcription and AS regulation itself. AS is also shown to be tightly linked to cell identity. By using closely related cell types, we captured alternative splicing events involved in specific stages of plant development. The columella cells, among others, show intensified AS regulation and an interesting splicing profile, especially regarding immunity‐related genes. Overall, our analysis brings a valuable tool in the study of cell type identity, plant immunity, and AS.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270022-list-10001" class="list" style="list-style-type:disc">
<li><p>Alternative splicing plays a role in regulating transcription.</p></li>
<li><p>Both alternative splicing and gene deregulation influence distinct sets of genes.</p></li>
<li><p>Alternative splicing is closely associated with determining cell identity.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270022-dl-0001">
<dt>AS</dt>
<dd><p>alternative splicing</p></dd>
<dt>DEG</dt>
<dd><p>differentially expressed cell type‐specific genes</p></dd>
<dt>DEX</dt>
<dd><p>dexamethasone</p></dd>
<dt>DSE</dt>
<dd><p>differentially spliced cell type‐specific event</p></dd>
<dt>DSG</dt>
<dd><p>differentially spliced cell type‐specific genes</p></dd>
<dt>PSI</dt>
<dd><p>percentage spliced‐in value</p></dd>
</dl>
<section id="tpg270022-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Alternative splicing (AS) is the process by which a single gene produces multiple messenger RNAs. It is believed that AS provides eukaryotes the level of transcriptome and proteome complexity necessary to adapt to environmental cues (Chaudhary et al., <a href="#tpg270022-bib-0007" class="usa-link" aria-describedby="tpg270022-bib-0007">2019</a>; Reddy et al., <a href="#tpg270022-bib-0055" class="usa-link" aria-describedby="tpg270022-bib-0055">2013</a>). This widespread process is well documented in mammals and higher plants. In the human genome, it is estimated that approximately 94% of multiexonic genes undergo AS (Wang et al., <a href="#tpg270022-bib-0069" class="usa-link" aria-describedby="tpg270022-bib-0069">2008</a>), while plants are estimated to alternatively splice up to 62% of multiexonic genes under normal developmental conditions (Marquez et al., <a href="#tpg270022-bib-0049" class="usa-link" aria-describedby="tpg270022-bib-0049">2012</a>; Martín et al., <a href="#tpg270022-bib-0050" class="usa-link" aria-describedby="tpg270022-bib-0050">2021</a>). The four main types of alternative splicing events (ASEs) are exon skipping (EX), intron retention (IR), alternative donor site (ALTD), and alternative acceptor site (ALTA). Intron retention is the most prevalent type of ASE in plants, with up to 65% of intron‐containing genes subjected to AS (Marquez et al., <a href="#tpg270022-bib-0049" class="usa-link" aria-describedby="tpg270022-bib-0049">2012</a>; Ner‐Gaon et al., <a href="#tpg270022-bib-0052" class="usa-link" aria-describedby="tpg270022-bib-0052">2004</a>), as opposed to 5% in humans (Wang et al., <a href="#tpg270022-bib-0069" class="usa-link" aria-describedby="tpg270022-bib-0069">2008</a>). AS may lead to (i) translation into functional proteins, thus diversifying proteome complexity, (ii) introduction of a premature terminal codon leading to degradation through nonsense‐mediated mRNA decay (NMD) (Drechsel et al., <a href="#tpg270022-bib-0011" class="usa-link" aria-describedby="tpg270022-bib-0011">2013</a>; Kalyna et al., <a href="#tpg270022-bib-0034" class="usa-link" aria-describedby="tpg270022-bib-0034">2012</a>; Ohtani &amp; Wachter, <a href="#tpg270022-bib-0054" class="usa-link" aria-describedby="tpg270022-bib-0054">2019</a>), (iii) nuclear sequestration for further mRNA processing on demand (Göhring et al., <a href="#tpg270022-bib-0018" class="usa-link" aria-describedby="tpg270022-bib-0018">2014</a>; Hartmann et al., <a href="#tpg270022-bib-0022" class="usa-link" aria-describedby="tpg270022-bib-0022">2018</a>; Yang et al., <a href="#tpg270022-bib-0073" class="usa-link" aria-describedby="tpg270022-bib-0073">2017</a>), and (iv) translation into truncated proteins (Liu et al., <a href="#tpg270022-bib-0041" class="usa-link" aria-describedby="tpg270022-bib-0041">2013</a>). While AS regulation mostly relies on splicing factors like serine–arginine‐rich (SR) proteins (Erkelenz et al., <a href="#tpg270022-bib-0015" class="usa-link" aria-describedby="tpg270022-bib-0015">2013</a>; Hertel &amp; Maniatis, <a href="#tpg270022-bib-0027" class="usa-link" aria-describedby="tpg270022-bib-0027">1999</a>) and heterogeneous nuclear ribonucleoproteins (hnRNPs) (Huelga et al., <a href="#tpg270022-bib-0031" class="usa-link" aria-describedby="tpg270022-bib-0031">2012</a>), AS is considered a co‐transcriptional process, thus it will be greatly impacted by chromatin structure and by the speed and phosphorylation state of the mRNA polymerase II (Zhu et al., <a href="#tpg270022-bib-0076" class="usa-link" aria-describedby="tpg270022-bib-0076">2018</a>).</p>
<p>Evidence of the impact of AS in plant development and cell type differentiation keeps emerging. While uncovering the role of specific AS isoforms in a precise stage of plant growth has been the focus of many, the advancement of high‐throughput transcriptomic analyses has enabled the mapping of the plant AS landscape to become increasingly common. For example, the AS mapping of the potato (<em>Solanum tuberosum</em> L.), the black gram (<em>Vigna mungo</em>), and even the cotton (<em>Gossypium hirsutum</em>) were recently published (Hazra et al., <a href="#tpg270022-bib-0025" class="usa-link" aria-describedby="tpg270022-bib-0025">2023</a>; Ogungbayi et al., <a href="#tpg270022-bib-0053" class="usa-link" aria-describedby="tpg270022-bib-0053">2023</a>; Zhang et al., <a href="#tpg270022-bib-0074" class="usa-link" aria-describedby="tpg270022-bib-0074">2022</a>). <em>Arabidopsis thaliana</em> (Arabidopsis) is one of the most studied plant species, including when it comes to AS research. High‐resolution databases compiling the AS landscapes of Arabidopsis have brought valuable information and tools to understand the global impact of plant AS (Guo et al., <a href="#tpg270022-bib-0020" class="usa-link" aria-describedby="tpg270022-bib-0020">2023</a>; Liu et al., <a href="#tpg270022-bib-0042" class="usa-link" aria-describedby="tpg270022-bib-0042">2023</a>; Martín et al., <a href="#tpg270022-bib-0050" class="usa-link" aria-describedby="tpg270022-bib-0050">2021</a>; Zhang et al., <a href="#tpg270022-bib-0074" class="usa-link" aria-describedby="tpg270022-bib-0074">2022</a>). Some studies focused on AS across cell types (Li et al., <a href="#tpg270022-bib-0039" class="usa-link" aria-describedby="tpg270022-bib-0039">2016</a>; Martín et al., <a href="#tpg270022-bib-0050" class="usa-link" aria-describedby="tpg270022-bib-0050">2021</a>; Misra et al., <a href="#tpg270022-bib-0051" class="usa-link" aria-describedby="tpg270022-bib-0051">2023</a>), while others focused on AS under various conditions like abiotic and biotic stresses (Guo et al., <a href="#tpg270022-bib-0020" class="usa-link" aria-describedby="tpg270022-bib-0020">2023</a>; Martín et al., <a href="#tpg270022-bib-0050" class="usa-link" aria-describedby="tpg270022-bib-0050">2021</a>), the latter highlighting the broader importance of AS when facing immune challenges.</p>
<p>Among all the factors that influence plant immunity, the impact of cell identity has until recently been seldom investigated. The interest in organ‐specific defense response emerged with the need to understand colonization patterns of naturally occurring pathogens. Some pathogens will favor given organs while others are able to infect the entire host (Balmer &amp; Mauch‐Mani, <a href="#tpg270022-bib-0001" class="usa-link" aria-describedby="tpg270022-bib-0001">2013</a>). One of the first studies recording organ‐specific disease was done on the oomycete <em>Hyaloperonospora arabidopsidis</em>. During incompatible interaction, <em>H. arabidopsidis</em> causes resistant‐like symptoms in Arabidopsis leaves but not in roots, although the plant still expresses the immune regulators <em>EDS1</em> and <em>NPR1</em> in both plant parts (Hermanns et al., <a href="#tpg270022-bib-0026" class="usa-link" aria-describedby="tpg270022-bib-0026">2003</a>). Although evidence of organ‐specific immune response has been mounting in the past two decades, the study of cell type‐specific immunity‐related genes is still at its starting point.</p>
<p>For example, the perception of the flagellin peptide flg22 by multiple Arabidopsis root cell types was shown to be sufficient to trigger PAMP‐Triggered Immunity (PTI); however, the intensity of the triggered response varies according to the cell type (Wyrsch et al., <a href="#tpg270022-bib-0070" class="usa-link" aria-describedby="tpg270022-bib-0070">2015</a>). In more detail, flg22 and the plant elicitor peptide 1 (Pep1) induce a different transcriptional network in the root epidermis, cortex, and pericycle by modulating the expression of genes with various transcription factor binding motifs (Rich‐Griffin et al., <a href="#tpg270022-bib-0056" class="usa-link" aria-describedby="tpg270022-bib-0056">2020</a>). Very recently, Tang et al. (<a href="#tpg270022-bib-0064" class="usa-link" aria-describedby="tpg270022-bib-0064">2023</a>) characterized the Arabidopsis leaf cell type‐specific responses to <em>Colletotrichum higginsianum</em> infection using single‐cell transcriptomics, further asserting the importance of cell identity on plant immunity.</p>
<p>Many steps of the plant defense response are paired with AS of resistance‐related transcripts. For instance, PTI induction will trigger AS of the CPK28 transcript, leading to the accumulation of a nonfunctional truncated CPK28 protein, thus alleviating its role as a negative regulator of plant defense (Dressano et al., <a href="#tpg270022-bib-0012" class="usa-link" aria-describedby="tpg270022-bib-0012">2020</a>). Similarly, the resistance gene <em>RPS4</em> from Arabidopsis, which confers resistance to <em>Pseudomonas syringae</em> pv <em>tomato</em> strain DC3000 (DC3000) expressing AvrRps4, is also subjected to AS. More specifically, removal of one or two of the introns found in the dominant splice variant of <em>RPS4</em> did not affect its expression but suppressed the function of the protein it encodes. This suggests that multiple variants of the RPS4 protein are required for proper immune response (Zhang &amp; Gassmann, <a href="#tpg270022-bib-0075" class="usa-link" aria-describedby="tpg270022-bib-0075">2003</a>). Phytopathogens are also able to modulate the host splicing machinery, as is the case for the <em>Phytophthora sojae</em> effector PsAvr3c (Huang et al., <a href="#tpg270022-bib-0030" class="usa-link" aria-describedby="tpg270022-bib-0030">2017</a>). Those are just a few examples showing the importance of proper mRNA splicing in plant defense.</p>
<p>Despite evidence of cell identity, immunity‐related genes, and AS being interconnected, very little effort has been invested so far to study these processes as a whole. While extensive mapping of the Arabidopsis AS landscape has already been performed, these databases were generated using publicly available aggregated datasets from varying sources. The isolation of different cell types is a limiting step, as the number of cell types varies from one developmental stage to another, rendering this process arduous. Furthermore, techniques like manual organ dissection and laser‐capture microdissection can be less precise and unpractical. In this study, we used cell type‐specific reporter transgenic lines coupled with fluorescence‐activated cell sorting (FACS) to collect 11 different Arabidopsis cell types from roots, stem, and leaves. In an effort to uncover the impact of AS on cell identity and immunity‐related genes, we compared gene expression and alternatively spliced events across these 11 different cell types with a particular focus on immunity‐related transcripts.</p>
<p>Our results demonstrate the feasibility of a pipeline combining flow cytometry sorting of fluorescent protoplasts from 11 cell types, followed by RNA extraction and deep sequencing. We observe that gene expression is poorly connected with AS and successfully identify deregulated genes and AS events that are specific to individual cell types or groups of cell types. We also uncover the previously unreported importance of AS in regulating immunity‐related genes, more specifically in columella cells. The use of closely related cell types enabled us to capture subtle changes in AS regulation needed during development and immune regulation. Overall, the dataset generated here provides a valuable tool in the study of cell type identity, plant immunity‐related genes, and AS.</p></section><section id="tpg270022-sec-0020"><h2 class="pmc_sec_title">2. METHODS</h2>
<section id="tpg270022-sec-0030"><h3 class="pmc_sec_title">2.1. Plant material and growth conditions</h3>
<p>The <em>Arabidopsis thaliana</em> Col‐0 transgenic plants used in this study were kindly provided by Pr. Sebastian Wolf from Universität Heidelberg (Schürholz et al., <a href="#tpg270022-bib-0059" class="usa-link" aria-describedby="tpg270022-bib-0059">2018</a>) and are listed in Table <a href="#tpg270022-tbl-0001" class="usa-link">1</a>. Plants were grown on Murashige and Skoog (MS) medium (2.15 g/L MS basal salt mixture [Phytotech], 1% [w/v] agar, and pH 5.8) supplemented with 30 µM dexamethasone (DEX) (Sigma‐Aldrich). Once sowed, plates were placed in growth chambers with 10‐/14‐h light/dark cycles. To ensure enough cells from each cell type were present, transgenic lines harboring aerial cell type‐specific promoters were grown for longer than the ones with root cell type‐specific promoters. Therefore, p<em>LTP1</em>, p<em>UFO</em>, p<em>REV</em>, p<em>ML1</em>, and p<em>APL</em> were incubated horizontally for 21 days, while p<em>ATHB8</em>, p<em>SCR</em>, p<em>TMO5</em>, p<em>SMLX5</em>, p<em>PXY</em>, and p<em>CASP1</em> transgenic lines were incubated vertically for 14 days.</p>
<section class="tw xbox font-sm" id="tpg270022-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Cell type‐specific promoters used in this study.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Category</th>
<th align="left" rowspan="1" colspan="1">Promoter</th>
<th align="left" rowspan="1" colspan="1">Gene</th>
<th align="left" rowspan="1" colspan="1">Gene ID</th>
<th align="left" rowspan="1" colspan="1">Cell type</th>
<th align="left" rowspan="1" colspan="1">Reference</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="4" align="left" colspan="1">Aerial</td>
<td align="left" rowspan="1" colspan="1">p<em>ML1</em>
</td>
<td align="left" rowspan="1" colspan="1">MERISTEM LAYER 1</td>
<td align="left" rowspan="1" colspan="1">AT4G21750</td>
<td align="left" rowspan="1" colspan="1">L1 Layer epidermis</td>
<td align="left" rowspan="1" colspan="1">(Session et al., <a href="#tpg270022-bib-0060" class="usa-link" aria-describedby="tpg270022-bib-0060">1999</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">p<em>REV</em>
</td>
<td align="left" rowspan="1" colspan="1">REVOLUTA</td>
<td align="left" rowspan="1" colspan="1">AT5G60690</td>
<td align="left" rowspan="1" colspan="1">Shoot apical meristem (SAM) central zone</td>
<td align="left" rowspan="1" colspan="1">(Otsuga et al., <a href="#tpg270022-bib-0080" class="usa-link" aria-describedby="tpg270022-bib-0080">2008</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">p<em>UFO</em>
</td>
<td align="left" rowspan="1" colspan="1">UNUSUAL FLORAL ORGANS</td>
<td align="left" rowspan="1" colspan="1">AT1G30950</td>
<td align="left" rowspan="1" colspan="1">SAM peripheral zone</td>
<td align="left" rowspan="1" colspan="1">(Hepworth et al., <a href="#tpg270022-bib-0077" class="usa-link" aria-describedby="tpg270022-bib-0077">2006</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">p<em>LTP1</em>
</td>
<td align="left" rowspan="1" colspan="1">LOCALIZED LIPID TRANSFERT PROTEIN1</td>
<td align="left" rowspan="1" colspan="1">AT2G38540</td>
<td align="left" rowspan="1" colspan="1">Epidermis in stem</td>
<td align="left" rowspan="1" colspan="1">(Thoma et al., <a href="#tpg270022-bib-0066" class="usa-link" aria-describedby="tpg270022-bib-0066">1994</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Root</td>
<td align="left" rowspan="1" colspan="1">p<em>APL</em>
</td>
<td align="left" rowspan="1" colspan="1">ALTERED PHLOEM DEVELOPMENT</td>
<td align="left" rowspan="1" colspan="1">AT1G79430</td>
<td align="left" rowspan="1" colspan="1">Phloem (differentiating)</td>
<td align="left" rowspan="1" colspan="1">(Bonke et al., <a href="#tpg270022-bib-0005" class="usa-link" aria-describedby="tpg270022-bib-0005">2003</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p<em>SCR</em>
</td>
<td align="left" rowspan="1" colspan="1">SCARECROW</td>
<td align="left" rowspan="1" colspan="1">AT3G54220</td>
<td align="left" rowspan="1" colspan="1">Endodermis, quiescent center (QC) in the root apical meristem (RAM), starch sheath in stem</td>
<td align="left" rowspan="1" colspan="1">(Malamy &amp; Benfey, <a href="#tpg270022-bib-0048" class="usa-link" aria-describedby="tpg270022-bib-0048">1997</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p<em>TMO5</em>
</td>
<td align="left" rowspan="1" colspan="1">TARGET OF MONOPTEROS5</td>
<td align="left" rowspan="1" colspan="1">AT3G25710</td>
<td align="left" rowspan="1" colspan="1">Xylem precursor</td>
<td align="left" rowspan="1" colspan="1">(De Rybel et al., <a href="#tpg270022-bib-0078" class="usa-link" aria-describedby="tpg270022-bib-0078">2013</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p<em>PXY</em>
</td>
<td align="left" rowspan="1" colspan="1">PHLOEM INTERCALATED WITH XYLEM</td>
<td align="left" rowspan="1" colspan="1">AT5G61480</td>
<td align="left" rowspan="1" colspan="1">(Pro)cambium</td>
<td align="left" rowspan="1" colspan="1">(Wang et al., <a href="#tpg270022-bib-0082" class="usa-link" aria-describedby="tpg270022-bib-0082">2013</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p<em>CASP1</em>
</td>
<td align="left" rowspan="1" colspan="1">CASPARIAN STRIP MEMBRANE DOMAIN PROTEINS</td>
<td align="left" rowspan="1" colspan="1">AT2G36100</td>
<td align="left" rowspan="1" colspan="1">Differentiating endodermis</td>
<td align="left" rowspan="1" colspan="1">(Roppolo et al., <a href="#tpg270022-bib-0057" class="usa-link" aria-describedby="tpg270022-bib-0057">2011</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p<em>SMXL5</em>
</td>
<td align="left" rowspan="1" colspan="1">SUPPRESSOR OF MAX2 1‐LIKE5</td>
<td align="left" rowspan="1" colspan="1">AT5G57130</td>
<td align="left" rowspan="1" colspan="1">Phloem (precursors)</td>
<td align="left" rowspan="1" colspan="1">(Wallner et al., <a href="#tpg270022-bib-0068" class="usa-link" aria-describedby="tpg270022-bib-0068">2017</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">p<em>ATHB‐8</em>
</td>
<td align="left" rowspan="1" colspan="1">HOMEOBOX GENE 8</td>
<td align="left" rowspan="1" colspan="1">AT4G32880</td>
<td align="left" rowspan="1" colspan="1">Procambium, xylem precursors and columella in RAM</td>
<td align="left" rowspan="1" colspan="1">(Baima et al., 1995)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270022-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270022-sec-0040"><h4 class="pmc_sec_title">Core Ideas</h4>
<ul id="tpg270022-list-0001" class="list" style="list-style-type:disc">
<li><p>Alternative splicing plays a role in regulating transcription.</p></li>
<li><p>Both alternative splicing and gene deregulation influence distinct sets of genes.</p></li>
<li><p>Alternative splicing is closely associated with determining cell identity.</p></li>
</ul></section></section></section><section id="tpg270022-sec-0050"><h3 class="pmc_sec_title">2.2. Protoplast extraction</h3>
<p>Protoplasts were extracted from the above‐mentioned seedlings following the protocol described by Bargmann and Birnbaum (<a href="#tpg270022-bib-0002" class="usa-link" aria-describedby="tpg270022-bib-0002">2010</a>) and Sanchez Carrillo et al. (<a href="#tpg270022-bib-0058" class="usa-link" aria-describedby="tpg270022-bib-0058">2023</a>) with the following modifications. For each Arabidopsis transgenic line, sections of plants containing the necessary cell type were cut from 30 MS plates with a minimum of 200 seedlings per plate. The sections were incubated in protoplasting solution (1% cellulase from <em>Trichoderma</em> sp. [Sigma‐Aldrich], 0.3% Macerozyme [Sigma‐Aldrich], 0.4 M mannitol, 20 mM KCl, 20 mM MES, 10 mM CaCl<sub>2</sub>, 5 mM β‐mercaptoethanol, and 0.1% BSA) for 90 min on an orbital shaker at 75 rpm. Protoplasts were passed through a 30 µm filter, centrifuged for 10 min at 500 <em>g</em>, and resuspended in fresh W5 buffer (154 mM NaCl, 125 mM CaCl<sub>2</sub>, 5 mM KCl, 2 mM MES, and pH 5.7). To ensure mTurquoise2 expression remained after extraction, protoplasts were checked under confocal microscopy before proceeding to FACS.</p></section><section id="tpg270022-sec-0060"><h3 class="pmc_sec_title">2.3. Confocal microscopy</h3>
<p>mTurquoise2 expression was observed on seedlings prior to protoplast isolation with a Leica TCS SP8 confocal laser scanning microscope (Leica Microsystem). Seedlings were stained with 1 ng/mL propidium iodide (PI) for 30 min before mounting in water under a coverslip. Excitation/Emission wavelengths during acquisition were 405 nm/460–520 nm for mTurquoise2 and 488 nm/590–660 nm for PI (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S1</a>). Protoplasts before and after cell sorting were also visualized for mTurquoise2 expression using the same excitation and emission wavelength mentioned above (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S2A,B</a>).</p></section><section id="tpg270022-sec-0070"><h3 class="pmc_sec_title">2.4. Fluorescence‐activated cell sorting</h3>
<p>FACS was performed on freshly extracted protoplasts using the BD FACS Melody Cell Sorter with a 100 µm nozzle and W5 buffer as sheath fluid. Flow rate was 1%, and event rate was kept under 6000 events/s. The BD FACS Melody Cell works with an operating pressure of 23 psi and a drop frequency of 34 kHz. mTurquoise2‐positive protoplasts were identified by plotting events obtained with a laser of 405 nm and a bandpass filter of 528/545 nm against events obtained with a laser of 488 nm and a bandpass of 527/532. For each experiment, protoplasts extracted from transgenic plants grown on MS media without DEX were used to differentiate mTurquoise2 fluorescence from autofluorescence (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S2C–H</a>). A total of 50,000–100,000 cells were sorted and collected in three to six replicates for each cell type. Samples were harvested at 4°C directly in RLT lysis buffer (Qiagen) and stored at −80°C. In total, 48 samples were collected and processed for sequencing (Table <a href="#tpg270022-supinfo-0001" class="usa-link">S1</a>).</p></section><section id="tpg270022-sec-0080"><h3 class="pmc_sec_title">2.5. RNA isolation and sequencing</h3>
<p>RNA was extracted with the RNeasy Micro kit (Qiagen), with the recommended DNase treatment according to the manufacturer's protocol. Quantification was performed with the Quanti‐iT Ribogreen RNA reagent kit (Invitrogen) using the manufacturer's low‐range standard curve. The Synergy H1 Multimode microplate reader (Biotek) was used to observe the Quant‐iT Ribogreen RNA reagent's fluorescence when bound to RNA, with an excitation/emission wavelength of 485/520 nm. QC was done with a 2100 Bioanalyzer (Agilent). Libraries were generated with a low input RNA Library Prep kit for Illumina (NEB) and sequenced on an Illumina NovaSeq 6000 S4 system in paired‐end reads (100 bp) by the <em>Centre d'expertise et de services Génome Québec</em>.</p></section><section id="tpg270022-sec-0090"><h3 class="pmc_sec_title">2.6. RNA‐seq data analysis</h3>
<p>Over 2 billion raw reads were trimmed using Trimmomatic v0.39 (Bolger et al., <a href="#tpg270022-bib-0004" class="usa-link" aria-describedby="tpg270022-bib-0004">2014</a>) to remove the sequencing adapters. The alignment was performed with vast‐tools v2.2.0 (Irimia et al., <a href="#tpg270022-bib-0032" class="usa-link" aria-describedby="tpg270022-bib-0032">2014</a>; Tapial et al., <a href="#tpg270022-bib-0065" class="usa-link" aria-describedby="tpg270022-bib-0065">2017</a>). This step is executed within vast‐tools with bowtie 1.0.0 (Langmead et al., <a href="#tpg270022-bib-0036" class="usa-link" aria-describedby="tpg270022-bib-0036">2009</a>). Trimmed reads were mapped to the Arabidopsis (araTha10 version 23.06.20) library from PASTDB. This database is composed of exon–exon and exon–intron junctions annotations generated using TAIR10, Ensembl Plants v31, and a collection of data as scaffold (Martín et al., <a href="#tpg270022-bib-0050" class="usa-link" aria-describedby="tpg270022-bib-0050">2021</a>). The vast‐tools alignment tool slices reads to 50 nt fragments with a sliding window of 25 nt to increase the mapping of junctions. This step consists of several modules that enable the identification of exon skipping, intron retention, and alternative donor and acceptor events (Tapial et al., <a href="#tpg270022-bib-0065" class="usa-link" aria-describedby="tpg270022-bib-0065">2017</a>).</p></section><section id="tpg270022-sec-0100"><h3 class="pmc_sec_title">2.7. Alternative splicing analysis</h3>
<p>All of the AS analyses were done using the vast‐tools v2.2.0 toolset. By mapping reads to exon–exon junctions and exon–intron junctions, vast‐tools is able to generate the percentage spliced‐in value (PSI) (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S3</a>). Only the splicing events with sufficient read coverage (above or equal to “LOW” coverage described by Martín et al. <a href="#tpg270022-bib-0050" class="usa-link" aria-describedby="tpg270022-bib-0050">2021</a>) in at least 10 samples across cell types were considered.</p>
<p>Furthermore, only the samples that passed the binomial test for IR were kept for further analysis. This test assesses proper balance of read at the exon–intron junction and is used to filter out IR events with read imbalance as described in Braunschweig et al. (<a href="#tpg270022-bib-0006" class="usa-link" aria-describedby="tpg270022-bib-0006">2014</a>). The script used to identify cell type‐specific ASE was developed by Martín et al. (<a href="#tpg270022-bib-0050" class="usa-link" aria-describedby="tpg270022-bib-0050">2021</a>) and is available at <a href="https://github.com/vastdb-pastdb/pastdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/vastdb‐pastdb/pastdb</a>. At this step, root and aerial cell types were analyzed separately (Table <a href="#tpg270022-tbl-0001" class="usa-link">1</a>). For the different cell types to be included in the analysis, each group needed a minimum of two replicates with sufficient coverage, and the minimum number of cell types with sufficient coverage from each organ was four. To be considered cell type‐specific, (i) a splicing event required an average |∆PSI| of at least 25 when compared to the average of all other samples from either root or aerial cell types (|global ∆PSI| &gt; 25). Moreover, (ii) the ASE needed a difference in average PSI of at least 15 when compared to any other cell type (|minimum ∆PSI| &gt; 15). Finally, the minimum PSI range (i.e., the difference between the maximum value from a cell type with the maximum value from another) for the event to be considered AS among groups was 2.</p>
<p>The immunity‐related gene list was compiled using the “defense response” annotation and children terms as a filter through the gene ontology (GO) annotation of the <em>Arabidopsis</em> list from TAIR (predicted and experimental) (Berardini et al., <a href="#tpg270022-bib-0003" class="usa-link" aria-describedby="tpg270022-bib-0003">2004</a>). This screening led to 4869 annotations and 2822 genes (Table <a href="#tpg270022-supinfo-0001" class="usa-link">S2</a>).</p></section><section id="tpg270022-sec-0110"><h3 class="pmc_sec_title">2.8. Differential gene expression assessment</h3>
<p>The differential gene expression step was executed with a vast‐tools script built by Martín et al. (<a href="#tpg270022-bib-0050" class="usa-link" aria-describedby="tpg270022-bib-0050">2021</a>) (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S3</a>). The script is accessible at the following link: <a href="https://github.com/vastdb-pastdb/pastdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/vastdb‐pastdb/pastdb</a>. For this analysis, aerial and root cell types were separated and only genes with a median expression cRPKM ≥ 5 in at least one cell type were considered. A gene with an absolute fold change ≥2 when compared to the median of each cell type, a fold change ≥5 when compared to the global median of all the other conditions, and an absolute difference in cRPKM ≥ 2 with every other cell type would be considered cell type specific. The normalized counts of the genes regulated by the cell type‐specific promoters (Table <a href="#tpg270022-tbl-0001" class="usa-link">1</a>) were calculated with DESeq2 (Love et al., <a href="#tpg270022-bib-0046" class="usa-link" aria-describedby="tpg270022-bib-0046">2014</a>) (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S4C</a>).</p></section><section id="tpg270022-sec-0120"><h3 class="pmc_sec_title">2.9. GO term analysis</h3>
<p>The functional annotation analysis was done with the DAVID tools v2023q2 (Huang et al., <a href="#tpg270022-bib-0029" class="usa-link" aria-describedby="tpg270022-bib-0029">2009</a>; Sherman et al., <a href="#tpg270022-bib-0061" class="usa-link" aria-describedby="tpg270022-bib-0061">2022</a>) using a custom background set of genes. This set is constituted of genes that pass the same quality control criteria used to define the cell type‐specific AS or genes.</p></section></section><section id="tpg270022-sec-0130"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270022-sec-0140"><h3 class="pmc_sec_title">3.1. Cell type identity is conserved upon cell sorting</h3>
<p>The transgenic plants of Arabidopsis Col‐0 used here carried the gene encoding for the fluorescent protein mTurquoise2 (Goedhart et al., <a href="#tpg270022-bib-0017" class="usa-link" aria-describedby="tpg270022-bib-0017">2012</a>), under the control of the inducible promoter p<em>OP</em>. This promoter relies upon the binding of the chimeric transcription factor LhG4. The interaction between LhG4 and the glucocorticoid DEX will cause the relocation of LhG4 to the nucleus, where it is able to bind pOP and activate it (Craft et al., <a href="#tpg270022-bib-0009" class="usa-link" aria-describedby="tpg270022-bib-0009">2005</a>). The cell type‐specificity aspect of this system comes from the expression of LhG4, which is controlled by a cell type‐specific promoter (Figure <a href="#tpg270022-fig-0001" class="usa-link">1A</a>) (Schürholz et al., <a href="#tpg270022-bib-0059" class="usa-link" aria-describedby="tpg270022-bib-0059">2018</a>). This multilayered system has enabled us to work with 11 different Arabidopsis transgenic lines, each harboring a different cell type‐specific promoter. To collect only the targeted cell types, the transgenic plants were grown on DEX‐containing medium (Figure <a href="#tpg270022-fig-0001" class="usa-link">1B</a>) and then used for protoplast extraction followed by sorting via FACS (Figure <a href="#tpg270022-fig-0001" class="usa-link">1B</a>). To prevent protoplast dedifferentiation, we acted in a timely fashion while still making sure that enough material was collected to proceed to FACS. The cell types subjected to Illumina sequencing are spread across the roots and aerial parts of the plants (Figure <a href="#tpg270022-fig-0001" class="usa-link">1A</a>, Table <a href="#tpg270022-tbl-0001" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="tpg270022-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/141e/11953613/831f8850b3f5/TPG2-18-e70022-g006.jpg" loading="lazy" id="jats-graphic-1" height="459" width="798" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270022-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cell type identity is conserved upon cell sorting. (A) Schematic representation of the cell types selected for this study. Green zone represents cell types found in aerial plant parts. Brown zone represents cell types in roots. Root cells are depicted either in the full root or in cross section. (B) Experimental workflow used to collect 11 different cell types of <em>Arabidopsis thaliana</em> for AS analysis. Plants are grown on a dexamethasone (DEX)‐containing medium and used for protoplast extraction. Protoplasts are then sorted through fluorescence‐activated cell sorting (FACS). To select only targeted cell types expressing mTurquoise2, protoplasts from plants grown on medium without DEX and therefore not producing mTurquoise2, are used as controls. The gating strategy to sort mTurquoise2‐positive cells shown here is of p<em>SMLX5</em>. Following FACS, sorted protoplast are proceeded to RNA extraction, library preparation and sequencing. The principal component analysis (PCA) done on (C) cRPKM of genes and (D) percentage spliced‐in values (PSIs) of splicing events that passed the quality filters described in Section <a href="#tpg270022-sec-0020" class="usa-link">2</a>.</p></figcaption></figure><p>To ensure that the experimental design of this study is reliable, we first confirmed that extracting protoplasts and sorting them would have minimal effect on the integrity and identity of the cells. The confocal images of protoplasts taken before and after cell sorting confirmed that protoplast extraction and cell sorting did not affect mTurquoise2 emission (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S2A</a>). They also confirmed that the protoplast population collected through FACS was indeed enriched with mTurquoise2‐emitting cells (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S2A,B</a>). Both the confocal images and the FACS data assured that protoplasts do conserve their integrity and therefore can be used for RNA sequencing. With the RNA‐seq data gathered, we were able to calculate the PSI of each ASE and the transcripts cRPKM (corrected for mappability) using vast‐tools. After applying the quality filters described in Section <a href="#tpg270022-sec-0020" class="usa-link">2</a>, we performed a principal component analysis (PCA) with the transcripts’ cRPKM values (Figure <a href="#tpg270022-fig-0001" class="usa-link">1C</a>) and the ASEs PSI (Figure <a href="#tpg270022-fig-0001" class="usa-link">1D</a>). This analysis shows that the different cell types cluster together according to the organ from which they originate. The clustering of the replicates is also visible on the Spearman cluster heatmap (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S4A,B</a>). In addition to the heatmap and PCA, we also looked at the normalized read counts of the genes natively controlled by the cell type‐specific promoters used in this study (Table <a href="#tpg270022-tbl-0001" class="usa-link">1</a>, Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S4C</a>). Most of the genes observed exhibit the highest counts in their respective condition. Although <em>REV</em>, <em>LTP1</em>, <em>PXY</em>, <em>CASP1</em>, and <em>APL</em> did not demonstrate the highest level of counts in the cell type they belong to, they do harbor the second to fourth highest count numbers. This can be explained by the fact that some of those genes are expressed in similar cell types. <em>LPT1</em>, for instance, has the highest count number in the cell type represented by p<em>ML1</em>, which is expected as both <em>ML1</em> and <em>LTP1</em> are expressed in the epidermis (Han et al., <a href="#tpg270022-bib-0021" class="usa-link" aria-describedby="tpg270022-bib-0021">2020</a>; Sessions et al., <a href="#tpg270022-bib-0060" class="usa-link" aria-describedby="tpg270022-bib-0060">1999</a>; Thoma et al., <a href="#tpg270022-bib-0066" class="usa-link" aria-describedby="tpg270022-bib-0066">1994</a>). It is also the case for <em>CASP1</em>, which has the highest count number in p<em>SCR</em>. While being expressed in the quiescent center, <em>SCR</em> is also expressed in the endodermis like <em>CASP1</em> (Malamy &amp; Benfey, <a href="#tpg270022-bib-0048" class="usa-link" aria-describedby="tpg270022-bib-0048">1997</a>; Roppolo et al., <a href="#tpg270022-bib-0057" class="usa-link" aria-describedby="tpg270022-bib-0057">2011</a>). <em>SMLX5</em> and <em>APL</em> are both specific to phloem cells, which is reflected by <em>SMLX5</em> having the highest number of counts in p<em>APL</em> (Bonke et al., <a href="#tpg270022-bib-0005" class="usa-link" aria-describedby="tpg270022-bib-0005">2003</a>; Wallner et al., <a href="#tpg270022-bib-0068" class="usa-link" aria-describedby="tpg270022-bib-0068">2017</a>). These results demonstrate that the procedure used herein does not significantly alter cell identity and can then be used to enrich cell populations for further characterization.</p></section><section id="tpg270022-sec-0150"><h3 class="pmc_sec_title">3.2. Gene expression is uncoupled from alternative splicing</h3>
<p>To assess differences in gene expression in a cell type‐specific manner from aerial or root organs, the script performed pairwise comparisons between the target cell type and a pool of all the other cell types. The goal here was to uncover genes whose expression did not overlap between cell types. Similarly, differences in gene splicing between cell types were measured by comparing PSIs in a targeted tissue against the average of all the other tissues, as described by Martín et al. (<a href="#tpg270022-bib-0050" class="usa-link" aria-describedby="tpg270022-bib-0050">2021</a>). These two analyses yielded 2984 and 718 differentially expressed cell type‐specific genes (DEGs) and differentially spliced cell type‐specific genes (DSGs) (Table <a href="#tpg270022-supinfo-0001" class="usa-link">S3</a>) (Figure <a href="#tpg270022-fig-0002" class="usa-link">2A</a>). The two sets only share 79 genes, which are both DEG and DSG. Overall, the sets of genes that are regulated at the transcriptional level and the AS level show only minimal overlap, suggesting the two processes act independently in managing cell identity. For a gene to be considered differentially spliced in a cell type‐specific manner, it needed at least one differentially spliced cell type‐specific event (DSE). From the 718 DSGs, 997 DSEs were identified, implying that some genes have more than one cell type‐specific DSE. The DEGs (Figure <a href="#tpg270022-fig-0002" class="usa-link">2B</a>) and DSEs (Figure <a href="#tpg270022-fig-0002" class="usa-link">2C</a>) exhibit distinct profiles. Interestingly, the splicing profiles also cluster according to the organ of origin, contrasting with the expression profiles, for which no evident pattern can be discerned (Figure <a href="#tpg270022-fig-0002" class="usa-link">2B,C</a>). Even though the two groups are strikingly different, a unique pattern can be observed for p<em>ATHB8</em> (Figure <a href="#tpg270022-fig-0002" class="usa-link">2B,C</a>). We also looked at the expression of various housekeeping genes in p<em>ATHB8</em> and ensured their expression remains unchanged compared to all the other cell types (Table <a href="#tpg270022-supinfo-0001" class="usa-link">S4</a>) (Czechowski et al., <a href="#tpg270022-bib-0010" class="usa-link" aria-describedby="tpg270022-bib-0010">2005</a>).</p>
<figure class="fig xbox font-sm" id="tpg270022-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/141e/11953613/0061f7322f73/TPG2-18-e70022-g005.jpg" loading="lazy" id="jats-graphic-3" height="393" width="705" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270022-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gene expression is uncoupled from alternative splicing. (A) Venn diagram of differentially expressed cell type‐specific genes (DEGs) and differentially spliced cell type‐specific genes (DSGs). Genes with at least one alternatively spliced event (ASE) were considered a DSG. <em>z</em>‐Score cluster heatmap of (B) normalized counts from DEG and (C) PSIs from differentially spliced events (DSEs). Clustering was done using the ward method. The conversion to <em>z</em>‐score was performed after the hierarchical clustering so that it only affects the color grading. Enriched gene ontology (GO) terms in DEGs (D) and DSGs (E). Red and blue indicates up‐ and downregulated genes. Bars indicate −log10 of the DAVID's EASE score modified <em>p</em>‐value.</p></figcaption></figure><p>The functional annotation analysis of each set of the DEG and DSG shows very little similarity (Figure <a href="#tpg270022-fig-0002" class="usa-link">2D,E</a>). Among the processes upregulated at the gene expression level, we notice tissue development. On the other hand, the downregulated processes are generally more focused on defense responses and stress (Figure <a href="#tpg270022-fig-0002" class="usa-link">2D</a>). The DSGs are enriched for processes like protein modifications and RNA/DNA metabolic process (Figure <a href="#tpg270022-fig-0002" class="usa-link">2E</a>). This is compatible with the fact that many splicing factors auto‐regulate themselves through AS (Lazar &amp; Goodman, <a href="#tpg270022-bib-0037" class="usa-link" aria-describedby="tpg270022-bib-0037">2000</a>; Lopato et al., <a href="#tpg270022-bib-0044" class="usa-link" aria-describedby="tpg270022-bib-0044">1999</a>, <a href="#tpg270022-bib-0045" class="usa-link" aria-describedby="tpg270022-bib-0045">1996</a>). Interestingly, annotations related to transcription regulation are also represented in the DSGs (Figure <a href="#tpg270022-fig-0002" class="usa-link">2E</a>), meaning that AS is involved in gene expression regulation. These results indicate that the regulation of transcription and of AS in a cell type‐specific manner are independent.</p></section><section id="tpg270022-sec-0160"><h3 class="pmc_sec_title">3.3. Cell identity shapes the need for transcription or alternative splicing regulation</h3>
<p>The importance of AS in cell differentiation has already been studied (Li et al., <a href="#tpg270022-bib-0039" class="usa-link" aria-describedby="tpg270022-bib-0039">2016</a>). To examine how the identity of this specific set of cell types affects the ASE population, we looked at the distribution of DEGs and DSEs across cell types. p<em>ATHB8</em>, a promoter mostly expressed in the procambium, xylem precursor, and columella in the root apical meristem, displays a notable distribution profile with the largest number of DEGs (985) (Figure <a href="#tpg270022-fig-0003" class="usa-link">3A</a>) and DSEs (452) (Figure <a href="#tpg270022-fig-0003" class="usa-link">3B</a>), regardless of whether they are up or downregulated. The cell types represented by p<em>ATHB8</em> make up 34% and 46% of the total DEGs and DSEs, respectively. Of the 985 DEGs in p<em>ATHB8</em>, 856 are unique to that promoter. Of the 129 genes left, 100 are shared between p<em>ML1</em>, p<em>REV</em>, p<em>UFO</em>, and p<em>LTP1</em>, which are from aerial organs. The other 29 genes are spread across the rest of the cell types (Figure <a href="#tpg270022-fig-0003" class="usa-link">3A</a>). At the other end of the spectrum, cells sorted with p<em>PXY</em> exhibited the smallest number of DEGs with only 46 (1.6%) deregulated genes (Figure <a href="#tpg270022-fig-0003" class="usa-link">3A</a>). At the splicing level, p<em>ATHB8</em> presents 452 DSEs, among which 413 are unique to that promoter, whereas p<em>SMLX5</em> has only 13 (1.3%), making those two the most and least prone to AS regulation, respectively (Figure <a href="#tpg270022-fig-0003" class="usa-link">3B</a>). Our analysis aimed at identifying nonoverlapping DSEs. Hence, p<em>APL</em> and p<em>SMLX5</em> both being expressed in the phloem led to the identification of fewer nonoverlapping DSEs between promoters. Interestingly, some cell types display twice as many DEGs as DSEs. This is the case for p<em>SMLX5</em>, p<em>CASP1</em>, p<em>SCR</em>, and p<em>ML1</em> (Figure <a href="#tpg270022-fig-0003" class="usa-link">3A,B</a>). This phenomenon is not observed for DSEs over DEGs, which can in part be explained by the fact that the overall number of observed DEGs is greater than the number of DSGs (2984 vs. 714). Nonetheless, p<em>LTP1</em> and p<em>REV</em> both have over 1.5‐fold more DEGs than DSEs (Figure <a href="#tpg270022-fig-0003" class="usa-link">3A,B</a>). This further implies that the importance of AS differs depending on the cell identity and the function they hold. The cell types represented by p<em>ATHB8</em>, for example, appear to host processes that require more intense transcriptional and posttranscriptional regulation. To have a better understanding of the impact of cell type‐specific AS, we looked at the proportion of the different types of events. As has been globally reported, the most prevalent type of ASE is intron retention, representing 69.1% of all the DSEs identified (Figure <a href="#tpg270022-fig-0003" class="usa-link">3C</a>). Intron retention is followed by alternative acceptor site, alternative donor site, and exon skipping (16.5%, 8.2%, and 6.2%, respectively). This corroborates previous reports of event type proportions found in plant cell type‐specific AS datasets (Li et al., <a href="#tpg270022-bib-0039" class="usa-link" aria-describedby="tpg270022-bib-0039">2016</a>; Martín et al., <a href="#tpg270022-bib-0050" class="usa-link" aria-describedby="tpg270022-bib-0050">2021</a>).</p>
<figure class="fig xbox font-sm" id="tpg270022-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/141e/11953613/8dcbe555eadc/TPG2-18-e70022-g003.jpg" loading="lazy" id="jats-graphic-5" height="363" width="722" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270022-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cell identity shapes the need for transcription or alternative splicing regulation. Upsetplots of (A) differentially expressed cell type‐specific genes (DEGs) and (B) differentially spliced cell type‐specific events (DSEs). Each color represents a cell type. Vertical bars show the number of overlapping genes or ASEs across cell types. Colored dots indicate intersections from which DEGs and DSEs overlap. The number of genes or ASEs in each intersection is indicated over the vertical bars, percentage is displayed when above 10%. Horizontal bars represent the total number of genes or ASEs that are differentially expressed or spliced according to different cell types. Green and brown bars represent aerial and root cell types, respectively. Proportions of the types of ASE from the DSEs, the upregulated and downregulated DSE (C). Numbers in bars indicate the percentage of ASEs that are either alternative acceptor site (ALTA) (dark violet), alternative donor site (ALTD) (teal), exon skipping (EX) (dark green), or intron retention (IR) (light green).</p></figcaption></figure><p>Even though the procedure to identify cell type‐specific genes and ASEs uncovers only nonoverlapping DEGs and DSEs, some are still shared between cell types. The analysis was done by separating cell types originating from different organs, which explains the overlap between cell types from aerial parts and from roots. The distribution of DEGs and DSEs across cell types highlights their varying needs in terms of splicing and transcription regulation.</p></section><section id="tpg270022-sec-0170"><h3 class="pmc_sec_title">3.4. Direction of gene expression and AS regulation is dependent on cell identity and function</h3>
<p>When DEGs are separated according to their upregulation (Figure <a href="#tpg270022-fig-0004" class="usa-link">4A</a>) or downregulation (Figure <a href="#tpg270022-fig-0004" class="usa-link">4B</a>), we can observe that the majority of p<em>ATHB8</em> DEGs are downregulated with 62 up‐DEGs (∼4% of the total up‐DEGs) (Figure <a href="#tpg270022-fig-0004" class="usa-link">4A</a>) and 842 down‐DEGs (50.6% of the total down‐DEGs) (Figure <a href="#tpg270022-fig-0004" class="usa-link">4B</a>). Although not at the same scale, p<em>APL</em> also shows more down‐DEGs (56) (Figure <a href="#tpg270022-fig-0004" class="usa-link">4B</a>) than up‐DEGs (7) (Figure <a href="#tpg270022-fig-0004" class="usa-link">4A</a>). While the overall trend of the DEGs tends toward downregulation, this is mostly due to p<em>ATHB8</em>, as other cell types like p<em>SCR</em>, p<em>TMO5</em>, p<em>PXY</em>, p<em>LTP1</em>, p<em>SMLX5</em>, p<em>REV</em>, and p<em>CASP1</em> have mostly up‐DEGs. Among those mainly upregulated DEGs cell types, p<em>REV</em>, p<em>CASP1</em>, and p<em>SCR</em> are the most striking as they have 271 (18%), 241 (16%), and 272 (18%) of up‐DEGs (Figure <a href="#tpg270022-fig-0004" class="usa-link">4A</a>) and 49 (∼3%), 22 (∼1.2%), and 59 (∼3.4%) down‐DEGs (Figure <a href="#tpg270022-fig-0004" class="usa-link">4B</a>), respectively. Looking at the upregulated (Figure <a href="#tpg270022-fig-0004" class="usa-link">4C</a>) and downregulated (Figure <a href="#tpg270022-fig-0004" class="usa-link">4D</a>) DSEs, a clear tendency toward upregulation can be seen. The only exceptions are p<em>ATHB8</em> and p<em>ML1</em>, which respectively have 149 (24.7%) and 17 (2.8%) up‐DSEs (Figure <a href="#tpg270022-fig-0004" class="usa-link">4C</a>) and 291 (65.7%) and 40 (9%) down‐DSEs (Figure <a href="#tpg270022-fig-0004" class="usa-link">4D</a>). Those results indicate that the need for up‐ or downregulation of DEGs varies according to the identity and the function of each cell type.</p>
<figure class="fig xbox font-sm" id="tpg270022-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/141e/11953613/028476c45aa3/TPG2-18-e70022-g002.jpg" loading="lazy" id="jats-graphic-7" height="322" width="534" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270022-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Direction of gene expression and AS regulation is dependent on cell identity and function. Distribution of (A) upregulated (log 2 fold change [FC] ≥ 2) and (B) downregulated (log 2 FC ≤ −2) differentially expressed DEGs, and of (C) upregulated (∆PSI &gt; 25) and (D) downregulated DSEs (∆PSI &lt; −25). As previously mentioned, cell types are colored as indicated; vertical bars represent the amount of genes or ASEs overlapping in the corresponding intersection. Numbers over the vertical bars indicate the quantity of genes or ASEs, percentage is displayed when above 10%. Green and brown bars represent aerial and root cell types, respectively.</p></figcaption></figure></section><section id="tpg270022-sec-0180"><h3 class="pmc_sec_title">3.5. Differential splicing of immunity‐related genes is cell type specific</h3>
<p>The functional annotation analysis (Figure <a href="#tpg270022-fig-0002" class="usa-link">2D</a>) highlighted a strong tendency toward various responses to biotic and abiotic stresses among the DEGs. We sought to investigate the impact of cell identity on AS regulation of immunity‐related genes. We first compiled a list of plant immunity‐related genes (Table <a href="#tpg270022-supinfo-0001" class="usa-link">S2</a>). From this list, we then identified 530 DEGs and 92 DSEs associated with genes involved in plant immunity, which we named immunity‐DEGs and immunity‐DSEs (Table <a href="#tpg270022-supinfo-0001" class="usa-link">S5</a>). The 92 immunity‐DSEs originate from 71 immunity‐related genes (immunity‐DSGs). Unsurprisingly, p<em>ATHB8</em> shows once again the highest amount of DEGs and DSEs, with 193 immunity‐DEGs (Figure <a href="#tpg270022-fig-0005" class="usa-link">5A</a>) and 39 immunity‐DSEs (Figure <a href="#tpg270022-fig-0005" class="usa-link">5B</a>), while p<em>APL</em> shows only three immunity‐DEGs (Figure <a href="#tpg270022-fig-0005" class="usa-link">5A</a>) and no immunity‐DSE (Figure <a href="#tpg270022-fig-0005" class="usa-link">5B</a>). Besides p<em>ATHB8</em>, most root‐associated promoters present very few immunity‐DSEs. It is the case for p<em>TMO5</em>, p<em>PXY</em>, p<em>CASP1</em>, and p<em>SMLX5</em>, which have only one or two immunity‐DSEs (Figure <a href="#tpg270022-fig-0005" class="usa-link">5B</a>).</p>
<figure class="fig xbox font-sm" id="tpg270022-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/141e/11953613/ef6da6b76171/TPG2-18-e70022-g004.jpg" loading="lazy" id="jats-graphic-9" height="450" width="643" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270022-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differential splicing of immunity‐related genes is cell type‐specific. Upsetplots of immunity‐related (A) differentially expressed cell type‐specific genes (Immunity‐DEGs) and (B) differentially spliced cell type‐specific events (Immunity‐DSEs) across cell types. Green and brown bars represent aerial and root cell types, respectively. Violin plot of ∆PSI for the 92 Immunity‐DSEs in each cell type (C). ∆PSI represents an ASE's difference of PSI between a target tissue and the average of all the other tissues together (∆PSI = PSI<sub>target</sub> − PSI<sub>average</sub>). Dotted lines show the thresholds used to determine if an ASE is significantly differentially spliced (25 &lt; ∆PSI &lt; −25). Predicted impact of the immunity‐DSEs on proteins (D). Bars represent the percentage of DSEs, which are predicted to cause either open reading frame (ORF) disruption, generate an alternative isoform, or overlap untranslated regions (UTRs).</p></figcaption></figure><p>When looking at the ΔPSI (∆PSI = PSI<sub>target</sub> − PSI<sub>average</sub>) of the immunity‐DSEs across cell types (Figure <a href="#tpg270022-fig-0005" class="usa-link">5C</a>), most of the immunity‐DSEs in this cell type are either over‐ or underrepresented, contrasting with the 10 other cell types where the ΔPSI median is closer to 0. Additionally, the functional analysis of the root cell types indicates that this promoter is mostly responsible for the immunity‐related annotations (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S5</a>) found in the DEGs.</p>
<p>To assess whether the 92 immunity‐DSEs identified in our analysis have an impact on protein functionality, we used PastDB. This database generated by Martín et al. (<a href="#tpg270022-bib-0050" class="usa-link" aria-describedby="tpg270022-bib-0050">2021</a>) regroups AS profiles of Arabidopsis and associated features, like the predicted protein impact (Irimia et al., <a href="#tpg270022-bib-0032" class="usa-link" aria-describedby="tpg270022-bib-0032">2014</a>). Using this database, we were able to extract the predicted protein impact of the immunity‐DSEs. Up to 56.5% of the immunity‐DSEs are predicted to cause ORF disruption (Figure <a href="#tpg270022-fig-0005" class="usa-link">5D</a>).</p>
<p>These observations align with the fact that, in addition to increased protein diversity, AS also plays a major role in gene translation regulation. In this instance, 60.4% of the immunity‐DSEs causing ORF disruption are upregulated, which can lead to the translation of a nonfunctional protein or to isoforms being processed through NMD, both resulting in a diminished quantity of functional protein. On the other hand, 39.6% of DSEs causing ORF disruption are downregulated, which implies a need for the other isoforms from the same genes to be upregulated. It is also important to note that at least 29.3% of immunity‐DSEs are predicted to generate alternative proteins (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S6</a>).</p>
<p>Among the genes from this group, the gene‐level expression varied significantly for five of them: <em>BCL‐2 associated athanogene 6</em> (<em>BAG6</em>), <em>plant and fungi atypical dual‐specificity phosphatase 3</em> (<em>PFA‐DSP3</em>), <em>mitochondrion‐localized small heat shock protein 23.6</em> (<em>HSP23.6‐MITO</em>), <em>A. thaliana</em>
<em>respiratory burst oxidase homolog F</em> (<em>RBOHF</em>), and AT4G28703 (Figure <a href="#tpg270022-fig-0006" class="usa-link">6</a>). These genes showed repressed gene expression and upregulated ASEs. This is the case for <em>BAG6</em> (LFC = −4.33) and <em>HSP23.6‐MITO</em> (LFC = −5.13), for which one ASE, AthINT0061548 and AthINT0080096 respectively, present a significant upregulation (ΔPSI<sub>AthINT0061548</sub> = 27.8 and ΔPSI<sub>AthINT0080096</sub> = 27.13) in p<em>LTP1</em> (Figure <a href="#tpg270022-fig-0006" class="usa-link">6</a>, Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S7</a>, Table <a href="#tpg270022-supinfo-0001" class="usa-link">S6</a>). <em>RBOHF</em>, <em>PFA‐DSP3</em>, and AT4G28703 also have negative gene expression and positive ΔPSIs, but this time in p<em>ATHB8</em> (Figure <a href="#tpg270022-fig-0006" class="usa-link">6</a>). Interestingly, AthINT0061548 from <em>BAG6</em>, AthINT0080096 from <em>HSP23.6‐MITO</em>, and AthINT0037028 from AT4G28703 are all predicted to result in alternative proteins (Table <a href="#tpg270022-supinfo-0001" class="usa-link">S6</a>). This observation can also be made for <em>PFA‐DSP3</em>, whose gene‐level expression is reduced (LFC = −2.79) but displays two ASEs, AthINT0021830 and AthINT0021831, with a positive ∆PSI (∆PSI<sub>AthINT0021830</sub> = 27.46, ∆PSI<sub>AthINT0021831</sub> = 39.65) in p<em>ATHB8</em> (Figure <a href="#tpg270022-fig-0006" class="usa-link">6</a>). <em>PFA‐DSP3</em>'s AthINT0021831 is also predicted to result in an alternative protein (Table <a href="#tpg270022-supinfo-0001" class="usa-link">S6</a>). Even upon reduction of the transcription level of specific genes, AS can lead to a significant change in isoforms, sometimes even leading to alternative proteins. Whether or not those alternative proteins are functional is yet to be determined. On the other hand, <em>RBOHF</em>'s AthINT0097993 and <em>PFA‐DSP3</em>'s AthINT0021830 are predicted to cause ORF disruptions (Table <a href="#tpg270022-supinfo-0001" class="usa-link">S5</a>), which implies that, just like the inhibition of their gene expression, these ASE's upregulation would result in a decreased level of their corresponding functional protein. We also looked at those events and five other randomly selected Immunity‐DSEs in a long‐read sequencing isoform database (AtRTD3) (Zhang et al., <a href="#tpg270022-bib-0074" class="usa-link" aria-describedby="tpg270022-bib-0074">2022</a>) to assess whether they had already been identified (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S8</a>). Out of those 10 events, seven were found in AtRTD3. This observation suggests that our results are not biased by the method and bioinformatic tools used here.</p>
<figure class="fig xbox font-sm" id="tpg270022-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/141e/11953613/d0aa6c390b00/TPG2-18-e70022-g001.jpg" loading="lazy" id="jats-graphic-11" height="526" width="480" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270022-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gene expression and AS variation of immunity‐related genes. Comparison of gene expression and differential alternative splicing for the 92 immunity‐related DSEs in the 11 cell types (A). Gene expression is represented as fold change in a logarithmic scale (log 2 FC) (see Section <a href="#tpg270022-sec-0020" class="usa-link">2</a> for details), while AS variation is represented as described previously with ΔPSI. Vertical dotted lines indicate the thresholds used to determine if an ASE is significantly differentially spliced (25 &lt; ∆PSI &lt; −25). Horizontal dotted lines indicate the thresholds used to determine if a gene is significantly differentially expressed (2 &lt; log 2 FC &lt; −2). DSEs that are differentially spliced and differentially expressed are identified by their gene name. Genomic representation of immunity DEGs that are also differentially spliced (B). Blue and black tracks represent exons and introns, respectively. Orange tracks show introns retained in the ASEs, orange lines indicate adjoining exons.</p></figcaption></figure><p>Besides the five immunity‐related genes mentioned previously, 58 of the immunity‐related ASEs that are differentially spliced do not vary at the gene‐level expression in at least one cell type. Out of those 58, 12 are predicted to encode for an alternative protein. This implies that, even though transcription regulation is not altered for those genes, a shift in isoform population still occurs (Table <a href="#tpg270022-supinfo-0001" class="usa-link">S7</a>).</p>
<p>Interestingly, our results accentuate the strong influence of AS on immunity‐associated genes, even leading to possible production of alternative proteins. Lastly, we also uncovered hints pointing toward the importance of cell identity on immunity regulation.</p></section></section><section id="tpg270022-sec-0190"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>The process of AS in plants has proven to be an important tool in the adaptation to external and internal cues. While its mechanistic intricacies have been extensively analyzed, its broad impact on cell identity and plant immunity remains to be explored. In this study, we gathered data from 11 plant cell types originating from different organs, all collected through the same experimental process, to obtain high‐resolution AS data. Through our analysis, we highlighted the significance of AS on transcription regulation as well as on AS regulation itself, all in a cell type‐specific manner. Moreover, we showed that AS holds a role in the shaping of cell identity, especially in the columella cells. We also uncovered the impact of AS on immunity‐related genes in the columella cells. This study attempts to bridge the gap between cell identity, AS, and response to biotic stress. In addition, the dataset generated provides the scientific community with reliable tools to study cell identity and AS.</p>
<section id="tpg270022-sec-0200"><h3 class="pmc_sec_title">4.1. Alternative splicing regulation is independent from transcription</h3>
<p>The first observation our findings indicate is that the genes targeted by expression regulation and the ones targeted by AS do not overlap (Figure <a href="#tpg270022-fig-0002" class="usa-link">2A</a>). Although AS occurs co‐transcriptionally, our results suggest that AS regulation is largely independent from gene expression regulation. This aligns with previous studies in which cell identity and stress triggered transcription and AS changes on nonoverlapping genes (Cheng et al., <a href="#tpg270022-bib-0008" class="usa-link" aria-describedby="tpg270022-bib-0008">2022</a>; Li et al., <a href="#tpg270022-bib-0040" class="usa-link" aria-describedby="tpg270022-bib-0040">2013</a>; Martín et al., <a href="#tpg270022-bib-0050" class="usa-link" aria-describedby="tpg270022-bib-0050">2021</a>). This phenomenon does not seem to be generalized to plant development overall, as the Arabidopsis germline shares a considerable number of DEGs and alternatively spliced genes (Misra et al., <a href="#tpg270022-bib-0051" class="usa-link" aria-describedby="tpg270022-bib-0051">2023</a>). The functional analysis of the DEGs highlighted tissue development, among others, as upregulated and immunity‐related processes as downregulated. On the other hand, genes targeted by AS are enriched for GO terms like DNA metabolic process, hence regulating gene expression. In addition, ASEs are also involved in RNA metabolic processes. This is expected as many splicing factors are themselves subjected to splicing, bringing another layer of regulation to AS (Jia et al., <a href="#tpg270022-bib-0033" class="usa-link" aria-describedby="tpg270022-bib-0033">2023</a>; Lazar &amp; Goodman, <a href="#tpg270022-bib-0037" class="usa-link" aria-describedby="tpg270022-bib-0037">2000</a>).</p></section><section id="tpg270022-sec-0210"><h3 class="pmc_sec_title">4.2. Alternative splicing shapes cell identity</h3>
<p>Through the use of specific promoters expressed on neighboring and sometimes even overlapping cell types, we managed to take a more refined look at the control AS holds over cell identity. Our dataset identified a greater number of cell type‐specific ASEs than previously done using the same analysis tools (Martin et al., <a href="#tpg270022-bib-0050" class="usa-link" aria-describedby="tpg270022-bib-0050">2021</a>). This could be due to the process by which our dataset was generated, as all cell types were harvested similarly using the same induction system, whereas other studies have aggregated publicly available data. Although lower than the number of DEGs, we were still able to identify several cell type‐specific differentially spliced ASEs. Martin et al. (<a href="#tpg270022-bib-0050" class="usa-link" aria-describedby="tpg270022-bib-0050">2021</a>) also documented tissue and cell identity as impacting gene expression at a greater scale than AS. Our functional analysis also suggests that tissue development is particularly supported by gene expression (Figure <a href="#tpg270022-fig-0002" class="usa-link">2D</a>). While the influence of gene‐level regulation on cell identity is considerable, the expression profile of ASEs clustering according to cell types still hints toward the effect of AS on cell identity to be impactful (Figure <a href="#tpg270022-fig-0002" class="usa-link">2B</a>).</p>
<p>One of our main goals was to find nonoverlapping regulated ASEs. Even though some root cell types studied here are either very close to each other in terms of identity or represented by multiple promoters, our analysis still yielded a few specific DSEs. We suspect that the difference in AS regulation between those cell types is due to the maturation stage of the root cells. The phloem cells, for example, are represented by both p<em>APL</em> and p<em>SMLX5</em>. The main difference between the two promoters resides in the developmental phase of the phloem cells they represent. The same rationale applies to p<em>SCR</em> and p<em>CASP1</em>, both expressed in the endodermis. Our analysis was able to capture the subtle changes in AS regulation necessary between different developmental stages of corresponding cell types. Hence, it identified genes better suited for future cell type‐specific studies, as very few cell type‐specific promoters available are truly cell type‐specific. Just like the promoters used here, many will have overlapping expression across neighboring cell types. Therefore, the present study provides a list of new genes and promoters to develop new cell type‐specific biological tools (Table <a href="#tpg270022-supinfo-0001" class="usa-link">S3</a>).</p>
<p>The shoot and root apical meristem provided substantial differences in pathways (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S5</a>) and intensity of gene‐level expression and AS regulation. This is to be expected as even though both perform active mitotic cell division, the two meristems are involved in largely different tissue growth (Xue et al., <a href="#tpg270022-bib-0072" class="usa-link" aria-describedby="tpg270022-bib-0072">2020</a>). While most aerial cell types differentially express a somewhat similar number of genes, the shoot apical meristem (SAM) central zone (<em>pREV</em>) is definitely subjected to a more intense ASE regulation (Figure <a href="#tpg270022-fig-0003" class="usa-link">3B</a>). This is corroborated by Tian et al. (<a href="#tpg270022-bib-0067" class="usa-link" aria-describedby="tpg270022-bib-0067">2019</a>), who noted the same effect when comparing the different zones of the SAM. AS is known to be involved in maintaining the SAM zone boundaries and cell division pattern through proteins like SKIP (Li et al., <a href="#tpg270022-bib-0038" class="usa-link" aria-describedby="tpg270022-bib-0038">2022</a>). Nevertheless, the broader implication and importance of AS in the SAM, more particularly in the central zone, have yet to be investigated. Similarly, the quiescent center of the root apical meristem also shows a certain degree of AS regulation. We hypothesize that AS may hold a role in maintaining the stem niches and therefore in the developmental plasticity of meristems.</p></section><section id="tpg270022-sec-0220"><h3 class="pmc_sec_title">4.3. Alternative splicing, cell identity, and immunity‐associated genes are interconnected</h3>
<p>To fill part of the gap between plant immune response, cell identity, and AS, we examined immunity‐related genes among cell type‐specific DEGs and DSGs. The distinction between transcription and AS regulation is also true for immunity‐related genes. At first glance, the majority of the immunity‐related processes seem transcriptionally regulated, as shown by the functional analysis of the DEGs (Figure <a href="#tpg270022-fig-0002" class="usa-link">2D</a>). However, when separating root and aerial regulated ASEs, we noticed that aerial ASEs are enriched for plant hypersensitive response (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S5</a>). The role of AS in immunity was proven multiple times through the study of key factors of plant defense (Golisz et al., <a href="#tpg270022-bib-0019" class="usa-link" aria-describedby="tpg270022-bib-0019">2021</a>; Howard et al., <a href="#tpg270022-bib-0028" class="usa-link" aria-describedby="tpg270022-bib-0028">2013</a>; Liu et al., <a href="#tpg270022-bib-0043" class="usa-link" aria-describedby="tpg270022-bib-0043">2022</a>; Xie et al., <a href="#tpg270022-bib-0071" class="usa-link" aria-describedby="tpg270022-bib-0071">2023</a>), and yet the global extent of its impact on resistance has yet to be discussed. Almost up to 10% of the cell type‐specific regulated ASEs are immunity‐related even in the absence of an actual defense stimulus. This implies that AS could participate in maintaining the growth to immune response balance. According to the predicted protein impact, half of the defense ASEs are expected to cause ORF disruption (Figure <a href="#tpg270022-fig-0005" class="usa-link">5E</a>). The aforementioned observation indicates that AS might act as another layer of immune transcriptional regulation. It possibly negatively regulates the accumulation of immunity‐related proteins in a context in which they are not needed while still maintaining a basal gene expression level to quickly produce the complete protein when needed. Still, the 30% of immunity‐related ASEs resulting in actual alternative protein isoforms also imply that AS potentially has a direct impact on immune signaling pathways in a cell type‐specific manner. As we know, the initiation of a defense response is very consuming to the plant (Smakowska et al., <a href="#tpg270022-bib-0062" class="usa-link" aria-describedby="tpg270022-bib-0062">2016</a>). Hence, the switch of energy allocated to growth versus defense needs intense regulation. Our result suggests that AS plays an important role at different levels to guard the induction of plant defense.</p></section><section id="tpg270022-sec-0230"><h3 class="pmc_sec_title">4.4. Columella cells defense system is regulated through alternative splicing</h3>
<p>One of the main highlights of this analysis comes from p<em>ATHB8</em> cells, which present the highest number of DEGs and DSGs (Figure <a href="#tpg270022-fig-0003" class="usa-link">3A,B</a>). These results could be partially attributed to the way the differentially spliced and differentially expressed analyses are inherently built. These analyses exclusively uncover nonoverlapping genes between conditions, and since many of the promoters used are expressed in overlapping cell types (Figure <a href="#tpg270022-fig-0001" class="usa-link">1</a>), we expected only a small number of DSGs or DEGs. Among the root cell types, the promoters used here, p<em>ATHB8</em> and p<em>SCR</em>, are the only ones specific to the columella and the quiescent center, respectively. Hence, p<em>ATHB8</em> yielded more specific genes and isoforms than the other promoters did. Moreover, we safely assume that most of those regulated genes and isoforms are specific to the columella cells. Following this logic, we would also expect p<em>SCR</em> to exhibit a higher number of specific regulated genes and isoforms. While not as much as p<em>ATHB8</em>, p<em>SCR</em> still presents more specific genes and isoforms than other root promoters do. The quiescent center being less represented than the endodermis in terms of quantity is probably one of the reasons p<em>SCR</em> does not show a higher number of DSEs and DEGs. However, this hypothesis only partially explains why p<em>ATHB8</em> and the columella in particular seem to undergo intense gene and isoform regulation. Additionally, it does not clarify why nearly all of these genes are mostly downregulated.</p>
<p>Columella cells play multiple physiological roles, including but not limited to the perception of gravity (Staehelin et al., <a href="#tpg270022-bib-0063" class="usa-link" aria-describedby="tpg270022-bib-0063">2000</a>), production of mucilage (Maeda et al., <a href="#tpg270022-bib-0047" class="usa-link" aria-describedby="tpg270022-bib-0047">2019</a>), acting as a physical barrier, and involvement in plant defense (Driouich et al., <a href="#tpg270022-bib-0013" class="usa-link" aria-describedby="tpg270022-bib-0013">2013</a>; Hawes et al., <a href="#tpg270022-bib-0024" class="usa-link" aria-describedby="tpg270022-bib-0024">2000</a>). Those cells also undergo tight coordination of stem cell production and cell release at the outer root cap layer. This process is crucial for proper root development. The columella is thus composed of three underlying cell types: the columella stem cells, the columella differentiated cells, and the border‐like cells, which ultimately go through programmed cell death (Kumar &amp; Iyer‐Pascuzzi, <a href="#tpg270022-bib-0035" class="usa-link" aria-describedby="tpg270022-bib-0035">2020</a>). Being responsible for such major functions could justify the need for a more intense regulation of gene expression and isoform splicing. We hypothesize that the imminent cell death could lead to major repression of gene expression. Additionally, columella cells and the root cap in general also act as barriers protecting the apical meristem (Driouich et al., <a href="#tpg270022-bib-0013" class="usa-link" aria-describedby="tpg270022-bib-0013">2013</a>; Fortier et al., <a href="#tpg270022-bib-0016" class="usa-link" aria-describedby="tpg270022-bib-0016">2023</a>; Hawes et al., <a href="#tpg270022-bib-0024" class="usa-link" aria-describedby="tpg270022-bib-0024">2000</a>; Hawes et al., <a href="#tpg270022-bib-0023" class="usa-link" aria-describedby="tpg270022-bib-0023">2016</a>). The mucilage they produce is, among other things, composed of antimicrobial proteins and reactive oxygen species (Driouich et al., <a href="#tpg270022-bib-0014" class="usa-link" aria-describedby="tpg270022-bib-0014">2021</a>; Maeda et al., <a href="#tpg270022-bib-0047" class="usa-link" aria-describedby="tpg270022-bib-0047">2019</a>). The border‐like cells in particular are responsible for the formation of the root extracellular trap, a biofilm‐like structure acting as the first physical line of defense of root meristems (Driouich et al., <a href="#tpg270022-bib-0014" class="usa-link" aria-describedby="tpg270022-bib-0014">2021</a>; Hawes et al., <a href="#tpg270022-bib-0024" class="usa-link" aria-describedby="tpg270022-bib-0024">2000</a>). According to our findings, the columella cells immune mechanism is subjected to intense regulation through gene expression, but also AS. The functional analysis of the DEG in p<em>ATHB8</em> (Figure <a href="#tpg270022-supinfo-0001" class="usa-link">S5</a>) also indicates that the columella is responsible for the immunity‐related annotations. We postulate that the crucial role of the border‐like cells in plant defense is strongly controlled by AS and is responsible for the distribution of the immunity‐related isoforms observed in p<em>ATHB8</em>. While more extensive work is necessary to explore further on the subject, our result certainly highlights the unsuspected importance of AS in the columella cells.</p>
<p>AS is a conserved process that enables eukaryotes to reach a higher level of transcriptome and proteome complexity. This mechanism is essential in the adaptation to various stresses and developmental stages of plants. To better understand the significance of AS on those numerous stimuli, many focused on the study of specific factors of plant development and stress tolerance. We, however, sought to observe AS's broader impact on plant growth and plant immunity. Here, we managed to establish links between AS, cell identity, and immunity‐related genes. Our results also highlighted the unsuspected importance of AS in the columella cells. AS might play an important part in the balance between the growth and defense states. Although the global Arabidopsis AS landscape upon elicitation of a defense response still needs to be investigated, our analysis gives an interesting insight into the interconnection between AS, immunity‐related genes, and cell identity.</p></section></section><section id="tpg270022-sec-0240"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Teura Barff</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; writing—original draft; writing—review and editing. <strong>Ingrid Berenice Sanchez Carrillo</strong>: Investigation; methodology. <strong>Valeria Paola Parra Gutiérrez</strong>: Methodology. <strong>Mélodie B. Plourde</strong>: Conceptualization; methodology; project administration; resources; writing—review and editing. <strong>David L. Joly</strong>: Conceptualization; supervision; writing—review and editing. <strong>Hugo Germain</strong>: Conceptualization; formal analysis; funding acquisition; investigation; methodology; project administration; resources; writing—review and editing.</p></section><section id="tpg270022-sec-0260"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec26"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270022-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270022-supitem-0001"><div class="caption p"><p>Table S1: Samples and replicates sequencing information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s005.xlsx</a><sup> (16.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270022-supitem-0002"><div class="caption p"><p>Table S2. List of defense‐related genes used to identify the immunity‐DEGs and DSEs</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s003.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s003.xlsx</a><sup> (139.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270022-supitem-0003"><div class="caption p"><p>Table S3. Cell type‐specific genes and ASE</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s008.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s008.xlsx</a><sup> (399.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270022-supitem-0004"><div class="caption p"><p>Table S4. Expression of housekeeping genes in p<em>ATHB8</em>
</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s002.xlsx</a><sup> (9.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270022-supitem-0005"><div class="caption p"><p>Table S5. Cell type‐specific immunity related genes and ASE</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s007.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s007.xlsx</a><sup> (67.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270022-supitem-0006"><div class="caption p"><p>Table S6. Immunity‐related splicing variant of interest</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s006.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s006.xlsx</a><sup> (11.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270022-supitem-0007"><div class="caption p"><p>Table S7. Immunity‐related DSE that do not vary at the gene expression level</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s001.xlsx</a><sup> (13.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270022-supitem-0008"><div class="caption p"><p>Supplementary Figures: Fig. S1 to S8</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s004.pdf</a><sup> (1.3MB, pdf) </sup>
</div></div></section></section><section id="tpg270022-sec-0250" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We thank Pr. Sebastian Wolf (Universität Heidelberg) for providing transgenic lines and clones used for the pOP6/LhG4‐mTurquoise2 expression system. We thank teams at the <em>Centre d'expertise et de services Génome Québec</em> for generating libraries and RNA sequencing. We would also like to thank Dr. Guiomar Martin (Universitat de Barcelona), Dr. Manuel Irimia (Universitat Pompeu Fabra), and teams for designing vast‐tools, creating PastDB, providing the scripts used for splicing analysis, and for their helpful insight. Finally, we thank Dr. Natacha Mérindol for her help optimizing the cell sorting process on protoplasts. The part of the schematic of the cell types (Figure <a href="#tpg270022-fig-0001" class="usa-link">1A</a>) and the schematic of the experimental design (Figure <a href="#tpg270022-fig-0001" class="usa-link">1B</a>) were created with BioRender.com. This work was supported by the NSERC (grant number RGPIN/4002‐2020), and Teura Barff was supported by the Fonds de Recherche du Québec en Nature et Technologies (FRQNT)‐DE PhD grant.</p></section><section id="notes1"><p>


Barff, T.
, 
Sanchez Carrillo, I. B.
, 
Parra Gutiérrez, V. P.
, 
B. Plourde, M.
, 
Joly, D. L.
, &amp; 
Germain, H.
 (2025). Comprehensive mapping of Arabidopsis alternative splicing landscape reveals key insights into plant development and immunity. The Plant Genome, 18, e70022. 10.1002/tpg2.70022

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270022-note-0001"><p>Assigned to Associate Editor Mukesh Jain.</p></div></div></section></section><section id="tpg270022-sec-0280"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>Raw sequencing data were submitted to Genbank under Bioproject accession number PRJNA1097566.</p></section><section id="tpg270022-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270022-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270022-bib-0001">
<cite>

Balmer, D.
, &amp; 
Mauch‐Mani, B.
 (2013). More beneath the surface? Root versus shoot antifungal plant defenses. Frontiers in Plant Science, 4, Article 256. 10.3389/fpls.2013.00256

</cite> [<a href="https://doi.org/10.3389/fpls.2013.00256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3709096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23874350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=More%20beneath%20the%20surface?%20Root%20versus%20shoot%20antifungal%20plant%20defenses&amp;author=D.%20Balmer&amp;author=B.%20Mauch%E2%80%90Mani&amp;volume=4&amp;publication_year=2013&amp;pages=256&amp;pmid=23874350&amp;doi=10.3389/fpls.2013.00256&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0002">
<cite>

Bargmann, B. O. R.
, &amp; 
Birnbaum, K. D.
 (2010). Fluorescence activated cell sorting of plant protoplasts. Journal of Visualized Experiments, 1, 2–5. 10.3791/1673
</cite> [<a href="https://doi.org/10.3791/1673" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2896036/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20168296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Visualized%20Experiments&amp;title=Fluorescence%20activated%20cell%20sorting%20of%20plant%20protoplasts&amp;author=B.%20O.%20R.%20Bargmann&amp;author=K.%20D.%20Birnbaum&amp;volume=1&amp;publication_year=2010&amp;pages=2-5&amp;pmid=20168296&amp;doi=10.3791/1673&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0003">
<cite>

Berardini, T. Z.
, 
Mundodi, S.
, 
Reiser, L.
, 
Huala, E.
, 
Garcia‐Hernandez, M.
, 
Zhang, P.
, 
Mueller, L. A.
, 
Yoon, J.
, 
Doyle, A.
, 
Lander, G.
, 
Moseyko, N.
, 
Yoo, D.
, 
Xu, I.
, 
Zoeckler, B.
, 
Montoya, M.
, 
Miller, N.
, 
Weems, D.
, &amp; 
Rhee, S. Y.
 (2004). Functional annotation of the Arabidopsis genome using controlled vocabularies. Plant Physiology, 135(2), 745–755. 10.1104/pp.104.040071

</cite> [<a href="https://doi.org/10.1104/pp.104.040071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC514112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15173566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Functional%20annotation%20of%20the%20Arabidopsis%20genome%20using%20controlled%20vocabularies&amp;author=T.%20Z.%20Berardini&amp;author=S.%20Mundodi&amp;author=L.%20Reiser&amp;author=E.%20Huala&amp;author=M.%20Garcia%E2%80%90Hernandez&amp;volume=135&amp;issue=2&amp;publication_year=2004&amp;pages=745-755&amp;pmid=15173566&amp;doi=10.1104/pp.104.040071&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0004">
<cite>

Bolger, A. M.
, 
Lohse, M.
, &amp; 
Usadel, B.
 (2014). Trimmomatic: A flexible trimmer for Illumina sequence data. Bioinformatics, 30, 2114–2120. 10.1093/bioinformatics/btu170

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4103590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Trimmomatic:%20A%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;volume=30&amp;publication_year=2014&amp;pages=2114-2120&amp;pmid=24695404&amp;doi=10.1093/bioinformatics/btu170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0005">
<cite>

Bonke, M.
, 
Thitamadee, S.
, 
Mähönen, A. P.
, 
Hauser, M.‐T.
, &amp; 
Helariutta, Y.
 (2003). APL regulates vascular tissue identity in Arabidopsis. Nature, 426(6963), 181‐‑186. 10.1038/nature02100

</cite> [<a href="https://doi.org/10.1038/nature02100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14614507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=APL%20regulates%20vascular%20tissue%20identity%20in%20Arabidopsis&amp;author=M.%20Bonke&amp;author=S.%20Thitamadee&amp;author=A.%20P.%20M%C3%A4h%C3%B6nen&amp;author=M.%E2%80%90T.%20Hauser&amp;author=Y.%20Helariutta&amp;volume=426&amp;issue=6963&amp;publication_year=2003&amp;pages=181%E2%80%90%E2%80%91186&amp;pmid=14614507&amp;doi=10.1038/nature02100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0006">
<cite>

Braunschweig, U.
, 
Barbosa‐Morais, N. L.
, 
Pan, Q.
, 
Nachman, E. N.
, 
Alipanahi, B.
, 
Gonatopoulos‐Pournatzis, T.
, 
Frey, B.
, 
Irimia, M.
, &amp; 
Blencowe, B. J.
 (2014). Widespread intron retention in mammals functionally tunes transcriptomes. Genome Research, 24(11), 1774–1786. 10.1101/gr.177790.114

</cite> [<a href="https://doi.org/10.1101/gr.177790.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4216919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25258385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=Widespread%20intron%20retention%20in%20mammals%20functionally%20tunes%20transcriptomes&amp;author=U.%20Braunschweig&amp;author=N.%20L.%20Barbosa%E2%80%90Morais&amp;author=Q.%20Pan&amp;author=E.%20N.%20Nachman&amp;author=B.%20Alipanahi&amp;volume=24&amp;issue=11&amp;publication_year=2014&amp;pages=1774-1786&amp;pmid=25258385&amp;doi=10.1101/gr.177790.114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0007">
<cite>

Chaudhary, S.
, 
Khokhar, W.
, 
Jabre, I.
, 
Reddy, A. S. N.
, 
Byrne, L. J.
, 
Wilson, C. M.
, &amp; 
Syed, N. H.
 (2019). Alternative splicing and protein diversity: Plants versus animals. Frontiers in Plant Science, 10(June), Article 708. 10.3389/fpls.2019.00708

</cite> [<a href="https://doi.org/10.3389/fpls.2019.00708" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6581706/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31244866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Alternative%20splicing%20and%20protein%20diversity:%20Plants%20versus%20animals&amp;author=S.%20Chaudhary&amp;author=W.%20Khokhar&amp;author=I.%20Jabre&amp;author=A.%20S.%20N.%20Reddy&amp;author=L.%20J.%20Byrne&amp;volume=10&amp;issue=June&amp;publication_year=2019&amp;pages=708&amp;pmid=31244866&amp;doi=10.3389/fpls.2019.00708&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0008">
<cite>

Cheng, X.
, 
Zhao, C.
, 
Gao, L.
, 
Zeng, L.
, 
Xu, Y.
, 
Liu, F.
, 
Huang, J.
, 
Liu, L.
, 
Liu, S.
, &amp; 
Zhang, X.
 (2022). Alternative splicing reprogramming in fungal pathogen <em>Sclerotinia sclerotiorum</em> at different infection stages on <em>Brassica napus</em>
. Frontiers in Plant Science, 13(October), 1008665. 10.3389/fpls.2022.1008665

</cite> [<a href="https://doi.org/10.3389/fpls.2022.1008665" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9597501/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36311105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Alternative%20splicing%20reprogramming%20in%20fungal%20pathogen%20Sclerotinia%20sclerotiorum%20at%20different%20infection%20stages%20on%20Brassica%20napus&amp;author=X.%20Cheng&amp;author=C.%20Zhao&amp;author=L.%20Gao&amp;author=L.%20Zeng&amp;author=Y.%20Xu&amp;volume=13&amp;issue=October&amp;publication_year=2022&amp;pages=1008665&amp;pmid=36311105&amp;doi=10.3389/fpls.2022.1008665&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0009">
<cite>

Craft, J.
, 
Samalova, M.
, 
Baroux, C.
, 
Townley, H.
, 
Martinez, A.
, 
Jepson, I.
, 
Tsiantis, M.
, &amp; 
Moore, I.
 (2005). New pOp/LhG4 vectors for stringent glucocorticoid‐dependent transgene expression in Arabidopsis. The Plant Journal, 41, 899–918. 10.1111/j.1365-313X.2005.02342.x

</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2005.02342.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15743453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=New%20pOp/LhG4%20vectors%20for%20stringent%20glucocorticoid%E2%80%90dependent%20transgene%20expression%20in%20Arabidopsis&amp;author=J.%20Craft&amp;author=M.%20Samalova&amp;author=C.%20Baroux&amp;author=H.%20Townley&amp;author=A.%20Martinez&amp;volume=41&amp;publication_year=2005&amp;pages=899-918&amp;pmid=15743453&amp;doi=10.1111/j.1365-313X.2005.02342.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0010">
<cite>

Czechowski, T.
, 
Stitt, M.
, 
Altmann, T.
, 
Udvardi, M. K.
, &amp; 
Scheible, W.‐R.
 (2005). Genome‐wide identification and testing of superior reference genes for transcript normalization in Arabidopsis. Plant Physiology, 139(1), 5‐‑17. 10.1104/pp.105.063743

</cite> [<a href="https://doi.org/10.1104/pp.105.063743" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1203353/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16166256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Genome%E2%80%90wide%20identification%20and%20testing%20of%20superior%20reference%20genes%20for%20transcript%20normalization%20in%20Arabidopsis&amp;author=T.%20Czechowski&amp;author=M.%20Stitt&amp;author=T.%20Altmann&amp;author=M.%20K.%20Udvardi&amp;author=W.%E2%80%90R.%20Scheible&amp;volume=139&amp;issue=1&amp;publication_year=2005&amp;pages=5%E2%80%90%E2%80%9117&amp;pmid=16166256&amp;doi=10.1104/pp.105.063743&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0078">
<cite>

De Rybel, B.
, 
Möller, B.
, 
Yoshida, S.
, 
Grabowicz, I.
, 
Barbier de Reuille, P.
, 
Boeren, S.
, 
Smith, R. S.
, 
Borst, J. W.
, &amp; 
Weijers, D.
 (2013). A bHLH complex controls embryonic vascular tissue establishment and indeterminate growth in Arabidopsis. Developmental Cell, 24(4), 426–437. 10.1016/j.devcel.2012.12.013

</cite> [<a href="https://doi.org/10.1016/j.devcel.2012.12.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23415953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Developmental%20Cell&amp;title=A%20bHLH%20complex%20controls%20embryonic%20vascular%20tissue%20establishment%20and%20indeterminate%20growth%20in%20Arabidopsis&amp;author=B.%20De%20Rybel&amp;author=B.%20M%C3%B6ller&amp;author=S.%20Yoshida&amp;author=I.%20Grabowicz&amp;author=P.%20Barbier%20de%20Reuille&amp;volume=24&amp;issue=4&amp;publication_year=2013&amp;pages=426-437&amp;pmid=23415953&amp;doi=10.1016/j.devcel.2012.12.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0011">
<cite>

Drechsel, G.
, 
Kahles, A.
, 
Kesarwani, A. K.
, 
Stauffer, E.
, 
Behr, J.
, 
Drewe, P.
, 
Rätsch, G.
, &amp; 
Wachter, A.
 (2013). Nonsense‐mediated decay of alternative precursor mRNA splicing variants is a major determinant of the Arabidopsis steady state transcriptome. Plant Cell, 25(10), 3726–3742. 10.1105/tpc.113.115485

</cite> [<a href="https://doi.org/10.1105/tpc.113.115485" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3877825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24163313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Nonsense%E2%80%90mediated%20decay%20of%20alternative%20precursor%20mRNA%20splicing%20variants%20is%20a%20major%20determinant%20of%20the%20Arabidopsis%20steady%20state%20transcriptome&amp;author=G.%20Drechsel&amp;author=A.%20Kahles&amp;author=A.%20K.%20Kesarwani&amp;author=E.%20Stauffer&amp;author=J.%20Behr&amp;volume=25&amp;issue=10&amp;publication_year=2013&amp;pages=3726-3742&amp;pmid=24163313&amp;doi=10.1105/tpc.113.115485&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0012">
<cite>

Dressano, K.
, 
Weckwerth, P. R.
, 
Poretsky, E.
, 
Takahashi, Y.
, 
Villarreal, C.
, 
Shen, Z.
, 
Schroeder, J. I.
, 
Briggs, S. P.
, &amp; 
Huffaker, A.
 (2020). Dynamic regulation of Pep‐induced immunity through post‐translational control of defence transcript splicing. Nature Plants, 6(8), 1008–1019. 10.1038/s41477-020-0724-1

</cite> [<a href="https://doi.org/10.1038/s41477-020-0724-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7482133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32690890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Plants&amp;title=Dynamic%20regulation%20of%20Pep%E2%80%90induced%20immunity%20through%20post%E2%80%90translational%20control%20of%20defence%20transcript%20splicing&amp;author=K.%20Dressano&amp;author=P.%20R.%20Weckwerth&amp;author=E.%20Poretsky&amp;author=Y.%20Takahashi&amp;author=C.%20Villarreal&amp;volume=6&amp;issue=8&amp;publication_year=2020&amp;pages=1008-1019&amp;pmid=32690890&amp;doi=10.1038/s41477-020-0724-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0013">
<cite>

Driouich, A.
, 
Follet‐Gueye, M. L.
, 
Vicré‐Gibouin, M.
, &amp; 
Hawes, M.
 (2013). Root border cells and secretions as critical elements in plant host defense. Current Opinion in Plant Biology, 16(4), 489–495. 10.1016/j.pbi.2013.06.010

</cite> [<a href="https://doi.org/10.1016/j.pbi.2013.06.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23856080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Plant%20Biology&amp;title=Root%20border%20cells%20and%20secretions%20as%20critical%20elements%20in%20plant%20host%20defense&amp;author=A.%20Driouich&amp;author=M.%20L.%20Follet%E2%80%90Gueye&amp;author=M.%20Vicr%C3%A9%E2%80%90Gibouin&amp;author=M.%20Hawes&amp;volume=16&amp;issue=4&amp;publication_year=2013&amp;pages=489-495&amp;pmid=23856080&amp;doi=10.1016/j.pbi.2013.06.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0014">
<cite>

Driouich, A.
, 
Gaudry, A.
, 
Pawlak, B.
, &amp; 
Moore, J. P.
 (2021). Root cap–derived cells and mucilage: A protective network at the root tip. Protoplasma, 258(6), 1179–1185. 10.1007/s00709-021-01660-y

</cite> [<a href="https://doi.org/10.1007/s00709-021-01660-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34196784/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protoplasma&amp;title=Root%20cap%E2%80%93derived%20cells%20and%20mucilage:%20A%20protective%20network%20at%20the%20root%20tip&amp;author=A.%20Driouich&amp;author=A.%20Gaudry&amp;author=B.%20Pawlak&amp;author=J.%20P.%20Moore&amp;volume=258&amp;issue=6&amp;publication_year=2021&amp;pages=1179-1185&amp;pmid=34196784&amp;doi=10.1007/s00709-021-01660-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0015">
<cite>

Erkelenz, S.
, 
Mueller, W. F.
, 
Evans, M. S.
, 
Busch, A.
, 
Schöneweis, K.
, 
Hertel, K. J.
, &amp; 
Schaal, H.
 (2013). Position‐dependent splicing activation and repression by SR and hnRNP proteins rely on common mechanisms. RNA, 19(7), 96–102. 10.1261/rna.037044.112.their

</cite> [<a href="https://doi.org/10.1261/rna.037044.112.their" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3527730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23175589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RNA&amp;title=Position%E2%80%90dependent%20splicing%20activation%20and%20repression%20by%20SR%20and%20hnRNP%20proteins%20rely%20on%20common%20mechanisms&amp;author=S.%20Erkelenz&amp;author=W.%20F.%20Mueller&amp;author=M.%20S.%20Evans&amp;author=A.%20Busch&amp;author=K.%20Sch%C3%B6neweis&amp;volume=19&amp;issue=7&amp;publication_year=2013&amp;pages=96-102&amp;pmid=23175589&amp;doi=10.1261/rna.037044.112.their&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0016">
<cite>

Fortier, M.
, 
Lemaitre, V.
, 
Gaudry, A.
, 
Pawlak, B.
, 
Driouich, A.
, 
Follet‐Gueye, M. L.
, &amp; 
Vicré, M.
 (2023). A fine‐tuned defense at the pea root caps: Involvement of border cells and arabinogalactan proteins against soilborne diseases. Frontiers in Plant Science, 14(February), 1132132. 10.3389/fpls.2023.1132132

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1132132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9947496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36844081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=A%20fine%E2%80%90tuned%20defense%20at%20the%20pea%20root%20caps:%20Involvement%20of%20border%20cells%20and%20arabinogalactan%20proteins%20against%20soilborne%20diseases&amp;author=M.%20Fortier&amp;author=V.%20Lemaitre&amp;author=A.%20Gaudry&amp;author=B.%20Pawlak&amp;author=A.%20Driouich&amp;volume=14&amp;issue=February&amp;publication_year=2023&amp;pages=1132132&amp;pmid=36844081&amp;doi=10.3389/fpls.2023.1132132&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0017">
<cite>

Goedhart, J.
, 
Von Stetten, D.
, 
Noirclerc‐Savoye, M.
, 
Lelimousin, M.
, 
Joosen, L.
, 
Hink, M. A.
, 
Van Weeren, L.
, 
Gadella, T. W. J.
, &amp; 
Royant, A.
 (2012). Structure‐guided evolution of cyan fluorescent proteins towards a quantum yield of 93%. Nature Communications, 3, Article 751. 10.1038/ncomms1738
</cite> [<a href="https://doi.org/10.1038/ncomms1738" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3316892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22434194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Structure%E2%80%90guided%20evolution%20of%20cyan%20fluorescent%20proteins%20towards%20a%20quantum%20yield%20of%2093%&amp;author=J.%20Goedhart&amp;author=D.%20Von%20Stetten&amp;author=M.%20Noirclerc%E2%80%90Savoye&amp;author=M.%20Lelimousin&amp;author=L.%20Joosen&amp;volume=3&amp;publication_year=2012&amp;pages=751&amp;pmid=22434194&amp;doi=10.1038/ncomms1738&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0018">
<cite>

Göhring, J.
, 
Jacak, J.
, &amp; 
Barta, A.
 (2014). Imaging of endogenous messenger RNA splice variants in living cells reveals nuclear retention of transcripts inaccessible to nonsense‐mediated decay in Arabidopsis. Plant Cell, 26(2), 754‐‑764. 10.1105/tpc.113.118075

</cite> [<a href="https://doi.org/10.1105/tpc.113.118075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3967038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24532591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Imaging%20of%20endogenous%20messenger%20RNA%20splice%20variants%20in%20living%20cells%20reveals%20nuclear%20retention%20of%20transcripts%20inaccessible%20to%20nonsense%E2%80%90mediated%20decay%20in%20Arabidopsis&amp;author=J.%20G%C3%B6hring&amp;author=J.%20Jacak&amp;author=A.%20Barta&amp;volume=26&amp;issue=2&amp;publication_year=2014&amp;pages=754%E2%80%90%E2%80%91764&amp;pmid=24532591&amp;doi=10.1105/tpc.113.118075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0019">
<cite>

Golisz, A.
, 
Krzyszton, M.
, 
Stepien, M.
, 
Dolata, J.
, 
Piotrowska, J.
, 
Szweykowska‐Kulinska, Z.
, 
Jarmolowski, A.
, &amp; 
Kufel, J.
 (2021). 
<em>Arabidopsis spliceosome</em> factor SmD3 modulates immunity to <em>Pseudomonas syringae</em> infection. Frontiers in Plant Science, 12, Article 765003. 10.3389/fpls.2021.765003

</cite> [<a href="https://doi.org/10.3389/fpls.2021.765003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8678131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34925413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Arabidopsis%20spliceosome%20factor%20SmD3%20modulates%20immunity%20to%20Pseudomonas%20syringae%20infection&amp;author=A.%20Golisz&amp;author=M.%20Krzyszton&amp;author=M.%20Stepien&amp;author=J.%20Dolata&amp;author=J.%20Piotrowska&amp;volume=12&amp;publication_year=2021&amp;pages=765003&amp;pmid=34925413&amp;doi=10.3389/fpls.2021.765003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0020">
<cite>

Guo, X.
, 
Wang, T.
, 
Jiang, L.
, 
Qi, H.
, &amp; 
Zhang, Z.
 (2023). PlaASDB: A comprehensive database of plant alternative splicing events in response to stress. BMC Plant Biology, 23(1), Article 225. 10.1186/s12870-023-04234-7

</cite> [<a href="https://doi.org/10.1186/s12870-023-04234-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10134664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37106367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=PlaASDB:%20A%20comprehensive%20database%20of%20plant%20alternative%20splicing%20events%20in%20response%20to%20stress&amp;author=X.%20Guo&amp;author=T.%20Wang&amp;author=L.%20Jiang&amp;author=H.%20Qi&amp;author=Z.%20Zhang&amp;volume=23&amp;issue=1&amp;publication_year=2023&amp;pages=225&amp;pmid=37106367&amp;doi=10.1186/s12870-023-04234-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0021">
<cite>

Han, H.
, 
Yan, A.
, 
Li, L.
, 
Zhu, Y.
, 
Feng, B.
, 
Liu, X.
, &amp; 
Zhou, Y.
 (2020). A signal cascade originated from epidermis defines apical‐basal patterning of Arabidopsis shoot apical meristems. Nature Communications, 11(1), Article 1214. 10.1038/s41467-020-14989-4
</cite> [<a href="https://doi.org/10.1038/s41467-020-14989-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7058014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32139673/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=A%20signal%20cascade%20originated%20from%20epidermis%20defines%20apical%E2%80%90basal%20patterning%20of%20Arabidopsis%20shoot%20apical%20meristems&amp;author=H.%20Han&amp;author=A.%20Yan&amp;author=L.%20Li&amp;author=Y.%20Zhu&amp;author=B.%20Feng&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=1214&amp;pmid=32139673&amp;doi=10.1038/s41467-020-14989-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0022">
<cite>

Hartmann, L.
, 
Wießner, T.
, &amp; 
Wachter, A.
 (2018). Subcellular compartmentation of alternatively spliced transcripts defines SERINE/ARGININE‐RICH PROTEIN30 expression. Plant Physiology, 176(4), 2886‑‐2903. 10.1104/pp.17.01260

</cite> [<a href="https://doi.org/10.1104/pp.17.01260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5884584/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29496883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Subcellular%20compartmentation%20of%20alternatively%20spliced%20transcripts%20defines%20SERINE/ARGININE%E2%80%90RICH%20PROTEIN30%20expression&amp;author=L.%20Hartmann&amp;author=T.%20Wie%C3%9Fner&amp;author=A.%20Wachter&amp;volume=176&amp;issue=4&amp;publication_year=2018&amp;pages=2886%E2%80%91%E2%80%902903&amp;pmid=29496883&amp;doi=10.1104/pp.17.01260&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0023">
<cite>

Hawes, M.
, 
Allen, C.
, 
Turgeon, B. G.
, 
Curlango‐Rivera, G.
, 
Minh Tran, T.
, 
Huskey, D. A.
, &amp; 
Xiong, Z.
 (2016). Root border cells and their role in plant defense. Annual Review of Phytopathology, 54, 143–161. 10.1146/annurev-phyto-080615-100140
</cite> [<a href="https://doi.org/10.1146/annurev-phyto-080615-100140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27215971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Phytopathology&amp;title=Root%20border%20cells%20and%20their%20role%20in%20plant%20defense&amp;author=M.%20Hawes&amp;author=C.%20Allen&amp;author=B.%20G.%20Turgeon&amp;author=G.%20Curlango%E2%80%90Rivera&amp;author=T.%20Minh%20Tran&amp;volume=54&amp;publication_year=2016&amp;pages=143-161&amp;pmid=27215971&amp;doi=10.1146/annurev-phyto-080615-100140&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0024">
<cite>

Hawes, M. C.
, 
Gunawardena, U.
, 
Miyasaka, S.
, &amp; 
Zhao, X.
 (2000). The role of root border cells in plant defense. Trends in Plant Science, 5(3), 128–133. 10.1016/S1360-1385(00)01556-9

</cite> [<a href="https://doi.org/10.1016/S1360-1385(00)01556-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10707079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=The%20role%20of%20root%20border%20cells%20in%20plant%20defense&amp;author=M.%20C.%20Hawes&amp;author=U.%20Gunawardena&amp;author=S.%20Miyasaka&amp;author=X.%20Zhao&amp;volume=5&amp;issue=3&amp;publication_year=2000&amp;pages=128-133&amp;pmid=10707079&amp;doi=10.1016/S1360-1385(00)01556-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0025">
<cite>

Hazra, A.
, 
Pal, A.
, &amp; 
Kundu, A.
 (2023). Alternative splicing shapes the transcriptome complexity in blackgram [<em>Vigna mungo</em> (L.) Hepper]. Functional &amp; Integrative Genomics, 23(2), 144. 10.1007/s10142-023-01066-4

</cite> [<a href="https://doi.org/10.1007/s10142-023-01066-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37133618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20&amp;%20Integrative%20Genomics&amp;title=Alternative%20splicing%20shapes%20the%20transcriptome%20complexity%20in%20blackgram%20%5BVigna%20mungo%20(L.)%20Hepper%5D&amp;author=A.%20Hazra&amp;author=A.%20Pal&amp;author=A.%20Kundu&amp;volume=23&amp;issue=2&amp;publication_year=2023&amp;pages=144&amp;pmid=37133618&amp;doi=10.1007/s10142-023-01066-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0077">
<cite>

Hepworth, S. R.
, 
Klenz, J. E.
, &amp; 
Haughn, G. W.
 (2006). UFO in the Arabidopsis inflorescence apex is required for floral‐meristem identity and bract suppression. Planta, 223, 769–778. 10.1007/s00425-005-0138-3

</cite> [<a href="https://doi.org/10.1007/s00425-005-0138-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16244866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=UFO%20in%20the%20Arabidopsis%20inflorescence%20apex%20is%20required%20for%20floral%E2%80%90meristem%20identity%20and%20bract%20suppression&amp;author=S.%20R.%20Hepworth&amp;author=J.%20E.%20Klenz&amp;author=G.%20W.%20Haughn&amp;volume=223&amp;publication_year=2006&amp;pages=769-778&amp;pmid=16244866&amp;doi=10.1007/s00425-005-0138-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0026">
<cite>

Hermanns, M.
, 
Slusarenko, A. J.
, &amp; 
Schlaich, N. L.
 (2003). Organ‐specificity in a plant disease is determined independently of <em>R</em> gene signaling. Molecular Plant‐Microbe Interactions, 16(9), 752–759. 10.1094/MPMI.2003.16.9.752

</cite> [<a href="https://doi.org/10.1094/MPMI.2003.16.9.752" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12971598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant%E2%80%90Microbe%20Interactions&amp;title=Organ%E2%80%90specificity%20in%20a%20plant%20disease%20is%20determined%20independently%20of%20R%20gene%20signaling&amp;author=M.%20Hermanns&amp;author=A.%20J.%20Slusarenko&amp;author=N.%20L.%20Schlaich&amp;volume=16&amp;issue=9&amp;publication_year=2003&amp;pages=752-759&amp;pmid=12971598&amp;doi=10.1094/MPMI.2003.16.9.752&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0027">
<cite>

Hertel, K. J.
, &amp; 
Maniatis, T.
 (1999). Serine‐arginine (SR)‐rich splicing factors have an exon‐independent function in pre‐mRNA splicing. Proceedings of the National Academy of Sciences of the United States of America, 96(6), 2651–2655. 10.1073/pnas.96.6.2651

</cite> [<a href="https://doi.org/10.1073/pnas.96.6.2651" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC15823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10077565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Serine%E2%80%90arginine%20(SR)%E2%80%90rich%20splicing%20factors%20have%20an%20exon%E2%80%90independent%20function%20in%20pre%E2%80%90mRNA%20splicing&amp;author=K.%20J.%20Hertel&amp;author=T.%20Maniatis&amp;volume=96&amp;issue=6&amp;publication_year=1999&amp;pages=2651-2655&amp;pmid=10077565&amp;doi=10.1073/pnas.96.6.2651&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0028">
<cite>

Howard, B. E.
, 
Hu, Q.
, 
Babaoglu, A. C.
, 
Chandra, M.
, 
Borghi, M.
, 
Tan, X.
, 
He, L.
, 
Winter‐Sederoff, H.
, 
Gassmann, W.
, 
Veronese, P.
, &amp; 
Heber, S.
 (2013). High‐throughput RNA sequencing of pseudomonas‐infected Arabidopsis reveals hidden transcriptome complexity and novel splice variants. PLoS One, 8(10), e74183. 10.1371/journal.pone.0074183

</cite> [<a href="https://doi.org/10.1371/journal.pone.0074183" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3788074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24098335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=High%E2%80%90throughput%20RNA%20sequencing%20of%20pseudomonas%E2%80%90infected%20Arabidopsis%20reveals%20hidden%20transcriptome%20complexity%20and%20novel%20splice%20variants&amp;author=B.%20E.%20Howard&amp;author=Q.%20Hu&amp;author=A.%20C.%20Babaoglu&amp;author=M.%20Chandra&amp;author=M.%20Borghi&amp;volume=8&amp;issue=10&amp;publication_year=2013&amp;pages=e74183&amp;pmid=24098335&amp;doi=10.1371/journal.pone.0074183&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0029">
<cite>

Huang, D. W.
, 
Sherman, B. T.
, &amp; 
Lempicki, R. A.
 (2009). Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources. Nature Protocols, 4(1), 44–57. 10.1038/nprot.2008.211

</cite> [<a href="https://doi.org/10.1038/nprot.2008.211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19131956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Protocols&amp;title=Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20bioinformatics%20resources&amp;author=D.%20W.%20Huang&amp;author=B.%20T.%20Sherman&amp;author=R.%20A.%20Lempicki&amp;volume=4&amp;issue=1&amp;publication_year=2009&amp;pages=44-57&amp;pmid=19131956&amp;doi=10.1038/nprot.2008.211&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0030">
<cite>

Huang, J.
, 
Gu, L.
, 
Zhang, Y.
, 
Yan, T.
, 
Kong, G.
, 
Kong, L.
, 
Guo, B.
, 
Qiu, M.
, 
Wang, Y.
, 
Jing, M.
, 
Xing, W.
, 
Ye, W.
, 
Wu, Z.
, 
Zhang, Z.
, 
Zheng, X.
, 
Gijzen, M.
, 
Wang, Y.
, &amp; 
Dong, S.
 (2017). An oomycete plant pathogen reprograms host pre‐mRNA splicing to subvert immunity. Nature Communications, 8(1), Article 2051. 10.1038/s41467-017-02233-5
</cite> [<a href="https://doi.org/10.1038/s41467-017-02233-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5727057/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29233978/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=An%20oomycete%20plant%20pathogen%20reprograms%20host%20pre%E2%80%90mRNA%20splicing%20to%20subvert%20immunity&amp;author=J.%20Huang&amp;author=L.%20Gu&amp;author=Y.%20Zhang&amp;author=T.%20Yan&amp;author=G.%20Kong&amp;volume=8&amp;issue=1&amp;publication_year=2017&amp;pages=2051&amp;pmid=29233978&amp;doi=10.1038/s41467-017-02233-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0031">
<cite>

Huelga, S. C.
, 
Vu, A. Q.
, 
Arnold, J. D.
, 
Liang, T. D.
, 
Liu, P. P.
, 
Yan, B. Y.
, 
Donohue, J. P.
, 
Shiue, L.
, 
Hoon, S.
, 
Brenner, S.
, 
Ares, M.
, &amp; 
Yeo, G. W.
 (2012). Integrative genome‐wide analysis reveals cooperative regulation of alternative splicing by hnRNP proteins. Cell Reports, 1(2), 167–178. 10.1016/j.celrep.2012.02.001

</cite> [<a href="https://doi.org/10.1016/j.celrep.2012.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3345519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22574288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Reports&amp;title=Integrative%20genome%E2%80%90wide%20analysis%20reveals%20cooperative%20regulation%20of%20alternative%20splicing%20by%20hnRNP%20proteins&amp;author=S.%20C.%20Huelga&amp;author=A.%20Q.%20Vu&amp;author=J.%20D.%20Arnold&amp;author=T.%20D.%20Liang&amp;author=P.%20P.%20Liu&amp;volume=1&amp;issue=2&amp;publication_year=2012&amp;pages=167-178&amp;pmid=22574288&amp;doi=10.1016/j.celrep.2012.02.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0032">
<cite>

Irimia, M.
, 
Weatheritt, R. J.
, 
Ellis, J. D.
, 
Parikshak, N. N.
, 
Gonatopoulos‐Pournatzis, T.
, 
Babor, M.
, 
Quesnel‐Vallières, M.
, 
Tapial, J.
, 
Raj, B.
, 
O'Hanlon, D.
, 
Barrios‐Rodiles, M.
, 
Sternberg, M. J. E.
, 
Cordes, S. P.
, 
Roth, F. P.
, 
Wrana, J. L.
, 
Geschwind, D. H.
, &amp; 
Blencowe, B. J.
 (2014). A highly conserved program of neuronal microexons is misregulated in autistic brains. Cell, 159(7), 1511–1523. 10.1016/j.cell.2014.11.035

</cite> [<a href="https://doi.org/10.1016/j.cell.2014.11.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4390143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25525873/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20highly%20conserved%20program%20of%20neuronal%20microexons%20is%20misregulated%20in%20autistic%20brains&amp;author=M.%20Irimia&amp;author=R.%20J.%20Weatheritt&amp;author=J.%20D.%20Ellis&amp;author=N.%20N.%20Parikshak&amp;author=T.%20Gonatopoulos%E2%80%90Pournatzis&amp;volume=159&amp;issue=7&amp;publication_year=2014&amp;pages=1511-1523&amp;pmid=25525873&amp;doi=10.1016/j.cell.2014.11.035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0033">
<cite>

Jia, Z.
, 
Das, D.
, 
Zhang, Y.
, 
Fernie, A. R.
, 
Liu, Y.
, 
Chen, M.
, &amp; 
Zhang, J.
 (2023). Plant serine/arginine‐rich proteins: Versatile players in RNA processing. Planta, 257(6), 109.
</cite> [<a href="https://doi.org/10.1007/s00425-023-04132-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37145304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Plant%20serine/arginine%E2%80%90rich%20proteins:%20Versatile%20players%20in%20RNA%20processing&amp;author=Z.%20Jia&amp;author=D.%20Das&amp;author=Y.%20Zhang&amp;author=A.%20R.%20Fernie&amp;author=Y.%20Liu&amp;volume=257&amp;issue=6&amp;publication_year=2023&amp;pages=109&amp;pmid=37145304&amp;doi=10.1007/s00425-023-04132-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0034">
<cite>

Kalyna, M.
, 
Simpson, C. G.
, 
Syed, N. H.
, 
Lewandowska, D.
, 
Marquez, Y.
, 
Kusenda, B.
, 
Marshall, J.
, 
Fuller, J.
, 
Cardle, L.
, 
McNicol, J.
, 
Dinh, H. Q.
, 
Barta, A.
, &amp; 
Brown, J. W. S.
 (2012). Alternative splicing and nonsense‐mediated decay modulate expression of important regulatory genes in Arabidopsis. Nucleic Acids Research, 40(6), 2454–2469. 10.1093/nar/gkr932

</cite> [<a href="https://doi.org/10.1093/nar/gkr932" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3315328/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22127866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Alternative%20splicing%20and%20nonsense%E2%80%90mediated%20decay%20modulate%20expression%20of%20important%20regulatory%20genes%20in%20Arabidopsis&amp;author=M.%20Kalyna&amp;author=C.%20G.%20Simpson&amp;author=N.%20H.%20Syed&amp;author=D.%20Lewandowska&amp;author=Y.%20Marquez&amp;volume=40&amp;issue=6&amp;publication_year=2012&amp;pages=2454-2469&amp;pmid=22127866&amp;doi=10.1093/nar/gkr932&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0035">
<cite>

Kumar, N.
, &amp; 
Iyer‐Pascuzzi, A. S.
 (2020). Shedding the last layer: Mechanisms of root cap cell release. Plants, 9(3), 308. 10.3390/plants9030308

</cite> [<a href="https://doi.org/10.3390/plants9030308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7154840/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32121604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Shedding%20the%20last%20layer:%20Mechanisms%20of%20root%20cap%20cell%20release&amp;author=N.%20Kumar&amp;author=A.%20S.%20Iyer%E2%80%90Pascuzzi&amp;volume=9&amp;issue=3&amp;publication_year=2020&amp;pages=308&amp;pmid=32121604&amp;doi=10.3390/plants9030308&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0036">
<cite>

Langmead, B.
, 
Trapnell, C.
, 
Pop, M.
, &amp; 
Salzberg, S. L.
 (2009). Ultrafast and memory‐efficient alignment of short DNA sequences to the human genome. Genome Biology, 10(3), Article R25. 10.1186/gb-2009-10-3-r25

</cite> [<a href="https://doi.org/10.1186/gb-2009-10-3-r25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2690996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19261174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Ultrafast%20and%20memory%E2%80%90efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.%20L.%20Salzberg&amp;volume=10&amp;issue=3&amp;publication_year=2009&amp;pages=R25&amp;pmid=19261174&amp;doi=10.1186/gb-2009-10-3-r25&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0037">
<cite>

Lazar, G.
, &amp; 
Goodman, H. M.
 (2000). The Arabidopsis splicing factor SR1 is regulated by alternative splicing. Plant Molecular Biology, 42(4), 571–581. 10.1023/A:1006394207479

</cite> [<a href="https://doi.org/10.1023/A:1006394207479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10809003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Molecular%20Biology&amp;title=The%20Arabidopsis%20splicing%20factor%20SR1%20is%20regulated%20by%20alternative%20splicing&amp;author=G.%20Lazar&amp;author=H.%20M.%20Goodman&amp;volume=42&amp;issue=4&amp;publication_year=2000&amp;pages=571-581&amp;pmid=10809003&amp;doi=10.1023/A:1006394207479&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0038">
<cite>

Li, R.
, 
Wei, Z.
, 
Li, Y.
, 
Shang, X.
, 
Cao, Y.
, 
Duan, L.
, &amp; 
Ma, L.
 (2022). SKI‐INTERACTING PROTEIN interacts with SHOOT MERISTEMLESS to regulate shoot apical meristem formation. Plant Physiology, 189(4), 2193–2209. 10.1093/plphys/kiac241

</cite> [<a href="https://doi.org/10.1093/plphys/kiac241" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9342996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35640153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=SKI%E2%80%90INTERACTING%20PROTEIN%20interacts%20with%20SHOOT%20MERISTEMLESS%20to%20regulate%20shoot%20apical%20meristem%20formation&amp;author=R.%20Li&amp;author=Z.%20Wei&amp;author=Y.%20Li&amp;author=X.%20Shang&amp;author=Y.%20Cao&amp;volume=189&amp;issue=4&amp;publication_year=2022&amp;pages=2193-2209&amp;pmid=35640153&amp;doi=10.1093/plphys/kiac241&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0039">
<cite>

Li, S.
, 
Yamada, M.
, 
Han, X.
, 
Ohler, U.
, &amp; 
Benfey, P. N.
 (2016). High‐resolution expression map of the Arabidopsis root reveals alternative splicing and lincRNA regulation. Developmental Cell, 39(4), 508–522. 10.1016/j.devcel.2016.10.012

</cite> [<a href="https://doi.org/10.1016/j.devcel.2016.10.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5125536/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27840108/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Developmental%20Cell&amp;title=High%E2%80%90resolution%20expression%20map%20of%20the%20Arabidopsis%20root%20reveals%20alternative%20splicing%20and%20lincRNA%20regulation&amp;author=S.%20Li&amp;author=M.%20Yamada&amp;author=X.%20Han&amp;author=U.%20Ohler&amp;author=P.%20N.%20Benfey&amp;volume=39&amp;issue=4&amp;publication_year=2016&amp;pages=508-522&amp;pmid=27840108&amp;doi=10.1016/j.devcel.2016.10.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0040">
<cite>

Li, W.
, 
Lin, W. D.
, 
Ray, P.
, 
Lan, P.
, &amp; 
Schmidt, W.
 (2013). Genome‐wide detection of condition‐sensitive alternative splicing in Arabidopsis roots. Plant Physiology, 162(3), 1750–1763. 10.1104/pp.113.217778

</cite> [<a href="https://doi.org/10.1104/pp.113.217778" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3700675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23735510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Genome%E2%80%90wide%20detection%20of%20condition%E2%80%90sensitive%20alternative%20splicing%20in%20Arabidopsis%20roots&amp;author=W.%20Li&amp;author=W.%20D.%20Lin&amp;author=P.%20Ray&amp;author=P.%20Lan&amp;author=W.%20Schmidt&amp;volume=162&amp;issue=3&amp;publication_year=2013&amp;pages=1750-1763&amp;pmid=23735510&amp;doi=10.1104/pp.113.217778&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0041">
<cite>

Liu, J.
, 
Sun, N.
, 
Liu, M.
, 
Liu, J.
, 
Du, B.
, 
Wang, X.
, &amp; 
Qi, X.
 (2013). An autoregulatory loop controlling Arabidopsis HsfA2 expression: Role of heat shock‐induced alternative splicing. Plant Physiology, 162(1), 512–521. 10.1104/pp.112.205864

</cite> [<a href="https://doi.org/10.1104/pp.112.205864" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3641227/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23503691/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=An%20autoregulatory%20loop%20controlling%20Arabidopsis%20HsfA2%20expression:%20Role%20of%20heat%20shock%E2%80%90induced%20alternative%20splicing&amp;author=J.%20Liu&amp;author=N.%20Sun&amp;author=M.%20Liu&amp;author=J.%20Liu&amp;author=B.%20Du&amp;volume=162&amp;issue=1&amp;publication_year=2013&amp;pages=512-521&amp;pmid=23503691&amp;doi=10.1104/pp.112.205864&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0042">
<cite>

Liu, J.
, 
Zhang, Y.
, 
Zheng, Y.
, 
Zhu, Y.
, 
Shi, Y.
, 
Guan, Z.
, 
Lang, K.
, 
Shen, D.
, 
Huang, W.
, &amp; 
Dou, D.
 (2023). PlantExp: A platform for exploration of gene expression and alternative splicing based on public plant RNA‐seq samples. Nucleic Acids Research, 51(D1), D1483–D1491. 10.1093/nar/gkac917

</cite> [<a href="https://doi.org/10.1093/nar/gkac917" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825497/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36271793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=PlantExp:%20A%20platform%20for%20exploration%20of%20gene%20expression%20and%20alternative%20splicing%20based%20on%20public%20plant%20RNA%E2%80%90seq%20samples&amp;author=J.%20Liu&amp;author=Y.%20Zhang&amp;author=Y.%20Zheng&amp;author=Y.%20Zhu&amp;author=Y.%20Shi&amp;volume=51&amp;issue=D1&amp;publication_year=2023&amp;pages=D1483-D1491&amp;pmid=36271793&amp;doi=10.1093/nar/gkac917&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0043">
<cite>

Liu, X.
, 
Guo, Q. H.
, 
Xu, W. B.
, 
Liu, P.
, &amp; 
Yan, K.
 (2022). Rapid regulation of alternative splicing in response to environmental stresses. Frontiers in Plant Science, 13, Article 832177. 10.3389/fpls.2022.832177

</cite> [<a href="https://doi.org/10.3389/fpls.2022.832177" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8931528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35310672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Rapid%20regulation%20of%20alternative%20splicing%20in%20response%20to%20environmental%20stresses&amp;author=X.%20Liu&amp;author=Q.%20H.%20Guo&amp;author=W.%20B.%20Xu&amp;author=P.%20Liu&amp;author=K.%20Yan&amp;volume=13&amp;publication_year=2022&amp;pages=832177&amp;pmid=35310672&amp;doi=10.3389/fpls.2022.832177&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0044">
<cite>

Lopato, S.
, 
Kalyna, M.
, 
Dorner, S.
, 
Kobayashi, R.
, 
Krainer, A. R.
, &amp; 
Barta, A.
 (1999). atSRp30, one of two SF2/ASF‐like proteins from <em>Arabidopsis thaliana</em>, regulates splicing of specific plant genes. Genes and Development, 13(8), 987–1001. 10.1101/gad.13.8.987

</cite> [<a href="https://doi.org/10.1101/gad.13.8.987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC316644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10215626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20and%20Development&amp;title=atSRp30,%20one%20of%20two%20SF2/ASF%E2%80%90like%20proteins%20from%20Arabidopsis%20thaliana,%20regulates%20splicing%20of%20specific%20plant%20genes&amp;author=S.%20Lopato&amp;author=M.%20Kalyna&amp;author=S.%20Dorner&amp;author=R.%20Kobayashi&amp;author=A.%20R.%20Krainer&amp;volume=13&amp;issue=8&amp;publication_year=1999&amp;pages=987-1001&amp;pmid=10215626&amp;doi=10.1101/gad.13.8.987&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0045">
<cite>

Lopato, S.
, 
Waigmann, E.
, &amp; 
Barta, A.
 (1996). Characterization of a novel arginine/serine‐rich splicing factor in Arabidopsis. The Plant Cell, 8(12), 2255–2264. 10.1105/tpc.8.12.2255

</cite> [<a href="https://doi.org/10.1105/tpc.8.12.2255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC161350/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8989882/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Cell&amp;title=Characterization%20of%20a%20novel%20arginine/serine%E2%80%90rich%20splicing%20factor%20in%20Arabidopsis&amp;author=S.%20Lopato&amp;author=E.%20Waigmann&amp;author=A.%20Barta&amp;volume=8&amp;issue=12&amp;publication_year=1996&amp;pages=2255-2264&amp;pmid=8989882&amp;doi=10.1105/tpc.8.12.2255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0046">
<cite>

Love, M. I.
, 
Huber, W.
, &amp; 
Anders, S.
 (2014). Moderated estimation of fold change and dispersion for RNA‐seq data with DESeq2. Genome Biology, 15(12), Article 550. 10.1186/s13059-014-0550-8

</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA%E2%80%90seq%20data%20with%20DESeq2&amp;author=M.%20I.%20Love&amp;author=W.%20Huber&amp;author=S.%20Anders&amp;volume=15&amp;issue=12&amp;publication_year=2014&amp;pages=550&amp;pmid=25516281&amp;doi=10.1186/s13059-014-0550-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0047">
<cite>

Maeda, K.
, 
Kunieda, T.
, 
Tamura, K.
, 
Hatano, K.
, 
Hara‐Nishimura, I.
, &amp; 
Shimada, T.
 (2019). Identification of periplasmic root‐cap mucilage in developing columella cells of <em>Arabidopsis thaliana</em>
. Plant and Cell Physiology, 60(6), 1296–1303. 10.1093/pcp/pcz047

</cite> [<a href="https://doi.org/10.1093/pcp/pcz047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30892660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20and%20Cell%20Physiology&amp;title=Identification%20of%20periplasmic%20root%E2%80%90cap%20mucilage%20in%20developing%20columella%20cells%20of%20Arabidopsis%20thaliana&amp;author=K.%20Maeda&amp;author=T.%20Kunieda&amp;author=K.%20Tamura&amp;author=K.%20Hatano&amp;author=I.%20Hara%E2%80%90Nishimura&amp;volume=60&amp;issue=6&amp;publication_year=2019&amp;pages=1296-1303&amp;pmid=30892660&amp;doi=10.1093/pcp/pcz047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0048">
<cite>

Malamy, J. E.
, &amp; 
Benfey, P. N.
 (1997). Analysis of SCARECROW expression using a rapid system for assessing transgene expression in Arabidopsis roots. Plant Journal, 12(4), 957–963. 10.1046/j.1365-313X.1997.12040957.x
</cite> [<a href="https://doi.org/10.1046/j.1365-313X.1997.12040957.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9375406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Journal&amp;title=Analysis%20of%20SCARECROW%20expression%20using%20a%20rapid%20system%20for%20assessing%20transgene%20expression%20in%20Arabidopsis%20roots&amp;author=J.%20E.%20Malamy&amp;author=P.%20N.%20Benfey&amp;volume=12&amp;issue=4&amp;publication_year=1997&amp;pages=957-963&amp;pmid=9375406&amp;doi=10.1046/j.1365-313X.1997.12040957.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0049">
<cite>

Marquez, Y.
, 
Brown, J. W. S.
, 
Simpson, C.
, 
Barta, A.
, &amp; 
Kalyna, M.
 (2012). Transcriptome survey reveals increased complexity of the alternative splicing landscape in Arabidopsis. Genome Research, 22(6), 1184–1195. 10.1101/gr.134106.111

</cite> [<a href="https://doi.org/10.1101/gr.134106.111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3371709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22391557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=Transcriptome%20survey%20reveals%20increased%20complexity%20of%20the%20alternative%20splicing%20landscape%20in%20Arabidopsis&amp;author=Y.%20Marquez&amp;author=J.%20W.%20S.%20Brown&amp;author=C.%20Simpson&amp;author=A.%20Barta&amp;author=M.%20Kalyna&amp;volume=22&amp;issue=6&amp;publication_year=2012&amp;pages=1184-1195&amp;pmid=22391557&amp;doi=10.1101/gr.134106.111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0050">
<cite>

Martín, G.
, 
Márquez, Y.
, 
Mantica, F.
, 
Duque, P.
, &amp; 
Irimia, M.
 (2021). Alternative splicing landscapes in Arabidopsis thaliana across tissues and stress conditions highlight major functional differences with animals. Genome Biology, 22(1), Article 35. 10.1186/s13059-020-02258-y

</cite> [<a href="https://doi.org/10.1186/s13059-020-02258-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7807721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33446251/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Alternative%20splicing%20landscapes%20in%20Arabidopsis%20thaliana%20across%20tissues%20and%20stress%20conditions%20highlight%20major%20functional%20differences%20with%20animals&amp;author=G.%20Mart%C3%ADn&amp;author=Y.%20M%C3%A1rquez&amp;author=F.%20Mantica&amp;author=P.%20Duque&amp;author=M.%20Irimia&amp;volume=22&amp;issue=1&amp;publication_year=2021&amp;pages=35&amp;pmid=33446251&amp;doi=10.1186/s13059-020-02258-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0051">
<cite>

Misra, C. S.
, 
Sousa, A. G. G.
, 
Barros, P. M.
, 
Kermanov, A.
, &amp; 
Becker, J. D.
 (2023). Cell‐type‐specific alternative splicing in the Arabidopsis germline. Plant Physiology, 192(1), 85–101. 10.1093/plphys/kiac574

</cite> [<a href="https://doi.org/10.1093/plphys/kiac574" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10152659/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36515615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Cell%E2%80%90type%E2%80%90specific%20alternative%20splicing%20in%20the%20Arabidopsis%20germline&amp;author=C.%20S.%20Misra&amp;author=A.%20G.%20G.%20Sousa&amp;author=P.%20M.%20Barros&amp;author=A.%20Kermanov&amp;author=J.%20D.%20Becker&amp;volume=192&amp;issue=1&amp;publication_year=2023&amp;pages=85-101&amp;pmid=36515615&amp;doi=10.1093/plphys/kiac574&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0052">
<cite>

Ner‐Gaon, H.
, 
Halachmi, R.
, 
Savaldi‐Goldstein, S.
, 
Rubin, E.
, 
Ophir, R.
, &amp; 
Fluhr, R.
 (2004). Intron retention is a major phenomenon in alternative splicing in Arabidopsis. Plant Journal, 39(6), 877–885. 10.1111/j.1365-313X.2004.02172.x
</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2004.02172.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15341630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Journal&amp;title=Intron%20retention%20is%20a%20major%20phenomenon%20in%20alternative%20splicing%20in%20Arabidopsis&amp;author=H.%20Ner%E2%80%90Gaon&amp;author=R.%20Halachmi&amp;author=S.%20Savaldi%E2%80%90Goldstein&amp;author=E.%20Rubin&amp;author=R.%20Ophir&amp;volume=39&amp;issue=6&amp;publication_year=2004&amp;pages=877-885&amp;pmid=15341630&amp;doi=10.1111/j.1365-313X.2004.02172.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0053">
<cite>

Ogungbayi, A.
, 
Lee, J.
, 
Vaghela, V.
, 
Yu, F.
, 
Min, X.
, &amp; 
Xiangjia, M.
 (2023). Systematic collection and analysis of alternative splicing events in potato plants. Journal of Plant Sciences, 11(3), 98–106. 10.11648/j.jps.20231103.19
</cite> [<a href="https://doi.org/10.11648/j.jps.20231103.19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Plant%20Sciences&amp;title=Systematic%20collection%20and%20analysis%20of%20alternative%20splicing%20events%20in%20potato%20plants&amp;author=A.%20Ogungbayi&amp;author=J.%20Lee&amp;author=V.%20Vaghela&amp;author=F.%20Yu&amp;author=X.%20Min&amp;volume=11&amp;issue=3&amp;publication_year=2023&amp;pages=98-106&amp;doi=10.11648/j.jps.20231103.19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0054">
<cite>

Ohtani, M.
, &amp; 
Wachter, A.
 (2019). NMD‐based gene regulation—A strategy for fitness enhancement in plants?
Plant and Cell Physiology, 60(9), 1953–1960. 10.1093/pcp/pcz090

</cite> [<a href="https://doi.org/10.1093/pcp/pcz090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31111919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20and%20Cell%20Physiology&amp;title=NMD%E2%80%90based%20gene%20regulation%E2%80%94A%20strategy%20for%20fitness%20enhancement%20in%20plants?&amp;author=M.%20Ohtani&amp;author=A.%20Wachter&amp;volume=60&amp;issue=9&amp;publication_year=2019&amp;pages=1953-1960&amp;pmid=31111919&amp;doi=10.1093/pcp/pcz090&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0080">
<cite>

Otsuga, D.
, 
DeGuzman, B.
, 
Prigge, M. J.
, 
Drews, G. N.
, &amp; 
Clark, S. E.
 (2001). REVOLUTA regulates meristem initiation at lateral positions. The Plant Journal, 25(2), 223–236. 10.1111/j.1365-313x.2001.00959.x

</cite> [<a href="https://doi.org/10.1111/j.1365-313x.2001.00959.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11169198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=REVOLUTA%20regulates%20meristem%20initiation%20at%20lateral%20positions&amp;author=D.%20Otsuga&amp;author=B.%20DeGuzman&amp;author=M.%20J.%20Prigge&amp;author=G.%20N.%20Drews&amp;author=S.%20E.%20Clark&amp;volume=25&amp;issue=2&amp;publication_year=2001&amp;pages=223-236&amp;pmid=11169198&amp;doi=10.1111/j.1365-313x.2001.00959.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0055">
<cite>

Reddy, A. S. N.
, 
Marquez, Y.
, 
Kalyna, M.
, &amp; 
Barta, A.
 (2013). Complexity of the alternative splicing landscape in plants. Plant Cell, 25(10), 3657–3683. 10.1105/tpc.113.117523

</cite> [<a href="https://doi.org/10.1105/tpc.113.117523" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3877793/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24179125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Complexity%20of%20the%20alternative%20splicing%20landscape%20in%20plants&amp;author=A.%20S.%20N.%20Reddy&amp;author=Y.%20Marquez&amp;author=M.%20Kalyna&amp;author=A.%20Barta&amp;volume=25&amp;issue=10&amp;publication_year=2013&amp;pages=3657-3683&amp;pmid=24179125&amp;doi=10.1105/tpc.113.117523&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0056">
<cite>

Rich‐Griffin, C.
, 
Eichmann, R.
, 
Reitz, M. U.
, 
Hermann, S.
, 
Woolley‐Allen, K.
, 
Brown, P. E.
, 
Wiwatdirekkul, K.
, 
Esteban, E.
, 
Pasha, A.
, 
Kogel, K. H.
, 
Provart, N. J.
, 
Ott, S.
, &amp; 
Schäfer, P.
 (2020). Regulation of cell type‐specific immunity networks in Arabidopsis roots. The Plant Cell, 32(9), 2742–2762. 10.1105/tpc.20.00154

</cite> [<a href="https://doi.org/10.1105/tpc.20.00154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7474276/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32699170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Cell&amp;title=Regulation%20of%20cell%20type%E2%80%90specific%20immunity%20networks%20in%20Arabidopsis%20roots&amp;author=C.%20Rich%E2%80%90Griffin&amp;author=R.%20Eichmann&amp;author=M.%20U.%20Reitz&amp;author=S.%20Hermann&amp;author=K.%20Woolley%E2%80%90Allen&amp;volume=32&amp;issue=9&amp;publication_year=2020&amp;pages=2742-2762&amp;pmid=32699170&amp;doi=10.1105/tpc.20.00154&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0057">
<cite>

Roppolo, D.
, 
De Rybel, B.
, 
Tendon, V. D.
, 
Pfister, A.
, 
Alassimone, J.
, 
Vermeer, J. E. M.
, 
Yamazaki, M.
, 
Stierhof, Y. D.
, 
Beeckman, T.
, &amp; 
Geldner, N.
 (2011). A novel protein family mediates Casparian strip formation in the endodermis. Nature, 473(7347), 381–384. 10.1038/nature10070
</cite> [<a href="https://doi.org/10.1038/nature10070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21593871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20novel%20protein%20family%20mediates%20Casparian%20strip%20formation%20in%20the%20endodermis&amp;author=D.%20Roppolo&amp;author=B.%20De%20Rybel&amp;author=V.%20D.%20Tendon&amp;author=A.%20Pfister&amp;author=J.%20Alassimone&amp;volume=473&amp;issue=7347&amp;publication_year=2011&amp;pages=381-384&amp;pmid=21593871&amp;doi=10.1038/nature10070&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0058">
<cite>

Sanchez Carrillo, I. B.
, 
Hoffmann, P. C.
, 
Barff, T.
, 
Beck, M.
, &amp; 
Germain, H.
 (2023). Preparing <em>Arabidopsis thaliana</em> root protoplasts for cryo electron tomography. Frontiers in Plant Science, 14, 1261180. 10.3389/fpls.2023.1261180

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1261180" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10556516/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37810374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Preparing%20Arabidopsis%20thaliana%20root%20protoplasts%20for%20cryo%20electron%20tomography&amp;author=I.%20B.%20Sanchez%20Carrillo&amp;author=P.%20C.%20Hoffmann&amp;author=T.%20Barff&amp;author=M.%20Beck&amp;author=H.%20Germain&amp;volume=14&amp;publication_year=2023&amp;pages=1261180&amp;pmid=37810374&amp;doi=10.3389/fpls.2023.1261180&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0059">
<cite>

Schürholz, A.‐K.
, 
Lopez‐Salmeron, V.
, 
Li, Z.
, 
Forner, J.
, 
Wenzl, C.
, 
Gaillochet, C.
, 
Augustin, S.
, 
Vilches‐Barro, A.
, 
Fuchs, M.
, 
Gebert, M.
, 
Lohmann, J. U.
, 
Greb, T.
, &amp; 
Wolf, S.
 (2018). A comprehensive toolkit for inducible, cell type‐specific gene expression in Arabidopsis. Plant Physiology, 178(September), 40–53. 10.1104/pp.18.00463

</cite> [<a href="https://doi.org/10.1104/pp.18.00463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6130011/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30026289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=A%20comprehensive%20toolkit%20for%20inducible,%20cell%20type%E2%80%90specific%20gene%20expression%20in%20Arabidopsis&amp;author=A.%E2%80%90K.%20Sch%C3%BCrholz&amp;author=V.%20Lopez%E2%80%90Salmeron&amp;author=Z.%20Li&amp;author=J.%20Forner&amp;author=C.%20Wenzl&amp;volume=178&amp;issue=September&amp;publication_year=2018&amp;pages=40-53&amp;pmid=30026289&amp;doi=10.1104/pp.18.00463&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0060">
<cite>

Sessions, A.
, 
Weigel, D.
, &amp; 
Yanofsky, M. F.
 (1999). The Arabidopsis thaliana MERISTEM LAYER 1 promoter specifies epidermal expression in meristems and young primordia. Plant Journal, 20(2), 259–263. 10.1046/j.1365-313X.1999.00594.x
</cite> [<a href="https://doi.org/10.1046/j.1365-313X.1999.00594.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10571886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Journal&amp;title=The%20Arabidopsis%20thaliana%20MERISTEM%20LAYER%201%20promoter%20specifies%20epidermal%20expression%20in%20meristems%20and%20young%20primordia&amp;author=A.%20Sessions&amp;author=D.%20Weigel&amp;author=M.%20F.%20Yanofsky&amp;volume=20&amp;issue=2&amp;publication_year=1999&amp;pages=259-263&amp;pmid=10571886&amp;doi=10.1046/j.1365-313X.1999.00594.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0061">
<cite>

Sherman, B. T.
, 
Hao, M.
, 
Qiu, J.
, 
Jiao, X.
, 
Baseler, M. W.
, 
Lane, H. C.
, 
Imamichi, T.
, &amp; 
Chang, W.
 (2022). DAVID: A web server for functional enrichment analysis and functional annotation of gene lists (2021 update). Nucleic Acids Research, 50(1), 216–221. 10.1093/nar/gkac194
</cite> [<a href="https://doi.org/10.1093/nar/gkac194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9252805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35325185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=DAVID:%20A%20web%20server%20for%20functional%20enrichment%20analysis%20and%20functional%20annotation%20of%20gene%20lists%20(2021%20update)&amp;author=B.%20T.%20Sherman&amp;author=M.%20Hao&amp;author=J.%20Qiu&amp;author=X.%20Jiao&amp;author=M.%20W.%20Baseler&amp;volume=50&amp;issue=1&amp;publication_year=2022&amp;pages=216-221&amp;pmid=35325185&amp;doi=10.1093/nar/gkac194&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0062">
<cite>

Smakowska, E.
, 
Kong, J.
, 
Busch, W.
, &amp; 
Belkhadir, Y.
 (2016). Organ‐specific regulation of growth‐defense tradeoffs by plants. Current Opinion in Plant Biology, 29, 129–137. 10.1016/j.pbi.2015.12.005

</cite> [<a href="https://doi.org/10.1016/j.pbi.2015.12.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26802804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Plant%20Biology&amp;title=Organ%E2%80%90specific%20regulation%20of%20growth%E2%80%90defense%20tradeoffs%20by%20plants&amp;author=E.%20Smakowska&amp;author=J.%20Kong&amp;author=W.%20Busch&amp;author=Y.%20Belkhadir&amp;volume=29&amp;publication_year=2016&amp;pages=129-137&amp;pmid=26802804&amp;doi=10.1016/j.pbi.2015.12.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0063">
<cite>

Staehelin, L. A.
, 
Zheng, H. Q.
, 
Yoder, T. L.
, 
Smith, J. D.
, &amp; 
Todd, P.
 (2000). Columella cells revisited: Novel structures, novel properties, and a novel Gravisensing Model. Gravitational and Space Biology Bulletin, 13(2), 95–100.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/11543286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gravitational%20and%20Space%20Biology%20Bulletin&amp;title=Columella%20cells%20revisited:%20Novel%20structures,%20novel%20properties,%20and%20a%20novel%20Gravisensing%20Model&amp;author=L.%20A.%20Staehelin&amp;author=H.%20Q.%20Zheng&amp;author=T.%20L.%20Yoder&amp;author=J.%20D.%20Smith&amp;author=P.%20Todd&amp;volume=13&amp;issue=2&amp;publication_year=2000&amp;pages=95-100&amp;pmid=11543286&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0064">
<cite>

Tang, B.
, 
Feng, L.
, 
Hulin, M. T.
, 
Ding, P.
, &amp; 
Ma, W.
 (2023). Resource cell‐type‐specific responses to fungal infection in plants revealed by single‐cell transcriptomics. Cell Host and Microbe, 31(10), 1732–1747.e5. 10.1016/j.chom.2023.08.019

</cite> [<a href="https://doi.org/10.1016/j.chom.2023.08.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37741284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20and%20Microbe&amp;title=Resource%20cell%E2%80%90type%E2%80%90specific%20responses%20to%20fungal%20infection%20in%20plants%20revealed%20by%20single%E2%80%90cell%20transcriptomics&amp;author=B.%20Tang&amp;author=L.%20Feng&amp;author=M.%20T.%20Hulin&amp;author=P.%20Ding&amp;author=W.%20Ma&amp;volume=31&amp;issue=10&amp;publication_year=2023&amp;pages=1732-1747.e5&amp;pmid=37741284&amp;doi=10.1016/j.chom.2023.08.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0065">
<cite>

Tapial, J.
, 
Ha, K. C. H.
, 
Sterne‐Weiler, T.
, 
Gohr, A.
, 
Braunschweig, U.
, 
Hermoso‐Pulido, A.
, 
Quesnel‐Vallières, M.
, 
Permanyer, J.
, 
Sodaei, R.
, 
Marquez, Y.
, 
Cozzuto, L.
, 
Wang, X.
, 
Gómez‐Velázquez, M.
, 
Rayon, T.
, 
Manzanares, M.
, 
Ponomarenko, J.
, 
Blencowe, B. J.
, &amp; 
Irimia, M.
 (2017). An atlas of alternative splicing profiles and functional associations reveals new regulatory programs and genes that simultaneously express multiple major isoforms. Genome Research, 27(10), 1759–1768. 10.1101/gr.220962.117

</cite> [<a href="https://doi.org/10.1101/gr.220962.117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5630039/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28855263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=An%20atlas%20of%20alternative%20splicing%20profiles%20and%20functional%20associations%20reveals%20new%20regulatory%20programs%20and%20genes%20that%20simultaneously%20express%20multiple%20major%20isoforms&amp;author=J.%20Tapial&amp;author=K.%20C.%20H.%20Ha&amp;author=T.%20Sterne%E2%80%90Weiler&amp;author=A.%20Gohr&amp;author=U.%20Braunschweig&amp;volume=27&amp;issue=10&amp;publication_year=2017&amp;pages=1759-1768&amp;pmid=28855263&amp;doi=10.1101/gr.220962.117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0066">
<cite>

Thoma, S.
, 
Hecht, U.
, 
Kippers, A.
, 
Botella, J.
, 
De Vries, S.
, &amp; 
Somerville, C.
 (1994). Tissue‐specific expression of a gene encoding a cell wall‐localized lipid transfer protein from Arabidopsis. Plant Physiology, 105(1), 35–45. 10.1104/pp.105.1.35

</cite> [<a href="https://doi.org/10.1104/pp.105.1.35" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC159326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8029357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Tissue%E2%80%90specific%20expression%20of%20a%20gene%20encoding%20a%20cell%20wall%E2%80%90localized%20lipid%20transfer%20protein%20from%20Arabidopsis&amp;author=S.%20Thoma&amp;author=U.%20Hecht&amp;author=A.%20Kippers&amp;author=J.%20Botella&amp;author=S.%20De%20Vries&amp;volume=105&amp;issue=1&amp;publication_year=1994&amp;pages=35-45&amp;pmid=8029357&amp;doi=10.1104/pp.105.1.35&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0067">
<cite>

Tian, C.
, 
Wang, Y.
, 
Yu, H.
, 
He, J.
, 
Wang, J.
, 
Shi, B.
, 
Du, Q.
, 
Provart, N. J.
, 
Meyerowitz, E. M.
, &amp; 
Jiao, Y.
 (2019). A gene expression map of shoot domains reveals regulatory mechanisms. Nature Communications, 10(1), Article 141. 10.1038/s41467-018-08083-z
</cite> [<a href="https://doi.org/10.1038/s41467-018-08083-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6329838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30635575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=A%20gene%20expression%20map%20of%20shoot%20domains%20reveals%20regulatory%20mechanisms&amp;author=C.%20Tian&amp;author=Y.%20Wang&amp;author=H.%20Yu&amp;author=J.%20He&amp;author=J.%20Wang&amp;volume=10&amp;issue=1&amp;publication_year=2019&amp;pages=141&amp;pmid=30635575&amp;doi=10.1038/s41467-018-08083-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0068">
<cite>

Wallner, E.‐S.
, 
López‐Salmerón, V.
, 
Belevich, I.
, 
Poschet, G.
, 
Jung, I.
, 
Grünwald, K.
, 
Sevilem, I.
, 
Jokitalo, E.
, 
Hell, R.
, 
Helariutta, Y.
, 
Agustí, J.
, 
Lebovka, I.
, &amp; 
Greb, T.
 (2017). Strigolactone‐ and karrikin‐independent SMXL proteins are central regulators of phloem formation. Current Biology, 27(8), 1241–1247. 10.1016/j.cub.2017.03.014

</cite> [<a href="https://doi.org/10.1016/j.cub.2017.03.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5405109/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28392107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Biology&amp;title=Strigolactone%E2%80%90%20and%20karrikin%E2%80%90independent%20SMXL%20proteins%20are%20central%20regulators%20of%20phloem%20formation&amp;author=E.%E2%80%90S.%20Wallner&amp;author=V.%20L%C3%B3pez%E2%80%90Salmer%C3%B3n&amp;author=I.%20Belevich&amp;author=G.%20Poschet&amp;author=I.%20Jung&amp;volume=27&amp;issue=8&amp;publication_year=2017&amp;pages=1241-1247&amp;pmid=28392107&amp;doi=10.1016/j.cub.2017.03.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0069">
<cite>

Wang, E. T.
, 
Sandberg, R.
, 
Luo, S.
, 
Khrebtukova, I.
, 
Zhang, L.
, 
Mayr, C.
, 
Kingsmore, S. F.
, 
Schroth, G. P.
, &amp; 
Burge, C. B.
 (2008). Alternative isoform regulation in human tissue transcriptomes. Nature, 456(7221), 470–476. 10.1038/nature07509.Alternative

</cite> [<a href="https://doi.org/10.1038/nature07509.Alternative" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2593745/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18978772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Alternative%20isoform%20regulation%20in%20human%20tissue%20transcriptomes&amp;author=E.%20T.%20Wang&amp;author=R.%20Sandberg&amp;author=S.%20Luo&amp;author=I.%20Khrebtukova&amp;author=L.%20Zhang&amp;volume=456&amp;issue=7221&amp;publication_year=2008&amp;pages=470-476&amp;pmid=18978772&amp;doi=10.1038/nature07509.Alternative&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0082">
<cite>

Wang, J.
, 
Kucukoglu, M.
, 
Zhang, L.
, 
Chen, P.
, 
Decker, D.
, 
Nilsson, O.
, 
Jones, B.
, 
Sandberg, G.
, &amp; 
Zheng, B.
 (2013). The Arabidopsis LRR‐RLK, PXC1, is a regulator of secondary wall formation correlated with the TDIF‐PXY/TDR‐WOX4 signaling pathway. BMC Plant Biology, 13(1). 10.1186/1471-2229-13-94
</cite> [<a href="https://doi.org/10.1186/1471-2229-13-94" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3716795/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23815750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=The%20Arabidopsis%20LRR%E2%80%90RLK,%20PXC1,%20is%20a%20regulator%20of%20secondary%20wall%20formation%20correlated%20with%20the%20TDIF%E2%80%90PXY/TDR%E2%80%90WOX4%20signaling%20pathway&amp;author=J.%20Wang&amp;author=M.%20Kucukoglu&amp;author=L.%20Zhang&amp;author=P.%20Chen&amp;author=D.%20Decker&amp;volume=13&amp;issue=1&amp;publication_year=2013&amp;pmid=23815750&amp;doi=10.1186/1471-2229-13-94&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0070">
<cite>

Wyrsch, I.
, 
Domínguez‐Ferreras, A.
, 
Geldner, N.
, &amp; 
Boller, T.
 (2015). Tissue‐specific FLAGELLIN‐SENSING 2 (FLS2) expression in roots restores immune responses in Arabidopsis <em>fls2</em> mutants. New Phytologist, 206(2), 774–784. 10.1111/nph.13280

</cite> [<a href="https://doi.org/10.1111/nph.13280" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25627577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Tissue%E2%80%90specific%20FLAGELLIN%E2%80%90SENSING%202%20(FLS2)%20expression%20in%20roots%20restores%20immune%20responses%20in%20Arabidopsis%20fls2%20mutants&amp;author=I.%20Wyrsch&amp;author=A.%20Dom%C3%ADnguez%E2%80%90Ferreras&amp;author=N.%20Geldner&amp;author=T.%20Boller&amp;volume=206&amp;issue=2&amp;publication_year=2015&amp;pages=774-784&amp;pmid=25627577&amp;doi=10.1111/nph.13280&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0071">
<cite>

Xie, J. Q.
, 
Zhou, X.
, 
Jia, Z. C.
, 
Su, C. F.
, 
Zhang, Y.
, 
Fernie, A. R.
, 
Zhang, J.
, 
Du, Z. Y.
, &amp; 
Chen, M. X.
 (2023). Alternative splicing, an overlooked defense frontier of plants with respect to bacterial infection. Journal of Agricultural and Food Chemistry, 71, 16883–16901. 10.1021/acs.jafc.3c04163
</cite> [<a href="https://doi.org/10.1021/acs.jafc.3c04163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37916838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Agricultural%20and%20Food%20Chemistry&amp;title=Alternative%20splicing,%20an%20overlooked%20defense%20frontier%20of%20plants%20with%20respect%20to%20bacterial%20infection&amp;author=J.%20Q.%20Xie&amp;author=X.%20Zhou&amp;author=Z.%20C.%20Jia&amp;author=C.%20F.%20Su&amp;author=Y.%20Zhang&amp;volume=71&amp;publication_year=2023&amp;pages=16883-16901&amp;pmid=37916838&amp;doi=10.1021/acs.jafc.3c04163&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0072">
<cite>

Xue, Z.
, 
Liu, L.
, &amp; 
Zhang, C.
 (2020). Regulation of shoot apical meristem and axillary meristem development in plants. International Journal of Molecular Sciences, 21, 2917. 10.3390/ijms21082917

</cite> [<a href="https://doi.org/10.3390/ijms21082917" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7216077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32326368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Regulation%20of%20shoot%20apical%20meristem%20and%20axillary%20meristem%20development%20in%20plants&amp;author=Z.%20Xue&amp;author=L.%20Liu&amp;author=C.%20Zhang&amp;volume=21&amp;publication_year=2020&amp;pages=2917&amp;pmid=32326368&amp;doi=10.3390/ijms21082917&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0073">
<cite>

Yang, W.
, 
Wightman, R.
, &amp; 
Meyerowitz, E. M.
 (2017). Cell cycle control by nuclear sequestration of CDC20 and CDH1 mRNA in plant stem cells. Molecular Cell, 68(6), 1108–1119.e3. 10.1016/j.molcel.2017.11.008

</cite> [<a href="https://doi.org/10.1016/j.molcel.2017.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6013263/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29225038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=Cell%20cycle%20control%20by%20nuclear%20sequestration%20of%20CDC20%20and%20CDH1%20mRNA%20in%20plant%20stem%20cells&amp;author=W.%20Yang&amp;author=R.%20Wightman&amp;author=E.%20M.%20Meyerowitz&amp;volume=68&amp;issue=6&amp;publication_year=2017&amp;pages=1108-1119.e3&amp;pmid=29225038&amp;doi=10.1016/j.molcel.2017.11.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0074">
<cite>

Zhang, R.
, 
Kuo, R.
, 
Coulter, M.
, 
Calixto, C. P. G.
, 
Entizne, J. C.
, 
Guo, W.
, 
Marquez, Y.
, 
Milne, L.
, 
Riegler, S.
, 
Matsui, A.
, 
Tanaka, M.
, 
Harvey, S.
, 
Gao, Y.
, 
Wießner‐Kroh, T.
, 
Paniagua, A.
, 
Crespi, M.
, 
Denby, K.
, 
Hur, A. B.
, 
Huq, E.
, … 
Brown, J. W. S.
 (2022). A high‐resolution single‐molecule sequencing‐based Arabidopsis transcriptome using novel methods of Iso‐seq analysis. Genome Biology, 23(1), Article 149. 10.1186/s13059-022-02711-0

</cite> [<a href="https://doi.org/10.1186/s13059-022-02711-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9264592/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35799267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=A%20high%E2%80%90resolution%20single%E2%80%90molecule%20sequencing%E2%80%90based%20Arabidopsis%20transcriptome%20using%20novel%20methods%20of%20Iso%E2%80%90seq%20analysis&amp;author=R.%20Zhang&amp;author=R.%20Kuo&amp;author=M.%20Coulter&amp;author=C.%20P.%20G.%20Calixto&amp;author=J.%20C.%20Entizne&amp;volume=23&amp;issue=1&amp;publication_year=2022&amp;pages=149&amp;pmid=35799267&amp;doi=10.1186/s13059-022-02711-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0075">
<cite>

Zhang, X. C.
, &amp; 
Gassmann, W.
 (2003). RPS4‐Mediated disease resistance requires the combined presence of RPS4 transcripts with full‐length and truncated open reading frames. Plant Cell, 15(10), 2333–2342. 10.1105/tpc.013474

</cite> [<a href="https://doi.org/10.1105/tpc.013474" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC197299/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14523247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=RPS4%E2%80%90Mediated%20disease%20resistance%20requires%20the%20combined%20presence%20of%20RPS4%20transcripts%20with%20full%E2%80%90length%20and%20truncated%20open%20reading%20frames&amp;author=X.%20C.%20Zhang&amp;author=W.%20Gassmann&amp;volume=15&amp;issue=10&amp;publication_year=2003&amp;pages=2333-2342&amp;pmid=14523247&amp;doi=10.1105/tpc.013474&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270022-bib-0076">
<cite>

Zhu, J.
, 
Liu, M.
, 
Liu, X.
, &amp; 
Dong, Z.
 (2018). RNA polymerase II activity revealed by GRO‐seq and pNET‐seq in Arabidopsis. Nature Plants, 4(12), 1112–1123. 10.1038/s41477-018-0280-0

</cite> [<a href="https://doi.org/10.1038/s41477-018-0280-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30374093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Plants&amp;title=RNA%20polymerase%20II%20activity%20revealed%20by%20GRO%E2%80%90seq%20and%20pNET%E2%80%90seq%20in%20Arabidopsis&amp;author=J.%20Zhu&amp;author=M.%20Liu&amp;author=X.%20Liu&amp;author=Z.%20Dong&amp;volume=4&amp;issue=12&amp;publication_year=2018&amp;pages=1112-1123&amp;pmid=30374093&amp;doi=10.1038/s41477-018-0280-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Table S1: Samples and replicates sequencing information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s005.xlsx</a><sup> (16.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>Table S2. List of defense‐related genes used to identify the immunity‐DEGs and DSEs</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s003.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s003.xlsx</a><sup> (139.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><p>Table S3. Cell type‐specific genes and ASE</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s008.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s008.xlsx</a><sup> (399.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p"><p>Table S4. Expression of housekeeping genes in p<em>ATHB8</em>
</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s002.xlsx</a><sup> (9.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p"><p>Table S5. Cell type‐specific immunity related genes and ASE</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s007.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s007.xlsx</a><sup> (67.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p"><p>Table S6. Immunity‐related splicing variant of interest</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s006.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s006.xlsx</a><sup> (11.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p"><p>Table S7. Immunity‐related DSE that do not vary at the gene expression level</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s001.xlsx</a><sup> (13.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="caption p"><p>Supplementary Figures: Fig. S1 to S8</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11953613/bin/TPG2-18-e70022-s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70022-s004.pdf</a><sup> (1.3MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Raw sequencing data were submitted to Genbank under Bioproject accession number PRJNA1097566.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70022"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70022.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11953613/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11953613/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11953613%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11953613/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11953613/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11953613/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40156198/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11953613/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40156198/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11953613/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11953613/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="bWTPlkBE9fbwGqPk0BYhYXYhh6545b9sskZMzP4os13VSATG2aM30J8A5bsxiqb9">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
