
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            A high-resolution, nanopore-based artificial intelligence assay for DNA replication stress in human cancer cells - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4B91C8AF3F82305B91C0027A44ED2.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365011/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="A high-resolution, nanopore-based artificial intelligence assay for DNA replication stress in human cancer cells">
<meta name="citation_author" content="Mathew J K Jones">
<meta name="citation_author_institution" content="Frazer Institute, Faculty of Health, Medicine, and Behavioural Sciences, University of Queensland, Brisbane, QLD Australia">
<meta name="citation_author_institution" content="School of Chemistry &amp; Molecular Biosciences, University of Queensland, Brisbane, QLD Australia">
<meta name="citation_author" content="Subash Kumar Rai">
<meta name="citation_author_institution" content="Frazer Institute, Faculty of Health, Medicine, and Behavioural Sciences, University of Queensland, Brisbane, QLD Australia">
<meta name="citation_author" content="Pauline L Pfuderer">
<meta name="citation_author_institution" content="Department of Pathology, University of Cambridge, Cambridge, UK">
<meta name="citation_author_institution" content="Cancer Research UK Cambridge Centre, Li Ka Shing Centre, Robinson Way, Cambridge, UK">
<meta name="citation_author" content="Alexis Bonfim-Melo">
<meta name="citation_author_institution" content="Frazer Institute, Faculty of Health, Medicine, and Behavioural Sciences, University of Queensland, Brisbane, QLD Australia">
<meta name="citation_author" content="Julia K Pagan">
<meta name="citation_author_institution" content="School of Biomedical Sciences, The University of Queensland, Saint Lucia, QLD Australia">
<meta name="citation_author" content="Paul R Clarke">
<meta name="citation_author_institution" content="Frazer Institute, Faculty of Health, Medicine, and Behavioural Sciences, University of Queensland, Brisbane, QLD Australia">
<meta name="citation_author_institution" content="Institute for Biomedicine and Glycomics, Griffith University, Southport, QLD Australia">
<meta name="citation_author" content="Francis Isidore Garcia Totañes">
<meta name="citation_author_institution" content="Wellcome Sanger Institute, Cambridge, UK">
<meta name="citation_author" content="Catherine J Merrick">
<meta name="citation_author_institution" content="Department of Pathology, University of Cambridge, Cambridge, UK">
<meta name="citation_author" content="Sarah E McClelland">
<meta name="citation_author_institution" content="Barts Cancer Institute, Queen Mary University of London, London, EC1M 6BQ UK">
<meta name="citation_author" content="Michael A Boemo">
<meta name="citation_author_institution" content="Department of Pathology, University of Cambridge, Cambridge, UK">
<meta name="citation_author_institution" content="Department of Genetics, University of Cambridge, Cambridge, CB2 3EH UK">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7732">
<meta name="citation_doi" content="10.1038/s41467-025-63168-w">
<meta name="citation_pmid" content="40830365">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365011/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365011/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365011/pdf/41467_2025_Article_63168.pdf">
<meta name="description" content="DNA replication stress is a hallmark of cancer that is exploited by chemotherapies. Current assays for replication stress have low throughput and poor resolution whilst being unable to map the movement of replication forks genome-wide. We present a ...">
<meta name="og:title" content="A high-resolution, nanopore-based artificial intelligence assay for DNA replication stress in human cancer cells">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="DNA replication stress is a hallmark of cancer that is exploited by chemotherapies. Current assays for replication stress have low throughput and poor resolution whilst being unable to map the movement of replication forks genome-wide. We present a ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365011/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365011">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63168-w"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63168.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365011%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365011/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365011/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365011/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 19;16:7732. doi: <a href="https://doi.org/10.1038/s41467-025-63168-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63168-w</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>A high-resolution, nanopore-based artificial intelligence assay for DNA replication stress in human cancer cells</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jones%20MJK%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Mathew J K Jones</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mathew J K Jones</span></h3>
<div class="p">
<sup>1</sup>Frazer Institute, Faculty of Health, Medicine, and Behavioural Sciences, University of Queensland, Brisbane, QLD Australia </div>
<div class="p">
<sup>2</sup>School of Chemistry &amp; Molecular Biosciences, University of Queensland, Brisbane, QLD Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jones%20MJK%22%5BAuthor%5D" class="usa-link"><span class="name western">Mathew J K Jones</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rai%20SK%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Subash Kumar Rai</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Subash Kumar Rai</span></h3>
<div class="p">
<sup>1</sup>Frazer Institute, Faculty of Health, Medicine, and Behavioural Sciences, University of Queensland, Brisbane, QLD Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rai%20SK%22%5BAuthor%5D" class="usa-link"><span class="name western">Subash Kumar Rai</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pfuderer%20PL%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Pauline L Pfuderer</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Pauline L Pfuderer</span></h3>
<div class="p">
<sup>3</sup>Department of Pathology, University of Cambridge, Cambridge, UK </div>
<div class="p">
<sup>4</sup>Cancer Research UK Cambridge Centre, Li Ka Shing Centre, Robinson Way, Cambridge, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pfuderer%20PL%22%5BAuthor%5D" class="usa-link"><span class="name western">Pauline L Pfuderer</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bonfim-Melo%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Alexis Bonfim-Melo</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Alexis Bonfim-Melo</span></h3>
<div class="p">
<sup>1</sup>Frazer Institute, Faculty of Health, Medicine, and Behavioural Sciences, University of Queensland, Brisbane, QLD Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bonfim-Melo%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexis Bonfim-Melo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pagan%20JK%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Julia K Pagan</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Julia K Pagan</span></h3>
<div class="p">
<sup>5</sup>School of Biomedical Sciences, The University of Queensland, Saint Lucia, QLD Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pagan%20JK%22%5BAuthor%5D" class="usa-link"><span class="name western">Julia K Pagan</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Clarke%20PR%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Paul R Clarke</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Paul R Clarke</span></h3>
<div class="p">
<sup>1</sup>Frazer Institute, Faculty of Health, Medicine, and Behavioural Sciences, University of Queensland, Brisbane, QLD Australia </div>
<div class="p">
<sup>6</sup>Institute for Biomedicine and Glycomics, Griffith University, Southport, QLD Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Clarke%20PR%22%5BAuthor%5D" class="usa-link"><span class="name western">Paul R Clarke</span></a>
</div>
</div>
<sup>1,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tota%C3%B1es%20FIG%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Francis Isidore Garcia Totañes</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Francis Isidore Garcia Totañes</span></h3>
<div class="p">
<sup>7</sup>Wellcome Sanger Institute, Cambridge, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tota%C3%B1es%20FIG%22%5BAuthor%5D" class="usa-link"><span class="name western">Francis Isidore Garcia Totañes</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Merrick%20CJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Catherine J Merrick</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Catherine J Merrick</span></h3>
<div class="p">
<sup>3</sup>Department of Pathology, University of Cambridge, Cambridge, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Merrick%20CJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Catherine J Merrick</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McClelland%20SE%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Sarah E McClelland</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Sarah E McClelland</span></h3>
<div class="p">
<sup>8</sup>Barts Cancer Institute, Queen Mary University of London, London, EC1M 6BQ UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McClelland%20SE%22%5BAuthor%5D" class="usa-link"><span class="name western">Sarah E McClelland</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Boemo%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Michael A Boemo</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Michael A Boemo</span></h3>
<div class="p">
<sup>3</sup>Department of Pathology, University of Cambridge, Cambridge, UK </div>
<div class="p">
<sup>9</sup>Department of Genetics, University of Cambridge, Cambridge, CB2 3EH UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Boemo%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael A Boemo</span></a>
</div>
</div>
<sup>3,</sup><sup>9,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Frazer Institute, Faculty of Health, Medicine, and Behavioural Sciences, University of Queensland, Brisbane, QLD Australia </div>
<div id="Aff2">
<sup>2</sup>School of Chemistry &amp; Molecular Biosciences, University of Queensland, Brisbane, QLD Australia </div>
<div id="Aff3">
<sup>3</sup>Department of Pathology, University of Cambridge, Cambridge, UK </div>
<div id="Aff4">
<sup>4</sup>Cancer Research UK Cambridge Centre, Li Ka Shing Centre, Robinson Way, Cambridge, UK </div>
<div id="Aff5">
<sup>5</sup>School of Biomedical Sciences, The University of Queensland, Saint Lucia, QLD Australia </div>
<div id="Aff6">
<sup>6</sup>Institute for Biomedicine and Glycomics, Griffith University, Southport, QLD Australia </div>
<div id="Aff7">
<sup>7</sup>Wellcome Sanger Institute, Cambridge, UK </div>
<div id="Aff8">
<sup>8</sup>Barts Cancer Institute, Queen Mary University of London, London, EC1M 6BQ UK </div>
<div id="Aff9">
<sup>9</sup>Department of Genetics, University of Cambridge, Cambridge, CB2 3EH UK </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2022 Nov 22; Accepted 2025 Aug 8; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365011  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830365/" class="usa-link">40830365</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">DNA replication stress is a hallmark of cancer that is exploited by chemotherapies. Current assays for replication stress have low throughput and poor resolution whilst being unable to map the movement of replication forks genome-wide. We present a new method that uses nanopore sequencing and artificial intelligence to map forks and measure their rates of movement and stalling in melanoma and colon cancer cells treated with chemotherapies. Our method can differentiate between fork slowing and fork stalling in cells treated with hydroxyurea, as well as inhibitors of ATR, WEE1, and PARP1. These different therapies yield different characteristic signatures of replication stress. We assess the role of the intra-S-phase checkpoint on fork slowing and stalling and show that replication stress dynamically changes over S-phase. Finally, we demonstrate that this method is applicable and consistent across two different flow cell chemistries (R9.4.1 and R10.4.1) from Oxford Nanopore Technologies. This method requires sequencing on only one nanopore flow cell per sample, and the cost-effectiveness enables functional screens to determine how human cancers respond to replication-targeted therapies.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Cancer genomics, Machine learning</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Determining how replication forks move across the human genome is critical for the effective use of agents that target replication stress. Here, the authors present DNAscent, an AI supported assay for DNA replication stress in human cells using Nanopore sequencing data.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">DNA replication stress is characterised by frequent slowing, stalling, and collapse of replication forks which causes unreplicated regions of the genome, DNA damage, mutations, and chromosomal instabilities that drive tumourigenesis<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Many important chemotherapeutic agents target the replication stress response (RSR) pathway<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup> including inhibitors of ATR<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>, PARP1<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, WEE1<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>, and dNTP synthesis with hydroxyurea (HU)<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> or 5-fluorouracil<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, as well as DNA crosslinkers like cisplatin<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. How these agents affect the movement and fidelity of replication forks across the genome remains unknown due to the low throughput and/or insufficient resolution of current assays. The location of DNA breaks that result from replication fork stalls can be mapped with techniques such as BLESS<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>, Break-seq<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, and TrAEL-seq<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> but these methods require the fork to stall such that it results in a break and no information about origin firing or fork movement prior to the break is captured. DNA fibre analysis is a single-molecule method that can measure fork velocity, origin density, and the frequency of fork stalling<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, but the throughput and spatial resolution are both low and fibres cannot be mapped to the genome without resorting to fibre-fish<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> which is not scalable to the whole genome. Recently, optical replication mapping (ORM) provided a high-throughput, single-molecule approach to map origin firing genome-wide<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, but the 15-kb resolution is too low to detect fork stalling. Long-read sequencing using the Oxford Nanopore Technologies (ONT) platform has enabled the detection of replication origins and fork movement in budding yeast<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>–<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> as well as the malaria parasites <em>Plasmodium falciparum</em><sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> and <em>Plasmodium knowlesi</em><sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. It has also been used to measure the replication rates of human mitochondrial DNA<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> as well as replication origins in the human genome<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Here, we introduce a method that measures fork speed and stalling on single molecules with up to single-nucleotide resolution and use it to show that chemotherapies create different “replication stress signatures”. Our method can clearly distinguish between replication fork slowing and stalling, and by mapping these molecules to the genome, we assess the role of the intra-S-phase checkpoint and show that replication stress dynamically changes over S-phase.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">DNAscent detects replication forks on single molecules from human cancer cells</h3>
<p id="Par4">Our DNAscent software can detect two thymidine analogues, BrdU and EdU, on single nanopore-sequenced molecules<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. When these base analogues are sequentially pulsed into S-phase cells, they are incorporated into the nascent strand by replication forks, leaving a “footprint” of fork movement. Therefore, we pulsed A2058 human melanoma cells and human RPE1 cells expressing PIP-FUCCI<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> with EdU, then BrdU, and followed by a thymidine chase. We then enriched for S-phase cells by fluorescence-activated cell sorting, extracted ultra-high-molecular weight DNA, and sequenced it on the Oxford Nanopore MinION platform (Fig. <a href="#Fig1" class="usa-link">1a, b</a>; Figure <a href="#MOESM1" class="usa-link">S1</a>). As shown previously in <em>S. cerevisiae</em><sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, fork stalling manifests as a sharp drop-off in analogue incorporation into nascent DNA (Fig. <a href="#Fig1" class="usa-link">1c</a>) and we observed stalled forks in A2058 melanoma cells (Fig. <a href="#Fig1" class="usa-link">1d</a>). The length (in base pairs) of the base analogue tracks, divided by the total pulse length, yielded a measure of fork speed that was reproducible between biological replicates with fork speeds of ~1.4 kb/min in RPE1 cells that are consistent with studies measuring fork speeds in RPE1 cells via DNA fibres<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> (Fig. <a href="#Fig1" class="usa-link">1e</a>). We performed DNA fibre analysis for A2058 cells where we observed a difference of approximately 0.3–0.5 kb/min between fork speeds measured by DNAscent and fork speeds measured by DNA fibre (Figure <a href="#MOESM1" class="usa-link">S2</a>). This difference could be due to variation in fibre stretching or differences in the resolution and sensitivity of analogue detection<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Our optimised ultra-long sequencing protocol yielded over 180,000 total reads with a mapping length greater than 20 kilobases (kb), an N50 read length of ~90 kb, and 4100 called forks from a single MinION flow cell (averages: 150,000 reads longer than 20 kb, 90 kb N50, 2050 fork calls; Table <a href="#MOESM1" class="usa-link">S1</a>) with a false positive rate of fork calls less than 0.004% (Table <a href="#MOESM1" class="usa-link">S2</a>). This read length and throughput allowed us to capture complex replication dynamics across the entire human genome with multiple forks, origins, and termination sites on the same molecule (Fig. <a href="#Fig1" class="usa-link">1f</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Replication dynamics on ultra-long nanopore reads from human cancer cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365011_41467_2025_63168_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a1a8/12365011/aed101066a24/41467_2025_63168_Fig1_HTML.jpg" loading="lazy" id="d33e516" height="750" width="798" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Cells expressing PIP-FUCCI were sequentially labelled with a 5-minute EdU pulse, a 10 min BrdU pulse, and a thymidine chase. The asynchronous population of cells was enriched for S-phase cells using FACS (mCherry+ mVenus-), and high molecular weight (HMW) DNA was extracted and sequenced on the Oxford Nanopore MinION platform. <strong>b</strong> PIP-FUCCI gates on high levels of geminin (Gem) and low levels of the PCNA-interacting protein (PIP) degron enabling S-phase enrichment without the cellular stress caused by arresting the cell cycle. <strong>c</strong> Diagram showing that fork stalling manifests as a sudden drop in BrdU incorporation into the nascent strand. <strong>d</strong> Two nanopore-sequenced molecules from A2058 melanoma cells analysed with DNAscent that show the patterns in (<strong>c</strong>). Tracks show the probability of BrdU (red) and EdU (blue) at each thymidine position along the molecule. <strong>e</strong> Distribution of fork speed measured by DNAscent in untreated RPE1 cells (upper) and A2058 cells (lower) for two biological replicates. Vertical red line median, IQR interquartile range, N number of fork calls. <strong>f</strong> Three ultra-long single molecules, each represented as a group of two tracks of bar graphs. The top track shows the DNAscent-called probability of BrdU (red) and the bottom track shows the DNAscent-called probability of EdU (blue). Each read is annotated with arrows to show fork direction. Reads were moved onto the same axis from different regions of the genome. Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">DNAscent distinguishes between fork stalling and slowing under chemotherapies</h3>
<p id="Par5">We treated A2058 cells with either a PARP inhibitor (Olaparib), WEE1 inhibitor (MK1775), ATR inhibitor (VE-821), or hydroxyurea (HU) (Fig. <a href="#Fig2" class="usa-link">2a</a>). These treatments had a marked effect on replication fork dynamics that was clearly visible on single molecules (Fig. <a href="#Fig2" class="usa-link">2b</a>). As expected, HU<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, ATR inhibition<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>, and WEE1 inhibition<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> all slowed forks (Fig. <a href="#Fig2" class="usa-link">2c</a>). To test the method further and provide an internal control, we added HU with the BrdU pulse and verified shortening of the BrdU track but not the EdU track (Figure <a href="#MOESM1" class="usa-link">S3</a>). The dose and treatment time of Olaparib mirrored that of Maya-Mendoza and colleagues<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> and, consistent with their findings using DNA fibre analysis, we observed a 30% increase in fork speed. Fork speeds from all conditions were consistent across biological replicates (Figure <a href="#MOESM1" class="usa-link">S4</a>) and, for each agent, the treatment effect exceeded expected variation between replicates (Figure <a href="#MOESM1" class="usa-link">S5</a>). To create a quantitative measure of fork stalling, we defined a “stall score” to measure how abruptly BrdU incorporation ends. Stall score is a measure between 0 and 1 of the proportional decrease in the frequency of positive BrdU calls at the BrdU end of the replication fork track. Forks assigned a stall score near 0 are unlikely to be stalled and forks assigned a stall score near 1 are likely to be stalls or pauses (Fig. <a href="#Fig3" class="usa-link">3a</a>). Stall score provides an additional layer of information about replication mobility that has the potential to distinguish between fork stalling and slowing: compared to untreated cells, HU and WEE1 inhibition resulted in fork slowing, and PARP inhibition resulted in a fork speed increase (see Fig. <a href="#Fig2" class="usa-link">2c</a>), but all three of these chemotherapies resulted in a similar distribution of stall scores to that of untreated cells (Fig. <a href="#Fig3" class="usa-link">3b</a>). ATR inhibition did not slow forks as much as HU, but treatment with ATR inhibitors resulted in a marked increase in stall score indicative of frequent fork stalling. While HU is generally thought to rapidly stall replication forks, multiple DNA fibre analysis studies have shown slow but continued fork progression during short and prolonged HU treatment<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Our approach builds on these DNA fibre methods by discriminating between fork slowing in cells treated with HU and rapid stalling in cells treated with ATRi. As before, stall scores from all conditions were consistent across biological replicates (Figure <a href="#MOESM1" class="usa-link">S6</a>). Notably, only ATRi showed strong evidence of a treatment effect beyond the expected variation between replicates (Figure <a href="#MOESM1" class="usa-link">S7</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. DNAscent measures disruptions to fork speed caused by inhibiting the replication stress response pathway.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365011_41467_2025_63168_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a1a8/12365011/f64103679b5c/41467_2025_63168_Fig2_HTML.jpg" loading="lazy" id="d33e598" height="750" width="798" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> A2058 cells were given a monotherapy of either HU, ATR inhibitor, WEE1 inhibitor, or PARP inhibitor at the specified timepoints relative to the BrdU and EdU pulses. <strong>b</strong> Five single molecules analysed by DNAscent, each showing a rightward-moving fork from untreated cells as well as from each of the monotherapies. Forks were from different regions of the genome, moved onto the same axis so that the start of the EdU tracks align. Each fork is represented as group of three tracks showing (from top to bottom) the probability of BrdU at that position on the read, the probability of EdU at that position on the read, and DNAscent’s segmentation of the read into EdU- and BrdU-positive regions. Fork length was measured by computing the genomic distance, in kilobases, between far ends of the EdU and BrdU segments (shown for the top fork) and dividing this distance by the 15 min total pulse duration. <strong>c</strong> Distribution of fork speeds for untreated cells and cells under each monotherapy. Forks near the end of reads, as well as forks together at origins or terminations, were excluded to avoid inaccuracies in fork speed (see Methods). Vertical red line median, IQR interquartile range, N number of fork calls. Treatment effect, as measured by a hierarchical Bayesian model (see Methods), is reported in Figure <a href="#MOESM1" class="usa-link">S5</a>. Source data are provided as a Source Data file.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Replication stress signatures stratify cells by treatment.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365011_41467_2025_63168_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a1a8/12365011/eb1ec88c6c83/41467_2025_63168_Fig3_HTML.jpg" loading="lazy" id="d33e635" height="646" width="798" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Each fork is assigned a raw stall score by computing the decrease in frequency of positive BrdU calls (A,B: frequencies) near the end of the fork. A nonlinear scaling is applied to create a final stall score. <strong>b</strong> Distribution of stall scores for untreated cells, as well as for each monotherapy. Vertical red line median, IQR interquartile range, N number of fork calls. Treatment effect is reported in Figure <a href="#MOESM1" class="usa-link">S7</a>. <strong>c</strong> Measurements of the speed, analogue incorporation, and stall score of each fork were used to create an 8-dimensional replication stress signature, shown embedded into two dimensions using UMAP. Left: Each point is the signature for a single fork, points are coloured according to the monotherapy delivered, and biological replicates are shown as different markers. The centroids from K-means clustering (K = 5) are shown as red boxed crosses. Right: Kernel density estimation to show distinct clusters, coloured from blue (low density of points) to red (high density of points). Clustering performance is shown in Figure <a href="#MOESM1" class="usa-link">S8</a>. To account for the treatment protocol in Fig. <a href="#Fig2" class="usa-link">2a</a> where HU was added with the BrdU pulse, Figure <a href="#MOESM1" class="usa-link">S9</a> repeats this analysis using the length of the BrdU track as the only metric of fork speed. <strong>d</strong> Repeat of (<strong>c</strong>) that excludes any measurement of fork speed in the replication stress signature. Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Chemotherapies create distinct signatures of replication stress</h3>
<p id="Par6">DNA replication stress is an umbrella term that refers to frequent fork slowing and/or stalling. DNAscent can measure each of these attributes, and to unify them into an overall measure of replication stress, we represented each fork as an 8-dimensional point consisting of features pertaining to fork speed, level of analogue incorporation, and stall score (see Methods for details) and reduced points on this 8-dimensional manifold to two dimensions using Uniform Manifold Approximation and Projection (UMAP)<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Forks measured from cells treated with each chemotherapy clustered together, showing that disrupting different elements of the DNA replication stress response pathway creates a replication stress signature in this 8-dimensional space (Fig. <a href="#Fig3" class="usa-link">3c</a>). To demonstrate that our new method captures fundamentally more meaningful information about DNA replication stress than existing methods that just measure fork speed, we repeated the procedure in a 5-dimensional space that excluded any measurement related to fork speed (the track lengths of EdU, BrdU, and the overall fork; see Methods). We found that stall score and the level of base analogue incorporation alone were sufficient to create distinct stress signatures of disruption to different elements of the RSR pathway (Fig. <a href="#Fig3" class="usa-link">3d</a>). While we anticipate variation due to the timing and dosage of treatment, these results show that different chemotherapies can create a characteristic replication stress signature based on the pathway that they target.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Mapping stressed forks to the genome reveals checkpoint-dependent changes over S-phase</h3>
<p id="Par7">To investigate whether fork stress changes across S-phase, we applied our method in HCT116 colon cancer cells to leverage existing high-resolution Repli-Seq replication timing data<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. We assessed both wild-type HCT116 cells and HCT116 cells with a <em>CDK2</em><sup>AF/AF</sup> mutation that prevents intra-S-phase checkpoint activation by WEE1<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Our measured fork speed in wild-type cells was consistent with DNA fibre analysis<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. While we observed a higher variability between repeats than in A2058 (wild-type median fork speed of 1.7 k/min with stall score median 0.3; <em>CDK2</em><sup>AF/AF</sup> median fork speed of 1.26 kb/min with stall score median 0.39; Figure <a href="#MOESM1" class="usa-link">S10</a>) the <em>CDK2</em><sup>AF/AF</sup> cells showed a slower fork speed and higher stall score than the wild-type within each repeat (Fig. <a href="#Fig4" class="usa-link">4a, b</a>). Moreover, there was moderate support for a treatment effect of the <em>CDK2</em><sup>AF/AF</sup> mutation (Figure <a href="#MOESM1" class="usa-link">S11</a>). This was consistent with the effect of WEE1 inhibition on fork speed and stall score in A2058 cells, as WEE1 inhibition functionally mirrors a <em>CDK2</em><sup>AF/AF</sup> mutation (Fig. <a href="#Fig2" class="usa-link">2c</a>; Fig. <a href="#Fig3" class="usa-link">3b</a>). We calculated the median replication time in S-phase (Trep) for the genomic position of each called fork using high-resolution Repli-Seq replication timing data for HCT116 cells<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> in order to show how the change in fork speed and stall score over S-phase can be measured from a single sequencing run. In wild-type cells, fork speed increased (in agreement with single-cell studies<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>) and stall score decreased in genomic positions that tended to replicate later in S-phase (Fig. <a href="#Fig4" class="usa-link">4c, d</a>). This relationship vanished in the <em>CDK2</em><sup>AF/AF</sup> mutant, as these cells maintained a constant slow fork speed and high stall score across regions of the genome that replicated both early and late in wild-type cells. These trends in stall score and fork speed over S-phase for both wild-type and <em>CDK2</em><sup>AF/AF</sup> HCT116 cells were consistent across biological replicates (Figure <a href="#MOESM1" class="usa-link">S10</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Replication stress dynamics change over S-phase and are checkpoint-dependent.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365011_41467_2025_63168_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a1a8/12365011/a8d63a39cde6/41467_2025_63168_Fig4_HTML.jpg" loading="lazy" id="d33e779" height="798" width="799" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Left column shows results for HCT116 wild-type cells and right column shows results for the HCT116 <em>CDK2</em><sup>AF/AF</sup> mutant. <strong>a</strong> Distribution of fork speeds and stall scores for HCT116 wild-type cells. <strong>b</strong> Distribution of fork speeds and stall scores for the HCT116 <em>CDK2</em><sup>AF/AF</sup> mutant. Vertical red line: median; IQR interquartile range, N number of fork calls. The equivalent analysis for a biological replicate is shown in Figure <a href="#MOESM1" class="usa-link">S10</a> and the effects of the mutation on both fork speed and stall score is reported in Figure <a href="#MOESM1" class="usa-link">S11</a>. <strong>c</strong> Distribution of fork speeds and (<strong>d</strong>) stall scores of forks grouped by the median replication time (Trep) of their genomic position. Fork speeds and stall scores mapping to each hour in S-phase are shown as strip plots, violin plots, and box plots with the horizontal black line indicating the median and boxes showing the IQR. Whisker length is 1.5⋅IQR. The Pearson correlation coefficient (R), slope, and p-value for each panel was computed via linear least-squares regression on the median at each time point. For fork speed, the alternative hypothesis was that the slope of the regression line was greater than zero; for stall score, the alternative hypothesis was that the slope was less than zero. The correlation was significant for both fork speed (<em>p</em> = 0.00018) and stall score (<em>p</em> = 0.0020) for wild-type cells and not significant for fork speed (<em>p</em> = 0.98) or stall score (<em>p</em> = 0.69) for the <em>CDK2</em><sup>AF/AF</sup> mutant. Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">A transition to R10.4.1 nanopore chemistry</h3>
<p id="Par8">To future-proof our method and show that our results hold across different generations of Oxford Nanopore flow cell chemistries, we developed and trained DNAscent to work on Oxford Nanopore flow cells using the latest R10.4.1 flow cells. Building upon the strategies we developed to train DNAscent v2<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> and DNAscent v3<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, we developed a new version of DNAscent (v4) for the R10.4.1 chemistry. The neural network in DNAscent v4 was trained using BrdU and EdU incorporated into human RPE1 cells (Figure <a href="#MOESM1" class="usa-link">S12</a>; Table <a href="#MOESM1" class="usa-link">S3</a> for model architecture). We observed cleaner fork tracks with DNAscent v4, particularly a very low false positive rate following the thymidine chase (Fig. <a href="#Fig5" class="usa-link">5a</a>). DNAscent v4 was benchmarked on human RPE1 reads as well as reads from the malaria parasite <em>Plasmodium falciparum</em> to stress test the model’s performance on an outlier 81% AT-rich genome. For both human cells and <em>P. falciparum</em>, DNAscent v4 showed considerably higher positive BrdU and EdU calls for a much lower false positive rate (Figure <a href="#MOESM1" class="usa-link">S13</a>; Table <a href="#MOESM1" class="usa-link">S4</a>). To ensure that our observations of fork speed and stall scores were consistent across R9.4.1 and R10.4.1, we used DNAscent v4 to measure fork speed in human RPE1 cells and found that fork speed on R10.4.1 (Fig. <a href="#Fig5" class="usa-link">5b</a>) was consistent with fork speed on R9.4.1 as measured with DNAscent v3 (Fig. <a href="#Fig1" class="usa-link">1e</a>). As expected, fork stalling was low. On treatment with ATRi, we again observed a decrease in fork speed and an increase in stall score when using R10.4.1 and DNAscent v4. Taken together, these data show that we can consistently estimate replication fork speed and stalling from Oxford Nanopore data across different nanopore chemistries.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. DNAscent v4 on R10.4.1 chemistry.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365011_41467_2025_63168_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a1a8/12365011/2cecac72249e/41467_2025_63168_Fig5_HTML.jpg" loading="lazy" id="d33e843" height="756" width="790" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Two representative origin calls from RPE1 cells. As in Figs. <a href="#Fig1" class="usa-link">1</a>–<a href="#Fig3" class="usa-link">3</a>, a 5-minute EdU pulse was followed by a 10-minute BrdU pulse and tracks show the probability of BrdU (red) and EdU (blue) at each thymidine position along the molecule. The upper molecule was sequenced using the R10.4.1 chemistry with BrdU and EdU probabilities annotated with DNAscent v4 while the lower molecule was sequenced using the R9.4.1 chemistry with analogue probabilities annotated with DNAscent v3.1.2. <strong>b</strong> Distribution of replication fork speeds and stall scores for untreated RPE1 cells and (<strong>c</strong>) RPE1 cells treated with the ATR inhibitor VE-821. Both samples were sequenced on the R10.4.1 chemistry. Vertical red line median, IQR interquartile range, N number of fork calls. Source data are provided as a Source Data file.</p></figcaption></figure></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par9">We have developed a new method for the high-throughput, high-resolution measurement of DNA replication stress across the human genome; demonstrated that an inexpensive handheld device produces enough data to create replication stress signatures that can accurately stratify cells according to the part of the replication stress response pathway that was inhibited; and used a single sequencing run to show how replication fork speed changes across replication timing domains. The result is a major step change over the previous “gold standard” of DNA fibre analysis for measuring replication stress in cancer cells: Our method categorically supersedes DNA fibre in throughput, resolution, cost-effectiveness, and automation, all while enabling us to map stressed forks to the genome which DNA fibre analysis is fundamentally unable to do.</p>
<p id="Par10">In addition, our method distinguishes between slow-moving and stalled forks. This distinction is critical for characterising the effect of chemotherapies on replication forks, as we have shown that HU causes the former while ATR inhibition causes the latter. We have demonstrated that the ability to map replication forks to genomic positions makes it possible to leverage pre-existing high resolution Repli-Seq data to investigate how replication timing impacts fork movement. This can naturally be extended to datasets measuring other genomic features such as epigenetic regulation and the chromatin landscape.</p>
<p id="Par11">The sensitivity of our approach is highlighted by its ability to identify increased fork stress and reduced fork speed in HCT116 <em>CDK2</em><sup>AF/AF</sup> cells that proliferate with a viable disruption of the intra-S-phase checkpoint<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Our findings provide a quantitative measure of the increased replication stress previously described in these cells and demonstrates how restricting CDK2 activity is important for suppressing fork stress and ensuring replication fork speeds rise throughout S-phase. Our study has not accounted for any potential changes in replication timing in the HCT116 <em>CDK2</em><sup>AF/AF</sup> and could have been improved by generating equivalent high resolution Repli-Seq timing reference for this cell line.</p>
<p id="Par12">With future development, we anticipate further improvements to this method’s throughput, utility, and scope. The 15 min analogue pulse represents 3% of an 8 h human cell S-phase, hence many of molecules sequenced do not include a fork track. A pulldown of EdU prior to sequencing would enrich the percentage of reads with a fork track and would also negate the need for S-phase enrichment with reporter systems. We have also observed a slight bias towards measuring leading strand synthesis, which may be due to 3’ overhangs on lagging strands reducing capture by transposase during ultra-high molecular weight library preparation. In addition, we envisage the future detection of 5-methylcytosine and 5-hydroxymethylcytosine alongside BrdU and EdU to show replication fork movement, origin firing, and epigenetic markers on the same molecule. This study focused on three cell lines and four treatments, but the method is usable for any human cell line and is extensible to tumour organoids and suspension cell cultures. While we have used the detection of BrdU and EdU for the purposes of studying DNA replication, the accurate detection of base analogues in single molecules carries much broader utility, with an example of future work being the incorporation of base analogues during nucleotide excision repair or long-patch base excision repair.</p>
<p id="Par13">Replication stress induced by inhibiting different parts of the RSR pathway is not created equally: the impact on replication forks will be highly dependent on the chemotherapy, dose, and timing which will become even more complex when treatments are given in combination. Our replication stress signature framework can scale with this complexity by incorporating further genomic features into the signature. This includes features from the sequenced single molecule such as mismatches and epigenetic markers, as well as population-level features such as chromatin accessibility, replication timing, and gene expression.</p>
<p id="Par14">Our approach to developing the DNAscent software has been guided by DNA fibre assays to measure fork dynamics. We have tuned the fork detection algorithm to call the start and end of replication tracks at the peaks of EdU and BrdU detection because this is the strategy that aligns closest to DNA fibre analysis. We do not expect DNAscent to report fork speeds that are consistent with DNA fibre assays in all experimental conditions, as they are fundamentally different analogue detection strategies. For example, one of our engineering considerations was that DNAscent would find it challenging to assess fork speed and stall score in regions of weak or variable analogue incorporation. Therefore, while the increased resolution and sensitivity of analogue detection with DNAscent identifies small stretches of low-level analogue incorporation, only regions of high analogue incorporation were used to calculate fork speed (Fig. <a href="#Fig2" class="usa-link">2b</a>). Stall scores are calculated in a manner that is unique to the nanopore sequencing platform and differences in stall scores cannot be easily validated with another approach. While the biological explanation for the large increase in the fork stall score observed with ATRi is not completely understood, we speculate that it could be due to increased stalling or pausing of forks during ATR inhibition but cannot rule out that ATR dependent regulation of nucleotide synthesis is also a contributing factor to this phenotype<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>–<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>.</p>
<p id="Par15">When engineering DNAscent, our design choices were made to prioritise the accuracy of replication fork speed and stall score. This necessarily required trade-offs in other areas, particularly on the location of the boundary between the EdU and BrdU regions of a fork track. We often observe noise at the EdU-to-BrdU transition whereby EdU incorporation is elevated within the BrdU region, as well as ambiguity over where this boundary should be placed. These two phenomena are visible, respectively, in the representative ATRi and WEE1i fork tracks of Fig. <a href="#Fig2" class="usa-link">2b</a>. Our design choices were made to minimise risk of the major segmentation error that would occur if, for example, a short segment of elevated EdU incorporation within a BrdU region was mistakenly called as an origin rather than ignored as noise. By making the calling of EdU and BrdU regions conservative to avoid these errors, DNAscent tends to leave a gap between the EdU and BrdU regions of a fork track. This can sometimes result in a ratio of BrdU-to-EdU track lengths that do not match the expected 2:1 ratio given the 5 min EdU pulse and 10 min BrdU pulse. Therefore, while we have added HU with the BrdU pulse in this study to serve as an internal control to verify shortening of the BrdU track relative to the EdU track (Figure <a href="#MOESM1" class="usa-link">S3</a>), we would not generally recommend this experimental setup to test the effectiveness of an inhibitor on replication fork dynamics. We instead suggest adding the inhibitor with or before the first analogue pulse.</p>
<p id="Par16">In summary, our method is cost-effective, automated, and provides fundamentally more layers of information to differentiate between stressed forks; all of these attributes are necessary to support high-throughput screens of how cancer subtypes respond to replication-based therapies.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Tissue culture, lentiviral transduction, and S-phase enrichment via FACS</h3>
<p id="Par17">A2058 cells were grown in DMEM with 10% FBS and 1% penicillin–streptomycin. HCT116 cells were grown in McCoy’s 5a modified media with 10% FBS and 1% penicillin–streptomycin. pLenti-CMV-Blast-PIP-FUCCI was a gift from Jean Cook (Addgene plasmid # 138715). RRID: Addgene_138715. The PIP-FUCCI lentiviral transfer plasmid was cotransfected with psPAX2 pMD2.G into 293 T cells. 48 h later, supernatants were filtered, mixed 1:1 with fresh medium containing polybrene (8 μg/mL), and applied to target cells for 16–24 h. A2058 and HCT116 cells expressing PIP-FUCCI were labelled with 50 μM EdU for 5 min, washed 3x in PBS (taking about 1 min for each) and labelled with 50 μM BrdU for 10 mins, washed 3x PBS and chased with 100 μM thymidine for 20 min then trypsinized in 0.05% trypsin. S-phase cells were isolated by FACS sorting (mCherry<sup>+</sup>, mVenus<sup>−</sup>) using a BD FACS Aria Fusion or MoFlo Astrios EQ. Sorted cells were pelleted and stored at −80c. Drug treatments: HU (2 mM) added during BrdU pulse, PARPi Olaparib (10 µM) for 24 h, Wee1i MK1775 (1 µM) 2 h and ATRi VE-821 (10 µM) 30 min.</p></section><section id="Sec11"><h3 class="pmc_sec_title">General purpose S-phase enrichment</h3>
<p id="Par18">The PIP-FUCCI system was used in this study only to increase the number of fork calls per MinION flow cell; it does not otherwise affect the results. Other cell synchronisation techniques could also be used to enrich S-phase cell, such as mimosine arrest or CDK4/6 inhibition and release<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Cell synchronisation techniques may impact fork stall scores depending on the end user’s application.</p></section><section id="Sec12"><h3 class="pmc_sec_title">DNA preparation</h3>
<p id="Par19">Ultra-high molecular weight (UHMW) DNA was extracted using Nanobind CBB Big DNA Kit (SKU NB-900-001-01, Circulomics) and UHMW DNA Aux Kit (NB-900-101-01, Circulomics) according to the manufacturer’s protocol (Document ID: EXT-CLU-001). DNA was incubated at 37 °C for 1 h at the elution step. The DNA was eluted in 760 µL Buffer EB (6 million cells for 6x MinION library sequencing) or in 506 µL (4–5.5 million cells for 4x MinION library sequencing). To measure the recovery and the quality of viscus UHMW DNA in Qubit and NanoDrop, the DNA sample was prepared according to the method described by Koetsier and Cantor<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. DNA purity was checked by the ratio of DNA concentration measured in NanoDrop and Qubit (1-1.5).</p></section><section id="Sec13"><h3 class="pmc_sec_title">Library preparation and nanopore sequencing</h3>
<p id="Par20">libraries were prepared using Ultra-Long DNA Sequencing Kit (SQK-ULK001, ONT) and Nanobind UL Library Prep Kit (NB-900-601-01, Circulomics) according to the NanoBind Library Prep - Ultra Long Sequencing protocol (Document ID: LBP-ULN-001; 03/24/2021, Circulomics). UHMW DNA library was eluted in 225 µL (6 million cells) or 150 µL (4–5.5 million cells) ONT Elution Buffer (EB; SQK-ULK001, ONT). The loading library was prepared according to the manufacturer’s protocol and was loaded onto MinION Flow Cell R9.4.1(FLO-MIN106D, ONT). Sequencing run was performed using MinKNOW (v21.11.7–22.08.4) for 96 h run script with the sequencing kit set to SQK-ULK001. To load the fresh loading library, the sequencing run was paused after 24 h and the flow cell was washed with Flow Cell Wash Kit (EXP-WSH004, ONT) according to the manufacturer’s guidelines.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Basecalling and Genome Alignment</h3>
<p id="Par21">On R9.4.1, Oxford Nanopore sequencing reads were basecalled using Guppy (v5.0.11) using the dna_r9.4.1_450bps_fast configuration. Basecalled reads were aligned to the human genome using minimap2 (v2.17-r941) using the map-ont setting. Reads from A2058 cells were aligned to the chm13v2.0 human reference genome from the Telomere-to-Telomere (T2T) Consortium<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Reads from HCT116 were aligned to the hg38 (GRCh38.p13) assembly for consistency with the high-resolution Repli-Seq data from Zhao and colleagues<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. On R10.4.1, Oxford Nanopore sequencing reads were basecalled with Dorado v0.7.2 using the dna_r10.4.1_e8.2_400bps_fast@v5.0.0model. Genome alignments were done at the time of basecalling using Dorado, and reads were aligned to the T2T chm13v2.0 human reference genome.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Fork calling with DNAscent</h3>
<p id="Par22">On the R9.4.1 chemistry, Oxford Nanopore reads were analysed with DNAscent v3.1.2 which is available under GPL-3.0 at <a href="https://github.com/MBoemo/DNAscent" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/MBoemo/DNAscent</a>. Each basecalled sequencing run was indexed with the DNAscent index subprogram, and the probability of BrdU and EdU at each nucleotide along each sequenced read was called using the DNAscent detect subprogram. DNAscent detect was run using a minimum mapping quality of 20 and a minimum mapping length of 20 kb. The DNAscent forkSense subprogram segmented each read into EdU- and BrdU-positive regions. To finalise the location of the start and end of the EdU track, DNAscent computes the average incorporation in the middle third of the track and trims back from the start of the track and the end of the track until it meets a part of the track with this average incorporation. The trimming then stops and this is given as the final track. DNAscent then does the same for the BrdU track. EdU and BrdU tracks are then matched into replication forks such that, for a rightward-moving fork, the replication fork track begins at the start of the EdU track and ends at the end of the BrdU track which is where BrdU incorporation starts to decrease at the start of the thymidine chase. Fork calls on each read were then matched into origins and termination sites. On the R10.4.1 chemistry, base analogue detection was done on DNAscent v4.0.3 with the same minimum thresholds for read length and mapping quality.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Measurement of fork speed with DNAscent</h3>
<p id="Par23">DNAscent forkSense outputs replication fork calls as bed files indicating the region of the genome each called fork moved through during the 15 min EdU-BrdU pulse. The length of each region (in kb) was divided by 15 min to compute the speed of each fork in kb per minute. These fork tracks will be shorter if (i.) the forks are from an origin that fired during the pulse, (ii.) the forks come together in a termination site during the pulse, or (iii.) the fork track runs off the end of the sequenced molecule. To present an accurate picture of fork speed, fork tracks were excluded from the analysis if they were on a read with a called origin or termination site, or if they started or ended within 3 kb of the end of the read.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Fork stall calling with DNAscent</h3>
<p id="Par24">The number of positive BrdU calls (probability &gt; 0.5 called by DNAscent detect) was calculated over a 2 kb window before (inside the BrdU-positive segment) the BrdU end of fork track and divided by the number of thymidine bases in the 2 kb window resulting in a fraction between 0 and 1 (denoted <em>B</em>). This fraction of positive BrdU calls divided by attempts was also calculated over a 2 kb window immediately after (outside the BrdU-positive segment) the BrdU end of the fork (denoted <em>A</em>). A raw score was computed:</p>
<table class="disp-formula p" id="Equ1"><tr><td class="formula"><math id="d33e959" display="block"><mi>R</mi><mo>=</mo><mspace width="0.25em"></mspace><mfrac><mrow><mi>B</mi><mo>−</mo><mi>A</mi></mrow><mrow><mi>B</mi></mrow></mfrac></math></td></tr></table>
<p id="Par25">A nonlinear scaling was applied to the raw score <em>R</em> to compute a stall score between 0 and 1:</p>
<table class="disp-formula p" id="Equ2"><tr><td class="formula"><math id="d33e981" display="block"><mi mathvariant="normal">stall score</mi><mo>=</mo><mi>α</mi><mi>log</mi><mfenced close=")" open="("><mrow><mn>1</mn><mo>+</mo><msup><mrow><mi>e</mi></mrow><mrow><mi>β</mi><mfenced close=")" open="("><mrow><mi>R</mi><mo>−</mo><mn>1</mn></mrow></mfenced></mrow></msup></mrow></mfenced><mo>−</mo><mi>α</mi><mi>log</mi><mfenced close=")" open="("><mrow><mn>1</mn><mo>+</mo><msup><mrow><mi>e</mi></mrow><mrow><mo>−</mo><mi>β</mi></mrow></msup></mrow></mfenced></math></td></tr></table>
<p id="Par26">On DNAscent v3.1.2, α = 1.55 and β = 3. This scaling is plotted in Fig. <a href="#Fig3" class="usa-link">3a</a> and serves a dual role. First, it accounts for the fact that a fork which has not stalled during the analogue pulse will produce a relatively high raw score approximately between 0.2 and 0.4 and rescales this range closer to zero. Second, it creates more conservative estimates of raw scores in the 0.6-0.9 range so that only high-confidence fork pauses or stalls have stall scores near 1. Each fork call is annotated with a stall score in DNAscent v3.1.2. To avoid erroneous results, DNAscent declines to assign a stall score to forks where (i.) the fork track runs off the read, (ii.) forks that come together in a termination site during the pulse, and (iii.) forks where there is an insertion or deletion longer than 100 bp in the read inside one of the 2 kb windows. These forks were excluded from the analysis in Figs. <a href="#Fig3" class="usa-link">3</a>–<a href="#Fig4" class="usa-link">4</a>. Note that for DNAscent v4.0.3, the cleaner BrdU and EdU calls meant that we tuned α = 2.63 and β = 1 to produce consistent results across v3.1.2 and v4.0.3.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Replication stress signatures with DNAscent</h3>
<p id="Par27">The following eight features were measured for each fork: (i.) the total length of the fork track (in bp), (ii.) the length of the EdU track (in bp), (iii.) the length of the BrdU track (in bp), (iv.) fraction of thymidine positions called as BrdU in the EdU segment, (v.) fraction of thymidine positions called as EdU in the EdU segment, (vi.) fraction of thymidine positions called as EdU in the BrdU segment, (vii.) fraction of thymidine positions called as BrdU in the BrdU segment, (viii.) the stall score. DNAscent v3.1.2 outputs these features as a bed file for ease of use. Each of these eight features, across all forks, was rescaled to the interquartile range and reduced to two dimensions using Uniform Manifold Approximation and Projection (UMAP)<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> with 25 nearest neighbours, a minimum embedded point distance of 0, and the Chebyshev metric. A total of 125 forks from each sequencing run (a total 250 forks per monotherapy, accounting for two biological repeats) were used. Kernel density estimates were made using Seaborn (v0.11.2) with the default settings.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Mapping forks to Trep</h3>
<p id="Par28">Gaussian smoothed and scaled high-resolution Repli-Seq data for HCT116 cells was accessed from NCBI GEO (accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE137764" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE137764</a>) and Trep was calculated via sigmoid fitting as per the authors’ instructions<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> for each 50 kb bin across the human genome. The Trep for each DNAscent fork call was computed using the Trep of the 50 kb genomic region that the fork call mapped to.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Human training data for 10.4.1 chemistry</h3>
<p id="Par29">RPE1 cells were labelled with 50 µM BrdU for 72 hours or 50 µM EdU for 24 hours. Genomic DNA was extracted using NEB Monarch HMW DNA Extraction Kit for Cells &amp; Blood (T3050L). Sequencing library was prepared using the Native Barcoding Kit 24 V14 SQK-NBD114.24.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Human samples analysed with 10.4.1 chemistry</h3>
<p id="Par30">Human cells were labelled and sorted as above for R9. Genomic DNA was extracted using R9 method or NEB Monarch HMW DNA Extraction Kit for Cells &amp; Blood (T3050L). Sequencing library was prepared using the Ultra-Long DNA Sequencing Kit V14 (SKU: SQK-ULK114) and samples were run on a MinION Flow Cell (R10.4.1) (SKU: FLO-MIN114.001).</p></section><section id="Sec22"><h3 class="pmc_sec_title">Parasite test data for 10.4.1 chemistry</h3>
<p id="Par31"><em>P. falciparum</em> parasites were tightly synchronised via Percoll gradient, as in our previous study<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, then incubated at 36 h post-invasion in 20 µM EdU for 1 h or 200 µM BrdU for 1 h. Parasite DNA was harvested and sequenced as above.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Model training on R10.4.1 chemistry</h3>
<p id="Par32">Our training procedure generally follows that of DNAscent v2<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> and DNAscent v3<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. To create pore models for EdU- and BrdU-containing 9mers on R10.4.1 chemistry, we used the DNAscent (v4.0.3) align subprogram on the BrdU-labelled, EdU-labelled, and untreated datasets to create a signal alignment between each possible 9mer and the measured normalised current as that 9mer passes through the nanopore. This resulted in a Gaussian distribution of normalised current measurements for each possible thymidine-containing 9mer, similar to the publicly available pore model for unlabelled 9mers released by Oxford Nanopore (<a href="https://github.com/nanoporetech/kmer_models/blob/master/dna_r10.4.1_e8.2_400bps/9mer_levels_v1.txt" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/nanoporetech/kmer_models/blob/master/dna_r10.4.1_e8.2_400bps/9mer_levels_v1.txt</a>). Note that while previous work<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> used a two-component Gaussian mixture model to identify the labelled and unlabelled distributions, the analogue concentration used to prepare our training datasets was sufficiently high that we assumed the labelled distribution would be dominant. We therefore fit a Gaussian distribution to the normalised current for each 9mer to create a pore models for unlabelled DNA as well as for EdU-labelled and BrdU-labelled DNA on the R10.4.1 chemistry. We used these pore models to perform hidden Markov-based detection<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> to determine the log-likelihood ratio of BrdU and EdU at each thymidine position of each sequenced read. For BrdU-labelled and EdU-labelled reads, a call was considered positive if the log-likelihood ratio at a thymidine position was greater than 1.5. Reads were used for further training if more than 30% of thymidine positions were called as EdU or BrdU by the hidden Markov-based detection algorithm. Following the labelling procedure outlined in the supplemental information for our manuscript on DNAscent v2<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, we used the results of hidden Markov-based detection to label each thymidine position on the read (TP = 0.6; C = 0.65; TN = 0.9). The model architecture is detailed in Figure <a href="#MOESM1" class="usa-link">S12</a> and Table <a href="#MOESM1" class="usa-link">S3</a>. The training data was comprised of 50,000 2-kilobase read segments for each of the EdU-labelled, BrdU-labelled, and unlabelled conditions. After training the model for 10 epochs, we used the model to call the probability of BrdU and EdU at each thymidine position across every read in the original BrdU-labelled and EdU-labelled datasets. We set the probability threshold above which a BrdU or EdU call is considered positive at 0.5 and pulled out reads for further training if more than 30% of thymidine positions were called as EdU or BrdU. This model was used to label the reads (TP = 0.8; C = 0.8; TN = 0.95). We then followed the data augmentation procedure outlined in Figure <a href="#MOESM1" class="usa-link">S2</a> of our DNAscent v2 manuscript<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> with the number of training 2-kilobase segments for each augmentation condition specified in Table <a href="#MOESM1" class="usa-link">S5</a>. This resulted in a total of 230,000 training segments, and we trained the model architecture in Figure <a href="#MOESM1" class="usa-link">S12</a> on this data for a total of 15 epochs.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Model inputs for R10.4.1 chemistry</h3>
<p id="Par33">A key change between DNAscent v3 and DNAscent v4 is how inputs to the neural network are handled. As shown in Figure <a href="#MOESM1" class="usa-link">S12</a>, the signal input is a tensor of normalised current signals that were aligned to each base on the sequenced molecule using an in-built hidden Markov alignment. At each base position, these signals are encoded by a GRU stack. To create the base sequence input, we determined that the magnitude of the signal shift was primarily governed by the middle 5mer of each 9mer. Therefore, each 9mer was decomposed into a middle 5mer and a flanking 4mer. For example, in 9mer given by bases N<sub>1</sub>N<sub>2</sub>N<sub>3</sub>N<sub>4</sub>N<sub>5</sub>N<sub>6</sub>N<sub>7</sub>N<sub>8</sub>N<sub>9</sub>, the middle 5mer is N<sub>3</sub>N<sub>4</sub>N<sub>5</sub>N<sub>6</sub>N<sub>7</sub> and the flanking 4mer is N<sub>1</sub>N<sub>2</sub>N<sub>8</sub>N<sub>9</sub>. We then built embeddings for the middle 5mer and embeddings for the flanking 4mer. A tensor of the 5mer embeddings, a tensor of the 4mer embeddings, and a tensor of the GRU-encoded signal are concatenated and passed to the first convolutional layer.</p></section><section id="Sec25"><h3 class="pmc_sec_title">DNA fibre analysis</h3>
<p id="Par34">A2058 cells were labelled with IdU and CldU (50 μM) for 20 min each. DNA fibres were prepared as previously described in ref. <sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Cells were trypsinized and resuspended at 1 × 10<sup>6</sup> cells/ml in PBS. 2 μl of cell suspension was placed onto a glass slide and lysed in 10 μL of lysis buffer (200 mM Tris-HCl (pH 7.4), 0.5% SDS, 50 mM EDTA). After 6 min, the slides were tilted at 15° to allow the DNA to spread. Slides were air-dried for 30 min, fixed in methanol and acetic acid (3:1) for 2 min, and refrigerated overnight before immunolabeling. DNA was denatured with 4 M HCl for 20 min. Slides were incubated in blocking buffer (PBS  +  0.1% Triton X-100 +  10% goat serum) for 1 h. IdU and CldU were detected using rat anti-BrdU (Abcam ab6326, 1:200) and mouse anti-BrdU (BD 347580, 1:200). Secondary antibody staining was performed using Alexa Fluor 488-labeled goat anti-mouse IgG antibody (Invitrogen A-11029 1:200) and Alexa Fluor 568-labeled goat anti-rat antibody (Invitrogen A11036 1:200). Slides were mounted in Prolong Gold and imaged on a DeltaVision Ultra. Replication track lengths were measured using Image J.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Treatment effect</h3>
<p id="Par35">We employed a hierarchical Bayesian model to estimate the posterior distribution of the effect of agents/mutations on replication fork speed and stall score, as well as determine whether the effect of this treatment exceeded background variation between replicates. Fork data from each replicate was modelled using a partial pooling structure to determine within-group variability and between-group differences. For each group, replicate-level means were modelled as draws from a group-level distribution, allowing for uncertainty at both the replicate and condition levels. Replication fork speed was assumed to follow a normal distribution and weakly informative priors were placed on group means (µ=1.5, σ = 0.5 for untreated and µ = 0.7, σ = 0.5 for treated) with half-normal (σ = 0.5) priors on standard deviations. Stall score was assumed to follow a Beta distribution to reflect the score being bound on [0,1]. These distributions were parametrised via a group mean and shared precision. The prior on the group mean was Beta-distributed (α = 2, β = 5 for both treated and untreated) and the prior on precision was Gamma-distributed (α = 2, β = 0.1). Posterior inference was done using the No-U-Turn Sampler (NUTS) which drew 2000 posterior samples after 1000 tuning steps. For guidance, we defined a Region of Practical Equivalence (ROPE) as ±0.05 for stall score and ±0.1 kb/min for fork speed. We reported a treatment effect (Δ<sub>1</sub>, the difference between group mean of treated and the group mean of untreated), control replicate variability (Δ<sub>2</sub>, the absolute value of the difference between the untreated replicate means), treated replicate variability (the absolute value of the difference between the treated replicate means), and adjusted treatment effect (Δ<sub>1</sub> -Δ<sub>2</sub>). These are reported for fork speed and stall score in Figures <a href="#MOESM1" class="usa-link">S5</a>, <a href="#MOESM1" class="usa-link">S7</a>, and S<a href="#MOESM1" class="usa-link">11</a>. We considered an effect to be significant if at least 95% of the treatment effect posterior lies outside the ROPE.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par36">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec28"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365011/bin/41467_2025_63168_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365011/bin/41467_2025_63168_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (81.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365011/bin/41467_2025_63168_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (419.7KB, pdf) </sup>
</div></div></section></section><section id="Sec29"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12365011/bin/41467_2025_63168_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (637.4KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors would like to thank Brian Gabrielli (Mater Research Institute), Valentine Murigneux (Genome Innovation Hub, University of Queensland), and all the members of the Jones and Clarke laboratories (Frazer Institute) as well as the Boemo laboratory (Cambridge Pathology) for guidance and helpful conversations. We would like to acknowledge the Translational Research Institute (TRI) for providing an excellent research environment and core facilities that enabled this research. We particularly thank Lucie Leveque-El Mouttie, David Sester, Dalia Khalil, Yitian Ding and Andy Wu from the Flow Cytometry Core Facility for invaluable help and technical assistance. The Translational Research Institute is supported by a grant from the Australian and Queensland Governments. Research by MAB is supported by the Isaac Newton Trust (19.39B), the Royal Society (RGS\R1\201251), the Leverhulme Trust (RPG-2022-028), and the Medical Sciences Research Council (MR/W031442/1). This work was supported by the Cancer Research UK Cambridge Centre (C9685/A25117) via a Cancer Research UK Cambridge Centre PhD Studentship to PLP. MJKJ and PRC are supported by the Australian Research Council grant DP210102704 and the Genome Innovation Hub at the University of Queensland. MJKJ is also supported by funding from the University of Queensland FACULTY OF SCIENCE BIRRST PARTNER Scheme. F.I.G.T. was supported by the European Research Council under the European Union’s Horizon 2020 research and innovation programme (ERC-2016-COG25126 to C.J.M.)</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>M.J.K.J., S.E.M. and M.A.B. conceptualised and designed the study. M.A.B. and P.L.P. developed the software and did the bioinformatics analyses. S.K.R., M.J.K.J., and A.B.M. conducted the wet lab research and generated nanopore sequencing for human cell lines. FIGT generated the nanopore sequencing data for <em>P. falciparum</em>. J.K.P. provided reagents and intellectual contributions. M.J.K.J., M.A.B., S.E.M., C.J.M., and P.R.C. acquired funding for the project. M.A.B., M.J.K.J. and A.B.M. made the figures. M.J.K.J. and M.A.B. supervised the project and wrote the manuscript with input from all authors.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par37"><em>Nature Communications</em> thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The Nanopore sequencing data generated in this study have been deposited in NCBI’s Gene Expression Omnibus under the GEO Series accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE216926" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE216926</a> and the European Nucleotide Archive under accession number <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB80561" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJEB80561</a>. <a href="#Sec29" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>The DNAscent software is available at <a href="https://github.com/MBoemo/DNAscent" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/MBoemo/DNAscent</a> under the GPL-3.0 open-source license<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. DNAscent v3.1.2 was used for analysis on the R9.4.1 Oxford Nanopore chemistry while DNAscent v4.0.3 was used for analysis on the R10.4.1 chemistry. Both versions are available as Singularity images at <a href="https://cloud.sylabs.io/library/mboemo/dnascent/dnascent" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cloud.sylabs.io/library/mboemo/dnascent/dnascent</a>.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par38">P.L.P. has received small grants from Oxford Nanopore Technologies to cover travel and fees associated with presenting research at scientific meetings. The remaining authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Mathew J. K. Jones, Email: mathew.jones@uq.edu.au</p>
<p>Michael A. Boemo, Email: mb915@cam.ac.uk</p></section><section id="sec30"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63168-w.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Zeman, M. K. &amp; Cimprich, K. A. Causes and consequences of replication stress. <em>Nat. Cell Biol.</em><strong>16</strong>, 2–9 (2014).
</cite> [<a href="https://doi.org/10.1038/ncb2897" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4354890/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24366029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeman,%20M.%20K.%20&amp;%20Cimprich,%20K.%20A.%20Causes%20and%20consequences%20of%20replication%20stress.%20Nat.%20Cell%20Biol.16,%202%E2%80%939%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Gaillard, H., García-Muse, T. &amp; Aguilera, A. Replication stress and cancer. <em>Nat. Rev. Cancer</em><strong>15</strong>, 276–289 (2015).
</cite> [<a href="https://doi.org/10.1038/nrc3916" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25907220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gaillard,%20H.,%20Garc%C3%ADa-Muse,%20T.%20&amp;%20Aguilera,%20A.%20Replication%20stress%20and%20cancer.%20Nat.%20Rev.%20Cancer15,%20276%E2%80%93289%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Zhang, J., Dai, Q., Park, D. &amp; Deng, X. Targeting DNA replication stress for cancer therapy. <em>Genes (Basel)</em><strong>7</strong>, 51 (2016).
</cite> [<a href="https://doi.org/10.3390/genes7080051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4999839/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27548226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20J.,%20Dai,%20Q.,%20Park,%20D.%20&amp;%20Deng,%20X.%20Targeting%20DNA%20replication%20stress%20for%20cancer%20therapy.%20Genes%20(Basel)7,%2051%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Karnitz, L. M. &amp; Zou, L. Molecular pathways: Targeting ATR in cancer therapy. <em>Clin. Cancer Res.</em><strong>21</strong>, 4780–4785 (2015).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-15-0479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4631635/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26362996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karnitz,%20L.%20M.%20&amp;%20Zou,%20L.%20Molecular%20pathways:%20Targeting%20ATR%20in%20cancer%20therapy.%20Clin.%20Cancer%20Res.21,%204780%E2%80%934785%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Farmer, H. et al. Targeting the DNA repair defect in BRCA mutant cells as a therapeutic strategy. <em>Nature</em><strong>434</strong>, 917–921 (2005).
</cite> [<a href="https://doi.org/10.1038/nature03445" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15829967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Farmer,%20H.%20et%20al.%20Targeting%20the%20DNA%20repair%20defect%20in%20BRCA%20mutant%20cells%20as%20a%20therapeutic%20strategy.%20Nature434,%20917%E2%80%93921%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Matheson, C. J., Backos, D. S. &amp; Reigan, P. Targeting WEE1 kinase in cancer. <em>Trends Pharmacol. Sci.</em><strong>37</strong>, 872–881 (2016).
</cite> [<a href="https://doi.org/10.1016/j.tips.2016.06.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27427153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Matheson,%20C.%20J.,%20Backos,%20D.%20S.%20&amp;%20Reigan,%20P.%20Targeting%20WEE1%20kinase%20in%20cancer.%20Trends%20Pharmacol.%20Sci.37,%20872%E2%80%93881%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Young Charles, W. &amp; Hodas, S. Hydroxyurea: Inhibitory effect on DNA metabolism. <em>Science</em><strong>146</strong>, 1172–1174 (1964).
</cite> [<a href="https://doi.org/10.1126/science.146.3648.1172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14203679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Young%20Charles,%20W.%20&amp;%20Hodas,%20S.%20Hydroxyurea:%20Inhibitory%20effect%20on%20DNA%20metabolism.%20Science146,%201172%E2%80%931174%20(1964)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Musiałek, M. W. &amp; Rybaczek, D. Hydroxyurea—The good, the bad and the ugly. <em>Genes (Basel)</em><strong>12</strong>, 1096 (2021).</cite> [<a href="https://doi.org/10.3390/genes12071096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8304116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34356112/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Hagenkort, A. et al. dUTPase inhibition augments replication defects of 5-Fluorouracil. <em>Oncotarget</em><strong>8</strong>, 23713–23726 (2017).</cite> [<a href="https://doi.org/10.18632/oncotarget.15785" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5410339/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28423595/" class="usa-link">PubMed</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Kelland, L. The resurgence of platinum-based cancer chemotherapy. <em>Nat. Rev. Cancer</em><strong>7</strong>, 573–584 (2007).
</cite> [<a href="https://doi.org/10.1038/nrc2167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17625587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kelland,%20L.%20The%20resurgence%20of%20platinum-based%20cancer%20chemotherapy.%20Nat.%20Rev.%20Cancer7,%20573%E2%80%93584%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Crosetto, N. et al. Nucleotide-resolution DNA double-strand break mapping by next-generation sequencing. <em>Nat. Methods</em><strong>10</strong>, 361–365 (2013).
</cite> [<a href="https://doi.org/10.1038/nmeth.2408" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3651036/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23503052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crosetto,%20N.%20et%20al.%20Nucleotide-resolution%20DNA%20double-strand%20break%20mapping%20by%20next-generation%20sequencing.%20Nat.%20Methods10,%20361%E2%80%93365%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Hoffman, E., McCulley, A., Haarer, B., Arnak, R. &amp; Feng, W. Break-seq reveals hydroxyurea-induced chromosome fragility as a result of unscheduled conflict between DNA replication and transcription. <em>Genome Res.</em><strong>25</strong>, 402–412 (2015).</cite> [<a href="https://doi.org/10.1101/gr.180497.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4352882/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25609572/" class="usa-link">PubMed</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Kara, N., Krueger, F., Rugg-Gunn, P. &amp; Houseley, J. Genome-wide analysis of DNA replication and DNA double-strand breaks using TrAEL-seq. <em>PLOS Biol.</em><strong>19</strong>, e3000886 (2021).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.3000886" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8021198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33760805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kara,%20N.,%20Krueger,%20F.,%20Rugg-Gunn,%20P.%20&amp;%20Houseley,%20J.%20Genome-wide%20analysis%20of%20DNA%20replication%20and%20DNA%20double-strand%20breaks%20using%20TrAEL-seq.%20PLOS%20Biol.19,%20e3000886%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Bensimon, A. et al. Alignment and Sensitive Detection of DNA by a Moving Interface. <em>Science</em><strong>265</strong>, 2096–2098 (1994).
</cite> [<a href="https://doi.org/10.1126/science.7522347" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7522347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bensimon,%20A.%20et%20al.%20Alignment%20and%20Sensitive%20Detection%20of%20DNA%20by%20a%20Moving%20Interface.%20Science265,%202096%E2%80%932098%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Merrick, C. J., Jackson, D. &amp; Diffley, J. F. X. Visualization of altered replication dynamics after DNA damage in human cells. <em>J. Biol. Chem.</em><strong>279</strong>, 20067–20075 (2004).
</cite> [<a href="https://doi.org/10.1074/jbc.M400022200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14982920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Merrick,%20C.%20J.,%20Jackson,%20D.%20&amp;%20Diffley,%20J.%20F.%20X.%20Visualization%20of%20altered%20replication%20dynamics%20after%20DNA%20damage%20in%20human%20cells.%20J.%20Biol.%20Chem.279,%2020067%E2%80%9320075%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Parra, I. &amp; Windle, B. High resolution visual mapping of stretched DNA by fluorescent hybridization. <em>Nat. Genet.</em><strong>5</strong>, 17–21 (1993).
</cite> [<a href="https://doi.org/10.1038/ng0993-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8106079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Parra,%20I.%20&amp;%20Windle,%20B.%20High%20resolution%20visual%20mapping%20of%20stretched%20DNA%20by%20fluorescent%20hybridization.%20Nat.%20Genet.5,%2017%E2%80%9321%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Wang, W. et al. Genome-wide mapping of human DNA replication by optical replication mapping supports a stochastic model of eukaryotic replication. <em>Mol. Cell</em><strong>81</strong>, 2975–2988.e2976 (2021).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2021.05.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8286344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34157308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20W.%20et%20al.%20Genome-wide%20mapping%20of%20human%20DNA%20replication%20by%20optical%20replication%20mapping%20supports%20a%20stochastic%20model%20of%20eukaryotic%20replication.%20Mol.%20Cell81,%202975%E2%80%932988.e2976%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Müller, C. A. et al. Capturing the dynamics of genome replication on individual ultra-long nanopore sequence reads. <em>Nat. Methods</em><strong>16</strong>, 429–436 (2019).
</cite> [<a href="https://doi.org/10.1038/s41592-019-0394-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7617212/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31011185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?M%C3%BCller,%20C.%20A.%20et%20al.%20Capturing%20the%20dynamics%20of%20genome%20replication%20on%20individual%20ultra-long%20nanopore%20sequence%20reads.%20Nat.%20Methods16,%20429%E2%80%93436%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Hennion, M. et al. FORK-seq: replication landscape of the Saccharomyces cerevisiae genome by nanopore sequencing. <em>Genome Biol.</em><strong>21</strong>, 125 (2020).
</cite> [<a href="https://doi.org/10.1186/s13059-020-02013-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7251829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32456659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hennion,%20M.%20et%20al.%20FORK-seq:%20replication%20landscape%20of%20the%20Saccharomyces%20cerevisiae%20genome%20by%20nanopore%20sequencing.%20Genome%20Biol.21,%20125%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Boemo, M. A. DNAscent v2: detecting replication forks in nanopore sequencing data with deep learning. <em>BMC Genomics</em><strong>22</strong>, 430 (2021).
</cite> [<a href="https://doi.org/10.1186/s12864-021-07736-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8191041/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34107894/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boemo,%20M.%20A.%20DNAscent%20v2:%20detecting%20replication%20forks%20in%20nanopore%20sequencing%20data%20with%20deep%20learning.%20BMC%20Genomics22,%20430%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Theulot, B. et al. Genome-wide mapping of individual replication fork velocities using nanopore sequencing. <em>Nat. Commun.</em><strong>13</strong>, 3295 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-31012-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9177527/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35676270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Theulot,%20B.%20et%20al.%20Genome-wide%20mapping%20of%20individual%20replication%20fork%20velocities%20using%20nanopore%20sequencing.%20Nat.%20Commun.13,%203295%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Claussin, C., Vazquez, J. &amp; Whitehouse, I. Single-molecule mapping of replisome progression. <em>Mol. Cell</em><strong>82</strong>, 1372–1382.e1374 (2022).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2022.02.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8995386/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35240057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Claussin,%20C.,%20Vazquez,%20J.%20&amp;%20Whitehouse,%20I.%20Single-molecule%20mapping%20of%20replisome%20progression.%20Mol.%20Cell82,%201372%E2%80%931382.e1374%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Totañes, F. rancisI. sidoreG. et al. A genome-wide map of DNA replication at single-molecule resolution in the malaria parasite Plasmodium falciparum. <em>Nucleic Acids Res.</em><strong>51</strong>, 2709–2724 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkad093" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10085703/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36808528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tota%C3%B1es,%20F.%20rancisI.%20sidoreG.%20et%20al.%20A%20genome-wide%20map%20of%20DNA%20replication%20at%20single-molecule%20resolution%20in%20the%20malaria%20parasite%20Plasmodium%20falciparum.%20Nucleic%20Acids%20Res.51,%202709%E2%80%932724%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Castellano, C. M. et al. The genetic landscape of origins of replication in P. falciparum. <em>Nucleic Acids Res.</em><strong>52</strong>, 660–676 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkad1103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10810204/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38038269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Castellano,%20C.%20M.%20et%20al.%20The%20genetic%20landscape%20of%20origins%20of%20replication%20in%20P.%20falciparum.%20Nucleic%20Acids%20Res.52,%20660%E2%80%93676%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Totañes, F. rancisI. sidoreG. et al. DNA replication dynamics are associated with genome composition in Plasmodium species. <em>Nucleic Acids Res.</em><strong>53</strong>, gkaf111 (2025).
</cite> [<a href="https://doi.org/10.1093/nar/gkaf111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11851108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39997219/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tota%C3%B1es,%20F.%20rancisI.%20sidoreG.%20et%20al.%20DNA%20replication%20dynamics%20are%20associated%20with%20genome%20composition%20in%20Plasmodium%20species.%20Nucleic%20Acids%20Res.53,%20gkaf111%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Georgieva, D., Liu, Q., Wang, K. &amp; Egli, D. Detection of base analogs incorporated during DNA replication by nanopore sequencing. <em>Nucleic Acids Res.</em><strong>48</strong>, e88–e88 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkaa517" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7470954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32710620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Georgieva,%20D.,%20Liu,%20Q.,%20Wang,%20K.%20&amp;%20Egli,%20D.%20Detection%20of%20base%20analogs%20incorporated%20during%20DNA%20replication%20by%20nanopore%20sequencing.%20Nucleic%20Acids%20Res.48,%20e88%E2%80%93e88%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Carrington, J. T. et al. Most human DNA replication initiation is dispersed throughout the genome with only a minority within previously identified initiation zones. <em>bioRxiv</em>, 2024.2004.2028.591325 (2024).</cite> [<a href="https://doi.org/10.1186/s13059-025-03591-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12063229/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40346587/" class="usa-link">PubMed</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Grant, G. D., Kedziora, K. M., Limas, J. C., Cook, J. G. &amp; Purvis, J. E. Accurate delineation of cell cycle phase transitions in living cells with PIP-FUCCI. <em>Cell Cycle</em><strong>17</strong>, 2496–2516 (2018).
</cite> [<a href="https://doi.org/10.1080/15384101.2018.1547001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6342071/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30421640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grant,%20G.%20D.,%20Kedziora,%20K.%20M.,%20Limas,%20J.%20C.,%20Cook,%20J.%20G.%20&amp;%20Purvis,%20J.%20E.%20Accurate%20delineation%20of%20cell%20cycle%20phase%20transitions%20in%20living%20cells%20with%20PIP-FUCCI.%20Cell%20Cycle17,%202496%E2%80%932516%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Chen, Y.-H. et al. ATR-Mediated Phosphorylation of FANCI Regulates Dormant Origin Firing in Response to Replication Stress. <em>Mol. Cell</em><strong>58</strong>, 323–338 (2015).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2015.02.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4408929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25843623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20Y.-H.%20et%20al.%20ATR-Mediated%20Phosphorylation%20of%20FANCI%20Regulates%20Dormant%20Origin%20Firing%20in%20Response%20to%20Replication%20Stress.%20Mol.%20Cell58,%20323%E2%80%93338%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Pennycook, B. R. et al. E2F-dependent transcription determines replication capacity and S phase length. <em>Nat. Commun.</em><strong>11</strong>, 3503 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-17146-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7360579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32665547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pennycook,%20B.%20R.%20et%20al.%20E2F-dependent%20transcription%20determines%20replication%20capacity%20and%20S%20phase%20length.%20Nat.%20Commun.11,%203503%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Benedict, B. et al. WAPL-Dependent Repair of Damaged DNA Replication Forks Underlies Oncogene-Induced Loss of Sister Chromatid Cohesion. <em>Developmental Cell</em><strong>52</strong>, 683–698.e687 (2020).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2020.01.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32084359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Benedict,%20B.%20et%20al.%20WAPL-Dependent%20Repair%20of%20Damaged%20DNA%20Replication%20Forks%20Underlies%20Oncogene-Induced%20Loss%20of%20Sister%20Chromatid%20Cohesion.%20Developmental%20Cell52,%20683%E2%80%93698.e687%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Técher, H. et al. Replication Dynamics: Biases and Robustness of DNA Fiber Analysis. <em>J. Mol. Biol.</em><strong>425</strong>, 4845–4855 (2013).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2013.03.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23557832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?T%C3%A9cher,%20H.%20et%20al.%20Replication%20Dynamics:%20Biases%20and%20Robustness%20of%20DNA%20Fiber%20Analysis.%20J.%20Mol.%20Biol.425,%204845%E2%80%934855%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Couch, F. B. et al. ATR phosphorylates SMARCAL1 to prevent replication fork collapse. <em>Genes Dev.</em><strong>27</strong>, 1610–1623 (2013).
</cite> [<a href="https://doi.org/10.1101/gad.214080.113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3731549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23873943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Couch,%20F.%20B.%20et%20al.%20ATR%20phosphorylates%20SMARCAL1%20to%20prevent%20replication%20fork%20collapse.%20Genes%20Dev.27,%201610%E2%80%931623%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Domínguez-Kelly, R. et al. WEE1 controls genomic stability during replication by regulating the Mus81-Eme1 endonuclease. <em>J. Cell Biol.</em><strong>194</strong>, 567–579 (2011).
</cite> [<a href="https://doi.org/10.1083/jcb.201101047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3160579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21859861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dom%C3%ADnguez-Kelly,%20R.%20et%20al.%20WEE1%20controls%20genomic%20stability%20during%20replication%20by%20regulating%20the%20Mus81-Eme1%20endonuclease.%20J.%20Cell%20Biol.194,%20567%E2%80%93579%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Maya-Mendoza, A. et al. High speed of fork progression induces DNA replication stress and genomic instability. <em>Nature</em><strong>559</strong>, 279–284 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0261-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29950726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maya-Mendoza,%20A.%20et%20al.%20High%20speed%20of%20fork%20progression%20induces%20DNA%20replication%20stress%20and%20genomic%20instability.%20Nature559,%20279%E2%80%93284%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Lossaint, G. et al. FANCD2 binds MCM proteins and controls replisome function upon activation of S phase checkpoint signaling. <em>Mol. Cell</em><strong>51</strong>, 678–690 (2013).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2013.07.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23993743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lossaint,%20G.%20et%20al.%20FANCD2%20binds%20MCM%20proteins%20and%20controls%20replisome%20function%20upon%20activation%20of%20S%20phase%20checkpoint%20signaling.%20Mol.%20Cell51,%20678%E2%80%93690%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Dungrawala, H. et al. The replication checkpoint prevents two types of fork collapse without regulating replisome stability. <em>Mol. Cell</em><strong>59</strong>, 998–1010 (2015).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2015.07.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4575883/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26365379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dungrawala,%20H.%20et%20al.%20The%20replication%20checkpoint%20prevents%20two%20types%20of%20fork%20collapse%20without%20regulating%20replisome%20stability.%20Mol.%20Cell59,%20998%E2%80%931010%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>McInnes, L., Healy, J., Saul, N. &amp; Großberger, L. UMAP: Uniform Manifold Approximation and Projection. <em>J. Open Source Softw.</em><strong>3</strong>, 861 (2018).</cite>
</li>
<li id="CR39">
<span class="label">39.</span><cite>Zhao, P. A., Sasaki, T. &amp; Gilbert, D. M. High-resolution Repli-Seq defines the temporal choreography of initiation, elongation and termination of replication in mammalian cells. <em>Genome Biol.</em><strong>21</strong>, 76 (2020).
</cite> [<a href="https://doi.org/10.1186/s13059-020-01983-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7092589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32209126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20P.%20A.,%20Sasaki,%20T.%20&amp;%20Gilbert,%20D.%20M.%20High-resolution%20Repli-Seq%20defines%20the%20temporal%20choreography%20of%20initiation,%20elongation%20and%20termination%20of%20replication%20in%20mammalian%20cells.%20Genome%20Biol.21,%2076%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Hughes, B. T., Sidorova, J., Swanger, J., Monnat, R. J. &amp; Clurman, B. E. Essential role for CDK2 inhibitory phosphorylation during replication stress revealed by a human CDK2 knockin mutation. <em>Proc. Natl Acad. Sci.</em><strong>110</strong>, 8954–8959 (2013).
</cite> [<a href="https://doi.org/10.1073/pnas.1302927110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3670391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23671119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hughes,%20B.%20T.,%20Sidorova,%20J.,%20Swanger,%20J.,%20Monnat,%20R.%20J.%20&amp;%20Clurman,%20B.%20E.%20Essential%20role%20for%20CDK2%20inhibitory%20phosphorylation%20during%20replication%20stress%20revealed%20by%20a%20human%20CDK2%20knockin%20mutation.%20Proc.%20Natl%20Acad.%20Sci.110,%208954%E2%80%938959%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Fu, H. et al. The DNA repair endonuclease Mus81 facilitates fast DNA replication in the absence of exogenous damage. <em>Nat. Commun.</em><strong>6</strong>, 6746 (2015).
</cite> [<a href="https://doi.org/10.1038/ncomms7746" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4400873/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25879486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fu,%20H.%20et%20al.%20The%20DNA%20repair%20endonuclease%20Mus81%20facilitates%20fast%20DNA%20replication%20in%20the%20absence%20of%20exogenous%20damage.%20Nat.%20Commun.6,%206746%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>van den Berg, J. et al. Quantifying DNA replication speeds in single cells by scEdU-seq. <em>Nat. Methods</em><strong>21</strong>, 1175–1184 (2024).
</cite> [<a href="https://doi.org/10.1038/s41592-024-02308-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11239516/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38886577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20den%20Berg,%20J.%20et%20al.%20Quantifying%20DNA%20replication%20speeds%20in%20single%20cells%20by%20scEdU-seq.%20Nat.%20Methods21,%201175%E2%80%931184%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Lopez-Contreras, A. J. et al. Increased Rrm2 gene dosage reduces fragile site breakage and prolongs survival of ATR mutant mice. <em>Genes Dev.</em><strong>29</strong>, 690–695 (2015).
</cite> [<a href="https://doi.org/10.1101/gad.256958.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4387711/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25838540/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lopez-Contreras,%20A.%20J.%20et%20al.%20Increased%20Rrm2%20gene%20dosage%20reduces%20fragile%20site%20breakage%20and%20prolongs%20survival%20of%20ATR%20mutant%20mice.%20Genes%20Dev.29,%20690%E2%80%93695%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Buisson, R., Boisvert, J. L., Benes, C. H. &amp; Zou, L. Distinct but Concerted Roles of ATR, DNA-PK, and Chk1 in Countering Replication Stress during S Phase. <em>Mol. Cell</em><strong>59</strong>, 1011–1024 (2015).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2015.07.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4575890/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26365377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buisson,%20R.,%20Boisvert,%20J.%20L.,%20Benes,%20C.%20H.%20&amp;%20Zou,%20L.%20Distinct%20but%20Concerted%20Roles%20of%20ATR,%20DNA-PK,%20and%20Chk1%20in%20Countering%20Replication%20Stress%20during%20S%C2%A0Phase.%20Mol.%20Cell59,%201011%E2%80%931024%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Zhang, Y.-W., Jones, T. L., Martin, S. E., Caplen, N. J. &amp; Pommier, Y. Implication of Checkpoint Kinase-dependent Up-regulation of Ribonucleotide Reductase R2 in DNA Damage Response *. <em>J. Biol. Chem.</em><strong>284</strong>, 18085–18095 (2009).
</cite> [<a href="https://doi.org/10.1074/jbc.M109.003020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2709352/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19416980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.-W.,%20Jones,%20T.%20L.,%20Martin,%20S.%20E.,%20Caplen,%20N.%20J.%20&amp;%20Pommier,%20Y.%20Implication%20of%20Checkpoint%20Kinase-dependent%20Up-regulation%20of%20Ribonucleotide%20Reductase%20R2%20in%20DNA%20Damage%20Response%20*.%20J.%20Biol.%20Chem.284,%2018085%E2%80%9318095%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Krude, T. Mimosine Arrests Proliferating Human Cells before Onset of DNA Replication in a Dose-Dependent Manner. <em>Exp. Cell Res.</em><strong>247</strong>, 148–159 (1999).
</cite> [<a href="https://doi.org/10.1006/excr.1998.4342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10047457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krude,%20T.%20Mimosine%20Arrests%20Proliferating%20Human%20Cells%20before%20Onset%20of%20DNA%20Replication%20in%20a%20Dose-Dependent%20Manner.%20Exp.%20Cell%20Res.247,%20148%E2%80%93159%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Koetsier, P. A. &amp; Cantor, E. J. A simple approach for effective shearing and reliable concentration measurement of ultra-high-molecular-weight DNA. <em>BioTechniques</em><strong>71</strong>, 439–444 (2021).
</cite> [<a href="https://doi.org/10.2144/btn-2021-0051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34232102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koetsier,%20P.%20A.%20&amp;%20Cantor,%20E.%20J.%20A%20simple%20approach%20for%20effective%20shearing%20and%20reliable%20concentration%20measurement%20of%20ultra-high-molecular-weight%20DNA.%20BioTechniques71,%20439%E2%80%93444%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Nurk, S. et al. The complete sequence of a human genome. <em>Science</em><strong>376</strong>, 44–53 (2022).
</cite> [<a href="https://doi.org/10.1126/science.abj6987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9186530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35357919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nurk,%20S.%20et%20al.%20The%20complete%20sequence%20of%20a%20human%20genome.%20Science376,%2044%E2%80%9353%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Jackson, D. A. &amp; Pombo, A. Replicon Clusters Are Stable Units of Chromosome Structure: Evidence That Nuclear Organization Contributes to the Efficient Activation and Propagation of S Phase in Human Cells. <em>J. Cell Biol.</em><strong>140</strong>, 1285–1295 (1998).
</cite> [<a href="https://doi.org/10.1083/jcb.140.6.1285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2132671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9508763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jackson,%20D.%20A.%20&amp;%20Pombo,%20A.%20Replicon%20Clusters%20Are%20Stable%20Units%20of%20Chromosome%20Structure:%20Evidence%20That%20Nuclear%20Organization%20Contributes%20to%20the%20Efficient%20Activation%20and%20Propagation%20of%20S%20Phase%20in%20Human%20Cells.%20J.%20Cell%20Biol.140,%201285%E2%80%931295%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Boemo, M. A., Pfuderer, P.L. &amp; Ye, S. (Zenodo, 2024).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365011/bin/41467_2025_63168_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365011/bin/41467_2025_63168_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (81.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365011/bin/41467_2025_63168_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (419.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365011/bin/41467_2025_63168_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (637.4KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The Nanopore sequencing data generated in this study have been deposited in NCBI’s Gene Expression Omnibus under the GEO Series accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE216926" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE216926</a> and the European Nucleotide Archive under accession number <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB80561" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJEB80561</a>. <a href="#Sec29" class="usa-link">Source data</a> are provided with this paper.</p>
<p>The DNAscent software is available at <a href="https://github.com/MBoemo/DNAscent" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/MBoemo/DNAscent</a> under the GPL-3.0 open-source license<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. DNAscent v3.1.2 was used for analysis on the R9.4.1 Oxford Nanopore chemistry while DNAscent v4.0.3 was used for analysis on the R10.4.1 chemistry. Both versions are available as Singularity images at <a href="https://cloud.sylabs.io/library/mboemo/dnascent/dnascent" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cloud.sylabs.io/library/mboemo/dnascent/dnascent</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63168-w"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63168.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365011/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365011/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365011%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365011/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365011/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365011/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830365/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365011/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830365/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365011/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365011/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="jFAu898B1HMfvnogeBEIlZwxzcO3uvHw4kUjt8vAHEMC1bIOeHVk2MVYk9ZT2Ldy">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
