
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Optimisation of Heated Electrospray Ionisation Parameters to Minimise In‐Source Generated Impurities in the Analysis of Oligonucleotide Therapeutics - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="37B2C0AB7FFF65B306C0AB003067061D.m_1" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11969060/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Rapid Communications in Mass Spectrometry">
<meta name="citation_title" content="Optimisation of Heated Electrospray Ionisation Parameters to Minimise In‐Source Generated Impurities in the Analysis of Oligonucleotide Therapeutics">
<meta name="citation_author" content="Mollie A Glenister">
<meta name="citation_author_institution" content="School of Chemical, Materials and Biological Engineering, University of Sheffield, Sheffield, UK">
<meta name="citation_author" content="Ulrik Mistarz">
<meta name="citation_author_institution" content="ThermoFisher Scientific, Allerød, Denmark">
<meta name="citation_author" content="Ken Cook">
<meta name="citation_author_institution" content="ThermoFisher Scientific, Hemel Hempstead, UK">
<meta name="citation_author" content="James L Stephenson">
<meta name="citation_author_institution" content="ThermoFisher Scientific, Lexington, Massachusetts, USA">
<meta name="citation_author" content="Mark J Dickman">
<meta name="citation_author_institution" content="School of Chemical, Materials and Biological Engineering, University of Sheffield, Sheffield, UK">
<meta name="citation_publication_date" content="2025 Apr 3">
<meta name="citation_volume" content="39">
<meta name="citation_issue" content="13">
<meta name="citation_firstpage" content="e10033">
<meta name="citation_doi" content="10.1002/rcm.10033">
<meta name="citation_pmid" content="40181565">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11969060/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11969060/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11969060/pdf/RCM-39-e10033.pdf">
<meta name="description" content="Oligonucleotides have emerged as an important new class of therapeutic. Due to their structural complexity, this presents significant challenges for the development of analytical methods to characterise and determine their impurity profile. In this ...">
<meta name="og:title" content="Optimisation of Heated Electrospray Ionisation Parameters to Minimise In‐Source Generated Impurities in the Analysis of Oligonucleotide Therapeutics">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Oligonucleotides have emerged as an important new class of therapeutic. Due to their structural complexity, this presents significant challenges for the development of analytical methods to characterise and determine their impurity profile. In this ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11969060/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11969060">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/rcm.10033"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/RCM-39-e10033.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11969060%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11969060/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11969060/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11969060/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/rcm.10033" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Rapid Commun Mass Spectrom</button></div>. 2025 Apr 3;39(13):e10033. doi: <a href="https://doi.org/10.1002/rcm.10033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/rcm.10033</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Rapid%20Commun%20Mass%20Spectrom%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rapid%20Commun%20Mass%20Spectrom%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Rapid%20Commun%20Mass%20Spectrom%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Rapid%20Commun%20Mass%20Spectrom%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Optimisation of Heated Electrospray Ionisation Parameters to Minimise In‐Source Generated Impurities in the Analysis of Oligonucleotide Therapeutics</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Glenister%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Mollie A Glenister</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mollie A Glenister</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>School of Chemical, Materials and Biological Engineering, University of Sheffield, Sheffield, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Glenister%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mollie A Glenister</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mistarz%20U%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ulrik Mistarz</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ulrik Mistarz</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>ThermoFisher Scientific, Allerød, Denmark</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mistarz%20U%22%5BAuthor%5D" class="usa-link"><span class="name western">Ulrik Mistarz</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cook%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Ken Cook</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Ken Cook</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>ThermoFisher Scientific, Hemel Hempstead, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cook%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Ken Cook</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stephenson%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">James L Stephenson</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">James L Stephenson</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>ThermoFisher Scientific, Lexington, Massachusetts, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stephenson%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">James L Stephenson</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dickman%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Mark J Dickman</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Mark J Dickman</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>School of Chemical, Materials and Biological Engineering, University of Sheffield, Sheffield, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dickman%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mark J Dickman</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="rcm10033-aff-0001">
<sup>
<sup>1</sup>
</sup>School of Chemical, Materials and Biological Engineering, University of Sheffield, Sheffield, UK</div>
<div id="rcm10033-aff-0002">
<sup>
<sup>2</sup>
</sup>ThermoFisher Scientific, Allerød, Denmark</div>
<div id="rcm10033-aff-0003">
<sup>
<sup>3</sup>
</sup>ThermoFisher Scientific, Hemel Hempstead, UK</div>
<div id="rcm10033-aff-0004">
<sup>
<sup>4</sup>
</sup>ThermoFisher Scientific, Lexington, Massachusetts, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Mark J. Dickman (<span>m.dickman@sheffield.ac.uk</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Mar 7; Received 2024 Nov 25; Accepted 2025 Mar 14; Issue date 2025 Jul 15.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Rapid Communications in Mass Spectrometry</em> published by John Wiley &amp; Sons Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11969060  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40181565/" class="usa-link">40181565</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<section id="rcm10033-sec-0001"><h3 class="pmc_sec_title">Rationale</h3>
<p>Oligonucleotides have emerged as an important new class of therapeutic. Due to their structural complexity, this presents significant challenges for the development of analytical methods to characterise and determine their impurity profile. In this study, we introduce a sensitive ion‐pair reverse phase method interfaced with mass spectrometry for analysis of antisense oligonucleotides and small interfering RNAs.</p></section><section id="rcm10033-sec-0002"><h3 class="pmc_sec_title">Methods</h3>
<p>Liquid chromatography–mass spectrometry analysis of antisense oligonucleotides and small interfering RNAs was performed using hexylamine: hexafluoro‐2‐propanol mobiles phases. LC‐MS analysis was performed in both negative and positive ion mode. Electrospray ionisation source conditions including collision energy and temperature were optimised to minimise in‐source generated impurities and alkylamine adducts in the analysis of oligonucleotide therapeutics.</p></section><section id="rcm10033-sec-0003"><h3 class="pmc_sec_title">Results</h3>
<p>The results show that under low or no in‐source collision energy the presence of hexylamine adducts are observed and are predominantly on the lowest charge states present. As the in‐source collision energy is increased, a reduction of hexylamine adducts is observed in conjunction with an increase in nucleobase loss in the gas phase, therefore generating in‐source impurities. In comparison to tributylammonium acetate, increased MS sensitivity, higher charge states and effective removal of hexylamine adducts using mild source conditions was achieved.</p></section><section id="rcm10033-sec-0004"><h3 class="pmc_sec_title">Conclusions</h3>
<p>Optimisation of the mild source conditions in conjunction with high pH mobile phases was combined with high‐resolution accurate mass spectrometry analysis and automated deconvolution workflows to develop a simplified and streamlined approach for characterising oligonucleotide therapeutics and their related impurities.</p></section><section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> electrospray ionisation, ion‐pair reversed phase, oligonucleotide therapeutics</p></section></section><section id="rcm10033-sec-0005"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Oligonucleotide (OGN) therapeutics are an emerging class of drugs which have the potential to treat a wide range of diseases and conditions, including cancer and neurological conditions [<a href="#rcm10033-bib-0001" class="usa-link" aria-describedby="rcm10033-bib-0001">1</a>, <a href="#rcm10033-bib-0002" class="usa-link" aria-describedby="rcm10033-bib-0002">2</a>, <a href="#rcm10033-bib-0003" class="usa-link" aria-describedby="rcm10033-bib-0003">3</a>]. Twenty oligonucleotide therapeutics have been approved by both the Food and Drug Administration (FDA) and the European Medical Agency (EMA) as of March 2024 [<a href="#rcm10033-bib-0004" class="usa-link" aria-describedby="rcm10033-bib-0004">4</a>], showcasing the growing interest in this field.</p>
<p>There are a number of different classes of OGN therapeutics including antisense OGNs (ASOs), small interfering RNAs (siRNAs), involved in RNA interference (RNAi), small activating RNAs (saRNAs), aptamers and guide RNAs (gRNAs) involved in CRISPR gene editing technology. The most dominant of these subgroups is ASOs which represent 65% of OGNs having obtained marketing authorisation [<a href="#rcm10033-bib-0005" class="usa-link" aria-describedby="rcm10033-bib-0005">5</a>]. This class of OGN therapeutic are single‐stranded nucleic acids which have complementary sequences to target mRNA and results in degradation and the downregulation of specific genes [<a href="#rcm10033-bib-0006" class="usa-link" aria-describedby="rcm10033-bib-0006">6</a>, <a href="#rcm10033-bib-0007" class="usa-link" aria-describedby="rcm10033-bib-0007">7</a>]. Modifications to OGN structure are often included in the therapeutic design to improve stability by preventing degradation by endonucleases, increasing cellular uptake and improving affinity to target strands [<a href="#rcm10033-bib-0008" class="usa-link" aria-describedby="rcm10033-bib-0008">8</a>].</p>
<p>Recent advances in nucleic acid chemistry have substantially influenced the design and optimisation of OGN therapies and their delivery mechanisms. Chemical stabilisation techniques have been used to enhance pharmacokinetic and pharmacodynamic responses. Chemical modifications are introduced into the structure of the nucleic acids or linkers and typically include replacing the non‐bridging oxygen atoms in the phosphate backbone with sulphur groups (phosphorothioate modifications), which reduces enzyme degradation and increases uptake into cells [<a href="#rcm10033-bib-0009" class="usa-link" aria-describedby="rcm10033-bib-0009">9</a>, <a href="#rcm10033-bib-0010" class="usa-link" aria-describedby="rcm10033-bib-0010">10</a>]. Adding an electron‐withdrawing group to the 2′ hydroxyl group of the ribose sugar, including methyl (Me) groups, methoxyethyl (MOE) groups and fluorine (F) groups. These modifications have also been shown to increase resistance to enzyme degradation and improve the stability of DNA–RNA duplex [<a href="#rcm10033-bib-0011" class="usa-link" aria-describedby="rcm10033-bib-0011">11</a>, <a href="#rcm10033-bib-0012" class="usa-link" aria-describedby="rcm10033-bib-0012">12</a>, <a href="#rcm10033-bib-0013" class="usa-link" aria-describedby="rcm10033-bib-0013">13</a>]. Lastly, chemical modification of the nucleotide bases, the most common includes modification of the 5′ positions of cytosine, thymine and uracil. Adding methyl groups to cytosine is widely used and is shown to improve binding affinity to the target strand [<a href="#rcm10033-bib-0014" class="usa-link" aria-describedby="rcm10033-bib-0014">14</a>].</p>
<p>Bioconjugates have been utilised to link OGNs to biomolecules such as proteins, antibodies and carbohydrates [<a href="#rcm10033-bib-0015" class="usa-link" aria-describedby="rcm10033-bib-0015">15</a>, <a href="#rcm10033-bib-0016" class="usa-link" aria-describedby="rcm10033-bib-0016">16</a>, <a href="#rcm10033-bib-0017" class="usa-link" aria-describedby="rcm10033-bib-0017">17</a>]. N‐acetylgalactosamine (GalNAc) conjugates have been shown to greatly improve cellular penetration [<a href="#rcm10033-bib-0014" class="usa-link" aria-describedby="rcm10033-bib-0014">14</a>] and aid in targeting the liver and kidneys to treat diseases which impact these organs [<a href="#rcm10033-bib-0018" class="usa-link" aria-describedby="rcm10033-bib-0018">18</a>, <a href="#rcm10033-bib-0019" class="usa-link" aria-describedby="rcm10033-bib-0019">19</a>].</p>
<p>OGN therapeutics have great potential to treat a wide range of conditions within the medical landscape. They are complex molecules, and therefore, robust analytical methods must be developed to allow them to reach their potential and ensure their safe use. Impurity profiling of nucleic acid therapeutics is predominantly performed using ion‐pair reverse‐phase high‐performance liquid chromatography (IP‐RP HPLC) with ultra‐violet (UV) detection and mass spectrometry (MS) or tandem mass spectrometry (MS/MS) [<a href="#rcm10033-bib-0020" class="usa-link" aria-describedby="rcm10033-bib-0020">20</a>, <a href="#rcm10033-bib-0021" class="usa-link" aria-describedby="rcm10033-bib-0021">21</a>, <a href="#rcm10033-bib-0022" class="usa-link" aria-describedby="rcm10033-bib-0022">22</a>].</p>
<p>IP‐RP HPLC typically utilises tertiary alkylamines ion‐pairing reagents including triethylammonium acetate (TEAA) and tributylammonium acetate (TBuAA) [<a href="#rcm10033-bib-0022" class="usa-link" aria-describedby="rcm10033-bib-0022">22</a>, <a href="#rcm10033-bib-0023" class="usa-link" aria-describedby="rcm10033-bib-0023">23</a>, <a href="#rcm10033-bib-0024" class="usa-link" aria-describedby="rcm10033-bib-0024">24</a>, <a href="#rcm10033-bib-0025" class="usa-link" aria-describedby="rcm10033-bib-0025">25</a>]. Using these alkylamines with acetic acid as the counter ion results in high‐resolution separations; however, signal suppression occurs when interfaced with mass spectrometry. Alkylamines with added fluoroalcohols are the most common mobile‐phase component used to improve mass spectrometry sensitivity [<a href="#rcm10033-bib-0024" class="usa-link" aria-describedby="rcm10033-bib-0024">24</a>, <a href="#rcm10033-bib-0026" class="usa-link" aria-describedby="rcm10033-bib-0026">26</a>, <a href="#rcm10033-bib-0027" class="usa-link" aria-describedby="rcm10033-bib-0027">27</a>, <a href="#rcm10033-bib-0028" class="usa-link" aria-describedby="rcm10033-bib-0028">28</a>, <a href="#rcm10033-bib-0029" class="usa-link" aria-describedby="rcm10033-bib-0029">29</a>]. Mobile phases using alkylamine and fluoroalcohols such as 1,1,1,3,3,3‐hexafluoro‐2‐propanol (HFIP) as the counter ion results in improved mass spectrometry sensitivity when IP RP separations are interfaced with heated electrospray ionisation mass spectrometry (HESI MS) and offer a powerful approach for characterising and quantifying OGNs therapeutics and their related impurities [<a href="#rcm10033-bib-0026" class="usa-link" aria-describedby="rcm10033-bib-0026">26</a>, <a href="#rcm10033-bib-0030" class="usa-link" aria-describedby="rcm10033-bib-0030">30</a>, <a href="#rcm10033-bib-0031" class="usa-link" aria-describedby="rcm10033-bib-0031">31</a>]. The development and application of a wide range of different alkylamine/fluoroalcohol mobile phases have been studied, including the effect of mobile phase ageing [<a href="#rcm10033-bib-0028" class="usa-link" aria-describedby="rcm10033-bib-0028">28</a>, <a href="#rcm10033-bib-0032" class="usa-link" aria-describedby="rcm10033-bib-0032">32</a>, <a href="#rcm10033-bib-0033" class="usa-link" aria-describedby="rcm10033-bib-0033">33</a>, <a href="#rcm10033-bib-0034" class="usa-link" aria-describedby="rcm10033-bib-0034">34</a>].</p>
<p>A number of proposed ion‐pair mechanisms and models have been discussed [<a href="#rcm10033-bib-0035" class="usa-link" aria-describedby="rcm10033-bib-0035">35</a>, <a href="#rcm10033-bib-0036" class="usa-link" aria-describedby="rcm10033-bib-0036">36</a>, <a href="#rcm10033-bib-0037" class="usa-link" aria-describedby="rcm10033-bib-0037">37</a>, <a href="#rcm10033-bib-0038" class="usa-link" aria-describedby="rcm10033-bib-0038">38</a>, <a href="#rcm10033-bib-0039" class="usa-link" aria-describedby="rcm10033-bib-0039">39</a>]. A dynamic ion exchange model proposed that the IP reagent first adsorbs at the surface of the stationary phase, providing exchange sites for analytes [<a href="#rcm10033-bib-0035" class="usa-link" aria-describedby="rcm10033-bib-0035">35</a>, <a href="#rcm10033-bib-0040" class="usa-link" aria-describedby="rcm10033-bib-0040">40</a>, <a href="#rcm10033-bib-0041" class="usa-link" aria-describedby="rcm10033-bib-0041">41</a>, <a href="#rcm10033-bib-0042" class="usa-link" aria-describedby="rcm10033-bib-0042">42</a>]. It was also proposed that both ion‐pairing and dynamic ion exchange occur depending on the experimental conditions [<a href="#rcm10033-bib-0035" class="usa-link" aria-describedby="rcm10033-bib-0035">35</a>, <a href="#rcm10033-bib-0036" class="usa-link" aria-describedby="rcm10033-bib-0036">36</a>]. The contribution of ionic interaction in IP‐RP LC depends on the hydrophobicity of IP reagent and is moderated by the amount of organic modifier in the mobile phase. Retention of OGNs in IP RP HPLC was shown to correlate with the alkylamine hydrophobicity [<a href="#rcm10033-bib-0043" class="usa-link" aria-describedby="rcm10033-bib-0043">43</a>]. Hydrophobic alkylamines act as more efficient (stronger) IP reagents due to their increased strength of adsorption to the stationary phase compared to less hydrophobic (weaker) IP reagents.</p>
<p>Modulating the mobile phases and IP reagents can be utilised for different purposes. Stronger IP reagents, including tributylamine (TBuA) and hexylamine (HA), are better suited for analysis of phosphorothioate (PS) OGNs as size‐dependent separations largely dominate, reducing the effect of the diastereoisomers present which detrimentally impact chromatographic resolution [<a href="#rcm10033-bib-0024" class="usa-link" aria-describedby="rcm10033-bib-0024">24</a>, <a href="#rcm10033-bib-0025" class="usa-link" aria-describedby="rcm10033-bib-0025">25</a>]. Although strong IP alkylamines offer improved chromatography for the analysis of PS OGNs, there are a number of caveats associated with interfacing with mass spectrometry analysis including the formation of ion‐pair adducts, which cause significant polydispersity of the target analyte signal [<a href="#rcm10033-bib-0044" class="usa-link" aria-describedby="rcm10033-bib-0044">44</a>]. This reduction of MS signal intensity can lead to difficulty identifying low‐level impurities and small mass changes caused by modification. Strong IP regents, like TBuA [<a href="#rcm10033-bib-0022" class="usa-link" aria-describedby="rcm10033-bib-0022">22</a>] and HA, are more prone to adduct formation, which is attributed to their high Henrys law partition coefficient, which results in their low volatility. During droplet formation in the HESI source, the OGN and the alkylamine can desorb as a complex [<a href="#rcm10033-bib-0045" class="usa-link" aria-describedby="rcm10033-bib-0045">45</a>, <a href="#rcm10033-bib-0046" class="usa-link" aria-describedby="rcm10033-bib-0046">46</a>]. Removal of these adducts is key to the benefits of strong IP analysis and improved spectra quality via increased signal.</p>
<p>Removing strong ion‐pairs adducts can be challenging; methods for their removal involve optimising ion source settings, such as temperature, gas flow and in‐source collision energy (<span class="font-variant-small-caps">ce</span>). However, this can also lead to the formation of in‐source impurities in the form of fragmentation ions. Specifically, base loss fragmentation with loss of adenine and guanine commonly seen [<a href="#rcm10033-bib-0047" class="usa-link" aria-describedby="rcm10033-bib-0047">47</a>, <a href="#rcm10033-bib-0048" class="usa-link" aria-describedby="rcm10033-bib-0048">48</a>]. Therefore, care is needed to balance the removal of adducts with unwanted fragmentation. Previous work using TBuA utilised different source settings, including drying gas flow rate and temperature, to remove TBuA adducts [<a href="#rcm10033-bib-0022" class="usa-link" aria-describedby="rcm10033-bib-0022">22</a>]. These settings were optimised for each sequence individually to achieve the ideal balance of adduct removal and fragmentation.</p>
<p>In this study, we utilised HA:HFIP mobiles phases and optimised HESI source conditions to minimise in‐source generated impurities in the analysis of OGN therapeutics. In‐source conditions were optimised to ensure efficient removal of alkylamine adducts, whilst minimising in‐source fragmentation of OGNs. We have studied the effect of altering a range of HESI parameters including in‐source <span class="font-variant-small-caps">ce</span>, ion transfer tube (ITT) temperature and vaporiser temperature (VT) on the formation of alkylamine adducts and production of in‐source related impurities in a wide range of different OGNs including ASOs and siRNAs. Furthermore, combined with high‐resolution accurate mass spectrometry (HRAM) analysis and automated deconvolution, a simplified and streamlined approach has been developed for the characterisation of OGN therapeutics and their related impurities.</p></section><section id="rcm10033-sec-0006"><h2 class="pmc_sec_title">2. Material and Methods</h2>
<section id="rcm10033-sec-0007"><h3 class="pmc_sec_title">2.1. Chemicals and Reagents</h3>
<p>Water (UHPLC‐MS grade, Thermo Scientific), acetonitrile (UHPLC‐MS grade, Thermo Scientific), 1,1,1,3,3,3,‐hexafluoro‐2‐propanol (HFIP, &gt;99.8% Fluka LC‐MS grade), hexylamine (HA, 99.7% extrapure Fisher Scientific), tributylamine (TBA, 99%, Acros Organics, Fisher Scientific) and glacial acetic acid (HPLC grade, VWR Chemicals) were used for LC‐MS mobile phases.</p></section><section id="rcm10033-sec-0008"><h3 class="pmc_sec_title">2.2. Oligonucleotides</h3>
<p>OGNs used in this study are shown in Table <a href="#rcm10033-tbl-0001" class="usa-link">1</a> and were purchased from Eurofins Genomics and MedChemExpress.</p>
<section class="tw xbox font-sm" id="rcm10033-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Oligonucleotides used in this study. OGN 1 (Patisiran‐antisense siRNA), OGN 2 (Patisiran‐sense siRNA). OGN 3 (Tofersen), OGN 4,5 (fully phoshorothioated 20 mers).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<tbody valign="top"><tr><td align="left" valign="top" rowspan="1" colspan="1">
<img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3026/11969060/15f132a36682/RCM-39-e10033-g002.jpg" loading="lazy" alt="graphic file with name RCM-39-e10033-g002.jpg">
</td></tr></tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/rcm10033-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="rcm10033-sec-0009"><h3 class="pmc_sec_title">2.3. HPLC Analysis</h3>
<p>IP RP HPLC analysis was performed on a U3000 HPLC (Thermo Fisher) using a DNAPac RP column (2.1 mm × 100 mm) (Thermo Fisher).</p>
<p>For HA/HFIP mobile phases, mobile phase (A) consisted of 15 mM HA and 60 mM HFIP, and Mobile phase (B) consisted of 15 mM HA, 60 mM HFIP, and 40% ACN,. HPLC separations were performed using a linear gradient starting at 20% (B) and ending at 90% (B) over 14.2 min, flow rate 300 μL/min at 50 °C and UV absorbance 260 nm. Fifty picomoles of OGN 1 and 2 and 100 pmol of OGNs 3, 4 and 5 was injected.</p>
<p>One hundred naomolars tributlyammonium acetate (TBuAA) stock was made using TBA and glacial acetic acid. Mobile phase (A) consisted of 5 mM TBuAA and 10% ACN, and mobile phase (B) consisted of 5 mM TBuAA, and 80% ACN. HPLC separations were performed using a linear gradient starting at 30% B and ending at 65% B over 25 min, flow rate 200 μL/min at 60 °C and UV absorbance 260 nm. One hundred picomoles of OGN 3 injected.</p></section><section id="rcm10033-sec-0010"><h3 class="pmc_sec_title">2.4. Mass Spectrometry Analysis</h3>
<p>Mass spectrometry analysis was performed using an Orbitrap Exploris 480 (Thermo Fisher) or Orbitrap Exploris 240 (Thermo Fisher Scientific). The MS settings include a vaporiser temperature of 300 °C, sheath gas set to 35 arbitrary units, aux gas set to 10 arbitrary units and sweep gas set to 5 arbitrary units unless otherwise stated. MS detection was performed in both negative and positive mode. The scan range was 400–2500 m/z. The ITT temperature was set to 275 °C, and the VT was set to 300 °C as default and varied accordingly. In‐source <span class="font-variant-small-caps">ce</span> was varied between 0 and 65 V for HA/HFIP in negative mode, 30 and 90 V for HA/HFIP in positive mode and 0–65 V for TBuAA in negative mode. Data was acquired in the peptide application mode at standard pressure (1 arbitrary unit), and the resolution was set to 120 000. The RF lens percentage was set to 70.</p></section><section id="rcm10033-sec-0011"><h3 class="pmc_sec_title">2.5. Data Analysis</h3>
<p>The Intact Protein Deconvolution function in Chromeleon 7.3.1 or 7.3.2 was used for deconvolution using XTRACT sliding windows deconvolution with the following settings: rel. intensity threshold = 1%, target avg. spectrum width = 0.390 min, number of detected intervals = 3, merge tolerance = 30 ppm, max RT gap = 1, output mass range = 1000–8000, S/N threshold = 3, charge range = 2–15 (OGN1 + 2), 2–10 (OGNs 3, 4 and 5), min number detected charge = 2 and rel abundance threshold = 0. For each OGN, the RT widow and m/z range were set individually to match the LC‐MS method. Custom columns, target mass (Da) and target tolerance (Da) were added to the sequence manager window. Target mass was set to the mass of the sample, and target tolerance was set to 1. A custom impurities table featuring the ions specific to this analysis was used with the reporting feature to quantify the target mass, IP adducts and fragmentation ions.</p></section></section><section id="rcm10033-sec-0012"><h2 class="pmc_sec_title">3. Results and Discussion</h2>
<section id="rcm10033-sec-0013"><h3 class="pmc_sec_title">3.1. Optimisation of LC HESI MS Source Conditions for the Analysis of OGN Therapeutics in Negative Mode Ionisation</h3>
<p>A range of different OGNs including therapeutic OGNs were analysed using IP‐RP LC‐MS in conjunction with 15 mM HA/60 mM HFIP mobile phases. Under these conditions, size‐dependent separations largely dominate, reducing the effect of the diastereoisomers present in phosphorothioate (PS) OGNs, which detrimentally impact chromatographic resolution of the full length product from associated manufacturing impurities. Initial work focused on the effect of in‐source <span class="font-variant-small-caps">ce</span> setting on an Orbitrap Exploris mass spectrometer on HA adduct removal and in‐source fragmentation of the corresponding OGN in negative ion mode. The LC‐MS spectra obtained for OGN1 (Patisiran‐antisense siRNA a 21 mer ssRNA partially 2’‐O‐methylated, see Table <a href="#rcm10033-tbl-0001" class="usa-link">1</a>) over a range of different in‐source <span class="font-variant-small-caps">ce</span> (0–65 V) is shown in Figure <a href="#rcm10033-fig-0001" class="usa-link">1A,B</a>. The results show that under low or no in‐source <span class="font-variant-small-caps">ce</span> the presence of HA adducts are observed and are predominantly on the lowest charge states present, while no significant HFIP or metal ion adducts were observed. This results in polydispersity of the signal on the low charge states. A combination of reduced coulombic repulsion, favourable conformational states, and solvation effects in lower charge states of the OGN ions makes the formation of alkylamine adducts more likely in these charge states during negative ion HESI. At 0 V, the presence of one to three HA adducts are highlighted (see Figure <a href="#rcm10033-fig-0001" class="usa-link">1A</a>, top left panel). As the in‐source <span class="font-variant-small-caps">ce</span> is increased, a reduction of HA adducts is observed. However, as in‐source <span class="font-variant-small-caps">ce</span> increases, this causes an increase OGN fragmentation with a predominant loss of adenosine (−A) and guanine (−G) consistent with the known gas phase fragmentation behaviour of OGNs [<a href="#rcm10033-bib-0048" class="usa-link" aria-describedby="rcm10033-bib-0048">48</a>]. Nucleobases can be lost as neutral or negatively charged species in the gas phase. Previous work has shown that the preference for neutral or charged base loss depends on a number of factors, including the charge state of the OGN, sequence position and whether the OGN is DNA or RNA [<a href="#rcm10033-bib-0049" class="usa-link" aria-describedby="rcm10033-bib-0049">49</a>, <a href="#rcm10033-bib-0050" class="usa-link" aria-describedby="rcm10033-bib-0050">50</a>, <a href="#rcm10033-bib-0051" class="usa-link" aria-describedby="rcm10033-bib-0051">51</a>]. For the analysis of OGN1 (siRNA), m/z values corresponding to the [M‐ 5H‐G<sup>−</sup>]<sup>4−</sup> were observed for charged base loss of G, neutral bass loss was observed for A and C with m/z values corresponding to [M‐4H‐B(A)]<sup>4−</sup> and [M‐4H‐B(C)] [<a href="#rcm10033-bib-0004" class="usa-link" aria-describedby="rcm10033-bib-0004">4</a>, <a href="#rcm10033-bib-0005" class="usa-link" aria-describedby="rcm10033-bib-0005">5</a>, <a href="#rcm10033-bib-0006" class="usa-link" aria-describedby="rcm10033-bib-0006">6</a>, <a href="#rcm10033-bib-0007" class="usa-link" aria-describedby="rcm10033-bib-0007">7</a>, <a href="#rcm10033-bib-0008" class="usa-link" aria-describedby="rcm10033-bib-0008">8</a>, <a href="#rcm10033-bib-0009" class="usa-link" aria-describedby="rcm10033-bib-0009">9</a>, <a href="#rcm10033-bib-0010" class="usa-link" aria-describedby="rcm10033-bib-0010">10</a>, <a href="#rcm10033-bib-0011" class="usa-link" aria-describedby="rcm10033-bib-0011">11</a>, <a href="#rcm10033-bib-0012" class="usa-link" aria-describedby="rcm10033-bib-0012">12</a>, <a href="#rcm10033-bib-0013" class="usa-link" aria-describedby="rcm10033-bib-0013">13</a>, <a href="#rcm10033-bib-0014" class="usa-link" aria-describedby="rcm10033-bib-0014">14</a>, <a href="#rcm10033-bib-0015" class="usa-link" aria-describedby="rcm10033-bib-0015">15</a>, <a href="#rcm10033-bib-0016" class="usa-link" aria-describedby="rcm10033-bib-0016">16</a>, <a href="#rcm10033-bib-0017" class="usa-link" aria-describedby="rcm10033-bib-0017">17</a>, <a href="#rcm10033-bib-0018" class="usa-link" aria-describedby="rcm10033-bib-0018">18</a>, <a href="#rcm10033-bib-0019" class="usa-link" aria-describedby="rcm10033-bib-0019">19</a>, <a href="#rcm10033-bib-0020" class="usa-link" aria-describedby="rcm10033-bib-0020">20</a>, <a href="#rcm10033-bib-0021" class="usa-link" aria-describedby="rcm10033-bib-0021">21</a>, <a href="#rcm10033-bib-0022" class="usa-link" aria-describedby="rcm10033-bib-0022">22</a>, <a href="#rcm10033-bib-0023" class="usa-link" aria-describedby="rcm10033-bib-0023">23</a>, <a href="#rcm10033-bib-0024" class="usa-link" aria-describedby="rcm10033-bib-0024">24</a>, <a href="#rcm10033-bib-0025" class="usa-link" aria-describedby="rcm10033-bib-0025">25</a>, <a href="#rcm10033-bib-0026" class="usa-link" aria-describedby="rcm10033-bib-0026">26</a>, <a href="#rcm10033-bib-0027" class="usa-link" aria-describedby="rcm10033-bib-0027">27</a>, <a href="#rcm10033-bib-0028" class="usa-link" aria-describedby="rcm10033-bib-0028">28</a>, <a href="#rcm10033-bib-0029" class="usa-link" aria-describedby="rcm10033-bib-0029">29</a>, <a href="#rcm10033-bib-0030" class="usa-link" aria-describedby="rcm10033-bib-0030">30</a>, <a href="#rcm10033-bib-0031" class="usa-link" aria-describedby="rcm10033-bib-0031">31</a>, <a href="#rcm10033-bib-0032" class="usa-link" aria-describedby="rcm10033-bib-0032">32</a>, <a href="#rcm10033-bib-0033" class="usa-link" aria-describedby="rcm10033-bib-0033">33</a>, <a href="#rcm10033-bib-0034" class="usa-link" aria-describedby="rcm10033-bib-0034">34</a>, <a href="#rcm10033-bib-0035" class="usa-link" aria-describedby="rcm10033-bib-0035">35</a>, <a href="#rcm10033-bib-0036" class="usa-link" aria-describedby="rcm10033-bib-0036">36</a>, <a href="#rcm10033-bib-0037" class="usa-link" aria-describedby="rcm10033-bib-0037">37</a>, <a href="#rcm10033-bib-0038" class="usa-link" aria-describedby="rcm10033-bib-0038">38</a>, <a href="#rcm10033-bib-0039" class="usa-link" aria-describedby="rcm10033-bib-0039">39</a>, <a href="#rcm10033-bib-0040" class="usa-link" aria-describedby="rcm10033-bib-0040">40</a>, <a href="#rcm10033-bib-0041" class="usa-link" aria-describedby="rcm10033-bib-0041">41</a>, <a href="#rcm10033-bib-0042" class="usa-link" aria-describedby="rcm10033-bib-0042">42</a>, <a href="#rcm10033-bib-0043" class="usa-link" aria-describedby="rcm10033-bib-0043">43</a>, <a href="#rcm10033-bib-0044" class="usa-link" aria-describedby="rcm10033-bib-0044">44</a>, <a href="#rcm10033-bib-0045" class="usa-link" aria-describedby="rcm10033-bib-0045">45</a>, <a href="#rcm10033-bib-0046" class="usa-link" aria-describedby="rcm10033-bib-0046">46</a>, <a href="#rcm10033-bib-0047" class="usa-link" aria-describedby="rcm10033-bib-0047">47</a>, <a href="#rcm10033-bib-0048" class="usa-link" aria-describedby="rcm10033-bib-0048">48</a>, <a href="#rcm10033-bib-0049" class="usa-link" aria-describedby="rcm10033-bib-0049">49</a>] (see Figure <a href="#rcm10033-fig-0001" class="usa-link">1B</a> and Supplementary Figure <a href="#rcm10033-supitem-0001" class="usa-link">S1A</a>). In addition, as <span class="font-variant-small-caps">ce</span> is increased, a reduction in signal intensity of the higher charge states (−8, −9) is observed, consistent with the reduced stability of the higher OGN charge states [<a href="#rcm10033-bib-0052" class="usa-link" aria-describedby="rcm10033-bib-0052">52</a>, <a href="#rcm10033-bib-0053" class="usa-link" aria-describedby="rcm10033-bib-0053">53</a>]. The greater coulombic repulsion of the higher charge states leads to significantly lower values for the critical energy of fragmentation.</p>
<figure class="fig xbox font-sm" id="rcm10033-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3026/11969060/30f0f1e83684/RCM-39-e10033-g005.jpg" loading="lazy" id="jats-graphic-1" height="754" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10033-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effects of in‐source energy on alkylamine adducts and in‐source fragmentation of OGN1 (Patisiran‐antisense siRNA). (A) HESI MS spectra at a range of in‐source <span class="font-variant-small-caps">ce</span> (0, 20, 25, 30, 45, 55, 60 and 65 V). (B) HESI MS spectra of the −4 charge state of OGN1 at 45 V and 60 V. Inset panels highlight the alkylamine adducts and corresponding OGN fragmentation and base loss.</p></figcaption></figure><p>LC‐MS spectra are shown for representative in‐source <span class="font-variant-small-caps">ce</span> settings of 45 V and 65 V for the corresponding 4− charge state (see Figure <a href="#rcm10033-fig-0001" class="usa-link">1B</a>). The results show the under optimised <span class="font-variant-small-caps">ce</span> settings of 45 V, this results in the effective reduction of HA adducts and no significant in‐source OGN fragmentation (base loss) is observed. However, at higher <span class="font-variant-small-caps">ce</span> voltages, increased OGN fragmentation (base loss) is observed, therefore generating significant in‐source impurities.</p></section><section id="rcm10033-sec-0014"><h3 class="pmc_sec_title">3.2. Effect of HESI and Source Temperature on Alkylamine Adduct Removal in Conjunction With In‐Source Collision Energy</h3>
<p>The results above demonstrated that using HA/HFIP mobile phase with optimised in‐source <span class="font-variant-small-caps">ce</span> voltages minimised LC‐MS generated impurities (HA adducts and OGN fragmentation). Therefore, additional experiments were performed to study the effects of additional HESI parameters including ion transfer tube (ITT) temperature and vaporiser temperatures (VT) in conjunction with in‐source <span class="font-variant-small-caps">ce</span> on the Orbitrap Exploris mass spectrometer.</p>
<p>A range of different ITT temperatures (150 °C, 250 °C, 275 °C and 350 °C) were studied at a constant VT of 300 °C and <span class="font-variant-small-caps">ce</span> set at 30 V (see Figure <a href="#rcm10033-fig-0002" class="usa-link">2A</a>). The results show that for the analysis of OGN2 (Patisiran‐sense siRNA) as ITT temperature is increased, alkylamine adducts present on the lower charge states (3−, 4−) decreased and no significant OGN fragmentation was observed across all ITT temperatures. The effect of VT was also investigated, ITT temperature was kept constant at 275 °C and in‐source <span class="font-variant-small-caps">ce</span> at 30 V and the VT was varied at 150 °C, 250 °C, 300 °C and 350 °C (see Figure <a href="#rcm10033-fig-0002" class="usa-link">2B</a>). The results show that similar to previous alterations in ITT temperature, as the VT increases, the abundance of alkylamine adducts present on the lower charge states (3−, 4−) decreases and no significant OGN fragmentation was observed across all vaporisation temperatures used shown in Figure <a href="#rcm10033-fig-0002" class="usa-link">2B</a>. The results show that at lower vaporisation temperature (150 °C) significantly higher alkylamine adducts were present compared to the lowest ITT temperature (150 °C) used. The VT also impacts the observed charge state distribution of the OGN, the results show that the −8, −7 charge states were only observed above 300 °C for OGN2 under the conditions analysed (see Figure <a href="#rcm10033-fig-0004" class="usa-link">4B</a>). These results were consistent with the analysis of OGN4 across varying ITT temperatures and vaporiser temperatures (see Supplementary Figure <a href="#rcm10033-supitem-0001" class="usa-link">S3A/B</a>).</p>
<figure class="fig xbox font-sm" id="rcm10033-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3026/11969060/77812ff61d41/RCM-39-e10033-g007.jpg" loading="lazy" id="jats-graphic-3" height="1187" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10033-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effects of ITT temperature and VT on alkylamine adducts and in‐source fragmentation. (A) HESI MS spectra of OGN2 (Patisiran‐sense siRNA) over a range of ITT temperatures (150 °C, 250 °C, 275 °C and 350 °C) at constant VT of 300 °C and <span class="font-variant-small-caps">ce</span> set at 30 V. (B) HESI MS spectra of OGN2 (Patisiran‐sense siRNA) over a range of VT (150 °C, 250 °C, 300 °C and 350 °C) at constant ITT temperature of 275 °C and <span class="font-variant-small-caps">ce</span> set at 30 V.</p></figcaption></figure><figure class="fig xbox font-sm" id="rcm10033-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3026/11969060/244965b04f31/RCM-39-e10033-g001.jpg" loading="lazy" id="jats-graphic-7" height="867" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10033-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effects of in‐source energy on alkylamine adducts and in‐source fragmentation of OGN3 (Tofersen). (A) HESI MS spectra at a range of in‐source <span class="font-variant-small-caps">ce</span> (0, 20, 25, 30, 45, 55, 60 and 65 V). (B) % Fractional abundance of the alkyamine adducts and base loss are shown for varying in‐source voltages following deconvolution of charge states −4 and above.</p></figcaption></figure><p>The results show that at higher ITT and vaporiser temperatures, less alkylamine adducts are observed on the lower charge states, consistent with a reduction in alkylamine binding to the phosphodiester backbone of the OGN. The alkylamine‐OGN complexes are not amenable for further deprotonation by counter ion in the gas phase, leading to lower charge states. Increasing source temperature reduces the alkylamine‐OGN complexes enabling further deprotonation of the free OGN by counterion in the gas phase, leading to high charge states [<a href="#rcm10033-bib-0046" class="usa-link" aria-describedby="rcm10033-bib-0046">46</a>]. Alternative proposed mechanisms can also explain the effects observed as linear alkylamines (such as HA) are proposed to keep the OGNs in a more folded conformation due to neutralisation of the phosphate backbone [<a href="#rcm10033-bib-0046" class="usa-link" aria-describedby="rcm10033-bib-0046">46</a>]. This, therefore, results in less charge repulsion; HA therefore enhances ionisation via the charge residual model (CRM) as opposed to the chain ejection model (CEM) [<a href="#rcm10033-bib-0054" class="usa-link" aria-describedby="rcm10033-bib-0054">54</a>], which results in lower charge states. This proposed mechanism is consistent with data where at lower vaporiser/ITT temperatures, increased amounts of HA are bound to the OGN resulting in neutralisation of the phosphate backbone enhancing ionisation via the CRM resulting in lower charge states. At higher ITT and VT (more denaturing conditions), reduced amounts of HA are bound to the OGN generating more highly charged OGNs that unfold and promote ionisation via the CEM resulting in higher charge states.</p>
<p>However, this is in contrast to previous data where an increase in higher charge states using octylamine:nonafluoro‐tert‐butyl alcohol was observed at lower source temperature conditions, 80 °C and 100 °C compared 120 °C [<a href="#rcm10033-bib-0055" class="usa-link" aria-describedby="rcm10033-bib-0055">55</a>]. This supports the idea that these HESI droplet microenvironments can be significantly affected by the mobile phase and source conditions, resulting in changes to the observed HESI mass spectra [<a href="#rcm10033-bib-0056" class="usa-link" aria-describedby="rcm10033-bib-0056">56</a>].</p>
<p>Following high‐resolution mass spectrometry analysis of the OGN and optimisation of the in‐source settings, a rapid streamlined mass deconvolution and data analysis workflow was developed in Chromeleon to identify and quantify impurities present based on HRAMS analysis (see Supplementary Figure <a href="#rcm10033-supitem-0001" class="usa-link">S2</a>).</p></section><section id="rcm10033-sec-0015"><h3 class="pmc_sec_title">3.3. Deconvolution Parameters for Identification and Quantification of OGN Impurity Masses</h3>
<p>From the data and optimisation described above, in‐source settings can influence the charge state distribution, alkylamine adduct removal and in‐source impurity generation. The ultimate goal would be to remove all the alkylamine adducts while preventing any generation of in‐source impurities. This is dependent on the OGN sequence and structure and usually cannot be achieved for the lower charge states. The higher charge states on the other hand, either do not have any adducts to begin with or they are removed much more easily. Using optimised source conditions and higher eluent pH values to generate the higher charge states, enables the utilisation of these higher charges states for deconvolution whilst ignoring the lower charge states that carry the adducts. There are two parameters in the deconvolution settings which can be exploited to focus the deconvolution on ignoring potential adducts in the final results. This allows milder source conditions to be employed which prevent in‐source generated impurities but may still leave some adducts present on the lower charge states.</p>
<p>Having multiple higher charge states available for deconvolution prevents the necessity of requiring the lowest charge state for deconvolution. The first parameter that can be employed is the m/z range, this can be set so that the higher value falls below the m/z of the lowest charge state. This would effectively put the lowest charge state out of the m/z range of the deconvolution. The second setting is the minimum number of detected charges. This specifies the minimum number of charge states required to produce a deconvolution result. No deconvolution results present with less than this minimum number of charge states appear in the results list. Which means if this is set to a value of 2, if a single low charge state is the only charge state that contains adducts, the non‐adduct m/z for this charge state would be used in the deconvolution but the m/z values from the adducts would not appear in the deconvoluted results as they are only present under one charge state.</p>
<p>Using these parameters for deconvolution allows mild source settings which may still leave adduct formation on some low charge states, but there would be no in‐source generation of impurities. This is important as source generation of abasic impurities would leave doubt about the impurity being present in the original sample or not. Figure <a href="#rcm10033-fig-0001" class="usa-link">1A</a> shows the full charge state spectrum of OGN1 analysed with an in‐source <span class="font-variant-small-caps">ce</span> of 45 V. The alkylamine adducts are clearly visible on the low charge state [−4 and −3] but are absent in all the other charge states. Using the above‐described parameters in the deconvolution settings, the deconvoluted results show all the impurities expected from the sample without any adducts or in‐source generated impurities. With optimisation of the source conditions to focus on prevention of in‐source impurity generation, whilst removing as many adducts as possible, the software parameters can deal with any remaining amine adducts.</p>
<p>Following mass deconvolution as described above a summary of the quantification of the HA adducts and OGN base loss over a range of source <span class="font-variant-small-caps">ce</span> for OGN1 is shown in Figure <a href="#rcm10033-fig-0003" class="usa-link">3A</a>. Consistent with LC‐MS spectra previously observed, the results show that alkylamine adducts are only present at low <span class="font-variant-small-caps">ce</span> voltages. As <span class="font-variant-small-caps">ce</span> is increased, there is an increase in the % abundance of in‐source generated impurities where gas‐phase base loss of A and G are the most abundant. Under these conditions, an optimum <span class="font-variant-small-caps">ce</span> of 45 V is determined for effective alkylamine adduct removal and with no significant OGN fragmentation (base loss observed) resulting in the highest % fractional abundance of the OGN target mass and therefore minimum in‐source generated impurities.</p>
<figure class="fig xbox font-sm" id="rcm10033-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3026/11969060/2e2ea013af2a/RCM-39-e10033-g003.jpg" loading="lazy" id="jats-graphic-5" height="839" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10033-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Deconvolution parameters for the quantification of OGNs and their related impurities. (A) % Fractional abundance of the alkyamine adducts and gas‐phase base loss are shown for varying in‐source voltages following deconvolution of charge states −4 and above. (B) % Fractional abundance of an OGN and gas‐phase base loss are shown for a range of ITT temperatures (left) and a range of VT (right), following deconvolution of charge states −4 and above. (C) % fractional abundance of OGN impurities. Five replicate injections of a phosphorothioated OGN are shown and deconvolution was performed using charge states −4 and above. (D) Comparative analysis of the intensity of the PO impurity (0–3%) spiked into a phosphorothioated OGN. Deconvolution was performed using charge states −4 and above (shown in orange) and only the most abundant charge state (−4 shown in blue).</p></figcaption></figure><p>In addition, a summary of the quantification of the HA adducts and OGN base loss over the range of different source conditions following mass deconvolution is shown in Figure <a href="#rcm10033-fig-0003" class="usa-link">3B</a> and Supplementary Figure <a href="#rcm10033-supitem-0001" class="usa-link">S3C</a>. The results highlight the decrease in alkyamine adducts at higher ITT and VTs as described above.</p>
<p>The ability to detect low level impurities in an OGN sample using the streamlined deconvolution settings is demonstrated in Figure <a href="#rcm10033-fig-0003" class="usa-link">3C</a>. The results show a series of replicate 1 μg injections of a phophorothioated OGN and the fractional abundance of the FLP and low level impurities including the PO. Further analysis using the deconvolution parameters excluding the lowest charge state was compared to deconvolution using only the most abundant charge state (4−) for the quantification of low level PO impurities. OGN samples were generated using a phosphorothioated OGN with the addition of a PO impurity spiked in (0.02 to 3%). Quantification of the PO impurity was determined using both of the deconvolution approaches described above (see Figure <a href="#rcm10033-fig-0003" class="usa-link">3D</a>). The results show consistent linearity and similar relative intensities determined for the PO impurities across the different deconvolution settings.</p></section><section id="rcm10033-sec-0016"><h3 class="pmc_sec_title">3.4. Optimisation of LC HESI MS Source Conditions for the Analysis of ASO OGNs</h3>
<p>Further LC‐MS analysis was performed using an alternative OGN therapeutic (Tofersen‐a 20 nt OGN with 2’ OMe, 2’ MOE and PS modifications) see Table <a href="#rcm10033-tbl-0001" class="usa-link">1</a> OGN3. The OGN was analysed using LC‐MS under a range of different in‐source collision energies as previously described (see Figure <a href="#rcm10033-fig-0004" class="usa-link">4A,B</a>). The results show that under these conditions, the predominant charge state observed was 4−. The results show the presence of HA adducts on the low charge states at low <span class="font-variant-small-caps">ce</span> settings when no in‐source <span class="font-variant-small-caps">ce</span> is applied (see Figure <a href="#rcm10033-fig-0004" class="usa-link">4A</a>). The HA adducts are effectively removed as the in‐source <span class="font-variant-small-caps">ce</span> increases with a reduction in signal intensity of the higher charge states (9− to −7) consistent with previous observations. Following mass deconvolution as previously described, a summary of the quantification of the HA adducts and OGN base loss over a range of source <span class="font-variant-small-caps">ce</span> is shown in Figure <a href="#rcm10033-fig-0004" class="usa-link">4B</a>. Under these conditions for the analysis of Tofersen, the optimum <span class="font-variant-small-caps">ce</span> where a reduction of HA adducts and minimising OGN fragmentation (generation of in‐source impurities) was obtained at 30 V as highlighted by the highest fractional abundance of the target mass of the OGN FLP (see Figure <a href="#rcm10033-fig-0002" class="usa-link">2B</a>). At high collision energies (&gt;30 V) m/z values corresponding to the [M‐ 4H‐B(A)]<sup>4−</sup> was the predominant species for this OGN (see Figure <a href="#rcm10033-fig-0002" class="usa-link">2B</a> and Supplementary Figure <a href="#rcm10033-supitem-0001" class="usa-link">S1B</a>). The abundant neutral base loss (A) observed for this OGN at high in‐source <span class="font-variant-small-caps">ce</span> is in contrast to the most abundant charged base loss of G which was observed for the siRNA at high in‐source <span class="font-variant-small-caps">ce</span>.</p>
<p>Finally, LC‐MS analysis of a fully phosphorothioated OGN (OGN4) was studied over a range of in‐source <span class="font-variant-small-caps">ce</span> settings (see Supplementary Figure <a href="#rcm10033-supitem-0001" class="usa-link">S4A/B</a>). As previously observed as in‐source <span class="font-variant-small-caps">ce</span> is increased, a decrease in HA adducts is observed which are present predominately on the lower charge states. At high <span class="font-variant-small-caps">ce</span> voltages (60 V), a clear reduction or removal of the adducts is observed in conjunction with a significant change in charge series compared to low or 0 V. Significant base loss was observed at high <span class="font-variant-small-caps">ce</span>, as an m/z value corresponding to the [M‐4H‐B(A)]<sup>4−</sup> was observed, demonstrating neutral base loss (predominantly A) (see Supplementary Figure <a href="#rcm10033-supitem-0001" class="usa-link">S1C</a> and 4B).</p>
<p>In summary, the results show that analysis of a range of OGNs using HA/HFIP mobile phases in conjunction with optimised <span class="font-variant-small-caps">ce</span> voltages and streamlined accurate mass deconvolution enables the identification and quantification of OGNs and their related impurities. Under optimised (mild) in‐source <span class="font-variant-small-caps">ce</span> settings enables a reduction in the LC‐MS generated impurities via the presence of HA adducts from the mobile phase and in‐source generated impurities from the OGN fragmentation (gas phase base loss). Optimisation of the appropriate <span class="font-variant-small-caps">ce</span> settings is dependent on the sequence and chemical composition of the OGN. The LC‐MS data indicate that the generation of in‐source impurities is dependent on the stability of the OGN in the gas phase. The results shows the enhanced stability of Patisiran ‐antisense siRNA and Tofersen compared to fully phosphorothioated ASO (ssDNA). It has been shown previously that base loss is less prevalent in RNA due to the 2′‐hydroxyl on the ribose stabilising the N‐glycosidic bond [<a href="#rcm10033-bib-0057" class="usa-link" aria-describedby="rcm10033-bib-0057">57</a>]. Furthermore, proposed mechanisms for base loss, such as neutral loss involves the 2′ hydrogen [<a href="#rcm10033-bib-0058" class="usa-link" aria-describedby="rcm10033-bib-0058">58</a>]. Therefore, the increased gas phase stability for Patisiran‐antisense siRNA and Tofersen compared to fully phosphorothioated ASO can be attributed to the 2′ modifications of the ribose sugars, stabilising the glycosidic bond and preventing subsequent base loss. These findings are consistent with previous studies in which MOE modifications improve stability under harsher conditions in terms of drying gas flow rate and temperature [<a href="#rcm10033-bib-0022" class="usa-link" aria-describedby="rcm10033-bib-0022">22</a>].</p></section><section id="rcm10033-sec-0017"><h3 class="pmc_sec_title">3.5. Comparative LC MS Analysis of OGNs Using HA/HFIP and TBuAA Mobile Phases</h3>
<p>Following optimisation of the HESI source settings and streamlined deconvolution methods for impurity profiling, a comparison was performed against the commonly used TBuAA mobile phase [<a href="#rcm10033-bib-0022" class="usa-link" aria-describedby="rcm10033-bib-0022">22</a>]. OGN3 was analysed using 5 mM TBuAA and compared against the HA/HFIP (see Figure <a href="#rcm10033-fig-0005" class="usa-link">5</a>). For the analysis using TBuAA (neutral pH) across a range of different in‐source <span class="font-variant-small-caps">ce</span>, the expected shift to the lower charge states is observed (see Figure <a href="#rcm10033-fig-0005" class="usa-link">5A</a>). The comparative data of the same source settings for the analysis of the OGN in TBuAA and HA/HFIP is shown in Figure <a href="#rcm10033-fig-0005" class="usa-link">5B</a>. It is interesting to note that the abundance of the 5− charge state in HA/HFIP is of similar intensity to the most abundant 4− charge state observed in 5 mM TBuAA. Figure <a href="#rcm10033-fig-0005" class="usa-link">5C</a> shows a comparison of the total signal intensity and shows that there is an increase in signal intensity achieved using the HA/HFIP mobile phases in comparison to the TBuAA. The results also demonstrate the increased abundance of the higher charge states using HA/HFIP, where alkylamine and metal ion adducts are less prevalent.</p>
<figure class="fig xbox font-sm" id="rcm10033-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3026/11969060/6cec957648f9/RCM-39-e10033-g004.jpg" loading="lazy" id="jats-graphic-9" height="981" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10033-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative LC‐MS analysis of OGN3 (Tofersen) using TBuAA and HA/HFIP. (A) HESI MS spectra at a range of in‐source <span class="font-variant-small-caps">ce</span> (0, 25, 30, 45, 55 and 65 V) using TBuAA. (B) HESI MS spectra using in‐source <span class="font-variant-small-caps">ce</span> 30 V for HA/HFIP (left) and TBuAA (right). (C) Comparative sum intensity of the HESI MS spectra of OGN3 (Tofersen). One hundred picomoles of OGN2 was analysed and normalised against UV peak area. (D) % Fractional abundance of the alkyamine adducts are shown for varying in‐source voltages following deconvolution of charge states −4 and above in TBuAA.</p></figcaption></figure><p>The data also shows that in the absence or low in‐source <span class="font-variant-small-caps">ce</span>, differences are observed in the abundance of alkylamine adducts. Focussing on the 4− charge state, increased abundance (and multiple) HA adducts are observed in comparison to the TBA and TBA oxide adducts observed. The increased abundance of the HA adducts likely reflects the higher concentration of HA used compared to TBA (15 mM vs 5 mM respectively). Therefore, these results also demonstrate the importance of optimising HESI parameters as outlined previously using the HA/HFIP mobile phases to effectively remove the HA adducts whilst minimising in‐source fragmentation and taking advantage of the benefits of the higher charge states observed and increased signal intensity (ionisation efficiency). Finally, the comparison also shows that as <span class="font-variant-small-caps">ce</span> is increased, the HA adducts are effectively removed from the −4 charge state at 45 V (see comparison with 0 V data) and are not present on the higher charge states across the majority of <span class="font-variant-small-caps">ce</span> used. In contrast, the TBA and TBA oxide adducts are still present on the −4 charge state at 45 V and above (see Figure <a href="#rcm10033-fig-0005" class="usa-link">5D</a> and Supplementary Figure <a href="#rcm10033-supitem-0001" class="usa-link">S6A‐C</a>).</p></section><section id="rcm10033-sec-0018"><h3 class="pmc_sec_title">3.6. Effect of In‐Source Collision Energy on Adduct Levels in Positive Mode Ionisation</h3>
<p>Negative mode ionisation HESI is predominantly used for analysing OGNs owing to their polyanionic nature, resulting in improved ionisation efficiency and greater signal intensity. However, positive mode HESI has been demonstrated for the analysis of OGNs [<a href="#rcm10033-bib-0059" class="usa-link" aria-describedby="rcm10033-bib-0059">59</a>, <a href="#rcm10033-bib-0060" class="usa-link" aria-describedby="rcm10033-bib-0060">60</a>]. Applications of positive mode HESI MS include the analysis of Morpholino antisense OGNs (absence of negatively charged phosphodiester backbone) which favours positive ion formation [<a href="#rcm10033-bib-0061" class="usa-link" aria-describedby="rcm10033-bib-0061">61</a>]. In this study, we also used positive ion mode LC HESI MS to analyse the OGN therapeutics under the same mobile phase conditions as described above (15 mM HA, 60 mM HFIP) and analysed the effect of in‐source collision energies on alkylamine adducts and in‐source fragmentation of the OGN. Figure <a href="#rcm10033-fig-0006" class="usa-link">6</a> and Supplementary Figure <a href="#rcm10033-supitem-0001" class="usa-link">S7</a> show the analysis of OGN1, 4 and 5 analysed in positive mode at in‐source collision energies of 30–80 V. The results for the analysis of OGN4 demonstrate that in positive mode, additional alkylamine adducts are present on the limited lower charge states observed compared to negative mode, generating significant polydispersity of the OGN signal. Higher in‐source collision energies are required to remove the alkylamine adducts. The results show that as <span class="font-variant-small-caps">ce</span> was increased, the amount of adducts present decreased, following the trend demonstrated in negative mode.</p>
<figure class="fig xbox font-sm" id="rcm10033-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3026/11969060/57e083c0207e/RCM-39-e10033-g006.jpg" loading="lazy" id="jats-graphic-11" height="1131" width="709" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10033-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>LC‐MS analysis in positive ion mode. (A) HESI MS spectra of OGN4 (21 mer ssDNA phosphorothioate) over a range of in‐source <span class="font-variant-small-caps">ce</span> (30–80 V). (B) HESI MS spectra of the −3 charge state of OGN3 at 45 and 65 V respectively. Inset panels highlight the alkylamine adducts and corresponding OGN fragmentation and base loss. (C) % Fractional abundance of the alkyamine adducts and gas‐phase fragmentation (base loss) in positive ionisation mode. Varying in‐source <span class="font-variant-small-caps">ce</span> (30 V, 45 V, 65 V and 80 V) were used for the analysis of OGN 3 (ssDNA phosphorothioate) prior to deconvolution and data analysis to determine the % fractional abundance of the alkyamine adducts and gas‐phase fragmentation (base loss).</p></figcaption></figure><p>Similar to previous observations in negative ion mode, the main form of fragmentation is base loss, with the purines again the most labile, neutral base loss (A) was the most abundant base loss observed for this OGN. However, greater stability is seen in positive mode, and therefore, higher in‐source <span class="font-variant-small-caps">ce</span> voltages can be applied before similar levels of base loss in negative mode are seen. This is clearly demonstrated with both the qualitative and quantitative analysis, where significant base loss only beginning to be seen at 65 V in positive mode. In addition, at 45 V, which was the optimum condition in negative mode ionisation for OGN4 (see Supplementary Figure <a href="#rcm10033-supitem-0001" class="usa-link">S4</a>), in positive mode, a significant amount of adducts are still present at this voltage. For positive mode, the optimum conditions were between 65 and 70 V, which saw the majority of alkylamine adducts removed and minimising the generation of in‐source impurities (see Figure <a href="#rcm10033-fig-0006" class="usa-link">6</a>). In negative mode, high voltages such as this would cause significant in‐source impurities. It is interesting to note that under optimised <span class="font-variant-small-caps">ce</span> conditions for both positive and negative HESI MS, there is no significant difference in signal intensity for the same amount for OGN analysed.</p></section></section><section id="rcm10033-sec-0019"><h2 class="pmc_sec_title">4. Conclusions</h2>
<p>OGN therapeutics are an emerging class of drugs that have the potential to treat a wide range of diseases. Characterisation of OGN therapeutics and their related impurities is predominantly performed using IP‐RP HPLC interfaced with mass spectrometry. In this study, we have developed a sensitive LC‐MS method for analysis of antisense OGNs and small interfering RNAs utilising HA/HFIP mobile phases in conjunction with optimised mild HESI source conditions to minimise in‐source generated impurities and alkylamine adducts in the analysis of OGN therapeutics.</p>
<p>Source conditions were optimised to ensure efficient removal of alkylamine adducts whilst minimising in‐source fragmentation of OGNs. We have studied the effect of altering a range of HESI parameters on the formation of alkylamine adducts and the production of source‐related impurities in a wide range of different OGNs, including ASOs and single stranded siRNAs. Optimisation of the appropriate source settings was shown to be dependent on the sequence and chemical composition of the OGN. The results show that under low or no in‐source <span class="font-variant-small-caps">ce</span> the presence of HA adducts are observed and are predominantly on the lowest charge states present. As the in‐source <span class="font-variant-small-caps">ce</span> is increased, a reduction of HA adducts is observed. However, as in‐source <span class="font-variant-small-caps">ce</span> increases, this causes an increase in nucleobase loss in the gas phase, therefore generating in‐source impurities.</p>
<p>Utilising HA/HFIP mobile phase results in increased overall HESI MS sensitivity compared to TBuAA mobile phase under the conditions used in this study. Using optimised source conditions in conjunction with the higher eluent pH of HA/HFIP mobile phases, results in higher charge states, which can be utilised for deconvolution whilst ignoring the lower charge states that carry the adducts.</p>
<p>Further analysis of the OGNs using HA/HFIP mobile phases in positive ion mode showed that additional alkylamine adducts are present on the limited lower charge states observed compared to negative mode, generating significant polydispersity of the OGN signal. Higher in‐source collision energies are required to remove the alkylamine adducts compared to those optimised in negative ion mode for the same corresponding OGN. However, greater stability is seen in the positive mode, and therefore, higher in‐source <span class="font-variant-small-caps">ce</span> voltages can be applied before similar levels of base loss in the negative mode are seen. Comparing positive and negative HESI MS, there was no significant difference in signal intensity under optimised conditions for the same amount of OGN analysed.</p>
<p>Optimisation of the mild source conditions in conjunction with high pH mobile phases was combined with high‐resolution accurate mass spectrometry analysis and automated deconvolution workflow. This simplified and streamlined approach for characterising OGN therapeutics and their related impurities, without any significant adducts present in the deconvoluted provides a rapid universal method for impurity analysis with automated GLP compliant data handling.</p></section><section id="rcm10033-sec-0023"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>
<strong>Mollie A. Glenister:</strong> investigation, writing – original draft, writing – review and editing, methodology. <strong>Ulrik Mistarz:</strong> investigation, software, writing – review and editing. <strong>Ken Cook:</strong> investigation, supervision, writing – review and editing, conceptualization. <strong>James L. Stephenson:</strong> writing – review and editing, supervision. <strong>Mark J. Dickman:</strong> conceptualization, funding acquisition, writing – original draft, writing – review and editing, supervision.</p></section><section id="rcm10033-sec-0024"><h3 class="pmc_sec_title">Peer Review</h3>
<p>The peer review history for this article is available at <a href="https://www.webofscience.com/api/gateway/wos/peer-review/10.1002/rcm.10033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.webofscience.com/api/gateway/wos/peer‐review/10.1002/rcm.10033</a>.</p></section><section id="sec23"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="rcm10033-supitem-0001"><div class="caption p"><p>
<strong>Data S1</strong> Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11969060/bin/RCM-39-e10033-s001.pptx" data-ga-action="click_feat_suppl" class="usa-link">RCM-39-e10033-s001.pptx</a><sup> (10.4MB, pptx) </sup>
</div></div></section></section><section id="rcm10033-sec-0020" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>MJD acknowledges support from the Biotechnology and Biological Sciences Research Council UK (BBSRC) (BB/M012166/1). MG is a funded on an Engineering and Physical Sciences Research Council (EPSRC) Doctoral Training Partnership CASE Conversion award in collaboration with ThermoFisher (EP/W524360/1).</p></section><section id="notes1"><section id="rcm10033-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="rcm10033-note-0001"><p>
<strong>Funding:</strong> MJD acknowledges support from the Biotechnology and Biological Sciences Research Council UK (BBSRC) (BB/M012166/1). MG is a funded on an Engineering and Physical Sciences Research Council (EPSRC) Doctoral Training Partnership CASE Conversion award in collaboration with ThermoFisher (EP/W524360/1).</p></div></div></section></section><section id="rcm10033-sec-0022"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The data that support the findings of this study are available from the corresponding author upon reasonable request.</p></section><section id="rcm10033-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="rcm10033-bibl-0001_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="rcm10033-bib-0001">
<span class="label">1.</span><cite>
Brentari I., Zadorozhna M., Denti M. A., and Giorgio E., “RNA Therapeutics for Neurological Diseases,” British Medical Bulletin
147, no. 1 (2023): 50–61, 10.1093/bmb/ldad010.
</cite> [<a href="https://doi.org/10.1093/bmb/ldad010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37210633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=British%20Medical%20Bulletin&amp;title=RNA%20Therapeutics%20for%20Neurological%20Diseases&amp;author=I.%20Brentari&amp;author=M.%20Zadorozhna&amp;author=M.%20A.%20Denti&amp;author=E.%20Giorgio&amp;volume=147&amp;issue=1&amp;publication_year=2023&amp;pages=50-61&amp;pmid=37210633&amp;doi=10.1093/bmb/ldad010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0002">
<span class="label">2.</span><cite>
Egli M. and Manoharan M., “Chemistry, Structure and Function of Approved Oligonucleotide Therapeutics,” Nucleic Acids Research
51, no. 6 (2023): 2529–2573, 10.1093/nar/gkad067.
</cite> [<a href="https://doi.org/10.1093/nar/gkad067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10085713/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36881759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Chemistry,%20Structure%20and%20Function%20of%20Approved%20Oligonucleotide%20Therapeutics&amp;author=M.%20Egli&amp;author=M.%20Manoharan&amp;volume=51&amp;issue=6&amp;publication_year=2023&amp;pages=2529-2573&amp;pmid=36881759&amp;doi=10.1093/nar/gkad067&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0003">
<span class="label">3.</span><cite>
Al Shaer D., Al Musaimi O., Albericio F., and de la Torre B. G., “2023 FDA TIDES (Peptides and Oligonucleotides) Harvest,” Pharmacy (Basel, Switzerland)
17, no. 2 (2024): 243, 10.3390/ph17020243.</cite> [<a href="https://doi.org/10.3390/ph17020243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10893093/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38399458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacy%20(Basel,%20Switzerland)&amp;title=2023%20FDA%20TIDES%20(Peptides%20and%20Oligonucleotides)%20Harvest&amp;author=D.%20Al%20Shaer&amp;author=O.%20Al%20Musaimi&amp;author=F.%20Albericio&amp;author=B.%20G.%20de%20la%20Torre&amp;volume=17&amp;issue=2&amp;publication_year=2024&amp;pages=243&amp;pmid=38399458&amp;doi=10.3390/ph17020243&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0004">
<span class="label">4.</span><cite>
Vinjamuri B. P., Pan J., and Peng P., “A Review on Commercial Oligonucleotide Drug Products,” Journal of Pharmaceutical Sciences
113, no. 7 (2024): 1749–1768, 10.1016/j.xphs.2024.04.021.
</cite> [<a href="https://doi.org/10.1016/j.xphs.2024.04.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38679232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Pharmaceutical%20Sciences&amp;title=A%20Review%20on%20Commercial%20Oligonucleotide%20Drug%20Products&amp;author=B.%20P.%20Vinjamuri&amp;author=J.%20Pan&amp;author=P.%20Peng&amp;volume=113&amp;issue=7&amp;publication_year=2024&amp;pages=1749-1768&amp;pmid=38679232&amp;doi=10.1016/j.xphs.2024.04.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0005">
<span class="label">5.</span><cite>
Moumné L., Marie A. C., and Crouvezier N., “Oligonucleotide Therapeutics: From Discovery and Development to Patentability,” Pharmaceutics.
14, no. 2 (2022): 260, 10.3390/pharmaceutics14020260.
</cite> [<a href="https://doi.org/10.3390/pharmaceutics14020260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8876811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35213992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmaceutics.&amp;title=Oligonucleotide%20Therapeutics:%20From%20Discovery%20and%20Development%20to%20Patentability&amp;author=L.%20Moumn%C3%A9&amp;author=A.%20C.%20Marie&amp;author=N.%20Crouvezier&amp;volume=14&amp;issue=2&amp;publication_year=2022&amp;pages=260&amp;pmid=35213992&amp;doi=10.3390/pharmaceutics14020260&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0006">
<span class="label">6.</span><cite>
Di Fusco D., Dinallo V., Marafini I., Figliuzzi M. M., Romano B., and Monteleone G., “Antisense Oligonucleotide: Basic Concepts and Therapeutic Application in Inflammatory Bowel Disease,” Frontiers in Pharmacology
10 (2019): 305, 10.3389/fphar.2019.00305.
</cite> [<a href="https://doi.org/10.3389/fphar.2019.00305" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6450224/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30983999/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Pharmacology&amp;title=Antisense%20Oligonucleotide:%20Basic%20Concepts%20and%20Therapeutic%20Application%20in%20Inflammatory%20Bowel%20Disease&amp;author=D.%20Di%20Fusco&amp;author=V.%20Dinallo&amp;author=I.%20Marafini&amp;author=M.%20M.%20Figliuzzi&amp;author=B.%20Romano&amp;volume=10&amp;publication_year=2019&amp;pages=305&amp;pmid=30983999&amp;doi=10.3389/fphar.2019.00305&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0007">
<span class="label">7.</span><cite>
Bennett C. F., Baker B. F., Pham N., Swayze E., and Geary R. S., “Pharmacology of Antisense Drugs,” Annual Review of Pharmacology and Toxicology
57, no. 1 (2017): 81–105, 10.1146/annurev-pharmtox-010716104846.</cite> [<a href="https://doi.org/10.1146/annurev-pharmtox-010716104846" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27732800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Pharmacology%20and%20Toxicology&amp;title=Pharmacology%20of%20Antisense%20Drugs&amp;author=C.%20F.%20Bennett&amp;author=B.%20F.%20Baker&amp;author=N.%20Pham&amp;author=E.%20Swayze&amp;author=R.%20S.%20Geary&amp;volume=57&amp;issue=1&amp;publication_year=2017&amp;pages=81-105&amp;pmid=27732800&amp;doi=10.1146/annurev-pharmtox-010716104846&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0008">
<span class="label">8.</span><cite>
Shen X. and Corey D. R., “Chemistry, Mechanism and Clinical Status of Antisense Oligonucleotides and Duplex RNAs,” Nucleic Acids Research
46, no. 4 (2018): 1584–1600, 10.1093/nar/gkx1239.
</cite> [<a href="https://doi.org/10.1093/nar/gkx1239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5829639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29240946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Chemistry,%20Mechanism%20and%20Clinical%20Status%20of%20Antisense%20Oligonucleotides%20and%20Duplex%20RNAs&amp;author=X.%20Shen&amp;author=D.%20R.%20Corey&amp;volume=46&amp;issue=4&amp;publication_year=2018&amp;pages=1584-1600&amp;pmid=29240946&amp;doi=10.1093/nar/gkx1239&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0009">
<span class="label">9.</span><cite>
Herdewijn P., “Heterocyclic Modifications of Oligonucleotides and Antisense Technology,” Antisense &amp; Nucleic Acid Drug Development
10, no. 4 (2000): 297–310, 10.1089/108729000421475.
</cite> [<a href="https://doi.org/10.1089/108729000421475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10984123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antisense%20&amp;%20Nucleic%20Acid%20Drug%20Development&amp;title=Heterocyclic%20Modifications%20of%20Oligonucleotides%20and%20Antisense%20Technology&amp;author=P.%20Herdewijn&amp;volume=10&amp;issue=4&amp;publication_year=2000&amp;pages=297-310&amp;pmid=10984123&amp;doi=10.1089/108729000421475&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0010">
<span class="label">10.</span><cite>
Dowdy S. F., “Overcoming Cellular Barriers for RNA Therapeutics,” Nature Biotechnology
35, no. 3 (2017): 222–229, 10.1038/nbt.3802.</cite> [<a href="https://doi.org/10.1038/nbt.3802" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28244992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Overcoming%20Cellular%20Barriers%20for%20RNA%20Therapeutics&amp;author=S.%20F.%20Dowdy&amp;volume=35&amp;issue=3&amp;publication_year=2017&amp;pages=222-229&amp;pmid=28244992&amp;doi=10.1038/nbt.3802&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0011">
<span class="label">11.</span><cite>
Smith C. I. E. and Zain R., “Therapeutic Oligonucleotides: State of the Art,” Annual Review of Pharmacology and Toxicology
59, no. 1 (2019): 605–630, 10.1146/annurev-pharmtox-010818021050.</cite> [<a href="https://doi.org/10.1146/annurev-pharmtox-010818021050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30285540/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Pharmacology%20and%20Toxicology&amp;title=Therapeutic%20Oligonucleotides:%20State%20of%20the%20Art&amp;author=C.%20I.%20E.%20Smith&amp;author=R.%20Zain&amp;volume=59&amp;issue=1&amp;publication_year=2019&amp;pages=605-630&amp;pmid=30285540&amp;doi=10.1146/annurev-pharmtox-010818021050&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0012">
<span class="label">12.</span><cite>
Monia B. P., Lesnik E. A., Gonzalez C., et al., “Evaluation of 2′‐Modified Oligonucleotides Containing 2′‐Deoxy Gaps as Antisense Inhibitors of Gene Expression,” Journal of Biological Chemistry
268, no. 19 (1993): 14514–14522.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/8390996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Evaluation%20of%202%E2%80%B2%E2%80%90Modified%20Oligonucleotides%20Containing%202%E2%80%B2%E2%80%90Deoxy%20Gaps%20as%20Antisense%20Inhibitors%20of%20Gene%20Expression&amp;author=B.%20P.%20Monia&amp;author=E.%20A.%20Lesnik&amp;author=C.%20Gonzalez&amp;volume=268&amp;issue=19&amp;publication_year=1993&amp;pages=14514-14522&amp;pmid=8390996&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0013">
<span class="label">13.</span><cite>
Geary R. S., Norris D., Yu R., and Bennett C. F., “Pharmacokinetics, Biodistribution and Cell Uptake of Antisense Oligonucleotides,” Advanced Drug Delivery Reviews
87 (2015): 46–51, 10.1016/j.addr.2015.01.008.
</cite> [<a href="https://doi.org/10.1016/j.addr.2015.01.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25666165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Advanced%20Drug%20Delivery%20Reviews&amp;title=Pharmacokinetics,%20Biodistribution%20and%20Cell%20Uptake%20of%20Antisense%20Oligonucleotides&amp;author=R.%20S.%20Geary&amp;author=D.%20Norris&amp;author=R.%20Yu&amp;author=C.%20F.%20Bennett&amp;volume=87&amp;publication_year=2015&amp;pages=46-51&amp;pmid=25666165&amp;doi=10.1016/j.addr.2015.01.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0014">
<span class="label">14.</span><cite>
Bost J. P., Barriga H., Holme M. N., et al., “Delivery of Oligonucleotide Therapeutics: Chemical Modifications, Lipid Nanoparticles, and Extracellular Vesicles,” ACS Nano
15, no. 9 (2021): 13993–14021, 10.1021/acsnano.1c05099.
</cite> [<a href="https://doi.org/10.1021/acsnano.1c05099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8482762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34505766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Nano&amp;title=Delivery%20of%20Oligonucleotide%20Therapeutics:%20Chemical%20Modifications,%20Lipid%20Nanoparticles,%20and%20Extracellular%20Vesicles&amp;author=J.%20P.%20Bost&amp;author=H.%20Barriga&amp;author=M.%20N.%20Holme&amp;volume=15&amp;issue=9&amp;publication_year=2021&amp;pages=13993-14021&amp;pmid=34505766&amp;doi=10.1021/acsnano.1c05099&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0015">
<span class="label">15.</span><cite>
Soutschek J., Akinc A., Bramlage B., et al., “Therapeutic Silencing of an Endogenous Gene by Systemic Administration of Modified siRNAs,” Nature
432, no. 7014 (2004): 173–178, 10.1038/nature03121.
</cite> [<a href="https://doi.org/10.1038/nature03121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15538359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Therapeutic%20Silencing%20of%20an%20Endogenous%20Gene%20by%20Systemic%20Administration%20of%20Modified%20siRNAs&amp;author=J.%20Soutschek&amp;author=A.%20Akinc&amp;author=B.%20Bramlage&amp;volume=432&amp;issue=7014&amp;publication_year=2004&amp;pages=173-178&amp;pmid=15538359&amp;doi=10.1038/nature03121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0016">
<span class="label">16.</span><cite>
Betts C., Saleh A. F., Arzumanov A. A., et al., “Pip6‐PMO, A New Generation of Peptide‐Oligonucleotide Conjugates With Improved Cardiac Exon Skipping Activity for DMD Treatment,” Molecular Therapy Nucleic Acids
1 (2012): e38, 10.1038/mtna.2012.30.
</cite> [<a href="https://doi.org/10.1038/mtna.2012.30" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3438601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23344180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Therapy%20Nucleic%20Acids&amp;title=Pip6%E2%80%90PMO,%20A%20New%20Generation%20of%20Peptide%E2%80%90Oligonucleotide%20Conjugates%20With%20Improved%20Cardiac%20Exon%20Skipping%20Activity%20for%20DMD%20Treatment&amp;author=C.%20Betts&amp;author=A.%20F.%20Saleh&amp;author=A.%20A.%20Arzumanov&amp;volume=1&amp;publication_year=2012&amp;pages=e38&amp;pmid=23344180&amp;doi=10.1038/mtna.2012.30&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0017">
<span class="label">17.</span><cite>
Song E., Zhu P., Lee S. K., et al., “Antibody Mediated In Vivo Delivery of Small Interfering RNAs via Cell‐Surface Receptors,” Nature Biotechnology
23, no. 6 (2005): 709–717, 10.1038/nbt1101.</cite> [<a href="https://doi.org/10.1038/nbt1101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15908939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Antibody%20Mediated%20In%C2%A0Vivo%20Delivery%20of%20Small%20Interfering%20RNAs%20via%20Cell%E2%80%90Surface%20Receptors&amp;author=E.%20Song&amp;author=P.%20Zhu&amp;author=S.%20K.%20Lee&amp;volume=23&amp;issue=6&amp;publication_year=2005&amp;pages=709-717&amp;pmid=15908939&amp;doi=10.1038/nbt1101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0018">
<span class="label">18.</span><cite>
Brown C. R., Gupta S., Qin J., et al., “Investigating the Pharmacodynamic Durability of GalNAc–siRNA Conjugates,” Nucleic Acids Research
48, no. 21 (2020): 11827–11844, 10.1093/nar/gkaa670.
</cite> [<a href="https://doi.org/10.1093/nar/gkaa670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7708070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32808038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Investigating%20the%20Pharmacodynamic%20Durability%20of%20GalNAc%E2%80%93siRNA%20Conjugates&amp;author=C.%20R.%20Brown&amp;author=S.%20Gupta&amp;author=J.%20Qin&amp;volume=48&amp;issue=21&amp;publication_year=2020&amp;pages=11827-11844&amp;pmid=32808038&amp;doi=10.1093/nar/gkaa670&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0019">
<span class="label">19.</span><cite>
Debacker A. J., Voutila J., Catley M., Blakey D., and Habib N., “Delivery of Oligonucleotides to the Liver With GalNAc: From Research to Registered Therapeutic Drug,” Molecular Therapy
28, no. 8 (2020): 1759–1771, 10.1016/j.ymthe.2020.06.015.
</cite> [<a href="https://doi.org/10.1016/j.ymthe.2020.06.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7403466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32592692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Therapy&amp;title=Delivery%20of%20Oligonucleotides%20to%20the%20Liver%20With%20GalNAc:%20From%20Research%20to%20Registered%20Therapeutic%20Drug&amp;author=A.%20J.%20Debacker&amp;author=J.%20Voutila&amp;author=M.%20Catley&amp;author=D.%20Blakey&amp;author=N.%20Habib&amp;volume=28&amp;issue=8&amp;publication_year=2020&amp;pages=1759-1771&amp;pmid=32592692&amp;doi=10.1016/j.ymthe.2020.06.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0020">
<span class="label">20.</span><cite>
Li N., El Zahar N. M., Saad J. G., van der Hage E. R. E., and Bartlett M. G., “Alkylamine Ion‐Pairing Reagents and the Chromatographic Separation of Oligonucleotides,” Journal of Chromatography A
1580 (2018): 110–119, 10.1016/j.chroma.2018.10.040.
</cite> [<a href="https://doi.org/10.1016/j.chroma.2018.10.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30409418/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chromatography%20A&amp;title=Alkylamine%20Ion%E2%80%90Pairing%20Reagents%20and%20the%20Chromatographic%20Separation%20of%20Oligonucleotides&amp;author=N.%20Li&amp;author=N.%20M.%20El%20Zahar&amp;author=J.%20G.%20Saad&amp;author=E.%20R.%20E.%20van%20der%20Hage&amp;author=M.%20G.%20Bartlett&amp;volume=1580&amp;publication_year=2018&amp;pages=110-119&amp;pmid=30409418&amp;doi=10.1016/j.chroma.2018.10.040&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0021">
<span class="label">21.</span><cite>
Roussis S. G., Rodriguez A. A., and Rentel C., “Determination of Individual Oligonucleotide Impurities by Small Amine ion Pair‐RP HPLC‐MS and MS/MS: n − 1 Impurities,” Journal of Chromatography B
1169 (2021): 122611, 10.1016/j.jchromb.2021.122611.</cite> [<a href="https://doi.org/10.1016/j.jchromb.2021.122611" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33706187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chromatography%20B&amp;title=Determination%20of%20Individual%20Oligonucleotide%20Impurities%20by%20Small%20Amine%20ion%20Pair%E2%80%90RP%20HPLC%E2%80%90MS%20and%20MS/MS:%20n%20%E2%88%92%201%20Impurities&amp;author=S.%20G.%20Roussis&amp;author=A.%20A.%20Rodriguez&amp;author=C.%20Rentel&amp;volume=1169&amp;publication_year=2021&amp;pages=122611&amp;pmid=33706187&amp;doi=10.1016/j.jchromb.2021.122611&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0022">
<span class="label">22.</span><cite>
Rentel C., Gaus H., Bradley K., et al., “Assay, Purity, and Impurity Profile of Phosphorothioate Oligonucleotide Therapeutics by Ion Pair–HPLC–MS,” Nucleic Acid Therapeutics
32, no. 3 (2022): 206–220, 10.1089/nat.2021.0056.
</cite> [<a href="https://doi.org/10.1089/nat.2021.0056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35238617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acid%20Therapeutics&amp;title=Assay,%20Purity,%20and%20Impurity%20Profile%20of%20Phosphorothioate%20Oligonucleotide%20Therapeutics%20by%20Ion%20Pair%E2%80%93HPLC%E2%80%93MS&amp;author=C.%20Rentel&amp;author=H.%20Gaus&amp;author=K.%20Bradley&amp;volume=32&amp;issue=3&amp;publication_year=2022&amp;pages=206-220&amp;pmid=35238617&amp;doi=10.1089/nat.2021.0056&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0023">
<span class="label">23.</span><cite>
Roussis S. G., Koch C., Capaldi D., and Rentel C., “Rapid Oligonucleotide Drug Impurity Determination by Direct Spectral Comparison of ion‐Pair Reversed‐Phase High‐Performance Liquid Chromatography Electrospray Ionization Mass Spectrometry Data,” Rapid Communications in Mass Spectrometry
32, no. 14 (2018): 1099–1106, 10.1002/rcm.8125.</cite> [<a href="https://doi.org/10.1002/rcm.8125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29575209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rapid%20Communications%20in%20Mass%20Spectrometry&amp;title=Rapid%20Oligonucleotide%20Drug%20Impurity%20Determination%20by%20Direct%20Spectral%20Comparison%20of%20ion%E2%80%90Pair%20Reversed%E2%80%90Phase%20High%E2%80%90Performance%20Liquid%20Chromatography%20Electrospray%20Ionization%20Mass%20Spectrometry%20Data&amp;author=S.%20G.%20Roussis&amp;author=C.%20Koch&amp;author=D.%20Capaldi&amp;author=C.%20Rentel&amp;volume=32&amp;issue=14&amp;publication_year=2018&amp;pages=1099-1106&amp;pmid=29575209&amp;doi=10.1002/rcm.8125&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0024">
<span class="label">24.</span><cite>
El Zahar N. M., Magdy N., El‐Kosasy A. M., and Bartlett M. G., “Chromatographic Approaches for the Characterization and Quality Control of Therapeutic Oligonucleotide Impurities,” Biomedical Chromatography
32, no. 1 (2018): e4088, 10.1002/bmc.4088.</cite> [<a href="https://doi.org/10.1002/bmc.4088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28869310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomedical%20Chromatography&amp;title=Chromatographic%20Approaches%20for%20the%20Characterization%20and%20Quality%20Control%20of%20Therapeutic%20Oligonucleotide%20Impurities&amp;author=N.%20M.%20El%20Zahar&amp;author=N.%20Magdy&amp;author=A.%20M.%20El%E2%80%90Kosasy&amp;author=M.%20G.%20Bartlett&amp;volume=32&amp;issue=1&amp;publication_year=2018&amp;pages=e4088&amp;pmid=28869310&amp;doi=10.1002/bmc.4088&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0025">
<span class="label">25.</span><cite>
McGinnis A. C., Chen B., and Bartlett M. G., “Chromatographic Methods for the Determination of Therapeutic Oligonucleotides,” Journal of Chromatography B
883–884 (2012): 76–94, 10.1016/j.jchromb.2011.09.007.</cite> [<a href="https://doi.org/10.1016/j.jchromb.2011.09.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21945211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chromatography%20B&amp;title=Chromatographic%20Methods%20for%20the%20Determination%20of%20Therapeutic%20Oligonucleotides&amp;author=A.%20C.%20McGinnis&amp;author=B.%20Chen&amp;author=M.%20G.%20Bartlett&amp;volume=883%E2%80%93884&amp;publication_year=2012&amp;pages=76-94&amp;pmid=21945211&amp;doi=10.1016/j.jchromb.2011.09.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0026">
<span class="label">26.</span><cite>
Goyon A., Yehl P., and Zhang K., “Characterization of Therapeutic Oligonucleotides by Liquid Chromatography,” Journal of Pharmaceutical and Biomedical Analysis
182 (2020): 113105, 10.1016/j.jpba.2020.113105.
</cite> [<a href="https://doi.org/10.1016/j.jpba.2020.113105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32004766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Pharmaceutical%20and%20Biomedical%20Analysis&amp;title=Characterization%20of%20Therapeutic%20Oligonucleotides%20by%20Liquid%20Chromatography&amp;author=A.%20Goyon&amp;author=P.%20Yehl&amp;author=K.%20Zhang&amp;volume=182&amp;publication_year=2020&amp;pages=113105&amp;pmid=32004766&amp;doi=10.1016/j.jpba.2020.113105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0027">
<span class="label">27.</span><cite>
Hannauer F., Black R., Ray A. D., Stulz E., Langley G. J., and Holman S. W., “Advancements in the Characterisation of Oligonucleotides by High Performance Liquid Chromatography‐Mass Spectrometry in 2021: A Short Review,” Analytical Science Advances
3, no. 3–4 (2022): 90–102, 10.1002/ansa.202100066.
</cite> [<a href="https://doi.org/10.1002/ansa.202100066" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10989539/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38715636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Science%20Advances&amp;title=Advancements%20in%20the%20Characterisation%20of%20Oligonucleotides%20by%20High%20Performance%20Liquid%20Chromatography%E2%80%90Mass%20Spectrometry%20in%202021:%20A%20Short%20Review&amp;author=F.%20Hannauer&amp;author=R.%20Black&amp;author=A.%20D.%20Ray&amp;author=E.%20Stulz&amp;author=G.%20J.%20Langley&amp;volume=3&amp;issue=3%E2%80%934&amp;publication_year=2022&amp;pages=90-102&amp;pmid=38715636&amp;doi=10.1002/ansa.202100066&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0028">
<span class="label">28.</span><cite>
McGinnis A. C., Grubb E. C., and Bartlett M. G., “Systematic Optimization of Ion‐Pairing Agents and Hexafluoroisopropanol for Enhanced Electrospray Ionization Mass Spectrometry of Oligonucleotides,” Rapid Communications in Mass Spectrometry
27, no. 23 (2013): 2655–2664, 10.1002/rcm.6733.
</cite> [<a href="https://doi.org/10.1002/rcm.6733" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24591027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rapid%20Communications%20in%20Mass%20Spectrometry&amp;title=Systematic%20Optimization%20of%20Ion%E2%80%90Pairing%20Agents%20and%20Hexafluoroisopropanol%20for%20Enhanced%20Electrospray%20Ionization%20Mass%20Spectrometry%20of%20Oligonucleotides&amp;author=A.%20C.%20McGinnis&amp;author=E.%20C.%20Grubb&amp;author=M.%20G.%20Bartlett&amp;volume=27&amp;issue=23&amp;publication_year=2013&amp;pages=2655-2664&amp;pmid=24591027&amp;doi=10.1002/rcm.6733&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0029">
<span class="label">29.</span><cite>
Studzińska S., Rola R., and Buszewski B., “The Impact of ion‐Pairing Reagents on the Selectivity and Sensitivity in the Analysis of Modified Oligonucleotides in Serum Samples by Liquid Chromatography Coupled With Tandem Mass Spectrometry,” Journal of Pharmaceutical and Biomedical Analysis
138 (2017): 146–152, 10.1016/j.jpba.2017.02.014.
</cite> [<a href="https://doi.org/10.1016/j.jpba.2017.02.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28208106/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Pharmaceutical%20and%20Biomedical%20Analysis&amp;title=The%20Impact%20of%20ion%E2%80%90Pairing%20Reagents%20on%20the%20Selectivity%20and%20Sensitivity%20in%20the%20Analysis%20of%20Modified%20Oligonucleotides%20in%20Serum%20Samples%20by%20Liquid%20Chromatography%20Coupled%20With%20Tandem%20Mass%20Spectrometry&amp;author=S.%20Studzi%C5%84ska&amp;author=R.%20Rola&amp;author=B.%20Buszewski&amp;volume=138&amp;publication_year=2017&amp;pages=146-152&amp;pmid=28208106&amp;doi=10.1016/j.jpba.2017.02.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0030">
<span class="label">30.</span><cite>
Gilar M., Fountain K. J., Budman Y., Holyoke J. L., Davoudi H., and Gebler J. C., “Characterization of Therapeutic Oligonucleotides Using Liquid Chromatography With On‐Line Mass Spectrometry Detection,” Oligonucleotides
13, no. 4 (2003): 229–243, 10.1089/154545703322460612.
</cite> [<a href="https://doi.org/10.1089/154545703322460612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15000838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oligonucleotides&amp;title=Characterization%20of%20Therapeutic%20Oligonucleotides%20Using%20Liquid%20Chromatography%20With%20On%E2%80%90Line%20Mass%20Spectrometry%20Detection&amp;author=M.%20Gilar&amp;author=K.%20J.%20Fountain&amp;author=Y.%20Budman&amp;author=J.%20L.%20Holyoke&amp;author=H.%20Davoudi&amp;volume=13&amp;issue=4&amp;publication_year=2003&amp;pages=229-243&amp;pmid=15000838&amp;doi=10.1089/154545703322460612&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0031">
<span class="label">31.</span><cite>
Apffel A., Chakel J. A., Fischer S., Lichtenwalter K., and Hancock W. S., “Analysis of Oligonucleotides by HPLC−Electrospray Ionization Mass Spectrometry,” Analytical Chemistry
69, no. 7 (1997): 1320–1325, 10.1021/ac960916h.
</cite> [<a href="https://doi.org/10.1021/ac960916h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21639339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Analysis%20of%20Oligonucleotides%20by%20HPLC%E2%88%92Electrospray%20Ionization%20Mass%20Spectrometry&amp;author=A.%20Apffel&amp;author=J.%20A.%20Chakel&amp;author=S.%20Fischer&amp;author=K.%20Lichtenwalter&amp;author=W.%20S.%20Hancock&amp;volume=69&amp;issue=7&amp;publication_year=1997&amp;pages=1320-1325&amp;pmid=21639339&amp;doi=10.1021/ac960916h&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0032">
<span class="label">32.</span><cite>
Guimaraes G. J., Saad J. G., Annavarapu V., and Bartlett M. G., “Mobile Phase Aging and Its Impact on Electrospray Ionization of Oligonucleotides,” Journal of the American Society for Mass Spectrometry
34, no. 12 (2023): 2691–2699, 10.1021/jasms.3c00264.
</cite> [<a href="https://doi.org/10.1021/jasms.3c00264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10704597/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37978939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=Mobile%20Phase%20Aging%20and%20Its%20Impact%20on%20Electrospray%20Ionization%20of%20Oligonucleotides&amp;author=G.%20J.%20Guimaraes&amp;author=J.%20G.%20Saad&amp;author=V.%20Annavarapu&amp;author=M.%20G.%20Bartlett&amp;volume=34&amp;issue=12&amp;publication_year=2023&amp;pages=2691-2699&amp;pmid=37978939&amp;doi=10.1021/jasms.3c00264&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0033">
<span class="label">33.</span><cite>
Basiri B., van Hattum H., van Dongen W. D., Murph M. M., and Bartlett M. G., “The Role of Fluorinated Alcohols as Mobile Phase Modifiers for LC‐MS Analysis of Oligonucleotides,” Journal of the American Society for Mass Spectrometry
28, no. 1 (2017): 190–199, 10.1007/s13361-016-1500-3.
</cite> [<a href="https://doi.org/10.1007/s13361-016-1500-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5500909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27644940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=The%20Role%20of%20Fluorinated%20Alcohols%20as%20Mobile%20Phase%20Modifiers%20for%20LC%E2%80%90MS%20Analysis%20of%20Oligonucleotides&amp;author=B.%20Basiri&amp;author=H.%20van%20Hattum&amp;author=W.%20D.%20van%20Dongen&amp;author=M.%20M.%20Murph&amp;author=M.%20G.%20Bartlett&amp;volume=28&amp;issue=1&amp;publication_year=2017&amp;pages=190-199&amp;pmid=27644940&amp;doi=10.1007/s13361-016-1500-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0034">
<span class="label">34.</span><cite>
Chen B. and Bartlett M. G., “Evaluation of Mobile Phase Composition for Enhancing Sensitivity of Targeted Quantification of Oligonucleotides Using Ultra‐High Performance Liquid Chromatography and Mass Spectrometry: Application to Phosphorothioate Deoxyribonucleic Acid,” Journal of Chromatography A
1288 (2013): 73–81, 10.1016/j.chroma.2013.03.003.
</cite> [<a href="https://doi.org/10.1016/j.chroma.2013.03.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23528868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chromatography%20A&amp;title=Evaluation%20of%20Mobile%20Phase%20Composition%20for%20Enhancing%20Sensitivity%20of%20Targeted%20Quantification%20of%20Oligonucleotides%20Using%20Ultra%E2%80%90High%20Performance%20Liquid%20Chromatography%20and%20Mass%20Spectrometry:%20Application%20to%20Phosphorothioate%20Deoxyribonucleic%20Acid&amp;author=B.%20Chen&amp;author=M.%20G.%20Bartlett&amp;volume=1288&amp;publication_year=2013&amp;pages=73-81&amp;pmid=23528868&amp;doi=10.1016/j.chroma.2013.03.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0035">
<span class="label">35.</span><cite>
Bidlingmeyer B. A., Deming S. N., Price W. P., Sachok B., and Petrusek M., “Retention Mechamism for Reversed‐Phase ion‐Pair Liquid Chromatography,” Journal of Chromatography A
186 (1979): 419–434, 10.1016/S0021-9673(00)952646.</cite> [<a href="https://doi.org/10.1016/S0021-9673(00)952646" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chromatography%20A&amp;title=Retention%20Mechamism%20for%20Reversed%E2%80%90Phase%20ion%E2%80%90Pair%20Liquid%20Chromatography&amp;author=B.%20A.%20Bidlingmeyer&amp;author=S.%20N.%20Deming&amp;author=W.%20P.%20Price&amp;author=B.%20Sachok&amp;author=M.%20Petrusek&amp;volume=186&amp;publication_year=1979&amp;pages=419-434&amp;doi=10.1016/S0021-9673(00)952646&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0036">
<span class="label">36.</span><cite>
Bidlingmeyer B. A., “Separation of Ionic Compounds by Reversed‐Phase Liquid Chromatography ‐ An Update of ion‐Pairing Techniques,” Journal of Chromatographic Science
18, no. 10 (1980): 525–539, 10.1093/chromsci/18.10.525.</cite> [<a href="https://doi.org/10.1093/chromsci/18.10.525" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chromatographic%20Science&amp;title=Separation%20of%20Ionic%20Compounds%20by%20Reversed%E2%80%90Phase%20Liquid%20Chromatography%20%E2%80%90%20An%20Update%20of%20ion%E2%80%90Pairing%20Techniques&amp;author=B.%20A.%20Bidlingmeyer&amp;volume=18&amp;issue=10&amp;publication_year=1980&amp;pages=525-539&amp;doi=10.1093/chromsci/18.10.525&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0037">
<span class="label">37.</span><cite>
Wittmer D. P., Nuessle N. O., and Haney W. G., “Simultaneous Analysis of Tartrazine and Its Intermediates by Reversed Phase Liquid Chromatography,” Analytical Chemistry
47, no. 8 (1975): 1422–1423, 10.1021/ac60358a072.
</cite> [<a href="https://doi.org/10.1021/ac60358a072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1147263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Simultaneous%20Analysis%20of%20Tartrazine%20and%20Its%20Intermediates%20by%20Reversed%20Phase%20Liquid%20Chromatography&amp;author=D.%20P.%20Wittmer&amp;author=N.%20O.%20Nuessle&amp;author=W.%20G.%20Haney&amp;volume=47&amp;issue=8&amp;publication_year=1975&amp;pages=1422-1423&amp;pmid=1147263&amp;doi=10.1021/ac60358a072&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0038">
<span class="label">38.</span><cite>
Cecchi T., “Ion Pairing Chromatography,” Critical Reviews in Analytical Chemistry
38, no. 3 (2008): 161–213, 10.1080/10408340802038882.
</cite> [<a href="https://doi.org/10.1080/10408340802038882" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28122458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Critical%20Reviews%20in%20Analytical%20Chemistry&amp;title=Ion%20Pairing%20Chromatography&amp;author=T.%20Cecchi&amp;volume=38&amp;issue=3&amp;publication_year=2008&amp;pages=161-213&amp;pmid=28122458&amp;doi=10.1080/10408340802038882&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0039">
<span class="label">39.</span><cite>
Knox J. H. and Laird G. R., “Soap Chromatography—A new High‐Performance Liquid Chromatographic Technique for Separation of Ionizable Materials: Dyestuff Intermediates,” Journal of Chromatography A
122 (1976): 17–34, 10.1016/S0021-9673(00)822347.</cite> [<a href="https://doi.org/10.1016/S0021-9673(00)822347" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chromatography%20A&amp;title=Soap%20Chromatography%E2%80%94A%20new%20High%E2%80%90Performance%20Liquid%20Chromatographic%20Technique%20for%20Separation%20of%20Ionizable%20Materials:%20Dyestuff%20Intermediates&amp;author=J.%20H.%20Knox&amp;author=G.%20R.%20Laird&amp;volume=122&amp;publication_year=1976&amp;pages=17-34&amp;doi=10.1016/S0021-9673(00)822347&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0040">
<span class="label">40.</span><cite>
Kraak J. C., Jonker K. M., and Huber J. F. K., “Solvent‐Generated ion‐Exchange Systems With Anionic Surfactants for Rapid Separations of Amino Acids,” Journal of Chromatography A
142 (1977): 671–688, 10.1016/S0021-9673(01)92076-X.</cite> [<a href="https://doi.org/10.1016/S0021-9673(01)92076-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/914941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chromatography%20A&amp;title=Solvent%E2%80%90Generated%20ion%E2%80%90Exchange%20Systems%20With%20Anionic%20Surfactants%20for%20Rapid%20Separations%20of%20Amino%20Acids&amp;author=J.%20C.%20Kraak&amp;author=K.%20M.%20Jonker&amp;author=J.%20F.%20K.%20Huber&amp;volume=142&amp;publication_year=1977&amp;pages=671-688&amp;pmid=914941&amp;doi=10.1016/S0021-9673(01)92076-X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0041">
<span class="label">41.</span><cite>
Kissinger P. T., “Reverse‐Phase ion‐Pair Partition Chromatography,” Comments. Analytical Chemistry
49, no. 6 (1977): 883–883, 10.1021/ac50014a054.</cite> [<a href="https://doi.org/10.1021/ac50014a054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comments.%20Analytical%20Chemistry&amp;title=Reverse%E2%80%90Phase%20ion%E2%80%90Pair%20Partition%20Chromatography&amp;author=P.%20T.%20Kissinger&amp;volume=49&amp;issue=6&amp;publication_year=1977&amp;pages=883-883&amp;doi=10.1021/ac50014a054&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0042">
<span class="label">42.</span><cite>
Hoffman N. E. and Liao J. C., “Reversed Phase High Performance Liquid Chromatographic Separations of Nucleotides in the Presence of Solvophobic Ions,” Analytical Chemistry
49, no. 14 (1977): 2231–2234, 10.1021/ac50022a030.
</cite> [<a href="https://doi.org/10.1021/ac50022a030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/931078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Reversed%20Phase%20High%20Performance%20Liquid%20Chromatographic%20Separations%20of%20Nucleotides%20in%20the%20Presence%20of%20Solvophobic%20Ions&amp;author=N.%20E.%20Hoffman&amp;author=J.%20C.%20Liao&amp;volume=49&amp;issue=14&amp;publication_year=1977&amp;pages=2231-2234&amp;pmid=931078&amp;doi=10.1021/ac50022a030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0043">
<span class="label">43.</span><cite>
Donegan M., Nguyen J. M., and Gilar M., “Effect of ion‐Pairing Reagent Hydrophobicity on Liquid Chromatography and Mass Spectrometry Analysis of Oligonucleotides,” Journal of Chromatography A
1666 (2022): 462860, 10.1016/j.chroma.2022.462860.
</cite> [<a href="https://doi.org/10.1016/j.chroma.2022.462860" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35123169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chromatography%20A&amp;title=Effect%20of%20ion%E2%80%90Pairing%20Reagent%20Hydrophobicity%20on%20Liquid%20Chromatography%20and%20Mass%20Spectrometry%20Analysis%20of%20Oligonucleotides&amp;author=M.%20Donegan&amp;author=J.%20M.%20Nguyen&amp;author=M.%20Gilar&amp;volume=1666&amp;publication_year=2022&amp;pages=462860&amp;pmid=35123169&amp;doi=10.1016/j.chroma.2022.462860&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0044">
<span class="label">44.</span><cite>
Sutton J. M., Guimaraes G. J., Annavarapu V., van Dongen W. D., and Bartlett M. G., “Current State of Oligonucleotide Characterization Using Liquid Chromatography‐Mass Spectrometry: Insight Into Critical Issues,” Journal of the American Society for Mass Spectrometry
31, no. 9 (2020): 1775–1782, 10.1021/jasms.0c00179.
</cite> [<a href="https://doi.org/10.1021/jasms.0c00179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32812756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=Current%20State%20of%20Oligonucleotide%20Characterization%20Using%20Liquid%20Chromatography%E2%80%90Mass%20Spectrometry:%20Insight%20Into%20Critical%20Issues&amp;author=J.%20M.%20Sutton&amp;author=G.%20J.%20Guimaraes&amp;author=V.%20Annavarapu&amp;author=W.%20D.%20van%20Dongen&amp;author=M.%20G.%20Bartlett&amp;volume=31&amp;issue=9&amp;publication_year=2020&amp;pages=1775-1782&amp;pmid=32812756&amp;doi=10.1021/jasms.0c00179&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0045">
<span class="label">45.</span><cite>
Metwally H., Duez Q., and Konermann L., “Chain Ejection Model for Electrospray Ionization of Unfolded Proteins: Evidence From Atomistic Simulations and Ion Mobility Spectrometry,” Analytical Chemistry
90, no. 16 (2018): 10069–10077, 10.1021/acs.analchem.8b02926.
</cite> [<a href="https://doi.org/10.1021/acs.analchem.8b02926" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30040388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Chain%20Ejection%20Model%20for%20Electrospray%20Ionization%20of%20Unfolded%20Proteins:%20Evidence%20From%20Atomistic%20Simulations%20and%20Ion%20Mobility%20Spectrometry&amp;author=H.%20Metwally&amp;author=Q.%20Duez&amp;author=L.%20Konermann&amp;volume=90&amp;issue=16&amp;publication_year=2018&amp;pages=10069-10077&amp;pmid=30040388&amp;doi=10.1021/acs.analchem.8b02926&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0046">
<span class="label">46.</span><cite>
Chen B., Mason S. F., and Bartlett M. G., “The Effect of Organic Modifiers on Electrospray Ionization Charge‐State Distribution and Desorption Efficiency for Oligonucleotides,” Journal of the American Society for Mass Spectrometry
24, no. 2 (2013): 257–264, 10.1007/s13361-012-0509-5.
</cite> [<a href="https://doi.org/10.1007/s13361-012-0509-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23325666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=The%20Effect%20of%20Organic%20Modifiers%20on%20Electrospray%20Ionization%20Charge%E2%80%90State%20Distribution%20and%20Desorption%20Efficiency%20for%20Oligonucleotides&amp;author=B.%20Chen&amp;author=S.%20F.%20Mason&amp;author=M.%20G.%20Bartlett&amp;volume=24&amp;issue=2&amp;publication_year=2013&amp;pages=257-264&amp;pmid=23325666&amp;doi=10.1007/s13361-012-0509-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0047">
<span class="label">47.</span><cite>
Fountain K. J., Gilar M., and Gebler J. C., “Analysis of Native and Chemically Modified Oligonucleotides by Tandem ion‐Pair Reversed‐Phase High‐Performance Liquid Chromatography/Electrospray Ionization Mass Spectrometry,” Rapid Communications in Mass Spectrometry
17, no. 7 (2003): 646–653, 10.1002/rcm.959.
</cite> [<a href="https://doi.org/10.1002/rcm.959" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12661016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rapid%20Communications%20in%20Mass%20Spectrometry&amp;title=Analysis%20of%20Native%20and%20Chemically%20Modified%20Oligonucleotides%20by%20Tandem%20ion%E2%80%90Pair%20Reversed%E2%80%90Phase%20High%E2%80%90Performance%20Liquid%20Chromatography/Electrospray%20Ionization%20Mass%20Spectrometry&amp;author=K.%20J.%20Fountain&amp;author=M.%20Gilar&amp;author=J.%20C.%20Gebler&amp;volume=17&amp;issue=7&amp;publication_year=2003&amp;pages=646-653&amp;pmid=12661016&amp;doi=10.1002/rcm.959&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0048">
<span class="label">48.</span><cite>
Wu J. and McLuckey S. A., “Gas‐Phase Fragmentation of Oligonucleotide Ions,” International Journal of Mass Spectrometry
237, no. 2 (2004): 197–241, 10.1016/j.ijms.2004.06.014.</cite> [<a href="https://doi.org/10.1016/j.ijms.2004.06.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Mass%20Spectrometry&amp;title=Gas%E2%80%90Phase%20Fragmentation%20of%20Oligonucleotide%20Ions&amp;author=J.%20Wu&amp;author=S.%20A.%20McLuckey&amp;volume=237&amp;issue=2&amp;publication_year=2004&amp;pages=197-241&amp;doi=10.1016/j.ijms.2004.06.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0049">
<span class="label">49.</span><cite>
Schürch S., Bernal‐Méndez E., and Leumann C. J., “Electrospray Tandem Mass Spectrometry of Mixed‐Sequence RNA/DNA Oligonucleotides,” Journal of the American Society for Mass Spectrometry
13, no. 8 (2002): 936–945, 10.1016/S1044-0305(02)004130.
</cite> [<a href="https://doi.org/10.1016/S1044-0305(02)004130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12216734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=Electrospray%20Tandem%20Mass%20Spectrometry%20of%20Mixed%E2%80%90Sequence%20RNA/DNA%20Oligonucleotides&amp;author=S.%20Sch%C3%BCrch&amp;author=E.%20Bernal%E2%80%90M%C3%A9ndez&amp;author=C.%20J.%20Leumann&amp;volume=13&amp;issue=8&amp;publication_year=2002&amp;pages=936-945&amp;pmid=12216734&amp;doi=10.1016/S1044-0305(02)004130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0050">
<span class="label">50.</span><cite>
Taucher M., Rieder U., and Breuker K., “Minimizing Base Loss and Internal Fragmentation in Collisionally Activated Dissociation of Multiply Deprotonated RNA,” Journal of the American Society for Mass Spectrometry
21, no. 2 (2010): 278–285, 10.1016/j.jasms.2009.10.010.
</cite> [<a href="https://doi.org/10.1016/j.jasms.2009.10.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19932627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=Minimizing%20Base%20Loss%20and%20Internal%20Fragmentation%20in%20Collisionally%20Activated%20Dissociation%20of%20Multiply%20Deprotonated%20RNA&amp;author=M.%20Taucher&amp;author=U.%20Rieder&amp;author=K.%20Breuker&amp;volume=21&amp;issue=2&amp;publication_year=2010&amp;pages=278-285&amp;pmid=19932627&amp;doi=10.1016/j.jasms.2009.10.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0051">
<span class="label">51.</span><cite>
Tang W., Zhu L., and Smith L. M., “Controlling DNA Fragmentation in MALDI‐MS by Chemical Modification,” Analytical Chemistry
69, no. 3 (1997): 302–312, 10.1021/ac960865o.
</cite> [<a href="https://doi.org/10.1021/ac960865o" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21639184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Controlling%20DNA%20Fragmentation%20in%20MALDI%E2%80%90MS%20by%20Chemical%20Modification&amp;author=W.%20Tang&amp;author=L.%20Zhu&amp;author=L.%20M.%20Smith&amp;volume=69&amp;issue=3&amp;publication_year=1997&amp;pages=302-312&amp;pmid=21639184&amp;doi=10.1021/ac960865o&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0052">
<span class="label">52.</span><cite>
Gawlig C. and Rühl M., “Investigation of the Influence of Charge State and Collision Energy on Oligonucleotide Fragmentation by Tandem Mass Spectrometry,” Molecules
28, no. 3 (2023): 1169, 10.3390/molecules28031169.
</cite> [<a href="https://doi.org/10.3390/molecules28031169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9921629/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36770836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Investigation%20of%20the%20Influence%20of%20Charge%20State%20and%20Collision%20Energy%20on%20Oligonucleotide%20Fragmentation%20by%20Tandem%20Mass%20Spectrometry&amp;author=C.%20Gawlig&amp;author=M.%20R%C3%BChl&amp;volume=28&amp;issue=3&amp;publication_year=2023&amp;pages=1169&amp;pmid=36770836&amp;doi=10.3390/molecules28031169&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0053">
<span class="label">53.</span><cite>
Huang T. Y., Kharlamova A., Liu J., and McLuckey S. A., “Ion Trap Collision‐Induced Dissociation of Multiply Deprotonated RNA: c/y‐Ions Versus (a‐B)/w‐Ions,” Journal of the American Society for Mass Spectrometry
19, no. 12 (2008): 1832–1840, 10.1016/j.jasms.2008.08.009.
</cite> [<a href="https://doi.org/10.1016/j.jasms.2008.08.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18799321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=Ion%20Trap%20Collision%E2%80%90Induced%20Dissociation%20of%20Multiply%20Deprotonated%20RNA:%20c/y%E2%80%90Ions%20Versus%20(a%E2%80%90B)/w%E2%80%90Ions&amp;author=T.%20Y.%20Huang&amp;author=A.%20Kharlamova&amp;author=J.%20Liu&amp;author=S.%20A.%20McLuckey&amp;volume=19&amp;issue=12&amp;publication_year=2008&amp;pages=1832-1840&amp;pmid=18799321&amp;doi=10.1016/j.jasms.2008.08.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0054">
<span class="label">54.</span><cite>
Konermann L., Ahadi E., Rodriguez A. D., and Vahidi S., “Unraveling the Mechanism of Electrospray Ionization,” Analytical Chemistry
85, no. 1 (2013): 2–9, 10.1021/ac302789c.
</cite> [<a href="https://doi.org/10.1021/ac302789c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23134552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Unraveling%20the%20Mechanism%20of%20Electrospray%20Ionization&amp;author=L.%20Konermann&amp;author=E.%20Ahadi&amp;author=A.%20D.%20Rodriguez&amp;author=S.%20Vahidi&amp;volume=85&amp;issue=1&amp;publication_year=2013&amp;pages=2-9&amp;pmid=23134552&amp;doi=10.1021/ac302789c&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0055">
<span class="label">55.</span><cite>
Sutton J. M., El Zahar N. M., and Bartlett M. G., “Oligonucleotide Anion Adduct Formation Using Negative Ion Electrospray Ion‐Mobility Mass Spectrometry,” Journal of the American Society for Mass Spectrometry
32, no. 2 (2021): 497–508, 10.1021/jasms.0c00380.
</cite> [<a href="https://doi.org/10.1021/jasms.0c00380" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33476148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=Oligonucleotide%20Anion%20Adduct%20Formation%20Using%20Negative%20Ion%20Electrospray%20Ion%E2%80%90Mobility%20Mass%20Spectrometry&amp;author=J.%20M.%20Sutton&amp;author=N.%20M.%20El%20Zahar&amp;author=M.%20G.%20Bartlett&amp;volume=32&amp;issue=2&amp;publication_year=2021&amp;pages=497-508&amp;pmid=33476148&amp;doi=10.1021/jasms.0c00380&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0056">
<span class="label">56.</span><cite>
Largy E., König A., Ghosh A., et al., “Mass Spectrometry of Nucleic Acid Noncovalent Complexes,” Chemical Reviews
122, no. 8 (2022): 7720–7839, 10.1021/acs.chemrev.1c00386.
</cite> [<a href="https://doi.org/10.1021/acs.chemrev.1c00386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34587741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Reviews&amp;title=Mass%20Spectrometry%20of%20Nucleic%20Acid%20Noncovalent%20Complexes&amp;author=E.%20Largy&amp;author=A.%20K%C3%B6nig&amp;author=A.%20Ghosh&amp;volume=122&amp;issue=8&amp;publication_year=2022&amp;pages=7720-7839&amp;pmid=34587741&amp;doi=10.1021/acs.chemrev.1c00386&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0057">
<span class="label">57.</span><cite>
Tromp J. M. and Schürch S., “Gas‐Phase Dissociation of Oligoribonucleotides and Their Analogs Studied by Electrospray Ionization Tandem Mass Spectrometry,” Journal of the American Society for Mass Spectrometry
16, no. 8 (2005): 1262–1268, 10.1016/j.jasms.2005.03.024.
</cite> [<a href="https://doi.org/10.1016/j.jasms.2005.03.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15978835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=Gas%E2%80%90Phase%20Dissociation%20of%20Oligoribonucleotides%20and%20Their%20Analogs%20Studied%20by%20Electrospray%20Ionization%20Tandem%20Mass%20Spectrometry&amp;author=J.%20M.%20Tromp&amp;author=S.%20Sch%C3%BCrch&amp;volume=16&amp;issue=8&amp;publication_year=2005&amp;pages=1262-1268&amp;pmid=15978835&amp;doi=10.1016/j.jasms.2005.03.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0058">
<span class="label">58.</span><cite>
Wan K. X., Gross J., Hillenkamp F., and Gross M. L., “Fragmentation Mechanisms of Oligodeoxynucleotides Studied by H/D Exchange and Electrospray Ionization Tandem Mass Spectrometry,” Journal of the American Society for Mass Spectrometry
12, no. 2 (2001): 193–205, 10.1016/S1044-0305(00)002087.
</cite> [<a href="https://doi.org/10.1016/S1044-0305(00)002087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11212004/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=Fragmentation%20Mechanisms%20of%20Oligodeoxynucleotides%20Studied%20by%20H/D%20Exchange%20and%20Electrospray%20Ionization%20Tandem%20Mass%20Spectrometry&amp;author=K.%20X.%20Wan&amp;author=J.%20Gross&amp;author=F.%20Hillenkamp&amp;author=M.%20L.%20Gross&amp;volume=12&amp;issue=2&amp;publication_year=2001&amp;pages=193-205&amp;pmid=11212004&amp;doi=10.1016/S1044-0305(00)002087&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0059">
<span class="label">59.</span><cite>
Wang P., Bartlett M. G., and Martin L. B., “Electrospray Collision‐Induced Dissociation Mass Spectra of Positively Charged Oligonucleotides,” Rapid Communications in Mass Spectrometry
11, no. 8 (1997): 846–856, .</cite> [<a href="https://doi.org/10.1002/(SICI)10970231(199705)11:8&amp;#x0003c;846::AID-RCM928&amp;#x0003e;3.0.CO;29" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rapid%20Communications%20in%20Mass%20Spectrometry&amp;title=Electrospray%20Collision%E2%80%90Induced%20Dissociation%20Mass%20Spectra%20of%20Positively%20Charged%20Oligonucleotides&amp;author=P.%20Wang&amp;author=M.%20G.%20Bartlett&amp;author=L.%20B.%20Martin&amp;volume=11&amp;issue=8&amp;publication_year=1997&amp;pages=846-856&amp;doi=10.1002/(SICI)10970231(199705)11:8&amp;#x0003c;846::AID-RCM928&amp;#x0003e;3.0.CO;29&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0060">
<span class="label">60.</span><cite>
Sannes‐Lowery K. A., Mack D. P., Hu P., Mei H. Y., and Loo J. A., “Positive ion Electrospray Ionization Mass Spectrometry of Oligonucleotides,” Journal of the American Society for Mass Spectrometry
8, no. 1 (1997): 90–95, 10.1016/S1044-0305(96)002024.</cite> [<a href="https://doi.org/10.1016/S1044-0305(96)002024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=Positive%20ion%20Electrospray%20Ionization%20Mass%20Spectrometry%20of%20Oligonucleotides&amp;author=K.%20A.%20Sannes%E2%80%90Lowery&amp;author=D.%20P.%20Mack&amp;author=P.%20Hu&amp;author=H.%20Y.%20Mei&amp;author=J.%20A.%20Loo&amp;volume=8&amp;issue=1&amp;publication_year=1997&amp;pages=90-95&amp;doi=10.1016/S1044-0305(96)002024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10033-bib-0061">
<span class="label">61.</span><cite>
Karasawa K., Duchoslav E., Burton L., Kawakami J., and Baba T., “Sequencing of Morpholino Antisense Oligonucleotides Using Electron Capture Dissociation Mass Spectrometry,” Analytical Chemistry
95, no. 44 (2023): 16352–16358, 10.1021/acs.analchem.3c03621.
</cite> [<a href="https://doi.org/10.1021/acs.analchem.3c03621" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37871344/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Sequencing%20of%20Morpholino%20Antisense%20Oligonucleotides%20Using%20Electron%20Capture%20Dissociation%20Mass%20Spectrometry&amp;author=K.%20Karasawa&amp;author=E.%20Duchoslav&amp;author=L.%20Burton&amp;author=J.%20Kawakami&amp;author=T.%20Baba&amp;volume=95&amp;issue=44&amp;publication_year=2023&amp;pages=16352-16358&amp;pmid=37871344&amp;doi=10.1021/acs.analchem.3c03621&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Data S1</strong> Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11969060/bin/RCM-39-e10033-s001.pptx" data-ga-action="click_feat_suppl" class="usa-link">RCM-39-e10033-s001.pptx</a><sup> (10.4MB, pptx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are available from the corresponding author upon reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Rapid Communications in Mass Spectrometry are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/rcm.10033"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/RCM-39-e10033.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11969060/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11969060/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11969060%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11969060/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11969060/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11969060/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40181565/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11969060/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40181565/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11969060/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11969060/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="xg83xUuAVe4aeHZDonCwmkoZnfulCKaPwJZp4tlQ4vdbWD9PhZaa3pYoNU5nTZOR">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
