
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Episodic transport of protein aggregates achieves a positive size selectivity in aggresome formation - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53146A8AEF59B305146A0025C4C30E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373839/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Episodic transport of protein aggregates achieves a positive size selectivity in aggresome formation">
<meta name="citation_author" content="Rui Fang">
<meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author_institution" content="School of Life Sciences, Westlake University, Hang Zhou, China">
<meta name="citation_author" content="Luolan Bai">
<meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Bert M Verheijen">
<meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Boyan Li">
<meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Kevin Dong">
<meta name="citation_author_institution" content="Department of Cell Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Joao A Paulo">
<meta name="citation_author_institution" content="Department of Cell Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Mengying Zhou">
<meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Yi-Chi Chu">
<meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Yuyu Song">
<meta name="citation_author_institution" content="Department of Neurology, Massachusetts General Hospital, Boston, MA USA">
<meta name="citation_author" content="Michael Y Sherman">
<meta name="citation_author_institution" content="Department of Molecular Biology, Ariel University, Ariel, Israel">
<meta name="citation_author" content="Steven Gygi">
<meta name="citation_author_institution" content="Department of Cell Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Christine M Field">
<meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Timothy J Mitchison">
<meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author" content="Ying Lu">
<meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7852">
<meta name="citation_doi" content="10.1038/s41467-025-62751-5">
<meta name="citation_pmid" content="40846843">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373839/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373839/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373839/pdf/41467_2025_Article_62751.pdf">
<meta name="description" content="Eukaryotic cells direct toxic misfolded proteins to various quality control pathways based on their chemical properties and aggregation status. Aggregated proteins are targeted to selective autophagy or specifically sequestered into the “aggresome”, ...">
<meta name="og:title" content="Episodic transport of protein aggregates achieves a positive size selectivity in aggresome formation">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Eukaryotic cells direct toxic misfolded proteins to various quality control pathways based on their chemical properties and aggregation status. Aggregated proteins are targeted to selective autophagy or specifically sequestered into the “aggresome”, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373839/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373839">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62751-5"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62751.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373839%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373839/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373839/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373839/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 22;16:7852. doi: <a href="https://doi.org/10.1038/s41467-025-62751-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62751-5</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Episodic transport of protein aggregates achieves a positive size selectivity in aggresome formation</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fang%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Rui Fang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Rui Fang</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">
<sup>2</sup>School of Life Sciences, Westlake University, Hang Zhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fang%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rui Fang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bai%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Luolan Bai</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Luolan Bai</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bai%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Luolan Bai</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Verheijen%20BM%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Bert M Verheijen</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Bert M Verheijen</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Verheijen%20BM%22%5BAuthor%5D" class="usa-link"><span class="name western">Bert M Verheijen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Boyan Li</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Boyan Li</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Boyan Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Kevin Dong</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Kevin Dong</span></h3>
<div class="p">
<sup>3</sup>Department of Cell Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kevin Dong</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Paulo%20JA%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Joao A Paulo</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Joao A Paulo</span></h3>
<div class="p">
<sup>3</sup>Department of Cell Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Paulo%20JA%22%5BAuthor%5D" class="usa-link"><span class="name western">Joao A Paulo</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Mengying Zhou</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Mengying Zhou</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mengying Zhou</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chu%20YC%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Yi-Chi Chu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Yi-Chi Chu</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chu%20YC%22%5BAuthor%5D" class="usa-link"><span class="name western">Yi-Chi Chu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Yuyu Song</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Yuyu Song</span></h3>
<div class="p">
<sup>4</sup>Department of Neurology, Massachusetts General Hospital, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuyu Song</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sherman%20MY%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Michael Y Sherman</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Michael Y Sherman</span></h3>
<div class="p">
<sup>5</sup>Department of Molecular Biology, Ariel University, Ariel, Israel </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sherman%20MY%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael Y Sherman</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gygi%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Steven Gygi</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Steven Gygi</span></h3>
<div class="p">
<sup>3</sup>Department of Cell Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gygi%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Steven Gygi</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Field%20CM%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Christine M Field</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Christine M Field</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Field%20CM%22%5BAuthor%5D" class="usa-link"><span class="name western">Christine M Field</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mitchison%20TJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Timothy J Mitchison</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Timothy J Mitchison</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mitchison%20TJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Timothy J Mitchison</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Ying Lu</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Ying Lu</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Ying Lu</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA USA </div>
<div id="Aff2">
<sup>2</sup>School of Life Sciences, Westlake University, Hang Zhou, China </div>
<div id="Aff3">
<sup>3</sup>Department of Cell Biology, Harvard Medical School, Boston, MA USA </div>
<div id="Aff4">
<sup>4</sup>Department of Neurology, Massachusetts General Hospital, Boston, MA USA </div>
<div id="Aff5">
<sup>5</sup>Department of Molecular Biology, Ariel University, Ariel, Israel </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jul 12; Accepted 2025 Jul 29; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373839  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846843/" class="usa-link">40846843</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>
<strong>Previous version available:</strong> This article is based on a previously available preprint posted on bioRxiv on August 7, 2024: "<a href="/articles/PMC11361152/" class="usa-link">
Episodic Transport of Protein Aggregates Achieves a Positive Size Selectivity in Aggresome Formation
</a>".</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Eukaryotic cells direct toxic misfolded proteins to various quality control pathways based on their chemical properties and aggregation status. Aggregated proteins are targeted to selective autophagy or specifically sequestered into the “aggresome”, a perinuclear inclusion at the microtubule-organizing center (MTOC). However, the mechanism for selective aggresome recruitment remains unclear. To investigate this process, here we reconstitute MTOC-directed aggregate transport in <em>Xenopus laevis</em> egg extract using AgDD, a chemically inducible aggregation system. High-resolution single-particle tracking reveals that dynein-mediated aggregate transport is highly episodic, with average velocity positively correlating with aggregate size. Mechanistic modeling suggests that recurrent formation of the dynein transport complex biases larger aggregates towards active transport, compensating for the slowdown due to viscosity. Both episodic transport and positive size selectivity are conferred by aggresome-specific dynein adapters. Coupling an aggresome adapter to polystyrene beads recapitulates positive size selectivity in transport, while recruiting conventional dynein adapters to protein aggregates perturbs aggresome formation and reverses the size selectivity.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Protein quality control, Motor protein function</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Aggregates of misfolded proteins are sequestered in the aggresome via dynein transport in an aggregate size-dependent manner. Here, authors find episodic transport kinetics mediated by aggresome-dynein adapters are central to this size-filter effect.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Accumulation of damaged or misfolded proteins leads to cytotoxicity and underlies a variety of human disorders, such as neurodegenerative diseases and type-II diabetes<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. To maintain protein homeostasis, eukaryotic cells possess multiple protein quality control (PQC) systems that operate complementarily to eliminate abnormal proteins with diverse chemical and physical properties. The proteasome system is responsible for degrading most soluble substrates. Proteins that aggregate into higher-order structures, such as oligomers and aggregates, are often resistant to proteasomal degradation and can even inhibit proteasome activities, necessitating their clearance through alternative PQC pathways, such as selective autophagy or aggresome sequestration. The aggresome is a membrane-less organelle localized at the microtubule-organizing center (MTOC)<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Aggresome formation represents a general cellular defense mechanism that is activated when the production of aggregation-prone proteins exceeds the degradation capacity<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Apart from its protective role, the aggresome may serve as a precursor to some pathological inclusions observed in liver and neurodegenerative diseases, and a similar process has been suggested in inflammasome formation and viral replication<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. The mechanism by which misfolded proteins are selectively targeted into the aggresome remains poorly understood.</p>
<p id="Par4">Formation of the aggresome typically initiates with the nucleation of aggregation-prone proteins into small aggregates in the peripheral cytoplasm, which are termed “pre-aggresome particles”, followed by the transport of these aggregates along microtubules to the perinuclear region by the dynein motor complex<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. A variety of enzymes, molecular chaperones, and adapter proteins have been discovered to regulate this process. For instance, HDAC6, SQSTM1/p62, CTIF/eEF1A1/Dynactin (CED), and the Hsp70/BAG3/14-3-3 complex recognize misfolded proteins and bridge them with the dynein machinery<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. HDAC6 interacts with the ubiquitin (Ub) or Ub chain on substrates, and bridges them with p150<sup>Glued</sup>, a core component in the dynactin complex responsible for dynein activation, while SQSTM1 may interact with dynein intermediate chains (DICs) and recruit the substrates via the associated Lys63-linked Ub chains<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Recognition of misfolded proteins by Hsp70 is independent of Ub and Hsp70’s interacting partner 14-3-3 then links these proteins to DIC<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Another adapter, the CED complex, preferentially interacts with pre-existing and newly-synthesized polypeptides and bridges them with dynein in a YTHDF2-dependent fashion<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. For simplicity, we refer to these factors that link aggresome substrates with the dynein motor as aggresome adapters. Different aggresome adapters can work concurrently in the same cell to secure the sequestration of protein aggregates<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, while the substrate specificity of each aggresome adapter has not been fully understood. Besides the aforementioned aggresome adapters, additional factors, such as ataxin-3, that interact with both dynein and misfolded proteins, have been reported to regulate aggresome formation<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>.</p>
<p id="Par5">Unlike protein aggregates, the transport of dynein’s conventional cargoes, such as cellular vesicles and organelles, is mainly mediated by the activating adapters, including BICD2, HOOK2, and HOOK3, which link these cargoes to the dynein motor. Besides substrate selection, the activating adapters also stabilize dynein–dynactin interaction and activate cargo transport by stimulating dynein’s speed and processivity<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. All activating adapters contain a coiled-coil domain that interacts extensively with both dynein and dynactin<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. However, aggresome-specific adapters lack the coiled-coil domain and may exhibit weak interactions with dynein, with HDAC6 as an example<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. The functional implications for employing structurally distinct adapters for aggregate transport have not been explored.</p>
<p id="Par6">Directing misfolded proteins into appropriate PQC pathways requires selectivity based on the aggregation status (or size) of the protein. Protein aggregates appear to be preferentially sorted into the autophagy pathway or loaded onto the dynein motor for aggresome sequestration, over soluble misfolded proteins<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. HDAC6’s BUZ domain interacts with the free C-terminus of Ub or Ub chain that is generated through substrate deubiquitylation. This free-Ub specificity may allow HDAC6 to interact with protein aggregates that contain trapped Ub<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. However, some aggregation-prone proteins, such as GFP-250 and synphilin 1, are sequestered by Ub-independent aggresome pathways, and their aggresome formation is still preceded by substrate nucleation<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. These observations suggest that ubiquitylation is not instrumental for aggregate selection, and a general feature of protein aggregates, e.g., their size, may guide this process. Contrary to the requirement to target aggregated proteins, dynein-mediated transport of conventional cargoes typically favors the movement of small objects, due to cytosolic viscous friction that increases with the cargo size<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. How this property of transport is reconciled with the desired specificity for protein aggregates during aggresome formation is unclear. Adding to this specificity puzzle, aggresome adapters Hsp70 and SQSTM1 are involved in diverse biological processes, including delivering soluble substrates for proteasomal degradation<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. How the same factors can participate in PQC pathways with distinct substrate specificities remains unclear.</p>
<p id="Par7">To elucidate the mechanism by which the aggresome pathways recognize the aggregated forms of misfolded proteins, we used a chemically inducible aggregation-prone protein, AgDD, and reconstituted the process of MTOC-directed transport of protein aggregates in both live cells and <em>Xenopus laevis</em> egg extract (XE). XE is one of the least perturbed cell-free systems available, and is used extensively to recapitulate dynamic intracellular structures and events<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Our reconstitution system successfully recapitulated the key features of protein aggregate transport in live cells, including the role of previously identified aggresome adapters, and allowed us to gain insights into the mechanism of an unusual positive size selectivity (PSS), where larger aggregates are transported more efficiently during aggresome formation. This transport feature is specifically associated with aggresome adapters and distinct from the transport of other cargoes, like cellular vesicles and organelles, where a negative size selectivity (NSS) is usually observed<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Through high-resolution single-particle tracking, we observed that protein aggregates exhibited “stop-and-go” movements with low average velocities. Interestingly, larger aggregates, despite having lower instantaneous velocities, experienced shorter pauses and hence achieved a higher average transport velocity. Our mechanistic model suggests that rapid disassembly and reformation of the aggregate-dynein-microtubule complex during episodic transport biases larger aggregates towards the active transport mode and quantitatively accounts for the observed PSS in aggresome formation.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Protein aggregates are selectively targeted to the aggresome</h3>
<p id="Par8">To confirm that protein aggregates are preferentially targeted for aggresome sequestration, we employed AgDD-sfGFP (AgDD) as a model for aggregation-prone proteins. AgDD consists of an FKBP-based destabilization domain (DD) fused with a short hydrophobic peptide at the N-terminus<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. The DD degron domain can be stabilized by the ligand Shield-1. Anti-GFP immunoblotting suggests that AgDD was stably expressed as a full-length protein in the cell lines (Fig. <a href="#MOESM1" class="usa-link">S1B</a>) and depleting Shield-1 from the culture medium led to rapid misfolding and aggregation of AgDD into cytoplasmic aggregates (Fig. <a href="#Fig1" class="usa-link">1A</a>, <a href="#MOESM1" class="usa-link">S1A</a>; Supp. Movie <a href="#MOESM3" class="usa-link">1</a>). These initial AgDD aggregates were then actively transported towards the nucleus to form a perinuclear aggresome-like punctum that colocalized with the centrosomal component pericentrin and was surrounded by a cage-like structure of vimentin, which are commonly utilized as markers of aggresome formation (Fig. <a href="#Fig1" class="usa-link">1B</a>)<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. The presence of AgDD aggresome did not significantly affect cell growth<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>, nor did it strongly perturb the cell-cycle phase durations as indicated by the localization of fluorescently labeled Histone H2A (Fig. <a href="#MOESM1" class="usa-link">S2</a>), which greatly simplified data interpretation.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. AgDD aggregates are selectively transported into the aggresome.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373839_41467_2025_62751_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fea/12373839/be0df4a9eeef/41467_2025_62751_Fig1_HTML.jpg" loading="lazy" id="d33e547" height="830" width="674" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic and representative live-cell confocal images of AgDD aggresome formation. 10 μM FKBP(F36V) was added to HEK293T cell culture stably expressing AgDD-sfGFP (AgDD) to deplete Shield-1. Representative cells with a high AgDD expression level that formed aggresomes after Shield-1 removal and cells with a low AgDD expression level that did not form detectable aggresomes were selected from the same field-of-view (FOV). <strong>B</strong> Colocalization analysis of AgDD with aggresome markers in cells with high AgDD and low AgDD levels. AgDD (green)-expressing U2OS cells were fixed and stained with anti-vimentin (red) and anti-pericentrin (magenta) antibodies, 4 h after Shield-1 removal. DNA was stained with DAPI (blue). Experiments were performed once using U2OS cells and once using HEK293T cells, with similar results. <strong>C</strong> Effects of drug treatment on aggresome formation. HEK293T cells stably expressing AgDD and H2B-mCherry were treated with 20 μM Nexturastat A, 10 μM MLN7243, or 1 μg/mL colchicine for 30 min before Shield-1 removal to induce aggresome formation, and imaged using a confocal microscope. The fraction of cells that had completed aggresome formation, as indicated by the sequestration of all peripheral aggregates into a single perinuclear punctum, was determined at the indicated time points. Error bars represent the standard error of the mean (SEM) over four randomly chosen FOVs; n: number of cells analyzed. <em>P</em>-values were determined by an unpaired two-tailed Student’s <em>t</em>-test as indicated. Below: representative images under each condition with AgDD (green) and H2B (red) channels, 8 h after Shield-1 removal. <strong>D</strong> Correlation between AgDD aggregation and aggresome formation. AgDD-expressing HEK293T cells were induced by Shield-1 removal with or without 1 μM bortezomib (BTZ) and subjected to time-lapse imaging at 10 min per frame for 4 h. Randomly chosen cells under each condition were classified according to the change of AgDD distribution during the time-lapse. D, diffusive AgDD signal; P, peripheral AgDD aggregates; A, AgDD aggresome. Cells with high (&gt;2.5 μM) and low (&lt;2.5 μM) initial AgDD levels were counted separately. Error bars represent SEM over 4 FOVs; n: number of cells analyzed. Representative images are shown in (<strong>E</strong>), with aggresomes indicated by arrowheads. <strong>F</strong> Degradation of diffusive AgDD upon Shield-1 removal. The diffusive AgDD signal was determined in the aggresome-free cells (“D → D” in (<strong>D</strong>)) during the time-lapse, normalized by the initial AgDD intensity right after Shield-1 removal (<em>t</em> = 0). Line shading represents SEM over n cells analyzed. <strong>G</strong> Live-cell confocal images of U2OS cells 3 h after Shield-1 removal, overlaid with trajectories of individual AgDD aggregates whose initial positions were marked with filled circles. Cells were treated with 1 μg/mL colchicine or DMSO for 30 min before aggresome induction and imaged once per minute. Nuclear contours were marked with white dashed lines. <strong>H</strong> The relationship between the MTOC-directed velocity and the diameter of AgDD aggregates. The MTOC-directed velocities of individual aggregates were determined frame-by-frame, and correlated with the aggregate diameter at each time point in a violin plot (n = 10286 velocity-diameter pairs from four cells). Positive velocity values represent movement towards the MTOC. Error bars in the zoom-in plot (left) represent the SEM within each size group. Source data are provided as a <a href="#MOESM16" class="usa-link">Source data</a> file. <strong>I</strong> Live-cell confocal images of MCF10A cells stably expressing GFP-tagged synphilin 1 (synph-GFP) after treatment with 100 nM MG132. Cells were imaged once per minute. <strong>J</strong> The relationship between the MTOC-directed velocity and the intensity of synph-GFP aggregates. The MTOC-directed velocities of individual aggregates were determined frame-by-frame, and correlated with the aggregate intensity at each time point in a violin plot as in (<strong>H</strong>) (n = 180934 velocity-intensity pairs from 16 cells). Source data are provided as a <a href="#MOESM16" class="usa-link">Source data</a> file.</p></figcaption></figure><p id="Par9">Similar to other aggregation-prone proteins, AgDD requires an intact microtubule network to localize to the aggresome. Colchicine, a microtubule destabilizer, impeded the perinuclear accumulation of misfolded AgDD, resulting in multiple small aggregates accumulating in the peripheral cytoplasm (Fig. <a href="#Fig1" class="usa-link">1C</a>). The HDAC6 inhibitor, Nexturastat A, delayed the completion of AgDD aggresome formation marked by all peripheral AgDD aggregates being sequestered into a unique perinuclear punctum, but did not completely block it, suggesting that the deacetylase activity of HDAC6 plays some role in the process as previously reported (Fig. <a href="#Fig1" class="usa-link">1C</a>)<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Inhibiting the ubiquitin-activating enzyme (E1) activity with MLN7243 did not alter the timing of perinuclear sequestration of AgDD, suggesting that substrate ubiquitylation is not essential for AgDD aggresome formation (Fig. <a href="#Fig1" class="usa-link">1C</a>, <a href="#MOESM1" class="usa-link">S1C</a>).</p>
<p id="Par10">To study the connection between AgDD nucleation and aggresome formation, we examined the fate of misfolded AgDD in either diffuse or aggregated states, leveraging the natural variation of AgDD expression levels in the cell population. The molar concentrations of AgDD in individual cells were determined based on calibration using purified GFP. As with other aggregation-prone proteins, the tendency of AgDD to aggregate increased with its concentration (Fig. <a href="#Fig1" class="usa-link">1</a>A, <a href="#Fig1" class="usa-link">D</a>). Cells constitutively expressing a low level of AgDD did not show detectable aggregates upon Shield-1 removal. We classified the cells according to the changes in AgDD localization in time-lapse and found that cells without detectable AgDD aggregation also lacked a perinuclear accumulation of AgDD (Fig. <a href="#Fig1" class="usa-link">1D</a>). However, AgDD misfolding still occurred in these aggregation-free cells, as indicated by the proteasome-dependent rapid decrease of the diffusive GFP signal, which could be stabilized by the proteasome inhibitor, bortezomib (BTZ) (Fig. <a href="#Fig1" class="usa-link">1F</a>). BTZ treatment is known to induce nonspecific protein aggregation and aggresome formation<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Upon proteasome inhibition, most of these low-AgDD cells formed an aggresome-like perinuclear punctum within 4 h after Shield-1 removal, though the intensity of the puncta appeared weak (Fig. <a href="#Fig1" class="usa-link">1</a>D, <a href="#Fig1" class="usa-link">E</a>), which excludes the possibility that these low-AgDD cells were inherently defective in aggresome formation. Taken together, these results suggest that the perinuclear deposition of misfolded AgDD correlates with, and potentially requires, its prior nucleation in the peripheral cytoplasm.</p>
<p id="Par11">To quantitatively compare the efficiency of aggresome formation under different conditions, we defined the “aggresome enrichment factor” as the ratio of the maximum fluorescence intensity of pixels within the perinuclear region to the average intensity of the cell before Shield-1 removal. Although this factor for BTZ-induced aggresome in low-AgDD cells was significantly larger than that for the aggresome-free cells, the value increased more dramatically for high-AgDD cells that formed aggresome spontaneously from peripheral aggregates without BTZ treatment (Fig. <a href="#MOESM1" class="usa-link">S1D</a>). Therefore, we conclude that, although misfolded protein may enter the aggresome without the “pre-aggresome particle” stage under proteasome inhibition, efficient aggresome formation still requires a prior aggregation of the misfolded protein. It is not clear how proteasome inhibition loosens this requirement. One possibility is that misfolded AgDD may join the nonspecific protein aggregates induced by proteasome inhibition to be recognized by the transport machinery<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p>
<p id="Par12">To further study how AgDD aggregation status modulates its tendency to form aggresomes, we performed single-particle tracking of live-cell time-lapse images to determine the MTOC-directed velocity of individual AgDD aggregates at each frame (Fig. <a href="#Fig1" class="usa-link">1G</a>). We then correlated the velocity with the aggregate diameter, which had been determined by fitting the aggregate’s image with a 2D-Gaussian function, upon calibration using fluorescent size standards (Fig. <a href="#Fig1" class="usa-link">1H</a>, <a href="#MOESM1" class="usa-link">S3</a>; Methods). Notably, the MTOC-directed velocity increased with the size of an aggregate, with a mean value of 0.6 nm/s for aggregates around 0.25 µm, rising to 1.4 nm/s for aggregates around 0.75 µm (Fig. <a href="#Fig1" class="usa-link">1H</a>). Both values were significantly lower than the typical dynein velocity around 500 nm/s when transporting cellular organelles and vesicles of similar sizes<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. This positive correlation between the transport velocity and aggregate size is not unique to AgDD, as a similar trend was detected for aggregates of GFP-tagged synphilin 1 in MCF10A cells upon proteasome inhibition by MG132 (Figs. <a href="#Fig1" class="usa-link">1I, J</a>; Supp. Movie <a href="#MOESM4" class="usa-link">2</a>)<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. We further excluded the possibility of AgDD co-translocating with other organelles. We stained live U2OS cells expressing AgDD with MitoTracker, LysoTracker, and ER-Tracker to detect mitochondria, lysosomes, and endoplasmic reticulum (ER) respectively, all of which are dynein cargoes. We found that AgDD aggregates were neither colocalized nor co-transported with these organelles, suggesting that the observed size dependency is an intrinsic feature of aggregate transport (Fig. <a href="#MOESM1" class="usa-link">S4</a>). In summary, our results indicate that sequestration of misfolded proteins into the aggresome is likely to require their nucleation into cytoplasmic aggregates. This dependency may be driven by the selectivity of the transport machinery, which preferentially engages with larger aggregates.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Reconstitution of dynein-mediated transport of protein aggregates in <em>Xenopus laevis</em> egg extract</h3>
<p id="Par13">Direct interpretation of the tracking results above is complicated by the potential involvement of the plus-end motor kinesins, changes in aggregate size, cytoplasm heterogeneity and other confounding factors in live cells. To investigate the mechanism of size selection in aggresome formation, we reconstituted the MTOC-directed transport of protein aggregates in <em>Xenopus laevis</em> egg extract (XE), an active cytoplasm that has been widely employed to study dynein activities in biological processes, such as spindle formation and the transport of organelles<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>.</p>
<p id="Par14">Aggregation of purified AgDD in XE was triggered by adding recombinant FKBP(F36V) to deplete free Shield-1 (Supp. Movie <a href="#MOESM5" class="usa-link">3</a>). AgDD aggregates were then diluted into interphase actin-depolymerized XE where the formation of microtubule asters was seeded by centrosomes from demembranated <em>Xenopus</em> sperms and probed with Alexa647-conjugated tubulin. The mobility of aggregates along microtubules was examined in a customized imaging chamber with dual-color fluorescence microscopy (Fig. <a href="#Fig2" class="usa-link">2A</a>; Methods)<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Reconstitution of dynein-dependent transport of AgDD aggregates in <em>Xenopus laevis</em> egg extract (XE).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373839_41467_2025_62751_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fea/12373839/05c3c4dc8457/41467_2025_62751_Fig2_HTML.jpg" loading="lazy" id="d33e784" height="728" width="799" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic and wide-field images of the aggregate transport assay. Recombinant AgDD (green) was induced to aggregate in XE and 1:20 diluted into a working XE containing Alexa647-labeled microtubule asters (magenta) and imaged in a 1 cm × 1 cm × 20 μm customized chamber once per minute for 30 min at 18 °C. Experiments were performed 10 times independently with similar results. <strong>B</strong> Example trajectories of AgDD aggregates. Measurement was performed as in (<strong>A</strong>), with or without 40 μg/mL CC1 to inhibit dynein-dynactin in XE. Aggregates were randomly selected and colored by diameter. The MTOC was marked with an empty red circle at the center. Among the 100 trajectories from the untreated group, Trajectories of the 10 largest and the 10 smallest aggregates are plotted in (<strong>C</strong>) (mean diameter ± standard deviation in legend), with the averaged trajectories shown as thick lines. <strong>D</strong> The relationship between the time-averaged transport velocity and diameter of individual aggregates. The average velocity of individual aggregates was calculated as the total travel distance towards the MTOC divided by the trajectory’s duration (illustrated in (<strong>E</strong>)). The velocities of individual aggregates were grouped by their diameter in the violin plot. Right: a zoom-in plot showing the mean ± SEM within each group. A total of 1152 (n) aggregates around 16 microtubule asters from 2 batches of XE were included in the analysis. Source data are provided as a <a href="#MOESM16" class="usa-link">Source data</a> file. <strong>E</strong> An example trajectory of AgDD aggregate to illustrate the result of trajectory segmentation. “Pause” and “Transport Engaged (TE)” segments were colored in pink and green respectively. <strong>F</strong> The relationship between aggregate diameter and lengths of the pause and the TE segments, shown as mean ± SEM within each size group. 851 (n) trajectories from the experiment in (<strong>D</strong>) that are longer than 15 min were selected and segmented as illustrated in (<strong>E</strong>). TE (+): transport-engaged segment when the aggregate moves towards the MTOC; TE (-): transport-engaged segment when the aggregate moves away from the MTOC. <strong>G</strong> The relationship between the aggregate diameter and the proportion of the pause segment, determined using the data in (<strong>F</strong>). The plot shows the mean proportion ± SEM within each size group. <strong>H</strong> Aggregates’ diffusion constants during pauses, with or without 40 μg/mL CC1 in XE. Indicated numbers (n) of aggregates that could be tracked for at least 15 min were selected and grouped by their diameters. Diffusion constants during pause were calculated as described in the methods and plotted against the inverse of the mean diameter within each size group. The dashed line represents the prediction by the Stokes-Einstein law as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e730"><msub><mrow><mi>k</mi></mrow><mrow><mi>B</mi></mrow></msub><mi>T</mi><mo>/</mo><mrow><mo>(</mo><mrow><mn>3</mn><mi>π</mi><mi>η</mi><mi>d</mi></mrow><mo>)</mo></mrow></math></span>, where <em>d</em> is the aggregate diameter, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e756"><mi>η</mi></math></span> is the dynamic viscosity of XE (0.01 Pa·s), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e765"><msub><mrow><mi>k</mi></mrow><mrow><mi>B</mi></mrow></msub></math></span> is the Boltzman constant, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e779"><mi>T</mi></math></span> is 291 Kelvin. Data are presented as mean values ± SEM.</p></figcaption></figure><p id="Par15">Upon incubation, we observed efficient MTOC-directed transport of AgDD aggregates resulting in most aggregates concentrated at the MTOC vicinity within 30 min, resembling aggresome formation in live cells (Figs. <a href="#Fig2" class="usa-link">2A, B</a>; Supp. Movie <a href="#MOESM6" class="usa-link">4</a>). Efficient transport was also observed in cycling XE with intact actin cytoskeleton (Supp. Movie <a href="#MOESM7" class="usa-link">5</a>). This transport depended on a functional dynein-dynactin complex, as the addition of a p150 fragment CC1, a dynein inhibitor, blocked the transport, consistent with the previous observations with other aggregation-prone proteins in cells (Fig. <a href="#Fig2" class="usa-link">2B</a>; Supp. Movie <a href="#MOESM8" class="usa-link">6</a>). Processive backward (i.e., away from MTOC) transport of aggregates was not observed upon inhibiting dynein-dynactin, which suggests a lack of kinesin activity in the transport process, consistent with a general absence of kinesin activity in <em>Xenopus</em> eggs<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Interestingly, AgDD aggregates formed in a buffer solution showed no transport when incubated in XE (Supp. Movie <a href="#MOESM9" class="usa-link">7</a>), suggesting that the cytoplasmic factors important for transport need to be present at the time of aggregate formation.</p>
<p id="Par16">Although substrate ubiquitylation is important for aggresome formation by certain aggregation-prone proteins<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>, adding a high concentration of a nonspecific deubiquitylating enzyme Usp2<sup>CD</sup> to deplete most Ub conjugates did not perturb the kinetics of AgDD aggregate transport in XE (Fig. <a href="#MOESM1" class="usa-link">S5A</a>). In addition, we did not detect significant transport of Dynabeads conjugated with either K48- or K63-linked ubiquitin chains in XE (Fig. <a href="#MOESM1" class="usa-link">S5B</a>). These results corroborate the findings in cells (Fig. <a href="#Fig1" class="usa-link">1C</a>) to suggest a negligible role of ubiquitylation in AgDD aggresome formation.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Average transport velocity of aggregates in <em>Xenopus</em> egg extract positively correlates with the size of aggregates</h3>
<p id="Par17">It took ~30 min for a peripheral aggregate to travel to the MTOC, much longer compared to the time constants of dynein molecule stepping or dynein-microtubule interaction, which typically last for seconds<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. We therefore examined the transport phenomenon over both long- and short-time scales to identify the mechanisms contributing to size selectivity.</p>
<p id="Par18">We first performed a long-term measurement at 1-min per frame to record the entire transport process. We tracked the movement of individual AgDD aggregates, calculated the time average of each aggregate’s velocity towards the MTOC and associated the velocity with the aggregate diameter (Fig. <a href="#Fig2" class="usa-link">2</a>C, <a href="#Fig2" class="usa-link">D</a>). Most aggregates’ size appeared stable during the experiment (Fig. <a href="#MOESM1" class="usa-link">S6</a>) and fusion events were rarely detected, as the aggregate concentration was low. Like the transport of AgDD aggregates in cells, the average transport velocity in XE positively correlated with the aggregate size and approached zero when the aggregate diameter reduced to ~0.7 µm (Fig. <a href="#Fig2" class="usa-link">2</a>C, <a href="#Fig2" class="usa-link">D</a>). In contrast, polystyrene beads, which non-specifically interact with cellular proteins, showed a negative velocity-size correlation in the MTOC-directed transport in XE (Fig. <a href="#MOESM1" class="usa-link">S7</a>), suggesting that PSS in aggregate transport is mediated by specific interactions between dynein and aggregates. We also confirmed that the average velocity of aggregates remained approximately constant within the single-particle tracking range in XE (50–150 μm from the MTOC) and in live cells (Fig. <a href="#MOESM1" class="usa-link">S8</a>), and the velocity of large aggregates remained consistently higher than that of small aggregates, suggesting that the observed PSS is not affected by the distance from the MTOC. Large aggregates slowed down in the close vicinity of the MTOC, which may be caused by molecular crowding or change of MT organization in the close zone<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>.</p>
<p id="Par19">PSS was robustly observed across experimental replicates or using different batches of extracts (Fig. <a href="#MOESM1" class="usa-link">S9</a>). Inhibiting the activities of proteasome, Hsp70, or Hsp90 did not perturb size selectivity, whereas inhibiting the catalytic activity of HDAC6 by Tubastatin A (TubA) reduced the average velocity and increased the size threshold for undergoing MTOC-directed transport (Fig. <a href="#MOESM1" class="usa-link">S10</a>), consistent with the role of HDAC6 in aggresome formation in cells (Fig. <a href="#Fig1" class="usa-link">1C</a>). The effect of TubA was confirmed by fully inhibiting the deacetylation of α-tubulin, a model substrate of HDAC6 (Fig. <a href="#MOESM1" class="usa-link">S10</a>).</p>
<p id="Par20">When traveling to the MTOC, AgDD aggregates exhibited “stop-and-go” movements (Fig. <a href="#Fig2" class="usa-link">2</a>C, <a href="#Fig2" class="usa-link">E</a>). To determine which parameters of aggregate transport were modulated by size, we divided each trajectory into distinct kinetic segments. For segmentation, we fitted each trajectory with a piecewise linear function to achieve the normalized root mean square error (NRMSE) of 0.072 and designated a segment as “Pause” if the net displacement within this segment could be accounted by simple diffusion and measurement errors with a 95% confidence level (Fig. <a href="#Fig2" class="usa-link">2E</a>; Methods). Otherwise, the segment was labeled as “Transport Engaged (TE)”, to be differentiated from the processive movement at a finer timescale (see below). Aggregates alternated between these two states when moving towards the MTOC. The increase in aggregate size correlated with the lengthening of the TE segment, while the length of the pause segment remained stable (Fig. <a href="#Fig2" class="usa-link">2F</a>). As a result, the mean proportion of the pause segment duration within a trajectory decreased with the aggregate size (Fig. <a href="#Fig2" class="usa-link">2G</a>). These results reveal that larger aggregates tend to be more persistently engaged in MTOC-directed transport.</p>
<p id="Par21">The pause segments are best characterized by a freely diffusing state, because the apparent diffusion constant of AgDD aggregates scaled inversely proportional to aggregates’ diameter during pauses, as predicted by the Stokes-Einstein equation, where the cytoplasmic coefficient of viscosity was taken as 0.01 Pa·s (Fig. <a href="#Fig2" class="usa-link">2H</a>)<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Consistently, the diffusion constant in the presence of CC1, which inhibits dynein-dynactin, showed a similar dependence on aggregate diameter.</p>
<p id="Par22">While aggregates in most TE segments moved towards the MTOC, brief backward movement was occasionally detected, accounting for an average of 8.6% ± 0.39% of the observation time (Fig. <a href="#Fig2" class="usa-link">2F</a>, “TE (-)”). Kinesin-independent backward movement has been reported for the transport of dynein cargoes, although the underlying mechanism remains unclear<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>–<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. We opted to neglect the contribution from backward segments in the following analysis, as they played a minor role in determining the average velocity and did not appear to vary with size (Fig. <a href="#Fig2" class="usa-link">2F</a>). In summary, aggregate transport shows distinct features from the transport of conventional dynein cargoes, including a slow average velocity and a PSS. Both features can be recapitulated in the XE reconstitution system.</p></section><section id="Sec6"><h3 class="pmc_sec_title">The size-selection mechanism in protein aggregate transport</h3>
<p id="Par23">We next conducted high-resolution single-particle tracking experiments to investigate the detailed mechanism underlying the PSS in aggregate transport. AgDD aggregates were tracked at 30 frames per second for 2 min, with ~10 nm uncertainty in sub-pixel localization as determined using surface-immobilized fluorescent beads (see Methods). These experiments confirmed the positive correlation between average transport velocity and aggregate size (Fig. <a href="#Fig3" class="usa-link">3</a>A, <a href="#Fig3" class="usa-link">B</a>) and uncovered additional kinetic features that are important for understanding the size-selection mechanism.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Positive size selectivity in aggregate transport originates from a size-dependent increase in the likelihood of engaging in active transport.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373839_41467_2025_62751_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fea/12373839/a02b7eee8f9d/41467_2025_62751_Fig3_HTML.jpg" loading="lazy" id="d33e1041" height="888" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> The relationship between the time-averaged velocity and the diameter of individual aggregates, presented in a violin plot with a zoom-in axis on the right. 799 trajectories around 6 microtubule asters were acquired at 30 frames per second for 120 s. Error bars represent the SEM within each size group. Source data are provided as a Source <a href="#MOESM16" class="usa-link">data file</a>. <strong>B</strong> Example trajectories of AgDD aggregates. Data were from the experiment in (<strong>A</strong>). Trajectories of the 10 largest (top 2%) and the 10 smallest (bottom 2%) aggregates are plotted as faint lines (mean diameter ± standard deviation in legend). The averaged trajectories of the large and the small groups are shown as thick lines. <strong>C</strong> An example trajectory of AgDD aggregate to illustrate the result of trajectory segmentation. “Pause”: pink; “Transport (+)”: moving towards the MTOC, green; “Transport (-)”: moving away from the MTOC, blue. The instantaneous velocity was calculated as the time derivative using a 2-s rolling time window, overlaid (gray curve) on the right axis, and the probability density is shown on the side. Zoom-in windows of the shaded regions, marked by T1, T2 and T3, are shown on the right. <strong>D</strong> The probability densities of instantaneous velocities of AgDD aggregates calculated as in (<strong>C</strong>) using only the “Transport (+)” segments. 799 trajectories from (<strong>A</strong>) were divided into three groups by aggregates’ diameter: &lt;1 μm (0–33 percentile); 1–1.4 μm (33–66 percentile); &gt; 1.4 μm (66–100 percentile). The probability density of the entire population is overlaid as gray bars. Inset: a view with a local <em>y</em>-axis. <strong>E</strong> The relationship between the intrinsic velocity and the diameter of AgDD aggregates, presented in a violin plot. Intrinsic velocity was defined for each aggregate as the mean of the top 20% of instantaneous velocities calculated as in <strong>D</strong>. The mean intrinsic velocity within each size group was plotted against the inverse of the aggregate diameter in (<strong>F</strong>). Similar results were obtained using the top 10% and top 50% of instantaneous velocities in the definition of intrinsic velocity. Error bars represent the SEM in each group. Data were from the experiment in (<strong>A</strong>). <strong>G</strong> The relationship between the proportion of the transport segment and the diameter of AgDD aggregates in a violin plot. Trajectories in (<strong>A</strong>) were grouped by size and segmented into transport (+/-) and pause segments as illustrated in (<strong>C</strong>) (Methods). The mean proportion within each size group is highlighted on the right. Error bars represent the SEM in each group. <strong>H</strong> Probability densities of the lengths of pause and transport segments. Trajectories of the largest 10% and the smallest 10% aggregates in (<strong>A</strong>) were included in the analysis. The mean and median values of the segment lengths are labeled on the top panel. Lower panel: time constants for the large aggregates (0.77 ± 0.03 s and 7.24 ± 3.12 s at the 95% confidence level) and the small aggregates (0.76 ± 0.04 s and 27.79 ± 14 s) were from fitting their pause length distributions with a double exponential function. The red and green lines represent the two exponential modes of the large aggregates in a semi-log plot. Inset: ratio of the pause length distribution of the small aggregates over that of the large aggregates. The statistical significance of the difference between groups of large and small aggregates was calculated using a Student’s two-tailed <em>t</em>-test (*<em>p</em> = 0.036; ****<em>p</em> = 1×10<sup>−</sup><sup>16</sup>). <strong>I</strong> The relationship between the diffusion constants of AgDD aggregates during pause and the pause length. The apparent diffusion constant during each pause segment was calculated based on a linear regression of the mean square displacement on time, grouped by the pause length, presented in a violin plot. For each group, the median values of the pause length and diffusion constant are labeled next to the distribution, and the error bars represent the first and third quartiles. The statistical significance of the difference between groups was calculated using a Student’s one-tailed <em>t</em>-test (ns: <em>p</em> = 0.16; **<em>p</em> = 0.01; ****<em>p</em> = 2×10<sup>−11</sup>). Data were from the experiment in (<strong>A</strong>).</p></figcaption></figure><p id="Par24">Larger aggregates may be carried by a greater number of dynein molecules, leading to faster transport. Alternatively, their size could predispose them to remain longer in the transport mode. To test the former possibility, we analyzed the instantaneous velocity calculated as the time derivative of an aggregate’s trajectory (Fig. <a href="#Fig3" class="usa-link">3C</a>). Although the time-averaged velocity of an aggregate was low, the instantaneous velocity spread significantly higher, approaching the typical dynein transport velocity around 500 nm/s (Fig. <a href="#Fig3" class="usa-link">3</a>C, <a href="#Fig3" class="usa-link">D</a>). To circumvent the influence of pausing during transport and stochasticity, we chose the mean value of the top 20% instantaneous velocities within transport segments in a trajectory to represent the velocity of that aggregate after forming an active transport complex with dynein and microtubule, and called it “intrinsic velocity”. Notably, the intrinsic velocity correlated inversely with aggregate size, aligning with the expectation by the viscosity rule (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e1058"><mi>v</mi><mo>~</mo><mfrac><mrow><msub><mrow><mi>N</mi><mo>×</mo><mi>F</mi></mrow><mrow><mi>S</mi></mrow></msub></mrow><mrow><mn>3</mn><mi>π</mi><mi>η</mi><mi>d</mi></mrow></mfrac></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e1082"><mi>N</mi></math></span> is the number of dynein; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e1092"><msub><mrow><mi>F</mi></mrow><mrow><mi>S</mi></mrow></msub></math></span> is the dynein stalling force; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e1105"><mi>η</mi></math></span> is the viscous coefficient; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e1114"><mi>d</mi></math></span> is the diameter) (Fig. <a href="#Fig3" class="usa-link">3</a>E, <a href="#Fig3" class="usa-link">F</a>)<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Choosing different instantaneous velocity thresholds to define the intrinsic velocity led to the same conclusion (Fig. <a href="#Fig3" class="usa-link">3F</a>). This observation also indicates that the average number of dynein molecules actively hauling an aggregate does not change significantly with aggregate size.</p>
<p id="Par25">To understand how a PSS in the average velocity emerges from a NSS in the intrinsic velocity, we delved deeper into the transport kinetics. Notably, under high temporal resolution, the movement of most aggregates appeared highly episodic. Trajectories with a net MTOC-directed movement over a 2-min observation period, corresponding to the TE segments defined previously (Fig. <a href="#Fig2" class="usa-link">2E</a>), are indeed composed of numerous short, processive transport segments interspersed with pauses of varying lengths (Fig. <a href="#Fig3" class="usa-link">3B, C</a>). The shape of the trajectory also varied with the aggregate size, with larger aggregates’ trajectories appearing “smoother” (Fig. <a href="#Fig3" class="usa-link">3B</a>). We applied a piecewise linear function with up to 153 segments to fit each trajectory, achieving a NRMSE of 0.0217. A segment was classified as “Transport”, if it exhibited a nonzero transport velocity with 90% confidence (Fig. <a href="#Fig3" class="usa-link">3C</a>; Methods); otherwise, it was classified as “Pause”. By analyzing the proportion of each kinetic mode, we concluded that larger aggregates were more likely to reside in the transport state (Fig. <a href="#Fig3" class="usa-link">3G</a>). This gain of transport probability was sufficiently large to offset the decrease in intrinsic velocity of larger aggregates due to viscosity.</p>
<p id="Par26">We next examined the time distribution of different kinetic segments to understand their contributions to the size-dependent modulation of the transport likelihood. The distribution of pause segment length was best described by a double exponential function, containing distinctly separated time constants. The fast time constant was 0.77 ± 0.03 s and remained invariant across sizes, whereas the slow time constant was about 3.8 times longer for the smallest 10% of the aggregates compared to the largest 10%, resulting in a significantly shorter mean pause length for the larger aggregates (Fig. <a href="#Fig3" class="usa-link">3H</a>). These findings indicate that larger aggregates more readily resume transport after a pause. The distribution of transport time did not appear to vary with size, as the smallest 10% of aggregates had an essentially identical distribution as the largest 10% (Fig. <a href="#Fig3" class="usa-link">3H</a>).</p>
<p id="Par27">In addition, we found that the apparent diffusion constant of aggregates in the pause state appeared to correlate with the pause duration: shorter pause segments were predominantly associated with slower diffusion, suggesting a stationary or motion-restricted state (Fig. <a href="#Fig3" class="usa-link">3I</a>) (see Discussion). In summary, a diffusing aggregate engages with the transport machinery for multiple rounds en route to the MTOC, and its movement is episodic at both long and short timescales. The PSS in transport velocity is primarily caused by the greater propensity of larger aggregates to participate in active transport, which effectively offsets the reduction in their intrinsic velocity due to viscosity.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Aggresome adapters mediate episodic transport in <em>Xenopus laevis</em> egg extract</h3>
<p id="Par28">The size-dependent modulation of the transport likelihood requires the likelihood value to be non-saturating or significantly less than one in the typical size range, which can be achieved through transport episodicity. We next studied whether transport episodicity depends on the properties of cargo or cargo adapters. To this end, we used a chemically defined cargo with specific adapters.</p>
<p id="Par29">We first identified factors mediating the aggregate transport in XE through characterizing the interactome of AgDD aggregates. We aggregated AgDD in XE and coupled the aggregates to Dynabeads, using soluble AgDD that was not transported by dynein as a control. After incubation in XE, the bead-bound proteins were analyzed by Tandem-Mass-Tag mass spectrometry (TMT-MS). This analysis identified several protein chaperones that were enriched with AgDD aggregates, including Hsp90, DNAJAs, and the known aggresome mediators Hsp70, together with a dynein subunit (Fig. <a href="#MOESM1" class="usa-link">S11</a>; Table <a href="#MOESM1" class="usa-link">S1</a>; Supp. Data <a href="#MOESM12" class="usa-link">1</a>).</p>
<p id="Par30">Several chaperonin (TRiC/CCT) subunits also showed enrichment with AgDD aggregates. Previous studies discovered that chaperonin subunits were recruited to the aggresome and interacted with DIC, suggesting a potential role of the chaperonin in aggresome formation<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. We found that the chaperonin subunits, CCT2 and CCT8, were enriched at the periphery of the aggresome formed by polyglutamine (polyQ) protein Htt(Q94)-CFP in HEK293T and U2OS cells by immunofluorescence staining (Fig. <a href="#MOESM1" class="usa-link">S12G-I</a>), akin to other aggresome adapters, namely HDAC6 and p62, with polyQ aggregates<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>–<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. However, no obvious enrichment of chaperonin at AgDD aggregates was observed. To further explore chaperonin’s role in aggresome formation, we transiently knocked down CCT2, CCT4, or CCT8 in HEK293T cells by RNAi and studied the consequences on aggresome formation (Fig. <a href="#MOESM1" class="usa-link">S12A</a>, <a href="#MOESM1" class="usa-link">S12B</a>). Depleting chaperonin subunits resulted in a modest, yet statistically significant, delay in AgDD aggresome formation, with little impact on cell growth (Fig. <a href="#MOESM1" class="usa-link">S12C</a>, <a href="#MOESM1" class="usa-link">S12D</a>). Similarly, a minor delay was observed in cells depleted of HDAC6 or BAG3 (Fig. <a href="#MOESM1" class="usa-link">S12D</a>). Consistent with the involvement of redundant aggresome pathways, double knockdowns of aggresome adapters resulted in longer delays in aggresome formation (Fig. <a href="#MOESM1" class="usa-link">S12D</a>). This delay was not unique to the AgDD system, as chaperonin knockdown also retarded polyQ aggresome formation in HEK293T cells (Fig. <a href="#MOESM1" class="usa-link">S12E</a>, <a href="#MOESM1" class="usa-link">S12F</a>).</p>
<p id="Par31">We then used beads coated with the identified factors or known dynein adapters, to address whether these factors can mediate cargo transport in XE and whether they incur different transport kinetics, which may be associated with distinct size selectivity. We prepared 2.8 µm-diameter fluorescent beads coated with specific antibodies to precipitate different factors from HeLa cell extract (Fig. <a href="#Fig4" class="usa-link">4A</a>; Methods). TMT-MS verified that each antibody-coated bead primarily precipitated its designated antigen (Table <a href="#MOESM1" class="usa-link">S2</a>; Supp. Data <a href="#MOESM13" class="usa-link">2</a>). No dynein or dynactin component was detected on beads by TMT-MS, suggesting their low abundance.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Aggresome adapters mediate dynein-dependent transport in <em>Xenopus laevis</em> egg extract (XE).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373839_41467_2025_62751_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fea/12373839/aaa0974014fd/41467_2025_62751_Fig4_HTML.jpg" loading="lazy" id="d33e1302" height="559" width="667" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> schematic workflow for studying the transport of adapter-coated beads in XE. Different adapters were immunoprecipitated from HeLa S3 extract using Dynabeads. Adapter-coated beads were incubated in interphase XE with Alexa647-labeled microtubule asters (cyan), imaged every 1 min for 30 min at 18 °C. Representative images from an experiment with chaperonin (CCT) coated beads (red) are shown on the right. Experiments were performed two times independently with similar results. <strong>B</strong> The averaged trajectories of different factor-coated beads in XE. Experiments were performed as described in (<strong>A</strong>), except that HOOK2 was immunoprecipitated from XE using Dynabeads as a positive control. Transport of AgDD aggregates (AgDD<sub>agg</sub>) was included for comparison. Line shading represents SEM. n: number of trajectories. <strong>C</strong> Transport segment length distribution for beads coated with indicated factors in a violin plot. Experiments were performed as in (<strong>A</strong>), but acquired at 30 frames per second for 120 s. The distributions of the transport segment proportion and the intrinsic velocity are presented in (<strong>D</strong>) and (<strong>E</strong>), respectively. Intrinsic velocity was defined as the mean of the top 20% of instantaneous velocities as in Fig. <a href="#Fig3" class="usa-link">3E</a>. n: number of trajectories. <strong>F</strong> The average velocity of beads with different surface density of HDAC6. Beads were prepared as described in (<strong>A</strong>), but using anti-HDAC6 antibody diluted with random IgG at the indicated ratios (%). The XE transport assay was performed as described in (<strong>A</strong>), but using Tau-mCherry to label microtubules. Average velocities of individual particles were calculated as in Fig. <a href="#Fig2" class="usa-link">2D</a> and are displayed in a violin plot with mean values indicated. n: number of trajectories. <strong>G</strong> Positive size selectivity for HDAC6-coated beads. Fluoresbrite® YG Carboxylate Microspheres with 1 μm or 0.5 μm diameter were passivated with PEG and then coated with HDAC6 (see Methods) or left untreated (“PEG only”). Transport assay and data analysis were performed as described in (<strong>F</strong>). n: number of trajectories. Representative images of HDAC6-coated beads at 30 min into the transport assay are shown in (<strong>H</strong>) (experiments were performed two times independently with similar results).</p></figcaption></figure><p id="Par32">Upon incubating adapter-coated beads in XE containing microtubule asters, we recorded efficient MTOC-directed transport of beads coated with HDAC6, SQSTM1/p62, and Hsc70 at an average velocity of 21 ~ 50 nm/s, similar to the transport of AgDD aggregates at 11 ~ 32 nm/s (Fig. <a href="#Fig4" class="usa-link">4B</a>). These activities of known aggresome adapters support the biological relevance of our reconstitution system. Notably, we recorded efficient MTOC-directed transport of CCT8-coated beads at a similar velocity (Figs. <a href="#Fig4" class="usa-link">4</a>A, <a href="#Fig4" class="usa-link">B</a>; Supp. Movie <a href="#MOESM10" class="usa-link">8</a>). These results suggest a likely role of the chaperonin, or its subcomplex, as another dynein adapter for protein aggregates. In contrast, the transport of proteasome- or Hsp90-coated beads was inefficient and indistinguishable from that of random-IgG-coated beads (Fig. <a href="#Fig4" class="usa-link">4B</a>), suggesting these factors by themselves may not directly engage with the transport machinery dynein.</p>
<p id="Par33">Beads coated with the dynein-activating adapter HOOK2 traveled at 480 nm/s, much faster than beads coated with the aggresome adapters (Fig. <a href="#Fig4" class="usa-link">4B</a>). Consistently, at high temporal resolution, the movement of aggresome adapter-coated beads was highly episodic, characterized by short processive runs punctuated by frequent pauses, akin to the transport of AgDD aggregates (Fig. <a href="#Fig4" class="usa-link">4C</a>). In contrast, HOOK2-coated beads exhibited a higher intrinsic velocity, and the movement was mostly devoid of short pauses (Fig. <a href="#Fig4" class="usa-link">4C–E</a>). Overall, the transport mode occupied ~40% of the observation time for the aggresome adapter-coated beads, whereas HOOK2-coated beads consistently maintained their transport velocity (Fig. <a href="#Fig4" class="usa-link">4D</a>). The similarity in transport kinetics of AgDD aggregates and aggresome-adapter-coated beads suggests that the aggregate transport in XE is likely to be mediated by the aggresome adapters, which is also corroborated by the effects of chemical or genetic perturbations of those adapters on the transport kinetics (Figs. <a href="#MOESM1" class="usa-link">S10</a>, <a href="#MOESM1" class="usa-link">S12</a>).</p>
<p id="Par34">In conclusion, our results show that aggresome adapters mediate episodic transport of protein aggregates, resulting in a lower likelihood of the cargo staying in the active transport state.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Aggresome adapter mediates positive size selectivity in dynein transport</h3>
<p id="Par35">To study what determines the velocity of aggresome-adapter-mediated transport, we first varied the density of HDAC6 on beads by serially diluting anti-HDAC6 antibodies with random IgG. Beads coated with a high density of HDAC6 moved fast towards the MTOC in XE, while reducing HDAC6 density slowed down the transport, suggesting the average velocity is not an intrinsic feature of the dynein motor (Fig. <a href="#Fig4" class="usa-link">4F</a>, <a href="#MOESM1" class="usa-link">S13A</a>).</p>
<p id="Par36">To address whether an aggresome adapter is sufficient to confer PSS, we varied the bead size and studied its effect on transport velocity. To this end, we developed a new passivation and conjugation protocol using polystyrene beads since Dynabeads are of the same size (see Methods). While untreated polystyrene beads that nonspecifically bound to dynein exhibited NSS in transport (Fig. <a href="#MOESM1" class="usa-link">S7</a>), passivation by PEG prevented nonspecific transport of these beads (Fig. <a href="#Fig4" class="usa-link">4G</a>). Further functionalization of the passivated beads with HDAC6 via antibody restored MTOC-directed transport (Fig. <a href="#Fig4" class="usa-link">4G</a>, <a href="#MOESM1" class="usa-link">S13B</a>). Notably, 1.0 μm HDAC6 beads moved at 20 nm/s on average, while 0.5 μm beads showed minimal movement, echoing the observation that the net velocity of small AgDD aggregates of less than 0.7 μm was close to zero in XE (Fig. <a href="#Fig2" class="usa-link">2D</a>). Overall, these results with chemically-defined cargoes support the conclusion that an aggresome adapter, such as HDAC6, is sufficient to confer PSS in dynein transport in a surface-density-dependent fashion.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Dynein’s activating adapter perturbs the normal size selectivity in aggresome formation</h3>
<p id="Par37">To examine whether the PSS in aggregate transport is specifically associated with the aggresome adapters in cells, we tethered dynein’s activating adapters, HOOK2 or HOOK3, to protein aggregates by expressing AgDD fused with these adapters in U2OS cells and studied the transport of the corresponding substrates. Anti-GFP immunoblotting suggested that most GFP signal was from the full-length fusion protein (Fig. <a href="#Fig5" class="usa-link">5A</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Dynein’s activating adapters HOOK2 and HOOK3 are associated with a negative size selectivity in aggregate transport.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373839_41467_2025_62751_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fea/12373839/51f1df9d7880/41467_2025_62751_Fig5_HTML.jpg" loading="lazy" id="d33e1428" height="476" width="799" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Anti-GFP western blot of U2OS cells stably expressing AgDD (41 kDa), AgDD-HOOK2 (124.5 kDa) and AgDD-HOOK3 (124.4 kDa). Blot was reprobed with anti-GAPDH antibody as the loading control. Source data are provided as a <a href="#MOESM16" class="usa-link">Source data</a> file. Experiment was performed one time. <strong>B</strong> Schematic and live-cell confocal images of U2OS cells stably expressing AgDD (top) or AgDD-HOOK2 (bottom), before and after Shield-1 removal. Peripheral aggregates were identified by <em>TrackMate</em> and circled in yellow at “10 min” as an example where 55 particles were identified in the AgDD cell, and 2 in the AgDD-HOOK2 cell. <strong>C</strong> Classification of the aggregation-to-aggresome phenotypes, presented as in Fig. <a href="#Fig1" class="usa-link">1D</a>. Bar plot shows the mean ± SEM, over 4 different FOVs. n: number of cells analyzed. D, P, and A are defined as in Fig. <a href="#Fig1" class="usa-link">1D</a>. A0: perinuclear puncta detected in the presence of Shield-1. Inset: cellular GFP signal in the analyzed AgDD and AgDD-HOOK2 cells before Shield-1 removal. <em>P</em>-value &gt; 0.2 by an unpaired Student’s two-tailed <em>t</em>-test. Boxplot shows mean (“x”), median (-), first and third quartiles; upper/lower whiskers extend to 1.5 × the interquartile range. <strong>D</strong> The relationship between the mean MTOC-directed velocity and the aggregate diameter, as presented in Fig. <a href="#Fig1" class="usa-link">1H</a>. Error bars represent the SEM within each size group. Source data are provided as a Source <a href="#MOESM16" class="usa-link">data file</a>. <strong>E</strong> Probability distributions of the MTOC-directed displacement over 1 min of AgDD, AgDD-HOOK2, and AgDD-HOOK3 aggregates in the experiment in (<strong>D</strong>). Dashed red lines mark the mean values.</p></figcaption></figure><p id="Par38">Upon Shield-1 removal, AgDD-HOOK2 rapidly accumulated at the perinuclear site that resembled an aggresome (Fig. <a href="#Fig5" class="usa-link">5B</a>; Supp. Movie <a href="#MOESM11" class="usa-link">9</a>). We identified the aggregates in each frame using the ImageJ plugin “<em>Trackmate</em>”, and found that, in 40.7% ± 4.62% of cells with an aggresome, fewer than three peripheral aggregates were detected before the appearance of the perinuclear punctum (Fig. <a href="#Fig5" class="usa-link">5C</a>, group “D → A”), while a typical AgDD cell showed more than 40 peripheral aggregates in this process. A perinuclear punctum of AgDD-HOOK2 was detected even in the presence of Shield-1 in 13.1% ± 1.36% of cells, much more frequently than in AgDD cells with a similar GFP level (Fig. <a href="#Fig5" class="usa-link">5C</a>, group “A0”; Supp. Movie <a href="#MOESM11" class="usa-link">9</a>). These distinct features of the aggresome formation from AgDD-HOOK2 indicate that diffusive AgDD-HOOK2 or small aggregates that are below our detection limit can be sequestered into the aggresome in these cells.</p>
<p id="Par39">In cells that formed distinguishable peripheral AgDD-HOOK2 aggregates, we tracked aggregate movement and recorded a significantly higher transport velocity compared to the AgDD aggregates (Fig. <a href="#Fig5" class="usa-link">5</a>D, <a href="#Fig5" class="usa-link">E</a>). Notably, AgDD-HOOK2 aggregates within the same size range as AgDD aggregates exhibited a NSS, with larger aggregates being transported more slowly, akin to the transport of conventional dynein cargoes (Fig. <a href="#Fig5" class="usa-link">5D</a>, right axis). AgDD fused with another activating adapter HOOK3 had a similar phenotype. Therefore, we conclude that the PSS during aggregate transport is uniquely associated with the aggresome adapters, as coupling activating adapters to protein aggregates perturbs the normal substrate selectivity in aggresome sequestration.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Physical modeling suggests that the stability of the cargo-dynein-microtubule complex determines the size selectivity in cargo transport</h3>
<p id="Par40">Although size-dependent phenomena are universal in biological systems, few biological mechanisms have been identified to account for size selectivity. We speculate that the observed PSS has a physical origin and ask whether we could establish a simple physical model to account for the main observations.</p>
<p id="Par41">Our single-particle measurements reveal that aggregate transport is highly episodic during aggresome formation. Episodic transport enables size-dependent modulation of the likelihood of the cargo dwelling in the transport mode, which serves as the primary mechanism underlying the PSS in aggresome formation. Furthermore, our kinetic analysis suggests that aggregate transport is modulated by two processes operating on different timescales. To engage in active transport, a freely diffusing protein aggregate must first be captured by a dynein motor and loaded onto the microtubule. In the slow process, the aggregates switch between the previously defined TE state and the freely diffusing state multiple times before reaching the MTOC. This TE-pause cycle accounts for ~40% of the overall PSS (Fig. <a href="#Fig2" class="usa-link">2</a>F, <a href="#Fig2" class="usa-link">G</a>). Movement within the TE state is not free of interruption but is instead characterized by frequent short pauses, which can be caused by the dissociation of dynein from either the cargo or the microtubule. Rapid reformation of the active transport complex consisting of cargo, dynein, and microtubule after a short pause indicates that the aggregate may remain in the vicinity of microtubules during this process, which is also supported by the slow diffusion observed during the short pause (Fig. <a href="#Fig3" class="usa-link">3I</a>)<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. We therefore call this fast process “Local Dynein Cycle”, which contributes ~60% of the overall PSS (Figs. <a href="#Fig6" class="usa-link">6</a>A, <a href="#Fig3" class="usa-link">3G</a>). Larger aggregates, despite slower diffusion, may be more efficiently captured by microtubules (or by microtubule-dynein). Their larger surface area and potentially more bound aggresome adapters could also favor dynein’s local reattachment. This size-dependent multivalency effect is akin to the avidity-entropy mechanism that enhances a ligand’s effective affinity with multivalent receptors<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. A physical model of dynein-mediated transport suggests the stability of the active transport complex determines cargo size selectivity.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373839_41467_2025_62751_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fea/12373839/1a9b45d010c2/41467_2025_62751_Fig6_HTML.jpg" loading="lazy" id="d33e1849" height="646" width="800" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic of the dynein transport model. See text for details. The transport complex formed by cargo, dynein, and microtubule is labeled on the graph. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e1508"><msub><mrow><mi>v</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></math></span>: intrinsic velocity of dynein transport. <strong>B</strong> Diagram showing the proposed mechanism that determines the size selectivity in cargo transport when the transport complex has low or high stability. The scaling of the average transport velocity <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e1525"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> as a function of cargo diameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e1538"><mi>d</mi></math></span> was derived under two scenarios: (1) When the transport complex is transient or has low stability, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e1547"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> changes in proportion to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e1561"><mi>α</mi><mo>⋅</mo><msup><mrow><mi>d</mi></mrow><mrow><mn>1.5</mn></mrow></msup></math></span>, which gives rise to PSS. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e1576"><mi>α</mi></math></span> is the probability of the freely-diffusing state as a function of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e1585"><mi>d</mi></math></span> (Fig. <a href="#Fig2" class="usa-link">2G</a>); (2) When the transport complex has high stability, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e1597"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> changes in proportion to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e1610"><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>α</mi></mrow><mo>)</mo></mrow><mo>⋅</mo><msup><mrow><mi>d</mi></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></math></span>, which gives rise to NSS. Expressions were derived based on an analytical model of the system (see Methods for details). <strong>C</strong> Comparison of the model predictions with experimental measurements. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e1636"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e1649"><mi>α</mi></math></span> were obtained from the experiments as in Fig. <a href="#Fig2" class="usa-link">2D</a> (<em>n</em> = 1154 aggregates), Fig. <a href="#Fig2" class="usa-link">2G</a> (<em>n</em> = 851 aggregates). Experimental results are replotted as the mean ± SEM of either <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e1671"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>/</mo><mi>α</mi></math></span> (blue, left axis, by the low-stability expression) or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e1686"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>/</mo><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>α</mi></mrow><mo>)</mo></mrow></math></span> (red, right axis, by the high-stability expression), against <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e1707"><mi>d</mi></math></span> in a log-log plot. Blue line represents the linear regression using the low-stability expression with the fitted slope <em>k</em> 1.36 and correlation coefficient <em>r</em> 0.96. The red dashed line is the best-fitting result by the high-stability expression (slope = −1). Source data are provided as a Source <a href="#MOESM16" class="usa-link">data file</a>. <strong>D</strong> Stochastic simulation of size selectivity in dynein-mediated transport. Simulated average transport velocity, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e1729"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span>, of single particles with different diameters when the stability of the transport complex was set to low (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e1742"><msub><mrow><mi>p</mi></mrow><mrow><mn>6</mn></mrow></msub></math></span> = 0.1; see Methods). Data are presented as empty circles connected by dashed lines and replotted as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e1755"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>/</mo><mi>α</mi></math></span> against diameter, with a linear regression fit (slope <em>k</em>) displayed in the log-log plot in (<strong>E</strong>). Simulation was performed using a numerical model of (<strong>B</strong>) (see Methods for details). <strong>F</strong> Stochastic simulation results when the stability of the transport complex was set to high (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e1783"><msub><mrow><mi>p</mi></mrow><mrow><mn>6</mn></mrow></msub></math></span> = 0.0001), presented as in (<strong>D</strong>). Data are replotted as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e1799"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>/</mo><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>α</mi></mrow><mo>)</mo></mrow></math></span> against diameter in (<strong>G</strong>). <strong>H</strong> Stochastic simulation results by continuously varying the transport complex stability parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e1827"><msub><mrow><mi>p</mi></mrow><mrow><mn>6</mn></mrow></msub></math></span> from 0.0001 to 0.1, denoting a decrease in stability (color-coded from red to blue). Data are presented as a surface plot, with white lines overlaid to indicate the velocity-diameter relationship at different <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e1840"><msub><mrow><mi>p</mi></mrow><mrow><mn>6</mn></mrow></msub></math></span> values.</p></figcaption></figure><p id="Par42">We next address the important question of whether this conceptual model is physically achievable and whether it is sufficient to account for the PSS in aggregate transport. Specifically, can the size-dependent modulation of transport likelihood through dynein rebinding cycles provide enough size bias to offset the reduction in intrinsic velocity due to viscosity? Furthermore, which system parameter dictates different size selectivities? While the mechanism of the size selectivity in the TE cycle is not entirely clear (see Discussion), we rewrote the Local Dynein Cycle in Fig. <a href="#Fig6" class="usa-link">6A</a> using statistical physics, which predicted how an aggregate’s average transport velocity should vary with its size (see Methods).</p>
<p id="Par43">In this exercise, we recognized that the stability of the cargo-adapter-dynein-microtubule transport complex critically determined the size selectivity and derived the analytic expressions for the average velocity <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e1862"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> under two conditions (Fig. <a href="#Fig6" class="usa-link">6B</a>): (1) Transient transport complex, whereby most cargoes are not engaged in active transport under a steady state. We obtained <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><math id="d33e1878"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>/</mo><mi>α</mi><mrow><mo>(</mo><mrow><mi>d</mi></mrow><mo>)</mo></mrow><mo>∝</mo><msup><mrow><mi>d</mi></mrow><mrow><mn>1.5</mn></mrow></msup></math></span>, i.e., a PSS, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><math id="d33e1904"><mi>d</mi></math></span> is the aggregate diameter. The contribution of the TE cycle in size selectivity is subtracted by dividing <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><math id="d33e1913"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><math id="d33e1927"><mi>α</mi><mfenced close=")" open="("><mrow><mi>d</mi></mrow></mfenced></math></span> which is the likelihood of cargo dwelling in the freely diffusing pause state (Fig. <a href="#Fig2" class="usa-link">2G</a>); (2) Stable transport complex, whereby most cargoes in the local dynein cycle are engaged in active transport. This gave rise to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><math id="d33e1942"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>/</mo><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>α</mi><mfenced close=")" open="("><mrow><mi>d</mi></mrow></mfenced></mrow><mo>)</mo></mrow><mo>∝</mo><msup><mrow><mi>d</mi></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></math></span>, i.e., a NSS if <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><math id="d33e1973"><mi>α</mi><mfenced close=")" open="("><mrow><mi>d</mi></mrow></mfenced></math></span> does not change dramatically with the cargo size, as we observed for AgDD aggregates. It is worth noting that these scaling exponents are determined by the system’s geometry rather than specific kinetic parameters in the model. The experimentally determined exponent of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><math id="d33e1985"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>/</mo><mi>α</mi><mrow><mo>(</mo><mrow><mi>d</mi></mrow><mo>)</mo></mrow></math></span> on <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><math id="d33e2005"><mi>d</mi></math></span> is 1.36 (Fig. <a href="#Fig6" class="usa-link">6C</a>), closely aligning with the theoretical prediction in the first scenario and indicating a low stability of the transport complex. Low transport complex stability is also corroborated by the observation that aggregates and aggresome-adapter-coated beads were transported in short bursts. In contrast, the movement of HOOK2-coated beads was more processive with fewer short pauses, suggesting a more stable transport complex. The stability of the active transport complex potentially accounts for the NSS in the transport of AgDD-HOOK2/HOOK3 aggregates.</p>
<p id="Par44">To validate the conclusion of the analytical model, we simulated the stochastic transitions between the system states with arbitrary stability of the transport complex (see Methods). This stochastic model recapitulated the positive or negative scaling between <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><math id="d33e2020"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><math id="d33e2033"><mi>d</mi></math></span> when the stability of the transport complex was set to low or high, respectively (Fig. <a href="#Fig6" class="usa-link">6D, E</a>, “low-stability” case; Fig. <a href="#Fig6" class="usa-link">6F, G</a>, “high-stability” case). The model also delineated the transition from NSS to PSS as the stability parameter varied continuously (Fig. <a href="#Fig6" class="usa-link">6H</a>), reinforcing the insights from the mechanistic model.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par45">Efficient targeting of misfolded proteins into the appropriate quality-control pathway is crucial for maintaining protein homeostasis. Aggresome formation preferentially sequesters aggregated misfolded proteins over their soluble counterparts. To elucidate the mechanism underlying the “size filter” effect, we reconstituted the dynein-mediated MTOC-directed transport of protein aggregates in a cell-free system, gaining valuable mechanistic insights.</p>
<p id="Par46">Transport of protein aggregates by dynein exhibits exquisite features, highlighting a PSS, in contrast to the transport of conventional dynein cargoes, where transport velocity and cargo size negatively correlate. The PSS in aggregate transport appears to be specifically associated with the aggresome adapters, as artificially attaching an activating adapter to the aggregates reversed size selectivity. In searching for the size-selection mechanism, we found that a simple physical model was able to recapitulate the scaling of the average transport velocity with aggregate size. This model suggests that the PSS in aggregate transport may not be attributed to a specific aggresome adapter. Rather, it is a general feature originating from the recurrent formation of the cargo-dynein-microtubule transport complex. This rebinding process can manifest an avidity entropy effect that favors larger aggregates in the transport state to overcome the reduction in their intrinsic velocity. Similar mechanisms have been proposed to enhance the receptor-ligand and enzyme-substrate interaction through multivalency<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>–<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>.</p>
<p id="Par47">To allow size-dependent modulation, the likelihood of cargo dwelling in the transport mode should not saturate, to maximize the avidity entropy effect. Having a low likelihood of transport, as mediated by the aggresome adapters, requires the cargo-dynein-microtubule transport complex to be transiently formed or unstable. Although the molecular basis for this condition has not been clearly elucidated, HDAC6 has been suggested to interact weakly with dynein<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Such weak dynein interaction may be common among the aggresome adapters, since they do not share the coiled-coil domain in the dynein-activating adapters, which extensively interacts with dynein-dynactin. The interaction between the aggresome adapter and aggregates may also contribute to the instability of the transport complex. For example, previous studies suggest that HDAC6 interacts with the free C-terminus of Ub (or Ub chains) trapped in aggregates<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. The high concentration of free Ub in the cytosol is likely to compete in this process, effectively weakening the interaction between HDAC6 and aggregates. In addition, a dynein motor associated with an aggresome adapter may have lower processivity on the microtubule, compared to dynein complexed with an activating adapter<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Interestingly, dynein appears to transiently interact with certain endosomal cargoes in HeLa cells and the interaction is presumably mediated by the activating adapters<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. The stability of the transport complex may also be modulated by dynein cofactors or through post-translational modifications<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. These effects may collectively strengthen the avidity entropy effect.</p>
<p id="Par48">Avidity entropy effect is a local effect requiring the cargo to approach the microtubule for transport complex formation. How the TE state is modulated by aggregate size is still unclear, and we treat this effect as a phenomenological parameter α in the model (Fig. <a href="#Fig6" class="usa-link">6B</a>). Modulation of the TE state may be achieved through different mechanisms. A super-resolution microscopy study observed that endosomal particles in transit remain close to microtubules after dynein dissociation in live cells<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. This dynein-independent microtubule tethering may be mediated by dynactin subunit p150<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Alternatively, cargoes may dwell close to the microtubule simply due to confined diffusion<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. The microtubule surface also appears to engage beads coated with a variety of proteins to perform one-dimensional diffusion along the microtubule<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. These mechanisms may prolong the local dynein cycle and enhance the size selectivity in transporting aggregates.</p>
<p id="Par49">In the current model, we only considered transport mediated by a single dynein molecule, which is supported by our results. The inverse correlation between the intrinsic velocity and aggregate size suggests that the number of dynein molecules mediating transport does not change significantly with size. The occasional processive backward transport, as reported in previous studies, is a signature of single-dynein-mediated transport<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. We cannot exclude the possibility of multi-dynein-mediated transport, especially when the dynein density is high on the cargo surface. Another limitation of the model is that it does not take into account any sudden change in dynein behavior while remaining bound. However, such effects are unlikely to be involved in cargo size selectivity.</p>
<p id="Par50">Protein aggregates are an important class of dynein cargo. Besides selectively targeting misfolded proteins into the aggresome, there may be other important advantages associated with the PSS. We previously reported that small clusters of aggregation-prone proteins are permanently present in the cell and serve to nucleate aggregates upon proteotoxic conditions<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. The PSS may be necessary to retain these nucleation sites throughout the cytoplasm to capture abnormal polypeptide species in the emerging stress. Our study indicates that the choice of cargo adapter is constrained by a trade-off between the transport speed and size selectivity: activating adapters may allow faster transport but favor smaller cargos; on the contrary, when PSS is desired, an aggresome adapter can be employed, but at the expense of the transport velocity.</p>
<p id="Par51">Multiple aggresome adapters mediate protein aggregate transport, while their substrate specificities are not fully understood. Using the reconstitution system developed in this study, we identified the chaperonin complex as sufficient to mediate dynein-dependent cargo transport and to promote efficient aggresome formation. Additional adapters may also contribute. For example, although YTHDF2 was not enriched on AgDD aggregates, its knockdown impaired AgDD aggresome formation in cells, especially of larger aggregates, mirroring the HDAC6 knockdown phenotype (Fig. <a href="#MOESM1" class="usa-link">S14</a>). Differences between live cells and the XE system may explain adapter discrepancies: live cells capture physiological complexity but are subject to confounding factors such as cytoplasmic heterogeneity and opposing motor activities, while XE isolates dynein-driven transport but lacks full cellular regulation, including organelle interactions. Despite these differences, both systems consistently highlight HDAC6, Hsp70, and chaperonin as key transport mediators (Fig. <a href="#Fig4" class="usa-link">4B</a> and <a href="#MOESM1" class="usa-link">S12</a>), strengthening the conclusion that unstable dynein-aggregate complexes—modulated by aggresome-specific adapters—enable selective aggresome sequestration of large aggregates. Importantly, adapter dependence may vary across aggresome models. For example, although ubiquitylation is commonly required for aggresome formation associated with proteasome inhibition, it appears to play a minimal role in AgDD aggresome formation (Fig. <a href="#Fig1" class="usa-link">1C</a>). Further studies are needed to define the specificities of aggresome adapters and their functional differences.</p>
<p id="Par52">In addition to PQC, many biological processes also involve the transport of protein aggregates or condensates. The avidity entropy effect we identified may be universal and can be applied to understand the specificity of other biological pathways processing protein aggregates.</p></section><section id="Sec12"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec13"><h3 class="pmc_sec_title">Plasmids</h3>
<p id="Par53">AgDD-sfGFP was cloned from the original plasmid (Addgene, #78289) into the pLenti6.3/V5-DEST gateway vector (Invitrogen) under the control of a CMV promoter. An HA-tag was inserted at its N-terminus. The open reading frames of human HOOK3 (Thermo Ultimate™ Human ORF Clones, #HORF96) and human HOOK2 (WZ Biosciences Inc.) were amplified and appended to the C-terminus of AgDD-sfGFP to create AgDD-sfGFP-HOOK3 and AgDD-sfGFP-HOOK2. HaloTag-Histone H2A in a retroviral vector with a hygromycin selection marker was from the Mitchison lab<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. Htt(Q94)-CFP in a lentiviral vector under the control of a Tet-On promoter was reported in a previous study<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>, while the Tet-Off version was created by inserting the same gene into the pCW57.1 vector (Addgene, #100521). The construct for purifying recombinant cys-Ub<sup>L73F</sup> was described previously<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. The construct for expressing recombinant HA-AgDD-sfGFP in <em>E. coli</em> was made by inserting the target gene into the pTXB1 vector (NEB) through the SapI, NdeI restriction sites. Histone H2B-mCherry in the CSII-EF lentiviral vector was a gift from Tobias Meyer<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. 6xHis-FKBP(F36V) in the pET15b vector was a gift from Thomas Wandless (Addgene, #73180)<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Recombinant protein purification</h3>
<p id="Par54">6xHis-FKBP(F36V) was purified from <em>E. coli</em> cells using His60 Ni Superflow Resin (Takara Bio, #635662) according to the manufacturer’s instructions. Eluted proteins were buffer-exchanged and concentrated to 1.75 mM in PBS + 10% glycerol and stored at −80 °C for future use. Protein concentration was determined using the Bio-Rad protein assay.</p>
<p id="Par55">HA-AgDD-sfGFP was purified from <em>E. coli</em> cells via the intein-tag at the C-terminus on Chitin Resin (NEB, #S6651L). 20 mL of resin was used for 1 L of cell culture. Cells were lysed by sonication in the wash buffer (30 mM Tris-HCl pH 8.0, 300 mM NaCl, 2 mM MgCl<sub>2</sub>, 5% glycerol, 1 mM PMSF, 0.5 mM EDTA) plus 0.5% Tween-20 and centrifuged at 35,000 × <em>g</em> for 30 min at 4 °C. The supernatant was incubated with the Chitin beads (NEB, # S6651L) for 1 h at 4 °C. Beads were washed twice with the wash buffer + 0.5% Tween-20 and twice with the wash buffer before being resuspended in the cleavage buffer (wash buffer + 50 mM dithiothreitol (DTT), 20 μM Shield−1) of 3 times the bead volume and rotated at 4 °C for 72 h to release HA-AgDD-sfGFP. Flow-through was collected, and proteins were further eluted three times using the wash buffer (+20 μM Shield-1) of 1/3 of the bead volume, with 10-min incubation in between. Eluted proteins and the flow-through were combined and concentrated by 10 kDa Amicon Ultra-15 centrifugal filters (Millipore, #UFC901008). The sample was buffer-exchanged once into the wash buffer (+20 μM Shield-1) using the same concentrator to remove 90% of the DTT in the solution. Protein concentration was determined by absorption at 488 nm using the extinction coefficient of sfGFP<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup> as 83300 M<sup>−1 </sup>cm<sup>−1</sup>. ~5 mg of AgDD was obtained from 1 L of culture. Proteins were aliquoted and stored at 200–400 μM at −80 °C. Multiple freeze-thaw cycles were avoided.</p>
<p id="Par56">Ubiquitin chains were synthesized using human ubiquitin mutant Ub<sup>L73F</sup> to minimize nonspecific deubiquitylation in the extract<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. Ub<sup>L73F</sup> with a cysteine residue and a 6xHis-tag at the N-terminus was purified from <em>E. coli</em> and labeled with Dylight-550-maleimide (Pierce, #62290) as described previously<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. Unreacted cysteines of Ub<sup>L73F</sup> were alkylated by N-Ethylmaleimide. Human E1 and E2-25K were purified from <em>E. coli</em><sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. Uev1a and UbcH13 were purchased from R&amp;D Systems.</p></section><section id="Sec15"><h3 class="pmc_sec_title">HeLa S3 cell extract preparation</h3>
<p id="Par57">HeLa S3 extract was prepared as described previously in ref. <sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. Briefly, 2 L of spinner culture of asynchronous HeLa S3 cells were collected and homogenized using nitrogen cavitation in the swelling buffer (20 mM HEPES pH 7.5, 5 mM KCl, 1.5 mM MgCl<sub>2</sub>, 1 mM DTT, 1× protease inhibitor cocktail). The homogenate was sequentially centrifuged at 5000 × <em>g</em> for 10 min and 20,000 × <em>g</em> for 30 min. The supernatant was collected, aliquoted, and stored at −80 °C.</p></section><section id="Sec16"><h3 class="pmc_sec_title">AgDD Aggregates formation</h3>
<p id="Par58">Purified AgDD stabilized by Shield-1 was centrifuged at 17000×<em>g</em> for 15 min at 4 °C before use. The supernatant was added to <em>Xenopus</em> egg extract or a physiological buffer (1 mM DTT, 100 mM KCH<sub>3</sub>COOH, 30 mM KCl, 1 mM MgCl<sub>2</sub>, 1 mM Na<sub>2</sub>ATP, 10 mM Na<sub>2</sub>HPO<sub>4</sub>) at a 10 μM final concentration in the presence of 200 μM FKBP(F36V) (or 20 μM Shield-1 for uninduced controls) and incubated at 18 °C. Samples were taken every 10 min after 30-min incubation and examined on microscope slides using an epi-fluorescent microscope equipped with a GFP channel filter and a 10x objective. Aggregates were allowed to form for an additional 30 min after they were first visible under the scope. Samples were then kept on ice and used for experiments on the same day.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Synthesis of fluorescent ubiquitin chains</h3>
<p id="Par59">Ubiquitin chains with K48 or K63 linkage were synthesized in reactions containing 100 μM Ub<sup>L73F</sup> and 1 μM DyLight 550-labeled Ub<sup>L73F</sup>, 0.2 μM E1, 15 μM E2-25K (for K48 linkage) or 5 μM Uev1a/UbcH13 (for K63 linkage) in the buffer (15 mM HEPES pH 7.5, 10 mM ATP, 10 mM MgCl<sub>2</sub>, 0.6 mM DTT, and 1 mM PMSF) and incubated at 37 °C for 5 h. The product was buffer-exchanged using a Zeba 40 K column (Thermo Scientific, #87764) into PBS. Protein concentration was determined by SDS-PAGE and Coomassie staining using BSA as a concentration standard.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Conjugation of ubiquitin chains to dynabeads</h3>
<p id="Par60">Dynabeads M-270 beads with amine groups were washed twice and resuspended in the freshly prepared passivation buffer (500 mM K<sub>2</sub>SO<sub>4</sub>, 100 mM NaHCO<sub>3</sub>). We used 5 kDa PEG plus 2.5% 5 kDa biotin-PEG (LaysanBio, mPEG-SVA-5000; Biotin-PEG-SVA-5000) in a “clouding-point” solution as described previously to passivate the beads for 2 h at 25 °C with slow tilt and rotation. Beads were then washed twice with TBS with 0.5 mg/mL BSA, 0.01% Tween-20, and resuspended to the original volume. Five microliters of beads were then incubated with 8 μg of Streptavidin (Invitrogen, #434301) for 30 min at 25 °C and washed three times with TBS.</p>
<p id="Par61">To conjugate the ubiquitin chains to the beads, 10 μL solutions of 50 μM ubiquitin chains or ubiquitin monomers were biotinylated by 100 μM NHS-PEG4-biotin (Thermo Scientific, #21363) at 25 °C for 1 h and buffer exchanged twice into PBS using a Zeba 7 K column. Samples were then incubated with 5 μL streptavidin-coated Dynabeads M-270 beads at 25 °C for 1 h and washed three times with PBS. 0.5 μL of beads was boiled and blotted by Alexa Fluor 790-streptavidin (ThermoFisher, #S11378). Biotin-XX goat anti-mouse IgG secondary antibody (ThermoFisher, #B2763) with a known concentration (2 mg/mL) and degree of biotin-XX labeling (4 biotins per antibody) was used as the standard. Surface density of ubiquitin on the beads was estimated as below:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e2285" display="block"><mi>#</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">ubiquitins</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">per</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">bead</mi><mo>=</mo><mfrac><mrow><mi mathvariant="normal">intensity</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">Ub</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">bands</mi></mrow><mrow><mi mathvariant="normal">intensity</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">Biotin</mi><mo>−</mo><mi mathvariant="normal">XX</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">antibody</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">band</mi></mrow></mfrac><mo>×</mo><mfrac><mrow><mn>4</mn><mspace width="0.25em"></mspace><mo>×</mo><mi>#</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">molecules</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">the</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">antibody</mi></mrow><mrow><mn>2</mn><mi>e</mi><mn>9</mn><mspace width="0.25em"></mspace><mi mathvariant="normal">beads</mi><mo>/</mo><mi mathvariant="normal">mL</mi><mspace width="0.25em"></mspace><mo>×</mo><mn>0.5</mn><mspace width="0.25em"></mspace><mi>μ</mi><mi mathvariant="normal">L</mi></mrow></mfrac><mo>.</mo></math></td>
<td class="label">1</td>
</tr></table>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e2381" display="block"><mi mathvariant="normal">surface</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">coverage</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">rate</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">ubiquitins</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">on</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">the</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">bead</mi><mo>=</mo><mfrac><mrow><mi>#</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">ubiquitins</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">per</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">bead</mi><mspace width="0.25em"></mspace><mo>×</mo><mi>π</mi><mo>×</mo><msup><mrow><mn>1.5</mn><mspace width="0.16em"></mspace><mi mathvariant="normal">nm</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow><mrow><mn>4</mn><mi>π</mi><mo>×</mo><msup><mrow><mn>1.4</mn><mspace width="0.16em"></mspace><mi>μ</mi><mi mathvariant="normal">m</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac><mo>.</mo></math></td>
<td class="label">2</td>
</tr></table>
<p id="Par62">The labeling protocol yielded, on average, one biotin molecule per ubiquitin. Given the bead diameter as 2.8 μm and ubiquitin molecule diameter as 3 nm, the surface coverage rate of the beads by ubiquitin chains or ubiquitin monomers was estimated to be 5%.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Cell line construction and general culture conditions</h3>
<p id="Par63">WT HEK293T and U2OS cells were obtained from ATCC and cultured in DMEM supplemented with 10% heat-inactivated FBS and 2× Antibiotic-Antimycotic (ThermoFisher, #15240062) at 37 °C and 5% CO<sub>2</sub>. AgDD-sfGFP, AgDD-sfGFP-HOOK2, AgDD-sfGFP-HOOK3, and Htt(Q94)-CFP were integrated into HEK293T or U2OS cells by lentivirus, followed by blasticidin selection (InvivoGen, #ant-bl-05) for two weeks to make stable cell lines. Cells transduced with the lentivirus carrying H2B-mCherry were sorted on a Beckman Colter MoFlo Astrios EQ high-speed cell sorter to obtain pure populations expressing mCherry. Lentivirus was prepared following a standard protocol: viruses were collected from 10 cm dish 48 h post transfection of the transfer vector plasmid, packaging plasmid (Addgene, #8455) and envelope plasmid (Addgene, #8454) at a ratio of 4:2:1 and were concentrated to 600 μL by 100 kDa Amicon Ultra-15 centrifugal filters (Millipore, #UFC910008) after centrifugation at 1500 × <em>g</em> for 20 min at 4 °C. 200 μL of virus solution was added to 1 well of HEK293T or U2OS cells seeded in a 12-well plate at 60% confluency. Cells were moved to larger containers after 24 h. Cells were maintained in the same media with 1 μM Shield-1 (AOBIUS, #AOB1848). U2OS cells co-expressing AgDD-sfGFP and HaloTag-H2A were generated together using a lentiviral system (AgDD-sfGFP) and a retroviral system (HaloTag-H2A) and selected using blasticidin and hygromycin (ThermoFisher, #10687010). Transient transfection of the Htt(Q94)-CFP construct into HEK293T cells was performed using TransIT-293 Transfection Reagent (Mirus Bio, #MIR 2700) according to the manufacturer’s instructions. 1 μg/mL doxycycline (Takara Bio, #631311) was used to induce protein expression under the Tet-On promoter and to maintain repression of expression in the Tet-Off system. MCF10A cells stably expressing GFP-tagged synphilin 1 were constructed by the Sherman lab in a previous study<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>.</p>
<p id="Par64">AgDD aggregation was induced by adding purified recombinant FKBP(F36V) (1.75 mM stock solution in PBS plus 10% glycerol) to the culture at a final concentration of 5 μM to deplete free Shield-1. Synphilin 1 aggregation was induced by adding 100 nM MG132 to the culture. Effects of drugs were tested by pretreating the cells with each drug for 30 min before aggregation induction, and drugs were kept in the media during the experiment.</p>
<p id="Par65">siRNAs targeting human CCT2 (13.1: sense strand 5′-GAUGGUGCCACUAUUCUAAAAAACA-3′, antisense strand 3′- UACUACCACGGUGAUAAGAUUUUUUGU-5′; 13.2: sense strand 5′-GAGUAAAUCAACCAAAACGAAUUGA-3′, antisense strand 3′-ACCUCAUUUAGUUGGUUUUGCUUAACU-5′), CCT4 (13.1: sense strand 5′- AGUGAUCUUGCAUUACACUUUCUGA-3′, antisense strand 3′- AAUCACUAGAACGUAAUGUGAAAGACU-5′; 13.2: sense strand 5′- CCUGAAGUUGUAUUGAAACAAUUTA-3′, antisense strand 3′- ACGGACUUCAACAUAACUUUGUUAAAU-5′), CCT8 (13.1: sense strand 5′- GUACUCAAGAAUCACCUGAUGUUTT-3′, antisense strand 3’-AUCAUGAGUUCUUAGUGGACUACAAAA-5′; 13.2: sense strand 5′-GGACUUGAACAGUAUGCUAUUAAGA-3′, antisense strand: 3′-GACCUGAACUUGUCAUACGAUAAUUCU-5′), HDAC6 (13.1: sense strand 5′-UUACGGGUCAUGAAGGUAGAAGACA-3′, antisense strand 3′-CGAAUGCCCAGUACUUCCAUCUUCUGU-5′; 13.2: sense strand 5′-GGAUGGCUAUUGCAUGUUCAACCAC-3′, antisense strand 3′-UACCUACCGAUAACGUACAAGUUGGUG-5′), BAG3 (13.1: sense strand 5′-GAAUAUCUGUAUGUUGGAUGACUTT-3′, antisense strand 3′-UCCUUAUAGACAUACAACCUACUGAAA-5′; 13.2: sense strand 5′-CGCAAAGAGGUGGAUUCUAAACCTG-3′, antisense strand 3′-AGGCGUUUCUCCACCUAAGAUUUGGAC-5′), and the negative control (NC) were purchased from IDT (DsiRNAs); siRNAs targeting YTHDF2 (1: sense strand 5′- CCGUUCCAUUAAGUAUAAUTT-3′, antisense strand 3′- AUUAUACUUAAUGGAACGGTT-5’; 2: sense strand 5′-GGACGUUCCCAAUAGCCAATT-3′, antisense strand 3′-UUGGCUAUUGGGAACGUCCUU-5′) were purchased from Shanghai GenePharma Co., Ltd. siRNAs were transfected to HEK293T cells using Lipofectamine™ RNAiMAX Transfection Reagent (Invitrogen, #13778150) at a final concentration of 20 nM. For double-knockdown (KD) of two genes, each siRNA was used at 10 nM. Cells were incubated at 37 °C for 24/48/72 h before being harvested for immunoblotting. Two siRNAs were tested for each target gene, and the one with the higher KD efficiency was used for functional tests unless otherwise stated. Cells were induced to form the aggresome 48 or 72 h after transfection of the siRNAs, and the time required to form a single punctum of aggresome or the transport kinetics of aggregates was measured through live-cell imaging.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Quantification of the intracellular AgDD concentration</h3>
<p id="Par66">A calibration curve between sfGFP fluorescent intensity and concentration was obtained for purified sfGFP protein using a TECAN Infinite F200 fluorescence microplate reader with a GFP filter (EX 485/20; EM 516/20). AgDD-sfGFP signal in the cell suspension (in PBS) was measured and converted to the molar concentration (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><math id="d33e2478"><msub><mrow><mrow><mo>[</mo><mrow><mi mathvariant="normal">AgDD</mi></mrow><mo>]</mo></mrow></mrow><mrow><mi mathvariant="normal">PBS suspension</mi></mrow></msub></math></span>) after multiplying by the coefficient determined as the slope of the calibration curve. sfGFP signal was mostly from full-length AgDD-sfGFP, as shown by immunoblotting (Fig. <a href="#MOESM1" class="usa-link">S1B</a>). The average intracellular AgDD concentration was calculated as:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e2501" display="block"><msub><mrow><mrow><mo>[</mo><mrow><mi mathvariant="normal">AgDD</mi></mrow><mo>]</mo></mrow></mrow><mrow><mi mathvariant="normal">intracellular</mi></mrow></msub><mo>=</mo><mfrac><mrow><msub><mrow><mrow><mo>[</mo><mrow><mi mathvariant="normal">AgDD</mi></mrow><mo>]</mo></mrow></mrow><mrow><mi mathvariant="normal">PBS</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">suspension</mi></mrow></msub><mspace width="0.25em"></mspace><mo>×</mo><mspace width="0.25em"></mspace><mi mathvariant="normal">Volume</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">PBS</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">suspension</mi></mrow><mrow><mi mathvariant="normal">Number</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.16em"></mspace><mn>293</mn><mi mathvariant="normal">T</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">in</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">PBS</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">suspension</mi><mo>×</mo><mspace width="0.25em"></mspace><mfrac><mrow><mn>4</mn></mrow><mrow><mn>3</mn></mrow></mfrac><mi>π</mi><msup><mrow><mi>R</mi></mrow><mrow><mn>3</mn></mrow></msup></mrow></mfrac></math></td>
<td class="label">3</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq75"><math id="d33e2584"><mi>R</mi></math></span> is the radius of the HEK293T cell, set at 5 μm.</p>
<p id="Par67">To estimate the molar concentration of AgDD in each single cell from microscopy data, 100 unperturbed cells were randomly selected from 5 fields-of-view and manually segmented to determine the cytoplasmic sfGFP signal (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq76"><math id="d33e2595"><msub><mrow><mi mathvariant="normal">sfGFP</mi></mrow><mrow><mi mathvariant="normal">single cell</mi></mrow></msub></math></span>). The average cytoplasmic sfGFP signal (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq77"><math id="d33e2610"><msub><mrow><mi mathvariant="normal">sfGFP</mi></mrow><mrow><mi mathvariant="normal">bulk</mi></mrow></msub></math></span>) was calculated as the mean of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq78"><math id="d33e2625"><msub><mrow><mi mathvariant="normal">sfGFP</mi></mrow><mrow><mi mathvariant="normal">single cell</mi></mrow></msub></math></span>. The molar concentration of AgDD in each cell was calculated using the value from Eq. <a href="#Equ3" class="usa-link">3</a>:</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e2644" display="block"><msub><mrow><mrow><mo>[</mo><mrow><mi mathvariant="normal">AgDD</mi></mrow><mo>]</mo></mrow></mrow><mrow><mi mathvariant="normal">single</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">cell</mi></mrow></msub><mo>=</mo><mfrac><mrow><msub><mrow><mi mathvariant="normal">sfGFP</mi></mrow><mrow><mi mathvariant="normal">single</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">cell</mi></mrow></msub><mspace width="0.25em"></mspace><mo>×</mo><mspace width="0.25em"></mspace><msub><mrow><mrow><mo>[</mo><mrow><mi mathvariant="normal">AgDD</mi></mrow><mo>]</mo></mrow></mrow><mrow><mi mathvariant="normal">intracellular</mi></mrow></msub></mrow><mrow><mi mathvariant="normal">sf</mi><msub><mrow><mi mathvariant="normal">GFP</mi></mrow><mrow><mi mathvariant="normal">bulk</mi></mrow></msub></mrow></mfrac><mo>.</mo></math></td>
<td class="label">4</td>
</tr></table></section><section id="Sec21"><h3 class="pmc_sec_title">Immunofluorescence staining</h3>
<p id="Par68">HEK293T or U2OS cells expressing AgDD-sfGFP were seeded on Poly-L-Lysine-coated (Millipore Sigma, #P4707-50ML) coverslips 24 h before experiments. 4 h after aggresome induction, cells were washed once with cold phosphate-buffered saline (PBS), fixed in 4% paraformaldehyde (Leagene Biotechnology, # DF0135) for 15 min at 25 °C, washed 3 × 5 min in PBS, permeabilized with PBS + 0.2% Triton X-100 for 10 min, and blocked for 30 min in PBS + 3% BSA, 0.1% Triton X-100. Centrosomes were stained by incubating overnight with 1:500 diluted rabbit polyclonal pericentrin antibody (Abcam, #AB4448) in blocking buffer at 4 °C. Vimentin was stained by incubating overnight with 1:200 diluted mouse monoclonal vimentin antibody (Abcam, #8069). Cells were then washed 3 × 5 min in PBS + 0.1% Triton X-100, incubated with 1:1000 dilution secondary antibodies (Donkey-anti-Rabbit 647 (Invitrogen, #A32795); Donkey-anti-Mouse 568 (Invitrogen, #A-10037)) in blocking buffer for 1 h at 25 °C, washed 3 × 5 min in PBS, and mounted on microscope slides using ProLong Gold Antifade Mountant with 4,6-diamidino-2-phenylindole (Invitrogen, #<a href="https://www.ncbi.nlm.nih.gov/protein/P36930" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P36930</a>) and imaged the following day. Images were taken using the same Nikon spinning-disk confocal microscope for live-cell imaging with a Nikon CFI Apo TIRF 60x Oil objective.</p>
<p id="Par69">To study colocalization of the aggresome and the chaperonin complex, HEK293T cells transfected with the Htt(Q94)-CFP construct were induced with 1 μg/mL doxycycline for 12 h before fixation. Cells were co-stained with 1:200 diluted mouse monoclonal antibody for mono- and poly ubiquitinylated proteins (FK2, Sigma-Aldrich, #04-263), 1:50 diluted rabbit polyclonal CCT2 antibody (Proteintech, #24896-1-AP), 1:1000 diluted rabbit polyclonal CCT4 antibody (Proteintech, #21524-1-AP), and 1:100 diluted rabbit polyclonal CCT8 antibody (Proteintech, #12263-1-AP) for 2 h at 25 °C. Similar experiments were performed using stable U2OS cell lines expressing Htt(Q94)-CFP under either a Tet-On or Tet-Off promoter. In the Tet-On system, cells were treated with 1 μg/mL doxycycline for 48 h before fixation. In the Tet-Off system, cells were maintained in 1 μg/mL doxycycline and induced to form aggresomes through doxycycline washout for 48 h.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Live-cell imaging</h3>
<p id="Par70">Cells were seeded onto a glass-bottom 8-well µ-Slide (ibidi, #80821) or glass-bottom 12-well plate (Cellvis, #P12-1.5H-N) coated with fibronectin (Sigma-Aldrich, #F0895) one day before the experiment in regular DMEM medium without phenol red (Cytiva, #SH30284.01) plus 10% heat-inactivated FBS and 2× Antibiotic-Antimycotic. Cells expressing HaloTag-H2A were stained with 500 nM JFX650-HaloTag ligand (Janelia) for 2 h right before imaging. Widefield fluorescence microscopy was performed on a Nikon Ti2 inverted microscope with epi-fluorescence optics, Lumencor Spectra-X light engine for fluorescence illumination, a sCMOS camera (Hamamatsu Flash4.0, 6.5 µm<sup>2</sup> photodiode) and a microscope incubator (OkoLab) for controlling temperature (37 °C), humidity, and CO<sub>2</sub> level (5%), and 20x (Nikon Plan Apo 0.75 NA) and 40x (Nikon Plan Apo 0.95 NA) objectives. Images were taken every 10 min. NIS Elements software was used for image acquisition. Cells were covered with mineral oil (Sigma, #M8410) to prevent evaporation. Confocal microscopy was performed on a Nikon spinning-disk confocal microscope (Nikon Ti2 inverted microscope, Yokogawa CSU-W1) with 405/488/561/640 nm lasers (iChrome MLE), a sCMOS camera (Photometrics Prime 95B), a Nikon CFI Plan Apochromat VC 60x water immersion objective, and a microscope incubator (OkoLab). Images were taken every 1 min for single particle tracking. Images were acquired using the NIS Elements software and examined using ImageJ (version 1.53t, National Institutes of Health).</p>
<p id="Par71">To test the aggresome formation efficiency in HEK293T cells expressing Htt(Q94)-CFP with or without chaperonin knockdown, cells were seeded on a glass-bottom 12-well plate and imaged using the IncuCyte® ZOOM Live-Cell Analysis System (Sartorius) with a 20x objective and phase and GFP filters. Images were taken every 30 min for 60 h.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Cargo transport assay in <em>Xenopus</em> egg extract (XE)</h3>
<section id="Sec24"><h4 class="pmc_sec_title">XE preparation</h4>
<p id="Par72">Actin-fragmented mitotic <em>Xenopus</em> egg extract (CSF extract) was used in all egg extract experiments except the one shown in Supp. Movie <a href="#MOESM7" class="usa-link">5</a>. Actin-fragmented CSF extract was either prepared with intact actin as described and treated with 20 μg/mL cytochalasin D (CytoD) right before experiments<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, or prepared from 100 μg/mL CytoD-treated <em>Xenopus</em> eggs and supplemented with 10 μg/mL CytoD<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> while otherwise following the same protocol<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Freshly prepared CSF extract was kept on ice and used for experiments within 8 h. Freshly prepared CSF extracts could also be aliquoted, slowly cooled in a Mr. Frosty™ Freezing Container (Thermo Scientific, #5100-0001), and stored at −80 °C. Frozen aliquots of egg extract were thawed on ice and used for later experiments while avoiding multiple freeze-thaw cycles. Whenever possible, extracts were pipetted with 200 μL wide-bore tips (Axygen, #T205WBC) to reduce shear damage. We compared behaviors of aggregate transport using freshly prepared XE, frozen XE, XE made with CytoD, or XE made with intact actin but treated with CytoD right before the experiment, and observed consistent trends of PSS (Fig. <a href="#MOESM1" class="usa-link">S9</a>).</p>
<p id="Par73">Actin-intact cycling egg extract used in Supp. Movie <a href="#MOESM7" class="usa-link">5</a> was prepared largely following Guan et al.<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>, without adding CytoD<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. Instead of enclosed imaging chambers (described below), cycling extract was loaded onto a passivated coverslip and covered with mineral oil for imaging.</p></section><section id="Sec25"><h4 class="pmc_sec_title">Cargo transport assay</h4>
<p id="Par74">To assemble and image interphase microtubule asters in egg extract, actin-fragmented CSF extract was supplemented with fluorescent tubulin to label microtubules, cycled to interphase to allow aster growth, and supplemented with demembranated <em>Xenopus</em> sperms to nucleate asters. Tubulin purified from bovine brain was labeled with Alexa Fluor 647 (tubulin-Alexa647), and Tau-mCherry was purified from <em>E. coli</em> cells as previously described in refs. <sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>,<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. In a typical reaction, MTs were imaged with either tubulin-Alexa647 at a final concentration of 50 nM or Tau-mCherry at a final concentration of 20 nM. To trigger exit from CSF arrest and entry to interphase, CaCl<sub>2</sub> was added to the extract at a 0.4 mM final concentration. The extract was mixed well immediately after calcium addition by gently flicking and incubated in an 18 °C water bath for 4 min, then returned to ice for 3 min. Demembranated <em>Xenopus</em> sperms were prepared as described in ref. <sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup> and added to the appropriate concentration that minimized contact between asters. For the experiment shown in Supplementary Movie <a href="#MOESM7" class="usa-link">5</a>, EB1-mApple was added to a final concentration of 200 nM to label the centrosomes<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>.</p>
<p id="Par75">To study dynein-dependent transport, AgDD aggregates, dynein-adapter-coated Dynabeads, and polystyrene beads (Polysciences, #21636-1) were 1:20 diluted into the working extract and mixed well by flicking the tube. The extract reaction was loaded into customized imaging chambers assembled from coverslips passivated with poly-L-lysine-g-polyethylene glycol (PLL-g-PEG), as described previously<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Imaging was started immediately using a Nikon Eclipse Ti2-E inverted microscope with SOLA SE V-nIR light engine and an Andor Zyla 4.2 PLUS sCMOS camera. The microscope room was kept at 18 °C. Low-temporal-resolution tracking experiments were performed using a Nikon CFI Plan Apo Lambda 10x NA 0.45 objective lens and imaged every 1 min unless otherwise stated. High-temporal-resolution tracking was performed using a Nikon CFI Plan Apo Lambda 40x NA 0.95 objective lens and imaged at 30 Hz. Two fluorescence channels were used in imaging, including the GFP channel for dynein cargoes (Filter cube: Nikon 96362/CHROMA 49002 ET GFP, excitation filter 470/40, emission filter 525/50) and the Cy5 channel for microtubules (Filter cube: CHROMA 49009 ET Cy5 NX, ex 640/30, em 690/50). In experiments with multiple conditions imaged in parallel, the slide holder was first chilled on ice for several seconds, so that aster growth would start at the same time across all conditions.</p>
<p id="Par76">To inhibit dynein, the p150-CC1 fragment of dynactin was added to a final concentration of 40 μg/mL<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>.</p>
<p id="Par77">To test the deubiquitylation activities of Usp2 in the extract, 1 μM Usp2 catalytic domain Usp2<sup>CD</sup> (R&amp;D Systems, stock solution at 50 μM in 50 mM HEPES pH 8.0, 150 mM NaCl, 0.1 mM EDTA, 1 mM DTT) or an equal volume of the same buffer was added to the Ca<sup>2+</sup>-activated extract and incubated at 18 °C. Samples were taken every 10 min during the incubation and immunoblotted with an antibody for ubiquitin (Cell Signaling, #43124). To remove any ubiquitin on the AgDD aggregates formed in the extract, the aggregates were incubated with 10 μM Usp2<sup>CD</sup> at 18 °C for 10 min before being diluted into the extract for the transport assay.</p></section><section id="Sec26"><h4 class="pmc_sec_title">Conjugation of dynein adapters to dynabeads</h4>
<p id="Par78">Dynabeads Protein G beads (Thermo Fisher, #10004D) were washed twice and resuspended in PBS to 10-fold bead volume (BV), then incubated with 5–10 μg of primary antibody per 20 μL beads (initial BV) overnight at 4 °C with slow tilt and rotation. Beads were washed with 10× BV of PBS to remove unbound antibodies and resuspended to 1/5 of the initial BV. Antibodies targeting human proteins VCP/p97 (Proteintech, #10736-1-AP), HSPA1A (Proteintech, #10995-1-AP), HSPA8 (Proteintech, #10654-1-AP), HSP90AA1 (Proteintech, #13171-1-AP), CCT8 (Proteintech, #12263-1-AP), proteasome subunit PSMA2 (Enzo Life Sciences, #BML-PW8105), SQSTM1/p62 (BD Biosciences, #610832), HDAC6 (Proteintech, #12834-1-AP), BAG3 (Proteintech, #10599-1-AP), as well as random rabbit IgG used as a negative control (Jackson ImmunoResearch, #011-000-003) were purchased. The antibody targeting <em>Xenopus</em> HOOK2 was made and purified from rabbit serum by the Mitchison lab<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>.</p>
<p id="Par79">To immunoprecipitate human adapter proteins from HeLa S3 extract, 20 μL beads coated with primary antibodies were incubated with 100 μL HeLa cell extract diluted with 200 μL of TBST (TBS + 0.05% Tween-20) with 1× protease inhibitor cocktail for 2 h at 4 °C and washed three times with TBST. Beads were then washed twice, resuspended, and stored in a physiological buffer (1 mM DTT, 100 mM KCH<sub>3</sub>COOH, 30 mM KCl, 1 mM MgCl<sub>2</sub>, 1 mM Na<sub>2</sub>ATP, 10 mM Na<sub>2</sub>HPO<sub>4</sub>). Beads were validated by mass spectrometry as described below. Beads conjugated with random IgG and <em>Xenopus</em> HOOK2 antibodies were prepared in parallel as negative controls.</p>
<p id="Par80">To prepare HOOK2-coated beads for the transport assay, 20 μL <em>Xenopus</em> HOOK2 antibody-conjugated beads were incubated with 100 μL CSF extract (actin depolymerized, Ca<sup>2+</sup> activated) for 60 min at 4 °C, washed 5 times with the wash buffer (20 mM KCl, 1 mM MgCl<sub>2</sub>, 10 mM K·HEPES pH 7.7, 1 mM EGTA) and resuspended and stored in the physiological buffer. Right before the transport assay, fluorescent secondary antibody was added to dynein adapter-coated beads to a final concentration of 20 μg/mL, including Goat anti-Rabbit IgG Secondary Antibody Alexa Fluor™ 488 (Invitrogen, #A-11008) for rabbit-derived primary antibodies and Goat anti-Mouse IgG Secondary Antibody Alexa Fluor™ 488 (Invitrogen, #A-11001) for mouse-derived primary antibodies.</p></section><section id="Sec27"><h4 class="pmc_sec_title">Passivation and conjugation of polystyrene beads</h4>
<p id="Par81">Fluoresbrite® YG Carboxylate Microspheres with different diameters (Polysciences, #21636-1) were passivated with amine-mPEG and amine-PEG-biotin as follows: Polystyrene beads were washed twice with H2O and resuspended in conjugation buffer (50 mM MES pH 6.5, 150 mM K<sub>2</sub>SO<sub>4</sub>, 20% w/v 2 kDa amine-mPEG (LaysanBio, #MPEG-NH2-2000-1GR), 0.8% w/v 3.4 kDa amine-PEG-biotin (LaysanBio, #BIOTIN-PEG-NH2-3400-100MG)) at 50x the bead volume. Immediately after, 50 mM EDC and 5 mM sulfo-NHS were added, followed by overnight incubation at 25 °C. The beads were then washed three times with H<sub>2</sub>O and twice with PBS.</p>
<p id="Par82">Passivated beads were either used directly in the transport assay or further conjugated with cargo adapters. For conjugation, purified streptavidin was mixed with biotinylated protein G (Pierce, #29988) at a 1:1 molar ratio, then incubated with passivated beads at 1 mg/ml for 2 h at 4 °C. Conjugated beads were then washed three times with PBS. HDAC6 antibody conjugation and immunoprecipitation procedures were performed identically to those used for Dynabeads.</p></section><section id="Sec28"><h4 class="pmc_sec_title">Immunoprecipitation of AgDD interacting proteins</h4>
<p id="Par83">Dynabeads Protein G beads were saturated with anti-HA primary antibodies (HA-7; Sigma, #H9658) as described previously. An extra step of crosslinking primary antibodies onto the beads was performed to minimize interference detection of antibodies in the mass spectrometry analysis. Beads were washed twice with 20× bead volume (BV) of 0.2 M sodium tetraborate pH 9.0, and resuspended in 10× BV of the same buffer. Dimethylpimelimidate was weighed and added as powder to reach a final concentration of 20 mM and dissolved by gentle mixing. Reactions were rotated for 30 min at 25 °C and quenched by washing the beads twice with 10× BV of the quenching buffer (150 mM NaCl, 200 mM Tris-HCl pH 7.5), with a 10-min incubation in between. Beads were then washed twice with the wash buffer (0.1 M KCl, 10 mM HEPES pH 7.7, 100 μg/mL BSA) and resuspended to 1/5 of the original volume. 4 μL of concentrated bead slurry was used to immunoprecipitate proteins from 100 μL of extract.</p>
<p id="Par84">Two batches of actin-depolymerized CSF extracts were freshly prepared in parallel and used for experiments as repeats. 15 μL AgDD aggregates formed in the extract (AgDD<sub>XE</sub>), in the physiological buffer (AgDD<sub>pb</sub>), and the uninduced control (AgDD<sub>sol</sub>) were prepared as described previously. Both extracts were activated by Ca<sup>2+</sup>, and each was split into 4 × 2 mL Protein LoBind Tubes (Eppendorf, #022431102), 100 μL each. 10 μL AgDD<sub>XE</sub>, AgDD<sub>pb</sub>, AgDD<sub>sol</sub>, or 10 μL extract was added to each tube. Four microliters HA-antibody-coated beads were then added to each tube and mixed well by flicking. Samples were incubated for 45 min at 16 °C with slow tilt and rotation. Beads were then collected on a magnet for 5 min on ice and washed five times with 1 mL wash buffer (25 mM KCl, 10 mM K·HEPES pH 7.7, 1 mM MgCl<sub>2</sub>, 1 mM EGTA, 1 mM ATP). Beads were transferred to a new tube twice after the 4th and 5th washes.</p>
<p id="Par85">Immunoprecipitated proteins were identified by mass spectrometry as described below.</p></section></section><section id="Sec29"><h3 class="pmc_sec_title">Quantitative mass spectrometry</h3>
<section id="Sec30"><h4 class="pmc_sec_title">Sample preparation</h4>
<p id="Par86">Proteins immunoprecipitated by the beads were determined by Tandem-Mass-Tag mass spectrometry (TMT-MS). On-bead digestion was performed by resuspending beads in 1× BV of 50 mM EPPS pH 8.3 and 2 M GuHCl with 1:100 diluted (v/v) lys-C (Promega, #VA1170; 2 mg/mL stock solution). The reactions were incubated at 37 °C for 4 h on an Eppendorf ThermoMixer C at the top speed. The same dose of lys-C was added again, and the reactions were incubated overnight. Supernatant was transferred to a new tube and diluted to 4× BV with 50 mM EPPS pH 8.3. 1:50 diluted (v/v) trypsin (Promega, #V511C; 0.5 mg/mL stock solution) was added to the sample and incubated at 37 °C for 4 h on a tube rotator. The peptide concentration was determined using CBQCA fluorescent assay following the manufacturer’s instruction (ThermoFisher, #C6667). For <em>Samples from Dynein-Adapter-Coated Beads</em>, TMTpro reagent 126, 127 N, 127 C, 128 N, 128 C, 129 N, 129 C, 130 N, 130 C, 131 N, 131 C, 132 N were used to label 0.73, 0.84, 0.91, 0.88, 1.20, 1.04, 0.63, 0.77, 0.91, 0.73, 1.03, 0.71 μg peptides from 12 conditions (ThermoFisher, #<a href="https://www.ncbi.nlm.nih.gov/protein/A44520" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">A44520</a>; Supplementary Data <a href="#MOESM13" class="usa-link">2</a>); For <em>Samples from AgDD-Coated Beads</em>, TMTpro reagent 126, 127 N, 127 C, 128 N, 128 C, 129 N, 129 C, 130 N, 130 C were used to label 0.46, 0.40, 0.24, 0.41, 0.88, 0.89, 1.06, 0.96, 0.75 μg peptides from 9 conditions (Supplementary Data <a href="#MOESM12" class="usa-link">1</a>). TMTpro 16plex labeling, alkylation/cysteine protection by iodoacetamide (IAA), stage tip desalting, and SpeedVac drying were performed following the manufacturer’s instructions and standard protocols<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Peptides were reconstituted in 0.1% formic acid for LC-MS analysis. TMT-labeling efficiency was verified to be more than 95%.</p></section><section id="Sec31"><h4 class="pmc_sec_title">LC-MS data collection and analysis</h4>
<section id="FPar1"><h5 class="pmc_sec_title">Samples from dynein-adapter-coated beads</h5>
<p id="Par87">Mass spectrometric data were collected on an Orbitrap Exploris480 instrument. The mass spectrometer was coupled to a Proxeon NanoLC-1200 UHPLC attached to a 100 µm capillary column packed with 35 cm of Accucore 150 resin (2.6 μm, 150 Å; ThermoFisher Scientific) at a flow rate of ~420 nL/min. The LC parameters consist of 5% acetonitrile (ACN) and 0.1% formic acid (FA) in mobile phase A, and 95% ACN with 0.1% FA in mobile phase B, with a gradient range of 5% to 27% of B. Data were acquired using multiple injections (<em>n</em> = 2) with varying combinations of FAIMS compensation voltages (CVs) between −30 and −80 V (3 CVs per set) over a 150 min gradient. A 1-s TopSpeed cycle was used for each CV. The scan sequence began with an MS1 spectrum (Orbitrap analysis, resolution 60,000, 350–1350 Th, automatic gain control (AGC) target 100%, maximum injection time “auto”). The HRMS2 stage consisted of fragmentation by higher energy collisional dissociation (HCD, normalized collision energy 32%) and analysis using the Orbitrap (AGC “standard”, maximum injection time 96 ms, isolation window 0.7 Th, resolution 45,000).</p>
<p id="Par88">The acquired data were searched using the open-source Comet algorithm (release_2019010)<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>–<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. Spectral searches utilized a custom FASTA-formatted database containing common contaminants and reversed sequences (Uniprot Human, 2021). The following parameters were used: 50 PPM precursor tolerance, fully tryptic peptides, a fragment ion tolerance of 0.02 Da, and a static modification by TMTPro16 (+304.2071 Da) on lysine and peptide N-termini. Additionally, carbamidomethylation of cysteine residues (+57.0214 Da) was applied as a static modification, and oxidation of methionine residues (+15.9949 Da) as a variable modification. Peptide spectral matches (PSMs) were filtered to achieve a peptide false discovery rate (FDR) of 1% using linear discriminant analysis and a target-decoy approach. Resulting peptides were further refined to reach a final protein-level FDR of 1% at the dataset level, with proteins grouped accordingly. Reporter ion intensities were adjusted for synthesis impurities of different TMT reagents as specified by the manufacturer. For each MS2 spectrum quantification, a minimum total signal-to-noise (S/N) of 100 or 160 for all reporter ions was required, depending on the experiment.</p>
<p id="Par89">Finally, protein abundance measurements were normalized such that the total signal-to-noise ratio across all channels for each protein was scaled to 100, providing a measure of its relative abundance.</p></section><section id="FPar2"><h5 class="pmc_sec_title">Samples from AgDD-coated beads</h5>
<p id="Par90">Mass spectrometric data were collected on an Orbitrap Fusion Lumos instrument (using HRMS2-mode). The mass spectrometer was coupled to a Proxeon NanoLC-1200 UHPLC attached to a 100 µm capillary column packed with 35 cm of Accucore 150 resin (2.6 μm, 150 Å; ThermoFisher Scientific) at a flow rate of ~440 nL/min. The LC parameters consist of 5% ACN and 0.1% formic acid FA in mobile phase A, and 95% ACN with 0.1% FA in mobile phase B, with a gradient range of 5% to 27% of B. Data were acquired using multiple injections (<em>n</em> = 3) with varying combinations of FAIMS compensation voltages (CVs) between −30 and −80 V (3 CVs per set) over a 150 min gradient. A 1-s TopSpeed cycle was used for each CV. The scan sequence began with an MS1 spectrum (Orbitrap analysis, resolution 60,000, 350–1350 Th, automatic gain control (AGC) target 100%, maximum injection time “auto”). The HRMS2 stage consisted of fragmentation by higher energy collisional dissociation (HCD, normalized collision energy 36%) and analysis using the Orbitrap (AGC 300%, maximum injection time 250 ms, isolation window 0.7 Th, resolution 50,000).</p>
<p id="Par91">The acquired data were similarly searched and filtered as previously described above. Spectral searches utilized a custom FASTA-formatted database containing common contaminants and reversed sequences (UniProt <em>Xenopus laevis</em>, 2021). For each MS2 spectrum quantification, a minimum total signal-to-noise (S/N) of 100 for all reporter ions was required. Again, protein abundance measurements were normalized such that the total signal-to-noise across ratio across all channels for each protein was scaled to 100, providing a measure of its relative abundance.</p></section></section><section id="Sec34"><h4 class="pmc_sec_title">Identification of AgDD interactome</h4>
<p id="Par92">Among all conditions (AgDD<sub>XE</sub>, AgDD<sub>sol</sub>, AgDD<sub>pb</sub>, naked beads), the enrichment of proteins captured by AgDD<sub>XE</sub> compared to AgDD<sub>sol</sub> was used to identify candidate proteins potentially involved in aggresome formation. For each identified protein, the mean of the AgDD<sub>XE</sub> to AgDD<sub>sol</sub> ratio of two repeats was calculated. <em>P</em>-values were calculated using a two-sample <em>t</em>-test. Proteins were selected if the mean ratio was greater than 1.15 and the <em>P</em>-value was smaller than 0.5. To be noted, <em>P</em>-value cutoff was chosen to help identify candidate proteins potentially associated with aggregates for subsequent validation via the bead-based transport assay rather than to make strict statistical claims. The cutoff value does not affect any conclusions in the study. Gene Ontology (GO) analysis was performed on the selected genes using shinyGo based on the category of Molecular Function and FDR cutoff set to 0.05<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>–<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>.</p></section></section><section id="Sec35"><h3 class="pmc_sec_title">Microscopy image analysis</h3>
<section id="Sec36"><h4 class="pmc_sec_title">Live-cell image analysis</h4>
<section id="FPar3"><h5 class="pmc_sec_title">Aggresome detection</h5>
<p id="Par93">AgDD aggresome formation is operationally defined as the appearance of a perinuclear punctum if the surrounding aggregates move toward it in subsequent frames and all peripheral aggregates are eventually sequestrated into it. The aggresome-containing cells were identified manually. Htt(Q94) aggresomes were identified using the ImageJ plugin “<em>TrackMate</em>”<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Briefly, a stable HEK293T cell line integrated with pTetOn-Htt(Q94)-CFP was transfected with CCT2-targeting siRNA and imaged using the IncuCyte® ZOOM System. Doxycycline was added to the media to induce Htt(Q94) expression at different time points as indicated in Fig. <a href="#MOESM1" class="usa-link">S12E, F</a>. Cell confluency was monitored via the IncuCyte® ZOOM Confluence Processing analysis tool (Basic Analyzer) based on the phase channel. The number of aggresomes in each frame was determined by <em>TrackMate</em>. Since most cells contained a single Htt(Q94) aggresome, the fraction of aggresome-containing cells at each time point was calculated as the ratio of the aggresome number to the total cell number. The latter was inferred from the confluency value based on a calibration using manually-counted cell numbers vs. confluency in 10 fields.</p></section><section id="FPar4"><h5 class="pmc_sec_title">AgDD Aggresome enrichment factor</h5>
<p id="Par94">Aggresome enrichment factor was defined as the maximum fluorescence intensity of pixels in the perinuclear region (<em>I</em><sub>max</sub>) divided by the average pixel intensity of the whole cell (<em>I</em><sub>0</sub>), before Shield-1 removal. The ring-shaped perinuclear region with a width of ~2 μm was manually segmented based on the contour of the nucleus. For aggresome-containing cells, the width was enlarged to enclose the entire aggresome.</p></section></section><section id="Sec37"><h4 class="pmc_sec_title">Single-particle tracking of aggregates</h4>
<section id="FPar5"><h5 class="pmc_sec_title">MTOC localization</h5>
<p id="Par95">For low-resolution tracking experiments in <em>Xenopus</em> egg extract (XE), the MTOC was manually registered as the center of the aster based on Alexa Fluor 647 signal using the ImageJ plugin “<em>Manual Tracking</em>”. For high-resolution tracking experiments in XE and on live cells, the MTOC was determined as the center of the aggregates that were already clustered around the centrosomes through the ImageJ plugin “<em>TrackMate</em>”. Clustered aggregates were first identified via “LoG detector” as a single object, whose diameter was adjusted to tightly enclose all the clustered aggregates. The object was then tracked via “LAP Tracker” with appropriate frame-to-frame linking and gap closing based on visual examination. The center of the object was used as the MTOC position. An extra step of aligning all frames through centering the MTOC was performed before tracking the aggregates for the low-resolution experiments in XE, to correct for the common drift of asters during the time-lapse imaging lasting for tens of minutes.</p></section><section id="FPar6"><h5 class="pmc_sec_title">Aggregates tracking</h5>
<p id="Par96">Aggregates were tracked using <em>TrackMate</em>. Aggregates were first identified via “LoG detector” with appropriate parameters:</p>
<ul class="list" style="list-style-type:none">
<li>
<span class="label">i.</span><p class="display-inline" id="Par97">“Estimated objective diameter”: adjusted to the minimal value that could cover all visible aggregates, named <em>r</em><sub><em>0</em></sub>, which will be used for size determination below.</p>
</li>
<li>
<span class="label">ii.</span><p class="display-inline" id="Par98">“Quality threshold”: adjusted based on visual examination.</p>
</li>
<li>
<span class="label">iii.</span><p class="display-inline" id="Par99">“Pre-process with median filter” and “Sub-pixel localization” were both selected.</p>
</li>
</ul>
<p id="Par100">Identified aggregates were then tracked via “LAP Tracker” with appropriate parameters:</p>
<ul class="list" style="list-style-type:none">
<li>
<span class="label">i.</span><p class="display-inline" id="Par101">“Frame-to-frame linking”: “Max distance” was adjusted to be slightly larger than the largest displacement detected between consecutive frames; “Feature penalties” included “Mean intensity ch1 (weight = 1)” and “Quality (weight = 1)”.</p>
</li>
<li>
<span class="label">ii.</span><p class="display-inline" id="Par102">“Track segment gap closing”: “Max frame gap” was set to “1”, and “Max distance” was chosen to be the same as “Max distance” in “Frame-to-frame linking”.</p>
</li>
</ul>
<p id="Par103">Lastly, after removing mis-tracked trajectories manually, the (<em>x</em>, <em>y</em>) positions of each trajectory at each time point <em>t</em> were saved and converted to the Euclidean distance <em>d</em>(<em>t</em>) to the MTOC in MATLAB (version R2023a, The MathWorks, Natick, MA). The “trajectory” over time was calculated as <em>d</em>(<em>t</em> = <em>0</em>) − <em>d</em>(<em>t</em>), named as “travel distance towards MTOC”, unless otherwise stated. A positive value means moving towards the MTOC.</p></section><section id="FPar7"><h5 class="pmc_sec_title">Size determination via 2D Gaussian fitting</h5>
<p id="Par104">For each timepoint from each trajectory, a (2<em>r</em><sub><em>0</em></sub> + 1) × (2<em>r</em><sub><em>0</em></sub> + 1) square centered at the (<em>x</em>, <em>y</em>) position of the aggregate was extracted from the original image and fitted by a 2D Gaussian function in MATLAB using the following equation:</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><math id="d33e3138" display="block"><mi>I</mi><mrow><mo>(</mo><mrow><mi>a</mi><mo>,</mo><mi>b</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>q</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>q</mi></mrow><mrow><mn>1</mn></mrow></msub><mi>exp</mi><mfenced open="["><mrow><mo>−</mo><msup><mrow><mfenced close=")" open="("><mrow><mfrac><mrow><mrow><mo>(</mo><mrow><mi>a</mi><mo>−</mo><msub><mrow><mi>q</mi></mrow><mrow><mn>5</mn></mrow></msub></mrow><mo>)</mo></mrow><mi>cos</mi><msub><mrow><mi>q</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>+</mo><mrow><mo>(</mo><mrow><mi>b</mi><mo>−</mo><msub><mrow><mi>q</mi></mrow><mrow><mn>6</mn></mrow></msub></mrow><mo>)</mo></mrow><mi>sin</mi><msub><mrow><mi>q</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow><mrow><msub><mrow><mi>q</mi></mrow><mrow><mn>3</mn></mrow></msub></mrow></mfrac></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfenced><mfenced close="]"><mrow><mo>−</mo><msup><mrow><mfenced close=")" open="("><mrow><mfrac><mrow><mo>−</mo><mrow><mo>(</mo><mrow><mi>a</mi><mo>−</mo><msub><mrow><mi>q</mi></mrow><mrow><mn>5</mn></mrow></msub></mrow><mo>)</mo></mrow><mi>sin</mi><msub><mrow><mi>q</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>+</mo><mrow><mo>(</mo><mrow><mi>b</mi><mo>−</mo><msub><mrow><mi>q</mi></mrow><mrow><mn>6</mn></mrow></msub></mrow><mo>)</mo></mrow><mi>cos</mi><msub><mrow><mi>q</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow><mrow><msub><mrow><mi>q</mi></mrow><mrow><mn>4</mn></mrow></msub></mrow></mfrac></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfenced></math></td>
<td class="label">5</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq79"><math id="d33e3272"><mi>a</mi></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq80"><math id="d33e3281"><mi>b</mi></math></span> are the coordinates in pixel, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq81"><math id="d33e3290"><mi>I</mi><mrow><mo>(</mo><mrow><mi>a</mi><mo>,</mo><mi>b</mi></mrow><mo>)</mo></mrow></math></span> is the intensity of the pixel, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq82"><math id="d33e3306"><msub><mrow><mi>q</mi></mrow><mrow><mn>0</mn><mo>−</mo><mn>6</mn></mrow></msub></math></span> are the parameters to fit. To be noted, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq83"><math id="d33e3321"><msub><mrow><mi>q</mi></mrow><mrow><mn>3</mn></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq84"><math id="d33e3335"><msub><mrow><mi>q</mi></mrow><mrow><mn>4</mn></mrow></msub></math></span> are restricted to positive values and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq85"><math id="d33e3348"><msub><mrow><mi>q</mi></mrow><mrow><mn>2</mn></mrow></msub></math></span> are restricted between 0 and 180 degrees without loss of generality. Other parameters constraining the fitting algorithm were defined in the script. Aggregates with irregular shapes or a low signal-to-noise ratio were discarded if satisfying any of the following conditions:</p>
<ul class="list" style="list-style-type:none">
<li>
<span class="label">i.</span><p class="display-inline" id="Par105"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq86"><math id="d33e3366"><msub><mrow><mi>q</mi></mrow><mrow><mn>5</mn></mrow></msub></math></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq87"><math id="d33e3379"><msub><mrow><mi>q</mi></mrow><mrow><mn>6</mn></mrow></msub></math></span> is smaller than 1 or larger than (2<em>r</em><sub><em>0</em></sub> + 1).</p>
</li>
<li>
<span class="label">ii.</span><p class="display-inline" id="Par106"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq88"><math id="d33e3402"><msub><mrow><mi>q</mi></mrow><mrow><mn>3</mn></mrow></msub></math></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq89"><math id="d33e3415"><msub><mrow><mi>q</mi></mrow><mrow><mn>4</mn></mrow></msub></math></span> is negative.</p>
</li>
<li>
<span class="label">iii.</span><p class="display-inline" id="Par107"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq90"><math id="d33e3432"><msub><mrow><mi>q</mi></mrow><mrow><mn>1</mn></mrow></msub></math></span> is smaller than <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq91"><math id="d33e3445"><msub><mrow><mi>q</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span>.</p>
</li>
</ul>
<p id="Par108">The diameter of the aggregate was calculated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq92"><math id="d33e3460"><msub><mrow><mi>c</mi></mrow><mrow><mn>0</mn></mrow></msub><msqrt><mrow><msub><mrow><mi>q</mi></mrow><mrow><mn>3</mn></mrow></msub><msub><mrow><mi>q</mi></mrow><mrow><mn>4</mn></mrow></msub></mrow></msqrt></math></span>, i.e., 2<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq93"><math id="d33e3485"><msub><mrow><mi>c</mi></mrow><mrow><mn>0</mn></mrow></msub><msqrt><mrow><msub><mrow><mi>σ</mi></mrow><mrow><mi mathvariant="normal">X</mi></mrow></msub><msub><mrow><mi>σ</mi></mrow><mrow><mi mathvariant="normal">Y</mi></mrow></msub></mrow></msqrt></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq94"><math id="d33e3512"><msub><mrow><mi>σ</mi></mrow><mrow><mi mathvariant="normal">X</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq95"><math id="d33e3526"><msub><mrow><mi>σ</mi></mrow><mrow><mi mathvariant="normal">Y</mi></mrow></msub></math></span> are the standard deviations (i.e., spread parameters) along the <em>x</em> and <em>y</em>-axes. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq96"><math id="d33e3547"><msub><mrow><mi>c</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> was determined from a calibration curve using standard polystyrene beads of known sizes (Polysciences, #21636-1; Fig. <a href="#MOESM1" class="usa-link">S3</a>).</p></section><section id="FPar8"><h5 class="pmc_sec_title">Determination of the localization error in the high-resolution tracking experiments</h5>
<p id="Par109">The same bead standards used for size calibration were immobilized on the coverslip via nonspecific attachment and imaged using the same settings as in the high-resolution tracking experiments for AgDD aggregates and adapter-coated beads. Data were then analyzed by <em>TrackMate</em> as described before. The localization error was calculated as the root-mean-square deviation (RMSD) of the bead’s position after subtracting the drift, which was determined as the linear regression line of the position over time.</p></section></section><section id="Sec42"><h4 class="pmc_sec_title">Trajectory segmentation and diffusion constant calculation</h4>
<p id="Par110">Segmentation was performed using a MATLAB package “<em>slmengine</em>” with a fixed root mean squared error (RMSE) or a fixed number of segments<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>. For low-resolution tracking experiments, we used a fixed RMSE = 0.025 and set the maximal segment number as 10. For high-resolution experiments, we first used a fixed RMSE = 0.01 and set the maximal segment number as 20 to process the entire trajectory, and then extracted the transport segments determined as described below to perform a second round of segmentation using a fixed segment number, calculated as the transport segment length in seconds divided by 0.5. All parameters mentioned above were chosen based on visual examination.</p>
<p id="Par111">For each trajectory, the velocity of each segment was determined as the slope after performing linear regression on the distance over time. Segments with the smallest absolute values of velocity were selected, which were most likely to be the “Pause” state. The number of segments selected as pauses was less than 1/3 of the total segment number. Selected pauses were then used to calculate the mean square displacement (MSD) over time and fitted by a linear function. The diffusion constant was calculated as half of the slope. The mean value of the diffusion constant calculated from all selected segments was used as the diffusion constant <em>D</em> for the trajectory. A hypothesis test for all other segments not selected as pauses was then performed, given the null hypothesis that “<em>the displacement given the segment length t should follow a Gaussian distribution with the mean equal to 0, and the standard deviation equal to the square root of 2Dt</em>”. The segment was designated as “Transport engaged (+)” for low-resolution experiments and “Transport (+)” for high-resolution experiments if the displacement fell to the right tail of the Gaussian distribution; and “Transport engaged (−)” or “Transport (−)” for the left tail. For the low-resolution experiments, the cutoff bounds on both tails were 0.025, achieving a goodness-of-fit = 0.072 (normalized root mean square error (NRMSE), calculated as RMSE divided by the range of the variable). For high-resolution experiments, the cutoff bounds are 0.05 on both tails for the first round of segmentation and 0.005 and 0.1 on the left and right tails, respectively, for the second round of segmentation, achieving a goodness-of-fit = 0.0217 (NRMSE). These parameters were chosen to allow the segmentation results most consistent with visual examination. Lastly, any adjacent segments with the same designations were connected as a single segment.</p></section><section id="Sec43"><h4 class="pmc_sec_title">Determination of the aggregate’s velocities</h4>
<section id="FPar9"><h5 class="pmc_sec_title">
<em>MTOC-directed frame-by-frame velocity</em> (Figs. <a href="#Fig1" class="usa-link">1</a>H, <a href="#Fig1" class="usa-link">1</a>J, <a href="#Fig5" class="usa-link">5D</a> and <a href="#MOESM1" class="usa-link">S8</a>, <a href="#MOESM1" class="usa-link">S9</a>, <a href="#MOESM1" class="usa-link">S10</a>, <a href="#MOESM1" class="usa-link">S4</a>)</h5>
<p id="Par112">For each aggregate particle, the distance between the MTOC and the aggregate at each frame was calculated. The MTOC-directed velocity of the aggregate was determined as the change in the distance between two consecutive frames divided by the time interval in between, with a positive value meaning movement towards the MTOC. For each particle, the velocity at each time point was associated with the particle size determined at the same time point.</p></section><section id="FPar10"><h5 class="pmc_sec_title">
<em>Time-averaged transport velocity of single particle</em> (Figs. <a href="#Fig2" class="usa-link">2</a>D, <a href="#Fig3" class="usa-link">3</a>A, <a href="#Fig4" class="usa-link">4</a>F, <a href="#Fig4" class="usa-link">4G</a> and <a href="#MOESM1" class="usa-link">S7</a>)</h5>
<p id="Par113">When the aggregate size remains largely stable during the time-lapse imaging (Fig. <a href="#MOESM1" class="usa-link">S5</a>), the transport velocity and particle size were determined as the time average over the entire trajectory. Specifically, the transport velocity of each aggregate particle was calculated as the total distance moving towards the MTOC divided by the duration of the trajectory, with a positive value meaning movement towards the MTOC.</p></section><section id="FPar11"><h5 class="pmc_sec_title">
<em>Instantaneous velocity</em> (Fig. <a href="#Fig3" class="usa-link">3</a>C, <a href="#Fig3" class="usa-link">D</a>)</h5>
<p id="Par114">The instantaneous velocity was determined as the time derivative of the trajectory. A 2-s or 0.5-s time window was used for high-resolution tracking data as indicated.</p></section><section id="FPar12"><h5 class="pmc_sec_title">
<em>Intrinsic velocity</em> (Figs. <a href="#Fig3" class="usa-link">3</a>E, <a href="#Fig3" class="usa-link">3</a>F and <a href="#Fig4" class="usa-link">4E</a>)</h5>
<p id="Par115">For each particle, the intrinsic velocity was defined as the mean of the top 20% values of the instantaneous velocity within transport segments, unless otherwise stated.</p></section></section><section id="Sec48"><h4 class="pmc_sec_title">Violin plot</h4>
<p id="Par116">All violin plots were generated using a MATLAB function “<em>violin</em>” with additional modifications<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup>.</p></section></section><section id="Sec49"><h3 class="pmc_sec_title">An analytical model for dynein-mediated cargo transport</h3>
<p id="Par117">Our goal here is to establish a physical model of dynein-mediated cargo transport and determine how the average transport velocity depends on the system parameters. The entities in the model are dynein (Dy), microtubule (MT), protein aggregates or cargo (C), and their complexes. The physical processes being modeled are binding-unbinding between the model entities and the viscosity-limited transport by dynein. The overall procedure is to derive the system partition function, which dictates the likelihoods of C in different states. The likelihood of C in the active transport state directly determines the average transport velocity. Although dynein transport is a non-equilibrium process, we assume it is transient, allowing us to treat binding and unbinding events as if they were in equilibrium.</p>
<p id="Par118">The cargo is identified with two states in our experiments: a freely diffusing state and a local state (denoted by the subscript “L”) that is close to MT. Cargoes starting from a freely diffusing state, either staying alone (C) or bound with free dynein (C-Dy), must first approach MT, i.e., enter the local state, before forming an active transport complex (C<sub>L</sub>-Dy-MT) and engaging in transport. We do not pre-define a transition sequence, and the cargo can jump freely among possible states as depicted in the figure on the right.<img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fea/12373839/45e1620946a0/41467_2025_62751_Figa_HTML.gif" loading="lazy" id="d33e3689" alt="Inline graphic"></p>
<p id="Par119">Our results suggest that the copy number of dynein involved in transport does not vary with aggregate size (Fig. <a href="#Fig3" class="usa-link">3F</a>). For simplicity, we let the dynein copy number be “1”. Therefore, the cargo, once forming the active transport complex, is transported at an intrinsic velocity <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq97"><math id="d33e3700"><msub><mrow><mi>v</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>=</mo><mfrac><mrow><msub><mrow><mi>F</mi></mrow><mrow><mi mathvariant="normal">s</mi></mrow></msub></mrow><mrow><mn>6</mn><mi>π</mi><mi>η</mi><mi>R</mi></mrow></mfrac></math></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq98"><math id="d33e3728"><msub><mrow><mi>F</mi></mrow><mrow><mi mathvariant="normal">s</mi></mrow></msub></math></span> is the dynein stalling force, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq99"><math id="d33e3742"><mi>R</mi><mo>=</mo><mi>d</mi><mo>/</mo><mn>2</mn></math></span> is the cargo radius, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq100"><math id="d33e3755"><mi>d</mi></math></span> is the diameter, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq101"><math id="d33e3765"><mi>η</mi></math></span> is the dynamic viscosity of the cytosol), as defined by Stokes’ law (Fig. <a href="#Fig3" class="usa-link">3F</a>)<sup><a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup>.</p>
<p id="Par120">The average transport velocity <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq102"><math id="d33e3783"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> can be written as the product of the probability of cargo forming the active transport complex <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq103"><math id="d33e3796"><msub><mrow><mi>P</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub></math></span> and the intrinsic velocity <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq104"><math id="d33e3821"><msub><mrow><mi>v</mi></mrow><mrow><mi mathvariant="normal">c</mi></mrow></msub></math></span>:</p>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><math id="d33e3836" display="block"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>=</mo><msub><mrow><mi>P</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub><mo>×</mo><msub><mrow><mi>v</mi></mrow><mrow><mi mathvariant="normal">c</mi></mrow></msub><mo>.</mo></math></td>
<td class="label">6</td>
</tr></table>
<p id="Par121">We next derive how <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq105"><math id="d33e3876"><msub><mrow><mi>P</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub></math></span> depends on system parameters. Under the equilibrium assumption, the likelihood of cargo in each state is determined by the system partition function <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq106"><math id="d33e3901"><msub><mrow><mi>Z</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq107"><math id="d33e3914"><mi>i</mi></math></span> = cargo states):</p>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><math id="d33e3924" display="block"><msub><mrow><mi>P</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub><mo>=</mo><mfrac><mrow><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub></mrow><mrow><msub><mrow><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub><mo>+</mo><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>+</mo><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub></mrow></msub><mo>+</mo><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi><mo>−</mo><mi mathvariant="normal">Dy</mi></mrow></msub><mo>+</mo><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi></mrow></msub></mrow></mfrac><mo>.</mo></math></td>
<td class="label">7</td>
</tr></table>
<p id="Par122">To calculate individual partition functions, we model cargo as a sphere and dynein as a rod that only constrains the distance between cargo and microtubule, but does not interfere with other degrees of freedom (DOFs). For each term:</p>
<p id="Par123">1) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq108"><math id="d33e4035"><msub><mrow><mi mathvariant="bold-italic">Z</mi></mrow><mrow><msub><mrow><mi mathvariant="bold">C</mi></mrow><mrow><mi mathvariant="bold">L</mi></mrow></msub><mo>−</mo><mi mathvariant="bold">Dy</mi><mo>−</mo><mi mathvariant="bold">MT</mi></mrow></msub></math></span></p>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><math id="d33e4061" display="block"><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub><mo>=</mo><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">M</mi><mi>T</mi><mo>,</mo><mi mathvariant="normal">rot</mi></mrow></msub><mo>×</mo><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">M</mi><mi>T</mi><mo>,</mo><mi mathvariant="normal">spin</mi></mrow></msub><mo>×</mo><msup><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mo>−</mo><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>−</mo><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">MT</mi></mrow></msub></mrow><mrow><mi>k</mi><mi>T</mi></mrow></mfrac></mrow></msup><mo>×</mo><msub><mrow><mi>N</mi></mrow><mrow><mi>r</mi></mrow></msub></math></td>
<td class="label">8</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq109"><math id="d33e4160"><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi><mo>,</mo><mi mathvariant="normal">rot</mi></mrow></msub></math></span> is the rotational partition function for the cargo around the Dy-MT joint <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq110"><math id="d33e4188"><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi><mo>,</mo><mi mathvariant="normal">spin</mi></mrow></msub></math></span> is the spin partition function, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq111"><math id="d33e4216"><msub><mrow><mi>N</mi></mrow><mrow><mi>r</mi></mrow></msub></math></span> is the number of dynein adapters on cargo, proportional to the surface area <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq112"><math id="d33e4229"><msup><mrow><mn>4</mn><mi>π</mi><mi>R</mi></mrow><mrow><mn>2</mn></mrow></msup></math></span>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq113"><math id="d33e4245"><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq114"><math id="d33e4259"><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">MT</mi></mrow></msub></math></span> are the binding energies between cargo and dynein or between dynein and microtubule. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq115"><math id="d33e4273"><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub></math></span> can be expanded as.<img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fea/12373839/6baead25780c/41467_2025_62751_Figb_HTML.gif" loading="lazy" id="d33e4294" alt="Inline graphic"></p>
<table class="disp-formula p" id="Equ9"><tr>
<td class="formula"><math id="d33e4301" display="block"><mtable><mtr><mtd columnalign="center"><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub><mo>=</mo><msup><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mo>−</mo><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>−</mo><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">MT</mi></mrow></msub></mrow><mrow><mi>k</mi><mi>T</mi></mrow></mfrac></mrow></msup><mo>×</mo><msub><mrow><mi>N</mi></mrow><mrow><mi>r</mi></mrow></msub><mo>×</mo><mfrac><mrow><mn>1</mn></mrow><mrow><msup><mrow><mi>h</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac><mi>∬</mi><msup><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mfenced close=")" open="("><mrow><mo>−</mo><msubsup><mrow><mi>p</mi></mrow><mrow><mi>α</mi></mrow><mrow><mn>2</mn></mrow></msubsup><mo>−</mo><msubsup><mrow><mi>p</mi></mrow><mrow><mi>β</mi></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></mfenced></mrow><mrow><mn>2</mn><mi>I</mi><mi>k</mi><mi>T</mi></mrow></mfrac></mrow></msup><mi>d</mi><msub><mrow><mi>p</mi></mrow><mrow><mi>α</mi></mrow></msub><mi>d</mi><msub><mrow><mi>p</mi></mrow><mrow><mi>β</mi></mrow></msub><mi>d</mi><mi>α</mi><mi>d</mi><mi>β</mi><mo>×</mo><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">spin</mi></mrow></msub></mtd></mtr><mtr><mtd columnalign="center"><mo>=</mo><msup><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mo>−</mo><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>−</mo><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">MT</mi></mrow></msub></mrow><mrow><mi>k</mi><mi>T</mi></mrow></mfrac></mrow></msup><mo>×</mo><mn>4</mn><mi>π</mi><msup><mrow><mi>R</mi></mrow><mrow><mn>2</mn></mrow></msup><mi>σ</mi><mo>×</mo><mfrac><mrow><mn>1</mn></mrow><mrow><msup><mrow><mi>h</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac><mo>×</mo><mn>4</mn><msup><mrow><mi>π</mi></mrow><mrow><mn>2</mn></mrow></msup><mi>∬</mi><msup><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mfenced close=")" open="("><mrow><mo>−</mo><msubsup><mrow><mi>p</mi></mrow><mrow><mi>α</mi></mrow><mrow><mn>2</mn></mrow></msubsup><mo>−</mo><msubsup><mrow><mi>p</mi></mrow><mrow><mi>β</mi></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></mfenced></mrow><mrow><mn>2</mn><mi>I</mi><mi>k</mi><mi>T</mi></mrow></mfrac></mrow></msup><mi>d</mi><msub><mrow><mi>p</mi></mrow><mrow><mi>α</mi></mrow></msub><mi>d</mi><msub><mrow><mi>p</mi></mrow><mrow><mi>β</mi></mrow></msub><mo>×</mo><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">spin</mi></mrow></msub></mtd></mtr><mtr><mtd columnalign="center"><mtable><mtr><mtd columnalign="center"><mo>=</mo><msup><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mo>−</mo><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>−</mo><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">MT</mi></mrow></msub></mrow><mrow><mi>k</mi><mi>T</mi></mrow></mfrac></mrow></msup><mo>×</mo><mn>4</mn><mi>π</mi><msup><mrow><mi>R</mi></mrow><mrow><mn>2</mn></mrow></msup><mi>σ</mi><mo>×</mo><mfrac><mrow><mn>1</mn></mrow><mrow><msup><mrow><mi>h</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac><mo>×</mo><mn>4</mn><msup><mrow><mi>π</mi></mrow><mrow><mn>2</mn></mrow></msup><mo>×</mo><msup><mrow><msqrt><mrow><mn>2</mn><mi>π</mi><mi>I</mi><mi>k</mi><mi>T</mi></mrow></msqrt></mrow><mrow><mn>2</mn></mrow></msup><mo>×</mo><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">spin</mi></mrow></msub></mtd></mtr><mtr><mtd columnalign="center"><mo>=</mo><msup><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mo>−</mo><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>−</mo><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">MT</mi></mrow></msub></mrow><mrow><mi>k</mi><mi>T</mi></mrow></mfrac></mrow></msup><mo>×</mo><mfrac><mrow><mn>32</mn><msup><mrow><mi>π</mi></mrow><mrow><mn>4</mn></mrow></msup><mi>σ</mi><mi>k</mi><mi>T</mi></mrow><mrow><msup><mrow><mi>h</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac><mo>×</mo><msup><mrow><mi>R</mi></mrow><mrow><mn>2</mn></mrow></msup><mo>×</mo><mfrac><mrow><mn>4</mn></mrow><mrow><mn>3</mn></mrow></mfrac><mi>π</mi><mi>ρ</mi><mfenced close=")" open="("><mrow><mfrac><mrow><mn>7</mn></mrow><mrow><mn>5</mn></mrow></mfrac><msup><mrow><mi>R</mi></mrow><mrow><mn>5</mn></mrow></msup><mo>+</mo><mn>2</mn><msup><mrow><mi>R</mi></mrow><mrow><mn>4</mn></mrow></msup><msub><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">Dy</mi></mrow></msub><mo>+</mo><msup><mrow><mi>R</mi></mrow><mrow><mn>3</mn></mrow></msup><msubsup><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">D</mi><mi>y</mi></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></mfenced><mo>×</mo><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">spin</mi></mrow></msub></mtd></mtr><mtr><mtd columnalign="center"><mo>=</mo><msup><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mo>−</mo><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>−</mo><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">MT</mi></mrow></msub></mrow><mrow><mi>k</mi><mi>T</mi></mrow></mfrac></mrow></msup><mo>×</mo><mfrac><mrow><mn>128</mn><msup><mrow><mi>π</mi></mrow><mrow><mn>5</mn></mrow></msup><mi>σ</mi><mi>ρ</mi><mi>k</mi><mi>T</mi></mrow><mrow><msup><mrow><mn>3</mn><mi>h</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac><mo>×</mo><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">spin</mi></mrow></msub><mo>×</mo><mfenced close=")" open="("><mrow><mfrac><mrow><mn>7</mn></mrow><mrow><mn>5</mn></mrow></mfrac><msup><mrow><mi>R</mi></mrow><mrow><mn>7</mn></mrow></msup><mo>+</mo><mn>2</mn><msup><mrow><mi>R</mi></mrow><mrow><mn>6</mn></mrow></msup><msub><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">Dy</mi></mrow></msub><mo>+</mo><msup><mrow><mi>R</mi></mrow><mrow><mn>5</mn></mrow></msup><msubsup><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">D</mi><mi>y</mi></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></mfenced></mtd></mtr></mtable></mtd></mtr></mtable></math></td>
<td class="label">9</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq116"><math id="d33e4800"><mi>α</mi></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq117"><math id="d33e4810"><mi>β</mi></math></span> are angular coordinates; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq118"><math id="d33e4819"><msub><mrow><mi>p</mi></mrow><mrow><mi>α</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq119"><math id="d33e4832"><msub><mrow><mi>p</mi></mrow><mrow><mi>β</mi></mrow></msub></math></span> are the conjugate angular momentum; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq120"><math id="d33e4845"><mi>σ</mi></math></span> is the surface density of the dynein binding sites; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq121"><math id="d33e4854"><mi>ρ</mi></math></span> is the density of the cargo; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq122"><math id="d33e4863"><mi>I</mi></math></span> is the moment of inertia as a function of dynein length <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq123"><math id="d33e4873"><msub><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">Dy</mi></mrow></msub></math></span> and cargo radius <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq124"><math id="d33e4887"><mi>R</mi></math></span>; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq125"><math id="d33e4896"><mi>T</mi></math></span> is the temperature; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq126"><math id="d33e4905"><mi>h</mi></math></span> is the Planck constant, a placeholder representing the unit of phase space; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq127"><math id="d33e4914"><mi>k</mi></math></span> is the Boltzmann constant.</p>
<p id="Par124">2) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq128"><math id="d33e4925"><msub><mrow><mi mathvariant="bold-italic">Z</mi></mrow><mrow><mi mathvariant="bold">C</mi></mrow></msub></math></span></p>
<p id="Par125">The partition function for free cargo only contains the translational and spin DOFs.</p>
<table class="disp-formula p" id="Equ10"><tr>
<td class="formula"><math id="d33e4942" display="block"><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><msup><mrow><mi>h</mi></mrow><mrow><mn>3</mn></mrow></msup></mrow></mfrac><mi>∬</mi><mspace width="0.3em"></mspace><mspace width="0.3em"></mspace><mo>∫</mo><msup><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mfenced close=")" open="("><mrow><mo>−</mo><msubsup><mrow><mi>p</mi></mrow><mrow><mi>x</mi></mrow><mrow><mn>2</mn></mrow></msubsup><mo>−</mo><msubsup><mrow><mi>p</mi></mrow><mrow><mi>y</mi></mrow><mrow><mn>2</mn></mrow></msubsup><mo>−</mo><msubsup><mrow><mi>p</mi></mrow><mrow><mi>z</mi></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></mfenced></mrow><mrow><mn>2</mn><mi>m</mi><mi>k</mi><mi>T</mi></mrow></mfrac></mrow></msup><mi>d</mi><msub><mrow><mi>p</mi></mrow><mrow><mi>x</mi></mrow></msub><mi>d</mi><msub><mrow><mi>p</mi></mrow><mrow><mi>y</mi></mrow></msub><mi>d</mi><msub><mrow><mi>p</mi></mrow><mrow><mi>z</mi></mrow></msub><mi>∬</mi><mspace width="0.3em"></mspace><mspace width="0.3em"></mspace><mo>∫</mo><mi>d</mi><mi>x</mi><mi>d</mi><mi>y</mi><mi>d</mi><mi>z</mi><mo>×</mo><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">spin</mi></mrow></msub><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><msup><mrow><mi>h</mi></mrow><mrow><mn>3</mn></mrow></msup></mrow></mfrac><mo>×</mo><msub><mrow><mi>V</mi></mrow><mrow><mi mathvariant="normal">free</mi></mrow></msub><mo>×</mo><msub><mrow><msup><mrow><mfenced close=")" open="("><mrow><mn>2</mn><mi>π</mi><mi>m</mi><mi>k</mi><mi>T</mi></mrow></mfenced></mrow><mrow><mfrac><mrow><mn>3</mn></mrow><mrow><mn>2</mn></mrow></mfrac></mrow></msup><mo>×</mo><mi>Z</mi></mrow><mrow><mi mathvariant="normal">spin</mi></mrow></msub><mo>=</mo><msub><mrow><msup><mrow><mfenced close=")" open="("><mrow><mfrac><mrow><mn>8</mn><msup><mrow><mi>π</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow><mrow><mn>3</mn><msup><mrow><mi>h</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac><mi>k</mi><mi>T</mi></mrow></mfenced></mrow><mrow><mn>3</mn><mo>/</mo><mn>2</mn></mrow></msup><mo>×</mo><msub><mrow><mi>V</mi></mrow><mrow><mi mathvariant="normal">free</mi></mrow></msub><mo>×</mo><mi>Z</mi></mrow><mrow><mi mathvariant="normal">spin</mi></mrow></msub><mo>×</mo><msup><mrow><mi>R</mi></mrow><mrow><mn>4.5</mn></mrow></msup></math></td>
<td class="label">10</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq129"><math id="d33e5131"><msub><mrow><mi>V</mi></mrow><mrow><mi mathvariant="normal">free</mi></mrow></msub></math></span> is the integration over the space where cargo can freely diffuse.</p>
<p id="Par126">3) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq130"><math id="d33e5147"><msub><mrow><mi mathvariant="bold-italic">Z</mi></mrow><mrow><mi mathvariant="bold">C</mi><mo>−</mo><mi mathvariant="bold">Dy</mi></mrow></msub></math></span></p>
<table class="disp-formula p" id="Equ11"><tr>
<td class="formula"><math id="d33e5165" display="block"><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi><mo>−</mo><mi mathvariant="normal">Dy</mi></mrow></msub><mo>=</mo><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>×</mo><msub><mrow><mi>N</mi></mrow><mrow><mi>r</mi></mrow></msub><mo>=</mo><msub><mrow><mn>4</mn><mi>π</mi><mi>σ</mi><msup><mrow><mfenced close=")" open="("><mrow><mfrac><mrow><mn>8</mn><msup><mrow><mi>π</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow><mrow><mn>3</mn><msup><mrow><mi>h</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac><mi>k</mi><mi>T</mi></mrow></mfenced></mrow><mrow><mn>3</mn><mo>/</mo><mn>2</mn></mrow></msup><mo>×</mo><msub><mrow><mi>V</mi></mrow><mrow><mi mathvariant="normal">free</mi></mrow></msub><mo>×</mo><mi>Z</mi></mrow><mrow><mi mathvariant="normal">spin</mi></mrow></msub><mo>×</mo><msup><mrow><mi>R</mi></mrow><mrow><mn>6.5</mn></mrow></msup><mo>×</mo><msup><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mo>−</mo><msub><mrow><mi>E</mi></mrow><mrow><mi mathvariant="normal">MT</mi></mrow></msub></mrow><mrow><mi>k</mi><mi>T</mi></mrow></mfrac></mrow></msup><mo>,</mo></math></td>
<td class="label">11</td>
</tr></table>
<p>given the multiplicity of dynein binding sites and the negligible size of dynein compared to the cargo.</p>
<p id="Par127">4) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq131"><math id="d33e5266"><msub><mrow><mi mathvariant="bold-italic">Z</mi></mrow><mrow><msub><mrow><mi mathvariant="bold">C</mi></mrow><mrow><mi mathvariant="bold">L</mi></mrow></msub></mrow></msub><mo>,</mo><msub><mrow><mi mathvariant="bold-italic">Z</mi></mrow><mrow><msub><mrow><mi mathvariant="bold">C</mi></mrow><mrow><mi mathvariant="bold">L</mi></mrow></msub><mo>−</mo><mi mathvariant="bold">Dy</mi></mrow></msub></math></span></p>
<p id="Par128">Generally, it is unclear how to write down the exact partition function for the local states, since its molecular nature is not clear. As explained below, this difficulty is circumvented by introducing an empirical parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq132"><math id="d33e5303"><mi>α</mi><mfenced close=")" open="("><mrow><mi>R</mi></mrow></mfenced></math></span>, the likelihood of cargo in the freely-diffusing state, which can be approximated by the measurement in Fig. <a href="#Fig2" class="usa-link">2G</a>.</p>
<p id="Par129">Analytical expressions for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq133"><math id="d33e5320"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> as a function of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq134"><math id="d33e5333"><mi>R</mi></math></span> can be obtained under two conditions:</p>
<ul class="list" style="list-style-type:none">
<li>
<span class="label">I.</span><div class="display-inline p" id="Par130">
<strong>The active transport complex is transient or has low stability</strong>, where most cargoes are not forming an active transport complex with dynein-microtubule at the steady state. This condition implies <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq135"><math id="d33e5350"><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>,</mo><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub></mrow></msub><mo>≫</mo><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi><mo>−</mo><mi mathvariant="normal">Dy</mi></mrow></msub><mo>,</mo><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi></mrow></msub></math></span>.<table class="disp-formula p" id="Equ12"><tr>
<td class="formula"><math id="d33e5402" display="block"><msub><mrow><mi>P</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub><mo>≈</mo><mfrac><mrow><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub></mrow><mrow><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>+</mo><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub></mrow></msub></mrow></mfrac><mo>=</mo><mfrac><mrow><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub></mrow><mrow><mfrac><mrow><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow><mrow><mi>α</mi><mfenced close=")" open="("><mrow><mi>R</mi></mrow></mfenced></mrow></mfrac></mrow></mfrac><mo>.</mo></math></td>
<td class="label">12</td>
</tr></table>
</div>
<div class="p" id="Par131">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq136"><math id="d33e5501"><mi>α</mi><mfenced close=")" open="("><mrow><mi>R</mi></mrow></mfenced></math></span> is the probability of cargo in the freely-diffusing state. Under the low-stability condition, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq137"><math id="d33e5513"><mi>α</mi><mfenced close=")" open="("><mrow><mi>R</mi></mrow></mfenced><mo>=</mo><mfrac><mrow><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow><mrow><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>+</mo><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub></mrow></msub></mrow></mfrac></math></span>. We can rewrite the above equation as<table class="disp-formula p" id="Equ13"><tr>
<td class="formula"><math id="d33e5554" display="block"><msub><mrow><mi>P</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub><mo>∝</mo><mfenced close=")" open="("><mrow><mfrac><mrow><mn>7</mn></mrow><mrow><mn>5</mn></mrow></mfrac><msup><mrow><mi>R</mi></mrow><mrow><mn>2.5</mn></mrow></msup><mo>+</mo><mn>2</mn><msup><mrow><mi>R</mi></mrow><mrow><mn>1.5</mn></mrow></msup><msub><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">Dy</mi></mrow></msub><mo>+</mo><msup><mrow><mi>R</mi></mrow><mrow><mn>0.5</mn></mrow></msup><msubsup><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">Dy</mi></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></mfenced><mo>×</mo><mi>α</mi><mfenced close=")" open="("><mrow><mi>R</mi></mrow></mfenced><mo>,</mo></math></td>
<td class="label">13</td>
</tr></table>and,<table class="disp-formula p" id="Equ14"><tr>
<td class="formula"><math id="d33e5626" display="block"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>∝</mo><mfenced close=")" open="("><mrow><mfrac><mrow><mn>7</mn></mrow><mrow><mn>5</mn></mrow></mfrac><msup><mrow><mi>R</mi></mrow><mrow><mn>2.5</mn></mrow></msup><mo>+</mo><mn>2</mn><msup><mrow><mi>R</mi></mrow><mrow><mn>1.5</mn></mrow></msup><msub><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">Dy</mi></mrow></msub><mo>+</mo><msup><mrow><mi>R</mi></mrow><mrow><mn>0.5</mn></mrow></msup><msubsup><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">Dy</mi></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></mfenced><mo>×</mo><mi>α</mi><mfenced close=")" open="("><mrow><mi>R</mi></mrow></mfenced><mo>×</mo><mfrac><mrow><msub><mrow><mi>F</mi></mrow><mrow><mi mathvariant="normal">s</mi></mrow></msub></mrow><mrow><mn>6</mn><mi>π</mi><mi>η</mi><mi>R</mi></mrow></mfrac><mo>.</mo></math></td>
<td class="label">14</td>
</tr></table>
</div>
<div class="p" id="Par132">Considering <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq138"><math id="d33e5700"><mi>R</mi><mo>≫</mo><msub><mrow><mi>l</mi></mrow><mrow><mi>D</mi><mi>y</mi></mrow></msub></math></span> in most examples, we get<table class="disp-formula p" id="Equ15"><tr>
<td class="formula"><math id="d33e5717" display="block"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>∝</mo><msup><mrow><mi>R</mi></mrow><mrow><mn>1.5</mn></mrow></msup><mo>×</mo><mi>α</mi><mfenced close=")" open="("><mrow><mi>R</mi></mrow></mfenced><mo>.</mo></math></td>
<td class="label">15</td>
</tr></table>
</div>
</li>
<li>
<span class="label">II.</span><p class="display-inline" id="Par133"><strong>The active transport complex has high stability</strong>, where most cargoes in the local state are engaged in the active transport complex. This condition implies <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq139"><math id="d33e5748"><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi><mo>−</mo><mi mathvariant="normal">Dy</mi></mrow></msub><mo>,</mo><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi></mrow></msub><mo>≪</mo><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub></math></span>.</p>
</li>
</ul>
<table class="disp-formula p" id="Equ16"><tr>
<td class="formula"><math id="d33e5799" display="block"><msub><mrow><mi>P</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub><mo>≈</mo><mfrac><mrow><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub></mrow><mrow><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub><mo>+</mo><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi><mo>−</mo><mi mathvariant="normal">Dy</mi></mrow></msub><mo>+</mo><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow></mfrac><mo>.</mo></math></td>
<td class="label">16</td>
</tr></table>
<p id="Par134">Under this high-stability condition, <em>α</em>(<em>R</em>) = <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq140"><math id="d33e5887"><mfrac><mrow><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi><mo>−</mo><mi mathvariant="normal">Dy</mi></mrow></msub><mo>+</mo><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow><mrow><msub><mrow><mi>Z</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub><mo>+</mo><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi><mo>−</mo><mi mathvariant="normal">Dy</mi></mrow></msub><mo>+</mo><msub><mrow><mi>Z</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow></mfrac></math></span>. Therefore,</p>
<table class="disp-formula p" id="Equ17"><tr>
<td class="formula"><math id="d33e5949" display="block"><msub><mrow><mi>P</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">C</mi></mrow><mrow><mi mathvariant="normal">L</mi></mrow></msub><mo>−</mo><mi mathvariant="normal">Dy</mi><mo>−</mo><mi mathvariant="normal">MT</mi></mrow></msub><mo>≈</mo><mn>1</mn><mo>−</mo><mi>α</mi><mfenced close=")" open="("><mrow><mi>R</mi></mrow></mfenced><mo>.</mo></math></td>
<td class="label">17</td>
</tr></table>
<p>And,</p>
<table class="disp-formula p" id="Equ18"><tr>
<td class="formula"><math id="d33e5983" display="block"><msub><mrow><mi>v</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>∝</mo><mfrac><mrow><msub><mrow><mi>F</mi></mrow><mrow><mi mathvariant="normal">s</mi></mrow></msub></mrow><mrow><mn>6</mn><mi>π</mi><mi>η</mi><mi>R</mi></mrow></mfrac><mo>×</mo><mfenced open="("><mrow><mn>1</mn><mo>−</mo><mi>α</mi><mfenced close=")" open="("><mrow><mi>R</mi></mrow></mfenced></mrow></mfenced><mo>.</mo></math></td>
<td class="label">18</td>
</tr></table></section><section id="Sec50"><h3 class="pmc_sec_title">A stochastic numerical model for dynein-mediated cargo transport</h3>
<section id="Sec51"><h4 class="pmc_sec_title">Model structure and assumptions</h4>
<p id="Par135">The goal here is to establish a stochastic numerical model to validate the conclusions of the analytical model by simulating the stochastic transitions between states at a given stability of the transport complex. The states and their transitions are identical to those in the analytical model, except that we use explicit values for the transition rates and the viscosity-limited velocity of the transport complex.<img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3fea/12373839/55ffedf1da5e/41467_2025_62751_Figc_HTML.gif" loading="lazy" id="d33e6021" alt="Inline graphic"></p>
<p id="Par136">We assume that the bindings among cargo, dynein, and microtubule reach equilibrium, the same as in the analytical model. The details of parameter determination are shown below (Table <a href="#Tab1" class="usa-link">1</a>). For parameters that were not directly measured in this study, we estimated their values by fitting the simulation results to satisfy the following criteria: (1) the simulated average transport velocity for the unstable transport complex case (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq141"><math id="d33e6032"><msub><mrow><mi>p</mi></mrow><mrow><mn>6</mn></mrow></msub></math></span> = 0.1, see below) approximates the measurements in Fig. <a href="#Fig2" class="usa-link">2D</a>; (2) the simulated average transport velocity for the stable transport complex case (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq142"><math id="d33e6048"><msub><mrow><mi>p</mi></mrow><mrow><mn>6</mn></mrow></msub></math></span> = 0.0001, see below) matches that of HOOK2-coated beads in Fig. <a href="#Fig4" class="usa-link">4</a>.</p>
<section class="tw xbox font-sm" id="Tab1"><h5 class="obj_head">Table 1.</h5>
<div class="caption p"><p>Table of model parameters</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Kinetic parameter</th>
<th colspan="1" rowspan="1">Expression</th>
<th colspan="1" rowspan="1">Parameter</th>
<th colspan="1" rowspan="1">Value</th>
<th colspan="1" rowspan="1">Source<sup>a</sup>
</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><math id="d33e6094"><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">on</mi></mrow></msub><mo>,</mo><msubsup><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">on</mi></mrow><mrow><mo>′</mo></mrow></msubsup></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><math id="d33e6118"><msub><mrow><msub><mrow><mi>p</mi></mrow><mrow><mn>0</mn></mrow></msub><mi>D</mi></mrow><mrow><mi mathvariant="normal">D</mi></mrow></msub><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><math id="d33e6144"><msub><mrow><mi>p</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span></td>
<td colspan="1" rowspan="1">2 × 10<sup>18 </sup>m<sup>−3</sup>
</td>
<td colspan="1" rowspan="1">this study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><math id="d33e6167"><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">off</mi></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><math id="d33e6182"><msubsup><mrow><mi>p</mi></mrow><mrow><mn>0</mn></mrow><mrow><mo>′</mo></mrow></msubsup><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">on</mi></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><math id="d33e6204"><msubsup><mrow><mi>p</mi></mrow><mrow><mn>0</mn></mrow><mrow><mo>′</mo></mrow></msubsup></math></span></td>
<td colspan="1" rowspan="1">0.1</td>
<td colspan="1" rowspan="1">this study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><math id="d33e6225"><msub><mrow><mi>k</mi></mrow><mrow><mn>1</mn></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><math id="d33e6239"><msub><mrow><mi>p</mi></mrow><mrow><mn>1</mn></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><math id="d33e6253"><msub><mrow><mi>p</mi></mrow><mrow><mn>1</mn></mrow></msub></math></span></td>
<td colspan="1" rowspan="1">0.0025 s<sup>−</sup><sup>1</sup>
</td>
<td colspan="1" rowspan="1">Fig. <a href="#Fig2" class="usa-link">2F</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><math id="d33e6278"><msub><mrow><mi>k</mi></mrow><mrow><mn>2</mn></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><math id="d33e6292"><msub><mrow><mi>p</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>/</mo><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><math id="d33e6313"><msub><mrow><mi>p</mi></mrow><mrow><mn>2</mn></mrow></msub></math></span></td>
<td colspan="1" rowspan="1">3.33 × 10<sup>−</sup><sup>9 </sup>m s<sup>−</sup><sup>1</sup>
</td>
<td colspan="1" rowspan="1">Fig. <a href="#Fig2" class="usa-link">2F</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><math id="d33e6341"><msub><mrow><mi>k</mi></mrow><mrow><mn>3</mn></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><math id="d33e6355"><msub><mrow><mi>p</mi></mrow><mrow><mn>3</mn></mrow></msub><msup><mrow><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow><mrow><mn>0.5</mn></mrow></msup></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><math id="d33e6379"><msub><mrow><mi>p</mi></mrow><mrow><mn>3</mn></mrow></msub></math></span></td>
<td colspan="1" rowspan="1">50.265 m<sup>−0.5</sup> s<sup>−</sup><sup>1</sup>
</td>
<td colspan="1" rowspan="1">this study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><math id="d33e6403"><msub><mrow><mi>k</mi></mrow><mrow><mn>4</mn></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><math id="d33e6417"><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">on</mi></mrow></msub><msub><mrow><mi>k</mi></mrow><mrow><mn>3</mn></mrow></msub><msub><mrow><mi>k</mi></mrow><mrow><mn>6</mn></mrow></msub><mo>/</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>k</mi></mrow><mrow><mn>5</mn></mrow></msub><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">off</mi></mrow></msub></mrow><mo>)</mo></mrow></math></span></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1">detailed balance</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><math id="d33e6463"><msub><mrow><mi>k</mi></mrow><mrow><mn>5</mn></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><math id="d33e6477"><msup><mrow><msub><mrow><mi>p</mi></mrow><mrow><mn>5</mn></mrow></msub><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow><mrow><mn>2</mn></mrow></msup><msub><mrow><mi>k</mi></mrow><mrow><mn>3</mn></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><math id="d33e6506"><msub><mrow><mi>p</mi></mrow><mrow><mn>5</mn></mrow></msub></math></span></td>
<td colspan="1" rowspan="1">5 × 10<sup>12 </sup>m<sup>−</sup><sup>2</sup>
</td>
<td colspan="1" rowspan="1">this study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><math id="d33e6530"><msub><mrow><mi>k</mi></mrow><mrow><mn>6</mn></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><math id="d33e6544"><msub><mrow><msub><mrow><mi>p</mi></mrow><mrow><mn>6</mn></mrow></msub><mi>k</mi></mrow><mrow><mi mathvariant="normal">off</mi></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><math id="d33e6564"><msub><mrow><mi>p</mi></mrow><mrow><mn>6</mn></mrow></msub></math></span></td>
<td colspan="1" rowspan="1">10<sup>−</sup><sup>1</sup> to 10<sup>−</sup><sup>4</sup>
</td>
<td colspan="1" rowspan="1">manually set</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><math id="d33e6589"><msub><mrow><mi>v</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><math id="d33e6604"><msub><mrow><mi>p</mi></mrow><mrow><mi mathvariant="normal">v</mi></mrow></msub><mo>/</mo><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></math></span></td>
<td colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><math id="d33e6626"><msub><mrow><mi>p</mi></mrow><mrow><mi mathvariant="normal">v</mi></mrow></msub></math></span></td>
<td colspan="1" rowspan="1">3 × 10<sup>−</sup><sup>13</sup> m<sup>2</sup> s<sup>−</sup><sup>1</sup>
</td>
<td colspan="1" rowspan="1">Fig. <a href="#Fig3" class="usa-link">3F</a>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p144"><p><sup>a</sup>All parameters in the model were either estimated or defined in this study. Specifically, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><math id="d33e6658"><msub><mrow><mi>k</mi></mrow><mrow><mn>1</mn></mrow></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><math id="d33e6671"><msub><mrow><mi>k</mi></mrow><mrow><mn>2</mn></mrow></msub></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><math id="d33e6684"><msub><mrow><mi>v</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></math></span> were inferred from experimental measurements; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><math id="d33e6698"><msubsup><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">on</mi></mrow><mrow><mo>′</mo></mrow></msubsup></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><math id="d33e6714"><msub><mrow><mi>k</mi></mrow><mrow><mn>4</mn></mrow></msub></math></span> were derived based on detailed balance; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><math id="d33e6728"><msub><mrow><mi>k</mi></mrow><mrow><mn>6</mn></mrow></msub></math></span> was manually set; and all remaining parameters were estimated by fitting simulation results.</p></div></div></section></section><section id="Sec52"><h4 class="pmc_sec_title">Determination of model parameters</h4>
<p id="Par137">We set the transition rates between the five states and the velocity of the active transport complex as follows:</p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par138"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq143"><math id="d33e6750"><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">on</mi></mrow></msub><mo>=</mo><mn>4</mn><mi>π</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">D</mi></mrow></msub><mo>+</mo><msub><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow></mfenced><mfenced close=")" open="("><mrow><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">D</mi></mrow></msub><mo>+</mo><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow></mfenced><mfenced close="]" open="["><mrow><mi mathvariant="normal">Dy</mi></mrow></mfenced><mo>×</mo><mn>1000</mn><msub><mrow><mi>N</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msub></math></span>, calculated as a diffusion-controlled kinetic rate<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq144"><math id="d33e6813"><msub><mrow><mi>N</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msub></math></span> is the Avogadro’s constant. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq145"><math id="d33e6827"><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">D</mi></mrow></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq146"><math id="d33e6841"><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></math></span> denote the radii of dynein and cargo. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq147"><math id="d33e6855"><msub><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">D</mi></mrow></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq148"><math id="d33e6870"><msub><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></math></span> are the diffusion constants of dynein and cargo, respectively, which can be calculated using the Stokes-Einstein equation: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq149"><math id="d33e6884"><msub><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">D</mi></mrow></msub><mo>=</mo><msub><mrow><mi>k</mi></mrow><mrow><mi>B</mi></mrow></msub><mi>T</mi><mo>/</mo><mrow><mo>(</mo><mrow><mn>6</mn><mi>π</mi><mi>η</mi><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">D</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>=</mo><mn>4.4</mn></math></span> μm<sup>2</sup>/s, given <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq150"><math id="d33e6923"><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">D</mi></mrow></msub><mo>=</mo><mn>10</mn></math></span> nm. Since <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq151"><math id="d33e6939"><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>≫</mo><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">D</mi></mrow></msub></math></span>, the expression <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq152"><math id="d33e6960"><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">on</mi></mrow></msub></math></span> can be simplified as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq153"><math id="d33e6975"><msub><mrow><msub><mrow><mi>p</mi></mrow><mrow><mn>0</mn></mrow></msub><mi>D</mi></mrow><mrow><mi mathvariant="normal">D</mi></mrow></msub><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq154"><math id="d33e7000"><msub><mrow><mi>p</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>=</mo><mn>4000</mn><mi>π</mi><msub><mrow><mi>N</mi></mrow><mrow><mi mathvariant="normal">A</mi></mrow></msub><mfenced close="]" open="["><mrow><mi mathvariant="normal">Dy</mi></mrow></mfenced></math></span>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq155"><math id="d33e7026"><mfenced close="]" open="["><mrow><mi mathvariant="normal">Dy</mi></mrow></mfenced></math></span> is the concentration of free dynein away from microtubules, in mol/L, which is estimated to be 0.3 nmol/L based on the simulation results.</p></li>
<li><p id="Par139"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq156"><math id="d33e7042"><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">off</mi></mrow></msub><mo>=</mo><msub><mrow><msub><mrow><mi>K</mi></mrow><mrow><mi mathvariant="normal">D</mi></mrow></msub><mi>k</mi></mrow><mrow><mi mathvariant="normal">on</mi></mrow></msub><mo>/</mo><mfenced close="]" open="["><mrow><mi mathvariant="normal">Dy</mi></mrow></mfenced><mo>=</mo><msubsup><mrow><mi>p</mi></mrow><mrow><mn>0</mn></mrow><mrow><mo>′</mo></mrow></msubsup><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">on</mi></mrow></msub></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq157"><math id="d33e7088"><msub><mrow><mi>K</mi></mrow><mrow><mi mathvariant="normal">D</mi></mrow></msub></math></span> is the dissociation constant of the adapter-dynein interaction and insensitive to whether the cargo is in the freely diffusing state or local dynein cycle. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq158"><math id="d33e7102"><msub><mrow><mi>K</mi></mrow><mrow><mi mathvariant="normal">D</mi></mrow></msub></math></span> is estimated to be 0.03 nmol/L based on the simulation results.</p></li>
<li><p id="Par140"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq159"><math id="d33e7120"><msub><mrow><mi>k</mi></mrow><mrow><mn>1</mn></mrow></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq160"><math id="d33e7133"><msub><mrow><mi>k</mi></mrow><mrow><mn>2</mn></mrow></msub></math></span> are the transition rates between the local dynein cycle and the freely diffusing state. We estimate <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq161"><math id="d33e7146"><msub><mrow><mi>k</mi></mrow><mrow><mn>1</mn></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq162"><math id="d33e7159"><msub><mrow><mi>k</mi></mrow><mrow><mn>2</mn></mrow></msub></math></span> based on experimentally determined pause and TE (+) lengths in Fig. <a href="#Fig2" class="usa-link">2F</a> as follows: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq163"><math id="d33e7175"><msub><mrow><mi>k</mi></mrow><mrow><mn>1</mn></mrow></msub></math></span> is a constant (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq164"><math id="d33e7189"><msub><mrow><mi>p</mi></mrow><mrow><mn>1</mn></mrow></msub></math></span>) because the mean pause length is independent of cargo radius. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq165"><math id="d33e7202"><msub><mrow><mi>k</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi mathvariant="normal">mean</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">TE</mi><mrow><mo>(</mo><mrow><mo>+</mo></mrow><mo>)</mo></mrow><mi mathvariant="normal">length</mi></mrow></mfrac><mo>∝</mo><msub><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>=</mo><mfrac><mrow><msub><mrow><mi>p</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow><mrow><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow></mfrac></math></span>, where the rate is proportional to the diffusion constant and thus inversely proportional to the cargo radius, with a parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq166"><math id="d33e7254"><msub><mrow><mi>p</mi></mrow><mrow><mn>2</mn></mrow></msub></math></span>.</p></li>
<li><p id="Par141"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq167"><math id="d33e7271"><msubsup><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">on</mi></mrow><mrow><mo>′</mo></mrow></msubsup><mo>=</mo><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">on</mi></mrow></msub></math></span>, to satisfy detailed balance.</p></li>
<li><p id="Par142"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq168"><math id="d33e7298"><msub><mrow><mi>k</mi></mrow><mrow><mn>3</mn></mrow></msub><mo>∝</mo><mi>S</mi><mi>e</mi><mo>∝</mo><msup><mrow><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow><mrow><mn>0.5</mn></mrow></msup><mo>=</mo><msub><mrow><mi>p</mi></mrow><mrow><mn>3</mn></mrow></msub><msup><mrow><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow><mrow><mn>0.5</mn></mrow></msup></math></span>, where given a fixed MT density and restricted diffusion within the local dynein cycle (Fig. <a href="#Fig3" class="usa-link">3I</a>), the binding rate between cargo and microtubule is proportional to the accessible surface area where the dynein can interact with the MT (boldly drawn in blue; see below for detailed calculation)<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup>. As suggested by our study, we only consider the case of a single dynein binding to the cargo. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq169"><math id="d33e7348"><msub><mrow><mi>p</mi></mrow><mrow><mn>3</mn></mrow></msub></math></span> is estimated to be 50.265 m<sup>−</sup><sup>0.5</sup> s<sup>−</sup><sup>1</sup> (see below).</p></li>
<li><p id="Par143"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq170"><math id="d33e7372"><msub><mrow><mi>k</mi></mrow><mrow><mn>5</mn></mrow></msub><mo>∝</mo><msup><mrow><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow><mrow><mn>2</mn></mrow></msup><msub><mrow><mi>k</mi></mrow><mrow><mn>3</mn></mrow></msub><mo>=</mo><msup><mrow><msub><mrow><mi>p</mi></mrow><mrow><mn>5</mn></mrow></msub><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow><mrow><mn>2</mn></mrow></msup><msub><mrow><mi>k</mi></mrow><mrow><mn>3</mn></mrow></msub></math></span>, where the number of adapters on cargo is proportional to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq171"><math id="d33e7422"><msup><mrow><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow><mrow><mn>2</mn></mrow></msup></math></span>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq172"><math id="d33e7440"><msub><mrow><mi>p</mi></mrow><mrow><mn>5</mn></mrow></msub></math></span> is estimated to be 5 × 10<sup>12 </sup>m<sup>−</sup><sup>2</sup>.</p></li>
<li><p id="Par144"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq173"><math id="d33e7462"><msub><mrow><msub><mrow><mi>k</mi></mrow><mrow><mn>6</mn></mrow></msub><mo>=</mo><msub><mrow><mi>p</mi></mrow><mrow><mn>6</mn></mrow></msub><mi>k</mi></mrow><mrow><mi mathvariant="normal">off</mi></mrow></msub></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq174"><math id="d33e7487"><msub><mrow><mi>p</mi></mrow><mrow><mn>6</mn></mrow></msub></math></span> is a parameter that accounts for the differences between the dissociation rate of cargo from free dynein versus MT-bound dynein. Note, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq175"><math id="d33e7500"><msub><mrow><mi>p</mi></mrow><mrow><mn>6</mn></mrow></msub></math></span> is set to a small value (0.0001) when simulating the case of stable transport complex while set to a large value (0.1) when simulating the case of unstable transport complex.</p></li>
<li><p id="Par145"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq176"><math id="d33e7517"><msub><mrow><mi>k</mi></mrow><mrow><mn>4</mn></mrow></msub><mo>=</mo><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">on</mi></mrow></msub><msub><mrow><mi>k</mi></mrow><mrow><mn>3</mn></mrow></msub><msub><mrow><mi>k</mi></mrow><mrow><mn>6</mn></mrow></msub><mo>/</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>k</mi></mrow><mrow><mn>5</mn></mrow></msub><msub><mrow><mi>k</mi></mrow><mrow><mi mathvariant="normal">off</mi></mrow></msub></mrow><mo>)</mo></mrow></math></span>, to satisfy detailed balance.</p></li>
<li><p id="Par146"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq177"><math id="d33e7567"><msub><mrow><mi>v</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>=</mo><msub><mrow><mi>p</mi></mrow><mrow><mi mathvariant="normal">v</mi></mrow></msub><mo>/</mo><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq178"><math id="d33e7595"><msub><mrow><mi>p</mi></mrow><mrow><mi mathvariant="normal">v</mi></mrow></msub></math></span> is estimated from Fig. <a href="#Fig3" class="usa-link">3F</a> to be 3 × 10<sup>−</sup><sup>13</sup> m<sup>2</sup> s<sup>−</sup><sup>1</sup>.</p></li>
</ol></section><section id="Sec53"><h4 class="pmc_sec_title">Accessible surface area of cargo for dynein-mt interaction, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq179"><math id="d33e7626"><mi mathvariant="bold-italic">Se</mi></math></span>
</h4>
<p id="Par147">To access MT, dynein must be within a distance <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq180"><math id="d33e7637"><msub><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">Dy</mi></mrow></msub></math></span> to the MT surface. In the direction parallel to the plane, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq181"><math id="d33e7651"><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mo>≫</mo><msub><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">Dy</mi></mrow></msub></math></span>, we have</p>
<table class="disp-formula p" id="Equ19"><tr>
<td class="formula"><math id="d33e7673" display="block"><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub><mfenced close=")" open="("><mrow><mn>1</mn><mo>−</mo><mi>cos</mi><msub><mrow><mi>θ</mi></mrow><mrow><mo>∥</mo></mrow></msub></mrow></mfenced><mo>=</mo><msub><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">Dy</mi></mrow></msub><mo>.</mo></math></td>
<td class="label">19</td>
</tr></table>
<p id="Par148">This leads to:</p>
<table class="disp-formula p" id="Equ20"><tr>
<td class="formula"><math id="d33e7707" display="block"><msub><mrow><mi>θ</mi></mrow><mrow><mo>∥</mo></mrow></msub><mo>=</mo><msqrt><mrow><mfrac><mrow><mn>2</mn><msub><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">Dy</mi></mrow></msub></mrow><mrow><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow></mfrac></mrow></msqrt><mo>.</mo></math></td>
<td class="label">20</td>
</tr></table>
<p id="Par149">In the direction perpendicular to the plane, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq182"><math id="d33e7741"><msub><mrow><mi>θ</mi></mrow><mrow><mo>⊥</mo></mrow></msub><mo>=</mo><mfrac><mrow><msub><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">Dy</mi></mrow></msub></mrow><mrow><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow></mfrac></math></span>.</p>
<p id="Par150">Therefore, the surface area on the cargo available for the dynein-MT interaction, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq183"><math id="d33e7772"><mi>S</mi><mi>e</mi></math></span>, can be calculated as follows:</p>
<table class="disp-formula p" id="Equ21"><tr>
<td class="formula"><math id="d33e7783" display="block"><mi>S</mi><mi>e</mi><mo>=</mo><mn>4</mn><mi>π</mi><msup><mrow><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow><mrow><mn>2</mn></mrow></msup><msub><mrow><mi>θ</mi></mrow><mrow><mo>∥</mo></mrow></msub><msub><mrow><mi>θ</mi></mrow><mrow><mo>⊥</mo></mrow></msub><mo>=</mo><mn>4</mn><msqrt><mrow><mn>2</mn></mrow></msqrt><mi>π</mi><msup><mrow><msub><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">Dy</mi></mrow></msub></mrow><mrow><mn>1.5</mn></mrow></msup><msup><mrow><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow><mrow><mn>0.5</mn></mrow></msup><mo>=</mo><mn>5.0265</mn><mo>×</mo><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>11</mn></mrow></msup><msup><mrow><msub><mrow><mi>R</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></mrow><mrow><mn>0.5</mn></mrow></msup></math></td>
<td class="label">21</td>
</tr></table>
<p>given <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq184"><math id="d33e7862"><msub><mrow><mi>l</mi></mrow><mrow><mi mathvariant="normal">Dy</mi></mrow></msub><mo>=</mo><mn>20</mn></math></span> nm.</p></section><section id="Sec54"><h4 class="pmc_sec_title">Simulation of the cargo-transport stochastic model</h4>
<p id="Par152">All simulations were performed in MATLAB with customized code based on Gillespie algorithm. The simulation begins with the cargo in the free state (State 1, “C”). The state of the cargo evolves according to the transition rates, modeled as a Poisson process. Each simulation is terminated once the total simulated time reaches 100,000 s, ensuring the system reaches a steady-state distribution. Cargo transport occurs only when the cargo is in State 4 (“C<sub>L</sub>-Dy-MT”), moving at a constant velocity <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq185"><math id="d33e7884"><msub><mrow><mi>v</mi></mrow><mrow><mi mathvariant="normal">C</mi></mrow></msub></math></span>. The average velocity is determined by simulating 100 particles for each condition.</p></section></section><section id="Sec55"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par153">Further information on research design is available in the <a href="#MOESM14" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec56"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (9.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62751_MOESM2_ESM.pdf</a><sup> (173.8KB, pdf) </sup><p>Description of Additional Supplementary File</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM3_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (9.6MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM4_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 2</a><sup> (21.5MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM5_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 3</a><sup> (731.9KB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM6_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 4</a><sup> (1.3MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM7_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 5</a><sup> (831.6KB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM8_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 6</a><sup> (1.3MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM9_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 7</a><sup> (2MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM10_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 8</a><sup> (1.3MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM11_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 9</a><sup> (12MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM12"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM12_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 1</a><sup> (9.3MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM13"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM13_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 2</a><sup> (45.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM14"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM14_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting summary</a><sup> (2.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM15"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM15_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (24.2MB, pdf) </sup>
</div></div></section></section><section id="Sec57"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM16"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM16_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (12.5MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We are grateful for funding from the National Institute of General Medical Sciences (R01 GM134064-01 to Y.L.; R35 GM131753 to T.M.) and want to thank Lisa McCalla for administrative support. We want to thank Daniel Finley, Tingting Yao, Louis Colson, Randall King, Donghoon Lee, Thuan Beng Saw, Shutao Qi, and Shih-Han Lin for commenting on the manuscript. We want to thank the Core for Imaging Technology &amp; Education (CITE) at Harvard Medical School for assisting with live-cell imaging and thank the staff at the Westlake University Microscopy Core Facility for advice and assistance in light microscopy data collection. We also thank Rachael Jonas-Closs for excellent <em>Xenopus</em> husbandry. Elements in the schematics in Figs. <a href="#Fig2" class="usa-link">2</a>A, <a href="#Fig4" class="usa-link">4</a>A and <a href="#Fig6" class="usa-link">6A</a> were created with BioRender.com with licenses to Y.L. and R.F.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>R.F. and Y.L. conceived and designed the project. R.F., L.B., B.V., B.L., M.Z., and Y.C. performed the experiments. K. D., J. P., and S. G. processed mass spec samples and data analysis. R.F., Y.L., L.B., B.V., Y.S., M.S., C.F., and T.M. participated in discussion and manuscript preparation.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar13"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par154"><em>Nature Communications</em> thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Stable cell lines and plasmids generated in this study are available from the corresponding authors with a completed material transfer agreement. The protein mass spectrometry data generated in this study have been deposited in PRIDE database with accession numbers <a href="https://www.ebi.ac.uk/pride/archive/projects/PXD054545" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD054545</a> and <a href="https://www.ebi.ac.uk/pride/archive/projects/PXD054601" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD054601</a>. The raw mass spectrometry data were deposited in ProteomeXchange database. All other data supporting the findings of this study are available from the corresponding authors upon request. <a href="#Sec57" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>All MATLAB code for microscopy data analysis, data presentation and stochastic model simulation has been deposited to Code Ocean (access# 7921289 &amp; 5827128).</p></section><section id="FPar14"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par155">The authors claim no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Rui Fang, Luolan Bai.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Rui Fang, Email: fangrui@westlake.edu.cn.</p>
<p>Ying Lu, Email: ying_lu@hms.harvard.edu.</p></section><section id="sec60"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62751-5.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Labbadia, J. &amp; Morimoto, R. I. The biology of proteostasis in aging and disease. <em>Annu. Rev. Biochem.</em><strong>84</strong>, 435–464 (2015).
</cite> [<a href="https://doi.org/10.1146/annurev-biochem-060614-033955" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4539002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25784053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Labbadia,%20J.%20&amp;%20Morimoto,%20R.%20I.%20The%20biology%20of%20proteostasis%20in%20aging%20and%20disease.%20Annu.%20Rev.%20Biochem.84,%20435%E2%80%93464%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Brundin, P., Melki, R. &amp; Kopito, R. Prion-like transmission of protein aggregates in neurodegenerative diseases. <em>Nat. Rev. Mol. Cell Biol.</em><strong>11</strong>, 301–307 (2010).
</cite> [<a href="https://doi.org/10.1038/nrm2873" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2892479/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20308987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brundin,%20P.,%20Melki,%20R.%20&amp;%20Kopito,%20R.%20Prion-like%20transmission%20of%20protein%20aggregates%20in%20neurodegenerative%20diseases.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.11,%20301%E2%80%93307%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Johnston, H. E. &amp; Samant, R. S. Alternative systems for misfolded protein clearance: life beyond the proteasome. <em>FEBS J</em>. 10.1111/febs.15617 (2020).</cite> [<a href="https://doi.org/10.1111/febs.15617" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33135311/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Johnston, J. A., Ward, C. L. &amp; Kopito, R. R. Aggresomes: a cellular response to misfolded proteins. <em>J. Cell Biol.</em><strong>143</strong>, 1883–1898 (1998).
</cite> [<a href="https://doi.org/10.1083/jcb.143.7.1883" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2175217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9864362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johnston,%20J.%20A.,%20Ward,%20C.%20L.%20&amp;%20Kopito,%20R.%20R.%20Aggresomes:%20a%20cellular%20response%20to%20misfolded%20proteins.%20J.%20Cell%20Biol.143,%201883%E2%80%931898%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Olanow, C. W., Perl, D. P., DeMartino, G. N. &amp; McNaught, K. S. P. Lewy-body formation is an aggresome-related process: a hypothesis. <em>Lancet Neurol.</em><strong>3</strong>, 496–503 (2004).
</cite> [<a href="https://doi.org/10.1016/S1474-4422(04)00827-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15261611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Olanow,%20C.%20W.,%20Perl,%20D.%20P.,%20DeMartino,%20G.%20N.%20&amp;%20McNaught,%20K.%20S.%20P.%20Lewy-body%20formation%20is%20an%20aggresome-related%20process:%20a%20hypothesis.%20Lancet%20Neurol.3,%20496%E2%80%93503%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Wileman, T. Aggresomes and autophagy generate sites for virus replication. <em>Science</em><strong>312</strong>, 875–878 (2006).
</cite> [<a href="https://doi.org/10.1126/science.1126766" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16690857/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wileman,%20T.%20Aggresomes%20and%20autophagy%20generate%20sites%20for%20virus%20replication.%20Science312,%20875%E2%80%93878%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Olzmann, J. A., Li, L. &amp; Chin, L. S. Aggresome formation and neurodegenerative diseases: therapeutic implications. <em>Curr. Med. Chem.</em><strong>15</strong>, 47–60 (2008).
</cite> [<a href="https://doi.org/10.2174/092986708783330692" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4403008/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18220762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Olzmann,%20J.%20A.,%20Li,%20L.%20&amp;%20Chin,%20L.%20S.%20Aggresome%20formation%20and%20neurodegenerative%20diseases:%20therapeutic%20implications.%20Curr.%20Med.%20Chem.15,%2047%E2%80%9360%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>French, S. W., Mendoza, A. S. &amp; Peng, Y. The mechanisms of Mallory-Denk body formation are similar to the formation of aggresomes in Alzheimer’s disease and other neurodegenerative disorders. <em>Exp. Mol. Pathol.</em><strong>100</strong>, 426–433 (2016).
</cite> [<a href="https://doi.org/10.1016/j.yexmp.2016.03.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4899135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27068270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?French,%20S.%20W.,%20Mendoza,%20A.%20S.%20&amp;%20Peng,%20Y.%20The%20mechanisms%20of%20Mallory-Denk%20body%20formation%20are%20similar%20to%20the%20formation%20of%20aggresomes%20in%20Alzheimer%E2%80%99s%20disease%20and%20other%20neurodegenerative%20disorders.%20Exp.%20Mol.%20Pathol.100,%20426%E2%80%93433%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Magupalli, V. G. et al. HDAC6 mediates an aggresome-like mechanism for NLRP3 and pyrin inflammasome activation. <em>Science</em><strong>369</strong>, eaas8995 (2020).</cite> [<a href="https://doi.org/10.1126/science.aas8995" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7814939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32943500/" class="usa-link">PubMed</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>García-Mata, R., Bebök, Z., Sorscher, E. J. &amp; Sztul, E. S. Characterization and dynamics of aggresome formation by a cytosolic GFP-chimera. <em>J. Cell Biol.</em><strong>146</strong>, 1239–1254 (1999).
</cite> [<a href="https://doi.org/10.1083/jcb.146.6.1239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2156127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10491388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garc%C3%ADa-Mata,%20R.,%20Beb%C3%B6k,%20Z.,%20Sorscher,%20E.%20J.%20&amp;%20Sztul,%20E.%20S.%20Characterization%20and%20dynamics%20of%20aggresome%20formation%20by%20a%20cytosolic%20GFP-chimera.%20J.%20Cell%20Biol.146,%201239%E2%80%931254%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Gamerdinger, M., Kaya, A. M., Wolfrum, U., Clement, A. M. &amp; Behl, C. BAG3 mediates chaperone-based aggresome-targeting and selective autophagy of misfolded proteins. <em>EMBO Rep.</em><strong>12</strong>, 149–156 (2011).
</cite> [<a href="https://doi.org/10.1038/embor.2010.203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3049430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21252941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gamerdinger,%20M.,%20Kaya,%20A.%20M.,%20Wolfrum,%20U.,%20Clement,%20A.%20M.%20&amp;%20Behl,%20C.%20BAG3%20mediates%20chaperone-based%20aggresome-targeting%20and%20selective%20autophagy%20of%20misfolded%20proteins.%20EMBO%20Rep.12,%20149%E2%80%93156%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Zaarur, N. et al. RuvbL1 and RuvbL2 enhance aggresome formation and disaggregate amyloid fibrils. <em>EMBO J.</em><strong>34</strong>, 2363–2382 (2015).
</cite> [<a href="https://doi.org/10.15252/embj.201591245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4570522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26303906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zaarur,%20N.%20et%20al.%20RuvbL1%20and%20RuvbL2%20enhance%20aggresome%20formation%20and%20disaggregate%20amyloid%20fibrils.%20EMBO%20J.34,%202363%E2%80%932382%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Kawaguchi, Y. et al. The deacetylase HDAC6 regulates aggresome formation and cell viability in response to misfolded protein stress. <em>Cell</em><strong>115</strong>, 727–738 (2003).
</cite> [<a href="https://doi.org/10.1016/s0092-8674(03)00939-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14675537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kawaguchi,%20Y.%20et%20al.%20The%20deacetylase%20HDAC6%20regulates%20aggresome%20formation%20and%20cell%20viability%20in%20response%20to%20misfolded%20protein%20stress.%20Cell115,%20727%E2%80%93738%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Xu, Z. et al. 14-3-3 protein targets misfolded chaperone-associated proteins to aggresomes. <em>J. Cell Sci.</em><strong>126</strong>, 4173–4186 (2013).
</cite> [<a href="https://doi.org/10.1242/jcs.126102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3772389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23843611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20Z.%20et%20al.%2014-3-3%20protein%20targets%20misfolded%20chaperone-associated%20proteins%20to%20aggresomes.%20J.%20Cell%20Sci.126,%204173%E2%80%934186%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Burnett, B. G. &amp; Pittman, R. N. The polyglutamine neurodegenerative protein ataxin 3 regulates aggresome formation. <em>Proc. Natl. Acad. Sci. USA.</em><strong>102</strong>, 4330–4335 (2005).
</cite> [<a href="https://doi.org/10.1073/pnas.0407252102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC555481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15767577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Burnett,%20B.%20G.%20&amp;%20Pittman,%20R.%20N.%20The%20polyglutamine%20neurodegenerative%20protein%20ataxin%203%20regulates%20aggresome%20formation.%20Proc.%20Natl.%20Acad.%20Sci.%20USA.102,%204330%E2%80%934335%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Hubbert, C. et al. HDAC6 is a microtubule-associated deacetylase. <em>Nature</em><strong>417</strong>, 455–458 (2002).
</cite> [<a href="https://doi.org/10.1038/417455a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12024216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hubbert,%20C.%20et%20al.%20HDAC6%20is%20a%20microtubule-associated%20deacetylase.%20Nature417,%20455%E2%80%93458%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Calderilla-Barbosa, L. et al. Interaction of SQSTM1 with the motor protein dynein-SQSTM1 is required for normal dynein function and trafficking. <em>J. Cell Sci.</em><strong>127</strong>, 4052–4063 (2014).
</cite> [<a href="https://doi.org/10.1242/jcs.152363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4163647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25015291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Calderilla-Barbosa,%20L.%20et%20al.%20Interaction%20of%20SQSTM1%20with%20the%20motor%20protein%20dynein-SQSTM1%20is%20required%20for%20normal%20dynein%20function%20and%20trafficking.%20J.%20Cell%20Sci.127,%204052%E2%80%934063%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Park, J. et al. Misfolded polypeptides are selectively recognized and transported toward aggresomes by a CED complex. <em>Nat. Commun.</em><strong>8</strong>, 15730 (2017).
</cite> [<a href="https://doi.org/10.1038/ncomms15730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5467238/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28589942/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Park,%20J.%20et%20al.%20Misfolded%20polypeptides%20are%20selectively%20recognized%20and%20transported%20toward%20aggresomes%20by%20a%20CED%20complex.%20Nat.%20Commun.8,%2015730%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Hwang, H. J. et al. YTHDF2 facilitates aggresome formation via UPF1 in an m6A-independent manner. <em>Nat. Commun</em><strong>14</strong>, 6248 (2023).</cite> [<a href="https://doi.org/10.1038/s41467-023-42015-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10558514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37803021/" class="usa-link">PubMed</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Adriaenssens, E. et al. BAG3 Pro209 mutants associated with myopathy and neuropathy relocate chaperones of the CASA-complex to aggresomes. <em>Sci. Rep.</em><strong>10</strong>, 8755 (2020).
</cite> [<a href="https://doi.org/10.1038/s41598-020-65664-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7260189/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32472079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adriaenssens,%20E.%20et%20al.%20BAG3%20Pro209%20mutants%20associated%20with%20myopathy%20and%20neuropathy%20relocate%20chaperones%20of%20the%20CASA-complex%20to%20aggresomes.%20Sci.%20Rep.10,%208755%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Cianfrocco, M. A., DeSantis, M. E., Leschziner, A. E. &amp; Reck-Peterson, S. L. Mechanism and regulation of cytoplasmic dynein. <em>Annu. Rev. Cell Dev. Biol.</em><strong>31</strong>, 83–108 (2015).
</cite> [<a href="https://doi.org/10.1146/annurev-cellbio-100814-125438" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4644480/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26436706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cianfrocco,%20M.%20A.,%20DeSantis,%20M.%20E.,%20Leschziner,%20A.%20E.%20&amp;%20Reck-Peterson,%20S.%20L.%20Mechanism%20and%20regulation%20of%20cytoplasmic%20dynein.%20Annu.%20Rev.%20Cell%20Dev.%20Biol.31,%2083%E2%80%93108%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Reck-Peterson, S. L., Redwine, W. B., Vale, R. D. &amp; Carter, A. P. The cytoplasmic dynein transport machinery and its many cargoes. <em>Nat. Rev. Mol. Cell Biol.</em><strong>19</strong>, 382–398 (2018).
</cite> [<a href="https://doi.org/10.1038/s41580-018-0004-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6457270/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29662141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reck-Peterson,%20S.%20L.,%20Redwine,%20W.%20B.,%20Vale,%20R.%20D.%20&amp;%20Carter,%20A.%20P.%20The%20cytoplasmic%20dynein%20transport%20machinery%20and%20its%20many%20cargoes.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.19,%20382%E2%80%93398%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Lu, K., den Brave, F. &amp; Jentsch, S. Receptor oligomerization guides pathway choice between proteasomal and autophagic degradation. <em>Nat. Cell Biol.</em><strong>19</strong>, 732–739 (2017).
</cite> [<a href="https://doi.org/10.1038/ncb3531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28504708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20K.,%20den%20Brave,%20F.%20&amp;%20Jentsch,%20S.%20Receptor%20oligomerization%20guides%20pathway%20choice%20between%20proteasomal%20and%20autophagic%20degradation.%20Nat.%20Cell%20Biol.19,%20732%E2%80%93739%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Ouyang, H. et al. Protein aggregates are recruited to aggresome by histone deacetylase 6 via unanchored ubiquitin C termini. <em>J. Biol. Chem.</em><strong>287</strong>, 2317–2327 (2012).
</cite> [<a href="https://doi.org/10.1074/jbc.M111.273730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3268394/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22069321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ouyang,%20H.%20et%20al.%20Protein%20aggregates%20are%20recruited%20to%20aggresome%20by%20histone%20deacetylase%206%20via%20unanchored%20ubiquitin%20C%20termini.%20J.%20Biol.%20Chem.287,%202317%E2%80%932327%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Zaarur, N., Meriin, A. B., Gabai, V. L. &amp; Sherman, M. Y. Triggering aggresome formation: dissecting aggresome-targeting and aggregation signals in synphilin 1. <em>J. Biol. Chem.</em><strong>283</strong>, 27575–27584 (2008).
</cite> [<a href="https://doi.org/10.1074/jbc.M802216200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18635553/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zaarur,%20N.,%20Meriin,%20A.%20B.,%20Gabai,%20V.%20L.%20&amp;%20Sherman,%20M.%20Y.%20Triggering%20aggresome%20formation:%20dissecting%20aggresome-targeting%20and%20aggregation%20signals%20in%20synphilin%201.%20J.%20Biol.%20Chem.283,%2027575%E2%80%9327584%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Shtridelman, Y., Cahyuti, T., Townsend, B., DeWitt, D. &amp; Macosko, J. C. Force-velocity curves of motor proteins cooperating in vivo. <em>Cell Biochem. Biophys.</em><strong>52</strong>, 19–29 (2008).
</cite> [<a href="https://doi.org/10.1007/s12013-008-9021-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2561921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18696014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shtridelman,%20Y.,%20Cahyuti,%20T.,%20Townsend,%20B.,%20DeWitt,%20D.%20&amp;%20Macosko,%20J.%20C.%20Force-velocity%20curves%20of%20motor%20proteins%20cooperating%20in%20vivo.%20Cell%20Biochem.%20Biophys.52,%2019%E2%80%9329%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Klumpp, S. &amp; Lipowsky, R. Cooperative cargo transport by several molecular motors. <em>Proc. Natl. Acad. Sci. USA.</em><strong>102</strong>, 17284–17289 (2005).
</cite> [<a href="https://doi.org/10.1073/pnas.0507363102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1283533/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16287974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Klumpp,%20S.%20&amp;%20Lipowsky,%20R.%20Cooperative%20cargo%20transport%20by%20several%20molecular%20motors.%20Proc.%20Natl.%20Acad.%20Sci.%20USA.102,%2017284%E2%80%9317289%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Fellows, A. D., Bruntraeger, M., Burgold, T., Bassett, A. R. &amp; Carter, A. P. Dynein and dynactin move long-range but are delivered separately to the axon tip. <em>J. Cell Biol</em>. <strong>223</strong>, e202309084 (2024).</cite> [<a href="https://doi.org/10.1083/jcb.202309084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10896695/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38407313/" class="usa-link">PubMed</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Mitchell, C. S. &amp; Lee, R. H. A quantitative examination of the role of cargo-exerted forces in axonal transport. <em>J. Theor. Biol.</em><strong>257</strong>, 430–437 (2009).
</cite> [<a href="https://doi.org/10.1016/j.jtbi.2008.12.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2673572/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19150364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mitchell,%20C.%20S.%20&amp;%20Lee,%20R.%20H.%20A%20quantitative%20examination%20of%20the%20role%20of%20cargo-exerted%20forces%20in%20axonal%20transport.%20J.%20Theor.%20Biol.257,%20430%E2%80%93437%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Palenzuela, H. et al. In vitro reconstitution of dynein force exertion in a bulk viscous medium. <em>Curr. Biol.</em><strong>30</strong>, 4534–4540.e7 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cub.2020.08.078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32946749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palenzuela,%20H.%20et%20al.%20In%20vitro%20reconstitution%20of%20dynein%20force%20exertion%20in%20a%20bulk%20viscous%20medium.%20Curr.%20Biol.30,%204534%E2%80%934540.e7%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Wang, T. et al. Radial contractility of actomyosin rings facilitates axonal trafficking and structural stability. <em>J. Cell Biol</em>. <strong>219</strong>, e201902001 (2020).</cite> [<a href="https://doi.org/10.1083/jcb.201902001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7199852/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32182623/" class="usa-link">PubMed</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Seibenhener, M. L. et al. Sequestosome 1/p62 is a polyubiquitin chain binding protein involved in ubiquitin proteasome degradation. <em>Mol. Cell. Biol.</em><strong>24</strong>, 8055–8068 (2004).
</cite> [<a href="https://doi.org/10.1128/MCB.24.18.8055-8068.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC515032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15340068/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seibenhener,%20M.%20L.%20et%20al.%20Sequestosome%201/p62%20is%20a%20polyubiquitin%20chain%20binding%20protein%20involved%20in%20ubiquitin%20proteasome%20degradation.%20Mol.%20Cell.%20Biol.24,%208055%E2%80%938068%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Hjerpe, R. et al. UBQLN2 mediates autophagy-independent protein aggregate clearance by the proteasome. <em>Cell</em><strong>166</strong>, 935–949 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.07.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5003816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27477512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hjerpe,%20R.%20et%20al.%20UBQLN2%20mediates%20autophagy-independent%20protein%20aggregate%20clearance%20by%20the%20proteasome.%20Cell166,%20935%E2%80%93949%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Field, C. M., Pelletier, J. F. &amp; Mitchison, T. J. Chapter 24 - Xenopus extract approaches to studying microtubule organization and signaling in cytokinesis. in <em>Methods in Cell Biology</em> (ed. Echard, A.) Vol. 137 395–435 (Academic Press, 2017).</cite> [<a href="https://doi.org/10.1016/bs.mcb.2016.04.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5322469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28065319/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Miyazaki, Y. et al. A method to rapidly create protein aggregates in living cells. <em>Nat. Commun.</em><strong>7</strong>, 11689 (2016).
</cite> [<a href="https://doi.org/10.1038/ncomms11689" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4894968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27229621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miyazaki,%20Y.%20et%20al.%20A%20method%20to%20rapidly%20create%20protein%20aggregates%20in%20living%20cells.%20Nat.%20Commun.7,%2011689%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Zhou, M. et al. HUWE1 amplifies ubiquitin modifications to broadly stimulate clearance of proteins and aggregates. <em>bioRxiv</em>10.1101/2023.05.30.542866 (2023).</cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Meriin, A. B. et al. Hsp70-Bag3 complex is a hub for proteotoxicity-induced signaling that controls protein aggregation. <em>Proc. Natl. Acad. Sci. USA.</em><strong>115</strong>, E7043–E7052 (2018).
</cite> [<a href="https://doi.org/10.1073/pnas.1803130115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6064996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29987014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meriin,%20A.%20B.%20et%20al.%20Hsp70-Bag3%20complex%20is%20a%20hub%20for%20proteotoxicity-induced%20signaling%20that%20controls%20protein%20aggregation.%20Proc.%20Natl.%20Acad.%20Sci.%20USA.115,%20E7043%E2%80%93E7052%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Bersuker, K., Brandeis, M. &amp; Kopito, R. R. Protein misfolding specifies recruitment to cytoplasmic inclusion bodies. <em>J. Cell Biol.</em><strong>213</strong>, 229–241 (2016).
</cite> [<a href="https://doi.org/10.1083/jcb.201511024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5084276/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27114501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bersuker,%20K.,%20Brandeis,%20M.%20&amp;%20Kopito,%20R.%20R.%20Protein%20misfolding%20specifies%20recruitment%20to%20cytoplasmic%20inclusion%20bodies.%20J.%20Cell%20Biol.213,%20229%E2%80%93241%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Pelletier, J. F., Field, C. M., Fürthauer, S., Sonnett, M. &amp; Mitchison, T. J. Co-movement of astral microtubules, organelles and F-actin by dynein and actomyosin forces in frog egg cytoplasm. <em>Elife</em><strong>9</strong>, e60047 (2020).</cite> [<a href="https://doi.org/10.7554/eLife.60047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7759381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33284105/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Levi, V., Serpinskaya, A. S., Gratton, E. &amp; Gelfand, V. Organelle transport along microtubules in Xenopus melanophores: evidence for cooperation between multiple motors. <em>Biophys. J.</em><strong>90</strong>, 318–327 (2006).
</cite> [<a href="https://doi.org/10.1529/biophysj.105.067843" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1367030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16214870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Levi,%20V.,%20Serpinskaya,%20A.%20S.,%20Gratton,%20E.%20&amp;%20Gelfand,%20V.%20Organelle%20transport%20along%20microtubules%20in%20Xenopus%20melanophores:%20evidence%20for%20cooperation%20between%20multiple%20motors.%20Biophys.%20J.90,%20318%E2%80%93327%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Mitchison, T. J. &amp; Field, C. M. Self-organization of cellular units. <em>Annu. Rev. Cell Dev. Biol.</em><strong>37</strong>, 23–41 (2021).
</cite> [<a href="https://doi.org/10.1146/annurev-cellbio-120319-025356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9059766/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34186005/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mitchison,%20T.%20J.%20&amp;%20Field,%20C.%20M.%20Self-organization%20of%20cellular%20units.%20Annu.%20Rev.%20Cell%20Dev.%20Biol.37,%2023%E2%80%9341%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Field, C. M. &amp; Mitchison, T. J. Assembly of spindles and asters in Xenopus egg extracts. <em>Cold Spring Harb. Protoc</em><strong>2018</strong>, db.prot099796 (2018).</cite> [<a href="https://doi.org/10.1101/pdb.prot099796" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5984692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29437996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Field,%20C.%20M.%20&amp;%20Mitchison,%20T.%20J.%20Assembly%20of%20spindles%20and%20asters%20in%20Xenopus%20egg%20extracts.%20Cold%20Spring%20Harb.%20Protoc2018,%20db.prot099796%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Lane, J. D. &amp; Allan, V. J. Microtubule-based endoplasmic reticulum motility in Xenopus laevis: activation of membrane-associated kinesin during development. <em>Mol. Biol. Cell</em><strong>10</strong>, 1909–1922 (1999).
</cite> [<a href="https://doi.org/10.1091/mbc.10.6.1909" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC25389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10359605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lane,%20J.%20D.%20&amp;%20Allan,%20V.%20J.%20Microtubule-based%20endoplasmic%20reticulum%20motility%20in%20Xenopus%20laevis:%20activation%20of%20membrane-associated%20kinesin%20during%20development.%20Mol.%20Biol.%20Cell10,%201909%E2%80%931922%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Reck-Peterson, S. L. et al. Single-molecule analysis of dynein processivity and stepping behavior. <em>Cell</em><strong>126</strong>, 335–348 (2006).
</cite> [<a href="https://doi.org/10.1016/j.cell.2006.05.046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2851639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16873064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reck-Peterson,%20S.%20L.%20et%20al.%20Single-molecule%20analysis%20of%20dynein%20processivity%20and%20stepping%20behavior.%20Cell126,%20335%E2%80%93348%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Ishihara, K., Nguyen, P. A., Groen, A. C., Field, C. M. &amp; Mitchison, T. J. Microtubule nucleation remote from centrosomes may explain how asters span large cells. <em>Proc. Natl. Acad. Sci. USA.</em><strong>111</strong>, 17715–17722 (2014).
</cite> [<a href="https://doi.org/10.1073/pnas.1418796111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4273342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25468969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ishihara,%20K.,%20Nguyen,%20P.%20A.,%20Groen,%20A.%20C.,%20Field,%20C.%20M.%20&amp;%20Mitchison,%20T.%20J.%20Microtubule%20nucleation%20remote%20from%20centrosomes%20may%20explain%20how%20asters%20span%20large%20cells.%20Proc.%20Natl.%20Acad.%20Sci.%20USA.111,%2017715%E2%80%9317722%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Ishihara, K. et al. Spatial variation of microtubule depolymerization in large asters. <em>Mol. Biol. Cell</em><strong>32</strong>, 869–879 (2021).
</cite> [<a href="https://doi.org/10.1091/mbc.E20-11-0723" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8108532/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33439671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ishihara,%20K.%20et%20al.%20Spatial%20variation%20of%20microtubule%20depolymerization%20in%20large%20asters.%20Mol.%20Biol.%20Cell32,%20869%E2%80%93879%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Valentine, M. T., Perlman, Z. E., Mitchison, T. J. &amp; Weitz, D. A. Mechanical properties of Xenopus egg cytoplasmic extracts. <em>Biophys. J.</em><strong>88</strong>, 680–689 (2005).
</cite> [<a href="https://doi.org/10.1529/biophysj.104.048025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1305045/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15501931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Valentine,%20M.%20T.,%20Perlman,%20Z.%20E.,%20Mitchison,%20T.%20J.%20&amp;%20Weitz,%20D.%20A.%20Mechanical%20properties%20of%20Xenopus%20egg%20cytoplasmic%20extracts.%20Biophys.%20J.88,%20680%E2%80%93689%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Ross, J. L., Wallace, K., Shuman, H., Goldman, Y. E. &amp; Holzbaur, E. L. F. Processive bidirectional motion of dynein-dynactin complexes in vitro. <em>Nat. Cell Biol.</em><strong>8</strong>, 562–570 (2006).
</cite> [<a href="https://doi.org/10.1038/ncb1421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16715075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ross,%20J.%20L.,%20Wallace,%20K.,%20Shuman,%20H.,%20Goldman,%20Y.%20E.%20&amp;%20Holzbaur,%20E.%20L.%20F.%20Processive%20bidirectional%20motion%20of%20dynein-dynactin%20complexes%20in%20vitro.%20Nat.%20Cell%20Biol.8,%20562%E2%80%93570%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Mallik, R., Petrov, D., Lex, S. A., King, S. J. &amp; Gross, S. P. Building complexity: an in vitro study of cytoplasmic dynein with in vivo implications. <em>Curr. Biol.</em><strong>15</strong>, 2075–2085 (2005).
</cite> [<a href="https://doi.org/10.1016/j.cub.2005.10.039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16332532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mallik,%20R.,%20Petrov,%20D.,%20Lex,%20S.%20A.,%20King,%20S.%20J.%20&amp;%20Gross,%20S.%20P.%20Building%20complexity:%20an%20in%20vitro%20study%20of%20cytoplasmic%20dynein%20with%20in%20vivo%20implications.%20Curr.%20Biol.15,%202075%E2%80%932085%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Gross, S. P., Welte, M. A., Block, S. M. &amp; Wieschaus, E. F. Dynein-mediated cargo transport in vivo. A switch controls travel distance. <em>J. Cell Biol.</em><strong>148</strong>, 945–956 (2000).
</cite> [<a href="https://doi.org/10.1083/jcb.148.5.945" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2174539/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10704445/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gross,%20S.%20P.,%20Welte,%20M.%20A.,%20Block,%20S.%20M.%20&amp;%20Wieschaus,%20E.%20F.%20Dynein-mediated%20cargo%20transport%20in%20vivo.%20A%20switch%20controls%20travel%20distance.%20J.%20Cell%20Biol.148,%20945%E2%80%93956%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Redwine, W. B. et al. The human cytoplasmic dynein interactome reveals novel activators of motility. <em>Elife</em><strong>6</strong>, e28257 (2017).</cite> [<a href="https://doi.org/10.7554/eLife.28257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5533585/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28718761/" class="usa-link">PubMed</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Dekker, C. et al. The interaction network of the chaperonin CCT. <em>EMBO J.</em><strong>27</strong>, 1827–1839 (2008).
</cite> [<a href="https://doi.org/10.1038/emboj.2008.108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2486426/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18511909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dekker,%20C.%20et%20al.%20The%20interaction%20network%20of%20the%20chaperonin%20CCT.%20EMBO%20J.27,%201827%E2%80%931839%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Ju, J.-S., Miller, S. E., Hanson, P. I. &amp; Weihl, C. C. Impaired protein aggregate handling and clearance underlie the pathogenesis of p97/VCP-associated disease. <em>J. Biol. Chem.</em><strong>283</strong>, 30289–30299 (2008).
</cite> [<a href="https://doi.org/10.1074/jbc.M805517200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2573070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18715868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ju,%20J.-S.,%20Miller,%20S.%20E.,%20Hanson,%20P.%20I.%20&amp;%20Weihl,%20C.%20C.%20Impaired%20protein%20aggregate%20handling%20and%20clearance%20underlie%20the%20pathogenesis%20of%20p97/VCP-associated%20disease.%20J.%20Biol.%20Chem.283,%2030289%E2%80%9330299%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Iwata, A., Riley, B. E., Johnston, J. A. &amp; Kopito, R. R. HDAC6 and microtubules are required for autophagic degradation of aggregated huntingtin. <em>J. Biol. Chem.</em><strong>280</strong>, 40282–40292 (2005).
</cite> [<a href="https://doi.org/10.1074/jbc.M508786200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16192271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Iwata,%20A.,%20Riley,%20B.%20E.,%20Johnston,%20J.%20A.%20&amp;%20Kopito,%20R.%20R.%20HDAC6%20and%20microtubules%20are%20required%20for%20autophagic%20degradation%20of%20aggregated%20huntingtin.%20J.%20Biol.%20Chem.280,%2040282%E2%80%9340292%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Bjørkøy, G. et al. p62/SQSTM1 forms protein aggregates degraded by autophagy and has a protective effect on huntingtin-induced cell death. <em>J. Cell Biol.</em><strong>171</strong>, 603–614 (2005).
</cite> [<a href="https://doi.org/10.1083/jcb.200507002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2171557/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16286508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bj%C3%B8rk%C3%B8y,%20G.%20et%20al.%20p62/SQSTM1%20forms%20protein%20aggregates%20degraded%20by%20autophagy%20and%20has%20a%20protective%20effect%20on%20huntingtin-induced%20cell%20death.%20J.%20Cell%20Biol.171,%20603%E2%80%93614%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Tirumala, N. A. et al. Single-molecule imaging of stochastic interactions that drive dynein activation and cargo movement in cells. <em>J. Cell Biol</em>. <strong>223</strong>, e202210026 (2024).</cite> [<a href="https://doi.org/10.1083/jcb.202210026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10798859/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38240798/" class="usa-link">PubMed</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Erbaş, A., Olvera de la Cruz, M. &amp; Marko, J. F. Receptor-ligand rebinding kinetics in confinement. <em>Biophys. J.</em><strong>116</strong>, 1609–1624 (2019).
</cite> [<a href="https://doi.org/10.1016/j.bpj.2019.02.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6506716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31029377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Erba%C5%9F,%20A.,%20Olvera%20de%20la%20Cruz,%20M.%20&amp;%20Marko,%20J.%20F.%20Receptor-ligand%20rebinding%20kinetics%20in%20confinement.%20Biophys.%20J.116,%201609%E2%80%931624%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Kitov, P. I. &amp; Bundle, D. R. On the nature of the multivalency effect: a thermodynamic model. <em>J. Am. Chem. Soc.</em><strong>125</strong>, 16271–16284 (2003).
</cite> [<a href="https://doi.org/10.1021/ja038223n" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14692768/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kitov,%20P.%20I.%20&amp;%20Bundle,%20D.%20R.%20On%20the%20nature%20of%20the%20multivalency%20effect:%20a%20thermodynamic%20model.%20J.%20Am.%20Chem.%20Soc.125,%2016271%E2%80%9316284%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Lu, Y., Lee, B.-H., King, R. W., Finley, D. &amp; Kirschner, M. W. Substrate degradation by the proteasome: a single-molecule kinetic analysis. <em>Science</em><strong>348</strong>, 1250834 (2015).
</cite> [<a href="https://doi.org/10.1126/science.1250834" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4450770/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25859050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20Y.,%20Lee,%20B.-H.,%20King,%20R.%20W.,%20Finley,%20D.%20&amp;%20Kirschner,%20M.%20W.%20Substrate%20degradation%20by%20the%20proteasome:%20a%20single-molecule%20kinetic%20analysis.%20Science348,%201250834%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Hao, R. et al. Proteasomes activate aggresome disassembly and clearance by producing unanchored ubiquitin chains. <em>Mol. Cell</em><strong>51</strong>, 819–828 (2013).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2013.08.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3791850/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24035499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hao,%20R.%20et%20al.%20Proteasomes%20activate%20aggresome%20disassembly%20and%20clearance%20by%20producing%20unanchored%20ubiquitin%20chains.%20Mol.%20Cell51,%20819%E2%80%93828%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>McKenney, R. J., Huynh, W., Tanenbaum, M. E., Bhabha, G. &amp; Vale, R. D. Activation of cytoplasmic dynein motility by dynactin-cargo adapter complexes. <em>Science</em><strong>345</strong>, 337–341 (2014).
</cite> [<a href="https://doi.org/10.1126/science.1254198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4224444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25035494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McKenney,%20R.%20J.,%20Huynh,%20W.,%20Tanenbaum,%20M.%20E.,%20Bhabha,%20G.%20&amp;%20Vale,%20R.%20D.%20Activation%20of%20cytoplasmic%20dynein%20motility%20by%20dynactin-cargo%20adapter%20complexes.%20Science345,%20337%E2%80%93341%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Jha, R. &amp; Surrey, T. Regulation of processive motion and microtubule localization of cytoplasmic dynein. <em>Biochem. Soc. Trans.</em><strong>43</strong>, 48–57 (2015).
</cite> [<a href="https://doi.org/10.1042/BST20140252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25619245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jha,%20R.%20&amp;%20Surrey,%20T.%20Regulation%20of%20processive%20motion%20and%20microtubule%20localization%20of%20cytoplasmic%20dynein.%20Biochem.%20Soc.%20Trans.43,%2048%E2%80%9357%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Culver-Hanlon, T. L., Lex, S. A., Stephens, A. D., Quintyne, N. J. &amp; King, S. J. A microtubule-binding domain in dynactin increases dynein processivity by skating along microtubules. <em>Nat. Cell Biol.</em><strong>8</strong>, 264–270 (2006).
</cite> [<a href="https://doi.org/10.1038/ncb1370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16474384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Culver-Hanlon,%20T.%20L.,%20Lex,%20S.%20A.,%20Stephens,%20A.%20D.,%20Quintyne,%20N.%20J.%20&amp;%20King,%20S.%20J.%20A%20microtubule-binding%20domain%20in%20dynactin%20increases%20dynein%20processivity%20by%20skating%20along%20microtubules.%20Nat.%20Cell%20Biol.8,%20264%E2%80%93270%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Berg, H. C. &amp; Purcell, E. M. Physics of chemoreception. <em>Biophys. J.</em><strong>20</strong>, 193–219 (1977).
</cite> [<a href="https://doi.org/10.1016/S0006-3495(77)85544-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1473391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/911982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berg,%20H.%20C.%20&amp;%20Purcell,%20E.%20M.%20Physics%20of%20chemoreception.%20Biophys.%20J.20,%20193%E2%80%93219%20(1977)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Wang, Z. &amp; Sheetz, M. P. One-dimensional diffusion on microtubules of particles coated with cytoplasmic dynein and immunoglobulins. <em>Cell Struct. Funct.</em><strong>24</strong>, 373–383 (1999).
</cite> [<a href="https://doi.org/10.1247/csf.24.373" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15216895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Z.%20&amp;%20Sheetz,%20M.%20P.%20One-dimensional%20diffusion%20on%20microtubules%20of%20particles%20coated%20with%20cytoplasmic%20dynein%20and%20immunoglobulins.%20Cell%20Struct.%20Funct.24,%20373%E2%80%93383%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Yoo, T. Y. &amp; Mitchison, T. J. O-GlcNAc modification of nuclear pore complexes accelerates bidirectional transport. <em>J. Cell Biol.</em><strong>220</strong>, e202010141 (2021).
</cite> [<a href="https://doi.org/10.1083/jcb.202010141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8091080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33909044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoo,%20T.%20Y.%20&amp;%20Mitchison,%20T.%20J.%20O-GlcNAc%20modification%20of%20nuclear%20pore%20complexes%20accelerates%20bidirectional%20transport.%20J.%20Cell%20Biol.220,%20e202010141%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Spencer, S. L. et al. The proliferation-quiescence decision is controlled by a bifurcation in CDK2 activity at mitotic exit. <em>Cell</em><strong>155</strong>, 369–383 (2013).
</cite> [<a href="https://doi.org/10.1016/j.cell.2013.08.062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4001917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24075009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spencer,%20S.%20L.%20et%20al.%20The%20proliferation-quiescence%20decision%20is%20controlled%20by%20a%20bifurcation%20in%20CDK2%20activity%20at%20mitotic%20exit.%20Cell155,%20369%E2%80%93383%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Egeler, E. L., Urner, L. M., Rakhit, R., Liu, C. W. &amp; Wandless, T. J. Ligand-switchable substrates for a ubiquitin-proteasome system. <em>J. Biol. Chem.</em><strong>286</strong>, 31328–31336 (2011).
</cite> [<a href="https://doi.org/10.1074/jbc.M111.264101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3173072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21768107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Egeler,%20E.%20L.,%20Urner,%20L.%20M.,%20Rakhit,%20R.,%20Liu,%20C.%20W.%20&amp;%20Wandless,%20T.%20J.%20Ligand-switchable%20substrates%20for%20a%20ubiquitin-proteasome%20system.%20J.%20Biol.%20Chem.286,%2031328%E2%80%9331336%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Békés, M. et al. DUB-resistant ubiquitin to survey ubiquitination switches in mammalian cells. <em>Cell Rep.</em><strong>5</strong>, 826–838 (2013).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2013.10.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3889155/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24210823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?B%C3%A9k%C3%A9s,%20M.%20et%20al.%20DUB-resistant%20ubiquitin%20to%20survey%20ubiquitination%20switches%20in%20mammalian%20cells.%20Cell%20Rep.5,%20826%E2%80%93838%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Lu, Y., Wang, W. &amp; Kirschner, M. W. Specificity of the anaphase-promoting complex: a single-molecule study. <em>Science</em><strong>348</strong>, 1248737 (2015).
</cite> [<a href="https://doi.org/10.1126/science.1248737" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4449139/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25859049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20Y.,%20Wang,%20W.%20&amp;%20Kirschner,%20M.%20W.%20Specificity%20of%20the%20anaphase-promoting%20complex:%20a%20single-molecule%20study.%20Science348,%201248737%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Desai, A., Murray, A., Mitchison, T. J. &amp; Walczak, C. E. The use of Xenopus egg extracts to study mitotic spindle assembly and function in vitro. <em>Methods Cell Biol.</em><strong>61</strong>, 385–412 (1999).
</cite> [<a href="https://doi.org/10.1016/s0091-679x(08)61991-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9891325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Desai,%20A.,%20Murray,%20A.,%20Mitchison,%20T.%20J.%20&amp;%20Walczak,%20C.%20E.%20The%20use%20of%20Xenopus%20egg%20extracts%20to%20study%20mitotic%20spindle%20assembly%20and%20function%20in%20vitro.%20Methods%20Cell%20Biol.61,%20385%E2%80%93412%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Guan, Y., Wang, S., Jin, M., Xu, H. &amp; Yang, Q. Reconstitution of cell-cycle oscillations in microemulsions of cell-free Xenopus egg extracts. <em>J. Vis. Exp</em>. 10.3791/58240 (2018).</cite> [<a href="https://doi.org/10.3791/58240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6235322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30320763/" class="usa-link">PubMed</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Hyman, A. et al. Preparation of modified tubulins. <em>Methods Enzymol.</em><strong>196</strong>, 478–485 (1991).
</cite> [<a href="https://doi.org/10.1016/0076-6879(91)96041-o" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2034137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hyman,%20A.%20et%20al.%20Preparation%20of%20modified%20tubulins.%20Methods%20Enzymol.196,%20478%E2%80%93485%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Mooney, P. et al. Tau-based fluorescent protein fusions to visualize microtubules. <em>Cytoskeleton</em><strong>74</strong>, 221–232 (2017).
</cite> [<a href="https://doi.org/10.1002/cm.21368" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5592782/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28407416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mooney,%20P.%20et%20al.%20Tau-based%20fluorescent%20protein%20fusions%20to%20visualize%20microtubules.%20Cytoskeleton74,%20221%E2%80%93232%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Hazel, J. W. &amp; Gatlin, J. C. Isolation and demembranation of Xenopus sperm nuclei. <em>Cold Spring Harb. Protoc.</em><strong>2018</strong>, db.prot099044 (2018).</cite> [<a href="https://doi.org/10.1101/pdb.prot099044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29438000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hazel,%20J.%20W.%20&amp;%20Gatlin,%20J.%20C.%20Isolation%20and%20demembranation%20of%20Xenopus%20sperm%20nuclei.%20Cold%20Spring%20Harb.%20Protoc.2018,%20db.prot099044%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Ishihara, K., Korolev, K. S. &amp; Mitchison, T. J. Physical basis of large microtubule aster growth. <em>Elife</em><strong>5</strong>, e19145 (2016).</cite> [<a href="https://doi.org/10.7554/eLife.19145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5207775/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27892852/" class="usa-link">PubMed</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>King, S. J., Brown, C. L., Maier, K. C., Quintyne, N. J. &amp; Schroer, T. A. Analysis of the dynein-dynactin interaction in vitro and in vivo. <em>Mol. Biol. Cell</em><strong>14</strong>, 5089–5097 (2003).
</cite> [<a href="https://doi.org/10.1091/mbc.E03-01-0025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC284810/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14565986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?King,%20S.%20J.,%20Brown,%20C.%20L.,%20Maier,%20K.%20C.,%20Quintyne,%20N.%20J.%20&amp;%20Schroer,%20T.%20A.%20Analysis%20of%20the%20dynein-dynactin%20interaction%20in%20vitro%20and%20in%20vivo.%20Mol.%20Biol.%20Cell14,%205089%E2%80%935097%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Beausoleil, S. A., Villén, J., Gerber, S. A., Rush, J. &amp; Gygi, S. P. A probability-based approach for high-throughput protein phosphorylation analysis and site localization. <em>Nat. Biotechnol.</em><strong>24</strong>, 1285–1292 (2006).
</cite> [<a href="https://doi.org/10.1038/nbt1240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16964243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beausoleil,%20S.%20A.,%20Vill%C3%A9n,%20J.,%20Gerber,%20S.%20A.,%20Rush,%20J.%20&amp;%20Gygi,%20S.%20P.%20A%20probability-based%20approach%20for%20high-throughput%20protein%20phosphorylation%20analysis%20and%20site%20localization.%20Nat.%20Biotechnol.24,%201285%E2%80%931292%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Huttlin, E. L. et al. A tissue-specific atlas of mouse protein phosphorylation and expression. <em>Cell</em><strong>143</strong>, 1174–1189 (2010).
</cite> [<a href="https://doi.org/10.1016/j.cell.2010.12.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3035969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21183079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huttlin,%20E.%20L.%20et%20al.%20A%20tissue-specific%20atlas%20of%20mouse%20protein%20phosphorylation%20and%20expression.%20Cell143,%201174%E2%80%931189%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Elias, J. E. &amp; Gygi, S. P. Target-decoy search strategy for mass spectrometry-based proteomics. <em>Methods Mol. Biol.</em><strong>604</strong>, 55–71 (2010).
</cite> [<a href="https://doi.org/10.1007/978-1-60761-444-9_5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2922680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20013364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elias,%20J.%20E.%20&amp;%20Gygi,%20S.%20P.%20Target-decoy%20search%20strategy%20for%20mass%20spectrometry-based%20proteomics.%20Methods%20Mol.%20Biol.604,%2055%E2%80%9371%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Elias, J. E. &amp; Gygi, S. P. Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry. <em>Nat. Methods</em><strong>4</strong>, 207–214 (2007).
</cite> [<a href="https://doi.org/10.1038/nmeth1019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17327847/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elias,%20J.%20E.%20&amp;%20Gygi,%20S.%20P.%20Target-decoy%20search%20strategy%20for%20increased%20confidence%20in%20large-scale%20protein%20identifications%20by%20mass%20spectrometry.%20Nat.%20Methods4,%20207%E2%80%93214%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>McAlister, G. C. et al. Increasing the multiplexing capacity of TMTs using reporter ion isotopologues with isobaric masses. <em>Anal. Chem.</em><strong>84</strong>, 7469–7478 (2012).
</cite> [<a href="https://doi.org/10.1021/ac301572t" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3715028/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22880955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McAlister,%20G.%20C.%20et%20al.%20Increasing%20the%20multiplexing%20capacity%20of%20TMTs%20using%20reporter%20ion%20isotopologues%20with%20isobaric%20masses.%20Anal.%20Chem.84,%207469%E2%80%937478%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Ashburner, M. et al. Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. <em>Nat. Genet.</em><strong>25</strong>, 25–29 (2000).
</cite> [<a href="https://doi.org/10.1038/75556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3037419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10802651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ashburner,%20M.%20et%20al.%20Gene%20ontology:%20tool%20for%20the%20unification%20of%20biology.%20The%20Gene%20Ontology%20Consortium.%20Nat.%20Genet.25,%2025%E2%80%9329%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Gene Ontology Consortium et al. The Gene Ontology knowledgebase in 2023. <em>Genetics</em><strong>224</strong>, iyad031 (2023).</cite> [<a href="https://doi.org/10.1093/genetics/iyad031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10158837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36866529/" class="usa-link">PubMed</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Ge, S. X., Jung, D. &amp; Yao, R. ShinyGO: a graphical gene-set enrichment tool for animals and plants. <em>Bioinformatics</em><strong>36</strong>, 2628–2629 (2020).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7178415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31882993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ge,%20S.%20X.,%20Jung,%20D.%20&amp;%20Yao,%20R.%20ShinyGO:%20a%20graphical%20gene-set%20enrichment%20tool%20for%20animals%20and%20plants.%20Bioinformatics36,%202628%E2%80%932629%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>SLM - Shape Language Modeling - File Exchange - MATLAB CentralFile Exchange - MATLAB Central. <a href="https://www.mathworks.com/matlabcentral/fileexchange/24443-slm-shape-language-modeling" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.mathworks.com/matlabcentral/fileexchange/24443-slm-shape-language-modeling</a> (2017).</cite>
</li>
<li id="CR87">
<span class="label">87.</span><cite>Violin Plot - File Exchange - MATLAB CentralFile Exchange - MATLAB Central. <a href="https://www.mathworks.com/matlabcentral/fileexchange/45134-violin-plot" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.mathworks.com/matlabcentral/fileexchange/45134-violin-plot</a> (2015).</cite>
</li>
<li id="CR88">
<span class="label">88.</span><cite>Stokes, G. G. On the effect of the internal friction of fluids on the motion of pendulums. in <em>Mathematical and Physical Papers</em> 1–10 (Cambridge University Press, 2009).</cite>
</li>
<li id="CR89">
<span class="label">89.</span><cite>Erickson, R. P., Jia, Z., Gross, S. P. &amp; Yu, C. C. How molecular motors are arranged on a cargo is important for vesicular transport. <em>PLoS Comput. Biol.</em><strong>7</strong>, e1002032 (2011).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1002032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3088656/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21573204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Erickson,%20R.%20P.,%20Jia,%20Z.,%20Gross,%20S.%20P.%20&amp;%20Yu,%20C.%20C.%20How%20molecular%20motors%20are%20arranged%20on%20a%20cargo%20is%20important%20for%20vesicular%20transport.%20PLoS%20Comput.%20Biol.7,%20e1002032%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (9.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62751_MOESM2_ESM.pdf</a><sup> (173.8KB, pdf) </sup><p>Description of Additional Supplementary File</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM3_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (9.6MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM4_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 2</a><sup> (21.5MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM5_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 3</a><sup> (731.9KB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM6_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 4</a><sup> (1.3MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM7_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 5</a><sup> (831.6KB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM8_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 6</a><sup> (1.3MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM9_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 7</a><sup> (2MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM10_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 8</a><sup> (1.3MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM11_ESM.avi" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 9</a><sup> (12MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM12_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 1</a><sup> (9.3MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM13_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 2</a><sup> (45.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material14_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM14_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting summary</a><sup> (2.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material15_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM15_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (24.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material16_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373839/bin/41467_2025_62751_MOESM16_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (12.5MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Stable cell lines and plasmids generated in this study are available from the corresponding authors with a completed material transfer agreement. The protein mass spectrometry data generated in this study have been deposited in PRIDE database with accession numbers <a href="https://www.ebi.ac.uk/pride/archive/projects/PXD054545" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD054545</a> and <a href="https://www.ebi.ac.uk/pride/archive/projects/PXD054601" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD054601</a>. The raw mass spectrometry data were deposited in ProteomeXchange database. All other data supporting the findings of this study are available from the corresponding authors upon request. <a href="#Sec57" class="usa-link">Source data</a> are provided with this paper.</p>
<p>All MATLAB code for microscopy data analysis, data presentation and stochastic model simulation has been deposited to Code Ocean (access# 7921289 &amp; 5827128).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62751-5"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62751.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373839/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373839/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373839%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373839/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373839/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373839/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846843/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373839/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846843/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373839/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373839/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="YN6qEB1CyR2GbjWcwpBYzpBtIyTX2M5PNUN3LQb520OBCHeFhJ4khHDxcy3026Sm">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
