
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Universal amplification and sequencing of foot-and-mouth disease virus complete genomes using nanopore technology - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532C3B8AF1ECE3052C3B0020F1E94E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcgeno">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372193/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Genomics">
<meta name="citation_title" content="Universal amplification and sequencing of foot-and-mouth disease virus complete genomes using nanopore technology">
<meta name="citation_author" content="Andrew E Shaw">
<meta name="citation_author_institution" content="The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK">
<meta name="citation_author" content="Kebaneilwe Lebani">
<meta name="citation_author_institution" content="The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK">
<meta name="citation_author_institution" content="School of Life Sciences, Botswana International University of Science and Technology, Palapye, Botswana">
<meta name="citation_author" content="Lina González Gordon">
<meta name="citation_author_institution" content="Division of Epidemiology, Roslin Institute and Royal (Dick) School of Veterinary Studies, The University of Edinburgh, Easter Bush Campus, Midlothian, EH25 9RG UK">
<meta name="citation_author" content="Ugonna E Ihearahu">
<meta name="citation_author_institution" content="The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK">
<meta name="citation_author" content="Jemma Wadsworth">
<meta name="citation_author_institution" content="The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK">
<meta name="citation_author" content="Hayley M Hicks">
<meta name="citation_author_institution" content="The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK">
<meta name="citation_author" content="Noemi Polo">
<meta name="citation_author_institution" content="The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK">
<meta name="citation_author" content="Graham Freimanis">
<meta name="citation_author_institution" content="The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK">
<meta name="citation_author" content="Dennis Muhanguzi">
<meta name="citation_author_institution" content="Makerere University, 7062 University Rd, Kampala, Uganda">
<meta name="citation_author" content="Chandana Tennakoon">
<meta name="citation_author_institution" content="The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK">
<meta name="citation_author" content="Richard J Orton">
<meta name="citation_author_institution" content="MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Garscube Campus, Glasgow, G61 1QH United Kingdom">
<meta name="citation_author" content="Nick J Knowles">
<meta name="citation_author_institution" content="The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK">
<meta name="citation_author" content="Antonello Di Nardo">
<meta name="citation_author_institution" content="The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK">
<meta name="citation_author" content="Ryan A Waters">
<meta name="citation_author_institution" content="The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK">
<meta name="citation_author" content="Barend MdeC Bronsvoort">
<meta name="citation_author_institution" content="Division of Epidemiology, Roslin Institute and Royal (Dick) School of Veterinary Studies, The University of Edinburgh, Easter Bush Campus, Midlothian, EH25 9RG UK">
<meta name="citation_author" content="Donald P King">
<meta name="citation_author_institution" content="The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="770">
<meta name="citation_doi" content="10.1186/s12864-025-11938-7">
<meta name="citation_pmid" content="40846908">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372193/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372193/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372193/pdf/12864_2025_Article_11938.pdf">
<meta name="description" content="Foot-and-mouth disease virus (FMDV) is capable of causing explosive outbreaks among domestic and wild cloven-hoofed animals. Genomic characterisation of FMDV is a crucial component of disease control enabling accurate tracing of disease outbreaks to ...">
<meta name="og:title" content="Universal amplification and sequencing of foot-and-mouth disease virus complete genomes using nanopore technology">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Foot-and-mouth disease virus (FMDV) is capable of causing explosive outbreaks among domestic and wild cloven-hoofed animals. Genomic characterisation of FMDV is a crucial component of disease control enabling accurate tracing of disease outbreaks to ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372193/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12372193">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12864-025-11938-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12864_2025_Article_11938.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372193%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12372193/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12372193/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372193/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcgeno.png" alt="BMC Genomics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Genomics" title="Link to BMC Genomics" shape="default" href="https://bmcgenomics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Genomics</button></div>. 2025 Aug 22;26:770. doi: <a href="https://doi.org/10.1186/s12864-025-11938-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12864-025-11938-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Genomics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Genomics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Universal amplification and sequencing of foot-and-mouth disease virus complete genomes using nanopore technology</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shaw%20AE%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Andrew E Shaw</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Andrew E Shaw</span></h3>
<div class="p">
<sup>1</sup>The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shaw%20AE%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew E Shaw</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lebani%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Kebaneilwe Lebani</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Kebaneilwe Lebani</span></h3>
<div class="p">
<sup>1</sup>The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK </div>
<div class="p">
<sup>2</sup>School of Life Sciences, Botswana International University of Science and Technology, Palapye, Botswana </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lebani%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kebaneilwe Lebani</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gonz%C3%A1lez%20Gordon%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Lina González Gordon</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Lina González Gordon</span></h3>
<div class="p">
<sup>3</sup>Division of Epidemiology, Roslin Institute and Royal (Dick) School of Veterinary Studies, The University of Edinburgh, Easter Bush Campus, Midlothian, EH25 9RG UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gonz%C3%A1lez%20Gordon%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lina González Gordon</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ihearahu%20UE%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ugonna E Ihearahu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ugonna E Ihearahu</span></h3>
<div class="p">
<sup>1</sup>The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ihearahu%20UE%22%5BAuthor%5D" class="usa-link"><span class="name western">Ugonna E Ihearahu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wadsworth%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Jemma Wadsworth</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jemma Wadsworth</span></h3>
<div class="p">
<sup>1</sup>The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wadsworth%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jemma Wadsworth</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hicks%20HM%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Hayley M Hicks</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Hayley M Hicks</span></h3>
<div class="p">
<sup>1</sup>The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hicks%20HM%22%5BAuthor%5D" class="usa-link"><span class="name western">Hayley M Hicks</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Polo%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Noemi Polo</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Noemi Polo</span></h3>
<div class="p">
<sup>1</sup>The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Polo%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Noemi Polo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Freimanis%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Graham Freimanis</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Graham Freimanis</span></h3>
<div class="p">
<sup>1</sup>The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Freimanis%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Graham Freimanis</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Muhanguzi%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Dennis Muhanguzi</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Dennis Muhanguzi</span></h3>
<div class="p">
<sup>4</sup>Makerere University, 7062 University Rd, Kampala, Uganda </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Muhanguzi%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dennis Muhanguzi</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tennakoon%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Chandana Tennakoon</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Chandana Tennakoon</span></h3>
<div class="p">
<sup>1</sup>The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tennakoon%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chandana Tennakoon</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Orton%20RJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Richard J Orton</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Richard J Orton</span></h3>
<div class="p">
<sup>5</sup>MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Garscube Campus, Glasgow, G61 1QH United Kingdom </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Orton%20RJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Richard J Orton</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Knowles%20NJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Nick J Knowles</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Nick J Knowles</span></h3>
<div class="p">
<sup>1</sup>The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Knowles%20NJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Nick J Knowles</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Di%20Nardo%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Antonello Di Nardo</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Antonello Di Nardo</span></h3>
<div class="p">
<sup>1</sup>The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Di%20Nardo%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Antonello Di Nardo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Waters%20RA%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Ryan A Waters</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Ryan A Waters</span></h3>
<div class="p">
<sup>1</sup>The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Waters%20RA%22%5BAuthor%5D" class="usa-link"><span class="name western">Ryan A Waters</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bronsvoort%20BM%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Barend MdeC Bronsvoort</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Barend MdeC Bronsvoort</span></h3>
<div class="p">
<sup>3</sup>Division of Epidemiology, Roslin Institute and Royal (Dick) School of Veterinary Studies, The University of Edinburgh, Easter Bush Campus, Midlothian, EH25 9RG UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bronsvoort%20BM%22%5BAuthor%5D" class="usa-link"><span class="name western">Barend MdeC Bronsvoort</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22King%20DP%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Donald P King</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Donald P King</span></h3>
<div class="p">
<sup>1</sup>The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22King%20DP%22%5BAuthor%5D" class="usa-link"><span class="name western">Donald P King</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>The Pirbright Institute, Ash Road, Pirbright, Surrey GU24 0NF UK </div>
<div id="Aff2">
<sup>2</sup>School of Life Sciences, Botswana International University of Science and Technology, Palapye, Botswana </div>
<div id="Aff3">
<sup>3</sup>Division of Epidemiology, Roslin Institute and Royal (Dick) School of Veterinary Studies, The University of Edinburgh, Easter Bush Campus, Midlothian, EH25 9RG UK </div>
<div id="Aff4">
<sup>4</sup>Makerere University, 7062 University Rd, Kampala, Uganda </div>
<div id="Aff5">
<sup>5</sup>MRC-University of Glasgow Centre for Virus Research, University of Glasgow, Garscube Campus, Glasgow, G61 1QH United Kingdom </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 23; Accepted 2025 Jul 23; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12372193  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846908/" class="usa-link">40846908</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Foot-and-mouth disease virus (FMDV) is capable of causing explosive outbreaks among domestic and wild cloven-hoofed animals. Genomic characterisation of FMDV is a crucial component of disease control enabling accurate tracing of disease outbreaks to be undertaken. Nanopore sequencing is an affordable and accessible form of high-throughput sequencing (HTS) technology. However, most published methods for FMDV only sequence genomic fragments or focus upon specific lineages. In this study, a universal FMDV sequencing protocol was developed alongside a bespoke analytical pipeline to sequence any FMDV genome in the absence of prior knowledge regarding the identity of the serotype or lineage.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par2">Universal multiplex RT-PCRs were used to amplify overlapping tiles encompassing the entire FMDV genome. The PCR products were pooled and subjected to nanopore sequencing using the portable MinION sequencing device. A bioinformatics pipeline was used to assemble genomes based upon blastn and reference assembly.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">Iterative changes in primer design and pooling resulted in two panels of primers; one set amplifying twenty short fragments (S_scheme), and another set amplifying six longer fragments (L_scheme). Both approaches were shown to be capable of generating FMDV genomes, however the L_scheme was simpler, more reliable and more cost-effective at generating complete genomes. The final L_scheme protocol was assessed using 30 FMDV isolates representing all the currently circulating lineages of FMDV. As part of the development, we successfully trialled the use of this technology in Uganda, a country endemic for FMD.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par4">The amplification, sequencing and bioinformatics strategy developed here has been assessed using a diverse array of FMDV lineages. Using two multiplex PCR reactions, this approach can successfully generate complete genomes of FMDV in a lineage agnostic fashion. Therefore, the primer sets and approaches described here represent a useful tool for expanding the capacity of laboratories to characterise FMDV at the genomic level.</p></section><section id="sec5"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12864-025-11938-7.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Foot-and-mouth disease, Nanopore, Sequencing, MinION, Whole genome sequencing, Complete genome</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par5">Foot-and-mouth disease (FMD) is one of the most economically consequential viral diseases of livestock [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. The disease is caused by FMD virus (FMDV), species <em>Aphthovirus vesiculae</em>, within the family <em>Picornaviridae</em>. FMDV is highly contagious causing vesicular lesions in the mouth and on the feet of wild and domesticated cloven-hoofed animals including cattle, sheep, goats, and pigs. Whilst mortality is usually low among adult animals, FMDV infection can result in severe production losses.</p>
<p id="Par6">FMDV exists as seven antigenically distinct serotypes. Serotypes O, A, Asia 1 and Southern African Territories (SAT) 1, 2 and 3 all circulate in endemic regions. In contrast, serotype C has not been detected anywhere since 2004 and is considered to be extinct [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. The epidemiology of FMDV is complex and is underpinned by endemic circulation of different virus serotypes in seven global ‘pools’ (pool 1 – pool 7), each of which maintains a unique constellation of circulating lineages [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Unpredictable inter-pool movements of FMDVs further complicate the epidemiological situation [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>] since the rapid spread of novel serotypes and lineages within a region can be facilitated by the lack of existing immunity in susceptible hosts derived from previous infection and/or vaccination. Indeed, the dynamic nature of FMD epidemiology has recently been exemplified by outbreaks in formerly FMD-free regions, for example recent outbreaks in Europe and Indonesia [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. These factors reinforce the importance of work undertaken to continually characterise FMD viruses collected from field cases of FMD to monitor the regional circulation of FMDV lineages and the emergence of new antigenic variants.</p>
<p id="Par7">Genetically, most serotypes encompass multiple ‘topotypes’, which comprise viral lineages (and sub-lineages) that are specific to the global regions in which they are found [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. The genetic diversity of FMDV is further heightened by the high error rate associated with the viral RNA dependent RNA polymerase [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Consequently, generating molecular assays to characterise FMDV at the lineage, serotype or species level can be extremely challenging.</p>
<p id="Par8">Differential lineage detection and characterisation can be achieved using lineage-specific real-time RT-PCR assays that have been developed for a broad range of FMDV lineages [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. However, these tests do not provide data that can be used to define the evolutionary origin of FMDVs or the precise genotype of viruses. The VP1-encoding 1D region of the FMDV genome is favoured for phylogenetic trees as its nucleotide variability and lack of recombination allow the robust reconstruction of evolutionary relationships. The generation of complete genomes as opposed to routine VP1-coding sequences offers multiple advantages, for example the identification of features within non-structural proteins which may relate to other facets of viral fitness, pathogenicity, host specificity [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>] or transmission [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Furthermore, the sequencing of complete RNA virus genomes can provide important data for disease control to allow transmission pathways to be resolved, with fine scale outbreak tracing applied to FMDV as well as multiple other viruses, including Ebolavirus, yellow fever virus, Zika virus, influenza virus and SARS-CoV-2 [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>–<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>].</p>
<p id="Par9">Historically, Sanger as well as short-read Illumina-based sequencing have been the dominant platforms used to sequence FMDV genomes [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. However, Illumina and Sanger sequencing technologies both suffer from the requirement for expensive hardware and dedicated infrastructure. Nanopore-based sequencing has been demonstrated as a suitable method which can be used to partially characterise FMDV [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>–<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Although direct sequencing of viral RNA has been reported (including for FMDV) [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>], the majority of protocols that use nanopore chemistry to sequence viruses amplify fragments of the genome using PCR as a form of enrichment above the background nucleic acid, e.g. host DNA [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. In contrast to approaches amplifying targeted regions of the genome, many protocols use multiplex PCR to amplify overlapping tiles encompassing the genome [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>, <a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>, <a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. Increasingly, nanopore technology is being used as the HTS method of choice to sequence amplified PCR products due to being economical, quick, and its ease of deployment [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>, <a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Here, we report the development of a universal approach to amplify and sequence complete FMDV genomes using nanopore technology.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Primer evaluation</h3>
<p id="Par10">The lack of <em>a priori</em> knowledge regarding the serotype/topotype of a FMDV positive sample means that an approach able to accommodate the wide range of phylogenetically distinct topotypes/lineages that could be present is required. We therefore designed PCR primer sets capable of amplifying highly diverse FMD genomes. Experiments to iteratively design the primer schemes were performed using a selection of virus isolates archived at the FAO World Reference Laboratory for FMD (WRLFMD), Pirbright, Table <a href="#Tab1" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>FMDV isolates used to evaluate primer combinations for the amplification of complete genomes</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Isolate</th>
<th align="left" colspan="1" rowspan="1">Serotype</th>
<th align="left" colspan="1" rowspan="1">Topotype</th>
<th align="left" colspan="1" rowspan="1">Lineage</th>
<th align="left" colspan="1" rowspan="1">Pool<sup>1</sup>
</th>
<th align="left" colspan="1" rowspan="1">Panel</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">VIT/1/2017</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">SEA</td>
<td align="left" colspan="1" rowspan="1">Mya-98</td>
<td align="center" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">SEA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">VIT/1/2018</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">ME-SA</td>
<td align="left" colspan="1" rowspan="1">PanAsia</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">SEA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">VIT/11/2017</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">ASIA</td>
<td align="left" colspan="1" rowspan="1">Sea-97</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">SEA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KEN/4/2018</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">EA-2</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ETH/4/2015</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">EA-3</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ETH/9/2019</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">EA-3</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ETH/30/2016</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">EA-4</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ETH/14/2019</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">EA-4</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ETH/2/2018</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">AFRICA</td>
<td align="left" colspan="1" rowspan="1">G-I</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UGA/28/2019</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">AFRICA</td>
<td align="left" colspan="1" rowspan="1">G-I</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SUD/9/2018</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">AFRICA</td>
<td align="left" colspan="1" rowspan="1">G-IV</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ETH/19/2019</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">AFRICA</td>
<td align="left" colspan="1" rowspan="1">G-IV</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TAN/27/2012</td>
<td align="left" colspan="1" rowspan="1">SAT1</td>
<td align="left" colspan="1" rowspan="1">I</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TAN/22/2013</td>
<td align="left" colspan="1" rowspan="1">SAT1</td>
<td align="left" colspan="1" rowspan="1">I</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KEN/10/2013</td>
<td align="left" colspan="1" rowspan="1">SAT1</td>
<td align="left" colspan="1" rowspan="1">I</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TAN/22/2014</td>
<td align="left" colspan="1" rowspan="1">SAT1</td>
<td align="left" colspan="1" rowspan="1">I</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KEN/19/2017</td>
<td align="left" colspan="1" rowspan="1">SAT2</td>
<td align="left" colspan="1" rowspan="1">IV</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ETH/16/2015</td>
<td align="left" colspan="1" rowspan="1">SAT2</td>
<td align="left" colspan="1" rowspan="1">VII</td>
<td align="left" colspan="1" rowspan="1">Alx-12</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">EGY/1/2018</td>
<td align="left" colspan="1" rowspan="1">SAT2</td>
<td align="left" colspan="1" rowspan="1">VII</td>
<td align="left" colspan="1" rowspan="1">Ghb-12</td>
<td align="center" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ETH/11/2018</td>
<td align="left" colspan="1" rowspan="1">SAT2</td>
<td align="left" colspan="1" rowspan="1">VII</td>
<td align="left" colspan="1" rowspan="1">Lib-12</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">EA</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p14">
<p><em>SEA</em> refers to virus isolates from Southeast Asia, <em>EA</em> refers to virus isolates from East Africa</p>
<p><sup>1‘</sup>Pool’ refers to the epidemiological pools where the lineage is observed [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]</p>
</div></div></section><p id="Par12">We first focused upon evaluating a primer scheme comprising 20 overlapping tiles (on average 572 bp in length) – termed the S_scheme - using the East African epidemiological pool as it contains a diverse array of serotypes and lineages, including serotypes O (4 lineages recorded), A (3 lineages recorded), SAT1 (4 lineages recorded) and SAT2 (5 lineages recorded) and thus represented a challenging scenario against which to evaluate the primers. We further evaluated the performance of the S_scheme by testing representative FMDV isolates from East and Southeast Asia. Recent isolates of viruses representing three key topotypes currently circulating within Southeast Asia (O/SEA/Mya-98, O/ME-SA/PanAsia and A/ASIA/Sea-97) were selected to assess the suitability of the primer scheme for Pool 1 viruses (Table <a href="#Tab1" class="usa-link">1</a>). As expected, the majority of amplification failures when using initial primer designs were associated with primers that targeted the highly variable P1 region (Fig. <a href="#Fig1" class="usa-link">1</a>A, Additional file 1 [Table S1]). Failed PCR reactions within the capsid coding region were particularly evident for serotype SAT1 where 13/24 PCRs failed across 4 viruses (Additional file 1 Table S1). Similar results were obtained for the Southeast Asian isolates, where the amplicon 6 PCR failed for A/VIT/11/2017 (Additional file 2 Table S2). In contrast, PCRs using primers located in P2 and P3 robustly amplified the target fragments. Iterative re-designs of faulty primer pairs resulted in universal amplification across the test panel of isolates.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372193_12864_2025_11938_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/679f/12372193/4956a0515c6d/12864_2025_11938_Fig1_HTML.jpg" loading="lazy" id="d33e802" height="183" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The FMDV genome. (<strong>A</strong>) A scaled representation of the FMDV region. The polyprotein is divided into P1, P2 and P3 and flanked by 5’ and 3’ untranslated regions (UTRs). The capsid coding regions 1A-1D are shaded blue. A poly cytosine (poly(C), (C<sub>n</sub>)) region separates the S fragment from the L fragment, and the 3’UTR is followed by a poly(A) tail (A<sub>n</sub>). (<strong>B</strong>) The six amplicons in the finalised L_scheme. One amplicon ~ 380 bp represents the S fragment, with five others covering the L fragment. <sup>1</sup>Amplicon length depends upon the serotype being amplified</p></figcaption></figure><p id="Par14">In addition to the S_scheme, a final arrangement of larger amplicons – termed the L_scheme - comprised one amplicon representing the S fragment and a further five overlapping amplicons encompassing the L fragment (Fig. <a href="#Fig1" class="usa-link">1</a>B; Table <a href="#Tab2" class="usa-link">2</a>). Primer combinations for the L_scheme were selected based upon the wider S_scheme testing but were also empirically tested to confirm performance (Fig. <a href="#Fig2" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>L_scheme primer composition of pools A and B for the universal amplification of FMDV</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Pool</th>
<th align="left" colspan="1" rowspan="1">Primer</th>
<th align="left" colspan="1" rowspan="1">Target serotype(s)</th>
<th align="left" colspan="1" rowspan="1">Sequence 5’−3’</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="16" colspan="1">A</td>
<td align="left" colspan="1" rowspan="1">UNI_For_1.1.1</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">GGCCACGCGTCGACTAGTACCCCCCCCCCCCCCCCCCCYAAR</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UNI_For_1.3.1</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">GGCCACGCGTCGACTAGTACCCCCCCCCCCCCCCCCCCRYN</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UNI_For_1.4.1</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">GGCCACGCGTCGACTAGTACCCCCCCCCCCCCCCCCCCRRN</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UNI_For_1.5.1</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">GGCCACGCGTCGACTAGTACCCCCCCCCCCCCCCCCCCTYW</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UNI_For_1.6.1</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">GGCCACGCGTCGACTAGTACCCCCCCCCCCCCCCCCCCTRN</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">O_UNI_For_6</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">CGGACGAACATGACRGCVCACAT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A_UNI_For_6.1</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">AGAACMAACATGACWGCVCACAT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">As1_UNI_For_6</td>
<td align="left" colspan="1" rowspan="1">Asia 1</td>
<td align="left" colspan="1" rowspan="1">CGCACYAACATGACGGCBCACAT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SAT1_UNI_For_6A</td>
<td align="left" colspan="1" rowspan="1">SAT1</td>
<td align="left" colspan="1" rowspan="1">ATRAACCCGCGYACCAACACCA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SAT1_UNI_For_6B</td>
<td align="left" colspan="1" rowspan="1">SAT1</td>
<td align="left" colspan="1" rowspan="1">ACCAATTCCTCAAYCCRMGRACGAACAC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SAT2_UNI_For_6.1</td>
<td align="left" colspan="1" rowspan="1">SAT2,3</td>
<td align="left" colspan="1" rowspan="1">TACCCRCACCAGTTCATYAACCC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UNI_For_15.1</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">GAAGAARCCTGTCGCYTTGAARGTGA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UNI_Rev_2.3</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">ACACCTCRCTGGGYCGYGARGC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UNI_Rev_2.4</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">CAGATCTCGCTRGGGCGAGT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UNI_Rev_9</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">GCYACAGCGGCCATRCAYGACA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FMD_UNI_RT</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">TACAACGCCTGTAGCATTCCTTTTTTTTTTTTTTTTTTTTGRWWW</td>
</tr>
<tr>
<td align="left" rowspan="12" colspan="1">B</td>
<td align="left" colspan="1" rowspan="1">UNI_For_2</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">CRTGTGTGCRACCCCRGCAC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UNI_For_10</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">CCAACCCTGGRCCCTTCTTYTT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Sfrag_F.1</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">TTGAAAGGGGGYRYTAGGG</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">O_A_UNI_Rev_5</td>
<td align="left" colspan="1" rowspan="1">O, A</td>
<td align="left" colspan="1" rowspan="1">GGGGCGATGTTGGCRTAVACCTTRAT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">As1_UNI_Rev_5.2</td>
<td align="left" colspan="1" rowspan="1">Asia 1</td>
<td align="left" colspan="1" rowspan="1">GGTGCTGCATTCATGTARACCTTRAT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SAT1_UNI_Rev_5A</td>
<td align="left" colspan="1" rowspan="1">SAT1,3</td>
<td align="left" colspan="1" rowspan="1">GTYTTGGGGTCHGTGTTTTGGAA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SAT1_UNI_Rev_5A.1</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">AAGTCTTCGGGTCGGTGTTTTGG</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SAT2_UNI_Rev_5</td>
<td align="left" colspan="1" rowspan="1">SAT2,3</td>
<td align="left" colspan="1" rowspan="1">GCRATGTTGGCGTACACCTCRAC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UNI_Rev_14A</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">CAGCAGATGGCYACYGTCTTVCC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UNI_Rev_14B</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">CAGCAGAGYGCAACBGTCTTMCC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UNI_Rev_14C</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">CAGCAGAKYGCMACKGTCTTMCC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Sfrag_R.1</td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">GGGGGGGGGGTGAAAGGYRG</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372193_12864_2025_11938_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/679f/12372193/08d687506985/12864_2025_11938_Fig2_HTML.jpg" loading="lazy" id="d33e1049" height="542" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gel electrophoresis of RT-PCR products generated for every serotype except C using the L_scheme amplification strategy. To the right-hand side of each gel the genomic regions which each product spans with the anticipated size underneath in base pairs (bp) is provided. M = Hyperladder 1KB molecular ladder or, for the S fragment, GeneRuler 100 bp (Thermo Fisher Scientific). Individual bands of the ladders are marked according to their size in base pairs (bp)</p></figcaption></figure><p id="Par17">Each L_scheme primer set was able to amplify every FMDV isolate tested, including representatives of each currently circulating serotype (Fig. <a href="#Fig2" class="usa-link">2</a>). The L_scheme was therefore selected for further development due to its comparative simplicity as well as its ability to amplify diverse FMDV isolates. Each primer pair was allocated to one of two multiplex pools of primers (Table <a href="#Tab2" class="usa-link">2</a>). Each pool amplified three amplicons which, once mixed encompassed the entire genome (Fig. <a href="#Fig1" class="usa-link">1</a>). The pooled L_scheme PCR products were in turn used as the input for the Oxford nanopore rapid barcoding kit prior to sequence generation.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Sequencing evaluation</h3>
<p id="Par18">To evaluate the universal nature of the sequencing protocol, we amplified and sequenced the genomes of 30 FMDV isolates. The list of isolates included representatives of all serotypes (except serotype C) and included examples of 27 lineages currently circulating in the field (Table <a href="#Tab3" class="usa-link">3</a>).</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Isolates used for L_scheme nanopore sequencing evaluation</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Isolate</th>
<th align="left" colspan="1" rowspan="1">Serotype</th>
<th align="left" colspan="1" rowspan="1">Topotype</th>
<th align="left" colspan="1" rowspan="1">Lineage</th>
<th align="left" colspan="1" rowspan="1">Sub-lineage</th>
<th align="left" colspan="1" rowspan="1">VP1 identity (%)</th>
<th align="left" colspan="1" rowspan="1">Accession number</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">IRN/24/2015</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">ASIA</td>
<td align="left" colspan="1" rowspan="1">Iran-05</td>
<td align="left" colspan="1" rowspan="1">SIS-10</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524752" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524752</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NEP/5/2021</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">ASIA</td>
<td align="left" colspan="1" rowspan="1">G-VII</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524754" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524754</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PAK/1/2020</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">ASIA</td>
<td align="left" colspan="1" rowspan="1">Iran-05</td>
<td align="left" colspan="1" rowspan="1">SIS-13</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524756" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524756</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PAK/30/2021</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">ASIA</td>
<td align="left" colspan="1" rowspan="1">Iran-05</td>
<td align="left" colspan="1" rowspan="1">FAR-11</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524757" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524757</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">VIT/19/2017</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">ASIA</td>
<td align="left" colspan="1" rowspan="1">Sea-97</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524758" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524758</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KEN/10/2021</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">AFRICA</td>
<td align="left" colspan="1" rowspan="1">G-I</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524753" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524753</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NIG/87/2020</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">AFRICA</td>
<td align="left" colspan="1" rowspan="1">G-IV</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524755" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524755</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PAK/47/2021</td>
<td align="left" colspan="1" rowspan="1">Asia1</td>
<td align="left" colspan="1" rowspan="1">ASIA</td>
<td align="left" colspan="1" rowspan="1">Sindh-08</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">99.68</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524759" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524759</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CAM/3/2018</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">ME-SA</td>
<td align="left" colspan="1" rowspan="1">PanAsia</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524741" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524741</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISR/5/2021</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">ME-SA</td>
<td align="left" colspan="1" rowspan="1">PanAsia-2</td>
<td align="left" colspan="1" rowspan="1">QOM-15</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524746" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524746</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SRL/13/2019</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">ME-SA</td>
<td align="left" colspan="1" rowspan="1">Ind-2001</td>
<td align="left" colspan="1" rowspan="1">d</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524749" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524749</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">JOR/2/2017</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">ME-SA</td>
<td align="left" colspan="1" rowspan="1">Ind-2001</td>
<td align="left" colspan="1" rowspan="1">e</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524747" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524747</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UAE/2/2021</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">ME-SA</td>
<td align="left" colspan="1" rowspan="1">PanAsia-2</td>
<td align="left" colspan="1" rowspan="1">ANT-10</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524750" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524750</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GHA/1/2016</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">WA</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524744" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524744</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">COD/43/2021</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">EA-2</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524742" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524742</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PAT/8/2021</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">EA-3</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524748" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524748</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ETH/24/2019</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">EA-4</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524743" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524743</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HKN/1/2019</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">CATHAY</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">99.84</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524745" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524745</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">VIT/47/2018</td>
<td align="left" colspan="1" rowspan="1">O</td>
<td align="left" colspan="1" rowspan="1">SEA</td>
<td align="left" colspan="1" rowspan="1">Mya-98</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524751" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524751</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KEN/80/2010</td>
<td align="left" colspan="1" rowspan="1">SAT1</td>
<td align="left" colspan="1" rowspan="1">I</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524760" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524760</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">QTR/4/2023</td>
<td align="left" colspan="1" rowspan="1">SAT1</td>
<td align="left" colspan="1" rowspan="1">I</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524917" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524917</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TAN/27/2012</td>
<td align="left" colspan="1" rowspan="1">SAT1</td>
<td align="left" colspan="1" rowspan="1">I</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524918" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524918</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NMB/1/2020</td>
<td align="left" colspan="1" rowspan="1">SAT2</td>
<td align="left" colspan="1" rowspan="1">III</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">99.54</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524765" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524765</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ZAM/8/2021</td>
<td align="left" colspan="1" rowspan="1">SAT2</td>
<td align="left" colspan="1" rowspan="1">IV</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524767" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524767</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NIG/57/2020</td>
<td align="left" colspan="1" rowspan="1">SAT2</td>
<td align="left" colspan="1" rowspan="1">VII</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524763" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524763</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NIG/91/2020</td>
<td align="left" colspan="1" rowspan="1">SAT2</td>
<td align="left" colspan="1" rowspan="1">VII</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524764" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524764</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SUD/12/2017</td>
<td align="left" colspan="1" rowspan="1">SAT2</td>
<td align="left" colspan="1" rowspan="1">VII</td>
<td align="left" colspan="1" rowspan="1">Alx-12</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524766" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524766</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">EGY/17/2012</td>
<td align="left" colspan="1" rowspan="1">SAT2</td>
<td align="left" colspan="1" rowspan="1">VII</td>
<td align="left" colspan="1" rowspan="1">Ghb-12</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524761" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524761</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NIG/5/2021</td>
<td align="left" colspan="1" rowspan="1">SAT2</td>
<td align="left" colspan="1" rowspan="1">VII</td>
<td align="left" colspan="1" rowspan="1">Lib-12</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524762" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524762</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ZAM/9/2018</td>
<td align="left" colspan="1" rowspan="1">SAT3</td>
<td align="left" colspan="1" rowspan="1">II</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV524768" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524768</a></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par20">Every isolate yielded amplicons that could be sequenced using the MinION. Sequences were generated across the entire L fragment and, in the majority of cases, the S fragment (Fig. <a href="#Fig3" class="usa-link">3</a>). However, occasional failures were observed for the S fragment, with 6/30 samples trialled resulting in low levels of coverage. For example, the sequence data for O/ETH/24/2019 and O/GHA/1/2016 only contained one and two reads specific for the S fragment respectively, despite substantial coverage of the L fragment.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372193_12864_2025_11938_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/679f/12372193/ca209d05ec70/12864_2025_11938_Fig3_HTML.jpg" loading="lazy" id="d33e1531" height="491" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Coverage plots for FMDV isolates sequenced for the evaluation of the final arrangement of L_scheme primer pools. Six serotypes are represented encompassing 27 topotypes/lineages currently circulating globally</p></figcaption></figure><p id="Par22">A consensus sequence was generated using the lineage-agnostic NanoFMDV pipeline for each isolate, and the VP1 encoding regions compared to those obtained by the established Sanger-based sequencing approaches [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. 100% nucleotide identity was observed for 27 of the 30 isolates sequenced (Table <a href="#Tab3" class="usa-link">3</a>). Two synonymous substitutions were present for Asia1/PAK/47/2021. A guanine to cytosine substitution in O/HKN/1/2019 resulted in a non-synonymous (T861S) amino acid change, although a mixture of the bases was observed in the BAM file. SAT2/NMB/1/2020 exhibited two nucleotide substitutions and the insertion of a thymine nucleotide; however, lower coverage across the VP1 coding region was observed for this isolate (Fig. <a href="#Fig3" class="usa-link">3</a>).</p>
<p id="Par23">The initial sequencing runs performed in this study used PCR products which had been cleaned using either GFX columns or AmpureXP beads. However, this clean-up process is expensive, laborious and previous studies have reported that the clean-up of the PCR reactions prior to library preparation is not always necessary [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. To evaluate the necessity for PCR cleanup of the amplicons designed here, twelve samples from the validation panel were re-amplified in duplicate. One set of each pair was purified using AmpureXP beads as previously described, whereas the paired sample was diluted 1:10, with 10 µl of the dilution being used directly as the input for library generation.</p>
<p id="Par24">The resulting data showed that the inclusion of a cleanup step had no impact upon the subsequent breadth and depth achieved (Fig. <a href="#Fig4" class="usa-link">4</a>). One sample (SAT2/903/MOUTH SWAB) achieved a lower depth of coverage relative to the remaining samples, including the failure of the S fragment (Fig. <a href="#Fig4" class="usa-link">4</a>). However, this deficiency applied to both the purified and unpurified PCR products implying that this is a PCR artefact, and still resulted in &gt; 96% of every genomic position being covered by at least 10 reads. Therefore, it was concluded that PCR product cleanup is not required.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372193_12864_2025_11938_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/679f/12372193/4130c505322d/12864_2025_11938_Fig4_HTML.jpg" loading="lazy" id="d33e1563" height="549" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Coverage plots of sequence data obtained using PCR products either cleaned using AmpureXP beads (bead-cleaned; green) or diluted 1:10 in nuclease free water. PCR products were generated using the final arrangement of L_scheme primer pools. 903 refers to sequences derived from clinical swabs from an animal experimentally infected with SAT2/JOR/19/2023</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">In -country deployment of nanopore sequencing in Uganda</h3>
<p id="Par25">To demonstrate the utility of nanopore sequencing in an endemic setting where laboratory conditions are often limited, the initial S_scheme and L_scheme primer mixes were trialled in parallel in Uganda, a FMDV endemic nation within FMD endemic Pool 4 [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. To reflect the limited capacity typical of laboratories in many endemic countries, we performed the RT as well as the PCR of some of the samples using a portable PCR machine (miniPCR bio™). Furthermore, we used a minicentrifuge and the easily portable Quantus fluorometer (Promega) for the quantification of DNA.</p>
<p id="Par26">Both the S_scheme and L_scheme achieved near complete genomes (Fig. <a href="#Fig5" class="usa-link">5</a>A and Additional files 3–4). However, the L_scheme outperformed the S_scheme, with the L_scheme/rapid barcoding approach resulting in 15 near complete genomes (median 94.3% of the reference genome covered) from samples with Ct values with a median Ct value of 23.6 (range 17.62–27.93) at an average depth of 518. While less consistent in generating complete genomes (Fig. <a href="#Fig5" class="usa-link">5</a>B), the depth of coverage was on average greater when using the S_scheme (Fig. <a href="#Fig5" class="usa-link">5</a>C). Whilst considerable variation in coverage depth was observed, for those samples generating near-full-length sequences, the depth was always sufficiently high to generate an accurate consensus (Fig. <a href="#Fig5" class="usa-link">5</a>C, filled points). The most closely related genome to those obtained using the nanopore methodology was in every case SAT2/TAN/5/2012 (Accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/KM268900" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KM268900</a>, 94.7–94.9% nucleotide identity). The amplification and sequencing of FMDV genomes was most successful (in terms of coverage) when using clinical samples. However, a large proportion of genome was sequenced using environmental swabs collected from a stick and a rope (Additional file 5).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372193_12864_2025_11938_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/679f/12372193/76f69550194f/12864_2025_11938_Fig5_HTML.jpg" loading="lazy" id="d33e1601" height="518" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance of nanopore sequencing on FMDV positive samples collected in Uganda. (<strong>A</strong>) R epresentative coverage plots of genome sequences recovered from clinical (left) and environmental swabs. The orange trace represents the L_scheme and the green trace represents the S_scheme. (<strong>B</strong>) Scatter plots for the L (left) and S (right) primer schemes showing the proportion of the genome covered relative to real-time RT-PCR Ct value. (<strong>C</strong>) A comparison of the average depth at each nucleotide achieved using the L_scheme and S_scheme amplification strategies. Filled points represent samples resulting in near complete genomes</p></figcaption></figure><p id="Par27">Phylogenetic reconstruction of publicly available VP1 sequences retrieved from FMDbase (<a href="https://www.fmdbase.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fmdbase.org</a>) revealed that Ugandan viruses were most closely related to viruses collected from Tanzania and Kenya between 2011 and 2016, whereas the most recent Ugandan SAT2/IV sequences were collected in 2013 (Fig. <a href="#Fig6" class="usa-link">6</a>). Among the sequences generated in this study, sequences clustered at both farm and district levels (Fig. <a href="#Fig6" class="usa-link">6</a>). In contrast, phylogenetic links between individual animals sampled on each farm were resolved using sequences of the full-polyprotein coding region (Fig. <a href="#Fig7" class="usa-link">7</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372193_12864_2025_11938_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/679f/12372193/e897b9d56c3d/12864_2025_11938_Fig6_HTML.jpg" loading="lazy" id="d33e1625" height="1016" width="789" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>FMDV SAT2/IV VP1 phylogenetic tree incorporating the Ugandan sample VP1 sequences. VP1 sequences from Ugandan samples (highlighted in red) were labelled according to their District, Farm (F), Animal (A), and Barcode number. The VP1 from the closest existing complete genome sequence (Tanzania, 2012) is highlighted in blue, whilst existing SAT2/IV VP1 sequences from Uganda on GenBank are highlighted in purple; all other SAT/IV sequences are shown in black. This is a cropped version of the full VP1 phylogenetic tree, only displaying the cluster containing the Ugandan sample sequences generated in this study; the complete tree is available as Additional file 6. SAT2/I and SAT2/III lineage sequences were used as an outgroup</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372193_12864_2025_11938_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/679f/12372193/2119b8435f8e/12864_2025_11938_Fig7_HTML.jpg" loading="lazy" id="d33e1635" height="544" width="669" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>FMDV SAT2/IV complete genome phylogenetic tree incorporating the Ugandan sample consensus genomes. Consensus genome sequences from Ugandan samples (highlighted in red) were labelled according to their District, Farm (F), Animal (A), and Barcode number. The closest existing genome sequence (Tanzania, 2012) is highlighted in blue whilst other SAT2/IV complete genome sequences available from GenBank are shown in black. SAT2/I and SAT2/III lineage sequences (highlighted in brown) were used as an outgroup. Bootstrap values are shown next to their corresponding branches</p></figcaption></figure></section></section><section id="Sec6"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par30">It is well established that multiple serotypes and topotypes of FMDV often co-circulate within the regional endemic pools, where inter-pool movements of FMDV lineages can further complicate the epidemiology of the disease [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. Tools are urgently needed to improve the surveillance of FMD in these settings to understand the burden of disease and regional risk pathways. Furthermore, rapid identification of field FMD viruses is critical when there are new incursions of FMDV into FMD-free countries [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. In view of the genomic diversity of FMDV, universal sequencing protocols are anticipated to provide capability for endemic and epidemic scenarios where a wide range of virus serotype and topotypes may need to be characterised.</p>
<p id="Par31">Primer designs for universal genome amplification strategies have previously been published [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. However, the necessity to locate primer binding sites within relatively restricted regions to generate approximately equally sized amplicons for the S_scheme necessitated a re-appraisal of all available primer sequences. Aligning primers to 1682 FMDV complete genomes enabled the rational selection of existing primers, as well as the design of new primer sequences that would be predicted to amplify all FMDV sequences. Whilst focusing upon currently circulating viruses may have resulted in a more streamlined set of primers appropriate for the current epidemiological situation, the use of historical sequences has instead allowed the identification of primer footprints that have not changed over several decades of evolution, providing greater confidence that these targets are highly conserved and unlikely to change. Furthermore, the design of primers capable of amplifying historical samples means that the schemes are appropriate for retrospective analyses of viruses held in laboratory archives.</p>
<p id="Par32">The quality of data obtained using nanopore-based sequencing continues to improve as the chemistry and base calling models evolve [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. We observed that using the super-high accuracy method greatly enhanced the accuracy of the resulting sequences. In common with many forms of HTS, homopolymeric regions often proved to be the areas that were most difficult to resolve. Insertions or deletions of nucleotides (indels) can be resolved manually for protein coding regions where amino acid conservation is required. However, in non-coding regions, where indels are frequently observed with FMDV [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>–<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>], base-calling errors could influence the accuracy of the consensus genomic sequence.</p>
<p id="Par33">The DNA PCR products used in the initial library preparations for this project (including in Uganda) were purified using GFX spin columns. Subsequently, we showed that the extent of data generated was not affected by the omission of this cleanup step. Whilst the exclusion of a cleanup step needs further evaluation, for example its impact upon sensitivity, it results in substantial savings in terms of both time and money, and is a finding in line with Freed e<em>t al</em> [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. It should be noted that the validation undertaken here used only SuperFiII Green mastermix and it remains possible that other mastermixes may be inhibitory to downstream library generation. In addition, not all library preparation kits may tolerate the omission of a purification step. Preliminary results suggest that Q5 polymerase (New England Biolabs) is also suitable for use in this protocol, although diluting the PCR products 1:2 rather than 1:10 (data not shown). Similarly, barcoding alongside the ability to wash and reuse flow cells makes nanopore sequencing more cost effective and particularly attractive for use in regions where support for more established workflows is limited [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. A previous study sequencing SARS-CoV-2 calculated that genomes could be sequenced at low cost using nanopore technology (at the time of publication £10/sample). The most efficient manner in which to reduce costs was found to be by multiplexing the samples and by reusing washed flowcells [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. However, when reusing flowcells, the selection of barcodes needs to be carefully considered due to the possibility of carry-over between runs.</p>
<p id="Par34">The coverage obtained for the S fragment was not always consistent. The occasional failure to obtain sequence for the S fragment may be due to weak PCR amplification as frequently observed via gel electrophoresis. An improvement to this method would therefore be to increase the efficiency of the S fragment within a multiplex format, although this is challenging due to the downstream flanking poly (C) tract. Alternatively, a three-reaction format could be used with a reaction dedicated to amplifying the S fragment. An alternative explanation is the fact that the rapid barcoding chemistry is based upon the random insertion of barcodes. The S fragment amplicon is ~ 380 bp, thus there are fewer positions at which a barcode can successfully be inserted and still result in a read longer than the required 200 bp (Oxford Nanopore Technologies, personal communication).</p>
<p id="Par35">A distinct advantage of using a larger fragment size is the ability to use the rapid barcoding kit as well as the requirement for fewer primers per reaction. Indeed, despite a larger amplicon size, the L_scheme outperformed the S_scheme when tested using clinical and environmental swabs collected in Uganda. This result suggests that there is scope for improvement in the efficiency of the S_scheme PCRs. Despite the clear advantages of the L_scheme when sequencing FMDV, in situations requiring greater analytical sensitivity, e.g. wastewater analysis, an optimised primer scheme based upon short fragments has merit.</p>
<p id="Par36">Trialling the sequencing approaches in Uganda highlighted the benefits of using swabs derived from animals exhibiting clinical signs relative to environmental swabs. Genomes were also generated using swabs collected from the lesions of experimentally infected cattle, further demonstrating the suitability of clinical lesion swabs as a sample matrix. It is anticipated that these results can be explained by the higher titre of FMD virus present in clinical samples, as well as difficulties to sequence degraded RNA in environmental swabs due to environmental factors such as heat and ultraviolet light exposure [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. However, the sequencing of FMDV from environmental samples complements the data obtained from clinical samples and can contribute to uncovering active transmission chains during outbreaks [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>, <a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. Furthermore, the integration of field-appropriate methods in resource-limited settings significantly enhances the scalability of FMDV sequencing in endemic areas. The successful generation of sequence data in Uganda demonstrated that this method is possible to use with limited equipment and resources. Nevertheless, the development of lyophilised/dried reagents with stability at room temperature would be hugely beneficial in locations with a propensity for power supplies to be interrupted.</p>
<p id="Par37">The generation of near complete genomes enabled phylogenetic trees to be constructed using the complete coding sequence of the virus. Using a greater proportion of the genome enabled individual animals to be distinguished on each farm. Thus, these data further demonstrate the power of complete genome sequence analyses when informing epidemiological studies. In particular, reconstructing the path of infection from animal-to-animal and farm-to-farm through complete genome sequencing complements network analyses of livestock movements via contact tracing and trade data. This approach can be integrated with other types of epidemiological data for accurately parameterizing dynamic models in complex eco-epidemiological systems involving multiple hosts, viral evolution and environmental persistence.</p>
<p id="Par38">The pipeline developed here was entirely agnostic with regards to FMDV serotype and lineage, and correctly assembled the genomes of every virus trialled. However, whilst infrequent, FMDV co-infections do occur, most notably the SAT serotypes which persist in African buffalo. Similarly, recombination readily occurs between co-infected viruses, suggesting that a pipeline reliant upon reference assembly may struggle to generate genome length sequences of recombinant viruses. Further work is required to fully establish the ability of the pipeline to detect the presence of, and assemble the genomes of, viruses derived from mixed infections.</p>
<p id="Par39">In summary, we have developed a universal RT-PCR and sequencing approach capable of generating complete FMDV genomes. The method, in combination with the NanoFMDV pipeline, was successfully trialled in Uganda and represents a route to straightforward and economical sequencing of FMDV genomes in laboratory settings where NGS infrastructure is lacking.</p></section><section id="Sec7"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec8"><h3 class="pmc_sec_title">Amplicon design</h3>
<p id="Par40">Primers predicted to encompass the full extent of FMDV diversity were designed using the publicly available sequences hosted in FMDbase. Complete FMDV genome sequences in FMDbase were downloaded and aligned using MAFFT v7.453. Many of the multiplex ‘ARTIC’-like approaches for sequencing small viral genomes utilise small amplicons approximately 400 bp in length. We therefore developed a similar approach, termed S_scheme, using two PCR reactions to amplify 20 overlapping tiles. However, the nucleotide diversity evident across the FMDV genome necessitated flexibility in the precise location of primer footprints. An amplicon strategy utilising fewer, but larger PCR amplicons (termed L_scheme, analogous to the ‘Midnight’ protocol used for SARS-CoV-2 [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]) divided across three PCR reactions was trialled in parallel with the S_scheme. In addition to requiring fewer PCR amplicons and therefore fewer conserved primer sites, the L_scheme utilised the ONT rapid barcoding kit, which is simpler, quicker and cheaper [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. An initial screen was undertaken to appraise previously published primer sites, for example those described in Knowles et al. [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>] and Dill et al. [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>], for their suitability in terms of i) cross-reactivity among serotypes and topotypes and ii) position within the genome. In certain cases, it was possible to modify some of the published primers, for example the FMD-L137F primer from Dill et al. [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>] was adjusted in terms of both its position and its incorporation of redundant bases. However, in the majority of cases, the necessity for specific regions to be targeted required the design of new primers with broad reactivity across the FMDV species. Software for the design of tiled amplicons, for example PrimalScheme, have previously been developed. However, the extent of genomic diversity tolerated by the original PrimalScheme algorithm precluded its use in the development of a pan-FMDV primer scheme. Therefore, primer locations were identified manually by visual inspection of alignments. The level of cross-reactivity of prospective primers was assessed using a custom R script (Additional files 7–8), and the melting temperatures predicted using OligoEvaluator™ (Sigma-Aldrich).</p>
<p id="Par41">With certain exceptions, for example A, the P2 and P3 regions of the FMDV genome are relatively conserved (68–91% nucleotide identity [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]), and a limited set of oligos was therefore sufficient to match the majority of the sequences. In contrast to the P2 and P3 regions, the P1 region of the FMDV ORF is highly diverse, with particularly high levels of sequence variability (48–75% nucleotide identity [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]), observed across the 1B, 1C, and 1D regions (encoding VP2, VP3 and VP1 respectively). A previous route to avoid the issue of variability in P1 has been to generate a PCR amplicon which spans the entire capsid coding region, with primers located in Lpro and 2A/2B [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. However, in order to achieve efficient multiplex amplification of the entire genome (most notably in the case of the S_scheme), we aimed to amplify amplicons of approximately equal size, resulting in some amplicons requiring multiple primers targeting the same (or a nearby) footprints (median = 3.5 primers per amplicon; range = 2–13). This was most evident in amplicon 7, which included seven sense primers and six antisense primers (Additional file 9; Table <a href="#MOESM3" class="usa-link">S3</a>). Furthermore, certain sites within the capsid coding region required multiple primers within a serotype, most notably for serotypes SAT1 and SAT2, where the sequences were sufficiently distinct between east and west African clades (Table <a href="#Tab2" class="usa-link">2</a>). In situations where the extent of diversity would require a multitude of primers, we adopted a rational approach by prioritising currently circulating lineages for successful amplification as they are the most likely sequences to be encountered. Separate alignments of the genomes for each serotype were prepared when designing primer schemes for footprints within regions encoding structural proteins.</p>
<p id="Par42">The region immediately following the poly(C) region of the FMDV region contains multiple pseudoknots and is extremely variable [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. We countered this by using primers comprising a string of cytosines followed by a short sequence of FMDV L fragment sequence. The initial four bases immediately following the poly(C) were extracted and tabulated revealing a wide variety of possible combinations (Fig. <a href="#Fig8" class="usa-link">8</a>A). Although variable, it was possible to allocate the majority of the sequences to one of just five consensus sequences (Fig. <a href="#Fig8" class="usa-link">8</a>B). Five sense primers were therefore used for the amplicon immediately following the poly(C) region. Each version of the sense primer for the amplicon following the poly(C) was modified by extending the primer in the 5’ direction as initial attempts without this extension were variable (Fig. <a href="#Fig8" class="usa-link">8</a>C). The 5’ extensions were shown to increase both reliability (Fig. <a href="#Fig8" class="usa-link">8</a>C) as well as efficiency (Fig. <a href="#Fig8" class="usa-link">8</a>D).</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372193_12864_2025_11938_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/679f/12372193/0104e8946fef/12864_2025_11938_Fig8_HTML.jpg" loading="lazy" id="d33e1788" height="549" width="669" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Optimisation of the UNI_For_1 primer region. (<strong>A</strong>) Proportion of each four-nucleotide sequence observed immediately following the poly(C) tract. Sequences were tallied from 1475 representative sequences. (<strong>B</strong>) The proportion of sequences encompassed by each of the UNI_For_1 primer variations. Only the 3’-most nucleotides specific to each primer are presented. ‘other’ equates to four nucleotide sequences not covered by any of the five primer variations (<em>n</em> = 6). (<strong>C</strong>) Agarose gel and semi-quantitation (<strong>D</strong>) of PCR products generated using the initial (1) and final (1.1) primer designs for UNI_For_1. L: molecular weight ladder (GeneRuler 100 bp, Thermo Fisher Scientific Scientific)</p></figcaption></figure><p id="Par44">We incorporated these modified UNI_For_1 primers into a final L_scheme arrangement of two pan-FMDV multiplex PCRs amplifying long fragments (range 1109–2237 bp) and the S fragment (~ 380 bp) (Fig. <a href="#Fig1" class="usa-link">1</a>B; Table <a href="#Tab2" class="usa-link">2</a>).</p>
<p id="Par45">The extreme 3’ of the 3’UTR immediately preceding the poly(A) tail is highly diverse, with insufficient homology to design primers capable of amplifying diverse viruses. We resolved this by using FMD_UNI_RT, which incorporates a 5’ extension of the poly-T stretch with a M13 primer sequence, as well as a short, redundant FMDV-specific consensus sequence (GRWWW) to help target and specifically anchor the primer to FMDV genomes. Importantly, this primer needed to be added to the reverse transcription reaction (Fig. <a href="#Fig9" class="usa-link">9</a>).</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372193_12864_2025_11938_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/679f/12372193/d25576052f3b/12864_2025_11938_Fig9_HTML.jpg" loading="lazy" id="d33e1811" height="247" width="669" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Agarose gel demonstrating the requirement for FMD_UNI_RT inclusion in the reverse transcription (RT) step. The 3’most amplicon (amplicon 19) was amplified from SAT1/QTR/4/2023 and A/CAM/5/2015 cDNAs prepared with (+) or without (-) FMD_UNI_RT in the RT reaction. A control amplicon (amplicon 10) amplifying the initial part of P2 not dependent upon the inclusion of FMD_UNI_RT. NTC = no template control. L: molecular weight ladder (GeneRuler 100 bp, Thermo Fisher Scientific Scientific)</p></figcaption></figure><p id="Par47">Redundant bases were judiciously included to increase the likelihood that diverse FMDV sequences would be amplified, with a focus upon reducing template-primer mismatches towards the 3’ end of the primer. The primer extends from this end and previous studies have indicated that mismatches in the five 3’-most nucleotides have a disproportionately greater impact upon the success of amplification relative to the 5’ end of a primer.</p>
<p id="Par48">Notably, serotype C viruses were not considered a priority for the schemes designed here as isolates of this serotype have not been observed in the field since 2004, leading to suggestions that serotype C is extinct in the wild [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. However, alignment of the primer sequences against serotype C genomes (<em>n</em> = 40), suggests that serotype C isolates would likely still be amplified using the selected primers (Additional file 10).</p></section><section id="Sec9"><h3 class="pmc_sec_title">Viruses and samples</h3>
<p id="Par49">Samples (<em>n</em> = 53) used for the initial development and evaluation comprised RNA derived from viruses isolated from samples submitted to the WRLFMD at The Pirbright Institute (Tables <a href="#Tab1" class="usa-link">1</a> and <a href="#Tab3" class="usa-link">3</a> and Additional file 11).</p></section><section id="Sec10"><h3 class="pmc_sec_title">Clinical and environmental samples</h3>
<p id="Par50">RNA samples were derived from swab samples collected from farms with ongoing outbreaks of FMDV in three districts in Uganda (Budaka, Bugiri and Bukedea). 4N6FLOQ™ swabs (Thermo Fisher Scientific) were used to swab possible lesions present on either the feet or mouth (‘clinical’ swabs, <em>n</em> = 40) and were placed into 1 mL PBS. Lesion swabs were collected from animals at different clinical stages of infection including at &gt; 10 days representing almost healed lesions. Environmental samples (<em>n</em> = 14) were collected as previously described [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>] at farms on which infected cattle were present. The Ugandan National Council of Science and Technology (UNCST) and the University of Edinburgh Animal Welfare and Ethical Review Board (AWERB) committee gave approval for both animal sampling (OS5-21) and human questionnaires. A 150 µL aliquot of the swab eluate was added to 337.5 µL of MagMax core lysis buffer (Thermo Fisher Scientific) and chilled whilst in the field. Upon return to the laboratory, samples in lysis buffer were stored at −80 °C until extraction. Swabs were first assessed for FMD presence using real-time RT-PCR as previously described [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. Fifty-four swab samples positive for FMDV by real-time RT-PCR (Ct range 16.23–43.65, mean 30.53) were subjected to sequencing using both the S_scheme and L_scheme protocols.</p>
<p id="Par51">Additional lesion swab samples were collected from cattle experimentally infected with FMDV at the Pirbright Institute. Animals used in this study were sourced from a commercial farm with defined health status. Prior to inoculation, animals were housed in groups to acclimatise for seven days prior to inoculation. Animals were sedated prior to FMDV challenge by administration of 0.3 mg/kg of xylazine via the intramuscular route. Cattle were placed into sternal recumbency, inoculated with FMDV (1 × 10<sup>5</sup> TCID<sub>50</sub> into the dorsal epithelium of the tongue), and then the sedation reversed by administering 0.1 mg/kg atipamezole. At the time of euthanasia, animals were secured and 140 mg/kg of sodium pentobarbital administered via the intravenous route. All experimental procedures were conducted in accordance with the Pirbright Institute’s AWERB, and the Home Office Animals (scientific procedures) Act 1986.</p></section><section id="Sec11"><h3 class="pmc_sec_title">RNA extraction</h3>
<section id="Sec12"><h4 class="pmc_sec_title">Virus isolates</h4>
<p id="Par52">Viral RNA was isolated from 0.5 mL cell culture supernatants maintaining the necessary ratios of lysis buffer to sample. RNA was extracted using the RNeasy mini kit columns (Qiagen) according to the manufacturer’s conditions. RNA samples were eluted from the columns in 50 µL nuclease free water and stored at −80 °C.</p></section><section id="Sec13"><h4 class="pmc_sec_title">Swab eluates</h4>
<p id="Par53">Viral RNA was isolated manually using a magnetic bead approach using the MagMax core reagents (Thermo Fisher Scientific). Briefly, 7.5 µL of beads/proteinase K mix (2:1 ratio) was added to 125 µL of lysis buffer/sample and mixed thoroughly followed by the addition of 87.5 µL of binding solution. 125 µL of wash 1 solution was added and mixed thoroughly. The beads were captured with a magnet, the supernatant discarded and washed with 125 µL of wash 2. Following capture and discard of the wash supernatant, the RNA was eluted from the beads in 90 µL of elution buffer and stored at −80 °C.</p></section></section><section id="Sec14"><h3 class="pmc_sec_title">Reverse transcription</h3>
<p id="Par54">Reverse transcription was performed using the LunaScript SuperMix (New England Biolabs). Although LunaScript SuperMix already contains oligo dT and random hexamers, reactions were performed with the addition of the FMD_UNI_RT primer. Briefly, RT mixes comprised 2 µL Lunascript SuperMix (5X), 0.5 µL FMD_UNI_RT (50 µM stock) and 7.5 µL sample RNA. The components were mixed and incubated for two minutes at 25 °C, 10 min at 55 °C and 1 min at 95 °C. Following incubation, cDNA samples were either used immediately or stored at −20 °C. cDNA samples were assessed for FMDV titre using real-time PCR and, on average had a Ct of 20.26 (Additional file 11).</p></section><section id="Sec15"><h3 class="pmc_sec_title">PCR amplification and cleanup of sequencing template</h3>
<p id="Par55">All conventional PCR reactions used SuperFiII Green PCR mix (Thermo Fisher Scientific) in a PCR volume of 10 µL. When individual amplicons were amplified, 5 pmols of each primer were used. Multiplex reactions were performed by first mixing equal volumes (except for S_Frag_F.1 and R.1, for which double the volume was used), of each forward and reverse stock primer (100 µM) in a single tube followed by a 1:10 dilution to make a working stock, of which 4 µL was used in the final 10 µL PCR reaction. Reaction volumes were adjusted to 9 µL to which 1 µL of cDNA was added. S_scheme PCR cycling was as follows: 98 °C for 1 min (x1), 30 cycles of 98 °C for 10 s, 60 °C–10 s and 72 °C for 15 s, followed by a final extension of 72 °C for 5 min. PCR cycling for the larger L_scheme fragments was as follows: 98 °C for 1 min (x1), 35 cycles of 98 °C for 10 s, 60 °C–15 s and 72 °C for 3 min, followed by a final extension of 72 °C for 5 min. PCR products were analysed using 1% agarose gels stained with SYBR Safe gel stain (Thermo Fisher Scientific) and were purified using either the GFX PCR DNA and Gel Band Purification kit (Cytivia) or AmpureXP Beads according to the manufacturer’s instructions, eluting the DNA in 50 or 15 µL water respectively.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Real-time RT-PCR</h3>
<section id="Sec17"><h4 class="pmc_sec_title">Hydrolysis probe based real-time PCR</h4>
<p id="Par56">RNA isolated from swab samples was tested for FMDV presence using the Callahan 3D PCR primers and probe targeting the viral 3D RNA polymerase [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. The real-time RT-PCR reactions were run as previously described [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>] using Toughmix RT-PCR mix (Quantabio, VWR) or Luna Universal qPCR probe mix on a QuantStudio 5 Real-Time PCR machine (Applied Biosystems).</p></section><section id="Sec18"><h4 class="pmc_sec_title">SYBR-based real-time PCR</h4>
<p id="Par57">A universal PCR mix (Luna Universal qPCR, New England Biolabs) was used to semi-quantitate the efficiency of amplification. Half reactions comprising 5 µL 2x Luna Universal qPCR mix, 2.5 pmols each primer, 3.5 µL nuclease free water and 1 µL PCR product (diluted 1:10,000). To measure specific amplicons the reverse primer of the amplicon of interest was used in combination with the forward primer of the downstream amplicon. Reactions were cycled in a QuantStudio 5 Real-Time PCR machine using the fast profile followed by a melt curve step.</p></section></section><section id="Sec19"><h3 class="pmc_sec_title">Nanopore sequencing</h3>
<p id="Par58">S_scheme products were sequenced using the Oxford Nanopore Technologies (ONT) native barcoding kit (NBD) and the L_scheme products were sequenced using the ONT rapid barcoding kit (RBK). Initial trials used SQK-NBD-110.96/SQK-RBK-110.96 chemistry and R9.4.1 flow cells. However, the final evaluation as well as sequencing in Uganda used the most recent chemistries (kits SQK-NBD114.96 and SQK-RBK114.24 (Oxford Nanopore Technologies, Oxford, UK) for S_scheme and L_scheme fragments respectively) in conjunction with R10.4.1 flow cells (Oxford Nanopore Technologies). Nanopore sequencing of FMDV positive Ugandan swab samples was undertaken at Makerere University College of Veterinary Medicine, Animal Resources and Bio-Security (COVAB).</p></section><section id="Sec20"><h3 class="pmc_sec_title">Evaluation of nanopore sequencing</h3>
<p id="Par59">Pilot sequencing runs utilised 400 ng of PCR product per sample to prepare libraries. To determine whether it would be possible to use a lower mass of input DNA, we performed sequencing reactions with a total of either 400 ng or 100 ng of PCR product. Reducing the mass of DNA had minimal effect regardless of serotype (Fig. <a href="#Fig10" class="usa-link">10</a>). Although we successfully obtained genomes using significantly less DNA, subsequent libraries were prepared using (where quantified) 100 ng of DNA.</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372193_12864_2025_11938_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/679f/12372193/11b407b23fd0/12864_2025_11938_Fig10_HTML.jpg" loading="lazy" id="d33e1914" height="476" width="669" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Impact of input mass on the coverage achieved across FMDV genomes. Representative isolates were used for serotypes O, A, Asia 1, SAT1, SAT2 and SAT3. Libraries were prepared using 100 (green trace) or 400 (orange trace) ng of input DNA. PCR products were generated using the L_scheme</p></figcaption></figure><p id="Par61">A target input mass for the SQK-NBD114.96 libraries was 70 ng whereas the target input for SQK-RBK114.24 libraries was 100ng. In situations where less than the target mass was achieved, library preparation continued with the maximum volume of product achievable (11.5 µl–10 µl for SQK-NBD114.96 and SQK-RBK114.24 respectively). Libraries were prepared according to the manufacturer’s instructions and included BSA in the flow cell priming mix. Flowcells were run using either a MinION Mk1B or Mk1C for 24–72 h. Raw data were basecalled using the super-high accuracy model within the MinKNOW software (MinKNOW version 24.06.5; Dorado version 7.4.12).</p></section><section id="Sec21"><h3 class="pmc_sec_title">Phylogenetic analysis</h3>
<p id="Par62">Both BLAST and FMDbase (<a href="https://www.fmdbase.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fmdbase.org</a>) were used to determine the FMDV serotype/topotype of the nanopore generated consensus sequences. Consensus sequences were aligned with all complete genomes of the identified SAT2/IV topotype available on GenBank (<em>n</em> = 4) with exemplar genome sequences representing the SAT2/I and SAT2/III topotype as an outgroup. Sequences were aligned using MAFFT v7.520, and a phylogenetic tree was constructed using IQ-TREE (v2.0.6) with 1000 bootstrap replicates; the Transversion Invariant Model TIM2 + F + I + G4 was identified as the best fit substitution model. Additionally, all available SAT2/IV sequences available on GenBank were aligned using MAFFT along with the nanopore generated consensus sequences and SAT2/I and SAT2/II outgroups. The corresponding VP1 section of the alignment was manually extracted (<em>n</em> = 204 GenBank VP1 sequences), and a phylogenetic tree was constructed using IQTREE-2 with 1000 bootstrap replicates; TIM2 + F + I + G4 was identified as the best fit substitution model.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Bioinformatics pipeline</h3>
<p id="Par63">The bespoke bioinformatics pipeline is currently implemented via a collection of bash scripts, which call existing bioinformatics tools as well as custom Python (version 3.8.15) scripts (Additional file 12). The pipeline, referred to as NanoFMDV, along with the reference FMDV sequences, and usage instructions is publicly available at: <a href="https://github.com/rjorton/nanofmdv" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/rjorton/nanofmdv</a>. The first step in the bioinformatics pipeline is to determine an appropriate FMDV reference sequence to use for read alignment. A database of 79 near complete FMDV genomes were downloaded from the FMDbase sequence database (<a href="https://www.fmdbase.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fmdbase.org</a>), representing approximately one per existing FMDV lineage; typically the FMDbase ‘RefSeq’ was selected for each lineage if available, unless a significantly longer genome sequence (ignoring N bases) was available. The VP1 sequence of each genome was extracted to create an equivalent FMDV-VP1 sequence data set. Each demultiplexed sample is analysed independently. After concatenating all of a sample’s FASTQ reads into a single file, minimap2 (version 2.24-r1122) [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>] is used to align the reads to the FMDV-VP1 sequence collection. The VP1 reference sequence with the highest number of reads mapping to it is selected as the reference sequence for the sample, with minimap2 again used to align all the reads on to the reference FMDV genome corresponding to the selected VP1 sequence. As tiled amplicons are used, amplicon primer sequences must be trimmed before consensus sequence generation. A simple Python script, based on the previous GoPrime [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>] methodology, is used to compare all the possible forward and reverse primers of each amplicon to the selected reference sequence to identify possible binding sites. A BED file is subsequently created representing the binding regions of each amplicon’s primers on the reference sequence. The samtools (version 1.16.1) [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>] ampliconclip function is then used to clip primer sequences from the aligned reads within the BAM file using the previously created BED file, with a primer-clipped read FASTQ file also generated using the samtools fastq function. A preliminary consensus sequence is generated using the samtools consensus function. This preliminary consensus sequence, along with the primer-clipped read FASTQ file, is then used by the nanopore variant/consensus caller medaka (<a href="https://github.com/nanoporetech/medaka" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/nanoporetech/medaka</a>, version 1.7.2) to call a ‘polished’ FMDV consensus genome sequence (medaka has models to address nanopore associated errors such as those at homopolymers), the consensus sequence is also depth masked to mask positions with read coverage less than 10 with N bases. A final step in the pipeline, is to BLASTn [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>] the final consensus sequence against a collection of 923 FMDV genome sequences to determine the closest sequence matches and their respective serotype/topotype. Assembly statistics were generated using samtools. Analyses were performed using R (version 4.4.0) [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. An example fastq dataset has been submitted to the ncbi short read archive (SRA, accession number PRJNA1286760; <a href="https://www.ncbi.nlm.nih.gov/sra/PRJNA1286760" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/sra/PRJNA1286760</a>) for SAT1/TAN/27/2012 which, when assembled should result in the same sequence as accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/MT602086" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MT602086</a>.</p></section></section><section id="Sec23"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (61.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (23.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3.</a><sup> (594.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4.</a><sup> (2.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5.</a><sup> (2.7MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6.</a><sup> (90.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7.</a><sup> (50.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 8.</a><sup> (126.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 9.</a><sup> (34.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM10_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 10.</a><sup> (38.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM11_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 11.</a><sup> (59.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM12"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM12_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 12.</a><sup> (134.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM13"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM13_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 13.</a><sup> (2.3MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM14"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM14_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 14.</a><sup> (2.4MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM15"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM15_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 15</a><sup> (3.1MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM16"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM16_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 16.</a><sup> (2.9MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM17"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM17_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 17.</a><sup> (2.9MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM18"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM18_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 18.</a><sup> (16.9KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Oxford Nanopore Technologies for the donation of reagents to undertake sequencing in Uganda, and the COVAB laboratory for assistance with logistics at Makerere University.</p></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>AES conceived the project, designed and developed methodology, performed analyses and wrote the paper (original draft and reviewing/editing). LGG conceived and undertook fieldwork in Uganda to collect/sequence samples, and obtained funding. RJO developed the NanoFMDV pipeline, performed analyses and wrote the paper. KL and UI evaluated the primer schemes. JW and HH provided RNA samples and assisted in validation of the protocol. NP and GF set up the nanopore system at the Pirbright Institute. DM obtained funding and assisted with testing the system in Uganda. CT, NJK and ADN performed computational analyses. RAW collected clinical samples from animals experimentally infected with FMDV at the Pirbright Institute. BMB and DPK conceived the project, assisted with data generation, reviewed the paper and acquired funding.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was primarily funded by the Department for Environment Food and Rural Affairs (defra projects, SE2945, SE2947 and GAP-DC2 (SE0574)). We would like to acknowledge the Pirbright Institute’s Strategic Program and High throughput Sequencing and Bioinformatics STP support from UK Research and Innovation (UKRI) Biotechnology and Biological Sciences Research Council (BBSRC) grants BBS/E/PI/23NB0003, BBS/E/PI/230002C and BBS/E/PI/23NB0004. BMB acknowledges support from the BBSRC (BBS/E/RL/230002D). BMB, LGG, AES, DK and DM thank the FAO European Commission for the Control of Foot-and-Mouth Disease (EuFMD) for support of the field work in Uganda via the Ninth Call EuFMD – Fund for Applied Research (EUFMD-FAR) – 2021.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Sequence data generated during the validation of this method have been submitted to GenBank and are available under accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PV524741:PV524918%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524741- PV524918</a>. Further sequences generated in Uganda have been submitted to GenBank and are available under accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PV883408:PV883420%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV883408 - PV883420</a>. A set of FASTQ files has been uploaded to the short read archive and are available under accession number PRJNA1286760.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par64">Samples were collected from animals experimentally infected with FMDV at the Pirbright Institute in accordance with local and national regulations. The farmer from whom the cattle for experimental infection were purchased was fully aware that the animals were purchased for the purpose of scientific research. All animals were euthanised via a Schedule 1 method (deemed most welfare friendly according to Animals (Scientific Procedures) Act 1986).</p>
<p id="Par65">Written informed consent was obtained from all of the Ugandan farmers who provided access to their premises to collect samples from their animals. Study protocols were approved by the University of Edinburgh Animal Welfare and Ethical Review Board (AWERB, Ref: OS5-21) and Human Ethical Review Committee (HERC, Ref: HERC_2022_19) at the Royal (Dick) School of Veterinary Sciences), the Ugandan National Council of Science and Technology (UNCST, Ref: A217ES), and the Institutional Animal Care and Use Committee (IACUC, Ref: SVAR-IACUC/108/2022) at the School of Veterinary Medicine and Animal Resources (SVAR), Makerere University.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par66">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par67">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Knight-Jones TJD, Rushton J. The economic impacts of foot and mouth disease – What are they, how big are they and where do they occur? Prev Vet Med. 2013;112(3–4):161–73.
</cite> [<a href="https://doi.org/10.1016/j.prevetmed.2013.07.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3989032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23958457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Knight-Jones%20TJD,%20Rushton%20J.%20The%20economic%20impacts%20of%20foot%20and%20mouth%20disease%20%E2%80%93%20What%20are%20they,%20how%20big%20are%20they%20and%20where%20do%20they%20occur?%20Prev%20Vet%20Med.%202013;112(3%E2%80%934):161%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Paton DJ, Di Nardo A, Knowles NJ, Wadsworth J, Pituco EM, Cosivi O, Rivera AM, Kassimi LB, Brocchi E, de Clercq K et al. The history of foot-and-mouth disease virus serotype C: the first known extinct serotype? Virus Evol. 2021;7(1):1–12.</cite> [<a href="https://doi.org/10.1093/ve/veab009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8102019/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35186323/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Paton DJ, Sumption KJ, Charleston B. Options for control of foot-and-mouth disease: knowledge, capability and policy. Philos Trans R Soc Lond B Biol Sci. 2009;364(1530):2657–67.
</cite> [<a href="https://doi.org/10.1098/rstb.2009.0100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2865093/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19687036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paton%20DJ,%20Sumption%20KJ,%20Charleston%20B.%20Options%20for%20control%20of%20foot-and-mouth%20disease:%20knowledge,%20capability%20and%20policy.%20Philos%20Trans%20R%20Soc%20Lond%20B%20Biol%20Sci.%202009;364(1530):2657%E2%80%9367." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Knowles NJ, Bachanek-Bankowska K, Wadsworth J, Mioulet V, Valdazo‐González B, Eldaghayes IM, Dayhum AS, Kammon AM, Sharif MA, Waight S et al. Outbreaks of Foot‐and‐Mouth disease in Libya and Saudi Arabia during 2013 due to an exotic O/ME‐SA/Ind‐2001 lineage virus. Transbound Emerg Dis. 2016;63(5):e431–5.</cite> [<a href="https://doi.org/10.1111/tbed.12299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25483996/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Bachanek-Bankowska K, Di Nardo A, Wadsworth J, Henry EKM, Parlak Ü, Timina A, Mischenko A, Qasim IA, Abdollahi D, Sultana M, et al. Foot-and-mouth disease in the middle East caused by an A/ASIA/G-VII virus lineage, 2015–2016. Emerg Infect Dis. 2018;24(6):1073–8.
</cite> [<a href="https://doi.org/10.3201/eid2406.170715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6004861/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29774839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bachanek-Bankowska%20K,%20Di%20Nardo%20A,%20Wadsworth%20J,%20Henry%20EKM,%20Parlak%20%C3%9C,%20Timina%20A,%20Mischenko%20A,%20Qasim%20IA,%20Abdollahi%20D,%20Sultana%20M,%20et%20al.%20Foot-and-mouth%20disease%20in%20the%20middle%20East%20caused%20by%20an%20A/ASIA/G-VII%20virus%20lineage,%202015%E2%80%932016.%20Emerg%20Infect%20Dis.%202018;24(6):1073%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Di Nardo A, Ferretti L, Wadsworth J, Mioulet V, Gelman B, Karniely S, Scherbakov A, Ziay G, Ozyoruk F, Parlak U, et al. Evolutionary and ecological drivers shape the emergence and extinction of Foot-and-Mouth disease virus lineages. Mol Biol Evol. 2021;38(10):4346–61.
</cite> [<a href="https://doi.org/10.1093/molbev/msab172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8476141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34115138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Di%20Nardo%20A,%20Ferretti%20L,%20Wadsworth%20J,%20Mioulet%20V,%20Gelman%20B,%20Karniely%20S,%20Scherbakov%20A,%20Ziay%20G,%20Ozyoruk%20F,%20Parlak%20U,%20et%20al.%20Evolutionary%20and%20ecological%20drivers%20shape%20the%20emergence%20and%20extinction%20of%20Foot-and-Mouth%20disease%20virus%20lineages.%20Mol%20Biol%20Evol.%202021;38(10):4346%E2%80%9361." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Eltahir YM, Ishag HZA, Wadsworth J, Hicks HM, Knowles NJ, Mioulet V, King DP, Mohamed MS, Bensalah OK, Yusof MF, et al. Molecular epidemiology of Foot-and-Mouth disease viruses in the emirate of Abu dhabi, united Arab emirates. Veterinary Sci. 2024;11(1):32.</cite> [<a href="https://doi.org/10.3390/vetsci11010032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11154577/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38250938/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eltahir%20YM,%20Ishag%20HZA,%20Wadsworth%20J,%20Hicks%20HM,%20Knowles%20NJ,%20Mioulet%20V,%20King%20DP,%20Mohamed%20MS,%20Bensalah%20OK,%20Yusof%20MF,%20et%20al.%20Molecular%20epidemiology%20of%20Foot-and-Mouth%20disease%20viruses%20in%20the%20emirate%20of%20Abu%20dhabi,%20united%20Arab%20emirates.%20Veterinary%20Sci.%202024;11(1):32." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Zainuddin N, Susila EB, Wibawa H, Daulay RSD, Wijayanti PE, Fitriani D, Hidayati DN, Idris S, Wadsworth J, Polo N, et al. Genome sequence of a foot-and-mouth disease virus detected in Indonesia in 2022. Microbiol Resour Announc. 2023;12(2):e0108122.
</cite> [<a href="https://doi.org/10.1128/mra.01081-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9933659/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36622181/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zainuddin%20N,%20Susila%20EB,%20Wibawa%20H,%20Daulay%20RSD,%20Wijayanti%20PE,%20Fitriani%20D,%20Hidayati%20DN,%20Idris%20S,%20Wadsworth%20J,%20Polo%20N,%20et%20al.%20Genome%20sequence%20of%20a%20foot-and-mouth%20disease%20virus%20detected%20in%20Indonesia%20in%202022.%20Microbiol%20Resour%20Announc.%202023;12(2):e0108122." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Saduakassova MA, Wood BA, Henry E, Gray AR, Mioulet V, Sultanov AA, Wadsworth J, Knowles NJ, Di Nardo A, King DP, et al. Establishing a molecular toolbox of lineage-specific real-time RT-PCR assays for the characterization of foot-and-mouth disease viruses circulating in Asia. Front Vet Sci. 2023;10: 1271690.
</cite> [<a href="https://doi.org/10.3389/fvets.2023.1271690" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10720594/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38098997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saduakassova%20MA,%20Wood%20BA,%20Henry%20E,%20Gray%20AR,%20Mioulet%20V,%20Sultanov%20AA,%20Wadsworth%20J,%20Knowles%20NJ,%20Di%20Nardo%20A,%20King%20DP,%20et%20al.%20Establishing%20a%20molecular%20toolbox%20of%20lineage-specific%20real-time%20RT-PCR%20assays%20for%20the%20characterization%20of%20foot-and-mouth%20disease%20viruses%20circulating%20in%20Asia.%20Front%20Vet%20Sci.%202023;10:%201271690." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Ahmed HA, Salem SAH, Habashi AR, Arafa AA, Aggour MGA, Salem GH, Gaber AS, Selem O, Abdelkader SH, Knowles NJ, et al. Emergence of foot-and-mouth disease virus SAT 2 in Egypt during 2012. Transbound Emerg Dis. 2012;59(6):476–81.
</cite> [<a href="https://doi.org/10.1111/tbed.12015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23025522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahmed%20HA,%20Salem%20SAH,%20Habashi%20AR,%20Arafa%20AA,%20Aggour%20MGA,%20Salem%20GH,%20Gaber%20AS,%20Selem%20O,%20Abdelkader%20SH,%20Knowles%20NJ,%20et%20al.%20Emergence%20of%20foot-and-mouth%20disease%20virus%20SAT%202%20in%20Egypt%20during%202012.%20Transbound%20Emerg%20Dis.%202012;59(6):476%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Saduakassova MA, Sultanov AA, Kutumbetov LB, Wadsworth J, A.Wood B, Knowles NJ, King DP, Bachanek-Bankowska K. Development and evaluation of a novel real-time RT-PCR to detect foot-and-mouth disease viruses from the emerging A/ASIA/G-VII lineage. J Virol Methods. 2018;252:37–41.
</cite> [<a href="https://doi.org/10.1016/j.jviromet.2017.10.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5764150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29113733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saduakassova%20MA,%20Sultanov%20AA,%20Kutumbetov%20LB,%20Wadsworth%20J,%20A.Wood%20B,%20Knowles%20NJ,%20King%20DP,%20Bachanek-Bankowska%20K.%20Development%20and%20evaluation%20of%20a%20novel%20real-time%20RT-PCR%20to%20detect%20foot-and-mouth%20disease%20viruses%20from%20the%20emerging%20A/ASIA/G-VII%20lineage.%20J%20Virol%20Methods.%202018;252:37%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Bachanek-Bankowska K, Mero HR, Wadsworth J, Mioulet V, Sallu R, Belsham GJ, Kasanga CJ, Knowles NJ, King DP. Development and evaluation of tailored specific real-time RT-PCR assays for detection of foot-and-mouth disease virus serotypes circulating in East Africa. J Virol Methods. 2016;237:114–20.
</cite> [<a href="https://doi.org/10.1016/j.jviromet.2016.08.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27575682/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bachanek-Bankowska%20K,%20Mero%20HR,%20Wadsworth%20J,%20Mioulet%20V,%20Sallu%20R,%20Belsham%20GJ,%20Kasanga%20CJ,%20Knowles%20NJ,%20King%20DP.%20Development%20and%20evaluation%20of%20tailored%20specific%20real-time%20RT-PCR%20assays%20for%20detection%20of%20foot-and-mouth%20disease%20virus%20serotypes%20circulating%20in%20East%20Africa.%20J%20Virol%20Methods.%202016;237:114%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Reid SM, Mioulet V, Knowles NJ, Shirazi N, Belsham GJ, King DP. Development of tailored real-time RT-PCR assays for the detection and differentiation of serotype O, A and Asia-1 foot-and-mouth disease virus lineages circulating in the middle East. J Virol Methods. 2014;207:146–53.
</cite> [<a href="https://doi.org/10.1016/j.jviromet.2014.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25016065/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reid%20SM,%20Mioulet%20V,%20Knowles%20NJ,%20Shirazi%20N,%20Belsham%20GJ,%20King%20DP.%20Development%20of%20tailored%20real-time%20RT-PCR%20assays%20for%20the%20detection%20and%20differentiation%20of%20serotype%20O,%20A%20and%20Asia-1%20foot-and-mouth%20disease%20virus%20lineages%20circulating%20in%20the%20middle%20East.%20J%20Virol%20Methods.%202014;207:146%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Jamal SM, Belsham GJ. Development and characterization of probe-based real time quantitative RT-PCR assays for detection and serotyping of foot-and-mouth disease viruses circulating in West Eurasia. PLoS One. 2015;10(8):e0135559.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0135559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4535879/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26270532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jamal%20SM,%20Belsham%20GJ.%20Development%20and%20characterization%20of%20probe-based%20real%20time%20quantitative%20RT-PCR%20assays%20for%20detection%20and%20serotyping%20of%20foot-and-mouth%20disease%20viruses%20circulating%20in%20West%20Eurasia.%20PLoS%20One.%202015;10(8):e0135559." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Pacheco JM, Gladue DP, Holinka LG, Arzt J, Bishop E, Smoliga G, Pauszek SJ, Bracht AJ, O’Donnell V, Fernandez-Sainz I, et al. A partial deletion in non-structural protein 3A can attenuate foot-and-mouth disease virus in cattle. Virology. 2013;446(1–2):260–7.
</cite> [<a href="https://doi.org/10.1016/j.virol.2013.08.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24074589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pacheco%20JM,%20Gladue%20DP,%20Holinka%20LG,%20Arzt%20J,%20Bishop%20E,%20Smoliga%20G,%20Pauszek%20SJ,%20Bracht%20AJ,%20O%E2%80%99Donnell%20V,%20Fernandez-Sainz%20I,%20et%20al.%20A%20partial%20deletion%20in%20non-structural%20protein%203A%20can%20attenuate%20foot-and-mouth%20disease%20virus%20in%20cattle.%20Virology.%202013;446(1%E2%80%932):260%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Ning X, Xia B, Wang J, Gao R, Ren H. Host-adaptive mutations in Chikungunya virus genome. Virulence. 2024. 10.1080/21505594.2024.2401985.
</cite> [<a href="https://doi.org/10.1080/21505594.2024.2401985" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11404619/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39263937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ning%20X,%20Xia%20B,%20Wang%20J,%20Gao%20R,%20Ren%20H.%20Host-adaptive%20mutations%20in%20Chikungunya%20virus%20genome.%20Virulence.%202024.%2010.1080/21505594.2024.2401985." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Cottam EM, Haydon DT, Paton DJ, Gloster J, Wilesmith JW, Ferris NP, Hutchings GH, King DP. Molecular epidemiology of the foot-and-mouth disease virus outbreak in the United Kingdom in 2001. J Virol. 2006;80(22):11274–82.
</cite> [<a href="https://doi.org/10.1128/JVI.01236-06" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1642183/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16971422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cottam%20EM,%20Haydon%20DT,%20Paton%20DJ,%20Gloster%20J,%20Wilesmith%20JW,%20Ferris%20NP,%20Hutchings%20GH,%20King%20DP.%20Molecular%20epidemiology%20of%20the%20foot-and-mouth%20disease%20virus%20outbreak%20in%20the%20United%20Kingdom%20in%202001.%20J%20Virol.%202006;80(22):11274%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Cottam EM, Wadsworth J, Shaw AE, Rowlands RJ, Goatley L, Maan S, Maan NS, Mertens PPC, Ebert K, Li Y, et al. Transmission pathways of foot-and-mouth disease virus in the United Kingdom in 2007. PLoS Pathog. 2008;4(4): e1000050.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1000050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2277462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18421380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cottam%20EM,%20Wadsworth%20J,%20Shaw%20AE,%20Rowlands%20RJ,%20Goatley%20L,%20Maan%20S,%20Maan%20NS,%20Mertens%20PPC,%20Ebert%20K,%20Li%20Y,%20et%20al.%20Transmission%20pathways%20of%20foot-and-mouth%20disease%20virus%20in%20the%20United%20Kingdom%20in%202007.%20PLoS%20Pathog.%202008;4(4):%20e1000050." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Hill SC, de Souza R, Thézé J, Claro I, Aguiar RS, Abade L, Santos FCP, Cunha MS, Nogueira JS, Salles FCS, et al. Genomic surveillance of yellow fever virus epizootic in São Paulo, Brazil, 2016–2018. PLoS Pathog. 2020;16(8):e1008699.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1008699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7437926/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32764827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hill%20SC,%20de%20Souza%20R,%20Th%C3%A9z%C3%A9%20J,%20Claro%20I,%20Aguiar%20RS,%20Abade%20L,%20Santos%20FCP,%20Cunha%20MS,%20Nogueira%20JS,%20Salles%20FCS,%20et%20al.%20Genomic%20surveillance%20of%20yellow%20fever%20virus%20epizootic%20in%20S%C3%A3o%20Paulo,%20Brazil,%202016%E2%80%932018.%20PLoS%20Pathog.%202020;16(8):e1008699." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Faria NR, Kraemer MUG, Hill SC, Goes de Jesus J, Aguiar RS, Iani FCM, Xavier J, Quick J, du Plessis L, Dellicour S, et al. Genomic and epidemiological monitoring of yellow fever virus transmission potential. Science. 2018;361(6405):894–9.
</cite> [<a href="https://doi.org/10.1126/science.aat7115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6874500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30139911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Faria%20NR,%20Kraemer%20MUG,%20Hill%20SC,%20Goes%20de%20Jesus%20J,%20Aguiar%20RS,%20Iani%20FCM,%20Xavier%20J,%20Quick%20J,%20du%20Plessis%20L,%20Dellicour%20S,%20et%20al.%20Genomic%20and%20epidemiological%20monitoring%20of%20yellow%20fever%20virus%20transmission%20potential.%20Science.%202018;361(6405):894%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Carroll MW, Matthews DA, Hiscox JA, Elmore MJ, Pollakis G, Rambaut A, Hewson R, García-Dorival I, Bore JA, Koundouno R, et al. Temporal and spatial analysis of the 2014–2015 Ebola virus outbreak in West Africa. Nature. 2015;524(7563):97–101.
</cite> [<a href="https://doi.org/10.1038/nature14594" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10601607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26083749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carroll%20MW,%20Matthews%20DA,%20Hiscox%20JA,%20Elmore%20MJ,%20Pollakis%20G,%20Rambaut%20A,%20Hewson%20R,%20Garc%C3%ADa-Dorival%20I,%20Bore%20JA,%20Koundouno%20R,%20et%20al.%20Temporal%20and%20spatial%20analysis%20of%20the%202014%E2%80%932015%20Ebola%20virus%20outbreak%20in%20West%20Africa.%20Nature.%202015;524(7563):97%E2%80%93101." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Dudas G, Carvalho LM, Bedford T, Tatem AJ, Baele G, Faria NR, Park DJ, Ladner JT, Arias A, Asogun D, et al. Virus genomes reveal factors that spread and sustained the Ebola epidemic. Nature. 2017;544(7650):309–15.
</cite> [<a href="https://doi.org/10.1038/nature22040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5712493/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28405027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dudas%20G,%20Carvalho%20LM,%20Bedford%20T,%20Tatem%20AJ,%20Baele%20G,%20Faria%20NR,%20Park%20DJ,%20Ladner%20JT,%20Arias%20A,%20Asogun%20D,%20et%20al.%20Virus%20genomes%20reveal%20factors%20that%20spread%20and%20sustained%20the%20Ebola%20epidemic.%20Nature.%202017;544(7650):309%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Grubaugh ND, Ladner JT, Lemey P, Pybus OG, Rambaut A, Holmes EC, Andersen KG. Tracking virus outbreaks in the twenty-first century. Nat Microbiol. 2019;4(1):10–9.
</cite> [<a href="https://doi.org/10.1038/s41564-018-0296-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6345516/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30546099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grubaugh%20ND,%20Ladner%20JT,%20Lemey%20P,%20Pybus%20OG,%20Rambaut%20A,%20Holmes%20EC,%20Andersen%20KG.%20Tracking%20virus%20outbreaks%20in%20the%20twenty-first%20century.%20Nat%20Microbiol.%202019;4(1):10%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Ladner JT, Grubaugh ND, Pybus OG, Andersen KG. Precision epidemiology for infectious disease control. Nat Med. 2019;25(2):206–11.
</cite> [<a href="https://doi.org/10.1038/s41591-019-0345-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7095960/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30728537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ladner%20JT,%20Grubaugh%20ND,%20Pybus%20OG,%20Andersen%20KG.%20Precision%20epidemiology%20for%20infectious%20disease%20control.%20Nat%20Med.%202019;25(2):206%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Vogels CBF, Hill V, Breban MI, Chaguza C, Paul LM, Sodeinde A, Taylor-Salmon E, Ott IM, Petrone ME, Dijk D, et al. DengueSeq: a pan-serotype whole genome amplicon sequencing protocol for dengue virus. BMC Genomics. 2024;25(1): 433.
</cite> [<a href="https://doi.org/10.1186/s12864-024-10350-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11062901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38693476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vogels%20CBF,%20Hill%20V,%20Breban%20MI,%20Chaguza%20C,%20Paul%20LM,%20Sodeinde%20A,%20Taylor-Salmon%20E,%20Ott%20IM,%20Petrone%20ME,%20Dijk%20D,%20et%20al.%20DengueSeq:%20a%20pan-serotype%20whole%20genome%20amplicon%20sequencing%20protocol%20for%20dengue%20virus.%20BMC%20Genomics.%202024;25(1):%20433." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Van Borm S, Ahrens AK, Bachofen C, Banyard AC, Boe CA, Briand FX, Dirbakova Z, Engelsma M, Fusaro A, Germeraad E, et al. Genesis and spread of novel highly pathogenic avian influenza A(H5N1) clade 2.3.4.4b virus genotype EA-2023-DG reassortant, Western Europe. Emerg Infect Dis. 2025;31(6):1100–8.
</cite> [<a href="https://doi.org/10.3201/eid3106.241870" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12123936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40323677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20Borm%20S,%20Ahrens%20AK,%20Bachofen%20C,%20Banyard%20AC,%20Boe%20CA,%20Briand%20FX,%20Dirbakova%20Z,%20Engelsma%20M,%20Fusaro%20A,%20Germeraad%20E,%20et%20al.%20Genesis%20and%20spread%20of%20novel%20highly%20pathogenic%20avian%20influenza%20A(H5N1)%20clade%202.3.4.4b%20virus%20genotype%20EA-2023-DG%20reassortant,%20Western%20Europe.%20Emerg%20Infect%20Dis.%202025;31(6):1100%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Knowles NJ, Wadsworth J, Bachanek-Bankowska K, King DP. VP1 sequencing protocol for foot and mouth disease virus molecular epidemiology. OIE Revue Scientifique Et Technique. 2016;35(3):741–55.</cite> [<a href="https://doi.org/10.20506/rst.35.3.2565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28332654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Knowles%20NJ,%20Wadsworth%20J,%20Bachanek-Bankowska%20K,%20King%20DP.%20VP1%20sequencing%20protocol%20for%20foot%20and%20mouth%20disease%20virus%20molecular%20epidemiology.%20OIE%20Revue%20Scientifique%20Et%20Technique.%202016;35(3):741%E2%80%9355." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Logan G, Freimanis GL, King DJ, Valdazo-González B, Bacha nek-Bankowska K, Sanderson ND, Knowles NJ, King DP, Cottam EM. A universal protocol to generate consensus level genome sequences for foot-and-mouth disease virus and other positive-sense polyadenylated RNA viruses using the illumina miseq. BMC Genomics. 2014. 10.1186/1471-2164-15-828.
</cite> [<a href="https://doi.org/10.1186/1471-2164-15-828" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4247156/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25269623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Logan%20G,%20Freimanis%20GL,%20King%20DJ,%20Valdazo-Gonz%C3%A1lez%20B,%20Bacha%20nek-Bankowska%20K,%20Sanderson%20ND,%20Knowles%20NJ,%20King%20DP,%20Cottam%20EM.%20A%20universal%20protocol%20to%20generate%20consensus%20level%20genome%20sequences%20for%20foot-and-mouth%20disease%20virus%20and%20other%20positive-sense%20polyadenylated%20RNA%20viruses%20using%20the%20illumina%20miseq.%20BMC%20Genomics.%202014.%2010.1186/1471-2164-15-828." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Brown E, Freimanis G, Shaw AE, Horton DL, Gubbins S, King D. Characterising Foot-and-Mouth disease virus in clinical samples using nanopore sequencing. Front Veterinary Sci. 2021;8:656256.</cite> [<a href="https://doi.org/10.3389/fvets.2021.656256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8165188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34079833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown%20E,%20Freimanis%20G,%20Shaw%20AE,%20Horton%20DL,%20Gubbins%20S,%20King%20D.%20Characterising%20Foot-and-Mouth%20disease%20virus%20in%20clinical%20samples%20using%20nanopore%20sequencing.%20Front%20Veterinary%20Sci.%202021;8:656256." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Hansen S, Dill V, Shalaby MA, Eschbaumer M, Böhlken-Fascher S, Hoffmann B, Czerny C-P. Abd El wahed A: serotyping of foot-and-mouth disease virus using Oxford nanopore sequencing. J Virol Methods. 2019;263:50–3.
</cite> [<a href="https://doi.org/10.1016/j.jviromet.2018.10.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30393148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hansen%20S,%20Dill%20V,%20Shalaby%20MA,%20Eschbaumer%20M,%20B%C3%B6hlken-Fascher%20S,%20Hoffmann%20B,%20Czerny%20C-P.%20Abd%20El%20wahed%20A:%20serotyping%20of%20foot-and-mouth%20disease%20virus%20using%20Oxford%20nanopore%20sequencing.%20J%20Virol%20Methods.%202019;263:50%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Bold D, Souza-Neto JA, Gombo-Ochir D, Gaudreault NN, Meekins DA, McDowell CD, Zayat B, Richt JA. Rapid identification of ASFV, CSFV and FMDV from Mongolian outbreaks with minion short amplicon sequencing. Pathogens. 2023;12(4): 533.
</cite> [<a href="https://doi.org/10.3390/pathogens12040533" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10140976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37111419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bold%20D,%20Souza-Neto%20JA,%20Gombo-Ochir%20D,%20Gaudreault%20NN,%20Meekins%20DA,%20McDowell%20CD,%20Zayat%20B,%20Richt%20JA.%20Rapid%20identification%20of%20ASFV,%20CSFV%20and%20FMDV%20from%20Mongolian%20outbreaks%20with%20minion%20short%20amplicon%20sequencing.%20Pathogens.%202023;12(4):%20533." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Wang S, Wei S, Ding Y, Zhang Y, Zhang Z, Sun S, Guo H, Yin S. Analysis of coinfection pathogens from Foot-and-Mouth disease virus persistently infected cattle using Oxford nanopore sequencing. Transbound Emerg Dis. 2024;2024(1):9703014.
</cite> [<a href="https://doi.org/10.1155/2024/9703014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12016752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40303157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20S,%20Wei%20S,%20Ding%20Y,%20Zhang%20Y,%20Zhang%20Z,%20Sun%20S,%20Guo%20H,%20Yin%20S.%20Analysis%20of%20coinfection%20pathogens%20from%20Foot-and-Mouth%20disease%20virus%20persistently%20infected%20cattle%20using%20Oxford%20nanopore%20sequencing.%20Transbound%20Emerg%20Dis.%202024;2024(1):9703014." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Xu L, Hurtle W, Rowland JM, Casteran KA, Bucko SM, Grau FR, Valdazo-González B, Knowles NJ, King DP, Beckham TR, et al. Development of a universal RT-PCR for amplifying and sequencing the leader and capsid-coding region of foot-and-mouth disease virus. J Virol Methods. 2013;189(1):70–6.
</cite> [<a href="https://doi.org/10.1016/j.jviromet.2013.01.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23380590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20L,%20Hurtle%20W,%20Rowland%20JM,%20Casteran%20KA,%20Bucko%20SM,%20Grau%20FR,%20Valdazo-Gonz%C3%A1lez%20B,%20Knowles%20NJ,%20King%20DP,%20Beckham%20TR,%20et%20al.%20Development%20of%20a%20universal%20RT-PCR%20for%20amplifying%20and%20sequencing%20the%20leader%20and%20capsid-coding%20region%20of%20foot-and-mouth%20disease%20virus.%20J%20Virol%20Methods.%202013;189(1):70%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Honeycutt E, Kizito F, Karn J, Sweet T. Direct analysis of HIV mRNA m6A methylation by nanopore sequencing. In; 2024: 209–27.</cite> [<a href="https://doi.org/10.1007/978-1-0716-3862-0_15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12120845/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38743231/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Keller MW, Rambo-Martin BL, Wilson MM, Ridenour CA, Shepard SS, Stark TJ, Neuhaus EB, Dugan VG, Wentworth DE, Barnes JR. Direct RNA sequencing of the coding complete influenza A virus genome. Sci Rep. 2018;8(1):14408.
</cite> [<a href="https://doi.org/10.1038/s41598-018-32615-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6158192/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30258076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keller%20MW,%20Rambo-Martin%20BL,%20Wilson%20MM,%20Ridenour%20CA,%20Shepard%20SS,%20Stark%20TJ,%20Neuhaus%20EB,%20Dugan%20VG,%20Wentworth%20DE,%20Barnes%20JR.%20Direct%20RNA%20sequencing%20of%20the%20coding%20complete%20influenza%20A%20virus%20genome.%20Sci%20Rep.%202018;8(1):14408." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Ji C-M, Feng X-Y, Huang Y-W, Chen R-A. The applications of nanopore sequencing technology in animal and human virus research. Viruses. 2024;16(5):798.
</cite> [<a href="https://doi.org/10.3390/v16050798" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11125791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38793679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ji%20C-M,%20Feng%20X-Y,%20Huang%20Y-W,%20Chen%20R-A.%20The%20applications%20of%20nanopore%20sequencing%20technology%20in%20animal%20and%20human%20virus%20research.%20Viruses.%202024;16(5):798." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Quick J, Grubaugh ND, Pullan ST, Claro IM, Smith AD, Gangavarapu K, Oliveira G, Robles-Sikisaka R, Rogers TF, Beutler NA, et al. Multiplex PCR method for minion and illumina sequencing of Zika and other virus genomes directly from clinical samples. Nat Protoc. 2017;12(6):1261–76.
</cite> [<a href="https://doi.org/10.1038/nprot.2017.066" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5902022/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28538739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quick%20J,%20Grubaugh%20ND,%20Pullan%20ST,%20Claro%20IM,%20Smith%20AD,%20Gangavarapu%20K,%20Oliveira%20G,%20Robles-Sikisaka%20R,%20Rogers%20TF,%20Beutler%20NA,%20et%20al.%20Multiplex%20PCR%20method%20for%20minion%20and%20illumina%20sequencing%20of%20Zika%20and%20other%20virus%20genomes%20directly%20from%20clinical%20samples.%20Nat%20Protoc.%202017;12(6):1261%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Brunker K, Jaswant G, Thumbi SM, Lushasi K, Lugelo A, Czupryna AM, Ade F, Wambura G, Chuchu V, Steenson R, et al. Rapid in-country sequencing of whole virus genomes to inform rabies elimination programmes. Wellcome Open Res. 2020;5:3.
</cite> [<a href="https://doi.org/10.12688/wellcomeopenres.15518.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7001756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32090172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brunker%20K,%20Jaswant%20G,%20Thumbi%20SM,%20Lushasi%20K,%20Lugelo%20A,%20Czupryna%20AM,%20Ade%20F,%20Wambura%20G,%20Chuchu%20V,%20Steenson%20R,%20et%20al.%20Rapid%20in-country%20sequencing%20of%20whole%20virus%20genomes%20to%20inform%20rabies%20elimination%20programmes.%20Wellcome%20Open%20Res.%202020;5:3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Quick J, Loman NJ, Duraffour S, Simpson JT, Severi E, Cowley L, Bore JA, Koundouno R, Dudas G, Mikhail A, et al. Real-time, portable genome sequencing for Ebola surveillance. Nature. 2016;530(7589):228–32.
</cite> [<a href="https://doi.org/10.1038/nature16996" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4817224/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26840485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quick%20J,%20Loman%20NJ,%20Duraffour%20S,%20Simpson%20JT,%20Severi%20E,%20Cowley%20L,%20Bore%20JA,%20Koundouno%20R,%20Dudas%20G,%20Mikhail%20A,%20et%20al.%20Real-time,%20portable%20genome%20sequencing%20for%20Ebola%20surveillance.%20Nature.%202016;530(7589):228%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Freed NE, Vlková M, Faisal MB, Silander OK. Rapid and inexpensive whole-genome sequencing of SARS-CoV-2 using 1200 bp tiled amplicons and Oxford Nanopore Rapid Barcoding. Biology Methods Protocols. 2020;5(1):1–7. </cite> [<a href="https://doi.org/10.1093/biomethods/bpaa014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7454405/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33029559/" class="usa-link">PubMed</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Casey-Bryars M, Reeve R, Bastola U, Knowles NJ, Auty H, Bachanek-Bankowska K, Fowler VL, Fyumagwa R, Kazwala R, Kibona T, et al. Waves of endemic foot-and-mouth disease in Eastern Africa suggest feasibility of proactive vaccination approaches. Nat Ecol Evol. 2018;2(9):1449–57.
</cite> [<a href="https://doi.org/10.1038/s41559-018-0636-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30082738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Casey-Bryars%20M,%20Reeve%20R,%20Bastola%20U,%20Knowles%20NJ,%20Auty%20H,%20Bachanek-Bankowska%20K,%20Fowler%20VL,%20Fyumagwa%20R,%20Kazwala%20R,%20Kibona%20T,%20et%20al.%20Waves%20of%20endemic%20foot-and-mouth%20disease%20in%20Eastern%20Africa%20suggest%20feasibility%20of%20proactive%20vaccination%20approaches.%20Nat%20Ecol%20Evol.%202018;2(9):1449%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Di Nardo A, Shaw AE, Gondard M, Wadsworth J, Girault G, Parekh K, Ludi A, Mioulet V, Bernelin-Cottet C, Hicks HM, et al. Eastern Africa origin of SAT2 topotype XIV foot-and-mouth disease virus outbreaks, Western Asia, 2023. Emerg Infect Dis. 2025;31(2):368–72.
</cite> [<a href="https://doi.org/10.3201/eid3102.240395" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11845145/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39983696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Di%20Nardo%20A,%20Shaw%20AE,%20Gondard%20M,%20Wadsworth%20J,%20Girault%20G,%20Parekh%20K,%20Ludi%20A,%20Mioulet%20V,%20Bernelin-Cottet%20C,%20Hicks%20HM,%20et%20al.%20Eastern%20Africa%20origin%20of%20SAT2%20topotype%20XIV%20foot-and-mouth%20disease%20virus%20outbreaks,%20Western%20Asia,%202023.%20Emerg%20Infect%20Dis.%202025;31(2):368%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Dill V, Beer M, Hoffmann B. Simple, quick and cost-efficient: a universal RT-PCR and sequencing strategy for genomic characterisation of foot-and-mouth disease viruses. J Virol Methods. 2017;246:58–64.
</cite> [<a href="https://doi.org/10.1016/j.jviromet.2017.04.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28442328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dill%20V,%20Beer%20M,%20Hoffmann%20B.%20Simple,%20quick%20and%20cost-efficient:%20a%20universal%20RT-PCR%20and%20sequencing%20strategy%20for%20genomic%20characterisation%20of%20foot-and-mouth%20disease%20viruses.%20J%20Virol%20Methods.%202017;246:58%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Kim BY, Gellert HR, Church SH, Suvorov A, Anderson SS, Barmina O, Beskid SG, Comeault AA, Crown KN, Diamond SE, et al. Single-fly genome assemblies fill major phylogenomic gaps across the drosophilidae tree of life. PLoS Biol. 2024;22(7):e3002697.
</cite> [<a href="https://doi.org/10.1371/journal.pbio.3002697" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11257246/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39024225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20BY,%20Gellert%20HR,%20Church%20SH,%20Suvorov%20A,%20Anderson%20SS,%20Barmina%20O,%20Beskid%20SG,%20Comeault%20AA,%20Crown%20KN,%20Diamond%20SE,%20et%20al.%20Single-fly%20genome%20assemblies%20fill%20major%20phylogenomic%20gaps%20across%20the%20drosophilidae%20tree%20of%20life.%20PLoS%20Biol.%202024;22(7):e3002697." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Knowles NJ, Davies PR, Henry T, O’Donnell V, Pacheco JM, Mason PW. Emergence in Asia of foot-and-mouth disease viruses with altered host range: characterization of alterations in the 3A protein. J Virol. 2001;75(3):1551–6.
</cite> [<a href="https://doi.org/10.1128/JVI.75.3.1551-1556.2001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC114061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11152528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Knowles%20NJ,%20Davies%20PR,%20Henry%20T,%20O%E2%80%99Donnell%20V,%20Pacheco%20JM,%20Mason%20PW.%20Emergence%20in%20Asia%20of%20foot-and-mouth%20disease%20viruses%20with%20altered%20host%20range:%20characterization%20of%20alterations%20in%20the%203A%20protein.%20J%20Virol.%202001;75(3):1551%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Yang F, Zhu Z, Cao W, Liu H, Wei T, Zheng M, Zhang K, Jin Y, He J, Guo J, et al. Genetic determinants of altered virulence of type O foot-and-mouth disease virus. J Virol. 2020. 10.1128/JVI.01657-19.
</cite> [<a href="https://doi.org/10.1128/JVI.01657-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7081894/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31915277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20F,%20Zhu%20Z,%20Cao%20W,%20Liu%20H,%20Wei%20T,%20Zheng%20M,%20Zhang%20K,%20Jin%20Y,%20He%20J,%20Guo%20J,%20et%20al.%20Genetic%20determinants%20of%20altered%20virulence%20of%20type%20O%20foot-and-mouth%20disease%20virus.%20J%20Virol.%202020.%2010.1128/JVI.01657-19." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Ward JC, Lasecka-Dykes L, Dobson SJ, Gold S, Kingston NJ, Herod MR, King DP, Tuthill TJ, Rowlands DJ, Stonehouse NJ. The dual role of a highly structured RNA (the S fragment) in the replication of foot-and-mouth disease virus. FASEB J. 2024;38(14):e23822.
</cite> [<a href="https://doi.org/10.1096/fj.202400500R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39072864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ward%20JC,%20Lasecka-Dykes%20L,%20Dobson%20SJ,%20Gold%20S,%20Kingston%20NJ,%20Herod%20MR,%20King%20DP,%20Tuthill%20TJ,%20Rowlands%20DJ,%20Stonehouse%20NJ.%20The%20dual%20role%20of%20a%20highly%20structured%20RNA%20(the%20S%20fragment)%20in%20the%20replication%20of%20foot-and-mouth%20disease%20virus.%20FASEB%20J.%202024;38(14):e23822." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Tyson JR, James P, Stoddart D, Sparks N, Wickenhagen A, Hall G, Choi JH, Lapointe H, Kamelian K, Smith AD et al. Improvements to the ARTIC multiplex PCR method for SARS-CoV-2 genome sequencing using nanopore. In.; 2020.</cite>
</li>
<li id="CR49">
<span class="label">49.</span><cite>Colenutt C, Shaw A, Esemu SN, Kfusi AJ, Willington Ojong B, Brown E, Wadsworth J, Knowles NJ, King DP, Ndip LM, et al. Detection and genomic characterisation of foot-and-mouth disease virus serotypes circulating in Cameroon using environmental sampling. Sci Rep. 2025;15(1):2834.
</cite> [<a href="https://doi.org/10.1038/s41598-024-84724-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11882811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39843930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Colenutt%20C,%20Shaw%20A,%20Esemu%20SN,%20Kfusi%20AJ,%20Willington%20Ojong%20B,%20Brown%20E,%20Wadsworth%20J,%20Knowles%20NJ,%20King%20DP,%20Ndip%20LM,%20et%20al.%20Detection%20and%20genomic%20characterisation%20of%20foot-and-mouth%20disease%20virus%20serotypes%20circulating%20in%20Cameroon%20using%20environmental%20sampling.%20Sci%20Rep.%202025;15(1):2834." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Gubbins S, Brown E, Wungak Y, Oyekan O, Adedeji AJ, Ijoma SI, Atai RB, Oguche MO, Samson M, Dogonyaro BB, et al. Longitudinal study of foot-and-mouth disease virus in Northern Nigeria: implications for the roles of small ruminants and environmental contamination in endemic settings. Vet Res. 2025;56(1): 76.
</cite> [<a href="https://doi.org/10.1186/s13567-025-01502-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11969707/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40181358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gubbins%20S,%20Brown%20E,%20Wungak%20Y,%20Oyekan%20O,%20Adedeji%20AJ,%20Ijoma%20SI,%20Atai%20RB,%20Oguche%20MO,%20Samson%20M,%20Dogonyaro%20BB,%20et%20al.%20Longitudinal%20study%20of%20foot-and-mouth%20disease%20virus%20in%20Northern%20Nigeria:%20implications%20for%20the%20roles%20of%20small%20ruminants%20and%20environmental%20contamination%20in%20endemic%20settings.%20Vet%20Res.%202025;56(1):%2076." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Carrillo C, Tulman ER, Delhon G, Lu Z, Carreno A, Vagnozzi A, Kutish GF, Rock DL. Comparative genomics of foot-and-mouth disease virus. J Virol. 2005;79(10):6487–504.
</cite> [<a href="https://doi.org/10.1128/JVI.79.10.6487-6504.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1091679/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15858032/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carrillo%20C,%20Tulman%20ER,%20Delhon%20G,%20Lu%20Z,%20Carreno%20A,%20Vagnozzi%20A,%20Kutish%20GF,%20Rock%20DL.%20Comparative%20genomics%20of%20foot-and-mouth%20disease%20virus.%20J%20Virol.%202005;79(10):6487%E2%80%93504." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Ward JC, Lasecka-Dykes L, Neil C, Adeyemi OO, Gold S, McLean-Pell N, Wright C, Herod MR, Kealy D, Warner E, et al. The RNA pseudoknots in foot-and-mouth disease virus are dispensable for genome replication, but essential for the production of infectious virus. PLoS Pathog. 2022;18(6):e1010589.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1010589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9203018/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35666744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ward%20JC,%20Lasecka-Dykes%20L,%20Neil%20C,%20Adeyemi%20OO,%20Gold%20S,%20McLean-Pell%20N,%20Wright%20C,%20Herod%20MR,%20Kealy%20D,%20Warner%20E,%20et%20al.%20The%20RNA%20pseudoknots%20in%20foot-and-mouth%20disease%20virus%20are%20dispensable%20for%20genome%20replication,%20but%20essential%20for%20the%20production%20of%20infectious%20virus.%20PLoS%20Pathog.%202022;18(6):e1010589." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Colenutt C, Brown E, Nelson N, Wadsworth J, Maud J, Adhikari B, Chapagain Kafle S, Upadhyaya M, Kafle Pandey S, Paton DJ, et al. Environmental sampling as a low-technology method for surveillance of foot-and-mouth disease virus in an area of endemicity. Appl Environ Microbiol. 2018;84(16):e00686-00618.
</cite> [<a href="https://doi.org/10.1128/AEM.00686-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6070757/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29959244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Colenutt%20C,%20Brown%20E,%20Nelson%20N,%20Wadsworth%20J,%20Maud%20J,%20Adhikari%20B,%20Chapagain%20Kafle%20S,%20Upadhyaya%20M,%20Kafle%20Pandey%20S,%20Paton%20DJ,%20et%20al.%20Environmental%20sampling%20as%20a%20low-technology%20method%20for%20surveillance%20of%20foot-and-mouth%20disease%20virus%20in%20an%20area%20of%20endemicity.%20Appl%20Environ%20Microbiol.%202018;84(16):e00686-00618." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Colenutt C, Brown E, Paton DJ, Mahapatra M, Parida S, Nelson N, Maud J, Motta P, Sumption K, Adhikari B, et al. Environmental sampling for the detection of foot-and‐mouth disease virus and peste des petits ruminants virus in a live goat market, Nepal. Transbound Emerg Dis. 2022;69(5):3041–6.
</cite> [<a href="https://doi.org/10.1111/tbed.14257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34331827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Colenutt%20C,%20Brown%20E,%20Paton%20DJ,%20Mahapatra%20M,%20Parida%20S,%20Nelson%20N,%20Maud%20J,%20Motta%20P,%20Sumption%20K,%20Adhikari%20B,%20et%20al.%20Environmental%20sampling%20for%20the%20detection%20of%20foot-and%E2%80%90mouth%20disease%20virus%20and%20peste%20des%20petits%20ruminants%20virus%20in%20a%20live%20goat%20market,%20Nepal.%20Transbound%20Emerg%20Dis.%202022;69(5):3041%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Colenutt C, Brown E, Nelson N, Paton DJ, Eblé P, Dekker A, Gonzales JL, Gubbins S. Quantifying the transmission of Foot-and-Mouth disease virus in cattle via a contaminated environment. mBio. 2020;11(4):e00381–00320.
</cite> [<a href="https://doi.org/10.1128/mBio.00381-20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7407078/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32753488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Colenutt%20C,%20Brown%20E,%20Nelson%20N,%20Paton%20DJ,%20Ebl%C3%A9%20P,%20Dekker%20A,%20Gonzales%20JL,%20Gubbins%20S.%20Quantifying%20the%20transmission%20of%20Foot-and-Mouth%20disease%20virus%20in%20cattle%20via%20a%20contaminated%20environment.%20mBio.%202020;11(4):e00381%E2%80%9300320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Callahan JD, Brown F, Osorio FA, Sur JH, Kramer E, Long GW, Lubroth J, Ellis SJ, Shoulars KS, Gaffney KL, et al. Use of a portable real-time reverse transcriptase polymerase chain reaction assay for rapid detection of foot-and-mouth disease virus. J Am Vet Med Assoc. 2002;220(11):1636–42.
</cite> [<a href="https://doi.org/10.2460/javma.2002.220.1636" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12051502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Callahan%20JD,%20Brown%20F,%20Osorio%20FA,%20Sur%20JH,%20Kramer%20E,%20Long%20GW,%20Lubroth%20J,%20Ellis%20SJ,%20Shoulars%20KS,%20Gaffney%20KL,%20et%20al.%20Use%20of%20a%20portable%20real-time%20reverse%20transcriptase%20polymerase%20chain%20reaction%20assay%20for%20rapid%20detection%20of%20foot-and-mouth%20disease%20virus.%20J%20Am%20Vet%20Med%20Assoc.%202002;220(11):1636%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Li H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics. 2018;34(18):3094–100.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6137996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29750242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H.%20Minimap2:%20pairwise%20alignment%20for%20nucleotide%20sequences.%20Bioinformatics.%202018;34(18):3094%E2%80%93100." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Howson ELA, Orton RJ, Mioulet V, Lembo T, King DP, Fowler VL. Goprime: development of an in silico framework to predict the performance of real-time PCR primers and probes using Foot-and-Mouth disease virus as a model. Pathogens. 2020. 10.3390/pathogens9040303.
</cite> [<a href="https://doi.org/10.3390/pathogens9040303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7238122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32326039/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Howson%20ELA,%20Orton%20RJ,%20Mioulet%20V,%20Lembo%20T,%20King%20DP,%20Fowler%20VL.%20Goprime:%20development%20of%20an%20in%20silico%20framework%20to%20predict%20the%20performance%20of%20real-time%20PCR%20primers%20and%20probes%20using%20Foot-and-Mouth%20disease%20virus%20as%20a%20model.%20Pathogens.%202020.%2010.3390/pathogens9040303." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Danecek P, Bonfield JK, Liddle J, Marshall J, Ohan V, Pollard MO, Whitwham A, Keane T, McCarthy SA, Davies RM, et al. Twelve years of SAMtools and BCFtools. Gigascience. 2021;10(2): giab008.
</cite> [<a href="https://doi.org/10.1093/gigascience/giab008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7931819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33590861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Danecek%20P,%20Bonfield%20JK,%20Liddle%20J,%20Marshall%20J,%20Ohan%20V,%20Pollard%20MO,%20Whitwham%20A,%20Keane%20T,%20McCarthy%20SA,%20Davies%20RM,%20et%20al.%20Twelve%20years%20of%20SAMtools%20and%20BCFtools.%20Gigascience.%202021;10(2):%20giab008." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Camacho C, Coulouris G, Avagyan V, Ma N, Papadopoulos J, Bealer K, Madden TL. BLAST+: architecture and applications. BMC Bioinformatics. 2009;10:421.
</cite> [<a href="https://doi.org/10.1186/1471-2105-10-421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2803857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20003500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Camacho%20C,%20Coulouris%20G,%20Avagyan%20V,%20Ma%20N,%20Papadopoulos%20J,%20Bealer%20K,%20Madden%20TL.%20BLAST+:%20architecture%20and%20applications.%20BMC%20Bioinformatics.%202009;10:421." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Team RC. R: A Language and Environment for Statistical Computing. In. Edited by Computing RFfS; 2021.</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (61.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (23.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3.</a><sup> (594.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4.</a><sup> (2.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5.</a><sup> (2.7MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6.</a><sup> (90.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7.</a><sup> (50.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 8.</a><sup> (126.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 9.</a><sup> (34.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM10_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 10.</a><sup> (38.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM11_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 11.</a><sup> (59.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM12_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 12.</a><sup> (134.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM13_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 13.</a><sup> (2.3MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material14_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM14_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 14.</a><sup> (2.4MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material15_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM15_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 15</a><sup> (3.1MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material16_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM16_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 16.</a><sup> (2.9MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material17_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM17_ESM.png" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 17.</a><sup> (2.9MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material18_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372193/bin/12864_2025_11938_MOESM18_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 18.</a><sup> (16.9KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Sequence data generated during the validation of this method have been submitted to GenBank and are available under accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PV524741:PV524918%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV524741- PV524918</a>. Further sequences generated in Uganda have been submitted to GenBank and are available under accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PV883408:PV883420%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV883408 - PV883420</a>. A set of FASTQ files has been uploaded to the short read archive and are available under accession number PRJNA1286760.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Genomics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12864-025-11938-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12864_2025_Article_11938.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12372193/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12372193/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372193%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372193/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12372193/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12372193/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846908/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12372193/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846908/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12372193/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12372193/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="pA41cbrjbzLtLRXMs72VYH4iwlHUZckhqDLaFfZ59MwQEP1cwztDJPPgSFXM2iln">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
