
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            The promise and peril of comparing fluorescence lifetime in biology revealed by simulations - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5335AB8AF2BA630535AB006CE9178D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="elife">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367299/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="eLife">
<meta name="citation_title" content="The promise and peril of comparing fluorescence lifetime in biology revealed by simulations">
<meta name="citation_author" content="Pingchuan Ma">
<meta name="citation_author_institution" content="Department of Neuroscience, Washington University in St. Louis, St. Louis, United States">
<meta name="citation_author_institution" content="Ph.D. Program in Neuroscience, Washington University in St. Louis, St. Louis, United States">
<meta name="citation_author" content="Peter Chen">
<meta name="citation_author_institution" content="Department of Neuroscience, Washington University in St. Louis, St. Louis, United States">
<meta name="citation_author_institution" content="Master’s Program in Biomedical Engineering, Washington University in St. Louis, St. Louis, United States">
<meta name="citation_author" content="Scott Sternson">
<meta name="citation_author_institution" content="Department of Neuroscience, Howard Hughes Medical Institute, University of California, San Diego, San Diego, United States">
<meta name="citation_author" content="Yao Chen">
<meta name="citation_author_institution" content="Department of Neuroscience, Washington University in St. Louis, St. Louis, United States">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="13">
<meta name="citation_firstpage" content="RP101559">
<meta name="citation_doi" content="10.7554/eLife.101559">
<meta name="citation_pmid" content="40832998">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367299/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367299/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367299/pdf/elife-101559.pdf">
<meta name="description" content="Signaling dynamics are crucial in biological systems, and biosensor-based real-time imaging has revolutionized their analysis. Fluorescence lifetime imaging microscopy (FLIM) excels over the widely used fluorescence intensity imaging by allowing the ...">
<meta name="og:title" content="The promise and peril of comparing fluorescence lifetime in biology revealed by simulations">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Signaling dynamics are crucial in biological systems, and biosensor-based real-time imaging has revolutionized their analysis. Fluorescence lifetime imaging microscopy (FLIM) excels over the widely used fluorescence intensity imaging by allowing the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367299/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12367299">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.7554/eLife.101559"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/elife-101559.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367299%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12367299/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12367299/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367299/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-elife.png" alt="eLife logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to eLife" title="Link to eLife" shape="default" href="https://doi.org/10.7554/eLife.101559" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">eLife</button></div>. 2025 Aug 20;13:RP101559. doi: <a href="https://doi.org/10.7554/eLife.101559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.101559</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22eLife%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22eLife%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22eLife%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22eLife%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>The promise and peril of comparing fluorescence lifetime in biology revealed by simulations</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Pingchuan Ma</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Pingchuan Ma</span></h3>
<div class="p">
<sup>1</sup>Department of Neuroscience, Washington University in St. Louis, St. Louis, United States</div>
<div class="p">
<sup>2</sup>Ph.D. Program in Neuroscience, Washington University in St. Louis, St. Louis, United States</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pingchuan Ma</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Peter Chen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Peter Chen</span></h3>
<div class="p">
<sup>1</sup>Department of Neuroscience, Washington University in St. Louis, St. Louis, United States</div>
<div class="p">
<sup>3</sup>Master’s Program in Biomedical Engineering, Washington University in St. Louis, St. Louis, United States</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peter Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sternson%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Scott Sternson</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Scott Sternson</span></h3>
<div class="p">
<sup>4</sup>Department of Neuroscience, Howard Hughes Medical Institute, University of California, San Diego, San Diego, United States</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sternson%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Scott Sternson</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yao Chen</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yao Chen</span></h3>
<div class="p">
<sup>1</sup>Department of Neuroscience, Washington University in St. Louis, St. Louis, United States</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yao Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<div class="cg p">Editors: <span class="name western">Jie Xiao</span><sup>5</sup>, <span class="name western">Panayiota Poirazi</span><sup>6</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>Department of Neuroscience, Washington University in St. Louis, St. Louis, United States</div>
<div id="aff2">
<sup>2</sup>Ph.D. Program in Neuroscience, Washington University in St. Louis, St. Louis, United States</div>
<div id="aff3">
<sup>3</sup>Master’s Program in Biomedical Engineering, Washington University in St. Louis, St. Louis, United States</div>
<div id="aff4">
<sup>4</sup>Department of Neuroscience, Howard Hughes Medical Institute, University of California, San Diego, San Diego, United States</div>
<div id="aff5">
<sup>5</sup>Johns Hopkins University, United States</div>
<div id="aff6">
<sup>6</sup>FORTH Institute of Molecular Biology and Biotechnology, Greece</div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Jie Xiao</span></strong>: <span class="role">Reviewing Editor</span>
</div>
<div>
<strong class="contrib"><span class="name western">Panayiota Poirazi</span></strong>: <span class="role">Senior Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2024, Ma et al</div>
<p>This article is distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use and redistribution provided that the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12367299  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40832998/" class="usa-link">40832998</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article">
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint posted on bioRxiv on June 6, 2025: "<a href="/articles/PMC10769356/" class="usa-link">The promise and peril of comparing fluorescence lifetime in biology revealed by simulations</a>".</div>
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint with doi: <a href="https://doi.org/10.7554/eLife.101559.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.7554/eLife.101559.1</a>.</div>
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint with doi: <a href="https://doi.org/10.7554/eLife.101559.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.7554/eLife.101559.2</a>.</div>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Signaling dynamics are crucial in biological systems, and biosensor-based real-time imaging has revolutionized their analysis. Fluorescence lifetime imaging microscopy (FLIM) excels over the widely used fluorescence intensity imaging by allowing the measurement of absolute signal levels independent of sensor concentration. This capability enables the comparison of signaling dynamics across different animals, body regions, and timeframes. However, FLIM’s advantage can be compromised by factors like autofluorescence in biological experiments. To address this, we introduce FLiSimBA, a flexible computational framework for realistic <span class="text-underline">F</span>luorescence <span class="text-underline">Li</span>fetime <span class="text-underline">Sim</span>ulation for <span class="text-underline">B</span>iological <span class="text-underline">A</span>pplications. Through simulations, we analyze the signal-to-noise ratios of fluorescence lifetime data, determining measurement uncertainty and providing necessary error bars for lifetime measurements. Furthermore, we challenge the belief that fluorescence lifetime is unaffected by sensor expression and establish quantitative limits to this insensitivity in biological applications. Additionally, we propose innovations, notably multiplexed dynamic imaging that combines fluorescence intensity and lifetime measurements. This innovation can transform the number of signals that can be simultaneously monitored, thereby enabling a systems approach in studying signaling dynamics. Thus, by incorporating different factors into our simulation framework, we uncover surprises, identify limitations, and propose advancements for fluorescence lifetime imaging in biology. This quantitative framework supports rigorous experimental design, facilitates accurate data interpretation, and paves the way for technological advancements in fluorescence lifetime imaging.</p>
<section id="kwd-group2" class="kwd-group"><p><strong>Research organism:</strong> None</p></section></section><section id="s1"><h2 class="pmc_sec_title">Introduction</h2>
<p>FLIM and fluorescence lifetime photometry (FLiP) are powerful methods for revealing the dynamics of biological signals (<a href="#bib3" class="usa-link" aria-describedby="bib3">Bastiaens and Squire, 1999</a>; <a href="#bib5" class="usa-link" aria-describedby="bib5">Becker, 2012</a>; <a href="#bib7" class="usa-link" aria-describedby="bib7">Brinks et al., 2015</a>; <a href="#bib10" class="usa-link" aria-describedby="bib10">Chen et al., 2017</a>; <a href="#bib9" class="usa-link" aria-describedby="bib9">Chen et al., 2014</a>; <a href="#bib24" class="usa-link" aria-describedby="bib24">Lakowicz et al., 1992</a>; <a href="#bib26" class="usa-link" aria-describedby="bib26">Laviv et al., 2020</a>; <a href="#bib27" class="usa-link" aria-describedby="bib27">Lazzari-Dean et al., 2019</a>; <a href="#bib28" class="usa-link" aria-describedby="bib28">Lee et al., 2019</a>; <a href="#bib29" class="usa-link" aria-describedby="bib29">Lee et al., 2021</a>; <a href="#bib31" class="usa-link" aria-describedby="bib31">Ma et al., 2018</a>; <a href="#bib32" class="usa-link" aria-describedby="bib32">Ma et al., 2024</a>; <a href="#bib36" class="usa-link" aria-describedby="bib36">Massengill et al., 2022</a>; <a href="#bib37" class="usa-link" aria-describedby="bib37">Mongeon et al., 2016</a>; <a href="#bib55" class="usa-link" aria-describedby="bib55">Tang and Yasuda, 2017</a>; <a href="#bib60" class="usa-link" aria-describedby="bib60">van der Linden et al., 2021</a>; <a href="#bib67" class="usa-link" aria-describedby="bib67">Yasuda, 2006a</a>; <a href="#bib68" class="usa-link" aria-describedby="bib68">Yasuda et al., 2006b</a>; <a href="#bib70" class="usa-link" aria-describedby="bib70">Zhang et al., 2021b</a>; <a href="#bib71" class="usa-link" aria-describedby="bib71">Zheng et al., 2015</a>). Fluorescence lifetime refers to the time between excitation of a fluorophore and emission of light. Compared with intensity-based imaging, the greatest advantage of fluorescence lifetime is its insensitivity to fluorophore concentration. As a result, fluorescence lifetime can be used to compare the dynamics of biological signals across animals or over long periods of time despite changes in sensor expression levels (<a href="#bib26" class="usa-link" aria-describedby="bib26">Laviv et al., 2020</a>; <a href="#bib32" class="usa-link" aria-describedby="bib32">Ma et al., 2024</a>). Furthermore, it offers the potential to quantitate absolute values of biological signals because of its quantitative nature and its insensitivity to sensor expression. Because of these advantages, FLIM-compatible sensors are being actively developed and FLIM has been increasingly adopted to elucidate the dynamics of many types of biological signals over multiple time scales.</p>
<p>Although fluorescence lifetime is independent of sensor expression when only the sensor is present, in biology experiments, this very advantage of FLIM breaks down when multiple other factors are present. These include autofluorescence (natural emission of light by biological tissue after light absorption), background light (e.g. ambient light), dark current, and afterpulse of the photomultiplier tube (PMT) (<a href="#bib1" class="usa-link" aria-describedby="bib1">Akgun et al., 2008</a>; <a href="#bib16" class="usa-link" aria-describedby="bib16">Georgakoudi and Quinn, 2023</a>; <a href="#bib34" class="usa-link" aria-describedby="bib34">Malak et al., 2022</a>). As sensor expression varies, the relative contribution of sensor fluorescence over these other sources of light or electrical noise varies correspondingly, leading to an apparent change in fluorescence lifetime. Thus, to harness the power and correctly interpret the results of FLIM and FLiP experiments in biological tissue, it is critical to quantitatively understand the regime in which fluorescence lifetime varies with sensor expression, and the range in which sensor expression does not significantly alter lifetime measurements. Furthermore, these additional factors introduce bias and noise to fluorescence lifetime measurements. As innovation pushes the technological boundary to image larger fields of view at higher speeds (<a href="#bib6" class="usa-link" aria-describedby="bib6">Bowman et al., 2023</a>; <a href="#bib30" class="usa-link" aria-describedby="bib30">Lodder et al., 2025</a>; <a href="#bib46" class="usa-link" aria-describedby="bib46">Raspe et al., 2016</a>; <a href="#bib50" class="usa-link" aria-describedby="bib50">Shcheslavskiy et al., 2018</a>; <a href="#bib69" class="usa-link" aria-describedby="bib69">Zhang et al., 2021a</a>), it is critical to understand how these factors contribute to signal-to-noise (SNR) ratio, and how many photons are required to achieve a certain SNR in biological settings.</p>
<p>An effective tool for exploring how experimental parameters contribute to outcomes is simulation. Both analytical and simulation methods have provided insights into issues such as the SNR (<a href="#bib12" class="usa-link" aria-describedby="bib12">Esposito et al., 2007</a>; <a href="#bib13" class="usa-link" aria-describedby="bib13">Esposito et al., 2008</a>; <a href="#bib17" class="usa-link" aria-describedby="bib17">Gerritsen et al., 2002</a>; <a href="#bib22" class="usa-link" aria-describedby="bib22">Köllner and Wolfrum, 1992</a>; <a href="#bib23" class="usa-link" aria-describedby="bib23">Kumar, 2012</a>; <a href="#bib25" class="usa-link" aria-describedby="bib25">Lakowicz, 2006</a>; <a href="#bib39" class="usa-link" aria-describedby="bib39">Nasser and Meller, 2022</a>; <a href="#bib40" class="usa-link" aria-describedby="bib40">Netaev et al., 2022</a>; <a href="#bib43" class="usa-link" aria-describedby="bib43">Philip and Carlsson, 2003</a>; <a href="#bib48" class="usa-link" aria-describedby="bib48">Roethlein et al., 2015</a>; <a href="#bib58" class="usa-link" aria-describedby="bib58">Trinh and Esposito, 2021</a>; <a href="#bib59" class="usa-link" aria-describedby="bib59">Turton et al., 2003</a>; <a href="#bib61" class="usa-link" aria-describedby="bib61">Walsh et al., 2016</a>; <a href="#bib65" class="usa-link" aria-describedby="bib65">Xiao et al., 2021</a>; <a href="#bib67" class="usa-link" aria-describedby="bib67">Yasuda, 2006a</a>). However, prior work usually assumes the presence of sensor fluorescence only, without considering the important contributions to noise and bias due to other factors, such as autofluorescence and afterpulse of the PMT. Consequently, these simulations are useful in vitro but are not readily applicable in biological settings. Therefore, to understand how experimental conditions influence lifetime estimates for biological applications, simulations with realistic and, ideally, measured data must be performed.</p>
<p>Here, we introduce <span class="text-underline">F</span>luorescence <span class="text-underline">Li</span>fetime <span class="text-underline">Sim</span>ulation for <span class="text-underline">B</span>iological <span class="text-underline">A</span>pplications (FLiSimBA) and use it to quantitatively define the potential and limitations of fluorescence lifetime experiments in biological settings. FLiSimBA is a flexible platform designed for realistic simulation of fluorescence lifetime data with empirically determined parameters through time-correlated single photon counting (TCSPC). FLiSimBA recapitulates experimental data. Using FLiSimBA, we determined the photon requirements for minimum detectable differences in fluorescence lifetime, thus providing realistic estimates of the SNR in biological tissue and the necessary quantification of measurement uncertainty for correct data interpretation. Furthermore, we challenge the conventional view that fluorescence lifetime is insensitive to sensor expression levels and establish quantitative limits of insensitivity, defining the potential and limits of comparing fluorescence lifetime measurements in biology. Finally, we use FLiSimBA to propel innovation in FLIM by assessing the impact of hardware improvement on the SNR, quantifying the value of developing sensors with spectra unaffected by autofluorescence, and specifying sensor characteristics that would greatly expand the power of simultaneous real-time measurements of multiple signals (multiplexing) with both intensity and lifetime properties of sensors. Our results show that considering relevant contributors and noise to fluorescence lifetime measurements in biological contexts does not merely add detail. Rather, these quantitative changes produce qualitatively different conclusions that require careful consideration to avoid misinterpretation of data and to harness the potential of FLIM. Thus, we provide a versatile tool for simulating experimental conditions, a quantitative framework for evaluating fluorescence lifetime results, and insights into the potential and limitations of fluorescence lifetime measurements in biological applications.</p></section><section id="s2"><h2 class="pmc_sec_title">Results</h2>
<section id="s2-1"><h3 class="pmc_sec_title">Simulation of fluorescence lifetime data in biological tissue</h3>
<p>To realistically mimic the fluorescence lifetime in biological tissue, we simulated contributions from sensor fluorescence, autofluorescence, afterpulse, and background, which is composed of both a small amount of light leakage and the dark current of the photon detectors (<a href="#fig1" class="usa-link">Figure 1</a>).</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">Figure 1. The simulation procedure of <span class="text-underline">F</span>luorescence <span class="text-underline">Li</span>fetime <span class="text-underline">Sim</span>ulation for <span class="text-underline">B</span>iological <span class="text-underline">A</span>pplications (FLiSimBA).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367299_elife-101559-fig1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2706/12367299/3f645baad531/elife-101559-fig1.jpg" loading="lazy" height="568" width="735" alt="Figure 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The sensor fluorescence lifetime distribution of a FRET-based sensor was modeled as a double exponential decay, as shown in <a href="#equ1" class="usa-link">Equation 1</a> (<em>τ<sub>1</sub></em>=2.14 ns, <em>τ<sub>2</sub></em>=0.69 ns) and the lifetimes of photons were sampled with replacement. After the sensor fluorescence was convolved with the probability density function (PDF) of the instrument response function (IRF), the following were added to produce the final simulated data: autofluorescence (autoF) empirically measured from brain tissue with photon number <em>F<sub>autoF</sub></em>, afterpulse of the photomultiplier tube (PMT), and background signal (consisting of the dark current of the PMT and light leak) with photon number <em>F<sub>background</sub></em>. The histograms of the final simulated data were similar to those of the experimental data. The simulation was repeated 500 times under each <em>P<sub>1</sub></em> and sensor fluorescence photon number (<em>F<sub>sensor</sub></em>) condition. The fitted <em>P<sub>1</sub></em> (based on a double exponential decay fitting of the final simulated data) and empirical lifetime were used for subsequent analyses and evaluations.</p></figcaption></figure><p>For sensor fluorescence, we sampled a specific number of photons (<em>F<sub>sensor</sub></em>) with replacement from an ideal distribution of photon lifetimes. In the examples in this study, we used the lifetime distribution of the FLIM-compatible A Kinase Activity Reporter (FLIM-AKAR), a Förster resonance energy transfer (FRET)-based biosensor that measures the activity of protein kinase A (PKA) in a variety of biological contexts, including brain slices and freely moving animals (<a href="#bib9" class="usa-link" aria-describedby="bib9">Chen et al., 2014</a>; <a href="#bib10" class="usa-link" aria-describedby="bib10">Chen et al., 2017</a>; <a href="#bib28" class="usa-link" aria-describedby="bib28">Lee et al., 2019</a>; <a href="#bib29" class="usa-link" aria-describedby="bib29">Lee et al., 2021</a>; <a href="#bib57" class="usa-link" aria-describedby="bib57">Tilden et al., 2024</a>). The fluorescence lifetime of FLIM-AKAR follows a double exponential decay defined by the following equation:</p>
<table class="disp-formula p" id="equ1"><tr>
<td class="formula"><math id="m1" display="block"><mstyle displaystyle="true" scriptlevel="0"><mrow><mi>F</mi><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>=</mo><msub><mi>F</mi><mrow><mn>0</mn></mrow></msub><mo>∗</mo><mo stretchy="false">(</mo><msub><mi>P</mi><mrow><mn>1</mn></mrow></msub><mo>∗</mo><msup><mi>e</mi><mrow><mrow><mo>(</mo><mrow><mo>−</mo><mfrac><mi>t</mi><msub><mi>τ</mi><mrow><mn>1</mn></mrow></msub></mfrac></mrow><mo>)</mo></mrow></mrow></msup><mo>+</mo><msub><mi>P</mi><mrow><mn>2</mn></mrow></msub><mo>∗</mo><msup><mi>e</mi><mrow><mrow><mo>(</mo><mrow><mo>−</mo><mfrac><mi>t</mi><msub><mi>τ</mi><mrow><mn>2</mn></mrow></msub></mfrac></mrow><mo>)</mo></mrow></mrow></msup><mo stretchy="false">)</mo></mrow></mstyle></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where <em>F(t</em>) is the number of photons that arrive at time t, <em>F<sub>0</sub></em> is the number of photons at time 0, <em>τ<sub>1</sub></em>, and <em>τ<sub>2</sub></em> are time constants corresponding to lifetime distributions of the donor fluorophore that is either free or undergoing FRET, and <em>P<sub>1</sub></em> and <em>P<sub>2</sub></em> are the proportions of the donor fluorophores in these two states (<a href="#fig1" class="usa-link">Figure 1</a>). Following sampling, we convolved the lifetime histogram with the probability density function (PDF) of an instrument response function (IRF) to account for instrument noise.</p>
<p>Subsequently, we added a specific number of sampled photons (<em>F<sub>autoF</sub></em>) from an autofluorescence curve, whose distribution was determined via fluorescence measurements in brain tissue without any sensor expression. The autofluorescence lifetime histogram exhibited faster decay than that of FLIM-AKAR (<a href="#fig2s1" class="usa-link">Figure 2—figure supplement 1A</a>). After that, we added the afterpulse of the PMT, long-lasting signals from ionization of residual gas inside the PMT following a photon detection event (<a href="#bib1" class="usa-link" aria-describedby="bib1">Akgun et al., 2008</a>). The afterpulse was modeled as an even lifetime distribution with the amount of signal as a fraction of sensor fluorescence. Finally, we added a specific number of background signals (<em>F<sub>background</sub></em>) that was empirically determined from measurements, accounted for largely by ambient light leakage and the dark current of the PMT. We generated 500 simulated fluorescence lifetime histograms for each <em>P<sub>1</sub></em> and sensor photon count (for additional details, see Materials and methods).</p>
<p>The simulated histogram closely matched the experimental histogram (<a href="#fig1" class="usa-link">Figure 1</a>). Following histogram generation, we used two commonly used fluorescence lifetime metrics to evaluate the simulated data (<a href="#bib10" class="usa-link" aria-describedby="bib10">Chen et al., 2017</a>; <a href="#bib19" class="usa-link" aria-describedby="bib19">Harvey et al., 2008</a>; <a href="#bib29" class="usa-link" aria-describedby="bib29">Lee et al., 2021</a>; <a href="#bib32" class="usa-link" aria-describedby="bib32">Ma et al., 2024</a>; <a href="#bib37" class="usa-link" aria-describedby="bib37">Mongeon et al., 2016</a>; <a href="#bib68" class="usa-link" aria-describedby="bib68">Yasuda et al., 2006b</a>). First, fluorescence lifetime data were evaluated with empirical lifetimes, defined as the average lifetime of all photons:</p>
<table class="disp-formula p" id="equ2"><tr>
<td class="formula"><math id="m2" display="block"><mstyle displaystyle="true" scriptlevel="0"><mrow><mstyle displaystyle="true" scriptlevel="0"><mi>e</mi><mi>m</mi><mi>p</mi><mi>i</mi><mi>r</mi><mi>i</mi><mi>c</mi><mi>a</mi><mi>l</mi><mspace width="thinmathspace"></mspace><mi>l</mi><mi>i</mi><mi>f</mi><mi>e</mi><mi>t</mi><mi>i</mi><mi>m</mi><mi>e</mi><mo>=</mo><mfrac><mrow><mo>∑</mo><mrow><mo>(</mo><mrow><mi>F</mi><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>∗</mo><mi>t</mi></mrow><mo>)</mo></mrow></mrow><mrow><mo>∑</mo><mi>F</mi><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow></mfrac></mstyle></mrow></mstyle></math></td>
<td class="label">(2)</td>
</tr></table>
<p>in which <em>t</em> is the lifetime of photons arriving at a specific time channel, and <em>F(t</em>) is the photon count from that time channel. Additionally, simulated histograms were fitted with a double exponential decay equation with Gauss-Newton nonlinear least-square fitting algorithm:</p>
<table class="disp-formula p" id="equ3"><tr>
<td class="formula"><math id="m3" display="block"><mstyle displaystyle="true" scriptlevel="0"><mrow><mi>F</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>=</mo><mrow><mo>[</mo><mrow><msub><mi>F</mi><mn>0</mn></msub><mo>∗</mo><mrow><mo>(</mo><mrow><msub><mi>P</mi><mn>1</mn></msub><mo>∗</mo><msup><mi>e</mi><mrow><mo>−</mo><mfrac><mi>t</mi><msub><mi>τ</mi><mn>1</mn></msub></mfrac></mrow></msup><mo>+</mo><msub><mi>P</mi><mn>2</mn></msub><mo>∗</mo><msup><mi>e</mi><mrow><mo>−</mo><mfrac><mi>t</mi><msub><mi>τ</mi><mn>2</mn></msub></mfrac></mrow></msup></mrow><mo>)</mo></mrow><mo>+</mo><mi>S</mi><mi>H</mi><mi>G</mi></mrow><mo>]</mo></mrow><mo>⊗</mo><mi>I</mi><mi>R</mi><mi>F</mi><mo>+</mo><msub><mi>F</mi><mrow><mi>b</mi><mi>a</mi><mi>c</mi><mi>k</mi><mi>g</mi><mi>r</mi><mi>o</mi><mi>u</mi><mi>n</mi><mi>a</mi><mi>d</mi></mrow></msub></mrow></mstyle></math></td>
<td class="label">(3)</td>
</tr></table>
<p>in which <em>F<sub>0</sub></em> is photon count from sensor fluorescence at time 0, <em>F<sub>background </sub></em>is background signal, SHG is second harmonic generation that is added at time 0, and ⊗ represents convolution. Both experimental and simulated data showed good fitting (<a href="#fig2s1" class="usa-link">Figure 2—figure supplement 1B</a>). For subsequent analyses and evaluations, empirical lifetime and fitted <em>P<sub>1</sub></em>, corresponding to the proportion of slower decay (free donor fluorophore) were used (<a href="#fig1" class="usa-link">Figure 1</a>).</p></section><section id="s2-2"><h3 class="pmc_sec_title">Bias and noise introduced by different sources of signals</h3>
<p>To understand how different fluorescence and electronic sources contribute to bias and noise, we analyzed the fitted <em>P<sub>1</sub></em> and empirical lifetime across a range of values for the simulated <em>P<sub>1</sub></em> and sensor photon counts. Autofluorescence (autoF) decreased the fitted <em>P<sub>1</sub></em> and empirical lifetime, consistent with the faster decay of autofluorescence compared with that of FLIM-AKAR sensor fluorescence (<a href="#fig2" class="usa-link">Figure 2</a>, <a href="#fig2s1" class="usa-link">Figure 2—figure supplement 1</a>; <a href="#fig2" class="usa-link">Figure 2B</a>: p&lt;0.0001, sensor +autoF vs sensor only under all sensor photon number conditions for both fitted <em>P<sub>1</sub></em> and empirical lifetime). Further addition of afterpulse and background did not significantly change the fitted <em>P<sub>1</sub></em> (<a href="#fig2" class="usa-link">Figure 2</a>; <a href="#fig2" class="usa-link">Figure 2B</a>: p&gt;0.6, final simulated data vs sensor + autoF condition for all sensor photon number conditions), consistent with consideration of the background term during the fitting procedure. In contrast, afterpulse and background increased the empirical lifetime, as they are evenly distributed across lifetime time channels and have a higher mean lifetime than FLIM-AKAR (<a href="#fig2" class="usa-link">Figure 2</a>; <a href="#fig2" class="usa-link">Figure 2B</a>: p&lt;0.0001, final simulated data vs sensor + autoF for all sensor photon number conditions). Afterpulse and background had a greater impact on empirical lifetime compared with autofluorescence. This is because the empirical lifetime of background/afterpulse (~4.90 ns) deviates more from FLIM-AKAR (2–2.3 ns) than the empirical lifetime of autofluorescence (around 1.69 ns) does. Thus, while autofluorescence affects both the fitted <em>P<sub>1</sub></em> and empirical lifetime, only empirical lifetime is sensitive to afterpulse and background.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">Figure 2. Simulated fluorescence lifetime in brain tissue.</h4>
<figcaption><p>(<strong>A</strong>) Simulated example histograms (upper panels) and heatmaps (middle and lower panels) showing the average fitted <em>P<sub>1</sub></em> and empirical lifetime of the simulated data across a range of <em>P<sub>1</sub></em> and sensor photon number conditions. Fluorescence lifetime histograms were simulated with the sensor only, sensor + autofluorescence (autoF), and sensor + autofluorescence + afterpulse + background (final simulated data). (<strong>B</strong>) Summaries of fitted <em>P<sub>1</sub></em> (left) and empirical lifetime (right) with simulated <em>P<sub>1</sub></em>=0.5854. *p&lt;0.05 for sensor + autoF (cyan) vs sensor only (orange); n.s. not significant for final simulated data (purple) vs sensor + autoF (cyan); #p&lt;0.05 for final simulated data (purple) vs sensor + autoF (cyan). Two-way ANOVA with Šídák’s multiple comparisons test. n=500 simulations. The data are represented as means and standard deviations.</p></figcaption><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367299_elife-101559-fig2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2706/12367299/e643afb4006f/elife-101559-fig2.jpg" loading="lazy" height="816" width="735" alt="Figure 2."></a></p>
<div class="p"><figure class="fig xbox font-sm" id="fig2s1"><h5 class="obj_head">Figure 2—figure supplement 1. Comparison between simulated and experimental data.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367299_elife-101559-fig2-figsupp1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2706/12367299/f83815cc6247/elife-101559-fig2-figsupp1.jpg" loading="lazy" height="596" width="735" alt="Figure 2—figure supplement 1."></a></p>
<figcaption><div class="p">(<strong>A</strong>) Fluorescence lifetime histograms of sensor fluorescence and autofluorescence. Note that autofluorescence has lower photon counts and shows faster decay compared with sensor fluorescence. (<strong>B</strong>) Examples of fluorescence decay histograms and fitted curves of experimental and simulated data. (<strong>C</strong>) Empirical lifetime, fitted <em>P<sub>1</sub></em>, and χ<sup>2</sup> from fitting (simulated <em>P<sub>1</sub></em>=0.5854) of simulated and experimental data. *p&lt;0.05, n.s. not significant. n=500 for simulations and n=7 for experimental data. One-way ANOVA with Šídák’s multiple comparisons test.</div></figcaption></figure></div>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Importantly, the biases introduced by autofluorescence, afterpulse, and background were less pronounced at higher sensor photon counts, which can be explained by the relatively small contribution of these factors when sensor fluorescence is high. Notably, as the number of sensor photons increased, the biases not only decreased but also plateaued, which made fluorescence lifetime less dependent on the number of sensor photons (<a href="#fig2" class="usa-link">Figure 2</a>). Furthermore, the variance also decreased at higher photon counts for both the fitted <em>P<sub>1</sub></em> and empirical lifetime (<a href="#fig2" class="usa-link">Figure 2B</a>). Thus, higher photon counts resulted in both less bias and less noise.</p>
<p>To quantify how well our simulation matches the experimental conditions, we calculated fitted <em>P<sub>1</sub></em> and empirical lifetime after each simulation step, with <em>P<sub>1</sub></em> matching the experimental condition. For both fitted <em>P<sub>1</sub></em> and empirical lifetime, the values from final simulated data were not significantly different from those from experimental data (<a href="#fig2s1" class="usa-link">Figure 2—figure supplement 1C</a>; adjusted p=0.81 for empirical lifetime and 0.98 for fitted <em>P<sub>1</sub></em>, final simulated data vs experimental data; n=500 and 7, respectively). Thus, our simulated fluorescence lifetime data recapitulated experimental data in biologically relevant settings.</p>
<p>In summary, using FLiSimBA, we realistically simulated fluorescence lifetime data in biological settings and quantitatively defined how different sources of signals contribute to bias and noise. These curves and quantitative approaches can, therefore, be used to evaluate the impact of optimization of these different sources on fluorescence lifetime measurements.</p></section><section id="s2-3"><h3 class="pmc_sec_title">Determination of minimum photon number requirements to achieve specific SNRs</h3>
<p>How many photons do we need for a given FLIM experiment in a biology experiment? Are more photons always better? A greater number of photons translates into a better SNR but also a lower sampling rate and reduced size for the field of view. How can we quantitatively find the optimal compromise among these factors? As sensor fluorescence increases, the variances of both the fitted <em>P<sub>1</sub></em> and empirical lifetime decrease (<a href="#fig2" class="usa-link">Figure 2B</a>), and the ability to detect a specific amount of fluorescence lifetime response increases. Although the number of photons required to achieve a certain SNR was analyzed previously (<a href="#bib12" class="usa-link" aria-describedby="bib12">Esposito et al., 2007</a>; <a href="#bib13" class="usa-link" aria-describedby="bib13">Esposito et al., 2008</a>; <a href="#bib17" class="usa-link" aria-describedby="bib17">Gerritsen et al., 2002</a>; <a href="#bib22" class="usa-link" aria-describedby="bib22">Köllner and Wolfrum, 1992</a>; <a href="#bib23" class="usa-link" aria-describedby="bib23">Kumar, 2012</a>; <a href="#bib25" class="usa-link" aria-describedby="bib25">Lakowicz, 2006</a>; <a href="#bib39" class="usa-link" aria-describedby="bib39">Nasser and Meller, 2022</a>; <a href="#bib40" class="usa-link" aria-describedby="bib40">Netaev et al., 2022</a>; <a href="#bib43" class="usa-link" aria-describedby="bib43">Philip and Carlsson, 2003</a>; <a href="#bib48" class="usa-link" aria-describedby="bib48">Roethlein et al., 2015</a>; <a href="#bib58" class="usa-link" aria-describedby="bib58">Trinh and Esposito, 2021</a>; <a href="#bib59" class="usa-link" aria-describedby="bib59">Turton et al., 2003</a>; <a href="#bib61" class="usa-link" aria-describedby="bib61">Walsh et al., 2016</a>; <a href="#bib65" class="usa-link" aria-describedby="bib65">Xiao et al., 2021</a>; <a href="#bib67" class="usa-link" aria-describedby="bib67">Yasuda, 2006a</a>), such analysis was not performed with consideration of biological samples in realistic experiments involving autofluorescence, background, and afterpulse.</p>
<p>To determine the minimum number of photons required for a certain SNR, we analyzed the minimum detectable differences (MDD) in both the fitted <em>P<sub>1</sub></em> and empirical lifetime for different numbers of sensor photons and repeated measurements (<a href="#fig3" class="usa-link">Figure 3</a>). The MDD was calculated with 80% power and a 5% significance level. As the number of sensor photons increased, the MDDs decreased. As the number of repeated measurements increased, the MDD also decreased (<a href="#fig3" class="usa-link">Figure 3B and C</a>). Importantly, the MDD curves provide quantitative information on the minimum number of photons required for a certain amount of expected signal. For example, to detect a <em>P<sub>1</sub></em> change of 0.006 or a lifetime change of 5 ps with five sample measurements in each comparison group, approximately 300,000 photons are needed. As the sensor fluorescence increased, the gain in SNR decreased. Thus, MDD curves generated with FLiSimBA are instrumental for determining the optimal experimental conditions (for example, optimal imaging speed and sizes of imaging fields) necessary to detect a specific amount of lifetime change.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">Figure 3. The minimum number of photons required to detect a specific fluorescence lifetime response.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367299_elife-101559-fig3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2706/12367299/5c27c563eb63/elife-101559-fig3.jpg" loading="lazy" height="256" width="735" alt="Figure 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Schematic illustrating the power analysis used to calculate the minimum detectable difference between two lifetime distributions. (<strong>B, C</strong>) Minimum detectable differences in the fitted <em>P<sub>1</sub></em> (<strong>B</strong>) and empirical lifetime (<strong>C</strong>) for different numbers of sensor photons and for different numbers of pairs of sampled data (n). The data were simulated with <em>P<sub>1</sub></em>=0.5.</p></figcaption></figure><p>Using realistic determination of the SNR with FLiSimBA, we also quantitatively evaluated the impact of specific hardware changes on the SNR. HBDs are advantageous over traditional gallium arsenide phosphide (GaAsP) PMTs because of their narrower IRF width and lack of afterpulsing (<a href="#bib4" class="usa-link" aria-describedby="bib4">Becker et al., 2011</a>). However, hybrid detectors are more expensive. Thus, it is valuable to quantify the benefits of HBDs over traditional GaAsP PMTs for SNRs. We generated fluorescence lifetime histograms with narrower IRFs and no afterpulses for HBDs and compared them with the simulated data for traditional GaAsP PMTs (<a href="#fig4" class="usa-link">Figure 4A and B</a>). Hybrid detectors and GaAsP PMTs displayed similar photon-dependent changes in fitted <em>P<sub>1</sub></em> and empirical lifetime (<a href="#fig4" class="usa-link">Figure 4</a>). Furthermore, a given sensor photon number gave a comparable MDD for HBDs and GaAsP PMTs (<a href="#fig4" class="usa-link">Figure 4</a>). The similarity in the MDD curves can be explained by similar levels of variance between HBDs and PMTs (<a href="#fig4" class="usa-link">Figure 4C and D</a>): although HBDs with a narrower IRF introduced less noise than GaAsP detectors (<a href="#fig4" class="usa-link">Figure 4G and H</a>; noise from sampling from the IRF: standard deviation (STD) of fitted <em>P<sub>1</sub></em>: 0.0008 for GaAsP PMTs and 0.0005 for HBDs; STD of empirical lifetime: 0.0003 for GaAsP PMTs and 0.0001 for HBDs), the noise of lifetime measurements was dominated by photon noise, and not by IRF noise (<a href="#fig4" class="usa-link">Figure 4G and H</a>; noise from sampling of photons: STD of fitted <em>P<sub>1</sub></em>: 0.0028 for both GaAsP PMTs and HBDs; STD of empirical lifetime: 0.0025 for both GaAsP PMTs and HBDs). Thus, although HBD can offer other advantages (<a href="#bib4" class="usa-link" aria-describedby="bib4">Becker et al., 2011</a>; <a href="#bib58" class="usa-link" aria-describedby="bib58">Trinh and Esposito, 2021</a>), HBDs with a narrower IRF and no afterpulse yield little improvement in the SNR for fluorescent protein-based sensors in biological applications.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">Figure 4. Comparison of the fluorescence lifetime responses of GaAsP photomultiplier tubes (PMTs) and hybrid detectors (HBDs).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367299_elife-101559-fig4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2706/12367299/b6cd1e59f3cd/elife-101559-fig4.jpg" loading="lazy" height="1460" width="735" alt="Figure 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Differences in instrument response function (IRF) widths and afterpulse ratios between the GaAsP PMT and HBD. These specific parameters are used in subsequent simulations. (<strong>B</strong>) Gaussian IRFs used for simulation, reflecting different Gaussian widths for the GaAsP PMT and HBD. (<strong>C, D</strong>) Distributions of the fitted <em>P<sub>1</sub></em> (<strong>C</strong>) and empirical lifetime (<strong>D</strong>) of the simulated data from the GaAsP PMT or HBD, with simulated <em>P<sub>1</sub></em>=0.5 and showing the sensor photon number dependence. The data are represented as means and standard deviations. (<strong>E, F</strong>) Minimum detectable difference in the fitted <em>P<sub>1</sub></em> (<strong>E</strong>) and empirical lifetime (<strong>F</strong>) for different numbers of sensor photons and different numbers of data samples. The data were simulated with <em>P<sub>1</sub></em>=0.5. (<strong>G, H</strong>) Distribution of the fitted <em>P<sub>1</sub></em> (<strong>G</strong>) and empirical lifetime (<strong>H</strong>) of the simulated data from the GaAsP PMT or HBD, with noise introduced from either sampling of photons or sampling from the IRF distribution. The data were simulated without autofluorescence, afterpulse, or background and are represented as means and standard deviations. n=500 simulations.</p></figcaption></figure><p>In summary, the quantitative and biologically realistic curves generated with FLiSimBA allow users to determine the appropriate trade-off in each experiment. Given the specific sensor brightness, sensor lifetime, and expected signal amplitude in specific biological applications, FLiSimBA allows the users to select the imaging speed and size of the imaging field to achieve the desired SNR. In addition, FLiSimBA allows users to evaluate the trade-off between performance improvement and price with specific hardware changes.</p></section><section id="s2-4"><h3 class="pmc_sec_title">Expression level dependence of fluorescence lifetime estimates</h3>
<p>Sensor expression levels often change over days and across animals and is usually assumed not to influence lifetime estimates because fluorescence lifetime is an intensive property of fluorophores. However, this assumption is true only if the biosensor is the only contributor to measurements. With autofluorescence, afterpulse, and background signals present in biological applications, the amount of sensor fluorescence relative to these contributing factors can lead to an apparent change in fluorescence lifetime estimates even if the biosensor is in the same conformational state (<a href="#bib32" class="usa-link" aria-describedby="bib32">Ma et al., 2024</a>). Here, we challenge the conventional view that fluorescence lifetime is independent of sensor expression and use simulation to define the range in which sensor expression has a negligible influence on lifetime.</p>
<p>We first determined how sensor expression levels altered the relative responses in fitted <em>P<sub>1</sub></em> and empirical lifetime (<a href="#fig5" class="usa-link">Figure 5A</a>). When <em>P<sub>1</sub></em> changed from 0.4 to 0.5, both fitted <em>P<sub>1</sub></em> and empirical lifetime increased as expected (<a href="#fig5" class="usa-link">Figure 5B and C</a>; adjusted p&lt;0.0001, <em>P<sub>1</sub></em>=0.4 vs <em>P<sub>1</sub></em>=0.5, under all photon number conditions for both fitted <em>P<sub>1</sub></em> and empirical lifetime). As the sensor photon count increased, fitted <em>P<sub>1</sub></em> increased and empirical lifetime decreased with both plateauing at high photon counts (<a href="#fig5" class="usa-link">Figure 5B and C</a>; p&lt;0.05, 800,000 photons vs 550,000 photons or less for fitted <em>P<sub>1</sub></em>, 800,000 photons vs 700,000 photons or less for empirical lifetime, for both <em>P<sub>1</sub></em>=0.4 and <em>P<sub>1</sub></em>=0.5). This photon-dependent change in fitted <em>P<sub>1</sub></em> or lifetime is consistent with the diminishing effect of autofluorescence, background, and afterpulse. We next determined whether the response amplitude also changed as the sensor photon count increased by comparing the changes in fitted <em>P<sub>1</sub></em> and empirical lifetime when actual <em>P<sub>1</sub></em> changed from 0.4 to 0.5. As the sensor photon count increased, the change in the fitted <em>P<sub>1</sub></em> and empirical lifetime also showed an apparent increase (<a href="#fig5" class="usa-link">Figure 5D and E</a>; p&lt;0.05, 800,000 photons vs 450,000 photons or less for fitted <em>P<sub>1</sub></em> and 400,000 photons or less for empirical lifetime). The response dependence on photons approached asymptotes at high photon counts: With the parameters used in our case study, the fluorescence lifetime responses were relatively stable at sensor photon counts of 500,000 and above (<a href="#fig5" class="usa-link">Figure 5D and E</a>). Thus, our results challenge the widely held view that relative fluorescence lifetime changes are independent of sensor expression level and define a quantitative threshold of sensor expression that allows comparison of lifetime responses across different sensor expression levels.</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">Figure 5. Impact of the number of sensor photons on the response amplitudes of the fitted <em>P<sub>1</sub></em> and the empirical lifetime.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367299_elife-101559-fig5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2706/12367299/102e463f4dd6/elife-101559-fig5.jpg" loading="lazy" height="1016" width="735" alt="Figure 5."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Schematic illustrating the following question under investigation: in biological systems, as autofluorescence, the afterpulse ratio, and the background remain constant, does the lifetime response change as sensor brightness increases? (<strong>B, C</strong>) Distribution of the fitted <em>P<sub>1</sub></em> (<strong>B</strong>) and empirical lifetime (<strong>C</strong>) from fluorescence lifetime data with simulated <em>P<sub>1</sub></em> values of 0.4 and 0.5 across different sensor photon numbers. *p&lt;0.05, <em>P<sub>1</sub></em>=0.4 vs <em>P<sub>1</sub></em>=0.5, two-way ANOVA with Šídák’s multiple comparisons test; <sup>#</sup>p&lt;0.05, vs photons = 800,000, one-way ANOVA with Dunnett’s multiple comparisons test. (<strong>D, E</strong>) Distribution of the changes in the fitted <em>P<sub>1</sub></em> (<strong>D</strong>) and empirical lifetime (<strong>E</strong>) for different sensor photon numbers. The simulated <em>P<sub>1</sub></em> varied from 0.4 to 0.5. *p&lt;0.05, vs photon count = 800,000, one-way ANOVA with Dunnett’s multiple comparisons test. The data are represented as means and standard deviations. n=500 simulations.</p></figcaption></figure><p>Since fluorescence lifetime allows comparison of absolute levels of analytes (<a href="#bib7" class="usa-link" aria-describedby="bib7">Brinks et al., 2015</a>; <a href="#bib27" class="usa-link" aria-describedby="bib27">Lazzari-Dean et al., 2019</a>; <a href="#bib32" class="usa-link" aria-describedby="bib32">Ma et al., 2024</a>; <a href="#bib60" class="usa-link" aria-describedby="bib60">van der Linden et al., 2021</a>; <a href="#bib71" class="usa-link" aria-describedby="bib71">Zheng et al., 2015</a>), we determined the range where changes in sensor expression do not significantly affect fitted <em>P<sub>1</sub></em> or empirical lifetime (<a href="#fig6" class="usa-link">Figure 6A</a>). A higher sensor photon count reduces the likelihood of significant changes in lifetime due to sensor expression (<a href="#fig6" class="usa-link">Figure 6A</a>). Thus, we determined the minimum number of sensor photons needed for a specific change in sensor expression to have a negligible impact on lifetime measurements (<a href="#fig6" class="usa-link">Figure 6A</a>). We first determined the apparent change in fitted <em>P<sub>1</sub></em> and empirical lifetime introduced by the change in sensor expression levels (<a href="#fig6" class="usa-link">Figure 6C</a>, purple traces). As sensor fluorescence increased, the sensor expression-induced apparent change in fitted <em>P<sub>1</sub></em> and empirical lifetime decreased. Additionally, by considering the variance associated with a specific photon count, we calculated the minimum change in fitted <em>P<sub>1</sub></em> or empirical lifetime that would result in a statistically significant difference due to sensor expression changes (<a href="#fig6" class="usa-link">Figure 6C</a>, black traces, t test threshold, see Materials and methods). The intersection of these two traces (<a href="#fig6" class="usa-link">Figure 6C</a>) indicates the minimum number of sensor photons required to tolerate a certain amount of sensor photon changes (i.e. t tests did not show a statistically significant difference) (<a href="#fig6" class="usa-link">Figure 6B</a>). As the total number of sensor photons increased, a greater difference in sensor expression was tolerated. Fitted <em>P<sub>1</sub></em> could tolerate more photon changes than empirical lifetime for a given number of sensor photons (<a href="#fig6" class="usa-link">Figure 6B</a>). These curves provide quantitative guidelines for determining the necessary sensor photon counts to compare absolute lifetime values across different sensor expression levels.</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">Figure 6. Conditions needed for the sensor expression-induced lifetime change to be statistically nonsignificant.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367299_elife-101559-fig6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2706/12367299/331651415cf1/elife-101559-fig6.jpg" loading="lazy" height="608" width="735" alt="Figure 6."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Schematic illustrating the question under investigation: as the biological analyte, autofluorescence, afterpulse ratio, and background signals remain constant, as the sensor expression/brightness increases, what is the minimum number of sensor photons needed to tolerate a specific change in sensor expression such that the apparent lifetimes are not significantly different? (<strong>B</strong>) Relationship between changes in photon number due to changes in expression level and the minimum number of sensor photons required not to reach statistical significance for both the fitted <em>P<sub>1</sub></em> and empirical lifetime. The data were simulated with <em>P<sub>1</sub></em>=0.5. The minimum number of sensor photons was calculated by interpolating the intersection between the two curves in (<strong>C</strong>). (<strong>C</strong>) Plots of changes reaching statistical significance according to t tests (calculated as 1.96*standard error of the difference in the mean) and apparent changes due to sensor expression for both the empirical lifetime (upper panels) and fitted <em>P<sub>1</sub></em> (lower panels). The data were simulated with <em>P<sub>1</sub></em>=0.5. Different sensor expression changes are plotted in each panel. The data are represented as means and standard deviations.</p></figcaption></figure><p>Together, the data in <a href="#fig5" class="usa-link">Figures 5</a> and <a href="#fig6" class="usa-link">6</a> not only challenge the conventional view that fluorescence lifetime is independent of sensor expression levels in biology experiments, but also, more importantly, quantitatively define the range of sensor expression levels that enable comparison of either relative responses or absolute lifetime values across diverse sensor expression levels. These results are critical for guiding experiments that involve comparisons across animals, across brain regions, and across chronic time periods.</p></section><section id="s2-5"><h3 class="pmc_sec_title">Innovating to reduce sensor expression dependence: avoiding autofluorescence and lowering background signal</h3>
<p>Sensor expression dependence limits comparisons of fluorescence lifetime values and responses across chronic periods and across animals. Thus, we investigated the extent to which the dependence of sensor expression could be mitigated by achievable technological advances, in particular, by developing sensors outside the autofluorescence spectra (e.g. red-shifted sensors), and/or by lowering background signal (e.g. with photodetectors with low dark current, together with better light proofing) (<a href="#fig7" class="usa-link">Figure 7</a>).</p>
<figure class="fig xbox font-sm" id="fig7"><h4 class="obj_head">Figure 7. Impact of no autofluorescence and/or low background on the lifetime dependence of sensor expression.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367299_elife-101559-fig7.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2706/12367299/3dad5e699b51/elife-101559-fig7.jpg" loading="lazy" height="909" width="735" alt="Figure 7."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A, B</strong>) Distribution of the change in fitted <em>P<sub>1</sub></em> (<strong>A</strong>) and empirical lifetime (<strong>B</strong>) with different sensor photon numbers, under no autofluorescence, low background, and both improvement conditions. The simulated <em>P<sub>1</sub></em> varied from 0.4 to 0.5. *p&lt;0.05, vs photon count = 800,000, one-way ANOVA with Dunnett’s multiple comparisons test. n=500 simulations. The data are represented as means and standard deviations. (<strong>C, D</strong>) Relationships between changes in photon number due to expression level and the minimum number of sensor photons required not to reach statistical significance for both the fitted <em>P<sub>1</sub></em> (<strong>C</strong>) and empirical lifetime (<strong>D</strong>) under no autofluorescence, low background, or both improvement conditions. The data were simulated with <em>P<sub>1</sub></em>=0.5. The orange traces are the same as the corresponding data in <a href="#fig5" class="usa-link">Figures 5</a> and <a href="#fig6" class="usa-link">6</a> and were reused here for comparison purposes.</p></figcaption></figure><p>If sensors could be made with emission spectra that do not overlap with tissue autofluorescence (e.g. red or far-red sensors), both the relative response and absolute values of fitted <em>P<sub>1</sub></em> no longer depended on sensor expression (<a href="#fig7" class="usa-link">Figure 7A and C</a> left panels; 7A: p=0.047 for 30,000 photons, p&gt;0.8 for the other photon counts, vs 800,000 photons). This expression independence can be explained by the fact that only autofluorescence contributes to the sensor expression-dependent bias of fitted <em>P<sub>1</sub></em> (<a href="#fig2" class="usa-link">Figure 2</a>). With no autofluorescence, the relative response in empirical lifetime showed less dependence on sensor expression (<a href="#fig7" class="usa-link">Figure 7B</a>, left, p&lt;0.05 for 150,000 photons or less, p&gt;0.2 for the other photon counts, vs 800,000 photons), whereas the absolute empirical lifetime could tolerate less change in sensor expression (<a href="#fig7" class="usa-link">Figure 7D</a>, left). Thus, sensors with spectra that avoid overlap with autofluorescence are completely independent of sensor expression levels when fitted <em>P<sub>1</sub></em> is used as the metric of evaluation.</p>
<p>When background signals were lower, both the relative response and absolute values of fitted <em>P<sub>1</sub></em> showed a similar sensor expression dependence to that of the higher background (<a href="#fig7" class="usa-link">Figure 7A and C</a>, middle panels; 7A: p&lt;0.05 for 350,000 photons or less, p&gt;0.1 for the other photon counts, vs 800,000 photons). This is because the background is considered in the fitting process as a mathematical term. For the empirical lifetime response, a lower background slightly reduced the sensor expression dependence (<a href="#fig7" class="usa-link">Figure 7B</a>, middle, p&lt;0.05 for 350,000 photons or less, p&gt;0.1 for the other photon counts, vs 800,000 photons). Notably, a lower background signal caused greater tolerance of sensor expression level changes for the measurement of absolute empirical lifetime (<a href="#fig7" class="usa-link">Figure 7D</a>, middle). Thus, a lower background does not significantly alter the sensor level dependence when fitted <em>P<sub>1</sub></em> is used as the metric of measurement, but it significantly reduces the sensor level dependence when empirical lifetime is used as the metric of measurement.</p>
<p>Consistent with the results above, when there was no autofluorescence and the background was lower, there was no longer sensor expression dependence for fitted <em>P<sub>1</sub></em> (<a href="#fig7" class="usa-link">Figure 7A and C</a>, right panels; 7A: p&gt;0.99 for all photon counts vs 800,000 photons), and the sensor expression dependence also became much less pronounced for empirical lifetime (<a href="#fig7" class="usa-link">Figure 7B and D</a>, right; 7B: p&lt;0.05 for 150,000 photons or less, p&gt;0.3 for the other photon counts, vs 800,000 photons).</p>
<p>In summary, using FLiSimBA, we show quantitatively the benefit gained from future technological innovations: Developing sensors outside autofluorescence spectra or having a lower background can significantly reduce the dependence of fluorescence lifetime on sensor expression in biological systems, increasing the versatility of lifetime comparison across animals, brain regions, and chronic time scales, as well as allowing us to quantify the absolute levels of analytes.</p></section><section id="s2-6"><h3 class="pmc_sec_title">Innovating for multiplexing: revealing the feasibility and specifying sensor characteristics for multiplexed imaging with both fluorescence lifetime and intensity</h3>
<p>Understanding of any biological systems requires the quantification of the dynamics of many biological signals that interact together. Multiplexed imaging with sensors with different excitation and emission spectra is powerful, but only a few color channels can be effectively separated. Combining fluorescence lifetime and intensity could greatly enhance the capacity to monitor more biological signals simultaneously, and this idea is becoming increasingly feasible with the development of genetically encoded and chemigenetic sensors (<a href="#bib14" class="usa-link" aria-describedby="bib14">Farrants et al., 2024</a>; <a href="#bib15" class="usa-link" aria-describedby="bib15">Frei et al., 2022</a>; <a href="#bib60" class="usa-link" aria-describedby="bib60">van der Linden et al., 2021</a>). However, this approach was only used to distinguish sensors with different lifetimes in different spatial compartments (<a href="#bib15" class="usa-link" aria-describedby="bib15">Frei et al., 2022</a>; <a href="#bib45" class="usa-link" aria-describedby="bib45">Rahim et al., 2022</a>), or fluorophores that do not change dynamically and thus do not act as sensors (<a href="#bib18" class="usa-link" aria-describedby="bib18">Hamilton and Sanabria, 2019</a>; <a href="#bib45" class="usa-link" aria-describedby="bib45">Rahim et al., 2022</a>; <a href="#bib47" class="usa-link" aria-describedby="bib47">Rice and Kumar, 2014</a>; <a href="#bib63" class="usa-link" aria-describedby="bib63">Widengren et al., 2006</a>), and there was no quantitative framework for exploring the multiplexing capacity to analyze the dynamics of biosensors. Here, we investigated whether the combined signal of sensors with different lifetimes can be effectively deconvolved to quantitate the dynamic intensity change of each sensor – if this is feasible, this new approach could be used to simultaneously track the dynamics of multiple signals in the same spatial location, or in photometry experiments (<a href="#fig8" class="usa-link">Figure 8A</a>), thus greatly enhancing our ability to understand the systems biology of signaling.</p>
<figure class="fig xbox font-sm" id="fig8"><h4 class="obj_head">Figure 8. Feasibility and parameter requirements for multiplexed fluorescence intensity and lifetime imaging.</h4>
<figcaption><p>(<strong>A</strong>) Schematic illustrating the motivation for the question. Sensor 1 and sensor 2 are intensity-based sensors with the same color but different fluorescence lifetimes (with single exponential decay constants <em>τ<sub>1</sub></em> and <em>τ<sub>2</sub></em>, respectively). The fluorescence signals are combined, and then mathematically deconvolved into individual components. Dotted lines: true fluorescence lifetime histograms of the two sensors; solid lines: simulated fluorescence lifetime histograms with standard deviation (gray shading) before and after deconvolution. (<strong>B</strong>) Schematic illustrating the question under investigation: when sensor 1 and sensor 2 change fluorescence intensities and their combined fluorescence is collected simultaneously, can the respective intensities be deconvoluted from the combined measurements? (<strong>C</strong>) Deconvoluted number of sensor 1 photons based on different simulated sensor 1 photons. Sensor 2 was simulated with a photon count of 500 k. The left and right graphs are based on either 0.5 or 4 ns differences in the single exponential decay constants between the two sensors. The left and right graphs show that the variance in deconvoluted sensor 1 photons is less when the lifetime separation between sensor 1 and sensor 2 is greater. n=500 simulations. The data are represented as means and standard deviations. (<strong>D</strong>) Schematic illustrating the calculation of the statistical power for detecting a specific change in sensor 1 photon counts. (<strong>E</strong>) Heatmaps showing the statistical power of detecting specific changes in the intensity of sensor 1 with different sensor 1 baseline photons, different sensor 2 photons, and different lifetime separations between the two sensors. The red line denotes the change in intensity that provides a statistical power of 80%. (<strong>F</strong>) Relationship between the minimum detectable intensity change in sensor 1 to reach 80% statistical power and the lifetime difference between the two sensors. Curves were plotted with different sensor 1 baseline photons (left, 500 k sensor 2 photons) and different sensor 2 photons (right, 200 k sensor 1 baseline photons).</p></figcaption><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367299_elife-101559-fig8.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2706/12367299/26ed00dc10f2/elife-101559-fig8.jpg" loading="lazy" height="870" width="735" alt="Figure 8."></a></p>
<div class="p"><figure class="fig xbox font-sm" id="fig8s1"><h5 class="obj_head">Figure 8—figure supplement 1. Minimum detectable intensity change across diverse experimental conditions for multiplexed fluorescence lifetime imaging.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367299_elife-101559-fig8-figsupp1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2706/12367299/8587707a5595/elife-101559-fig8-figsupp1.jpg" loading="lazy" height="795" width="735" alt="Figure 8—figure supplement 1."></a></p>
<figcaption><div class="p">(<strong>A</strong>) Method of interpolating the minimum detectable intensity change to reach 80% statistical power. Statistical power and sensor 1 intensity change were fitted to a 5-parameter logistic function to calculate sensor 1 intensity change that gives a statistical power of 80%. Data are based on 500 k sensor 2 photons, 200 k sensor 1 baseline, and 0.5 ns lifetime separation between the two sensors. (<strong>B</strong>) Heatmaps showing the minimum detectable intensity change of sensor 1 to reach 80% statistical power, with different sensor 2 photons, sensor 1 baseline, and lifetime separation between the two sensors. Note that minimum detectable intensity change decreases as sensor 2 photons decreases, sensor 1 baseline increases, and lifetime separation increases. (<strong>C</strong>) Minimum detectable intensity changes of sensor 1 and sensor 2. Simulation assumed that the two sensors had the same baseline intensity, and both had a dynamic range in intensity that spanned from 1 to 2 fold of the baseline.</div></figcaption></figure></div>
<div class="p text-right font-secondary"><a href="figure/fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>We modeled a multiplexed system with two sensors, each displaying a single exponential decay in its fluorescence lifetime histogram but with different decay constants. Furthermore, in response to a change in the concentration of a biological analyte, the fluorescence lifetime of each sensor remains constant, whereas the fluorescence intensity is modulated. We used FLiSimBA to simulate multiplexed fluorescence lifetime imaging with realistic parameters and noise in biological systems. Subsequently, we deconvolved the combined histogram by fitting it with a double exponential decay equation (<a href="#equ3" class="usa-link">Equation 3</a>, Gauss-Newton nonlinear least-square fitting algorithm), whose decay time constants corresponded to those of the two sensor species (<a href="#fig8" class="usa-link">Figure 8A</a>).</p>
<p>We first investigated whether varying sensor intensities could be accurately deconvolved (<a href="#fig8" class="usa-link">Figure 8B</a>). Deconvolution accurately recovered the photon numbers attributed to each sensor compared with the actual sensor photons inputted into the model (<a href="#fig8" class="usa-link">Figure 8C</a>). The number of fitted sensor 1 photons (with shorter lifetime) was greater than the number of simulated sensor 1 photons, consistent with the contribution of autofluorescence whose average lifetime is closer to that of Sensor 1. Furthermore, when the lifetime decay constants of the two species were further separated, the variance in the fitted sensor photons decreased (<a href="#fig8" class="usa-link">Figure 8C</a>, STD = 0.012 for 4 ns separation and STD = 0.053–0.061 for 0.5 ns separation), indicating that further separation of lifetimes between the two sensors resulted in a better SNR.</p>
<p>To quantitatively evaluate the feasibility of multiplexed fluorescence lifetime imaging of sensors with the same spectra, we calculated the statistical power for detecting specific intensity changes of sensor 1 photons (<a href="#fig8" class="usa-link">Figure 8D</a>). The statistical power for detecting a specific intensity change was greater when the intensity change was greater and when the difference between the lifetime constants of the two sensors was greater (<a href="#fig8" class="usa-link">Figure 8E</a>, for any given heatmap). Furthermore, the statistical power was also greater when the number of starting sensor 1 photons was greater and the number of sensor 2 photons were lower (<a href="#fig8" class="usa-link">Figure 8E</a>, comparing between the heatmaps). Therefore, these results were valuable for demonstrating the feasibility of using fluorescence lifetime for multiplexed imaging and for quantitatively specifying the parameter space that provided sufficient statistical power.</p>
<p>To systematically define the parameter space, we determined the minimum intensity change that was detectable under different conditions. For any given sensor 1 starting photon count, sensor 2 photon count, and lifetime separation, we fitted the statistical power versus intensity change relationship to a 5-parameter logistic curve (<a href="#fig8s1" class="usa-link">Figure 8—figure supplement 1A</a>). Using the fitted curve, we calculated the intensity change that would yield a statistical power of 80% and defined it as the minimum detectable ΔF/F<sub>0</sub> (<a href="#fig8s1" class="usa-link">Figure 8—figure supplement 1A</a>). The minimum detectable ΔF/F<sub>0</sub> for sensor 1 decreased as the lifetime separation increased, as sensor 1 expression level increased, or as the number of sensor 2 photons decreased (<a href="#fig8" class="usa-link">Figure 8F</a> and <a href="#fig8s1" class="usa-link">Figure 8—figure supplement 1B</a>). If the two sensors had comparable dynamic ranges and comparable expression levels, the minimum detectable ΔF/F<sub>0</sub> for sensor 2 was similar to that of sensor 1 (<a href="#fig8s1" class="usa-link">Figure 8—figure supplement 1C</a>). Remarkably, only a 0.5 ns separation in lifetime constants between the two sensors could result in a minimum detectable ΔF/F<sub>0</sub> of 5% in sensor 1 if sensor 1 showed high but reasonable expression (800,000 photons or higher) and sensor 2 showed reasonable expression (500,000 photons or lower) (<a href="#fig8" class="usa-link">Figure 8F</a>). These results showed how the sensitivity of the intensity response quantitatively depended on the sensor expression and differences in lifetime between the two sensors.</p>
<p>In summary, our simulation showed that multiplexed fluorescence lifetime imaging was feasible with small shifts in the sensor properties. Furthermore, FLiSimBA could be used to quantitatively define the parameters of the sensors to yield a desired sensitivity. Thus, these results provide sensor developers with specific sensor properties to evolve toward. With simultaneous spectral and lifetime multiplexing, our proposed method enables the detection of a much larger number of signals dynamically, opening doors to better systems understanding of signaling within the cell and physiological interactions between different organs.</p></section></section><section id="s3"><h2 class="pmc_sec_title">Discussion</h2>
<p>Here, we provide a quantitative framework for analyzing fluorescence lifetime in realistic biological settings. We introduce FLiSimBA, a platform that accurately simulates fluorescence lifetime data for biological applications (<a href="#fig1" class="usa-link">Figures 1</a> and <a href="#fig2" class="usa-link">2</a>). With FLiSimBA, we address key questions in FLIM. First, to understand SNRs, we determined the number of photons required for different minimum detectable differences (<a href="#fig3" class="usa-link">Figure 3</a>). Additionally, we assessed the impact of hardware changes on SNRs by comparing GaAsP PMTs with hybrid detectors (<a href="#fig4" class="usa-link">Figure 4</a>). Moreover, we challenged the conventional view that biosensor expression levels do not affect fluorescence lifetime. We revealed how variation in sensor expression influences response amplitude (<a href="#fig5" class="usa-link">Figure 5</a>) and identified the amount of expression level variation that does not significantly alter fluorescence lifetime estimates (<a href="#fig6" class="usa-link">Figure 6</a>). Finally, we evaluated the feasibility, quantified the benefits, and specified the parameter space for FLIM innovations that minimize expression level dependence (<a href="#fig7" class="usa-link">Figure 7</a>) or enable multiplexed imaging with lifetime and intensity (<a href="#fig8" class="usa-link">Figure 8</a>). In summary, our study provides valuable insights and a quantitative framework for defining the power and limitations of fluorescence lifetime experiments in biological applications.</p>
<section id="s3-1"><h3 class="pmc_sec_title">FLiSimBA as a tool for realistic simulation of FLIM and FLiP experiments</h3>
<p>FLiSimBA is a necessary and useful tool for FLIM and FLiP experiments for a few reasons. First, it realistically simulates lifetime data from biological applications. Second, FLiSimBA and the quantitative framework we provide can precisely define the benefits and limitations of FLIM or FLiP experiments, enabling rigorous experimental design and data interpretation. Third, the versatility of FLiSimBA allows easy adaption to different sensors, tissues, organisms, and analysis methods (<a href="#bib2" class="usa-link" aria-describedby="bib2">Bajzer et al., 1991</a>; <a href="#bib11" class="usa-link" aria-describedby="bib11">Chen et al., 2022</a>; <a href="#bib21" class="usa-link" aria-describedby="bib21">Jo et al., 2005</a>; <a href="#bib35" class="usa-link" aria-describedby="bib35">Mannam et al., 2020</a>; <a href="#bib42" class="usa-link" aria-describedby="bib42">Periasamy, 1988</a>; <a href="#bib49" class="usa-link" aria-describedby="bib49">Rowley et al., 2011</a>; <a href="#bib51" class="usa-link" aria-describedby="bib51">Shen et al., 2024</a>; <a href="#bib52" class="usa-link" aria-describedby="bib52">Smith et al., 2019</a>; <a href="#bib53" class="usa-link" aria-describedby="bib53">Steinbach, 2012</a>; <a href="#bib54" class="usa-link" aria-describedby="bib54">Stringari et al., 2011</a>; <a href="#bib62" class="usa-link" aria-describedby="bib62">Wang et al., 2022</a>; <a href="#bib64" class="usa-link" aria-describedby="bib64">Wu et al., 2016</a>; <a href="#bib66" class="usa-link" aria-describedby="bib66">Xiao et al., 2023</a>; <a href="#bib72" class="usa-link" aria-describedby="bib72">Zickus et al., 2020</a>).</p>
<p>While this study used specific parameters from FLIM-AKAR and our imaging systems to demonstrate the power of FLiSimBA, FLiSimBA allows customizable inputs (<a href="#fig1" class="usa-link">Figure 1</a>) with minimal model assumptions to suit specific experimental settings. Users should carefully select mathematical models for their fluorescence sensors (<a href="#bib9" class="usa-link" aria-describedby="bib9">Chen et al., 2014</a>; <a href="#bib53" class="usa-link" aria-describedby="bib53">Steinbach, 2012</a>), measure autofluorescence that matches the age, brain region, and data collection conditions (e.g. ex vivo or in vivo) of the tissue (<a href="#bib8" class="usa-link" aria-describedby="bib8">Campbell et al., 2024</a>; <a href="#bib20" class="usa-link" aria-describedby="bib20">Jang et al., 2022</a>; <a href="#bib38" class="usa-link" aria-describedby="bib38">Morrow et al., 2024</a>; <a href="#bib56" class="usa-link" aria-describedby="bib56">Tehrani et al., 2023</a>), as well as measure afterpulse ratio, background, and dark currents relevant to their specific FLIM or FLiP settings. While these input parameters do not alter the general insights from this study, they affect quantitative values and it is thus crucial to accurately match experimental conditions to enhance the rigor of experimental design and data interpretation.</p></section><section id="s3-2"><h3 class="pmc_sec_title">Insights into measurement uncertainty of FLIM</h3>
<p>Beyond providing a package to realistically simulate FLIM data, our results also provide insights into critical questions for FLIM usage in biology that were not previously available. Quantifying the uncertainty of fluorescence lifetime measurements is a key aspect of any measurement but is often neglected. Our results provide the amplitude of the error bars and reveal the sources of bias and noise in fluorescence lifetime measurements in realistic biological experiments. Therefore, these results facilitate the disambiguation of signals from noise in data interpretation and empower FLIM users to design optimal experiments by precisely evaluating the compromise between the SNR, field-of-view size, and imaging speed.</p></section><section id="s3-3"><h3 class="pmc_sec_title">Insights into expression level dependence of FLIM</h3>
<p>Our analysis of expression level dependence demonstrates that small quantitative changes can impact qualitative conclusions: Because of the influence of autofluorescence, afterpulse, and background, changes in sensor expression levels can lead to changes in lifetime measurements. This challenges the widely held assumptions about the very advantage of fluorescence lifetime measurements: Its insensitivity to sensor expression levels. While this issue is less problematic when the same sample is compared over short periods (for example, minutes), it can lead to misinterpretation when fluorescence lifetime is compared across prolonged periods (for example, days or weeks) or between samples with different sensor expression levels (for example, between individual animals). Thus, apparent changes in fluorescence lifetime observed over days, across cell types, or subcellular compartments may reflect variations in sensor expression levels rather than true differences in biological signals (<a href="#fig6" class="usa-link">Figure 6</a>). Therefore, considering biologically realistic factors in FLiSimBA is essential, as it qualitatively impacts the conclusions.</p>
<p>Importantly, our results not only highlight this issue but also provide a solution by defining the range where expression levels do not significantly affect lifetime (<a href="#fig5" class="usa-link">Figures 5</a> and <a href="#fig6" class="usa-link">6</a>). Practically, for a sensor with medium brightness delivered via in utero electroporation, adeno-associated virus, or as a knock-in gene, the brightness may not always fall within the expression level-independent regime. Therefore, these analyses are critical for identifying the conditions in which FLIM and FLiP can be reliably used to compare biosensor measurements over chronic time periods, across animals, and to quantify absolute levels of biological signals.</p></section><section id="s3-4"><h3 class="pmc_sec_title">Insights into future FLIM innovation</h3>
<p>Our quantitative platform demonstrates how we can improve FLIM through future innovation. First, whereas our simulations showed little advantage of hybrid detectors over GaAsP PMTs in terms of the SNR (<a href="#fig4" class="usa-link">Figure 4</a>), different types of hybrid detectors offer narrower IRFs at the cost of lower quantum efficiency (QE). FLiSimBA enables the evaluation of the trade-off between IRF width and QE. Additionally, hybrid detectors are more advantageous for sensors with shorter fluorescence lifetimes (<a href="#bib58" class="usa-link" aria-describedby="bib58">Trinh and Esposito, 2021</a>) and FLiSimBA allows exploration of the range in which they provide clear benefits in biological settings.</p>
<p>Second, we quantified how the development of red or far-red sensors outside the autofluorescence spectrum, along with minimizing detector dark currents, can significantly reduce or eliminate the expression level dependence of lifetime measurements (<a href="#fig7" class="usa-link">Figure 7</a>). This opens exciting directions for sensor makers and hardware developers. The impact of these innovations on fluorescence lifetime consistency depends on the metrics used to measure lifetime (<a href="#fig7" class="usa-link">Figure 7</a>). Fitted parameters, such as fitted <em>P<sub>1</sub></em>, are preferred over empirical lifetime when the lifetime histogram can be described by multi-exponential decay equations and sufficient photons are collected for fitting, because they are less sensitive to background interference (<a href="#fig7" class="usa-link">Figure 7</a>). In contrast, empirical lifetime is used when sensor lifetime decay is too complex, or when photon counts are too low for fitting. FLiSimBA can be used to test multiple fitting methods and lifetime metrics (<a href="#bib2" class="usa-link" aria-describedby="bib2">Bajzer et al., 1991</a>; <a href="#bib11" class="usa-link" aria-describedby="bib11">Chen et al., 2022</a>; <a href="#bib21" class="usa-link" aria-describedby="bib21">Jo et al., 2005</a>; <a href="#bib35" class="usa-link" aria-describedby="bib35">Mannam et al., 2020</a>; <a href="#bib42" class="usa-link" aria-describedby="bib42">Periasamy, 1988</a>; <a href="#bib49" class="usa-link" aria-describedby="bib49">Rowley et al., 2011</a>; <a href="#bib51" class="usa-link" aria-describedby="bib51">Shen et al., 2024</a>; <a href="#bib52" class="usa-link" aria-describedby="bib52">Smith et al., 2019</a>; <a href="#bib53" class="usa-link" aria-describedby="bib53">Steinbach, 2012</a>; <a href="#bib54" class="usa-link" aria-describedby="bib54">Stringari et al., 2011</a>; <a href="#bib62" class="usa-link" aria-describedby="bib62">Wang et al., 2022</a>; <a href="#bib64" class="usa-link" aria-describedby="bib64">Wu et al., 2016</a>; <a href="#bib66" class="usa-link" aria-describedby="bib66">Xiao et al., 2023</a>; <a href="#bib72" class="usa-link" aria-describedby="bib72">Zickus et al., 2020</a>), and this will be an important future direction for identifying the best analysis method for specific experimental conditions.</p>
<p>Third, we demonstrate the feasibility and specify sensor parameters that enable the use of fluorescence lifetime and color spectrum as two orthogonal axes to distinguish many biological signals through intensity-based real-time imaging of biosensors (<a href="#fig8" class="usa-link">Figure 8</a>). This approach promises to open doors to enhancing the ability of multiplexed imaging by orders of magnitude.</p>
<p>Thus, FLiSimBA and our quantitative framework are instrumental for enabling optimal experimental design, rigorous data interpretation, and pushing the limit of FLIM imaging through innovations.</p></section></section><section id="s4"><h2 class="pmc_sec_title">Materials and methods</h2>
<section class="tw xbox font-sm" id="keyresource"><h3 class="obj_head">Key resources table.</h3>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" valign="bottom" rowspan="1" colspan="1">Reagent type (species) or resource</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">Designation</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">Source or reference</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">Identifiers</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">Additional information</th>
</tr></thead>
<tbody>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">Software, algorithm</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">Matlab</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">Matlab</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">RRID:<a href="https://identifiers.org/RRID:SCR_001622" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SCR_001622</a>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">Software, algorithm</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">Python</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<a href="https://www.python.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Python.org</a>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">RRID:<a href="https://identifiers.org/RRID:SCR_008394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SCR_008394</a>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">Software, algorithm</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">Fluorescence Lifetime Simulation for Biological Applications (FLiSimBA)</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">Yao Chen’s Laboratory (<a href="#bib33" class="usa-link" aria-describedby="bib33">Ma et al., 2025</a>)</td>
<td align="left" valign="bottom" rowspan="1" colspan="1"></td>
<td align="left" valign="bottom" rowspan="1" colspan="1">Authors: Pingchuan Ma, Peter Chen, Yao Chen, 2025, version 1, <a href="https://github.com/YaoChenLabWashU/Publication/tree/main/Simulation_manuscript" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/YaoChenLabWashU/Publication/tree/main/Simulation_manuscript</a>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/keyresource/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section id="s4-1"><h3 class="pmc_sec_title">Animals</h3>
<p>All procedures for rodent husbandry and surgery were performed following protocols approved by the Washington University Institutional Animal Care and Use Committee and in accordance with National Institutes of Health guidelines. CD-1 mice (Envigo #030) were used. The experiments were performed according to the Animal Research: Reporting of In Vivo Experiments (ARRIVE) guidelines (<a href="#bib41" class="usa-link" aria-describedby="bib41">Percie du Sert et al., 2020</a>).</p></section><section id="s4-2"><h3 class="pmc_sec_title">DNA plasmid</h3>
<p>For experimentally determined FLIM-AKAR data, AAV-FLIM-AKAR (<a href="#bib9" class="usa-link" aria-describedby="bib9">Chen et al., 2014</a>) (Addgene #63058) was used to express the FLIM-AKAR sensor in the primary somatosensory cortex by in utero electroporation (<a href="#bib10" class="usa-link" aria-describedby="bib10">Chen et al., 2017</a>).</p></section><section id="s4-3"><h3 class="pmc_sec_title">Acute brain slice preparation</h3>
<p>Mice were anesthetized with isoflurane at 15–19 days of age and then decapitated. Their brains were rapidly dissected and put in sucrose-based cutting solution (concentrations in mM: 75 sucrose, 2.5 KCl, 1.25 NaH<sub>2</sub>PO<sub>4</sub>, 25 NaHCO<sub>3</sub>, 87 NaCl, 25 glucose, 1 MgCl<sub>2</sub>). 300 μm-thick coronal sections containing the primary somatosensory cortex were obtained with a vibratome (Leica Instruments, VT1200S) in cold sucrose-based cutting solution. After sectioning, the slices were transferred to artificial cerebral spinal fluid (ACSF) (concentrations in mM: 127 NaCl, 2.5 KCl, 1.25 NaH<sub>2</sub>PO<sub>4</sub>, 25 NaHCO<sub>3</sub>, 2 CaCl<sub>2</sub>, 1 MgCl<sub>2</sub>, and 25 glucose) and incubated at 34℃ for 10 min for recovery. The slices were kept at room temperature in ACSF with 5% CO<sub>2</sub> and 95% O<sub>2</sub>. The slices were then transferred to a microscope chamber and ACSF was perfused at a flow rate of 2–4 mL/min for imaging.</p></section><section id="s4-4"><h3 class="pmc_sec_title">Two-photon fluorescence lifetime imaging microscopy (2pFLIM)</h3>
<p>2pFLIM was performed as described previously (<a href="#bib9" class="usa-link" aria-describedby="bib9">Chen et al., 2014</a>; <a href="#bib10" class="usa-link" aria-describedby="bib10">Chen et al., 2017</a>; <a href="#bib32" class="usa-link" aria-describedby="bib32">Ma et al., 2024</a>; <a href="#bib57" class="usa-link" aria-describedby="bib57">Tilden et al., 2024</a>). A custom-built microscope with a mode-locked laser source (Spectra-Physics, Insight X3 operating at 80 MHz) was used. Photons were collected with fast photomultiplier tubes (PMTs, Hamamatsu, H10770PB-40). A 60 X (Olympus, NA 1.1) objective was used. Image acquisition was performed with the custom-written software ScanImage (<a href="#bib9" class="usa-link" aria-describedby="bib9">Chen et al., 2014</a>; <a href="#bib10" class="usa-link" aria-describedby="bib10">Chen et al., 2017</a>; <a href="#bib32" class="usa-link" aria-describedby="bib32">Ma et al., 2024</a>; <a href="#bib44" class="usa-link" aria-describedby="bib44">Pologruto et al., 2003</a>; <a href="#bib57" class="usa-link" aria-describedby="bib57">Tilden et al., 2024</a>) in MATLAB 2012b. The excitation wavelength was 920 nm. Emission light was collected through a dichroic mirror (FF580-FDi01−25X36, Semrock) and a band-pass filter (FF03-525/50-25, Semrock). Images covering 15 μm × 15 μm fields of view were collected at 128x128 pixels via a frame scan at 4 Hz. The FLIM board SPC-150 (Becker and Hickl GmbH) was used, and time-correlated single photon counting was performed with 256 time channels. Photons from 20 frames were pooled for fluorescence lifetime calculations. Only healthy cells (judged by gradient contrast images) at 30–50 μm below the slice surface were selected. Each individual cell was analyzed as a region of interest (ROI). Photons from a given ROI were pooled for further analysis.</p></section><section id="s4-5"><h3 class="pmc_sec_title">Experimental data collection, parameter determination, and simulation</h3>
<p>The simulation packages were provided in MATLAB and Python. The simulation was performed in MATLAB 2022a or Python with the following steps (<a href="#fig1" class="usa-link">Figure 1A</a>). The final simulated histograms consisted of the IRF-convolved sensor fluorescence, autofluorescence, afterpulse, and background fluorescence. The simulations were performed with 256 time channels for each laser cycle from an 80 MHz laser (12.5 ns interpulse interval). For each <em>P<sub>1</sub></em> and sensor photon number condition, the simulation was repeated 500 times.</p>
<section id="s4-5-1"><h4 class="pmc_sec_title">Generation of photon populations for sensor fluorescence</h4>
<p>For FLIM-AKAR sensor fluorescence, <em>τ<sub>1</sub></em> and <em>τ<sub>2</sub></em> were previously determined to be 2.14 ns and 0.69 ns, respectively (<a href="#bib9" class="usa-link" aria-describedby="bib9">Chen et al., 2014</a>; <a href="#bib10" class="usa-link" aria-describedby="bib10">Chen et al., 2017</a>). To determine the appropriate photon counts and <em>P<sub>1</sub></em> range for simulation, sensor fluorescence histogram was fitted with <a href="#equ3" class="usa-link">Equation 3</a>.</p>
<p>For <a href="#fig1" class="usa-link">Figures 1</a>—<a href="#fig7" class="usa-link">7</a>, to simulate the fluorescence lifetime with double exponential decay, we generated a population of photons with <a href="#equ1" class="usa-link">Equation 1</a> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="inf1"><msub><mrow><mi>F</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> equal to 1,000,000. We generated photon populations and corresponding fluorescence lifetime histograms with <em>P<sub>1</sub></em> ranging from 0.4 to 0.6 with an increment of 0.01.</p></section><section id="s4-5-2"><h4 class="pmc_sec_title">Sensor fluorescence sampling and IRF convolution</h4>
<p>For sensor fluorescence sampling, a specific number of photons were randomly drawn with replacement from the corresponding population generated with double exponential decay. IRF convolution of the fluorescence lifetime histogram was performed: the lifetime of each photon in the sample was redistributed along the time channels based on the probability of the IRF distribution. Following convolution, the histogram was wrapped around such that any photons whose lifetimes were beyond 12.5 ns (inter-pulse interval of laser) were redistributed to the next cycle.</p>
<p>In all the figures except for <a href="#fig4" class="usa-link">Figure 4</a>, the IRF was empirically measured by second harmonic generation of the mouse tail with excitation at 1050 nm. The lifetime histogram was normalized to the total photon number and used as the IRF. For <a href="#fig4" class="usa-link">Figure 4</a>, the IRFs of both systems were modeled as Gaussian distributions with different Gaussian widths. The mean of the Gaussian distribution (<em>μ</em>) was set as the peak channel of the experimentally collected IRF. The full width at half maximum (FWHM) of the Gaussian IRF of the GaAsP PMT was set to match the FWHM of the experimentally collected IRF (340 ps). The FWHM of the hybrid detector Gaussian IRF was set as 120 ps based on the HPM-100–40 model (Becker &amp; Hickl) (<a href="#bib4" class="usa-link" aria-describedby="bib4">Becker et al., 2011</a>). The standard deviation (STD, <em>σ</em>) of the Gaussian distribution of the IRFs was determined based on the relationship with the FWHM:</p>
<table class="disp-formula p" id="equ4"><tr>
<td class="formula"><math id="m4" display="block"><mstyle displaystyle="true" scriptlevel="0"><mrow><mstyle displaystyle="true" scriptlevel="0"><mi>F</mi><mi>W</mi><mi>H</mi><mi>M</mi><mo>=</mo><mn>2</mn><msqrt><mn>2</mn><mi>l</mi><mi>n</mi><mn>2</mn></msqrt><mo>∗</mo><mi>σ</mi></mstyle></mrow></mstyle></math></td>
<td class="label">(4)</td>
</tr></table>
<p>The Gaussian distribution was defined as follows: </p>
<table class="disp-formula p" id="equ5"><tr>
<td class="formula"><math id="m5" display="block"><mstyle displaystyle="true" scriptlevel="0"><mrow><mstyle displaystyle="true" scriptlevel="0"><mi>G</mi><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mrow><mi>σ</mi><msqrt><mn>2</mn><mi>π</mi></msqrt></mrow></mfrac><mo>∗</mo><msup><mi>e</mi><mrow><mfrac><mrow><mo>−</mo><mn>1</mn></mrow><mn>2</mn></mfrac><msup><mrow><mo>(</mo><mfrac><mrow><mi>t</mi><mo>−</mo><mi>μ</mi></mrow><mi>σ</mi></mfrac><mo>)</mo></mrow><mrow><mn>2</mn></mrow></msup></mrow></msup></mstyle></mrow></mstyle></math></td>
<td class="label">(5)</td>
</tr></table>
<p>The Gaussian IRF was generated by normalization of <em>G(t</em>) against the total photon count.</p>
<p>For <a href="#fig4" class="usa-link">Figure 4G and H</a>, to isolate the noise from the sampling of photons, sensor fluorescence sampling was performed 500 times, followed by analytical IRF convolution of each sensor fluorescence decay histogram. To isolate the noise from sampling from the IRF distribution, only one sensor photon decay histogram was generated, and each photon was reassigned to a single lifetime value based on the probability density distribution of the IRF, and this procedure was repeated 500 times.</p></section><section id="s4-5-3"><h4 class="pmc_sec_title">Autofluorescence</h4>
<p>All biological tissues exhibit autofluorescence due to fluorescent cellular components and metabolites, such as nicotinamide-adenine dinucleotide (NAD), flavins, and aromatic amino acids (<a href="#bib16" class="usa-link" aria-describedby="bib16">Georgakoudi and Quinn, 2023</a>; <a href="#bib32" class="usa-link" aria-describedby="bib32">Ma et al., 2024</a>; <a href="#bib34" class="usa-link" aria-describedby="bib34">Malak et al., 2022</a>).</p>
<p>To determine the autofluorescence of brain tissue, we imaged acute brain slices from mice aged 15–19 days postnatal that did not express sensors. Neurons from the primary somatosensory cortex were imaged under the following conditions: 920 nm excitation light, 2.5 mW power, 30–50 μm below the slice surface, 20 frames pooled over 5 s, and a field of view of 15 μm x 15 μm. Fluorescence decay histograms from 19 images of two brain slices from a single mouse were averaged. These histograms included contributions from both autofluorescence and background signals (due to dark current and ambient light leakage). The average autofluorescence histogram was fitted with a double exponential function with background (<a href="#equ3" class="usa-link">Equation 3</a>) to determine the parameters <em>τ<sub>1</sub></em>, <em>τ<sub>2</sub></em>, <em>P<sub>1</sub></em>, and <em>P<sub>2</sub></em> of autofluorescence as well as background signals. The number of photons from autofluorescence (<em>F<sub>auto</sub></em>) was calculated as the remaining photons after subtracting the background signals (<em>F<sub>background</sub></em>) from the average histogram. In this study, <em>F<sub>auto</sub></em> was determined to be 4560 photons and <em>F<sub>background</sub></em> was 3484 photons.</p>
<p>For simulations, we introduced up to 10% fluctuation in the number of photons by randomly drawing an integer within the range of <em>F<sub>auto</sub></em>*(1 ± 5%).</p>
<p>For <a href="#fig4" class="usa-link">Figure 4</a>, the autofluorescence lifetime was simulated as double exponential decay (<a href="#equ1" class="usa-link">Equation 1</a>). For the remaining figures, the autofluorescence lifetime was sampled from the empirical autofluorescence lifetime distribution, where the background was subtracted from the average autofluorescence lifetime histogram.</p></section><section id="s4-5-4"><h4 class="pmc_sec_title">Afterpulse and background fluorescence</h4>
<p>The afterpulse ratio of the PMT was derived from the IRF histogram and the background fluorescence measurement described above. Then the photons per channel were averaged at the end of the IRF histograms, where the distribution was even across time channels. The number was subtracted by the background fluorescence and the result was used as the afterpulse. Subsequently, the ratio between the number of photons contributing to the afterpulse and the total number of photons from the IRF histogram was calculated as the afterpulse ratio. For GaAsP PMTs, an afterpulse ratio of 0.32% was used in all figures except for <a href="#fig2" class="usa-link">Figure 2</a> and <a href="#fig2s1" class="usa-link">Figure 2—figure supplement 1</a>, where a ratio of 0.40% was used to match the specific experimental data.</p>
<p>Afterpulse and background fluorescence were simulated by sampling with replacement from an even distribution across time channels. The number of photons contributing to background fluorescence (<em>F<sub>background</sub></em>) was determined from autofluorescence fitting. Up to 10% fluctuation was introduced to the number of photons with the random draw of an integer within the range of <em>F<sub>background</sub></em>*(1 ± 5%). For the low background conditions in <a href="#fig7" class="usa-link">Figure 7</a>, the number of photons contributing to the background was half of the background photon count for the remaining figures. The number of photons from the afterpulse was determined by the afterpulse ratio (0.32% or 0.40% for GaAsP PMTs and 0 for hybrid detectors) multiplied by the number of photons from the sensor fluorescence.</p>
<p>Please note that we used the specific parameters described above for this study, including using the double-exponential decay model and parameters for FLIM-AKAR, the measured autofluorescence amount and distribution from acute mouse brain slices at a specific age, the empirically measured IRF, afterpulse ratio, and background fluorescence for our 2pFLIM microscope. These input parameters (<a href="#fig1" class="usa-link">Figure 1</a>) should be altered to adapt to different biological applications. Although they do not change the conclusions of this study (e.g. there is an expression level-dependent and -independent regime for fluorescence lifetime), the specific input parameters would alter the quantitative thresholds (e.g. the precise threshold above which fluorescence lifetime is not significantly altered by expression levels).</p></section></section><section id="s4-6"><h3 class="pmc_sec_title">FLIM analysis</h3>
<p>Two metrics were used for subsequent data visualization and analysis. First, after the fluorescence lifetime histograms were fitted with <a href="#equ3" class="usa-link">Equation 3</a>, fitted <em>P<sub>1</sub></em>, corresponding to the proportion of slower decay (2.14 ns), was used for data visualization and analysis. Second, the empirical lifetimes of all the photons were calculated based on <a href="#equ2" class="usa-link">Equation 2</a>.</p>
<p>For all figures except for <a href="#fig2" class="usa-link">Figure 2</a> and <a href="#fig2s1" class="usa-link">Figure 2—figure supplement 1</a>, the time range from 0.489 ns to 11.5 ns was used for both <em>P<sub>1</sub></em> fitting and empirical lifetime calculations, with the first time channel within this range used as time 0.0488 ns (12.5 ns/256 time channels) in the calculation.</p>
<p>For comparison between experimental and simulated data, we determined the photon counts and <em>P<sub>1</sub></em> of experimental conditions from fluorescence lifetime histograms from whole fields of view in brain slices. We subtracted empirically collected autofluorescence (which includes dark currents, SHG, and background) and afterpulse (with an afterpulse ratio of 0.4%) from the histogram to derive the sensor histogram. Sensor histograms from seven experimental acquisitions were averaged and fitted with <a href="#equ3" class="usa-link">Equation 3</a>, using a lifetime range of 1.8–11.5 ns, with <em>SHG</em> and <em>F<sub>background</sub></em> set to 0. The fitted <em>P<sub>1</sub></em> (0.5854) and photon number (652,126) were used for simulations in <a href="#fig2" class="usa-link">Figure 2B</a>, <a href="#fig2s1" class="usa-link">Figure 2—figure supplement 1B and C</a>. To minimize system artifact interference at the histogram edges, fitted <em>P<sub>1</sub></em> and empirical lifetime were calculated within the 1.8–11.5 ns range for <a href="#fig2" class="usa-link">Figure 2</a> and <a href="#fig2s1" class="usa-link">Figure 2—figure supplement 1B and C</a>.</p></section><section id="s4-7"><h3 class="pmc_sec_title">Quantification and statistical analysis</h3>
<p>For each simulated condition, the mean and STD of the fitted <em>P<sub>1</sub></em> or the empirical lifetime of the 500 simulation repeats were calculated.</p>
<p>For <a href="#fig3" class="usa-link">Figures 3</a> and <a href="#fig4" class="usa-link">4</a>, the minimum detectable difference (MDD) was calculated by:</p>
<table class="disp-formula p" id="equ6"><tr>
<td class="formula"><math id="m6" display="block"><mstyle displaystyle="true" scriptlevel="0"><mrow><mstyle displaystyle="true" scriptlevel="0"><mi>M</mi><mi>D</mi><mi>D</mi><mo>=</mo><mi>z</mi><mo>∗</mo><mi>S</mi><mi>E</mi><mo stretchy="false">(</mo><mrow><mover><mrow><mi>d</mi><mi>i</mi><mi>f</mi><mi>f</mi></mrow><mo>^</mo></mover></mrow><mo stretchy="false">)</mo></mstyle></mrow></mstyle></math></td>
<td class="label">(6)</td>
</tr></table>
<p>where SE refers to the standard error, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="inf2"><mstyle displaystyle="true" scriptlevel="0"><mrow><mrow><mover><mrow><mi>d</mi><mi>i</mi><mi>f</mi><mi>f</mi></mrow><mo>^</mo></mover></mrow></mrow></mstyle></math></span> is the estimated mean difference between two distributions. With a significance level of 0.05 and a power of 0.8, the z value is 2.806. which was used for the calculations in this study. Under a certain sensor photon number, the STD of the two metrics under different <em>P<sub>1</sub></em> conditions (condition 1 and condition 2) were similar. Thus, given a certain sensor photon count, the STD under <em>P<sub>1</sub></em> condition (0.5 in <a href="#fig2" class="usa-link">Figure 2</a>) was used to calculate the MDD:</p>
<table class="disp-formula p" id="equ7"><tr>
<td class="formula"><math id="m7" display="block"><mstyle displaystyle="true" scriptlevel="0"><mrow><mstyle displaystyle="true" scriptlevel="0"><mi>M</mi><mi>D</mi><mi>D</mi><mo>=</mo><mi>z</mi><mo>∗</mo><mfrac><msqrt><mi>S</mi><mi>T</mi><msub><mi>D</mi><mrow><msup><mrow><mo>(</mo><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>d</mi><mi>i</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi><mspace width="thinmathspace"></mspace><mn>1</mn></mrow><mo>)</mo></mrow><mrow><mn>2</mn></mrow></msup></mrow></msub><mo>+</mo><mi>S</mi><mi>T</mi><msub><mi>D</mi><mrow><msup><mrow><mo>(</mo><mrow><mi>c</mi><mi>o</mi><mi>n</mi><mi>d</mi><mi>i</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi><mspace width="thinmathspace"></mspace><mn>2</mn></mrow><mo>)</mo></mrow><mrow><mn>2</mn></mrow></msup></mrow></msub></msqrt><msqrt><mi>n</mi></msqrt></mfrac><mo>∽</mo><mtext> </mtext><mfrac><mrow><mi>z</mi><mo>∗</mo><msqrt><mn>2</mn></msqrt><mo>∗</mo><mi>S</mi><mi>T</mi><mi>D</mi></mrow><msqrt><mi>n</mi></msqrt></mfrac></mstyle></mrow></mstyle></math></td>
<td class="label">(7)</td>
</tr></table>
<p>where n is the number of data pairs used to analyze whether there was a significant change in the fitted <em>P<sub>1</sub></em> or empirical lifetime.</p>
<p>For <a href="#fig6" class="usa-link">Figures 6</a> and <a href="#fig7" class="usa-link">7</a>, to determine whether a certain amount of sensor expression-induced apparent change in the fitted <em>P<sub>1</sub></em> or empirical lifetime was significant, t tests were used to compare whether there was any statistically significant difference between two distributions with the same simulated <em>P<sub>1</sub></em> but different photon numbers. The critical value of the t statistic is z=1.96 for a significance level of 0.05. The equation <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="inf3"><mstyle displaystyle="true" scriptlevel="0"><mrow><mi>z</mi><mo>∗</mo><mi>S</mi><mi>E</mi><mo stretchy="false">(</mo><mrow><mover><mrow><mi>d</mi><mi>i</mi><mi>f</mi><mi>f</mi></mrow><mo>^</mo></mover></mrow><mo stretchy="false">)</mo></mrow></mstyle></math></span> was used to calculate the fitted <em>P<sub>1</sub></em> or empirical lifetime difference that would be statistically significant. As the photon count increased, there was an intersection where the mean fitted <em>P<sub>1</sub></em> or empirical lifetime difference between the two distributions became less than the difference that would reach statistical significance, and this intersection point (determined by linear interpolation of the curves plotted on a log scale for both axes) was used to determine the minimum number of sensor photons required to tolerate a specific amount of change in sensor fluorescence.</p>
<p>Detailed information on the quantification, sample size, and statistical tests used are summarized in the Figure Legends, Figures, and Results. T tests were performed to test whether two distributions had equal means. For analysis of variance, one-way or two-way ANOVA was performed followed by multiple comparison tests.</p></section><section id="s4-8"><h3 class="pmc_sec_title">Simulation and analysis of multiplexed imaging with fluorescence intensity and lifetime data</h3>
<p>For <a href="#fig8" class="usa-link">Figure 8</a> and <a href="#fig8s1" class="usa-link">Figure 8—figure supplement 1</a>, the simulations were performed with a laser cycle of 50 ns to capture the fluorescence decay curve of Sensor 2 with a longer lifetime constant. The fluorescence lifetime histograms of sensor 1 and sensor 2 were simulated as single exponential decays:</p>
<table class="disp-formula p" id="equ8"><tr>
<td class="formula"><math id="m8" display="block"><mstyle displaystyle="true" scriptlevel="0"><mrow><mi>F</mi><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>=</mo><msub><mi>F</mi><mrow><mn>0</mn></mrow></msub><msup><mi>e</mi><mrow><mfrac><mrow><mo>−</mo><mi>t</mi></mrow><mi>τ</mi></mfrac></mrow></msup><mo>⊗</mo><mi>I</mi><mi>R</mi><mi>F</mi></mrow></mstyle></math></td>
<td class="label">(8)</td>
</tr></table>
<p> where the decay constants <em>τ</em> for sensor 1 and sensor 2 are different.</p>
<p>To simulate the fluorescence lifetime with single exponential decay, we generated a population of photons with <a href="#equ8" class="usa-link">Equation 8</a> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="inf4"><msub><mrow><mi>F</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> equal to 1,000,000. For sensor 1, the population was generated with <em>τ</em>=2.0 ns. For sensor 2, the population was generated with <em>τ</em>=2.5, 3.0, 3.5, 4.0, or 6.0 ns.</p>
<p>Fluorescence lifetime histograms of sensor 1 and sensor 2 were generated by sampling from the corresponding population, followed by IRF convolution. The histograms of sensor 1 and sensor 2 were added together as the combined histogram. Autofluorescence and background signals were simulated with the same probability density function as above but with 3.2-fold of photon numbers to accommodate for the longer acquisition time needed with a longer laser cycle to reach the same number of photons. Afterpulse was added proportional to the sensor photons as described above.</p>
<p>The combined histograms were fitted by a double exponential decay (<a href="#equ3" class="usa-link">Equation 3</a>), with fixed <em>τ<sub>1</sub></em> and <em>τ<sub>2</sub></em> that were used for the simulation that generated the histograms. The fitting was performed with the Gauss-Newton nonlinear least-square fitting algorithm. <em>F<sub>0,</sub> P<sub>1</sub></em>, and <em>P<sub>2</sub></em> generated from the fitting were used to calculate the number of photons that contributed to sensor 1 and sensor 2, respectively.</p>
<p>The statistical power to detect the difference between two distributions of sensor intensity was calculated as follows:</p>
<table class="disp-formula p" id="equ9"><tr>
<td class="formula"><math id="m9" display="block"><mstyle displaystyle="true" scriptlevel="0"><mrow><mstyle displaystyle="true" scriptlevel="0"><mi>β</mi><mo>=</mo><mn>1</mn><mo>−</mo><mi>n</mi><mi>o</mi><mi>r</mi><mi>m</mi><mi>c</mi><mi>d</mi><mi>f</mi><mrow><mo>(</mo><mrow><mn>1.96</mn><mo>−</mo><mfrac><mrow><mi mathvariant="normal">Δ</mi><mi>I</mi></mrow><mrow><mi>S</mi><mi>E</mi><mo stretchy="false">(</mo><mrow><mover><mrow><mi>d</mi><mi>l</mi><mi>f</mi><mi>f</mi></mrow><mo>^</mo></mover></mrow><mo stretchy="false">)</mo></mrow></mfrac></mrow><mo>)</mo></mrow></mstyle></mrow></mstyle></math></td>
<td class="label"> (9)</td>
</tr></table>
<p>where <em>normcdf</em> is the cumulative distribution function of a normal distribution, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="inf5"><mstyle displaystyle="true" scriptlevel="0"><mrow><mi mathvariant="normal">Δ</mi><mi>I</mi></mrow></mstyle></math></span> is the intensity (photon number) change to be detected, SE refers to the standard error, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="inf6"><mstyle displaystyle="true" scriptlevel="0"><mrow><mrow><mover><mrow><mi>d</mi><mi>i</mi><mi>f</mi><mi>f</mi></mrow><mo>^</mo></mover></mrow></mrow></mstyle></math></span> is the estimator of the mean difference between two distributions.</p>
<p>The relationship between the statistical power <em>β</em> and the intensity change ratio (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="inf7"><mstyle displaystyle="true" scriptlevel="0"><mrow><mi mathvariant="normal">Δ</mi><mi>F</mi><mrow><mo>/</mo></mrow><msub><mi>F</mi><mrow><mn>0</mn></mrow></msub></mrow></mstyle></math></span>) was fitted to a 5-parameter logistic curve:</p>
<table class="disp-formula p" id="equ10"><tr>
<td class="formula"><math id="m10" display="block"><mstyle displaystyle="true" scriptlevel="0"><mrow><msub><mi>β</mi><mrow><mrow><mo>(</mo><mrow><mi mathvariant="normal">Δ</mi><mi>F</mi><mrow><mo>/</mo></mrow><msub><mi>F</mi><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow></mrow></msub><mo>=</mo><mi>D</mi><mo>+</mo><mfrac><mrow><mi>A</mi><mo>−</mo><mi>D</mi></mrow><msup><mrow><mo>(</mo><mrow><mn>1</mn><mo>+</mo><msup><mrow><mo>(</mo><mfrac><mrow><mi mathvariant="normal">Δ</mi><mi>F</mi><mrow><mo>/</mo></mrow><msub><mi>F</mi><mrow><mn>0</mn></mrow></msub></mrow><mi>C</mi></mfrac><mo>)</mo></mrow><mrow><mi>B</mi></mrow></msup></mrow><mo>)</mo></mrow><mrow><mi>E</mi></mrow></msup></mfrac></mrow></mstyle></math></td>
<td class="label"> (10)</td>
</tr></table>
<p>where A, B, C, D, and E are constants in the equation. The intensity change ratio corresponding to a statistical power of 80% is determined by solving the equation with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="inf8"><msub><mrow><mi>β</mi></mrow><mrow><mfenced separators="|"><mrow><mo>∆</mo><mrow><mrow><mi>F</mi></mrow><mo>/</mo><mrow><msub><mrow><mi>F</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow></mrow></mrow></mfenced></mrow></msub><mo>=</mo><mn>0.8</mn></math></span> and defined as the minimum detectable <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="inf9"><mo>∆</mo><mrow><mrow><mi>F</mi></mrow><mo>/</mo><mrow><msub><mrow><mi>F</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow></mrow></math></span>.</p></section></section><section id="ack" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Mikhail Berezin, Hanley Chiang, and the members of Yao Chen’s lab, Tim Holy’s lab, and Daniel Kerschensteiner’s lab for helpful feedback on the study. We thank Kerry Grens, Chao Zhou, and Erwin Arias for critical comments on the manuscript. We thank Elizabeth Tilden for code review, and Aditi Maduska for sharing FLIM-AKAR imaging data. We thank the following sources for funding support for this work: U.S. National Institute of Neurological Disorders and Stroke (R01 NS119821, to YC), Brain &amp; Behavior Research Foundation (NARSAD Young Investigator Grant 28323, to YC), Whitehall Foundation (2019-08-64, to YC), and McDonnell International Scholars Academy of Washington University in St. Louis (to PM).</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</p></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Yao Chen, Email: Yaochen@wustl.edu.</p>
<p>Jie Xiao, Johns Hopkins University, United States.</p>
<p>Panayiota Poirazi, FORTH Institute of Molecular Biology and Biotechnology, Greece.</p></section><section id="sec27"><h2 class="pmc_sec_title">Funding Information</h2>
<p>This paper was supported by the following grants:</p>
<ul class="list" style="list-style-type:disc">
<li><p>
National Institute of Neurological Disorders and Stroke

R01 NS119821 to Yao Chen.</p></li>
<li><p>
Brain and Behavior Research Foundation

NARSAD Young Investigator Grant 28323 to Yao Chen.</p></li>
<li><p>
Whitehall Foundation

2019-08-64 to Yao Chen.</p></li>
<li><p>
McDonnell International Scholars Academy, Washington University in St. Louis
 to Pingchuan Ma.</p></li>
</ul></section><section id="s5"><h2 class="pmc_sec_title">Additional information</h2>
<section id="fn-group1" class="fn-group"><h3 class="pmc_sec_title">Competing interests</h3>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="conf1"><p>No competing interests declared.</p></div></div></section><section id="fn-group2" class="fn-group"><h3 class="pmc_sec_title">Author contributions</h3>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="con1"><p>Conceptualization, Data curation, Software, Formal analysis, Validation, Investigation, Visualization, Methodology, Writing – original draft, Writing – review and editing.</p></div>
<div class="fn p" id="con2"><p>Software, Validation, Methodology, Writing – review and editing.</p></div>
<div class="fn p" id="con3"><p>Conceptualization, Writing – review and editing.</p></div>
<div class="fn p" id="con4"><p>Conceptualization, Resources, Software, Formal analysis, Supervision, Funding acquisition, Validation, Investigation, Visualization, Methodology, Writing – original draft, Project administration, Writing – review and editing.</p></div>
</div></section><section id="fn-group3" class="fn-group"><h3 class="pmc_sec_title">Ethics</h3>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn6"><p>All procedures for rodent husbandry and surgery were performed following protocols approved by the Washington University Institutional Animal Care and Use Committee (#21-0390) and in accordance with the recommendations in the Guide for the Care and Use of Laboratory Animals of the National Institutes of Health. The experiments were performed according to the Animal Research: Reporting of In Vivo Experiments (ARRIVE) guidelines (Percie du Sert et al., 2020).</p></div></div></section></section><section id="s6"><h2 class="pmc_sec_title">Additional files</h2>
<section class="sm xbox font-sm" id="mdar"><div class="caption p"><span>MDAR checklist</span></div>
<div class="media p" id="d67e3212"><div class="caption">
<a href="/articles/instance/12367299/bin/elife-101559-mdarchecklist1.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-101559-mdarchecklist1.docx</a><sup> (89KB, docx) </sup>
</div></div></section></section><section id="s7"><h2 class="pmc_sec_title">Data availability</h2>
<p>The current manuscript is a computational study. The MATLAB and Python functions for simulation and data analyses are deposited at <a href="https://github.com/YaoChenLabWashU/Publication/tree/main/Simulation_manuscript" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GitHub</a> (<a href="#bib33" class="usa-link" aria-describedby="bib33">Ma et al., 2025</a>).</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="bib1">
<cite>Akgun U, Ayan AS, Aydin G, Duru F, Olson J, Onel Y. Afterpulse timing and rate investigation of three different Hamamatsu Photomultiplier tubes. Journal of Instrumentation. 2008;3:T01001.  doi: 10.1088/1748-0221/3/01/T01001.</cite> [<a href="https://doi.org/10.1088/1748-0221/3/01/T01001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Instrumentation&amp;title=Afterpulse%20timing%20and%20rate%20investigation%20of%20three%20different%20Hamamatsu%20Photomultiplier%20tubes&amp;author=U%20Akgun&amp;author=AS%20Ayan&amp;author=G%20Aydin&amp;author=F%20Duru&amp;author=J%20Olson&amp;volume=3&amp;publication_year=2008&amp;pages=T01001&amp;doi=10.1088/1748-0221/3/01/T01001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib2">
<cite>Bajzer E, Therneau TM, Sharp JC, Prendergast FG. Maximum likelihood method for the analysis of time-resolved fluorescence decay curves. European Biophysics Journal. 1991;20:247–262. doi: 10.1007/BF00450560.</cite> [<a href="https://doi.org/10.1007/BF00450560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2323321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Biophysics%20Journal&amp;title=Maximum%20likelihood%20method%20for%20the%20analysis%20of%20time-resolved%20fluorescence%20decay%20curves&amp;author=E%20Bajzer&amp;author=TM%20Therneau&amp;author=JC%20Sharp&amp;author=FG%20Prendergast&amp;volume=20&amp;publication_year=1991&amp;pages=247-262&amp;pmid=2323321&amp;doi=10.1007/BF00450560&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib3">
<cite>Bastiaens PIH, Squire A. Fluorescence lifetime imaging microscopy: spatial resolution of biochemical processes in the cell. Trends in Cell Biology. 1999;9:48–52. doi: 10.1016/s0962-8924(98)01410-x.</cite> [<a href="https://doi.org/10.1016/s0962-8924(98)01410-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10087617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Cell%20Biology&amp;title=Fluorescence%20lifetime%20imaging%20microscopy:%20spatial%20resolution%20of%20biochemical%20processes%20in%20the%20cell&amp;author=PIH%20Bastiaens&amp;author=A%20Squire&amp;volume=9&amp;publication_year=1999&amp;pages=48-52&amp;pmid=10087617&amp;doi=10.1016/s0962-8924(98)01410-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib4">
<cite>Becker W, Su B, Holub O, Weisshart K. FLIM and FCS detection in laser-scanning microscopes: increased efficiency by GaAsP hybrid detectors. Microscopy Research and Technique. 2011;74:804–811. doi: 10.1002/jemt.20959.</cite> [<a href="https://doi.org/10.1002/jemt.20959" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23939667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microscopy%20Research%20and%20Technique&amp;title=FLIM%20and%20FCS%20detection%20in%20laser-scanning%20microscopes:%20increased%20efficiency%20by%20GaAsP%20hybrid%20detectors&amp;author=W%20Becker&amp;author=B%20Su&amp;author=O%20Holub&amp;author=K%20Weisshart&amp;volume=74&amp;publication_year=2011&amp;pages=804-811&amp;pmid=23939667&amp;doi=10.1002/jemt.20959&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib5">
<cite>Becker W. Fluorescence lifetime imaging--techniques and applications. Journal of Microscopy. 2012;247:119–136. doi: 10.1111/j.1365-2818.2012.03618.x.</cite> [<a href="https://doi.org/10.1111/j.1365-2818.2012.03618.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22621335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Microscopy&amp;title=Fluorescence%20lifetime%20imaging--techniques%20and%20applications&amp;author=W%20Becker&amp;volume=247&amp;publication_year=2012&amp;pages=119-136&amp;pmid=22621335&amp;doi=10.1111/j.1365-2818.2012.03618.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib6">
<cite>Bowman AJ, Huang C, Schnitzer MJ, Kasevich MA. Wide-field fluorescence lifetime imaging of neuron spiking and subthreshold activity in vivo. Science. 2023;380:1270–1275. doi: 10.1126/science.adf9725.</cite> [<a href="https://doi.org/10.1126/science.adf9725" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10361454/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37347862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Wide-field%20fluorescence%20lifetime%20imaging%20of%20neuron%20spiking%20and%20subthreshold%20activity%20in%20vivo&amp;author=AJ%20Bowman&amp;author=C%20Huang&amp;author=MJ%20Schnitzer&amp;author=MA%20Kasevich&amp;volume=380&amp;publication_year=2023&amp;pages=1270-1275&amp;pmid=37347862&amp;doi=10.1126/science.adf9725&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib7">
<cite>Brinks D, Klein AJ, Cohen AE. Two-photon lifetime imaging of voltage indicating proteins as a probe of absolute membrane voltage. Biophysical Journal. 2015;109:914–921. doi: 10.1016/j.bpj.2015.07.038.</cite> [<a href="https://doi.org/10.1016/j.bpj.2015.07.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4564826/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26331249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophysical%20Journal&amp;title=Two-photon%20lifetime%20imaging%20of%20voltage%20indicating%20proteins%20as%20a%20probe%20of%20absolute%20membrane%20voltage&amp;author=D%20Brinks&amp;author=AJ%20Klein&amp;author=AE%20Cohen&amp;volume=109&amp;publication_year=2015&amp;pages=914-921&amp;pmid=26331249&amp;doi=10.1016/j.bpj.2015.07.038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib8">
<cite>Campbell JM, Gosnell M, Agha A, Handley S, Knab A, Anwer AG, Bhargava A, Goldys EM. Label-free assessment of key biological autofluorophores: material characteristics and opportunities for clinical applications. Advanced Materials. 2024;36:e2403761.  doi: 10.1002/adma.202403761.</cite> [<a href="https://doi.org/10.1002/adma.202403761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38775184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Advanced%20Materials&amp;title=Label-free%20assessment%20of%20key%20biological%20autofluorophores:%20material%20characteristics%20and%20opportunities%20for%20clinical%20applications&amp;author=JM%20Campbell&amp;author=M%20Gosnell&amp;author=A%20Agha&amp;author=S%20Handley&amp;author=A%20Knab&amp;volume=36&amp;publication_year=2024&amp;pages=e2403761&amp;pmid=38775184&amp;doi=10.1002/adma.202403761&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib9">
<cite>Chen Y, Saulnier JL, Yellen G, Sabatini BL. A PKA activity sensor for quantitative analysis of endogenous GPCR signaling via 2-photon FRET-FLIM imaging. Frontiers in Pharmacology. 2014;5:e56.  doi: 10.3389/fphar.2014.00056.</cite> [<a href="https://doi.org/10.3389/fphar.2014.00056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3980114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24765076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Pharmacology&amp;title=A%20PKA%20activity%20sensor%20for%20quantitative%20analysis%20of%20endogenous%20GPCR%20signaling%20via%202-photon%20FRET-FLIM%20imaging&amp;author=Y%20Chen&amp;author=JL%20Saulnier&amp;author=G%20Yellen&amp;author=BL%20Sabatini&amp;volume=5&amp;publication_year=2014&amp;pages=e56&amp;pmid=24765076&amp;doi=10.3389/fphar.2014.00056&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib10">
<cite>Chen Y, Granger AJ, Tran T, Saulnier JL, Kirkwood A, Sabatini BL. Endogenous Gαq-coupled neuromodulator receptors activate protein kinase A. Neuron. 2017;96:1070–1083. doi: 10.1016/j.neuron.2017.10.023.</cite> [<a href="https://doi.org/10.1016/j.neuron.2017.10.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5726796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29154125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Endogenous%20G%CE%B1q-coupled%20neuromodulator%20receptors%20activate%20protein%20kinase%20A&amp;author=Y%20Chen&amp;author=AJ%20Granger&amp;author=T%20Tran&amp;author=JL%20Saulnier&amp;author=A%20Kirkwood&amp;volume=96&amp;publication_year=2017&amp;pages=1070-1083&amp;pmid=29154125&amp;doi=10.1016/j.neuron.2017.10.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib11">
<cite>Chen YI, Chang YJ, Liao SC, Nguyen TD, Yang J, Kuo YA, Hong S, Liu YL, Rylander HG, III, Santacruz SR, Yankeelov TE, Yeh HC. Generative adversarial network enables rapid and robust fluorescence lifetime image analysis in live cells. Communications Biology. 2022;5:e18.  doi: 10.1038/s42003-021-02938-w.</cite> [<a href="https://doi.org/10.1038/s42003-021-02938-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8752789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35017629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Communications%20Biology&amp;title=Generative%20adversarial%20network%20enables%20rapid%20and%20robust%20fluorescence%20lifetime%20image%20analysis%20in%20live%20cells&amp;author=YI%20Chen&amp;author=YJ%20Chang&amp;author=SC%20Liao&amp;author=TD%20Nguyen&amp;author=J%20Yang&amp;volume=5&amp;publication_year=2022&amp;pages=e18&amp;pmid=35017629&amp;doi=10.1038/s42003-021-02938-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib12">
<cite>Esposito A, Gerritsen HC, Wouters FS. Optimizing frequency-domain fluorescence lifetime sensing for high-throughput applications: photon economy and acquisition speed. Journal of the Optical Society of America. A, Optics, Image Science, and Vision. 2007;24:3261–3273. doi: 10.1364/josaa.24.003261.</cite> [<a href="https://doi.org/10.1364/josaa.24.003261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17912319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Optical%20Society%20of%20America.%20A,%20Optics,%20Image%20Science,%20and%20Vision&amp;title=Optimizing%20frequency-domain%20fluorescence%20lifetime%20sensing%20for%20high-throughput%20applications:%20photon%20economy%20and%20acquisition%20speed&amp;author=A%20Esposito&amp;author=HC%20Gerritsen&amp;author=FS%20Wouters&amp;volume=24&amp;publication_year=2007&amp;pages=3261-3273&amp;pmid=17912319&amp;doi=10.1364/josaa.24.003261&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib13">
<cite>Esposito A, Gerritsen HC, Wouters FS.  In: Standardization and Quality Assurance in Fluorescence Measurements II: Bioanalytical and Biomedical Applications. Esposito A, editor. Springer; 2008. Fluorescence lifetime imaging microscopy: quality assessment and standards; pp. 117–142.</cite> [<a href="https://doi.org/10.1007/4243_2008_030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Standardization%20and%20Quality%20Assurance%20in%20Fluorescence%20Measurements%20II:%20Bioanalytical%20and%20Biomedical%20Applications&amp;author=A%20Esposito&amp;author=HC%20Gerritsen&amp;author=FS%20Wouters&amp;publication_year=2008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib14">
<cite>Farrants H, Shuai Y, Lemon WC, Monroy Hernandez C, Zhang D, Yang S, Patel R, Qiao G, Frei MS, Plutkis SE, Grimm JB, Hanson TL, Tomaska F, Turner GC, Stringer C, Keller PJ, Beyene AG, Chen Y, Liang Y, Lavis LD, Schreiter ER. A modular chemigenetic calcium indicator for multiplexed in vivo functional imaging. Nature Methods. 2024;21:1916–1925. doi: 10.1038/s41592-024-02411-6.</cite> [<a href="https://doi.org/10.1038/s41592-024-02411-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11466818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39304767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=A%20modular%20chemigenetic%20calcium%20indicator%20for%20multiplexed%20in%20vivo%20functional%20imaging&amp;author=H%20Farrants&amp;author=Y%20Shuai&amp;author=WC%20Lemon&amp;author=C%20Monroy%20Hernandez&amp;author=D%20Zhang&amp;volume=21&amp;publication_year=2024&amp;pages=1916-1925&amp;pmid=39304767&amp;doi=10.1038/s41592-024-02411-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib15">
<cite>Frei MS, Tarnawski M, Roberti MJ, Koch B, Hiblot J, Johnsson K. Engineered halotag variants for fluorescence lifetime multiplexing. Nature Methods. 2022;19:65–70. doi: 10.1038/s41592-021-01341-x.</cite> [<a href="https://doi.org/10.1038/s41592-021-01341-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8748199/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34916672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Engineered%20halotag%20variants%20for%20fluorescence%20lifetime%20multiplexing&amp;author=MS%20Frei&amp;author=M%20Tarnawski&amp;author=MJ%20Roberti&amp;author=B%20Koch&amp;author=J%20Hiblot&amp;volume=19&amp;publication_year=2022&amp;pages=65-70&amp;pmid=34916672&amp;doi=10.1038/s41592-021-01341-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib16">
<cite>Georgakoudi I, Quinn KP. Label-free optical metabolic imaging in cells and tissues. Annual Review of Biomedical Engineering. 2023;25:413–443. doi: 10.1146/annurev-bioeng-071516-044730.</cite> [<a href="https://doi.org/10.1146/annurev-bioeng-071516-044730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10733979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37104650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Biomedical%20Engineering&amp;title=Label-free%20optical%20metabolic%20imaging%20in%20cells%20and%20tissues&amp;author=I%20Georgakoudi&amp;author=KP%20Quinn&amp;volume=25&amp;publication_year=2023&amp;pages=413-443&amp;pmid=37104650&amp;doi=10.1146/annurev-bioeng-071516-044730&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib17">
<cite>Gerritsen HC, Asselbergs MAH, Agronskaia AV, Van Sark W. Fluorescence lifetime imaging in scanning microscopes: acquisition speed, photon economy and lifetime resolution. Journal of Microscopy. 2002;206:218–224. doi: 10.1046/j.1365-2818.2002.01031.x.</cite> [<a href="https://doi.org/10.1046/j.1365-2818.2002.01031.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12067366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Microscopy&amp;title=Fluorescence%20lifetime%20imaging%20in%20scanning%20microscopes:%20acquisition%20speed,%20photon%20economy%20and%20lifetime%20resolution&amp;author=HC%20Gerritsen&amp;author=MAH%20Asselbergs&amp;author=AV%20Agronskaia&amp;author=W%20Van%20Sark&amp;volume=206&amp;publication_year=2002&amp;pages=218-224&amp;pmid=12067366&amp;doi=10.1046/j.1365-2818.2002.01031.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib18">
<cite>Hamilton G, Sanabria H.  In: In Spectroscopy and Dynamics of Single Molecules: Methods and Applications. Hamilton G, editor. Elsevier; 2019. Multiparameter fluorescence spectroscopy of single molecules; pp. 269–333.</cite> [<a href="https://doi.org/10.1016/B978-0-12-816463-1.00006-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=In%20Spectroscopy%20and%20Dynamics%20of%20Single%20Molecules:%20Methods%20and%20Applications&amp;author=G%20Hamilton&amp;author=H%20Sanabria&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib19">
<cite>Harvey CD, Yasuda R, Zhong H, Svoboda K. The spread of Ras activity triggered by activation of a single dendritic spine. Science. 2008;321:136–140. doi: 10.1126/science.1159675.</cite> [<a href="https://doi.org/10.1126/science.1159675" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2745709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18556515/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20spread%20of%20Ras%20activity%20triggered%20by%20activation%20of%20a%20single%20dendritic%20spine&amp;author=CD%20Harvey&amp;author=R%20Yasuda&amp;author=H%20Zhong&amp;author=K%20Svoboda&amp;volume=321&amp;publication_year=2008&amp;pages=136-140&amp;pmid=18556515&amp;doi=10.1126/science.1159675&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib20">
<cite>Jang H, Li Z, Li Y, Bagheri P, Ackerstaff E, Koutcher J, Shi L. Ultrafast nonlinear multimodal metabolic imaging platform for studying aging and diseases. Ultrafast Nonlinear Imaging and Spectroscopy X; 2022. pp. 27–31.</cite> [<a href="https://doi.org/10.1117/12.2636458" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Jang%20H,%20Li%20Z,%20Li%20Y,%20Bagheri%20P,%20Ackerstaff%20E,%20Koutcher%20J,%20Shi%20L.%20Ultrafast%20nonlinear%20multimodal%20metabolic%20imaging%20platform%20for%20studying%20aging%20and%20diseases.%20Ultrafast%20Nonlinear%20Imaging%20and%20Spectroscopy%20X;%202022.%20pp.%2027%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib21">
<cite>Jo JA, Qiyin F, Marcu L. Ultrafast method for the analysis of fluorescence lifetime imaging microscopy data based on the Laguerre expansion technique. IEEE Journal of Selected Topics in Quantum Electronics. 2005;11:835–845. doi: 10.1109/JSTQE.2005.857685.</cite> [<a href="https://doi.org/10.1109/JSTQE.2005.857685" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2681327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19444338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Journal%20of%20Selected%20Topics%20in%20Quantum%20Electronics&amp;title=Ultrafast%20method%20for%20the%20analysis%20of%20fluorescence%20lifetime%20imaging%20microscopy%20data%20based%20on%20the%20Laguerre%20expansion%20technique&amp;author=JA%20Jo&amp;author=F%20Qiyin&amp;author=L%20Marcu&amp;volume=11&amp;publication_year=2005&amp;pages=835-845&amp;pmid=19444338&amp;doi=10.1109/JSTQE.2005.857685&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib22">
<cite>Köllner M, Wolfrum J. How many photons are necessary for fluorescence-lifetime measurements? Chemical Physics Letters. 1992;200:199–204. doi: 10.1016/0009-2614(92)87068-Z.</cite> [<a href="https://doi.org/10.1016/0009-2614(92)87068-Z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Physics%20Letters&amp;title=How%20many%20photons%20are%20necessary%20for%20fluorescence-lifetime%20measurements?&amp;author=M%20K%C3%B6llner&amp;author=J%20Wolfrum&amp;volume=200&amp;publication_year=1992&amp;pages=199-204&amp;doi=10.1016/0009-2614(92)87068-Z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib23">
<cite>Kumar ATN. Direct Monte Carlo computation of time-resolved fluorescence in heterogeneous turbid media. Optics Letters. 2012;37:4783–4785. doi: 10.1364/ol.37.004783.</cite> [<a href="https://doi.org/10.1364/ol.37.004783" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3544305/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23164912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Optics%20Letters&amp;title=Direct%20Monte%20Carlo%20computation%20of%20time-resolved%20fluorescence%20in%20heterogeneous%20turbid%20media&amp;author=ATN%20Kumar&amp;volume=37&amp;publication_year=2012&amp;pages=4783-4785&amp;pmid=23164912&amp;doi=10.1364/ol.37.004783&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib24">
<cite>Lakowicz JR, Szmacinski H, Johnson ML. Calcium imaging using fluorescence lifetimes and long-wavelength probes. Journal of Fluorescence. 1992;2:47–62. doi: 10.1007/BF00866388.</cite> [<a href="https://doi.org/10.1007/BF00866388" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6885754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24243158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Fluorescence&amp;title=Calcium%20imaging%20using%20fluorescence%20lifetimes%20and%20long-wavelength%20probes&amp;author=JR%20Lakowicz&amp;author=H%20Szmacinski&amp;author=ML%20Johnson&amp;volume=2&amp;publication_year=1992&amp;pages=47-62&amp;pmid=24243158&amp;doi=10.1007/BF00866388&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib25">
<cite>Lakowicz JR.  Principles of Fluorescence Spectroscopy. Springer; 2006. </cite> [<a href="https://doi.org/10.1007/978-1-4757-3061-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Principles%20of%20Fluorescence%20Spectroscopy&amp;author=JR%20Lakowicz&amp;publication_year=2006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib26">
<cite>Laviv T, Scholl B, Parra-Bueno P, Foote B, Zhang C, Yan L, Hayano Y, Chu J, Yasuda R. In vivo imaging of the coupling between Neuronal and CREB activity in the mouse brain. Neuron. 2020;105:799–812. doi: 10.1016/j.neuron.2019.11.028.</cite> [<a href="https://doi.org/10.1016/j.neuron.2019.11.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7144870/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31883788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=In%20vivo%20imaging%20of%20the%20coupling%20between%20Neuronal%20and%20CREB%20activity%20in%20the%20mouse%20brain&amp;author=T%20Laviv&amp;author=B%20Scholl&amp;author=P%20Parra-Bueno&amp;author=B%20Foote&amp;author=C%20Zhang&amp;volume=105&amp;publication_year=2020&amp;pages=799-812&amp;pmid=31883788&amp;doi=10.1016/j.neuron.2019.11.028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib27">
<cite>Lazzari-Dean JR, Gest AMM, Miller EW. Optical estimation of absolute membrane potential using fluorescence lifetime imaging. eLife. 2019;8:e44522.  doi: 10.7554/eLife.44522.</cite> [<a href="https://doi.org/10.7554/eLife.44522" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6814365/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31545164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Optical%20estimation%20of%20absolute%20membrane%20potential%20using%20fluorescence%20lifetime%20imaging&amp;author=JR%20Lazzari-Dean&amp;author=AMM%20Gest&amp;author=EW%20Miller&amp;volume=8&amp;publication_year=2019&amp;pages=e44522&amp;pmid=31545164&amp;doi=10.7554/eLife.44522&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib28">
<cite>Lee SJ, Chen Y, Lodder B, Sabatini BL. Monitoring behaviorally induced biochemical changes using fluorescence lifetime photometry. Frontiers in Neuroscience. 2019;13:e766.  doi: 10.3389/fnins.2019.00766.</cite> [<a href="https://doi.org/10.3389/fnins.2019.00766" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6685078/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31417343/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Neuroscience&amp;title=Monitoring%20behaviorally%20induced%20biochemical%20changes%20using%20fluorescence%20lifetime%20photometry&amp;author=SJ%20Lee&amp;author=Y%20Chen&amp;author=B%20Lodder&amp;author=BL%20Sabatini&amp;volume=13&amp;publication_year=2019&amp;pages=e766&amp;pmid=31417343&amp;doi=10.3389/fnins.2019.00766&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib29">
<cite>Lee SJ, Lodder B, Chen Y, Patriarchi T, Tian L, Sabatini BL. Cell-type-specific asynchronous modulation of PKA by dopamine in learning. Nature. 2021;590:451–456. doi: 10.1038/s41586-020-03050-5.</cite> [<a href="https://doi.org/10.1038/s41586-020-03050-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7889726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33361810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Cell-type-specific%20asynchronous%20modulation%20of%20PKA%20by%20dopamine%20in%20learning&amp;author=SJ%20Lee&amp;author=B%20Lodder&amp;author=Y%20Chen&amp;author=T%20Patriarchi&amp;author=L%20Tian&amp;volume=590&amp;publication_year=2021&amp;pages=451-456&amp;pmid=33361810&amp;doi=10.1038/s41586-020-03050-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib30">
<cite>Lodder B, Kamath T, Savenco E, Röring B, Siegel M, Chouinard J, Lee SJ, Zagoren C, Rosen P, Adan R, Tian L, Sabatini BL. Absolute measurement of fast and slow neuronal signals with fluorescence lifetime photometry at high temporal resolution. bioRxiv. 2025 doi: 10.1101/2025.01.10.632162.</cite> [<a href="https://doi.org/10.1101/2025.01.10.632162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="bib31">
<cite>Ma L, Jongbloets BC, Xiong WH, Melander JB, Qin M, Lameyer TJ, Harrison MF, Zemelman BV, Mao T, Zhong H. A ihghly sensitive A-kinase activity reporter for imaging neuromodulatory events in awake mice. Neuron. 2018;99:665–679. doi: 10.1016/j.neuron.2018.07.020.</cite> [<a href="https://doi.org/10.1016/j.neuron.2018.07.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6152931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30100256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=A%20ihghly%20sensitive%20A-kinase%20activity%20reporter%20for%20imaging%20neuromodulatory%20events%20in%20awake%20mice&amp;author=L%20Ma&amp;author=BC%20Jongbloets&amp;author=WH%20Xiong&amp;author=JB%20Melander&amp;author=M%20Qin&amp;volume=99&amp;publication_year=2018&amp;pages=665-679&amp;pmid=30100256&amp;doi=10.1016/j.neuron.2018.07.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib32">
<cite>Ma P, Chen P, Tilden EI, Aggarwal S, Oldenborg A, Chen Y. Fast and slow: Recording neuromodulator dynamics across both transient and chronic time scales. Science Advances. 2024;10:eadi0643.  doi: 10.1126/sciadv.adi0643.</cite> [<a href="https://doi.org/10.1126/sciadv.adi0643" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10881037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38381826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20Advances&amp;title=Fast%20and%20slow:%20Recording%20neuromodulator%20dynamics%20across%20both%20transient%20and%20chronic%20time%20scales&amp;author=P%20Ma&amp;author=P%20Chen&amp;author=EI%20Tilden&amp;author=S%20Aggarwal&amp;author=A%20Oldenborg&amp;volume=10&amp;publication_year=2024&amp;pages=eadi0643&amp;pmid=38381826&amp;doi=10.1126/sciadv.adi0643&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib33"><cite>Ma P, Chen P, Chen Y. Simulation_manuscript. 25753d7GitHub. 2025 <a href="https://github.com/YaoChenLabWashU/Publication/tree/main/Simulation_manuscript" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/YaoChenLabWashU/Publication/tree/main/Simulation_manuscript</a></cite></li>
<li id="bib34">
<cite>Malak M, James J, Grantham J, Ericson MB. Contribution of autofluorescence from intracellular proteins in multiphoton fluorescence lifetime imaging. Scientific Reports. 2022;12:e16584.  doi: 10.1038/s41598-022-20857-6.</cite> [<a href="https://doi.org/10.1038/s41598-022-20857-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9534927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36198710/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Contribution%20of%20autofluorescence%20from%20intracellular%20proteins%20in%20multiphoton%20fluorescence%20lifetime%20imaging&amp;author=M%20Malak&amp;author=J%20James&amp;author=J%20Grantham&amp;author=MB%20Ericson&amp;volume=12&amp;publication_year=2022&amp;pages=e16584&amp;pmid=36198710&amp;doi=10.1038/s41598-022-20857-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib35">
<cite>Mannam V, Zhang Y, Yuan X, Ravasio C, Howard SS. Machine learning for faster and smarter fluorescence lifetime imaging microscopy. Journal of Physics. 2020;2:e042005.  doi: 10.1088/2515-7647/abac1a.</cite> [<a href="https://doi.org/10.1088/2515-7647/abac1a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Physics&amp;title=Machine%20learning%20for%20faster%20and%20smarter%20fluorescence%20lifetime%20imaging%20microscopy&amp;author=V%20Mannam&amp;author=Y%20Zhang&amp;author=X%20Yuan&amp;author=C%20Ravasio&amp;author=SS%20Howard&amp;volume=2&amp;publication_year=2020&amp;pages=e042005&amp;doi=10.1088/2515-7647/abac1a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib36">
<cite>Massengill CI, Bayless-Edwards L, Ceballos CC, Cebul ER, Cahill J, Bharadwaj A, Wilson E, Qin M, Whorton MR, Baconguis I, Ye B, Mao T, Zhong H. Sensitive genetically encoded sensors for population and subcellular imaging of cAMP in vivo. Nature Methods. 2022;19:1461–1471. doi: 10.1038/s41592-022-01646-5.</cite> [<a href="https://doi.org/10.1038/s41592-022-01646-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10171401/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36303019/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Sensitive%20genetically%20encoded%20sensors%20for%20population%20and%20subcellular%20imaging%20of%20cAMP%20in%20vivo&amp;author=CI%20Massengill&amp;author=L%20Bayless-Edwards&amp;author=CC%20Ceballos&amp;author=ER%20Cebul&amp;author=J%20Cahill&amp;volume=19&amp;publication_year=2022&amp;pages=1461-1471&amp;pmid=36303019&amp;doi=10.1038/s41592-022-01646-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib37">
<cite>Mongeon R, Venkatachalam V, Yellen G. Cytosolic NADH-NAD + redox visualized in brain slices by two-photon fluorescence lifetime biosensor imaging. Antioxidants &amp; Redox Signaling. 2016;25:553–563. doi: 10.1089/ars.2015.6593.</cite> [<a href="https://doi.org/10.1089/ars.2015.6593" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5041510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26857245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antioxidants%20&amp;%20Redox%20Signaling&amp;title=Cytosolic%20NADH-NAD%20+%20redox%20visualized%20in%20brain%20slices%20by%20two-photon%20fluorescence%20lifetime%20biosensor%20imaging&amp;author=R%20Mongeon&amp;author=V%20Venkatachalam&amp;author=G%20Yellen&amp;volume=25&amp;publication_year=2016&amp;pages=553-563&amp;pmid=26857245&amp;doi=10.1089/ars.2015.6593&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib38">
<cite>Morrow CS, Tweed K, Farhadova S, Walsh AJ, Lear BP, Roopra A, Risgaard RD, Klosa PC, Arndt ZP, Peterson ER, Chi MM, Harris AG, Skala MC, Moore DL. Autofluorescence is a biomarker of neural stem cell activation state. Cell Stem Cell. 2024;31:570–581. doi: 10.1016/j.stem.2024.02.011.</cite> [<a href="https://doi.org/10.1016/j.stem.2024.02.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10997463/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38521057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Stem%20Cell&amp;title=Autofluorescence%20is%20a%20biomarker%20of%20neural%20stem%20cell%20activation%20state&amp;author=CS%20Morrow&amp;author=K%20Tweed&amp;author=S%20Farhadova&amp;author=AJ%20Walsh&amp;author=BP%20Lear&amp;volume=31&amp;publication_year=2024&amp;pages=570-581&amp;pmid=38521057&amp;doi=10.1016/j.stem.2024.02.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib39">
<cite>Nasser M, Meller A. Lifetime-based analysis of binary fluorophores mixtures in the low photon count limit. iScience. 2022;25:e103554.  doi: 10.1016/j.isci.2021.103554.</cite> [<a href="https://doi.org/10.1016/j.isci.2021.103554" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8689154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34977508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=Lifetime-based%20analysis%20of%20binary%20fluorophores%20mixtures%20in%20the%20low%20photon%20count%20limit&amp;author=M%20Nasser&amp;author=A%20Meller&amp;volume=25&amp;publication_year=2022&amp;pages=e103554&amp;pmid=34977508&amp;doi=10.1016/j.isci.2021.103554&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib40">
<cite>Netaev A, Schierbaum N, Seidl K. Advantages and limitations of fluorescence lifetime measurements using Single-Photon Avalanche Diode (SPAD) array detector: a comprehensive theoretical and experimental study. Sensors. 2022;22:e3822.  doi: 10.3390/s22103822.</cite> [<a href="https://doi.org/10.3390/s22103822" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9144122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35632231/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sensors&amp;title=Advantages%20and%20limitations%20of%20fluorescence%20lifetime%20measurements%20using%20Single-Photon%20Avalanche%20Diode%20(SPAD)%20array%20detector:%20a%20comprehensive%20theoretical%20and%20experimental%20study&amp;author=A%20Netaev&amp;author=N%20Schierbaum&amp;author=K%20Seidl&amp;volume=22&amp;publication_year=2022&amp;pages=e3822&amp;pmid=35632231&amp;doi=10.3390/s22103822&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib41">
<cite>Percie du Sert N, Hurst V, Ahluwalia A, Alam S, Avey MT, Baker M, Browne WJ, Clark A, Cuthill IC, Dirnagl U, Emerson M, Garner P, Holgate ST, Howells DW, Karp NA, Lazic SE, Lidster K, MacCallum CJ, Macleod M, Pearl EJ, Petersen OH, Rawle F, Reynolds P, Rooney K, Sena ES, Silberberg SD, Steckler T, Würbel H. The ARRIVE guidelines 2.0: Updated guidelines for reporting animal research*. Journal of Cerebral Blood Flow &amp; Metabolism. 2020;40:1769–1777. doi: 10.1177/0271678X20943823.</cite> [<a href="https://doi.org/10.1177/0271678X20943823" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7430098/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32663096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cerebral%20Blood%20Flow%20&amp;%20Metabolism&amp;title=The%20ARRIVE%20guidelines%202.0:%20Updated%20guidelines%20for%20reporting%20animal%20research*&amp;author=N%20Percie%20du%20Sert&amp;author=V%20Hurst&amp;author=A%20Ahluwalia&amp;author=S%20Alam&amp;author=MT%20Avey&amp;volume=40&amp;publication_year=2020&amp;pages=1769-1777&amp;pmid=32663096&amp;doi=10.1177/0271678X20943823&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib42">
<cite>Periasamy N. Analysis of fluorescence decay by the nonlinear least squares method. Biophysical Journal. 1988;54:961–967. doi: 10.1016/S0006-3495(88)83032-7.</cite> [<a href="https://doi.org/10.1016/S0006-3495(88)83032-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1330404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19431735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophysical%20Journal&amp;title=Analysis%20of%20fluorescence%20decay%20by%20the%20nonlinear%20least%20squares%20method&amp;author=N%20Periasamy&amp;volume=54&amp;publication_year=1988&amp;pages=961-967&amp;pmid=19431735&amp;doi=10.1016/S0006-3495(88)83032-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib43">
<cite>Philip J, Carlsson K. Theoretical investigation of the signal-to-noise ratio in fluorescence lifetime imaging. Journal of the Optical Society of America. A, Optics, Image Science, and Vision. 2003;20:368–379. doi: 10.1364/josaa.20.000368.</cite> [<a href="https://doi.org/10.1364/josaa.20.000368" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12570304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Optical%20Society%20of%20America.%20A,%20Optics,%20Image%20Science,%20and%20Vision&amp;title=Theoretical%20investigation%20of%20the%20signal-to-noise%20ratio%20in%20fluorescence%20lifetime%20imaging&amp;author=J%20Philip&amp;author=K%20Carlsson&amp;volume=20&amp;publication_year=2003&amp;pages=368-379&amp;pmid=12570304&amp;doi=10.1364/josaa.20.000368&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib44">
<cite>Pologruto TA, Sabatini BL, Svoboda K. ScanImage: flexible software for operating laser scanning microscopes. Biomedical Engineering Online. 2003;2:1–9. doi: 10.1186/1475-925X-2-13.</cite> [<a href="https://doi.org/10.1186/1475-925X-2-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC161784/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12801419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomedical%20Engineering%20Online&amp;title=ScanImage:%20flexible%20software%20for%20operating%20laser%20scanning%20microscopes&amp;author=TA%20Pologruto&amp;author=BL%20Sabatini&amp;author=K%20Svoboda&amp;volume=2&amp;publication_year=2003&amp;pages=1-9&amp;pmid=12801419&amp;doi=10.1186/1475-925X-2-13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib45">
<cite>Rahim MK, Zhao J, Patel HV, Lagouros HA, Kota R, Fernandez I, Gratton E, Haun JB. Phasor analysis of fluorescence lifetime enables quantitative multiplexed molecular imaging of three probes. Analytical Chemistry. 2022;94:14185–14194. doi: 10.1021/acs.analchem.2c02149.</cite> [<a href="https://doi.org/10.1021/acs.analchem.2c02149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10681155/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36190014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Phasor%20analysis%20of%20fluorescence%20lifetime%20enables%20quantitative%20multiplexed%20molecular%20imaging%20of%20three%20probes&amp;author=MK%20Rahim&amp;author=J%20Zhao&amp;author=HV%20Patel&amp;author=HA%20Lagouros&amp;author=R%20Kota&amp;volume=94&amp;publication_year=2022&amp;pages=14185-14194&amp;pmid=36190014&amp;doi=10.1021/acs.analchem.2c02149&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib46">
<cite>Raspe M, Kedziora KM, van den Broek B, Zhao Q, de Jong S, Herz J, Mastop M, Goedhart J, Gadella TWJ, Young IT, Jalink K. siFLIM: single-image frequency-domain FLIM provides fast and photon-efficient lifetime data. Nature Methods. 2016;13:501–504. doi: 10.1038/nmeth.3836.</cite> [<a href="https://doi.org/10.1038/nmeth.3836" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27088314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=siFLIM:%20single-image%20frequency-domain%20FLIM%20provides%20fast%20and%20photon-efficient%20lifetime%20data&amp;author=M%20Raspe&amp;author=KM%20Kedziora&amp;author=B%20van%20den%20Broek&amp;author=Q%20Zhao&amp;author=S%20de%20Jong&amp;volume=13&amp;publication_year=2016&amp;pages=501-504&amp;pmid=27088314&amp;doi=10.1038/nmeth.3836&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib47">
<cite>Rice WL, Kumar ATN. Preclinical whole body time domain fluorescence lifetime multiplexing of fluorescent proteins. Journal of Biomedical Optics. 2014;19:e046005.  doi: 10.1117/1.JBO.19.4.046005.</cite> [<a href="https://doi.org/10.1117/1.JBO.19.4.046005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3979168/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24715027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biomedical%20Optics&amp;title=Preclinical%20whole%20body%20time%20domain%20fluorescence%20lifetime%20multiplexing%20of%20fluorescent%20proteins&amp;author=WL%20Rice&amp;author=ATN%20Kumar&amp;volume=19&amp;publication_year=2014&amp;pages=e046005&amp;pmid=24715027&amp;doi=10.1117/1.JBO.19.4.046005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib48">
<cite>Roethlein C, Miettinen MS, Ignatova Z. A flexible approach to assess fluorescence decay functions in complex energy transfer systems. BMC Biophysics. 2015;8:e5.  doi: 10.1186/s13628-015-0020-z.</cite> [<a href="https://doi.org/10.1186/s13628-015-0020-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4403788/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25897394/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Biophysics&amp;title=A%20flexible%20approach%20to%20assess%20fluorescence%20decay%20functions%20in%20complex%20energy%20transfer%20systems&amp;author=C%20Roethlein&amp;author=MS%20Miettinen&amp;author=Z%20Ignatova&amp;volume=8&amp;publication_year=2015&amp;pages=e5&amp;pmid=25897394&amp;doi=10.1186/s13628-015-0020-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib49">
<cite>Rowley MI, Barber PR, Coolen ACC, Vojnovic B. Bayesian analysis of fluorescence lifetime imaging data. SPIE BiOS; 2011. pp. 339–350.</cite> [<a href="https://doi.org/10.1117/12.873890" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Rowley%20MI,%20Barber%20PR,%20Coolen%20ACC,%20Vojnovic%20B.%20Bayesian%20analysis%20of%20fluorescence%20lifetime%20imaging%20data.%20SPIE%20BiOS;%202011.%20pp.%20339%E2%80%93350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib50">
<cite>Shcheslavskiy VI, Shirmanova MV, Dudenkova VV, Lukyanov KA, Gavrina AI, Shumilova AV, Zagaynova E, Becker W. Fluorescence time-resolved macroimaging. Optics Letters. 2018;43:3152–3155. doi: 10.1364/OL.43.003152.</cite> [<a href="https://doi.org/10.1364/OL.43.003152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29957804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Optics%20Letters&amp;title=Fluorescence%20time-resolved%20macroimaging&amp;author=VI%20Shcheslavskiy&amp;author=MV%20Shirmanova&amp;author=VV%20Dudenkova&amp;author=KA%20Lukyanov&amp;author=AI%20Gavrina&amp;volume=43&amp;publication_year=2018&amp;pages=3152-3155&amp;pmid=29957804&amp;doi=10.1364/OL.43.003152&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib51">
<cite>Shen B, Lu Y, Guo F, Lin F, Hu R, Rao F, Qu J, Liu L. Overcoming photon and spatiotemporal sparsity in fluorescence lifetime imaging with SparseFLIM. Communications Biology. 2024;7:e1359.  doi: 10.1038/s42003-024-07080-x.</cite> [<a href="https://doi.org/10.1038/s42003-024-07080-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11494201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39433929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Communications%20Biology&amp;title=Overcoming%20photon%20and%20spatiotemporal%20sparsity%20in%20fluorescence%20lifetime%20imaging%20with%20SparseFLIM&amp;author=B%20Shen&amp;author=Y%20Lu&amp;author=F%20Guo&amp;author=F%20Lin&amp;author=R%20Hu&amp;volume=7&amp;publication_year=2024&amp;pages=e1359&amp;pmid=39433929&amp;doi=10.1038/s42003-024-07080-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib52">
<cite>Smith JT, Yao R, Sinsuebphon N, Rudkouskaya A, Un N, Mazurkiewicz J, Barroso M, Yan P, Intes X. Fast fit-free analysis of fluorescence lifetime imaging via deep learning. PNAS. 2019;116:24019–24030. doi: 10.1073/pnas.1912707116.</cite> [<a href="https://doi.org/10.1073/pnas.1912707116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6883809/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31719196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Fast%20fit-free%20analysis%20of%20fluorescence%20lifetime%20imaging%20via%20deep%20learning&amp;author=JT%20Smith&amp;author=R%20Yao&amp;author=N%20Sinsuebphon&amp;author=A%20Rudkouskaya&amp;author=N%20Un&amp;volume=116&amp;publication_year=2019&amp;pages=24019-24030&amp;pmid=31719196&amp;doi=10.1073/pnas.1912707116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib53">
<cite>Steinbach PJ. Filtering artifacts from lifetime distributions when maximizing entropy using a bootstrapped model. Analytical Biochemistry. 2012;427:102–105. doi: 10.1016/j.ab.2012.04.008.</cite> [<a href="https://doi.org/10.1016/j.ab.2012.04.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3390977/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22504734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Biochemistry&amp;title=Filtering%20artifacts%20from%20lifetime%20distributions%20when%20maximizing%20entropy%20using%20a%20bootstrapped%20model&amp;author=PJ%20Steinbach&amp;volume=427&amp;publication_year=2012&amp;pages=102-105&amp;pmid=22504734&amp;doi=10.1016/j.ab.2012.04.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib54">
<cite>Stringari C, Cinquin A, Cinquin O, Digman MA, Donovan PJ, Gratton E. Phasor approach to fluorescence lifetime microscopy distinguishes different metabolic states of germ cells in a live tissue. PNAS. 2011;108:13582–13587. doi: 10.1073/pnas.1108161108.</cite> [<a href="https://doi.org/10.1073/pnas.1108161108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3158156/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21808026/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Phasor%20approach%20to%20fluorescence%20lifetime%20microscopy%20distinguishes%20different%20metabolic%20states%20of%20germ%20cells%20in%20a%20live%20tissue&amp;author=C%20Stringari&amp;author=A%20Cinquin&amp;author=O%20Cinquin&amp;author=MA%20Digman&amp;author=PJ%20Donovan&amp;volume=108&amp;publication_year=2011&amp;pages=13582-13587&amp;pmid=21808026&amp;doi=10.1073/pnas.1108161108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib55">
<cite>Tang S, Yasuda R. Imaging ERK and PKA activation in single dendritic spines during structural plasticity. Neuron. 2017;93:1315–1324. doi: 10.1016/j.neuron.2017.02.032.</cite> [<a href="https://doi.org/10.1016/j.neuron.2017.02.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6042854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28285819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Imaging%20ERK%20and%20PKA%20activation%20in%20single%20dendritic%20spines%20during%20structural%20plasticity&amp;author=S%20Tang&amp;author=R%20Yasuda&amp;volume=93&amp;publication_year=2017&amp;pages=1315-1324&amp;pmid=28285819&amp;doi=10.1016/j.neuron.2017.02.032&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib56">
<cite>Tehrani KF, Park J, Renteria C, Boppart SA. Label-free identification of Alzheimer’s disease plaques using multiple co-registered nonlinear optical biomarkers. Clinical and Translational Neurophotonics. 2023;12364:e1236405.  doi: 10.1117/12.2648013.</cite> [<a href="https://doi.org/10.1117/12.2648013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clinical%20and%20Translational%20Neurophotonics&amp;title=Label-free%20identification%20of%20Alzheimer%E2%80%99s%20disease%20plaques%20using%20multiple%20co-registered%20nonlinear%20optical%20biomarkers&amp;author=KF%20Tehrani&amp;author=J%20Park&amp;author=C%20Renteria&amp;author=SA%20Boppart&amp;volume=12364&amp;publication_year=2023&amp;pages=e1236405&amp;doi=10.1117/12.2648013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib57">
<cite>Tilden EI, Maduskar A, Oldenborg A, Sabatini BL, Chen Y. A Cre-dependent reporter mouse for quantitative real-time imaging of protein kinase A activity dynamics. Scientific Reports. 2024;14:3054.  doi: 10.1038/s41598-024-53313-8.</cite> [<a href="https://doi.org/10.1038/s41598-024-53313-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10847463/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38321128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=A%20Cre-dependent%20reporter%20mouse%20for%20quantitative%20real-time%20imaging%20of%20protein%20kinase%20A%20activity%20dynamics&amp;author=EI%20Tilden&amp;author=A%20Maduskar&amp;author=A%20Oldenborg&amp;author=BL%20Sabatini&amp;author=Y%20Chen&amp;volume=14&amp;publication_year=2024&amp;pages=3054&amp;pmid=38321128&amp;doi=10.1038/s41598-024-53313-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib58">
<cite>Trinh AL, Esposito A. Biochemical resolving power of fluorescence lifetime imaging: untangling the roles of the instrument response function and photon-statistics. Biomedical Optics Express. 2021;12:3775–3788. doi: 10.1364/BOE.428070.</cite> [<a href="https://doi.org/10.1364/BOE.428070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8367261/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34457379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomedical%20Optics%20Express&amp;title=Biochemical%20resolving%20power%20of%20fluorescence%20lifetime%20imaging:%20untangling%20the%20roles%20of%20the%20instrument%20response%20function%20and%20photon-statistics&amp;author=AL%20Trinh&amp;author=A%20Esposito&amp;volume=12&amp;publication_year=2021&amp;pages=3775-3788&amp;pmid=34457379&amp;doi=10.1364/BOE.428070&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib59">
<cite>Turton DA, Reid GD, Beddard GS. Accurate analysis of fluorescence decays from single molecules in photon counting experiments. Analytical Chemistry. 2003;75:4182–4187. doi: 10.1021/ac034325k.</cite> [<a href="https://doi.org/10.1021/ac034325k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14632133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Accurate%20analysis%20of%20fluorescence%20decays%20from%20single%20molecules%20in%20photon%20counting%20experiments&amp;author=DA%20Turton&amp;author=GD%20Reid&amp;author=GS%20Beddard&amp;volume=75&amp;publication_year=2003&amp;pages=4182-4187&amp;pmid=14632133&amp;doi=10.1021/ac034325k&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib60">
<cite>van der Linden FH, Mahlandt EK, Arts JJG, Beumer J, Puschhof J, de Man SMA, Chertkova AO, Ponsioen B, Clevers H, van Buul JD, Postma M, Gadella TJ, Goedhart J. A turquoise fluorescence lifetime-based biosensor for quantitative imaging of intracellular calcium. Nature Communications. 2021;12:e7159.  doi: 10.1038/s41467-021-27249-w.</cite> [<a href="https://doi.org/10.1038/s41467-021-27249-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8660884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34887382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=A%20turquoise%20fluorescence%20lifetime-based%20biosensor%20for%20quantitative%20imaging%20of%20intracellular%20calcium&amp;author=FH%20van%20der%20Linden&amp;author=EK%20Mahlandt&amp;author=JJG%20Arts&amp;author=J%20Beumer&amp;author=J%20Puschhof&amp;volume=12&amp;publication_year=2021&amp;pages=e7159&amp;pmid=34887382&amp;doi=10.1038/s41467-021-27249-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib61">
<cite>Walsh AJ, Sharick JT, Skala MC, Beier HT. Temporal binning of time-correlated single photon counting data improves exponential decay fits and imaging speed. Biomedical Optics Express. 2016;7:1385–1399. doi: 10.1364/BOE.7.001385.</cite> [<a href="https://doi.org/10.1364/BOE.7.001385" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4929649/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27446663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomedical%20Optics%20Express&amp;title=Temporal%20binning%20of%20time-correlated%20single%20photon%20counting%20data%20improves%20exponential%20decay%20fits%20and%20imaging%20speed&amp;author=AJ%20Walsh&amp;author=JT%20Sharick&amp;author=MC%20Skala&amp;author=HT%20Beier&amp;volume=7&amp;publication_year=2016&amp;pages=1385-1399&amp;pmid=27446663&amp;doi=10.1364/BOE.7.001385&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib62">
<cite>Wang Q, Li Y, Xiao D, Zang Z, Jiao Z, Chen Y, Li DDU. Simple and robust deep learning approach for fast fluorescence lifetime imaging. Sensors. 2022;22:e7293.  doi: 10.3390/s22197293.</cite> [<a href="https://doi.org/10.3390/s22197293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9572653/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36236390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sensors&amp;title=Simple%20and%20robust%20deep%20learning%20approach%20for%20fast%20fluorescence%20lifetime%20imaging&amp;author=Q%20Wang&amp;author=Y%20Li&amp;author=D%20Xiao&amp;author=Z%20Zang&amp;author=Z%20Jiao&amp;volume=22&amp;publication_year=2022&amp;pages=e7293&amp;pmid=36236390&amp;doi=10.3390/s22197293&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib63">
<cite>Widengren J, Kudryavtsev V, Antonik M, Berger S, Gerken M, Seidel CAM. Single-molecule detection and identification of multiple species by multiparameter fluorescence detection. Analytical Chemistry. 2006;78:2039–2050. doi: 10.1021/ac0522759.</cite> [<a href="https://doi.org/10.1021/ac0522759" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16536444/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Single-molecule%20detection%20and%20identification%20of%20multiple%20species%20by%20multiparameter%20fluorescence%20detection&amp;author=J%20Widengren&amp;author=V%20Kudryavtsev&amp;author=M%20Antonik&amp;author=S%20Berger&amp;author=M%20Gerken&amp;volume=78&amp;publication_year=2006&amp;pages=2039-2050&amp;pmid=16536444&amp;doi=10.1021/ac0522759&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib64">
<cite>Wu G, Nowotny T, Zhang Y, Yu HQ, Li DDU. Artificial neural network approaches for fluorescence lifetime imaging techniques. Optics Letters. 2016;41:2561–2564. doi: 10.1364/OL.41.002561.</cite> [<a href="https://doi.org/10.1364/OL.41.002561" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27244414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Optics%20Letters&amp;title=Artificial%20neural%20network%20approaches%20for%20fluorescence%20lifetime%20imaging%20techniques&amp;author=G%20Wu&amp;author=T%20Nowotny&amp;author=Y%20Zhang&amp;author=HQ%20Yu&amp;author=DDU%20Li&amp;volume=41&amp;publication_year=2016&amp;pages=2561-2564&amp;pmid=27244414&amp;doi=10.1364/OL.41.002561&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib65">
<cite>Xiao D, Sapermsap N, Safar M, Cunningham MR, Chen Y, Li DD. On synthetic instrument response functions of time-correlated single-photon counting based fluorescence lifetime imaging analysis. Frontiers in Physics. 2021;9:e635645.  doi: 10.3389/fphy.2021.635645.</cite> [<a href="https://doi.org/10.3389/fphy.2021.635645" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Physics&amp;title=On%20synthetic%20instrument%20response%20functions%20of%20time-correlated%20single-photon%20counting%20based%20fluorescence%20lifetime%20imaging%20analysis&amp;author=D%20Xiao&amp;author=N%20Sapermsap&amp;author=M%20Safar&amp;author=MR%20Cunningham&amp;author=Y%20Chen&amp;volume=9&amp;publication_year=2021&amp;pages=e635645&amp;doi=10.3389/fphy.2021.635645&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib66">
<cite>Xiao D, Sapermsap N, Chen Y, Li DDU. Deep learning enhanced fast fluorescence lifetime imaging with a few photons. Optica. 2023;10:e944.  doi: 10.1364/OPTICA.491798.</cite> [<a href="https://doi.org/10.1364/OPTICA.491798" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Optica&amp;title=Deep%20learning%20enhanced%20fast%20fluorescence%20lifetime%20imaging%20with%20a%20few%20photons&amp;author=D%20Xiao&amp;author=N%20Sapermsap&amp;author=Y%20Chen&amp;author=DDU%20Li&amp;volume=10&amp;publication_year=2023&amp;pages=e944&amp;doi=10.1364/OPTICA.491798&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib67">
<cite>Yasuda R. Imaging spatiotemporal dynamics of neuronal signaling using fluorescence resonance energy transfer and fluorescence lifetime imaging microscopy. Current Opinion in Neurobiology. 2006a;16:551–561. doi: 10.1016/j.conb.2006.08.012.</cite> [<a href="https://doi.org/10.1016/j.conb.2006.08.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16971112/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Neurobiology&amp;title=Imaging%20spatiotemporal%20dynamics%20of%20neuronal%20signaling%20using%20fluorescence%20resonance%20energy%20transfer%20and%20fluorescence%20lifetime%20imaging%20microscopy&amp;author=R%20Yasuda&amp;volume=16&amp;publication_year=2006a&amp;pages=551-561&amp;pmid=16971112&amp;doi=10.1016/j.conb.2006.08.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib68">
<cite>Yasuda R, Harvey CD, Zhong H, Sobczyk A, van Aelst L, Svoboda K. Supersensitive Ras activation in dendrites and spines revealed by two-photon fluorescence lifetime imaging. Nature Neuroscience. 2006b;9:283–291. doi: 10.1038/nn1635.</cite> [<a href="https://doi.org/10.1038/nn1635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16429133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Neuroscience&amp;title=Supersensitive%20Ras%20activation%20in%20dendrites%20and%20spines%20revealed%20by%20two-photon%20fluorescence%20lifetime%20imaging&amp;author=R%20Yasuda&amp;author=CD%20Harvey&amp;author=H%20Zhong&amp;author=A%20Sobczyk&amp;author=L%20van%20Aelst&amp;volume=9&amp;publication_year=2006b&amp;pages=283-291&amp;pmid=16429133&amp;doi=10.1038/nn1635&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib69">
<cite>Zhang Y, Guldner IH, Nichols EL, Benirschke D, Smith CJ, Zhang S, Howard SS. Instant FLIM enables 4D in vivo lifetime imaging of intact and injured zebrafish and mouse brains. Optica. 2021a;8:885–897. doi: 10.1364/optica.426870.</cite> [<a href="https://doi.org/10.1364/optica.426870" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11759494/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39867356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Optica&amp;title=Instant%20FLIM%20enables%204D%20in%20vivo%20lifetime%20imaging%20of%20intact%20and%20injured%20zebrafish%20and%20mouse%20brains&amp;author=Y%20Zhang&amp;author=IH%20Guldner&amp;author=EL%20Nichols&amp;author=D%20Benirschke&amp;author=CJ%20Smith&amp;volume=8&amp;publication_year=2021a&amp;pages=885-897&amp;pmid=39867356&amp;doi=10.1364/optica.426870&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib70">
<cite>Zhang SX, Lutas A, Yang S, Diaz A, Fluhr H, Nagel G, Gao S, Andermann ML. Hypothalamic dopamine neurons motivate mating through persistent cAMP signalling. Nature. 2021b;597:245–249. doi: 10.1038/s41586-021-03845-0.</cite> [<a href="https://doi.org/10.1038/s41586-021-03845-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8884112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34433964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Hypothalamic%20dopamine%20neurons%20motivate%20mating%20through%20persistent%20cAMP%20signalling&amp;author=SX%20Zhang&amp;author=A%20Lutas&amp;author=S%20Yang&amp;author=A%20Diaz&amp;author=H%20Fluhr&amp;volume=597&amp;publication_year=2021b&amp;pages=245-249&amp;pmid=34433964&amp;doi=10.1038/s41586-021-03845-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib71">
<cite>Zheng K, Bard L, Reynolds JP, King C, Jensen TP, Gourine AV, Rusakov DA. Time-resolved imaging reveals heterogeneous landscapes of nanomolar Ca(2+) in neurons and astroglia. Neuron. 2015;88:277–288. doi: 10.1016/j.neuron.2015.09.043.</cite> [<a href="https://doi.org/10.1016/j.neuron.2015.09.043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4622934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26494277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Time-resolved%20imaging%20reveals%20heterogeneous%20landscapes%20of%20nanomolar%20Ca(2+)%20in%20neurons%20and%20astroglia&amp;author=K%20Zheng&amp;author=L%20Bard&amp;author=JP%20Reynolds&amp;author=C%20King&amp;author=TP%20Jensen&amp;volume=88&amp;publication_year=2015&amp;pages=277-288&amp;pmid=26494277&amp;doi=10.1016/j.neuron.2015.09.043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib72">
<cite>Zickus V, Wu ML, Morimoto K, Kapitany V, Fatima A, Turpin A, Insall R, Whitelaw J, Machesky L, Bruschini C, Faccio D, Charbon E. Fluorescence lifetime imaging with a megapixel SPAD camera and neural network lifetime estimation. Scientific Reports. 2020;10:e20986.  doi: 10.1038/s41598-020-77737-0.</cite> [<a href="https://doi.org/10.1038/s41598-020-77737-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7710711/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33268900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Fluorescence%20lifetime%20imaging%20with%20a%20megapixel%20SPAD%20camera%20and%20neural%20network%20lifetime%20estimation&amp;author=V%20Zickus&amp;author=ML%20Wu&amp;author=K%20Morimoto&amp;author=V%20Kapitany&amp;author=A%20Fatima&amp;volume=10&amp;publication_year=2020&amp;pages=e20986&amp;pmid=33268900&amp;doi=10.1038/s41598-020-77737-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section></section><article class="sub-article" id="sa0"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. doi: <a href="https://doi.org/10.7554/eLife.101559.3.sa0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.101559.3.sa0</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>eLife Assessment</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiao%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Jie Xiao</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jie Xiao</span></h3>
<div class="p">
<sup>1</sup>Johns Hopkins University, United States</div>
<div>Reviewing Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiao%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jie Xiao</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.c" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a.c" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.c" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.c" class="d-panel p" style="display: none">
<div class="p" id="aff7">
<sup>1</sup>Johns Hopkins University, United States</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Jie Xiao</span></strong>: <span class="role">Reviewing Editor</span>
</div>
</div>
<div id="anp_a.c" class="d-panel p" style="display: none"><div class="notes p">
<section id="kwd-group3" class="kwd-group"><p><strong>Keywords:</strong> Compelling</p></section><section id="kwd-group4" class="kwd-group"><p><strong>Keywords:</strong> Important</p></section>
</div></div>
<div id="clp_a.c" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>This study presents an <strong>important</strong> computational framework, FLiSimBA (Fluorescence Lifetime Simulation for Biological Applications), for modeling experimental limitations in Fluorescence Lifetime Imaging Microscopy (FLIM). FLiSimBA is readily available in MATLAB and Python, enables users to simulate effects of noise and varying sensor expression levels, and provides practical guidance for both lifetime imaging experiments and biosensor development. The analyses are robust, and the evidence supporting the tool's utility in distinguishing between multiple lifetime signals is <strong>compelling</strong>, indicating strong potential for multiplexed dynamic imaging. However, users should also consider that the tool's effectiveness depends on the suitability of a two-component discrete exponential model.</p></section></article><article class="sub-article" id="sa1"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. doi: <a href="https://doi.org/10.7554/eLife.101559.3.sa1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.101559.3.sa1</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Reviewer #1 (Public review):</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name given-only">Anonymous</span></a><div hidden="hidden" id="id6">
<h3><span class="name given-only">Anonymous</span></h3>
<div>Reviewer</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link"><span class="name given-only">Anonymous</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.d" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.d" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.d" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name given-only">Anonymous</span></strong>: <span class="role">Reviewer</span>
</div>
</div>
<div id="clp_a.d" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>In this study, Ma et al. aimed to determine previously uncharacterized contributions of tissue autofluorescence, detector afterpulse, and background noise on fluorescence lifetime measurement interpretations. They introduce a computational framework they named "Fluorescence Lifetime Simulation for Biological Applications (FLiSimBA)" to model experimental limitations in Fluorescence Lifetime Imaging Microscopy (FLIM) and determine parameters for achieving multiplexed imaging of dynamic biosensors using lifetime and intensity. By quantitatively defining sensor photon effects on signal to noise in either fitting or averaging methods of determining lifetime, the authors contradict any claims of FLIM sensor expression insensitivity to fluorescence lifetime and highlight how these artifacts occur differently depending on analysis method. Finally, the authors quantify how statistically meaningful experiments using multiplexed imaging could be achieved.</p>
<p>A major strength of the study is the effort to present results in a clear and understandable way given that most researcher do not think about these factors on a day-to-day basis. Additionally, the model code is readily available in Matlab and Python, which should allow for open access to a larger community.</p>
<p>Overall, the authors' achieved their aims of demonstrating how common factors (autofluorescence, background, and sensor expression) will affect lifetime measurements and they present a clear strategy for understanding how sensor expression may confound results if not properly considered. This work should bring to awareness an issue that new users of lifetime biosensors may not be aware of and that experts, while aware, have not quantitatively determine the conditions where these issues arise. This work will also point to future directions for improving experiments using fluorescence lifetime biosensors and the development of new sensors with more favorable properties.</p></section></article><article class="sub-article" id="sa2"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. doi: <a href="https://doi.org/10.7554/eLife.101559.3.sa2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.101559.3.sa2</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Reviewer #3 (Public review):</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name given-only">Anonymous</span></a><div hidden="hidden" id="id7">
<h3><span class="name given-only">Anonymous</span></h3>
<div>Reviewer</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link"><span class="name given-only">Anonymous</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.e" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.e" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.e" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name given-only">Anonymous</span></strong>: <span class="role">Reviewer</span>
</div>
</div>
<div id="clp_a.e" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>Summary:</p>
<p>This study presents a useful computational tool, termed FLiSimBA. The MATLAB-based FLiSimBA simulations allow users to examine the effects of various noise factors (such as autofluorescence, afterpulse of the photomultiplier tube detector, and other background signals) and varying sensor expression levels. Under the conditions explored, the simulations unveiled how these factors affect the observed lifetime measurements, thereby providing useful guidelines for experimental designs. Further simulations with two distinct fluorophores uncovered conditions in which two different lifetime signals could be distinguished, indicating multiplexed dynamic imaging may be possible.</p>
<p>Strengths:</p>
<p>The simulations and their analyses were done systematically and rigorously. FliSimba can be useful for guiding and validating fluorescence lifetime imaging studies. The simulations could define useful parameters such as the minimum number of photons required to detect a specific lifetime, how sensor protein expression level may affect the lifetime data, the conditions under which the lifetime would be insensitive to the sensor expression levels, and whether certain multiplexing could be feasible.</p>
<p>Weaknesses:</p>
<p>The analyses have relied on a key premise that the fluorescence lifetime in the system can be described as a two-component discrete exponential decay. This means that the experimenter should ensure that this is the right model for their fluorophores a priori.</p></section></article><article class="sub-article" id="sa3"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. 2025 Aug 20;13:RP101559. doi: <a href="https://doi.org/10.7554/eLife.101559.3.sa3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.101559.3.sa3</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Pingchuan Ma</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Pingchuan Ma</span></h3>
<div class="p">
<sup>1</sup>Washington University in St. Louis, St. Louis, United States</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pingchuan Ma</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Peter Chen</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Peter Chen</span></h3>
<div class="p">
<sup>2</sup>Washington University in St. Louis, St. Louis, United States</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peter Chen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sternson%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Scott Sternson</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Scott Sternson</span></h3>
<div class="p">
<sup>3</sup>Howard Hughes Medical Institute, University of California, San Diego, La Jolla, United States</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sternson%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Scott Sternson</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Yao Chen</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Yao Chen</span></h3>
<div class="p">
<sup>4</sup>Washington University in St. Louis, St. Louis, United States</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yao Chen</span></a>
</div>
</div>
<sup>4</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.f" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a.f" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.f" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.f" class="d-panel p" style="display: none">
<div class="p" id="aff8">
<sup>1</sup>Washington University in St. Louis, St. Louis, United States</div>
<div id="aff9">
<sup>2</sup>Washington University in St. Louis, St. Louis, United States</div>
<div id="aff10">
<sup>3</sup>Howard Hughes Medical Institute, University of California, San Diego, La Jolla, United States</div>
<div id="aff11">
<sup>4</sup>Washington University in St. Louis, St. Louis, United States</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Pingchuan Ma</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Peter Chen</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Scott Sternson</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yao Chen</span></strong>: <span class="role">Author</span>
</div>
</div>
<div id="anp_a.f" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub4" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a.f" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>The following is the authors’ response to the original reviews.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #1 (Public review):</strong>
</p>
<p>In this study, Ma et al. aimed to determine previously uncharacterized contributions of tissue autofluorescence, detector afterpulse, and background noise on fluorescence lifetime measurement interpretations. They introduce a computational framework they named "Fluorescence Lifetime Simulation for Biological Applications (FLiSimBA)" to model experimental limitations in Fluorescence Lifetime Imaging Microscopy (FLIM) and determine parameters for achieving multiplexed imaging of dynamic biosensors using lifetime and intensity. By quantitatively defining sensor photon effects on signal-to-noise in either fitting or averaging methods of determining lifetime, the authors contradict any claims of FLIM sensor expression insensitivity to fluorescence lifetime and highlight how these artifacts occur differently depending on the analysis method. Finally, the authors quantify how statistically meaningful experiments using multiplexed imaging could be achieved.</p>
<p>A major strength of the study is the effort to present results in a clear and understandable way given that most researchers do not think about these factors on a day-to-day basis. The model code is available and written in Matlab, which should make it readily accessible, although a version in other common languages such as Python might help with dissemination in the community. One potential weakness is that the model uses parameters that are determined in a</p>
<p>specific way by the authors, and it is not clear how vastly other biological tissue and microscope setups may differ from the values used by the authors.</p>
<p>Overall, the authors achieved their aims of demonstrating how common factors</p>
<p>(autofluorescence, background, and sensor expression) will affect lifetime measurements and they present a clear strategy for understanding how sensor expression may confound results if not properly considered. This work should bring to awareness an issue that new users of lifetime biosensors may not be aware of and that experts, while aware, have not quantitatively determined the conditions where these issues arise. This work will also point to future directions for improving experiments using fluorescence lifetime biosensors and the development of new sensors with more favorable properties.</p>
</blockquote>
<p>We appreciate the comments and helpful suggestions. We now also include FLiSimBA simulation code in Python in addition to Matlab to make it more accessible to the community.</p>
<p>One advantage of FLiSimBA is that the simulation package is flexible and adaptable, allowing users to input parameters based on the specific sensors, hardware, and autofluorescence measurements for their biological and optical systems. We used parameters based on a FRETbased sensor, measured autofluorescence from mouse tissue, and measured dark count/after pulse of our specific GaAsP PMT in this manuscript as examples. In Discussion and Materials and methods, we now emphasize this advantage and further clarify how these parameters can be adapted to diverse tissues, imaging systems, and sensors based on individual experiments. We further explain that these input parameters will not affect the conclusions of our study, but the specific input parameters would alter the quantitative thresholds.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #2 (Public review):</strong>
</p>
<p>Summary:</p>
<p>By using simulations of common signal artefacts introduced by acquisition hardware and the sample itself, the authors are able to demonstrate methods to estimate their influence on the estimated lifetime, and lifetime proportions, when using signal fitting for fluorescence lifetime imaging.</p>
<p>Strengths:</p>
<p>They consider a range of effects such as after-pulsing and background signal, and present a range of situations that are relevant to many experimental situations.</p>
<p>Weaknesses:</p>
<p>A weakness is that they do not present enough detail on the fitting method that they used to estimate lifetimes and proportions. The method used will influence the results significantly. They seem to only use the "empirical lifetime" which is not a state of the art algorithm. The method used to deconvolve two multiplexed exponential signals is not given.</p>
</blockquote>
<p>We appreciate the comments and constructive feedback. Our revision based on the reviewer’s suggestions has made our manuscript clearer and more user friendly. We originally described the detail of the fitting methods in Materials and methods. Given the importance of these methodological details for evaluating the conclusions of this study, we have moved the description of the fitting method from Materials and methods to Results. In addition, we provide further clarification and more details of the rationale of using these different methods of lifetime estimates in Discussion to aid users in choosing the best metric for evaluating fluorescence lifetime data.</p>
<p>More specifically, we modified our writing to highlight the following.</p>
<p>(1) In Results, we describe that lifetime histograms were fitted to Equation 3 with the GaussNewton nonlinear least-square fitting algorithm and the fitted P</p>
<p>(2) In Results, we clarify that our simulation of multiplexed imaging was modeled with two sensors, each displaying a single exponential decay, but the two sensors have different decay constants. We also describe that Equation 3 with the Gauss-Newton nonlinear least-square fitting algorithm was used to deconvolve the two multiplexed exponential signals (Fig. 8)</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #3 (Public review):</strong>
</p>
<p>Summary:</p>
<p>This study presents a useful computational tool, termed FLiSimBA. The MATLAB-based FLiSimBA simulations allow users to examine the effects of various noise factors (such as autofluorescence, afterpulse of the photomultiplier tube detector, and other background signals) and varying sensor expression levels. Under the conditions explored, the simulations unveiled how these factors affect the observed lifetime measurements, thereby providing useful guidelines for experimental designs. Further simulations with two distinct fluorophores uncovered conditions in which two different lifetime signals could be distinguished, indicating multiplexed dynamic imaging may be possible.</p>
<p>Strengths:</p>
<p>The simulations and their analyses were done systematically and rigorously. FliSimba can be useful for guiding and validating fluorescence lifetime imaging studies. The simulations could define useful parameters such as the minimum number of photons required to detect a specific lifetime, how sensor protein expression level may affect the lifetime data, the conditions under which the lifetime would be insensitive to the sensor expression levels, and whether certain multiplexing could be feasible.</p>
<p>Weaknesses:</p>
<p>The analyses have relied on a key premise that the fluorescence lifetime in the system can be described as two-component discrete exponential decay. This means that the experimenter should ensure that this is the right model for their fluorophores a priori and should keep in mind that the fluorescence lifetime of the fluorophores may not be perfectly described by a twocomponent discrete exponential (for which alternative algorithms have been implemented: e.g., Steinbach, P. J. Anal. Biochem. 427, 102-105, (2012)). In this regard, I also couldn't find how good the fits were for each simulation and experimental data to the given fitting equation (Equation 2, for example, for Figure 2C data).</p>
</blockquote>
<p>We thank the reviewer for the constructive feedback. We agree that the FLiSimBA users should ensure that the right decay equations are used to describe the fluorescent sensors. In this study, we used a FRET-based PKA sensor FLIM-AKAR to provide proof-of-principle demonstration of the capability of FLiSimBA. The donor fluorophore of FLIM-AKAR, truncated monomeric enhanced GFP, displays a single exponential decay. FLIM-AKAR, a FRET-based sensor, displays a double exponential decay. The time constants of the two exponential components were determined and reported previously (Chen, et al, Neuron (2017)). Thus, a double exponential decay equation with known τ<sub>1</sub> and τ<sub>2</sub> was used for both simulation and fitting. The goodness of fit is now provided in Supplementary Fig. 1 for both simulated and experimental data. In addition to referencing our prior study characterizing the double exponential decay model of FLIM-AKAR in Materials and methods, we have emphasized in Discussion the versality of FLiSimBA to adapt to different sensors, tissues, and analysis methods, and the importance of using the right mathematical models to describe the fluorescence decay of specific sensors.</p>
<blockquote class="text-italic"><p>Also, in Figure 2C, the 'sensor only' simulation without accounting for autofluorescence (as seen in Sensor + autoF) or afterpulse and background fluorescence (as seen in Final simulated data) seems to recapitulate the experimental data reasonably well. So, at least in this particular case where experimental data is limited by its broad spread with limited data points, being able to incorporate the additional noise factors into the simulation tool didn't seem to matter too much.</p></blockquote>
<p>In the original Fig 2C, the sensor fluorescence was much higher than the contributions from autofluorescence, afterpulse, and background signals, resulting in minimal effects of these other factors, as the reviewer noted. This original figure was based on photon counts from single neurons expressing FLIM-AKAR. For the rest of the manuscript, photon counts were based on whole fields of view (FOV). Since the FOV includes cells that do not express fluorescent sensors, the influence of autofluorescence, dark currents, and background is much more pronounced, as shown in Fig. 2B.</p>
<p>Both approaches – using photon counts from the whole FOV or from individual neurons – have their justifications. Photon counts from the whole FOV simulate data from fluorescence lifetime photometry (FLiP), whereas photon counts from individual neurons simulate data from fluorescence lifetime imaging microscopy (FLIM). However, the choice of approach does not affect the conclusions of the manuscript, as a range of photon count values are simulated. To maintain consistency throughout the manuscript, we have revised the photon counts in this figure (now Supplementary Fig. 1C) to match those from the whole FOV.</p>
<p>Additionally, we have made some modifications in our analyses of Supplementary Fig. 1C and Fig. 2B, detailed in the “FLIM analysis” section of Materials and methods. For instance, to minimize system artifact interference at the histogram edges, we now use a narrower time range (1.8 to 11.5 ns) for fitting and empirical lifetime calculation.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #1 (Recommendations for the authors):</strong>
</p>
<p>(1) The authors report how autofluorescence was measured from "imaged brain slices from mice at postnatal 15 to 19 days of age without sensor expression." However, it remains unclear how many acute slices and animals were used (for example, were all 15um x 15um FOV from a single slice) and if mouse age affects autofluorescence quantification. Furthermore, would in vivo measurements have different autofluorescence conditions given that blood flow would be active? It would help if the authors more clearly explained how reliable their autofluorescence measurement is by clarifying how they obtained it, whether this would vary across brain areas, and whether in vitro vs in vivo conditions would affect autofluorescence.</p>
</blockquote>
<p>We have added description in Materials and methods that for autofluorescence ‘Fluorescence decay histograms from 19 images of two brain slices from a single mouse were averaged.’ We have added in Discussion that users should carefully ‘measure autofluorescence that matches the age, brain region, and data collection conditions (e.g., ex vivo or in vivo) of their tissue…’, and emphasize that FLiSimBA offers customization of inputs, and it is important for users to adapt the inputs such as autofluorescence to their experimental conditions. We also clarify in Discussion that the change of input parameters such as autofluorescence across age and brain region would not affect the general insights from this study, but will affect quantitative values.</p>
<blockquote class="text-italic"><p>(2) Does sensor expression level issues arise more with in-utero electroporation compared to AAV-based delivery of biosensors? A brief comment on this in the discussion may help as most users in the field today may be using AAV strategies to deliver biosensors.</p></blockquote>
<p>In our experience, in-utero electroporation results in higher sensor expression than AAV-based delivery, and so pose less concern for expression-level dependence. However, both delivery methods can result in expression level dependence, especially with a sensor that is not bright. We have added in Discussion ‘For a sensor with medium brightness delivered via in utero electroporation, adeno-associated virus, or as a knock-in gene, the brightness may not always fall within the expression level-independent regime.’</p>
<blockquote class="text-italic">
<p>(3) Figure 1. Should the x-axis on the top figures be "Time (ns)" instead of "Lifetime (ns)"?</p>
<p>Similarly in Figure 8A&amp;B, wouldn't it make more sense to have the x-axis be Time not Lifetime?</p>
</blockquote>
<p>The x-axis labels in Fig. 1 and Fig. 8A-8B have been changed to ‘Time (ns)’.</p>
<blockquote class="text-italic">
<p>(4) Figure 2b: why is the empirical lifetime close to 3.5ns? Shouldn't it be somewhere between</p>
<p>2.14 and 0.69?</p>
</blockquote>
<p>In our empirical lifetime calculation, we did not set the peak channel to have a time of 0.0488 ns (i.e. the laser cycle 12.5 ns divided by 256 time channels). Rather, we set the first time channel within a defined calculation range (i.e. 1.8 ns in Supplementary Fig. 1B) to have a time of 0.0488 ns (i.e.). Thus, the empirical lifetime exceeds 2.14 ns and depends on the time range of the histogram used for calculation.</p>
<p>For Fig. 2B and Supplementary Fig. 1C, we have now adjusted the range to 1.8-11.5 ns to eliminate FLIM artifacts at the histogram edges in our experimental data, resulting in an empirical lifetime around 2.255 ns. In contrast, the range for calculating the empirical lifetime of simulated data in the rest of the study (e.g. Fig. 4D) is 0.489-11.5 ns, yielding a larger lifetime of ~3.35 ns.</p>
<p>We have clarified these details and our rationale in Materials and methods.</p>
<blockquote class="text-italic"><p>(5) Figure 2b: how come the afterpulse+background contributes more to the empirical lifetime than the autofluorescence (shorter lifetime). This was unclear in the results text why autofluorescence photons did not alter empirical lifetime as much as did the afterpulse/background.</p></blockquote>
<p>With a histogram range from 1.8 ns to 11.5 ns used in Fig. 2B, the empirical lifetime for FLIM-AKAR sensor fluorescence, autofluorescence, and background/afterpulse are: 2-2.3 ns, around 1.69 ns, and around 4.90 ns. The larger difference of background/afterpulse from FLIM-AKAR sensor fluorescence leads to larger influence of afterpulse+background than autofluorescence. We have added an explanation of this in Results.</p>
<blockquote class="text-italic"><p>(6) One overall suggestion for an improvement that could help active users of lifetime biosensors understand the consequences would be to show either a real or simulated example of a "typical experiment" conducted using FLIM-AKAR and how an incorrect interpretation could be drawn as a consequence of these artifacts. For example, do these confounds affect experiments involving comparisons across animals more than within-subject experiments such as washing a drug onto the brain slice, and the baseline period is used to normalize the change in signal? I think this type of direct discussion will help biosensor users more deeply grasp how these factors play out in common experiments being conducted.</p></blockquote>
<p>We have added the following in Discussion, ‘…While this issue is less problematic when the same sample is compared over short periods (e.g. minutes), It can lead to misinterpretation when fluorescence lifetime is compared across prolonged periods or between samples when comparison is made across chronic time periods or between samples with different sensor expression levels. For example, apparent changes in fluorescence lifetime observed over days, across cell types, or subcellular compartments may actually reflect variations in sensor expression levels rather than true differences in biological signals (Fig. 6), Therefore, considering biologically realistic factors in FLiSimBA is essential, as it qualitatively impacts the conclusions.’</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #2 (Recommendations for the authors):</strong>
</p>
<p>The paper would be improved with more detail on the fitting methods, and the use of state-of-theart methods. Consult for example the introduction of this paper where many methods are listed: <a href="https://www.mdpi.com/1424-8220/22/19/7293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.mdpi.com/1424-8220/22/19/7293</a></p>
</blockquote>
<p>We have moved the description of the Gauss-Newton nonlinear least-square fitting algorithm from Materials and methods to Results to enhance clarity. We appreciate the reviewer’s suggestion to combine FLiSimBA with various analysis methods. However, the primary focus of our manuscript is to call for attention of how specific contributing factors in biological experiments influence FLIM data, and to provide a tool that rigorously considers these factors to simulate FLIM data, which can then be used for fitting. Therefore, we did not expand the scope of our manuscript. Instead, we have added in the Discussion that ‘‘FLiSimBA can be used to test multiple fitting methods and lifetime metrics as an exciting future direction for identifying the best analysis method for specific experimental conditions’, citing relevant references.</p>
<blockquote class="text-italic"><p>I would also improve the content of the GitHub repository as it is very hard to identify to source code used for simulation and fitting.</p></blockquote>
<p>We have reorganized and relabeled our GitHub repository and now have three folders labeled as ‘Simulation_inMatlab’, ‘DataAnalysis_inMatlab’, and ‘SimulationAnalysis_inPython’. We also updated the clarification of the contents of each folder in the README file.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #3 (Recommendations for the authors):</strong>
</p>
<p>(1) P. 10 "For example, to detect a P1 change of 0.006 or a lifetime change of 5 ps with one sample measurement in each comparison group, approximately 300,000 photons are needed." If I am reading the graphs in Figures 3B and C, this sentence is talking about the red line. However, the intersection of 0.006 in the MDD of P1 in 3B and red is not 3E5 photons. And the intersection of 0.005 ns and red in 3C is not 3E5 photons either. Are you sure you are talking about n=1? Maybe the values are correct for the blue curve with n=5.</p>
</blockquote>
<p>Thank you for catching our error. We have corrected the text to ‘with five sample measurements’.</p>
<blockquote class="text-italic"><p>(2) Figure 2 (B) legend: It would be helpful to specify what is being compared in the legend. For example, consider revising "* p &lt; 0.05 vs sensor only; n.s. not significant vs sensor + autoF; # p &lt; 0.05 vs sensor + autoF. Two-way ANOVA with Šídák's multiple comparisons test" to "* p &lt;0.05 for sensor + auto F (cyan) vs sensor only; n.s. not significant for final simulated data (purple) vs sensor + autoF; # p &lt; 0.05 for final simulated data (purple) vs sensor + autoF. Twoway ANOVA with Šídák's multiple comparisons test".</p></blockquote>
<p>We’ve made the change and thanks for the suggestion to make it clearer.</p>
<blockquote class="text-italic"><p>(3) Figure 2 (c) Can you please show the same Two-way ANOVA test values for Experimental vs. Sensor only and for Experimental vs. Sensor + autoF? Currently, the value (n.s.) is marked only for Experimental vs. Final simulation. Given that the experimental data are sparse (compared to the simulations), it seems likely that there may be no significant difference among the 3 different simulations regarding how well they match the experimental data. Also, can you specify the P1 and P2 of the experimental data used to generate the simulated data on this panel? Also, what is the reason why P1=0.5 was used for panels A and B, instead of the value matching the experimental value?</p></blockquote>
<p>As the reviewer suggested, we have included statistical tests in the figure (now Supplementary Fig. 1C). Please see our response to the Public Review of Reviewer 3’s comments as well as our changes in Materials and Methods on other changes and their rationale for this figure. We have now specified the P<sub>1</sub> value of the experimental data used to generate the simulated data on this panel both in Figure Legends and Materials and Methods. Based on the suggestion, we have now used the same P<sub>1</sub> value in Fig. 2B.</p></section></article><article class="sub-article" id="_ad93_"><section class="pmc-layout__citation font-secondary font-xs"><div></div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Associated Data</h1></hgroup><ul class="d-buttons inline-list"></ul>
<div class="d-panels font-secondary-light"></div>
<div></div>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
</div></section><section class="body sub-article-body"><section id="_adsm93_" lang="en" class="supplementary-materials"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>MDAR checklist</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367299/bin/elife-101559-mdarchecklist1.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-101559-mdarchecklist1.docx</a><sup> (89KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The current manuscript is a computational study. The MATLAB and Python functions for simulation and data analyses are deposited at <a href="https://github.com/YaoChenLabWashU/Publication/tree/main/Simulation_manuscript" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GitHub</a> (<a href="#bib33" class="usa-link" aria-describedby="bib33">Ma et al., 2025</a>).</p></section></section></article><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from eLife are provided here courtesy of <strong>eLife Sciences Publications, Ltd</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.7554/eLife.101559"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/elife-101559.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12367299/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12367299/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367299%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367299/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12367299/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12367299/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40832998/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12367299/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40832998/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12367299/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12367299/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="x9TiyUOozncXyl80beLs1YnmmFctxwwDLkCzkQUJgLE2cPLA3vEXna7STmOG40G0">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
