
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Virome characteristics of small mammals and their associated environments in pastoral areas on the Qinghai-Tibet Plateau - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532DAA8AF209D3052DAA0032EDB67C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="npjbiofilms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371082/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="NPJ Biofilms and Microbiomes">
<meta name="citation_title" content="Virome characteristics of small mammals and their associated environments in pastoral areas on the Qinghai-Tibet Plateau">
<meta name="citation_author" content="Xiang Lu">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China">
<meta name="citation_author_institution" content="Institute of Critical Care Medicine, The Affiliated People’s Hospital, Jiangsu University, Zhenjiang, China">
<meta name="citation_author" content="Qing Zhang">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China">
<meta name="citation_author_institution" content="Qinghai Institute of Endemic Disease Prevention and Control, Xining, China">
<meta name="citation_author" content="Haisheng Wu">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China">
<meta name="citation_author_institution" content="Qinghai Institute of Endemic Disease Prevention and Control, Xining, China">
<meta name="citation_author" content="Likai Ji">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China">
<meta name="citation_author_institution" content="Institute of Critical Care Medicine, The Affiliated People’s Hospital, Jiangsu University, Zhenjiang, China">
<meta name="citation_author" content="Yongshun Wang">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China">
<meta name="citation_author_institution" content="Qinghai Institute of Endemic Disease Prevention and Control, Xining, China">
<meta name="citation_author" content="Juan Xu">
<meta name="citation_author_institution" content="Clinical Laboratory Center, The Affiliated Taizhou People’s Hospital of Nanjing Medical University, Taizhou, China">
<meta name="citation_author" content="Jia Liu">
<meta name="citation_author_institution" content="Qinghai Institute of Endemic Disease Prevention and Control, Xining, China">
<meta name="citation_author" content="Jiamin Pan">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China">
<meta name="citation_author" content="Xiaojie Jiang">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China">
<meta name="citation_author" content="Yuan Xi">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China">
<meta name="citation_author" content="Shixing Yang">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China">
<meta name="citation_author_institution" content="Institute of Critical Care Medicine, The Affiliated People’s Hospital, Jiangsu University, Zhenjiang, China">
<meta name="citation_author" content="Quan Shen">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China">
<meta name="citation_author" content="Xiaochun Wang">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China">
<meta name="citation_author_institution" content="Institute of Critical Care Medicine, The Affiliated People’s Hospital, Jiangsu University, Zhenjiang, China">
<meta name="citation_author" content="Ping Wu">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China">
<meta name="citation_author" content="Yuwei Liu">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China">
<meta name="citation_author" content="Xiao Ma">
<meta name="citation_author_institution" content="Qinghai Institute of Endemic Disease Prevention and Control, Xining, China">
<meta name="citation_author" content="Chenglin Zhou">
<meta name="citation_author_institution" content="Clinical Laboratory Center, The Affiliated Taizhou People’s Hospital of Nanjing Medical University, Taizhou, China">
<meta name="citation_author" content="Tongling Shan">
<meta name="citation_author_institution" content="Shanghai Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Shanghai, China">
<meta name="citation_author" content="Wen Zhang">
<meta name="citation_author_institution" content="Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China">
<meta name="citation_author_institution" content="Institute of Critical Care Medicine, The Affiliated People’s Hospital, Jiangsu University, Zhenjiang, China">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="11">
<meta name="citation_firstpage" content="168">
<meta name="citation_doi" content="10.1038/s41522-025-00814-7">
<meta name="citation_pmid" content="40841544">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371082/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371082/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371082/pdf/41522_2025_Article_814.pdf">
<meta name="description" content="The Qinghai-Tibet Plateau (QTP), known for its rich biodiversity and unique ecological niche, harbors numerous endemic species and serves as a reservoir for both established human pathogens and numerous poorly characterized pathogens. Despite ...">
<meta name="og:title" content="Virome characteristics of small mammals and their associated environments in pastoral areas on the Qinghai-Tibet Plateau">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The Qinghai-Tibet Plateau (QTP), known for its rich biodiversity and unique ecological niche, harbors numerous endemic species and serves as a reservoir for both established human pathogens and numerous poorly characterized pathogens. Despite ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371082/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371082">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41522-025-00814-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41522_2025_Article_814.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371082%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371082/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371082/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371082/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-npjbiofilms.jpg" alt="NPJ Biofilms and Microbiomes logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NPJ Biofilms and Microbiomes" title="Link to NPJ Biofilms and Microbiomes" shape="default" href="http://www.nature.com/npjbiofilms/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">NPJ Biofilms Microbiomes</button></div>. 2025 Aug 22;11:168. doi: <a href="https://doi.org/10.1038/s41522-025-00814-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41522-025-00814-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22NPJ%20Biofilms%20Microbiomes%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Virome characteristics of small mammals and their associated environments in pastoral areas on the Qinghai-Tibet Plateau</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Xiang Lu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xiang Lu</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div class="p">
<sup>2</sup>Institute of Critical Care Medicine, The Affiliated People’s Hospital, Jiangsu University, Zhenjiang, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiang Lu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Qing Zhang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Qing Zhang</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div class="p">
<sup>3</sup>Qinghai Institute of Endemic Disease Prevention and Control, Xining, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qing Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Haisheng Wu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Haisheng Wu</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div class="p">
<sup>3</sup>Qinghai Institute of Endemic Disease Prevention and Control, Xining, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haisheng Wu</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ji%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Likai Ji</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Likai Ji</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div class="p">
<sup>2</sup>Institute of Critical Care Medicine, The Affiliated People’s Hospital, Jiangsu University, Zhenjiang, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ji%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Likai Ji</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yongshun Wang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yongshun Wang</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div class="p">
<sup>3</sup>Qinghai Institute of Endemic Disease Prevention and Control, Xining, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yongshun Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Juan Xu</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Juan Xu</span></h3>
<div class="p">
<sup>4</sup>Clinical Laboratory Center, The Affiliated Taizhou People’s Hospital of Nanjing Medical University, Taizhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Juan Xu</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jia Liu</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jia Liu</span></h3>
<div class="p">
<sup>3</sup>Qinghai Institute of Endemic Disease Prevention and Control, Xining, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jia Liu</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pan%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Jiamin Pan</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jiamin Pan</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pan%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiamin Pan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Xiaojie Jiang</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Xiaojie Jiang</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaojie Jiang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xi%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Yuan Xi</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Yuan Xi</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xi%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuan Xi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Shixing Yang</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Shixing Yang</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div class="p">
<sup>2</sup>Institute of Critical Care Medicine, The Affiliated People’s Hospital, Jiangsu University, Zhenjiang, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shixing Yang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shen%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Quan Shen</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Quan Shen</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shen%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Quan Shen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Xiaochun Wang</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Xiaochun Wang</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div class="p">
<sup>2</sup>Institute of Critical Care Medicine, The Affiliated People’s Hospital, Jiangsu University, Zhenjiang, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaochun Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Ping Wu</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Ping Wu</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Ping Wu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Yuwei Liu</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Yuwei Liu</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuwei Liu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Xiao Ma</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Xiao Ma</span></h3>
<div class="p">
<sup>3</sup>Qinghai Institute of Endemic Disease Prevention and Control, Xining, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiao Ma</span></a>
</div>
</div>
<sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Chenglin Zhou</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Chenglin Zhou</span></h3>
<div class="p">
<sup>4</sup>Clinical Laboratory Center, The Affiliated Taizhou People’s Hospital of Nanjing Medical University, Taizhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chenglin Zhou</span></a>
</div>
</div>
<sup>4,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shan%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Tongling Shan</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Tongling Shan</span></h3>
<div class="p">
<sup>5</sup>Shanghai Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shan%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tongling Shan</span></a>
</div>
</div>
<sup>5,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Wen Zhang</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Wen Zhang</span></h3>
<div class="p">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div class="p">
<sup>2</sup>Institute of Critical Care Medicine, The Affiliated People’s Hospital, Jiangsu University, Zhenjiang, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wen Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Laboratory Medicine, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu China </div>
<div id="Aff2">
<sup>2</sup>Institute of Critical Care Medicine, The Affiliated People’s Hospital, Jiangsu University, Zhenjiang, China </div>
<div id="Aff3">
<sup>3</sup>Qinghai Institute of Endemic Disease Prevention and Control, Xining, China </div>
<div id="Aff4">
<sup>4</sup>Clinical Laboratory Center, The Affiliated Taizhou People’s Hospital of Nanjing Medical University, Taizhou, China </div>
<div id="Aff5">
<sup>5</sup>Shanghai Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Shanghai, China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 12; Accepted 2025 Aug 13; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371082  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841544/" class="usa-link">40841544</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The Qinghai-Tibet Plateau (QTP), known for its rich biodiversity and unique ecological niche, harbors numerous endemic species and serves as a reservoir for both established human pathogens and numerous poorly characterized pathogens. Despite certain advances, virological research on small mammals in this region is still limited. In this study, we investigated the diversity and evolution of viral communities in plateau voles, plateau pikas, and their associated environmental samples collected near pastoral settlements on the QTP. Our findings revealed 291,165 species-level viral operational taxonomic units, highlighting a diverse array of bacteriophages. In addition, we further characterized 774 novel eukaryotic viral genomes, primarily classified into 22 established viral families. Among them, 95.5% shared less than 80% amino acid sequence identity with known viruses, some of which may represent new genera or species. Furthermore, some eukaryotic viruses display distinct tissue distribution patterns in these small mammals. Notably, we identified 32 mammalian viruses capable of cross-species transmission, among which 22 are considered potential spillover risks, and most of them were detected for the first time in plateau pikas and plateau voles. Overall, this study expands the viral spectrum of small mammals on the QTP and broadens the host range of previously identified viruses, emphasizing the need for further research into their potential public health implications.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Microbial communities, Environmental microbiology, Virology</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Emerging zoonotic diseases of wildlife origin caused by previously unknown pathogens have become a major global public health challenge<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. The Qinghai-Tibet Plateau (QTP), one of the world’s most biodiverse regions and a habitat for numerous endemic species, is home to widely distributed and abundant small mammals, including lagomorphs and rodents, which are closely associated with humans. They play a crucial role in ecosystems by facilitating energy transfer and nutrient cycling, dispersing seeds, and contributing to biodiversity maintenance<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. However, as dominant species in this region, the plateau pika (<em>Ochotona curzoniae</em>) and plateau vole (<em>Neodon fuscus</em>) serve as key reservoirs for the transmission of infectious diseases such as echinococcosis and plague<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>, which can spread to humans through predation, excretion, and various other pathways<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>.</p>
<p id="Par3">To date, research on plateau pikas and plateau voles on the QTP has primarily focused on parasites and gut microbiota<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. However, virological research remains limited, with only a few studies, such as Zhu et al., having characterized the RNA virome of birds and small mammals on the QTP<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Plateau pikas have been identified as potential hosts for influenza viruses, including H7N2<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, H9N2<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, and H5N1<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Recently, Zhu et al. identified a novel alphacoronavirus, plateau pika coronavirus (PPCoV) P83, in gut and respiratory samples from plateau pikas<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Likewise, Xu et al. detected coronaviruses in plateau pikas and Gansu pikas, classifying them as variants of pika alphacoronaviruses or betacoronaviruses<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Luo et al. characterized a novel arenavirus, plateau pika virus (PPV)<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>, from plateau pikas on the QTP. These findings have not only revealed pathogenic viruses hidden in pikas but also suggested the possible presence of numerous previously undiscovered viruses in this region, which may circulate among a broader range of hosts. Therefore, these findings underscore the urgent need for systematic virome investigations in small mammals on the QTP. Additionally, studies have demonstrated that plateau pikas influence soil microbial networks and the multifunctionality of ecosystems through their activities, functioning as “ecosystem engineers”<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. This ecological impact is further reflected in the higher plant species richness observed in regions inhabited by pikas compared to pika-free areas<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Thus, analyzing environmental samples from the surroundings of these animals’ burrows can help assess whether viruses persist in the ecosystem and spread through excretions, bodily fluids, or direct contact, thereby offering insights into potential cross-host transmission.</p>
<p id="Par4">In this study, we captured plateau pikas and plateau voles active around pastoral settlements on the QTP and conducted a comprehensive virome analysis. Additionally, we collected samples from arthropods, plants, fungi, and soil in close proximity to burrow sites, as well as fresh feces from yaks grazing near these sites. Our aim was to identify novel viruses that may be hidden in these environments, characterize the host range and distribution of known viruses, and evaluate the potential spillover risks of viruses carried by these small mammals.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Virome of QTP small mammals and their associated environments</h3>
<p id="Par5">Virome analysis was conducted on samples from small mammals (plateau voles and plateau pikas) and their associated environments in multiple pastoral settlements across three county-level administrative regions at an average altitude of 4133 m (see Methods for details). Environment samples were collected from areas surrounding animal burrows, specifically near capture sites where burrows were visible. Notably, samples from Chengduo had the highest biodiversity, while in Jiuzhi, plateau voles were found only in barren landscapes slightly farther from pastoral settlements (Fig. <a href="#Fig1" class="usa-link">1a</a>). A total of 392,069 viral contigs were identified across all samples following de novo assembly (Fig. <a href="#Fig1" class="usa-link">1b</a>). After deduplication and clustering, these contigs were classified into 291,165 species-, 76,106 genus-, and 28,704 family-level viral operational taxonomic units (vOTUs). The viral species richness of these samples was assessed using rarefaction curves (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). As the number of sampled sequences increased, the curves gradually approached a plateau, suggesting that the sequencing depth was sufficient to capture the majority of viral diversity present in these libraries. This indicates that additional sequencing would yield only a limited number of new species. Only 15.7% of the vOTUs were successfully annotated at the family level, comprising 1.9% vertebrate-associated viruses and 13.8% other annotated viruses. Among the vertebrate viruses, the most frequently assigned families were <em>Retroviridae</em>, <em>Picobirnaviridae</em>, <em>Smacoviridae</em>, <em>Parvoviridae</em>, and <em>Circoviridae</em>. The other annotated viruses were dominated by <em>Microviridae</em> and several families within the class <em>Caudoviricetes</em>, such as <em>Intestiviridae</em>, <em>Steigviridae</em>, <em>Herelleviridae</em>, and <em>Suoliviridae</em>, which are mainly associated with prokaryotic hosts (Fig. <a href="#Fig1" class="usa-link">1c</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Map of sampling sites from this study.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371082_41522_2025_814_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/726e/12371082/d4c60a0f97b0/41522_2025_814_Fig1_HTML.jpg" loading="lazy" id="d33e451" height="1000" width="712" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> An overview of the sampling sites in Qinghai Province, China, with three distinct sampling locations indicated by pie charts. The top-left image was taken by Qing Zhang. <strong>b</strong> The number of viral contigs and species-level vOTUs clustered from different sample categories, with the phylogenetic relationships of the samples indicated. <strong>c</strong> The distribution of 291,165 species-level vOTUs is plotted based on their length and GC content. The bar charts at the top show the distribution of these vOTUs further clustered at the genus and family levels. Dots are colored according to the number of vOTUs. The pie chart shows the proportion of well-annotated vOTUs across different classifications.</p></figcaption></figure><p id="Par6">The assignment of most sequences to bacteriophages (phages) was expected (Fig. <a href="#Fig2" class="usa-link">2a</a>). Overall, eukaryotic viruses constitute a minor component of microbial communities, yet their potential impacts are often evident<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Identifying these viruses, especially novel eukaryotic viruses, is essential for anticipating and mitigating potential future threats. We assembled 774 complete or nearly complete eukaryotic viral genomes from the majority of animal tissue and environmental samples included in this study, with nearly complete genomes being those that contain at least one annotatable conserved motif. These viruses shared sequence identity with 22 established viral families known to infect vertebrates, invertebrates, plants, and fungi, or with two groups of unclassified Circular Replication (Rep)-Encoding Single-Stranded (CRESS)-DNA viruses and unclassified RNA viruses, 95.5% of which shared less than 80% amino acid (aa) sequence identity with any known viruses (Supplementary Data <a href="#MOESM2" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Comparison of viral community composition and diversity across different regions and animal tissues.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371082_41522_2025_814_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/726e/12371082/cd4bc16249cb/41522_2025_814_Fig2_HTML.jpg" loading="lazy" id="d33e501" height="999" width="708" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Heatmap of viral families in the collected samples. The differently colored bands at the top of the heatmap indicate the sample sources, while the row names on the right represent the names of viral families. The data is log-transformed with a base of log<sub>10</sub>. <strong>b</strong> The left panel presents boxplots of Shannon diversity (alpha diversity) of viral communities in the overall tissues of plateau pikas and voles. The right panel presents boxplots of Shannon diversity of viral communities across four tissue types from plateau voles and plateau pikas. The Shannon index was analyzed using the Wilcoxon Rank-Sum Test. Statistical significance was defined as follows: *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, **<em>*P</em> &lt; 0.001, and ****<em>P</em> &lt; 0.0001; comparisons were made between the two groups. <strong>c</strong> The left panel presents boxplots of Shannon diversity of viral communities across different plateau pika tissues. The right panel shows the PCoA analysis, which revealed significant differences among viral communities from different plateau pika tissues. The two principal components accounted for 25% (PC1) and 19% (PC2) of the total variation, respectively. Each symbol represents an individual sample. <strong>d</strong> The boxplots show the Shannon index of viral communities across different vole tissues in three sampling regions. <strong>e</strong> The PCoA analysis separately illustrates the significant differences in viral communities among different plateau vole tissues across sampling sites, as well as the variations in viral communities of the four tissue types across the three sampling sites.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Virome diversity of plateau pikas and plateau voles</h3>
<p id="Par7">To gain a deeper understanding of the virome in plateau pikas and plateau voles, we constructed separate libraries for each individual’s liver, lung, intestine, and heart/spleen/kidney/brain (HSKB) samples. The heart, spleen, kidney, and brain samples were pooled into a single library, as these tissues were expected to yield very low viral loads, thereby improving detection sensitivity. Overall, the viral diversity in plateau vole tissues was higher than that in plateau pika tissues (Wilcoxon Rank-Sum Test, <em>P</em> &lt; 0.0001), a consistent pattern observed across four distinct tissue types (Fig. <a href="#Fig2" class="usa-link">2b</a>). This could be attributed to the smaller sample size of plateau pikas relative to plateau voles. Moreover, plateau pika samples were collected only from pastoral settlements in a single county-level administrative region, whereas plateau voles inhabited more biodiverse environments. Including a larger number of samples and incorporating sampling sites with greater geographic and ecological diversity may alter the currently observed virome landscape.</p>
<p id="Par8">In plateau pikas, the viral diversity in intestinal samples was significantly higher than that in other tissues, which is expected, as the intestine is typically the most virus-rich organ in most mammals<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> and humans<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. However, there was no significant difference in viral diversity among lung, liver, and HSKB samples (Fig. <a href="#Fig2" class="usa-link">2c</a>). Additionally, Principal Coordinate Analysis (PCoA) revealed that the viral composition of intestinal tissues was distinctly different from that of other tissues. Notably, although α diversity did not differ significantly among lung, liver, and HSKB samples, their overall viral community structures showed significant differences in β diversity analysis. This indicated that while these tissues had similar viral richness and evenness, their community compositions remained distinct.</p>
<p id="Par9">Similarly, plateau voles from Chengduo and Gande exhibited a viral diversity pattern comparable to that of plateau pikas (Fig. <a href="#Fig2" class="usa-link">2d, e</a>). However, in voles sampled from Jiuzhi, the distribution of viral diversity followed a different trend, with significant differences observed only between the intestine and lung tissues. Furthermore, PCoA analysis failed to distinguish the overall viral community structures among different tissues in Jiuzhi voles. Given this, increasing the sample size may help reveal a clearer diversity pattern. Additionally, we observed significant β diversity differences in the same tissue of plateau voles across different regions, which may be partially attributed to the interaction between the voles and their surrounding environments.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Highly diverse phages harbored in QTP small mammals and their associated environments</h3>
<p id="Par10">Phages are widely prevalent across diverse habitats and play a critical role in maintaining bacterial community dynamics and ecosystem stability. The majority of the annotated viruses in this study were phages (over 70%), primarily belonging to <em>Caudoviricetes</em> and <em>Microviridae</em>, while the unannotated sequences (84.3%) may also represent a substantial proportion of phages (Fig. <a href="#Fig1" class="usa-link">1c</a>). <em>Caudoviricetes</em> comprise a diverse class of tailed double-stranded DNA (dsDNA) phages found in various environments and were highly represented in gut metagenomes<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. To investigate the evolutionary patterns of this group in QTP small mammals and their associated environments, we generated a phylogenetic tree using a concatenated alignment of 77 protein-coding marker genes<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> (Fig. <a href="#Fig3" class="usa-link">3a</a>). After excluding genomes with insufficient data (fewer than three markers or less than 5% alignment representation), the final phylogenetic tree consisted of 4380 genus-level viral genomes, with reference genomes sourced from the most comprehensive human gut virome database, MGV<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Compared to the MGV database, this study identified 830 putative novel vOTUs at the genus level, with an average length of 36,859 bp. Notably, as more viral genomic data become available, the resolution and accuracy of detecting novel vOTUs are expected to improve. Among them, 235 were from plateau pikas, 213 from plateau voles, 187 from yak feces, 72 from arthropod samples, 62 from plant and fungal samples, and 61 from soil samples. Intestinal vOTUs were the most abundant, totaling 609 (73.4%), with 220 from plateau pikas, 202 from plateau voles, and 187 from yaks. Moreover, <em>Caudoviricetes</em> were not detected in the HSKB and lung samples of plateau pikas, nor in the HSKB samples of plateau voles. Additionally, the majority of the vOTUs were predicted to be assigned to host phyla <em>Actinobacteria</em>, <em>Bacteroidetes</em>, <em>Firmicutes</em>, and <em>Proteobacteria</em>, while only two genomes were classified as archaeaphages. The percentage of lysogenic viruses varied across host phyla, with most exhibiting a low proportion (Supplementary Data <a href="#MOESM2" class="usa-link">1</a>). <em>Microviridae</em>, a family of viruses that infect bacteria, is widely recognized as one of the most diverse and globally distributed viral families<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. They inhabit a variety of environments, including the guts of animals and humans<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, insects<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, freshwater<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>, seawater<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, and sediments<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. A total of 512 distinct major capsid proteins (MCPs) were identified at the genus level, with an average sequence length of 514 aa (Supplementary Data <a href="#MOESM2" class="usa-link">1</a>). Among these, 360 MCPs exhibited less than 60% aa sequence identity with their closest matches in the GenBank database. Furthermore, network clustering analysis showed that a subset of MCPs identified in this study grouped into several major clusters with known sequences, the majority of which could not be classified into specific viral genera or species. In contrast, another group of MCPs formed clusters consisting of a single or a few MCPs, suggesting novel genera (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). The evolutionary diversity of phages is thought to be driven by horizontal gene transfer with other microbes or host genomes<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. To explore how gut viruses differ in their evolutionary relationships and functional potential among plateau voles, plateau pikas, and yaks living in the same region, we examined the relationship between nucleotide-level genetic distance and gene content similarity among viral genomes from these hosts (Fig. <a href="#Fig3" class="usa-link">3b</a> and Supplementary Data <a href="#MOESM2" class="usa-link">1</a>). Overall, gut viruses from different hosts showed substantial nucleotide-level divergence and low gene content similarity. In contrast, the viromes of plateau pikas and plateau voles displayed greater functional similarity, suggesting a closer ecological relationship between the two species. Furthermore, the moderate gene content similarity observed between the viromes of pikas and yaks may indicate potential ecological interactions or viral exchange.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Phylogenetic analysis of <em>Caudoviricetes</em> and cross-host evolutionary relationships of gut viromes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371082_41522_2025_814_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/726e/12371082/d8a5513d96f1/41522_2025_814_Fig3_HTML.jpg" loading="lazy" id="d33e649" height="276" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> To enhance visualization, only one representative genome is shown for each genus-level vOTU. Branch colors indicate their sources: black represents lineages from a previously published study, while green denotes those unique to this study. The outer rings display four metadata attributes for each vOTU. <strong>b</strong> The scatter plot illustrates the relationship between pairwise nucleotide distance and gene content similarity among gut viromes of plateau pikas, plateau voles, and yaks. Each dot represents a comparison between two viral genomes derived from different host species. The x-axis shows the nucleotide distance, while the y-axis indicates gene content similarity, reflecting the extent of functional overlap between viral genomes.</p></figcaption></figure><p id="Par11">Phage sequence-derived genes were annotated with the KEGG search function in eggNOG-mapper v2, enabling the investigation of their potential roles (Supplementary Data <a href="#MOESM3" class="usa-link">2</a>). To focus on host-associated phages and reduce environmental interference, we restricted our analysis to those detected in plateau pikas and voles. As expected, these functional genes were predominantly detected in intestinal tissues (Fig. <a href="#Fig4" class="usa-link">4a</a>). The results revealed that the majority of annotated genes were associated with genetic information processing, particularly its subcategory, replication and repair, with a significant portion remaining unclassified, indicating their critical roles in preserving genomic integrity and cellular functions. To enhance the visualization of relationships between these genes and host tissues, we included only genes with a weight ≥ 50 in the network diagram. Among these genes, those associated with pyrimidine metabolism (<em>thyX</em> and <em>dut</em>) and DNA replication (<em>dnaE</em> and <em>polA</em>) had the highest numbers and were detected more frequently in the intestinal tissues of plateau pikas than in those of plateau voles (Fig. <a href="#Fig4" class="usa-link">4b</a>). Interestingly, although more plateau vole samples were included in this study compared to plateau pikas, and their intestinal tissues exhibited greater viral diversity (Fig. <a href="#Fig2" class="usa-link">2b</a>), these genes were still present in higher numbers in plateau pikas. One possible explanation is the differences in adaptive strategies between plateau pikas and plateau voles for high-altitude adaptation.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Functional gene analysis of viruses in plateau pika and vole tissues.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371082_41522_2025_814_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/726e/12371082/25d0ea2a7137/41522_2025_814_Fig4_HTML.jpg" loading="lazy" id="d33e690" height="915" width="800" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The bar chart on the left shows the number of functional genes in different plateau pika and vole tissue categories annotated based on the eggNOG database, while the pie chart on the right illustrates the classification of these functional genes. <strong>b</strong> Network diagram of viral functional genes and their source tissues. Arrows always point from genes to their sources, with genes shown as red circles and sources as green circles. The thickness and color intensity of the arrows are positively correlated with the degree of association between genes and their sources.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">Evolutionary diversification of novel eukaryotic viruses</h3>
<p id="Par12">We manually validated the genomes of 774 viral sequences and inferred phylogenetic trees based on RNA-dependent RNA polymerase (RdRp) sequences for RNA viruses, replicase or other non-structural proteins for DNA viruses, and full-length ORF1 sequences for anelloviruses (Fig. <a href="#Fig5" class="usa-link">5</a> and Supplementary Data <a href="#MOESM2" class="usa-link">1</a>). The 22 unedited phylogenetic trees with detailed sequence information are available in Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Phylogenetic analysis of 22 major viral families.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371082_41522_2025_814_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/726e/12371082/8faee83d6979/41522_2025_814_Fig5_HTML.jpg" loading="lazy" id="d33e715" height="877" width="800" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The phylogenetic trees were primarily constructed based on the amino acid sequences of RdRp for RNA viruses and replicase and other non-structural proteins for DNA viruses. Additionally, following ICTV classification guidelines, the phylogenetic tree for <em>Anelloviridae</em> was constructed using the nucleotide sequences of the complete ORF1.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">Positive-sense ssRNA viruses and retrotranscribing RNA viruses</h3>
<p id="Par13">A total of 99 positive-sense ssRNA viruses were detected, classified into eleven viral families (<em>Arteriviridae</em>, <em>Deltaflexiviridae</em>, <em>Flaviviridae</em>, <em>Astroviridae</em>, <em>Benyviridae</em>, <em>Hepeviridae</em>, <em>Dicistroviridae</em>, <em>Picornaviridae</em>, <em>Mitoviridae</em>, <em>Nodaviridae</em>, and <em>Iflaviridae</em>), as well as <em>Betaflexi</em>-like viruses. In addition, 13 members of the <em>Retroviridae</em> family (ssRNA-RT viruses) were also identified. Within the <em>Arteriviridae</em> family, four viruses closely related to Jingmen rodent arterivirus 1 were identified in the lungs of plateau voles from Gande and Chengduo, sharing 84.3–85.3% aa sequence identity with this virus. Interestingly, viral fragments were also detected in the liver and HSKB samples of voles, suggesting potential systemic spread within the host (see the section ‘Cross-species transmission of viruses’ for details). Twenty-four viruses were classified into the <em>Flaviviridae</em> family, with phylogenetic analysis effectively grouping them into distinct clades based on host preference. Among them, two <em>Pegivirus</em> viruses shared 83.8–83.9% aa sequence identity with <em>Phaiomys leucurus</em> pegivirus. Additionally, 18 <em>Hepacivirus</em> viruses were identified as being related to <em>rodent hepacivirus</em>, sharing 67.4–87% aa sequence identity. Both <em>Pegivirus</em> and <em>Hepacivirus</em> viruses were detected in the liver and lungs of plateau voles. Two hepeviruses identified from the intestines of Chengduo plateau voles shared less than 55% aa sequence identity with any known viruses, suggesting the presence of novel viral species or possibly even new genera. Similarly, 24 picornavirus-related sequences were identified in the intestines of plateau pikas and voles, as well as in arthropods and fungi, indicating that these viruses occupy the ecological interface between QTP small mammals and their associated environments. Some of these sequences may represent novel species, as they formed distinct phylogenetic clades. Four astrovirus-related viruses were detected. Among them, two were identified in the intestines of plateau voles, sharing 67–68.8% aa sequence identity with their closest matches from voles in South Dakota. The other two viruses, detected in arthropod samples, exhibited low identity to known viruses and were therefore classified into the <em>Astro</em>-like group. Moreover, novel viruses related to <em>Iflaviridae</em>, <em>Deltaflexiviridae</em>, <em>Betaflexiviridae</em>, <em>Dicistroviridae</em>, <em>Benyviridae</em>, and <em>Mitoviridae</em> were detected, with natural hosts primarily including invertebrates, plants, fungi, and insects. However, these viruses were only sporadically identified in small mammals, suggesting that they may have a limited role in the interactions between QTP small mammals and their associated environments. In contrast, <em>Nodaviridae</em> appears to be more frequently involved in viral transmission between small mammals and their associated environments.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Negative-sense ssRNA viruses</h3>
<p id="Par14">A total of 12 negative-sense ssRNA viruses associated with the families <em>Chuviridae</em>, <em>Rhabdoviridae</em>, <em>Xinmoviridae</em>, and <em>Phenuiviridae</em> were identified from arthropods, plants, and fungi. Among the five viruses associated with <em>Chuviridae</em>, one shared 74.7% aa sequence identity with a Chuvirus previously identified in the feces of a rat from the QTP, suggesting it represents a novel species. The other four exhibited 34–45.8% aa sequence identity with their closest matches in GenBank, indicating that they likely represent four distinct novel genera<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. A novel virus belonging to <em>Rhabdoviridae</em> was identified, with a complete genome length of 14,494 bp. It shared 36.7% aa sequence identity with its closest match, which was originally detected in <em>Lasius neglectus</em>. Phylogenetic analysis revealed that it formed an independent clade between the genera <em>Alphahymrhavirus</em> and <em>Betapaprhavirus</em>, suggesting that it may represent a novel genus or a higher taxonomic rank. Similarly, a virus with a full genome length of 13,447 bp was classified within the <em>Xinmoviridae</em> family. It shared 38% aa sequence identity with its closest match, <em>Soldier fly-associated anphevirus</em>, identified in <em>Inopus flavus</em>. Given its low sequence identity and phylogenetic analysis, it is likely to represent at least a novel species. Additionally, five viruses related to <em>Phenuiviridae</em> were identified. Among them, one virus shared 95.7% aa sequence identity with its closest match, while the remaining four exhibited less than 55% identity with their respective closest matches. Based on phylogenetic analysis, each of these four viruses likely represents a novel species.</p></section><section id="Sec9"><h3 class="pmc_sec_title">dsRNA viruses</h3>
<p id="Par15">In this study, a total of 154 picobirnaviruses were identified, with 144 derived from the intestines of plateau pikas and voles, 6 from the lungs of voles, 1 from the liver of a pika, and the remaining 3 from environmental samples. These viruses exhibited aa sequence identities ranging from 25.2% to 79.8% with their most closely related sequences in GenBank, including those detected in pigs, foxes, chickens, marmots, mongooses, and cattle. Notably, the vast majority of sequences were detected in the intestines of small mammals, suggesting an association with the host’s diet. However, the relatively low number of viruses detected in environmental samples, compared with the number found in the host intestines, suggests that these viruses are primarily transmitted among mammals rather than being widely circulating in the environment.</p></section><section id="Sec10"><h3 class="pmc_sec_title">ssDNA and dsDNA viruses</h3>
<p id="Par16">A total of 63 sequences related to <em>Parvoviridae</em> were identified from the intestines of plateau pikas, the intestines and HSKB of plateau voles, as well as from arthropods, plants, fungi, and soil samples. Among them, 49 sequences shared less than 70% aa sequence identity with their closest matches. Additionally, these parvoviruses span a wide taxonomic range from the subfamilies <em>Densovirinae</em> to <em>Parvovirinae</em>, suggesting their widespread distribution on the QTP. Moreover, four viruses belonging to the genus <em>Mastadenovirus</em> were detected in the tissues of plateau pikas and voles. Additionally, four viruses related to the family <em>Polyomaviridae</em> were identified from the liver and lungs of plateau voles, the lungs of plateau pikas, and arthropods. Phylogenetic analysis based on the aa sequences of the large tumor antigen revealed that these viruses clustered into distinct evolutionary clades.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Two novel monophyletic clades of <em>Anelloviridae</em>
</h3>
<p id="Par17">A total of 25 viruses belonging to the family <em>Anelloviridae</em> were identified, all derived from various tissues of plateau pikas and voles. Interestingly, BLASTn searches of their ORF1 nucleotide sequences returned no significant matches. Additionally, phylogenetic analysis revealed that these viruses formed two distinct clades, comprising 5 and 20 viruses, respectively. These two clades should at least be considered to represent two distinct new genera.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Unclassified RdRp-encoding RNA and CRESS DNA viruses</h3>
<p id="Par18">We recovered 108 RdRp sequences from the dataset, which exhibit a notable degree of divergence from all currently known RdRp sequences, making it difficult to assign them to any established viral taxonomy. Sequence similarity network analysis revealed that most sequences formed distinct clusters, suggesting an evolutionary relationship with known but yet-to-be-classified viruses (Fig. <a href="#Fig6" class="usa-link">6</a>). In contrast, some isolated RdRp sequences may represent new viral species, genera, or families. CRESS DNA viruses are widely distributed and have been detected across a wide range of eukaryotic hosts<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. These viruses display remarkable genetic diversity and continue to proliferate without signs of decline<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. To investigate the diversity of CRESS DNA viruses in small mammals and their associated environments on the QTP, we recovered 292 Rep protein sequences from the dataset. These sequences included 11 circoviruses, 15 genomoviruses, 4 geminiviruses, 25 smacoviruses, and 237 unclassified CRESS DNA viruses. Further sequence analysis showed that 177 Rep proteins exhibited less than 60% aa sequence identity in comparison to known viruses, suggesting that they may represent potentially novel viral species. Notably, CRESS DNA viruses exhibited distinct host specificity. Compared to other eukaryotic viruses, smacoviruses were detected at significantly higher proportions in yak feces, whereas the representation of other eukaryotic viruses in yak feces was relatively low. This finding suggests that the viromes of the yak gut and small mammals have limited ecological interactions.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Sequence similarity networks of two unclassified virus groups.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371082_41522_2025_814_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/726e/12371082/2638e35900c8/41522_2025_814_Fig6_HTML.jpg" loading="lazy" id="d33e915" height="840" width="800" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Viruses identified with RdRp / Rep sequences but unable to be classified were included in the sequence similarity network analysis along with their best matches from the current GenBank database and other representative viruses.</p></figcaption></figure></section><section id="Sec13"><h3 class="pmc_sec_title">Tissue-associated differences in the presence and composition of eukaryotic viruses in plateau pikas and plateau voles</h3>
<p id="Par19">To better understand the distribution of eukaryotic viruses across different tissues in plateau pikas and plateau voles, we clustered sequences annotated as eukaryotic viruses using 95% ANI across 90% of the shortest assembled contig to define species-level clusters. Consequently, these viruses were classified into 14 viral families and two unclassified viral groups (Supplementary Data <a href="#MOESM4" class="usa-link">3</a>). The heatmap illustrates the distribution of different viral groups across various tissues (Fig. <a href="#Fig7" class="usa-link">7a</a>). Notably, the unclassified <em>Cressdnaviricota</em> family was detected in a large proportion of samples but appeared more frequently in the intestinal tissues of voles (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). Additionally, as noted earlier, smacoviruses were more commonly found in yak feces. <em>Flaviviridae</em> was mainly present in the liver and lungs of voles, whereas <em>Anelloviridae</em> was found more often in pika tissues than in vole tissues. More specifically, <em>Adenoviridae</em> was detected only in the intestinal tissues or the HSKB of voles, with no evidence of cross-tissue transmission. It is worth noting that analyzing the heart, spleen, kidney, and brain separately when constructing libraries could provide a clearer picture of their distribution. <em>Arteriviridae</em> appears to be exclusively associated with voles. Additionally, <em>Anelloviridae</em> was widely detected across multiple tissues and individuals of both plateau pikas and plateau voles, suggesting a strong capacity for tissue-level transmission, though no clear evidence of cross-species transmission has been observed. Similarly, <em>Circoviridae</em> and several yet-to-be-classified viruses exhibited comparable distribution patterns (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). Additionally, two representative contigs from the <em>Retroviridae</em> family, Hvole267_72163 (2978 bp) and Hvole267_1029205 (2974 bp), were originally detected in the intestine of plateau voles and share only 32.1% nucleotide identity, yet were unexpectedly found across various tissues of plateau pikas, suggesting the possibility of cross-species transmission (Fig. <a href="#Fig7" class="usa-link">7b</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. Mammalian viruses with potential for cross-species transmission.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371082_41522_2025_814_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/726e/12371082/b601886c232c/41522_2025_814_Fig7_HTML.jpg" loading="lazy" id="d33e981" height="761" width="800" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Circle heatmap illustrating the associations between viruses and host samples (tissues). Each circle represents a virus-host association, and the circle size is positively associated with the number of viruses detected in each host sample. <strong>b</strong> The heatmap on the left shows the distribution of viral families detected in various tissues of plateau voles and pikas. Color intensity represents the number of samples in which each viral family was identified. The heatmap on the right highlights two species-level representative contigs initially detected in the intestines of plateau voles that were also identified in different tissues of plateau pikas, indicating potential cross-species transmission. <strong>c</strong> The cross-species transmission pattern of 47 known viruses was mapped based on the data generated in this study and viral records from the NCBI database. The Latin names of mammalian samples collected in this study are displayed in yellow at the top of the figure. Blue circles indicate the presence of these viruses in different hosts (see Supplementary Data <a href="#MOESM5" class="usa-link">4</a> for details). On the right, purple and skyblue circles represent the presence of these viruses in plateau pika and vole tissues, respectively.</p></figcaption></figure></section><section id="Sec14"><h3 class="pmc_sec_title">Cross-species transmission of viruses</h3>
<p id="Par20">To evaluate the viruses identified in this study, particularly those associated with vertebrates and their potential public health impact, we focused on viral taxa with cross-species transmission potential (detected in at least two mammalian species) and spillover risk (detected in mammalian hosts from at least two distinct orders)<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Species-level classification of known eukaryotic viruses was based on ≥80% amino acid identity in conserved proteins<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. All known viruses included in the analysis were obtained from viral records in the NCBI database and the corresponding literature (Supplementary Data <a href="#MOESM5" class="usa-link">4</a>). We identified 32 mammalian viruses capable of cross-species transmission, 22 of which are considered at risk of spillover, including Rodent hepacivirus, Rodent hepatovirus, Canine parvovirus, Porcine picobirnavirus, and Canine-associated porprismacovirus, among others (Fig. <a href="#Fig7" class="usa-link">7c</a>). The genomic organizations of these viruses can be found in Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>.</p>
<p id="Par21">Notably, some viruses that had not been detected in other regions since their initial identification were also found in this study. Adenovirus myodes38640 (with spillover risk) was identified in <em>Myodes glareolus</em> from Ukraine; Chaetfec virus (with cross-species transmission potential) was detected in the feces of <em>Chaetodipus</em> sp. in Arizona, USA; and Mosavirus A2 was initially identified in the feces of <em>Coracias garrulus</em> collected in Hungary<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Mosavirus A2 is likely transmitted via bird droppings, raising the possibility that plateau voles may have acquired the virus through environmental exposure. In this study, however, Mosavirus A2 and Chaetfec virus were detected in the lungs and intestines of plateau voles (Supplementary Data <a href="#MOESM5" class="usa-link">4</a>). Additionally, some viruses are estimated to have spread globally, including Rodent hepacivirus, Porcine picobirnavirus, Rodent hepatovirus, Wigfec virus, Circoviridae sp. isolate ctda274, and Lama-associated gemycircularvirus 1.</p>
<p id="Par22">We identified canine parvovirus (<em>Protoparvovirus carnivoran1</em>) in various tissues of plateau voles collected from three county-level administrative regions, as well as in yak feces. This virus was previously identified in canine fecal samples obtained from the QTP<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Additionally, several viruses that had been previously identified in <em>Rodentia</em>, including Jingmen rodent arterivirus 1, Mamastrovirus sp. isolate XZS01, Phaiomys leucurus pegivirus, Rodent hepacvirus, Rodent hepatovirus, Chaetfec virus, Anelloviridae sp. isolate ratane153-zj2, and Anelloviridae sp. isolate AnelloviridaeDogfe362C1, were further characterized in various tissues of plateau voles.</p></section></section><section id="Sec15"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par23">The QTP has a unique ecological environment, featuring a large population of small rodents and the frequent activity of stray dogs, domestic dogs, Tibetan foxes, and red foxes<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>, thereby creating favorable conditions for the transmission of viral diseases. Small mammals, particularly rodents, exhibit regional differences in viral transmission and prevalence. However, some viruses they carry have achieved a global distribution<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, highlighting their potential role in long-distance viral dissemination. Living in close proximity to humans, rodents serve as key mediators of interactions among humans, arthropod vectors, and wildlife<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, facilitating the spread of viruses across ecological boundaries. Our study provides a comprehensive virome analysis of plateau pikas and plateau voles, along with their associated environments, significantly expanding the known viral diversity in this region.</p>
<p id="Par24">Our findings revealed 774 novel eukaryotic viral genomes, the majority of which shared low sequence identity with known viruses, suggesting the presence of previously uncharacterized viral lineages (Fig. <a href="#Fig5" class="usa-link">5</a>). The virome composition varied across different tissues, with intestinal samples harboring the highest viral diversity, consistent with previous studies on mammals and humans. Interestingly, although plateau voles exhibited higher overall viral diversity than plateau pikas, the latter harbored a greater number of viral-encoded functional genes. This discrepancy cannot be easily explained, as plateau voles exceed pikas in both habitat diversity and sample size. These findings underscore the need for further research to elucidate the ecological factors shaping viral diversity in these species. Although this study did not identify any known human pathogenic viruses in plateau pikas or plateau voles, the detection of nearly identical viral sequences across various animal samples and between animals and environmental sources, such as arthropods, suggests ongoing viral transmission between these hosts and their surrounding environment (Supplementary Data <a href="#MOESM4" class="usa-link">3</a>). Moreover, the diversity of viruses carried by arthropods was remarkably high. In this study, the newly identified arthropod-associated viruses were classified into 13 viral families, with some exhibiting substantial evolutionary divergence within their respective families. Notably, host interactions involving <em>Parvoviridae</em> appeared to be more frequent than those of other viral families, whereas <em>Iflaviridae</em> was exclusively found in arthropods (Fig. <a href="#Fig5" class="usa-link">5</a>).</p>
<p id="Par25">This study identified a large number of <em>Picobirnaviridae</em> viruses from plateau voles and pikas, which is consistent with previous studies<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>, suggesting that the virome of small mammals on the QTP exhibits a distinct composition. This distinctiveness is further reflected in the discovery of novel viruses, particularly the identification of new viral genera. For example, this study proposes two novel monophyletic clades of <em>Anelloviridae</em>. Notably, anelloviruses are not only found in animals but have also been detected colonizing more than 90% of the adult human population<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Therefore, this finding is more positive, as these sequences help fill gaps in known viral families, especially those identified in a wide range of samples, including humans, where they occupy a “narrow” yet distinct phylogenetic niche.</p>
<p id="Par26">A striking feature of the QTP virome was the dominance of phages, particularly those from <em>Microviridae</em> and <em>Caudoviricetes</em>, which accounted for over 70% of the annotated viral sequences. This suggests that phages play a fundamental role in shaping microbial communities in high-altitude ecosystems. Phylogenetic analysis of <em>Caudoviricetes</em> phages and network clustering analysis of <em>Microviridae</em> viruses revealed substantial genetic divergence from known phage genomes, indicating that the phage communities in plateau small mammals remain largely unexplored. While we used the MGV database in this study, which may introduce a bias toward detecting viral elements with potential relevance to human spillover, the increasing availability of phage genome sequences through databases such as IMG/VR<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup> is expected to mitigate this limitation and enable a more comprehensive understanding of phage diversity and evolution.</p>
<p id="Par27">A critical aspect of our study was the identification of 32 mammalian viruses with cross-species transmission potential, including 22 considered at risk of spillover. The detection of canine parvovirus in multiple plateau vole tissues and yak feces suggests potential interspecies transmission, likely facilitated by environmental exposure or direct contact. Similarly, Rodent hepacivirus and Rodent hepatovirus, previously reported in rodents, were detected in plateau voles, raising concerns about their expanded host range and zoonotic potential. Notably, Mosavirus A2 and Chaetfec virus, which had never been reported outside their original discovery locations, were identified in plateau voles, suggesting that viral dispersal across geographic regions may be more common than previously thought. Given that Mosavirus A2 is likely transmitted via bird droppings, its presence in plateau voles supports the possibility of an environmental transmission route that warrants further investigation. Interestingly, although both plateau pikas and plateau voles are dominant species on the QTP, no viruses with cross-species transmission potential have been found to be shared between them. The only exception is Adenovirus myodes38640, which crosses the host orders Rodentia and Lagomorpha, as it has been detected in both plateau pikas and another rodent species, <em>Myodes glareolus</em>. Notably, both plateau pikas and plateau voles exhibit cross-species viral transmission with <em>Phrynocephalus erythrurus</em>, a reptile species also collected from the QTP<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, as indicated by the presence of <em>Parvoviridae</em> sp. isolate QTP 1 and <em>Cressdnaviricota</em> sp. isolate QTP 1 (Supplementary Data <a href="#MOESM5" class="usa-link">4</a>). Therefore, future studies should incorporate a broader range of species to provide a more comprehensive perspective.</p>
<p id="Par28">In conclusion, this study provides key data for understanding the virome and evolutionary dynamics of small mammals on the QTP, particularly plateau voles and plateau pikas. These findings not only contribute to viral taxonomy but also help evaluate viral spillover risk, thereby mitigating the public health impact of emerging infectious diseases. However, preventive virological research should be continuously expanded to incorporate a broader range of samples for a more comprehensive risk assessment.</p></section><section id="Sec16"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec17"><h3 class="pmc_sec_title">Ethics approval</h3>
<p id="Par29">This study obtained approval from the Ethics Committee of the Qinghai Institute of Endemic Disease Prevention and Control (Approval No. 2024006) and adhered to the Wildlife Protection Law of the People’s Republic of China for sample collection. All sample pre-treatment was conducted in accordance with Biosafety Level 2 laboratory guidelines.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Sample collection, processing, and quality control</h3>
<p id="Par30">In June 2023, a total of 76 plateau voles and 43 plateau pikas, along with 100 environmental samples collected near burrows, were obtained from multiple pastoral settlements across three county-level administrative regions on the QTP. Specifically, 6 plateau voles were collected from Jiuzhi; 28 plateau voles, 13 arthropod samples, and 16 plant and fungal samples were collected from Gande; and 42 plateau voles, 43 plateau pikas, 6 yak fecal samples, 29 arthropod samples, 17 soil samples, and 19 plant and fungal samples were collected from Chengduo (Supplementary Data <a href="#MOESM2" class="usa-link">1</a>). All samples were stored in sterile containers and transported on dry ice. To better understand the viruses carried by small mammals, we separately collected the liver, lungs, and intestines of each individual plateau pika and plateau vole into distinct 1.5 mL microcentrifuge tubes, while the heart, spleen, kidney, and brain of the same individual were combined into a single tube. As a result, each individual plateau pika and plateau vole was associated with four sample tubes, resulting in a total of 576 tubes for subsequent library construction.</p>
<p id="Par31">Before conducting viral metagenomic analysis, samples were homogenized using a cryogenic grinder, suspended in 0.5 mL of Dulbecco’s phosphate-buffered saline (DPBS), and vigorously vortexed for 5 min, followed by three cycles of freeze-thaw to further release viral particles. Afterward, the samples were kept at 4 °C for 30 min. Following centrifugation at 15,000 × <em>g</em> for 10 min, the supernatants were transferred into 1.5 mL microcentrifuge tubes and preserved at −80 °C for later use<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. For each library, 100 µL of the supernatant was pipetted from an individual sample and transferred to a fresh 1.5 mL tube. These samples were centrifuged at 12,000 × <em>g</em> for 5 min at 4 °C and then filtered through a 0.45 µm membrane to enrich viral particles. The resulting filtrates were treated with RNase and DNase, and unprotected nucleic acids were digested at 37 °C for 60 min<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Total nucleic acids were subsequently extracted using the QIAamp MinElute Virus Spin Kit (Qiagen) following the manufacturer’s protocol. These nucleic acids, containing both DNA and RNA viral sequences, were subjected to reverse transcription with SuperScript III reverse transcriptase (Invitrogen) using 100 pmol of a random hexamer primer, followed by a single round of DNA synthesis via Klenow fragment polymerase (New England BioLabs). Libraries were prepared with the Nextera XT DNA Sample Preparation Kit (Illumina), and sequencing was carried out on the Illumina NovaSeq 6000 platform, generating 150 bp paired-end reads with dual barcoding.</p>
<p id="Par32">During the experiment, all procedures were conducted under strict precautions to prevent sample cross-contamination and nucleic acid degradation. We used aerosol filter tips to minimize the risk of sample cross-contamination. Additionally, all experimental consumables, such as microcentrifuge tubes and tips, that came into direct contact with nucleic acid samples were free of DNase and RNase. The samples were dissolved in DEPC-treated water containing RNase inhibitors. For blank controls, we simultaneously prepared sterile ddH2O and processed it under the same experimental conditions. Sample quality was assessed using agarose gel electrophoresis and the Agilent 2100 Bioanalyzer. During sequencing on the Illumina NovaSeq 6000 platform, the control pool yielded only a minimal number of reads.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Assembly and identification of viral contigs</h3>
<p id="Par33">Primers and low-quality reads were trimmed using Trim Galore v0.6.5 (<a href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.bioinformatics.babraham.ac.uk/projects/trim_galore</a>), and quality control was performed on the files using the following specific options: ‘–phred33 –length 50 –stringency 3 –paired‘. The paired-end reads were assembled using MEGAHIT v1.2.9<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> with the default parameters.</p>
<p id="Par34">We identified the viral origin of the contigs through a series of steps. First, the assembled contigs were screened against a local non-redundant (nr) protein database downloaded in May 2024. Contigs initially annotated as eukaryotic viruses were manually assembled in Geneious Prime v2022.0.1 (<a href="https://www.geneious.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.geneious.com</a>) and subsequently inspected, and then used as reference sequences for mapping to the raw data using the default recommended parameters to obtain complete or nearly complete genomes. The obtained genomic sequences were screened for potential vector contamination using VecScreen (<a href="https://www.ncbi.nlm.nih.gov/tools/vecscreen" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/tools/vecscreen</a>), followed by genomic clustering using MMseqs2<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup> with the parameters: ‘-k 0 -e 0.001 –min-seq-id 0.95 -c 0.9 –cluster-mode 0‘. Subsequently, these sequences were added to the non-redundant viral dataset along with sequences further identified as phage contigs.</p>
<p id="Par35">Phage contigs were recognized in accordance with the viral sequence identification SOP (10.17504/protocols.io.bwm5pc86). Contigs were validated using VirSorter2<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> with the option ‘–min-score 0.5‘ and were subsequently analyzed with CheckV<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> to remove host sequences flanking prophages. The potential phage contigs were screened based on data from VirSorter2 and CheckV outcomes, which took into account the counts of viral and host genes, VirSorter2 viral scores, and the presence of hallmark genes. These sequences underwent further validation using VIBRANT<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> with the option ‘-virome‘, and only those reconfirmed as viral were retained as bona fide phage sequences. These phage contigs were subsequently clustered at 95% average nucleotide identity (ANI) across 85% of the shortest contig following MIUViG standards<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>, using a custom script from the CheckV repository, to define phage populations.</p>
<p id="Par36">The complete non-redundant viral dataset (species-level vOTUs) was then compared against the nr database using the BLASTx program implemented in DIAMOND v2.0.15<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>, and significant sequences with an E-value cutoff of &lt;10<sup>−5</sup> were filtered. Taxonomic identification of the viral dataset was performed using TaxonKit<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> and the rma2info program within MEGAN6<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Phages that could not be classified by comparison with known sequences were further categorized using PhaGCN2<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>, following the latest ICTV 2024 release.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Clustering of viral sequences into vOTUs</h3>
<p id="Par37">To further cluster the species-level vOTU representatives at the genus and family levels, we adopted the scripts developed by Nayfach et al.<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, using a combination of gene sharing and AAI. Briefly, the ORFs of the vOTU representatives were predicted using Prodigal v2.6.3<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> with the option ‘-p meta‘. BLASTp (DIAMOND) was used with the parameters ‘–evalue 1e-5 –max-target-seqs 10000‘ to align all viral proteins. For each genome pair, genes with an e-value &lt; 1 × 10<sup>−5</sup> were retained to calculate the proportion of shared genes and their AAI. Genomes were assigned to the same family if they shared at least 10% or 8 genes, with a minimum AAI of 20% (parameters: ‘-min_percent_shared 10, -min_num_shared 8, -min_aai 20‘). Pairs meeting these criteria were retained and clustered using the Markov Cluster Algorithm (MCL) v14-137 with an inflation factor of 1.2. At the genus level, genomes sharing at least 20% or 16 genes, along with a minimum AAI of 40% (parameters: ‘-min_percent_shared 20, -min_num_shared 16, -min_aai 40‘), were clustered with an MCL inflation factor of 2.0.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Annotation of viral genomes</h3>
<p id="Par38">Geneious Prime was used with the parameters: minimum size = 100, start codon = ATG, to predict potential open reading frames (ORFs). These ORFs were subsequently validated through comparison with similar viruses in the GenBank database. The annotations of these ORFs were assigned based on comparisons with the built-in CDD v3.21, a version of the Conserved Domain Database (CDD)<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Functional annotation and ecological classification of phages</h3>
<p id="Par39">The ORFs of the phage contigs were functionally annotated by comparing them to the eggNOG v5.0<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> database using eggNOG-mapper v2<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> with the parameters ‘–translate –itype metagenome‘. eggNOG-mapper v2 is a tool for functional annotation based on precomputed orthology assignments. The KEGG functional annotation was obtained from the eggNOG-mapper analysis results.</p>
<p id="Par40">The host range and lifestyle of phages were predicted using PhaBOX v2<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. Phages identified as prophages or those encoding an integrase were classified as lysogenic. The lysogenic rate (%) was calculated based on VIBRANT results, representing the proportion of lysogenic viruses among all detected viruses within each probable host phylum<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Phylogenetic analysis and sequence similarity network analysis</h3>
<p id="Par41">To elucidate phylogenetic relationships, sequences belonging to different groups of related viruses were downloaded from the GenBank database, along with sequences of proposed species pending ratification. Nucleotide or protein sequences were aligned using MUSCLE, as implemented in MEGA-X<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>, and the results were trimmed with trimAl<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. Phylogenetic trees were constructed using IQ-TREE v2.3.6<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>, with 1,000 bootstrap replicates (-bb 1000) and the ModelFinder function (-m MFP). Interactive Tree Of Life (iTOL) was used to visualize and edit phylogenetic trees<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>.</p>
<p id="Par42">We have also assembled a sub-dataset comprising the protein sequences of the unclassified RdRp obtained in this study, which serves as a hallmark gene for RNA viruses, along with the five RdRp sequences that show the highest sequence identity to these sequences, as well as sequences from other known RNA virus families retrieved from GenBank. We employed MMseqs2 to cluster this sub-dataset and performed sequence similarity network analysis on the non-redundant sub-dataset using EFI-EST<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>, with an alignment score threshold of 86, corresponding to 35% sequence identity. The obtained network was visualized using Cytoscape v3.10<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> for subsequent analysis. Similarly, a sub-dataset of Reps from unclassified CRESS DNA viruses was generated using an alignment score threshold of 37. Likewise, a sub-dataset of MCPs from the family <em>Microviridae</em> was generated using an alignment score threshold of 84.</p>
<p id="Par43">For viruses assigned to the class <em>Caudoviricetes</em>, we downloaded the HMM profiles of 77 marker genes<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> from VOGDB<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup> and integrated the genus-level vOTUs identified in this study with the human gut virome database (MGV)<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> for marker gene analysis using HMMER v3.3.2<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. Only viruses with &gt;50% completeness, as determined by CheckV, were included in the analysis. Each marker gene was aligned using MAFFT<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>, and the resulting alignments were trimmed with trimAl to retain positions with less than 50% gaps. The filtered marker alignments were then concatenated, with gaps introduced where necessary to account for missing markers. Finally, only concatenated marker gene sets containing at least three marker genes and present in &gt;5% of all aligned concatemers were retained. We then constructed a concatenated protein phylogenetic tree from the multiple sequence alignment using FastTree v2.1.9<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>, employing the WAG + G model with the additional parameters ‘-mlacc 2 -slownni’.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Statistics and reproducibility</h3>
<p id="Par44">Statistical analyses and normalization were performed using MEGAN6 and R. Alpha- and beta-diversity analyses were performed using the vegan package, applying a statistical significance threshold of <em>P</em> &lt; 0.05. The Shannon index was assessed using the Wilcoxon Rank-Sum Test. Data visualization was carried out with the ggplot2 and ggpubr packages. PCoA based on Bray–Curtis dissimilarity was performed using the Permute, lattice, vegan, and ape packages. PERMANOVA was conducted with the adonis() function from the vegan package.</p></section></section><section id="Sec25"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371082/bin/41522_2025_814_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (11.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12371082/bin/41522_2025_814_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary data1</a><sup> (791.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12371082/bin/41522_2025_814_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary data2</a><sup> (5MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12371082/bin/41522_2025_814_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary data3</a><sup> (111.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12371082/bin/41522_2025_814_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary data4</a><sup> (143.5KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This research was financially supported by National Key Research and Development Programs of China (Nos. 2023YFD1801301 and 2022YFC2603801) and the National Natural Science Foundation of China (no. 82341106) to W.Z.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>All authors participated in the design, interpretation of the studies and analysis of the data and review of the manuscript; X.M., C.Z., T.S. and W.Z. contributed to the conception and design; X.L., Q.Z., H.W., L.J., Y.W., J.X., J.L., J.P., X.J., Y.X., S.Y., Q.S., X.W., P.W. and Y.L. contributed to the collection and assembly of data; X.L., Q.Z., H.W. and L.J. contributed to the data analysis and interpretation.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>The raw sequence data reported in this paper have been deposited in the Genome Sequence Archive (GSA) at the National Genomics Data Center (NGDC), China National Center for Bioinformation, Beijing Institute of Genomics, Chinese Academy of Sciences (GSA: CRA015789-CRA015791), and are publicly accessible at <a href="https://ngdc.cncb.ac.cn/gsa" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa</a>. The novel sequences reported in this paper have also been deposited in GenBase at NGDC and are freely accessible at <a href="https://ngdc.cncb.ac.cn/genbase" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/genbase</a> without any access restrictions. Detailed accession numbers can be found in Supplementary Data 1. Additionally, phages reported in this study have been archived in the Zenodo repository and can be accessed at 10.5281/zenodo.14992764.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par45">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Xiang Lu, Qing Zhang, Haisheng Wu, Likai Ji.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Xiao Ma, Email: maxiao0971@163.com.</p>
<p>Chenglin Zhou, Email: zhouchenglin@njmu.edu.cn.</p>
<p>Tongling Shan, Email: shantongling@shvri.ac.cn.</p>
<p>Wen Zhang, Email: zhangwen@ujs.edu.cn.</p></section><section id="sec26"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41522-025-00814-7.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Luo, X. L. et al. Emergence of an ancient and pathogenic mammarenavirus. <em>Emerg. Microbes Infect.</em><strong>12</strong>, e2192816 (2023).
</cite> [<a href="https://doi.org/10.1080/22221751.2023.2192816" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10337645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36939609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luo,%20X.%20L.%20et%20al.%20Emergence%20of%20an%20ancient%20and%20pathogenic%20mammarenavirus.%20Emerg.%20Microbes%20Infect.12,%20e2192816%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Zheng, J. X. et al. Species composition of a small mammal community and prevalence of Echinococcus spp. in the alpine pastoral area of the Eastern Tibetan Plateau. <em>Pathogens</em><strong>13</strong>, 10.3390/pathogens13070558 (2024).</cite> [<a href="https://doi.org/10.3390/pathogens13070558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11280319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39057785/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Wang, X. et al. Investigation on population dynamics and Echinococcus infections in small rodents around human settlement in Yushu City, Qinghai Province]. <em>Zhongguo Xue Xi Chong Bing. Fang. Zhi Za Zhi</em><strong>33</strong>, 346–352 (2021).
</cite> [<a href="https://doi.org/10.16250/j.32.1374.2020002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34505440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20X.%20et%20al.%20Investigation%20on%20population%20dynamics%20and%20Echinococcus%20infections%20in%20small%20rodents%20around%20human%20settlement%20in%20Yushu%20City,%20Qinghai%20Province%5D.%20Zhongguo%20Xue%20Xi%20Chong%20Bing.%20Fang.%20Zhi%20Za%20Zhi33,%20346%E2%80%93352%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Smith, A. T. &amp; Foggin, J. M. The plateau pika (Ochotona curzoniae) is a keystone species for biodiversity on the Tibetan plateau. <em>Animal Conserv.</em><strong>2</strong>, 235–240 (1999).</cite>
</li>
<li id="CR5">
<span class="label">5.</span><cite>Mills, J. N. &amp; Childs, J. E. Ecologic studies of rodent reservoirs: their relevance for human health. <em>Emerg. Infect. Dis.</em><strong>4</strong>, 529–537 (1998).
</cite> [<a href="https://doi.org/10.3201/eid0404.980403" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2640244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9866729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mills,%20J.%20N.%20&amp;%20Childs,%20J.%20E.%20Ecologic%20studies%20of%20rodent%20reservoirs:%20their%20relevance%20for%20human%20health.%20Emerg.%20Infect.%20Dis.4,%20529%E2%80%93537%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Mu, X. et al. Gut microbiome and antibiotic resistance genes in plateau model animal (Ochotona curzoniae) exhibit a relative stability under cold stress. <em>J. Hazard Mater.</em><strong>478</strong>, 135472 (2024).
</cite> [<a href="https://doi.org/10.1016/j.jhazmat.2024.135472" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39137548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mu,%20X.%20et%20al.%20Gut%20microbiome%20and%20antibiotic%20resistance%20genes%20in%20plateau%20model%20animal%20(Ochotona%20curzoniae)%20exhibit%20a%20relative%20stability%20under%20cold%20stress.%20J.%20Hazard%20Mater.478,%20135472%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Zhang, L. et al. Dysbiosis of gut microbiome aggravated male infertility in captivity of plateau pika. <em>Biomolecules</em><strong>14</strong>, 10.3390/biom14040403 (2024).</cite> [<a href="https://doi.org/10.3390/biom14040403" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11047922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38672421/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Zhao, J. et al. Livestock grazing is associated with the gut microbiota and antibiotic resistance genes in sympatric plateau pika (Ochotona curzoniae). <em>Integr. Zool.</em><strong>19</strong>, 646–661 (2024).
</cite> [<a href="https://doi.org/10.1111/1749-4877.12778" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37828802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20J.%20et%20al.%20Livestock%20grazing%20is%20associated%20with%20the%20gut%20microbiota%20and%20antibiotic%20resistance%20genes%20in%20sympatric%20plateau%20pika%20(Ochotona%20curzoniae).%20Integr.%20Zool.19,%20646%E2%80%93661%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Wu, Y. D. et al. Genetic evolution and implications of the mitochondrial genomes of two newly identified Taenia spp. in rodents from Qinghai-Tibet Plateau. <em>Front. Microbiol.</em><strong>12</strong>, 647119 (2021).
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.647119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8021716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33833747/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20Y.%20D.%20et%20al.%20Genetic%20evolution%20and%20implications%20of%20the%20mitochondrial%20genomes%20of%20two%20newly%20identified%20Taenia%20spp.%20in%20rodents%20from%20Qinghai-Tibet%20Plateau.%20Front.%20Microbiol.12,%20647119%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Yao, Z., Zhao, W., Tang, B., Li, Q. &amp; Wang, Z. Effects of host identity on the gut microbiota: a comparative study on three microtinae species. <em>Anim. Model Exp. Med</em>. <strong>7</strong>, 98–105 (2024).</cite> [<a href="https://doi.org/10.1002/ame2.12404" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11079152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38567747/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yao,%20Z.,%20Zhao,%20W.,%20Tang,%20B.,%20Li,%20Q.%20&amp;%20Wang,%20Z.%20Effects%20of%20host%20identity%20on%20the%20gut%20microbiota:%20a%20comparative%20study%20on%20three%20microtinae%20species.%20Anim.%20Model%20Exp.%20Med.%207,%2098%E2%80%93105%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Zhu, W. et al. RNA virus diversity in birds and small mammals from Qinghai-Tibet Plateau of China. <em>Front. Microbiol.</em><strong>13</strong>, 780651 (2022).
</cite> [<a href="https://doi.org/10.3389/fmicb.2022.780651" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8894885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35250920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20W.%20et%20al.%20RNA%20virus%20diversity%20in%20birds%20and%20small%20mammals%20from%20Qinghai-Tibet%20Plateau%20of%20China.%20Front.%20Microbiol.13,%20780651%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Su, S. et al. Characterization of H7N2 avian influenza virus in wild birds and pikas in Qinghai-Tibet Plateau area. <em>Sci. Rep.</em><strong>6</strong>, 30974 (2016).
</cite> [<a href="https://doi.org/10.1038/srep30974" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4995509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27553660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su,%20S.%20et%20al.%20Characterization%20of%20H7N2%20avian%20influenza%20virus%20in%20wild%20birds%20and%20pikas%20in%20Qinghai-Tibet%20Plateau%20area.%20Sci.%20Rep.6,%2030974%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Yu, Z. et al. Lowly pathogenic avian influenza (H9N2) infection in plateau pika (Ochotona curzoniae), Qinghai Lake, China. <em>Vet. Microbiol.</em><strong>173</strong>, 132–135 (2014).
</cite> [<a href="https://doi.org/10.1016/j.vetmic.2014.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25069623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20Z.%20et%20al.%20Lowly%20pathogenic%20avian%20influenza%20(H9N2)%20infection%20in%20plateau%20pika%20(Ochotona%20curzoniae),%20Qinghai%20Lake,%20China.%20Vet.%20Microbiol.173,%20132%E2%80%93135%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Zhou, J. et al. Characterization of the H5N1 highly pathogenic avian influenza virus derived from wild pikas in China. <em>J. Virol.</em><strong>83</strong>, 8957–8964 (2009).
</cite> [<a href="https://doi.org/10.1128/JVI.00793-09" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2738197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19553321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20J.%20et%20al.%20Characterization%20of%20the%20H5N1%20highly%20pathogenic%20avian%20influenza%20virus%20derived%20from%20wild%20pikas%20in%20China.%20J.%20Virol.83,%208957%E2%80%938964%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Zhu, W. et al. Discovery and evolution of a divergent coronavirus in the plateau pika from China that extends the host range of alphacoronaviruses. <em>Front. Microbiol.</em><strong>12</strong>, 755599 (2021).
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.755599" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8529330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34691006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20W.%20et%20al.%20Discovery%20and%20evolution%20of%20a%20divergent%20coronavirus%20in%20the%20plateau%20pika%20from%20China%20that%20extends%20the%20host%20range%20of%20alphacoronaviruses.%20Front.%20Microbiol.12,%20755599%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Xu, L. et al. Five-year longitudinal surveillance reveals the continual circulation of both alpha- and beta-coronaviruses in Plateau and Gansu pikas (Ochotona spp.) at Qinghai Lake, China. <em>Emerg. Microbes Infect.</em><strong>13</strong>, 2392693 (2024).
</cite> [<a href="https://doi.org/10.1080/22221751.2024.2392693" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11346322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39137298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20L.%20et%20al.%20Five-year%20longitudinal%20surveillance%20reveals%20the%20continual%20circulation%20of%20both%20alpha-%20and%20beta-coronaviruses%20in%20Plateau%20and%20Gansu%20pikas%20(Ochotona%20spp.)%20at%20Qinghai%20Lake,%20China.%20Emerg.%20Microbes%20Infect.13,%202392693%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Yang, J. et al. Animal activities of the key herbivore plateau pika (Ochotona curzoniae) on the Qinghai-Tibetan plateau affect grassland microbial networks and ecosystem functions. <em>Front. Microbiol.</em><strong>13</strong>, 950811 (2022).
</cite> [<a href="https://doi.org/10.3389/fmicb.2022.950811" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9298508/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35875528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20J.%20et%20al.%20Animal%20activities%20of%20the%20key%20herbivore%20plateau%20pika%20(Ochotona%20curzoniae)%20on%20the%20Qinghai-Tibetan%20plateau%20affect%20grassland%20microbial%20networks%20and%20ecosystem%20functions.%20Front.%20Microbiol.13,%20950811%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Smith, A. T., Badingqiuying, Wilson, M. C. &amp; Hogan, B. W. Functional-trait ecology of the plateau pika Ochotona curzoniae in the Qinghai-Tibetan Plateau ecosystem. <em>Integr. Zool.</em><strong>14</strong>, 87–103 (2019).
</cite> [<a href="https://doi.org/10.1111/1749-4877.12300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29316275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20A.%20T.,%20Badingqiuying,%20Wilson,%20M.%20C.%20&amp;%20Hogan,%20B.%20W.%20Functional-trait%20ecology%20of%20the%20plateau%20pika%20Ochotona%20curzoniae%20in%20the%20Qinghai-Tibetan%20Plateau%20ecosystem.%20Integr.%20Zool.14,%2087%E2%80%93103%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Wylie, K. M., Weinstock, G. M. &amp; Storch, G. A. Emerging view of the human virome. <em>Transl. Res.</em><strong>160</strong>, 283–290 (2012).
</cite> [<a href="https://doi.org/10.1016/j.trsl.2012.03.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3701101/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22683423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wylie,%20K.%20M.,%20Weinstock,%20G.%20M.%20&amp;%20Storch,%20G.%20A.%20Emerging%20view%20of%20the%20human%20virome.%20Transl.%20Res.160,%20283%E2%80%93290%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Hu, J. et al. Characterizing the gut phageome and phage-borne antimicrobial resistance genes in pigs. <em>Microbiome</em><strong>12</strong>, 102 (2024).
</cite> [<a href="https://doi.org/10.1186/s40168-024-01818-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11151549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38840247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu,%20J.%20et%20al.%20Characterizing%20the%20gut%20phageome%20and%20phage-borne%20antimicrobial%20resistance%20genes%20in%20pigs.%20Microbiome12,%20102%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Garmaeva, S. et al. Transmission and dynamics of mother-infant gut viruses during pregnancy and early life. <em>Nat. Commun.</em><strong>15</strong>, 1945 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-45257-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10908809/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38431663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garmaeva,%20S.%20et%20al.%20Transmission%20and%20dynamics%20of%20mother-infant%20gut%20viruses%20during%20pregnancy%20and%20early%20life.%20Nat.%20Commun.15,%201945%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Cao, Z. et al. The gut virome: a new microbiome component in health and disease. <em>EBioMedicine</em><strong>81</strong>, 104113 (2022).
</cite> [<a href="https://doi.org/10.1016/j.ebiom.2022.104113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9240800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35753153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cao,%20Z.%20et%20al.%20The%20gut%20virome:%20a%20new%20microbiome%20component%20in%20health%20and%20disease.%20EBioMedicine81,%20104113%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Gulyaeva, A. et al. Diversity and ecology of caudoviricetes phages with genome terminal repeats in fecal metagenomes from four Dutch cohorts. <em>Viruses</em><strong>14</strong>, 10.3390/v14102305 (2022).</cite> [<a href="https://doi.org/10.3390/v14102305" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9610469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36298860/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Low, S. J., Dzunkova, M., Chaumeil, P. A., Parks, D. H. &amp; Hugenholtz, P. Evaluation of a concatenated protein phylogeny for classification of tailed double-stranded DNA viruses belonging to the order Caudovirales. <em>Nat. Microbiol.</em><strong>4</strong>, 1306–1315 (2019).
</cite> [<a href="https://doi.org/10.1038/s41564-019-0448-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31110365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Low,%20S.%20J.,%20Dzunkova,%20M.,%20Chaumeil,%20P.%20A.,%20Parks,%20D.%20H.%20&amp;%20Hugenholtz,%20P.%20Evaluation%20of%20a%20concatenated%20protein%20phylogeny%20for%20classification%20of%20tailed%20double-stranded%20DNA%20viruses%20belonging%20to%20the%20order%20Caudovirales.%20Nat.%20Microbiol.4,%201306%E2%80%931315%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Nayfach, S. et al. Metagenomic compendium of 189,680 DNA viruses from the human gut microbiome. <em>Nat. Microbiol.</em><strong>6</strong>, 960–970 (2021).
</cite> [<a href="https://doi.org/10.1038/s41564-021-00928-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8241571/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34168315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nayfach,%20S.%20et%20al.%20Metagenomic%20compendium%20of%20189,680%20DNA%20viruses%20from%20the%20human%20gut%20microbiome.%20Nat.%20Microbiol.6,%20960%E2%80%93970%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Wang, H. et al. Gut virome of mammals and birds reveals high genetic diversity of the family Microviridae. <em>Virus Evol.</em><strong>5</strong>, vez013 (2019).
</cite> [<a href="https://doi.org/10.1093/ve/vez013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6555873/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31191981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20H.%20et%20al.%20Gut%20virome%20of%20mammals%20and%20birds%20reveals%20high%20genetic%20diversity%20of%20the%20family%20Microviridae.%20Virus%20Evol.5,%20vez013%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Walters, M. et al. Novel single-stranded DNA virus genomes recovered from chimpanzee feces sampled from the Mambilla Plateau in Nigeria. <em>Genome Announc.</em><strong>5</strong>, 10.1128/genomeA.01715-16 (2017).</cite> [<a href="https://doi.org/10.1128/genomeA.01715-16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5334589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28254982/" class="usa-link">PubMed</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Shkoporov, A. N. et al. The human gut virome is highly diverse, stable, and individual specific. <em>Cell Host Microbe</em><strong>26</strong>, 527–541 e525 (2019).
</cite> [<a href="https://doi.org/10.1016/j.chom.2019.09.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31600503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shkoporov,%20A.%20N.%20et%20al.%20The%20human%20gut%20virome%20is%20highly%20diverse,%20stable,%20and%20individual%20specific.%20Cell%20Host%20Microbe26,%20527%E2%80%93541%20e525%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Kraberger, S., Schmidlin, K., Fontenele, R. S., Walters, M. &amp; Varsani, A. Unravelling the single-stranded DNA virome of the New Zealand Blackfly. <em>Viruses</em><strong>11</strong>, 10.3390/v11060532 (2019).</cite> [<a href="https://doi.org/10.3390/v11060532" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6630596/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31181730/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Tseng, C. H. et al. Microbial and viral metagenomes of a subtropical freshwater reservoir subject to climatic disturbances. <em>ISME J.</em><strong>7</strong>, 2374–2386 (2013).
</cite> [<a href="https://doi.org/10.1038/ismej.2013.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3834851/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23842651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tseng,%20C.%20H.%20et%20al.%20Microbial%20and%20viral%20metagenomes%20of%20a%20subtropical%20freshwater%20reservoir%20subject%20to%20climatic%20disturbances.%20ISME%20J.7,%202374%E2%80%932386%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Cheng, R. et al. Virus diversity and interactions with hosts in deep-sea hydrothermal vents. <em>Microbiome</em><strong>10</strong>, 235 (2022).
</cite> [<a href="https://doi.org/10.1186/s40168-022-01441-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9789665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36566239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20R.%20et%20al.%20Virus%20diversity%20and%20interactions%20with%20hosts%20in%20deep-sea%20hydrothermal%20vents.%20Microbiome10,%20235%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Deng, Z. et al. Phage-prokaryote coexistence strategy mediates microbial community diversity in the intestine and sediment microhabitats of shrimp culture pond ecosystem. <em>Front. Microbiol.</em><strong>13</strong>, 1011342 (2022).
</cite> [<a href="https://doi.org/10.3389/fmicb.2022.1011342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9537357/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36212844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deng,%20Z.%20et%20al.%20Phage-prokaryote%20coexistence%20strategy%20mediates%20microbial%20community%20diversity%20in%20the%20intestine%20and%20sediment%20microhabitats%20of%20shrimp%20culture%20pond%20ecosystem.%20Front.%20Microbiol.13,%201011342%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Mavrich, T. N. &amp; Hatfull, G. F. Bacteriophage evolution differs by host, lifestyle and genome. <em>Nat. Microbiol.</em><strong>2</strong>, 17112 (2017).
</cite> [<a href="https://doi.org/10.1038/nmicrobiol.2017.112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5540316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28692019/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mavrich,%20T.%20N.%20&amp;%20Hatfull,%20G.%20F.%20Bacteriophage%20evolution%20differs%20by%20host,%20lifestyle%20and%20genome.%20Nat.%20Microbiol.2,%2017112%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Di Paola, N. et al. Jingchuvirales: a new taxonomical framework for a rapidly expanding order of unusual monjiviricete viruses broadly distributed among arthropod subphyla. <em>Appl. Environ. Microbiol.</em><strong>88</strong>, e0195421 (2022).
</cite> [<a href="https://doi.org/10.1128/aem.01954-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8939347/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35108077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Di%20Paola,%20N.%20et%20al.%20Jingchuvirales:%20a%20new%20taxonomical%20framework%20for%20a%20rapidly%20expanding%20order%20of%20unusual%20monjiviricete%20viruses%20broadly%20distributed%20among%20arthropod%20subphyla.%20Appl.%20Environ.%20Microbiol.88,%20e0195421%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Desingu, P. A. &amp; Nagarajan, K. Genetic diversity and characterization of circular replication (Rep)-encoding single-stranded (CRESS) DNA viruses. <em>Microbiol. Spectr.</em><strong>10</strong>, e0105722 (2022).
</cite> [<a href="https://doi.org/10.1128/spectrum.01057-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9769708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36346238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Desingu,%20P.%20A.%20&amp;%20Nagarajan,%20K.%20Genetic%20diversity%20and%20characterization%20of%20circular%20replication%20(Rep)-encoding%20single-stranded%20(CRESS)%20DNA%20viruses.%20Microbiol.%20Spectr.10,%20e0105722%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Zhao, L., Rosario, K., Breitbart, M. &amp; Duffy, S. Eukaryotic circular rep-encoding single-stranded DNA (CRESS DNA) viruses: ubiquitous viruses with small genomes and a diverse host range. <em>Adv. Virus Res.</em><strong>103</strong>, 71–133 (2019).
</cite> [<a href="https://doi.org/10.1016/bs.aivir.2018.10.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30635078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20L.,%20Rosario,%20K.,%20Breitbart,%20M.%20&amp;%20Duffy,%20S.%20Eukaryotic%20circular%20rep-encoding%20single-stranded%20DNA%20(CRESS%20DNA)%20viruses:%20ubiquitous%20viruses%20with%20small%20genomes%20and%20a%20diverse%20host%20range.%20Adv.%20Virus%20Res.103,%2071%E2%80%93133%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Zhang, J. T. et al. Decoding the RNA viromes in shrew lungs along the eastern coast of China. <em>NPJ Biofilms Microbiomes</em><strong>10</strong>, 68 (2024).
</cite> [<a href="https://doi.org/10.1038/s41522-024-00543-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11310413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39117662/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20J.%20T.%20et%20al.%20Decoding%20the%20RNA%20viromes%20in%20shrew%20lungs%20along%20the%20eastern%20coast%20of%20China.%20NPJ%20Biofilms%20Microbiomes10,%2068%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Chen, Y. M. et al. Host traits shape virome composition and virus transmission in wild small mammals. <em>Cell</em><strong>186</strong>, 4662–4675.e4612 (2023).
</cite> [<a href="https://doi.org/10.1016/j.cell.2023.08.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37734372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20Y.%20M.%20et%20al.%20Host%20traits%20shape%20virome%20composition%20and%20virus%20transmission%20in%20wild%20small%20mammals.%20Cell186,%204662%E2%80%934675.e4612%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Liu, Q. et al. Association of virome dynamics with mosquito species and environmental factors. <em>Microbiome</em><strong>11</strong>, 101 (2023).
</cite> [<a href="https://doi.org/10.1186/s40168-023-01556-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10165777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37158937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Q.%20et%20al.%20Association%20of%20virome%20dynamics%20with%20mosquito%20species%20and%20environmental%20factors.%20Microbiome11,%20101%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Reuter, G., Boros, A., Kiss, T., Delwart, E. &amp; Pankovics, P. Complete genome characterization of mosavirus (family Picornaviridae) identified in droppings of a European roller (Coracias garrulus) in Hungary. <em>Arch. Virol.</em><strong>159</strong>, 2723–2729 (2014).
</cite> [<a href="https://doi.org/10.1007/s00705-014-2113-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24824348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reuter,%20G.,%20Boros,%20A.,%20Kiss,%20T.,%20Delwart,%20E.%20&amp;%20Pankovics,%20P.%20Complete%20genome%20characterization%20of%20mosavirus%20(family%20Picornaviridae)%20identified%20in%20droppings%20of%20a%20European%20roller%20(Coracias%20garrulus)%20in%20Hungary.%20Arch.%20Virol.159,%202723%E2%80%932729%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Jiang, X. et al. Virome of high-altitude canine digestive tract and genetic characterization of novel viruses potentially threatening human health. <em>mSphere</em><strong>8</strong>, e0034523 (2023).
</cite> [<a href="https://doi.org/10.1128/msphere.00345-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10597464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37724888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang,%20X.%20et%20al.%20Virome%20of%20high-altitude%20canine%20digestive%20tract%20and%20genetic%20characterization%20of%20novel%20viruses%20potentially%20threatening%20human%20health.%20mSphere8,%20e0034523%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Mills, J. N. Regulation of rodent-borne viruses in the natural host: implications for human disease. <em>Arch. Virol. Suppl</em>. <strong>19</strong>, 45–57 (2005).</cite> [<a href="https://doi.org/10.1007/3-211-29981-5_5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16355867/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Rabiee, M. H., Mahmoudi, A., Siahsarvie, R., Krystufek, B. &amp; Mostafavi, E. Rodent-borne diseases and their public health importance in Iran. <em>PLoS Negl. Trop. Dis.</em><strong>12</strong>, e0006256 (2018).
</cite> [<a href="https://doi.org/10.1371/journal.pntd.0006256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5908068/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29672510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rabiee,%20M.%20H.,%20Mahmoudi,%20A.,%20Siahsarvie,%20R.,%20Krystufek,%20B.%20&amp;%20Mostafavi,%20E.%20Rodent-borne%20diseases%20and%20their%20public%20health%20importance%20in%20Iran.%20PLoS%20Negl.%20Trop.%20Dis.12,%20e0006256%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Blanga-Kanfi, S. et al. Rodent phylogeny revised: analysis of six nuclear genes from all major rodent clades. <em>BMC Evol. Biol.</em><strong>9</strong>, 71 (2009).
</cite> [<a href="https://doi.org/10.1186/1471-2148-9-71" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2674048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19341461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blanga-Kanfi,%20S.%20et%20al.%20Rodent%20phylogeny%20revised:%20analysis%20of%20six%20nuclear%20genes%20from%20all%20major%20rodent%20clades.%20BMC%20Evol.%20Biol.9,%2071%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Meerburg, B. G., Singleton, G. R. &amp; Kijlstra, A. Rodent-borne diseases and their risks for public health. <em>Crit. Rev. Microbiol</em><strong>35</strong>, 221–270 (2009).
</cite> [<a href="https://doi.org/10.1080/10408410902989837" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19548807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meerburg,%20B.%20G.,%20Singleton,%20G.%20R.%20&amp;%20Kijlstra,%20A.%20Rodent-borne%20diseases%20and%20their%20risks%20for%20public%20health.%20Crit.%20Rev.%20Microbiol35,%20221%E2%80%93270%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Taylor, L. J., Keeler, E. L., Bushman, F. D. &amp; Collman, R. G. The enigmatic roles of Anelloviridae and Redondoviridae in humans. <em>Curr. Opin. Virol.</em><strong>55</strong>, 101248 (2022).
</cite> [<a href="https://doi.org/10.1016/j.coviro.2022.101248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35870315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Taylor,%20L.%20J.,%20Keeler,%20E.%20L.,%20Bushman,%20F.%20D.%20&amp;%20Collman,%20R.%20G.%20The%20enigmatic%20roles%20of%20Anelloviridae%20and%20Redondoviridae%20in%20humans.%20Curr.%20Opin.%20Virol.55,%20101248%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Camargo, A. P. et al. IMG/VR v4: an expanded database of uncultivated virus genomes within a framework of extensive functional, taxonomic, and ecological metadata. <em>Nucleic Acids Res.</em><strong>51</strong>, D733–D743 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkac1037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36399502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Camargo,%20A.%20P.%20et%20al.%20IMG/VR%20v4:%20an%20expanded%20database%20of%20uncultivated%20virus%20genomes%20within%20a%20framework%20of%20extensive%20functional,%20taxonomic,%20and%20ecological%20metadata.%20Nucleic%20Acids%20Res.51,%20D733%E2%80%93D743%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Lu, J. et al. Gut Virome of the world’s highest-elevation lizard species (Phrynocephalus erythrurus and Phrynocephalus theobaldi) reveals versatile commensal viruses. <em>Microbiol. Spectr.</em><strong>10</strong>, e0187221 (2022).
</cite> [<a href="https://doi.org/10.1128/spectrum.01872-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8865479/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35196818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20J.%20et%20al.%20Gut%20Virome%20of%20the%20world%E2%80%99s%20highest-elevation%20lizard%20species%20(Phrynocephalus%20erythrurus%20and%20Phrynocephalus%20theobaldi)%20reveals%20versatile%20commensal%20viruses.%20Microbiol.%20Spectr.10,%20e0187221%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Zhang, W. et al. Faecal virome of cats in an animal shelter. <em>J. Gen. Virol.</em><strong>95</strong>, 2553–2564 (2014).
</cite> [<a href="https://doi.org/10.1099/vir.0.069674-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4202271/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25078300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20W.%20et%20al.%20Faecal%20virome%20of%20cats%20in%20an%20animal%20shelter.%20J.%20Gen.%20Virol.95,%202553%E2%80%932564%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Li, D. et al. MEGAHIT v1.0: A fast and scalable metagenome assembler driven by advanced methodologies and community practices. <em>Methods</em><strong>102</strong>, 3–11 (2016).
</cite> [<a href="https://doi.org/10.1016/j.ymeth.2016.02.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27012178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20D.%20et%20al.%20MEGAHIT%20v1.0:%20A%20fast%20and%20scalable%20metagenome%20assembler%20driven%20by%20advanced%20methodologies%20and%20community%20practices.%20Methods102,%203%E2%80%9311%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Mirdita, M., Steinegger, M. &amp; Soding, J. MMseqs2 desktop and local web server app for fast, interactive sequence searches. <em>Bioinformatics</em><strong>35</strong>, 2856–2858 (2019).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty1057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6691333/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30615063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mirdita,%20M.,%20Steinegger,%20M.%20&amp;%20Soding,%20J.%20MMseqs2%20desktop%20and%20local%20web%20server%20app%20for%20fast,%20interactive%20sequence%20searches.%20Bioinformatics35,%202856%E2%80%932858%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Guo, J. et al. VirSorter2: a multi-classifier, expert-guided approach to detect diverse DNA and RNA viruses. <em>Microbiome</em><strong>9</strong>, 37 (2021).
</cite> [<a href="https://doi.org/10.1186/s40168-020-00990-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7852108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33522966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20J.%20et%20al.%20VirSorter2:%20a%20multi-classifier,%20expert-guided%20approach%20to%20detect%20diverse%20DNA%20and%20RNA%20viruses.%20Microbiome9,%2037%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Nayfach, S. et al. CheckV assesses the quality and completeness of metagenome-assembled viral genomes. <em>Nat. Biotechnol.</em><strong>39</strong>, 578–585 (2021).
</cite> [<a href="https://doi.org/10.1038/s41587-020-00774-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8116208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33349699/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nayfach,%20S.%20et%20al.%20CheckV%20assesses%20the%20quality%20and%20completeness%20of%20metagenome-assembled%20viral%20genomes.%20Nat.%20Biotechnol.39,%20578%E2%80%93585%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Kieft, K., Zhou, Z. &amp; Anantharaman, K. VIBRANT: automated recovery, annotation and curation of microbial viruses, and evaluation of viral community function from genomic sequences. <em>Microbiome</em><strong>8</strong>, 90 (2020).
</cite> [<a href="https://doi.org/10.1186/s40168-020-00867-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7288430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32522236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kieft,%20K.,%20Zhou,%20Z.%20&amp;%20Anantharaman,%20K.%20VIBRANT:%20automated%20recovery,%20annotation%20and%20curation%20of%20microbial%20viruses,%20and%20evaluation%20of%20viral%20community%20function%20from%20genomic%20sequences.%20Microbiome8,%2090%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Roux, S. et al. Minimum information about an uncultivated virus genome (MIUViG). <em>Nat. Biotechnol.</em><strong>37</strong>, 29–37 (2019).
</cite> [<a href="https://doi.org/10.1038/nbt.4306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6871006/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30556814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roux,%20S.%20et%20al.%20Minimum%20information%20about%20an%20uncultivated%20virus%20genome%20(MIUViG).%20Nat.%20Biotechnol.37,%2029%E2%80%9337%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Buchfink, B., Reuter, K. &amp; Drost, H. G. Sensitive protein alignments at tree-of-life scale using DIAMOND. <em>Nat. Methods</em><strong>18</strong>, 366–368 (2021).
</cite> [<a href="https://doi.org/10.1038/s41592-021-01101-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8026399/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33828273/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buchfink,%20B.,%20Reuter,%20K.%20&amp;%20Drost,%20H.%20G.%20Sensitive%20protein%20alignments%20at%20tree-of-life%20scale%20using%20DIAMOND.%20Nat.%20Methods18,%20366%E2%80%93368%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Shen, W. &amp; Ren, H. TaxonKit: a practical and efficient NCBI taxonomy toolkit. <em>J. Genet Genom.</em><strong>48</strong>, 844–850 (2021).</cite> [<a href="https://doi.org/10.1016/j.jgg.2021.03.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34001434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen,%20W.%20&amp;%20Ren,%20H.%20TaxonKit:%20a%20practical%20and%20efficient%20NCBI%20taxonomy%20toolkit.%20J.%20Genet%20Genom.48,%20844%E2%80%93850%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Gautam, A., Felderhoff, H., Bagci, C. &amp; Huson, D. H. Using AnnoTree to get more assignments, faster, in DIAMOND+MEGAN microbiome analysis. <em>mSystems</em><strong>7</strong>, e0140821 (2022).
</cite> [<a href="https://doi.org/10.1128/msystems.01408-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8862659/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35191776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gautam,%20A.,%20Felderhoff,%20H.,%20Bagci,%20C.%20&amp;%20Huson,%20D.%20H.%20Using%20AnnoTree%20to%20get%20more%20assignments,%20faster,%20in%20DIAMOND+MEGAN%20microbiome%20analysis.%20mSystems7,%20e0140821%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Shang, J., Jiang, J. &amp; Sun, Y. Bacteriophage classification for assembled contigs using graph convolutional network. <em>Bioinformatics</em><strong>37</strong>, i25–i33 (2021).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btab293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8275337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34252923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shang,%20J.,%20Jiang,%20J.%20&amp;%20Sun,%20Y.%20Bacteriophage%20classification%20for%20assembled%20contigs%20using%20graph%20convolutional%20network.%20Bioinformatics37,%20i25%E2%80%93i33%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Hyatt, D. et al. Prodigal: prokaryotic gene recognition and translation initiation site identification. <em>BMC Bioinforma.</em><strong>11</strong>, 119 (2010).</cite> [<a href="https://doi.org/10.1186/1471-2105-11-119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2848648/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20211023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hyatt,%20D.%20et%20al.%20Prodigal:%20prokaryotic%20gene%20recognition%20and%20translation%20initiation%20site%20identification.%20BMC%20Bioinforma.11,%20119%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Wang, J. et al. The conserved domain database in 2023. <em>Nucleic Acids Res.</em><strong>51</strong>, D384–D388 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkac1096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825596/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36477806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20J.%20et%20al.%20The%20conserved%20domain%20database%20in%202023.%20Nucleic%20Acids%20Res.51,%20D384%E2%80%93D388%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Huerta-Cepas, J. et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. <em>Nucleic Acids Res.</em><strong>47</strong>, D309–D314 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gky1085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6324079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30418610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huerta-Cepas,%20J.%20et%20al.%20eggNOG%205.0:%20a%20hierarchical,%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses.%20Nucleic%20Acids%20Res.47,%20D309%E2%80%93D314%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Cantalapiedra, C. P., Hernandez-Plaza, A., Letunic, I., Bork, P. &amp; Huerta-Cepas, J. eggNOG-mapper v2: functional annotation, orthology assignments, and domain prediction at the metagenomic scale. <em>Mol. Biol. Evol.</em><strong>38</strong>, 5825–5829 (2021).
</cite> [<a href="https://doi.org/10.1093/molbev/msab293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8662613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34597405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cantalapiedra,%20C.%20P.,%20Hernandez-Plaza,%20A.,%20Letunic,%20I.,%20Bork,%20P.%20&amp;%20Huerta-Cepas,%20J.%20eggNOG-mapper%20v2:%20functional%20annotation,%20orthology%20assignments,%20and%20domain%20prediction%20at%20the%20metagenomic%20scale.%20Mol.%20Biol.%20Evol.38,%205825%E2%80%935829%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Shang, J., Peng, C., Liao, H., Tang, X. &amp; Sun, Y. PhaBOX: a web server for identifying and characterizing phage contigs in metagenomic data. <em>Bioinform Adv.</em><strong>3</strong>, vbad101 (2023).
</cite> [<a href="https://doi.org/10.1093/bioadv/vbad101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10460485/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37641717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shang,%20J.,%20Peng,%20C.,%20Liao,%20H.,%20Tang,%20X.%20&amp;%20Sun,%20Y.%20PhaBOX:%20a%20web%20server%20for%20identifying%20and%20characterizing%20phage%20contigs%20in%20metagenomic%20data.%20Bioinform%20Adv.3,%20vbad101%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Yan, M. et al. Interrogating the viral dark matter of the rumen ecosystem with a global virome database. <em>Nat. Commun.</em><strong>14</strong>, 5254 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-41075-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10465536/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37644066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan,%20M.%20et%20al.%20Interrogating%20the%20viral%20dark%20matter%20of%20the%20rumen%20ecosystem%20with%20a%20global%20virome%20database.%20Nat.%20Commun.14,%205254%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Kumar, S., Stecher, G., Li, M., Knyaz, C. &amp; Tamura, K. MEGA X: molecular evolutionary genetics analysis across computing platforms. <em>Mol. Biol. Evol.</em><strong>35</strong>, 1547–1549 (2018).
</cite> [<a href="https://doi.org/10.1093/molbev/msy096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5967553/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29722887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20S.,%20Stecher,%20G.,%20Li,%20M.,%20Knyaz,%20C.%20&amp;%20Tamura,%20K.%20MEGA%20X:%20molecular%20evolutionary%20genetics%20analysis%20across%20computing%20platforms.%20Mol.%20Biol.%20Evol.35,%201547%E2%80%931549%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Capella-Gutierrez, S., Silla-Martinez, J. M. &amp; Gabaldon, T. trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses. <em>Bioinformatics</em><strong>25</strong>, 1972–1973 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2712344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Capella-Gutierrez,%20S.,%20Silla-Martinez,%20J.%20M.%20&amp;%20Gabaldon,%20T.%20trimAl:%20a%20tool%20for%20automated%20alignment%20trimming%20in%20large-scale%20phylogenetic%20analyses.%20Bioinformatics25,%201972%E2%80%931973%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Minh, B. Q. et al. IQ-TREE 2: new models and efficient methods for phylogenetic inference in the genomic era. <em>Mol. Biol. Evol.</em><strong>37</strong>, 1530–1534 (2020).
</cite> [<a href="https://doi.org/10.1093/molbev/msaa015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7182206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32011700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Minh,%20B.%20Q.%20et%20al.%20IQ-TREE%202:%20new%20models%20and%20efficient%20methods%20for%20phylogenetic%20inference%20in%20the%20genomic%20era.%20Mol.%20Biol.%20Evol.37,%201530%E2%80%931534%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Letunic, I. &amp; Bork, P. Interactive Tree of Life (iTOL) v6: recent updates to the phylogenetic tree display and annotation tool. <em>Nucleic Acids Res.</em>10.1093/nar/gkae268 (2024).</cite> [<a href="https://doi.org/10.1093/nar/gkae268" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11223838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38613393/" class="usa-link">PubMed</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Oberg, N., Zallot, R. &amp; Gerlt, J. A. EFI-EST, EFI-GNT, and EFI-CGFP: enzyme function initiative (EFI) web resource for genomic enzymology tools. <em>J. Mol. Biol.</em><strong>435</strong>, 168018 (2023).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2023.168018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10291204/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37356897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oberg,%20N.,%20Zallot,%20R.%20&amp;%20Gerlt,%20J.%20A.%20EFI-EST,%20EFI-GNT,%20and%20EFI-CGFP:%20enzyme%20function%20initiative%20(EFI)%20web%20resource%20for%20genomic%20enzymology%20tools.%20J.%20Mol.%20Biol.435,%20168018%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Shannon, P. et al. Cytoscape: a software environment for integrated models of biomolecular interaction networks. <em>Genome Res.</em><strong>13</strong>, 2498–2504 (2003).
</cite> [<a href="https://doi.org/10.1101/gr.1239303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC403769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shannon,%20P.%20et%20al.%20Cytoscape:%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks.%20Genome%20Res.13,%202498%E2%80%932504%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Trgovec-Greif, L. et al. VOGDB-database of virus orthologous groups. <em>Viruses</em><strong>16</strong>, 10.3390/v16081191 (2024).</cite> [<a href="https://doi.org/10.3390/v16081191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11360334/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39205165/" class="usa-link">PubMed</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Eddy, S. R. Accelerated profile HMM searches. <em>PLoS Comput. Biol.</em><strong>7</strong>, e1002195 (2011).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1002195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3197634/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22039361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eddy,%20S.%20R.%20Accelerated%20profile%20HMM%20searches.%20PLoS%20Comput.%20Biol.7,%20e1002195%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Kuraku, S., Zmasek, C. M., Nishimura, O. &amp; Katoh, K. aLeaves facilitates on-demand exploration of metazoan gene family trees on MAFFT sequence alignment server with enhanced interactivity. <em>Nucleic Acids Res.</em><strong>41</strong>, W22–W28 (2013).
</cite> [<a href="https://doi.org/10.1093/nar/gkt389" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3692103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23677614/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuraku,%20S.,%20Zmasek,%20C.%20M.,%20Nishimura,%20O.%20&amp;%20Katoh,%20K.%20aLeaves%20facilitates%20on-demand%20exploration%20of%20metazoan%20gene%20family%20trees%20on%20MAFFT%20sequence%20alignment%20server%20with%20enhanced%20interactivity.%20Nucleic%20Acids%20Res.41,%20W22%E2%80%93W28%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Price, M. N., Dehal, P. S. &amp; Arkin, A. P. FastTree 2-approximately maximum-likelihood trees for large alignments. <em>PLoS One</em><strong>5</strong>, e9490 (2010).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0009490" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2835736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20224823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Price,%20M.%20N.,%20Dehal,%20P.%20S.%20&amp;%20Arkin,%20A.%20P.%20FastTree%202-approximately%20maximum-likelihood%20trees%20for%20large%20alignments.%20PLoS%20One5,%20e9490%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371082/bin/41522_2025_814_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (11.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371082/bin/41522_2025_814_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary data1</a><sup> (791.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371082/bin/41522_2025_814_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary data2</a><sup> (5MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371082/bin/41522_2025_814_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary data3</a><sup> (111.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371082/bin/41522_2025_814_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary data4</a><sup> (143.5KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The raw sequence data reported in this paper have been deposited in the Genome Sequence Archive (GSA) at the National Genomics Data Center (NGDC), China National Center for Bioinformation, Beijing Institute of Genomics, Chinese Academy of Sciences (GSA: CRA015789-CRA015791), and are publicly accessible at <a href="https://ngdc.cncb.ac.cn/gsa" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa</a>. The novel sequences reported in this paper have also been deposited in GenBase at NGDC and are freely accessible at <a href="https://ngdc.cncb.ac.cn/genbase" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/genbase</a> without any access restrictions. Detailed accession numbers can be found in Supplementary Data 1. Additionally, phages reported in this study have been archived in the Zenodo repository and can be accessed at 10.5281/zenodo.14992764.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from NPJ Biofilms and Microbiomes are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41522-025-00814-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41522_2025_Article_814.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371082/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371082/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371082%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371082/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371082/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371082/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841544/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371082/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841544/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371082/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371082/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="LzBYls8XWFIGSDMgWX0U7vFofMGjZMjz2dQo5cCky2qoQxUSFGtd2ZEvMNGxyiPa">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
