
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Multi-omic identification of perineurial hyperplasia and lipid-associated nerve macrophages in human polyneuropathies - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E486248AEF44B3058624001E68CA9A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375038/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Multi-omic identification of perineurial hyperplasia and lipid-associated nerve macrophages in human polyneuropathies">
<meta name="citation_author" content="Michael Heming">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Münster, Münster, Germany">
<meta name="citation_author" content="Anna-Lena Börsch">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Münster, Münster, Germany">
<meta name="citation_author" content="Jolien Wolbert">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Münster, Münster, Germany">
<meta name="citation_author" content="Christian Thomas">
<meta name="citation_author_institution" content="Institute of Neuropathology, University Hospital Münster, Münster, Germany">
<meta name="citation_author" content="Anne K Mausberg">
<meta name="citation_author_institution" content="Department of Neurology, Center for Translational Neuro- and Behavioural Sciences (C-TNBS), University Hospital Essen, Essen, Germany">
<meta name="citation_author" content="Fabian Szepanowski">
<meta name="citation_author_institution" content="Department of Neurology, Center for Translational Neuro- and Behavioural Sciences (C-TNBS), University Hospital Essen, Essen, Germany">
<meta name="citation_author" content="Bianca Eggert">
<meta name="citation_author_institution" content="Department of Neurology, Center for Translational Neuro- and Behavioural Sciences (C-TNBS), University Hospital Essen, Essen, Germany">
<meta name="citation_author" content="I-Na Lu">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Münster, Münster, Germany">
<meta name="citation_author" content="Julia Tietz">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Münster, Münster, Germany">
<meta name="citation_author" content="Finja Dienhart">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Münster, Münster, Germany">
<meta name="citation_author" content="Maja Meschnark">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Münster, Münster, Germany">
<meta name="citation_author" content="Jan-Kolja Strecker">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Schleswig-Holstein, Lübeck, Germany">
<meta name="citation_author" content="Michael Glatza">
<meta name="citation_author_institution" content="Institute of Neuropathology, University Hospital Münster, Münster, Germany">
<meta name="citation_author" content="Carolina Thomas">
<meta name="citation_author_institution" content="Paul-Flechsig-Institute of Neuropathology, University Hospital Leipzig, Leipzig, Germany">
<meta name="citation_author" content="Noemi Gmahl">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Münster, Münster, Germany">
<meta name="citation_author" content="Christine Dambietz">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Münster, Münster, Germany">
<meta name="citation_author" content="Michael Müther">
<meta name="citation_author_institution" content="Department of Neurosurgery, University Hospital Münster, Münster, Germany">
<meta name="citation_author" content="Anne-Kathrin Uerschels">
<meta name="citation_author_institution" content="Department of Neurosurgery, University Hospital Essen, Essen, Germany">
<meta name="citation_author" content="Kathy Keyvani">
<meta name="citation_author_institution" content="Department of Neuropathology, University Hospital Essen, Essen, Germany">
<meta name="citation_author" content="Jens Minnerup">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Schleswig-Holstein, Lübeck, Germany">
<meta name="citation_author" content="Kathrin Doppler">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Würzburg, Würzburg, Germany">
<meta name="citation_author" content="Nurcan Üçeyler">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Würzburg, Würzburg, Germany">
<meta name="citation_author" content="Julieta Aprea">
<meta name="citation_author_institution" content="DRESDEN-concept Genome Center, Center for Molecular and Cellular Bioengineering, Dresden University of Technology, Dresden, Germany">
<meta name="citation_author" content="Andreas Dahl">
<meta name="citation_author_institution" content="DRESDEN-concept Genome Center, Center for Molecular and Cellular Bioengineering, Dresden University of Technology, Dresden, Germany">
<meta name="citation_author" content="Ruth Stassart">
<meta name="citation_author_institution" content="Paul-Flechsig-Institute of Neuropathology, University Hospital Leipzig, Leipzig, Germany">
<meta name="citation_author" content="Robert Fledrich">
<meta name="citation_author_institution" content="Institute of Anatomy, University of Leipzig, Leipzig, Germany">
<meta name="citation_author" content="Heinz Wiendl">
<meta name="citation_author_institution" content="Clinic of Neurology and Neurophysiology, University of Freiburg, Freiburg, Germany">
<meta name="citation_author" content="Claudia Sommer">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Würzburg, Würzburg, Germany">
<meta name="citation_author" content="Mark Stettner">
<meta name="citation_author_institution" content="Department of Neurology, Center for Translational Neuro- and Behavioural Sciences (C-TNBS), University Hospital Essen, Essen, Germany">
<meta name="citation_author" content="Gerd Meyer zu Hörste">
<meta name="citation_author_institution" content="Department of Neurology, University Hospital Münster, Münster, Germany">
<meta name="citation_publication_date" content="2025 Aug 23">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7872">
<meta name="citation_doi" content="10.1038/s41467-025-62964-8">
<meta name="citation_pmid" content="40849297">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375038/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375038/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375038/pdf/41467_2025_Article_62964.pdf">
<meta name="description" content="Diseases affecting multiple peripheral nerves, termed polyneuropathies (PNPs), are common, mechanistically heterogeneous, and their causes are challenging to identify. Here, we integrated single-nucleus transcriptomics of peripheral nerves from 33 ...">
<meta name="og:title" content="Multi-omic identification of perineurial hyperplasia and lipid-associated nerve macrophages in human polyneuropathies">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Diseases affecting multiple peripheral nerves, termed polyneuropathies (PNPs), are common, mechanistically heterogeneous, and their causes are challenging to identify. Here, we integrated single-nucleus transcriptomics of peripheral nerves from 33 ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375038/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375038">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62964-8"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62964.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375038%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375038/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375038/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375038/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 23;16:7872. doi: <a href="https://doi.org/10.1038/s41467-025-62964-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62964-8</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Multi-omic identification of perineurial hyperplasia and lipid-associated nerve macrophages in human polyneuropathies</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heming%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Michael Heming</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Michael Heming</span></h3>
<div class="p">
<sup>1</sup>Department of Neurology, University Hospital Münster, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heming%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael Heming</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22B%C3%B6rsch%20AL%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Anna-Lena Börsch</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Anna-Lena Börsch</span></h3>
<div class="p">
<sup>1</sup>Department of Neurology, University Hospital Münster, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22B%C3%B6rsch%20AL%22%5BAuthor%5D" class="usa-link"><span class="name western">Anna-Lena Börsch</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolbert%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jolien Wolbert</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jolien Wolbert</span></h3>
<div class="p">
<sup>1</sup>Department of Neurology, University Hospital Münster, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolbert%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jolien Wolbert</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thomas%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Christian Thomas</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Christian Thomas</span></h3>
<div class="p">
<sup>2</sup>Institute of Neuropathology, University Hospital Münster, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thomas%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christian Thomas</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mausberg%20AK%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Anne K Mausberg</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Anne K Mausberg</span></h3>
<div class="p">
<sup>3</sup>Department of Neurology, Center for Translational Neuro- and Behavioural Sciences (C-TNBS), University Hospital Essen, Essen, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mausberg%20AK%22%5BAuthor%5D" class="usa-link"><span class="name western">Anne K Mausberg</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Szepanowski%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Fabian Szepanowski</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Fabian Szepanowski</span></h3>
<div class="p">
<sup>3</sup>Department of Neurology, Center for Translational Neuro- and Behavioural Sciences (C-TNBS), University Hospital Essen, Essen, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Szepanowski%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fabian Szepanowski</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Eggert%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Bianca Eggert</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Bianca Eggert</span></h3>
<div class="p">
<sup>3</sup>Department of Neurology, Center for Translational Neuro- and Behavioural Sciences (C-TNBS), University Hospital Essen, Essen, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Eggert%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bianca Eggert</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20IN%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">I-Na Lu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">I-Na Lu</span></h3>
<div class="p">
<sup>1</sup>Department of Neurology, University Hospital Münster, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20IN%22%5BAuthor%5D" class="usa-link"><span class="name western">I-Na Lu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tietz%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Julia Tietz</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Julia Tietz</span></h3>
<div class="p">
<sup>1</sup>Department of Neurology, University Hospital Münster, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tietz%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Julia Tietz</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dienhart%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Finja Dienhart</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Finja Dienhart</span></h3>
<div class="p">
<sup>1</sup>Department of Neurology, University Hospital Münster, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dienhart%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Finja Dienhart</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meschnark%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Maja Meschnark</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Maja Meschnark</span></h3>
<div class="p">
<sup>1</sup>Department of Neurology, University Hospital Münster, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meschnark%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maja Meschnark</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Strecker%20JK%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Jan-Kolja Strecker</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Jan-Kolja Strecker</span></h3>
<div class="p">
<sup>4</sup>Department of Neurology, University Hospital Schleswig-Holstein, Lübeck, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Strecker%20JK%22%5BAuthor%5D" class="usa-link"><span class="name western">Jan-Kolja Strecker</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Glatza%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Michael Glatza</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Michael Glatza</span></h3>
<div class="p">
<sup>2</sup>Institute of Neuropathology, University Hospital Münster, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Glatza%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael Glatza</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thomas%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Carolina Thomas</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Carolina Thomas</span></h3>
<div class="p">
<sup>5</sup>Paul-Flechsig-Institute of Neuropathology, University Hospital Leipzig, Leipzig, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thomas%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Carolina Thomas</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gmahl%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Noemi Gmahl</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Noemi Gmahl</span></h3>
<div class="p">
<sup>1</sup>Department of Neurology, University Hospital Münster, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gmahl%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Noemi Gmahl</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dambietz%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Christine Dambietz</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Christine Dambietz</span></h3>
<div class="p">
<sup>1</sup>Department of Neurology, University Hospital Münster, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dambietz%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christine Dambietz</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22M%C3%BCther%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Michael Müther</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Michael Müther</span></h3>
<div class="p">
<sup>6</sup>Department of Neurosurgery, University Hospital Münster, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22M%C3%BCther%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael Müther</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Uerschels%20AK%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Anne-Kathrin Uerschels</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Anne-Kathrin Uerschels</span></h3>
<div class="p">
<sup>7</sup>Department of Neurosurgery, University Hospital Essen, Essen, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Uerschels%20AK%22%5BAuthor%5D" class="usa-link"><span class="name western">Anne-Kathrin Uerschels</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Keyvani%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Kathy Keyvani</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Kathy Keyvani</span></h3>
<div class="p">
<sup>8</sup>Department of Neuropathology, University Hospital Essen, Essen, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Keyvani%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kathy Keyvani</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Minnerup%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Jens Minnerup</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Jens Minnerup</span></h3>
<div class="p">
<sup>4</sup>Department of Neurology, University Hospital Schleswig-Holstein, Lübeck, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Minnerup%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jens Minnerup</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doppler%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Kathrin Doppler</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Kathrin Doppler</span></h3>
<div class="p">
<sup>9</sup>Department of Neurology, University Hospital Würzburg, Würzburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doppler%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kathrin Doppler</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22%C3%9C%C3%A7eyler%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id22"><span class="name western">Nurcan Üçeyler</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Nurcan Üçeyler</span></h3>
<div class="p">
<sup>9</sup>Department of Neurology, University Hospital Würzburg, Würzburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22%C3%9C%C3%A7eyler%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nurcan Üçeyler</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aprea%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id23"><span class="name western">Julieta Aprea</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Julieta Aprea</span></h3>
<div class="p">
<sup>10</sup>DRESDEN-concept Genome Center, Center for Molecular and Cellular Bioengineering, Dresden University of Technology, Dresden, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aprea%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Julieta Aprea</span></a>
</div>
</div>
<sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dahl%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id24"><span class="name western">Andreas Dahl</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Andreas Dahl</span></h3>
<div class="p">
<sup>10</sup>DRESDEN-concept Genome Center, Center for Molecular and Cellular Bioengineering, Dresden University of Technology, Dresden, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dahl%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andreas Dahl</span></a>
</div>
</div>
<sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stassart%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id25"><span class="name western">Ruth Stassart</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Ruth Stassart</span></h3>
<div class="p">
<sup>5</sup>Paul-Flechsig-Institute of Neuropathology, University Hospital Leipzig, Leipzig, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stassart%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ruth Stassart</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fledrich%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id26"><span class="name western">Robert Fledrich</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">Robert Fledrich</span></h3>
<div class="p">
<sup>11</sup>Institute of Anatomy, University of Leipzig, Leipzig, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fledrich%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Robert Fledrich</span></a>
</div>
</div>
<sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wiendl%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id27"><span class="name western">Heinz Wiendl</span></a><div hidden="hidden" id="id27">
<h3><span class="name western">Heinz Wiendl</span></h3>
<div class="p">
<sup>12</sup>Clinic of Neurology and Neurophysiology, University of Freiburg, Freiburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wiendl%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Heinz Wiendl</span></a>
</div>
</div>
<sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sommer%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id28"><span class="name western">Claudia Sommer</span></a><div hidden="hidden" id="id28">
<h3><span class="name western">Claudia Sommer</span></h3>
<div class="p">
<sup>9</sup>Department of Neurology, University Hospital Würzburg, Würzburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sommer%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Claudia Sommer</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stettner%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id29"><span class="name western">Mark Stettner</span></a><div hidden="hidden" id="id29">
<h3><span class="name western">Mark Stettner</span></h3>
<div class="p">
<sup>3</sup>Department of Neurology, Center for Translational Neuro- and Behavioural Sciences (C-TNBS), University Hospital Essen, Essen, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stettner%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mark Stettner</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meyer%20zu%20H%C3%B6rste%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id30"><span class="name western">Gerd Meyer zu Hörste</span></a><div hidden="hidden" id="id30">
<h3><span class="name western">Gerd Meyer zu Hörste</span></h3>
<div class="p">
<sup>1</sup>Department of Neurology, University Hospital Münster, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meyer%20zu%20H%C3%B6rste%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gerd Meyer zu Hörste</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Neurology, University Hospital Münster, Münster, Germany </div>
<div id="Aff2">
<sup>2</sup>Institute of Neuropathology, University Hospital Münster, Münster, Germany </div>
<div id="Aff3">
<sup>3</sup>Department of Neurology, Center for Translational Neuro- and Behavioural Sciences (C-TNBS), University Hospital Essen, Essen, Germany </div>
<div id="Aff4">
<sup>4</sup>Department of Neurology, University Hospital Schleswig-Holstein, Lübeck, Germany </div>
<div id="Aff5">
<sup>5</sup>Paul-Flechsig-Institute of Neuropathology, University Hospital Leipzig, Leipzig, Germany </div>
<div id="Aff6">
<sup>6</sup>Department of Neurosurgery, University Hospital Münster, Münster, Germany </div>
<div id="Aff7">
<sup>7</sup>Department of Neurosurgery, University Hospital Essen, Essen, Germany </div>
<div id="Aff8">
<sup>8</sup>Department of Neuropathology, University Hospital Essen, Essen, Germany </div>
<div id="Aff9">
<sup>9</sup>Department of Neurology, University Hospital Würzburg, Würzburg, Germany </div>
<div id="Aff10">
<sup>10</sup>DRESDEN-concept Genome Center, Center for Molecular and Cellular Bioengineering, Dresden University of Technology, Dresden, Germany </div>
<div id="Aff11">
<sup>11</sup>Institute of Anatomy, University of Leipzig, Leipzig, Germany </div>
<div id="Aff12">
<sup>12</sup>Clinic of Neurology and Neurophysiology, University of Freiburg, Freiburg, Germany </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Aug 5; Accepted 2025 Aug 6; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375038  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40849297/" class="usa-link">40849297</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Diseases affecting multiple peripheral nerves, termed polyneuropathies (PNPs), are common, mechanistically heterogeneous, and their causes are challenging to identify. Here, we integrated single-nucleus transcriptomics of peripheral nerves from 33 human PNP patients and four controls (365,708 nuclei) with subcellular spatial transcriptomics. We identified nerve cell type markers and uncovered unexpected heterogeneity of perineurial cells. PNPs shared a loss of myelinating and an increase in repair Schwann cells and endoneurial lipid-phagocytizing macrophages. Transcriptional changes affected multiple cells outside of the endoneurium across PNPs, suggesting PNPs as ‘pan-nerve diseases’. Spatially, PNPs—particularly those mediated by autoimmunity—exhibited focal perineurial hyperplasia and increased expression of <em>CXCL14</em>, identified as perineurial cell marker. Multi-omic characterization of human nerve biopsies thus identified novel mechanisms in PNPs with diagnostic potential.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Peripheral neuropathies, Somatic system</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Polyneuropathy causes are difficult to pinpoint. Here, authors used single-nucleus and spatial transcriptomics of 37 human peripheral nerves to identify focal perineurial hyperplasia and endoneurial lipid-associated macrophages in polyneuropathies.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Polyneuropathies (PNPs) denote diseases of multiple peripheral nerves and range among the most common neurological diseases<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Patients affected by PNPs often suffer from progressive sensory and motor impairment with high socio-economic impact<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. The underlying causes of PNPs are very diverse<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Despite a comprehensive diagnostic work-up, the etiology of PNPs often remains unclear (up to 20–30%)<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. A biopsy of the sensory sural nerve at the lateral ankle is often the final diagnostic step, but even such a biopsy does not identify the cause of PNPs in many patients<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Fully exploiting this precious biomaterial for mechanistic understanding and diagnostic potential is especially important to detect treatable causes such as immune-mediated neuropathies, which account for up to 10% of the PNPs<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. We and others previously performed single-cell transcriptomic analyses of rodent peripheral nerves and thereby deepened our mechanistic understanding of peripheral nerve diseases<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. However, human peripheral nerves have not been characterized in health or disease using similar technologies.</p>
<p id="Par4">Here, we provide a single-cell atlas of 37 human peripheral nerves (365,708 nuclei) integrated with spatial transcriptomics. In addition to identifying novel cell type markers, we redefined the unexpected heterogeneous perineurial cells. In diseases, repair and damage to Schwann cells (SC) and lipid-phagocytosing macrophages increased across PNP etiologies. PNPs also affected multiple cell types beyond the endoneurium. Specifically, we identified perineurial hyperplasia, especially in immune-mediated polyneuropathies, and this was associated with increased expression of the novel perineurial marker CXCL14. Multi-omic profiling of human peripheral nerves thus identified novel mechanisms with diagnostic potential.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Comprehensive species-specific single-nucleus transcriptional atlas of human sensory nerves</h3>
<p id="Par5">We sought to better understand human PNPs and first created a large single-nucleus transcriptomics atlas of human sural nerves of 37 donors (Fig. <a href="#Fig1" class="usa-link">1A</a>). Sural nerves from 33 PNP patients and four controls were collected in three centers (Supplementary Fig. <a href="#MOESM1" class="usa-link">1A</a> and Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). Control samples (CTRL) were residual material of surgical sural nerve autografts (‘interpositions’) from patients with traumatic nerve injuries. We performed single-nucleus RNA-sequencing (snRNA-seq) of all samples (“Methods”, Supplementary Fig. <a href="#MOESM1" class="usa-link">1B</a> and Supplementary Data <a href="#MOESM4" class="usa-link">2</a>). After analytical removal of doublets and low-quality nuclei and batch correction, this resulted in 365,708 total high-quality nuclei (Supplementary Fig. <a href="#MOESM1" class="usa-link">1C, D</a> and Supplementary Data <a href="#MOESM4" class="usa-link">2</a>). We then clustered the nuclei, resulting in 24 individual clusters (Fig. <a href="#Fig1" class="usa-link">1B</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">1E</a>), and annotated them using predefined marker genes (Supplementary Fig. <a href="#MOESM1" class="usa-link">2A</a> and Supplementary Data <a href="#MOESM5" class="usa-link">3</a>) and automatic annotation based on rodent data<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">2B</a>). We identified Schwann cells (<em>S100B, SOX10</em>) of both myelinating (mySC; <em>MPZ, MBP, PRX</em>) and non-myelinating (nmSC; <em>NCAM1</em>, <em>L1CAM</em>) types (Fig. <a href="#Fig1" class="usa-link">1B</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2A</a>). Schwann cells expressed features of repair<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> (repairSC; <em>NGFR</em>, <em>ATF3, GDNF, RUNX2</em>) and of damage (damageSC; <em>EGR1, FOS, JUN</em>). Moreover, we found endoneurial (endoC; <em>SOX9, PLXDC1, ABCA9</em>), perineurial (periC1-3; <em>SLC2A1</em>/GLUT1, <em>KRT19, CLDN1</em>), and epineurial (epiC; <em>CCBE1, COMP</em>) cells (Fig. <a href="#Fig1" class="usa-link">1B</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2A</a>). This perineurial cell heterogeneity was greater than previously described in rodents<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. The periC3 cluster was distinguished from other perineurial clusters by genes associated with extracellular matrix and collagen fibril organization (Supplementary Fig. <a href="#MOESM1" class="usa-link">2C</a>). Vascular cells included vascular smooth muscle cells (VSMC; <em>ACTA2</em>, <em>CARMN</em>), pericytes (PC1-2; <em>RGS5, PDGFRB</em>), and endothelial cells (EC; <em>EGFL7, PECAM1</em>) (Supplementary Fig. <a href="#MOESM1" class="usa-link">2A</a>). Based on known markers (Supplementary Fig. <a href="#MOESM1" class="usa-link">2A</a>) and published data<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">2B</a>), endothelial cells (EC) separated into a venous (ven_EC: <em>PLVAP</em>, <em>ACKR1</em>) to capillary (capEC: <em>ABCG2</em>, <em>MFSD2A</em>) to arterial (artEC: <em>SEMA3G</em>, <em>HEY1</em>, <em>GJA5</em>) continuum and lymphatic endothelial cells (LEC; <em>PROX1, LYVE1, FLT4</em>). We identified a cluster of venous/capillary EC (ven_cap_EC2), which expressed the blood-nerve barrier (BNB) markers <em>ABCB1</em> and <em>SLC1A1</em><sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, blood-brain barrier marker <em>MFSD2A</em><sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, and tight junction transcript <em>GJA1</em> and therefore likely represented EC of the BNB (Fig. <a href="#Fig1" class="usa-link">1B</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2A</a>). Nerve-associated leukocytes were mainly of myeloid lineage, outnumbering T/NK cells and B cells (Fig. <a href="#Fig1" class="usa-link">1B</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2A</a>). We thus created a transcriptional cellular atlas of human peripheral nerves, considerably extending rodent studies<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Defining the species-specific cellular landscape of human peripheral nerves.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375038_41467_2025_62964_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9eea/12375038/5b189c3d38e4/41467_2025_62964_Fig1_HTML.jpg" loading="lazy" id="d33e734" height="646" width="683" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic overview of the experimental procedure. Sural nerve biopsies were collected from 33 polyneuropathy (PNP) patients and four controls (CTRL) from three German centers (Essen, Münster, Würzburg) and processed by single-nucleus RNA-sequencing (snRNA-seq). In addition, subcellular spatial transcriptomics (spatial-seq) was performed in a subgroup of eight patients. All 37 tissues were histologically characterized by quantifying myelin thickness (g-ratio), number of intactly myelinated axons, and axon diameters. Perineurium thickness was measured in 12 CIDP and 18 CTRL patients using immunohistochemistry (IHC) targeting epithelial membrane antigen. Markers were confirmed using RNA in situ hybridization (RNA-ISH) and immunofluorescence (IF). <strong>B</strong> UMAP of 365,708 high-quality nuclei from 37 human sural nerves, showing 24 main clusters. <strong>C</strong> Schematic depicting if marker genes of human myelinating Schwann cells (mySC), non-myelinating Schwann cells (nmSC), and perineurial cells (periC) found in this dataset using snRNA-seq and spatial-seq were detected when reanalyzing published rodent datasets (Yim et al., Gerber et al., Wolbert et al., Supplementary Data <a href="#MOESM1" class="usa-link">4</a>–<a href="#MOESM1" class="usa-link">6</a>) when performing RNA-ISH/IF histology in rodents and whether these markers were previously described in rodent or human literature. <strong>D</strong>, <strong>E</strong> Spatial-seq was performed on sural nerve samples from a total of eight patients. Representation sections from CTRL patient S24 are shown. <strong>D</strong> Representative section of an H&amp;E staining overlaid with spatial-seq showing predicted clusters in spatial-seq of a sural nerve cross-section. Each dot represents one cell. <strong>E</strong> Representative spatial-seq images of selected genes expressed in perineurial cells. <strong>F</strong> Immunofluorescence staining was performed on a minimum of two sural nerve sections per patient from a total of four patients. Representative immunofluorescence image showing CXCL14 staining in perineurial cells from CIAP patient S10. The dotted line marks the perineurium. <strong>G</strong>,<strong> H</strong> Spatial-seq was performed on sural nerve samples from a total of eight patients. Representation sections from CTRL patient S24 are shown. Representative spatial-seq images of selected genes expressed in (<strong>G</strong>) cells of the blood nerve barrier (BNB) and (<strong>H</strong>) in mySC (blue highlight) and nmSC (red highlight). Each dot represents the expression of one transcript, a dotted line marks the perineurium, and a solid line surrounds individual vessels. Icons were created in BioRender. Meyer zu Hörste, G. (2025) <a href="https://BioRender.com/l74v346" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/l74v346</a>.</p></figcaption></figure><p id="Par6">We systematically compared cell type markers in humans with their published rodent counterparts<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup> (Supplementary Data <a href="#MOESM6" class="usa-link">4</a>–<a href="#MOESM8" class="usa-link">6</a>). As expected, the majority of transcripts were shared between species. However, several genes expressed in mySC (<em>MLIP</em>), nmSC (<em>GRIK3</em>, <em>PRIMA1</em>), and periC (<em>CXCL14</em>) were undescribed in rodent or human literature (Fig. <a href="#Fig1" class="usa-link">1C</a>). When reanalyzing rodent sc/snRNA-seq datasets<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>, we detected <em>Mlip</em> in mySC in two studies<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>; albeit previously unmentioned (Fig. <a href="#Fig1" class="usa-link">1C</a> and Supplementary Data <a href="#MOESM6" class="usa-link">4</a>–<a href="#MOESM8" class="usa-link">6</a>). In contrast, <em>Grik3</em>, <em>Prima1</em>, and <em>Cxcl14</em> were not detectable in rodent sc/nRNA-seq datasets (Supplementary Fig. <a href="#MOESM1" class="usa-link">2D</a> and Supplementary Data <a href="#MOESM6" class="usa-link">4</a>–<a href="#MOESM8" class="usa-link">6</a>). We thus identified novel transcripts across nerve-associated cells.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Spatially confirming novel cell type markers across species</h3>
<p id="Par7">For morphological confirmation, we first employed in situ amplification-based spatial transcriptomics with subcellular resolution (‘spatial-seq’)<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. We designed a custom marker panel of 99 known (e.g., <em>PRX</em>) and novel (e.g., <em>GRIK3</em>) transcripts (Supplementary Data <a href="#MOESM9" class="usa-link">7</a>) while excluding highly transcribed genes to prevent optical overcrowding (“Methods”). Visualizing these targets in a formalin-fixed paraffin-embedded (FFPE) cross-section of a human sural nerve graft (CTRL) replicated the overall cellular organization of peripheral nerves. We matched each cell in spatial-seq to an snRNA-seq cluster, aggregated these into 8 larger groups, and then visualized these groups (Fig. <a href="#Fig1" class="usa-link">1D</a>). As expected, the nerve consisted of multiple fascicles formed by endoneurial cell types (SC, endoC) surrounded by perineurial cells (periC). Vascular cell types formed vessels of their respective morphology (Fig. <a href="#Fig1" class="usa-link">1D</a>). Epineurial cells (epiC) were located in between fascicles and were interspersed with immune cells that were rare in the endoneurial areas (Fig. <a href="#Fig1" class="usa-link">1D</a>). We thus morphologically confirmed the unbiased annotation of human peripheral nerve cell clusters.</p>
<p id="Par8">In addition, previously undescribed perineurial markers (<em>CXCL14</em>) indeed colocalized with known perineurial cell transcripts (<em>CLDN1, SLC2A1, KRT19</em>)<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup> in the perineurium (Fig. <a href="#Fig1" class="usa-link">1E</a>). In addition, we confirmed CXCL14 expression in the perineurium using immunofluorescence (Fig. <a href="#Fig1" class="usa-link">1F</a>). We found that <em>CXCR4</em>—the receptor for <em>CXCL14</em>—was primarily expressed in the B and T/NK clusters (Supplementary Fig. <a href="#MOESM1" class="usa-link">3A</a>). In addition, we performed a computational analysis of cell-cell interactions and found that perineurial cells, particularly the periC2 cluster, were predicted to interact with B cells via <em>CXCL14</em>-<em>CXCR4</em> binding (Supplementary Fig. <a href="#MOESM1" class="usa-link">3B</a>). Immune cells may thus signal to perineurial cells in human peripheral nerves.</p>
<p id="Par9">The BNB markers <em>ABCB1</em> and <em>SLC1A1</em>, expressed by the ven_capEC2 cluster in snRNA-seq, were enriched in endoneurial vessels together with known EC markers (<em>PECAM1, EGFL7</em>)<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> (Fig. <a href="#Fig1" class="usa-link">1G</a>). This supported that the ven_capEC2 cluster represents BNB cells. Other vascular cell markers (e.g., <em>SEMA3G</em> in artEC) were associated with their respective vessel type (Supplementary Fig. <a href="#MOESM1" class="usa-link">3C</a>). Known pan-SC markers (<em>SOX10, S100B, EGR2</em>)<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup> were expressed across the endoneurium (Supplementary Fig. <a href="#MOESM1" class="usa-link">3D</a>). Known markers of mySC (<em>PRX, CDH1, SLC36A2</em>)<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> colocalized with <em>MLIP</em> identified in the mySC cluster (Fig. <a href="#Fig1" class="usa-link">1H</a>). Markers of nmSC (<em>CDH2, L1CAM, CHL1</em>)<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> colocalized with novel transcripts of the nmSC cluster (<em>PRIMA1, GRIK3</em>) in human nerve sections (Fig. <a href="#Fig1" class="usa-link">1H</a>).</p>
<p id="Par10">Next, we tested whether the novelty of these markers (<em>CXCL14, PRIMA1, GRIK3</em>) and their orthologues was due to technical differences between published and our dataset or due to human-specific expression by combining RNA in situ hybridization with immunofluorescence (RNA-ISH/IF) in rodent material. While <em>Grik3</em> was not detectable in rodent sc/snRNA-seq data (Supplementary Fig. <a href="#MOESM1" class="usa-link">2D</a> and Supplementary Data <a href="#MOESM6" class="usa-link">4</a>–<a href="#MOESM8" class="usa-link">6</a>). <em>Grik3</em> was expressed in murine nerves using RNA-ISH/IF, albeit at a much lower intensity than in positive control tissues (Supplementary Fig. <a href="#MOESM1" class="usa-link">3E</a>). <em>Grik3</em> mostly colocalized with the nmSC marker L1CAM (Supplementary Fig. <a href="#MOESM1" class="usa-link">3E</a>), consistent with our findings in the human tissue (Supplementary Fig. <a href="#MOESM1" class="usa-link">3E</a>). <em>Prima1</em> expression was not detectable in the murine nerve or positive control tissue. In contrast to rodent snRNA-seq, <em>Cxcl14</em> was detected in the murine perineurium using RNA-ISH. As expected, <em>Cxcl14</em> colocalized with the perineurial marker <em>Krt19</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">3E</a>). What appeared to be human specificity in snRNA-seq was therefore at least partly attributable to technical factors. Nonetheless, we provide multi-modal confirmation of several novel cell type markers: <em>MLIP</em> (myelinating Schwann cells), <em>GRIK3</em> and <em>PRIMA1</em> (non-myelinating Schwann cells), and <em>CXCL14</em> (perineurial cells).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Diseased human peripheral nerves contain myelin-phagocytosing macrophages</h3>
<p id="Par11">We next deeply subclustered all immune cell (IC) nuclei (<em>n</em> = 18,436) at high resolution (Fig. <a href="#Fig2" class="usa-link">2A</a>). In addition to the initial clustering, we identified multiple smaller myeloid clusters that could be assigned to classical dendritic cells (DC) type 1 (cDC1; <em>CLEC9A</em>, <em>XCR1</em>, <em>BATF3</em>), plasmacytoid DC (pDC; <em>CLEC4C</em>, <em>IRF8</em>), and classical DC type 2 (cDC2; <em>FCER1A</em>, <em>CD1C</em>, <em>CLEC10A</em>) (Fig. <a href="#Fig2" class="usa-link">2A</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">4A</a>). cDC2 had not been previously described in nerves<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Nerve-associated T/NK cell clusters, naive B cells (<em>CD19</em>, <em>MS4A1/</em>CD20, <em>CD79B</em>), and plasma cells (<em>JCHAIN</em>, <em>SDC1</em>/CD138) were also detected (Fig. <a href="#Fig2" class="usa-link">2A</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">4A</a>). B lineage cells expressed class-switched (<em>IGHG1</em>, <em>IGHG3</em>, <em>IGHG4</em>, <em>IGHA1</em>) heavy chain genes (Supplementary Fig. <a href="#MOESM1" class="usa-link">4B</a>). We thus delineated nerve-associated leukocytes in humans with high resolution.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Heterogeneity of human nerve-associated immune cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375038_41467_2025_62964_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9eea/12375038/5b344e2ef9bc/41467_2025_62964_Fig2_HTML.jpg" loading="lazy" id="d33e1144" height="622" width="674" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> UMAP of 18,436 nuclei representing 35 immune cell (IC) subclusters of 37 human sural nerves. <strong>B</strong> Feature plots of known endoneurial and epineurial macrophage marker genes. Color encodes gene expression. <strong>C</strong> Spatial-seq was performed on sural nerve samples from a total of eight patients. Representative spatial-seq images of the macrophage marker <em>MS4A7</em> and the endoneurial macrophage markers <em>CX3CR1</em> and <em>TREM2</em> in the sural nerve of CTRL patient S24. Each dot represents the expression of one transcript, a dotted line marks the perineurium, and a solid line surrounds individual vessels. <strong>D</strong> Gene ontology term enrichment analysis of marker genes (log<sub>2</sub> fold change &gt; 2, adjusted<em> p</em>-value &lt; 0.001) expressed by the Macro18/LAM cluster in a one vs. all immune cell cluster comparison. Statistical significance was assessed using enrichR (one-sided Fisher’s exact test with p-values adjusted for multiple comparisons using the Benjamini-Hochberg method). <strong>E</strong> Gene expression of lipid-associated macrophages in the immune cell subclusters. <strong>F</strong> Stroke-associated myeloid cells (SAMC) from Beuker et al. were projected onto the immune cell clustering. Each red dot denotes a nucleus predicted to be a SAMC. <strong>G</strong> Representative histological section characterizing Macro18/LAM cluster by co-staining marker genes with myelin and BODIPY (labels neutral lipids). <em>SPP1</em> encodes osteopontin. Nuclei were stained with DAPI. Stainings were performed on a minimum of two sural nerve sections per patient from a total of three patients.</p></figcaption></figure><p id="Par12">The majority of IC clusters (79% of all IC nuclei) were of myeloid lineage, mainly macrophages (Macro1-18: <em>LYZ</em>, <em>CD14</em>, <em>MRC1</em>, <em>CD163</em>, <em>MS4A7</em>) (Fig. <a href="#Fig2" class="usa-link">2A</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4A</a> and Supplementary Data <a href="#MOESM10" class="usa-link">8</a>). The phenotype and ontogeny of nerve-associated macrophages in rodents differ depending on their endo- vs. epineurial location<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. In our human dataset, macrophage clusters (<em>MS4A7</em>) also exhibited a gradient of expression from known endoneurial (<em>CX3CR1, TREM2)</em> to epineurial (<em>LYVE1, FOLR2, TIMD4</em>) markers (Fig. <a href="#Fig2" class="usa-link">2B</a>). Location-specific markers of epi- (<em>FOLR2</em>) vs. endoneurial (<em>CX3CR1</em>) macrophages (<em>MS4A7</em>)<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> were accordingly confirmed in spatial transcriptomics (Fig. <a href="#Fig2" class="usa-link">2C</a>). The phenotype of nerve-associated macrophages is thus conserved across species.</p>
<p id="Par13">One representative tentatively endoneurial macrophage cluster (Macro18) was notably enriched in lipid metabolism-related pathways (Fig. <a href="#Fig2" class="usa-link">2D</a>). Specifically, Macro18 showed high expression of <em>SPP1, APOE, LPL, FABP5</em>, and <em>GPNMB</em> (Fig. <a href="#Fig2" class="usa-link">2E</a>), and thus a transcriptional phenotype similar to lipid-associated macrophages (LAMs)<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> and to a previously described CNS myeloid population in stroke with a phagocytic and lipid-sensing phenotype<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Accordingly, when projected onto our dataset, the stroke-associated myeloid cells (SAMC) predominantly aligned with the Macro18 cluster (Fig. <a href="#Fig2" class="usa-link">2F</a>). Histologically, Macro18 markers (<em>SPP1</em>/Osteopontin, PLIN2, CD36, FABP5; Fig. <a href="#Fig2" class="usa-link">2E</a> and Supplementary Data <a href="#MOESM7" class="usa-link">5</a>) colocalized with myelin and BODIPY, which labels neutral lipids in samples from PNP patients (Fig. <a href="#Fig2" class="usa-link">2G</a>). Macro18 thus likely represents a population with myelin-phagocytizing capacity similar to myelin debris-clearing stroke-associated myeloid cells<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Damage response mechanisms in the presence of lipid excess may thus be shared between the central and peripheral nervous systems.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Non-endoneurial cell types profoundly respond to polyneuropathy</h3>
<p id="Par14">We next systematically characterized how PNP affected peripheral nerves using the snRNA-seq data (Supplementary Fig. <a href="#MOESM1" class="usa-link">1E</a>). The most apparent change in PNP was a loss of mySC and the occurrence of damageSC and repairSC (Fig. <a href="#Fig3" class="usa-link">3A</a>), which are known to be induced by nerve damage in rodents<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>–<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Among IC subclusters, the most prominent change in PNP patients was an increase in the Macro18 subcluster, indicative of endoneurial LAM (Fig. <a href="#Fig3" class="usa-link">3A</a>) with a potential role in myelin debris clearance. In addition, the perineurial cell cluster periC3 also increased in PNP (Fig. <a href="#Fig3" class="usa-link">3A</a>). When testing which individual single nuclei were associated with disease, the PNP status positively correlated with nuclei in the leukocyte and repairSC clusters and negatively correlated with the mySC cluster after correcting for confounders (Methods) (Fig. <a href="#Fig3" class="usa-link">3B</a>, left panel). This means that mySC are less likely to be present in PNP patients on a per-nucleus level. In addition, the periC3 cluster positively correlated with PNP disease status (Fig. <a href="#Fig3" class="usa-link">3B</a>, left panel). These clusters similarly correlated with clinical disease severity as quantified by the clinical INCAT score (Fig. <a href="#Fig3" class="usa-link">3B</a>, middle panel) and myelin thickness (inversely measured by g-ratio) (Fig. <a href="#Fig3" class="usa-link">3B</a>, right panel). PNPs thus induced profound compositional changes that correlated with disease severity and established measures of PNP, and they strongly affected perineurial cells.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Identifying a pan-neuropathy cellular transcriptional response pattern.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375038_41467_2025_62964_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9eea/12375038/80695ce5a355/41467_2025_62964_Fig3_HTML.jpg" loading="lazy" id="d33e1327" height="530" width="683" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Differences in cluster abundance in patients with polyneuropathy (PNP) vs. controls (CTRL), left: main clusters (Fig. <a href="#Fig1" class="usa-link">1B</a>), right: immune cell subclusters (Fig. <a href="#Fig2" class="usa-link">2A</a>). Only clusters with a log<sub>2</sub> fold change &gt; 0.5 are shown. <strong>B</strong> Correlations between individual single nuclei and PNP status (right), Inflammatory Neuropathy Cause and Treatment (INCAT) disability score (middle), and g-ratio of myelinated axons (right) are shown. Red indicates a positive, blue a negative correlation. <strong>C</strong> The number of differentially expressed genes (DEG) per cell cluster was determined in a pseudobulk approach and is visualized. <strong>D</strong> Neighborhood graph illustrating the number of DEG between PNP and CTRL in a cluster-free neighborhood-based approach. Each dot denotes a neighborhood. Edges reflect the number of cells shared between neighborhoods. The panel on the right displays a higher magnification of the neighborhoods with the largest number of DEGs. <strong>E</strong> Volcano plots of DEG in 0.1, and the vertical dashed lines display a log<sub>2</sub> fold change of 2. Selected transcripts above those thresholds are labeled. Statistical significance was assessed using a two-sided likelihood ratio test, with p-values adjusted for multiple comparisons using the Benjamini-Hochberg method.</p></figcaption></figure><p id="Par15">We next characterized which and how nerve cells <em>transcriptionally</em> responded to the disease. The number of differentially expressed genes (DEG) was highest in vascular clusters (ven_capEC1, PC2, artEC), followed by endoneurial fibroblasts (endoC) (Fig. <a href="#Fig3" class="usa-link">3C</a>). Using a cluster-free neighborhood-based approach, we accordingly found that neighborhoods with the most DEG were located in vascular clusters (PC2, ven_capEC1, artEC) and in the Macro2 cluster (Fig. <a href="#Fig3" class="usa-link">3D</a>). Next, we more specifically tested how PNP influenced gene expression of selected nerve-associated cells (mySC, nmSC, repairSC, PC2) (Fig. <a href="#Fig3" class="usa-link">3E</a>). We found that many of the DEG in the mySC cluster were associated with fibrotic tissue remodeling (e.g., <em>DCN, TNXB</em>), extracellular matrix formation (e.g., <em>COL1A1, COL15A1</em>), and immune regulation (e.g., <em>CD53, IL4R, CD74</em>). The repairSC cluster expressed genes for nerve repair and neuronal differentiation (<em>GALR1, TMEM47</em>) (Fig. <a href="#Fig3" class="usa-link">3E</a> and Supplementary Data <a href="#MOESM11" class="usa-link">9</a>). GO term enrichment analysis showed that genes upregulated in PNP were associated with cell differentiation in the mySC cluster and with cell migration in the nmSC cluster (Supplementary Fig. <a href="#MOESM1" class="usa-link">4C</a>). In summary, PNPs induced transcriptional changes in Schwann cells but also widely outside of the endoneurium, especially in non-endoneurial vascular cells. PNPs thus surprisingly affected the entire cellular micro-milieu of peripheral nerves even beyond the endoneurium and may thus constitute ‘pan-nerve diseases’.</p></section><section id="Sec7"><h3 class="pmc_sec_title">PNP subtypes preferentially affect different cellular compartments of peripheral nerves</h3>
<p id="Par16">We next sought to better understand the poorly defined heterogeneity of PNP mechanisms. We classified the available PNP patients by integrating all diagnostic information (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>) into seven subtypes of PNPs: vasculitic (VN, <em>n</em> = 5), chronic inflammatory demyelinating (CIDP, <em>n</em> = 9), chronic idiopathic axonal (CIAP, <em>n</em> = 11), cancer-associated paraproteinemic (PPN, <em>n</em> = 2), diabetic (DPN, <em>n</em> = 2), other inflammatory (OIN, <em>n</em> = 2), and other non-inflammatory (ONIN, <em>n</em> = 2) (Fig. <a href="#Fig4" class="usa-link">4A</a>). Histological characterization (Supplementary Fig. <a href="#MOESM1" class="usa-link">5A–G</a>) identified that myelin thickness (Supplementary Fig. <a href="#MOESM1" class="usa-link">5B, C</a>) and the average number of intactly myelinated axons (Supplementary Fig. <a href="#MOESM1" class="usa-link">5D</a>) decreased in PNPs, while axon diameter was less affected (Supplementary Fig. <a href="#MOESM1" class="usa-link">5E</a>). Histological findings in PNP subtypes were in accordance with expectations and correlated with electrophysiology (Supplementary Fig. <a href="#MOESM1" class="usa-link">5F</a>) and with cluster proportions determined by snRNA-seq (Supplementary Fig. <a href="#MOESM1" class="usa-link">5G</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. PNP subtypes affect different cellular compartments of peripheral nerves.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375038_41467_2025_62964_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9eea/12375038/4b9bfd1de85a/41467_2025_62964_Fig4_HTML.jpg" loading="lazy" id="d33e1468" height="629" width="683" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Proportion of cells split by disease group and colored by the main clusters. <strong>B</strong>, <strong>C</strong> Changes in cluster abundance in patients with vasculitic (VN), chronic inflammatory demyelinating (CIDP), and chronic idiopathic axonal (CIAP) polyneuropathy vs. controls (CTRL). The right panels display changes of the main clusters (as in Fig. <a href="#Fig1" class="usa-link">1B</a>), the right panels show differences in the immune cell subclusters (Fig. <a href="#Fig2" class="usa-link">2A</a>). Only clusters with a log<sub>2</sub> fold change &gt; 0.5 are shown. <strong>D</strong>–<strong>F</strong> Spatial-seq was performed on sural nerve samples from a total of eight patients (two per disease). <strong>D</strong> Representative sections of H&amp;E staining of sural nerves overlaid with spatial-seq showing predicted Schwann cell clusters in CTRL (S24, left panel) and VN (S30, right panel) patients. Each dot represents one cell. <strong>E</strong> The density of <em>PTPRC</em> transcripts in the endoneurium (left pot) and epineurium (right plot) per disease group (<em>n</em> = 2 per disease group). Box plots show the median, interquartile range (IQR), and whiskers extending to 1.5 × IQR. Dots represent individual measurements. <strong>F</strong> Representative spatial-seq images showing <em>TREM2</em> transcripts (yellow) in the perineurium of CTRL (S24), VN (S30), CIDP (S01), and CIAP (S04) patients. Each square indicates the expression of one transcript. <strong>G</strong> Neighborhood graphs displaying the number of DEG between PNP patients with VN (left), CIDP (middle) or CIAP (right), and CTRL in a cluster-free neighborhood-based approach. Edges reflect the number of cells shared between neighborhoods.</p></figcaption></figure><p id="Par17">We next compared cluster proportions between groups with at least four samples per group (VN, CIDP, CIAP, CTRL), while disregarding others. In VN, CIDP, and CIAP, we observed a relative increase in damageSC and repairSC, alongside a reduction in mySC, compared to controls (Fig. <a href="#Fig4" class="usa-link">4B</a>). Schwann cell pathology thus occurred widely across PNPs. A gain of the periC3 and the venEC cluster was shared between VN and CIDP. An increase in the VSMC cluster was found in both VN and CIAP. Loss of the LEC cluster was specific to VN (Fig. <a href="#Fig4" class="usa-link">4B</a>). The lipid-associated Macro18/LAM immune cell cluster was increased in all three PNP groups compared to CTRL (Fig. <a href="#Fig4" class="usa-link">4C</a>). Only VN showed an expansion of the plasma cell cluster (Fig. <a href="#Fig4" class="usa-link">4C</a>). Compositional cellular analysis of PNP subtypes thus revealed unique and shared cellular alterations, including a shared increase of Macro18/LAM.</p>
<p id="Par18">We next analyzed spatial transcriptomics of sural nerves from VN, CIDP, and CIAP patients vs. CTRL (<em>n</em> = 2 per group). Predicted repairSCs were increased in the endoneurium in VN (Fig. <a href="#Fig4" class="usa-link">4D</a>). Predicted mySCs were considerably less abundant in VN than in CTRL (Fig. <a href="#Fig4" class="usa-link">4D</a>), and this was also true but less prominent for CIDP and CIAP (Supplementary Fig. <a href="#MOESM1" class="usa-link">5H, I</a>). Quantification of leukocyte-associated transcripts (<em>PTPRC</em>/CD45, <em>MS4A1</em>/CD20, <em>CD3E</em>) showed that the density of endo- and epineurial B and T cells increased in VN, CIDP, and CIAP compared to CTRL (Fig. <a href="#Fig4" class="usa-link">4E</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5J, K</a>). Predicted T_NK cells were more abundant in VN and less so in CIDP and CIAP than in CTRL (Supplementary Fig. <a href="#MOESM1" class="usa-link">5L, M</a>). <em>TREM2</em>, as a marker expressed by Macro18/LAM (Fig. <a href="#Fig2" class="usa-link">2B</a>), was predominantly detected in the endoneurium (Fig. <a href="#Fig4" class="usa-link">4F</a>) and was increased in CIDP, CIAP, and most notably in VN compared to CTRL (Supplementary Fig. <a href="#MOESM1" class="usa-link">5N</a>). We thus spatially validated the gain of repairSC, leukocytes, and LAM along with the loss of mySC in PNPs, with the most prominent effects in VN.</p>
<p id="Par19">Next, we tested for subtype-specific transcriptional alterations. In VN, most DEGs were located in neighborhoods in the PC2 (pericyte) cluster (Fig. <a href="#Fig4" class="usa-link">4G</a>) in accordance with vascular-focused inflammation. In CIDP, DEGs were also identified in neighborhoods of vascular clusters (ven_capEC1, artEC, PC2) and in the Macro2 cluster. In CIAP, DEGs were distributed in a more widespread pattern (Fig. <a href="#Fig4" class="usa-link">4G</a>). Each PNP subtype thus induced unique compositional and transcriptional responses.</p></section><section id="Sec8"><h3 class="pmc_sec_title">PNPs, particularly immune-mediated subtypes, exhibit focal perineurial hyperplasia</h3>
<p id="Par20">We next aimed to spatially validate the notable increase in the perineurial cell clusters in snRNA-seq. Plotting known (<em>CLDN1, SLC2A1, KRT19</em>) and novel (<em>CXCL14</em>) perineurial markers identified the perineurium as a single cell layer in CTRL but as 3–5 cell layers in PNP samples. This perineurial hyperplasia was most pronounced in CIDP and appeared to be regionalized (Fig. <a href="#Fig5" class="usa-link">5A</a>). The perineurium was also more dispersed and heterogeneous in PNP samples (Fig. <a href="#Fig5" class="usa-link">5A</a>). Moreover, the expression of the novel perineurial marker <em>CXCL14</em> was increased in PNPs compared to CTRL in snRNA-seq and immunofluorescence (Fig. <a href="#Fig5" class="usa-link">5B, C</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. PNPs, particularly immune-mediated subtypes, exhibit focal perineurial hyperplasia.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375038_41467_2025_62964_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9eea/12375038/6433a4b3c63e/41467_2025_62964_Fig5_HTML.jpg" loading="lazy" id="d33e1604" height="456" width="676" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Spatial-seq was performed on sural nerve samples from a total of eight patients (two per disease). Representative spatial transcriptomics images of perineurial marker genes of CTRL (S24), VN (S30), CIDP (S01), and CIAP (S14). Each dot indicates the expression of one transcript. <strong>B</strong> Average gene expression of <em>CXCL14</em> expression per disease group in snRNA-seq. <strong>C</strong> Representative immunofluorescence images showing CXCL14 staining in CTRL (S24) vs. CIDP (S08) (left panel). Immunofluorescence staining was performed on a minimum of six sural nerve sections per patient from a total of four patients. The quantification of the CXCL14-positive area in the perineurium is visualized in a boxplot (right panel). Box plots show the median, interquartile range (IQR), and whiskers extending to 1.5 × IQR. Dots represent individual measurements (technical replicates; S24/S08/S09: <em>n</em> = 8, S25: <em>n </em>= 6). <strong>D</strong> EMA staining was performed on sural nerve samples from a total of 30 patients. Representative images of epithelial membrane antigen (EMA) staining in CTRL (S71) and CIDP (S76). <strong>E</strong>, <strong>F</strong> The inner and outer perimeters, along with the corresponding area, were measured in CTRL (<em>n</em> = 18) and CIDP (<em>n</em> = 12) patients. A linear mixed model was used to adjust for confounding variables (Methods) and to determine statistical significance (two-sided <em>t</em> test). Box plots show the median, interquartile range (IQR), and whiskers extending to 1.5 × IQR.</p></figcaption></figure><p id="Par21">To validate the perineurial hyperplasia in CIDP, we histologically stained epithelial membrane antigen (EMA)<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> in 18 additional controls (residual material of surgical sural nerve autografts) and 12 CIDP patients (Supplementary Data <a href="#MOESM12" class="usa-link">10</a>) and measured the area and perimeter of the perineurium for each fascicle (370 fascicles in total) (Fig. <a href="#Fig5" class="usa-link">5D</a>). We applied a linear mixed model to account for potential confounders (age and sex as fixed effects) and center- and sample-specific variability (random effects). We found that the perineurial area and perimeter were significantly increased in CIDP patients compared to CTRL (Fig. <a href="#Fig5" class="usa-link">5E</a>). Moreover, their variance significantly increased in CIDP (Fig. <a href="#Fig5" class="usa-link">5F</a>), suggesting that fascicular involvement was regionalized. In conclusion, CIDP exhibited focal perineurial hyperplasia driven by an expansion of PNP-specific perineurial cells, potentially induced by immune-perineurial <em>CXCL14</em> signaling.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par22">We aimed to exploit the full potential of human nerve biopsies using state-of-the-art techniques and present a single-nucleus transcriptomics atlas comprising 365,708 nuclei of 33 PNP patients and four controls integrated with subcellular spatial transcriptomics. Capitalizing on this novel application to human peripheral nerves and a dataset 10-fold larger than previous rodent studies<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, we discovered and validated previously undescribed transcriptional markers of perineurial fibroblasts (<em>CXCL14</em>) and myelinating (<em>MLIP</em>) and non-myelinating SC (<em>GRIK3, PRIMA1</em>). In human PNPs, especially in immune-mediated forms, we identified perineurial hyperplasia potentially driven by <em>CXCL14</em> signaling. Myelin-phagocytosing nerve and lipid-associated macrophages accumulated across PNPs, and transcriptional changes widely affected multiple non-endoneurial cell populations, suggesting PNPs as ‘pan-nerve’ diseases. In conclusion, we identified a PNP subtype-specific composition and transcriptome with diagnostic potential.</p>
<p id="Par23">Our study has limitations: Our patient cohort as a whole is large in comparison to other single-cell transcriptomics studies<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, but limited patient numbers within each PNP subtype might be insufficient to fully explore differences between subtypes. Our results are potentially confounded by imbalances in disease duration, sex, center of origin, and immunosuppressive therapy in some patients (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). Larger multi-centric, multi-national funding, collection, and analysis efforts will be required in the future. Sural nerve biopsies are invasive with some inherent risk and are usually performed in difficult-to-diagnose patients<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Our patient cohort is thus biased towards rare PNP etiologies and, for example, includes only two diabetic PNP, the most common cause of PNPs<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. We mitigate this by focusing on groups with at least four patients and analytically correcting for biases when possible. We believe that novel techniques to enrich rare cell types<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> could identify PNP-associated cellular patterns in more readily available tissues, such as skin biopsies, in the future. Whether less expensive and less labor-intensive methods, such as bulk transcriptomic or methylome analysis of peripheral nerves, could be similarly powerful remains to be determined.</p>
<p id="Par24">Among novel cell type transcripts, the <em>MLIP</em> gene encodes a muscular laminin-interacting protein required to maintain muscular integrity<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, which could serve similar functions in SC. The <em>PRIMA1</em>-encoded protein organizes esterases at the neuromuscular junction in terminal SC<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> but has not been described in nmSC. <em>GRIK3</em> encodes a glutamate ionotropic receptor<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> and variants in <em>GRIK3</em> have been associated with hereditary neuropathy<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. These genes may thus be involved in the structural organization of SC in human peripheral nerves. Human <em>CXCL14</em> (synonymously <em>BRAK</em>) is constitutively expressed by epithelial tissues and regulates microglial development in the brain<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> and neurovascular patterning in the eye<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. It is thus conceivable that the perineurium, regarded as an epithelial barrier<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, expresses <em>CXCL14</em> and thereby contributes to nerve organization. Upregulation of <em>CXCL14</em> was previously described in a mouse model of hereditary neuropathy<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Here, we found an increase in <em>CXCL14</em> expression in perineurial cells in human PNPs, suggesting <em>CXCL14</em> as a potential disease marker. The predicted interaction of <em>CXCL14</em> with B cell-derived <em>CXCR4</em> signals hints towards immunity-driven perineurial hyperplasia.</p>
<p id="Par25">The perineurium is a lamellated structure made up of concentric cell layers bordered on each side by a basement membrane<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. The perineurium forms an active diffusion barrier to maintain the homeostasis of the endoneurium<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. A linear relationship has been described between fascicle diameter and perineurium thickness<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, i.e., larger fascicles have a thicker perineurium. In addition, distal sections have a greater perineurium thickness than proximal segments<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Perineurial thickening is a characteristic of diabetic PNP<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> and Borrelia neuropathy<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Focal or generalized perineurial thickening also profoundly occurs in ‘perineuritis’, a rare inflammatory disease<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Proliferation of PNP-specific perineurial cells could reflect a maladaptive response of the nerve microstructure to chronic damage or inflammation. Overall, perineurial thickening per se has not been studied systematically in PNPs and may be diagnostically more relevant than previously appreciated.</p>
<p id="Par26">Conditions such as obesity, stroke, and atherosclerosis induce tissue damage accompanied by local lipid excess and trigger the accumulation of lipid-associated macrophages (LAM) in the affected tissues<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. LAMs share a common set of lipid-metabolizing transcripts (e.g., <em>Fabp5, Lpl, Spp1, Trem2</em>) and likely represent an adaptive response aimed at clearing excess lipids from the site of injury. Nerve-associated macrophages feature complex location-specific ontogeny<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> and are implicated in multiple nerve diseases and in nerve regeneration<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. By defining LAMs in the peripheral nerves of human PNP patients, we here identified that the innate immune response to damage is conserved across lipid-rich tissues and across diverse injury conditions. This may hold even wider implications for PNPs, given that the lipid-sensing TREM2/APOE pathway on disease-associated macrophages (DAM) constitutes a potential biomarker and preclinical therapeutic target of neurodegenerative CNS diseases<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Collection of patient samples</h3>
<p id="Par27">We collected sural nerves from 33 PNP patients from three centers: University Hospital of Münster (10 patients), Essen (11 patients), and Würzburg (12 patients). In addition, we obtained sural nerves from four control patients in Essen with traumatic nerve injuries who received a sural nerve autograft as interposition but were unaffected by polyneuropathy (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). Residual material from the sural nerve grafts was used for control samples in this study. In PNP patients, sural nerve biopsy was performed as part of the clinical diagnostic workup, and a small portion of the nerve (approximately 0.2 cm) was used for this study. In Münster and Essen, the samples were immediately fresh-frozen after surgery in dry ice-cooled methylbutane and stored in liquid nitrogen (0−14 months, Supplementary Data <a href="#MOESM3" class="usa-link">1</a>) until nuclei extraction. In Würzburg, the samples were embedded into an OCT-embedding matrix (Roth) until nuclei extraction (33−77 months, Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). In Essen and Münster, samples were collected from newly recruited patients. In Würzburg, samples were collected from an existing biobank of OCT-embedded cryo-preserved sural nerve biopsies collected between 2016 and 2020. All experiments were carried out in accordance with the Declaration of Helsinki and were approved by the local ethical committees in Münster (2018-719-f-S), Essen (21-10376-BO), and Würzburg (238/17; 15/19). All patients gave written informed consent to sample collection.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Patient characteristics used for sequencing</h3>
<p id="Par28">Patients were characterized clinically regarding diagnosis, disease activity, therapy response, disease duration, immunosuppressive therapy, relevant secondary diagnosis, electrophysiological studies, and CSF analysis, outlined in Supplementary Data <a href="#MOESM3" class="usa-link">1</a>. Patients were diagnosed according to the EAN/PNS criteria 2021 for CIDP<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup> and according to the PNS criteria for VN.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Nuclei extraction and purification</h3>
<p id="Par29">For each patient, approximately 10 mg pieces (median of 13.9 mg) were cut on dry ice as starting material from the original samples. Nerve pieces were cut into smaller pieces and processed using the Miltenyi Biotec nuclei extraction protocol. In summary, samples were transferred to a gentleMACS C-tube (Cat. no. 130-093-237) containing 2 ml lysis buffer (nuclei extraction buffer Cat. no. 130-128-024 + 0.2 µ/µl RNase inhibitor EO0381) and processed on a gentleMACS dissociator with program 4C_nuclei_1. After dissociation, the nuclei suspension was filtered through a pluriStrainer Mini 70 µm cell strainer (pluriSelect®) and washed in 200–400 µl resuspension buffer (PBS with 0.1% BSA and 0.2 µ/μL RNase inhibitor EO0381), depending on sample size and expected nuclei count. Nuclei suspension was then filtered through a 40 μm cell strainer (pluriSelect®), placed on a 1.5 ml DNA-LoBind tube (Merck). The nuclei suspension was stained with Trypan Blue to manually assess nuclei viability and count using a Fuchs-Rosenthal chamber. Equal volumes between samples were used for the downstream application of single-nucleus RNA seq.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Single-nucleus RNA sequencing and generation of count matrices</h3>
<p id="Par30">Single-nucleus suspensions were loaded into a Chromium Next GEM Chip G and placed into the Chromium X Single Cell Controller and processed with Chromium Next GEM Single Cell 3´ Kit v3.1 reagents (all 10X Genomics). Sequencing was performed on Illumina Nextseq 2000 and Novaseq 6000 with a 28-8-0-91 read setup. We used cellranger v7.0.1 (10X Genomics) to generate count matrices with default parameters but an optimized transcriptome reference v1.1<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> for GRCh38.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Single-nucleus analysis</h3>
<p id="Par31">CellBender v0.3.0<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup> was used to remove background noise. In CellBender, the number of expected cells and total droplets was based on the UMI curve, and the learning rate was reduced based on the automated output report. Further downstream analysis was performed with the R package Seurat v5.0.1<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. To increase computational efficiency and reduce the memory usage of this large dataset, we used BPCells v0.1.0. Low-quality cells were filtered for each sample individually by inspecting quality control plots and removing cells with higher mitochondrial percentages (range: 1–5%), low (&lt; 200) or very high molecule counts per cell (range: 6000–9000). Doublets were removed using scDblFinder v1.16.0<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> with default parameters. We used Seurat for normalization (LogNormalize, default parameters), identification of highly variable genes (vst method, 2000 features), scaling, and performing PCA (default parameters). Next, we used atomic sketch integration (method LeverageScore, 5000 cells) in Seurat to reduce memory usage. Batch effects were accounted for by integrating the samples with scVI v1.0.4<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. After inspecting the effect of batch removal, the full dataset was integrated with Seurat. The scVI integrated full dataset was used to calculate the UMAP embeddings with Seurat (30 dimensions). We identified clusters with the FindNeighbors and FindClusters (resolution 0.7) functions. We calculated the top-expressing genes of each cluster with FindMarkers (min.pct = 0.1, logfc_threshold = 0.25, p_val_adj &lt; 0.05, two-sided Wilcoxon rank-sum test). To subcluster the immune cells, we selected the immune cell clusters of the main clusters. We then performed identification of highly variable genes, scaling, and PCA as explained above. The data were integrated with reciprocal PCA using Seurat, and after plausibility checks of the batch removal, UMAP was calculated (30 dimensions). Clusters were identified with FindNeighbors and FindClusters (resolution 2.3). The top-expressing genes were identified as explained for the main clusters. Enrichment analysis of top markers (avg_log2FC &gt; 1, p_val_adj &lt; 0.001) was carried out with enrichR 3.2<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> with the GO Biological Process 2023 database. Differentially expressed genes between conditions were identified using a pseudobulk method with Libra v1.0.0<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> (edgeR 4.0.7<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>, two-sided likelihood ratio test). Volcano plots of DE genes were created with Enhanced Volcano v1.20.0. In addition, we determined DE genes in a cluster-independent manner with miloDE v0.0.0.9<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> following the tutorial. Briefly, neighborhoods were assigned based on the scvi integrated data (<em>k</em> = 30, prop = 0.1, <em>d</em> = 30), and DE testing was conducted between conditions (min_count = 10). To determine the cluster abundance, we used propeller<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> (part of speckle v1.2.0). We used the co-varying neighborhood analysis (CNA), implemented in the rcna package v0.0.99<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>, to compute correlations between clinical phenotypes or histological measures and our single-cell data independent of clustering, controlling for age and sex. To analyze cell-cell interactions, we used LIANA v0.1.14<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup> based on the methods NATMI, Connectome, SingleCellSignalR, and logFC.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Comparison with published datasets</h3>
<p id="Par32">We downloaded the publicly available annotated data from Yim et al.<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> (GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE182098" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE182098</a>, sciatic nerve), Gerber et al.<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup> (<a href="https://snat.ethz.ch/seurat-objects.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://snat.ethz.ch/seurat-objects.html</a>, 10X Genomics P60), Mathys et al. (<a href="https://compbio.mit.edu/scBBB/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://compbio.mit.edu/scBBB/</a> ROSMAP vascular cells)<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, Wolbert et al.<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> (GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142541" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE142541</a>, mouse), and Beuker et al. (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE189432" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE189432</a>). If necessary, the data were preprocessed with Seurat, i.e., normalized with LogNormalize, highly variable genes were identified and scaled, and PCA was performed with default parameters. To identify novel marker genes, cell markers in the published datasets were determined with the FindMarkers function in Seurat (min.pct = 0.1, logfc.threshold = 0.25, p_val_ad &lt; 0.05). To annotate our dataset based on the published dataset, rodent gene names were converted to orthologues using homologene v.1.4.68 (homologeneData2 database). We then classified our cells based on the annotated reference dataset using FindTransferAnchors and TransferData functions in Seurat with default parameters. Cells were labeled as unknown if the Seurat prediction score was below 0.3.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Histology: Semi-thin sections and analysis</h3>
<p id="Par33">After biopsy, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e1943"><mn>1</mn><mo>/</mo><mn>3</mn></math></span> of the sural nerve was embedded in epoxy resin (Fig. <a href="#Fig1" class="usa-link">1A</a>, experimental setup). Semithin sections (1 µm) of sural nerves were cut using an EM UC7 ultramicrotome (Leica Microsystems) and stained with toluidine blue. The sections were scanned with a slide scanner (Münster/Würzburg: Grundium Ocus40; Essen: Zeiss Axio Scan Z.1, Hitachi HV F203SCL camera). After blinding, the entire number of total myelinated axons per sural nerve was counted manually by one investigator using the CellCounter plug-in of ImageJ v1.36. Physiologically unmyelinated axons (diameter &lt; 1 µm) and Remak bundle fibers were not included.</p>
<p id="Par34">The ImageJ g-ratio plugin was used to determine the myelin thickness and axonal diameter of 15% of the total myelinated axons per sural nerve. In randomly chosen myelinated axons, the inner and outer rim of a myelin sheath were manually traced by one investigator. The g-ratio was calculated by dividing the axonal circumference (i.e., inner rim of the myelin sheath) by the outer circumference of the respective myelin sheath, presuming circular axons. The g-ratio values of each patient were plotted against the calculated axon diameters.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Preparation of sural nerve cross sections and spatial transcriptomics</h3>
<p id="Par35">We created a custom, standalone gene panel for spatial transcriptomics using the commercial Xenium Analyzer platform. The panel was designed based on our single-nucleus analysis, mostly focusing on cells that demonstrated a potential role in polyneuropathies, including Schwann cells (mySC, nmSC, repairSC), perineurial cells, and vascular cells such as pericytes and endothelial cells. We included a minimum of four top marker genes of each of the clusters and also defined twenty DE genes that were differentially expressed between the different PNP subtypes. We additionally included known markers of both epi- and endoneurial cells and different subtypes of leukocytes, including B cells, T cells, and macrophages, adding up to a total of 99 genes (Supplementary Data <a href="#MOESM9" class="usa-link">7</a>).</p>
<p id="Par36">In total, 8 sural nerve formalin-fixed paraffin-embedded (FFPE) samples were included for Xenium spatial transcriptomics: 2 CTRL (S22, S24), 2 CIDP (S01, S11), 2 CIAP (S04, S14), and 2 VN (S29, S30) samples (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). Samples were primarily selected based on their optimal/intact morphology in the semithin sections and their center, with the aim to limit center bias.</p>
<p id="Par37">Sample preparation and processing were carried out by the CMCB Technology Platform Core Facility EM and Histology, and the DRESDEN-concept Genome Center at the Technical University Dresden, in Germany. Samples were processed according to manufacturer protocols (10x protocols, <a href="https://www.ncbi.nlm.nih.gov/nuccore/CG000580" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CG000580</a>, CG00582 and CG00584). Briefly, 4 µm cross sections were collected, floated in a 37 ˚C water bath, and adhered to Xenium slides (10x Genomics, PN 1000460). Samples were deparaffinized in 2x xylene and rehydrated in a descending series from 100% ethanol to MilliQ water. After inserting slides into Xenium cassettes, samples were decross-linked and incubated overnight (22 h) with padlock probes, followed by a post-hybridization wash. Subsequently, padlock probes were ligated, followed by rolling circle amplification, autofluorescence quenching, and nuclear staining (DAPI). Slides were loaded on a 10X Xenium Analyzer for region selection, with each region corresponding to one entire nerve bundle.</p>
<p id="Par38">Primary analysis, image processing and decoding, and secondary analysis, cell segmentation, and transcript mapping were performed on the instrument with the analysis software Xenium v1.6.0.8, resulting in a cell-feature matrix and an initial clustering. For cell segmentation, a custom neural network on DAPI images was initially used for nucleus segmentation. Then nucleus boundaries are expanded by 15 µm or until they encounter another cell boundary in X-Y to define cell boundaries. Transcripts were mapped to these 2D shapes according to their X and Y coordinates. Gene expression mapping was visualized using the Xenium Explorer software v.1.3. The post-Xenium H&amp;E staining was processed according to the manufacturer protocol, including quencher removal in advance (see 10x Genomics protocol <a href="https://www.ncbi.nlm.nih.gov/nuccore/CG000613" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CG000613</a>/A).</p></section><section id="Sec19"><h3 class="pmc_sec_title">Spatial transcriptomics data analysis</h3>
<p id="Par39">Xenium data was loaded into Seurat. We removed cells with less than 10 molecules per cell. We performed normalization with SCTransform (default parameters) and computed PCA for each sample. To match our snRNA-seq data to our Xenium data, each cell was classified with the FindTransferAnchors and TransferData function based on our previously annotated snRNA-seq data (downsampled to 1000 cells per cluster). Cells with a prediction score below 0.3 were labeled as unknown. The 24 main clusters were then aggregated into 8 larger groups (SC, endoC, periC, epiC, VSMC, PC, EC, IC). For manual quantification of endo- vs. epineurial transcripts, the Xenium images were loaded into Xenium Explorer v1.3.0 (10x Genomics) and endoneurial areas were manually outlined using the selection tool while using visualization of Schwann cell marker transcripts (<em>EGR2, NGFR, SOX10, S100B)</em> to reliably identify such endoneurial areas. The area of each image visibly unoccupied by nerve-associated tissue was selected as epineurium. The area (in μm<sup>2</sup>), the number of total segmented cells, and the number of immune cell-associated transcripts (leukocytes: <em>PTPRC</em>/CD45, B cells: <em>MS4A1</em>/CD20, T cells: <em>CD3E</em>) were quantified in each selection area. The density of the respective transcript per total endoneurial and epineurial area was calculated for each sample. For the <em>TREM2</em> quantification, the entire nerve area (in µm<sup>2</sup>), the number of total segmented cells, and the number of <em>TREM2</em> transcripts were quantified using the selection tool in the Xenium Explorer. Selected transcripts were visualized in Xenium Explorer. Images were integrated with corresponding H&amp;E staining to visualize the spatial orientation.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Combined immunofluorescence and RNA in situ hybridization (RNA-ISH) in murine tissue</h3>
<p id="Par40">10–15 week old female mice were perfused with phosphate-buffered saline (PBS) and sacrificed. The sciatic nerve and the brain were isolated and directly frozen in Tissue-Tek O.C.T.<sup>™</sup> compound (Sakura Finetek Europe). 10 µm sections were cut at − 20 °C using a cryostat (Epredia CryoStar NX50). Samples were processed according to the HCR<sup>TM</sup> RNA-FISH protocol for frozen or fixed frozen tissue sections and the multiplexed HCR<sup>TM</sup> RNA-FISH protocol from Molecular Instruments, followed by immunofluorescence staining. Probes <em>Krt19</em> (B1; <a href="https://www.ncbi.nlm.nih.gov/nuccore/AB472075.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AB472075.1</a>), <em>Cxcl14</em> (B2; <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_019568.2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_019568.2</a>), and <em>Grik3</em> (B2; <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_001081097.3" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_001081097.3</a>) were used to detect target RNA within intact cells, and the primary antibody mouse anti-L1cam (1:100, Abcam, ab24345) and AF488-labeled secondary antibody (1:100, Life, A-11001) were used for the immunofluorescence staining. Briefly, the fresh frozen samples were fixed in 4% paraformaldehyde for 15 minutes at 4 °C and then dehydrated in a series of 50%-70%-100% ethanol each for 5 min at room temperature. Samples were washed with PBS and incubated with a hybridization buffer for 30 min at 37 °C. A concentration of 0.5 pmol of each probe was used and incubated at 37 °C overnight. Excess probes were removed by applying a series of 25%-50%-75%-100% 5xSSCT (5x sodium chloride, sodium citrate (SSC) and 0.1% Tween 20) for 15 min each at 37 °C, except for the final 100% 5 x SSCT, which was incubated for 15 min at room temperature. Samples were pre-amplified in a humidified chamber for 30 min at room temperature and incubated with a hairpin solution (4.5 pmol/hairpin) for the respective probes overnight at room temperature. The next day, slides were washed with 5 x SSCT, and the immunofluorescence staining was performed. Samples were blocked using a blocking reagent for 15 minutes at room temperature, followed by incubation with the primary antibody for 1 h at room temperature. Subsequently, samples were incubated with a fluorescent-labeled secondary antibody for an additional hour at room temperature. Sections were mounted in a fluorescence preservative mounting medium containing DAPI (4’,6-diamidino-2-phenylindole, Fluoromount-G, Invitrogen 00-4959-52), and images were obtained with a Zeiss AxioVision Apotome (Carl Zeiss) and processed in ZEN 2.3 lite (blue edition).</p></section><section id="Sec21"><h3 class="pmc_sec_title">Immunohistochemistry of human tissue</h3>
<p id="Par41">Formalin-fixed, paraffin-embedded nerve sections (5 µm) were rehydrated and antigen retrieved using EnVisionTM FLEX Target Retrieval Solution (pH 6.0). After washing (PBS, pH 7.4), sections were incubated in blocking reagent (Roche, Mannheim, Germany) for 20 min (room temperature) and stained with the following primary antibodies: rabbit anti-Osteopontin (1:100, Abcam, ab63856), goat anti-FABP5 (1:100, R&amp;D Systems, AF1476), rabbit anti-Perilipin-2 (1:100, Novusbio, NBP2-48532), rabbit anti-CD36 (1:100, Abcam, ab124515), and rabbit anti-CXCL14 (1:100, Abcam, ab264467). Myelin was stained with FluoroMyelin Green (1:100, 20 min, Thermofisher, <a href="https://www.ncbi.nlm.nih.gov/nuccore/F34651" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">F34651</a>). All nerve sections were then incubated with the appropriate AlexaFluor secondary antibody for target antigen visualization (1:100, 45 min, room temperature). For lipid droplet and CXCL14 staining, frozen sural nerve tissue was first fixed in paraformaldehyde (pH 7.4, 4 °C) overnight, soaked in sucrose (30%, 4 °C) for up to 3 days, embedded in TissueTek, and cut with a cryostat (Epredia CryoStar NX50). Transverse nerve sections (10 µm) were blocked with blocking reagent (20 min, room temperature) and stained with the above primary antibodies. Lipid droplets were visualized using BODIPY 493/503 (1:1000, Molecular Probes, D3922). Cell nuclei were counterstained with a fluorescence preservative mounting medium containing DAPI (4’,6-diamidino-2-phenylindole, Fluoromount-G, Invitrogen 00-4959-52). Fluorescence images were taken using a Zeiss AxioVision Apotome (Carl Zeiss).</p>
<p id="Par42">To quantify CXCL14, images were acquired at 20x magnification with a consistent exposure time. Up to eight images of up to eight perineurial regions were taken per patient (two CTRL and two CIAP patients). ImageJ v.1.54 f was used for analysis. The perineurium was manually outlined to determine its total area, and CXCL14-positive regions were identified using a color threshold with a brightness value of 100. The CXCL14-positive area was measured, and the percentage of CXCL14-positive area within the perineurium was calculated relative to the total perineurial area.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Perineurial thickness measurements</h3>
<p id="Par43">To quantify the thickness of the perineurium, we identified sural nerve histologies from 12 CIDP and 18 CTRL patients from three centers: University Hospital of Münster, Essen, and Leipzig (Supplementary Data <a href="#MOESM12" class="usa-link">10</a>). CTRL samples were residual material from sural nerve grafts. CIDP patients fulfilled the EAN/PNS criteria 2021 for CIDP<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>.</p>
<p id="Par44">In Münster, immunohistochemical detection of epithelial membrane antigen (EMA) was performed using an EMA antibody (Dako, clone E29, undiluted) on an automated Dako Omnis staining system. In Leipzig, automated immunostaining with the Benchmark Ultra Staining System and a primary antibody against EMA monoclonal antibody (Dako, clone E29, 1:50) were used. The slides were digitized using a Grundium Ocus 40 slide scanner microscope.</p>
<p id="Par45">The area and the perimeter of the perineurium were manually measured for each fascicle in ImageJ v1.54 d by manually outlining the inner and outer perimeter of the perineurium of each intact nerve fascicle in images of EMA-stained sural nerve biopsies by an investigator blinded towards patient identity. The area occupied by the perineurium was calculated as the difference between the outer area and the inner area. The ratio of the areas and perimeters was calculated and used for the analysis.</p>
<p id="Par46">To control for potential confounders, we used a linear mixed model with lme4 v1.1-35<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>, including age and sex as fixed effects and center and sample ID as random effects. The model was employed to determine the significance between the CIDP and CTRL groups and to adjust the perineurium measurements. To test for variance heterogeneity, we calculated the standard deviation of the adjusted perineurium measurements. We built an additional linear mixed model with lme4 v1.1-35, including age and sex as fixed effects and center as a random effect to assess the significance of the standard deviation between the CIDP and CTRL groups.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par47">Further information on research design is available in the <a href="#MOESM13" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (8.7MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62964_MOESM2_ESM.pdf</a><sup> (81.8KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (9.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (6.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (2.5MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (331KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (911.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 6</a><sup> (315.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 7</a><sup> (11.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 8</a><sup> (2.3MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM11_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 9</a><sup> (111.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM12"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM12_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 10</a><sup> (6.5MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM13"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM13_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (2.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM14"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM14_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (25.5KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We are deeply indebted to the patients for their participation. We thank Rebecca Ley from the Institute of Neuropathology Münster, Birgit Schmeddes and Sophie Linnenbaum from the Department of Neurology Münster, Susanne Weiche from the Dresden-concept Genome Center, and Kristina Wagner from the Department of Neurology in Essen for technical assistance. This work was supported by the Core EM and Histology, a core facility of the CMCB Technology Platform at the Technische Universität Dresden. Part of the calculations were performed on the high-performance computing (HPC) cluster PALMA II of the University of Münster, subsidized by the DFG (INST 211/667-1). This project was supported by the National Competence Network for Immune-Mediated Neuropathies in Germany (Kompetenznetz Peripherer Nerv). This project was mainly funded by a grant from the Bundesministerium für Bildung und Forschung (BMBF) ‘Lipid Immune Neuropathy Consortium’ (to G.M.z.H., M.S., R.S., R.F.) and a grant from the Interdisziplinäres Zentrum für Klinische Forschung (IZKF) Münster (SEED/016/21 to M.H.). In addition, GMzH was supported by grants from the Deutsche Forschungsgemeinschaft (DFG) (ME4050/12-1, ME4050/13-1, ME4050/8-1). This project was also supported by the DFG Sonderforschungsbereich Transregio 128 (to H.W.). J.A. was supported by the DFG Research Infrastructure NGS_CC (INST 269/768, project 407482635, DRESDEN-concept Genome Center). Open Access funding enabled and organized by Projekt DEAL.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>M.H. and G.M.z.H. conceived and supervised the study. M.H., J.T., A.K.M., M.M., M.S., A.-K.U., C.D., K.K., F.S., B.E., K.D., N.Ü., and C.S. were involved in collecting human sural nerves. J.T. performed nuclei extraction and snRNA-seq experiments. I.-N.L. carried out sequencing of snRNA-seq. J.W. planned the Xenium analysis. J.A. and A.D. performed Xenium experiments. F.D. determined g-ratios. M.M. measured the perineurium thickness. M.G., Christian T. and Carolina T. supervised perineurial stainings. M.H., N.G., and C.S. gathered sample and patient metadata. A.-L.B. and J.-K.S. performed RNA-ISH/IF and IH experiments. M.H. performed computational and statistical analyses. A.-L.B. designed the figures and schemes. J.M., R.S., R.F., H.W., C.S., and M.S. co-supervised the study. M.H., A.-L.B., J.W., and G.M.z.H. wrote the manuscript. All authors read and approved the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par48"><em>Nature Communications</em> thanks Maureen Su and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Funding</h2>
<p>Open Access funding enabled and organized by Projekt DEAL.</p></section><section id="notes4"><h2 class="pmc_sec_title">Data availability</h2>
<p>The raw and processed single-cell sequencing data with sample and cluster annotations and spatial data generated in this study have been deposited in the GEO database under accession code <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE285983" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE285983</a> and GSE <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE285984" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">285984</a>. An interactive version of the snRNA-seq data, created with cerebroAppLite v1.5.2<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> and an interactive version of the Xenium data, created with TissUUmaps v3.1<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>, are available at <a href="https://pns-atlas.mzhlab.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pns-atlas.mzhlab.com</a>. Source data are available on Zenodo (10.5281/zenodo.14226218)<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>.</p></section><section id="notes5"><h2 class="pmc_sec_title">Code availability</h2>
<p>The code is publicly available at <a href="https://github.com/mihem/pns_atlas" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/mihem/pns_atlas</a><span class="text-underline">)</span>. and has been archived on Zenodo <span class="text-underline">(</span><a href="https://zenodo.org/records/15750104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/15750104</a><span class="text-underline">)</span><sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. A Docker image is available to restore the coding environment. Figures are reproduced in a markdown document that automatically downloads all relevant source data from Zenodo<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> available at <a href="https://mihem.github.io/pns_atlas/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://mihem.github.io/pns_atlas/</a>.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par49">M.H., J.W. and G.M.z.H. have submitted a patent application for the diagnostic classification of PNP patients by single-nucleus transcriptomics of human nerve material. The remaining authors report no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Michael Heming, Anna-Lena Börsch, Jolien Wolbert.</p></div>
</div></section><section id="sec25"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62964-8.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Hanewinckel, R. et al. Prevalence of polyneuropathy in the general middle-aged and elderly population. <em>Neurology</em><strong>87</strong>, 1892–1898 (2016).
</cite> [<a href="https://doi.org/10.1212/WNL.0000000000003293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27683845/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hanewinckel,%20R.%20et%20al.%20Prevalence%20of%20polyneuropathy%20in%20the%20general%20middle-aged%20and%20elderly%20population.%20Neurology87,%201892%E2%80%931898%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Hoffman, E. M. et al. Impairments and comorbidities of polyneuropathy revealed by population-based analyses. <em>Neurology</em><strong>84</strong>, 1644–1651 (2015).
</cite> [<a href="https://doi.org/10.1212/WNL.0000000000001492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4409579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25832668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoffman,%20E.%20M.%20et%20al.%20Impairments%20and%20comorbidities%20of%20polyneuropathy%20revealed%20by%20population-based%20analyses.%20Neurology84,%201644%E2%80%931651%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Jennum, P., Ibsen, R. &amp; Kjellberg, J. Health, social and economic consequences of polyneuropathy: a controlled national study evaluating societal effects on patients and their partners. <em>Eur. Neurol.</em><strong>73</strong>, 81–88 (2015).
</cite> [<a href="https://doi.org/10.1159/000367653" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25401731/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jennum,%20P.,%20Ibsen,%20R.%20&amp;%20Kjellberg,%20J.%20Health,%20social%20and%20economic%20consequences%20of%20polyneuropathy:%20a%20controlled%20national%20study%20evaluating%20societal%20effects%20on%20patients%20and%20their%20partners.%20Eur.%20Neurol.73,%2081%E2%80%9388%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Visser, N. A., Notermans, N. C., Linssen, R. S. N., van den Berg, L. H. &amp; Vrancken, A. F. J. E. Incidence of polyneuropathy in Utrecht, the Netherlands. <em>Neurology</em><strong>84</strong>, 259–264 (2015).
</cite> [<a href="https://doi.org/10.1212/WNL.0000000000001160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25503982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Visser,%20N.%20A.,%20Notermans,%20N.%20C.,%20Linssen,%20R.%20S.%20N.,%20van%20den%20Berg,%20L.%20H.%20&amp;%20Vrancken,%20A.%20F.%20J.%20E.%20Incidence%20of%20polyneuropathy%20in%20Utrecht,%20the%20Netherlands.%20Neurology84,%20259%E2%80%93264%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Hanewinckel, R., van Oijen, M., Ikram, M. A. &amp; van Doorn, P. A. The epidemiology and risk factors of chronic polyneuropathy. <em>Eur. J. Epidemiol.</em><strong>31</strong>, 5–20 (2016).
</cite> [<a href="https://doi.org/10.1007/s10654-015-0094-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4756033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26700499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hanewinckel,%20R.,%20van%20Oijen,%20M.,%20Ikram,%20M.%20A.%20&amp;%20van%20Doorn,%20P.%20A.%20The%20epidemiology%20and%20risk%20factors%20of%20chronic%20polyneuropathy.%20Eur.%20J.%20Epidemiol.31,%205%E2%80%9320%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Verghese, J., Bieri, P. L., Gellido, C., Schaumburg, H. H. &amp; Herskovitz, S. Peripheral neuropathy in young-old and old-old patients. <em>Muscle Nerve</em><strong>24</strong>, 1476–1481 (2001).
</cite> [<a href="https://doi.org/10.1002/mus.1171" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11745949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Verghese,%20J.,%20Bieri,%20P.%20L.,%20Gellido,%20C.,%20Schaumburg,%20H.%20H.%20&amp;%20Herskovitz,%20S.%20Peripheral%20neuropathy%20in%20young-old%20and%20old-old%20patients.%20Muscle%20Nerve24,%201476%E2%80%931481%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Prada, V. et al. Diagnostic value of sural nerve biopsy: retrospective analysis of clinical cases from 1981 to 2017. <em>Front. Neurol.</em><strong>10</strong>, 1218 (2019).
</cite> [<a href="https://doi.org/10.3389/fneur.2019.01218" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6884026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31824401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prada,%20V.%20et%20al.%20Diagnostic%20value%20of%20sural%20nerve%20biopsy:%20retrospective%20analysis%20of%20clinical%20cases%20from%201981%20to%202017.%20Front.%20Neurol.10,%201218%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Wolbert, J. et al. Redefining the heterogeneity of peripheral nerve cells in health and autoimmunity. <em>Proc. Natl. Acad. Sci. USA</em><strong>117</strong>, 9466–9476 (2020).
</cite> [<a href="https://doi.org/10.1073/pnas.1912139117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7196786/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32295886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolbert,%20J.%20et%20al.%20Redefining%20the%20heterogeneity%20of%20peripheral%20nerve%20cells%20in%20health%20and%20autoimmunity.%20Proc.%20Natl.%20Acad.%20Sci.%20USA117,%209466%E2%80%939476%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Yim, A. K. Y. et al. Disentangling glial diversity in peripheral nerves at single-nuclei resolution. <em>Nat. Neurosci.</em><strong>25</strong>, 238–251 (2022).
</cite> [<a href="https://doi.org/10.1038/s41593-021-01005-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9060899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35115729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yim,%20A.%20K.%20Y.%20et%20al.%20Disentangling%20glial%20diversity%20in%20peripheral%20nerves%20at%20single-nuclei%20resolution.%20Nat.%20Neurosci.25,%20238%E2%80%93251%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Gerber, D. et al. Transcriptional profiling of mouse peripheral nerves to the single-cell level to build a sciatic nerve ATlas (SNAT). <em>ELife</em><strong>10</strong>, e58591 (2021).
</cite> [<a href="https://doi.org/10.7554/eLife.58591" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8064760/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33890853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gerber,%20D.%20et%20al.%20Transcriptional%20profiling%20of%20mouse%20peripheral%20nerves%20to%20the%20single-cell%20level%20to%20build%20a%20sciatic%20nerve%20ATlas%20(SNAT).%20ELife10,%20e58591%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Carr, M. J. et al. Mesenchymal precursor cells in adult nerves contribute to mammalian tissue repair and regeneration. <em>Cell Stem Cell</em><strong>24</strong>, 240–256 (2019).
</cite> [<a href="https://doi.org/10.1016/j.stem.2018.10.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30503141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carr,%20M.%20J.%20et%20al.%20Mesenchymal%20precursor%20cells%20in%20adult%20nerves%20contribute%20to%20mammalian%20tissue%20repair%20and%20regeneration.%20Cell%20Stem%20Cell24,%20240%E2%80%93256%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Toma, J. S. et al. Peripheral nerve single-cell analysis identifies mesenchymal ligands that promote axonal growth. <em>eNeuro</em><strong>7</strong>, ENEURO.0066–20.2020 (2020).
</cite> [<a href="https://doi.org/10.1523/ENEURO.0066-20.2020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7294463/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32349983/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Toma,%20J.%20S.%20et%20al.%20Peripheral%20nerve%20single-cell%20analysis%20identifies%20mesenchymal%20ligands%20that%20promote%20axonal%20growth.%20eNeuro7,%20ENEURO.0066%E2%80%9320.2020%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Lovatt, D. et al. scRNA-seq generates a molecular map of emerging cell subtypes after sciatic nerve injury in rats. <em>Commun. Biol.</em><strong>5</strong>, 1105 (2022).
</cite> [<a href="https://doi.org/10.1038/s42003-022-03970-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9581950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36261573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lovatt,%20D.%20et%20al.%20scRNA-seq%20generates%20a%20molecular%20map%20of%20emerging%20cell%20subtypes%20after%20sciatic%20nerve%20injury%20in%20rats.%20Commun.%20Biol.5,%201105%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Garcia, F. J. et al. Single-cell dissection of the human brain vasculature. <em>Nature</em><strong>603</strong>, 893–899 (2022).
</cite> [<a href="https://doi.org/10.1038/s41586-022-04521-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9680899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35158371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garcia,%20F.%20J.%20et%20al.%20Single-cell%20dissection%20of%20the%20human%20brain%20vasculature.%20Nature603,%20893%E2%80%93899%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Arthur-Farraj, P. J. et al. Changes in the coding and non-coding transcriptome and DNA methylome that define the Schwann cell repair phenotype after nerve injury. <em>Cell Rep.</em><strong>20</strong>, 2719–2734 (2017).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.08.064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5608958/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28903050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arthur-Farraj,%20P.%20J.%20et%20al.%20Changes%20in%20the%20coding%20and%20non-coding%20transcriptome%20and%20DNA%20methylome%20that%20define%20the%20Schwann%20cell%20repair%20phenotype%20after%20nerve%20injury.%20Cell%20Rep.20,%202719%E2%80%932734%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Palladino, S. P. et al. The human blood-nerve barrier transcriptome. <em>Sci. Rep.</em><strong>7</strong>, 17477 (2017).
</cite> [<a href="https://doi.org/10.1038/s41598-017-17475-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5727190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29234067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palladino,%20S.%20P.%20et%20al.%20The%20human%20blood-nerve%20barrier%20transcriptome.%20Sci.%20Rep.7,%2017477%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Wood, C. A. P. et al. Structure and mechanism of blood-brain-barrier lipid transporter MFSD2A. <em>Nature</em><strong>596</strong>, 444–448 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03782-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8884080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34349262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wood,%20C.%20A.%20P.%20et%20al.%20Structure%20and%20mechanism%20of%20blood-brain-barrier%20lipid%20transporter%20MFSD2A.%20Nature596,%20444%E2%80%93448%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Janesick, A. et al. High resolution mapping of the tumor microenvironment using integrated single-cell, spatial and in situ analysis. <em>Nat. Commun.</em><strong>14</strong>, 8353 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-43458-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10730913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38114474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Janesick,%20A.%20et%20al.%20High%20resolution%20mapping%20of%20the%20tumor%20microenvironment%20using%20integrated%20single-cell,%20spatial%20and%20in%20situ%20analysis.%20Nat.%20Commun.14,%208353%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Chen, B., Banton, M. C., Singh, L., Parkinson, D. B. &amp; Dun, X.-P. Single cell transcriptome data analysis defines the heterogeneity of peripheral nerve cells in homeostasis and regeneration. <em>Front. Cell. Neurosci.</em><strong>15</strong>, 624826 (2021).
</cite> [<a href="https://doi.org/10.3389/fncel.2021.624826" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8019921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33828460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20B.,%20Banton,%20M.%20C.,%20Singh,%20L.,%20Parkinson,%20D.%20B.%20&amp;%20Dun,%20X.-P.%20Single%20cell%20transcriptome%20data%20analysis%20defines%20the%20heterogeneity%20of%20peripheral%20nerve%20cells%20in%20homeostasis%20and%20regeneration.%20Front.%20Cell.%20Neurosci.15,%20624826%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Kalinski, A. L. et al. Analysis of the immune response to sciatic nerve injury identifies efferocytosis as a key mechanism of nerve debridement. <em>ELife</em><strong>9</strong>, e60223 (2020).
</cite> [<a href="https://doi.org/10.7554/eLife.60223" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7735761/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33263277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kalinski,%20A.%20L.%20et%20al.%20Analysis%20of%20the%20immune%20response%20to%20sciatic%20nerve%20injury%20identifies%20efferocytosis%20as%20a%20key%20mechanism%20of%20nerve%20debridement.%20ELife9,%20e60223%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Ydens, E. et al. Profiling peripheral nerve macrophages reveals two macrophage subsets with distinct localization, transcriptome and response to injury. <em>Nat. Neurosci.</em><strong>23</strong>, 676–689 (2020).
</cite> [<a href="https://doi.org/10.1038/s41593-020-0618-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7611025/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32284604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ydens,%20E.%20et%20al.%20Profiling%20peripheral%20nerve%20macrophages%20reveals%20two%20macrophage%20subsets%20with%20distinct%20localization,%20transcriptome%20and%20response%20to%20injury.%20Nat.%20Neurosci.23,%20676%E2%80%93689%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Wang, P. L. et al. Peripheral nerve resident macrophages share tissue-specific programming and features of activated microglia. <em>Nat. Commun.</em><strong>11</strong>, 2552 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-16355-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7242366/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32439942/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20P.%20L.%20et%20al.%20Peripheral%20nerve%20resident%20macrophages%20share%20tissue-specific%20programming%20and%20features%20of%20activated%20microglia.%20Nat.%20Commun.11,%202552%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Xu, R. et al. Lipid-associated macrophages between aggravation and alleviation of metabolic diseases. <em>Trends Endocrinol. Metab.</em><strong>35</strong>, 981–995 (2024).
</cite> [<a href="https://doi.org/10.1016/j.tem.2024.04.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38705759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20R.%20et%20al.%20Lipid-associated%20macrophages%20between%20aggravation%20and%20alleviation%20of%20metabolic%20diseases.%20Trends%20Endocrinol.%20Metab.35,%20981%E2%80%93995%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Beuker, C. et al. Stroke induces disease-specific myeloid cells in the brain parenchyma and pia. <em>Nat. Commun.</em><strong>13</strong>, 945 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-28593-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8854573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35177618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beuker,%20C.%20et%20al.%20Stroke%20induces%20disease-specific%20myeloid%20cells%20in%20the%20brain%20parenchyma%20and%20pia.%20Nat.%20Commun.13,%20945%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Gomez-Sanchez, J. A. et al. After nerve injury, lineage tracing shows that myelin and remak schwann cells elongate extensively and branch to form repair schwann cells, which shorten radically on remyelination. <em>J. Neurosci.</em><strong>37</strong>, 9086–9099 (2017).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.1453-17.2017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5597985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28904214/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gomez-Sanchez,%20J.%20A.%20et%20al.%20After%20nerve%20injury,%20lineage%20tracing%20shows%20that%20myelin%20and%20remak%20schwann%20cells%20elongate%20extensively%20and%20branch%20to%20form%20repair%20schwann%20cells,%20which%20shorten%20radically%20on%20remyelination.%20J.%20Neurosci.37,%209086%E2%80%939099%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Norrmén, C. et al. mTORC1 Is Transiently Reactivated in Injured Nerves to Promote c-Jun Elevation and Schwann Cell Dedifferentiation. <em>J. Neurosci.</em><strong>38</strong>, 4811–4828 (2018).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.3619-17.2018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5956991/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29695414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Norrm%C3%A9n,%20C.%20et%20al.%20mTORC1%20Is%20Transiently%20Reactivated%20in%20Injured%20Nerves%20to%20Promote%20c-Jun%20Elevation%20and%20Schwann%20Cell%20Dedifferentiation.%20J.%20Neurosci.38,%204811%E2%80%934828%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Szepanowski, F. et al. LPA1 signaling drives Schwann cell dedifferentiation in experimental autoimmune neuritis. <em>J. Neuroinflammation</em><strong>18</strong>, 293 (2021).
</cite> [<a href="https://doi.org/10.1186/s12974-021-02350-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8680309/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34920725/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Szepanowski,%20F.%20et%20al.%20LPA1%20signaling%20drives%20Schwann%20cell%20dedifferentiation%20in%20experimental%20autoimmune%20neuritis.%20J.%20Neuroinflammation18,%20293%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Perentes, E., Nakagawa, Y., Ross, G. W., Stanton, C. &amp; Rubinstein, L. J. Expression of epithelial membrane antigen in perineurial cells and their derivatives. An immunohistochemical study with multiple markers. <em>Acta Neuropathol.</em><strong>75</strong>, 160–165 (1987).
</cite> [<a href="https://doi.org/10.1007/BF00687077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3434224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perentes,%20E.,%20Nakagawa,%20Y.,%20Ross,%20G.%20W.,%20Stanton,%20C.%20&amp;%20Rubinstein,%20L.%20J.%20Expression%20of%20epithelial%20membrane%20antigen%20in%20perineurial%20cells%20and%20their%20derivatives.%20An%20immunohistochemical%20study%20with%20multiple%20markers.%20Acta%20Neuropathol.75,%20160%E2%80%93165%20(1987)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Heming, M., Börsch, A.-L., Wiendl, H. &amp; Meyer Zu Hörste, G. High-dimensional investigation of the cerebrospinal fluid to explore and monitor CNS immune responses. <em>Genome Med.</em><strong>14</strong>, 94 (2022).
</cite> [<a href="https://doi.org/10.1186/s13073-022-01097-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9385102/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35978442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heming,%20M.,%20B%C3%B6rsch,%20A.-L.,%20Wiendl,%20H.%20&amp;%20Meyer%20Zu%20H%C3%B6rste,%20G.%20High-dimensional%20investigation%20of%20the%20cerebrospinal%20fluid%20to%20explore%20and%20monitor%20CNS%20immune%20responses.%20Genome%20Med.14,%2094%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Clark, I. C. et al. Identification of astrocyte regulators by nucleic acid cytometry. <em>Nature</em><strong>614</strong>, 326–333 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-022-05613-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9980163/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36599367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clark,%20I.%20C.%20et%20al.%20Identification%20of%20astrocyte%20regulators%20by%20nucleic%20acid%20cytometry.%20Nature614,%20326%E2%80%93333%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Cattin, M.-E. et al. Expression of murine muscle-enriched A-type lamin-interacting protein (MLIP) is regulated by tissue-specific alternative transcription start sites. <em>J. Biol. Chem.</em><strong>293</strong>, 19761–19770 (2018).
</cite> [<a href="https://doi.org/10.1074/jbc.RA118.003758" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6314141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30389785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cattin,%20M.-E.%20et%20al.%20Expression%20of%20murine%20muscle-enriched%20A-type%20lamin-interacting%20protein%20(MLIP)%20is%20regulated%20by%20tissue-specific%20alternative%20transcription%20start%20sites.%20J.%20Biol.%20Chem.293,%2019761%E2%80%9319770%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Petrov, K. A. et al. Schwann cells sense and control acetylcholine spillover at the neuromuscular junction by α7 nicotinic receptors and butyrylcholinesterase. <em>J. Neurosci.</em><strong>34</strong>, 11870–11883 (2014).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.0329-14.2014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6608466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25186736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Petrov,%20K.%20A.%20et%20al.%20Schwann%20cells%20sense%20and%20control%20acetylcholine%20spillover%20at%20the%20neuromuscular%20junction%20by%20%CE%B17%20nicotinic%20receptors%20and%20butyrylcholinesterase.%20J.%20Neurosci.34,%2011870%E2%80%9311883%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Strutz, N. et al. Identification of domains and amino acids involved in GLuR7 ion channel function. <em>J. Neurosci.</em><strong>21</strong>, 401–411 (2001).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.21-02-00401.2001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6763793/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11160421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Strutz,%20N.%20et%20al.%20Identification%20of%20domains%20and%20amino%20acids%20involved%20in%20GLuR7%20ion%20channel%20function.%20J.%20Neurosci.21,%20401%E2%80%93411%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Vedi, M. et al. 2022 updates to the Rat Genome Database: a Findable, Accessible, Interoperable, and Reusable (FAIR) resource. <em>Genetics</em><strong>224</strong>, iyad042 (2023).
</cite> [<a href="https://doi.org/10.1093/genetics/iyad042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10474928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36930729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vedi,%20M.%20et%20al.%202022%20updates%20to%20the%20Rat%20Genome%20Database:%20a%20Findable,%20Accessible,%20Interoperable,%20and%20Reusable%20(FAIR)%20resource.%20Genetics224,%20iyad042%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Li, Z., Li, Y. &amp; Jiao, J. Neural progenitor cells mediated by H2A.Z.2 regulate microglial development via Cxcl14 in the embryonic brain. <em>Proc. Natl. Acad. Sci. USA</em><strong>116</strong>, 24122–24132 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1913978116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6883828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31712428/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Z.,%20Li,%20Y.%20&amp;%20Jiao,%20J.%20Neural%20progenitor%20cells%20mediated%20by%20H2A.Z.2%20regulate%20microglial%20development%20via%20Cxcl14%20in%20the%20embryonic%20brain.%20Proc.%20Natl.%20Acad.%20Sci.%20USA116,%2024122%E2%80%9324132%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Ojeda, A. F., Munjaal, R. P. &amp; Lwigale, P. Y. Knockdown of CXCL14 disrupts neurovascular patterning during ocular development. <em>Dev. Biol.</em><strong>423</strong>, 77–91 (2017).
</cite> [<a href="https://doi.org/10.1016/j.ydbio.2017.01.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5334790/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28095300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ojeda,%20A.%20F.,%20Munjaal,%20R.%20P.%20&amp;%20Lwigale,%20P.%20Y.%20Knockdown%20of%20CXCL14%20disrupts%20neurovascular%20patterning%20during%20ocular%20development.%20Dev.%20Biol.423,%2077%E2%80%9391%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Ubogu, E. E. Biology of the human blood-nerve barrier in health and disease. <em>Exp. Neurol.</em><strong>328</strong>, 113272 (2020).
</cite> [<a href="https://doi.org/10.1016/j.expneurol.2020.113272" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7145763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32142802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ubogu,%20E.%20E.%20Biology%20of%20the%20human%20blood-nerve%20barrier%20in%20health%20and%20disease.%20Exp.%20Neurol.328,%20113272%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Barbaria, E. M. et al. The alpha-chemokine CXCL14 is up-regulated in the sciatic nerve of a mouse model of Charcot-Marie-Tooth disease type 1A and alters myelin gene expression in cultured Schwann cells. <em>Neurobiol. Dis.</em><strong>33</strong>, 448–458 (2009).
</cite> [<a href="https://doi.org/10.1016/j.nbd.2008.11.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19111616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barbaria,%20E.%20M.%20et%20al.%20The%20alpha-chemokine%20CXCL14%20is%20up-regulated%20in%20the%20sciatic%20nerve%20of%20a%20mouse%20model%20of%20Charcot-Marie-Tooth%20disease%20type%201A%20and%20alters%20myelin%20gene%20expression%20in%20cultured%20Schwann%20cells.%20Neurobiol.%20Dis.33,%20448%E2%80%93458%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Hill, R. E. &amp; Williams, R. E. A quantitative analysis of perineurial cell basement membrane collagen IV, laminin and fibronectin in diabetic and non-diabetic human sural nerve. <em>J. Anat.</em><strong>201</strong>, 185–192 (2002).
</cite> [<a href="https://doi.org/10.1046/j.1469-7580.2002.00083.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1570905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12220126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hill,%20R.%20E.%20&amp;%20Williams,%20R.%20E.%20A%20quantitative%20analysis%20of%20perineurial%20cell%20basement%20membrane%20collagen%20IV,%20laminin%20and%20fibronectin%20in%20diabetic%20and%20non-diabetic%20human%20sural%20nerve.%20J.%20Anat.201,%20185%E2%80%93192%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Peltonen, S., Alanne, M. &amp; Peltonen, J. Barriers of the peripheral nerve. <em>Tissue Barriers</em><strong>1</strong>, e24956 (2013).
</cite> [<a href="https://doi.org/10.4161/tisb.24956" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3867511/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24665400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peltonen,%20S.,%20Alanne,%20M.%20&amp;%20Peltonen,%20J.%20Barriers%20of%20the%20peripheral%20nerve.%20Tissue%20Barriers1,%20e24956%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Andreis, F. R. et al. On the relationship between fascicle diameter and perineurium thickness in the ulnar nerve of pigs. In <em>Proceedings</em><em>2023</em><em>11th International IEEE/EMBS Conference on Neural Engineering (NER)</em> 1–4 (IEEE, 2023).</cite>
</li>
<li id="CR42">
<span class="label">42.</span><cite>Hill, R. E. &amp; Williams, P. E. Perineurial cell basement membrane thickening and myelinated nerve fibre loss in diabetic and nondiabetic peripheral nerve. <em>J. Neurol. Sci.</em><strong>217</strong>, 157–163 (2004).
</cite> [<a href="https://doi.org/10.1016/j.jns.2003.09.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14706218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hill,%20R.%20E.%20&amp;%20Williams,%20P.%20E.%20Perineurial%20cell%20basement%20membrane%20thickening%20and%20myelinated%20nerve%20fibre%20loss%20in%20diabetic%20and%20nondiabetic%20peripheral%20nerve.%20J.%20Neurol.%20Sci.217,%20157%E2%80%93163%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Schäfers, M., Neukirchen, S., Toyka, K. V. &amp; Sommer, C. Diagnostic value of sural nerve biopsy in patients with suspected Borrelia neuropathy. <em>J. Peripher. Nerv. Syst.</em><strong>13</strong>, 81–91 (2008).
</cite> [<a href="https://doi.org/10.1111/j.1529-8027.2008.00161.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18346234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sch%C3%A4fers,%20M.,%20Neukirchen,%20S.,%20Toyka,%20K.%20V.%20&amp;%20Sommer,%20C.%20Diagnostic%20value%20of%20sural%20nerve%20biopsy%20in%20patients%20with%20suspected%20Borrelia%20neuropathy.%20J.%20Peripher.%20Nerv.%20Syst.13,%2081%E2%80%9391%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Collins, M. P., Hadden, R. D. M. &amp; Shahnoor, N. Primary perineuritis, a rare but treatable neuropathy: Review of perineurial anatomy, clinicopathological features, and differential diagnosis. <em>Muscle Nerve</em><strong>68</strong>, 696–713 (2023).
</cite> [<a href="https://doi.org/10.1002/mus.27949" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37602939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Collins,%20M.%20P.,%20Hadden,%20R.%20D.%20M.%20&amp;%20Shahnoor,%20N.%20Primary%20perineuritis,%20a%20rare%20but%20treatable%20neuropathy:%20Review%20of%20perineurial%20anatomy,%20clinicopathological%20features,%20and%20differential%20diagnosis.%20Muscle%20Nerve68,%20696%E2%80%93713%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Jaitin, D. A. et al. Lipid-associated macrophages control metabolic homeostasis in a Trem2-dependent manner. <em>Cell</em><strong>178</strong>, 686–698 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.05.054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7068689/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31257031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jaitin,%20D.%20A.%20et%20al.%20Lipid-associated%20macrophages%20control%20metabolic%20homeostasis%20in%20a%20Trem2-dependent%20manner.%20Cell178,%20686%E2%80%93698%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Liu, P. et al. Role of macrophages in peripheral nerve injury and repair. <em>Neural Regen. Res.</em><strong>14</strong>, 1335–1342 (2019).
</cite> [<a href="https://doi.org/10.4103/1673-5374.253510" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6524518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30964051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20P.%20et%20al.%20Role%20of%20macrophages%20in%20peripheral%20nerve%20injury%20and%20repair.%20Neural%20Regen.%20Res.14,%201335%E2%80%931342%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Yoo, Y., Neumayer, G., Shibuya, Y., Mader, M. M.-D. &amp; Wernig, M. A cell therapy approach to restore microglial Trem2 function in a mouse model of Alzheimer’s disease. <em>Cell Stem Cell</em><strong>30</strong>, 1043–1053 (2023).
</cite> [<a href="https://doi.org/10.1016/j.stem.2023.07.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37541210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoo,%20Y.,%20Neumayer,%20G.,%20Shibuya,%20Y.,%20Mader,%20M.%20M.-D.%20&amp;%20Wernig,%20M.%20A%20cell%20therapy%20approach%20to%20restore%20microglial%20Trem2%20function%20in%20a%20mouse%20model%20of%20Alzheimer%E2%80%99s%20disease.%20Cell%20Stem%20Cell30,%201043%E2%80%931053%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Morenas-Rodríguez, E. et al. Soluble TREM2 in CSF and its association with other biomarkers and cognition in autosomal-dominant Alzheimer’s disease: a longitudinal observational study. <em>Lancet Neurol.</em><strong>21</strong>, 329–341 (2022).
</cite> [<a href="https://doi.org/10.1016/S1474-4422(22)00027-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8926925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35305339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morenas-Rodr%C3%ADguez,%20E.%20et%20al.%20Soluble%20TREM2%20in%20CSF%20and%20its%20association%20with%20other%20biomarkers%20and%20cognition%20in%20autosomal-dominant%20Alzheimer%E2%80%99s%20disease:%20a%20longitudinal%20observational%20study.%20Lancet%20Neurol.21,%20329%E2%80%93341%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Van den Bergh, P. Y. K. et al. European Academy of Neurology/Peripheral Nerve Society guideline on diagnosis and treatment of chronic inflammatory demyelinating polyradiculoneuropathy: Report of a joint task force-second revision. <em>Eur. J. Neurol.</em><strong>28</strong>, 3556–3583 (2021).
</cite> [<a href="https://doi.org/10.1111/ene.14959" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34327760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20den%20Bergh,%20P.%20Y.%20K.%20et%20al.%20European%20Academy%20of%20Neurology/Peripheral%20Nerve%20Society%20guideline%20on%20diagnosis%20and%20treatment%20of%20chronic%20inflammatory%20demyelinating%20polyradiculoneuropathy:%20Report%20of%20a%20joint%20task%20force-second%20revision.%20Eur.%20J.%20Neurol.28,%203556%E2%80%933583%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Pool, A.-H., Poldsam, H., Chen, S., Thomson, M. &amp; Oka, Y. Recovery of missing single-cell RNA-sequencing data with optimized transcriptomic references. <em>Nat. Methods</em><strong>20</strong>, 1506–1515 (2023).
</cite> [<a href="https://doi.org/10.1038/s41592-023-02003-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37697162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pool,%20A.-H.,%20Poldsam,%20H.,%20Chen,%20S.,%20Thomson,%20M.%20&amp;%20Oka,%20Y.%20Recovery%20of%20missing%20single-cell%20RNA-sequencing%20data%20with%20optimized%20transcriptomic%20references.%20Nat.%20Methods20,%201506%E2%80%931515%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Fleming, S. J. et al. Unsupervised removal of systematic background noise from droplet-based single-cell experiments using CellBender. <em>Nat. Methods</em><strong>20</strong>, 1323–1335 (2023).
</cite> [<a href="https://doi.org/10.1038/s41592-023-01943-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37550580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fleming,%20S.%20J.%20et%20al.%20Unsupervised%20removal%20of%20systematic%20background%20noise%20from%20droplet-based%20single-cell%20experiments%20using%20CellBender.%20Nat.%20Methods20,%201323%E2%80%931335%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Hao, Y. et al. Dictionary learning for integrative, multimodal and scalable single-cell analysis. <em>Nat. Biotechnol.</em><strong>42</strong>, 293–304 (2024).
</cite> [<a href="https://doi.org/10.1038/s41587-023-01767-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10928517/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37231261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hao,%20Y.%20et%20al.%20Dictionary%20learning%20for%20integrative,%20multimodal%20and%20scalable%20single-cell%20analysis.%20Nat.%20Biotechnol.42,%20293%E2%80%93304%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Germain, P.-L., Lun, A., Garcia Meixide, C., Macnair, W. &amp; Robinson, M. D. Doublet identification in single-cell sequencing data using scDblFinder. <em>F1000Res.</em><strong>10</strong>, 979 (2021).
</cite> [<a href="https://doi.org/10.12688/f1000research.73600.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9204188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35814628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Germain,%20P.-L.,%20Lun,%20A.,%20Garcia%20Meixide,%20C.,%20Macnair,%20W.%20&amp;%20Robinson,%20M.%20D.%20Doublet%20identification%20in%20single-cell%20sequencing%20data%20using%20scDblFinder.%20F1000Res.10,%20979%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Lopez, R., Regier, J., Cole, M. B., Jordan, M. I. &amp; Yosef, N. Deep generative modeling for single-cell transcriptomics. <em>Nat. Methods</em><strong>15</strong>, 1053–1058 (2018).
</cite> [<a href="https://doi.org/10.1038/s41592-018-0229-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6289068/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30504886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lopez,%20R.,%20Regier,%20J.,%20Cole,%20M.%20B.,%20Jordan,%20M.%20I.%20&amp;%20Yosef,%20N.%20Deep%20generative%20modeling%20for%20single-cell%20transcriptomics.%20Nat.%20Methods15,%201053%E2%80%931058%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. <em>Nucleic Acids Res.</em><strong>44</strong>, W90–W97 (2016).
</cite> [<a href="https://doi.org/10.1093/nar/gkw377" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4987924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27141961/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuleshov,%20M.%20V.%20et%20al.%20Enrichr:%20a%20comprehensive%20gene%20set%20enrichment%20analysis%20web%20server%202016%20update.%20Nucleic%20Acids%20Res.44,%20W90%E2%80%93W97%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Squair, J. W. et al. Confronting false discoveries in single-cell differential expression. <em>Nat. Commun.</em><strong>12</strong>, 5692 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-25960-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8479118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34584091/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Squair,%20J.%20W.%20et%20al.%20Confronting%20false%20discoveries%20in%20single-cell%20differential%20expression.%20Nat.%20Commun.12,%205692%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. <em>Bioinformatics</em><strong>26</strong>, 139–140 (2010).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2796818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Robinson,%20M.%20D.,%20McCarthy,%20D.%20J.%20&amp;%20Smyth,%20G.%20K.%20edgeR:%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data.%20Bioinformatics26,%20139%E2%80%93140%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Missarova, A., Dann, E., Rosen, L., Satija, R. &amp; Marioni, J. Leveraging neighborhood representations of single-cell data to achieve sensitive DE testing with miloDE. <em>Genome Biol.</em><strong>25</strong>, 189 (2024).
</cite> [<a href="https://doi.org/10.1186/s13059-024-03334-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11256449/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39026254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Missarova,%20A.,%20Dann,%20E.,%20Rosen,%20L.,%20Satija,%20R.%20&amp;%20Marioni,%20J.%20Leveraging%20neighborhood%20representations%20of%20single-cell%20data%20to%20achieve%20sensitive%20DE%20testing%20with%20miloDE.%20Genome%20Biol.25,%20189%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Phipson, B. et al. propeller: testing for differences in cell type proportions in single cell data. <em>Bioinformatics</em><strong>38</strong>, 4720–4726 (2022).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btac582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9563678/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36005887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Phipson,%20B.%20et%20al.%20propeller:%20testing%20for%20differences%20in%20cell%20type%20proportions%20in%20single%20cell%20data.%20Bioinformatics38,%204720%E2%80%934726%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Reshef, Y. A. et al. Co-varying neighborhood analysis identifies cell populations associated with phenotypes of interest from single-cell transcriptomics. <em>Nat. Biotechnol.</em><strong>40</strong>, 355–363 (2022).
</cite> [<a href="https://doi.org/10.1038/s41587-021-01066-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8930733/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34675423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reshef,%20Y.%20A.%20et%20al.%20Co-varying%20neighborhood%20analysis%20identifies%20cell%20populations%20associated%20with%20phenotypes%20of%20interest%20from%20single-cell%20transcriptomics.%20Nat.%20Biotechnol.40,%20355%E2%80%93363%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Dimitrov, D. et al. Comparison of methods and resources for cell-cell communication inference from single-cell RNA-Seq data. <em>Nat. Commun.</em><strong>13</strong>, 3224 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-30755-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9184522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35680885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dimitrov,%20D.%20et%20al.%20Comparison%20of%20methods%20and%20resources%20for%20cell-cell%20communication%20inference%20from%20single-cell%20RNA-Seq%20data.%20Nat.%20Commun.13,%203224%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Bates, D., Mächler, M., Bolker, B. &amp; Walker, S. Fitting linear mixed-effects models using lme4. <em>J. Stat. Softw.</em><strong>67</strong>, 1–48 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bates,%20D.,%20M%C3%A4chler,%20M.,%20Bolker,%20B.%20&amp;%20Walker,%20S.%20Fitting%20linear%20mixed-effects%20models%20using%20lme4.%20J.%20Stat.%20Softw.67,%201%E2%80%9348%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Hillje, R., Pelicci, P. G. &amp; Luzi, L. Cerebro: interactive visualization of scRNA-seq data. <em>Bioinformatics</em><strong>36</strong>, 2311–2313 (2020).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz877" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7141853/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31764967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hillje,%20R.,%20Pelicci,%20P.%20G.%20&amp;%20Luzi,%20L.%20Cerebro:%20interactive%20visualization%20of%20scRNA-seq%20data.%20Bioinformatics36,%202311%E2%80%932313%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Pielawski, N. et al. TissUUmaps 3: Improvements in interactive visualization, exploration, and quality assessment of large-scale spatial omics data. <em>Heliyon</em><strong>9</strong>, e15306 (2023).
</cite> [<a href="https://doi.org/10.1016/j.heliyon.2023.e15306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10149187/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37131430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pielawski,%20N.%20et%20al.%20TissUUmaps%203:%20Improvements%20in%20interactive%20visualization,%20exploration,%20and%20quality%20assessment%20of%20large-scale%20spatial%20omics%20data.%20Heliyon9,%20e15306%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Heming, M. Source Data: Multi-omic identification of perineurial hyperplasia and lipid-associated nerve macrophages in human polyneuropathies. <em>Zenodo</em>10.5281/zenodo.14226218 (2024)</cite> [<a href="https://doi.org/10.1038/s41467-025-62964-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12375038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40849297/" class="usa-link">PubMed</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Heming, M. Code: Multi-omic identification of perineurial hyperplasia and lipid-associated nerve macrophages in human polyneuropathies. <em>Zenodo</em>10.5281/zenodo.15750104 (2025).</cite> [<a href="https://doi.org/10.1038/s41467-025-62964-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12375038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40849297/" class="usa-link">PubMed</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (8.7MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62964_MOESM2_ESM.pdf</a><sup> (81.8KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (9.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (6.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (2.5MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (331KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (911.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 6</a><sup> (315.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 7</a><sup> (11.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 8</a><sup> (2.3MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM11_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 9</a><sup> (111.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM12_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 10</a><sup> (6.5MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM13_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (2.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material14_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375038/bin/41467_2025_62964_MOESM14_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (25.5KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The raw and processed single-cell sequencing data with sample and cluster annotations and spatial data generated in this study have been deposited in the GEO database under accession code <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE285983" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE285983</a> and GSE <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE285984" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">285984</a>. An interactive version of the snRNA-seq data, created with cerebroAppLite v1.5.2<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> and an interactive version of the Xenium data, created with TissUUmaps v3.1<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>, are available at <a href="https://pns-atlas.mzhlab.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pns-atlas.mzhlab.com</a>. Source data are available on Zenodo (10.5281/zenodo.14226218)<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>.</p>
<p>The code is publicly available at <a href="https://github.com/mihem/pns_atlas" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/mihem/pns_atlas</a><span class="text-underline">)</span>. and has been archived on Zenodo <span class="text-underline">(</span><a href="https://zenodo.org/records/15750104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/15750104</a><span class="text-underline">)</span><sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. A Docker image is available to restore the coding environment. Figures are reproduced in a markdown document that automatically downloads all relevant source data from Zenodo<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> available at <a href="https://mihem.github.io/pns_atlas/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://mihem.github.io/pns_atlas/</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62964-8"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62964.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375038/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375038/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375038%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375038/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375038/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375038/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40849297/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375038/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40849297/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375038/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375038/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="SCAWZttjlF6bzfN30Wyi739CjKjltnyVvg7NBV6tEC9RPp3Md3RMGu7mWJVCQTjs">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
