
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Structural mechanism of strand exchange by the RAD51 filament - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53470C8AF4444305470C001AA7834F.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="elife">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360782/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="eLife">
<meta name="citation_title" content="Structural mechanism of strand exchange by the RAD51 filament">
<meta name="citation_author" content="Luay Joudeh">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom">
<meta name="citation_author" content="Robert E Appleby">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom">
<meta name="citation_author" content="Joseph D Maman">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom">
<meta name="citation_author" content="Luca Pellegrini">
<meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="14">
<meta name="citation_firstpage" content="RP107114">
<meta name="citation_doi" content="10.7554/eLife.107114">
<meta name="citation_pmid" content="40820711">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360782/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360782/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360782/pdf/elife-107114.pdf">
<meta name="description" content="Homologous recombination (HR) preserves genomic stability by repairing double-strand DNA breaks and ensuring efficient DNA replication. Central to HR is the strand-exchange reaction taking place within the three-stranded synapsis wherein a RAD51 ...">
<meta name="og:title" content="Structural mechanism of strand exchange by the RAD51 filament">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Homologous recombination (HR) preserves genomic stability by repairing double-strand DNA breaks and ensuring efficient DNA replication. Central to HR is the strand-exchange reaction taking place within the three-stranded synapsis wherein a RAD51 ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360782/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12360782">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.7554/eLife.107114"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/elife-107114.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12360782%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12360782/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12360782/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360782/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-elife.png" alt="eLife logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to eLife" title="Link to eLife" shape="default" href="https://doi.org/10.7554/eLife.107114" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">eLife</button></div>. 2025 Aug 18;14:RP107114. doi: <a href="https://doi.org/10.7554/eLife.107114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.107114</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22eLife%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22eLife%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22eLife%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22eLife%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Structural mechanism of strand exchange by the RAD51 filament</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joudeh%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Luay Joudeh</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Luay Joudeh</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joudeh%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Luay Joudeh</span></a>
</div>
</div>
<sup>1,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Appleby%20RE%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Robert E Appleby</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Robert E Appleby</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Appleby%20RE%22%5BAuthor%5D" class="usa-link"><span class="name western">Robert E Appleby</span></a>
</div>
</div>
<sup>1,</sup><sup>†,</sup><sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maman%20JD%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Joseph D Maman</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Joseph D Maman</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maman%20JD%22%5BAuthor%5D" class="usa-link"><span class="name western">Joseph D Maman</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pellegrini%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Luca Pellegrini</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Luca Pellegrini</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pellegrini%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Luca Pellegrini</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<div class="cg p">Editors: <span class="name western">Akira Shinohara</span><sup>2</sup>, <span class="name western">Adèle L Marston</span><sup>3</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom</div>
<div id="aff2">
<sup>2</sup>Osaka University, Osaka, Japan</div>
<div id="aff3">
<sup>3</sup>University of Edinburgh, Edinburgh, United Kingdom</div>
<div class="author-notes p">
<div class="fn" id="equal-contrib1">
<sup>†</sup><p class="display-inline">These authors contributed equally to this work.</p>
</div>
<div class="fn" id="pa1">
<sup>‡</sup><p class="display-inline">Francis Crick Institute, London, United Kingdom.</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Akira Shinohara</span></strong>: <span class="role">Reviewing Editor</span>
</div>
<div>
<strong class="contrib"><span class="name western">Adèle L Marston</span></strong>: <span class="role">Senior Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025, Joudeh, Appleby et al</div>
<p>This article is distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use and redistribution provided that the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12360782  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40820711/" class="usa-link">40820711</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article">
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint with doi: <a href="https://doi.org/10.1101/2025.03.26.645421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1101/2025.03.26.645421</a>.</div>
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint with doi: <a href="https://doi.org/10.7554/eLife.107114.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.7554/eLife.107114.1</a>.</div>
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint with doi: <a href="https://doi.org/10.7554/eLife.107114.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.7554/eLife.107114.2</a>.</div>
<div>See commentary "<a href="/articles/PMC12360781/" class="usa-link">Snapshot of an intermediate structure</a>", e108402.</div>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Homologous recombination (HR) preserves genomic stability by repairing double-strand DNA breaks and ensuring efficient DNA replication. Central to HR is the strand-exchange reaction taking place within the three-stranded synapsis wherein a RAD51 nucleoprotein filament binds to a donor DNA. Here, we present the cryoEM structure of a displacement loop of human RAD51 that captures the synaptic state when the filament has become tightly bound to the donor DNA. The structure elucidates the mechanism of strand exchange by RAD51, including the filament engagement with the donor DNA, the strand invasion and pairing with the complementary sequence of the donor DNA, the capture of the non-complementary strand and the polarity of the strand-exchange reaction. Our findings provide fundamental mechanistic insights into the biochemical reaction of eukaryotic HR.</p>
<section id="kwd-group2" class="kwd-group"><p><strong>Research organism:</strong> Human</p></section></section><section id="s1"><h2 class="pmc_sec_title">Introduction</h2>
<p>The transfer of genetic information between DNA molecules promoted by homologous recombination (HR) is essential in the repair of DNA double-strand breaks (<a href="#bib33" class="usa-link" aria-describedby="bib33">Kowalczykowski, 2015</a>; <a href="#bib61" class="usa-link" aria-describedby="bib61">Wright et al., 2018</a>) and to overcome replicative stress during chromosomal DNA synthesis (<a href="#bib12" class="usa-link" aria-describedby="bib12">Chakraborty et al., 2023</a>; <a href="#bib59" class="usa-link" aria-describedby="bib59">Triplett et al., 2024</a>). The molecular apparatus of HR is also responsible for the crossover reactions that generate genetic diversity in meiosis (<a href="#bib8" class="usa-link" aria-describedby="bib8">Brown and Bishop, 2014</a>). HR deficiency causes genetic instability that predisposes to cancer (<a href="#bib59" class="usa-link" aria-describedby="bib59">Triplett et al., 2024</a>; <a href="#bib44" class="usa-link" aria-describedby="bib44">Prakash et al., 2015</a>; <a href="#bib37" class="usa-link" aria-describedby="bib37">Matos-Rodrigues et al., 2021</a>), and its synthetic lethality with other DNA repair pathways is exploited to induce cancer cell death in therapy (<a href="#bib36" class="usa-link" aria-describedby="bib36">Lord and Ashworth, 2017</a>; <a href="#bib18" class="usa-link" aria-describedby="bib18">Dibitetto et al., 2024</a>).</p>
<p>In eukaryotic cells, the reaction of HR is performed by RAD51 (<a href="#bib4" class="usa-link" aria-describedby="bib4">Baumann et al., 1996</a>; <a href="#bib56" class="usa-link" aria-describedby="bib56">Sung, 1994</a>), a member of a highly conserved family of ATPases that includes bacterial RecA and archaeal RadA (<a href="#bib35" class="usa-link" aria-describedby="bib35">Lin et al., 2006</a>; <a href="#bib14" class="usa-link" aria-describedby="bib14">Chintapalli et al., 2013</a>). RAD51 polymerises on the single-stranded (ss) DNA overhangs that form upon long-range resection of DNA ends (<a href="#bib11" class="usa-link" aria-describedby="bib11">Cejka and Symington, 2021</a>). The RAD51 nucleoprotein filament invades the donor DNA duplex and searches for sequence homology within a three-stranded synaptic filament intermediate (<a href="#bib49" class="usa-link" aria-describedby="bib49">Renkawitz et al., 2014</a>; <a href="#bib23" class="usa-link" aria-describedby="bib23">Greene, 2016</a>). When homology is found, the invading strand of the filament becomes stably paired to the complementary sequence of the donor DNA, yielding a structure known as a displacement loop (D-loop) (<a href="#bib52" class="usa-link" aria-describedby="bib52">Shibata et al., 1979</a>; <a href="#bib32" class="usa-link" aria-describedby="bib32">Jain et al., 1995</a>; <a href="#bib43" class="usa-link" aria-describedby="bib43">Piazza et al., 2019</a>; <a href="#fig1" class="usa-link">Figure 1A</a>), which ultimately progresses to strand exchange. Although cellular HR is controlled by a complex network of recombination mediators such as BRCA2 (<a href="#bib6" class="usa-link" aria-describedby="bib6">Bell et al., 2023</a>), the RAD51 paralogues (<a href="#bib24" class="usa-link" aria-describedby="bib24">Greenhough et al., 2023</a>; <a href="#bib58" class="usa-link" aria-describedby="bib58">Taylor et al., 2015</a>; <a href="#bib48" class="usa-link" aria-describedby="bib48">Rawal et al., 2023</a>), and FIGNL1 (<a href="#bib31" class="usa-link" aria-describedby="bib31">Ito et al., 2023</a>; <a href="#bib10" class="usa-link" aria-describedby="bib10">Carver et al., 2025</a>), RAD51 can catalyse the biochemical reaction of strand exchange independently.</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">Figure 1. CryoEM structure of the RAD51 displacement loop (D-loop).</h3>
<figcaption><p>(<strong>A</strong>) Schematic drawing of RAD51 D-loop formation during double-strand DNA break repair by homologous recombination. (<strong>B</strong>) CryoEM map of the RAD51 D-loop, coloured according to local resolution. (<strong>C</strong>) Drawing of the RAD51 D-loop structure. The nine RAD51 protomers in the filament are shown as cylinder cartoons, coloured alternatively in lavender and pink. The DNA strands of the D-loop are represented as light yellow tubes for the phosphoribose backbone, and the nucleotides are drawn in stick representation with filled rings. (<strong>D</strong>) D-loop drawing that highlights the trajectories of the DNA strands of the donor DNA and the invading DNA. The complementary and exchanged strands of the donor DNA are coloured light green and orange; the invading strand of the filament is coloured as the exchanged strand to emphasise that they would share sequence homology in vivo. The protein filament is drawn as a transparent surface in light blue. (<strong>E</strong>) Nucleotide sequences and observed base pairing of the D-loop DNA. Sequences are coloured as in panel <strong>D</strong>. (<strong>F</strong>) Cartoon drawing of the DNA strands at the 5’-arm junction. The phosphoribose backbone is shown as a narrow tube, the nucleotide bases as filled rings. DNA strands are coloured as in panel <strong>D</strong>. The Watson-Crick hydrogen bonds between base pairs are drawn as dashed purple lines. (<strong>G</strong>) Cartoon drawing of the DNA strands at the 3’-arm junction, coloured and annotated as in panel<strong> F</strong>.</p></figcaption><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/54791dc71cf9/elife-107114-fig1.jpg" loading="lazy" height="1039" width="731" alt="Figure 1."></a></p>
<div class="p"><figure class="fig xbox font-sm" id="fig1s1"><h4 class="obj_head">Figure 1—figure supplement 1. Displacement loop (D-loop) reconstitution with human RAD51.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig1-figsupp1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/5cba19151b91/elife-107114-fig1-figsupp1.jpg" loading="lazy" height="817" width="757" alt="Figure 1—figure supplement 1."></a></p>
<figcaption><div class="p">(<strong>A</strong>) Sequences and designed base pairing of oligonucleotides used in D-loop reconstitution for cryoEM analysis. Nucleotides in lower case were not included in the final atomic model. (<strong>B</strong>, <strong>C</strong>) Electrophoretic mobility shift assays (EMSAs), shown as composite fluorescent images: the Cy5-labelled (<strong>B</strong>) or SYBR Gold-stained single-stranded DNA (ssDNA) (<strong>B</strong>) is coloured red, while the Cy3-labelled strand in double-stranded DNA (dsDNA) is green. Schematic drawings of the DNA substrates coloured according to the fluorescence label are shown above each experiment. (<strong>B</strong>) RAD51 titration on ssDNA only or ssDNA and complementary dsDNA. (<strong>C</strong>) RAD51 binding to doubly biotinylated ssDNA only or to doubly biotinylated ssDNA and complementary doubly biotinylated duplex DNA, in the presence of mono-streptavidin (mSA).</div>
<div class="p">
<section class="sm xbox font-sm" id="fig1s1sdata1"><div class="caption p"><span>Figure 1—figure supplement 1—source data 1. Original gel images for <a href="#fig1s1" class="usa-link">Figure 1—figure supplement 1B and C</a>.</span></div>
<div class="media p" id="d67e392"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-fig1-figsupp1-data1.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-107114-fig1-figsupp1-data1.zip</a><sup> (8.1MB, zip) </sup>
</div></div></section>
</div>
<div class="p">
<section class="sm xbox font-sm" id="fig1s1sdata2"><div class="caption p"><span>Figure 1—figure supplement 1—source data 2. Uncropped gel images for <a href="#fig1s1" class="usa-link">Figure 1—figure supplement 1A and B</a>.</span></div>
<div class="media p" id="d67e404"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-fig1-figsupp1-data2.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-107114-fig1-figsupp1-data2.zip</a><sup> (667.6KB, zip) </sup>
</div></div></section>
</div></figcaption></figure></div>
<div class="p"><figure class="fig xbox font-sm" id="fig1s2"><h4 class="obj_head">Figure 1—figure supplement 2. CryoEM dataset #1 (free-end DNA) processing.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig1-figsupp2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/d80d28ff13ae/elife-107114-fig1-figsupp2.jpg" loading="lazy" height="1039" width="728" alt="Figure 1—figure supplement 2."></a></p>
<figcaption><div class="p">3D reconstructions in light grey were used for further processing. (<strong>A</strong>) Representative micrograph with superimposed particles from final particle set. (<strong>B</strong>) Processing workflow. (<strong>C</strong>) Fourier shell correlation (FSC) curve with 0.143 threshold line (top) and conical FSC curve for directional resolution analysis (bottom).</div></figcaption></figure></div>
<div class="p"><figure class="fig xbox font-sm" id="fig1s3"><h4 class="obj_head">Figure 1—figure supplement 3. CryoEM dataset #2 (streptavidin-capped DNA) processing.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig1-figsupp3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/dc4ae23674a9/elife-107114-fig1-figsupp3.jpg" loading="lazy" height="1012" width="744" alt="Figure 1—figure supplement 3."></a></p>
<figcaption><div class="p">3D reconstructions in light grey were used for further processing. (<strong>A</strong>) Representative micrograph with superimposed particles from final particle set. (<strong>B</strong>) Processing workflow. (<strong>C</strong>) Fourier shell correlation (FSC) curve with 0.143 threshold line (left) and conical FSC curve for directional resolution analysis (right).</div></figcaption></figure></div>
<div class="p"><figure class="fig xbox font-sm" id="fig1s4"><h4 class="obj_head">Figure 1—figure supplement 4. Geometric analysis of displacement loop (D-loop) DNA.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig1-figsupp4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/d92a1eaebff0/elife-107114-fig1-figsupp4.jpg" loading="lazy" height="942" width="756" alt="Figure 1—figure supplement 4."></a></p>
<figcaption><div class="p">(<strong>A</strong>) Superposition of D-loop, pre-synaptic (PDB ID: 8BQ2) and post-synaptic (PDB ID: 8BR2) filament structures. Only the DNA is shown, as a tube ribbon coloured white (D-loop), grey (post-synaptic DNA), and dark grey (pre-synaptic DNA). The centroid positions of the RAD51 protomers in each structure are joined to generate a trajectory representative of the filament pitch for each structure. (<strong>B</strong>) Details of the cryoEM map for the D-loop DNA, with superimposed atomic model of the DNA strands. (<strong>C</strong>) Superposition of D-loop and post-synaptic DNA, oriented and coloured as in panel A, showing the base pairs in the homologous pairing sequence of the D-loop and post-synaptic double-stranded DNA (dsDNA). (<strong>D</strong>) Orientation of the axes of the arms of the donor dsDNA relative to the filament axis. The axes are drawn as cylinders and were fitted based on the centroid positions of the base pairs (for the donor arms) or the nucleotides of single-stranded DNA (ssDNA) in the filament. The tilt angles of the dsDNA arms relative to the filament axis are indicated.</div></figcaption></figure></div>
<div class="p"><figure class="fig xbox font-sm" id="fig1s5"><h4 class="obj_head">Figure 1—figure supplement 5. CryoEM density for homologous base pairing.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig1-figsupp5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/96036cb9ab24/elife-107114-fig1-figsupp5.jpg" loading="lazy" height="1039" width="745" alt="Figure 1—figure supplement 5."></a></p>
<figcaption><div class="p">Details of the cryoEM map for the base pairs formed by the invading strand (is) of the filament DNA with the complementary strand (cs) of the donor DNA. Nucleotides designed to base pair in the displacement loop (D-loop) structure are in bold. (<strong>A</strong>) Homologous base pairing. (<strong>B</strong>) Base conformation for the designed C8<sub>is</sub>:G26<sub>cs</sub> pair. Occupancy values for the two G26 conformations are reported in the panel.</div></figcaption></figure></div>
<div class="p"><figure class="fig xbox font-sm" id="fig1video1"><h4 class="obj_head">Figure 1—video 1. CryoEM structure of human RAD51 D-loop.</h4>
<div class="media-box p text-center margin-x-neg-2 tablet:margin-x-0"><video controls="" preload="none" height="360" width="522" poster="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/video_poster.png"><source src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/9767dc8bbd15/elife-107114-fig1-video1-pmcvs_normal.mp4"></source><div class="media-fallback margin-x-2 tablet:margin-x-0">
<div>Your browser is not supporting the HTML5 &lt;video&gt; element.</div>
<div>You may download that video as a file and play it with a player of you choice.</div>
<p><a href="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/9767dc8bbd15/elife-107114-fig1-video1-pmcvs_normal.mp4" class="usa-link">Download video stream</a>.</p>
</div></video></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-fig1-video1.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (28.9MB, mp4) </sup>
</div></div></figure></div>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The RAD51/Reca/RadA protein family has been extensively investigated to elucidate the mechanistic basis for the reaction of strand exchange. Low-resolution electron microscopy (EM) and X-ray crystallography have demonstrated conservation in the helical parameters of the nucleoprotein filament (<a href="#bib41" class="usa-link" aria-describedby="bib41">Ogawa et al., 1993</a>; <a href="#bib65" class="usa-link" aria-describedby="bib65">Yu et al., 2001</a>), the structure of the ATPase domain (<a href="#bib55" class="usa-link" aria-describedby="bib55">Story et al., 1992</a>; <a href="#bib42" class="usa-link" aria-describedby="bib42">Pellegrini et al., 2002</a>), and the protomer self-association into a filament (<a href="#bib16" class="usa-link" aria-describedby="bib16">Conway et al., 2004</a>; <a href="#bib62" class="usa-link" aria-describedby="bib62">Wu et al., 2004</a>). CryoEM analysis of the RAD51 nucleoprotein filament has revealed further conservation with RecA in the mode of interaction with DNA, such as the observation that DNA stretching results from its separation into B-form nucleotide triplets (<a href="#bib53" class="usa-link" aria-describedby="bib53">Short et al., 2016</a>; <a href="#bib63" class="usa-link" aria-describedby="bib63">Xu et al., 2017</a>; <a href="#bib2" class="usa-link" aria-describedby="bib2">Appleby et al., 2023a</a>; <a href="#bib13" class="usa-link" aria-describedby="bib13">Chen et al., 2008</a>). Furthermore, single-molecule experiments have determined the kinetic parameters of filament nucleation and growth (<a href="#bib21" class="usa-link" aria-describedby="bib21">Galletto et al., 2006</a>; <a href="#bib9" class="usa-link" aria-describedby="bib9">Candelli et al., 2014</a>), have shown that a minimum length of eight homologous nucleotides is required for the formation of stable synaptic joints (<a href="#bib28" class="usa-link" aria-describedby="bib28">Hsieh et al., 1992</a>; <a href="#bib46" class="usa-link" aria-describedby="bib46">Qi et al., 2015</a>) and that homologous pairing takes place in steps of three nucleotides (<a href="#bib47" class="usa-link" aria-describedby="bib47">Ragunathan et al., 2011</a>; <a href="#bib34" class="usa-link" aria-describedby="bib34">Lee et al., 2015</a>).</p>
<p>High-resolution information about the RAD51 synaptic state is still missing, limiting our mechanistic understanding of the strand-exchange reaction. In this study, we have carried out the biochemical reconstitution, cryoEM structure, and biophysical validation of a D-loop of human RAD51. The structure elucidates the key steps of strand exchange, including the mechanism of filament engagement with the donor DNA, strand invasion and pairing with the complementary strand, and capture of the exchanged strand. Comparison with the synaptic state of RecA filaments (<a href="#bib64" class="usa-link" aria-describedby="bib64">Yang et al., 2020</a>) reveals both conserved and distinct features of the strand-exchange reaction, shedding light on the molecular evolution of the biochemical reaction of HR.</p></section><section id="s2"><h2 class="pmc_sec_title">Results</h2>
<p>EM analysis of filamentous RAD51 relies normally on helical averaging, which is unsuitable for reconstruction of an aperiodic structure such as a D-loop. We therefore sought to prepare a D-loop sample of human RAD51 that is amenable to single-particle cryoEM. We designed a 32-nucleotide (nt) poly(dT) with a central, unique 10-nt-long sequence complementary to a 50-nt donor double-stranded (ds) DNA; the complementary sequence of the donor DNA was incorporated in a 10-nt mismatch bubble to favour D-loop formation (<a href="#fig1s1" class="usa-link">Figure 1—figure supplement 1A</a> and <a href="#supp1" class="usa-link">Supplementary file 1</a>). The length of the complementary sequence was decided based on evidence that RAD51 requires a homologous tract of at least eight nucleotides for stable capture of the donor DNA (<a href="#bib46" class="usa-link" aria-describedby="bib46">Qi et al., 2015</a>). As RAD51 can bind both ss- and dsDNA, we defined experimentally appropriate stoichiometry and buffer conditions for D-loop reconstitution. Electrophoretic mobility shift assay (EMSA) with fluorescently labelled oligos showed successful D-loop formation (<a href="#fig1s1" class="usa-link">Figure 1—figure supplement 1B</a> and <a href="#supp1" class="usa-link">Supplementary file 1</a>).</p>
<p>We collected an initial cryoEM dataset of 2792 movies of the RAD51 D-loop sample and processed the data by single-particle analysis. 2D classification showed clear evidence of the presence of synaptic particles and yielded a D-loop reconstruction at 3.31 Å resolution (<a href="#fig1s2" class="usa-link">Figure 1—figure supplement 2</a> and <a href="#supp2" class="usa-link">Supplementary file 2</a>). To improve data homogeneity and resolution, we collected a larger dataset of 8960 movies of a D-loop sample with ss- and dsDNA that had been biotinylated at both ends and capped with monomeric streptavidin (<a href="#fig1s1" class="usa-link">Figure 1—figure supplement 1C</a>; <a href="#bib3" class="usa-link" aria-describedby="bib3">Appleby et al., 2023b</a>), yielding a structure of the D-loop at 2.64 Å (<a href="#fig1s3" class="usa-link">Figure 1—figure supplement 3</a> and <a href="#supp2" class="usa-link">Supplementary file 2</a>). The final single-particle reconstruction based on 102,596 particles comprises a nucleoprotein filament of 9 RAD51 protomers and 26 nucleotides of ssDNA, bound to a dsDNA containing the central bubble region flanked by 15- and 16-bp-long segments of donor DNA. The cryoEM map ranges in resolution between 2.5 Å and 2.9 Å for the homologous pairing region of the D-loop and 3 Å and 6 Å for the flanking dsDNA arms (<a href="#fig1" class="usa-link">Figure 1B</a>).</p>
<section id="s2-1"><h3 class="pmc_sec_title">D-loop structure</h3>
<p>In the D-loop structure, RAD51 maintains the right-handed helical arrangement of protomers observed in pre- and post-synaptic filaments (<a href="#bib53" class="usa-link" aria-describedby="bib53">Short et al., 2016</a>; <a href="#bib63" class="usa-link" aria-describedby="bib63">Xu et al., 2017</a>), with the same helical twist and a slightly compressed pitch relative to the pre-synaptic filament (<a href="#fig1" class="usa-link">Figure 1C</a>, <a href="#fig1s4" class="usa-link">Figure 1—figure supplement 4A</a>, and <a href="#fig1video1" class="usa-link">Figure 1—video 1</a>). Continuous density for the exchanged strand (es), that becomes excluded upon homologous pairing of the invading strand (is) with the complementary strand (cs) of the donor DNA, connects the arms of the donor DNA (<a href="#fig1s4" class="usa-link">Figure 1—figure supplement 4B</a>). The filament-bound esDNA follows a peripheral path along the filament groove that runs parallel to the isDNA (<a href="#fig1" class="usa-link">Figure 1D</a>). The conformation of the duplex formed by pairing of the isDNA and csDNA strands resembles closely that of the dsDNA in the post-synaptic filament (<a href="#fig1s4" class="usa-link">Figure 1—figure supplement 4C</a>; <a href="#bib63" class="usa-link" aria-describedby="bib63">Xu et al., 2017</a>; <a href="#bib2" class="usa-link" aria-describedby="bib2">Appleby et al., 2023a</a>), with sharp bends at the junctions where the donor DNA enters and exits the filament (<a href="#fig1" class="usa-link">Figure 1C and D</a>).</p>
<p>The arms of the donor DNA flanking the complementary sequence project away from the filament in opposite directions and with different tilt angles relative to the filament axis: the dsDNA arm closer to the 5’-end of the filament, hereinafter referred to as the 5’-arm, is almost perpendicular to the filament axis, with its filament-proximal end pointing slightly towards the 3’-end of the filament (3’-tilt), whereas the dsDNA arm closer to the 3’-end of the filament, hereinafter referred to as the 3’-arm, exhibits a more pronounced tilt in the opposite direction, towards the 5’-end of the ssDNA (5’-tilt) (<a href="#fig1" class="usa-link">Figure 1D</a> and <a href="#fig1s4" class="usa-link">Figure 1—figure supplement 4D</a>). The opposite tilt of the donor arms favours their connection via the esDNA strand in the D-loop structure.</p></section><section id="s2-2"><h3 class="pmc_sec_title">Homologous pairing</h3>
<p>A continuous span of 10 base-pair interactions is present between the isDNA and csDNA (<a href="#fig1" class="usa-link">Figure 1E</a> and <a href="#fig1s5" class="usa-link">Figure 1—figure supplement 5A</a>). Base pairing includes the entirety of the csDNA sequence designed to be homologous to the invading strand, with the exception of G26<sub>cs</sub> (<a href="#fig1" class="usa-link">Figure 1E and F</a> and <a href="#fig1s5" class="usa-link">Figure 1—figure supplement 5A</a>): its guanine base adopts a minor-occupancy conformation in the direction of C8 in the isDNA but too distant for Watson-Crick hydrogen bonding, while the predominant conformation is pointing away (<a href="#fig1s5" class="usa-link">Figure 1—figure supplement 5B</a>).</p>
<p>Base pairing is fully maintained in the transition between the csDNA:isDNA duplex and the flanking arms of the donor DNA (<a href="#fig1" class="usa-link">Figure 1F and G</a>), with the exception of the aforementioned G26<sub>cs</sub>. Unexpectedly, an additional, mismatched base pair is formed at the junction of the csDNA:isDNA duplex with the 3’-arm of the donor DNA, where T18<sub>is</sub> becomes base paired with G16<sub>cs</sub> (<a href="#fig1" class="usa-link">Figure 1E, G</a> and <a href="#fig1s5" class="usa-link">Figure 1—figure supplement 5A</a>). Thus, the complementary pairing observed in the D-loop structure does not entirely follow our experimental design, with one unrealised G26<sub>cs</sub>:C8<sub>is</sub> base pair and one mismatched G16<sub>cs</sub>:T18<sub>is</sub> base pair (<a href="#fig1s5" class="usa-link">Figure 1—figure supplement 5</a>). This observation indicates that factors in addition to hydrogen-bonding potential contribute to determining pairing between the donor DNA and the invading strand of the filament.</p></section><section id="s2-3"><h3 class="pmc_sec_title">Unwinding and strand invasion of donor DNA</h3>
<p>Biochemical and structural analyses of pre- and post-synaptic filaments have revealed that RAD51 loops L1 and L2 mediate DNA binding, with prominent roles for amino acids L1 R235 and L2 V273 in partitioning filament DNA into nucleotide triplets (<a href="#bib63" class="usa-link" aria-describedby="bib63">Xu et al., 2017</a>; <a href="#bib38" class="usa-link" aria-describedby="bib38">Matsuo et al., 2006</a>). However, L2 residues Q272 to P283 remained disordered in these filament structures (<a href="#fig2" class="usa-link">Figure 2A</a>).</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">Figure 2. Donor DNA unwinding.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/529ce90ded71/elife-107114-fig2.jpg" loading="lazy" height="1000" width="756" alt="Figure 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Multiple sequence alignment of RAD51 L2 loop sequences, coloured according to conservation based on the Clustal scheme (<em>Hsa</em>: human, <em>Dre</em>: zebrafish, <em>Dme</em>: fly, <em>Ath</em>: <em>arabidopsis</em>, <em>Sce</em>: budding yeast, <em>Spo</em>: fission yeast). (<strong>B</strong>) Schematic drawing of the RAD51 displacement loop (D-loop). The N- and C-terminal domains of each RAD51 protomer are shown as spheres of size proportional to mass. RAD51 E, F, G, and H are coloured in blue, the other protomers in pale blue. (<strong>C</strong>) Ribbon drawing of the L2 loop insertion into the donor DNA for RAD51-E, F, G, and H. The L2 loops are drawn as thick blue tubes, and the rest of the RAD51 protein is represented as transparent surface. The DNA strands are drawn as narrow tubes with the nucleotides shown as sticks; the DNA strands are coloured dark magenta except for the homologous sequences of the invading and complementary strands, coloured in pink. (<strong>D</strong>) Nucleotide sequences and observed base pairing of the D-loop DNA. Sequences are coloured in dark magenta and pink as in panel <strong>C</strong>. The approximate position of the L2-loop insertion in the unwound donor DNA is marked with the name of the RAD51 protomer. Vertical dashed lines mark the extent of donor DNA unwinding for each L2 insertion. (<strong>E</strong>–<strong>H</strong>) Cartoon drawings of the interface between the L2 loop of RAD51-E (panel <strong>E</strong>), RAD51-F (panel <strong>F</strong>), RAD51-G (panel <strong>G</strong>), RAD51-H (panel <strong>H</strong>) with the donor DNA. L2-loop amino acids and DNA-strand nucleotides are drawn as sticks and carbon-coloured as in panel <strong>C</strong>. The nucleotide bases are shown as filled rings, and the Watson-Crick hydrogen bonds are drawn as dashed lines.</p></figcaption></figure><p>In our D-loop structure, the L2 sequences of four contiguous RAD51 protomers – E, F, G, and H (<a href="#fig2" class="usa-link">Figure 2B</a>) – have become ordered and traceable in the density map, folding in random-coil conformation with 3<sub>10</sub>-helix elements. The structure shows that filamentous RAD51 achieves strand separation by sequential L2-loop insertion of adjacent RAD51 protomers between the strands of the donor DNA (<a href="#fig2" class="usa-link">Figure 2C and D</a>). Insertion of the RAD51-H L2 loop at the 5’-arm junction causes a drastic widening of the minor groove, progressing to full separation of the complementary and exchanged strands upon L2-loop insertion by RAD51-G (<a href="#fig2" class="usa-link">Figure 2C, D, G, and H</a>). In total, donor DNA unwinding by RAD51-H and -G exposes a continuous csDNA stretch of five nucleotides for homologous pairing: A22-C25 of csDNA base pair with G9-T12 of isDNA, physically organised as a base-pair triplet of (A22-A24)<sub>cs</sub> with (T10-T12)<sub>is</sub> and one unstacked C25<sub>cs</sub>:G9<sub>is</sub> base pair, while G26<sub>cs</sub> remains unpaired (<a href="#fig1" class="usa-link">Figures 1F</a>, <a href="#fig2" class="usa-link">2C, D, G, and H</a>).</p>
<p>Successive L2-loop insertions extend the unwinding of donor DNA, with each insertion exposing a triplet of csDNA nucleotides for homology pairing with the invading strand (<a href="#fig2" class="usa-link">Figure 2C</a>). Thus, L2-loop insertions by RAD51-F and -E present (A19-C21)<sub>cs</sub> for base pairing with (G13-T15)<sub>is</sub>, and (G16-T18)<sub>cs</sub> for base pairing with (A16-T18)<sub>is</sub>, respectively (<a href="#fig2" class="usa-link">Figure 2C–F</a>). After completion of homologous pairing, the strands of the donor DNA rejoin in double-helical form at the 3’-arm junction (<a href="#fig1" class="usa-link">Figures 1G</a> and <a href="#fig2" class="usa-link">2C</a>).</p>
<p>The L2 loop acts as a physical spacer, directly separating the unwound strands of the donor DNA. Its sequence is dominated by hydrophobic amino acids (<a href="#fig2" class="usa-link">Figure 2A</a>) that make extensive contacts with the phosphoribose backbone of the esDNA and culminate in stacking of F279’s phenylalanine side chain against the last base pair of the 5’-arm duplex (RAD51-H) or the aromatic bases of nucleotides in the exchanged strand (RAD51-G, F, E) (<a href="#fig2" class="usa-link">Figure 2E–H</a>). The L2 loop displays a considerable degree of conformational heterogeneity, likely because each loop engages in both common and unique contacts with the DNA. This is best exemplified by the L2 loops of RAD51-E and -H that interact with the duplex arms of the donor DNA in different ways. RAD51-H L2 initiates donor DNA unwinding by buttressing the F279 side chain against G27<sub>cs</sub>:C15<sub>is</sub>, the last base pair of the 5’-arm duplex (<a href="#fig2" class="usa-link">Figure 2H</a>). Conversely, contact with RAD51-E L2 does not alter the conformation of the 3’-arm, as its F279 side chain stacks against C26 of the exchanged strand (<a href="#fig2" class="usa-link">Figure 2E</a>).</p>
<p>The asymmetric binding of RAD51-H and -E L2 loops at the boundaries between the arms of the donor DNA and the three-stranded synaptic DNA suggests that strand invasion and pairing follows a preferred direction. Thus, invasion by filament DNA likely begins at the 3’-end of the homology region in the complementary strand of the donor DNA and extends in the 5’-direction, as incrementally more nucleotides of csDNA are exposed by L2-loop insertions and become available for homologous pairing.</p></section><section id="s2-4"><h3 class="pmc_sec_title">Capture of the exchanged strand</h3>
<p>Strand invasion leads to displacement of the exchanged strand in the donor DNA. The structure of the D-loop shows that the esDNA is sequestered in a basic channel running parallel to the ssDNA-binding groove of the RAD51 nucleoprotein filament (<a href="#fig3" class="usa-link">Figure 3A and B</a>). One side of the channel is formed by RAD51 residues 303–313 (<a href="#fig3" class="usa-link">Figure 3C</a>), that fold in a beta hairpin on the inside of the filament groove, whereas the opposite side is formed by the C-terminal half of the L2-loop sequence (<a href="#fig3" class="usa-link">Figure 3B and D</a>). In the filament, the beta hairpins of successive protomers align to generate a continuous ssDNA-binding stripe (<a href="#fig3" class="usa-link">Figure 3D</a>). The esDNA is bound in the basic channel so that its aromatic bases remain largely exposed to solvent whereas the phosphodiester backbone faces the protein (<a href="#fig3" class="usa-link">Figure 3B and D</a>).</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">Figure 3. Capture of the exchanged strand.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/988895585d1e/elife-107114-fig3.jpg" loading="lazy" height="1038" width="696" alt="Figure 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Nucleotide sequences and observed base pairing of the displacement loop (D-loop) DNA. The exchanged strand DNA (esDNA) sequence is in yellow, the rest of the D-loop DNA in light brown. The span of esDNA bound by each of RAD51-H, -G, and -F is underlined. (<strong>B</strong>) Drawing of esDNA capture in a basic channel of the filament, aligned with the filament axis. The DNA is shown as a full-atom model with filled rings for the ribose and the bases, coloured in yellow for the captured esDNA nucleotides and light brown for the rest. Filament RAD51 is shown in surface representation, coloured according to electrostatic potential from blue (basic) to red (acidic). (<strong>C</strong>) Multiple sequence alignment of RAD51 esDNA-binding sequences, coloured according to conservation based on the Clustal scheme (<em>Hsa</em>: human, <em>Dre</em>: zebrafish, <em>Dme</em>: fly, <em>Ath</em>: <em>arabidopsis</em>, <em>Sce</em>: budding yeast, <em>Spo</em>: fission yeast). (<strong>D</strong>) Ribbon drawing of the interface between the filament and the captured esDNA, highlighting the position of the esDNA-binding beta hairpins of RAD51-H, -G, and -F in blue. The DNA is drawn as in panel <strong>B</strong>. The inset shows a schematic cartoon of the RAD51 D-loop, drawn as in <a href="#fig2" class="usa-link">Figure 2B</a>, to emphasise the position of RAD51-H, -G, -F in the filament. (<strong>E</strong>–<strong>G</strong>) Drawings of the interface between the beta hairpin amino acids of RAD51-H (panel <strong>E</strong>), RAD51-G (panel <strong>F</strong>), RAD51-F (panel <strong>G</strong>), and the esDNA. Beta hairpin residues and esDNA nucleotides are drawn as sticks; amino acids are coloured according to atom type, whereas esDNA nucleotides are coloured as in panel <strong>D</strong>.</p></figcaption></figure><p>Three adjacent RAD51 protomers – F, G, and H – engage the exchanged strand in the D-loop (<a href="#fig3" class="usa-link">Figure 3D</a>). The majority of contacts comprise charged interactions between four invariant RAD51 residues R303, K304, R306, and K313 and the phosphate groups of the esDNA (<a href="#fig3" class="usa-link">Figure 3E–G</a>). In addition to the electrostatic interactions, R306 is part of a GRG motif at the tip of the beta hairpin that comes into close contact with the phosphoribose backbone of the esDNA and makes stacking interactions with the aromatic bases of nucleotides A19<sub>es</sub> and T25<sub>es</sub> in RAD51-H and -G, respectively (<a href="#fig3" class="usa-link">Figure 3E and F</a>). The interaction of the 305-GRG-307 motif with esDNA resembles that of L1-loop 234-GRG-236 with csDNA in the homologous duplex of the D-loop.</p>
<p>Of the three RAD51 protomers F, G, and H that bind the esDNA, RAD51-G makes the most extensive set of contacts, spanning five nucleotides, from A21<sub>es</sub> to T25<sub>es</sub>, involving all four basic residues R303, K304, R306, and K313 (<a href="#fig3" class="usa-link">Figure 3F</a>). Binding of RAD51-G to the esDNA is further strengthened by phosphate-group contacts of L2-loop K284 with C22<sub>es</sub> and of R130 with C23<sub>es</sub> and T24<sub>es</sub>. RAD51-H contacts four nucleotides, from A17<sub>es</sub> to A20<sub>es</sub>, with R303, K304, and 305-GRG-307 (<a href="#fig3" class="usa-link">Figure 3E</a>), whereas RAD51-F makes a single long-range interaction between K313 and T25<sub>es</sub> (<a href="#fig3" class="usa-link">Figure 3G</a>). In addition, R303 and R306 of RAD51-F engage the phosphate groups of T27, A28, and T29 of the 3’-arm duplex, possibly to guide the 3’-arm in the appropriate direction as it exits the filament groove, towards the RAD51-A NTD (<a href="#fig3" class="usa-link">Figure 3G</a>).</p>
<p>In total, the filament interacts with 9 of the 11 nucleotides of unpaired esDNA via contiguous interactions of the two RAD51 protomers G and H (5 and 4 nt, respectively). This compares with donor DNA opening by L2-loop insertion, which progresses by exposure of three nucleotides per RAD51 protomer.</p></section><section id="s2-5"><h3 class="pmc_sec_title">Filament binding of donor DNA arms</h3>
<p>The D-loop structure shows that the nucleoprotein filament interacts with the double-stranded arms of the donor DNA via the N-terminal domains of RAD51 protomers A and D (<a href="#fig4" class="usa-link">Figure 4A and B</a>). Although the position of the RAD51-A and -D NTDs relative to the donor DNA is broadly similar, their DNA-binding interfaces differ significantly in the extent and mode of their engagement with the 3’- and 5’-arm duplexes, respectively.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">Figure 4. Filament engagement of the donor DNA arms.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/3206176580df/elife-107114-fig4.jpg" loading="lazy" height="1087" width="708" alt="Figure 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Drawing of the displacement loop (D-loop) structure that highlights the interaction of RAD51-A and -D with the 3’- and 5’-arms of the duplex donor DNA, respectively. RAD51-A and -D are drawn as cylinder cartoons in pale blue with their NTDs in blue. The donor DNA arms are shown as molecular surfaces in light brown while the rest of the DNA is drawn as narrow tubes. The filament is shown as a transparent surface. The inset shows a schematic cartoon of the D-loop structure to highlight the position of the RAD51-A and -D protomers. (<strong>B</strong>) Multiple sequence alignment of RAD51 NTD sequences, coloured according to conservation based on the Clustal scheme (<em>Hsa</em>: human, <em>Dre</em>: zebrafish, <em>Dme</em>: fly, <em>Ath</em>: <em>arabidopsis</em>, <em>Sce</em>: budding yeast, <em>Spo</em>: fission yeast). (<strong>C</strong>, <strong>D</strong>) The interaction of RAD51-D and -A NTD with the 5’- and 3’-arms, respectively. The protein is drawn as a ribbon, with the side chains as sticks. The hydrogen bond between the main-chain nitrogen of G65 and the phosphate group of T12 is drawn as a dashed line. The DNA is in spacefill representation; the nucleotides contacted by the NTD are coloured yellow, and the rest of the DNA is light brown. (<strong>E</strong>) Nucleotide sequence of the donor DNA, coloured as in panels <strong>C</strong> and<strong> D</strong>.</p></figcaption></figure><p>The RAD51-D NTD engages the minor groove of the 5’-arm DNA (<a href="#fig4" class="usa-link">Figure 4C</a>). The side chains of K39, K40, and K64 contact the phosphate groups of 10-GCTC-13 and 34-CG-35 in the upper and lower rungs of the minor groove, respectively, while K70 and K73 contribute long-range electrostatic interactions to the binding (<a href="#fig4" class="usa-link">Figure 4C and E</a>). The association of RAD51-D NTD with 5’-arm is closer and more extensive than that of RAD51-A with the 3’-arm, and includes a direct hydrogen bond between the main-chain nitrogen of G65 and the phosphate group of T12 (<a href="#fig4" class="usa-link">Figure 4C and E</a>). In contrast, RAD51-A NTD binds in the major groove of the 3’-arm DNA, using electrostatic interactions mediated by the same set of lysine residues as in RAD51-D (<a href="#fig4" class="usa-link">Figure 4D</a>). However, the RAD51-A association with the 3’-arm involves a more limited set of direct contacts with the phosphate groups of C5 and 31-TTG-33 across the major groove (<a href="#fig4" class="usa-link">Figure 4D and E</a>).</p>
<p>The interaction of the NTD with the arms of the donor DNA highlights a lack of direct contacts with the bases and the predominantly electrostatic nature of the association. Such plasticity in the NTD-DNA interaction might be required for effective D-loop formation, as the direction at which the 3’-arm exits the filament upon D-loop formation is a function of the number of nucleotides involved in homologous pairing.</p></section><section id="s2-6"><h3 class="pmc_sec_title">Structure-based mutagenesis of RAD51 residues important for D-loop formation</h3>
<p>The atomic model of the RAD51 D-loop has revealed several new protein-DNA interfaces, including those involved in strand separation, capture of the exchanged strand, and anchoring of the flanking DNA arms to the filament. To assess the relative importance and contribution of RAD51 residues involved in D-loop formation, we performed alanine mutagenesis of F279 (strand separation); R303, K304, R306, K313 (esDNA capture); K39, K40, K64, K70, and K73 (arm duplex binding) (<a href="#fig5" class="usa-link">Figure 5A</a>).</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">Figure 5. Structure-based mutagenesis of the RAD51 displacement loop (D-loop).</h4>
<figcaption><p>(<strong>A</strong>) Ribbon drawing of RAD51, highlighting residues targeted for alanine mutagenesis, colour-coded according to their role in D-loop formation (F279 in pink; R303, K304, R306, K313 in magenta; K39, K40, K64, K70, K73 in light blue). (<strong>B</strong>) Strand-exchange assay showing the change in fluorescence for each RAD51 mutant following injection of the complementary strand after 9 min, with traces coloured according to amino acid as in panel <strong>A</strong>. Error bars denote the standard deviation of three independent replicates. The upper panel shows a diagrammatic representation of the assay, with DNA shown as light or dark brown lines and the fluorophore and quencher represented as in the key. (<strong>C</strong>) Schematic drawing of the experimental setup for single molecule analysis of RAD51 D-loop formation. (<strong>D</strong>) Kymograph of a representative D-loop formation event. The upper panel shows a time-lapse fluorescence intensity scan of the λ DNA, with a stable binding event at the expected target site (10.6 kb, indicated by the arrow). The white dashed line marks the first 60 s interval that was used in frame acquisition. The lower panel shows a summed projection of the fluorescence signal over the acquisition time. (<strong>E</strong>) Quantification of RAD51 filament binding events. The left-side histogram displays the binding profile of wild-type RAD51 (0.5 μM), showing a Gaussian distribution of binding events centred on the expected position of the complementary sequence (~10.6 kb), consistent with specific single-stranded DNA (ssDNA) pairing and D-loop formation. The right-side histogram shows the binding profile of the KK39,40AA RAD51 double mutant (2.5 μM). (<strong>F</strong>) Mechanism of D-loop formation by RAD51 nucleoprotein filaments.</p></figcaption><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/a5c5aa9a1ba9/elife-107114-fig5.jpg" loading="lazy" height="1040" width="766" alt="Figure 5."></a></p>
<div class="p"><figure class="fig xbox font-sm" id="fig5s1"><h5 class="obj_head">Figure 5—figure supplement 1. Electrophoretic mobility shift assay for the interaction of RAD51 mutants with single-stranded DNA (ssDNA) (<strong>A</strong>) and double-stranded DNA (dsDNA) (<strong>B</strong>).</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig5-figsupp1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/0cf52f35a4ea/elife-107114-fig5-figsupp1.jpg" loading="lazy" height="577" width="754" alt="Figure 5—figure supplement 1."></a></p>
<figcaption><div class="p">For each mutant, increasing protein concentrations (0.5, 1, 2, 4, 6, 8, 10 µM) were titrated against a fixed amount of DNA (0.5 µM).</div>
<div class="p">
<section class="sm xbox font-sm" id="fig5s1sdata1"><div class="caption p"><span>Figure 5—figure supplement 1—source data 1. Original gel images for <a href="#fig5s1" class="usa-link">Figure 5—figure supplement 1A and B</a>.</span></div>
<div class="media p" id="d67e1052"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-fig5-figsupp1-data1.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-107114-fig5-figsupp1-data1.zip</a><sup> (18.5MB, zip) </sup>
</div></div></section>
</div>
<div class="p">
<section class="sm xbox font-sm" id="fig5s1sdata2"><div class="caption p"><span>Figure 5—figure supplement 1—source data 2. Uncropped gel images for <a href="#fig5s1" class="usa-link">Figure 5—figure supplement 1A and B</a>.</span></div>
<div class="media p" id="d67e1064"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-fig5-figsupp1-data2.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-107114-fig5-figsupp1-data2.zip</a><sup> (766.8KB, zip) </sup>
</div></div></section>
</div></figcaption></figure></div>
<div class="p"><figure class="fig xbox font-sm" id="fig5s2"><h5 class="obj_head">Figure 5—figure supplement 2. Fluorescence intensity traces extracted from 2D scans of trapped DNA molecules.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig5-figsupp2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/e2b13c260463/elife-107114-fig5-figsupp2.jpg" loading="lazy" height="723" width="734" alt="Figure 5—figure supplement 2."></a></p>
<figcaption><div class="p">(<strong>A</strong>) Traces for the wild-type RAD51 filaments. (<strong>B</strong>) Traces for KK39,40AA RAD51 mutant filaments, at 0.5 mM (left) and 2.5 mM (right). (<strong>C</strong>) Gaussian peak fitting for a representative trace of wild-type RAD51.</div></figcaption></figure></div>
<div class="p"><figure class="fig xbox font-sm" id="fig5s3"><h5 class="obj_head">Figure 5—figure supplement 3. Mechanism of displacement loop (D-loop) formation by the RecA nucleoprotein filament.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig5-figsupp3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/ffce9bae3191/elife-107114-fig5-figsupp3.jpg" loading="lazy" height="505" width="755" alt="Figure 5—figure supplement 3."></a></p></figure></div>
<div class="p"><figure class="fig xbox font-sm" id="fig5s4"><h5 class="obj_head">Figure 5—figure supplement 4. Comparison of RAD51 and RecA displacement loop (D-loop) structures.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig5-figsupp4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/831b567fe065/elife-107114-fig5-figsupp4.jpg" loading="lazy" height="960" width="755" alt="Figure 5—figure supplement 4."></a></p>
<figcaption><div class="p">(<strong>A</strong>) Side-by-side transparent surface representation of RAD51 (left, in light blue) and RecA (right, in light pink) D-loops, with DNA strands drawn as purple tubes. (<strong>B</strong>) Superposition of RAD51 and RecA protomers, coloured according to the colour key. (<strong>C</strong>) Side-by-side cylinder representation of RAD51 (left) and RecA (right) D-loops. The DNA-binding RAD51-NTD and RecA-CTD are coloured blue and magenta, respectively. The RAD51-NTD and RecA-CTD that contact the DNA are highlighted with dashed circles. The PDB accession code for the RecA D-loop is 7JY9.</div></figcaption></figure></div>
<div class="p"><figure class="fig xbox font-sm" id="fig5s5"><h5 class="obj_head">Figure 5—figure supplement 5. Proposed steps in the functional evolution of the RecA/RAD51 recombinases.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360782_elife-107114-fig5-figsupp5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/44c5/12360782/96fef3786009/elife-107114-fig5-figsupp5.jpg" loading="lazy" height="1298" width="667" alt="Figure 5—figure supplement 5."></a></p></figure></div>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The designed mutations are distant from the binding site of ssDNA in filamentous RAD51 and therefore are not predicted to reduce the ability of the mutant protein to form nucleoprotein filaments on ssDNA. As expected, EMSA analysis showed that the F279 and NTD mutants retained wild-type ability to form filaments on ssDNA (<a href="#fig5s1" class="usa-link">Figure 5—figure supplement 1A</a>). However, RAD51 mutants R303A, K304A, R306A, and K313A showed a clear decrease in ssDNA binding (<a href="#fig5s1" class="usa-link">Figure 5—figure supplement 1A</a>), suggesting that, in addition to their role in esDNA capture, they might mediate the initial contact of the incoming ssDNA during pre-synaptic filament nucleation. All RAD51 mutants showed a moderate decrease in affinity for dsDNA, with the exception of K304 (<a href="#fig5s1" class="usa-link">Figure 5—figure supplement 1B</a>).</p>
<p>We evaluated the ability of the alanine mutants to perform DNA-strand exchange using an in vitro assay that measures the increase in fluorescent signal when a fluorescently labelled 60-nt oligo that is quenched in duplex form is strand-exchanged with the unlabelled sequence (<a href="#fig5" class="usa-link">Figure 5B</a>; <a href="#bib29" class="usa-link" aria-describedby="bib29">Huang et al., 2011</a>). The results of the strand-exchange assay revealed that all mutants decreased the efficiency of the reaction, but with varying degrees of impairment (<a href="#fig5" class="usa-link">Figure 5B</a>). The L2-loop F279A showed a modest impact, in agreement with earlier findings (<a href="#bib38" class="usa-link" aria-describedby="bib38">Matsuo et al., 2006</a>), whereas the esDNA-capture mutants R303A, K304A, R306A, and K313A displayed a more pronounced effect. Notably, the NTD mutations showed the largest negative impact on strand exchange: the double lysine-to-alanine substitutions of K39, K40 and K70, K73 reduced strand exchange to a fraction of the yield for the wild-type protein.</p>
<p>To provide further support for the role of the RAD51 NTD in D-loop formation, we developed a single-molecule D-loop assay using fluorescence imaging with optical tweezers (<a href="#bib26" class="usa-link" aria-describedby="bib26">Heller et al., 2014</a>). In the assay, RAD51 filaments were reconstituted on an Alexa 488-labelled 49-nt ssDNA designed to anneal at position 10.6 kb of λ DNA (<a href="#fig5" class="usa-link">Figure 5C</a>). The filaments interacted with doubly biotinylated λ DNA held between streptavidin-coated beads and D-loop formation was monitored by the presence of a fluorescent signal at the expected locus on λ DNA. Wild-type RAD51 filaments were capable of D-loop formation at the expected site, as assessed by kymograph and binding events analysis (<a href="#fig5" class="usa-link">Figure 5D and E</a> and <a href="#fig5s2" class="usa-link">Figure 5—figure supplement 2</a>). In contrast, filaments of the KK39,40AA RAD51 double mutant failed to show measurable binding events; a fivefold increase in the concentration of the double mutant protein showed non-specific binding and no stable D-loop formation at the expected λ DNA site (<a href="#fig5" class="usa-link">Figure 5E</a> and <a href="#fig5s2" class="usa-link">Figure 5—figure supplement 2</a>).</p>
<p>These findings validate our model for the D-loop structure and highlight the important role of the RAD51 NTD in the strand-exchange reaction, by anchoring the donor dsDNA to the nucleoprotein filament in the correct orientation for the strand invasion and homology search. The biochemical behaviour of our esDNA-capture site mutants agrees with the impaired D-loop formation of a yeast RAD51 protein bearing alanine mutations at amino acids corresponding to R130, K303, and K313 of human RAD51 (<a href="#bib15" class="usa-link" aria-describedby="bib15">Cloud et al., 2012</a>).</p></section></section><section id="s3"><h2 class="pmc_sec_title">Discussion</h2>
<p>The architecture of the RAD51 D-loop structure reveals the mechanism of strand invasion and homology search within a synaptic RAD51 filament. The close engagement of the RAD51-D NTD with the 5’-arm duplex, coupled to minor-groove widening by the RAD51-H L2 loop that initiates DNA unwinding, indicates that synapsis formation starts by interaction of the filament with the 5’-arm of the donor DNA. In comparison, the interaction of the RAD51-A NTD with the 3’-arm is more limited, and insertion of the L2 loop of RAD51-E does not remodel the proximal end of the 3’-arm duplex. Furthermore, the first isDNA:csDNA base pair formed at the 3’-arm junction is a T:G mismatch that is unlikely to stabilise the initial strand pairing.</p>
<p>Our findings form the basis for a mechanistic model of strand exchange by a RAD51 filament (<a href="#fig5" class="usa-link">Figure 5F</a>). In the model, RAD51-D NTD binding directs the donor duplex towards the filament groove, where L2-loop insertions by RAD51-H and -G initiate strand separation and capture of the exchanged strand. Such unwinding exposes five nucleotides of csDNA, of which four are available for pairing with the invading strand of the filament. Initial base pair formation in the presence of homology would drive further donor DNA unwinding by L2-loop insertions of RAD51-F and -E; each insertion exposes a triplet of csDNA nucleotides, in agreement with experimental evidence that homologous pairing progresses in triplet steps (<a href="#bib46" class="usa-link" aria-describedby="bib46">Qi et al., 2015</a>; <a href="#bib34" class="usa-link" aria-describedby="bib34">Lee et al., 2015</a>). Concomitant capture of the exchanged strand by RAD51-G after insertion of a third L2 loop by RAD51-F would lead to sequestration of nine esDNA nucleotides, contributing to the formation of a stable synaptic intermediate.</p>
<p>Successful homology pairing drives the engagement of the 3’-arm of the donor dsDNA with the NTD of the RAD51 molecule three protomer positions away towards the 3’-end of the filament, completing D-loop formation. The observed plasticity in the interaction of the NTD with dsDNA would facilitate capture of the 3’-arm DNA emerging at slightly different angles from the filament axis, depending on the exact number of paired nucleotides. Overall, the structural features of the RAD51 D-loop provide a strong indication that strand pairing and exchange begins at the 3’-end of the complementary strand in the donor DNA and progresses with 3’-to-5’ polarity (<a href="#fig5" class="usa-link">Figure 5F</a>).</p>
<p>The observation of an unrealised base pair involving G26<sub>cs</sub> at the 3’-end of the unwound csDNA, which was designed to base pair with C8<sub>is</sub>, indicates that the mechanism of unwinding of the 5’-arm prevents base pairing of the first available csDNA nucleotide, likely due to physical constraints. Further unwinding of the 5’-arm to allow G26<sub>cs</sub> pairing was not observed, possibly because the presence of three consecutive G:C base pairs in the 5’-arm duplex acted as a GC clamp.</p>
<p>The presence of a T18<sub>is</sub>:G16<sub>cs</sub> mismatch as the last base pair at the 5’-end of the paired csDNA is intriguing (<a href="#fig1s5" class="usa-link">Figure 1—figure supplement 5</a>). Of note, T18<sub>is</sub> and G16<sub>cs</sub> do not form a ‘wobble’ base pair (<a href="#bib30" class="usa-link" aria-describedby="bib30">Hunter et al., 1987</a>), indicating that fulfilment of hydrogen bonding potential is not critical. Rather, mismatch pairing that completes a base-pair triplet might occur regardless of perfect homology, to achieve a full complement of base stacking and protein-DNA interactions with RAD51 L1 and L2 loops. This observation provides a potential mechanism for how RAD51 can tolerate mismatches at certain positions of the homologous sequence during strand exchange.</p>
<p>The observed 3’-to-5’ polarity of exchange of the complementary strand by RAD51 is opposite to the 5’-to-3’ polarity of bacterial RecA (<a href="#fig5s3" class="usa-link">Figure 5—figure supplement 3</a>), which was determined based on cryoEM structures of RecA D-loops (<a href="#bib64" class="usa-link" aria-describedby="bib64">Yang et al., 2020</a>). Comparison of RAD51 and RecA D-loop structures reveals that their different polarity follows from the opposite position occupied by their dsDNA-binding domains in the filament: in a vertically oriented filament with the 5’-end of the ssDNA at the top and the 3’-end at the bottom, the RAD51 NTDs line the bottom of the filament groove whereas the RecA CTDs hang from the top (<a href="#fig5s4" class="usa-link">Figure 5—figure supplement 4</a>). The shared mode of minor-groove binding by RAD51-NTD and RecA-CTD and L2-loop insertion into the minor groove half a turn away leads naturally to opposite polarity in the progression of strand invasion and pairing with the complementary strand of the donor DNA. Our mechanistic interpretation of static D-loop structures awaits full reconciliation with earlier efforts to determine strand-exchange polarity for RecA and RAD51 measured under a variety of experimental conditions (<a href="#bib25" class="usa-link" aria-describedby="bib25">Gupta et al., 1998</a>; <a href="#bib5" class="usa-link" aria-describedby="bib5">Baumann and West, 1997</a>; <a href="#bib57" class="usa-link" aria-describedby="bib57">Sung and Robberson, 1995</a>; <a href="#bib40" class="usa-link" aria-describedby="bib40">Murayama et al., 2008</a>).</p>
<p>Altogether, the available evidence suggests that the ability to perform DNA-strand exchange by members of the RecA-family of recombinases results from a process of convergent evolution (<a href="#fig5s5" class="usa-link">Figure 5—figure supplement 5</a>). Thus, an ancestral RecA-fold ATPase protein might first have evolved to bind ssDNA using the L1 and L2 loops, possibly as a means of protecting the exposed DNA template during DNA replication. Independent acquisition of dsDNA binding by RecA and RAD51 via incorporation of carboxy- and amino-terminal domains, respectively, might have further endowed these proteins with the ability to engage simultaneously ss- and dsDNA and catalyse the exchange of homologous DNA strands.</p></section><section id="s4"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="s4-1"><h3 class="pmc_sec_title">RAD51 expression and purification</h3>
<p>Human RAD51 was expressed and purified as described (<a href="#bib7" class="usa-link" aria-describedby="bib7">Brouwer et al., 2018</a>). Briefly, RAD51 was co-expressed with a HisTag-MBP (Maltose Binding Protein)-BRCA2 BRC4 fusion protein in Rosetta BL21(DE3) <em>Escherichia coli</em> cells, and purified over successive steps of Ni<sup>2+</sup> chromatography, heparin chromatography, and size-exclusion chromatography. The purified protein was concentrated and stored in small aliquots at –70°C. Structure-based RAD51 mutations were introduced according to standard protocols, and the mutant RAD51 proteins were purified as for the wild-type protein.</p></section><section id="s4-2"><h3 class="pmc_sec_title">DNA substrate preparation</h3>
<p>DNA oligos were purchased from Integrated DNA Technologies (IDT). Unmodified or biotinylated oligos were supplied PAGE-purified. Fluorescently labelled oligos were HPLC-purified. All oligos were resuspended to 100 μM in TE buffer (10 mM Tris pH 7.5, 1 mM EDTA). dsDNA reagents were prepared at a concentration of 10 μM duplex in TE buffer, by mixing equimolar concentrations of the constituent complementary ssDNA oligos and heating at 95°C for 5 min before slowly cooling to room temperature.</p>
<p>For the strand-exchange assay, a duplex DNA 60mer in which one strand was 5’-labelled with fluorescein and the complementary strand 3’-labelled with an Iowa Black quencher was prepared by mixing quencher strand to fluorescent strand at a 1.2:1 molar ratio to ensure full quenching and purified on 10% native PAGE.</p>
<p>All DNA sequences can be found in <a href="#supp1" class="usa-link">Supplementary file 1</a>.</p></section><section id="s4-3"><h3 class="pmc_sec_title">Electrophoretic mobility shift assay</h3>
<p>EMSA reactions were prepared in buffer: 25 mM HEPES pH 7.5, 150 mM NaCl, 2 mM DTT, 5 mM CaCl<sub>2</sub>, and 2 mM ATP, by mixing fluorescein-labelled ss- or dsDNA 60mer at 250 nM concentration with increasing concentrations of wild-type or mutant RAD51 (0.5, 1.0, 2.0, 4.0, 6.0, 8.0, and 10.0 µM) in a 20 µL volume, and incubated at room temperature for 10 min. 10 µL of 50% sucrose (wt/vol) was added to each reaction mixture prior to electrophoresis, and samples were resolved on a 0.5% agarose gel in TB buffer (45 mM Tris, 45 mM boric acid) at 45 V for 2 hr at 4°C.</p>
<p>For D-loop reconstitution experiments, EMSA reactions were prepared in the same buffer by mixing 500 nM of either Cy5-labelled ssDNA or doubly biotinylated ssDNA with increasing concentrations of RAD51 (1.0, 2.5, 5.0, 10.0, 20.0, 30.0 µM), followed by incubation at room temperature for 10 min. 500 nM of either Cy3-labelled dsDNA or doubly-biotinylated Cy3-labelled DNA was then added to the reaction and incubated for a further 10 min before electrophoresis. In experiments with mono-streptavidin (mSA) capped DNA, mSA was first added to the DNA at 5× molar excess over biotin labels and incubated for 15 min at room temperature before adding it to EMSA reactions. SAvPhire mSA was purchased from Sigma-Aldrich and resuspended to a working stock of 4 mg/mL from lyophilised powder according to supplier instructions.</p>
<p>In experiments where unlabelled DNA was used, the gel was stained in a 2× SYBR Gold solution made up in milliQ water (10 min, 25°C) and then destained in milliQ water. Gels were visualised using a Typhoon FLA 9000 fluorescent gel imager, by excitation at 473 nm for fluorescein-labelled DNA or SYBR Gold staining, 532 nm excitation for Cy3-labelled DNA or 635 nm excitation for Cy5-labelled DNA.</p></section><section id="s4-4"><h3 class="pmc_sec_title">DNA strand-exchange assay</h3>
<p>Strand-exchange assays were performed using a PHERAstar FS instrument (BMG Labtech) equipped with a fluorescence intensity optic module (excitation at 485 nm, emission at 520 nm, 10 nm bandwidth). The gain was calibrated to 80% of the maximum fluorescence intensity using a 25 nM sample of fluorescein-labelled ssDNA in strand-exchange buffer: 150 mM NaCl, 25 mM HEPES pH 7.5, 2 mM DTT, 2 mM CaCl<sub>2</sub>, 2 mM ATP, 0.1 mg/mL BSA.</p>
<p>Reactions were prepared in 50 µL volumes of strand-exchange buffer containing 500 nM RAD51 and 50 nM ssDNA 60mer, corresponding to a 1:2 ratio of RAD51 to RAD51-binding sites in the DNA. RAD51 was incubated with ssDNA at 30°C for 9 min prior to initiation of the strand-exchange reaction by 50 µL injection of dsDNA labelled with fluorescein at the 5’-end of one strand and with the Iowa Black quencher at the 3’-end of the complementary strand, using the PHERAstar FI Slow Kinetics mode. Fluorescence measurements began 2 s after duplex injection, and data were collected at 1 min intervals for an additional 30 min. As a specificity control for strand exchange, the reaction was performed with an oligo(dT) ssDNA 60mer. Blank measurements were performed by injecting dsDNA into strand-exchange buffer lacking both ssDNA and RAD51. All reactions were performed in triplicate.</p></section><section id="s4-5"><h3 class="pmc_sec_title">CryoEM</h3>
<section id="s4-5-1"><h4 class="pmc_sec_title">Sample preparation</h4>
<p>5 μM RAD51 was first incubated with 250 nM ssDNA in buffer: 150 mM NaCl, 25 mM HEPES pH 7.5, 2 mM DTT, 2 mM ATP, 5 mM CaCl<sub>2</sub>, for 10 min to assemble a nucleoprotein filament; 250 nM dsDNA was then added to the sample and incubated for a further 10 min. In experiments with mSA-capped DNA, mSA was first added to the DNA at 5× molar excess over biotin labels and incubated for 15 min before the addition of other components.</p></section><section id="s4-5-2"><h4 class="pmc_sec_title">Vitrification protocol</h4>
<p>UltrAuFoil R1.2/R1.3 300 mesh gold grids (Quantifoil) were glow-discharged twice for 1 min using a PELCO easiGlow system (0.38 mBar, 30 mA, negative polarity). 3 μL of sample was applied to the mesh side of each grid before plunge-freezing in liquid ethane using a Vitrobot Mark IV robot (FEI), set to 100% humidity, 4°C, 2 s blot time, and –3 blot force.</p></section><section id="s4-5-3"><h4 class="pmc_sec_title">Grid screening and data collection</h4>
<p>Grids were screened using a Talos Arctica 200 keV transmission electron microscope fitted with a Falcon III direct electron detector (Thermo Fisher). Data were collected using a Titan Krios G3 300 keV transmission electron microscope (Thermo Fisher) fitted with a K3 (Gatan) direct electron detector, using the EPU package (Thermo Fisher). Two datasets were collected: a first dataset with free-end DNA (#1; 2792 movies in super-resolution mode at 0.326 Å/pixel) and a second dataset with mSA-capped ss- and dsDNA (#2, 8960 movies in super-resolution mode at 0.326 Å/pixel and 2× binned). All data were recorded at the cryoEM facility of the University of Cambridge in the Department of Biochemistry. Data collection parameters are reported in <a href="#supp2" class="usa-link">Supplementary file 2</a>.</p></section><section id="s4-5-4"><h4 class="pmc_sec_title">Data processing</h4>
<p>Data was processed using cryoSPARC 4.4.1 (<a href="#bib45" class="usa-link" aria-describedby="bib45">Punjani et al., 2017</a>). Micrographs were motion-corrected with Patch Motion Correction with output F-crop factor set to ½. The contrast transfer function (CTF) was estimated using Patch CTF Estimation, and exposures were manually curated to remove outliers.</p>
<p>Dataset #1 1477 particles were manually picked from 2711 micrographs and, after 2D classification, 578 particles were selected to generate filament templates for automated picking. Template picking yielded 948,693 particles, which were reduced to 838,403 after outlier removal. 705,678 particles were extracted using a 400-pixel box. Multiple rounds of 2D classification led to the identification of a set of 2522 particles that yielded classes with clear evidence of synaptic filaments and was therefore used for Topaz training. The trained Topaz model picked 66,799 particles, which were manually curated to 64,942 particles; of these, 60,328 were extracted with a 416-pixel box. Ab initio reconstruction of two 3D volumes followed by heterogeneous refinement yielded a 3D shape of a recognisable D-loop structure. The corresponding set of 44,864 particles was subject to 3D classification into five classes, three of which presented pronounced D-loop features with strong density for the synaptic dsDNA; these classes were pooled and the combined 29,398 particles refined by homogenous refinement to 4.43 Å. The resulting particle set was used for a second round of Topaz training and picking, which yielded 167,005 particles and from which 153,235 particles were extracted with a 416-pixel box. The particles were used directly for ab initio reconstruction and heterogeneous refinement, generating a D-loop structure from a set of 117,041 particles. These were fractionated into 10 classes by 3D classification, and the particles for four of these classes that presented the best D-loop shapes (50,363 particles) were pooled and subject to homogenous refinement to 3.62 Å, followed by non-uniform refinement to yield a final D-loop reconstruction at 3.31 Å.</p>
<p>Dataset #2 The final 3D reconstruction obtained from dataset #1 processing was low-pass filtered at 8 Å and used to generate templates for particle picking. 2,940,045 particles were picked from 8834 micrographs, 2,680,398 particles were selected after manual curation, and 2,245,379 particles extracted with a 208-pixel box. After multiple rounds of 2D classification, classes from a 19,116 particle set showed clear synaptic features and were used for ab initio reconstruction and homogenous refinement of a 3D volume from 13,626 particles, which presented the expected filament shape and D-loop features such as density attributable to the dsDNA arms of the target DNA. These particles were used for Topaz training and picking of 415,845 particles, of which 413,320 particles were extracted with a 208-pixel box. The extracted particles were fed directly into ab initio reconstruction and heterogeneous refinement, which yielded a 3D volume of a filament with recognisable D-loop features at lower map contouring for 328,724 extracted particles. 3D classification into 10 classes identified a single class of 43,824 particles of a complete D-loop structure, which refined in non-uniform refinement to 2.96 Å. This particle set was used in a second round of Topaz training and picking that yielded 922,204 particles, of which 900,213 were extracted with a 208-pixel box and used directly in ab initio reconstruction and heterogenous refinement of a filament volume of 711,480 particles with recognisable D-loop features at lower map contouring. 3D classification of this particle set over 10 classes identified one class of 74,483 particles corresponding to a high-quality D-loop structure, which was improved by further classification and refinement by heterogenous refinement. The resulting set of 102,679 particles was subject to non-uniform refinement to yield a 3D reconstruction at 2.72 Å, followed by reference-based motion correction of 102,596 particles and another step of non-uniform refinement, to yield a final reconstruction at 2.64 Å resolution.</p></section><section id="s4-5-5"><h4 class="pmc_sec_title">Model building and refinement</h4>
<p>An atomic model comprising nine RAD51-ATP protomers, 26 nucleotides of ssDNA and 41 nucleotides for each strand of the ‘mismatch bubble’ dsDNA was built into the 3D volume of the final dataset #2 reconstruction at 2.64 Å resolution using Coot (<a href="#bib19" class="usa-link" aria-describedby="bib19">Emsley and Cowtan, 2004</a>). Fitting of the model to the map was improved by real-space refinement in Phenix (<a href="#bib1" class="usa-link" aria-describedby="bib1">Afonine et al., 2018</a>) and model rebuilding in Coot (<a href="#bib19" class="usa-link" aria-describedby="bib19">Emsley and Cowtan, 2004</a>). In the later stages of refinement, map fitting and model stereochemistry were improved using ISOLDE (<a href="#bib17" class="usa-link" aria-describedby="bib17">Croll, 2018</a>). The register of the DNA sequence in the structure could be unambiguously assigned based on fitting of the nucleoside bases in the homologous dsDNA region, where the map resolution was highest.</p></section></section><section id="s4-6"><h3 class="pmc_sec_title">Optical tweezers-based single-molecule fluorescence imaging</h3>
<p>To reconstitute RAD51 D-loop formation at the single-molecule level, we used a C-Trap (LUMICKS) instrument that combines optical tweezers and confocal fluorescence microscopy. The experimental setup featured a microfluidic chip with four distinct flow channels separated by laminar flow: one for streptavidin-coated polystyrene beads (Spherotech, 4.43 μm diameter), one for biotinylated λ DNA, and two for buffer exchange or RAD51 nucleoprotein filaments reservoirs.</p>
<p>Doubly biotinylated λ DNA molecules (48.5 kbp) prepared as previously described (<a href="#bib9" class="usa-link" aria-describedby="bib9">Candelli et al., 2014</a>) were tethered between two streptavidin-coated beads in the microfluidic flow cell. Successful capture of a single λ DNA was confirmed by measuring force-extension curves (<a href="#bib54" class="usa-link" aria-describedby="bib54">Smith et al., 1996</a>). The optical traps were calibrated to a stiffness of 0.4 pN/nm, enabling stable manipulation of the DNA tether under tension (<a href="#bib50" class="usa-link" aria-describedby="bib50">Rouzina and Bloomfield, 2001a</a>; <a href="#bib51" class="usa-link" aria-describedby="bib51">Rouzina and Bloomfield, 2001b</a>). Fluorescence imaging was performed using a single 488 nm laser for excitation with an illumination intensity of 2 μW. Fluorescence signals were detected on a highly sensitive single-photon counting detector. Confocal fluorescence scans were performed over a region of 300 × 50 pixels to visualise the DNA tether and the edges of the beads. Pixel width was set to 0.1 µm, and each pixel was illuminated for 0.1 ms. During imaging, DNA was held at a constant force of 20 pN, and the flow channels were closed.</p>
<p>RAD51 D-loop formation reactions were examined by incubating 20 µM RAD51 with 1 µM Alexa 488-labelled ssDNA 49mer in buffer: 25 mM HEPES pH 7.5, 150 mM NaCl, 2 mM ATP, 2 mM CaCl<sub>2</sub>, 2 mM DTT at room temperature for 20 min, 40-fold diluted in the same buffer and injected into the microfluidic flow cell for analysis of the interaction with the λ DNA. The 49-nt-long sequence of the ssDNA is complementary to nucleotides 10,617–10,666 of λ DNA, to promote D-loop formation at a single, specific site on the DNA tether. During imaging, the DNA tether tension was maintained at 20 pN, ensuring optimal conditions for detecting RAD51 interactions along the DNA substrate.</p></section><section id="s4-7"><h3 class="pmc_sec_title">Data extraction and analysis</h3>
<p>Two-dimensional (2D) fluorescence scans (300×50 pixels) of individual λ DNA molecules were recorded for a total acquisition time of 180 s at a temporal resolution of one frame every 2 s. Given that RAD51 binding induces DNA extension (<a href="#bib27" class="usa-link" aria-describedby="bib27">Hilario et al., 2009</a>; <a href="#bib60" class="usa-link" aria-describedby="bib60">van Mameren et al., 2009</a>), leading to drift of the D-loop position under the constant applied force regime, only the initial 30 frames were considered for analysis to ensure accurate localisation. The stack of frames was summed using the open-source image processing package FIJI (ImageJ), to enhance fluorescence signal detection by summing pixel intensity values across all frames in a stack (<a href="#bib20" class="usa-link" aria-describedby="bib20">Finkelstein et al., 2010</a>). Fluorescence intensity profiles were extracted by applying a linear scan along the DNA contour and bead peripheries, excluding regions near the bead peripheries to minimise fluorescence distortion, enabling precise quantification of binding events.</p>
<p>Due to fluorescence interference and reduced sensitivity near the beads, regions adjacent to each bead were excluded from analysis. The extent of this exclusion region was determined by performing a linear fit of the loss of fluorescence intensity measured at the junction areas versus the physical diameter of the beads and the contour length of λ DNA. Based on this analysis, the exclusion region was estimated to be approximately 0.75 kbp from each end of the DNA molecule.</p>
<p>To automate peak identification, fluorescence intensity profiles were plotted against position along λ DNA. Using the peak fitting module in Origin Pro (OriginLab), fluorescence intensity peaks were identified and fitted with a 2D-Gaussian model which tracks each peak using a symmetrical bell-shaped curve characterised by amplitude, width (full width at half maximum), and centre position (X-centre) to accurately locate and quantify binding events (<a href="#bib39" class="usa-link" aria-describedby="bib39">Morin et al., 2022</a>; <a href="#bib22" class="usa-link" aria-describedby="bib22">Gorman et al., 2007</a>). The X-centre values of the detected peaks were extracted to determine the precise binding positions along λ DNA. Binding event frequencies were binned at 1 kbp intervals along the DNA molecule length. Similar peak detection and Gaussian fitting were applied to the histogram of binding events, allowing for the identification of the focal binding site.</p></section></section><section id="ack" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We would like to thank Chen Qian for early work on the project and the Biochemistry cryoEM facility team for assistance with data collection. This work was funded by a Wellcome Trust Investigator award to LP (221892/Z/20/Z) and a UKRI BBSRC PhD studentship award to LP and RA (grant number 2271086).</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication. For the purpose of Open Access, the authors have applied a CC BY public copyright license to any Author Accepted Manuscript version arising from this submission.</p></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Luca Pellegrini, Email: lp212@cam.ac.uk.</p>
<p>Akira Shinohara, Osaka University, Osaka, Japan.</p>
<p>Adèle L Marston, University of Edinburgh, Edinburgh, United Kingdom.</p></section><section id="sec23"><h2 class="pmc_sec_title">Funding Information</h2>
<p>This paper was supported by the following grants:</p>
<ul class="list" style="list-style-type:disc">
<li><p>
Wellcome Trust

10.35802/221892 to Luca Pellegrini.</p></li>
<li><p>
Biotechnology and Biological Sciences Research Council

2271086 to Robert E Appleby.</p></li>
</ul></section><section id="s5"><h2 class="pmc_sec_title">Additional information</h2>
<section id="fn-group1" class="fn-group"><h3 class="pmc_sec_title">Competing interests</h3>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="conf1"><p>No competing interests declared.</p></div></div></section><section id="fn-group2" class="fn-group"><h3 class="pmc_sec_title">Author contributions</h3>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="con1"><p>Data curation, Supervision, Investigation, Methodology, Writing – review and editing.</p></div>
<div class="fn p" id="con2"><p>Data curation, Funding acquisition, Investigation, Methodology, Writing – review and editing.</p></div>
<div class="fn p" id="con3"><p>Conceptualization, Supervision, Methodology, Writing – review and editing.</p></div>
<div class="fn p" id="con4"><p>Conceptualization, Supervision, Funding acquisition, Writing – original draft, Writing – review and editing.</p></div>
</div></section></section><section id="s6"><h2 class="pmc_sec_title">Additional files</h2>
<section class="sm xbox font-sm" id="supp1"><div class="caption p"><span>Supplementary file 1. Oligonucleotide sequence, size, and labelling.</span></div>
<div class="media p" id="d67e1415"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-supp1.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-107114-supp1.docx</a><sup> (15.6KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="supp2"><div class="caption p"><span>Supplementary file 2. CryoEM data collection and real-space refinement.</span></div>
<div class="media p" id="d67e1420"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-supp2.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-107114-supp2.docx</a><sup> (28.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="mdar"><div class="caption p"><span>MDAR checklist</span></div>
<div class="media p" id="d67e1423"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-mdarchecklist1.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-107114-mdarchecklist1.docx</a><sup> (87.4KB, docx) </sup>
</div></div></section></section><section id="s7"><h2 class="pmc_sec_title">Data availability</h2>
<p>Atomic coordinates and cryoEM maps for the human RAD51 D-loop have been deposited in the PDB and EMDB under accession codes 9I62 and EMD-52646, respectively.</p>
<p>The following datasets were generated:</p>
<p>
Pellegrini L, Joudeh L, Appleby RE. 2025. CryoEM structure of a RAD51 D-loop. Worldwide Protein Data Bank. 
</p>
<p>
Pellegrini L, Joudeh L, Appleby RE. 2025. CryoEM structure of a RAD51 D-loop. Electron Microscopy Data Bank. EMD-52646
</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="bib1">
<cite>Afonine PV, Poon BK, Read RJ, Sobolev OV, Terwilliger TC, Urzhumtsevf A, Adams PD. Real‐space refinement in PHENIX for cryo‐EM and crystallography. Acta Crystallographica Section D. 2018;74:531–544. doi: 10.1107/S2059798318006551.</cite> [<a href="https://doi.org/10.1107/S2059798318006551" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6096492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29872004/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallographica%20Section%20D&amp;title=Real%E2%80%90space%20refinement%20in%20PHENIX%20for%20cryo%E2%80%90EM%20and%20crystallography&amp;author=PV%20Afonine&amp;author=BK%20Poon&amp;author=RJ%20Read&amp;author=OV%20Sobolev&amp;author=TC%20Terwilliger&amp;volume=74&amp;publication_year=2018&amp;pages=531-544&amp;pmid=29872004&amp;doi=10.1107/S2059798318006551&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib2">
<cite>Appleby R, Bollschweiler D, Chirgadze DY, Joudeh L, Pellegrini L. A metal ion-dependent mechanism of RAD51 nucleoprotein filament disassembly. iScience. 2023a;26:106689.  doi: 10.1016/j.isci.2023.106689.</cite> [<a href="https://doi.org/10.1016/j.isci.2023.106689" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10192527/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37216117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=A%20metal%20ion-dependent%20mechanism%20of%20RAD51%20nucleoprotein%20filament%20disassembly&amp;author=R%20Appleby&amp;author=D%20Bollschweiler&amp;author=DY%20Chirgadze&amp;author=L%20Joudeh&amp;author=L%20Pellegrini&amp;volume=26&amp;publication_year=2023a&amp;pages=106689&amp;pmid=37216117&amp;doi=10.1016/j.isci.2023.106689&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib3">
<cite>Appleby R, Joudeh L, Cobbett K, Pellegrini L. Structural basis for stabilisation of the RAD51 nucleoprotein filament by BRCA2. Nature Communications. 2023b;14:7003.  doi: 10.1038/s41467-023-42830-1.</cite> [<a href="https://doi.org/10.1038/s41467-023-42830-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10622577/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37919288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Structural%20basis%20for%20stabilisation%20of%20the%20RAD51%20nucleoprotein%20filament%20by%20BRCA2&amp;author=R%20Appleby&amp;author=L%20Joudeh&amp;author=K%20Cobbett&amp;author=L%20Pellegrini&amp;volume=14&amp;publication_year=2023b&amp;pages=7003&amp;pmid=37919288&amp;doi=10.1038/s41467-023-42830-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib4">
<cite>Baumann P, Benson FE, West SC. Human Rad51 protein promotes ATP-dependent homologous pairing and strand transfer reactions in vitro. Cell. 1996;87:757–766. doi: 10.1016/s0092-8674(00)81394-x.</cite> [<a href="https://doi.org/10.1016/s0092-8674(00)81394-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8929543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Human%20Rad51%20protein%20promotes%20ATP-dependent%20homologous%20pairing%20and%20strand%20transfer%20reactions%20in%20vitro&amp;author=P%20Baumann&amp;author=FE%20Benson&amp;author=SC%20West&amp;volume=87&amp;publication_year=1996&amp;pages=757-766&amp;pmid=8929543&amp;doi=10.1016/s0092-8674(00)81394-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib5">
<cite>Baumann P, West SC. The human Rad51 protein: polarity of strand transfer and stimulation by hRP-A. The EMBO Journal. 1997;16:5198–5206. doi: 10.1093/emboj/16.17.5198.</cite> [<a href="https://doi.org/10.1093/emboj/16.17.5198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1170152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9311980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20EMBO%20Journal&amp;title=The%20human%20Rad51%20protein:%20polarity%20of%20strand%20transfer%20and%20stimulation%20by%20hRP-A&amp;author=P%20Baumann&amp;author=SC%20West&amp;volume=16&amp;publication_year=1997&amp;pages=5198-5206&amp;pmid=9311980&amp;doi=10.1093/emboj/16.17.5198&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib6">
<cite>Bell JC, Dombrowski CC, Plank JL, Jensen RB, Kowalczykowski SC. BRCA2 chaperones RAD51 to single molecules of RPA-coated ssDNA. PNAS. 2023;120:e2221971120.  doi: 10.1073/pnas.2221971120.</cite> [<a href="https://doi.org/10.1073/pnas.2221971120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10083600/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36976771/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=BRCA2%20chaperones%20RAD51%20to%20single%20molecules%20of%20RPA-coated%20ssDNA&amp;author=JC%20Bell&amp;author=CC%20Dombrowski&amp;author=JL%20Plank&amp;author=RB%20Jensen&amp;author=SC%20Kowalczykowski&amp;volume=120&amp;publication_year=2023&amp;pages=e2221971120&amp;pmid=36976771&amp;doi=10.1073/pnas.2221971120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib7">
<cite>Brouwer I, Moschetti T, Candelli A, Garcin EB, Modesti M, Pellegrini L, Wuite GJ, Peterman EJ. Two distinct conformational states define the interaction of human RAD51-ATP with single-stranded DNA. The EMBO Journal. 2018;37:e98162.  doi: 10.15252/embj.201798162.</cite> [<a href="https://doi.org/10.15252/embj.201798162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5881629/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29507080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20EMBO%20Journal&amp;title=Two%20distinct%20conformational%20states%20define%20the%20interaction%20of%20human%20RAD51-ATP%20with%20single-stranded%20DNA&amp;author=I%20Brouwer&amp;author=T%20Moschetti&amp;author=A%20Candelli&amp;author=EB%20Garcin&amp;author=M%20Modesti&amp;volume=37&amp;publication_year=2018&amp;pages=e98162&amp;pmid=29507080&amp;doi=10.15252/embj.201798162&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib8">
<cite>Brown MS, Bishop DK. DNA strand exchange and RecA homologs in meiosis. Cold Spring Harbor Perspectives in Biology. 2014;7:a016659.  doi: 10.1101/cshperspect.a016659.</cite> [<a href="https://doi.org/10.1101/cshperspect.a016659" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4292170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25475089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Perspectives%20in%20Biology&amp;title=DNA%20strand%20exchange%20and%20RecA%20homologs%20in%20meiosis&amp;author=MS%20Brown&amp;author=DK%20Bishop&amp;volume=7&amp;publication_year=2014&amp;pages=a016659&amp;pmid=25475089&amp;doi=10.1101/cshperspect.a016659&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib9">
<cite>Candelli A, Holthausen JT, Depken M, Brouwer I, Franker MAM, Marchetti M, Heller I, Bernard S, Garcin EB, Modesti M, Wyman C, Wuite GJL, Peterman EJG. Visualization and quantification of nascent RAD51 filament formation at single-monomer resolution. PNAS. 2014;111:15090–15095. doi: 10.1073/pnas.1307824111.</cite> [<a href="https://doi.org/10.1073/pnas.1307824111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4210327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25288749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Visualization%20and%20quantification%20of%20nascent%20RAD51%20filament%20formation%20at%20single-monomer%20resolution&amp;author=A%20Candelli&amp;author=JT%20Holthausen&amp;author=M%20Depken&amp;author=I%20Brouwer&amp;author=MAM%20Franker&amp;volume=111&amp;publication_year=2014&amp;pages=15090-15095&amp;pmid=25288749&amp;doi=10.1073/pnas.1307824111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib10">
<cite>Carver A, Yu TY, Yates LA, White T, Wang R, Lister K, Jasin M, Zhang X. Molecular basis of FIGNL1 in dissociating RAD51 from DNA and chromatin. Science. 2025;387:426–431. doi: 10.1126/science.adr7920.</cite> [<a href="https://doi.org/10.1126/science.adr7920" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7617353/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39636933/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Molecular%20basis%20of%20FIGNL1%20in%20dissociating%20RAD51%20from%20DNA%20and%20chromatin&amp;author=A%20Carver&amp;author=TY%20Yu&amp;author=LA%20Yates&amp;author=T%20White&amp;author=R%20Wang&amp;volume=387&amp;publication_year=2025&amp;pages=426-431&amp;pmid=39636933&amp;doi=10.1126/science.adr7920&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib11">
<cite>Cejka P, Symington LS. DNA end resection: mechanism and control. Annual Review of Genetics. 2021;55:285–307. doi: 10.1146/annurev-genet-071719-020312.</cite> [<a href="https://doi.org/10.1146/annurev-genet-071719-020312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34813349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Genetics&amp;title=DNA%20end%20resection:%20mechanism%20and%20control&amp;author=P%20Cejka&amp;author=LS%20Symington&amp;volume=55&amp;publication_year=2021&amp;pages=285-307&amp;pmid=34813349&amp;doi=10.1146/annurev-genet-071719-020312&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib12">
<cite>Chakraborty S, Schirmeisen K, Lambert SA. The multifaceted functions of homologous recombination in dealing with replication-associated DNA damages. DNA Repair. 2023;129:103548.  doi: 10.1016/j.dnarep.2023.103548.</cite> [<a href="https://doi.org/10.1016/j.dnarep.2023.103548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37541027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=DNA%20Repair&amp;title=The%20multifaceted%20functions%20of%20homologous%20recombination%20in%20dealing%20with%20replication-associated%20DNA%20damages&amp;author=S%20Chakraborty&amp;author=K%20Schirmeisen&amp;author=SA%20Lambert&amp;volume=129&amp;publication_year=2023&amp;pages=103548&amp;pmid=37541027&amp;doi=10.1016/j.dnarep.2023.103548&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib13">
<cite>Chen Z, Yang H, Pavletich NP. Mechanism of homologous recombination from the RecA–ssDNA/dsDNA structures. Nature. 2008;453:489–494. doi: 10.1038/nature06971.</cite> [<a href="https://doi.org/10.1038/nature06971" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18497818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mechanism%20of%20homologous%20recombination%20from%20the%20RecA%E2%80%93ssDNA/dsDNA%20structures&amp;author=Z%20Chen&amp;author=H%20Yang&amp;author=NP%20Pavletich&amp;volume=453&amp;publication_year=2008&amp;pages=489-494&amp;pmid=18497818&amp;doi=10.1038/nature06971&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib14">
<cite>Chintapalli SV, Bhardwaj G, Babu J, Hadjiyianni L, Hong Y, Todd GK, Boosalis CA, Zhang Z, Zhou X, Ma H, Anishkin A, van Rossum DB, Patterson RL. Reevaluation of the evolutionary events within recA/RAD51 phylogeny. BMC Genomics. 2013;14:240.  doi: 10.1186/1471-2164-14-240.</cite> [<a href="https://doi.org/10.1186/1471-2164-14-240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3637515/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23574621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Reevaluation%20of%20the%20evolutionary%20events%20within%20recA/RAD51%20phylogeny&amp;author=SV%20Chintapalli&amp;author=G%20Bhardwaj&amp;author=J%20Babu&amp;author=L%20Hadjiyianni&amp;author=Y%20Hong&amp;volume=14&amp;publication_year=2013&amp;pages=240&amp;pmid=23574621&amp;doi=10.1186/1471-2164-14-240&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib15">
<cite>Cloud V, Chan YL, Grubb J, Budke B, Bishop DK. Rad51 is an accessory factor for Dmc1-mediated joint molecule formation during meiosis. Science. 2012;337:1222–1225. doi: 10.1126/science.1219379.</cite> [<a href="https://doi.org/10.1126/science.1219379" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4056682/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22955832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Rad51%20is%20an%20accessory%20factor%20for%20Dmc1-mediated%20joint%20molecule%20formation%20during%20meiosis&amp;author=V%20Cloud&amp;author=YL%20Chan&amp;author=J%20Grubb&amp;author=B%20Budke&amp;author=DK%20Bishop&amp;volume=337&amp;publication_year=2012&amp;pages=1222-1225&amp;pmid=22955832&amp;doi=10.1126/science.1219379&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib16">
<cite>Conway AB, Lynch TW, Zhang Y, Fortin GS, Fung CW, Symington LS, Rice PA. Crystal structure of a Rad51 filament. Nature Structural &amp; Molecular Biology. 2004;11:791–796. doi: 10.1038/nsmb795.</cite> [<a href="https://doi.org/10.1038/nsmb795" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15235592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Structural%20&amp;%20Molecular%20Biology&amp;title=Crystal%20structure%20of%20a%20Rad51%20filament&amp;author=AB%20Conway&amp;author=TW%20Lynch&amp;author=Y%20Zhang&amp;author=GS%20Fortin&amp;author=CW%20Fung&amp;volume=11&amp;publication_year=2004&amp;pages=791-796&amp;pmid=15235592&amp;doi=10.1038/nsmb795&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib17">
<cite>Croll TI. ISOLDE: a physically realistic environment for model building into low-resolution electron-density maps. Acta Crystallographica Section D Structural Biology. 2018;74:519–530. doi: 10.1107/S2059798318002425.</cite> [<a href="https://doi.org/10.1107/S2059798318002425" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6096486/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29872003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallographica%20Section%20D%20Structural%20Biology&amp;title=ISOLDE:%20a%20physically%20realistic%20environment%20for%20model%20building%20into%20low-resolution%20electron-density%20maps&amp;author=TI%20Croll&amp;volume=74&amp;publication_year=2018&amp;pages=519-530&amp;pmid=29872003&amp;doi=10.1107/S2059798318002425&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib18">
<cite>Dibitetto D, Widmer CA, Rottenberg S. PARPi, BRCA, and gaps: controversies and future research. Trends in Cancer. 2024;10:857–869. doi: 10.1016/j.trecan.2024.06.008.</cite> [<a href="https://doi.org/10.1016/j.trecan.2024.06.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39004561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Cancer&amp;title=PARPi,%20BRCA,%20and%20gaps:%20controversies%20and%20future%20research&amp;author=D%20Dibitetto&amp;author=CA%20Widmer&amp;author=S%20Rottenberg&amp;volume=10&amp;publication_year=2024&amp;pages=857-869&amp;pmid=39004561&amp;doi=10.1016/j.trecan.2024.06.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib19">
<cite>Emsley P, Cowtan K. Coot: model-building tools for molecular graphics. Acta Crystallographica. Section D, Biological Crystallography. 2004;60:2126–2132. doi: 10.1107/S0907444904019158.</cite> [<a href="https://doi.org/10.1107/S0907444904019158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15572765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallographica.%20Section%20D,%20Biological%20Crystallography&amp;title=Coot:%20model-building%20tools%20for%20molecular%20graphics&amp;author=P%20Emsley&amp;author=K%20Cowtan&amp;volume=60&amp;publication_year=2004&amp;pages=2126-2132&amp;pmid=15572765&amp;doi=10.1107/S0907444904019158&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib20">
<cite>Finkelstein IJ, Visnapuu ML, Greene EC. Single-molecule imaging reveals mechanisms of protein disruption by a DNA translocase. Nature. 2010;468:983–987. doi: 10.1038/nature09561.</cite> [<a href="https://doi.org/10.1038/nature09561" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3230117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21107319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Single-molecule%20imaging%20reveals%20mechanisms%20of%20protein%20disruption%20by%20a%20DNA%20translocase&amp;author=IJ%20Finkelstein&amp;author=ML%20Visnapuu&amp;author=EC%20Greene&amp;volume=468&amp;publication_year=2010&amp;pages=983-987&amp;pmid=21107319&amp;doi=10.1038/nature09561&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib21">
<cite>Galletto R, Amitani I, Baskin RJ, Kowalczykowski SC. Direct observation of individual RecA filaments assembling on single DNA molecules. Nature. 2006;443:875–878. doi: 10.1038/nature05197.</cite> [<a href="https://doi.org/10.1038/nature05197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16988658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Direct%20observation%20of%20individual%20RecA%20filaments%20assembling%20on%20single%20DNA%20molecules&amp;author=R%20Galletto&amp;author=I%20Amitani&amp;author=RJ%20Baskin&amp;author=SC%20Kowalczykowski&amp;volume=443&amp;publication_year=2006&amp;pages=875-878&amp;pmid=16988658&amp;doi=10.1038/nature05197&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib22">
<cite>Gorman J, Chowdhury A, Surtees JA, Shimada J, Reichman DR, Alani E, Greene EC. Dynamic basis for one-dimensional DNA scanning by the mismatch repair complex Msh2-Msh6. Molecular Cell. 2007;28:359–370. doi: 10.1016/j.molcel.2007.09.008.</cite> [<a href="https://doi.org/10.1016/j.molcel.2007.09.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2953642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17996701/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=Dynamic%20basis%20for%20one-dimensional%20DNA%20scanning%20by%20the%20mismatch%20repair%20complex%20Msh2-Msh6&amp;author=J%20Gorman&amp;author=A%20Chowdhury&amp;author=JA%20Surtees&amp;author=J%20Shimada&amp;author=DR%20Reichman&amp;volume=28&amp;publication_year=2007&amp;pages=359-370&amp;pmid=17996701&amp;doi=10.1016/j.molcel.2007.09.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib23">
<cite>Greene EC. DNA sequence alignment during homologous recombination. The Journal of Biological Chemistry. 2016;291:11572–11580. doi: 10.1074/jbc.R116.724807.</cite> [<a href="https://doi.org/10.1074/jbc.R116.724807" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4882428/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27129270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Biological%20Chemistry&amp;title=DNA%20sequence%20alignment%20during%20homologous%20recombination&amp;author=EC%20Greene&amp;volume=291&amp;publication_year=2016&amp;pages=11572-11580&amp;pmid=27129270&amp;doi=10.1074/jbc.R116.724807&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib24">
<cite>Greenhough LA, Liang CC, Belan O, Kunzelmann S, Maslen S, Rodrigo-Brenni MC, Anand R, Skehel M, Boulton SJ, West SC. Structure and function of the RAD51B-RAD51C-RAD51D-XRCC2 tumour suppressor. Nature. 2023;619:650–657. doi: 10.1038/s41586-023-06179-1.</cite> [<a href="https://doi.org/10.1038/s41586-023-06179-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7614784/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37344587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structure%20and%20function%20of%20the%20RAD51B-RAD51C-RAD51D-XRCC2%20tumour%20suppressor&amp;author=LA%20Greenhough&amp;author=CC%20Liang&amp;author=O%20Belan&amp;author=S%20Kunzelmann&amp;author=S%20Maslen&amp;volume=619&amp;publication_year=2023&amp;pages=650-657&amp;pmid=37344587&amp;doi=10.1038/s41586-023-06179-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib25">
<cite>Gupta RC, Golub EI, Wold MS, Radding CM. Polarity of DNA strand exchange promoted by recombination proteins of the RecA family. PNAS. 1998;95:9843–9848. doi: 10.1073/pnas.95.17.9843.</cite> [<a href="https://doi.org/10.1073/pnas.95.17.9843" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC21424/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9707563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Polarity%20of%20DNA%20strand%20exchange%20promoted%20by%20recombination%20proteins%20of%20the%20RecA%20family&amp;author=RC%20Gupta&amp;author=EI%20Golub&amp;author=MS%20Wold&amp;author=CM%20Radding&amp;volume=95&amp;publication_year=1998&amp;pages=9843-9848&amp;pmid=9707563&amp;doi=10.1073/pnas.95.17.9843&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib26">
<cite>Heller I, Hoekstra TP, King GA, Peterman EJG, Wuite GJL. Optical tweezers analysis of DNA-protein complexes. Chemical Reviews. 2014;114:3087–3119. doi: 10.1021/cr4003006.</cite> [<a href="https://doi.org/10.1021/cr4003006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24443844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Reviews&amp;title=Optical%20tweezers%20analysis%20of%20DNA-protein%20complexes&amp;author=I%20Heller&amp;author=TP%20Hoekstra&amp;author=GA%20King&amp;author=EJG%20Peterman&amp;author=GJL%20Wuite&amp;volume=114&amp;publication_year=2014&amp;pages=3087-3119&amp;pmid=24443844&amp;doi=10.1021/cr4003006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib27">
<cite>Hilario J, Amitani I, Baskin RJ, Kowalczykowski SC. Direct imaging of human Rad51 nucleoprotein dynamics on individual DNA molecules. PNAS. 2009;106:361–368. doi: 10.1073/pnas.0811965106.</cite> [<a href="https://doi.org/10.1073/pnas.0811965106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2613362/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19122145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Direct%20imaging%20of%20human%20Rad51%20nucleoprotein%20dynamics%20on%20individual%20DNA%20molecules&amp;author=J%20Hilario&amp;author=I%20Amitani&amp;author=RJ%20Baskin&amp;author=SC%20Kowalczykowski&amp;volume=106&amp;publication_year=2009&amp;pages=361-368&amp;pmid=19122145&amp;doi=10.1073/pnas.0811965106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib28">
<cite>Hsieh P, Camerini-Otero CS, Camerini-Otero RD. The synapsis event in the homologous pairing of DNAs: RecA recognizes and pairs less than one helical repeat of DNA. PNAS. 1992;89:6492–6496. doi: 10.1073/pnas.89.14.6492.</cite> [<a href="https://doi.org/10.1073/pnas.89.14.6492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC49527/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1631148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=The%20synapsis%20event%20in%20the%20homologous%20pairing%20of%20DNAs:%20RecA%20recognizes%20and%20pairs%20less%20than%20one%20helical%20repeat%20of%20DNA&amp;author=P%20Hsieh&amp;author=CS%20Camerini-Otero&amp;author=RD%20Camerini-Otero&amp;volume=89&amp;publication_year=1992&amp;pages=6492-6496&amp;pmid=1631148&amp;doi=10.1073/pnas.89.14.6492&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib29">
<cite>Huang F, Motlekar NA, Burgwin CM, Napper AD, Diamond SL, Mazin AV. Identification of specific inhibitors of human RAD51 recombinase using high-throughput screening. ACS Chemical Biology. 2011;6:628–635. doi: 10.1021/cb100428c.</cite> [<a href="https://doi.org/10.1021/cb100428c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3117970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21428443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Chemical%20Biology&amp;title=Identification%20of%20specific%20inhibitors%20of%20human%20RAD51%20recombinase%20using%20high-throughput%20screening&amp;author=F%20Huang&amp;author=NA%20Motlekar&amp;author=CM%20Burgwin&amp;author=AD%20Napper&amp;author=SL%20Diamond&amp;volume=6&amp;publication_year=2011&amp;pages=628-635&amp;pmid=21428443&amp;doi=10.1021/cb100428c&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib30">
<cite>Hunter WN, Brown T, Kneale G, Anand NN, Rabinovich D, Kennard O. The structure of guanosine-thymidine mismatches in B-DNA at 2.5-A resolution. The Journal of Biological Chemistry. 1987;262:9962–9970. doi: 10.2210/pdb113d/pdb.</cite> [<a href="https://doi.org/10.2210/pdb113d/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3611072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Biological%20Chemistry&amp;title=The%20structure%20of%20guanosine-thymidine%20mismatches%20in%20B-DNA%20at%202.5-A%20resolution&amp;author=WN%20Hunter&amp;author=T%20Brown&amp;author=G%20Kneale&amp;author=NN%20Anand&amp;author=D%20Rabinovich&amp;volume=262&amp;publication_year=1987&amp;pages=9962-9970&amp;pmid=3611072&amp;doi=10.2210/pdb113d/pdb&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib31">
<cite>Ito M, Furukohri A, Matsuzaki K, Fujita Y, Toyoda A, Shinohara A. FIGNL1 AAA+ ATPase remodels RAD51 and DMC1 filaments in pre-meiotic DNA replication and meiotic recombination. Nature Communications. 2023;14:6857.  doi: 10.1038/s41467-023-42576-w.</cite> [<a href="https://doi.org/10.1038/s41467-023-42576-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10611733/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37891173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=FIGNL1%20AAA+%20ATPase%20remodels%20RAD51%20and%20DMC1%20filaments%20in%20pre-meiotic%20DNA%20replication%20and%20meiotic%20recombination&amp;author=M%20Ito&amp;author=A%20Furukohri&amp;author=K%20Matsuzaki&amp;author=Y%20Fujita&amp;author=A%20Toyoda&amp;volume=14&amp;publication_year=2023&amp;pages=6857&amp;pmid=37891173&amp;doi=10.1038/s41467-023-42576-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib32">
<cite>Jain SK, Cox MM, Inman RB. Occurrence of three-stranded DNA within a RecA protein filament. The Journal of Biological Chemistry. 1995;270:4943–4949. doi: 10.1074/jbc.270.9.4943.</cite> [<a href="https://doi.org/10.1074/jbc.270.9.4943" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7876269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Biological%20Chemistry&amp;title=Occurrence%20of%20three-stranded%20DNA%20within%20a%20RecA%20protein%20filament&amp;author=SK%20Jain&amp;author=MM%20Cox&amp;author=RB%20Inman&amp;volume=270&amp;publication_year=1995&amp;pages=4943-4949&amp;pmid=7876269&amp;doi=10.1074/jbc.270.9.4943&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib33">
<cite>Kowalczykowski SC. An overview of the molecular mechanisms of recombinational DNA repair. Cold Spring Harbor Perspectives in Biology. 2015;7:a016410.  doi: 10.1101/cshperspect.a016410.</cite> [<a href="https://doi.org/10.1101/cshperspect.a016410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4632670/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26525148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Perspectives%20in%20Biology&amp;title=An%20overview%20of%20the%20molecular%20mechanisms%20of%20recombinational%20DNA%20repair&amp;author=SC%20Kowalczykowski&amp;volume=7&amp;publication_year=2015&amp;pages=a016410&amp;pmid=26525148&amp;doi=10.1101/cshperspect.a016410&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib34">
<cite>Lee JY, Terakawa T, Qi Z, Steinfeld JB, Redding S, Kwon Y, Gaines WA, Zhao W, Sung P, Greene EC. DNA RECOMBINATION: Base triplet stepping by the Rad51/RecA family of recombinases. Science. 2015;349:977–981. doi: 10.1126/science.aab2666.</cite> [<a href="https://doi.org/10.1126/science.aab2666" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4580133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26315438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=DNA%20RECOMBINATION:%20Base%20triplet%20stepping%20by%20the%20Rad51/RecA%20family%20of%20recombinases&amp;author=JY%20Lee&amp;author=T%20Terakawa&amp;author=Z%20Qi&amp;author=JB%20Steinfeld&amp;author=S%20Redding&amp;volume=349&amp;publication_year=2015&amp;pages=977-981&amp;pmid=26315438&amp;doi=10.1126/science.aab2666&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib35">
<cite>Lin Z, Kong H, Nei M, Ma H. Origins and evolution of the recA / RAD51 gene family: Evidence for ancient gene duplication and endosymbiotic gene transfer. PNAS. 2006;103:10328–10333. doi: 10.1073/pnas.0604232103.</cite> [<a href="https://doi.org/10.1073/pnas.0604232103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1502457/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16798872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Origins%20and%20evolution%20of%20the%20recA%20/%20RAD51%20gene%20family:%20Evidence%20for%20ancient%20gene%20duplication%20and%20endosymbiotic%20gene%20transfer&amp;author=Z%20Lin&amp;author=H%20Kong&amp;author=M%20Nei&amp;author=H%20Ma&amp;volume=103&amp;publication_year=2006&amp;pages=10328-10333&amp;pmid=16798872&amp;doi=10.1073/pnas.0604232103&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib36">
<cite>Lord CJ, Ashworth A. PARP inhibitors: Synthetic lethality in the clinic. Science. 2017;355:1152–1158. doi: 10.1126/science.aam7344.</cite> [<a href="https://doi.org/10.1126/science.aam7344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6175050/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28302823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=PARP%20inhibitors:%20Synthetic%20lethality%20in%20the%20clinic&amp;author=CJ%20Lord&amp;author=A%20Ashworth&amp;volume=355&amp;publication_year=2017&amp;pages=1152-1158&amp;pmid=28302823&amp;doi=10.1126/science.aam7344&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib37">
<cite>Matos-Rodrigues G, Guirouilh-Barbat J, Martini E, Lopez BS. Homologous recombination, cancer and the “RAD51 paradox”. NAR Cancer. 2021;3:zcab016.  doi: 10.1093/narcan/zcab016.</cite> [<a href="https://doi.org/10.1093/narcan/zcab016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8209977/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34316706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NAR%20Cancer&amp;title=Homologous%20recombination,%20cancer%20and%20the%20%E2%80%9CRAD51%20paradox%E2%80%9D&amp;author=G%20Matos-Rodrigues&amp;author=J%20Guirouilh-Barbat&amp;author=E%20Martini&amp;author=BS%20Lopez&amp;volume=3&amp;publication_year=2021&amp;pages=zcab016&amp;pmid=34316706&amp;doi=10.1093/narcan/zcab016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib38">
<cite>Matsuo Y, Sakane I, Takizawa Y, Takahashi M, Kurumizaka H. Roles of the human Rad51 L1 and L2 loops in DNA binding. The FEBS Journal. 2006;273:3148–3159. doi: 10.1111/j.1742-4658.2006.05323.x.</cite> [<a href="https://doi.org/10.1111/j.1742-4658.2006.05323.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16780572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20FEBS%20Journal&amp;title=Roles%20of%20the%20human%20Rad51%20L1%20and%20L2%20loops%20in%20DNA%20binding&amp;author=Y%20Matsuo&amp;author=I%20Sakane&amp;author=Y%20Takizawa&amp;author=M%20Takahashi&amp;author=H%20Kurumizaka&amp;volume=273&amp;publication_year=2006&amp;pages=3148-3159&amp;pmid=16780572&amp;doi=10.1111/j.1742-4658.2006.05323.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib39">
<cite>Morin JA, Wittmann S, Choubey S, Klosin A, Golfier S, Hyman AA, Jülicher F, Grill SW. Sequence-dependent surface condensation of a pioneer transcription factor on DNA. Nature Physics. 2022;18:271–276. doi: 10.1038/s41567-021-01462-2.</cite> [<a href="https://doi.org/10.1038/s41567-021-01462-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Physics&amp;title=Sequence-dependent%20surface%20condensation%20of%20a%20pioneer%20transcription%20factor%20on%20DNA&amp;author=JA%20Morin&amp;author=S%20Wittmann&amp;author=S%20Choubey&amp;author=A%20Klosin&amp;author=S%20Golfier&amp;volume=18&amp;publication_year=2022&amp;pages=271-276&amp;doi=10.1038/s41567-021-01462-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib40">
<cite>Murayama Y, Kurokawa Y, Mayanagi K, Iwasaki H. Formation and branch migration of Holliday junctions mediated by eukaryotic recombinases. Nature. 2008;451:1018–1021. doi: 10.1038/nature06609.</cite> [<a href="https://doi.org/10.1038/nature06609" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18256600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Formation%20and%20branch%20migration%20of%20Holliday%20junctions%20mediated%20by%20eukaryotic%20recombinases&amp;author=Y%20Murayama&amp;author=Y%20Kurokawa&amp;author=K%20Mayanagi&amp;author=H%20Iwasaki&amp;volume=451&amp;publication_year=2008&amp;pages=1018-1021&amp;pmid=18256600&amp;doi=10.1038/nature06609&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib41">
<cite>Ogawa T, Yu X, Shinohara A, Egelman EH. Similarity of the yeast RAD51 filament to the bacterial RecA filament. Science. 1993;259:1896–1899. doi: 10.1126/science.8456314.</cite> [<a href="https://doi.org/10.1126/science.8456314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8456314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Similarity%20of%20the%20yeast%20RAD51%20filament%20to%20the%20bacterial%20RecA%20filament&amp;author=T%20Ogawa&amp;author=X%20Yu&amp;author=A%20Shinohara&amp;author=EH%20Egelman&amp;volume=259&amp;publication_year=1993&amp;pages=1896-1899&amp;pmid=8456314&amp;doi=10.1126/science.8456314&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib42">
<cite>Pellegrini L, Yu DS, Lo T, Anand S, Lee M, Blundell TL, Venkitaraman AR. Insights into DNA recombination from the structure of a RAD51-BRCA2 complex. Nature. 2002;420:287–293. doi: 10.1038/nature01230.</cite> [<a href="https://doi.org/10.1038/nature01230" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12442171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Insights%20into%20DNA%20recombination%20from%20the%20structure%20of%20a%20RAD51-BRCA2%20complex&amp;author=L%20Pellegrini&amp;author=DS%20Yu&amp;author=T%20Lo&amp;author=S%20Anand&amp;author=M%20Lee&amp;volume=420&amp;publication_year=2002&amp;pages=287-293&amp;pmid=12442171&amp;doi=10.1038/nature01230&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib43">
<cite>Piazza A, Shah SS, Wright WD, Gore SK, Koszul R, Heyer WD. Dynamic processing of displacement loops during recombinational DNA repair. Molecular Cell. 2019;73:1255–1266. doi: 10.1016/j.molcel.2019.01.005.</cite> [<a href="https://doi.org/10.1016/j.molcel.2019.01.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6532985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30737186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=Dynamic%20processing%20of%20displacement%20loops%20during%20recombinational%20DNA%20repair&amp;author=A%20Piazza&amp;author=SS%20Shah&amp;author=WD%20Wright&amp;author=SK%20Gore&amp;author=R%20Koszul&amp;volume=73&amp;publication_year=2019&amp;pages=1255-1266&amp;pmid=30737186&amp;doi=10.1016/j.molcel.2019.01.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib44">
<cite>Prakash R, Zhang Y, Feng W, Jasin M. Homologous recombination and human health: the roles of BRCA1, BRCA2, and associated proteins. Cold Spring Harbor Perspectives in Biology. 2015;7:a016600.  doi: 10.1101/cshperspect.a016600.</cite> [<a href="https://doi.org/10.1101/cshperspect.a016600" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4382744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25833843/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Perspectives%20in%20Biology&amp;title=Homologous%20recombination%20and%20human%20health:%20the%20roles%20of%20BRCA1,%20BRCA2,%20and%20associated%20proteins&amp;author=R%20Prakash&amp;author=Y%20Zhang&amp;author=W%20Feng&amp;author=M%20Jasin&amp;volume=7&amp;publication_year=2015&amp;pages=a016600&amp;pmid=25833843&amp;doi=10.1101/cshperspect.a016600&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib45">
<cite>Punjani A, Rubinstein JL, Fleet DJ, Brubaker MA. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Nature Methods. 2017;14:290–296. doi: 10.1038/nmeth.4169.</cite> [<a href="https://doi.org/10.1038/nmeth.4169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28165473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=cryoSPARC:%20algorithms%20for%20rapid%20unsupervised%20cryo-EM%20structure%20determination&amp;author=A%20Punjani&amp;author=JL%20Rubinstein&amp;author=DJ%20Fleet&amp;author=MA%20Brubaker&amp;volume=14&amp;publication_year=2017&amp;pages=290-296&amp;pmid=28165473&amp;doi=10.1038/nmeth.4169&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib46">
<cite>Qi Z, Redding S, Lee JY, Gibb B, Kwon Y, Niu H, Gaines WA, Sung P, Greene EC. DNA sequence alignment by microhomology sampling during homologous recombination. Cell. 2015;160:856–869. doi: 10.1016/j.cell.2015.01.029.</cite> [<a href="https://doi.org/10.1016/j.cell.2015.01.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4344887/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25684365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=DNA%20sequence%20alignment%20by%20microhomology%20sampling%20during%20homologous%20recombination&amp;author=Z%20Qi&amp;author=S%20Redding&amp;author=JY%20Lee&amp;author=B%20Gibb&amp;author=Y%20Kwon&amp;volume=160&amp;publication_year=2015&amp;pages=856-869&amp;pmid=25684365&amp;doi=10.1016/j.cell.2015.01.029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib47">
<cite>Ragunathan K, Joo C, Ha T. Real-time observation of strand exchange reaction with high spatiotemporal resolution. Structure. 2011;19:1064–1073. doi: 10.1016/j.str.2011.06.009.</cite> [<a href="https://doi.org/10.1016/j.str.2011.06.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3163668/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21827943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Real-time%20observation%20of%20strand%20exchange%20reaction%20with%20high%20spatiotemporal%20resolution&amp;author=K%20Ragunathan&amp;author=C%20Joo&amp;author=T%20Ha&amp;volume=19&amp;publication_year=2011&amp;pages=1064-1073&amp;pmid=21827943&amp;doi=10.1016/j.str.2011.06.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib48">
<cite>Rawal Y, Jia L, Meir A, Zhou S, Kaur H, Ruben EA, Kwon Y, Bernstein KA, Jasin M, Taylor AB, Burma S, Hromas R, Mazin AV, Zhao W, Zhou D, Wasmuth EV, Greene EC, Sung P, Olsen SK. Structural insights into BCDX2 complex function in homologous recombination. Nature. 2023;619:640–649. doi: 10.1038/s41586-023-06219-w.</cite> [<a href="https://doi.org/10.1038/s41586-023-06219-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10712684/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37344589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structural%20insights%20into%20BCDX2%20complex%20function%20in%20homologous%20recombination&amp;author=Y%20Rawal&amp;author=L%20Jia&amp;author=A%20Meir&amp;author=S%20Zhou&amp;author=H%20Kaur&amp;volume=619&amp;publication_year=2023&amp;pages=640-649&amp;pmid=37344589&amp;doi=10.1038/s41586-023-06219-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib49">
<cite>Renkawitz J, Lademann CA, Jentsch S. Mechanisms and principles of homology search during recombination. Nature Reviews. Molecular Cell Biology. 2014;15:369–383. doi: 10.1038/nrm3805.</cite> [<a href="https://doi.org/10.1038/nrm3805" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24824069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews.%20Molecular%20Cell%20Biology&amp;title=Mechanisms%20and%20principles%20of%20homology%20search%20during%20recombination&amp;author=J%20Renkawitz&amp;author=CA%20Lademann&amp;author=S%20Jentsch&amp;volume=15&amp;publication_year=2014&amp;pages=369-383&amp;pmid=24824069&amp;doi=10.1038/nrm3805&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib50">
<cite>Rouzina I, Bloomfield VA. Force-induced melting of the DNA double helix. 2. Effect of solution conditions. Biophysical Journal. 2001a;80:894–900. doi: 10.1016/S0006-3495(01)76068-7.</cite> [<a href="https://doi.org/10.1016/S0006-3495(01)76068-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1301287/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11159456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophysical%20Journal&amp;title=Force-induced%20melting%20of%20the%20DNA%20double%20helix.%202.%20Effect%20of%20solution%20conditions&amp;author=I%20Rouzina&amp;author=VA%20Bloomfield&amp;volume=80&amp;publication_year=2001a&amp;pages=894-900&amp;pmid=11159456&amp;doi=10.1016/S0006-3495(01)76068-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib51">
<cite>Rouzina I, Bloomfield VA. Force-induced melting of the DNA double helix 1. Thermodynamic analysis. Biophysical Journal. 2001b;80:882–893. doi: 10.1016/S0006-3495(01)76067-5.</cite> [<a href="https://doi.org/10.1016/S0006-3495(01)76067-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1301286/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11159455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophysical%20Journal&amp;title=Force-induced%20melting%20of%20the%20DNA%20double%20helix%201.%20Thermodynamic%20analysis&amp;author=I%20Rouzina&amp;author=VA%20Bloomfield&amp;volume=80&amp;publication_year=2001b&amp;pages=882-893&amp;pmid=11159455&amp;doi=10.1016/S0006-3495(01)76067-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib52">
<cite>Shibata T, DasGupta C, Cunningham RP, Radding CM. Purified Escherichia coli recA protein catalyzes homologous pairing of superhelical DNA and single-stranded fragments. PNAS. 1979;76:1638–1642. doi: 10.1073/pnas.76.4.1638.</cite> [<a href="https://doi.org/10.1073/pnas.76.4.1638" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC383445/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/156361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Purified%20Escherichia%20coli%20recA%20protein%20catalyzes%20homologous%20pairing%20of%20superhelical%20DNA%20and%20single-stranded%20fragments&amp;author=T%20Shibata&amp;author=C%20DasGupta&amp;author=RP%20Cunningham&amp;author=CM%20Radding&amp;volume=76&amp;publication_year=1979&amp;pages=1638-1642&amp;pmid=156361&amp;doi=10.1073/pnas.76.4.1638&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib53">
<cite>Short JM, Liu Y, Chen S, Soni N, Madhusudhan MS, Shivji MKK, Venkitaraman AR. High-resolution structure of the presynaptic RAD51 filament on single-stranded DNA by electron cryo-microscopy. Nucleic Acids Research. 2016;44:9017–9030. doi: 10.1093/nar/gkw783.</cite> [<a href="https://doi.org/10.1093/nar/gkw783" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5100573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27596592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=High-resolution%20structure%20of%20the%20presynaptic%20RAD51%20filament%20on%20single-stranded%20DNA%20by%20electron%20cryo-microscopy&amp;author=JM%20Short&amp;author=Y%20Liu&amp;author=S%20Chen&amp;author=N%20Soni&amp;author=MS%20Madhusudhan&amp;volume=44&amp;publication_year=2016&amp;pages=9017-9030&amp;pmid=27596592&amp;doi=10.1093/nar/gkw783&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib54">
<cite>Smith SB, Cui Y, Bustamante C. Overstretching B-DNA: the elastic response of individual double-stranded and single-stranded DNA molecules. Science. 1996;271:795–799. doi: 10.1126/science.271.5250.795.</cite> [<a href="https://doi.org/10.1126/science.271.5250.795" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8628994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Overstretching%20B-DNA:%20the%20elastic%20response%20of%20individual%20double-stranded%20and%20single-stranded%20DNA%20molecules&amp;author=SB%20Smith&amp;author=Y%20Cui&amp;author=C%20Bustamante&amp;volume=271&amp;publication_year=1996&amp;pages=795-799&amp;pmid=8628994&amp;doi=10.1126/science.271.5250.795&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib55">
<cite>Story RM, Weber IT, Steitz TA. The structure of the E. coli recA protein monomer and polymer. Nature. 1992;355:318–325. doi: 10.1038/355318a0.</cite> [<a href="https://doi.org/10.1038/355318a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1731246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20structure%20of%20the%20E.%20coli%20recA%20protein%20monomer%20and%20polymer&amp;author=RM%20Story&amp;author=IT%20Weber&amp;author=TA%20Steitz&amp;volume=355&amp;publication_year=1992&amp;pages=318-325&amp;pmid=1731246&amp;doi=10.1038/355318a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib56">
<cite>Sung P. Catalysis of ATP-dependent homologous DNA pairing and strand exchange by yeast RAD51 protein. Science. 1994;265:1241–1243. doi: 10.1126/science.8066464.</cite> [<a href="https://doi.org/10.1126/science.8066464" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8066464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Catalysis%20of%20ATP-dependent%20homologous%20DNA%20pairing%20and%20strand%20exchange%20by%20yeast%20RAD51%20protein&amp;author=P%20Sung&amp;volume=265&amp;publication_year=1994&amp;pages=1241-1243&amp;pmid=8066464&amp;doi=10.1126/science.8066464&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib57">
<cite>Sung P, Robberson DL. DNA strand exchange mediated by a RAD51-ssDNA nucleoprotein filament with polarity opposite to that of RecA. Cell. 1995;82:453–461. doi: 10.1016/0092-8674(95)90434-4.</cite> [<a href="https://doi.org/10.1016/0092-8674(95)90434-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7634335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=DNA%20strand%20exchange%20mediated%20by%20a%20RAD51-ssDNA%20nucleoprotein%20filament%20with%20polarity%20opposite%20to%20that%20of%20RecA&amp;author=P%20Sung&amp;author=DL%20Robberson&amp;volume=82&amp;publication_year=1995&amp;pages=453-461&amp;pmid=7634335&amp;doi=10.1016/0092-8674(95)90434-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib58">
<cite>Taylor MRG, Špírek M, Chaurasiya KR, Ward JD, Carzaniga R, Yu X, Egelman EH, Collinson LM, Rueda D, Krejci L, Boulton SJ. Rad51 paralogs remodel pre-synaptic Rad51 filaments to stimulate homologous recombination. Cell. 2015;162:271–286. doi: 10.1016/j.cell.2015.06.015.</cite> [<a href="https://doi.org/10.1016/j.cell.2015.06.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4518479/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26186187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Rad51%20paralogs%20remodel%20pre-synaptic%20Rad51%20filaments%20to%20stimulate%20homologous%20recombination&amp;author=MRG%20Taylor&amp;author=M%20%C5%A0p%C3%ADrek&amp;author=KR%20Chaurasiya&amp;author=JD%20Ward&amp;author=R%20Carzaniga&amp;volume=162&amp;publication_year=2015&amp;pages=271-286&amp;pmid=26186187&amp;doi=10.1016/j.cell.2015.06.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib59">
<cite>Triplett MK, Johnson MJ, Symington LS. Induction of homologous recombination by site-specific replication stress. DNA Repair. 2024;142:103753.  doi: 10.1016/j.dnarep.2024.103753.</cite> [<a href="https://doi.org/10.1016/j.dnarep.2024.103753" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11425181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39190984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=DNA%20Repair&amp;title=Induction%20of%20homologous%20recombination%20by%20site-specific%20replication%20stress&amp;author=MK%20Triplett&amp;author=MJ%20Johnson&amp;author=LS%20Symington&amp;volume=142&amp;publication_year=2024&amp;pages=103753&amp;pmid=39190984&amp;doi=10.1016/j.dnarep.2024.103753&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib60">
<cite>van Mameren J, Modesti M, Kanaar R, Wyman C, Peterman EJG, Wuite GJL. Counting RAD51 proteins disassembling from nucleoprotein filaments under tension. Nature. 2009;457:745–748. doi: 10.1038/nature07581.</cite> [<a href="https://doi.org/10.1038/nature07581" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3871861/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19060884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Counting%20RAD51%20proteins%20disassembling%20from%20nucleoprotein%20filaments%20under%20tension&amp;author=J%20van%20Mameren&amp;author=M%20Modesti&amp;author=R%20Kanaar&amp;author=C%20Wyman&amp;author=EJG%20Peterman&amp;volume=457&amp;publication_year=2009&amp;pages=745-748&amp;pmid=19060884&amp;doi=10.1038/nature07581&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib61">
<cite>Wright WD, Shah SS, Heyer WD. Homologous recombination and the repair of DNA double-strand breaks. The Journal of Biological Chemistry. 2018;293:10524–10535. doi: 10.1074/jbc.TM118.000372.</cite> [<a href="https://doi.org/10.1074/jbc.TM118.000372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6036207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29599286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Biological%20Chemistry&amp;title=Homologous%20recombination%20and%20the%20repair%20of%20DNA%20double-strand%20breaks&amp;author=WD%20Wright&amp;author=SS%20Shah&amp;author=WD%20Heyer&amp;volume=293&amp;publication_year=2018&amp;pages=10524-10535&amp;pmid=29599286&amp;doi=10.1074/jbc.TM118.000372&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib62">
<cite>Wu Y, He Y, Moya IA, Qian X, Luo Y. Crystal structure of archaeal recombinase RADA: a snapshot of its extended conformation. Molecular Cell. 2004;15:423–435. doi: 10.1016/j.molcel.2004.07.014.</cite> [<a href="https://doi.org/10.1016/j.molcel.2004.07.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15304222/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=Crystal%20structure%20of%20archaeal%20recombinase%20RADA:%20a%20snapshot%20of%20its%20extended%20conformation&amp;author=Y%20Wu&amp;author=Y%20He&amp;author=IA%20Moya&amp;author=X%20Qian&amp;author=Y%20Luo&amp;volume=15&amp;publication_year=2004&amp;pages=423-435&amp;pmid=15304222&amp;doi=10.1016/j.molcel.2004.07.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib63">
<cite>Xu J, Zhao L, Xu Y, Zhao W, Sung P, Wang H-W. Cryo-EM structures of human RAD51 recombinase filaments during catalysis of DNA-strand exchange. Nature Structural &amp; Molecular Biology. 2017;24:40–46. doi: 10.1038/nsmb.3336.</cite> [<a href="https://doi.org/10.1038/nsmb.3336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5471492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27941862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Structural%20&amp;%20Molecular%20Biology&amp;title=Cryo-EM%20structures%20of%20human%20RAD51%20recombinase%20filaments%20during%20catalysis%20of%20DNA-strand%20exchange&amp;author=J%20Xu&amp;author=L%20Zhao&amp;author=Y%20Xu&amp;author=W%20Zhao&amp;author=P%20Sung&amp;volume=24&amp;publication_year=2017&amp;pages=40-46&amp;pmid=27941862&amp;doi=10.1038/nsmb.3336&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib64">
<cite>Yang H, Zhou C, Dhar A, Pavletich NP. Mechanism of strand exchange from RecA-DNA synaptic and D-loop structures. Nature. 2020;586:801–806. doi: 10.1038/s41586-020-2820-9.</cite> [<a href="https://doi.org/10.1038/s41586-020-2820-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8366275/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33057191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mechanism%20of%20strand%20exchange%20from%20RecA-DNA%20synaptic%20and%20D-loop%20structures&amp;author=H%20Yang&amp;author=C%20Zhou&amp;author=A%20Dhar&amp;author=NP%20Pavletich&amp;volume=586&amp;publication_year=2020&amp;pages=801-806&amp;pmid=33057191&amp;doi=10.1038/s41586-020-2820-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib65">
<cite>Yu X, Jacobs SA, West SC, Ogawa T, Egelman EH. Domain structure and dynamics in the helical filaments formed by RecA and Rad51 on DNA. PNAS. 2001;98:8419–8424. doi: 10.1073/pnas.111005398.</cite> [<a href="https://doi.org/10.1073/pnas.111005398" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC37452/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11459984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Domain%20structure%20and%20dynamics%20in%20the%20helical%20filaments%20formed%20by%20RecA%20and%20Rad51%20on%20DNA&amp;author=X%20Yu&amp;author=SA%20Jacobs&amp;author=SC%20West&amp;author=T%20Ogawa&amp;author=EH%20Egelman&amp;volume=98&amp;publication_year=2001&amp;pages=8419-8424&amp;pmid=11459984&amp;doi=10.1073/pnas.111005398&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section></section><article class="sub-article" id="sa0"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. doi: <a href="https://doi.org/10.7554/eLife.107114.3.sa0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.107114.3.sa0</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>eLife Assessment</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shinohara%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Akira Shinohara</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Akira Shinohara</span></h3>
<div class="p">
<sup>1</sup>Osaka University, Japan</div>
<div>Reviewing Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shinohara%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Akira Shinohara</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.c" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a.c" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.c" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.c" class="d-panel p" style="display: none">
<div class="p" id="aff4">
<sup>1</sup>Osaka University, Japan</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Akira Shinohara</span></strong>: <span class="role">Reviewing Editor</span>
</div>
</div>
<div id="anp_a.c" class="d-panel p" style="display: none"><div class="notes p">
<section id="kwd-group3" class="kwd-group"><p><strong>Keywords:</strong> Compelling</p></section><section id="kwd-group4" class="kwd-group"><p><strong>Keywords:</strong> Landmark</p></section>
</div></div>
<div id="clp_a.c" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>This <strong>landmark</strong> study describes the structure of the human RAD51 filament with a recombination intermediate called the displacement loop (D-loop). Using cryogenic structural, biochemical, and single-molecule analyses, the authors provide <strong>compelling</strong> evidence on how the RAD51 filament promotes strand exchange between single-stranded and double-stranded DNAs. The findings are highly relevant to the fields of homologous recombination, DNA repair, and genome stability.</p></section></article><article class="sub-article" id="sa1"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. doi: <a href="https://doi.org/10.7554/eLife.107114.3.sa1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.107114.3.sa1</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Reviewer #1 (Public review):</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name given-only">Anonymous</span></a><div hidden="hidden" id="id6">
<h3><span class="name given-only">Anonymous</span></h3>
<div>Reviewer</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link"><span class="name given-only">Anonymous</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.d" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.d" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.d" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name given-only">Anonymous</span></strong>: <span class="role">Reviewer</span>
</div>
</div>
<div id="clp_a.d" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>Summary:</p>
<p>The paper describes the cryoEM structure of RAD51 filament on the recombination intermediate. In the RAD51 filament, the insertion of a DNA-binding loop called the L2 loop stabilizes the separation of the complementary strand for the base-pairing with an incoming ssDNA and the non-complementary strand, which is captured by the second DNA-binding channel called the site II. The molecular structure of the RAD51 filament with a recombination intermediate provides a new insight into the mechanism of homology search and strand exchange between ssDNA and dsDNA.</p>
<p>Strong points:</p>
<p>This is the first human RAD51 filament structure with a recombination intermediate called the D-loop. The work has been done with great care, and the results shown in the paper are compelling based on cryo-EM and biochemical analyses. The paper is really nice and important for researchers in the field of homologous recombination, which gives a new view on the molecular mechanism of RAD51-mediated homology search and strand exchange.</p>
<p>Comments on revisions:</p>
<p>The authors nicely address most of the previous points.</p></section></article><article class="sub-article" id="sa2"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. doi: <a href="https://doi.org/10.7554/eLife.107114.3.sa2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.107114.3.sa2</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Reviewer #2 (Public review):</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name given-only">Anonymous</span></a><div hidden="hidden" id="id7">
<h3><span class="name given-only">Anonymous</span></h3>
<div>Reviewer</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link"><span class="name given-only">Anonymous</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.e" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.e" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.e" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name given-only">Anonymous</span></strong>: <span class="role">Reviewer</span>
</div>
</div>
<div id="clp_a.e" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>Homologous recombination is essential for DNA double-strand break repair, with RAD51-catalyzed strand exchange at its core. This study presents a 2.64 Å resolution cryogenic electron microscopy structure of the RAD51 D-loop complex, achieved through reconstitution of a RAD51 mini-filament. The structure uncovers how specific RAD51 residues drive strand exchange, offering atomic-level insight into the mechanics of eukaryotic HR and DNA repair.</p>
<p>Comments on revisions:</p>
<p>Authors acknowledged:</p>
<p>"We acknowledge that there exists an extensive body of literature that has investigated the polarity of strand exchange by RecA and RAD51 under a variety of experimental conditions, and we have added a brief comment to the text to reflect this, as well as some of the key citations. Undoubtedly, and as we also mention in our reply to the public reviews, further experimental work will be needed for a full reconciliation of the available evidence."</p>
<p>In the revised manuscript, this is reflected in the statement:</p>
<p>"Our mechanistic interpretation of static D-loop structures awaits full reconciliation with earlier efforts to determine strand-exchange polarity for RecA and RAD51 measured under a variety of experimental conditions."</p>
<p>Among the four cited studies, my understanding (as a person who has never studied this subject of polarity) is as follows:</p>
<p>•References 50 (EMBO J. 1997), 51 (Cell. 1995), and 52 (Nature. 2008) suggest that the strand exchange by human RAD51 occurs with a polarity opposite to that of RecA-that is, in the 5′→3′ direction relative to the complementary strand, or 3′→5′ relative to the initiating single-stranded DNA (isDNA).</p>
<p>• In contrast, reference 49 (PNAS 1998) proposed that 5′→3′ polarity (relative to isDNA) is conserved across RecA, human RAD51, and yeast RAD51.</p>
<p>Given the substantial structural analysis provided in the current manuscript, it would strengthen the work to include a concise description of these earlier biochemical findings, rather than citing them without context. This would benefit readers who are not familiar with the longstanding studies in the field and allow for a more informed interpretation of how the structural observations may reconcile or contrast with previous work.</p></section></article><article class="sub-article" id="sa3"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. doi: <a href="https://doi.org/10.7554/eLife.107114.3.sa3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.107114.3.sa3</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Reviewer #3 (Public review):</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name given-only">Anonymous</span></a><div hidden="hidden" id="id8">
<h3><span class="name given-only">Anonymous</span></h3>
<div>Reviewer</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link"><span class="name given-only">Anonymous</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.f" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.f" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.f" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name given-only">Anonymous</span></strong>: <span class="role">Reviewer</span>
</div>
</div>
<div id="clp_a.f" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>Summary:</p>
<p>Built on their previous pioneer expertise in studying RAD51 biology, in this paper, the authors aim to capture and investigate the structural mechanism of human RAD51 filament bound with a displacement loop (D-loop), which occurs during the dynamic synaptic state of the homologous recombination (HR) strand-exchange step. As the structures of both pre- and post-synaptic RAD51 filaments were previously determined, a complex structure of RAD51 filament during strand exchange is one of the key missing pieces of information for a complete understanding of how RAD51 functions in HR pathway. This paper aims to determine the high-resolution cryo-EM structure of RAD51 filament bound with D-loop. Combined with mutagenesis analysis and biophysical assays, the authors aim to investigate the D-loop DNA structure, RAD51 mediated strand separation and polarity, and a working model of RAD51 during HR strand invasion in comparison with RecA.</p>
<p>Strengths:</p>
<p>(1) The structural work and associated biophysical assays in this paper are solid, elegantly designed and interpreted. These results provide novel insights into RAD51's function in HR.</p>
<p>(2) The DNA substrate used was well designed, taking into consideration of the nucleotide number requirement of RAD51 for stable capture of donor DNA. This DNA substrate choice lays the foundation for successfully determining the structure of the RAD51 filament on D-loop DNA using single-partial cryo-EM.</p>
<p>(3) The authors utilised their previous expertise in capping DNA ends using monometric streptavidin and combined their careful data collection and processing to determine the cryo-EM structure of full-length human RAD51 bound at D-loop in high resolution. This interesting structure forms the core part of this work and allows detailed mapping of DNA-DNA and DNA-protein interaction among RAD51, invading strands, and donor DNA arms (Figures 1, 2, 3, 4). The geometric analysis of D-loop DNA bound with RAD51 and EM density for homologous DNA pairing are also impressive (Figure S5). The previously disordered RAD51's L2-loop is now ordered and traceable in the density map and functions as a physical spacer when bound with D-loop DNA. Interestingly, the authors identified that the side chain position of F279 in the L2_loop of RAD51_H differs from other F279 residues in L2-loops of E, F and G protomers. This asymmetric binding of L2 loops and RAD51_NTD binding with donor DNA arms forms the basis of the proposed working model about the polarity on csDNA during RAD51-mediated strand exchange.</p>
<p>(4) This work also includes mutagenesis analysis and biophysical experiments, especially EMSA, single-molecule fluorescence imaging using an optical tweezer, and DNA strand exchange assay, which are all suitable methods to study the key residues of RAD51 for strand exchange and D-loop formation (Figure 5).</p>
<p>Weaknesses:</p>
<p>(1) The proposed model for the 3'-5' polarity of RAD51-mediated strand invasion is based on the structural observations in the cryo-EM structure. This study lacks follow-up biochemical/biophysical experiments to validate the proposed model compared to RecA or developing methods to capture structures of any intermediate states with different polarity models.</p>
<p>(2) The functional impact of key mutants designed based on structure has not been tested in cells to evaluate how these mutants impact the HR pathway.</p>
<p>The significance of the work for the DNA repair field and beyond:</p>
<p>Homologous recombination (HR) is a key pathway for repairing DNA double-strand breaks and involves multiple steps. RAD51 forms nucleoprotein filaments first with 3' overhang single-strand DNA (ssDNA), followed by a search and exchange with a homology strand. This function serves as the basis of an accurate template-based DNA repair during HR. This research addressed a long-standing challenge of capturing RAD51 bound with the dynamic synaptic DNA and provided the first structural insight into how RAD51 performs this function. The significance of this work extends beyond the discovery biology for the DNA repair field, into its medical relevance. RAD51 is a potential drug target for inhibiting DNA repair in cancer cells to overcome drug resistance. This work offers a structural understanding of RAD51's function with D-loop and provides new strategies for targeting RAD51 to improve cancer therapies.</p></section></article><article class="sub-article" id="sa4"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. 2025 Aug 18;14:RP107114. doi: <a href="https://doi.org/10.7554/eLife.107114.3.sa4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.107114.3.sa4</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joudeh%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Luay Joudeh</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Luay Joudeh</span></h3>
<div class="p">
<sup>1</sup>University of Cambridge, Cambridge, United Kingdom</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joudeh%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Luay Joudeh</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Appleby%20RA%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Robert A Appleby</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Robert A Appleby</span></h3>
<div class="p">
<sup>2</sup>The Francis Crick Institute, London, United Kingdom</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Appleby%20RA%22%5BAuthor%5D" class="usa-link"><span class="name western">Robert A Appleby</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maman%20JD%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Joseph D Maman</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Joseph D Maman</span></h3>
<div class="p">
<sup>3</sup>University of Cambridge, Cambridge, United Kingdom</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maman%20JD%22%5BAuthor%5D" class="usa-link"><span class="name western">Joseph D Maman</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pellegrini%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Luca Pellegrini</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Luca Pellegrini</span></h3>
<div class="p">
<sup>4</sup>University of Cambridge, Cambridge, United Kingdom</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pellegrini%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Luca Pellegrini</span></a>
</div>
</div>
<sup>4</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.g" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a.g" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.g" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.g" class="d-panel p" style="display: none">
<div class="p" id="aff5">
<sup>1</sup>University of Cambridge, Cambridge, United Kingdom</div>
<div id="aff6">
<sup>2</sup>The Francis Crick Institute, London, United Kingdom</div>
<div id="aff7">
<sup>3</sup>University of Cambridge, Cambridge, United Kingdom</div>
<div id="aff8">
<sup>4</sup>University of Cambridge, Cambridge, United Kingdom</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Luay Joudeh</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Robert A Appleby</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Joseph D Maman</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Luca Pellegrini</span></strong>: <span class="role">Author</span>
</div>
</div>
<div id="anp_a.g" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub5" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a.g" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>The following is the authors’ response to the original reviews.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #1 (Public review):</strong>
</p>
<p>Summary:</p>
<p>The paper describes the cryoEM structure of RAD51 filament on the recombination intermediate. In the RAD51 filament, the insertion of a DNA-binding loop called the L2 loop stabilizes the separation of the complementary strand for the base-pairing with an incoming ssDNA and the non-complementary strand, which is captured by the second DNA-binding channel called the site II. The molecular structure of the RAD51 filament with a recombination intermediate provides a new insight into the mechanism of homology search and strand exchange between ssDNA and dsDNA.</p>
<p>Strengths:</p>
<p>This is the first human RAD51 filament structure with a recombination intermediate called the D-loop. The work has been done with great care, and the results shown in the paper are compelling based on cryo-EM and biochemical analyses. The paper is really nice and important for researchers in the field of homologous recombination, which gives a new view on the molecular mechanism of RAD51-mediated homology search and strand exchange.</p>
<p>Weaknesses:</p>
<p>The authors need more careful text writing. Without page and line numbers, it is hard to give comments.</p>
</blockquote>
<p>We would like to thank the reviewer for their kind words of appreciation of our work.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #2 (Public review):</strong>
</p>
<p>Summary:</p>
<p>Homologous recombination (HR) is a critical pathway for repairing double-strand DNA breaks and ensuring genomic stability. At the core of HR is the RAD51-mediated strand-exchange process, in which the RAD51-ssDNA filament binds to homologous double-stranded DNA (dsDNA) to form a characteristic D-loop structure. While decades of biochemical, genetic, and single-molecule studies have elucidated many aspects of this mechanism, the atomic-level details of the strand-exchange process remained unresolved due to a lack of atomic-resolution structure of RAD51 D-loop complex.</p>
<p>In this study, the authors achieved this by reconstituting a RAD51 mini-filament, allowing them to solve the RAD51 D-loop complex at 2.64 Å resolution using a single particle approach. The atomic resolution structure reveals how specific residues of RAD51 facilitate the strand exchange reaction. Ultimately, this work provides unprecedented structural insight into the eukaryotic HR process and deepens the understanding of RAD51 function at the atomic level, advancing the broader knowledge of DNA repair mechanisms.</p>
<p>Strengths:</p>
<p>The authors overcame the challenge of RAD51's helical symmetry by designing a minifilament system suitable for single-particle cryo-EM, enabling them to resolve the RAD51 D-loop structure at 2.64 Å without imposed symmetry. This high resolution revealed precise roles of key residues, including F279 in Loop 2, which facilitates strand separation, and basic residues on site II that capture the displaced strand. Their findings were supported by mutagenesis, strand exchange assays, and single-molecule analysis, providing strong validation of the structural insights.</p>
<p>Weaknesses:</p>
<p>Despite the detailed structural data, some structure-based mutagenesis data interpretation lacks clarity. Additionally, the proposed 3′-to-5′ polarity of strand exchange relies on assumptions from static structural features, such as stronger binding of the 5′-arm-which are not directly supported by other experiments. This makes the directional model compelling but contradicts several well-established biochemical studies that support a 5'-to-3' polarity relative to the complementary strand (e.g., Cell 1995, PMID: 7634335; JBC 1996, PMID: 8910403; Nature 2008, PMID: 18256600).</p>
<p>Overall:</p>
<p>The 2.6 Å resolution cryoEM structure of the RAD51 D-loop complex provides remarkably detailed insights into the residues involved in D-loop formation. The high-quality cryoEM density enables precise placement of each nucleotide, which is essential for interpreting the molecular interactions between RAD51 and DNA. Particularly, the structural analysis highlights specific roles for key domains, such as the N-terminal domain (NTD), in engaging the donor DNA duplex.</p>
<p>This structural interpretation is further substantiated by single-molecule fluorescence experiments using the KK39,40AA NTD mutant. The data clearly show a significant reduction in D-loop formation by the mutant compared to wild-type, supporting the proposed functional role of the NTD observed in the cryoEM model.</p>
<p>However, the strand exchange activity interpretation presented in Figure 5B could benefit from a more rigorous experimental design. The current assay measures an increase in fluorescence intensity, which depends heavily on the formation of RAD51-ssDNA filaments. As shown in Figure S6A, several mutants exhibit reduced ability to form such filaments, which could confound the interpretation of strand exchange efficiency. To address this, the assay should either: (1) normalize for equivalent levels of RAD51-ssDNA filaments across samples, or (2) compare the initial rates of fluorescence increase (i.e., the slope of the reaction curve), rather than endpoint fluorescence, to better isolate the strand exchange activity itself.</p>
</blockquote>
<p>We agree with the reviewer that the reduced filament-forming ability of some of the RAD51 mutants complicates a straightforward interpretation of their strand-exchange assay. Interestingly, the RAD51 mutants that appear most impaired are the esDNA-capture mutants that do not contact the ssDNA in the structure of the pre-synaptic filament. However, the RAD51 NTD mutants, that display the most severe defect in strand-exchange, have a near-WT filament forming ability.</p>
<blockquote class="text-italic"><p>Based on the structural features of the D-loop, the authors propose that strand pairing and exchange initiate at the 3'-end of the complementary strand in the donor DNA and proceed with a 3'-to-5' polarity. This conclusion, drawn from static structural observations, contrasts with several well-established biochemical studies that support a 5'-to-3' polarity relative to the complementary strand (e.g., Cell 1995, PMID: 7634335; JBC 1996, PMID: 8910403; Nature 2008, PMID: 18256600). While the structural model is compelling and methodologically robust, this discrepancy underscores the need for further experiments.</p></blockquote>
<p>We would like to thank the reviewer for highlighting the importance of our findings to our understanding of the mechanism of homologous recombination.</p>
<p>The reviewer correctly points out that the polarity of strand exchange by RecA and RAD51 is an extensively researched topic that has been characterised in several authoritative studies. In our paper, we simply describe the mechanistic insights obtained from the structural D-loop models of RAD51 (our work) and RecA (Yang et al, PMID: 33057191).The structures illustrate a very similar mechanism of Dloop formation that proceeds with opposite polarity of strand exchange for RAD51 and RecA. Comparison of the D-loop structures for RecA and RAD51 provides an attractive explanation for the opposite polarity, as caused by the different positions of their dsDNA-binding domains in the filament structure.</p>
<p>We agree with the reviewer that further investigation will be needed for an adequate rationalisation of the available evidence. We will mention the relevant literature in the revised version of the manuscript.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #3 (Public review):</strong>
</p>
<p>Summary:</p>
<p>Built on their previous pioneer expertise in studying RAD51 biology, in this paper, the authors aim to capture and investigate the structural mechanism of human RAD51 filament bound with a displacement loop (D-loop), which occurs during the dynamic synaptic state of the homologous recombination (HR) strand-exchange step. As the structures of both pre- and post-synaptic RAD51 filaments were previously determined, a complex structure of RAD51 filaments during strand exchange is one of the key missing pieces of information for a complete understanding of how RAD51 functions in the HR pathway. This paper aims to determine the high-resolution cryo-EM structure of RAD51 filament bound with the D-loop. Combined with mutagenesis analysis and biophysical assays, the authors aim to investigate the D-loop DNA structure, RAD51-mediated strand separation and polarity, and a working model of RAD51 during HR strand invasion in comparison with RecA.</p>
<p>Strengths:</p>
<p>(1) The structural work and associated biophysical assays in this paper are solid, elegantly designed, and interpreted. These results provide novel insights into RAD51's function in HR.</p>
<p>(2) The DNA substrate used was well designed, taking into consideration the nucleotide number requirement of RAD51 for stable capture of donor DNA. This DNA substrate choice lays the foundation for successfully determining the structure of the RAD51 filament on D-loop DNA using single-particle cryo-EM.</p>
<p>(3) The authors utilised their previous expertise in capping DNA ends using monomeric streptavidin and combined their careful data collection and processing to determine the cryo-EM structure of full-length human RAD51 bound at the D-loop in high resolution. This interesting structure forms the core part of this work and allows detailed mapping of DNA-DNA and DNA-protein interaction among RAD51, invading strands, and donor DNA arms (Figures 1, 2, 3, 4). The geometric analysis of D-loop DNA bound with RAD51 and EM density for homologous DNA pairing is also impressive (Figure S5). The previously disordered RAD51's L2-loop is now ordered and traceable in the density map and functions as a physical spacer when bound with D-loop DNA. Interestingly, the authors identified that the side chain position of F279 in the L2_loop of RAD51_H differs from other F279 residues in L2-loops of E, F, and G protomers. This asymmetric binding of L2 loops and RAD51_NTD binding with donor DNA arms forms the basis of the proposed working model about the polarity of csDNA during RAD51-mediated strand exchange.</p>
<p>(4) This work also includes mutagenesis analysis and biophysical experiments, especially EMSA, singlemolecule fluorescence imaging using an optical tweezer, and DNA strand exchange assay, which are all suitable methods to study the key residues of RAD51 for strand exchange and D-loop formation (Figure 5).</p>
<p>Weaknesses:</p>
<p>(1) The proposed model for the 3'-5' polarity of RAD51-mediated strand invasion is based on the structural observations in the cryo-EM structure. This study lacks follow-up biochemical/biophysical experiments to validate the proposed model compared to RecA or developing methods to capture structures of any intermediate states with different polarity models.</p>
<p>(2) The functional impact of key mutants designed based on structure has not been tested in cells to evaluate how these mutants impact the HR pathway.</p>
<p>The significance of the work for the DNA repair field and beyond:</p>
<p>Homologous recombination (HR) is a key pathway for repairing DNA double-strand breaks and involves multiple steps. RAD51 forms nucleoprotein filaments first with 3' overhang single-strand DNA (ssDNA), followed by a search and exchange with a homologous strand. This function serves as the basis of an accurate template-based DNA repair during HR. This research addressed a long-standing challenge of capturing RAD51 bound with the dynamic synaptic DNA and provided the first structural insight into how RAD51 performs this function. The significance of this work extends beyond the discovery of biology for the DNA repair field, into its medical relevance. RAD51 is a potential drug target for inhibiting DNA repair in cancer cells to overcome drug resistance. This work offers a structural understanding of RAD51's function with the D-loop and provides new strategies for targeting RAD51 to improve cancer therapies.</p>
</blockquote>
<p>We thank the reviewer for their positive comments on the significance of our work. Concerning the proposed polarity of strand exchange based on our structural finding, please see our reply to the previous reviewer; we agree with the reviewer that further experimentation will be needed to to reach a settled view on this.</p>
<p>Testing the functional effects of the RAD51 mutants on HR in cells was not an aim of the current work but we agree that it would be a very interesting experiment, which would likely provide further important insights into the mechanism of strand exchange at the core of the HR reaction.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #1 (Recommendations for the authors):</strong>
</p>
<p>Major points:</p>
<p>(1) Structural analysis showed a critical role of F279 in the L2 loop. However, the biochemical study showed that the F279A substitution did not provide a strong defect in the in vitro strand exchange, as shown in Figure 5B. Moreover, a previous study by Matsuo et al. FEBS J, 2006; ref 43 showed human RAD51-F279A is proficient in the in vitro strand exchange. These suggest that human RAD51 F279 is not critical for the strand exchange. The authors need more discussions of the role of F279 or the L2 for the RAD51-mediated reactions in the Discussion.</p>
</blockquote>
<p>In the strand-exchange essay of Figure 5B, the F279A mutant shows the mildest phenotype, in agreement with the findings of Matsuo et al. Accordingly, in the text we describe the F279A mutant as having a “modest impact” on strand-exchange.</p>
<p>We have now added a brief comment to the relevant text, pointing out that the result of the strand exchange assay for F279A are in agreement with the previous findings by Matsuo et al., and adding the reference.</p>
<blockquote class="text-italic"><p>(2) In some parts, the authors cited the newest references rather than the paper describing the original findings. For RAD51 paralogs, why are these three (refs 21,22, 23) selected here? For FIGNL1, why is only one (ref 24) chosen?</p></blockquote>
<p>The cited publications were chosen to acquaint the reader with the latest structural and mechanistic advances about the function of some of the most important and well-studied recombination mediator proteins. For completeness, we have now added a further reference for FIGNL1 - Ito, Masaru et al, Nat Comm, 2023 – in the Introduction, to provide the reader with an additional pointer to our current knowledge about the mechanism of FIGNL1 in Homologous Recombination.</p>
<blockquote class="text-italic">
<p>Minor points:</p>
<p>(1) Page 3, line 1 in the second paragraph, the reaction of "HR": HR should be homology search and strand exchange. HR is used incorrectly throughout the text, please check them. Remove "strandexchange" from ATPases in line 2.</p>
</blockquote>
<p>We believe that HR is used correctly in this context, as we refer to the biochemical reactions of HR, which includes the search for homology and strand exchange.</p>
<p>We have removed “strand-exchange” from ATPases in line 2, as requested by the reviewer.</p>
<blockquote class="text-italic"><p>(2) Supplementary Figure 1B, C, "EMSA" experiment: Please indicate an experimental condition in the legend: how ssDNA and dsDNA were mixed with RAD51. In (B), this is not an actual EMSA result, but rather a native gel analysis of reaction products with the D-loop. In (C), was the binding of RAD51 to the pre-formed D-loop examined? Which is correct here? Moreover, why do the authors need streptavidin in this experiment? Please explain why this is necessary for the EMSA assay. Please show where is Cy3 or Cy5 labels on the DNAs should be shown in the schematic drawing.</p></blockquote>
<p>The conditions for the experiments of Supplementary figure 1B, C are reported in the Methods section.</p>
<p>Panel B shows the mobility shifts of the ssDNA and dsDNA sequences in panel A, so it is appropriate to describe it as an EMSA.</p>
<p>We did not examine the binding of RAD51 to a pre-formed D-loop.</p>
<p>We used streptavidine in the experiment of Supplementary Figure 1C to show that streptavidine binding did not interfere with D-loop reconstitution.</p>
<p>The position of the Cy3, Cy5 labels in the DNAs is reported in Table S1.</p>
<blockquote class="text-italic"><p>(3) Figure S4B, page 6, line 6 from the top, 5'-arm and 3'-arm: please add them to the figure. And also, please explain what 5'-arm and 3'-arm are here in the text, as shown in lines 3-5 in the second paragraph of the same page.</p></blockquote>
<p>We thank the reviewer for spotting this slight incongruity. We have removed the reference to 5’- and 3’arms of the donor DNA in the initial description of the D-loop (first paragraph of the “D-loop structure” section, 6 lines from the top), as the nomenclature for the arms of the donor DNA is introduced more appropriately in the following paragraph. Thus, there is no need to re-label Figure S4B; we note that the 5’- and 3’-labels are added to the arms of the donor DNA in Figure S4D.</p>
<blockquote class="text-italic"><p>(4) Page 7, line 4, and Figure 2E, "C24": C24 should be C26 here (Figure 2D shows that position 24 in esDNA is "T").</p></blockquote>
<p>We thank the reviewer for spotting this typo, that is now corrected in the revised version of Figure 2 and in the text.</p>
<blockquote class="text-italic"><p>(5) Page 8, line 1, K284: It would be nice to show "K284" in Figure 3F.</p></blockquote>
<p>We have added the side chain of K284 to Figure 3F, as suggested by the reviewer.</p>
<blockquote class="text-italic"><p>(6) Page 8, second paragraph, line 3 from the bottom, "5'-arm" should be "3'-arm" for the binding of RAD51A NTD to ds DNA (Figure 4D).</p></blockquote>
<p>We thank the reviewer for spotting this typo, that is now corrected in the revised version of the text.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #2 (Recommendations for the authors):</strong>
</p>
<p>I understand that the strand exchange polarity of RAD51 should be opposite to that of RecA. But in the RecA manuscript (Nature 2020), it states (in the extended figure 1) " Because the mini-filament consists of fused RecA protomers, it does not reflect the effects a preferential polarity of RecA polymerization might have on the directionality of strand exchange. Also, our strand exchange reactions do not include the single-stranded DNA binding protein SSB that is involved in strand exchange in vivo and may sequester released DNA strands."</p>
</blockquote>
<p>We are aware that the findings by Yang et al, 2020 were obtained with a multi-protomeric RecA chimera and that their construct might not therefore recapitulate a potential effect of RecA polymerisation on the directionality of strand-exchange.</p>
<p>Comparison of the RecA and RAD51 D-loop structures shows that RecA and RAD51 adopt the same asymmetric mechanism of D-loop formation, which begins at one arm of the donor DNA and proceeds with donor unwinding and strand invasion until the second arm is captured, completing D-loop formation. However, the cryoEM structures provide compelling evidence that, after engagement with the donor DNA, RecA and RAD51 proceed to unwind the donor with opposite polarity; the structures provide a clear rationale for this, because of the different position of their dsDNA-binding domains relative to the ATPase domain.</p>
<p>We acknowledge that there exists an extensive body of literature that has investigated the polarity of strand exchange by RecA and RAD51 under a variety of experimental conditions, and we have added a brief comment to the text to reflect this, as well as some of the key citations. Undoubtedly, and as we also mention in our reply to the public reviews, further experimental work will be needed for a full reconciliation of the available evidence.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #3 (Recommendations for the authors):</strong>
</p>
<p>(1) I have a minor comment regarding the DNA shown in the structural figures in this work. The authors have used different colours to differentiate between isDNA, esDNA, and csDNA for easier interpretation. However, these colour codes are inconsistent across Figures 1, 2, 3, and 5. This inconsistency makes it difficult to interpret which strand is which, particularly for readers unfamiliar with D-loops and strand invasion. A consistent colour scheme for the DNA strands would enhance the quality of the structural figures.</p>
</blockquote>
<p>We appreciate the reviewer’s comment about the colour scheme of the strands in the D-loop. We chose a unique colour scheme for each figure, to help the reader focus on the particular structural features that we wanted to highlight in the figure. So for instance, in figure 1D we chose to highlight the relationship (complementary vs identical) of the donor DNA strands with the the invading strand; in figure 2, the emphasis is on distinguishing the homologously paired dsDNA (pink) from the exchanged strand (magenta), as a consequence of L2 loop binding; etc.</p>
<blockquote class="text-italic"><p>(2) I have another comment regarding the rationale behind naming the RAD51 protomers (A to H) within the structure, which could confuse general readers if not clearly explained. In this paper, the RAD51 protomer is RAD51_A when closest to the 3' end of the isDNA. I assume the authors chose this order because HR generates a 3' ssDNA overhang before strand invasion. It would be beneficial for the introduction and results sections to mention this property of the 3' ssDNA overhang and the reasoning behind this naming strategy. This explanation will help readers understand how it differs from other naming orders used in RecA/RAD51 with ssDNA, where protomer A is closer to the 5' ssDNA.</p></blockquote>
<p>We thank the reviewer for their insightful comment. We chose to name as chain A the RAD51 protomer nearest to the 3’-end of the isDNA to be consistent with the naming scheme that we use for all our published RAD51 filament structures.</p>
<blockquote class="text-italic">
<p>(3) I have highlighted some text within this paper that has contradicting parts for authors to clarify and correct:</p>
<p>"Overall, the structural features of the RAD51 D-loop provide a strong indication that strand pairing and exchange begins at the 3'-end of the complementary strand in the donor DNA and progresses with 3'-to5' polarity (Fig. 5F)"</p>
<p>"The observed 5'-to-3' polarity of strand-exchange by RAD51 is opposite to the 3'-to-5' polarity of bacterial RecA (Fig. S8), that was determined based on cryoEM structures of RecA D-loops".</p>
</blockquote>
<p>We thank the reviewer for alerting us to this inconsistency that has now been corrected in the revised manuscript.</p>
<blockquote class="text-italic"><p>(4) Figure S8 last model: NTD should be CTD in the title; Figure 2B: resolution scale bar needs A unit. We thank the reviewer for spotting this typo that has now been corrected in the revised version of figure S8.</p></blockquote>
<p>We couldn’t find a missing resolution scale bar in Figure 2B; however, we have added a missing resolution bar with A unit to Fig. S3B.</p></section></article><article class="sub-article" id="_ad93_"><section class="pmc-layout__citation font-secondary font-xs"><div></div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Associated Data</h1></hgroup><ul class="d-buttons inline-list"></ul>
<div class="d-panels font-secondary-light"></div>
<div></div>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
</div></section><section class="body sub-article-body"><section id="_adc93_" lang="en" class="data-citations"><h2 class="pmc_sec_title">Data Citations</h2>
<ol class="ref-list font-sm">
<li id="_dbx_ref_dataset1">
<cite>Pellegrini L, Joudeh L, Appleby RE. 2025. CryoEM structure of a RAD51 D-loop. Worldwide Protein Data Bank. </cite> [<a href="https://doi.org/10.2210/pdb9i62/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="_dbx_ref_dataset2"><cite>Pellegrini L, Joudeh L, Appleby RE. 2025. CryoEM structure of a RAD51 D-loop. Electron Microscopy Data Bank. EMD-52646</cite></li>
</ol></section><section id="_adsm93_" lang="en" class="supplementary-materials"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Figure 1—figure supplement 1—source data 1. Original gel images for <a href="#fig1s1" class="usa-link">Figure 1—figure supplement 1B and C</a>.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-fig1-figsupp1-data1.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-107114-fig1-figsupp1-data1.zip</a><sup> (8.1MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><span>Figure 1—figure supplement 1—source data 2. Uncropped gel images for <a href="#fig1s1" class="usa-link">Figure 1—figure supplement 1A and B</a>.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-fig1-figsupp1-data2.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-107114-fig1-figsupp1-data2.zip</a><sup> (667.6KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><span>Figure 5—figure supplement 1—source data 1. Original gel images for <a href="#fig5s1" class="usa-link">Figure 5—figure supplement 1A and B</a>.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-fig5-figsupp1-data1.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-107114-fig5-figsupp1-data1.zip</a><sup> (18.5MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><span>Figure 5—figure supplement 1—source data 2. Uncropped gel images for <a href="#fig5s1" class="usa-link">Figure 5—figure supplement 1A and B</a>.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-fig5-figsupp1-data2.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-107114-fig5-figsupp1-data2.zip</a><sup> (766.8KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p"><span>Supplementary file 1. Oligonucleotide sequence, size, and labelling.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-supp1.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-107114-supp1.docx</a><sup> (15.6KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p"><span>Supplementary file 2. CryoEM data collection and real-space refinement.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-supp2.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-107114-supp2.docx</a><sup> (28.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p"><span>MDAR checklist</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360782/bin/elife-107114-mdarchecklist1.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-107114-mdarchecklist1.docx</a><sup> (87.4KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>Atomic coordinates and cryoEM maps for the human RAD51 D-loop have been deposited in the PDB and EMDB under accession codes 9I62 and EMD-52646, respectively.</p>
<p>The following datasets were generated:</p>
<p>
Pellegrini L, Joudeh L, Appleby RE. 2025. CryoEM structure of a RAD51 D-loop. Worldwide Protein Data Bank. 
</p>
<p>
Pellegrini L, Joudeh L, Appleby RE. 2025. CryoEM structure of a RAD51 D-loop. Electron Microscopy Data Bank. EMD-52646
</p></section></section></article><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from eLife are provided here courtesy of <strong>eLife Sciences Publications, Ltd</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.7554/eLife.107114"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/elife-107114.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (10.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12360782/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12360782/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12360782%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360782/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12360782/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12360782/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40820711/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12360782/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40820711/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12360782/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12360782/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="JeZp0feLI7Hhejx97FnxoHgT2Mc3Uyh5Gyolr1p1tFgKfHUcz9I62ELs57zg2CqC">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
