
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genetic regulation of sperm DNA methylation in cattle through meQTL mapping - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15253B7D8AF23023053B7D000034E404.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcgeno">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374274/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Genomics">
<meta name="citation_title" content="Genetic regulation of sperm DNA methylation in cattle through meQTL mapping">
<meta name="citation_author" content="Corentin Fouéré">
<meta name="citation_author_institution" content="Eliance, 149 Rue de Bercy, Paris, 75012 France">
<meta name="citation_author_institution" content="Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France">
<meta name="citation_author" content="Valentin Costes">
<meta name="citation_author_institution" content="Eliance, 149 Rue de Bercy, Paris, 75012 France">
<meta name="citation_author_institution" content="Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France">
<meta name="citation_author_institution" content="Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France">
<meta name="citation_author" content="Chris Hozé">
<meta name="citation_author_institution" content="Eliance, 149 Rue de Bercy, Paris, 75012 France">
<meta name="citation_author_institution" content="Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France">
<meta name="citation_author" content="Amrita Raja Ravi Shankar">
<meta name="citation_author_institution" content="Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France">
<meta name="citation_author_institution" content="Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France">
<meta name="citation_author" content="Florian Besnard">
<meta name="citation_author_institution" content="Eliance, 149 Rue de Bercy, Paris, 75012 France">
<meta name="citation_author_institution" content="Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France">
<meta name="citation_author" content="Gabriel Costa Monteiro Moreira">
<meta name="citation_author_institution" content="Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France">
<meta name="citation_author_institution" content="Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France">
<meta name="citation_author" content="Valentin Sorin">
<meta name="citation_author_institution" content="Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France">
<meta name="citation_author" content="Chrystelle Le Danvic">
<meta name="citation_author_institution" content="Eliance, 149 Rue de Bercy, Paris, 75012 France">
<meta name="citation_author_institution" content="Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France">
<meta name="citation_author_institution" content="Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France">
<meta name="citation_author" content="Aurélie Chaulot-Talmon">
<meta name="citation_author_institution" content="Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France">
<meta name="citation_author_institution" content="Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France">
<meta name="citation_author" content="Francesca Ali">
<meta name="citation_author_institution" content="Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France">
<meta name="citation_author_institution" content="Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France">
<meta name="citation_author" content="Marie Christine Deloche">
<meta name="citation_author_institution" content="Eliance, 149 Rue de Bercy, Paris, 75012 France">
<meta name="citation_author_institution" content="Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France">
<meta name="citation_author_institution" content="Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France">
<meta name="citation_author" content="Aurélie Bonnet">
<meta name="citation_author_institution" content="Eliance, 149 Rue de Bercy, Paris, 75012 France">
<meta name="citation_author_institution" content="Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France">
<meta name="citation_author_institution" content="Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France">
<meta name="citation_author" content="Eliaou Sellem">
<meta name="citation_author_institution" content="Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France">
<meta name="citation_author_institution" content="Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France">
<meta name="citation_author" content="Hélène Jammes">
<meta name="citation_author_institution" content="Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France">
<meta name="citation_author_institution" content="Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France">
<meta name="citation_author" content="Sébastien Fritz">
<meta name="citation_author_institution" content="Eliance, 149 Rue de Bercy, Paris, 75012 France">
<meta name="citation_author_institution" content="Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France">
<meta name="citation_author" content="Mekki Boussaha">
<meta name="citation_author_institution" content="Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France">
<meta name="citation_author" content="Didier Boichard">
<meta name="citation_author_institution" content="Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France">
<meta name="citation_author" content="Hélène Kiefer">
<meta name="citation_author_institution" content="Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France">
<meta name="citation_author_institution" content="Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France">
<meta name="citation_author" content="Marie-Pierre Sanchez">
<meta name="citation_author_institution" content="Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="771">
<meta name="citation_doi" content="10.1186/s12864-025-11934-x">
<meta name="citation_pmid" content="40847283">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374274/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374274/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374274/pdf/12864_2025_Article_11934.pdf">
<meta name="description" content="DNA methylation (DNAm) plays an important functional role and is influenced by genetic variants known as methylation QTLs (meQTLs). The majority of meQTL studies have been conducted in human blood. Despite its unique landscape, the genetic ...">
<meta name="og:title" content="Genetic regulation of sperm DNA methylation in cattle through meQTL mapping">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="DNA methylation (DNAm) plays an important functional role and is influenced by genetic variants known as methylation QTLs (meQTLs). The majority of meQTL studies have been conducted in human blood. Despite its unique landscape, the genetic ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374274/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12374274">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12864-025-11934-x"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12864_2025_Article_11934.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374274%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12374274/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12374274/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374274/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcgeno.png" alt="BMC Genomics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Genomics" title="Link to BMC Genomics" shape="default" href="https://bmcgenomics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Genomics</button></div>. 2025 Aug 22;26:771. doi: <a href="https://doi.org/10.1186/s12864-025-11934-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12864-025-11934-x</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Genomics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Genomics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genetic regulation of sperm DNA methylation in cattle through meQTL mapping</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fou%C3%A9r%C3%A9%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Corentin Fouéré</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Corentin Fouéré</span></h3>
<div class="p">
<sup>1</sup>Eliance, 149 Rue de Bercy, Paris, 75012 France </div>
<div class="p">
<sup>2</sup>Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fou%C3%A9r%C3%A9%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Corentin Fouéré</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Costes%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Valentin Costes</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Valentin Costes</span></h3>
<div class="p">
<sup>1</sup>Eliance, 149 Rue de Bercy, Paris, 75012 France </div>
<div class="p">
<sup>3</sup>Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France </div>
<div class="p">
<sup>4</sup>Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Costes%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Valentin Costes</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoz%C3%A9%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Chris Hozé</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Chris Hozé</span></h3>
<div class="p">
<sup>1</sup>Eliance, 149 Rue de Bercy, Paris, 75012 France </div>
<div class="p">
<sup>2</sup>Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoz%C3%A9%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chris Hozé</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Raja%20Ravi%20Shankar%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Amrita Raja Ravi Shankar</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Amrita Raja Ravi Shankar</span></h3>
<div class="p">
<sup>3</sup>Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France </div>
<div class="p">
<sup>4</sup>Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Raja%20Ravi%20Shankar%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Amrita Raja Ravi Shankar</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Besnard%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Florian Besnard</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Florian Besnard</span></h3>
<div class="p">
<sup>1</sup>Eliance, 149 Rue de Bercy, Paris, 75012 France </div>
<div class="p">
<sup>2</sup>Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Besnard%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Florian Besnard</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Costa%20Monteiro%20Moreira%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Gabriel Costa Monteiro Moreira</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Gabriel Costa Monteiro Moreira</span></h3>
<div class="p">
<sup>3</sup>Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France </div>
<div class="p">
<sup>4</sup>Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Costa%20Monteiro%20Moreira%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gabriel Costa Monteiro Moreira</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sorin%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Valentin Sorin</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Valentin Sorin</span></h3>
<div class="p">
<sup>2</sup>Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sorin%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Valentin Sorin</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Le%20Danvic%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Chrystelle Le Danvic</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Chrystelle Le Danvic</span></h3>
<div class="p">
<sup>1</sup>Eliance, 149 Rue de Bercy, Paris, 75012 France </div>
<div class="p">
<sup>3</sup>Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France </div>
<div class="p">
<sup>4</sup>Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Le%20Danvic%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chrystelle Le Danvic</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chaulot-Talmon%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Aurélie Chaulot-Talmon</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Aurélie Chaulot-Talmon</span></h3>
<div class="p">
<sup>3</sup>Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France </div>
<div class="p">
<sup>4</sup>Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chaulot-Talmon%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Aurélie Chaulot-Talmon</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ali%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Francesca Ali</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Francesca Ali</span></h3>
<div class="p">
<sup>3</sup>Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France </div>
<div class="p">
<sup>4</sup>Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ali%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Francesca Ali</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deloche%20MC%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Marie Christine Deloche</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Marie Christine Deloche</span></h3>
<div class="p">
<sup>1</sup>Eliance, 149 Rue de Bercy, Paris, 75012 France </div>
<div class="p">
<sup>3</sup>Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France </div>
<div class="p">
<sup>4</sup>Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deloche%20MC%22%5BAuthor%5D" class="usa-link"><span class="name western">Marie Christine Deloche</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bonnet%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Aurélie Bonnet</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Aurélie Bonnet</span></h3>
<div class="p">
<sup>1</sup>Eliance, 149 Rue de Bercy, Paris, 75012 France </div>
<div class="p">
<sup>3</sup>Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France </div>
<div class="p">
<sup>4</sup>Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bonnet%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Aurélie Bonnet</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sellem%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Eliaou Sellem</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Eliaou Sellem</span></h3>
<div class="p">
<sup>3</sup>Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France </div>
<div class="p">
<sup>4</sup>Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sellem%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Eliaou Sellem</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jammes%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Hélène Jammes</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Hélène Jammes</span></h3>
<div class="p">
<sup>3</sup>Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France </div>
<div class="p">
<sup>4</sup>Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jammes%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hélène Jammes</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fritz%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Sébastien Fritz</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Sébastien Fritz</span></h3>
<div class="p">
<sup>1</sup>Eliance, 149 Rue de Bercy, Paris, 75012 France </div>
<div class="p">
<sup>2</sup>Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fritz%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sébastien Fritz</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Boussaha%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Mekki Boussaha</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Mekki Boussaha</span></h3>
<div class="p">
<sup>2</sup>Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Boussaha%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mekki Boussaha</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Boichard%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Didier Boichard</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Didier Boichard</span></h3>
<div class="p">
<sup>2</sup>Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Boichard%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Didier Boichard</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kiefer%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Hélène Kiefer</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Hélène Kiefer</span></h3>
<div class="p">
<sup>3</sup>Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France </div>
<div class="p">
<sup>4</sup>Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kiefer%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hélène Kiefer</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanchez%20MP%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Marie-Pierre Sanchez</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Marie-Pierre Sanchez</span></h3>
<div class="p">
<sup>2</sup>Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanchez%20MP%22%5BAuthor%5D" class="usa-link"><span class="name western">Marie-Pierre Sanchez</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Eliance, 149 Rue de Bercy, Paris, 75012 France </div>
<div id="Aff2">
<sup>2</sup>Université Paris-Saclay, INRAE, AgroParisTech, GABI, Jouy-en-Josas, 78352 France </div>
<div id="Aff3">
<sup>3</sup>Université Paris-Saclay, UVSQ, INRAE, BREED, Jouy-en-Josas, 78350 France </div>
<div id="Aff4">
<sup>4</sup>Ecole Nationale Vétérinaire d’Alfort, BREED, Maisons-Alfort, 94700 France </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 27; Accepted 2025 Jul 22; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12374274  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40847283/" class="usa-link">40847283</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">DNA methylation (DNAm) plays an important functional role and is influenced by genetic variants known as methylation QTLs (meQTLs). The majority of meQTL studies have been conducted in human blood. Despite its unique landscape, the genetic regulation of sperm DNAm remains largely unexplored. In this study, we leveraged DNAm measured in sperm from 405 Holstein bulls using reduced representation bisulfite sequencing (RRBS) and performed sequence-level genome-wide association studies for 166,985 variable CpGs (s.d. &gt;5%). We reported heritability estimates and have mapped both <em>cis-</em>meQTLs and <em>trans-</em>meQTLs.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">Heritability estimates ranged from 0 to 1 and averaged 0.26 across all selected CpGs, with 76% of estimates above 0.1. The meQTL mapping revealed that 32.9% of the CpGs had a <em>cis-</em>meQTL, 3.6% had a <em>trans-</em>meQTL and 1.0% had both <em>cis-</em> and <em>trans-</em>meQTLs. The <em>cis-</em>CpGs were located on average 261 kb (absolute mean) from their <em>cis-</em>meQTL top SNPs (defined by the most significant association). MeQTLs were enriched in featured genomic annotations, including regions surrounding transcription start sites and ATAC-seq peaks. We also identified spurious <em>trans-</em>associations by analyzing data across multiple genome assemblies, including the construction of a partial pangenome. Additionally, eight <em>trans-</em>meQTL hotspots, defined as variants associated with at least 30 <em>trans-</em>CpGs, were identified and overlapped with genes involved in epigenetic regulation. Using peripheral blood mononuclear cell DNAm from 54 out of the 405 bulls, we did not observe a similar effect of the <em>trans-</em>meQTL hotspots to that one observed in sperm.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">For the first time, meQTLs have been detected and characterized in bovine sperm, contributing to a better understanding of the transmission of paternally inherited DNAm marks. These findings provide useful information for further research aimed at integrating epigenetic information into the prediction of performance traits.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12864-025-11934-x.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> MeQTL mapping, GWAS, DNA methylation, Cattle, Semen</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par4">DNA methylation (DNAm) in eukaryotes corresponds to the addition of a methyl group to the 5-carbon position of the cytosine pyrimidine ring mainly within CpG dinucleotides, resulting in 5-methylcytosine [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. DNAm is a key epigenetic mark that regulates gene expression, is involved in cell differentiation [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>] and maintenance of cell identity [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>], and can lead to phenotypic changes in response to environmental exposures [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>].</p>
<p id="Par5">Large-scale epigenotyping (e.g., DNAm arrays) providing DNAm for several thousands to several hundreds of thousands of CpGs, has facilitated the study of genetic influence on DNAm, mainly in humans. The heritability reported for these sites is predominantly low to moderate, ranging between 0.1 and 0.3 [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Genome-wide association studies (GWAS) have significantly advanced our understanding of gene regulation by mapping methylation quantitative trait loci (meQTL) that influence DNAm either at nearby (<em>cis</em>; within 1 Mb) or distant (<em>trans</em>; &gt;1 Mb or interchromosomal) genomic locations [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. The majority of meQTL are <em>cis</em>-acting [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>] and the proportion of reported <em>trans</em>-meQTL varies depending on the statistical power and significance threshold considered [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. A recent study, based on blood DNAm from the 450 K array of 27,750 European individuals, reported meQTL associated with 45% of CpGs, out of which 8.5% were <em>trans</em>-meQTLs [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. To further explore the biological links between gene regulation, environment, and phenotype, several studies have integrated the identification of meQTL into the study of complex diseases, e.g., by combining meQTL with expression QTL (eQTL) [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. However, only a few studies have investigated DNAm genetics in farm animal species [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>] and notably, their heritability’s estimates for DNAm measured in blood, are comparable to those found in human studies (i.e., 0.2 for global DNAm in sheep [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>] and ranging from 0.18 to 0.72 across 34,324 CpGs using the HorvathMammalMethylChip40 array in cattle [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]).</p>
<p id="Par6">Germ cells undergo extensive epigenetic reprogramming during their development, involving the erasure and re-establishment of their methylome [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Similar to other mammalian species, bovine spermatozoa display distinct methylation profile compared to somatic cells [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. This unique remodeling of DNAm landscape plays a crucial role in biological processes related to fertilization, post-fertilization reprogramming and the acquisition of totipotency [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. As most studies have focused on blood DNAm, the genetic impacts of DNAm in sperm remains poorly understood, highlighting a need for further investigation.</p>
<p id="Par7">Disentangling genetic and environmental factors affecting DNAm is of particular interest in sperm as it contains epigenetic marks that could be transmitted to the next generation. In the present study, we aimed to characterize the genetic architecture of DNAm determined through Reduced Representation Bisulfite Sequencing (RRBS), in sperm of 405 Holstein bulls. We estimated heritability and identified both <em>cis-</em> and <em>trans-</em>meQTLs associated with CpGs displaying variable DNAm, and further investigated these meQTL and their associated CpGs.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Animals, samples and genotypes</h3>
<p id="Par8">Semen samples were collected from 405 marketed Holstein bulls born between 2001 and 2021. The samples were obtained from frozen straws stored at −196 °C for on-farm artificial insemination. At the time of semen collection, the bulls ranged in age from 10 months to 7.3 years. Depending on the bulls, either pools of ejaculates or individual ejaculates (from one to nine samples per bull) were assessed. For 54 out of the 405 bulls, blood samples were collected from 15 months to 21 months of age. Peripheral blood mononuclear cells (PBMCs) were isolated from blood by centrifugation using a Ficoll gradient. The metadata for the samples can be found in Supplementary Table 1. All biological samples were collected with the informed consent of the breeders to whom the bulls belonged.</p>
<p id="Par9">All bulls were genotyped using medium density SNP arrays (including the 50 K SNP Beadchip (Illumina, San Diego, CA, USA) and EuroGMD Beadchip (<a href="https://www.eurogenomics.com/actualites/the-eurog-md:-a-unique-genotyping-microarray-for-cattle-.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.eurogenomics.com/actualites/the-eurog-md:-a-unique-genotyping-microarray-for-cattle-.html</a>, Illumina, San Diego, CA, USA)). After quality control (as previously described by Sanchez et al. [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]) and imputation to the EuroGMD Beadchip, 53,469 autosomal SNPs were retained. The EuroGMD genotypes were then imputed up to whole-genome sequence level in a two-step approach: from EuroGMD to 777 K SNPs (BovineHD BeadChip; Illumina, San Diego, CA, USA) using FImpute [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>], and then from 777 K SNPs to whole-genome sequence using Minimac [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>], as described in Sanchez et al. [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. We selected only variants with a Minimac coefficient of determination (r<sup>2</sup>) above 0.3. Finally, following the recommendations of Sahana et al., [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>] we used a 3% minor allele frequency (MAF) threshold to ensure at least 20 copies of the rare allele, yielding a final dataset of 12,930,389 variants.</p></section><section id="Sec4"><h3 class="pmc_sec_title">DNA methylation</h3>
<p id="Par10">The equivalent of one straw of bull semen was used for DNA extraction (about 20 million spermatozoa). After thawing at 37 °C, the semen was washed with phosphate buffer saline (PBS) to remove the extender, washed again with H<sub>2</sub>O to lysate somatic cells, and incubated overnight at 55 °C in 200 µl lysis buffer (10 mM Tris-HCl pH 7.5, 25 mM EDTA, 1% SDS, 75 mM NaCl, 50 mM dithiothreitol (DTT) and 0.5 µg glycogen) in the presence of 0.2 mg/ml proteinase K. After incubation with 25 µg/ml RNAse A for 1 h at 37 °C, genomic DNA was extracted twice using phenol: chloroform (1:1) and chloroform, then ethanol precipitated and washed. The dried pellet was re-suspended in TE buffer (10 mM Tris HCl pH7.5, 2 mM EDTA) and the DNA concentration was measured using a Qubit 2.0 Fluorometer (Invitrogen). A similar procedure was applied to extract DNA from PBMCs, except that DTT was omitted. The integrity of DNA was checked using agarose electrophoresis.</p>
<p id="Par11">For one subset of sperm samples (see Supplementary Table 1 for the detailed list), the RRBS libraries were constructed using a semi-automated procedure described in details in Costes et al. [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. The other libraries were constructed by Integragen SA using a similar protocol based on MspI and the selection of genomic DNA fragments spanning from 40 to 290 bp. Sequencing was carried-out by Integragen SA to produce 75 bp paired-end reads (HiSeq4000, Illumina) or 100 bp paired-end reads (NovaSeq6000 and NovaSeq X Plus, Illumina).</p>
<p id="Par12">On average, sequencing generated around 48 million read pairs per library. The sequences displayed the expected nucleotide composition based on MspI digestion and bisulfite conversion according to FastQC quality control. Subsequent quality checks and trimming were carried out using TrimGalore v0.4.4 [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] which removed adapter sequences, poor quality bases (Phred score below 20) and reads shorter than 20 nucleotides. The average bisulfite conversion rate was 99% and was estimated from unmethylated cytosine added in vitro during the end-repair step. High quality reads were aligned on the bovine reference genome (ARS-UCD1.2) using Bismark v0.20.0 [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>] in the default mode with hisat2. After alignment, per sample, only CpGs covered by at least 10 uniquely mapped reads were retained (median = 27 reads per CpG). For these CpGs, a methylation percentage was calculated from Bismark methylation calling (number of reads with “C” × 100)/(number of reads with “C” + number of reads with “T”).</p>
<p id="Par13">For the bulls with several RRBS samples analyzed, we computed the average DNAm percentage by CpG weighted by the number of reads in each sample. We then retained all CpGs with methylation information for at least 80% of the bulls. Any CpG that overlapped with known variants [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>] was discarded. We then selected CpGs with a DNAm standard deviation (s.d.) &gt;5% and corrected the methylation levels at each site for age at collection using four classes (&lt; 500 d; [500d; 750 d];]750d; 1000 d]; &gt;1000d).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Genetic analysis</h3>
<section id="Sec6"><h4 class="pmc_sec_title">meQTL mapping</h4>
<p id="Par14">We performed single-trait analyses between the 12,930,389 variants and the 166,985 CpGs. The software GCTA (version 1.93) [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>] with the <em>mlma</em> option was used to test the effect of each variant in the following mixed linear model:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/1fbee93696dc/d33e573.gif" loading="lazy" id="d33e573" alt="graphic file with name d33e573.gif"></td>
<td class="label">1</td>
</tr></table>
<p id="Par15">where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/efcc4ad69ea7/d33e581.gif" loading="lazy" id="d33e581" alt="Inline graphic"></span> is the vector of methylation values for a given CpG, <strong>1</strong> is a vector of 1 s, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/522dcb2e5063/d33e590.gif" loading="lazy" id="d33e590" alt="Inline graphic"></span> is the overall mean, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/07f08451fa63/d33e596.gif" loading="lazy" id="d33e596" alt="Inline graphic"></span> is the additive fixed effect of the variant to be tested for association, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/fdfff813a087/d33e602.gif" loading="lazy" id="d33e602" alt="Inline graphic"></span> is the vector of SNP genotypes coded as 0, 1 or 2, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/cda2e8a88806/d33e609.gif" loading="lazy" id="d33e609" alt="Inline graphic"></span> is the vector of random polygenic effects, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/19a66f96ad67/d33e615.gif" loading="lazy" id="d33e615" alt="Inline graphic"></span> the genomic relationship matrix (GRM), calculated using the EuroGMD SNPs genotypes, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/77d84138d3ea/d33e621.gif" loading="lazy" id="d33e621" alt="Inline graphic"></span> is the polygenic variance, estimated based on the null model (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/c2c087749ef7/d33e627.gif" loading="lazy" id="d33e627" alt="Inline graphic"></span>) and then fixed while testing for the association between each variant and the given CpG. Genomic inflation factor (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/989f9b29148a/d33e633.gif" loading="lazy" id="d33e633" alt="Inline graphic"></span>) was computed for each GWAS (from the median <em>P</em>-value). <em>P</em>-values were then adjusted for their corresponding <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/989f9b29148a/d33e646.gif" loading="lazy" id="d33e646" alt="Inline graphic"></span>. MeQTLs were classified as <em>cis</em> if the distance between the CpG and the variant was within 1 Mb and as <em>trans</em> if this distance was greater than 2 Mb on the same chromosome or if the variant and the CpG were located on different chromosomes. Intrachromosomal <em>trans-</em>associations within 2 and 20 Mb for CpG that also shared a <em>cis-</em>meQTL were considered as long-range <em>cis-</em>meQTL and were excluded from the analysis.</p></section><section id="Sec7"><h4 class="pmc_sec_title">Multiple testing correction for meQTL mapping</h4>
<p id="Par16">We applied permutation testing to determine significant thresholds for both <em>cis-</em> and <em>trans-</em>meQTLs. For each permutation, genotypes were randomly reassigned to bulls while maintaining the original EuroGMD SNP-based GRM. This approach accounted for population structure while disrupting the link between genotype and phenotype. To reduce the computational burden, GWASs were performed for 10% of CpGs (randomly selected) with permuted genotypes, and the lowest <em>P</em>-value (for <em>trans-</em>meQTLs), as well as the lowest <em>P</em>-value within 1 Mb (for <em>cis-</em>meQTL), were retained to approximate the distributions of <em>P</em>-values under the null hypothesis. This process was repeated 10 times, similar to previous studies [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. The resulted significance thresholds were <em>P</em> &lt; 1.38 × 10<sup>−4</sup> for <em>cis-</em>meQTLs and <em>P</em> &lt; 1.25 × 10<sup>−8</sup> for <em>trans-</em>meQTLs, which corresponded to an empirical false positive rate of 5%. Supplementary Table 2 shows the result of each of the 10 permutations.</p></section><section id="Sec8"><h4 class="pmc_sec_title">Identification of spurious <em>trans-</em>meQTLs</h4>
<p id="Par17">Some <em>trans</em> associations showed a specific pattern in which several CpGs, clustered within a few hundred to a few thousand bp, were almost exclusively associated with variants clustered in a different genomic region, sometimes spanning tens of Mb. To investigate these associations, we retrieved the reference genome sequence corresponding to the CpGs cluster and the associated group of variants and aligned them to the publicly available Holstein-Friesian genome assembly (accession number = PRJNA668863) using BLASTN [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] available online [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. If the sequence of a CpG cluster aligned to the genomic region containing its <em>trans-</em>associated variants, the CpG-SNP <em>trans-</em>pairs were considered false positives and discarded from further analyses. One notable case involved a CpG cluster spanning 1152 bp on BTA3 associated with variants located in a large region (40 Mb-65 Mb) on BTA9, for which BLASTN alignment of the CpG region did not identify a match in the BTA9 region defined by the variants with the significant effects. To further investigate this CpG cluster, we built a partial pangenome of the 35 Mb-70 Mb region on BTA9 using the reference genome and 8 Holstein genome assemblies (accession number = PRJEB68295). This was achieved through reference-free all-to-all alignment using PanGenome Graph Builder [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. The resulting pangenome graph was visualized with BandageNG [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>] to assess potential alignments and structural variations. This tool allowed BLAST searching of the CpG region within this pangenome.</p></section><section id="Sec10"><h4 class="pmc_sec_title">Heritability</h4>
<p id="Par18">GRM-based heritability <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/fcf36405dd94/d33e751.gif" loading="lazy" id="d33e751" alt="Inline graphic"></span> was estimated individually for each CpG. Estimates of variance explained by the EuroGMD variants (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/77d84138d3ea/d33e757.gif" loading="lazy" id="d33e757" alt="Inline graphic"></span>) and residual variance (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/5593c52984df/d33e763.gif" loading="lazy" id="d33e763" alt="Inline graphic"></span>) were obtained from the REML method implemented in GCTA [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>].</p></section></section><section id="Sec11"><h3 class="pmc_sec_title">Functional annotations</h3>
<p id="Par19">CpGs and SNPs were annotated according to several datasets. First we derived annotations retrieved from Ensembl v112 [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. Each CpG or variant was assigned to a genomic feature following this priority order: TSS200 (0-200 bases upstream of the transcription start site (TSS)) &gt; TSS1500 (0-1500 bases upstream of the TSS) &gt; 5′ UTR &gt;1 st exon &gt; 3′UTR &gt; body &gt; intergenic. We also retrieved CGI coordinates from the UCSC genome browser (“cpgIslandExt” table) [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. To further characterize the CGI landscape, we have defined two additional regions around the CpG islands: (1) CGI shore regions, located 0-2 kb from CpG islands and (2) CGI shelf regions, located 2-4 kb regions from CpG islands. CpGs and SNPs outside these regions were assigned to the “open sea” feature. Genomic coordinates of Transposable Elements (TE) were obtained from the UCSC genome browser’s RepeatMasker Table [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>], focusing on regions annotated as LTR, SINE, LINE (class I Retrotransposons), and DNA (class II Transposons). In addition to these annotations, we have also made use of a recent bovine TSS database obtained from Cap Analysis Gene Expression sequencing from the H2020 BovReg project [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>], differentiating TSS reported in testis and ubiquitous TSS (minimum coverage of 10 reads and, commonly detected in a minimum of 66 out of the 102 tissues utilized in the study). For the TSS in testis, we annotated CpGs or SNPs using the following priority order: TSS+−200 (200 bp around the TSS) &gt; TSS+−1500 (1500 bp around the TSS) &gt; intergenic. For the ubiquitous TSS, the pattern was similar: TSS+−200 &gt; TSS+−1500 &gt; intergenic. Finally, we included ATAC-seq peak annotations from various cattle tissues (obtained from Yuan et al. [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]), retaining only peaks classified under the largest nonnegative matrix factorization components as “ubiquitous” or “testis”.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Enrichment analyses</h3>
<p id="Par20">We computed odds ratios (OR) to look for enrichment and depletion in genomic regions. As neighbouring CpGs may be highly correlated and share a common methylation pattern, enrichment analyses were performed by considering a reduced set of “independent” CpGs. From the 166,985 CpGs, all adjacent CpGs with a correlation (R) &gt; 0.5 were grouped into clusters and the genomic position considered was defined by its central CpG, resulting in 103,719 independent CpGs (see Supplementary Table 3 for the detailed list). We first tested enrichment of CpGs (<em>N</em> = 103,719) by genomic annotation to determine whether some genomic regions were enriched for CpGs with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/e8c2d1182da1/d33e798.gif" loading="lazy" id="d33e798" alt="Inline graphic"></span>&gt;0.1 as described in Huan et al. [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>], using 2-sided Fisher exact test. <em>P</em>-values were adjusted for multiple comparisons across two genomic annotation categories (CGI, genomic features). We then performed separate enrichment analysis for <em>cis-</em> and <em>trans-</em>CpG, along with their corresponding most significant <em>cis-</em> and <em>trans-</em>SNP (“top SNP”), considering only independent CpGs. For each, the OR was calculated against a randomly selected background set of the same size as the set of interest. This procedure was repeated 1,000 times to obtain the mean OR and 95% confidence interval (CI). For <em>cis-</em> and <em>trans-</em>CpGs, the background set consisted of randomly selected CpGs from all the independent CpGs (<em>N</em> = 103,719). For top <em>cis-</em>SNPs, we randomly selected the background set from all <em>cis-</em>SNPs tested (approximately 1.5 billion associations CpG-SNP) while maintaining MAF and distance to CpG similar to the one observed in the top <em>cis-</em>SNP set (set of interest). For that, we used four MAF classes ([0.03; 0.05[, [0.05; 0.1[, [0.1; 0.2[, [0.2; 0.5]) and seven distance classes relative to the corresponding CpG ([−1 Mb; −500 kb[, [−500 kb; −100 kb[, [−100 kb; −10 kb[, [−10 kb; 10 kb[, [10 kb; 100 kb[, [100 kb; 500 kb[, [500 kb; 1 Mb]). For the top <em>trans-</em>SNPs, the background set was randomly selected from the full 12.9 million variants set, while maintaining the MAF proportion (classes mentioned above) observed for the top <em>trans-</em>meQTL SNPs set.</p></section><section id="Sec13"><h3 class="pmc_sec_title">
<em>Trans-</em>meQTL hotspots</h3>
<p id="Par21">We further investigated <em>trans-</em>meQTLs and their associated CpGs (<em>trans</em>-CpGs) by identifying <em>trans-</em>meQTL hotspots. For each lead SNP, defined as the SNP associated with the highest number of associated <em>trans-</em>CpGs (more than 30 as suggested by Huan et <em>al.</em> [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]<em>)</em>, we calculated LD with all the adjacent SNPs within a ± 2 Mb window around the lead SNP. SNPs with a high LD (r<sup>2</sup> &gt; 0.8) with the lead variant were retained, and these regions were defined as <em>trans-</em>meQTL hotspots. Next, we manually curated genes within or close to the previously defined hotspot intervals, focusing on genes involved in epigenetic processes. To facilitate this, we used an external database (Epifactors database [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]), which contains 802 genes involved in epigenetic processes, and checked for overlaps with the <em>trans-</em>meQTL hotspots.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Enrichment of <em>trans-</em>meQTL hotspots CpG in transcription factor binding sites</h3>
<p id="Par22">The enrichment in transcription factor binding sites (TFBS) was examined for the CpGs of each <em>trans-</em>meQTL hotspot. The background set consisted of all the CpGs analyzed. 841 motifs corresponding to the JASPAR 2022 CORE vertebrates non-redundant release [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>] were tested. We considered a 100-bp CpG window [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>], centered on the CpG, and for CpGs that were less than 100 bp apart, the window was set from 50 bp before the first CpG to 50 bp after the last CpG. Following the merging of closely located CpGs, only the <em>trans-</em>meQTL hotspots with more than 20 sequences were retained. Enrichment were computed using SEA [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>] and were considered significant when the false discovery rate (FDR) adjusted <em>P</em>-value (i.e., q-value) was below 0.05.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Effect of sperm <em>trans-</em>meQTL hotspots in PBMCs</h3>
<p id="Par23">To investigate the effects of the <em>trans-</em>meQTL hotspots identified in sperm on PBMC DNAm, we examined the CpGs targeted by lead SNPs of the <em>trans-</em>meQTL hotspots in PBMCs. This analysis was conducted in a cohort of 54 bulls, which were also part of the sperm meQTL mapping design. For each <em>trans-</em>meQTL hotspot, we plotted the DNAm level of the associated <em>trans-</em>CpGs in PBMCs, classified by genotypes. If the observed DNAm pattern in PBMCs did not align with those observed in sperm, we concluded that the <em>trans-</em>meQTL hotspot was not shared between sperm and PBMCs for the investigated <em>trans</em>-CpGs.</p></section></section><section id="Sec16"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec17"><h3 class="pmc_sec_title">Selection of CpGs with variable DNA methylation</h3>
<p id="Par24">From the sperm samples of 405 marketed Holstein bulls, we identified and selected 166,985 CpGs non-overlapping with a known DNA polymorphism and with a DNAm s.d. above 5%. These selected CpGs represented 13.7% of the total number of CpGs covered by RRBS (<em>N</em> = 1,212,361) and excluded most CpGs with extreme DNA methylation values (Fig. <a href="#Fig1" class="usa-link">1</a>a). The distribution of retained CpGs varied notably across autosomes, with counts ranging from 1,493 CpGs on BTA28 to 12,600 CpGs on BTA18 (Fig. <a href="#Fig1" class="usa-link">1</a>b). These CpGs with variable DNAm were subsequently analyzed for genetic association using GWAS on imputed whole-genome sequences for Holstein bulls. The numbers and proportions of CpGs reported in the following results are therefore relative to the 166,985 CpGs with variable DNAm.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374274_12864_2025_11934_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/692b28eaca7d/12864_2025_11934_Fig1_HTML.jpg" loading="lazy" id="MO1" height="683" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>DNA methylation measured by RRBS. (<strong>a</strong>) Scatter plot of CpG methylation levels, showing the mean methylation ratio (x-axis) vs. the standard deviation (s.d.) (y-axis) for all analyzed CpGs (<em>N</em> = 1,212,361). Black dots, above the red line (s.d. &gt;0.05), were retained for the study (<em>N</em> = 166,985). The upper and right-sided histograms represent the overall distribution of methylation levels and standard deviations, with retained CpGs highlighted in dark grey. (<strong>b</strong>) Distribution of the retained CpGs (<em>N</em> = 166,985) per Mb across the genome, with alternating colors distinguishing chromosomes</p></figcaption></figure></section><section id="Sec18"><h3 class="pmc_sec_title">Heritability of DNA methylation in sperm</h3>
<p id="Par26">Heritability of DNAm (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/e8c2d1182da1/d33e987.gif" loading="lazy" id="d33e987" alt="Inline graphic"></span>) was estimated for the 166,985 CpGs, and the distribution of heritability estimates is shown in Fig. <a href="#Fig2" class="usa-link">2</a> (see Supplementary Table 4 for individual values). The average <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/e8c2d1182da1/d33e996.gif" loading="lazy" id="d33e996" alt="Inline graphic"></span> across all CpGs was 0.26. Notably, 90% of CpGs had non-zero heritability estimate, and 76% had <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/e8c2d1182da1/d33e1002.gif" loading="lazy" id="d33e1002" alt="Inline graphic"></span> estimates &gt; 0.1. Compared to the background set of 103,719 independent analyzed CpGs (bearing non redundant methylation information), independent CpGs with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/e8c2d1182da1/d33e1008.gif" loading="lazy" id="d33e1008" alt="Inline graphic"></span> estimates &gt; 0.1 were significantly enriched in intergenic regions (odds ratio (OR) = 1.21, <em>P</em> = 2.7 × 10<sup>−33</sup>) and in regions of low CpG density (OR = 1.36, <em>P</em> = 6.3 × 10<sup>−107</sup>), denoted “open-sea” regions, in contrast to CpG islands (CGI). Conversely, these CpGs were depleted in the 0–200 bp region upstream of transcription start sites (TSS200), gene bodies, CGI, CGI shores, and CGI shelves. Odds ratios and corresponding <em>P</em>-values are summarized in Table <a href="#Tab1" class="usa-link">1</a>.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374274_12864_2025_11934_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/d1a0f224e2e5/12864_2025_11934_Fig2_HTML.jpg" loading="lazy" id="MO2" height="326" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of heritability estimates (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/e8c2d1182da1/d33e1039.gif" loading="lazy" id="d33e1039" alt="Inline graphic"></span>) for 166,985 CpGs with variable DNA methylation. Dashed lines represent the first quartile, the median and the third quartile</p></figcaption></figure><section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Characterization of the CpGs with variable DNA methylation (<em>N</em> = 166,985) by genomic region: counts, mean heritability (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/94faa179290a/d33e1059.gif" loading="lazy" id="d33e1059" alt="Inline graphic"></span>, and enrichment analysis results for the independent CpGs with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/e8c2d1182da1/d33e1065.gif" loading="lazy" id="d33e1065" alt="Inline graphic"></span>&gt;0.1 compared to the 103,719 independent CpGs<sup>a</sup></p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Genomic region</th>
<th align="left" rowspan="2" colspan="1"></th>
<th align="left" rowspan="2" colspan="1">Number of variable CpGs</th>
<th align="left" rowspan="2" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/e8c2d1182da1/d33e1089.gif" loading="lazy" id="d33e1089" alt="Inline graphic"></span> mean ±s.d.</th>
<th align="left" rowspan="2" colspan="1">Number of independent CpGs<sup>a</sup>
</th>
<th align="left" colspan="2" rowspan="1">Enrichment for<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/94faa179290a/d33e1102.gif" loading="lazy" id="d33e1102" alt="Inline graphic"></span> &gt;0.1</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Odds Ratio</th>
<th align="left" colspan="1" rowspan="1">
<em>P</em>-value</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="7" colspan="1">Genomic features</td>
<td align="left" colspan="1" rowspan="1">TSS 200<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">2726 (1.6%)</td>
<td align="left" colspan="1" rowspan="1">0.24 ± 0.19</td>
<td align="left" colspan="1" rowspan="1">1830 (1.8%)</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">1.3 × 10<sup>−3</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TSS 1500<sup>c</sup>
</td>
<td align="left" colspan="1" rowspan="1">10,071 (6%)</td>
<td align="left" colspan="1" rowspan="1">0.25 ± 0.20</td>
<td align="left" colspan="1" rowspan="1">6844 (6.6%)</td>
<td align="left" colspan="1" rowspan="1">0.99</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5’UTR</td>
<td align="left" colspan="1" rowspan="1">2215 (1.3%)</td>
<td align="left" colspan="1" rowspan="1">0.26 ± 0.20</td>
<td align="left" colspan="1" rowspan="1">1327 (1.3%)</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">8.8 × 10<sup>−3</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1 st exon</td>
<td align="left" colspan="1" rowspan="1">5842 (3.5%)</td>
<td align="left" colspan="1" rowspan="1">0.27 ± 0.21</td>
<td align="left" colspan="1" rowspan="1">3330 (3.2%)</td>
<td align="left" colspan="1" rowspan="1">0.87</td>
<td align="left" colspan="1" rowspan="1">7.6 × 10<sup>−4</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Gene body</td>
<td align="left" colspan="1" rowspan="1">97,553 (58.4%)</td>
<td align="left" colspan="1" rowspan="1">0.26 ± 0.20</td>
<td align="left" colspan="1" rowspan="1">60,190 (58%)</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">5.1 × 10<sup>−12</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3’UTR</td>
<td align="left" colspan="1" rowspan="1">6019 (3.6%)</td>
<td align="left" colspan="1" rowspan="1">0.24 ± 0.18</td>
<td align="left" colspan="1" rowspan="1">3972 (3.8%)</td>
<td align="left" colspan="1" rowspan="1">0.99</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Intergenic</td>
<td align="left" colspan="1" rowspan="1">42,559 (25.5%)</td>
<td align="left" colspan="1" rowspan="1">0.28 ± 0.20</td>
<td align="left" colspan="1" rowspan="1">26,226 (25.3%)</td>
<td align="left" colspan="1" rowspan="1">1.21</td>
<td align="left" colspan="1" rowspan="1">2.7 × 10<sup>−33</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1">CpG landscape<sup>d</sup>
</td>
<td align="left" colspan="1" rowspan="1">CGI</td>
<td align="left" colspan="1" rowspan="1">60,466 (36.2%)</td>
<td align="left" colspan="1" rowspan="1">0.26 ± 0.20</td>
<td align="left" colspan="1" rowspan="1">33,756 (32.5%)</td>
<td align="left" colspan="1" rowspan="1">0.8</td>
<td align="left" colspan="1" rowspan="1">4.1 × 10<sup>−54</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CGI shores</td>
<td align="left" colspan="1" rowspan="1">41,433 (24.8%)</td>
<td align="left" colspan="1" rowspan="1">0.25 ± 0.19</td>
<td align="left" colspan="1" rowspan="1">28,232 (27.2%)</td>
<td align="left" colspan="1" rowspan="1">0.94</td>
<td align="left" colspan="1" rowspan="1">2.1 × 10<sup>−4</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CGI shelves</td>
<td align="left" colspan="1" rowspan="1">9854 (5.9%)</td>
<td align="left" colspan="1" rowspan="1">0.24 ± 0.20</td>
<td align="left" colspan="1" rowspan="1">6571 (6.3%)</td>
<td align="left" colspan="1" rowspan="1">0.87</td>
<td align="left" colspan="1" rowspan="1">2.2 × 10<sup>−6</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">open sea</td>
<td align="left" colspan="1" rowspan="1">55,232 (33.1%)</td>
<td align="left" colspan="1" rowspan="1">0.28 ± 0.21</td>
<td align="left" colspan="1" rowspan="1">35,150 (33.9%)</td>
<td align="left" colspan="1" rowspan="1">1.36</td>
<td align="left" colspan="1" rowspan="1">6.3 × 10<sup>−107</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p33">
<p><sup>a</sup>Adjacents CpGs with DNAm R &gt;0.5 were grouped</p>
<p><sup>b</sup>TSS 200: 0–200 bases upstream to transcription start sites (TSS)</p>
<p><sup>c</sup>TSS 1500: 0–1500 bases upstream to TSS</p>
<p><sup>d</sup>CGI: CpG island; CGI shores: 0-2kb from CGI; CGI shelves: 2-4kb regions from CGI; open-sea: &gt;4kb regions from CGI</p>
</div></div></section></section><section id="Sec19"><h3 class="pmc_sec_title">Methylation QTL mapping</h3>
<p id="Par29">GWAS were performed for all 166,985 CpGs to identify both <em>cis-</em> (within 1 Mb upstream or downstream of the associated CpG) and <em>trans-</em>meQTLs (intrachromosomal &gt; 2 Mb away or interchromosomal from the associated CpG). Intrachromosomal associations between 1 and 2 Mb were considered potential long-range <em>cis</em> associations and were excluded. At a significance threshold of <em>P</em> &lt; 1.38 × 10<sup>−4</sup> (Supplementary Fig. 1), a total of 57,450,466 significant <em>cis-</em>associations were detected, representing <em>cis-</em>meQTLs for 54,934 CpGs (32.9% of the analysed CpGs) (Fig. <a href="#Fig3" class="usa-link">3</a>a). On average, a <em>cis-</em>CpG was located 260.8 kb (absolute mean; s.d. = 299.4 kb; median = 114.2 kb) away from its top <em>cis-</em>SNP (defined as the associated SNP with the lowest <em>p</em>-value). The median number of SNPs per <em>cis-</em>CpG was 384 (mean = 1,046, s.d. = 1,434). Figure <a href="#Fig3" class="usa-link">3</a>b shows the distribution of distances between CpGs and their top <em>cis-</em>SNPs. The top <em>cis-</em>SNPs and associated <em>P</em>-values are reported in Supplementary Table 4.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374274_12864_2025_11934_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/701854b9d899/12864_2025_11934_Fig3_HTML.jpg" loading="lazy" id="MO3" height="786" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mapping of <em>cis-</em> and <em>trans-</em>methylation QTLs. (<strong>a</strong>) Distribution of meQTLs types for the 166,985 CpGs. (<strong>b</strong>) Distribution of the distance between the CpG and the top <em>cis-</em>SNP. (<strong>c</strong>) Significant <em>trans</em> associations between SNPs located on BTA9 and CpGs located on BTA3. “<em>N</em>” indicates the number of associated <em>trans-</em>CpG per SNP. Purple line = BTA9:64,209,331 − 64,210,048 bp. Blue line = BTA3:117,543,159 − 117,544,310 bp (containing up to 19 <em>trans-</em>CpGs). (<strong>d</strong>) Pangenome bubble within the 35 to 70 Mb region of BTA9 extracted from a pangenome graph constructed with ARS-UCD1.2 and 8 Holstein genome assemblies. Purple segment = BTA9:64,209,331 − 64,210,048 bp. Pink segment = BTA9:64,209,243 − 64,209,316 bp. Blue segment = region with a 99.74% identity with BTA3:117,543,159 − 117,544,310 bp. The reported genomic coordinates corresponded to positions within the ARS-UCD1.2 reference genome. (<strong>e</strong>) Number of <em>trans-</em>CpGs (y-axis) associated to each SNP (chromosome coordinates on x-axis). Red line = 30 CpGs</p></figcaption></figure><p id="Par30">At a threshold of <em>P</em> &lt; 1.25 × 10<sup>−8</sup> (Supplementary Fig. 1), we identified 17,593,537 <em>trans-</em>CpG-SNP pairs involving 23,327 CpGs. These associations included intrachromosomal pairs (&gt; 2 Mb apart) and interchromosomal pairs. A large proportion of <em>trans</em> associations (79.3%) involved intrachromosomal CpG-SNP pairs with a genomic distance within 2 Mb to 20 Mb. Among these 2–20 Mb intrachromosomal associations, 99.4% of the associated CpGs also had a <em>cis-</em>meQTL. Since these associations are likely to result from long-range LD, they were excluded from further <em>trans-</em>meQTL analyses. Furthermore, to address potential mapping inaccuracies and structural polymorphisms, we aligned several sequences defined by groups of CpGs or SNPs to a Holstein-Friesian genome assembly (accession number = PRJNA668863) different from the Hereford reference genome (ARS-UCD1.2). This verification step was conducted on genomic regions for which SNPs were associated with at least 10 distant CpGs within the same genomic region. This revealed 11 regions (ranging from 54 to 196,313 bp) that aligned near their respective <em>trans-</em>SNPs or <em>trans-</em>CpGs, with &gt; 91% query coverage and &gt; 98.48% of identity (Supplementary Table 5). These associations were discarded as they were likely false positive <em>trans-</em>associations. For one region, showing a strong association between SNPs located on BTA9 and a cluster of 19 CpGs spanning over 1,152 bp on BTA3 (Fig. <a href="#Fig3" class="usa-link">3</a>c), the alignment with the Holstein-Friesian genome assembly did not reveal mapping errors. However, the presence of such an exclusive long-range association between these CpGs and distant SNPs is biologically intriguing. To investigate further this region, we constructed a reduced pangenome graph using the reference genome and eight Holstein <em>de novo</em> genome assemblies (Fig. <a href="#Fig3" class="usa-link">3</a>d). We identified a 4,791 bp structural variant (SV) on BTA9 corresponding to an insertion (carried by one Holstein individual) relative to the reference genome located between 64,209,316 and 64,209,331 bp (ARS-UCD1.2 coordinates). This SV contained, with 99.74% identity, the 1,152 bp segment initially located on BTA3 on the reference genome. Therefore, associations involving this structural variant were excluded from the <em>trans-</em>meQTL analyses. After applying all filters, 2,422,352 <em>trans-</em>CpG-SNP pairs, corresponding to 7,666 CpGs (4.6% of all CpGs) remained (top <em>trans-</em>SNPs are reported in Supplementary Table 4). Of these, 440 CpGs had intrachromosomal <em>trans-</em>meQTL, and 1,680 CpGs (1.0%) had both <em>cis-</em> and <em>trans-</em>meQTLs (Fig. <a href="#Fig3" class="usa-link">3</a>a). In total, 680,122 SNPs were associated with at least one <em>trans-</em>CpG, with 1.2% (8,082 SNPs) linked to 30 or more <em>trans-</em>CpGs, as shown in Fig. <a href="#Fig3" class="usa-link">3</a>e.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Genomic enrichments of meQTLs and their CpGs</h3>
<p id="Par31">We performed enrichment analyses distinguishing independent <em>cis-</em>CpGs, <em>trans-</em>CpGs and their corresponding top <em>cis-</em>SNPs and top <em>trans-</em>SNPs (based on the lowest <em>P</em>-value). OR and 95% confidence interval are shown in Fig. <a href="#Fig4" class="usa-link">4</a> and reported in Supplementary Table 6. Overall, <em>cis-</em> and <em>trans-</em>CpGs showed moderate enrichment patterns toward functional elements (0.67 &lt; OR &lt; 1.57). In contrast, the corresponding top SNPs presented stronger over- and under-representation patterns. With regards to the CpG Island (CGI) landscape, both CGI (OR<sub>cis</sub> = 1.59, OR<sub>trans</sub> = 2.36) and CGI shores (OR<sub>cis</sub> = 1.08; OR<sub>trans</sub> = 1.96), were significantly enriched for top <em>cis-</em> and <em>trans-</em>meQTL SNPs, while open-sea regions were depleted (OR<sub>cis</sub> = 0.89; OR<sub>trans</sub> = 0.46). Both top <em>cis-</em> and <em>trans-</em>meQTL SNPs were overrepresented in TSS200, TSS1500, and 3’UTR regions (OR &gt; 1) and the <em>trans-</em>meQTL SNPs were under-represented in intergenic regions (OR &lt; 1). Furthermore, the meQTL SNPs were preferentially located in regions depleted of transposable elements (“no_TE”: OR &gt; 1), while their associated CpGs showed enrichment for TEs (depletion in “no_TE”: OR<sub>cis</sub> = 0.78; OR<sub>trans</sub> = 0.81), in particular Long Terminal Repeats (LTR: OR<sub>cis</sub> = 1.25; OR<sub>trans</sub> = 1.40) and Short Interspersed Repetitive Elements (SINE: OR<sub>cis</sub> = 1.33; OR<sub>trans</sub> = 1.30) SINE. When analyzing TSS surrounding regions annotations (TSS ± 200 and TSS ± 1500) derived from TSS recently identified by Cap Analysis Gene Expression sequencing (CAGE) [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>], we found consistent meQTL SNPs enrichment for both testis-specific TSS (“CAGE_testis”) and ubiquitous TSS (“CAGE”). For the TSS ± 200 regions, this enrichment was stronger compared to Ensembl derived TSS ± 200 annotation [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] (TSS ± 200 in testis: OR<sub>cis</sub> = 1.75; OR<sub>trans</sub> = 1.79; ubiquitous TSS ± 200: OR<sub>cis</sub> = 1.60; OR<sub>trans</sub> = 2.23). Lastly, the top meQTL SNPs were enriched in combined testis-specific and ubiquitous ATAC-seq peaks (regions of open chromatin) (OR<sub>cis</sub> = 1.42; OR<sub>trans</sub> = 2.56), in line with the observed enrichment in TSS regions. This enrichment was particularly marked for <em>trans-</em>meQTLs.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374274_12864_2025_11934_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/ca390dfb3094/12864_2025_11934_Fig4_HTML.jpg" loading="lazy" id="MO4" height="576" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Enrichment in genomic annotations of independent <em>cis-</em>CpGs, <em>trans-</em>CpGs and their corresponding top <em>cis-</em> and <em>trans-</em>meQTL SNPs (selected by lowest <em>P</em>-value) for common annotation databases (“cpgIslandExt” Table [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] from “CGI” to “open_sea”; Ensembl release 112 [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] from “TSS200” to “INTERGENIC”; and RepeatMasker [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] from “DNA” to “SINE”) or novel catalogs (CAGE annotation track [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] from “TSS+−200_CAGE_testis” to “INTERGENIC_CAGE” and ATAC-seq bovine catalog [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>] from “ATAC_seq_peaks”, from which only “ubiquitous” or “testis” peaks were retained). The error bars represent the 95% confidence interval. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001</p></figcaption></figure></section><section id="Sec21"><h3 class="pmc_sec_title">Investigation of<em> trans-</em>meQTL hotspots</h3>
<p id="Par32"><em>Trans-</em>meQTL mapping revealed the presence of several genomic regions associated with multiple <em>trans-</em>CpGs. We focused on <em>trans-</em>meQTL hotspots, defined as regions associated with more than 30 <em>trans-</em>CpGs (Table <a href="#Tab2" class="usa-link">2</a>). A total of eight <em>trans-</em>meQTL hotspots (H1 to H8) were identified on seven autosomes, collectively targeting 1,058 <em>trans-</em>CpGs, representing 481 independent <em>trans-</em>CpGs (considered by a DNAm <em>r</em><sup>2</sup> &lt; 0.25 between two adjacent <em>trans-</em>CpGs). For each <em>trans</em>-meQTL hotspot, genomic intervals were defined using all variants in high LD (i.e., <em>r</em><sup>2</sup> &gt; 0.8) with the lead variant (associated with the highest number of <em>trans-</em>CpGs) located within ± 2 Mb. For each interval, we manually curated candidate genes in the hotspot regions, retaining only genes known to be involved in epigenetic regulation.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Description of eight <em>trans-</em>meQTL hotspots</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">hotspot</th>
<th align="left" rowspan="2" colspan="1">BTA</th>
<th align="left" rowspan="2" colspan="1">Interval (in Mb)</th>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="4" rowspan="1">Most significant SNP</th>
<th align="left" rowspan="2" colspan="1">Candidate genes</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Position</th>
<th align="left" colspan="2" rowspan="1">#<em> trans</em>-CpGs</th>
<th align="left" colspan="1" rowspan="1">
<em>trans</em>-CpGs with opposite effects to the hotspot majority (%)</th>
<th align="left" colspan="1" rowspan="1"># independent CpGs<sup>a</sup>
</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">H1</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">156.8-157.6</td>
<td align="left" colspan="1" rowspan="1">157,443,428</td>
<td align="left" colspan="2" rowspan="1">92</td>
<td align="left" colspan="1" rowspan="1">10.9</td>
<td align="left" colspan="1" rowspan="1">76</td>
<td align="left" colspan="1" rowspan="1">
<em>KAT2B</em><sup><em>b</em></sup>, <em>PRDM9</em><sup><em>b</em></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H2</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">120.5–123.0</td>
<td align="left" colspan="1" rowspan="1">122,297,963</td>
<td align="left" colspan="2" rowspan="1">553</td>
<td align="left" colspan="1" rowspan="1">9.2</td>
<td align="left" colspan="1" rowspan="1">214</td>
<td align="left" colspan="1" rowspan="1">
<em>RBBP4</em><sup><em>b</em></sup>, <em>HDAC1</em><sup><em>b</em></sup>, <em>SPOCD1</em>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3</td>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="left" colspan="1" rowspan="1">12.7–13.9</td>
<td align="left" colspan="1" rowspan="1">13,370,307</td>
<td align="left" colspan="2" rowspan="1">31</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">
<em>BANP</em><sup><em>b</em></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H4</td>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="left" colspan="1" rowspan="1">63.8–63.9</td>
<td align="left" colspan="1" rowspan="1">63,854,493</td>
<td align="left" colspan="2" rowspan="1">84</td>
<td align="left" colspan="1" rowspan="1">8.3</td>
<td align="left" colspan="1" rowspan="1">31</td>
<td align="left" colspan="1" rowspan="1">
<em>PRPF31</em><sup><em>bc</em></sup>, <em>TFPT</em><sup><em>bc</em></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H5</td>
<td align="left" colspan="1" rowspan="1">21</td>
<td align="left" colspan="1" rowspan="1">60.3–60.5</td>
<td align="left" colspan="1" rowspan="1">60,522,514</td>
<td align="left" colspan="2" rowspan="1">177</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">91</td>
<td align="left" colspan="1" rowspan="1">
<em>TCL1B</em>,<em> TCL1A</em><sup><em>bc</em></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H6</td>
<td align="left" colspan="1" rowspan="1">25</td>
<td align="left" colspan="1" rowspan="1">21.9–22.3</td>
<td align="left" colspan="1" rowspan="1">22,201,331</td>
<td align="left" colspan="2" rowspan="1">35</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">
<em>PRKCB</em><sup><em>bc</em></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H7</td>
<td align="left" colspan="1" rowspan="1">28</td>
<td align="left" colspan="1" rowspan="1">4.6–6.5</td>
<td align="left" colspan="1" rowspan="1">5,294,954</td>
<td align="left" colspan="2" rowspan="1">51</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">24</td>
<td align="left" colspan="1" rowspan="1"><em>-</em></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H8</td>
<td align="left" colspan="1" rowspan="1">29</td>
<td align="left" colspan="1" rowspan="1">48.7–48.8</td>
<td align="left" colspan="1" rowspan="1">48,763,272</td>
<td align="left" colspan="2" rowspan="1">35</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">
<em>NAP1L4</em><sup><em>bc</em></sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p44">
<p><sup>a</sup>Adjacents CpGs with DNAm R &gt;0.5 were grouped</p>
<p><sup>b</sup>In Epifactor database [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]</p>
<p><sup>c</sup>Outside the hotspot interval (distance from the interval: <em>PRPF31</em> = 560 kb, <em>TFPT</em> = 551 kb, <em>TCL1A</em> = 0.6 kb, <em>PRKCB</em> = 68 kb, <em>NAP1L4</em> = 59 kb)</p>
</div></div></section><p id="Par33">The first <em>trans-</em>meQTL hotspot (H1) overlapped with several genes, including <em>KAT2B</em> (a histone acetyltransferase gene, HAT) and <em>PRDM9</em> (a histone methyltransferase, HMT), which catalyzes H3K4 trimethylation and has been reported to bind to recombination hotspots [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. The lead variant of H2 (BTA2:122297963 C &gt; T) was associated with 553 CpGs located across the 29 autosomes, with the T &gt; C allele effect increasing DNAm by on average of 5.5% (s.d. = 4.6%), positive for 91% of the 553 CpGs. The interval of the second hotspot (~ 2.5 Mb) contained genes <em>HDAC1</em>, encoding a histone deacetylase enzyme (HDAC), a histone chaperone (<em>RBBP4</em>), and <em>SPOCD1</em>, a gene involved in piRNA directed <em>de novo</em> methylation of TE in the germline [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>], which interacts with <em>DNMT3A</em> (DNA (cytosine-5)-methyltransferase 3 A) encoding an enzyme that catalyzes the <em>de novo</em> transfer of methyl groups to DNA. Of the two <em>trans-</em>meQTL hotspots contained in BTA18, the first one (H3) included <em>BANP</em>, a protein-coding gene that binds to matrix attachment regions [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] involved in chromatin structure and activates CGI [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. <em>BANP</em> interacts with histone modifying enzymes, including <em>HDAC6</em> [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] and <em>HDAC1</em> [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. No known gene overlaps the 22.7-kb interval of H4 on BTA18. At a distance of ± 0.6 Mb, two genes, <em>PRPF31</em> and <em>TFPT</em>, were found to be both components of complexes involved in epigenetic processes [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. H5 overlapped with 4 genes, including <em>TCL1B</em>, the closest gene to the lead SNP. While <em>TCL1B</em> does not interact directly with <em>DNMT3A</em>, <em>TCL1A</em>, a highly similar gene located 26 kb from the lead variant, physically interacts with <em>DNMT3A</em> and <em>DNMT3B</em>, with <em>TCL1A</em> overexpression inhibiting <em>DNMT3A</em> [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>] activity. The H6 lead variant was located 320 kb away from <em>PRKCB</em>, a histone kinase gene. Although several genes overlapped with the H7 interval, we did not identify any epigenetic regulatory genes. <em>NAP1L4</em>, a histone chaperone gene, is located close to the H8 hotspot. Of the eight hotspots, five (H3, H5, H6, H7, and H8) had unidirectional estimated effects on their associated <em>trans-</em>CpGs. H1 showed the highest frequency (11%) of <em>trans-</em>CpGs with effects opposite to the majority. In comparison with the full set of analyzed CpGs (<em>N</em> = 166,985), no significant enrichment in transcription factor binding sites (TFBSs) was observed for CpGs associated with any <em>trans-</em>meQTL hotspots (5% FDR adjusted <em>P</em>-value &gt; 0.05).</p>
<p id="Par34">We then investigated whether the lead variants from the <em>trans</em>-meQTL hotspots affecting DNAm at more than 30 <em>trans-</em>CpGs in sperm also affected DNAm at the same <em>trans-</em>CpGs in peripheral blood mononuclear cells (PBMCs). PBMC DNAm data, also measured with RRBS, was available for 54 of the 405 bulls. Hotspots H7 and H8 were not considered in this analysis due to the low number of <em>trans-</em>CpGs shared between sperm and PBMCs (5 CpGs, 9.6%; and 7 CpGs, 20%, respectively). The remaining hotspots shared DNAm data for at least 78.2% of their <em>trans-</em>CpGs between the two tissues. To assess genetic influence, we plotted the DNAm levels of the <em>trans-</em>CpGs across sperm and PBMCs for each hotspot, stratified by genotypes of the lead variant. The results showed no evidence of a genetic influence of the lead variants on DNAm levels of the shared <em>trans-</em>CpGs in PBMCs (Supplementary Fig. 2).</p></section></section><section id="Sec22"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par35">We performed meQTL mapping analyses for 166,985 CpGs with variable DNAm and 12,930,389 variants to investigate the genetic architecture of sperm DNAm in bulls. Paving the way for further studies aimed at integrating epigenetic information into the prediction of performance traits (e.g., milk yield, fertility), we reported that, on average, genetic variance accounted for 26% of the DNAm variability, with heterogeneity observed across different genomic features. We identified <em>cis-</em>meQTLs for approximately one-third of the CpGs analyzed and <em>trans-</em>meQTLs for 4.6% of CpGs. These proportions are consistent with previous studies [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>, <a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>], despite significant differences in experimental design, including: (1) the tissue analyzed (sperm vs. blood), (2) the species studied (cattle vs. human), (3) the technology used for methylation measurement (RRBS vs. DNAm array), and (4) the germline-specific epigenetic reprogramming. Additionally, we found eight <em>trans-</em>meQTL hotspots, of which four overlapped with genes involved in epigenetic processes. Interestingly, the genetic effects on associated CpGs in PBMCs did not mirror the patterns observed in sperm. Further research is needed to determine whether this observation reflects a sperm-specific mechanism involved in the genetic regulation of DNAm.</p>
<p id="Par36">Highly variable CpGs have been reported to be more heritable than less variable ones [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>], which is an important consideration in our study, as we selected CpGs based on their individual variability. This observation may be attributed to the technical variability, which is proportionally larger when total variability is low. Besides, the majority of the CpGs have non-variable DNAm and we can hypothesize that they are under strong genetic control. Despite the selection criterion based on variability, our heritability estimates were similar to those derived from DNAm array data in human blood, even though they were estimated in a different tissue [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>], corroborating our findings. Heritable CpGs (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4897/12374274/e8c2d1182da1/d33e2125.gif" loading="lazy" id="d33e2125" alt="Inline graphic"></span> &gt;0.1) are depleted in TSS200 regions and in high-CpG dense regions, as also observed in human blood data [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>].</p>
<p id="Par37">Without considering transposable elements, the general enrichment of meQTLs in specific genomic annotations (e.g., CGI, TSS surrounding regions) and the depletion of open-sea and intergenic regions were similar to observations reported by Villicaña et al. [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. These findings remain consistent despite comparing results across different tissues and species. Min et al. [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>] underpinned that variants associated with meQTLs are enriched in chromatin states, although the enrichments depended on the type of association (active chromatin domains for <em>cis</em> and <em>cis-</em> + <em>trans-</em>meQTLs, heterochromatin for <em>trans-</em>meQTLs). Using recently published bovine ATAC-seq data [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>], we observed significant over-representation of <em>cis-</em> and <em>trans-</em>meQTLs in ATAC-seq peaks, which mark regions of open chromatin, corroborating that regulatory variant preferentially map to tissue-specific ATAC-seq peaks (as described in Yuan et al. [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]). Enrichment was also observed in TSS ± 200 regions derived from a recent and improved TSS annotation track based on CAGE-seq data [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>], with notably higher enrichment levels compared to TSS200 regions annotations from Ensembl v112 [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>].</p>
<p id="Par38">While the DNAm landscape is highly tissue-specific and dependent on the developmental stage [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>], several studies have identified shared meQTLs across tissues, such as blood, various brain regions [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>, <a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>], and saliva [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>], although these associations were examined over short genomic distances. For example, Lin et al. [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>] reported 45–73% overlap in <em>cis-</em>meQTL among tissues. To our knowledge, cross-tissue meQTLs comparisons have not yet included a direct blood-sperm comparison analysis. In the present study, we did not identify a shared genetic architecture for <em>trans-</em>meQTLs affecting multiple CpGs across both sperm and blood. However, because our analysis focused on genetic variants affecting multiple distant CpGs, broader conclusions regarding the genetic regulation of DNA methylation across tissues should be drawn with caution. Following these observations, despite most meQTL studies focusing on blood, identifying sperm meQTLs is essential for addressing key scientific questions, including understanding the transmission of paternally inherited DNA marks [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>].</p>
<p id="Par39">Previous research has suggested that changes in DNAm may result from altered chromatin conformation [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. Several studies investigating the genetic architecture of DNA methylation have identified histone modifier genes [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>, <a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>], highlighting the well-established interdependence between histones and DNA methylation [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. In this study, the reported candidate genes overlapping the <em>trans-</em>meQTL hotspots align with this hypothesis. Specifically, chromatin in male germ cells exhibits a unique structure, characterized by extreme compaction, with most histones substituted by protamines [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. Notably, two of the identified candidate genes from <em>trans-</em>meQTL hotspots investigations, <em>PRDM9</em> and <em>SPOCD1</em>, are particularly compelling due to their active roles during spermatogenesis. <em>PRMD9</em> acts as a hotspot regulator through a <em>trans-</em>acting mechanism [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>], aligning with our findings. <em>SPOCD1</em> is involved in piRNA-mediated DNA methylation targeting TE and its deficiency leads to male-specific infertility [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. This is aligns with a tendency for LINE L1 hypomethylation in subfertile bulls [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>], thus reinforcing the interest in understanding the underlying mechanisms.</p>
<p id="Par40">Exploration of <em>trans-</em>meQTL mapping revealed several spurious <em>trans-</em>associations between CpGs and variants, likely arising from inaccuracies in the reference genome for our population. These inaccuracies were confirmed by analyses of multiple genome assemblies. More broadly, the search for associations between genomic markers located in different genomic regions, i.e., SNPs and CpGs, underlined a general pattern: a high proportion of short-range associations but very few long-range ones. Strong and exclusive associations between distant genomic regions are rare and may result from assembly errors or structural variants. The construction of a cattle pangenome has facilitated the discovery of novel structural variants (SVs) molecular QTLs [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>] (molQTLs; i.e., to refer to molecular phenotypes such as gene expression data, protein abundance, and methylation). Here, we extended this concept, emphasizing that pangenomes can also refine molQTL mapping by highlighting SVs, which allow the detection and elimination of spurious distant associations. However, the pangenome approach is computationally demanding and requires a large and diverse set of genomes to increase the probability of observing rare events.</p>
<p id="Par41">The extensive catalog of sperm meQTLs generated in this study represents a valuable resource to improve our understanding of the biological mechanisms underlying complex traits. In cattle, DNA methylation has already been associated with traits of economic importance [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>, <a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. Specifically regarding field fertility, Costes et al. [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>] identified distinct methylation patterns between fertile and sub-fertile Montbéliard bulls. Building on these findings, integrating meQTL data with GWAS results and fertility-associated methylation signatures offers a promising avenue to uncover the molecular mechanisms involved in bull fertility. SNPs that influence DNA methylation in sperm may contribute to variation in gene regulation or TE activity during spermatogenesis or early embryonic development, potentially affecting fertility outcomes. By identifying loci where genetic variants modulate epigenetic marks, we can begin to disentangle causal pathways and prioritize candidate genes or regulatory elements for functional validation. Ultimately, this integrative approach could improve the accuracy of fertility-related trait prediction and inform breeding strategies.</p>
<p id="Par42">While our findings are promising, several limitations of our study should be acknowledged. First, the CpGs analyzed were selected from a population of CpGs generated by RRBS (using the MspI restriction enzyme), a technology that does not provide a uniform representation of CpGs across the genome [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>] (bias towards CpG-dense regions). In addition, the CpGs were selected solely on the basis of their individual variability. As a result, the analyzed CpGs were not entirely independent, as physically proximal CpGs may show similar DNAm patterns. The use of independent CpGs in enrichment analysis attempted to overcome this phenomenon. Third, the limited sample size may have increased the risk of false positives or overestimation of effect size [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. Although we controlled for false positive rate using permutations, the limited sample size reduced our ability to identify rare variants affecting DNA methylation. The use of widely available epigenotyping tools, such as DNAm chips, could enable larger-scale phenotyping efforts, although outcomes would depend on marker selection and the tissue studied.</p></section><section id="Sec23"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par43">The identification of molQTLs is the subject of intense research to gain a better understanding of the links between genes and phenotypes. To this end, we mapped both <em>cis-</em> and <em>trans-</em>meQTLs in bull semen. In particular, we showed that the identification of <em>trans-</em>meQTLs is sensitive to the reference genome assembly, which can lead to spurious associations. Our results also support the hypothesis that the influence of genetic variants on multiple CpGs may be mediated through changes in chromatin structure during germ cell differentiation, possibly involving histone modifiers genes. These findings further our understanding of the genetic architecture of bovine semen methylation, a relatively unexplored field.</p></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12374274/bin/12864_2025_11934_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (342.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12374274/bin/12864_2025_11934_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (13.2MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank the breeding companies that provided biological samples and access to bull genotypes. Imputations were carried out with sequences from the “1000 Bull Genomes” project.</p></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>CF, CH, SF, MB, DB, HK, and MPS conceived the study. CF did the analysis and wrote the original draft. VC, ARRS, CLD, ACT, FA, MCD, AB, ES, HJ and HK have carried out methylation data collection. VS built the partial pangenome. VC, CH, FB, GCMM, VS, MB, DB, HK, and MPS contributed to writing the manuscript. All authors reviewed and approved the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>CF is recipient of a CIFRE PhD grant from ANRT and APIS-GENE. VS and ARRS are recipients of a PhD grant from INRAE. This work was part of the RUMIGEN project funded by the European Union’s Horizon 2020 research and innovation program under grant agreement No. 101000226. This work was part of the POLYPHEME project funded by ANR (ANR-21-CE20-0021) and APIS-GENE.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>RRBS fastq files are available through the NCBI SRA under accession number PRJNA1225939 (<a href="http://www.ncbi.nlm.nih.gov/bioproject/1225939" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.ncbi.nlm.nih.gov/bioproject/1225939</a>). The fasta files of the eight 8 Holstein used to build the partial pangenome are available in the European Nucleotide Archive (ENA) at EMBL-EBI under accession number PRJEB68295 (<a href="https://www.ebi.ac.uk/ena/data/view/PRJEB68295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/ena/data/view/PRJEB68295</a>). Genotypes of the 405 bulls are commercial information belonging to the French breeders and are not publicly available. All other datasets supporting findings are included in the article and its supplementary tables and figures.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par44">Blood samples were collected for routine procedures by licensed veterinarians complying with the ethical guidelines of the French National Research Institute for Agriculture, Food and Environment (INRAE) with the informed consent of the breeders to whom the bulls belonged.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par45">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par46">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Corentin Fouéré, Email: corentin.fouere@inrae.fr.</p>
<p>Marie-Pierre Sanchez, Email: marie-pierre.sanchez@inrae.fr.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Singal R, Ginder GD. DNA Methylation. Blood. 1999;93:4059–70.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/10361102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singal%20R,%20Ginder%20GD.%20DNA%20Methylation.%20Blood.%201999;93:4059%E2%80%9370." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Moore LD, Le T, Fan G. DNA methylation and its basic function. Neuropsychopharmacol. 2013;38:23–38.</cite> [<a href="https://doi.org/10.1038/npp.2012.112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3521964/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22781841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moore%20LD,%20Le%20T,%20Fan%20G.%20DNA%20methylation%20and%20its%20basic%20function.%20Neuropsychopharmacol.%202013;38:23%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Perino M, Veenstra GJC. Chromatin control of developmental dynamics and plasticity. Dev Cell. 2016;38:610–20.
</cite> [<a href="https://doi.org/10.1016/j.devcel.2016.08.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27676434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perino%20M,%20Veenstra%20GJC.%20Chromatin%20control%20of%20developmental%20dynamics%20and%20plasticity.%20Dev%20Cell.%202016;38:610%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Leenen FAD, Muller CP, Turner JD. DNA methylation: conducting the orchestra from exposure to phenotype? Clin Epigenet. 2016;8: 92.</cite> [<a href="https://doi.org/10.1186/s13148-016-0256-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5012062/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27602172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leenen%20FAD,%20Muller%20CP,%20Turner%20JD.%20DNA%20methylation:%20conducting%20the%20orchestra%20from%20exposure%20to%20phenotype?%20Clin%20Epigenet.%202016;8:%2092." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Villicaña S, Bell JT. Genetic impacts on DNA methylation: research findings and future perspectives. Genome Biol. 2021;22: 127.
</cite> [<a href="https://doi.org/10.1186/s13059-021-02347-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8086086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33931130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Villica%C3%B1a%20S,%20Bell%20JT.%20Genetic%20impacts%20on%20DNA%20methylation:%20research%20findings%20and%20future%20perspectives.%20Genome%20Biol.%202021;22:%20127." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Villicaña S, Castillo-Fernandez J, Hannon E, Christiansen C, Tsai P-C, Maddock J, et al. Genetic impacts on DNA methylation help elucidate regulatory genomic processes. Genome Biol. 2023;24: 176.
</cite> [<a href="https://doi.org/10.1186/s13059-023-03011-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10391992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37525248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Villica%C3%B1a%20S,%20Castillo-Fernandez%20J,%20Hannon%20E,%20Christiansen%20C,%20Tsai%20P-C,%20Maddock%20J,%20et%20al.%20Genetic%20impacts%20on%20DNA%20methylation%20help%20elucidate%20regulatory%20genomic%20processes.%20Genome%20Biol.%202023;24:%20176." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>McRae AF, Marioni RE, Shah S, Yang J, Powell JE, Harris SE, et al. Identification of 55,000 replicated DNA methylation QTL. Sci Rep. 2018;8:17605.
</cite> [<a href="https://doi.org/10.1038/s41598-018-35871-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6279736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30514905/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McRae%20AF,%20Marioni%20RE,%20Shah%20S,%20Yang%20J,%20Powell%20JE,%20Harris%20SE,%20et%20al.%20Identification%20of%2055,000%20replicated%20DNA%20methylation%20QTL.%20Sci%20Rep.%202018;8:17605." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Min JL, Hemani G, Hannon E, Dekkers KF, Castillo-Fernandez J, Luijk R, et al. Genomic and phenotypic insights from an atlas of genetic effects on DNA methylation. Nat Genet. 2021;53:1311–21.
</cite> [<a href="https://doi.org/10.1038/s41588-021-00923-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7612069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34493871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Min%20JL,%20Hemani%20G,%20Hannon%20E,%20Dekkers%20KF,%20Castillo-Fernandez%20J,%20Luijk%20R,%20et%20al.%20Genomic%20and%20phenotypic%20insights%20from%20an%20atlas%20of%20genetic%20effects%20on%20DNA%20methylation.%20Nat%20Genet.%202021;53:1311%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Hawe JS, Wilson R, Schmid KT, Zhou L, Lakshmanan LN, Lehne BC, et al. Genetic variation influencing DNA methylation provides insights into molecular mechanisms regulating genomic function. Nat Genet. 2022;54:18–29.
</cite> [<a href="https://doi.org/10.1038/s41588-021-00969-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7617265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34980917/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hawe%20JS,%20Wilson%20R,%20Schmid%20KT,%20Zhou%20L,%20Lakshmanan%20LN,%20Lehne%20BC,%20et%20al.%20Genetic%20variation%20influencing%20DNA%20methylation%20provides%20insights%20into%20molecular%20mechanisms%20regulating%20genomic%20function.%20Nat%20Genet.%202022;54:18%E2%80%9329." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Huan T, Joehanes R, Song C, Peng F, Guo Y, Mendelson M, et al. Genome-wide identification of DNA methylation QTLs in whole blood highlights pathways for cardiovascular disease. Nat Commun. 2019;10: 4267.
</cite> [<a href="https://doi.org/10.1038/s41467-019-12228-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6753136/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31537805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huan%20T,%20Joehanes%20R,%20Song%20C,%20Peng%20F,%20Guo%20Y,%20Mendelson%20M,%20et%20al.%20Genome-wide%20identification%20of%20DNA%20methylation%20QTLs%20in%20whole%20blood%20highlights%20pathways%20for%20cardiovascular%20disease.%20Nat%20Commun.%202019;10:%204267." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Perzel Mandell KA, Eagles NJ, Wilton R, Price AJ, Semick SA, Collado-Torres L, et al. Genome-wide sequencing-based identification of methylation quantitative trait loci and their role in schizophrenia risk. Nat Commun. 2021;12: 5251.
</cite> [<a href="https://doi.org/10.1038/s41467-021-25517-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8413445/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34475392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perzel%20Mandell%20KA,%20Eagles%20NJ,%20Wilton%20R,%20Price%20AJ,%20Semick%20SA,%20Collado-Torres%20L,%20et%20al.%20Genome-wide%20sequencing-based%20identification%20of%20methylation%20quantitative%20trait%20loci%20and%20their%20role%20in%20schizophrenia%20risk.%20Nat%20Commun.%202021;12:%205251." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Zhang T, Choi J, Dilshat R, Einarsdóttir BÓ, Kovacs MA, Xu M, et al. Cell-type-specific MeQTLs extend melanoma GWAS annotation beyond eQTLs and inform melanocyte gene-regulatory mechanisms. Am J Hum Genet. 2021;108:1631–46.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2021.06.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8456160/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34293285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20T,%20Choi%20J,%20Dilshat%20R,%20Einarsd%C3%B3ttir%20B%C3%93,%20Kovacs%20MA,%20Xu%20M,%20et%20al.%20Cell-type-specific%20MeQTLs%20extend%20melanoma%20GWAS%20annotation%20beyond%20eQTLs%20and%20inform%20melanocyte%20gene-regulatory%20mechanisms.%20Am%20J%20Hum%20Genet.%202021;108:1631%E2%80%9346." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Hazard D, Plisson-Petit F, Moreno-Romieux C, Fabre S, Drouilhet L. Genetic determinism exists for the global DNA methylation rate in sheep. Front Genet. 2020;11: 616960.
</cite> [<a href="https://doi.org/10.3389/fgene.2020.616960" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7786236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33424937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hazard%20D,%20Plisson-Petit%20F,%20Moreno-Romieux%20C,%20Fabre%20S,%20Drouilhet%20L.%20Genetic%20determinism%20exists%20for%20the%20global%20DNA%20methylation%20rate%20in%20sheep.%20Front%20Genet.%202020;11:%20616960." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Shen Q, Wang K, Wang S, Zhao Z, Ji X, Chen D, et al. Identifying key DNA methylation sites and their cis-methylation quantitative loci for intramuscular fatty acid traits using genome and methylome data in Yorkshire pigs. Italian J Anim Sci. 2023;22:829–40.</cite> [<a href="https://scholar.google.com/scholar_lookup?Shen%20Q,%20Wang%20K,%20Wang%20S,%20Zhao%20Z,%20Ji%20X,%20Chen%20D,%20et%20al.%20Identifying%20key%20DNA%20methylation%20sites%20and%20their%20cis-methylation%20quantitative%20loci%20for%20intramuscular%20fatty%20acid%20traits%20using%20genome%20and%20methylome%20data%20in%20Yorkshire%20pigs.%20Italian%20J%20Anim%20Sci.%202023;22:829%E2%80%9340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Ribeiro AMF, Sanglard LP, Wijesena HR, Ciobanu DC, Horvath S, Spangler ML. DNA methylation profile in beef cattle is influenced by additive genetics and age. Sci Rep. 2022;12(1): 12016.
</cite> [<a href="https://doi.org/10.1038/s41598-022-16350-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9283455/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35835812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ribeiro%20AMF,%20Sanglard%20LP,%20Wijesena%20HR,%20Ciobanu%20DC,%20Horvath%20S,%20Spangler%20ML.%20DNA%20methylation%20profile%20in%20beef%20cattle%20is%20influenced%20by%20additive%20genetics%20and%20age.%20Sci%20Rep.%202022;12(1):%2012016." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Smallwood SA, Kelsey G. De novo DNA methylation: a germ cell perspective. Trends Genet. 2012;28:33–42.
</cite> [<a href="https://doi.org/10.1016/j.tig.2011.09.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22019337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smallwood%20SA,%20Kelsey%20G.%20De%20novo%20DNA%20methylation:%20a%20germ%20cell%20perspective.%20Trends%20Genet.%202012;28:33%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Zhou Y, Connor EE, Bickhart DM, Li C, Baldwin RL, Schroeder SG, et al. Comparative whole genome DNA methylation profiling of cattle sperm and somatic tissues reveals striking hypomethylated patterns in sperm. Gigascience. 2018;7:giy039.
</cite> [<a href="https://doi.org/10.1093/gigascience/giy039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5928411/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29635292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20Y,%20Connor%20EE,%20Bickhart%20DM,%20Li%20C,%20Baldwin%20RL,%20Schroeder%20SG,%20et%20al.%20Comparative%20whole%20genome%20DNA%20methylation%20profiling%20of%20cattle%20sperm%20and%20somatic%20tissues%20reveals%20striking%20hypomethylated%20patterns%20in%20sperm.%20Gigascience.%202018;7:giy039." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Perrier J-P, Sellem E, Prézelin A, Gasselin M, Jouneau L, Piumi F, et al. A multi-scale analysis of bull sperm methylome revealed both species peculiarities and conserved tissue-specific features. BMC Genomics. 2018;19: 404.
</cite> [<a href="https://doi.org/10.1186/s12864-018-4764-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5975405/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29843609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perrier%20J-P,%20Sellem%20E,%20Pr%C3%A9zelin%20A,%20Gasselin%20M,%20Jouneau%20L,%20Piumi%20F,%20et%20al.%20A%20multi-scale%20analysis%20of%20bull%20sperm%20methylome%20revealed%20both%20species%20peculiarities%20and%20conserved%20tissue-specific%20features.%20BMC%20Genomics.%202018;19:%20404." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Sanchez M-P, Guatteo R, Davergne A, Saout J, Grohs C, Deloche M-C, et al. Identification of the ABCC4, IER3, and CBFA2T2 candidate genes for resistance to paratuberculosis from sequence-based GWAS in Holstein and Normande dairy cattle. Genet Selection Evol. 2020;52:14.</cite> [<a href="https://doi.org/10.1186/s12711-020-00535-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7077142/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32183688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanchez%20M-P,%20Guatteo%20R,%20Davergne%20A,%20Saout%20J,%20Grohs%20C,%20Deloche%20M-C,%20et%20al.%20Identification%20of%20the%20ABCC4,%20IER3,%20and%20CBFA2T2%20candidate%20genes%20for%20resistance%20to%20paratuberculosis%20from%20sequence-based%20GWAS%20in%20Holstein%20and%20Normande%20dairy%20cattle.%20Genet%20Selection%20Evol.%202020;52:14." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Sargolzaei M, Chesnais JP, Schenkel FS. A new approach for efficient genotype imputation using information from relatives. BMC Genomics. 2014;15:478.
</cite> [<a href="https://doi.org/10.1186/1471-2164-15-478" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4076979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24935670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sargolzaei%20M,%20Chesnais%20JP,%20Schenkel%20FS.%20A%20new%20approach%20for%20efficient%20genotype%20imputation%20using%20information%20from%20relatives.%20BMC%20Genomics.%202014;15:478." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. Nat Genet. 2012;44:955–9.
</cite> [<a href="https://doi.org/10.1038/ng.2354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3696580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22820512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Howie%20B,%20Fuchsberger%20C,%20Stephens%20M,%20Marchini%20J,%20Abecasis%20GR.%20Fast%20and%20accurate%20genotype%20imputation%20in%20genome-wide%20association%20studies%20through%20pre-phasing.%20Nat%20Genet.%202012;44:955%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Sanchez M-P, Ramayo-Caldas Y, Wolf V, Laithier C, El Jabri M, Michenet A, et al. Sequence-based GWAS, network and pathway analyses reveal genes co-associated with milk cheese-making properties and milk composition in Montbéliarde cows. Genet Selection Evol. 2019;51:34.</cite> [<a href="https://doi.org/10.1186/s12711-019-0473-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6604208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31262251/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanchez%20M-P,%20Ramayo-Caldas%20Y,%20Wolf%20V,%20Laithier%20C,%20El%20Jabri%20M,%20Michenet%20A,%20et%20al.%20Sequence-based%20GWAS,%20network%20and%20pathway%20analyses%20reveal%20genes%20co-associated%20with%20milk%20cheese-making%20properties%20and%20milk%20composition%20in%20Montb%C3%A9liarde%20cows.%20Genet%20Selection%20Evol.%202019;51:34." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Sahana G, Cai Z, Sanchez MP, Bouwman AC, Boichard D. Invited review: good practices in genome-wide association studies to identify candidate sequence variants in dairy cattle. J Dairy Sci. 2023;106:5218–41.
</cite> [<a href="https://doi.org/10.3168/jds.2022-22694" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37349208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sahana%20G,%20Cai%20Z,%20Sanchez%20MP,%20Bouwman%20AC,%20Boichard%20D.%20Invited%20review:%20good%20practices%20in%20genome-wide%20association%20studies%20to%20identify%20candidate%20sequence%20variants%20in%20dairy%20cattle.%20J%20Dairy%20Sci.%202023;106:5218%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Costes V, Chaulot-Talmon A, Sellem E, Perrier J-P, Aubert-Frambourg A, Jouneau L, et al. Predicting male fertility from the sperm methylome: application to 120 bulls with hundreds of artificial insemination records. Clin Epigenetics. 2022;14:54.
</cite> [<a href="https://doi.org/10.1186/s13148-022-01275-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9047354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35477426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Costes%20V,%20Chaulot-Talmon%20A,%20Sellem%20E,%20Perrier%20J-P,%20Aubert-Frambourg%20A,%20Jouneau%20L,%20et%20al.%20Predicting%20male%20fertility%20from%20the%20sperm%20methylome:%20application%20to%20120%20bulls%20with%20hundreds%20of%20artificial%20insemination%20records.%20Clin%20Epigenetics.%202022;14:54." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Krueger F. Trim Galore! A wrapper around Cutadapt and FastQC to consistently apply adapter and quality trimming to FastQ files, with extra functionality for RRBS data. Babraham Institute. 2015. <a href="https://github.com/FelixKrueger/TrimGalore" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/FelixKrueger/TrimGalore</a>.</cite>
</li>
<li id="CR26">
<span class="label">26.</span><cite>Krueger F, Andrews SR. Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications. Bioinformatics. 2011;27:1571–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3102221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21493656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krueger%20F,%20Andrews%20SR.%20Bismark:%20a%20flexible%20aligner%20and%20methylation%20caller%20for%20Bisulfite-Seq%20applications.%20Bioinformatics.%202011;27:1571%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Daetwyler HD, Capitan A, Pausch H, Stothard P, Van Binsbergen R, Brøndum RF, et al. Whole-genome sequencing of 234 bulls facilitates mapping of monogenic and complex traits in cattle. Nat Genet. 2014;46:858–65.
</cite> [<a href="https://doi.org/10.1038/ng.3034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25017103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Daetwyler%20HD,%20Capitan%20A,%20Pausch%20H,%20Stothard%20P,%20Van%20Binsbergen%20R,%20Br%C3%B8ndum%20RF,%20et%20al.%20Whole-genome%20sequencing%20of%20234%20bulls%20facilitates%20mapping%20of%20monogenic%20and%20complex%20traits%20in%20cattle.%20Nat%20Genet.%202014;46:858%E2%80%9365." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: A tool for Genome-wide complex trait analysis. Am J Hum Genet. 2011;88:76–82.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2010.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3014363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21167468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20J,%20Lee%20SH,%20Goddard%20ME,%20Visscher%20PM.%20GCTA:%20A%20tool%20for%20Genome-wide%20complex%20trait%20analysis.%20Am%20J%20Hum%20Genet.%202011;88:76%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Bonder MJ, Luijk R, Zhernakova DV, Moed M, Deelen P, Vermaat M, et al. Disease variants alter transcription factor levels and methylation of their binding sites. Nat Genet. 2017;49:131–8.
</cite> [<a href="https://doi.org/10.1038/ng.3721" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27918535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bonder%20MJ,%20Luijk%20R,%20Zhernakova%20DV,%20Moed%20M,%20Deelen%20P,%20Vermaat%20M,%20et%20al.%20Disease%20variants%20alter%20transcription%20factor%20levels%20and%20methylation%20of%20their%20binding%20sites.%20Nat%20Genet.%202017;49:131%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Morrow JD, Glass K, Cho MH, Hersh CP, Pinto-Plata V, Celli B, et al. Human lung DNA methylation quantitative trait loci colocalize with chronic obstructive pulmonary disease genome-wide association loci. Am J Respir Crit Care Med. 2018;197:1275–84.
</cite> [<a href="https://doi.org/10.1164/rccm.201707-1434OC" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5955059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29313708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morrow%20JD,%20Glass%20K,%20Cho%20MH,%20Hersh%20CP,%20Pinto-Plata%20V,%20Celli%20B,%20et%20al.%20Human%20lung%20DNA%20methylation%20quantitative%20trait%20loci%20colocalize%20with%20chronic%20obstructive%20pulmonary%20disease%20genome-wide%20association%20loci.%20Am%20J%20Respir%20Crit%20Care%20Med.%202018;197:1275%E2%80%9384." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Zhang Z, Schwartz S, Wagner L, Miller W. A greedy algorithm for aligning DNA sequences. J Comput Biol. 2000;7:203–14.
</cite> [<a href="https://doi.org/10.1089/10665270050081478" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10890397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Z,%20Schwartz%20S,%20Wagner%20L,%20Miller%20W.%20A%20greedy%20algorithm%20for%20aligning%20DNA%20sequences.%20J%20Comput%20Biol.%202000;7:203%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Nucleotide BLAST. Search nucleotide databases using a nucleotide query. <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastSearch%26;PROG_DEF=blastn%26;BLAST_SPEC=GDH_GCA_947034695.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastSearch%26;PROG_DEF=blastn%26;BLAST_SPEC=GDH_GCA_947034695.1</a>. Accessed 25 Oct 2024.</cite>
</li>
<li id="CR33">
<span class="label">33.</span><cite>Garrison E, Guarracino A, Heumos S, Villani F, Bao Z, Tattini L, et al. Building pangenome graphs. Nat Methods. 2024;21:2008–12.
</cite> [<a href="https://doi.org/10.1038/s41592-024-02430-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39433878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garrison%20E,%20Guarracino%20A,%20Heumos%20S,%20Villani%20F,%20Bao%20Z,%20Tattini%20L,%20et%20al.%20Building%20pangenome%20graphs.%20Nat%20Methods.%202024;21:2008%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Wick RR, Schultz MB, Zobel J, Holt KE. Bandage: interactive visualization of de novo genome assemblies. Bioinformatics. 2015;31(20):3350–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4595904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26099265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wick%20RR,%20Schultz%20MB,%20Zobel%20J,%20Holt%20KE.%20Bandage:%20interactive%20visualization%20of%20de%20novo%20genome%20assemblies.%20Bioinformatics.%202015;31(20):3350%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Ensembl - Biomart. <a href="https://www.ensembl.org/biomart/martview" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ensembl.org/biomart/martview</a>. Accessed 9 Sep 2024.</cite>
</li>
<li id="CR36">
<span class="label">36.</span><cite>UCSC Genome Browser Gateway. <a href="https://genome-euro.ucsc.edu/cgi-bin/hgGateway?redirect=manual&amp;source=genome.ucsc.edu" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://genome-euro.ucsc.edu/cgi-bin/hgGateway?redirect=manual&amp;source=genome.ucsc.edu</a>. Accessed 9 Sep 2024.</cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Salavati M, Clark R, Becker D, Kühn C, Plastow G, Dupont S, et al. Improving the annotation of the cattle genome by annotating transcription start sites in a diverse set of tissues and populations using cap analysis gene expression sequencing. G3 Genes|Genomes|Genetics. 2023;13:jkad108.
</cite> [<a href="https://doi.org/10.1093/g3journal/jkad108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10411599/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37216666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Salavati%20M,%20Clark%20R,%20Becker%20D,%20K%C3%BChn%20C,%20Plastow%20G,%20Dupont%20S,%20et%20al.%20Improving%20the%20annotation%20of%20the%20cattle%20genome%20by%20annotating%20transcription%20start%20sites%20in%20a%20diverse%20set%20of%20tissues%20and%20populations%20using%20cap%20analysis%20gene%20expression%20sequencing.%20G3%20Genes%7CGenomes%7CGenetics.%202023;13:jkad108." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Yuan C, Tang L, Lopdell T, Petrov VA, Oget-Ebrad C, Moreira GCM, et al. An organism-wide ATAC-seq peak catalog for the bovine and its use to identify regulatory variants. Genome Res. 2023;33:1848–64.
</cite> [<a href="https://doi.org/10.1101/gr.277947.123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10691486/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37751945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan%20C,%20Tang%20L,%20Lopdell%20T,%20Petrov%20VA,%20Oget-Ebrad%20C,%20Moreira%20GCM,%20et%20al.%20An%20organism-wide%20ATAC-seq%20peak%20catalog%20for%20the%20bovine%20and%20its%20use%20to%20identify%20regulatory%20variants.%20Genome%20Res.%202023;33:1848%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Marakulina D, Vorontsov IE, Kulakovskiy IV, Lennartsson A, Drabløs F, Medvedeva YA. Epifactors 2022: expansion and enhancement of a curated database of human epigenetic factors and complexes. Nucleic Acids Res. 2023;51:D564-70.
</cite> [<a href="https://doi.org/10.1093/nar/gkac989" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825597/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36350659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marakulina%20D,%20Vorontsov%20IE,%20Kulakovskiy%20IV,%20Lennartsson%20A,%20Drabl%C3%B8s%20F,%20Medvedeva%20YA.%20Epifactors%202022:%20expansion%20and%20enhancement%20of%20a%20curated%20database%20of%20human%20epigenetic%20factors%20and%20complexes.%20Nucleic%20Acids%20Res.%202023;51:D564-70." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Castro-Mondragon JA, Riudavets-Puig R, Rauluseviciute I, Berhanu Lemma R, Turchi L, Blanc-Mathieu R, et al. JASPAR 2022: the 9th release of the open-access database of transcription factor binding profiles. Nucleic Acids Res. 2022;50:D165–73.
</cite> [<a href="https://doi.org/10.1093/nar/gkab1113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34850907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Castro-Mondragon%20JA,%20Riudavets-Puig%20R,%20Rauluseviciute%20I,%20Berhanu%20Lemma%20R,%20Turchi%20L,%20Blanc-Mathieu%20R,%20et%20al.%20JASPAR%202022:%20the%209th%20release%20of%20the%20open-access%20database%20of%20transcription%20factor%20binding%20profiles.%20Nucleic%20Acids%20Res.%202022;50:D165%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Bailey TL, Grant CE. SEA: Simple Enrichment Analysis of motifs. BioRxiv. 2021;2021(08):23.457422. 10.1101/2021.08.23.457422.</cite>
</li>
<li id="CR42">
<span class="label">42.</span><cite>Powers NR, Parvanov ED, Baker CL, Walker M, Petkov PM, Paigen K. The meiotic recombination activator PRDM9 trimethylates both H3K36 and H3K4 at recombination hotspots in vivo. PLoS Genet. 2016;12:e1006146.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1006146" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4928815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27362481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Powers%20NR,%20Parvanov%20ED,%20Baker%20CL,%20Walker%20M,%20Petkov%20PM,%20Paigen%20K.%20The%20meiotic%20recombination%20activator%20PRDM9%20trimethylates%20both%20H3K36%20and%20H3K4%20at%20recombination%20hotspots%20in%20vivo.%20PLoS%20Genet.%202016;12:e1006146." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Zoch A, Auchynnikava T, Berrens RV, Kabayama Y, Schöpp T, Heep M, et al. SPOCD1 is an essential executor of piRNA-directed de novo DNA methylation. Nature. 2020;584:635–9.
</cite> [<a href="https://doi.org/10.1038/s41586-020-2557-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7612247/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32674113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zoch%20A,%20Auchynnikava%20T,%20Berrens%20RV,%20Kabayama%20Y,%20Sch%C3%B6pp%20T,%20Heep%20M,%20et%20al.%20SPOCD1%20is%20an%20essential%20executor%20of%20piRNA-directed%20de%20novo%20DNA%20methylation.%20Nature.%202020;584:635%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Malonia SK, Sinha S, Lakshminarasimhan P, Singh K, Jalota-Badhwar A, Rampalli S, et al. Gene regulation by SMAR1: role in cellular homeostasis and cancer. Biochimica et Biophysica Acta (BBA). 2011;1815:1–12.
</cite> [<a href="https://doi.org/10.1016/j.bbcan.2010.08.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20709157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Malonia%20SK,%20Sinha%20S,%20Lakshminarasimhan%20P,%20Singh%20K,%20Jalota-Badhwar%20A,%20Rampalli%20S,%20et%20al.%20Gene%20regulation%20by%20SMAR1:%20role%20in%20cellular%20homeostasis%20and%20cancer.%20Biochimica%20et%20Biophysica%20Acta%20(BBA).%202011;1815:1%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Grand RS, Burger L, Gräwe C, Michael AK, Isbel L, Hess D, et al. BANP opens chromatin and activates CpG-island-regulated genes. Nature. 2021;596:133–7.
</cite> [<a href="https://doi.org/10.1038/s41586-021-03689-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34234345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grand%20RS,%20Burger%20L,%20Gr%C3%A4we%20C,%20Michael%20AK,%20Isbel%20L,%20Hess%20D,%20et%20al.%20BANP%20opens%20chromatin%20and%20activates%20CpG-island-regulated%20genes.%20Nature.%202021;596:133%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Nakka KK, Chaudhary N, Joshi S, Bhat J, Singh K, Chatterjee S, et al. Nuclear matrix-associated protein SMAR1 regulates alternative splicing via HDAC6-mediated deacetylation of Sam68. Proc Natl Acad Sci U S A. 2015;112:E3374-83.
</cite> [<a href="https://doi.org/10.1073/pnas.1418603112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4491761/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26080397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakka%20KK,%20Chaudhary%20N,%20Joshi%20S,%20Bhat%20J,%20Singh%20K,%20Chatterjee%20S,%20et%20al.%20Nuclear%20matrix-associated%20protein%20SMAR1%20regulates%20alternative%20splicing%20via%20HDAC6-mediated%20deacetylation%20of%20Sam68.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202015;112:E3374-83." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Alam A, Taye N, Patel S, Thube M, Mullick J, Shah VK, et al. SMAR1 favors immunosurveillance of cancer cells by modulating calnexin and MHC I expression. Neoplasia. 2019;21:945–62.
</cite> [<a href="https://doi.org/10.1016/j.neo.2019.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6706529/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31422285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alam%20A,%20Taye%20N,%20Patel%20S,%20Thube%20M,%20Mullick%20J,%20Shah%20VK,%20et%20al.%20SMAR1%20favors%20immunosurveillance%20of%20cancer%20cells%20by%20modulating%20calnexin%20and%20MHC%20I%20expression.%20Neoplasia.%202019;21:945%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Palamarchuk A, Yan PS, Zanesi N, Wang L, Rodrigues B, Murphy M, et al. Tcl1 protein functions as an inhibitor of de novo DNA methylation in B-cell chronic lymphocytic leukemia (CLL). Proc Natl Acad Sci U S A. 2012;109:2555–60.
</cite> [<a href="https://doi.org/10.1073/pnas.1200003109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3289317/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22308499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palamarchuk%20A,%20Yan%20PS,%20Zanesi%20N,%20Wang%20L,%20Rodrigues%20B,%20Murphy%20M,%20et%20al.%20Tcl1%20protein%20functions%20as%20an%20inhibitor%20of%20de%20novo%20DNA%20methylation%20in%20B-cell%20chronic%20lymphocytic%20leukemia%20(CLL).%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202012;109:2555%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Grundberg E, Meduri E, Sandling JK, Hedman AK, Keildson S, Buil A, et al. Global analysis of DNA methylation variation in adipose tissue from twins reveals links to disease-associated variants in distal regulatory elements. Am J Hum Genet. 2013;93:876–90.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2013.10.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3824131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24183450/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grundberg%20E,%20Meduri%20E,%20Sandling%20JK,%20Hedman%20AK,%20Keildson%20S,%20Buil%20A,%20et%20al.%20Global%20analysis%20of%20DNA%20methylation%20variation%20in%20adipose%20tissue%20from%20twins%20reveals%20links%20to%20disease-associated%20variants%20in%20distal%20regulatory%20elements.%20Am%20J%20Hum%20Genet.%202013;93:876%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Gaunt TR, Shihab HA, Hemani G, Min JL, Woodward G, Lyttleton O, et al. Systematic identification of genetic influences on methylation across the human life course. Genome Biol. 2016;17:61.
</cite> [<a href="https://doi.org/10.1186/s13059-016-0926-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4818469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27036880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gaunt%20TR,%20Shihab%20HA,%20Hemani%20G,%20Min%20JL,%20Woodward%20G,%20Lyttleton%20O,%20et%20al.%20Systematic%20identification%20of%20genetic%20influences%20on%20methylation%20across%20the%20human%20life%20course.%20Genome%20Biol.%202016;17:61." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Slieker RC, Roost MS, van Iperen L, Suchiman HED, Tobi EW, Carlotti F, et al. DNA methylation landscapes of human fetal development. PLoS Genet. 2015;11: e1005583.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1005583" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4619663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26492326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Slieker%20RC,%20Roost%20MS,%20van%20Iperen%20L,%20Suchiman%20HED,%20Tobi%20EW,%20Carlotti%20F,%20et%20al.%20DNA%20methylation%20landscapes%20of%20human%20fetal%20development.%20PLoS%20Genet.%202015;11:%20e1005583." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Smith AK, Kilaru V, Kocak M, Almli LM, Mercer KB, Ressler KJ, et al. Methylation quantitative trait loci (meQTLs) are consistently detected across ancestry, developmental stage, and tissue type. BMC Genomics. 2014;15: 145.
</cite> [<a href="https://doi.org/10.1186/1471-2164-15-145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4028873/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24555763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith%20AK,%20Kilaru%20V,%20Kocak%20M,%20Almli%20LM,%20Mercer%20KB,%20Ressler%20KJ,%20et%20al.%20Methylation%20quantitative%20trait%20loci%20(meQTLs)%20are%20consistently%20detected%20across%20ancestry,%20developmental%20stage,%20and%20tissue%20type.%20BMC%20Genomics.%202014;15:%20145." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Lin D, Chen J, Perrone-Bizzozero N, Bustillo JR, Du Y, Calhoun VD, et al. Characterization of cross-tissue genetic-epigenetic effects and their patterns in schizophrenia. Genome Med. 2018;10: 13.
</cite> [<a href="https://doi.org/10.1186/s13073-018-0519-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5828480/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29482655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20D,%20Chen%20J,%20Perrone-Bizzozero%20N,%20Bustillo%20JR,%20Du%20Y,%20Calhoun%20VD,%20et%20al.%20Characterization%20of%20cross-tissue%20genetic-epigenetic%20effects%20and%20their%20patterns%20in%20schizophrenia.%20Genome%20Med.%202018;10:%2013." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Feinberg JI, Schrott R, Ladd-Acosta C, Newschaffer CJ, Hertz-Picciotto I, Croen LA, et al. Epigenetic changes in sperm are associated with paternal and child quantitative autistic traits in an autism-enriched cohort. Mol Psychiatry. 2024;29:43–53.
</cite> [<a href="https://doi.org/10.1038/s41380-023-02046-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37100868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feinberg%20JI,%20Schrott%20R,%20Ladd-Acosta%20C,%20Newschaffer%20CJ,%20Hertz-Picciotto%20I,%20Croen%20LA,%20et%20al.%20Epigenetic%20changes%20in%20sperm%20are%20associated%20with%20paternal%20and%20child%20quantitative%20autistic%20traits%20in%20an%20autism-enriched%20cohort.%20Mol%20Psychiatry.%202024;29:43%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Hop PJ, Luijk R, Daxinger L, Van Iterson M, Dekkers KF, Jansen R, et al. Genome-wide identification of genes regulating DNA methylation using genetic anchors for causal inference. Genome Biol. 2020;21: 220.
</cite> [<a href="https://doi.org/10.1186/s13059-020-02114-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7453518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32859263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hop%20PJ,%20Luijk%20R,%20Daxinger%20L,%20Van%20Iterson%20M,%20Dekkers%20KF,%20Jansen%20R,%20et%20al.%20Genome-wide%20identification%20of%20genes%20regulating%20DNA%20methylation%20using%20genetic%20anchors%20for%20causal%20inference.%20Genome%20Biol.%202020;21:%20220." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Banovich NE, Lan X, McVicker G, Geijn B, van de, Degner JF, Blischak JD, et al. Methylation QTLs are associated with coordinated changes in transcription factor binding, histone modifications, and gene expression levels. PLoS Genet. 2014;10:e1004663.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1004663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4169251/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25233095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banovich%20NE,%20Lan%20X,%20McVicker%20G,%20Geijn%20B,%20van%20de,%20Degner%20JF,%20Blischak%20JD,%20et%20al.%20Methylation%20QTLs%20are%20associated%20with%20coordinated%20changes%20in%20transcription%20factor%20binding,%20histone%20modifications,%20and%20gene%20expression%20levels.%20PLoS%20Genet.%202014;10:e1004663." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Cedar H, Bergman Y. Linking DNA methylation and histone modification: patterns and paradigms. Nat Rev Genet. 2009;10:295–304.
</cite> [<a href="https://doi.org/10.1038/nrg2540" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19308066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cedar%20H,%20Bergman%20Y.%20Linking%20DNA%20methylation%20and%20histone%20modification:%20patterns%20and%20paradigms.%20Nat%20Rev%20Genet.%202009;10:295%E2%80%93304." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Kiefer H, Perrier J-P. DNA methylation in bull spermatozoa: evolutionary impacts, interindividual variability, and contribution to the embryo. Can J Anim Sci. 2020;100:1–16.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kiefer%20H,%20Perrier%20J-P.%20DNA%20methylation%20in%20bull%20spermatozoa:%20evolutionary%20impacts,%20interindividual%20variability,%20and%20contribution%20to%20the%20embryo.%20Can%20J%20Anim%20Sci.%202020;100:1%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Sandor C, Li W, Coppieters W, Druet T, Charlier C, Georges M. Genetic variants in REC8, RNF212, and PRDM9 influence male recombination in cattle. PLoS Genet. 2012;8: e1002854.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1002854" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3406008/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22844258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sandor%20C,%20Li%20W,%20Coppieters%20W,%20Druet%20T,%20Charlier%20C,%20Georges%20M.%20Genetic%20variants%20in%20REC8,%20RNF212,%20and%20PRDM9%20influence%20male%20recombination%20in%20cattle.%20PLoS%20Genet.%202012;8:%20e1002854." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Leonard AS, Mapel XM, Pausch H. Pangenome-genotyped structural variation improves molecular phenotype mapping in cattle. Genome Res. 2024;34:300–9.
</cite> [<a href="https://doi.org/10.1101/gr.278267.123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10984387/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38355307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leonard%20AS,%20Mapel%20XM,%20Pausch%20H.%20Pangenome-genotyped%20structural%20variation%20improves%20molecular%20phenotype%20mapping%20in%20cattle.%20Genome%20Res.%202024;34:300%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Attree E, Griffiths B, Panchal K, Xia D, Werling D, Banos G, et al. Identification of DNA methylation markers for age and bovine respiratory disease in dairy cattle: a pilot study based on reduced representation bisulfite sequencing. Commun Biol. 2024;7:1–15.
</cite> [<a href="https://doi.org/10.1038/s42003-024-06925-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11450024/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39363014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Attree%20E,%20Griffiths%20B,%20Panchal%20K,%20Xia%20D,%20Werling%20D,%20Banos%20G,%20et%20al.%20Identification%20of%20DNA%20methylation%20markers%20for%20age%20and%20bovine%20respiratory%20disease%20in%20dairy%20cattle:%20a%20pilot%20study%20based%20on%20reduced%20representation%20bisulfite%20sequencing.%20Commun%20Biol.%202024;7:1%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Dong W, Yang J, Zhang Y, Liu S, Ning C, Ding X, et al. Integrative analysis of genome-wide DNA methylation and gene expression profiles reveals important epigenetic genes related to milk production traits in dairy cattle. J Anim Breed Genet. 2021;138:562–73.
</cite> [<a href="https://doi.org/10.1111/jbg.12530" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33620112/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dong%20W,%20Yang%20J,%20Zhang%20Y,%20Liu%20S,%20Ning%20C,%20Ding%20X,%20et%20al.%20Integrative%20analysis%20of%20genome-wide%20DNA%20methylation%20and%20gene%20expression%20profiles%20reveals%20important%20epigenetic%20genes%20related%20to%20milk%20production%20traits%20in%20dairy%20cattle.%20J%20Anim%20Breed%20Genet.%202021;138:562%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Meissner A, Mikkelsen TS, Gu H, Wernig M, Hanna J, Sivachenko A, et al. Genome-scale DNA methylation maps of pluripotent and differentiated cells. Nature. 2008;454:766–70.
</cite> [<a href="https://doi.org/10.1038/nature07107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2896277/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18600261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meissner%20A,%20Mikkelsen%20TS,%20Gu%20H,%20Wernig%20M,%20Hanna%20J,%20Sivachenko%20A,%20et%20al.%20Genome-scale%20DNA%20methylation%20maps%20of%20pluripotent%20and%20differentiated%20cells.%20Nature.%202008;454:766%E2%80%9370." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Göring HHH, Terwilliger JD, Blangero J. Large upward bias in estimation of locus-specific effects from genomewide scans. Am J Hum Genet. 2001;69:1357–69.
</cite> [<a href="https://doi.org/10.1086/324471" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1235546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11593451/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?G%C3%B6ring%20HHH,%20Terwilliger%20JD,%20Blangero%20J.%20Large%20upward%20bias%20in%20estimation%20of%20locus-specific%20effects%20from%20genomewide%20scans.%20Am%20J%20Hum%20Genet.%202001;69:1357%E2%80%9369." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374274/bin/12864_2025_11934_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (342.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374274/bin/12864_2025_11934_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (13.2MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>RRBS fastq files are available through the NCBI SRA under accession number PRJNA1225939 (<a href="http://www.ncbi.nlm.nih.gov/bioproject/1225939" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.ncbi.nlm.nih.gov/bioproject/1225939</a>). The fasta files of the eight 8 Holstein used to build the partial pangenome are available in the European Nucleotide Archive (ENA) at EMBL-EBI under accession number PRJEB68295 (<a href="https://www.ebi.ac.uk/ena/data/view/PRJEB68295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/ena/data/view/PRJEB68295</a>). Genotypes of the 405 bulls are commercial information belonging to the French breeders and are not publicly available. All other datasets supporting findings are included in the article and its supplementary tables and figures.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Genomics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12864-025-11934-x"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12864_2025_Article_11934.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12374274/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12374274/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374274%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374274/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12374274/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12374274/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40847283/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12374274/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40847283/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12374274/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12374274/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="xlFdRCIYz5yJwA47l44595KAtN18SPVtxuWe1dzN8QaMj9zGfML0xwpLDlOIoE7q">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
