
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Molecular characterization of Neospora caninum major antigens NcSAG1 and NcSRS2 - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AF3E8AF3180305AF3E000F966CC4.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="rsos">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365351/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Royal Society Open Science">
<meta name="citation_title" content="Molecular characterization of Neospora caninum major antigens NcSAG1 and NcSRS2">
<meta name="citation_author" content="Soledad Echeverría">
<meta name="citation_author_institution" content="Laboratorio de Interacciones Hospedero Patogeno, Institut Pasteur de Montevideo, Montevideo, Uruguay">
<meta name="citation_author" content="Federico Carrión">
<meta name="citation_author_institution" content="Laboratorio de Inmunovirología / Unidad de Biofísica de Proteínas, Institut Pasteur de Montevideo, Montevideo, Uruguay">
<meta name="citation_author" content="Martín Soñora">
<meta name="citation_author_institution" content="Laboratorio de Simulaciones Biomoleculares, Institut Pasteur de Montevideo, Montevideo, Uruguay">
<meta name="citation_author" content="Andrés Cabrera">
<meta name="citation_author_institution" content="Laboratorio de Interacciones Hospedero Patogeno, Institut Pasteur de Montevideo, Montevideo, Uruguay">
<meta name="citation_author_institution" content="Unidad Académica de Parasitología, Facultad de Medicina, UDELAR, Uruguay">
<meta name="citation_author" content="Carlos Robello">
<meta name="citation_author_institution" content="Laboratorio de Interacciones Hospedero Patogeno, Institut Pasteur de Montevideo, Montevideo, Uruguay">
<meta name="citation_author_institution" content="Unidad Académica de Bioquímica, Facultad de Medicina, UDELAR, Uruguay">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="12">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="250239">
<meta name="citation_doi" content="10.1098/rsos.250239">
<meta name="citation_pmid" content="40843189">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365351/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365351/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365351/pdf/rsos.250239.pdf">
<meta name="description" content="The SAG1-related sequence (SRS) protein family was initially identified in Toxoplasma gondii as glycosyl-phosphatidylinositol-anchored surface antigens. More recently, they have been identified in Neospora caninum, the causative agent of ...">
<meta name="og:title" content="Molecular characterization of Neospora caninum major antigens NcSAG1 and NcSRS2">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The SAG1-related sequence (SRS) protein family was initially identified in Toxoplasma gondii as glycosyl-phosphatidylinositol-anchored surface antigens. More recently, they have been identified in Neospora caninum, the causative agent of ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365351/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365351">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1098/rsos.250239"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/rsos.250239.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365351%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365351/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365351/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365351/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-rsos.gif" alt="Royal Society Open Science logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Royal Society Open Science" title="Link to Royal Society Open Science" shape="default" href="http://rsos.royalsocietypublishing.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">R Soc Open Sci</button></div>. 2025 Aug 20;12(8):250239. doi: <a href="https://doi.org/10.1098/rsos.250239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1098/rsos.250239</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22R%20Soc%20Open%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22R%20Soc%20Open%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22R%20Soc%20Open%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22R%20Soc%20Open%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Molecular characterization of <em>Neospora caninum</em> major antigens NcSAG1 and NcSRS2</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Echeverr%C3%ADa%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Soledad Echeverría</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Soledad Echeverría</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Laboratorio de Interacciones Hospedero Patogeno, Institut Pasteur de Montevideo, Montevideo, Uruguay</div>
<div>Investigation, Methodology, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Echeverr%C3%ADa%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Soledad Echeverría</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carri%C3%B3n%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Federico Carrión</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Federico Carrión</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Laboratorio de Inmunovirología / Unidad de Biofísica de Proteínas, Institut Pasteur de Montevideo, Montevideo, Uruguay</div>
<div>Investigation, Methodology, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carri%C3%B3n%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Federico Carrión</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22So%C3%B1ora%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Martín Soñora</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Martín Soñora</span></h3>
<div class="p">
<sup><sup>3</sup></sup>Laboratorio de Simulaciones Biomoleculares, Institut Pasteur de Montevideo, Montevideo, Uruguay</div>
<div>Investigation, Methodology, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22So%C3%B1ora%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martín Soñora</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cabrera%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Andrés Cabrera</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Andrés Cabrera</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Laboratorio de Interacciones Hospedero Patogeno, Institut Pasteur de Montevideo, Montevideo, Uruguay</div>
<div class="p">
<sup><sup>4</sup></sup>Unidad Académica de Parasitología, Facultad de Medicina, UDELAR, Uruguay</div>
<div>Conceptualization, Investigation, Supervision, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cabrera%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrés Cabrera</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Robello%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Carlos Robello</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Carlos Robello</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Laboratorio de Interacciones Hospedero Patogeno, Institut Pasteur de Montevideo, Montevideo, Uruguay</div>
<div class="p">
<sup><sup>5</sup></sup>Unidad Académica de Bioquímica, Facultad de Medicina, UDELAR, Uruguay</div>
<div>Conceptualization, Funding acquisition, Investigation, Project administration, Supervision, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Robello%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Carlos Robello</span></a>
</div>
</div>
<sup>1,</sup><sup>5,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>Laboratorio de Interacciones Hospedero Patogeno, Institut Pasteur de Montevideo, Montevideo, Uruguay</div>
<div id="aff2">
<sup><sup>2</sup></sup>Laboratorio de Inmunovirología / Unidad de Biofísica de Proteínas, Institut Pasteur de Montevideo, Montevideo, Uruguay</div>
<div id="aff3">
<sup><sup>3</sup></sup>Laboratorio de Simulaciones Biomoleculares, Institut Pasteur de Montevideo, Montevideo, Uruguay</div>
<div id="aff4">
<sup><sup>4</sup></sup>Unidad Académica de Parasitología, Facultad de Medicina, UDELAR, Uruguay</div>
<div id="aff5">
<sup><sup>5</sup></sup>Unidad Académica de Bioquímica, Facultad de Medicina, UDELAR, Uruguay</div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>Electronic supplementary material is available online at <a href="https://doi.org/10.6084/m9.figshare.c.7948755" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.6084/m9.figshare.c.7948755</a>.</p></div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Soledad Echeverría</span></strong>: <span class="role">Investigation, Methodology, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Federico Carrión</span></strong>: <span class="role">Investigation, Methodology, Writing – original draft, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Martín Soñora</span></strong>: <span class="role">Investigation, Methodology, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Andrés Cabrera</span></strong>: <span class="role">Conceptualization, Investigation, Supervision, Writing – original draft, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Carlos Robello</span></strong>: <span class="role">Conceptualization, Funding acquisition, Investigation, Project administration, Supervision, Writing – review and editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 2; Revised 2025 Jun 10; Accepted 2025 Jun 11; Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors.</div>
<p>Published by the Royal Society under the terms of the Creative Commons Attribution License <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>, which permits unrestricted use, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365351  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40843189/" class="usa-link">40843189</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The SAG1-related sequence (SRS) protein family was initially identified in <em>Toxoplasma gondii</em> as glycosyl-phosphatidylinositol-anchored surface antigens. More recently, they have been identified in <em>Neospora caninum</em>, the causative agent of neosporosis, a leading cause of bovine abortion worldwide. These proteins are implicated in parasite adhesion to and invasion of host cells, immune response modulation and structural roles in the cyst wall. In this study, we characterized two key <em>N. caninum</em> SRS proteins, NcSAG1 and NcSRS2, through sequence analysis, structural modelling, biophysical characterization and immunochemical assessment. Sequence analyses revealed conserved domains, including hallmark D1 and D2 regions, but with significant sequence divergence. Using A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span>, we constructed reliable structural models, confirming conserved features such as disulfide bond patterns and dimerization. Structural comparisons demonstrated a high degree of conservation within D1 domains despite low sequence similarity. Recombinant NcSAG1 and NcSRS2 were expressed as soluble and stable proteins in <em>Drosophila melanogaster</em> S2 cells, achieving yields comparable to the most efficient prokaryote expression systems. Size exclusion chromatography and dynamic light scattering demonstrated their dimeric nature and structural stability, with melting temperatures exceeding 50°C. Circular dichroism spectroscopy confirmed their correct secondary structure content, validating proper folding and structural integrity. Antigenicity assays demonstrated universal recognition by sera from experimentally and naturally infected hosts, highlighting their potential as diagnostic markers or vaccine candidates. Comparative structural analysis of 219 SRS family members, based on sequence and A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span>-predicted structures, revealed conserved cysteine, proline and tryptophan motifs. Hierarchical clustering and phylogenetic analyses identified key evolutionary clusters, correlating structural divergence with functional specialization. Discrepancies between sequence- and structure-based trees underscored instances of structural evolution not reflected in sequence data. This comprehensive analysis bridges sequence divergence, structural conservation and biological function, providing a robust framework for investigating SRS proteins’ roles in pathogenesis and immunity. Our findings lay the groundwork for future research into <em>N. caninum</em>’s molecular mechanisms and their implications for controlling neosporosis.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> SAG1-related sequence proteins, SRS, NcSAG1, NcSRS2, <em>Neospora caninum</em>, apicomplexa</p></section></section><section id="s1"><h2 class="pmc_sec_title">1.  Introduction</h2>
<p>The apicomplexan <em>Neospora caninum</em> is an obligate intracellular parasite closely related to <em>Toxoplasma gondii</em> [<a href="#B1" class="usa-link" aria-describedby="B1">1</a>,<a href="#B2" class="usa-link" aria-describedby="B2">2</a>]. Dogs act as the definitive hosts in the life cycle of <em>N. caninum</em>, where the sexual replication of the parasite occurs in the intestinal epithelium, culminating in the production of oocysts [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>]. These oocysts are shed in the faeces of infected dogs, contaminating the environment and serving as a source of infection for both other canids and a wide range of herbivorous intermediate hosts, such as cattle and sheep [<a href="#B4" class="usa-link" aria-describedby="B4">4</a>,<a href="#B5" class="usa-link" aria-describedby="B5">5</a>]. In some cases, <em>N. caninum</em> infection in dogs can disseminate to the central nervous system, leading to chronic neurological manifestations, including ataxia, hind limb paresis and muscle atrophy [<a href="#B6" class="usa-link" aria-describedby="B6">6</a>]. The economic impact of neosporosis is most significant in cattle, where it causes bovine neosporosis [<a href="#B7" class="usa-link" aria-describedby="B7">7</a>], the leading cause of bovine abortion worldwide [<a href="#B8" class="usa-link" aria-describedby="B8">8</a>]. Infected cattle can abort from three months of pregnancy onwards, or the parasite can be transmitted vertically from mother to offspring [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>]. Congenitally infected calves may or may not exhibit symptoms but can serve as carriers, maintaining the disease within the herd [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>]. Various strategies have been proposed for controlling and diagnosing neosporosis, with surface antigen proteins showing high potential for diagnosis and vaccine development [<a href="#B9" class="usa-link" aria-describedby="B9">9</a>].</p>
<p>The SAG1-related sequence (SRS) protein family was initially identified in <em>T. gondii</em> as major surface antigens and named based on their apparent molecular weight (MW) in sodium dodecyl sulfate–polyacrylamide gel electrophoresis (SDS-PAGE) [<a href="#B10" class="usa-link" aria-describedby="B10">10</a>]. In 1991, a nomenclature was proposed using capital letters (e.g. P30 was called SAG1, and P22 was called SAG2) [<a href="#B11" class="usa-link" aria-describedby="B11">11</a>]. The SRS superfamily is also present in <em>N. caninum</em>, where orthologues for SRS proteins described in <em>T. gondii</em> were found [<a href="#B12" class="usa-link" aria-describedby="B12">12</a>]. Recent studies involving the genomes of both <em>T. gondii</em> and <em>N. caninum</em> revealed that the SRS family appears more extensive in <em>N. caninum</em> than in <em>T. gondii</em> [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>,<a href="#B14" class="usa-link" aria-describedby="B14">14</a>]. SRS proteins exhibit a sequence identity of approximately 25%–30% and share several common characteristics, including an N-terminal signal peptide with 8–12 conserved cysteine residues, four conserved prolines, a conserved tryptophan in the membrane-proximal domain and a C-terminal glycosyl-phosphatidylinositol (GPI) signal. Most of them are anchored to the membrane by GPI (GPI-anchored) [<a href="#B15" class="usa-link" aria-describedby="B15">15</a>,<a href="#B16" class="usa-link" aria-describedby="B16">16</a>]. The crystal structures of SAG1, BSR4 and SporoSAG from <em>T. gondii</em> have been obtained [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>–<a href="#B19" class="usa-link" aria-describedby="B19">19</a>]. These structures—crystallized without the signal peptide and the GPI anchor region—revealed the presence of two domains, D1 and D2, connected by a flexible hinge linker. These domains form a β-sandwich structure consisting of antiparallel and parallel strands stabilized by three conserved disulfide bonds. The reported structure appears as a homodimer, with the dimeric form resembling an immunoglobulin domain, suggesting a potential relationship to its proposed function in ligand recognition. This implies that the dimer would play a significant role in the protein’s ability to recognize and bind to specific ligands [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>].</p>
<p>Studies on the tachyzoite-to-bradyzoite transition have revealed significant changes in the expression profiles of SRS proteins, probably reflecting their distinct roles in the parasite’s life cycle [<a href="#B15" class="usa-link" aria-describedby="B15">15</a>,<a href="#B20" class="usa-link" aria-describedby="B20">20</a>]. These expression patterns also vary among strains, correlating with infectivity and virulence: less virulent strains typically express a broader range of SRS proteins, including those associated with bradyzoites [<a href="#B21" class="usa-link" aria-describedby="B21">21</a>–<a href="#B23" class="usa-link" aria-describedby="B23">23</a>]. In <em>T. gondii,</em> the attachment to the host cell is primarily mediated by SAG1; although it is not the only interaction, it is important for adhesion and invasion [<a href="#B24" class="usa-link" aria-describedby="B24">24</a>]. In the case of <em>N. caninum</em>, two SRSs have been described for host cell attachment: NcSAG1 (also known as Ncp36) and NcSRS2 (also known as Ncp43 or Ncp35), playing essential roles in the invasion process [<a href="#B25" class="usa-link" aria-describedby="B25">25</a>–<a href="#B29" class="usa-link" aria-describedby="B29">29</a>]. The work of Nishikawa <em>et al</em>. [<a href="#B27" class="usa-link" aria-describedby="B27">27</a>] shows the relevance of NcSAG1 and NcSRS2 in the invasion process; by using monoclonal antibodies, the capacity of parasites to invade the host cell decreases significantly [<a href="#B27" class="usa-link" aria-describedby="B27">27</a>]. The role of NcSAG1 in the pathogenesis of <em>N. caninum</em> has been demonstrated by gene deletion studies, which revealed that knockout of NcSAG1 significantly reduces parasite infectivity and egress, in murine models [<a href="#B30" class="usa-link" aria-describedby="B30">30</a>]. The importance of NcSRS2 in the attachment and invasion was demonstrated using polyclonal and monoclonal antibodies for NcSRS2 and showing the decrease of attachment and invasion of <em>N. caninum</em> [<a href="#B28" class="usa-link" aria-describedby="B28">28</a>]. It was demonstrated that Nc-p43 (NcSRS2) is localized not only on the tachyzoite surface but also within the rhoptries and dense granules, suggesting that this protein is secreted [<a href="#B31" class="usa-link" aria-describedby="B31">31</a>].</p>
<p>Furthermore, its expression was observed in both tachyzoites and bradyzoites. NcSAG1 and NcSRS2 have been widely used in the development of ELISA-based diagnostic assays, as well as in vaccine research [<a href="#B32" class="usa-link" aria-describedby="B32">32</a>,<a href="#B33" class="usa-link" aria-describedby="B33">33</a>]. Despite extensive studies, the function, structure and physicochemical properties of SRS proteins remain poorly understood, including the nature of ligands they recognize on target cell surfaces and how their diversity influences host range. In this study, we expressed NcSRS2 and NcSAG1 in a eukaryotic system and performed biochemical and structural characterization of these proteins. Our findings provide helpful insights that could enhance the understanding of these proteins.</p></section><section id="s2"><h2 class="pmc_sec_title">2.  Material and methods</h2>
<section id="s2-1"><h3 class="pmc_sec_title">2.1.  Cloning, protein expression and purification</h3>
<p>Based on predictions from the SignalP algorithm, we excluded the signal peptide to produce the recombinant proteins. DNA sequences coding for amino acids 31−317 of <em>NcSAG1</em> (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/AF123660" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AF123660</a>) and 54−401 of <em>NcSRS2</em> (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/AF061249" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AF061249</a>), both fused to a C-terminal enterokinase cleavage site followed by a Twin-Strep-tag® (IBA) and a stop codon, were optimized for <em>Drosophila melanogaster</em> codon usage and synthesized by G<span class="font-variant-small-caps">en</span>S<span class="font-variant-small-caps">cript</span>. Synthetic genes were cloned into a pMT/V5-His expression vector (Invitrogen) immediately after the BiP sequence as previously described [<a href="#B34" class="usa-link" aria-describedby="B34">34</a>].</p>
<p>S2 stable cell lines expressing rNcSAG1 and rNcSRS2 were obtained by co-transfection with the respective expression vector and pCoPuro selection plasmid using a 2 : 1 ratio, as previously described [<a href="#B34" class="usa-link" aria-describedby="B34">34</a>]. Transfections were achieved with an ExPERT STx transfection system using OC-400 processing assemblies (MaxCyte). Transfected cells were selected at 28°C in Insect Xpress medium (LONZA), added with 6 µg ml<sup>−1</sup> puromycin. Stable S2 cell lines were grown in glass flasks at 28°C with 110 rpm agitation in a standard orbital shaker and induced at 5 × 10<sup>6</sup> cells ml<sup>−1</sup> with 5 µM CdCl<sub>2</sub>. After 3 days post-induction, cells were harvested by centrifugation at 150 g for 5 min, and both proteins were purified from the culture supernatant by affinity chromatography (AC) as before [<a href="#B34" class="usa-link" aria-describedby="B34">34</a>]. Briefly, culture supernatants were centrifuged at 6000<em>g</em>, loaded in 5 ml Strep-Tactin®XT 4Flow® columns (IBA), and eluted with 10 ml of 1 × Buffer BXT (Strep‑Tactin®XT elution buffer containing 50 mM biotin; IBA), following the manufacturer’s recommendations. An extra purification step by size exclusion chromatography (SEC) in 0.1 M Tris-Cl pH 8.0, 0.15 M NaCl and 1 mM EDTA (storage buffer) was conducted using Superdex 200 Increase 10/300 GL or HiLoad Superdex 200 16/600 pg columns (Cytiva). When required, the purification tag was removed by incubating SEC-purified proteins overnight at 25°C with Enterokinase-His (Genscript) (20 U mg<sup>−1</sup>) and subjected to a second AC step in the storage buffer. Cleaved proteins (rNcSRS2<sup>*</sup> and rNcSAG1<sup>*</sup>) were collected in the flow-through fraction and subjected to SEC under the same conditions.</p></section><section id="s2-2"><h3 class="pmc_sec_title">2.2.  Analytical size exclusion chromatography</h3>
<p>A volume of 100 μl was injected into a Superdex 200 Increase 10/300 GL column equilibrated in a storage buffer and eluted at 0.5 ml min<sup>-1</sup>. Under the same conditions, calibration was performed using low molecular weight (LMW) and high molecular weight (HMW) gel filtration calibration kits (Cytiva).</p></section><section id="s2-3"><h3 class="pmc_sec_title">2.3.  Nano-scale differential scanning fluorimetry</h3>
<p>The thermostability of rNcSAG1-ST and rNcSRS2-ST was analysed by nano-scale differential scanning fluorimetry (nanoDSF) using a Prometheus NT.48 (Nanotemper). Unfolding and refolding curves were obtained by measuring intrinsic emission fluorescence at 350 and 330 nm upon sequential heating and cooling at 1°C min<sup>−1</sup> between 20°C and 90°C, and melting temperature (<em>T</em><sub>M</sub>) values were calculated from inflection points by following the first derivative of the <em>F</em><sub>350</sub>/<em>F</em><sub>330</sub> ratio.</p></section><section id="s2-4"><h3 class="pmc_sec_title">2.4.  Dynamic light scattering</h3>
<p>The hydrodynamic radius (R<sub>H</sub>) of recombinant proteins was analysed by dynamic light scattering (DLS) in a Zetasizer Nano S (Malvern) using low-volume quartz cuvettes (ZEN2112) at 25°C. Measurements were conducted in triplicate with rNcSAG1<sup>*</sup> and rNcSRS2<sup>*</sup> after samples were concentrated to 1 mg ml<sup>−1</sup> by centrifugal ultrafiltration using Amicon® Ultra-15 tubes (Millipore).</p></section><section id="s2-5"><h3 class="pmc_sec_title">2.5.  Far-UV circular dichroism</h3>
<p>Secondary structure analysis of rNcSAG1* and rNcSRS2* was achieved using a Chirascan V100 (Applied Photophysics) at 25°C. After Strep-tag removal, followed by protein ultrafiltration, sample absorbance at 280 nm was measured using a 10 mm pathlength quartz cuvette to calculate protein concentration. To obtain far-UV circular dichroism (far-UV CD) spectra, samples were diluted 1 : 20 in deuterium oxide to 3.04 and 3.58 μM, respectively, and measured between 260 and 180 nm at 1 s per point with 0.1 mm pathlength quartz cuvettes. Initial data processing was achieved with P<span class="font-variant-small-caps">ro</span>D<span class="font-variant-small-caps">ata</span> V<span class="font-variant-small-caps">iewer</span> software v:4.4.2.0, and secondary structure analysis was done with the BeStSel web server [<a href="#B35" class="usa-link" aria-describedby="B35">35</a>] for single spectrum and Protein Data Bank (PDB) ID analysis.</p></section><section id="s2-6"><h3 class="pmc_sec_title">2.6.  Mass spectrometry</h3>
<p>Proteomic studies were conducted at the Analytical Biochemistry and Proteomics Unit of the Institut Pasteur de Montevideo. Briefly, protein bands were excised from a Coomassie-stained gel, destained, reduced with 10 mM dithiothreitol (DTT), and alkylated with iodoacetamide, 50 mM. Subsequently, the proteins were digested overnight with trypsin. The resulting peptides were extracted, concentrated, desalted with Stage-Tips and resuspended in 0.1% formic acid. Samples were then injected into a liquid chromatography–mass spectrometry (LC‑MS) spectrometer (LTQ Velos+ETD, Thermo) and analyzed using the MASCOT search engine.</p></section><section id="s2-7"><h3 class="pmc_sec_title">2.7.  Mouse antiserum against rNcSRS2-ST and rNcSAG1-ST</h3>
<p>Two female BALB/c mice (six weeks old) were immunized intraperitoneally once with 100 μg of purified protein mixed with an equal volume of protein and complete Freund’s adjuvant (1 : 1 ratio). Then, they were immunized twice with purified protein and an equal volume of incomplete Freund’s adjuvant (1 : 1 ratio) at 15 day intervals. Mice were bled 15 days after the final immunization [<a href="#B27" class="usa-link" aria-describedby="B27">27</a>]. This protocol was performed for each of the proteins separately, as well as the antisera.</p></section><section id="s2-8"><h3 class="pmc_sec_title">2.8.  Western blot</h3>
<p><em>Neospora caninum</em> tachyzoite protein extracts from the Liverpool and Uru1 strains [<a href="#B36" class="usa-link" aria-describedby="B36">36</a>] or purified proteins were loaded into a 12% SDS-PAGE gel with equal loads and transferred to polyvinylidene fluoride (PVDF) membranes (Millipore, MA, USA) together with a visible prestained protein marker (PageRuler™ Prestained Protein Ladder, Thermo Scientific™) after electrophoresis [<a href="#B33" class="usa-link" aria-describedby="B33">33</a>]. The membranes were blocked with 5% (w/v) skim milk in PBS for 1 h at 37°C, rinsed with a washing buffer and incubated with anti-NcSAG1 and anti-NcSRS2 in 0.1% Tween-20 in PBS (PBST) for 1 h at 37°C (these antibodies were produced as described below). The blots were washed five times with PBST (0.1% Tween−20), followed by incubation with horseradish peroxidase (HRP)-labelled goat anti-IgG (H+L) (1 : 30 000, Sigma, USA). Finally, enhanced chemiluminescence reagents (SuperSignal™ West Pico PLUS Chemiluminescent Substrate, Thermo Scientific™) were used to observe the reaction bands after a 30 s exposure time.</p>
<p>Western blot analysis was also conducted to assess recombinant proteins' expression, size and integrity after induction, using Strep-Tactin-HRP (IBA) following the manufacturer’s recommendations. Briefly, 20 µl of culture supernatant collected after 48 h post-induction were loaded in reducing SDS-PAGE and transferred to a PVDF membrane, as described above. Membranes were blocked with 3% bovine serum albumin (BSA) and 0.05% Tween-20 in PBS for 2 h at 37°C, and after washing three times with 0.1% (v/v) Tween 20 in PBS, were incubated for 1 h with Strep-Tactin® HRP conjugate 1 : 10 000 and revealed by chemiluminescence as described above. We also evaluated whether antibodies from naturally infected animals could recognize the recombinant proteins. To this end, Western blotting was performed as described above, using pooled sera from dogs, cattle and deer that tested positive for <em>N. caninum</em> by commercial ELISA. Additionally, sera from mice experimentally infected with 1 × 10⁶ tachyzoites were included. All sera were used at a 1 : 100 dilution, and detection was carried out using protein G-HRP at a 1 : 10 000 dilution.</p></section><section id="s2-9"><h3 class="pmc_sec_title">2.9.  Structural modelling with A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span>
</h3>
<p>To generate three-dimensional structures from amino acid sequences, we used A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span> [<a href="#B37" class="usa-link" aria-describedby="B37">37</a>] in monomer mode, running individual predictions for each protein sequence independently. This approach was chosen to evaluate the intrinsic folding of each polypeptide chain. Our focus in this stage was on the tertiary structure of single chains, aiming to characterize sequence-dependent folding features and local structural motifs. The following parameters were applied during A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span> running: msa_mode: MMseqs2 (for multiple sequence alignments), num_models: 5 (five models per sequence), num_recycles: 5 (iterative structure refinement) and model_preset: full_dbs (using comprehensive databases). Custom bash scripts automated the process, managing job queuing, resource allocation and error handling to ensure robust execution. The 219 SRS protein sequences analysed were previously identified through a Protein BLAST analysis of the translated <em>N. caninum</em> genome [<a href="#B14" class="usa-link" aria-describedby="B14">14</a>]. To address A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span>’s tendency to model low-confidence regions as unstructured loops, we trimmed the amino and carboxyl termini of each protein, retaining the globular regions and the coil linker loops connecting them. These linker regions, while inherently unstructured, were included for their potential functional relevance. For the identification of structured regions and the selection of truncation sites, we used MKDSSP (version 3.0) to assign secondary structure elements to each A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span> model. The resulting annotations were then processed with custom scripts to systematically analyse the secondary structure content across the models. Based on this analysis, we trimmed poorly structured or disordered regions at the N- and C-termini. This approach ensured that only the well-folded, structurally relevant core of each model was retained for downstream analyses.</p></section><section id="s2-10"><h3 class="pmc_sec_title">2.10.  Structural comparison with pairwise structure alignment</h3>
<p>Once the protein structures of NcSAG1 and NcSRS2 were generated, we compared them with the structure of TgSAG1 (PDB ID: 1KZQ) using the web server of Pairwise Structure Alignment [<a href="#B38" class="usa-link" aria-describedby="B38">38</a>] with Template Modelling-alignment algorithm (Template Modelling (TM) score). The TM score ranges from 0 to 1; a score greater than 0.5 typically suggests that the two structures have the same fold or topology, meaning they are structurally similar.</p></section><section id="s2-11"><h3 class="pmc_sec_title">2.11.  Structural comparison with Distance Matrix Alignment</h3>
<p>To explore structural similarities and differences within the SRS protein family, we employed Distance Matrix Alignment (DALI) for a detailed structural analysis. Using this approach, we analysed 219 A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span> models, using DALI’s robust framework for three-dimensional protein structure comparison [<a href="#B39" class="usa-link" aria-describedby="B39">39</a>,<a href="#B40" class="usa-link" aria-describedby="B40">40</a>]. DALI conducts pairwise structural comparisons by assessing distance matrices to identify structurally equivalent residues. The method optimizes a scoring function that maximizes the weighted sum of similarities in intramolecular distances between equivalent atom pairs, enabling accurate structural alignment and comparison. We performed all-against-all structure comparison analyses using DALI. This analysis generated a structural similarity dendrogram by systematically comparing each protein structure against all others in the dataset. The resulting dendrogram visually illustrates the structural relationships within the SRS protein family, providing valuable insights into their evolutionary and functional connections.</p></section><section id="s2-12"><h3 class="pmc_sec_title">2.12.  Structural phylogeny</h3>
<p>To investigate the evolutionary relationships within the SRS protein family, we generated amino acid sequence alignments for the 219 sequences mentioned above. These alignments were focused on conserved regions and structural similarities, serving as the basis for subsequent phylogenetic and structural analyses. G<span class="font-variant-small-caps">blocks</span> [<a href="#B41" class="usa-link" aria-describedby="B41">41</a>] refine the alignment by excluding poorly aligned or ambiguous positions, enhancing the reliability of evolutionary inferences. A maximum likelihood (ML) phylogenetic tree was constructed using MEGA software [<a href="#B42" class="usa-link" aria-describedby="B42">42</a>]. The reconstruction employed the JTT+G4+I model of amino acid substitution, selected based on its statistical robustness and suitability for modelling the evolutionary dynamics of these proteins.</p></section><section id="s2-13"><h3 class="pmc_sec_title">2.13.  Visualization and analysis</h3>
<p>Using the similarity matrix obtained from DALI, we constructed a heatmap to visualize the structural clusters of the protein structures. This visualization was created using custom Python scripts, using libraries such as Matplotlib and Seaborn for graphical representation. The heatmap highlights groups of proteins with high structural similarity, facilitating the identification of potential functional or evolutionary relationships. Protein structures were visualized using VMD [<a href="#B43" class="usa-link" aria-describedby="B43">43</a>].</p></section><section id="s2-14"><h3 class="pmc_sec_title">2.14.  Ethics statement</h3>
<p>All animal experiments followed international animal care and use guidelines, respecting National Law 18.611 with pre-approved protocols by the Institutional Ethics Committee (CEUA no. 010-17) at the Institut Pasteur de Montevideo.</p></section></section><section id="s3"><h2 class="pmc_sec_title">3.  Results</h2>
<section id="s3-1"><h3 class="pmc_sec_title">3.1.  Sequence analysis and three-dimensional modelling</h3>
<p>In the literature, NcSAG1 and NcSRS2 are also called p36 (Genbank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/AJ005664.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AJ005664.1</a>) and p35/p43 (Genbank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/AF061249" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AF061249</a>), respectively. Both belong to the SRS protein family and comprise 321 (expected MW: 33,1 kDa) and 401 (expected MW: 38,1 kDa) amino acids, respectively. As members of this family, they share hallmark features, including an N-terminal signal peptide, a C-terminal GPI-anchor, D1 and D2 domains connected by a flexible linker, and six conserved cysteines within each domain (<a href="#F1" class="usa-link">figure 1A</a>). NcSAG1 and NcSRS2 exhibit low overall sequence identity at the amino acid level (electronic supplementary material, table S1). The D1 domains of the two proteins show the highest similarity, with 48% sequence identity, while the D2 domains are more divergent, with only 35% identity. Also, the sequence similarity between the D1 and D2 domains within each protein differs significantly: 24% for NcSAG1 and 38% for NcSRS2.</p>
<figure class="fig xbox font-sm" id="F1" title="AlphaFold modelling and disulfide bridges in SAG1 and SRS2 of N. caninum."><h4 class="obj_head">Figure 1. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7e33/12365351/d00df8821a18/rsos.250239.f001.jpg" loading="lazy" height="971" width="772" alt="AlphaFold modelling and disulfide bridges in SAG1 and SRS2 of N. caninum."></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span> modelling and disulfide bridges in SAG1 and SRS2 of <em>N. caninum</em>. (A) Schematic representation of the primary structure, the domains, and the disulfide bridges between cysteines 1 and 6. SP, signal peptide; L, linker; GPI is the anchor site for GPI. (B) A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span> model of NcSAG1 (left) and the disulfide bonds (right). (C) A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span> model of NcSRS2 (left) and the disulfide bonds (right). For A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span> modelling, the SP and GPI anchor regions were removed.</p></figcaption></figure><p>By using A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span> [<a href="#B37" class="usa-link" aria-describedby="B37">37</a>], we modelled both proteins, finding the expected D1 and D2 domains, the linker sequence and the conserved cysteines with the pairings 1−6, 2−5 and 3−4 (<a href="#F1" class="usa-link">figure 1B,C</a>). Despite sequence divergence, NcSAG1 and NcSRS2 exhibit significant structural similarity, with a TM score of 0.86 (electronic supplementary material, table S1). The comparison of the D1 domains exhibits a TM score of 0.83, indicating high structural conservation despite their low sequence similarity. By contrast, the D2 domains are more divergent, with a TM score of 0.58. The A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span> modelling predicted that NcSAG1 forms a dimer closely resembling the crystallized dimer of TgSAG1, a structural feature potentially relevant to their biological function (<a href="#F2" class="usa-link">figure 2A</a>). We then compared NcSAG1 to the crystallized structure of TgSAG1 through a pairwise structure alignment, obtaining a TM score of 0.61 (<a href="#F2" class="usa-link">figure 2B</a>). Although disulfide bridges in NcSAG1 and NcSRS2 differ in their relative positions to those in TgSAG1 (<a href="#F2" class="usa-link">figure 2C</a>), the overall order of disulfide bonding is conserved (<a href="#F2" class="usa-link">figure 2C</a>).</p>
<figure class="fig xbox font-sm" id="F2" title="Dimer modelling and structure comparison of NcSAG1 and TgSAG1."><h4 class="obj_head">Figure 2. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7e33/12365351/ba0d67895ac6/rsos.250239.f002.jpg" loading="lazy" height="618" width="602" alt="Dimer modelling and structure comparison of NcSAG1 and TgSAG1."></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Dimer modelling and structure comparison of NcSAG1 and TgSAG1. (A) A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span> model of the NcSAG1 dimer. (B) Structural comparison of NcSAG1 (yellow) with TgSAG1 (PDB 1KZQ; blue). (C) Comparison of disulfide bonds. On the right, the zoomed-in view shows an example of the cysteine residue location.</p></figcaption></figure></section><section id="s3-2"><h3 class="pmc_sec_title">3.2.  Recombinant Strep-tagged NcSAG1 and NcSRS2 are expressed as stable and soluble proteins</h3>
<p>Both proteins were expressed in S2 cells, as described above. To assess their presence in the soluble fraction, the culture supernatant was collected 48 hours after induction with CdCl<sub>2</sub> and subjected to AC purification. The expression of rNcSAG1-ST and rNcSRS2-ST was confirmed by Western blot analysis, targeting Strep-tagged proteins. Both proteins appeared as single bands on SDS-PAGE and Western blot, with apparent MWs of approximately 42 and 52 kDa (theoretical MWs 30 and 37 kDa), respectively (<a href="#F3" class="usa-link">figure 3A</a>). Purification by AC allowed us to elute total protein amounts ranging from 2 to 13 mg based on culture scales of 50–450 ml (electronic supplementary material, figure S1). Production yields were 22−65 mg l<sup>−1</sup> for rNcSAG1-ST and 37−85 mg l<sup>−1</sup> for rNcSRS2-ST. After affinity purification, both proteins were visible on Coomassie-stained SDS-PAGE (<a href="#F3" class="usa-link">figure 3B</a>). The identity of the bands was confirmed by mass spectrometry: the unique band for rNcSAG1-ST was identified as the p29 surface antigen of <em>N. caninum</em>, while both bands observed for rNcSRS2-ST were identified as Ncp35 (<a href="#F3" class="usa-link">figure 3B</a>; electronic supplementary material, table S2). Discrepancies between theoretical and estimated MWs are expected and attributed to post-translational modifications.</p>
<figure class="fig xbox font-sm" id="F3" title="Expression, purification and identification of rNcSAG1-ST and rNcSSR2-ST."><h4 class="obj_head">Figure 3. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365351_rsos.250239.f003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7e33/12365351/f23c7d8210a3/rsos.250239.f003.jpg" loading="lazy" height="568" width="765" alt="Expression, purification and identification of rNcSAG1-ST and rNcSSR2-ST."></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Expression, purification and identification of rNcSAG1-ST and rNcSSR2-ST. (A) The plasmid construction used for protein expression in the <em>D. melanogaste</em>r system. Protein expressions were visualized by Western blot in the medium of expression using the Strep-tag. (B) SDS-PAGE analysis of the purification process, where lanes 1 and 4 are total medium fractions, 2 and 5 are the flow-through fractions, and 3 and 6 correspond to the elution fractions from affinity chromatography. Bands stained with Coomassie blue were analysed by mass spectroscopy, identifying the proteins listed on the right.</p></figcaption></figure></section><section id="s3-3"><h3 class="pmc_sec_title">3.3.  Antigenicity and immunogenicity of rNcSAG1-ST and rNcSRS2-ST</h3>
<p>To evaluate the antigenicity of rNcSAG1-ST and rNcSRS2-ST, we used serum pools from mice, cows, dogs and deer that were positive for neosporosis. Except for the sera from mice, which were derived from experimental infections, all originated from naturally infected animals. They all recognized both proteins, as demonstrated by Western blot (<a href="#F4" class="usa-link">figure 4A</a>). This suggests that they serve as universal antigens, being specifically identified by serum samples from different host species, regardless of whether they were naturally or experimentally infected.</p>
<figure class="fig xbox font-sm" id="F4" title="Antigenicity and immunogenicity of rNcSAG1-ST and rNcSRS2-ST."><h4 class="obj_head">Figure 4. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365351_rsos.250239.f004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7e33/12365351/cf079b8eeadb/rsos.250239.f004.jpg" loading="lazy" height="776" width="656" alt="Antigenicity and immunogenicity of rNcSAG1-ST and rNcSRS2-ST."></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Antigenicity and immunogenicity of rNcSAG1-ST and rNcSRS2-ST. (A) Western blot with antisera from mice immunized with rNcSAG1-ST and rNcSRS2-ST. Lanes 1 and 2 correspond to protein extracts of <em>N. caninum</em> from <em>N. caninum</em> URU1 and Liverpool strains, respectively. The immune sera recognize native proteins in <em>N. caninum</em> extracts. Black arrows indicate the bands corresponding to NcSAG1 and NcSRS2. (B) Western blot analysis of recombinant proteins detected by sera from various infected hosts, including experimentally infected mice and naturally infected cows, dogs and deer.</p></figcaption></figure><p>To assess their immunogenicity, we separated total <em>N. caninum</em> protein extracts by SDS-PAGE and analysed them via Western blotting using rNcSAG1-ST and rNcSRS2-ST antisera. This approach revealed bands of the expected sizes, with no observed cross-reaction between the antisera (<a href="#F4" class="usa-link">figure 4B</a>). The size differences between recombinant and native proteins were anticipated owing to the addition of biotin in recombinant proteins and variations in post-translational modification profiles.</p></section><section id="s3-4"><h3 class="pmc_sec_title">3.4.  Stable and dimeric conformation of soluble proteins</h3>
<p>The inclusion of an extra purification step by SEC enabled us to obtain several fractions with different oligomerization states. Within each purification batch, we isolated both proteins as a main peak preceded by high MW aggregates (<a href="#F5" class="usa-link">figure 5A,B</a>). When these main fractions were isolated and re-injected in analytical SEC, rNcSAG1-ST and rNcSRS2-ST eluted as unique peaks with estimated MWs of 105.5 and 76.6 kDa, suggesting that both proteins are obtained as stable dimers (<a href="#F5" class="usa-link">figure 5C,D</a>). The thermostability of rNcSAG1-ST and rNcSRS2-ST was assessed by nanoDSF, showing thermal unfolding with <em>T</em><sub>M</sub> above 50°C (<em>T</em><sub>M</sub> = 58.6°C for rNcSAG1-ST and <em>T</em><sub>M</sub> = 53.1°C for rNcSRS2-ST). Additionally, both proteins showed reversibility with no signs of aggregation upon heating to 90°C (<a href="#F5" class="usa-link">figure 5E,F</a>). Both dimeric rNcSAG1-ST and rNcSRS2-ST were purified by preparative SEC and treated overnight with bovine enterokinase to remove the Strep-tag. Subsequently, cleaved proteins (rNcSAG1* and rNcSRS2*) were re-injected in a preparative SEC, showing a unique peak with a slight shift in exclusion volume towards smaller sizes, as a consequence of the Strep-tag removal (electronic supplementary material, figure S2). The main peak was collected and concentrated 10-fold to obtain batches of cleaved versions of dimeric proteins at 1 mg ml<sup>−1</sup>. Batches of both dimeric rNcSAG1* and rNcSRS2* were analysed by DLS to evaluate their behaviour in solution after Strep-tag removal and sample concentration. Both proteins showed size distributions by intensity, containing a main population with an average hydrodynamic radius (<em>R</em><sub>H</sub>) of 3.40 nm for rNcSAG1* and 3.53 nm for rNcSRS2*, and high-sized aggregates, which are negligible, as observed in size distributions by volume (<a href="#F6" class="usa-link">figure 6</a>).</p>
<figure class="fig xbox font-sm" id="F5" title="Biochemical and physicochemical characterization of rNcSAG1-ST and rNcSRS2-ST."><h4 class="obj_head">Figure 5. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7e33/12365351/8711c76fbf6c/rsos.250239.f005.jpg" loading="lazy" height="639" width="692" alt="Biochemical and physicochemical characterization of rNcSAG1-ST and rNcSRS2-ST."></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Biochemical and physicochemical characterization of rNcSAG1-ST and rNcSRS2-ST. (A and B) Preparative size exclusion chromatography (SEC) of rNCSAG1-ST and rNcSRS2-ST, respectively, with the main collected peak shadowed in red. (C and D) Analytical SEC of the main fractions of rNcSAG1-ST and rNcSRS2-ST obtained in (A) and (B), respectively. The molecular weight (MW) calibration curve is included with values in grey, and the estimated MW of each protein is shown in kilodaltons (kDa) next to each peak. (E) Nano differential scanning fluorimetry showing changes in intrinsic fluorescence of rNcSAG1-ST (red) and rNcSRS2-ST (green) upon heating and cooling (shown in bright and pale colours, respectively, accompanied by grey arrows indicating the direction of each transition). (F) The first derivative of the curves obtained in (E), shows the melting temperature (<em>T</em><sub>M</sub>) as the peak of each transition at a temperature of Celsius degrees. ST means that the protein contains a C-terminal double Strep-tag.</p></figcaption></figure><figure class="fig xbox font-sm" id="F6" title="Evaluation of rNcSAG1* and rNcSRS2* behaviour in solution."><h4 class="obj_head">Figure 6. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7e33/12365351/67acae488469/rsos.250239.f006.jpg" loading="lazy" height="482" width="672" alt="Evaluation of rNcSAG1* and rNcSRS2* behaviour in solution."></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Evaluation of rNcSAG1* and rNcSRS2* behaviour in solution. Hydrodynamic radius (<em>R</em><sub>H</sub>) of rNcSAG1* and rNcSRS2*, respectively, measured by DLS after treatment with bovine enterokinase to remove the Strep-tag. Distributions calculated by intensity (upper panels) show a main peak at 3.53 and 3.40 nm, respectively, accompanied by high molecular weight aggregates, which are not detected in distributions measured by volume (lower panels). Asterisk suffixes represent that the Strep-tag was removed.</p></figcaption></figure></section><section id="s3-5"><h3 class="pmc_sec_title">3.5.  Circular dichroism</h3>
<p>Far-UV CD spectra of rNcSAG1* and rNcSRS2* were found to be very similar, suggesting that both proteins have a comparable secondary structure composition. Deconvolution of spectra using BeStSel allowed us to identify a secondary structure content of approximately 30% β-strands (mainly relaxed and right-twisted antiparallel β-strands) followed by 14%–15.3% of turns and 8.2%–5.3% of α-helices (mainly distorted α-helices) as seen in <a href="#F7" class="usa-link">figure 7</a>. To validate our structural interpretation of CD spectra using BeStSel, we carried out an orthogonal approach based on the PDB structures of related proteins such as TgSAG1, TgBSR4 and TgSporoSAG from <em>T. gondii</em>, obtaining quite similar secondary structure compositions for all of them (electronic supplementary material, figure S3).</p>
<figure class="fig xbox font-sm" id="F7" title="Secondary structural analysis by circular dichroism."><h4 class="obj_head">Figure 7. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7e33/12365351/7eab256d410b/rsos.250239.f007.jpg" loading="lazy" height="540" width="748" alt="Secondary structural analysis by circular dichroism."></p>
<div class="p text-right font-secondary"><a href="figure/F7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Secondary structural analysis by circular dichroism. (A and B) Spectral data and deconvolution interpretation for rNcSAG1* and rNcSRS2*, respectively.</p></figcaption></figure></section><section id="s3-6"><h3 class="pmc_sec_title">3.6.  Comparative structural analysis</h3>
<p>From a protein BLAST analysis of the translated <em>N. caninum</em> genome, 219 sequences belonging to the SRS family were selected (electronic supplementary material, table S3; [<a href="#B14" class="usa-link" aria-describedby="B14">14</a>]). These sequences retain key SAG domain features, such as conserved cysteines, prolines and the signature tryptophan. To compare these proteins, we employed A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span> for structure prediction and DALI for structural comparison, enabling a systematic exploration of their structural features and relationships. DALI generated a similarity matrix for the 219 predicted protein structures, quantified by <em>Z</em>-scores that indicate the degree of structural similarity between protein pairs. Higher <em>Z</em>-scores signify greater similarity. The triangular heatmap (<a href="#F8" class="usa-link">figure 8A</a>) illustrates this matrix, with yellow indicating high similarity and purple denoting divergence. Hierarchical clustering along the axes groups proteins with similar structural characteristics, revealing distinct clusters within the SRS family. Representative structures from key clusters are highlighted, illustrating conserved and variable motifs. Some clusters belonging to previously biologically characterized proteins are marked in red, blue, orange and purple, which include Ncaninum_LIV_000104000 (an orthologue of SporoSAG), Ncaninum_LIV_000061800 (NcSAG1), Ncaninum_LIV_000678300 (NcBSR4) and Ncaninum_LIV_000533300 (NcSAG4), respectively (<a href="#F8" class="usa-link">figure 8A,B</a>).</p>
<figure class="fig xbox font-sm" id="F8" title="Comprehensive analysis of structural similarity through heatmap and dendrogram."><h4 class="obj_head">Figure 8. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365351_rsos.250239.f008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7e33/12365351/d2e1ded70160/rsos.250239.f008.jpg" loading="lazy" height="376" width="719" alt="Comprehensive analysis of structural similarity through heatmap and dendrogram."></a></p>
<div class="p text-right font-secondary"><a href="figure/F8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comprehensive analysis of structural similarity through heatmap and dendrogram. (A) The heatmap visualizes the structural similarity among proteins using a colour gradient that ranges from blue to yellow, indicating low to high similarity, respectively. This heatmap displays the lower triangular region of the similarity matrix. Along the axes of the heatmap, a dendrogram is plotted to illustrate the hierarchical clustering of these structures. Key structures of interest relevant to this study are distinctly highlighted: Ncaninum000533300 (SAG4) is marked in purple, Ncaninum000104000 (an orthologue of SporoSAG) in red, Ncaninum000061800 (SAG1) in blue and Ncaninum000678300 (BSR4) in yellow. (B) The star-shaped dendrogram, derived from the structural similarity matrix, effectively depicts the structural relationships and diversity among the analysed proteins. Each clade within the dendrogram is represented by a unique, randomly selected structure, coloured in silver, to exemplify the variation and commonalities within the clusters. This representation underscores the breadth of structural diversity and aids in identifying significant similarities and differences within the dataset. The relevant protein structures highlighted in this study are coloured as in panel (A).</p></figcaption></figure><p>To evaluate the robustness of our methodology, we compared the ML phylogenetic tree, based on sequence data processed with G<span class="font-variant-small-caps">blocks</span>, with the structure-based tree derived from DALI (electronic supplementary material, figure S5). Both trees highlighted the major clusters, demonstrating a strong correlation between sequence- and structure-based phylogenetic analyses. While congruences reinforced the reliability of our approach, some discrepancies between sequence and structural evolution suggest instances of structural divergence not mirrored in sequence data. These findings underscore the complementarity of sequence- and structure-based methodologies, offering a comprehensive understanding of the SRS protein family in <em>N. caninum</em>.</p>
<p>These results establish a framework for further investigation into the roles and interactions of SRS proteins, bridging structural diversity and evolutionary patterns to their biological significance.</p></section></section><section id="s4"><h2 class="pmc_sec_title">4.  Discussion</h2>
<p>This study reports the successful expression, purification and characterization of NcSAG1 and NcSRS2 proteins. The <em>D. melanogaster</em> S2 expression system was chosen since it has proven advantageous, enabling efficient production of secreted proteins while simplifying purification and scalability [<a href="#B34" class="usa-link" aria-describedby="B34">34</a>,<a href="#B44" class="usa-link" aria-describedby="B44">44</a>]. In addition, it could address challenges often encountered with prokaryotic systems, including insolubility and low expression levels, and preserve the disulfide bridges, which play a crucial role in the structural integrity of these proteins, as described for <em>T. gondii</em> SRS proteins [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>–<a href="#B19" class="usa-link" aria-describedby="B19">19</a>]. Moreover, since they are glycoproteins, incorporating post-translational modifications is necessary. It should be noted that the glycosylation pattern in S2 cells may not fully replicate that of <em>N. caninum</em>, which could influence epitope recognition and modulate the immune response. Both rNcSAG1-ST and rNcSRS2-ST were expressed as soluble proteins, with production yields comparable to the most efficient prokaryotic systems, making the S2 system highly recommended for expressing proteins of eukaryotic origin. The stable and soluble production of rNcSAG1-ST and rNcSRS2-ST enabled us to complement the structural studies with physicochemical characterization, as discussed later.</p>
<p>Both proteins exhibit hallmark features of the SRS family, including an N-terminal signal peptide, a C-terminal GPI-anchor, D1 and D2 domains connected by a flexible linker, and 12 conserved cysteines forming characteristic disulfide bonds. Despite their low overall sequence identity, NcSAG1 and NcSRS2 share significant structural similarities, as evidenced by their TM scores of 0.86 for full-length proteins and 0.83 for the D1 domains. These findings suggest that structural conservation may be critical in maintaining their functional integrity. The D1 domains, which exhibit higher sequence identity than the D2 domains, may contribute to a conserved core function, whereas the more divergent D2 domains could support specialized or distinct roles. The A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span>-predicted structures highlight the importance of disulfide bonding patterns in stabilizing the architecture of these proteins. While the relative positions of disulfide bridges differ between NcSAG1, NcSRS2 and the structurally resolved TgSAG1, the overall disulfide bonding order remains conserved. The more stable predicted structures are dimeric, and its resemblance to the crystallized dimer of TgSAG1 suggests that this dimeric arrangement may be essential for biological functions such as host cell attachment or immune evasion, as previously reported for other SRS proteins. This result is supported here by experimental assays, as discussed below. The moderate TM score (0.61) obtained from the structural comparison between NcSAG1 and TgSAG1 underscores both conserved and divergent aspects of their structures, potentially reflecting adaptations to different host environments or immune pressures.</p>
<p>The information obtained from structure prediction was confirmed through biochemical and biophysical studies. In the first purification step, both proteins were eluted with a main peak consistent with the formation of a dimer. Including an additional SEC step significantly enhanced the purity and resolution of oligomeric states, enabling the isolation of rNcSAG1-ST and rNcSRS2-ST as stable dimers. This stability is further corroborated by the high <em>T</em><sub>M</sub> observed in nanoDSF assays (58.6°C for rNcSAG1-ST and 53.1°C for rNcSRS2-ST), which are notable for recombinant proteins. The ability of these proteins to refold after heating to 90°C without aggregation suggests robust structural integrity, making them suitable for diverse experimental studies. We can suppose that the dimer is the functional unit in both cases, with direct implications for future studies on potential ligands and inhibitors. In addition, the thermostability may reflect evolutionary adaptations that ensure functionality in varying environments, a feature worth exploring in the context of their native biological roles.</p>
<p>The far-UV CD analysis of rNcSAG1* and rNcSRS2* provides valuable insights into their secondary structure and strengthens the robustness of the A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span> modelling. The close agreement between the CD-based secondary structure estimates for rNcSAG1* and rNcSRS2* and those derived from TgSAG1, TgBSR4 and TgSporoSAG suggests a conserved architectural motif within this protein family. This conservation underscores the evolutionary pressure to maintain structural elements essential for function, such as ligand binding or host-pathogen interactions. The predominance of relaxed and right-twisted antiparallel β-strands indicates a structural preference that may be key to their stability and interactions with other molecules. Additionally, the presence of distorted α-helices and turns reflects structural versatility, which could play a role in facilitating dynamic conformational changes required for their functions. These results suggest that the β-strands constitute the backbone of this protein family, providing stability and conservation, enhanced by the disulfide bridges between the six highly conserved cysteines for each domain; meanwhile, the α-helices and turns act as sources of variability under evolutionary pressure, enabling functional and structural diversity.</p>
<p>The biochemical and physicochemical analyses suggest that the recombinant proteins are of high quality and suitable for developing advanced diagnostic techniques and vaccines. Proteins that accurately mimic native folding and post-translational modifications are expected to enhance immune system recognition, which is critical for effective diagnostic assays and vaccine development. Previous vaccine development efforts have used protein fragments or insoluble forms, limiting antigenic recognition to specific sequences [<a href="#B26" class="usa-link" aria-describedby="B26">26</a>,<a href="#B28" class="usa-link" aria-describedby="B28">28</a>,<a href="#B45" class="usa-link" aria-describedby="B45">45</a>]. The absence of NcSAG1 reduces parasite infectivity, suggesting that antibody-mediated neutralization of this protein could significantly decrease infective capacity. However, redundancy within the protein family may necessitate the inclusion of multiple antigens for full inhibition [<a href="#B30" class="usa-link" aria-describedby="B30">30</a>]. A recent study with recombinant NcSAG1 expressed in <em>Escherichia coli</em> demonstrated protective efficacy in cows, eliciting strong humoral responses and activation of peripheral blood mononuclear cells (PBMCs) [<a href="#B46" class="usa-link" aria-describedby="B46">46</a>]. Incorporating NcSAG1 and NcSRS2 in forms closer to their native structures could further enhance these results. In this sense, we generated murine antisera against both recombinant proteins, and they successfully recognized <em>N. caninum</em> lysates, including reference and local strains. Additionally, sera from infected animals recognized these proteins, confirming specificity. The absence of cross-reactivity among antibodies targeting different recombinant proteins also highlights their diagnostic potential. Combining multiple antigens could improve serological test sensitivity and specificity [<a href="#B47" class="usa-link" aria-describedby="B47">47</a>].</p>
<p>The strong correlation between structural modelling and CD spectroscopy data enabled the construction of the first structure-based phylogenetic tree for the SRS protein family. This integrative analysis, based on A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span>-generated monomeric models and DALI-based structural comparisons, provides a comprehensive overview of the family’s structural landscape and evolutionary relationships. It is important to emphasize that all computational analyses, including structure prediction, clustering and tree reconstruction, were conducted using monomeric forms of the proteins, and the results should be interpreted as structural hypotheses rather than definitive conclusions.</p>
<p>Our results revealed distinct structural clades that align with known sequence-based groupings and support hypotheses regarding potential functional specialization. In <em>T. gondii</em>, SRS proteins cluster into two well-characterized groups (SAG1 and SAG2A) [<a href="#B15" class="usa-link" aria-describedby="B15">15</a>,<a href="#B48" class="usa-link" aria-describedby="B48">48</a>]; a similar pattern is observed in <em>N. caninum</em>, where NcSAG1 and NcSRS2 form a distinct subclade within the SAG1-like group, possibly reflecting species-specific adaptations. Unlike previous studies reporting stage-specific clustering for <em>T. gondii</em>, our structural tree does not segregate proteins by life stage, suggesting that structural similarity might relate more to functional features than to temporal expression, though this remains a working hypothesis.</p>
<p>The subset of proteins expressed in this study falls within the same clade and groups coherently with sequence-based classifications. Secondary structure content derived from CD spectroscopy exhibited strong agreement with the structural features predicted by A<span class="font-variant-small-caps">lpha</span>F<span class="font-variant-small-caps">old</span>, supporting the validity of the models.</p>
<p>Representative structures shown in <a href="#F8" class="usa-link">figure 8A</a> span from approximately 180–420 amino acids in length. Shorter models typically feature a single β-sandwich domain, which may correspond to naturally occurring isoforms or truncations potentially derived from alternative splicing. The presence of a single domain raises the possibility that this structural unit alone could fulfill essential biological functions—an intriguing hypothesis that warrants future experimental validation. Additionally, several models display extended α-helices projecting from the core fold. Based on their orientation and hydrophobic character, these helices may represent membrane-anchoring regions or transmembrane domains. While detailed biophysical or topological analyses were beyond the scope of this work, these features merit further investigation to fully elucidate their functional relevance.</p>
<p>In conclusion, our results and generated tools provide a foundation for further exploration of these proteins' biological roles. The successful expression of soluble, stable proteins positions this system as a promising method for producing antigens for both diagnostic and vaccine applications, offering new avenues for improving the management and understanding of neosporosis.</p></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Soledad Echeverría, Email: echeverria@pasteur.edu.uy.</p>
<p>Federico Carrión, Email: fcarrion@pasteur.edu.uy.</p>
<p>Martín Soñora, Email: msonora@pasteur.edu.uy.</p>
<p>Andrés Cabrera, Email: cabrera@pasteur.edu.uy.</p>
<p>Carlos Robello, Email: robello@pasteur.edu.uy.</p></section><section id="s5"><h2 class="pmc_sec_title">Ethics</h2>
<p>All animal experiments followed international animal care and use guidelines, respecting National Law 18.611 with pre-approved protocols by the Institutional Ethics Committee (CEUA no. 010-17) at the Institut Pasteur de Montevideo (Uruguay).</p></section><section id="s6"><h2 class="pmc_sec_title">Data accessibility</h2>
<p>Data can be accessed online [<a href="#B49" class="usa-link" aria-describedby="B49">49</a>].</p>
<p>Supplementary material available online [<a href="#B50" class="usa-link" aria-describedby="B50">50</a>].</p></section><section id="s7"><h2 class="pmc_sec_title">Declaration of AI use</h2>
<p>We have not used AI-assisted technologies in creating this article.</p></section><section id="s8"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>S.E.: investigation, methodology, writing—original draft; F.C.: investigation, methodology, writing—original draft, writing—review and editing; M.S.: investigation, methodology, writing—review and editing; A.C.: conceptualization, investigation, supervision, writing—original draft, writing—review and editing; C.R.: conceptualization, funding acquisition, investigation, project administration, supervision, writing—review and editing.</p>
<p>All authors gave final approval for publication and agreed to be held accountable for the work performed therein.</p></section><section id="s9"><h2 class="pmc_sec_title">Conflict of interest declaration</h2>
<p>We declare we have no competing interests.</p></section><section id="s10"><h2 class="pmc_sec_title">Funding</h2>
<p>This project was funded by grant FSSA_1_2019_1_160691 from the Uruguayan National Agency for Research and Innovation (ANII), awarded to C.R. S.E. was supported by a postgraduate fellowship from ANII and CSIC (Universidad de la República, Uruguay). A.C., M.S., and C.R. are researchers from PEDECIBA (Programa de Desarrollo de las Ciencias Básicas, Uruguay) and the Sistema Nacional de Investigadores (SNI-ANII, Uruguay).</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>
Dubey JP, Carpenter JL, Speer CA, Topper MJ, Uggla A. 1988. 
Newly recognized fatal protozoan disease of dogs. J. Am. Vet. Med. Assoc.
192, 1269–1285.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/3391851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Vet.%20Med.%20Assoc.&amp;title=Newly%20recognized%20fatal%20protozoan%20disease%20of%20dogs&amp;volume=192&amp;publication_year=1988&amp;pages=1269-1285&amp;pmid=3391851&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>
Dubey JP, Schares G. 2011. 
Neosporosis in animals—the last five years. Vet. Parasitol.
180, 90–108. ( 10.1016/j.vetpar.2011.05.031)
</cite> [<a href="https://doi.org/10.1016/j.vetpar.2011.05.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21704458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vet.%20Parasitol.&amp;title=Neosporosis%20in%20animals%E2%80%94the%20last%20five%20years&amp;volume=180&amp;publication_year=2011&amp;pages=90-108&amp;pmid=21704458&amp;doi=10.1016/j.vetpar.2011.05.031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>
Dubey JP, Schares G, Ortega-Mora LM. 2007. 
Epidemiology and control of neosporosis and Neospora caninum. Clin. Microbiol. Rev.
20, 323–367. ( 10.1128/CMR.00031-06)
</cite> [<a href="https://doi.org/10.1128/CMR.00031-06" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1865591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17428888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Microbiol.%20Rev.&amp;title=Epidemiology%20and%20control%20of%20neosporosis%20and%20Neospora%20caninum&amp;volume=20&amp;publication_year=2007&amp;pages=323-367&amp;pmid=17428888&amp;doi=10.1128/CMR.00031-06&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>
McAllister MM, Dubey JP, Lindsay DS, Jolley WR, Wills RA, McGuire AM. 1998. 
Dogs are definitive hosts of Neospora caninum. Int. J. Parasitol.
28, 1473–1478.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/9770635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Parasitol.&amp;title=Dogs%20are%20definitive%20hosts%20of%20Neospora%20caninum&amp;volume=28&amp;publication_year=1998&amp;pages=1473-1478&amp;pmid=9770635&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>
Goodswen SJ, Kennedy PJ, Ellis JT. 2013. 
A review of the infection, genetics, and evolution of Neospora caninum: from the past to the present. Infect. Genet. Evol.
13, 133–150. ( 10.1016/j.meegid.2012.08.012)
</cite> [<a href="https://doi.org/10.1016/j.meegid.2012.08.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22985682/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Genet.%20Evol.&amp;title=A%20review%20of%20the%20infection,%20genetics,%20and%20evolution%20of%20Neospora%20caninum:%20from%20the%20past%20to%20the%20present&amp;volume=13&amp;publication_year=2013&amp;pages=133-150&amp;pmid=22985682&amp;doi=10.1016/j.meegid.2012.08.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>
Barber JS, Trees AJ. 1996. 
Clinical aspects of 27 cases of neosporosis in dogs. Vet. Rec.
139, 439–443. ( 10.1136/vr.139.18.439)
</cite> [<a href="https://doi.org/10.1136/vr.139.18.439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8931299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vet.%20Rec.&amp;title=Clinical%20aspects%20of%2027%20cases%20of%20neosporosis%20in%20dogs&amp;volume=139&amp;publication_year=1996&amp;pages=439-443&amp;pmid=8931299&amp;doi=10.1136/vr.139.18.439&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>
Cavalcante GT, Monteiro RM, Soares RM, Nishi SM, Alves Neto AF, Esmerini P de O, Sercundes MK, Martins J, Gennari SM. 2011. 
Shedding of Neospora caninum oocysts by dogs fed different tissues from naturally infected cattle. Vet. Parasitol.
179, 220–223. ( 10.1016/j.vetpar.2011.02.026)
</cite> [<a href="https://doi.org/10.1016/j.vetpar.2011.02.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7131750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21450407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vet.%20Parasitol.&amp;title=Shedding%20of%20Neospora%20caninum%20oocysts%20by%20dogs%20fed%20different%20tissues%20from%20naturally%20infected%20cattle&amp;volume=179&amp;publication_year=2011&amp;pages=220-223&amp;pmid=21450407&amp;doi=10.1016/j.vetpar.2011.02.026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>
Reichel MP, Alejandra Ayanegui-Alcérreca M, Gondim LFP, Ellis JT. 2013. 
What is the global economic impact of Neospora caninum in cattle – the billion dollar question. Int. J. Parasitol.
43, 133–142. ( 10.1016/j.ijpara.2012.10.022)
</cite> [<a href="https://doi.org/10.1016/j.ijpara.2012.10.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23246675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Parasitol.&amp;title=What%20is%20the%20global%20economic%20impact%20of%20Neospora%20caninum%20in%20cattle%20%E2%80%93%20the%20billion%20dollar%20question&amp;volume=43&amp;publication_year=2013&amp;pages=133-142&amp;pmid=23246675&amp;doi=10.1016/j.ijpara.2012.10.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>
Wang P, et al. 2022. 
Evaluation of protective immune responses induced in BALB/c mice and goats by the Neospora caninum surface SRS proteins and interleukin-18. Animals
12, 2952. ( 10.3390/ani12212952)
</cite> [<a href="https://doi.org/10.3390/ani12212952" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9658137/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36359077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Animals&amp;title=Evaluation%20of%20protective%20immune%20responses%20induced%20in%20BALB/c%20mice%20and%20goats%20by%20the%20Neospora%20caninum%20surface%20SRS%20proteins%20and%20interleukin-18&amp;volume=12&amp;publication_year=2022&amp;pages=2952&amp;pmid=36359077&amp;doi=10.3390/ani12212952&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>
Manger ID, Hehl AB, Boothroyd JC. 1998. 
The surface of Toxoplasma tachyzoites is dominated by a family of glycosylphosphatidylinositol-anchored antigens related to SAG1. Infect. Immun.
66, 2237–2244. ( 10.1128/iai.66.5.2237-2244.1998)
</cite> [<a href="https://doi.org/10.1128/iai.66.5.2237-2244.1998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC108187/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9573113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&amp;title=The%20surface%20of%20Toxoplasma%20tachyzoites%20is%20dominated%20by%20a%20family%20of%20glycosylphosphatidylinositol-anchored%20antigens%20related%20to%20SAG1&amp;volume=66&amp;publication_year=1998&amp;pages=2237-2244&amp;pmid=9573113&amp;doi=10.1128/iai.66.5.2237-2244.1998&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>
Sibley LD, Pfefferkorn ER, Boothroyd JC. 1991. 
Proposal for a uniform genetic nomenclature in Toxoplasma gondii. Parasitol. Today
7, 327–328. ( 10.1016/0169-4758(91)90210-f)
</cite> [<a href="https://doi.org/10.1016/0169-4758(91)90210-f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15463406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Parasitol.%20Today&amp;title=Proposal%20for%20a%20uniform%20genetic%20nomenclature%20in%20Toxoplasma%20gondii&amp;volume=7&amp;publication_year=1991&amp;pages=327-328&amp;pmid=15463406&amp;doi=10.1016/0169-4758(91)90210-f&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>
Fuchs N, Sonda S, Bütikofer P, Hemphill A. 1999. 
Detection of surface-associated and intracellular glycoconjugates and glycoproteins in Neospora caninum tachyzoites. Int. J. Parasitol.
29, 1597–1611. ( 10.1016/s0020-7519(99)00118-6)
</cite> [<a href="https://doi.org/10.1016/s0020-7519(99)00118-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10608447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Parasitol.&amp;title=Detection%20of%20surface-associated%20and%20intracellular%20glycoconjugates%20and%20glycoproteins%20in%20Neospora%20caninum%20tachyzoites&amp;volume=29&amp;publication_year=1999&amp;pages=1597-1611&amp;pmid=10608447&amp;doi=10.1016/s0020-7519(99)00118-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>
Reid AJ, et al. 2012. 
Comparative genomics of the apicomplexan parasites Toxoplasma gondii and Neospora caninum: coccidia differing in host range and transmission strategy. PLoS Pathog.
8, e1002567. ( 10.1371/journal.ppat.1002567)
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1002567" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3310773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22457617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&amp;title=Comparative%20genomics%20of%20the%20apicomplexan%20parasites%20Toxoplasma%20gondii%20and%20Neospora%20caninum:%20coccidia%20differing%20in%20host%20range%20and%20transmission%20strategy&amp;volume=8&amp;publication_year=2012&amp;pages=e1002567&amp;pmid=22457617&amp;doi=10.1371/journal.ppat.1002567&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B14">
<span class="label">14.</span><cite>
Berná L, Marquez P, Cabrera A, Greif G, Francia ME, Robello C. 2021. 
Reevaluation of the Toxoplasma gondii and Neospora caninum genomes reveals misassembly, karyotype differences, and chromosomal rearrangements. Genome Res.
31, 823–833. ( 10.1101/gr.262832.120)
</cite> [<a href="https://doi.org/10.1101/gr.262832.120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8092007/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33906964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Reevaluation%20of%20the%20Toxoplasma%20gondii%20and%20Neospora%20caninum%20genomes%20reveals%20misassembly,%20karyotype%20differences,%20and%20chromosomal%20rearrangements&amp;volume=31&amp;publication_year=2021&amp;pages=823-833&amp;pmid=33906964&amp;doi=10.1101/gr.262832.120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B15">
<span class="label">15.</span><cite>
Jung C, Lee CYF, Grigg ME. 2004. 
The SRS superfamily of Toxoplasma surface proteins. Int. J. Parasitol.
34, 285–296. ( 10.1016/j.ijpara.2003.12.004)
</cite> [<a href="https://doi.org/10.1016/j.ijpara.2003.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15003490/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Parasitol.&amp;title=The%20SRS%20superfamily%20of%20Toxoplasma%20surface%20proteins&amp;volume=34&amp;publication_year=2004&amp;pages=285-296&amp;pmid=15003490&amp;doi=10.1016/j.ijpara.2003.12.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B16">
<span class="label">16.</span><cite>
Bezerra MA, Pereira LM, Baroni L, Yatsuda AP. 2019. 
The soluble fraction of Neospora caninum treated with PI-PLC is dominated by NcSRS29B and NcSRS29C. Exp. Parasitol.
204, 107731. ( 10.1016/j.exppara.2019.107731)
</cite> [<a href="https://doi.org/10.1016/j.exppara.2019.107731" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31374185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp.%20Parasitol.&amp;title=The%20soluble%20fraction%20of%20Neospora%20caninum%20treated%20with%20PI-PLC%20is%20dominated%20by%20NcSRS29B%20and%20NcSRS29C&amp;volume=204&amp;publication_year=2019&amp;pages=107731&amp;pmid=31374185&amp;doi=10.1016/j.exppara.2019.107731&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B17">
<span class="label">17.</span><cite>
Crawford J, Grujic O, Bruic E, Czjzek M, Grigg ME, Boulanger MJ. 2009. 
Structural characterization of the bradyzoite surface antigen (BSR4) from Toxoplasma gondii, a unique addition to the surface antigen glycoprotein 1-related superfamily. J. Biol. Chem.
284, 9192–9198. ( 10.1074/jbc.m808714200)
</cite> [<a href="https://doi.org/10.1074/jbc.m808714200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2666571/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19155215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Structural%20characterization%20of%20the%20bradyzoite%20surface%20antigen%20(BSR4)%20from%20Toxoplasma%20gondii,%20a%20unique%20addition%20to%20the%20surface%20antigen%20glycoprotein%201-related%20superfamily&amp;volume=284&amp;publication_year=2009&amp;pages=9192-9198&amp;pmid=19155215&amp;doi=10.1074/jbc.m808714200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B18">
<span class="label">18.</span><cite>
Crawford J, Lamb E, Wasmuth J, Grujic O, Grigg ME, Boulanger MJ. 2010. 
Structural and functional characterization of SporoSAG. J. Biol. Chem.
285, 12063–12070. ( 10.1074/jbc.m109.054866)
</cite> [<a href="https://doi.org/10.1074/jbc.m109.054866" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2852944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20164173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Structural%20and%20functional%20characterization%20of%20SporoSAG&amp;volume=285&amp;publication_year=2010&amp;pages=12063-12070&amp;pmid=20164173&amp;doi=10.1074/jbc.m109.054866&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B19">
<span class="label">19.</span><cite>
He X lin, Grigg ME, Boothroyd JC, Garcia KC. 2002. 
Structure of the immunodominant surface antigen from the Toxoplasma gondii SRS superfamily. Nat. Struct. Biol.
9, 606–611. ( 10.1038/nsb819)
</cite> [<a href="https://doi.org/10.1038/nsb819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12091874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Struct.%20Biol.&amp;title=Structure%20of%20the%20immunodominant%20surface%20antigen%20from%20the%20Toxoplasma%20gondii%20SRS%20superfamily&amp;volume=9&amp;publication_year=2002&amp;pages=606-611&amp;pmid=12091874&amp;doi=10.1038/nsb819&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B20">
<span class="label">20.</span><cite>
Kim SK, Boothroyd JC. 2005. 
Stage-specific expression of surface antigens by Toxoplasma gondii as a mechanism to facilitate parasite persistence. J. Immunol.
174, 8038–8048. ( 10.4049/jimmunol.174.12.8038)
</cite> [<a href="https://doi.org/10.4049/jimmunol.174.12.8038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15944311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.&amp;title=Stage-specific%20expression%20of%20surface%20antigens%20by%20Toxoplasma%20gondii%20as%20a%20mechanism%20to%20facilitate%20parasite%20persistence&amp;volume=174&amp;publication_year=2005&amp;pages=8038-8048&amp;pmid=15944311&amp;doi=10.4049/jimmunol.174.12.8038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B21">
<span class="label">21.</span><cite>
Horcajo P, Xia D, Randle N, Collantes-Fernández E, Wastling J, Ortega-Mora LM, Regidor-Cerrillo J. 2018. 
Integrative transcriptome and proteome analyses define marked differences between Neospora caninum isolates throughout the tachyzoite lytic cycle. J. Proteom.
180, 108–119. ( 10.1016/j.jprot.2017.11.007)</cite> [<a href="https://doi.org/10.1016/j.jprot.2017.11.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29154927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteom.&amp;title=Integrative%20transcriptome%20and%20proteome%20analyses%20define%20marked%20differences%20between%20Neospora%20caninum%20isolates%20throughout%20the%20tachyzoite%20lytic%20cycle&amp;volume=180&amp;publication_year=2018&amp;pages=108-119&amp;pmid=29154927&amp;doi=10.1016/j.jprot.2017.11.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B22">
<span class="label">22.</span><cite>
Rico-San Román L, Horcajo P, Regidor-Cerrillo J, Fernández-Escobar M, Collantes-Fernández E, Gutiérrez-Blázquez D, Hernáez-Sánchez ML, Saeij JPJ, Ortega-Mora LM. 2020. 
Comparative tachyzoite proteome analyses among six Neospora caninum isolates with different virulence. Int. J. Parasitol.
50, 377–388. ( 10.1016/j.ijpara.2020.02.003)
</cite> [<a href="https://doi.org/10.1016/j.ijpara.2020.02.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32360428/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Parasitol.&amp;title=Comparative%20tachyzoite%20proteome%20analyses%20among%20six%20Neospora%20caninum%20isolates%20with%20different%20virulence&amp;volume=50&amp;publication_year=2020&amp;pages=377-388&amp;pmid=32360428&amp;doi=10.1016/j.ijpara.2020.02.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B23">
<span class="label">23.</span><cite>
Risco-Castillo V, Fernández-García A, Ortega-Mora LM. 2004. 
Comparative analysis of stress agents in a simplified in vitro system of Neospora caninum bradyzoite production. J. Parasitol.
90, 466–470. ( 10.1645/GE-3298)
</cite> [<a href="https://doi.org/10.1645/GE-3298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15270086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Parasitol.&amp;title=Comparative%20analysis%20of%20stress%20agents%20in%20a%20simplified%20in%20vitro%20system%20of%20Neospora%20caninum%20bradyzoite%20production&amp;volume=90&amp;publication_year=2004&amp;pages=466-470&amp;pmid=15270086&amp;doi=10.1645/GE-3298&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B24">
<span class="label">24.</span><cite>
Naguleswaran A, Müller N, Hemphill A. 2003. 
Neospora caninum and Toxoplasma gondii: a novel adhesion/invasion assay reveals distinct differences in tachyzoite–host cell interactions. Exp. Parasitol
104, 149–158. ( 10.1016/s0014-4894(03)00137-1)
</cite> [<a href="https://doi.org/10.1016/s0014-4894(03)00137-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14552862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp.%20Parasitol&amp;title=Neospora%20caninum%20and%20Toxoplasma%20gondii:%20a%20novel%20adhesion/invasion%20assay%20reveals%20distinct%20differences%20in%20tachyzoite%E2%80%93host%20cell%20interactions&amp;volume=104&amp;publication_year=2003&amp;pages=149-158&amp;pmid=14552862&amp;doi=10.1016/s0014-4894(03)00137-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B25">
<span class="label">25.</span><cite>
Sonda S, Fuchs N, Connolly B, Fernandez P, Gottstein B, Hemphill A. 1998. 
The major 36 kDa Neospora caninum tachyzoite surface protein is closely related to the major Toxoplasma gondii surface antigen. Mol. Biochem. Parasitol.
97, 97–108. ( 10.1016/s0166-6851(98)00133-9)
</cite> [<a href="https://doi.org/10.1016/s0166-6851(98)00133-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9879890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biochem.%20Parasitol.&amp;title=The%20major%2036%20kDa%20Neospora%20caninum%20tachyzoite%20surface%20protein%20is%20closely%20related%20to%20the%20major%20Toxoplasma%20gondii%20surface%20antigen&amp;volume=97&amp;publication_year=1998&amp;pages=97-108&amp;pmid=9879890&amp;doi=10.1016/s0166-6851(98)00133-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B26">
<span class="label">26.</span><cite>
Cho JH, Chung WS, Song KJ, Na BK, Kang SW, Song CY, Kim TS. 2005. 
Protective efficacy of vaccination with Neospora caninum multiple recombinant antigens against experimental Neospora caninum infection. Korean J. Parasitol.
43, 19. ( 10.3347/kjp.2005.43.1.19)
</cite> [<a href="https://doi.org/10.3347/kjp.2005.43.1.19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2712005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15793355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Korean%20J.%20Parasitol.&amp;title=Protective%20efficacy%20of%20vaccination%20with%20Neospora%20caninum%20multiple%20recombinant%20antigens%20against%20experimental%20Neospora%20caninum%20infection&amp;volume=43&amp;publication_year=2005&amp;pages=19&amp;pmid=15793355&amp;doi=10.3347/kjp.2005.43.1.19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B27">
<span class="label">27.</span><cite>
Nishikawa Y, Xuan X, Nagasawa H, Igarashi I, Fujisaki K, Otsuka H, Mikami T. 2000. 
Monoclonal antibody inhibition of Neospora caninum tachyzoite invasion into host cells. Int. J. Parasitol.
30, 51–58. ( 10.1016/s0020-7519(99)00162-9)
</cite> [<a href="https://doi.org/10.1016/s0020-7519(99)00162-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10675744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Parasitol.&amp;title=Monoclonal%20antibody%20inhibition%20of%20Neospora%20caninum%20tachyzoite%20invasion%20into%20host%20cells&amp;volume=30&amp;publication_year=2000&amp;pages=51-58&amp;pmid=10675744&amp;doi=10.1016/s0020-7519(99)00162-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B28">
<span class="label">28.</span><cite>
Haldorson GJ, Mathison BA, Wenberg K, Conrad PA, Dubey JP, Trees AJ, Yamane I, Baszler TV. 2005. 
Immunization with native surface protein NcSRS2 induces a Th2 immune response and reduces congenital Neospora caninum transmission in mice. Int. J. Parasitol.
35, 1407–1415. ( 10.1016/j.ijpara.2005.05.013)
</cite> [<a href="https://doi.org/10.1016/j.ijpara.2005.05.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16051244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Parasitol.&amp;title=Immunization%20with%20native%20surface%20protein%20NcSRS2%20induces%20a%20Th2%20immune%20response%20and%20reduces%20congenital%20Neospora%20caninum%20transmission%20in%20mice&amp;volume=35&amp;publication_year=2005&amp;pages=1407-1415&amp;pmid=16051244&amp;doi=10.1016/j.ijpara.2005.05.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B29">
<span class="label">29.</span><cite>
Son ES, Ahn HJ, Kim JH, Kim DY, Nam HW. 2001. 
Determination of antigenic domain in GST fused major surface protein (Nc-p43) of Neospora caninum. Korean J. Parasitol.
39, 241–246. ( 10.3347/kjp.2001.39.3.241)
</cite> [<a href="https://doi.org/10.3347/kjp.2001.39.3.241" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2721073/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11590914/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Korean%20J.%20Parasitol.&amp;title=Determination%20of%20antigenic%20domain%20in%20GST%20fused%20major%20surface%20protein%20(Nc-p43)%20of%20Neospora%20caninum&amp;volume=39&amp;publication_year=2001&amp;pages=241-246&amp;pmid=11590914&amp;doi=10.3347/kjp.2001.39.3.241&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B30">
<span class="label">30.</span><cite>
Abdelbaky HH, Rahman MdM, Shimoda N, Chen Y, Hasan T, Ushio N, Nishikawa Y. 2023. 
Neospora caninum surface antigen 1 is a major determinant of the pathogenesis of neosporosis in nonpregnant and pregnant mice. Front. Microbiol.
14, 1334447. ( 10.3389/fmicb.2023.1334447)
</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1334447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10800813/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38260884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Neospora%20caninum%20surface%20antigen%201%20is%20a%20major%20determinant%20of%20the%20pathogenesis%20of%20neosporosis%20in%20nonpregnant%20and%20pregnant%20mice&amp;volume=14&amp;publication_year=2023&amp;pages=1334447&amp;pmid=38260884&amp;doi=10.3389/fmicb.2023.1334447&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B31">
<span class="label">31.</span><cite>
Hemphill A, Vonlaufen N, Naguleswaran A. 2006. 
Cellular and immunological basis of the host-parasite relationship during infection with Neospora caninum. Parasitology
133, 261–278. ( 10.1017/S0031182006000485)
</cite> [<a href="https://doi.org/10.1017/S0031182006000485" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16753081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Parasitology&amp;title=Cellular%20and%20immunological%20basis%20of%20the%20host-parasite%20relationship%20during%20infection%20with%20Neospora%20caninum&amp;volume=133&amp;publication_year=2006&amp;pages=261-278&amp;pmid=16753081&amp;doi=10.1017/S0031182006000485&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B32">
<span class="label">32.</span><cite>
Alves Sinnott F, da Silva Leal K, de Oliveira Silva MT, Barros de Pinho R, Pappen F, da Rosa Farias NA, Bedoya Llano HA, Pereira Garcia Melo D, Borsuk S. 2020. 
An indirect ELISA for neosporosis: associating recombinant Neospora caninum proteins NcSRS2 and NcSAG1. Vet. Parasitol.
281, 109101. ( 10.1016/j.vetpar.2020.109101)
</cite> [<a href="https://doi.org/10.1016/j.vetpar.2020.109101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32302944/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vet.%20Parasitol.&amp;title=An%20indirect%20ELISA%20for%20neosporosis:%20associating%20recombinant%20Neospora%20caninum%20proteins%20NcSRS2%20and%20NcSAG1&amp;volume=281&amp;publication_year=2020&amp;pages=109101&amp;pmid=32302944&amp;doi=10.1016/j.vetpar.2020.109101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B33">
<span class="label">33.</span><cite>
Wilkowsky SE, Bareiro GG, Mon ML, Moore DP, Caspe G, Campero C, Fort M, Romano MI. 2011. 
An applied printing immunoassay with recombinant Nc-SAG1 for detection of antibodies to Neospora caninum in cattle. J. Vet. Diagn. Invest.
23, 971–976. ( 10.1177/1040638711416845)
</cite> [<a href="https://doi.org/10.1177/1040638711416845" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21908357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Vet.%20Diagn.%20Invest.&amp;title=An%20applied%20printing%20immunoassay%20with%20recombinant%20Nc-SAG1%20for%20detection%20of%20antibodies%20to%20Neospora%20caninum%20in%20cattle&amp;volume=23&amp;publication_year=2011&amp;pages=971-976&amp;pmid=21908357&amp;doi=10.1177/1040638711416845&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B34">
<span class="label">34.</span><cite>
Carrión F, Rammauro F, Olivero‐Deibe N, Fló M, Portela MM, Lima A, Durán R, Pritsch O, Bianchi S. 2023. 
Soluble SARS‐CoV‐2 RBD and human ACE2 peptidase domain produced in Drosophila S2 cells show functions evoking virus–cell interface. Protein Sci.
32, e4721. ( 10.1002/pro.4721)
</cite> [<a href="https://doi.org/10.1002/pro.4721" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10382795/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37405395/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&amp;title=Soluble%20SARS%E2%80%90CoV%E2%80%902%20RBD%20and%20human%20ACE2%20peptidase%20domain%20produced%20in%20Drosophila%20S2%20cells%20show%20functions%20evoking%20virus%E2%80%93cell%20interface&amp;volume=32&amp;publication_year=2023&amp;pages=e4721&amp;pmid=37405395&amp;doi=10.1002/pro.4721&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B35">
<span class="label">35.</span><cite>
Micsonai A, et al. 2022. 
BeStSel: webserver for secondary structure and fold prediction for protein CD spectroscopy. Nucleic Acids Res.
50, W90–W98. ( 10.1093/nar/gkac345)
</cite> [<a href="https://doi.org/10.1093/nar/gkac345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9252784/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35544232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=BeStSel:%20webserver%20for%20secondary%20structure%20and%20fold%20prediction%20for%20protein%20CD%20spectroscopy&amp;volume=50&amp;publication_year=2022&amp;pages=W90-W98&amp;pmid=35544232&amp;doi=10.1093/nar/gkac345&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B36">
<span class="label">36.</span><cite>
Cabrera A, et al. 2019. 
Isolation and molecular characterization of four novel Neospora caninum strains. Parasitol. Res.
118, 3535–3542. ( 10.1007/s00436-019-06474-9)
</cite> [<a href="https://doi.org/10.1007/s00436-019-06474-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31701296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Parasitol.%20Res.&amp;title=Isolation%20and%20molecular%20characterization%20of%20four%20novel%20Neospora%20caninum%20strains&amp;volume=118&amp;publication_year=2019&amp;pages=3535-3542&amp;pmid=31701296&amp;doi=10.1007/s00436-019-06474-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B37">
<span class="label">37.</span><cite>
Jumper J, et al. 2021. 
Highly accurate protein structure prediction with AlphaFold. Nature
596, 583–589. ( 10.1038/s41586-021-03819-2)
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&amp;volume=596&amp;publication_year=2021&amp;pages=583-589&amp;pmid=34265844&amp;doi=10.1038/s41586-021-03819-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B38">
<span class="label">38.</span><cite>
Bittrich S, Segura J, Duarte JM, Burley SK, Rose Y. 2024. 
RCSB protein Data Bank: exploring protein 3D similarities via comprehensive structural alignments. Bioinformatics
40, e370. ( 10.1093/bioinformatics/btae370)</cite> [<a href="https://doi.org/10.1093/bioinformatics/btae370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11212067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38870521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=RCSB%20protein%20Data%20Bank:%20exploring%20protein%203D%20similarities%20via%20comprehensive%20structural%20alignments&amp;volume=40&amp;publication_year=2024&amp;pages=e370&amp;pmid=38870521&amp;doi=10.1093/bioinformatics/btae370&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B39">
<span class="label">39.</span><cite>
Holm L. 2020. 
DALI and the persistence of protein shape. Protein Sci.
29, 128–140. ( 10.1002/pro.3749)
</cite> [<a href="https://doi.org/10.1002/pro.3749" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6933842/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31606894/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&amp;title=DALI%20and%20the%20persistence%20of%20protein%20shape&amp;volume=29&amp;publication_year=2020&amp;pages=128-140&amp;pmid=31606894&amp;doi=10.1002/pro.3749&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B40">
<span class="label">40.</span><cite>
Holm L, Laiho A, Törönen P, Salgado M. 2023. 
DALI shines a light on remote homologs: one hundred discoveries. Protein Sci.
32, e4519. ( 10.1002/pro.4519)
</cite> [<a href="https://doi.org/10.1002/pro.4519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9793968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36419248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&amp;title=DALI%20shines%20a%20light%20on%20remote%20homologs:%20one%20hundred%20discoveries&amp;volume=32&amp;publication_year=2023&amp;pages=e4519&amp;pmid=36419248&amp;doi=10.1002/pro.4519&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B41">
<span class="label">41.</span><cite>
Talavera G, Castresana J. 2007. 
Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments. Syst. Biol.
56, 564–577. ( 10.1080/10635150701472164)
</cite> [<a href="https://doi.org/10.1080/10635150701472164" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17654362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Syst.%20Biol.&amp;title=Improvement%20of%20phylogenies%20after%20removing%20divergent%20and%20ambiguously%20aligned%20blocks%20from%20protein%20sequence%20alignments&amp;volume=56&amp;publication_year=2007&amp;pages=564-577&amp;pmid=17654362&amp;doi=10.1080/10635150701472164&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B42">
<span class="label">42.</span><cite>
Tamura K, Stecher G, Kumar S. 2021. 
MEGA11: molecular evolutionary genetics analysis version 11. Mol. Biol. Evol.
38, 3022–3027. ( 10.1093/molbev/msab120)
</cite> [<a href="https://doi.org/10.1093/molbev/msab120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8233496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33892491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&amp;title=MEGA11:%20molecular%20evolutionary%20genetics%20analysis%20version%2011&amp;volume=38&amp;publication_year=2021&amp;pages=3022-3027&amp;pmid=33892491&amp;doi=10.1093/molbev/msab120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B43">
<span class="label">43.</span><cite>
Humphrey W, Dalke A, Schulten K. 1996. 
VMD: visual molecular dynamics. J. Mol. Graph.
14, 33–38. ( 10.1016/0263-7855(96)00018-5)
</cite> [<a href="https://doi.org/10.1016/0263-7855(96)00018-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8744570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Graph.&amp;title=VMD:%20visual%20molecular%20dynamics&amp;volume=14&amp;publication_year=1996&amp;pages=33-38&amp;pmid=8744570&amp;doi=10.1016/0263-7855(96)00018-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B44">
<span class="label">44.</span><cite>
Tomé-Poderti L, et al. 2024. 
Characterization and application of recombinant bovine leukemia virus Env protein. Sci. Rep.
14, 12190. ( 10.1038/s41598-024-62811-8)
</cite> [<a href="https://doi.org/10.1038/s41598-024-62811-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11133380/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38806566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Characterization%20and%20application%20of%20recombinant%20bovine%20leukemia%20virus%20Env%20protein&amp;volume=14&amp;publication_year=2024&amp;pages=12190&amp;pmid=38806566&amp;doi=10.1038/s41598-024-62811-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B45">
<span class="label">45.</span><cite>
Pownall WR, Imhof D, Trigo NF, Ganal-Vonarburg SC, Plattet P, Monney C, Forterre F, Hemphill A, Oevermann A. 2021. 
Safety of a novel Listeria monocytogenes-based vaccine vector expressing NcSAG1 (Neospora caninum surface antigen 1). Front. Cell. Infect. Microbiol.
11, 675219. ( 10.3389/fcimb.2021.675219)
</cite> [<a href="https://doi.org/10.3389/fcimb.2021.675219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8506043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34650932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Cell.%20Infect.%20Microbiol.&amp;title=Safety%20of%20a%20novel%20Listeria%20monocytogenes-based%20vaccine%20vector%20expressing%20NcSAG1%20(Neospora%20caninum%20surface%20antigen%201)&amp;volume=11&amp;publication_year=2021&amp;pages=675219&amp;pmid=34650932&amp;doi=10.3389/fcimb.2021.675219&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B46">
<span class="label">46.</span><cite>
Mendoza-Morales LF, et al. 2024. 
Immunogenicity, safety and dual DIVA-like character of a recombinant candidate vaccine against neosporosis in cattle. Acta Trop.
257, 107293. ( 10.1016/j.actatropica.2024.107293)
</cite> [<a href="https://doi.org/10.1016/j.actatropica.2024.107293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38901525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Trop.&amp;title=Immunogenicity,%20safety%20and%20dual%20DIVA-like%20character%20of%20a%20recombinant%20candidate%20vaccine%20against%20neosporosis%20in%20cattle&amp;volume=257&amp;publication_year=2024&amp;pages=107293&amp;pmid=38901525&amp;doi=10.1016/j.actatropica.2024.107293&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B47">
<span class="label">47.</span><cite>
Gondim LFP, Mineo JR, Schares G. 2017. 
Importance of serological cross-reactivity among Toxoplasma gondii, Hammondia spp., Neospora spp., Sarcocystis spp. and Besnoitia besnoiti. Parasitology
144, 851–868. ( 10.1017/s0031182017000063)
</cite> [<a href="https://doi.org/10.1017/s0031182017000063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5471829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28241894/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Parasitology&amp;title=Importance%20of%20serological%20cross-reactivity%20among%20Toxoplasma%20gondii,%20Hammondia%20spp.,%20Neospora%20spp.,%20Sarcocystis%20spp.%20and%20Besnoitia%20besnoiti&amp;volume=144&amp;publication_year=2017&amp;pages=851-868&amp;pmid=28241894&amp;doi=10.1017/s0031182017000063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B48">
<span class="label">48.</span><cite>
Lekutis C, Ferguson DJP, Grigg ME, Camps M, Boothroyd JC. 2001. 
Surface antigens of Toxoplasma gondii: variations on a theme. Int. J. Parasitol.
31, 1285–1292. ( 10.1016/s0020-7519(01)00261-2)
</cite> [<a href="https://doi.org/10.1016/s0020-7519(01)00261-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11566296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Parasitol.&amp;title=Surface%20antigens%20of%20Toxoplasma%20gondii:%20variations%20on%20a%20theme&amp;volume=31&amp;publication_year=2001&amp;pages=1285-1292&amp;pmid=11566296&amp;doi=10.1016/s0020-7519(01)00261-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B49">
<span class="label">49.</span><cite>
Echeverria S, Carrión F, Soñora Met al. 2025. Molecular Characterization of Neospora Caninum Major Antigens NcSAG1 and NcSRS2. Dryad Digital Repository. ( 10.5061/dryad.44j0zpcrh)</cite> [<a href="https://doi.org/10.5061/dryad.44j0zpcrh" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="B50">
<span class="label">50.</span><cite>
Echeverria S, Carrión F, Soñora M, Cabrera A, Robello C. 2025. 
Supplementary material from: Molecular Characterization of Neospora caninum Major Antigens NcSAG1 and NcSRS2. Figshare. ( 10.6084/m9.figshare.c.7948755)</cite> [<a href="https://doi.org/10.6084/m9.figshare.c.7948755" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data can be accessed online [<a href="#B49" class="usa-link" aria-describedby="B49">49</a>].</p>
<p>Supplementary material available online [<a href="#B50" class="usa-link" aria-describedby="B50">50</a>].</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Royal Society Open Science are provided here courtesy of <strong>The Royal Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1098/rsos.250239"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/rsos.250239.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365351/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365351/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365351%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365351/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365351/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365351/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40843189/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365351/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40843189/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365351/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365351/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="FfpWGvuZt2bcDdXEcbRTrww5QQ14yPsR66uhdIDQKTppNpiTuKEhxBt117vWIS8n">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
