
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Conserved function of the HAUS6 calponin homology domain in anchoring augmin for microtubule branching - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A4338AF21F7305A43300146388F1.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373997/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Conserved function of the HAUS6 calponin homology domain in anchoring augmin for microtubule branching">
<meta name="citation_author" content="Martin Würtz">
<meta name="citation_author_institution" content="Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany">
<meta name="citation_author_institution" content="European Molecular Biology Laboratory (EMBL), Heidelberg, Germany">
<meta name="citation_author" content="Giulia Tonon">
<meta name="citation_author_institution" content="Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany">
<meta name="citation_author_institution" content="Present Address: Department of Molecular Sociology, Max Planck Institute of Biophysics, Frankfurt, Germany">
<meta name="citation_author" content="Bram J A Vermeulen">
<meta name="citation_author_institution" content="Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany">
<meta name="citation_author" content="Maja Zezlina">
<meta name="citation_author_institution" content="Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany">
<meta name="citation_author" content="Qi Gao">
<meta name="citation_author_institution" content="Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany">
<meta name="citation_author" content="Annett Neuner">
<meta name="citation_author_institution" content="Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany">
<meta name="citation_author" content="Angelika Seidl">
<meta name="citation_author_institution" content="Zoological Institute, Karlsruhe Institute of Technology (KIT), Karlsruhe, Germany">
<meta name="citation_author" content="Melanie König">
<meta name="citation_author_institution" content="Biochemie-Zentrum der Universität Heidelberg (BZH), Heidelberg, Germany">
<meta name="citation_author" content="Maximilian Harkenthal">
<meta name="citation_author_institution" content="Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany">
<meta name="citation_author" content="Sebastian Eustermann">
<meta name="citation_author_institution" content="European Molecular Biology Laboratory (EMBL), Heidelberg, Germany">
<meta name="citation_author" content="Sylvia Erhardt">
<meta name="citation_author_institution" content="Zoological Institute, Karlsruhe Institute of Technology (KIT), Karlsruhe, Germany">
<meta name="citation_author_institution" content="Institute of Biological and Chemical Systems - Functional Molecular Systems (IBACS-FMS), Karlsruhe Institute of Technology (KIT), Karlsruhe, Germany">
<meta name="citation_author" content="Fabio Lolicato">
<meta name="citation_author_institution" content="Department of Physics, University of Helsinki, Helsinki, Finland">
<meta name="citation_author_institution" content="Present Address: Biochemie-Zentrum der Universität Heidelberg (BZH), Heidelberg, Germany">
<meta name="citation_author" content="Elmar Schiebel">
<meta name="citation_author_institution" content="Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany">
<meta name="citation_author" content="Stefan Pfeffer">
<meta name="citation_author_institution" content="Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7845">
<meta name="citation_doi" content="10.1038/s41467-025-63165-z">
<meta name="citation_pmid" content="40846850">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373997/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373997/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373997/pdf/41467_2025_Article_63165.pdf">
<meta name="description" content="Branching microtubule nucleation is a key mechanism for mitotic and meiotic spindle assembly and requires the hetero-octameric augmin complex. Augmin recruits the major microtubule nucleator, the γ-tubulin ring complex, to pre-existing microtubules ...">
<meta name="og:title" content="Conserved function of the HAUS6 calponin homology domain in anchoring augmin for microtubule branching">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Branching microtubule nucleation is a key mechanism for mitotic and meiotic spindle assembly and requires the hetero-octameric augmin complex. Augmin recruits the major microtubule nucleator, the γ-tubulin ring complex, to pre-existing microtubules ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373997/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373997">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63165-z"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63165.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373997%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373997/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373997/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373997/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 22;16:7845. doi: <a href="https://doi.org/10.1038/s41467-025-63165-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63165-z</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Conserved function of the HAUS6 calponin homology domain in anchoring augmin for microtubule branching</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22W%C3%BCrtz%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Martin Würtz</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Martin Würtz</span></h3>
<div class="p">
<sup>1</sup>Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany </div>
<div class="p">
<sup>2</sup>European Molecular Biology Laboratory (EMBL), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22W%C3%BCrtz%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Würtz</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tonon%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Giulia Tonon</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Giulia Tonon</span></h3>
<div class="p">
<sup>1</sup>Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany </div>
<div class="p">
<sup>7</sup>Present Address: Department of Molecular Sociology, Max Planck Institute of Biophysics, Frankfurt, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tonon%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Giulia Tonon</span></a>
</div>
</div>
<sup>1,</sup><sup>7,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vermeulen%20BJA%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Bram J A Vermeulen</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Bram J A Vermeulen</span></h3>
<div class="p">
<sup>1</sup>Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vermeulen%20BJA%22%5BAuthor%5D" class="usa-link"><span class="name western">Bram J A Vermeulen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zezlina%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Maja Zezlina</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Maja Zezlina</span></h3>
<div class="p">
<sup>1</sup>Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zezlina%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maja Zezlina</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Qi Gao</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Qi Gao</span></h3>
<div class="p">
<sup>1</sup>Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qi Gao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Neuner%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Annett Neuner</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Annett Neuner</span></h3>
<div class="p">
<sup>1</sup>Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Neuner%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Annett Neuner</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Seidl%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Angelika Seidl</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Angelika Seidl</span></h3>
<div class="p">
<sup>3</sup>Zoological Institute, Karlsruhe Institute of Technology (KIT), Karlsruhe, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Seidl%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Angelika Seidl</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22K%C3%B6nig%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Melanie König</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Melanie König</span></h3>
<div class="p">
<sup>4</sup>Biochemie-Zentrum der Universität Heidelberg (BZH), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22K%C3%B6nig%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Melanie König</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Harkenthal%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Maximilian Harkenthal</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Maximilian Harkenthal</span></h3>
<div class="p">
<sup>1</sup>Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Harkenthal%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maximilian Harkenthal</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Eustermann%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Sebastian Eustermann</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Sebastian Eustermann</span></h3>
<div class="p">
<sup>2</sup>European Molecular Biology Laboratory (EMBL), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Eustermann%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sebastian Eustermann</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Erhardt%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Sylvia Erhardt</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Sylvia Erhardt</span></h3>
<div class="p">
<sup>3</sup>Zoological Institute, Karlsruhe Institute of Technology (KIT), Karlsruhe, Germany </div>
<div class="p">
<sup>5</sup>Institute of Biological and Chemical Systems - Functional Molecular Systems (IBACS-FMS), Karlsruhe Institute of Technology (KIT), Karlsruhe, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Erhardt%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sylvia Erhardt</span></a>
</div>
</div>
<sup>3,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lolicato%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Fabio Lolicato</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Fabio Lolicato</span></h3>
<div class="p">
<sup>6</sup>Department of Physics, University of Helsinki, Helsinki, Finland </div>
<div class="p">
<sup>8</sup>Present Address: Biochemie-Zentrum der Universität Heidelberg (BZH), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lolicato%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fabio Lolicato</span></a>
</div>
</div>
<sup>6,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schiebel%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Elmar Schiebel</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Elmar Schiebel</span></h3>
<div class="p">
<sup>1</sup>Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schiebel%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Elmar Schiebel</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pfeffer%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Stefan Pfeffer</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Stefan Pfeffer</span></h3>
<div class="p">
<sup>1</sup>Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pfeffer%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Stefan Pfeffer</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Zentrum für Molekulare Biologie der Universität Heidelberg (ZMBH), Heidelberg, Germany </div>
<div id="Aff2">
<sup>2</sup>European Molecular Biology Laboratory (EMBL), Heidelberg, Germany </div>
<div id="Aff3">
<sup>3</sup>Zoological Institute, Karlsruhe Institute of Technology (KIT), Karlsruhe, Germany </div>
<div id="Aff4">
<sup>4</sup>Biochemie-Zentrum der Universität Heidelberg (BZH), Heidelberg, Germany </div>
<div id="Aff5">
<sup>5</sup>Institute of Biological and Chemical Systems - Functional Molecular Systems (IBACS-FMS), Karlsruhe Institute of Technology (KIT), Karlsruhe, Germany </div>
<div id="Aff6">
<sup>6</sup>Department of Physics, University of Helsinki, Helsinki, Finland </div>
<div id="Aff7">
<sup>7</sup>Present Address: Department of Molecular Sociology, Max Planck Institute of Biophysics, Frankfurt, Germany </div>
<div id="Aff8">
<sup>8</sup>Present Address: Biochemie-Zentrum der Universität Heidelberg (BZH), Heidelberg, Germany </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 4; Accepted 2025 Aug 12; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373997  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846850/" class="usa-link">40846850</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Branching microtubule nucleation is a key mechanism for mitotic and meiotic spindle assembly and requires the hetero-octameric augmin complex. Augmin recruits the major microtubule nucleator, the γ-tubulin ring complex, to pre-existing microtubules to direct the formation of new microtubules in a defined orientation. Although recent structural work has provided key insights into the structural organization of augmin, molecular details of its interaction with microtubules remain elusive. Here, we identify the minimal conserved microtubule-binding unit of augmin across species and demonstrate that stable microtubule anchoring is predominantly mediated via the calponin homology (CH) domain in Dgt6/HAUS6. Comparative sequence and functional analyses in vitro and in vivo reveal a highly conserved functional role of the HAUS6 CH domain in microtubule binding. Using cryo-electron microscopy and molecular dynamics simulations in combination with AlphaFold structure predictions, we show that the <em>D. melanogaster</em> Dgt6/HAUS6 CH domain binds microtubules at the inter-protofilament groove between two adjacent β-tubulin subunits and thereby orients augmin on microtubules. Altogether, our findings reveal how augmin binds microtubules to pre-determine the branching angle during microtubule nucleation and facilitate the rapid assembly of complex microtubule networks.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Cryoelectron microscopy, Microtubules</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Augmin is essential for microtubule nucleation during cell division. This study reveals how the conserved calponin homology domain of the augmin subunit HAUS6 binds to microtubules to orient augmin for microtubule branching.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Microtubules (MTs) are fundamental components of the eukaryotic cytoskeleton, playing crucial roles in diverse cellular processes such as compartmentalization, chromosome segregation, cell motility, and intracellular transport. At the onset of division, cells assemble a dense and polarized spindle composed of interconnected MT networks, including pole-to-pole, pole-to-kinetochore, and branched MTs, to ensure the precise segregation of the genetic material<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. The MT branching mechanism is governed by the coordinated interplay of many different components, most notably the γ-tubulin ring complex (γ-TuRC), the primary MT nucleator<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, and the augmin complex, which recruits the γ-TuRC to pre-existing MTs<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Through this bridging function, the augmin complex determines the orientation and polarity of newly formed MTs<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> and therefore is of utmost importance for the rapid amplification of spindle MTs, as well as for establishing connections with kinetochore fibers<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Consistent with these central functions, loss of augmin results in mitotic and meiotic spindle assembly defects, chromosome missegregation, and ultimately cell lethality<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Beyond its spindle-associated roles, augmin is essential for non-centrosomal MT networks in neurons, where it supports axonal MT organization, influences MT polarity, and plays an important role in central nervous system development<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>–<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>.</p>
<p id="Par4">Recent structural studies shed light on the molecular architecture of the augmin complex<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> revealing that its previously reported h-shaped structure<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> is formed by two tetrameric subcomplexes (Fig. <a href="#Fig1" class="usa-link">1a</a>): the rod-like TIII subcomplex (composed of HAUS1, HAUS3, HAUS4, and HAUS5) and the clamp-like TII subcomplex (composed of HAUS2, HAUS6, HAUS7, and HAUS8). These two subcomplexes serve complementary functions, providing the binding sites for the γ-TuRC (TIII) and the pre-existing MT (TII), respectively<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> (Fig. <a href="#Fig1" class="usa-link">1a</a>). While the interaction between TIII and the γ-TuRC remains elusive, the N-terminal half of TII, termed N-clamp, was proposed to function as the MT binding site<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. In particular, the unstructured positively charged N-terminal extension of the TII subunit HAUS8 (HAUS8-NT) was suggested to bind the negatively charged MT surface, thereby serving as an essential MT-binding element of human augmin<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Yet, interaction data suggest that additional parts of TII are required to establish a stable and effective interaction<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, and the defined angle at which MT branching occurs is challenging to reconcile with a MT binding site formed purely by the disordered HAUS8-NT.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. Augmin N-clamp is a structurally conserved minimal MT-binding unit.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373997_41467_2025_63165_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a79/12373997/297bbfaddbbf/41467_2025_63165_Fig1_HTML.jpg" loading="lazy" id="d33e508" height="990" width="696" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic representation of the augmin holocomplex (left)<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> and a cartoon illustrating the augmin N-clamp architecture (right, <em>H. sapiens</em>). Coloring as indicated. <strong>b</strong> Organization of the CH domains in HAUS6 and HAUS7. Middle: Overlay of augmin’s CH domains with CH domains from other MT-binding proteins. Shown are human proteins: HAUS6, HAUS7 (<a href="https://www.rcsb.org/structure/7SQK" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDB-7SQK</a>), Ndc80 (<a href="https://doi.org/10.2210/pdb3IZ0/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDB-3IZ0</a>) and CEP44 (<a href="https://doi.org/10.2210/pdb7PT5/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDB-7PT5</a>). <strong>c</strong> Top: Top-ranked AF2 models of the augmin N-clamp from different species, with coloring as indicated (see Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). The unstructured region of the HAUS8-NT is indicated with a red arrowhead. The missing second CH domain (CH7) domain for <em>D. melanogaster</em> is indicated with a circle. Bottom<strong>:</strong> Representative negative stain EM 2D classes of purified augmin N-clamp from different species. <strong>d</strong> Left: Schematic representation of tubulin co-sedimentation assay. Right: Representative SDS-PAGE analyses of tubulin co-sedimentation assays for wild-type augmin N-clamp from different species. Tubulin concentration is indicated on the top. Supernatant (S) and Pellet (P) fractions are indicated (see Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). <strong>e</strong> Pelleted fraction from tubulin co-sedimentation assays for different species plotted against the tubulin concentration. Shown as mean ± SD (see Methods and Table <a href="#Tab1" class="usa-link">1</a>) with fitted binding curves, coloring as indicated. Experiments were performed in at least <em>n</em> = 3 repetitions: <em>H. sapiens</em>
<em>n</em> = 5; <em>X. laevis</em>
<em>n</em> = 3; <em>D. melanogaster</em>
<em>n</em> = 4; <em>A. thaliana</em>
<em>n</em> = 3. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par5">Recent structural analysis of vertebrate augmin revealed that the HAUS8-NT extends from a hammerhead-like structure comprising two additional potential MT-binding elements: the globular calponin homology (CH) domains of HAUS6 and HAUS7<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, hereafter referred to as CH6 and CH7, respectively (Fig. <a href="#Fig1" class="usa-link">1b</a>). CH domains adopt a compact, globular fold consisting of four main alpha-helices and form the MT-binding motifs in numerous MT-associated proteins, like the MT plus-end binding EB proteins<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, the kinetochore proteins Ndc80 and Nuf2<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> and the inner centriole protein CEP44<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. In these complexes, the CH domains are often accompanied by unstructured extensions, which typically play an important role in the ability of the protein to bind MTs<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Per similarity, the composition of the N-clamp thus suggests that augmin binds MTs through a composite interface involving the unstructured HAUS8-NT and one or two of the CH domains<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p>
<p id="Par6">Here, we elucidate the molecular basis for the augmin-MT interaction by reconstituting the augmin N-clamp as a minimal MT-binding unit across various model organisms. We identify the low-complexity MT-binding unit from <em>D. melanogaster</em> as an optimal model for dissecting the central function of CH6 in MT binding. By integrating cryo-EM analysis, in silico modeling and in vitro assays with in vivo data, we characterize how CH6 anchors augmin to the inter-protofilament groove between adjacent β-tubulin subunits. Our study reveals a conserved MT-binding mechanism utilized by augmin to achieve initiation of branched MT formation and proper spindle assembly.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Augmin N-clamp is a structurally conserved MT-binding unit</h3>
<p id="Par7">To identify the minimal structurally conserved MT-binding unit of augmin, we analyzed augmin TII structures predicted by AlphaFold2 (AF2)<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> across a range of model organisms. Our comparative analysis, including vertebrates (<em>X. laevis</em> and <em>H. sapiens</em>), invertebrates (<em>D. melanogaster</em>) and plants (<em>A. thaliana</em>), revealed that the general architecture of augmin TII as predicted by AF2 is remarkably conserved (Fig. <a href="#Fig1" class="usa-link">1c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">1a–c</a>). The predicted TII architecture is characterized by extensive coiled-coil bundles, bisected by a hinge region that provides conformational flexibility between the N- and C-clamp, consistent with previous studies<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Notably, the augmin N-clamp in <em>D. melanogaster</em> included only a single CH domain formed by the HAUS6 ortholog Dgt6 (hereafter named Dgt6<sub>HAUS6</sub>), and lacked the second CH domain mapping at the N-terminus of Msd5<sub>HAUS7</sub> in other organisms<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Additionally, we observed substantial length and charge variability in the unstructured N-termini of HAUS8 orthologs (Fig. <a href="#Fig1" class="usa-link">1c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>), especially for the <em>D. melanogaster</em> Dgt4<sub>HAUS8</sub>-NT, which is the shortest amongst those considered and contains no positively charged side chains.</p>
<p id="Par8">To experimentally test and compare the MT binding function of augmin N-clamp from these key model organisms and validate the AF2 structure predictions, we established structure-guided recombinant reconstitution of the minimal augmin MT-binding units from <em>H. sapiens</em>, <em>X. laevis</em>, <em>A. thaliana</em>, and <em>D. melanogaster</em> in <em>E. coli</em>.</p>
<p id="Par9">We co-expressed the augmin N-clamp components based on standardized dual expression constructs (<em>HAUS2</em>, <em>HAUS6</em>, <em>HAUS7-EGFP-HIS</em><sub><em>8</em></sub>, and <em>HAUS8-HIS</em><sub><em>8</em></sub>) (Supplementary Fig. <a href="#MOESM1" class="usa-link">3a</a>). Using His-affinity purification combined with size exclusion chromatography (SEC) and anion exchange chromatography (AEC), we successfully purified augmin N-clamp from all indicated organisms (Supplementary Fig. <a href="#MOESM1" class="usa-link">3b</a>), as also confirmed by mass spectrometry (MS) analysis (Supplementary Tables <a href="#MOESM1" class="usa-link">1</a>–<a href="#MOESM1" class="usa-link">4</a>).</p>
<p id="Par10">Negative stain electron microscopy (EM) analysis and 2D class averaging verified complex formation and validated general features of the AF2 structure predictions (Fig. <a href="#Fig1" class="usa-link">1c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">3c</a>). The 2D class averages recapitulated all ordered structural elements of the augmin N-clamp complexes, including two globular densities representing the CH domains of HAUS6 and HAUS7 orthologs, along with an elongated density segment corresponding to the extended coiled-coil region formed by HAUS2, 6, 7 and 8. As predicted, only a single CH domain was resolved in the <em>D. melanogaster</em> augmin N-clamp. This was further reflected in the reduced molecular weight of the <em>D. melanogaster</em> augmin N-clamp compared to human augmin N-clamp in mass photometry experiments (Supplementary Fig. <a href="#MOESM1" class="usa-link">3d</a>). Due to its flexibility, the N-terminal extension of HAUS8 orthologs was not resolved by negative stain EM analysis.</p>
<p id="Par11">To assess the function of the purified augmin N-clamp complexes in MT binding, we conducted tubulin co-sedimentation assays, an established method for characterizing the MT binding properties of proteins. All recombinant augmin N-clamp complexes demonstrated MT binding activity in vitro, co-pelleting with polymerized α/β-tubulin in a concentration-dependent manner (Fig. <a href="#Fig1" class="usa-link">1</a>d, e). Importantly, human augmin N-clamp bound to MTs with an affinity comparable to that reported for the full TII subcomplex<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> (Table <a href="#Tab1" class="usa-link">1</a>, Fig. <a href="#Fig1" class="usa-link">1</a>d, e), indicating that the augmin N-clamp is a fully functional minimal MT-binding unit.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Quantification of MT binding affinity for wild-type and mutated augmin N-clamp complexes from different species as determined using tubulin co-sedimentation assays</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Sample</th>
<th colspan="1" rowspan="1">K<sub>d</sub> [μM]</th>
<th colspan="1" rowspan="1">SD of K<sub>d</sub>
</th>
<th colspan="1" rowspan="1">R<sup>2</sup>
</th>
<th colspan="1" rowspan="1">Number of experiments</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">H. sapiens<sup>wild-type</sup>
</td>
<td align="left" colspan="1" rowspan="1">0.8</td>
<td align="left" colspan="1" rowspan="1">0.4</td>
<td align="left" colspan="1" rowspan="1">0.94</td>
<td align="left" colspan="1" rowspan="1">5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">X. laevis<sup>wild-type</sup>
</td>
<td align="left" colspan="1" rowspan="1">0.6</td>
<td align="left" colspan="1" rowspan="1">0.4</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A. thaliana<sup>wild-type</sup>
</td>
<td align="left" colspan="1" rowspan="1">0.01</td>
<td align="left" colspan="1" rowspan="1">0.09</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D. melanogaster<sup>wild-type</sup>
</td>
<td align="left" colspan="1" rowspan="1">0.4</td>
<td align="left" colspan="1" rowspan="1">0.3</td>
<td align="left" colspan="1" rowspan="1">0.90</td>
<td align="left" colspan="1" rowspan="1">4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D. melanogaster<sup>Msd5-GST</sup>
</td>
<td align="left" colspan="1" rowspan="1">0.04</td>
<td align="left" colspan="1" rowspan="1">0.01</td>
<td align="left" colspan="1" rowspan="1">0.90</td>
<td align="left" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H. sapiens<sup>HAUS8-ΔNT</sup>
</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">X. laevis<sup>HAUS8-ΔNT</sup>
</td>
<td align="left" colspan="1" rowspan="1">2.2</td>
<td align="left" colspan="1" rowspan="1">2.1</td>
<td align="left" colspan="1" rowspan="1">0.69</td>
<td align="left" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A. thaliana<sup>AUG8-ΔNT</sup>
</td>
<td align="left" colspan="1" rowspan="1">2.1</td>
<td align="left" colspan="1" rowspan="1">1.5</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D. melanogaster<sup>Dgt4- ΔNT</sup>
</td>
<td align="left" colspan="1" rowspan="1">2.9</td>
<td align="left" colspan="1" rowspan="1">0.5</td>
<td align="left" colspan="1" rowspan="1">0.99</td>
<td align="left" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H. sapiens<sup>HAUS6-4A-mut</sup>
</td>
<td align="left" colspan="1" rowspan="1">5.6</td>
<td align="left" colspan="1" rowspan="1">1.8</td>
<td align="left" colspan="1" rowspan="1">0.98</td>
<td align="left" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D. melanogaster<sup>Dgt6-3A-mut</sup>
</td>
<td align="left" colspan="1" rowspan="1">25.5</td>
<td align="left" colspan="1" rowspan="1">31.9</td>
<td align="left" colspan="1" rowspan="1">0.94</td>
<td align="left" colspan="1" rowspan="1">3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D. melanogaster<sup>Msd5- Δ81–93</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.0</td>
<td align="left" colspan="1" rowspan="1">1.0</td>
<td align="left" colspan="1" rowspan="1">0.61</td>
<td align="left" colspan="1" rowspan="1">3</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p14"><p>Apparent <em>K</em><sub><em>d</em></sub> values, standard deviation (SD) and <em>R</em><sup>2</sup> of the fitted curve (see Methods, Figs. <a href="#Fig1" class="usa-link">1</a>–<a href="#Fig3" class="usa-link">3</a>) are shown. Dgt4/AUG8 and Dgt6 are the <em>D. melanogaster/A. thaliana</em> orthologs of HAUS8 and HAUS6, respectively. Number of experiments is given. No binding was observed for <em>H. sapiens</em><sup>HAUS8-ΔNT</sup>(-).</p></div></div></section><p id="Par12">Notably, the bacterial expression system does not replicate the extensive post-translational modifications that augmin subunits undergo during mitosis. As a result, the properties of the augmin N-clamp observed in biochemical assays with recombinant augmin N-clamp may represent only a subset of its properties in vivo. However, our results demonstrate that the augmin N-clamp is a highly suitable, low-complexity model for dissecting the interaction between augmin and MTs in vitro from both mechanistic and structural perspectives.</p></section><section id="Sec4"><h3 class="pmc_sec_title">The role of HAUS8-NT for MT binding varies between species</h3>
<p id="Par13">Next, we used our recombinant in vitro system (Supplementary Fig. <a href="#MOESM1" class="usa-link">4a</a>) combined with tubulin co-sedimentation assays to systematically dissect the contribution of different augmin N-clamp components to MT binding. The unstructured HAUS8-NT of human augmin was reported to be essential for the augmin-MT interaction<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Consistently, deletion of the HAUS8-NT in the human augmin N-clamp completely abolished MT binding (Fig. <a href="#Fig2" class="usa-link">2a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4b</a>, Table <a href="#Tab1" class="usa-link">1</a>). While MT binding was also impaired in both <em>X. laevis</em> and <em>A. thaliana</em> augmin N-clamp constructs lacking the HAUS8-NT (Fig. <a href="#Fig2" class="usa-link">2</a>b, c, Supplementary Fig. <a href="#MOESM1" class="usa-link">4b</a>, Table <a href="#Tab1" class="usa-link">1</a>), they still exhibited concentration-dependent MT-binding activity, albeit with reduced apparent binding affinities and maximum binding capacities. In contrast, a much milder effect was observed upon HAUS8-NT deletion in <em>D. melanogaster</em> (Fig. <a href="#Fig2" class="usa-link">2d</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4b</a>, Table <a href="#Tab1" class="usa-link">1</a>), where the Dgt4<sub>HAUS8</sub>-NT is substantially shorter than in other species (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>) and completely devoid of the positively charged residues that have been postulated to drive MT binding in its vertebrate counterparts<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Overall, these data reveal a differential effect of the HAUS8-NT on MT binding in different species, likely reflecting evolutionary adaptations of the MT branching machinery.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. The HAUS8-NT differentially contributes to MT binding across species.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373997_41467_2025_63165_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a79/12373997/e4cf66630daa/41467_2025_63165_Fig2_HTML.jpg" loading="lazy" id="d33e1010" height="579" width="799" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a-d</strong> Pelleted fraction from tubulin co-sedimentation assays for wild-type and HAUS8-ΔNT augmin N-clamp variants from different species plotted against tubulin concentration. Schematic icons indicate the organization of the corresponding augmin N-clamp and the deleted amino acids of HAUS8-NT. Shown as mean ± SD (see Methods and Table <a href="#Tab1" class="usa-link">1</a>) with fitted binding curves, coloring as indicated. Curves of the wild-type complexes from Fig. <a href="#Fig1" class="usa-link">1</a>. Experiments with HAUS8-ΔNT constructs were performed in <em>n</em> = 3 repetitions. Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">The CH domain of HAUS6 is a conserved MT-binding element</h3>
<p id="Par14">Having tested the role of HAUS8-NT across species, we next turned our attention to the CH6 and CH7 domains, first focusing on structural conservation and electrostatic charge distribution. Generally, surface residues on CH6 and CH7 were more conserved on the surface distal than proximal to the coiled-coil region (Fig. <a href="#Fig3" class="usa-link">3a</a>), suggesting that the distal surface is more likely to participate in a conserved function like MT binding. Interestingly, while the CH7 surface is predominantly negatively charged, the CH6 surface is largely basic (Fig. <a href="#Fig3" class="usa-link">3b</a>). This indicates a higher potential for CH6 to mediate interaction with the overall negatively charged surface of MTs, which is also consistent with the complete absence of CH7 in <em>D. melanogaster</em>. Consistently, sequence alignment of HAUS6 orthologs across eukaryotes (Supplementary Fig. <a href="#MOESM1" class="usa-link">5a</a>), from plants to human, identified several conserved basic residues on CH6, specifically K84, R90, K91, K98, and K123 (human residue numbers), forming a solvent-exposed basic patch (Fig. <a href="#Fig3" class="usa-link">3b</a>) that may underlie specific CH6-MT interactions to orient augmin on MTs.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. CH6 is a conserved MT-binding element.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373997_41467_2025_63165_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a79/12373997/f884b275280c/41467_2025_63165_Fig3_HTML.jpg" loading="lazy" id="d33e1167" height="959" width="799" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Model of the human augmin N-clamp hammerhead region (<a href="https://doi.org/10.2210/pdb7SQK/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDB-7SQK</a>) colored according to conservation score, shown from viewing angles distal and proximal to the coiled-coil (CC) region. Coiled-coil region and HAUS8-NT are indicated. Coloring scheme and conserved residues are indicated. <strong>b</strong> Augmin N-clamp hammerhead region from human (<a href="https://doi.org/10.2210/pdb7SQK/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDB-7SQK</a>), <em>X. laevis</em> (<a href="https://doi.org/10.2210/pdb8AT3/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDB-8AT3</a>), and AF2 predictions for <em>D. melanogaster</em> and <em>A. thaliana</em> were colored according to charge distribution. View distal to the coiled-coil region (see Supplementary Fig. <a href="#MOESM1" class="usa-link">5a</a>). <strong>c</strong> Pelleted fraction from tubulin co-sedimentation assays for wild-type and HAUS6<sup>4A-mut</sup> (K84A, R90A, K91A, K98A) human augmin N-clamp variants plotted against tubulin concentration. Shown as mean ± SD with fitted binding curves (see Methods and Table <a href="#Tab1" class="usa-link">1</a>), coloring as indicated. Curve of wild-type complex from Fig. <a href="#Fig1" class="usa-link">1</a>. Experiments were performed in n = 3 repetitions. <strong>d</strong> Pelleted fraction from tubulin co-sedimentation assays for wild-type and Dgt6<sub>HAUS6</sub><sup>3A-mut</sup> (K82A, R88A, K96A) N-clamp variants from <em>D. melanogaster</em> plotted against tubulin concentration. Shown as mean ± SD with fitted binding curves (see Methods and Table <a href="#Tab1" class="usa-link">1</a>), coloring as indicated. Curve of wild-type complex from Fig. <a href="#Fig1" class="usa-link">1</a>. Experiments were performed in <em>n</em> = 3 repetitions. <strong>e</strong> Immunofluorescence images following dsRNA-mediated Dgt6<sub>HAUS6</sub> knockdown in <em>D. melanogaster</em> S2 cells. Cells expressing either <em>Dgt6</em><sub><em>HAUS6</em></sub><sup><em>wild-type</em></sup><em>-</em>GFP or <em>Dgt6</em><sub><em>HAUS6</em></sub><sup><em>3A-mut</em></sup><em>-</em>GFP constructs were stained for α-tubulin, DAPI, and GFP (Dgt6<sub>HAUS6</sub>). Representative sections of images showing metaphase cells from wild-type <em>(left)</em> and 3A-mut <em>(right)</em>. Scale bars: 5 µm. Clear localization to spindle MTs is indicated with white arrowheads in case of wild-type (see Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>). <strong>f</strong> Quantification of Dgt6<sub>HAUS6</sub> spindle localization after <em>Dgt6</em><sub><em>HAUS6</em></sub> dsRNA treatment (3 or 4 days, see Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>). The intensity values of eight line plots per spindle (GFP vs α-tubulin) were compared using Pearson correlation coefficients (PCC) and PCC values were plotted for WT = 68 cells (purple) and 3A-mut = 75 cells (gray). <em>n</em> = 4 experiments (see Supplementary Fig. <a href="#MOESM1" class="usa-link">6e</a>). The bars represent mean ± SD, statistical significance is indicated by p-values determined using two-sided Mann–Whitney <em>U</em> test. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par15">To test this prediction, we introduced point mutations in human CH6 (K84A, R90A, K91A, K98A, referred to as HAUS6<sup>4A-mut</sup>, Supplementary Fig. <a href="#MOESM1" class="usa-link">5b</a>) and performed tubulin co-sedimentation assays with wild-type and mutant human augmin N-clamp. We observed a sevenfold reduction in MT binding affinity for HAUS6<sup>4A-mut</sup> compared to human wild-type HAUS6 (Fig. <a href="#Fig3" class="usa-link">3c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">5c</a>, Table <a href="#Tab1" class="usa-link">1</a>). The residual MT binding ability likely originates from the HAUS8-NT. These data suggest that while the HAUS8-NT of human augmin is essential for initial binding, CH6 likely is required for stably anchoring augmin to MTs.</p>
<p id="Par16">To test the function of the conserved CH6 residues in a low-complexity system without interference from other MT-binding N-clamp elements, we analyzed <em>D. melanogaster</em> augmin N-clamp, where the CH7 domain is absent and the Dgt4<sub>HAUS8</sub>-NT deletion had little impact on MT binding (Fig. <a href="#Fig2" class="usa-link">2d</a>). Strikingly, mutating three of the conserved residues in <em>D. melanogaster</em> Dgt6<sub>HAUS6</sub> (K82A, R88A, and K96A; referred to as Dgt6<sub>HAUS6</sub><sup>3A-mut</sup>) resulted in a much more severe impairment in augmin N-clamp-MT binding (Fig. <a href="#Fig3" class="usa-link">3d</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>b, c, Table <a href="#Tab1" class="usa-link">1</a>) compared to the human HAUS6<sup>4A-mut</sup> N-clamp variant. This indicates that the conserved basic patch in the CH6 domain plays an important role in the augmin-MT interaction, which is more critical for establishing high-affinity MT binding in <em>D. melanogaster</em> than in human augmin.</p>
<p id="Par17">Finally, to confirm the role of the CH6 domain in MT binding in <em>D. melanogaster</em> in vivo, we generated stable S2 cell lines expressing either <em>Dgt6</em><sub><em>HAUS6</em></sub><sup><em>WT</em></sup><em>-GFP</em> or <em>Dgt6</em><sub><em>HAUS6</em></sub><sup><em>3A-mut</em></sup><em>-GFP</em> under the control of the metallothionein promoter. Endogenous Dgt6<sub>HAUS6</sub> was depleted using Dgt6<sub>HAUS6</sub> dsRNA (3 days or 4 days), and metaphase cells expressing either wild-type or mutant <em>Dgt6</em><sub><em>HAUS6</em></sub> were analyzed by immunofluorescence staining (Fig. <a href="#Fig3" class="usa-link">3e</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">6a–d</a>). Dgt6<sub>HAUS6</sub><sup>3A-mut</sup>-GFP exhibited clearly reduced colocalization with metaphase spindles compared to Dgt6<sub>HAUS6</sub><sup>WT</sup>-GFP (Fig. <a href="#Fig3" class="usa-link">3</a>e, f, Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>d, e), which localized to spindles as expected for the functional augmin complex<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>.</p>
<p id="Par18">These results establish the CH6 domain as a conserved MT-binding element across species, linking in silico analysis with both in vitro and in vivo functional data.</p></section><section id="Sec6"><h3 class="pmc_sec_title">CH6 directly binds to MTs to orient augmin on the MT lattice</h3>
<p id="Par19">Next, we aimed at elucidating the structural basis for how CH6 anchors and orients augmin on MTs using cryo-EM. To this end, we used the <em>D. melanogaster</em> augmin N-clamp as a low-complexity model system, in which CH6 plays a dominant role in the augmin-MT interaction, while the CH7 domain is absent and the Dgt4<sub>HAUS8</sub>-NT deletion had little impact on MT binding. Pre-incubation of taxol-stabilized MTs (see Methods) with <em>D. melanogaster</em> augmin N-clamp resulted in pronounced MT bundling complicating cryo-EM analysis (Supplementary Fig. <a href="#MOESM1" class="usa-link">7a–c</a>). However, sequential application of MTs and augmin N-clamp onto cryo-EM grids yielded individual MTs decorated with <em>D. melanogaster</em> augmin N-clamp, suitable for cryo-EM analysis (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>d, e). Following established cryo-EM image processing workflows (see Methods)<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, we reconstructed the 14-protofilament MT to ~4 Å resolution at the tubulin level (Fig. <a href="#Fig4" class="usa-link">4a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">8a–e</a>), enabling us to unambiguously distinguish α- and β-tubulin (Fig. <a href="#Fig4" class="usa-link">4b</a>). Inspection of the cryo-EM reconstruction at low density threshold level identified the augmin N-clamp binding site in the groove between β-tubulin subunit pairs on adjacent protofilaments (Fig. <a href="#Fig4" class="usa-link">4a</a>). Subsequent symmetry expansion and 3D classification focused on the asymmetric unit provided a cryo-EM reconstruction of the augmin N-clamp bound to the MT (Supplementary Fig. <a href="#MOESM1" class="usa-link">8f, g</a>). However, the limited local resolution for the decorating density (Supplementary Fig. <a href="#MOESM1" class="usa-link">8f</a>) and the absence of density attributable to more distal regions of the augmin N-clamp prevented confident docking of the N-clamp model predicted by AF2.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. CH6 orients <em>D. melanogaster</em> augmin N-clamp on MTs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373997_41467_2025_63165_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a79/12373997/6274c7e3c285/41467_2025_63165_Fig4_HTML.jpg" loading="lazy" id="d33e1390" height="568" width="781" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Helical reconstruction of a 14-PF MT decorated with the <em>D. melanogaster</em> augmin N-clamp shown at high confidence threshold (top) and overlaid with the same density at lower confidence threshold (transparent) after low-pass filtering to 15 Å (bottom). Green circles indicate additional density in the groove between neighboring β-tubulin subunits. Coloring as indicated. <strong>b</strong> MT-lumenal view of one α/β-tubulin dimer in the cryo-EM density from (a) superposed with the atomic model of an α/β-tubulin dimer (<a href="https://doi.org/10.2210/pdb6WVR/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDB-6WVR</a>)<sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup>. Loop S9-S10 region, distinguishing α- and β-tubulin, is outlined and shown in blue along with taxol (stick representation); otherwise, coloring as in panel (<strong>a</strong>). <strong>c</strong> Same as (<strong>b</strong>), but showing the reconstruction of the 13 PF MT decorated with GST-N-clamp shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">10c</a>. <strong>d</strong> 3D reconstruction of 13PF MT-bound augmin GST-N-clamp after focused 3D classification of symmetry-expanded particles, filtered to local resolution. The AF2 prediction of <em>D. melanogaster</em> augmin N-clamp was rigid-body docked. Distal segments of the AF2 prediction are not shown. Coloring as indicated. <strong>e</strong> AF2 prediction of the full <em>D. melanogaster</em> augmin N-clamp placed on the MT lattice based on the N-clamp fit shown in (<strong>d</strong>). Coloring as in (<strong>d</strong>).</p></figcaption></figure><p id="Par20">To improve the cryo-EM sample quality, we next aimed at boosting the overall N-clamp decoration of MTs by enhancing the N-clamp binding affinity, which is relatively low (~0.4 µM) compared to similar MT-binding complexes successfully analyzed by cryo-EM, such as Ndc80-related constructs<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. To preserve the native binding mode, we aimed at increasing binding affinity by GST-mediated dimerization of the augmin N-clamp. We placed a GST tag after the C-terminal EGFP tag on Msd5<sub>HAUS7</sub>, separated by a flexible linker (Supplementary Fig. <a href="#MOESM1" class="usa-link">9a</a>), generating a GST-tagged <em>D. melanogaster</em> augmin N-clamp version (GST-N-clamp). Complex formation and GST-induced dimerization were confirmed by mass photometry (Supplementary Fig. <a href="#MOESM1" class="usa-link">9b, c</a>). Tubulin co-sedimentation assays showed that GST-mediated dimerization decreased the apparent K<sub>d</sub> approximately tenfold to ~40 nM (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>d, e). We repeated cryo-EM experiments with the GST-N-clamp (see Methods) analogously to wild-type N-clamp (Supplementary Fig. <a href="#MOESM1" class="usa-link">10a–d</a>). After assigning the α/β-tubulin register (Fig. <a href="#Fig4" class="usa-link">4c</a>), we observed decorating density at the same position as for wild-type N-clamp, confirming the MT binding site of augmin. Compared to our cryo-EM reconstruction of the wild-type augmin N-clamp, the local estimated resolution for the GST-N-clamp was substantially improved (Supplementary Fig. <a href="#MOESM1" class="usa-link">10b</a>) and the more distal coiled-coil regions of the GST-N-clamp were better visible at low threshold level (Supplementary Fig. <a href="#MOESM1" class="usa-link">10c</a>). Subsequent recentering of particles on the asymmetric unit, combined with additional rounds of focused 3D classification and 3D refinement, further improved the cryo-EM reconstruction to secondary structure level for the entire CH6 domain and MT-proximal segments of Msd5<sub>HAUS7</sub> and Dgt4<sub>HAUS8</sub> (Fig. <a href="#Fig4" class="usa-link">4d</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>d, e). This enabled highly confident rigid body docking of the augmin N-clamp model predicted by AF2 and revealed the binding site and orientation of the augmin N-clamp on MTs (Fig. <a href="#Fig4" class="usa-link">4</a>d, e). In this configuration, the conserved basic patch in the augmin CH6 domain is positioned to directly bind to the MT surface, indicating that it centrally contributes to orienting augmin on the MT lattice. Notably, MT-proximal α-helices of Msd5<sub>HAUS7</sub> and Dgt4<sub>HAUS8</sub> were slightly differently oriented between the cryo-EM reconstruction and the AF2 prediction, which may reflect local inaccuracies of the model predicted by AF2 or suggest small conformational rearrangements of the augmin N-clamp upon MT binding.</p></section><section id="Sec7"><h3 class="pmc_sec_title">MD simulations identify central interacting residues</h3>
<p id="Par21">To assess the stability of the interface and gain residue-level insight into the interactions involved, we conducted atomistic Molecular Dynamics (MD) simulations of the composite <em>D. melanogaster</em> augmin N-clamp–MT model (see Methods). We simulated three independent replicas for 1 µs each and analyzed the final 200 ns and 500 ns of each simulation. Throughout all simulations, augmin N-clamp remained stably bound at the inter-protofilament groove between two adjacent β-tubulin subunits (see Supplementary Movie <a href="#MOESM3" class="usa-link">1</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">11a</a>). Residue-level contacts between the augmin N-clamp and surrounding tubulin subunits were analyzed within a 0.6 nm cutoff and averaged across the three replicas. The average contact frequencies of augmin N-clamp with β-tubulin indicated that the majority of interactions were driven by the CH6 domain and are located in the regions of residues 5-13, 71-88 and 110-121 (Fig. <a href="#Fig5" class="usa-link">5</a>a, b, Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>b, c; Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>a, b). Additionally, we identified a flexible loop region in Msd5<sub>HAUS7</sub> (residues 85–92) that strongly interacted with the disordered C-termini of β-tubulin rich in acidic residues, known as the E-hook (Fig. <a href="#Fig5" class="usa-link">5</a>a, b, Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>b, c; Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>a, b; Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>a, b)<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. This loop in Msd5<sub>HAUS7</sub> appears to be fly-specific and correlates with the absence of the CH7 domain and reduced length and charge in the Dgt4<sub>HAUS8</sub>-NT in these species (Supplementary Fig. <a href="#MOESM1" class="usa-link">14a</a>). Notably, our cryo-EM density shows unassigned density segments extending from the tubulin E-hook region adjacent to the N-clamp binding site (Supplementary Fig. <a href="#MOESM1" class="usa-link">10f</a>), aligning with MD-simulation data that suggest E-hook contacts contribute to augmin N-clamp binding in <em>D. melanogaster</em>.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Molecular characterization of the augmin N-clamp-MT interface via atomistic molecular dynamics simulations.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373997_41467_2025_63165_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a79/12373997/37198c5d9238/41467_2025_63165_Fig5_HTML.jpg" loading="lazy" id="d33e1536" height="638" width="780" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Pairwise interaction energy map highlighting augmin N-clamp residues interacting with the two β-tubulin monomers, displaying the average interaction energy as the sum of electrostatic and van der Waals contributions, based on the contacts identified in Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>. Key interacting residues with an interaction energy of less than −20 kJ/mol are shown. <strong>b</strong> Representative structure of the MT-bound augmin N-clamp, with close-up views (red, black, orange) highlighting all residues identified as involved in contacts. Key interacting residues with an interaction energy of less than -20 kJ/mol are labeled, see Supplementary Fig. <a href="#MOESM1" class="usa-link">12b</a>. <strong>c</strong> Root-Mean-Square-Fluctuation (RMSF) of the MT-bound augmin N-clamp, with the color gradient representing the degree of fluctuations (in nm) along the structure. Blue regions indicate minimal fluctuation, while red regions emphasize higher flexibility. The RMSF is calculated for the entire trajectory of each run and subsequently averaged across all runs, coloring is indicated. <strong>d</strong> Orientation of the full <em>D. melanogaster</em> augmin complex placed on the MT, based on a composite AF2 prediction of the augmin octamer (see Supplementary Fig. <a href="#MOESM1" class="usa-link">16</a>). TII and TIII tetramers, MT polarity and possible daughter MT direction are indicated. Coloring of TII as in (<strong>b</strong>), TIII as indicated.</p></figcaption></figure><p id="Par22">To refine the simulation-based contact map, we calculated the interaction energy between residues, highlighting the contributions of both short- and long-range interactions (Fig. <a href="#Fig5" class="usa-link">5a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">13a</a>). This analysis indicated that the interaction network between augmin N-clamp and β-tubulin is primarily driven by electrostatic interactions and ion pairs (Fig. <a href="#Fig5" class="usa-link">5</a>a, b, Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>a–c). Key residues, making frequent interactions throughout both time ranges include R3, K10, K48, K81, K82, R88 and R121 of Dgt6<sub>HAUS6</sub>, as well as K86, K87 and R88 in Msd5<sub>HAUS7</sub>. In contrast to β-tubulin, α-tubulin plays a minor role in interacting with the augmin N-clamp. Only α-tubulin E-hook residues D431 and E434 of α-tubulin form strong interactions with R3 and the N-terminus of Dgt6<sub>HAUS6</sub> (Supplementary Fig. <a href="#MOESM1" class="usa-link">15a–f</a>).</p>
<p id="Par23">To further investigate the contribution of the Msd5<sub>HAUS7</sub> loop region in MT binding, we deleted the Msd5<sub>HAUS7</sub> loop residues forming strong interactions in the MD simulations (K86, K87, and R88; Msd5<sub>HAUS7</sub><sup>Δ81–93</sup>) from the <em>D. melanogaster</em> N-clamp. Following purification, complex integrity of the mutant N-clamp was confirmed by mass photometry (Supplementary Fig. <a href="#MOESM1" class="usa-link">14b–d</a>). MT co-sedimentation assays demonstrated that mutant Msd5<sub>HAUS7</sub><sup>Δ81–93</sup> N-clamp was strongly compromised in MT binding (Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>e, f, Table <a href="#Tab1" class="usa-link">1</a>), validating an important role of the Msd5<sub>HAUS7</sub> loop region in MT binding and thereby supporting a model in which this loop may substitute for the HAUS8-NT conserved in other eukaryotes.</p>
<p id="Par24">Next, we calculated the root-mean-square-fluctuation (RMSF) to assess the dynamics of the MT-bound augmin N-clamp. While CH6 segments in contact with the MT are highly rigid, consistent with the strong network of electrostatic interactions, fluctuations increase significantly as they propagate away from the MT (Fig. <a href="#Fig5" class="usa-link">5c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">13c</a>), which may explain why only the MT-bound region of the N-clamp could be resolved using cryo-EM at high density confidence threshold. Overall, these data are fully consistent with our structural, functional and evolutionary analysis (Fig. <a href="#Fig3" class="usa-link">3</a>, Fig. <a href="#Fig4" class="usa-link">4</a>), further underlining the critical role of residues in the CH6 basic patch for MT binding.</p>
<p id="Par25">Finally, we investigated how the orientation of the <em>D. melanogaster</em> augmin N-clamp consistently observed in MD simulations and cryo-EM structural analysis positions the full augmin complex on MTs. Using a “divide-and-conquer” approach adapted from a previously published protocol<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, we modeled the complete <em>D. melanogaster</em> augmin complex based on AF2 predicted models (Supplementary Fig. <a href="#MOESM1" class="usa-link">16a–d</a>). Combining these predictions with the observed MT binding mode of the <em>D. melanogaster</em> N-clamp, we generated a composite model of augmin bound to MTs (Fig. <a href="#Fig5" class="usa-link">5d</a>). Anchoring via CH6 aligns the γ-TuRC-binding augmin TIII subcomplex along the MT axis, oriented toward the MT plus end. In this configuration, augmin TIII is sterically highly accessible to interact with γ-TuRCs for nucleation of daughter MTs.</p></section><section id="Sec8"><h3 class="pmc_sec_title">CH6-MT interactions are likely conserved</h3>
<p id="Par26">Next, we aimed to understand whether the CH6 binding mode experimentally observed for <em>D. melanogaster</em> augmin N-clamp was evolutionarily conserved and compatible with the more complex augmin N-clamp architecture in other organisms. We utilized AF2 to predict an α/β-tubulin dimer in complex with the <em>D. melanogaster</em> CH6 domain. In 15 out of 25 models, the position of the <em>D. melanogaster</em> CH6 domain matched our cryo-EM reconstruction (Fig. <a href="#Fig6" class="usa-link">6a</a>). The small deviations between the predicted and experimental position of CH6 most likely arise from the absence of the second β-tubulin subunit that the CH6 domain binds to in the cryo-EM reconstruction. This high similarity between the predicted and experimental models of the CH6-MT complex in <em>D. melanogaster</em> indicated that AF2 is generally capable of identifying correct MT-CH6 interactions. Therefore, we extended our AF2 analysis of CH6-MT interactions to <em>H. sapiens</em>, <em>X. laevis</em>, and <em>A. thaliana</em> CH6 domains (Fig. <a href="#Fig6" class="usa-link">6b</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">17</a>a, b). In all top-ranked models, CH6 was predicted to bind β-tubulin in an orientation similar to that observed in <em>D. melanogaster</em>, suggesting a conserved binding mode for CH6 domains in all inspected organisms (Fig. <a href="#Fig6" class="usa-link">6b</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. CH6 binding is conserved across species and compatible with complex augmin N-clamp architecture.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373997_41467_2025_63165_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a79/12373997/cfb631bc121b/41467_2025_63165_Fig6_HTML.jpg" loading="lazy" id="d33e1747" height="757" width="783" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Top-ranked AF2 predictions of α/β-tubulin (porcine) in complex with the <em>D. melanogaster</em> CH6 domain compared to the cryo-EM model of GST-N-clamp bound to MT (Fig. <a href="#Fig4" class="usa-link">4e</a>), presented in two views. Coloring as indicated. <strong>b</strong> Top-ranked AF2 predictions of α/β-tubulin (porcine) in complex with the CH6 domains from four different species, presented in two views. For prediction metrics and sequences used for the predictions see Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a> and Supplementary Table <a href="#MOESM1" class="usa-link">7</a>, respectively. <strong>c</strong> Human augmin N-clamp (<a href="https://doi.org/10.2210/pdb7SQK/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDB-7SQK</a>) placed on the MT based on the AF2 predicted position of CH6 shown in (b). Potential direction of HAUS8-NT binding is indicated. <strong>d</strong> Representative immunofluorescence images of pRetroX-Tet-On RPE1 cells expressing siRNA-resistant <em>HAUS6</em><sup><em>wild-type</em></sup>-mRuby (WT) and <em>HAUS6</em><sup><em>4A-mut</em></sup>-mRuby (4A-mut) in metaphase. Cells were treated with either siRNA against HAUS6 (siHAUS6) or a non-specific control siRNA (siControl). Expression of <em>HAUS6</em>-mRuby constructs was induced by 20 ng/ml doxycycline (+Dox) or left uninduced (-Dox). Cells were stained for α-tubulin, DAPI, and HAUS6, and imaged as well for HAUS6-mRuby channel. Scale bars: 5 µm. <strong>e</strong> Spindle-pole-to-spindle-pole distance was measured for all conditions in wild-type (WT, purple) and 4A-mut (4A, gray) cells. The data are presented as boxplots with median as central line and 25<sup>th</sup> and 75<sup>th</sup> percentile as borders of the box with minimum and maximum values as whiskers. Sample sizes for each condition: WT siControl, <em>n</em> = 102 cells; WT siHAUS6, <em>n</em> = 136 cells; 4A siControl, <em>n</em> = 158 cells; 4 A siHAUS6 <em>n</em> = 187 cells; WT siHAUS6 (−Dox), <em>n</em> = 166 cels; 4 A siHAUS6 (−Dox), <em>n</em> = 130 cells. Data were pooled from three independent experiments. Statistical significance is indicated by <em>p</em>-values determined using two-sided Kruskal–Wallis test with Dunn’s multiple comparison test. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par27">Superposing the full human augmin N-clamp<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> onto the human CH6 domain as positioned on the MT surface by AF2 (Fig. <a href="#Fig6" class="usa-link">6c</a>) revealed that the more complex N-clamp architecture in vertebrates and plants, including a longer HAUS8-NT and the additional CH7 domain, is fully compatible with the CH6 orientation predicted by AF2 and experimentally determined by cryo-EM in case of <em>D. melanogaster</em>. This further supports a conserved binding mode for CH6 across species.</p>
<p id="Par28">To test whether the conserved and predicted CH6-MT interface is functionally relevant in human augmin, we generated pRetroX-Tet-On RPE1 cell lines expressing either siRNA-resistant wild-type or 4A-mutant <em>HAUS6-mRuby</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>a, b) and tested the impact of the CH6-4A mutations under siRNA-mediated knockdown of endogenous <em>HAUS6</em> (Fig. <a href="#Fig6" class="usa-link">6d</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">18c</a>). Expression of <em>HAUS6-mRuby</em> constructs was confirmed via immunoblotting (Supplementary Fig. <a href="#MOESM1" class="usa-link">18b</a>) and immunofluorescence analysis demonstrated robust centrosome localization during interphase and MT localization during cell division, consistent with previous findings<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>d, e). By immunoblotting, we verified knockdown of endogenous <em>HAUS6</em>, but not siRNA-resistant <em>HAUS6-mRuby</em>, upon siRNA treatment and doxycycline-induced expression (Supplementary Fig. <a href="#MOESM1" class="usa-link">18f–h</a>). Previous studies have established that knockdown of augmin components results in severe mitotic spindle defects, most prominently manifested in shortened spindles<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Consistently, siRNA-mediated knockdown of <em>HAUS6</em> (siHAUS6) resulted in visibly impaired spindles (Fig. <a href="#Fig6" class="usa-link">6d</a>), characterized by a significant reduction in spindle-pole-to-spindle-pole distance (Fig. <a href="#Fig6" class="usa-link">6e</a>). Wild-type <em>HAUS6-mRuby</em> expression rescued the spindle length defects, restoring spindle dimensions to near-wild-type levels. In contrast, expression of the 4A-mutant phenocopied <em>HAUS6</em> knockdown and resulted in significantly shorter mitotic spindles (Fig. <a href="#Fig6" class="usa-link">6e</a>), demonstrating that the conserved residues in CH6 are critical for the function of human augmin. Collectively, these results suggest that the MT-CH6 interactions characterized here represent a conserved mechanism essential for correct binding of augmin to spindle MTs and the proper assembly of the mitotic spindle.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par29">In this study, we identify how augmin stably binds to MTs via the N-clamp, a structurally conserved composite MT-binding unit shared across plants, invertebrates and vertebrates. Our data establish that the conserved CH6 domain plays a critical role in MT binding throughout all species studied. We focused our study on <em>D. melanogaster</em>, where CH6 is central for the augmin-MT interaction by binding to the inter-protofilament groove between two β-tubulins, as demonstrated by a combination of cryo-EM structure determination, AF2 predictions and MD simulations, in conjunction with in vitro tubulin co-sedimentation and in vivo spindle co-localization assays. These findings considerably advance our mechanistic understanding of augmin’s composite MT-binding interface, which was hypothesized to involve a second critical contact site in addition to the HAUS8-NT<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> over a decade ago, which we now identify as the CH6 domain.</p>
<p id="Par30">Interestingly, we found that the requirement for HAUS8-NT in MT binding varies between species. While its deletion in <em>D. melanogaster</em> has the mildest effect, the much more extended HAUS8-NT in humans is essential for MT binding, likely due to its positively charged residues that interact with the negatively charged MTs through ionic interactions<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Surprisingly, in <em>X. laevis</em> and <em>A. thaliana</em>, HAUS8-NT deletion compromises but not completely abolish MT binding, underscoring the important role of HAUS6 in augmin´s MT binding affinity and function. This aligns with experimental data demonstrating that the human HAUS8-NT alone binds MTs with substantially lower efficiency than the full augmin complex or the augmin TII tetramer<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. In light of our results, this indicates that while HAUS6/Dgt6 is universally important for MT binding, the necessity of HAUS8-NT in recruiting and stabilizing the CH6-MT interaction is species-dependent. In case of <em>D. melanogaster</em>, our MD simulations and tubulin co-sedimentation assays for instance indicate that the MT binding function of the very short Dgt4<sub>HAUS8</sub>-NT could be partially replaced by the disordered loop of Msd5<sub>HAUS7</sub>, which engaged tubulin E-hook residues through a patch of positively charged residues (K86, K87, R88, R92) in the simulations — reminiscent of the K-loop of KIF1A<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Overall, this suggests that stable anchoring of CH6 may generally require additional electrostatic interactions, provided for example by the positively charged human HAUS8-NT or the <em>D. melanogaster</em> Msd5<sub>HAUS7</sub> loop. In some instances, posttranslational modifications such as phosphorylation may fine-tune those electrostatic interactions, as reported for vertebrate and plant HAUS8-NT<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>–<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>.</p>
<p id="Par31">Although we provide insights into the functional interplay between CH6 and HAUS8-NT, the role of CH7 remains unclear. The appearance of multiple CH domains in a protein complex is not uncommon, especially in actin-binding proteins, where one CH domain often assumes a stabilizing or regulatory function, while the other mediates binding<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. The conserved negative charge distribution in CH7 domains suggests a conserved mechanism of action, perhaps in recruiting interaction partners or balancing augmin’s electrostatics to enable augmin clustering at MT branching sites (see below)<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>.</p>
<p id="Par32">MT branching angles have been analyzed across species, including <em>H. sapiens</em>, <em>X. laevis</em>, <em>D. melanogaster</em>, and plants, revealing context-dependent patterns. In human in vitro systems and <em>Xenopus</em> egg extracts, branching nucleation mediated by γ-TuRC and augmin predominantly occurs at shallow angles below 30°, reflecting the dense MT packing necessary for spindle assembly and function<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. In plants, branching angles have been characterized within cortical arrays and phragmoplasts, typically ranging from 20° to 60°<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. In <em>D. melanogaster</em>, both in vitro and in vivo analyses reveal branching angles typically ranging from 30° to 60°<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. While the unstructured HAUS8-NT is unlikely to provide a uniform and stable platform for orienting the γ-TuRC on pre-existing MTs, the conserved CH6 domain and its defined binding site on the MT surface are a prime candidate to fulfill this function. Consistently, while the human augmin N-clamp with CH6-4A mutations still bound to MTs in pelleting assays due to the contribution of HAUS8-NT, functional analyses showed severely disrupted spindle function and assembly in cells expressing <em>HAUS6</em><sup><em>4A-mut</em></sup>, as witnessed by reduced pole-to-pole distance, a phenotype that is typical for loss of augmin function<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. How regulatory and augmin-cooperating factors may affect the branching angle by modulating the augmin-MT interaction remains to be investigated in molecular detail. For example, the plant protein MACET4 has been shown to affect MT branching angles and MT organization by modulating binding of the augmin TII subunit AUG7<sub>HAUS7</sub> to MTs, highlighting the potential for additional regulatory layers to fine-tune augmin anchoring to MTs<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>.</p>
<p id="Par33">Similar to the cooperativity observed in Ndc80 complexes and other multivalent assemblies, where clustering amplifies both binding affinity and functional specificity<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>,<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>, a recent study suggested a potential clustering mechanism of augmin at MT branching sites<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. In light of our structural data, clustering of multiple augmin complexes per branch site could involve binding of several copies of augmin to β-tubulin subunit pairs in close spatial proximity. Augmin’s conformational flexibility<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, in particular in the hinge region separating the augmin TII N- and C-clamp, may play a central role in enabling γ-TuRC interactions from MT-binding sites with a slight offset. In vertebrates, augmin clusters may also involve multiple copies of augmin, which are not stably docked to the MT lattice, but only flexibly tethered by the HAUS8-NT. Key posttranslational modifications and augmin-interacting branching factors like γ-TuRC, HURP, NEDD1 or TPX2<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> may facilitate or further assist augmin clustering.</p>
<p id="Par34">Notably, we observed pronounced MT bundling activity of our N-clamp construct in vitro (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>), consistent with previous reports for HAUS8 (formerly HICE1)<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Although MT bundle formation has been reported in vivo in the context of kinetochore fibers and dense spindle arrays<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, whether augmin directly contributes to this organization through bundling activity is unknown. In fact, MT bundling is a common feature of multivalent MT-binding proteins in vitro<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>,<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> but may arise from high local protein concentrations or other non-physiological experimental conditions. Addressing whether the MT bundling effect of the augmin N-clamp reflects a specific biological function will require further investigations.</p>
<p id="Par35">Apart from augmin, the MT-binding protein TPX2 has also been described as a pivotal component of MT branching in <em>X. laevis</em> and is able to recruit augmin and γ-TuRC to MTs<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. Cryo-EM analysis has revealed the TPX2 binding site in the inter-protofilament groove between two adjacent β-tubulin units close to the CH6 binding site<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>, suggesting that direct N-clamp-TPX2 interactions may underlie the augmin-TPX2 cooperativity during MT branching<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>.</p>
<p id="Par36">In sum, by characterizing the N-clamp as a minimal MT-binding unit of augmin and demonstrating the conserved role of CH6 in orienting augmin on MTs, we provide a basis for exploring species-specific regulatory mechanisms, additional MT binding partners, and evolutionary adaptation of augmin. In particular, investigating augmin’s interplay with the γ-TuRC, TPX2, or HURP<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> will be essential for uncovering the complexity of its regulation and its indispensable role in cell division.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Molecular cloning</h3>
<p id="Par37">For each cloning step, the vectors were linearized and amplified by PCR (Q5, NEB). All PCR-amplified fragments were assembled using the NEBuilder Assembly Kit following the manufacturer’s instructions (NEB). DNA fragments of <em>HAUS2</em>, <em>HAUS6</em>, <em>HAUS7</em>, and <em>HAUS8</em> from different species (Supplementary Table <a href="#MOESM1" class="usa-link">5</a>) were cloned into two separate <em>E. coli</em> expression plasmids based on the pETDuet-1 and pET-26b expression vectors using primers listed in Supplementary Table <a href="#MOESM1" class="usa-link">6</a>.</p>
<p id="Par38">For <em>H. sapiens</em> and <em>X. laevis</em>, in two subsequent cloning steps, <em>HAUS8</em> and <em>HAUS6</em> encoding sequences were inserted into the two pETDuet-1 cloning sites, adding a C-terminal 8xHis tag to the <em>HAUS8</em> sequence. <em>HAUS2</em> and <em>HAUS7</em> were inserted into another pETDuet-1 vector. Afterwards, the pET-26b expression cassette was replaced with the pETDuet-1 expression cassette including the gene fragments of <em>HAUS2</em> and <em>HAUS7</em> using primers listed in Supplementary Table <a href="#MOESM1" class="usa-link">6</a>. During this step, the sequence of <em>TEV-EGFP-3C-8xHis</em> tag was added to the C-terminus on the <em>HAUS7</em> fragment using a pET26b backbone as described<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Gene fragments from <em>A. thaliana</em> and <em>D. melanogaster</em> were codon-optimized for <em>E. coli</em> expression (IDT, USA) and ordered as gene fragments (TWIST Bioscience) for direct insertion into the pETDuet-1/pET-26b vector scheme replacing the corresponding gene fragments. For the generation of EGFP-GST tagged Msd5<sub>HAUS7</sub> the GST sequence was amplified from a pGEX plasmid and pET-26b Msd1<sub>HAUS2</sub>/Msd5<sub>HAUS7</sub> plasmid was amplified with primers listed in Supplementary Table <a href="#MOESM1" class="usa-link">6</a>.</p>
<p id="Par39">For protein expression, these constructs were used for co-transformation. Alternatively, to enhance transformation and expression efficiency, the two expression cassettes of pETDuet-1 and pET-26b were combined into the pETDuet-1 vector. To achieve this, the pETDuet1 backbone containing the <em>HAUS6</em> and <em>HAUS8</em> gene fragments, and the gene cassette from the modified pET-26b vectors were amplified separately. In this step, the 8xHis tag following <em>HAUS7-TEV-EGFP</em> was removed by replacing it with a stop codon. These components were then combined, resulting in a pETDuet-1 vector containing all four gene cassettes (Supplementary Fig. <a href="#MOESM1" class="usa-link">3a</a>).</p>
<p id="Par40">Mutations and deletions in the constructs were introduced by PCR amplification followed by DpnI (NEB) digestion and transformation into DH5α <em>E. coli</em>, as for the other constructs.</p>
<p id="Par41">Full-length pMT <em>Dgt6</em><sub><em>HAUS6</em></sub><sup><em>WT</em></sup><em>-GFP</em> and <em>Dgt6</em><sub><em>HAUS6</em></sub><sup><em>3A-mut</em></sup><em>-GFP</em> constructs for expression in <em>D. melanogaster</em> were cloned into the pMT-GFP expression vector. For this purpose, the remaining C-terminal segment of <em>Dgt6</em><sub><em>HAUS6</em></sub> was ordered as a gene fragment (TWIST Bioscience), including a C-terminal TEV-ALFA tag, and then cloned together with the N-terminal segment of <em>Dgt6</em><sub><em>HAUS6</em></sub> into the GFP-V5 region of the pMT plasmid (see Supplementary Fig. <a href="#MOESM1" class="usa-link">6a</a>).</p>
<p id="Par42">pRetroX-Tet3G and pVSVG (Retro-X Tet-On 3G Inducible Expression System, Clontech) were used to generate the RPE1 cells with Tet-On 3 G System. The pRetroX-Tet3G with <em>HAUS6</em> WT or 4A-mut constructs were cloned sequentially. First, non-coding base-exchange mutations for siRNA resistance were introduced into the pETDuet-1 <em>HAUS6</em> N-clamp constructs. Next, point mutation PCR was performed to incorporate the 4A mutations using primers listed in Supplementary Table <a href="#MOESM1" class="usa-link">6</a>. Then, the siRNA-resistant wild-type or mutated N-terminal part of <em>HAUS6</em> was cloned into a full-length <em>HAUS6</em> construct (pACEBac1 HAUS6-2xFLAG). Finally, the full-length <em>HAUS6</em> from the pRetroX-Tet3G vector and the mRuby sequence flanked by ALFA-tag were amplified and combined via a NEBuilder reaction.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Protein expression</h3>
<p id="Par43">For the expression of augmin N-clamp constructs, 100-150 ng of DNA constructs were transformed into 100 μl of competent <em>E. coli</em> BL21 CodonPlus-RIL (Stratagene). Transformed cells were cultured in 2xYT medium (Roth) at 37 °C in volumes up to 500 ml until an optical density (OD) of maximal 0.8 was reached. The expression cultures were then shifted to 18 °C and diluted with an equal volume of cold 2xYT medium. Protein expression was induced at an OD of 0.6-0.8 by adding 0.5 mM IPTG, and the cultures were incubated at 18 °C for 20-21 h with shaking. Cells were harvested by centrifugation (4000<em>g</em>, 4 °C for 20 min), washed with PBS (Gibco), and either used immediately for protein purification or flash-frozen in liquid nitrogen and stored at −80 °C until further use.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Protein purification</h3>
<p id="Par44">For the purification of recombinant augmin N-clamp complexes, expression pellets from 1 liter culture were resuspended in 30–35 ml cold lysis buffer (20 mM HEPES, pH 7.4, 300 mM KCl, 1 mM MgCl<sub>2</sub>, 1 mM EGTA, 2 mM DTT, 0.1% Tween-20), supplemented with one EDTA-free protease inhibitor tablet (Roche) and 10 μl Benzonase (Sigma Aldrich). Resuspended cells were kept on ice and sonicated (6 × 1 min at 100% amplitude; Hielscher UP50H). The lysate was clarified by centrifugation (20,000<em>g</em> for 30 min at 4 °C). The total soluble fraction of the lysate was incubated with Ni-NTA beads (Qiagen) pre-equilibrated with lysis buffer for 90 min at 4 °C with gentle rotation. Typically, 400 μl of Ni-NTA beads were used per 1 l of culture. After incubation, beads were separated from the flow-through by centrifugation (800<em>g</em>, 3 min at 4 °C) and washed once with lysis buffer, followed by two washes with wash buffer (20 mM HEPES, pH 7.4, 200 mM KCl, 15 mM imidazole, 1 mM MgCl<sub>2</sub>, 1 mM EGTA, 1 mM DTT). After each washing step, the beads were sedimented by centrifugation (800<em>g</em>, 3 min at 4 °C). Recombinant proteins were eluted three times from the Ni-NTA beads by incubating with 1–1.25 bead volumes of elution buffer (20 mM HEPES, pH 7.4, 200 mM KCl, 400 mM imidazole, 1 mM MgCl<sub>2</sub>, 1 mM EGTA, 1 mM DTT) for 10 min at 4 °C. Elution fractions were collected by centrifugation (800<em>g</em>, 3 min at 4 °C) and subjected to SEC using a Superose 6 Increase (10/300) GL column equilibrated in SEC buffer (20 mM HEPES, pH 7.4, 150 mM KCl, 1 mM MgCl<sub>2</sub>, 1 mM EGTA, 1 mM DTT, 2.5% glycerol) at 0.5 ml/min. Peak fractions were verified by Coomassie-stained SDS-PAGE analysis (4-20% gradient SDS Precast-gel; BIO-RAD), pooled, and subjected to AEC. GST-N-clamp was used for cryo-EM experiments and mass photometry after SEC. For AEC, peak fractions from SEC were loaded onto a Capto HiRes™ Q 5/50 GL column (Cytiva) equilibrated with low salt buffer (20 mM HEPES, pH 7.4, 75 mM KCl, 1 mM MgCl<sub>2</sub>, 1 mM EGTA, 1 mM DTT, 2.5% glycerol). Recombinant N-clamp complexes were eluted using a gradient from 75 mM to 1 M KCl at a flow rate of 0.5 ml/min over 20 column volumes. All chromatography runs were performed on an ÄKTA Pure system or ÄKTA go system (Cytiva) controlled by Unicorn software (version 7.9 or 7.5). Peak fractions were verified via Coomassie-stained SDS-PAGE (4–20% gradient SDS Precast-gel; BIO-RAD), combined, and concentrated using Amicon concentrators (30 kDa MWCO). The purified proteins were used directly or flash-frozen in liquid nitrogen and stored at −80°C until further use. Porcine tubulin was purified according standard protocols as previously described<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Stable RPE1 cell lines</h3>
<p id="Par45">Human telomerase-immortalized RPE1 cells (hTERT-RPE1 TP53−/−) were cultured in Dulbecco’s Modified Eagle’s Medium (DMEM)/F-12 (Gibco) supplemented with 10% FBS, 1% penicillin–streptomycin, and 1% L-glutamine. Cells were maintained at 37 °C in a 5% CO<sub>2</sub>-humidified incubator. Stable cell lines were firstly generated by introducing the Retro-X™ Tet-On® 3 G Inducible Expression System (Clontech) into the RPE1 <em>TP53</em><sup>−/−</sup> cells. Different <em>HAUS6</em> siRNA-resistant constructs were then integrated under the TRE3G promotor via Retrovirus infection (Clontech).</p></section><section id="Sec15"><h3 class="pmc_sec_title">RNA interference and rescue in RPE1 cells</h3>
<p id="Par46">Endogenous HAUS6 depletion was achieved by transfecting cells with siRNA oligonucleotides targeting <em>HAUS6</em> (5′-CAGUUAAGCAGGUACGAA-3′) (Dharmacon™)<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> using Lipofectamine RNAiMAX (Invitrogen). Transfection reactions were prepared in Opti-MEM™ medium according to the manufacturer’s protocol. RPE1 cells were seeded to 60–80% confluency in 6-well plates. siRNA oligonucleotides (50 nM) were incubated with RNAiMAX in serum-free Opti-MEM medium (Gibco) for 20 min, then added to the cells. After 24 h, cells were reseeded into 24-well plates containing 12 mm coverslips for another 48 h. To rescue the depletion and to analyze the function, the <em>HAUS6</em> wild-type or 4A-mut construct expression were induced by adding 20 ng/ml of doxycycline to the cell medium 48 h before the end of the siRNA depletion experiment.</p></section><section id="Sec16"><h3 class="pmc_sec_title">
<em>D. melanogaster</em> S2 cell transfection</h3>
<p id="Par47">Sterile culture conditions were maintained for <em>D. melanogaster</em> S2 cells at 25 °C. The culture medium consisted of Schneider’s medium supplemented with 10% heat-inactivated fetal bovine serum (FBS) and 100 μg/ml Penicillin/Streptomycin. The expression of the <em>Dgt6</em><sub><em>HAUS6</em></sub><em>-GFP</em> wild-type and 3A-mut gene, driven by the metallothionein promoter (pMT vector), was induced using CuSO<sub>4</sub> (500 μM for 16 h). <em>D. melanogaster</em> S2 cells stably expressing pMT constructs were generated via Cellfectin II mediated transfection (Life Technologies™), with hygromycin used as the selection marker. Two days post-transfection, the medium was supplemented with 200 μg/ml hygromycin B. The medium was refreshed weekly. After 4 weeks, once the cells resumed exponential growth, the hygromycin concentration was reduced to 100 μg/ml.</p></section><section id="Sec17"><h3 class="pmc_sec_title">dsRNA-mediated knockdown</h3>
<p id="Par48">Double-stranded RNA (dsRNA) was synthesized using the MEGAscript T7 kit (Ambion) following the manufacturer’s protocol. Templates were generated by PCR from cDNA using specific primers for <em>Dgt6</em><sub><em>HAUS6</em></sub> and brown (control). The sequences of primers and dsRNA are listed in Supplementary Table <a href="#MOESM1" class="usa-link">6</a>. Knockdown efficiency was assessed via qPCR. A total of 6 × 10<sup>5</sup> logarithmically growing S2 cells were seeded in 12-well plates, with 2 ml of medium per well. On the following day, cells were incubated for 1 h with 8 μg of dsRNA in 0.5 ml of serum-free medium. Next, 1 ml of medium containing 20% serum was added, and cells were incubated for 3 or 4 days. 16 h before harvesting, the <em>Dgt6</em><sub><em>HAUS6</em></sub><em>-GFP</em> construct was induced with 500 μM CuSO<sub>4</sub>. Cells were used for immunofluorescence (IF) or RNA extraction.</p></section><section id="Sec18"><h3 class="pmc_sec_title">RNA qPCR</h3>
<p id="Par49">RNA isolation was performed using Trizol® reagent according to the manufacturer’s protocol. Any remaining genomic DNA was digested using TURBO<sup>TM</sup> DNase (Ambion by Life Technologies). cDNA synthesis was carried out using the Quantitect Reverse Transcription Kit (Qiagen). A control reaction without reverse transcriptase was included to detect potential gDNA contamination. qPCR was performed in triplicates on a LightCycler 480 multiwell plate, using LightCycler 480 SYBR Green I Master (Roche) in combination with the LightCycler 480 instrument (Roche). The primer pair PP14341 for quantifying <em>Dgt6</em><sub><em>HAUS6</em></sub> was obtained from FlyPrimerBank. Its efficiency was tested and measured at 97%. dGapdH served as the reference gene for normalization. The qPCR data were analyzed using the 2^ − ΔΔCT method to determine relative gene expression levels<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. Wild-type S2 cells were treated with 8 μg of <em>Dgt6</em><sub><em>HAUS6</em></sub> dsRNA, while the control group received brown dsRNA. Harvesting was performed at 3- and 6-day post-treatment.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Immunofluorescence</h3>
<p id="Par50">For <em>D. melanogaster</em> S2 cells, a 100 μl cell suspension was placed on a Poly-L-lysine-treated glass coverslip. Cells were allowed to settle for 10 min before fixation with 4% paraformaldehyde (PFA) for an additional 10 min. After three washes with PBS, cells were permeabilized for 7 min using PBS containing 0.3% Triton X-100. Nonspecific binding was blocked by incubating cells for 1 h with 4% BSA in PBS. All antibodies were diluted in blocking solution and incubated for 1 h at room temperature. GFP was stained using Thermo Fisher A10262 antibody (1:200, Thermo Fisher), and α-tubulin was stained with Sigma T9026 antibody (1:700). After three washes with PBS, cells were incubated with the corresponding Alexa Fluor secondary antibodies (1:500), protected from light. After another three washes with PBS, cells were mounted in ProLong™ Diamond Antifade Mountant with DAPI (Invitrogen™) on a glass slide.</p>
<p id="Par51">For experiments in RPE1 cells, <em>HAUS6</em> wild-type or 4A-mut cells were grown on 12 mm coverslips and fixed with cold methanol at −20 °C for 5 min. They were then blocked with PBS supplemented with 10% FBS and 0.1% Triton X-100 for 1 h at room temperature (RT). Afterwards, cells were incubated with primary antibody, diluted in 3% BSA solution, for 1 h. Following primary antibody incubation, cells were stained with secondary antibody and DAPI in 3% BSA solution for 45 min, then mounted with Moviol. Image acquisition was performed using a DeltaVision RT system (Applied Precision) with an Olympus IX71 microscope, equipped with 60×/1.42 and 100×/1.40 oil objective lenses.</p>
<p id="Par52">Metaphase cells from the <em>Drosophila</em> S2 cell experiment were analyzed to compare Dgt6<sub>HAUS6</sub><sup>wild-type</sup>-GFP and Dgt6<sub>HAUS6</sub><sup>3A-mut</sup>-GFP cells following <em>Dgt6</em><sub><em>HAUS6</em></sub> dsRNA treatment. For each metaphase spindle, four lines were drawn through each spindle pole side, resulting in eight lines in total (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>). Signal intensity along these lines in all channels was measured using the line plot tool in Fiji software<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. The intensity values between the GFP and tubulin channels for each line were compared by calculating the Pearson correlation coefficient (PCC) between the two channels. The data of 3 or 4 days <em>Dgt6</em><sub><em>HAUS6</em></sub> dsRNA treatment were compared and plotted individually (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>, as well as pooled in Fig. <a href="#Fig3" class="usa-link">3f</a>). For the RPE1 experiments, spindle pole distances were measured using the Fiji software<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Antibodies</h3>
<p id="Par53">Primary antibodies used in this study for immunofluorescence (IF) and immunoblot were: GFP was stained using A10262 chicken antibody (1:200, Thermo Fisher). In <em>D. melanogaster</em> S2 cells α-tubulin was stained with Sigma T9026 mouse antibody (1:700), whereas for RPE1 cells mouse α-tubulin (monoclonal, clone 1E4C11, Proteintech, IF 1:500, REF 66031-1IG, LOT 10028345) was used. Alpaca HRP-conjugated sdAb anti-ALFA (monoclonal, clone 1G5, NanoTag Biotechnologies GmbH, WB 1:1000, REF N1505-HRP, LOT 032303B), rabbit GAPDH (monoclonal, clone 14C10F5G9, Cell Signaling Technology, WB 1:1000, REF #2118, LOT 16), rabbit polyclonal antibodies against HAUS6 (polyclonal, AA 448–955, WB 1:2000, IF 1:1000)<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>.</p>
<p id="Par54">Secondary HRP-conjugated antibody used in immunoblot was HRP-conjugated anti-rabbit (polyclonal, Jackson ImmunoResearch, WB 1:5000, REF 711-035-152, 172401). Secondary antibodies used for immunofluorescence were: Alexa Fluor 488 goat anti-chicken (Thermo Fisher Scientific, A-11039, 1:500), Alexa Fluor 647 goat anti-mouse (Thermo Fisher Scientific, A-21235, 1:500), Alexa Fluor Plus 488 donkey anti-mouse IgG (polyclonal, Thermo Fisher Scientific, IF 1:500, REF A-11001, LOT 2816171), Alexa Fluor Plus 647 donkey anti-rabbit IgG (polyclonal, Thermo Fisher Scientific, IF 1:500, REF A32733, LOT 2577247).</p></section><section id="Sec21"><h3 class="pmc_sec_title">LC-MS/MS analysis</h3>
<p id="Par55">Each aliquot of human, <em>X. laevis</em>, <em>A. thaliana</em>, and <em>D. melanogaster</em> augmin N-clamp was thawed and clarified by centrifugation at 20,000<em>g</em> for 5 min at 4 °C. The clarified samples were then mixed with an equal volume of 2x Lämmli buffer and boiled at 95 °C for 3 min. Protein separation was carried out by SDS-PAGE (minigel, 10%; Invitrogen). Coomassie-stained bands were excised with a scalpel and processed as previously described<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>,<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. Briefly, the samples were reduced, alkylated, and digested with trypsin. Peptides were extracted from the gel pieces, concentrated using a vacuum centrifuge, and dissolved in 15 μl of 0.1% TFA.</p>
<p id="Par56">Nanoflow LC-MS/MS analysis of the four distinct augmin N-clamp samples was conducted utilizing two distinct LC-MS platforms: an Ultimate 3000 liquid chromatography (LC) system coupled to an Orbitrap Q Exactive HF (QE HF; Thermo Fisher Scientific), and a Vanquish Neo coupled to an Orbitrap Tribrid Eclipse mass spectrometer (Thermo Fisher Scientific). In both configurations, peptide separation was performed on an in-house packed analytical column (75 μm × 200 mm) using Reprosil-Pur AQ 120 C18 stationary phase material with 1.9 μm particle size (Dr. Maisch, Germany). Each sample analysis was performed with a single biological replicate (<em>n</em> = 1).</p>
<p id="Par57">For analyses performed using the Ultimate 3000 LC system coupled to the Orbitrap QE HF, mobile phase solvents consisted of solvent A (0.1% formic acid, 1% acetonitrile) and solvent B (0.1% formic acid, 89.9% acetonitrile). Peptides were separated using a linear gradient over 25 min, initiated at 3% solvent B and increased to 23% over the first 21 min, then raised further to 38% solvent B over the next 4 min, followed by a wash step at 95% solvent B. The QE HF mass spectrometer was operated in data-dependent acquisition (DDA) mode, automatically switching between MS1 and MS2 scans. Full MS1 spectra were recorded over a mass range of m/z 400–1600 at a resolution of 60,000 at <em>m/z</em> 400. MS2 spectra were generated from up to 15 precursor ions using higher-energy collisional dissociation (HCD) with a normalized collision energy (NCE) of 27 and an isolation window of 1.4 <em>m/z</em>.</p>
<p id="Par58">For analyses performed on the Vanquish Neo coupled to the Orbitrap Tribrid Eclipse mass spectrometer, mobile phase solvents consisted of solvent A (0.1% formic acid) and solvent B (0.1% formic acid, 80% acetonitrile). Peptides were separated using a 30 min linear gradient, starting from 4% solvent B and linearly increasing to 32% over 25 min, then further increasing to 49% solvent B within the subsequent 5 min, and finally concluding with a wash step at 99% solvent B. The Orbitrap Tribrid Eclipse was operated in data-dependent acquisition mode. MS1 spectra were acquired at a resolution of 120,000 with an AGC target of 1.2 × 10<sup>6</sup>, a maximum injection time (MaxIT) of 50 ms, an RF lens setting of 30%, and a mass range of <em>m/z</em> 400–1600. Dynamic exclusion of precursor ions was set to 10 s, with all charge states of a given precursor excluded upon selection. Monoisotopic peak determination was enabled, with the isolation window centered on the most abundant peak. Singly charged ions and ions with charge states above six were excluded, and an intensity threshold filter was set to 5 × 10<sup>3</sup>. MS2 spectra were acquired in the linear ion trap using a turbo scan rate, custom AGC target set to 2.5 × 10<sup>4</sup>, MaxIT of 14 ms, and NCE (HCD) set to 30%.</p>
<p id="Par59">Raw data obtained from the Orbitrap QE HF instrument were processed using Proteome Discoverer 2.5 software with the Sequest HT search algorithm (Thermo Fisher Scientific)<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. Fragment ion mass tolerance was set to 0.02 Da, and precursor ion mass tolerance was set to 5 ppm. Trypsin was specified as the digestion enzyme. Carbamidomethylation of cysteine residues was set as a fixed modification, whereas oxidation of methionine, deamidation of asparagine and glutamine, acetylation at protein N-terminus, and methionine loss at protein N-terminus were specified as variable modifications. Peptide quantification was conducted using the precursor ion quantifier node employing the Top N Average method, with N set to 3 for protein abundance calculation.</p>
<p id="Par60">Raw data from the Orbitrap Tribrid Eclipse instrument were processed using Proteome Discoverer 3.1 software with Sequest HT (Thermo Fisher Scientific)<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. Fragment ion mass tolerance was set to 0.6 Da, and precursor ion mass tolerance to 10 ppm. Trypsin was again selected as the digestion enzyme, carbamidomethylation of cysteine residues as a fixed modification, and oxidation of methionine and methionine loss at the protein N-terminus as variable modifications. Peptide quantification utilized the precursor ion quantifier node, with protein abundance calculated via the Summed Abundance method.</p>
<p id="Par61">MS/MS spectra were searched against several protein sequence databases, including a customized contaminant database (part of MaxQuant, MPI Martinsried)<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>, UniProt <em>E. coli</em> protein database (UP000000625_83333; modified November 2019, containing 4,349 sequences), UniProt <em>A. thaliana</em> protein database (UP000006548_3702; modified November 2019, containing 27,402 sequences), UniProt <em>H. sapiens</em> protein database (modified June 2020, containing 20,531 sequences), UniProt <em>D. melanogaster</em> protein database (modified November 2019, containing 13,776 sequences), and a custom database containing modified sequences of the protein of interest.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Mass photometry</h3>
<p id="Par62">Mass photometry measurements were conducted using a Refeyn TwoMP mass photometer (Refeyn Ltd, Oxford, UK). Videos were recorded for 1 min with the default image size using Refeyn AcquireMP 2024 R1 software (Refeyn Ltd, Oxford, UK). Freshly washed and dried high-precision microscope coverslips (24 ×50 mm) were used for the measurements. A silicone gasket with six cavities was positioned at the center of the coverslip to create measurement wells. For each measurement, 19 μl of MP buffer (20 mM HEPES, pH 7.4, 150 mM KCl, 1 mM MgCl<sub>2</sub>, 1 mM EGTA or 1xPBS) was added to each gasket cavity, and autofocus was performed before every measurement. Subsequently, 1 μl of protein solution at a concentration of 400 nM was mixed with the MP buffer. Data analysis and plotting were carried out using Refeyn DiscoverMP 2024 R1 software (Refeyn Ltd, Oxford, UK). A standard contrast-to-mass calibration curve was generated using bovine serum albumin (BSA, 66 kDa) and immunoglobulin G (IgG, 150 kDa and 300 kDa).</p></section><section id="Sec23"><h3 class="pmc_sec_title">Tubulin co-sedimentation assay</h3>
<p id="Par63">Taxol-stabilized MTs were nucleated in vitro based on a previously published protocol<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. Briefly, purified porcine brain tubulin was mixed in BRB80 buffer (80 mM PIPES, 1 mM MgCl2, 1 mM EGTA, pH 6.8) at a final concentration of 50 µM. The nucleation reaction was performed at 37 °C for 20 min in the presence of 0.5 mM GTP (Sigma Aldrich) and 5 µM paclitaxel (Thermo Fisher Scientific) in BRB80-MT buffer (80 mM PIPES, 1 mM MgCl<sub>2</sub>, 1 mM EGTA, 12.5% (w/v) glycerol, pH 6.8). MTs were subsequently layered on top of a 50 µl 30% (w/v) glycerol cushion containing paclitaxel and GTP and pelleted at 200,000 g for 10 min at 27 °C using an S100-AT3 rotor (Thermo Fisher Scientific). The supernatant was discarded, and the pellet was resuspended in 40 µl of BRB80-MT buffer for further experiments. For co-sedimentation assays, MT concentration in each reaction refers to α/β-tubulin heterodimer molarity. Taxol-stabilized MTs were incubated with 1 µM purified N-clamp, 0.25 mg/ml BSA, and BRB80-MT buffer in a final reaction volume of 30 µl. After 20 min of incubation at room temperature to reach binding equilibrium, the reaction mixtures were sedimented at 200,000<em>g</em> for 10 min at 27 °C in an S100-AT3 rotor. Pellet and supernatant fractions were analyzed by SDS-PAGE (12% or 4-20% SDS Precast gel; BIO-RAD) and Coomassie staining (Coomassie Brilliant Blue G250; Sigma Aldrich). Gel images were acquired using an LAS400IR, and gel densitometry was performed with Fiji software<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. Only samples that did not co-sediment in the negative control (0 µM tubulin) were used for MT binding quantification. For the comparative analysis, only concentration data points present in every sample were quantified. Normalized MT binding data were obtained by integrating peak intensity values for HAUS6/AUG6/Dgt6 bands or the combined HAUS6/HAUS8 bands. Normalized MT binding data were calculated as the fraction of protein co-sedimented with MTs (pellet fraction) and fitted to a modified quadratic equation as previously<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>:</p>
<table class="disp-formula p" id="Equ1"><tr><td class="formula"><math id="d33e2548" display="block"><mi mathvariant="normal">complex</mi><mo>_</mo><mi mathvariant="normal">fraction</mi><mo>=</mo><mn>2</mn><mfenced close=")" open="("><mrow><msub><mrow><mi>B</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>+</mo><msub><mrow><mi>K</mi></mrow><mrow><mi>d</mi></mrow></msub><mo>+</mo><mi>x</mi></mrow></mfenced><mo>−</mo><mn>2</mn><msup><mrow><mfenced close=")" open="("><mrow><msup><mrow><mfenced close=")" open="("><mrow><msub><mrow><mi>B</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>+</mo><msub><mrow><mi>K</mi></mrow><mrow><mi>d</mi></mrow></msub><mo>+</mo><mi>x</mi></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup><mo>−</mo><mn>4</mn><mi>x</mi><msub><mrow><mi>B</mi></mrow><mrow><mi>max</mi></mrow></msub></mrow></mfenced></mrow><mrow><mn>1</mn><mo>/</mo><mn>2</mn></mrow></msup></math></td></tr></table>
<p id="Par64">where <em>B</em><sub><em>max</em></sub> is the maximal fraction of N-clamp bound to MTs, <em>K</em><sub><em>d</em></sub> is the apparent dissociation constant, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e2628"><mi>x</mi></math></span> is the tubulin heterodimer concentration. Fitting and kinetic parameter calculations were performed using least squares fitting with the SciPy Python library<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>, and data were plotted using Matplotlib<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>.</p></section><section id="Sec24"><h3 class="pmc_sec_title">AlphaFold2 predictions</h3>
<p id="Par65">AF2 predictions were performed using AlphaFold v2.2.0, 2.3.1 and 2.3.2 in multimer mode with default settings, yielding 25 models per prediction. PAE values were plotted using PAE viewer<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. A list of AF2 predictions and the identifiers of the used sequences is given in Supplementary Table <a href="#MOESM1" class="usa-link">7</a>.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Conservation of HAUS8</h3>
<p id="Par66">Sequences of HAUS8 and its orthologs were fetched from UniProt for the following organisms: <em>Homo sapiens</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/Q9BT25" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q9BT25</a>), <em>Mus musculus</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/Q99L00" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q99L00</a>), <em>Xenopus laevis</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/Q0IHJ3" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q0IHJ3</a>), <em>Danio rerio</em> (A0A2R8QND7), <em>Drosophila melanogaster</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/Q9W4M8" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q9W4M8</a>), <em>Arabidopsis thaliana</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/Q9SUH5" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q9SUH5</a>), <em>Emericella nidulans</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/Q5BGP4" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q5BGP4</a>), <em>Acanthamoeba castellanii</em> (L8HF95). MSA was generated with the MAFFT algorithm<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup> using standard parameters and visualized in JalView<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Conservation of CH domains</h3>
<p id="Par67">The conservation scores of human augmin CH domains shown in Fig. <a href="#Fig3" class="usa-link">3</a> were estimated using the ConSurf server<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup> on <a href="https://doi.org/10.2210/pdb7SQK/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDB-7SQK</a> with standard parameters. For the multiple sequence alignment (MSA) of HAUS6, sequences of its orthologs were fetched for the following organisms according to their accession numbers: <em>Homo sapiens</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/Q7Z4H7" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q7Z4H7</a>), <em>Danio rerio</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/A0JMF7" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">A0JMF7</a>), <em>Drosophila melanogaster</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/Q9VAP2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q9VAP2</a>), <em>Xenopus laevis</em> (A0JPI0), <em>Bos taurus</em> (E1BG87), <em>Mus musculus</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/Q6NV99" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q6NV99</a>), <em>Canis familiaris</em> (A0A8C0MNQ7), <em>Rattus norvegicus</em> (A0A0G2K4Y6), <em>Arabidopsis thaliana</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/Q94BP7" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q94BP7</a>), <em>Acanthamoeba castellanii</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/XP_004336105.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_004336105.1</a>), <em>Apis mellifera</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/NP_001242967.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NP_001242967.1</a>), <em>Aspergillus nidulans</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/XP_663791.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_663791.1</a>), <em>Oryza sativa</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/XP_015625562.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_015625562.1</a>), <em>Chlamydomonas reinhardtii</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/XP_042915056.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_042915056.1</a>), <em>Physcomitrium patens</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/XP_024393130.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_024393130.1</a>), <em>Selaginella moellendorffii</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/XP_024525773.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_024525773.1</a>). MSA was generated with the MAFFT algorithm<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup> using standard parameters and sequence identity of the residues was calculated with JalView<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Conservation of HAUS7 and HAUS8 functional elements</h3>
<p id="Par68">Orthologs of HAUS7/Msd5 and HAUS8/Dgt4 were identified as annotated in UniProtKB or using PSI-BLAST using A0A0N0U5X0, <a href="https://www.ncbi.nlm.nih.gov/protein/Q9W0G6" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q9W0G6</a>, A0A195DAI7 or <a href="https://www.ncbi.nlm.nih.gov/protein/Q9W4M8" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q9W4M8</a> as query sequence. Identifiers for used sequences are provided in Supplementary Table <a href="#MOESM1" class="usa-link">8</a>. Sequences were compared by multiple sequence alignment using MAFFT through the EMBL-EBI job dispatcher<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup>. The HAUS7/Msd5 positively charged loop was defined as a stretch of amino acids with a net charge of at least +3 aligning with the positively charged loop in <em>D. melanogaster</em> Msd5<sub>HAUS7</sub>. Human HAUS8 residue 142 and accordingly aligned residues were defined as the final residue of the HAUS8/Dgt4-NT. HAUS8/Dgt4-NT charge was calculated as (K + R) – (D + E) + 1, accounting for the charge of the N-terminal amino group.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Generating the <em>D. melanogaster</em> augmin holocomplex model</h3>
<p id="Par69">The model of <em>D. melanogaster</em> augmin octamer was generated following a ‘divide-and-conquer’ approach adapted from<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Briefly, the highest-scoring model predicted for TIII Dgt3/5<sub>HAUS3/5</sub> in complex with TII was superposed to the highest-scoring model for TIII using the matchmake command in UCSF Chimera (v. 1.16<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup> according to the following residues: Dgt3<sub>HAUS3</sub> 135-173, 370-392 and Dgt5<sub>HAUS5</sub> 139-169, 395-429. Afterwards, the two models were merged in UCSF Chimera<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup> by retaining complementary segments of Dgt3<sub>HAUS3</sub> and Dgt5<sub>HAUS5</sub> from the two predicted models. The model predicted for the TIII HAUS<sub>Dgt</sub>3/5 in complex with TII contributed residues 174-369 from Dgt3<sub>HAUS3</sub> and residues 170-510 from Dgt5<sub>HAUS5</sub>, as well as the TII tetramer. The model predicted for TIII contributed to the remaining segments of Dgt3<sub>HAUS3</sub> and Dgt5<sub>HAUS5</sub>, as well as Wac<sub>HAUS1</sub> and Dgt2<sub>HAUS4</sub>. In the final step, atom numbering was adjusted using the pdb_reatom command from pdb-tools<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup>.</p></section><section id="Sec29"><h3 class="pmc_sec_title">Negative stain EM and 2D class averaging</h3>
<p id="Par70">Copper/palladium 400 EM mesh grids (G2400D, Plano GmbH) coated with continuous carbon ( ~ 10 nm thickness) were glow-discharged before sample application. 5 µl of purified augmin N-clamp sample was applied to the grids and incubated for 30 sec. Grids were then blotted with Whatman filter paper 50 and washed with three drops of distilled water. Sample staining was done using 3% uranyl acetate in water, and excess staining solution was removed by blotting. All datasets were acquired at RT at a Talos L120C TEM (Thermo Fisher Scientific) operated at 120 kV, equipped with a 4k x 4k Ceta CMOS camera (Thermo Fisher Scientific). Micrographs were taken using the EPU software package (version 2.9, Thermo Fisher Scientific) at a nominal defocus of approximately −2 µm and an object pixel size of 0.1992 nm. The final datasets included 303 images of human augmin N-clamp, 245 images of <em>X. laevis</em> augmin N-clamp, 549 images of <em>A. thaliana</em> augmin N-clamp and 205 images of <em>D. melanogaster</em> augmin N-clamp. Image processing was performed in RELION 3.1<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>,<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>. Contrast transfer function (CTF) parameters were estimated using Gctf<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>. Automated particle picking was done using 2D class averages obtained after manual particle picking. The selected 2D class averages were low-pass filtered to 30 Å and used as templates for all datasets with a mask diameter of 250 Å or 300 Å. The number of automatically picked particles was 50,630 for human augmin N-clamp, 32,928 for <em>X. laevis</em> augmin N-clamp, 135,869 for <em>A. thaliana</em> augmin N-clamp, and 43,840 for <em>D. melanogaster</em> augmin N-clamp. Particles were extracted in 128 or 256-px boxes, either rescaled (3.984 Å/px) or at full spatial resolution (1.992 Å/px). Extracted particles underwent up to three rounds of 2D classification into 50-200 classes, with a T-factor of 2 and a circular mask of 250 or 300 Å.</p></section><section id="Sec30"><h3 class="pmc_sec_title">Cryo-EM of MTs preincubated with wild-type augmin N-clamp</h3>
<p id="Par71">After AEC, purified wild-type <em>D. melanogaster</em> augmin N-clamp was desalted in BRB80 using a HiTrap 5 ml column followed by concentration using Amicon, 30 kDa MWCO (Merck) to a final concentration of 4 µM. Taxol-stabilized MTs were prepared as described above and diluted to a concentration of 1 mg/ml in BRB80 buffer at RT. Before plunge freezing taxol-stabilized MTs were mixed in a 1:1 ratio with the desalted <em>D. melanogaster</em> augmin N-clamp (4 µM concentration). Holey carbon grids (Cu R2/1, 200 mesh; Quantifoil) were glow-discharged twice for 1 min using an easiGlow plasma cleaner (PELCO). 3 µl of the incubation mixture was applied on the carbon side of the EM grids mounted into a Vitrobot Mark IV (Thermo Fisher Scientific) operated at 30 °C and 100% humidity. After 30 s of incubation, 4 µl of purified <em>D. melanogaster</em> augmin N-clamp sample (4 µM concentration) was added to maximize binding. After a waiting time of 1 min and 40 s, the grids were blotted for 1 s from both sides and plunge-frozen in liquid ethane. The grids were screened at a Glacios TEM (Thermo Fisher Scientific) to assess their quality and representative images were collected at different magnifications.</p></section><section id="Sec31"><h3 class="pmc_sec_title">Cryo-EM of MTs decorated with wild-type augmin N-clamp</h3>
<p id="Par72">Purified <em>D. melanogaster</em> augmin N-clamp and MTs were prepared as described above (pre-incubation). Holey carbon grids (Cu R2/1, 200 mesh; Quantifoil) were glow-discharged twice for 1 min using an easiGlow plasma cleaner (PELCO). 3 µl of MT solution (1 mg/ml) was applied on the carbon side of the EM grids and MTs were allowed to adsorb for 1 min at RT and ambient atmosphere before a short manual blotting step (Whatman filter paper #1). Subsequently, the grids were incubated three times with 3 µl of purified <em>D. melanogaster</em> augmin N-clamp sample (4 µM concentration), waiting 2 min and manually removing excess buffer (Whatman filter paper #1) between each application. Afterwards, the grids were mounted into a Vitrobot Mark IV (Thermo Fisher Scientific) operated at 30 °C and 100% humidity. Finally, 4 µl of purified <em>D. melanogaster</em> augmin N-clamp sample (4 µM concentration) was added to maximize binding. After a waiting time of 1 min and 40 s, the grids were blotted for 1 s from both sides and plunge-frozen in liquid ethane. Grids were stored in liquid nitrogen until further use.</p></section><section id="Sec32"><h3 class="pmc_sec_title">Cryo-EM of MTs decorated with augmin GST-N-clamp</h3>
<p id="Par73">MTs were prepared as described above and diluted to a final concentration of (0.5 mg/ml). Purified <em>D. melanogaster</em> augmin GST-N-clamp was desalted and concentrated as described above to a final concentration of 4 µM. Holey carbon grids (Cu R2/1, 200 mesh; Quantifoil) were glow-discharged twice for 1 min using an easiGlow plasma cleaner (PELCO). 3 µl of MT sample (0.5 mg/ml) was applied on the carbon side of the EM grids and MTs were allowed to adsorb for 1 min at RT and ambient atmosphere before a short manual blotting step (Whatman filter paper #1). Subsequently, the grids were incubated four times with 3 µl of purified <em>D. melanogaster</em> augmin GST-N-clamp sample (4 µM concentration), waiting 30 s and manually removing excess buffer (Whatman filter paper #1) between each application. Afterwards, the grids were mounted into a Vitrobot Mark IV (Thermo Fisher Scientific) operated at 30 °C and 100% humidity. Finally, 4 µl of purified <em>D. melanogaster</em> augmin GST-N-clamp sample (4 µM concentration) was added and after 30 s the grids were blotted for 1 s from both sides and plunge-frozen in liquid ethane. Grids were stored in liquid nitrogen until further use.</p></section><section id="Sec33"><h3 class="pmc_sec_title">Cryo-EM data acquisition</h3>
<p id="Par74">Micrograph movie stacks were recorded on a Titan Krios G1 (Thermo Fisher Scientific), operated at 300 kV and equipped with a K3 (Gatan, Inc.) direct electron detector and a Quantum Gatan Imaging Filter (Gatan, Inc.) with a slit width of 20 eV. Automated high-resolution data acquisition was conducted in EPU (Thermo Fisher Scientific) with four images per hole at a nominal magnification of ×81,000 (1.069 Å/px) and a nominal defocus ranging from -1.0 μm to −3.0 μm. The 8528 micrographs of <em>D. melanogaster</em> augmin N-clamp were acquired with a cumulative dose of 49.8 e<sup>-</sup>/Å<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> and a dose rate of 12.8 e<sup>−</sup>/Å<sup>2</sup>/s distributed over 50 fractions. For the augmin GST-N-clamp 8,882 micrographs were acquired with a cumulative dose of 40.4 e<sup>−</sup>/Å<sup>2</sup> and of 12.4 e<sup>−</sup>/Å<sup>2</sup>/s. distributed over 50 fractions. Details of data collection are summarized in Supplementary Table <a href="#MOESM1" class="usa-link">9</a>.</p></section><section id="Sec34"><h3 class="pmc_sec_title">Processing of WT augmin N-clamp-MT cryo-EM data</h3>
<p id="Par75">Initial steps of cryo-EM data processing were performed using RELION 3.1<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>,<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>. Beam-induced motion was corrected with MotionCor2<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup> using 5×5 patches, and CTF parameters were estimated using Gctf<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>. For particle picking, 25 micrographs were randomly selected to train a filament-picking model in crYOLO<sup><a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup> (version 1.8). MT segments were then located on the full dataset using the trained crYOLO model, with a spacing of 80 Å along the MT axis. 441,250 MT segments were extracted in 600 px boxes, rescaled to 5.01 Å/px and were subjected to 2D classification. 423,115 retained MT segments were classified by protofilament (PF) number in 4 random subsets through supervised 3D classification, as described previously<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, with 13PF (149,266 segments) and 14PF (192,313 particles) MTs as most abundant classes.</p>
<p id="Par76">14 PF MTs were used for further processing using the MiRP pipeline (v2)<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Following seam correction, 102,166 MT segments were extracted at full spatial resolution (1.069 Å/px, 432 px box). Extracted MT segments were subjected to refinement with C1 symmetry using parameters outlined in the MiRP manual<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, employing a loose mask with a z-length of 30% of the MT and the final 3D class of the MiRP pipeline before seam correction as reference. Afterwards, the outputs of the particle refinement runs were used as references.</p>
<p id="Par77">From here on, processing was performed in RELION 3.0, unless mentioned otherwise. Successive CTF refinement and Bayesian polishing (each performed twice) interspersed with 3D refinements (C1 symmetry) were applied, followed by an additional round of C1 refinement. From this point, MT segments were processed through two branches. In one branch, MT segments were subjected to helical refinement with a rise of 8.7 Å and a twist of -25.8 Å applying 14-fold helical symmetry, reaching a global resolution of 4.0 Å, followed by sharpening according to the automatically determined B-factor and filtering to local resolution. In the second branch, MT segments were symmetry expanded after the aforementioned C1 refinement using the relion_particle_symmetry_expand command, with a 14-fold expansion using a twist of -25.76 Å and a rise of 8.71 Å, yielding 1,430,324 MT segments. Expanded MT segments were subsequently subjected to C1 refinement using the same parameters as the initial refinement and transferred for further processing to CryoSPARC 4.5.3<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup>.</p>
<p id="Par78">In CryoSPARC, MT segments were classified into 3 classes using a soft focus mask covering an α/β-tubulin dimer and associated decorating protein density opposite the seam. The class containing the most well-defined decorating density in uniform tubulin register was selected (469,266 MT segments) and MT segments were subjected to non-uniform homogeneous refinement (C1 symmetry) with local angular sampling, using a solvent mask covering all 14 protofilaments and decorating protein. Finally, the reconstruction, reaching 4.7 Å, was subjected to local resolution and local filter jobs.</p></section><section id="Sec35"><h3 class="pmc_sec_title">Processing of augmin GST-N-clamp-MT cryo-EM data</h3>
<p id="Par79">Initial steps of cryo-EM data processing were performed as done as for the wild-type data set. For particle picking in crYOLO, the model trained with the wild-type data was used. For the GST-N-clamp dataset, 348,721 MT segments at 80 Å spacing were extracted in 600 px boxes, rescaled to 5.01 Å/px and subjected to 2D classification. 335,829 retained MT segments were classified by protofilament (PF) number through supervised 3D classification, as described previously<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, with 13PF (175,657 segments) and 14PF (98,204 particles) as the most abundant MT classes.</p>
<p id="Par80">13 PF MTs were used for further processing using the MiRP pipeline (v2)<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Following seam correction, 141,148 MT segments were extracted at full spatial resolution (1.069 Å/px, 432 px box). Extracted MT segments were subjected to refinement with C1 symmetry using parameters outlined in the MiRP manual<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, employing a loose mask with a z-length of 30% of the MT and the final 3D class of the MiRP pipeline before seam correction as reference. Afterwards, the outputs of the particle refinement runs were used as references.</p>
<p id="Par81">From here on, processing was performed in RELION 3.0, unless mentioned otherwise. Successive CTF refinement and Bayesian polishing (each performed twice) interspersed with 3D refinements (applying helical symmetry) were applied, followed by an additional round of C1 refinement. Afterwards, MT segments were symmetry expanded after the aforementioned C1 refinement using the relion_particle_symmetry_expand command, with a 13-fold expansion using a rise of 9.521 Å and a twist of -27.657 Å, yielding 1,834,924 MT segments. Expanded MT segments were subsequently subjected to C1 refinement using the same parameters as the initial refinement, reaching 3.6 Å, resolution and transferred to CryoSPARC 4.4.1<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup> for further processing.</p>
<p id="Par82">In CryoSPARC, MT segments were classified into 4 classes using a soft focus mask covering decorating protein density opposite the seam. One class containing well-defined decorating density was selected (457,096 MT segments) and MT segments were subjected to non-uniform homogeneous refinement (C1 symmetry) with local angular sampling, using a solvent mask covering 2×3 tubulin monomers and the central decorating density. The reconstruction, reaching 3.9 Å and was filtered locally. Using the pyEM package<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>, the particle set was subsequently transferred back to RELION 3.1, and particles were recentered on the augmin N-clamp-MT interface in 256-pixel box at full spatial resolution. To retain the particles in a polished state, they were recentered using a signal subtraction job without actual signal subtraction, by providing a wide mask around the MT<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup>. These particles were reconstructed to generate a 3D reference and subsequently, a 3D refinement was performed with a mask covering 2 × 5 tubulins and the central decorating density. The resulting map was then submitted to 3D classification without alignment in RELION 5.0, with Blush regularization applied<sup><a href="#CR92" class="usa-link" aria-describedby="CR92">92</a></sup>. The class with most well-defined N-clamp density was selected, containing 117,870 particles. Finally, the particles were subjected to local refinement and filtered to local resolution. Details of data processing are summarized in Supplementary Table <a href="#MOESM1" class="usa-link">9</a>.</p></section><section id="Sec36"><h3 class="pmc_sec_title">Analysis of cryo-EM reconstructions</h3>
<p id="Par83">All rigid-body docking was performed using the fitmap command in ChimeraX (1.3, 1.7.1 and 1.9)<sup><a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup>. The AF2 prediction of <em>D. melanogaster</em> N-clamp was rigid-body docked into the density obtained after 3D classification in CryoSPARC, which had sufficient resolution to distinguish a- and β-tubulin (Supplementary Fig. <a href="#MOESM1" class="usa-link">10b</a>). For the final cryo-EM density (Supplementary Fig. <a href="#MOESM1" class="usa-link">10c</a>), an atomic model for one protofilament of a Taxol-stabilized MT (<a href="https://doi.org/10.2210/pdb6WVR/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDB-6WVR</a>) was rigid-body docked twice into adjacent PFs of the density, along with the AF2 prediction of <em>D. melanogaster</em> augmin N-clamp (CH6 K10-L145). Then, individual chains from the AF2 prediction of porcine α/β-tubulin (Supplementary Table <a href="#MOESM1" class="usa-link">7</a>) were separately aligned to the rigid-body docked bovine MT (<a href="https://doi.org/10.2210/pdb6WVR/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDB-6WVR</a>) by matchmaking in ChimeraX. The full <em>D. melanogaster</em> N-clamp AF2 prediction was placed into the density by matchmaking its Dgt6<sub>HAUS6</sub> CH domain onto the Dgt6<sub>HAUS6</sub> CH domain fitted as described above.</p></section><section id="Sec37"><h3 class="pmc_sec_title">Molecular dynamics simulations</h3>
<p id="Par84">MD simulations were conducted at atomic-level resolution, using the CHARMM36m force field (February 2021) for proteins, the CHARMM TIP3P force field for water, and the standard CHARMM36 force field for ions<sup><a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup>. The parameters for GDP and GTP molecules were generated using CHARMM-GUI v1.7, specifically through the Ligand Reader &amp; Modeler tool<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup>.</p>
<p id="Par85">All simulations were performed using the GROMACS 2024.2 simulation package<sup><a href="#CR96" class="usa-link" aria-describedby="CR96">96</a></sup>. AlphaFold 2.3.2, with default settings, was employed to generate the full-length structures of the α-tubulin (<a href="https://www.ncbi.nlm.nih.gov/protein/P81947" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P81947</a>) and β-tubulin (<a href="https://www.ncbi.nlm.nih.gov/protein/Q6B856" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q6B856</a>) subunits. The predicted structures were aligned using the ChimeraX Matchmaker tool<sup><a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup> to the tubulin dimer resolved in <a href="https://doi.org/10.2210/pdb6WVR/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDB 6WVR</a><sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup>, allowing verification of AlphaFold’s predictions for the resolved regions. The dimers in the PDB structure contained GDP and GTP molecules as well as magnesium ions, which were correctly incorporated into the AF2-predicted structures.</p>
<p id="Par86">To model the MT-bound <em>D. melanogaster</em> augmin N-clamp complex, we utilized a structure comprising four protofilaments, each containing two α-tubulin and two β-tubulin subunits. The augmin N-clamp was placed based on a rigid body fit into the cryo-EM reconstruction of wild-type N-clamp. N- and C-termini of all proteins were modeled as charged residues due to their location in flexible loops exposed to solvent. Additionally, the proximity of N-terminal groups of both the N-clamp and tubulin dimers predicted by AF2 hypothesized that they could contribute to stabilizing or destabilizing the interaction interface in their charged forms. To retain the overall shape of the truncated MT filament without simulating the entire tube, we constructed a system comprising four rows of tubulin dimers. Overall, the system consisted of 8 α-tubulin and 8 β-tubulin molecules, the <em>D. melanogaster</em> augmin N-clamp complex, as well as 8 GDP and GTP molecules, and 8 magnesium ions bound to the tubulins. It also included 298,034 water molecules, 1,026 potassium ions, and 592 chloride ions, totaling 1,016,448 atoms. The system was enclosed in a rectangular simulation box with dimensions 22.97 × 27.07 × 16.57 nm. During the production runs, positional restraints were applied only to the two outer protofilaments, while the two central rows (those directly interacting with the N-clamp) remained unrestrained. To restrain the two outer protofilaments, a force constant of 1000 kJ/mol/nm² was applied in all dimensions to both the side chains and backbone atoms. The periodic boundary conditions were applied in all dimensions and the box size was configured to simulate an infinite MT filament in the z-axis, effectively mimicking a realistic section of a MT filament.</p>
<p id="Par87">Initially, all systems were energy minimized in a vacuum, followed by hydration. To mimic experimental conditions, we added 150 mM potassium chloride and an appropriate number of counter ions to neutralize the extra charges. Subsequently, an equilibration phase was conducted under NpT ensemble conditions. During this phase, all proteins were restrained with force constants of 400 kJ/mol/nm<sup>2</sup> for backbone atoms and 40 kJ/mol/nm<sup>2</sup> for side chains. The V-rescale thermostat maintained the temperature at 303.15 K with a 1.0 ps time constant<sup><a href="#CR98" class="usa-link" aria-describedby="CR98">98</a></sup>. A pressure of 1 atm was regulated using the C-rescale barostat with a time constant of 5.0 ps and an isothermal compressibility of 4.5×10<sup>-5</sup> bar<sup>-1</sup> <sup><a href="#CR99" class="usa-link" aria-describedby="CR99">99</a></sup> and an isotropic pressure-coupling scheme. Neighbor searching utilized the Verlet scheme, updating every 20 steps<sup><a href="#CR100" class="usa-link" aria-describedby="CR100">100</a></sup>. Electrostatic interactions were calculated using the particle mesh Ewald method<sup><a href="#CR101" class="usa-link" aria-describedby="CR101">101</a></sup> with a 0.12 nm grid spacing, a 10<sup>-5</sup> tolerance, and a 1.2 nm cutoff. Three independent simulations were performed by varying the initial atomic velocities. Each simulation was carried out with a 2 fs time step and extended to a total duration of 1000 ns.</p>
<p id="Par88">Analyses were conducted using an in-house tool in combination with MDAnalysis<sup><a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup> for contact, RMSD and RMSF calculations, and standard GROMACS tool (gmx energy) for interaction energy calculations. Interaction energy calculations were used to refine the contact map and distinguish truly interacting residues from those simply nearby, as previously done in refs. <sup><a href="#CR103" class="usa-link" aria-describedby="CR103">103</a>,<a href="#CR104" class="usa-link" aria-describedby="CR104">104</a></sup>. Visualization snapshots and video were generated using VMD<sup><a href="#CR105" class="usa-link" aria-describedby="CR105">105</a></sup>.</p></section><section id="Sec38"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par89">Protein band intensities on immunoblots or Coomassie blue-stained gels were quantified using Fiji software (v2.14.0)<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. Signal intensity measurements using the line plot tool or distance measurements of immunofluorescence images were performed on maximum intensity-projected images using Fiji software<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>.</p>
<p id="Par90">Data plotting and statistical analysis were done using GraphPad Prism (Version 10.0). Data normality was assessed using the Shapiro–Wilk test. Subsequently, data were analyzed using a two-tailed <em>t</em> test, Mann–Whitney <em>U</em> test or the Kruskal–Wallis test with Dunn’s multiple comparison post-hoc analysis. A significance threshold of <em>p</em> &lt; 0.05 was applied.</p></section><section id="Sec39"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par91">Further information on research design is available in the <a href="#MOESM4" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec40"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373997/bin/41467_2025_63165_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (18MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373997/bin/41467_2025_63165_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63165_MOESM2_ESM.pdf</a><sup> (171.2KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12373997/bin/41467_2025_63165_MOESM3_ESM.mp4" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (3.9MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12373997/bin/41467_2025_63165_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (110.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12373997/bin/41467_2025_63165_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (2.7MB, pdf) </sup>
</div></div></section></section><section id="Sec41"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12373997/bin/41467_2025_63165_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (17.2MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors thank Ursula Jäkle, Annika Hanraths, Nethaji Kuruppu and Ramiro Monge Lozano for help in protein expression and purification. We thank Oliver Gruss, Simone Reber, Luca Troman and Mamata Bangera for discussion and support during cryo-EM data processing, and Anne-Laure Pauleau for technical suggestions. Human <em>HAUS2</em>, <em>HAUS6</em>, <em>HAUS7</em> and <em>HAUS8</em> genes were a gift from Laurence Pelletier. We thank Jens Lüders for the HAUS6 antibody. We further thank Karine Lapouge from the European Molecular Biology Laboratory (EMBL Heidelberg, Germany) Protein Expression and Purification Core Facility (PEPCF) for Mass Photometry. We acknowledge the technical support of the Core Facility for Mass Spectrometry and Proteomics of the Center for Molecular Biology of Heidelberg University (ZMBH). We thank Marcin Luzarowski and Sabine Merker for support with data analysis and deposition. The Core Facility for Mass Spectrometry and Proteomics is funded by the ZMBH and partially funded by the CellNetworks Core Technology Platform (CCTP) of Heidelberg University. The CCTP is funded in part by the Federal Ministry of Education and Research (BMBF) and the Ministry of Science Baden Württemberg within the framework of the Excellence Strategy of the Federal and State Governments of Germany. The purchase of the Orbitrap Tribrid Eclipse used in this study was funded in part by the German Research Foundation (DFG)—project number: 538758380. Moreover, we would like to acknowledge access to the infrastructure and support provided by the Cryo-EM Network at the Heidelberg University (HDcryoNet), which is funded and supported by the German Research Foundation (DFG), the Federal Ministry of Education and Research (BMBF) and the Ministry of Science Baden-Württemberg, among others, within the framework of the Excellence Strategy of the Federal and State Governments of Germany. We also acknowledge the services SDS@hd and bwHPC supported by the Ministry of Science, Research and the Arts; DFG Sch Baden-Württemberg, as well as the German Research Foundation (INST 35/1314-1 FUGG, INST 35/1503-1 FUGG and INST 35/1134-1 FUGG). We thank EMBL IT for computational and data storage support. This work is supported by grants of the Deutsche Forschungsgemeinschaft (DFG) to ES (DFG Schi 295/4-4; DFG SCHI 295/11-1; DFG SCHI 295/9-1 and to SP (DFG PF 963/1-4; DFG PF 963/4-1), to FL (SFB-1638/1-511488495-Z01, SFB/TRR 186, project A1, DFG LO 2821/1-1) and to MK (SFB-1638/1-511488495-Z01). FL acknowledges the computing resources provided by the CSC-IT Center for Science Ltd. (Espoo, Finland) and supported by the state of Baden-Württemberg through bwHPC and the German Research Foundation (DFG) through grant INST 35/1597-1 FUGG. S.P. also acknowledges funding by the Aventis Foundation and the Chica and Heinz Schaller Foundation. M.W. acknowledges funding by the Health + Life Science Alliance Heidelberg Mannheim.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>M.W., G.T., and M.Z. performed cloning, protein expression, protein purification and tubulin co-sedimentation assays. G.T., M.W., and A.N. performed plunge freezing experiments. G.T., B.J.A.V., and M.H. acquired cryo-EM data. G.T., B.J.A.V., and M.W. analyzed cryo-EM data. A.N. performed negative staining and negative stain EM image acquisition and G.T. and M.W. analyzed negative stain EM data. A.S. and S.E. performed experiments in S2 cells. Q.G. and M.Z. performed microscopy and experiments in RPE1 cells. M.K. and F.L. performed and analyzed MD simulations. G.T., B.J.A.V., and W.M. performed AlphaFold2 predictions. M.W., E.S., S. Eustermann and S.P. supervised the experiments. M.W., G.T., B.J.A.V., E.S., and S.P. wrote the manuscript with input from the other authors. All authors discussed the data and gave final approval for publication.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par92"><em>Nature Communications</em> thanks Sarah Harris, James Wakefield, and the other, anonymous, reviewer for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Funding</h2>
<p>Open Access funding enabled and organized by Projekt DEAL.</p></section><section id="notes4"><h2 class="pmc_sec_title">Data availability</h2>
<p>Atomic coordinates and cryo-EM densities have been deposited at the Protein Data Bank and the Electron Microscopy Data Bank under accession codes: PDB <a href="https://doi.org/10.2210/pdb9RPD/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9RPD</a> and <a href="https://www.ebi.ac.uk/emdb/EMD-54161" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-54161</a> (<em>D. melanogaster</em> augmin GST-N-clamp bound to a MT, well-defined subset of particles), <a href="https://www.ebi.ac.uk/emdb/EMD-54174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-54174</a> (<em>D. melanogaster</em> augmin N-clamp bound to a MT), <a href="https://www.ebi.ac.uk/emdb/EMD-52832" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-52832</a> (helical reconstruction of a <em>D. melanogaster</em> N-clamp-decorated MT), <a href="https://www.ebi.ac.uk/emdb/EMD-54160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-54160</a> (<em>D. melanogaster</em> augmin GST-N-clamp bound to a MT). Published structural data used in this article are: PDB <a href="https://doi.org/10.2210/pdb7SQK/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7SQK</a>, PDB <a href="https://doi.org/10.2210/pdb6WVR/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6WVR</a>, PDB <a href="https://doi.org/10.2210/pdb3IZ0/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">3IZ0</a>, PDB <a href="https://doi.org/10.2210/pdb7PT5/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7PT5</a> and PDB <a href="https://doi.org/10.2210/pdb8AT3/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8AT3</a>. The models predicted by AF2 generated in this study have been deposited in the ModelArchive database with the identifiers ma-vxlav [<a href="https://modelarchive.org/doi/10.5452/ma-vxlav" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-vxlav</a>] (<em>X. laevis</em> augmin N-clamp), ma-apjck [<a href="https://modelarchive.org/doi/10.5452/ma-apjck" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-apjck</a>] (human augmin N-clamp), ma-a7krk [<a href="https://modelarchive.org/doi/10.5452/ma-a7krk" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-a7krk</a>] (<em>D. melanogaster</em> augmin N-clamp), ma-u3pr2 [<a href="https://modelarchive.org/doi/10.5452/ma-u3pr2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-u3pr2</a>] (<em>A. thaliana</em> augmin N-clamp), ma-t3gx3 [<a href="https://modelarchive.org/doi/10.5452/ma-t3gx3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-t3gx3</a>] (<em>A. thaliana</em> augmin TII), ma-5p38z [<a href="https://modelarchive.org/doi/10.5452/ma-5p38z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-5p38z</a>] (<em>D. melanogaster</em> augmin TII), ma-8pbts [<a href="https://modelarchive.org/doi/10.5452/ma-8pbts" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-8pbts</a>] (<em>D. melanogaster</em> augmin TIII), ma-7w5lz [<a href="https://modelarchive.org/doi/10.5452/ma-7w5lz" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-7w5lz</a>] (D. melanogaster augmin TII + III(interface)), ma-3drto [<a href="https://modelarchive.org/doi/10.5452/ma-3drto" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-3drto</a>] (<em>D. melanogaster</em> augmin CH6 with tubulin dimer), ma-5rtzo [<a href="https://modelarchive.org/doi/10.5452/ma-5rtzo" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-5rtzo</a>] (human augmin CH6 with tubulin dimer), ma-xcax9 [<a href="https://modelarchive.org/doi/10.5452/ma-xcax9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-xcax9</a>] (<em>A. thaliana</em> augmin CH6 with tubulin dimer), ma-upjzh [<a href="https://modelarchive.org/doi/10.5452/ma-upjzh" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-upjzh</a>] (<em>X. laevis</em> augmin CH6 with tubulin dimer). Mass spectrometry data of purified augmin N-clamp complexes are available from the ProteomeXchange Consortium<sup><a href="#CR106" class="usa-link" aria-describedby="CR106">106</a></sup> via the PRIDE partner repository<sup><a href="#CR107" class="usa-link" aria-describedby="CR107">107</a></sup> with the dataset identifier <a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD060099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD060099</a>. The initial configuration files for the Molecular Dynamics simulations associated with this study and the trajectories have been deposited on ZENODO. They are publicly available at the following address: 10.5281/zenodo.16452427<sup><a href="#CR108" class="usa-link" aria-describedby="CR108">108</a></sup>. The script used for quantitative analysis and plotting of the tubulin co-sedimentation assays is available at <a href="https://github.com/gt-biomodel/tubulin-co-sedimentation-plots" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/gt-biomodel/tubulin-co-sedimentation-plots</a> and at 10.5281/zenodo.16452427<sup><a href="#CR108" class="usa-link" aria-describedby="CR108">108</a></sup>. Source Data are provided as a Source Data file. <a href="#Sec41" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par93">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Martin Würtz, Giulia Tonon.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Martin Würtz, Email: m.wuertz@zmbh.uni-heidelberg.de.</p>
<p>Stefan Pfeffer, Email: s.pfeffer@zmbh.uni-heidelberg.de.</p></section><section id="sec42"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63165-z.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Goshima, G., Mayer, M., Zhang, N., Stuurman, N. &amp; Vale, R. D. Augmin: a protein complex required for centrosome-independent microtubule generation within the spindle. <em>J. Cell Biol.</em><strong>181</strong>, 421–429 (2008).
</cite> [<a href="https://doi.org/10.1083/jcb.200711053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2364697/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18443220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goshima,%20G.,%20Mayer,%20M.,%20Zhang,%20N.,%20Stuurman,%20N.%20&amp;%20Vale,%20R.%20D.%20Augmin:%20a%20protein%20complex%20required%20for%20centrosome-independent%20microtubule%20generation%20within%20the%20spindle.%20J.%20Cell%20Biol.181,%20421%E2%80%93429%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Petry, S., Groen, A. C., Ishihara, K., Mitchison, T. J. &amp; Vale, R. D. Branching microtubule nucleation in Xenopus egg extracts mediated by augmin and TPX2. <em>Cell</em><strong>152</strong>, 768–777 (2013).
</cite> [<a href="https://doi.org/10.1016/j.cell.2012.12.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3680348/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23415226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Petry,%20S.,%20Groen,%20A.%20C.,%20Ishihara,%20K.,%20Mitchison,%20T.%20J.%20&amp;%20Vale,%20R.%20D.%20Branching%20microtubule%20nucleation%20in%20Xenopus%20egg%20extracts%20mediated%20by%20augmin%20and%20TPX2.%20Cell152,%20768%E2%80%93777%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Prosser, S. L. &amp; Pelletier, L. Mitotic spindle assembly in animal cells: a fine balancing act. <em>Nat. Rev. Mol. Cell Biol.</em><strong>18</strong>, 187–201 (2017).
</cite> [<a href="https://doi.org/10.1038/nrm.2016.162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28174430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prosser,%20S.%20L.%20&amp;%20Pelletier,%20L.%20Mitotic%20spindle%20assembly%20in%20animal%20cells:%20a%20fine%20balancing%20act.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.18,%20187%E2%80%93201%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Kiewisz, R. et al. Three-dimensional structure of kinetochore-fibers in human mitotic spindles. <em>Elife</em><strong>11</strong>, e75459 (2022).
</cite> [<a href="https://doi.org/10.7554/eLife.75459" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9365394/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35894209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kiewisz,%20R.%20et%20al.%20Three-dimensional%20structure%20of%20kinetochore-fibers%20in%20human%20mitotic%20spindles.%20Elife11,%20e75459%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Gao, Q. et al. The structure of the γ-TuRC at the microtubule minus end—not just one solution. <em>BioEssays</em><strong>46</strong>, 2400117 (2024).</cite> [<a href="https://doi.org/10.1002/bies.202400117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39044599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20Q.%20et%20al.%20The%20structure%20of%20the%20%CE%B3-TuRC%20at%20the%20microtubule%20minus%20end%E2%80%94not%20just%20one%20solution.%20BioEssays46,%202400117%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Vermeulen, B. J. A. et al. γ-TuRC asymmetry induces local protofilament mismatch at the RanGTP-stimulated microtubule minus end. <em>EMBO J.</em><strong>43</strong>, 2062–2085 (2024).
</cite> [<a href="https://doi.org/10.1038/s44318-024-00087-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11099078/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38600243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vermeulen,%20B.%20J.%20A.%20et%20al.%20%CE%B3-TuRC%20asymmetry%20induces%20local%20protofilament%20mismatch%20at%20the%20RanGTP-stimulated%20microtubule%20minus%20end.%20EMBO%20J.43,%202062%E2%80%932085%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Brito, C., Serna, M., Guerra, P., Llorca, O. &amp; Surrey, T. Transition of human γ-tubulin ring complex into a closed conformation during microtubule nucleation. <em>Science</em><strong>383</strong>, 870–876 (2024).
</cite> [<a href="https://doi.org/10.1126/science.adk6160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38305685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brito,%20C.,%20Serna,%20M.,%20Guerra,%20P.,%20Llorca,%20O.%20&amp;%20Surrey,%20T.%20Transition%20of%20human%20%CE%B3-tubulin%20ring%20complex%20into%20a%20closed%20conformation%20during%20microtubule%20nucleation.%20Science383,%20870%E2%80%93876%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Aher, A., Urnavicius, L., Xue, A., Neselu, K. &amp; Kapoor, T. M. Structure of the γ-tubulin ring complex-capped microtubule. <em>Nat. Struct. Mol. Biol.</em>10.1038/s41594-024-01264-z (2024).
</cite> [<a href="https://doi.org/10.1038/s41594-024-01264-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11257807/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38609661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aher,%20A.,%20Urnavicius,%20L.,%20Xue,%20A.,%20Neselu,%20K.%20&amp;%20Kapoor,%20T.%20M.%20Structure%20of%20the%20%CE%B3-tubulin%20ring%20complex-capped%20microtubule.%20Nat.%20Struct.%20Mol.%20Biol.10.1038/s41594-024-01264-z%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Dendooven, T. et al. Structure of the native γ-tubulin ring complex capping spindle microtubules. <em>Nat. Struct. Mol. Biol.</em>10.1038/s41594-024-01281-y (2024).</cite> [<a href="https://doi.org/10.1038/s41594-024-01281-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11257966/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38609662/" class="usa-link">PubMed</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Alfaro-Aco, R., Thawani, A. &amp; Petry, S. Biochemical reconstitution of branching microtubule nucleation. <em>Elife</em><strong>9</strong>, e49797 (2020).
</cite> [<a href="https://doi.org/10.7554/eLife.49797" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6959992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31933480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alfaro-Aco,%20R.,%20Thawani,%20A.%20&amp;%20Petry,%20S.%20Biochemical%20reconstitution%20of%20branching%20microtubule%20nucleation.%20Elife9,%20e49797%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Tariq, A., Green, L., Jeynes, J. C. G., Soeller, C. &amp; Wakefield, J. G. In vitro reconstitution of branching microtubule nucleation. <em>Elife</em><strong>9</strong>, e49769 (2020).
</cite> [<a href="https://doi.org/10.7554/eLife.49769" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6959987/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31933481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tariq,%20A.,%20Green,%20L.,%20Jeynes,%20J.%20C.%20G.,%20Soeller,%20C.%20&amp;%20Wakefield,%20J.%20G.%20In%20vitro%20reconstitution%20of%20branching%20microtubule%20nucleation.%20Elife9,%20e49769%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Zhang, Y., Hong, X., Hua, S. &amp; Jiang, K. Reconstitution and mechanistic dissection of the human microtubule branching machinery. <em>J. Cell Biol.</em><strong>221</strong>, e202109053 (2022).
</cite> [<a href="https://doi.org/10.1083/jcb.202109053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9129923/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35604367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.,%20Hong,%20X.,%20Hua,%20S.%20&amp;%20Jiang,%20K.%20Reconstitution%20and%20mechanistic%20dissection%20of%20the%20human%20microtubule%20branching%20machinery.%20J.%20Cell%20Biol.221,%20e202109053%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>David, A. F. et al. Augmin accumulation on long-lived microtubules drives amplification and kinetochore-directed growth. <em>J. Cell Biol.</em><strong>218</strong>, 2150–2168 (2019).
</cite> [<a href="https://doi.org/10.1083/jcb.201805044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6605806/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31113824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?David,%20A.%20F.%20et%20al.%20Augmin%20accumulation%20on%20long-lived%20microtubules%20drives%20amplification%20and%20kinetochore-directed%20growth.%20J.%20Cell%20Biol.218,%202150%E2%80%932168%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Almeida, A. C. et al. Augmin-dependent microtubule self-organization drives kinetochore fiber maturation in mammals. <em>Cell Rep.</em><strong>39</strong>, 110610 (2022).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2022.110610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8994134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35385739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Almeida,%20A.%20C.%20et%20al.%20Augmin-dependent%20microtubule%20self-organization%20drives%20kinetochore%20fiber%20maturation%20in%20mammals.%20Cell%20Rep.39,%20110610%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Štimac, V., Koprivec, I., Manenica, M., Simunić, J. &amp; Tolić, I. M. Augmin prevents merotelic attachments by promoting proper arrangement of bridging and kinetochore fibers. <em>Elife</em><strong>11</strong>, e83287 (2022).
</cite> [<a href="https://doi.org/10.7554/eLife.83287" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9640188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36269126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?%C5%A0timac,%20V.,%20Koprivec,%20I.,%20Manenica,%20M.,%20Simuni%C4%87,%20J.%20&amp;%20Toli%C4%87,%20I.%20M.%20Augmin%20prevents%20merotelic%20attachments%20by%20promoting%20proper%20arrangement%20of%20bridging%20and%20kinetochore%20fibers.%20Elife11,%20e83287%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Trupinić, M. et al. The chirality of the mitotic spindle provides a mechanical response to forces and depends on microtubule motors and augmin. <em>Curr. Biol.</em><strong>32</strong>, 2480–2493.e6 (2022).
</cite> [<a href="https://doi.org/10.1016/j.cub.2022.04.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9235856/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35537456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trupini%C4%87,%20M.%20et%20al.%20The%20chirality%20of%20the%20mitotic%20spindle%20provides%20a%20mechanical%20response%20to%20forces%20and%20depends%20on%20microtubule%20motors%20and%20augmin.%20Curr.%20Biol.32,%202480%E2%80%932493.e6%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Lawo, S. et al. HAUS, the 8-subunit human augmin complex, regulates centrosome and spindle integrity. <em>Curr. Biol.</em><strong>19</strong>, 816–826 (2009).
</cite> [<a href="https://doi.org/10.1016/j.cub.2009.04.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19427217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lawo,%20S.%20et%20al.%20HAUS,%20the%208-subunit%20human%20augmin%20complex,%20regulates%20centrosome%20and%20spindle%20integrity.%20Curr.%20Biol.19,%20816%E2%80%93826%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Uehara, R. et al. The augmin complex plays a critical role in spindle microtubule generation for mitotic progression and cytokinesis in human cells. <em>Proc. Natl Acad. Sci. USA</em><strong>106</strong>, 6998–7003 (2009).
</cite> [<a href="https://doi.org/10.1073/pnas.0901587106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2668966/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19369198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Uehara,%20R.%20et%20al.%20The%20augmin%20complex%20plays%20a%20critical%20role%20in%20spindle%20microtubule%20generation%20for%20mitotic%20progression%20and%20cytokinesis%20in%20human%20cells.%20Proc.%20Natl%20Acad.%20Sci.%20USA106,%206998%E2%80%937003%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Wu, T. et al. Mechanisms of minor pole–mediated spindle bipolarization in human oocytes. <em>Science</em><strong>385</strong>, eado1022 (2025).</cite> [<a href="https://doi.org/10.1126/science.ado1022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39172836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20T.%20et%20al.%20Mechanisms%20of%20minor%20pole%E2%80%93mediated%20spindle%20bipolarization%20in%20human%20oocytes.%20Science385,%20eado1022%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Uehara, R. &amp; Goshima, G. Functional central spindle assembly requires de novo microtubule generation in the interchromosomal region during anaphase. <em>J. Cell Biol.</em><strong>191</strong>, 259–267 (2010).
</cite> [<a href="https://doi.org/10.1083/jcb.201004150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2958471/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20937700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Uehara,%20R.%20&amp;%20Goshima,%20G.%20Functional%20central%20spindle%20assembly%20requires%20de%20novo%20microtubule%20generation%20in%20the%20interchromosomal%20region%20during%20anaphase.%20J.%20Cell%20Biol.191,%20259%E2%80%93267%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Weiner, A. T., Thyagarajan, P., Shen, Y. &amp; Rolls, M. M. To nucleate or not, that is the question in neurons. <em>Neurosci. Lett.</em><strong>751</strong>, 135806 (2021).
</cite> [<a href="https://doi.org/10.1016/j.neulet.2021.135806" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8058292/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33705928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weiner,%20A.%20T.,%20Thyagarajan,%20P.,%20Shen,%20Y.%20&amp;%20Rolls,%20M.%20M.%20To%20nucleate%20or%20not,%20that%20is%20the%20question%20in%20neurons.%20Neurosci.%20Lett.751,%20135806%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Sánchez-Huertas, C. &amp; Lüders, J. The augmin connection in the geometry of microtubule networks. <em>Curr. Biol.</em><strong>25</strong>, R294–R299 (2015).
</cite> [<a href="https://doi.org/10.1016/j.cub.2015.02.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25829017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?S%C3%A1nchez-Huertas,%20C.%20&amp;%20L%C3%BCders,%20J.%20The%20augmin%20connection%20in%20the%20geometry%20of%20microtubule%20networks.%20Curr.%20Biol.25,%20R294%E2%80%93R299%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Cunha-Ferreira, I. et al. The HAUS complex is a key regulator of non-centrosomal microtubule organization during neuronal development. <em>Cell Rep.</em><strong>24</strong>, 791–800 (2018).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2018.06.093" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6083040/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30044976/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cunha-Ferreira,%20I.%20et%20al.%20The%20HAUS%20complex%20is%20a%20key%20regulator%20of%20non-centrosomal%20microtubule%20organization%20during%20neuronal%20development.%20Cell%20Rep.24,%20791%E2%80%93800%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Viais, R. et al. Augmin deficiency in neural stem cells causes p53-dependent apoptosis and aborts brain development. <em>Elife</em><strong>10</strong>, e67989 (2021).
</cite> [<a href="https://doi.org/10.7554/eLife.67989" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8456695/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34427181/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Viais,%20R.%20et%20al.%20Augmin%20deficiency%20in%20neural%20stem%20cells%20causes%20p53-dependent%20apoptosis%20and%20aborts%20brain%20development.%20Elife10,%20e67989%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Mukherjee, A. et al. TuRCs and the augmin complex are required for the development of highly branched dendritic arbors in Drosophila. <em>J. Cell Sci.</em><strong>137</strong>, jcs261534 (2024).
</cite> [<a href="https://doi.org/10.1242/jcs.261534" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11128279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38606636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mukherjee,%20A.%20et%20al.%20TuRCs%20and%20the%20augmin%20complex%20are%20required%20for%20the%20development%20of%20highly%20branched%20dendritic%20arbors%20in%20Drosophila.%20J.%20Cell%20Sci.137,%20jcs261534%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Zhang, Y. et al. Augmin complex activity finetunes dendrite morphology through non-centrosomal microtubule nucleation in vivo. <em>J. Cell Sci.</em><strong>137</strong>, jcs261512 (2024).
</cite> [<a href="https://doi.org/10.1242/jcs.261512" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11128282/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38587100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.%20et%20al.%20Augmin%20complex%20activity%20finetunes%20dendrite%20morphology%20through%20non-centrosomal%20microtubule%20nucleation%20in%20vivo.%20J.%20Cell%20Sci.137,%20jcs261512%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Sanchez-Huertas, C. et al. Non-centrosomal nucleation mediated by augmin organizes microtubules in post-mitotic neurons and controls axonal microtubule polarity. <em>Nat. Commun.</em><strong>7</strong>, 12187 (2016).
</cite> [<a href="https://doi.org/10.1038/ncomms12187" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4947180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27405868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanchez-Huertas,%20C.%20et%20al.%20Non-centrosomal%20nucleation%20mediated%20by%20augmin%20organizes%20microtubules%20in%20post-mitotic%20neurons%20and%20controls%20axonal%20microtubule%20polarity.%20Nat.%20Commun.7,%2012187%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Travis, S. M. et al. Integrated model of the vertebrate augmin complex. <em>Nat. Commun.</em><strong>14</strong>, 2072 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-37519-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10102177/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37055408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Travis,%20S.%20M.%20et%20al.%20Integrated%20model%20of%20the%20vertebrate%20augmin%20complex.%20Nat.%20Commun.14,%202072%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Zupa, E. et al. The augmin complex architecture reveals structural insights into microtubule branching. <em>Nat. Commun.</em><strong>13</strong>, 5635 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-33228-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9512787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36163468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zupa,%20E.%20et%20al.%20The%20augmin%20complex%20architecture%20reveals%20structural%20insights%20into%20microtubule%20branching.%20Nat.%20Commun.13,%205635%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Gabel, C. A. et al. Molecular architecture of the augmin complex. <em>Nat. Commun.</em><strong>13</strong>, 5449 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-33227-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9481612/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36114186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gabel,%20C.%20A.%20et%20al.%20Molecular%20architecture%20of%20the%20augmin%20complex.%20Nat.%20Commun.13,%205449%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Manka, S. W. Structural insights into how augmin augments the mitotic spindle. <em>Nat. Commun.</em><strong>14</strong>, 2073 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-37625-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10102323/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37055387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manka,%20S.%20W.%20Structural%20insights%20into%20how%20augmin%20augments%20the%20mitotic%20spindle.%20Nat.%20Commun.14,%202073%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Hsia, K.-C. et al. Reconstitution of the augmin complex provides insights into its architecture and function. <em>Nat. Cell Biol.</em><strong>16</strong>, 852–863 (2014).
</cite> [<a href="https://doi.org/10.1038/ncb3030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4244909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25173975/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hsia,%20K.-C.%20et%20al.%20Reconstitution%20of%20the%20augmin%20complex%20provides%20insights%20into%20its%20architecture%20and%20function.%20Nat.%20Cell%20Biol.16,%20852%E2%80%93863%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Song, J.-G. et al. Mechanism of how augmin directly targets the gamma-tubulin ring complex to microtubules. <em>J. Cell Biol.</em><strong>217</strong>, 2417–2428 (2018).
</cite> [<a href="https://doi.org/10.1083/jcb.201711090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6028527/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29875259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song,%20J.-G.%20et%20al.%20Mechanism%20of%20how%20augmin%20directly%20targets%20the%20gamma-tubulin%20ring%20complex%20to%20microtubules.%20J.%20Cell%20Biol.217,%202417%E2%80%932428%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Chen, J. W. C. et al. Cross-linking mass spectrometry identifies new interfaces of Augmin required to localise the gamma-tubulin ring complex to the mitotic spindle. <em>Biol. Open</em><strong>6</strong>, 654–663 (2017).
</cite> [<a href="https://doi.org/10.1242/bio.022905" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5450317/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28351835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20J.%20W.%20C.%20et%20al.%20Cross-linking%20mass%20spectrometry%20identifies%20new%20interfaces%20of%20Augmin%20required%20to%20localise%20the%20gamma-tubulin%20ring%20complex%20to%20the%20mitotic%20spindle.%20Biol.%20Open6,%20654%E2%80%93663%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Travis, S. M., Mahon, B. P. &amp; Petry, S. How microtubules build the spindle branch by branch. <em>Annu. Rev. Cell Dev. Biol.</em>10.1146/annurev-cellbio-120420-114559 (2022).</cite> [<a href="https://doi.org/10.1146/annurev-cellbio-120420-114559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9619725/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35759800/" class="usa-link">PubMed</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Wu, G. et al. Hice1, a novel microtubule-associated protein required for maintenance of spindle integrity and chromosomal stability in human cells. <em>Mol. Cell Biol.</em><strong>28</strong>, 3652–3662 (2008).
</cite> [<a href="https://doi.org/10.1128/MCB.01923-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2423303/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18362163/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20G.%20et%20al.%20Hice1,%20a%20novel%20microtubule-associated%20protein%20required%20for%20maintenance%20of%20spindle%20integrity%20and%20chromosomal%20stability%20in%20human%20cells.%20Mol.%20Cell%20Biol.28,%203652%E2%80%933662%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Komarova, Y. et al. Mammalian end binding proteins control persistent microtubule growth. <em>J. Cell Biol.</em><strong>184</strong>, 691–706 (2009).
</cite> [<a href="https://doi.org/10.1083/jcb.200807179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2686402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19255245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Komarova,%20Y.%20et%20al.%20Mammalian%20end%20binding%20proteins%20control%20persistent%20microtubule%20growth.%20J.%20Cell%20Biol.184,%20691%E2%80%93706%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Hayashi, I. &amp; Ikura, M. Crystal structure of the amino-terminal microtubule-binding domain of end-binding protein 1 (EB1). <em>J. Biol. Chem.</em><strong>278</strong>, 36430–36434 (2003).
</cite> [<a href="https://doi.org/10.1074/jbc.M305773200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12857735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hayashi,%20I.%20&amp;%20Ikura,%20M.%20Crystal%20structure%20of%20the%20amino-terminal%20microtubule-binding%20domain%20of%20end-binding%20protein%201%20(EB1).%20J.%20Biol.%20Chem.278,%2036430%E2%80%9336434%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Wei, R. R., Al-Bassam, J. &amp; Harrison, S. C. The Ndc80/HEC1 complex is a contact point for kinetochore-microtubule attachment. <em>Nat. Struct. Mol. Biol.</em><strong>14</strong>, 54–59 (2007).
</cite> [<a href="https://doi.org/10.1038/nsmb1186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17195848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei,%20R.%20R.,%20Al-Bassam,%20J.%20&amp;%20Harrison,%20S.%20C.%20The%20Ndc80/HEC1%20complex%20is%20a%20contact%20point%20for%20kinetochore-microtubule%20attachment.%20Nat.%20Struct.%20Mol.%20Biol.14,%2054%E2%80%9359%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Ciferri, C. et al. Implications for kinetochore-microtubule attachment from the structure of an engineered Ndc80 complex. <em>Cell</em><strong>133</strong>, 427–439 (2008).
</cite> [<a href="https://doi.org/10.1016/j.cell.2008.03.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4754795/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18455984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ciferri,%20C.%20et%20al.%20Implications%20for%20kinetochore-microtubule%20attachment%20from%20the%20structure%20of%20an%20engineered%20Ndc80%20complex.%20Cell133,%20427%E2%80%93439%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>van Breugel, M. Rosa e Silva, I. &amp; Andreeva, A. Structural validation and assessment of AlphaFold2 predictions for centrosomal and centriolar proteins and their complexes. <em>Commun. Biol.</em><strong>5</strong>, 312 (2022).</cite> [<a href="https://doi.org/10.1038/s42003-022-03269-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8983713/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35383272/" class="usa-link">PubMed</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Alushin, G. M. et al. Multimodal microtubule binding by the Ndc80 kinetochore complex. <em>Nat. Struct. Mol. Biol.</em><strong>19</strong>, 1161–1167 (2012).
</cite> [<a href="https://doi.org/10.1038/nsmb.2411" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3492541/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23085714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alushin,%20G.%20M.%20et%20al.%20Multimodal%20microtubule%20binding%20by%20the%20Ndc80%20kinetochore%20complex.%20Nat.%20Struct.%20Mol.%20Biol.19,%201161%E2%80%931167%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Buey, R. M. et al. Insights into EB1 structure and the role of its C-terminal domain for discriminating microtubule tips from the lattice. <em>Mol. Biol. Cell</em><strong>22</strong>, 2912–2923 (2011).
</cite> [<a href="https://doi.org/10.1091/mbc.E11-01-0017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3154886/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21737692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buey,%20R.%20M.%20et%20al.%20Insights%20into%20EB1%20structure%20and%20the%20role%20of%20its%20C-terminal%20domain%20for%20discriminating%20microtubule%20tips%20from%20the%20lattice.%20Mol.%20Biol.%20Cell22,%202912%E2%80%932923%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. <em>Nature</em>10.1038/s41586-021-03819-2 (2021).</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Evans, R. et al. Protein complex prediction with AlphaFold-Multimer. <em>bioRxiv</em>10.1101/2021.10.04.463034 (2022).</cite>
</li>
<li id="CR46">
<span class="label">46.</span><cite>Cook, A. D., Manka, S. W., Wang, S., Moores, C. A. &amp; Atherton, J. A microtubule RELION-based pipeline for cryo-EM image processing. <em>J. Struct. Biol.</em><strong>209</strong>, 107402 (2020).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2019.10.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6961209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31610239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cook,%20A.%20D.,%20Manka,%20S.%20W.,%20Wang,%20S.,%20Moores,%20C.%20A.%20&amp;%20Atherton,%20J.%20A%20microtubule%20RELION-based%20pipeline%20for%20cryo-EM%20image%20processing.%20J.%20Struct.%20Biol.209,%20107402%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Kikkawa, M., Okada, Y. &amp; Hirokawa, N. 15 A; Resolution model of the monomeric Kinesin Motor, KIF1A. <em>Cell</em><strong>100</strong>, 241–252 (2000).
</cite> [<a href="https://doi.org/10.1016/s0092-8674(00)81562-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10660047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kikkawa,%20M.,%20Okada,%20Y.%20&amp;%20Hirokawa,%20N.%2015%E2%80%89A;%20Resolution%20model%20of%20the%20monomeric%20Kinesin%20Motor,%20KIF1A.%20Cell100,%20241%E2%80%93252%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Schweizer, N. et al. Sub-centrosomal mapping identifies augmin-γTuRC as part of a centriole-stabilizing scaffold. <em>Nat. Commun.</em><strong>12</strong>, 6042 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-26252-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8519919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34654813/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schweizer,%20N.%20et%20al.%20Sub-centrosomal%20mapping%20identifies%20augmin-%CE%B3TuRC%20as%20part%20of%20a%20centriole-stabilizing%20scaffold.%20Nat.%20Commun.12,%206042%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Ustinova, K., Ruhnow, F., Gili, M. &amp; Surrey, T. Microtubule binding of the human augmin complex is directly controlled by importins and Ran-GTP. <em>J. Cell Sci.</em><strong>136</strong>, jcs261096 (2023).
</cite> [<a href="https://doi.org/10.1242/jcs.261096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10323247/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37357828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ustinova,%20K.,%20Ruhnow,%20F.,%20Gili,%20M.%20&amp;%20Surrey,%20T.%20Microtubule%20binding%20of%20the%20human%20augmin%20complex%20is%20directly%20controlled%20by%20importins%20and%20Ran-GTP.%20J.%20Cell%20Sci.136,%20jcs261096%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Kraus, J., Travis, S. M., King, M. R. &amp; Petry, S. Augmin is a Ran-regulated spindle assembly factor. <em>J. Biol. Chem.</em><strong>299</strong>, 104736 (2023).
</cite> [<a href="https://doi.org/10.1016/j.jbc.2023.104736" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10318467/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37086784/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kraus,%20J.,%20Travis,%20S.%20M.,%20King,%20M.%20R.%20&amp;%20Petry,%20S.%20Augmin%20is%20a%20Ran-regulated%20spindle%20assembly%20factor.%20J.%20Biol.%20Chem.299,%20104736%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Benoit, M. P. M. H., Rao, L., Asenjo, A. B., Gennerich, A. &amp; Sosa, H. Cryo-EM unveils kinesin KIF1A’s processivity mechanism and the impact of its pathogenic variant P305L. <em>Nat. Commun.</em><strong>15</strong>, 5530 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-48720-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11219953/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38956021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Benoit,%20M.%20P.%20M.%20H.,%20Rao,%20L.,%20Asenjo,%20A.%20B.,%20Gennerich,%20A.%20&amp;%20Sosa,%20H.%20Cryo-EM%20unveils%20kinesin%20KIF1A%E2%80%99s%20processivity%20mechanism%20and%20the%20impact%20of%20its%20pathogenic%20variant%20P305L.%20Nat.%20Commun.15,%205530%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Tsai, C. Y. et al. Aurora-A phosphorylates augmin complex component Hice1 protein at an N-terminal serine/threonine cluster to modulate its microtubule binding activity during spindle assembly*. <em>J. Biol. Chem.</em><strong>286</strong>, 30097–30106 (2011).
</cite> [<a href="https://doi.org/10.1074/jbc.M111.266767" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3191049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21705324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsai,%20C.%20Y.%20et%20al.%20Aurora-A%20phosphorylates%20augmin%20complex%20component%20Hice1%20protein%20at%20an%20N-terminal%20serine/threonine%20cluster%20to%20modulate%20its%20microtubule%20binding%20activity%20during%20spindle%20assembly*.%20J.%20Biol.%20Chem.286,%2030097%E2%80%9330106%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Johmura, Y. et al. Regulation of microtubule-based microtubule nucleation by mammalian polo-like kinase 1. <em>Proc. Natl Acad. Sci.</em><strong>108</strong>, 11446 LP–11411451 (2011).
</cite> [<a href="https://doi.org/10.1073/pnas.1106223108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3136274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21690413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johmura,%20Y.%20et%20al.%20Regulation%20of%20microtubule-based%20microtubule%20nucleation%20by%20mammalian%20polo-like%20kinase%201.%20Proc.%20Natl%20Acad.%20Sci.108,%2011446%20LP%E2%80%9311411451%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Romeiro Motta, M. et al. The cell cycle controls spindle architecture in <em>Arabidopsis</em> by activating the augmin pathway. <em>Dev. Cell</em><strong>59</strong>, 2947–2961.e9 (2024).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2024.08.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39191252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Romeiro%20Motta,%20M.%20et%20al.%20The%20cell%20cycle%20controls%20spindle%20architecture%20in%20Arabidopsis%20by%20activating%20the%20augmin%20pathway.%20Dev.%20Cell59,%202947%E2%80%932961.e9%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Bandi, S., Singh, S. &amp; Mallela, K. Structural stability of tandem calponin-homology (CH) domains originates from their C-terminal CH2 domain. <em>Biophys. J.</em><strong>104</strong>, 396a (2013).23442861
</cite> [<a href="https://scholar.google.com/scholar_lookup?Bandi,%20S.,%20Singh,%20S.%20&amp;%20Mallela,%20K.%20Structural%20stability%20of%20tandem%20calponin-homology%20(CH)%20domains%20originates%20from%20their%20C-terminal%20CH2%20domain.%20Biophys.%20J.104,%20396a%20(2013).23442861" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Harris, A. R. et al. Steric regulation of tandem calponin homology domain actin-binding affinity. <em>Mol. Biol. Cell</em><strong>30</strong>, 3112–3122 (2019).
</cite> [<a href="https://doi.org/10.1091/mbc.E19-06-0317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6938246/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31693446/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harris,%20A.%20R.%20et%20al.%20Steric%20regulation%20of%20tandem%20calponin%20homology%20domain%20actin-binding%20affinity.%20Mol.%20Biol.%20Cell30,%203112%E2%80%933122%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Yagi, N., Fujita, S. &amp; Nakamura, M. Plant microtubule nucleating apparatus and its potential signaling pathway. <em>Curr. Opin. Plant Biol.</em><strong>82</strong>, 102624 (2024).
</cite> [<a href="https://doi.org/10.1016/j.pbi.2024.102624" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39232346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yagi,%20N.,%20Fujita,%20S.%20&amp;%20Nakamura,%20M.%20Plant%20microtubule%20nucleating%20apparatus%20and%20its%20potential%20signaling%20pathway.%20Curr.%20Opin.%20Plant%20Biol.82,%20102624%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Verma, V. &amp; Maresca, T. J. Direct observation of branching MT nucleation in living animal cells. <em>J. Cell Biol.</em><strong>218</strong>, 2829–2840 (2019).
</cite> [<a href="https://doi.org/10.1083/jcb.201904114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6719462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31340987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Verma,%20V.%20&amp;%20Maresca,%20T.%20J.%20Direct%20observation%20of%20branching%20MT%20nucleation%20in%20living%20animal%20cells.%20J.%20Cell%20Biol.218,%202829%E2%80%932840%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Schmidt-Marcec, S. et al. The microtubule-nucleating factor MACERATOR tethers AUGMIN7 to microtubules and governs phragmoplast architecture. <em>Plant Cell</em><strong>36</strong>, 1072–1097 (2024).
</cite> [<a href="https://doi.org/10.1093/plcell/koad304" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11181950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38079222/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schmidt-Marcec,%20S.%20et%20al.%20The%20microtubule-nucleating%20factor%20MACERATOR%20tethers%20AUGMIN7%20to%20microtubules%20and%20governs%20phragmoplast%20architecture.%20Plant%20Cell36,%201072%E2%80%931097%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Cheeseman, I. M., Chappie, J. S., Wilson-Kubalek, E. M. &amp; Desai, A. The conserved KMN network constitutes the core microtubule-binding site of the kinetochore. <em>Cell</em><strong>127</strong>, 983–997 (2006).
</cite> [<a href="https://doi.org/10.1016/j.cell.2006.09.039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17129783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheeseman,%20I.%20M.,%20Chappie,%20J.%20S.,%20Wilson-Kubalek,%20E.%20M.%20&amp;%20Desai,%20A.%20The%20conserved%20KMN%20network%20constitutes%20the%20core%20microtubule-binding%20site%20of%20the%20kinetochore.%20Cell127,%20983%E2%80%93997%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Muir, K. W. et al. Structural mechanism of outer kinetochore Dam1-Ndc80 complex assembly on microtubules. <em>Science</em><strong>382</strong>, 1184–1190 (2023).
</cite> [<a href="https://doi.org/10.1126/science.adj8736" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7615550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38060647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Muir,%20K.%20W.%20et%20al.%20Structural%20mechanism%20of%20outer%20kinetochore%20Dam1-Ndc80%20complex%20assembly%20on%20microtubules.%20Science382,%201184%E2%80%931190%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>King, M. R. &amp; Petry, S. Phase separation of TPX2 enhances and spatially coordinates microtubule nucleation. <em>Nat. Commun.</em><strong>11</strong>, 270 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-019-14087-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6959270/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31937751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?King,%20M.%20R.%20&amp;%20Petry,%20S.%20Phase%20separation%20of%20TPX2%20enhances%20and%20spatially%20coordinates%20microtubule%20nucleation.%20Nat.%20Commun.11,%20270%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Valdez, V. A., Ma, M., Gouveia, B., Zhang, R. &amp; Petry, S. HURP facilitates spindle assembly by stabilizing microtubules and working synergistically with TPX2. <em>Nat. Commun.</em><strong>15</strong>, 9689 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-53630-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11549357/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39516491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Valdez,%20V.%20A.,%20Ma,%20M.,%20Gouveia,%20B.,%20Zhang,%20R.%20&amp;%20Petry,%20S.%20HURP%20facilitates%20spindle%20assembly%20by%20stabilizing%20microtubules%20and%20working%20synergistically%20with%20TPX2.%20Nat.%20Commun.15,%209689%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Niedzialkowska, E. et al. Chromosomal passenger complex condensates generate parallel microtubule bundles in vitro. <em>J. Biol. Chem.</em><strong>300</strong>, 105669 (2024).
</cite> [<a href="https://doi.org/10.1016/j.jbc.2024.105669" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10876603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38272221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Niedzialkowska,%20E.%20et%20al.%20Chromosomal%20passenger%20complex%20condensates%20generate%20parallel%20microtubule%20bundles%20in%20vitro.%20J.%20Biol.%20Chem.300,%20105669%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Subramanian, R. et al. Insights into Antiparallel Microtubule Crosslinking by PRC1, a Conserved Nonmotor Microtubule Binding Protein. <em>Cell</em><strong>142</strong>, 433–443 (2010).
</cite> [<a href="https://doi.org/10.1016/j.cell.2010.07.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2966277/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20691902/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Subramanian,%20R.%20et%20al.%20Insights%20into%20Antiparallel%20Microtubule%20Crosslinking%20by%20PRC1,%20a%20Conserved%20Nonmotor%20Microtubule%20Binding%20Protein.%20Cell142,%20433%E2%80%93443%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Zhang, R., Roostalu, J., Surrey, T. &amp; Nogales, E. Structural insight into TPX2-stimulated microtubule assembly. <em>Elife</em><strong>6</strong>, e30959 (2017).
</cite> [<a href="https://doi.org/10.7554/eLife.30959" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5679754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29120325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20R.,%20Roostalu,%20J.,%20Surrey,%20T.%20&amp;%20Nogales,%20E.%20Structural%20insight%20into%20TPX2-stimulated%20microtubule%20assembly.%20Elife6,%20e30959%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Liu, P. et al. Insights into the assembly and activation of the microtubule nucleator γ-TuRC. <em>Nature</em> (2019) 10.1038/s41586-019-1896-6 (2019).</cite> [<a href="https://doi.org/10.1038/s41586-019-1896-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31856152/" class="usa-link">PubMed</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Livak, K. J. &amp; Schmittgen, T. D. Analysis of relative gene expression data using real-time quantitative PCR and the 2 − ΔΔCT method. <em>Methods</em><strong>25</strong>, 402–408 (2001).
</cite> [<a href="https://doi.org/10.1006/meth.2001.1262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11846609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Livak,%20K.%20J.%20&amp;%20Schmittgen,%20T.%20D.%20Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202%E2%80%89%E2%88%92%E2%80%89%CE%94%CE%94CT%20method.%20Methods25,%20402%E2%80%93408%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. <em>Nat. Methods</em><strong>9</strong>, 676 (2012).
</cite> [<a href="https://doi.org/10.1038/nmeth.2019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3855844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schindelin,%20J.%20et%20al.%20Fiji:%20an%20open-source%20platform%20for%20biological-image%20analysis.%20Nat.%20Methods9,%20676%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Lecland, N. &amp; Lüders, J. The dynamics of microtubule minus ends in the human mitotic spindle. <em>Nat. Cell Biol.</em><strong>16</strong>, 770–778 (2014).
</cite> [<a href="https://doi.org/10.1038/ncb2996" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24976384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lecland,%20N.%20&amp;%20L%C3%BCders,%20J.%20The%20dynamics%20of%20microtubule%20minus%20ends%20in%20the%20human%20mitotic%20spindle.%20Nat.%20Cell%20Biol.16,%20770%E2%80%93778%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Shevchenko, A., Tomas, H., Havli, J., Olsen, J. V. &amp; Mann, M. In-gel digestion for mass spectrometric characterization of proteins and proteomes. <em>Nat. Protoc.</em><strong>1</strong>, 2856–2860 (2006).
</cite> [<a href="https://doi.org/10.1038/nprot.2006.468" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17406544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shevchenko,%20A.,%20Tomas,%20H.,%20Havli,%20J.,%20Olsen,%20J.%20V.%20&amp;%20Mann,%20M.%20In-gel%20digestion%20for%20mass%20spectrometric%20characterization%20of%20proteins%20and%20proteomes.%20Nat.%20Protoc.1,%202856%E2%80%932860%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Eden, M. et al. Mst4, a novel cardiac STRIPAK complex–associated kinase, regulates cardiomyocyte growth and survival and is upregulated in human cardiomyopathy. <em>J. Biol. Chem.</em><strong>300</strong>, 107255 (2024).
</cite> [<a href="https://doi.org/10.1016/j.jbc.2024.107255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11087964/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38579991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eden,%20M.%20et%20al.%20Mst4,%20a%20novel%20cardiac%20STRIPAK%20complex%E2%80%93associated%20kinase,%20regulates%20cardiomyocyte%20growth%20and%20survival%20and%20is%20upregulated%20in%20human%20cardiomyopathy.%20J.%20Biol.%20Chem.300,%20107255%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Eng, J. K., McCormack, A. L. &amp; Yates, J. R. An approach to correlate tandem mass spectral data of peptides with amino acid sequences in a protein database. <em>J. Am. Soc. Mass Spectrom.</em><strong>5</strong>, 976–989 (1994).
</cite> [<a href="https://doi.org/10.1016/1044-0305(94)80016-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24226387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eng,%20J.%20K.,%20McCormack,%20A.%20L.%20&amp;%20Yates,%20J.%20R.%20An%20approach%20to%20correlate%20tandem%20mass%20spectral%20data%20of%20peptides%20with%20amino%20acid%20sequences%20in%20a%20protein%20database.%20J.%20Am.%20Soc.%20Mass%20Spectrom.5,%20976%E2%80%93989%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Cox, J. &amp; Mann, M. MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. <em>Nat. Biotechnol.</em><strong>26</strong>, 1367–1372 (2008).
</cite> [<a href="https://doi.org/10.1038/nbt.1511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19029910/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cox,%20J.%20&amp;%20Mann,%20M.%20MaxQuant%20enables%20high%20peptide%20identification%20rates,%20individualized%20p.p.b.-range%20mass%20accuracies%20and%20proteome-wide%20protein%20quantification.%20Nat.%20Biotechnol.26,%201367%E2%80%931372%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. <em>Nat. Methods</em><strong>17</strong>, 261–272 (2020).
</cite> [<a href="https://doi.org/10.1038/s41592-019-0686-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7056644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32015543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Virtanen,%20P.%20et%20al.%20SciPy%201.0:%20fundamental%20algorithms%20for%20scientific%20computing%20in%20Python.%20Nat.%20Methods17,%20261%E2%80%93272%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Hunter, J. D. Matplotlib: A 2D graphics environment. <em>Comput Sci. Eng.</em><strong>9</strong>, 90–95 (2007).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hunter,%20J.%20D.%20Matplotlib:%20A%202D%20graphics%20environment.%20Comput%20Sci.%20Eng.9,%2090%E2%80%9395%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Elfmann, C. &amp; Stülke, J. PAE viewer: a webserver for the interactive visualization of the predicted aligned error for multimer structure predictions and crosslinks. <em>Nucleic Acids Res.</em><strong>51</strong>, W404–W410 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkad350" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10320053/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37140053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Elfmann,%20C.%20&amp;%20St%C3%BClke,%20J.%20PAE%20viewer:%20a%20webserver%20for%20the%20interactive%20visualization%20of%20the%20predicted%20aligned%20error%20for%20multimer%20structure%20predictions%20and%20crosslinks.%20Nucleic%20Acids%20Res.51,%20W404%E2%80%93W410%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Katoh, K., Misawa, K., Kuma, K. &amp; Miyata, T. MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform. <em>Nucleic Acids Res.</em><strong>30</strong>, 3059–3066 (2002).
</cite> [<a href="https://doi.org/10.1093/nar/gkf436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC135756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12136088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katoh,%20K.,%20Misawa,%20K.,%20Kuma,%20K.%20&amp;%20Miyata,%20T.%20MAFFT:%20a%20novel%20method%20for%20rapid%20multiple%20sequence%20alignment%20based%20on%20fast%20Fourier%20transform.%20Nucleic%20Acids%20Res.30,%203059%E2%80%933066%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Waterhouse, A. M., Procter, J. B., Martin, D. M. A., Clamp, M. &amp; Barton, G. J. Jalview Version 2—a multiple sequence alignment editor and analysis workbench. <em>Bioinformatics</em><strong>25</strong>, 1189–1191 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2672624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19151095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Waterhouse,%20A.%20M.,%20Procter,%20J.%20B.,%20Martin,%20D.%20M.%20A.,%20Clamp,%20M.%20&amp;%20Barton,%20G.%20J.%20Jalview%20Version%202%E2%80%94a%20multiple%20sequence%20alignment%20editor%20and%20analysis%20workbench.%20Bioinformatics25,%201189%E2%80%931191%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Yariv, B. et al. Using evolutionary data to make sense of macromolecules with a “face-lifted” ConSurf. <em>Protein Sci.</em><strong>32</strong>, e4582 (2023).
</cite> [<a href="https://doi.org/10.1002/pro.4582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9942591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36718848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yariv,%20B.%20et%20al.%20Using%20evolutionary%20data%20to%20make%20sense%20of%20macromolecules%20with%20a%20%E2%80%9Cface-lifted%E2%80%9D%20ConSurf.%20Protein%20Sci.32,%20e4582%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Madeira, F. et al. The EMBL-EBI Job Dispatcher sequence analysis tools framework in 2024. <em>Nucleic Acids Res.</em><strong>52</strong>, W521–W525 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkae241" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11223882/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38597606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Madeira,%20F.%20et%20al.%20The%20EMBL-EBI%20Job%20Dispatcher%20sequence%20analysis%20tools%20framework%20in%202024.%20Nucleic%20Acids%20Res.52,%20W521%E2%80%93W525%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Pettersen, E. F. et al. UCSF Chimera—A visualization system for exploratory research and analysis. <em>J. Comput. Chem.</em><strong>25</strong>, 1605–1612 (2004).
</cite> [<a href="https://doi.org/10.1002/jcc.20084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15264254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pettersen,%20E.%20F.%20et%20al.%20UCSF%20Chimera%E2%80%94A%20visualization%20system%20for%20exploratory%20research%20and%20analysis.%20J.%20Comput.%20Chem.25,%201605%E2%80%931612%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Rodrigues, J. P. G. L. M., Teixeira, J. M. C., Trellet, M. &amp; Bonvin, A. M. J. J. pdb-tools: a swiss army knife for molecular structures. <em>F1000Res</em><strong>7</strong>, 1961 (2018).
</cite> [<a href="https://doi.org/10.12688/f1000research.17456.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6343223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30705752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rodrigues,%20J.%20P.%20G.%20L.%20M.,%20Teixeira,%20J.%20M.%20C.,%20Trellet,%20M.%20&amp;%20Bonvin,%20A.%20M.%20J.%20J.%20pdb-tools:%20a%20swiss%20army%20knife%20for%20molecular%20structures.%20F1000Res7,%201961%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Zivanov, J. et al. New tools for automated high-resolution cryo-EM structure determination in RELION-3. <em>Elife</em><strong>7</strong>, e42166 (2018).
</cite> [<a href="https://doi.org/10.7554/eLife.42166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6250425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30412051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zivanov,%20J.%20et%20al.%20New%20tools%20for%20automated%20high-resolution%20cryo-EM%20structure%20determination%20in%20RELION-3.%20Elife7,%20e42166%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Scheres, S. H. W. RELION: implementation of a Bayesian approach to cryo-EM structure determination. <em>J. Struct. Biol.</em><strong>180</strong>, 519–530 (2012).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2012.09.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3690530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23000701/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scheres,%20S.%20H.%20W.%20RELION:%20implementation%20of%20a%20Bayesian%20approach%20to%20cryo-EM%20structure%20determination.%20J.%20Struct.%20Biol.180,%20519%E2%80%93530%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Zhang, K. Gctf: Real-time C. T. F. determination and correction. <em>J. Struct. Biol.</em><strong>193</strong>, 1–12 (2016).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2015.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4711343/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26592709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20K.%20Gctf:%20Real-time%20C.%20T.%20F.%20determination%20and%20correction.%20J.%20Struct.%20Biol.193,%201%E2%80%9312%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Zheng, S. Q. et al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. <em>Nat. Methods</em><strong>14</strong>, 331–332 (2017).
</cite> [<a href="https://doi.org/10.1038/nmeth.4193" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5494038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28250466/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20S.%20Q.%20et%20al.%20MotionCor2:%20anisotropic%20correction%20of%20beam-induced%20motion%20for%20improved%20cryo-electron%20microscopy.%20Nat.%20Methods14,%20331%E2%80%93332%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Wagner, T. et al. SPHIRE-crYOLO is a fast and accurate fully automated particle picker for cryo-EM. <em>Commun. Biol.</em><strong>2</strong>, 218 (2019).
</cite> [<a href="https://doi.org/10.1038/s42003-019-0437-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6584505/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31240256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wagner,%20T.%20et%20al.%20SPHIRE-crYOLO%20is%20a%20fast%20and%20accurate%20fully%20automated%20particle%20picker%20for%20cryo-EM.%20Commun.%20Biol.2,%20218%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Punjani, A., Rubinstein, J. L., Fleet, D. J. &amp; Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. <em>Nat. Methods</em><strong>14</strong>, 290–296 (2017).
</cite> [<a href="https://doi.org/10.1038/nmeth.4169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28165473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Punjani,%20A.,%20Rubinstein,%20J.%20L.,%20Fleet,%20D.%20J.%20&amp;%20Brubaker,%20M.%20A.%20cryoSPARC:%20algorithms%20for%20rapid%20unsupervised%20cryo-EM%20structure%20determination.%20Nat.%20Methods14,%20290%E2%80%93296%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Asarnow D.; Palovcak, E. C. Y. asarnow/pyem: UCSF pyem v0.5. 10.1101/2023.06.09.544258 (2019).</cite>
</li>
<li id="CR91">
<span class="label">91.</span><cite>Atherton, J. et al. Microtubule association induces a Mg-free apo-like ADP pre-release conformation in kinesin-1 that is unaffected by its autoinhibitory tail. <em>bioRxiv</em>10.1101/2024.11.11.622991(2024).</cite> [<a href="https://doi.org/10.1038/s41467-025-61498-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12228791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40617823/" class="usa-link">PubMed</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Kimanius, D. et al. Data-driven regularization lowers the size barrier of cryo-EM structure determination. <em>Nat. Methods</em><strong>21</strong>, 1216–1221 (2024).
</cite> [<a href="https://doi.org/10.1038/s41592-024-02304-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11239489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38862790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kimanius,%20D.%20et%20al.%20Data-driven%20regularization%20lowers%20the%20size%20barrier%20of%20cryo-EM%20structure%20determination.%20Nat.%20Methods21,%201216%E2%80%931221%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Meng, E. C. et al. UCSF ChimeraX: Tools for structure building and analysis. <em>Protein Sci.</em><strong>32</strong>, e4792 (2023).
</cite> [<a href="https://doi.org/10.1002/pro.4792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10588335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37774136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meng,%20E.%20C.%20et%20al.%20UCSF%20ChimeraX:%20Tools%20for%20structure%20building%20and%20analysis.%20Protein%20Sci.32,%20e4792%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Huang, J. et al. CHARMM36m: an improved force field for folded and intrinsically disordered proteins. <em>Nat. Methods</em><strong>14</strong>, 71–73 (2017).
</cite> [<a href="https://doi.org/10.1038/nmeth.4067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5199616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27819658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20J.%20et%20al.%20CHARMM36m:%20an%20improved%20force%20field%20for%20folded%20and%20intrinsically%20disordered%20proteins.%20Nat.%20Methods14,%2071%E2%80%9373%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Kim, S. et al. CHARMM-GUI ligand reader and modeler for CHARMM force field generation of small molecules. <em>J. Comput. Chem.</em><strong>38</strong>, 1879–1886 (2017).
</cite> [<a href="https://doi.org/10.1002/jcc.24829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5488718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28497616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20S.%20et%20al.%20CHARMM-GUI%20ligand%20reader%20and%20modeler%20for%20CHARMM%20force%20field%20generation%20of%20small%20molecules.%20J.%20Comput.%20Chem.38,%201879%E2%80%931886%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Abraham, M. J. et al. GROMACS: High performance molecular simulations through multi-level parallelism from laptops to supercomputers. <em>SoftwareX</em><strong>1–2</strong>, 19–25 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Abraham,%20M.%20J.%20et%20al.%20GROMACS:%20High%20performance%20molecular%20simulations%20through%20multi-level%20parallelism%20from%20laptops%20to%20supercomputers.%20SoftwareX1%E2%80%932,%2019%E2%80%9325%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Debs, G. E., Cha, M., Liu, X., Huehn, A. R. &amp; Sindelar, C. V. Dynamic and asymmetric fluctuations in the microtubule wall captured by high-resolution cryoelectron microscopy. <em>Proc. Natl Acad. Sci.</em><strong>117</strong>, 16976–16984 (2020).
</cite> [<a href="https://doi.org/10.1073/pnas.2001546117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7382274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32636254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Debs,%20G.%20E.,%20Cha,%20M.,%20Liu,%20X.,%20Huehn,%20A.%20R.%20&amp;%20Sindelar,%20C.%20V.%20Dynamic%20and%20asymmetric%20fluctuations%20in%20the%20microtubule%20wall%20captured%20by%20high-resolution%20cryoelectron%20microscopy.%20Proc.%20Natl%20Acad.%20Sci.117,%2016976%E2%80%9316984%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Bussi, G., Donadio, D. &amp; Parrinello, M. Canonical sampling through velocity rescaling. <em>J. Chem. Phys.</em><strong>126</strong>, 014101 (2007).
</cite> [<a href="https://doi.org/10.1063/1.2408420" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17212484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bussi,%20G.,%20Donadio,%20D.%20&amp;%20Parrinello,%20M.%20Canonical%20sampling%20through%20velocity%20rescaling.%20J.%20Chem.%20Phys.126,%20014101%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Bernetti, M. &amp; Bussi, G. Pressure control using stochastic cell rescaling. <em>J. Chem. Phys.</em><strong>153</strong>, 114107 (2020).
</cite> [<a href="https://doi.org/10.1063/5.0020514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32962386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bernetti,%20M.%20&amp;%20Bussi,%20G.%20Pressure%20control%20using%20stochastic%20cell%20rescaling.%20J.%20Chem.%20Phys.153,%20114107%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Verlet, L. Computer ‘experiments’ on classical fluids. I. thermodynamical properties of lennard-jones molecules. <em>Phys. Rev.</em><strong>159</strong>, 98–103 (1967).</cite> [<a href="https://scholar.google.com/scholar_lookup?Verlet,%20L.%20Computer%20%E2%80%98experiments%E2%80%99%20on%20classical%20fluids.%20I.%20thermodynamical%20properties%20of%20lennard-jones%20molecules.%20Phys.%20Rev.159,%2098%E2%80%93103%20(1967)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Essmann, U. et al. A smooth particle mesh Ewald method. <em>J. Chem. Phys.</em><strong>103</strong>, 8577–8593 (1995).</cite> [<a href="https://scholar.google.com/scholar_lookup?Essmann,%20U.%20et%20al.%20A%20smooth%20particle%20mesh%20Ewald%20method.%20J.%20Chem.%20Phys.103,%208577%E2%80%938593%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Michaud-Agrawal, N., Denning, E. J., Woolf, T. B. &amp; Beckstein, O. MDAnalysis: A toolkit for the analysis of molecular dynamics simulations. <em>J. Comput. Chem.</em><strong>32</strong>, 2319–2327 (2011).
</cite> [<a href="https://doi.org/10.1002/jcc.21787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3144279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21500218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Michaud-Agrawal,%20N.,%20Denning,%20E.%20J.,%20Woolf,%20T.%20B.%20&amp;%20Beckstein,%20O.%20MDAnalysis:%20A%20toolkit%20for%20the%20analysis%20of%20molecular%20dynamics%20simulations.%20J.%20Comput.%20Chem.32,%202319%E2%80%932327%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Steringer, J. P. et al. Key steps in unconventional secretion of fibroblast growth factor 2 reconstituted with purified components. <em>Elife</em><strong>6</strong>, e28985 (2017).
</cite> [<a href="https://doi.org/10.7554/eLife.28985" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5601999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28722655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Steringer,%20J.%20P.%20et%20al.%20Key%20steps%20in%20unconventional%20secretion%20of%20fibroblast%20growth%20factor%202%20reconstituted%20with%20purified%20components.%20Elife6,%20e28985%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Legrand, C. et al. The Na,K-ATPase acts upstream of phosphoinositide PI(4,5)P2 facilitating unconventional secretion of Fibroblast Growth Factor 2. <em>Commun. Biol.</em><strong>3</strong>, 141 (2020).
</cite> [<a href="https://doi.org/10.1038/s42003-020-0871-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7096399/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32214225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Legrand,%20C.%20et%20al.%20The%20Na,K-ATPase%20acts%20upstream%20of%20phosphoinositide%20PI(4,5)P2%20facilitating%20unconventional%20secretion%20of%20Fibroblast%20Growth%20Factor%202.%20Commun.%20Biol.3,%20141%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Humphrey, W., Dalke, A. &amp; Schulten, K. VMD: Visual molecular dynamics. <em>J. Mol. Graph</em><strong>14</strong>, 33–38 (1996).
</cite> [<a href="https://doi.org/10.1016/0263-7855(96)00018-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8744570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Humphrey,%20W.,%20Dalke,%20A.%20&amp;%20Schulten,%20K.%20VMD:%20Visual%20molecular%20dynamics.%20J.%20Mol.%20Graph14,%2033%E2%80%9338%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>Deutsch, E. W. et al. The ProteomeXchange consortium in 2020: enabling ‘big data’ approaches in proteomics. <em>Nucleic Acids Res.</em><strong>48</strong>, D1145–D1152 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkz984" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7145525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31686107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deutsch,%20E.%20W.%20et%20al.%20The%20ProteomeXchange%20consortium%20in%202020:%20enabling%20%E2%80%98big%20data%E2%80%99%20approaches%20in%20proteomics.%20Nucleic%20Acids%20Res.48,%20D1145%E2%80%93D1152%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Perez-Riverol, Y. et al. The PRIDE database resources in 2022: a hub for mass spectrometry-based proteomics evidences. <em>Nucleic Acids Res.</em><strong>50</strong>, D543–D552 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkab1038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728295/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34723319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perez-Riverol,%20Y.%20et%20al.%20The%20PRIDE%20database%20resources%20in%202022:%20a%20hub%20for%20mass%20spectrometry-based%20proteomics%20evidences.%20Nucleic%20Acids%20Res.50,%20D543%E2%80%93D552%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite>Lolicato, F. Molecular characterization of the MT-Bound N-Clamp complex interface via atomistic Molecular Dynamics simulations. 10.5281/zenodo.16452427 (2025).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373997/bin/41467_2025_63165_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (18MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373997/bin/41467_2025_63165_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63165_MOESM2_ESM.pdf</a><sup> (171.2KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373997/bin/41467_2025_63165_MOESM3_ESM.mp4" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (3.9MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373997/bin/41467_2025_63165_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (110.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373997/bin/41467_2025_63165_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (2.7MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373997/bin/41467_2025_63165_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (17.2MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Atomic coordinates and cryo-EM densities have been deposited at the Protein Data Bank and the Electron Microscopy Data Bank under accession codes: PDB <a href="https://doi.org/10.2210/pdb9RPD/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9RPD</a> and <a href="https://www.ebi.ac.uk/emdb/EMD-54161" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-54161</a> (<em>D. melanogaster</em> augmin GST-N-clamp bound to a MT, well-defined subset of particles), <a href="https://www.ebi.ac.uk/emdb/EMD-54174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-54174</a> (<em>D. melanogaster</em> augmin N-clamp bound to a MT), <a href="https://www.ebi.ac.uk/emdb/EMD-52832" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-52832</a> (helical reconstruction of a <em>D. melanogaster</em> N-clamp-decorated MT), <a href="https://www.ebi.ac.uk/emdb/EMD-54160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-54160</a> (<em>D. melanogaster</em> augmin GST-N-clamp bound to a MT). Published structural data used in this article are: PDB <a href="https://doi.org/10.2210/pdb7SQK/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7SQK</a>, PDB <a href="https://doi.org/10.2210/pdb6WVR/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6WVR</a>, PDB <a href="https://doi.org/10.2210/pdb3IZ0/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">3IZ0</a>, PDB <a href="https://doi.org/10.2210/pdb7PT5/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7PT5</a> and PDB <a href="https://doi.org/10.2210/pdb8AT3/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8AT3</a>. The models predicted by AF2 generated in this study have been deposited in the ModelArchive database with the identifiers ma-vxlav [<a href="https://modelarchive.org/doi/10.5452/ma-vxlav" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-vxlav</a>] (<em>X. laevis</em> augmin N-clamp), ma-apjck [<a href="https://modelarchive.org/doi/10.5452/ma-apjck" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-apjck</a>] (human augmin N-clamp), ma-a7krk [<a href="https://modelarchive.org/doi/10.5452/ma-a7krk" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-a7krk</a>] (<em>D. melanogaster</em> augmin N-clamp), ma-u3pr2 [<a href="https://modelarchive.org/doi/10.5452/ma-u3pr2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-u3pr2</a>] (<em>A. thaliana</em> augmin N-clamp), ma-t3gx3 [<a href="https://modelarchive.org/doi/10.5452/ma-t3gx3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-t3gx3</a>] (<em>A. thaliana</em> augmin TII), ma-5p38z [<a href="https://modelarchive.org/doi/10.5452/ma-5p38z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-5p38z</a>] (<em>D. melanogaster</em> augmin TII), ma-8pbts [<a href="https://modelarchive.org/doi/10.5452/ma-8pbts" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-8pbts</a>] (<em>D. melanogaster</em> augmin TIII), ma-7w5lz [<a href="https://modelarchive.org/doi/10.5452/ma-7w5lz" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-7w5lz</a>] (D. melanogaster augmin TII + III(interface)), ma-3drto [<a href="https://modelarchive.org/doi/10.5452/ma-3drto" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-3drto</a>] (<em>D. melanogaster</em> augmin CH6 with tubulin dimer), ma-5rtzo [<a href="https://modelarchive.org/doi/10.5452/ma-5rtzo" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-5rtzo</a>] (human augmin CH6 with tubulin dimer), ma-xcax9 [<a href="https://modelarchive.org/doi/10.5452/ma-xcax9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-xcax9</a>] (<em>A. thaliana</em> augmin CH6 with tubulin dimer), ma-upjzh [<a href="https://modelarchive.org/doi/10.5452/ma-upjzh" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://modelarchive.org/doi/10.5452/ma-upjzh</a>] (<em>X. laevis</em> augmin CH6 with tubulin dimer). Mass spectrometry data of purified augmin N-clamp complexes are available from the ProteomeXchange Consortium<sup><a href="#CR106" class="usa-link" aria-describedby="CR106">106</a></sup> via the PRIDE partner repository<sup><a href="#CR107" class="usa-link" aria-describedby="CR107">107</a></sup> with the dataset identifier <a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD060099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD060099</a>. The initial configuration files for the Molecular Dynamics simulations associated with this study and the trajectories have been deposited on ZENODO. They are publicly available at the following address: 10.5281/zenodo.16452427<sup><a href="#CR108" class="usa-link" aria-describedby="CR108">108</a></sup>. The script used for quantitative analysis and plotting of the tubulin co-sedimentation assays is available at <a href="https://github.com/gt-biomodel/tubulin-co-sedimentation-plots" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/gt-biomodel/tubulin-co-sedimentation-plots</a> and at 10.5281/zenodo.16452427<sup><a href="#CR108" class="usa-link" aria-describedby="CR108">108</a></sup>. Source Data are provided as a Source Data file. <a href="#Sec41" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63165-z"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63165.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373997/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373997/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373997%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373997/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373997/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373997/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846850/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373997/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846850/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373997/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373997/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="AONcwKRInKCV54gaYkDd0hDI7UHa9sfNRxPi3rJ1fdZXaWUQtjf5QLjb5jee5VHE">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
