
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            OncoMet: a deep learning framework for the prediction of oncogenic signaling pathways and metastasis in esophageal cancer patients using histopathology images from primary tumors - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532DAA8AF209D3052DAA0057E5C89F.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="transmed">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372372/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Journal of Translational Medicine">
<meta name="citation_title" content="OncoMet: a deep learning framework for the prediction of oncogenic signaling pathways and metastasis in esophageal cancer patients using histopathology images from primary tumors">
<meta name="citation_author" content="Syed Wajid Aalam">
<meta name="citation_author_institution" content="Department of Computer Science, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India">
<meta name="citation_author_institution" content="Centre for Artificial Intelligence, Islamic University of Science and Technology, Awantipora, Kashmir India">
<meta name="citation_author" content="Abdul Basit Ahanger">
<meta name="citation_author_institution" content="Department of Computer Science, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India">
<meta name="citation_author" content="Tabasum Majeed">
<meta name="citation_author_institution" content="Department of Computer Science and Engineering, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India">
<meta name="citation_author" content="Ab Naffi Ahanger">
<meta name="citation_author_institution" content="Department of Computer Science, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India">
<meta name="citation_author" content="Tariq Masoodi">
<meta name="citation_author_institution" content="Canary Oncoceutics Inc, 20E Thomas Road, Suite 2203, Phoenix, AZ USA">
<meta name="citation_author" content="Ajaz A Bhat">
<meta name="citation_author_institution" content="Metabolic and Mendelian Disorders Clinical Research Program, Precision OMICs Research and Translational Science, Sidra Medicine, Doha, Qatar">
<meta name="citation_author" content="Assif Assad">
<meta name="citation_author_institution" content="Centre for Artificial Intelligence, Islamic University of Science and Technology, Awantipora, Kashmir India">
<meta name="citation_author" content="Muzafar Ahmad Macha">
<meta name="citation_author_institution" content="Watson-Crick Centre for Molecular Medicine, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India">
<meta name="citation_author" content="Muzafar Rasool Bhat">
<meta name="citation_author_institution" content="Department of Computer Science, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India">
<meta name="citation_author_institution" content="Centre for Artificial Intelligence, Islamic University of Science and Technology, Awantipora, Kashmir India">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="23">
<meta name="citation_firstpage" content="945">
<meta name="citation_doi" content="10.1186/s12967-025-06914-4">
<meta name="citation_pmid" content="40842012">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372372/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372372/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372372/pdf/12967_2025_Article_6914.pdf">
<meta name="description" content="Despite recent advancements in the diagnosis and prognosis of Esophageal cancer (EC), it remains among the leading causes of cancer-related mortality. Timely and cost-effective diagnosis, particularly in predicting the risk of metastasis and ...">
<meta name="og:title" content="OncoMet: a deep learning framework for the prediction of oncogenic signaling pathways and metastasis in esophageal cancer patients using histopathology images from primary tumors">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Despite recent advancements in the diagnosis and prognosis of Esophageal cancer (EC), it remains among the leading causes of cancer-related mortality. Timely and cost-effective diagnosis, particularly in predicting the risk of metastasis and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372372/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12372372">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12967-025-06914-4"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12967_2025_Article_6914.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372372%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12372372/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12372372/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372372/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-transmed.png" alt="Journal of Translational Medicine logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Journal of Translational Medicine" title="Link to Journal of Translational Medicine" shape="default" href="https://translational-medicine.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">J Transl Med</button></div>. 2025 Aug 21;23:945. doi: <a href="https://doi.org/10.1186/s12967-025-06914-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12967-025-06914-4</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22J%20Transl%20Med%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%20Transl%20Med%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22J%20Transl%20Med%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22J%20Transl%20Med%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>OncoMet: a deep learning framework for the prediction of oncogenic signaling pathways and metastasis in esophageal cancer patients using histopathology images from primary tumors</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aalam%20SW%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Syed Wajid Aalam</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Syed Wajid Aalam</span></h3>
<div class="p">
<sup>1</sup>Department of Computer Science, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India </div>
<div class="p">
<sup>2</sup>Centre for Artificial Intelligence, Islamic University of Science and Technology, Awantipora, Kashmir India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aalam%20SW%22%5BAuthor%5D" class="usa-link"><span class="name western">Syed Wajid Aalam</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahanger%20AB%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Abdul Basit Ahanger</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Abdul Basit Ahanger</span></h3>
<div class="p">
<sup>1</sup>Department of Computer Science, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahanger%20AB%22%5BAuthor%5D" class="usa-link"><span class="name western">Abdul Basit Ahanger</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Majeed%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Tabasum Majeed</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Tabasum Majeed</span></h3>
<div class="p">
<sup>3</sup>Department of Computer Science and Engineering, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Majeed%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tabasum Majeed</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahanger%20AN%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ab Naffi Ahanger</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ab Naffi Ahanger</span></h3>
<div class="p">
<sup>1</sup>Department of Computer Science, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahanger%20AN%22%5BAuthor%5D" class="usa-link"><span class="name western">Ab Naffi Ahanger</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Masoodi%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Tariq Masoodi</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Tariq Masoodi</span></h3>
<div class="p">
<sup>4</sup>Canary Oncoceutics Inc, 20E Thomas Road, Suite 2203, Phoenix, AZ USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Masoodi%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tariq Masoodi</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhat%20AA%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Ajaz A Bhat</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Ajaz A Bhat</span></h3>
<div class="p">
<sup>5</sup>Metabolic and Mendelian Disorders Clinical Research Program, Precision OMICs Research and Translational Science, Sidra Medicine, Doha, Qatar </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhat%20AA%22%5BAuthor%5D" class="usa-link"><span class="name western">Ajaz A Bhat</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Assad%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Assif Assad</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Assif Assad</span></h3>
<div class="p">
<sup>2</sup>Centre for Artificial Intelligence, Islamic University of Science and Technology, Awantipora, Kashmir India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Assad%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Assif Assad</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Macha%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Muzafar Ahmad Macha</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Muzafar Ahmad Macha</span></h3>
<div class="p">
<sup>6</sup>Watson-Crick Centre for Molecular Medicine, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Macha%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Muzafar Ahmad Macha</span></a>
</div>
</div>
<sup>6,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhat%20MR%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Muzafar Rasool Bhat</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Muzafar Rasool Bhat</span></h3>
<div class="p">
<sup>1</sup>Department of Computer Science, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India </div>
<div class="p">
<sup>2</sup>Centre for Artificial Intelligence, Islamic University of Science and Technology, Awantipora, Kashmir India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhat%20MR%22%5BAuthor%5D" class="usa-link"><span class="name western">Muzafar Rasool Bhat</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Computer Science, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India </div>
<div id="Aff2">
<sup>2</sup>Centre for Artificial Intelligence, Islamic University of Science and Technology, Awantipora, Kashmir India </div>
<div id="Aff3">
<sup>3</sup>Department of Computer Science and Engineering, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India </div>
<div id="Aff4">
<sup>4</sup>Canary Oncoceutics Inc, 20E Thomas Road, Suite 2203, Phoenix, AZ USA </div>
<div id="Aff5">
<sup>5</sup>Metabolic and Mendelian Disorders Clinical Research Program, Precision OMICs Research and Translational Science, Sidra Medicine, Doha, Qatar </div>
<div id="Aff6">
<sup>6</sup>Watson-Crick Centre for Molecular Medicine, Islamic University of Science and Technology (IUST), Awantipora, Kashmir India </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 19; Accepted 2025 Jul 21; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12372372  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40842012/" class="usa-link">40842012</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Despite recent advancements in the diagnosis and prognosis of Esophageal cancer (EC), it remains among the leading causes of cancer-related mortality. Timely and cost-effective diagnosis, particularly in predicting the risk of metastasis and identifying the deregulation of oncogenic signaling pathways, could open new frontiers towards precision medicine and targeted therapy of EC. However, current diagnostic practices in identifying metastasis and deregulated oncogenic pathways involve molecular testing, which is time-consuming and costly. Advances in deep learning analysis of digital pathological imagery data offer promising avenues for automating and enhancing cancer diagnosis and risk stratification.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par2">High-resolution H&amp;E-stained diagnostic whole slide images were obtained from the open repository of The Cancer Genome Atlas (TCGA). The WSIs underwent several pre-processing steps, including patching, color normalization and augmentation. A deep learning model was designed and trained on WSI data and tissue-level labels to generate image feature representations for predicting metastatic potential and identifying the deregulation of four major oncogenic signaling pathways, viz. mTOR, PTEN, p53, and PI3K/AKT.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">The proposed model achieved an AUC of 0.92 for predicting metastatic risk and AUCs ranging from 0.64 to 0.92 for the identification of deregulated oncogenic pathways. In a first, we were able to operate the model without the need for exhaustive patch-level annotations, relying instead on slide-level annotations only.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par4">In this work, we highlighted the transformative potential of deep learning in accurately detecting metastasis and identifying deregulated oncogenic pathways from H&amp;E slides using slide-level annotation, thus opening new doors in precision medicine and targeted therapy.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Esophageal cancer, Histopathology, Metastasis, Signaling pathways, Deep learning, Machine learning, Whole slide images, Color normalization, Patching</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par5">Esophageal cancer (EC) poses a significant global health challenge, with approximately 6,04,000 new cases and 5,44,000 deaths annually. It is the sixth leading cause of cancer-related mortality and the eighth most common cancer worldwide [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. EC is highly aggressive, often metastasizing early to organs such as the lungs, pleura, liver, stomach, peritoneum, kidneys, adrenal glands, and bones [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Despite advancements in oncology, EC remains understudied, resulting in low 5-year survival rates of less than 25%, primarily due to loco-regional and distant metastasis [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Alarmingly, up to 50% of EC cases are diagnosed with metastasis at initial presentation. Even superficially located primary tumors can lead to metastatic spread, underscoring the formidable challenge in prognostication and treatment planning [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Esophageal cancer has been on the rise globally, particularly among men, contributing significantly to cancer-related mortality. By 2040, the number of new esophageal cancer cases worldwide is expected to reach nearly 1 million annually [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>].</p>
<p id="Par6">EC consists of two main subtypes: squamous cell carcinoma (ESCC), arising from the squamous cells lining the esophagus, and adenocarcinoma (EAC), originating from glandular cells in the lower esophagus. While early detection and classification of these subtypes are essential, the more significant challenge lies in addressing metastasis, where cancer cells spread to secondary sites. The formation of the secondary cancer sites, known as metastases, is a continuous process that can start early in the growth of the primary tumor. Metastases can also create new lesions in other organs, essentially “metastasizing within metastases” [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>].</p>
<p id="Par7">The heterogeneity of metastatic tumors varying in size, location, age, and cellular composition - poses a formidable obstacle to current treatment modalities. This complexity limits the efficacy of therapeutic interventions and contributes to poor prognoses. Most fatalities in metastatic cancer cases result from organ failure caused by the cumulative burden of widespread metastases or complications related to aggressive treatment strategies [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Hence, an advanced and highly accurate diagnostic system is imperative. Such a system could assist pathologists in predicting metastatic potential during the early stages of cancer, thereby improving patient outcomes and survival rates [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>].</p>
<p id="Par8">Recent studies have explored the use of histological features to predict metastatic involvement in certain cancers, such as lung, renal, and prostate carcinomas. Leveraging deep learning (DL) techniques, these studies achieved area under the curve (AUC) values of up to 0.78, 0.86, and 0.68 for receiver operating characteristic (ROC) analyses [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. While these advancements highlight the potential of artificial intelligence in improving metastasis prediction, none of these approaches have explored signaling pathway identification using histopathological whole-slide images (WSIs) at the slide level. Furthermore, our study introduces a novel method for metastasis prediction that achieves higher accuracy, bridges this gap, and advances the field by offering unique insights into the underlying biological mechanisms.</p>
<p id="Par9">A critical aspect of understanding EC lies in the diverse oncogenic signaling pathways involved in its progression. Key pathways such as mTOR, p53, PTEN, and PI3K/AKT are pivotal in cellular functions like growth, proliferation, differentiation, invasion, and apoptosis. Dysregulation of these pathways contributes significantly to cancer development and metastasis [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. For instance, the mTOR pathway is essential for cell growth and metabolism, and its dysregulation can lead to uncontrolled cell proliferation. The p53 pathway is necessary for DNA repair and apoptosis; mutations in the p53 gene are common in EC and lead to evasion of apoptosis and increased survival of malignant cells. The PTEN pathway acts as a tumor suppressor, and its inactivation can enhance cell survival and growth. Similarly, the PI3K/AKT pathway promotes cell survival and development, and its hyperactivation is frequently observed in various cancers, including EC [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>].</p>
<p id="Par10">Traditional cancer diagnosis and grading have relied on the microscopic analysis of Hematoxylin and Eosin (H&amp;E) stained sections. This process has become increasingly important with the rise in cancer cases and the need for personalized treatments. Pathologists must now examine numerous slides, often with additional stains, and extract various quantitative parameters for grading systems, making the process time-intensive and susceptible to variation [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. To address these challenges, integrating computational science with histopathology through whole-slide scanning systems has opened new avenues for enhancing pathological procedures. Digitizing high-resolution tissue slides allows for efficient study and analysis, potentially increasing metastasis detection’s speed, sensitivity, and consistency [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p>
<p id="Par11">Machine learning and artificial intelligence (AI) algorithms, intense learning models such as Convolutional Neural Networks (CNNs), have demonstrated superior capabilities in diagnosing, prognosing, and classifying malignancies based on medical imaging data [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. These technologies hold promise for automating the initial assessment of images, providing supplementary information to pathologists, and streamlining the diagnostic process.</p>
<p id="Par12">Using histopathology images, we present OncoMet a novel DL framework designed to predict metastasis and identify oncogenic signaling pathways in esophageal cancer patients. OncoMet employs DL classifiers to enable direct slide-level predictions, utilizing unique feature representations of whole slide images (WSIs). By efficiently handling and transforming large-sized WSIs through pre-processing, OncoMet consolidates them into single feature vectors for classification using machine learning classifiers. We adopted a three-step data augmentation strategy for the first time by applying Reinhard normalization three times, each with a different reference image, to introduce greater variability and richness in the training data.</p>
<p id="Par13">The performance and generalizability of the proposed framework are validated through external validation using the TCGA-STAD dataset of gastric cancer patients, providing additional insights into its efficacy. This innovative approach aims to support pathologists in predicting metastasis at early stages and identifying critical oncogenic pathways such as mTOR, p53, PTEN, and PI3K/AKT, ultimately improving patient prognosis and survival outcomes.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Dataset description</h3>
<p id="Par14">For this study, we selected WSIs from 124 patients diagnosed with esophageal carcinoma (TCGA-ESCA) from The Cancer Genome Atlas (TCGA) via the GDC data transfer tool (<a href="https://portal.gdc.cancer.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.gdc.cancer.gov/</a>). These WSIs were captured at x40 magnification, featuring an average pixel density of 1,00,000 × 80,000 pixels and an 8-bit color depth, with each pixel corresponding to a spatial resolution of 0.25 micrometers (µm). The images, provided in.svs format, specifically focused on primary tumors stained with H&amp;E. The dataset was randomly partitioned into an 80:20 ratio, adhering to the Pareto principle, a commonly applied rule of thumb in practice.</p>
<p id="Par15">Additionally, we incorporated diagnostic slides from 20 patients with stomach adenocarcinoma (TCGA-STAD) to validate our method. The clinical and demographic characteristics of both TCGA-ESCA and TCGA-STAD cohorts, including gender distribution, age groups, vital status, and tumor location, are summarized in Table <a href="#Tab1" class="usa-link">1</a>. Notably, esophageal tumors were predominantly located in the lower third (76 cases) and middle third (32 cases) of the esophagus. In comparison, stomach tumors were primarily found in the cardia (4 cases), body (4 cases), and pyloric antrum (12 cases) of the stomach.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Clinical and demographic characteristics of TCGA-ESCA and TCGA-STAD cohorts</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Characteristics</th>
<th align="left" colspan="2" rowspan="1">Cohort</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">TCGA-ESCA</th>
<th align="left" colspan="1" rowspan="1">TCGA-STAD</th>
</tr>
</thead>
<tbody>
<tr><td align="left" colspan="3" rowspan="1">Gender</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Male</td>
<td align="left" colspan="1" rowspan="1">112</td>
<td align="left" colspan="1" rowspan="1">14</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Female</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">6</td>
</tr>
<tr><td align="left" colspan="3" rowspan="1">Age</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> &lt;=60</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> &gt; 60</td>
<td align="left" colspan="1" rowspan="1">114</td>
<td align="left" colspan="1" rowspan="1">18</td>
</tr>
<tr><td align="left" colspan="3" rowspan="1">Vital status</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Dead</td>
<td align="left" colspan="1" rowspan="1">56</td>
<td align="left" colspan="1" rowspan="1">12</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Alive</td>
<td align="left" colspan="1" rowspan="1">68</td>
<td align="left" colspan="1" rowspan="1">8</td>
</tr>
<tr><td align="left" colspan="3" rowspan="1">Location (Esophagus)</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Lower Third</td>
<td align="left" colspan="1" rowspan="1">76</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Middle Third</td>
<td align="left" colspan="1" rowspan="1">32</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Thoracic</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Upper Third</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Unspecified Location</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr><td align="left" colspan="3" rowspan="1">Location (Stomach)</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Cardia</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Central region (Body)</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Pyloric antrum</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">12</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par17">We conducted an integrative molecular analysis using somatic mutation and copy number variation (CNV) data derived from the TCGA-ESCA and TCGA-STAD cohorts to identify patients with potential for distant metastasis. We prioritized known metastasis-associated biomarkers, including Matrix Metalloproteinases (MMPs), vascular endothelial growth factor (VEGF), and E-cadherin (CDH1), based on their well-established roles in promoting tissue invasion, angiogenesis, and epithelial-to-mesenchymal transition (EMT), hallmark processes in metastatic progression.</p>
<p id="Par18">Somatic mutation profiles were downloaded from cBioPortal [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. Recurrently mutated genes were identified (mutated in at least two samples). These included key oncogenes and tumor suppressor genes such as <em>TP53</em>,<em> CDKN2A</em>,<em> FAT1</em>,<em> NOTCH1</em>, and <em>PIK3CA</em>, among others, known to regulate critical cellular pathways involved in tumor progression, EMT, and metastatic spread. We also incorporated CNVs, focusing on amplifications of oncogenes and deletions of tumor suppressors, as these structural variations are often associated with aggressive tumor phenotypes.</p>
<p id="Par19">The complete list of recurrently mutated genes was submitted to Ingenuity Pathway Analysis (IPA) [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. To identify significantly enriched canonical pathways with functional relevance to metastasis. The analysis yielded 10 top-ranked pathways commonly altered in carcinoma cases and implicated in the metastatic process. These included pathways involved in cell cycle dysregulation, apoptosis evasion, PI3K/AKT signaling, and EMT-related cascades. Patients showing alterations in any of these metastasis-associated pathways were labeled as having a high metastatic potential. These molecular labels were subsequently used to guide histopathological image processing and model training, linking morphological features to the presence of metastatic-driving alterations.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Dataset pre-processing</h3>
<p id="Par20">To ensure consistent quality in our dataset, we filtered low-quality whole-slide images from the training and validation sets based on stringent criteria such as visual clarity, image blurring, staining abnormalities, and ink marks. This process refined our collection to 124 diagnostic whole-slide images from the 154 diagnostic WSIs available on the TCGA portal. We allocated 80% of these images for training and the remaining 20% for testing, maintaining the same proportion of metastatic and non-metastatic images as outlined in previous studies [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>–<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>].</p>
<p id="Par21">The selected 124 WSIs were utilized to analyze four oncogenic signaling pathways. To address data imbalance, we supplemented our dataset with additional cases from the TCGA-HNSCC cohort, thereby increasing the representation of minority class cases. This approach ensured that our ML models were trained on a more balanced and representative dataset, crucial for accurately identifying and predicting patterns in minority classes. A balanced dataset reduces the risk of model bias towards the majority class, thereby enhancing the robustness and generalizability of the predictions. The supplemented datasets were then randomly divided into an 80:20 ratio for training and testing the models.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Whole slide image pre-processing</h3>
<p id="Par22">With images captured at various magnification levels, WSIs possess a pyramidal data structure and are exceedingly large [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. To balance storage needs and information retention, we processed the data at a 20x magnification level using an image segmentation tool built on the OpenSlide library [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. This process yielded a total of 3,90,532 tiles. In WSI analysis, tile dimensions and overlap selection are critical parameters. We extracted non-overlapping tiles with dimensions of 512 × 512 pixels. These 512 × 512 tiles were resized to 224 × 224 pixels during the feature extraction phase. Additionally, tiles with more than 90% background coverage were excluded, adhering to the methodology outlined in [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>].</p>
<p id="Par23">Histopathology images are inherently prone to color variations due to fluctuations in staining concentrations, disparities in equipment usage, and inconsistencies during tissue sectioning. When these variations occur within WSIs, they introduce significant variability into the training data, which can profoundly impact the performance of DL models. In medical image analysis, where precision is paramount, addressing these color discrepancies is essential to ensure accurate diagnoses and reliable feature extraction.</p>
<p id="Par24">Several stain normalization techniques have been developed to address color inconsistencies in histopathology images, including Histogram Specification, Structure-Preserving Color Normalization, and the widely used Reinhard Algorithm (RA). Among these, RA has shown strong potential in improving the accuracy and reliability of deep learning models in medical image analysis [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>].</p>
<p id="Par25">Reinhard color normalization is a widely used technique in histopathology image analysis to reduce staining variability across whole-slide images (WSIs). It works by transforming the RGB color space of an image into the Lab color space, which separates luminance (L) from color information (a and b channels). Once the image is converted to Lab color space, Reinhard normalization adjusts the brightness and color channels so that the target image has the same average color and contrast as a chosen reference image. This helps ensure that all slides, no matter where or how they were stained, end up with a similar overall appearance, making it easier for models to learn consistent patterns. After normalization, the image is converted back to the RGB space for further processing. By harmonizing color distributions, Reinhard normalization enhances the consistency of image features extracted by deep learning models and improves the generalizability of predictions across datasets acquired under different staining conditions [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>].</p>
<p id="Par26">In our study, we successfully mitigated color variations within our histopathology dataset by applying Reinhard color normalization, facilitating feature extraction and simplifying subsequent analysis. This approach is invaluable in medical image analysis and DL, where precision and consistency are paramount. By harmonizing the color properties of all images, RA ensures a consistent dataset, ultimately improving the accuracy and reliability of DL models in medical diagnostics.</p></section><section id="Sec6"><h3 class="pmc_sec_title">OncoMet algorithm</h3>
<p id="Par27">OncoMet employs a supervised workflow to achieve cost-effective automated identification of metastasis and oncogenic signaling pathways in esophageal carcinoma. The OncoMet algorithm comprises three distinct phases: (i) The Feature extraction phase is dedicated to effectively characterizing the input data. Each tile of every whole-slide image is passed through the pre-trained ResNet101 network with the top layer removed and the pre-trained parameter set to True. The extracted features are then stored for subsequent analysis. This sub-network is pre-trained on the ImageNet dataset, maintaining consistent parameters during training and testing. Consequently, each input tile yields a feature vector with a substantial dimensionality of 2048. This process generates an N x 2048 feature matrix for each patient slide, where N represents the total number of tumor tiles within the slide. (ii) Feature aggregation: The feature aggregation process consolidates the extracted features from each tile into a cohesive representation for the entire slide. Formally defined by Eqs. <a href="#Equ1" class="usa-link">1</a> and <a href="#Equ2" class="usa-link">2</a>, this step ensures that the high-dimensional feature vectors are effectively summarized, providing a comprehensive overview of the slide’s characteristics. (iii) Classification phase: Following feature aggregation, we shift our focus to classification tasks using a variety of machine learning classifiers, including K-Nearest Neighbors (KNN), Support Vector Machine (SVM), Logistic Regression (LR), Random Forest, AdaBoost, and Decision Tree. These classifiers are employed to make predictions based on the feature vectors derived from the WSIs, enhancing the accuracy and reliability of the diagnostic process.</p>
<p id="Par28">Given a set of feature vectors <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/d2b355283c52/d33e590.gif" loading="lazy" id="d33e590" alt="Inline graphic"></span> of all the tiles in a WSI <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/78b515de073d/d33e596.gif" loading="lazy" id="d33e596" alt="Inline graphic"></span> we calculate the mean <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/dec551cc9736/d33e602.gif" loading="lazy" id="d33e602" alt="Inline graphic"></span>and median <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/229cb3a06487/d33e608.gif" loading="lazy" id="d33e608" alt="Inline graphic"></span> of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/50215e7340da/d33e614.gif" loading="lazy" id="d33e614" alt="Inline graphic"></span> as follows:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/98272c31de41/d33e622.gif" loading="lazy" id="d33e622" alt="graphic file with name d33e622.gif"></td>
<td class="label">1</td>
</tr></table>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/be3ac7378081/d33e631.gif" loading="lazy" id="d33e631" alt="graphic file with name d33e631.gif"></td>
<td class="label">2</td>
</tr></table>
<p id="Par31">where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8000"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/600e8c40e111/d33e640.gif" loading="lazy" id="d33e640" alt="Inline graphic"></span>.</p>
<p id="Par32">For each WSI in our dataset, we calculate two feature vectors mean (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/1aed52cd66b3/d33e648.gif" loading="lazy" id="d33e648" alt="Inline graphic"></span>) and median (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/09368c89e639/d33e654.gif" loading="lazy" id="d33e654" alt="Inline graphic"></span>) for all the tiles within that WSI, and concatenate them sequentially to form a single feature vector, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/630f1bad0840/d33e660.gif" loading="lazy" id="d33e660" alt="Inline graphic"></span>. For each WSI W, the generated feature vector <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/625d5d747829/d33e666.gif" loading="lazy" id="d33e666" alt="Inline graphic"></span>is computed as:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/187e67655191/d33e674.gif" loading="lazy" id="d33e674" alt="graphic file with name d33e674.gif"></td>
<td class="label">3</td>
</tr></table></section><section id="Sec7"><h3 class="pmc_sec_title">Model setup</h3>
<p id="Par34">The feature extraction process employs the ResNet101 model pre-trained on ImageNet, with the top (classification) layer removed and the training for the remaining layers frozen. Before feature extraction, each 512 × 512 pixel tile is resized to 224 × 224 pixels. A crucial step in our approach is color normalization, where each whole-slide image undergoes normalization using mean and standard deviations derived from the Reinhard Algorithm (RA) applied to a chosen reference image. This ensures that the color properties are consistent across all photos. Subsequently, every tile within each whole-slide image is passed through the feature extractor to obtain its feature vector, consisting of 2048 deep features. These extracted features for each tile are saved to disk to expedite the training and evaluation process.</p>
<p id="Par35">Given the variations in the dimensions of WSIs, the tiling process produces tile sets of varying lengths, leading to inconsistent feature set lengths for each whole slide image. To establish consistent feature representations, we calculate the mean and median of each feature set, resulting in two distinct feature vectors, each with 2048 dimensions. These two feature vectors are then concatenated into a single feature vector of length 4096, serving as a comprehensive representation of the WSI. Subsequently, these concatenated feature vectors are fed into various supervised machine-learning classifiers and corresponding WSI labels. This enables the classifiers to learn the feature representations and accurately classify the test set into metastasis and non-metastasis categories.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Model evaluation</h3>
<p id="Par36">A range of quantitative metrics is employed to comprehensively assess the OncoMet algorithm’s performance and predictive capability. These metrics offer intuitive insights into classification performance, including accuracy, precision, recall, and F1-score, while concealing the underlying mathematical complexity.</p>
<p id="Par37">Accuracy (ACC) denotes the percentage of correct predictions, serving as a straightforward measure of overall model performance. Precision quantifies the proportion of true positives among all instances classified as positive, reflecting the model’s ability to avoid false positives. Recall, or sensitivity, indicates how effectively the model identifies all actual positive cases of metastasis, highlighting its capacity to detect actual positives. The F1-score, calculated as the harmonic mean of precision and recall, offers a balanced perspective on the model’s performance, considering both false positives and false negatives.</p>
<p id="Par38">Furthermore, the Area Under the Curve (AUC) score is critical for evaluating the model’s ability to distinguish between positive (metastatic) and negative cases. A higher AUC signifies superior discrimination capacity, indicating more accurate cancer prognosis and diagnosis predictions. The Receiver Operating Characteristic (ROC) curve, which plots sensitivity against 1specificity across all classification thresholds, visually assesses the model’s discriminatory power. An ideal ROC curve is characterized by high sensitivity and specificity, demonstrating the model’s proficiency in metastasis detection.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Prediction of metastasis using OncoMet</h3>
<p id="Par39">In the experiment illustrated in Fig. <a href="#Fig1" class="usa-link">1</a>, the training set comprised 100 diagnostic WSIs. Features were extracted using the ResNet101 pre-trained architecture and subsequently fed into various machine-learning algorithms for classification in an end-to-end supervised manner. Data augmentation techniques were employed, given the limited dataset size and the risk of model overfitting. Data augmentation involves applying various transformations to the existing data, enhancing the dataset’s diversity, and reducing overfitting, thereby improving the generalizability and robustness of DL models. To address these challenges, our approach involved applying data augmentation techniques, which include Reinhard stain normalization, to our training data only.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372372_12967_2025_6914_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/5b21c1af061b/12967_2025_6914_Fig1_HTML.jpg" loading="lazy" id="d33e712" height="590" width="774" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Pipeline for Metastasis Prediction: This figure illustrates the training workflow on diagnostic whole slide images (WSIs), using ResNet101 for feature extraction and various machine learning algorithms for classification. Data augmentation techniques, including Reinhard stain normalization, were applied three times with different reference images to prevent overfitting, expanding the dataset to 300 images</p></figcaption></figure><p id="Par41">We performed this process three times, each time with a different reference image, making the data three times bigger with a total image of 300. These augmented samples were named with the original patient’s name and the reference image name used for normalization. This augmentation strategy contributed significantly to our DL models’ robustness and generalization capacity, making them more adept at handling the inherent variations in WSI characteristics across different medical institutions. Notably, testing data was not augmented to ensure a realistic evaluation of the model’s performance. Finally, the model was tested using 24 images separated during the train test split.</p>
<p id="Par42">The extracted features are passed to different Machine Learning models for classification. KNN performed better than other models including Support Vector Machine, Logistic Regression, Decision Tree, AdaBoost, and Random Forest. The model performs strongly in predicting metastasis, achieving a precision of 0.88, a recall of 1.00, and an F1-score of 0.94. For non-metastasis, the model attained a precision of 1.00, a recall of 0.78, and an F1-score of 0.88. The model’s overall accuracy and AUC score of 92%. Among the 24 test images, 15 were metastatic, and 9 were non-metastatic. The model correctly identified all 15 metastatic slides and 7 out of 9 non-metastatic slides, underscoring the accuracy and effectiveness of the OncoMet algorithm. The performance of all the models on the test set and validation set are illustrated in Table <a href="#Tab2" class="usa-link">2</a>.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Performance metrics for predicting metastatic and Non-Metastatic esophageal cancer using various algorithms on test set and external dataset</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Algorithms</th>
<th align="left" colspan="2" rowspan="1">Precision</th>
<th align="left" colspan="2" rowspan="1">Recall</th>
<th align="left" colspan="2" rowspan="1">F1-Score</th>
<th align="left" rowspan="2" colspan="1">ACC</th>
<th align="left" rowspan="2" colspan="1">AUC</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Meta</th>
<th align="left" colspan="1" rowspan="1">Non-Meta</th>
<th align="left" colspan="1" rowspan="1">Meta</th>
<th align="left" colspan="1" rowspan="1">Non-Meta</th>
<th align="left" colspan="1" rowspan="1">Meta</th>
<th align="left" colspan="1" rowspan="1">Non-Meta</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">SVC</td>
<td align="center" colspan="1" rowspan="1">0.71</td>
<td align="left" colspan="1" rowspan="1">0.50</td>
<td align="left" colspan="1" rowspan="1">0.67</td>
<td align="center" colspan="1" rowspan="1">0.56</td>
<td align="center" colspan="1" rowspan="1">0.69</td>
<td align="center" colspan="1" rowspan="1">0.53</td>
<td align="center" colspan="1" rowspan="1">0.62</td>
<td align="center" colspan="1" rowspan="1">0.63</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LR</td>
<td align="center" colspan="1" rowspan="1">0.79</td>
<td align="left" colspan="1" rowspan="1">0.60</td>
<td align="left" colspan="1" rowspan="1">0.73</td>
<td align="center" colspan="1" rowspan="1">0.67</td>
<td align="center" colspan="1" rowspan="1">0.76</td>
<td align="center" colspan="1" rowspan="1">0.63</td>
<td align="center" colspan="1" rowspan="1">0.71</td>
<td align="center" colspan="1" rowspan="1">0.83</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DT</td>
<td align="center" colspan="1" rowspan="1">0.75</td>
<td align="left" colspan="1" rowspan="1">0.44</td>
<td align="left" colspan="1" rowspan="1">0.40</td>
<td align="center" colspan="1" rowspan="1">0.78</td>
<td align="center" colspan="1" rowspan="1">0.52</td>
<td align="center" colspan="1" rowspan="1">0.56</td>
<td align="center" colspan="1" rowspan="1">0.54</td>
<td align="center" colspan="1" rowspan="1">0.49</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AdaBoost</td>
<td align="center" colspan="1" rowspan="1">0.69</td>
<td align="left" colspan="1" rowspan="1">045</td>
<td align="left" colspan="1" rowspan="1">0.60</td>
<td align="center" colspan="1" rowspan="1">0.56</td>
<td align="center" colspan="1" rowspan="1">0.64</td>
<td align="center" colspan="1" rowspan="1">0.50</td>
<td align="center" colspan="1" rowspan="1">0.65</td>
<td align="center" colspan="1" rowspan="1">0.58</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RFC</td>
<td align="center" colspan="1" rowspan="1">0.85</td>
<td align="left" colspan="1" rowspan="1">0.64</td>
<td align="left" colspan="1" rowspan="1">0.73</td>
<td align="center" colspan="1" rowspan="1">0.78</td>
<td align="center" colspan="1" rowspan="1">0.79</td>
<td align="center" colspan="1" rowspan="1">0.70</td>
<td align="center" colspan="1" rowspan="1">0.75</td>
<td align="center" colspan="1" rowspan="1">0.75</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KNN</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.88</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>1</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>1</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.78</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.94</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.88</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.92</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.92</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>KNN</p>
<p>(External Test set)</p>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.82</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>0.89</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>0.90</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.80</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.86</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.84</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.85</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.84</strong>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec11"><h3 class="pmc_sec_title">Model validation on an external dataset</h3>
<p id="Par44">To validate our best-performing model, we employed an external dataset of gastric cancer from TCGA. This external validation was crucial for assessing the model’s generalizability and robustness beyond the training data. We selected an equal number of metastatic and non-metastatic patients to ensure the data was balanced. The pre-processing of gastric cancer WSIs was conducted using PyHIST with the same parameters applied during the training phase. Color normalization was performed using Reinhard Normalization, utilizing one of the three reference images in the pre-processing step. The difference between patches before and after normalization is shown in Fig. <a href="#Fig2" class="usa-link">2</a>.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372372_12967_2025_6914_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/3bd9960441cb/12967_2025_6914_Fig2_HTML.jpg" loading="lazy" id="d33e980" height="380" width="774" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Color Normalization effects: The difference between patches of gastric cancer WSIs before and after color normalization using Reinhard Normalization is visible. This normalization was applied during the pre-processing of an external dataset from TCGA to validate our model. The figure demonstrates how the normalization process standardizes the color variations, ensuring consistency across WSIs</p></figcaption></figure><p id="Par46">The consistency in pre-processing ensured that the WSIs were standardized across training and validation datasets. Unlike the training set, the validation set did not undergo any augmentation, guaranteeing an unbiased evaluation of the model’s performance. The KNN model, trained on esophageal cancer data, achieved an accuracy of 85% on the gastric cancer validation set, with an AUC score of 84%. The precision for non-metastatic slides was 0.89, while the precision for metastatic slides was 0.82. The recall and F1 score for metastatic slides were higher than those for non-metastatic slides, with a recall of 0.90 and an F1-score of 0.86 for metastasis, compared to a recall of 0.80 and an F1-score of 0.84 for non-metastatic slides. These results indicate that the model effectively generalized to the external dataset, maintaining high-performance metrics across both metastasis and non-metastasis classifications.</p>
<p id="Par47">Gastric cancer was chosen for external validation due to its close histological and pathological resemblance to esophageal adenocarcinoma. Both cancers originate from the upper gastrointestinal tract and often share similar glandular morphologies, tumor microenvironments, and progression patterns. Moreover, they exhibit overlapping molecular features, including dysregulation of pathways such as PI3K/AKT and p53, which are critical to our study. These similarities make gastric cancer a biologically relevant choice for validating the generalizability of our model beyond esophageal carcinoma, particularly in the context of metastasis prediction using histopathological images [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>, <a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>].</p></section><section id="Sec12"><h3 class="pmc_sec_title">Prediction of signaling pathways using OncoMet</h3>
<p id="Par48">We utilized the extracted features from metastasis prediction to explore four signaling pathways associated with EC: mTOR, p53, PI3K/AKT, and PTEN. The whole experimentation is illustrated in Fig. <a href="#Fig3" class="usa-link">3</a>. The label for each WSI images was annotated as present (1) or absent (0). There were significant data imbalances between the two classes in each path. For instance, the p53 pathway showed an evident imbalance, with only 13% of cases where the path was absent and 87% where it was present. Similar patterns were observed for the other pathways: the mTOR pathway had 89% of cases absent and 11% present, the PI3K/AKT pathway had 85% present and 15% absent, and the PTEN pathway had 92% absent and only 8% present. These imbalances highlight the challenges in training robust models and underscore the necessity for strategies such as data augmentation and the inclusion of external datasets to ensure balanced representation.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372372_12967_2025_6914_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/71c5c349fe4c/12967_2025_6914_Fig3_HTML.jpg" loading="lazy" id="d33e1007" height="602" width="774" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Signaling Pathways pipeline: This figure illustrates four signaling pathways-mTOR, p53, PI3K/AKT, and PTEN in esophageal cancer, explored using features extracted from metastasis prediction. Each pathway is categorized as present (1) or absent (0) in WSI images, highlighting significant data imbalances between pathway states. Strategies such as data augmentation and external dataset inclusion were crucial to address these imbalances and enhance model reliability for pathway analysis</p></figcaption></figure><p id="Par50">Addressing these imbalances is crucial for improving our models’ reliability and predictive performance. We addressed the class imbalance by incorporating additional TCGA- HNSCC cohort data. This strategy aimed to balance the minority class representations within our dataset. Specifically, we identified 21 patients with mTOR pathway activation, eight patients with p53 pathway activation, 10 with PI3K/AKT pathway activation, and 26 with PTEN pathway activation from the HNSCC cohort. These patients were selected to match the minority class instances and were incorporated to balance the pathway data in our study. This integration helped balance the class distribution. The cases from the TCGA-HNSCC cohort were pre-processed similarly to those from the TCGA-ESCA cohort. Each WSI was divided into patches, and those containing less than 10% tissue were discarded to ensure quality. These patches were then passed through a pre-trained ResNet101 model (with the top layer removed) for feature extraction. The features extracted from ResNet101 were saved for subsequent analysis. The resulting dataset was randomly split into an 80:20 ratio for training and testing.</p>
<p id="Par51">Despite incorporating additional HNSCC data, some imbalances persisted. To further mitigate this issue in the training dataset, we applied the Synthetic Minority Over-sampling Technique (SMOTE). SMOTE is a powerful data augmentation method that helps address class imbalances in machine learning datasets. It generates synthetic samples for the minority class by intelligently interpolating between existing samples. This process creates a more balanced training dataset, ensuring the model is exposed to a more equitable representation of the majority and minority classes. The test set remained untouched to preserve its originality and provide an unbiased model performance evaluation. This approach ensured that the training data was balanced, enhancing the model’s ability to learn from diverse examples and reducing the risk of bias. Using the SMOTE technique, 187 training samples were generated for both classes for each pathway. The resulting balanced dataset is depicted in Fig. <a href="#Fig4" class="usa-link">4</a>, showcasing the equitable representation of both minority and majority groups.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372372_12967_2025_6914_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/3d2c97904c32/12967_2025_6914_Fig4_HTML.jpg" loading="lazy" id="d33e1026" height="696" width="774" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SMOTE Balancing in Training Dataset: This figure illustrates the application of the Synthetic Minority Over-sampling Technique (SMOTE) to mitigate class imbalances in the training dataset for pathway analysis. We incorporated additional HNSCC data and the SMOTE to generate synthetic samples for the minority class. This process created a balanced training dataset with equitable representation across minority and majority classes, which is crucial for enhancing model robustness and reducing bias. The test set was kept untouched to maintain its originality and ensure unbiased model performance evaluation</p></figcaption></figure><p id="Par52">Despite applying SMOTE to address class imbalance, the relatively low performance in PTEN pathway prediction is attributed to under-represented alterations of PTEN class which affected performance even after SMOTE-based oversampling. As synthetic samples generated by SMOTE are linear interpolations of minority instances, they have not adequately captured the non-linear and high-dimensional variability typical of histopathological image features, especially in complex molecular phenotypes such as PTEN inactivation.</p>
<p id="Par54">For each pathway, we experimented with multiple classifiers including KNN, SVM, Logistic Regression, Random Forest, Decision Tree, and AdaBoost. The classifier with the best performance based on accuracy, precision, recall, F1-score, and AUC was selected. We observed that AdaBoost performed better for the p53 and PI3K/AKT pathways, likely due to its ability to handle complex patterns and class imbalances, whereas Decision Tree achieved superior results for the mTOR and PTEN pathways, benefiting from simpler decision boundaries.</p>
<p id="Par55">The Decision Tree classifier achieved the best performance for predicting the mTOR signaling pathway. It attained an accuracy of 87% and an AUC (area under the ROC curve) of 64%. When the pathway is absent, the model demonstrated high precision (0.88), recall (0.96), and F1-score (0.92). Even when the pathway is present, the model maintained the precision (0.80), although recall (0.57) and F1-score (0.67) are lower. The sensitivity is 0.57, and the specificity is 0.95. For the p53 signaling pathway, the AdaBoost model outperformed other classifiers. It achieved balanced precision, recall, and F1-scores of 0.67 when the pathway was absent and 0.96 when present. This model exhibited high sensitivity (0.95) but lower specificity (0.66). Overall, it attained an accuracy of 93% and an AUC of 76%.</p>
<p id="Par56">AdaBoost also showed superior performance in predicting the PI3K/AKT signaling pathway. It achieved precision, recall, and F1-scores of 0.75 for the absence and 0.96 for the presence of the pathway. The model demonstrated high sensitivity (0.95) and specificity (0.75), impressive accuracy of 93%, and an AUC of 92%. For the PTEN signaling pathway, the Decision Tree classifier provided the best results. It achieved precision, recall, and F1-scores of 0.83, 0.87, and 0.85, respectively, when the pathway is absent. When the pathway is present, the scores are 0.57, 0.50, and 0.53. The sensitivity is 0.50, specificity is 0.86, accuracy is 77%, and AUC is 74%. The performance metrics of the best models for predicting signaling pathways in esophageal cancer are summarized in Table <a href="#Tab3" class="usa-link">3</a>. These models were selected based on their superior accuracy, precision, recall, and F1 score, demonstrating their effectiveness in identifying key signaling pathways associated with esophageal cancer.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Performance metrics of the best models for predicting signaling pathways in esophageal cancer</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Pathway</th>
<th align="left" rowspan="2" colspan="1">Models (best)</th>
<th align="left" colspan="2" rowspan="1">Precision</th>
<th align="left" colspan="2" rowspan="1">Recall</th>
<th align="left" colspan="2" rowspan="1">F1-Score</th>
<th align="left" rowspan="2" colspan="1">ACC</th>
<th align="left" rowspan="2" colspan="1">AUC</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">0</th>
<th align="left" colspan="1" rowspan="1">1</th>
<th align="left" colspan="1" rowspan="1">0</th>
<th align="left" colspan="1" rowspan="1">1</th>
<th align="left" colspan="1" rowspan="1">0</th>
<th align="left" colspan="1" rowspan="1">1</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">PI3K</td>
<td align="left" colspan="1" rowspan="1">
<strong>AdaBoost</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.75</td>
<td align="center" colspan="1" rowspan="1">0.96</td>
<td align="center" colspan="1" rowspan="1">0.75</td>
<td align="center" colspan="1" rowspan="1">0.96</td>
<td align="center" colspan="1" rowspan="1">0.75</td>
<td align="center" colspan="1" rowspan="1">0.96</td>
<td align="center" colspan="1" rowspan="1">0.93</td>
<td align="center" colspan="1" rowspan="1">0.92</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mTOR</td>
<td align="left" colspan="1" rowspan="1">
<strong>DT</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.88</td>
<td align="center" colspan="1" rowspan="1">0.80</td>
<td align="center" colspan="1" rowspan="1">0.96</td>
<td align="center" colspan="1" rowspan="1">0.57</td>
<td align="center" colspan="1" rowspan="1">0.92</td>
<td align="center" colspan="1" rowspan="1">0.67</td>
<td align="center" colspan="1" rowspan="1">0.87</td>
<td align="center" colspan="1" rowspan="1">0.64</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">p53</td>
<td align="left" colspan="1" rowspan="1">
<strong>AdaBoost</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.67</td>
<td align="center" colspan="1" rowspan="1">0.96</td>
<td align="center" colspan="1" rowspan="1">0.67</td>
<td align="center" colspan="1" rowspan="1">0.96</td>
<td align="center" colspan="1" rowspan="1">0.67</td>
<td align="center" colspan="1" rowspan="1">0.96</td>
<td align="center" colspan="1" rowspan="1">0.93</td>
<td align="center" colspan="1" rowspan="1">0.76</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PTEN</td>
<td align="left" colspan="1" rowspan="1">
<strong>DT</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.83</td>
<td align="center" colspan="1" rowspan="1">0.57</td>
<td align="center" colspan="1" rowspan="1">0.87</td>
<td align="center" colspan="1" rowspan="1">0.50</td>
<td align="center" colspan="1" rowspan="1">0.85</td>
<td align="center" colspan="1" rowspan="1">0.53</td>
<td align="center" colspan="1" rowspan="1">0.77</td>
<td align="center" colspan="1" rowspan="1">0.74</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section></section><section id="Sec600"><h2 class="pmc_sec_title">Discussion and conclusion</h2>
<p id="Par58">EC is a highly aggressive cancer, with around 6,04,000 new cases and 5,44,000 deaths annually [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. It is the sixth leading cause of cancer-related mortality and commonly metastasizes early to multiple organs, resulting in low survival rates of 15–20%. The two main types of EC are ESCC, linked to tobacco and alcohol use; EAC, primarily associated with obesity, GERD, and Barrett’s esophagus. Key oncogenic pathways involved in EC progression include mTOR, p53, PTEN, and PI3K/AKT [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Traditional diagnostic methods rely on time-consuming microscopic analysis of tissue slides. Still, integrating AI and ML, such as CNNs, promises faster, more accurate detection and classification, aiding pathologists in diagnosing EC and predicting metastasis.</p>
<p id="Par59">This study showed the capability of DL models to predict both metastasis and signaling pathways in EC using WSIs. We trained our model on the EC dataset. Further, we validated its performance of predicting metastasis on an external set of gastric cancer WSIs, demonstrating its potential as a reliable tool for cancer prognosis in clinical settings. Despite the promising results, we faced several challenges inherent to the domain, particularly the limited availability of sufficiently large training samples, which can lead to overfitting issues.</p>
<p id="Par60">We explored data augmentation strategies to address these challenges, including Reinhard stain normalization and the Synthetic Minority Over-sampling Technique (SMOTE). These efforts mitigated the challenges posed by limited training data and potential overfitting, ultimately contributing to a more reliable and accurate metastasis prediction model for esophageal cancer analysis. The total number of tiles used in this study, including those generated through data augmentation, was 10,78,883, effectively increasing the dataset size and providing the model with a more diverse range of examples to learn from. Tile distribution in different cohorts used in prediction of metastasis is illustrated in Fig. <a href="#Fig5" class="usa-link">5</a>.</p>
<figure class="fig xbox font-sm" id="Fig5"><h3 class="obj_head">Fig. 5.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372372_12967_2025_6914_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/299b2758b383/12967_2025_6914_Fig5_HTML.jpg" loading="lazy" id="d33e1212" height="455" width="774" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Tile Distribution: This illustrates the distribution of tiles used for predicting metastasis across different cohorts. It displays the number of tiles in the training set before and after augmentation and in the test set and external validation set. The increase in tile numbers post-augmentation highlights the importance of data augmentation in deep learning for improved feature extraction and model performance</p></figcaption></figure><p id="Par62">Handling WSIs is challenging due to their large size, necessitating the breaking up of WSIs into tiles or patches for model training. We extracted non-overlapping tiles to avoid repetition, ensure consistent tile dimensions, reduce computational load, prevent redundancy, and ensure overall resource efficiency. This approach streamlined the processing of WSIs, ensuring that machine learning models could effectively learn from the data. Decomposing a single WSI into tiles, followed by normalization and feature extraction, is time-consuming, averaging 15 to 25 min per slide. To expedite these processes, we utilized parallel processing on an NVIDIA Tesla V100 GPU with 32 GB memory, significantly enhancing our capability for intensive data analysis.</p>
<p id="Par63">Our research highlights the benefits of tissue-level classification over traditional patch-level analysis in digital pathology. Although widely used for disease progression and cancer metastasis detection, patch-level approaches are limited by several significant drawbacks. These methods require extensive manual annotation by pathologists, who must meticulously mark each patch within WSIs. This process is labor-intensive, time-consuming, and susceptible to human error. Furthermore, Patch-level analysis is vulnerable to sampling bias, as selected patches may not accurately represent the entire slide, potentially obscuring critical pathological details. In contrast, our whole slide image analysis method provides a comprehensive view of the pathological context, improving diagnostic performance and reliability. By analyzing the entire WSI, our approach preserves essential spatial relationships and contextual information often lost in patch-level analysis. This comprehensive method reduces the annotation burden on pathologists and mitigates the risk of sampling bias. As a result, it enhances the accuracy and generalizability of cancer detection and classification, offering a more robust and efficient alternative to patch-level techniques [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>].</p>
<p id="Par64">This study provides a substantial advancement in the field by validating the OncoMet algorithm’s efficacy in predicting metastasis in esophageal cancer through the analysis of WSIs. The study also extended the OncoMet algorithm to predict the activation status of key signaling pathways (mTOR, PTEN, p53, and PI3K/AKT) in esophageal cancer, addressing data imbalances and enhancing model reliability through advanced techniques like additional dataset incorporation and SMOTE. The findings underscore the OncoMet algorithm’s potential as a powerful tool for cancer prognosis and pathway analysis, with implications for targeted treatment strategies. This research contributes to advancing the field of digital pathology and computational oncology, paving the way for future studies and applications in cancer research and clinical practice.</p>
<p id="Par65">The predictions made by our model are consistent with findings reported in the existing literature on EC. Specifically, the following molecular alterations predicted by the model align closely with known biological patterns in EC: (a) High frequencies of p53 pathway alterations, PTEN loss, and aberrant activation of the PI3K/AKT and mTOR pathways have been well-documented in EC, particularly in esophageal adenocarcinoma. Our model reflected these trends, with p53 and PI3K/AKT showing high predicted activation rates, while PTEN frequently appeared inactivated, consistent with its tumor suppressor loss reported in multiple genomic studies [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>–<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. (b) A pathway alteration score of 0.93 for PI3K/AKT implies strong pathway activation. It is frequently hyperactivated in EC, driving proliferation, survival and metastasis. For example, phosphorylated AKT is constitutively elevated in the majority of tumors: one study found AKT active in ~ 75% of esophageal cancers versus matched normal tissue [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. (c) mTOR lies downstream of PI3K/AKT and orchestrates protein synthesis, metabolism and cell growth. In ESCC, elevated mTOR or phosphorylated-mTOR expression strongly predicts worse survival. Analyses show that mTOR overexpression is a robust biomarker of poor overall and disease-free survival [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. A high mTOR accuracy of 0.87 correlates with its aggressiveness. (d) In EC, TP53 is by far the most commonly altered gene with over 90% of dysplastic and cancerous lesions exhibiting TP53 loss [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. In adenocarcinomas of the esophagus and gastroesophageal junction, nearly half of tumors harbor TP53 mutations, with significantly worse survival rates [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. (e) In EC, PTEN downregulation is well documented with studies reporting significantly lower PTEN protein in tumors than in adjacent normal mucosa [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. PTEN loss correlates with aggressive featuresand tend to have deeper invasion and higher stage.</p>
<p id="Par66">Our model’s capability to predict the activation of these pathways further emphasizes its potential utility in clinical settings. Identifying the status of mTOR, PTEN, p53, and PI3K/AKT pathways can provide valuable insights into the molecular foundations of esophageal cancer in individual patients, thereby guiding personalized treatment strategies. The significant data imbalances encountered, with most instances showing the pathways as absent, highlight the necessity for data augmentation and balancing techniques. Incorporating additional data from the TCGA-HNSCC cohort helped to mitigate these imbalances. Yet, some persisted, underscoring the ongoing challenge of achieving perfectly balanced datasets.</p>
<p id="Par67">By employing a combination of ResNet101 for feature extraction and various machine learning classifiers, we achieved an overall accuracy and an AUC score of 92% in detecting metastasis on test set. The model’s ability was further validated on an external dataset of gastric cancer WSIs, maintaining robust performance with an accuracy of 85% and an AUC score of 84%. The AUC ROC curves on test and validation sets are shown in Fig. <a href="#Fig6" class="usa-link">6</a>. Addressing the inherent challenges posed by limited training data and data imbalance, we applied data augmentation techniques, including reinhard stain normalization and SMOTE, to enhance the model’s robustness and mitigate overfitting. The incorporation of additional data from the TCGA-HNSCC cohort and the careful pre-processing of WSIs ensured a balanced representation of minority classes, further improving the reliability of the model.</p>
<figure class="fig xbox font-sm" id="Fig6"><h3 class="obj_head">Fig. 6.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372372_12967_2025_6914_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c992/12372372/756304db2865/12967_2025_6914_Fig6_HTML.jpg" loading="lazy" id="d33e1263" height="331" width="774" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>AUC ROC Curves for Metastasis: This figure presents the AUC ROC curves for both the test and external validation sets. The curves demonstrate the model’s performance in distinguishing between metastatic and non-metastatic esophageal cancer cases. A higher AUC indicates better discrimination ability, showcasing the model’s robustness and generalizability across different datasets</p></figcaption></figure><p id="Par69">Our investigation into the mTOR, PTEN, p53, and PI3K/AKT signaling pathways revealed significant data imbalances, effectively addressed through data augmentation using two different techniques. The results highlight the importance of balanced datasets in training robust predictive models and demonstrate the feasibility of transferring learned features from metastasis prediction to pathway analysis. Future work should further increase dataset diversity, explore advanced augmentation techniques, and integrate multiomics data to enhance model performance. Developing more sophisticated algorithms capable of handling imbalanced data more effectively would further improve predictive accuracy and robustness.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>SWA, TAM, AA, MAM, and MRB contributed to the study’s concept and design, wrote the manuscript, generated figures, and critically edited it. SWA, ABA, TM, ANA, and TAM performed experiments. SWA, ABA, TM, ANA, TAM, AAB, AA, MRB, and MAM critically revised and edited the scientific content. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study was funded, in part, by a Research Grant (Grant number: ID No. 2022–16465) from the Indian Council of Medical Research (ICMR) Govt. of India, New Delhi to Muzafar A. Macha. Promotion of University Research and Scientific Excellence (PURSE) (SR/PURSE/2022/121) grant from the Department of Science and Technology, Govt. of India, New Delhi to the Islamic University of Science and Technology (IUST), Awantipora.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Not applicable.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar3"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par70">Not applicable.</p></section><section id="FPar1"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par71">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par72">The authors declare that they have no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Muzafar Ahmad Macha, Email: muzafar.macha@iust.ac.in.</p>
<p>Muzafar Rasool Bhat, Email: muzafarrasool@gmail.com.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Sung H, Ferlay J, Siegel RL, Laversanne M, Soerjomataram I, Jemal A, Bray F. Global cancer statistics 2020: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. Cancer J Clin. 2021;71:209–49.</cite> [<a href="https://doi.org/10.3322/caac.21660" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33538338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sung%20H,%20Ferlay%20J,%20Siegel%20RL,%20Laversanne%20M,%20Soerjomataram%20I,%20Jemal%20A,%20Bray%20F.%20Global%20cancer%20statistics%202020:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries.%20Cancer%20J%20Clin.%202021;71:209%E2%80%9349." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Koizumi W, Kitago M, Shinoda M, Yagi H, Abe Y, Oshima G, Hori S, Inomata K, Kawakubo H, Kawaida M. Successful resection of pancreatic metastasis from oesophageal squamous cell carcinoma: a case report and review of the literature. BMC Cancer. 2019;19:1–5.
</cite> [<a href="https://doi.org/10.1186/s12885-019-5549-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6451211/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30953505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koizumi%20W,%20Kitago%20M,%20Shinoda%20M,%20Yagi%20H,%20Abe%20Y,%20Oshima%20G,%20Hori%20S,%20Inomata%20K,%20Kawakubo%20H,%20Kawaida%20M.%20Successful%20resection%20of%20pancreatic%20metastasis%20from%20oesophageal%20squamous%20cell%20carcinoma:%20a%20case%20report%20and%20review%20of%20the%20literature.%20BMC%20Cancer.%202019;19:1%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Weinberg JS, Suki D, Hanbali F, Cohen ZR, Lenzi R, Sawaya R. Metastasis of esophageal carcinoma to the brain. Cancer. 2003;98:1925–33.
</cite> [<a href="https://doi.org/10.1002/cncr.11737" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14584076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weinberg%20JS,%20Suki%20D,%20Hanbali%20F,%20Cohen%20ZR,%20Lenzi%20R,%20Sawaya%20R.%20Metastasis%20of%20esophageal%20carcinoma%20to%20the%20brain.%20Cancer.%202003;98:1925%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Pennathur A, Gibson MK, Jobe BA, Luketich JD. Oesophageal carcinoma. Lancet. 2013;381:400–12.
</cite> [<a href="https://doi.org/10.1016/S0140-6736(12)60643-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23374478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pennathur%20A,%20Gibson%20MK,%20Jobe%20BA,%20Luketich%20JD.%20Oesophageal%20carcinoma.%20Lancet.%202013;381:400%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Enzinger PC, Mayer RJ. Esophageal cancer. N Engl J Med. 2003;349:2241–52.
</cite> [<a href="https://doi.org/10.1056/NEJMra035010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14657432/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Enzinger%20PC,%20Mayer%20RJ.%20Esophageal%20cancer.%20N%20Engl%20J%20Med.%202003;349:2241%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Dempsey D. Esophagectomy for T1 esophageal cancer: outcomes in 100 patients and implications for endoscopic therapy. In: Pennathur A, Farkas A, Krasinskas AM et al. (eds) (University of Pittsburgh Med Ctr; The Univ of Pittsburgh Cancer Inst Biostatistics Facility, Pittsburgh, PA) Ann Thorac Surg. 2009;87:1048–1055. Year Book of Gastroenterology 2009, 2009;157–158.</cite> [<a href="https://doi.org/10.1016/j.athoracsur.2008.12.060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2912110/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19324126/" class="usa-link">PubMed</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Vijayakumar S, Saravanan A, Sayeed N, Kirezi NGR, Duggirala NK, El-Hashash AH, Al Hussein H. Analyzing mortality patterns and location of death in patients with malignant esophageal neoplasms: a two-decade study in the united States. Cureus 2023;15.</cite> [<a href="https://doi.org/10.7759/cureus.50455" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10788881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38226107/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Santucci C, Mignozzi S, Malvezzi M, Collatuzzo G, Levi F, La Vecchia C, Negri E. Global trends in esophageal cancer mortality with predictions to 2025, and in incidence by histotype. Cancer Epidemiol. 2023;87:102486.
</cite> [<a href="https://doi.org/10.1016/j.canep.2023.102486" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37956470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Santucci%20C,%20Mignozzi%20S,%20Malvezzi%20M,%20Collatuzzo%20G,%20Levi%20F,%20La%20Vecchia%20C,%20Negri%20E.%20Global%20trends%20in%20esophageal%20cancer%20mortality%20with%20predictions%20to%202025,%20and%20in%20incidence%20by%20histotype.%20Cancer%20Epidemiol.%202023;87:102486." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Kim N. Sex difference of esophageal cancer: esophageal squamous cell carcinoma vs. esophageal adenocarcinoma. Sex/Gender-Specific medicine in the Gastrointestinal diseases. Springer; 2022. pp. 69–92.</cite>
</li>
<li id="CR10">
<span class="label">10.</span><cite>Gerstberger S, Jiang Q, Ganesh K. Metastasis. Cell. 2023;186:1564–79.
</cite> [<a href="https://doi.org/10.1016/j.cell.2023.03.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10511214/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37059065/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gerstberger%20S,%20Jiang%20Q,%20Ganesh%20K.%20Metastasis.%20Cell.%202023;186:1564%E2%80%9379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Sheikh M, Roshandel G, McCormack V, Malekzadeh R. Current status and future prospects for esophageal cancer. Cancers. 2023;15:765.
</cite> [<a href="https://doi.org/10.3390/cancers15030765" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9913274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36765722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sheikh%20M,%20Roshandel%20G,%20McCormack%20V,%20Malekzadeh%20R.%20Current%20status%20and%20future%20prospects%20for%20esophageal%20cancer.%20Cancers.%202023;15:765." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Lee V, King ALO, Sritharan D, Moore NS, Chadha S, Maresca R, Hager T, Aneja S. Lymph node metastasis prediction with non-small cell lung cancer histopathology imaging. American Society of Clinical Oncology; 2024.</cite>
</li>
<li id="CR13">
<span class="label">13.</span><cite>Gao F, Jiang L, Guo T, Lin J, Xu W, Yuan L, Han Y, Yang J, Pan Q, Chen E. Deep learning-based pathological prediction of lymph node metastasis for patient with renal cell carcinoma from primary whole slide images. J Translational Med. 2024;22:568.</cite> [<a href="https://doi.org/10.1186/s12967-024-05382-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11177484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38877591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao%20F,%20Jiang%20L,%20Guo%20T,%20Lin%20J,%20Xu%20W,%20Yuan%20L,%20Han%20Y,%20Yang%20J,%20Pan%20Q,%20Chen%20E.%20Deep%20learning-based%20pathological%20prediction%20of%20lymph%20node%20metastasis%20for%20patient%20with%20renal%20cell%20carcinoma%20from%20primary%20whole%20slide%20images.%20J%20Translational%20Med.%202024;22:568." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Wessels F, Schmitt M, Krieghoff-Henning E, Jutzi T, Worst TS, Waldbillig F, Neuberger M, Maron RC, Steeg M, Gaiser T. Deep learning approach to predict lymph node metastasis directly from primary tumour histology in prostate cancer. BJU Int. 2021;128:352–60.
</cite> [<a href="https://doi.org/10.1111/bju.15386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33706408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wessels%20F,%20Schmitt%20M,%20Krieghoff-Henning%20E,%20Jutzi%20T,%20Worst%20TS,%20Waldbillig%20F,%20Neuberger%20M,%20Maron%20RC,%20Steeg%20M,%20Gaiser%20T.%20Deep%20learning%20approach%20to%20predict%20lymph%20node%20metastasis%20directly%20from%20primary%20tumour%20histology%20in%20prostate%20cancer.%20BJU%20Int.%202021;128:352%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Wu N, Du Z, Zhu Y, Song Y, Pang L, Chen Z. The expression and prognostic impact of the PI3K/AKT/mTOR signaling pathway in advanced esophageal squamous cell carcinoma. Technol Cancer Res Treat. 2018;17:1533033818758772.
</cite> [<a href="https://doi.org/10.1177/1533033818758772" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5826005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29463194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20N,%20Du%20Z,%20Zhu%20Y,%20Song%20Y,%20Pang%20L,%20Chen%20Z.%20The%20expression%20and%20prognostic%20impact%20of%20the%20PI3K/AKT/mTOR%20signaling%20pathway%20in%20advanced%20esophageal%20squamous%20cell%20carcinoma.%20Technol%20Cancer%20Res%20Treat.%202018;17:1533033818758772." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Basson MA. Signaling in cell differentiation and morphogenesis. Cold Spring Harb Perspect Biol 2012;4.</cite> [<a href="https://doi.org/10.1101/cshperspect.a008151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3367549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22570373/" class="usa-link">PubMed</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>da Silva HB, Amaral EP, Nolasco EL, de Victo NC, Atique R, Jank CC, Anschau V, Zerbini LF, Correa RG. Dissecting major signaling pathways throughout the development of prostate cancer. Prostate Cancer. 2013;2013:920612.
</cite> [<a href="https://doi.org/10.1155/2013/920612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3657461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23738079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?da%20Silva%20HB,%20Amaral%20EP,%20Nolasco%20EL,%20de%20Victo%20NC,%20Atique%20R,%20Jank%20CC,%20Anschau%20V,%20Zerbini%20LF,%20Correa%20RG.%20Dissecting%20major%20signaling%20pathways%20throughout%20the%20development%20of%20prostate%20cancer.%20Prostate%20Cancer.%202013;2013:920612." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Nisar S, Hashem S, Macha MA, Yadav SK, Muralitharan S, Therachiyil L, Sageena G, Al-Naemi H, Haris M, Bhat AA. Exploring dysregulated signaling pathways in cancer. Curr Pharm Des. 2020;26:429–45.
</cite> [<a href="https://doi.org/10.2174/1381612826666200115095937" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31939726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nisar%20S,%20Hashem%20S,%20Macha%20MA,%20Yadav%20SK,%20Muralitharan%20S,%20Therachiyil%20L,%20Sageena%20G,%20Al-Naemi%20H,%20Haris%20M,%20Bhat%20AA.%20Exploring%20dysregulated%20signaling%20pathways%20in%20cancer.%20Curr%20Pharm%20Des.%202020;26:429%E2%80%9345." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Luo Q, Du R, Liu W, Huang G, Dong Z, Li X. PI3K/AKT/mTOR signaling pathway: role in esophageal squamous cell carcinoma, regulatory mechanisms and opportunities for targeted therapy. Front Oncol. 2022;12:852383.
</cite> [<a href="https://doi.org/10.3389/fonc.2022.852383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8980269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35392233/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luo%20Q,%20Du%20R,%20Liu%20W,%20Huang%20G,%20Dong%20Z,%20Li%20X.%20PI3K/AKT/mTOR%20signaling%20pathway:%20role%20in%20esophageal%20squamous%20cell%20carcinoma,%20regulatory%20mechanisms%20and%20opportunities%20for%20targeted%20therapy.%20Front%20Oncol.%202022;12:852383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Litjens G, Sánchez CI, Timofeeva N, Hermsen M, Nagtegaal I, Kovacs I, Hulsbergen-Van De Kaa C, Bult P, Van Ginneken B, Van Der Laak J. Deep learning as a tool for increased accuracy and efficiency of histopathological diagnosis. Sci Rep. 2016;6:26286.
</cite> [<a href="https://doi.org/10.1038/srep26286" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4876324/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27212078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Litjens%20G,%20S%C3%A1nchez%20CI,%20Timofeeva%20N,%20Hermsen%20M,%20Nagtegaal%20I,%20Kovacs%20I,%20Hulsbergen-Van%20De%20Kaa%20C,%20Bult%20P,%20Van%20Ginneken%20B,%20Van%20Der%20Laak%20J.%20Deep%20learning%20as%20a%20tool%20for%20increased%20accuracy%20and%20efficiency%20of%20histopathological%20diagnosis.%20Sci%20Rep.%202016;6:26286." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Wang X, Chen Y, Gao Y, Zhang H, Guan Z, Dong Z, Zheng Y, Jiang J, Yang H, Wang L. Predicting gastric cancer outcome from resected lymph node histopathology images using deep learning. Nat Commun. 2021;12:1637.
</cite> [<a href="https://doi.org/10.1038/s41467-021-21674-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7954798/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33712598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20X,%20Chen%20Y,%20Gao%20Y,%20Zhang%20H,%20Guan%20Z,%20Dong%20Z,%20Zheng%20Y,%20Jiang%20J,%20Yang%20H,%20Wang%20L.%20Predicting%20gastric%20cancer%20outcome%20from%20resected%20lymph%20node%20histopathology%20images%20using%20deep%20learning.%20Nat%20Commun.%202021;12:1637." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>He L, Long LR, Antani S, Thoma GR. Histology image analysis for carcinoma detection and grading. Comput Methods Programs Biomed. 2012;107:538–56.
</cite> [<a href="https://doi.org/10.1016/j.cmpb.2011.12.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3587978/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22436890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20L,%20Long%20LR,%20Antani%20S,%20Thoma%20GR.%20Histology%20image%20analysis%20for%20carcinoma%20detection%20and%20grading.%20Comput%20Methods%20Programs%20Biomed.%202012;107:538%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Cerami E, Gao J, Dogrusoz U, Gross BE, Sumer SO, Aksoy BA, Jacobsen A, Byrne CJ, Heuer ML, Larsson E, et al. The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. Cancer Discov. 2012;2:401–4.
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-12-0095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3956037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22588877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cerami%20E,%20Gao%20J,%20Dogrusoz%20U,%20Gross%20BE,%20Sumer%20SO,%20Aksoy%20BA,%20Jacobsen%20A,%20Byrne%20CJ,%20Heuer%20ML,%20Larsson%20E,%20et%20al.%20The%20cBio%20cancer%20genomics%20portal:%20an%20open%20platform%20for%20exploring%20multidimensional%20cancer%20genomics%20data.%20Cancer%20Discov.%202012;2:401%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Krämer A, Green J, Pollard J Jr., Tugendreich S. Causal analysis approaches in ingenuity pathway analysis. Bioinformatics. 2014;30:523–30.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3928520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24336805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kr%C3%A4mer%20A,%20Green%20J,%20Pollard%20J%20Jr.,%20Tugendreich%20S.%20Causal%20analysis%20approaches%20in%20ingenuity%20pathway%20analysis.%20Bioinformatics.%202014;30:523%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Talebi A, Celis-Morales CA, Borumandnia N, Abbasi S, Pourhoseingholi MA, Akbari A, Yousefi J. Predicting metastasis in gastric cancer patients: machine learning-based approaches. Sci Rep. 2023;13:4163.
</cite> [<a href="https://doi.org/10.1038/s41598-023-31272-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10011363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36914697/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Talebi%20A,%20Celis-Morales%20CA,%20Borumandnia%20N,%20Abbasi%20S,%20Pourhoseingholi%20MA,%20Akbari%20A,%20Yousefi%20J.%20Predicting%20metastasis%20in%20gastric%20cancer%20patients:%20machine%20learning-based%20approaches.%20Sci%20Rep.%202023;13:4163." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Talebi R, Celis-Morales CA, Akbari A, Talebi A, Borumandnia N, Pourhoseingholi MA. Machine learning-based classifiers to predict metastasis in colorectal cancer patients. Front Artif Intell. 2024;7:1285037.
</cite> [<a href="https://doi.org/10.3389/frai.2024.1285037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10847339/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38327669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Talebi%20R,%20Celis-Morales%20CA,%20Akbari%20A,%20Talebi%20A,%20Borumandnia%20N,%20Pourhoseingholi%20MA.%20Machine%20learning-based%20classifiers%20to%20predict%20metastasis%20in%20colorectal%20cancer%20patients.%20Front%20Artif%20Intell.%202024;7:1285037." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Aalam SW, Ahanger AB, Masoodi TA, Bhat AA, Akil ASA, Khan MA, Assad A, Macha MA, Bhat MR. Deep learning-based identification of esophageal cancer subtypes through analysis of high-resolution histopathology images. Front Mol Biosci. 2024;11:1346242.
</cite> [<a href="https://doi.org/10.3389/fmolb.2024.1346242" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10985197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38567100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aalam%20SW,%20Ahanger%20AB,%20Masoodi%20TA,%20Bhat%20AA,%20Akil%20ASA,%20Khan%20MA,%20Assad%20A,%20Macha%20MA,%20Bhat%20MR.%20Deep%20learning-based%20identification%20of%20esophageal%20cancer%20subtypes%20through%20analysis%20of%20high-resolution%20histopathology%20images.%20Front%20Mol%20Biosci.%202024;11:1346242." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Berman AG, Orchard WR, Gehrung M, Markowetz F. PathML: a unified framework for whole-slide image analysis with deep learning. <em>MedRxiv</em> 2021:2021.2007. 2007.21260138.</cite>
</li>
<li id="CR29">
<span class="label">29.</span><cite>Goode A, Gilbert B, Harkes J, Jukic D, Satyanarayanan M. OpenSlide: a vendor-neutral software foundation for digital pathology. J Pathol Inf. 2013;4:27.</cite> [<a href="https://doi.org/10.4103/2153-3539.119005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3815078/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24244884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goode%20A,%20Gilbert%20B,%20Harkes%20J,%20Jukic%20D,%20Satyanarayanan%20M.%20OpenSlide:%20a%20vendor-neutral%20software%20foundation%20for%20digital%20pathology.%20J%20Pathol%20Inf.%202013;4:27." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Hoffman RA, Kothari S, Phan JH, Wang MD. A high-resolution tile-based approach for classifying biological regions in whole-slide histopathological images. IFMBE Proc. 2014;42:280–3.
</cite> [<a href="https://doi.org/10.1007/978-3-319-03005-0_71" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4983443/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27532012/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoffman%20RA,%20Kothari%20S,%20Phan%20JH,%20Wang%20MD.%20A%20high-resolution%20tile-based%20approach%20for%20classifying%20biological%20regions%20in%20whole-slide%20histopathological%20images.%20IFMBE%20Proc.%202014;42:280%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Lakshmanan B, Anand S, Jenitha T. Stain removal through color normalization of haematoxylin and eosin images: a review. In: Journal of physics: conference series. IOP Publishing; 2019: 012108.</cite>
</li>
<li id="CR32">
<span class="label">32.</span><cite>Qu H, Zhou M, Yan Z, Wang H, Rustgi VK, Zhang S, Gevaert O, Metaxas DN. Genetic mutation and biological pathway prediction based on whole slide images in breast carcinoma using deep learning. NPJ Precis Oncol. 2021;5:87.
</cite> [<a href="https://doi.org/10.1038/s41698-021-00225-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8460699/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34556802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qu%20H,%20Zhou%20M,%20Yan%20Z,%20Wang%20H,%20Rustgi%20VK,%20Zhang%20S,%20Gevaert%20O,%20Metaxas%20DN.%20Genetic%20mutation%20and%20biological%20pathway%20prediction%20based%20on%20whole%20slide%20images%20in%20breast%20carcinoma%20using%20deep%20learning.%20NPJ%20Precis%20Oncol.%202021;5:87." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Roy S, Panda S, Jangid M. (2021, August). Modified reinhard algorithm for color normalization of colorectal cancer histopathology images. In 2021 29th European Signal Processing Conference (EUSIPCO) (pp. 1231–1235). IEEE.</cite>
</li>
<li id="CR34">
<span class="label">34.</span><cite>Quante M, Wang TC, Bass AJ. Adenocarcinoma of the oesophagus: is it gastric cancer? Gut. 2023;72(6):1027–9.
</cite> [<a href="https://doi.org/10.1136/gutjnl-2022-327096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10176420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35365571/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quante%20M,%20Wang%20TC,%20Bass%20AJ.%20Adenocarcinoma%20of%20the%20oesophagus:%20is%20it%20gastric%20cancer?%20Gut.%202023;72(6):1027%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Hu B, Hajj E, Sittler N, Lammert S, Barnes N, R., Meloni-Ehrig A. Gastric cancer: classification, histology and application of molecular pathology. J Gastrointest Oncol. 2012;3(3):251.
</cite> [<a href="https://doi.org/10.3978/j.issn.2078-6891.2012.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3418539/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22943016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu%20B,%20Hajj%20E,%20Sittler%20N,%20Lammert%20S,%20Barnes%20N,%20R.,%20Meloni-Ehrig%20A.%20Gastric%20cancer:%20classification,%20histology%20and%20application%20of%20molecular%20pathology.%20J%20Gastrointest%20Oncol.%202012;3(3):251." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Qu H-T, Li Q, Hao L, Ni Y-J, Luan W-Y, Yang Z, Chen X-D, Zhang T-T, Miao Y-D, Zhang F. Esophageal cancer screening, early detection and treatment: current insights and future directions. World J Gastrointest Oncol. 2024;16:1180.
</cite> [<a href="https://doi.org/10.4251/wjgo.v16.i4.1180" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11037049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38660654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qu%20H-T,%20Li%20Q,%20Hao%20L,%20Ni%20Y-J,%20Luan%20W-Y,%20Yang%20Z,%20Chen%20X-D,%20Zhang%20T-T,%20Miao%20Y-D,%20Zhang%20F.%20Esophageal%20cancer%20screening,%20early%20detection%20and%20treatment:%20current%20insights%20and%20future%20directions.%20World%20J%20Gastrointest%20Oncol.%202024;16:1180." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Hölscher AH, Bollschweiler E, Schneider PM, Siewert JR. Prognosis of early esophageal cancer. Comparison between adeno-and squamous cell carcinoma. Cancer. 1995;76:178–86.
</cite> [<a href="https://doi.org/10.1002/1097-0142(19950715)76:2&lt;178::aid-cncr2820760204&gt;3.0.co;2-d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8625089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?H%C3%B6lscher%20AH,%20Bollschweiler%20E,%20Schneider%20PM,%20Siewert%20JR.%20Prognosis%20of%20early%20esophageal%20cancer.%20Comparison%20between%20adeno-and%20squamous%20cell%20carcinoma.%20Cancer.%201995;76:178%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Li B, Qin W, Yang L, Li H, Jiang C, Yao Y, Cheng S, Zou B, Fan B, Dong T. From pixels to patient care: deep learning-enabled pathomics signature offers precise outcome predictions for immunotherapy in esophageal squamous cell cancer. J Transl Med. 2024;22:195.
</cite> [<a href="https://doi.org/10.1186/s12967-024-04997-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10885627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38388379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20B,%20Qin%20W,%20Yang%20L,%20Li%20H,%20Jiang%20C,%20Yao%20Y,%20Cheng%20S,%20Zou%20B,%20Fan%20B,%20Dong%20T.%20From%20pixels%20to%20patient%20care:%20deep%20learning-enabled%20pathomics%20signature%20offers%20precise%20outcome%20predictions%20for%20immunotherapy%20in%20esophageal%20squamous%20cell%20cancer.%20J%20Transl%20Med.%202024;22:195." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Zhong L, Li H, Chang W, Ao Y, Wen Z, Chen Y. TP53 mutations in esophageal squamous cell carcinoma. Front Biosci Landmark. 2023;28(9):219.</cite> [<a href="https://doi.org/10.31083/j.fbl2809219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37796679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhong%20L,%20Li%20H,%20Chang%20W,%20Ao%20Y,%20Wen%20Z,%20Chen%20Y.%20TP53%20mutations%20in%20esophageal%20squamous%20cell%20carcinoma.%20Front%20Biosci%20Landmark.%202023;28(9):219." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Wang G, Guo S, Zhang W, Li Z, Xu J, Li D, Zhan Q. A comprehensive analysis of alterations in DNA damage repair pathways reveals a potential way to enhance the radio-sensitivity of esophageal squamous cell cancer. Front Oncol. 2020;10:575711.
</cite> [<a href="https://doi.org/10.3389/fonc.2020.575711" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7596747/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33178606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20G,%20Guo%20S,%20Zhang%20W,%20Li%20Z,%20Xu%20J,%20Li%20D,%20Zhan%20Q.%20A%20comprehensive%20analysis%20of%20alterations%20in%20DNA%20damage%20repair%20pathways%20reveals%20a%20potential%20way%20to%20enhance%20the%20radio-sensitivity%20of%20esophageal%20squamous%20cell%20cancer.%20Front%20Oncol.%202020;10:575711." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Fusco N, Sajjadi E, Venetis K, Gaudioso G, Lopez G, Corti C, Rocco EG, Criscitiello C, Malapelle U, Invernizzi M. PTEN alterations and their role in cancer management: are we making headway on precision medicine? Genes. 2020;11(7):719.
</cite> [<a href="https://doi.org/10.3390/genes11070719" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7397204/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32605290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fusco%20N,%20Sajjadi%20E,%20Venetis%20K,%20Gaudioso%20G,%20Lopez%20G,%20Corti%20C,%20Rocco%20EG,%20Criscitiello%20C,%20Malapelle%20U,%20Invernizzi%20M.%20PTEN%20alterations%20and%20their%20role%20in%20cancer%20management:%20are%20we%20making%20headway%20on%20precision%20medicine?%20Genes.%202020;11(7):719." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Li B, Li J, Xu WW, Guan XY, Qin YR, Zhang LY, Cheung AL. Suppression of esophageal tumor growth and chemoresistance by directly targeting the PI3K/AKT pathway. Oncotarget. 2014;5(22):11576.
</cite> [<a href="https://doi.org/10.18632/oncotarget.2596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4294385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25344912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20B,%20Li%20J,%20Xu%20WW,%20Guan%20XY,%20Qin%20YR,%20Zhang%20LY,%20Cheung%20AL.%20Suppression%20of%20esophageal%20tumor%20growth%20and%20chemoresistance%20by%20directly%20targeting%20the%20PI3K/AKT%20pathway.%20Oncotarget.%202014;5(22):11576." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Lu J, Pan Y, Xia X, Gu Y, Lei Y. Prognostic significance of mTOR and PTEN in patients with esophageal squamous cell carcinoma. Biomed Res Int. 2015;20151:417210.</cite> [<a href="https://doi.org/10.1155/2015/417210" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4555449/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26357655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20J,%20Pan%20Y,%20Xia%20X,%20Gu%20Y,%20Lei%20Y.%20Prognostic%20significance%20of%20mTOR%20and%20PTEN%20in%20patients%20with%20esophageal%20squamous%20cell%20carcinoma.%20Biomed%20Res%20Int.%202015;20151:417210." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Murai K, Dentro S, Ong SH, Sood R, Fernandez-Antoran D, Herms A, Kostiou V, Abnizova I, Hall BA, Gerstung M, Jones PH. p53 mutation in normal esophagus promotes multiple stages of carcinogenesis but is constrained by clonal competition. Nat Commun. 2022;13(1):6206.
</cite> [<a href="https://doi.org/10.1038/s41467-022-33945-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9584949/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36266286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murai%20K,%20Dentro%20S,%20Ong%20SH,%20Sood%20R,%20Fernandez-Antoran%20D,%20Herms%20A,%20Kostiou%20V,%20Abnizova%20I,%20Hall%20BA,%20Gerstung%20M,%20Jones%20PH.%20p53%20mutation%20in%20normal%20esophagus%20promotes%20multiple%20stages%20of%20carcinogenesis%20but%20is%20constrained%20by%20clonal%20competition.%20Nat%20Commun.%202022;13(1):6206." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Ireland AP, Shibata DK, Chandrasoma P, Lord RV, Peters JH, DeMeester TR. Clinical significance of p53 mutations in adenocarcinoma of the esophagus and cardia. Ann Surg. 2000;231(2):179–87.
</cite> [<a href="https://doi.org/10.1097/00000658-200002000-00005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1420984/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10674608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ireland%20AP,%20Shibata%20DK,%20Chandrasoma%20P,%20Lord%20RV,%20Peters%20JH,%20DeMeester%20TR.%20Clinical%20significance%20of%20p53%20mutations%20in%20adenocarcinoma%20of%20the%20esophagus%20and%20cardia.%20Ann%20Surg.%202000;231(2):179%E2%80%9387." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Chang D, Wang TY, Li HC, Wei JC, Song JX. Prognostic significance of PTEN expression in esophageal squamous cell carcinoma from Linzhou city, a high incidence area of Northern China. Dis Esophagus. 2007;20(6):491–6.
</cite> [<a href="https://doi.org/10.1111/j.1442-2050.2007.00695.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17958724/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chang%20D,%20Wang%20TY,%20Li%20HC,%20Wei%20JC,%20Song%20JX.%20Prognostic%20significance%20of%20PTEN%20expression%20in%20esophageal%20squamous%20cell%20carcinoma%20from%20Linzhou%20city,%20a%20high%20incidence%20area%20of%20Northern%20China.%20Dis%20Esophagus.%202007;20(6):491%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Not applicable.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Journal of Translational Medicine are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12967-025-06914-4"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12967_2025_Article_6914.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12372372/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12372372/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372372%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372372/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12372372/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12372372/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40842012/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12372372/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40842012/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12372372/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12372372/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="xBSwK1Yfc7b1GFAwXbPdIU4D2t1X6C58IxU70WiocD5cioAMc88dV6gA0H8xMrWp">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
